
<html lang="en"     class="pb-page"  data-request-id="378aef5e-2e66-47d6-bc1f-65ceb31e3b92"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-6;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b00083;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair" /></meta><meta name="dc.Creator" content="Qi-Kun  Yin" /></meta><meta name="dc.Creator" content="Chen-Xi  Wang" /></meta><meta name="dc.Creator" content="Yu-Qing  Wang" /></meta><meta name="dc.Creator" content="Qian-Liang  Guo" /></meta><meta name="dc.Creator" content="Zi-Lin  Zhang" /></meta><meta name="dc.Creator" content="Tian-Miao  Ou" /></meta><meta name="dc.Creator" content="Shi-Liang  Huang" /></meta><meta name="dc.Creator" content="Ding  Li" /></meta><meta name="dc.Creator" content="Hong-Gen  Wang" /></meta><meta name="dc.Creator" content="Jia-Heng  Tan" /></meta><meta name="dc.Creator" content="Shuo-Bin  Chen" /></meta><meta name="dc.Creator" content="Zhi-Shu  Huang" /></meta><meta name="dc.Description" content="Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design. The Bloom syndrome protein (BLM) is a 3′–5′ DNA helicase t..." /></meta><meta name="Description" content="Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design. The Bloom syndrome protein (BLM) is a 3′–5′ DNA helicase t..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 2, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00083" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00083" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00083" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00083" /></link>
        
    
    

<title>Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00083" /></meta><meta property="og:title" content="Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0011.jpeg" /></meta><meta property="og:description" content="Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design. The Bloom syndrome protein (BLM) is a 3′–5′ DNA helicase that performs an important role in HRR regulation. However, limited studies about BLM inhibitors and their biological effects have been reported. Here, we identified a class of isaindigotone derivatives as novel BLM inhibitors by synthesis, screening, and evaluating. Among them, compound 29 was found as an effective BLM inhibitor with a high binding affinity and good inhibitory effect on BLM. Cellular evaluation indicated that 29 effectively disrupted the recruitment of BLM at DNA double-strand break sites, promoted an accumulation of RAD51, and regulated the HRR process. Meanwhile, 29 significantly induced DNA damage responses, as well as apoptosis and proliferation arrest in cancer cells. Our finding provides a potential anticancer strategy based on interfering with BLM via small molecules." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00083"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00083">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00083&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00083&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00083&amp;href=/doi/10.1021/acs.jmedchem.9b00083" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3147-3162</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00069" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00073" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Qi-Kun Yin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qi-Kun Yin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qi-Kun++Yin">Qi-Kun Yin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chen-Xi Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chen-Xi Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chen-Xi++Wang">Chen-Xi Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu-Qing Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu-Qing Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu-Qing++Wang">Yu-Qing Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qian-Liang Guo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qian-Liang Guo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qian-Liang++Guo">Qian-Liang Guo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zi-Lin Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zi-Lin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zi-Lin++Zhang">Zi-Lin Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tian-Miao Ou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tian-Miao Ou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tian-Miao++Ou">Tian-Miao Ou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8176-4576" title="Orcid link">http://orcid.org/0000-0002-8176-4576</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shi-Liang Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shi-Liang Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shi-Liang++Huang">Shi-Liang Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ding Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ding Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ding++Li">Ding Li</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3581-7630" title="Orcid link">http://orcid.org/0000-0002-3581-7630</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong-Gen Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong-Gen Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong-Gen++Wang">Hong-Gen Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jia-Heng Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jia-Heng Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jia-Heng++Tan">Jia-Heng Tan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1612-7482" title="Orcid link">http://orcid.org/0000-0002-1612-7482</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuo-Bin Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuo-Bin Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#67040f0209140f055f270a060e0b49141e141249020312490409"><span class="__cf_email__" data-cfemail="84e7ece1eaf7ece6bcc4e9e5ede8aaf7fdf7f1aae1e0f1aae7ea">[email protected]</span></a>. Phone: 8620-39943068 (S.-B.C.).</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuo-Bin++Chen">Shuo-Bin Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9118-2185" title="Orcid link">http://orcid.org/0000-0001-9118-2185</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Zhi-Shu Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhi-Shu Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#92f1f7e1fae8e1d2fff3fbfebce1ebe1e7bcf7f6e7bcf1fc"><span class="__cf_email__" data-cfemail="95f6f0e6fdefe6d5f8f4fcf9bbe6ece6e0bbf0f1e0bbf6fb">[email protected]</span></a>. Phone, 8620-39943056 (Z.-S.H.).</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhi-Shu++Huang">Zhi-Shu Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6211-5482" title="Orcid link">http://orcid.org/0000-0002-6211-5482</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00083&amp;href=/doi/10.1021%2Facs.jmedchem.9b00083" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3147–3162</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 2, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 January 2019</li><li><span class="item_label"><b>Published</b> online</span>2 March 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00083" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00083</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3147%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DQi-Kun%2BYin%252C%2BChen-Xi%2BWang%252C%2BYu-Qing%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D6%26contentID%3Dacs.jmedchem.9b00083%26title%3DDiscovery%2Bof%2BIsaindigotone%2BDerivatives%2Bas%2BNovel%2BBloom%25E2%2580%2599s%2BSyndrome%2BProtein%2B%2528BLM%2529%2BHelicase%2BInhibitors%2BThat%2BDisrupt%2Bthe%2BBLM%252FDNA%2BInteractions%2Band%2BRegulate%2Bthe%2BHomologous%2BRecombination%2BRepair%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3162%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00083"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2148</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00083" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Qi-Kun&quot;,&quot;last_name&quot;:&quot;Yin&quot;},{&quot;first_name&quot;:&quot;Chen-Xi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yu-Qing&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Qian-Liang&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Zi-Lin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Tian-Miao&quot;,&quot;last_name&quot;:&quot;Ou&quot;},{&quot;first_name&quot;:&quot;Shi-Liang&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Ding&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Hong-Gen&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jia-Heng&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Shuo-Bin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Zhi-Shu&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;3147-3162&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00083&quot;},&quot;abstract&quot;:&quot;Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design. The Bloom syndrome protein (BLM) is a 3′–5′ DNA helicase that performs an important role in HRR regulation. However, limited studies about BLM inhibitors and their biological effects have been reported. Here, we identified a class of isaindigotone derivatives as novel BLM inhibitors by synthesis, screening, and evaluating. Among them, compound 29 was found as an effective BLM inhibitor with a high binding affinity and good inhibitory effect on BLM. Cellular evaluation indicated that 29 effectively disrupted the recruitment of BLM at DNA double-strand break sites, promoted an accumulation of RAD51, and regulated the HRR process. Meanwhile, 29 significantly induced DNA damage responses, as well as apoptosis and proliferation arrest in cancer cells. Our finding provides a potential anticancer strategy based on interfering with BLM via small molecules.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00083&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00083" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00083&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00083" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00083&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00083" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00083&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00083&amp;href=/doi/10.1021/acs.jmedchem.9b00083" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00083" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00083" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00083%26sid%3Dliteratum%253Aachs%26pmid%3D30827110%26genre%3Darticle%26aulast%3DYin%26date%3D2019%26atitle%3DDiscovery%2Bof%2BIsaindigotone%2BDerivatives%2Bas%2BNovel%2BBloom%25E2%2580%2599s%2BSyndrome%2BProtein%2B%2528BLM%2529%2BHelicase%2BInhibitors%2BThat%2BDisrupt%2Bthe%2BBLM%252FDNA%2BInteractions%2Band%2BRegulate%2Bthe%2BHomologous%2BRecombination%2BRepair%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D6%26spage%3D3147%26epage%3D3162%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291428" title="Physiology">Physiology</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/jmcmar.2019.62.issue-6/20190328/jmcmar.2019.62.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design. The Bloom syndrome protein (BLM) is a 3′–5′ DNA helicase that performs an important role in HRR regulation. However, limited studies about BLM inhibitors and their biological effects have been reported. Here, we identified a class of isaindigotone derivatives as novel BLM inhibitors by synthesis, screening, and evaluating. Among them, compound <b>29</b> was found as an effective BLM inhibitor with a high binding affinity and good inhibitory effect on BLM. Cellular evaluation indicated that <b>29</b> effectively disrupted the recruitment of BLM at DNA double-strand break sites, promoted an accumulation of RAD51, and regulated the HRR process. Meanwhile, <b>29</b> significantly induced DNA damage responses, as well as apoptosis and proliferation arrest in cancer cells. Our finding provides a potential anticancer strategy based on interfering with BLM via small molecules.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">DNA damage is generally caused by normal biological events and environmental factors, such as pollution and radiation. A single human cell experiences more than ten thousand DNA molecular lesions per day, such as DNA single-strand breaks, DNA double-strand breaks (DSBs), nucleotide mutations, and other lesions.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In response to such DNA damage, a series of DNA damage repair pathways are involved in to ensure genomic stability and prevent genomic diseases, and such pathways include base excision repair, nucleotide excision repair, mismatch repair, nonhomologous DNA end joining, and homologous recombination repair (HRR).<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> Compared with normal cells, cancer cells harbor prevalent mutations and deficiencies in repair pathways, and in some cases, they even activate harmful repair pathways to drive tumor progression.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> The replication stress and oxidative damage make repair systems more fragile and impressionable in cancer cells. Therefore, cancer cells are more reliant on certain repair pathways and sensitive to their specific inhibitors.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> These differences make intervening the process of the DNA damage repair pathways via small molecules an effective anticancer strategy. Impressively, several inhibitors aimed at repair pathways have already entered the clinical trial stage.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">HRR is performed by various DNA damage repair proteins, such as MRN protein complex, RPA, BRCA1, RAD51, Bloom syndrome protein (BLM), and so on.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This repair is a primary means for the damaged DSBs, which represent the most serious damage in eukaryotes. In HRR pathway, a corresponding DNA sequence from an undamaged sister chromatid is used as a template. Because of its accurate and integral repair, HRR plays an essential role in genome stability.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> In addition, dysregulation of HRR usually causes repair dysfunctions, harmful genome rearrangements, and loss of heterozygosity.<a onclick="showRef(event, 'ref7 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref12">(7,11,12)</a> Synthetic lethality originally means a genetic interaction between two individual viable mutations, resulting in a lethal phenotype.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Because of the common mutations and abnormalities of HRR pathways in cancer cells, Hartwell and colleagues proposed the application of synthetic lethality in anticancer therapy.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> To expand the spectrum and enhance the therapeutic effect of anticancer drugs, combination therapy with drugs which target different DNA damage repair pathways has become a hot and potential field in research. For instance, <b>RI-1</b> (RAD51 inhibitor) sensitizes cancer cells to chemotherapy drugs, and <b>NSC19630</b> (WRN helicase inhibitor) increases cellular sensitivity to DNA damage agents.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Therefore, targeting repair-related proteins in HRR process is considered as an alternative anticancer therapeutic approach in combination therapy, and the development of inhibitors that target novel proteins in the core HRR pathways becomes necessary and urgent.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a></div><div class="NLM_p">BLM, a 3′–5′ ATP-driven DNA helicase, can unwind multiple nucleic acid structures including duplexes, Holliday junctions, G-quadruplexes, and DNA displacement loops.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> As a member of the highly conserved RecQ helicase family (RECQ1, BLM, WRN, RECQ4, RECQ5), BLM is highly expressed in cancer cells and considered to play a significant role in recombination.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Studies on the biological functions in cells present a detailed characterization of BLM in each stage of the HRR process, for example, BLM displays a broad spectrum of activities that either negatively or positively regulate HRR: (i) it can facilitate the dissociation of the branch migration of Holliday junctions and the unwinding of intermediates during DNA synthesis to promote HRR<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and (ii) it can prevent recombinant related protein RAD51 binding to DNA strands to restrain the progress of repair.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In addition, recent research showed that the deficiency of BLM induced a proliferation arrest and damage susceptibility in cancer cells, meanwhile BLM had a potential synthetic lethal effect with other repair-related proteins.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> In brief, BLM is an attractive protein that has potential to be developed as an anticancer target. However, only <b>ML216</b> is reported as a BLM inhibitor, and the pharmacology study of this inhibitor on cancer cells is still not clear.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Thus, it is necessary and urgent to develop novel BLM inhibitors and explore the potential anticancer strategy by targeting BLM.</div><div class="NLM_p">To identify novel BLM inhibitors, we first performed a screening in our in-house small-molecule library and found that isaindigotone derivatives showed inhibitory effects on BLM. Interestingly, we had found several isaindigotone compounds could effectively disrupt the protein/DNA interactions via specific binding on the DNA binding pocket of NM23-H2, a transcriptional factor that can activate <i>c-MYC</i> transcription by recognizing the G-quadruplex in the promoter of the gene, in the previous research.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> On this basis, we conducted an isaindigotone derivatives library containing 20 newly synthetic compounds (<b>5–24</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and 35 reported compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S1</a>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In these new compounds, some closed-ring basic terminal groups, including <i>N</i>-methyl-piperazinyl, 4-fluorophenethyl, or 4-(pyridin-2-yl)-piperazinyl groups, were introduced at the 6-position, meanwhile the types of substituents at 4′-position were further expanded (such as isopropyl, isobutyl, and tertiary butyl groups) or the styryl portion was converted into more complicated aromatic structures (such as carbazolyl and benzofuranyl groups). Various experiments were carried out to make systematic evaluation for BLM inhibitors. Prominent compound <b>29</b> was then selected as the potential BLM inhibitor for further in-depth studies. Impressively, compound <b>29</b> exhibited remarkable activities on disrupting BLM from DSB sites, regulating the HRR process and inducing DNA damage response. Furthermore, this compound could also induce cell apoptosis and inhibit cell proliferation in human colon cancer HCT116 cells. These results revealed that intervening repair factors via chemical molecules to affect important damage repair pathways might be an effective anticancer strategy.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and construction of isaindigotone derivatives library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures of the Newly Synthesized Isaindigotone Derivatives (<b>5–24</b>)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0010.gif" alt="" id="gr10" /></img><div></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthetic pathway for the isaindigotone derivatives was depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. According to our previous reports, the intermediate <b>2</b> was attained by 2-amino-4,5-difluorobenzoic acid heated with pyrrolidin-2-one in the presence of POCl<sub>3</sub>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Next, the preparation of the key intermediates <b>3a–3h</b> began with N-nucleophiles substituting the 6-position fluorine atom of intermediate <b>2</b>, and meanwhile treatment of terephthalaldehyde or isophthalaldehyde with different substituted acetophenones produced the intermediates <b>4a–4d</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Scheme S1</a>). Finally, intermediates <b>3a–3h</b> reacted with various aldehydes via the Knoevenagel reaction and provided the target compounds <b>5–24</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Route of Isaindigotone Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) pyrrolidin-2-one, POCl<sub>3</sub>, reflux, 8 h; (b) R<sub>1</sub>NH<sub>2</sub>, 100 °C, 12 h; (c) aldehydes, DMF, TMSCl, 100 °C, 12–48 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Screening and Evaluation of BLM Helicase Inhibitors</h3><div class="NLM_p">To screen BLM inhibitors, DNA helicase unwinding assay was performed to determinate the inhibitory effects of compounds on BLM unwinding DNA. We purified a truncated structure BLM that retained helicase activity in vitro and selected a duplex-forked DNA as a substrate.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Meanwhile, the reported inhibitor, <b>ML216</b>, was used as a reference compound during the screening. Fifty-five isaindigotone derivatives containing 20 newly synthetic compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and 35 reported compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S1</a>) were conducted as an isaindigotone compounds library. The library was then simultaneously evaluated following the process below (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The screening was performed at decreased concentrations (50, 10, and 1.0 μM) of compounds for three steps. First, we evaluated the inhibitory effects of 55 isaindigotone derivatives at the concentration of 50 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S1</a>). Twenty-nine candidate compounds that exhibited over 80% inhibitory effects on helicase unwinding were then selected for second-round screening treated with 10 μM of the compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S1</a>). The results showed that there were 12 isaindigotone derivatives that exhibited more than 80% inhibitory effects at 10 μM. Next, these 12 compounds were further evaluated at the concentration of 1.0 μM. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, compound <b>29</b> showed a best inhibitory effect on BLM helicase among these compounds, and three compounds (5, 6, and 26) also showed better inhibitory effects than <b>ML216</b>.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification of BLM inhibitors. (A) Diagram of BLM inhibitors identification and structure of hit compound <b>29</b>. (B) Inhibitory effects on BLM unwinding at the concentration of 1.0 μM for 12 candidate compounds. (C) Expression level of γ-H2AX with the treatment of 12 candidate compounds in HCT116 cells for 24 h, the concentration of each compound used in the treatment was half of IC<sub>50</sub> value in MTT assay, β-actin was used as a control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">During the screening, some information about the structure–activity relationship was obtained and described below. First, the size and type of the amine side chains (R<sub>1</sub>) at 6-position obviously impacted the inhibitory effects on BLM helicase. For example, the introduction of open-ring basic terminal groups, such as a diethylamino group (<b>29</b>), showed a remarkable improvement in the inhibitory effect of BLM compared with the corresponding closed-ring basic terminal groups, such as piperidinyl and 4-fluorophenethyl groups (<b>12</b> and <b>15</b>). Second, the role of a single substituent (R<sub>4</sub>) at the 4′-position was also investigated. <b>29</b>, <b>5</b>, and <b>6</b>, which presented simple alkyl substituents (4′-isopropyl, 4′-<i>tert</i>-butyl and 4′-isobutyl group, respectively), exhibited better inhibitory effects than <b>27</b> and <b>28</b>, which presented sterically hindered groups (4′-morpholinyl and 4′-piperidinyl group, respectively). Then, we quantified the inhibitory effect of <b>29</b> on BLM helicase. The IC<sub>50</sub> value of <b>29</b> was 0.95 μM. Meanwhile, the IC<sub>50</sub> value of <b>ML216</b> was 2.15 μM, which was consistent with previous report (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S2</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In conclusion, we found that compound <b>29</b> had a significant inhibitory effect on BLM unwinding, stronger than that reported inhibitor <b>ML216</b>.</div><div class="NLM_p last">BLM was a critical repair factor in the HHR pathway, and the deficiency of BLM could induce DNA damage response and potential toxicity in cancer cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S3</a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> We then determined the cytotoxicity and γ-H2AX expression (a DNA damage marker) on HCT116 cells upon treatment of 12 candidate compounds to evaluate their anticancer activities. The methyl thiazolyl tetrazolium (MTT) assay results showed that most compounds had good cytotoxicity (IC<sub>50</sub> < 10 μM) for HCT116 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S2</a>). The cytotoxicity of compounds was also relative to their BLM inhibition activity. Among them, <b>29</b> induced remarkable increase of γ-H2AX level in cells, which suggested that <b>29</b> was more likely to act as a BLM inhibitor in cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). In conclusion, we selected <b>29</b> as a hit compound from the library for further study.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Disrupting Effect of Compound 29 on the BLM/DNA Interaction</h3><div class="NLM_p">Besides unwinding DNA structures, BLM could play roles in HRR through recruiting to DSB sites.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> To investigate disruption of compound on the binding of BLM with DNA, we first evaluated the binding between BLM and DNA by enzyme-linked immunosorbent assay (ELISA) and filter-binding assay. In the ELISA assay, biotin-labeled DNA was immobilized on a plate and incubated with purified BLM upon treatment of compounds. The unbound BLM was removed after rinsing, and the BLM-bound DNA was quantified using BLM-antibody with horse radish peroxidase. In filter-binding assay, the BLM/DNA complex was separated using vacuum filtration. The BLM-bound DNA would retain in the upper membrane, and the lower membrane would hold the rest. The unbound DNA was quantified by counting the gray level of the dots in the lower membranes. First, the EC<sub>50</sub> value of BLM to DNA was determined. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S4</a>, EC<sub>50</sub> values were determined as 105.0 and 133.8 nM in ELISA and filter-binding assay, respectively. We then evaluated the potential disrupting effect of 29 on BLM/DNA interaction by ELISA and filter-binding assays. Upon treatment of 29, the percentage of unbound DNA in both assays was increased in a dose-dependent manner, and the IC<sub>50</sub> value in ELISA was 3.95 μM, which indicated that <b>29</b> could effectively disrupt BLM binding to DNA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">S5A</a>). In addition, <b>29</b> showed a stronger ability than <b>ML216</b> in disruption of BLM/DNA interaction. Afterward, a pull-down experiment was performed to investigate the disrupting effect of <b>29</b> on cellular BLM. Biotin-labeled DNA was immobilized on the magnetic beads, coincubated with <b>29</b> and cell lysis solution for 1 h. Then, the quantity of BLM in both eluent and concentrate was detected using western blot. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, <b>29</b> could disrupt binding of cellular BLM to DNA with efficiency up to 84.3% at the concentration of 1.0 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S5B</a>). In brief, the BLM inhibitor <b>29</b> could not only inhibit the unwinding activity of BLM but also disrupt the BLM binding to DNA.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Disrupting effect of <b>29</b> on the interaction of BLM and DNA. (A) Filter-binding dots of BLM with DNA upon addition of <b>29</b>. (B) Dose-dependent disrupting of BLM/DNA binding upon addition of <b>29</b> and <b>ML216</b> using ELISA assay, the IC<sub>50</sub> value was fitted by Hill model. (C) Disrupting effect of <b>29</b> on cellular BLM binding to DNA determined by pull-down assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Regulation of Homologous Recombination by Compound <b>29</b> via Disrupting the Recruitment of BLM to DSB Sites</h3><div class="NLM_p">We then investigated the possible relation between cellular effect and BLM inhibition of <b>29</b>. BLM is considered as a regulatory factor involved in HRR pathway; therefore, we used the HRR reporter assay to evaluate the possible effect of <b>29</b> on the HRR level.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The HRR reporter consists of two mutant <i>green fluorescent protein</i> (GFP) genes, <i>SceGFP</i> and <i>iGFP</i>. The <i>SceGFP</i> is disrupted by an 18-bp restriction site for I-SceI endonuclease, and the <i>iGFP</i> is truncated at both 5′ and 3′ ends (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In the assay, a DSB site would first be generated by restrict cleavage of I-SceI nuclease on <i>SceGFP</i>. Second, the <i>SceGFP</i> could be repaired though the HRR pathway with the <i>iGFP</i> gene as a homologous template and turned into a functional <i>GFP</i> gene. Thus, a percentage of GFP expressed cells could represent the level of HRR. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, the level of HRR was upregulated in a <b>29</b>-dependent manner and increased up to threefold at the concentration of 1.0 μM, indicating that <b>29</b> could improve the level of HRR in cells. <b>ML216</b> also induced a similar effect, but the effective concentration (50 μM) was much higher than <b>29</b>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> To obtain more details about the connection between the cellular activity of <b>29</b> and BLM inhibition, we detected the HRR level in siBLM HCT116 cells where the BLM expression level was decreased by small interfering RNA (siRNA), and the efficiency of siRNA is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S3</a>. The results showed that the decrease of BLM could significantly upregulate the level of HRR (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). This finding was consistent with the previous report that the hyper recombination level was prevalent in BLM deficiency cells.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Then, we noticed that the HRR level in siBLM HCT116 cells was not altered upon treatment of <b>29</b>. This result suggested that <b>29</b> regulated the level of HRR probably because of the inhibition of BLM.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. HR regulation mediated by <b>29</b> via disrupting the interaction of BLM and DNA in cells. (A) Scheme of HRR and DSB model. (B) Relative HRR level in HCT116 cells after being treated with <b>29</b> and <b>ML216</b>. (C) Relative HRR level in HCT116 and siBLM HCT116 cells after being treated with <b>29</b>. The HRR level was quantified by the percentage of GFP expressed cells and normalized to untreated samples. (D) Recruitment of BLM to the indicated DSB sites on HRR (−) reporters in <b>29</b>-treated and <b>ML216</b>-treated HCT116 cells. (E) Recruitment of BLM to the indicated DSB sites on chromosomes in <b>29</b>-treated HCT116 cells. BLM ChIP efficiency were determined by qPCR using specific primers and normalized to untreated samples. The data in experiment were made in triplicate and expressed as the mean ± SD: (*) <i>P</i> < 0.05, (**) <i>P</i> < 0.01, and (***) <i>P</i> < 0.001 compared with untreated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">During the HRR process, many repair-related proteins including BLM are recruited to the DSB sites to perform repair functions.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Because <b>29</b> could disrupt the interaction of BLM and DNA in vitro, it might also be able to disrupt BLM recruiting to the DSB sites in cells. To verify this hypothesis, the HRR reporter with DSB sites was used as the DSB model to evaluate the recruitment of BLM. To validate the DSB model, the cleavage efficiency was quantified by fluorogenic quantitative PCR (qPCR) assay, and activation of damage response on DSB sites was confirmed by the expression level of γ-H2AX. The results showed that the cleavage efficiency reached approximately 60% and effective damage response (γ-H2AX) was observed after the cleavage (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S6A,B</a>). On the basis of the DSB model on HRR (−) reporter, the chromatin immunoprecipitation (ChIP) assay was applied to evaluate the potential disrupting of <b>29</b> toward BLM on the DSB sites. The binding effects of BLM to DNA in different ranges (−100 to +2000 bp) around the DSB sites are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D. BLM was recruited significantly near the DSB sites (in DNA fragments that −92 bp and +559 bp from the DSB sites). In addition, the recruitment of BLM on DSB sites was decreased significantly upon treatment of <b>29</b>, but not on treatment of <b>ML216</b>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> These results revealed that <b>29</b> might disrupt the interaction of BLM and the plasmid reporter in cells is more effective than the reported BLM inhibitor.</div><div class="NLM_p last">In addition, I-SceI was also applied to cleavage DSB sites on chromosomes and generated chromosomic DSBs.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Upon treatment of I-SceI alone, the expression level of γ-H2AX increased significantly, suggesting that I-SceI successfully induced DSBs on chromosomes (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S6C</a>). To further evaluate the disrupting effect of <b>29</b>, the I-SceI induced DSB sites were figured out, and the binding effects of BLM to the I-SceI induced DSB sites were determined (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figures S6D</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>E, respectively). Similar to the plasmid reporter, BLM was also recruited significantly near the DSB sites on chromosomes, and addition of <b>29</b> could effectively disrupt the recruitment. The data from these experiments indicated that <b>29</b> significantly disrupted the interaction of BLM and DNA near the DSBs sites both on the plasmid reporter and chromosomes. Hence, the transcription and translation levels of BLM were both upregulated upon treatment of <b>29</b>, suggesting that the decreased recruitment of BLM was not associated with its expression level but with the action of <b>29</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S7</a>). In brief, compound <b>29</b> could disrupt the interaction of BLM and DNA in cells.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Accumulation of RAD51 at DSB Sites Induced by Compound <b>29</b></h3><div class="NLM_p">RAD51, a single-strand DNA (ssDNA) binding protein, plays a crucial role in the HRR process. During the repair, RAD51 binds to ssDNA and forms a nucleoprotein filament to promote the progress of repair.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Recent research suggested that BLM could bind to ssDNA and prevent the formation of the RAD51-ssDNA filaments in vitro.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This evidence indicated us to consider whether <b>29</b> could affect the behavior of RAD51 by inhibiting BLM. First, RAD51 was purchased and its DNA binding ability was detected as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S8A</a>; the data revealed that RAD51 had a high binding ability to DNA substrate (EC<sub>50</sub> = 108 nM). Second, the DNA exchange activity assay was performed. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, the quantities of RAD51-ssDNA filaments decreased in the presence of BLM, but this decrease was recovered with the increasing concentration of <b>29</b>. These results indicated that <b>29</b> could promote RAD51 binding to the ssDNA in vitro. Furthermore, the ELISA assay was applied to discovery this promotion of RAD51 induced by <b>29</b>. Interestingly, the consequence revealed that <b>29</b> could not facilitate RAD51 binding to ssDNA in vitro (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S8B</a>). In this case, the promotion of RAD51 binding to DNA in vitro might be derived from the disruption of the BLM/DNA interaction caused by <b>29</b>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Accumulation of RAD51 at the DSB sites mediated by <b>29</b>. (A) <b>29</b> reversed the decrease of RAD51-ssDNA filaments induced by BLM in vitro. (B) Recruitment of RAD51 to the indicated DSB sites on HRR (−) reporters in <b>29</b>-treated and siBLM HCT116 cells, same as <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D. (C) Recruitment of RAD51 to the indicated DSB sites on chromosomes in <b>29</b>-treated HCT116 cells, same as <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E. (D) Distribution and colocalization of RAD51 and γ-H2AX in HCT116 cells treated with <b>29</b> for 24 h using the immunofluorescence assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of the DSB model, the ChIP assay was performed to verify whether <b>29</b> could promote RAD51 binding to the DSB sites at the cellular level. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, RAD51 was significantly recruited near the DSB sites in siBLM HCT116 cells, indicating that the deficiency of BLM could promote RAD51 recruiting to the DSB sites. In addition, an increasing recruitment of RAD51 was also observed in a <b>29</b>-dependent manner around the DSB sites on the plasmid reporter. Furthermore, similar results appeared around the DSB sites on chromosomes (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). On the basis of the above results, <b>29</b> could significantly alter the behavior of RAD51 and promote RAD51 binding to DNA chains around the DSBs sites both on the plasmid reporter and chromosomes.</div><div class="NLM_p last">Because <b>29</b> induced an increase in the level of γ-H2AX in cells, as well as the behavior of RAD51 is related to DNA damage response and repair as reported,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> we verified whether the occurrence of damage was related to the inhibition of BLM by <b>29</b>. The immunofluorescence assay was performed to detect the distribution and localization of RAD51 and DNA damages (DSBs were represented by γ-H2AX foci in cells), and then we observed an increasing RAD51 foci (green) accumulated on the γ-H2AX foci (red) in <b>29</b>-treated cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D), suggesting that <b>29</b> could induce DNA damage response and this damage might be associated with RAD51. Furthermore, the expression level of RAD51 was not significantly changed under the treatment with <b>29</b>, indicating that it was probably behavior not expression of RAD51 affecting this process (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S8C</a>). Taken together, the results revealed that <b>29</b> could induce DNA damage and promote the accumulation of RAD51 to the damage sites in cells, and this process might be related with the disrupting effect of <b>29</b> on BLM.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> DNA Damage Response and Apoptosis Induced by Compound 29 in Cells</h3><div class="NLM_p">Because compound <b>29</b> has a significant impact on the DNA damage, we evaluated its effect on the DNA damage pathway.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, the level of major damage sensors in damage response, such as p-ATM and p-ATR, was obviously upregulated with the treatment of <b>29</b>. Afterward, the expression of other damage-related proteins in ATM-CHK2-p53 and ATR-CHK1-p53 pathways was remarkably upregulated as the concentration of <b>29</b> increased. These results demonstrated that <b>29</b> could activate two DNA damage response pathways. The deficiency of BLM in prostate cancer cells from patients could induce apoptosis, which suggested us that <b>29</b> might also induce a cellular apoptosis.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, we found that the ratio of apoptosis cells (staining by Hoechst and PI) increased from 1.09 to 71.59% in a concentration-dependent fashion of <b>29</b> (from 0 to 2.0 μM). Further, the expression level of cleaved caspase-3, cleaved caspase-7, and cleaved PARP was clearly upregulated in <b>29</b>-treated cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Taken together, the BLM inhibitor <b>29</b> could significant activate DNA damage pathways and induce a p53-dependent apoptosis (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). The mechanism of <b>29</b> was also consistent with appearances in BLM-deficient cells, suggesting that the damage and apoptosis might be induced by the suppression of BLM via <b>29</b>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DNA damage response and apoptosis in HCT116 cells induced by <b>29</b>. (A) Expression level of major signaling proteins in response to DNA damage in HCT116 cells treated with <b>29</b> for 24 h, β-actin was used as a control. (B) Apoptosis evaluation of HCT116 cells treated with <b>29</b> for 48 h. (C) Expression level of apoptosis pathway-related proteins in HCT116 cells treated with <b>29</b> for 48 h, β-actin was used as a control. (D) Action pathway diagram of <b>29</b> in HCT116 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Evaluation of Proliferation Arrest and Combination Treatment by Compound <b>29</b> in Cells</h3><div class="NLM_p">Because the deficiency of BLM induced a proliferation arrest in cancer cells, we then verified whether <b>29</b> had the similar antiproliferation effect.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> First, we used a real-time cellular analysis to investigate the impact of BLM on cellular proliferation. The result showed an obvious proliferative inhibition in HCT116 cells but not in human fibroblast BJ cells treated with siBLM for 48 h, indicating that the absence of BLM inhibited the proliferation of cancer cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S9</a>). In this case, we evaluated the cytotoxicity of <b>29</b> in different cells and found that <b>29</b> showed a stronger antiproliferative effect in cancer cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S3</a>). Then, the expression level of BLM in different cells was detected using western blot, and the results showed that the cytotoxicity of <b>29</b> increased in a BLM dependent fashion (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S10</a>). To gain more details evidences, we further evaluated the cytotoxicity of <b>29</b> in siBLM HCT116 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, a significant proliferation arrest appeared in a dose-dependent manner of <b>29</b> in HCT116 cells; however, the growth of siBLM HCT116 cells was not inhibited with the same treatment of <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). Meanwhile, a long-term proliferation assay was performed and showed the similar results, indicating that the arrest of cell proliferation induced by <b>29</b> might be due to the inhibition of BLM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C,D).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Proliferation arrest and synthetic interaction induced by <b>29</b>. (A,B) Proliferative ability in HCT116 and siBLM HCT116 cells treated with <b>29</b> for 48 h. (C,D) Proliferative ability in HCT116 and siBLM HCT116 cells treated with <b>29</b> in a long-term proliferation assay. (E,F) Antiproliferative effect of <b>29</b> in combined therapy with a RAD51 inhibitor (<b>RI-1</b>) and a DNA damage agent (<b>Cisplatin</b>) in HCT116 cells. The data were derived from three experiments and expressed as the mean ± SEM: (*) <i>P</i> < 0.05, (**) <i>P</i> < 0.01, and (***) <i>P</i> < 0.001 compared with untreated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the previous research, Mao proposed that the decrease of BLM enhanced the chemotherapy sensitivity of cancer cells.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In this case, we selected <b>Cisplatin</b> (a traditional DNA cross-linking agent) for the combination treatment.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E, the results showed that HCT116 cells became more sensitive to <b>Cisplatin</b> in the presence of <b>29</b> (1.25 μM), and the antiproliferative effect of <b>Cisplatin</b> (100 μM) increased from 15.4 to 71.2%. In addition, <b>29</b> induced an obvious DNA damage, and the potential synergistic lethal effect of <b>29</b> is of interest. Recent research indicated that BLM deficiency cells increased cell death associated with RAD51 deficiency.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Herein, <b>RI-1</b>, a RAD51 inhibitor, was applied to evaluate the antiproliferative effect in the synthetic interaction.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F, a strong antiproliferative effect emerged under the combined administration of <b>29</b> and <b>RI-1</b>, such as the proliferative effect of <b>RI-1</b> (50 μM) in the combined medicine group increased up to fourfold higher than the simple medicine group.</div><div class="NLM_p last">Furthermore, the antiproliferative effects of <b>RI-1</b> and <b>Cisplatin</b> were also enhanced by siBLM treatment in HCT116 cells, suggesting that these sensitization effects in cancer cells were related to the decrease of BLM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S11A,B</a>). Then, <b>29</b> was added, and there were not significant differences between combined and simple medicine groups yet, indicating that the enhanced antiproliferative effects by <b>29</b> was due to the inhibition of BLM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S11C,D</a>). In brief, compound <b>29</b> could cause proliferation arrest, sensitize cancer cells to chemotherapeutic agents, and damage repair inhibitors.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding of Compound <b>29</b> to BLM Protein</h3><div class="NLM_p">Previous research showed that <b>29</b> effectively inhibited the activity of BLM and displayed great cellular activity. To further illuminate the interaction mode between <b>29</b> and BLM, binding studies were performed. First, an isothermal titration calorimetry assay (ITC) was applied to detect the binding affinity of <b>29</b> to BLM. The disassociated constant (<i>K</i><sub>D</sub>) value of compound <b>29</b> to BLM was determined as 1.81 μM, suggesting that <b>29</b> perfectly bound to BLM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding mode of <b>29</b> to BLM. (A) ITC curve and fitted affinity of <b>29</b> to wild-type BLM, the top panel showed the heat change upon ligand titration, the bottom panel showed the integrated data and ITC isotherm (solid line) fitted by a single-site binding model. (B) Predicted binding mode of <b>29</b> with BLM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>), the diagram showed the interaction between ligand <b>29</b> (cyan) and predicted residues (green). (C) Binding affinity of <b>29</b> to wild-type and mutant BLM using ITC assay. “N.D.” in the table meant the <i>K</i><sub>D</sub> value was not obtained by fitting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The active core of BLM consists of three functional domains: helicase core domain capable of ATP hydrolysis, RecQ C-terminal domain (RQC) for DNA binding, and HRDC domain with unwinding activity.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> To further evaluate the binding sites, we performed a molecular docking simulation of <b>29</b> with all these domains in BLM based on the reported X-ray structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>) using Schrodinger software.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S12</a>, <b>29</b> preferred to bind to the 3′-tailed duplex DNA binding pocket of the RQC domain. To figure out the reliable binding geometry of <b>29</b> with BLM, molecular dynamics (MD) studies were then carried out.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The top-ranked predict binding mode in MD simulation is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B. In this binding mode, <b>29</b> had π–π stacking interactions with Tyr995 and His996, hydrophobic interactions with Met1111 and Ile1168,<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> and hydrogen-bonding interactions with Glu1143, implying that these amino acids might be the crucial residues in <b>29</b> binding to BLM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S4</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S12</a>). In addition, slight conformation changes of the DNA binding pocket were observed in the presence of <b>29</b> in the MD simulations (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Table S5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S13</a>). To further confirm this binding mode, the binding affinity of compound <b>29</b> with five single-point amino acid mutants (Y995A, H996A, M1111A, E1143A, and I1168A) was determined by ITC. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">S14</a>, the binding constants of <b>29</b> for mutants were all beyond that of <b>29</b> with the wild-type BLM, indicating that these five amino acids were involved in the binding of <b>29</b> with BLM. In addition, His996 seemed most important for the binding because no binding between <b>29</b> and BLM occurred when it was mutated. Above all, this result was consistent with our predicted binding mode, in which <b>29</b> was bound to the pocket along the DNA binding pocket in the RQC domain. In addition, the binding mode explained that the quinazolinone portion of <b>29</b> had a dominant role in binding to BLM, meanwhile the amine side chains and styryl portion also had partial effects, and these findings roughly correspond to the results of the SAR analysis in preliminary screening. Moreover, the ELISA results (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S15</a>) showed that the disrupting effect of <b>29</b> significantly decreased in the five mutants compared with wild-type protein, further indicating that these residues were vital for the interaction between <b>29</b> and BLM. Additionally, no binding of <b>29</b> to DNA was observed using the ITC assay (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S16</a>). Furthermore, the ATP hydrolysis activity of BLM was evaluated upon treatment of <b>29</b>, and the result suggested that <b>29</b> had no effect on BLM using ATP in vitro (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S17</a>). Taken together, <b>29</b> might competitively bind to the DNA binding pocket in the RQC domain of BLM, disrupt BLM/DNA interaction, and inhibit the DNA unwinding activity of BLM.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82148" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82148" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we screened our in-house library and first found that isaindigotone derivatives had inhibitory effects of BLM. On this basis, we newly synthesized and screened a series of derivatives to explore an optimized compound and discovery the influence of structural diversity on the inhibitory effects of BLM. The results from SAR analysis indicated that long amine side chains (R<sub>1</sub>) at 6-position and simple alkyl substituents (R<sub>4</sub>) at 4-position might be the important structural units. After screening, we identified that compound <b>29</b> possessed the best inhibitory effect on BLM for further study.</div><div class="NLM_p">A panel of in vitro assays, including ELISA, filter-binding assay, and pull-down assay, were performed to support that <b>29</b> could disrupt the interaction of BLM and DNA in vitro. In cells, subsequent biological assays, including HRR reporter assay, ChIP, western blot, and immunofluorescence assay, were applied and confirmed that <b>29</b> altered the level of HRR, disrupted the interaction of BLM/DNA, induced DNA damage, and promoted the accumulation of RAD51 to the DSB sites. At present, BLM and RAD51 are considered to play crucial roles in repair of DSBs in cells. In this case, the hyper-recombination caused by <b>29</b> via inhibiting BLM might break the balance of repair and induce DNA damage response. In addition, <b>29</b> activated the DNA damage response pathways, induced cellular apoptosis, caused proliferation arrest, and enhanced the sensitizing effects of <b>RI-1</b> and <b>Cisplatin</b> in cancer cells. Compared with the reported inhibitor <b>ML216</b>, compound <b>29</b> showed a lower effective concentration in inducing DNA damage and stronger cell cytotoxicity to cancer cells. As both compounds could uptake into cells with high efficiency (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Figure S18</a>), the better disruption of <b>29</b> on BLM/DNA interaction both in vitro and in cells might be the reason of its better inhibitory effect on cancer cells.</div><div class="NLM_p">To further elucidate the interaction between <b>29</b> and BLM, we performed the ITC study and demonstrated that <b>29</b> had a high binding affinity to BLM. Molecular docking suggested that <b>29</b> was suitable to the DNA binding pocket in the RQC domain of BLM. The binding mode was confirmed by ITC studies of <b>29</b> with five single-point mutants with predict binding amino acids (Y995, H996, M1111, E1143, and I1168A). This binding mode might be valuable to design more effective BLM inhibitors in the feature.</div><div class="NLM_p last">In summary, we identified a novel BLM inhibitor, isaindigotone derivative <b>29</b>, and first determined fairly complete interaction studies and pharmacological evaluations of the present BLM inhibitor. The different cellular effects of <b>29</b> and <b>ML216</b> suggested that the disrupting ability of compounds on BLM/DNA interaction might be more important in impacting BLM functions in cells. On the basis of this, the identification of <b>29</b> provides an important tool in the research of BLM functions, and detailed evaluation also gives a new perspective on the development of BLM in anticancer treatment.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87033" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87033" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemical Synthesis</h3><div class="NLM_p last">All chemicals were purchased from commercial sources unless otherwise specified. All the solvents were of analytical reagent grade and were used without further purification. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded using tetramethylsilane (TMS) as the internal standard in DMSO-<i>d</i><sub>6</sub> or CDCl<sub>3</sub> with a Bruker BioSpin GmbH spectrometer at 400 and 100 MHz, respectively. Chemical shifts are expressed in parts per million downfield from TMS as an internal standard and coupling constants in hertz. Mass spectra (MS) were recorded on a Shimadzu LCMS-2010A instrument with an ESI or ACPI mass selective detector and high-resolution MS (HRMS) on Shimadzu LCMSITTOF. Flash column chromatography was performed with silica gel (200–300 mesh) purchased from Qingdao Haiyang Chemical Co. Ltd. The purities of synthesized compounds were confirmed to be higher than 95% by analytical HPLC performed with a dual pump Shimadzu LC-20AB system equipped with a Ultimate XB-C18 column (4.6 mm × 250 mm, 5 μm) and eluted with methanol/water (15:85) containing 0.1% TFA at a flow rate of 0.5 mL/min.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Procedure for the Synthesis of Intermediate 6,7-Difluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>2</b>)</h3><div class="NLM_p last">POCl<sub>3</sub> (45 mL) was carefully added to a solution of 2-amino-4,5-difluorobezoic acid (1 equiv) and pyrrolidin-2-one (2 equiv) at room temperature and then stirred at 120 °C for 4 h and monitored by TLC. The crude product was adsorbed on silica and purified by flash chromatography. Yield: 3.9 g, 64%, white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.02 (t, <i>J</i> = 9.2 Hz, 1H), 7.49–7.34 (m, 1H), 4.20 (t, <i>J</i> = 7.1 Hz, 2H), 3.17 (t, <i>J</i> = 7.8 Hz, 2H), 2.37–2.23 (m, 2H).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure for the Synthesis of Intermediates <b>3a–3h</b></h3><div class="NLM_p">The mixture of intermediate <b>2</b> (1 equiv) and aliphatic amines (2 equiv) stirring in a 38 mL sealed tube was heated at 100 °C for 12 h and monitored by TLC. After cooling to room temperature, the crude product was adsorbed on silica and purified by flash chromatography.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 6-((3-(Diethylamino)propyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3a</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3a</b> was obtained. Yield: 3.5 g, 74%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.72 (d, <i>J</i> = 11.7 Hz, 1H), 6.72 (s, 1H), 6.67 (d, <i>J</i> = 7.7 Hz, 1H), 4.19–4.12 (m, 2H), 3.31 (dd, <i>J</i> = 10.6, 5.8 Hz, 2H), 3.11 (t, <i>J</i> = 7.9 Hz, 2H), 2.63–2.59 (m, 2H), 2.55 (q, <i>J</i> = 7.1 Hz, 4H), 2.30–2.19 (m, 2H), 1.89–1.81 (m, 2H), 1.06 (t, <i>J</i> = 7.1 Hz, 6H).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 7-Fluoro-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3b</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3b</b> was obtained. Yield: 2.7 g, 67%, milk white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.67 (d, <i>J</i> = 11.7 Hz, 1H), 6.62 (d, <i>J</i> = 7.7 Hz, 1H), 4.13–4.05 (m, 2H), 3.72–3.67 (m, 2H), 3.66–3.61 (m, 2H), 3.25 (dd, <i>J</i> = 11.1, 5.8 Hz, 2H), 3.04 (t, <i>J</i> = 7.9 Hz, 4H), 2.51–2.46 (m, 3H), 2.24–2.12 (m, 4H), 1.86–1.75 (m, 2H).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 7-Fluoro-6-((3-(pyrrolidin-1-yl)propyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3c</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3c</b> was obtained. Yield: 2.3 g, 61%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.74 (d, <i>J</i> = 11.7 Hz, 1H), 6.70 (d, <i>J</i> = 7.7 Hz, 1H), 4.22–4.12 (m, 2H), 3.33 (dd, <i>J</i> = 11.1, 6.0 Hz, 2H), 3.13 (t, <i>J</i> = 7.9 Hz, 2H), 2.68 (t, <i>J</i> = 6.2 Hz, 2H), 2.56 (t, <i>J</i> = 5.9 Hz, 5H), 2.31–2.22 (m, 2H), 1.90 (dt, <i>J</i> = 12.4, 6.2 Hz, 2H), 1.85–1.80 (m, 4H).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 6-((2-(Dimethylamino)ethyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3d</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3d</b> was obtained. Yield: 2.1 g, 59%, milk white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.71 (d, <i>J</i> = 11.5 Hz, 1H), 6.61 (d, <i>J</i> = 7.4 Hz, 1H), 4.22–4.01 (m, 2H), 3.44 (dd, <i>J</i> = 12.6, 6.5 Hz, 2H), 3.14 (t, <i>J</i> = 7.8 Hz, 2H), 2.47 (t, <i>J</i> = 6.5 Hz, 4H), 2.18 (s, 6H).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 7-Fluoro-6-((2-(piperidin-1-yl)ethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3e</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3e</b> was obtained. Yield: 2.6 g, 79%, milk white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.75 (dd, <i>J</i> = 11.6, 4.6 Hz, 1H), 6.71 (d, <i>J</i> = 7.5 Hz, 1H), 5.32 (s, 1H), 4.16 (t, <i>J</i> = 7.0 Hz, 2H), 3.33–3.20 (m, 2H), 3.12 (dd, <i>J</i> = 9.6, 5.9 Hz, 2H), 2.64 (t, <i>J</i> = 5.6 Hz, 2H), 2.42 (s, 4H), 2.30–2.21 (m, 2H), 1.59 (d, <i>J</i> = 4.8 Hz, 4H), 1.46 (d, <i>J</i> = 3.3 Hz, 2H).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 7-Fluoro-6-(4-methylpiperazin-1-yl)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3f</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3f</b> was obtained. Yield: 1.8 g, 54%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41 (d, <i>J</i> = 12.7 Hz, 1H), 6.79 (d, <i>J</i> = 8.1 Hz, 1H), 3.43 (t, <i>J</i> = 8.2 Hz, 4H), 3.17–3.05 (m, 4H), 2.34 (t, <i>J</i> = 8.2 Hz, 4H), 2.31–2.22 (m, 2H), 2.21 (s, 3H).</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 7-Fluoro-6-(4-(pyridin-2-yl)piperazin-1-yl)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3g</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3g</b> was obtained. Yield: 3.6 g, 87%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.03 (dd, <i>J</i> = 12.0, 2.4 Hz, 1H), 7.54 (m, 1H), 7.42 (d, <i>J</i> = 12.7 Hz, 1H), 7.22 (m, 1H), 6.79 (d, <i>J</i> = 8.1 Hz, 1H), 6.72 (m, 1H), 3.63–3.45 (m, 8H), 3.22–3.02 (m, 4H), 2.36–2.18 (m, 2H).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 7-Fluoro-6-((4-fluorophenethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>3h</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>3h</b> was obtained. Yield: 2.2 g, 69%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.51 (d, <i>J</i> = 12.7 Hz, 1H), 7.30–7.09 (m, 4H), 6.74 (d, <i>J</i> = 7.9 Hz, 1H), 3.41 (m, 3H), 3.17–3.06 (m, 4H), 2.93 (t, <i>J</i> = 8.3 Hz, 2H), 2.35–2.15 (m, 2H).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedure for the Synthesis of Intermediates <b>4a–4d</b></h3><div class="NLM_p">The mixture of terephthalaldehyde or isophthalaldehyde (1 equiv) and different substituted acetophenones (2 equiv) in a 38 mL sealed tube was stirred at 0 °C for 2 h first and then at room temperature for another 4 h, which was monitored by TLC. The crude product was adsorbed on silica and purified by flash chromatography.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>E</i>)-4-(3-(4-Morpholinophenyl)-3-oxoprop-1-en-1-yl)benzaldehyde (<b>4a</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>4a</b> was obtained. Yield: 1.2 g, 49%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.05 (s, 1H), 8.02 (d, <i>J</i> = 9.0 Hz, 2H), 7.92 (d, <i>J</i> = 8.2 Hz, 2H), 7.80 (m, 3H), 7.66 (d, <i>J</i> = 15.7 Hz, 1H), 6.93 (d, <i>J</i> = 9.0 Hz, 2H), 3.92–3.83 (m, 4H), 3.40–3.32 (m, 4H).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>E</i>)-4-(3-(4-Fluorophenyl)-3-oxoprop-1-en-1-yl)benzaldehyde (<b>4b</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>4b</b> was obtained. Yield: 2.1 g, 55%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.88 (s, 1H), 8.05 (d, <i>J</i> = 24.2 Hz, 1H), 7.96–7.88 (m, 2H), 7.79–7.68 (m, 4H), 7.59 (d, <i>J</i> = 24.1 Hz, 1H), 7.53–7.42 (m, 2H).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>E</i>)-3-(3-(Benzo[<i>d</i>][1,3]dioxol-5-yl)-3-oxoprop-1-en-1-yl)benzaldehyde (<b>4c</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>4c</b> was obtained. Yield: 2.5 g, 59%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.87 (s, 1H), 8.11–7.82 (m, 4H), 7.69 (m, 1H), 7.58 (d, <i>J</i> = 24.2 Hz, 1H), 7.45 (t, <i>J</i> = 2.4 Hz, 1H), 7.33 (d, <i>J</i> = 2.5 Hz, 1H), 7.15 (d, <i>J</i> = 12.0 Hz, 1H), 6.05 (s, 2H).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>E</i>)-3-(3-(4-Morpholinophenyl)-3-oxoprop-1-en-1-yl)benzaldehyde (<b>4d</b>)</h4><div class="NLM_p last">According to the general procedure above, the intermediate <b>4d</b> was obtained. Yield: 1.9 g, 51%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.08 (s, 1H), 8.19–7.99 (m, 3H), 7.93–7.76 (m, 3H), 7.70–7.50 (m, 2H), 6.93 (d, <i>J</i> = 9.0 Hz, 2H), 3.95–3.77 (m, 4H), 3.43–3.29 (m, 4H).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> General Procedure for the Synthesis of Final Compounds <b>5–24</b></h3><div class="NLM_p">The mixture of intermediates <b>3a–3h</b> (1 equiv) and different aldehydes (1.2 equiv) was dissolved in DMF and catalyzed by TMSCl in a 38 mL sealed tube at 100 °C for 12–48 h and monitored by TLC. The crude product was adsorbed on silica and purified by flash chromatography.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>E</i>)-3-(4-(<i>tert</i>-Butyl)benzylidene)-6-((3-(diethylamino)propyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>5</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>5</b> was obtained. Yield: 115 mg, 38%, white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (m, 2H), 7.47 (q, <i>J</i> = 8.5 Hz, 4H), 6.77 (d, <i>J</i> = 7.7 Hz, 1H), 6.65 (s, 1H), 4.21 (t, <i>J</i> = 7.3 Hz, 2H), 3.34 (dd, <i>J</i> = 10.7, 5.6 Hz, 2H), 3.24 (m, 2H), 2.61 (t, <i>J</i> = 5.9 Hz, 2H), 2.55 (q, <i>J</i> = 7.1 Hz, 4H), 1.89–1.80 (m, 2H), 1.35 (s, 9H), 1.06 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.51 (d, <i>J</i> = 3.5 Hz), 155.33 (d, <i>J</i> = 1.6 Hz), 152.11, 151.94, 149.51, 148.79, 143.39 (d, <i>J</i> = 14.0 Hz), 132.94, 131.14, 129.56 (2C), 125.76 (2C), 109.64 (d, <i>J</i> = 20.9 Hz), 109.31 (d, <i>J</i> = 7.7 Hz), 105.51 (d, <i>J</i> = 4.1 Hz), 52.82, 46.90 (2C), 43.86, 43.70, 34.80, 31.19 (3C), 25.60, 25.26, 11.71 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 477.3024, calcd for C<sub>29</sub>H<sub>37</sub>N<sub>4</sub>OF, 477.3029. Purity: 100.0% by HPLC.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>E</i>)-6-((3-(Diethylamino)propyl)amino)-7-fluoro-3-(4-isobutylbenzylidene)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>6</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>6</b> was obtained. Yield: 193 mg, 47%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.76–7.70 (m, 2H), 7.48–7.42 (m, 2H), 7.23–7.17 (m, 2H), 6.79–6.74 (m, 1H), 6.63 (s, 1H), 4.20 (m, 2H), 3.34 (dd, <i>J</i> = 10.1, 5.1 Hz, 2H), 3.22 (m, 2H), 2.61 (t, <i>J</i> = 5.9 Hz, 2H), 2.53 (m, 6H), 1.95–1.81 (m, 3H), 1.06 (t, <i>J</i> = 7.1 Hz, 6H), 0.92 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.51 (d, <i>J</i> = 2.5 Hz), 155.33, 151.92, 149.50, 148.80, 143.37 (d, <i>J</i> = 14.1 Hz), 142.80, 133.22, 130.98, 129.62 (d, <i>J</i> = 11.4 Hz) (2C), 109.63 (d, <i>J</i> = 20.8 Hz), 109.30 (d, <i>J</i> = 7.6 Hz), 105.51 (d, <i>J</i> = 4.1 Hz), 52.80, 46.90 (2C), 45.28, 43.86, 43.67, 30.16, 25.63, 25.25, 22.37 (2C), 11.69 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 477.3024, calcd for C<sub>29</sub>H<sub>37</sub>N<sub>4</sub>OF, 477.3028. Purity: 100.0% by HPLC.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>E</i>)-3-Benzylidene-6-((3-(diethylamino)propyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>7</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>7</b> was obtained. Yield: 141 mg, 52%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.75 (dd, <i>J</i> = 7.1, 4.6 Hz, 2H), 7.54 (d, <i>J</i> = 7.6 Hz, 2H), 7.43 (t, <i>J</i> = 7.5 Hz, 2H), 7.35 (t, <i>J</i> = 7.3 Hz, 1H), 6.78 (s, 1H), 6.76 (s, 1H), 4.26–4.18 (m, 2H), 3.34 (dd, <i>J</i> = 10.8, 5.6 Hz, 2H), 3.28–3.20 (m, 2H), 2.65–2.59 (m, 2H), 2.55 (q, <i>J</i> = 7.1 Hz, 4H), 1.90–1.81 (m, 2H), 1.06 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.50 (d, <i>J</i> = 3.4 Hz), 155.17 (d, <i>J</i> = 1.7 Hz), 151.99, 149.57, 148.73, 143.51, 143.37, 135.67, 132.09, 129.65 (d, <i>J</i> = 5.8 Hz) (2C), 128.76 (d, <i>J</i> = 7.6 Hz) (2C), 109.66 (d, <i>J</i> = 20.8 Hz), 109.29 (d, <i>J</i> = 7.7 Hz), 105.46 (d, <i>J</i> = 4.2 Hz), 52.87, 46.85 (2C), 43.88, 43.76, 25.64, 25.08, 11.69 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 421.2398, calcd for C<sub>25</sub>H<sub>29</sub>N<sub>4</sub>OF, 421.2394. Purity: 99.9% by HPLC.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>E</i>)-7-Fluoro-3-(4-fluorobenzylidene)-6-((3-(pyrrolidin-1-yl)propyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>8</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>8</b> was obtained. Yield: 177 mg, 52%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.79–7.70 (m, 2H), 7.53 (dd, <i>J</i> = 8.3, 5.6 Hz, 2H), 7.13 (t, <i>J</i> = 8.5 Hz, 2H), 6.78 (d, <i>J</i> = 7.7 Hz, 1H), 6.36 (s, 1H), 4.25 (t, <i>J</i> = 7.1 Hz, 2H), 3.37 (t, <i>J</i> = 6.1 Hz, 2H), 3.23 (t, <i>J</i> = 5.9 Hz, 2H), 2.76 (t, <i>J</i> = 6.3 Hz, 2H), 2.66 (m, 4H), 2.00–1.92 (m, 2H), 1.86 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.94, 161.45, 160.48 (d, <i>J</i> = 3.3 Hz), 155.12, 150.71 (d, <i>J</i> = 244.3 Hz), 148.67, 143.38 (d, <i>J</i> = 14.0 Hz), 131.75 (dd, <i>J</i> = 32.2, 2.8 Hz), 131.47 (d, <i>J</i> = 8.3 Hz) (2C), 128.51, 115.98 (d, <i>J</i> = 21.7 Hz) (2C), 109.78 (d, <i>J</i> = 20.8 Hz), 109.45 (d, <i>J</i> = 7.6 Hz), 105.51 (d, <i>J</i> = 4.0 Hz), 54.63, 53.92 (2C), 43.88, 42.83, 26.40, 25.51, 23.45 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 437.2147, calcd for C<sub>25</sub>H<sub>26</sub>N<sub>4</sub>OF<sub>2</sub>, 437.2143. Purity: 99.3% by HPLC.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>E</i>)-3-Benzylidene-6-((2-(diethylamino)ethyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>9</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>9</b> was obtained. Yield: 140 mg, 55%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.16 (s, 1H), 7.82 (d, <i>J</i> = 11.0 Hz, 1H), 7.60 (d, <i>J</i> = 7.4 Hz, 2H), 7.47 (t, <i>J</i> = 7.3 Hz, 2H), 7.44–7.38 (m, 1H), 7.09 (s, 1H), 6.58 (s, 1H), 4.37–4.29 (m, 2H), 3.96–3.87 (m, 2H), 3.33 (m, 4H), 3.28–3.12 (m, 4H), 1.43 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (126 MHz, DMSO): δ 159.74 (d, <i>J</i> = 3.3 Hz), 155.70, 151.26, 149.09 (d, <i>J</i> = 61.3 Hz), 143.05 (d, <i>J</i> = 13.7 Hz), 135.86, 133.68, 130.08 (2C), 129.36 (2C), 129.16, 128.65, 109.56 (d, <i>J</i> = 20.4 Hz), 109.07 (d, <i>J</i> = 7.1 Hz), 106.12 (d, <i>J</i> = 3.9 Hz), 51.09, 46.91 (2C), 44.31, 40.77, 25.60, 12.34 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 407.2242, calcd for C<sub>24</sub>H<sub>27</sub>N<sub>4</sub>OF, 404.2237. Purity: 99.9% by HPLC.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>E</i>)-3-(4-(Diethylamino)benzylidene)-6-((2-(dimethylamino)ethyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>10</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>10</b> was obtained. Yield: 237 mg, 66%, testaceous solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.77 (d, <i>J</i> = 11.5 Hz, 1H), 7.68 (m, 1H), 7.45 (d, <i>J</i> = 8.3 Hz, 2H), 6.80 (d, <i>J</i> = 7.5 Hz, 1H), 6.70 (d, <i>J</i> = 8.2 Hz, 2H), 5.20 (s, 1H), 4.24 (t, <i>J</i> = 7.1 Hz, 2H), 3.42 (dd, <i>J</i> = 13.7, 6.8 Hz, 4H), 3.34 (m, 2H), 3.27–3.21 (m, 2H), 2.70 (m, 2H), 2.34 (s, 6H), 1.21 (t, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.69 (d, <i>J</i> = 3.3 Hz), 156.42 (d, <i>J</i> = 1.3 Hz), 151.48, 149.00 (d, <i>J</i> = 11.8 Hz), 148.04, 142.66 (d, <i>J</i> = 13.8 Hz), 131.80 (2C), 130.54, 125.42, 122.80, 111.31 (2C), 109.97 (d, <i>J</i> = 20.6 Hz), 109.71 (d, <i>J</i> = 7.5 Hz), 105.85 (d, <i>J</i> = 3.6 Hz), 57.31, 45.03 (2C), 44.43 (2C), 43.96, 40.03, 25.60, 12.63 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 450.2664, calcd for C<sub>26</sub>H<sub>32</sub>N<sub>5</sub>OF, 450.2652. Purity: 99.5% by HPLC.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>E</i>)-3-(4-(Benzyloxy)-2-hydroxybenzylidene)-7-fluoro-6-((2-(piperidin-1-yl)ethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>11</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>11</b> was obtained. Yield: 112 mg, 43%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (s, 1H), 7.76 (d, <i>J</i> = 11.3 Hz, 1H), 7.40 (m, 2H), 7.33–7.26 (m, 4H), 6.83 (d, <i>J</i> = 6.9 Hz, 1H), 6.55 (d, <i>J</i> = 7.4 Hz, 1H), 6.44 (s, 1H), 5.20 (s, 1H), 4.94 (s, 2H), 4.25–4.10 (m, 2H), 3.17 (m, 4H), 2.57 (m, 2H), 2.39 (s, 4H), 1.57 (m, 4H), 1.44 (s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.94, 151.61, 143.09 (d, <i>J</i> = 13.7 Hz), 136.59, 129.88, 128.52 (2C), 127.90, 127.10 (2C), 125.62, 116.92, 110.06 (d, <i>J</i> = 21.2 Hz), 107.60, 105.36, 102.94, 100.00, 70.01, 56.61, 54.27 (2C), 44.09, 39.53, 25.85 (2C), 25.53, 24.29. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 541.2609, calcd for C<sub>32</sub>H<sub>33</sub>N<sub>4</sub>O<sub>3</sub>F, 541.2591. Purity: 95.8% by HPLC.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>E</i>)-7-Fluoro-3-(4-isopropylbenzylidene)-6-((2-(piperidin-1-yl)ethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>12</b> was obtained. Yield: 77 mg, 32%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.76 (m, 2H), 7.48 (d, <i>J</i> = 8.1 Hz, 2H), 7.30 (d, <i>J</i> = 8.1 Hz, 2H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 5.31 (s, 1H), 4.23 (t, <i>J</i> = 7.2 Hz, 2H), 3.34–3.19 (m, 4H), 3.00–2.87 (m, 1H), 2.67 (t, <i>J</i> = 6.0 Hz, 2H), 2.44 (m, 4H), 1.61 (dt, <i>J</i> = 10.7, 5.4 Hz, 4H), 1.52–1.40 (m, 2H), 1.28 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.55 (d, <i>J</i> = 3.1 Hz), 155.50, 151.79, 149.97, 149.37, 148.68, 143.00 (d, <i>J</i> = 13.5 Hz), 133.27, 130.90, 129.86 (2C), 126.95 (2C), 109.96, 109.72 (d, <i>J</i> = 6.1 Hz), 106.13 (d, <i>J</i> = 3.6 Hz), 56.64, 54.27 (2C), 43.94, 39.46, 34.04, 25.96 (2C), 25.61, 24.38, 23.82 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 461.2711, calcd for C<sub>28</sub>H<sub>33</sub>N<sub>4</sub>OF, 461.2703. Purity: 99.8% by HPLC.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>E</i>)-3-(4-(<i>tert</i>-Butyl)benzylidene)-7-fluoro-6-((2-(piperidin-1-yl)ethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>13</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>13</b> was obtained. Yield: 164 mg, 50%, milk white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.77 (m, 2H), 7.48 (dd, <i>J</i> = 19.8, 8.5 Hz, 4H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 5.33 (s, 1H), 4.29–4.19 (m, 2H), 3.28 (m, 4H), 2.68 (t, <i>J</i> = 5.8 Hz, 2H), 2.46 (s, 4H), 1.68–1.57 (m, 4H), 1.48 (s, 2H), 1.35 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 160.53 (d, <i>J</i> = 3.3 Hz), 155.51 (d, <i>J</i> = 1.8 Hz), 152.21, 151.56, 149.62, 148.69, 142.97 (d, <i>J</i> = 13.6 Hz), 132.89, 131.04, 129.74, 129.61 (2C), 125.82 (2C), 110.02–109.76 (m), 106.16 (d, <i>J</i> = 3.7 Hz), 56.61 (2C), 54.26 (2C), 43.94, 39.41, 34.84, 31.21 (3C), 25.91, 25.61, 24.35. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 475.2868, calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>OF, 475.2874. Purity: 100.0% by HPLC.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>E</i>)-7-Fluoro-3-(4-isobutylbenzylidene)-6-((2-(piperidin-1-yl)ethyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>14</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>14</b> was obtained. Yield: 198 mg, 71%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.81–7.72 (m, 2H), 7.47 (d, <i>J</i> = 8.1 Hz, 2H), 7.22 (d, <i>J</i> = 8.1 Hz, 2H), 6.80 (d, <i>J</i> = 7.7 Hz, 1H), 5.52 (s, 1H), 4.29–4.20 (m, 2H), 3.43 (m, 2H), 3.30–3.21 (m, 2H), 2.81 (m, 2H), 2.61 (s, 4H), 2.52 (d, <i>J</i> = 7.2 Hz, 2H), 2.00–1.81 (m, 1H), 1.73 (m, 4H), 1.52 (m, 2H), 0.93 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.50 (d, <i>J</i> = 3.3 Hz), 155.56, 151.80, 149.37, 148.65, 142.99, 142.66 (d, <i>J</i> = 13.9 Hz), 133.14, 130.86, 129.96 (2C), 129.63 (2C), 110.11 (d, <i>J</i> = 8.9 Hz), 109.97 (d, <i>J</i> = 3.1 Hz), 106.06 (d, <i>J</i> = 3.7 Hz), 56.52, 54.29 (2C), 45.29, 43.95, 39.08, 30.22 (2C), 25.65, 25.23, 23.85, 22.41 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 475.2868, calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>OF, 475.2858. Purity: 100.0% by HPLC.</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>E</i>)-7-Fluoro-6-((4-fluorophenethyl)amino)-3-(4-isopropylbenzylidene)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>15</b> was obtained. Yield: 103 mg, 47%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.90–7.74 (m, 2H), 7.51 (m, 2H), 7.31 (m, 2H), 7.21 (m, 2H), 7.03 (t, <i>J</i> = 8.0 Hz, 2H), 6.96 (s, 1H), 4.56 (s, 1H), 4.26 (s, 2H), 3.55 (s, 2H), 3.28 (s, 2H), 3.09–2.86 (m, 3H), 1.28 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.00, 160.57, 160.30, 155.78, 151.60, 150.25, 149.18, 142.41 (d, <i>J</i> = 13.3 Hz), 134.13 (d, <i>J</i> = 3.1 Hz), 133.16, 130.24, 130.16, 130.07 (2C), 127.04 (2C), 115.72 (2C), 115.51 (2C), 110.16 (d, <i>J</i> = 21.2 Hz), 105.92, 44.37, 44.25, 34.24, 34.07, 25.72, 23.83 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 472.2195, calcd for C<sub>29</sub>H<sub>27</sub>N<sub>3</sub>OF<sub>2</sub>, 472.2182. Purity: 97.1% by HPLC.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>E</i>)-3-(4-(<i>tert</i>-Butyl)benzylidene)-7-fluoro-6-(4-methylpiperazin-1-yl)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>16</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>16</b> was obtained. Yield: 109 mg, 57%, milk white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.83 (d, <i>J</i> = 12.9 Hz, 1H), 7.77 (m, 1H), 7.49 (q, <i>J</i> = 8.5 Hz, 4H), 7.17 (d, <i>J</i> = 7.8 Hz, 1H), 4.26 (t, <i>J</i> = 7.2 Hz, 2H), 3.30 (m, 6H), 2.66 (m, 4H), 2.39 (s, 3H), 1.35 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 160.32 (d, <i>J</i> = 3.0 Hz), 155.70 (d, <i>J</i> = 1.8 Hz), 153.98 (d, <i>J</i> = 249.1 Hz), 152.42, 147.74, 146.24 (d, <i>J</i> = 10.0 Hz), 132.74, 130.67, 130.22, 129.67 (2C), 125.87 (2C), 115.19 (d, <i>J</i> = 3.3 Hz), 114.51 (d, <i>J</i> = 8.4 Hz), 112.01 (d, <i>J</i> = 23.8 Hz), 54.91 (2C), 49.88, 46.08 (2C), 44.03, 34.87, 31.20 (3C), 25.57. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 447.2555, calcd for C<sub>27</sub>H<sub>31</sub>N<sub>4</sub>OF, 447.2553. Purity: 100.0% by HPLC.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>E</i>)-7-Fluoro-3-(4-isopropylbenzylidene)-6-(4-(pyridin-2-yl)piperazin-1-yl)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>17</b> was obtained. Yield: 181 mg, 62%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.23 (dd, <i>J</i> = 4.9, 1.1 Hz, 1H), 7.85 (dd, <i>J</i> = 12.8, 1.4 Hz, 1H), 7.77 (m, 1H), 7.56–7.45 (m, 3H), 7.31 (d, <i>J</i> = 8.1 Hz, 2H), 7.19 (d, <i>J</i> = 7.8 Hz, 1H), 6.73–6.64 (m, 2H), 4.25 (t, <i>J</i> = 7.2 Hz, 2H), 3.78–3.69 (m, 4H), 3.43–3.37 (m, 4H), 3.27 (t, <i>J</i> = 6.8 Hz, 2H), 3.02–2.87 (m, 1H), 1.28 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.27 (d, <i>J</i> = 2.7 Hz), 159.36, 155.74 (d, <i>J</i> = 1.8 Hz), 155.18, 152.70, 150.18, 147.87 (d, <i>J</i> = 20.9 Hz), 146.16 (d, <i>J</i> = 10.3 Hz), 137.65, 133.14, 130.47 (d, <i>J</i> = 18.9 Hz), 129.91 (2C), 126.99 (2C), 115.22 (d, <i>J</i> = 3.0 Hz), 114.68 (d, <i>J</i> = 8.3 Hz), 113.78, 112.20, 111.97, 107.22, 49.88 (d, <i>J</i> = 4.3 Hz), 45.26 (2C), 44.01, 34.04 (2C), 25.57, 23.78 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 496.2507, calcd for C<sub>30</sub>H<sub>30</sub>N<sub>5</sub>OF, 496.2517. Purity: 99.3% by HPLC.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 7-Fluoro-3-((<i>E</i>)-4-((<i>E</i>)-3-(4-morpholinophenyl)-3-oxoprop-1-en-1-yl)benzylidene)-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>18</b> was obtained. Yield: 62 mg, 29%, yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.01 (d, <i>J</i> = 8.9 Hz, 2H), 7.78 (m, 3H), 7.69 (d, <i>J</i> = 8.2 Hz, 2H), 7.59 (m, 3H), 6.92 (d, <i>J</i> = 8.9 Hz, 2H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 6.48 (s, 1H), 4.26 (t, <i>J</i> = 7.1 Hz, 2H), 3.92–3.83 (m, 4H), 3.84–3.75 (m, 4H), 3.43–3.32 (m, 6H), 3.31–3.25 (m, 2H), 2.58 (m, <i>J</i> = 16.8, 11.1 Hz, 6H), 1.92 (m, <i>J</i> = 11.5, 5.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 187.75, 160.43 (d, <i>J</i> = 3.6 Hz), 155.06 (d, <i>J</i> = 1.9 Hz), 154.27, 152.05, 149.62, 148.69, 143.32 (d, <i>J</i> = 13.9 Hz), 142.14, 137.31, 135.58, 133.25, 130.67, 130.13, 128.77 (d, <i>J</i> = 18.8 Hz), 128.71, 122.67, 113.40, 109.83 (d, <i>J</i> = 21.1 Hz), 109.61 (d, <i>J</i> = 7.6 Hz), 105.65 (d, <i>J</i> = 3.8 Hz), 66.84 (2C), 66.57 (2C), 58.05, 53.77 (2C), 47.47 (2C), 43.93, 43.37, 25.79, 24.03. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 650.3137, calcd for C<sub>38</sub>H<sub>40</sub>N<sub>5</sub>O<sub>4</sub>F, 650.3139. Purity: 98.8% by HPLC.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 7-Fluoro-3-((<i>E</i>)-4-((<i>E</i>)-3-(4-fluorophenyl)-3-oxoprop-1-en-1-yl)benzylidene)-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>19</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>19</b> was obtained. Yield: 114 mg, 40%, yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.08 (m, 2H), 7.89–7.67 (m, 5H), 7.57 (m, 3H), 7.19 (t, <i>J</i> = 7.9 Hz, 2H), 6.81 (d, <i>J</i> = 7.3 Hz, 1H), 6.52 (s, 1H), 4.33–4.20 (m, 2H), 3.80 (s, 4H), 3.38 (m, 2H), 3.30 (m, 2H), 2.58 (m, 6H), 1.93 (s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 188.55, 166.97, 164.44, 160.42 (d, <i>J</i> = 2.9 Hz), 155.01, 148.66, 143.89 (2C), 137.82, 134.99, 134.43 (d, <i>J</i> = 2.7 Hz), 133.63, 131.13 (d, <i>J</i> = 9.3 Hz), 130.19 (2C), 128.91 (2C), 128.71, 122.20 (2C), 115.96, 115.74, 109.85 (d, <i>J</i> = 21.2 Hz), 109.61 (d, <i>J</i> = 6.9 Hz), 105.63 (d, <i>J</i> = 3.7 Hz), 66.81 (2C), 58.06, 53.75 (2C), 43.94, 43.37, 25.83, 23.95. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 583.2515, calcd for C<sub>34</sub>H<sub>32</sub>N<sub>4</sub>O<sub>3</sub>F<sub>2</sub>, 583.2514. Purity: 97.7% by HPLC.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 3-((<i>E</i>)-3-((<i>E</i>)-3-(Benzo[<i>d</i>][1,3]dioxol-5-yl)-3-oxoprop-1-en-1-yl)benzylidene)-7-fluoro-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>20</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>20</b> was obtained. Yield: 136 mg, 37%, yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO): δ 8.04 (s, 1H), 7.93 (d, <i>J</i> = 19.6 Hz, 1H), 7.85 (m, 3H), 7.74 (d, <i>J</i> = 15.6 Hz, 1H), 7.68 (d, <i>J</i> = 7.6 Hz, 1H), 7.62 (m, 2H), 7.56 (t, <i>J</i> = 7.7 Hz, 1H), 7.06 (d, <i>J</i> = 8.1 Hz, 1H), 6.93 (d, <i>J</i> = 7.8 Hz, 1H), 6.14 (s, 2H), 4.16 (t, <i>J</i> = 7.0 Hz, 2H), 3.93 (m, 4H), 3.38 (t, <i>J</i> = 6.5 Hz, 2H), 3.34 (m, 2H), 3.24–3.15 (m, 2H), 2.89 (s, 2H), 2.74 (s, 2H), 2.17–2.08 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO): δ 187.72, 162.74, 159.58, 155.83, 152.08, 148.55, 143.21, 143.04 (d, <i>J</i> = 13.9 Hz), 136.56, 136.08, 134.25, 132.93, 131.34, 130.42, 129.87, 129.22, 129.07, 125.49, 123.68, 109.85 (d, <i>J</i> = 20.9 Hz), 108.58, 108.45, 105.85 (d, <i>J</i> = 3.8 Hz), 102.50, 63.61 (2C), 54.63, 51.64 (2C), 44.57, 36.17, 31.31, 25.60, 22.70. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 609.2508, calcd for C<sub>35</sub>H<sub>33</sub>N<sub>4</sub>O<sub>5</sub>F, 609.2503. Purity: 99.3% by HPLC.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 7-Fluoro-3-((<i>E</i>)-3-((<i>E</i>)-3-(4-morpholinophenyl)-3-oxoprop-1-en-1-yl)benzylidene)-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>21</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>21</b> was obtained. Yield: 123 mg, 34%, yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.02 (d, <i>J</i> = 8.8 Hz, 2H), 7.79 (m, 4H), 7.68–7.55 (m, 3H), 7.49 (t, <i>J</i> = 7.6 Hz, 1H), 6.93 (d, <i>J</i> = 8.9 Hz, 2H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 6.53 (s, 1H), 4.27 (t, <i>J</i> = 7.1 Hz, 2H), 3.92–3.84 (m, 4H), 3.84–3.76 (m, 4H), 3.43–3.33 (m, 6H), 3.30 (m, 2H), 2.58 (m, 6H), 1.96–1.87 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 187.82, 160.47 (d, <i>J</i> = 3.1 Hz), 155.05 (d, <i>J</i> = 1.6 Hz), 154.30, 150.82 (d, <i>J</i> = 244.3 Hz), 148.67, 143.35 (d, <i>J</i> = 13.9 Hz), 142.55, 136.33, 135.93, 133.02, 130.87, 130.72 (2C), 129.81, 129.40, 129.00, 128.58, 128.05, 122.69, 113.41 (2C), 109.82 (d, <i>J</i> = 21.0 Hz), 109.57 (d, <i>J</i> = 7.5 Hz), 105.62 (d, <i>J</i> = 4.0 Hz), 66.86 (2C), 66.58 (2C), 58.09, 53.77 (2C), 47.46 (2C), 43.96, 43.42, 25.67, 23.98. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 650.3137, calcd for C<sub>38</sub>H<sub>40</sub>N<sub>5</sub>O<sub>4</sub>F, 650.3153. Purity: 99.1% by HPLC.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>E</i>)-3-(Benzofuran-2-ylmethylene)-6-((3-(diethylamino)propyl)amino)-7-fluoro-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>22</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>22</b> was obtained. Yield: 230 mg, 46%, yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.76 (d, <i>J</i> = 11.7 Hz, 1H), 7.63 (t, <i>J</i> = 2.7 Hz, 1H), 7.60 (d, <i>J</i> = 7.7 Hz, 1H), 7.49 (d, <i>J</i> = 8.2 Hz, 1H), 7.36–7.30 (m, 1H), 7.25 (m, 1H), 6.92 (s, 1H), 6.76 (d, <i>J</i> = 7.7 Hz, 1H), 6.71 (s, 1H), 4.30–4.23 (m, 2H), 3.47 (td, <i>J</i> = 7.9, 2.6 Hz, 2H), 3.34 (dd, <i>J</i> = 10.3, 5.7 Hz, 2H), 2.62 (t, <i>J</i> = 5.9 Hz, 2H), 2.56 (q, <i>J</i> = 7.1 Hz, 4H), 1.87 (m, 2H), 1.07 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.48 (d, <i>J</i> = 3.3 Hz), 155.61, 154.58 (d, <i>J</i> = 1.9 Hz), 153.87, 150.90 (d, <i>J</i> = 244.5 Hz), 148.69, 143.46 (d, <i>J</i> = 14.0 Hz), 133.24, 128.41, 125.72, 123.35, 121.51, 116.62, 111.29, 110.09, 109.75 (d, <i>J</i> = 21.0 Hz), 109.40 (d, <i>J</i> = 7.6 Hz), 105.48 (d, <i>J</i> = 4.1 Hz), 52.79, 46.87 (2C), 43.94, 43.67, 25.62, 25.12, 11.64 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 461.2347, calcd for C<sub>27</sub>H<sub>29</sub>N<sub>4</sub>O<sub>2</sub>F, 461.2344. Purity: 97.6% by HPLC.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>E</i>)-3-((9-Ethyl-9H-carbazol-3-yl)methylene)-7-fluoro-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>23</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>23</b> was obtained. Yield: 101 mg, 40%, pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.26 (s, 1H), 8.10 (d, <i>J</i> = 7.7 Hz, 1H), 7.96 (s, 1H), 7.76 (d, <i>J</i> = 11.7 Hz, 1H), 7.67 (d, <i>J</i> = 8.3 Hz, 1H), 7.50 (t, <i>J</i> = 7.6 Hz, 1H), 7.43 (m, 2H), 7.26 (m, 1H), 6.80 (d, <i>J</i> = 7.7 Hz, 1H), 6.41 (s, 1H), 4.37 (q, <i>J</i> = 7.1 Hz, 2H), 4.26 (t, <i>J</i> = 7.2 Hz, 2H), 3.80 (s, 4H), 3.36 (m, 4H), 2.66–2.48 (m, 6H), 1.98–1.88 (m, 2H), 1.46 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.63 (d, <i>J</i> = 3.3 Hz), 155.92, 151.76, 149.35, 148.93, 143.17 (d, <i>J</i> = 13.9 Hz), 140.44, 140.03, 131.04, 128.32, 128.06, 126.79, 126.27, 123.40, 122.82, 122.08, 120.51, 119.50, 109.75 (d, <i>J</i> = 20.8 Hz), 109.41 (d, <i>J</i> = 7.6 Hz), 108.82 (d, <i>J</i> = 8.4 Hz), 105.51 (d, <i>J</i> = 3.9 Hz), 66.82, 58.04, 53.75 (2C), 43.99, 43.33, 37.75, 25.78, 24.04, 13.89 (2C). ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 552.2769, calcd for C<sub>33</sub>H<sub>34</sub>N<sub>5</sub>O<sub>2</sub>F, 552.2757. Purity: 96.1% by HPLC.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>E</i>)-3-(Benzofuran-2-ylmethylene)-7-fluoro-6-((3-morpholinopropyl)amino)-2,3-dihydropyrrolo[2,1-<i>b</i>]quinazolin-9(1<i>H</i>)-one (<b>24</b>)</h4><div class="NLM_p last">According to the general procedure above, the compound <b>24</b> was obtained. Yield: 211 mg, 63%, yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.80 (d, <i>J</i> = 11.7 Hz, 1H), 7.65 (t, <i>J</i> = 2.7 Hz, 1H), 7.62 (d, <i>J</i> = 7.6 Hz, 1H), 7.51 (d, <i>J</i> = 8.2 Hz, 1H), 7.39–7.33 (m, 1H), 7.29 (m, 1H), 6.94 (s, 1H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 6.49 (s, 1H), 4.33–4.25 (m, 2H), 3.81 (t, <i>J</i> = 4.3 Hz, 4H), 3.55–3.45 (m, 2H), 3.39 (t, <i>J</i> = 5.6 Hz, 2H), 2.60 (m, 6H), 1.93 (dt, <i>J</i> = 11.4, 5.6 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 160.45 (d, <i>J</i> = 3.2 Hz), 155.62, 154.68 (d, <i>J</i> = 2.0 Hz), 153.82, 150.85 (d, <i>J</i> = 244.2 Hz), 148.68, 143.28 (d, <i>J</i> = 14.0 Hz), 133.18, 129.87 (d, <i>J</i> = 206.9 Hz), 128.40, 125.76, 123.37, 121.52, 116.68, 111.30, 110.16, 109.86 (d, <i>J</i> = 21.0 Hz), 105.62 (d, <i>J</i> = 3.9 Hz), 66.83 (2C), 58.04, 53.76 (2C), 43.96, 43.35, 25.61, 24.02. ESI-HRMS [M + H]<sup>+</sup><i>m</i>/<i>z</i>: 475.2140, calcd for C<sub>27</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub>F, 475.2154. Purity: 97.0% by HPLC.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> DNA Preparation</h3><div class="NLM_p last">All oligonucleotides used in experiments were shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">Tables S6 and S7</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Plasmid Construction</h3><div class="NLM_p last">A truncated protein BLM (BLM<sub>642–1296</sub>) protein expression plasmid was a gift from Tan.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> BLM were constructed by digesting and inserting the corresponding cDNA into the pET28b plasmid using the Bam HI and XhoI restriction enzyme. In the HRR reporter assay, pDRGFP (HRR (+) reporter) was a gift from Maria Jasin (Addgene plasmid # 26475) and pCBASceI was a gift from Maria Jasin (Addgene plasmid # 26477). To conduct the HRR (−) reporter, the reverse GFP template was digested using the HindIII restriction enzyme, and the recombinant sequences were spliced and inserted into the PGK-puro plasmid.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Protein Preparation</h3><div class="NLM_p last">In brief, BLM and mutant variants were cloned into the pET28b plasmids containing the His-tag and transformed into <i>E. coli</i> strain BL21 (DE3) cells. The positive-clones were enlarged overnight at 37 °C and 0.1 μM. Isopropyl β-D-1-thiogalactopyranoside (IPTG) were added to induce protein expression at 16 °C for 24 h. Then, cells were collected and lysed using lysozyme (MP Biomedicals). The purification of protein was harvested by HisTrap HP (Roche), and the purified protein was stored in buffer of 10 mM Tris-HCl (pH 7.4), 50 mM NaCl, 2 mM MgCl<sub>2</sub>, 2 mM DTT, and 50% (v/v) glycerol conditions. RAD51 protein was purchased from Abcam (ab63808, Abcam).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Helicase Unwinding Assay</h3><div class="NLM_p last">A 5′-biotin labeled forked-DNA was used as a DNA substrate. The DNA substrate was diluted in buffer of 10 mM Tris-HCl (pH 7.4), 20 mM NaCl, and heated to 95 °C for 5 min, cooling down to 25 °C slowly. The purified BLM protein was incubated with a series of different concentration of compounds (1.0, 10, and 50 μM) at 4 °C for 1 h with no more than 1% DMSO. After 1 h of incubation, the duplex forked-DNA was added. Finally, the reaction sample contained 8 nM duplex forked-DNA, 32 nM purified BLM, 2 mM ATP (Sigma), 0.1% bovine serum albumin (BSA) and compounds. After 1 h of coincubation, the reaction was stopped by DNA loading buffer (Takara), and samples were run on the 20% native polyacrylamide gels at 8 V·cm<sup>–1</sup> and 4 °C in 0.5× TB buffer for 180 min. After the electrophoresis, the gels were transferred to nylon membranes. The membranes were cross-linked under UV irradiation at 265 nm for 10 min, and the visualization of DNA was performed using Nucleic Acid Detection Module Kit (Thermo Scientific). The upper bands were duplex DNA and the lower bands were single strand DNA after unwinding. The image analysis was performed using ImageJ 2.0, the untreated group was set as 100%, and the other bands were quantified to gain the inhibitory effect of each compound. The half maximal inhibitory concentration of BLM helicase (IC<sub>50</sub> value) was calculated via a nonlinear regression model in GraphPad software.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> MTT Assay</h3><div class="NLM_p last">Various cells (5 × 10<sup>3</sup>) were planted per well of the 96-well plate and cultured for 24 h. Then, a series concentration of <b>29</b> (from 0 to 50 μM) were added into cells and incubated for different times. For BLM RNA interference, cells were pretreated with 50 nM BLM siRNA for 24 h and treated with a series concentration of <b>29</b> (from 0 to 50 μM) for indicated times. Then, cells were incubated with 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide for 4 h at 37 °C at the final concentration of 0.5 mg/mL. Finally, removing the medium and dissolving the blue crystal in 100 μL of DMSO. The absorbance was measured at 470 nm by a microplate reader (BioTek. USA). The IC<sub>50</sub> was calculated via Hill Model in GraphPad software.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Enzyme-Linked Immunosorbent Assay</h3><div class="NLM_p last">A 5′-biotin labeled forked-DNA and the StreptaWell High Bind plates (Roche) were used in ELISA assay. Plates were hydrated with PBS for 30 min, and DNA substrates were diluted to 50 nM in Tris-HCl buffer (10 mM, pH 7.4) and immobilized on the plate at 4 °C overnight. The purified BLM and mutant variants were final diluted to 100 nM. BLM variants and half-diluted <b>29</b> were blended in the blocking buffer (3% BSA in ELISA buffer) with DMSO no more than 1%. After 1 h of incubation on ice, the plates were washed by ELISA buffer (50 mM KH<sub>2</sub>PO<sub>4</sub>, 100 mM KCl, pH 7.4) three times, blocked in blocking buffer for 3 h at room temperature, incubated with protein–ligand complexes at 37 °C for 1 h, repeat washed three times by washing buffer (0.1% Tween-20 in ELISA buffer), incubated with primary antibody (ab86664, Abcam) at 4 °C overnight, washed by washing buffer for three times, incubated with secondary HRP-conjugated antibody (no. 7074S Cell Signaling Technology) at 25 °C for 2 h, and repeat washed with washing buffer three times and 100 μL of TMB Chromogen Solution (Life Technologies) was added, and the plates were incubated for 5 min and then stopped by 50 μL of 2 M H<sub>2</sub>SO<sub>4</sub>. Absorbance was measured at 450 nm and the disrupting effect of <b>29</b> was obtained using GraphPad software via Hill fitting.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Filter-Binding Assay</h3><div class="NLM_p last">The nitrocellulose membrane was activated with 0.5 M KOH for 10 min at 25 °C and washed three times with 0.5× TB buffer. The nylon membrane was placed under the nitrocellulose membrane to arrest the free DNA that was not bound to the upper layer. Biotin-labeled duplex forked-DNA (8 nM) was incubated with increasing concentration of BLM for 1 h at 37 °C. The protein–DNA complexes penetrated the membrane under vacuum and were washed three times with 0.5× TB buffer. The membranes were cross-linked under UV irradiation at 265 nm for 10 min and the visualization of DNA was performed using Nucleic Acid Detection Module Kit (Thermo Scientific). The image analysis was applied using ImageJ 2.0, and we set the gray level of the left dot (total of the dots in upper and lower membranes) as 100%, and the subsequent dots were quantified to obtain the relative level of binding status, of which the upper dots were protein–DNA complex and the lower were free DNA. The effective binding concentration (EC<sub>50</sub>) was calculated via the Hill model using GraphPad software. In addition, BLM (100 nM) was incubated with two concentrates of <b>29</b> (0.5 and 1.0 μM) for 1 h at 37 °C before mixing with duplex forked-DNA and then following the above steps to quantify the disrupting effect of <b>29</b>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Pull-Down Assay</h3><div class="NLM_p last">Streptavidin magnetic beads (Thermo Scientific) were used in this experiment. After prewashing, excessive biotin-labeled duplex forked-DNA was added to be incubated with streptavidin magnetic beads at 4 °C overnight. HCT116 cells were harvested with cold PBS. The cell pellets were lysed by RIPA lysis buffer and centrifugated at 12 000 rpm for 10 min to obtain the supernatant. Then, the cell lysate was incubated with labeled streptavidin magnetic beads at 25 °C for 1 h with the treatment of <b>29</b>. After immunoprecipitation, the beads and supernatant were collected, and the washing steps were carried out following the manufacturer’s protocol. Finally, both the elution and supernatant were heated to 95 °C for 5 min and complemented with loading buffer. The protein was separated using SDS-PAGE and detected by western-blot.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Cell Culture and Treatment</h3><div class="NLM_p last">Colon cancer HCT116 cells were grown in GIBCO PPMI 1640 medium complemented with 10% fetal bovine serum at 37 °C constant temperature incubator with 5% CO<sub>2</sub>. For the drug treatment experiments, cells were treated with <b>29</b> and collected for various times in different experiments.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> HRR Reporter Assay</h3><div class="NLM_p last">HCT116 cells (2 × 10<sup>5</sup>) were seeded at culture dishes and cultured for 24 h. Then, cells were cotransfected with HRR reporters and I-SceI plasmids following the manufacturer’s protocol of lipo3000 (Invitrogen). After 24 h of cotransfection, the medium was discarded and a series concentration of <b>29</b> (from 0 to 2 μM) and <b>ML216</b> (from 0 to 50 μM) were added into per well. Finally, the cells were harvested after 24 h of treatment. The cell pellets were washed three times and resuspended in cold PBS. The quantities of GFP expressed cells were detected using flow cytometry (EPICS XL).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> ChIP Assay</h3><div class="NLM_p last">A Pierce Magnetic ChIP Kit (Thermo Scientific) was used to perform the ChIP experiments according to the manufacturer’s protocol. HCT116 cells were cotransfected with DSB models and I-SceI plasmids. After 24 h cotransfection, a series concentration of <b>29</b> (from 0 to 2 μM) and <b>ML216</b> (50 μM) were added, and cells were harvested and fixed with 1% formaldehyde for 15 min and lysed after 24 h treatment. The chromatins were dissociated into an average size of 0.2–0.5 kb using micrococcal nuclease (10 U/μL) provided in the kit. Then, 10% of the lysis was used as the input, 2 μg primary antibody of BLM (ab2179, Abcam) and RAD51 (ab133534, Abcam) was used for immunoprecipitation, and 2 μg of normal rabbit IgG (no. 7074S Cell Signaling Technology) was used as negative control. IP reactions were performed overnight at 4 °C. The antigen–antibody complexes were harvested using ChIP Grade Protein A/G M magnetic beads supplied in the kit. After the elution and DNA recovery, the purified DNA were amplified and detected by qPCR.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> DNA Exchange Activity Assay</h3><div class="NLM_p last">StreptaWell High Bind plates (Roche) were used in this experiment. The final concentration of 50 nM biotin-labeled duplex forked-DNA was planted on the plate at 4 °C overnight. A series concentration of purified RAD51 (from 0 to 500 nM) was added following the ELISA. The absorbance was measured at 450 nm and the effective binding concentration (EC<sub>50</sub>) of RAD51 with duplex forked-DNA was obtained using GraphPad software via Hill fitting. Then, a series of increasing concentration of purified BLM (from 10 to 1000 nM) was incubated with 250 nM of RAD51 at 37 °C for 1 h following the procedure of ELISA. The amount of RAD51 was measured in 450 nm absorbance to present the level of the formation of RAD51-ssDNA filaments. For ligand intervention, BLM and different concentrations of <b>29</b> were pretreated at 37 °C for 1 h, and the complex was added to RAD51 solution following the procedure of ELISA.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Immunofluorescence Assay</h3><div class="NLM_p last">Cells grown on glass coverslips were fixed in 4% paraformaldehyde/PBS for 15 min, then permeabilized with 0.5% Triton-X100/PBS at 37 °C for 20 min, and finally blocked with 5% goat serum/PBS at 37 °C for 3 h. The protocol was followed with the γ-H2AX antibody (no. 9718, Cell Signaling Technology) and RAD51 antibody (ab133534, Abcam) at 4 °C overnight. The glass coverslips were washed six times with blocking buffer and were then incubated with anti-rabbit Alexa 488-conjugated antibody (A21206, Life Technology) and anti-mouse Alexa 647-conjugated antibody (A21235, Life Technology), and 2 μg/mL of 4,6-diamidino-2-phenylindole (DAPI, Invitrogen) was diluted in 5% goat serum/PBS at 37 °C for 3 h. The glass coverslips were again washed six times with the blocking buffer, and then, digital images were recorded using an LSM710 microscope (Zeiss, GER) and analyzed with ZEN software.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Protein Extract and Western Blot Assay</h3><div class="NLM_p last">The HCT116 cells were washed with cold PBS and lysed by RIPA lysis buffer (50 mM Tris, 150 mM NaCl, 1% TritonX-100, 1% sodium deoxycholate, 0.1% SDS, 1 mM EDTA, pH 7.4) at 4 °C for 30 min. The lysis of cells was centrifuged at 12 000 rpm at 4 °C for 5 min. The supernatant was diluted to BCA determination and denatured at 95 °C for 5 min with addition of the loading buffer (50 mM Tris-HCl, 6 M urea, 6% 2-mercaptoethanol, 3% SDS, 0.003% bromophenol blue, pH 6.8), and 40 μg of protein was loaded on each lane, run on the SDS-PAGE, transferred to a microporous polyvinylidene difluoride membranes, and then detected by western blot. The primary antibodies used in this experiment were β-actin (no. 4970S, Cell Signaling Technology), BLM (ab2179, Abcam), RAD51 (ab133534, Abcam), DNA Damage Antibody Sampler Kit (no. 9947T Cell Signaling Technology), and Apoptosis Antibody Sampler Kit (no. 9915T Cell Signaling Technology). In addition, secondary antibodies were horse radish peroxidase-conjugated anti-mouse (no. 7076S Cell Signaling Technology) and anti-rabbit (no. 7074S Cell Signaling Technology). The visualization of protein bands replied on the chemiluminescence substrate.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Annexin V-FITC/PI Apoptosis Detection</h3><div class="NLM_p last">HCT116 cells (2 × 10<sup>5</sup>) were planted in culture dish and treated with <b>29</b> for 48 h. The apoptosis detection was applied using the FITC Annexin V/PI apoptosis Detection Kit (KeyGEN). The collected cells were resuspended in 500 μL binding buffer and added with 5 μL of FITC Annexin V and PI dyes. Slowly blending the samples and incubating at 25 °C for 30 min away from light. Finally, the fluorescence-positive cells were quantified by flow cytometry (EPICS XL).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> ITC Studies</h3><div class="NLM_p last">The interactions between <b>29</b>, BLM variants (wild-type, Y995A, H996A, M1111A, E1143A, I1168A), and DNA were measured using isothermal titration calorimetry (ITC) with a VP-isothermal titration calorimeter from Microcal, Inc. (Northampton, USA). Calorimetric titrations of <b>29</b> (0.6 mM in the syringe), BLM variants (0.04 mM in the cuvette), and DNA (0.04 mM in the cuvette) were carried out at 25 °C in 2 mM MgCl<sub>2</sub>, and 10 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.5. <b>29</b> was titrated into BLM variants/DNA in 10 μL injections with a spacing of 300 s between injections. Calorimetric data were analyzed by integrating heat effects normalized to the amount of injected protein and curve-fitting based on a 1:1 binding model using the Origin software package (Microcal). The dissociation constant was derived from data using standard procedures.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Molecular Modeling Studies</h3><div class="NLM_p last">Docking studies were performed using the Schrodinger Maestro software. The crystal structure of BLM (code ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>) was obtained from the Protein Data Bank. The duplex DNA and ADP in the crystal structure were removed. The structure of <b>29</b> was generated using the LigPrep module. Possible binding pockets over all the domains in BLM were predicted by SiteMap module. The docking analysis was first performed using the Glide module. Two hundred poses were generated for each pocket, and post-docking minimization was performed. The binding poses with a strong predicted docking score were obtained and clustered. Simulations of MD were performed using the sander module of the AMBER 12 program suite using the parm99bsc0 parameters.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> Partial atomic charges for compound <b>29</b> were derived using the HF/6-31G* basis set, followed by an RESP calculation, whereas force-field parameters were obtained from the generalized Amber force field using the ANTECHAMBER module.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The complexes of BLMs and compounds were solvated in a rectangular box of TIP3P water molecules with solvent layers of 8 Å. The solvated structures were subjected to initial minimization to equilibrate the solvent and counter cations. The system was then heated from 0 to 300 K in a 100 ps simulation, followed by a 100 ps simulation to equilibrate the density of the system. Subsequently, a constant pressure MD simulation of 20 ns was performed in an <i>NPT</i> ensemble at 1 atm and 300 K. The complete trajectory analysis was performed with the Ptraj module in the Amber 10 suite. The molecular mechanics Poisson–Boltzmann surface area method implemented in the AMBER 12 suite was applied to calculate the binding free energy between the BLM and compound <b>29</b>.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The models were visualized using the Discovery Studio software package.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00083" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56738" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00083" class="ext-link">10.1021/acs.jmedchem.9b00083</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional experimental results, <sup>1</sup>H NMR, <sup>13</sup>C NMR, HPLC and HRMS spectra for final compounds, molecular formula strings and coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">We provide the molecular information for a model (<b>29</b>/<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>) in this study. The authors will release the atomic coordinates and experimental data upon article publication (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Protein Data Bank, crystal structure of BLM, and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>(<a href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf">jm9b00083_si_001.pdf (11.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_002.csv">jm9b00083_si_002.csv (5.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_003.pdb">jm9b00083_si_003.pdb (693.02 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00083" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12537" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12537" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuo-Bin Chen</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9118-2185" title="Orcid link">http://orcid.org/0000-0001-9118-2185</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#76151e1318051e144e361b171f1a58050f050358131203581518"><span class="__cf_email__" data-cfemail="1a79727f74697278225a777b7376346963696f347f7e6f347974">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhi-Shu Huang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6211-5482" title="Orcid link">http://orcid.org/0000-0002-6211-5482</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#385b5d4b50424b7855595154164b414b4d165d5c4d165b56"><span class="__cf_email__" data-cfemail="86e5e3f5eefcf5c6ebe7efeaa8f5fff5f3a8e3e2f3a8e5e8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qi-Kun Yin</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chen-Xi Wang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu-Qing Wang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qian-Liang Guo</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zi-Lin Zhang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tian-Miao Ou</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8176-4576" title="Orcid link">http://orcid.org/0000-0002-8176-4576</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shi-Liang Huang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ding Li</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3581-7630" title="Orcid link">http://orcid.org/0000-0002-3581-7630</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong-Gen Wang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jia-Heng Tan</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical Sciences, Guangdong Provincial
Key Laboratory of New Drug Design and Evaluation, Sun Yat-sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1612-7482" title="Orcid link">http://orcid.org/0000-0002-1612-7482</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87480" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87480" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (81872732, 21708053, 81330077, and 21672265), the Natural Science Foundation of Guangdong Province (2017A030308003 and 2017A030313040), the Ministry of Education of China (no. IRT-17R111), the Fundamental Research Funds for the Central Universities (17ykpy18), and the Guangdong Provincial Key Laboratory of Construction Foundation (2017B030314030).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BLM</td><td class="NLM_def"><p class="first last">bloom’s syndrome protein</p></td></tr><tr><td class="NLM_term">HRR</td><td class="NLM_def"><p class="first last">homologous recombination repair</p></td></tr><tr><td class="NLM_term">DSBs</td><td class="NLM_def"><p class="first last">double-strand breaks</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">methyl thiazolyl tetrazolium</p></td></tr><tr><td class="NLM_term">ELISA</td><td class="NLM_def"><p class="first last">enzyme-linked immunosorbent assay</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">qPCR</td><td class="NLM_def"><p class="first last">fluorogenic quantitative PCR</p></td></tr><tr><td class="NLM_term">CHIP</td><td class="NLM_def"><p class="first last">chromatin immunoprecipitation</p></td></tr><tr><td class="NLM_term">ssDNA</td><td class="NLM_def"><p class="first last">single-strand DNA</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">RQC</td><td class="NLM_def"><p class="first last">RecQ C-terminal</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40508" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40508" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage response in human biology and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi"> DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=1071-1078&author=S.+P.+Jacksonauthor=J.+Bartek&title=The+DNA-damage+response+in+human+biology+and+disease&doi=10.1038%2Fnature08467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0liQgPajnNaKnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520response%2520in%2520human%2520biology%2520and%2520disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, S. A.</span></span> <span> </span><span class="NLM_article-title">Genomic approaches to DNA repair and mutagenesis</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2015.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2015.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26411877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWls7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=146-155&author=J.+J.+Wyrickauthor=S.+A.+Roberts&title=Genomic+approaches+to+DNA+repair+and+mutagenesis&doi=10.1016%2Fj.dnarep.2015.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic approaches to DNA repair and mutagenesis</span></div><div class="casAuthors">Wyrick, John J.; Roberts, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">146-155</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  DNA damage is a const. threat to cells, causing cytotoxicity as well as inducing genetic alterations.  The steady-state abundance of DNA lesions in a cell is minimized by a variety of DNA repair mechanisms, including DNA strand break repair, mismatch repair, nucleotide excision repair, base excision repair, and ribonucleotide excision repair.  The efficiencies and mechanisms by which these pathways remove damage from chromosomes have been primarily characterized by investigating the processing of lesions at defined genomic loci, among bulk genomic DNA, on episomal DNA constructs, or using in vitro substrates.  However, the structure of a chromosome is heterogeneous, consisting of heavily protein-bound heterochromatic regions, open regulatory regions, actively transcribed genes, and even areas of transient single stranded DNA.  Consequently, DNA repair pathways function in a much more diverse set of chromosomal contexts than can be readily assessed using previous methods.  Recent efforts to develop whole genome maps of DNA damage, repair processes, and even mutations promise to greatly expand our understanding of DNA repair and mutagenesis.  Here we review the current efforts to utilize whole genome maps of DNA damage and mutation to understand how different chromosomal contexts affect DNA excision repair pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJpUXQJ0AGcrVg90H21EOLACvtfcHk0lhttU8d5R5PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWls7jL&md5=0bf15f45ed5052821415f4323618425b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DWyrick%26aufirst%3DJ.%2BJ.%26aulast%3DRoberts%26aufirst%3DS.%2BA.%26atitle%3DGenomic%2520approaches%2520to%2520DNA%2520repair%2520and%2520mutagenesis%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D36%26spage%3D146%26epage%3D155%26doi%3D10.1016%2Fj.dnarep.2015.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span> <span> </span><span class="NLM_article-title">Single-strand break repair and genetic disease</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1038/nrg2380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrg2380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18626472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVOku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=619-631&author=K.+W.+Caldecott&title=Single-strand+break+repair+and+genetic+disease&doi=10.1038%2Fnrg2380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Single-strand break repair and genetic disease</span></div><div class="casAuthors">Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">619-631</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Hereditary defects in the repair of DNA damage are implicated in a variety of diseases, many of which are typified by neurol. dysfunction and/or increased genetic instability and cancer.  Of the different types of DNA damage that arise in cells, single-strand breaks (SSBs) are the most common, arising at a frequency of tens of thousands per cell per day from direct attack by intracellular metabolites and from spontaneous DNA decay.  Here, the mol. mechanisms and organization of the DNA-repair pathways that remove SSBs are reviewed and the connection between defects in these pathways and hereditary neurodegenerative disease are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrLPZKetl1rVg90H21EOLACvtfcHk0lhttU8d5R5PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVOku7Y%253D&md5=5677671cc74547583cfb595952106b48</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrg2380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2380%26sid%3Dliteratum%253Aachs%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DSingle-strand%2520break%2520repair%2520and%2520genetic%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2008%26volume%3D9%26spage%3D619%26epage%3D631%26doi%3D10.1038%2Fnrg2380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">DNA double-strand breaks: signaling, repair and the cancer connection</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1038/85798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2F85798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11242102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2001&pages=247-254&author=K.+K.+Khannaauthor=S.+P.+Jackson&title=DNA+double-strand+breaks%3A+signaling%2C+repair+and+the+cancer+connection&doi=10.1038%2F85798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">DNA double-strand breaks: signaling, repair and the cancer connection</span></div><div class="casAuthors">Khanna, Kum Kum; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">247-254</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review, with 76 refs.,.  To ensure the high-fidelity transmission of genetic information, cells have evolved mechanisms to monitor genome integrity.  Cells respond to DNA damage by activating a complex DNA-damage-response pathway that includes cell-cycle arrest, the transcriptional and post-transcriptional activation of a subset of genes including those assocd. with DNA repair, and under some circumstances, the triggering of programmed cell death.  An inability to respond properly to, or to repair, DNA damage leads to genetic instability, which in turn may enhance the rate of cancer development.  Indeed, it is becoming increasingly clear that deficiencies in DNA-damage signaling and repair pathways are fundamental to the etiol. of most, if not all, human cancers.  Here, recent progress in the understanding of how cells detect and signal the presence and repair of one particularly important form of DNA damage induced by ionizing radiation, the DNA double-strand break (DSB), is described.  Moreover, how tumor suppressor proteins such as p53, ATM, Brca1 and Brca2 have been linked to such pathways and how accumulating evidence is connecting deficiencies in cellular responses to DNA DSBs with tumorigenesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2_Aw1LLx3LVg90H21EOLACvtfcHk0lhttU8d5R5PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslKqtLg%253D&md5=e5ccb5ac64c8dbf39375d33a056f34ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F85798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F85798%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DK.%2BK.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DDNA%2520double-strand%2520breaks%253A%2520signaling%252C%2520repair%2520and%2520the%2520cancer%2520connection%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D27%26spage%3D247%26epage%3D254%26doi%3D10.1038%2F85798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krejci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmannova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spirek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination and its regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5795</span>– <span class="NLM_lpage">5818</span>, <span class="refDoi"> DOI: 10.1093/nar/gks270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgks270" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=5795-5818&author=L.+Krejciauthor=V.+Altmannovaauthor=M.+Spirekauthor=X.+Zhao&title=Homologous+recombination+and+its+regulation&doi=10.1093%2Fnar%2Fgks270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks270%26sid%3Dliteratum%253Aachs%26aulast%3DKrejci%26aufirst%3DL.%26aulast%3DAltmannova%26aufirst%3DV.%26aulast%3DSpirek%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DHomologous%2520recombination%2520and%2520its%2520regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D5795%26epage%3D5818%26doi%3D10.1093%2Fnar%2Fgks270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0li4lcTqcHwI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">DNA repair dysregulation from cancer driver to therapeutic target</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nrc3399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc3399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=23175119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=801-817&author=N.+J.+Curtin&title=DNA+repair+dysregulation+from+cancer+driver+to+therapeutic+target&doi=10.1038%2Fnrc3399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair dysregulation from cancer driver to therapeutic target</span></div><div class="casAuthors">Curtin, Nicola J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">801-817</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dysregulation of DNA damage repair and signalling to cell cycle checkpoints, known as the DNA damage response (DDR), is assocd. with a predisposition to cancer and affects responses to DNA-damaging anticancer therapy.  Dysfunction of one DNA repair pathway may be compensated for by the function of another compensatory DDR pathway, which may be increased and contribute to resistance to DNA-damaging chemotherapy and radiotherapy.  Therefore, DDR pathways make an ideal target for therapeutic intervention; first, to prevent or reverse therapy resistance; and second, using a synthetic lethal approach to specifically kill cancer cells that are dependent on a compensatory DNA repair pathway for survival in the context of cancer-assocd. oxidative and replicative stress.  These hypotheses are currently being tested in the lab. and are being translated into clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNDBV6krLGBbVg90H21EOLACvtfcHk0li4lcTqcHwI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N&md5=1bcb90d4742e46696c8a30a4fce2400c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc3399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3399%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DDNA%2520repair%2520dysregulation%2520from%2520cancer%2520driver%2520to%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D801%26epage%3D817%26doi%3D10.1038%2Fnrc3399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garber, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domchek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audeh, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzel, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedlander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span> <span> </span><span class="NLM_article-title">Phase II trial of the oral PARP inhibitor olaparib in BRCA-deficient advanced breast cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">CRA501</span>, <span class="refDoi"> DOI: 10.1200/jco.2009.27.18_suppl.cra501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1200%2Fjco.2009.27.18_suppl.cra501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=CRA501&author=A.+Tuttauthor=M.+Robsonauthor=J.+E.+Garberauthor=S.+Domchekauthor=M.+W.+Audehauthor=J.+N.+Weitzelauthor=M.+Friedlanderauthor=J.+Carmichael&title=Phase+II+trial+of+the+oral+PARP+inhibitor+olaparib+in+BRCA-deficient+advanced+breast+cancer&doi=10.1200%2Fjco.2009.27.18_suppl.cra501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1200%2Fjco.2009.27.18_suppl.cra501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2009.27.18_suppl.cra501%26sid%3Dliteratum%253Aachs%26aulast%3DTutt%26aufirst%3DA.%26aulast%3DRobson%26aufirst%3DM.%26aulast%3DGarber%26aufirst%3DJ.%2BE.%26aulast%3DDomchek%26aufirst%3DS.%26aulast%3DAudeh%26aufirst%3DM.%2BW.%26aulast%3DWeitzel%26aufirst%3DJ.%2BN.%26aulast%3DFriedlander%26aufirst%3DM.%26aulast%3DCarmichael%26aufirst%3DJ.%26atitle%3DPhase%2520II%2520trial%2520of%2520the%2520oral%2520PARP%2520inhibitor%2520olaparib%2520in%2520BRCA-deficient%2520advanced%2520breast%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2009%26volume%3D27%26spage%3DCRA501%26doi%3D10.1200%2Fjco.2009.27.18_suppl.cra501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ristic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanaar, R.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination-mediated double-strand break repair</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2004.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2004.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=15279767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVeqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=827-833&author=C.+Wymanauthor=D.+Risticauthor=R.+Kanaar&title=Homologous+recombination-mediated+double-strand+break+repair&doi=10.1016%2Fj.dnarep.2004.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination-mediated double-strand break repair</span></div><div class="casAuthors">Wyman, Claire; Ristic, Dejan; Kanaar, Roland</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">827-833</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Exchange of DNA strands between homologous DNA mols. via recombination ensures accurate genome duplication and preservation of genome integrity.  Biochem. studies have provided insights into the mol. mechanisms by which homologous recombination proteins perform these essential tasks.  More recent cell biol. expts. are addressing the behavior of homologous recombination proteins in cells.  The challenge ahead is to uncover the relationship between the individual biochem. activities of homologous recombination proteins and their coordinated action in the context of the living cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRDQngo6NrWrVg90H21EOLACvtfcHk0li4lcTqcHwI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVeqsb8%253D&md5=ec00b3ee185ba32a9cfdaab48ad65187</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2004.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2004.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DWyman%26aufirst%3DC.%26aulast%3DRistic%26aufirst%3DD.%26aulast%3DKanaar%26aufirst%3DR.%26atitle%3DHomologous%2520recombination-mediated%2520double-strand%2520break%2520repair%26jtitle%3DDNA%2520Repair%26date%3D2004%26volume%3D3%26spage%3D827%26epage%3D833%26doi%3D10.1016%2Fj.dnarep.2004.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyer, W.-D.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination in DNA repair and DNA damage tolerance</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1038/cr.2008.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fcr.2008.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18166982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisVSjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=99-113&author=X.+Liauthor=W.-D.+Heyer&title=Homologous+recombination+in+DNA+repair+and+DNA+damage+tolerance&doi=10.1038%2Fcr.2008.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in DNA repair and DNA damage tolerance</span></div><div class="casAuthors">Li, Xuan; Heyer, Wolf-Dietrich</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-113</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Homologous recombination (HR) comprises a series of interrelated pathways that function in the repair of DNA double-stranded breaks (DSBs) and interstrand crosslinks (ICLs).  In addn., recombination provides crit. support for DNA replication in the recovery of stalled or broken replication forks, contributing to tolerance of DNA damage.  A central core of proteins, most critically the RecA homolog Rad51, catalyzes the key reactions that typify HR: homol. search and DNA strand invasion.  The diverse functions of recombination are reflected in the need for context-specific factors that perform supplemental functions in conjunction with the core proteins.  The inability to properly repair complex DNA damage and resolve DNA replication stress leads to genomic instability and contributes to cancer etiol.  Mutations in the BRCA2 recombination gene cause predisposition to breast and ovarian cancer as well as Fanconi anemia, a cancer predisposition syndrome characterized by a defect in the repair of DNA interstrand crosslinks.  The cellular functions of recombination are also germane to DNA-based treatment modalities of cancer, which target replicating cells by the direct or indirect induction of DNA lesions that are substrates for recombination pathways.  This review focuses on mechanistic aspects of HR relating to DSB and ICL repair as well as replication fork support.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprinxRSKXk47Vg90H21EOLACvtfcHk0lj6xucIPGmUDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisVSjsw%253D%253D&md5=103977d80ae5366ccb2b8d4621fd7358</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fcr.2008.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2008.1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHeyer%26aufirst%3DW.-D.%26atitle%3DHomologous%2520recombination%2520in%2520DNA%2520repair%2520and%2520DNA%2520damage%2520tolerance%26jtitle%3DCell%2520Res.%26date%3D2008%26volume%3D18%26spage%3D99%26epage%3D113%26doi%3D10.1038%2Fcr.2008.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedberg, E. C.</span></span> <span> </span><span class="NLM_article-title">DNA damage and repair</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nature01408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature01408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=12540918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFGqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=436-440&author=E.+C.+Friedberg&title=DNA+damage+and+repair&doi=10.1038%2Fnature01408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage and repair</span></div><div class="casAuthors">Friedberg, Errol C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">6921, Suppl.</span>),
    <span class="NLM_cas:pages">436-440</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aesthetic appeal of the DNA double helix initially hindered notions of DNA mutation and repair, which would necessarily interfere with its pristine state.  But it has since been recognized that DNA is subject to continuous damage and the cell has an arsenal of ways of responding to such injury.  Although mutations or deficiencies in repair can have catastrophic consequences, causing a range of human diseases, mutations are nonetheless fundamental to life and evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofETXt2LmXt7Vg90H21EOLACvtfcHk0lj6xucIPGmUDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFGqtb8%253D&md5=92475bfb2d2c155798ef6198dabcc701</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature01408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01408%26sid%3Dliteratum%253Aachs%26aulast%3DFriedberg%26aufirst%3DE.%2BC.%26atitle%3DDNA%2520damage%2520and%2520repair%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D436%26epage%3D440%26doi%3D10.1038%2Fnature01408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishijima, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youssoufian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span> <span> </span><span class="NLM_article-title">Cancer predisposition caused by elevated mitotic recombination in Bloom mice</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1038/82548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2F82548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11101838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVWgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2000&pages=424-429&author=G.+Luoauthor=I.+M.+Santoroauthor=L.+D.+McDanielauthor=I.+Nishijimaauthor=M.+Millsauthor=H.+Youssoufianauthor=H.+Vogelauthor=R.+A.+Schultzauthor=A.+Bradley&title=Cancer+predisposition+caused+by+elevated+mitotic+recombination+in+Bloom+mice&doi=10.1038%2F82548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer predisposition caused by elevated mitotic recombination in Bloom mice</span></div><div class="casAuthors">Luo, Guangbin; Santoro, Irma M.; McDaniel, Lisa D.; Nishijima, Ichiko; Mills, Michael; Youssoufian, Hagop; Vogel, Hannes; Schultz, Roger A.; Bradley, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-429</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Bloom syndrome is a disorder assocd. with genomic instability that causes affected people to be prone to cancer.  Bloom cell lines show increased sister chromatid exchange, yet are proficient in the repair of various DNA lesions.  The underlying cause of this disease are mutations in a gene encoding a RECQ DNA helicase.  Using embryonic stem cell technol., the authors have generated viable Bloom mice that are prone to a wide variety of cancers.  Cell lines from these mice show elevations in the rates of mitotic recombination.  The authors demonstrate that the increased rate of loss of heterozygosity (LOH) resulting from mitotic recombination in vivo constitutes the underlying mechanism causing tumor susceptibility in these mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2NUHWpnQPL7Vg90H21EOLACvtfcHk0lj6xucIPGmUDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVWgtr4%253D&md5=66b50d2784f4a289712d078ce73b4990</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F82548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F82548%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DSantoro%26aufirst%3DI.%2BM.%26aulast%3DMcDaniel%26aufirst%3DL.%2BD.%26aulast%3DNishijima%26aufirst%3DI.%26aulast%3DMills%26aufirst%3DM.%26aulast%3DYoussoufian%26aufirst%3DH.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DR.%2BA.%26aulast%3DBradley%26aufirst%3DA.%26atitle%3DCancer%2520predisposition%2520caused%2520by%2520elevated%2520mitotic%2520recombination%2520in%2520Bloom%2520mice%26jtitle%3DNat.%2520Genet.%26date%3D2000%26volume%3D26%26spage%3D424%26epage%3D429%26doi%3D10.1038%2F82548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobzhansky, T.</span></span> <span> </span><span class="NLM_article-title">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1946</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20985721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1946&pages=269-290&author=T.+Dobzhansky&title=Genetics+of+natural+populations%3B+recombination+and+variability+in+populations+of+Drosophila+pseudoobscura"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span></div><div class="casAuthors">DOBZHANSKY T</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1946</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-90</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXehIoFNdovudfST00W0BcfW6udTcc2eYyuqVy3O8QwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D&md5=51f55835f5a3053a235bdea89458573c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDobzhansky%26aufirst%3DT.%26atitle%3DGenetics%2520of%2520natural%2520populations%253B%2520recombination%2520and%2520variability%2520in%2520populations%2520of%2520Drosophila%2520pseudoobscura%26jtitle%3DGenetics%26date%3D1946%26volume%3D31%26spage%3D269%26epage%3D290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartwell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szankasi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friend, S. H.</span></span> <span> </span><span class="NLM_article-title">Integrating genetic approaches into the discovery of anticancer drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">1064</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1126/science.278.5340.1064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1126%2Fscience.278.5340.1064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=9353181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADyaK2sXnt1Wju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=1997&pages=1064-1068&author=L.+H.+Hartwellauthor=P.+Szankasiauthor=C.+J.+Robertsauthor=A.+W.+Murrayauthor=S.+H.+Friend&title=Integrating+genetic+approaches+into+the+discovery+of+anticancer+drugs&doi=10.1126%2Fscience.278.5340.1064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating genetic approaches into the discovery of anticancer drugs</span></div><div class="casAuthors">Hartwell, Leland H.; Szankasi, Philippe; Roberts, Christopher J.; Murray, Andrew W.; Friend, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">5340</span>),
    <span class="NLM_cas:pages">1064-1068</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 48 refs. The discovery of anticancer drugs is now driven by the numerous mol. alterations identified in tumor cells over the past decade.  To exploit these alterations, it is necessary to understand how they define a mol. context that allows increased sensitivity to particular compds.  Traditional genetic approaches together with the new wealth of genomic information for both human and model organisms open up strategies by which drugs can be profiled for their ability to selectively kill cells in a mol. context that matches those found in tumors.  Similarly, it may be possible to identify and validate new targets for drugs that would selectively kill tumor cells with a particular mol. context.  This article outlines some of the ways that yeast genetics can be used to streamline anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbQLv4QwxtZrVg90H21EOLACvtfcHk0liebu4FEUywbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt1Wju7k%253D&md5=6bd5b219e8b5f5dd28b400003a9e4edb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.278.5340.1064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.278.5340.1064%26sid%3Dliteratum%253Aachs%26aulast%3DHartwell%26aufirst%3DL.%2BH.%26aulast%3DSzankasi%26aufirst%3DP.%26aulast%3DRoberts%26aufirst%3DC.%2BJ.%26aulast%3DMurray%26aufirst%3DA.%2BW.%26aulast%3DFriend%26aufirst%3DS.%2BH.%26atitle%3DIntegrating%2520genetic%2520approaches%2520into%2520the%2520discovery%2520of%2520anticancer%2520drugs%26jtitle%3DScience%26date%3D1997%26volume%3D278%26spage%3D1064%26epage%3D1068%26doi%3D10.1126%2Fscience.278.5340.1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelivianskaia, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron McGuire, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, P. P.</span></span> <span> </span><span class="NLM_article-title">RI-1: a chemical inhibitor of RAD51 that disrupts homologous recombination in human cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">7347</span>– <span class="NLM_lpage">7357</span>, <span class="refDoi"> DOI: 10.1093/nar/gks353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgks353" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=7347-7357&author=B.+Budkeauthor=H.+L.+Loganauthor=J.+H.+Kalinauthor=A.+S.+Zelivianskaiaauthor=W.+Cameron+McGuireauthor=L.+L.+Millerauthor=J.+M.+Starkauthor=A.+P.+Kozikowskiauthor=D.+K.+Bishopauthor=P.+P.+Connell&title=RI-1%3A+a+chemical+inhibitor+of+RAD51+that+disrupts+homologous+recombination+in+human+cells&doi=10.1093%2Fnar%2Fgks353"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks353%26sid%3Dliteratum%253Aachs%26aulast%3DBudke%26aufirst%3DB.%26aulast%3DLogan%26aufirst%3DH.%2BL.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DZelivianskaia%26aufirst%3DA.%2BS.%26aulast%3DCameron%2BMcGuire%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DL.%2BL.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DBishop%26aufirst%3DD.%2BK.%26aulast%3DConnell%26aufirst%3DP.%2BP.%26atitle%3DRI-1%253A%2520a%2520chemical%2520inhibitor%2520of%2520RAD51%2520that%2520disrupts%2520homologous%2520recombination%2520in%2520human%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D7347%26epage%3D7357%26doi%3D10.1093%2Fnar%2Fgks353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1073/pnas.1006423108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.1006423108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=21220316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1525-1530&author=M.+Aggarwalauthor=J.+A.+Sommersauthor=R.+H.+Shoemakerauthor=R.+M.+Brosh&title=Inhibition+of+helicase+activity+by+a+small+molecule+impairs+Werner+syndrome+helicase+%28WRN%29+function+in+the+cellular+response+to+DNA+damage+or+replication+stress&doi=10.1073%2Fpnas.1006423108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span></div><div class="casAuthors">Aggarwal, Monika; Sommers, Joshua A.; Shoemaker, Robert H.; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1525-1530, S1525/1-S1525/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modulation of DNA repair proteins by small mols. has attracted great interest.  An in vitro helicase activity screen was used to identify mols. that modulate DNA unwinding by Werner syndrome helicase (WRN), mutated in the premature aging disorder Werner syndrome.  A small mol. from the National Cancer Institute Diversity Set designated NSC 19630 [1-(propoxymethyl)-maleimide] was identified that inhibited WRN helicase activity but did not affect other DNA helicases [Bloom syndrome (BLM), Fanconi anemia group J (FANCJ), RECQ1, RecQ, UvrD, or DnaB].  Exposure of human cells to NSC 19630 dramatically impaired growth and proliferation, induced apoptosis in a WRN-dependent manner, and resulted in elevated γ-H2AX and proliferating cell nuclear antigen (PCNA) foci.  NSC 19630 exposure led to delayed S-phase progression, consistent with the accumulation of stalled replication forks, and to DNA damage in a WRN-dependent manner.  Exposure to NSC 19630 sensitized cancer cells to the G-quadruplex-binding compd. telomestatin or a poly(ADP ribose) polymerase (PARP) inhibitor.  Sublethal dosage of NSC 19630 and the chemotherapy drug topotecan acted synergistically to inhibit cell proliferation and induce DNA damage.  The use of this WRN helicase inhibitor mol. may provide insight into the importance of WRN-mediated pathway(s) important for DNA repair and the replicational stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRIkmkwi9yCrVg90H21EOLACvtfcHk0liebu4FEUywbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D&md5=d01ee377da5c65cdec72b0403494fa0e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1006423108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1006423108%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DM.%26aulast%3DSommers%26aufirst%3DJ.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DInhibition%2520of%2520helicase%2520activity%2520by%2520a%2520small%2520molecule%2520impairs%2520Werner%2520syndrome%2520helicase%2520%2528WRN%2529%2520function%2520in%2520the%2520cellular%2520response%2520to%2520DNA%2520damage%2520or%2520replication%2520stress%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D1525%26epage%3D1530%26doi%3D10.1073%2Fpnas.1006423108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination in cancer development, treatment and development of drug resistance</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fcarcin%2Fbgq064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20351092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=955-960&author=T.+Helleday&title=Homologous+recombination+in+cancer+development%2C+treatment+and+development+of+drug+resistance&doi=10.1093%2Fcarcin%2Fbgq064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in cancer development, treatment and development of drug resistance</span></div><div class="casAuthors">Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">955-960</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although DNA double-strand breaks (DSBs) are substrates for homologous recombination (HR) repair, it is becoming apparent that DNA lesions produced at replication forks, for instance by many anticancer drugs, are more significant substrates for HR repair.  Cells defective in HR are hypersensitive to a wide variety of anticancer drugs, including those that do not produce DSBs.  Several cancers have mutations in or epigenetically silenced HR genes, which explain the genetic instability that drives cancer development.  There are an increasing no. of reports suggesting that mutation or epigenetic silencing of HR genes explains the sensitivity of cancers to current chemotherapy treatments.  Furthermore, there are also many examples of re-expression of HR genes in tumors to explain drug resistance.  Emerging data suggest that there are several different subpathways of HR, which can compensate for each other.  Unravelling the overlapping pathways in HR showed that BRCA1- and BRCA2-defective cells rely on the PARP protein for survival.  This synthetic lethal interaction is now being exploited for selective treatment of BRCA1- and BRCA2-defective cancers with PARP inhibitors.  Here, I discuss the diversity of HR and how it impacts on cancer with a particular focus on how HR can be exploited in future anticancer strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaL2tqlRLWrVg90H21EOLACvtfcHk0lhKODdkdoBqog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D&md5=1385e375ac39dd982ff9892fc8492486</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq064%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DHomologous%2520recombination%2520in%2520cancer%2520development%252C%2520treatment%2520and%2520development%2520of%2520drug%2520resistance%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D955%26epage%3D960%26doi%3D10.1093%2Fcarcin%2Fbgq064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hengel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors targeting DNA repair and DNA repair deficiency in research and cancer therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1101</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.chembiol.2017.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=28938088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1101-1119&author=S.+R.+Hengelauthor=M.+A.+Spiesauthor=M.+Spies&title=Small-molecule+inhibitors+targeting+DNA+repair+and+DNA+repair+deficiency+in+research+and+cancer+therapy&doi=10.1016%2Fj.chembiol.2017.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span></div><div class="casAuthors">Hengel, Sarah R.; Spies, M. Ashley; Spies, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1101-1119</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To maintain stable genomes and to avoid cancer and aging, cells need to repair a multitude of deleterious DNA lesions, which arise constantly in every cell.  Processes that support genome integrity in normal cells, however, allow cancer cells to develop resistance to radiation and DNA-damaging chemotherapeutics.  Chem. inhibition of the key DNA repair proteins and pharmacol. induced synthetic lethality have become instrumental in both dissecting the complex DNA repair networks and as promising anticancer agents.  The difficulty in capitalizing on synthetically lethal interactions in cancer cells is that many potential targets do not possess well-defined small-mol. binding determinates.  In this review, we discuss several successful campaigns to identify and leverage small-mol. inhibitors of the DNA repair proteins, from PARP1, a paradigm case for clin. successful small-mol. inhibitors, to coveted new targets, such as RAD51 recombinase, RAD52 DNA repair protein, MRE11 nuclease, and WRN DNA helicase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZweRKd9gHmLVg90H21EOLACvtfcHk0lhKODdkdoBqog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE&md5=b8b865f8df43100effd60037efdc96b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DHengel%26aufirst%3DS.%2BR.%26aulast%3DSpies%26aufirst%3DM.%2BA.%26aulast%3DSpies%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520targeting%2520DNA%2520repair%2520and%2520DNA%2520repair%2520deficiency%2520in%2520research%2520and%2520cancer%2520therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1101%26epage%3D1119%26doi%3D10.1016%2Fj.chembiol.2017.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karow, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s syndrome gene product promotes branch migration of holliday junctions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6508</span>, <span class="refDoi"> DOI: 10.1073/pnas.100448097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.100448097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10823897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktFajt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=6504-6508&author=J.+K.+Karowauthor=A.+Constantinouauthor=J.-L.+Liauthor=S.+C.+Westauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+syndrome+gene+product+promotes+branch+migration+of+holliday+junctions&doi=10.1073%2Fpnas.100448097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's syndrome gene product promotes branch migration of Holliday junctions</span></div><div class="casAuthors">Karow, Julia K.; Constantinou, Angelos; Li, Ji-Liang; West, Stephen C.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6504-6508</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bloom's syndrome (BS) is an autosomal recessive disorder assocd. with dwarfism, immunodeficiency, reduced fertility, and elevated levels of many types of cancer.  BS cells show marked genomic instability; in particular, hyperrecombination between sister chromatids and homologous chromosomes.  This instability is thought to result from defective processing of DNA replication intermediates.  The gene mutated in BS, BLM, encodes a member of the RecQ family of DExH box DNA helicases, which also includes the Werner's syndrome gene product.  The authors have investigated the mechanism by which BLM suppresses hyperrecombination.  Here, the authors show that BLM selectively binds Holliday junctions in vitro and acts on recombination intermediates contg. a Holliday junction to promote ATP-dependent branch migration.  The authors present a model in which BLM disrupts potentially recombinogenic mols. that arise at sites of stalled replication forks.  The authors' results have implications for the role of BLM as an anti-recombinase in the suppression of tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeN_anR_Fmi7Vg90H21EOLACvtfcHk0lhKODdkdoBqog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktFajt7o%253D&md5=de09613fb82612020f6e12a1695e1fb0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.100448097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.100448097%26sid%3Dliteratum%253Aachs%26aulast%3DKarow%26aufirst%3DJ.%2BK.%26aulast%3DConstantinou%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.-L.%26aulast%3DWest%26aufirst%3DS.%2BC.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520syndrome%2520gene%2520product%2520promotes%2520branch%2520migration%2520of%2520holliday%2520junctions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D6504%26epage%3D6508%26doi%3D10.1073%2Fpnas.100448097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budhathoki, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yodh, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balci, H.</span></span> <span> </span><span class="NLM_article-title">RecQ-core of BLM unfolds telomeric G-quadruplex in the absence of ATP</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">11528</span>– <span class="NLM_lpage">11545</span>, <span class="refDoi"> DOI: 10.1093/nar/gku856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgku856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25245947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKmurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=11528-11545&author=J.+B.+Budhathokiauthor=S.+Rayauthor=V.+Urbanauthor=P.+Janscakauthor=J.+G.+Yodhauthor=H.+Balci&title=RecQ-core+of+BLM+unfolds+telomeric+G-quadruplex+in+the+absence+of+ATP&doi=10.1093%2Fnar%2Fgku856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">RecQ-core of BLM unfolds telomeric G-quadruplex in the absence of ATP</span></div><div class="casAuthors">Budhathoki, Jagat B.; Ray, Sujay; Urban, Vaclav; Janscak, Pavel; Yodh, Jaya G.; Balci, Hamza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11528-11545</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Various helicases and single-stranded DNA (ss-DNA) binding proteins are known to destabilize G-quadruplex (GQ) structures, which otherwise result in genomic instability.  Bulk biochem. studies have shown that Bloom helicase (BLM) unfolds both intermol. and intramol. GQ in the presence of ATP.  Using single mol. FRET, we show that binding of RecQ-core of BLM (will be referred to as BLM) to ssDNA in the vicinity of an intramol. GQ leads to destabilization and unfolding of the GQ in the absence of ATP.  We show that the efficiency of BLM-mediated GQ unfolding correlates with the binding stability of BLM to ssDNA overhang, as modulated by the nucleotide state, ionic conditions, overhang length and overhang directionality.  In particular, we obsd. enhanced GQ unfolding by BLM in the presence of non-hydrolysable ATP analogs, which has implications for the underlying mechanism.  We also show that increasing GQ stability, via shorter loops or higher ionic strength, reduces BLM-mediated GQ unfolding.  Finally, we show that while WRN has similar activity as BLM, RecQ and RECQ5 helicases do not unfold GQ in the absence of ATP at physiol. ionic strength.  In summary, our study points to a novel and potentially very common mechanism of GQ destabilization mediated by proteins binding to the vicinity of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJvDxC7oRfHLVg90H21EOLACvtfcHk0lhKODdkdoBqog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKmurrK&md5=eb27480107144f5e8039039696f6e3ed</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku856%26sid%3Dliteratum%253Aachs%26aulast%3DBudhathoki%26aufirst%3DJ.%2BB.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DUrban%26aufirst%3DV.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DYodh%26aufirst%3DJ.%2BG.%26aulast%3DBalci%26aufirst%3DH.%26atitle%3DRecQ-core%2520of%2520BLM%2520unfolds%2520telomeric%2520G-quadruplex%2520in%2520the%2520absence%2520of%2520ATP%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D11528%26epage%3D11545%26doi%3D10.1093%2Fnar%2Fgku856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Fatah, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleskandarany, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshareeda, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic and protein expression analysis reveals clinicopathological significance of Bloom syndrome helicase (BLM) in breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1065</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1535-7163.mct-14-0939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25673821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlCitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1057-1065&author=A.+Aroraauthor=T.+M.+A.+Abdel-Fatahauthor=D.+Agarwalauthor=R.+Dohertyauthor=P.+M.+Moseleyauthor=M.+A.+Aleskandaranyauthor=A.+R.+Greenauthor=G.+Ballauthor=A.+T.+Alshareedaauthor=E.+A.+Rakhaauthor=S.+Y.+T.+Chanauthor=I.+O.+Ellisauthor=S.+Madhusudan&title=Transcriptomic+and+protein+expression+analysis+reveals+clinicopathological+significance+of+Bloom+syndrome+helicase+%28BLM%29+in+breast+cancer&doi=10.1158%2F1535-7163.mct-14-0939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic and Protein Expression Analysis Reveals Clinicopathological Significance of Bloom Syndrome Helicase (BLM) in Breast Cancer</span></div><div class="casAuthors">Arora, Arvind; Abdel-Fatah, Tarek M. A.; Agarwal, Devika; Doherty, Rachel; Moseley, Paul M.; Aleskandarany, Mohammed A.; Green, Andrew R.; Ball, Graham; Alshareeda, Alaa T.; Rakha, Emad A.; Chan, Stephen Y. T.; Ellis, Ian O.; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1057-1065</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bloom syndrome helicase (BLM) has key roles in homologous recombination repair, telomere maintenance, and DNA replication.  Germ-line mutations in the BLM gene causes Bloom syndrome, a rare disorder characterized by premature aging and predisposition to multiple cancers, including breast cancer.  The clinicopathol. significance of BLM in sporadic breast cancers is unknown.  We investigated BLM mRNA expression in the Mol. Taxonomy of Breast Cancer International Consortium cohort (n = 1,950) and validated in an external dataset of 2,413 tumors.  BLM protein level was evaluated in the Nottingham Tenovus series comprising 1,650 breast tumors.  BLM mRNA overexpression was significantly assocd. with high histol. grade, larger tumor size, estrogen receptor-neg. (ER-), progesterone receptor-neg. (PR-), and triple-neg. phenotypes (ps < 0.0001).  BLM mRNA overexpression was also linked to aggressive mol. phenotypes, including PAM50.Her2 (P < 0.0001), PAM50.Basal (P < 0.0001), and PAM50.LumB (P < 0.0001) and Genufu subtype (ER+/Her2-/high proliferation; P < 0.0001). (ER+/Her2-/low proliferation) were more likely to express low levels of BLM mRNA (ps < 0.0001).  Integrative mol. clusters (intClust) intClust.1 (P < 0.0001), intClust.5 (P < 0.0001), intClust.9 (P < 0.0001), and intClust.10 (P < 0.0001) were also more likely in tumors with high BLM mRNA expression.  BLM mRNA overexpression was assocd. with poor breast cancer-specific survival (BCSS; ps < 0.000001).  At the protein level, altered subcellular localization with high cytoplasmic BLM and low nuclear BLM was linked to aggressive phenotypes.  In multivariate anal., BLM mRNA and BLM protein levels independently influenced BCSS.  This is the first and the largest study to provide evidence that BLM is a promising biomarker in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgXUCTp8NP27Vg90H21EOLACvtfcHk0li5TPHTg2QYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlCitrw%253D&md5=8dc54a6fd7887f63f676c147de97273b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0939%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DAbdel-Fatah%26aufirst%3DT.%2BM.%2BA.%26aulast%3DAgarwal%26aufirst%3DD.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DMoseley%26aufirst%3DP.%2BM.%26aulast%3DAleskandarany%26aufirst%3DM.%2BA.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DBall%26aufirst%3DG.%26aulast%3DAlshareeda%26aufirst%3DA.%2BT.%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DChan%26aufirst%3DS.%2BY.%2BT.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DTranscriptomic%2520and%2520protein%2520expression%2520analysis%2520reveals%2520clinicopathological%2520significance%2520of%2520Bloom%2520syndrome%2520helicase%2520%2528BLM%2529%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3D1057%26epage%3D1065%26doi%3D10.1158%2F1535-7163.mct-14-0939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s syndrome helicase suppresses crossing over during homologous recombination</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1038/nature02253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature02253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=14685245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVGmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2003&pages=870-874&author=L.+Wuauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+syndrome+helicase+suppresses+crossing+over+during+homologous+recombination&doi=10.1038%2Fnature02253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's syndrome helicase suppresses crossing over during homologous recombination</span></div><div class="casAuthors">Wu, Leonard; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">6968</span>),
    <span class="NLM_cas:pages">870-874</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in BLM, which encodes a RecQ helicase, give rise to Bloom's syndrome, a disorder assocd. with cancer predisposition and genomic instability.  A defining feature of Bloom's syndrome is an elevated frequency of sister chromatid exchanges.  These arise from crossing over of chromatid arms during homologous recombination, a ubiquitous process that exists to repair DNA double-stranded breaks and damaged replication forks.  Whereas crossing over is required in meiosis, in mitotic cells it can be assocd. with detrimental loss of heterozygosity.  BLM forms an evolutionarily conserved complex with human topoisomerase IIIα (hTOPO IIIα), which can break and rejoin DNA to alter its topol.  Inactivation of homologs of either protein leads to hyper-recombination in unicellular organisms.  Here, the authors show that BLM and hTOPO IIIα together effect the resoln. of a recombination intermediate contg. a double Holliday junction.  The mechanism, which the authors term double-junction dissoln., is distinct from classical Holliday junction resoln. and prevents exchange of flanking sequences.  Loss of such an activity explains many of the cellular phenotypes of Bloom's syndrome.  These results have wider implications for our understanding of the process of homologous recombination and the mechanisms that exist to prevent tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUs7F5HuS1KrVg90H21EOLACvtfcHk0li5TPHTg2QYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVGmtbs%253D&md5=f98f21ab5b3557fe2edd1359678402fd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature02253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature02253%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%2520suppresses%2520crossing%2520over%2520during%2520homologous%2520recombination%26jtitle%3DNature%26date%3D2003%26volume%3D426%26spage%3D870%26epage%3D874%26doi%3D10.1038%2Fnature02253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bugreev, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egelman, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazin, A. V.</span></span> <span> </span><span class="NLM_article-title">Novel pro- and anti-recombination activities of the Bloom’s syndrome helicase</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3085</span>– <span class="NLM_lpage">3094</span>, <span class="refDoi"> DOI: 10.1101/gad.1609007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.1609007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18003860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3085-3094&author=D.+V.+Bugreevauthor=X.+Yuauthor=E.+H.+Egelmanauthor=A.+V.+Mazin&title=Novel+pro-+and+anti-recombination+activities+of+the+Bloom%E2%80%99s+syndrome+helicase&doi=10.1101%2Fgad.1609007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pro- and anti-recombination activities of the Bloom's syndrome helicase</span></div><div class="casAuthors">Bugreev, Dmitry V.; Yu, Xiong; Egelman, Edward H.; Mazin, Alexander V.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3085-3094</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bloom's syndrome (BS) is an autosomal recessive disorder characterized by a strong cancer predisposition.  The defining feature of BS is extreme genome instability.  The gene mutated in Bloom's syndrome, BLM, encodes a DNA helicase (BLM) of the RecQ family.  BLM plays a role in homologous recombination; however, its exact function remains controversial.  Mutations in the BLM cause hyperrecombination between sister chromatids and homologous chromosomes, indicating an anti-recombination role.  Conversely, other data show that BLM is required for recombination.  It was previously shown that in vitro BLM helicase promotes disruption of recombination intermediates, regression of stalled replication forks, and dissoln. of double Holliday junctions.  Here, we demonstrate two novel activities of BLM: disruption of the Rad51-ssDNA (single-stranded DNA) filament, an active species that promotes homologous recombination, and stimulation of DNA repair synthesis.  Using in vitro reconstitution reactions, we analyzed how different biochem. activities of BLM contribute to its functions in homologous recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr--SBOK6pjb7Vg90H21EOLACvtfcHk0li5TPHTg2QYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vP&md5=1ddadff75bf07fea1a83263eab68f80c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1101%2Fgad.1609007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1609007%26sid%3Dliteratum%253Aachs%26aulast%3DBugreev%26aufirst%3DD.%2BV.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DEgelman%26aufirst%3DE.%2BH.%26aulast%3DMazin%26aufirst%3DA.%2BV.%26atitle%3DNovel%2520pro-%2520and%2520anti-recombination%2520activities%2520of%2520the%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3085%26epage%3D3094%26doi%3D10.1101%2Fgad.1609007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misenko, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Her, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S. F.</span></span> <span> </span><span class="NLM_article-title">BLM helicase regulates DNA repair by counteracting RAD51 loading at DNA double-strand break sites</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>216</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3534</span>, <span class="refDoi"> DOI: 10.1083/jcb.201703144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1083%2Fjcb.201703144" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=216&publication_year=2017&pages=3521-3534&author=D.+S.+Patelauthor=S.+M.+Misenkoauthor=J.+Herauthor=S.+F.+Bunting&title=BLM+helicase+regulates+DNA+repair+by+counteracting+RAD51+loading+at+DNA+double-strand+break+sites&doi=10.1083%2Fjcb.201703144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201703144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201703144%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BS.%26aulast%3DMisenko%26aufirst%3DS.%2BM.%26aulast%3DHer%26aufirst%3DJ.%26aulast%3DBunting%26aufirst%3DS.%2BF.%26atitle%3DBLM%2520helicase%2520regulates%2520DNA%2520repair%2520by%2520counteracting%2520RAD51%2520loading%2520at%2520DNA%2520double-strand%2520break%2520sites%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2017%26volume%3D216%26spage%3D3521%26epage%3D3534%26doi%3D10.1083%2Fjcb.201703144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidorova, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauper, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emond, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnat, R. J.</span></span> <span> </span><span class="NLM_article-title">The human WRN and BLM RecQ helicases differentially regulate cell proliferation and survival after chemotherapeutic DNA damage</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">6548</span>– <span class="NLM_lpage">6555</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-0475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F0008-5472.can-10-0475" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=6548-6555&author=F.+J.+Maoauthor=J.+M.+Sidorovaauthor=J.+M.+Lauperauthor=M.+J.+Emondauthor=R.+J.+Monnat&title=The+human+WRN+and+BLM+RecQ+helicases+differentially+regulate+cell+proliferation+and+survival+after+chemotherapeutic+DNA+damage&doi=10.1158%2F0008-5472.can-10-0475"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-10-0475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-10-0475%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%2BJ.%26aulast%3DSidorova%26aufirst%3DJ.%2BM.%26aulast%3DLauper%26aufirst%3DJ.%2BM.%26aulast%3DEmond%26aufirst%3DM.%2BJ.%26aulast%3DMonnat%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520human%2520WRN%2520and%2520BLM%2520RecQ%2520helicases%2520differentially%2520regulate%2520cell%2520proliferation%2520and%2520survival%2520after%2520chemotherapeutic%2520DNA%2520damage%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D6548%26epage%3D6555%26doi%3D10.1158%2F0008-5472.can-10-0475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahkim Bennani-Belhaj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouzeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhagiar-Labarchede, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabosseau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onclercq-Delic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayart, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordelieres, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couturier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amor-Gueret, M.</span></span> <span> </span><span class="NLM_article-title">The Bloom syndrome protein limits the lethality associated with RAD51 deficiency</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-09-0534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1541-7786.mcr-09-0534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20215422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ptV2nsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=385-394&author=K.+Lahkim+Bennani-Belhajauthor=S.+Rouzeauauthor=G.+Buhagiar-Labarchedeauthor=P.+Chabosseauauthor=R.+Onclercq-Delicauthor=E.+Bayartauthor=F.+Cordelieresauthor=J.+Couturierauthor=M.+Amor-Gueret&title=The+Bloom+syndrome+protein+limits+the+lethality+associated+with+RAD51+deficiency&doi=10.1158%2F1541-7786.mcr-09-0534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom syndrome protein limits the lethality associated with RAD51 deficiency</span></div><div class="casAuthors">Lahkim Bennani-Belhaj Kenza; Rouzeau Sebastien; Buhagiar-Labarchede Geraldine; Chabosseau Pauline; Onclercq-Delic Rosine; Bayart Emilie; Cordelieres Fabrice; Couturier Jerome; Amor-Gueret Mounira</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer research : MCR</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Little is known about the functional interaction between the Bloom's syndrome protein (BLM) and the recombinase RAD51 within cells.  Using RNA interference technology, we provide the first demonstration that RAD51 acts upstream from BLM to prevent anaphase bridge formation.  RAD51 downregulation was associated with an increase in the frequency of BLM-positive anaphase bridges, but not of BLM-associated ultrafine bridges.  Time-lapse live microscopy analysis of anaphase bridge cells revealed that BLM promoted cell survival in the absence of Rad51.  Our results directly implicate BLM in limiting the lethality associated with RAD51 deficiency through the processing of anaphase bridges resulting from the RAD51 defect.  These findings provide insight into the molecular basis of some cancers possibly associated with variants of the RAD51 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSiFpylsdl8U8NIRw0GKUoEfW6udTcc2ebd59ktdeY7rrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ptV2nsw%253D%253D&md5=dd11389bab5efe0daa81c38ce3a8e9fe</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-09-0534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-09-0534%26sid%3Dliteratum%253Aachs%26aulast%3DLahkim%2BBennani-Belhaj%26aufirst%3DK.%26aulast%3DRouzeau%26aufirst%3DS.%26aulast%3DBuhagiar-Labarchede%26aufirst%3DG.%26aulast%3DChabosseau%26aufirst%3DP.%26aulast%3DOnclercq-Delic%26aufirst%3DR.%26aulast%3DBayart%26aufirst%3DE.%26aulast%3DCordelieres%26aufirst%3DF.%26aulast%3DCouturier%26aufirst%3DJ.%26aulast%3DAmor-Gueret%26aufirst%3DM.%26atitle%3DThe%2520Bloom%2520syndrome%2520protein%2520limits%2520the%2520lethality%2520associated%2520with%2520RAD51%2520deficiency%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2010%26volume%3D8%26spage%3D385%26epage%3D394%26doi%3D10.1158%2F1541-7786.mcr-09-0534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosedale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachrati, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">A small molecule inhibitor of the BLM helicase modulates chromosome stability in human cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.chembiol.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=23352139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=55-62&author=G.+H.+Nguyenauthor=T.+S.+Dexheimerauthor=A.+S.+Rosenthalauthor=W.+K.+Chuauthor=D.+K.+Singhauthor=G.+Mosedaleauthor=C.+Z.+Bachratiauthor=L.+Schultzauthor=M.+Sakuraiauthor=P.+Savitskyauthor=M.+Abuauthor=P.+J.+McHughauthor=V.+A.+Bohrauthor=C.+C.+Harrisauthor=A.+Jadhavauthor=O.+Gileadiauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=I.+D.+Hickson&title=A+small+molecule+inhibitor+of+the+BLM+helicase+modulates+chromosome+stability+in+human+cells&doi=10.1016%2Fj.chembiol.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span></div><div class="casAuthors">Nguyen, Giang Huong; Dexheimer, Thomas S.; Rosenthal, Andrew S.; Chu, Wai Kit; Singh, Dharmendra Kumar; Mosedale, Georgina; Bachrati, Csanad Z.; Schultz, Lena; Sakurai, Masaaki; Savitsky, Pavel; Abu, Mika; McHugh, Peter J.; Bohr, Vilhelm A.; Harris, Curtis C.; Jadhav, Ajit; Gileadi, Opher; Maloney, David J.; Simeonov, Anton; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bloom's syndrome protein, BLM, is a member of the conserved RecQ helicase family.  Although cell lines lacking BLM exist, these exhibit progressive genomic instability that makes distinguishing primary from secondary effects of BLM loss problematic.  In order to be able to acutely disable BLM function in cells, we undertook a high throughput screen of a chem. compd. library for small mol. inhibitors of BLM.  We present ML216, a potent inhibitor of the DNA unwinding activity of BLM.  ML216 shows cell-based activity and can induce sister chromatid exchanges, enhance the toxicity of aphidicolin, and exert antiproliferative activity in cells expressing BLM, but not those lacking BLM.  These data indicate that ML216 shows strong selectivity for BLM in cultured cells.  We discuss the potential utility of such a BLM-targeting compd. as an anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhJ5QlteU3zLVg90H21EOLACvtfcHk0liAAYb3L65HXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D&md5=26bfa8e45f2253aa8807d055be0689f2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DG.%2BH.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DChu%26aufirst%3DW.%2BK.%26aulast%3DSingh%26aufirst%3DD.%2BK.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DBachrati%26aufirst%3DC.%2BZ.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAbu%26aufirst%3DM.%26aulast%3DMcHugh%26aufirst%3DP.%2BJ.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DA%2520small%2520molecule%2520inhibitor%2520of%2520the%2520BLM%2520helicase%2520modulates%2520chromosome%2520stability%2520in%2520human%2520cells%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.chembiol.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of new selective NM23-H2 binders as c-MYC transcription inhibitors via disruption of the NM23-H2/G-Quadruplex interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6924</span>– <span class="NLM_lpage">6941</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00421</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00421" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6924-6941&author=Y.-Q.+Wangauthor=Z.-L.+Huangauthor=S.-B.+Chenauthor=C.-X.+Wangauthor=C.+Shanauthor=Q.-K.+Yinauthor=T.-M.+Ouauthor=D.+Liauthor=L.-Q.+Guauthor=J.-H.+Tanauthor=Z.-S.+Huang&title=Design%2C+synthesis%2C+and+evaluation+of+new+selective+NM23-H2+binders+as+c-MYC+transcription+inhibitors+via+disruption+of+the+NM23-H2%2FG-Quadruplex+interaction&doi=10.1021%2Facs.jmedchem.7b00421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of New Selective NM23-H2 Binders as c-MYC Transcription Inhibitors via Disruption of the NM23-H2/G-Quadruplex Interaction</span></div><div class="casAuthors">Wang, Yu-Qing; Huang, Zhou-Li; Chen, Shuo-Bin; Wang, Chen-Xi; Shan, Chan; Yin, Qi-Kun; Ou, Tian-Miao; Li, Ding; Gu, Lian-Quan; Tan, Jia-Heng; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6924-6941</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-MYC is one of the important human proto-oncogenes, and transcriptional factor NM23-H2 can activate c-MYC transcription by recognizing the G-quadruplex in the promoter of the gene.  Small mols. that inhibit c-MYC transcription by disrupting the NM23-H2/G-quadruplex interaction might be a promising strategy for developing selective anticancer agents.  In recent studies, we developed a series of isaindigotone derivs., which can bind to G-quadruplex and NM23-H2, thus down-regulating c-MYC (J. Med. Chem. 2017, 60, 1292-1308).  Herein, a series of novel isaindigotone derivs. were designed, synthesized, and screened for NM23-H2 selective binding ligands.  Among them, compd. I showed a high specific binding affinity to NM23-H2, effectively disrupting the interaction of NM23-H2 with G-quadruplex, and it strongly down-regulated c-MYC transcription.  Furthermore, I induced cell cycle arrest and apoptosis, and it exhibited good tumor growth inhibition in a mouse xenograft model.  This work provides a new strategy to modulate c-MYC transcription for the development of selective anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH6VlXYOPCY7Vg90H21EOLACvtfcHk0liAAYb3L65HXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jL&md5=7ba64697833652269acc158833534ce5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00421%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DHuang%26aufirst%3DZ.-L.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DWang%26aufirst%3DC.-X.%26aulast%3DShan%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DQ.-K.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.-Q.%26aulast%3DTan%26aufirst%3DJ.-H.%26aulast%3DHuang%26aufirst%3DZ.-S.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520new%2520selective%2520NM23-H2%2520binders%2520as%2520c-MYC%2520transcription%2520inhibitors%2520via%2520disruption%2520of%2520the%2520NM23-H2%252FG-Quadruplex%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6924%26epage%3D6941%26doi%3D10.1021%2Facs.jmedchem.7b00421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohaghegh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s and Werner’s syndrome proteins are DNA structure-specific helicases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2843</span>– <span class="NLM_lpage">2849</span>, <span class="refDoi"> DOI: 10.1093/nar/29.13.2843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2F29.13.2843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11433031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlejs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=2843-2849&author=P.+Mohagheghauthor=J.+K.+Karowauthor=R.+M.+Broshauthor=V.+A.+Bohrauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+and+Werner%E2%80%99s+syndrome+proteins+are+DNA+structure-specific+helicases&doi=10.1093%2Fnar%2F29.13.2843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's and Werner's syndrome proteins are DNA structure-specific helicases</span></div><div class="casAuthors">Mohaghegh, Payam; Karow, Julia K.; Brosh, Robert M., Jr.; Bohr, Vilhelm A.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2843-2849</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BLM and WRN, the products of the Bloom's and Werner's syndrome genes, are members of the RecQ family of DNA helicases.  Although both have been shown previously to unwind simple, partial duplex DNA substrates with 3'→5' polarity, little is known about the structural features of DNA that det. the substrate specificities of these enzymes.  We have compared the substrate specificities of the BLM and WRN proteins using a variety of partial duplex DNA mols., which are based upon a common core nucleotide sequence.  We show that neither BLM nor WRN is capable of unwinding duplex DNA from a blunt-ended terminus or from an internal nick.  However, both enzymes efficiently unwind the same blunt-ended duplex contg. a centrally located 12 nt single-stranded "bubble", as well as a synthetic X-structure (a model for the Holliday junction recombination intermediate) in which each "arm" of the 4-way junction is blunt-ended.  Surprisingly, a 3'-tailed duplex, a std. substrate for 3'→5' helicases, is unwound much less efficiently by BLM and WRN than are the bubble and X-structure substrates.  These data show conclusively that a single-stranded 3'-tail is not a structural requirement for unwinding of std. B-form DNA by these helicases.  BLM and WRN also both unwind a variety of different forms of G-quadruplex DNA, a structure that can form at guanine-rich sequences present at several genomic loci.  Our data indicate that BLM and WRN are atypical helicases that are highly DNA structure specific and have similar substrate specificities.  We interpret these data in the light of the genomic instability and hyper-recombination characteristics of cells from individuals with Bloom's or Werner's syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY0ZC4RKKynLVg90H21EOLACvtfcHk0liAAYb3L65HXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlejs7s%253D&md5=8a32f9fddbf47c0c54141d80b83431d5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F29.13.2843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F29.13.2843%26sid%3Dliteratum%253Aachs%26aulast%3DMohaghegh%26aufirst%3DP.%26aulast%3DKarow%26aufirst%3DJ.%2BK.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520and%2520Werner%25E2%2580%2599s%2520syndrome%2520proteins%2520are%2520DNA%2520structure-specific%2520helicases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2001%26volume%3D29%26spage%3D2843%26epage%3D2849%26doi%3D10.1093%2Fnar%2F29.13.2843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guirouilh-Barbat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacca, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Endogenous -H2AX-ATM-Chk2 Checkpoint Activation in Bloom’s Syndrome Helicase Deficient Cells Is Related to DNA Replication Arrested Forks</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-07-0028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1541-7786.mcr-07-0028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17634426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVChtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=713-724&author=V.+A.+Raoauthor=C.+Contiauthor=J.+Guirouilh-Barbatauthor=A.+Nakamuraauthor=Z.-H.+Miaoauthor=S.+L.+Daviesauthor=B.+Saccaauthor=I.+D.+Hicksonauthor=A.+Bensimonauthor=Y.+Pommier&title=Endogenous+-H2AX-ATM-Chk2+Checkpoint+Activation+in+Bloom%E2%80%99s+Syndrome+Helicase+Deficient+Cells+Is+Related+to+DNA+Replication+Arrested+Forks&doi=10.1158%2F1541-7786.mcr-07-0028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous γ-H2AX-ATM-Chk2 Checkpoint Activation in Bloom's Syndrome Helicase-Deficient Cells Is Related to DNA Replication Arrested Forks</span></div><div class="casAuthors">Rao, V. Ashutosh; Conti, Chiara; Guirouilh-Barbat, Josee; Nakamura, Asako; Miao, Ze-Hong; Davies, Sally L.; Sacca, Barbara; Hickson, Ian D.; Bensimon, Aaron; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">713-724</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The Bloom syndrome helicase (BLM) is crit. for genomic stability.  A defect in BLM activity results in the cancer-predisposing Bloom syndrome (BS).  Here, the authors report that BLM-deficient cell lines and primary fibroblasts display an endogenously activated DNA double-strand break checkpoint response with prominent levels of phosphorylated histone H2AX (γ-H2AX), Chk2 (pT68Chk2), and ATM (pS1981ATM) colocalizing in nuclear foci.  Interestingly, the mitotic fraction of γ-H2AX foci did not seem to be higher in BLM-deficient cells, indicating that these lesions form transiently during interphase.  Pulse labeling with iododeoxyuridine and immunofluorescence microscopy showed the colocalization of γ-H2AX, ATM, and Chk2 together with replication foci.  Those foci costained for Rad51, indicating homologous recombination at these replication sites.  The authors therefore analyzed replication in BS cells using a single mol. approach on combed DNA fibers.  In addn. to a higher frequency of replication fork barriers, BS cells displayed a reduced av. fork velocity and global redn. of interorigin distances indicative of an elevated frequency of origin firing.  Because BS is one of the most penetrant cancer-predisposing hereditary diseases, it is likely that the lack of BLM engages the cells in a situation similar to precancerous tissues with replication stress.  To the authors' knowledge, this is the first report of high ATM-Chk2 kinase activation and its linkage to replication defects in a BS model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ0QOY7Yc62rVg90H21EOLACvtfcHk0lg60nUw19LACw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvVChtb4%253D&md5=8d75a91c20823851a205af82d6d07a7a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-07-0028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-07-0028%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DV.%2BA.%26aulast%3DConti%26aufirst%3DC.%26aulast%3DGuirouilh-Barbat%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DA.%26aulast%3DMiao%26aufirst%3DZ.-H.%26aulast%3DDavies%26aufirst%3DS.%2BL.%26aulast%3DSacca%26aufirst%3DB.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DBensimon%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DEndogenous%2520-H2AX-ATM-Chk2%2520Checkpoint%2520Activation%2520in%2520Bloom%25E2%2580%2599s%2520Syndrome%2520Helicase%2520Deficient%2520Cells%2520Is%2520Related%2520to%2520DNA%2520Replication%2520Arrested%2520Forks%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2007%26volume%3D5%26spage%3D713%26epage%3D724%26doi%3D10.1158%2F1541-7786.mcr-07-0028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nimonkar, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozsoy, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genschel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczykowski, S. C.</span></span> <span> </span><span class="NLM_article-title">Human exonuclease 1 and BLM helicase interact to resect DNA and initiate DNA repair</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">16906</span>– <span class="NLM_lpage">16911</span>, <span class="refDoi"> DOI: 10.1073/pnas.0809380105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.0809380105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18971343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlKmu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16906-16911&author=A.+V.+Nimonkarauthor=A.+Z.+Ozsoyauthor=J.+Genschelauthor=P.+Modrichauthor=S.+C.+Kowalczykowski&title=Human+exonuclease+1+and+BLM+helicase+interact+to+resect+DNA+and+initiate+DNA+repair&doi=10.1073%2Fpnas.0809380105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Human exonuclease 1 and BLM helicase interact to resect DNA and initiate DNA repair</span></div><div class="casAuthors">Nimonkar, Amitabh V.; Ozsoy, A. Zeynep; Genschel, Jochen; Modrich, Paul; Kowalczykowski, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">16906-16911</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The error-free repair of double-stranded DNA breaks by homologous recombination requires processing of broken ends.  These processed ends are substrates for assembly of DNA strand exchange proteins that mediate DNA strand invasion.  Here, we establish that human BLM helicase, a member of the RecQ family, stimulates the nucleolytic activity of human exonuclease 1 (hExo1), a 5' → 3' double-stranded DNA exonuclease.  The stimulation is specific because other RecQ homologs fail to stimulate hExol.  Stimulation of DNA resection by hExo1 is independent of BLM helicase activity and is, instead, mediated by an interaction between the 2 proteins.  Finally, we show that DNA ends resected by hExo1 and BLM are used by human Rad51, but not its yeast or bacterial counterparts, to promote homologous DNA pairing.  This in vitro system recapitulates initial steps of homologous recombination and provides biochem. evidence for a role of BLM and Exo1 in the initiation of recombinational DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE87ZrQuRE7Vg90H21EOLACvtfcHk0lg60nUw19LACw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlKmu7nI&md5=c5c9381a86a5712595b55412f0bd8575</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0809380105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0809380105%26sid%3Dliteratum%253Aachs%26aulast%3DNimonkar%26aufirst%3DA.%2BV.%26aulast%3DOzsoy%26aufirst%3DA.%2BZ.%26aulast%3DGenschel%26aufirst%3DJ.%26aulast%3DModrich%26aufirst%3DP.%26aulast%3DKowalczykowski%26aufirst%3DS.%2BC.%26atitle%3DHuman%2520exonuclease%25201%2520and%2520BLM%2520helicase%2520interact%2520to%2520resect%2520DNA%2520and%2520initiate%2520DNA%2520repair%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D16906%26epage%3D16911%26doi%3D10.1073%2Fpnas.0809380105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasin, M.</span></span> <span> </span><span class="NLM_article-title">XRCC3 promotes homology-directed repair of DNA damage in mammalian cells</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2638</span>, <span class="refDoi"> DOI: 10.1101/gad.13.20.2633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.13.20.2633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10541549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADyaK1MXnt1Wqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=2633-2638&author=A.+J.+Pierceauthor=R.+D.+Johnsonauthor=L.+H.+Thompsonauthor=M.+Jasin&title=XRCC3+promotes+homology-directed+repair+of+DNA+damage+in+mammalian+cells&doi=10.1101%2Fgad.13.20.2633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">XRCC3 promotes homology-directed repair of DNA damage in mammalian cells</span></div><div class="casAuthors">Pierce, Andrew J.; Johnson, Roger D.; Thompson, Larry H.; Jasin, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2633-2638</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Homol.-directed repair of DNA damage has recently emerged as a major mechanism for the maintenance of genomic integrity in mammalian cells.  The highly conserved strand transferase, Rad51, is expected to be crit. for this process.  XRCC3 possesses a limited sequence similarity to Rad51 and interacts with it.  Using a novel fluorescence-based assay, the authors demonstrate here that error-free homol.-directed repair of DNA double-strand breaks is decreased 25-fold in an XRCC3-deficient hamster cell line and can be restored to wild-type levels through XRCC3 expression.  These results establish that XRCC3-mediated homologous recombination can reverse DNA damage that would otherwise be mutagenic or lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc5mIYaRKGbVg90H21EOLACvtfcHk0lg60nUw19LACw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnt1Wqsrk%253D&md5=8301dc45408d2f4ec6d5d6965f973d8d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.20.2633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.20.2633%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BJ.%26aulast%3DJohnson%26aufirst%3DR.%2BD.%26aulast%3DThompson%26aufirst%3DL.%2BH.%26aulast%3DJasin%26aufirst%3DM.%26atitle%3DXRCC3%2520promotes%2520homology-directed%2520repair%2520of%2520DNA%2520damage%2520in%2520mammalian%2520cells%26jtitle%3DGenes%2520Dev.%26date%3D1999%26volume%3D13%26spage%3D2633%26epage%3D2638%26doi%3D10.1101%2Fgad.13.20.2633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Traverso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettegowda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzler, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span> <span> </span><span class="NLM_article-title">Hyper-recombination and genetic instability in BLM-deficient epithelial cells</span>. <i>Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8578</span>– <span class="NLM_lpage">8581</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=8578-8581&author=G.+Traversoauthor=C.+Bettegowdaauthor=J.+Krausauthor=M.+R.+Speicherauthor=K.+W.+Kinzlerauthor=B.+Vogelsteinauthor=C.+Lengauer&title=Hyper-recombination+and+genetic+instability+in+BLM-deficient+epithelial+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraverso%26aufirst%3DG.%26aulast%3DBettegowda%26aufirst%3DC.%26aulast%3DKraus%26aufirst%3DJ.%26aulast%3DSpeicher%26aufirst%3DM.%2BR.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DHyper-recombination%2520and%2520genetic%2520instability%2520in%2520BLM-deficient%2520epithelial%2520cells%26jtitle%3DCancer%2520Res%26date%3D2003%26volume%3D63%26spage%3D8578%26epage%3D8581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nimonkar, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genschel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polaczek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczykowski, S. C.</span></span> <span> </span><span class="NLM_article-title">BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1101/gad.2003811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.2003811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=21325134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVait7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=350-362&author=A.+V.+Nimonkarauthor=J.+Genschelauthor=E.+Kinoshitaauthor=P.+Polaczekauthor=J.+L.+Campbellauthor=C.+Wymanauthor=P.+Modrichauthor=S.+C.+Kowalczykowski&title=BLM-DNA2-RPA-MRN+and+EXO1-BLM-RPA-MRN+constitute+two+DNA+end+resection+machineries+for+human+DNA+break+repair&doi=10.1101%2Fgad.2003811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair</span></div><div class="casAuthors">Nimonkar, Amitabh V.; Genschel, Jochen; Kinoshita, Eri; Polaczek, Piotr; Campbell, Judith L.; Wyman, Claire; Modrich, Paul; Kowalczykowski, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">350-362</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Repair of dsDNA breaks requires processing to produce 3'-terminated ssDNA.  We biochem. reconstituted DNA end resection using purified human proteins: Bloom helicase (BLM); DNA2 helicase/nuclease; Exonuclease 1 (EXO1); the complex comprising MRE11, RAD50, and NBS1 (MRN); and Replication protein A (RPA).  Resection occurs via two routes.  In one, BLM and DNA2 phys. and specifically interact to resect DNA in a process that is ATP-dependent and requires BLM helicase and DNA2 nuclease functions.  RPA is essential for both DNA unwinding by BLM and enforcing 5' → 3' resection polarity by DNA2.  MRN accelerates processing by recruiting BLM to the end.  In the other, EXO1 resects the DNA and is stimulated by BLM, MRN, and RPA.  BLM increases the affinity of EXO1 for ends, and MRN recruits and enhances the processivity of EXO1.  Our results establish two of the core machineries that initiate recombinational DNA repair in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfq3dsij9QbbVg90H21EOLACvtfcHk0lh4sef-VRynBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVait7Y%253D&md5=794da119abc7dbadc0ce8aebafbf027b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1101%2Fgad.2003811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2003811%26sid%3Dliteratum%253Aachs%26aulast%3DNimonkar%26aufirst%3DA.%2BV.%26aulast%3DGenschel%26aufirst%3DJ.%26aulast%3DKinoshita%26aufirst%3DE.%26aulast%3DPolaczek%26aufirst%3DP.%26aulast%3DCampbell%26aufirst%3DJ.%2BL.%26aulast%3DWyman%26aufirst%3DC.%26aulast%3DModrich%26aufirst%3DP.%26aulast%3DKowalczykowski%26aufirst%3DS.%2BC.%26atitle%3DBLM-DNA2-RPA-MRN%2520and%2520EXO1-BLM-RPA-MRN%2520constitute%2520two%2520DNA%2520end%2520resection%2520machineries%2520for%2520human%2520DNA%2520break%2520repair%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D350%26epage%3D362%26doi%3D10.1101%2Fgad.2003811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span> <span> </span><span class="NLM_article-title">MRN complex-dependent recruitment of ubiquitylated BLM helicase to DSBs negatively regulates DNA repair pathways</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1016</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-03393-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fs41467-018-03393-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=29523790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC1MrpsF2hug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1016&author=V.+Tripathiauthor=H.+Agarwalauthor=S.+Priyaauthor=H.+Batraauthor=P.+Modiauthor=M.+Pandeyauthor=D.+Sahaauthor=S.+C.+Raghavanauthor=S.+Sengupta&title=MRN+complex-dependent+recruitment+of+ubiquitylated+BLM+helicase+to+DSBs+negatively+regulates+DNA+repair+pathways&doi=10.1038%2Fs41467-018-03393-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MRN complex-dependent recruitment of ubiquitylated BLM helicase to DSBs negatively regulates DNA repair pathways</span></div><div class="casAuthors">Tripathi Vivek; Agarwal Himanshi; Priya Swati; Batra Harish; Modi Priyanka; Sengupta Sagar; Pandey Monica; Raghavan Sathees C; Saha Dhurjhoti</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1016</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mutations in BLM in Bloom Syndrome patients predispose them to multiple types of cancers.  Here we report that BLM is recruited in a biphasic manner to annotated DSBs.  BLM recruitment is dependent on the presence of NBS1, MRE11 and ATM.  While ATM activity is essential for BLM recruitment in early phase, it is dispensable in late phase when MRE11 exonuclease activity and RNF8-mediated ubiquitylation of BLM are the key determinants.  Interaction between polyubiquitylated BLM and NBS1 is essential for the helicase to be retained at the DSBs.  The helicase activity of BLM is required for the recruitment of HR and c-NHEJ factors onto the chromatin in S- and G1-phase, respectively.  During the repair phase, BLM inhibits HR in S-phase and c-NHEJ in G1-phase.  Consequently, inhibition of helicase activity of BLM enhances the rate of DNA alterations.  Thus BLM utilizes its pro- and anti-repair functions to maintain genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAACZdsi8QiZI8NvIb08rbfW6udTcc2eaNLrcmHs0z-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrpsF2hug%253D%253D&md5=c537c155acda024a05e7e6ef91c61698</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-03393-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-03393-8%26sid%3Dliteratum%253Aachs%26aulast%3DTripathi%26aufirst%3DV.%26aulast%3DAgarwal%26aufirst%3DH.%26aulast%3DPriya%26aufirst%3DS.%26aulast%3DBatra%26aufirst%3DH.%26aulast%3DModi%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DM.%26aulast%3DSaha%26aufirst%3DD.%26aulast%3DRaghavan%26aufirst%3DS.%2BC.%26aulast%3DSengupta%26aufirst%3DS.%26atitle%3DMRN%2520complex-dependent%2520recruitment%2520of%2520ubiquitylated%2520BLM%2520helicase%2520to%2520DSBs%2520negatively%2520regulates%2520DNA%2520repair%2520pathways%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1016%26doi%3D10.1038%2Fs41467-018-03393-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Honma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuraba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takashima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, M.</span></span> <span> </span><span class="NLM_article-title">Non-homologous end-joining for repairing I-SceI-induced DNA double strand breaks in human cells</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2007.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17296333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFOkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=781-788&author=M.+Honmaauthor=M.+Sakurabaauthor=T.+Koizumiauthor=Y.+Takashimaauthor=H.+Sakamotoauthor=M.+Hayashi&title=Non-homologous+end-joining+for+repairing+I-SceI-induced+DNA+double+strand+breaks+in+human+cells&doi=10.1016%2Fj.dnarep.2007.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Non-homologous end-joining for repairing I-SceI-induced DNA double strand breaks in human cells</span></div><div class="casAuthors">Honma, Masamitsu; Sakuraba, Mayumi; Koizumi, Tomoko; Takashima, Yoshio; Sakamoto, Hiroko; Hayashi, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA double strand breaks (DSBs) are usually repaired through either non-homologous end-joining (NHEJ) or homologous recombination (HR).  While HR is basically error-free repair, NHEJ is a mutagenic pathway that leads to deletion.  NHEJ must be precisely regulated to maintain genomic integrity.  To clarify the role of NHEJ, we investigated the genetic consequences of NHEJ repair of DSBs in human cells.  Human lymphoblastoid cell lines TSCE5 and TSCE105 have, resp., single and double I-SceI endonuclease sites in the endogenous thymidine kinase gene (TK) located on chromosome 17q.  I-SceI expression generated DSBs at the TK gene.  We used the novel transfection system (Amaxa Nucleofector) to introduce an I-SceI expression vector into the cells and randomly isolated clones.  We found mutations involved in the DSBs in the TK gene in 3% of TSCE5 cells and 30% of TSCE105 cell clones.  Most of the mutations in TSCE5 were small (1-30 bp) deletions with a 0-4 bp microhomol. at the junction.  The others consisted of large (>60) bp deletions, an insertion, and a rearrangement.  Mutants resulting from interallelic HR also occurred, but infrequently.  Most of the mutations in TSCE105, on the other hand, were deletions that encompassed the two I-SceI sites generated by NHEJ at DSBs.  The sequence joint was similar to that found in TSCE5 mutants.  Interestingly, some mutants formed a new I-SceI site by perfectly joining the two original I-SceI sites without deletion of the broken-ends.  These results support the idea that NHEJ for repairing I-SceI-induced DSBs mainly results in small or no deletions.  Thus, NHEJ must help maintain genomic integrity in mammalian cells by repairing DSBs as well as by preventing many deleterious alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzF7gA3vX_fLVg90H21EOLACvtfcHk0lh4sef-VRynBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFOkt78%253D&md5=504da114588a8d895b8b308f8386c442</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DM.%26aulast%3DSakuraba%26aufirst%3DM.%26aulast%3DKoizumi%26aufirst%3DT.%26aulast%3DTakashima%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DH.%26aulast%3DHayashi%26aufirst%3DM.%26atitle%3DNon-homologous%2520end-joining%2520for%2520repairing%2520I-SceI-induced%2520DNA%2520double%2520strand%2520breaks%2520in%2520human%2520cells%26jtitle%3DDNA%2520Repair%26date%3D2007%26volume%3D6%26spage%3D781%26epage%3D788%26doi%3D10.1016%2Fj.dnarep.2007.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scully, R.</span></span> <span> </span><span class="NLM_article-title">BRCA1, BRCA2, and Rad51 operate in a common DNA damage response pathway</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1756</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1999&pages=1752-1756&author=J.+J.+Chenauthor=D.+Silverauthor=S.+Cantorauthor=D.+M.+Livingstonauthor=R.+Scully&title=BRCA1%2C+BRCA2%2C+and+Rad51+operate+in+a+common+DNA+damage+response+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DSilver%26aufirst%3DD.%26aulast%3DCantor%26aufirst%3DS.%26aulast%3DLivingston%26aufirst%3DD.%2BM.%26aulast%3DScully%26aufirst%3DR.%26atitle%3DBRCA1%252C%2520BRCA2%252C%2520and%2520Rad51%2520operate%2520in%2520a%2520common%2520DNA%2520damage%2520response%2520pathway%26jtitle%3DCancer%2520Res.%26date%3D1999%26volume%3D59%26spage%3D1752%26epage%3D1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaina, B.</span></span> <span> </span><span class="NLM_article-title">DNA damage and the balance between survival and death in cancer biology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrc.2015.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc.2015.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26678314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVWqtrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=20-33&author=W.+P.+Roosauthor=A.+D.+Thomasauthor=B.+Kaina&title=DNA+damage+and+the+balance+between+survival+and+death+in+cancer+biology&doi=10.1038%2Fnrc.2015.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage and the balance between survival and death in cancer biology</span></div><div class="casAuthors">Roos, Wynand P.; Thomas, Adam D.; Kaina, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-33</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA is vulnerable to damage resulting from endogenous metabolites, environmental and dietary carcinogens, some anti-inflammatory drugs, and genotoxic cancer therapeutics.  Cells respond to DNA damage by activating complex signalling networks that decide cell fate, promoting not only DNA repair and survival but also cell death.  The decision between cell survival and death following DNA damage rests on factors that are involved in DNA damage recognition, and DNA repair and damage tolerance, as well as on factors involved in the activation of apoptosis, necrosis, autophagy and senescence.  The pathways that dictate cell fate are entwined and have key roles in cancer initiation and progression.  Furthermore, they det. the outcome of cancer therapy with genotoxic drugs.  Understanding the mol. basis of these pathways is important not only for gaining insight into carcinogenesis, but also in promoting successful cancer therapy.  In this Review, we describe key decision-making nodes in the complex interplay between cell survival and death following DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp63s4k3-gcbVg90H21EOLACvtfcHk0lh10G4KnnvPJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVWqtrvO&md5=443e30f3b7b6ff733eeda166e703b7ef</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2015.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2015.2%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DW.%2BP.%26aulast%3DThomas%26aufirst%3DA.%2BD.%26aulast%3DKaina%26aufirst%3DB.%26atitle%3DDNA%2520damage%2520and%2520the%2520balance%2520between%2520survival%2520and%2520death%2520in%2520cancer%2520biology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D20%26epage%3D33%26doi%3D10.1038%2Fnrc.2015.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">RecQ helicase BLM regulates prostate cancer cell proliferation and apoptosis</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4206</span>– <span class="NLM_lpage">4212</span>, <span class="refDoi"> DOI: 10.3892/ol.2017.6704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.3892%2Fol.2017.6704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=28943928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhvFSlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=4206-4212&author=X.+Qianauthor=S.+Fengauthor=D.+Xieauthor=D.+Fengauthor=Y.+Jiangauthor=X.+Zhang&title=RecQ+helicase+BLM+regulates+prostate+cancer+cell+proliferation+and+apoptosis&doi=10.3892%2Fol.2017.6704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RecQ helicase BLM regulates prostate cancer cell proliferation and apoptosis</span></div><div class="casAuthors">Qian Xiaosong; Feng Sujuan; Xie Dawei; Feng Dalin; Jiang Yihang; Zhang Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">4206-4212</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">Prostate cancer (PCa) is a common malignant tumor and the second leading cause of morbidity and mortality in men worldwide.  Considering the prevalence and effects of PCa in males, an understanding of the molecular mechanisms underlying PCa tumorigenesis are essential and may provide novel therapeutic strategies for treating PCa.  Bloom syndrome protein (BLM) is a member of the RecQ helicase family.  The major function of BLM is to uncoil the double-stranded DNA structure.  It has previously been demonstrated that BLM acts as a 'genome caretaker', and dysregulation of BLM function has been implicated in the development of multiple tumor types; however, its potential for inducing PCa tumorigenesis remains undetermined.  The present study aimed to explore the function of BLM in PCa progression.  Reverse transcription-polymerase chain reaction, immunohistochemistry and western blot analyses were performed to detect the BLM expression pattern in PCa patients and cell lines.  The proliferation, and migration and invasion capacities of prostate cells were determined by EdU and Transwell assays following transfection with BLM-targeting short hairpin RNA (shRNA).  The expression of BLM was significantly increased in PCa tissues and PC3 cells compared with non-PCa tissues and benign prostatic hyperplasia cells.  Knockdown of BLM via shRNA inhibited PCa cell proliferation, and promoted PCa cell apoptosis.  Notably, reducing the expression of BLM had no effect on the migration or invasive abilities of PCa cells.  These results suggest that downregulation of BLM may alleviate PCa development, providing a novel perspective for PCa tumorigenesis and a potential therapeutic target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR02yiRS-GsPyM8MEzWGjS9fW6udTcc2ebkZzyGwB_Vzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhvFSlsA%253D%253D&md5=0535a20b3078a465b668fad530888d28</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6704%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DRecQ%2520helicase%2520BLM%2520regulates%2520prostate%2520cancer%2520cell%2520proliferation%2520and%2520apoptosis%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D4206%26epage%3D4212%26doi%3D10.3892%2Fol.2017.6704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sears, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooney, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin-Sinex, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchi, J. J.</span></span> <span> </span><span class="NLM_article-title">DNA damage response (DDR) pathway engagement in cisplatin radiosensitization of non-small cell lung cancer</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2016.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2016.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26991853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=35-46&author=C.+R.+Searsauthor=S.+A.+Cooneyauthor=H.+Chin-Sinexauthor=M.+S.+Mendoncaauthor=J.+J.+Turchi&title=DNA+damage+response+%28DDR%29+pathway+engagement+in+cisplatin+radiosensitization+of+non-small+cell+lung+cancer&doi=10.1016%2Fj.dnarep.2016.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage response (DDR) pathway engagement in cisplatin radiosensitization of non-small cell lung cancer</span></div><div class="casAuthors">Sears, Catherine R.; Cooney, Sean A.; Chin-Sinex, Helen; Mendonca, Marc S.; Turchi, John J.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-46</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-small cell lung cancers (NSCLC) are commonly treated with a platinum-based chemotherapy such as cisplatin (CDDP) in combination with ionizing radiation (IR).  Although clin. trials have demonstrated that the combination of CDDP and IR appear to be synergistic in terms of therapeutic efficacy, the mechanism of synergism remains largely uncharacterized.  We investigated the role of the DNA damage response (DDR) in CDDP radiosensitization using two NSCLC cell lines.  Using clonogenic survival assays, we detd. that the cooperative cytotoxicity of CDDP and IR treatment is sequence dependent, requiring administration of CDDP prior to IR (CDDP-IR).  We identified and interrogated the unique time and agent-dependent activation of the DDR in NSCLC cells treated with cisplatin-IR combination therapy.  Compared to treatment with CDDP or IR alone, CDDP-IR combination treatment led to persistence of γH2Ax foci, a marker of DNA double-strand breaks (DSB), for up to 24 h after treatment.  Interestingly, pharmacol. inhibition of DDR sensor kinases revealed the persistence of γ-H2Ax foci in CDDP-IR treated cells is independent of kinase activation.  Taken together, our data suggest that delayed repair of DSBs in NSCLC cells treated with CDDP-IR contributes to CDDP radiosensitization and that alterations of the DDR pathways by inhibition of specific DDR kinases can augment CDDP-IR cytotoxicity by a complementary mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTraxlo6nDrVg90H21EOLACvtfcHk0lh10G4KnnvPJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWgt7g%253D&md5=95038974165090ad89ad757d278b75ce</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2016.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2016.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DSears%26aufirst%3DC.%2BR.%26aulast%3DCooney%26aufirst%3DS.%2BA.%26aulast%3DChin-Sinex%26aufirst%3DH.%26aulast%3DMendonca%26aufirst%3DM.%2BS.%26aulast%3DTurchi%26aufirst%3DJ.%2BJ.%26atitle%3DDNA%2520damage%2520response%2520%2528DDR%2529%2520pathway%2520engagement%2520in%2520cisplatin%2520radiosensitization%2520of%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DDNA%2520Repair%26date%3D2016%26volume%3D40%26spage%3D35%26epage%3D46%26doi%3D10.1016%2Fj.dnarep.2016.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagelbaum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenberg, E.</span></span> <span> </span><span class="NLM_article-title">Mechanistic insight into the interaction of BLM helicase with intra-strand G-quadruplex structures</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5556</span>, <span class="refDoi"> DOI: 10.1038/ncomms6556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fncomms6556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25418155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVCis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5556&author=S.+Chatterjeeauthor=J.+Zagelbaumauthor=P.+Savitskyauthor=A.+Sturzeneggerauthor=D.+Huttnerauthor=P.+Janscakauthor=I.+D.+Hicksonauthor=O.+Gileadiauthor=E.+Rothenberg&title=Mechanistic+insight+into+the+interaction+of+BLM+helicase+with+intra-strand+G-quadruplex+structures&doi=10.1038%2Fncomms6556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into the interaction of BLM helicase with intra-strand G-quadruplex structures</span></div><div class="casAuthors">Chatterjee, Sujoy; Zagelbaum, Jennifer; Savitsky, Pavel; Sturzenegger, Andreas; Huttner, Diana; Janscak, Pavel; Hickson, Ian D.; Gileadi, Opher; Rothenberg, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5556</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bloom syndrome is an autosomal recessive disorder caused by mutations in the RecQ family helicase BLM that is assocd. with growth retardation and predisposition to cancer.  BLM helicase has a high specificity for non-canonical G-quadruplex (G4) DNA structures, which are formed by G-rich DNA strands and play an important role in the maintenance of genomic integrity.  Here we used single-mol. FRET to define the mechanism of interaction of BLM helicase with intra-stranded G4 structures.  We show that the activity of BLM is substrate dependent, and highly regulated by a short-strand DNA (ssDNA) segment that separates the G4 motif from double-stranded DNA (dsDNA).  We demonstrate cooperativity between the RQC (RecQ C-terminal) and HRDC (helicase and RNaseD C-terminal) domains of BLM during binding and unfolding of the G4 structure, where the RQC domain interaction with G4 is stabilized by HRDC binding to ssDNA.  We present a model that proposes a unique role for G4 structures in modulating the activity of DNA processing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzxOggEhZ6XbVg90H21EOLACvtfcHk0lhADUr-_dEvcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVCis7k%253D&md5=d855358965757181f48cf7d1c9e6e535</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms6556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6556%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DZagelbaum%26aufirst%3DJ.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DSturzenegger%26aufirst%3DA.%26aulast%3DHuttner%26aufirst%3DD.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DRothenberg%26aufirst%3DE.%26atitle%3DMechanistic%2520insight%2520into%2520the%2520interaction%2520of%2520BLM%2520helicase%2520with%2520intra-strand%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5556%26doi%3D10.1038%2Fncomms6556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerston, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bizard, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özer, Ö.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarlós, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Bloom’s syndrome helicase indicates a role for the HRDC domain in conformational changes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">5221</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgkv373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25901030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFait7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=5221-5235&author=J.+A.+Newmanauthor=P.+Savitskyauthor=C.+K.+Allerstonauthor=A.+H.+Bizardauthor=%C3%96.+%C3%96zerauthor=K.+Sarl%C3%B3sauthor=Y.+Liuauthor=E.+Pardonauthor=J.+Steyaertauthor=I.+D.+Hicksonauthor=O.+Gileadi&title=Crystal+structure+of+the+Bloom%E2%80%99s+syndrome+helicase+indicates+a+role+for+the+HRDC+domain+in+conformational+changes&doi=10.1093%2Fnar%2Fgkv373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Bloom's syndrome helicase indicates a role for the HRDC domain in conformational changes</span></div><div class="casAuthors">Newman, Joseph A.; Savitsky, Pavel; Allerston, Charles K.; Bizard, Anna H.; Ozer, Ozgun; Sarlos, Kata; Liu, Ying; Pardon, Els; Steyaert, Jan; Hickson, Ian D.; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5221-5235</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bloom syndrome helicase (BLM) is a member of the RecQ family of DNA helicases, which play key roles in the maintenance of genome integrity in all organism groups.  Here, the authors describe crystal structures of the BLM helicase domain in complex with DNA and with an antibody fragment, as well as SAXS and domain assocn. studies in soln.  The authors showed an unexpected nucleotide-dependent interaction of the core helicase domain with the conserved, poorly characterized HRDC domain.  The BLM-DNA complex showed an unusual base-flipping mechanism with unique positioning of the DNA duplex relative to the helicase core domains.  Comparison with other crystal structures of RecQ helicases permitted the definition of structural transitions underlying ATP-driven helicase action, and the identification of a nucleotide-regulated tunnel that may play a role in interactions with complex DNA substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHuEtr4ILMbrVg90H21EOLACvtfcHk0lhADUr-_dEvcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFait7zP&md5=3b4c02568175564af56d1cc44891dd42</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv373%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAllerston%26aufirst%3DC.%2BK.%26aulast%3DBizard%26aufirst%3DA.%2BH.%26aulast%3D%25C3%2596zer%26aufirst%3D%25C3%2596.%26aulast%3DSarl%25C3%25B3s%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%2520indicates%2520a%2520role%2520for%2520the%2520HRDC%2520domain%2520in%2520conformational%2520changes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D5221%26epage%3D5235%26doi%3D10.1093%2Fnar%2Fgkv373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">The dynamic multisite interactions between two intrinsically disordered proteins</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7515</span>– <span class="NLM_lpage">7519</span>, <span class="refDoi"> DOI: 10.1002/anie.201701883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fanie.201701883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot12isLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=7515-7519&author=S.+Wuauthor=D.+Wangauthor=J.+Liuauthor=Y.+Fengauthor=J.+Wengauthor=Y.+Liauthor=X.+Gaoauthor=J.+Liuauthor=W.+Wang&title=The+dynamic+multisite+interactions+between+two+intrinsically+disordered+proteins&doi=10.1002%2Fanie.201701883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic multisite interactions between two intrinsically disordered proteins</span></div><div class="casAuthors">Wu, Shaowen; Wang, Dongdong; Liu, Jin; Feng, Yitao; Weng, Jingwei; Li, Yu; Gao, Xin; Liu, Jianwei; Wang, Wenning</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7515-7519</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein interactions involving intrinsically disordered proteins (IDPs) comprise a variety of binding modes, from the well-characterized folding upon binding to dynamic fuzzy complexes.  To date, most studies concern the binding of an IDP to a structured protein, while the interaction between two IDPs is poorly understood.  In this study, NMR, smFRET, and mol. dynamics (MD) simulation are combined to characterize the interaction between two IDPs, the C-terminal domain (CTD) of protein 4.1G and the nuclear mitotic app. (NuMA) protein.  It is revealed that CTD and NuMA form a fuzzy complex with remaining structural disorder.  Multiple binding sites on both proteins were identified by mol. dynamics and mutagenesis studies.  This study provides an at. scenario in which two IDPs bearing multiple binding sites interact with each other in dynamic equil.  The combined approach employed here could be widely applicable for investigating IDPs and their dynamic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtk7C2PKKDibVg90H21EOLACvtfcHk0lhADUr-_dEvcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot12isLg%253D&md5=a4e9878f24ffa2bc17cd987642b42219</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701883%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DThe%2520dynamic%2520multisite%2520interactions%2520between%2520two%2520intrinsically%2520disordered%2520proteins%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D7515%26epage%3D7519%26doi%3D10.1002%2Fanie.201701883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottiger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaffen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leutwyler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachorz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klopper, W.</span></span> <span> </span><span class="NLM_article-title">Strong N–H···π Hydrogen Bonding in Amide–Benzene Interactions</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2937</span>– <span class="NLM_lpage">2943</span>, <span class="refDoi"> DOI: 10.1021/jp8110474</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp8110474" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2937-2943&author=P.+Ottigerauthor=C.+Pfaffenauthor=R.+Leistauthor=S.+Leutwylerauthor=R.+A.+Bachorzauthor=W.+Klopper&title=Strong+N%E2%80%93H%C2%B7%C2%B7%C2%B7%CF%80+Hydrogen+Bonding+in+Amide%E2%80%93Benzene+Interactions&doi=10.1021%2Fjp8110474"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjp8110474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp8110474%26sid%3Dliteratum%253Aachs%26aulast%3DOttiger%26aufirst%3DP.%26aulast%3DPfaffen%26aufirst%3DC.%26aulast%3DLeist%26aufirst%3DR.%26aulast%3DLeutwyler%26aufirst%3DS.%26aulast%3DBachorz%26aufirst%3DR.%2BA.%26aulast%3DKlopper%26aufirst%3DW.%26atitle%3DStrong%2520N%25E2%2580%2593H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580%2520Hydrogen%2520Bonding%2520in%2520Amide%25E2%2580%2593Benzene%2520Interactions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D2937%26epage%3D2943%26doi%3D10.1021%2Fjp8110474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlakos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendorf, J. R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arif, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tizzard, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliev, A. E.</span></span> <span> </span><span class="NLM_article-title">Noncovalent Interactions of π Systems with Sulfur: The Atomic Chameleon of Molecular Recognition</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1002/anie.201708485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fanie.201708485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Cqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1193-1198&author=W.+B.+Motherwellauthor=R.+B.+Morenoauthor=I.+Pavlakosauthor=J.+R.+T.+Arendorfauthor=T.+Arifauthor=G.+J.+Tizzardauthor=S.+J.+Colesauthor=A.+E.+Aliev&title=Noncovalent+Interactions+of+%CF%80+Systems+with+Sulfur%3A+The+Atomic+Chameleon+of+Molecular+Recognition&doi=10.1002%2Fanie.201708485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Noncovalent Interactions of π Systems with Sulfur: The Atomic Chameleon of Molecular Recognition</span></div><div class="casAuthors">Motherwell, William B.; Moreno, Rafael B.; Pavlakos, Ilias; Arendorf, Josephine R. T.; Arif, Tanzeel; Tizzard, Graham J.; Coles, Simon J.; Aliev, Abil E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1198</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The relative strength of noncovalent interactions between a thioether sulfur atom and various π systems in designed top pan mol. balances was detd. by NMR spectroscopy.  Compared to its oxygen counterpart, the sulfur atom displays a remarkable ability to interact with almost equal facility over the entire range of π systems studied, with the simple alkene emerging as the most powerful partner.  With the exception of the O···heteroarene interaction, all noncovalent interactions of sulfur with π systems are favored over oxygen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsh-5TchNJm7Vg90H21EOLACvtfcHk0ljQyid0m1xMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Cqug%253D%253D&md5=8811fc6c0721c31c36e4d07d875f1f8b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.201708485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201708485%26sid%3Dliteratum%253Aachs%26aulast%3DMotherwell%26aufirst%3DW.%2BB.%26aulast%3DMoreno%26aufirst%3DR.%2BB.%26aulast%3DPavlakos%26aufirst%3DI.%26aulast%3DArendorf%26aufirst%3DJ.%2BR.%2BT.%26aulast%3DArif%26aufirst%3DT.%26aulast%3DTizzard%26aufirst%3DG.%2BJ.%26aulast%3DColes%26aufirst%3DS.%2BJ.%26aulast%3DAliev%26aufirst%3DA.%2BE.%26atitle%3DNoncovalent%2520Interactions%2520of%2520%25CF%2580%2520Systems%2520with%2520Sulfur%253A%2520The%2520Atomic%2520Chameleon%2520of%2520Molecular%2520Recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1193%26epage%3D1198%26doi%3D10.1002%2Fanie.201708485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span> <span> </span><span class="NLM_article-title">G-quadruplex hinders translocation of BLM helicase on DNA: a real-time fluorescence spectroscopic unwinding study and comparison with duplex substrates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">10521</span>– <span class="NLM_lpage">10527</span>, <span class="refDoi"> DOI: 10.1021/ja1038165</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1038165" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10521-10527&author=J.-q.+Liuauthor=C.-y.+Chenauthor=Y.+Xueauthor=Y.-h.+Haoauthor=Z.+Tan&title=G-quadruplex+hinders+translocation+of+BLM+helicase+on+DNA%3A+a+real-time+fluorescence+spectroscopic+unwinding+study+and+comparison+with+duplex+substrates&doi=10.1021%2Fja1038165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja1038165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1038165%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.-q.%26aulast%3DChen%26aufirst%3DC.-y.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DY.-h.%26aulast%3DTan%26aufirst%3DZ.%26atitle%3DG-quadruplex%2520hinders%2520translocation%2520of%2520BLM%2520helicase%2520on%2520DNA%253A%2520a%2520real-time%2520fluorescence%2520spectroscopic%2520unwinding%2520study%2520and%2520comparison%2520with%2520duplex%2520substrates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10521%26epage%3D10527%26doi%3D10.1021%2Fja1038165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchán, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svozil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of α/γ Conformers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">3817</span>– <span class="NLM_lpage">3829</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.097782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1529%2Fbiophysj.106.097782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17351000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2007&pages=3817-3829&author=A.+P%C3%A9rezauthor=I.+March%C3%A1nauthor=D.+Svozilauthor=J.+Sponerauthor=T.+E.+Cheathamauthor=C.+A.+Laughtonauthor=M.+Orozco&title=Refinement+of+the+AMBER+Force+Field+for+Nucleic+Acids%3A+Improving+the+Description+of+%CE%B1%2F%CE%B3+Conformers&doi=10.1529%2Fbiophysj.106.097782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of the AMBER force field for nucleic acids: improving the description of α/γ conformers</span></div><div class="casAuthors">Perez, Alberto; Marchan, Ivan; Svozil, Daniel; Sponer, Jiri; Cheatham, Thomas E., III; Laughton, Charles A.; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3817-3829</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">We present here the parmbsc0 force field, a refinement of the AMBER parm99 force field, where emphasis has been made on the correct representation of the α/γ concerted rotation in nucleic acids (NAs).  The modified force field corrects overpopulations of the α/γ = (g+,t) backbone that were seen in long (more than 10 ns) simulations with previous AMBER parameter sets (parm94-99).  The force field has been derived by fitting to high-level quantum mech. data and verified by comparison with very high-level quantum mech. calcns. and by a very extensive comparison between simulations and exptl. data.  The set of validation simulations includes two of the longest trajectories published to date for the DNA duplex (200 ns each) and the largest variety of NA structures studied to date (15 different NA families and 97 individual structures).  The total simulation time used to validate the force field includes near 1 μs of state-of-the-art mol. dynamics simulations in aq. soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriZsH_9T_yB7Vg90H21EOLACvtfcHk0ljQyid0m1xMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D&md5=e0f8d87e5fe7710e59cc41f5e8c25f7b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.097782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.097782%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DI.%26aulast%3DSvozil%26aufirst%3DD.%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DRefinement%2520of%2520the%2520AMBER%2520Force%2520Field%2520for%2520Nucleic%2520Acids%253A%2520Improving%2520the%2520Description%2520of%2520%25CE%25B1%252F%25CE%25B3%2520Conformers%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D3817%26epage%3D3829%26doi%3D10.1529%2Fbiophysj.106.097782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span> <span> </span><span class="NLM_article-title">Tracking the dynamic folding and unfolding of RNA G-Quadruplexes in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">4792</span>– <span class="NLM_lpage">4796</span>, <span class="refDoi"> DOI: 10.1002/ange.201801999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fange.201801999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2018&pages=4792-4796&author=X.-C.+Chenauthor=S.-B.+Chenauthor=J.+Daiauthor=J.-H.+Yuanauthor=T.-M.+Ouauthor=Z.-S.+Huangauthor=J.-H.+Tan&title=Tracking+the+dynamic+folding+and+unfolding+of+RNA+G-Quadruplexes+in+live+cells&doi=10.1002%2Fange.201801999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fange.201801999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201801999%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.-C.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.-H.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DHuang%26aufirst%3DZ.-S.%26aulast%3DTan%26aufirst%3DJ.-H.%26atitle%3DTracking%2520the%2520dynamic%2520folding%2520and%2520unfolding%2520of%2520RNA%2520G-Quadruplexes%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D130%26spage%3D4792%26epage%3D4796%26doi%3D10.1002%2Fange.201801999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Cheatham, T. E.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Crowley, M.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Zhang, W.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Hayik, S.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Wong, K. F.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Vanicek, J.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Steinbrecher, T.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Yang, L.</span>; <span class="NLM_string-name">Tan, C.</span>; <span class="NLM_string-name">Mongan, J.</span>; <span class="NLM_string-name">Hornak, V.</span>; <span class="NLM_string-name">Cui, G.</span>; <span class="NLM_string-name">Mathews, D. H.</span>; <span class="NLM_string-name">Seetin, M. G.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 10</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>, <span class="NLM_year">2008</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=M.+Crowley&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Wang&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=L.+Yang&author=C.+Tan&author=J.+Mongan&author=V.+Hornak&author=G.+Cui&author=D.+H.+Mathews&author=M.+G.+Seetin&author=C.+Sagui&author=V.+Babin&author=P.+A.+Kollman&title=AMBER+10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252010%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the Molecular Mechanics Poisson–Boltzmann Surface Area Free Energy Method Using a Congeneric Series of Ligands to p38 MAP Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">7796</span>– <span class="NLM_lpage">7807</span>, <span class="refDoi"> DOI: 10.1021/jm050306m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050306m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFynurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7796-7807&author=D.+A.+Pearlman&title=Evaluating+the+Molecular+Mechanics+Poisson%E2%80%93Boltzmann+Surface+Area+Free+Energy+Method+Using+a+Congeneric+Series+of+Ligands+to+p38+MAP+Kinase&doi=10.1021%2Fjm050306m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the molecular mechanics Poisson-Boltzmann surface area free energy method using a congeneric series of ligands to p38 MAP kinase</span></div><div class="casAuthors">Pearlman, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7796-7807</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently described mol. mechanics Poisson-Boltzmann surface area (MM-PBSA) method for calcg. free energies is applied to a congeneric series of 16 ligands to p38 MAP kinase whose binding consts. span approx. 2 orders of magnitude.  These compds. have previously been used to test and compare other free energy calcn. methods, including thermodn. integration (TI), OWFEG, ChemScore, PLPScore, and Dock Energy Score.  We find that the MM-PBSA performs relatively poorly for this set of ligands, yielding results much inferior to those from TI or OWFEG, inferior to Dock Energy Score, and not appreciably better than ChemScore or PLPScore but at an appreciably larger computational cost than any of these other methods.  This suggests that one should be selective in applying the MM-PBSA method and that for systems that are amenable to other free energy approaches, these other approaches may be preferred.  We also examine the single simulation approxn. for MM-PBSA, whereby the required ligand and protein trajectories are extd. from a single MD simulation rather than two sep. MD runs.  This assumption, sometimes used to speed the MM-PBSA calcn., is found to yield significantly inferior results with only a moderate net percentage redn. in total simulation time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOT67YZ9ZNj7Vg90H21EOLACvtfcHk0liDVH_TT1po5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFynurfL&md5=4a5169f9f3ead9723e080bc16ca43272</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm050306m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050306m%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26atitle%3DEvaluating%2520the%2520Molecular%2520Mechanics%2520Poisson%25E2%2580%2593Boltzmann%2520Surface%2520Area%2520Free%2520Energy%2520Method%2520Using%2520a%2520Congeneric%2520Series%2520of%2520Ligands%2520to%2520p38%2520MAP%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D7796%26epage%3D7807%26doi%3D10.1021%2Fjm050306m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Saikat Chakraborty, Kartik Dutta, Pooja Gupta, Anubrata Das, Amit Das, Sunil Kumar Ghosh, <span class="NLM_string-name hlFld-ContribAuthor">Birija Sankar Patro</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1524-1544. <a href="https://doi.org/10.1021/acs.jmedchem.0c01692" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01692</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01692%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BRECQL5%252BFunctions%25252C%252Bby%252Ba%252BSmall%252BMolecule%25252C%252BSelectively%252BKills%252BBreast%252BCancer%252Bin%252BVitro%252Band%252Bin%252BVivo%26aulast%3DChakraborty%26aufirst%3DSaikat%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28092020%26date%3D02022021%26volume%3D64%26issue%3D3%26spage%3D1524%26epage%3D1544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chen-Xi Wang, Zi-Lin Zhang, Qi-Kun Yin, Jia-Li Tu, Jia-En Wang, Yao-Hao Xu, Yong Rao, Tian-Miao Ou, Shi-Liang Huang, Ding Li, Hong-Gen Wang, Qing-Jiang Li, Jia-Heng Tan, Shuo-Bin Chen, <span class="NLM_string-name hlFld-ContribAuthor">Zhi-Shu Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Evaluation of New Quinazolinone Derivatives that Inhibit Bloom Syndrome Protein (BLM) Helicase, Trigger DNA Damage at the Telomere Region, and Synergize with PARP Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9752-9772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00917" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00917</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00917%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BEvaluation%252Bof%252BNew%252BQuinazolinone%252BDerivatives%252Bthat%252BInhibit%252BBloom%252BSyndrome%252BProtein%252B%252528BLM%252529%252BHelicase%25252C%252BTrigger%252BDNA%252BDamage%252Bat%252Bthe%252BTelomere%252BRegion%25252C%252Band%252BSynergize%252Bwith%252BPARP%252BInhibitors%26aulast%3DWang%26aufirst%3DChen-Xi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30052020%26date%3D07082020%26date%3D22072020%26volume%3D63%26issue%3D17%26spage%3D9752%26epage%3D9772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yin  Lu</span>, <span class="hlFld-ContribAuthor ">Qin  Ma</span>, <span class="hlFld-ContribAuthor ">Xiaolu  Xu</span>, <span class="hlFld-ContribAuthor ">Zhefu  Yu</span>, <span class="hlFld-ContribAuthor ">Tianjiao  Guo</span>, <span class="hlFld-ContribAuthor ">Yangkai  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxicity and genotoxicity evaluation of polystyrene microplastics on Vicia faba roots. </span><span class="cited-content_cbyCitation_journal-name">Environmental Pollution</span><span> <strong>2021,</strong> <em>288 </em>, 117821. <a href="https://doi.org/10.1016/j.envpol.2021.117821" title="DOI URL">https://doi.org/10.1016/j.envpol.2021.117821</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envpol.2021.117821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envpol.2021.117821%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520Pollution%26atitle%3DCytotoxicity%252Band%252Bgenotoxicity%252Bevaluation%252Bof%252Bpolystyrene%252Bmicroplastics%252Bon%252BVicia%252Bfaba%252Broots%26aulast%3DLu%26aufirst%3DYin%26date%3D2021%26volume%3D288%26spage%3D117821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mouna  Ababou</span>. </span><span class="cited-content_cbyCitation_article-title">Bloom syndrome and the underlying causes of genetic instability. </span><span class="cited-content_cbyCitation_journal-name">Molecular Genetics and Metabolism</span><span> <strong>2021,</strong> <em>133 </em>
                                    (1)
                                     , 35-48. <a href="https://doi.org/10.1016/j.ymgme.2021.03.003" title="DOI URL">https://doi.org/10.1016/j.ymgme.2021.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymgme.2021.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymgme.2021.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Genetics%2520and%2520Metabolism%26atitle%3DBloom%252Bsyndrome%252Band%252Bthe%252Bunderlying%252Bcauses%252Bof%252Bgenetic%252Binstability%26aulast%3DAbabou%26aufirst%3DMouna%26date%3D2021%26volume%3D133%26issue%3D1%26spage%3D35%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Larissa Costa  de Almeida</span>, <span class="hlFld-ContribAuthor ">Felipe Antunes  Calil</span>, <span class="hlFld-ContribAuthor ">João Agostinho  Machado-Neto</span>, <span class="hlFld-ContribAuthor ">Leticia Veras  Costa-Lotufo</span>. </span><span class="cited-content_cbyCitation_article-title">DNA damaging agents and DNA repair: From carcinogenesis to cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancer Genetics</span><span> <strong>2021,</strong> <em>252-253 </em>, 6-24. <a href="https://doi.org/10.1016/j.cancergen.2020.12.002" title="DOI URL">https://doi.org/10.1016/j.cancergen.2020.12.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cancergen.2020.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cancergen.2020.12.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Genetics%26atitle%3DDNA%252Bdamaging%252Bagents%252Band%252BDNA%252Brepair%25253A%252BFrom%252Bcarcinogenesis%252Bto%252Bcancer%252Btherapy%26aulast%3Dde%2BAlmeida%26aufirst%3DLarissa%2BCosta%26date%3D2021%26volume%3D252-253%26spage%3D6%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ekjot  Kaur</span>, <span class="hlFld-ContribAuthor ">Ritu  Agrawal</span>, <span class="hlFld-ContribAuthor ">Sagar  Sengupta</span>. </span><span class="cited-content_cbyCitation_article-title">Functions of BLM Helicase in Cells: Is It Acting Like a Double-Edged Sword?. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fgene.2021.634789" title="DOI URL">https://doi.org/10.3389/fgene.2021.634789</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2021.634789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2021.634789%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DFunctions%252Bof%252BBLM%252BHelicase%252Bin%252BCells%25253A%252BIs%252BIt%252BActing%252BLike%252Ba%252BDouble-Edged%252BSword%25253F%26aulast%3DKaur%26aufirst%3DEkjot%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangrong  Chen</span>, <span class="hlFld-ContribAuthor ">Yusuf I  Ali</span>, <span class="hlFld-ContribAuthor ">Charlotte EL  Fisher</span>, <span class="hlFld-ContribAuthor ">Raquel  Arribas-Bosacoma</span>, <span class="hlFld-ContribAuthor ">Mohan B  Rajasekaran</span>, <span class="hlFld-ContribAuthor ">Gareth  Williams</span>, <span class="hlFld-ContribAuthor ">Sarah  Walker</span>, <span class="hlFld-ContribAuthor ">Jessica R  Booth</span>, <span class="hlFld-ContribAuthor ">Jessica JR  Hudson</span>, <span class="hlFld-ContribAuthor ">S Mark  Roe</span>, <span class="hlFld-ContribAuthor ">Laurence H  Pearl</span>, <span class="hlFld-ContribAuthor ">Simon E  Ward</span>, <span class="hlFld-ContribAuthor ">Frances MG  Pearl</span>, <span class="hlFld-ContribAuthor ">Antony W  Oliver</span>. </span><span class="cited-content_cbyCitation_article-title">Uncovering an allosteric mode of action for a selective inhibitor of human Bloom syndrome protein. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2021,</strong> <em>10 </em><a href="https://doi.org/10.7554/eLife.65339" title="DOI URL">https://doi.org/10.7554/eLife.65339</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.65339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.65339%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DUncovering%252Ban%252Ballosteric%252Bmode%252Bof%252Baction%252Bfor%252Ba%252Bselective%252Binhibitor%252Bof%252Bhuman%252BBloom%252Bsyndrome%252Bprotein%26aulast%3DChen%26aufirst%3DXiangrong%26date%3D2021%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Faisal A.  Alzahrani</span>, <span class="hlFld-ContribAuthor ">Firoz  Ahmed</span>, <span class="hlFld-ContribAuthor ">Monika  Sharma</span>, <span class="hlFld-ContribAuthor ">Mohd  Rehan</span>, <span class="hlFld-ContribAuthor ">Maryam  Mahfuz</span>, <span class="hlFld-ContribAuthor ">Mohammed N.  Baeshen</span>, <span class="hlFld-ContribAuthor ">Yousef  Hawsawi</span>, <span class="hlFld-ContribAuthor ">Ahmed  Almatrafi</span>, <span class="hlFld-ContribAuthor ">Suliman Abdallah  Alsagaby</span>, <span class="hlFld-ContribAuthor ">Mohammad Azhar  Kamal</span>, <span class="hlFld-ContribAuthor ">Mohiuddin Khan  Warsi</span>, <span class="hlFld-ContribAuthor ">Hani  Choudhry</span>, <span class="hlFld-ContribAuthor ">Mohammad Sarwar  Jamal</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the pathogenic SNPs in BLM helicase and their biological consequences by computational approach. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-69033-8" title="DOI URL">https://doi.org/10.1038/s41598-020-69033-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-69033-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-69033-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DInvestigating%252Bthe%252Bpathogenic%252BSNPs%252Bin%252BBLM%252Bhelicase%252Band%252Btheir%252Bbiological%252Bconsequences%252Bby%252Bcomputational%252Bapproach%26aulast%3DAlzahrani%26aufirst%3DFaisal%2BA.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xi  Chen</span>, <span class="hlFld-ContribAuthor ">Guo  Wu</span>, <span class="hlFld-ContribAuthor ">Qiong  Ma</span>, <span class="hlFld-ContribAuthor ">Jin-long  Lai</span>, <span class="hlFld-ContribAuthor ">Xue-gang  Luo</span>, <span class="hlFld-ContribAuthor ">Xiao-hui  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxic and genotoxic evaluation and the toxicological mechanism of uranium in Vicia faba root. </span><span class="cited-content_cbyCitation_journal-name">Environmental and Experimental Botany</span><span> <strong>2020,</strong> <em>179 </em>, 104227. <a href="https://doi.org/10.1016/j.envexpbot.2020.104227" title="DOI URL">https://doi.org/10.1016/j.envexpbot.2020.104227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envexpbot.2020.104227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envexpbot.2020.104227%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520and%2520Experimental%2520Botany%26atitle%3DCytotoxic%252Band%252Bgenotoxic%252Bevaluation%252Band%252Bthe%252Btoxicological%252Bmechanism%252Bof%252Buranium%252Bin%252BVicia%252Bfaba%252Broot%26aulast%3DChen%26aufirst%3DXi%26date%3D2020%26volume%3D179%26spage%3D104227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, <span class="hlFld-ContribAuthor ">Joseph A.  Newman</span>, <span class="hlFld-ContribAuthor ">Opher  Gileadi</span>. </span><span class="cited-content_cbyCitation_article-title">RecQ helicases in DNA repair and cancer targets. </span><span class="cited-content_cbyCitation_journal-name">Essays in Biochemistry</span><span> <strong>2020,</strong> <em>64 </em>
                                    (5)
                                     , 819-830. <a href="https://doi.org/10.1042/EBC20200012" title="DOI URL">https://doi.org/10.1042/EBC20200012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/EBC20200012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FEBC20200012%26sid%3Dliteratum%253Aachs%26jtitle%3DEssays%2520in%2520Biochemistry%26atitle%3DRecQ%252Bhelicases%252Bin%252BDNA%252Brepair%252Band%252Bcancer%252Btargets%26aulast%3DWu%26aufirst%3DQian%26date%3D2020%26date%3D2020%26volume%3D64%26issue%3D5%26spage%3D819%26epage%3D830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditya  Mojumdar</span>. </span><span class="cited-content_cbyCitation_article-title">Mutations in conserved functional domains of human RecQ helicases are associated with diseases and cancer: A review. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Chemistry</span><span> <strong>2020,</strong> <em>265 </em>, 106433. <a href="https://doi.org/10.1016/j.bpc.2020.106433" title="DOI URL">https://doi.org/10.1016/j.bpc.2020.106433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpc.2020.106433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpc.2020.106433%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Chemistry%26atitle%3DMutations%252Bin%252Bconserved%252Bfunctional%252Bdomains%252Bof%252Bhuman%252BRecQ%252Bhelicases%252Bare%252Bassociated%252Bwith%252Bdiseases%252Band%252Bcancer%25253A%252BA%252Breview%26aulast%3DMojumdar%26aufirst%3DAditya%26date%3D2020%26volume%3D265%26spage%3D106433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chetan K.  Jain</span>, <span class="hlFld-ContribAuthor ">Swagata  Mukhopadhyay</span>, <span class="hlFld-ContribAuthor ">Agneyo  Ganguly</span>. </span><span class="cited-content_cbyCitation_article-title">RecQ Family Helicases in Replication Fork Remodeling and Repair: Opening New Avenues towards the Identification of Potential Targets for Cancer Chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Agents in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (11)
                                     , 1311-1326. <a href="https://doi.org/10.2174/1871520620666200518082433" title="DOI URL">https://doi.org/10.2174/1871520620666200518082433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871520620666200518082433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871520620666200518082433%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Agents%2520in%2520Medicinal%2520Chemistry%26atitle%3DRecQ%252BFamily%252BHelicases%252Bin%252BReplication%252BFork%252BRemodeling%252Band%252BRepair%25253A%252BOpening%252BNew%252BAvenues%252Btowards%252Bthe%252BIdentification%252Bof%252BPotential%252BTargets%252Bfor%252BCancer%252BChemotherapy%26aulast%3DJain%26aufirst%3DChetan%2BK.%26date%3D2020%26volume%3D20%26issue%3D11%26spage%3D1311%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and construction of isaindigotone derivatives library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Route of Isaindigotone Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) pyrrolidin-2-one, POCl<sub>3</sub>, reflux, 8 h; (b) R<sub>1</sub>NH<sub>2</sub>, 100 °C, 12 h; (c) aldehydes, DMF, TMSCl, 100 °C, 12–48 h.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification of BLM inhibitors. (A) Diagram of BLM inhibitors identification and structure of hit compound <b>29</b>. (B) Inhibitory effects on BLM unwinding at the concentration of 1.0 μM for 12 candidate compounds. (C) Expression level of γ-H2AX with the treatment of 12 candidate compounds in HCT116 cells for 24 h, the concentration of each compound used in the treatment was half of IC<sub>50</sub> value in MTT assay, β-actin was used as a control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Disrupting effect of <b>29</b> on the interaction of BLM and DNA. (A) Filter-binding dots of BLM with DNA upon addition of <b>29</b>. (B) Dose-dependent disrupting of BLM/DNA binding upon addition of <b>29</b> and <b>ML216</b> using ELISA assay, the IC<sub>50</sub> value was fitted by Hill model. (C) Disrupting effect of <b>29</b> on cellular BLM binding to DNA determined by pull-down assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. HR regulation mediated by <b>29</b> via disrupting the interaction of BLM and DNA in cells. (A) Scheme of HRR and DSB model. (B) Relative HRR level in HCT116 cells after being treated with <b>29</b> and <b>ML216</b>. (C) Relative HRR level in HCT116 and siBLM HCT116 cells after being treated with <b>29</b>. The HRR level was quantified by the percentage of GFP expressed cells and normalized to untreated samples. (D) Recruitment of BLM to the indicated DSB sites on HRR (−) reporters in <b>29</b>-treated and <b>ML216</b>-treated HCT116 cells. (E) Recruitment of BLM to the indicated DSB sites on chromosomes in <b>29</b>-treated HCT116 cells. BLM ChIP efficiency were determined by qPCR using specific primers and normalized to untreated samples. The data in experiment were made in triplicate and expressed as the mean ± SD: (*) <i>P</i> < 0.05, (**) <i>P</i> < 0.01, and (***) <i>P</i> < 0.001 compared with untreated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Accumulation of RAD51 at the DSB sites mediated by <b>29</b>. (A) <b>29</b> reversed the decrease of RAD51-ssDNA filaments induced by BLM in vitro. (B) Recruitment of RAD51 to the indicated DSB sites on HRR (−) reporters in <b>29</b>-treated and siBLM HCT116 cells, same as <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D. (C) Recruitment of RAD51 to the indicated DSB sites on chromosomes in <b>29</b>-treated HCT116 cells, same as <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E. (D) Distribution and colocalization of RAD51 and γ-H2AX in HCT116 cells treated with <b>29</b> for 24 h using the immunofluorescence assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DNA damage response and apoptosis in HCT116 cells induced by <b>29</b>. (A) Expression level of major signaling proteins in response to DNA damage in HCT116 cells treated with <b>29</b> for 24 h, β-actin was used as a control. (B) Apoptosis evaluation of HCT116 cells treated with <b>29</b> for 48 h. (C) Expression level of apoptosis pathway-related proteins in HCT116 cells treated with <b>29</b> for 48 h, β-actin was used as a control. (D) Action pathway diagram of <b>29</b> in HCT116 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Proliferation arrest and synthetic interaction induced by <b>29</b>. (A,B) Proliferative ability in HCT116 and siBLM HCT116 cells treated with <b>29</b> for 48 h. (C,D) Proliferative ability in HCT116 and siBLM HCT116 cells treated with <b>29</b> in a long-term proliferation assay. (E,F) Antiproliferative effect of <b>29</b> in combined therapy with a RAD51 inhibitor (<b>RI-1</b>) and a DNA damage agent (<b>Cisplatin</b>) in HCT116 cells. The data were derived from three experiments and expressed as the mean ± SEM: (*) <i>P</i> < 0.05, (**) <i>P</i> < 0.01, and (***) <i>P</i> < 0.001 compared with untreated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/medium/jm-2019-000838_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding mode of <b>29</b> to BLM. (A) ITC curve and fitted affinity of <b>29</b> to wild-type BLM, the top panel showed the heat change upon ligand titration, the bottom panel showed the integrated data and ITC isotherm (solid line) fitted by a single-site binding model. (B) Predicted binding mode of <b>29</b> with BLM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>), the diagram showed the interaction between ligand <b>29</b> (cyan) and predicted residues (green). (C) Binding affinity of <b>29</b> to wild-type and mutant BLM using ITC assay. “N.D.” in the table meant the <i>K</i><sub>D</sub> value was not obtained by fitting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.9b00083/20190321/images/large/jm-2019-000838_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00083&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i73">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52135" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage response in human biology and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi"> DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=1071-1078&author=S.+P.+Jacksonauthor=J.+Bartek&title=The+DNA-damage+response+in+human+biology+and+disease&doi=10.1038%2Fnature08467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0lgj5UqIOQLP5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520response%2520in%2520human%2520biology%2520and%2520disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, S. A.</span></span> <span> </span><span class="NLM_article-title">Genomic approaches to DNA repair and mutagenesis</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2015.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2015.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26411877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWls7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=146-155&author=J.+J.+Wyrickauthor=S.+A.+Roberts&title=Genomic+approaches+to+DNA+repair+and+mutagenesis&doi=10.1016%2Fj.dnarep.2015.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic approaches to DNA repair and mutagenesis</span></div><div class="casAuthors">Wyrick, John J.; Roberts, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">146-155</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  DNA damage is a const. threat to cells, causing cytotoxicity as well as inducing genetic alterations.  The steady-state abundance of DNA lesions in a cell is minimized by a variety of DNA repair mechanisms, including DNA strand break repair, mismatch repair, nucleotide excision repair, base excision repair, and ribonucleotide excision repair.  The efficiencies and mechanisms by which these pathways remove damage from chromosomes have been primarily characterized by investigating the processing of lesions at defined genomic loci, among bulk genomic DNA, on episomal DNA constructs, or using in vitro substrates.  However, the structure of a chromosome is heterogeneous, consisting of heavily protein-bound heterochromatic regions, open regulatory regions, actively transcribed genes, and even areas of transient single stranded DNA.  Consequently, DNA repair pathways function in a much more diverse set of chromosomal contexts than can be readily assessed using previous methods.  Recent efforts to develop whole genome maps of DNA damage, repair processes, and even mutations promise to greatly expand our understanding of DNA repair and mutagenesis.  Here we review the current efforts to utilize whole genome maps of DNA damage and mutation to understand how different chromosomal contexts affect DNA excision repair pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJpUXQJ0AGcrVg90H21EOLACvtfcHk0lhDOxT7dA4--w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWls7jL&md5=0bf15f45ed5052821415f4323618425b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DWyrick%26aufirst%3DJ.%2BJ.%26aulast%3DRoberts%26aufirst%3DS.%2BA.%26atitle%3DGenomic%2520approaches%2520to%2520DNA%2520repair%2520and%2520mutagenesis%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D36%26spage%3D146%26epage%3D155%26doi%3D10.1016%2Fj.dnarep.2015.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span> <span> </span><span class="NLM_article-title">Single-strand break repair and genetic disease</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1038/nrg2380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrg2380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18626472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVOku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=619-631&author=K.+W.+Caldecott&title=Single-strand+break+repair+and+genetic+disease&doi=10.1038%2Fnrg2380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Single-strand break repair and genetic disease</span></div><div class="casAuthors">Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">619-631</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Hereditary defects in the repair of DNA damage are implicated in a variety of diseases, many of which are typified by neurol. dysfunction and/or increased genetic instability and cancer.  Of the different types of DNA damage that arise in cells, single-strand breaks (SSBs) are the most common, arising at a frequency of tens of thousands per cell per day from direct attack by intracellular metabolites and from spontaneous DNA decay.  Here, the mol. mechanisms and organization of the DNA-repair pathways that remove SSBs are reviewed and the connection between defects in these pathways and hereditary neurodegenerative disease are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrLPZKetl1rVg90H21EOLACvtfcHk0lhDOxT7dA4--w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVOku7Y%253D&md5=5677671cc74547583cfb595952106b48</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrg2380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2380%26sid%3Dliteratum%253Aachs%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DSingle-strand%2520break%2520repair%2520and%2520genetic%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2008%26volume%3D9%26spage%3D619%26epage%3D631%26doi%3D10.1038%2Fnrg2380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">DNA double-strand breaks: signaling, repair and the cancer connection</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1038/85798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2F85798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11242102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2001&pages=247-254&author=K.+K.+Khannaauthor=S.+P.+Jackson&title=DNA+double-strand+breaks%3A+signaling%2C+repair+and+the+cancer+connection&doi=10.1038%2F85798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">DNA double-strand breaks: signaling, repair and the cancer connection</span></div><div class="casAuthors">Khanna, Kum Kum; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">247-254</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review, with 76 refs.,.  To ensure the high-fidelity transmission of genetic information, cells have evolved mechanisms to monitor genome integrity.  Cells respond to DNA damage by activating a complex DNA-damage-response pathway that includes cell-cycle arrest, the transcriptional and post-transcriptional activation of a subset of genes including those assocd. with DNA repair, and under some circumstances, the triggering of programmed cell death.  An inability to respond properly to, or to repair, DNA damage leads to genetic instability, which in turn may enhance the rate of cancer development.  Indeed, it is becoming increasingly clear that deficiencies in DNA-damage signaling and repair pathways are fundamental to the etiol. of most, if not all, human cancers.  Here, recent progress in the understanding of how cells detect and signal the presence and repair of one particularly important form of DNA damage induced by ionizing radiation, the DNA double-strand break (DSB), is described.  Moreover, how tumor suppressor proteins such as p53, ATM, Brca1 and Brca2 have been linked to such pathways and how accumulating evidence is connecting deficiencies in cellular responses to DNA DSBs with tumorigenesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2_Aw1LLx3LVg90H21EOLACvtfcHk0lhDOxT7dA4--w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslKqtLg%253D&md5=e5ccb5ac64c8dbf39375d33a056f34ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F85798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F85798%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DK.%2BK.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DDNA%2520double-strand%2520breaks%253A%2520signaling%252C%2520repair%2520and%2520the%2520cancer%2520connection%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D27%26spage%3D247%26epage%3D254%26doi%3D10.1038%2F85798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krejci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmannova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spirek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination and its regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5795</span>– <span class="NLM_lpage">5818</span>, <span class="refDoi"> DOI: 10.1093/nar/gks270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgks270" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=5795-5818&author=L.+Krejciauthor=V.+Altmannovaauthor=M.+Spirekauthor=X.+Zhao&title=Homologous+recombination+and+its+regulation&doi=10.1093%2Fnar%2Fgks270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks270%26sid%3Dliteratum%253Aachs%26aulast%3DKrejci%26aufirst%3DL.%26aulast%3DAltmannova%26aufirst%3DV.%26aulast%3DSpirek%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DHomologous%2520recombination%2520and%2520its%2520regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D5795%26epage%3D5818%26doi%3D10.1093%2Fnar%2Fgks270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0lgQyGsCnMJsBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">DNA repair dysregulation from cancer driver to therapeutic target</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nrc3399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc3399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=23175119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=801-817&author=N.+J.+Curtin&title=DNA+repair+dysregulation+from+cancer+driver+to+therapeutic+target&doi=10.1038%2Fnrc3399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair dysregulation from cancer driver to therapeutic target</span></div><div class="casAuthors">Curtin, Nicola J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">801-817</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dysregulation of DNA damage repair and signalling to cell cycle checkpoints, known as the DNA damage response (DDR), is assocd. with a predisposition to cancer and affects responses to DNA-damaging anticancer therapy.  Dysfunction of one DNA repair pathway may be compensated for by the function of another compensatory DDR pathway, which may be increased and contribute to resistance to DNA-damaging chemotherapy and radiotherapy.  Therefore, DDR pathways make an ideal target for therapeutic intervention; first, to prevent or reverse therapy resistance; and second, using a synthetic lethal approach to specifically kill cancer cells that are dependent on a compensatory DNA repair pathway for survival in the context of cancer-assocd. oxidative and replicative stress.  These hypotheses are currently being tested in the lab. and are being translated into clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNDBV6krLGBbVg90H21EOLACvtfcHk0lgQyGsCnMJsBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N&md5=1bcb90d4742e46696c8a30a4fce2400c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc3399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3399%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DDNA%2520repair%2520dysregulation%2520from%2520cancer%2520driver%2520to%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D801%26epage%3D817%26doi%3D10.1038%2Fnrc3399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garber, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domchek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audeh, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzel, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedlander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span> <span> </span><span class="NLM_article-title">Phase II trial of the oral PARP inhibitor olaparib in BRCA-deficient advanced breast cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">CRA501</span>, <span class="refDoi"> DOI: 10.1200/jco.2009.27.18_suppl.cra501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1200%2Fjco.2009.27.18_suppl.cra501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=CRA501&author=A.+Tuttauthor=M.+Robsonauthor=J.+E.+Garberauthor=S.+Domchekauthor=M.+W.+Audehauthor=J.+N.+Weitzelauthor=M.+Friedlanderauthor=J.+Carmichael&title=Phase+II+trial+of+the+oral+PARP+inhibitor+olaparib+in+BRCA-deficient+advanced+breast+cancer&doi=10.1200%2Fjco.2009.27.18_suppl.cra501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1200%2Fjco.2009.27.18_suppl.cra501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2009.27.18_suppl.cra501%26sid%3Dliteratum%253Aachs%26aulast%3DTutt%26aufirst%3DA.%26aulast%3DRobson%26aufirst%3DM.%26aulast%3DGarber%26aufirst%3DJ.%2BE.%26aulast%3DDomchek%26aufirst%3DS.%26aulast%3DAudeh%26aufirst%3DM.%2BW.%26aulast%3DWeitzel%26aufirst%3DJ.%2BN.%26aulast%3DFriedlander%26aufirst%3DM.%26aulast%3DCarmichael%26aufirst%3DJ.%26atitle%3DPhase%2520II%2520trial%2520of%2520the%2520oral%2520PARP%2520inhibitor%2520olaparib%2520in%2520BRCA-deficient%2520advanced%2520breast%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2009%26volume%3D27%26spage%3DCRA501%26doi%3D10.1200%2Fjco.2009.27.18_suppl.cra501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ristic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanaar, R.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination-mediated double-strand break repair</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2004.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2004.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=15279767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVeqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=827-833&author=C.+Wymanauthor=D.+Risticauthor=R.+Kanaar&title=Homologous+recombination-mediated+double-strand+break+repair&doi=10.1016%2Fj.dnarep.2004.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination-mediated double-strand break repair</span></div><div class="casAuthors">Wyman, Claire; Ristic, Dejan; Kanaar, Roland</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">827-833</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Exchange of DNA strands between homologous DNA mols. via recombination ensures accurate genome duplication and preservation of genome integrity.  Biochem. studies have provided insights into the mol. mechanisms by which homologous recombination proteins perform these essential tasks.  More recent cell biol. expts. are addressing the behavior of homologous recombination proteins in cells.  The challenge ahead is to uncover the relationship between the individual biochem. activities of homologous recombination proteins and their coordinated action in the context of the living cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRDQngo6NrWrVg90H21EOLACvtfcHk0lgJYM1Mvss2AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVeqsb8%253D&md5=ec00b3ee185ba32a9cfdaab48ad65187</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2004.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2004.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DWyman%26aufirst%3DC.%26aulast%3DRistic%26aufirst%3DD.%26aulast%3DKanaar%26aufirst%3DR.%26atitle%3DHomologous%2520recombination-mediated%2520double-strand%2520break%2520repair%26jtitle%3DDNA%2520Repair%26date%3D2004%26volume%3D3%26spage%3D827%26epage%3D833%26doi%3D10.1016%2Fj.dnarep.2004.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyer, W.-D.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination in DNA repair and DNA damage tolerance</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1038/cr.2008.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fcr.2008.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18166982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisVSjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=99-113&author=X.+Liauthor=W.-D.+Heyer&title=Homologous+recombination+in+DNA+repair+and+DNA+damage+tolerance&doi=10.1038%2Fcr.2008.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in DNA repair and DNA damage tolerance</span></div><div class="casAuthors">Li, Xuan; Heyer, Wolf-Dietrich</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-113</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Homologous recombination (HR) comprises a series of interrelated pathways that function in the repair of DNA double-stranded breaks (DSBs) and interstrand crosslinks (ICLs).  In addn., recombination provides crit. support for DNA replication in the recovery of stalled or broken replication forks, contributing to tolerance of DNA damage.  A central core of proteins, most critically the RecA homolog Rad51, catalyzes the key reactions that typify HR: homol. search and DNA strand invasion.  The diverse functions of recombination are reflected in the need for context-specific factors that perform supplemental functions in conjunction with the core proteins.  The inability to properly repair complex DNA damage and resolve DNA replication stress leads to genomic instability and contributes to cancer etiol.  Mutations in the BRCA2 recombination gene cause predisposition to breast and ovarian cancer as well as Fanconi anemia, a cancer predisposition syndrome characterized by a defect in the repair of DNA interstrand crosslinks.  The cellular functions of recombination are also germane to DNA-based treatment modalities of cancer, which target replicating cells by the direct or indirect induction of DNA lesions that are substrates for recombination pathways.  This review focuses on mechanistic aspects of HR relating to DSB and ICL repair as well as replication fork support.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprinxRSKXk47Vg90H21EOLACvtfcHk0lgJYM1Mvss2AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisVSjsw%253D%253D&md5=103977d80ae5366ccb2b8d4621fd7358</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fcr.2008.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2008.1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHeyer%26aufirst%3DW.-D.%26atitle%3DHomologous%2520recombination%2520in%2520DNA%2520repair%2520and%2520DNA%2520damage%2520tolerance%26jtitle%3DCell%2520Res.%26date%3D2008%26volume%3D18%26spage%3D99%26epage%3D113%26doi%3D10.1038%2Fcr.2008.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedberg, E. C.</span></span> <span> </span><span class="NLM_article-title">DNA damage and repair</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nature01408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature01408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=12540918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFGqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=436-440&author=E.+C.+Friedberg&title=DNA+damage+and+repair&doi=10.1038%2Fnature01408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage and repair</span></div><div class="casAuthors">Friedberg, Errol C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">6921, Suppl.</span>),
    <span class="NLM_cas:pages">436-440</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aesthetic appeal of the DNA double helix initially hindered notions of DNA mutation and repair, which would necessarily interfere with its pristine state.  But it has since been recognized that DNA is subject to continuous damage and the cell has an arsenal of ways of responding to such injury.  Although mutations or deficiencies in repair can have catastrophic consequences, causing a range of human diseases, mutations are nonetheless fundamental to life and evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofETXt2LmXt7Vg90H21EOLACvtfcHk0lgJYM1Mvss2AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFGqtb8%253D&md5=92475bfb2d2c155798ef6198dabcc701</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature01408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01408%26sid%3Dliteratum%253Aachs%26aulast%3DFriedberg%26aufirst%3DE.%2BC.%26atitle%3DDNA%2520damage%2520and%2520repair%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D436%26epage%3D440%26doi%3D10.1038%2Fnature01408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishijima, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youssoufian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span> <span> </span><span class="NLM_article-title">Cancer predisposition caused by elevated mitotic recombination in Bloom mice</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1038/82548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2F82548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11101838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVWgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2000&pages=424-429&author=G.+Luoauthor=I.+M.+Santoroauthor=L.+D.+McDanielauthor=I.+Nishijimaauthor=M.+Millsauthor=H.+Youssoufianauthor=H.+Vogelauthor=R.+A.+Schultzauthor=A.+Bradley&title=Cancer+predisposition+caused+by+elevated+mitotic+recombination+in+Bloom+mice&doi=10.1038%2F82548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer predisposition caused by elevated mitotic recombination in Bloom mice</span></div><div class="casAuthors">Luo, Guangbin; Santoro, Irma M.; McDaniel, Lisa D.; Nishijima, Ichiko; Mills, Michael; Youssoufian, Hagop; Vogel, Hannes; Schultz, Roger A.; Bradley, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-429</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Bloom syndrome is a disorder assocd. with genomic instability that causes affected people to be prone to cancer.  Bloom cell lines show increased sister chromatid exchange, yet are proficient in the repair of various DNA lesions.  The underlying cause of this disease are mutations in a gene encoding a RECQ DNA helicase.  Using embryonic stem cell technol., the authors have generated viable Bloom mice that are prone to a wide variety of cancers.  Cell lines from these mice show elevations in the rates of mitotic recombination.  The authors demonstrate that the increased rate of loss of heterozygosity (LOH) resulting from mitotic recombination in vivo constitutes the underlying mechanism causing tumor susceptibility in these mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2NUHWpnQPL7Vg90H21EOLACvtfcHk0ljMqFYzTj5s3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVWgtr4%253D&md5=66b50d2784f4a289712d078ce73b4990</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F82548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F82548%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DSantoro%26aufirst%3DI.%2BM.%26aulast%3DMcDaniel%26aufirst%3DL.%2BD.%26aulast%3DNishijima%26aufirst%3DI.%26aulast%3DMills%26aufirst%3DM.%26aulast%3DYoussoufian%26aufirst%3DH.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DR.%2BA.%26aulast%3DBradley%26aufirst%3DA.%26atitle%3DCancer%2520predisposition%2520caused%2520by%2520elevated%2520mitotic%2520recombination%2520in%2520Bloom%2520mice%26jtitle%3DNat.%2520Genet.%26date%3D2000%26volume%3D26%26spage%3D424%26epage%3D429%26doi%3D10.1038%2F82548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobzhansky, T.</span></span> <span> </span><span class="NLM_article-title">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1946</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20985721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1946&pages=269-290&author=T.+Dobzhansky&title=Genetics+of+natural+populations%3B+recombination+and+variability+in+populations+of+Drosophila+pseudoobscura"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span></div><div class="casAuthors">DOBZHANSKY T</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1946</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-90</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXehIoFNdovudfST00W0BcfW6udTcc2ebbS8ksCitMn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D&md5=51f55835f5a3053a235bdea89458573c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDobzhansky%26aufirst%3DT.%26atitle%3DGenetics%2520of%2520natural%2520populations%253B%2520recombination%2520and%2520variability%2520in%2520populations%2520of%2520Drosophila%2520pseudoobscura%26jtitle%3DGenetics%26date%3D1946%26volume%3D31%26spage%3D269%26epage%3D290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartwell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szankasi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friend, S. H.</span></span> <span> </span><span class="NLM_article-title">Integrating genetic approaches into the discovery of anticancer drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">1064</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1126/science.278.5340.1064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1126%2Fscience.278.5340.1064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=9353181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADyaK2sXnt1Wju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=1997&pages=1064-1068&author=L.+H.+Hartwellauthor=P.+Szankasiauthor=C.+J.+Robertsauthor=A.+W.+Murrayauthor=S.+H.+Friend&title=Integrating+genetic+approaches+into+the+discovery+of+anticancer+drugs&doi=10.1126%2Fscience.278.5340.1064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating genetic approaches into the discovery of anticancer drugs</span></div><div class="casAuthors">Hartwell, Leland H.; Szankasi, Philippe; Roberts, Christopher J.; Murray, Andrew W.; Friend, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">5340</span>),
    <span class="NLM_cas:pages">1064-1068</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 48 refs. The discovery of anticancer drugs is now driven by the numerous mol. alterations identified in tumor cells over the past decade.  To exploit these alterations, it is necessary to understand how they define a mol. context that allows increased sensitivity to particular compds.  Traditional genetic approaches together with the new wealth of genomic information for both human and model organisms open up strategies by which drugs can be profiled for their ability to selectively kill cells in a mol. context that matches those found in tumors.  Similarly, it may be possible to identify and validate new targets for drugs that would selectively kill tumor cells with a particular mol. context.  This article outlines some of the ways that yeast genetics can be used to streamline anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbQLv4QwxtZrVg90H21EOLACvtfcHk0ljMqFYzTj5s3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt1Wju7k%253D&md5=6bd5b219e8b5f5dd28b400003a9e4edb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.278.5340.1064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.278.5340.1064%26sid%3Dliteratum%253Aachs%26aulast%3DHartwell%26aufirst%3DL.%2BH.%26aulast%3DSzankasi%26aufirst%3DP.%26aulast%3DRoberts%26aufirst%3DC.%2BJ.%26aulast%3DMurray%26aufirst%3DA.%2BW.%26aulast%3DFriend%26aufirst%3DS.%2BH.%26atitle%3DIntegrating%2520genetic%2520approaches%2520into%2520the%2520discovery%2520of%2520anticancer%2520drugs%26jtitle%3DScience%26date%3D1997%26volume%3D278%26spage%3D1064%26epage%3D1068%26doi%3D10.1126%2Fscience.278.5340.1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelivianskaia, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron McGuire, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, P. P.</span></span> <span> </span><span class="NLM_article-title">RI-1: a chemical inhibitor of RAD51 that disrupts homologous recombination in human cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">7347</span>– <span class="NLM_lpage">7357</span>, <span class="refDoi"> DOI: 10.1093/nar/gks353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgks353" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=7347-7357&author=B.+Budkeauthor=H.+L.+Loganauthor=J.+H.+Kalinauthor=A.+S.+Zelivianskaiaauthor=W.+Cameron+McGuireauthor=L.+L.+Millerauthor=J.+M.+Starkauthor=A.+P.+Kozikowskiauthor=D.+K.+Bishopauthor=P.+P.+Connell&title=RI-1%3A+a+chemical+inhibitor+of+RAD51+that+disrupts+homologous+recombination+in+human+cells&doi=10.1093%2Fnar%2Fgks353"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks353%26sid%3Dliteratum%253Aachs%26aulast%3DBudke%26aufirst%3DB.%26aulast%3DLogan%26aufirst%3DH.%2BL.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DZelivianskaia%26aufirst%3DA.%2BS.%26aulast%3DCameron%2BMcGuire%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DL.%2BL.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DBishop%26aufirst%3DD.%2BK.%26aulast%3DConnell%26aufirst%3DP.%2BP.%26atitle%3DRI-1%253A%2520a%2520chemical%2520inhibitor%2520of%2520RAD51%2520that%2520disrupts%2520homologous%2520recombination%2520in%2520human%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D7347%26epage%3D7357%26doi%3D10.1093%2Fnar%2Fgks353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1073/pnas.1006423108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.1006423108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=21220316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1525-1530&author=M.+Aggarwalauthor=J.+A.+Sommersauthor=R.+H.+Shoemakerauthor=R.+M.+Brosh&title=Inhibition+of+helicase+activity+by+a+small+molecule+impairs+Werner+syndrome+helicase+%28WRN%29+function+in+the+cellular+response+to+DNA+damage+or+replication+stress&doi=10.1073%2Fpnas.1006423108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span></div><div class="casAuthors">Aggarwal, Monika; Sommers, Joshua A.; Shoemaker, Robert H.; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1525-1530, S1525/1-S1525/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modulation of DNA repair proteins by small mols. has attracted great interest.  An in vitro helicase activity screen was used to identify mols. that modulate DNA unwinding by Werner syndrome helicase (WRN), mutated in the premature aging disorder Werner syndrome.  A small mol. from the National Cancer Institute Diversity Set designated NSC 19630 [1-(propoxymethyl)-maleimide] was identified that inhibited WRN helicase activity but did not affect other DNA helicases [Bloom syndrome (BLM), Fanconi anemia group J (FANCJ), RECQ1, RecQ, UvrD, or DnaB].  Exposure of human cells to NSC 19630 dramatically impaired growth and proliferation, induced apoptosis in a WRN-dependent manner, and resulted in elevated γ-H2AX and proliferating cell nuclear antigen (PCNA) foci.  NSC 19630 exposure led to delayed S-phase progression, consistent with the accumulation of stalled replication forks, and to DNA damage in a WRN-dependent manner.  Exposure to NSC 19630 sensitized cancer cells to the G-quadruplex-binding compd. telomestatin or a poly(ADP ribose) polymerase (PARP) inhibitor.  Sublethal dosage of NSC 19630 and the chemotherapy drug topotecan acted synergistically to inhibit cell proliferation and induce DNA damage.  The use of this WRN helicase inhibitor mol. may provide insight into the importance of WRN-mediated pathway(s) important for DNA repair and the replicational stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRIkmkwi9yCrVg90H21EOLACvtfcHk0lj6YxUNOwxk5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D&md5=d01ee377da5c65cdec72b0403494fa0e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1006423108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1006423108%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DM.%26aulast%3DSommers%26aufirst%3DJ.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DInhibition%2520of%2520helicase%2520activity%2520by%2520a%2520small%2520molecule%2520impairs%2520Werner%2520syndrome%2520helicase%2520%2528WRN%2529%2520function%2520in%2520the%2520cellular%2520response%2520to%2520DNA%2520damage%2520or%2520replication%2520stress%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D1525%26epage%3D1530%26doi%3D10.1073%2Fpnas.1006423108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination in cancer development, treatment and development of drug resistance</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fcarcin%2Fbgq064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20351092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=955-960&author=T.+Helleday&title=Homologous+recombination+in+cancer+development%2C+treatment+and+development+of+drug+resistance&doi=10.1093%2Fcarcin%2Fbgq064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in cancer development, treatment and development of drug resistance</span></div><div class="casAuthors">Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">955-960</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although DNA double-strand breaks (DSBs) are substrates for homologous recombination (HR) repair, it is becoming apparent that DNA lesions produced at replication forks, for instance by many anticancer drugs, are more significant substrates for HR repair.  Cells defective in HR are hypersensitive to a wide variety of anticancer drugs, including those that do not produce DSBs.  Several cancers have mutations in or epigenetically silenced HR genes, which explain the genetic instability that drives cancer development.  There are an increasing no. of reports suggesting that mutation or epigenetic silencing of HR genes explains the sensitivity of cancers to current chemotherapy treatments.  Furthermore, there are also many examples of re-expression of HR genes in tumors to explain drug resistance.  Emerging data suggest that there are several different subpathways of HR, which can compensate for each other.  Unravelling the overlapping pathways in HR showed that BRCA1- and BRCA2-defective cells rely on the PARP protein for survival.  This synthetic lethal interaction is now being exploited for selective treatment of BRCA1- and BRCA2-defective cancers with PARP inhibitors.  Here, I discuss the diversity of HR and how it impacts on cancer with a particular focus on how HR can be exploited in future anticancer strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaL2tqlRLWrVg90H21EOLACvtfcHk0lj6YxUNOwxk5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D&md5=1385e375ac39dd982ff9892fc8492486</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq064%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DHomologous%2520recombination%2520in%2520cancer%2520development%252C%2520treatment%2520and%2520development%2520of%2520drug%2520resistance%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D955%26epage%3D960%26doi%3D10.1093%2Fcarcin%2Fbgq064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hengel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors targeting DNA repair and DNA repair deficiency in research and cancer therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1101</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.chembiol.2017.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=28938088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1101-1119&author=S.+R.+Hengelauthor=M.+A.+Spiesauthor=M.+Spies&title=Small-molecule+inhibitors+targeting+DNA+repair+and+DNA+repair+deficiency+in+research+and+cancer+therapy&doi=10.1016%2Fj.chembiol.2017.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span></div><div class="casAuthors">Hengel, Sarah R.; Spies, M. Ashley; Spies, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1101-1119</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To maintain stable genomes and to avoid cancer and aging, cells need to repair a multitude of deleterious DNA lesions, which arise constantly in every cell.  Processes that support genome integrity in normal cells, however, allow cancer cells to develop resistance to radiation and DNA-damaging chemotherapeutics.  Chem. inhibition of the key DNA repair proteins and pharmacol. induced synthetic lethality have become instrumental in both dissecting the complex DNA repair networks and as promising anticancer agents.  The difficulty in capitalizing on synthetically lethal interactions in cancer cells is that many potential targets do not possess well-defined small-mol. binding determinates.  In this review, we discuss several successful campaigns to identify and leverage small-mol. inhibitors of the DNA repair proteins, from PARP1, a paradigm case for clin. successful small-mol. inhibitors, to coveted new targets, such as RAD51 recombinase, RAD52 DNA repair protein, MRE11 nuclease, and WRN DNA helicase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZweRKd9gHmLVg90H21EOLACvtfcHk0lj6YxUNOwxk5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE&md5=b8b865f8df43100effd60037efdc96b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DHengel%26aufirst%3DS.%2BR.%26aulast%3DSpies%26aufirst%3DM.%2BA.%26aulast%3DSpies%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520targeting%2520DNA%2520repair%2520and%2520DNA%2520repair%2520deficiency%2520in%2520research%2520and%2520cancer%2520therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1101%26epage%3D1119%26doi%3D10.1016%2Fj.chembiol.2017.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karow, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s syndrome gene product promotes branch migration of holliday junctions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6508</span>, <span class="refDoi"> DOI: 10.1073/pnas.100448097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.100448097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10823897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktFajt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=6504-6508&author=J.+K.+Karowauthor=A.+Constantinouauthor=J.-L.+Liauthor=S.+C.+Westauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+syndrome+gene+product+promotes+branch+migration+of+holliday+junctions&doi=10.1073%2Fpnas.100448097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's syndrome gene product promotes branch migration of Holliday junctions</span></div><div class="casAuthors">Karow, Julia K.; Constantinou, Angelos; Li, Ji-Liang; West, Stephen C.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6504-6508</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bloom's syndrome (BS) is an autosomal recessive disorder assocd. with dwarfism, immunodeficiency, reduced fertility, and elevated levels of many types of cancer.  BS cells show marked genomic instability; in particular, hyperrecombination between sister chromatids and homologous chromosomes.  This instability is thought to result from defective processing of DNA replication intermediates.  The gene mutated in BS, BLM, encodes a member of the RecQ family of DExH box DNA helicases, which also includes the Werner's syndrome gene product.  The authors have investigated the mechanism by which BLM suppresses hyperrecombination.  Here, the authors show that BLM selectively binds Holliday junctions in vitro and acts on recombination intermediates contg. a Holliday junction to promote ATP-dependent branch migration.  The authors present a model in which BLM disrupts potentially recombinogenic mols. that arise at sites of stalled replication forks.  The authors' results have implications for the role of BLM as an anti-recombinase in the suppression of tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeN_anR_Fmi7Vg90H21EOLACvtfcHk0liB4ZofV64JmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktFajt7o%253D&md5=de09613fb82612020f6e12a1695e1fb0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.100448097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.100448097%26sid%3Dliteratum%253Aachs%26aulast%3DKarow%26aufirst%3DJ.%2BK.%26aulast%3DConstantinou%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.-L.%26aulast%3DWest%26aufirst%3DS.%2BC.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520syndrome%2520gene%2520product%2520promotes%2520branch%2520migration%2520of%2520holliday%2520junctions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D6504%26epage%3D6508%26doi%3D10.1073%2Fpnas.100448097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budhathoki, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yodh, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balci, H.</span></span> <span> </span><span class="NLM_article-title">RecQ-core of BLM unfolds telomeric G-quadruplex in the absence of ATP</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">11528</span>– <span class="NLM_lpage">11545</span>, <span class="refDoi"> DOI: 10.1093/nar/gku856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgku856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25245947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKmurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=11528-11545&author=J.+B.+Budhathokiauthor=S.+Rayauthor=V.+Urbanauthor=P.+Janscakauthor=J.+G.+Yodhauthor=H.+Balci&title=RecQ-core+of+BLM+unfolds+telomeric+G-quadruplex+in+the+absence+of+ATP&doi=10.1093%2Fnar%2Fgku856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">RecQ-core of BLM unfolds telomeric G-quadruplex in the absence of ATP</span></div><div class="casAuthors">Budhathoki, Jagat B.; Ray, Sujay; Urban, Vaclav; Janscak, Pavel; Yodh, Jaya G.; Balci, Hamza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11528-11545</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Various helicases and single-stranded DNA (ss-DNA) binding proteins are known to destabilize G-quadruplex (GQ) structures, which otherwise result in genomic instability.  Bulk biochem. studies have shown that Bloom helicase (BLM) unfolds both intermol. and intramol. GQ in the presence of ATP.  Using single mol. FRET, we show that binding of RecQ-core of BLM (will be referred to as BLM) to ssDNA in the vicinity of an intramol. GQ leads to destabilization and unfolding of the GQ in the absence of ATP.  We show that the efficiency of BLM-mediated GQ unfolding correlates with the binding stability of BLM to ssDNA overhang, as modulated by the nucleotide state, ionic conditions, overhang length and overhang directionality.  In particular, we obsd. enhanced GQ unfolding by BLM in the presence of non-hydrolysable ATP analogs, which has implications for the underlying mechanism.  We also show that increasing GQ stability, via shorter loops or higher ionic strength, reduces BLM-mediated GQ unfolding.  Finally, we show that while WRN has similar activity as BLM, RecQ and RECQ5 helicases do not unfold GQ in the absence of ATP at physiol. ionic strength.  In summary, our study points to a novel and potentially very common mechanism of GQ destabilization mediated by proteins binding to the vicinity of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJvDxC7oRfHLVg90H21EOLACvtfcHk0liB4ZofV64JmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKmurrK&md5=eb27480107144f5e8039039696f6e3ed</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku856%26sid%3Dliteratum%253Aachs%26aulast%3DBudhathoki%26aufirst%3DJ.%2BB.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DUrban%26aufirst%3DV.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DYodh%26aufirst%3DJ.%2BG.%26aulast%3DBalci%26aufirst%3DH.%26atitle%3DRecQ-core%2520of%2520BLM%2520unfolds%2520telomeric%2520G-quadruplex%2520in%2520the%2520absence%2520of%2520ATP%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D11528%26epage%3D11545%26doi%3D10.1093%2Fnar%2Fgku856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Fatah, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleskandarany, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshareeda, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic and protein expression analysis reveals clinicopathological significance of Bloom syndrome helicase (BLM) in breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1065</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1535-7163.mct-14-0939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25673821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlCitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1057-1065&author=A.+Aroraauthor=T.+M.+A.+Abdel-Fatahauthor=D.+Agarwalauthor=R.+Dohertyauthor=P.+M.+Moseleyauthor=M.+A.+Aleskandaranyauthor=A.+R.+Greenauthor=G.+Ballauthor=A.+T.+Alshareedaauthor=E.+A.+Rakhaauthor=S.+Y.+T.+Chanauthor=I.+O.+Ellisauthor=S.+Madhusudan&title=Transcriptomic+and+protein+expression+analysis+reveals+clinicopathological+significance+of+Bloom+syndrome+helicase+%28BLM%29+in+breast+cancer&doi=10.1158%2F1535-7163.mct-14-0939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic and Protein Expression Analysis Reveals Clinicopathological Significance of Bloom Syndrome Helicase (BLM) in Breast Cancer</span></div><div class="casAuthors">Arora, Arvind; Abdel-Fatah, Tarek M. A.; Agarwal, Devika; Doherty, Rachel; Moseley, Paul M.; Aleskandarany, Mohammed A.; Green, Andrew R.; Ball, Graham; Alshareeda, Alaa T.; Rakha, Emad A.; Chan, Stephen Y. T.; Ellis, Ian O.; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1057-1065</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bloom syndrome helicase (BLM) has key roles in homologous recombination repair, telomere maintenance, and DNA replication.  Germ-line mutations in the BLM gene causes Bloom syndrome, a rare disorder characterized by premature aging and predisposition to multiple cancers, including breast cancer.  The clinicopathol. significance of BLM in sporadic breast cancers is unknown.  We investigated BLM mRNA expression in the Mol. Taxonomy of Breast Cancer International Consortium cohort (n = 1,950) and validated in an external dataset of 2,413 tumors.  BLM protein level was evaluated in the Nottingham Tenovus series comprising 1,650 breast tumors.  BLM mRNA overexpression was significantly assocd. with high histol. grade, larger tumor size, estrogen receptor-neg. (ER-), progesterone receptor-neg. (PR-), and triple-neg. phenotypes (ps < 0.0001).  BLM mRNA overexpression was also linked to aggressive mol. phenotypes, including PAM50.Her2 (P < 0.0001), PAM50.Basal (P < 0.0001), and PAM50.LumB (P < 0.0001) and Genufu subtype (ER+/Her2-/high proliferation; P < 0.0001). (ER+/Her2-/low proliferation) were more likely to express low levels of BLM mRNA (ps < 0.0001).  Integrative mol. clusters (intClust) intClust.1 (P < 0.0001), intClust.5 (P < 0.0001), intClust.9 (P < 0.0001), and intClust.10 (P < 0.0001) were also more likely in tumors with high BLM mRNA expression.  BLM mRNA overexpression was assocd. with poor breast cancer-specific survival (BCSS; ps < 0.000001).  At the protein level, altered subcellular localization with high cytoplasmic BLM and low nuclear BLM was linked to aggressive phenotypes.  In multivariate anal., BLM mRNA and BLM protein levels independently influenced BCSS.  This is the first and the largest study to provide evidence that BLM is a promising biomarker in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgXUCTp8NP27Vg90H21EOLACvtfcHk0liB4ZofV64JmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlCitrw%253D&md5=8dc54a6fd7887f63f676c147de97273b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0939%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DAbdel-Fatah%26aufirst%3DT.%2BM.%2BA.%26aulast%3DAgarwal%26aufirst%3DD.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DMoseley%26aufirst%3DP.%2BM.%26aulast%3DAleskandarany%26aufirst%3DM.%2BA.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DBall%26aufirst%3DG.%26aulast%3DAlshareeda%26aufirst%3DA.%2BT.%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DChan%26aufirst%3DS.%2BY.%2BT.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DTranscriptomic%2520and%2520protein%2520expression%2520analysis%2520reveals%2520clinicopathological%2520significance%2520of%2520Bloom%2520syndrome%2520helicase%2520%2528BLM%2529%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3D1057%26epage%3D1065%26doi%3D10.1158%2F1535-7163.mct-14-0939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s syndrome helicase suppresses crossing over during homologous recombination</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1038/nature02253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnature02253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=14685245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVGmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2003&pages=870-874&author=L.+Wuauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+syndrome+helicase+suppresses+crossing+over+during+homologous+recombination&doi=10.1038%2Fnature02253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's syndrome helicase suppresses crossing over during homologous recombination</span></div><div class="casAuthors">Wu, Leonard; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">6968</span>),
    <span class="NLM_cas:pages">870-874</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in BLM, which encodes a RecQ helicase, give rise to Bloom's syndrome, a disorder assocd. with cancer predisposition and genomic instability.  A defining feature of Bloom's syndrome is an elevated frequency of sister chromatid exchanges.  These arise from crossing over of chromatid arms during homologous recombination, a ubiquitous process that exists to repair DNA double-stranded breaks and damaged replication forks.  Whereas crossing over is required in meiosis, in mitotic cells it can be assocd. with detrimental loss of heterozygosity.  BLM forms an evolutionarily conserved complex with human topoisomerase IIIα (hTOPO IIIα), which can break and rejoin DNA to alter its topol.  Inactivation of homologs of either protein leads to hyper-recombination in unicellular organisms.  Here, the authors show that BLM and hTOPO IIIα together effect the resoln. of a recombination intermediate contg. a double Holliday junction.  The mechanism, which the authors term double-junction dissoln., is distinct from classical Holliday junction resoln. and prevents exchange of flanking sequences.  Loss of such an activity explains many of the cellular phenotypes of Bloom's syndrome.  These results have wider implications for our understanding of the process of homologous recombination and the mechanisms that exist to prevent tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUs7F5HuS1KrVg90H21EOLACvtfcHk0ljSt7o6bDv2EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVGmtbs%253D&md5=f98f21ab5b3557fe2edd1359678402fd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature02253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature02253%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%2520suppresses%2520crossing%2520over%2520during%2520homologous%2520recombination%26jtitle%3DNature%26date%3D2003%26volume%3D426%26spage%3D870%26epage%3D874%26doi%3D10.1038%2Fnature02253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bugreev, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egelman, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazin, A. V.</span></span> <span> </span><span class="NLM_article-title">Novel pro- and anti-recombination activities of the Bloom’s syndrome helicase</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3085</span>– <span class="NLM_lpage">3094</span>, <span class="refDoi"> DOI: 10.1101/gad.1609007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.1609007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18003860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3085-3094&author=D.+V.+Bugreevauthor=X.+Yuauthor=E.+H.+Egelmanauthor=A.+V.+Mazin&title=Novel+pro-+and+anti-recombination+activities+of+the+Bloom%E2%80%99s+syndrome+helicase&doi=10.1101%2Fgad.1609007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pro- and anti-recombination activities of the Bloom's syndrome helicase</span></div><div class="casAuthors">Bugreev, Dmitry V.; Yu, Xiong; Egelman, Edward H.; Mazin, Alexander V.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3085-3094</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bloom's syndrome (BS) is an autosomal recessive disorder characterized by a strong cancer predisposition.  The defining feature of BS is extreme genome instability.  The gene mutated in Bloom's syndrome, BLM, encodes a DNA helicase (BLM) of the RecQ family.  BLM plays a role in homologous recombination; however, its exact function remains controversial.  Mutations in the BLM cause hyperrecombination between sister chromatids and homologous chromosomes, indicating an anti-recombination role.  Conversely, other data show that BLM is required for recombination.  It was previously shown that in vitro BLM helicase promotes disruption of recombination intermediates, regression of stalled replication forks, and dissoln. of double Holliday junctions.  Here, we demonstrate two novel activities of BLM: disruption of the Rad51-ssDNA (single-stranded DNA) filament, an active species that promotes homologous recombination, and stimulation of DNA repair synthesis.  Using in vitro reconstitution reactions, we analyzed how different biochem. activities of BLM contribute to its functions in homologous recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr--SBOK6pjb7Vg90H21EOLACvtfcHk0ljSt7o6bDv2EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vP&md5=1ddadff75bf07fea1a83263eab68f80c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1101%2Fgad.1609007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1609007%26sid%3Dliteratum%253Aachs%26aulast%3DBugreev%26aufirst%3DD.%2BV.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DEgelman%26aufirst%3DE.%2BH.%26aulast%3DMazin%26aufirst%3DA.%2BV.%26atitle%3DNovel%2520pro-%2520and%2520anti-recombination%2520activities%2520of%2520the%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3085%26epage%3D3094%26doi%3D10.1101%2Fgad.1609007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misenko, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Her, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S. F.</span></span> <span> </span><span class="NLM_article-title">BLM helicase regulates DNA repair by counteracting RAD51 loading at DNA double-strand break sites</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>216</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3534</span>, <span class="refDoi"> DOI: 10.1083/jcb.201703144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1083%2Fjcb.201703144" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=216&publication_year=2017&pages=3521-3534&author=D.+S.+Patelauthor=S.+M.+Misenkoauthor=J.+Herauthor=S.+F.+Bunting&title=BLM+helicase+regulates+DNA+repair+by+counteracting+RAD51+loading+at+DNA+double-strand+break+sites&doi=10.1083%2Fjcb.201703144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201703144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201703144%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BS.%26aulast%3DMisenko%26aufirst%3DS.%2BM.%26aulast%3DHer%26aufirst%3DJ.%26aulast%3DBunting%26aufirst%3DS.%2BF.%26atitle%3DBLM%2520helicase%2520regulates%2520DNA%2520repair%2520by%2520counteracting%2520RAD51%2520loading%2520at%2520DNA%2520double-strand%2520break%2520sites%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2017%26volume%3D216%26spage%3D3521%26epage%3D3534%26doi%3D10.1083%2Fjcb.201703144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidorova, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauper, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emond, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnat, R. J.</span></span> <span> </span><span class="NLM_article-title">The human WRN and BLM RecQ helicases differentially regulate cell proliferation and survival after chemotherapeutic DNA damage</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">6548</span>– <span class="NLM_lpage">6555</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-0475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F0008-5472.can-10-0475" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=6548-6555&author=F.+J.+Maoauthor=J.+M.+Sidorovaauthor=J.+M.+Lauperauthor=M.+J.+Emondauthor=R.+J.+Monnat&title=The+human+WRN+and+BLM+RecQ+helicases+differentially+regulate+cell+proliferation+and+survival+after+chemotherapeutic+DNA+damage&doi=10.1158%2F0008-5472.can-10-0475"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-10-0475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-10-0475%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%2BJ.%26aulast%3DSidorova%26aufirst%3DJ.%2BM.%26aulast%3DLauper%26aufirst%3DJ.%2BM.%26aulast%3DEmond%26aufirst%3DM.%2BJ.%26aulast%3DMonnat%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520human%2520WRN%2520and%2520BLM%2520RecQ%2520helicases%2520differentially%2520regulate%2520cell%2520proliferation%2520and%2520survival%2520after%2520chemotherapeutic%2520DNA%2520damage%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D6548%26epage%3D6555%26doi%3D10.1158%2F0008-5472.can-10-0475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahkim Bennani-Belhaj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouzeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhagiar-Labarchede, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabosseau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onclercq-Delic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayart, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordelieres, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couturier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amor-Gueret, M.</span></span> <span> </span><span class="NLM_article-title">The Bloom syndrome protein limits the lethality associated with RAD51 deficiency</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-09-0534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1541-7786.mcr-09-0534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=20215422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ptV2nsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=385-394&author=K.+Lahkim+Bennani-Belhajauthor=S.+Rouzeauauthor=G.+Buhagiar-Labarchedeauthor=P.+Chabosseauauthor=R.+Onclercq-Delicauthor=E.+Bayartauthor=F.+Cordelieresauthor=J.+Couturierauthor=M.+Amor-Gueret&title=The+Bloom+syndrome+protein+limits+the+lethality+associated+with+RAD51+deficiency&doi=10.1158%2F1541-7786.mcr-09-0534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom syndrome protein limits the lethality associated with RAD51 deficiency</span></div><div class="casAuthors">Lahkim Bennani-Belhaj Kenza; Rouzeau Sebastien; Buhagiar-Labarchede Geraldine; Chabosseau Pauline; Onclercq-Delic Rosine; Bayart Emilie; Cordelieres Fabrice; Couturier Jerome; Amor-Gueret Mounira</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer research : MCR</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Little is known about the functional interaction between the Bloom's syndrome protein (BLM) and the recombinase RAD51 within cells.  Using RNA interference technology, we provide the first demonstration that RAD51 acts upstream from BLM to prevent anaphase bridge formation.  RAD51 downregulation was associated with an increase in the frequency of BLM-positive anaphase bridges, but not of BLM-associated ultrafine bridges.  Time-lapse live microscopy analysis of anaphase bridge cells revealed that BLM promoted cell survival in the absence of Rad51.  Our results directly implicate BLM in limiting the lethality associated with RAD51 deficiency through the processing of anaphase bridges resulting from the RAD51 defect.  These findings provide insight into the molecular basis of some cancers possibly associated with variants of the RAD51 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSiFpylsdl8U8NIRw0GKUoEfW6udTcc2ebvIK6RUjgU-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ptV2nsw%253D%253D&md5=dd11389bab5efe0daa81c38ce3a8e9fe</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-09-0534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-09-0534%26sid%3Dliteratum%253Aachs%26aulast%3DLahkim%2BBennani-Belhaj%26aufirst%3DK.%26aulast%3DRouzeau%26aufirst%3DS.%26aulast%3DBuhagiar-Labarchede%26aufirst%3DG.%26aulast%3DChabosseau%26aufirst%3DP.%26aulast%3DOnclercq-Delic%26aufirst%3DR.%26aulast%3DBayart%26aufirst%3DE.%26aulast%3DCordelieres%26aufirst%3DF.%26aulast%3DCouturier%26aufirst%3DJ.%26aulast%3DAmor-Gueret%26aufirst%3DM.%26atitle%3DThe%2520Bloom%2520syndrome%2520protein%2520limits%2520the%2520lethality%2520associated%2520with%2520RAD51%2520deficiency%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2010%26volume%3D8%26spage%3D385%26epage%3D394%26doi%3D10.1158%2F1541-7786.mcr-09-0534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosedale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachrati, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">A small molecule inhibitor of the BLM helicase modulates chromosome stability in human cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.chembiol.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=23352139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=55-62&author=G.+H.+Nguyenauthor=T.+S.+Dexheimerauthor=A.+S.+Rosenthalauthor=W.+K.+Chuauthor=D.+K.+Singhauthor=G.+Mosedaleauthor=C.+Z.+Bachratiauthor=L.+Schultzauthor=M.+Sakuraiauthor=P.+Savitskyauthor=M.+Abuauthor=P.+J.+McHughauthor=V.+A.+Bohrauthor=C.+C.+Harrisauthor=A.+Jadhavauthor=O.+Gileadiauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=I.+D.+Hickson&title=A+small+molecule+inhibitor+of+the+BLM+helicase+modulates+chromosome+stability+in+human+cells&doi=10.1016%2Fj.chembiol.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span></div><div class="casAuthors">Nguyen, Giang Huong; Dexheimer, Thomas S.; Rosenthal, Andrew S.; Chu, Wai Kit; Singh, Dharmendra Kumar; Mosedale, Georgina; Bachrati, Csanad Z.; Schultz, Lena; Sakurai, Masaaki; Savitsky, Pavel; Abu, Mika; McHugh, Peter J.; Bohr, Vilhelm A.; Harris, Curtis C.; Jadhav, Ajit; Gileadi, Opher; Maloney, David J.; Simeonov, Anton; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bloom's syndrome protein, BLM, is a member of the conserved RecQ helicase family.  Although cell lines lacking BLM exist, these exhibit progressive genomic instability that makes distinguishing primary from secondary effects of BLM loss problematic.  In order to be able to acutely disable BLM function in cells, we undertook a high throughput screen of a chem. compd. library for small mol. inhibitors of BLM.  We present ML216, a potent inhibitor of the DNA unwinding activity of BLM.  ML216 shows cell-based activity and can induce sister chromatid exchanges, enhance the toxicity of aphidicolin, and exert antiproliferative activity in cells expressing BLM, but not those lacking BLM.  These data indicate that ML216 shows strong selectivity for BLM in cultured cells.  We discuss the potential utility of such a BLM-targeting compd. as an anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhJ5QlteU3zLVg90H21EOLACvtfcHk0lgiDdml8TC9iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D&md5=26bfa8e45f2253aa8807d055be0689f2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DG.%2BH.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DChu%26aufirst%3DW.%2BK.%26aulast%3DSingh%26aufirst%3DD.%2BK.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DBachrati%26aufirst%3DC.%2BZ.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAbu%26aufirst%3DM.%26aulast%3DMcHugh%26aufirst%3DP.%2BJ.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DA%2520small%2520molecule%2520inhibitor%2520of%2520the%2520BLM%2520helicase%2520modulates%2520chromosome%2520stability%2520in%2520human%2520cells%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.chembiol.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of new selective NM23-H2 binders as c-MYC transcription inhibitors via disruption of the NM23-H2/G-Quadruplex interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6924</span>– <span class="NLM_lpage">6941</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00421</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00421" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6924-6941&author=Y.-Q.+Wangauthor=Z.-L.+Huangauthor=S.-B.+Chenauthor=C.-X.+Wangauthor=C.+Shanauthor=Q.-K.+Yinauthor=T.-M.+Ouauthor=D.+Liauthor=L.-Q.+Guauthor=J.-H.+Tanauthor=Z.-S.+Huang&title=Design%2C+synthesis%2C+and+evaluation+of+new+selective+NM23-H2+binders+as+c-MYC+transcription+inhibitors+via+disruption+of+the+NM23-H2%2FG-Quadruplex+interaction&doi=10.1021%2Facs.jmedchem.7b00421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of New Selective NM23-H2 Binders as c-MYC Transcription Inhibitors via Disruption of the NM23-H2/G-Quadruplex Interaction</span></div><div class="casAuthors">Wang, Yu-Qing; Huang, Zhou-Li; Chen, Shuo-Bin; Wang, Chen-Xi; Shan, Chan; Yin, Qi-Kun; Ou, Tian-Miao; Li, Ding; Gu, Lian-Quan; Tan, Jia-Heng; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6924-6941</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-MYC is one of the important human proto-oncogenes, and transcriptional factor NM23-H2 can activate c-MYC transcription by recognizing the G-quadruplex in the promoter of the gene.  Small mols. that inhibit c-MYC transcription by disrupting the NM23-H2/G-quadruplex interaction might be a promising strategy for developing selective anticancer agents.  In recent studies, we developed a series of isaindigotone derivs., which can bind to G-quadruplex and NM23-H2, thus down-regulating c-MYC (J. Med. Chem. 2017, 60, 1292-1308).  Herein, a series of novel isaindigotone derivs. were designed, synthesized, and screened for NM23-H2 selective binding ligands.  Among them, compd. I showed a high specific binding affinity to NM23-H2, effectively disrupting the interaction of NM23-H2 with G-quadruplex, and it strongly down-regulated c-MYC transcription.  Furthermore, I induced cell cycle arrest and apoptosis, and it exhibited good tumor growth inhibition in a mouse xenograft model.  This work provides a new strategy to modulate c-MYC transcription for the development of selective anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH6VlXYOPCY7Vg90H21EOLACvtfcHk0lgiDdml8TC9iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jL&md5=7ba64697833652269acc158833534ce5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00421%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DHuang%26aufirst%3DZ.-L.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DWang%26aufirst%3DC.-X.%26aulast%3DShan%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DQ.-K.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.-Q.%26aulast%3DTan%26aufirst%3DJ.-H.%26aulast%3DHuang%26aufirst%3DZ.-S.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520new%2520selective%2520NM23-H2%2520binders%2520as%2520c-MYC%2520transcription%2520inhibitors%2520via%2520disruption%2520of%2520the%2520NM23-H2%252FG-Quadruplex%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6924%26epage%3D6941%26doi%3D10.1021%2Facs.jmedchem.7b00421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohaghegh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">The Bloom’s and Werner’s syndrome proteins are DNA structure-specific helicases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2843</span>– <span class="NLM_lpage">2849</span>, <span class="refDoi"> DOI: 10.1093/nar/29.13.2843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2F29.13.2843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=11433031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlejs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=2843-2849&author=P.+Mohagheghauthor=J.+K.+Karowauthor=R.+M.+Broshauthor=V.+A.+Bohrauthor=I.+D.+Hickson&title=The+Bloom%E2%80%99s+and+Werner%E2%80%99s+syndrome+proteins+are+DNA+structure-specific+helicases&doi=10.1093%2Fnar%2F29.13.2843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Bloom's and Werner's syndrome proteins are DNA structure-specific helicases</span></div><div class="casAuthors">Mohaghegh, Payam; Karow, Julia K.; Brosh, Robert M., Jr.; Bohr, Vilhelm A.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2843-2849</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BLM and WRN, the products of the Bloom's and Werner's syndrome genes, are members of the RecQ family of DNA helicases.  Although both have been shown previously to unwind simple, partial duplex DNA substrates with 3'→5' polarity, little is known about the structural features of DNA that det. the substrate specificities of these enzymes.  We have compared the substrate specificities of the BLM and WRN proteins using a variety of partial duplex DNA mols., which are based upon a common core nucleotide sequence.  We show that neither BLM nor WRN is capable of unwinding duplex DNA from a blunt-ended terminus or from an internal nick.  However, both enzymes efficiently unwind the same blunt-ended duplex contg. a centrally located 12 nt single-stranded "bubble", as well as a synthetic X-structure (a model for the Holliday junction recombination intermediate) in which each "arm" of the 4-way junction is blunt-ended.  Surprisingly, a 3'-tailed duplex, a std. substrate for 3'→5' helicases, is unwound much less efficiently by BLM and WRN than are the bubble and X-structure substrates.  These data show conclusively that a single-stranded 3'-tail is not a structural requirement for unwinding of std. B-form DNA by these helicases.  BLM and WRN also both unwind a variety of different forms of G-quadruplex DNA, a structure that can form at guanine-rich sequences present at several genomic loci.  Our data indicate that BLM and WRN are atypical helicases that are highly DNA structure specific and have similar substrate specificities.  We interpret these data in the light of the genomic instability and hyper-recombination characteristics of cells from individuals with Bloom's or Werner's syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY0ZC4RKKynLVg90H21EOLACvtfcHk0lgiDdml8TC9iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlejs7s%253D&md5=8a32f9fddbf47c0c54141d80b83431d5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F29.13.2843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F29.13.2843%26sid%3Dliteratum%253Aachs%26aulast%3DMohaghegh%26aufirst%3DP.%26aulast%3DKarow%26aufirst%3DJ.%2BK.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DThe%2520Bloom%25E2%2580%2599s%2520and%2520Werner%25E2%2580%2599s%2520syndrome%2520proteins%2520are%2520DNA%2520structure-specific%2520helicases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2001%26volume%3D29%26spage%3D2843%26epage%3D2849%26doi%3D10.1093%2Fnar%2F29.13.2843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guirouilh-Barbat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacca, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Endogenous -H2AX-ATM-Chk2 Checkpoint Activation in Bloom’s Syndrome Helicase Deficient Cells Is Related to DNA Replication Arrested Forks</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-07-0028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1158%2F1541-7786.mcr-07-0028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17634426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVChtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=713-724&author=V.+A.+Raoauthor=C.+Contiauthor=J.+Guirouilh-Barbatauthor=A.+Nakamuraauthor=Z.-H.+Miaoauthor=S.+L.+Daviesauthor=B.+Saccaauthor=I.+D.+Hicksonauthor=A.+Bensimonauthor=Y.+Pommier&title=Endogenous+-H2AX-ATM-Chk2+Checkpoint+Activation+in+Bloom%E2%80%99s+Syndrome+Helicase+Deficient+Cells+Is+Related+to+DNA+Replication+Arrested+Forks&doi=10.1158%2F1541-7786.mcr-07-0028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous γ-H2AX-ATM-Chk2 Checkpoint Activation in Bloom's Syndrome Helicase-Deficient Cells Is Related to DNA Replication Arrested Forks</span></div><div class="casAuthors">Rao, V. Ashutosh; Conti, Chiara; Guirouilh-Barbat, Josee; Nakamura, Asako; Miao, Ze-Hong; Davies, Sally L.; Sacca, Barbara; Hickson, Ian D.; Bensimon, Aaron; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">713-724</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The Bloom syndrome helicase (BLM) is crit. for genomic stability.  A defect in BLM activity results in the cancer-predisposing Bloom syndrome (BS).  Here, the authors report that BLM-deficient cell lines and primary fibroblasts display an endogenously activated DNA double-strand break checkpoint response with prominent levels of phosphorylated histone H2AX (γ-H2AX), Chk2 (pT68Chk2), and ATM (pS1981ATM) colocalizing in nuclear foci.  Interestingly, the mitotic fraction of γ-H2AX foci did not seem to be higher in BLM-deficient cells, indicating that these lesions form transiently during interphase.  Pulse labeling with iododeoxyuridine and immunofluorescence microscopy showed the colocalization of γ-H2AX, ATM, and Chk2 together with replication foci.  Those foci costained for Rad51, indicating homologous recombination at these replication sites.  The authors therefore analyzed replication in BS cells using a single mol. approach on combed DNA fibers.  In addn. to a higher frequency of replication fork barriers, BS cells displayed a reduced av. fork velocity and global redn. of interorigin distances indicative of an elevated frequency of origin firing.  Because BS is one of the most penetrant cancer-predisposing hereditary diseases, it is likely that the lack of BLM engages the cells in a situation similar to precancerous tissues with replication stress.  To the authors' knowledge, this is the first report of high ATM-Chk2 kinase activation and its linkage to replication defects in a BS model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ0QOY7Yc62rVg90H21EOLACvtfcHk0lg2TRVNkANwTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvVChtb4%253D&md5=8d75a91c20823851a205af82d6d07a7a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-07-0028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-07-0028%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DV.%2BA.%26aulast%3DConti%26aufirst%3DC.%26aulast%3DGuirouilh-Barbat%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DA.%26aulast%3DMiao%26aufirst%3DZ.-H.%26aulast%3DDavies%26aufirst%3DS.%2BL.%26aulast%3DSacca%26aufirst%3DB.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DBensimon%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DEndogenous%2520-H2AX-ATM-Chk2%2520Checkpoint%2520Activation%2520in%2520Bloom%25E2%2580%2599s%2520Syndrome%2520Helicase%2520Deficient%2520Cells%2520Is%2520Related%2520to%2520DNA%2520Replication%2520Arrested%2520Forks%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2007%26volume%3D5%26spage%3D713%26epage%3D724%26doi%3D10.1158%2F1541-7786.mcr-07-0028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nimonkar, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozsoy, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genschel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczykowski, S. C.</span></span> <span> </span><span class="NLM_article-title">Human exonuclease 1 and BLM helicase interact to resect DNA and initiate DNA repair</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">16906</span>– <span class="NLM_lpage">16911</span>, <span class="refDoi"> DOI: 10.1073/pnas.0809380105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1073%2Fpnas.0809380105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=18971343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlKmu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16906-16911&author=A.+V.+Nimonkarauthor=A.+Z.+Ozsoyauthor=J.+Genschelauthor=P.+Modrichauthor=S.+C.+Kowalczykowski&title=Human+exonuclease+1+and+BLM+helicase+interact+to+resect+DNA+and+initiate+DNA+repair&doi=10.1073%2Fpnas.0809380105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Human exonuclease 1 and BLM helicase interact to resect DNA and initiate DNA repair</span></div><div class="casAuthors">Nimonkar, Amitabh V.; Ozsoy, A. Zeynep; Genschel, Jochen; Modrich, Paul; Kowalczykowski, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">16906-16911</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The error-free repair of double-stranded DNA breaks by homologous recombination requires processing of broken ends.  These processed ends are substrates for assembly of DNA strand exchange proteins that mediate DNA strand invasion.  Here, we establish that human BLM helicase, a member of the RecQ family, stimulates the nucleolytic activity of human exonuclease 1 (hExo1), a 5' → 3' double-stranded DNA exonuclease.  The stimulation is specific because other RecQ homologs fail to stimulate hExol.  Stimulation of DNA resection by hExo1 is independent of BLM helicase activity and is, instead, mediated by an interaction between the 2 proteins.  Finally, we show that DNA ends resected by hExo1 and BLM are used by human Rad51, but not its yeast or bacterial counterparts, to promote homologous DNA pairing.  This in vitro system recapitulates initial steps of homologous recombination and provides biochem. evidence for a role of BLM and Exo1 in the initiation of recombinational DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE87ZrQuRE7Vg90H21EOLACvtfcHk0lg2TRVNkANwTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlKmu7nI&md5=c5c9381a86a5712595b55412f0bd8575</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0809380105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0809380105%26sid%3Dliteratum%253Aachs%26aulast%3DNimonkar%26aufirst%3DA.%2BV.%26aulast%3DOzsoy%26aufirst%3DA.%2BZ.%26aulast%3DGenschel%26aufirst%3DJ.%26aulast%3DModrich%26aufirst%3DP.%26aulast%3DKowalczykowski%26aufirst%3DS.%2BC.%26atitle%3DHuman%2520exonuclease%25201%2520and%2520BLM%2520helicase%2520interact%2520to%2520resect%2520DNA%2520and%2520initiate%2520DNA%2520repair%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D16906%26epage%3D16911%26doi%3D10.1073%2Fpnas.0809380105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasin, M.</span></span> <span> </span><span class="NLM_article-title">XRCC3 promotes homology-directed repair of DNA damage in mammalian cells</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2638</span>, <span class="refDoi"> DOI: 10.1101/gad.13.20.2633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.13.20.2633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10541549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADyaK1MXnt1Wqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=2633-2638&author=A.+J.+Pierceauthor=R.+D.+Johnsonauthor=L.+H.+Thompsonauthor=M.+Jasin&title=XRCC3+promotes+homology-directed+repair+of+DNA+damage+in+mammalian+cells&doi=10.1101%2Fgad.13.20.2633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">XRCC3 promotes homology-directed repair of DNA damage in mammalian cells</span></div><div class="casAuthors">Pierce, Andrew J.; Johnson, Roger D.; Thompson, Larry H.; Jasin, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2633-2638</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Homol.-directed repair of DNA damage has recently emerged as a major mechanism for the maintenance of genomic integrity in mammalian cells.  The highly conserved strand transferase, Rad51, is expected to be crit. for this process.  XRCC3 possesses a limited sequence similarity to Rad51 and interacts with it.  Using a novel fluorescence-based assay, the authors demonstrate here that error-free homol.-directed repair of DNA double-strand breaks is decreased 25-fold in an XRCC3-deficient hamster cell line and can be restored to wild-type levels through XRCC3 expression.  These results establish that XRCC3-mediated homologous recombination can reverse DNA damage that would otherwise be mutagenic or lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc5mIYaRKGbVg90H21EOLACvtfcHk0lg2TRVNkANwTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnt1Wqsrk%253D&md5=8301dc45408d2f4ec6d5d6965f973d8d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.20.2633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.20.2633%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BJ.%26aulast%3DJohnson%26aufirst%3DR.%2BD.%26aulast%3DThompson%26aufirst%3DL.%2BH.%26aulast%3DJasin%26aufirst%3DM.%26atitle%3DXRCC3%2520promotes%2520homology-directed%2520repair%2520of%2520DNA%2520damage%2520in%2520mammalian%2520cells%26jtitle%3DGenes%2520Dev.%26date%3D1999%26volume%3D13%26spage%3D2633%26epage%3D2638%26doi%3D10.1101%2Fgad.13.20.2633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Traverso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettegowda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzler, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span> <span> </span><span class="NLM_article-title">Hyper-recombination and genetic instability in BLM-deficient epithelial cells</span>. <i>Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8578</span>– <span class="NLM_lpage">8581</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=8578-8581&author=G.+Traversoauthor=C.+Bettegowdaauthor=J.+Krausauthor=M.+R.+Speicherauthor=K.+W.+Kinzlerauthor=B.+Vogelsteinauthor=C.+Lengauer&title=Hyper-recombination+and+genetic+instability+in+BLM-deficient+epithelial+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraverso%26aufirst%3DG.%26aulast%3DBettegowda%26aufirst%3DC.%26aulast%3DKraus%26aufirst%3DJ.%26aulast%3DSpeicher%26aufirst%3DM.%2BR.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DHyper-recombination%2520and%2520genetic%2520instability%2520in%2520BLM-deficient%2520epithelial%2520cells%26jtitle%3DCancer%2520Res%26date%3D2003%26volume%3D63%26spage%3D8578%26epage%3D8581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nimonkar, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genschel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polaczek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczykowski, S. C.</span></span> <span> </span><span class="NLM_article-title">BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1101/gad.2003811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1101%2Fgad.2003811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=21325134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVait7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=350-362&author=A.+V.+Nimonkarauthor=J.+Genschelauthor=E.+Kinoshitaauthor=P.+Polaczekauthor=J.+L.+Campbellauthor=C.+Wymanauthor=P.+Modrichauthor=S.+C.+Kowalczykowski&title=BLM-DNA2-RPA-MRN+and+EXO1-BLM-RPA-MRN+constitute+two+DNA+end+resection+machineries+for+human+DNA+break+repair&doi=10.1101%2Fgad.2003811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair</span></div><div class="casAuthors">Nimonkar, Amitabh V.; Genschel, Jochen; Kinoshita, Eri; Polaczek, Piotr; Campbell, Judith L.; Wyman, Claire; Modrich, Paul; Kowalczykowski, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">350-362</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Repair of dsDNA breaks requires processing to produce 3'-terminated ssDNA.  We biochem. reconstituted DNA end resection using purified human proteins: Bloom helicase (BLM); DNA2 helicase/nuclease; Exonuclease 1 (EXO1); the complex comprising MRE11, RAD50, and NBS1 (MRN); and Replication protein A (RPA).  Resection occurs via two routes.  In one, BLM and DNA2 phys. and specifically interact to resect DNA in a process that is ATP-dependent and requires BLM helicase and DNA2 nuclease functions.  RPA is essential for both DNA unwinding by BLM and enforcing 5' → 3' resection polarity by DNA2.  MRN accelerates processing by recruiting BLM to the end.  In the other, EXO1 resects the DNA and is stimulated by BLM, MRN, and RPA.  BLM increases the affinity of EXO1 for ends, and MRN recruits and enhances the processivity of EXO1.  Our results establish two of the core machineries that initiate recombinational DNA repair in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfq3dsij9QbbVg90H21EOLACvtfcHk0liSVfKfRgNo4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVait7Y%253D&md5=794da119abc7dbadc0ce8aebafbf027b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1101%2Fgad.2003811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2003811%26sid%3Dliteratum%253Aachs%26aulast%3DNimonkar%26aufirst%3DA.%2BV.%26aulast%3DGenschel%26aufirst%3DJ.%26aulast%3DKinoshita%26aufirst%3DE.%26aulast%3DPolaczek%26aufirst%3DP.%26aulast%3DCampbell%26aufirst%3DJ.%2BL.%26aulast%3DWyman%26aufirst%3DC.%26aulast%3DModrich%26aufirst%3DP.%26aulast%3DKowalczykowski%26aufirst%3DS.%2BC.%26atitle%3DBLM-DNA2-RPA-MRN%2520and%2520EXO1-BLM-RPA-MRN%2520constitute%2520two%2520DNA%2520end%2520resection%2520machineries%2520for%2520human%2520DNA%2520break%2520repair%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D350%26epage%3D362%26doi%3D10.1101%2Fgad.2003811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span> <span> </span><span class="NLM_article-title">MRN complex-dependent recruitment of ubiquitylated BLM helicase to DSBs negatively regulates DNA repair pathways</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1016</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-03393-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fs41467-018-03393-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=29523790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC1MrpsF2hug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1016&author=V.+Tripathiauthor=H.+Agarwalauthor=S.+Priyaauthor=H.+Batraauthor=P.+Modiauthor=M.+Pandeyauthor=D.+Sahaauthor=S.+C.+Raghavanauthor=S.+Sengupta&title=MRN+complex-dependent+recruitment+of+ubiquitylated+BLM+helicase+to+DSBs+negatively+regulates+DNA+repair+pathways&doi=10.1038%2Fs41467-018-03393-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MRN complex-dependent recruitment of ubiquitylated BLM helicase to DSBs negatively regulates DNA repair pathways</span></div><div class="casAuthors">Tripathi Vivek; Agarwal Himanshi; Priya Swati; Batra Harish; Modi Priyanka; Sengupta Sagar; Pandey Monica; Raghavan Sathees C; Saha Dhurjhoti</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1016</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mutations in BLM in Bloom Syndrome patients predispose them to multiple types of cancers.  Here we report that BLM is recruited in a biphasic manner to annotated DSBs.  BLM recruitment is dependent on the presence of NBS1, MRE11 and ATM.  While ATM activity is essential for BLM recruitment in early phase, it is dispensable in late phase when MRE11 exonuclease activity and RNF8-mediated ubiquitylation of BLM are the key determinants.  Interaction between polyubiquitylated BLM and NBS1 is essential for the helicase to be retained at the DSBs.  The helicase activity of BLM is required for the recruitment of HR and c-NHEJ factors onto the chromatin in S- and G1-phase, respectively.  During the repair phase, BLM inhibits HR in S-phase and c-NHEJ in G1-phase.  Consequently, inhibition of helicase activity of BLM enhances the rate of DNA alterations.  Thus BLM utilizes its pro- and anti-repair functions to maintain genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAACZdsi8QiZI8NvIb08rbfW6udTcc2eZJ9TQWCY0oorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrpsF2hug%253D%253D&md5=c537c155acda024a05e7e6ef91c61698</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-03393-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-03393-8%26sid%3Dliteratum%253Aachs%26aulast%3DTripathi%26aufirst%3DV.%26aulast%3DAgarwal%26aufirst%3DH.%26aulast%3DPriya%26aufirst%3DS.%26aulast%3DBatra%26aufirst%3DH.%26aulast%3DModi%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DM.%26aulast%3DSaha%26aufirst%3DD.%26aulast%3DRaghavan%26aufirst%3DS.%2BC.%26aulast%3DSengupta%26aufirst%3DS.%26atitle%3DMRN%2520complex-dependent%2520recruitment%2520of%2520ubiquitylated%2520BLM%2520helicase%2520to%2520DSBs%2520negatively%2520regulates%2520DNA%2520repair%2520pathways%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1016%26doi%3D10.1038%2Fs41467-018-03393-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Honma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuraba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takashima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, M.</span></span> <span> </span><span class="NLM_article-title">Non-homologous end-joining for repairing I-SceI-induced DNA double strand breaks in human cells</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2007.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17296333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFOkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=781-788&author=M.+Honmaauthor=M.+Sakurabaauthor=T.+Koizumiauthor=Y.+Takashimaauthor=H.+Sakamotoauthor=M.+Hayashi&title=Non-homologous+end-joining+for+repairing+I-SceI-induced+DNA+double+strand+breaks+in+human+cells&doi=10.1016%2Fj.dnarep.2007.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Non-homologous end-joining for repairing I-SceI-induced DNA double strand breaks in human cells</span></div><div class="casAuthors">Honma, Masamitsu; Sakuraba, Mayumi; Koizumi, Tomoko; Takashima, Yoshio; Sakamoto, Hiroko; Hayashi, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA double strand breaks (DSBs) are usually repaired through either non-homologous end-joining (NHEJ) or homologous recombination (HR).  While HR is basically error-free repair, NHEJ is a mutagenic pathway that leads to deletion.  NHEJ must be precisely regulated to maintain genomic integrity.  To clarify the role of NHEJ, we investigated the genetic consequences of NHEJ repair of DSBs in human cells.  Human lymphoblastoid cell lines TSCE5 and TSCE105 have, resp., single and double I-SceI endonuclease sites in the endogenous thymidine kinase gene (TK) located on chromosome 17q.  I-SceI expression generated DSBs at the TK gene.  We used the novel transfection system (Amaxa Nucleofector) to introduce an I-SceI expression vector into the cells and randomly isolated clones.  We found mutations involved in the DSBs in the TK gene in 3% of TSCE5 cells and 30% of TSCE105 cell clones.  Most of the mutations in TSCE5 were small (1-30 bp) deletions with a 0-4 bp microhomol. at the junction.  The others consisted of large (>60) bp deletions, an insertion, and a rearrangement.  Mutants resulting from interallelic HR also occurred, but infrequently.  Most of the mutations in TSCE105, on the other hand, were deletions that encompassed the two I-SceI sites generated by NHEJ at DSBs.  The sequence joint was similar to that found in TSCE5 mutants.  Interestingly, some mutants formed a new I-SceI site by perfectly joining the two original I-SceI sites without deletion of the broken-ends.  These results support the idea that NHEJ for repairing I-SceI-induced DSBs mainly results in small or no deletions.  Thus, NHEJ must help maintain genomic integrity in mammalian cells by repairing DSBs as well as by preventing many deleterious alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzF7gA3vX_fLVg90H21EOLACvtfcHk0lgff0FoiPlmgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFOkt78%253D&md5=504da114588a8d895b8b308f8386c442</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DM.%26aulast%3DSakuraba%26aufirst%3DM.%26aulast%3DKoizumi%26aufirst%3DT.%26aulast%3DTakashima%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DH.%26aulast%3DHayashi%26aufirst%3DM.%26atitle%3DNon-homologous%2520end-joining%2520for%2520repairing%2520I-SceI-induced%2520DNA%2520double%2520strand%2520breaks%2520in%2520human%2520cells%26jtitle%3DDNA%2520Repair%26date%3D2007%26volume%3D6%26spage%3D781%26epage%3D788%26doi%3D10.1016%2Fj.dnarep.2007.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scully, R.</span></span> <span> </span><span class="NLM_article-title">BRCA1, BRCA2, and Rad51 operate in a common DNA damage response pathway</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1756</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1999&pages=1752-1756&author=J.+J.+Chenauthor=D.+Silverauthor=S.+Cantorauthor=D.+M.+Livingstonauthor=R.+Scully&title=BRCA1%2C+BRCA2%2C+and+Rad51+operate+in+a+common+DNA+damage+response+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DSilver%26aufirst%3DD.%26aulast%3DCantor%26aufirst%3DS.%26aulast%3DLivingston%26aufirst%3DD.%2BM.%26aulast%3DScully%26aufirst%3DR.%26atitle%3DBRCA1%252C%2520BRCA2%252C%2520and%2520Rad51%2520operate%2520in%2520a%2520common%2520DNA%2520damage%2520response%2520pathway%26jtitle%3DCancer%2520Res.%26date%3D1999%26volume%3D59%26spage%3D1752%26epage%3D1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaina, B.</span></span> <span> </span><span class="NLM_article-title">DNA damage and the balance between survival and death in cancer biology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrc.2015.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fnrc.2015.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26678314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVWqtrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=20-33&author=W.+P.+Roosauthor=A.+D.+Thomasauthor=B.+Kaina&title=DNA+damage+and+the+balance+between+survival+and+death+in+cancer+biology&doi=10.1038%2Fnrc.2015.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage and the balance between survival and death in cancer biology</span></div><div class="casAuthors">Roos, Wynand P.; Thomas, Adam D.; Kaina, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-33</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA is vulnerable to damage resulting from endogenous metabolites, environmental and dietary carcinogens, some anti-inflammatory drugs, and genotoxic cancer therapeutics.  Cells respond to DNA damage by activating complex signalling networks that decide cell fate, promoting not only DNA repair and survival but also cell death.  The decision between cell survival and death following DNA damage rests on factors that are involved in DNA damage recognition, and DNA repair and damage tolerance, as well as on factors involved in the activation of apoptosis, necrosis, autophagy and senescence.  The pathways that dictate cell fate are entwined and have key roles in cancer initiation and progression.  Furthermore, they det. the outcome of cancer therapy with genotoxic drugs.  Understanding the mol. basis of these pathways is important not only for gaining insight into carcinogenesis, but also in promoting successful cancer therapy.  In this Review, we describe key decision-making nodes in the complex interplay between cell survival and death following DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp63s4k3-gcbVg90H21EOLACvtfcHk0lgff0FoiPlmgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVWqtrvO&md5=443e30f3b7b6ff733eeda166e703b7ef</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2015.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2015.2%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DW.%2BP.%26aulast%3DThomas%26aufirst%3DA.%2BD.%26aulast%3DKaina%26aufirst%3DB.%26atitle%3DDNA%2520damage%2520and%2520the%2520balance%2520between%2520survival%2520and%2520death%2520in%2520cancer%2520biology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D20%26epage%3D33%26doi%3D10.1038%2Fnrc.2015.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">RecQ helicase BLM regulates prostate cancer cell proliferation and apoptosis</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4206</span>– <span class="NLM_lpage">4212</span>, <span class="refDoi"> DOI: 10.3892/ol.2017.6704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.3892%2Fol.2017.6704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=28943928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhvFSlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=4206-4212&author=X.+Qianauthor=S.+Fengauthor=D.+Xieauthor=D.+Fengauthor=Y.+Jiangauthor=X.+Zhang&title=RecQ+helicase+BLM+regulates+prostate+cancer+cell+proliferation+and+apoptosis&doi=10.3892%2Fol.2017.6704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RecQ helicase BLM regulates prostate cancer cell proliferation and apoptosis</span></div><div class="casAuthors">Qian Xiaosong; Feng Sujuan; Xie Dawei; Feng Dalin; Jiang Yihang; Zhang Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">4206-4212</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">Prostate cancer (PCa) is a common malignant tumor and the second leading cause of morbidity and mortality in men worldwide.  Considering the prevalence and effects of PCa in males, an understanding of the molecular mechanisms underlying PCa tumorigenesis are essential and may provide novel therapeutic strategies for treating PCa.  Bloom syndrome protein (BLM) is a member of the RecQ helicase family.  The major function of BLM is to uncoil the double-stranded DNA structure.  It has previously been demonstrated that BLM acts as a 'genome caretaker', and dysregulation of BLM function has been implicated in the development of multiple tumor types; however, its potential for inducing PCa tumorigenesis remains undetermined.  The present study aimed to explore the function of BLM in PCa progression.  Reverse transcription-polymerase chain reaction, immunohistochemistry and western blot analyses were performed to detect the BLM expression pattern in PCa patients and cell lines.  The proliferation, and migration and invasion capacities of prostate cells were determined by EdU and Transwell assays following transfection with BLM-targeting short hairpin RNA (shRNA).  The expression of BLM was significantly increased in PCa tissues and PC3 cells compared with non-PCa tissues and benign prostatic hyperplasia cells.  Knockdown of BLM via shRNA inhibited PCa cell proliferation, and promoted PCa cell apoptosis.  Notably, reducing the expression of BLM had no effect on the migration or invasive abilities of PCa cells.  These results suggest that downregulation of BLM may alleviate PCa development, providing a novel perspective for PCa tumorigenesis and a potential therapeutic target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR02yiRS-GsPyM8MEzWGjS9fW6udTcc2eZz-8IzhcjnBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhvFSlsA%253D%253D&md5=0535a20b3078a465b668fad530888d28</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6704%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DRecQ%2520helicase%2520BLM%2520regulates%2520prostate%2520cancer%2520cell%2520proliferation%2520and%2520apoptosis%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D4206%26epage%3D4212%26doi%3D10.3892%2Fol.2017.6704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sears, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooney, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin-Sinex, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchi, J. J.</span></span> <span> </span><span class="NLM_article-title">DNA damage response (DDR) pathway engagement in cisplatin radiosensitization of non-small cell lung cancer</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2016.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1016%2Fj.dnarep.2016.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=26991853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=35-46&author=C.+R.+Searsauthor=S.+A.+Cooneyauthor=H.+Chin-Sinexauthor=M.+S.+Mendoncaauthor=J.+J.+Turchi&title=DNA+damage+response+%28DDR%29+pathway+engagement+in+cisplatin+radiosensitization+of+non-small+cell+lung+cancer&doi=10.1016%2Fj.dnarep.2016.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage response (DDR) pathway engagement in cisplatin radiosensitization of non-small cell lung cancer</span></div><div class="casAuthors">Sears, Catherine R.; Cooney, Sean A.; Chin-Sinex, Helen; Mendonca, Marc S.; Turchi, John J.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-46</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-small cell lung cancers (NSCLC) are commonly treated with a platinum-based chemotherapy such as cisplatin (CDDP) in combination with ionizing radiation (IR).  Although clin. trials have demonstrated that the combination of CDDP and IR appear to be synergistic in terms of therapeutic efficacy, the mechanism of synergism remains largely uncharacterized.  We investigated the role of the DNA damage response (DDR) in CDDP radiosensitization using two NSCLC cell lines.  Using clonogenic survival assays, we detd. that the cooperative cytotoxicity of CDDP and IR treatment is sequence dependent, requiring administration of CDDP prior to IR (CDDP-IR).  We identified and interrogated the unique time and agent-dependent activation of the DDR in NSCLC cells treated with cisplatin-IR combination therapy.  Compared to treatment with CDDP or IR alone, CDDP-IR combination treatment led to persistence of γH2Ax foci, a marker of DNA double-strand breaks (DSB), for up to 24 h after treatment.  Interestingly, pharmacol. inhibition of DDR sensor kinases revealed the persistence of γ-H2Ax foci in CDDP-IR treated cells is independent of kinase activation.  Taken together, our data suggest that delayed repair of DSBs in NSCLC cells treated with CDDP-IR contributes to CDDP radiosensitization and that alterations of the DDR pathways by inhibition of specific DDR kinases can augment CDDP-IR cytotoxicity by a complementary mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTraxlo6nDrVg90H21EOLACvtfcHk0lj8wQpqZBi-bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWgt7g%253D&md5=95038974165090ad89ad757d278b75ce</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2016.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2016.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DSears%26aufirst%3DC.%2BR.%26aulast%3DCooney%26aufirst%3DS.%2BA.%26aulast%3DChin-Sinex%26aufirst%3DH.%26aulast%3DMendonca%26aufirst%3DM.%2BS.%26aulast%3DTurchi%26aufirst%3DJ.%2BJ.%26atitle%3DDNA%2520damage%2520response%2520%2528DDR%2529%2520pathway%2520engagement%2520in%2520cisplatin%2520radiosensitization%2520of%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DDNA%2520Repair%26date%3D2016%26volume%3D40%26spage%3D35%26epage%3D46%26doi%3D10.1016%2Fj.dnarep.2016.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagelbaum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenberg, E.</span></span> <span> </span><span class="NLM_article-title">Mechanistic insight into the interaction of BLM helicase with intra-strand G-quadruplex structures</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5556</span>, <span class="refDoi"> DOI: 10.1038/ncomms6556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1038%2Fncomms6556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25418155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVCis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5556&author=S.+Chatterjeeauthor=J.+Zagelbaumauthor=P.+Savitskyauthor=A.+Sturzeneggerauthor=D.+Huttnerauthor=P.+Janscakauthor=I.+D.+Hicksonauthor=O.+Gileadiauthor=E.+Rothenberg&title=Mechanistic+insight+into+the+interaction+of+BLM+helicase+with+intra-strand+G-quadruplex+structures&doi=10.1038%2Fncomms6556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into the interaction of BLM helicase with intra-strand G-quadruplex structures</span></div><div class="casAuthors">Chatterjee, Sujoy; Zagelbaum, Jennifer; Savitsky, Pavel; Sturzenegger, Andreas; Huttner, Diana; Janscak, Pavel; Hickson, Ian D.; Gileadi, Opher; Rothenberg, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5556</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bloom syndrome is an autosomal recessive disorder caused by mutations in the RecQ family helicase BLM that is assocd. with growth retardation and predisposition to cancer.  BLM helicase has a high specificity for non-canonical G-quadruplex (G4) DNA structures, which are formed by G-rich DNA strands and play an important role in the maintenance of genomic integrity.  Here we used single-mol. FRET to define the mechanism of interaction of BLM helicase with intra-stranded G4 structures.  We show that the activity of BLM is substrate dependent, and highly regulated by a short-strand DNA (ssDNA) segment that separates the G4 motif from double-stranded DNA (dsDNA).  We demonstrate cooperativity between the RQC (RecQ C-terminal) and HRDC (helicase and RNaseD C-terminal) domains of BLM during binding and unfolding of the G4 structure, where the RQC domain interaction with G4 is stabilized by HRDC binding to ssDNA.  We present a model that proposes a unique role for G4 structures in modulating the activity of DNA processing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzxOggEhZ6XbVg90H21EOLACvtfcHk0lj8wQpqZBi-bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVCis7k%253D&md5=d855358965757181f48cf7d1c9e6e535</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms6556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6556%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DZagelbaum%26aufirst%3DJ.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DSturzenegger%26aufirst%3DA.%26aulast%3DHuttner%26aufirst%3DD.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DRothenberg%26aufirst%3DE.%26atitle%3DMechanistic%2520insight%2520into%2520the%2520interaction%2520of%2520BLM%2520helicase%2520with%2520intra-strand%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5556%26doi%3D10.1038%2Fncomms6556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerston, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bizard, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özer, Ö.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarlós, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Bloom’s syndrome helicase indicates a role for the HRDC domain in conformational changes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">5221</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1093%2Fnar%2Fgkv373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=25901030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFait7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=5221-5235&author=J.+A.+Newmanauthor=P.+Savitskyauthor=C.+K.+Allerstonauthor=A.+H.+Bizardauthor=%C3%96.+%C3%96zerauthor=K.+Sarl%C3%B3sauthor=Y.+Liuauthor=E.+Pardonauthor=J.+Steyaertauthor=I.+D.+Hicksonauthor=O.+Gileadi&title=Crystal+structure+of+the+Bloom%E2%80%99s+syndrome+helicase+indicates+a+role+for+the+HRDC+domain+in+conformational+changes&doi=10.1093%2Fnar%2Fgkv373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Bloom's syndrome helicase indicates a role for the HRDC domain in conformational changes</span></div><div class="casAuthors">Newman, Joseph A.; Savitsky, Pavel; Allerston, Charles K.; Bizard, Anna H.; Ozer, Ozgun; Sarlos, Kata; Liu, Ying; Pardon, Els; Steyaert, Jan; Hickson, Ian D.; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5221-5235</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bloom syndrome helicase (BLM) is a member of the RecQ family of DNA helicases, which play key roles in the maintenance of genome integrity in all organism groups.  Here, the authors describe crystal structures of the BLM helicase domain in complex with DNA and with an antibody fragment, as well as SAXS and domain assocn. studies in soln.  The authors showed an unexpected nucleotide-dependent interaction of the core helicase domain with the conserved, poorly characterized HRDC domain.  The BLM-DNA complex showed an unusual base-flipping mechanism with unique positioning of the DNA duplex relative to the helicase core domains.  Comparison with other crystal structures of RecQ helicases permitted the definition of structural transitions underlying ATP-driven helicase action, and the identification of a nucleotide-regulated tunnel that may play a role in interactions with complex DNA substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHuEtr4ILMbrVg90H21EOLACvtfcHk0lj8wQpqZBi-bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFait7zP&md5=3b4c02568175564af56d1cc44891dd42</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv373%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAllerston%26aufirst%3DC.%2BK.%26aulast%3DBizard%26aufirst%3DA.%2BH.%26aulast%3D%25C3%2596zer%26aufirst%3D%25C3%2596.%26aulast%3DSarl%25C3%25B3s%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Bloom%25E2%2580%2599s%2520syndrome%2520helicase%2520indicates%2520a%2520role%2520for%2520the%2520HRDC%2520domain%2520in%2520conformational%2520changes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D5221%26epage%3D5235%26doi%3D10.1093%2Fnar%2Fgkv373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">The dynamic multisite interactions between two intrinsically disordered proteins</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7515</span>– <span class="NLM_lpage">7519</span>, <span class="refDoi"> DOI: 10.1002/anie.201701883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fanie.201701883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot12isLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=7515-7519&author=S.+Wuauthor=D.+Wangauthor=J.+Liuauthor=Y.+Fengauthor=J.+Wengauthor=Y.+Liauthor=X.+Gaoauthor=J.+Liuauthor=W.+Wang&title=The+dynamic+multisite+interactions+between+two+intrinsically+disordered+proteins&doi=10.1002%2Fanie.201701883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic multisite interactions between two intrinsically disordered proteins</span></div><div class="casAuthors">Wu, Shaowen; Wang, Dongdong; Liu, Jin; Feng, Yitao; Weng, Jingwei; Li, Yu; Gao, Xin; Liu, Jianwei; Wang, Wenning</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7515-7519</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein interactions involving intrinsically disordered proteins (IDPs) comprise a variety of binding modes, from the well-characterized folding upon binding to dynamic fuzzy complexes.  To date, most studies concern the binding of an IDP to a structured protein, while the interaction between two IDPs is poorly understood.  In this study, NMR, smFRET, and mol. dynamics (MD) simulation are combined to characterize the interaction between two IDPs, the C-terminal domain (CTD) of protein 4.1G and the nuclear mitotic app. (NuMA) protein.  It is revealed that CTD and NuMA form a fuzzy complex with remaining structural disorder.  Multiple binding sites on both proteins were identified by mol. dynamics and mutagenesis studies.  This study provides an at. scenario in which two IDPs bearing multiple binding sites interact with each other in dynamic equil.  The combined approach employed here could be widely applicable for investigating IDPs and their dynamic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtk7C2PKKDibVg90H21EOLACvtfcHk0lg6oOJ0bqw2KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot12isLg%253D&md5=a4e9878f24ffa2bc17cd987642b42219</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701883%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DThe%2520dynamic%2520multisite%2520interactions%2520between%2520two%2520intrinsically%2520disordered%2520proteins%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D7515%26epage%3D7519%26doi%3D10.1002%2Fanie.201701883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottiger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaffen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leutwyler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachorz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klopper, W.</span></span> <span> </span><span class="NLM_article-title">Strong N–H···π Hydrogen Bonding in Amide–Benzene Interactions</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2937</span>– <span class="NLM_lpage">2943</span>, <span class="refDoi"> DOI: 10.1021/jp8110474</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp8110474" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2937-2943&author=P.+Ottigerauthor=C.+Pfaffenauthor=R.+Leistauthor=S.+Leutwylerauthor=R.+A.+Bachorzauthor=W.+Klopper&title=Strong+N%E2%80%93H%C2%B7%C2%B7%C2%B7%CF%80+Hydrogen+Bonding+in+Amide%E2%80%93Benzene+Interactions&doi=10.1021%2Fjp8110474"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjp8110474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp8110474%26sid%3Dliteratum%253Aachs%26aulast%3DOttiger%26aufirst%3DP.%26aulast%3DPfaffen%26aufirst%3DC.%26aulast%3DLeist%26aufirst%3DR.%26aulast%3DLeutwyler%26aufirst%3DS.%26aulast%3DBachorz%26aufirst%3DR.%2BA.%26aulast%3DKlopper%26aufirst%3DW.%26atitle%3DStrong%2520N%25E2%2580%2593H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580%2520Hydrogen%2520Bonding%2520in%2520Amide%25E2%2580%2593Benzene%2520Interactions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D2937%26epage%3D2943%26doi%3D10.1021%2Fjp8110474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlakos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendorf, J. R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arif, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tizzard, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliev, A. E.</span></span> <span> </span><span class="NLM_article-title">Noncovalent Interactions of π Systems with Sulfur: The Atomic Chameleon of Molecular Recognition</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1002/anie.201708485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fanie.201708485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Cqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1193-1198&author=W.+B.+Motherwellauthor=R.+B.+Morenoauthor=I.+Pavlakosauthor=J.+R.+T.+Arendorfauthor=T.+Arifauthor=G.+J.+Tizzardauthor=S.+J.+Colesauthor=A.+E.+Aliev&title=Noncovalent+Interactions+of+%CF%80+Systems+with+Sulfur%3A+The+Atomic+Chameleon+of+Molecular+Recognition&doi=10.1002%2Fanie.201708485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Noncovalent Interactions of π Systems with Sulfur: The Atomic Chameleon of Molecular Recognition</span></div><div class="casAuthors">Motherwell, William B.; Moreno, Rafael B.; Pavlakos, Ilias; Arendorf, Josephine R. T.; Arif, Tanzeel; Tizzard, Graham J.; Coles, Simon J.; Aliev, Abil E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1198</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The relative strength of noncovalent interactions between a thioether sulfur atom and various π systems in designed top pan mol. balances was detd. by NMR spectroscopy.  Compared to its oxygen counterpart, the sulfur atom displays a remarkable ability to interact with almost equal facility over the entire range of π systems studied, with the simple alkene emerging as the most powerful partner.  With the exception of the O···heteroarene interaction, all noncovalent interactions of sulfur with π systems are favored over oxygen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsh-5TchNJm7Vg90H21EOLACvtfcHk0lg6oOJ0bqw2KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Cqug%253D%253D&md5=8811fc6c0721c31c36e4d07d875f1f8b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.201708485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201708485%26sid%3Dliteratum%253Aachs%26aulast%3DMotherwell%26aufirst%3DW.%2BB.%26aulast%3DMoreno%26aufirst%3DR.%2BB.%26aulast%3DPavlakos%26aufirst%3DI.%26aulast%3DArendorf%26aufirst%3DJ.%2BR.%2BT.%26aulast%3DArif%26aufirst%3DT.%26aulast%3DTizzard%26aufirst%3DG.%2BJ.%26aulast%3DColes%26aufirst%3DS.%2BJ.%26aulast%3DAliev%26aufirst%3DA.%2BE.%26atitle%3DNoncovalent%2520Interactions%2520of%2520%25CF%2580%2520Systems%2520with%2520Sulfur%253A%2520The%2520Atomic%2520Chameleon%2520of%2520Molecular%2520Recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1193%26epage%3D1198%26doi%3D10.1002%2Fanie.201708485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span> <span> </span><span class="NLM_article-title">G-quadruplex hinders translocation of BLM helicase on DNA: a real-time fluorescence spectroscopic unwinding study and comparison with duplex substrates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">10521</span>– <span class="NLM_lpage">10527</span>, <span class="refDoi"> DOI: 10.1021/ja1038165</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1038165" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10521-10527&author=J.-q.+Liuauthor=C.-y.+Chenauthor=Y.+Xueauthor=Y.-h.+Haoauthor=Z.+Tan&title=G-quadruplex+hinders+translocation+of+BLM+helicase+on+DNA%3A+a+real-time+fluorescence+spectroscopic+unwinding+study+and+comparison+with+duplex+substrates&doi=10.1021%2Fja1038165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja1038165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1038165%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.-q.%26aulast%3DChen%26aufirst%3DC.-y.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DY.-h.%26aulast%3DTan%26aufirst%3DZ.%26atitle%3DG-quadruplex%2520hinders%2520translocation%2520of%2520BLM%2520helicase%2520on%2520DNA%253A%2520a%2520real-time%2520fluorescence%2520spectroscopic%2520unwinding%2520study%2520and%2520comparison%2520with%2520duplex%2520substrates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10521%26epage%3D10527%26doi%3D10.1021%2Fja1038165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchán, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svozil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of α/γ Conformers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">3817</span>– <span class="NLM_lpage">3829</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.097782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1529%2Fbiophysj.106.097782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=17351000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2007&pages=3817-3829&author=A.+P%C3%A9rezauthor=I.+March%C3%A1nauthor=D.+Svozilauthor=J.+Sponerauthor=T.+E.+Cheathamauthor=C.+A.+Laughtonauthor=M.+Orozco&title=Refinement+of+the+AMBER+Force+Field+for+Nucleic+Acids%3A+Improving+the+Description+of+%CE%B1%2F%CE%B3+Conformers&doi=10.1529%2Fbiophysj.106.097782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of the AMBER force field for nucleic acids: improving the description of α/γ conformers</span></div><div class="casAuthors">Perez, Alberto; Marchan, Ivan; Svozil, Daniel; Sponer, Jiri; Cheatham, Thomas E., III; Laughton, Charles A.; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3817-3829</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">We present here the parmbsc0 force field, a refinement of the AMBER parm99 force field, where emphasis has been made on the correct representation of the α/γ concerted rotation in nucleic acids (NAs).  The modified force field corrects overpopulations of the α/γ = (g+,t) backbone that were seen in long (more than 10 ns) simulations with previous AMBER parameter sets (parm94-99).  The force field has been derived by fitting to high-level quantum mech. data and verified by comparison with very high-level quantum mech. calcns. and by a very extensive comparison between simulations and exptl. data.  The set of validation simulations includes two of the longest trajectories published to date for the DNA duplex (200 ns each) and the largest variety of NA structures studied to date (15 different NA families and 97 individual structures).  The total simulation time used to validate the force field includes near 1 μs of state-of-the-art mol. dynamics simulations in aq. soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriZsH_9T_yB7Vg90H21EOLACvtfcHk0lg6oOJ0bqw2KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D&md5=e0f8d87e5fe7710e59cc41f5e8c25f7b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.097782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.097782%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DI.%26aulast%3DSvozil%26aufirst%3DD.%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DRefinement%2520of%2520the%2520AMBER%2520Force%2520Field%2520for%2520Nucleic%2520Acids%253A%2520Improving%2520the%2520Description%2520of%2520%25CE%25B1%252F%25CE%25B3%2520Conformers%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D3817%26epage%3D3829%26doi%3D10.1529%2Fbiophysj.106.097782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span> <span> </span><span class="NLM_article-title">Tracking the dynamic folding and unfolding of RNA G-Quadruplexes in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">4792</span>– <span class="NLM_lpage">4796</span>, <span class="refDoi"> DOI: 10.1002/ange.201801999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=10.1002%2Fange.201801999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2018&pages=4792-4796&author=X.-C.+Chenauthor=S.-B.+Chenauthor=J.+Daiauthor=J.-H.+Yuanauthor=T.-M.+Ouauthor=Z.-S.+Huangauthor=J.-H.+Tan&title=Tracking+the+dynamic+folding+and+unfolding+of+RNA+G-Quadruplexes+in+live+cells&doi=10.1002%2Fange.201801999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fange.201801999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201801999%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.-C.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.-H.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DHuang%26aufirst%3DZ.-S.%26aulast%3DTan%26aufirst%3DJ.-H.%26atitle%3DTracking%2520the%2520dynamic%2520folding%2520and%2520unfolding%2520of%2520RNA%2520G-Quadruplexes%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D130%26spage%3D4792%26epage%3D4796%26doi%3D10.1002%2Fange.201801999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Cheatham, T. E.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Crowley, M.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Zhang, W.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Hayik, S.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Wong, K. F.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Vanicek, J.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Steinbrecher, T.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Yang, L.</span>; <span class="NLM_string-name">Tan, C.</span>; <span class="NLM_string-name">Mongan, J.</span>; <span class="NLM_string-name">Hornak, V.</span>; <span class="NLM_string-name">Cui, G.</span>; <span class="NLM_string-name">Mathews, D. H.</span>; <span class="NLM_string-name">Seetin, M. G.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 10</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>, <span class="NLM_year">2008</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=M.+Crowley&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Wang&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=L.+Yang&author=C.+Tan&author=J.+Mongan&author=V.+Hornak&author=G.+Cui&author=D.+H.+Mathews&author=M.+G.+Seetin&author=C.+Sagui&author=V.+Babin&author=P.+A.+Kollman&title=AMBER+10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252010%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the Molecular Mechanics Poisson–Boltzmann Surface Area Free Energy Method Using a Congeneric Series of Ligands to p38 MAP Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">7796</span>– <span class="NLM_lpage">7807</span>, <span class="refDoi"> DOI: 10.1021/jm050306m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050306m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFynurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7796-7807&author=D.+A.+Pearlman&title=Evaluating+the+Molecular+Mechanics+Poisson%E2%80%93Boltzmann+Surface+Area+Free+Energy+Method+Using+a+Congeneric+Series+of+Ligands+to+p38+MAP+Kinase&doi=10.1021%2Fjm050306m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the molecular mechanics Poisson-Boltzmann surface area free energy method using a congeneric series of ligands to p38 MAP kinase</span></div><div class="casAuthors">Pearlman, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7796-7807</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently described mol. mechanics Poisson-Boltzmann surface area (MM-PBSA) method for calcg. free energies is applied to a congeneric series of 16 ligands to p38 MAP kinase whose binding consts. span approx. 2 orders of magnitude.  These compds. have previously been used to test and compare other free energy calcn. methods, including thermodn. integration (TI), OWFEG, ChemScore, PLPScore, and Dock Energy Score.  We find that the MM-PBSA performs relatively poorly for this set of ligands, yielding results much inferior to those from TI or OWFEG, inferior to Dock Energy Score, and not appreciably better than ChemScore or PLPScore but at an appreciably larger computational cost than any of these other methods.  This suggests that one should be selective in applying the MM-PBSA method and that for systems that are amenable to other free energy approaches, these other approaches may be preferred.  We also examine the single simulation approxn. for MM-PBSA, whereby the required ligand and protein trajectories are extd. from a single MD simulation rather than two sep. MD runs.  This assumption, sometimes used to speed the MM-PBSA calcn., is found to yield significantly inferior results with only a moderate net percentage redn. in total simulation time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOT67YZ9ZNj7Vg90H21EOLACvtfcHk0lg6kVKUpgwXRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFynurfL&md5=4a5169f9f3ead9723e080bc16ca43272</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm050306m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050306m%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26atitle%3DEvaluating%2520the%2520Molecular%2520Mechanics%2520Poisson%25E2%2580%2593Boltzmann%2520Surface%2520Area%2520Free%2520Energy%2520Method%2520Using%2520a%2520Congeneric%2520Series%2520of%2520Ligands%2520to%2520p38%2520MAP%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D7796%26epage%3D7807%26doi%3D10.1021%2Fjm050306m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ','PDB','4CGZ'); return false;">PDB: 4CGZ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i70"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00083">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_46883"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00083">10.1021/acs.jmedchem.9b00083</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional experimental results, <sup>1</sup>H NMR, <sup>13</sup>C NMR, HPLC and HRMS spectra for final compounds, molecular formula strings and coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf">PDF</a>)</p></li><li><p class="inline">We provide the molecular information for a model (<b>29</b>/<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>) in this study. The authors will release the atomic coordinates and experimental data upon article publication (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_002.csv">CSV</a>)</p></li><li><p class="inline">Protein Data Bank, crystal structure of BLM, and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CGZ">4CGZ</a>(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_001.pdf">jm9b00083_si_001.pdf (11.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_002.csv">jm9b00083_si_002.csv (5.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00083/suppl_file/jm9b00083_si_003.pdb">jm9b00083_si_003.pdb (693.02 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-6%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00083%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00083" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d26c7b4d3c10","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
