
<html lang="en"     class="pb-page"  data-request-id="5122a1de-34c5-44ba-80b2-993bddb20f07"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02274;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Hinge Binder Scaffold Hopping Identifies Potent Calcium/Calmodulin-Dependent Protein Kinase Kinase 2 (CAMKK2) Inhibitor Chemotypes" /></meta><meta name="dc.Creator" content="Benjamin J.  Eduful" /></meta><meta name="dc.Creator" content="Sean N.  O’Byrne" /></meta><meta name="dc.Creator" content="Louisa  Temme" /></meta><meta name="dc.Creator" content="Christopher R. M.  Asquith" /></meta><meta name="dc.Creator" content="Yi  Liang" /></meta><meta name="dc.Creator" content="Alfredo  Picado" /></meta><meta name="dc.Creator" content="Joseph R.  Pilotte" /></meta><meta name="dc.Creator" content="Mohammad Anwar  Hossain" /></meta><meta name="dc.Creator" content="Carrow I.  Wells" /></meta><meta name="dc.Creator" content="William J.  Zuercher" /></meta><meta name="dc.Creator" content="Carolina M. C.  Catta-Preta" /></meta><meta name="dc.Creator" content="Priscila  Zonzini Ramos" /></meta><meta name="dc.Creator" content="André de S.  Santiago" /></meta><meta name="dc.Creator" content="Rafael M.  Couñago" /></meta><meta name="dc.Creator" content="Christopher G.  Langendorf" /></meta><meta name="dc.Creator" content="Kévin  Nay" /></meta><meta name="dc.Creator" content="Jonathan S.  Oakhill" /></meta><meta name="dc.Creator" content="Thomas L.  Pulliam" /></meta><meta name="dc.Creator" content="Chenchu  Lin" /></meta><meta name="dc.Creator" content="Dominik  Awad" /></meta><meta name="dc.Creator" content="Timothy M.  Willson" /></meta><meta name="dc.Creator" content="Daniel E.  Frigo" /></meta><meta name="dc.Creator" content="John W.  Scott" /></meta><meta name="dc.Creator" content="David H.  Drewry" /></meta><meta name="dc.Description" content="CAMKK2 is a serine/threonine kinase and an activator of AMPK whose dysregulation is linked with multiple diseases. Unfortunately, STO-609, the tool inhibitor commonly used to probe CAMKK2 signaling..." /></meta><meta name="Description" content="CAMKK2 is a serine/threonine kinase and an activator of AMPK whose dysregulation is linked with multiple diseases. Unfortunately, STO-609, the tool inhibitor commonly used to probe CAMKK2 signaling..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 15, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02274" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02274" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02274" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02274" /></link>
        
    
    

<title>Hinge Binder Scaffold Hopping Identifies Potent Calcium/Calmodulin-Dependent Protein Kinase Kinase 2 (CAMKK2) Inhibitor Chemotypes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02274" /></meta><meta property="og:title" content="Hinge Binder Scaffold Hopping Identifies Potent Calcium/Calmodulin-Dependent Protein Kinase Kinase 2 (CAMKK2) Inhibitor Chemotypes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0026.jpeg" /></meta><meta property="og:description" content="CAMKK2 is a serine/threonine kinase and an activator of AMPK whose dysregulation is linked with multiple diseases. Unfortunately, STO-609, the tool inhibitor commonly used to probe CAMKK2 signaling, has limitations. To identify promising scaffolds as starting points for the development of high-quality CAMKK2 chemical probes, we utilized a hinge-binding scaffold hopping strategy to design new CAMKK2 inhibitors. Starting from the potent but promiscuous disubstituted 7-azaindole GSK650934, a total of 32 compounds, composed of single-ring, 5,6-, and 6,6-fused heteroaromatic cores, were synthesized. The compound set was specifically designed to probe interactions with the kinase hinge-binding residues. Compared to GSK650394 and STO-609, 13 compounds displayed similar or better CAMKK2 inhibitory potency in vitro, while compounds 13g and 45 had improved selectivity for CAMKK2 across the kinome. Our systematic survey of hinge-binding chemotypes identified several potent and selective inhibitors of CAMKK2 to serve as starting points for medicinal chemistry programs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02274"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02274">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02274&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02274&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02274&amp;href=/doi/10.1021/acs.jmedchem.0c02274" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.0c02178" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00781" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Hinge Binder Scaffold Hopping Identifies Potent Calcium/Calmodulin-Dependent Protein Kinase Kinase 2 (CAMKK2) Inhibitor Chemotypes</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Benjamin J. Eduful</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin J. Eduful</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin+J.++Eduful">Benjamin J. Eduful</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sean N. O’Byrne</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sean N. O’Byrne</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sean+N.++O%E2%80%99Byrne">Sean N. O’Byrne</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Louisa Temme</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Louisa Temme</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Louisa++Temme">Louisa Temme</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher R. M. Asquith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher R. M. Asquith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div><div class="loa-info-affiliations-info">Department of Pharmacology, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+R.+M.++Asquith">Christopher R. M. Asquith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5871-3458" title="Orcid link">https://orcid.org/0000-0001-5871-3458</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi Liang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi Liang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Liang">Yi Liang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alfredo Picado</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alfredo Picado</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alfredo++Picado">Alfredo Picado</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph R. Pilotte</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph R. Pilotte</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+R.++Pilotte">Joseph R. Pilotte</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohammad Anwar Hossain</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohammad Anwar Hossain</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad+Anwar++Hossain">Mohammad Anwar Hossain</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8684-8755" title="Orcid link">https://orcid.org/0000-0001-8684-8755</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carrow I. Wells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carrow I. Wells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carrow+I.++Wells">Carrow I. Wells</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4799-6792" title="Orcid link">https://orcid.org/0000-0003-4799-6792</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William J. Zuercher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William J. Zuercher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+J.++Zuercher">William J. Zuercher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carolina M. C. Catta-Preta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carolina M. C. Catta-Preta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Química Medicinal (CQMED), Centro de Biologia Molecular e Engenharia Genética (CBMEG), Universidade Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</div><div class="loa-info-affiliations-info">Structural Genomics Consortium, Departamento de Genética e Evolução, Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carolina+M.+C.++Catta-Preta">Carolina M. C. Catta-Preta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Priscila Zonzini Ramos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Priscila Zonzini Ramos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Química Medicinal (CQMED), Centro de Biologia Molecular e Engenharia Genética (CBMEG), Universidade Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</div><div class="loa-info-affiliations-info">Structural Genomics Consortium, Departamento de Genética e Evolução, Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Priscila++Zonzini+Ramos">Priscila Zonzini Ramos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">André de S. Santiago</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">André de S. Santiago</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Química Medicinal (CQMED), Centro de Biologia Molecular e Engenharia Genética (CBMEG), Universidade Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</div><div class="loa-info-affiliations-info">Structural Genomics Consortium, Departamento de Genética e Evolução, Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andr%C3%A9+de+S.++Santiago">André de S. Santiago</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rafael M. Couñago</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rafael M. Couñago</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Química Medicinal (CQMED), Centro de Biologia Molecular e Engenharia Genética (CBMEG), Universidade Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</div><div class="loa-info-affiliations-info">Structural Genomics Consortium, Departamento de Genética e Evolução, Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rafael+M.++Cou%C3%B1ago">Rafael M. Couñago</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher G. Langendorf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher G. Langendorf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">St Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+G.++Langendorf">Christopher G. Langendorf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kévin Nay</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kévin Nay</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">St Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</div><div class="loa-info-affiliations-info">Mary MacKillop Institute for Health Research, Australian Catholic University, 215 Spring Street, Melbourne 3000, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=K%C3%A9vin++Nay">Kévin Nay</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jonathan S. Oakhill</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan S. Oakhill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">St Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</div><div class="loa-info-affiliations-info">Mary MacKillop Institute for Health Research, Australian Catholic University, 215 Spring Street, Melbourne 3000, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+S.++Oakhill">Jonathan S. Oakhill</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas L. Pulliam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas L. Pulliam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</div><div class="loa-info-affiliations-info">Center for Nuclear Receptors and Cell Signaling, University of Houston, Houston, Texas 77204, United States</div><div class="loa-info-affiliations-info">Department of Biology and Biochemistry, University of Houston, Houston, Texas 77204, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+L.++Pulliam">Thomas L. Pulliam</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chenchu Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chenchu Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</div><div class="loa-info-affiliations-info">The University of Texas MD Anderson Cancer Center UTHealth Graduate School of Biomedical Sciences, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chenchu++Lin">Chenchu Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominik Awad</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominik Awad</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</div><div class="loa-info-affiliations-info">The University of Texas MD Anderson Cancer Center UTHealth Graduate School of Biomedical Sciences, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominik++Awad">Dominik Awad</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Timothy M. Willson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Timothy M. Willson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Timothy+M.++Willson">Timothy M. Willson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4181-8223" title="Orcid link">https://orcid.org/0000-0003-4181-8223</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel E. Frigo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel E. Frigo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</div><div class="loa-info-affiliations-info">Center for Nuclear Receptors and Cell Signaling, University of Houston, Houston, Texas 77204, United States</div><div class="loa-info-affiliations-info">Department of Biology and Biochemistry, University of Houston, Houston, Texas 77204, United States</div><div class="loa-info-affiliations-info">Department of Genitourinary Medical Oncology, University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div><div class="loa-info-affiliations-info">The Methodist Hospital Research Institute, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+E.++Frigo">Daniel E. Frigo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0713-471X" title="Orcid link">https://orcid.org/0000-0002-0713-471X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John W. Scott</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John W. Scott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">St Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</div><div class="loa-info-affiliations-info">Mary MacKillop Institute for Health Research, Australian Catholic University, 215 Spring Street, Melbourne 3000, Australia</div><div class="loa-info-affiliations-info">The Florey Institute of Neuroscience and Mental Health, 30 Royal Parade, Parkville 3052, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3e544d5d514a4a7e4d4857105b5a4b105f4b"><span class="__cf_email__" data-cfemail="b3d9c0d0dcc7c7f3c0c5da9dd6d7c69dd2c6">[email protected]</span></a>. Phone: +61-3-9231-3510.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+W.++Scott">John W. Scott</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">David H. Drewry</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David H. Drewry</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium and Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div><div class="loa-info-affiliations-info">UNC Lineberger Comprehensive Cancer Center, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d0b4b1a6b9b4feb4a2b5a7a2a990a5beb3feb5b4a5"><span class="__cf_email__" data-cfemail="791d180f101d571d0b1c0e0b00390c171a571c1d0c">[email protected]</span></a>. Phone: +1-919-602-1327.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+H.++Drewry">David H. Drewry</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5973-5798" title="Orcid link">https://orcid.org/0000-0001-5973-5798</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02274&amp;href=/doi/10.1021%2Facs.jmedchem.0c02274" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 15, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 December 2020</li><li><span class="item_label"><b>Published</b> online</span>15 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02274" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02274</a></div><div class="article_header-article-copyright"><strong>© 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D29%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBenjamin%2BJ.%2BEduful%252C%2BSean%2BN.%2BO%25E2%2580%2599Byrne%252C%2BLouisa%2BTemme%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.0c02274%26title%3DHinge%2BBinder%2BScaffold%2BHopping%2BIdentifies%2BPotent%2BCalcium%252FCalmodulin-Dependent%2BProtein%2BKinase%2BKinase%2B2%2B%2528CAMKK2%2529%2BInhibitor%2BChemotypes%26numPages%3D29%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D29%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted" /></img><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted" /></img><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02274"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1325</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02274" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Hinge Binder Scaffold Hopping Identifies Potent Calcium/Calmodulin-Dependent Protein Kinase Kinase 2 (CAMKK2) Inhibitor Chemotypes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;J. Eduful&quot;},{&quot;first_name&quot;:&quot;Sean&quot;,&quot;last_name&quot;:&quot;N. O’Byrne&quot;},{&quot;first_name&quot;:&quot;Louisa&quot;,&quot;last_name&quot;:&quot;Temme&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;R. M. Asquith&quot;},{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Alfredo&quot;,&quot;last_name&quot;:&quot;Picado&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;R. Pilotte&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;Anwar Hossain&quot;},{&quot;first_name&quot;:&quot;Carrow&quot;,&quot;last_name&quot;:&quot;I. Wells&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;J. Zuercher&quot;},{&quot;first_name&quot;:&quot;Carolina&quot;,&quot;last_name&quot;:&quot;M. C. Catta-Preta&quot;},{&quot;first_name&quot;:&quot;Priscila&quot;,&quot;last_name&quot;:&quot;Zonzini Ramos&quot;},{&quot;first_name&quot;:&quot;André&quot;,&quot;last_name&quot;:&quot;de S. Santiago&quot;},{&quot;first_name&quot;:&quot;Rafael&quot;,&quot;last_name&quot;:&quot;M. Couñago&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;G. Langendorf&quot;},{&quot;first_name&quot;:&quot;Kévin&quot;,&quot;last_name&quot;:&quot;Nay&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;S. Oakhill&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;L. Pulliam&quot;},{&quot;first_name&quot;:&quot;Chenchu&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Dominik&quot;,&quot;last_name&quot;:&quot;Awad&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;M. Willson&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;E. Frigo&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;W. Scott&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;H. Drewry&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02274&quot;},&quot;abstract&quot;:&quot;CAMKK2 is a serine/threonine kinase and an activator of AMPK whose dysregulation is linked with multiple diseases. Unfortunately, STO-609, the tool inhibitor commonly used to probe CAMKK2 signaling, has limitations. To identify promising scaffolds as starting points for the development of high-quality CAMKK2 chemical probes, we utilized a hinge-binding scaffold hopping strategy to design new CAMKK2 inhibitors. Starting from the potent but promiscuous disubstituted 7-azaindole GSK650934, a total of 32 compounds, composed of single-ring, 5,6-, and 6,6-fused heteroaromatic cores, were synthesized. The compound set was specifically designed to probe interactions with the kinase hinge-binding residues. Compared to GSK650394 and STO-609, 13 compounds displayed similar or better CAMKK2 inhibitory potency in vitro, while compounds 13g and 45 had improved selectivity for CAMKK2 across the kinome. Our systematic survey of hinge-binding chemotypes identified several potent and selective inhibitors of CAMKK2 to serve as &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02274&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02274" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02274&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02274" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02274&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02274" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02274&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02274&amp;href=/doi/10.1021/acs.jmedchem.0c02274" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02274" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02274" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02274%26sid%3Dliteratum%253Aachs%26pmid%3D34264658%26genre%3Darticle%26aulast%3DEduful%26date%3D2021%26atitle%3DHinge%2BBinder%2BScaffold%2BHopping%2BIdentifies%2BPotent%2BCalcium%252FCalmodulin-Dependent%2BProtein%2BKinase%2BKinase%2B2%2B%2528CAMKK2%2529%2BInhibitor%2BChemotypes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291241" title="Phenyls">Phenyls</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">CAMKK2 is a serine/threonine kinase and an activator of AMPK whose dysregulation is linked with multiple diseases. Unfortunately, STO-609, the tool inhibitor commonly used to probe CAMKK2 signaling, has limitations. To identify promising scaffolds as starting points for the development of high-quality CAMKK2 chemical probes, we utilized a hinge-binding scaffold hopping strategy to design new CAMKK2 inhibitors. Starting from the potent but promiscuous disubstituted 7-azaindole GSK650934, a total of 32 compounds, composed of single-ring, 5,6-, and 6,6-fused heteroaromatic cores, were synthesized. The compound set was specifically designed to probe interactions with the kinase hinge-binding residues. Compared to GSK650394 and STO-609, 13 compounds displayed similar or better CAMKK2 inhibitory potency <i>in vitro</i>, while compounds <b>13g</b> and <b>45</b> had improved selectivity for CAMKK2 across the kinome. Our systematic survey of hinge-binding chemotypes identified several potent and selective inhibitors of CAMKK2 to serve as starting points for medicinal chemistry programs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Calcium (Ca<sup>2+</sup>)/calmodulin-dependent protein kinase kinase 2 (CAMKK2) is a serine/threonine kinase that is one of the calmodulin (CaM)-binding proteins of the CaMK family.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Ca<sup>2+</sup> is an important second messenger that aids in the regulation of a wide range of signaling events in part through binding to its intracellular receptor CaM.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Ca<sup>2+</sup>-bound CaM modulates various cellular responses <i>via</i> activation of an array of downstream proteins including CAMKK2.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Upon activation, CAMKK2 phosphorylates and activates its substrates including CAMK1, CAMK4, AMP-activated protein kinase, (AMPK) and in some cases AKT. This signal transduction results in the regulation of many important physiological and pathological processes.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16">(6−16)</a></div><div class="NLM_p">Due to CAMKK2’s important role in cell signaling, dysregulation of CAMKK2 has been implicated in several diseases. Aberrant activation and overexpression of CAMKK2 have been linked to multiple cancer types including prostate, breast, ovarian, gastric, and hepatic cancers.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref17 ref18">(8−14,17,18)</a> Knockdown of CAMKK2 <i>via</i> siRNA or pharmacological inhibition of CAMKK2 reduced cell proliferation, migration, and invasion as well as induced apoptosis and cell cycle arrest in numerous cancer cell lines and tumor models.<a onclick="showRef(event, 'ref8 ref12 ref13 ref14 ref17 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref8 ref12 ref13 ref14 ref17 ref19 ref20 ref21">(8,12−14,17,19−21)</a> Mechanistically, CAMKK2 is an important regulator of metabolic and inflammatory processes, which are contributory factors to its impact on cancer growth.<a onclick="showRef(event, 'ref9 ref14'); return false;" href="javascript:void(0);" class="ref ref9 ref14">(9,14)</a></div><div class="NLM_p">In addition, therapeutic interventions targeting CAMKK2 may be beneficial beyond oncology. For example, hepatocellular carcinoma (HCC) is often preceded by non-alcoholic fatty liver disease (NAFLD), which is driven by several risk factors including obesity and type 2 diabetes.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a> Inhibition of CAMKK2 reduced food intake in mice, and <i>Camkk2</i>-null mice are leaner than wild-type mice when fed a high-fat diet.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Consistent with these findings, CaMKK2 was recently shown to be inhibited by liraglutide, a glucagon-like peptide-1 receptor agonist that decreases food intake and promotes weight loss.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In relation to skeletal disease, CAMKK2 is expressed in osteoblasts and osteoclasts, which play an essential role in bone tissue maintenance.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Inhibition of CAMKK2 stimulated bone formation and reversed age-associated decline in bone health by promoting osteoblast differentiation and inhibiting osteoclast differentiation.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Taken together, these findings suggest that inhibition of CAMKK2 may be effective for the treatment of a variety of diseases including various types of cancers, metabolic disorders, and bone diseases like osteoporosis. While CAMKK2 has emerged as an attractive therapeutic target, there remains a shortage of high-quality CAMKK2 inhibitors, which has impaired progress in the field.</div><div class="NLM_p">At present, almost all CAMKK2-related pharmacological studies rely on the use of the ATP-competitive inhibitor STO-609 to study CAMKK2 signaling events (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref8 ref11 ref17 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref8 ref11 ref17 ref28 ref29 ref30 ref31 ref32">(8,11,17,28−32)</a> However, STO-609 is not an ideal chemical tool.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> It binds to multiple kinases beyond CAMKK2. When screened against a panel of over 400 wild-type human kinases (KINOME<i>scan</i>, Eurofins DiscoverX) at 1 μM, STO-609 bound with strong affinity to 14 kinases (<20% activity remaining).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Among these kinases were CDKL2, GRK3, and CK2, which are all overexpressed in several cancer types.<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34−36)</a> In addition to these collateral kinase targets, STO-609 potently activates the aryl hydrocarbon receptor.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> STO-609 is a planar molecule with poor aqueous solubility, yet it is routinely used at high doses in cell culture (>10 μM) and <i>in vivo</i>.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Due to the liabilities of STO-609, the field would benefit from the discovery and development of potent and highly selective small-molecule inhibitors of CAMKK2 as high-quality probes.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known CAMKK2 inhibitors: STO-609 and GSK650394.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We recently conducted an extensive literature search for CAMKK2 inhibitors to identify starting points for medicinal chemistry optimization.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> A promising series of potent CAMKK2 inhibitors were recently disclosed by GlaxoSmithKline (GSK).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The 7-azaindole GSK650394 (also called a pyrrolopyridine) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was a potent CAMKK2 inhibitor (IC<sub>50</sub> = 0.63 nM). The co-crystal structure of GSK650394 with CAMKK2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>) revealed the critical role of the carboxylic acid and cyclopentyl moiety (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a), which was consistent with the reported structure–activity relationship (SAR) studies.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The co-crystal structure reveals that the nitrogen atoms of the pyridine and pyrrole rings act as hydrogen bond acceptor and donor pairs, respectively, and form interactions with the protein backbone of the ATP-binding site. The acid functionality contributes to critical hydrogen bonding interactions with both the protonated amine of Lys194 and the carboxylate group of Glu236 in a water-mediated manner.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Additionally, CH−π interactions (aromatic edge–face interactions) between the aromatic ring of gatekeeper Phe267 with both the CH in the 2-position of the 7-azaindole and the methine group in the ortho-position of the 3-aryl moiety of the inhibitor stabilize GSK650394 in this orientation. The pendant phenyl group in the 5-position of the 7-azaindole scaffold occupies a hydrophobic pocket which can potentially be exploited to gain selectivity and potency and modulate physical properties to optimize pharmacokinetic parameters. Although a crystal structure of CAMKK2 co-crystallized with ATP is not available, the cyclopentyl group appears to mimic at least the position of the ribosyl moiety of ATP based on the proximity of the cyclopentyl group to the P-loop of CAMKK2. The 7-azaindole pharmacophore in GSK650394 is commonly found in kinase inhibitors and is considered a powerful but sometimes promiscuous hinge binder.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43">(39−43)</a> When screened against a panel of 334 kinases (Reaction Biology Corporation), GSK650394 inhibited 29 kinases by more than 90%, limiting its utility as a tool compound for studying CAMKK2 or any other kinase.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Binding mode of the co-crystallized ligand GSK650394 in the ATP-binding site of CAMKK2 (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>). The image was generated with PyMOL. The protein is colored in gray. Blue-dashed lines indicate H-bond interactions; green-dashed lines display CH−π interactions. GSK650394 is shown as purple sticks and the water molecule as a red sphere. Oxygen and nitrogen atoms are colored in red and blue, respectively. (b) Structure of GSK650394 highlighting important sites for design of new CAMKK2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To improve the kinase selectivity of GSK650394, we targeted replacement of the 7-azaindole with other heterocycles that would modulate interactions with the hinge-binding residues and in some cases reduce interactions with the hinge-binding residues. Our compound design strategy is depicted in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b. Our central hypothesis was based on the belief that the 3-position aryl ring, decorated with the carboxylic acid and cyclopentyl groups, plays a dominant role in CAMKK2 recognition and that inhibitors with enhanced selectivity would arise from compounds with the binding contribution from the hinge binder diminished. This led to the synthesis and evaluation of structurally diverse novel chemotypes as potential inhibitors of CAMKK2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. New molecules based on scaffold hopping from GSK650394 designed to interrogate several aspects of hinge binding interactions between CAMKK2 and the proposed inhibitors. Top row: Inhibitors retaining the topology of GSK650394 (5,6-fused ring systems with the cyclopentyl-substituted benzoic acid moiety attached to the 5-membered ring). Middle row: 5,6-fused systems with the cyclopentyl-substituted benzoic acid moiety appended to the 6-membered ring. Bottom row: Inhibitors with ring expansion (6,6-fused ring system) or ring deletion (single 6-membered ring as the hinge-binding core).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Thus, we performed a scaffold-hopping exercise wherein the hinge-binding 7-azaindole core of GSK650394 was substituted with structurally diverse alternate hinge-binding moieties. We retained the <i>ortho</i>-cyclopentyl benzoic acid moiety hypothesizing that the interactions observed in the crystal structure would bias the new set toward CAMKK2 affinity.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58056" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58056" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">To develop analogues of GSK650394 that could be potent and selective CAMKK2 inhibitor scaffolds, all our synthesized compounds incorporated the critical pharmacophore, <i>ortho</i>-cyclopentyl benzoic acid. In parallel, to investigate the importance of the pendant phenyl ring, we synthesized matched pairs with and without this group. We theorized that removal of the phenyl ring could significantly lower the cLog <i>P</i> and increase ligand efficiency (LE) if it was dispensable without being detrimental to potency. To this end, we first synthesized pinacol borate esters <b>3</b> and <b>4</b> to enable attachment to the various hinge-binding pharmacophores <i>via</i> Suzuki coupling reactions (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). 4-Bromo-2-fluorobenzoic acid <b>1</b> underwent a Grignard reaction with cyclopentylmagnesium bromide to afford the <i>ortho</i>-cyclopentyl-substituted benzoic acid <b>2</b>, which was then converted into the corresponding pinacol boronate ester using Miyaura borylation conditions yielding <b>3</b>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The analogous methyl ester <b>4</b> was made in a similar fashion following esterification of <b>2</b> in methanol in the presence of thionyl chloride.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pinacol Boronate Esters <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) cyclopentylmagnesium bromide, THF, −10 to 25 °C, 5 h, then aq HCl; (b) PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, KOAc, bis(pinacolato)diboron, 100 °C, 2 h; (c) SOCl<sub>2</sub>, MeOH, 16 h, 75 °C.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> 3- and 3,5-Substituted Fused 6,5-Ring Systems</h3><div class="NLM_p">Initial analogues were fused 5,6-ring systems with 3- and 3,5- substitution patterns that retained the topology of GSK650394 but modify the nature and/or location of heteroatoms in the ring system and thus can alter the compound’s ability to form H-bond interactions with the hinge region. GSK650394 is commercially available, but its matched pair <b>7</b>, truncated at the 5-position, required synthesis (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). 3-Bromo-7-azaindole <b>5</b> was converted to the silylethoxymethyl (SEM)-protected azaindole and then subjected to Suzuki–Miyaura conditions to afford cyclopentyl analogue <b>6</b>. SEM deprotection, followed by base-mediated saponification, afforded the target azaindole <b>7</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Azaindole <b>7</b>, the Matched Pair for GSK650394<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SEM-Cl, NaH, DMF, 0–21 °C, 2 h; (b) <b>4</b>, Pd(OAc)<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, P(Cy)<sub>3</sub>, PhMe/H<sub>2</sub>O, 80 °C, 16 h; (c) CF<sub>3</sub>COOH, CH<sub>2</sub>Cl<sub>2</sub>, then NaOAc, EtOH, 21 °C, 24 h; (d) aq NaOH, MeOH, 75 °C, 1 h, then aq HCl.</p></p></figure><div class="NLM_p">The <i>N</i>-methyl analogues of GSK650394 and <b>7</b> were synthesized as depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. To access <b>10</b>, 5-chloro-3-iodo azaindole <b>8b</b> was <i>N</i>-alkylated.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Consecutive Suzuki reactions with the <i>ortho</i>-cyclopentyl methyl benzoate <b>4</b> and then phenyl boronic acid placed the key pharmacophore in the 3-position and a phenyl ring in the 5-position. Saponification of the methyl ester proceeded smoothly affording azaindole <b>10</b>. Coupling of boronate ester <b>4</b> with 3-bromo-<i>N</i>-methyl azaindole, followed by saponification of the methyl ester, afforded target molecule <b>11</b>.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <i>N</i>-Methyl Azaindole Hinge Binders <b>10</b> and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; (b) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (c) <b>4</b>, PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, rt, 16 h; (d) PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><div class="NLM_p">The synthesis of the structurally similar furopyridine and thienopyridine cores is described in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Nicotinic acid derivatives <b>12a</b> and <b>13a</b> were converted into ethyl esters <b>12–14b</b> and then treated with ethyl glycolate or ethyl thioglycolate in the presence of sodium hydride (NaH) to give the respective furopyridines or thienopyridines <b>12–14c</b>.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> A one-pot saponification–decarboxylation of the β-keto esters afforded the 2-unsubstituted heterocycles <b>12–14d</b>. These aryl alcohols were converted to the corresponding triflates <b>12–14d</b>, which were able to undergo Suzuki–Miyaura reactions to yield analogues <b>12f</b>, <b>13g</b>, and <b>14g</b>.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a><b>12g</b> was obtained directly from the commercially available 3-bromothieno[2,3-<i>b</i>]pyridine <b>15e</b> using Suzuki–Miyaura reaction.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Route to the Furo- and Thienopyridines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (EtO)<sub>3</sub>CH, PhMe, 100 °C, 2 h or H<sub>2</sub>SO<sub>4</sub>, EtOH, reflux, 24 h; (b) ethyl glycolate or ethylthioglycolate, NaH, DME, 0–75 °C, 2.5 h; (c) aq NaOH, EtOH then aq HCl, 100 °C, 2 h; (d) Tf<sub>2</sub>O, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 3 h; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h; (f) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h or PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O 90 °C, 16 h; (g) aq NaOH, MeOH then aq HCl, 1 h; (h) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h.</p></p></figure><div class="NLM_p">The subsequent set of fused 5,6-ring derivatives synthesized in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a> (pyrazolopyridines and pyrazolopyrimidines) dramatically altered the placement of the heteroatoms based on the parent compound. Analogues <b>19</b> and <b>20</b> were obtained by stepwise Suzuki couplings, first with <b>4</b> and then with phenylboronic acid, followed by methyl ester hydrolysis. The monosubstituted complementary matched pairs were synthesized from commercially available 3-bromopyrazolo[1,5-<i>a</i>]pyridine or -pyrimidine <i>via</i> Suzuki couplings with <b>4</b> to afford <b>21</b> and <b>22</b>. The same set of reaction conditions was used to afford the triazolopyridazine analogues <b>25</b> and <b>26</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Pyrazolopyridine and Pyrazolopyrimidine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; <b>17</b><i>via</i> (b) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; <b>18</b><i>via</i> (c) PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 80 °C, 16 h; (d) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 30 min, μW; (e) aq LiOH, dioxane, 100 °C, 16 h then aq HCl.</p></p></figure><figure id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of the Triazolopyridazine Analogues <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (b) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (c) Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (d) aq NaOH, MeOH 75 °C, 1 h, then aq HCl.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> 2- and 2,4-Substituted Fused 5,6-Ring Systems</h3><div class="NLM_p">The first scaffolds in this category were <i>N</i>-methyl azaindoles and are described in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. The disubstituted azaindole <b>29</b> was obtained from coupling phenylboronic acid at the more reactive 2-position of 2-iodo-4-chloro-<i>N</i>-methyl azaindole, followed by coupling with intermediate <b>4</b> at the 4-position. Saponification of the methyl ester afforded acid <b>29</b>. The monosubstituted compound <b>30</b> was synthesized from commercially available 4-chloro-<i>N</i>-methyl-7-azaindole with a Suzuki coupling reaction, followed by saponification of the methyl ester.</div><figure id="sch7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of 2,4-Substituted <i>N</i>-Methyl Azaindoles<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (b) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (c) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (d) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><div class="NLM_p">Imidazopyridine analogues were obtained from initial coupling of 2,3-diamino-4-chloropyridine with <b>4</b> to afford the aryl-substituted pyridine <b>32</b> (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). Imidazopyridine <b>33</b> was synthesized by condensation of diamine <b>32</b> with trimethyl orthoformate in methanol, followed by saponification of the ester to afford the desired carboxylic acid. The 2-aryl analogue <b>34</b> was obtained by converting benzoic acid into an activated ester with 1,1′-carbonyldiimidazole (CDI) and addition of diamine <b>32</b> to form a putative amide intermediate that underwent a ring closure under thermal conditions resulting in the imidazopyridine intermediate. Ester hydrolysis yielded the desired imidazopyridine <b>34</b>.</div><figure id="sch8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of the Imidazopyridine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 100 °C, 16 h; (b) CH(OMe)<sub>3</sub>, H<sub>3</sub>NSO<sub>3</sub>, MeOH, rt, 1 h; (c) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (d) benzoic acid, CDI, DMF, 0 °C, 30 min, then <b>32</b>, 200 °C, 10 min; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><div class="NLM_p">Mono- and disubstituted thienopyrimidine analogues of the [3,2-<i>d</i>] and [2,3-<i>d</i>] core scaffolds were synthesized from commercially available starting materials (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>). The monosubstituted analogues <b>38</b> and <b>42</b> were easily accessed in one step <i>via</i> Suzuki coupling using commercially available <b>35a</b> and <b>39a</b>, respectively. <b>37</b> was synthesized by first coupling <b>35b</b> with phenylboronic acid to afford 2-aryl-substituted intermediate <b>36</b>, which was then coupled with <b>3</b> to yield the disubstituted thieno[3,2-<i>d</i>]pyrimidine. The reaction sequence was reversed in the synthesis of the disubstituted thieno[2,3-<i>d</i>]pyrimidine <b>41</b> since the chlorine substituent in <b>39b</b> was found to be more reactive than the thiophene bromine. Intermediate <b>4</b> was coupled to the 4-position of <b>39b</b> affording <b>40</b>. Subsequent Suzuki coupling with phenylboronic acid, followed by saponification of the ester, yielded disubstituted analogue <b>41</b>.</div><figure id="sch9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Thieno[3,2-<i>d</i>]pyrimidine and Thieno[2,3-<i>d</i>]pyrimidine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, μW, 15 min; (b) PhB(OH)<sub>2</sub>, NaHCO<sub>3</sub>, CsF, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O, 95 °C, 3 h; (c) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, 15 min; (d) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, μW, 15 min; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 145 °C, μW, 15 min; (f) PhB(OH)<sub>2</sub>, NaHCO<sub>3</sub>, CsF, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O, 80 °C, 4 h; (g) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis of Substituted Fused 6,6-Ring Hinge Binders</h3><div class="NLM_p">This set of hinge binders moves from 5,6-fused ring systems to 6,6-fused systems. The disubstituted quinoline analogue <b>45</b> was synthesized in a straightforward manner from commercially available 4,6-dichloroquinoline <b>43b</b><i>via</i> sequential Suzuki couplings with <b>4</b> and phenylboronic acid, followed by saponification of the methyl ester. The monosubstituted quinoline <b>46</b> was synthesized in one step <i>via</i> Suzuki coupling between the commercially available <b>43a</b> and <b>3</b>.</div><div class="NLM_p">The quinazoline hinge binder derivatives were obtained via similar routes utilized in the synthesis of the quinolines (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>) with the exception of compound <b>49</b>. The disubstituted quinazoline <b>49</b> was prepared from 4-chloroquinazolin-6-ol <b>47b</b> with initial installation of boronate <b>4</b> and subsequent conversion of the hydroxy group of <b>48</b> into a triflate which readily underwent Suzuki coupling with phenylboronic acid. Saponification then afforded the target compound. 4-Aryl-2-methylquinoline <b>52</b> and 1,6-naphthyridine <b>54</b> analogues were synthesized in one step <i>via</i> Suzuki coupling with <b>3</b> (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>c).</div><figure id="sch10" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Quinoline and Quinazoline Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (b) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, rt, 16 h; (c) PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, 120 °C, 16 h; (d) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 16 h; (f) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (g) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (h) (i) DIPEA, Tf<sub>2</sub>O, CH<sub>2</sub>Cl<sub>2</sub>, 0–21 °C, 2 h, (ii) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH, 90 °C, 1 h; (i) aq NaOH, MeOH, 75 °C, 1 h, then aq HCl; (j) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H2O, 90 °C, 16 h.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Pyrimidine Hinge Binders</h3><div class="NLM_p">The final set of hinge binders we explored consisted of substituted pyrimidines. The monosubstituted pyrimidine <b>56</b> was obtained <i>via</i> Suzuki coupling of <b>55</b> with <b>3</b> (<a class="ref internalNav" href="#sch11" aria-label="Scheme 11">Scheme 11</a>a). 2-Phenyl-4-aryl pyrimidine <b>58</b> was synthesized from 2-phenyl-4-chloropyrimidine <b>57</b> by Suzuki coupling with <b>3</b>. The synthesis of aminopyrimidines <b>61</b> and <b>62</b> is described in <a class="ref internalNav" href="#sch11" aria-label="Scheme 11">Scheme 11</a>b. Starting from commercially available 4-chloropyrimidin-2-amine <b>59</b>, Suzuki coupling with <b>4</b> afforded <b>60</b>. Saponification of the resulting methyl ester afforded <b>61</b>. <b>60</b> was utilized in a Buchwald–Hartwig coupling with bromobenzene, and subsequent ester hydrolysis yielded 2-phenylaminopyrimidine <b>62</b>.</div><figure id="sch11" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0020.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis of Pyrimidine and Aminopyrimidine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,2-DME, H<sub>2</sub>O, 120 °C, 0.5 h. (b) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,2-DME, H<sub>2</sub>O, 120 °C, 0.5 h; (c) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (d) PhBr, Pd<sub>2</sub>(dba)<sub>3</sub>, XantPhos, NaO<sup><i>t</i></sup>Bu, PhMe, 80 °C, 19 h.</p></p></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Assays Used for <i>In Vitro</i> Compound Affinity Evaluation</h3><div id="sec2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> DSF Assay</h4><div class="NLM_p last">We used a thermal-shift assay (differential scanning fluorimetry, DSF) to detect protein–ligand interactions.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> For many proteins, including kinases, this is a useful high-throughput method to identify compounds that bind to a protein of interest, and in many cases, the melting temperature correlates with binding affinity. CAMKK1 and CAMKK2 are ∼60% identical at the amino acid sequence level.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> We measured DSF Δ<i>T</i><sub>m</sub> for all analogues against both CAMKK2 and CAMKK1 using the purified kinase domains of these two proteins. As in the end we did not utilize the DSF data to drive the project, all DSF data are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>In Vitro</i> CAMKK1 and CAMKK2 Enzyme Inhibition Assays</h4><div class="NLM_p last">We used purified recombinant CAMKK1 and CAMKK2 in an assay that measured the transfer of radiolabeled phosphate from [γ-<sup>32</sup>P]-ATP to a synthetic CAMKK2 substrate (CAMKKtide). Initially, we evaluated CAMKK2 inhibitory activity of all synthesized compounds at three different concentrations (10, 100, and 1000 nM) to rank compounds and provide preliminary dose response information. In the percent of control (PoC) experiments, the assay with no inhibitor present serves as the control. Half-maximal inhibitory concentrations (IC<sub>50</sub>) were then determined for analogues with PoC values <10 at the 1 μM screening concentration. For a subset of CAMKK2 inhibitors, we also determined half-maximal inhibitory concentration (IC<sub>50</sub>) values. STO-609 and GSK650394 were routinely used as reference compounds in the assay to ensure that the assay performed correctly and CAMKK2 inhibition was observed.</div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>In Vitro</i> Compound Screening Results</h3><div class="NLM_p">The initial set of compounds evaluated was focused on those with similar topology to GSK650394. Data in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> depict compounds, the enzyme CAMKK2 inhibition data (PoC) at three concentrations, and CAMKK2 enzyme inhibition IC<sub>50</sub> data for selected compounds. Although we collected thermal shift (DSF) data for our inhibitors with both CAMKK1 and CAMKK2, we found that they did not provide enough information to drive chemistry decisions and did not always correlate with the enzymatic activity. The DSF data are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a> for reference. We thus relied on the CAMKK2 enzyme inhibition data as our primary assay.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. CAMKK2 Enzyme Inhibition Data for Compounds with 3- and 3,5-Substituted Fused 5,6-Ring Systems<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0021.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">R = <i>ortho</i>-cyclopentyl benzoic acid moiety; PoC (percent of control = percent of enzyme activity remaining, when compared with the control); IC<sub>50</sub> (half-maximal inhibitory concentration); NG (not generated); IC<sub>50</sub> values were generated in an 8-point full dose response assay.</p></div></div><div></div></div><div class="NLM_p">Removal of the phenyl group from the 5-position of GSK650394 led to a roughly 3-fold decrease in enzyme inhibitory activity (IC<sub>50</sub> = 26 vs 3 nM), implying that the phenyl group or an appropriately adorned phenyl group may be useful to enhance CAMKK2 enzyme affinity and/or activity. Similar pairs of analogues for the different hinge binders allowed for direct comparison (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">The goal of this project was to discover alternate scaffolds for CAMKK2 with sufficient potency and suitable kinome selectivity that can be used as starting points for medicinal chemistry optimization programs. We hypothesized that carefully modulating the hydrogen bonding interactions at the hinge-binding region <i>via</i> a variety of heterocyclic scaffolds could identify a new series that maintained CAMKK2 affinity and improved selectivity profiles. Kinase inhibitors with multiple hinge-binding interactions can suffer from off-target effects owing to their potential for poor selectivity across the kinome.<a onclick="showRef(event, 'ref39 ref41 ref50'); return false;" href="javascript:void(0);" class="ref ref39 ref41 ref50">(39,41,50)</a> However, careful modification of even promiscuous starting points can in some cases lead to desired levels of selectivity. There are examples demonstrating that modifications to the hydrogen bonding interactions between a kinase inhibitor and the hinge region of the kinase can improve selectivity without severely impacting potency.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a><i>N</i>-Methyl azaindoles <b>10</b> and <b>11</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) offer a particularly drastic change in hinge binding by blocking the H-bonding donor property of the pyrrole ring. This resulted in a complete loss in enzyme inhibition potency <i>in vitro</i>. This suggested that the steric bulk of the <i>N</i>-methyl group is not accommodated within the binding pocket of CAMKK2 when the cyclopentyl benzoic acid moiety is attached at the 3-position of the azaindole scaffold.</div><div class="NLM_p">Further structural modifications to the hinge-binding moiety that maintained the fused 5,6-ring system are outlined here. Compounds <b>13g</b> and <b>12f</b> replaced the pyrrole ring with a furan ring. Compounds <b>14g</b> and <b>12g</b> replaced the pyrrole ring of GSK650394 with a thiophene ring. The pyridine nitrogen that makes a key hydrogen bond with the hinge is maintained, but the NH hydrogen bonding opportunity has been removed. Compounds <b>19</b>, <b>20</b>, <b>21</b>, <b>22</b>, <b>25</b>, and <b>26</b> all maintain the 5,6-fused core but have key differences from GSK650394. They do not have a nitrogen in a position analogous to the nitrogen in the 7-position of GSK650394, thus necessitating different hinge-binding interactions. They also incorporate nitrogen atoms in other locations within the 5,6-ring system.</div><div class="NLM_p">Nearly all the phenyl-substituted analogues of active CAMKK2 inhibitor scaffolds showed greater enzyme potency than their corresponding truncated analogues. An exception was compound <b>19</b> (phenyl version), which had an IC<sub>50</sub> = 145 nM compared to an IC<sub>50</sub> = 44 nM for compound <b>21</b> (truncated version). Replacement of the pyrrole ring in the azaindole with either a furan or thiophene ring retained potency in the phenyl-substituted analogues (<b>13g</b> IC<sub>50</sub> = 65 nM, <b>14g</b> IC<sub>50</sub> = 5 nM), but their corresponding truncated analogues (<b>12f</b>, <b>12g</b>) were relatively inactive. We would like to point out that an analogue of compound <b>13g</b> is listed as a CAMKK2 chemical probe (<a href="https://www.thesgc.org/chemical-probes/SGC-CAMKK2-1" class="extLink">https://www.thesgc.org/chemical-probes/SGC-CAMKK2-1</a>) with the name SGC-CAMKK2-1. The kinome-wide selectivity profiles and cellular potencies of these two close analogues are comparable (<i>vide infra</i> and the SGC probe link above). In addition, SGC-CAMKK2-1 is currently commercially available from Sigma-Aldrich. The results depicted in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> demonstrate that the choice of a hinge binder plays a role in the type of substitutions that are tolerated. Both versions of the triazolopyridazine hinge binder (<b>25</b>, <b>26</b>) showed no activity. We hypothesize that this is due to repulsion between the lone pair on the nitrogen in the 2-position and a backbone protein carbonyl (carbonyl from E268 in CAMKK2). In many kinase/inhibitor co-crystal structures, this carbonyl is engaged in hydrogen bonding with a NH from the inhibitor or at least pointing toward a polarized CH on the inhibitor. In summary, four scaffolds (furopyridine <b>13g</b>, thienopyridine <b>14g</b>, pyrazolopyridine <b>19</b> and <b>21</b>, and pyrazolopyrimidine <b>20</b>) in this set exhibited robust CAMKK2 enzyme inhibition (IC<sub>50</sub> values < 150 nM), with <b>14g</b> (IC<sub>50</sub> = 5 nM) exhibiting comparable enzymatic inhibitory activity to GSK650394 (IC<sub>50</sub> = 3 nM).</div><div class="NLM_p">To further evaluate fused 5,6-ring structures as CAMKK2 inhibitors, we switched from 3,5- to 2,4-ring substitutions as depicted in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. <i>N</i>-methyl azaindoles <b>29</b> and <b>30</b> were the first pair of analogues synthesized. Interestingly, these two <i>N</i>-methyl-substituted azaindole analogues were well tolerated, displaying good CAMKK2 enzyme inhibitory activity (<b>29</b> IC<sub>50</sub> = 120 nM, <b>30</b> IC<sub>50</sub> = 56 nM). This result is in stark contrast to 3,5-<i>N</i>-methyl azaindoles <b>10</b> and <b>11</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The truncated analogue <b>30</b> was significantly more potent than its corresponding phenyl analogue <b>29</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. CAMKK2 Enzyme Inhibition Data for Compounds with 2- and 2,4-Substituted Fused 6,5-Ring Systems<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0022.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">R = <i>ortho</i>-cyclopentyl benzoic acid moiety; PoC (percent of control = percent of enzyme activity remaining, when compared with control); IC<sub>50</sub> (half-maximal inhibitory concentration); IC<sub>50</sub> values were generated in an 8-point full dose response assay.</p></div></div><div></div></div><div class="NLM_p">The 2,4-substituted analogues of <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> are all inhibitors of CAMKK2. Incorporation of an extra nitrogen into the pyrrole ring to create imidazopyridine analogues <b>33</b> and <b>34</b> was well tolerated, with the phenyl-substituted analogue demonstrating higher potency (<b>34</b> IC<sub>50</sub> = 23 nM) than the corresponding non-phenyl version (<b>33</b> IC<sub>50</sub> = 193 nM). Similar to <i>N</i>-methyl azaindole <b>30</b>, non-phenyl-substituted thienopyrimidines (<b>38</b> IC<sub>50</sub> = 24 nM, <b>42</b> IC<sub>50</sub> = 31 nM) were more potent than their corresponding phenyl-substituted analogues (<b>37</b> IC<sub>50</sub> = 169 nM, <b>41</b> IC<sub>50</sub> = 232 nM). Both [3,2-<i>d</i>] analogues <b>37</b> and <b>38</b> (with the sulfur oriented away from the hinge-binding region) and [2,3-<i>d</i>] analogues <b>41</b> and <b>42</b> (with the sulfur oriented toward the hinge) inhibit CAMKK2. For both these isomeric thienopyrimidines, removal of the pendant phenyl actually increases CAMKK2 potency (<b>37</b> with phenyl IC<sub>50</sub> = 169 nM <b>38</b> without phenyl IC<sub>50</sub> = 24 nM; <b>41</b> with phenyl IC<sub>50</sub> = 232 nM <b>42</b> without phenyl IC<sub>50</sub> = 31 nM).</div><div class="NLM_p">Next, we explored replacement of the 5,6-bicyclic cores with 6,6-bicyclic structures. Quinazolines and quinolines are frequently used as kinase hinge-binding heterocycles and were identified as active scaffolds for CAMKK2 inhibition (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Both quinoline <b>45</b>, with a phenyl in the 6-position, and <b>46</b>, unsubstituted in the 6-position, inhibited CAMKK2 (<b>45</b> IC<sub>50</sub> = 137 nM, IC<sub>50</sub><b>46</b> = 13 nM). The potency of the low-molecular-weight compound <b>46</b> suggests that further exploration may be fruitful. Similarly, the two quinazolines <b>49</b> (IC<sub>50</sub> = 12 nM) and <b>50</b> (IC<sub>50</sub> = 96 nM) were also potent CAMKK2 inhibitors.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. CAMKK2 Enzyme Inhibition Data for Compounds with Fused 6,6-Ring Systems<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0023.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">R = <i>ortho</i>-cyclopentyl benzoic acid moiety; PoC (percent of control = percent of enzyme activity remaining, when compared with the control); IC<sub>50</sub> (half-maximal inhibitory concentration); NG (not generated); IC<sub>50</sub> values were generated in an 8-point full dose response assay.</p></div></div><div></div></div><div class="NLM_p">As expected, introduction of a methyl group at the 2-position of <b>52</b> resulted in a complete loss of CAMKK2 enzyme activity. The steric bulk of the 2′ methyl group is not tolerated and likely hinders the ability of the quinoline nitrogen to effectively participate in hydrogen bonding with the hinge region. Naphthyridine <b>54</b> showed poor CAMKK2 enzymatic activity.</div><div class="NLM_p">Finally, we evaluated a small set of pyrimidines, well-known kinase ATP-competitive inhibitor scaffolds, for their CAMKK2 inhibitory activity (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Pyrimidine <b>56</b>, with a phenyl in the 2-position, had a CAMKK2 IC<sub>50</sub> = 21 nM. 2-Anilino-pyrimidine <b>62</b> was also potent with CAMKK2 IC<sub>50</sub> = 51 nM. Pyrimidine <b>58</b>, unsubstituted in the 2-position, lost considerable activity relative to <b>56</b>. 2-Amino-pyrimidine <b>61</b> retained CAMKK2 potency (IC<sub>50</sub> = 108 nM).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. CAMKK2 Enzyme Inhibition Data for Substituted Pyrimidines and 4-Aminopyrimidines<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0024.gif" alt="" id="gr23" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">R = <i>ortho</i>-cyclopentyl benzoic acid moiety; PoC (percent of control = percent of enzyme activity remaining, when compared with the control); IC<sub>50</sub> (half-maximal inhibitory concentration); NG (not generated); IC<sub>50</sub> values were generated in an 8-point full dose response assay.</p></div></div><div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> X-ray Crystallography and <i>In Silico</i> Docking Studies</h3><div class="NLM_p">In order to more fully understand the binding modes and to plan optimization studies on these scaffolds, we turned to X-ray crystallography and in silico docking analysis of key molecules. We previously reported the crystal structure of 7-azaindole GSK650394 bound to CAMKK2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> A crystal structure of the closely related 7-azaindole GSK650393 is also available (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CMJ">6CMJ</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> These two structures clearly demonstrate a hydrogen bond interaction between the NH of the pyrrole in the 7-azaindole to the carbonyl group of Glu268 at the CAMKK2 hinge region. The nitrogen atom at the 7-position forms an interaction with the hinge <i>via</i> the hydrogen bond with the NH of Val270. We were able to obtain a crystal structure of furopyridine <b>13g</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a>). Data collection statistics and crystallization conditions are shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Crystallization Conditions and Data Collection Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center"><b>13g</b></th><th class="colsep0 rowsep0" align="center">UNC10244803</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data collection</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">X-ray source</td><td class="colsep0 rowsep0" align="left">APS 24-ID-C</td><td class="colsep0 rowsep0" align="left">DLS I03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.9791</td><td class="colsep0 rowsep0" align="left">0.9763</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>3</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>3</sub>2<sub>1</sub>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">73.3, 73.3, 122.0</td><td class="colsep0 rowsep0" align="left">73.2, 73.2, 120.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">19.74–1.70 (1.73–1.70)</td><td class="colsep0 rowsep0" align="left">19.61–1.60 (1.63–1.60)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of unique reflections</td><td class="colsep0 rowsep0" align="left">37,374 (1,767)</td><td class="colsep0 rowsep0" align="left">43,864 (3,161)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub> (%)</td><td class="colsep0 rowsep0" align="left">14.7 (183.7)</td><td class="colsep0 rowsep0" align="left">12.2 (31.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>I</i>/σ<i>I</i></td><td class="colsep0 rowsep0" align="left">14.2 (2.0)</td><td class="colsep0 rowsep0" align="left">21.3 (10.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean CC(1/2)</td><td class="colsep0 rowsep0" align="left">1.0 (0.75)</td><td class="colsep0 rowsep0" align="left">1.0 (0.99)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.9 (100)</td><td class="colsep0 rowsep0" align="left">99.9 (100)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">21.7 (22.1)</td><td class="colsep0 rowsep0" align="left">25.8 (25.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">refinement</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">19.75–1.70 (1.75–1.70)</td><td class="colsep0 rowsep0" align="left">19.61–1.60 (1.64–1.60)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>cryst</sub>/<i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">16.0/18.2</td><td class="colsep0 rowsep0" align="left">15.3/17.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of non-hydrogen atoms/mean B-factor (Å)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein atoms</td><td class="colsep0 rowsep0" align="left">2230/28.1</td><td class="colsep0 rowsep0" align="left">2229/21.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent atoms</td><td class="colsep0 rowsep0" align="left">370/44.3</td><td class="colsep0 rowsep0" align="left">338/33.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand atoms</td><td class="colsep0 rowsep0" align="left">29/19.8</td><td class="colsep0 rowsep0" align="left">26/12.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.010</td><td class="colsep0 rowsep0" align="left">0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd bond angles (deg)</td><td class="colsep0 rowsep0" align="left">1.05</td><td class="colsep0 rowsep0" align="left">1.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran statistics (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored</td><td class="colsep0 rowsep0" align="left">98.2</td><td class="colsep0 rowsep0" align="left">98.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDB ID</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crystallization conditions</td><td class="colsep0 rowsep0" align="left">25% PEG 3350, 0.2 M ammonium sulfate, 0.1 M bis–tris buffer, pH 6.5</td><td class="colsep0 rowsep0" align="left">20% PEG 3350, 0.02 M ammonium sulfate, 0.1 M CHC buffer system, pH 8.5</td></tr></tbody></table></div></div><div class="NLM_p">Furopyridine <b>13g</b> displays a similar binding mode to GSK650394. The 5,6-fused ring systems are oriented in the same way in the CAMKK2 active site, and the pyridine moieties of the two heterocycles form comparable hydrogen bond interactions with the NH group of Val270, and the furan oxygen atom and pyrrole NH group are in the same region of space. Although we have no crystal structure of thienopyridine <b>14g</b>, we hypothesize that it would likely bind in a similar fashion.</div><div class="NLM_p">To assess the binding modes of <b>14g</b> and other compounds, we performed in silico docking studies with the docking software Glide version 2014 (Schrödinger).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> For our docking studies, we utilized three of the X-ray crystal structures of CAMKK2 (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>). We based our choice of the CAMKK2 PDB protein structure to use for our modeling on the structural similarity between the co-crystallized ligand and the compound we planned to use in the in silico docking. In order to ensure reliability of the docking protocol and the ensuing results for hypothesis generation, the co-crystallized ligands GSK650394, <b>13g</b>, and UNC10244803 [2-cyclopentyl-4-(7-methoxyquinolin-4-yl)benzoic acid, CAMKK2 enzyme IC<sub>50</sub> = 33 nM] were removed from the binding site and the ligands were re-docked into the binding pocket. The docking poses were then compared with the original pose from the crystal structures. The root-mean-square deviation (rmsd) value was lower than 1 Å, indicating a reliable docking procedure. The predicted binding modes of active compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) show conserved H-bonds or close proximity between the compound’s carboxylate group and both the protonated amine of Lys194 and the carboxylate group of Glu236 in a water-mediated manner, analogous to what was observed in the crystal structures. The overall orientation of the tested compounds and their interactions with the hinge region is also of interest.</div><div class="NLM_p">As expected, the predicted binding mode of thienopyridine <b>14g</b> overlaid almost perfectly with the pose of furopyridine <b>13g</b> in the X-ray structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). The backbone NH group of Val270 and the thienopyridine N-atom are well positioned for a hydrogen bond interaction. Interestingly, there appears to be an additional favorable interaction between the sulfur atom of <b>14g</b> and the carbonyl group of Glu268. Inter- and intra-molecular interactions between sulfur and oxygen atoms are relatively common and can be used to the medicinal chemists’ advantage.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a></div><figure id="fig4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray structures and in silico docking. In silico docking was performed with Glide, and images were generated with PyMOL. The protein is colored in gray. Blue-dashed lines indicate H-bond interactions, while green-dashed lines display CH−π interactions and orange-dashed lines refer to sulfur−σ hole bonding. Docked ligands are shown as yellow sticks or orange sticks, co-crystallized ligands as purple sticks, and the water molecule as a red sphere. Oxygen and nitrogen atoms are colored in red and blue, respectively. (a) Predicted binding mode of <b>14g</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a> compared to the co-crystallized ligand <b>13g</b> (purple). (b) Predicted binding modes of <b>46</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a> and the co-crystallized ligand UNC10244803 (purple). (c) Predicted binding mode of <b>56</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>. (d) Predicted binding mode of <b>22</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>. (e) Predicted binding modes of <b>10</b> (orange) and <b>29</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a> (light gray) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a> (dark gray), respectively. (f) Predicted binding mode of <b>38</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We were also able to obtain a crystal structure of quinoline UNC10244803 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>). The quinoline nitrogen atom is positioned to form a hydrogen bond with the NH group of Val270. The predicted binding pose of <b>46</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b) is highly comparable to the binding mode of the co-crystallized ligand UNC10244803. Similar to quinoline <b>46</b>, 2-phenyl-pyrimidine <b>56</b> has only one heteroatom that is capable of forming hydrogen bonds with the hinge (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). The N-atom of pyridine in the 1-position of <b>56</b> shows a H-bond with the NH group of Val270. Together with the presumed H-bond formed between the protonated N-atom of Lys194 and the carboxylate group of <b>56</b>, these two interactions anchor <b>56</b> in the active site. In this orientation, the phenyl ring in the 2-position is tolerated by the binding pocket. 2-Aryl-pyrimidines are much less common as kinase inhibitors than 2-anilinopyrimidines and so warrant further exploration. The 2-position phenyl ring of <b>56</b> is adjacent to Leu269 of CAMKK2. We speculate that kinases incorporating amino acids with larger side chains such as Phe or Tyr in this location at the hinge region will be less tolerant of this scaffold, perhaps offering a path to enhance selectivity.</div><div class="NLM_p last">Pyrazolopyrimidine <b>22</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d) forms one hinge-binding interaction between the N-atom in position 1 and the backbone NH group of Val270. The distance between the NH<sub>3</sub><sup>+</sup> group of Lys194 and the carboxylate group of <b>22</b> is greater compared to the predicted binding modes of the other compounds (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b,e). The <i>N</i>-methyl group of compound <b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e) appears to preclude binding of this compound in the same orientation as GSK650394. This compound is inactive in the enzyme assay, and the docking predicts a flipped orientation of the compound. <i>N</i>-methyl compound <b>29</b>, however, retains activity, and the in silico docking result suggests a conformation that allows the pyridyl nitrogen atom to interact with the NH group of Val270. The methyl group in this orientation is tolerated, perhaps analogously to the 2-phenyl of compound <b>56</b>. This flipped binding mode is possibly due to the shift of the attachment position of the cyclopentyl benzoic acid moiety from the 5-position of <b>10</b> to the 4-position in <b>29</b>. The hinge-binding interactions shown in the predicted binding mode of thienopyrimidine <b>38</b> are comparable (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>f) to those of <b>56</b>. Additionally, the orientation of <b>38</b> is further stabilized by an interaction that is formed between the O-atom of the carbonyl moiety of Ile171 and the S-atom of the thienopyrimidine scaffold.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Compound Selectivity</h3><div class="NLM_p">In order to begin to understand kinome-wide selectivity for these new compounds, we profiled 9 exemplars in a panel of over 400 wild-type human kinases using the Eurofins DiscoverX’s KINOME<i>scan</i> technology<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (Freemont, CA USA). A summary of the kinome selectivity results is depicted in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, and the complete KINOME<i>scan</i> data sets are available in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>. <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> contains the structures of the compounds profiled, the PoC values for CAMKK1 and CAMKK2 at the 1 μM screening concentration, and a list of all the kinases in the panel that bound with a PoC < 10. The S-score is a quantitative measure of a compound’s selectivity at a particular screening concentration, calculated by dividing the number of kinases that a particular compound binds to at a chosen threshold by the total number of distinct kinases tested. All the compounds we tested from this scaffold-hopping effort have an S<sub>10</sub> (1 μM) < 0.05, which implies that they bind to fewer than 5% of the wild-type kinases tested in this panel with a PoC of 10 or less, thus offering promising selectivity for a starting point for further optimization.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Selectivity Screening Summary Results for Nine Different Exemplars</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0025.gif" alt="" id="gr24" /></img><div></div></div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> is a visual representation of the KINOME<i>scan</i> data for two of the compounds, azaindole compound <b>7</b> and 2-anilino-pyrimidine compound <b>62</b>. All kinases bound with a PoC < 10 are depicted as red dots. These kinases are listed in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, and the full data sets are available in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>. For compound <b>7</b>, 12 kinases have a PoC less than or equal to 10, including CAMKK2 and CAMKK1. For compound <b>7</b>, S<sub>10</sub> (1 μM) = 0.03 implies that 3% of the kinases tested had a PoC < 10 at an inhibitor screening concentration of 1 μM. For compound <b>62</b>, only two kinases (CAMKK1 and CAMKK2) have a PoC < 10. S<sub>10</sub> (1 μM) = 0.005 implies that 0.5% of the kinases tested had a PoC < 10 at an inhibitor screening concentration of 1 μM. Next steps will need to include testing of the compounds in enzyme inhibition assays and cell-based assays for the identified kinases for each scaffold.</div><figure id="fig5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>In vitro</i> kinase selectivity profile of compound <b>7</b> (<b>5a</b>) and compound <b>62</b> (<b>5b</b>) at 1 μM (Eurofins DiscoverX KINOME<i>scan</i>). TREEspot interaction maps for <b>7</b> and <b>62</b> profiled against >400 human kinase targets. In this assay, primary screen binding interactions are reported as PoC, where lower values are an indication of stronger hits. Red dots depict the kinases for which PoC at 1 μM < 10. The size of the dot is based on the PoC values, with larger dots having smaller PoC values (more potent binders). PoC values for all kinases can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> CAMKK1 activity</h3><div class="NLM_p">CAMKK1 is a closely related and also understudied kinase. In order to learn if selectivity between CAMKK1 and CAMKK2 may be attainable in these series, we determined IC50 values for CAMKK1 inhibition for the 18 compounds with CAMKK2 IC50 < 150 nM, with the results depicted in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. STO-609 is reported to have some selectivity for CAMKK2 over CAMKK1, and this is recapitulated in our assays.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> In general, the compounds are more potent inhibitors of CAMKK2 than CAMKK1. Compound <b>34</b> (CAMKK1 IC<sub>50</sub> = 27 nM, CAMKK2 IC<sub>50</sub> = 23 nM) and compound 62 (CAMKK1 IC<sub>50</sub> = 31 nM, CAMKK2 IC<sub>50</sub> = 51 nM) are exceptions, being equipotent or slightly more potent on CAMKK1. For STO-609, a switch in a residue near the inhibitor hinge-binding group from a valine in CAMKK2 (Val269) to a leucine in CAMKK1 (Leu233) has been hypothesized as the source of selectivity.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Further experiments will need to be done to understand and exploit the differences in CAMKK1 and CAMKK2 activity observed for the inhibitors described here.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. CAMKK1 and CAMKK2 Enzyme Inhibition Data, CAMKK2 NanoBRET in Cell Target Engagement, LE Metrics, and cLog <i>P</i> and Solubility Data for Potent CAMKK2 Inhibitors<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">CAMKK1 enzyme IC<sub>50</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±">CAMKK2 enzyme IC<sub>50</sub> [nM]</th><th class="colsep0 rowsep0" align="center">CAMKK2 NB IC<sub>50</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char=".">cLog <i>P</i></th><th class="colsep0 rowsep0" align="center" char=".">LE</th><th class="colsep0 rowsep0" align="center" char=".">LLE</th><th class="colsep0 rowsep0" align="center" char=".">solubility [μg/mL]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STO-609</td><td class="colsep0 rowsep0" align="left">408 ± 22</td><td class="colsep0 rowsep0" align="char" char="±">58 ± 5.7</td><td class="colsep0 rowsep0" align="left">NG</td><td class="colsep0 rowsep0" align="char" char=".">3.59</td><td class="colsep0 rowsep0" align="char" char=".">0.41</td><td class="colsep0 rowsep0" align="char" char=".">3.61</td><td class="colsep0 rowsep0" align="char" char=".">60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GSK650394</td><td class="colsep0 rowsep0" align="left">33 ± 6.9</td><td class="colsep0 rowsep0" align="char" char="±">3 ± 0.4</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="char" char=".">5.67</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td><td class="colsep0 rowsep0" align="char" char=".">2.83</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">195 ± 25</td><td class="colsep0 rowsep0" align="char" char="±">26 ± 6.1</td><td class="colsep0 rowsep0" align="left">NG</td><td class="colsep0 rowsep0" align="char" char=".">3.78</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">3.82</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13g</b></td><td class="colsep0 rowsep0" align="left">961 ± 132</td><td class="colsep0 rowsep0" align="char" char="±">65 ± 15</td><td class="colsep0 rowsep0" align="left">700 ± 9.3</td><td class="colsep0 rowsep0" align="char" char=".">5.68</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">1.52</td><td class="colsep0 rowsep0" align="char" char=".">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14g</b></td><td class="colsep0 rowsep0" align="left">384 ± 53</td><td class="colsep0 rowsep0" align="char" char="±">5 ± 1.1</td><td class="colsep0 rowsep0" align="left">210 ± 24</td><td class="colsep0 rowsep0" align="char" char=".">6.32</td><td class="colsep0 rowsep0" align="char" char=".">0.39</td><td class="colsep0 rowsep0" align="char" char=".">1.98</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">1480 ± 120</td><td class="colsep0 rowsep0" align="char" char="±">145 ± 30</td><td class="colsep0 rowsep0" align="left">530 ± 95</td><td class="colsep0 rowsep0" align="char" char=".">6.01</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char=".">42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">238 ± 32</td><td class="colsep0 rowsep0" align="char" char="±">21 ± 1.3</td><td class="colsep0 rowsep0" align="left">190 ± 19</td><td class="colsep0 rowsep0" align="char" char=".">5.17</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td><td class="colsep0 rowsep0" align="char" char=".">2.53</td><td class="colsep0 rowsep0" align="char" char=".">50.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">2013 ± 174</td><td class="colsep0 rowsep0" align="char" char="±">44 ± 9.1</td><td class="colsep0 rowsep0" align="left">200 ± 11</td><td class="colsep0 rowsep0" align="char" char=".">4.13</td><td class="colsep0 rowsep0" align="char" char=".">0.44</td><td class="colsep0 rowsep0" align="char" char=".">3.27</td><td class="colsep0 rowsep0" align="char" char=".">48.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="char" char="±">120 ± 21</td><td class="colsep0 rowsep0" align="left">240 ± 7</td><td class="colsep0 rowsep0" align="char" char=".">6.06</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">0.84</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">1345 ± 222</td><td class="colsep0 rowsep0" align="char" char="±">56 ± 14</td><td class="colsep0 rowsep0" align="left">470 ± 44</td><td class="colsep0 rowsep0" align="char" char=".">3.97</td><td class="colsep0 rowsep0" align="char" char=".">0.42</td><td class="colsep0 rowsep0" align="char" char=".">3.33</td><td class="colsep0 rowsep0" align="char" char=".">40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">27 ± 4.5</td><td class="colsep0 rowsep0" align="char" char="±">23 ± 4.2</td><td class="colsep0 rowsep0" align="left">8.1 ± 0.8</td><td class="colsep0 rowsep0" align="char" char=".">5.42</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td><td class="colsep0 rowsep0" align="char" char=".">2.18</td><td class="colsep0 rowsep0" align="char" char=".">55.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">407 ± 25</td><td class="colsep0 rowsep0" align="char" char="±">24 ± 1.1</td><td class="colsep0 rowsep0" align="left">170 ± 18</td><td class="colsep0 rowsep0" align="char" char=".">3.99</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">3.61</td><td class="colsep0 rowsep0" align="char" char=".">55.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">7280 ± 216</td><td class="colsep0 rowsep0" align="char" char="±">31 ± 4.3</td><td class="colsep0 rowsep0" align="left">1400 ± 250</td><td class="colsep0 rowsep0" align="char" char=".">3.78</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">3.72</td><td class="colsep0 rowsep0" align="char" char=".">62.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">1838 ± 348</td><td class="colsep0 rowsep0" align="char" char="±">137 ± 12</td><td class="colsep0 rowsep0" align="left">540 ± 62</td><td class="colsep0 rowsep0" align="char" char=".">6.59</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td><td class="colsep0 rowsep0" align="char" char=".">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">239 ± 20</td><td class="colsep0 rowsep0" align="char" char="±">13 ± 1.0</td><td class="colsep0 rowsep0" align="left">140 ± 12</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">64.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">2614 ± 349</td><td class="colsep0 rowsep0" align="char" char="±">12 ± 3.9</td><td class="colsep0 rowsep0" align="left">890 ± 71</td><td class="colsep0 rowsep0" align="char" char=".">5.86</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td><td class="colsep0 rowsep0" align="char" char=".">2.04</td><td class="colsep0 rowsep0" align="char" char=".">70.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">5481 ± 130</td><td class="colsep0 rowsep0" align="char" char="±">96 ± 14</td><td class="colsep0 rowsep0" align="left">1900 ± 200</td><td class="colsep0 rowsep0" align="char" char=".">3.97</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td><td class="colsep0 rowsep0" align="char" char=".">3.03</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="char" char="±">21 ± 2.6</td><td class="colsep0 rowsep0" align="left">290 ± 23</td><td class="colsep0 rowsep0" align="char" char=".">4.68</td><td class="colsep0 rowsep0" align="char" char=".">0.41</td><td class="colsep0 rowsep0" align="char" char=".">3.02</td><td class="colsep0 rowsep0" align="char" char=".">19.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="char" char="±">108 ± 6.9</td><td class="colsep0 rowsep0" align="left">950 ± 110</td><td class="colsep0 rowsep0" align="char" char=".">2.57</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td><td class="colsep0 rowsep0" align="char" char=".">4.43</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">31 ± 6.5</td><td class="colsep0 rowsep0" align="char" char="±">51 ± 13</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="char" char=".">5.38</td><td class="colsep0 rowsep0" align="char" char=".">0.37</td><td class="colsep0 rowsep0" align="char" char=".">1.92</td><td class="colsep0 rowsep0" align="char" char=".">10.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">NB = data from the NanoBRET in the cell target engagement assay; NG = data not generated.</p></div></div></div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Compound Properties</h3><div class="NLM_p">A number of key physicochemical screens and calculations are considered in early-stage drug discovery, including lipophilicity, p<i>K</i><sub>a</sub>, solubility, permeability, and stability. Solubility is one of the most critical physicochemical properties as poor solubility can lead to an underestimation of activity, inaccurate SAR, inaccurate <i>in vitro</i> ADMET test results, and downstream compound development issues.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> Highlighting the critical importance of solubility, it has been shown that 87% of commercial drugs have solubility ≥65 μg/mL, but only 7% had solubility ≤20 μg/mL.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Therefore, solubility data can identify a series liability that needs to be fixed and highlight promising series with enhanced solubility. This information can help guide optimization strategies. LE metrics are also a commonly used tool that can guide hit-to-lead optimization for drug candidates.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> To this end, we calculated LE and lipophilic ligand efficiency (LLE) and collected kinetic solubility data for compounds with IC<sub>50</sub> values <150 nM in the CAMKK2 enzyme assay (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div><div class="NLM_p">LE values are influenced by molecular size; hence, it was unsurprising to see two clear ranges depending on the presence or absence of the phenyl ring. The hinge binders without the phenyl ring had LE values ranging from 0.41 to 0.46. These are smaller compounds with molecular weights between 300 and 325, and generally, LE values above 0.3 are acceptable for this MW range.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The LLE values for these compounds were between 3.02 and 4.43. Aminopyridine <b>61</b> had the highest LE and LLE values of 0.46 and 4.43, respectively. Both <b>7</b> and <b>38</b> had comparable LE values of 0.45 but significantly lower LLE values of 3.82 and 3.61, respectively.</div><div class="NLM_p">Both the LE and LLE metrics of the phenyl-substituted analogues have lower values. This is reflective of the increased atom count and higher cLog <i>P</i> values due to the additional lipophilicity of the phenyl ring. The LE values are between 0.32 and 0.40 for these compounds. GSK650394 has the highest LE and LLE values of 0.40 and 2.83, respectively. The analogous thienopyridine <b>14g</b> has the next highest LE value of 0.39 but a lower LLE value of 1.98. Quinoline <b>45</b> had the lowest LE and LLE values of 0.32 and 0.31, respectively. Solely relying on LE metrics to select compounds is unadvisable as many other factors, such as solubility, underpin successful drug development. A future direction could be to focus on polar substituents on the pendant phenyl to lower log <i>P</i>, enhance solubility, and mitigate the metabolic liability that may be seen in high-logP compounds. Similarly, non-aromatic substituents could be explored to access this same region of the active site.</div><div class="NLM_p last">Of the phenyl-substituted analogues, GSK650394 and <b>14g</b> were the most potent, with IC<sub>50</sub> values of 3 and 5 nM, respectively, and had the best LE and LLE values. Although promising in this regard, these compounds have extremely poor solubility values of 2 and 1.5 μg/mL, respectively. Optimization of these compounds will thus require extensive structure–property-relationship studies in parallel to SAR studies to develop them into useful tools with adequate solubility for further study. Interestingly, furopyridine <b>13g</b> had the highest measured solubility value of 77 μg/mL. Substitution of the azaindole’s NH with an oxygen greatly improved solubility. The effect was reversed when a more lipophilic sulfur atom is incorporated into the same position, <b>14g</b>. Pyrimidine analogues <b>56</b> and <b>62</b> are also only modestly soluble (10.6 and 19.4 μg/mL). Optimization to tool compounds will likely require the introduction of solubilizing functional groups. A number of the compounds tested had reasonable solubility ranging from 38 to 55 μg/mL. Four compounds had a promising solubility above that recorded for <b>STO-609</b>, 60 μg/mL. These were the thienopyrimidine <b>42</b> (62.6 μg/mL) and quinoline <b>46</b> (64.4 μg/mL), both without the pendant phenyl ring. The phenyl-substituted quinazoline <b>49</b> has a solubility of 70.4 μg/mL, second only to furopyridine <b>13g</b>. Overall, the solubility of several potent compounds was encouraging but will need to be monitored as optimization proceeds.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> CAMKK2 NanoBRET Cellular Target Engagement</h3><div class="NLM_p">Having demonstrated potent CAMKK2 inhibition and promising selectivity profiles for these compounds we tested in kinome-wide profiling, we turned our attention to the cellular activity of the compounds (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). To this end, we developed a CAMKK2 NanoBRET target engagement assay.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> This assay uses bioluminescence resonance energy transfer (BRET) between nanoluciferase (NL) fused to the kinase domain of CAMKK2 (BRET donor) and a tracer molecule that binds to the ATP-binding site of the kinase (BRET acceptor). The addition of cell penetrant CAMKK2 inhibitors leads to displacement of the tracer molecule and a quantifiable reduction in BRET signal. This assay is conducted in living cells, and the observation of binding not only depends on affinity between the compound and CAMKK2 but also on compound cell penetrance. The parent molecule, GSK650394, was very potent in this assay (NB IC<sub>50</sub> < 3 nM). Several additional key compounds were evaluated in this assay (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a> and the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>). A key takeaway here is that all the compounds that advanced to this assay (those with CAMKK2 enzyme IC<sub>50</sub> < 150 nM) demonstrated measurable CAMKK2 target engagement in cells. The potency was below 500 nM for thienopyridine <b>14g</b> (210 nM), pyrazolopyrimidines <b>20</b> and <b>21</b> (190 and 200 nM), <i>N</i>-methyl azaindoles <b>29</b> and <b>30</b> (240 and 470 nM), imidazopyridine <b>34</b> (8 nM), thienopyrimidine <b>38</b> (170 nM), quinoline <b>46</b> (140 nM), 2-phenylpyrimidine <b>56</b> (290 nM), and 2-amino-pyrimidine <b>62</b> (<3 nM). The results for exemplar compounds from five different scaffolds (<b>13g</b>, <b>45</b>, <b>46</b>, <b>56</b>, and <b>62</b>) are depicted in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>.</div><figure id="fig6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. CAMKK2 NanoBRET dose–response curves for compounds <b>13g</b>, <b>45</b>, <b>46</b>, <b>56</b>, and <b>62</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> On-Target Cellular Effect (Phosphorylation Inhibition Assay)</h4><div class="NLM_p">We also assessed the impact of our inhibitors on phosphorylation downstream from CAMKK2 using Western blot analysis with C4-2 prostate cancer cells to provide evidence of a phenotypic and on-target effect of our CAMKK2 inhibitors in intact cells. AMPK (Thr172) was chosen because in intact cells, CAMKK2, but not the related CAMKK1, can readily phosphorylate this substrate.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> We performed preliminary Western blot analysis at a single inhibitor concentration of 1 μM for the 18 compounds with CAMKK2 enzyme IC<sub>50</sub> < 150 nM along with STO-609. These results are shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</div><figure id="fig7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blots of the 18 compounds with IC<sub>50</sub> < 150 nM in the CAMKK2 enzyme assay, together with STO-609, screened at a single inhibitor concentration = 1 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on these results from screening at a single concentration, compounds <b>7</b>, <b>20</b>, <b>29</b>, <b>34</b>, <b>38</b>, <b>46</b>, <b>56</b>, and <b>62</b> showed the most robust reduction of the p-AMPK band and were advanced into full dose response experiments in this same assay format to provide IC<sub>50</sub> estimates for inhibition of phosphorylation of AMPK at Thr172. The IC<sub>50</sub> values were determined from dose response experiments using 0–10 μM of the compound. The Western blot dose response experiments for the most potent compound, <b>62</b>, are depicted in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a. The Western blot dose response data for compounds 7, 20, 29, 34, 38, 46, and 56 can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>. In addition, dose response experiments for compound 62 utilizing lower inhibitor concentrations can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>. The calculated IC<sub>50</sub> values for the dose response experiments for all eight compounds are shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b. Compounds <b>62</b> (IC<sub>50</sub> = 10 nM), <b>46</b> (IC<sub>50</sub> = 390 nM), <b>56</b> (IC<sub>50</sub> = 790 nM), <b>20</b> (IC<sub>50</sub> = 900 nM), and <b>29</b> (IC<sub>50</sub> 980 nM) all have IC<sub>50</sub> values for inhibition of phosphorylation of AMPK at Thr172 below 1 μM, suggesting that optimization into compounds with potent cellular activity will be achievable for these series.</div><figure id="fig8" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Dose response for inhibition of phosphorylation of AMPK at Thr172 in C4-2 prostate cancer cells by compound <b>62</b>. (b). Calculated IC<sub>50</sub> values for inhibition of phosphorylation of AMPK at Thr172 in C4-2 prostate cancer cells for the top compounds from the single concentration experiment depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Through a hinge-binder scaffold hopping strategy, we have designed, synthesized, and biologically evaluated as CAMKK2 inhibitors a series of 32 compounds that utilize 5,6-bicyclic, 6,6-bicyclic, and single-ring hinge-binding moieties that are based on the 7-azaindole GSK650394. These inhibitors were designed with the aim of retaining CAMKK2 potency and improving the kinase selectivity of the promiscuous azaindole by changing the strong H-bond interactions that the parent azaindole makes with the kinase hinge-binding residues. Additionally, we sought to identify inhibitors with improved physicochemical properties, drug likeness, and selectivity all while retaining CAMKK2 inhibitory potency. Several compounds with similar or better potency than GSK650394 and STO-609 have been identified, some of which also showed improved physicochemical properties. We found that a number of these compounds, but not all, have some selectivity over the closely related enzyme CAMKK1 in enzyme assays. Further work will be needed to understand selectivity determinants, to optimize this selectivity, and to determine if the observed selectivity holds in a cellular context. Kinome-wide profiling revealed that the nine exemplars we screened in the Eurofins DiscoverX KINOME<i>scan</i> platform are not broadly promiscuous, with S<sub>10</sub> (1 μM) < 0.05. In addition, we have demonstrated that exemplars have cellular activity in two orthogonal assays, a CAMKK2 NanoBRET in cell target engagement assay, and Western blot experiments looking at inhibition of the production of p-AMPK. Our work has shown that kinase inhibitors with weakened (non-ideal) hinge-binding interactions can show highly selective kinase inhibition and still have useful on-target enzyme potency and cellular potency. This work has led to the identification of several CAMKK2 scaffolds with alternate hinge-binding moieties that hold promise as starting points for the discovery of potent, selective, and cell-active CAMKK2 inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Biology</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Protein Expression and Purification/DSF Assay</h4><div class="NLM_p last">Small-molecule screening by DSF was performed as described previously.<a onclick="showRef(event, 'ref48 ref65'); return false;" href="javascript:void(0);" class="ref ref48 ref65">(48,65)</a> Briefly, the DSF assay was performed in the 96-well format. Purified CAMKK1 or CAMKK2 was diluted to 2 μM kinase in 100 mM potassium phosphate pH 7.5, 150 mM NaCl, and 10% glycerol supplemented with 5 × SYPRO Orange (Invitrogen, Carlsbad, CA, USA). All assay experiments used 19.5 μL of 2 μM kinase and SYPRO Orange mixture. Compounds solubilized in dimethyl sulfoxide (DMSO) were used at a 12.5 μM final concentration, with a 2.5% concentration of DMSO per well. PCR plates were sealed using optically clear films and transferred to a C1000 thermal cycler with CFX-96 RT-PCR head (BioRad, Hercules, CA, USA). The fluorescence intensity was measured over a temperature gradient from 25 to 95 °C at a constant rate of 0.05 °C/s. Curve fitting and protein melting temperatures were calculated based on a Boltzmann function fitting to experimental data (GraphPad Prism 8). Protein with the addition of 2.5% DMSO was used as a reference. All experiments were carried out in triplicate, and the mean of the Δ<i>T</i><sub>m</sub> is reported. Compounds that provided negative values are presented as having a Δ<i>T</i><sub>m</sub> of 0 °C.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> CAMKK1 and CAMKK2 Enzyme Assays</h4><div class="NLM_p last">CAMKK1 and CAMKK2 activity was determined by measuring the transfer of radiolabeled phosphate from [γ-<sup>32</sup>P]-ATP to a synthetic peptide substrate (CaMKKtide) as previously described.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Briefly, purified recombinant CAMKK1 or CAMKK2 (100 pM) was incubated in the assay buffer [50 mM HEPES (pH 7.4), 1 mM dithiothreitol, 0.02% (v/v) Brij-35] containing 200 μM CaMKKtide (Genscript), 100 μM CaCl<sub>2</sub>, 1 μM CaM (Sigma-Aldrich, Castle Hill, NSW, Australia), 200 μM [γ-<sup>32</sup>P]-ATP (Perkin Elmer, Boston, MA, USA), 5 mM MgCl<sub>2</sub> (Sigma-Aldrich, Castle Hill, NSW, Australia), and various concentrations of inhibitors (0–1 μM) in a standard 30 μL assay for 10 min at 30 °C. Reactions were terminated by spotting 15 μl onto P81 phosphocellulose paper (GE Lifesciences, Paramatta, NSW, Australia) and washing extensively in 1% phosphoric acid (Sigma-Aldrich, Castle Hill, NSW, Australia). Radioactivity was quantified by liquid scintillation counting.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> CAMKK2 NanoBRET Assay</h4><div class="NLM_p last">To quantify the cellular activity of these inhibitors, we developed a CAMKK2 NanoBRET target engagement assay.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Briefly, this assay utilizes an NL fused to the kinase domain of CAMKK2. This NL kinase fusion is then transiently transfected into HEK293 cells, and after 24 h, the tracer is added to the cells. When the tracer and the NL-CAMKK2 fusion come into proximity, they create a BRET signal that can be competed in a dose-dependent manner by the addition of cell-penetrant CAMKK2 inhibitors.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Western Blot Analysis</h4><div class="NLM_p">C4-2 cells were plated in 6-well plates in the IMEM medium containing 0.5% fetal bovine serum. After 72 h, the cells were then treated with the compounds for 24 h before the media were aspirated and wells were washed twice in ice-cold phosphate-buffered saline. Cells were lysed using RIPA buffer containing phosphatase and the protease inhibitor cocktail while rotating for 30 min at 4 °C. In each lane, 30 μg/well of protein lysate was loaded into a 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis gel and run for 1 h and 30 min. Gels were then transferred overnight in a TRIS-glycine/methanol transfer buffer onto a poly(vinylidene difluoride) membrane at 4 °C. Membranes were blocked, incubated with primary overnight at 4 °C, washed, incubated with secondary at room temperature (rt) for 1 h, washed, and then developed on an Azure Biosystems C-600 imager.</div><div class="NLM_p last">[Cell signaling: phospho-AMPKα (Thr172) (40H9) rabbit mAb: Cat#: 2535; AMPKα (D5A2) Rabbit mAb Cat#: 5831; BD Bioscience: CAMKK mouse mAb Cat# 610544; Sigma: GAPDH rabbit pAb: Cat# G9545; secondary antibody: goat anti-rabbit IgG (H + L)-HRP conjugate was from Bio-Rad (Cat#:1706515)].</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Crystallization, Data Collection, and Structure Determination</h4><div class="NLM_p">Crystallization of the CAMKK2-kinase domain bound to <b>13g</b> or UNC10244803 followed a previously established protocol.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Briefly, the inhibitor (dissolved in 100% DMSO) was added to the protein in 3-fold molar excess and incubated on ice for approximately 30 min. The mixture was centrifuged at 21,000<i>g</i> for 10 minutes at 4 °C before setting up 150 nL volume sitting drops at three ratios (2:1, 1:1, or 1:2 protein–inhibitor complex to reservoir solution). Crystallization experiments were performed at 20 °C. Crystal optimization used Newman’s buffer system.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Crystals were cryoprotected in the mother liquor supplemented with 25–30% glycerol before flash cooling in liquid nitrogen for data collection. Diffraction data were collected at 100 K at the Advanced Photon Source 24ID-C or at the Diamond Light Source (DLS) I03 beamline.</div><div class="NLM_p last">Diffraction data were integrated with XDS<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and scaled using AIMLESS from the CCP4 software suite.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The structure was solved by molecular replacement using Phaser<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> and the kinase domain of CAMKK2 as the search model (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZV2">2ZV2</a>).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Refinement was performed using REFMAC5,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> and Coot<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> was used for model building. Structure validation was performed using MolProbity.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> Structure factors and coordinates for the structure were deposited at the PDB (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a>).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Chemistry</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> General Chemistry Information</h4><div class="NLM_p last">All reagents and solvents, unless specifically stated, were used as obtained from their commercial sources without further purification. Solvents were degassed with nitrogen for cross-coupling reactions. Air- and moisture-sensitive reactions were performed under an inert atmosphere using nitrogen in a previously oven-dried or flame-dried reaction flask, and addition of reagents was done using a syringe. All microwave (μW) reactions were carried out in a Biotage Initiator EXP US 400W microwave synthesizer. Thin-layer chromatography (TLC) analyses were performed using 200 μm pre-coated sorbtech fluorescent TLC plates, and spots were visualized using UV light. High-resolution mass spectrometry samples were analyzed with a ThermoFisher Q Exactive HF-X (ThermoFisher, Bremen, Germany) mass spectrometer coupled with a Waters Acquity H-class liquid chromatograph system. All high-resolution mass spectrometry (HRMS) spectra were recorded <i>via</i> electrospray ionization (ESI). Column chromatography was undertaken with a Biotage Isolera One or Prime instrument. Nuclear magnetic resonance (NMR) spectrometry was run on a Varian Inova 400 MHz or Bruker AVANCE III 700 MHz spectrometer equipped with a TCI H-C/N-D 5 mm cryoprobe, and data were processed using the MestReNova processor. Chemical shifts are reported in ppm with residual solvent peaks referenced as the internal standard. All compounds were >95% pure by analytical LC.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-Bromo-2-cyclopentylbenzoic Acid (<b>2</b>)</h4><div class="NLM_p">4-Bromo-2-fluorobenzoic acid (2.00 g, 9.00 mmol) was dissolved in tetrahydrofuran (THF) (20.0 mL) and cooled to 0 °C. Cyclopentylmagnesium bromide solution (16.0 mL of a 2 M solution, 32.0 mmol) was added dropwise. The reaction was stirred at 0 °C for 4 h. 2 M HCl (25.0 mL) was then slowly added to the solution, followed by EtOAc (40.0 mL). The organic phase was separated, concentrated, and purified using column chromatography (10% EtOAc/hexane) to afford 4-bromo-2-cyclopentylbenzoic acid <b>2</b> (1.70 g, 71%) as a pure white solid. The NMR data for this compound match those previously reported.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.07 (s, 1H), 7.60–7.56 (m, 2H), 7.45 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 3.68 (tt, <i>J</i> = 9.5, 7.5 Hz, 1H), 2.03–1.94 (m, 2H), 1.82–1.72 (m, 2H), 1.67–1.46 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.9, 148.6, 131.2, 131.1, 129.5, 128.6, 125.1, 41.2, 34.2, 25.2; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 269.0.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2-Cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic Acid (<b>3</b>)</h4><div class="NLM_p">To a stirred solution of 4-bromo-2-cyclopentylbenzoic acid <b>2</b> (2.50 g, 9.30 mmol) and bis(pinacolato)diboron (2.80 g, 11.0 mmol) in dioxane (50.0 mL) were added PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> (0.76 g, 0.96 mmol) and KOAc (3.60 g, 37.0 mmol). The solution was heated to 95 °C for 2 h. Once cooled to rt, the solution was diluted with EtOAc and filtered through a celite pad. The solution was concentrated, and the crude product was purified by column chromatography (10% EtOAc/hexane) to afford 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (2.30 g, 78%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.00 (s, 1H), 7.69 (d, <i>J</i> = 1.1 Hz, 1H), 7.60 (d, <i>J</i> = 7.6 Hz, 1H), 7.53 (dd, <i>J</i> = 7.6, 1.1 Hz, 1H), 3.62 (tt, <i>J</i> = 9.8, 7.4 Hz, 1H), 2.05–1.95 (m, 2H), 1.83–1.73 (m, 2H), 1.69–1.45 (m, 4H), 1.29 (s, 12H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.7, 144.4, 134.9, 132.1, 131.5, 128.2, 83.9, 41.2, 34.4, 25.2, 24.7; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 317.2.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Methyl 2-Cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate (<b>4</b>)</h4><div class="NLM_p">To a stirred solution of methyl 4-bromo-2-cyclopentylbenzoate <b>SI1</b> (1.50 g, 5.29 mmol) and bis(pinacolato)diboron (1.88 g, 7.42 mmol) in dioxane (35.0 mL) were added Pd(dppf)Cl<sub>2</sub> (194 mg, 0.26 mmol) and KOAc (1.56 g, 15.9 mmol). The solution was heated to 100 °C for 2 h. Once cooled to rt, the solution was filtered through a pad of Celite and washed with 100 mL of EtOAc. The volatiles were removed <i>in vacuo</i>, and the crude residue was purified <i>via</i> column chromatography (0–5% EtOAc/hexane) to afford methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (1.37 g, 78%) as oil which slowly solidified into a white solid under vacuum.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.70 (d, <i>J</i> = 1.1 Hz, 1H), 7.60 (d, <i>J</i> = 7.7 Hz, 1H), 7.56 (d, <i>J</i> = 1.1 Hz, 1H), 3.83 (s, 3H), 3.50 (tt, <i>J</i> = 9.9, 7.5 Hz, 1H), 2.04–1.93 (m, 2H), 1.85–1.70 (m, 2H), 1.70–1.44 (m, 4H), 1.30 (s, 12H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.3, 144.6, 133.6, 132.2, 131.6, 128.3, 84.0, 52.2, 41.3, 34.3, 25.2, 24.7; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 330.2.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Methyl 2-Cyclopentyl-4-(1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoate (<b>6</b>)</h4><div class="NLM_p">To a stirred solution of 3-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (450 mg, 1.37 mmol) and methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (681 mg, 2.06 mmol) in PhMe (8.0 mL) and H<sub>2</sub>O (1.00 mL) were added Pd(OAc)2 (15.4 mg, 68.7 μmol), K3PO4 (934 mg, 4.40 mmol), and P(Cy)3 (38.6 mg, 137 μmol). The reaction mixture was heated to 100 °C for 18 h and allowed to cool to rt, filtered through a pad of Celite, and washed with EtOAc (10 mL). The filtrate was concentrated <i>in vacuo</i>, and the crude was purified by column chromatography eluting with 0–30% EtOAc/hexane to afford methyl 2-cyclopentyl-4-(1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoate (545 mg, 88%) as a light-yellow solid (LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 327.3). A solution of the above material (“SEM-protected <b>6</b>”), 2-cyclopentyl-4-(1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoate (500 mg, 1.11 mmol), in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was treated with trifluoroacetyl (3.50 mL, 45.5 mmol). The solution was stirred at 25 °C for 2 h and then neutralized with sat. NaHCO<sub>3</sub> (50 mL). The solution was extracted with EtOAc (3 × 100 mL), and the combined organic phases were washed with sat. NaCl solution, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was dissolved in ethanol (40.0 mL), and NaOAc (1.82 mg, 22.2 mmol) was added. The mixture was stirred at 50 °C for 20 h and allowed to cool to rt. The solution was diluted with H<sub>2</sub>O (50 mL) and EtOAc (75.0 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 75 mL). The combined organics were washed with a saturated NaCl solution (50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude material was purified by column chromatography (0–30% EtOAc/hexane) to afford methyl 2-cyclopentyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoate <b>6</b> (245 mg, 69%) as a deep-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.10–12.05 (m, 1H), 8.32–8.24 (m, 2H), 8.05 (d, <i>J</i> = 2.7 Hz, 1H), 7.78–7.72 (m, 2H), 7.63 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.20 (dd, <i>J</i> = 8.0, 4.6 Hz, 1H), 3.84 (s, 3H), 3.79–3.67 (m, 1H), 2.11–1.98 (m, 2H), 1.88–1.77 (m, 2H), 1.74–1.58 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.9, 149.2, 147.2, 143.1, 138.7, 130.3, 127.4, 127.2, 125.1, 124.1, 123.2, 117.2, 116.4, 113.4, 51.9, 41.3, 34.3, 25.2; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 320.4.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-Cyclopentyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>7</b>)</h4><div class="NLM_p">Methyl 2-cyclopentyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoate (100 mg, 0.312 mmol) was dissolved in methanol (8.00 mL), followed by the addition of 50% NaOH solution (170 μL, 3.12 mmol). The mixture was heated to 75 °C for 1 h, allowed to cool, and extracted with diethyl ether (20.0 mL). The aqueous layer was then acidified to pH 5 with 1 M HCl and extracted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL). The organic layer was then dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to afford 2-cyclopentyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoic acid <b>7</b> (85.0 mg, 89%) as a pale-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.64 (s, 1H), 12.12–12.08 (m, 1H), 8.32–8.27 (m, 2H), 8.03 (d, <i>J</i> = 2.6 Hz, 1H), 7.78–7.71 (m, 2H), 7.60 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.22 (dd, <i>J</i> = 8.0, 4.7 Hz, 1H), 3.92–3.81 (m, 1H), 2.07 (d, <i>J</i> = 14.0 Hz, 2H), 1.82 (t, <i>J</i> = 5.8 Hz, 2H), 1.73–1.61 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 148.3, 147.1, 142.3, 138.1, 130.4, 128.6, 128.2, 125.2, 124.2, 123.2, 117.7, 116.4, 113.8, 41.2, 34.4, 25.3; HRMS: calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 307.1447; found, 307.1426; mp range 238–242 °C.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Methyl 4-(5-Chloro-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate (<b>9</b>)</h4><div class="NLM_p">5-Chloro-3-iodo-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.34 mmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate (113 mg, 0.34 mmol), Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub> (28.0 mg, 34.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (334 mg, 1.00 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2 mL) was added before the vial was flushed with nitrogen and capped. The solution was stirred at rt for 16 h. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–20% EtOAc/hexane) to afford methyl 4-(5-chloro-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>9</b> (217 mg, 69%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 2.2 Hz, 1H), 8.15 (d, <i>J</i> = 2.0 Hz, 1H), 7.86 (d, <i>J</i> = 8.1 Hz, 1H), 7.60 (d, <i>J</i> = 1.6 Hz, 1H), 7.49 (s, 1H), 7.42 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 3.97 (s, 3H), 3.93 (s, 3H), 3.92–3.84 (m, 1H), 2.31–2.06 (m, 2H), 1.92–1.80 (m, 2H), 1.80–1.71 (m, 2H), 1.71–1.59 (m, 4H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 168.5, 148.6, 146.3, 141.7, 137.6, 130.8, 128.5, 128.3, 127.6, 125.0, 124.4, 123.6, 119.3, 114.2, 52.0, 41.7, 34.9, 31.8, 25.7; HRMS: calcd for C<sub>21</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub>Cl [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 369.1369; found: <i>m</i>/<i>z</i>, 369.1348; mp range 144–148 °C.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2-Cyclopentyl-4-(1-methyl-5-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>10</b>)</h4><div class="NLM_p">Methyl 4-(5-chloro-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>9</b> (100 mg, 0.27 mmol), phenylboronic acid (40.0 mg, 0.32 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (12.0 mg, 13.0 μmol), XPhos (13.0 mg, 27.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (265 mg, 0.81 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2 ml) was added, and the vial was flushed with nitrogen and capped. The solution was heated to 120 °C for 16 h. Once cooled, the solution was diluted with EtOAc (2 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–10% EtOAc/hexane) to afford the methyl ester intermediate. The methyl ester was saponified in a solution of aq LiOH (1.00 mL, 1 M) and dioxane (1.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2 mL), and then acidified with aq HCl (1 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford 2-cyclopentyl-4-(1-methyl-5-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoic acid <b>10</b> (77.0 mg, 72%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.75 (br s, 1H), 8.63 (d, <i>J</i> = 2.1 Hz, 1H), 8.43 (d, <i>J</i> = 2.1 Hz, 1H), 8.15 (s, 1H), 7.83–7.67 (m, 5H), 7.50 (dd, <i>J</i> = 8.4, 7.0 Hz, 2H), 7.43–7.34 (m, 1H), 3.94–3.85 (m, 1H), 3.92 (s, 3H), 2.12–1.99 (m, 2H), 1.88–1.76 (m, 2H), 1.75–1.58 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.7, 148.0, 147.7, 142.4, 139.2, 138.2, 131.0, 130.0, 129.6, 129.5, 128.9, 127.6, 126.1, 124.5, 123.6, 117.9, 113.3, 41.4, 34.8, 31.6, 25.8; HRMS: calcd for C<sub>26</sub>H<sub>25</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 397.1916; found <i>m</i>/<i>z</i>, 397.1911; mp range 199–201 °C.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-Cyclopentyl-4-(1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>11</b>)</h4><div class="NLM_p">3-Bromo-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.47 mmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate (180 mg, 0.54 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (22.0 mg, 24 μmol), XPhos (23.0 mg, 48.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (540 mg, 1.66 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was heated to 120 °C for 16 h. Once cooled, the solution was diluted with EtOAc (2.0 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (0–20% EtOAc/hexane) to afford the methyl ester intermediate. The methyl ester was saponified in a solution of aq LiOH (2.00 mL, 1.00 M) and dioxane (2.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2.00 mL), and then acidified with aq HCl (1.00 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford <b>11</b> (113 mg, 75%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.33 (d, <i>J</i> = 2.2 Hz, 1H), 8.29 (d, <i>J</i> = 2.2 Hz, 1H), 8.20 (s, 1H), 7.75 (d, <i>J</i> = 8.1 Hz, 1H), 7.65 (d, <i>J</i> = 1.8 Hz, 1H), 7.59 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 3.91–3.80 (m, 1H) 3.87 (s, 3H), 2.12–1.96 (m, 2H), 1.82 (m, 2H), 1.71–159 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 147.0, 146.3, 141.0, 136.9, 130.7, 130.5, 129.1, 126.8, 124.0, 123.4, 123.1, 118.2, 112.3, 41.1, 34.3, 31.3, 25.3; HRMS: calcd for C<sub>20</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 321.1603; found <i>m</i>/<i>z</i>, 321.1596; mp range 181–184 °C.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Ethyl 2-Chloronicotinate (<b>12b</b>)</h4><div class="NLM_p">To a solution of 2-chloronicotinic acid (5.00 g, 31.7 mmol) in toluene (35.0 mL) was added dropwise with stirring triethyl orthoacetate (17.5 mL, 95.2 mmol). The mixture was heated to reflux for 16 h and allowed to cool to rt, and the resultant solution was washed with sat. NaHCO<sub>3</sub> (50.0 mL). The organic phase was dried with MgSO<sub>4</sub>, and the solvent was removed <i>in vacuo</i> to afford the title compound, ethyl 2-chloronicotinate <b>12b</b> (5.40 g, 92%), as a clear oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (dd, <i>J</i> = 4.8, 2.0 Hz, 1H), 8.24 (dd, <i>J</i> = 7.7, 2.0 Hz, 1H), 7.57 (dd, <i>J</i> = 7.7, 4.8 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 1.32 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.1, 152.2, 147.8, 140.2, 127.1, 123.2, 61.8, 13.9; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 186.0.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Ethyl 5-Bromo-2-chloronicotinate (<b>13b</b>)</h4><div class="NLM_p">To a solution of 5-bromo-2-chloronicotinic acid (10.0 g, 42.0 mmol) in ethanol (60.0 mL) was added slowly H<sub>2</sub>SO<sub>4</sub> (9.10 g, 5.0 mL, 93.0 mmol). The reaction mixture was heated to reflux for 16 h. The mixture was concentrated <i>in vacuo</i>, re-dissolved in aq NaHCO<sub>3</sub> (150 mL), and extracted with EtOAc (2 × 300 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the desired product, <b>13b</b> (10.5 g, 92%), as a light-yellowish oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.76 (d, <i>J</i> = 2.5 Hz, 1H), 8.47 (d, <i>J</i> = 2.5 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 1.33 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.88, 152.79, 146.65, 142.12, 128.34, 118.85, 62.24, 13.83; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 186.0.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Ethyl 2,5-Dichloronicotinate (<b>14b</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12b</b> using 2,5-dichloronicotinic acid (10.0 g, 42.0 mmol). After purification, ethyl 2,5-dichloronicotinate <b>14b</b> (11.0 g, 99%) was afforded as a clear oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (d, <i>J</i> = 2.6 Hz, 1H), 8.38 (d, <i>J</i> = 2.6 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 1.33 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.9, 150.6, 146.1, 139.5, 130.3, 128.0, 62.3, 13.8; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 220.1.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Ethyl 3-Hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate (<b>12c</b>)</h4><div class="NLM_p">To a suspension of sodium hydride, 60% dispersed in mineral oil, (5.60 g, 0.14 mol) in 1,2-dimethoxyethane (60.0 mL) was added ethyl 2-hydroxyacetate (13.0 mL, 0.13 mol) under ice cooling and stirring. The ice bath was removed, and the solution was allowed warm to rt. After 30 min, a solution of ethyl 2-chloronicotinate (10.0 g, 54.0 mmol) in 100 mL of 1,2-dimethoxyethane was slowly added and the mixture was heated to 75 °C for 2 h. The solvent was removed <i>in vacuo</i>, and the residual solid was re-dissolved in aq NaHCO<sub>3</sub> solution (150 mL) and EtOAc (250 mL). The aq layer was acidified with AcOH (pH 4) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 200 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification by flash column chromatography (10% EtOAc/hexane) afforded ethyl 3-hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate <b>12c</b> (8.5 g, 76%) as a pale-white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.53 (dd, <i>J</i> = 4.8, 1.8 Hz, 1H), 8.11 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.30 (dd, <i>J</i> = 7.8, 4.8 Hz, 1H), 4.48 (q, <i>J</i> = 7.1 Hz, 2H), 1.45 (t, <i>J</i> = 7.1 Hz, 3H).</div><div class="NLM_p last">LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 208.1.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Ethyl 5-Bromo-3-hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate (<b>13c</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12c</b> using ethyl 5-bromo-2-chloronicotinate <b>13b</b> (18.0 g, 68.0 mmol). After purification, ethyl 5-bromo-3-hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate <b>13c</b> (19.0 g, 76%) was afforded as a light-brown solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.34–11.30 (m, 1H), 8.58 (d, <i>J</i> = 2.3 Hz, 1H), 8.53 (d, <i>J</i> = 2.3 Hz, 1H), 4.32 (q, <i>J</i> = 7.1 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.7, 156.7, 148.8, 144.7, 133.5, 127.2, 115.9, 114.3, 60.4, 14.3; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 286.0.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Ethyl 5-Chloro-3-hydroxythieno[2,3-<i>b</i>]pyridine-2-carboxylate (<b>14c</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12c</b> using ethyl 2,5-dichloronicotinate <b>14b</b> (15.0 g, 68.0 mmol) and ethyl 2-mercaptoacetate (15.0 mL, 0.14 mmol). After purification, ethyl 5-chloro-3-hydroxythieno[2,3-<i>b</i>]pyridine-2-carboxylate <b>14c</b> (12.4 g, 71%) was afforded as a light-orange solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.04 (s, 1H), 8.73 (d, <i>J</i> = 2.4 Hz, 1H), 8.41 (d, <i>J</i> = 2.4 Hz, 1H), 4.33 (q, <i>J</i> = 7.1 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.6, 155.7, 152.2, 149.4, 130.3, 127.7, 126.7, 105.5, 61.1, 14.2; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 258.0.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Furo[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one (<b>12d</b>)</h4><div class="NLM_p">A mixture of ethyl 3-hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate <b>12c</b> (6.00 g, 30.0 mmol) in ethanol (100 mL) and sodium hydroxide (5.00 g, 100 mmol) dissolved in 15.0 mL of water was refluxed for 1 h. After evaporation of the solvent, the yellow-orange crystalline mass was dissolved in water (100 mL), acidified to pH 2–3 with conc. HCl, and heated to reflux for 45 min. Once cooled, the solution was neutralized with sodium bicarbonate and extracted with chloroform. The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>) and filtered, and the solvent was removed <i>in vacuo</i> to afford furo[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one <b>12d</b> (2.70 g, 67%) as a light-brown solid. The title compound was isolated as a mixture of keto–enol tautomers (9:1).</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.71 (s, 0H), 8.60 (dd, <i>J</i> = 4.9, 2.0 Hz, 1H), 8.15 (dd, <i>J</i> = 7.5, 1.9 Hz, 1H), 7.27–7.22 (m, 1H), 4.89 (s, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 197.7, 177.1, 156.9, 134.3, 118.8, 113.6, 74.6; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 136.1.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 5-Bromofuro[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one (<b>13d</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12d</b> using ethyl 5-bromo-3-hydroxyfuro[2,3-<i>b</i>]pyridine-2-carboxylate <b>13c</b> (10.0 g, 35.0 mmol). After purification, 5-bromofuro[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one <b>13d</b> (6.30 g, 84%) was afforded as a brown solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.71 (d, <i>J</i> = 2.5 Hz, 1H), 8.42 (d, <i>J</i> = 2.5 Hz, 1H), 4.96 (s, 2H); <sup>13</sup>C NMR (100 MHz, DMSO): δ 196.4, 175.6, 156.9, 136.3, 115.5, 113.2, 75.8; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 213.4.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 5-Chlorothieno[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one (<b>14d</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12d</b> using ethyl 5-chloro-3-hydroxythieno[2,3-<i>b</i>]pyridine-2-carboxylate <b>14c</b> (3.0g, 12 mmol). After purification, 5-chlorothieno[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one <b>14d</b> (1.0 g, 50%) was isolated as an orange solid composed of keto- and enol-isomers.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.46 (s, 1H), 8.77 (d, <i>J</i> = 2.5 Hz, 1H), 8.56 (dd, <i>J</i> = 2.3, 0.5 Hz, 1H), 8.16 (d, <i>J</i> = 2.4 Hz, 1H), 8.11 (d, <i>J</i> = 2.5 Hz, 1H), 6.70 (d, <i>J</i> = 0.5 Hz, 1H), 4.18 (s, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 196.8, 172.1, 156.5, 154.7, 145.4, 145.3, 133.2, 127.9, 127.8, 126.9, 126.0, 100.9, 40.8; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 186.4.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Furo[2,3-<i>b</i>]pyridin-3-yl Trifluoromethanesulfonate (<b>12e</b>)</h4><div class="NLM_p">A solution of <b>12d</b> (450 mg, 3.33 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (8.00 mL) was cooled to 0 °C. DIPEA (0.70 mL, 4.00 mmol) was added, followed by dropwise addition of trifluoromethanesulfonic anhydride (788 μL, 4.66 mmol). The reaction mixture was slowly warmed to rt and stirred for 2 h and then quenched with water. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and the combined organic extracts were dried, filtered, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (10% EtOAc/hexane) to afford furo[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>12e</b> (640 mg, 72%) as a brown oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.55–8.41 (m, 1H), 8.03 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.91 (s, 1H), 7.43–7.35 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 158.81, 146.35, 135.28, 132.54, 129.95, 128.17, 120.11, 118.71 (q, <i>J</i> = 321.6 Hz); LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 268.2.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 5-Bromofuro[2,3-<i>b</i>]pyridin-3-yl Trifluoromethanesulfonate (<b>13e</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12e</b> using 5-bromofuro[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one <b>13d</b> (10.0 g, 47.0 mmol). After purification, 5-bromofuro[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>13e</b> (12.8 g, 79%) was isolated as a light-brown oil which solidified upon standing.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.50 (dd, <i>J</i> = 2.2, 0.4 Hz, 1H), 8.14 (d, <i>J</i> = 2.2 Hz, 1H), 7.92 (s, 1H); LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 347.1.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 5-Chlorothieno[2,3-<i>b</i>]pyridin-3-yl Trifluoromethanesulfonate (<b>14e</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12e</b> using 5-chlorothieno[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one <b>14d</b> (800 mg, 4.31 mmol). After purification, 5-Chlorothieno[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>14e</b> (1.00 g, 74%) was isolated as a brown oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.60 (d, <i>J</i> = 2.3 Hz, 1H), 8.02 (d, <i>J</i> = 2.3 Hz, 1H), 7.55 (d, <i>J</i> = 0.5 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 155.34, 147.62, 134.57, 129.60, 127.41, 125.23, 118.67 (q, <i>J</i> = 321.2 Hz), 118.13; LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 318.4.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-Cyclopentyl-4-(furo[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>12f</b>)</h4><div class="NLM_p">A microwave vial was loaded with furo[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>12e</b> (250 mg, 936 μmol), 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (444 mg, 1.40 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (11.0 mg, 9.50 μmol), sodium carbonate (397 mg, 3.74 mmol), MeOH (2.00 mL), and CH<sub>2</sub>Cl<sub>2</sub> (0.50 mL). The vial was sealed and heated to 90 °C for 2 h. The solution was neutralized with aq HCl (10 mL) and then extracted with ethyl acetate (2 × 20.0 mL). The organic phases were combined and dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (30% EtOAc/hexane) to afford <b>12f</b> (217 mg, 76%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.94 (s, 1H), 8.67 (s, 1H), 8.41 (d, <i>J</i> = 6.3 Hz, 2H), 7.81–7.75 (m, 2H), 7.67 (dd, <i>J</i> = 8.0, 1.8 Hz, 1H), 7.52–7.46 (m, 1H), 3.84–3.73 (m, 1H), 2.07–2.03 (m, 2H), 1.86–1.81 (m, 2H), 1.73–1.62 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 161.9, 147.0, 144.5, 143.0, 133.7, 131.0, 130.2, 124.8, 123.9, 120.2, 119.9, 117.4, 41.3, 34.3, 25.2; HRMS calcd for C<sub>19</sub>H<sub>17</sub>NO<sub>3</sub>: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 308.1287; found, 308.1266; mp range 164–168 °C.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Methyl 4-(5-Bromofuro[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate (<b>13f</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12f</b> using 5-bromofuro[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>13e</b> (100 mg, 0.29 mmol) and <b>4</b> (95.4 mg, 0.29 mmol). After purification, methyl 4-(5-bromofuro[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>13f</b> (68 mg, 59%) was isolated as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.74 (s, 1H), 8.56 (d, <i>J</i> = 1.8 Hz, 1H), 8.48 (d, <i>J</i> = 1.6 Hz, 1H), 7.75 (d, <i>J</i> = 8.2 Hz, 2H), 7.69 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 3.85 (s, 3H), 3.71–3.58 (m, 1H), 2.08–1.96 (m, 2H), 1.87–1.76 (m, 2H), 1.73–1.61 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.9, 160.5, 147.0, 144.9, 144.8, 133.4, 132.3, 130.2, 130.0, 125.0, 124.1, 119.6, 119.5, 115.3, 52.2, 41.4, 34.2, 25.2.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Methyl 4-(5-Chlorothieno[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate (<b>14f</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12f</b> using 5-chlorothieno[2,3-<i>b</i>]pyridin-3-yl trifluoromethanesulfonate <b>14e</b> (950 mg, 2.99 mmol) and <b>4</b> (988 mg, 2.99 mmol). After purification, methyl 4-(5-chlorothieno[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>14f</b> (870 mg, 78%) was isolated as a near-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (d, <i>J</i> = 2.3 Hz, 1H), 8.27–8.24 (m, 2H), 7.79 (d, <i>J</i> = 8.0 Hz, 1H), 7.66 (d, <i>J</i> = 1.8 Hz, 1H), 7.57 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 3.87 (s, 3H), 3.71–3.61 (m, 1H), 2.08–2.01 (m, 2H), 1.85–1.75 (m, 2H), 1.69–1.62 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.0, 159.5, 147.1, 145.4, 137.1, 133.6, 131.3, 130.2, 130.0, 129.7, 128.1, 128.0, 126.6, 125.3, 52.2, 41.4, 34.3, 25.2.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 2-Cyclopentyl-4-(5-phenylfuro[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>13g</b>)</h4><div class="NLM_p">4-(5-Bromofuro[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoic acid, <b>13f</b> (48.0 mg, 0.12 mmol), phenylboronic acid (30.0 mg, 0.25 mmol), Na<sub>2</sub>CO<sub>3</sub> (40.0 mg, 0.37 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (14.0 mg, 12 μmol) were dissolved in a 4:1 mixture of dioxane (2.5 mL) and heated to 90 °C under nitrogen for 16 h. The mixture was neutralized to pH 6–7 with aq HCl and extracted with ethyl acetate (5.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude residue was purified using flash column chromatography (30% EtOAc/hexane) to obtain the methyl ester intermediate which was immediately dissolved in MeOH (3.0 mL), added aq NaOH (small excess), and heated to 70 °C for 1 h. After cooling, H<sub>2</sub>O (4.0 mL) was added and the mixture was extracted with diethyl ether (2 × 4.0 mL). The aqueous phase was acidified to pH 6 with aq HCl, extracted with CH<sub>2</sub>Cl<sub>2</sub> (6.00 mL), and concentrated to afford 2-cyclopentyl-4-(5-phenylfuro[2,3-<i>b</i>]pyridin-3-yl)benzoic acid <b>13g</b> (39 mg, 82%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.95 (s, 1H), 8.72 (s, 1H), 8.67 (d, <i>J</i> = 2.2 Hz, 1H), 8.53 (d, <i>J</i> = 2.2 Hz, 1H), 7.82 (d, <i>J</i> = 1.5 Hz, 1H), 7.81–7.79 (m, 4H), 7.56–7.50 (m, 2H), 7.47–7.41 (m, 1H), 3.86–3.77 (m, 1H), 2.11–2.01 (m, 2H), 1.87–1.79 (m, 2H), 1.73–1.62 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 161.6, 147.0, 143.8, 143.2, 137.5, 133.6, 132.9, 131.0, 130.3, 129.1, 128.2, 127.8, 127.5, 125.0, 124.1, 120.2, 117.6, 41.2, 34.4, 25.3; HRMS: calcd for C<sub>25</sub>H<sub>21</sub>NO<sub>3</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 384.1600; found, 384.1593; mp range 259–262 °C.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-Cyclopentyl-4-(5-phenylthieno[2,3-<i>b</i>]pyridin-3-yl)benzoic Acid (<b>14g</b>)</h4><div class="NLM_p">Methyl 4-(5-chlorothieno[2,3-<i>b</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>14f</b> (150 mg, 403 μmol), phenylboronic acid (73.8 mg, 605 μmol), Pd<sub>2</sub>(dba)<sub>3</sub> (18.5 mg, 20.2 μmol), dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphane (28.8 mg, 60.5 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (394 mg, 1.21 mmol) were dissolved in dioxane (3.00 mL) and H<sub>2</sub>O (0.8 mL). The solution was heated to 90 °C for 16 h. The reaction mixture was allowed to cool to rt, neutralized to pH 7 with aq HCl, and extracted with ethyl acetate (15 mL). The combined organic phases were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified by column chromatography (25% EtOAc/hexane) to afford the methyl ester intermediate. This material was dissolved in MeOH (6 mL), added aq NaOH (small excess), and heated to 70 °C for 1 h. After cooling, H<sub>2</sub>O (8 mL) was added and the mixture was extracted with diethyl ether (2 × 10.0 mL). The aqueous phase was acidified to pH 6, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL), and concentrated to afford 2-cyclopentyl-4-(5-phenylthieno[2,3-<i>b</i>]pyridin-3-yl)benzoic acid <b>14g</b> (116 mg, 72%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.94 (s, 1H), 8.96 (d, <i>J</i> = 2.1 Hz, 1H), 8.38 (d, <i>J</i> = 2.2 Hz, 1H), 8.17 (s, 1H), 7.83–7.77 (m, 3H), 7.72 (d, <i>J</i> = 1.7 Hz, 1H), 7.62 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.55–7.50 (m, 2H), 7.47–7.41 (m, 1H), 3.87–3.78 (m, 1H), 2.09–2.04 (m, 2H), 1.83–1.78 (m, 2H), 1.71–1.63 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.4, 160.6, 147.0, 145.8, 137.3, 137.3, 134.4, 132.5, 131.2, 130.4, 130.1, 129.2, 128.1, 128.1, 127.3, 126.5, 126.2, 125.3, 41.2, 34.5, 25.3; HRMS: calcd for C<sub>25</sub>H<sub>21</sub>NO<sub>2</sub>S [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 400.1371; found, 400.1365; mp range 269–273 °C.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-Cyclopentyl-4-(thieno[2,3-b]pyridin-3-yl)benzoic Acid (<b>12g</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>12f</b> using 3-bromothieno[2,3-<i>b</i>]pyridine <b>15e</b> (50.0 mg, 0.23 mmol). After purification, 2-cyclopentyl-4-(thieno[2,3-<i>b</i>]pyridin-3-yl)benzoic acid <b>12g</b> (66.0 mg, 88%) was afforded as a light-brown solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.99 (s, 1H), 8.65 (dd, <i>J</i> = 4.6, 1.6 Hz, 1H), 8.27 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 8.10 (s, 1H), 7.80 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 1.8 Hz, 1H), 7.56–7.50 (m, 2H), 3.85–3.76 (m, 1H), 2.08–2.03 (m, 2H), 1.82–1.78 (m, 2H), 1.72–1.58 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.6, 164.7, 150.1, 149.9, 140.4, 137.4, 134.5, 133.7, 133.4, 133.1, 129.5, 128.4, 128.4, 123.4, 44.4, 37.5, 28.4; HRMS: calcd for C<sub>19</sub>H<sub>17</sub>NO<sub>2</sub>S [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 324.1058; found, 324.1054; mp range 194–197 °C.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Methyl 4-(5-Bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)-2-cyclopentylbenzoate (<b>17</b>)</h4><div class="NLM_p">5-Bromo-3-iodopyrazolo[1,5-<i>a</i>]pyridine (200 mg, 0.62 mmol) and methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (194 mg, 0.60 mmol) were dissolved in a 3:1 solution of dioxane/water (4.00 mL). Pd<sub>2</sub>(dba)<sub>3</sub> (28 mg, 31 μmol), XPhos (30.0 mg, 62.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (605 mg, 1.90 mmol) were added. The mixture was stirred at rt for 16 h. The aqueous phase was removed, and the solution was concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (0–20% EtOAc/hexane) to afford ethyl 4-(5-bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>17</b> (189 mg, 76%) as a mustard yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.36 (dd, <i>J</i> = 7.3, 0.8 Hz, 1H), 8.16 (s, 1H), 7.94 (dd, <i>J</i> = 2.1, 0.8 Hz, 1H), 7.86 (dd, <i>J</i> = 8.1, 0.4 Hz, 1H), 7.57 (d, <i>J</i> = 1.8 Hz, 1H), 7.40 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 6.91 (ddd, <i>J</i> = 7.3, 2.1, 0.4 Hz, 1H), 3.93 (s, 3H), 3.91–3.83 (m, 1H), 2.24–2.11 (m, 2H), 1.93–1.81 (m, 2H), 1.81–1.72 (m, 2H), 1.71–1.61 (m, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 168.5, 148.7, 141.6, 137.6, 135.8, 130.9, 129.7, 128.4, 125.3, 123.7, 119.7, 118.7, 116.0, 112.4, 52.0, 41.7, 34.8, 25.7; HRMS: calcd for C<sub>20</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>Br [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 399.0708; found <i>m</i>/<i>z</i>, 399.0705; mp range 182–184 °C.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Methyl 4-(5-Chloropyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)-2-cyclopentylbenzoate (<b>18</b>)</h4><div class="NLM_p">5-Chloro-3-iodopyrazolo[1,5-<i>a</i>]pyrimidine (500 mg, 1.79 mmol) and methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (591 mg) were dissolved in a 10:1 mixture of dioxane/water (10.0 mL). Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub> (73.0 mg, 0.9 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (1.75 g, 5.37 mmol) were added, and the solution was heated to 80 °C for 16 h. The solution was allowed to cool to rt, the aqueous phase was removed, and the organic phase was concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (10–20% EtOAc/hexane) to afford methyl 4-(5-chloropyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)-2-cyclopentylbenzoate <b>18</b> (270 mg, 44%) as a bright-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.59 (d, <i>J</i> = 7.2 Hz, 1H), 8.48 (s, 1H), 8.12 (d, <i>J</i> = 1.5 Hz, 1H), 7.92–7.78 (m, 2H), 6.86 (d, <i>J</i> = 7.2 Hz, 1H), 3.99–3.86 (m, 4H), 2.28–2.04 (m, 2H), 1.97–1.84 (m, 2H), 1.84–1.61 (m, 4H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 168.6, 150.8, 148.6, 143.9, 136.5, 134.4, 130.6, 128.3, 124.5, 122.8, 110.1, 109.4, 51.9, 41.7, 34.8, 25.8; HRMS: calcd for C<sub>19</sub>H<sub>19</sub>N<sub>3</sub>O<sub>2</sub>Cl [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 356.1165; found <i>m</i>/<i>z</i>, 356.1161; mp range 163–166 °C.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-Cyclopentyl-4-(5-phenylpyrazolo[1,5-<i>a</i>]pyridin-3-yl)benzoic Acid (<b>19</b>)</h4><div class="NLM_p">Methyl 4-(5-bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)-2-cyclopentylbenzoate <b>17</b> (100 mg, 0.25 mmol), phenylboronic acid (37.0 mg, 0.30 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (29.0 mg, 25.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (245 mg, 0.75 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (1.50 mL) was added, and the vial was flushed with nitrogen and sealed. The mixture was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–10% EtOAc/hexane) to afford the methyl ester intermediate. The methyl ester was saponified in a solution of aq LiOH (1.00 mL, 1.00 M) and dioxane (1.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2.00 mL), and then acidified with aq HCl (1 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford the acid (68.0 mg, 71%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.83 (d, <i>J</i> = 6.9 Hz, 1H), 8.50 (s, 1H), 8.10 (s, 1H), 8.02 (s, 1H), 7.85 (d, <i>J</i> = 7.1 Hz, 2H), 7.79 (d, <i>J</i> = 7.8 Hz, 1H), 7.72 (s, 1H), 7.67 (d, <i>J</i> = 7.6 Hz, 1H), 7.53 (t, <i>J</i> = 6.9 Hz, 2H), 7.45 (t, <i>J</i> = 6.8 Hz, 1H), 7.33 (d, <i>J</i> = 6.8 Hz, 1H), 3.91–3.85 (m, 1H), 2.10–2.01 (m, 2H), 1.88–1.78 (m, 2H), 1.73–1.60 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 147.2, 141.4, 137.6, 136.9, 136.5, 135.7, 131.5, 131.4, 130.5, 129.7, 129.3, 129.1, 128.9, 128.7, 128.5, 126.7, 124.5, 123.4, 113.5, 111.9, 111.7, 41.1, 34.3, 25.3; HRMS: calcd for C<sub>25</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 383.1759; found <i>m</i>/<i>z</i>, 383.1754; mp range 153–155 °C.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-Cyclopentyl-4-(5-phenylpyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)benzoic Acid (<b>20</b>)</h4><div class="NLM_p">Methyl 4-(5-chloropyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)-2-cyclopentylbenzoate <b>18</b> (100 mg, 0.28 mmol), phenylboronic acid (41.0 mg, 0.33 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (32.0 mg, 28.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (275 mg, 0.84 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–10% EtOAc/hexane) to afford the methyl ester intermediate. The methyl ester was saponified in a solution of aq LiOH (1.00 mL, 1.00 M) and dioxane (1.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2.00 mL), and then acidified with aq HCl (1.00 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na2SO4), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford 2-cyclopentyl-4-(5-phenylpyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)benzoic acid <b>20</b> (87.0 mg, 81%) as a bright-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.94 (s, 1H), 9.24 (dd, <i>J</i> = 7.6, 2.8 Hz, 1H), 8.87 (s, 1H), 8.65 (s, 1H), 8.39–8.28 (m, 2H), 7.95 (d, <i>J</i> = 8.1 Hz, 1H), 7.84–7.72 (m, 2H), 7.61–7.55 (m, 3H), 3.98–3.91 (m, 1H), 2.15–2.09 (m, 2H), 1.93–1.86 (m, 2H), 1.77–1.68 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): 169.2, 155.9, 147.3, 144.3, 143.6, 137.0, 136.6, 135.3, 131.0, 130.1, 129.0 (2 × ArCH), 128.4, 127.2 (2 × ArCH), 123.8, 122.1, 108.2, 105.9, 41.0, 34.7, 25.6; HRMS: calcd for C<sub>24</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 384.1712; found, <i>m</i>/<i>z</i> 384.1707; mp range 210–214 °C.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 2-Cyclopentyl-4-(pyrazolo[1,5-<i>a</i>]pyridin-3-yl)benzoic Acid (<b>21</b>)</h4><div class="NLM_p">3-Bromopyrazolo[1,5-<i>a</i>]pyridine (100 mg, 0.50 mmol), 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (193 mg, 0.61 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (23.0 mg, 25.0 μmol), XPhos (24.0 mg, 51.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (579 mg, 1.80 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (3.00 mL) was added, and the vial was flushed with nitrogen and sealed. The mixture was heated to 120 °C for 16 h. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The mixture was acidified to pH 4 with 1.00 M aq HCl solution. The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (50–100% EtOAc/hexane) to afford 2-cyclopentyl-4-(pyrazolo[1,5-<i>a</i>]pyridin-3-yl)benzoic acid <b>21</b> (64.0 mg, 41%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.78 (br s, 1H), 8.76 (d, <i>J</i> = 7.0 Hz, 1H), 8.47 (s, 1H), 7.94 (d, <i>J</i> = 9.0 Hz, 1H), 7.77 (d, <i>J</i> = 8.1 Hz, 1H), 7.67 (d, <i>J</i> = 1.6 Hz, 1H), 7.57 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.39 (ddd, <i>J</i> = 8.9, 6.7, 1.0 Hz, 1H), 6.99 (td, <i>J</i> = 6.9, 1.1 Hz, 1H), 3.90–3.79 (m, 1H), 2.09–1.97 (m, 2H), 1.88–1.76 (m, 2H), 1.73–1.58 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 147.1, 140.8, 136.3, 135.8, 130.5, 129.6, 128.7, 125.4, 124.2, 123.2, 117.1, 112.7, 110.9, 41.2, 34.3, 25.2; HRMS (ESI): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 307.1446; found <i>m</i>/<i>z</i>, 307.1439; mp range 167–169 °C.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 2-Cyclopentyl-4-(pyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)benzoic Acid (<b>22</b>)</h4><div class="NLM_p">3-Bromopyrazolo[1,5-<i>a</i>]pyrimidine (150 mg, 0.75 mmol), 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (287 mg, 0.90 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (87.0 mg, 75.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (864 mg, 2.65 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The mixture was acidified to pH 4 with 1 M aq HCl solution. The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (50–100% EtOAc/hexane) to afford 2-cyclopentyl-4-(pyrazolo[1,5-<i>a</i>]pyrimidin-3-yl)benzoic acid <b>22</b> (137 mg, 59%) as a bright-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.74 (br s, 1H), 9.24–9.12 (d, <i>J</i> = 6.0 Hz, 1H), 8.87 (d, <i>J</i> = 2.9 Hz, 1H), 8.72 (s, 1H), 8.25 (s, 1H), 8.03 (d, <i>J</i> = 8.2 Hz, 1H), 7.75 (dd, <i>J</i> = 8.1, 2.8 Hz, 1H), 7.14 (dt, <i>J</i> = 7.4, 3.5 Hz, 1H), 3.89–3.83 (m, 1H), 2.08–2.01 (m, 2H), 1.88–1.82 (m, 2H), 1.71–1.63 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 150.9, 146.9, 144.5, 143.2, 136.6, 135.0, 130.1, 128.6, 123.6, 122.5, 109.0, 108.2, 41.2, 34.3, 25.2; HRMS: calcd for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 308.1399; found <i>m</i>/<i>z</i>, 308.1393; mp range 196–199 °C.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Methyl 4-(6-Chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)-2-cyclopentylbenzoate (<b>24</b>)</h4><div class="NLM_p last">3-Bromo-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (924 mg, 3.96 mmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (1.31 g, 3.96 mmol), K<sub>2</sub>CO<sub>3</sub> (1.64 g, 11.9 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (320 mg, 0.28 mmol) in a mixture of dioxane (12.0 mL) and water (3.00 mL) were degassed and put under a N<sub>2</sub> atmosphere. The reaction mixture was heated to 90 °C and stirred for 16 h. The reaction mixture was diluted with aq HCl (50.0 mL) to pH 6 and extracted with EtOAc (2 × 75.0 mL), and the combined organic phases were concentrated <i>in vacuo</i>. The crude residue was purified <i>via</i> column chromatography (35% EtOAc/hexane) to afford methyl 4-(6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)-2-cyclopentylbenzoate <b>24</b> (1.20 g, 85%) as a grayish solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 9.7 Hz, 1H), 8.47 (d, <i>J</i> = 1.7 Hz, 1H), 8.21 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.87 (d, <i>J</i> = 8.2 Hz, 1H), 7.59 (d, <i>J</i> = 2.8 Hz, 1H), 3.88 (s, 3H), 3.77–3.66 (m, 1H), 2.14–2.03 (m, 2H), 1.91–1.79 (m, 2H), 1.75–1.58 (m, 4H). LCMS: [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 356.2.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 2-Cyclopentyl-4-(6-phenyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)benzoic Acid (<b>25</b>)</h4><div class="NLM_p">Following the procedure for <b>14g</b> using 4-(6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)-2-cyclopentylbenzoate <b>24</b> (220 mg, 0.62 mmol). After purification, cyclopentyl-4-(6-phenyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)benzoic acid <b>25</b> (200 mg, 84%) was afforded as a near-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.74 (d, <i>J</i> = 1.7 Hz, 1H), 8.58 (d, <i>J</i> = 9.8 Hz, 1H), 8.32 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 8.21–8.18 (m, 2H), 8.07 (d, <i>J</i> = 9.8 Hz, 1H), 7.89 (d, <i>J</i> = 8.2 Hz, 1H), 7.65–7.59 (m, 3H), 3.91–3.82 (m, 1H), 2.15–2.11 (m, 2H), 1.88–1.83 (m, 2H), 1.72–1.66 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 153.5, 146.6, 146.4, 144.5, 134.1, 133.0, 131.1, 129.7, 129.1, 128.7, 127.4, 125.6, 125.2, 124.2, 119.9, 41.1, 34.7, 25.6; HRMS: calcd for C<sub>23</sub>H<sub>20</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 385.1665; found, 385.1659; mp range 228–231 °C.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 4-([1,2,4]Triazolo[4,3-<i>b</i>]pyridazin-3-yl)-2-cyclopentylbenzoic Acid (<b>26</b>)</h4><div class="NLM_p">3-Bromo-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (150 mg, 754 μmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (373 mg, 1.13 mmol), XPhos (53.9 mg, 113 μmol), Cs<sub>2</sub>CO<sub>3</sub> (516 mg, 1.58 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (34.5 mg, 0.04 mmol) in a 3:1 mixture of dioxane/water (4.00 mL) were heated to 90 °C under a N<sub>2</sub> atmosphere for 16 h. The reaction mixture was diluted with aq HCl (25.0 mL, pH 6) and extracted with EtOAc (2 × 50.0 mL); the organic phases were combined, concentrated, and purified using flash column chromatography (40% EtOAc/hexane) to afford 4-([1,2,4]triazolo[4,3-<i>b</i>]pyridazin-3-yl)-2-cyclopentylbenzoic acid <b>26</b> (178.0 mg, 77%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.79 (dd, <i>J</i> = 4.3, 1.5 Hz, 1H), 8.50–8.46 (m, 2H), 8.29 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.86 (d, <i>J</i> = 8.2 Hz, 1H), 7.44 (dd, <i>J</i> = 9.4, 4.3 Hz, 1H), 3.86–3.75 (m, 1H), 2.14–2.04 (m, 2H), 1.89–1.79 (m, 2H), 1.74–1.58 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 146.8, 146.4, 146.3, 145.1, 133.1, 129.6, 128.6, 125.4, 125.3, 124.1, 120.9, 41.3, 34.4, 25.3; HRMS: calcd for C<sub>17</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 309.1352; found, 309.1346; mp range 208–211 °C.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 4-Chloro-1-methyl-2-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>28</b>)</h4><div class="NLM_p">4-Chloro-2-iodo-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (75.0 mg, 0.25 mmol), phenylboronic acid (31.0 mg, 0.25 mmol), Cs<sub>2</sub>CO<sub>3</sub> (251 mg, 0.77 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (30.0 mg, 0.02 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (10% EtOAc/hexane) to afford 4-chloro-1-methyl-2-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>28</b> (58.0 mg, 93%) as a white-gray solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.23 (d, <i>J</i> = 5.2 Hz, 1H), 7.63–7.40 (m, 5H), 7.12 (d, <i>J</i> = 5.2 Hz, 1H), 6.63 (s, 1H), 3.89 (s, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 149.6, 142.6, 142.5, 135.3, 131.7, 129.1 (2 × ArCH), 128.7 (2 × ArCH), 128.6, 119.9, 116.2, 97.9, 30.3; mp range 113–116 °C.</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 2-Cyclopentyl-4-(1-methyl-2-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzoic Acid (<b>29</b>)</h4><div class="NLM_p">4-Chloro-1-methyl-2-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>28</b> (60.0 mg, 0.25 mmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (98.0 mg, 0.30 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (29.0 mg, 25.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (242 mg, 0.74 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (2.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–10% EtOAc/hexane) to afford the methyl-substituted ester intermediate. The methyl ester was saponified in a solution of aq LiOH (1.00 mL, 1.00 M) and dioxane (1.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2.00 mL), and then acidified with aq HCl (1.00 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford 2-cyclopentyl-4-(1-methyl-2-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzoic acid <b>29</b> (79.0 mg, 80%) as a brown-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.40 (d, <i>J</i> = 4.9 Hz, 1H), 7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.80 (d, <i>J</i> = 1.7 Hz, 1H), 7.71 (t, <i>J</i> = 7.7 Hz, 3H), 7.56 (t, <i>J</i> = 7.5 Hz, 2H), 7.50 (t, <i>J</i> = 7.4 Hz, 1H), 7.33 (d, <i>J</i> = 4.9 Hz, 1H), 6.73 (s, 1H), 3.89 (s, 3H), 3.86–3.76 (m, 1H), 2.12–2.04 (m, 2H), 1.84–1.77 (m, 2H), 1.70–1.59 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 149.5, 146.7, 143.0, 142.2, 140.8, 139.4, 131.8, 131.6, 130.0, 128.9 (2 × ArCH), 128.8 (2 × ArCH), 128.6, 126.5, 125.5, 117.5, 115.1, 98.1, 41.2, 34.4, 29.9, 25.3; HRMS: calcd for C<sub>26</sub>H<sub>25</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 397.1916; found <i>m</i>/<i>z</i>, 397.1908; mp range 187–189 °C.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 2-Cyclopentyl-4-(1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzoic Acid (<b>30</b>)</h4><div class="NLM_p">4-Chloro-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (75.0 mg, 0.45 mmol), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (180 mg, 0.54 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (52.0 mg, 45.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (440 mg, 1.40 mmol) were loaded into a microwave vial. A 3:1 mixture of dioxane/water (3.00 mL) was added, and the vial was flushed with nitrogen and capped. The solution was irradiated at 120 °C for 30 min. Once cooled, the solution was diluted with EtOAc (2.00 mL) and water (2.00 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (5–10% EtOAc/hexane) to afford the methyl ester intermediate. The methyl ester was saponified in a solution of aq LiOH (1.00 mL, 1.00 M) and dioxane (1.00 mL) at 100 °C for 16 h. The solution was cooled to rt, diluted with water (2.00 mL), and then acidified with aq HCl (1.00 M) until pH 4. The solution was extracted with EtOAc (2 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The solid was washed with cold water (5.00 mL), followed by hexane (10.0 mL), and dried to afford 2-cyclopentyl-4-(1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzoic acid <b>30</b> (97.0 mg, 67%) as a pale-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.01 (s, 1H), 8.36 (s, 1H), 7.81 (d, <i>J</i> = 7.6 Hz, 1H), 7.76 (s, 1H), 7.67–7.61 (m, 2H), 7.27 (s, 1H), 6.57 (s, 1H), 3.87 (s, 3H), 3.80 (dd, <i>J</i> = 19.7, 12.5 Hz, 1H), 2.13–2.00 (m, 2H), 1.87–1.74 (m, 2H), 1.71–1.56 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 147.9, 146.6, 142.6, 140.8, 139.9, 131.7, 131.0, 129.9, 126.5, 125.4, 117.5, 114.3, 97.8, 41.1, 34.4, 31.2, 25.3; HRMS: calcd for C<sub>20</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 321.1603; found <i>m</i>/<i>z</i>, 321.1595; mp range 201–203 °C.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Methyl 2-Cyclopentyl-4-(2,3-diaminopyridin-4-yl)benzoate (<b>32</b>)</h4><div class="NLM_p">4-Chloropyridine-2,3-diamine (103 mg, 0.71 mmol) and methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (250 mg, 0.75 mmol) were dissolved in a 10:1 mixture of dioxane/H<sub>2</sub>O (10.0 mL). Pd<sub>2</sub>(dba)<sub>3</sub> (69.0 mg, 75 μmol), XPhos (72.0 mg, 150 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (740 mg, 2.30 mmol) were added. The solution was heated to 100 °C for 16 h. Once cooled to rt, the aqueous phase was removed and the organic phase was concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (25–75% EtOAc/hexane) to afford methyl 2-cyclopentyl-4-(2,3-diaminopyridin-4-yl)benzoate <b>32</b> (169 mg, 71%) as a dark-brown solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.66 (d, <i>J</i> = 5.3 Hz, 1H), 7.47 (d, <i>J</i> = 1.8 Hz, 1H), 7.31–7.23 (dd, 1H), 6.61 (d, <i>J</i> = 5.3 Hz, 1H), 4.63 (br s, 2H), 3.93 (s, 3H), 3.86–3.71 (m, 1H), 3.61 (br s, 2H), 2.19–2.07 (m, 2H), 1.90–1.73 (m, 2H), 1.75–1.65 (m, 2H), 1.65–1.50 (m, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 168.5, 149.1, 148.4, 140.6, 136.8, 133.8, 130.4, 130.4, 127.2, 126.6, 125.5, 116.3, 52.2, 41.7, 35.0, 25.7; HRMS: calcd for C<sub>18</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 312.1712; found <i>m</i>/<i>z</i>, 312.1704; mp range 125–129 °C.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 2-Cyclopentyl-4-(3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-7-yl)benzoic Acid (<b>33</b>)</h4><div class="NLM_p">Methyl 2-cyclopentyl-4-(2,3-diaminopyridin-4-yl)benzoate <b>32</b> (100 mg, 0.32 mmol) was dissolved in MeOH (1 mL). Sulfamic acid (3.00 mg, 32.0 μmol) and trimethyl orthoformate (85.0 mg, 0.80 mmol) were added, and the mixture was stirred at rt for 1 h. The solvent was removed <i>in vacuo</i>, and the residue was dissolved in EtOAc (10.0 mL) and water (10.0 mL). The phases were separated, and the aqueous phase was extracted with EtOAc (2 × 10.0 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude residue was purified by column chromatography (50–100% EtOAc/hexane). The isolated methyl ester was dissolved in dioxane (1.00 mL) and aq LiOH (1.00 mL of a 1.00 M solution). The solution was heated to 100 °C for 16 h, cooled, and diluted with EtOAc (3.00 mL) and water (3.00 mL). The mixture was acidified to pH 4 with 1.00 M aq HCl solution. The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to afford 2-cyclopentyl-4-(3H-imidazo[4,5-<i>b</i>]pyridin-7-yl)benzoic acid <b>33</b> (45 mg, 46%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.23 (s, 1H), 8.51 (s, 1H), 8.45 (s, 1H), 8.41 (d, <i>J</i> = 4.7 Hz, 1H), 8.11 (d, <i>J</i> = 8.1 Hz, 1H), 7.79 (d, <i>J</i> = 8.1 Hz, 1H), 7.59 (d, <i>J</i> = 4.7 Hz, 1H), 3.80 (t, <i>J</i> = 8.7 Hz, 1H), 2.10–2.03 (br m, 2H), 1.87–1.73 (br m, 2H), 1.67 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.9, 169.4, 148.8, 146.1, 144.2, 143.8, 138.2, 136.5, 132.1, 131.8, 129.3, 127.6, 125.8, 115.3, 41.3, 34.4, 25.2; HRMS: calcd for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 308.1399; found, <i>m</i>/<i>z</i> 308.1392; mp range 183–186 °C.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 2-Cyclopentyl-4-(2-phenyl-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-7-yl)benzoic Acid (<b>34</b>)</h4><div class="NLM_p">Benzoic acid (47.0 mg, 0.38 mmol) and CDI (63.0 mg, 0.38 mmol) were loaded into a microwave vial and dissolved in dimethylformamide (DMF) (1.0 mL). The solution was cooled to 0 °C and stirred for 30 min. Methyl 2-cyclopentyl-4-(2,3-diaminopyridin-4-yl)benzoate <b>4</b> (100 mg, 0.32 mmol) was added, and the vial was sealed. The mixture was heated to 200 °C for 10 min and allowed to cool to rt, and water (3.00 mL) was added, causing a precipitate to form. The precipitate was filtered and washed with cold water (10.0 mL) and hexane (10.0 mL). The precipitate was dissolved in dioxane (1.00 mL) and aq LiOH (1.00 mL of a 1.00 M solution). The solution was heated to 100 °C for 16 h and then diluted with EtOAc (3.00 mL) and water (3.00 mL). The mixture was acidified to pH 4 with 1 M aq HCl solution. The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 3.00 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (50–100% EtOAc/hexane) to afford 2-cyclopentyl-4-(2-phenyl-3H-imidazo[4,5-<i>b</i>]pyridin-7-yl)benzoic acid <b>34</b> (93.0 mg, 76%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55–8.44 (m, 2H), 8.33 (d, <i>J</i> = 6.3 Hz, 2H), 8.05 (d, <i>J</i> = 6.7 Hz, 1H), 7.84 (d, <i>J</i> = 7.8 Hz, 1H), 7.67 (s, 1H), 7.61 (d, <i>J</i> = 6.6 Hz, 3H), 3.93–3.79 (m, 1H), 2.18–2.05 (m, 2H), 1.93–1.83 (m, 2H), 1.82–1.66 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.4, 150.2, 146.6, 137.7, 132.4, 131.1, 129.5, 129.1, 128.0, 127.1, 125.6, 116.2, 41.1, 39.5, 34.5, 25.4; HRMS: calcd for C<sub>24</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 384.1712; found <i>m</i>/<i>z</i>, 384.1704; mp range 196–199 °C.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 4-Chloro-6-iodothieno[3,2-<i>d</i>]pyrimidine (<b>35b</b>)</h4><div class="NLM_p">An oven-dried 250 mL three-neck round-bottom flask provided with a pressure-equalizing addition funnel was cooled under N<sub>2</sub>. 4-Chlorothieno[3,2-<i>d</i>]pyrimidine <b>35a</b> (3.40 g, 20.0 mmol) was added under positive N<sub>2</sub> pressure, followed by the addition of anhydrous THF (65.0 mL). The solution was cooled down to −78 °C; LDA (12.0 mL, 24.0 mmol) was added dropwise over 5 min, and the mixture was stirred at −78 °C for 20 min. A solution of I<sub>2</sub> (6.08 g, 24.0 mmol) in anhydrous THF (15.0 mL) was added dropwise over 10 min using the equalizing addition funnel. The resulting mixture was stirred at −78 °C for 30 min, allowed to warm up to rt over 1 h, and then stirred for 16 h. The reaction was quenched with saturated NH<sub>4</sub>Cl/ice and diluted with CHCl<sub>3</sub> (200 mL). The layers were separated, and the aqueous layer was extracted with CHCl<sub>3</sub> (2 × 50.0 mL). The combined organics were rinsed with aq Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (2 × 25.0 mL) and sat. NaCl solution (2 × 25.0 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (0–100% hexane/EtOAc), affording 4-chloro-6-iodothieno[3,2-<i>d</i>]pyrimidine <b>35b</b> (4.60 g, 68%) as a clear brown solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (s, 1H), 7.84 (s, 1H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.6, 154.9, 154.8, 151.7, 134.8, 134.2, 97.4.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 4-Chloro-6-phenylthieno[3,2-<i>d</i>]pyrimidine (<b>36</b>)</h4><div class="NLM_p">4-Chloro-6-iodothieno[3,2-<i>d</i>]pyrimidine <b>35b</b> (1.01 mmol, 300 mg), phenylboronic acid (1.01 mmol, 125 mg), NaHCO<sub>3</sub> (1.01 mmol, 85.0 mg), CsF (1.01 mmol, 155 mg), and Pd(Ph<sub>3</sub>)<sub>4</sub> (35.0 mg, 0.03 mmol) were dissolved in a 3:1 mixture of dioxane/water (7.00 mL). The reaction mixture was stirred at reflux for 3 h. The volatiles were removed under reduced pressure, and the crude was purified <i>via</i> column chromatography (hexane/EtOAc 0–30%) to afford 4-chloro-6-phenylthieno[3,2-<i>d</i>]pyrimidine <b>36</b> (175 mg, 70%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.04 (s, 1H), 8.26 (s, 1H), 8.05–7.97 (m, 2H), 7.62–7.52 (m, 3H).</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 2-Cyclopentyl-4-(6-phenylthieno[3,2-<i>d</i>]pyrimidin-4-yl)benzoic Acid (<b>37</b>)</h4><div class="NLM_p">A 5 mL microwave vial was loaded with pyrimidine <b>36</b> (55.0 mg, 0.22 mmol) and 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (70.0 mg, 0.22 mmol), Cs<sub>2</sub>CO<sub>3</sub> (0.45 mmol, 150 mg), and Pd(Ph<sub>3</sub>)<sub>4</sub> (5 mol %, 13.0 mg). The solids were dissolved in a solution of 1,4-dioxane/water (3:1, 1.50 mL). The vial was sealed, and the reaction mixture was irradiated at 125 °C for 15 min. The volatiles were removed <i>in vacuo</i>, and remaining solids were suspended in ice/water before the pH was adjusted to 2–3 with 2 M HCl. After an initial purification over silica gel, the compound was further purified by recrystallization from diethyl ether, affording 2-cyclopentyl-4-(6-phenylthieno[3,2-<i>d</i>]pyrimidin-4-yl)benzoic acid <b>37</b> (12.0 mg, 15%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.25 (s, 1H), 9.30 (s, 1H), 8.23 (s, 2H), 8.07 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 8.00 (dd, <i>J</i> = 7.1, 2.5 Hz, 2H), 7.89 (d, <i>J</i> = 8.1 Hz, 1H), 7.62–7.52 (m, 3H), 3.80 (p, <i>J</i> = 8.3 Hz, 1H), 2.18–2.06 (m, 2H), 1.92–1.78 (m, 2H), 1.75–1.60 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 162.98, 157.7, 155.2, 153.6, 146.6, 138.9, 134.4, 131.9, 130.6, 129.8, 129.5, 127.4, 126.8, 126.6, 125.4, 120.3, 41.3, 34.5, 25.3; HRMS calcd for C<sub>24</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup>, 401.1324; found, 401.1329; mp range >250 °C.</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 2-Cyclopentyl-4-(thieno[3,2-<i>d</i>]pyrimidin-4-yl)benzoic Acid (<b>38</b>)</h4><div class="NLM_p">A 5 mL microwave vial was loaded with 4-chlorothieno[3,2-<i>d</i>]pyrimidine <b>35a</b> (0.41 mmol, 70.0 mg), 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (0.41 mmol, 130 mg), Cs<sub>2</sub>CO<sub>3</sub> (0.82 mmol, 270 mg), and Pd(Ph<sub>3</sub>)<sub>4</sub> (5 mol %, 25.0 mg). The solids were dissolved in a solution of 1,4-dioxane/water (3:1, 1.50 mL). The vial was sealed, and the reaction mixture was irradiated at 125 °C for 15 min. The volatiles were removed <i>in vacuo</i>, and the remaining solids were suspended with ice/water before the pH was adjusted to 2–3 with 2 M HCl(aq). The crude was purified <i>via</i> column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH/HOAc, 95:4:1–90:9:1), affording 2-cyclopentyl-4-(thieno[3,2-<i>d</i>]pyrimidin-4-yl)benzoic acid <b>38</b> (70.0 mg, 48%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.24 (s, 1H), 9.31 (s, 1H), 8.60 (d, <i>J</i> = 5.5 Hz, 1H), 8.22 (d, <i>J</i> = 1.7 Hz, 1H), 8.04 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.88 (d, <i>J</i> = 8.1 Hz, 1H), 7.76 (d, <i>J</i> = 5.5 Hz, 1H), 3.80 (p, <i>J</i> = 8.4, 7.9 Hz, 1H), 2.16–2.06 (m, 2H), 1.89–1.79 (m, 2H), 1.74–1.60 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 162.1, 158.3, 154.7, 146.6, 139.0, 138.9, 134.4, 129.8, 127.6, 126.6, 125.5, 124.5, 41.2, 34.5, 25.4; HRMS calcd for C<sub>18</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup>, 325.1011; found, 325.1011; mp range 205–209 °C.</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Methyl 4-(6-Bromothieno[2,3-<i>d</i>]pyrimidin-4-yl)-2-cyclopentylbenzoate (<b>40</b>)</h4><div class="NLM_p">A 5 mL microwave vial was loaded with 6-bromo-4-chlorothieno[2,3-<i>d</i>]pyrimidine <b>39b</b> (0.40 mmol, 100 mg), methyl 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (0.36 mmol, 120 mg), Cs<sub>2</sub>CO<sub>3</sub> (0.60 mmol, 195 mg), and Pd(Ph<sub>3</sub>)<sub>4</sub> (5 mol %, 23.0 mg). The solids were dissolved in a solution of 1,4-dioxane/water (3:1, 1.5 mL). The vial was sealed, and the reaction mixture was irradiated at 145 °C for 15 min. The volatiles were removed <i>in vacuo</i>, and the crude was purified <i>via</i> column chromatography (0–5% hexane/EtOAc) to afford methyl 4-(6-bromothieno[2,3-<i>d</i>]pyrimidin-4-yl)-2-cyclopentylbenzoate <b>40</b> (50.0 mg, 30%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.95 (s, 1H), 8.22 (s, 1H), 7.98 (d, <i>J</i> = 1.9 Hz, 1H), 7.83–7.76 (m, 2H), 3.86 (s, 3H), 3.63–3.57 (m, 1H), 2.08–2.00 (m, 2H), 1.90–1.81 (m, 2H), 1.75–1.63 (m, 4H).</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 2-Cyclopentyl-4-(6-phenylthieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoic Acid (<b>41</b>)</h4><div class="NLM_p">Methyl 4-(6-bromothieno[2,3-<i>d</i>]pyrimidin-4-yl)-2-cyclopentylbenzoate <b>40</b> (0.11 mmol, 45.0 mg), phenylboronic acid (0.12 mmol, 15.0 mg), Cs<sub>2</sub>CO<sub>3</sub> (0.38 mmol, 52.0 mg), and Pd(Ph<sub>3</sub>)<sub>4</sub> (5 mol %, 6.00 mg) were dissolved in a solution of 1,4-dioxane/water (3:1, 2.00 mL). The reaction mixture was stirred at reflux for 3 h. The volatiles were removed <i>in vacuo</i>, and the crude was purified <i>via</i> column chromatography (0–10% EtOAc/hexane). The material was further purified using preparative TLC silica plates, affording methyl 2-cyclopentyl-4-(6-phenylthieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoate (25 mg, 55%) as a white solid. The methyl ester was saponified as follows: methyl 2-cyclopentyl-4-(6-phenylthieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoate (0.06 mmol, 25.0 mg) was dissolved in 1,4-dioxane (2.50 mL). LiOH (0.30 mmol, 7.00 mg) was dissolved in H<sub>2</sub>O (0.50 mL) and added to the solution. This mixture was stirred at 80 °C for 4 h. Solvents were removed <i>in vacuo</i>; the material was suspended in ice/water, and the pH was adjusted to 2–3 with 2 M HCl. The resulting solid was filtrated, thoroughly rinsed with water, and air- and vacuum-dried, affording 2-cyclopentyl-4-(6-phenylthieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoic acid <b>41</b> (17.0 mg, 70%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.11 (s, 1H), 9.16 (s, 1H), 8.16 (s, 1H), 8.13–8.06 (m, 2H), 7.83 (dd, <i>J</i> = 4.3, 2.4 Hz, 2H), 7.78–7.74 (m, 1H), 7.68–7.63 (m, 3H), 3.68–3.62 (m, 1H), 2.10–1.98 (m, 2H), 1.87–1.79 (m, 2H), 1.70–1.62 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.1, 168.8, 159.9, 153.4, 146.8, 142.8, 136.9, 134.9, 130.7, 130.2, 129.4, 129.0, 128.9, 126.7, 124.9, 124.4, 117.7, 41.4, 34.2, 25.1; HRMS (HESI) calcd for C<sub>24</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup>, 401.1324; found, 401.1330; mp range >250 °C.</div></div><div id="sec4_2_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 2-Cyclopentyl-4-(thieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoic Acid (<b>42</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>38</b> using 4-chlorothieno[2,3-<i>d</i>]pyrimidine <b>35a</b> (70.0 mg, 0.41 mmol) and 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoic acid <b>3</b> (130 mg, 0.41 mmol). After purification, 2-cyclopentyl-4-(thieno[2,3-<i>d</i>]pyrimidin-4-yl)benzoic acid <b>42</b> (72.0 mg, 55%) was afforded as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.19 (s, 1H), 8.09 (d, <i>J</i> = 6.1 Hz, 1H), 7.98 (d, <i>J</i> = 1.6 Hz, 1H), 7.87–7.79 (m, 2H), 7.68 (d, <i>J</i> = 6.1 Hz, 1H), 3.83–3.70 (m, 1H), 2.12–2.02 (m, 2H), 1.86–1.74 (m, 2H), 1.65–1.56 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 169.3, 159.2, 153.1, 146.2, 139.4, 133.9, 129.5, 129.3, 127.4, 127.4, 126.4, 120.7, 41.3, 34.4, 25.3; HRMS (HESI) calcd for C<sub>18</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup>, 325.1011; found, 325.1013; mp range 146–149.5 °C.</div></div><div id="sec4_2_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Methyl 4-(6-Chloroquinolin-4-yl)-2-cyclopentylbenzoate (<b>44</b>)</h4><div class="NLM_p">4,6-Dichloroquinoline (1.00 g, 5.05 mmol) and 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (1.58 g, 4.80 mmol) were dissolved in a 10:1 solution of dioxane/water (27.5 mL). Pd<sub>2</sub>(dba)<sub>3</sub> (231 mg, 0.25 mmol), XPhos (240 mg, 0.51 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (4.94 g, 15.2 mmol) were added, and the mixture was stirred for 16 h at 25 °C. The solution was diluted with EtOAc (20.0 mL) and water (25.0 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 20.0 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (10–30% EtOAc/hexane) to afford methyl 4-(6-chloroquinolin-4-yl)-2-cyclopentylbenzoate <b>44</b> (1.54 g, 83%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.96 (d, <i>J</i> = 4.4 Hz, 1H), 8.15 (d, <i>J</i> = 9.0 Hz, 1H), 7.90 (d, <i>J</i> = 8.0 Hz, 1H), 7.84 (d, <i>J</i> = 2.3 Hz, 1H), 7.69 (dd, <i>J</i> = 9.0, 2.3 Hz, 1H), 7.51 (d, <i>J</i> = 1.6 Hz, 1H), 7.42–7.33 (m, 2H), 3.97 (s, 3H), 3.89–3.78 (m, 1H), 2.23–2.12 (m, 2H), 1.86–1.69 (m, 4H), 1.63 (ddd, <i>J</i> = 16.2, 12.9, 8.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 168.5, 145.0, 148.1, 147.2, 146.9, 140.4, 133.0, 131.5, 131.1, 130.5, 130.0, 128.1, 127.2, 126.4, 124.5, 121.8, 77.0, 52.3, 41.8, 35.9, 25.7; HRMS: calcd for C<sub>22</sub>H<sub>21</sub>ClNO<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 366.1260; found <i>m</i>/<i>z</i>, 366.1246; mp range 137–139 °C.</div></div><div id="sec4_2_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 2-Cyclopentyl-4-(6-phenylquinolin-4-yl)benzoic Acid (<b>45</b>)</h4><div class="NLM_p">Methyl 4-(6-chloroquinolin-4-yl)-2-cyclopentylbenzoate <b>44</b> (100 mg, 0.27 mmol) and phenylboronic acid (66.0 mg, 0.32 mmol) were dissolved in a 10:1 solution of dioxane/water (12 mL). Pd<sub>2</sub>(dba)<sub>3</sub> (25 mg, 0.03 mmol), XPhos (29.0 mg, 0.06 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (264 mg, 0.81 mmol) were added, and the solution was heated to 80 °C for 16 h. Once cooled, the solution was diluted with EtOAc (10 mL) and water (10 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 10 mL). The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography (10–30% EtOAc/hexane) to afford methyl 2-cyclopentyl-4-(6-phenylquinolin-4-yl)benzoate as an intermediate. The intermediate methyl ester was dissolved in dioxane (1 mL) and aq LiOH (1 mL of a 1 M solution). The solution was heated to 100 °C for 16 h and then acidified to pH 4 with 1 M aq HCl, causing a precipitate to form. The solid was collected by filtration, washed with cold water (4 mL) and Et<sub>2</sub>O (10 mL), and dried under vacuum to afford 2-cyclopentyl-4-(6-phenylquinolin-4-yl)benzoic acid <b>45</b> (81 mg, 76%) as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.15 (d, <i>J</i> = 4.8 Hz, 1H), 8.39 (d, <i>J</i> = 8.6 Hz, 1H), 8.32 (d, <i>J</i> = 8.7 Hz, 1H), 8.13 (d, <i>J</i> = 1.7 Hz, 1H), 7.87 (d, <i>J</i> = 7.9 Hz, 1H), 7.84–7.79 (m, 1H), 7.74–7.69 (m, 3H), 7.59 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.51 (t, <i>J</i> = 7.4 Hz, 2H), 7.47–7.41 (m, 1H), 3.88–3.72 (m, 1H), 2.14–2.01 (m, 2H), 1.84–1.71 (m, 2H), 1.71–1.50 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO): δ 169.3, 146.4, 145.2, 139.8, 138.9, 138.8, 132.8, 130.7, 129.6, 129.3, 129.2, 128.3, 128.1, 127.1, 126.8, 126.2, 123.0, 122.3, 119.4, 99.5, 41.2, 34.5, 25.3; HRMS: calcd for C<sub>27</sub>H<sub>24</sub>NO<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 394.1807; found <i>m</i>/<i>z</i>, 394.1786; mp range 164–167 °C.</div></div><div id="sec4_2_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 2-Cyclopentyl-4-(quinolin-4-yl)benzoic Acid (<b>46</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>21</b> using 4-chloroquinoline (100 mg, 611 μmol). After purification, 2-cyclopentyl-4-(quinolin-4-yl)benzoic acid <b>46</b> (l60 mg, 83%) was afforded as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.09 (s, 1H), 8.97 (d, <i>J</i> = 4.4 Hz, 1H), 8.14–8.11 (m, 1H), 7.8–7.78 (m, 3H), 7.62 (ddd, <i>J</i> = 8.3, 6.9, 1.3 Hz, 1H), 7.56 (d, <i>J</i> = 1.7 Hz, 1H), 7.50 (d, <i>J</i> = 4.4 Hz, 1H), 7.44 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 3.85–3.73 (m, 1H), 2.11–2.00 (m, 2H), 1.81–1.71 (m, 2H), 1.70–1.54 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.4, 150.2, 148.1, 146.8, 146.6, 140.0, 132.1, 129.6, 129.6, 129.5, 127.8, 127.2, 126.7, 125.7, 125.2, 121.5, 41.2, 34.5, 25.3; HRMS: calcd for C<sub>21</sub>H<sub>19</sub>NO<sub>2</sub> [M – H]<sup>−</sup><i>m</i>/<i>z</i>, 316.1338; found, 316.1339; mp range 259–263 °C.</div></div><div id="sec4_2_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Methyl 2-Cyclopentyl-4-(6-hydroxyquinazolin-4-yl)benzoate (<b>48</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>32</b> using 4-chloroquinazolin-6-ol <b>47b</b> (50.0 mg, 0.28 mmol). After purification, methyl 2-cyclopentyl-4-(6-hydroxyquinazolin-4-yl)benzoate <b>48</b> (63.0 mg, 65%) was afforded as a thick oil.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.42 (s, 1H), 8.16 (d, <i>J</i> = 9.0 Hz, 1H), 8.10 (dd, <i>J</i> = 9.0, 2.3 Hz, 1H), 7.99–7.98 (m, 1H), 7.88–7.84 (m, 3H), 7.71 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 3.70–3.62 (m, 1H), 2.13–2.01 (m, 2H), 1.81–1.75 (m, 2H), 1.72–1.57 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.9, 166.2, 154.7, 149.0, 146.4, 138.9, 134.8, 132.6, 132.3, 130.8, 129.4, 128.4, 127.1, 125.2, 122.9, 52.4, 41.3, 34.4, 25.2.</div></div><div id="sec4_2_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 2-Cyclopentyl-4-(6-phenylquinazolin-4-yl)benzoic Acid (<b>49</b>)</h4><div class="NLM_p">Methyl 2-cyclopentyl-4-(6-phenylquinazolin-4-yl)benzoate <b>SI4</b> (170 mg, 0.41 mmol) was dissolved in MeOH (3.5 mL) and aq NaOH (1 mL). The resulting solution was heated to 70 °C for 1 h and allowed to cool to rt, added water (4 mL), and extracted with ether (2 × 10 mL). The aqueous phase was acidified with 1 N HCl (pH 6) and then extracted with EtOAc. The organic phase was dried (Na<sub>2</sub>CO<sub>3</sub>) and concentrated to afford 2-cyclopentyl-4-(6-phenylquinazolin-4-yl)benzoic acid <b>49</b> (150 mg, 91%) as a light-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19 (s, 1H), 9.39 (s, 1H), 8.39 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 8.24–8.19 (m, 2H), 7.92 (d, <i>J</i> = 1.7 Hz, 1H), 7.87 (d, <i>J</i> = 7.9 Hz, 1H), 7.78–7.73 (m, 3H), 7.51 (t, <i>J</i> = 7.4 Hz, 2H), 7.45 (d, <i>J</i> = 7.3 Hz, 1H), 3.86–3.75 (m, 1H), 2.12–2.04 (m, 2H), 1.80–1.74 (m, 2H), 1.69–1.60 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.4, 167.0, 154.4, 149.9, 146.1, 139.8, 139.0, 138.8, 133.5, 133.4, 129.3, 129.2, 129.2, 128.5, 128.3, 127.2, 127.1, 123.7, 122.5, 41.2, 34.5, 25.3; HRMS: calcd for C<sub>26</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup><i>m</i>/<i>z</i>, 393.1607; found, 393.1603; mp range 265–269 °C.</div></div><div id="sec4_2_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 2-Cyclopentyl-4-(quinazolin-4-yl)benzoic Acid (<b>50</b>)</h4><div class="NLM_p">4-Chloroquinazoline (100 mg, 0.60 mmol), 2-cyclopentyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzoate <b>4</b> (240 mg, 0.72 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (70.0 mg, 60.0 μmol), and Cs<sub>2</sub>CO<sub>3</sub> (0.60 g, 1.80 mmol) were loaded into a microwave vial. A solution of dioxane/water (6.0 mL of a 10:1 mix) was added, and the vial was flushed with nitrogen. The mixture was irradiated at 100 °C for 30 min. Once cooled, the aqueous layer was removed and the remaining volatiles were removed <i>in vacuo</i>. The crude was purified <i>via</i> column chromatography, and the intermediate ester was isolated. The ester was dissolved in dioxane (1 mL) and aq LiOH (1 mL of a 1 M solution). The solution was heated to 100 °C for 16 h and then acidified to pH 4 with 1 M aq HCl, causing a precipitate to form. The solid was collected by filtration, washed with cold H<sub>2</sub>O, and then dried under vacuum to afford the quinazoline (128 mg, 67%) as a pale-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 8.15–8.03 (m, 3H), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.78 (ddd, <i>J</i> = 9.7, 6.8, 1.4 Hz, 3H), 7.66 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 3.83–3.73 (m, 1H), 2.11–2.02 (m, 2H), 1.81–1.71 (m, 2H), 1.70–1.57 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 167.0, 154.3, 150.3, 146.0, 139.0, 134.4, 133.5, 129.1, 128.6, 128.4, 128.2, 127.08, 126.6, 122.3, 41.3, 34.4, 25.2; HRMS: calcd for C<sub>20</sub>H<sub>19</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 319.1446; found, <i>m</i>/<i>z</i> 319.1427; mp range 120–122 °C.</div></div><div id="sec4_2_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 2-Cyclopentyl-4-(2-methylquinolin-4-yl)benzoic Acid (<b>52</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>21</b> using 4-chloro-2-methylquinoline <b>51</b> (200 mg, 1.13 mmol). After purification, 2-cyclopentyl-4-(2-methylquinolin-4-yl)benzoic acid <b>52</b> (290 mg, 77%) was afforded as an off-white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.08 (s, 1H), 8.01 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 7.81 (d, <i>J</i> = 7.9 Hz, 1H), 7.78–7.72 (m, 2H), 7.56–7.52 (m, 2H), 7.43–7.39 (m, 2H), 3.86–3.74 (m, 1H), 2.70 (s, 3H), 2.12–2.00 (m, 2H), 1.78–1.74 (m, 2H), 1.67–1.56 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.6, 161.6, 151.0, 150.0, 149.3, 143.2, 135.4, 132.6, 132.5, 132.0, 130.8, 129.7, 129.3, 128.1, 127.2, 125.3, 44.4, 37.6, 28.4, 27.9; HRMS: calcd for C<sub>22</sub>H<sub>21</sub>NO<sub>2</sub> [M – H]<sup>−</sup><i>m</i>/<i>z</i>, 332.1651; found, 332.1642; mp range 271–275 °C.</div></div><div id="sec4_2_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 2-Cyclopentyl-4-(1,6-naphthyridin-4-yl)benzoic Acid (<b>54</b>)</h4><div class="NLM_p">Prepared following the procedure for <b>21</b> using 4-chloro-1,6-naphthyridine <b>53</b> (80.0 mg, 0.49 mmol). After purification, 2-cyclopentyl-4-(1,6-naphthyridin-4-yl)benzoic acid <b>54</b> (110 mg, 54%) was afforded as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 9.24–9.15 (m, 2H), 8.79 (d, <i>J</i> = 5.8 Hz, 1H), 8.01 (dd, <i>J</i> = 5.9, 0.9 Hz, 1H), 7.84 (d, <i>J</i> = 7.9 Hz, 1H), 7.71–7.65 (m, 2H), 7.54 (dd, <i>J</i> = 7.9, 1.8 Hz, 1H), 3.84–3.73 (m, 1H), 2.09–2.04 (m, 2H), 1.80–1.71 (m, 2H), 1.71–1.56 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.88, 155.32, 150.85, 150.45, 147.84, 146.88, 146.7, 138.4, 133.1, 129.9, 128.4, 127.3, 123.5, 122.3, 121.5, 41.6, 34.8, 25.6; HRMS calcd for C<sub>20</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup><i>m</i>/<i>z</i>, 319.1447; found, 319.1427; mp range 195–200 °C.</div></div><div id="sec4_2_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 2-Cyclopentyl-4-(pyrimidin-4-yl)benzoic Acid (<b>56</b>)</h4><div class="NLM_p">4-Chloropyrimidine (162 mg, 1.40 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (1.32 g, 4.00 mmol) were dissolved in 1,2-DME (4.00 mL) and H<sub>2</sub>O (0.70 mL). Pd(PPh<sub>3</sub>)<sub>4</sub> (78 mg, 68 μmol) and <b>3</b> (503 mg, 1.60 mmol) were added, and the solution was stirred at 120 °C for 30 min. The resulting suspension was filtered through Celite under vacuum. The filter cake was washed with ethyl acetate, and the filtrate was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude product was purified by column chromatography [0–50% EtOAc(0.01% acetic acid)/hexane] to afford 2-cyclopentyl-4-(pyrimidin-4-yl)benzoic acid <b>56</b> (29 mg, 8%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (br s, 1H), 9.28 (s, 1H), 8.92–8.87 (m, 1H), 8.25 (s, 1H), 8.20–8.16 (m, 1H), 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 7.77 (d, <i>J</i> = 7.7 Hz, 1H), 3.72 (p, <i>J</i> = 8.0 Hz, 1H), 2.09–2.01 (m, 2H), 1.87–1.79 (m, 2H), 1.73–1.59 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 158.8, 158.8, 158.3, 146.4, 138.3, 134.5, 129.7, 125.1, 124.2, 117.7, 41.4, 34.4, 25.3; HRMS: calcd for C<sub>16</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup><i>m</i>/<i>z</i>, 267.1138; found, 267.1139; mp range 245 °C.</div></div><div id="sec4_2_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 2-Cyclopentyl-4-(2-phenylpyrimidin-4-yl)benzoic Acid (<b>58</b>)</h4><div class="NLM_p">4-Chloro-2-phenylpyrimidine (155 mg, 0.81 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (775 mg, 2.40 mmol) were dissolved in 1,2-DME (2.40 mL) and H<sub>2</sub>O (0.40 mL). Pd(PPh<sub>3</sub>)<sub>4</sub> (48.0 mg, 42.0 μmol) and <b>3</b> (307 mg, 0.97 mmol) were added, and the solution was stirred at 120 °C for 30 min. The resulting suspension was filtered through Celite under vacuum. The filter cake was washed with ethyl acetate, and the filtrate was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude product was purified by column chromatography [0–10% EtOAc(0.01% acetic acid)/hexane] to afford 2-cyclopentyl-4-(2-phenylpyrimidin-4-yl)benzoic acid <b>58</b> (21 mg, 8%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (br s, 1H), 9.04–8.95 (m, 1H), 8.52 (d, <i>J</i> = 6.0 Hz, 2H), 8.33 (s, 1H), 8.21 (d, <i>J</i> = 7.6 Hz, 1H), 8.15–8.07 (m, 1H), 7.82 (d, <i>J</i> = 7.6 Hz, 1H), 7.63–7.53 (m, 3H), 3.76 (p, <i>J</i> = 8.8 Hz, 1H), 2.12–2.03 (m, 2H), 1.90–1.82 (m, 2H), 1.75–1.65 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.4, 163.4, 162.3, 158.9, 146.4, 138.5, 137.2, 134.5, 131.0, 129.7, 128.8, 127.8, 125.3, 124.3, 115.7, 41.4, 34.4, 25.3; HRMS: calcd for C<sub>22</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 345.1598; found, 345.1600; mp range >250 °C.</div></div><div id="sec4_2_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Methyl 4-(2-Aminopyrimidin-4-yl)-2-cyclopentylbenzoate (<b>60</b>)</h4><div class="NLM_p">4-Chloropyrimidin-2-amine (501 mg, 3.90 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (3.81 mg, 12.0 mmol) were dissolved in 1,2-DME (8.00 mL) and H<sub>2</sub>O (2.00 mL). Pd(PPh<sub>3</sub>)<sub>4</sub> (238 mg, 0.21 mmol) and <b>4</b> (1.50 g, 4.50 mmol) were added, and the solution was irradiated at 120 °C for 30 min. The resulting suspension was filtered through Celite under vacuum. The filter cake was washed with ethyl acetate, and the filtrate was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The crude product was purified by column chromatography (20% EtOAc/hexane) to afford methyl 4-(2-aminopyrimidin-4-yl)-2-cyclopentylbenzoate <b>60</b> (523 mg, 45%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.34 (d, <i>J</i> = 5.2 Hz, 1H), 8.13 (d, <i>J</i> = 1.7 Hz, 1H), 7.93 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.73 (d, <i>J</i> = 8.1 Hz, 1H), 7.19 (d, <i>J</i> = 5.2 Hz, 1H), 6.74 (s, 2H), 3.85 (s, 3H), 3.69–3.54 (m, 1H), 2.11–1.96 (m, 2H), 1.90–1.75 (m, 2H), 1.73–1.55 (m, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.00, 163.81, 162.68, 159.31, 146.28, 139.90, 132.37, 129.51, 124.83, 124.03, 106.29, 52.23, 41.48, 34.34, 25.20; mp range 178 °C.</div></div><div id="sec4_2_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 4-(2-Aminopyrimidin-4-yl)-2-cyclopentylbenzoic Acid Hydrochloride (<b>61</b>·HCl)</h4><div class="NLM_p">Methyl 4-(2-aminopyrimidin-4-yl)-2-cyclopentylbenzoate <b>60</b> (40.0 mg, 0.13 mmol) was dissolved in 1,4-dioxane (1.00 mL). 1 M LiOH (1.00 mL) was added, and the solution was stirred at 100 °C for 17 h. HCl (6.00 M, 6.00 mL) was added, resulting in precipitate formation. The precipitate was filtered and washed with HCl (6.00 M). The precipitate was dried under reduced pressure to afford 4-(2-aminopyrimidin-4-yl)-2-cyclopentylbenzoic acid hydrochloride <b>61</b>·HCl (35.0 mg, 85%) as a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.54–8.14 (br s, 1H) 8.53 (d, <i>J</i> = 6.3 Hz, 1H), 8.22 (d, <i>J</i> = 1.8 Hz, 1H), 8.05 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 7.77 (d, <i>J</i> = 8.2 Hz, 1H), 7.61 (d, <i>J</i> = 6.3 Hz, 1H), 3.73–3.60 (m, 1H), 2.10–1.98 (m, 2H), 1.90–1.76 (m, 2H), 1.73–1.57 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.2, 168.3, 157.1, 150.3, 146.1, 136.7, 136.0, 129.5, 126.1, 125.3, 106.5, 41.5, 34.4, 25.2; HRMS: calcd for C<sub>16</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 284.1394; found, 284.1394; mp range >250 °C.</div></div><div id="sec4_2_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 2-Cyclopentyl-4-(2-(phenylamino)pyrimidin-4-yl)benzoic Acid Hydrochloride (<b>62</b>·HCl)</h4><div class="NLM_p">Methyl 2-cyclopentyl-4-(2-(phenylamino)pyrimidin-4-yl)benzoate <b>SI5</b> (20 mg, 54 μmol) was dissolved in 1,4-dioxane (1.0 mL). 1 M LiOH (1.0 mL) was added, and the solution was stirred for at 100 °C for 16 h. HCl (6.0 M, 6.0 mL) was added, resulting in precipitate formation. The precipitate was filtered and washed with HCl (6.0 M). The precipitate was dried under reduced pressure to afford 2-cyclopentyl-4-(2-(phenylamino)pyrimidin-4-yl)benzoic acid hydrochloride (<b>62</b>·HCl) (12 mg, 63%) as a yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.10 (br s, 1H), 9.74 (s, 1H), 8.61–8.55 (m, 1H), 8.29 (s, 1H), 8.00 (d, <i>J</i> = 7.4 Hz, 1H), 7.86 (d, <i>J</i> = 7.3 Hz, 2H), 7.77 (d, <i>J</i> = 7.4 Hz, 1H), 7.50–7.46 (m, 1H), 7.33–7.27 (m, 2H), 7.01–6.96 (m, 1H), 3.77 (p, <i>J</i> = 9.6 Hz, 1H), 2.13–2.03 (m, 2H), 1.89–1.80 (m, 2H), 1.73–1.61 (m, 4H); <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.3, 162.6, 160.1, 159.3, 146.4, 140.5, 139.0, 134.0, 129.6, 128.4, 125.1, 124.0, 121.5, 119.0, 108.3, 41.2, 34.5, 25.4; HRMS: calcd for C<sub>22</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 360.4365; found, 360.1712; mp range >250 °C.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i108"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02274" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01101" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01101" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02274?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02274</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Mass spectrometry methods, compound characterization spectra, CAMKK2 in cell target engagement NanoBRET data, <i>p</i>-AMPK Western blot images, details on docking studies, and treespot images of kinase selectivity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Compound selectivity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Compound molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_003.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf">jm0c02274_si_001.pdf (31.51 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_002.csv">jm0c02274_si_002.csv (3.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_003.xlsx">jm0c02274_si_003.xlsx (41.42 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02274" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77576" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John W. Scott</span> - <span class="hlFld-Affiliation affiliation">St
Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Mary MacKillop
Institute for Health Research, Australian
Catholic University, 215 Spring Street, Melbourne 3000, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">The Florey Institute of Neuroscience and Mental Health, 30 Royal Parade, Parkville 3052, Australia</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbd1c8d8d4cfcffbc8cdd295dedfce95dace"><span class="__cf_email__" data-cfemail="84eef7e7ebf0f0c4f7f2edaae1e0f1aae5f1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David H. Drewry</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">UNC Lineberger Comprehensive Cancer Center,
UNC Eshelman School of
Pharmacy, University of North Carolina at
Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5973-5798" title="Orcid link">https://orcid.org/0000-0001-5973-5798</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#afcbced9c6cb81cbddcad8ddd6efdac1cc81cacbda"><span class="__cf_email__" data-cfemail="5034312639347e34223527222910253e337e353425">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin J. Eduful</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sean N. O’Byrne</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Louisa Temme</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher R. M. Asquith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, School of Medicine, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5871-3458" title="Orcid link">https://orcid.org/0000-0001-5871-3458</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Liang</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alfredo Picado</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph R. Pilotte</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad Anwar Hossain</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8684-8755" title="Orcid link">https://orcid.org/0000-0001-8684-8755</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carrow I. Wells</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4799-6792" title="Orcid link">https://orcid.org/0000-0003-4799-6792</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William J. Zuercher</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carolina M. C. Catta-Preta</span> - <span class="hlFld-Affiliation affiliation">Centro
de Química Medicinal (CQMED), Centro de Biologia Molecular
e Engenharia Genética (CBMEG), Universidade
Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Departamento de Genética e Evolução,
Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Priscila Zonzini Ramos</span> - <span class="hlFld-Affiliation affiliation">Centro
de Química Medicinal (CQMED), Centro de Biologia Molecular
e Engenharia Genética (CBMEG), Universidade
Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Departamento de Genética e Evolução,
Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">André de S. Santiago</span> - <span class="hlFld-Affiliation affiliation">Centro
de Química Medicinal (CQMED), Centro de Biologia Molecular
e Engenharia Genética (CBMEG), Universidade
Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Departamento de Genética e Evolução,
Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rafael M. Couñago</span> - <span class="hlFld-Affiliation affiliation">Centro
de Química Medicinal (CQMED), Centro de Biologia Molecular
e Engenharia Genética (CBMEG), Universidade
Estadual de Campinas (UNICAMP), Campinas, São Paulo 13083-875, Brazil</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Departamento de Genética e Evolução,
Instituto de Biologia, UNICAMP, Campinas, São Paulo 13083-886, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher G. Langendorf</span> - <span class="hlFld-Affiliation affiliation">St
Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kévin Nay</span> - <span class="hlFld-Affiliation affiliation">St
Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Mary MacKillop
Institute for Health Research, Australian
Catholic University, 215 Spring Street, Melbourne 3000, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan S. Oakhill</span> - <span class="hlFld-Affiliation affiliation">St
Vincent’s Institute and Department of Medicine, The University of Melbourne, 41 Victoria Parade, Fitzroy 3065, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Mary MacKillop
Institute for Health Research, Australian
Catholic University, 215 Spring Street, Melbourne 3000, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas L. Pulliam</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Systems Imaging, University of
Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Nuclear Receptors and Cell Signaling, University of Houston, Houston, Texas 77204, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biology and Biochemistry, University
of Houston, Houston, Texas 77204, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chenchu Lin</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Systems Imaging, University of
Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</span>; 
    <span class="hlFld-Affiliation affiliation">The University of Texas MD Anderson Cancer Center UTHealth
Graduate
School of Biomedical Sciences, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominik Awad</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Systems Imaging, University of
Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</span>; 
    <span class="hlFld-Affiliation affiliation">The University of Texas MD Anderson Cancer Center UTHealth
Graduate
School of Biomedical Sciences, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy M. Willson</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium and Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4181-8223" title="Orcid link">https://orcid.org/0000-0003-4181-8223</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel E. Frigo</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Systems Imaging, University of
Texas MD Anderson Cancer Center, Houston, Texas 77054, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Nuclear Receptors and Cell Signaling, University of Houston, Houston, Texas 77204, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biology and Biochemistry, University
of Houston, Houston, Texas 77204, United
States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Genitourinary Medical Oncology, University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span>; 
    <span class="hlFld-Affiliation affiliation">The Methodist Hospital Research Institute, Houston, Texas 77030, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0713-471X" title="Orcid link">https://orcid.org/0000-0002-0713-471X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>B.J.E., S.N.O., and L.T. contributed equally to this work. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This research was funded in part by the National Cancer Institute of the National Institutes of Health (grant numbers R01CA218442 and R01CA184208) and was supported by the NIH Illuminating the Druggable Genome program (grant number 1U24DK116204-01), FAPESP (Fundação de Amparo à Pesquisa do Estado de São Paulo) (2013/50724-5 and 2014/50897-0), and CNPq (Conselho Nacional de Desenvolvimento Científico e Tecnológico) (465651/2014-3). The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, the Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA Darmstadt Germany, MSD, Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, Takeda, and Wellcome [106169/ZZ14/Z]. J.W.S. and J.S.O. are funded by National Health and Medical Research Council (NHMRC) grants GNT1138102 and GNT1145265, respectively. P.Z.R. received CAPES (Coordenação de Aperfeiçoamento de Pessoal de Nível Superior) postdoctoral fellowship (88887.136432/2017-00). A.d.S.S. is an FAPESP postdoctoral fellow (2019/14275-8). C.G.L. is an NHMRC early career fellow (GNT1143080). C.L. is supported by an Antje Wuelfrath Gee and Harry Gee, Jr. Family Legacy Scholarship. D.A. is supported by an American Legion Auxiliary Fellowship. Acknowledgements for beamlines/crystallography: this work is based upon research conducted at the Northeastern Collaborative Access Team beamlines (GU51510, GU56413), which are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P41 GM103403). The Pilatus 6M detector on the 24-ID-C beamline is funded by an NIH-ORIP HEI grant (S10 RR029205). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility, operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357. We thank Diamond Light Source for access to beam line I03 (proposal number MX10619-74).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">PDB ID codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a>; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>. Authors will release the atomic coordinates upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i111">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10768" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10768" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for LC–MS/HRMS support provided by Dr. Brandie Ehrmann and Diane E. Wallace in the Mass Spectrometry Core Laboratory at the University of North Carolina at Chapel Hill. The core is supported by the National Science Foundation under grant no. CHE-1726291.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CAMKK2</td><td class="NLM_def"><p class="first last">calcium-calmodulin protein kinase kinase 2</p></td></tr><tr><td class="NLM_term">AMPK</td><td class="NLM_def"><p class="first last">AMP-activated protein kinase</p></td></tr><tr><td class="NLM_term">AKT</td><td class="NLM_def"><p class="first last">protein kinase B (PKB)</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">NAFLD</td><td class="NLM_def"><p class="first last">non-alcoholic fatty liver disease</p></td></tr><tr><td class="NLM_term">GRK3</td><td class="NLM_def"><p class="first last">G protein-coupled receptor kinase 3</p></td></tr><tr><td class="NLM_term">CK2</td><td class="NLM_def"><p class="first last">casein kinase 2</p></td></tr><tr><td class="NLM_term">CDKL2</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase-like 3</p></td></tr><tr><td class="NLM_term">AhR</td><td class="NLM_def"><p class="first last">aryl hydrocarbon receptor</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">trimethylsilylethoxymethyl</p></td></tr><tr><td class="NLM_term">CDI</td><td class="NLM_def"><p class="first last">1,1-carbodiimidazole</p></td></tr><tr><td class="NLM_term">DME</td><td class="NLM_def"><p class="first last">dimethoxyethane</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">PoC</td><td class="NLM_def"><p class="first last">percentage of control</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicology</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i113">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 74 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Calmodulin: a prototypical calcium sensor</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/s0962-8924(00)01800-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2FS0962-8924%2800%2901800-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10884684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkvVyrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=322-328&author=D.+Chinauthor=A.+R.+Means&title=Calmodulin%3A+a+prototypical+calcium+sensor&doi=10.1016%2Fs0962-8924%2800%2901800-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin: a prototypical calcium sensor</span></div><div class="casAuthors">Chin, D.; Means, A. R.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">322-328</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 60 refs.  Calmodulin is the best studied and prototypical example of the E-F-hand family of Ca2+-sensing proteins.  Changes in intracellular Ca2+ concn. regulate calmodulin in 3 distinct ways.  First, at the cellular level, by directing its subcellular distribution.  Second, at the mol. level, by promoting different modes of assocn. with many target proteins.  Third, by directing a variety of conformational states in calmodulin that result in target-specific activation.  The calmodulin-dependent regulation of protein kinases illustrates the potential mechanisms by which Ca2+-sensing proteins can recognize and generate affinity and specificity for effectors in a Ca2+-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ThLB_NAqhbVg90H21EOLACvtfcHk0lg3TP5r7J1FEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkvVyrtLw%253D&md5=9296d0539a7923ea86ad388ca387a689</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0962-8924%2800%2901800-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0962-8924%252800%252901800-6%26sid%3Dliteratum%253Aachs%26aulast%3DChin%26aufirst%3DD.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCalmodulin%253A%2520a%2520prototypical%2520calcium%2520sensor%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2000%26volume%3D10%26spage%3D322%26epage%3D328%26doi%3D10.1016%2Fs0962-8924%2800%2901800-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hook, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Ca(2+)/CaM-dependent kinases: from activation to function</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.41.1.471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1146%2Fannurev.pharmtox.41.1.471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11264466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVaqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=471-505&author=S.+S.+Hookauthor=A.+R.+Means&title=Ca%282%2B%29%2FCaM-dependent+kinases%3A+from+activation+to+function&doi=10.1146%2Fannurev.pharmtox.41.1.471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ca2+/CaM-dependent kinases: from activation to function</span></div><div class="casAuthors">Hook, Sara S.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-505</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 214 refs.  Calmodulin (CaM) is an essential protein that serves as a ubiquitous intracellular receptor for Ca2+.  The Ca2+/CaM complex initiates a plethora of signaling cascades that culminate in alteration of cellular functions.  Among the many Ca2+/CaM-binding proteins to be discovered, the multifunctional protein kinases CaMKI, II, and IV play pivotal roles.  Our review focuses on this class of CaM kinases to illustrate the structural and biochem. basis for Ca2+/CaM interaction with and regulation of its target enzymes.  Gene transcription has been chosen as the functional endpoint to illustrate the recent advances in Ca2+/CaM-mediated signal transduction mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb-524IthxV7Vg90H21EOLACvtfcHk0ljR9v1BliyQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVaqt74%253D&md5=7784f5c4d753bbc15ff5cb77fcfc1161</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.41.1.471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.41.1.471%26sid%3Dliteratum%253Aachs%26aulast%3DHook%26aufirst%3DS.%2BS.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCa%25282%252B%2529%252FCaM-dependent%2520kinases%253A%2520from%2520activation%2520to%2520function%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2001%26volume%3D41%26spage%3D471%26epage%3D505%26doi%3D10.1146%2Fannurev.pharmtox.41.1.471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parameswaran, S.</span></span> <span> </span><span class="NLM_article-title">Calmodulin-binding proteins: A journey of 40 years</span>. <i>Cell Calcium</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.ceca.2018.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.ceca.2018.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=30205293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSnt73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=89-100&author=R.+K.+Sharmaauthor=S.+Parameswaran&title=Calmodulin-binding+proteins%3A+A+journey+of+40+years&doi=10.1016%2Fj.ceca.2018.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin-binding proteins: A journey of 40 years</span></div><div class="casAuthors">Sharma, Rajendra K.; Parameswaran, Sreejit</div><div class="citationInfo"><span class="NLM_cas:title">Cell Calcium</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">CECADV</span>;
        ISSN:<span class="NLM_cas:issn">0143-4160</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The proteins which bind to calmodulin in a Ca2+-dependent and reversible manner are known as calmodulin-binding proteins.  These proteins are involved in a multitude of processes in which Ca2+ and calmodulin play crucial roles.  Our group elucidated the mechanism and importance of these proteins in normal and diseased conditions.  Various calmodulin-binding proteins were discovered and purified from bovine tissue including a heat stable calmodulin-binding protein 70, calmodulin-dependent protein kinase VI and a high mol. wt. calmodulin-binding protein (HMWCaMBP).  We obsd. a complex interplay occurs between these and other Ca2+ and calmodulin-binding proteins during cardiac ischemia and reperfusion.  Purified cardiac HMWCaMBP is a homolog form of calpastatin and an inhibitor of the Ca2+-activated cysteine proteases, calpains and therefore can have cardioprotective role in ischemic conditions.  Calcineurin is a Ca2+ and calmodulin-dependent serine/threonine protein phosphatase showed increased phosphatase activity in ischemic heart through its direct interaction with Hsp70 and expression of calcineurin following ischemia suggests self-repair and favorable survival outcomes.  Calcineurin was also found to be present in other tissues including the eye; where its expression and calcineurin phosphatase activity varied.  In neurons, calcineurin may play a key role in initiating apoptosis-related pathways esp. in epilepsy.  In colorectal cancer we demonstrated high calcineurin phosphatase activity and simultaneous overexpression of calcineurin.  The impact of calcineurin signaling on neuronal apoptosis in epilepsy and its use as a diagnostic marker for colorectal cancer requires in-depth study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2LvL3lccyxLVg90H21EOLACvtfcHk0ljR9v1BliyQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSnt73E&md5=66bfa8ecdae78cea42ef30e1c092fdd2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ceca.2018.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceca.2018.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DR.%2BK.%26aulast%3DParameswaran%26aufirst%3DS.%26atitle%3DCalmodulin-binding%2520proteins%253A%2520A%2520journey%2520of%252040%2520years%26jtitle%3DCell%2520Calcium%26date%3D2018%26volume%3D75%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.ceca.2018.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clapham, D. E.</span></span> <span> </span><span class="NLM_article-title">Calcium signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">1047</span>– <span class="NLM_lpage">1058</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cell.2007.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18083096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2007&pages=1047-1058&author=D.+E.+Clapham&title=Calcium+signaling&doi=10.1016%2Fj.cell.2007.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium signaling</span></div><div class="casAuthors">Clapham, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1047-1058</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ca2+ ions impact nearly every aspect of cellular life.  Here, the author examines the principles of Ca2+ signaling, from changes in protein conformations driven by Ca2+ to the mechanisms that control Ca2+ levels in the cytoplasm and organelles.  Also discussed is the highly localized nature of Ca2+-mediated signal transduction and its specific role in excitability, exocytosis, motility, apoptosis, and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojw-AGybzZYrVg90H21EOLACvtfcHk0ljR9v1BliyQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFGgsA%253D%253D&md5=268945d7ca1ec7a455f69cb72272be85</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DClapham%26aufirst%3DD.%2BE.%26atitle%3DCalcium%2520signaling%26jtitle%3DCell%26date%3D2007%26volume%3D131%26spage%3D1047%26epage%3D1058%26doi%3D10.1016%2Fj.cell.2007.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Racioppi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Calcium/calmodulin-dependent protein kinase kinase 2: roles in signaling and pathophysiology</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">31658</span>– <span class="NLM_lpage">31665</span>, <span class="refDoi"> DOI: 10.1074/jbc.r112.356485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.R112.356485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22778263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGgsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=31658-31665&author=L.+Racioppiauthor=A.+R.+Means&title=Calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2%3A+roles+in+signaling+and+pathophysiology&doi=10.1074%2Fjbc.r112.356485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium/Calmodulin-dependent Protein Kinase Kinase 2: Roles in Signaling and Pathophysiology</span></div><div class="casAuthors">Racioppi, Luigi; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">31658-31665</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Many cellular Ca2+-dependent signaling cascades utilize calmodulin (CaM) as the intracellular Ca2+ receptor.  Ca2+/CaM binds and activates a plethora of enzymes, including CaM kinases (CaMKs).  CaMKK2 is one of the most versatile of the CaMKs and will phosphorylate and activate CaMKI, CaMKIV, and AMP-activated protein kinase.  Cell expression of CaMKK2 is limited, yet CaMKK2 is involved in regulating many important physiol. and pathophysiol. processes, including energy balance, adiposity, glucose homeostasis, hematopoiesis, inflammation, and cancer.  Here, we explore known functions of CaMKK2 and discuss its potential as a target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8CI0yjQ9IcrVg90H21EOLACvtfcHk0ljR9v1BliyQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGgsbnL&md5=c0eaa88730dd66d0c2add46ee57c176b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.356485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.356485%26sid%3Dliteratum%253Aachs%26aulast%3DRacioppi%26aufirst%3DL.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCalcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%253A%2520roles%2520in%2520signaling%2520and%2520pathophysiology%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D31658%26epage%3D31665%26doi%3D10.1074%2Fjbc.r112.356485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Components of a calmodulin-dependent protein kinase cascade. Molecular cloning, functional characterization and cellular localization of Ca2+/calmodulin-dependent protein kinase kinase beta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">31880</span>– <span class="NLM_lpage">31889</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.48.31880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.273.48.31880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=9822657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVeltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=31880-31889&author=K.+A.+Andersonauthor=R.+L.+Meansauthor=Q.-H.+Huangauthor=B.+E.+Kempauthor=E.+G.+Goldsteinauthor=M.+A.+Selbertauthor=A.+M.+Edelmanauthor=R.+T.+Fremeauauthor=A.+R.+Means&title=Components+of+a+calmodulin-dependent+protein+kinase+cascade.+Molecular+cloning%2C+functional+characterization+and+cellular+localization+of+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+beta&doi=10.1074%2Fjbc.273.48.31880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Components of a calmodulin-dependent protein kinase cascade. Molecular cloning, functional characterization and cellular localization of Ca2+/calmodulin-dependent protein kinase kinase β</span></div><div class="casAuthors">Anderson, Kristin A.; Means, Raylene L.; Huang, Qi-Hui; Kemp, Bruce E.; Goldstein, Elaine G.; Selbert, Michele A.; Edelman, Arthur M.; Fremeau, Robert T.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">31880-31889</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ca2+/calmodulin-dependent protein kinases I and IV (CaMKI and CaMKIV, resp.) require phosphorylation on an equiv. single Thr in the activation loop of subdomain VIII for maximal activity.  Two distinct CaMKI/IV kinases, CaMKKα and CaMKKβ, were purified from rat brain and partially sequenced (Edelman, A. M., Mitchelhill, K., Selbert, M. A., Anderson, K. A., Hook, S. S., Stapleton, D., Goldstein, E. G., Means, A. R., and Kemp, B. E. (1996) J. Biol. Chem. 271, 10806-10810).  We report here the cloning and sequencing of cDNAs for human and rat CaMKKβ, tissue and regional brain localization of CaMKKβ protein, and mRNA and functional characterization of recombinant CaMKKβ in vitro and in Jurkat T cells.  The sequences of human and rat CaMKKβ demonstrate 65% identity and 80% similarity with CaMKKα and 30-40% identity with CaMKI and CaMKIV themselves.  CaMKKβ is broadly distributed among rat tissues with highest levels in CaMKIV-expressing tissues such as brain, thymus, spleen, and testis.  In brain, CaMKKβ tracks more closely with CaMKIV than does CaMKKα.  Bacterially expressed CaMKKβ undergoes intramol. autophosphorylation, is regulated by Ca2+/CaM, and phosphorylates CaMKI and CaMKIV on Thr177 and Thr200, resp.  CaMKKβ activates both CaMKI and CaMKIV when coexpressed in Jurkat T cells as judged by phosphorylated cAMP response element-binding protein-dependent reporter gene expression.  CaMKKβ activity is enhanced by elevation of intracellular Ca2+, although substantial activity is obsd. at the resting Ca2+ concn.  The strict Ca2+ requirement of CaMKIV-dependent phosphorylation of cAMP response element-binding protein, is therefore controlled at the level of CaMKIV rather than CaMKK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokPLunLpqVSbVg90H21EOLACvtfcHk0lgkE4nvpRojxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvVeltr4%253D&md5=dd9aad89eb2c0e06f02ce47233455d75</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.48.31880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.48.31880%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DMeans%26aufirst%3DR.%2BL.%26aulast%3DHuang%26aufirst%3DQ.-H.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DGoldstein%26aufirst%3DE.%2BG.%26aulast%3DSelbert%26aufirst%3DM.%2BA.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DComponents%2520of%2520a%2520calmodulin-dependent%2520protein%2520kinase%2520cascade.%2520Molecular%2520cloning%252C%2520functional%2520characterization%2520and%2520cellular%2520localization%2520of%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520beta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D31880%26epage%3D31889%26doi%3D10.1074%2Fjbc.273.48.31880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawley, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustard, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenguelli, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardie, D. G.</span></span> <span> </span><span class="NLM_article-title">Calmodulin-dependent protein kinase kinase-beta is an alternative upstream kinase for AMP-activated protein kinase</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2005.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cmet.2005.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=16054095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntlCruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=9-19&author=S.+A.+Hawleyauthor=D.+A.+Panauthor=K.+J.+Mustardauthor=L.+Rossauthor=J.+Bainauthor=A.+M.+Edelmanauthor=B.+G.+Frenguelliauthor=D.+G.+Hardie&title=Calmodulin-dependent+protein+kinase+kinase-beta+is+an+alternative+upstream+kinase+for+AMP-activated+protein+kinase&doi=10.1016%2Fj.cmet.2005.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin-dependent protein kinase kinase-β is an alternative upstream kinase for AMP-activated protein kinase</span></div><div class="casAuthors">Hawley, Simon A.; Pan, David A.; Mustard, Kirsty J.; Ross, Louise; Bain, Jenny; Edelman, Arthur M.; Frenguelli, Bruno G.; Hardie, D. Grahame</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-19</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The AMP-activated protein kinase (AMPK) is a crit. regulator of energy balance at both the cellular and whole-body levels.  Two upstream kinases have been reported to activate AMPK in cell-free assays, i.e., the tumor suppressor LKB1 and calmodulin-dependent protein kinase kinase.  However, evidence that this is physiol. relevant currently only exists for LKB1.  We now report that there is a significant basal activity and phosphorylation of AMPK in LKB1-deficient cells that can be stimulated by Ca2+ ionophores, and studies using the CaMKK inhibitor STO-609 and isoform-specific siRNAs show that CaMKKβ is required for this effect.  CaMKKβ also activates AMPK much more rapidly than CaMKKα in cell-free assays.  K+-induced depolarization in rat cerebrocortical slices, which increases intracellular Ca2+ without disturbing cellular adenine nucleotide levels, activates AMPK, and this is blocked by STO-609.  Our results suggest a potential Ca2+-dependent neuroprotective pathway involving phosphorylation and activation of AMPK by CaMKKβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHr770gkRJILVg90H21EOLACvtfcHk0lgkE4nvpRojxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntlCruro%253D&md5=6de7ad1e166f6aefb027bf74bbdd9b03</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2005.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2005.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DHawley%26aufirst%3DS.%2BA.%26aulast%3DPan%26aufirst%3DD.%2BA.%26aulast%3DMustard%26aufirst%3DK.%2BJ.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DBain%26aufirst%3DJ.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26aulast%3DFrenguelli%26aufirst%3DB.%2BG.%26aulast%3DHardie%26aufirst%3DD.%2BG.%26atitle%3DCalmodulin-dependent%2520protein%2520kinase%2520kinase-beta%2520is%2520an%2520alternative%2520upstream%2520kinase%2520for%2520AMP-activated%2520protein%2520kinase%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D2%26spage%3D9%26epage%3D19%26doi%3D10.1016%2Fj.cmet.2005.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karacosta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azabdaftari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feliciano, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span> <span> </span><span class="NLM_article-title">A regulatory feedback loop between Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) and the androgen receptor in prostate cancer progression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">24832</span>– <span class="NLM_lpage">24843</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.370783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M112.370783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22654108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWgtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=24832-24843&author=L.+G.+Karacostaauthor=B.+A.+Fosterauthor=G.+Azabdaftariauthor=D.+M.+Felicianoauthor=A.+M.+Edelman&title=A+regulatory+feedback+loop+between+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+and+the+androgen+receptor+in+prostate+cancer+progression&doi=10.1074%2Fjbc.m112.370783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A Regulatory Feedback Loop Between Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) and the androgen receptor in Prostate Cancer Progression</span></div><div class="casAuthors">Karacosta, Loukia G.; Foster, Barbara A.; Azabdaftari, Gissou; Feliciano, David M.; Edelman, Arthur M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">24832-24843</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The androgen receptor (AR) plays a crit. role in prostate cancer (PCa) progression, however, the mol. mechanisms by which the AR regulates cell proliferation in androgen-dependent and castration-resistant PCa are incompletely understood.  We report that Ca2+/calmodulin-dependent kinase kinase 2 (CaMKK2) expression increases and becomes nuclear or perinuclear in advanced PCa.  In the TRAMP (transgenic adenocarcinoma of mouse prostate) model of PCa, CaMKK2 expression increases with PCa progression with many cells exhibiting nuclear staining.  CaMKK2 expression is higher in human castration-resistant tumor xenografts compared with androgen-responsive xenografts and is markedly higher in the AR-expressing, tumorigenic cell line LNCaP compared with cell lines that are AR-nonexpressing and/or nontumorigenic.  In LNCaP cells, dihydrotestosterone induced CaMKK2 mRNA and protein expression and translocation of CaMKK2 to the nucleus.  Conversely, androgen withdrawal suppressed CaMKK2 expression.  Knockdown of CaMKK2 expression by RNAi reduced LNCaP cell proliferation and increased percentages of cells in G1 phase, whereas correspondingly reducing percentages in S phase, of the cell cycle.  CaMKK2 knockdown reduced expression of the AR target gene prostate-specific antigen at both mRNA and protein levels, AR transcriptional activity driven by androgen responsive elements from the prostate-specific probasin gene promoter and levels of the AR-regulated cell cycle proteins, cyclin D1 and hyperphosphorylated Rb.  Our results suggest that in PCa progression, CaMKK2 and the AR are in a feedback loop in which CaMKK2 is induced by the AR to maintain AR activity, AR-dependent cell cycle control, and continued cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFq7oX7Qhp9bVg90H21EOLACvtfcHk0lgkE4nvpRojxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWgtbzM&md5=6335c6e09e17c5e511f4f2a8a379cea9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.370783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.370783%26sid%3Dliteratum%253Aachs%26aulast%3DKaracosta%26aufirst%3DL.%2BG.%26aulast%3DFoster%26aufirst%3DB.%2BA.%26aulast%3DAzabdaftari%26aufirst%3DG.%26aulast%3DFeliciano%26aufirst%3DD.%2BM.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26atitle%3DA%2520regulatory%2520feedback%2520loop%2520between%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520and%2520the%2520androgen%2520receptor%2520in%2520prostate%2520cancer%2520progression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D24832%26epage%3D24843%26doi%3D10.1074%2Fjbc.m112.370783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilganowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bissig, K.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldie, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, B.</span></span> <span> </span><span class="NLM_article-title">The camKK2/camKIV relay is an essential regulator of hepatic cancer</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1002/hep.27832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fhep.27832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25847065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KmsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=505-520&author=F.+Linauthor=K.+L.+Marceloauthor=K.+Rajapaksheauthor=C.+Coarfaauthor=A.+Deanauthor=N.+Wilganowskiauthor=H.+Robinsonauthor=E.+Sevickauthor=K.-D.+Bissigauthor=L.+C.+Goldieauthor=A.+R.+Meansauthor=B.+York&title=The+camKK2%2FcamKIV+relay+is+an+essential+regulator+of+hepatic+cancer&doi=10.1002%2Fhep.27832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The camKK2/camKIV relay is an essential regulator of hepatic cancer</span></div><div class="casAuthors">Lin, Fumin; Marcelo, Kathrina L.; Rajapakshe, Kimal; Coarfa, Cristian; Dean, Adam; Wilganowski, Nathaniel; Robinson, Holly; Sevick, Eva; Bissig, Karl-Dimiter; Goldie, Lauren C.; Means, Anthony R.; York, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">505-520</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatic cancer is one of the most lethal cancers worldwide.  Here, we report that the expression of Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) is significantly up-regulated in hepatocellular carcinoma (HCC) and neg. correlated with HCC patient survival.  The CaMKK2 protein is highly expressed in all eight hepatic cancer cell lines evaluated and is markedly up-regulated relative to normal primary hepatocytes.  Loss of CaMKK2 function is sufficient to inhibit liver cancer cell growth, and the growth defect resulting from loss of CaMKK2 can be rescued by ectopic expression of wild-type CaMKK2 but not by kinase-inactive mutants.  Cellular ablation of CaMKK2 using RNA interference yields a gene signature that correlates with improvement in HCC patient survival, and ablation or pharmacol. inhibition of CaMKK2 with STO-609 impairs tumorigenicity of liver cancer cells in vivo.  Moreover, CaMKK2 expression is up-regulated in a time-dependent manner in a carcinogen-induced HCC mouse model, and STO-609 treatment regresses hepatic tumor burden in this model.  Mechanistically, CaMKK2 signals through Ca2+/calmodulin-dependent protein kinase 4 (CaMKIV) to control liver cancer cell growth.  Further anal. revealed that CaMKK2 serves as a scaffold to assemble CaMKIV with key components of the mammalian target of rapamycin/ribosomal protein S6 kinase, 70 kDa, pathway and thereby stimulate protein synthesis through protein phosphorylation.  Conclusion: The CaMKK2/CaMKIV relay is an upstream regulator of the oncogenic mammalian target of rapamycin/ribosomal protein S6 kinase, 70 kDa, pathway, and the importance of this CaMKK2/CaMKIV axis in HCC growth is confirmed by the potent growth inhibitory effects of genetically or pharmacol. decreasing CaMKK2 activity; collectively, these findings suggest that CaMKK2 and CaMKIV may represent potential targets for hepatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsIuVll-cevbVg90H21EOLACvtfcHk0lhXlmdrPdpGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KmsLzK&md5=62e123eedf5f49c210aeb2d8edb695c3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fhep.27832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.27832%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DF.%26aulast%3DMarcelo%26aufirst%3DK.%2BL.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DDean%26aufirst%3DA.%26aulast%3DWilganowski%26aufirst%3DN.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DSevick%26aufirst%3DE.%26aulast%3DBissig%26aufirst%3DK.-D.%26aulast%3DGoldie%26aufirst%3DL.%2BC.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26aulast%3DYork%26aufirst%3DB.%26atitle%3DThe%2520camKK2%252FcamKIV%2520relay%2520is%2520an%2520essential%2520regulator%2520of%2520hepatic%2520cancer%26jtitle%3DHepatology%26date%3D2015%26volume%3D62%26spage%3D505%26epage%3D520%26doi%3D10.1002%2Fhep.27832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-L.</span></span> <span> </span><span class="NLM_article-title">CAMKK2, regulated by promoter methylation, is a prognostic marker in diffuse gliomas</span>. <i>CNS Neurosci. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">518</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1111/cns.12531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1111%2Fcns.12531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27012733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvVCrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=518-524&author=D.-M.+Liuauthor=H.-J.+Wangauthor=B.+Hanauthor=X.-Q.+Mengauthor=M.-H.+Chenauthor=D.-B.+Yangauthor=Y.+Sunauthor=Y.-L.+Liauthor=C.-L.+Jiang&title=CAMKK2%2C+regulated+by+promoter+methylation%2C+is+a+prognostic+marker+in+diffuse+gliomas&doi=10.1111%2Fcns.12531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">CAMKK2, Regulated by Promoter Methylation, is a Prognostic Marker in Diffuse Gliomas</span></div><div class="casAuthors">Liu, Da-Ming; Wang, Hong-Jun; Han, Bo; Meng, Xiang-Qi; Chen, Ming-Hui; Yang, Dong-Bo; Sun, Ying; Li, Yong-Li; Jiang, Chuan-Lu</div><div class="citationInfo"><span class="NLM_cas:title">CNS Neuroscience & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">518-524</span>CODEN:
                <span class="NLM_cas:coden">CNTNAB</span>;
        ISSN:<span class="NLM_cas:issn">1755-5930</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Aims : To explore the expression, methylation pattern, the prognostic value, and the biol. consequences of CAMKK2 in gliomas.  Methods : The expression and methylation pattern of CAMKK2 was inferred and validated from mRNA expression profile (N = 866) and methylation profile (N = 426) of glioma tissue samples, and independent samples were used for further validation by IHC and pyrosequencing.  To explore the function of CAMKK2 in gliomas, in vitro studies, colony formation assays and migration and invasion assays were performed.  Results : We found the upregulation of CAMKK2 in high-grade glioma samples was assocd. with promoter hypomethylation.  An elevated expression of CAMKK2 was assocd. with worse prognosis.  By in vitro assays, we demonstrated that CAMKK2 could promote cell migration, invasion, and proliferation.  Conclusions : The expression level of CAMKK2 could be regulated by promoter methylation.  CAMKK2 serves as a prognostic marker in gliomas and could be a potential therapeutic target in gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2555BynUjLVg90H21EOLACvtfcHk0lhXlmdrPdpGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvVCrurs%253D&md5=1d8eba304bd0184cd36d36aef7dd8c3c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fcns.12531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcns.12531%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DD.-M.%26aulast%3DWang%26aufirst%3DH.-J.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DMeng%26aufirst%3DX.-Q.%26aulast%3DChen%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DD.-B.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.-L.%26aulast%3DJiang%26aufirst%3DC.-L.%26atitle%3DCAMKK2%252C%2520regulated%2520by%2520promoter%2520methylation%252C%2520is%2520a%2520prognostic%2520marker%2520in%2520diffuse%2520gliomas%26jtitle%3DCNS%2520Neurosci.%2520Ther.%26date%3D2016%26volume%3D22%26spage%3D518%26epage%3D524%26doi%3D10.1111%2Fcns.12531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massie, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos-Montoya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecchini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarthur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adryan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleave, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennie, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, I. G.</span></span> <span> </span><span class="NLM_article-title">The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2733</span>, <span class="refDoi"> DOI: 10.1038/emboj.2011.158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Femboj.2011.158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21602788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlemtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=2719-2733&author=C.+E.+Massieauthor=A.+Lynchauthor=A.+Ramos-Montoyaauthor=J.+Borenauthor=R.+Starkauthor=L.+Fazliauthor=A.+Warrenauthor=H.+Scottauthor=B.+Madhuauthor=N.+Sharmaauthor=H.+Bonauthor=V.+Zecchiniauthor=D.-M.+Smithauthor=G.+M.+Denicolaauthor=N.+Mathewsauthor=M.+Osborneauthor=J.+Hadfieldauthor=S.+Macarthurauthor=B.+Adryanauthor=S.+K.+Lyonsauthor=K.+M.+Brindleauthor=J.+Griffithsauthor=M.+E.+Gleaveauthor=P.+S.+Rennieauthor=D.+E.+Nealauthor=I.+G.+Mills&title=The+androgen+receptor+fuels+prostate+cancer+by+regulating+central+metabolism+and+biosynthesis&doi=10.1038%2Femboj.2011.158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis</span></div><div class="casAuthors">Massie, Charles E.; Lynch, Andy; Ramos-Montoya, Antonio; Boren, Joan; Stark, Rory; Fazli, Ladan; Warren, Anne; Scott, Helen; Madhu, Basetti; Sharma, Naomi; Bon, Helene; Zecchini, Vinny; Smith, Donna-Michelle; DeNicola, Gina M.; Mathews, Nik; Osborne, Michelle; Hadfield, James; MacArthur, Stewart; Adryan, Boris; Lyons, Scott K.; Brindle, Kevin M.; Griffiths, John; Gleave, Martin E.; Rennie, Paul S.; Neal, David E.; Mills, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2719-2733</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is a key regulator of prostate growth and the principal drug target for the treatment of prostate cancer.  Previous studies have mapped AR targets and identified some candidates which may contribute to cancer progression, but did not characterize AR biol. in an integrated manner.  In this study, we took an interdisciplinary approach, integrating detailed genomic studies with metabolomic profiling and identify an anabolic transcriptional network involving AR as the core regulator.  Restricting flux through anabolic pathways is an attractive approach to deprive tumors of the building blocks needed to sustain tumor growth.  Therefore, we searched for targets of the AR that may contribute to these anabolic processes and could be amenable to therapeutic intervention by virtue of differential expression in prostate tumors.  This highlighted calcium/calmodulin-dependent protein kinase kinase 2, which we show is overexpressed in prostate cancer and regulates cancer cell growth via its unexpected role as a hormone-dependent modulator of anabolic metab.  In conclusion, it is possible to progress from transcriptional studies to a promising therapeutic target by taking an unbiased interdisciplinary approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraxaKmz8_gabVg90H21EOLACvtfcHk0lhXlmdrPdpGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlemtb4%253D&md5=a28d6643fb447b79372eeb8922960f2c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.158%26sid%3Dliteratum%253Aachs%26aulast%3DMassie%26aufirst%3DC.%2BE.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DRamos-Montoya%26aufirst%3DA.%26aulast%3DBoren%26aufirst%3DJ.%26aulast%3DStark%26aufirst%3DR.%26aulast%3DFazli%26aufirst%3DL.%26aulast%3DWarren%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DH.%26aulast%3DMadhu%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DBon%26aufirst%3DH.%26aulast%3DZecchini%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DD.-M.%26aulast%3DDenicola%26aufirst%3DG.%2BM.%26aulast%3DMathews%26aufirst%3DN.%26aulast%3DOsborne%26aufirst%3DM.%26aulast%3DHadfield%26aufirst%3DJ.%26aulast%3DMacarthur%26aufirst%3DS.%26aulast%3DAdryan%26aufirst%3DB.%26aulast%3DLyons%26aufirst%3DS.%2BK.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26aulast%3DGriffiths%26aufirst%3DJ.%26aulast%3DGleave%26aufirst%3DM.%2BE.%26aulast%3DRennie%26aufirst%3DP.%2BS.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMills%26aufirst%3DI.%2BG.%26atitle%3DThe%2520androgen%2520receptor%2520fuels%2520prostate%2520cancer%2520by%2520regulating%2520central%2520metabolism%2520and%2520biosynthesis%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D2719%26epage%3D2733%26doi%3D10.1038%2Femboj.2011.158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Mora, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaHair, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCubrey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, R. A.</span></span> <span> </span><span class="NLM_article-title">Calcium/calmodulin-dependent kinase I and calcium/calmodulin-dependent kinase kinase participate in the control of cell cycle progression in MCF-7 human breast cancer cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">5408</span>– <span class="NLM_lpage">5416</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-0271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-05-0271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=15958590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFels70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=5408-5416&author=O.+G.+Rodriguez-Moraauthor=M.+M.+LaHairauthor=J.+A.+McCubreyauthor=R.+A.+Franklin&title=Calcium%2Fcalmodulin-dependent+kinase+I+and+calcium%2Fcalmodulin-dependent+kinase+kinase+participate+in+the+control+of+cell+cycle+progression+in+MCF-7+human+breast+cancer+cells&doi=10.1158%2F0008-5472.can-05-0271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium/Calmodulin-Dependent Kinase I and Calcium/Calmodulin-Dependent Kinase Kinase Participate in the Control of Cell Cycle Progression in MCF-7 Human Breast Cancer Cells</span></div><div class="casAuthors">Rodriguez-Mora, Oswaldo G.; LaHair, Michelle M.; McCubrey, James A.; Franklin, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5408-5416</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Ca is universally required for cell growth and proliferation.  Calmodulin is the main intracellular receptor for Ca.  Although Ca and calmodulin are well known to be required for cell cycle regulation, the target pathways for their action remain poorly defined.  Potential targets include the Ca/calmodulin-dependent kinases (CaM-K).  The aim of this study was to det. the role of the CaM-Ks on cell proliferation and progress through the cell cycle in breast cancer cells.  CaM-KI inhibition with either KN-93 or specific interfering RNA (siRNA) caused an arrest in the cell cycle in the human breast cancer cell line, MCF-7.  This arrest occurred in the G1 phase of the cell cycle.  Supporting this finding, CaM-K inhibition using KN-93 also resulted in a redn. of cyclin D1 protein and pRb phosphorylation when cells were compared with control cultures.  Furthermore, inhibition of the upstream activator of CaM-KI, CaM-KK, using siRNA also resulted in cell cycle arrest.  In summary, CaM-KK and CaM-KI participate in the control of the G0-G1 restriction check point of the cell cycle in human breast cancer cells.  This arrest seems due to an inhibition in cyclin D1 synthesis and a redn. in pRb phosphorylation.  To the best of our knowledge, this is the first time that CaM-KK was reported to be involved in mammalian cell cycle regulation and that CaM-Ks are regulating breast cancer cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-cqYzjPCAnbVg90H21EOLACvtfcHk0liulXujTw8QQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFels70%253D&md5=8929eeac7e0401bcf6a8b6cf1fae1571</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-0271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-0271%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Mora%26aufirst%3DO.%2BG.%26aulast%3DLaHair%26aufirst%3DM.%2BM.%26aulast%3DMcCubrey%26aufirst%3DJ.%2BA.%26aulast%3DFranklin%26aufirst%3DR.%2BA.%26atitle%3DCalcium%252Fcalmodulin-dependent%2520kinase%2520I%2520and%2520calcium%252Fcalmodulin-dependent%2520kinase%2520kinase%2520participate%2520in%2520the%2520control%2520of%2520cell%2520cycle%2520progression%2520in%2520MCF-7%2520human%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D5408%26epage%3D5416%26doi%3D10.1158%2F0008-5472.can-05-0271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofude, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namiki, M.</span></span> <span> </span><span class="NLM_article-title">Down-regulation of calcium/calmodulin-dependent protein kinase kinase 2 by androgen deprivation induces castration-resistant prostate cancer</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1801</span>, <span class="refDoi"> DOI: 10.1002/pros.22533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fpros.22533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22549914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGqtLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=1789-1801&author=T.+Shimaauthor=A.+Mizokamiauthor=T.+Miyagiauthor=K.+Kawaiauthor=K.+Izumiauthor=M.+Kumakiauthor=M.+Ofudeauthor=J.+Zhangauthor=E.+T.+Kellerauthor=M.+Namiki&title=Down-regulation+of+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2+by+androgen+deprivation+induces+castration-resistant+prostate+cancer&doi=10.1002%2Fpros.22533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulation of calcium/calmodulin-dependent protein kinase kinase 2 by androgen deprivation induces castration-resistant prostate cancer</span></div><div class="casAuthors">Shima, Takashi; Mizokami, Atsushi; Miyagi, Toru; Kawai, Keiichi; Izumi, Kouji; Kumaki, Misako; Ofude, Mitsuo; Zhang, Jian; Keller, Evan T.; Namiki, Mikio</div><div class="citationInfo"><span class="NLM_cas:title">Prostate (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1789-1801</span>CODEN:
                <span class="NLM_cas:coden">PRSTDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-4137</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND : Conversion into androgen-hypersensitive state and adaptation to the low concn. of androgen during ADT cause relapse of prostate cancer (PCa).  It is important to identify differentially expressed genes between PCa and normal prostate tissues and to reveal the function of these genes that are involved in progression of PCa.  METHODS : We performed cDNA microarray anal. to identify differentially expressed genes, calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2).  Immunohistochem. anal. was conducted to investigate the relationship between the CAMKK2 expression level and prognosis.  The function of CAMKK2 was assessed by generating CAMKK2 overexpressed LNCaP cells and by knockdown of CAMKK2.  RESULTS : We identified CAMKK2 overexpressed six times in PCa more than normal prostate by cDNA microarray anal.  Immunohistochem. anal. of CAMKK2 protein showed that CAMKK2 protein was expressed more in PCa than normal tissue.  However, the expression in the high-grade PCa diminished.  Moreover, the narrowness of CAMKK2-pos. area before ADT was a poor prognostic factor.  Androgen-deprivation treatment from the medium in which LNCaP cells were cultured in the presence of 10 nM DHT repressed CAMKK2 expression.  CAMKK2 overexpressed LNCaP cells (LNCaP/GFP-CAMKK2) attenuated androgen-sensitivity.  Tumorigenesis of LNCaP/GFP-CAMKK2 cells in male SCID mice was decreased compared with control cells irresp. of castration.  Finally, knockdown of CAMKK2 mRNA in LNCaP cells induced androgen-hypersensitivity and stimulated LNCaP cell proliferation.  CONCLUSIONS : Induction of androgen-hypersensitivity after ADT may be involved in down-regulation of CAMKK2.  This result may provide new therapeutic approach to keep androgen-sensitivity of PCa after ADT.  Prostate 72:1789-1801, 2012.  © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTdlaoqDTAMrVg90H21EOLACvtfcHk0liulXujTw8QQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGqtLzM&md5=cc116cc36ce22df02fb8f8da0e6dc737</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fpros.22533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22533%26sid%3Dliteratum%253Aachs%26aulast%3DShima%26aufirst%3DT.%26aulast%3DMizokami%26aufirst%3DA.%26aulast%3DMiyagi%26aufirst%3DT.%26aulast%3DKawai%26aufirst%3DK.%26aulast%3DIzumi%26aufirst%3DK.%26aulast%3DKumaki%26aufirst%3DM.%26aulast%3DOfude%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DE.%2BT.%26aulast%3DNamiki%26aufirst%3DM.%26atitle%3DDown-regulation%2520of%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520by%2520androgen%2520deprivation%2520induces%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProstate%26date%3D2012%26volume%3D72%26spage%3D1789%26epage%3D1801%26doi%3D10.1002%2Fpros.22533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subbannayya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbhuiya, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renuse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manju, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zameer, M. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brait, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srikumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaya Kumar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A.</span></span> <span> </span><span class="NLM_article-title">Calcium calmodulin dependent kinase kinase 2 - a novel therapeutic target for gastric adenocarcinoma</span>. <i>Canc. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">336</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.4161/15384047.2014.972264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.4161%2F15384047.2014.972264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25756516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWqt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=336-345&author=Y.+Subbannayyaauthor=N.+Syedauthor=M.+A.+Barbhuiyaauthor=R.+Rajaauthor=A.+Marimuthuauthor=N.+Sahasrabuddheauthor=S.+M.+Pintoauthor=S.+S.+Mandaauthor=S.+Renuseauthor=H.+Manjuauthor=M.+A.+L.+Zameerauthor=J.+Sharmaauthor=M.+Braitauthor=K.+Srikumarauthor=J.+C.+Roaauthor=M.+Vijaya+Kumarauthor=K.+V.+Kumarauthor=T.+K.+Prasadauthor=G.+Ramaswamyauthor=R.+V.+Kumarauthor=A.+Pandeyauthor=H.+Gowdaauthor=A.+Chatterjee&title=Calcium+calmodulin+dependent+kinase+kinase+2+-+a+novel+therapeutic+target+for+gastric+adenocarcinoma&doi=10.4161%2F15384047.2014.972264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium calmodulin dependent kinase kinase 2 - a novel therapeutic target for gastric adenocarcinoma</span></div><div class="casAuthors">Subbannayya, Yashwanth; Syed, Nazia; Barbhuiya, Mustafa A.; Raja, Remya; Marimuthu, Arivusudar; Sahasrabuddhe, Nandini; Pinto, Sneha M.; Manda, Srikanth Srinivas; Renuse, Santosh; Manju, Hc; Zameer, Mohammed Abdul Lateef; Sharma, Jyoti; Brait, Mariana; Srikumar, Kotteazeth; Roa, Juan Carlos; Vijaya Kumar, M.; Kumar, Kv veerendra; Prasad, Ts keshava; Ramaswamy, Girija; Kumar, Rekha Vijay; Pandey, Akhilesh; Gowda, Harsha; Chatterjee, Aditi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">336-345</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1555-8576</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Gastric cancer is one of the most common gastrointestinal malignancies and is assocd. with poor prognosis.  Exploring alterations in the proteomic landscape of gastric cancer is likely to provide potential biomarkers for early detection and mols. for targeted therapeutic intervention.  Using iTRAQ-based quant. proteomic anal., we identified 22 proteins that were overexpressed and 17 proteins that were downregulated in gastric tumor tissues as compared to the adjacent normal tissue.  Calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) was found to be 7-fold overexpressed in gastric tumor tissues.  Immunohistochem. labeling of tumor tissue microarrays for validation of CAMKK2 overexpression revealed that it was indeed overexpressed in 94% (92 of 98) of gastric cancer cases.  Silencing of CAMKK2 using siRNA significantly reduced cell proliferation, colony formation and invasion of gastric cancer cells.  Our results demonstrate that CAMKK2 signals in gastric cancer through AMPK activation and suggest that CAMKK2 could be a novel therapeutic target in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw67QaWTwi5rVg90H21EOLACvtfcHk0liulXujTw8QQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWqt7fL&md5=6969f1236282780560fbde50b1fa98c5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2F15384047.2014.972264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384047.2014.972264%26sid%3Dliteratum%253Aachs%26aulast%3DSubbannayya%26aufirst%3DY.%26aulast%3DSyed%26aufirst%3DN.%26aulast%3DBarbhuiya%26aufirst%3DM.%2BA.%26aulast%3DRaja%26aufirst%3DR.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DSahasrabuddhe%26aufirst%3DN.%26aulast%3DPinto%26aufirst%3DS.%2BM.%26aulast%3DManda%26aufirst%3DS.%2BS.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DManju%26aufirst%3DH.%26aulast%3DZameer%26aufirst%3DM.%2BA.%2BL.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DBrait%26aufirst%3DM.%26aulast%3DSrikumar%26aufirst%3DK.%26aulast%3DRoa%26aufirst%3DJ.%2BC.%26aulast%3DVijaya%2BKumar%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DK.%2BV.%26aulast%3DPrasad%26aufirst%3DT.%2BK.%26aulast%3DRamaswamy%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DR.%2BV.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DGowda%26aufirst%3DH.%26aulast%3DChatterjee%26aufirst%3DA.%26atitle%3DCalcium%2520calmodulin%2520dependent%2520kinase%2520kinase%25202%2520-%2520a%2520novel%2520therapeutic%2520target%2520for%2520gastric%2520adenocarcinoma%26jtitle%3DCanc.%2520Biol.%2520Ther.%26date%3D2015%26volume%3D16%26spage%3D336%26epage%3D345%26doi%3D10.4161%2F15384047.2014.972264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soderling, T. R.</span></span> <span> </span><span class="NLM_article-title">Calcium promotes cell survival through CaM-K kinase activation of the protein-kinase-B pathway</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/25147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2F25147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=9859994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWnsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=1998&pages=584-587&author=S.+Yanoauthor=H.+Tokumitsuauthor=T.+R.+Soderling&title=Calcium+promotes+cell+survival+through+CaM-K+kinase+activation+of+the+protein-kinase-B+pathway&doi=10.1038%2F25147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium promotes cell survival through CaM-K kinase activation of the protein-kinase-B pathway</span></div><div class="casAuthors">Yano, Shigetoshi; Tokumitsu, Hiroshi; Soderling, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">6711</span>),
    <span class="NLM_cas:pages">584-587</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The protection against apoptosis provided by growth factors in several cell lines is due to stimulation of the phosphatidylinositol-3-kinase (PI3K) pathway, which results in activation of protein kinase B (PKB; also known as c-Akt and Rac) and phosphorylation and sequestration to protein 14-3-3 of proapoptotic Bcl-2 family member BAD.  A modest increase in intracellular Ca2+ concn. also promotes survival of some cultured neurons through a pathway that requires calmodulin but is independent of PI3K and the MAP kinases.  Here, the authors report that Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK) activates PKB directly, resulting in phosphorylation of BAD on Ser-136 and the interaction of BAD with protein 14-3-3.  Serum withdrawal induced a 3- to 4-fld increase in cell death of NG108 neuroblastoma cells, and this apoptosis was largely blocked by increasing the intracellular Ca2+ concn. with NMDA (N-methyl-D-aspartate) or KCl or by transfection with constitutively active CaM-KK.  The effect of NMDA on cell survival was blocked by transfection with dominant-neg. forms of CaM-KK or PKB.  These results identify a Ca2+-triggered signaling cascade in which CaM-KK activates PKB, which in turn phosphorylates BAD and protects cells from apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPtIJx03Rrk7Vg90H21EOLACvtfcHk0ljaX8BYzQtkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWnsLo%253D&md5=a93a337e6ca963a4607fc294c1f7c5a8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F25147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F25147%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DS.%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DSoderling%26aufirst%3DT.%2BR.%26atitle%3DCalcium%2520promotes%2520cell%2520survival%2520through%2520CaM-K%2520kinase%2520activation%2520of%2520the%2520protein-kinase-B%2520pathway%26jtitle%3DNature%26date%3D1998%26volume%3D396%26spage%3D584%26epage%3D587%26doi%3D10.1038%2F25147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spengler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zibrova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, R.</span></span> <span> </span><span class="NLM_article-title">Protein kinase A negatively regulates VEGF-induced AMPK activation by phosphorylating CaMKK2 at serine 495</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">3453</span>– <span class="NLM_lpage">3469</span>, <span class="refDoi"> DOI: 10.1042/bcj20200555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1042%2FBCJ20200555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32869834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB38bivVOguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2020&pages=3453-3469&author=K.+Spenglerauthor=D.+Zibrovaauthor=A.+Woodsauthor=C.+G.+Langendorfauthor=J.+W.+Scottauthor=D.+Carlingauthor=R.+Heller&title=Protein+kinase+A+negatively+regulates+VEGF-induced+AMPK+activation+by+phosphorylating+CaMKK2+at+serine+495&doi=10.1042%2Fbcj20200555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase A negatively regulates VEGF-induced AMPK activation by phosphorylating CaMKK2 at serine 495</span></div><div class="casAuthors">Spengler Katrin; Zibrova Darya; Heller Regine; Woods Angela; Carling David; Langendorf Christopher G; Scott John W; Scott John W; Scott John W</div><div class="citationInfo"><span class="NLM_cas:title">The Biochemical journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3453-3469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activation of AMP-activated protein kinase (AMPK) in endothelial cells by vascular endothelial growth factor (VEGF) via the Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) represents a pro-angiogenic pathway, whose regulation and function is incompletely understood.  This study investigates whether the VEGF/AMPK pathway is regulated by cAMP-mediated signalling.  We show that cAMP elevation in endothelial cells by forskolin, an activator of the adenylate cyclase, and/or 3-isobutyl-1-methylxanthine (IBMX), an inhibitor of phosphodiesterases, triggers protein kinase A (PKA)-mediated phosphorylation of CaMKK2 (serine residues S495, S511) and AMPK (S487).  Phosphorylation of CaMKK2 by PKA led to an inhibition of its activity as measured in CaMKK2 immunoprecipitates of forskolin/IBMX-treated cells.  This inhibition was linked to phosphorylation of S495, since it was not seen in cells expressing a non-phosphorylatable CaMKK2 S495C mutant.  Phosphorylation of S511 alone in these cells was not able to inhibit CaMKK2 activity.  Moreover, phosphorylation of AMPK at S487 was not sufficient to inhibit VEGF-induced AMPK activation in cells, in which PKA-mediated CaMKK2 inhibition was prevented by expression of the CaMKK2 S495C mutant. cAMP elevation in endothelial cells reduced basal and VEGF-induced acetyl-CoA carboxylase (ACC) phosphorylation at S79 even if AMPK was not inhibited.  Together, this study reveals a novel regulatory mechanism of VEGF-induced AMPK activation by cAMP/PKA, which may explain, in part, inhibitory effects of PKA on angiogenic sprouting and play a role in balancing pro- and anti-angiogenic mechanisms in order to ensure functional angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJPdtvc19VG-IHSmpohpd0fW6udTcc2eZBKldEBb0efrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38bivVOguw%253D%253D&md5=8f29b7bf157e45a59cafaea00f8f6914</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2FBCJ20200555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20200555%26sid%3Dliteratum%253Aachs%26aulast%3DSpengler%26aufirst%3DK.%26aulast%3DZibrova%26aufirst%3DD.%26aulast%3DWoods%26aufirst%3DA.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DHeller%26aufirst%3DR.%26atitle%3DProtein%2520kinase%2520A%2520negatively%2520regulates%2520VEGF-induced%2520AMPK%2520activation%2520by%2520phosphorylating%2520CaMKK2%2520at%2520serine%2520495%26jtitle%3DBiochem.%2520J.%26date%3D2020%26volume%3D477%26spage%3D3453%26epage%3D3469%26doi%3D10.1042%2Fbcj20200555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gocher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azabdaftari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Euscher, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karacosta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franke, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span> <span> </span><span class="NLM_article-title">Akt activation by Ca(2+)/calmodulin-dependent protein kinase kinase 2 (CaMKK2) in ovarian cancer cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">14188</span>– <span class="NLM_lpage">14204</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.778464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M117.778464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=28634229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2iurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=14188-14204&author=A.+M.+Gocherauthor=G.+Azabdaftariauthor=L.+M.+Euscherauthor=S.+Daiauthor=L.+G.+Karacostaauthor=T.+F.+Frankeauthor=A.+M.+Edelman&title=Akt+activation+by+Ca%282%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+in+ovarian+cancer+cells&doi=10.1074%2Fjbc.m117.778464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Akt activation by Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) in ovarian cancer cells</span></div><div class="casAuthors">Gocher, Angela M.; Azabdaftari, Gissou; Euscher, Lindsey M.; Dai, Shuhang; Karacosta, Loukia G.; Franke, Thomas F.; Edelman, Arthur M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">14188-14204</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hyperactivation of Akt is assocd. with oncogenic changes in the growth, survival, and chemoresistance of cancer cells.  The PI3K/phosphoinositide-dependent kinase (PDK) 1 pathway represents the canonical mechanism for phosphorylation of Akt at its primary activation site, Thr-308.  We obsd. that Ca2+/calmodulin (CaM)-dependent protein kinase kinase 2 (β) (CaMKK2) is highly expressed in high-grade serous ovarian cancer, and we investigated its role in Akt activation in ovarian cancer (OVCa) cell lines (OVCAR-3, SKOV-3, and Caov-3).  Knockdown or pharmacol. inhibition of CaMKK2 produced phenotypes expected of Akt inhibition, including redns. in cell growth and cell viability and in the regulation of Akt downstream targets involved in G1/S transition and apoptosis.  CaMKK2 knockdown or inhibition decreased Akt phosphorylation at Thr-308 and Ser-473 to extents similar to those of PDK1 knockdown or PI3K inhibition.  Combined CaMKK2 and PDK1 knockdown or CaMKK and PI3K inhibition, resp., produced additive effects on p-Akt and cell growth, consistent with direct Akt phosphorylation by CaMKK2.  This conclusion was supported by the absence of effects of CaMKK2 knockdown/inhibition on alternative means of activating Akt via p-Akt Thr-450, p-PDK1 Ser-241, or p-IRS1 Ser-636/639.  Recombinant CaMKK2 directly activated recombinant Akt by phosphorylation at Thr-308 in a Ca2+/CaM-dependent manner.  In OVCa cells, p-Akt Thr-308 was significantly inhibited by intracellular Ca2+i chelation or CaM inhibition.  Ionomycin-induced Ca2+ influx promoted p-Akt, an effect blocked by PDK1, and/or CaMKK2, siRNAs, and by PI3K and/or CaMKK inhibitors.  CaMKK2 knockdown potentiated the effects of the chemotherapeutic drugs carboplatin and PX-866 to reduce proliferation and survival of OVCa cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7fTocaUYvgLVg90H21EOLACvtfcHk0ljaX8BYzQtkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2iurvE&md5=6926c83bd4825d1818c2fee251b0ac3c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.778464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.778464%26sid%3Dliteratum%253Aachs%26aulast%3DGocher%26aufirst%3DA.%2BM.%26aulast%3DAzabdaftari%26aufirst%3DG.%26aulast%3DEuscher%26aufirst%3DL.%2BM.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DKaracosta%26aufirst%3DL.%2BG.%26aulast%3DFranke%26aufirst%3DT.%2BF.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26atitle%3DAkt%2520activation%2520by%2520Ca%25282%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520in%2520ovarian%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D14188%26epage%3D14204%26doi%3D10.1074%2Fjbc.m117.778464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davare, M. A.</span></span> <span> </span><span class="NLM_article-title">ERK activation and cell growth require CaM kinases in MCF-7 breast cancer cells</span>. <i>Mol. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1007/s11010-009-0252-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2Fs11010-009-0252-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=19763792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2010&pages=155-171&author=J.+M.+Schmittauthor=E.+Abellauthor=A.+Wagnerauthor=M.+A.+Davare&title=ERK+activation+and+cell+growth+require+CaM+kinases+in+MCF-7+breast+cancer+cells&doi=10.1007%2Fs11010-009-0252-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">ERK activation and cell growth require CaM kinases in MCF-7 breast cancer cells</span></div><div class="casAuthors">Schmitt, John M.; Abell, Ellen; Wagner, Andrea; Davare, Monika A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">1&2</span>),
    <span class="NLM_cas:pages">155-171</span>CODEN:
                <span class="NLM_cas:coden">MCBIB8</span>;
        ISSN:<span class="NLM_cas:issn">0300-8177</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Previous studies on MCF-7 breast cancer cells have shown that the G-protein coupled receptor (GPCR) agonist carbachol increases intracellular calcium levels and the activation of extracellular signal-regulated kinase (ERK).  Calcium and calmodulin regulate the calcium/calmodulin-dependent kinase (CaM kinase) family of proteins that have been proposed to regulate ERK and gene transcription.  Our results suggest that both estrogen (E2) and carbachol treatment of MCF-7 breast cancer cells trigger phosphorylation of ERK1/2 and the transcription factor Elk-1.  Carbachol and estrogen triggered nearly a 4-6-fold increase in MCF-7 cell proliferation by 96 h, resp.  Carbachol-stimulated ERK activation and cell growth was completely blocked by the Muscarinic M3-subtype GPCR inhibitor, 4-DAMP, and siRNA against the M3-subtype GPCR.  Interestingly, blockade of CaM KK with the selective inhibitor STO-609 prevented carbachol activation CaM KI, ERK, Elk-1, and cell growth.  Consistent with these observations, knockdown of CaM KKα and CaM KIγ with shRNA-contg. plasmids blocked ERK activation by carbachol.  In addn., Elk-1 phosphorylation and luciferase activity in response to carbachol treatment was also dependent upon CaM kinases and was inhibited by U 0126, STO-609, and siRNA knockdown of CaM kinases and ERK2.  Finally, blockade of either CaM KK (with STO-609) or ERK (with U 0126) activities resulted in the inhibition of carbachol- and estrogen-mediated cyclin D1 expression and MCF-7 cell growth.  Taken together, our results suggest that carbachol treatment of MCF-7 cells activates CaM KI, ERK, the transcription factor Elk-1, cyclin D1, and cell growth through CaM KK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMIgFw4WiYhbVg90H21EOLACvtfcHk0lg_o_XyroHvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCksQ%253D%253D&md5=e6b4fa78e7d8148c99b6a1a715fbca56</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11010-009-0252-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11010-009-0252-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DJ.%2BM.%26aulast%3DAbell%26aufirst%3DE.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DDavare%26aufirst%3DM.%2BA.%26atitle%3DERK%2520activation%2520and%2520cell%2520growth%2520require%2520CaM%2520kinases%2520in%2520MCF-7%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Cell.%2520Biochem.%26date%3D2010%26volume%3D335%26spage%3D155%26epage%3D171%26doi%3D10.1007%2Fs11010-009-0252-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">CaM kinase kinase beta-mediated activation of the growth regulatory kinase AMPK is required for androgen-dependent migration of prostate cancer cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">528</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-2581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-10-2581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21098087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsFKltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=528-537&author=D.+E.+Frigoauthor=M.+K.+Howeauthor=B.+M.+Wittmannauthor=A.+M.+Brunnerauthor=I.+Cushmanauthor=Q.+Wangauthor=M.+Brownauthor=A.+R.+Meansauthor=D.+P.+McDonnell&title=CaM+kinase+kinase+beta-mediated+activation+of+the+growth+regulatory+kinase+AMPK+is+required+for+androgen-dependent+migration+of+prostate+cancer+cells&doi=10.1158%2F0008-5472.can-10-2581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CaM Kinase Kinase β-Mediated Activation of the Growth Regulatory Kinase AMPK Is Required for Androgen-Dependent Migration of Prostate Cancer Cells</span></div><div class="casAuthors">Frigo, Daniel E.; Howe, Matthew K.; Wittmann, Bryan M.; Brunner, Abigail M.; Cushman, Ian; Wang, Qianben; Brown, Myles; Means, Anthony R.; McDonnell, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">528-537</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">While patients with advanced prostate cancer initially respond favorably to androgen ablation therapy, most experience a relapse of the disease within 1-2 years.  Although hormone-refractory disease is unresponsive to androgen-deprivation, androgen receptor (AR)-regulated signaling pathways remain active and are necessary for cancer progression.  Thus, both AR itself and the processes downstream of the receptor remain viable targets for therapeutic intervention.  Microarray anal. of multiple clin. cohorts showed that the serine/threonine kinase Ca2+/calmodulin-dependent protein kinase kinase β (CaMKKβ) is both highly expressed in the prostate and further elevated in prostate cancers.  Using cellular models of prostate cancer, we have detd. that androgens (a) directly increase the expression of a CaMKKβ splice variant and (b) increase functional CaMKKβ protein levels as detd. by the phosphorylation of both CaMKI and AMP-activated protein kinase (AMPK), two of CaMKKβ's primary substrates.  Importantly, inhibition of the CaMKKβ-AMPK, but not CaMKI, signaling axis in prostate cancer cells by pharmacol. inhibitors or siRNA-mediated knockdown blocks androgen-mediated migration and invasion.  Conversely, overexpression of CaMKKβ alone leads to both increased AMPK phosphorylation and cell migration.  Given the key roles of CaMKKβ and AMPK in the biol. of prostate cancer cells, we propose that these enzymes are potential therapeutic targets in prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkTeJeY6bKQrVg90H21EOLACvtfcHk0lg_o_XyroHvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsFKltw%253D%253D&md5=d621e7ca9bd660cc0bb2c2a332adf4ca</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2581%26sid%3Dliteratum%253Aachs%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26aulast%3DHowe%26aufirst%3DM.%2BK.%26aulast%3DWittmann%26aufirst%3DB.%2BM.%26aulast%3DBrunner%26aufirst%3DA.%2BM.%26aulast%3DCushman%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DCaM%2520kinase%2520kinase%2520beta-mediated%2520activation%2520of%2520the%2520growth%2520regulatory%2520kinase%2520AMPK%2520is%2520required%2520for%2520androgen-dependent%2520migration%2520of%2520prostate%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D528%26epage%3D537%26doi%3D10.1158%2F0008-5472.can-10-2581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.-d.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.-x.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.-s.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.-n.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W.-d.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-224 and its target CAMKK2 synergistically influence tumor progression and patient prognosis in prostate cancer</span>. <i>Tumour Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1991</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2805-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2Fs13277-014-2805-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25394900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC2M3nvFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1983-1991&author=H.+Fuauthor=H.-c.+Heauthor=Z.-d.+Hanauthor=Y.-p.+Wanauthor=H.-w.+Luoauthor=Y.-q.+Huangauthor=C.+Caiauthor=Y.-x.+Liangauthor=Q.-s.+Daiauthor=F.-n.+Jiangauthor=W.-d.+Zhong&title=MicroRNA-224+and+its+target+CAMKK2+synergistically+influence+tumor+progression+and+patient+prognosis+in+prostate+cancer&doi=10.1007%2Fs13277-014-2805-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-224 and its target CAMKK2 synergistically influence tumor progression and patient prognosis in prostate cancer</span></div><div class="casAuthors">Fu Hao; He Hui-chan; Han Zhao-dong; Wan Yue-ping; Luo Hong-wei; Huang Ya-qiang; Cai Chao; Liang Yu-xiang; Dai Qi-shan; Jiang Fu-neng; Zhong Wei-de</div><div class="citationInfo"><span class="NLM_cas:title">Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1983-91</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We previously demonstrated that microRNA (miR)-224 expression was significantly reduced in human prostate cancer (PCa) tissues and predicted unfavorable prognosis in patients.  However, the underlying mechanisms of miR-224 have not been fully elucidated.  In this study, calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) was identified as a target gene of miR-224.  Then, we found that enforced expression of miR-224 could suppress PCa cell proliferation and cell cycle by regulating the expression of CAMKK2 in vitro.  In addition, the expression levels of miR-224 in PCa tissues were negatively correlated with those of CAMKK2 mRNA significantly (Spearman's correlation: r = -0.66, P = 0.004).  Moreover, combined low miR-224 expression and high CAMKK2 expression (miR-224-low/CAMKK2-high) was closely correlated with advanced clinical stage (P = 0.028).  Furthermore, PCa patients with miR-224-low/CAMKK2-high expression more frequently had shorter overall survival than those in groups with other expression patterns of two molecules.  In conclusion, our data offer the convincing evidence that miR-224 and its target gene CAMKK2 may synergistically contribute to the malignant progression of PCa.  Combined detection of miR-224 and CAMKK2 expressions represents an efficient predictor of patient prognosis and may be a novel marker which can provide additional prognostic information in PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyfAqmJ0zmKxwLZxei1HN8fW6udTcc2eYYuVDJylBCXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3nvFertQ%253D%253D&md5=8794d11ca3f885f8ccdbd45681caa965</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2805-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2805-0%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.-c.%26aulast%3DHan%26aufirst%3DZ.-d.%26aulast%3DWan%26aufirst%3DY.-p.%26aulast%3DLuo%26aufirst%3DH.-w.%26aulast%3DHuang%26aufirst%3DY.-q.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DY.-x.%26aulast%3DDai%26aufirst%3DQ.-s.%26aulast%3DJiang%26aufirst%3DF.-n.%26aulast%3DZhong%26aufirst%3DW.-d.%26atitle%3DMicroRNA-224%2520and%2520its%2520target%2520CAMKK2%2520synergistically%2520influence%2520tumor%2520progression%2520and%2520patient%2520prognosis%2520in%2520prostate%2520cancer%26jtitle%3DTumour%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D1983%26epage%3D1991%26doi%3D10.1007%2Fs13277-014-2805-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsouko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkenfeld, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakili, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulliam, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katreddy, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span> <span> </span><span class="NLM_article-title">GLUT12 promotes prostate cancer cell growth and is regulated by androgens and CaMKK2 signaling</span>. <i>Endocr. Relat. Canc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1530/erc-17-0051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1530%2FERC-17-0051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29431615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCksbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=453-469&author=M.+A.+Whiteauthor=E.+Tsoukoauthor=C.+Linauthor=K.+Rajapaksheauthor=J.+M.+Spencerauthor=S.+R.+Wilkenfeldauthor=S.+S.+Vakiliauthor=T.+L.+Pulliamauthor=D.+Awadauthor=F.+Nikolosauthor=R.+R.+Katreddyauthor=B.+A.+Kaipparettuauthor=A.+Sreekumarauthor=X.+Zhangauthor=E.+Cheungauthor=C.+Coarfaauthor=D.+E.+Frigo&title=GLUT12+promotes+prostate+cancer+cell+growth+and+is+regulated+by+androgens+and+CaMKK2+signaling&doi=10.1530%2Ferc-17-0051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">GLUT12 promotes prostate cancer cell growth and is regulated by androgens and CaMKK2 signaling</span></div><div class="casAuthors">White, Mark A.; Tsouko, Efrosini; Lin, Chenchu; Rajapakshe, Kimal; Spencer, Jeffrey M.; Wilkenfeld, Sandi R.; Vakili, Sheiva S.; Pulliam, Thomas L.; Awad, Dominik; Nikolos, Fotis; Katreddy, Rajasekhara Reddy; Kaipparettu, Benny Abraham; Sreekumar, Arun; Zhang, Xiaoliu; Cheung, Edwin; Coarfa, Cristian; Frigo, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine-Related Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-469</span>CODEN:
                <span class="NLM_cas:coden">ERCAE9</span>;
        ISSN:<span class="NLM_cas:issn">1351-0088</span>.
    
            (<span class="NLM_cas:orgname">BioScientifica Ltd.</span>)
        </div><div class="casAbstract">Despite altered metab. being an accepted hallmark of cancer, it is still not completely understood which signaling pathways regulate these processes.  Given the central role of androgen receptor (AR) signaling in prostate cancer, we hypothesized that AR could promote prostate cancer cell growth in part through increasing glucose uptake via the expression of distinct glucose transporters.  Here, we detd. that AR directly increased the expression of SLC2A12, the gene that encodes the glucose transporter GLUT12.  In support of these findings, gene signatures of AR activity correlated with SLC2A12 expression in multiple clin. cohorts.  Functionally, GLUT12 was required for maximal androgen mediated glucose uptake and cell growth in LNCaP and VCaP cells.  Knockdown of GLUT12 also decreased the growth of C4-2, 22Rv1 and AR-neg. PC-3 cells.  This latter observation corresponded with a significant redn. in glucose uptake, indicating that addnl. signaling mechanisms could augment GLUT12 function in an AR-independent manner.  Interestingly, GLUT12 trafficking to the plasma membrane was modulated by calcium/ calmodulin-dependent protein kinase kinase 2 (CaMKK2)-5'-AMP-activated protein kinase (AMPK) signaling, a pathway we previously demonstrated to be a downstream effector of AR.  Inhibition of CaMKK2-AMPK signaling decreased GLUT12 translocation to the plasma membrane by inhibiting the phosphorylation of TBC1D4, a known regulator of glucose transport.  Further, AR increased TBC1D4 expression.  Correspondingly, expression of TBC1D4 correlated with AR activity in prostate cancer patient samples.  Taken together, these data demonstrate that prostate cancer cells can increase the functional levels of GLUT12 through multiple mechanisms to promote glucose uptake and subsequent cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqvR94mNVUB7Vg90H21EOLACvtfcHk0lii6Tp18kRrDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCksbjK&md5=ad31763ec66f56ea721a4d5887249004</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1530%2FERC-17-0051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-17-0051%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DM.%2BA.%26aulast%3DTsouko%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DSpencer%26aufirst%3DJ.%2BM.%26aulast%3DWilkenfeld%26aufirst%3DS.%2BR.%26aulast%3DVakili%26aufirst%3DS.%2BS.%26aulast%3DPulliam%26aufirst%3DT.%2BL.%26aulast%3DAwad%26aufirst%3DD.%26aulast%3DNikolos%26aufirst%3DF.%26aulast%3DKatreddy%26aufirst%3DR.%2BR.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DSreekumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheung%26aufirst%3DE.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26atitle%3DGLUT12%2520promotes%2520prostate%2520cancer%2520cell%2520growth%2520and%2520is%2520regulated%2520by%2520androgens%2520and%2520CaMKK2%2520signaling%26jtitle%3DEndocr.%2520Relat.%2520Canc.%26date%3D2018%26volume%3D25%26spage%3D453%26epage%3D469%26doi%3D10.1530%2Ferc-17-0051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komorizono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanbara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saibara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okanoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Japan Nash
Study Group, M. o. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welfare of, J.</span></span> <span> </span><span class="NLM_article-title">Characteristics of patients with nonalcoholic steatohepatitis who develop hepatocellular carcinoma</span>. <i>Clin. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.cgh.2011.01.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cgh.2011.01.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21320639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvls1ylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=428-433&author=K.+Yasuiauthor=E.+Hashimotoauthor=Y.+Komorizonoauthor=K.+Koikeauthor=S.+Ariiauthor=Y.+Imaiauthor=T.+Shimaauthor=Y.+Kanbaraauthor=T.+Saibaraauthor=T.+Moriauthor=S.+Kawataauthor=H.+Utoauthor=S.+Takamiauthor=Y.+Sumidaauthor=T.+Takamuraauthor=M.+Kawanakaauthor=T.+Okanoueauthor=M.+o.+H.+L.+Japan+Nash%0AStudy+Groupauthor=J.+Welfare+of&title=Characteristics+of+patients+with+nonalcoholic+steatohepatitis+who+develop+hepatocellular+carcinoma&doi=10.1016%2Fj.cgh.2011.01.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Characteristics of patients with nonalcoholic steatohepatitis who develop hepatocellular carcinoma</span></div><div class="casAuthors">Yasui Kohichiroh; Hashimoto Etsuko; Komorizono Yasuji; Koike Kazuhiko; Arii Shigeki; Imai Yasuharu; Shima Toshihide; Kanbara Yoshihiro; Saibara Toshiji; Mori Takahiro; Kawata Sumio; Uto Hirofumi; Takami Shiro; Sumida Yoshio; Takamura Toshinari; Kawanaka Miwa; Okanoue Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical gastroenterology and hepatology : the official clinical practice journal of the American Gastroenterological Association</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">428-33; quiz e50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  Nonalcoholic steatohepatitis (NASH) can progress to hepatocellular carcinoma (HCC).  We aimed to characterize the clinical features of NASH patients with HCC.  METHODS:  In a cross-sectional multicenter study in Japan, we examined 87 patients (median age, 72 years; 62% male) with histologically proven NASH who developed HCC.  The clinical data were collected at the time HCC was diagnosed.  RESULTS:  Obesity (body mass index ≥25 kg/m(2)), diabetes, dyslipidemia, and hypertension were present in 54 (62%), 51 (59%), 24 (28%), and 47 (55%) patients, respectively.  In nontumor liver tissues, the degree of fibrosis was stage 1 in 10 patients (11%), stage 2 in 15 (17%), stage 3 in 18 (21%), and stage 4 (ie, liver cirrhosis) in 44 (51%).  The prevalence of cirrhosis was significantly lower among male patients (21 of 54, 39%) compared with female patients (23 of 33, 70%) (P = .008).  CONCLUSIONS:  Most patients with NASH who develop HCC are men; the patients have high rates of obesity, diabetes, and hypertension.  Male patients appear to develop HCC at a less advanced stage of liver fibrosis than female patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQPAIlg7wMzs0kUaC_kzhvyfW6udTcc2eZBkxGs6Dk2Ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvls1ylsA%253D%253D&md5=d86986d6edede3d8d3eb97bfd2e6579b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cgh.2011.01.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cgh.2011.01.023%26sid%3Dliteratum%253Aachs%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DE.%26aulast%3DKomorizono%26aufirst%3DY.%26aulast%3DKoike%26aufirst%3DK.%26aulast%3DArii%26aufirst%3DS.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DShima%26aufirst%3DT.%26aulast%3DKanbara%26aufirst%3DY.%26aulast%3DSaibara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DKawata%26aufirst%3DS.%26aulast%3DUto%26aufirst%3DH.%26aulast%3DTakami%26aufirst%3DS.%26aulast%3DSumida%26aufirst%3DY.%26aulast%3DTakamura%26aufirst%3DT.%26aulast%3DKawanaka%26aufirst%3DM.%26aulast%3DOkanoue%26aufirst%3DT.%26aulast%3DJapan%2BNash%2BStudy%2BGroup%26aufirst%3DM.%2Bo.%2BH.%2BL.%26aulast%3DWelfare%2Bof%26aufirst%3DJ.%26atitle%3DCharacteristics%2520of%2520patients%2520with%2520nonalcoholic%2520steatohepatitis%2520who%2520develop%2520hepatocellular%2520carcinoma%26jtitle%3DClin.%2520Gastroenterol.%2520Hepatol.%26date%3D2011%26volume%3D9%26spage%3D428%26epage%3D433%26doi%3D10.1016%2Fj.cgh.2011.01.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouso, K.</span></span> <span> </span><span class="NLM_article-title">Nonalcoholic steatohepatitis-associated hepatocellular carcinoma: our case series and literature review</span>. <i>World J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1436</span>– <span class="NLM_lpage">1441</span>, <span class="refDoi"> DOI: 10.3748/wjg.v16.i12.1436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3748%2Fwjg.v16.i12.1436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20333782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1Crs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1436-1441&author=Y.+Takumaauthor=K.+Nouso&title=Nonalcoholic+steatohepatitis-associated+hepatocellular+carcinoma%3A+our+case+series+and+literature+review&doi=10.3748%2Fwjg.v16.i12.1436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Nonalcoholic steatohepatitis-associated hepatocellular carcinoma: our case series and literature review</span></div><div class="casAuthors">Takuma, Yoshitaka; Nouso, Kazuhiro</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1436-1441</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Recently, nonalcoholic steatohepatitis (NASH) has been considered to be another cause of liver cirrhosis and hepatocellular carcinoma (HCC).  The natural history and prognosis of NASH are controversial.  Accordingly, we assessed the clinicopathol. features of NASH-assocd. HCC in our experience and reviewed the literature of NASH-assocd. HCC.  We experienced 11 patients with NASH-assocd. HCC (6 male, 5 female; mean age 73.8 ± 4.9 years) who received curative treatments.  Most (91%) patients had been diagnosed with obesity, diabetes, hypertension, or dyslipidemia.  Seven patients (64%) also had a non-cirrhotic liver.  The recurrence-free survival rates at 1, 3 and 5 years were 72%, 60%, and 60%.  We also summarized and reviewed 94 cases of NASH-assocd. HCC which were reported in the literature (64 male; mean age 66 years).  The majority of patients (68%) were obese, 66% of patients had diabetes, and 24% had dyslipidemia.  Furthermore, 26% of the HCCs arose from the non-cirrhotic liver.  In conclusion, patients with non-cirrhotic NASH may be a high-risk group for HCC, and regular surveillance for HCC is necessary in non-cirrhotic NASH patients as well as cirrhotic patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr78pJsnUvf07Vg90H21EOLACvtfcHk0lhFt7uCgWnheQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1Crs7Y%253D&md5=0654b46d6f47aee1c673c6ae8663020f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v16.i12.1436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v16.i12.1436%26sid%3Dliteratum%253Aachs%26aulast%3DTakuma%26aufirst%3DY.%26aulast%3DNouso%26aufirst%3DK.%26atitle%3DNonalcoholic%2520steatohepatitis-associated%2520hepatocellular%2520carcinoma%253A%2520our%2520case%2520series%2520and%2520literature%2520review%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2010%26volume%3D16%26spage%3D1436%26epage%3D1441%26doi%3D10.3748%2Fwjg.v16.i12.1436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">York, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gooden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maity, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putluri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of CaMKK2 with the selective antagonist STO-609 regresses NAFLD</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11793</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12139-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-017-12139-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=28924233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC1cbpvFeqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11793&author=B.+Yorkauthor=F.+Liauthor=F.+Linauthor=K.+L.+Marceloauthor=J.+Maoauthor=A.+Deanauthor=N.+Gonzalesauthor=D.+Goodenauthor=S.+Maityauthor=C.+Coarfaauthor=N.+Putluriauthor=A.+R.+Means&title=Pharmacological+inhibition+of+CaMKK2+with+the+selective+antagonist+STO-609+regresses+NAFLD&doi=10.1038%2Fs41598-017-12139-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of CaMKK2 with the selective antagonist STO-609 regresses NAFLD</span></div><div class="casAuthors">York Brian; Li Feng; Lin Fumin; Marcelo Kathrina L; Mao Jianqiang; Dean Adam; Gonzales Naomi; Maity Suman; Coarfa Cristian; Putluri Nagireddy; Means Anthony R; York Brian; Maity Suman; Coarfa Cristian; Means Anthony R; Li Feng; Gonzales Naomi; Means Anthony R; Gooden David</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11793</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Binding of calcium to its intracellular receptor calmodulin (CaM) activates a family of Ca(2+)/CaM-dependent protein kinases.  CaMKK2 (Ca(2+)/CaM-dependent protein kinase kinase 2) is a central member of this kinase family as it controls the actions of a CaMK cascade involving CaMKI, CaMKIV or AMPK.  CaMKK2 controls insulin signaling, metabolic homeostasis, inflammation and cancer cell growth highlighting its potential as a therapeutic target for a variety of diseases.  STO-609 is a selective, small molecule inhibitor of CaMKK2.  Although STO-609 has been used extensively in vitro and in cells to characterize and define new mechanistic functions of CaMKK2, only a few studies have reported the in vivo use of STO-609.  We synthesized functional STO-609 and assessed its pharmacological properties through in vitro (kinase assay), ex vivo (human liver microsomes) and in vivo (mouse) model systems.  We describe the metabolic processing of STO-609, its toxicity, pharmacokinetics and bioavailability in a variety of mouse tissues.  Utilizing these data, we show STO-609 treatment to inhibit CaMKK2 function confers protection against non-alcoholic fatty liver disease.  These data provide a valuable resource by establishing criteria for use of STO-609 to inhibit the in vivo functions of CaMKK2 and demonstrate its utility for treating metabolically-related hepatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBljNsbBpYgkh5eUSa2oe_fW6udTcc2ea_C7Vp9l8AsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbpvFeqtQ%253D%253D&md5=6dc5caaa28569c21b3cae14375ed714b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12139-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12139-3%26sid%3Dliteratum%253Aachs%26aulast%3DYork%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DMarcelo%26aufirst%3DK.%2BL.%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DDean%26aufirst%3DA.%26aulast%3DGonzales%26aufirst%3DN.%26aulast%3DGooden%26aufirst%3DD.%26aulast%3DMaity%26aufirst%3DS.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DPutluri%26aufirst%3DN.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DPharmacological%2520inhibition%2520of%2520CaMKK2%2520with%2520the%2520selective%2520antagonist%2520STO-609%2520regresses%2520NAFLD%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11793%26doi%3D10.1038%2Fs41598-017-12139-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckholz, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selley, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearer, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faggioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piubelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. Y.</span></span> <span> </span><span class="NLM_article-title">An orally available, brain-penetrant CAMKK2 inhibitor reduces food intake in rodent model</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1958</span>– <span class="NLM_lpage">1963</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bmcl.2018.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29653895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1SitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1958-1963&author=D.+J.+Priceauthor=D.+H.+Drewryauthor=L.+T.+Schallerauthor=B.+D.+Thompsonauthor=P.+R.+Reidauthor=P.+R.+Maloneyauthor=X.+Liangauthor=P.+Bankerauthor=R.+G.+Buckholzauthor=P.+K.+Selleyauthor=O.+B.+McDonaldauthor=J.+L.+Smithauthor=T.+W.+Shearerauthor=R.+F.+Coxauthor=S.+P.+Williamsauthor=R.+A.+Reidauthor=S.+Tacconiauthor=F.+Faggioniauthor=C.+Piubelliauthor=I.+Sartoriauthor=M.+Tessariauthor=T.+Y.+Wang&title=An+orally+available%2C+brain-penetrant+CAMKK2+inhibitor+reduces+food+intake+in+rodent+model&doi=10.1016%2Fj.bmcl.2018.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">An orally available, brain-penetrant CAMKK2 inhibitor reduces food intake in rodent model</span></div><div class="casAuthors">Price, Daniel J.; Drewry, David H.; Schaller, Lee T.; Thompson, Brian D.; Reid, Paul R.; Maloney, Patrick R.; Liang, Xi; Banker, Periette; Buckholz, Richard G.; Selley, Paula K.; McDonald, Octerloney B.; Smith, Jeffery L.; Shearer, Todd W.; Cox, Richard F.; Williams, Shawn P.; Reid, Robert A.; Tacconi, Stefano; Faggioni, Federico; Piubelli, Chiara; Sartori, Ilaria; Tessari, Michela; Wang, Tony Y.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1958-1963</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hypothalamic CAMKK2 represents a potential mechanism for chem. affecting satiety and promoting wt. loss in clin. obese patients.  Single-digit nanomolar inhibitors of CAMKK2 were identified in three related ATP-competitive series.  Limited optimization of kinase selectivity, soly., and pharmacokinetic properties were undertaken on all three series, as SAR was often transferrable.  Ultimately, a 2,4-diaryl 7-azaindole was optimized to afford a tool mol. that potently inhibits AMPK phosphorylation in a hypothalamus-derived cell line, is orally bioavailable, and crosses the blood-brain barrier.  When dosed orally in rodents, compd. 4t limited ghrelin-induced food intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE-unQLVd-VrVg90H21EOLACvtfcHk0liWoS8mO5oHQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1SitLg%253D&md5=5013a14295c1eb6bf07a0ff7f3bef684</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DSchaller%26aufirst%3DL.%2BT.%26aulast%3DThompson%26aufirst%3DB.%2BD.%26aulast%3DReid%26aufirst%3DP.%2BR.%26aulast%3DMaloney%26aufirst%3DP.%2BR.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DBanker%26aufirst%3DP.%26aulast%3DBuckholz%26aufirst%3DR.%2BG.%26aulast%3DSelley%26aufirst%3DP.%2BK.%26aulast%3DMcDonald%26aufirst%3DO.%2BB.%26aulast%3DSmith%26aufirst%3DJ.%2BL.%26aulast%3DShearer%26aufirst%3DT.%2BW.%26aulast%3DCox%26aufirst%3DR.%2BF.%26aulast%3DWilliams%26aufirst%3DS.%2BP.%26aulast%3DReid%26aufirst%3DR.%2BA.%26aulast%3DTacconi%26aufirst%3DS.%26aulast%3DFaggioni%26aufirst%3DF.%26aulast%3DPiubelli%26aufirst%3DC.%26aulast%3DSartori%26aufirst%3DI.%26aulast%3DTessari%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%2BY.%26atitle%3DAn%2520orally%2520available%252C%2520brain-penetrant%2520CAMKK2%2520inhibitor%2520reduces%2520food%2520intake%2520in%2520rodent%2520model%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D1958%26epage%3D1963%26doi%3D10.1016%2Fj.bmcl.2018.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribar, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noeldner, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlbauer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witters, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Hypothalamic CaMKK2 contributes to the regulation of energy balance</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2008.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cmet.2008.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18460329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtFeitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=377-388&author=K.+A.+Andersonauthor=T.+J.+Ribarauthor=F.+Linauthor=P.+K.+Noeldnerauthor=M.+F.+Greenauthor=M.+J.+Muehlbauerauthor=L.+A.+Wittersauthor=B.+E.+Kempauthor=A.+R.+Means&title=Hypothalamic+CaMKK2+contributes+to+the+regulation+of+energy+balance&doi=10.1016%2Fj.cmet.2008.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Hypothalamic CaMKK2 contributes to the regulation of energy balance</span></div><div class="casAuthors">Anderson, Kristin A.; Ribar, Thomas J.; Lin, Fumin; Noeldner, Pamela K.; Green, Michelle F.; Muehlbauer, Michael J.; Witters, Lee A.; Kemp, Bruce E.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">377-388</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Detailed knowledge of the pathways by which ghrelin and leptin signal to AMPK in hypothalamic neurons and lead to regulation of appetite and glucose homeostasis is central to the development of effective means to combat obesity.  Here the authors identify CaMKK2 as a component of one of these pathways, show that it regulates hypothalamic prodn. of the orexigenic hormone NPY, provide evidence that it functions as an AMPKα kinase in the hypothalamus, and demonstrate that it forms a unique signaling complex with AMPKα and β.  Acute pharmacol. inhibition of CaMKK2 in wild-type mice, but not CaMKK2 null mice, inhibits appetite and promotes wt. loss consistent with decreased NPY and AgRP mRNAs.  Moreover, the loss of CaMKK2 protects mice from high-fat diet-induced obesity, insulin resistance, and glucose intolerance.  These data underscore the potential of targeting CaMKK2 as a therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn7ikRZ3NqH7Vg90H21EOLACvtfcHk0liWoS8mO5oHQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtFeitbc%253D&md5=4cb56cffd6e8aa32bf1b38355223ad16</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DRibar%26aufirst%3DT.%2BJ.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DNoeldner%26aufirst%3DP.%2BK.%26aulast%3DGreen%26aufirst%3DM.%2BF.%26aulast%3DMuehlbauer%26aufirst%3DM.%2BJ.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DHypothalamic%2520CaMKK2%2520contributes%2520to%2520the%2520regulation%2520of%2520energy%2520balance%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D7%26spage%3D377%26epage%3D388%26doi%3D10.1016%2Fj.cmet.2008.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngoei, K. R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAloon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhagat, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, N. X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dite, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span> <span> </span><span class="NLM_article-title">CaMKK2 is inactivated by cAMP-PKA signaling and 14-3-3 adaptor proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">16239</span>– <span class="NLM_lpage">16250</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra120.013756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.RA120.013756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32913128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1CjsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=16239-16250&author=C.+G.+Langendorfauthor=M.+T.+O%E2%80%99Brienauthor=K.+R.+W.+Ngoeiauthor=L.+M.+McAloonauthor=U.+Dhagatauthor=A.+Hoqueauthor=N.+X.+Y.+Lingauthor=T.+A.+Diteauthor=S.+Galicauthor=K.+Lohauthor=M.+W.+Parkerauthor=J.+S.+Oakhillauthor=B.+E.+Kempauthor=J.+W.+Scott&title=CaMKK2+is+inactivated+by+cAMP-PKA+signaling+and+14-3-3+adaptor+proteins&doi=10.1074%2Fjbc.ra120.013756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">CaMKK2 is inactivated by cAMP-PKA signaling and 14-3-3 adaptor proteins</span></div><div class="casAuthors">Langendorf, Christopher G.; O'Brien, Matthew T.; Ngoei, Kevin R. W.; McAloon, Luke M.; Dhagat, Urmi; Hoque, Ashfaqul; Ling, Naomi X. Y.; Dite, Toby A.; Galic, Sandra; Loh, Kim; Parker, Michael W.; Oakhill, Jonathan S.; Kemp, Bruce E.; Scott, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16239-16250</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The calcium-calmodulin-dependent protein kinase kinase-2 (CaMKK2) is a key regulator of cellular and whole-body energy metab.  It is known to be activated by increases in intracellular Ca2+, but the mechanisms by which it is inactivated are less clear.  CaMKK2 inhibition protects against prostate cancer, hepatocellular carcinoma, and metabolic derangements induced by a high-fat diet; therefore, elucidating the intracellular mechanisms that inactivate CaMKK2 has important therapeutic implications.  Here we show that stimulation of cAMP-dependent protein kinase A (PKA) signaling in cells inactivates CaMKK2 by phosphorylation of three conserved serine residues.  PKA-dependent phosphorylation of Ser495 directly impairs calcium-calmodulin activation, whereas phosphorylation of Ser100 and Ser511 mediate recruitment of 14-3-3 adaptor proteins that hold CaMKK2 in the inactivated state by preventing dephosphorylation of phospho-Ser495.  We also report the crystal structure of 14-3-3ζ bound to a synthetic diphosphorylated peptide that reveals how the canonical (Ser511) and noncanonical (Ser100) 14-3-3 consensus sites on CaMKK2 cooperate to bind 14-3-3 proteins.  Our findings provide detailed mol. insights into how cAMP-PKA signaling inactivates CaMKK2 and reveals a pathway to inhibit CaMKK2 with potential for treating human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSuF1Vc9L8ObVg90H21EOLACvtfcHk0lgP73NDmOOnzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1CjsLrF&md5=9c2ebf5e227d37cb77074dd10134c2b1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA120.013756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA120.013756%26sid%3Dliteratum%253Aachs%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BT.%26aulast%3DNgoei%26aufirst%3DK.%2BR.%2BW.%26aulast%3DMcAloon%26aufirst%3DL.%2BM.%26aulast%3DDhagat%26aufirst%3DU.%26aulast%3DHoque%26aufirst%3DA.%26aulast%3DLing%26aufirst%3DN.%2BX.%2BY.%26aulast%3DDite%26aufirst%3DT.%2BA.%26aulast%3DGalic%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26atitle%3DCaMKK2%2520is%2520inactivated%2520by%2520cAMP-PKA%2520signaling%2520and%252014-3-3%2520adaptor%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D16239%26epage%3D16250%26doi%3D10.1074%2Fjbc.ra120.013756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cary, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racioppi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novack, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ca(2)(+)/calmodulin-dependent protein kinase kinase 2 stimulates osteoblast formation and inhibits osteoclast differentiation</span>. <i>J. Bone Miner. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1002/jbmr.1890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fjbmr.1890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=23408651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVOmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2013&pages=1599-1610&author=R.+L.+Caryauthor=S.+Waddellauthor=L.+Racioppiauthor=F.+Longauthor=D.+V.+Novackauthor=M.+J.+Voorauthor=U.+Sankar&title=Inhibition+of+Ca%282%29%28%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+2+stimulates+osteoblast+formation+and+inhibits+osteoclast+differentiation&doi=10.1002%2Fjbmr.1890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Ca2+/Calmodulin-Dependent Protein Kinase Kinase 2 Stimulates Osteoblast Formation and Inhibits Osteoclast Differentiation</span></div><div class="casAuthors">Cary, Rachel L.; Waddell, Seid; Racioppi, Luigi; Long, Fanxin; Novack, Deborah V.; Voor, Michael J.; Sankar, Uma</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bone and Mineral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">JBMREJ</span>;
        ISSN:<span class="NLM_cas:issn">0884-0431</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Bone remodeling, a physiol. process characterized by bone formation by osteoblasts (OBs) and resorption of preexisting bone matrix by osteoclasts (OCs), is vital for the maintenance of healthy bone tissue in adult humans.  Imbalances in this vital process result in pathol. conditions including osteoporosis.  Owing to its initial asymptomatic nature, osteoporosis is often detected only after the patient has sustained significant bone loss or a fracture.  Hence, anabolic therapeutics that stimulate bone accrual is in high clin. demand.  Here we identify Ca2+/calmodulin (CaM)-dependent protein kinase kinase 2 (CaMKK2) as a potential target for such therapeutics because its inhibition enhances OB differentiation and bone growth and suppresses OC differentiation.  Mice null for CaMKK2 possess higher trabecular bone mass in their long bones, along with significantly more OBs and fewer multinuclear OCs.  In vitro, although Camkk2-/- mesenchymal stem cells (MSCs) yield significantly higher nos. of OBs, bone marrow cells from Camkk2-/- mice produce fewer multinuclear OCs.  Acute inhibition of CaMKK2 by its selective, cell-permeable pharmacol. inhibitor STO-609 also results in increased OB and diminished OC formation.  Further, we find phospho-protein kinase A (PKA) and Ser133 phosphorylated form of cyclic adenosine monophosphate (cAMP) response element binding protein (pCREB) to be markedly elevated in OB progenitors deficient in CaMKK2.  On the other hand, genetic ablation of CaMKK2 or its pharmacol. inhibition in OC progenitors results in reduced pCREB as well as significantly reduced levels of its transcriptional target, nuclear factor of activated T cells, cytoplasmic (NFATc1).  Moreover, in vivo administration of STO-609 results in increased OBs and diminished OCs, conferring significant protection from ovariectomy (OVX)-induced osteoporosis in adult mice.  Thus, a novel function was shown for CaMKK2 in bone remodeling and highlight the potential for its therapeutic inhibition as a valuable bone anabolic strategy that also inhibits OC differentiation in the treatment of osteoporosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH8_fa3hgfgbVg90H21EOLACvtfcHk0lgP73NDmOOnzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVOmsb4%253D&md5=56d88615d2500c579e6b5741a9b3c0d4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjbmr.1890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjbmr.1890%26sid%3Dliteratum%253Aachs%26aulast%3DCary%26aufirst%3DR.%2BL.%26aulast%3DWaddell%26aufirst%3DS.%26aulast%3DRacioppi%26aufirst%3DL.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNovack%26aufirst%3DD.%2BV.%26aulast%3DVoor%26aufirst%3DM.%2BJ.%26aulast%3DSankar%26aufirst%3DU.%26atitle%3DInhibition%2520of%2520Ca%25282%2529%2528%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520stimulates%2520osteoblast%2520formation%2520and%2520inhibits%2520osteoclast%2520differentiation%26jtitle%3DJ.%2520Bone%2520Miner.%2520Res.%26date%3D2013%26volume%3D28%26spage%3D1599%26epage%3D1610%26doi%3D10.1002%2Fjbmr.1890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novack, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of CaMKK2 reverses age-associated decline in bone mass</span>. <i>Bone</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1016/j.bone.2015.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bone.2015.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25724145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslCntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=120-127&author=Z.+J.+Pritchardauthor=R.+L.+Caryauthor=C.+Yangauthor=D.+V.+Novackauthor=M.+J.+Voorauthor=U.+Sankar&title=Inhibition+of+CaMKK2+reverses+age-associated+decline+in+bone+mass&doi=10.1016%2Fj.bone.2015.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of CaMKK2 reverses age-associated decline in bone mass</span></div><div class="casAuthors">Pritchard, Zachary J.; Cary, Rachel L.; Yang, Chang; Novack, Deborah V.; Voor, Michael J.; Sankar, Uma</div><div class="citationInfo"><span class="NLM_cas:title">Bone (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">BONEDL</span>;
        ISSN:<span class="NLM_cas:issn">1873-2763</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Decline in bone formation is a major contributing factor to the loss of bone mass assocd. with aging.  We previously showed that the genetic ablation of the tissue-restricted and multifunctional Ca2 +/calmodulin (CaM)-dependent protein kinase kinase 2 (CaMKK2) stimulates trabecular bone mass accrual, mainly by promoting anabolic pathways and inhibiting catabolic pathways of bone remodeling.  In this study, we investigated whether inhibition of this kinase using its selective cell-permeable inhibitor STO-609 will stimulate bone formation in 32 wk old male WT mice and reverse age-assocd. of decline in bone vol. and strength.  Tri-weekly i.p. injections of saline or STO-609 (10 μM) were performed for six weeks followed by metabolic labeling with calcein and alizarin red.  New bone formation was assessed by dynamic histomorphometry whereas micro-computed tomog. was employed to measure trabecular bone vol., microarchitecture and femoral mid-shaft geometry.  Cortical and trabecular bone biomech. properties were assessed using three-point bending and punch compression methods resp.  Our results reveal that as they progress from 12 to 32 wk of age, WT mice sustain a significant decline in trabecular bone vol., microarchitecture and strength as well as cortical bone strength.  However, treatment of the 32 wk old WT mice with STO-609 stimulated apposition of new bone and completely reversed the age-assocd. decrease in bone vol., quality, as well as trabecular and cortical bone strength.  We also obsd. that regardless of age, male Camkk2-/- mice possessed significantly elevated trabecular bone vol., microarchitecture and compressive strength as well as cortical bone strength compared to age-matched WT mice, implying that the chronic loss of this kinase attenuates age-assocd. decline in bone mass.  Further, whereas STO-609 treatment and/or the absence of CaMKK2 significantly enhanced the femoral mid-shaft geometry, the mid-shaft cortical wall thickness and material bending stress remained similar among the cohorts, implying that regardless of treatment, the material properties of the bone remain similar.  Thus, our cumulative results provide evidence for the pharmacol. inhibition of CaMKK2 as a bone anabolic strategy in combating age-assocd. osteoporosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHR9HKD1SNbVg90H21EOLACvtfcHk0lh6cSc_00tI4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslCntb8%253D&md5=0a19057ffeda873dd4b33c566006d856</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bone.2015.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bone.2015.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DPritchard%26aufirst%3DZ.%2BJ.%26aulast%3DCary%26aufirst%3DR.%2BL.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DNovack%26aufirst%3DD.%2BV.%26aulast%3DVoor%26aufirst%3DM.%2BJ.%26aulast%3DSankar%26aufirst%3DU.%26atitle%3DInhibition%2520of%2520CaMKK2%2520reverses%2520age-associated%2520decline%2520in%2520bone%2520mass%26jtitle%3DBone%26date%3D2015%26volume%3D75%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.bone.2015.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribar, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">The Ca2+/calmodulin-dependent protein kinase kinase, CaMKK2, inhibits preadipocyte differentiation</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">3668</span>– <span class="NLM_lpage">3679</span>, <span class="refDoi"> DOI: 10.1210/en.2011-1107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1210%2Fen.2011-1107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21862616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVanurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=3668-3679&author=F.+Linauthor=T.+J.+Ribarauthor=A.+R.+Means&title=The+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase%2C+CaMKK2%2C+inhibits+preadipocyte+differentiation&doi=10.1210%2Fen.2011-1107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Ca2+/calmodulin-dependent protein kinase kinase, CaMKK2, inhibits preadipocyte differentiation</span></div><div class="casAuthors">Lin, Fumin; Ribar, Thomas J.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3668-3679</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">When fed a std. chow diet, CaMKK2 null mice have increased adiposity and larger adipocytes than do wild-type mice, whereas energy balance is unchanged.  Here, we show that Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) is expressed in preadipocytes, where it functions as an AMP-activated protein kinase (AMPK)α kinase.  Acute inhibition or deletion of CaMKK2 in preadipocytes enhances their differentiation into mature adipocytes, which can be reversed by 5-aminoimidazole-4-carboxamide ribonucleotide-mediated activation of AMPK.  During adipogenesis, CaMKK2 expression is markedly decreased and temporally accompanied by increases in mRNA encoding the early adipogenic genes CCAAT/enhancer binding protein (C/EBP) β and C/EBP δ.  Preadipocyte factor 1 has been reported to inhibit adipogenesis by up-regulating sex detg. region Y-box 9 (Sox9) expression in preadipocytes and Sox9 suppresses C/EBPβ and C/EBPδ transcription.  We show that inhibition of the CaMKK2/AMPK signaling cascade in preadipocytes reduces preadipocyte factor 1 and Sox9 mRNA resulting in accelerated adipogenesis.  We conclude that CaMKK2 and AMPK function in a signaling pathway that participates in the regulation of adiposity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPYor3uM5dU7Vg90H21EOLACvtfcHk0lh6cSc_00tI4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVanurvE&md5=30187fd728913b1f6765b8b228b9dd48</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fen.2011-1107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2011-1107%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DF.%26aulast%3DRibar%26aufirst%3DT.%2BJ.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DThe%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%252C%2520CaMKK2%252C%2520inhibits%2520preadipocyte%2520differentiation%26jtitle%3DEndocrinology%26date%3D2011%26volume%3D152%26spage%3D3668%26epage%3D3679%26doi%3D10.1210%2Fen.2011-1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müschen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. C.</span></span> <span> </span><span class="NLM_article-title">CAMKs support development of acute myeloid leukemia</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/s13045-018-0574-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1186%2Fs13045-018-0574-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29482582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=30&author=X.+Kangauthor=C.+Cuiauthor=C.+Wangauthor=G.+Wuauthor=H.+Chenauthor=Z.+Luauthor=X.+Chenauthor=L.+Wangauthor=J.+Huangauthor=H.+Gengauthor=M.+Zhaoauthor=Z.+Chenauthor=M.+M%C3%BCschenauthor=H.-Y.+Wangauthor=C.+C.+Zhang&title=CAMKs+support+development+of+acute+myeloid+leukemia&doi=10.1186%2Fs13045-018-0574-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">CAMKs support development of acute myeloid leukemia</span></div><div class="casAuthors">Kang, Xunlei; Cui, Changhao; Wang, Chen; Wu, Guojin; Chen, Heyu; Lu, Zhigang; Chen, Xiaoli; Wang, Li; Huang, Jie; Geng, Huimin; Zhao, Meng; Chen, Zhengshan; Muschen, Markus; Wang, Huan-You; Zhang, Cheng Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30/1-30/12</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: We recently identified the human leukocyte Ig-like receptor B2 (LILRB2) and its mouse ortholog-paired Ig-like receptor (PirB) as receptors for several angiopoietin-like proteins (Angptls).  We also demonstrated that PirB is important for the development of acute myeloid leukemia (AML), but exactly how an inhibitory receptor such as PirB can support cancer development is intriguing.  Results: Here, we showed that the activation of Ca (2+)/calmodulin-dependent protein kinases (CAMKs) is coupled with PirB signaling in AML cells.  High expression of CAMKs is assocd. with a poor overall survival probability in patients with AML.  Knockdown of CAMKI or CAMKIV decreased human acute leukemia development in vitro and in vivo.  Mouse AML cells that are defective in PirB signaling had decreased activation of CAMKs, and the forced expression of CAMK partially rescued the PirB-defective phenotype in the MLL-AF9 AML mouse model.  The inhibition of CAMK kinase activity or deletion of CAMKIV significantly slowed AML development and decreased the AML stem cell activity.  We also found that CAMKIV acts through the phosphorylation of one of its well-known target (CREB) in AML cells.  Conclusion: CAMKs are essential for the growth of human and mouse AML.  The inhibition of CAMK signaling may become an effective strategy for treating leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcfARnv35UB7Vg90H21EOLACvtfcHk0lh6cSc_00tI4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyqt7w%253D&md5=c9c18a264c577c442e8bae29e44dafbf</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs13045-018-0574-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-018-0574-8%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DM%25C3%25BCschen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.-Y.%26aulast%3DZhang%26aufirst%3DC.%2BC.%26atitle%3DCAMKs%2520support%2520development%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2018%26volume%3D11%26spage%3D30%26doi%3D10.1186%2Fs13045-018-0574-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span> <span> </span><span class="NLM_article-title">Growth inhibition of human gastric adenocarcinoma cells in vitro by STO-609 is independent of calcium/calmodulin-dependent protein kinase kinase-beta and adenosine monophosphate-activated protein kinase</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1164</span>– <span class="NLM_lpage">1171</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27158402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFWlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1164-1171&author=Z.+Maauthor=D.+Wenauthor=X.+Wangauthor=L.+Yangauthor=T.+Liuauthor=J.+Liuauthor=J.+Zhuauthor=X.+Fang&title=Growth+inhibition+of+human+gastric+adenocarcinoma+cells+in+vitro+by+STO-609+is+independent+of+calcium%2Fcalmodulin-dependent+protein+kinase+kinase-beta+and+adenosine+monophosphate-activated+protein+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Growth inhibition of human gastric adenocarcinoma cells in vitro by STO-609 is independent of calcium/ calmodulin-dependent protein kinase kinase-beta and adenosine monophosphate-activated protein kinase</span></div><div class="casAuthors">Ma, Zhiming; Wen, Dacheng; Wang, Xudong; Yang, Longfei; Liu, Tianzhou; Liu, Jingjing; Zhu, Jiaming; Fang, Xuedong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1164-1171</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Adenosine monophosphate (AMP)-activated protein kinase is a recently identified downstream target of calcium/calmodulin-dependent protein kinase kinase-beta, and is involved in the regulation of cell metab. and cell proliferation.  STO-609 is a selective antagonist of calcium/calmodulin-dependent protein kinase kinase-beta.  In the present study, we found that STO-609 suppressed AMP-activated protein kinase activity, reduced expression of Akt and ERK, and increased cell apoptosis in SNU-1 and N87 cells but not normal gastric epithelial cells (CCL-241).  Interestingly, we found such effects of STO-609 on gastric cancer cells were not affected after the knock-down of CaMKK- β and AMPK.  In conclusion, STO-609 is an effective cytotoxic agent for gastric adenocarcinoma in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIgiOWvdZlULVg90H21EOLACvtfcHk0lgl4GeKcghxDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFWlt7w%253D&md5=4118e22a54738fcd8b16234543fefd3a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DX.%26atitle%3DGrowth%2520inhibition%2520of%2520human%2520gastric%2520adenocarcinoma%2520cells%2520in%2520vitro%2520by%2520STO-609%2520is%2520independent%2520of%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase-beta%2520and%2520adenosine%2520monophosphate-activated%2520protein%2520kinase%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2016%26volume%3D8%26spage%3D1164%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Byrne, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotte, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eduful, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">In depth analysis of kinase cross screening data to identify CAMKK2 inhibitory scaffolds</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">325</span>, <span class="refDoi"> DOI: 10.3390/molecules25020325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3390%2Fmolecules25020325" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslCjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=325&author=S.+N.+O%E2%80%99Byrneauthor=J.+W.+Scottauthor=J.+R.+Pilotteauthor=A.+D.+S.+Santiagoauthor=C.+G.+Langendorfauthor=J.+S.+Oakhillauthor=B.+J.+Edufulauthor=R.+M.+Cou%C3%B1agoauthor=C.+I.+Wellsauthor=W.+J.+Zuercherauthor=T.+M.+Willsonauthor=D.+H.+Drewry&title=In+depth+analysis+of+kinase+cross+screening+data+to+identify+CAMKK2+inhibitory+scaffolds&doi=10.3390%2Fmolecules25020325"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">In depth analysis of kinase cross screening data to identify CAMKK2 inhibitory scaffolds</span></div><div class="casAuthors">O'Byrne, Sean N.; Scott, John W.; Pilotte, Joseph R.; Santiago, Andre Da S.; Langendorf, Christopher G.; Oakhill, Jonathan S.; Eduful, Benjamin J.; Counago, Rafael M.; Wells, Carrow I.; Zuercher, William J.; Willson, Timothy M.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">325</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) activates CAMK1, CAMK4, AMPK, and AKT, leading to numerous physiol. responses.  The deregulation of CAMKK2 is linked to several diseases, suggesting the utility of CAMKK2 inhibitors for oncol., metabolic and inflammatory indications.  In this work, we demonstrate that STO-609, frequently described as a selective inhibitor for CAMKK2, potently inhibits a significant no. of other kinases.  Through an anal. of literature and public databases, we have identified other potent CAMKK2 inhibitors and verified their activities in differential scanning fluorimetry and enzyme inhibition assays.  These inhibitors are potential starting points for the development of selective CAMKK2 inhibitors and will lead to tools that delineate the roles of this kinase in disease biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzFTNmh85XgLVg90H21EOLACvtfcHk0lgl4GeKcghxDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslCjsbo%253D&md5=fed1cf447694b4bd7fdb39744c27442b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25020325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25020325%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Byrne%26aufirst%3DS.%2BN.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26aulast%3DPilotte%26aufirst%3DJ.%2BR.%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DEduful%26aufirst%3DB.%2BJ.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DIn%2520depth%2520analysis%2520of%2520kinase%2520cross%2520screening%2520data%2520to%2520identify%2520CAMKK2%2520inhibitory%2520scaffolds%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D325%26doi%3D10.3390%2Fmolecules25020325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amato, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">CDKL2 promotes epithelial-mesenchymal transition and breast cancer progression</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10840</span>– <span class="NLM_lpage">10853</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.18632%2Foncotarget.2535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25333262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC2M3itVKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10840-10853&author=L.+Liauthor=C.+Liuauthor=R.+J.+Amatoauthor=J.+T.+Changauthor=G.+Duauthor=W.+Li&title=CDKL2+promotes+epithelial-mesenchymal+transition+and+breast+cancer+progression&doi=10.18632%2Foncotarget.2535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CDKL2 promotes epithelial-mesenchymal transition and breast cancer progression</span></div><div class="casAuthors">Li Linna; Liu Chunping; Amato Robert J; Chang Jeffrey T; Du Guangwei; Li Wenliang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10840-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epithelial-mesenchymal transition (EMT) confers mesenchymal properties on epithelial cells and has been closely associated with the acquisition of aggressive traits by epithelial cancer cells.  To identify novel regulators of EMT, we carried out cDNA screens that covered 500 human kinases.  Subsequent characterization of candidate kinases led us to uncover cyclin-dependent kinase-like 2 (CDKL2) as a novel potent promoter for EMT and breast cancer progression.  CDKL2-expressing human mammary gland epithelial cells displayed enhanced mesenchymal traits and stem cell-like phenotypes, which was acquired through activating a ZEB1/E-cadherin/β-catenin positive feedback loop and regulating CD44 mRNA alternative splicing to promote conversion of CD24(high) cells to CD44(high) cells.  Furthermore, CDKL2 enhanced primary tumor formation and metastasis in a breast cancer xenograft model.  Notably, CDKL2 is expressed significantly higher in mesenchymal human breast cancer cell lines than in epithelial lines, and its over-expression/amplification in human breast cancers is associated with shorter disease-free survival.  Taken together, our study uncovered a major role for CDKL2 in promoting EMT and breast cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQC9qYy_17RnIJKJbLTgsvCfW6udTcc2eadwzE8HpZtK7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3itVKhsw%253D%253D&md5=49752f32dbc5808a474bdb42f1db87c8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2535%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DAmato%26aufirst%3DR.%2BJ.%26aulast%3DChang%26aufirst%3DJ.%2BT.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DCDKL2%2520promotes%2520epithelial-mesenchymal%2520transition%2520and%2520breast%2520cancer%2520progression%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10840%26epage%3D10853%26doi%3D10.18632%2Foncotarget.2535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billard, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzhugh, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozowski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timoshchenko, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lininger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klauber-Demore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahagian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serody, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrant, T. K.</span></span> <span> </span><span class="NLM_article-title">G protein coupled receptor kinase 3 regulates breast cancer migration, invasion, and metastasis</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0152856</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0152856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1371%2Fjournal.pone.0152856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27049755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaks7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=M.+J.+Billardauthor=D.+J.+Fitzhughauthor=J.+S.+Parkerauthor=J.+M.+Brozowskiauthor=M.+W.+McGinnisauthor=R.+G.+Timoshchenkoauthor=D.+S.+Serafinauthor=R.+Liningerauthor=N.+Klauber-Demoreauthor=G.+Sahagianauthor=Y.+K.+Truongauthor=M.+F.+Sassanoauthor=J.+S.+Serodyauthor=T.+K.+Tarrant&title=G+protein+coupled+receptor+kinase+3+regulates+breast+cancer+migration%2C+invasion%2C+and+metastasis&doi=10.1371%2Fjournal.pone.0152856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">G protein coupled receptor kinase 3 regulates breast cancer migration, invasion, and metastasis</span></div><div class="casAuthors">Billard, Matthew J.; Fitzhugh, David J.; Parker, Joel S.; Brozowski, Jaime M.; McGinnis, Marcus W.; Timoshchenko, Roman G.; Stephen, Serafin D.; Lininger, Ruth; Klauber-Demore, Nancy; Sahagian, Gary; Truong, Young K.; Sassano, Maria F.; Serody, Jonathan S.; Tarrant, Teresa K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0152856/1-e0152856/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC) is a heterogeneous disease that has a poor prognosis and limited treatment options.  Chemokine receptor interactions are important modulators of breast cancer metastasis; however, it is now recognized that quant. surface expression of one important chemokine receptor, CXCR4, may not directly correlate with metastasis and that its functional activity in breast cancer may better inform tumor pathogenicity.  G protein coupled receptor kinase 3 (GRK3) is a neg. regulator of CXCR4 activity, and we show that GRK expression correlates with tumorigenicity, mol. subtype, and metastatic potential in human tumor microarray anal.  Using established human breast cancer cell lines and an immunocompetent in vivo mouse model, we further demonstrate that alterations in GRK3 expression levels in tumor cells directly affect migration and invasion in vitro and the establishment of distant metastasis in vivo.  The effects of GRK3 modulation appear to be specific to chemokine-mediated migration behaviors without influencing tumor cell proliferation or survival.  These data demonstrate that GRK3 dysregulation may play an important part in TNBC metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7p1mNnYr53rVg90H21EOLACvtfcHk0lhkEqPXX7vNjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaks7zN&md5=e97281c582eef4661bf7cebd5dc9dcb5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0152856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0152856%26sid%3Dliteratum%253Aachs%26aulast%3DBillard%26aufirst%3DM.%2BJ.%26aulast%3DFitzhugh%26aufirst%3DD.%2BJ.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DBrozowski%26aufirst%3DJ.%2BM.%26aulast%3DMcGinnis%26aufirst%3DM.%2BW.%26aulast%3DTimoshchenko%26aufirst%3DR.%2BG.%26aulast%3DSerafin%26aufirst%3DD.%2BS.%26aulast%3DLininger%26aufirst%3DR.%26aulast%3DKlauber-Demore%26aufirst%3DN.%26aulast%3DSahagian%26aufirst%3DG.%26aulast%3DTruong%26aufirst%3DY.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DSerody%26aufirst%3DJ.%2BS.%26aulast%3DTarrant%26aufirst%3DT.%2BK.%26atitle%3DG%2520protein%2520coupled%2520receptor%2520kinase%25203%2520regulates%2520breast%2520cancer%2520migration%252C%2520invasion%252C%2520and%2520metastasis%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0152856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landesman-Bollag, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romieu-Mourez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonenshein, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seldin, D. C.</span></span> <span> </span><span class="NLM_article-title">Protein kinase CK2 in mammary gland tumorigenesis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3247</span>– <span class="NLM_lpage">3257</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fsj.onc.1204411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11423974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1GktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3247-3257&author=E.+Landesman-Bollagauthor=R.+Romieu-Mourezauthor=D.+H.+Songauthor=G.+E.+Sonensheinauthor=R.+D.+Cardiffauthor=D.+C.+Seldin&title=Protein+kinase+CK2+in+mammary+gland+tumorigenesis&doi=10.1038%2Fsj.onc.1204411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase CK2 in mammary gland tumorigenesis</span></div><div class="casAuthors">Landesman-Bollag, Esther; Romieu-Mourez, Raphaelle; Song, Diane H.; Sonenshein, Gail E.; Cardiff, Robert D.; Seldin, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3247-3257</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein kinase CK2 is a ubiquitous and evolutionarily conserved serine/threonine kinase that is upregulated in many human cancers and can serve as an oncogene in lymphocytes.  Recently, the authors have demonstrated that CK2 potentiates Wnt/β-catenin signaling in mammary epithelial cells.  To det. whether CK2 overexpression contributes to mammary tumorigenesis, the authors have performed comparative studies of human and rat breast cancer specimens and the authors have engineered transgenic mice with dysregulated expression of CK2α in the mammary gland.  The authors find that CK2 is highly expressed in human breast tumor specimens and in carcinogen-induced rat mammary tumors.  Overexpression of CK2α in the mammary gland of transgenic mice, under control of the MMTV-LTR, causes hyperplasia and dysplasia of the female mammary gland.  Thirty per cent of the female MMTV-CK2α transgenic mice develop mammary adenocarcinomas at a median of 23 mo of age, often assocd. with Wnt pathway activation, as evidenced by upregulation of β-catenin protein.  NF-κB activation and upregulation of c-Myc also occur frequently.  Thus, in mice, rats, and humans, dysregulated expression of CK2 is assocd. with and is capable of contributing to mammary tumorigenesis.  Targeted inhibition of CK2 could be useful in the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDbe-6fAKYLVg90H21EOLACvtfcHk0lgyNSjep0UUww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1GktLk%253D&md5=a68a69b007fd48038580645bd847f4de</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204411%26sid%3Dliteratum%253Aachs%26aulast%3DLandesman-Bollag%26aufirst%3DE.%26aulast%3DRomieu-Mourez%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DD.%2BH.%26aulast%3DSonenshein%26aufirst%3DG.%2BE.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DSeldin%26aufirst%3DD.%2BC.%26atitle%3DProtein%2520kinase%2520CK2%2520in%2520mammary%2520gland%2520tumorigenesis%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D3247%26epage%3D3257%26doi%3D10.1038%2Fsj.onc.1204411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langouet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fardel, O.</span></span> <span> </span><span class="NLM_article-title">Activation of the aryl hydrocarbon receptor by the calcium/calmodulin-dependent protein kinase kinase inhibitor 7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid (STO-609)</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2556</span>– <span class="NLM_lpage">2563</span>, <span class="refDoi"> DOI: 10.1124/dmd.108.023333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1124%2Fdmd.108.023333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18755850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=2556-2563&author=P.+Monteiroauthor=D.+Gilotauthor=S.+Langouetauthor=O.+Fardel&title=Activation+of+the+aryl+hydrocarbon+receptor+by+the+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+inhibitor+7-oxo-7H-benzimidazo%5B2%2C1-a%5Dbenz%5Bde%5Disoquinoline-3-carboxylic+acid+%28STO-609%29&doi=10.1124%2Fdmd.108.023333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the aryl hydrocarbon receptor by the calcium/calmodulin-dependent protein kinase kinase inhibitor 7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid (STO-609)</span></div><div class="casAuthors">Monteiro, Patricia; Gilot, David; Langouet, Sophie; Fardel, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2556-2563</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">This study was designed to analyze the effects of the Ca2+/calmodulin-dependent protein kinase kinase (CaMKK) inhibitor STO-609 (7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid) toward the aryl hydrocarbon receptor (AhR) pathway because Ca2+/calmodulin-dependent protein kinase (CaMK) Iα, known as a downstream CaMKK effector, has been recently shown to contribute to the AhR cascade.  STO-609 failed to alter up-regulation of the AhR target CYP1A1 in response to the potent AhR ligand 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) in MCF-7 cells.  STO-609, used at a 25 μM concn. known to fully inhibit CaMKK activity, was surprisingly found to markedly induce CYP1A1 expression and activity by itself in MCF-7 cells; it similarly up-regulated various other AhR target genes in human macrophages.  STO-609-related CYP1A1 induction was prevented by chem. inhibition or small interfering RNA-mediated knockdown expression of AhR.  Moreover, STO-609 was demonstrated to phys. interact with the ligand-binding domain of AhR, as assessed by TCDD binding competition assay, and to induce AhR translocation to the nucleus.  As already reported for AhR agonists, STO-609 triggered the increase of [Ca2+]i and activation of CaMKIα, whose inhibition through the use of the Ca2+ chelator BAPTA-acetoxymethyl ester or the CaMK inhibitor KN-93 (2-[[N-(2-hydroxyethyl)]-N-(4-methoxybenzenesulfonyl)]amino-N-(4-chlorocinnamyl)-N-methylbenzylamine), resp., prevented STO-609-mediated CYP1A1 activity induction.  Taken together, these results demonstrate that the CaMKK inhibitor STO-609 can act as an AhR ligand and, in this way, fully activates the Ca2+/CaMKIα/AhR cascade.  Such data, therefore, make unlikely any contribution of CaMKK activity to the AhR pathway and, moreover, suggest that caution may be required when using STO-609 as a specific inhibitor of CaMKKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4LM4lXUX03LVg90H21EOLACvtfcHk0lgyNSjep0UUww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCltrzK&md5=037d08b34bc2e8aca51365f8354d1594</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1124%2Fdmd.108.023333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.108.023333%26sid%3Dliteratum%253Aachs%26aulast%3DMonteiro%26aufirst%3DP.%26aulast%3DGilot%26aufirst%3DD.%26aulast%3DLangouet%26aufirst%3DS.%26aulast%3DFardel%26aufirst%3DO.%26atitle%3DActivation%2520of%2520the%2520aryl%2520hydrocarbon%2520receptor%2520by%2520the%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520inhibitor%25207-oxo-7H-benzimidazo%255B2%252C1-a%255Dbenz%255Bde%255Disoquinoline-3-carboxylic%2520acid%2520%2528STO-609%2529%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2008%26volume%3D36%26spage%3D2556%26epage%3D2563%26doi%3D10.1124%2Fdmd.108.023333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asquith, C. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koutentis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalogirou, S. A.</span></span> <span> </span><span class="NLM_article-title">1,2,6-Thiadiazinones as novel narrow spectrum calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2) inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1221</span>, <span class="refDoi"> DOI: 10.3390/molecules23051221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3390%2Fmolecules23051221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyksrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1221&author=C.+R.+M.+Asquithauthor=H.+P.+Godoiauthor=M.+R.+Cou%C3%B1agoauthor=T.+Laitinenauthor=W.+J.+Scottauthor=G.+C.+Langendorfauthor=S.+J.+Oakhillauthor=H.+D.+Drewryauthor=J.+W.+Zuercherauthor=A.+P.+Koutentisauthor=M.+T.+Willsonauthor=S.+A.+Kalogirou&title=1%2C2%2C6-Thiadiazinones+as+novel+narrow+spectrum+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+inhibitors&doi=10.3390%2Fmolecules23051221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">1,2,6-thiadiazinones as novel narrow spectrum calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2) inhibitors</span></div><div class="casAuthors">Asquith, Christopher R. M.; Godoi, Paulo H.; Counago, Rafael M.; Laitinen, Tuomo; Scott, John W.; Langendorf, Christopher G.; Oakhill, Jonathan S.; Drewry, David H.; Zuercher, William J.; Koutentis, Panayiotis A.; Willson, Timothy M.; Kalogirou, Andreas S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1221/1-1221/23</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We demonstrate for the first time that 4H-1,2,6-thiadiazin-4-one (TDZ) can function as a chemotype for the design of ATP-competitive kinase inhibitors.  Using insights from a co-crystal structure of a 3,5-bis(arylamino)-4H-1,2,6-thiadiazin-4-one bound to calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2), several analogs were identified with micromolar activity through targeted displacement of bound water mols. in the active site.  Since the TDZ analogs showed reduced promiscuity compared to their 2,4-dianilinopyrimidine counter parts, they represent starting points for development of highly selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobtAr-yNWZabVg90H21EOLACvtfcHk0ljig3D45oOkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyksrvI&md5=ad009ea9adb9dfd9a084606d26c4f652</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23051221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23051221%26sid%3Dliteratum%253Aachs%26aulast%3DAsquith%26aufirst%3DC.%2BR.%2BM.%26aulast%3DGodoi%26aufirst%3DH.%2BP.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DM.%2BR.%26aulast%3DLaitinen%26aufirst%3DT.%26aulast%3DScott%26aufirst%3DW.%2BJ.%26aulast%3DLangendorf%26aufirst%3DG.%2BC.%26aulast%3DOakhill%26aufirst%3DS.%2BJ.%26aulast%3DDrewry%26aufirst%3DH.%2BD.%26aulast%3DZuercher%26aufirst%3DJ.%2BW.%26aulast%3DKoutentis%26aufirst%3DA.%2BP.%26aulast%3DWillson%26aufirst%3DM.%2BT.%26aulast%3DKalogirou%26aufirst%3DS.%2BA.%26atitle%3D1%252C2%252C6-Thiadiazinones%2520as%2520novel%2520narrow%2520spectrum%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520inhibitors%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1221%26doi%3D10.3390%2Fmolecules23051221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghose, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbertz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallamo, J. P.</span></span> <span> </span><span class="NLM_article-title">Knowledge based prediction of ligand binding modes and rational inhibitor design for kinase drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5149</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/jm800475y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800475y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5149-5171&author=A.+K.+Ghoseauthor=T.+Herbertzauthor=D.+A.+Pippinauthor=J.+M.+Salvinoauthor=J.+P.+Mallamo&title=Knowledge+based+prediction+of+ligand+binding+modes+and+rational+inhibitor+design+for+kinase+drug+discovery&doi=10.1021%2Fjm800475y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge Based Prediction of Ligand Binding Modes and Rational Inhibitor Design for Kinase Drug Discovery</span></div><div class="casAuthors">Ghose, Arup K.; Herbertz, Torsten; Pippin, Douglas A.; Salvino, Joseph M.; Mallamo, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5149-5171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review describing the use of Protein Data Bank (PDB) entries specifically for kinase inhibitor drug discovery.  Topics discussed include: the current state of structure-aided ligand design, kinase structure queries and extn. from the PDB, alignment of protein structures, structural classification of ligands, anal. of ligands and binding modes, and ligand design strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDINDYVE15cbVg90H21EOLACvtfcHk0ljuW1R9rAwOKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWmurnM&md5=a1afc4d6c7f6fa4dd6aa9da8d78f1ee3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm800475y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800475y%26sid%3Dliteratum%253Aachs%26aulast%3DGhose%26aufirst%3DA.%2BK.%26aulast%3DHerbertz%26aufirst%3DT.%26aulast%3DPippin%26aufirst%3DD.%2BA.%26aulast%3DSalvino%26aufirst%3DJ.%2BM.%26aulast%3DMallamo%26aufirst%3DJ.%2BP.%26atitle%3DKnowledge%2520based%2520prediction%2520of%2520ligand%2520binding%2520modes%2520and%2520rational%2520inhibitor%2520design%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5149%26epage%3D5171%26doi%3D10.1021%2Fjm800475y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">Toward a pharmacophore for kinase frequent hitters</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5616</span>– <span class="NLM_lpage">5619</span>, <span class="refDoi"> DOI: 10.1021/jm049793g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049793g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5616-5619&author=A.+M.+Aronovauthor=M.+A.+Murcko&title=Toward+a+pharmacophore+for+kinase+frequent+hitters&doi=10.1021%2Fjm049793g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a Pharmacophore for Kinase Frequent Hitters</span></div><div class="casAuthors">Aronov, Alex M.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5616-5619</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. protein kinase inhibitors are widely employed as biol. reagents and as leads in the design of drugs for a variety of diseases.  One of the hardest challenges in kinase inhibitor design is achieving target selectivity.  By utilizing x-ray structural information for four promiscuous inhibitors, we propose a five-point pharmacophore for kinase frequent hitters, demonstrate its ability to discriminate between frequent hitters and selective ligands, and suggest a strategy for selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj2-pbD58y0bVg90H21EOLACvtfcHk0ljuW1R9rAwOKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlSisrY%253D&md5=4ebc59812016803b8cbd64849d104064</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm049793g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049793g%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DToward%2520a%2520pharmacophore%2520for%2520kinase%2520frequent%2520hitters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5616%26epage%3D5619%26doi%3D10.1021%2Fjm049793g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klug-Mcleod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span> <span> </span><span class="NLM_article-title">Kinase hinge binding scaffolds and their hydrogen bond patterns</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bmc.2015.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=26358279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6520-6527&author=L.+Xingauthor=J.+Klug-Mcleodauthor=B.+Raiauthor=E.+A.+Lunney&title=Kinase+hinge+binding+scaffolds+and+their+hydrogen+bond+patterns&doi=10.1016%2Fj.bmc.2015.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase hinge binding scaffolds and their hydrogen bond patterns</span></div><div class="casAuthors">Xing, Li; Klug-Mcleod, Jacquelyn; Rai, Brajesh; Lunney, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6520-6527</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinases constitute a major class of intracellular signaling mols., and describe some of the most prominent drug targets.  Kinase inhibitors commonly employ small chem. scaffolds that form hydrogen bonds with the kinase hinge residues connecting the N- and C-terminal lobes of the catalytic domain.  In general the satisfied hydrogen bonds are required for potent inhibition, therefore constituting a conserved feature in the majority of inhibitor-kinase interactions.  From systematically analyzing the kinase scaffolds extd. from Pfizer crystal structure database (CSDb) the authors recognize that large no. of kinase inhibitors of diverse chem. structures are derived from a relatively small no. of common scaffolds.  Depending on specific substitution patterns, scaffolds may demonstrate versatile binding capacities to interact with kinase hinge.  Afforded by thousands of ligand-protein binary complexes, the hinge hydrogen bond patterns were analyzed with a focus on their three-dimensional configurations.  Most of the compds. engage H6 NH for hinge recognition.  Dual hydrogen bonds are commonly obsd. with addnl. recruitment of H4 CO upstream and/or H6 CO downstream.  Triple hydrogen bonds accounts for small no. of binary complexes.  An unusual hydrogen bond with a non-canonical H5 conformation is obsd., requiring a peptide bond flip by a glycine residue at the H6 position.  Addnl. hydrogen bonds to kinase hinge do not necessarily correlate with an increase in potency; conversely they appear to compromise kinase selectivity.  Such learnings could enhance the prospect of successful therapy design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRC-xiTiNvELVg90H21EOLACvtfcHk0ljuW1R9rAwOKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L&md5=048b4b99628c6e1ae273b00a47ab2377</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DKlug-Mcleod%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DB.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26atitle%3DKinase%2520hinge%2520binding%2520scaffolds%2520and%2520their%2520hydrogen%2520bond%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6520%26epage%3D6527%26doi%3D10.1016%2Fj.bmc.2015.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span> <span> </span><span class="NLM_article-title">7-Azaindole: A versatile scaffold for developing kinase inhibitors</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1248%2Fcpb.c17-00380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29311509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=29-36&author=T.+Irieauthor=M.+Sawa&title=7-Azaindole%3A+A+versatile+scaffold+for+developing+kinase+inhibitors&doi=10.1248%2Fcpb.c17-00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">7-Azaindole: a versatile scaffold for developing kinase inhibitors</span></div><div class="casAuthors">Irie, Takayuki; Sawa, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review.  This review summarized recent examples of 7-azaindole-based kinase inhibitors and discusses their binding interactions with the kinase hinge regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9QuvBOqQ8GrVg90H21EOLACvtfcHk0liiBPPsCYeMjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK&md5=69a5097cb54f2598d2e02414e74b3e6b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00380%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DT.%26aulast%3DSawa%26aufirst%3DM.%26atitle%3D7-Azaindole%253A%2520A%2520versatile%2520scaffold%2520for%2520developing%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D29%26epage%3D36%26doi%3D10.1248%2Fcpb.c17-00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherk, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnackenberg, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laping, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trizna, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">Development of a small-molecule serum- and glucocorticoid-regulated kinase-1 antagonist and its evaluation as a prostate cancer therapeutic</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">7475</span>– <span class="NLM_lpage">7483</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-08-1047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-08-1047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18794135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWltb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=7475-7483&author=A.+B.+Sherkauthor=D.+E.+Frigoauthor=C.+G.+Schnackenbergauthor=J.+D.+Brayauthor=N.+J.+Lapingauthor=W.+Triznaauthor=M.+Hammondauthor=J.+R.+Pattersonauthor=S.+K.+Thompsonauthor=D.+Kazminauthor=J.+D.+Norrisauthor=D.+P.+McDonnell&title=Development+of+a+small-molecule+serum-+and+glucocorticoid-regulated+kinase-1+antagonist+and+its+evaluation+as+a+prostate+cancer+therapeutic&doi=10.1158%2F0008-5472.can-08-1047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Small-Molecule Serum- and Glucocorticoid-Regulated Kinase-1 Antagonist and Its Evaluation as a Prostate Cancer Therapeutic</span></div><div class="casAuthors">Sherk, Andrea B.; Frigo, Daniel E.; Schnackenberg, Christine G.; Bray, Jeffrey D.; Laping, Nicholas J.; Trizna, Walter; Hammond, Marlys; Patterson, Jaclyn R.; Thompson, Scott K.; Kazmin, Dmitri; Norris, John D.; McDonnell, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7475-7483</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgens, through their actions on the androgen receptor (AR), are required for the development of the prostate and contribute to the pathol. growth dysregulation obsd. in prostate cancers.  Consequently, androgen ablation has become an essential component of the pharmacotherapy of prostate cancer.  In this study, we explored the utility of targeting processes downstream of AR as an alternate approach for therapy.  Specifically, we show that the serum and glucocorticoid-regulated kinase 1 (SGK1) gene is an androgen-regulated target gene in cellular models of prostate cancer.  Furthermore, functional serum- and glucocorticoid-regulated kinase 1 (SGK1) protein, as detd. by the phosphorylation of its target Nedd4-2, was also increased with androgen treatment.  Importantly, we detd. that RNA interference-mediated knockdown of SGK1 expression attenuates the androgen-mediated growth of the prostate cancer cell line LNCaP.  Given these findings, we explored the utility of SGK1 as a therapeutic target in prostate cancer by developing and evaluating a small-mol. inhibitor of this enzyme.  From these studies emerged GSK650394, a competitive inhibitor that quant. blocks the effect of androgens on LNCaP cell growth.  Thus, in addn. to androgen ablation, inhibition of pathways downstream of AR is likely to have therapeutic utility in prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTPdHEs-8bW7Vg90H21EOLACvtfcHk0liiBPPsCYeMjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFWltb7L&md5=bd7e0bb3e7a3e0bb8b514c3cf58f0a8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1047%26sid%3Dliteratum%253Aachs%26aulast%3DSherk%26aufirst%3DA.%2BB.%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26aulast%3DSchnackenberg%26aufirst%3DC.%2BG.%26aulast%3DBray%26aufirst%3DJ.%2BD.%26aulast%3DLaping%26aufirst%3DN.%2BJ.%26aulast%3DTrizna%26aufirst%3DW.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DPatterson%26aufirst%3DJ.%2BR.%26aulast%3DThompson%26aufirst%3DS.%2BK.%26aulast%3DKazmin%26aufirst%3DD.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DDevelopment%2520of%2520a%2520small-molecule%2520serum-%2520and%2520glucocorticoid-regulated%2520kinase-1%2520antagonist%2520and%2520its%2520evaluation%2520as%2520a%2520prostate%2520cancer%2520therapeutic%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D7475%26epage%3D7483%26doi%3D10.1158%2F0008-5472.can-08-1047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span> <span> </span><span class="NLM_article-title">Palladium(0)-catalyzed cross-coupling reaction of alkoxydiboron with haloarenes: a direct procedure for arylboronic esters</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7508</span>– <span class="NLM_lpage">7510</span>, <span class="refDoi"> DOI: 10.1021/jo00128a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00128a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK2MXovFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=7508-7510&author=T.+Ishiyamaauthor=M.+Murataauthor=N.+Miyaura&title=Palladium%280%29-catalyzed+cross-coupling+reaction+of+alkoxydiboron+with+haloarenes%3A+a+direct+procedure+for+arylboronic+esters&doi=10.1021%2Fjo00128a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium(0)-Catalyzed Cross-Coupling Reaction of Alkoxydiboron with Haloarenes: A Direct Procedure for Arylboronic Esters</span></div><div class="casAuthors">Ishiyama, Tatsuo; Murata, Miki; Miyaura, Norio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7508-10</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The palladium-catalyzed cross-coupling reaction of the pinacol ester of diboronic acid I with haloarenes gave a direct procedure for arylboronic esters II (Ar = aryl) from aryl halides in a range of 60-98%.  The reaction was catalyzed by PdCl2(dppf) (3 mol %) at 80° in the presence of KOAc (3 equiv) in DMSO and available with various functional groups such as nitrile, cyanide, ester, and carbonyl groups.  The trans-ArPd(II)(OAc)(PPh3)2 intermediate was isolated and characterized to propose the catalytic cycle involving the transmetalation between the phenylpalladium(II) acetate complex and I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobehJBGFjDH7Vg90H21EOLACvtfcHk0li2w6CHVen7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXovFSku7g%253D&md5=2666902270ec1efc5742480b133f1255</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjo00128a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00128a024%26sid%3Dliteratum%253Aachs%26aulast%3DIshiyama%26aufirst%3DT.%26aulast%3DMurata%26aufirst%3DM.%26aulast%3DMiyaura%26aufirst%3DN.%26atitle%3DPalladium%25280%2529-catalyzed%2520cross-coupling%2520reaction%2520of%2520alkoxydiboron%2520with%2520haloarenes%253A%2520a%2520direct%2520procedure%2520for%2520arylboronic%2520esters%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D7508%26epage%3D7510%26doi%3D10.1021%2Fjo00128a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merour, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and reactivity of 7-azaindoles (1H-pyrrolo(2,3-b)pyridine)</span>. <i>Curr. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.2174/1385272013375427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.2174%2F1385272013375427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFejurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=471-506&author=J.-Y.+Merourauthor=B.+Joseph&title=Synthesis+and+reactivity+of+7-azaindoles+%281H-pyrrolo%282%2C3-b%29pyridine%29&doi=10.2174%2F1385272013375427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and reactivity of 7-azaindoles (1H-pyrrolo[2,3-b]pyridines)</span></div><div class="casAuthors">Merour, Jean-Yves; Joseph, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Current Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-506</span>CODEN:
                <span class="NLM_cas:coden">CORCFE</span>;
        ISSN:<span class="NLM_cas:issn">1385-2728</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 133 refs. on research over the last 20 yr into the chem. of 7-azaindoles.  New synthetic methods for 7-azaindole structures were developed, including palladium-catalyzed cross-coupling reactions.  The reactivity of 3-formyl-, 3-halo-, and 2-lithio-7-azaindole derivs. is reported.  Functionalization of the pyridine ring via the N-oxide is also described.  In the last part, biol. activity of various 7-azaindole derivs. is briefly indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPCylwwn-8rLVg90H21EOLACvtfcHk0li2w6CHVen7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFejurY%253D&md5=27e3562dcfbbee1fc4f701e0ee8514e6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2174%2F1385272013375427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1385272013375427%26sid%3Dliteratum%253Aachs%26aulast%3DMerour%26aufirst%3DJ.-Y.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520reactivity%2520of%25207-azaindoles%2520%25281H-pyrrolo%25282%252C3-b%2529pyridine%2529%26jtitle%3DCurr.%2520Org.%2520Chem.%26date%3D2001%26volume%3D5%26spage%3D471%26epage%3D506%26doi%3D10.2174%2F1385272013375427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotani, S.</span></span> <span> </span><span class="NLM_article-title">Furopyridines. VI. Preparation and reactions of 2- and 3-substituted furo[2,3-b]pyridines</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570230542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fjhet.5570230542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaL2sXksFyru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1986&pages=1465-1469&author=H.+Moritaauthor=S.+Shiotani&title=Furopyridines.+VI.+Preparation+and+reactions+of+2-+and+3-substituted+furo%5B2%2C3-b%5Dpyridines&doi=10.1002%2Fjhet.5570230542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Furopyridines.  VI.  Preparation and reactions of 2- and 3-substituted furo[2,3-b]pyridines</span></div><div class="casAuthors">Morita, Hiroyuki; Shiotani, Shunsaku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1465-9</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    </div><div class="casAbstract">The title compds. I (R, R1 = H, Me, CH2OMe, CHO, cyano; R≠R1, except H) were prepd. starting from Et 2-chloronicotinate using known methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHJmUW8Nd4dbVg90H21EOLACvtfcHk0ljUARIFhxAD9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXksFyru70%253D&md5=c50a8a83a95e4d7f789dde494493510b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570230542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570230542%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DH.%26aulast%3DShiotani%26aufirst%3DS.%26atitle%3DFuropyridines.%2520VI.%2520Preparation%2520and%2520reactions%2520of%25202-%2520and%25203-substituted%2520furo%255B2%252C3-b%255Dpyridines%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1986%26volume%3D23%26spage%3D1465%26epage%3D1469%26doi%3D10.1002%2Fjhet.5570230542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. C.</span></span> <span> </span><span class="NLM_article-title">Trifluoromethyl triflate: synthesis and reactions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4147</span>– <span class="NLM_lpage">4156</span>, <span class="refDoi"> DOI: 10.1021/jo00228a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00228a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlKjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1987&pages=4147-4156&author=S.+L.+Taylorauthor=J.+C.+Martin&title=Trifluoromethyl+triflate%3A+synthesis+and+reactions&doi=10.1021%2Fjo00228a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Trifluoromethyl triflate: synthesis and reactions.</span></div><div class="casAuthors">Taylor, S. L.; Martin, J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4147-56</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">A new and convenient synthesis of F3CSO3CF3 (I) was devised.  The addn. of triflic anhydride to a catalytic amt. of SbF5 at 25° produced I in 94% yield, utilizing a new and stronger Lewis acid catalyst produced in the reaction, F4SbOSO2CF3.  This reaction makes pure I conveniently and economically available in large quantities.  A mechanism is presented that accounts for the substitution of a triflate ligand for a fluorine ligand on Sb, as well as for the formation of I.  I does not trifluoromethylate nucleophiles (pyridine, Et3N, iodide, PhLi, PhMgBr, lithium thiophenolate, or sodium naphthalenide) but gives products that result from an initial attack of the nucleophile at S.  Fluoride ion, which was formed by fragmentation of the trifluoromethoxide ion displaced from S, is a chain carrier in the rapid decompn. of I to give F3CSO2F and fluorophosgene.  This limits the synthetic utility of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqniKWCskQxZLVg90H21EOLACvtfcHk0ljUARIFhxAD9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmtlKjtrw%253D&md5=3241f9714404f33c2702770b0ed8ce12</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjo00228a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00228a001%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DS.%2BL.%26aulast%3DMartin%26aufirst%3DJ.%2BC.%26atitle%3DTrifluoromethyl%2520triflate%253A%2520synthesis%2520and%2520reactions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1987%26volume%3D52%26spage%3D4147%26epage%3D4156%26doi%3D10.1021%2Fjo00228a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lio6XOeo17iXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, P. H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massirer, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of inhibitor binding to CAMKK1 Identifies Features Necessary for Design of Specific Inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14800</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-33043-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-018-33043-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=30287839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB3czmtVajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=14800&author=A.+D.+S.+Santiagoauthor=R.+M.+Cou%C3%B1agoauthor=P.+Z.+Ramosauthor=P.+H.+C.+Godoiauthor=K.+B.+Massirerauthor=O.+Gileadiauthor=J.+M.+Elkins&title=Structural+analysis+of+inhibitor+binding+to+CAMKK1+Identifies+Features+Necessary+for+Design+of+Specific+Inhibitors&doi=10.1038%2Fs41598-018-33043-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of Inhibitor Binding to CAMKK1 Identifies Features Necessary for Design of Specific Inhibitors</span></div><div class="casAuthors">Santiago Andre da Silva; Counago Rafael M; Ramos Priscila Zonzini; Godoi Paulo H C; Massirer Katlin B; Elkins Jonathan M; Counago Rafael M; Massirer Katlin B; Gileadi Opher; Elkins Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14800</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The calcium/calmodulin-dependent protein kinases (CAMKKs) are upstream activators of CAMK1 and CAMK4 signalling and have important functions in neural development, maintenance and signalling, as well as in other aspects of biology such as Ca(2+) signalling in the cardiovascular system.  To support the development of specific inhibitors of CAMKKs we have determined the crystal structure of CAMKK1 with two ATP-competitive inhibitors.  The structures reveal small but exploitable differences between CAMKK1 and CAMKK2, despite the high sequence identity, which could be used in the generation of specific inhibitors.  Screening of a kinase inhibitor library revealed molecules that bind potently to CAMKK1.  Isothermal titration calorimetry revealed that the most potent inhibitors had binding energies largely dependent on favourable enthalpy.  Together, the data provide a foundation for future inhibitor development activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTiTkdNpgO7swHqQV89nF-2fW6udTcc2eYGh_htGAGCGLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czmtVajug%253D%253D&md5=3757fbb8f8e0e9ba37c3bb0520594787</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-33043-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-33043-4%26sid%3Dliteratum%253Aachs%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26aulast%3DRamos%26aufirst%3DP.%2BZ.%26aulast%3DGodoi%26aufirst%3DP.%2BH.%2BC.%26aulast%3DMassirer%26aufirst%3DK.%2BB.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26atitle%3DStructural%2520analysis%2520of%2520inhibitor%2520binding%2520to%2520CAMKK1%2520Identifies%2520Features%2520Necessary%2520for%2520Design%2520of%2520Specific%2520Inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D14800%26doi%3D10.1038%2Fs41598-018-33043-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljpL1D6dVOzRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Profeta, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Reis, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, P. H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fala, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmazo, A. P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massirer, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span> <span> </span><span class="NLM_article-title">Binding and structural analyses of potent inhibitors of the human Ca(2+)/calmodulin dependent protein kinase kinase 2 (CAMKK2) identified from a collection of commercially-available kinase inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">16452</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-52795-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-019-52795-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=31712618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB3Mjnsleqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=16452&author=G.+S.+Profetaauthor=C.+V.+Dos+Reisauthor=A.+D.+S.+Santiagoauthor=P.+H.+C.+Godoiauthor=A.+M.+Falaauthor=C.+I.+Wellsauthor=R.+Sartoriauthor=A.+P.+T.+Salmazoauthor=P.+Z.+Ramosauthor=K.+B.+Massirerauthor=J.+M.+Elkinsauthor=D.+H.+Drewryauthor=O.+Gileadiauthor=R.+M.+Cou%C3%B1ago&title=Binding+and+structural+analyses+of+potent+inhibitors+of+the+human+Ca%282%2B%29%2Fcalmodulin+dependent+protein+kinase+kinase+2+%28CAMKK2%29+identified+from+a+collection+of+commercially-available+kinase+inhibitors&doi=10.1038%2Fs41598-019-52795-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Binding and structural analyses of potent inhibitors of the human Ca(2+)/calmodulin dependent protein kinase kinase 2 (CAMKK2) identified from a collection of commercially-available kinase inhibitors</span></div><div class="casAuthors">Profeta Gerson S; Dos Reis Caio V; Santiago Andre da S; Godoi Paulo H C; Fala Angela M; Ramos Priscila Z; Massirer Katlin B; Counago Rafael M; Profeta Gerson S; Dos Reis Caio V; Santiago Andre da S; Godoi Paulo H C; Fala Angela M; Sartori Roger; Salmazo Anita P T; Ramos Priscila Z; Massirer Katlin B; Elkins Jonathan M; Counago Rafael M; Wells Carrow I; Drewry David H; Elkins Jonathan M; Gileadi Opher</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16452</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Calcium/Calmodulin-dependent Protein Kinase Kinase 2 (CAMKK2) acts as a signaling hub, receiving signals from various regulatory pathways and decoding them via phosphorylation of downstream protein kinases - such as AMPK (AMP-activated protein kinase) and CAMK types I and IV.  CAMKK2 relevance is highlighted by its constitutive activity being implicated in several human pathologies.  However, at present, there are no selective small-molecule inhibitors available for this protein kinase.  Moreover, CAMKK2 and its closest human homolog, CAMKK1, are thought to have overlapping biological roles.  Here we present six new co-structures of potent ligands bound to CAMKK2 identified from a library of commercially-available kinase inhibitors.  Enzyme assays confirmed that most of these compounds are equipotent inhibitors of both human CAMKKs and isothermal titration calorimetry (ITC) revealed that binding to some of these molecules to CAMKK2 is enthalpy driven.  We expect our results to advance current efforts to discover small molecule kinase inhibitors selective to each human CAMKK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4tR5q1L6mjO-MdeMhXAHkfW6udTcc2eZOUCJ4Qe57_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mjnsleqtg%253D%253D&md5=d0605c597cecd456b71a85e88e45b38f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-52795-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-52795-1%26sid%3Dliteratum%253Aachs%26aulast%3DProfeta%26aufirst%3DG.%2BS.%26aulast%3DDos%2BReis%26aufirst%3DC.%2BV.%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DGodoi%26aufirst%3DP.%2BH.%2BC.%26aulast%3DFala%26aufirst%3DA.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DSartori%26aufirst%3DR.%26aulast%3DSalmazo%26aufirst%3DA.%2BP.%2BT.%26aulast%3DRamos%26aufirst%3DP.%2BZ.%26aulast%3DMassirer%26aufirst%3DK.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520and%2520structural%2520analyses%2520of%2520potent%2520inhibitors%2520of%2520the%2520human%2520Ca%25282%252B%2529%252Fcalmodulin%2520dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CAMKK2%2529%2520identified%2520from%2520a%2520collection%2520of%2520commercially-available%2520kinase%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D16452%26doi%3D10.1038%2Fs41598-019-52795-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lihSJDpeQOo-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beno, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">A Survey of the role of noncovalent sulfur interactions in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4383</span>– <span class="NLM_lpage">4438</span>, <span class="refDoi"> DOI: 10.1021/jm501853m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501853m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Giu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4383-4438&author=B.+R.+Benoauthor=K.-S.+Yeungauthor=M.+D.+Bartbergerauthor=L.+D.+Penningtonauthor=N.+A.+Meanwell&title=A+Survey+of+the+role+of+noncovalent+sulfur+interactions+in+drug+design&doi=10.1021%2Fjm501853m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Survey of the Role of Noncovalent Sulfur Interactions in Drug Design</span></div><div class="casAuthors">Beno, Brett R.; Yeung, Kap-Sun; Bartberger, Michael D.; Pennington, Lewis D.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4383-4438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Electron deficient, bivalent sulfur atoms have two areas of pos. electrostatic potential, a consequence of the low-lying σ* orbitals of the C-S bond that are available for interaction with electron donors including oxygen and nitrogen atoms and, possibly, π-systems.  Intramol. interactions are by far the most common manifestation of this effect, which offers a means of modulating the conformational preferences of a mol.  Although a well-documented phenomenon, a priori applications in drug design are relatively sparse and this interaction, which is often isosteric with an intramol. hydrogen-bonding interaction, appears to be underappreciated by the medicinal chem. community.  In this Perspective, we discuss the theor. basis for sulfur σ* orbital interactions and illustrate their importance in the context of drug design and org. synthesis.  The role of sulfur interactions in protein structure and function is discussed and although relatively rare, intermol. interactions between ligand C-S σ* orbitals and proteins are illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYKXRsl4BQYrVg90H21EOLACvtfcHk0lihSJDpeQOo-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Giu7g%253D&md5=d12f53eb3271c161cfc833fb9f7c02b3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm501853m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501853m%26sid%3Dliteratum%253Aachs%26aulast%3DBeno%26aufirst%3DB.%2BR.%26aulast%3DYeung%26aufirst%3DK.-S.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DPennington%26aufirst%3DL.%2BD.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DA%2520Survey%2520of%2520the%2520role%2520of%2520noncovalent%2520sulfur%2520interactions%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4383%26epage%3D4438%26doi%3D10.1021%2Fjm501853m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Intermolecular sulfur...oxygen interactions: theoretical and statistical investigations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2138</span>– <span class="NLM_lpage">2153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2138-2153&author=X.+Zhangauthor=Z.+Gongauthor=J.+Liauthor=T.+Lu&title=Intermolecular+sulfur...oxygen+interactions%3A+theoretical+and+statistical+investigations&doi=10.1021%2Facs.jcim.5b00177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span></div><div class="casAuthors">Zhang, Xuejin; Gong, Zhen; Li, Jian; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2138-2153</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intermol. S···O interactions are very common and are important in biol. systems, but until recently, the presence of these contacts in protein-ligand systems largely depended on serendipitous discovery instead of rational design.  Here we provide insight into the phenomenon of intermol. S···O contacts by focusing on three sulfur-contg. arom. rings.  Quantum mechanics is employed to characterize the strength and directionality of the S···O interactions and to det. their energy dependence on their geometric parameters.  Protein Data Bank mining is performed to systematically det. the occurrence and geometry of intermol. S···O interactions, and several representative examples are discussed.  Three typical cases are investigated using a combined quantum mechanics/mol. mechanics approach to demonstrate the potential of these interactions in improving binding affinities and physiochem. properties.  Overall, our work elucidates the structures and energy features of intermol. S···O interactions and addresses their use in mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2J_2Nq9PDQLVg90H21EOLACvtfcHk0lh_TM6bUEEQ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI&md5=7192e3b57e66fc6eb9dd53165427d449</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00177%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DIntermolecular%2520sulfur...oxygen%2520interactions%253A%2520theoretical%2520and%2520statistical%2520investigations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2138%26epage%3D2153%26doi%3D10.1021%2Facs.jcim.5b00177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span> <span> </span><span class="NLM_article-title">STO-609, a specific inhibitor of the Ca(2+)/calmodulin-dependent protein kinase kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">15813</span>– <span class="NLM_lpage">15818</span>, <span class="refDoi"> DOI: 10.1074/jbc.m201075200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M201075200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11867640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjs1Wjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=15813-15818&author=H.+Tokumitsuauthor=H.+Inuzukaauthor=Y.+Ishikawaauthor=M.+Ikedaauthor=I.+Sajiauthor=R.+Kobayashi&title=STO-609%2C+a+specific+inhibitor+of+the+Ca%282%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase&doi=10.1074%2Fjbc.m201075200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">STO-609, a specific inhibitor of the Ca2+/calmodulin-dependent protein kinase kinase</span></div><div class="casAuthors">Tokumitsu, Hiroshi; Inuzuka, Hiroyuki; Ishikawa, Yumi; Ikeda, Masahiko; Saji, Ikutaro; Kobayashi, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">15813-15818</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">STO-609, a selective inhibitor of Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK) was synthesized, and its inhibitory properties were investigated both in vitro and in vivo.  STO-609 inhibits the activities of recombinant CaM-KKα and CaM-KKβ isoforms, with Ki values of 80 and 15 ng/mL, resp., and also inhibits their autophosphorylation activities.  Comparison of the inhibitory potency of the compd. against various protein kinases revealed that STO-609 is highly selective for CaM-KK without any significant effect on the downstream CaM kinases (CaM-KI and -IV), and the IC50 value of the compd. against CaM-KII is ∼ 10 μg/mL.  STO-609 inhibits constitutively active CaM-KKα (glutathione S-transferase (GST)-CaM-KK-(84-434)) as well as the wild-type enzyme.  Kinetic anal. indicates that the compd. is a competitive inhibitor of ATP.  In transfected HeLa cells, STO-609 suppresses the Ca2+-induced activation of CaM-KIV in a dose-dependent manner.  In agreement with this observation, the inhibitor significantly reduces the endogenous activity of CaM-KK in SH-SY5Y neuroblastoma cells at a concn. of 1 μg/mL (∼80% inhibitory rate).  Taken together, these results indicate that STO-609 is a selective and cell-permeable inhibitor of CaM-KK and that it may be a useful tool for evaluating the physiol. significance of the CaM-KK-mediated pathway in vivo as well as in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOo8PRZ4cKCbVg90H21EOLACvtfcHk0lh_TM6bUEEQ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjs1Wjsbs%253D&md5=03e436ce8278669ff528ecad280d4955</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M201075200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M201075200%26sid%3Dliteratum%253Aachs%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DSaji%26aufirst%3DI.%26aulast%3DKobayashi%26aufirst%3DR.%26atitle%3DSTO-609%252C%2520a%2520specific%2520inhibitor%2520of%2520the%2520Ca%25282%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D15813%26epage%3D15818%26doi%3D10.1074%2Fjbc.m201075200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span> <span> </span><span class="NLM_article-title">A single amino acid difference between alpha and beta Ca2+/calmodulin-dependent protein kinase kinase dictates sensitivity to the specific inhibitor, STO-609</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">10908</span>– <span class="NLM_lpage">10913</span>, <span class="refDoi"> DOI: 10.1074/jbc.m213183200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M213183200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=12540834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1Khurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=10908-10913&author=H.+Tokumitsuauthor=H.+Inuzukaauthor=Y.+Ishikawaauthor=R.+Kobayashi&title=A+single+amino+acid+difference+between+alpha+and+beta+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+dictates+sensitivity+to+the+specific+inhibitor%2C+STO-609&doi=10.1074%2Fjbc.m213183200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Single Amino Acid Difference between α and β Ca2+/Calmodulin-dependent Protein Kinase Kinase Dictates Sensitivity to the Specific Inhibitor, STO-609</span></div><div class="casAuthors">Tokumitsu, Hiroshi; Inuzuka, Hiroyuki; Ishikawa, Yumi; Kobayashi, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10908-10913</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We recently developed STO-609, a selective inhibitor of Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK), and we demonstrated that CaM-KKβ is more sensitive to STO-609 than the CaM-KKα isoform.  By using catalytic chimera and point mutants of both isoforms, we demonstrated that Val269 in CaM-KKβ/Leu233 in CaM-KKα confers a distinct sensitivity (~10-fold) to STO-609 on CaM-KK isoforms.  Various mutations of Val269 in CaM-KKβ indicate that substitution by hydrophobic residues with bulky side chains significantly decreases drug sensitivity and that the V269F mutant is the most effective drug-resistant enzyme (~80-fold higher IC50 value).  These findings are consistent with a result obtained with a full-length mutant expressed in COS-7 cells.  Furthermore, suppression of CaM-KK-mediated CaM-KIV activation in transfected HeLa cells by STO-609 treatment was completely abolished by the co-expression of the CaM-KKβ V269F mutant.  Based on the results that the distinct sensitivity of CaM-KK isoforms to STO-609 is because of a single amino acid substitution (Val/Leu) in the ATP-binding pocket, we have generated an STO-609-resistant CaM-KK mutant, which might be useful for validating the pharmacol. effects and specificity of STO-609 in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5Fl3MmunxrVg90H21EOLACvtfcHk0lgx_iN96JSaYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1Khurk%253D&md5=6a372bc54100e90b7d37dfe2935ea2a8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M213183200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M213183200%26sid%3Dliteratum%253Aachs%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DKobayashi%26aufirst%3DR.%26atitle%3DA%2520single%2520amino%2520acid%2520difference%2520between%2520alpha%2520and%2520beta%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520dictates%2520sensitivity%2520to%2520the%2520specific%2520inhibitor%252C%2520STO-609%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D10908%26epage%3D10913%26doi%3D10.1074%2Fjbc.m213183200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span> <span> </span><span class="NLM_article-title">High throughput solubility measurement in drug discovery and development</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.addr.2007.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=17604872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=546-567&author=J.+Alsenzauthor=M.+Kansy&title=High+throughput+solubility+measurement+in+drug+discovery+and+development&doi=10.1016%2Fj.addr.2007.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput solubility measurement in drug discovery and development</span></div><div class="casAuthors">Alsenz, Jochem; Kansy, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">546-567</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Measurement of drug soly. in various solvents is one of the key elements of compd. characterization during the whole discovery and development process.  This review summarizes current exptl. approaches and addresses recent advances in the exptl. methods used to det. drug soly. in drug discovery and early development.  This paper focuses on high throughput methods designed to det. kinetic and thermodn. (equil.) soly. but traditional methods are also presented.  The focus, positioning, exptl. setup, pros and cons, and limitations of individual assays are discussed and differences in soly. studies in discovery and development environments are highlighted.  Finally, future needs and trends in soly. assay development designed to overcome current bottlenecks and trade-offs between speed and quality/quantity of measurements are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsvOlLd-yoxLVg90H21EOLACvtfcHk0lgx_iN96JSaYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D&md5=8d55ead3ea0c13fa49b032db725842b0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3DKansy%26aufirst%3DM.%26atitle%3DHigh%2520throughput%2520solubility%2520measurement%2520in%2520drug%2520discovery%2520and%2520development%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D546%26epage%3D567%26doi%3D10.1016%2Fj.addr.2007.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lgs69cAqxfw4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2010%26volume%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2Fs0169-409x%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lgs69cAqxfw4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2Fs0169-409x%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majoral, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Present drug-likeness filters in medicinal chemistry during the hit and lead optimization process: how far can they be simplified?</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">615</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.drudis.2018.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29330127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC1MvgtlCksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=605-615&author=S.+Mignaniauthor=J.+Rodriguesauthor=H.+Tomasauthor=R.+Jalalauthor=P.+P.+Singhauthor=J.-P.+Majoralauthor=R.+A.+Vishwakarma&title=Present+drug-likeness+filters+in+medicinal+chemistry+during+the+hit+and+lead+optimization+process%3A+how+far+can+they+be+simplified%3F&doi=10.1016%2Fj.drudis.2018.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Present drug-likeness filters in medicinal chemistry during the hit and lead optimization process: how far can they be simplified?</span></div><div class="casAuthors">Mignani Serge; Rodrigues Joao; Tomas Helena; Jalal Rachid; Singh Parvinder Pal; Vishwakarma Ram A; Majoral Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">605-615</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">During the past decade, decreasing the attrition rate of drug development candidates reaching the market has become one of the major challenges in pharmaceutical research and drug development (R&D).  To facilitate the decision-making process, and to increase the probability of rapidly finding and developing high-quality compounds, a variety of multiparametric guidelines, also known as rules and ligand efficiency (LE) metrics, have been developed.  However, what are the 'best' descriptors and how far can we simplify these drug-likeness prediction tools in terms of the numerous, complex properties that they relate to?</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2XT6-2EQJ3n6oPH0mYWL1fW6udTcc2eZebvQHR2dpB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvgtlCksw%253D%253D&md5=a61e00552a35b455199f0a8d351d0522</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DMignani%26aufirst%3DS.%26aulast%3DRodrigues%26aufirst%3DJ.%26aulast%3DTomas%26aufirst%3DH.%26aulast%3DJalal%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DP.%2BP.%26aulast%3DMajoral%26aufirst%3DJ.-P.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DPresent%2520drug-likeness%2520filters%2520in%2520medicinal%2520chemistry%2520during%2520the%2520hit%2520and%2520lead%2520optimization%2520process%253A%2520how%2520far%2520can%2520they%2520be%2520simplified%253F%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2018%26volume%3D23%26spage%3D605%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2018.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lhrBSWcwYbCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petereit, A. C.</span></span> <span> </span><span class="NLM_article-title">Optimizing solubility: kinetic versus thermodynamic solubility temptations and risks</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.ejps.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22885099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=589-595&author=C.+Saalauthor=A.+C.+Petereit&title=Optimizing+solubility%3A+kinetic+versus+thermodynamic+solubility+temptations+and+risks&doi=10.1016%2Fj.ejps.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing solubility: Kinetic versus thermodynamic solubility temptations and risks</span></div><div class="casAuthors">Saal, Christoph; Petereit, Anna Christine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to assess the usefulness of kinetic and thermodn. soly. data in guiding medicinal chem. during lead optimization.  The soly. of 465 research compds. was measured using a kinetic and a thermodn. soly. assay.  In the thermodn. assay, polarized-light microscopy was used to investigate whether the result referred to the cryst. or to the amorphous compd.  From the comparison of kinetic and thermodn. soly. data it was noted that kinetic soly. measurements frequently yielded results which show considerably higher soly. compared to thermodn. soly.  This observation is ascribed to the fact that a kinetic soly. assay typically delivers results which refer to the amorphous compd.  In contrast, results from thermodn. soly. detns. more frequently refer to a cryst. phase.  Accordingly, thermodn. soly. data - esp. when used together with an assessment of the solid state form - are deemed to be more useful in guiding soly. optimization for research compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxKTQHBpynbVg90H21EOLACvtfcHk0lhrBSWcwYbCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ&md5=ace8b962c04c3a62f028ec3e66522922</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DSaal%26aufirst%3DC.%26aulast%3DPetereit%26aufirst%3DA.%2BC.%26atitle%3DOptimizing%2520solubility%253A%2520kinetic%2520versus%2520thermodynamic%2520solubility%2520temptations%2520and%2520risks%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2012%26volume%3D47%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fj.ejps.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0liN6sgP4MRRsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of the CaMKKbeta-AMPK signaling complex</span>. <i>Cell. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2005</span>– <span class="NLM_lpage">2012</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2011.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cellsig.2011.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21807092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kku7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2011&pages=2005-2012&author=M.+F.+Greenauthor=K.+A.+Andersonauthor=A.+R.+Means&title=Characterization+of+the+CaMKKbeta-AMPK+signaling+complex&doi=10.1016%2Fj.cellsig.2011.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CaMKKβ-AMPK signaling complex</span></div><div class="casAuthors">Green, Michelle F.; Anderson, Kristin A.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2005-2012</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The AMP-activated protein kinase (AMPK) is a crit. regulator of energy homeostasis, and is a potential target for treatment of metabolic diseases as well as cancer.  AMPK can be phosphorylated and activated by the tumor suppressor LKB1 or the Ca2+/CaM-dependent protein kinase kinase β (CaMKKβ).  We previously identified a phys. complex between CaMKKβ and AMPK (Anderson, K. A., Ribar, T. J., Lin, F., Noeldner, P. K., Green, M. F., Muehlbauer, M. J., Witters, L. A., Kemp, B. E., and Means, A. R. (2008) Cell Metab. 7, 377-388).  Here we expand our anal. of the CaMKKβ-AMPK signaling complex and show that whereas CaMKKβ can form a complex with and activate AMPK, CaMKKα cannot.  In addn., we show that CaMKKβ and AMPK assoc. through their kinase domains, and CaMKKβ must be in an active conformation in order to bind AMPK but not to assoc. with an alternative substrate, Ca2+/Calmodulin-dependent protein kinase IV (CaMKIV).  Our results demonstrate that CaMKKβ and AMPK form a unique signaling complex.  This raises the possibility that the CaMKKβ-AMPK complex can be specifically targeted by small mol. drugs to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBivJ0b2Ntj7Vg90H21EOLACvtfcHk0liN6sgP4MRRsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kku7zN&md5=edbf3362e408d902b4a580e96ad03207</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2011.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2011.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BF.%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCharacterization%2520of%2520the%2520CaMKKbeta-AMPK%2520signaling%2520complex%26jtitle%3DCell.%2520Signal.%26date%3D2011%26volume%3D23%26spage%3D2005%26epage%3D2012%26doi%3D10.1016%2Fj.cellsig.2011.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0licba5UElnIxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xy
Ling, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundlach, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of an R311C variant of Ca(2+) -calmodulin-dependent protein kinase kinase-2 (CaMKK2) found as a de novo mutation in a patient with bipolar disorder</span>. <i>Bipolar Disord.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1111/bdi.12901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1111%2Fbdi.12901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32216002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB383nvVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=841-848&author=N.+Xy%0ALingauthor=C.+G.+Langendorfauthor=A.+Hoqueauthor=S.+Galicauthor=K.+Lohauthor=B.+E.+Kempauthor=A.+L.+Gundlachauthor=J.+S.+Oakhillauthor=J.+W.+Scott&title=Functional+analysis+of+an+R311C+variant+of+Ca%282%2B%29+-calmodulin-dependent+protein+kinase+kinase-2+%28CaMKK2%29+found+as+a+de+novo+mutation+in+a+patient+with+bipolar+disorder&doi=10.1111%2Fbdi.12901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of an R311C variant of Ca(2+) -calmodulin-dependent protein kinase kinase-2 (CaMKK2) found as a de novo mutation in a patient with bipolar disorder</span></div><div class="casAuthors">Xy Ling Naomi; Langendorf Christopher G; Hoque Ashfaqul; Galic Sandra; Loh Kim; Kemp Bruce E; Oakhill Jonathan S; Scott John W; Kemp Bruce E; Oakhill Jonathan S; Scott John W; Gundlach Andrew L; Scott John W</div><div class="citationInfo"><span class="NLM_cas:title">Bipolar disorders</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">841-848</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Loss-of-function mutations in the gene encoding the calcium-calmodulin (Ca(2+) -CaM)-dependent protein kinase kinase-2 (CaMKK2) enzyme are linked to bipolar disorder.  Recently, a de novo arginine to cysteine (R311C) mutation in CaMKK2 was identified from a whole exome sequencing study of bipolar patients and their unaffected parents.  The aim of the present study was to determine the functional consequences of the R311C mutation on CaMKK2 activity and regulation by Ca(2+) -CaM.  METHODS:  The effects of the R311C mutation on CaMKK2 activity and Ca(2+) -CaM activation were examined using a radiolabeled adenosine triphosphate (ATP) kinase assay.  We performed immunoblot analysis to determine whether the R311C mutation impacts threonine-85 (T85) autophosphorylation, an activating phosphorylation site on CaMKK2 that has also been implicated in bipolar disorder.  We also expressed the R311C mutant in CaMKK2 knockout HAP1 cells and used immunoblot analysis and an MTS reduction assay to study its effects on Ca(2+) -dependent downstream signaling and cell viability, respectively.  RESULTS:  The R311C mutation maps to the conserved HRD motif within the catalytic loop of CaMKK2 and caused a marked reduction in kinase activity and Ca(2+) -CaM activation.  The R311C mutation virtually abolished T85 autophosphorylation in response to Ca(2+) -CaM and exerted a dominant-negative effect in cells as it impaired the ability of wild-type CaMKK2 to initiate downstream signaling and maintain cell viability.  CONCLUSIONS:  The highly disruptive, loss-of-function impact of the de novo R311C mutation in human CaMKK2 provides a compelling functional rationale for being considered a potential rare monogenic cause of bipolar disorder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0-Sz3Da-0oGHOrQ6qeX-mfW6udTcc2ea5nJAyY6PRqrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383nvVSquw%253D%253D&md5=73d279451881e35e7b8ac881bf5583da</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1111%2Fbdi.12901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbdi.12901%26sid%3Dliteratum%253Aachs%26aulast%3DXy%2BLing%26aufirst%3DN.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DHoque%26aufirst%3DA.%26aulast%3DGalic%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DK.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DGundlach%26aufirst%3DA.%2BL.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26atitle%3DFunctional%2520analysis%2520of%2520an%2520R311C%2520variant%2520of%2520Ca%25282%252B%2529%2520-calmodulin-dependent%2520protein%2520kinase%2520kinase-2%2520%2528CaMKK2%2529%2520found%2520as%2520a%2520de%2520novo%2520mutation%2520in%2520a%2520patient%2520with%2520bipolar%2520disorder%26jtitle%3DBipolar%2520Disord.%26date%3D2020%26volume%3D22%26spage%3D841%26epage%3D848%26doi%3D10.1111%2Fbdi.12901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span> <span> </span><span class="NLM_article-title">Novel buffer systems for macromolecular crystallization</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1107/s0907444903029640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444903029640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=14993709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1OrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=610-612&author=J.+Newman&title=Novel+buffer+systems+for+macromolecular+crystallization&doi=10.1107%2Fs0907444903029640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel buffer systems for macromolecular crystallization</span></div><div class="casAuthors">Newman, Janet</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">610-612</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">In protein crystn., screening is initially performed to obtain an indication of the conditions under which a macromol. might crystallize.  These preliminary conditions are then optimized to produce (in a perfect world) well diffracting crystals; this process of optimization often involves fine grid screening around the initial conditions.  An issue in optimization is to find factors which are independent, so as to simplify the anal. of the results of optimization trials.  This is necessarily difficult with buffers, as a buffer and its pH range tend to be very highly correlated.  Multi-buffer systems for pH modulation are presented which enable a broad pH range to be sampled without changing the chem. compn. of the buffering component.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBsN7Yw2gPhbVg90H21EOLACvtfcHk0lhm9gnvQmZWPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1OrtLY%253D&md5=2153f6feed4c3e2ca920e5f10d25d521</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0907444903029640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903029640%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%26atitle%3DNovel%2520buffer%2520systems%2520for%2520macromolecular%2520crystallization%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D610%26epage%3D612%26doi%3D10.1107%2Fs0907444903029640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhm9gnvQmZWPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhHQdl1oXtK7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhHQdl1oXtK7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomabechi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada-Iwabu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwabu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minokoshi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Ca(2)(+)/calmodulin-dependent protein kinase kinase in complex with the inhibitor STO-609</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">22570</span>– <span class="NLM_lpage">22579</span>, <span class="refDoi"> DOI: 10.1074/jbc.m111.251710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M111.251710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21504895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1Kkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=22570-22579&author=M.+Kukimoto-Niinoauthor=S.+Yoshikawaauthor=T.+Takagiauthor=N.+Ohsawaauthor=Y.+Tomabechiauthor=T.+Teradaauthor=M.+Shirouzuauthor=A.+Suzukiauthor=S.+Leeauthor=T.+Yamauchiauthor=M.+Okada-Iwabuauthor=M.+Iwabuauthor=T.+Kadowakiauthor=Y.+Minokoshiauthor=S.+Yokoyama&title=Crystal+structure+of+the+Ca%282%29%28%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+in+complex+with+the+inhibitor+STO-609&doi=10.1074%2Fjbc.m111.251710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Ca2+/calmodulin-dependent protein kinase kinase in complex with the inhibitor STO-609</span></div><div class="casAuthors">Kukimoto-Niino, Mutsuko; Yoshikawa, Seiko; Takagi, Tetsuo; Ohsawa, Noboru; Tomabechi, Yuri; Terada, Takaho; Shirouzu, Mikako; Suzuki, Atsushi; Lee, Suni; Yamauchi, Toshimasa; Okada-Iwabu, Miki; Iwabu, Masato; Kadowaki, Takashi; Minokoshi, Yasuhiko; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">22570-22579</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ca2+/calmodulin (CaM)-dependent protein kinase (CaMK) kinase (calmodulin kinase kinase; CaMKK) is a member of the CaMK cascade that mediates the response to intracellular Ca2+ elevation.  CaMKK phosphorylates and activates CaMKI and CaMKIV, which directly activate transcription factors.  Here, the authors detd. the 2.4-Å crystal structure of the catalytic kinase domain of the human CaMKK β-isoform (CaMKKβ) complexed with its selective inhibitor, STO-609.  The structure revealed that CaMKKβ lacked the αD helix and that the equiv. region displayed a hydrophobic mol. surface, which may reflect its unique substrate recognition and autoinhibition.  Although CaMKKβ lacked the activation loop phosphorylation site, the activation loop was folded in an active-state conformation, which was stabilized by a no. of interactions between amino acid residues conserved among the CaMKK isoforms.  An in vitro anal. of the kinase activity confirmed the intrinsic activity of the CaMKKβ kinase domain.  Structure and sequence analyses of the STO-609-binding site revealed amino acid replacements that may affect the inhibitor binding.  Indeed, mutagenesis demonstrated that the CaMKKβ residue Pro-274, which replaced the conserved acidic residue of other protein kinases, was an important determinant for the selective inhibition by STO-609.  Therefore, the present structure provides a mol. basis for clarifying the known biochem. properties of CaMKKβ and for designing novel inhibitors targeting CaMKKβ and the related protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFyEDOtLOpbVg90H21EOLACvtfcHk0lhHQdl1oXtK7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1Kkt7o%253D&md5=ff72e627808eb4880e34c847767295d3</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.251710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.251710%26sid%3Dliteratum%253Aachs%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DS.%26aulast%3DTakagi%26aufirst%3DT.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DTomabechi%26aufirst%3DY.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYamauchi%26aufirst%3DT.%26aulast%3DOkada-Iwabu%26aufirst%3DM.%26aulast%3DIwabu%26aufirst%3DM.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DMinokoshi%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Ca%25282%2529%2528%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520in%2520complex%2520with%2520the%2520inhibitor%2520STO-609%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D22570%26epage%3D22579%26doi%3D10.1074%2Fjbc.m111.251710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0ljALg9HBTF2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljALg9HBTF2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljALg9HBTF2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known CAMKK2 inhibitors: STO-609 and GSK650394.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Binding mode of the co-crystallized ligand GSK650394 in the ATP-binding site of CAMKK2 (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>). The image was generated with PyMOL. The protein is colored in gray. Blue-dashed lines indicate H-bond interactions; green-dashed lines display CH−π interactions. GSK650394 is shown as purple sticks and the water molecule as a red sphere. Oxygen and nitrogen atoms are colored in red and blue, respectively. (b) Structure of GSK650394 highlighting important sites for design of new CAMKK2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. New molecules based on scaffold hopping from GSK650394 designed to interrogate several aspects of hinge binding interactions between CAMKK2 and the proposed inhibitors. Top row: Inhibitors retaining the topology of GSK650394 (5,6-fused ring systems with the cyclopentyl-substituted benzoic acid moiety attached to the 5-membered ring). Middle row: 5,6-fused systems with the cyclopentyl-substituted benzoic acid moiety appended to the 6-membered ring. Bottom row: Inhibitors with ring expansion (6,6-fused ring system) or ring deletion (single 6-membered ring as the hinge-binding core).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pinacol Boronate Esters <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) cyclopentylmagnesium bromide, THF, −10 to 25 °C, 5 h, then aq HCl; (b) PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, KOAc, bis(pinacolato)diboron, 100 °C, 2 h; (c) SOCl<sub>2</sub>, MeOH, 16 h, 75 °C.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Azaindole <b>7</b>, the Matched Pair for GSK650394<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SEM-Cl, NaH, DMF, 0–21 °C, 2 h; (b) <b>4</b>, Pd(OAc)<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, P(Cy)<sub>3</sub>, PhMe/H<sub>2</sub>O, 80 °C, 16 h; (c) CF<sub>3</sub>COOH, CH<sub>2</sub>Cl<sub>2</sub>, then NaOAc, EtOH, 21 °C, 24 h; (d) aq NaOH, MeOH, 75 °C, 1 h, then aq HCl.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <i>N</i>-Methyl Azaindole Hinge Binders <b>10</b> and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; (b) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (c) <b>4</b>, PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, rt, 16 h; (d) PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Route to the Furo- and Thienopyridines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (EtO)<sub>3</sub>CH, PhMe, 100 °C, 2 h or H<sub>2</sub>SO<sub>4</sub>, EtOH, reflux, 24 h; (b) ethyl glycolate or ethylthioglycolate, NaH, DME, 0–75 °C, 2.5 h; (c) aq NaOH, EtOH then aq HCl, 100 °C, 2 h; (d) Tf<sub>2</sub>O, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 3 h; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h; (f) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h or PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O 90 °C, 16 h; (g) aq NaOH, MeOH then aq HCl, 1 h; (h) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 90 °C, 1 h.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Pyrazolopyridine and Pyrazolopyrimidine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; <b>17</b><i>via</i> (b) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 16 h; <b>18</b><i>via</i> (c) PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 80 °C, 16 h; (d) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 120 °C, 30 min, μW; (e) aq LiOH, dioxane, 100 °C, 16 h then aq HCl.</p></p></figure><figure data-id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of the Triazolopyridazine Analogues <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (b) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (c) Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (d) aq NaOH, MeOH 75 °C, 1 h, then aq HCl.</p></p></figure><figure data-id="sch7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of 2,4-Substituted <i>N</i>-Methyl Azaindoles<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (b) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (c) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (d) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 30 min; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><figure data-id="sch8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of the Imidazopyridine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 100 °C, 16 h; (b) CH(OMe)<sub>3</sub>, H<sub>3</sub>NSO<sub>3</sub>, MeOH, rt, 1 h; (c) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (d) benzoic acid, CDI, DMF, 0 °C, 30 min, then <b>32</b>, 200 °C, 10 min; (e) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><figure data-id="sch9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Thieno[3,2-<i>d</i>]pyrimidine and Thieno[2,3-<i>d</i>]pyrimidine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, μW, 15 min; (b) PhB(OH)<sub>2</sub>, NaHCO<sub>3</sub>, CsF, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O, 95 °C, 3 h; (c) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, 15 min; (d) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 125 °C, μW, 15 min; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/water, 145 °C, μW, 15 min; (f) PhB(OH)<sub>2</sub>, NaHCO<sub>3</sub>, CsF, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O, 80 °C, 4 h; (g) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl.</p></p></figure><figure data-id="sch10" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Quinoline and Quinazoline Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (b) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, rt, 16 h; (c) PhB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, 120 °C, 16 h; (d) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (e) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 120 °C, 16 h; (f) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (g) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 16 h; (h) (i) DIPEA, Tf<sub>2</sub>O, CH<sub>2</sub>Cl<sub>2</sub>, 0–21 °C, 2 h, (ii) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH, 90 °C, 1 h; (i) aq NaOH, MeOH, 75 °C, 1 h, then aq HCl; (j) <b>4</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H2O, 90 °C, 16 h.</p></p></figure><figure data-id="sch11" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0020.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis of Pyrimidine and Aminopyrimidine Hinge Binders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>3</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,2-DME, H<sub>2</sub>O, 120 °C, 0.5 h. (b) <b>4</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,2-DME, H<sub>2</sub>O, 120 °C, 0.5 h; (c) aq LiOH, dioxane, 100 °C, 16 h, then aq HCl; (d) PhBr, Pd<sub>2</sub>(dba)<sub>3</sub>, XantPhos, NaO<sup><i>t</i></sup>Bu, PhMe, 80 °C, 19 h.</p></p></figure><figure data-id="fig4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray structures and in silico docking. In silico docking was performed with Glide, and images were generated with PyMOL. The protein is colored in gray. Blue-dashed lines indicate H-bond interactions, while green-dashed lines display CH−π interactions and orange-dashed lines refer to sulfur−σ hole bonding. Docked ligands are shown as yellow sticks or orange sticks, co-crystallized ligands as purple sticks, and the water molecule as a red sphere. Oxygen and nitrogen atoms are colored in red and blue, respectively. (a) Predicted binding mode of <b>14g</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6">5UY6</a> compared to the co-crystallized ligand <b>13g</b> (purple). (b) Predicted binding modes of <b>46</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a> and the co-crystallized ligand UNC10244803 (purple). (c) Predicted binding mode of <b>56</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>. (d) Predicted binding mode of <b>22</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a>. (e) Predicted binding modes of <b>10</b> (orange) and <b>29</b> (yellow) using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU">6BKU</a> (light gray) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a> (dark gray), respectively. (f) Predicted binding mode of <b>38</b> using the protein structure from PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ">5UYJ</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>In vitro</i> kinase selectivity profile of compound <b>7</b> (<b>5a</b>) and compound <b>62</b> (<b>5b</b>) at 1 μM (Eurofins DiscoverX KINOME<i>scan</i>). TREEspot interaction maps for <b>7</b> and <b>62</b> profiled against >400 human kinase targets. In this assay, primary screen binding interactions are reported as PoC, where lower values are an indication of stronger hits. Red dots depict the kinases for which PoC at 1 μM < 10. The size of the dot is based on the PoC values, with larger dots having smaller PoC values (more potent binders). PoC values for all kinases can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. CAMKK2 NanoBRET dose–response curves for compounds <b>13g</b>, <b>45</b>, <b>46</b>, <b>56</b>, and <b>62</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blots of the 18 compounds with IC<sub>50</sub> < 150 nM in the CAMKK2 enzyme assay, together with STO-609, screened at a single inhibitor concentration = 1 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/medium/jm0c02274_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Dose response for inhibition of phosphorylation of AMPK at Thr172 in C4-2 prostate cancer cells by compound <b>62</b>. (b). Calculated IC<sub>50</sub> values for inhibition of phosphorylation of AMPK at Thr172 in C4-2 prostate cancer cells for the top compounds from the single concentration experiment depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02274/20210715/images/large/jm0c02274_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02274&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i113">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14570" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14570" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 74 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Calmodulin: a prototypical calcium sensor</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/s0962-8924(00)01800-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2FS0962-8924%2800%2901800-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10884684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkvVyrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=322-328&author=D.+Chinauthor=A.+R.+Means&title=Calmodulin%3A+a+prototypical+calcium+sensor&doi=10.1016%2Fs0962-8924%2800%2901800-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin: a prototypical calcium sensor</span></div><div class="casAuthors">Chin, D.; Means, A. R.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">322-328</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 60 refs.  Calmodulin is the best studied and prototypical example of the E-F-hand family of Ca2+-sensing proteins.  Changes in intracellular Ca2+ concn. regulate calmodulin in 3 distinct ways.  First, at the cellular level, by directing its subcellular distribution.  Second, at the mol. level, by promoting different modes of assocn. with many target proteins.  Third, by directing a variety of conformational states in calmodulin that result in target-specific activation.  The calmodulin-dependent regulation of protein kinases illustrates the potential mechanisms by which Ca2+-sensing proteins can recognize and generate affinity and specificity for effectors in a Ca2+-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ThLB_NAqhbVg90H21EOLACvtfcHk0lidqXNGThEcCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkvVyrtLw%253D&md5=9296d0539a7923ea86ad388ca387a689</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0962-8924%2800%2901800-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0962-8924%252800%252901800-6%26sid%3Dliteratum%253Aachs%26aulast%3DChin%26aufirst%3DD.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCalmodulin%253A%2520a%2520prototypical%2520calcium%2520sensor%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2000%26volume%3D10%26spage%3D322%26epage%3D328%26doi%3D10.1016%2Fs0962-8924%2800%2901800-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hook, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Ca(2+)/CaM-dependent kinases: from activation to function</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.41.1.471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1146%2Fannurev.pharmtox.41.1.471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11264466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVaqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=471-505&author=S.+S.+Hookauthor=A.+R.+Means&title=Ca%282%2B%29%2FCaM-dependent+kinases%3A+from+activation+to+function&doi=10.1146%2Fannurev.pharmtox.41.1.471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ca2+/CaM-dependent kinases: from activation to function</span></div><div class="casAuthors">Hook, Sara S.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-505</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 214 refs.  Calmodulin (CaM) is an essential protein that serves as a ubiquitous intracellular receptor for Ca2+.  The Ca2+/CaM complex initiates a plethora of signaling cascades that culminate in alteration of cellular functions.  Among the many Ca2+/CaM-binding proteins to be discovered, the multifunctional protein kinases CaMKI, II, and IV play pivotal roles.  Our review focuses on this class of CaM kinases to illustrate the structural and biochem. basis for Ca2+/CaM interaction with and regulation of its target enzymes.  Gene transcription has been chosen as the functional endpoint to illustrate the recent advances in Ca2+/CaM-mediated signal transduction mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb-524IthxV7Vg90H21EOLACvtfcHk0lhMCqpoFqkspQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVaqt74%253D&md5=7784f5c4d753bbc15ff5cb77fcfc1161</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.41.1.471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.41.1.471%26sid%3Dliteratum%253Aachs%26aulast%3DHook%26aufirst%3DS.%2BS.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCa%25282%252B%2529%252FCaM-dependent%2520kinases%253A%2520from%2520activation%2520to%2520function%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2001%26volume%3D41%26spage%3D471%26epage%3D505%26doi%3D10.1146%2Fannurev.pharmtox.41.1.471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parameswaran, S.</span></span> <span> </span><span class="NLM_article-title">Calmodulin-binding proteins: A journey of 40 years</span>. <i>Cell Calcium</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.ceca.2018.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.ceca.2018.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=30205293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSnt73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=89-100&author=R.+K.+Sharmaauthor=S.+Parameswaran&title=Calmodulin-binding+proteins%3A+A+journey+of+40+years&doi=10.1016%2Fj.ceca.2018.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin-binding proteins: A journey of 40 years</span></div><div class="casAuthors">Sharma, Rajendra K.; Parameswaran, Sreejit</div><div class="citationInfo"><span class="NLM_cas:title">Cell Calcium</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">CECADV</span>;
        ISSN:<span class="NLM_cas:issn">0143-4160</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The proteins which bind to calmodulin in a Ca2+-dependent and reversible manner are known as calmodulin-binding proteins.  These proteins are involved in a multitude of processes in which Ca2+ and calmodulin play crucial roles.  Our group elucidated the mechanism and importance of these proteins in normal and diseased conditions.  Various calmodulin-binding proteins were discovered and purified from bovine tissue including a heat stable calmodulin-binding protein 70, calmodulin-dependent protein kinase VI and a high mol. wt. calmodulin-binding protein (HMWCaMBP).  We obsd. a complex interplay occurs between these and other Ca2+ and calmodulin-binding proteins during cardiac ischemia and reperfusion.  Purified cardiac HMWCaMBP is a homolog form of calpastatin and an inhibitor of the Ca2+-activated cysteine proteases, calpains and therefore can have cardioprotective role in ischemic conditions.  Calcineurin is a Ca2+ and calmodulin-dependent serine/threonine protein phosphatase showed increased phosphatase activity in ischemic heart through its direct interaction with Hsp70 and expression of calcineurin following ischemia suggests self-repair and favorable survival outcomes.  Calcineurin was also found to be present in other tissues including the eye; where its expression and calcineurin phosphatase activity varied.  In neurons, calcineurin may play a key role in initiating apoptosis-related pathways esp. in epilepsy.  In colorectal cancer we demonstrated high calcineurin phosphatase activity and simultaneous overexpression of calcineurin.  The impact of calcineurin signaling on neuronal apoptosis in epilepsy and its use as a diagnostic marker for colorectal cancer requires in-depth study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2LvL3lccyxLVg90H21EOLACvtfcHk0lhMCqpoFqkspQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSnt73E&md5=66bfa8ecdae78cea42ef30e1c092fdd2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ceca.2018.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceca.2018.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DR.%2BK.%26aulast%3DParameswaran%26aufirst%3DS.%26atitle%3DCalmodulin-binding%2520proteins%253A%2520A%2520journey%2520of%252040%2520years%26jtitle%3DCell%2520Calcium%26date%3D2018%26volume%3D75%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.ceca.2018.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clapham, D. E.</span></span> <span> </span><span class="NLM_article-title">Calcium signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">1047</span>– <span class="NLM_lpage">1058</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cell.2007.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18083096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2007&pages=1047-1058&author=D.+E.+Clapham&title=Calcium+signaling&doi=10.1016%2Fj.cell.2007.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium signaling</span></div><div class="casAuthors">Clapham, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1047-1058</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ca2+ ions impact nearly every aspect of cellular life.  Here, the author examines the principles of Ca2+ signaling, from changes in protein conformations driven by Ca2+ to the mechanisms that control Ca2+ levels in the cytoplasm and organelles.  Also discussed is the highly localized nature of Ca2+-mediated signal transduction and its specific role in excitability, exocytosis, motility, apoptosis, and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojw-AGybzZYrVg90H21EOLACvtfcHk0lhMCqpoFqkspQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFGgsA%253D%253D&md5=268945d7ca1ec7a455f69cb72272be85</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DClapham%26aufirst%3DD.%2BE.%26atitle%3DCalcium%2520signaling%26jtitle%3DCell%26date%3D2007%26volume%3D131%26spage%3D1047%26epage%3D1058%26doi%3D10.1016%2Fj.cell.2007.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Racioppi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Calcium/calmodulin-dependent protein kinase kinase 2: roles in signaling and pathophysiology</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">31658</span>– <span class="NLM_lpage">31665</span>, <span class="refDoi"> DOI: 10.1074/jbc.r112.356485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.R112.356485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22778263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGgsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=31658-31665&author=L.+Racioppiauthor=A.+R.+Means&title=Calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2%3A+roles+in+signaling+and+pathophysiology&doi=10.1074%2Fjbc.r112.356485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium/Calmodulin-dependent Protein Kinase Kinase 2: Roles in Signaling and Pathophysiology</span></div><div class="casAuthors">Racioppi, Luigi; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">31658-31665</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Many cellular Ca2+-dependent signaling cascades utilize calmodulin (CaM) as the intracellular Ca2+ receptor.  Ca2+/CaM binds and activates a plethora of enzymes, including CaM kinases (CaMKs).  CaMKK2 is one of the most versatile of the CaMKs and will phosphorylate and activate CaMKI, CaMKIV, and AMP-activated protein kinase.  Cell expression of CaMKK2 is limited, yet CaMKK2 is involved in regulating many important physiol. and pathophysiol. processes, including energy balance, adiposity, glucose homeostasis, hematopoiesis, inflammation, and cancer.  Here, we explore known functions of CaMKK2 and discuss its potential as a target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8CI0yjQ9IcrVg90H21EOLACvtfcHk0ljkNr_bn5tLDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGgsbnL&md5=c0eaa88730dd66d0c2add46ee57c176b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.356485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.356485%26sid%3Dliteratum%253Aachs%26aulast%3DRacioppi%26aufirst%3DL.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCalcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%253A%2520roles%2520in%2520signaling%2520and%2520pathophysiology%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D31658%26epage%3D31665%26doi%3D10.1074%2Fjbc.r112.356485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Components of a calmodulin-dependent protein kinase cascade. Molecular cloning, functional characterization and cellular localization of Ca2+/calmodulin-dependent protein kinase kinase beta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">31880</span>– <span class="NLM_lpage">31889</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.48.31880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.273.48.31880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=9822657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVeltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=31880-31889&author=K.+A.+Andersonauthor=R.+L.+Meansauthor=Q.-H.+Huangauthor=B.+E.+Kempauthor=E.+G.+Goldsteinauthor=M.+A.+Selbertauthor=A.+M.+Edelmanauthor=R.+T.+Fremeauauthor=A.+R.+Means&title=Components+of+a+calmodulin-dependent+protein+kinase+cascade.+Molecular+cloning%2C+functional+characterization+and+cellular+localization+of+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+beta&doi=10.1074%2Fjbc.273.48.31880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Components of a calmodulin-dependent protein kinase cascade. Molecular cloning, functional characterization and cellular localization of Ca2+/calmodulin-dependent protein kinase kinase β</span></div><div class="casAuthors">Anderson, Kristin A.; Means, Raylene L.; Huang, Qi-Hui; Kemp, Bruce E.; Goldstein, Elaine G.; Selbert, Michele A.; Edelman, Arthur M.; Fremeau, Robert T.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">31880-31889</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ca2+/calmodulin-dependent protein kinases I and IV (CaMKI and CaMKIV, resp.) require phosphorylation on an equiv. single Thr in the activation loop of subdomain VIII for maximal activity.  Two distinct CaMKI/IV kinases, CaMKKα and CaMKKβ, were purified from rat brain and partially sequenced (Edelman, A. M., Mitchelhill, K., Selbert, M. A., Anderson, K. A., Hook, S. S., Stapleton, D., Goldstein, E. G., Means, A. R., and Kemp, B. E. (1996) J. Biol. Chem. 271, 10806-10810).  We report here the cloning and sequencing of cDNAs for human and rat CaMKKβ, tissue and regional brain localization of CaMKKβ protein, and mRNA and functional characterization of recombinant CaMKKβ in vitro and in Jurkat T cells.  The sequences of human and rat CaMKKβ demonstrate 65% identity and 80% similarity with CaMKKα and 30-40% identity with CaMKI and CaMKIV themselves.  CaMKKβ is broadly distributed among rat tissues with highest levels in CaMKIV-expressing tissues such as brain, thymus, spleen, and testis.  In brain, CaMKKβ tracks more closely with CaMKIV than does CaMKKα.  Bacterially expressed CaMKKβ undergoes intramol. autophosphorylation, is regulated by Ca2+/CaM, and phosphorylates CaMKI and CaMKIV on Thr177 and Thr200, resp.  CaMKKβ activates both CaMKI and CaMKIV when coexpressed in Jurkat T cells as judged by phosphorylated cAMP response element-binding protein-dependent reporter gene expression.  CaMKKβ activity is enhanced by elevation of intracellular Ca2+, although substantial activity is obsd. at the resting Ca2+ concn.  The strict Ca2+ requirement of CaMKIV-dependent phosphorylation of cAMP response element-binding protein, is therefore controlled at the level of CaMKIV rather than CaMKK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokPLunLpqVSbVg90H21EOLACvtfcHk0ljkNr_bn5tLDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvVeltr4%253D&md5=dd9aad89eb2c0e06f02ce47233455d75</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.48.31880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.48.31880%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DMeans%26aufirst%3DR.%2BL.%26aulast%3DHuang%26aufirst%3DQ.-H.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DGoldstein%26aufirst%3DE.%2BG.%26aulast%3DSelbert%26aufirst%3DM.%2BA.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DComponents%2520of%2520a%2520calmodulin-dependent%2520protein%2520kinase%2520cascade.%2520Molecular%2520cloning%252C%2520functional%2520characterization%2520and%2520cellular%2520localization%2520of%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520beta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D31880%26epage%3D31889%26doi%3D10.1074%2Fjbc.273.48.31880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawley, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustard, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenguelli, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardie, D. G.</span></span> <span> </span><span class="NLM_article-title">Calmodulin-dependent protein kinase kinase-beta is an alternative upstream kinase for AMP-activated protein kinase</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2005.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cmet.2005.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=16054095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntlCruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=9-19&author=S.+A.+Hawleyauthor=D.+A.+Panauthor=K.+J.+Mustardauthor=L.+Rossauthor=J.+Bainauthor=A.+M.+Edelmanauthor=B.+G.+Frenguelliauthor=D.+G.+Hardie&title=Calmodulin-dependent+protein+kinase+kinase-beta+is+an+alternative+upstream+kinase+for+AMP-activated+protein+kinase&doi=10.1016%2Fj.cmet.2005.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Calmodulin-dependent protein kinase kinase-β is an alternative upstream kinase for AMP-activated protein kinase</span></div><div class="casAuthors">Hawley, Simon A.; Pan, David A.; Mustard, Kirsty J.; Ross, Louise; Bain, Jenny; Edelman, Arthur M.; Frenguelli, Bruno G.; Hardie, D. Grahame</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-19</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The AMP-activated protein kinase (AMPK) is a crit. regulator of energy balance at both the cellular and whole-body levels.  Two upstream kinases have been reported to activate AMPK in cell-free assays, i.e., the tumor suppressor LKB1 and calmodulin-dependent protein kinase kinase.  However, evidence that this is physiol. relevant currently only exists for LKB1.  We now report that there is a significant basal activity and phosphorylation of AMPK in LKB1-deficient cells that can be stimulated by Ca2+ ionophores, and studies using the CaMKK inhibitor STO-609 and isoform-specific siRNAs show that CaMKKβ is required for this effect.  CaMKKβ also activates AMPK much more rapidly than CaMKKα in cell-free assays.  K+-induced depolarization in rat cerebrocortical slices, which increases intracellular Ca2+ without disturbing cellular adenine nucleotide levels, activates AMPK, and this is blocked by STO-609.  Our results suggest a potential Ca2+-dependent neuroprotective pathway involving phosphorylation and activation of AMPK by CaMKKβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHr770gkRJILVg90H21EOLACvtfcHk0lhV5CZHZ_2MFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntlCruro%253D&md5=6de7ad1e166f6aefb027bf74bbdd9b03</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2005.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2005.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DHawley%26aufirst%3DS.%2BA.%26aulast%3DPan%26aufirst%3DD.%2BA.%26aulast%3DMustard%26aufirst%3DK.%2BJ.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DBain%26aufirst%3DJ.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26aulast%3DFrenguelli%26aufirst%3DB.%2BG.%26aulast%3DHardie%26aufirst%3DD.%2BG.%26atitle%3DCalmodulin-dependent%2520protein%2520kinase%2520kinase-beta%2520is%2520an%2520alternative%2520upstream%2520kinase%2520for%2520AMP-activated%2520protein%2520kinase%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D2%26spage%3D9%26epage%3D19%26doi%3D10.1016%2Fj.cmet.2005.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karacosta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azabdaftari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feliciano, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span> <span> </span><span class="NLM_article-title">A regulatory feedback loop between Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) and the androgen receptor in prostate cancer progression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">24832</span>– <span class="NLM_lpage">24843</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.370783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M112.370783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22654108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWgtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=24832-24843&author=L.+G.+Karacostaauthor=B.+A.+Fosterauthor=G.+Azabdaftariauthor=D.+M.+Felicianoauthor=A.+M.+Edelman&title=A+regulatory+feedback+loop+between+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+and+the+androgen+receptor+in+prostate+cancer+progression&doi=10.1074%2Fjbc.m112.370783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A Regulatory Feedback Loop Between Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) and the androgen receptor in Prostate Cancer Progression</span></div><div class="casAuthors">Karacosta, Loukia G.; Foster, Barbara A.; Azabdaftari, Gissou; Feliciano, David M.; Edelman, Arthur M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">24832-24843</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The androgen receptor (AR) plays a crit. role in prostate cancer (PCa) progression, however, the mol. mechanisms by which the AR regulates cell proliferation in androgen-dependent and castration-resistant PCa are incompletely understood.  We report that Ca2+/calmodulin-dependent kinase kinase 2 (CaMKK2) expression increases and becomes nuclear or perinuclear in advanced PCa.  In the TRAMP (transgenic adenocarcinoma of mouse prostate) model of PCa, CaMKK2 expression increases with PCa progression with many cells exhibiting nuclear staining.  CaMKK2 expression is higher in human castration-resistant tumor xenografts compared with androgen-responsive xenografts and is markedly higher in the AR-expressing, tumorigenic cell line LNCaP compared with cell lines that are AR-nonexpressing and/or nontumorigenic.  In LNCaP cells, dihydrotestosterone induced CaMKK2 mRNA and protein expression and translocation of CaMKK2 to the nucleus.  Conversely, androgen withdrawal suppressed CaMKK2 expression.  Knockdown of CaMKK2 expression by RNAi reduced LNCaP cell proliferation and increased percentages of cells in G1 phase, whereas correspondingly reducing percentages in S phase, of the cell cycle.  CaMKK2 knockdown reduced expression of the AR target gene prostate-specific antigen at both mRNA and protein levels, AR transcriptional activity driven by androgen responsive elements from the prostate-specific probasin gene promoter and levels of the AR-regulated cell cycle proteins, cyclin D1 and hyperphosphorylated Rb.  Our results suggest that in PCa progression, CaMKK2 and the AR are in a feedback loop in which CaMKK2 is induced by the AR to maintain AR activity, AR-dependent cell cycle control, and continued cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFq7oX7Qhp9bVg90H21EOLACvtfcHk0lhV5CZHZ_2MFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWgtbzM&md5=6335c6e09e17c5e511f4f2a8a379cea9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.370783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.370783%26sid%3Dliteratum%253Aachs%26aulast%3DKaracosta%26aufirst%3DL.%2BG.%26aulast%3DFoster%26aufirst%3DB.%2BA.%26aulast%3DAzabdaftari%26aufirst%3DG.%26aulast%3DFeliciano%26aufirst%3DD.%2BM.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26atitle%3DA%2520regulatory%2520feedback%2520loop%2520between%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520and%2520the%2520androgen%2520receptor%2520in%2520prostate%2520cancer%2520progression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D24832%26epage%3D24843%26doi%3D10.1074%2Fjbc.m112.370783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilganowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bissig, K.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldie, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, B.</span></span> <span> </span><span class="NLM_article-title">The camKK2/camKIV relay is an essential regulator of hepatic cancer</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1002/hep.27832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fhep.27832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25847065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KmsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=505-520&author=F.+Linauthor=K.+L.+Marceloauthor=K.+Rajapaksheauthor=C.+Coarfaauthor=A.+Deanauthor=N.+Wilganowskiauthor=H.+Robinsonauthor=E.+Sevickauthor=K.-D.+Bissigauthor=L.+C.+Goldieauthor=A.+R.+Meansauthor=B.+York&title=The+camKK2%2FcamKIV+relay+is+an+essential+regulator+of+hepatic+cancer&doi=10.1002%2Fhep.27832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The camKK2/camKIV relay is an essential regulator of hepatic cancer</span></div><div class="casAuthors">Lin, Fumin; Marcelo, Kathrina L.; Rajapakshe, Kimal; Coarfa, Cristian; Dean, Adam; Wilganowski, Nathaniel; Robinson, Holly; Sevick, Eva; Bissig, Karl-Dimiter; Goldie, Lauren C.; Means, Anthony R.; York, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">505-520</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatic cancer is one of the most lethal cancers worldwide.  Here, we report that the expression of Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) is significantly up-regulated in hepatocellular carcinoma (HCC) and neg. correlated with HCC patient survival.  The CaMKK2 protein is highly expressed in all eight hepatic cancer cell lines evaluated and is markedly up-regulated relative to normal primary hepatocytes.  Loss of CaMKK2 function is sufficient to inhibit liver cancer cell growth, and the growth defect resulting from loss of CaMKK2 can be rescued by ectopic expression of wild-type CaMKK2 but not by kinase-inactive mutants.  Cellular ablation of CaMKK2 using RNA interference yields a gene signature that correlates with improvement in HCC patient survival, and ablation or pharmacol. inhibition of CaMKK2 with STO-609 impairs tumorigenicity of liver cancer cells in vivo.  Moreover, CaMKK2 expression is up-regulated in a time-dependent manner in a carcinogen-induced HCC mouse model, and STO-609 treatment regresses hepatic tumor burden in this model.  Mechanistically, CaMKK2 signals through Ca2+/calmodulin-dependent protein kinase 4 (CaMKIV) to control liver cancer cell growth.  Further anal. revealed that CaMKK2 serves as a scaffold to assemble CaMKIV with key components of the mammalian target of rapamycin/ribosomal protein S6 kinase, 70 kDa, pathway and thereby stimulate protein synthesis through protein phosphorylation.  Conclusion: The CaMKK2/CaMKIV relay is an upstream regulator of the oncogenic mammalian target of rapamycin/ribosomal protein S6 kinase, 70 kDa, pathway, and the importance of this CaMKK2/CaMKIV axis in HCC growth is confirmed by the potent growth inhibitory effects of genetically or pharmacol. decreasing CaMKK2 activity; collectively, these findings suggest that CaMKK2 and CaMKIV may represent potential targets for hepatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsIuVll-cevbVg90H21EOLACvtfcHk0lhJDakGZ_se3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KmsLzK&md5=62e123eedf5f49c210aeb2d8edb695c3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fhep.27832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.27832%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DF.%26aulast%3DMarcelo%26aufirst%3DK.%2BL.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DDean%26aufirst%3DA.%26aulast%3DWilganowski%26aufirst%3DN.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DSevick%26aufirst%3DE.%26aulast%3DBissig%26aufirst%3DK.-D.%26aulast%3DGoldie%26aufirst%3DL.%2BC.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26aulast%3DYork%26aufirst%3DB.%26atitle%3DThe%2520camKK2%252FcamKIV%2520relay%2520is%2520an%2520essential%2520regulator%2520of%2520hepatic%2520cancer%26jtitle%3DHepatology%26date%3D2015%26volume%3D62%26spage%3D505%26epage%3D520%26doi%3D10.1002%2Fhep.27832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-L.</span></span> <span> </span><span class="NLM_article-title">CAMKK2, regulated by promoter methylation, is a prognostic marker in diffuse gliomas</span>. <i>CNS Neurosci. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">518</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1111/cns.12531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1111%2Fcns.12531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27012733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvVCrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=518-524&author=D.-M.+Liuauthor=H.-J.+Wangauthor=B.+Hanauthor=X.-Q.+Mengauthor=M.-H.+Chenauthor=D.-B.+Yangauthor=Y.+Sunauthor=Y.-L.+Liauthor=C.-L.+Jiang&title=CAMKK2%2C+regulated+by+promoter+methylation%2C+is+a+prognostic+marker+in+diffuse+gliomas&doi=10.1111%2Fcns.12531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">CAMKK2, Regulated by Promoter Methylation, is a Prognostic Marker in Diffuse Gliomas</span></div><div class="casAuthors">Liu, Da-Ming; Wang, Hong-Jun; Han, Bo; Meng, Xiang-Qi; Chen, Ming-Hui; Yang, Dong-Bo; Sun, Ying; Li, Yong-Li; Jiang, Chuan-Lu</div><div class="citationInfo"><span class="NLM_cas:title">CNS Neuroscience & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">518-524</span>CODEN:
                <span class="NLM_cas:coden">CNTNAB</span>;
        ISSN:<span class="NLM_cas:issn">1755-5930</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Aims : To explore the expression, methylation pattern, the prognostic value, and the biol. consequences of CAMKK2 in gliomas.  Methods : The expression and methylation pattern of CAMKK2 was inferred and validated from mRNA expression profile (N = 866) and methylation profile (N = 426) of glioma tissue samples, and independent samples were used for further validation by IHC and pyrosequencing.  To explore the function of CAMKK2 in gliomas, in vitro studies, colony formation assays and migration and invasion assays were performed.  Results : We found the upregulation of CAMKK2 in high-grade glioma samples was assocd. with promoter hypomethylation.  An elevated expression of CAMKK2 was assocd. with worse prognosis.  By in vitro assays, we demonstrated that CAMKK2 could promote cell migration, invasion, and proliferation.  Conclusions : The expression level of CAMKK2 could be regulated by promoter methylation.  CAMKK2 serves as a prognostic marker in gliomas and could be a potential therapeutic target in gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2555BynUjLVg90H21EOLACvtfcHk0lhJDakGZ_se3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvVCrurs%253D&md5=1d8eba304bd0184cd36d36aef7dd8c3c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fcns.12531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcns.12531%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DD.-M.%26aulast%3DWang%26aufirst%3DH.-J.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DMeng%26aufirst%3DX.-Q.%26aulast%3DChen%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DD.-B.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.-L.%26aulast%3DJiang%26aufirst%3DC.-L.%26atitle%3DCAMKK2%252C%2520regulated%2520by%2520promoter%2520methylation%252C%2520is%2520a%2520prognostic%2520marker%2520in%2520diffuse%2520gliomas%26jtitle%3DCNS%2520Neurosci.%2520Ther.%26date%3D2016%26volume%3D22%26spage%3D518%26epage%3D524%26doi%3D10.1111%2Fcns.12531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massie, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos-Montoya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecchini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarthur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adryan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleave, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennie, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, I. G.</span></span> <span> </span><span class="NLM_article-title">The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2733</span>, <span class="refDoi"> DOI: 10.1038/emboj.2011.158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Femboj.2011.158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21602788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlemtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=2719-2733&author=C.+E.+Massieauthor=A.+Lynchauthor=A.+Ramos-Montoyaauthor=J.+Borenauthor=R.+Starkauthor=L.+Fazliauthor=A.+Warrenauthor=H.+Scottauthor=B.+Madhuauthor=N.+Sharmaauthor=H.+Bonauthor=V.+Zecchiniauthor=D.-M.+Smithauthor=G.+M.+Denicolaauthor=N.+Mathewsauthor=M.+Osborneauthor=J.+Hadfieldauthor=S.+Macarthurauthor=B.+Adryanauthor=S.+K.+Lyonsauthor=K.+M.+Brindleauthor=J.+Griffithsauthor=M.+E.+Gleaveauthor=P.+S.+Rennieauthor=D.+E.+Nealauthor=I.+G.+Mills&title=The+androgen+receptor+fuels+prostate+cancer+by+regulating+central+metabolism+and+biosynthesis&doi=10.1038%2Femboj.2011.158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis</span></div><div class="casAuthors">Massie, Charles E.; Lynch, Andy; Ramos-Montoya, Antonio; Boren, Joan; Stark, Rory; Fazli, Ladan; Warren, Anne; Scott, Helen; Madhu, Basetti; Sharma, Naomi; Bon, Helene; Zecchini, Vinny; Smith, Donna-Michelle; DeNicola, Gina M.; Mathews, Nik; Osborne, Michelle; Hadfield, James; MacArthur, Stewart; Adryan, Boris; Lyons, Scott K.; Brindle, Kevin M.; Griffiths, John; Gleave, Martin E.; Rennie, Paul S.; Neal, David E.; Mills, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2719-2733</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is a key regulator of prostate growth and the principal drug target for the treatment of prostate cancer.  Previous studies have mapped AR targets and identified some candidates which may contribute to cancer progression, but did not characterize AR biol. in an integrated manner.  In this study, we took an interdisciplinary approach, integrating detailed genomic studies with metabolomic profiling and identify an anabolic transcriptional network involving AR as the core regulator.  Restricting flux through anabolic pathways is an attractive approach to deprive tumors of the building blocks needed to sustain tumor growth.  Therefore, we searched for targets of the AR that may contribute to these anabolic processes and could be amenable to therapeutic intervention by virtue of differential expression in prostate tumors.  This highlighted calcium/calmodulin-dependent protein kinase kinase 2, which we show is overexpressed in prostate cancer and regulates cancer cell growth via its unexpected role as a hormone-dependent modulator of anabolic metab.  In conclusion, it is possible to progress from transcriptional studies to a promising therapeutic target by taking an unbiased interdisciplinary approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraxaKmz8_gabVg90H21EOLACvtfcHk0ljtGtWPJ3-hVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlemtb4%253D&md5=a28d6643fb447b79372eeb8922960f2c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.158%26sid%3Dliteratum%253Aachs%26aulast%3DMassie%26aufirst%3DC.%2BE.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DRamos-Montoya%26aufirst%3DA.%26aulast%3DBoren%26aufirst%3DJ.%26aulast%3DStark%26aufirst%3DR.%26aulast%3DFazli%26aufirst%3DL.%26aulast%3DWarren%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DH.%26aulast%3DMadhu%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DBon%26aufirst%3DH.%26aulast%3DZecchini%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DD.-M.%26aulast%3DDenicola%26aufirst%3DG.%2BM.%26aulast%3DMathews%26aufirst%3DN.%26aulast%3DOsborne%26aufirst%3DM.%26aulast%3DHadfield%26aufirst%3DJ.%26aulast%3DMacarthur%26aufirst%3DS.%26aulast%3DAdryan%26aufirst%3DB.%26aulast%3DLyons%26aufirst%3DS.%2BK.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26aulast%3DGriffiths%26aufirst%3DJ.%26aulast%3DGleave%26aufirst%3DM.%2BE.%26aulast%3DRennie%26aufirst%3DP.%2BS.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMills%26aufirst%3DI.%2BG.%26atitle%3DThe%2520androgen%2520receptor%2520fuels%2520prostate%2520cancer%2520by%2520regulating%2520central%2520metabolism%2520and%2520biosynthesis%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D2719%26epage%3D2733%26doi%3D10.1038%2Femboj.2011.158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Mora, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaHair, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCubrey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, R. A.</span></span> <span> </span><span class="NLM_article-title">Calcium/calmodulin-dependent kinase I and calcium/calmodulin-dependent kinase kinase participate in the control of cell cycle progression in MCF-7 human breast cancer cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">5408</span>– <span class="NLM_lpage">5416</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-0271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-05-0271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=15958590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFels70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=5408-5416&author=O.+G.+Rodriguez-Moraauthor=M.+M.+LaHairauthor=J.+A.+McCubreyauthor=R.+A.+Franklin&title=Calcium%2Fcalmodulin-dependent+kinase+I+and+calcium%2Fcalmodulin-dependent+kinase+kinase+participate+in+the+control+of+cell+cycle+progression+in+MCF-7+human+breast+cancer+cells&doi=10.1158%2F0008-5472.can-05-0271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium/Calmodulin-Dependent Kinase I and Calcium/Calmodulin-Dependent Kinase Kinase Participate in the Control of Cell Cycle Progression in MCF-7 Human Breast Cancer Cells</span></div><div class="casAuthors">Rodriguez-Mora, Oswaldo G.; LaHair, Michelle M.; McCubrey, James A.; Franklin, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5408-5416</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Ca is universally required for cell growth and proliferation.  Calmodulin is the main intracellular receptor for Ca.  Although Ca and calmodulin are well known to be required for cell cycle regulation, the target pathways for their action remain poorly defined.  Potential targets include the Ca/calmodulin-dependent kinases (CaM-K).  The aim of this study was to det. the role of the CaM-Ks on cell proliferation and progress through the cell cycle in breast cancer cells.  CaM-KI inhibition with either KN-93 or specific interfering RNA (siRNA) caused an arrest in the cell cycle in the human breast cancer cell line, MCF-7.  This arrest occurred in the G1 phase of the cell cycle.  Supporting this finding, CaM-K inhibition using KN-93 also resulted in a redn. of cyclin D1 protein and pRb phosphorylation when cells were compared with control cultures.  Furthermore, inhibition of the upstream activator of CaM-KI, CaM-KK, using siRNA also resulted in cell cycle arrest.  In summary, CaM-KK and CaM-KI participate in the control of the G0-G1 restriction check point of the cell cycle in human breast cancer cells.  This arrest seems due to an inhibition in cyclin D1 synthesis and a redn. in pRb phosphorylation.  To the best of our knowledge, this is the first time that CaM-KK was reported to be involved in mammalian cell cycle regulation and that CaM-Ks are regulating breast cancer cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-cqYzjPCAnbVg90H21EOLACvtfcHk0ljtGtWPJ3-hVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFels70%253D&md5=8929eeac7e0401bcf6a8b6cf1fae1571</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-0271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-0271%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Mora%26aufirst%3DO.%2BG.%26aulast%3DLaHair%26aufirst%3DM.%2BM.%26aulast%3DMcCubrey%26aufirst%3DJ.%2BA.%26aulast%3DFranklin%26aufirst%3DR.%2BA.%26atitle%3DCalcium%252Fcalmodulin-dependent%2520kinase%2520I%2520and%2520calcium%252Fcalmodulin-dependent%2520kinase%2520kinase%2520participate%2520in%2520the%2520control%2520of%2520cell%2520cycle%2520progression%2520in%2520MCF-7%2520human%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D5408%26epage%3D5416%26doi%3D10.1158%2F0008-5472.can-05-0271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofude, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namiki, M.</span></span> <span> </span><span class="NLM_article-title">Down-regulation of calcium/calmodulin-dependent protein kinase kinase 2 by androgen deprivation induces castration-resistant prostate cancer</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1801</span>, <span class="refDoi"> DOI: 10.1002/pros.22533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fpros.22533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22549914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGqtLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=1789-1801&author=T.+Shimaauthor=A.+Mizokamiauthor=T.+Miyagiauthor=K.+Kawaiauthor=K.+Izumiauthor=M.+Kumakiauthor=M.+Ofudeauthor=J.+Zhangauthor=E.+T.+Kellerauthor=M.+Namiki&title=Down-regulation+of+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2+by+androgen+deprivation+induces+castration-resistant+prostate+cancer&doi=10.1002%2Fpros.22533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulation of calcium/calmodulin-dependent protein kinase kinase 2 by androgen deprivation induces castration-resistant prostate cancer</span></div><div class="casAuthors">Shima, Takashi; Mizokami, Atsushi; Miyagi, Toru; Kawai, Keiichi; Izumi, Kouji; Kumaki, Misako; Ofude, Mitsuo; Zhang, Jian; Keller, Evan T.; Namiki, Mikio</div><div class="citationInfo"><span class="NLM_cas:title">Prostate (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1789-1801</span>CODEN:
                <span class="NLM_cas:coden">PRSTDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-4137</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND : Conversion into androgen-hypersensitive state and adaptation to the low concn. of androgen during ADT cause relapse of prostate cancer (PCa).  It is important to identify differentially expressed genes between PCa and normal prostate tissues and to reveal the function of these genes that are involved in progression of PCa.  METHODS : We performed cDNA microarray anal. to identify differentially expressed genes, calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2).  Immunohistochem. anal. was conducted to investigate the relationship between the CAMKK2 expression level and prognosis.  The function of CAMKK2 was assessed by generating CAMKK2 overexpressed LNCaP cells and by knockdown of CAMKK2.  RESULTS : We identified CAMKK2 overexpressed six times in PCa more than normal prostate by cDNA microarray anal.  Immunohistochem. anal. of CAMKK2 protein showed that CAMKK2 protein was expressed more in PCa than normal tissue.  However, the expression in the high-grade PCa diminished.  Moreover, the narrowness of CAMKK2-pos. area before ADT was a poor prognostic factor.  Androgen-deprivation treatment from the medium in which LNCaP cells were cultured in the presence of 10 nM DHT repressed CAMKK2 expression.  CAMKK2 overexpressed LNCaP cells (LNCaP/GFP-CAMKK2) attenuated androgen-sensitivity.  Tumorigenesis of LNCaP/GFP-CAMKK2 cells in male SCID mice was decreased compared with control cells irresp. of castration.  Finally, knockdown of CAMKK2 mRNA in LNCaP cells induced androgen-hypersensitivity and stimulated LNCaP cell proliferation.  CONCLUSIONS : Induction of androgen-hypersensitivity after ADT may be involved in down-regulation of CAMKK2.  This result may provide new therapeutic approach to keep androgen-sensitivity of PCa after ADT.  Prostate 72:1789-1801, 2012.  © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTdlaoqDTAMrVg90H21EOLACvtfcHk0liwKen0IjUCLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGqtLzM&md5=cc116cc36ce22df02fb8f8da0e6dc737</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fpros.22533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22533%26sid%3Dliteratum%253Aachs%26aulast%3DShima%26aufirst%3DT.%26aulast%3DMizokami%26aufirst%3DA.%26aulast%3DMiyagi%26aufirst%3DT.%26aulast%3DKawai%26aufirst%3DK.%26aulast%3DIzumi%26aufirst%3DK.%26aulast%3DKumaki%26aufirst%3DM.%26aulast%3DOfude%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DE.%2BT.%26aulast%3DNamiki%26aufirst%3DM.%26atitle%3DDown-regulation%2520of%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520by%2520androgen%2520deprivation%2520induces%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProstate%26date%3D2012%26volume%3D72%26spage%3D1789%26epage%3D1801%26doi%3D10.1002%2Fpros.22533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subbannayya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbhuiya, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renuse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manju, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zameer, M. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brait, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srikumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaya Kumar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A.</span></span> <span> </span><span class="NLM_article-title">Calcium calmodulin dependent kinase kinase 2 - a novel therapeutic target for gastric adenocarcinoma</span>. <i>Canc. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">336</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.4161/15384047.2014.972264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.4161%2F15384047.2014.972264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25756516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWqt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=336-345&author=Y.+Subbannayyaauthor=N.+Syedauthor=M.+A.+Barbhuiyaauthor=R.+Rajaauthor=A.+Marimuthuauthor=N.+Sahasrabuddheauthor=S.+M.+Pintoauthor=S.+S.+Mandaauthor=S.+Renuseauthor=H.+Manjuauthor=M.+A.+L.+Zameerauthor=J.+Sharmaauthor=M.+Braitauthor=K.+Srikumarauthor=J.+C.+Roaauthor=M.+Vijaya+Kumarauthor=K.+V.+Kumarauthor=T.+K.+Prasadauthor=G.+Ramaswamyauthor=R.+V.+Kumarauthor=A.+Pandeyauthor=H.+Gowdaauthor=A.+Chatterjee&title=Calcium+calmodulin+dependent+kinase+kinase+2+-+a+novel+therapeutic+target+for+gastric+adenocarcinoma&doi=10.4161%2F15384047.2014.972264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium calmodulin dependent kinase kinase 2 - a novel therapeutic target for gastric adenocarcinoma</span></div><div class="casAuthors">Subbannayya, Yashwanth; Syed, Nazia; Barbhuiya, Mustafa A.; Raja, Remya; Marimuthu, Arivusudar; Sahasrabuddhe, Nandini; Pinto, Sneha M.; Manda, Srikanth Srinivas; Renuse, Santosh; Manju, Hc; Zameer, Mohammed Abdul Lateef; Sharma, Jyoti; Brait, Mariana; Srikumar, Kotteazeth; Roa, Juan Carlos; Vijaya Kumar, M.; Kumar, Kv veerendra; Prasad, Ts keshava; Ramaswamy, Girija; Kumar, Rekha Vijay; Pandey, Akhilesh; Gowda, Harsha; Chatterjee, Aditi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">336-345</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1555-8576</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Gastric cancer is one of the most common gastrointestinal malignancies and is assocd. with poor prognosis.  Exploring alterations in the proteomic landscape of gastric cancer is likely to provide potential biomarkers for early detection and mols. for targeted therapeutic intervention.  Using iTRAQ-based quant. proteomic anal., we identified 22 proteins that were overexpressed and 17 proteins that were downregulated in gastric tumor tissues as compared to the adjacent normal tissue.  Calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) was found to be 7-fold overexpressed in gastric tumor tissues.  Immunohistochem. labeling of tumor tissue microarrays for validation of CAMKK2 overexpression revealed that it was indeed overexpressed in 94% (92 of 98) of gastric cancer cases.  Silencing of CAMKK2 using siRNA significantly reduced cell proliferation, colony formation and invasion of gastric cancer cells.  Our results demonstrate that CAMKK2 signals in gastric cancer through AMPK activation and suggest that CAMKK2 could be a novel therapeutic target in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw67QaWTwi5rVg90H21EOLACvtfcHk0liwKen0IjUCLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWqt7fL&md5=6969f1236282780560fbde50b1fa98c5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2F15384047.2014.972264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384047.2014.972264%26sid%3Dliteratum%253Aachs%26aulast%3DSubbannayya%26aufirst%3DY.%26aulast%3DSyed%26aufirst%3DN.%26aulast%3DBarbhuiya%26aufirst%3DM.%2BA.%26aulast%3DRaja%26aufirst%3DR.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DSahasrabuddhe%26aufirst%3DN.%26aulast%3DPinto%26aufirst%3DS.%2BM.%26aulast%3DManda%26aufirst%3DS.%2BS.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DManju%26aufirst%3DH.%26aulast%3DZameer%26aufirst%3DM.%2BA.%2BL.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DBrait%26aufirst%3DM.%26aulast%3DSrikumar%26aufirst%3DK.%26aulast%3DRoa%26aufirst%3DJ.%2BC.%26aulast%3DVijaya%2BKumar%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DK.%2BV.%26aulast%3DPrasad%26aufirst%3DT.%2BK.%26aulast%3DRamaswamy%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DR.%2BV.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DGowda%26aufirst%3DH.%26aulast%3DChatterjee%26aufirst%3DA.%26atitle%3DCalcium%2520calmodulin%2520dependent%2520kinase%2520kinase%25202%2520-%2520a%2520novel%2520therapeutic%2520target%2520for%2520gastric%2520adenocarcinoma%26jtitle%3DCanc.%2520Biol.%2520Ther.%26date%3D2015%26volume%3D16%26spage%3D336%26epage%3D345%26doi%3D10.4161%2F15384047.2014.972264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soderling, T. R.</span></span> <span> </span><span class="NLM_article-title">Calcium promotes cell survival through CaM-K kinase activation of the protein-kinase-B pathway</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/25147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2F25147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=9859994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWnsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=1998&pages=584-587&author=S.+Yanoauthor=H.+Tokumitsuauthor=T.+R.+Soderling&title=Calcium+promotes+cell+survival+through+CaM-K+kinase+activation+of+the+protein-kinase-B+pathway&doi=10.1038%2F25147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Calcium promotes cell survival through CaM-K kinase activation of the protein-kinase-B pathway</span></div><div class="casAuthors">Yano, Shigetoshi; Tokumitsu, Hiroshi; Soderling, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">6711</span>),
    <span class="NLM_cas:pages">584-587</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The protection against apoptosis provided by growth factors in several cell lines is due to stimulation of the phosphatidylinositol-3-kinase (PI3K) pathway, which results in activation of protein kinase B (PKB; also known as c-Akt and Rac) and phosphorylation and sequestration to protein 14-3-3 of proapoptotic Bcl-2 family member BAD.  A modest increase in intracellular Ca2+ concn. also promotes survival of some cultured neurons through a pathway that requires calmodulin but is independent of PI3K and the MAP kinases.  Here, the authors report that Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK) activates PKB directly, resulting in phosphorylation of BAD on Ser-136 and the interaction of BAD with protein 14-3-3.  Serum withdrawal induced a 3- to 4-fld increase in cell death of NG108 neuroblastoma cells, and this apoptosis was largely blocked by increasing the intracellular Ca2+ concn. with NMDA (N-methyl-D-aspartate) or KCl or by transfection with constitutively active CaM-KK.  The effect of NMDA on cell survival was blocked by transfection with dominant-neg. forms of CaM-KK or PKB.  These results identify a Ca2+-triggered signaling cascade in which CaM-KK activates PKB, which in turn phosphorylates BAD and protects cells from apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPtIJx03Rrk7Vg90H21EOLACvtfcHk0liwKen0IjUCLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWnsLo%253D&md5=a93a337e6ca963a4607fc294c1f7c5a8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F25147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F25147%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DS.%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DSoderling%26aufirst%3DT.%2BR.%26atitle%3DCalcium%2520promotes%2520cell%2520survival%2520through%2520CaM-K%2520kinase%2520activation%2520of%2520the%2520protein-kinase-B%2520pathway%26jtitle%3DNature%26date%3D1998%26volume%3D396%26spage%3D584%26epage%3D587%26doi%3D10.1038%2F25147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spengler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zibrova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, R.</span></span> <span> </span><span class="NLM_article-title">Protein kinase A negatively regulates VEGF-induced AMPK activation by phosphorylating CaMKK2 at serine 495</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">3453</span>– <span class="NLM_lpage">3469</span>, <span class="refDoi"> DOI: 10.1042/bcj20200555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1042%2FBCJ20200555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32869834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB38bivVOguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2020&pages=3453-3469&author=K.+Spenglerauthor=D.+Zibrovaauthor=A.+Woodsauthor=C.+G.+Langendorfauthor=J.+W.+Scottauthor=D.+Carlingauthor=R.+Heller&title=Protein+kinase+A+negatively+regulates+VEGF-induced+AMPK+activation+by+phosphorylating+CaMKK2+at+serine+495&doi=10.1042%2Fbcj20200555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase A negatively regulates VEGF-induced AMPK activation by phosphorylating CaMKK2 at serine 495</span></div><div class="casAuthors">Spengler Katrin; Zibrova Darya; Heller Regine; Woods Angela; Carling David; Langendorf Christopher G; Scott John W; Scott John W; Scott John W</div><div class="citationInfo"><span class="NLM_cas:title">The Biochemical journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3453-3469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activation of AMP-activated protein kinase (AMPK) in endothelial cells by vascular endothelial growth factor (VEGF) via the Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) represents a pro-angiogenic pathway, whose regulation and function is incompletely understood.  This study investigates whether the VEGF/AMPK pathway is regulated by cAMP-mediated signalling.  We show that cAMP elevation in endothelial cells by forskolin, an activator of the adenylate cyclase, and/or 3-isobutyl-1-methylxanthine (IBMX), an inhibitor of phosphodiesterases, triggers protein kinase A (PKA)-mediated phosphorylation of CaMKK2 (serine residues S495, S511) and AMPK (S487).  Phosphorylation of CaMKK2 by PKA led to an inhibition of its activity as measured in CaMKK2 immunoprecipitates of forskolin/IBMX-treated cells.  This inhibition was linked to phosphorylation of S495, since it was not seen in cells expressing a non-phosphorylatable CaMKK2 S495C mutant.  Phosphorylation of S511 alone in these cells was not able to inhibit CaMKK2 activity.  Moreover, phosphorylation of AMPK at S487 was not sufficient to inhibit VEGF-induced AMPK activation in cells, in which PKA-mediated CaMKK2 inhibition was prevented by expression of the CaMKK2 S495C mutant. cAMP elevation in endothelial cells reduced basal and VEGF-induced acetyl-CoA carboxylase (ACC) phosphorylation at S79 even if AMPK was not inhibited.  Together, this study reveals a novel regulatory mechanism of VEGF-induced AMPK activation by cAMP/PKA, which may explain, in part, inhibitory effects of PKA on angiogenic sprouting and play a role in balancing pro- and anti-angiogenic mechanisms in order to ensure functional angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJPdtvc19VG-IHSmpohpd0fW6udTcc2eaSsfdqaYPsI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38bivVOguw%253D%253D&md5=8f29b7bf157e45a59cafaea00f8f6914</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2FBCJ20200555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20200555%26sid%3Dliteratum%253Aachs%26aulast%3DSpengler%26aufirst%3DK.%26aulast%3DZibrova%26aufirst%3DD.%26aulast%3DWoods%26aufirst%3DA.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DHeller%26aufirst%3DR.%26atitle%3DProtein%2520kinase%2520A%2520negatively%2520regulates%2520VEGF-induced%2520AMPK%2520activation%2520by%2520phosphorylating%2520CaMKK2%2520at%2520serine%2520495%26jtitle%3DBiochem.%2520J.%26date%3D2020%26volume%3D477%26spage%3D3453%26epage%3D3469%26doi%3D10.1042%2Fbcj20200555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gocher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azabdaftari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Euscher, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karacosta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franke, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, A. M.</span></span> <span> </span><span class="NLM_article-title">Akt activation by Ca(2+)/calmodulin-dependent protein kinase kinase 2 (CaMKK2) in ovarian cancer cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">14188</span>– <span class="NLM_lpage">14204</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.778464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M117.778464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=28634229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2iurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=14188-14204&author=A.+M.+Gocherauthor=G.+Azabdaftariauthor=L.+M.+Euscherauthor=S.+Daiauthor=L.+G.+Karacostaauthor=T.+F.+Frankeauthor=A.+M.+Edelman&title=Akt+activation+by+Ca%282%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+in+ovarian+cancer+cells&doi=10.1074%2Fjbc.m117.778464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Akt activation by Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) in ovarian cancer cells</span></div><div class="casAuthors">Gocher, Angela M.; Azabdaftari, Gissou; Euscher, Lindsey M.; Dai, Shuhang; Karacosta, Loukia G.; Franke, Thomas F.; Edelman, Arthur M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">14188-14204</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hyperactivation of Akt is assocd. with oncogenic changes in the growth, survival, and chemoresistance of cancer cells.  The PI3K/phosphoinositide-dependent kinase (PDK) 1 pathway represents the canonical mechanism for phosphorylation of Akt at its primary activation site, Thr-308.  We obsd. that Ca2+/calmodulin (CaM)-dependent protein kinase kinase 2 (β) (CaMKK2) is highly expressed in high-grade serous ovarian cancer, and we investigated its role in Akt activation in ovarian cancer (OVCa) cell lines (OVCAR-3, SKOV-3, and Caov-3).  Knockdown or pharmacol. inhibition of CaMKK2 produced phenotypes expected of Akt inhibition, including redns. in cell growth and cell viability and in the regulation of Akt downstream targets involved in G1/S transition and apoptosis.  CaMKK2 knockdown or inhibition decreased Akt phosphorylation at Thr-308 and Ser-473 to extents similar to those of PDK1 knockdown or PI3K inhibition.  Combined CaMKK2 and PDK1 knockdown or CaMKK and PI3K inhibition, resp., produced additive effects on p-Akt and cell growth, consistent with direct Akt phosphorylation by CaMKK2.  This conclusion was supported by the absence of effects of CaMKK2 knockdown/inhibition on alternative means of activating Akt via p-Akt Thr-450, p-PDK1 Ser-241, or p-IRS1 Ser-636/639.  Recombinant CaMKK2 directly activated recombinant Akt by phosphorylation at Thr-308 in a Ca2+/CaM-dependent manner.  In OVCa cells, p-Akt Thr-308 was significantly inhibited by intracellular Ca2+i chelation or CaM inhibition.  Ionomycin-induced Ca2+ influx promoted p-Akt, an effect blocked by PDK1, and/or CaMKK2, siRNAs, and by PI3K and/or CaMKK inhibitors.  CaMKK2 knockdown potentiated the effects of the chemotherapeutic drugs carboplatin and PX-866 to reduce proliferation and survival of OVCa cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7fTocaUYvgLVg90H21EOLACvtfcHk0ljm_sSres-BBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2iurvE&md5=6926c83bd4825d1818c2fee251b0ac3c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.778464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.778464%26sid%3Dliteratum%253Aachs%26aulast%3DGocher%26aufirst%3DA.%2BM.%26aulast%3DAzabdaftari%26aufirst%3DG.%26aulast%3DEuscher%26aufirst%3DL.%2BM.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DKaracosta%26aufirst%3DL.%2BG.%26aulast%3DFranke%26aufirst%3DT.%2BF.%26aulast%3DEdelman%26aufirst%3DA.%2BM.%26atitle%3DAkt%2520activation%2520by%2520Ca%25282%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520in%2520ovarian%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D14188%26epage%3D14204%26doi%3D10.1074%2Fjbc.m117.778464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davare, M. A.</span></span> <span> </span><span class="NLM_article-title">ERK activation and cell growth require CaM kinases in MCF-7 breast cancer cells</span>. <i>Mol. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1007/s11010-009-0252-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2Fs11010-009-0252-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=19763792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2010&pages=155-171&author=J.+M.+Schmittauthor=E.+Abellauthor=A.+Wagnerauthor=M.+A.+Davare&title=ERK+activation+and+cell+growth+require+CaM+kinases+in+MCF-7+breast+cancer+cells&doi=10.1007%2Fs11010-009-0252-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">ERK activation and cell growth require CaM kinases in MCF-7 breast cancer cells</span></div><div class="casAuthors">Schmitt, John M.; Abell, Ellen; Wagner, Andrea; Davare, Monika A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">1&2</span>),
    <span class="NLM_cas:pages">155-171</span>CODEN:
                <span class="NLM_cas:coden">MCBIB8</span>;
        ISSN:<span class="NLM_cas:issn">0300-8177</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Previous studies on MCF-7 breast cancer cells have shown that the G-protein coupled receptor (GPCR) agonist carbachol increases intracellular calcium levels and the activation of extracellular signal-regulated kinase (ERK).  Calcium and calmodulin regulate the calcium/calmodulin-dependent kinase (CaM kinase) family of proteins that have been proposed to regulate ERK and gene transcription.  Our results suggest that both estrogen (E2) and carbachol treatment of MCF-7 breast cancer cells trigger phosphorylation of ERK1/2 and the transcription factor Elk-1.  Carbachol and estrogen triggered nearly a 4-6-fold increase in MCF-7 cell proliferation by 96 h, resp.  Carbachol-stimulated ERK activation and cell growth was completely blocked by the Muscarinic M3-subtype GPCR inhibitor, 4-DAMP, and siRNA against the M3-subtype GPCR.  Interestingly, blockade of CaM KK with the selective inhibitor STO-609 prevented carbachol activation CaM KI, ERK, Elk-1, and cell growth.  Consistent with these observations, knockdown of CaM KKα and CaM KIγ with shRNA-contg. plasmids blocked ERK activation by carbachol.  In addn., Elk-1 phosphorylation and luciferase activity in response to carbachol treatment was also dependent upon CaM kinases and was inhibited by U 0126, STO-609, and siRNA knockdown of CaM kinases and ERK2.  Finally, blockade of either CaM KK (with STO-609) or ERK (with U 0126) activities resulted in the inhibition of carbachol- and estrogen-mediated cyclin D1 expression and MCF-7 cell growth.  Taken together, our results suggest that carbachol treatment of MCF-7 cells activates CaM KI, ERK, the transcription factor Elk-1, cyclin D1, and cell growth through CaM KK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMIgFw4WiYhbVg90H21EOLACvtfcHk0ljm_sSres-BBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCksQ%253D%253D&md5=e6b4fa78e7d8148c99b6a1a715fbca56</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11010-009-0252-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11010-009-0252-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DJ.%2BM.%26aulast%3DAbell%26aufirst%3DE.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DDavare%26aufirst%3DM.%2BA.%26atitle%3DERK%2520activation%2520and%2520cell%2520growth%2520require%2520CaM%2520kinases%2520in%2520MCF-7%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Cell.%2520Biochem.%26date%3D2010%26volume%3D335%26spage%3D155%26epage%3D171%26doi%3D10.1007%2Fs11010-009-0252-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">CaM kinase kinase beta-mediated activation of the growth regulatory kinase AMPK is required for androgen-dependent migration of prostate cancer cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">528</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-2581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-10-2581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21098087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsFKltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=528-537&author=D.+E.+Frigoauthor=M.+K.+Howeauthor=B.+M.+Wittmannauthor=A.+M.+Brunnerauthor=I.+Cushmanauthor=Q.+Wangauthor=M.+Brownauthor=A.+R.+Meansauthor=D.+P.+McDonnell&title=CaM+kinase+kinase+beta-mediated+activation+of+the+growth+regulatory+kinase+AMPK+is+required+for+androgen-dependent+migration+of+prostate+cancer+cells&doi=10.1158%2F0008-5472.can-10-2581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CaM Kinase Kinase β-Mediated Activation of the Growth Regulatory Kinase AMPK Is Required for Androgen-Dependent Migration of Prostate Cancer Cells</span></div><div class="casAuthors">Frigo, Daniel E.; Howe, Matthew K.; Wittmann, Bryan M.; Brunner, Abigail M.; Cushman, Ian; Wang, Qianben; Brown, Myles; Means, Anthony R.; McDonnell, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">528-537</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">While patients with advanced prostate cancer initially respond favorably to androgen ablation therapy, most experience a relapse of the disease within 1-2 years.  Although hormone-refractory disease is unresponsive to androgen-deprivation, androgen receptor (AR)-regulated signaling pathways remain active and are necessary for cancer progression.  Thus, both AR itself and the processes downstream of the receptor remain viable targets for therapeutic intervention.  Microarray anal. of multiple clin. cohorts showed that the serine/threonine kinase Ca2+/calmodulin-dependent protein kinase kinase β (CaMKKβ) is both highly expressed in the prostate and further elevated in prostate cancers.  Using cellular models of prostate cancer, we have detd. that androgens (a) directly increase the expression of a CaMKKβ splice variant and (b) increase functional CaMKKβ protein levels as detd. by the phosphorylation of both CaMKI and AMP-activated protein kinase (AMPK), two of CaMKKβ's primary substrates.  Importantly, inhibition of the CaMKKβ-AMPK, but not CaMKI, signaling axis in prostate cancer cells by pharmacol. inhibitors or siRNA-mediated knockdown blocks androgen-mediated migration and invasion.  Conversely, overexpression of CaMKKβ alone leads to both increased AMPK phosphorylation and cell migration.  Given the key roles of CaMKKβ and AMPK in the biol. of prostate cancer cells, we propose that these enzymes are potential therapeutic targets in prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkTeJeY6bKQrVg90H21EOLACvtfcHk0ljNMhgzUx1Nuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsFKltw%253D%253D&md5=d621e7ca9bd660cc0bb2c2a332adf4ca</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2581%26sid%3Dliteratum%253Aachs%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26aulast%3DHowe%26aufirst%3DM.%2BK.%26aulast%3DWittmann%26aufirst%3DB.%2BM.%26aulast%3DBrunner%26aufirst%3DA.%2BM.%26aulast%3DCushman%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DCaM%2520kinase%2520kinase%2520beta-mediated%2520activation%2520of%2520the%2520growth%2520regulatory%2520kinase%2520AMPK%2520is%2520required%2520for%2520androgen-dependent%2520migration%2520of%2520prostate%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D528%26epage%3D537%26doi%3D10.1158%2F0008-5472.can-10-2581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.-d.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.-x.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.-s.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.-n.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W.-d.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-224 and its target CAMKK2 synergistically influence tumor progression and patient prognosis in prostate cancer</span>. <i>Tumour Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1991</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2805-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2Fs13277-014-2805-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25394900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC2M3nvFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1983-1991&author=H.+Fuauthor=H.-c.+Heauthor=Z.-d.+Hanauthor=Y.-p.+Wanauthor=H.-w.+Luoauthor=Y.-q.+Huangauthor=C.+Caiauthor=Y.-x.+Liangauthor=Q.-s.+Daiauthor=F.-n.+Jiangauthor=W.-d.+Zhong&title=MicroRNA-224+and+its+target+CAMKK2+synergistically+influence+tumor+progression+and+patient+prognosis+in+prostate+cancer&doi=10.1007%2Fs13277-014-2805-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-224 and its target CAMKK2 synergistically influence tumor progression and patient prognosis in prostate cancer</span></div><div class="casAuthors">Fu Hao; He Hui-chan; Han Zhao-dong; Wan Yue-ping; Luo Hong-wei; Huang Ya-qiang; Cai Chao; Liang Yu-xiang; Dai Qi-shan; Jiang Fu-neng; Zhong Wei-de</div><div class="citationInfo"><span class="NLM_cas:title">Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1983-91</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We previously demonstrated that microRNA (miR)-224 expression was significantly reduced in human prostate cancer (PCa) tissues and predicted unfavorable prognosis in patients.  However, the underlying mechanisms of miR-224 have not been fully elucidated.  In this study, calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) was identified as a target gene of miR-224.  Then, we found that enforced expression of miR-224 could suppress PCa cell proliferation and cell cycle by regulating the expression of CAMKK2 in vitro.  In addition, the expression levels of miR-224 in PCa tissues were negatively correlated with those of CAMKK2 mRNA significantly (Spearman's correlation: r = -0.66, P = 0.004).  Moreover, combined low miR-224 expression and high CAMKK2 expression (miR-224-low/CAMKK2-high) was closely correlated with advanced clinical stage (P = 0.028).  Furthermore, PCa patients with miR-224-low/CAMKK2-high expression more frequently had shorter overall survival than those in groups with other expression patterns of two molecules.  In conclusion, our data offer the convincing evidence that miR-224 and its target gene CAMKK2 may synergistically contribute to the malignant progression of PCa.  Combined detection of miR-224 and CAMKK2 expressions represents an efficient predictor of patient prognosis and may be a novel marker which can provide additional prognostic information in PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyfAqmJ0zmKxwLZxei1HN8fW6udTcc2eZ-oCCYk_4GsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3nvFertQ%253D%253D&md5=8794d11ca3f885f8ccdbd45681caa965</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2805-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2805-0%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.-c.%26aulast%3DHan%26aufirst%3DZ.-d.%26aulast%3DWan%26aufirst%3DY.-p.%26aulast%3DLuo%26aufirst%3DH.-w.%26aulast%3DHuang%26aufirst%3DY.-q.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DY.-x.%26aulast%3DDai%26aufirst%3DQ.-s.%26aulast%3DJiang%26aufirst%3DF.-n.%26aulast%3DZhong%26aufirst%3DW.-d.%26atitle%3DMicroRNA-224%2520and%2520its%2520target%2520CAMKK2%2520synergistically%2520influence%2520tumor%2520progression%2520and%2520patient%2520prognosis%2520in%2520prostate%2520cancer%26jtitle%3DTumour%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D1983%26epage%3D1991%26doi%3D10.1007%2Fs13277-014-2805-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsouko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkenfeld, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakili, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulliam, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katreddy, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span> <span> </span><span class="NLM_article-title">GLUT12 promotes prostate cancer cell growth and is regulated by androgens and CaMKK2 signaling</span>. <i>Endocr. Relat. Canc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1530/erc-17-0051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1530%2FERC-17-0051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29431615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCksbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=453-469&author=M.+A.+Whiteauthor=E.+Tsoukoauthor=C.+Linauthor=K.+Rajapaksheauthor=J.+M.+Spencerauthor=S.+R.+Wilkenfeldauthor=S.+S.+Vakiliauthor=T.+L.+Pulliamauthor=D.+Awadauthor=F.+Nikolosauthor=R.+R.+Katreddyauthor=B.+A.+Kaipparettuauthor=A.+Sreekumarauthor=X.+Zhangauthor=E.+Cheungauthor=C.+Coarfaauthor=D.+E.+Frigo&title=GLUT12+promotes+prostate+cancer+cell+growth+and+is+regulated+by+androgens+and+CaMKK2+signaling&doi=10.1530%2Ferc-17-0051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">GLUT12 promotes prostate cancer cell growth and is regulated by androgens and CaMKK2 signaling</span></div><div class="casAuthors">White, Mark A.; Tsouko, Efrosini; Lin, Chenchu; Rajapakshe, Kimal; Spencer, Jeffrey M.; Wilkenfeld, Sandi R.; Vakili, Sheiva S.; Pulliam, Thomas L.; Awad, Dominik; Nikolos, Fotis; Katreddy, Rajasekhara Reddy; Kaipparettu, Benny Abraham; Sreekumar, Arun; Zhang, Xiaoliu; Cheung, Edwin; Coarfa, Cristian; Frigo, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine-Related Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-469</span>CODEN:
                <span class="NLM_cas:coden">ERCAE9</span>;
        ISSN:<span class="NLM_cas:issn">1351-0088</span>.
    
            (<span class="NLM_cas:orgname">BioScientifica Ltd.</span>)
        </div><div class="casAbstract">Despite altered metab. being an accepted hallmark of cancer, it is still not completely understood which signaling pathways regulate these processes.  Given the central role of androgen receptor (AR) signaling in prostate cancer, we hypothesized that AR could promote prostate cancer cell growth in part through increasing glucose uptake via the expression of distinct glucose transporters.  Here, we detd. that AR directly increased the expression of SLC2A12, the gene that encodes the glucose transporter GLUT12.  In support of these findings, gene signatures of AR activity correlated with SLC2A12 expression in multiple clin. cohorts.  Functionally, GLUT12 was required for maximal androgen mediated glucose uptake and cell growth in LNCaP and VCaP cells.  Knockdown of GLUT12 also decreased the growth of C4-2, 22Rv1 and AR-neg. PC-3 cells.  This latter observation corresponded with a significant redn. in glucose uptake, indicating that addnl. signaling mechanisms could augment GLUT12 function in an AR-independent manner.  Interestingly, GLUT12 trafficking to the plasma membrane was modulated by calcium/ calmodulin-dependent protein kinase kinase 2 (CaMKK2)-5'-AMP-activated protein kinase (AMPK) signaling, a pathway we previously demonstrated to be a downstream effector of AR.  Inhibition of CaMKK2-AMPK signaling decreased GLUT12 translocation to the plasma membrane by inhibiting the phosphorylation of TBC1D4, a known regulator of glucose transport.  Further, AR increased TBC1D4 expression.  Correspondingly, expression of TBC1D4 correlated with AR activity in prostate cancer patient samples.  Taken together, these data demonstrate that prostate cancer cells can increase the functional levels of GLUT12 through multiple mechanisms to promote glucose uptake and subsequent cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqvR94mNVUB7Vg90H21EOLACvtfcHk0ljNMhgzUx1Nuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCksbjK&md5=ad31763ec66f56ea721a4d5887249004</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1530%2FERC-17-0051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-17-0051%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DM.%2BA.%26aulast%3DTsouko%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DSpencer%26aufirst%3DJ.%2BM.%26aulast%3DWilkenfeld%26aufirst%3DS.%2BR.%26aulast%3DVakili%26aufirst%3DS.%2BS.%26aulast%3DPulliam%26aufirst%3DT.%2BL.%26aulast%3DAwad%26aufirst%3DD.%26aulast%3DNikolos%26aufirst%3DF.%26aulast%3DKatreddy%26aufirst%3DR.%2BR.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DSreekumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheung%26aufirst%3DE.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26atitle%3DGLUT12%2520promotes%2520prostate%2520cancer%2520cell%2520growth%2520and%2520is%2520regulated%2520by%2520androgens%2520and%2520CaMKK2%2520signaling%26jtitle%3DEndocr.%2520Relat.%2520Canc.%26date%3D2018%26volume%3D25%26spage%3D453%26epage%3D469%26doi%3D10.1530%2Ferc-17-0051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komorizono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanbara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saibara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okanoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Japan Nash
Study Group, M. o. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welfare of, J.</span></span> <span> </span><span class="NLM_article-title">Characteristics of patients with nonalcoholic steatohepatitis who develop hepatocellular carcinoma</span>. <i>Clin. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.cgh.2011.01.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cgh.2011.01.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21320639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvls1ylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=428-433&author=K.+Yasuiauthor=E.+Hashimotoauthor=Y.+Komorizonoauthor=K.+Koikeauthor=S.+Ariiauthor=Y.+Imaiauthor=T.+Shimaauthor=Y.+Kanbaraauthor=T.+Saibaraauthor=T.+Moriauthor=S.+Kawataauthor=H.+Utoauthor=S.+Takamiauthor=Y.+Sumidaauthor=T.+Takamuraauthor=M.+Kawanakaauthor=T.+Okanoueauthor=M.+o.+H.+L.+Japan+Nash%0AStudy+Groupauthor=J.+Welfare+of&title=Characteristics+of+patients+with+nonalcoholic+steatohepatitis+who+develop+hepatocellular+carcinoma&doi=10.1016%2Fj.cgh.2011.01.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Characteristics of patients with nonalcoholic steatohepatitis who develop hepatocellular carcinoma</span></div><div class="casAuthors">Yasui Kohichiroh; Hashimoto Etsuko; Komorizono Yasuji; Koike Kazuhiko; Arii Shigeki; Imai Yasuharu; Shima Toshihide; Kanbara Yoshihiro; Saibara Toshiji; Mori Takahiro; Kawata Sumio; Uto Hirofumi; Takami Shiro; Sumida Yoshio; Takamura Toshinari; Kawanaka Miwa; Okanoue Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical gastroenterology and hepatology : the official clinical practice journal of the American Gastroenterological Association</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">428-33; quiz e50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  Nonalcoholic steatohepatitis (NASH) can progress to hepatocellular carcinoma (HCC).  We aimed to characterize the clinical features of NASH patients with HCC.  METHODS:  In a cross-sectional multicenter study in Japan, we examined 87 patients (median age, 72 years; 62% male) with histologically proven NASH who developed HCC.  The clinical data were collected at the time HCC was diagnosed.  RESULTS:  Obesity (body mass index ≥25 kg/m(2)), diabetes, dyslipidemia, and hypertension were present in 54 (62%), 51 (59%), 24 (28%), and 47 (55%) patients, respectively.  In nontumor liver tissues, the degree of fibrosis was stage 1 in 10 patients (11%), stage 2 in 15 (17%), stage 3 in 18 (21%), and stage 4 (ie, liver cirrhosis) in 44 (51%).  The prevalence of cirrhosis was significantly lower among male patients (21 of 54, 39%) compared with female patients (23 of 33, 70%) (P = .008).  CONCLUSIONS:  Most patients with NASH who develop HCC are men; the patients have high rates of obesity, diabetes, and hypertension.  Male patients appear to develop HCC at a less advanced stage of liver fibrosis than female patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQPAIlg7wMzs0kUaC_kzhvyfW6udTcc2eaKozNdMUOrg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvls1ylsA%253D%253D&md5=d86986d6edede3d8d3eb97bfd2e6579b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cgh.2011.01.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cgh.2011.01.023%26sid%3Dliteratum%253Aachs%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DE.%26aulast%3DKomorizono%26aufirst%3DY.%26aulast%3DKoike%26aufirst%3DK.%26aulast%3DArii%26aufirst%3DS.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DShima%26aufirst%3DT.%26aulast%3DKanbara%26aufirst%3DY.%26aulast%3DSaibara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DKawata%26aufirst%3DS.%26aulast%3DUto%26aufirst%3DH.%26aulast%3DTakami%26aufirst%3DS.%26aulast%3DSumida%26aufirst%3DY.%26aulast%3DTakamura%26aufirst%3DT.%26aulast%3DKawanaka%26aufirst%3DM.%26aulast%3DOkanoue%26aufirst%3DT.%26aulast%3DJapan%2BNash%2BStudy%2BGroup%26aufirst%3DM.%2Bo.%2BH.%2BL.%26aulast%3DWelfare%2Bof%26aufirst%3DJ.%26atitle%3DCharacteristics%2520of%2520patients%2520with%2520nonalcoholic%2520steatohepatitis%2520who%2520develop%2520hepatocellular%2520carcinoma%26jtitle%3DClin.%2520Gastroenterol.%2520Hepatol.%26date%3D2011%26volume%3D9%26spage%3D428%26epage%3D433%26doi%3D10.1016%2Fj.cgh.2011.01.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouso, K.</span></span> <span> </span><span class="NLM_article-title">Nonalcoholic steatohepatitis-associated hepatocellular carcinoma: our case series and literature review</span>. <i>World J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1436</span>– <span class="NLM_lpage">1441</span>, <span class="refDoi"> DOI: 10.3748/wjg.v16.i12.1436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3748%2Fwjg.v16.i12.1436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20333782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1Crs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1436-1441&author=Y.+Takumaauthor=K.+Nouso&title=Nonalcoholic+steatohepatitis-associated+hepatocellular+carcinoma%3A+our+case+series+and+literature+review&doi=10.3748%2Fwjg.v16.i12.1436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Nonalcoholic steatohepatitis-associated hepatocellular carcinoma: our case series and literature review</span></div><div class="casAuthors">Takuma, Yoshitaka; Nouso, Kazuhiro</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1436-1441</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Recently, nonalcoholic steatohepatitis (NASH) has been considered to be another cause of liver cirrhosis and hepatocellular carcinoma (HCC).  The natural history and prognosis of NASH are controversial.  Accordingly, we assessed the clinicopathol. features of NASH-assocd. HCC in our experience and reviewed the literature of NASH-assocd. HCC.  We experienced 11 patients with NASH-assocd. HCC (6 male, 5 female; mean age 73.8 ± 4.9 years) who received curative treatments.  Most (91%) patients had been diagnosed with obesity, diabetes, hypertension, or dyslipidemia.  Seven patients (64%) also had a non-cirrhotic liver.  The recurrence-free survival rates at 1, 3 and 5 years were 72%, 60%, and 60%.  We also summarized and reviewed 94 cases of NASH-assocd. HCC which were reported in the literature (64 male; mean age 66 years).  The majority of patients (68%) were obese, 66% of patients had diabetes, and 24% had dyslipidemia.  Furthermore, 26% of the HCCs arose from the non-cirrhotic liver.  In conclusion, patients with non-cirrhotic NASH may be a high-risk group for HCC, and regular surveillance for HCC is necessary in non-cirrhotic NASH patients as well as cirrhotic patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr78pJsnUvf07Vg90H21EOLACvtfcHk0lhX3OKkiDhA8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1Crs7Y%253D&md5=0654b46d6f47aee1c673c6ae8663020f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v16.i12.1436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v16.i12.1436%26sid%3Dliteratum%253Aachs%26aulast%3DTakuma%26aufirst%3DY.%26aulast%3DNouso%26aufirst%3DK.%26atitle%3DNonalcoholic%2520steatohepatitis-associated%2520hepatocellular%2520carcinoma%253A%2520our%2520case%2520series%2520and%2520literature%2520review%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2010%26volume%3D16%26spage%3D1436%26epage%3D1441%26doi%3D10.3748%2Fwjg.v16.i12.1436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">York, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gooden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maity, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putluri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of CaMKK2 with the selective antagonist STO-609 regresses NAFLD</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11793</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12139-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-017-12139-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=28924233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC1cbpvFeqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11793&author=B.+Yorkauthor=F.+Liauthor=F.+Linauthor=K.+L.+Marceloauthor=J.+Maoauthor=A.+Deanauthor=N.+Gonzalesauthor=D.+Goodenauthor=S.+Maityauthor=C.+Coarfaauthor=N.+Putluriauthor=A.+R.+Means&title=Pharmacological+inhibition+of+CaMKK2+with+the+selective+antagonist+STO-609+regresses+NAFLD&doi=10.1038%2Fs41598-017-12139-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of CaMKK2 with the selective antagonist STO-609 regresses NAFLD</span></div><div class="casAuthors">York Brian; Li Feng; Lin Fumin; Marcelo Kathrina L; Mao Jianqiang; Dean Adam; Gonzales Naomi; Maity Suman; Coarfa Cristian; Putluri Nagireddy; Means Anthony R; York Brian; Maity Suman; Coarfa Cristian; Means Anthony R; Li Feng; Gonzales Naomi; Means Anthony R; Gooden David</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11793</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Binding of calcium to its intracellular receptor calmodulin (CaM) activates a family of Ca(2+)/CaM-dependent protein kinases.  CaMKK2 (Ca(2+)/CaM-dependent protein kinase kinase 2) is a central member of this kinase family as it controls the actions of a CaMK cascade involving CaMKI, CaMKIV or AMPK.  CaMKK2 controls insulin signaling, metabolic homeostasis, inflammation and cancer cell growth highlighting its potential as a therapeutic target for a variety of diseases.  STO-609 is a selective, small molecule inhibitor of CaMKK2.  Although STO-609 has been used extensively in vitro and in cells to characterize and define new mechanistic functions of CaMKK2, only a few studies have reported the in vivo use of STO-609.  We synthesized functional STO-609 and assessed its pharmacological properties through in vitro (kinase assay), ex vivo (human liver microsomes) and in vivo (mouse) model systems.  We describe the metabolic processing of STO-609, its toxicity, pharmacokinetics and bioavailability in a variety of mouse tissues.  Utilizing these data, we show STO-609 treatment to inhibit CaMKK2 function confers protection against non-alcoholic fatty liver disease.  These data provide a valuable resource by establishing criteria for use of STO-609 to inhibit the in vivo functions of CaMKK2 and demonstrate its utility for treating metabolically-related hepatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBljNsbBpYgkh5eUSa2oe_fW6udTcc2ea2z-sC3pszBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbpvFeqtQ%253D%253D&md5=6dc5caaa28569c21b3cae14375ed714b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12139-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12139-3%26sid%3Dliteratum%253Aachs%26aulast%3DYork%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DMarcelo%26aufirst%3DK.%2BL.%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DDean%26aufirst%3DA.%26aulast%3DGonzales%26aufirst%3DN.%26aulast%3DGooden%26aufirst%3DD.%26aulast%3DMaity%26aufirst%3DS.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DPutluri%26aufirst%3DN.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DPharmacological%2520inhibition%2520of%2520CaMKK2%2520with%2520the%2520selective%2520antagonist%2520STO-609%2520regresses%2520NAFLD%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11793%26doi%3D10.1038%2Fs41598-017-12139-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckholz, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selley, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearer, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faggioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piubelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. Y.</span></span> <span> </span><span class="NLM_article-title">An orally available, brain-penetrant CAMKK2 inhibitor reduces food intake in rodent model</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1958</span>– <span class="NLM_lpage">1963</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bmcl.2018.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29653895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1SitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1958-1963&author=D.+J.+Priceauthor=D.+H.+Drewryauthor=L.+T.+Schallerauthor=B.+D.+Thompsonauthor=P.+R.+Reidauthor=P.+R.+Maloneyauthor=X.+Liangauthor=P.+Bankerauthor=R.+G.+Buckholzauthor=P.+K.+Selleyauthor=O.+B.+McDonaldauthor=J.+L.+Smithauthor=T.+W.+Shearerauthor=R.+F.+Coxauthor=S.+P.+Williamsauthor=R.+A.+Reidauthor=S.+Tacconiauthor=F.+Faggioniauthor=C.+Piubelliauthor=I.+Sartoriauthor=M.+Tessariauthor=T.+Y.+Wang&title=An+orally+available%2C+brain-penetrant+CAMKK2+inhibitor+reduces+food+intake+in+rodent+model&doi=10.1016%2Fj.bmcl.2018.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">An orally available, brain-penetrant CAMKK2 inhibitor reduces food intake in rodent model</span></div><div class="casAuthors">Price, Daniel J.; Drewry, David H.; Schaller, Lee T.; Thompson, Brian D.; Reid, Paul R.; Maloney, Patrick R.; Liang, Xi; Banker, Periette; Buckholz, Richard G.; Selley, Paula K.; McDonald, Octerloney B.; Smith, Jeffery L.; Shearer, Todd W.; Cox, Richard F.; Williams, Shawn P.; Reid, Robert A.; Tacconi, Stefano; Faggioni, Federico; Piubelli, Chiara; Sartori, Ilaria; Tessari, Michela; Wang, Tony Y.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1958-1963</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hypothalamic CAMKK2 represents a potential mechanism for chem. affecting satiety and promoting wt. loss in clin. obese patients.  Single-digit nanomolar inhibitors of CAMKK2 were identified in three related ATP-competitive series.  Limited optimization of kinase selectivity, soly., and pharmacokinetic properties were undertaken on all three series, as SAR was often transferrable.  Ultimately, a 2,4-diaryl 7-azaindole was optimized to afford a tool mol. that potently inhibits AMPK phosphorylation in a hypothalamus-derived cell line, is orally bioavailable, and crosses the blood-brain barrier.  When dosed orally in rodents, compd. 4t limited ghrelin-induced food intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE-unQLVd-VrVg90H21EOLACvtfcHk0lgHJY9U61W2MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1SitLg%253D&md5=5013a14295c1eb6bf07a0ff7f3bef684</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DSchaller%26aufirst%3DL.%2BT.%26aulast%3DThompson%26aufirst%3DB.%2BD.%26aulast%3DReid%26aufirst%3DP.%2BR.%26aulast%3DMaloney%26aufirst%3DP.%2BR.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DBanker%26aufirst%3DP.%26aulast%3DBuckholz%26aufirst%3DR.%2BG.%26aulast%3DSelley%26aufirst%3DP.%2BK.%26aulast%3DMcDonald%26aufirst%3DO.%2BB.%26aulast%3DSmith%26aufirst%3DJ.%2BL.%26aulast%3DShearer%26aufirst%3DT.%2BW.%26aulast%3DCox%26aufirst%3DR.%2BF.%26aulast%3DWilliams%26aufirst%3DS.%2BP.%26aulast%3DReid%26aufirst%3DR.%2BA.%26aulast%3DTacconi%26aufirst%3DS.%26aulast%3DFaggioni%26aufirst%3DF.%26aulast%3DPiubelli%26aufirst%3DC.%26aulast%3DSartori%26aufirst%3DI.%26aulast%3DTessari%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%2BY.%26atitle%3DAn%2520orally%2520available%252C%2520brain-penetrant%2520CAMKK2%2520inhibitor%2520reduces%2520food%2520intake%2520in%2520rodent%2520model%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D1958%26epage%3D1963%26doi%3D10.1016%2Fj.bmcl.2018.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribar, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noeldner, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlbauer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witters, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Hypothalamic CaMKK2 contributes to the regulation of energy balance</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2008.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cmet.2008.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18460329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtFeitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=377-388&author=K.+A.+Andersonauthor=T.+J.+Ribarauthor=F.+Linauthor=P.+K.+Noeldnerauthor=M.+F.+Greenauthor=M.+J.+Muehlbauerauthor=L.+A.+Wittersauthor=B.+E.+Kempauthor=A.+R.+Means&title=Hypothalamic+CaMKK2+contributes+to+the+regulation+of+energy+balance&doi=10.1016%2Fj.cmet.2008.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Hypothalamic CaMKK2 contributes to the regulation of energy balance</span></div><div class="casAuthors">Anderson, Kristin A.; Ribar, Thomas J.; Lin, Fumin; Noeldner, Pamela K.; Green, Michelle F.; Muehlbauer, Michael J.; Witters, Lee A.; Kemp, Bruce E.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">377-388</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Detailed knowledge of the pathways by which ghrelin and leptin signal to AMPK in hypothalamic neurons and lead to regulation of appetite and glucose homeostasis is central to the development of effective means to combat obesity.  Here the authors identify CaMKK2 as a component of one of these pathways, show that it regulates hypothalamic prodn. of the orexigenic hormone NPY, provide evidence that it functions as an AMPKα kinase in the hypothalamus, and demonstrate that it forms a unique signaling complex with AMPKα and β.  Acute pharmacol. inhibition of CaMKK2 in wild-type mice, but not CaMKK2 null mice, inhibits appetite and promotes wt. loss consistent with decreased NPY and AgRP mRNAs.  Moreover, the loss of CaMKK2 protects mice from high-fat diet-induced obesity, insulin resistance, and glucose intolerance.  These data underscore the potential of targeting CaMKK2 as a therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn7ikRZ3NqH7Vg90H21EOLACvtfcHk0lgHJY9U61W2MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtFeitbc%253D&md5=4cb56cffd6e8aa32bf1b38355223ad16</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DRibar%26aufirst%3DT.%2BJ.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DNoeldner%26aufirst%3DP.%2BK.%26aulast%3DGreen%26aufirst%3DM.%2BF.%26aulast%3DMuehlbauer%26aufirst%3DM.%2BJ.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DHypothalamic%2520CaMKK2%2520contributes%2520to%2520the%2520regulation%2520of%2520energy%2520balance%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D7%26spage%3D377%26epage%3D388%26doi%3D10.1016%2Fj.cmet.2008.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngoei, K. R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAloon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhagat, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, N. X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dite, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span> <span> </span><span class="NLM_article-title">CaMKK2 is inactivated by cAMP-PKA signaling and 14-3-3 adaptor proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">16239</span>– <span class="NLM_lpage">16250</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra120.013756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.RA120.013756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32913128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1CjsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=16239-16250&author=C.+G.+Langendorfauthor=M.+T.+O%E2%80%99Brienauthor=K.+R.+W.+Ngoeiauthor=L.+M.+McAloonauthor=U.+Dhagatauthor=A.+Hoqueauthor=N.+X.+Y.+Lingauthor=T.+A.+Diteauthor=S.+Galicauthor=K.+Lohauthor=M.+W.+Parkerauthor=J.+S.+Oakhillauthor=B.+E.+Kempauthor=J.+W.+Scott&title=CaMKK2+is+inactivated+by+cAMP-PKA+signaling+and+14-3-3+adaptor+proteins&doi=10.1074%2Fjbc.ra120.013756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">CaMKK2 is inactivated by cAMP-PKA signaling and 14-3-3 adaptor proteins</span></div><div class="casAuthors">Langendorf, Christopher G.; O'Brien, Matthew T.; Ngoei, Kevin R. W.; McAloon, Luke M.; Dhagat, Urmi; Hoque, Ashfaqul; Ling, Naomi X. Y.; Dite, Toby A.; Galic, Sandra; Loh, Kim; Parker, Michael W.; Oakhill, Jonathan S.; Kemp, Bruce E.; Scott, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16239-16250</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The calcium-calmodulin-dependent protein kinase kinase-2 (CaMKK2) is a key regulator of cellular and whole-body energy metab.  It is known to be activated by increases in intracellular Ca2+, but the mechanisms by which it is inactivated are less clear.  CaMKK2 inhibition protects against prostate cancer, hepatocellular carcinoma, and metabolic derangements induced by a high-fat diet; therefore, elucidating the intracellular mechanisms that inactivate CaMKK2 has important therapeutic implications.  Here we show that stimulation of cAMP-dependent protein kinase A (PKA) signaling in cells inactivates CaMKK2 by phosphorylation of three conserved serine residues.  PKA-dependent phosphorylation of Ser495 directly impairs calcium-calmodulin activation, whereas phosphorylation of Ser100 and Ser511 mediate recruitment of 14-3-3 adaptor proteins that hold CaMKK2 in the inactivated state by preventing dephosphorylation of phospho-Ser495.  We also report the crystal structure of 14-3-3ζ bound to a synthetic diphosphorylated peptide that reveals how the canonical (Ser511) and noncanonical (Ser100) 14-3-3 consensus sites on CaMKK2 cooperate to bind 14-3-3 proteins.  Our findings provide detailed mol. insights into how cAMP-PKA signaling inactivates CaMKK2 and reveals a pathway to inhibit CaMKK2 with potential for treating human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSuF1Vc9L8ObVg90H21EOLACvtfcHk0lg97G9Ynr-2YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1CjsLrF&md5=9c2ebf5e227d37cb77074dd10134c2b1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA120.013756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA120.013756%26sid%3Dliteratum%253Aachs%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BT.%26aulast%3DNgoei%26aufirst%3DK.%2BR.%2BW.%26aulast%3DMcAloon%26aufirst%3DL.%2BM.%26aulast%3DDhagat%26aufirst%3DU.%26aulast%3DHoque%26aufirst%3DA.%26aulast%3DLing%26aufirst%3DN.%2BX.%2BY.%26aulast%3DDite%26aufirst%3DT.%2BA.%26aulast%3DGalic%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26atitle%3DCaMKK2%2520is%2520inactivated%2520by%2520cAMP-PKA%2520signaling%2520and%252014-3-3%2520adaptor%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D16239%26epage%3D16250%26doi%3D10.1074%2Fjbc.ra120.013756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cary, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racioppi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novack, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ca(2)(+)/calmodulin-dependent protein kinase kinase 2 stimulates osteoblast formation and inhibits osteoclast differentiation</span>. <i>J. Bone Miner. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1002/jbmr.1890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fjbmr.1890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=23408651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVOmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2013&pages=1599-1610&author=R.+L.+Caryauthor=S.+Waddellauthor=L.+Racioppiauthor=F.+Longauthor=D.+V.+Novackauthor=M.+J.+Voorauthor=U.+Sankar&title=Inhibition+of+Ca%282%29%28%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+2+stimulates+osteoblast+formation+and+inhibits+osteoclast+differentiation&doi=10.1002%2Fjbmr.1890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Ca2+/Calmodulin-Dependent Protein Kinase Kinase 2 Stimulates Osteoblast Formation and Inhibits Osteoclast Differentiation</span></div><div class="casAuthors">Cary, Rachel L.; Waddell, Seid; Racioppi, Luigi; Long, Fanxin; Novack, Deborah V.; Voor, Michael J.; Sankar, Uma</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bone and Mineral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">JBMREJ</span>;
        ISSN:<span class="NLM_cas:issn">0884-0431</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Bone remodeling, a physiol. process characterized by bone formation by osteoblasts (OBs) and resorption of preexisting bone matrix by osteoclasts (OCs), is vital for the maintenance of healthy bone tissue in adult humans.  Imbalances in this vital process result in pathol. conditions including osteoporosis.  Owing to its initial asymptomatic nature, osteoporosis is often detected only after the patient has sustained significant bone loss or a fracture.  Hence, anabolic therapeutics that stimulate bone accrual is in high clin. demand.  Here we identify Ca2+/calmodulin (CaM)-dependent protein kinase kinase 2 (CaMKK2) as a potential target for such therapeutics because its inhibition enhances OB differentiation and bone growth and suppresses OC differentiation.  Mice null for CaMKK2 possess higher trabecular bone mass in their long bones, along with significantly more OBs and fewer multinuclear OCs.  In vitro, although Camkk2-/- mesenchymal stem cells (MSCs) yield significantly higher nos. of OBs, bone marrow cells from Camkk2-/- mice produce fewer multinuclear OCs.  Acute inhibition of CaMKK2 by its selective, cell-permeable pharmacol. inhibitor STO-609 also results in increased OB and diminished OC formation.  Further, we find phospho-protein kinase A (PKA) and Ser133 phosphorylated form of cyclic adenosine monophosphate (cAMP) response element binding protein (pCREB) to be markedly elevated in OB progenitors deficient in CaMKK2.  On the other hand, genetic ablation of CaMKK2 or its pharmacol. inhibition in OC progenitors results in reduced pCREB as well as significantly reduced levels of its transcriptional target, nuclear factor of activated T cells, cytoplasmic (NFATc1).  Moreover, in vivo administration of STO-609 results in increased OBs and diminished OCs, conferring significant protection from ovariectomy (OVX)-induced osteoporosis in adult mice.  Thus, a novel function was shown for CaMKK2 in bone remodeling and highlight the potential for its therapeutic inhibition as a valuable bone anabolic strategy that also inhibits OC differentiation in the treatment of osteoporosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH8_fa3hgfgbVg90H21EOLACvtfcHk0lg97G9Ynr-2YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVOmsb4%253D&md5=56d88615d2500c579e6b5741a9b3c0d4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjbmr.1890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjbmr.1890%26sid%3Dliteratum%253Aachs%26aulast%3DCary%26aufirst%3DR.%2BL.%26aulast%3DWaddell%26aufirst%3DS.%26aulast%3DRacioppi%26aufirst%3DL.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNovack%26aufirst%3DD.%2BV.%26aulast%3DVoor%26aufirst%3DM.%2BJ.%26aulast%3DSankar%26aufirst%3DU.%26atitle%3DInhibition%2520of%2520Ca%25282%2529%2528%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520stimulates%2520osteoblast%2520formation%2520and%2520inhibits%2520osteoclast%2520differentiation%26jtitle%3DJ.%2520Bone%2520Miner.%2520Res.%26date%3D2013%26volume%3D28%26spage%3D1599%26epage%3D1610%26doi%3D10.1002%2Fjbmr.1890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novack, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of CaMKK2 reverses age-associated decline in bone mass</span>. <i>Bone</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1016/j.bone.2015.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bone.2015.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25724145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslCntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=120-127&author=Z.+J.+Pritchardauthor=R.+L.+Caryauthor=C.+Yangauthor=D.+V.+Novackauthor=M.+J.+Voorauthor=U.+Sankar&title=Inhibition+of+CaMKK2+reverses+age-associated+decline+in+bone+mass&doi=10.1016%2Fj.bone.2015.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of CaMKK2 reverses age-associated decline in bone mass</span></div><div class="casAuthors">Pritchard, Zachary J.; Cary, Rachel L.; Yang, Chang; Novack, Deborah V.; Voor, Michael J.; Sankar, Uma</div><div class="citationInfo"><span class="NLM_cas:title">Bone (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">BONEDL</span>;
        ISSN:<span class="NLM_cas:issn">1873-2763</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Decline in bone formation is a major contributing factor to the loss of bone mass assocd. with aging.  We previously showed that the genetic ablation of the tissue-restricted and multifunctional Ca2 +/calmodulin (CaM)-dependent protein kinase kinase 2 (CaMKK2) stimulates trabecular bone mass accrual, mainly by promoting anabolic pathways and inhibiting catabolic pathways of bone remodeling.  In this study, we investigated whether inhibition of this kinase using its selective cell-permeable inhibitor STO-609 will stimulate bone formation in 32 wk old male WT mice and reverse age-assocd. of decline in bone vol. and strength.  Tri-weekly i.p. injections of saline or STO-609 (10 μM) were performed for six weeks followed by metabolic labeling with calcein and alizarin red.  New bone formation was assessed by dynamic histomorphometry whereas micro-computed tomog. was employed to measure trabecular bone vol., microarchitecture and femoral mid-shaft geometry.  Cortical and trabecular bone biomech. properties were assessed using three-point bending and punch compression methods resp.  Our results reveal that as they progress from 12 to 32 wk of age, WT mice sustain a significant decline in trabecular bone vol., microarchitecture and strength as well as cortical bone strength.  However, treatment of the 32 wk old WT mice with STO-609 stimulated apposition of new bone and completely reversed the age-assocd. decrease in bone vol., quality, as well as trabecular and cortical bone strength.  We also obsd. that regardless of age, male Camkk2-/- mice possessed significantly elevated trabecular bone vol., microarchitecture and compressive strength as well as cortical bone strength compared to age-matched WT mice, implying that the chronic loss of this kinase attenuates age-assocd. decline in bone mass.  Further, whereas STO-609 treatment and/or the absence of CaMKK2 significantly enhanced the femoral mid-shaft geometry, the mid-shaft cortical wall thickness and material bending stress remained similar among the cohorts, implying that regardless of treatment, the material properties of the bone remain similar.  Thus, our cumulative results provide evidence for the pharmacol. inhibition of CaMKK2 as a bone anabolic strategy in combating age-assocd. osteoporosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHR9HKD1SNbVg90H21EOLACvtfcHk0lg97G9Ynr-2YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslCntb8%253D&md5=0a19057ffeda873dd4b33c566006d856</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bone.2015.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bone.2015.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DPritchard%26aufirst%3DZ.%2BJ.%26aulast%3DCary%26aufirst%3DR.%2BL.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DNovack%26aufirst%3DD.%2BV.%26aulast%3DVoor%26aufirst%3DM.%2BJ.%26aulast%3DSankar%26aufirst%3DU.%26atitle%3DInhibition%2520of%2520CaMKK2%2520reverses%2520age-associated%2520decline%2520in%2520bone%2520mass%26jtitle%3DBone%26date%3D2015%26volume%3D75%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.bone.2015.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribar, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">The Ca2+/calmodulin-dependent protein kinase kinase, CaMKK2, inhibits preadipocyte differentiation</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">3668</span>– <span class="NLM_lpage">3679</span>, <span class="refDoi"> DOI: 10.1210/en.2011-1107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1210%2Fen.2011-1107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21862616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVanurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=3668-3679&author=F.+Linauthor=T.+J.+Ribarauthor=A.+R.+Means&title=The+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase%2C+CaMKK2%2C+inhibits+preadipocyte+differentiation&doi=10.1210%2Fen.2011-1107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Ca2+/calmodulin-dependent protein kinase kinase, CaMKK2, inhibits preadipocyte differentiation</span></div><div class="casAuthors">Lin, Fumin; Ribar, Thomas J.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3668-3679</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">When fed a std. chow diet, CaMKK2 null mice have increased adiposity and larger adipocytes than do wild-type mice, whereas energy balance is unchanged.  Here, we show that Ca2+/calmodulin-dependent protein kinase kinase 2 (CaMKK2) is expressed in preadipocytes, where it functions as an AMP-activated protein kinase (AMPK)α kinase.  Acute inhibition or deletion of CaMKK2 in preadipocytes enhances their differentiation into mature adipocytes, which can be reversed by 5-aminoimidazole-4-carboxamide ribonucleotide-mediated activation of AMPK.  During adipogenesis, CaMKK2 expression is markedly decreased and temporally accompanied by increases in mRNA encoding the early adipogenic genes CCAAT/enhancer binding protein (C/EBP) β and C/EBP δ.  Preadipocyte factor 1 has been reported to inhibit adipogenesis by up-regulating sex detg. region Y-box 9 (Sox9) expression in preadipocytes and Sox9 suppresses C/EBPβ and C/EBPδ transcription.  We show that inhibition of the CaMKK2/AMPK signaling cascade in preadipocytes reduces preadipocyte factor 1 and Sox9 mRNA resulting in accelerated adipogenesis.  We conclude that CaMKK2 and AMPK function in a signaling pathway that participates in the regulation of adiposity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPYor3uM5dU7Vg90H21EOLACvtfcHk0lg97G9Ynr-2YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVanurvE&md5=30187fd728913b1f6765b8b228b9dd48</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fen.2011-1107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2011-1107%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DF.%26aulast%3DRibar%26aufirst%3DT.%2BJ.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DThe%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%252C%2520CaMKK2%252C%2520inhibits%2520preadipocyte%2520differentiation%26jtitle%3DEndocrinology%26date%3D2011%26volume%3D152%26spage%3D3668%26epage%3D3679%26doi%3D10.1210%2Fen.2011-1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müschen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. C.</span></span> <span> </span><span class="NLM_article-title">CAMKs support development of acute myeloid leukemia</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/s13045-018-0574-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1186%2Fs13045-018-0574-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29482582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=30&author=X.+Kangauthor=C.+Cuiauthor=C.+Wangauthor=G.+Wuauthor=H.+Chenauthor=Z.+Luauthor=X.+Chenauthor=L.+Wangauthor=J.+Huangauthor=H.+Gengauthor=M.+Zhaoauthor=Z.+Chenauthor=M.+M%C3%BCschenauthor=H.-Y.+Wangauthor=C.+C.+Zhang&title=CAMKs+support+development+of+acute+myeloid+leukemia&doi=10.1186%2Fs13045-018-0574-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">CAMKs support development of acute myeloid leukemia</span></div><div class="casAuthors">Kang, Xunlei; Cui, Changhao; Wang, Chen; Wu, Guojin; Chen, Heyu; Lu, Zhigang; Chen, Xiaoli; Wang, Li; Huang, Jie; Geng, Huimin; Zhao, Meng; Chen, Zhengshan; Muschen, Markus; Wang, Huan-You; Zhang, Cheng Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30/1-30/12</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: We recently identified the human leukocyte Ig-like receptor B2 (LILRB2) and its mouse ortholog-paired Ig-like receptor (PirB) as receptors for several angiopoietin-like proteins (Angptls).  We also demonstrated that PirB is important for the development of acute myeloid leukemia (AML), but exactly how an inhibitory receptor such as PirB can support cancer development is intriguing.  Results: Here, we showed that the activation of Ca (2+)/calmodulin-dependent protein kinases (CAMKs) is coupled with PirB signaling in AML cells.  High expression of CAMKs is assocd. with a poor overall survival probability in patients with AML.  Knockdown of CAMKI or CAMKIV decreased human acute leukemia development in vitro and in vivo.  Mouse AML cells that are defective in PirB signaling had decreased activation of CAMKs, and the forced expression of CAMK partially rescued the PirB-defective phenotype in the MLL-AF9 AML mouse model.  The inhibition of CAMK kinase activity or deletion of CAMKIV significantly slowed AML development and decreased the AML stem cell activity.  We also found that CAMKIV acts through the phosphorylation of one of its well-known target (CREB) in AML cells.  Conclusion: CAMKs are essential for the growth of human and mouse AML.  The inhibition of CAMK signaling may become an effective strategy for treating leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcfARnv35UB7Vg90H21EOLACvtfcHk0liXvOehOaWRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyqt7w%253D&md5=c9c18a264c577c442e8bae29e44dafbf</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs13045-018-0574-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-018-0574-8%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DM%25C3%25BCschen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.-Y.%26aulast%3DZhang%26aufirst%3DC.%2BC.%26atitle%3DCAMKs%2520support%2520development%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2018%26volume%3D11%26spage%3D30%26doi%3D10.1186%2Fs13045-018-0574-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span> <span> </span><span class="NLM_article-title">Growth inhibition of human gastric adenocarcinoma cells in vitro by STO-609 is independent of calcium/calmodulin-dependent protein kinase kinase-beta and adenosine monophosphate-activated protein kinase</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1164</span>– <span class="NLM_lpage">1171</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27158402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFWlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1164-1171&author=Z.+Maauthor=D.+Wenauthor=X.+Wangauthor=L.+Yangauthor=T.+Liuauthor=J.+Liuauthor=J.+Zhuauthor=X.+Fang&title=Growth+inhibition+of+human+gastric+adenocarcinoma+cells+in+vitro+by+STO-609+is+independent+of+calcium%2Fcalmodulin-dependent+protein+kinase+kinase-beta+and+adenosine+monophosphate-activated+protein+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Growth inhibition of human gastric adenocarcinoma cells in vitro by STO-609 is independent of calcium/ calmodulin-dependent protein kinase kinase-beta and adenosine monophosphate-activated protein kinase</span></div><div class="casAuthors">Ma, Zhiming; Wen, Dacheng; Wang, Xudong; Yang, Longfei; Liu, Tianzhou; Liu, Jingjing; Zhu, Jiaming; Fang, Xuedong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1164-1171</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Adenosine monophosphate (AMP)-activated protein kinase is a recently identified downstream target of calcium/calmodulin-dependent protein kinase kinase-beta, and is involved in the regulation of cell metab. and cell proliferation.  STO-609 is a selective antagonist of calcium/calmodulin-dependent protein kinase kinase-beta.  In the present study, we found that STO-609 suppressed AMP-activated protein kinase activity, reduced expression of Akt and ERK, and increased cell apoptosis in SNU-1 and N87 cells but not normal gastric epithelial cells (CCL-241).  Interestingly, we found such effects of STO-609 on gastric cancer cells were not affected after the knock-down of CaMKK- β and AMPK.  In conclusion, STO-609 is an effective cytotoxic agent for gastric adenocarcinoma in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIgiOWvdZlULVg90H21EOLACvtfcHk0liXvOehOaWRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFWlt7w%253D&md5=4118e22a54738fcd8b16234543fefd3a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DX.%26atitle%3DGrowth%2520inhibition%2520of%2520human%2520gastric%2520adenocarcinoma%2520cells%2520in%2520vitro%2520by%2520STO-609%2520is%2520independent%2520of%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase-beta%2520and%2520adenosine%2520monophosphate-activated%2520protein%2520kinase%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2016%26volume%3D8%26spage%3D1164%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Byrne, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotte, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eduful, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">In depth analysis of kinase cross screening data to identify CAMKK2 inhibitory scaffolds</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">325</span>, <span class="refDoi"> DOI: 10.3390/molecules25020325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3390%2Fmolecules25020325" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslCjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=325&author=S.+N.+O%E2%80%99Byrneauthor=J.+W.+Scottauthor=J.+R.+Pilotteauthor=A.+D.+S.+Santiagoauthor=C.+G.+Langendorfauthor=J.+S.+Oakhillauthor=B.+J.+Edufulauthor=R.+M.+Cou%C3%B1agoauthor=C.+I.+Wellsauthor=W.+J.+Zuercherauthor=T.+M.+Willsonauthor=D.+H.+Drewry&title=In+depth+analysis+of+kinase+cross+screening+data+to+identify+CAMKK2+inhibitory+scaffolds&doi=10.3390%2Fmolecules25020325"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">In depth analysis of kinase cross screening data to identify CAMKK2 inhibitory scaffolds</span></div><div class="casAuthors">O'Byrne, Sean N.; Scott, John W.; Pilotte, Joseph R.; Santiago, Andre Da S.; Langendorf, Christopher G.; Oakhill, Jonathan S.; Eduful, Benjamin J.; Counago, Rafael M.; Wells, Carrow I.; Zuercher, William J.; Willson, Timothy M.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">325</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The calcium/calmodulin-dependent protein kinase kinase 2 (CAMKK2) activates CAMK1, CAMK4, AMPK, and AKT, leading to numerous physiol. responses.  The deregulation of CAMKK2 is linked to several diseases, suggesting the utility of CAMKK2 inhibitors for oncol., metabolic and inflammatory indications.  In this work, we demonstrate that STO-609, frequently described as a selective inhibitor for CAMKK2, potently inhibits a significant no. of other kinases.  Through an anal. of literature and public databases, we have identified other potent CAMKK2 inhibitors and verified their activities in differential scanning fluorimetry and enzyme inhibition assays.  These inhibitors are potential starting points for the development of selective CAMKK2 inhibitors and will lead to tools that delineate the roles of this kinase in disease biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzFTNmh85XgLVg90H21EOLACvtfcHk0liXvOehOaWRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslCjsbo%253D&md5=fed1cf447694b4bd7fdb39744c27442b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25020325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25020325%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Byrne%26aufirst%3DS.%2BN.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26aulast%3DPilotte%26aufirst%3DJ.%2BR.%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DEduful%26aufirst%3DB.%2BJ.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DIn%2520depth%2520analysis%2520of%2520kinase%2520cross%2520screening%2520data%2520to%2520identify%2520CAMKK2%2520inhibitory%2520scaffolds%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D325%26doi%3D10.3390%2Fmolecules25020325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amato, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">CDKL2 promotes epithelial-mesenchymal transition and breast cancer progression</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10840</span>– <span class="NLM_lpage">10853</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.18632%2Foncotarget.2535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=25333262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC2M3itVKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10840-10853&author=L.+Liauthor=C.+Liuauthor=R.+J.+Amatoauthor=J.+T.+Changauthor=G.+Duauthor=W.+Li&title=CDKL2+promotes+epithelial-mesenchymal+transition+and+breast+cancer+progression&doi=10.18632%2Foncotarget.2535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CDKL2 promotes epithelial-mesenchymal transition and breast cancer progression</span></div><div class="casAuthors">Li Linna; Liu Chunping; Amato Robert J; Chang Jeffrey T; Du Guangwei; Li Wenliang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10840-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epithelial-mesenchymal transition (EMT) confers mesenchymal properties on epithelial cells and has been closely associated with the acquisition of aggressive traits by epithelial cancer cells.  To identify novel regulators of EMT, we carried out cDNA screens that covered 500 human kinases.  Subsequent characterization of candidate kinases led us to uncover cyclin-dependent kinase-like 2 (CDKL2) as a novel potent promoter for EMT and breast cancer progression.  CDKL2-expressing human mammary gland epithelial cells displayed enhanced mesenchymal traits and stem cell-like phenotypes, which was acquired through activating a ZEB1/E-cadherin/β-catenin positive feedback loop and regulating CD44 mRNA alternative splicing to promote conversion of CD24(high) cells to CD44(high) cells.  Furthermore, CDKL2 enhanced primary tumor formation and metastasis in a breast cancer xenograft model.  Notably, CDKL2 is expressed significantly higher in mesenchymal human breast cancer cell lines than in epithelial lines, and its over-expression/amplification in human breast cancers is associated with shorter disease-free survival.  Taken together, our study uncovered a major role for CDKL2 in promoting EMT and breast cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQC9qYy_17RnIJKJbLTgsvCfW6udTcc2eaka9iXGZdifrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3itVKhsw%253D%253D&md5=49752f32dbc5808a474bdb42f1db87c8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2535%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DAmato%26aufirst%3DR.%2BJ.%26aulast%3DChang%26aufirst%3DJ.%2BT.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DCDKL2%2520promotes%2520epithelial-mesenchymal%2520transition%2520and%2520breast%2520cancer%2520progression%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10840%26epage%3D10853%26doi%3D10.18632%2Foncotarget.2535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billard, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzhugh, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozowski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timoshchenko, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lininger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klauber-Demore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahagian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serody, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrant, T. K.</span></span> <span> </span><span class="NLM_article-title">G protein coupled receptor kinase 3 regulates breast cancer migration, invasion, and metastasis</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0152856</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0152856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1371%2Fjournal.pone.0152856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=27049755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaks7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=M.+J.+Billardauthor=D.+J.+Fitzhughauthor=J.+S.+Parkerauthor=J.+M.+Brozowskiauthor=M.+W.+McGinnisauthor=R.+G.+Timoshchenkoauthor=D.+S.+Serafinauthor=R.+Liningerauthor=N.+Klauber-Demoreauthor=G.+Sahagianauthor=Y.+K.+Truongauthor=M.+F.+Sassanoauthor=J.+S.+Serodyauthor=T.+K.+Tarrant&title=G+protein+coupled+receptor+kinase+3+regulates+breast+cancer+migration%2C+invasion%2C+and+metastasis&doi=10.1371%2Fjournal.pone.0152856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">G protein coupled receptor kinase 3 regulates breast cancer migration, invasion, and metastasis</span></div><div class="casAuthors">Billard, Matthew J.; Fitzhugh, David J.; Parker, Joel S.; Brozowski, Jaime M.; McGinnis, Marcus W.; Timoshchenko, Roman G.; Stephen, Serafin D.; Lininger, Ruth; Klauber-Demore, Nancy; Sahagian, Gary; Truong, Young K.; Sassano, Maria F.; Serody, Jonathan S.; Tarrant, Teresa K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0152856/1-e0152856/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC) is a heterogeneous disease that has a poor prognosis and limited treatment options.  Chemokine receptor interactions are important modulators of breast cancer metastasis; however, it is now recognized that quant. surface expression of one important chemokine receptor, CXCR4, may not directly correlate with metastasis and that its functional activity in breast cancer may better inform tumor pathogenicity.  G protein coupled receptor kinase 3 (GRK3) is a neg. regulator of CXCR4 activity, and we show that GRK expression correlates with tumorigenicity, mol. subtype, and metastatic potential in human tumor microarray anal.  Using established human breast cancer cell lines and an immunocompetent in vivo mouse model, we further demonstrate that alterations in GRK3 expression levels in tumor cells directly affect migration and invasion in vitro and the establishment of distant metastasis in vivo.  The effects of GRK3 modulation appear to be specific to chemokine-mediated migration behaviors without influencing tumor cell proliferation or survival.  These data demonstrate that GRK3 dysregulation may play an important part in TNBC metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7p1mNnYr53rVg90H21EOLACvtfcHk0lhWLtU8gA_QJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaks7zN&md5=e97281c582eef4661bf7cebd5dc9dcb5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0152856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0152856%26sid%3Dliteratum%253Aachs%26aulast%3DBillard%26aufirst%3DM.%2BJ.%26aulast%3DFitzhugh%26aufirst%3DD.%2BJ.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DBrozowski%26aufirst%3DJ.%2BM.%26aulast%3DMcGinnis%26aufirst%3DM.%2BW.%26aulast%3DTimoshchenko%26aufirst%3DR.%2BG.%26aulast%3DSerafin%26aufirst%3DD.%2BS.%26aulast%3DLininger%26aufirst%3DR.%26aulast%3DKlauber-Demore%26aufirst%3DN.%26aulast%3DSahagian%26aufirst%3DG.%26aulast%3DTruong%26aufirst%3DY.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DSerody%26aufirst%3DJ.%2BS.%26aulast%3DTarrant%26aufirst%3DT.%2BK.%26atitle%3DG%2520protein%2520coupled%2520receptor%2520kinase%25203%2520regulates%2520breast%2520cancer%2520migration%252C%2520invasion%252C%2520and%2520metastasis%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0152856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landesman-Bollag, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romieu-Mourez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonenshein, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seldin, D. C.</span></span> <span> </span><span class="NLM_article-title">Protein kinase CK2 in mammary gland tumorigenesis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3247</span>– <span class="NLM_lpage">3257</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fsj.onc.1204411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11423974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1GktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3247-3257&author=E.+Landesman-Bollagauthor=R.+Romieu-Mourezauthor=D.+H.+Songauthor=G.+E.+Sonensheinauthor=R.+D.+Cardiffauthor=D.+C.+Seldin&title=Protein+kinase+CK2+in+mammary+gland+tumorigenesis&doi=10.1038%2Fsj.onc.1204411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase CK2 in mammary gland tumorigenesis</span></div><div class="casAuthors">Landesman-Bollag, Esther; Romieu-Mourez, Raphaelle; Song, Diane H.; Sonenshein, Gail E.; Cardiff, Robert D.; Seldin, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3247-3257</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein kinase CK2 is a ubiquitous and evolutionarily conserved serine/threonine kinase that is upregulated in many human cancers and can serve as an oncogene in lymphocytes.  Recently, the authors have demonstrated that CK2 potentiates Wnt/β-catenin signaling in mammary epithelial cells.  To det. whether CK2 overexpression contributes to mammary tumorigenesis, the authors have performed comparative studies of human and rat breast cancer specimens and the authors have engineered transgenic mice with dysregulated expression of CK2α in the mammary gland.  The authors find that CK2 is highly expressed in human breast tumor specimens and in carcinogen-induced rat mammary tumors.  Overexpression of CK2α in the mammary gland of transgenic mice, under control of the MMTV-LTR, causes hyperplasia and dysplasia of the female mammary gland.  Thirty per cent of the female MMTV-CK2α transgenic mice develop mammary adenocarcinomas at a median of 23 mo of age, often assocd. with Wnt pathway activation, as evidenced by upregulation of β-catenin protein.  NF-κB activation and upregulation of c-Myc also occur frequently.  Thus, in mice, rats, and humans, dysregulated expression of CK2 is assocd. with and is capable of contributing to mammary tumorigenesis.  Targeted inhibition of CK2 could be useful in the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDbe-6fAKYLVg90H21EOLACvtfcHk0lhWLtU8gA_QJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1GktLk%253D&md5=a68a69b007fd48038580645bd847f4de</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204411%26sid%3Dliteratum%253Aachs%26aulast%3DLandesman-Bollag%26aufirst%3DE.%26aulast%3DRomieu-Mourez%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DD.%2BH.%26aulast%3DSonenshein%26aufirst%3DG.%2BE.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DSeldin%26aufirst%3DD.%2BC.%26atitle%3DProtein%2520kinase%2520CK2%2520in%2520mammary%2520gland%2520tumorigenesis%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D3247%26epage%3D3257%26doi%3D10.1038%2Fsj.onc.1204411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langouet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fardel, O.</span></span> <span> </span><span class="NLM_article-title">Activation of the aryl hydrocarbon receptor by the calcium/calmodulin-dependent protein kinase kinase inhibitor 7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid (STO-609)</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2556</span>– <span class="NLM_lpage">2563</span>, <span class="refDoi"> DOI: 10.1124/dmd.108.023333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1124%2Fdmd.108.023333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18755850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=2556-2563&author=P.+Monteiroauthor=D.+Gilotauthor=S.+Langouetauthor=O.+Fardel&title=Activation+of+the+aryl+hydrocarbon+receptor+by+the+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+inhibitor+7-oxo-7H-benzimidazo%5B2%2C1-a%5Dbenz%5Bde%5Disoquinoline-3-carboxylic+acid+%28STO-609%29&doi=10.1124%2Fdmd.108.023333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the aryl hydrocarbon receptor by the calcium/calmodulin-dependent protein kinase kinase inhibitor 7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid (STO-609)</span></div><div class="casAuthors">Monteiro, Patricia; Gilot, David; Langouet, Sophie; Fardel, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2556-2563</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">This study was designed to analyze the effects of the Ca2+/calmodulin-dependent protein kinase kinase (CaMKK) inhibitor STO-609 (7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid) toward the aryl hydrocarbon receptor (AhR) pathway because Ca2+/calmodulin-dependent protein kinase (CaMK) Iα, known as a downstream CaMKK effector, has been recently shown to contribute to the AhR cascade.  STO-609 failed to alter up-regulation of the AhR target CYP1A1 in response to the potent AhR ligand 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) in MCF-7 cells.  STO-609, used at a 25 μM concn. known to fully inhibit CaMKK activity, was surprisingly found to markedly induce CYP1A1 expression and activity by itself in MCF-7 cells; it similarly up-regulated various other AhR target genes in human macrophages.  STO-609-related CYP1A1 induction was prevented by chem. inhibition or small interfering RNA-mediated knockdown expression of AhR.  Moreover, STO-609 was demonstrated to phys. interact with the ligand-binding domain of AhR, as assessed by TCDD binding competition assay, and to induce AhR translocation to the nucleus.  As already reported for AhR agonists, STO-609 triggered the increase of [Ca2+]i and activation of CaMKIα, whose inhibition through the use of the Ca2+ chelator BAPTA-acetoxymethyl ester or the CaMK inhibitor KN-93 (2-[[N-(2-hydroxyethyl)]-N-(4-methoxybenzenesulfonyl)]amino-N-(4-chlorocinnamyl)-N-methylbenzylamine), resp., prevented STO-609-mediated CYP1A1 activity induction.  Taken together, these results demonstrate that the CaMKK inhibitor STO-609 can act as an AhR ligand and, in this way, fully activates the Ca2+/CaMKIα/AhR cascade.  Such data, therefore, make unlikely any contribution of CaMKK activity to the AhR pathway and, moreover, suggest that caution may be required when using STO-609 as a specific inhibitor of CaMKKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4LM4lXUX03LVg90H21EOLACvtfcHk0lhc1AOJlWzLYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCltrzK&md5=037d08b34bc2e8aca51365f8354d1594</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1124%2Fdmd.108.023333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.108.023333%26sid%3Dliteratum%253Aachs%26aulast%3DMonteiro%26aufirst%3DP.%26aulast%3DGilot%26aufirst%3DD.%26aulast%3DLangouet%26aufirst%3DS.%26aulast%3DFardel%26aufirst%3DO.%26atitle%3DActivation%2520of%2520the%2520aryl%2520hydrocarbon%2520receptor%2520by%2520the%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520inhibitor%25207-oxo-7H-benzimidazo%255B2%252C1-a%255Dbenz%255Bde%255Disoquinoline-3-carboxylic%2520acid%2520%2528STO-609%2529%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2008%26volume%3D36%26spage%3D2556%26epage%3D2563%26doi%3D10.1124%2Fdmd.108.023333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asquith, C. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koutentis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalogirou, S. A.</span></span> <span> </span><span class="NLM_article-title">1,2,6-Thiadiazinones as novel narrow spectrum calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2) inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1221</span>, <span class="refDoi"> DOI: 10.3390/molecules23051221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.3390%2Fmolecules23051221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyksrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1221&author=C.+R.+M.+Asquithauthor=H.+P.+Godoiauthor=M.+R.+Cou%C3%B1agoauthor=T.+Laitinenauthor=W.+J.+Scottauthor=G.+C.+Langendorfauthor=S.+J.+Oakhillauthor=H.+D.+Drewryauthor=J.+W.+Zuercherauthor=A.+P.+Koutentisauthor=M.+T.+Willsonauthor=S.+A.+Kalogirou&title=1%2C2%2C6-Thiadiazinones+as+novel+narrow+spectrum+calcium%2Fcalmodulin-dependent+protein+kinase+kinase+2+%28CaMKK2%29+inhibitors&doi=10.3390%2Fmolecules23051221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">1,2,6-thiadiazinones as novel narrow spectrum calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2) inhibitors</span></div><div class="casAuthors">Asquith, Christopher R. M.; Godoi, Paulo H.; Counago, Rafael M.; Laitinen, Tuomo; Scott, John W.; Langendorf, Christopher G.; Oakhill, Jonathan S.; Drewry, David H.; Zuercher, William J.; Koutentis, Panayiotis A.; Willson, Timothy M.; Kalogirou, Andreas S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1221/1-1221/23</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We demonstrate for the first time that 4H-1,2,6-thiadiazin-4-one (TDZ) can function as a chemotype for the design of ATP-competitive kinase inhibitors.  Using insights from a co-crystal structure of a 3,5-bis(arylamino)-4H-1,2,6-thiadiazin-4-one bound to calcium/calmodulin-dependent protein kinase kinase 2 (CaMKK2), several analogs were identified with micromolar activity through targeted displacement of bound water mols. in the active site.  Since the TDZ analogs showed reduced promiscuity compared to their 2,4-dianilinopyrimidine counter parts, they represent starting points for development of highly selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobtAr-yNWZabVg90H21EOLACvtfcHk0lhc1AOJlWzLYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyksrvI&md5=ad009ea9adb9dfd9a084606d26c4f652</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23051221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23051221%26sid%3Dliteratum%253Aachs%26aulast%3DAsquith%26aufirst%3DC.%2BR.%2BM.%26aulast%3DGodoi%26aufirst%3DH.%2BP.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DM.%2BR.%26aulast%3DLaitinen%26aufirst%3DT.%26aulast%3DScott%26aufirst%3DW.%2BJ.%26aulast%3DLangendorf%26aufirst%3DG.%2BC.%26aulast%3DOakhill%26aufirst%3DS.%2BJ.%26aulast%3DDrewry%26aufirst%3DH.%2BD.%26aulast%3DZuercher%26aufirst%3DJ.%2BW.%26aulast%3DKoutentis%26aufirst%3DA.%2BP.%26aulast%3DWillson%26aufirst%3DM.%2BT.%26aulast%3DKalogirou%26aufirst%3DS.%2BA.%26atitle%3D1%252C2%252C6-Thiadiazinones%2520as%2520novel%2520narrow%2520spectrum%2520calcium%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CaMKK2%2529%2520inhibitors%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1221%26doi%3D10.3390%2Fmolecules23051221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghose, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbertz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallamo, J. P.</span></span> <span> </span><span class="NLM_article-title">Knowledge based prediction of ligand binding modes and rational inhibitor design for kinase drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5149</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/jm800475y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800475y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5149-5171&author=A.+K.+Ghoseauthor=T.+Herbertzauthor=D.+A.+Pippinauthor=J.+M.+Salvinoauthor=J.+P.+Mallamo&title=Knowledge+based+prediction+of+ligand+binding+modes+and+rational+inhibitor+design+for+kinase+drug+discovery&doi=10.1021%2Fjm800475y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge Based Prediction of Ligand Binding Modes and Rational Inhibitor Design for Kinase Drug Discovery</span></div><div class="casAuthors">Ghose, Arup K.; Herbertz, Torsten; Pippin, Douglas A.; Salvino, Joseph M.; Mallamo, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5149-5171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review describing the use of Protein Data Bank (PDB) entries specifically for kinase inhibitor drug discovery.  Topics discussed include: the current state of structure-aided ligand design, kinase structure queries and extn. from the PDB, alignment of protein structures, structural classification of ligands, anal. of ligands and binding modes, and ligand design strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDINDYVE15cbVg90H21EOLACvtfcHk0lhc1AOJlWzLYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWmurnM&md5=a1afc4d6c7f6fa4dd6aa9da8d78f1ee3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm800475y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800475y%26sid%3Dliteratum%253Aachs%26aulast%3DGhose%26aufirst%3DA.%2BK.%26aulast%3DHerbertz%26aufirst%3DT.%26aulast%3DPippin%26aufirst%3DD.%2BA.%26aulast%3DSalvino%26aufirst%3DJ.%2BM.%26aulast%3DMallamo%26aufirst%3DJ.%2BP.%26atitle%3DKnowledge%2520based%2520prediction%2520of%2520ligand%2520binding%2520modes%2520and%2520rational%2520inhibitor%2520design%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5149%26epage%3D5171%26doi%3D10.1021%2Fjm800475y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">Toward a pharmacophore for kinase frequent hitters</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5616</span>– <span class="NLM_lpage">5619</span>, <span class="refDoi"> DOI: 10.1021/jm049793g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049793g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5616-5619&author=A.+M.+Aronovauthor=M.+A.+Murcko&title=Toward+a+pharmacophore+for+kinase+frequent+hitters&doi=10.1021%2Fjm049793g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a Pharmacophore for Kinase Frequent Hitters</span></div><div class="casAuthors">Aronov, Alex M.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5616-5619</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. protein kinase inhibitors are widely employed as biol. reagents and as leads in the design of drugs for a variety of diseases.  One of the hardest challenges in kinase inhibitor design is achieving target selectivity.  By utilizing x-ray structural information for four promiscuous inhibitors, we propose a five-point pharmacophore for kinase frequent hitters, demonstrate its ability to discriminate between frequent hitters and selective ligands, and suggest a strategy for selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj2-pbD58y0bVg90H21EOLACvtfcHk0ljzehgRSrsFlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlSisrY%253D&md5=4ebc59812016803b8cbd64849d104064</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm049793g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049793g%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DToward%2520a%2520pharmacophore%2520for%2520kinase%2520frequent%2520hitters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5616%26epage%3D5619%26doi%3D10.1021%2Fjm049793g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klug-Mcleod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span> <span> </span><span class="NLM_article-title">Kinase hinge binding scaffolds and their hydrogen bond patterns</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.bmc.2015.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=26358279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6520-6527&author=L.+Xingauthor=J.+Klug-Mcleodauthor=B.+Raiauthor=E.+A.+Lunney&title=Kinase+hinge+binding+scaffolds+and+their+hydrogen+bond+patterns&doi=10.1016%2Fj.bmc.2015.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase hinge binding scaffolds and their hydrogen bond patterns</span></div><div class="casAuthors">Xing, Li; Klug-Mcleod, Jacquelyn; Rai, Brajesh; Lunney, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6520-6527</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinases constitute a major class of intracellular signaling mols., and describe some of the most prominent drug targets.  Kinase inhibitors commonly employ small chem. scaffolds that form hydrogen bonds with the kinase hinge residues connecting the N- and C-terminal lobes of the catalytic domain.  In general the satisfied hydrogen bonds are required for potent inhibition, therefore constituting a conserved feature in the majority of inhibitor-kinase interactions.  From systematically analyzing the kinase scaffolds extd. from Pfizer crystal structure database (CSDb) the authors recognize that large no. of kinase inhibitors of diverse chem. structures are derived from a relatively small no. of common scaffolds.  Depending on specific substitution patterns, scaffolds may demonstrate versatile binding capacities to interact with kinase hinge.  Afforded by thousands of ligand-protein binary complexes, the hinge hydrogen bond patterns were analyzed with a focus on their three-dimensional configurations.  Most of the compds. engage H6 NH for hinge recognition.  Dual hydrogen bonds are commonly obsd. with addnl. recruitment of H4 CO upstream and/or H6 CO downstream.  Triple hydrogen bonds accounts for small no. of binary complexes.  An unusual hydrogen bond with a non-canonical H5 conformation is obsd., requiring a peptide bond flip by a glycine residue at the H6 position.  Addnl. hydrogen bonds to kinase hinge do not necessarily correlate with an increase in potency; conversely they appear to compromise kinase selectivity.  Such learnings could enhance the prospect of successful therapy design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRC-xiTiNvELVg90H21EOLACvtfcHk0ljzehgRSrsFlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L&md5=048b4b99628c6e1ae273b00a47ab2377</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DKlug-Mcleod%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DB.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26atitle%3DKinase%2520hinge%2520binding%2520scaffolds%2520and%2520their%2520hydrogen%2520bond%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6520%26epage%3D6527%26doi%3D10.1016%2Fj.bmc.2015.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span> <span> </span><span class="NLM_article-title">7-Azaindole: A versatile scaffold for developing kinase inhibitors</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1248%2Fcpb.c17-00380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29311509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=29-36&author=T.+Irieauthor=M.+Sawa&title=7-Azaindole%3A+A+versatile+scaffold+for+developing+kinase+inhibitors&doi=10.1248%2Fcpb.c17-00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">7-Azaindole: a versatile scaffold for developing kinase inhibitors</span></div><div class="casAuthors">Irie, Takayuki; Sawa, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review.  This review summarized recent examples of 7-azaindole-based kinase inhibitors and discusses their binding interactions with the kinase hinge regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9QuvBOqQ8GrVg90H21EOLACvtfcHk0ljzehgRSrsFlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK&md5=69a5097cb54f2598d2e02414e74b3e6b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00380%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DT.%26aulast%3DSawa%26aufirst%3DM.%26atitle%3D7-Azaindole%253A%2520A%2520versatile%2520scaffold%2520for%2520developing%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D29%26epage%3D36%26doi%3D10.1248%2Fcpb.c17-00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherk, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frigo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnackenberg, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laping, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trizna, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">Development of a small-molecule serum- and glucocorticoid-regulated kinase-1 antagonist and its evaluation as a prostate cancer therapeutic</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">7475</span>– <span class="NLM_lpage">7483</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-08-1047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1158%2F0008-5472.CAN-08-1047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=18794135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWltb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=7475-7483&author=A.+B.+Sherkauthor=D.+E.+Frigoauthor=C.+G.+Schnackenbergauthor=J.+D.+Brayauthor=N.+J.+Lapingauthor=W.+Triznaauthor=M.+Hammondauthor=J.+R.+Pattersonauthor=S.+K.+Thompsonauthor=D.+Kazminauthor=J.+D.+Norrisauthor=D.+P.+McDonnell&title=Development+of+a+small-molecule+serum-+and+glucocorticoid-regulated+kinase-1+antagonist+and+its+evaluation+as+a+prostate+cancer+therapeutic&doi=10.1158%2F0008-5472.can-08-1047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Small-Molecule Serum- and Glucocorticoid-Regulated Kinase-1 Antagonist and Its Evaluation as a Prostate Cancer Therapeutic</span></div><div class="casAuthors">Sherk, Andrea B.; Frigo, Daniel E.; Schnackenberg, Christine G.; Bray, Jeffrey D.; Laping, Nicholas J.; Trizna, Walter; Hammond, Marlys; Patterson, Jaclyn R.; Thompson, Scott K.; Kazmin, Dmitri; Norris, John D.; McDonnell, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7475-7483</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgens, through their actions on the androgen receptor (AR), are required for the development of the prostate and contribute to the pathol. growth dysregulation obsd. in prostate cancers.  Consequently, androgen ablation has become an essential component of the pharmacotherapy of prostate cancer.  In this study, we explored the utility of targeting processes downstream of AR as an alternate approach for therapy.  Specifically, we show that the serum and glucocorticoid-regulated kinase 1 (SGK1) gene is an androgen-regulated target gene in cellular models of prostate cancer.  Furthermore, functional serum- and glucocorticoid-regulated kinase 1 (SGK1) protein, as detd. by the phosphorylation of its target Nedd4-2, was also increased with androgen treatment.  Importantly, we detd. that RNA interference-mediated knockdown of SGK1 expression attenuates the androgen-mediated growth of the prostate cancer cell line LNCaP.  Given these findings, we explored the utility of SGK1 as a therapeutic target in prostate cancer by developing and evaluating a small-mol. inhibitor of this enzyme.  From these studies emerged GSK650394, a competitive inhibitor that quant. blocks the effect of androgens on LNCaP cell growth.  Thus, in addn. to androgen ablation, inhibition of pathways downstream of AR is likely to have therapeutic utility in prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTPdHEs-8bW7Vg90H21EOLACvtfcHk0ljzehgRSrsFlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFWltb7L&md5=bd7e0bb3e7a3e0bb8b514c3cf58f0a8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1047%26sid%3Dliteratum%253Aachs%26aulast%3DSherk%26aufirst%3DA.%2BB.%26aulast%3DFrigo%26aufirst%3DD.%2BE.%26aulast%3DSchnackenberg%26aufirst%3DC.%2BG.%26aulast%3DBray%26aufirst%3DJ.%2BD.%26aulast%3DLaping%26aufirst%3DN.%2BJ.%26aulast%3DTrizna%26aufirst%3DW.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DPatterson%26aufirst%3DJ.%2BR.%26aulast%3DThompson%26aufirst%3DS.%2BK.%26aulast%3DKazmin%26aufirst%3DD.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DDevelopment%2520of%2520a%2520small-molecule%2520serum-%2520and%2520glucocorticoid-regulated%2520kinase-1%2520antagonist%2520and%2520its%2520evaluation%2520as%2520a%2520prostate%2520cancer%2520therapeutic%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D7475%26epage%3D7483%26doi%3D10.1158%2F0008-5472.can-08-1047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span> <span> </span><span class="NLM_article-title">Palladium(0)-catalyzed cross-coupling reaction of alkoxydiboron with haloarenes: a direct procedure for arylboronic esters</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7508</span>– <span class="NLM_lpage">7510</span>, <span class="refDoi"> DOI: 10.1021/jo00128a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00128a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK2MXovFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=7508-7510&author=T.+Ishiyamaauthor=M.+Murataauthor=N.+Miyaura&title=Palladium%280%29-catalyzed+cross-coupling+reaction+of+alkoxydiboron+with+haloarenes%3A+a+direct+procedure+for+arylboronic+esters&doi=10.1021%2Fjo00128a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium(0)-Catalyzed Cross-Coupling Reaction of Alkoxydiboron with Haloarenes: A Direct Procedure for Arylboronic Esters</span></div><div class="casAuthors">Ishiyama, Tatsuo; Murata, Miki; Miyaura, Norio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7508-10</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The palladium-catalyzed cross-coupling reaction of the pinacol ester of diboronic acid I with haloarenes gave a direct procedure for arylboronic esters II (Ar = aryl) from aryl halides in a range of 60-98%.  The reaction was catalyzed by PdCl2(dppf) (3 mol %) at 80° in the presence of KOAc (3 equiv) in DMSO and available with various functional groups such as nitrile, cyanide, ester, and carbonyl groups.  The trans-ArPd(II)(OAc)(PPh3)2 intermediate was isolated and characterized to propose the catalytic cycle involving the transmetalation between the phenylpalladium(II) acetate complex and I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobehJBGFjDH7Vg90H21EOLACvtfcHk0ljypwJ4spgNcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXovFSku7g%253D&md5=2666902270ec1efc5742480b133f1255</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjo00128a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00128a024%26sid%3Dliteratum%253Aachs%26aulast%3DIshiyama%26aufirst%3DT.%26aulast%3DMurata%26aufirst%3DM.%26aulast%3DMiyaura%26aufirst%3DN.%26atitle%3DPalladium%25280%2529-catalyzed%2520cross-coupling%2520reaction%2520of%2520alkoxydiboron%2520with%2520haloarenes%253A%2520a%2520direct%2520procedure%2520for%2520arylboronic%2520esters%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D7508%26epage%3D7510%26doi%3D10.1021%2Fjo00128a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merour, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and reactivity of 7-azaindoles (1H-pyrrolo(2,3-b)pyridine)</span>. <i>Curr. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.2174/1385272013375427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.2174%2F1385272013375427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFejurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=471-506&author=J.-Y.+Merourauthor=B.+Joseph&title=Synthesis+and+reactivity+of+7-azaindoles+%281H-pyrrolo%282%2C3-b%29pyridine%29&doi=10.2174%2F1385272013375427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and reactivity of 7-azaindoles (1H-pyrrolo[2,3-b]pyridines)</span></div><div class="casAuthors">Merour, Jean-Yves; Joseph, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Current Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-506</span>CODEN:
                <span class="NLM_cas:coden">CORCFE</span>;
        ISSN:<span class="NLM_cas:issn">1385-2728</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 133 refs. on research over the last 20 yr into the chem. of 7-azaindoles.  New synthetic methods for 7-azaindole structures were developed, including palladium-catalyzed cross-coupling reactions.  The reactivity of 3-formyl-, 3-halo-, and 2-lithio-7-azaindole derivs. is reported.  Functionalization of the pyridine ring via the N-oxide is also described.  In the last part, biol. activity of various 7-azaindole derivs. is briefly indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPCylwwn-8rLVg90H21EOLACvtfcHk0ljypwJ4spgNcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFejurY%253D&md5=27e3562dcfbbee1fc4f701e0ee8514e6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2174%2F1385272013375427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1385272013375427%26sid%3Dliteratum%253Aachs%26aulast%3DMerour%26aufirst%3DJ.-Y.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520reactivity%2520of%25207-azaindoles%2520%25281H-pyrrolo%25282%252C3-b%2529pyridine%2529%26jtitle%3DCurr.%2520Org.%2520Chem.%26date%3D2001%26volume%3D5%26spage%3D471%26epage%3D506%26doi%3D10.2174%2F1385272013375427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotani, S.</span></span> <span> </span><span class="NLM_article-title">Furopyridines. VI. Preparation and reactions of 2- and 3-substituted furo[2,3-b]pyridines</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570230542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1002%2Fjhet.5570230542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaL2sXksFyru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1986&pages=1465-1469&author=H.+Moritaauthor=S.+Shiotani&title=Furopyridines.+VI.+Preparation+and+reactions+of+2-+and+3-substituted+furo%5B2%2C3-b%5Dpyridines&doi=10.1002%2Fjhet.5570230542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Furopyridines.  VI.  Preparation and reactions of 2- and 3-substituted furo[2,3-b]pyridines</span></div><div class="casAuthors">Morita, Hiroyuki; Shiotani, Shunsaku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1465-9</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    </div><div class="casAbstract">The title compds. I (R, R1 = H, Me, CH2OMe, CHO, cyano; R≠R1, except H) were prepd. starting from Et 2-chloronicotinate using known methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHJmUW8Nd4dbVg90H21EOLACvtfcHk0ljypwJ4spgNcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXksFyru70%253D&md5=c50a8a83a95e4d7f789dde494493510b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570230542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570230542%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DH.%26aulast%3DShiotani%26aufirst%3DS.%26atitle%3DFuropyridines.%2520VI.%2520Preparation%2520and%2520reactions%2520of%25202-%2520and%25203-substituted%2520furo%255B2%252C3-b%255Dpyridines%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1986%26volume%3D23%26spage%3D1465%26epage%3D1469%26doi%3D10.1002%2Fjhet.5570230542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. C.</span></span> <span> </span><span class="NLM_article-title">Trifluoromethyl triflate: synthesis and reactions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4147</span>– <span class="NLM_lpage">4156</span>, <span class="refDoi"> DOI: 10.1021/jo00228a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00228a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlKjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1987&pages=4147-4156&author=S.+L.+Taylorauthor=J.+C.+Martin&title=Trifluoromethyl+triflate%3A+synthesis+and+reactions&doi=10.1021%2Fjo00228a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Trifluoromethyl triflate: synthesis and reactions.</span></div><div class="casAuthors">Taylor, S. L.; Martin, J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4147-56</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">A new and convenient synthesis of F3CSO3CF3 (I) was devised.  The addn. of triflic anhydride to a catalytic amt. of SbF5 at 25° produced I in 94% yield, utilizing a new and stronger Lewis acid catalyst produced in the reaction, F4SbOSO2CF3.  This reaction makes pure I conveniently and economically available in large quantities.  A mechanism is presented that accounts for the substitution of a triflate ligand for a fluorine ligand on Sb, as well as for the formation of I.  I does not trifluoromethylate nucleophiles (pyridine, Et3N, iodide, PhLi, PhMgBr, lithium thiophenolate, or sodium naphthalenide) but gives products that result from an initial attack of the nucleophile at S.  Fluoride ion, which was formed by fragmentation of the trifluoromethoxide ion displaced from S, is a chain carrier in the rapid decompn. of I to give F3CSO2F and fluorophosgene.  This limits the synthetic utility of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqniKWCskQxZLVg90H21EOLACvtfcHk0lhutRwabzPFSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmtlKjtrw%253D&md5=3241f9714404f33c2702770b0ed8ce12</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjo00228a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00228a001%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DS.%2BL.%26aulast%3DMartin%26aufirst%3DJ.%2BC.%26atitle%3DTrifluoromethyl%2520triflate%253A%2520synthesis%2520and%2520reactions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1987%26volume%3D52%26spage%3D4147%26epage%3D4156%26doi%3D10.1021%2Fjo00228a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lhutRwabzPFSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, P. H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massirer, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of inhibitor binding to CAMKK1 Identifies Features Necessary for Design of Specific Inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14800</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-33043-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-018-33043-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=30287839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB3czmtVajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=14800&author=A.+D.+S.+Santiagoauthor=R.+M.+Cou%C3%B1agoauthor=P.+Z.+Ramosauthor=P.+H.+C.+Godoiauthor=K.+B.+Massirerauthor=O.+Gileadiauthor=J.+M.+Elkins&title=Structural+analysis+of+inhibitor+binding+to+CAMKK1+Identifies+Features+Necessary+for+Design+of+Specific+Inhibitors&doi=10.1038%2Fs41598-018-33043-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of Inhibitor Binding to CAMKK1 Identifies Features Necessary for Design of Specific Inhibitors</span></div><div class="casAuthors">Santiago Andre da Silva; Counago Rafael M; Ramos Priscila Zonzini; Godoi Paulo H C; Massirer Katlin B; Elkins Jonathan M; Counago Rafael M; Massirer Katlin B; Gileadi Opher; Elkins Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14800</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The calcium/calmodulin-dependent protein kinases (CAMKKs) are upstream activators of CAMK1 and CAMK4 signalling and have important functions in neural development, maintenance and signalling, as well as in other aspects of biology such as Ca(2+) signalling in the cardiovascular system.  To support the development of specific inhibitors of CAMKKs we have determined the crystal structure of CAMKK1 with two ATP-competitive inhibitors.  The structures reveal small but exploitable differences between CAMKK1 and CAMKK2, despite the high sequence identity, which could be used in the generation of specific inhibitors.  Screening of a kinase inhibitor library revealed molecules that bind potently to CAMKK1.  Isothermal titration calorimetry revealed that the most potent inhibitors had binding energies largely dependent on favourable enthalpy.  Together, the data provide a foundation for future inhibitor development activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTiTkdNpgO7swHqQV89nF-2fW6udTcc2eZTj_6lQLWF7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czmtVajug%253D%253D&md5=3757fbb8f8e0e9ba37c3bb0520594787</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-33043-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-33043-4%26sid%3Dliteratum%253Aachs%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26aulast%3DRamos%26aufirst%3DP.%2BZ.%26aulast%3DGodoi%26aufirst%3DP.%2BH.%2BC.%26aulast%3DMassirer%26aufirst%3DK.%2BB.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26atitle%3DStructural%2520analysis%2520of%2520inhibitor%2520binding%2520to%2520CAMKK1%2520Identifies%2520Features%2520Necessary%2520for%2520Design%2520of%2520Specific%2520Inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D14800%26doi%3D10.1038%2Fs41598-018-33043-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhutRwabzPFSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Profeta, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Reis, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, A. D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoi, P. H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fala, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmazo, A. P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massirer, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couñago, R. M.</span></span> <span> </span><span class="NLM_article-title">Binding and structural analyses of potent inhibitors of the human Ca(2+)/calmodulin dependent protein kinase kinase 2 (CAMKK2) identified from a collection of commercially-available kinase inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">16452</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-52795-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fs41598-019-52795-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=31712618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB3Mjnsleqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=16452&author=G.+S.+Profetaauthor=C.+V.+Dos+Reisauthor=A.+D.+S.+Santiagoauthor=P.+H.+C.+Godoiauthor=A.+M.+Falaauthor=C.+I.+Wellsauthor=R.+Sartoriauthor=A.+P.+T.+Salmazoauthor=P.+Z.+Ramosauthor=K.+B.+Massirerauthor=J.+M.+Elkinsauthor=D.+H.+Drewryauthor=O.+Gileadiauthor=R.+M.+Cou%C3%B1ago&title=Binding+and+structural+analyses+of+potent+inhibitors+of+the+human+Ca%282%2B%29%2Fcalmodulin+dependent+protein+kinase+kinase+2+%28CAMKK2%29+identified+from+a+collection+of+commercially-available+kinase+inhibitors&doi=10.1038%2Fs41598-019-52795-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Binding and structural analyses of potent inhibitors of the human Ca(2+)/calmodulin dependent protein kinase kinase 2 (CAMKK2) identified from a collection of commercially-available kinase inhibitors</span></div><div class="casAuthors">Profeta Gerson S; Dos Reis Caio V; Santiago Andre da S; Godoi Paulo H C; Fala Angela M; Ramos Priscila Z; Massirer Katlin B; Counago Rafael M; Profeta Gerson S; Dos Reis Caio V; Santiago Andre da S; Godoi Paulo H C; Fala Angela M; Sartori Roger; Salmazo Anita P T; Ramos Priscila Z; Massirer Katlin B; Elkins Jonathan M; Counago Rafael M; Wells Carrow I; Drewry David H; Elkins Jonathan M; Gileadi Opher</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16452</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Calcium/Calmodulin-dependent Protein Kinase Kinase 2 (CAMKK2) acts as a signaling hub, receiving signals from various regulatory pathways and decoding them via phosphorylation of downstream protein kinases - such as AMPK (AMP-activated protein kinase) and CAMK types I and IV.  CAMKK2 relevance is highlighted by its constitutive activity being implicated in several human pathologies.  However, at present, there are no selective small-molecule inhibitors available for this protein kinase.  Moreover, CAMKK2 and its closest human homolog, CAMKK1, are thought to have overlapping biological roles.  Here we present six new co-structures of potent ligands bound to CAMKK2 identified from a library of commercially-available kinase inhibitors.  Enzyme assays confirmed that most of these compounds are equipotent inhibitors of both human CAMKKs and isothermal titration calorimetry (ITC) revealed that binding to some of these molecules to CAMKK2 is enthalpy driven.  We expect our results to advance current efforts to discover small molecule kinase inhibitors selective to each human CAMKK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4tR5q1L6mjO-MdeMhXAHkfW6udTcc2ea_U6ctS364D7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mjnsleqtg%253D%253D&md5=d0605c597cecd456b71a85e88e45b38f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-52795-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-52795-1%26sid%3Dliteratum%253Aachs%26aulast%3DProfeta%26aufirst%3DG.%2BS.%26aulast%3DDos%2BReis%26aufirst%3DC.%2BV.%26aulast%3DSantiago%26aufirst%3DA.%2BD.%2BS.%26aulast%3DGodoi%26aufirst%3DP.%2BH.%2BC.%26aulast%3DFala%26aufirst%3DA.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DSartori%26aufirst%3DR.%26aulast%3DSalmazo%26aufirst%3DA.%2BP.%2BT.%26aulast%3DRamos%26aufirst%3DP.%2BZ.%26aulast%3DMassirer%26aufirst%3DK.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DCou%25C3%25B1ago%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520and%2520structural%2520analyses%2520of%2520potent%2520inhibitors%2520of%2520the%2520human%2520Ca%25282%252B%2529%252Fcalmodulin%2520dependent%2520protein%2520kinase%2520kinase%25202%2520%2528CAMKK2%2529%2520identified%2520from%2520a%2520collection%2520of%2520commercially-available%2520kinase%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D16452%26doi%3D10.1038%2Fs41598-019-52795-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgEpT-wFrByzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beno, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">A Survey of the role of noncovalent sulfur interactions in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4383</span>– <span class="NLM_lpage">4438</span>, <span class="refDoi"> DOI: 10.1021/jm501853m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501853m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Giu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4383-4438&author=B.+R.+Benoauthor=K.-S.+Yeungauthor=M.+D.+Bartbergerauthor=L.+D.+Penningtonauthor=N.+A.+Meanwell&title=A+Survey+of+the+role+of+noncovalent+sulfur+interactions+in+drug+design&doi=10.1021%2Fjm501853m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Survey of the Role of Noncovalent Sulfur Interactions in Drug Design</span></div><div class="casAuthors">Beno, Brett R.; Yeung, Kap-Sun; Bartberger, Michael D.; Pennington, Lewis D.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4383-4438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Electron deficient, bivalent sulfur atoms have two areas of pos. electrostatic potential, a consequence of the low-lying σ* orbitals of the C-S bond that are available for interaction with electron donors including oxygen and nitrogen atoms and, possibly, π-systems.  Intramol. interactions are by far the most common manifestation of this effect, which offers a means of modulating the conformational preferences of a mol.  Although a well-documented phenomenon, a priori applications in drug design are relatively sparse and this interaction, which is often isosteric with an intramol. hydrogen-bonding interaction, appears to be underappreciated by the medicinal chem. community.  In this Perspective, we discuss the theor. basis for sulfur σ* orbital interactions and illustrate their importance in the context of drug design and org. synthesis.  The role of sulfur interactions in protein structure and function is discussed and although relatively rare, intermol. interactions between ligand C-S σ* orbitals and proteins are illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYKXRsl4BQYrVg90H21EOLACvtfcHk0lgEpT-wFrByzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Giu7g%253D&md5=d12f53eb3271c161cfc833fb9f7c02b3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm501853m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501853m%26sid%3Dliteratum%253Aachs%26aulast%3DBeno%26aufirst%3DB.%2BR.%26aulast%3DYeung%26aufirst%3DK.-S.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DPennington%26aufirst%3DL.%2BD.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DA%2520Survey%2520of%2520the%2520role%2520of%2520noncovalent%2520sulfur%2520interactions%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4383%26epage%3D4438%26doi%3D10.1021%2Fjm501853m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Intermolecular sulfur...oxygen interactions: theoretical and statistical investigations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2138</span>– <span class="NLM_lpage">2153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2138-2153&author=X.+Zhangauthor=Z.+Gongauthor=J.+Liauthor=T.+Lu&title=Intermolecular+sulfur...oxygen+interactions%3A+theoretical+and+statistical+investigations&doi=10.1021%2Facs.jcim.5b00177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span></div><div class="casAuthors">Zhang, Xuejin; Gong, Zhen; Li, Jian; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2138-2153</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intermol. S···O interactions are very common and are important in biol. systems, but until recently, the presence of these contacts in protein-ligand systems largely depended on serendipitous discovery instead of rational design.  Here we provide insight into the phenomenon of intermol. S···O contacts by focusing on three sulfur-contg. arom. rings.  Quantum mechanics is employed to characterize the strength and directionality of the S···O interactions and to det. their energy dependence on their geometric parameters.  Protein Data Bank mining is performed to systematically det. the occurrence and geometry of intermol. S···O interactions, and several representative examples are discussed.  Three typical cases are investigated using a combined quantum mechanics/mol. mechanics approach to demonstrate the potential of these interactions in improving binding affinities and physiochem. properties.  Overall, our work elucidates the structures and energy features of intermol. S···O interactions and addresses their use in mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2J_2Nq9PDQLVg90H21EOLACvtfcHk0lhyp8efDlzaCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI&md5=7192e3b57e66fc6eb9dd53165427d449</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00177%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DIntermolecular%2520sulfur...oxygen%2520interactions%253A%2520theoretical%2520and%2520statistical%2520investigations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2138%26epage%3D2153%26doi%3D10.1021%2Facs.jcim.5b00177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span> <span> </span><span class="NLM_article-title">STO-609, a specific inhibitor of the Ca(2+)/calmodulin-dependent protein kinase kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">15813</span>– <span class="NLM_lpage">15818</span>, <span class="refDoi"> DOI: 10.1074/jbc.m201075200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M201075200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=11867640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjs1Wjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=15813-15818&author=H.+Tokumitsuauthor=H.+Inuzukaauthor=Y.+Ishikawaauthor=M.+Ikedaauthor=I.+Sajiauthor=R.+Kobayashi&title=STO-609%2C+a+specific+inhibitor+of+the+Ca%282%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase&doi=10.1074%2Fjbc.m201075200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">STO-609, a specific inhibitor of the Ca2+/calmodulin-dependent protein kinase kinase</span></div><div class="casAuthors">Tokumitsu, Hiroshi; Inuzuka, Hiroyuki; Ishikawa, Yumi; Ikeda, Masahiko; Saji, Ikutaro; Kobayashi, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">15813-15818</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">STO-609, a selective inhibitor of Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK) was synthesized, and its inhibitory properties were investigated both in vitro and in vivo.  STO-609 inhibits the activities of recombinant CaM-KKα and CaM-KKβ isoforms, with Ki values of 80 and 15 ng/mL, resp., and also inhibits their autophosphorylation activities.  Comparison of the inhibitory potency of the compd. against various protein kinases revealed that STO-609 is highly selective for CaM-KK without any significant effect on the downstream CaM kinases (CaM-KI and -IV), and the IC50 value of the compd. against CaM-KII is ∼ 10 μg/mL.  STO-609 inhibits constitutively active CaM-KKα (glutathione S-transferase (GST)-CaM-KK-(84-434)) as well as the wild-type enzyme.  Kinetic anal. indicates that the compd. is a competitive inhibitor of ATP.  In transfected HeLa cells, STO-609 suppresses the Ca2+-induced activation of CaM-KIV in a dose-dependent manner.  In agreement with this observation, the inhibitor significantly reduces the endogenous activity of CaM-KK in SH-SY5Y neuroblastoma cells at a concn. of 1 μg/mL (∼80% inhibitory rate).  Taken together, these results indicate that STO-609 is a selective and cell-permeable inhibitor of CaM-KK and that it may be a useful tool for evaluating the physiol. significance of the CaM-KK-mediated pathway in vivo as well as in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOo8PRZ4cKCbVg90H21EOLACvtfcHk0lhyp8efDlzaCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjs1Wjsbs%253D&md5=03e436ce8278669ff528ecad280d4955</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M201075200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M201075200%26sid%3Dliteratum%253Aachs%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DSaji%26aufirst%3DI.%26aulast%3DKobayashi%26aufirst%3DR.%26atitle%3DSTO-609%252C%2520a%2520specific%2520inhibitor%2520of%2520the%2520Ca%25282%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D15813%26epage%3D15818%26doi%3D10.1074%2Fjbc.m201075200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokumitsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span> <span> </span><span class="NLM_article-title">A single amino acid difference between alpha and beta Ca2+/calmodulin-dependent protein kinase kinase dictates sensitivity to the specific inhibitor, STO-609</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">10908</span>– <span class="NLM_lpage">10913</span>, <span class="refDoi"> DOI: 10.1074/jbc.m213183200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M213183200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=12540834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1Khurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=10908-10913&author=H.+Tokumitsuauthor=H.+Inuzukaauthor=Y.+Ishikawaauthor=R.+Kobayashi&title=A+single+amino+acid+difference+between+alpha+and+beta+Ca2%2B%2Fcalmodulin-dependent+protein+kinase+kinase+dictates+sensitivity+to+the+specific+inhibitor%2C+STO-609&doi=10.1074%2Fjbc.m213183200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Single Amino Acid Difference between α and β Ca2+/Calmodulin-dependent Protein Kinase Kinase Dictates Sensitivity to the Specific Inhibitor, STO-609</span></div><div class="casAuthors">Tokumitsu, Hiroshi; Inuzuka, Hiroyuki; Ishikawa, Yumi; Kobayashi, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10908-10913</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We recently developed STO-609, a selective inhibitor of Ca2+/calmodulin-dependent protein kinase kinase (CaM-KK), and we demonstrated that CaM-KKβ is more sensitive to STO-609 than the CaM-KKα isoform.  By using catalytic chimera and point mutants of both isoforms, we demonstrated that Val269 in CaM-KKβ/Leu233 in CaM-KKα confers a distinct sensitivity (~10-fold) to STO-609 on CaM-KK isoforms.  Various mutations of Val269 in CaM-KKβ indicate that substitution by hydrophobic residues with bulky side chains significantly decreases drug sensitivity and that the V269F mutant is the most effective drug-resistant enzyme (~80-fold higher IC50 value).  These findings are consistent with a result obtained with a full-length mutant expressed in COS-7 cells.  Furthermore, suppression of CaM-KK-mediated CaM-KIV activation in transfected HeLa cells by STO-609 treatment was completely abolished by the co-expression of the CaM-KKβ V269F mutant.  Based on the results that the distinct sensitivity of CaM-KK isoforms to STO-609 is because of a single amino acid substitution (Val/Leu) in the ATP-binding pocket, we have generated an STO-609-resistant CaM-KK mutant, which might be useful for validating the pharmacol. effects and specificity of STO-609 in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5Fl3MmunxrVg90H21EOLACvtfcHk0ljni-judAMlhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1Khurk%253D&md5=6a372bc54100e90b7d37dfe2935ea2a8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M213183200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M213183200%26sid%3Dliteratum%253Aachs%26aulast%3DTokumitsu%26aufirst%3DH.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DKobayashi%26aufirst%3DR.%26atitle%3DA%2520single%2520amino%2520acid%2520difference%2520between%2520alpha%2520and%2520beta%2520Ca2%252B%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520dictates%2520sensitivity%2520to%2520the%2520specific%2520inhibitor%252C%2520STO-609%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D10908%26epage%3D10913%26doi%3D10.1074%2Fjbc.m213183200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span> <span> </span><span class="NLM_article-title">High throughput solubility measurement in drug discovery and development</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.addr.2007.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=17604872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=546-567&author=J.+Alsenzauthor=M.+Kansy&title=High+throughput+solubility+measurement+in+drug+discovery+and+development&doi=10.1016%2Fj.addr.2007.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput solubility measurement in drug discovery and development</span></div><div class="casAuthors">Alsenz, Jochem; Kansy, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">546-567</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Measurement of drug soly. in various solvents is one of the key elements of compd. characterization during the whole discovery and development process.  This review summarizes current exptl. approaches and addresses recent advances in the exptl. methods used to det. drug soly. in drug discovery and early development.  This paper focuses on high throughput methods designed to det. kinetic and thermodn. (equil.) soly. but traditional methods are also presented.  The focus, positioning, exptl. setup, pros and cons, and limitations of individual assays are discussed and differences in soly. studies in discovery and development environments are highlighted.  Finally, future needs and trends in soly. assay development designed to overcome current bottlenecks and trade-offs between speed and quality/quantity of measurements are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsvOlLd-yoxLVg90H21EOLACvtfcHk0ljni-judAMlhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D&md5=8d55ead3ea0c13fa49b032db725842b0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3DKansy%26aufirst%3DM.%26atitle%3DHigh%2520throughput%2520solubility%2520measurement%2520in%2520drug%2520discovery%2520and%2520development%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D546%26epage%3D567%26doi%3D10.1016%2Fj.addr.2007.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0ljni-judAMlhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2010%26volume%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2Fs0169-409x%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lh7UOnR5J6IIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2Fs0169-409x%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majoral, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Present drug-likeness filters in medicinal chemistry during the hit and lead optimization process: how far can they be simplified?</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">615</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.drudis.2018.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29330127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BC1MvgtlCksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=605-615&author=S.+Mignaniauthor=J.+Rodriguesauthor=H.+Tomasauthor=R.+Jalalauthor=P.+P.+Singhauthor=J.-P.+Majoralauthor=R.+A.+Vishwakarma&title=Present+drug-likeness+filters+in+medicinal+chemistry+during+the+hit+and+lead+optimization+process%3A+how+far+can+they+be+simplified%3F&doi=10.1016%2Fj.drudis.2018.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Present drug-likeness filters in medicinal chemistry during the hit and lead optimization process: how far can they be simplified?</span></div><div class="casAuthors">Mignani Serge; Rodrigues Joao; Tomas Helena; Jalal Rachid; Singh Parvinder Pal; Vishwakarma Ram A; Majoral Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">605-615</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">During the past decade, decreasing the attrition rate of drug development candidates reaching the market has become one of the major challenges in pharmaceutical research and drug development (R&D).  To facilitate the decision-making process, and to increase the probability of rapidly finding and developing high-quality compounds, a variety of multiparametric guidelines, also known as rules and ligand efficiency (LE) metrics, have been developed.  However, what are the 'best' descriptors and how far can we simplify these drug-likeness prediction tools in terms of the numerous, complex properties that they relate to?</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2XT6-2EQJ3n6oPH0mYWL1fW6udTcc2eZgg9pkfn8oe7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvgtlCksw%253D%253D&md5=a61e00552a35b455199f0a8d351d0522</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DMignani%26aufirst%3DS.%26aulast%3DRodrigues%26aufirst%3DJ.%26aulast%3DTomas%26aufirst%3DH.%26aulast%3DJalal%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DP.%2BP.%26aulast%3DMajoral%26aufirst%3DJ.-P.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DPresent%2520drug-likeness%2520filters%2520in%2520medicinal%2520chemistry%2520during%2520the%2520hit%2520and%2520lead%2520optimization%2520process%253A%2520how%2520far%2520can%2520they%2520be%2520simplified%253F%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2018%26volume%3D23%26spage%3D605%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2018.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lh7UOnR5J6IIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petereit, A. C.</span></span> <span> </span><span class="NLM_article-title">Optimizing solubility: kinetic versus thermodynamic solubility temptations and risks</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.ejps.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=22885099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=589-595&author=C.+Saalauthor=A.+C.+Petereit&title=Optimizing+solubility%3A+kinetic+versus+thermodynamic+solubility+temptations+and+risks&doi=10.1016%2Fj.ejps.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing solubility: Kinetic versus thermodynamic solubility temptations and risks</span></div><div class="casAuthors">Saal, Christoph; Petereit, Anna Christine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to assess the usefulness of kinetic and thermodn. soly. data in guiding medicinal chem. during lead optimization.  The soly. of 465 research compds. was measured using a kinetic and a thermodn. soly. assay.  In the thermodn. assay, polarized-light microscopy was used to investigate whether the result referred to the cryst. or to the amorphous compd.  From the comparison of kinetic and thermodn. soly. data it was noted that kinetic soly. measurements frequently yielded results which show considerably higher soly. compared to thermodn. soly.  This observation is ascribed to the fact that a kinetic soly. assay typically delivers results which refer to the amorphous compd.  In contrast, results from thermodn. soly. detns. more frequently refer to a cryst. phase.  Accordingly, thermodn. soly. data - esp. when used together with an assessment of the solid state form - are deemed to be more useful in guiding soly. optimization for research compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxKTQHBpynbVg90H21EOLACvtfcHk0lji6ipHtWLNfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ&md5=ace8b962c04c3a62f028ec3e66522922</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DSaal%26aufirst%3DC.%26aulast%3DPetereit%26aufirst%3DA.%2BC.%26atitle%3DOptimizing%2520solubility%253A%2520kinetic%2520versus%2520thermodynamic%2520solubility%2520temptations%2520and%2520risks%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2012%26volume%3D47%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fj.ejps.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lji6ipHtWLNfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Means, A. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of the CaMKKbeta-AMPK signaling complex</span>. <i>Cell. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2005</span>– <span class="NLM_lpage">2012</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2011.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1016%2Fj.cellsig.2011.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21807092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kku7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2011&pages=2005-2012&author=M.+F.+Greenauthor=K.+A.+Andersonauthor=A.+R.+Means&title=Characterization+of+the+CaMKKbeta-AMPK+signaling+complex&doi=10.1016%2Fj.cellsig.2011.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CaMKKβ-AMPK signaling complex</span></div><div class="casAuthors">Green, Michelle F.; Anderson, Kristin A.; Means, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2005-2012</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The AMP-activated protein kinase (AMPK) is a crit. regulator of energy homeostasis, and is a potential target for treatment of metabolic diseases as well as cancer.  AMPK can be phosphorylated and activated by the tumor suppressor LKB1 or the Ca2+/CaM-dependent protein kinase kinase β (CaMKKβ).  We previously identified a phys. complex between CaMKKβ and AMPK (Anderson, K. A., Ribar, T. J., Lin, F., Noeldner, P. K., Green, M. F., Muehlbauer, M. J., Witters, L. A., Kemp, B. E., and Means, A. R. (2008) Cell Metab. 7, 377-388).  Here we expand our anal. of the CaMKKβ-AMPK signaling complex and show that whereas CaMKKβ can form a complex with and activate AMPK, CaMKKα cannot.  In addn., we show that CaMKKβ and AMPK assoc. through their kinase domains, and CaMKKβ must be in an active conformation in order to bind AMPK but not to assoc. with an alternative substrate, Ca2+/Calmodulin-dependent protein kinase IV (CaMKIV).  Our results demonstrate that CaMKKβ and AMPK form a unique signaling complex.  This raises the possibility that the CaMKKβ-AMPK complex can be specifically targeted by small mol. drugs to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBivJ0b2Ntj7Vg90H21EOLACvtfcHk0lji6ipHtWLNfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kku7zN&md5=edbf3362e408d902b4a580e96ad03207</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2011.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2011.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DM.%2BF.%26aulast%3DAnderson%26aufirst%3DK.%2BA.%26aulast%3DMeans%26aufirst%3DA.%2BR.%26atitle%3DCharacterization%2520of%2520the%2520CaMKKbeta-AMPK%2520signaling%2520complex%26jtitle%3DCell.%2520Signal.%26date%3D2011%26volume%3D23%26spage%3D2005%26epage%3D2012%26doi%3D10.1016%2Fj.cellsig.2011.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0li5lb5P9gYe0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xy
Ling, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langendorf, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundlach, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakhill, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. W.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of an R311C variant of Ca(2+) -calmodulin-dependent protein kinase kinase-2 (CaMKK2) found as a de novo mutation in a patient with bipolar disorder</span>. <i>Bipolar Disord.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1111/bdi.12901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1111%2Fbdi.12901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=32216002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A280%3ADC%252BB383nvVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=841-848&author=N.+Xy%0ALingauthor=C.+G.+Langendorfauthor=A.+Hoqueauthor=S.+Galicauthor=K.+Lohauthor=B.+E.+Kempauthor=A.+L.+Gundlachauthor=J.+S.+Oakhillauthor=J.+W.+Scott&title=Functional+analysis+of+an+R311C+variant+of+Ca%282%2B%29+-calmodulin-dependent+protein+kinase+kinase-2+%28CaMKK2%29+found+as+a+de+novo+mutation+in+a+patient+with+bipolar+disorder&doi=10.1111%2Fbdi.12901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of an R311C variant of Ca(2+) -calmodulin-dependent protein kinase kinase-2 (CaMKK2) found as a de novo mutation in a patient with bipolar disorder</span></div><div class="casAuthors">Xy Ling Naomi; Langendorf Christopher G; Hoque Ashfaqul; Galic Sandra; Loh Kim; Kemp Bruce E; Oakhill Jonathan S; Scott John W; Kemp Bruce E; Oakhill Jonathan S; Scott John W; Gundlach Andrew L; Scott John W</div><div class="citationInfo"><span class="NLM_cas:title">Bipolar disorders</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">841-848</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Loss-of-function mutations in the gene encoding the calcium-calmodulin (Ca(2+) -CaM)-dependent protein kinase kinase-2 (CaMKK2) enzyme are linked to bipolar disorder.  Recently, a de novo arginine to cysteine (R311C) mutation in CaMKK2 was identified from a whole exome sequencing study of bipolar patients and their unaffected parents.  The aim of the present study was to determine the functional consequences of the R311C mutation on CaMKK2 activity and regulation by Ca(2+) -CaM.  METHODS:  The effects of the R311C mutation on CaMKK2 activity and Ca(2+) -CaM activation were examined using a radiolabeled adenosine triphosphate (ATP) kinase assay.  We performed immunoblot analysis to determine whether the R311C mutation impacts threonine-85 (T85) autophosphorylation, an activating phosphorylation site on CaMKK2 that has also been implicated in bipolar disorder.  We also expressed the R311C mutant in CaMKK2 knockout HAP1 cells and used immunoblot analysis and an MTS reduction assay to study its effects on Ca(2+) -dependent downstream signaling and cell viability, respectively.  RESULTS:  The R311C mutation maps to the conserved HRD motif within the catalytic loop of CaMKK2 and caused a marked reduction in kinase activity and Ca(2+) -CaM activation.  The R311C mutation virtually abolished T85 autophosphorylation in response to Ca(2+) -CaM and exerted a dominant-negative effect in cells as it impaired the ability of wild-type CaMKK2 to initiate downstream signaling and maintain cell viability.  CONCLUSIONS:  The highly disruptive, loss-of-function impact of the de novo R311C mutation in human CaMKK2 provides a compelling functional rationale for being considered a potential rare monogenic cause of bipolar disorder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0-Sz3Da-0oGHOrQ6qeX-mfW6udTcc2eZCbSBHS21Yzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383nvVSquw%253D%253D&md5=73d279451881e35e7b8ac881bf5583da</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1111%2Fbdi.12901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbdi.12901%26sid%3Dliteratum%253Aachs%26aulast%3DXy%2BLing%26aufirst%3DN.%26aulast%3DLangendorf%26aufirst%3DC.%2BG.%26aulast%3DHoque%26aufirst%3DA.%26aulast%3DGalic%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DK.%26aulast%3DKemp%26aufirst%3DB.%2BE.%26aulast%3DGundlach%26aufirst%3DA.%2BL.%26aulast%3DOakhill%26aufirst%3DJ.%2BS.%26aulast%3DScott%26aufirst%3DJ.%2BW.%26atitle%3DFunctional%2520analysis%2520of%2520an%2520R311C%2520variant%2520of%2520Ca%25282%252B%2529%2520-calmodulin-dependent%2520protein%2520kinase%2520kinase-2%2520%2528CaMKK2%2529%2520found%2520as%2520a%2520de%2520novo%2520mutation%2520in%2520a%2520patient%2520with%2520bipolar%2520disorder%26jtitle%3DBipolar%2520Disord.%26date%3D2020%26volume%3D22%26spage%3D841%26epage%3D848%26doi%3D10.1111%2Fbdi.12901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span> <span> </span><span class="NLM_article-title">Novel buffer systems for macromolecular crystallization</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1107/s0907444903029640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444903029640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=14993709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1OrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=610-612&author=J.+Newman&title=Novel+buffer+systems+for+macromolecular+crystallization&doi=10.1107%2Fs0907444903029640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel buffer systems for macromolecular crystallization</span></div><div class="casAuthors">Newman, Janet</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">610-612</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">In protein crystn., screening is initially performed to obtain an indication of the conditions under which a macromol. might crystallize.  These preliminary conditions are then optimized to produce (in a perfect world) well diffracting crystals; this process of optimization often involves fine grid screening around the initial conditions.  An issue in optimization is to find factors which are independent, so as to simplify the anal. of the results of optimization trials.  This is necessarily difficult with buffers, as a buffer and its pH range tend to be very highly correlated.  Multi-buffer systems for pH modulation are presented which enable a broad pH range to be sampled without changing the chem. compn. of the buffering component.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBsN7Yw2gPhbVg90H21EOLACvtfcHk0li5lb5P9gYe0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1OrtLY%253D&md5=2153f6feed4c3e2ca920e5f10d25d521</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0907444903029640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903029640%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%26atitle%3DNovel%2520buffer%2520systems%2520for%2520macromolecular%2520crystallization%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D610%26epage%3D612%26doi%3D10.1107%2Fs0907444903029640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lh1Mq4CaoZMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh1Mq4CaoZMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh1Mq4CaoZMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomabechi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada-Iwabu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwabu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minokoshi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Ca(2)(+)/calmodulin-dependent protein kinase kinase in complex with the inhibitor STO-609</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">22570</span>– <span class="NLM_lpage">22579</span>, <span class="refDoi"> DOI: 10.1074/jbc.m111.251710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1074%2Fjbc.M111.251710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21504895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1Kkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=22570-22579&author=M.+Kukimoto-Niinoauthor=S.+Yoshikawaauthor=T.+Takagiauthor=N.+Ohsawaauthor=Y.+Tomabechiauthor=T.+Teradaauthor=M.+Shirouzuauthor=A.+Suzukiauthor=S.+Leeauthor=T.+Yamauchiauthor=M.+Okada-Iwabuauthor=M.+Iwabuauthor=T.+Kadowakiauthor=Y.+Minokoshiauthor=S.+Yokoyama&title=Crystal+structure+of+the+Ca%282%29%28%2B%29%2Fcalmodulin-dependent+protein+kinase+kinase+in+complex+with+the+inhibitor+STO-609&doi=10.1074%2Fjbc.m111.251710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Ca2+/calmodulin-dependent protein kinase kinase in complex with the inhibitor STO-609</span></div><div class="casAuthors">Kukimoto-Niino, Mutsuko; Yoshikawa, Seiko; Takagi, Tetsuo; Ohsawa, Noboru; Tomabechi, Yuri; Terada, Takaho; Shirouzu, Mikako; Suzuki, Atsushi; Lee, Suni; Yamauchi, Toshimasa; Okada-Iwabu, Miki; Iwabu, Masato; Kadowaki, Takashi; Minokoshi, Yasuhiko; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">22570-22579</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ca2+/calmodulin (CaM)-dependent protein kinase (CaMK) kinase (calmodulin kinase kinase; CaMKK) is a member of the CaMK cascade that mediates the response to intracellular Ca2+ elevation.  CaMKK phosphorylates and activates CaMKI and CaMKIV, which directly activate transcription factors.  Here, the authors detd. the 2.4-Å crystal structure of the catalytic kinase domain of the human CaMKK β-isoform (CaMKKβ) complexed with its selective inhibitor, STO-609.  The structure revealed that CaMKKβ lacked the αD helix and that the equiv. region displayed a hydrophobic mol. surface, which may reflect its unique substrate recognition and autoinhibition.  Although CaMKKβ lacked the activation loop phosphorylation site, the activation loop was folded in an active-state conformation, which was stabilized by a no. of interactions between amino acid residues conserved among the CaMKK isoforms.  An in vitro anal. of the kinase activity confirmed the intrinsic activity of the CaMKKβ kinase domain.  Structure and sequence analyses of the STO-609-binding site revealed amino acid replacements that may affect the inhibitor binding.  Indeed, mutagenesis demonstrated that the CaMKKβ residue Pro-274, which replaced the conserved acidic residue of other protein kinases, was an important determinant for the selective inhibition by STO-609.  Therefore, the present structure provides a mol. basis for clarifying the known biochem. properties of CaMKKβ and for designing novel inhibitors targeting CaMKKβ and the related protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFyEDOtLOpbVg90H21EOLACvtfcHk0ljVQsPXcnYQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1Kkt7o%253D&md5=ff72e627808eb4880e34c847767295d3</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.251710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.251710%26sid%3Dliteratum%253Aachs%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DS.%26aulast%3DTakagi%26aufirst%3DT.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DTomabechi%26aufirst%3DY.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYamauchi%26aufirst%3DT.%26aulast%3DOkada-Iwabu%26aufirst%3DM.%26aulast%3DIwabu%26aufirst%3DM.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DMinokoshi%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Ca%25282%2529%2528%252B%2529%252Fcalmodulin-dependent%2520protein%2520kinase%2520kinase%2520in%2520complex%2520with%2520the%2520inhibitor%2520STO-609%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D22570%26epage%3D22579%26doi%3D10.1074%2Fjbc.m111.251710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0ljVQsPXcnYQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljVQsPXcnYQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljd_0a5clm32A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKU','PDB','6BKU'); return false;">PDB: 6BKU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CMJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CMJ','PDB','6CMJ'); return false;">PDB: 6CMJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UY6','PDB','5UY6'); return false;">PDB: 5UY6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UYJ','PDB','5UYJ'); return false;">PDB: 5UYJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZV2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZV2','PDB','2ZV2'); return false;">PDB: 2ZV2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i108"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02274">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_46027"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02274?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02274</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Mass spectrometry methods, compound characterization spectra, CAMKK2 in cell target engagement NanoBRET data, <i>p</i>-AMPK Western blot images, details on docking studies, and treespot images of kinase selectivity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Compound selectivity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_002.csv">CSV</a>)</p></li><li><p class="inline">Compound molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_003.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_001.pdf">jm0c02274_si_001.pdf (31.51 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_002.csv">jm0c02274_si_002.csv (3.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02274/suppl_file/jm0c02274_si_003.xlsx">jm0c02274_si_003.xlsx (41.42 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02274&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02274%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02274" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a14852e303cb2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
