
<html lang="en"     class="pb-page"  data-request-id="d51da3e7-f1aa-4280-a783-c01d716d83ee"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/jm201217q;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2012.55.issue-5"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Binding Affinity Prediction for Ligands and Receptors Forming Tautomers and Ionization Species: Inhibition of Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MK2)" /></meta><meta name="dc.Creator" content="Senthil  Natesan" /></meta><meta name="dc.Creator" content="Rajesh  Subramaniam" /></meta><meta name="dc.Creator" content="Charles  Bergeron" /></meta><meta name="dc.Creator" content="Stefan  Balaz" /></meta><meta name="dc.Description" content="Treatment of ionization and tautomerism of ligands and receptors is one of the unresolved issues in structure-based prediction of binding affinities. Our solution utilizes the thermodynamic master ..." /></meta><meta name="Description" content="Treatment of ionization and tautomerism of ligands and receptors is one of the unresolved issues in structure-based prediction of binding affinities. Our solution utilizes the thermodynamic master ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 17, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm201217q" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm201217q" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm201217q" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm201217q" /></link>
        
    
    

<title>Binding Affinity Prediction for Ligands and Receptors Forming Tautomers and Ionization Species: Inhibition of Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MK2) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm201217q" /></meta><meta property="og:title" content="Binding Affinity Prediction for Ligands and Receptors Forming Tautomers and Ionization Species: Inhibition of Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MK2)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0012.jpeg" /></meta><meta property="og:description" content="Treatment of ionization and tautomerism of ligands and receptors is one of the unresolved issues in structure-based prediction of binding affinities. Our solution utilizes the thermodynamic master equation, expressing the experimentally observed association constant as the sum of products, each valid for a specific ligand–receptor species pair, consisting of the association microconstant and the fractions of the involved ligand and receptor species. The microconstants are characterized by structure-based simulations, which are run for individual species pairs. Here we incorporated the multispecies approach into the QM/MM linear response method and used it for structural correlation of published inhibition data on mitogen-activated protein kinase (MAPK)-activated protein kinase (MK2) by 66 benzothiophene and pyrrolopyridine analogues, forming up to five tautomers and seven ionization species under experimental conditions. Extensive cross-validation showed that the resulting models were stable and predictive. Inclusion of all tautomers and ionization ligand species was essential: the explained variance increased to 90% from 66% for the single-species model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm201217q"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm201217q">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm201217q&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm201217q&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm201217q&amp;href=/doi/10.1021/jm201217q" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2035-2047</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm2011595" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm201218m" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Binding Affinity Prediction for Ligands and Receptors Forming Tautomers and Ionization Species: Inhibition of Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MK2)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Senthil++Natesan">Senthil Natesan</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rajesh++Subramaniam">Rajesh Subramaniam</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Charles++Bergeron">Charles Bergeron</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Balaz">Stefan Balaz</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Pharmaceutical Sciences, Albany College of Pharmacy and Health Sciences, Vermont Campus, 261 Mountain View Drive, Colchester, Vermont 05446, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 802-735-2615. E-mail: <a href="/cdn-cgi/l/email-protection#5e2d2a3b383f30703c3f323f241e3f3d2e362d703b3a2b"><span class="__cf_email__" data-cfemail="ee9d9a8b888f80c08c8f828f94ae8f8d9e869dc08b8a9b">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm201217q&amp;href=/doi/10.1021%2Fjm201217q" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2035–2047</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 26, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 September 2011</li><li><span class="item_label"><b>Published</b> online</span>17 February 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 March 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm201217q" title="DOI URL">https://doi.org/10.1021/jm201217q</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm201217q"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1784</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">23</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm201217q" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Binding Affinity Prediction for Ligands and Receptors Forming Tautomers and Ionization Species: Inhibition of Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MK2)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Senthil&quot;,&quot;last_name&quot;:&quot;Natesan&quot;},{&quot;first_name&quot;:&quot;Rajesh&quot;,&quot;last_name&quot;:&quot;Subramaniam&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;Bergeron&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Balaz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;2035-2047&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm201217q&quot;},&quot;abstract&quot;:&quot;Treatment of ionization and tautomerism of ligands and receptors is one of the unresolved issues in structure-based prediction of binding affinities. Our solution utilizes the thermodynamic master equation, expressing the experimentally observed association constant as the sum of products, each valid for a specific ligand–receptor species pair, consisting of the association microconstant and the fractions of the involved ligand and receptor species. The microconstants are characterized by structure-based simulations, which are run for individual species pairs. Here we incorporated the multispecies approach into the QM/MM linear response method and used it for structural correlation of published inhibition data on mitogen-activated protein kinase (MAPK)-activated protein kinase (MK2) by 66 benzothiophene and pyrrolopyridine analogues, forming up to five tautomers and seven ionization species under experimental conditions. Extensive cross-validation showed that the resulting models were stable and predictive. I&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201217q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201217q" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201217q&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201217q" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201217q&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201217q" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm201217q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201217q&amp;href=/doi/10.1021/jm201217q" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm201217q" /></input><a href="/doi/pdf/10.1021/jm201217q" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm201217q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm201217q%26sid%3Dliteratum%253Aachs%26pmid%3D22280316%26genre%3Darticle%26aulast%3DNatesan%26date%3D2012%26atitle%3DBinding%2BAffinity%2BPrediction%2Bfor%2BLigands%2Band%2BReceptors%2BForming%2BTautomers%2Band%2BIonization%2BSpecies%253A%2BInhibition%2Bof%2BMitogen-Activated%2BProtein%2BKinase-Activated%2BProtein%2BKinase%2B2%2B%2528MK2%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D5%26spage%3D2035%26epage%3D2047%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292238" title="QM/MM">QM/MM</a>,</li><li><a href="/action/doSearch?ConceptID=292244" title="Ionization">Ionization</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jmcmar.2012.55.issue-5/production/jmcmar.2012.55.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Treatment of ionization and tautomerism of ligands and receptors is one of the unresolved issues in structure-based prediction of binding affinities. Our solution utilizes the thermodynamic master equation, expressing the experimentally observed association constant as the sum of products, each valid for a specific ligand–receptor species pair, consisting of the association microconstant and the fractions of the involved ligand and receptor species. The microconstants are characterized by structure-based simulations, which are run for individual species pairs. Here we incorporated the multispecies approach into the QM/MM linear response method and used it for structural correlation of published inhibition data on mitogen-activated protein kinase (MAPK)-activated protein kinase (MK2) by 66 benzothiophene and pyrrolopyridine analogues, forming up to five tautomers and seven ionization species under experimental conditions. Extensive cross-validation showed that the resulting models were stable and predictive. Inclusion of all tautomers and ionization ligand species was essential: the explained variance increased to 90% from 66% for the single-species model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Quantitative prediction of binding affinities of ligands interacting with target macromolecules is one of the most important tasks for lead optimization and other procedures in computational medicinal chemistry. The majority of approved drugs and drug candidates contain tautomerism-prone heteroaromatic ring systems and heteroatom-rich substructures<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> as well as one or more ionizing groups.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> Components of the receptor binding sites, e.g., several amino acid residues,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> cofactors (porphyrin,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> NAD<sup>+</sup>, biotin,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and others), and nucleobases,<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> are also prone to ionization and tautomerism under physiological conditions.</div><div class="NLM_p">Structural differences of tautomer and ionization species lead to varying interactions with the binding site and cause the dependence of overall affinity on several factors. In addition to pH and temperature, the influence of medium polarity on tautomer and ionization equilibria plays a role because the interactions with the receptors may happen in an aqueous medium (blood/plasma, extra- and intracellular fluids) or in a nonpolar medium such as the bilayer core of the cell membrane.</div><div class="NLM_p">The time scale of establishing the tautomeric equilibria depends on the nature of broken and created bonds. Tautomers that interchange by CH bond cleavage and formation can often be isolated because their half-lives (<i>t</i><sub>1/2</sub>) are measured in hours thanks to high (>30 kcal/mol) interconversion energy barriers. The CH-to-NH, −OH, and −SH tautomer conversions have much lower energy barriers (∼20 kcal/mol), and their <i>t</i><sub>1/2</sub> values are in the range of a second.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For the NH-to-OH and OH-to-OH tautomerism, the rates are much faster: some keto–enol tautomers convert on the picosecond time scale.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The fast tautomer interconversions of the last two categories facilitate the description of the ligand binding to macromolecules because the tautomer fractions remain constant during the binding process and are fully characterized by the equilibrium constants, eliminating the need to consider the kinetics of the tautomer interconversion process.</div><div class="NLM_p">Treatment of tautomers presents a challenge to methods utilizing classical force fields because of the increased occurrence of less common structures for which the parameters may not be readily available. Along with this nuisance, the static charge distribution and other approximations of the classical force fields contribute to the increasing use of combined quantum mechanics/molecular mechanics (QM/MM) methods in structure-based binding affinity predictions.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> The application of hybrid QM/MM methods alleviates several issues in the classical description of biomolecular interactions and enables their more accurate description at a reasonable computational expense.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We previously reported an extension of the linear response method<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21 ref22">(16-22)</a> by using the QM/MM energies of the time-averaged structures after MD simulation.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> The QM/MM-LR approach was successfully applied to the affinity prediction of inhibitor binding to Zn-dependent matrix metalloproteinases (MMPs).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The model was capable of distinguishing subtle differences in the binding sites of two related MMPs and provide intricate selectivity clues<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> by including the phenomena that are out of reach of classical force fields, such as the coordination bonds made by the studied hydroxamates with the binding-site zinc and the proton transfer from the hydroxamate OH group to carboxyl of a neighboring Glu side chain in the binding site.</div><div class="NLM_p">Treatment of multiple species resulting from ionization and tautomerism is a challenge for current force-field based simulations in spite of the progress in the area.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Keeping in mind the advantages the QM/MM techniques offer for the treatment of tautomers and polarizing interactions, we wanted to examine the ability of the QM/MM-LR approach to describe binding of ligands involving tautomer and ionization equilibria. Suitable chemotypes, forming several tautomers and ionization species in aqueous media under physiological conditions, were found among the inhibitors of mitogen-activated protein kinase (MAPK)-activated protein kinase 2 (MK2) for the complexes of which a number of X-ray structures were available.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27-31)</a></div><div class="NLM_p">MK2 has two isoforms, α and β, which are produced by alternative splicing with 400 and 370 residues, respectively.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The enzyme consists of N-terminal proline-rich region (residues 10–44), catalytic region with DFG motif (residues 51–325), and C-terminal autoinhibitory region (residues 328–364). The C-terminal region includes a nuclear export signal (NES) of hydrophobic residues 356–368 and a nuclear localization signal (NLS) containing basic residues 373–389. The isoform β lacks the NLS region. During the presence of the inactive complex of p38α and MK2 in the nucleus, the C-terminal NLS is functional and the NES is masked. Upon stress stimulation, upstream kinases like MAPK kinase-6 activate p38, which in turn phosphorylates MK2, unmasking the NES, and this active complex translocates to cytoplasm. This cascade of events leads to coexport of activated p38 from the nucleus to cytoplasm.</div><div class="NLM_p">MK2 is a serine/threonine protein kinase regulating, by a post-transcriptional mechanism, biosynthesis of tumor necrosis factor α (TNFα) that is overproduced in inflammatory diseases such as rheumatoid arthritis and inflammatory bowel disease. MK2 knockout mice showed reduced expression of TNFα when stimulated with lipopolysaccharide (LPS) and were resistant to developing disease in arthritis models.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Mice that lack MK2 show increased stress resistance and survive LPS-induced endotoxic shock,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> thanks to significant reduction in the production of TNFα. Unlike p38 MAP kinase inhibition leading to several serious side effects,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> MK2 knockout mice are healthy and have a normal phenotype. The role of MK2 and the benefits of MK2 inhibition in other inflammatory diseases such as Alzheimer’s disease, atherosclerosis, and cancer are being actively investigated.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_p last">This study focuses on characterization of structural and energetic determinants of MK2 inhibitor binding involving multispecies equilibria using the newly developed MS-QM/MM-LR approach. The approach is applicable to binding predictions of speciated small molecules to speciated macromolecules, which are of interest in several areas of pure and applied chemistry.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67648" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67648" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The structures, tautomer and species, and inhibitory activities of the studied 66 MK2 inhibitors,<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27-29)</a> analogues of benzothiophene (benzothienodiazepinones, except <b>6</b> and <b>7</b>) and pyrrolopyridine, are summarized in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, respectively. Under physiological conditions, the compounds exhibit ionization as well as prototropic and annular tautomerism: they form up to five different skeleton tautomers, with some tautomers creating two to four ionization species (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>), occasionally thanks to the ionization centers in the side moieties R (Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). All potential tautomers within the two series are formed by the shifts of hydrogen between C and N, C and O, and N and O (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>), so the tautomer equilibria in an aqueous solution are expected to be established practically instantaneously.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Benzothiophene Analogues: Structures, Considered Tautomers and Species, and MK2 Inhibition IC<sub>50</sub> Values (M).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0003.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">considered</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">log(1/IC<sub>50</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand no.</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R1</th><th class="colsep0 rowsep0" align="center">R2</th><th class="colsep0 rowsep0" align="center">R3</th><th class="colsep0 rowsep0" align="center">R4</th><th class="colsep0 rowsep0" align="center">tautomers</th><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">exp</th><th class="colsep0 rowsep0" align="center" char=".">calcd</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.745</td><td class="colsep0 rowsep0" align="char" char=".">6.739</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">7.398</td><td class="colsep0 rowsep0" align="char" char=".">7.635</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.523</td><td class="colsep0 rowsep0" align="char" char=".">6.713</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">5.818</td><td class="colsep0 rowsep0" align="char" char=".">5.486</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2,T5</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S11</td><td class="colsep0 rowsep0" align="char" char=".">7.854</td><td class="colsep0 rowsep0" align="char" char=".">7.446</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T3</td><td class="colsep0 rowsep0" align="left">S1,S9</td><td class="colsep0 rowsep0" align="char" char=".">5.963</td><td class="colsep0 rowsep0" align="char" char=".">5.547</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T3</td><td class="colsep0 rowsep0" align="left">S1,S9</td><td class="colsep0 rowsep0" align="char" char=".">6.301</td><td class="colsep0 rowsep0" align="char" char=".">6.281</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6</td><td class="colsep0 rowsep0" align="char" char=".">8.301</td><td class="colsep0 rowsep0" align="char" char=".">8.436</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NHCH<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6</td><td class="colsep0 rowsep0" align="char" char=".">6.244</td><td class="colsep0 rowsep0" align="char" char=".">6.021</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NH(CH<sub>2</sub>)<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6</td><td class="colsep0 rowsep0" align="char" char=".">6.347</td><td class="colsep0 rowsep0" align="char" char=".">6.621</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NHCH<sub>2</sub>Ph<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2,T5</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6,S12</td><td class="colsep0 rowsep0" align="char" char=".">6.244</td><td class="colsep0 rowsep0" align="char" char=".">6.418</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>N(CH<sub>3</sub>)CH<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2,T5</td><td class="colsep0 rowsep0" align="left">S1,S2,S5–S7,S12</td><td class="colsep0 rowsep0" align="char" char=".">5.719</td><td class="colsep0 rowsep0" align="char" char=".">6.074</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6–S8</td><td class="colsep0 rowsep0" align="char" char=".">6.721</td><td class="colsep0 rowsep0" align="char" char=".">6.944</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>(3-OCH<sub>3</sub>-Ph)</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.959</td><td class="colsep0 rowsep0" align="char" char=".">7.022</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>-<i>c</i>-C<sub>3</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">5.879</td><td class="colsep0 rowsep0" align="char" char=".">5.874</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O(CH<sub>2</sub>)<sub>2</sub>OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.569</td><td class="colsep0 rowsep0" align="char" char=".">6.769</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>CH(CH<sub>3</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.398</td><td class="colsep0 rowsep0" align="char" char=".">6.238</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>COOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">7.523</td><td class="colsep0 rowsep0" align="char" char=".">7.212</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>-<i>c</i>-C<sub>6</sub>H<sub>11</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">5.463</td><td class="colsep0 rowsep0" align="char" char=".">5.593</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">O–CH═CH<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">7.796</td><td class="colsep0 rowsep0" align="char" char=".">7.775</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">O–CH<sub>2</sub>–CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">7.553</td><td class="colsep0 rowsep0" align="char" char=".">7.593</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═CH–CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">9.000</td><td class="colsep0 rowsep0" align="char" char=".">8.549</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">OC(Ph)═CH</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">6.824</td><td class="colsep0 rowsep0" align="char" char=".">6.801</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(Cl)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">8.699</td><td class="colsep0 rowsep0" align="char" char=".">8.474</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(Ph)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">8.046</td><td class="colsep0 rowsep0" align="char" char=".">7.758</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(4-Py)-CH═CH<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5–S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">7.854</td><td class="colsep0 rowsep0" align="char" char=".">7.967</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(3-Py)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5–S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">8.301</td><td class="colsep0 rowsep0" align="char" char=".">7.705</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(5-Pm)-CH═CH<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">7.638</td><td class="colsep0 rowsep0" align="char" char=".">7.356</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(2-OCH<sub>3</sub>-Ph)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S5,S7</td><td class="colsep0 rowsep0" align="char" char=".">7.699</td><td class="colsep0 rowsep0" align="char" char=".">7.517</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(3-OCH<sub>3</sub>-Ph)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">7.387</td><td class="colsep0 rowsep0" align="char" char=".">7.199</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(2-F-Ph)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">7.854</td><td class="colsep0 rowsep0" align="char" char=".">7.705</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(2-CH<sub>3</sub>-Ph)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S5,S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">7.523</td><td class="colsep0 rowsep0" align="char" char=".">7.771</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">N═C(4-CH3-3-Py)-CH═CH</td><td class="colsep0 rowsep0" align="left">T1,T2,T4</td><td class="colsep0 rowsep0" align="left">S1,S2,S5–S7,S10</td><td class="colsep0 rowsep0" align="char" char=".">8.301</td><td class="colsep0 rowsep0" align="char" char=".">8.265</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>N-Tp<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S6</td><td class="colsep0 rowsep0" align="char" char="."><4.699</td><td class="colsep0 rowsep0" align="char" char=".">4.879</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>N-Mo<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">T1,T2,T5</td><td class="colsep0 rowsep0" align="left">S1,S2,S5,S7,S11</td><td class="colsep0 rowsep0" align="char" char="."><4.699</td><td class="colsep0 rowsep0" align="char" char=".">4.812</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Ph: phenyl.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">R3 and R4 join the phenyl ring to form cyclic derivatives in ligands <b>20</b>–<b>33</b>.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Py: pyridyl.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Pm: pyrimidinyl.</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">Tp: tetrahydropyrrolyl.</p></div><div class="footnote" id="t1fn6"><sup>f</sup><p class="last">Mo: morpholinyl.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Pyrrolopyridine Analogues: Structures, Considered Species,<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> and MK2 Enzyme Inhibition IC<sub>50</sub> Values (M)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0004.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">log(1/IC<sub>50</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">considered species</th><th class="colsep0 rowsep0" align="center" char=".">exp</th><th class="colsep0 rowsep0" align="center" char=".">calcd</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">6.767</td><td class="colsep0 rowsep0" align="char" char=".">6.561</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">Ph<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.180</td><td class="colsep0 rowsep0" align="char" char=".">7.520</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">4-Py<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">S1,S2,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.252</td><td class="colsep0 rowsep0" align="char" char=".">7.339</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">3-Py</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.319</td><td class="colsep0 rowsep0" align="char" char=".">7.486</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">3-Th<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.119</td><td class="colsep0 rowsep0" align="char" char=".">7.550</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">2-Np<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.284</td><td class="colsep0 rowsep0" align="char" char=".">6.934</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">2-Bt<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.523</td><td class="colsep0 rowsep0" align="char" char=".">7.347</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">3-Q<a class="ref internalNav" href="#t2fn7" aria-label="g">g</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">8.071</td><td class="colsep0 rowsep0" align="char" char=".">7.994</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">2-OH-Ph</td><td class="colsep0 rowsep0" align="left">S1,S2,S3–S5</td><td class="colsep0 rowsep0" align="char" char=".">6.387</td><td class="colsep0 rowsep0" align="char" char=".">6.920</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">3-OH-Ph</td><td class="colsep0 rowsep0" align="left">S1,S2,S3–S5</td><td class="colsep0 rowsep0" align="char" char=".">7.602</td><td class="colsep0 rowsep0" align="char" char=".">7.829</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">4-OH-Ph</td><td class="colsep0 rowsep0" align="left">S1,S2,S3–S5</td><td class="colsep0 rowsep0" align="char" char=".">7.678</td><td class="colsep0 rowsep0" align="char" char=".">7.669</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">2-Cl-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">6.218</td><td class="colsep0 rowsep0" align="char" char=".">6.430</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">3-Cl-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.432</td><td class="colsep0 rowsep0" align="char" char=".">7.747</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">4-Cl-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.495</td><td class="colsep0 rowsep0" align="char" char=".">7.568</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">2-F-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">6.900</td><td class="colsep0 rowsep0" align="char" char=".">7.165</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left">3-F-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.523</td><td class="colsep0 rowsep0" align="char" char=".">7.659</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">4-F-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.301</td><td class="colsep0 rowsep0" align="char" char=".">7.422</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub>-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.149</td><td class="colsep0 rowsep0" align="char" char=".">7.181</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="left">4-COCH<sub>3</sub>-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.292</td><td class="colsep0 rowsep0" align="char" char=".">7.228</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub>-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.260</td><td class="colsep0 rowsep0" align="char" char=".">7.205</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">4-NH<sub>2</sub>-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.387</td><td class="colsep0 rowsep0" align="char" char=".">7.340</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="left">4-[CONH-<i>c</i>-C<sub>5</sub>H<sub>9</sub>]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">8.097</td><td class="colsep0 rowsep0" align="char" char=".">8.153</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">4-[CONH-<i>c</i>-C<sub>6</sub>H<sub>11</sub>]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.770</td><td class="colsep0 rowsep0" align="char" char=".">7.662</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">4-[CONHCH<sub>2</sub>Ph]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">8.097</td><td class="colsep0 rowsep0" align="char" char=".">7.786</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">4-[CONH(CH<sub>2</sub>)<sub>2</sub>Ph]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.337</td><td class="colsep0 rowsep0" align="char" char=".">7.508</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">4-[CONH(CH<sub>3</sub>)CH<sub>2</sub>Ph]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.252</td><td class="colsep0 rowsep0" align="char" char=".">7.309</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">6.216</td><td class="colsep0 rowsep0" align="char" char=".">6.270</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="left">5-Pm<a class="ref internalNav" href="#t2fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.081</td><td class="colsep0 rowsep0" align="char" char=".">7.115</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64</b></td><td class="colsep0 rowsep0" align="left">4-CN-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.208</td><td class="colsep0 rowsep0" align="char" char=".">7.218</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b></td><td class="colsep0 rowsep0" align="left">4-COOH-Ph</td><td class="colsep0 rowsep0" align="left">S2,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.658</td><td class="colsep0 rowsep0" align="char" char=".">7.683</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">4-[CONH-<i>c</i>-C<sub>3</sub>H<sub>5</sub>]-Ph</td><td class="colsep0 rowsep0" align="left">S1,S3,S5</td><td class="colsep0 rowsep0" align="char" char=".">7.824</td><td class="colsep0 rowsep0" align="char" char=".">7.816</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">All compounds present as tautomers T1 and T2.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Phenyl.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">Pyridinyl.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">Thienyl.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">Naphthyl (marked as 3-Np in the original publication).</p></div><div class="footnote" id="t2fn6"><sup>f</sup><p class="last">Benzothienyl.</p></div><div class="footnote" id="t2fn7"><sup>g</sup><p class="last">Quinolinyl.</p></div><div class="footnote" id="t2fn8"><sup>h</sup><p class="last">Pyrimidinyl.</p></div></div></div><div id="sec2_98" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Ligand Fractions of Tautomers and Ionization Species</h3><div class="NLM_p">Ligand fractions of tautomers and ionization species in aqueous solution were estimated for water under the conditions of experiments (pH 7.5 and 30 °C), using the SPARC<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> online calculator. The fractions of individual tautomers were calculated from the relative abundances of all possible tautomers at equilibrium (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). Ionization estimates were only performed for tautomers with at least 0.01% fraction to determine the fraction of each species. The resulting 233 tautomers (T1–T5) and species (S1–S12) for the studied 66 compounds were grouped and numbered as shown in Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The species classification is based on the ionization state as well as on the parent tautomer. As a result, the species with the same number may have different overall charges as indicated in Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Tautomer and Ionization Equilibria of Studied Benzothiophene Analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Species, present in at least 0.01% fraction under experimental conditions, are labeled S1–S12 and their parent tautomers are labeled as T1–T5. The neutral forms of the T3, T4, and T5 neutral-zwitterion pairs, marked with a prime, are present in negligible fractions and are listed only as intermediates in the formation of more abundant zwitterionic counterparts. The protonated form of T2 is shown as T2+. The charges of substituent groups are given in brackets. The R3–R4 substituent group for S2/T1 and S6/T2 refers to the cyclic derivatives (<b>20</b>–<b>33</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Species S3 and S4 are not present in this series.</p></p></figure><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Tautomer and Ionization Equilibria of the Studied Pyrrolopyridine Analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), Showing the Species Present in at Least 0.01% Fraction under Experimental Conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Tautomers are labeled as T1–T2 and species are labeled S1–S5. Protonated form of T1 is shown as T1+. The charges of substituent groups are given in brackets. Species S1, S2, and S5 are similar to those in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> in the approximate position of charged atoms with regard to the CO-NH in T1. Species S6–S12 are not present in this series.</p></p></figure><div class="NLM_p">The neutral-zwitterion tautomer pairs T3, T4, and T5 are mainly present as zwitterions, with the proton transferred from the hydroxyl in position 1 to N in position 2 or 5 (atom X in the Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> structure). Charged skeletons are produced by external protonation, as seen in benzothiophene tautomer T2<sup>+</sup> (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) and pyrrolopyridine tautomer T1<sup>+</sup>, with the protonated N in position 5 or in the pyridine ring, respectively. Further ionizable groups are present in the side chains R1, R3, R4, or cyclic substructures R3–R4. Combinations of the skeleton tautomers and the side chains ionized to varying degrees give rise to altogether 12 species.</div><div class="NLM_p">While both series are dominated mainly by tautomers T1 and/or T2, the presence of other tautomers cannot be neglected. The formation of T2 and T5 results in the loss of planarity of thiophene ring forming a new chiral center, which exhibited stronger interactions with the MK2 enzyme when set in <i>S</i> configuration in all MD simulations (described below). An overview of speciation of individual compounds is given in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, with all details listed in <a class="ref internalNav" href="#notes-1" aria-label="Table S4 in Supporting Information">Table S4 in Supporting Information</a>.</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Species fractions of the studied benzothiophenes (<b>1</b>–<b>35</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) and pyrrolopyridines (<b>36</b>–<b>66</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) in water under experimental conditions: species S1/T1, S2/T1, S3/T1, S4/T1, S5/T2, and S6/T2 (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) are shown in black, red, blue, yellow, gray, and green, respectively. Only major species (>10% in at least one compound) are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The majority of the benzothiophene analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) do not ionize in water under experimental conditions. For 26 of 35 compounds, species 5/tautomer 2 (S5/T2) is the predominant species. Compounds <b>6</b> and <b>7</b> are mainly present as S1/T1. Only compounds <b>8</b>–<b>12</b> and <b>34</b> are mostly available as ionized S6/T2 and S2/T1.</div><div class="NLM_p last">All pyrrolopyridine analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) are present mainly as T1 in water. While neutral species S1 dominates (>70%) for compounds <b>42</b>, <b>56</b>, and <b>62</b> and ionized species S3 for compounds <b>36</b>, <b>37</b>, <b>40</b>, <b>41</b>, <b>46</b>, <b>55</b>, <b>64</b>, and <b>65</b>, most compounds share preferences for both species 1 and 3. Compound <b>65</b> with carboxyl group substituent is always ionized and present as both species S2 and S3. Compound <b>44</b> also exhibits preference for neutral species S4 in addition to species S1 and S3.</div></div><div id="sec2_99" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Protonation States of Ionizable Protein Residues</h3><div class="NLM_p">Protonation of ionizable residues of MK2 was determined by p<i>K</i><sub>a</sub> calculations using the PROPKA 2.0 web server<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> at the experimental conditions (pH 7.5 and 30 °C). This empirical structure-based approach includes desolvation effects and intramolecular interactions such as H-bonds and charge–charge interactions. The p<i>K</i><sub>a</sub> values for ionizable residues in the binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> below), Glu139 (p<i>K</i><sub>a</sub> = 1.88), Asp142 (2.51), Asp207 (1.24), and Lys93 (12.83), differ from the pH 7.5 of experiment by 5 or more logarithmic units, indicating that the binding site has just one ionization state in fraction exceeding 0.01%, with the side chain carboxyl groups of both Glu and Asp residues deprotonated and the side chain ε-amino group of Lys93 protonated. No tautomerization is expected for the binding site residues.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Time-averaged structure of MK2 with bound compound <b>8</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) used in step 4. The protein is shown as C<sub>α</sub>-trace with the binding site indicated as transparent surface of the binding site residues Leu70, Gly71, Leu72, Gly73, Ile74, Ala91, Leu92, Lys93, Met138, Glu139, Cys140, Leu141, Asp142, Asn191, Leu192, Leu193, Thr206, and Asp207. The key residues and the ligand, shown in atom-type-colored ball and stick representation and the other binding site residues represented as red sticks are included in QM and optimized MM regions of the QM/MM geometry optimization, respectively (step 2). The ligand and all binding site residues were included in the flexible region in FlexiDock docking (step 1) and the QM-region in the single-point QM/MM energy calculation (step 4).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Multispecies Binding Equilibria</h3><div class="NLM_p">Individual ligand (L) and receptor (R) species present in the solution differ in structures of some fragments and, consequently, the binding event results in different structures of the complexes, which have different association microconstants. For the receptor, only the ionization or tautomer species of the fragments, which either directly participate in the binding or are in a sufficient proximity of the site to affect the binding, will differ in the association microconstant and need to be considered.</div><div class="NLM_p">The mutually exclusive 1:1 binding of multiple ligand species (total number <i>l</i>) to a macromolecule receptor site species (total number <i>s</i>) is illustrated in eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a> for the <i>i</i>th ligand species.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0007.gif" alt="" id="GRAPHIC-d2967e2260-autogenerated" /></img></span></div><div class="NLM_p">The microconstants <i>K</i><sub><i>ij</i></sub> characterizing affinities of the <i>l</i> × <i>s</i> ligand–receptor complex species are defined, for the <i>i</i>th ligand species bound to the <i>j</i>th receptor species, as<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/jm-2011-01217q_m001.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p">The association microconstants are the relevant quantities for the correlation with structure. The measured equilibrium constant <i>K</i> of the ligand, however, contains the total concentration of the ligand–receptor complexes without distinguishing between complexes differing in interacting species. To express <i>K</i> as a function of <i>K</i><sub><i>ij</i></sub>s, a series of rearrangements needs to be made, as shown in eq <a class="ref internalNav" href="#eq3" aria-label="3">3</a>: (1) the total bound concentration, [LR], is given as the sum of concentration of the <i>l</i> × <i>s</i> complex species, to get the third term; (2) in the numerator of the third term, each summand is formally multiplied by [L<sub><i>i</i></sub>][R<sub><i>j</i></sub>]/[L<sub><i>i</i></sub>][R<sub><i>j</i></sub>] and, in this way, the fractions of each ligand species, <i>f</i><sub><i>i</i></sub> = [L<sub><i>i</i></sub>]/[L], and each receptor species, <i>f</i><sub><i>j</i></sub> = [R<sub><i>j</i></sub>]/[R], are introduced, as shown in the fourth term; (3) finally, the microconstants <i>K</i><sub><i>ij</i></sub> are incorporated using their definition in eq <a class="ref internalNav" href="#eq2" aria-label="2">2</a>.<span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/jm-2011-01217q_m002.gif" alt="" /></img><span class="labelSpan">(3)</span></span>The fractions <i>f</i><sub><i>i</i></sub> and <i>f</i><sub><i>j</i></sub> depend on the medium and remain constant as long as the medium is not changing. The test media and intra/extracellular body fluids are buffered, so the key property, the pH value, remains invariant and the fractions <i>f</i><sub><i>i</i></sub> and <i>f</i><sub><i>j</i></sub> can usually be calculated before optimization.</div><div class="NLM_p">For ligand ionization, the expressions for the fractions <i>f</i><sub><i>i</i></sub> of the <i>i</i>th species can be derived from the definition of the ionization constants. In the case of two or more ionization groups, attention needs to be paid to the actual macroscopic p<i>K</i><sub>a</sub> values. If the p<i>K</i><sub>a</sub> values are closer than 3–4 units, the ionization tendencies of the two ionizable groups are not clearly separated, and mixed species may occur. Consequently, such p<i>K</i><sub>a</sub> values are no longer equal to the ionization microconstants, and multiple species with the same charge (e.g., neutral molecules and zwitterions as two species with the net zero charge, such as the neutral-zwitterion tautomer pairs T3–T5 in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) can be present for a given pH value of the medium.</div><div class="NLM_p">For the receptors, the equilibria can be affected by through-space proximity to other groups. Specialized techniques have been developed to estimate the ionization constants of individual ionizable groups<a onclick="showRef(event, 'ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47">(37-47)</a> and the equilibrium constants for tautomers.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p last">The prevalence of the <i>i</i>th bound species for a ligand can be calculated using the sum of all its complexes with the receptor species:<span class="NLM_disp-formula" id="eq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/jm-2011-01217q_m003.gif" alt="" /></img><span class="labelSpan">(4)</span></span>The numerator and denominator come from eqs <a class="ref internalNav" href="#eq2" aria-label="2">2</a> and <a class="ref internalNav" href="#eq3" aria-label="3">3</a>, respectively. The third quasi-equality in eq <a class="ref internalNav" href="#eq4" aria-label="4">4</a> ensures that the sum of prevalences for all <i>l</i> species of a ligand equals unity. The prevalence of the receptor species for the given ligand is calculated in a similar way; just the summations in the numerators of eq <a class="ref internalNav" href="#eq4" aria-label="4">4</a> would run through all <i>l</i> ligand species instead of the <i>s</i> receptor species.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Multispecies Inhibitor Studies</h3><div class="NLM_p">In the single-species QM/MM-LR approach, a linear combination of the QM/MM energy term and the solvent-accessible surface area (SASA) term is correlated with enzyme inhibition potency, which may be given as log(1/IC<sub>50</sub>) values, where IC<sub>50</sub> is the concentration of an inhibitor that decreases the rate of an enzyme-catalyzed reaction by 50%. For reversible and competitive enzyme inhibition, the direct proportionality between IC<sub>50</sub> and ligand–receptor association constant is given by the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> For the multispecies correlation, which uses the total association constant <i>K</i> as the dependent variable, each of the species microconstants <i>K</i><sub><i>ij</i></sub> in eq <a class="ref internalNav" href="#eq3" aria-label="3">3</a> needs to be expressed as an exponential with the exponent equal to the linear combination of the ligand–protein QM/MM interaction energy term and the SASA term and eq <a class="ref internalNav" href="#eq3" aria-label="3">3</a> needs to be logarithmized. Assuming that the regression coefficients α, γ, and κ maintain the same values for each pair of ligand and receptor species, the inclusion of multiple species does not increase the number of the regression coefficients. The MS-QM/MM-LR correlation equation is then<span class="NLM_disp-formula" id="eq5"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/jm-2011-01217q_m004.gif" alt="" /></img><span class="labelSpan">(5)</span></span>The term ⟨Δ<i>E</i><sub>QM/MM</sub>⟩ denotes the binding energy defined as the difference between the QM/MM energies of the complex and those of the unbound interaction partners, all calculated for the time-averaged structures from MD simulations. The change in SASA upon binding is defined analogously. The QM/MM term in eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> replaced the van der Waals and electrostatic energy terms in the classical LR approach, which were scaled by coefficients α and β, respectively.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21 ref22">(16-22)</a></div><div class="NLM_p last">The QM/MM and SASA terms of 233 tautomers and species of 66 ligands complexed with MK2 enzyme were calculated using a slight modification of our previously published four-tier approach.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> The protocol consists of (1) flexible docking with the poses selected based on the docking score, (2) QM/MM geometry optimization of the docked complexes, (3) MD simulation of the geometry optimized structures, solvated with explicit TIP3P water molecules,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and (4) for the time-averaged structures from step 3, calculation of the single-point QM/MM interaction energy term and the SASA term. The two terms were correlated with the enzyme inhibition potencies using eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Computational Protocol</h3><div class="NLM_p">In step 1 (docking), the studied inhibitors were docked into the recently published<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> X-ray structure of MK2 in active conformation, bound to a 3-aminopyrazole derivative (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA">3KGA</a><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>) with 2.55 Å resolution, taken from the Protein Data Bank.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> MK2 features the β-sheet dominated N-terminal lobe and a purely α-helical C-terminal lobe, enclosing the ATP-binding site for which the inhibitors compete. The binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) is lined by conserved glycine-rich motif GXGXXG of residues 71–76 and the hinge region, extending from the gate-keeper residue Met138 and including residues 139–142. The interaction of conserved Lys93 with the conserved Glu104 from helix C is considered a hallmark of the active conformation of protein kinases.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Docking was performed using the FlexiDock module from SYBYL-X. Two available X-ray structures, one for each series (for ligands <b>33</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and <b>50</b><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>), were used to guide prepositioning of studied compounds. In addition to ligands, rotatable bonds of side chains of binding site residues were allowed to move during the docking procedure. This step was critical in selecting the optimal binding mode with the highest FlexiDock score, especially for more flexible compounds. The deviations between the optimal binding modes (poses) of individual tautomers/species of the same compounds were significant. The respective heavy-atom rmsd values varied between 0.55 and 10.95 Å and are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Table S1 in Supporting Information">Table S1 in Supporting Information</a>. To speed up the QM/MM convergence, docked complexes were briefly minimized using Generalized Born implicit solvent method in Amber 10.</div><div class="NLM_p">In step 2 (QM/MM optimization), geometries of minimized protein–ligand complexes were further optimized by the ONIOM method,<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> a hybrid QM/MM method available in Gaussian 09.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Ligands and key binding site components (backbone atoms of Cys140, Leu141, and Asp142; whole residues of Thr206 and Asp207) were included in the QM region, and the rest of the system was defined as the MM region. The ligand and binding site residues were allowed to move during geometry optimization, and the rest of the system was frozen (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). No tautomer/species conversion was observed during the geometry optimization procedure.</div><div class="NLM_p">In step 3 (MD simulations), the entire hydrated and neutralized QM/MM-optimized complexes were heated and subjected to 1 ns MD simulation in Amber 10. Analysis of the trajectories for energy terms (potential, kinetic, and total energies), density, volume, temperature, pressure, and rmsd values of C<sub>α</sub> atoms revealed that that the secondary and tertiary structure of the ligand–receptor complex remained stable throughout the simulation and all studied ligand–receptor complexes attained equilibrium within the 1 ns simulation period. For several compounds, 20 ns MD simulations were run, showing no major differences to the 1 ns trajectories.</div><div class="NLM_p">The MD trajectories were also analyzed to track critical intermolecular interactions, including water bridges that were involved in ligand binding (see below). For all complexes, the conserved Lys93 was found to be interacting with a conserved Glu104 from helix C, indicating the active conformation of protein kinases.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The time-averaged structures from the final 900 ps of MD simulations were calculated using the <i>ptraj</i> program in AmberTools,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> keeping only 100 water molecules closest to the ligands and stripping off the remaining water molecules and counterions used for neutralizing the system. The resulting structures were briefly minimized to remove unnatural features.</div><div class="NLM_p">Step 4 (single-point QM/MM energy and SASA calculations) was performed for time-averaged structures resulting from step 3. The QM region consisted of ligands, binding site residues, and water molecule for complexes, in the cases when water-mediated interactions were observed. The heavy-atom rmsd values between the time-averaged structures of individual tautomers/species varied between 1.30 and 11.73 Å and are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Table S2 in Supporting Information">Table S2 in Supporting Information</a>.</div><div class="NLM_p last">For steps 1–4, the computations for each ligand–receptor species took on average 15–20 min, 40–60 h, 200–240 h, and 12–24 h of single-processor time, respectively. For the 233 complexes, plus free interaction partners, this study used about ∼130000 h of CPU time.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Multispecies 3D-QSAR Correlation</h3><div class="NLM_p">The MK2 enzyme inhibition potencies, given as logarithmized 1/IC<sub>50</sub> values, were correlated with the QM/MM interaction terms and the SASA terms, both calculated in step 4. The nonlinear regression model (eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>), with <i>s</i> = 1 as only one ionization/tautomer state was predicted for the binding site, was optimized using the Solver<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> software.</div><div class="NLM_p">Contributions of individual steps of the calculation protocol to the correlation were examined, and the results are summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. The use of the FlexiDock scores in place of the QM/MM energies in eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> resulted in no correlation (<i>r</i><sup>2</sup> = 0.002). The QM/MM energies and SASA, calculated from the geometry optimization step 2, gave <i>r</i><sup>2</sup> = 0.202. The use of van der Waals and electrostatic energy terms from the MD simulation in the MS-LR fashion in eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>, as summarized in step 3, slightly improved the correlation (<i>r</i><sup>2</sup> = 0.353), but the signs of the coefficients α and β for the van der Waals and electrostatic energy terms, respectively, were incorrect. The full MS-QM/MM-LR treatment (eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>) was necessary to achieve an agreement between experimental and calculated inhibitory activities, with the correct signs of the optimized coefficients α and γ (<i>r</i><sup>2</sup> = 0.906).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Linear Response Correlations Using eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> for Individual Steps: Optimized Coefficients and Descriptive Statistical Indices<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" valign="top">step</th><th class="colsep0 rowsep0" align="left" valign="top" char="±">α × 10<sup>–3</sup>(mol/kcal)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="left" valign="top" char="±">β × 10<sup>–4</sup>(mol/kcal)<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="left" valign="top" char="±">γ × 10<sup>–3</sup>(1/Å<sup>2</sup>)<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="left" valign="top" char="±">κ</th><th class="colsep0 rowsep0" align="left" valign="top" char="."><i>r</i><sup>2</sup></th><th class="colsep0 rowsep0" align="left" valign="top" char=".">RMSE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1. docking</td><td class="colsep0 rowsep0" align="char" char="±">0.0878 ± 0.115</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">7.292 ± 0.238</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td><td class="colsep0 rowsep0" align="char" char=".">0.741</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2. QM/MM optimization</td><td class="colsep0 rowsep0" align="char" char="±">–0.3868 ± 21.25</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">–7.354 ± 46.51</td><td class="colsep0 rowsep0" align="char" char="±">4.107 ± 27.44</td><td class="colsep0 rowsep0" align="char" char=".">0.202</td><td class="colsep0 rowsep0" align="char" char=".">0.663</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3. MD simulation</td><td class="colsep0 rowsep0" align="char" char="±">10.47 ± 54.29</td><td class="colsep0 rowsep0" align="char" char="±">5.709 ± 62.32</td><td class="colsep0 rowsep0" align="char" char="±">–11.61 ± 86.56</td><td class="colsep0 rowsep0" align="char" char="±">5.417 ± 38.22</td><td class="colsep0 rowsep0" align="char" char=".">0.353</td><td class="colsep0 rowsep0" align="char" char=".">0.597</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4. single-point QM/MM</td><td class="colsep0 rowsep0" align="char" char="±">–1.326 ± 0.591</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">–7.761 ± 2.261</td><td class="colsep0 rowsep0" align="char" char="±">1.160 ± 0.358</td><td class="colsep0 rowsep0" align="char" char=".">0.906</td><td class="colsep0 rowsep0" align="char" char=".">0.228</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">The squared correlation coefficient (<i>r</i><sup>2</sup>) and the root-mean-square error (RMSE).</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">Scales the QM/MM energy term (eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>) in rows 2 and 4, and the vdW energy term in row 3.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">Scales the electrostatic energy term.</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last">Scales the SASA term.</p></div></div></div><div class="NLM_p">An analysis of the importance of tautomeric and ionization species for the correlation is summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. The simplest model (row 1) uses only the QM/MM and SASA terms for tautomer 1 (no ionization species are included), in eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>, which thus became a standard, one-mode QM/MM-LR equation and exhibits a comparatively weak correlation (<i>r</i><sup>2</sup> = 0.662). The addition of ionization species for tautomer 1 (row 2) lead to a better performance (<i>r</i><sup>2</sup> = 0.734). The consideration of all tautomers and no ionization species (row 3) brought further improvement (<i>r</i><sup>2</sup> = 0.839). The model including all tautomers and all ionization species (row 4) provided the best results (<i>r</i><sup>2</sup> = 0.906). The comparison of experimental and calculated inhibitory potencies for different species compositions (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>), shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>, also confirms the need for inclusion of all tautomers and ionization species.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. The MS-QM/MM-LR Correlations (eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>) for Different Tautomer and Species Composition: Optimized Coefficients and Statistical Indices</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">LOO<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MC-LGO<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">tautomer</th><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char="±">α × 10<sup>–3</sup>(mol/kcal)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±">γ × 10<sup>–3</sup>(1/Å<sup>2</sup>)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">κ</th><th class="colsep0 rowsep0" align="center" char="."><i>r</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">RMSE</th><th class="colsep0 rowsep0" align="center" char="."><i>q</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">RMSE</th><th class="colsep0 rowsep0" align="center" char="."><i>q</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">RMSE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">single</td><td class="colsep0 rowsep0" align="left">single</td><td class="colsep0 rowsep0" align="char" char="±">–1.212 ± 1.289</td><td class="colsep0 rowsep0" align="char" char="±">–8.985 ± 8.451</td><td class="colsep0 rowsep0" align="char" char="±">1.151 ± 0.896</td><td class="colsep0 rowsep0" align="char" char=".">0.662</td><td class="colsep0 rowsep0" align="char" char=".">0.431</td><td class="colsep0 rowsep0" align="char" char=".">0.617</td><td class="colsep0 rowsep0" align="char" char=".">0.458</td><td class="colsep0 rowsep0" align="char" char=".">0.616</td><td class="colsep0 rowsep0" align="char" char=".">0.459</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">single</td><td class="colsep0 rowsep0" align="left">multi</td><td class="colsep0 rowsep0" align="char" char="±">–1.291 ± 1.026</td><td class="colsep0 rowsep0" align="char" char="±">–8.664 ± 7.211</td><td class="colsep0 rowsep0" align="char" char="±">1.000 ± 0.758</td><td class="colsep0 rowsep0" align="char" char=".">0.734</td><td class="colsep0 rowsep0" align="char" char=".">0.382</td><td class="colsep0 rowsep0" align="char" char=".">0.690</td><td class="colsep0 rowsep0" align="char" char=".">0.412</td><td class="colsep0 rowsep0" align="char" char=".">0.691</td><td class="colsep0 rowsep0" align="char" char=".">0.411</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multi</td><td class="colsep0 rowsep0" align="left">single</td><td class="colsep0 rowsep0" align="char" char="±">–1.338 ± 0.801</td><td class="colsep0 rowsep0" align="char" char="±">–8.334 ± 5.002</td><td class="colsep0 rowsep0" align="char" char="±">0.960 ± 0.507</td><td class="colsep0 rowsep0" align="char" char=".">0.839</td><td class="colsep0 rowsep0" align="char" char=".">0.297</td><td class="colsep0 rowsep0" align="char" char=".">0.819</td><td class="colsep0 rowsep0" align="char" char=".">0.315</td><td class="colsep0 rowsep0" align="char" char=".">0.819</td><td class="colsep0 rowsep0" align="char" char=".">0.315</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multi</td><td class="colsep0 rowsep0" align="left">multi</td><td class="colsep0 rowsep0" align="char" char="±">–1.326 ± 0.591</td><td class="colsep0 rowsep0" align="char" char="±">–7.761 ± 2.261</td><td class="colsep0 rowsep0" align="char" char="±">1.160 ± 0.358</td><td class="colsep0 rowsep0" align="char" char=".">0.906</td><td class="colsep0 rowsep0" align="char" char=".">0.228</td><td class="colsep0 rowsep0" align="char" char=".">0.897</td><td class="colsep0 rowsep0" align="char" char=".">0.237</td><td class="colsep0 rowsep0" align="char" char=".">0.894</td><td class="colsep0 rowsep0" align="char" char=".">0.241</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Scales the QM/MM energy term.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">Scales the SASA term.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">Leave-one-out cross-validation: the results reported as RMSE and the squared predictive correlation coefficient (<i>q</i><sup>2</sup>).</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last">Monte Carlo leave-group-out cross-validation: the compounds were randomly divided into 7 groups, and each group of ∼9 compounds was used once as a test set; the procedure was repeated 10 times.</p></div></div></div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Correlations between experimental and calculated MK2 inhibition potencies for studied compounds are shown for all models: single-tautomer, single-species (green squares), single-tautomer, multispecies (red triangles), multitautomer, single-species (blue diamonds), and multitautomer, multispecies (black circles).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The agreement between the model and experiment is very satisfactory, especially for the complete setup (row 4). The LOO-cross-validation and, more importantly, the rigorous MC-LGO cross-validation confirmed that all models are stable, exhibit no overfitting, and have adequate predictive power because the values of the predictive indices RMSE and <i>q</i><sup>2</sup> are similar to those of the descriptive indices RMSE and <i>r</i><sup>2</sup>. Interestingly, the optimized values of the coefficients α and γ did not vary significantly with the addition of species, which is also a sign of extraordinary stability of the models. As an additional model validation attempt, we performed the correlations with swapped values of the fractions <i>f</i><sub><i>i</i></sub> of individual tautomers/species in water. The numbers of tautomers/species differ but are equal to three for most compounds (Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Therefore, we swapped the <i>f</i><sub><i>i</i></sub> values in the correlation eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> for the first three species (the <i>f</i><sub>i</sub> values for compounds <b>6</b> and <b>7</b>, having only two species, were always exchanged). The descriptive abilities of the correlations using eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> were characterized by <i>r</i><sup>2</sup> = 0.906 for the correct (1–2–3) order of the <i>f</i><sub><i>i</i></sub> values, <i>r</i><sup>2</sup> = 0.831 for the order 3–2–1 (the <i>f</i><sub><i>i</i></sub> values for species 1 and 3 were exchanged), <i>r</i><sup>2</sup> = 0.807 for the 2–1–3 order, <i>r</i><sup>2</sup> = 0.629 for the 1–3–2 order, <i>r</i><sup>2</sup> = 0.702 for the 2–3–1 order, and 0.673 for the 3–2–1 order. The swapping of some <i>f</i><sub><i>i</i></sub> values, although providing only limited randomization, clearly leads to deterioration of the correlation.</div><div class="NLM_p">For compounds <b>34</b> and <b>35</b> with semiquantitative enzyme inhibitory potencies (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), which were excluded from the model fitting process, predictions were made using all models. Activities of both compounds were predicted to be very close to the observed limits using the complete model (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, line 4) although not below them (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><div class="NLM_p">Contribution of the QM/MM energy to the correlation with inhibitory potency is greater than that of the SASA term. Coefficient γ associated with SASA term is approximately 7 times larger than that of the QM/MM term; however, when the overall contributions, i.e., the products of coefficients and the variables, are considered, the values of the QM/MM contributions are ∼10 times larger than those of the SASA term. There is no cross-correlation between the QM/MM energy and SASA terms (<i>r</i><sup>2</sup> = −0.334).</div><div class="NLM_p last">Standard deviations of inhibitory activities (IC<sub>50</sub>) were only published for the series 2 compounds (<b>36</b>–<b>66</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) and are listed in <a class="ref internalNav" href="#notes-1" aria-label="Table S3 in Supporting Information">Table S3 in Supporting Information</a>. No information was published about the error distribution, so a detailed comparison of model performance with respect to experimental errors cannot be made. However, we can use the standard deviations as the estimates of experimental variance. On the logarithmic scale, which was used in the correlation, the average error interval for log(1/IC<sub>50</sub>) is 0.220 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>), with the average log(1/IC<sub>50</sub>) located asymmetrically in this interval. The average error interval, which includes both positive and negative deviations, is smaller than RMSE = 0.228 for the best prediction (step 4 in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), indicating that the complexity of the model is adequate: the model is not overly detailed and does not describe experimental errors.</div></div><div id="sec2_4_98" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Critical MK2–Inhibitor Interactions</h3><div class="NLM_p">Important interactions between MK2 and inhibitors  were tracked by the analysis of MD trajectories of 233 complexes from the production phase.</div><div id="xec2_4_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> H-Bond Interactions</h4><div class="NLM_p">H-bond interactions, both at the catalytic end (Asp207 and Lys93) and in the hinge region (Leu141), shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, are typical for all compounds exhibiting higher inhibitory potencies. The amino and carbonyl groups in the lactam ring make H-bond interactions with catalytic Asp/Lys pairs and methoxy oxygen (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) or pyridinyl nitrogen (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) groups interact with the backbone NH group of Leu141 in hinge region (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). The magnitudes of the average bond lengths (<2 Å) and angles (>150°, i.e., not deviating much from the ideal 180° magnitude) indicate strong H-bond interactions that seem to be critical for enzyme inhibition.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0010.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Interaction map of compound <b>8</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) as species 5/tautomer 2 (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) in the binding site of MK2. Nitrogen, oxygen, and sulfur atoms are printed in blue, red, and yellow colors, respectively. H-Bond interactions are drawn in red color, and the bond lengths and bond angles (in italics) for the time-averaged structure are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Water-Mediated Interactions</h4><div class="NLM_p last">Water-mediated interactions were observed in many studied ligands bound to MK2. The protonated N atom in position X (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) of the lactam ring in tautomers T2 and/or T4 of compounds <b>23</b> and <b>33</b> makes the water-mediated H-bond interaction with the backbone carbonyl oxygen of Glu190. Additionally, in compound <b>33</b>, a water molecule creates an H-bonded bridge between the protonated N atom of pyridine ring and the carboxyl group of the Asp142 side chain in the hinge region. Water-mediated interactions were also observed for tautomer T2 of compounds <b>5</b>, <b>8</b>, <b>9</b>, <b>11</b>, <b>17</b>, and <b>19</b>, as well as for tautomers T2 and T4 of compounds <b>25</b>, <b>26</b>, and <b>29</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), in which the protonated N atom in the X position of the lactam ring forms H-bond interaction with the backbone carbonyl oxygen of Leu70 in the glycine-rich loop through a water molecule. Pyrrolopyridines (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) binding as S3/T1 possess protonated pyridine ring, which interacts either directly with Glu139 backbone carbonyl oxygen (compounds <b>40</b>, <b>41</b>, <b>43</b>–<b>46</b>, <b>49</b>, <b>52</b>, and <b>57</b>) or with Leu141 backbone NH group through a water-mediated interaction (compounds <b>36</b>, <b>38</b>, <b>39</b>, <b>56</b>, <b>58</b>, <b>59</b>, and <b>60</b>).</div></div><div id="sec2_4_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Hydrophobic Interactions</h4><div class="NLM_p last">Hydrophobic interactions with the hinge region (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) are formed by several substructures in the part of the molecules that is opposite to the lactam ring. In benzothiophene series (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), the rings attached in positions R<sub>3</sub>–R<sub>4</sub> make compounds <b>22</b>–<b>33</b> relatively potent inhibitors. On the contrary, the absence of the aryl ring attached to the pyridyl ring decreases the potency of pyrrolopyridines <b>36</b> and <b>62</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). However, the structural dependence is more complex: substituents in the <i>o</i>-position of the aryl ring attached to the pyridyl ring negatively affect potency for pyrrolopyridines <b>44</b>, <b>47</b>, and <b>49</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The side chain atoms of residue Leu70 (glycine-rich loop, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) and Leu193 make hydrophobic interactions from both sides of the binding site with many compounds having aryl ring substituents positioned near the hinge region of the MK2 enzyme.</div></div></div><div id="sec2_4_99" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Prevalences of Bound Tautomers and Species</h3><div class="NLM_p">The prevalences, as predicted by eq <a class="ref internalNav" href="#eq4" aria-label="4">4</a> with the <i>K</i><sub><i>ij</i></sub> values calculated from eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a> with optimized coefficients (last lines of Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>or <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>), are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Tables S4 and S5 in Supporting Information">Tables S4 and S5 in Supporting Information</a>. The majority (28 of 35) of benzothiophenes (<b>1</b>–<b>5</b>, <b>12</b>–<b>33</b>, and <b>35</b>; Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) preferably bind as neutral species 5/tautomer 2 (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Several benzothiophenes (<b>1</b>–<b>4</b>, <b>12</b>–<b>14</b>, <b>17</b>, <b>18</b>, <b>24</b>, <b>29</b>, <b>30</b>, and <b>35</b>) also exhibit some fraction of bound S1/T1 species. Only compounds <b>6</b> and <b>7</b> bind exclusively as S1/T1 (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Compounds <b>8</b>–<b>10</b> and <b>34</b> prefer to bind as protonated S6/T2 thanks to the presence of amino substituent in the lactam ring. Compound <b>11</b> shows almost equal binding preferences for S2/T1, S5/T2, and S6/T2. Compounds <b>9</b> and <b>11</b> bind (∼10%) in S2/T1 form as well.</div><div class="NLM_p">Pyrrolopyridine analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) mostly bind in neutral form as S1/T1 (<b>42</b>, <b>50</b>, <b>60</b>, and <b>62</b>) or protonized as S3/T1 (<b>36</b>, <b>41</b>, <b>43</b>, <b>45</b>, <b>46</b>, <b>49</b>, <b>54</b>–<b>57</b>, <b>64</b>, and <b>65</b>) or both (<b>37</b>–<b>40</b>, <b>43</b>, <b>47</b>, <b>48</b>, <b>51</b>–<b>53</b>, <b>58</b>, <b>59</b>, <b>61</b>, <b>63</b>, and <b>66</b>). Compound <b>44</b> binds as both S3/T1 and S4/T1 species, the latter having the phenyl ring hydroxyl group in the deprotonated form. The predominantly bound tautomer 1 has the planar lactam ring attached to aromatic pyrrole ring. All compounds from this series make favorable interactions with the Asp/Lys catalytic pair, which may explain their comparatively narrow potency range.</div><div class="NLM_p">The fractions of free vs bound tautomers and species for all studied inhibitors are plotted in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. For the majority of the compounds, the prevalent species in solution are the bound species as well. Nevertheless, the correlation is rather loose and there are several noticeable differences between binding prevalences and the fractions in solution.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0011.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. A comparison of predicted bound species prevalences and species fractions in the aqueous solution for studied ligands (the numbers correspond to the compound numbers in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Species 1–6 (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) are referred to by black, cyan, red, magenta, blue, and green colors, respectively. Only major species (at least one compound with >10%) are shown here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For example, compounds <b>11</b> and <b>12</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) are mainly present as protonated species S6 in solution (>60%) but they are bound as nonionized species S5. Compound <b>13</b> binds to a significant extent as species S1, S5, and S6, although only species S5 is prevalent in solution. Compound <b>50</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) is present as both species S1 and S3 in solution but shows a higher preference for species S1 while binding.The situation is opposite for compounds <b>43</b>, <b>56</b>, and <b>57</b>, which prefer to bind as species S3, even though they have about equal concentrations of species S1 and S3 in solution.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The QM/MM linear response method was extended for multiple ligand and receptor species, resulting from tautomerism and ionization, using rigorous thermodynamic principles, to fill a gap in the armory of the computational methods for structure-based, ensemble-utilizing, and empirically calibrated prediction of binding affinities. Ionization and tautomerism, as widespread phenomena among the drug-like molecules, need to be included in the methods for prediction of binding affinities to bring the models closer to reality. Our approach is based on performing the simulations individually for each species pair and combining the results of these simulations in a correlation equation that is calibrated using the experimental data. Interestingly, inclusion of multiple species does not increase the number of optimized coefficients, at least in the studied case: the same regression coefficients could be used for all species without a detrimental impact on the quality of the fit. The developed MS-QM/MM LR method was applied to a set of 66 MK2 inhibitors forming up to five tautomers and seven ionization species under the experimental conditions. The treatment of multiple ligand species significantly improved the correlation between experimental inhibition potencies and the QM/MM interaction energy and SASA terms for the time-averaged structures from MD simulations. The structural and energetic information obtained from the time-averaged structures highlights the critical interactions necessary for optimal enzyme inhibition. Strength of the H-bond interactions of ligands with the backbone amide group of Leu141 in the hinge region, and carboxyl and ε-amino groups in the side chains of catalytic Asp207 and Lys93, respectively, significantly affect the inhibition potency. Additional interactions with backbone carbonyl group of Glu190 and glycine-rich loop made by some potent inhibitors represent hints for further improvements in potency. If future studies, preferably dealing with the experimentally observed prevalences of bound species, confirm the utility of the MS-QM/MM-LR approach, it may become a welcome addition to the methods for lead optimization in drug design.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51848" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51848" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Data Set</h3><div class="NLM_p last">The MK2 inhibitory activities for 66 benzothiophene and pyrrolopyridine derivatives were reported by Anderson et al.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27-29)</a> The X-ray structures of the MK2 complexes with a benzothiophene analogue <b>33</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), a pyrrolopyridine analogue <b>50</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), and an untested 3-aminopyrazole derivative with the PDB<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ">3FYJ</a>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P3G">2P3G</a>,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA">3KGA</a>,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> respectively, facilitated the use of structure-based techniques. The structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ">3FYJ</a> was preferred to <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYK">3FYK</a><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> because its ligand is larger and provides more clues for docking larger benzothiophene analogues. The inhibitors compete with ATP for binding to MK2. The isoeffective concentrations (IC<sub>50</sub>) were determined in the assay, monitoring the amount of HSP-peptide (KKKALSRQLSVAA) that was phosphorylated by MK2 after 30 min incubation at pH 7.5 and 30 °C.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27-29)</a> Experimental errors were only given for the pyrrolopyridine series<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and are listed in <a class="ref internalNav" href="#notes-1" aria-label="Table S3 in Supporting Information">Table S3 in Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Tautomerism and Ionization of Ligands</h3><div class="NLM_p last">The enzyme inhibitory IC<sub>50</sub> values of the studied ligands were determined at pH 7.5 and 30 °C. The relative abundances of all possible tautomers at equilibrium were estimated under the conditions of the experiment. Subsequently, ionization estimates were performed only for tautomer having the fraction of at least 0.01% to determine the fraction of individual species as a function of pH. In both cases, the SPARC web server<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> was used.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Ionization and Tautomerism of Binding Site Residues</h3><div class="NLM_p last">The protonation states of ionizable protein residues were estimated by p<i>K</i><sub>a</sub> calculations using the PROPKA 2.0 web server<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> at the experimental conditions (pH 7.5 and 30 °C). There was one dominant ionization state of the binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) predicted, with Glu and Asp side chain carboxyl groups deprotonated and Lys side chain ε-amino groups protonated. No tautomerism was expected for the binding site residues (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Ligand Preparation</h3><div class="NLM_p last">Ligand structures were built in Sybyl-X 1.1.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Geometry optimization and Mulliken charge<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> calculations were done in Jaguar<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> (Schrodinger) using the DFT/B3LYP method<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> with basis set 6-31G**.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a> Charge and spin multiplicity were entered manually for different protonation states.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Protein Preparation</h3><div class="NLM_p last">The recently published X-ray structure of MK2 in active conformation bound to a 3-aminopyrazole derivative (PDB<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA">3KGA</a><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>) with 2.55 Å resolution is of better quality than other available structures (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ">3FYJ</a>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYK">3FYK</a>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P3G">2P3G</a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>), as documented by resolution (2.55 Å vs >3.5 Å) and distribution of φ–ψ dihedrals (90% vs <70%) in the most favored regions. The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA">3KGA</a> structure was modified with Biopolymer structure preparation tool in Sybyl-X:<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> the bound ligand and water molecules were removed, heavy atoms missing from residues were added, and the N-terminal and the C-terminal residues were capped with <i>N</i>-acetyl and <i>N</i>-methyl amide groups. Protonation types were set for His (ε-N protonated), Glu (negatively charged), and Lys (positively charged) residues. Hydrogen atoms were added, and charges were loaded using the AMBER 7 FF02<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> force field. The resulting structure was briefly (100 steps) minimized only for H-atoms by the Powell conjugate gradient method in Sybyl-X.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec4_99" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Flexible Protein–Ligand Docking</h3><div class="NLM_p">Docking of ligands into MK2 binding site was performed using the FlexiDock program in Sybyl-X.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> FlexiDock deploys a genetic alghorithm-based optimization in torsional space for both the protein binding site (rotatable bonds in side chains) and the ligand. The fitness function uses a subset of the Tripos force field: the van der Waals, electrostatic, torsional, and constraint energy terms. The number of rotatable bonds in the binding site residues (see Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) of the protein was up to 57 and that of ligands ranged from 1 to 8. The number of generations used varied from 35000 to 60000 depending upon the total number of rotatable bonds, satisfying the rule-of-thumb recommendation of 500–1000 generations per each rotatable bond in protein and ligand plus six. The site-point scoring feature was activated in FlexiDock by defining the H-bond interaction partners in the protein and ligands: the backbone NH of Leu141 in the hinge region of MK2 as H-bond donor, and, as acceptor, the first atom, if it was O or N, in substituent R<sub>3</sub> of benzothiophenes (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) or the N atom in the pyridyl ring of pyrrolopyridines (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The available X-ray structures for the data set series, with the PDB<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ">3FYJ</a> (compound <b>33</b> In Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> below)<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P3G">2P3G</a> (compound <b>50</b> in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> below),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> were used to aid the prepositioning of benzothiophene and pyrrolopyridine analogues, respectively, in the binding site prior to docking. The protein parts of these structures were aligned with that of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA">3KGA</a> structure by homology in Biopolymer module in Sybyl-X. The remaining ligands from both the series were superimposed using the fit atoms method. The best structures were chosen based on the FlexiDock scores and visual inspection. As tautomers with nonplanar lactam ring for most benzothiophenes (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) did not always make catalytic end H-bond interactions, the H-bond distances to catalytic residues Asp207 and Lys93 were not considered mandatory for the pose selection.</div><div class="NLM_p last">The difference in the docked poses (conformations) of tautomers/species of individual compounds were reported as the heavy-atom rmsd values (<a class="ref internalNav" href="#notes-1" aria-label="Table S1 in Supporting Information">Table S1 in Supporting Information</a>), calculated using Chimera.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> For all compounds, species 1 served as reference structure except for compound <b>65</b>, where species 2 was used for this purpose. The same approach was used to calculate the heavy-atom rmsd values for the time-averaged ligand geometries of tautomers/species after MD simulation (steps 3/4), which are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Table S2 in Supporting Information">Table S2 in Supporting Information</a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> QM/MM Geometry Optimization</h3><div class="NLM_p last">Docked protein–ligand complexes were briefly (1000 steps) optimized to remove any bad contacts by modified Generalized Born implicit solvent model in Amber 10. Optimization started with steepest descent method to quickly remove the largest strains and was switched after 100 cycles to the conjugate gradient method with root-mean-square of 1 cal/(mol × Å) as the convergence criterion for fine optimization close to the energy minimum. The nonbonded cutoff was set to 16.0 Å. Input files for ONIOM calculations<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> in Gaussian 09<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> were prepared in GaussView 5.0.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Ligands and selected binding site components (entire residues of Thr206 and Asp207, backbone atoms of Cys140, Leu141, and Asp142) were included in the QM region, and the rest of the system was defined as the MM region. The partitioning of the system in ONIOM was based on published recommendations:<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> the QM/MM cuts were always made between the C–C bonds and at least three bonds away from any potential interacting atoms. In the MM region, the remaining binding site residues (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) were allowed to move during geometry optimization and the rest of the system was frozen. Charges were manually entered for both QM (along with spin multiplicity) and MM regions. The defined QM and MM regions were treated with DFT/B3LYP functional<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> using 6-31G(d,p) basis set<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a> and Amber ff99SB force field,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> respectively. Electronic embedding option was activated during calculations.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> MD Simulation</h3><div class="NLM_p">Molecular dynamics simulations were performed using Amber 10 package<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> under isothermal/isobaric (NPT) conditions with Amber ff99SB<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and general Amber (GAFF)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> force fields for protein and ligand molecules, respectively.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Structure Preparation</h4><div class="NLM_p last">The Leap program from Antechamber tools (AmberTools 1.4)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> was used to prepare the parameter/topology (<i>prmtop</i>) and input coordinate (<i>inpcrd</i>) files using the Mulliken charges<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> from the QM/MM optimization, which were kept for protein–ligand complexes during MD simulations. For each system, protein–ligand complexes and unbound ligands were simulated separately. The unbound protein was simulated only once. The net charge of the protein–ligand complexes varied from +3 to +6 depending upon the overall charge of the ligand and was neutralized by adding Cl<sup>–</sup> ions at positions of high positive electron potential around the complexes. The system was immersed in a truncated octahedral box of pre-equilibrated TIP3P<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> water molecules<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> in the way that no atoms in the protein–ligand complexes were closer than 8 Å to any of the sides of the water box. The counterions and solvent molecules were briefly minimized for 1000 steps to remove any bad contacts with the complexes, whereby the protein and ligands were position-restrained using force constant of 500 kcal/(mol × Å<sup>2</sup>).</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Heating, Equilibration, and Production Phases</h4><div class="NLM_p last">To allow the readjustment of solvent molecules to the potential field of the ligand–receptor complex, the solvent equilibration step was performed in three stages. During the first heating phase, MD simulation was carried out for 10 ps, with constant volume periodic boundaries with an initial temperature of 0 K, allowing to heat up to 100 K. The second heating phase was performed for 20 ps under constant pressure periodic boundaries with an average pressure of 1 atm. Isotropic position scaling was used to maintain the pressure, and a relaxation time of 2 ps was used. The system was allowed to heat up from the initial temperature 100 K to the final temperature 300 K. The final solvent equilibration step was performed for 50 ps under constant pressure periodic boundary conditions as mentioned above. The Langevin dynamics was used in all stages to control the temperature using a collision frequency of 1.0 ps<sup>–1</sup>. During these three stages, the protein–ligand complexes were position-restrained with a force constant of 10 kcal/(mol × Å<sup>2</sup>). The final production phase with the entire system was carried out under isothermal/isobaric conditions for 1 ns including 100 ps equilibration phase. The SHAKE algorithm<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> was used to constrain bonds involving hydrogen, allowing time step of 2 fs, for a total of 500000 steps. The trajectory file was written for every 100 steps, resulting in 5000 frames. The cutoff was set to 10 Å in all steps.</div></div><div id="sec4_7_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Tracking H-Bond and Water-Mediated Interactions.</h4><div class="NLM_p last">MD trajectory analysis for H-bond and water-mediated interactions for 233 complexes from the production phase was performed using the <i>ptraj</i> program in AmberTools 1.4.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The default cutoff values for distance (3.5 Å) and angle (no value) were chosen. The following pairwise interactions were monitored: (1) the backbone NH of Leu141 in the hinge region of MK2 as H-bond donor, and, as H-bond acceptor, the first atom, if it was O or N, in substituent R<sub>3</sub> of benzothiophenes (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) or the N atom in the pyridyl ring of pyrrolopyridines (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), (2) Asp207 carboxyl oxygen atoms and the NH group of the six- or seven-membered lactam ring of ligands, and (3) the O atom of the lactam ring and the amino group of the Lys93 side chain. To keep track of important water-mediated interactions, “solvent” keywords <i>solventneighbor</i> (value = 6), <i>solventdonor</i> (WAT O), and <i>solventacceptor</i> (WAT O H1 and WAT O H2) were specified. The <i>solventneighbor</i> value specifies the maximum number of possible interactions per a given donor or acceptor.</div></div><div id="sec4_7_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Time-Averaged Structures</h4><div class="NLM_p last">Time-averaged structures were calculated from 4000 trajectory frames (final 900 ps) of the production phase of the MD simulation using the <i>ptraj</i> program in AmberTools 1.4.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The hundred water molecules closest to the ligands were kept for all complexes, and the remaining waters and ions were stripped. The structures were briefly minimized to relieve conflicts, resulting in structures with close-to-standard bond length and angles and the dihedrals representing the ensemble.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Single Point QM/MM Energy Calculations</h3><div class="NLM_p last">The time-averaged structures from MD simulations were used for single point QM/MM energy calculation in ONIOM<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> using DFT/B3LYP functional<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> method with 6-31G(d,p) basis set<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a> for QM region and Amber ff99SB<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> force field for MM region, respectively. Electronic embedding option was activated during calculations. The QM region consisted of ligands, binding site residues (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), and a water molecule for complexes, where water-mediated interactions were observed. The binding energy term ⟨Δ<i>E</i><sub>QM/MM</sub>⟩ term was calculated as the difference between the QM/MM energies of the complex and those of the unbound interaction partners.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> SASA Term Calculation</h3><div class="NLM_p last">The solvent accessible surface area (SASA) terms for time-averaged structures of protein–ligand complexes and unbound interaction partners were calculated using the program Naccess,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> which implements the Lee and Richards algorithm.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> The default values (probe radius 1.4 Å; <i>z</i>-slices 0.05 Å; van der Waals radii) were used in the calculations. The burial of solvent accessible surface area upon protein–ligand binding was calculated as the difference between the SASAs of the complex and those of unbound interaction partners.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Regression and Cross-Validation</h3><div class="NLM_p last">The coefficient values in the model (eq <a class="ref internalNav" href="#eq5" aria-label="5">5</a>) were optimized by nonlinear regression analysis in the Premium Solver Platform.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The stability and predictive power of the model was tested by the Monte Carlo leave-group-out (MC-LGO) cross-validation. In this test, compounds were randomly divided into seven groups of about equal size and the model calibration process involved only compounds from six groups in any given cycle leaving one of the groups outside as the test set. The resulting model was used to predict the activities for the omitted test set compounds. This process was repeated 10 times, reshuffling the compounds in each group, resulting in 70 models in total. The leave-one-out (LOO) cross-validation was also performed, omitting and predicting one compound at a time.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/jm201217q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74384" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74384" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Tables (XLS) summarizing the heavy-atom rmsd values between the docked poses (step 1) and between the time-averaged poses (steps 3/4) of tautomer/species for individual compounds, published standard deviations of inhibitory activities of pyrrolopyridines (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), fractions of free tautomers/species and prevalences of bound tautomers/species for benzothiophenes (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) and pyrrolopyridines (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), respectively. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm201217q/suppl_file/jm201217q_si_001">jm201217q_si_001 (98.0 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm201217q" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Balaz</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Sciences, Albany College of Pharmacy and Health Sciences, Vermont
Campus, 261 Mountain View Drive, Colchester, Vermont 05446, United
States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fa898e9f9c9b94d4989b969b80ba9b998a9289d49f9e8f"><span class="__cf_email__" data-cfemail="1566617073747b3b777479746f557476657d663b707160">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Senthil Natesan</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Sciences, Albany College of Pharmacy and Health Sciences, Vermont
Campus, 261 Mountain View Drive, Colchester, Vermont 05446, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rajesh Subramaniam</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Sciences, Albany College of Pharmacy and Health Sciences, Vermont
Campus, 261 Mountain View Drive, Colchester, Vermont 05446, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles Bergeron</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Sciences, Albany College of Pharmacy and Health Sciences, Vermont
Campus, 261 Mountain View Drive, Colchester, Vermont 05446, United
States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2967e4678-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85047" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85047" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by the NIH NIGMS grant R01 GM80508 and by the NSF TG-MCB110017 program through Teragrid resources provided by NCSA and Pittsburgh Supercomputing Center. We thank Dr. Mahmoud A. Ibrahim, School of Chemistry, The University of Manchester, UK, for providing Amber parameter file for Gaussian 09.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i39" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i39"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i40" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i40"> Abbreviations Used</h2><tr><td class="NLM_term">LGO</td><td class="NLM_def"><p class="first last">leave-group-out (cross-validation)</p></td></tr><tr><td class="NLM_term">LOO</td><td class="NLM_def"><p class="first last">leave-one-out (cross-validation)</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr><tr><td class="NLM_term">LR</td><td class="NLM_def"><p class="first last">linear response (method)</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">MC</td><td class="NLM_def"><p class="first last">Monte Carlo</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">MK2</td><td class="NLM_def"><p class="first last">MAPK-activated protein kinase 2</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">multispecies (approach)</p></td></tr><tr><td class="NLM_term">NES</td><td class="NLM_def"><p class="first last">nuclear export signal</p></td></tr><tr><td class="NLM_term">NLS</td><td class="NLM_def"><p class="first last">nuclear localization signal</p></td></tr><tr><td class="NLM_term">ONIOM</td><td class="NLM_def"><p class="first last">our own N-layered integrated molecular orbital and molecular mechanics</p></td></tr><tr><td class="NLM_term">QM/MM</td><td class="NLM_def"><p class="first last">quantum mechanics/molecular mechanics</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RMSE</td><td class="NLM_def"><p class="first last">root-mean-square error</p></td></tr><tr><td class="NLM_term">SASA</td><td class="NLM_def"><p class="first last">solvent-accessible surface area</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term">TNFα</td><td class="NLM_def"><p class="first last">tumor necrosis factor α</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 72 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lj9AJxJWu9dlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lee, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayyampalayam, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreira, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattachar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coselmon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span> </span><span class="NLM_article-title">In silico prediction of ionization constants of drugs</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">498</span><span class="NLM_x">–</span> <span class="NLM_lpage">512</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp070019%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1CmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=498-512&author=P.+H.+Leeauthor=S.+N.+Ayyampalayamauthor=L.+A.+Carreiraauthor=M.+Shalaevaauthor=S.+Bhattacharauthor=R.+Coselmonauthor=S.+Pooleauthor=E.+Giffordauthor=F.+Lombardo&title=In+silico+prediction+of+ionization+constants+of+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Prediction of Ionization Constants of Drugs</span></div><div class="casAuthors">Lee, Pil H.; Ayyampalayam, Saravanaraj N.; Carreira, Lionel A.; Shalaeva, Marina; Bhattachar, Shobha; Coselmon, Rachel; Poole, Salwa; Gifford, Eric; Lombardo, Franco</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">498-512</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most pharmacol. active mols. contain one or more ionizing groups, and it is well-known that knowledge of the ionization state of a drug, indicated by the pKa value, is crit. for understanding many properties important to the drug discovery and development process.  The ionization state of a compd. directly influences such important pharmaceutical characteristics as aq. soly., permeability, crystal structure, etc.  Tremendous advances have been made in the field of exptl. detn. of pKa, in terms of both quantity/speed and quality/accuracy.  However, there still remains a need for accurate in silico predictions of pKa both to est. this parameter for virtual compds. and to focus screening efforts of real compds.  The computer program SPARC (SPARC Performs Automated Reasoning in Chem.) was used to predict the ionization state of a drug.  This program has been developed based on the solid phys. chem. of reactivity models and applied to successfully predict numerous phys. properties as well as chem. reactivity parameters.  SPARC predicts both macroscopic and microscopic pKa values strictly from mol. structure.  In this paper, we describe the details of the SPARC reactivity computational methods and its performance on predicting the pKa values of known drugs as well as Pfizer internal discovery/development compds.  A high correlation (r2 = 0.92) between exptl. and the SPARC calcd. pKa values was obtained with root-mean-square error (RMSE) of 0.78 log unit for a set of 123 compds. including many known drugs.  For a set of 537 compds. from the Pfizer internal dataset, correlation coeff. r2 = 0.80 and RMSE = 1.05 were obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3S-24g-rak7Vg90H21EOLACvtfcHk0lj9AJxJWu9dlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1CmsLw%253D&md5=8363ba511dbb3a98ed8d0f8f4389d88e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fmp070019%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp070019%252B%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BH.%26aulast%3DAyyampalayam%26aufirst%3DS.%2BN.%26aulast%3DCarreira%26aufirst%3DL.%2BA.%26aulast%3DShalaeva%26aufirst%3DM.%26aulast%3DBhattachar%26aufirst%3DS.%26aulast%3DCoselmon%26aufirst%3DR.%26aulast%3DPoole%26aufirst%3DS.%26aulast%3DGifford%26aufirst%3DE.%26aulast%3DLombardo%26aufirst%3DF.%26atitle%3DIn%2520silico%2520prediction%2520of%2520ionization%2520constants%2520of%2520drugs%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2007%26volume%3D4%26spage%3D498%26epage%3D512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzur, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassanali, M.</span><span> </span><span class="NLM_article-title">DrugBank: A knowledgebase for drugs, drug actions and drug targets</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D901</span><span class="NLM_x">–</span> <span class="NLM_lpage">D906</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fnar%2Fgkm958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=18048412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D901-D906&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=D.+Chengauthor=S.+Shrivastavaauthor=D.+Tzurauthor=B.+Gautamauthor=M.+Hassanali&title=DrugBank%3A+A+knowledgebase+for+drugs%2C+drug+actions+and+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Cheng, Dean; Shrivastava, Savita; Tzur, Dan; Gautam, Bijaya; Hassanali, Murtaza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D901-D906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a richly annotated resource that combines detailed drug data with comprehensive drug target and drug action information.  Since its first release in 2006, DrugBank has been widely used to facilitate in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  The latest version of DrugBank (release 2.0) has been expanded significantly over the previous release.  With ∼4900 drug entries, it now contains 60% more FDA-approved small mol. and biotech drugs including 10% more exptl.' drugs.  Significantly, more protein target data has also been added to the database, with the latest version of DrugBank contg. three times as many non-redundant protein or drug target sequences as before (1565 vs. 524).  Each DrugCard entry now contains more than 100 data fields with half of the information being devoted to drug/chem. data and the other half devoted to pharmacol., pharmacogenomic and mol. biol. data.  A no. of new data fields, including food-drug interactions, drug-drug interactions and exptl. ADME data have been added in response to numerous user requests.  DrugBank has also significantly improved the power and simplicity of its structure query and text query searches.  DrugBank is available at http://www.drugbank.ca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqKjSWZe1ULVg90H21EOLACvtfcHk0lhmKEwX7eRlSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D&md5=f8f8ca15f248b8c8b9c6d3bd0e85eec8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm958%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DTzur%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DB.%26aulast%3DHassanali%26aufirst%3DM.%26atitle%3DDrugBank%253A%2520A%2520knowledgebase%2520for%2520drugs%252C%2520drug%2520actions%2520and%2520drug%2520targets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD901%26epage%3DD906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Shimba, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serber, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledwidge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doetsch, V.</span><span> </span><span class="NLM_article-title">Quantitative identification of the protonation state of histidines in vitro and in vivo</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">9227</span><span class="NLM_x">–</span> <span class="NLM_lpage">9234</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi0344679" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=9227-9234&author=N.+Shimbaauthor=Z.+Serberauthor=R.+Ledwidgeauthor=S.+M.+Millerauthor=C.+S.+Craikauthor=V.+Doetsch&title=Quantitative+identification+of+the+protonation+state+of+histidines+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbi0344679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0344679%26sid%3Dliteratum%253Aachs%26aulast%3DShimba%26aufirst%3DN.%26aulast%3DSerber%26aufirst%3DZ.%26aulast%3DLedwidge%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DS.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DDoetsch%26aufirst%3DV.%26atitle%3DQuantitative%2520identification%2520of%2520the%2520protonation%2520state%2520of%2520histidines%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D9227%26epage%3D9234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Vila, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnautova, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobjev, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H. A.</span><span> </span><span class="NLM_article-title">Assessing the fractions of tautomeric forms of the imidazole ring of histidine in proteins as a function of pH</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">5602</span><span class="NLM_x">–</span> <span class="NLM_lpage">5607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=5602-5607&author=J.+A.+Vilaauthor=Y.+A.+Arnautovaauthor=Y.+Vorobjevauthor=H.+A.+Scheraga&title=Assessing+the+fractions+of+tautomeric+forms+of+the+imidazole+ring+of+histidine+in+proteins+as+a+function+of+pH"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVila%26aufirst%3DJ.%2BA.%26aulast%3DArnautova%26aufirst%3DY.%2BA.%26aulast%3DVorobjev%26aufirst%3DY.%26aulast%3DScheraga%26aufirst%3DH.%2BA.%26atitle%3DAssessing%2520the%2520fractions%2520of%2520tautomeric%2520forms%2520of%2520the%2520imidazole%2520ring%2520of%2520histidine%2520in%2520proteins%2520as%2520a%2520function%2520of%2520pH%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D5602%26epage%3D5607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Clarke, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rochester, C. H.</span><span> </span><span class="NLM_article-title">Spectroscopic study of the acid ionization of porphyrins</span> <span class="citation_source-journal">J. Chem. Soc., Perkin. Trans. 2</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1039%2Fp29730000414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE3sXht1ersL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1973&pages=414-416&author=J.+A.+Clarkeauthor=P.+J.+Dawsonauthor=R.+Griggauthor=C.+H.+Rochester&title=Spectroscopic+study+of+the+acid+ionization+of+porphyrins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Spectroscopic study of the acid ionization of porphyrins</span></div><div class="casAuthors">Clarke, J. A.; Dawson, P. J.; Grigg, R.; Rochester, C. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">414-16</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">Spectral studies showed that N-methyloctaethylporphyrin and the  corresponding N-(3-bromopropyl)porphyrin ionize in solns. contg. aq. NaOH to give their monoanions.  Octaethylporphyrin, etioporphyrin I, and monozaethioporphyrin I ionize to their dianions by loss of 2 protons; their monoanions were not detected in solns. contg. NaOH.  N-Me substitution in a porphyrin increases the acidity of the remaining NH group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxd38-n0HVW7Vg90H21EOLACvtfcHk0lhmKEwX7eRlSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXht1ersL8%253D&md5=a946e4f6d826d855be5021176975fbc0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2Fp29730000414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29730000414%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BA.%26aulast%3DDawson%26aufirst%3DP.%2BJ.%26aulast%3DGrigg%26aufirst%3DR.%26aulast%3DRochester%26aufirst%3DC.%2BH.%26atitle%3DSpectroscopic%2520study%2520of%2520the%2520acid%2520ionization%2520of%2520porphyrins%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin.%2520Trans.%25202%26date%3D1973%26spage%3D414%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Braun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koecher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlabach, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehrle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limbach, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, E.</span><span> </span><span class="NLM_article-title">NMR study of the tautomerism of porphyrin including the kinetic HH/HD/DD isotope effects in the liquid and the solid state</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">6593</span><span class="NLM_x">–</span> <span class="NLM_lpage">6604</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja00094a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2cXltVygsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1994&pages=6593-6604&author=J.+Braunauthor=M.+Koecherauthor=M.+Schlabachauthor=B.+Wehrleauthor=H.+H.+Limbachauthor=E.+Vogel&title=NMR+study+of+the+tautomerism+of+porphyrin+including+the+kinetic+HH%2FHD%2FDD+isotope+effects+in+the+liquid+and+the+solid+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">NMR Study of the Tautomerism of Porphyrin Including the Kinetic HH/HD/DD Isotope Effects in the Liquid and the Solid State</span></div><div class="casAuthors">Braun, Juergen; Koecher, Martin; Schlabach, Martin; Wehrle, Bernd; Limbach, Han-Heinrich; Vogel, Emanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6593-604</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Using dynamic liq.-state 1H NMR and solid-state 15N CPMAS NMR spectroscopy (CP = cross polarization, MAS = magic angle spinning), the intramol. double-proton transfer (tautomerism) of 15N-labeled porphyrin has been studied.  Rate consts. including kinetic HH/HD/DD isotope effects were obtained as a function of temp. not only for the liq. phase but also for the polycryst. solid.  Within the margin of error, the liq.-state degeneracy of the tautomerism is maintained in the solid state; moreover, rate consts. and kinetic isotope effects are the same for both environments.  Therefore, it is justified to combine rate consts. of the porphyrin tautomerism detd. for various environments into a single Arrhenius diagram.  The Arrhenius curves of the different isotopic reactions indicate a stepwise tautomerism via a metastable intermediate as postulated previously, involving thermally activated tunneling at low temps.  Furthermore, the obsd. kinetic isotope effects are not in agreement with a solid-state process consisting of combined nondegenerate proton transfers and mol. 90° rotations.  This possibility has been discussed in order to reconcile the observations of proton disorder in solid porphyrin by NMR and proton order by X-ray crystallog.  Finally, a novel process has been obsd. which consists of a complete deuteration or dedeuteration of the mobile proton sites of solid porphyrin by contact with gaseous or liq. D2O or H2O.  The process involves (i) penetration of gaseous water into the porphyrin crystals, (ii) diffusion processes of water or of porphyrin inside the crystals, and (iii) exchange of protons or deuterons in mol. hydrogen-bonded water-porphyrin complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwIM86pqDWrVg90H21EOLACvtfcHk0lhmKEwX7eRlSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltVygsb8%253D&md5=9fcef5707c746ae9a0cf4879ddfa6c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja00094a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00094a014%26sid%3Dliteratum%253Aachs%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DKoecher%26aufirst%3DM.%26aulast%3DSchlabach%26aufirst%3DM.%26aulast%3DWehrle%26aufirst%3DB.%26aulast%3DLimbach%26aufirst%3DH.%2BH.%26aulast%3DVogel%26aufirst%3DE.%26atitle%3DNMR%2520study%2520of%2520the%2520tautomerism%2520of%2520porphyrin%2520including%2520the%2520kinetic%2520HH%252FHD%252FDD%2520isotope%2520effects%2520in%2520the%2520liquid%2520and%2520the%2520solid%2520state%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D6593%26epage%3D6604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Sponer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leszczynski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobza, P.</span><span> </span><span class="NLM_article-title">Electronic properties, hydrogen bonding, stacking, and cation binding of DNA and RNA bases</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=3-31&author=J.+Sponerauthor=J.+Leszczynskiauthor=P.+Hobza&title=Electronic+properties%2C+hydrogen+bonding%2C+stacking%2C+and+cation+binding+of+DNA+and+RNA+bases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DLeszczynski%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DElectronic%2520properties%252C%2520hydrogen%2520bonding%252C%2520stacking%252C%2520and%2520cation%2520binding%2520of%2520DNA%2520and%2520RNA%2520bases%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D61%26spage%3D3%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Sigel, H.</span><span> </span><span class="NLM_article-title">Acid–base properties of purine residues and the effect of metal ions: Quantification of rare nucleobase tautomers</span> <span class="citation_source-journal">Pure Appl. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">1869</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2004&pages=1869-1886&author=H.+Sigel&title=Acid%E2%80%93base+properties+of+purine+residues+and+the+effect+of+metal+ions%3A+Quantification+of+rare+nucleobase+tautomers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSigel%26aufirst%3DH.%26atitle%3DAcid%25E2%2580%2593base%2520properties%2520of%2520purine%2520residues%2520and%2520the%2520effect%2520of%2520metal%2520ions%253A%2520Quantification%2520of%2520rare%2520nucleobase%2520tautomers%26jtitle%3DPure%2520Appl.%2520Chem.%26date%3D2004%26volume%3D76%26spage%3D1869%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Lippert, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, D.</span><span> </span><span class="NLM_article-title">Promotion of rare nucleobase tautomers by metal binding</span> <span class="citation_source-journal">Dalton Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_fpage">4619</span><span class="NLM_x">–</span> <span class="NLM_lpage">4634</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=4619-4634&author=B.+Lippertauthor=D.+Gupta&title=Promotion+of+rare+nucleobase+tautomers+by+metal+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLippert%26aufirst%3DB.%26aulast%3DGupta%26aufirst%3DD.%26atitle%3DPromotion%2520of%2520rare%2520nucleobase%2520tautomers%2520by%2520metal%2520binding%26jtitle%3DDalton%2520Trans.%26date%3D2009%26spage%3D4619%26epage%3D4634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Katritzky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Gendy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draghici, B.</span><span> </span><span class="NLM_article-title">Tautomerism in drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">475</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2Fs10822-010-9359-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=20490619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=475-484&author=A.+Katritzkyauthor=C.+Hallauthor=B.+El+Gendyauthor=B.+Draghici&title=Tautomerism+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tautomerism in drug discovery</span></div><div class="casAuthors">Katritzky, Alan R.; Hall, C. Dennis; El-Gendy, Bahaa El-Dien M.; Draghici, Bogdan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">475-484</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The influence of tautomerism on the precise structure of drugs and thus of their potential to interact with biol. systems is discussed from thermodn. and kinetic aspects.  The types of tautomerism encountered in the structure of drugs in current use are surveyed together with the effect of pH, solvent polarity, and temp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8wFMBpeaetbVg90H21EOLACvtfcHk0lidkyWdsF3rDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbk%253D&md5=06e63cd44d4c7a9f974c8257f07fc275</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9359-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9359-z%26sid%3Dliteratum%253Aachs%26aulast%3DKatritzky%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DC.%26aulast%3DEl%2BGendy%26aufirst%3DB.%26aulast%3DDraghici%26aufirst%3DB.%26atitle%3DTautomerism%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D475%26epage%3D484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Chou, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y. M.</span><span> </span><span class="NLM_article-title">Excited-state intramolecular proton transfer in 10-hydroxybenzo[<i>h</i>]quinoline</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1731</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp002942w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFynsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=1731-1740&author=P.+T.+Chouauthor=Y.+C.+Chenauthor=W.+S.+Yuauthor=Y.+H.+Chouauthor=C.+Y.+Weiauthor=Y.+M.+Cheng&title=Excited-state+intramolecular+proton+transfer+in+10-hydroxybenzo%5Bh%5Dquinoline"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Excited-State Intramolecular Proton Transfer in 10-Hydroxybenzo[h]quinoline</span></div><div class="casAuthors">Chou, Pi-Tai; Chen, Youn-Chan; Yu, Wei-Shan; Chou, Yi-Hsuan; Wei, Ching-Yen; Cheng, Yi-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1731-1740</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamics of excited-state intramol. enol-keto proton-transfer tautomerism in 10-hydroxybenzoquinoline (HBQ) and its deuterated analog (DBQ) have been investigated by steady-state absorption and fluorescence spectroscopy, femtosecond fluorescence upconversion in combination with pump-probe transient absorption expts. in nonpolar solvents.  In cyclohexane, the time scale for both proton and deuterium transfer in the excited state cannot be resolved under the response limit of ca. 160 and 200 fs, resp., of our current upconversion and transient absorption systems.  The initially prepd. keto tautomer is in a higher lying excited state, possibly the S'2 state (prime indicates the keto-tautomer form) which then undergoes a ∼330 fs S'2 → S'1 internal conversion, resulting in a highly vibrationally excited S'1 state. Subsequently, a solvent-induced vibrational relaxation takes place in a time scale of 8-10 ps followed by a relatively much longer, thermally cooled S'1 → S'0 decay rate of 3.3 × 109 s-1 (τf ∼ 300 ps-1).  The results in combination with extremely weak enol fluorescence resolved from the steady-state measurement lead us to conclude that excited-state intramol. proton transfer (ESIPT) is essentially barrier-less.  The rate of ESIPT upon 385-405 nm excitation may be detd. within the period of low-frequency, large-amplitude vibrations incorporating the motion of atoms assocd. with the hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFT4XR0IpgLVg90H21EOLACvtfcHk0lidkyWdsF3rDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFynsLs%253D&md5=bb76bfc34af6abef6a3f046e9a47ab56</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjp002942w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp002942w%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DP.%2BT.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DYu%26aufirst%3DW.%2BS.%26aulast%3DChou%26aufirst%3DY.%2BH.%26aulast%3DWei%26aufirst%3DC.%2BY.%26aulast%3DCheng%26aufirst%3DY.%2BM.%26atitle%3DExcited-state%2520intramolecular%2520proton%2520transfer%2520in%252010-hydroxybenzo%255Bh%255Dquinoline%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2001%26volume%3D105%26spage%3D1731%26epage%3D1740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Raha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wollacott, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westerhoff, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, J.</span><span> </span><span class="NLM_article-title">The role of quantum mechanics in structure-based drug design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2Fj.drudis.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=17826685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=725-731&author=K.+Rahaauthor=M.+B.+Petersauthor=B.+Wangauthor=N.+Yuauthor=A.+M.+Wollacottauthor=L.+M.+Westerhoffauthor=J.+Merz&title=The+role+of+quantum+mechanics+in+structure-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The role of quantum mechanics in structure-based drug design</span></div><div class="casAuthors">Raha, Kaushik; Peters, Martin B.; Wang, Bing; Yu, Ning; Wollacott, Andrew M.; Westerhoff, Lance M.; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">17&18</span>),
    <span class="NLM_cas:pages">725-731</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Herein we will focus on the use of quantum mechanics (QM) in drug design (DD) to solve disparate problems from scoring protein-ligand poses to building QM QSAR models.  Through the variational principle of QM we know that we can obtain a more accurate representation of mol. systems than classical models, and while this is not a matter of debate, it still has not been shown that the expense of QM approaches is offset by improved accuracy in DD applications.  Objectively validating the improved applicability and performance of QM over classical-based models in DD will be the focus of research in the coming years along with research on the conformational sampling problem as it relates to protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ISn2DWdN9LVg90H21EOLACvtfcHk0lhcxCd-D7zW9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK&md5=67fecc13843c7961ff820026c85a9290</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DK.%26aulast%3DPeters%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DN.%26aulast%3DWollacott%26aufirst%3DA.%2BM.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26aulast%3DMerz%26aufirst%3DJ.%26atitle%3DThe%2520role%2520of%2520quantum%2520mechanics%2520in%2520structure-based%2520drug%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D725%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Senn, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, W.</span><span> </span><span class="NLM_article-title">QM/MM methods for biomolecular systems</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1198</span><span class="NLM_x">–</span> <span class="NLM_lpage">1229</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fanie.200802019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=19173328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitFOqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=1198-1229&author=H.+M.+Sennauthor=W.+Thiel&title=QM%2FMM+methods+for+biomolecular+systems"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">QM/MM methods for biomolecular systems</span></div><div class="casAuthors">Senn, Hans Martin; Thiel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1198-1229</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Combined quantum-mechanics/mol.-mechanics (QM/MM) approaches have become the method of choice for modeling reactions in biomol. systems.  Quantum-mech. (QM) methods are required for describing chem. reactions and other electronic processes, such as charge transfer or electronic excitation.  However, QM methods are restricted to systems of up to a few hundred atoms.  However, the size and conformational complexity of biopolymers calls for methods capable of treating up to several 100,000 atoms and allowing for simulations over time scales of tens of nanoseconds.  This is achieved by highly efficient, force-field-based mol. mechanics (MM) methods.  Thus to model large biomols. the logical approach is to combine the two techniques and, to use a QM method for the chem. active region (e.g., substrates and co-factors in an enzymic reaction) and an MM treatment for the surroundings (e.g., protein and solvent).  The resulting schemes are commonly referred to as combined or hybrid QM/MM methods.  They enable the modeling of reactive biomol. systems at a reasonable computational effort while providing the necessary accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDAQMRLVHxZrVg90H21EOLACvtfcHk0lhcxCd-D7zW9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitFOqs7g%253D&md5=c51da58b0525651c71f9c393a79023be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.200802019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200802019%26sid%3Dliteratum%253Aachs%26aulast%3DSenn%26aufirst%3DH.%2BM.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DQM%252FMM%2520methods%2520for%2520biomolecular%2520systems%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D1198%26epage%3D1229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Major, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truhlar, D. G.</span><span> </span><span class="NLM_article-title">Mechanisms and free energies of enzymatic reactions</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">3188</span><span class="NLM_x">–</span> <span class="NLM_lpage">3209</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr050293k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt12nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3188-3209&author=J.+Gaoauthor=S.+Maauthor=D.+T.+Majorauthor=K.+Namauthor=J.+Puauthor=D.+G.+Truhlar&title=Mechanisms+and+free+energies+of+enzymatic+reactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and free energies of enzymatic reactions</span></div><div class="casAuthors">Gao, Jiali; Ma, Shuhua; Major, Dan T.; Nam, Kwangho; Pu, Jingzhi; Truhlar, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3188-3209</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here, the authors summarize the theory and computational techniques for studies of the catalytic mechanisms and free energies of selected enzymic reactions.  Computational approaches for enzymic reactions with special emphasis on 2 key elements that affect the computational accuracy, namely, the potential energy function and statistical mech. sampling of the enzyme system are highlighted.  The choice of the reaction coordinate for detg. the free energy of activation to characterize the enzymic mechanism is discussed.  By examg. a variety of enzymic reactions that are well-understood from expts. and computations, it was found that natural selection enables enzymes to achieve catalytic efficiency in many different ways.  The factors promoting catalysis include transition state stabilization through electrostatic interactions, desolvation and reactant state destabilization, protein conformational change induced by differential interactions between the substrate and enzyme in the reactant state and transition state, and general acid and general base catalysis.  Quantum mech. tunneling appears also to be crit. in some enzyme reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosG2fgv8E8A7Vg90H21EOLACvtfcHk0lhcxCd-D7zW9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt12nsro%253D&md5=8b5d54ea600fbc9622c4bdd7ec6730f5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcr050293k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050293k%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DMajor%26aufirst%3DD.%2BT.%26aulast%3DNam%26aufirst%3DK.%26aulast%3DPu%26aufirst%3DJ.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26atitle%3DMechanisms%2520and%2520free%2520energies%2520of%2520enzymatic%2520reactions%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3188%26epage%3D3209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Aqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, J. E.</span><span> </span><span class="NLM_article-title">A new method for predicting binding affinity in computer-aided drug design</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fprotein%2F7.3.385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8177887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=385-391&author=J.+Aqvistauthor=C.+Medinaauthor=J.+E.+Samuelsson&title=A+new+method+for+predicting+binding+affinity+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A new method for predicting binding affinity in computer-aided drug design</span></div><div class="casAuthors">Aqvist J; Medina C; Samuelsson J E</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-91</span>
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    </div><div class="casAbstract">A new semi-empirical method for calculating free energies of binding from molecular dynamics (MD) simulations is presented.  It is based on standard thermodynamic cycles and on a linear approximation of polar and non-polar free energy contributions from the corresponding MD averages.  The method is tested on a set of endothiapepsin inhibitors and found to give accurate results both for absolute as well as relative free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTEx3uc8suH_BOk4JA3soTqfW6udTcc2eYTYfakQPB9J7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D&md5=c45572201328309d4fc849844d1c3b71</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F7.3.385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F7.3.385%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DSamuelsson%26aufirst%3DJ.%2BE.%26atitle%3DA%2520new%2520method%2520for%2520predicting%2520binding%2520affinity%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DProtein%2520Eng.%26date%3D1994%26volume%3D7%26spage%3D385%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1144</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fprotein%2F8.11.1137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8819979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=1137-1144&author=T.+Hanssonauthor=J.+Aqvist&title=Estimation+of+binding+free+energies+for+HIV+proteinase+inhibitors+by+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span></div><div class="casAuthors">Hansson, Tomas; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1137-44</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Abs. binding free energies for inhibitors of HIV-1 proteinase were estd. from mol. dynamics simulations by a recently reported linear approxn. procedure.  The results were in fairly good agreement with exptl. binding data.  Two of the inhibitors were very similar and, for comparison, their relative free energies of binding were also calcd. by free energy perturbation methods, giving virtually the same result.  Effects of cutoff radii and charge states of the protein model were examd.  The effects of pH on binding of one of the inhibitors were predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcv6yv6JgB1rVg90H21EOLACvtfcHk0lh5KISl0s8DFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D&md5=c77820746da2a692e7543f5f79789cca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.11.1137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.11.1137%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DEstimation%2520of%2520binding%2520free%2520energies%2520for%2520HIV%2520proteinase%2520inhibitors%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D1137%26epage%3D1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Calculation of absolute binding free energies for charged ligands and effects of long-range electrostatic interactions</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1587</span><span class="NLM_x">–</span> <span class="NLM_lpage">1597</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=1587-1597&author=J.+Aqvist&title=Calculation+of+absolute+binding+free+energies+for+charged+ligands+and+effects+of+long-range+electrostatic+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DCalculation%2520of%2520absolute%2520binding%2520free%2520energies%2520for%2520charged%2520ligands%2520and%2520effects%2520of%2520long-range%2520electrostatic%2520interactions%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D1587%26epage%3D1597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">An extended linear response method for determining free energies of hydration</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">10667</span><span class="NLM_x">–</span> <span class="NLM_lpage">10673</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100026a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=10667-10673&author=H.+A.+Carlsonauthor=W.+L.+Jorgensen&title=An+extended+linear+response+method+for+determining+free+energies+of+hydration"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">An Extended Linear Response Method for Determining Free Energies of Hydration</span></div><div class="casAuthors">Carlson, Heather A.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10667-73</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fourteen neutral org. mols. with diverse functionalities were used to det. the utility of equations based on linear response theory for the calcn. of free energies of hydration.  The equations contain terms proportional to the Coulombic and van der Waals components of the solute-solvent interaction energy and to an index for the cost of cavity formation such as solvent-accessible surface area.  The energy components can be obtained from Monte Carlo or mol. dynamics simulations of the solutes in water.  The methodol. was tested by using partial charges from fitting to the electrostatic potentials of ab initio 6-31G* wave functions and from OPLS potential functions.  Root-mean-square deviations between the results from the recommended linear response methods and exptl. free energies of hydration are ca. 0.8 kcal/mol.  Substituted benzenes, not included in the parameter development, were used to further evaluate the method performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcOGUicelNrVg90H21EOLACvtfcHk0lh5KISl0s8DFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D&md5=b91a7f287b63063c81c51082f694a78b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fj100026a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100026a034%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DAn%2520extended%2520linear%2520response%2520method%2520for%2520determining%2520free%2520energies%2520of%2520hydration%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D10667%26epage%3D10673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Jones-Hertzog, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Binding affinities for sulfonamide inhibitors with human thrombin using Monte Carlo simulations with a linear response method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1549</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm960684e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2sXis1ahu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=1539-1549&author=D.+K.+Jones-Hertzogauthor=W.+L.+Jorgensen&title=Binding+affinities+for+sulfonamide+inhibitors+with+human+thrombin+using+Monte+Carlo+simulations+with+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Affinities for Sulfonamide Inhibitors with Human Thrombin Using Monte Carlo Simulations with a Linear Response Method</span></div><div class="casAuthors">Jones-Hertzog, Deborah K.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1539-1549</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of sulfonamide inhibitors to human thrombin is examd. to evaluate the viability of calcg. free energies of binding, ΔGb, utilizing Monte Carlo (MC) statistical mechanics with a linear response approach.  Coulombic and van der Waals energy components detd. from MC simulations of the bound and unbound inhibitors solvated in water plus a solvent-accessible surface area term, as an index for cavity formation, were correlated with the free energies of binding for the inhibitor MD-805 and six derivs.  The best correlations yield an av. error of 0.8 kcal/mol for the seven binding affinities, which cover an obsd. range of 6.0 kcal/mol.  The MC simulations also provided insights into the interactions occurring in the active site and the origins of variations in ΔGb.  Equatorial placement of the carboxylate group at C2 in the piperidine ring of the inhibitors causes electrostatic destabilization with the side chain of Glu-H192, while axial disposition of the C4-Me group reduces favorable hydrophobic interactions in the P-pocket of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP22Kg6fLIUrVg90H21EOLACvtfcHk0lh5KISl0s8DFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1ahu70%253D&md5=ab9aa0e119329ab539c0dd5fcbd1a187</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm960684e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960684e%26sid%3Dliteratum%253Aachs%26aulast%3DJones-Hertzog%26aufirst%3DD.%2BK.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DBinding%2520affinities%2520for%2520sulfonamide%2520inhibitors%2520with%2520human%2520thrombin%2520using%2520Monte%2520Carlo%2520simulations%2520with%2520a%2520linear%2520response%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D1539%26epage%3D1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Estimation of the binding affinities of FKBP12 inhibitors using a linear response method</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">860</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=851-860&author=M.+L.+Lambauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Estimation+of+the+binding+affinities+of+FKBP12+inhibitors+using+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEstimation%2520of%2520the%2520binding%2520affinities%2520of%2520FKBP12%2520inhibitors%2520using%2520a%2520linear%2520response%2520method%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D851%26epage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">What determines the van der Waals coefficient beta in the LIE (linear interaction energy) method to estimate binding free energies using molecular dynamics simulations?</span> <span class="citation_source-journal">Proteins.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">402</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=395-402&author=W.+Wangauthor=J.+Wangauthor=P.+A.+Kollman&title=What+determines+the+van+der+Waals+coefficient+beta+in+the+LIE+%28linear+interaction+energy%29+method+to+estimate+binding+free+energies+using+molecular+dynamics+simulations%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DWhat%2520determines%2520the%2520van%2520der%2520Waals%2520coefficient%2520beta%2520in%2520the%2520LIE%2520%2528linear%2520interaction%2520energy%2529%2520method%2520to%2520estimate%2520binding%2520free%2520energies%2520using%2520molecular%2520dynamics%2520simulations%253F%26jtitle%3DProteins.%26date%3D1999%26volume%3D34%26spage%3D395%26epage%3D402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukacova, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">A combination of docking, QM/MM methods, and MD simulation for binding affinity estimation of metalloprotein ligands</span> <span class="citation_source-journal">J.Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5437</span><span class="NLM_x">–</span> <span class="NLM_lpage">5447</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049050v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5437-5447&author=A.+Khandelwalauthor=V.+Lukacovaauthor=D.+Comezauthor=D.+M.+Krollauthor=S.+Rahaauthor=S.+Balaz&title=A+combination+of+docking%2C+QM%2FMM+methods%2C+and+MD+simulation+for+binding+affinity+estimation+of+metalloprotein+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm049050v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049050v%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DLukacova%26aufirst%3DV.%26aulast%3DComez%26aufirst%3DD.%26aulast%3DKroll%26aufirst%3DD.%2BM.%26aulast%3DRaha%26aufirst%3DS.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DA%2520combination%2520of%2520docking%252C%2520QM%252FMM%2520methods%252C%2520and%2520MD%2520simulation%2520for%2520binding%2520affinity%2520estimation%2520of%2520metalloprotein%2520ligands%26jtitle%3DJ.Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5437%26epage%3D5447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">QM/MM linear response method distinguishes ligand affinities for closely related metalloproteins</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=326-339&author=A.+Khandelwalauthor=S.+Balaz&title=QM%2FMM+linear+response+method+distinguishes+ligand+affinities+for+closely+related+metalloproteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DQM%252FMM%2520linear%2520response%2520method%2520distinguishes%2520ligand%2520affinities%2520for%2520closely%2520related%2520metalloproteins%26jtitle%3DProteins%26date%3D2007%26volume%3D69%26spage%3D326%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">Improved estimation of ligand/macromolecule binding affinities by linear response approach using a combination of multi-mode MD simulation and QM/MM methods</span> <span class="citation_source-journal">J.Comput-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2Fs10822-007-9104-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=17333483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Kntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=131-137&author=A.+Khandelwalauthor=S.+Balaz&title=Improved+estimation+of+ligand%2Fmacromolecule+binding+affinities+by+linear+response+approach+using+a+combination+of+multi-mode+MD+simulation+and+QM%2FMM+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved estimation of ligand-macromolecule binding affinities by linear response approach using a combination of multi-mode MD simulation and QM/MM methods</span></div><div class="casAuthors">Khandelwal, Akash; Balaz, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">131-137</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based predictions of binding affinities of ligands binding to proteins by coordination bonds with transition metals, covalent bonds, and bonds involving charge re-distributions are hindered by the absence of proper force fields.  This shortcoming affects all methods which use force-field-based mol. simulation data on complex formation for affinity predictions.  One of the most frequently used methods in this category is the Linear Response (LR) approach of Åquist, correlating binding affinities with van der Waals and electrostatic energies, as extended by Jorgensen's inclusion of solvent-accessible surface areas.  All these terms represent the differences, upon binding, in the ensemble avs. of pertinent quantities, obtained from mol. dynamics (MD) or Monte Carlo simulations of the complex and of single components.  Here the authors report a modification of the LR approach by: (1) the replacement of the 2 energy terms through the single-point QM/MM energy of the time-averaged complex structure from an MD simulation; and (2) a rigorous consideration of multiple modes (mm) of binding.  The first extension alleviates the force-field related problems, while the second extension deals with the ligands exhibiting large-scale motions in the course of an MD simulation.  The second modification results in the correlation equation that is nonlinear in optimized coeffs., but does not lead to an increase in the no. of optimized coeffs.  The application of the resulting mm QM/MM LR approach to the inhibition of zinc-dependent gelatinase B (matrix metalloproteinase 9) by 28 hydroxamate ligands indicates a significant improvement of descriptive and predictive abilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsCwk74Or7LVg90H21EOLACvtfcHk0limiA_C5Du0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Kntr8%253D&md5=2eaa3d7972aa401f69c4ab60b0660a6b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9104-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9104-4%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DImproved%2520estimation%2520of%2520ligand%252Fmacromolecule%2520binding%2520affinities%2520by%2520linear%2520response%2520approach%2520using%2520a%2520combination%2520of%2520multi-mode%2520MD%2520simulation%2520and%2520QM%252FMM%2520methods%26jtitle%3DJ.Comput-Aided.%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D131%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Donnini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmueller, H.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics in explicit solvent with λ-dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1978</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct200061r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1962-1978&author=S.+Donniniauthor=F.+Tegelerauthor=G.+Groenhofauthor=H.+Grubmueller&title=Constant+pH+molecular+dynamics+in+explicit+solvent+with+%CE%BB-dynamics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics</span></div><div class="casAuthors">Donnini, Serena; Tegeler, Florian; Groenhof, Gerrit; Grubmuller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1962-1978</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PH is an important parameter in condensed-phase systems, because it dets. the protonation state of titratable groups and thus influences the structure, dynamics, and function of mols. in soln.  In most force field simulation protocols, however, the protonation state of a system (rather than its pH) is kept fixed and cannot adapt to changes of the local environment.  Here, we present a method, implemented within the MD package GROMACS, for const. pH mol. dynamics simulations in explicit solvent that is based on the λ-dynamics approach.  In the latter, the dynamics of the titrn. coordinate λ, which interpolates between the protonated and deprotonated states, is driven by generalized forces between the protonated and deprotonated states.  The hydration free energy, as a function of pH, is included to facilitate const. pH simulations.  The protonation states of titratable groups are allowed to change dynamically during a simulation, thus reproducing av. protonation probabilities at a certain pH.  The accuracy of the method is tested against titrn. curves of single amino acids and a dipeptide in explicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp1iVOhE1DLVg90H21EOLACvtfcHk0limiA_C5Du0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D&md5=0637c4d334ea58836db9e465401624c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fct200061r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200061r%26sid%3Dliteratum%253Aachs%26aulast%3DDonnini%26aufirst%3DS.%26aulast%3DTegeler%26aufirst%3DF.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DGrubmueller%26aufirst%3DH.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520with%2520%25CE%25BB-dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D1962%26epage%3D1978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vernier, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schindler, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reitz, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span> </span><span class="NLM_article-title">Pyrrolopyridine inhibitors of mitogen-activated protein kinase-activated protein kinase 2 (MK-2)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2647</span><span class="NLM_x">–</span> <span class="NLM_lpage">2654</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0611004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2647-2654&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=W.+F.+Vernierauthor=M.+W.+Mahoneyauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=J.+F.+Schindlerauthor=D.+B.+Reitzauthor=R.+J.+Mourey&title=Pyrrolopyridine+inhibitors+of+mitogen-activated+protein+kinase-activated+protein+kinase+2+%28MK-2%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm0611004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0611004%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DVernier%26aufirst%3DW.%2BF.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DSchindler%26aufirst%3DJ.%2BF.%26aulast%3DReitz%26aufirst%3DD.%2BB.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26atitle%3DPyrrolopyridine%2520inhibitors%2520of%2520mitogen-activated%2520protein%2520kinase-activated%2520protein%2520kinase%25202%2520%2528MK-2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2647%26epage%3D2654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span> </span><span class="NLM_article-title">Benzothiophene inhibitors of MK2. Part 1: Structure–activity relationships, assessments of selectivity and cellular potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4878</span><span class="NLM_x">–</span> <span class="NLM_lpage">4881</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4878-4881&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=S.+A.+Longauthor=B.+S.+Pierceauthor=M.+W.+Mahoneyauthor=R.+J.+Mourey&title=Benzothiophene+inhibitors+of+MK2.+Part+1%3A+Structure%E2%80%93activity+relationships%2C+assessments+of+selectivity+and+cellular+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DLong%26aufirst%3DS.%2BA.%26aulast%3DPierce%26aufirst%3DB.%2BS.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26atitle%3DBenzothiophene%2520inhibitors%2520of%2520MK2.%2520Part%25201%253A%2520Structure%25E2%2580%2593activity%2520relationships%252C%2520assessments%2520of%2520selectivity%2520and%2520cellular%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4878%26epage%3D4881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, M. D.</span><span> </span><span class="NLM_article-title">Benzothiophene inhibitors of MK2. Part 2: Improvements in kinase selectivity and cell potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4882</span><span class="NLM_x">–</span> <span class="NLM_lpage">4884</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4882-4884&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=S.+A.+Longauthor=B.+S.+Pierceauthor=M.+W.+Mahoneyauthor=R.+J.+Moureyauthor=M.+D.+Parikh&title=Benzothiophene+inhibitors+of+MK2.+Part+2%3A+Improvements+in+kinase+selectivity+and+cell+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DLong%26aufirst%3DS.%2BA.%26aulast%3DPierce%26aufirst%3DB.%2BS.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26atitle%3DBenzothiophene%2520inhibitors%2520of%2520MK2.%2520Part%25202%253A%2520Improvements%2520in%2520kinase%2520selectivity%2520and%2520cell%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4882%26epage%3D4884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Velcicky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feifel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawtin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heng, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppertz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroemer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moebitz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revesz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheufler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlapbach, A.</span><span> </span><span class="NLM_article-title">Novel 3-aminopyrazole inhibitors of MK-2 discovered by scaffold hopping strategy</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1293</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1293-1297&author=J.+Velcickyauthor=R.+Feifelauthor=S.+Hawtinauthor=R.+Hengauthor=C.+Huppertzauthor=G.+Kochauthor=M.+Kroemerauthor=H.+Moebitzauthor=L.+Reveszauthor=C.+Scheuflerauthor=A.+Schlapbach&title=Novel+3-aminopyrazole+inhibitors+of+MK-2+discovered+by+scaffold+hopping+strategy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVelcicky%26aufirst%3DJ.%26aulast%3DFeifel%26aufirst%3DR.%26aulast%3DHawtin%26aufirst%3DS.%26aulast%3DHeng%26aufirst%3DR.%26aulast%3DHuppertz%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DG.%26aulast%3DKroemer%26aufirst%3DM.%26aulast%3DMoebitz%26aufirst%3DH.%26aulast%3DRevesz%26aufirst%3DL.%26aulast%3DScheufler%26aufirst%3DC.%26aulast%3DSchlapbach%26aufirst%3DA.%26atitle%3DNovel%25203-aminopyrazole%2520inhibitors%2520of%2520MK-2%2520discovered%2520by%2520scaffold%2520hopping%2520strategy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D1293%26epage%3D1297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Hillig, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberspaecher, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteclaro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller-Tiemann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span> </span><span class="NLM_article-title">Structural basis for a high affinity inhibitor bound to protein kinase MK2</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">369</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">745</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2007&pages=735-745&author=R.+C.+Hilligauthor=U.+Eberspaecherauthor=F.+Monteclaroauthor=M.+Huberauthor=D.+Nguyenauthor=A.+Mengelauthor=B.+Muller-Tiemannauthor=U.+Egner&title=Structural+basis+for+a+high+affinity+inhibitor+bound+to+protein+kinase+MK2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DEberspaecher%26aufirst%3DU.%26aulast%3DMonteclaro%26aufirst%3DF.%26aulast%3DHuber%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DMengel%26aufirst%3DA.%26aulast%3DMuller-Tiemann%26aufirst%3DB.%26aulast%3DEgner%26aufirst%3DU.%26atitle%3DStructural%2520basis%2520for%2520a%2520high%2520affinity%2520inhibitor%2520bound%2520to%2520protein%2520kinase%2520MK2%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D369%26spage%3D735%26epage%3D745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Gaestel, M.</span><span> </span><span class="NLM_article-title">MAPKAP kinases, MKs—two’s company, three’s a crowd</span> <span class="citation_source-journal">Nature Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1038%2Fnrm1834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16421520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=120-130&author=M.+Gaestel&title=MAPKAP+kinases%2C+MKs%E2%80%94two%E2%80%99s+company%2C+three%E2%80%99s+a+crowd"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MAPKAP kinases - MKs - two's company, three's a crowd</span></div><div class="casAuthors">Gaestel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-130</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Downstream of MAP kinases (MAPKs), 3 structurally related MAPK-activated protein kinases (MAPKAPKs or MKs), MK2, MK3, and MK5, signal to diverse cellular targets.  Although there is no known common function for all 3 MKs, these kinases are involved in important processes.  MKs regulate gene expression at the transcriptional and post-transcriptional level, control cytoskeletal architecture and cell cycle progression, and are implicated in inflammation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI91xfvhmqhrVg90H21EOLACvtfcHk0lhuIz3iG1ookg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWntr4%253D&md5=1ae824b7d680a574a7c764bd61ac43aa</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrm1834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1834%26sid%3Dliteratum%253Aachs%26aulast%3DGaestel%26aufirst%3DM.%26atitle%3DMAPKAP%2520kinases%252C%2520MKs%25E2%2580%2594two%25E2%2580%2599s%2520company%252C%2520three%25E2%2580%2599s%2520a%2520crowd%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D120%26epage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Kotlyarov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neininger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birchmeier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volk, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaestel, M.</span><span> </span><span class="NLM_article-title">MAPKAP kinase 2 is essential for LPS-induced TNFalpha biosynthesis</span> <span class="citation_source-journal">Nature Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1038%2F10061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=10559880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVGqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=94-97&author=A.+Kotlyarovauthor=A.+Neiningerauthor=C.+Schubertauthor=R.+Eckertauthor=C.+Birchmeierauthor=H.+D.+Volkauthor=M.+Gaestel&title=MAPKAP+kinase+2+is+essential+for+LPS-induced+TNFalpha+biosynthesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">MAPKAP kinase 2 is essential for LPS-induced TNF-α biosynthesis</span></div><div class="casAuthors">Kotlyarov, Alexey; Neininger, Armin; Schubert, Carola; Eckert, Rolf; Birchmeier, Carmen; Volk, Hans-Dieter; Gaestel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">94-97</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">MAPKAP kinase 2 (MK2) is one of several kinases that are regulated through direct phosphorylation by p38 MAP kinase.  By introducing a targeted mutation into the mouse MK2 gene, the authors have detd. the physiol. function of MK2 in vivo.  Mice that lack MK2 show increased stress resistance and survive LPS-induced endotoxic shock.  This is due to a redn. of ∼90% in the prodn. of tumor necrosis factor-α (TNF-α) and not to a change in signaling from the TNF receptor.  The level and stability of TNF-α mRNA is not reduced and TNF-α secretion is not affected.  Thus, MK2 is an essential component in the inflammatory response which regulates biosynthesis of TNF-α at a post-transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtiobBLSnV77Vg90H21EOLACvtfcHk0lgEB6IH9oNtyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVGqsLY%253D&md5=5fb502616639b298341431eea8009cc2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2F10061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10061%26sid%3Dliteratum%253Aachs%26aulast%3DKotlyarov%26aufirst%3DA.%26aulast%3DNeininger%26aufirst%3DA.%26aulast%3DSchubert%26aufirst%3DC.%26aulast%3DEckert%26aufirst%3DR.%26aulast%3DBirchmeier%26aufirst%3DC.%26aulast%3DVolk%26aufirst%3DH.%2BD.%26aulast%3DGaestel%26aufirst%3DM.%26atitle%3DMAPKAP%2520kinase%25202%2520is%2520essential%2520for%2520LPS-induced%2520TNFalpha%2520biosynthesis%26jtitle%3DNature%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D94%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Dambach, D. M.</span><span> </span><span class="NLM_article-title">Potential adverse effects associated with inhibition of p38α/β MAP Kinases</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">929</span><span class="NLM_x">–</span> <span class="NLM_lpage">939</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.2174%2F1568026054985911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16178738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFeitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=929-939&author=D.+M.+Dambach&title=Potential+adverse+effects+associated+with+inhibition+of+p38%CE%B1%2F%CE%B2+MAP+Kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Potential adverse effects associated with inhibition of p38α/β MAP kinases</span></div><div class="casAuthors">Dambach, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibitors of p38 MAP kinases show promise for the treatment of inflammatory and immunol. disorders and some cancers.  There is a substantial body of exptl. evidence across several organ systems suggesting that p38 also mediates developmental, differentiation and proliferation processes.  As a consequence of the wide-ranging regulatory role of p38 kinase in diverse cellular processes, the possibility of adverse events resulting from undesired pharmacol. activity is a major concern for the p38 inhibitor drug class.  Taking into consideration the limitations of exptl. modeling systems, together the data may indicate that profound inhibition of p38 has the potential to impact these processes during fetal or neonatal development.  The difficulty comes in extrapolating these findings to predict potential adverse effects under conditions of partial inhibition of p38 activity, and in an adult population in which these processes are typically only recapitulated during repair or adaptive responses.  As such, the goal of this review of the targets of p38 activity is to bring an awareness of the those organ systems that should be monitored for potential toxicity, as well as to present a potential mechanistic basis for such monitoring or for investigation of adverse effects that may develop with administration of a p38 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9YoS9QjJnfrVg90H21EOLACvtfcHk0lh0JrFil4vE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFeitrbM&md5=ece8a45395222f3039bc51485453ac15</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568026054985911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054985911%26sid%3Dliteratum%253Aachs%26aulast%3DDambach%26aufirst%3DD.%2BM.%26atitle%3DPotential%2520adverse%2520effects%2520associated%2520with%2520inhibition%2520of%2520p38%25CE%25B1%252F%25CE%25B2%2520MAP%2520Kinases%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D929%26epage%3D939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Duraisamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajpai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bughani, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastidar, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, P.</span><span> </span><span class="NLM_article-title">MK2: A novel molecular target for anti-inflammatory therapy</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">921</span><span class="NLM_x">–</span> <span class="NLM_lpage">936</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1517%2F14728222.12.8.921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=18620516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ChtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=921-936&author=S.+Duraisamyauthor=M.+Bajpaiauthor=U.+Bughaniauthor=S.+G.+Dastidarauthor=A.+Rayauthor=P.+Chopra&title=MK2%3A+A+novel+molecular+target+for+anti-inflammatory+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MK2: a novel molecular target for anti-inflammatory therapy</span></div><div class="casAuthors">Duraisamy, Senthil; Bajpai, Malini; Bughani, Usha; Dastidar, Sunanda G.; Ray, Abhijit; Chopra, Puneet</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">921-936</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinase (MAPK)-activated protein kinase 2 (MK2) is activated upon stress by p38 MAPK.  MK2 is stimulated in a wide range of inflammatory conditions and its catalytic activity is required for cytokine prodn., cell migration and is a potential drug target for inflammatory diseases.  Disruption of MK2 leads to a redn. in TNF-α prodn.  MK2-mediated pro-inflammatory cytokine prodn. has been demonstrated in several inflammatory conditions where TNF-α plays a role.  We discuss the development of specific MK2 inhibitors for the treatment of inflammatory diseases.  Inhibition of the p38 MAPK pathway may have therapeutic uses for inflammatory diseases.  However, blocking p38 MAPK activation in vivo is not advisable due to toxicity, significant off-target effects, and lack of oral bioavailability.  This concern may be countered by the use of MK2 inhibitors that can dissect the pathways downstream of p38 without affecting addnl. cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_tMhx866ALbVg90H21EOLACvtfcHk0lh0JrFil4vE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ChtL0%253D&md5=46eefd34bc965b84b34234314fdb6c98</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F14728222.12.8.921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.12.8.921%26sid%3Dliteratum%253Aachs%26aulast%3DDuraisamy%26aufirst%3DS.%26aulast%3DBajpai%26aufirst%3DM.%26aulast%3DBughani%26aufirst%3DU.%26aulast%3DDastidar%26aufirst%3DS.%2BG.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DChopra%26aufirst%3DP.%26atitle%3DMK2%253A%2520A%2520novel%2520molecular%2520target%2520for%2520anti-inflammatory%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2008%26volume%3D12%26spage%3D921%26epage%3D936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Hilal, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karickhoff, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreira, L. A.</span><span> </span><span class="NLM_article-title">Estimation of microscopic, zwitterionic ionization constants, isoelectric point and molecular speciation of organic compounds</span> <span class="citation_source-journal">Talanta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">827</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1999&pages=827-840&author=S.+H.+Hilalauthor=S.+W.+Karickhoffauthor=L.+A.+Carreira&title=Estimation+of+microscopic%2C+zwitterionic+ionization+constants%2C+isoelectric+point+and+molecular+speciation+of+organic+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHilal%26aufirst%3DS.%2BH.%26aulast%3DKarickhoff%26aufirst%3DS.%2BW.%26aulast%3DCarreira%26aufirst%3DL.%2BA.%26atitle%3DEstimation%2520of%2520microscopic%252C%2520zwitterionic%2520ionization%2520constants%252C%2520isoelectric%2520point%2520and%2520molecular%2520speciation%2520of%2520organic%2520compounds%26jtitle%3DTalanta%26date%3D1999%26volume%3D50%26spage%3D827%26epage%3D840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J. H.</span><span> </span><span class="NLM_article-title">Very fast empirical prediction and rationalization of protein p<i>K</i><sub>a</sub> values</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">704</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fprot.20660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16231289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=704-721&author=H.+Liauthor=A.+D.+Robertsonauthor=J.+H.+Jensen&title=Very+fast+empirical+prediction+and+rationalization+of+protein+pKa+values"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Very fast empirical prediction and rationalization of protein pKa values</span></div><div class="casAuthors">Li, Hui; Robertson, Andrew D.; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">704-721</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A very fast empirical method is presented for structure-based protein pKa prediction and rationalization.  The desolvation effects and intra-protein interactions, which cause variations in pKa values of protein ionizable groups, are empirically related to the positions and chem. nature of the groups proximate to the pKa sites.  A computer program is written to automatically predict pKa values based on these empirical relationships within a couple of seconds.  Unusual pKa values at buried active sites, which are among the most interesting protein pKa values, are predicted very well with the empirical method.  A test on 233 carboxyl, 12 cysteine, 45 histidine, and 24 lysine pKa values in various proteins shows a root-mean-square deviation (RMSD) of 0.89 from exptl. values.  Removal of the 29 pKa values that are upper or lower limits results in an RMSD = 0.79 for the remaining 285 pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYkiHyYz6FobVg90H21EOLACvtfcHk0lhxqosDLelMVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN&md5=9e33529954c65867929326b99dac493b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fprot.20660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20660%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520empirical%2520prediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%26jtitle%3DProteins%26date%3D2005%26volume%3D61%26spage%3D704%26epage%3D721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Multiple-site titration and molecular modeling: Two rapid methods for computing energies and forces for ionizable groups in proteins</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1993&pages=266-282&author=M.+K.+Gilson&title=Multiple-site+titration+and+molecular+modeling%3A+Two+rapid+methods+for+computing+energies+and+forces+for+ionizable+groups+in+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DMultiple-site%2520titration%2520and%2520molecular%2520modeling%253A%2520Two%2520rapid%2520methods%2520for%2520computing%2520energies%2520and%2520forces%2520for%2520ionizable%2520groups%2520in%2520proteins%26jtitle%3DProteins%26date%3D1993%26volume%3D15%26spage%3D266%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Antosiewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Prediction of pH-dependent properties of proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">238</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1006%2Fjmbi.1994.1301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8176733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2cXktVCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=1994&pages=415-436&author=J.+Antosiewiczauthor=J.+A.+McCammonauthor=M.+K.+Gilson&title=Prediction+of+pH-dependent+properties+of+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pH-dependent properties of proteins</span></div><div class="casAuthors">Antosiewicz, Jan; McCammon, J. Andrew; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">415-36</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe what may be the most accurate approach currently available for the calcn. of the pKas of ionizable groups in proteins.  The accuracy is assessed by comparison of computed pKas with 60 measured pKas in a total of seven proteins.  The overall root-mean-square error is 0.89 pKa units.  Linear regression anal. of computed vs. measured pKas yields a slope of 0.95, yr-intercept of -0.02 and a correlation coeff. of 0.96.  The proposed approach also picks out many of the shifted pKas of groups in enzyme active sites and special salt bridges.  However, it does yield several over-shifted pKas and tends to underest. pKa shifts which result from desolvation effects.  The authors examine the ability of the new approach to reproduce the dependence of protein stability upon pH, using the ionization polynomial formalism.  Overall features of the stability curves are reproduced, but the quant. agreement is not particularly good.  The reasons for the disagreement may have to do both with insufficient accuracy in the theory and with uncertainty in the nature of the unfolded state of proteins.  The methodol. described here is based upon finite difference solns. of the Poisson-Boltzmann equation.  Its success depends upon the use of the rather high protein dielec. const. of 20.  However, theor. considerations and the fact that pKa shifts which result from desolvation are underestd. here imply that the dielec. const. of the protein interior actually is lower than 20.  The authors suggest that the high protein dielec. const. improves the overall agreement with expt. because it accounts approx. for phenomena which tend to mitigate pKa shifts and which are not specifically included in the model.  These include conformational relaxation and specific ion-binding.  Future models based upon a low protein dielec. const. and treating such phenomena explicitly might yield improved agreement with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy0UmLyANFBrVg90H21EOLACvtfcHk0lhxqosDLelMVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVCntbk%253D&md5=f029ddd8fcbee0708fe44e0ee5e0e29c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1301%26sid%3Dliteratum%253Aachs%26aulast%3DAntosiewicz%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DPrediction%2520of%2520pH-dependent%2520properties%2520of%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D238%26spage%3D415%26epage%3D436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Antosiewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computing ionization states of proteins with a detailed charge model</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1633</span><span class="NLM_x">–</span> <span class="NLM_lpage">1644</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=1633-1644&author=J.+Antosiewiczauthor=J.+W.+Briggsauthor=A.+H.+Elcockauthor=M.+K.+Gilsonauthor=J.+A.+McCammon&title=Computing+ionization+states+of+proteins+with+a+detailed+charge+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntosiewicz%26aufirst%3DJ.%26aulast%3DBriggs%26aufirst%3DJ.%2BW.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputing%2520ionization%2520states%2520of%2520proteins%2520with%2520a%2520detailed%2520charge%2520model%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D1633%26epage%3D1644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Modeling protonation equilibria in biomolecules</span>. In  <span class="citation_source-book">Computer Simulation of Biomolecular Systems</span>; <span class="NLM_contrib-group">van Gunsteren, W. F.; Weiner, P. K.; Wilkinson, A. J.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">3</span>, pp  <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2F978-94-017-1120-3_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=199-222&author=M.+K.+Gilsonauthor=W.+F.+van+Gunsteren&author=P.+K.+Weiner&author=A.+J.+Wilkinson&title=Computer+Simulation+of+Biomolecular+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-94-017-1120-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-017-1120-3_7%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DModeling%2520protonation%2520equilibria%2520in%2520biomolecules%26btitle%3DComputer%2520Simulation%2520of%2520Biomolecular%2520Systems%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26pub%3DSpringer%26date%3D1997%26volume%3D3%26spage%3D199%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Improving the accuracy of protein p<i>K</i><sub>a</sub> calculations: Conformational averaging versus the average structure</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">145</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=145-158&author=H.+W.+T.+Van+Vlijmenauthor=M.+Schaeferauthor=M.+Karplus&title=Improving+the+accuracy+of+protein+pKa+calculations%3A+Conformational+averaging+versus+the+average+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DImproving%2520the%2520accuracy%2520of%2520protein%2520pKa%2520calculations%253A%2520Conformational%2520averaging%2520versus%2520the%2520average%2520structure%26jtitle%3DProteins%26date%3D1998%26volume%3D33%26spage%3D145%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Prediction of p<i>K</i><sub>a</sub> shifts in proteins using a combination of molecular mechanical and continuum solvent calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1865</span><span class="NLM_x">–</span> <span class="NLM_lpage">1872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=15376253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1865-1872&author=B.+Kuhnauthor=P.+A.+Kollmanauthor=M.+Stahl&title=Prediction+of+pKa+shifts+in+proteins+using+a+combination+of+molecular+mechanical+and+continuum+solvent+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pKa shifts in proteins using a combination of molecular mechanical and continuum solvent calculations</span></div><div class="casAuthors">Kuhn, Bernd; Kollman, Peter A.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1865-1872</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The prediction of pKa shifts of ionizable groups in proteins is of great relevance for a no. of important biol. phenomena.  We present an implementation of the MM-GBSA approach, which combines mol. mech. (MM) and generalized Born (GB) continuum solvent energy terms, to the calcn. of pKa values of a panel of nine proteins, including 69 individual comparisons with expt.  While applied so far mainly to the calcn. of biomol. binding free energies, we show that this method can also be used for the estn. of protein pKa shifts, with an accuracy around 1 pKa unit, even for strongly shifted residues.  Our anal. reveals that the nonelectrostatic terms that are part of the MM-GBSA free energy expression are important contributors to improved prediction accuracy.  This suggests that most of the previous approaches that focus only on electrostatic interactions could be improved by adding other nonpolar energy terms to their free energy expression.  Interestingly, our method yields best accuracy at protein dielec. consts. of εint = 2-4, which is in contrast to previous approaches that peak at higher εint ≥ 8.  An important component of our procedure is an intermediate minimization step of each protonation state involving different rotamers and tautomers as a way to explicitly model protein relaxation upon (de)protonation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ5j9GZtXBELVg90H21EOLACvtfcHk0lh_-e1NFr_LPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVyjsr0%253D&md5=e91b3f62ac9f8cd67beeeb064fd96aed</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20111%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DPrediction%2520of%2520pKa%2520shifts%2520in%2520proteins%2520using%2520a%2520combination%2520of%2520molecular%2520mechanical%2520and%2520continuum%2520solvent%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1865%26epage%3D1872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Jensen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, P. A.</span><span> </span><span class="NLM_article-title">Prediction and rationalization of protein p<i>K</i><sub>a</sub> values using QM and QM/MM methods</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6634</span><span class="NLM_x">–</span> <span class="NLM_lpage">6643</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp051922x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVChtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6634-6643&author=J.+H.+Jensenauthor=H.+Liauthor=A.+D.+Robertsonauthor=P.+A.+Molina&title=Prediction+and+rationalization+of+protein+pKa+values+using+QM+and+QM%2FMM+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction and Rationalization of Protein pKa Values Using QM and QM/MM Methods</span></div><div class="casAuthors">Jensen, Jan H.; Li, Hui; Robertson, Andrew D.; Molina, Pablo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6634-6643</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe the development and application of a computational method for the prediction and rationalization of pKa values of ionizable residues in proteins, based on ab initio quantum mechanics (QM) and the effective fragment potential (EFPs) method (a hybrid QM/MM method).  The theor. developments include (1) a covalent boundary method based on frozen localized orbitals, (2) divide-and-conquer methods for the ab initio computation of protein EFPs consisting of multipoles up to octupoles and dipole polarizability tensors, (3) a method for computing vibrational free energies for a localized mol. region, and (4) solns. of the polarized continuum model of bulk solvation equations for protein-sized systems.  The QM-based pKa prediction method is one of the most accurate methods currently available and can be used in cases where other pKa prediction methods fail.  Preliminary anal. of the computed results indicate that many pKa values (1) are primarily detd. by hydrogen bonds rather than long-range charge-charge interactions and (2) are relatively insensitive to large-scale dynamical fluctuations of the protein structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHFgRei3zrGLVg90H21EOLACvtfcHk0lh_-e1NFr_LPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVChtr0%253D&md5=58696f960a47c34c95d0592d6b9299d0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjp051922x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp051922x%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DMolina%26aufirst%3DP.%2BA.%26atitle%3DPrediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%2520using%2520QM%2520and%2520QM%252FMM%2520methods%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2005%26volume%3D109%26spage%3D6634%26epage%3D6643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Gordon, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoja, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heath, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span> </span><span class="NLM_article-title">H++: A server for estimating p<i>K</i><sub>a</sub>s and adding missing hydrogens to macromolecules</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">W368</span><span class="NLM_x">–</span> <span class="NLM_lpage">W371</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+A+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520A%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Bas, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J. H.</span><span> </span><span class="NLM_article-title">Very fast prediction and rationalization of p<i>K</i><sub>a</sub> values for protein–ligand complexes</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">765</span><span class="NLM_x">–</span> <span class="NLM_lpage">783</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=765-783&author=D.+C.+Basauthor=D.+M.+Rogersauthor=J.+H.+Jensen&title=Very+fast+prediction+and+rationalization+of+pKa+values+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBas%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DD.%2BM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520prediction%2520and%2520rationalization%2520of%2520pKa%2520values%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DProteins%26date%3D2008%26volume%3D73%26spage%3D765%26epage%3D783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Song, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunner, M. R.</span><span> </span><span class="NLM_article-title">MCCE2: Improving protein p<i>K</i><sub>a</sub> calculations with extensive side chain rotamer sampling</span> <span class="citation_source-journal">J.Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2231</span><span class="NLM_x">–</span> <span class="NLM_lpage">2247</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=19274707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOrsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2231-2247&author=Y.+F.+Songauthor=J.+J.+Maoauthor=M.+R.+Gunner&title=MCCE2%3A+Improving+protein+pKa+calculations+with+extensive+side+chain+rotamer+sampling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MCCE2: Improving protein pKa calculations with extensive side chain rotamer sampling</span></div><div class="casAuthors">Song, Yifan; Mao, Junjun; Gunner, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2231-2247</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Multiconformation continuum electrostatics (MCCE) explores different conformational degrees of freedom in Monte Carlo calcns. of protein residue and ligand pKas.  Explicit changes in side chain conformations throughout a titrn. create a position dependent, heterogeneous dielec. response giving a more accurate picture of coupled ionization and position changes.  The MCCE2 methods for choosing a group of input heavy atom and proton positions are described.  The pKas calcd. with different isosteric conformers, heavy atom rotamers and proton positions, with different degrees of optimization are tested against a curated group of 305 exptl. pKas in 33 proteins.  QUICK calcns., with rotation around Asn and Gln termini, sampling His tautomers and torsion min. hydroxyls yield an RMSD of 1.34 with 84% of the errors being <1.5 pH units.  FULL calcns. adding heavy atom rotamers and side chain optimization yield an RMSD of 0.90 with 90% of the errors <1.5 pH unit.  Good results are also found for pKas in the membrane protein bacteriorhodopsin.  The inclusion of extra side chain positions distorts the dielec. boundary and also biases the calcd. pKas by creating more neutral than ionized conformers.  Methods for correcting these errors are introduced.  Calcns. are compared with multiple x-ray and NMR derived structures in 36 sol. proteins.  Calcns. with x-ray structures give significantly better pKas.  Results with the default protein dielec. const. of 4 are as good as those using a value of 8.  The MCCE2 program can be downloaded from http://www.sci.ccny.cuny.edu/~mcce.  © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY__IipXaBcLVg90H21EOLACvtfcHk0lh_-e1NFr_LPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOrsLjM&md5=43c57ec68c03af687d898702f8cc8ca5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%2BF.%26aulast%3DMao%26aufirst%3DJ.%2BJ.%26aulast%3DGunner%26aufirst%3DM.%2BR.%26atitle%3DMCCE2%253A%2520Improving%2520protein%2520pKa%2520calculations%2520with%2520extensive%2520side%2520chain%2520rotamer%2520sampling%26jtitle%3DJ.Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2231%26epage%3D2247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K</i><sub>I</sub>) and the concentration of inhibitor which causes 50% inhibition (<i>I</i><sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lgewNK9sCQCnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgewNK9sCQCnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0lgewNK9sCQCnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Vreven, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morokuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farkas, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisch, M. J.</span><span> </span><span class="NLM_article-title">Geometry optimization with QM/MM, ONIOM, and other combined methods. I. Microiterations and constraints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">769</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.10156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12666168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFWqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=760-769&author=T.+Vrevenauthor=K.+Morokumaauthor=O.+Farkasauthor=H.+B.+Schlegelauthor=M.+J.+Frisch&title=Geometry+optimization+with+QM%2FMM%2C+ONIOM%2C+and+other+combined+methods.+I.+Microiterations+and+constraints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Geometry optimization with QM/MM, ONIOM, and other combined methods. I. Microiterations and constraints</span></div><div class="casAuthors">Vreven, Thom; Morokuma, Keiji; Farkas, Odon; Schlegel, H. Bernhard; Frisch, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">760-769</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hybrid energy methods such as QM/MM and ONIOM, that combine different levels of theory into one calcn., have been very successful in describing large systems.  Geometry optimization methods can take advantage of the partitioning of these calcns. into a region treated at a quantum mech. (QM) level of theory and the larger, remaining region treated by an inexpensive method such as mol. mechanics (MM).  A series of microiterations can be employed to fully optimize the MM region for each optimization step in the QM region.  Cartesian coordinates are used for the MM region and are chosen so that the internal coordinates of the QM region remain const. during the microiterations.  The coordinates of the MM region are augmented to permit rigid body translation and rotation of the QM region.  This is essential if any atoms in the MM region are constrained, but it also improves the efficiency of unconstrained optimizations.  Because of the microiterations, special care is needed for the optimization step in the QM region so that the system remains in the same local valley during the course of the optimization.  The optimization methodol. with microiterations, constraints, and step-size control are illustrated by calcns. on bacteriorhodopsin and other systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLKbGjxXu0LVg90H21EOLACvtfcHk0lgewNK9sCQCnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFWqtrc%253D&md5=d8dbb3fc8ca569cf5ec7ef85bd20694d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10156%26sid%3Dliteratum%253Aachs%26aulast%3DVreven%26aufirst%3DT.%26aulast%3DMorokuma%26aufirst%3DK.%26aulast%3DFarkas%26aufirst%3DO.%26aulast%3DSchlegel%26aufirst%3DH.%2BB.%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26atitle%3DGeometry%2520optimization%2520with%2520QM%252FMM%252C%2520ONIOM%252C%2520and%2520other%2520combined%2520methods.%2520I.%2520Microiterations%2520and%2520constraints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D760%26epage%3D769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Vreven, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byun, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komiromi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dapprich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morokuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisch, M. J.</span><span> </span><span class="NLM_article-title">Combining quantum mechanics methods with molecular mechanics methods in ONIOM</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct050289g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFWqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=815-826&author=T.+Vrevenauthor=K.+S.+Byunauthor=I.+Komiromiauthor=S.+Dapprichauthor=A.+Johnauthor=K.+Morokumaauthor=M.+J.+Frisch&title=Combining+quantum+mechanics+methods+with+molecular+mechanics+methods+in+ONIOM"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Quantum Mechanics Methods with Molecular Mechanics Methods in ONIOM</span></div><div class="casAuthors">Vreven, Thom; Byun, K. Suzie; Komaromi, Istvan; Dapprich, Stefan; Montgomery, John A., Jr.; Morokuma, Keiji; Frisch, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The purpose of this paper is 2-fold.  First, we present several extensions to the ONIOM(QM:MM) scheme.  In its original formulation, the electrostatic interaction between the regions is included at the classical level.  Here we present the extension to electronic embedding.  We show how the behavior of ONIOM with electronic embedding can be more stable than QM/MM with electronic embedding.  We also investigate the link atom correction, which is implicit in ONIOM but not in QM/MM.  Second, we demonstrate some of the practical aspects of ONIOM(QM:MM) calcns.  Specifically, we show that the potential surface can be discontinuous when there is bond breaking and forming closer than three bonds from the MM region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorbf2n2RA2NbVg90H21EOLACvtfcHk0liOA65xOAqDqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFWqsb0%253D&md5=3cf69310302f73c284f6721b3996d4a5</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct050289g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct050289g%26sid%3Dliteratum%253Aachs%26aulast%3DVreven%26aufirst%3DT.%26aulast%3DByun%26aufirst%3DK.%2BS.%26aulast%3DKomiromi%26aufirst%3DI.%26aulast%3DDapprich%26aufirst%3DS.%26aulast%3DJohn%26aufirst%3DA.%26aulast%3DMorokuma%26aufirst%3DK.%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26atitle%3DCombining%2520quantum%2520mechanics%2520methods%2520with%2520molecular%2520mechanics%2520methods%2520in%2520ONIOM%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2006%26volume%3D2%26spage%3D815%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Frisch, M. J.</span>Gaussian 09, version A.1; <span class="NLM_publisher-name">Gaussian Inc.</span>, <span class="NLM_publisher-loc">Wallingford, CT 06492, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=M.+J.+Frisch&title=Gaussian+09%2C+version+A.1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%252C%2520version%2520A.1%26pub%3DGaussian%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0liOA65xOAqDqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span>Premier Solver Platform version 10.5; <span class="NLM_publisher-name">Frontline Systems Inc.</span>: <span class="NLM_publisher-loc">Incline Village, NV, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Premier+Solver+Platform+version+10.5%3B+Frontline+Systems+Inc.%3A+Incline+Village%2C+NV%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPremier%2520Solver%2520Platform%2520version%252010.5%26pub%3DFrontline%2520Systems%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span>Sybyl-X version 1.1; <span class="NLM_publisher-name">Tripos Inc.</span>: <span class="NLM_publisher-loc">St. Louis, MO, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sybyl-X+version+1.1%3B+Tripos+Inc.%3A+St.+Louis%2C+MO%2C+USA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSybyl-X%2520version%25201.1%26pub%3DTripos%2520Inc%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Mulliken, R. S.</span><span> </span><span class="NLM_article-title">Electronic population analysis on LCAO-MO [linear combination of atomic orbital-molecular orbital] molecular wave functions. I</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1955</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1833</span><span class="NLM_x">–</span> <span class="NLM_lpage">1840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1955&pages=1833-1840&author=R.+S.+Mulliken&title=Electronic+population+analysis+on+LCAO-MO+%5Blinear+combination+of+atomic+orbital-molecular+orbital%5D+molecular+wave+functions.+I"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMulliken%26aufirst%3DR.%2BS.%26atitle%3DElectronic%2520population%2520analysis%2520on%2520LCAO-MO%2520%255Blinear%2520combination%2520of%2520atomic%2520orbital-molecular%2520orbital%255D%2520molecular%2520wave%2520functions.%2520I%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1955%26volume%3D23%26spage%3D1833%26epage%3D1840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span>Jaguar; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">LLC Portland, OR, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Jaguar%3B+Schr%C3%B6dinger%3A+LLC+Portland%2C+OR%2C+USA%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DJaguar%26pub%3DSchr%25C3%25B6dinger%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. III. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+III.+The+role+of+exact+exchange"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0ljJCcw62EFcnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520III.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Hay, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wadt, W. R.</span><span> </span><span class="NLM_article-title">Ab initio effective core potentials for molecular calculations. Potentials for the transition metal atoms scandium to mercury</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">270</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.448799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL2MXhtlyju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1985&pages=270-283&author=P.+J.+Hayauthor=W.+R.+Wadt&title=Ab+initio+effective+core+potentials+for+molecular+calculations.+Potentials+for+the+transition+metal+atoms+scandium+to+mercury"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Ab initio effective core potentials for molecular calculations.  Potentials for the transition metal atoms scandium to mercury</span></div><div class="casAuthors">Hay, P. Jeffrey; Wadt, Willard R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">270-83</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Ab initio effective core potentials (ECP's) were generated to replace the Coulomb, exchange, and core-orthogonality effects of the chem. inert core electron in the transition metal atoms Sc to Hg.  For the second and third transition series relative ECP's were generated which also incorporate the mass-velocity and Darwin relativistic effects into the potential.  The ab initio ECP's should facilitate valence electron calcns. on mols. contg. transition-metal atoms with accuracies approaching all-electron calcns. at a fraction of the computational cost.  Analytic fits to the potentials are presented for use in multicenter integral evaluation.  Gaussian orbital valence basis sets are developed for the (3d,4s,4p), (4d,5s,5p), and (5d,6s,6p) orbitals of the first, second, and third transition series atoms, resp.  All-electron and valence-electron at. excitation energies are also compared for the low-lying states of Sc-Hg, and the valence-electron calcns. reproduce the all-electron excitation energies (typically within a few tenths of an eV).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIEl284u2R9rVg90H21EOLACvtfcHk0ljJCcw62EFcnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhtlyju70%253D&md5=29271d2a54b5c81acd19762c570e64d7</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1063%2F1.448799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448799%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DP.%2BJ.%26aulast%3DWadt%26aufirst%3DW.%2BR.%26atitle%3DAb%2520initio%2520effective%2520core%2520potentials%2520for%2520molecular%2520calculations.%2520Potentials%2520for%2520the%2520transition%2520metal%2520atoms%2520scandium%2520to%2520mercury%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D82%26spage%3D270%26epage%3D283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Wadt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, P. J.</span><span> </span><span class="NLM_article-title">Ab initio effective core potentials for molecular calculations. Potentials for main group elements sodium to bismuth</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.448800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL2MXht1SjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1985&pages=284-298&author=W.+R.+Wadtauthor=P.+J.+Hay&title=Ab+initio+effective+core+potentials+for+molecular+calculations.+Potentials+for+main+group+elements+sodium+to+bismuth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Ab initio effective core potentials for molecular calculations.  Potentials for main group elements sodium to bismuth</span></div><div class="casAuthors">Wadt, Willard R.; Hay, P. Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">284-98</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A consistent set of ab initio effective core potentials (ECP) was generated for the main group elements from Na to Bi by using the procedure originally developed by Kahn (1976).  For Rb to Bi the ECP's are generated from the relativistic Hartree-Fock at. wave functions of Cowan which incorporate the Darwin and mass-velocity terms.  Energy-optimized valence basis sets of (3s3p) primitive Gaussians are presented for use with the ECP's.  Comparisons between all-electron and valence-electron ECP calcns. are presented for NaF, NaCl, Cl2, Cl2-, Br2, Br2-, and Xe2+.  The results show that the av. errors introduced by the ECP's are generally only a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt2lGOejwqOLVg90H21EOLACvtfcHk0ljJCcw62EFcnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXht1SjtLk%253D&md5=16288166d7c3a65ba452c4673b7c7d2a</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1063%2F1.448800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448800%26sid%3Dliteratum%253Aachs%26aulast%3DWadt%26aufirst%3DW.%2BR.%26aulast%3DHay%26aufirst%3DP.%2BJ.%26atitle%3DAb%2520initio%2520effective%2520core%2520potentials%2520for%2520molecular%2520calculations.%2520Potentials%2520for%2520main%2520group%2520elements%2520sodium%2520to%2520bismuth%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D82%26spage%3D284%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, J. W.</span><span> </span><span class="NLM_article-title">Consistent treatment of inter- and intramolecular polarization in molecular mechanics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1506</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.10127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12395419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1497-1506&author=P.+Renauthor=J.+W.+Ponder&title=Consistent+treatment+of+inter-+and+intramolecular+polarization+in+molecular+mechanics+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Consistent treatment of inter- and intramolecular polarization in molecular mechanics calculations</span></div><div class="casAuthors">Ren, Pengyu; Ponder, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1497-1506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A protocol is described for the treatment of mol. polarization in force field calcns.  The resulting model is consistent in that both inter- and intramol. polarization are handled within a single scheme.  An anal. formula for removing intramol. polarization from a set of at. multipoles for an arbitrary static structure or conformation is given.  With the help of the intramol. polarization, these permanent at. multipoles can then be applied in modeling alternative conformations of a mol.  Equipped with this simple technique, one can derive transferable electrostatic parameters for peptides and proteins using flexible model compds. such as dipeptides.  The proposed procedure is tested for its ability to describe the electrostatic potential around various configurations of the N-methylacetamide dimer.  The effect of different intramol. polarization schemes on the accuracy of a force field model of the electrostatic potential of alanine dipeptide is investigated.  A group-based scheme for including direct intramol. polarization is shown to be most successful in accounting for the conformational dependence of electrostatic potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzG8zSIw6usrVg90H21EOLACvtfcHk0ljJCcw62EFcnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rtr4%253D&md5=4f43548e95124d23ba655451cd332232</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10127%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DP.%26aulast%3DPonder%26aufirst%3DJ.%2BW.%26atitle%3DConsistent%2520treatment%2520of%2520inter-%2520and%2520intramolecular%2520polarization%2520in%2520molecular%2520mechanics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1497%26epage%3D1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span>GaussView version 5.0; <span class="NLM_publisher-name">Gaussian Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT 06492, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GaussView+version+5.0%3B+Gaussian+Inc.%3A+Wallingford%2C+CT+06492%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGaussView%2520version%25205.0%26pub%3DGaussian%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Clemente, F. R.; Vreven, T.; Frisch, M. J.</span><span> </span><span class="NLM_article-title">Getting the most out of ONIOM: guidelines and pitfalls</span>. In  <span class="citation_source-book">Quantum Biochemistry</span>; <span class="NLM_contrib-group">Matta, C. F.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2F9783527629213.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-83&author=F.+R.+Clemente&author=T.+Vreven&author=M.+J.+Frischauthor=C.+F.+Matta&title=Quantum+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2F9783527629213.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527629213.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DClemente%26aufirst%3DF.%2BR.%26atitle%3DGetting%2520the%2520most%2520out%2520of%2520ONIOM%253A%2520guidelines%2520and%2520pitfalls%26btitle%3DQuantum%2520Biochemistry%26aulast%3DMatta%26aufirst%3DC.%2BF.%26pub%3DWiley-VCH%26date%3D2010%26spage%3D61%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lgZ60aCRBYg7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.,  III; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Crowley, M.; Walker, R. C.; Zhang, W.; Merz, K. M.; Wang, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Kolossváry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Yang, L.; Tan, C.; Mongan, J.; Hornak, V.; Cui, G.; Mathews, D. H.; Seetin, M. G.; Sagui, C.; Babin, V.; Kollman, P. A.</span>Amber 10; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Fransisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=M.+Crowley&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Wang&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=L.+Yang&author=C.+Tan&author=J.+Mongan&author=V.+Hornak&author=G.+Cui&author=D.+H.+Mathews&author=M.+G.+Seetin&author=C.+Sagui&author=V.+Babin&author=P.+A.+Kollman&title=Amber+10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber%252010%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general Amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+Amber+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lgZ60aCRBYg7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520Amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lj4j-9F37GMbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Ryckaert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the Cartesian equations of motion of a system with constraints: molecular dynamics of <i>n</i>-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+Cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lj4j-9F37GMbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520Cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Hubbard, S. J.; Thornton, J. M.</span>NACCESS; <span class="NLM_publisher-name">Department of Biochemistry and Molecular Biology, University College London</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">1993</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=S.+J.+Hubbard&author=J.+M.+Thornton&title=NACCESS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DS.%2BJ.%26jtitle%3DNACCESS%26pub%3DDepartment%2520of%2520Biochemistry%2520and%2520Molecular%2520Biology%252C%2520University%2520College%2520London%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Lee, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, F. M.</span><span> </span><span class="NLM_article-title">The interpretation of protein structures: Estimation of static accessibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0022-2836%2871%2990324-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=5551392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE3MXht1Shtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1971&pages=379-400&author=B.+Leeauthor=F.+M.+Richards&title=The+interpretation+of+protein+structures%3A+Estimation+of+static+accessibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretation of protein structures:  estimation of static accessibility</span></div><div class="casAuthors">Lee, Byungkook; Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-400</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A program is described for drawing the van der Waals surface of a protein mol.  An extension of the program permits the accessibility of atoms, or groups of atoms, to solvent or solute mols. of specified size to be quant. assessed.  Accessibility is proportional to surface area.  The accessibility of all atoms in the 20 common amino acids in model tripeptides of the type Ala-X-Ala are given for defined conformation.  The accessibilities are also given for all atoms in RNase S, lysozyme, and myogoblin.  Internal cavities are defined and discussed.  Of the surface area of each protein 40-50% is occupied by nonpolar atoms.  The actual numerical results are sensitive to the values chosen for the van der Waals radii of the various groups.  Since there is uncertainty over the correct values for these radii, the derived nos. should only be used as qual. guide at this stage.  The av. change in accessibility for the atoms of all 3 proteins in going from a hypothetical extended chain to the folded conformation of the native protein is about a factor of 3.  This no. applies to both polar (N and O) and nonpolar (C and S) atoms considered sep.  The larger nonpolar amino acids tend to be more buried in the native form of all 3 proteins.  However, for all classes and for residues within a given class, the accessibility changes on folding tend to be highly variable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobxHItjODI-LVg90H21EOLACvtfcHk0lj4j-9F37GMbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXht1Shtrk%253D&md5=05fe695f57006e8a7f782e6c5a8db122</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2871%2990324-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252871%252990324-X%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DB.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DThe%2520interpretation%2520of%2520protein%2520structures%253A%2520Estimation%2520of%2520static%2520accessibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1971%26volume%3D55%26spage%3D379%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 23 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Acassio Rocha-Santos, Elton José Ferreira Chaves, Igor Barden Grillo, Amanara Souza de Freitas, Demétrius Antônio Machado Araújo, <span class="NLM_string-name hlFld-ContribAuthor">Gerd Bruno Rocha</span>. </span><span class="cited-content_cbyCitation_article-title">Thermochemical and Quantum Descriptor Calculations for Gaining Insight into Ricin Toxin A (RTA) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (13)
                                     , 8764-8777. <a href="https://doi.org/10.1021/acsomega.0c02588" title="DOI URL">https://doi.org/10.1021/acsomega.0c02588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c02588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c02588%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DThermochemical%252Band%252BQuantum%252BDescriptor%252BCalculations%252Bfor%252BGaining%252BInsight%252Binto%252BRicin%252BToxin%252BA%252B%252528RTA%252529%252BInhibitors%26aulast%3DRocha-Santos%26aufirst%3DAcassio%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D01062020%26date%3D30122020%26date%3D23032021%26volume%3D6%26issue%3D13%26spage%3D8764%26epage%3D8777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ulf  Ryde</span> and <span class="hlFld-ContribAuthor ">Pär  Söderhjelm</span>  . </span><span class="cited-content_cbyCitation_article-title">Ligand-Binding Affinity Estimates Supported by Quantum-Mechanical Methods. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2016,</strong> <em>116 </em>
                                    (9)
                                     , 5520-5566. <a href="https://doi.org/10.1021/acs.chemrev.5b00630" title="DOI URL">https://doi.org/10.1021/acs.chemrev.5b00630</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.5b00630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.5b00630%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DLigand-Binding%252BAffinity%252BEstimates%252BSupported%252Bby%252BQuantum-Mechanical%252BMethods%26aulast%3DRyde%26aufirst%3DUlf%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D23102015%26date%3D14042016%26date%3D11052016%26volume%3D116%26issue%3D9%26spage%3D5520%26epage%3D5566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mario  Fiore</span>, <span class="hlFld-ContribAuthor ">Stefano  Forli</span>, and <span class="hlFld-ContribAuthor ">Fabrizio  Manetti</span>  . </span><span class="cited-content_cbyCitation_article-title">Targeting Mitogen-Activated Protein Kinase-Activated Protein Kinase 2 (MAPKAPK2, MK2): Medicinal Chemistry Efforts To Lead Small Molecule Inhibitors to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (8)
                                     , 3609-3634. <a href="https://doi.org/10.1021/acs.jmedchem.5b01457" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01457%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BMitogen-Activated%252BProtein%252BKinase-Activated%252BProtein%252BKinase%252B2%252B%252528MAPKAPK2%25252C%252BMK2%252529%25253A%252BMedicinal%252BChemistry%252BEfforts%252BTo%252BLead%252BSmall%252BMolecule%252BInhibitors%252Bto%252BClinical%252BTrials%26aulast%3DFiore%26aufirst%3DMario%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D18092015%26date%3D09112015%26date%3D28042016%26date%3D26102015%26volume%3D59%26issue%3D8%26spage%3D3609%26epage%3D3634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prasad  Chaskar</span>, <span class="hlFld-ContribAuthor ">Vincent  Zoete</span>, and <span class="hlFld-ContribAuthor ">Ute F.  Röhrig</span>  . </span><span class="cited-content_cbyCitation_article-title">Toward On-The-Fly Quantum Mechanical/Molecular Mechanical (QM/MM) Docking: Development and Benchmark of a Scoring Function. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (11)
                                     , 3137-3152. <a href="https://doi.org/10.1021/ci5004152" title="DOI URL">https://doi.org/10.1021/ci5004152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci5004152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci5004152%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DToward%252BOn-The-Fly%252BQuantum%252BMechanical%25252FMolecular%252BMechanical%252B%252528QM%25252FMM%252529%252BDocking%25253A%252BDevelopment%252Band%252BBenchmark%252Bof%252Ba%252BScoring%252BFunction%26aulast%3DChaskar%26aufirst%3DPrasad%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D14072014%26date%3D22102014%26date%3D24112014%26date%3D09102014%26volume%3D54%26issue%3D11%26spage%3D3137%26epage%3D3152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">R. Thomas  Ullmann</span>, <span class="hlFld-ContribAuthor ">Susana L. A.  Andrade</span>, and <span class="hlFld-ContribAuthor ">G. Matthias  Ullmann</span>  . </span><span class="cited-content_cbyCitation_article-title">Thermodynamics of Transport Through the Ammonium Transporter Amt-1 Investigated with Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2012,</strong> <em>116 </em>
                                    (32)
                                     , 9690-9703. <a href="https://doi.org/10.1021/jp305440f" title="DOI URL">https://doi.org/10.1021/jp305440f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jp305440f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjp305440f%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DThermodynamics%252Bof%252BTransport%252BThrough%252Bthe%252BAmmonium%252BTransporter%252BAmt-1%252BInvestigated%252Bwith%252BFree%252BEnergy%252BCalculations%26aulast%3DUllmann%26aufirst%3DR.%2BThomas%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D03062012%26date%3D13072012%26date%3D03082012%26date%3D16082012%26date%3D17072012%26volume%3D116%26issue%3D32%26spage%3D9690%26epage%3D9703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">The latest automated docking technologies for novel drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (6)
                                     , 625-645. <a href="https://doi.org/10.1080/17460441.2021.1858793" title="DOI URL">https://doi.org/10.1080/17460441.2021.1858793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1858793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1858793%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Blatest%252Bautomated%252Bdocking%252Btechnologies%252Bfor%252Bnovel%252Bdrug%252Bdiscovery%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D6%26spage%3D625%26epage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Quan  Liu</span>, <span class="hlFld-ContribAuthor ">Liangxuan  Wang</span>, <span class="hlFld-ContribAuthor ">Juan Carlos  Roldao</span>, <span class="hlFld-ContribAuthor ">Pierre-Michel  Adam</span>, <span class="hlFld-ContribAuthor ">Marc  Brecht</span>, <span class="hlFld-ContribAuthor ">Johannes  Gierschner</span>, <span class="hlFld-ContribAuthor ">Frank  Wackenhut</span>, <span class="hlFld-ContribAuthor ">Alfred J.  Meixner</span>. </span><span class="cited-content_cbyCitation_article-title">Theoretical and Experimental Evidence of Two‐Step Tautomerization in Hypericin. </span><span class="cited-content_cbyCitation_journal-name">Advanced Photonics Research</span><span> <strong>2021,</strong> <em>2 </em>
                                    (6)
                                     , 2000170. <a href="https://doi.org/10.1002/adpr.202000170" title="DOI URL">https://doi.org/10.1002/adpr.202000170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adpr.202000170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadpr.202000170%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Photonics%2520Research%26atitle%3DTheoretical%252Band%252BExperimental%252BEvidence%252Bof%252BTwo%2525E2%252580%252590Step%252BTautomerization%252Bin%252BHypericin%26aulast%3DLiu%26aufirst%3DQuan%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D6%26spage%3D2000170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zbigniew  Dutkiewicz</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods for calculation of protein-ligand binding affinities in structure-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Physical Sciences Reviews</span><span> <strong>2020</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1515/psr-2020-0034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1515%2Fpsr-2020-0034%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Sciences%2520Reviews%26atitle%3DComputational%252Bmethods%252Bfor%252Bcalculation%252Bof%252Bprotein-ligand%252Bbinding%252Baffinities%252Bin%252Bstructure-based%252Bdrug%252Bdesign%26aulast%3DDutkiewicz%26aufirst%3DZbigniew%26date%3D2020%26date%3D2020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Parichehr  Hassanzadeh</span>. </span><span class="cited-content_cbyCitation_article-title">Towards the quantum-enabled technologies for development of drugs or delivery systems. </span><span class="cited-content_cbyCitation_journal-name">Journal of Controlled Release</span><span> <strong>2020,</strong> <em>324 </em>, 260-279. <a href="https://doi.org/10.1016/j.jconrel.2020.04.050" title="DOI URL">https://doi.org/10.1016/j.jconrel.2020.04.050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jconrel.2020.04.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jconrel.2020.04.050%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Controlled%2520Release%26atitle%3DTowards%252Bthe%252Bquantum-enabled%252Btechnologies%252Bfor%252Bdevelopment%252Bof%252Bdrugs%252Bor%252Bdelivery%252Bsystems%26aulast%3DHassanzadeh%26aufirst%3DParichehr%26date%3D2020%26volume%3D324%26spage%3D260%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad Rizki Fadhil  Pratama</span>, <span class="hlFld-ContribAuthor ">Hadi  Poerwono</span>, <span class="hlFld-ContribAuthor ">Siswandono  Siswodihardjo</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking of novel 5-O-benzoylpinostrobin derivatives as wild type and L858R/T790M/V948R mutant EGFR inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Basic and Clinical Physiology and Pharmacology</span><span> <strong>2019,</strong> <em>30 </em>
                                    (6)
                                     <a href="https://doi.org/10.1515/jbcpp-2019-0301" title="DOI URL">https://doi.org/10.1515/jbcpp-2019-0301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1515/jbcpp-2019-0301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1515%2Fjbcpp-2019-0301%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Basic%2520and%2520Clinical%2520Physiology%2520and%2520Pharmacology%26atitle%3DMolecular%252Bdocking%252Bof%252Bnovel%252B5-O-benzoylpinostrobin%252Bderivatives%252Bas%252Bwild%252Btype%252Band%252BL858R%25252FT790M%25252FV948R%252Bmutant%252BEGFR%252Binhibitor%26aulast%3DPratama%26aufirst%3DMohammad%2BRizki%2BFadhil%26date%3D2019%26volume%3D30%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Zhou</span>, <span class="hlFld-ContribAuthor ">Yulin  Huang</span>, <span class="hlFld-ContribAuthor ">Guihua  Tang</span>, <span class="hlFld-ContribAuthor ">Xiaofang  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of dispiro[1-benzothiophene-2,3'-pyrrolidine-2',3”-indoline]-2”,3-diones in cycloaddition reaction. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2019,</strong> <em>55 </em>
                                    (11)
                                     , 1044-1049. <a href="https://doi.org/10.1007/s10593-019-02575-6" title="DOI URL">https://doi.org/10.1007/s10593-019-02575-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-019-02575-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-019-02575-6%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DSynthesis%252Bof%252Bdispiro%25255B1-benzothiophene-2%25252C3%252527-pyrrolidine-2%252527%25252C3%2525E2%252580%25259D-indoline%25255D-2%2525E2%252580%25259D%25252C3-diones%252Bin%252Bcycloaddition%252Breaction%26aulast%3DZhou%26aufirst%3DYing%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D11%26spage%3D1044%26epage%3D1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin  Li</span>, <span class="hlFld-ContribAuthor ">Ailing  Fu</span>, <span class="hlFld-ContribAuthor ">Le  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">An Overview of Scoring Functions Used for Protein–Ligand Interactions in Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">Interdisciplinary Sciences: Computational Life Sciences</span><span> <strong>2019,</strong> <em>11 </em>
                                    (2)
                                     , 320-328. <a href="https://doi.org/10.1007/s12539-019-00327-w" title="DOI URL">https://doi.org/10.1007/s12539-019-00327-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12539-019-00327-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12539-019-00327-w%26sid%3Dliteratum%253Aachs%26jtitle%3DInterdisciplinary%2520Sciences%253A%2520Computational%2520Life%2520Sciences%26atitle%3DAn%252BOverview%252Bof%252BScoring%252BFunctions%252BUsed%252Bfor%252BProtein%2525E2%252580%252593Ligand%252BInteractions%252Bin%252BMolecular%252BDocking%26aulast%3DLi%26aufirst%3DJin%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D2%26spage%3D320%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cosimo  Toma</span>, <span class="hlFld-ContribAuthor ">Domenico  Gadaleta</span>, <span class="hlFld-ContribAuthor ">Alessandra  Roncaglioni</span>, <span class="hlFld-ContribAuthor ">Andrey  Toropov</span>, <span class="hlFld-ContribAuthor ">Alla  Toropova</span>, <span class="hlFld-ContribAuthor ">Marco  Marzo</span>, <span class="hlFld-ContribAuthor ">Emilio  Benfenati</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR Development for Plasma Protein Binding: Influence of the Ionization State. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Research</span><span> <strong>2019,</strong> <em>36 </em>
                                    (2)
                                     <a href="https://doi.org/10.1007/s11095-018-2561-8" title="DOI URL">https://doi.org/10.1007/s11095-018-2561-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11095-018-2561-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11095-018-2561-8%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Research%26atitle%3DQSAR%252BDevelopment%252Bfor%252BPlasma%252BProtein%252BBinding%25253A%252BInfluence%252Bof%252Bthe%252BIonization%252BState%26aulast%3DToma%26aufirst%3DCosimo%26date%3D2019%26date%3D2018%26volume%3D36%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rakesh Kumar  Singh</span>, <span class="hlFld-ContribAuthor ">Abul Kalam  Najmi</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutic Potential of Mitogen-Activated Protein Kinase Activated Protein Kinase 2 (MK2) in Chronic Airway Inflammatory Disorders. </span><span class="cited-content_cbyCitation_journal-name">Current Drug Targets</span><span> <strong>2019,</strong> <em>20 </em>
                                    (4)
                                     , 367-379. <a href="https://doi.org/10.2174/1389450119666180816121323" title="DOI URL">https://doi.org/10.2174/1389450119666180816121323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1389450119666180816121323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1389450119666180816121323%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Drug%2520Targets%26atitle%3DNovel%252BTherapeutic%252BPotential%252Bof%252BMitogen-Activated%252BProtein%252BKinase%252BActivated%252BProtein%252BKinase%252B2%252B%252528MK2%252529%252Bin%252BChronic%252BAirway%252BInflammatory%252BDisorders%26aulast%3DSingh%26aufirst%3DRakesh%2BKumar%26date%3D2019%26volume%3D20%26issue%3D4%26spage%3D367%26epage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adebayo A.  Adeniyi</span>, <span class="hlFld-ContribAuthor ">Mahmoud E.S.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Implementing QM in docking calculations: is it a waste of computational time?. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2017,</strong> <em>22 </em>
                                    (8)
                                     , 1216-1223. <a href="https://doi.org/10.1016/j.drudis.2017.06.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2017.06.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2017.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2017.06.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DImplementing%252BQM%252Bin%252Bdocking%252Bcalculations%25253A%252Bis%252Bit%252Ba%252Bwaste%252Bof%252Bcomputational%252Btime%25253F%26aulast%3DAdeniyi%26aufirst%3DAdebayo%2BA.%26date%3D2017%26volume%3D22%26issue%3D8%26spage%3D1216%26epage%3D1223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rakesh Kumar  Singh</span>, <span class="hlFld-ContribAuthor ">Abul Kalam  Najmi</span>, <span class="hlFld-ContribAuthor ">Sunanda Ghosh  Dastidar</span>. </span><span class="cited-content_cbyCitation_article-title">Biological functions and role of mitogen-activated protein kinase activated protein kinase 2 (MK2) in inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reports</span><span> <strong>2017,</strong> <em>69 </em>
                                    (4)
                                     , 746-756. <a href="https://doi.org/10.1016/j.pharep.2017.03.023" title="DOI URL">https://doi.org/10.1016/j.pharep.2017.03.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharep.2017.03.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharep.2017.03.023%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reports%26atitle%3DBiological%252Bfunctions%252Band%252Brole%252Bof%252Bmitogen-activated%252Bprotein%252Bkinase%252Bactivated%252Bprotein%252Bkinase%252B2%252B%252528MK2%252529%252Bin%252Binflammatory%252Bdiseases%26aulast%3DSingh%26aufirst%3DRakesh%2BKumar%26date%3D2017%26volume%3D69%26issue%3D4%26spage%3D746%26epage%3D756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rabab  Boyaala</span>, <span class="hlFld-ContribAuthor ">Rachid  Touzani</span>, <span class="hlFld-ContribAuthor ">Véronique  Guerchais</span>, <span class="hlFld-ContribAuthor ">Jean-François  Soulé</span>, <span class="hlFld-ContribAuthor ">Henri  Doucet</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 2-(fluorinated aryl)pyridine derivatives via palladium-catalyzed C H bond arylation of fluorobenzenes using 2-halopyridines as aryl sources. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (33)
                                     , 3205-3208. <a href="https://doi.org/10.1016/j.tetlet.2017.06.075" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.06.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.06.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.06.075%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DSynthesis%252Bof%252B2-%252528fluorinated%252Baryl%252529pyridine%252Bderivatives%252Bvia%252Bpalladium-catalyzed%252BC%252BH%252Bbond%252Barylation%252Bof%252Bfluorobenzenes%252Busing%252B2-halopyridines%252Bas%252Baryl%252Bsources%26aulast%3DBoyaala%26aufirst%3DRabab%26date%3D2017%26volume%3D58%26issue%3D33%26spage%3D3205%26epage%3D3208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adela  Bobovská</span>, <span class="hlFld-ContribAuthor ">Igor  Tvaroška</span>, <span class="hlFld-ContribAuthor ">Juraj  Kóňa</span>. </span><span class="cited-content_cbyCitation_article-title">Using DFT methodology for more reliable predictive models: Design of inhibitors of Golgi α-Mannosidase II. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2016,</strong> <em>66 </em>, 47-57. <a href="https://doi.org/10.1016/j.jmgm.2016.03.004" title="DOI URL">https://doi.org/10.1016/j.jmgm.2016.03.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2016.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2016.03.004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DUsing%252BDFT%252Bmethodology%252Bfor%252Bmore%252Breliable%252Bpredictive%252Bmodels%25253A%252BDesign%252Bof%252Binhibitors%252Bof%252BGolgi%252B%2525CE%2525B1-Mannosidase%252BII%26aulast%3DBobovsk%25C3%25A1%26aufirst%3DAdela%26date%3D2016%26volume%3D66%26spage%3D47%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elizabeth  Yuriev</span>, <span class="hlFld-ContribAuthor ">Jessica  Holien</span>, <span class="hlFld-ContribAuthor ">Paul A.  Ramsland</span>. </span><span class="cited-content_cbyCitation_article-title">Improvements, trends, and new ideas in molecular docking: 2012-2013 in review. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Recognition</span><span> <strong>2015,</strong> <em>28 </em>
                                    (10)
                                     , 581-604. <a href="https://doi.org/10.1002/jmr.2471" title="DOI URL">https://doi.org/10.1002/jmr.2471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jmr.2471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjmr.2471%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Recognition%26atitle%3DImprovements%25252C%252Btrends%25252C%252Band%252Bnew%252Bideas%252Bin%252Bmolecular%252Bdocking%25253A%252B2012-2013%252Bin%252Breview%26aulast%3DYuriev%26aufirst%3DElizabeth%26date%3D2015%26date%3D2015%26volume%3D28%26issue%3D10%26spage%3D581%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Harjeet  Juneja</span>, <span class="hlFld-ContribAuthor ">Dhanashri  Panchbhai</span>, <span class="hlFld-ContribAuthor ">Javed  Sheikh</span>, <span class="hlFld-ContribAuthor ">Vishwas  Ingle</span>, <span class="hlFld-ContribAuthor ">Taibi Ben  Hadda</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, antibacterial screening, and POM analyses of novel bis-isoxazolyl/pyrazolyl-1,3-diols. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2014,</strong> <em>23 </em>
                                    (3)
                                     , 1537-1547. <a href="https://doi.org/10.1007/s00044-013-0755-5" title="DOI URL">https://doi.org/10.1007/s00044-013-0755-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-013-0755-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-013-0755-5%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DSynthesis%25252C%252Bantibacterial%252Bscreening%25252C%252Band%252BPOM%252Banalyses%252Bof%252Bnovel%252Bbis-isoxazolyl%25252Fpyrazolyl-1%25252C3-diols%26aulast%3DJuneja%26aufirst%3DHarjeet%26date%3D2014%26date%3D2013%26volume%3D23%26issue%3D3%26spage%3D1537%26epage%3D1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Taibi Ben  Hadda</span>, <span class="hlFld-ContribAuthor ">Mushtaq  Ahmad</span>, <span class="hlFld-ContribAuthor ">Shazia  Sultana</span>, <span class="hlFld-ContribAuthor ">Usama  Shaheen</span>, <span class="hlFld-ContribAuthor ">Ammar  Bader</span>, <span class="hlFld-ContribAuthor ">Sajal  Srivastava</span>, <span class="hlFld-ContribAuthor ">Barnali  Das</span>, <span class="hlFld-ContribAuthor ">Héctor  Salgado-Zamora</span>. </span><span class="cited-content_cbyCitation_article-title">POM analyses for antimicrobial evaluation of thienopyrimidinones derivatives: a rapid method for drug design. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2014,</strong> <em>23 </em>
                                    (1)
                                     , 16-24. <a href="https://doi.org/10.1007/s00044-013-0614-4" title="DOI URL">https://doi.org/10.1007/s00044-013-0614-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-013-0614-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-013-0614-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DPOM%252Banalyses%252Bfor%252Bantimicrobial%252Bevaluation%252Bof%252Bthienopyrimidinones%252Bderivatives%25253A%252Ba%252Brapid%252Bmethod%252Bfor%252Bdrug%252Bdesign%26aulast%3DHadda%26aufirst%3DTaibi%2BBen%26date%3D2014%26date%3D2013%26volume%3D23%26issue%3D1%26spage%3D16%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Mucs</span>, <span class="hlFld-ContribAuthor ">Richard A  Bryce</span>. </span><span class="cited-content_cbyCitation_article-title">The application of quantum mechanics in structure-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2013,</strong> <em>8 </em>
                                    (3)
                                     , 263-276. <a href="https://doi.org/10.1517/17460441.2013.752812" title="DOI URL">https://doi.org/10.1517/17460441.2013.752812</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2013.752812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2013.752812%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bapplication%252Bof%252Bquantum%252Bmechanics%252Bin%252Bstructure-based%252Bdrug%252Bdesign%26aulast%3DMucs%26aufirst%3DDaniel%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D3%26spage%3D263%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine H.  Schein</span>, <span class="hlFld-ContribAuthor ">Deliang  Chen</span>, <span class="hlFld-ContribAuthor ">Lili  Ma</span>, <span class="hlFld-ContribAuthor ">John J.  Kanalas</span>, <span class="hlFld-ContribAuthor ">Jian  Gao</span>, <span class="hlFld-ContribAuthor ">Maria Estrella  Jimenez</span>, <span class="hlFld-ContribAuthor ">Laurie E.  Sower</span>, <span class="hlFld-ContribAuthor ">Mary A.  Walter</span>, <span class="hlFld-ContribAuthor ">Scott R.  Gilbertson</span>, <span class="hlFld-ContribAuthor ">Johnny W.  Peterson</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore Selection and Redesign of Non-nucleotide Inhibitors of Anthrax Edema Factor. </span><span class="cited-content_cbyCitation_journal-name">Toxins</span><span> <strong>2012,</strong> <em>4 </em>
                                    (11)
                                     , 1288-1300. <a href="https://doi.org/10.3390/toxins4111288" title="DOI URL">https://doi.org/10.3390/toxins4111288</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/toxins4111288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Ftoxins4111288%26sid%3Dliteratum%253Aachs%26jtitle%3DToxins%26atitle%3DPharmacophore%252BSelection%252Band%252BRedesign%252Bof%252BNon-nucleotide%252BInhibitors%252Bof%252BAnthrax%252BEdema%252BFactor%26aulast%3DSchein%26aufirst%3DCatherine%2BH.%26date%3D2012%26date%3D2012%26volume%3D4%26issue%3D11%26spage%3D1288%26epage%3D1300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Tautomer and Ionization Equilibria of Studied Benzothiophene Analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Species, present in at least 0.01% fraction under experimental conditions, are labeled S1–S12 and their parent tautomers are labeled as T1–T5. The neutral forms of the T3, T4, and T5 neutral-zwitterion pairs, marked with a prime, are present in negligible fractions and are listed only as intermediates in the formation of more abundant zwitterionic counterparts. The protonated form of T2 is shown as T2+. The charges of substituent groups are given in brackets. The R3–R4 substituent group for S2/T1 and S6/T2 refers to the cyclic derivatives (<b>20</b>–<b>33</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Species S3 and S4 are not present in this series.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Tautomer and Ionization Equilibria of the Studied Pyrrolopyridine Analogues (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), Showing the Species Present in at Least 0.01% Fraction under Experimental Conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Tautomers are labeled as T1–T2 and species are labeled S1–S5. Protonated form of T1 is shown as T1+. The charges of substituent groups are given in brackets. Species S1, S2, and S5 are similar to those in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> in the approximate position of charged atoms with regard to the CO-NH in T1. Species S6–S12 are not present in this series.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Species fractions of the studied benzothiophenes (<b>1</b>–<b>35</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) and pyrrolopyridines (<b>36</b>–<b>66</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) in water under experimental conditions: species S1/T1, S2/T1, S3/T1, S4/T1, S5/T2, and S6/T2 (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) are shown in black, red, blue, yellow, gray, and green, respectively. Only major species (>10% in at least one compound) are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Time-averaged structure of MK2 with bound compound <b>8</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) used in step 4. The protein is shown as C<sub>α</sub>-trace with the binding site indicated as transparent surface of the binding site residues Leu70, Gly71, Leu72, Gly73, Ile74, Ala91, Leu92, Lys93, Met138, Glu139, Cys140, Leu141, Asp142, Asn191, Leu192, Leu193, Thr206, and Asp207. The key residues and the ligand, shown in atom-type-colored ball and stick representation and the other binding site residues represented as red sticks are included in QM and optimized MM regions of the QM/MM geometry optimization, respectively (step 2). The ligand and all binding site residues were included in the flexible region in FlexiDock docking (step 1) and the QM-region in the single-point QM/MM energy calculation (step 4).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Correlations between experimental and calculated MK2 inhibition potencies for studied compounds are shown for all models: single-tautomer, single-species (green squares), single-tautomer, multispecies (red triangles), multitautomer, single-species (blue diamonds), and multitautomer, multispecies (black circles).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0010.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Interaction map of compound <b>8</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) as species 5/tautomer 2 (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) in the binding site of MK2. Nitrogen, oxygen, and sulfur atoms are printed in blue, red, and yellow colors, respectively. H-Bond interactions are drawn in red color, and the bond lengths and bond angles (in italics) for the time-averaged structure are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/medium/jm-2011-01217q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0011.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. A comparison of predicted bound species prevalences and species fractions in the aqueous solution for studied ligands (the numbers correspond to the compound numbers in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Species 1–6 (Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) are referred to by black, cyan, red, magenta, blue, and green colors, respectively. Only major species (at least one compound with >10%) are shown here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-5/jm201217q/production/images/large/jm-2011-01217q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201217q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 72 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lg2xJtsM0j6zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lee, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayyampalayam, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreira, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattachar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coselmon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span> </span><span class="NLM_article-title">In silico prediction of ionization constants of drugs</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">498</span><span class="NLM_x">–</span> <span class="NLM_lpage">512</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp070019%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1CmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=498-512&author=P.+H.+Leeauthor=S.+N.+Ayyampalayamauthor=L.+A.+Carreiraauthor=M.+Shalaevaauthor=S.+Bhattacharauthor=R.+Coselmonauthor=S.+Pooleauthor=E.+Giffordauthor=F.+Lombardo&title=In+silico+prediction+of+ionization+constants+of+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Prediction of Ionization Constants of Drugs</span></div><div class="casAuthors">Lee, Pil H.; Ayyampalayam, Saravanaraj N.; Carreira, Lionel A.; Shalaeva, Marina; Bhattachar, Shobha; Coselmon, Rachel; Poole, Salwa; Gifford, Eric; Lombardo, Franco</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">498-512</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most pharmacol. active mols. contain one or more ionizing groups, and it is well-known that knowledge of the ionization state of a drug, indicated by the pKa value, is crit. for understanding many properties important to the drug discovery and development process.  The ionization state of a compd. directly influences such important pharmaceutical characteristics as aq. soly., permeability, crystal structure, etc.  Tremendous advances have been made in the field of exptl. detn. of pKa, in terms of both quantity/speed and quality/accuracy.  However, there still remains a need for accurate in silico predictions of pKa both to est. this parameter for virtual compds. and to focus screening efforts of real compds.  The computer program SPARC (SPARC Performs Automated Reasoning in Chem.) was used to predict the ionization state of a drug.  This program has been developed based on the solid phys. chem. of reactivity models and applied to successfully predict numerous phys. properties as well as chem. reactivity parameters.  SPARC predicts both macroscopic and microscopic pKa values strictly from mol. structure.  In this paper, we describe the details of the SPARC reactivity computational methods and its performance on predicting the pKa values of known drugs as well as Pfizer internal discovery/development compds.  A high correlation (r2 = 0.92) between exptl. and the SPARC calcd. pKa values was obtained with root-mean-square error (RMSE) of 0.78 log unit for a set of 123 compds. including many known drugs.  For a set of 537 compds. from the Pfizer internal dataset, correlation coeff. r2 = 0.80 and RMSE = 1.05 were obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3S-24g-rak7Vg90H21EOLACvtfcHk0lg2xJtsM0j6zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1CmsLw%253D&md5=8363ba511dbb3a98ed8d0f8f4389d88e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fmp070019%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp070019%252B%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BH.%26aulast%3DAyyampalayam%26aufirst%3DS.%2BN.%26aulast%3DCarreira%26aufirst%3DL.%2BA.%26aulast%3DShalaeva%26aufirst%3DM.%26aulast%3DBhattachar%26aufirst%3DS.%26aulast%3DCoselmon%26aufirst%3DR.%26aulast%3DPoole%26aufirst%3DS.%26aulast%3DGifford%26aufirst%3DE.%26aulast%3DLombardo%26aufirst%3DF.%26atitle%3DIn%2520silico%2520prediction%2520of%2520ionization%2520constants%2520of%2520drugs%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2007%26volume%3D4%26spage%3D498%26epage%3D512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzur, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassanali, M.</span><span> </span><span class="NLM_article-title">DrugBank: A knowledgebase for drugs, drug actions and drug targets</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D901</span><span class="NLM_x">–</span> <span class="NLM_lpage">D906</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fnar%2Fgkm958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=18048412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D901-D906&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=D.+Chengauthor=S.+Shrivastavaauthor=D.+Tzurauthor=B.+Gautamauthor=M.+Hassanali&title=DrugBank%3A+A+knowledgebase+for+drugs%2C+drug+actions+and+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Cheng, Dean; Shrivastava, Savita; Tzur, Dan; Gautam, Bijaya; Hassanali, Murtaza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D901-D906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a richly annotated resource that combines detailed drug data with comprehensive drug target and drug action information.  Since its first release in 2006, DrugBank has been widely used to facilitate in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  The latest version of DrugBank (release 2.0) has been expanded significantly over the previous release.  With ∼4900 drug entries, it now contains 60% more FDA-approved small mol. and biotech drugs including 10% more exptl.' drugs.  Significantly, more protein target data has also been added to the database, with the latest version of DrugBank contg. three times as many non-redundant protein or drug target sequences as before (1565 vs. 524).  Each DrugCard entry now contains more than 100 data fields with half of the information being devoted to drug/chem. data and the other half devoted to pharmacol., pharmacogenomic and mol. biol. data.  A no. of new data fields, including food-drug interactions, drug-drug interactions and exptl. ADME data have been added in response to numerous user requests.  DrugBank has also significantly improved the power and simplicity of its structure query and text query searches.  DrugBank is available at http://www.drugbank.ca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqKjSWZe1ULVg90H21EOLACvtfcHk0ljmaTKcPPPq6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D&md5=f8f8ca15f248b8c8b9c6d3bd0e85eec8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm958%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DTzur%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DB.%26aulast%3DHassanali%26aufirst%3DM.%26atitle%3DDrugBank%253A%2520A%2520knowledgebase%2520for%2520drugs%252C%2520drug%2520actions%2520and%2520drug%2520targets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD901%26epage%3DD906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Shimba, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serber, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledwidge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doetsch, V.</span><span> </span><span class="NLM_article-title">Quantitative identification of the protonation state of histidines in vitro and in vivo</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">9227</span><span class="NLM_x">–</span> <span class="NLM_lpage">9234</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi0344679" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=9227-9234&author=N.+Shimbaauthor=Z.+Serberauthor=R.+Ledwidgeauthor=S.+M.+Millerauthor=C.+S.+Craikauthor=V.+Doetsch&title=Quantitative+identification+of+the+protonation+state+of+histidines+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbi0344679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0344679%26sid%3Dliteratum%253Aachs%26aulast%3DShimba%26aufirst%3DN.%26aulast%3DSerber%26aufirst%3DZ.%26aulast%3DLedwidge%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DS.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DDoetsch%26aufirst%3DV.%26atitle%3DQuantitative%2520identification%2520of%2520the%2520protonation%2520state%2520of%2520histidines%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D9227%26epage%3D9234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Vila, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnautova, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobjev, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H. A.</span><span> </span><span class="NLM_article-title">Assessing the fractions of tautomeric forms of the imidazole ring of histidine in proteins as a function of pH</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">5602</span><span class="NLM_x">–</span> <span class="NLM_lpage">5607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=5602-5607&author=J.+A.+Vilaauthor=Y.+A.+Arnautovaauthor=Y.+Vorobjevauthor=H.+A.+Scheraga&title=Assessing+the+fractions+of+tautomeric+forms+of+the+imidazole+ring+of+histidine+in+proteins+as+a+function+of+pH"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVila%26aufirst%3DJ.%2BA.%26aulast%3DArnautova%26aufirst%3DY.%2BA.%26aulast%3DVorobjev%26aufirst%3DY.%26aulast%3DScheraga%26aufirst%3DH.%2BA.%26atitle%3DAssessing%2520the%2520fractions%2520of%2520tautomeric%2520forms%2520of%2520the%2520imidazole%2520ring%2520of%2520histidine%2520in%2520proteins%2520as%2520a%2520function%2520of%2520pH%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D5602%26epage%3D5607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Clarke, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rochester, C. H.</span><span> </span><span class="NLM_article-title">Spectroscopic study of the acid ionization of porphyrins</span> <span class="citation_source-journal">J. Chem. Soc., Perkin. Trans. 2</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1039%2Fp29730000414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE3sXht1ersL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1973&pages=414-416&author=J.+A.+Clarkeauthor=P.+J.+Dawsonauthor=R.+Griggauthor=C.+H.+Rochester&title=Spectroscopic+study+of+the+acid+ionization+of+porphyrins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Spectroscopic study of the acid ionization of porphyrins</span></div><div class="casAuthors">Clarke, J. A.; Dawson, P. J.; Grigg, R.; Rochester, C. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">414-16</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">Spectral studies showed that N-methyloctaethylporphyrin and the  corresponding N-(3-bromopropyl)porphyrin ionize in solns. contg. aq. NaOH to give their monoanions.  Octaethylporphyrin, etioporphyrin I, and monozaethioporphyrin I ionize to their dianions by loss of 2 protons; their monoanions were not detected in solns. contg. NaOH.  N-Me substitution in a porphyrin increases the acidity of the remaining NH group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxd38-n0HVW7Vg90H21EOLACvtfcHk0ljmaTKcPPPq6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXht1ersL8%253D&md5=a946e4f6d826d855be5021176975fbc0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2Fp29730000414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29730000414%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BA.%26aulast%3DDawson%26aufirst%3DP.%2BJ.%26aulast%3DGrigg%26aufirst%3DR.%26aulast%3DRochester%26aufirst%3DC.%2BH.%26atitle%3DSpectroscopic%2520study%2520of%2520the%2520acid%2520ionization%2520of%2520porphyrins%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin.%2520Trans.%25202%26date%3D1973%26spage%3D414%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Braun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koecher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlabach, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehrle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limbach, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, E.</span><span> </span><span class="NLM_article-title">NMR study of the tautomerism of porphyrin including the kinetic HH/HD/DD isotope effects in the liquid and the solid state</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">6593</span><span class="NLM_x">–</span> <span class="NLM_lpage">6604</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja00094a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2cXltVygsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1994&pages=6593-6604&author=J.+Braunauthor=M.+Koecherauthor=M.+Schlabachauthor=B.+Wehrleauthor=H.+H.+Limbachauthor=E.+Vogel&title=NMR+study+of+the+tautomerism+of+porphyrin+including+the+kinetic+HH%2FHD%2FDD+isotope+effects+in+the+liquid+and+the+solid+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">NMR Study of the Tautomerism of Porphyrin Including the Kinetic HH/HD/DD Isotope Effects in the Liquid and the Solid State</span></div><div class="casAuthors">Braun, Juergen; Koecher, Martin; Schlabach, Martin; Wehrle, Bernd; Limbach, Han-Heinrich; Vogel, Emanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6593-604</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Using dynamic liq.-state 1H NMR and solid-state 15N CPMAS NMR spectroscopy (CP = cross polarization, MAS = magic angle spinning), the intramol. double-proton transfer (tautomerism) of 15N-labeled porphyrin has been studied.  Rate consts. including kinetic HH/HD/DD isotope effects were obtained as a function of temp. not only for the liq. phase but also for the polycryst. solid.  Within the margin of error, the liq.-state degeneracy of the tautomerism is maintained in the solid state; moreover, rate consts. and kinetic isotope effects are the same for both environments.  Therefore, it is justified to combine rate consts. of the porphyrin tautomerism detd. for various environments into a single Arrhenius diagram.  The Arrhenius curves of the different isotopic reactions indicate a stepwise tautomerism via a metastable intermediate as postulated previously, involving thermally activated tunneling at low temps.  Furthermore, the obsd. kinetic isotope effects are not in agreement with a solid-state process consisting of combined nondegenerate proton transfers and mol. 90° rotations.  This possibility has been discussed in order to reconcile the observations of proton disorder in solid porphyrin by NMR and proton order by X-ray crystallog.  Finally, a novel process has been obsd. which consists of a complete deuteration or dedeuteration of the mobile proton sites of solid porphyrin by contact with gaseous or liq. D2O or H2O.  The process involves (i) penetration of gaseous water into the porphyrin crystals, (ii) diffusion processes of water or of porphyrin inside the crystals, and (iii) exchange of protons or deuterons in mol. hydrogen-bonded water-porphyrin complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwIM86pqDWrVg90H21EOLACvtfcHk0lhv7OReYt4fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltVygsb8%253D&md5=9fcef5707c746ae9a0cf4879ddfa6c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja00094a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00094a014%26sid%3Dliteratum%253Aachs%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DKoecher%26aufirst%3DM.%26aulast%3DSchlabach%26aufirst%3DM.%26aulast%3DWehrle%26aufirst%3DB.%26aulast%3DLimbach%26aufirst%3DH.%2BH.%26aulast%3DVogel%26aufirst%3DE.%26atitle%3DNMR%2520study%2520of%2520the%2520tautomerism%2520of%2520porphyrin%2520including%2520the%2520kinetic%2520HH%252FHD%252FDD%2520isotope%2520effects%2520in%2520the%2520liquid%2520and%2520the%2520solid%2520state%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D6593%26epage%3D6604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Sponer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leszczynski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobza, P.</span><span> </span><span class="NLM_article-title">Electronic properties, hydrogen bonding, stacking, and cation binding of DNA and RNA bases</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=3-31&author=J.+Sponerauthor=J.+Leszczynskiauthor=P.+Hobza&title=Electronic+properties%2C+hydrogen+bonding%2C+stacking%2C+and+cation+binding+of+DNA+and+RNA+bases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DLeszczynski%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DElectronic%2520properties%252C%2520hydrogen%2520bonding%252C%2520stacking%252C%2520and%2520cation%2520binding%2520of%2520DNA%2520and%2520RNA%2520bases%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D61%26spage%3D3%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Sigel, H.</span><span> </span><span class="NLM_article-title">Acid–base properties of purine residues and the effect of metal ions: Quantification of rare nucleobase tautomers</span> <span class="citation_source-journal">Pure Appl. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">1869</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2004&pages=1869-1886&author=H.+Sigel&title=Acid%E2%80%93base+properties+of+purine+residues+and+the+effect+of+metal+ions%3A+Quantification+of+rare+nucleobase+tautomers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSigel%26aufirst%3DH.%26atitle%3DAcid%25E2%2580%2593base%2520properties%2520of%2520purine%2520residues%2520and%2520the%2520effect%2520of%2520metal%2520ions%253A%2520Quantification%2520of%2520rare%2520nucleobase%2520tautomers%26jtitle%3DPure%2520Appl.%2520Chem.%26date%3D2004%26volume%3D76%26spage%3D1869%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Lippert, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, D.</span><span> </span><span class="NLM_article-title">Promotion of rare nucleobase tautomers by metal binding</span> <span class="citation_source-journal">Dalton Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_fpage">4619</span><span class="NLM_x">–</span> <span class="NLM_lpage">4634</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=4619-4634&author=B.+Lippertauthor=D.+Gupta&title=Promotion+of+rare+nucleobase+tautomers+by+metal+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLippert%26aufirst%3DB.%26aulast%3DGupta%26aufirst%3DD.%26atitle%3DPromotion%2520of%2520rare%2520nucleobase%2520tautomers%2520by%2520metal%2520binding%26jtitle%3DDalton%2520Trans.%26date%3D2009%26spage%3D4619%26epage%3D4634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Katritzky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Gendy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draghici, B.</span><span> </span><span class="NLM_article-title">Tautomerism in drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">475</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2Fs10822-010-9359-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=20490619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=475-484&author=A.+Katritzkyauthor=C.+Hallauthor=B.+El+Gendyauthor=B.+Draghici&title=Tautomerism+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tautomerism in drug discovery</span></div><div class="casAuthors">Katritzky, Alan R.; Hall, C. Dennis; El-Gendy, Bahaa El-Dien M.; Draghici, Bogdan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">475-484</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The influence of tautomerism on the precise structure of drugs and thus of their potential to interact with biol. systems is discussed from thermodn. and kinetic aspects.  The types of tautomerism encountered in the structure of drugs in current use are surveyed together with the effect of pH, solvent polarity, and temp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8wFMBpeaetbVg90H21EOLACvtfcHk0lhv7OReYt4fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbk%253D&md5=06e63cd44d4c7a9f974c8257f07fc275</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9359-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9359-z%26sid%3Dliteratum%253Aachs%26aulast%3DKatritzky%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DC.%26aulast%3DEl%2BGendy%26aufirst%3DB.%26aulast%3DDraghici%26aufirst%3DB.%26atitle%3DTautomerism%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D475%26epage%3D484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Chou, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y. M.</span><span> </span><span class="NLM_article-title">Excited-state intramolecular proton transfer in 10-hydroxybenzo[<i>h</i>]quinoline</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1731</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp002942w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFynsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=1731-1740&author=P.+T.+Chouauthor=Y.+C.+Chenauthor=W.+S.+Yuauthor=Y.+H.+Chouauthor=C.+Y.+Weiauthor=Y.+M.+Cheng&title=Excited-state+intramolecular+proton+transfer+in+10-hydroxybenzo%5Bh%5Dquinoline"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Excited-State Intramolecular Proton Transfer in 10-Hydroxybenzo[h]quinoline</span></div><div class="casAuthors">Chou, Pi-Tai; Chen, Youn-Chan; Yu, Wei-Shan; Chou, Yi-Hsuan; Wei, Ching-Yen; Cheng, Yi-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1731-1740</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamics of excited-state intramol. enol-keto proton-transfer tautomerism in 10-hydroxybenzoquinoline (HBQ) and its deuterated analog (DBQ) have been investigated by steady-state absorption and fluorescence spectroscopy, femtosecond fluorescence upconversion in combination with pump-probe transient absorption expts. in nonpolar solvents.  In cyclohexane, the time scale for both proton and deuterium transfer in the excited state cannot be resolved under the response limit of ca. 160 and 200 fs, resp., of our current upconversion and transient absorption systems.  The initially prepd. keto tautomer is in a higher lying excited state, possibly the S'2 state (prime indicates the keto-tautomer form) which then undergoes a ∼330 fs S'2 → S'1 internal conversion, resulting in a highly vibrationally excited S'1 state. Subsequently, a solvent-induced vibrational relaxation takes place in a time scale of 8-10 ps followed by a relatively much longer, thermally cooled S'1 → S'0 decay rate of 3.3 × 109 s-1 (τf ∼ 300 ps-1).  The results in combination with extremely weak enol fluorescence resolved from the steady-state measurement lead us to conclude that excited-state intramol. proton transfer (ESIPT) is essentially barrier-less.  The rate of ESIPT upon 385-405 nm excitation may be detd. within the period of low-frequency, large-amplitude vibrations incorporating the motion of atoms assocd. with the hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFT4XR0IpgLVg90H21EOLACvtfcHk0lhv7OReYt4fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFynsLs%253D&md5=bb76bfc34af6abef6a3f046e9a47ab56</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjp002942w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp002942w%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DP.%2BT.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DYu%26aufirst%3DW.%2BS.%26aulast%3DChou%26aufirst%3DY.%2BH.%26aulast%3DWei%26aufirst%3DC.%2BY.%26aulast%3DCheng%26aufirst%3DY.%2BM.%26atitle%3DExcited-state%2520intramolecular%2520proton%2520transfer%2520in%252010-hydroxybenzo%255Bh%255Dquinoline%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2001%26volume%3D105%26spage%3D1731%26epage%3D1740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Raha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wollacott, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westerhoff, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, J.</span><span> </span><span class="NLM_article-title">The role of quantum mechanics in structure-based drug design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2Fj.drudis.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=17826685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=725-731&author=K.+Rahaauthor=M.+B.+Petersauthor=B.+Wangauthor=N.+Yuauthor=A.+M.+Wollacottauthor=L.+M.+Westerhoffauthor=J.+Merz&title=The+role+of+quantum+mechanics+in+structure-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The role of quantum mechanics in structure-based drug design</span></div><div class="casAuthors">Raha, Kaushik; Peters, Martin B.; Wang, Bing; Yu, Ning; Wollacott, Andrew M.; Westerhoff, Lance M.; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">17&18</span>),
    <span class="NLM_cas:pages">725-731</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Herein we will focus on the use of quantum mechanics (QM) in drug design (DD) to solve disparate problems from scoring protein-ligand poses to building QM QSAR models.  Through the variational principle of QM we know that we can obtain a more accurate representation of mol. systems than classical models, and while this is not a matter of debate, it still has not been shown that the expense of QM approaches is offset by improved accuracy in DD applications.  Objectively validating the improved applicability and performance of QM over classical-based models in DD will be the focus of research in the coming years along with research on the conformational sampling problem as it relates to protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ISn2DWdN9LVg90H21EOLACvtfcHk0lhCFe7WBM09xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK&md5=67fecc13843c7961ff820026c85a9290</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DK.%26aulast%3DPeters%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DN.%26aulast%3DWollacott%26aufirst%3DA.%2BM.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26aulast%3DMerz%26aufirst%3DJ.%26atitle%3DThe%2520role%2520of%2520quantum%2520mechanics%2520in%2520structure-based%2520drug%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D725%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Senn, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, W.</span><span> </span><span class="NLM_article-title">QM/MM methods for biomolecular systems</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1198</span><span class="NLM_x">–</span> <span class="NLM_lpage">1229</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fanie.200802019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=19173328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitFOqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=1198-1229&author=H.+M.+Sennauthor=W.+Thiel&title=QM%2FMM+methods+for+biomolecular+systems"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">QM/MM methods for biomolecular systems</span></div><div class="casAuthors">Senn, Hans Martin; Thiel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1198-1229</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Combined quantum-mechanics/mol.-mechanics (QM/MM) approaches have become the method of choice for modeling reactions in biomol. systems.  Quantum-mech. (QM) methods are required for describing chem. reactions and other electronic processes, such as charge transfer or electronic excitation.  However, QM methods are restricted to systems of up to a few hundred atoms.  However, the size and conformational complexity of biopolymers calls for methods capable of treating up to several 100,000 atoms and allowing for simulations over time scales of tens of nanoseconds.  This is achieved by highly efficient, force-field-based mol. mechanics (MM) methods.  Thus to model large biomols. the logical approach is to combine the two techniques and, to use a QM method for the chem. active region (e.g., substrates and co-factors in an enzymic reaction) and an MM treatment for the surroundings (e.g., protein and solvent).  The resulting schemes are commonly referred to as combined or hybrid QM/MM methods.  They enable the modeling of reactive biomol. systems at a reasonable computational effort while providing the necessary accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDAQMRLVHxZrVg90H21EOLACvtfcHk0lhCFe7WBM09xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitFOqs7g%253D&md5=c51da58b0525651c71f9c393a79023be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.200802019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200802019%26sid%3Dliteratum%253Aachs%26aulast%3DSenn%26aufirst%3DH.%2BM.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DQM%252FMM%2520methods%2520for%2520biomolecular%2520systems%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D1198%26epage%3D1229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Major, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truhlar, D. G.</span><span> </span><span class="NLM_article-title">Mechanisms and free energies of enzymatic reactions</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">3188</span><span class="NLM_x">–</span> <span class="NLM_lpage">3209</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr050293k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt12nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3188-3209&author=J.+Gaoauthor=S.+Maauthor=D.+T.+Majorauthor=K.+Namauthor=J.+Puauthor=D.+G.+Truhlar&title=Mechanisms+and+free+energies+of+enzymatic+reactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and free energies of enzymatic reactions</span></div><div class="casAuthors">Gao, Jiali; Ma, Shuhua; Major, Dan T.; Nam, Kwangho; Pu, Jingzhi; Truhlar, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3188-3209</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here, the authors summarize the theory and computational techniques for studies of the catalytic mechanisms and free energies of selected enzymic reactions.  Computational approaches for enzymic reactions with special emphasis on 2 key elements that affect the computational accuracy, namely, the potential energy function and statistical mech. sampling of the enzyme system are highlighted.  The choice of the reaction coordinate for detg. the free energy of activation to characterize the enzymic mechanism is discussed.  By examg. a variety of enzymic reactions that are well-understood from expts. and computations, it was found that natural selection enables enzymes to achieve catalytic efficiency in many different ways.  The factors promoting catalysis include transition state stabilization through electrostatic interactions, desolvation and reactant state destabilization, protein conformational change induced by differential interactions between the substrate and enzyme in the reactant state and transition state, and general acid and general base catalysis.  Quantum mech. tunneling appears also to be crit. in some enzyme reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosG2fgv8E8A7Vg90H21EOLACvtfcHk0liJk8roAeEdbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt12nsro%253D&md5=8b5d54ea600fbc9622c4bdd7ec6730f5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcr050293k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050293k%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DMajor%26aufirst%3DD.%2BT.%26aulast%3DNam%26aufirst%3DK.%26aulast%3DPu%26aufirst%3DJ.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26atitle%3DMechanisms%2520and%2520free%2520energies%2520of%2520enzymatic%2520reactions%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3188%26epage%3D3209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Aqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, J. E.</span><span> </span><span class="NLM_article-title">A new method for predicting binding affinity in computer-aided drug design</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fprotein%2F7.3.385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8177887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=385-391&author=J.+Aqvistauthor=C.+Medinaauthor=J.+E.+Samuelsson&title=A+new+method+for+predicting+binding+affinity+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A new method for predicting binding affinity in computer-aided drug design</span></div><div class="casAuthors">Aqvist J; Medina C; Samuelsson J E</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-91</span>
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    </div><div class="casAbstract">A new semi-empirical method for calculating free energies of binding from molecular dynamics (MD) simulations is presented.  It is based on standard thermodynamic cycles and on a linear approximation of polar and non-polar free energy contributions from the corresponding MD averages.  The method is tested on a set of endothiapepsin inhibitors and found to give accurate results both for absolute as well as relative free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTEx3uc8suH_BOk4JA3soTqfW6udTcc2eYBaIZIV-SqdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D&md5=c45572201328309d4fc849844d1c3b71</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F7.3.385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F7.3.385%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DSamuelsson%26aufirst%3DJ.%2BE.%26atitle%3DA%2520new%2520method%2520for%2520predicting%2520binding%2520affinity%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DProtein%2520Eng.%26date%3D1994%26volume%3D7%26spage%3D385%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1144</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fprotein%2F8.11.1137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8819979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=1137-1144&author=T.+Hanssonauthor=J.+Aqvist&title=Estimation+of+binding+free+energies+for+HIV+proteinase+inhibitors+by+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span></div><div class="casAuthors">Hansson, Tomas; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1137-44</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Abs. binding free energies for inhibitors of HIV-1 proteinase were estd. from mol. dynamics simulations by a recently reported linear approxn. procedure.  The results were in fairly good agreement with exptl. binding data.  Two of the inhibitors were very similar and, for comparison, their relative free energies of binding were also calcd. by free energy perturbation methods, giving virtually the same result.  Effects of cutoff radii and charge states of the protein model were examd.  The effects of pH on binding of one of the inhibitors were predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcv6yv6JgB1rVg90H21EOLACvtfcHk0liJk8roAeEdbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D&md5=c77820746da2a692e7543f5f79789cca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.11.1137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.11.1137%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DEstimation%2520of%2520binding%2520free%2520energies%2520for%2520HIV%2520proteinase%2520inhibitors%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D1137%26epage%3D1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Calculation of absolute binding free energies for charged ligands and effects of long-range electrostatic interactions</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1587</span><span class="NLM_x">–</span> <span class="NLM_lpage">1597</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=1587-1597&author=J.+Aqvist&title=Calculation+of+absolute+binding+free+energies+for+charged+ligands+and+effects+of+long-range+electrostatic+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DCalculation%2520of%2520absolute%2520binding%2520free%2520energies%2520for%2520charged%2520ligands%2520and%2520effects%2520of%2520long-range%2520electrostatic%2520interactions%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D1587%26epage%3D1597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">An extended linear response method for determining free energies of hydration</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">10667</span><span class="NLM_x">–</span> <span class="NLM_lpage">10673</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100026a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=10667-10673&author=H.+A.+Carlsonauthor=W.+L.+Jorgensen&title=An+extended+linear+response+method+for+determining+free+energies+of+hydration"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">An Extended Linear Response Method for Determining Free Energies of Hydration</span></div><div class="casAuthors">Carlson, Heather A.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10667-73</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fourteen neutral org. mols. with diverse functionalities were used to det. the utility of equations based on linear response theory for the calcn. of free energies of hydration.  The equations contain terms proportional to the Coulombic and van der Waals components of the solute-solvent interaction energy and to an index for the cost of cavity formation such as solvent-accessible surface area.  The energy components can be obtained from Monte Carlo or mol. dynamics simulations of the solutes in water.  The methodol. was tested by using partial charges from fitting to the electrostatic potentials of ab initio 6-31G* wave functions and from OPLS potential functions.  Root-mean-square deviations between the results from the recommended linear response methods and exptl. free energies of hydration are ca. 0.8 kcal/mol.  Substituted benzenes, not included in the parameter development, were used to further evaluate the method performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcOGUicelNrVg90H21EOLACvtfcHk0lieOQz5qmm-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D&md5=b91a7f287b63063c81c51082f694a78b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fj100026a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100026a034%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DAn%2520extended%2520linear%2520response%2520method%2520for%2520determining%2520free%2520energies%2520of%2520hydration%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D10667%26epage%3D10673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Jones-Hertzog, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Binding affinities for sulfonamide inhibitors with human thrombin using Monte Carlo simulations with a linear response method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1549</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm960684e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2sXis1ahu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=1539-1549&author=D.+K.+Jones-Hertzogauthor=W.+L.+Jorgensen&title=Binding+affinities+for+sulfonamide+inhibitors+with+human+thrombin+using+Monte+Carlo+simulations+with+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Affinities for Sulfonamide Inhibitors with Human Thrombin Using Monte Carlo Simulations with a Linear Response Method</span></div><div class="casAuthors">Jones-Hertzog, Deborah K.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1539-1549</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of sulfonamide inhibitors to human thrombin is examd. to evaluate the viability of calcg. free energies of binding, ΔGb, utilizing Monte Carlo (MC) statistical mechanics with a linear response approach.  Coulombic and van der Waals energy components detd. from MC simulations of the bound and unbound inhibitors solvated in water plus a solvent-accessible surface area term, as an index for cavity formation, were correlated with the free energies of binding for the inhibitor MD-805 and six derivs.  The best correlations yield an av. error of 0.8 kcal/mol for the seven binding affinities, which cover an obsd. range of 6.0 kcal/mol.  The MC simulations also provided insights into the interactions occurring in the active site and the origins of variations in ΔGb.  Equatorial placement of the carboxylate group at C2 in the piperidine ring of the inhibitors causes electrostatic destabilization with the side chain of Glu-H192, while axial disposition of the C4-Me group reduces favorable hydrophobic interactions in the P-pocket of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP22Kg6fLIUrVg90H21EOLACvtfcHk0lieOQz5qmm-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1ahu70%253D&md5=ab9aa0e119329ab539c0dd5fcbd1a187</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm960684e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960684e%26sid%3Dliteratum%253Aachs%26aulast%3DJones-Hertzog%26aufirst%3DD.%2BK.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DBinding%2520affinities%2520for%2520sulfonamide%2520inhibitors%2520with%2520human%2520thrombin%2520using%2520Monte%2520Carlo%2520simulations%2520with%2520a%2520linear%2520response%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D1539%26epage%3D1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Estimation of the binding affinities of FKBP12 inhibitors using a linear response method</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">860</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=851-860&author=M.+L.+Lambauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Estimation+of+the+binding+affinities+of+FKBP12+inhibitors+using+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEstimation%2520of%2520the%2520binding%2520affinities%2520of%2520FKBP12%2520inhibitors%2520using%2520a%2520linear%2520response%2520method%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D851%26epage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">What determines the van der Waals coefficient beta in the LIE (linear interaction energy) method to estimate binding free energies using molecular dynamics simulations?</span> <span class="citation_source-journal">Proteins.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">402</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=395-402&author=W.+Wangauthor=J.+Wangauthor=P.+A.+Kollman&title=What+determines+the+van+der+Waals+coefficient+beta+in+the+LIE+%28linear+interaction+energy%29+method+to+estimate+binding+free+energies+using+molecular+dynamics+simulations%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DWhat%2520determines%2520the%2520van%2520der%2520Waals%2520coefficient%2520beta%2520in%2520the%2520LIE%2520%2528linear%2520interaction%2520energy%2529%2520method%2520to%2520estimate%2520binding%2520free%2520energies%2520using%2520molecular%2520dynamics%2520simulations%253F%26jtitle%3DProteins.%26date%3D1999%26volume%3D34%26spage%3D395%26epage%3D402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukacova, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">A combination of docking, QM/MM methods, and MD simulation for binding affinity estimation of metalloprotein ligands</span> <span class="citation_source-journal">J.Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5437</span><span class="NLM_x">–</span> <span class="NLM_lpage">5447</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049050v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5437-5447&author=A.+Khandelwalauthor=V.+Lukacovaauthor=D.+Comezauthor=D.+M.+Krollauthor=S.+Rahaauthor=S.+Balaz&title=A+combination+of+docking%2C+QM%2FMM+methods%2C+and+MD+simulation+for+binding+affinity+estimation+of+metalloprotein+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm049050v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049050v%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DLukacova%26aufirst%3DV.%26aulast%3DComez%26aufirst%3DD.%26aulast%3DKroll%26aufirst%3DD.%2BM.%26aulast%3DRaha%26aufirst%3DS.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DA%2520combination%2520of%2520docking%252C%2520QM%252FMM%2520methods%252C%2520and%2520MD%2520simulation%2520for%2520binding%2520affinity%2520estimation%2520of%2520metalloprotein%2520ligands%26jtitle%3DJ.Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5437%26epage%3D5447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">QM/MM linear response method distinguishes ligand affinities for closely related metalloproteins</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=326-339&author=A.+Khandelwalauthor=S.+Balaz&title=QM%2FMM+linear+response+method+distinguishes+ligand+affinities+for+closely+related+metalloproteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DQM%252FMM%2520linear%2520response%2520method%2520distinguishes%2520ligand%2520affinities%2520for%2520closely%2520related%2520metalloproteins%26jtitle%3DProteins%26date%3D2007%26volume%3D69%26spage%3D326%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Khandelwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balaz, S.</span><span> </span><span class="NLM_article-title">Improved estimation of ligand/macromolecule binding affinities by linear response approach using a combination of multi-mode MD simulation and QM/MM methods</span> <span class="citation_source-journal">J.Comput-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2Fs10822-007-9104-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=17333483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Kntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=131-137&author=A.+Khandelwalauthor=S.+Balaz&title=Improved+estimation+of+ligand%2Fmacromolecule+binding+affinities+by+linear+response+approach+using+a+combination+of+multi-mode+MD+simulation+and+QM%2FMM+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved estimation of ligand-macromolecule binding affinities by linear response approach using a combination of multi-mode MD simulation and QM/MM methods</span></div><div class="casAuthors">Khandelwal, Akash; Balaz, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">131-137</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based predictions of binding affinities of ligands binding to proteins by coordination bonds with transition metals, covalent bonds, and bonds involving charge re-distributions are hindered by the absence of proper force fields.  This shortcoming affects all methods which use force-field-based mol. simulation data on complex formation for affinity predictions.  One of the most frequently used methods in this category is the Linear Response (LR) approach of Åquist, correlating binding affinities with van der Waals and electrostatic energies, as extended by Jorgensen's inclusion of solvent-accessible surface areas.  All these terms represent the differences, upon binding, in the ensemble avs. of pertinent quantities, obtained from mol. dynamics (MD) or Monte Carlo simulations of the complex and of single components.  Here the authors report a modification of the LR approach by: (1) the replacement of the 2 energy terms through the single-point QM/MM energy of the time-averaged complex structure from an MD simulation; and (2) a rigorous consideration of multiple modes (mm) of binding.  The first extension alleviates the force-field related problems, while the second extension deals with the ligands exhibiting large-scale motions in the course of an MD simulation.  The second modification results in the correlation equation that is nonlinear in optimized coeffs., but does not lead to an increase in the no. of optimized coeffs.  The application of the resulting mm QM/MM LR approach to the inhibition of zinc-dependent gelatinase B (matrix metalloproteinase 9) by 28 hydroxamate ligands indicates a significant improvement of descriptive and predictive abilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsCwk74Or7LVg90H21EOLACvtfcHk0ljV4-2ElKm3dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Kntr8%253D&md5=2eaa3d7972aa401f69c4ab60b0660a6b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9104-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9104-4%26sid%3Dliteratum%253Aachs%26aulast%3DKhandelwal%26aufirst%3DA.%26aulast%3DBalaz%26aufirst%3DS.%26atitle%3DImproved%2520estimation%2520of%2520ligand%252Fmacromolecule%2520binding%2520affinities%2520by%2520linear%2520response%2520approach%2520using%2520a%2520combination%2520of%2520multi-mode%2520MD%2520simulation%2520and%2520QM%252FMM%2520methods%26jtitle%3DJ.Comput-Aided.%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D131%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Donnini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmueller, H.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics in explicit solvent with λ-dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1978</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct200061r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1962-1978&author=S.+Donniniauthor=F.+Tegelerauthor=G.+Groenhofauthor=H.+Grubmueller&title=Constant+pH+molecular+dynamics+in+explicit+solvent+with+%CE%BB-dynamics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics</span></div><div class="casAuthors">Donnini, Serena; Tegeler, Florian; Groenhof, Gerrit; Grubmuller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1962-1978</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PH is an important parameter in condensed-phase systems, because it dets. the protonation state of titratable groups and thus influences the structure, dynamics, and function of mols. in soln.  In most force field simulation protocols, however, the protonation state of a system (rather than its pH) is kept fixed and cannot adapt to changes of the local environment.  Here, we present a method, implemented within the MD package GROMACS, for const. pH mol. dynamics simulations in explicit solvent that is based on the λ-dynamics approach.  In the latter, the dynamics of the titrn. coordinate λ, which interpolates between the protonated and deprotonated states, is driven by generalized forces between the protonated and deprotonated states.  The hydration free energy, as a function of pH, is included to facilitate const. pH simulations.  The protonation states of titratable groups are allowed to change dynamically during a simulation, thus reproducing av. protonation probabilities at a certain pH.  The accuracy of the method is tested against titrn. curves of single amino acids and a dipeptide in explicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp1iVOhE1DLVg90H21EOLACvtfcHk0ljV4-2ElKm3dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D&md5=0637c4d334ea58836db9e465401624c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fct200061r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200061r%26sid%3Dliteratum%253Aachs%26aulast%3DDonnini%26aufirst%3DS.%26aulast%3DTegeler%26aufirst%3DF.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DGrubmueller%26aufirst%3DH.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520with%2520%25CE%25BB-dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D1962%26epage%3D1978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vernier, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schindler, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reitz, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span> </span><span class="NLM_article-title">Pyrrolopyridine inhibitors of mitogen-activated protein kinase-activated protein kinase 2 (MK-2)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2647</span><span class="NLM_x">–</span> <span class="NLM_lpage">2654</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0611004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2647-2654&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=W.+F.+Vernierauthor=M.+W.+Mahoneyauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=J.+F.+Schindlerauthor=D.+B.+Reitzauthor=R.+J.+Mourey&title=Pyrrolopyridine+inhibitors+of+mitogen-activated+protein+kinase-activated+protein+kinase+2+%28MK-2%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm0611004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0611004%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DVernier%26aufirst%3DW.%2BF.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DSchindler%26aufirst%3DJ.%2BF.%26aulast%3DReitz%26aufirst%3DD.%2BB.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26atitle%3DPyrrolopyridine%2520inhibitors%2520of%2520mitogen-activated%2520protein%2520kinase-activated%2520protein%2520kinase%25202%2520%2528MK-2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2647%26epage%3D2654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span> </span><span class="NLM_article-title">Benzothiophene inhibitors of MK2. Part 1: Structure–activity relationships, assessments of selectivity and cellular potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4878</span><span class="NLM_x">–</span> <span class="NLM_lpage">4881</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4878-4881&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=S.+A.+Longauthor=B.+S.+Pierceauthor=M.+W.+Mahoneyauthor=R.+J.+Mourey&title=Benzothiophene+inhibitors+of+MK2.+Part+1%3A+Structure%E2%80%93activity+relationships%2C+assessments+of+selectivity+and+cellular+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DLong%26aufirst%3DS.%2BA.%26aulast%3DPierce%26aufirst%3DB.%2BS.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26atitle%3DBenzothiophene%2520inhibitors%2520of%2520MK2.%2520Part%25201%253A%2520Structure%25E2%2580%2593activity%2520relationships%252C%2520assessments%2520of%2520selectivity%2520and%2520cellular%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4878%26epage%3D4881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Anderson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyers, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caspers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourey, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, M. D.</span><span> </span><span class="NLM_article-title">Benzothiophene inhibitors of MK2. Part 2: Improvements in kinase selectivity and cell potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4882</span><span class="NLM_x">–</span> <span class="NLM_lpage">4884</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4882-4884&author=D.+R.+Andersonauthor=M.+J.+Meyersauthor=R.+G.+Kurumbailauthor=N.+Caspersauthor=G.+I.+Podaauthor=S.+A.+Longauthor=B.+S.+Pierceauthor=M.+W.+Mahoneyauthor=R.+J.+Moureyauthor=M.+D.+Parikh&title=Benzothiophene+inhibitors+of+MK2.+Part+2%3A+Improvements+in+kinase+selectivity+and+cell+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DMeyers%26aufirst%3DM.%2BJ.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DLong%26aufirst%3DS.%2BA.%26aulast%3DPierce%26aufirst%3DB.%2BS.%26aulast%3DMahoney%26aufirst%3DM.%2BW.%26aulast%3DMourey%26aufirst%3DR.%2BJ.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26atitle%3DBenzothiophene%2520inhibitors%2520of%2520MK2.%2520Part%25202%253A%2520Improvements%2520in%2520kinase%2520selectivity%2520and%2520cell%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4882%26epage%3D4884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Velcicky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feifel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawtin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heng, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppertz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroemer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moebitz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revesz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheufler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlapbach, A.</span><span> </span><span class="NLM_article-title">Novel 3-aminopyrazole inhibitors of MK-2 discovered by scaffold hopping strategy</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1293</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1293-1297&author=J.+Velcickyauthor=R.+Feifelauthor=S.+Hawtinauthor=R.+Hengauthor=C.+Huppertzauthor=G.+Kochauthor=M.+Kroemerauthor=H.+Moebitzauthor=L.+Reveszauthor=C.+Scheuflerauthor=A.+Schlapbach&title=Novel+3-aminopyrazole+inhibitors+of+MK-2+discovered+by+scaffold+hopping+strategy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVelcicky%26aufirst%3DJ.%26aulast%3DFeifel%26aufirst%3DR.%26aulast%3DHawtin%26aufirst%3DS.%26aulast%3DHeng%26aufirst%3DR.%26aulast%3DHuppertz%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DG.%26aulast%3DKroemer%26aufirst%3DM.%26aulast%3DMoebitz%26aufirst%3DH.%26aulast%3DRevesz%26aufirst%3DL.%26aulast%3DScheufler%26aufirst%3DC.%26aulast%3DSchlapbach%26aufirst%3DA.%26atitle%3DNovel%25203-aminopyrazole%2520inhibitors%2520of%2520MK-2%2520discovered%2520by%2520scaffold%2520hopping%2520strategy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D1293%26epage%3D1297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Hillig, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberspaecher, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteclaro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller-Tiemann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span> </span><span class="NLM_article-title">Structural basis for a high affinity inhibitor bound to protein kinase MK2</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">369</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">745</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2007&pages=735-745&author=R.+C.+Hilligauthor=U.+Eberspaecherauthor=F.+Monteclaroauthor=M.+Huberauthor=D.+Nguyenauthor=A.+Mengelauthor=B.+Muller-Tiemannauthor=U.+Egner&title=Structural+basis+for+a+high+affinity+inhibitor+bound+to+protein+kinase+MK2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DEberspaecher%26aufirst%3DU.%26aulast%3DMonteclaro%26aufirst%3DF.%26aulast%3DHuber%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DMengel%26aufirst%3DA.%26aulast%3DMuller-Tiemann%26aufirst%3DB.%26aulast%3DEgner%26aufirst%3DU.%26atitle%3DStructural%2520basis%2520for%2520a%2520high%2520affinity%2520inhibitor%2520bound%2520to%2520protein%2520kinase%2520MK2%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D369%26spage%3D735%26epage%3D745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Gaestel, M.</span><span> </span><span class="NLM_article-title">MAPKAP kinases, MKs—two’s company, three’s a crowd</span> <span class="citation_source-journal">Nature Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1038%2Fnrm1834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16421520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=120-130&author=M.+Gaestel&title=MAPKAP+kinases%2C+MKs%E2%80%94two%E2%80%99s+company%2C+three%E2%80%99s+a+crowd"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MAPKAP kinases - MKs - two's company, three's a crowd</span></div><div class="casAuthors">Gaestel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-130</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Downstream of MAP kinases (MAPKs), 3 structurally related MAPK-activated protein kinases (MAPKAPKs or MKs), MK2, MK3, and MK5, signal to diverse cellular targets.  Although there is no known common function for all 3 MKs, these kinases are involved in important processes.  MKs regulate gene expression at the transcriptional and post-transcriptional level, control cytoskeletal architecture and cell cycle progression, and are implicated in inflammation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI91xfvhmqhrVg90H21EOLACvtfcHk0lho9Qq0euw0Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWntr4%253D&md5=1ae824b7d680a574a7c764bd61ac43aa</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrm1834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1834%26sid%3Dliteratum%253Aachs%26aulast%3DGaestel%26aufirst%3DM.%26atitle%3DMAPKAP%2520kinases%252C%2520MKs%25E2%2580%2594two%25E2%2580%2599s%2520company%252C%2520three%25E2%2580%2599s%2520a%2520crowd%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D120%26epage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Kotlyarov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neininger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birchmeier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volk, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaestel, M.</span><span> </span><span class="NLM_article-title">MAPKAP kinase 2 is essential for LPS-induced TNFalpha biosynthesis</span> <span class="citation_source-journal">Nature Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1038%2F10061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=10559880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVGqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=94-97&author=A.+Kotlyarovauthor=A.+Neiningerauthor=C.+Schubertauthor=R.+Eckertauthor=C.+Birchmeierauthor=H.+D.+Volkauthor=M.+Gaestel&title=MAPKAP+kinase+2+is+essential+for+LPS-induced+TNFalpha+biosynthesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">MAPKAP kinase 2 is essential for LPS-induced TNF-α biosynthesis</span></div><div class="casAuthors">Kotlyarov, Alexey; Neininger, Armin; Schubert, Carola; Eckert, Rolf; Birchmeier, Carmen; Volk, Hans-Dieter; Gaestel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">94-97</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">MAPKAP kinase 2 (MK2) is one of several kinases that are regulated through direct phosphorylation by p38 MAP kinase.  By introducing a targeted mutation into the mouse MK2 gene, the authors have detd. the physiol. function of MK2 in vivo.  Mice that lack MK2 show increased stress resistance and survive LPS-induced endotoxic shock.  This is due to a redn. of ∼90% in the prodn. of tumor necrosis factor-α (TNF-α) and not to a change in signaling from the TNF receptor.  The level and stability of TNF-α mRNA is not reduced and TNF-α secretion is not affected.  Thus, MK2 is an essential component in the inflammatory response which regulates biosynthesis of TNF-α at a post-transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtiobBLSnV77Vg90H21EOLACvtfcHk0lho9Qq0euw0Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVGqsLY%253D&md5=5fb502616639b298341431eea8009cc2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2F10061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10061%26sid%3Dliteratum%253Aachs%26aulast%3DKotlyarov%26aufirst%3DA.%26aulast%3DNeininger%26aufirst%3DA.%26aulast%3DSchubert%26aufirst%3DC.%26aulast%3DEckert%26aufirst%3DR.%26aulast%3DBirchmeier%26aufirst%3DC.%26aulast%3DVolk%26aufirst%3DH.%2BD.%26aulast%3DGaestel%26aufirst%3DM.%26atitle%3DMAPKAP%2520kinase%25202%2520is%2520essential%2520for%2520LPS-induced%2520TNFalpha%2520biosynthesis%26jtitle%3DNature%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D94%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Dambach, D. M.</span><span> </span><span class="NLM_article-title">Potential adverse effects associated with inhibition of p38α/β MAP Kinases</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">929</span><span class="NLM_x">–</span> <span class="NLM_lpage">939</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.2174%2F1568026054985911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16178738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFeitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=929-939&author=D.+M.+Dambach&title=Potential+adverse+effects+associated+with+inhibition+of+p38%CE%B1%2F%CE%B2+MAP+Kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Potential adverse effects associated with inhibition of p38α/β MAP kinases</span></div><div class="casAuthors">Dambach, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibitors of p38 MAP kinases show promise for the treatment of inflammatory and immunol. disorders and some cancers.  There is a substantial body of exptl. evidence across several organ systems suggesting that p38 also mediates developmental, differentiation and proliferation processes.  As a consequence of the wide-ranging regulatory role of p38 kinase in diverse cellular processes, the possibility of adverse events resulting from undesired pharmacol. activity is a major concern for the p38 inhibitor drug class.  Taking into consideration the limitations of exptl. modeling systems, together the data may indicate that profound inhibition of p38 has the potential to impact these processes during fetal or neonatal development.  The difficulty comes in extrapolating these findings to predict potential adverse effects under conditions of partial inhibition of p38 activity, and in an adult population in which these processes are typically only recapitulated during repair or adaptive responses.  As such, the goal of this review of the targets of p38 activity is to bring an awareness of the those organ systems that should be monitored for potential toxicity, as well as to present a potential mechanistic basis for such monitoring or for investigation of adverse effects that may develop with administration of a p38 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9YoS9QjJnfrVg90H21EOLACvtfcHk0lglRbEcgu8PLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFeitrbM&md5=ece8a45395222f3039bc51485453ac15</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568026054985911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054985911%26sid%3Dliteratum%253Aachs%26aulast%3DDambach%26aufirst%3DD.%2BM.%26atitle%3DPotential%2520adverse%2520effects%2520associated%2520with%2520inhibition%2520of%2520p38%25CE%25B1%252F%25CE%25B2%2520MAP%2520Kinases%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D929%26epage%3D939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Duraisamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajpai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bughani, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastidar, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, P.</span><span> </span><span class="NLM_article-title">MK2: A novel molecular target for anti-inflammatory therapy</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">921</span><span class="NLM_x">–</span> <span class="NLM_lpage">936</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1517%2F14728222.12.8.921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=18620516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ChtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=921-936&author=S.+Duraisamyauthor=M.+Bajpaiauthor=U.+Bughaniauthor=S.+G.+Dastidarauthor=A.+Rayauthor=P.+Chopra&title=MK2%3A+A+novel+molecular+target+for+anti-inflammatory+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MK2: a novel molecular target for anti-inflammatory therapy</span></div><div class="casAuthors">Duraisamy, Senthil; Bajpai, Malini; Bughani, Usha; Dastidar, Sunanda G.; Ray, Abhijit; Chopra, Puneet</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">921-936</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinase (MAPK)-activated protein kinase 2 (MK2) is activated upon stress by p38 MAPK.  MK2 is stimulated in a wide range of inflammatory conditions and its catalytic activity is required for cytokine prodn., cell migration and is a potential drug target for inflammatory diseases.  Disruption of MK2 leads to a redn. in TNF-α prodn.  MK2-mediated pro-inflammatory cytokine prodn. has been demonstrated in several inflammatory conditions where TNF-α plays a role.  We discuss the development of specific MK2 inhibitors for the treatment of inflammatory diseases.  Inhibition of the p38 MAPK pathway may have therapeutic uses for inflammatory diseases.  However, blocking p38 MAPK activation in vivo is not advisable due to toxicity, significant off-target effects, and lack of oral bioavailability.  This concern may be countered by the use of MK2 inhibitors that can dissect the pathways downstream of p38 without affecting addnl. cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_tMhx866ALbVg90H21EOLACvtfcHk0lglRbEcgu8PLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ChtL0%253D&md5=46eefd34bc965b84b34234314fdb6c98</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F14728222.12.8.921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.12.8.921%26sid%3Dliteratum%253Aachs%26aulast%3DDuraisamy%26aufirst%3DS.%26aulast%3DBajpai%26aufirst%3DM.%26aulast%3DBughani%26aufirst%3DU.%26aulast%3DDastidar%26aufirst%3DS.%2BG.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DChopra%26aufirst%3DP.%26atitle%3DMK2%253A%2520A%2520novel%2520molecular%2520target%2520for%2520anti-inflammatory%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2008%26volume%3D12%26spage%3D921%26epage%3D936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Hilal, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karickhoff, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreira, L. A.</span><span> </span><span class="NLM_article-title">Estimation of microscopic, zwitterionic ionization constants, isoelectric point and molecular speciation of organic compounds</span> <span class="citation_source-journal">Talanta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">827</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1999&pages=827-840&author=S.+H.+Hilalauthor=S.+W.+Karickhoffauthor=L.+A.+Carreira&title=Estimation+of+microscopic%2C+zwitterionic+ionization+constants%2C+isoelectric+point+and+molecular+speciation+of+organic+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHilal%26aufirst%3DS.%2BH.%26aulast%3DKarickhoff%26aufirst%3DS.%2BW.%26aulast%3DCarreira%26aufirst%3DL.%2BA.%26atitle%3DEstimation%2520of%2520microscopic%252C%2520zwitterionic%2520ionization%2520constants%252C%2520isoelectric%2520point%2520and%2520molecular%2520speciation%2520of%2520organic%2520compounds%26jtitle%3DTalanta%26date%3D1999%26volume%3D50%26spage%3D827%26epage%3D840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J. H.</span><span> </span><span class="NLM_article-title">Very fast empirical prediction and rationalization of protein p<i>K</i><sub>a</sub> values</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">704</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fprot.20660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16231289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=704-721&author=H.+Liauthor=A.+D.+Robertsonauthor=J.+H.+Jensen&title=Very+fast+empirical+prediction+and+rationalization+of+protein+pKa+values"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Very fast empirical prediction and rationalization of protein pKa values</span></div><div class="casAuthors">Li, Hui; Robertson, Andrew D.; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">704-721</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A very fast empirical method is presented for structure-based protein pKa prediction and rationalization.  The desolvation effects and intra-protein interactions, which cause variations in pKa values of protein ionizable groups, are empirically related to the positions and chem. nature of the groups proximate to the pKa sites.  A computer program is written to automatically predict pKa values based on these empirical relationships within a couple of seconds.  Unusual pKa values at buried active sites, which are among the most interesting protein pKa values, are predicted very well with the empirical method.  A test on 233 carboxyl, 12 cysteine, 45 histidine, and 24 lysine pKa values in various proteins shows a root-mean-square deviation (RMSD) of 0.89 from exptl. values.  Removal of the 29 pKa values that are upper or lower limits results in an RMSD = 0.79 for the remaining 285 pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYkiHyYz6FobVg90H21EOLACvtfcHk0lglRbEcgu8PLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN&md5=9e33529954c65867929326b99dac493b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fprot.20660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20660%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520empirical%2520prediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%26jtitle%3DProteins%26date%3D2005%26volume%3D61%26spage%3D704%26epage%3D721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Multiple-site titration and molecular modeling: Two rapid methods for computing energies and forces for ionizable groups in proteins</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1993&pages=266-282&author=M.+K.+Gilson&title=Multiple-site+titration+and+molecular+modeling%3A+Two+rapid+methods+for+computing+energies+and+forces+for+ionizable+groups+in+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DMultiple-site%2520titration%2520and%2520molecular%2520modeling%253A%2520Two%2520rapid%2520methods%2520for%2520computing%2520energies%2520and%2520forces%2520for%2520ionizable%2520groups%2520in%2520proteins%26jtitle%3DProteins%26date%3D1993%26volume%3D15%26spage%3D266%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Antosiewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Prediction of pH-dependent properties of proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">238</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1006%2Fjmbi.1994.1301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=8176733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK2cXktVCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=1994&pages=415-436&author=J.+Antosiewiczauthor=J.+A.+McCammonauthor=M.+K.+Gilson&title=Prediction+of+pH-dependent+properties+of+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pH-dependent properties of proteins</span></div><div class="casAuthors">Antosiewicz, Jan; McCammon, J. Andrew; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">415-36</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe what may be the most accurate approach currently available for the calcn. of the pKas of ionizable groups in proteins.  The accuracy is assessed by comparison of computed pKas with 60 measured pKas in a total of seven proteins.  The overall root-mean-square error is 0.89 pKa units.  Linear regression anal. of computed vs. measured pKas yields a slope of 0.95, yr-intercept of -0.02 and a correlation coeff. of 0.96.  The proposed approach also picks out many of the shifted pKas of groups in enzyme active sites and special salt bridges.  However, it does yield several over-shifted pKas and tends to underest. pKa shifts which result from desolvation effects.  The authors examine the ability of the new approach to reproduce the dependence of protein stability upon pH, using the ionization polynomial formalism.  Overall features of the stability curves are reproduced, but the quant. agreement is not particularly good.  The reasons for the disagreement may have to do both with insufficient accuracy in the theory and with uncertainty in the nature of the unfolded state of proteins.  The methodol. described here is based upon finite difference solns. of the Poisson-Boltzmann equation.  Its success depends upon the use of the rather high protein dielec. const. of 20.  However, theor. considerations and the fact that pKa shifts which result from desolvation are underestd. here imply that the dielec. const. of the protein interior actually is lower than 20.  The authors suggest that the high protein dielec. const. improves the overall agreement with expt. because it accounts approx. for phenomena which tend to mitigate pKa shifts and which are not specifically included in the model.  These include conformational relaxation and specific ion-binding.  Future models based upon a low protein dielec. const. and treating such phenomena explicitly might yield improved agreement with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy0UmLyANFBrVg90H21EOLACvtfcHk0ljDdqPfeLtc7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVCntbk%253D&md5=f029ddd8fcbee0708fe44e0ee5e0e29c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1301%26sid%3Dliteratum%253Aachs%26aulast%3DAntosiewicz%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DPrediction%2520of%2520pH-dependent%2520properties%2520of%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D238%26spage%3D415%26epage%3D436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Antosiewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computing ionization states of proteins with a detailed charge model</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1633</span><span class="NLM_x">–</span> <span class="NLM_lpage">1644</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=1633-1644&author=J.+Antosiewiczauthor=J.+W.+Briggsauthor=A.+H.+Elcockauthor=M.+K.+Gilsonauthor=J.+A.+McCammon&title=Computing+ionization+states+of+proteins+with+a+detailed+charge+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntosiewicz%26aufirst%3DJ.%26aulast%3DBriggs%26aufirst%3DJ.%2BW.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputing%2520ionization%2520states%2520of%2520proteins%2520with%2520a%2520detailed%2520charge%2520model%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D1633%26epage%3D1644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Modeling protonation equilibria in biomolecules</span>. In  <span class="citation_source-book">Computer Simulation of Biomolecular Systems</span>; <span class="NLM_contrib-group">van Gunsteren, W. F.; Weiner, P. K.; Wilkinson, A. J.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">3</span>, pp  <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1007%2F978-94-017-1120-3_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=199-222&author=M.+K.+Gilsonauthor=W.+F.+van+Gunsteren&author=P.+K.+Weiner&author=A.+J.+Wilkinson&title=Computer+Simulation+of+Biomolecular+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-94-017-1120-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-017-1120-3_7%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DModeling%2520protonation%2520equilibria%2520in%2520biomolecules%26btitle%3DComputer%2520Simulation%2520of%2520Biomolecular%2520Systems%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26pub%3DSpringer%26date%3D1997%26volume%3D3%26spage%3D199%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Improving the accuracy of protein p<i>K</i><sub>a</sub> calculations: Conformational averaging versus the average structure</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">145</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=145-158&author=H.+W.+T.+Van+Vlijmenauthor=M.+Schaeferauthor=M.+Karplus&title=Improving+the+accuracy+of+protein+pKa+calculations%3A+Conformational+averaging+versus+the+average+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DImproving%2520the%2520accuracy%2520of%2520protein%2520pKa%2520calculations%253A%2520Conformational%2520averaging%2520versus%2520the%2520average%2520structure%26jtitle%3DProteins%26date%3D1998%26volume%3D33%26spage%3D145%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Prediction of p<i>K</i><sub>a</sub> shifts in proteins using a combination of molecular mechanical and continuum solvent calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1865</span><span class="NLM_x">–</span> <span class="NLM_lpage">1872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=15376253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1865-1872&author=B.+Kuhnauthor=P.+A.+Kollmanauthor=M.+Stahl&title=Prediction+of+pKa+shifts+in+proteins+using+a+combination+of+molecular+mechanical+and+continuum+solvent+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pKa shifts in proteins using a combination of molecular mechanical and continuum solvent calculations</span></div><div class="casAuthors">Kuhn, Bernd; Kollman, Peter A.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1865-1872</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The prediction of pKa shifts of ionizable groups in proteins is of great relevance for a no. of important biol. phenomena.  We present an implementation of the MM-GBSA approach, which combines mol. mech. (MM) and generalized Born (GB) continuum solvent energy terms, to the calcn. of pKa values of a panel of nine proteins, including 69 individual comparisons with expt.  While applied so far mainly to the calcn. of biomol. binding free energies, we show that this method can also be used for the estn. of protein pKa shifts, with an accuracy around 1 pKa unit, even for strongly shifted residues.  Our anal. reveals that the nonelectrostatic terms that are part of the MM-GBSA free energy expression are important contributors to improved prediction accuracy.  This suggests that most of the previous approaches that focus only on electrostatic interactions could be improved by adding other nonpolar energy terms to their free energy expression.  Interestingly, our method yields best accuracy at protein dielec. consts. of εint = 2-4, which is in contrast to previous approaches that peak at higher εint ≥ 8.  An important component of our procedure is an intermediate minimization step of each protonation state involving different rotamers and tautomers as a way to explicitly model protein relaxation upon (de)protonation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ5j9GZtXBELVg90H21EOLACvtfcHk0ljDdqPfeLtc7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVyjsr0%253D&md5=e91b3f62ac9f8cd67beeeb064fd96aed</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20111%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DPrediction%2520of%2520pKa%2520shifts%2520in%2520proteins%2520using%2520a%2520combination%2520of%2520molecular%2520mechanical%2520and%2520continuum%2520solvent%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1865%26epage%3D1872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Jensen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, P. A.</span><span> </span><span class="NLM_article-title">Prediction and rationalization of protein p<i>K</i><sub>a</sub> values using QM and QM/MM methods</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6634</span><span class="NLM_x">–</span> <span class="NLM_lpage">6643</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp051922x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVChtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6634-6643&author=J.+H.+Jensenauthor=H.+Liauthor=A.+D.+Robertsonauthor=P.+A.+Molina&title=Prediction+and+rationalization+of+protein+pKa+values+using+QM+and+QM%2FMM+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction and Rationalization of Protein pKa Values Using QM and QM/MM Methods</span></div><div class="casAuthors">Jensen, Jan H.; Li, Hui; Robertson, Andrew D.; Molina, Pablo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6634-6643</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe the development and application of a computational method for the prediction and rationalization of pKa values of ionizable residues in proteins, based on ab initio quantum mechanics (QM) and the effective fragment potential (EFPs) method (a hybrid QM/MM method).  The theor. developments include (1) a covalent boundary method based on frozen localized orbitals, (2) divide-and-conquer methods for the ab initio computation of protein EFPs consisting of multipoles up to octupoles and dipole polarizability tensors, (3) a method for computing vibrational free energies for a localized mol. region, and (4) solns. of the polarized continuum model of bulk solvation equations for protein-sized systems.  The QM-based pKa prediction method is one of the most accurate methods currently available and can be used in cases where other pKa prediction methods fail.  Preliminary anal. of the computed results indicate that many pKa values (1) are primarily detd. by hydrogen bonds rather than long-range charge-charge interactions and (2) are relatively insensitive to large-scale dynamical fluctuations of the protein structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHFgRei3zrGLVg90H21EOLACvtfcHk0lhtL157sNhspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVChtr0%253D&md5=58696f960a47c34c95d0592d6b9299d0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjp051922x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp051922x%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DMolina%26aufirst%3DP.%2BA.%26atitle%3DPrediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%2520using%2520QM%2520and%2520QM%252FMM%2520methods%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2005%26volume%3D109%26spage%3D6634%26epage%3D6643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Gordon, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoja, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heath, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span> </span><span class="NLM_article-title">H++: A server for estimating p<i>K</i><sub>a</sub>s and adding missing hydrogens to macromolecules</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">W368</span><span class="NLM_x">–</span> <span class="NLM_lpage">W371</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+A+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520A%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Bas, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J. H.</span><span> </span><span class="NLM_article-title">Very fast prediction and rationalization of p<i>K</i><sub>a</sub> values for protein–ligand complexes</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">765</span><span class="NLM_x">–</span> <span class="NLM_lpage">783</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=765-783&author=D.+C.+Basauthor=D.+M.+Rogersauthor=J.+H.+Jensen&title=Very+fast+prediction+and+rationalization+of+pKa+values+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBas%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DD.%2BM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520prediction%2520and%2520rationalization%2520of%2520pKa%2520values%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DProteins%26date%3D2008%26volume%3D73%26spage%3D765%26epage%3D783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Song, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunner, M. R.</span><span> </span><span class="NLM_article-title">MCCE2: Improving protein p<i>K</i><sub>a</sub> calculations with extensive side chain rotamer sampling</span> <span class="citation_source-journal">J.Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2231</span><span class="NLM_x">–</span> <span class="NLM_lpage">2247</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=19274707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOrsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2231-2247&author=Y.+F.+Songauthor=J.+J.+Maoauthor=M.+R.+Gunner&title=MCCE2%3A+Improving+protein+pKa+calculations+with+extensive+side+chain+rotamer+sampling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MCCE2: Improving protein pKa calculations with extensive side chain rotamer sampling</span></div><div class="casAuthors">Song, Yifan; Mao, Junjun; Gunner, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2231-2247</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Multiconformation continuum electrostatics (MCCE) explores different conformational degrees of freedom in Monte Carlo calcns. of protein residue and ligand pKas.  Explicit changes in side chain conformations throughout a titrn. create a position dependent, heterogeneous dielec. response giving a more accurate picture of coupled ionization and position changes.  The MCCE2 methods for choosing a group of input heavy atom and proton positions are described.  The pKas calcd. with different isosteric conformers, heavy atom rotamers and proton positions, with different degrees of optimization are tested against a curated group of 305 exptl. pKas in 33 proteins.  QUICK calcns., with rotation around Asn and Gln termini, sampling His tautomers and torsion min. hydroxyls yield an RMSD of 1.34 with 84% of the errors being <1.5 pH units.  FULL calcns. adding heavy atom rotamers and side chain optimization yield an RMSD of 0.90 with 90% of the errors <1.5 pH unit.  Good results are also found for pKas in the membrane protein bacteriorhodopsin.  The inclusion of extra side chain positions distorts the dielec. boundary and also biases the calcd. pKas by creating more neutral than ionized conformers.  Methods for correcting these errors are introduced.  Calcns. are compared with multiple x-ray and NMR derived structures in 36 sol. proteins.  Calcns. with x-ray structures give significantly better pKas.  Results with the default protein dielec. const. of 4 are as good as those using a value of 8.  The MCCE2 program can be downloaded from http://www.sci.ccny.cuny.edu/~mcce.  © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY__IipXaBcLVg90H21EOLACvtfcHk0lhtL157sNhspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOrsLjM&md5=43c57ec68c03af687d898702f8cc8ca5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%2BF.%26aulast%3DMao%26aufirst%3DJ.%2BJ.%26aulast%3DGunner%26aufirst%3DM.%2BR.%26atitle%3DMCCE2%253A%2520Improving%2520protein%2520pKa%2520calculations%2520with%2520extensive%2520side%2520chain%2520rotamer%2520sampling%26jtitle%3DJ.Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2231%26epage%3D2247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K</i><sub>I</sub>) and the concentration of inhibitor which causes 50% inhibition (<i>I</i><sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lhtL157sNhspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0liFuHDTqOqdPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0liFuHDTqOqdPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Vreven, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morokuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farkas, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisch, M. J.</span><span> </span><span class="NLM_article-title">Geometry optimization with QM/MM, ONIOM, and other combined methods. I. Microiterations and constraints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">769</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.10156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12666168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFWqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=760-769&author=T.+Vrevenauthor=K.+Morokumaauthor=O.+Farkasauthor=H.+B.+Schlegelauthor=M.+J.+Frisch&title=Geometry+optimization+with+QM%2FMM%2C+ONIOM%2C+and+other+combined+methods.+I.+Microiterations+and+constraints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Geometry optimization with QM/MM, ONIOM, and other combined methods. I. Microiterations and constraints</span></div><div class="casAuthors">Vreven, Thom; Morokuma, Keiji; Farkas, Odon; Schlegel, H. Bernhard; Frisch, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">760-769</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hybrid energy methods such as QM/MM and ONIOM, that combine different levels of theory into one calcn., have been very successful in describing large systems.  Geometry optimization methods can take advantage of the partitioning of these calcns. into a region treated at a quantum mech. (QM) level of theory and the larger, remaining region treated by an inexpensive method such as mol. mechanics (MM).  A series of microiterations can be employed to fully optimize the MM region for each optimization step in the QM region.  Cartesian coordinates are used for the MM region and are chosen so that the internal coordinates of the QM region remain const. during the microiterations.  The coordinates of the MM region are augmented to permit rigid body translation and rotation of the QM region.  This is essential if any atoms in the MM region are constrained, but it also improves the efficiency of unconstrained optimizations.  Because of the microiterations, special care is needed for the optimization step in the QM region so that the system remains in the same local valley during the course of the optimization.  The optimization methodol. with microiterations, constraints, and step-size control are illustrated by calcns. on bacteriorhodopsin and other systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLKbGjxXu0LVg90H21EOLACvtfcHk0liFuHDTqOqdPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFWqtrc%253D&md5=d8dbb3fc8ca569cf5ec7ef85bd20694d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10156%26sid%3Dliteratum%253Aachs%26aulast%3DVreven%26aufirst%3DT.%26aulast%3DMorokuma%26aufirst%3DK.%26aulast%3DFarkas%26aufirst%3DO.%26aulast%3DSchlegel%26aufirst%3DH.%2BB.%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26atitle%3DGeometry%2520optimization%2520with%2520QM%252FMM%252C%2520ONIOM%252C%2520and%2520other%2520combined%2520methods.%2520I.%2520Microiterations%2520and%2520constraints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D760%26epage%3D769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Vreven, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byun, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komiromi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dapprich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morokuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisch, M. J.</span><span> </span><span class="NLM_article-title">Combining quantum mechanics methods with molecular mechanics methods in ONIOM</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct050289g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFWqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=815-826&author=T.+Vrevenauthor=K.+S.+Byunauthor=I.+Komiromiauthor=S.+Dapprichauthor=A.+Johnauthor=K.+Morokumaauthor=M.+J.+Frisch&title=Combining+quantum+mechanics+methods+with+molecular+mechanics+methods+in+ONIOM"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Quantum Mechanics Methods with Molecular Mechanics Methods in ONIOM</span></div><div class="casAuthors">Vreven, Thom; Byun, K. Suzie; Komaromi, Istvan; Dapprich, Stefan; Montgomery, John A., Jr.; Morokuma, Keiji; Frisch, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The purpose of this paper is 2-fold.  First, we present several extensions to the ONIOM(QM:MM) scheme.  In its original formulation, the electrostatic interaction between the regions is included at the classical level.  Here we present the extension to electronic embedding.  We show how the behavior of ONIOM with electronic embedding can be more stable than QM/MM with electronic embedding.  We also investigate the link atom correction, which is implicit in ONIOM but not in QM/MM.  Second, we demonstrate some of the practical aspects of ONIOM(QM:MM) calcns.  Specifically, we show that the potential surface can be discontinuous when there is bond breaking and forming closer than three bonds from the MM region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorbf2n2RA2NbVg90H21EOLACvtfcHk0li_qvF38pL65g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFWqsb0%253D&md5=3cf69310302f73c284f6721b3996d4a5</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct050289g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct050289g%26sid%3Dliteratum%253Aachs%26aulast%3DVreven%26aufirst%3DT.%26aulast%3DByun%26aufirst%3DK.%2BS.%26aulast%3DKomiromi%26aufirst%3DI.%26aulast%3DDapprich%26aufirst%3DS.%26aulast%3DJohn%26aufirst%3DA.%26aulast%3DMorokuma%26aufirst%3DK.%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26atitle%3DCombining%2520quantum%2520mechanics%2520methods%2520with%2520molecular%2520mechanics%2520methods%2520in%2520ONIOM%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2006%26volume%3D2%26spage%3D815%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Frisch, M. J.</span>Gaussian 09, version A.1; <span class="NLM_publisher-name">Gaussian Inc.</span>, <span class="NLM_publisher-loc">Wallingford, CT 06492, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=M.+J.+Frisch&title=Gaussian+09%2C+version+A.1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%252C%2520version%2520A.1%26pub%3DGaussian%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0li_qvF38pL65g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span>Premier Solver Platform version 10.5; <span class="NLM_publisher-name">Frontline Systems Inc.</span>: <span class="NLM_publisher-loc">Incline Village, NV, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Premier+Solver+Platform+version+10.5%3B+Frontline+Systems+Inc.%3A+Incline+Village%2C+NV%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPremier%2520Solver%2520Platform%2520version%252010.5%26pub%3DFrontline%2520Systems%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span>Sybyl-X version 1.1; <span class="NLM_publisher-name">Tripos Inc.</span>: <span class="NLM_publisher-loc">St. Louis, MO, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sybyl-X+version+1.1%3B+Tripos+Inc.%3A+St.+Louis%2C+MO%2C+USA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSybyl-X%2520version%25201.1%26pub%3DTripos%2520Inc%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Mulliken, R. S.</span><span> </span><span class="NLM_article-title">Electronic population analysis on LCAO-MO [linear combination of atomic orbital-molecular orbital] molecular wave functions. I</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1955</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1833</span><span class="NLM_x">–</span> <span class="NLM_lpage">1840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1955&pages=1833-1840&author=R.+S.+Mulliken&title=Electronic+population+analysis+on+LCAO-MO+%5Blinear+combination+of+atomic+orbital-molecular+orbital%5D+molecular+wave+functions.+I"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMulliken%26aufirst%3DR.%2BS.%26atitle%3DElectronic%2520population%2520analysis%2520on%2520LCAO-MO%2520%255Blinear%2520combination%2520of%2520atomic%2520orbital-molecular%2520orbital%255D%2520molecular%2520wave%2520functions.%2520I%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1955%26volume%3D23%26spage%3D1833%26epage%3D1840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span>Jaguar; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">LLC Portland, OR, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Jaguar%3B+Schr%C3%B6dinger%3A+LLC+Portland%2C+OR%2C+USA%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DJaguar%26pub%3DSchr%25C3%25B6dinger%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. III. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+III.+The+role+of+exact+exchange"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0li_qvF38pL65g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520III.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Hay, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wadt, W. R.</span><span> </span><span class="NLM_article-title">Ab initio effective core potentials for molecular calculations. Potentials for the transition metal atoms scandium to mercury</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">270</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.448799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL2MXhtlyju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1985&pages=270-283&author=P.+J.+Hayauthor=W.+R.+Wadt&title=Ab+initio+effective+core+potentials+for+molecular+calculations.+Potentials+for+the+transition+metal+atoms+scandium+to+mercury"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Ab initio effective core potentials for molecular calculations.  Potentials for the transition metal atoms scandium to mercury</span></div><div class="casAuthors">Hay, P. Jeffrey; Wadt, Willard R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">270-83</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Ab initio effective core potentials (ECP's) were generated to replace the Coulomb, exchange, and core-orthogonality effects of the chem. inert core electron in the transition metal atoms Sc to Hg.  For the second and third transition series relative ECP's were generated which also incorporate the mass-velocity and Darwin relativistic effects into the potential.  The ab initio ECP's should facilitate valence electron calcns. on mols. contg. transition-metal atoms with accuracies approaching all-electron calcns. at a fraction of the computational cost.  Analytic fits to the potentials are presented for use in multicenter integral evaluation.  Gaussian orbital valence basis sets are developed for the (3d,4s,4p), (4d,5s,5p), and (5d,6s,6p) orbitals of the first, second, and third transition series atoms, resp.  All-electron and valence-electron at. excitation energies are also compared for the low-lying states of Sc-Hg, and the valence-electron calcns. reproduce the all-electron excitation energies (typically within a few tenths of an eV).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIEl284u2R9rVg90H21EOLACvtfcHk0lj37UN29J3QSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhtlyju70%253D&md5=29271d2a54b5c81acd19762c570e64d7</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1063%2F1.448799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448799%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DP.%2BJ.%26aulast%3DWadt%26aufirst%3DW.%2BR.%26atitle%3DAb%2520initio%2520effective%2520core%2520potentials%2520for%2520molecular%2520calculations.%2520Potentials%2520for%2520the%2520transition%2520metal%2520atoms%2520scandium%2520to%2520mercury%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D82%26spage%3D270%26epage%3D283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Wadt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, P. J.</span><span> </span><span class="NLM_article-title">Ab initio effective core potentials for molecular calculations. Potentials for main group elements sodium to bismuth</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.448800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL2MXht1SjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1985&pages=284-298&author=W.+R.+Wadtauthor=P.+J.+Hay&title=Ab+initio+effective+core+potentials+for+molecular+calculations.+Potentials+for+main+group+elements+sodium+to+bismuth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Ab initio effective core potentials for molecular calculations.  Potentials for main group elements sodium to bismuth</span></div><div class="casAuthors">Wadt, Willard R.; Hay, P. Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">284-98</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A consistent set of ab initio effective core potentials (ECP) was generated for the main group elements from Na to Bi by using the procedure originally developed by Kahn (1976).  For Rb to Bi the ECP's are generated from the relativistic Hartree-Fock at. wave functions of Cowan which incorporate the Darwin and mass-velocity terms.  Energy-optimized valence basis sets of (3s3p) primitive Gaussians are presented for use with the ECP's.  Comparisons between all-electron and valence-electron ECP calcns. are presented for NaF, NaCl, Cl2, Cl2-, Br2, Br2-, and Xe2+.  The results show that the av. errors introduced by the ECP's are generally only a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt2lGOejwqOLVg90H21EOLACvtfcHk0lj37UN29J3QSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXht1SjtLk%253D&md5=16288166d7c3a65ba452c4673b7c7d2a</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1063%2F1.448800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448800%26sid%3Dliteratum%253Aachs%26aulast%3DWadt%26aufirst%3DW.%2BR.%26aulast%3DHay%26aufirst%3DP.%2BJ.%26atitle%3DAb%2520initio%2520effective%2520core%2520potentials%2520for%2520molecular%2520calculations.%2520Potentials%2520for%2520main%2520group%2520elements%2520sodium%2520to%2520bismuth%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D82%26spage%3D284%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, J. W.</span><span> </span><span class="NLM_article-title">Consistent treatment of inter- and intramolecular polarization in molecular mechanics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1506</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.10127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=12395419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1497-1506&author=P.+Renauthor=J.+W.+Ponder&title=Consistent+treatment+of+inter-+and+intramolecular+polarization+in+molecular+mechanics+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Consistent treatment of inter- and intramolecular polarization in molecular mechanics calculations</span></div><div class="casAuthors">Ren, Pengyu; Ponder, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1497-1506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A protocol is described for the treatment of mol. polarization in force field calcns.  The resulting model is consistent in that both inter- and intramol. polarization are handled within a single scheme.  An anal. formula for removing intramol. polarization from a set of at. multipoles for an arbitrary static structure or conformation is given.  With the help of the intramol. polarization, these permanent at. multipoles can then be applied in modeling alternative conformations of a mol.  Equipped with this simple technique, one can derive transferable electrostatic parameters for peptides and proteins using flexible model compds. such as dipeptides.  The proposed procedure is tested for its ability to describe the electrostatic potential around various configurations of the N-methylacetamide dimer.  The effect of different intramol. polarization schemes on the accuracy of a force field model of the electrostatic potential of alanine dipeptide is investigated.  A group-based scheme for including direct intramol. polarization is shown to be most successful in accounting for the conformational dependence of electrostatic potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzG8zSIw6usrVg90H21EOLACvtfcHk0lj37UN29J3QSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rtr4%253D&md5=4f43548e95124d23ba655451cd332232</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10127%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DP.%26aulast%3DPonder%26aufirst%3DJ.%2BW.%26atitle%3DConsistent%2520treatment%2520of%2520inter-%2520and%2520intramolecular%2520polarization%2520in%2520molecular%2520mechanics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1497%26epage%3D1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span>GaussView version 5.0; <span class="NLM_publisher-name">Gaussian Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT 06492, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GaussView+version+5.0%3B+Gaussian+Inc.%3A+Wallingford%2C+CT+06492%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGaussView%2520version%25205.0%26pub%3DGaussian%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Clemente, F. R.; Vreven, T.; Frisch, M. J.</span><span> </span><span class="NLM_article-title">Getting the most out of ONIOM: guidelines and pitfalls</span>. In  <span class="citation_source-book">Quantum Biochemistry</span>; <span class="NLM_contrib-group">Matta, C. F.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2F9783527629213.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-83&author=F.+R.+Clemente&author=T.+Vreven&author=M.+J.+Frischauthor=C.+F.+Matta&title=Quantum+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2F9783527629213.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527629213.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DClemente%26aufirst%3DF.%2BR.%26atitle%3DGetting%2520the%2520most%2520out%2520of%2520ONIOM%253A%2520guidelines%2520and%2520pitfalls%26btitle%3DQuantum%2520Biochemistry%26aulast%3DMatta%26aufirst%3DC.%2BF.%26pub%3DWiley-VCH%26date%3D2010%26spage%3D61%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lhw6tT9ywjnGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.,  III; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Crowley, M.; Walker, R. C.; Zhang, W.; Merz, K. M.; Wang, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Kolossváry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Yang, L.; Tan, C.; Mongan, J.; Hornak, V.; Cui, G.; Mathews, D. H.; Seetin, M. G.; Sagui, C.; Babin, V.; Kollman, P. A.</span>Amber 10; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Fransisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=M.+Crowley&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Wang&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=L.+Yang&author=C.+Tan&author=J.+Mongan&author=V.+Hornak&author=G.+Cui&author=D.+H.+Mathews&author=M.+G.+Seetin&author=C.+Sagui&author=V.+Babin&author=P.+A.+Kollman&title=Amber+10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber%252010%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general Amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+Amber+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lhw6tT9ywjnGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520Amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lhw6tT9ywjnGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Ryckaert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the Cartesian equations of motion of a system with constraints: molecular dynamics of <i>n</i>-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+Cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0ljIOnMcIx9Oiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520Cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Hubbard, S. J.; Thornton, J. M.</span>NACCESS; <span class="NLM_publisher-name">Department of Biochemistry and Molecular Biology, University College London</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">1993</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=S.+J.+Hubbard&author=J.+M.+Thornton&title=NACCESS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DS.%2BJ.%26jtitle%3DNACCESS%26pub%3DDepartment%2520of%2520Biochemistry%2520and%2520Molecular%2520Biology%252C%2520University%2520College%2520London%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Lee, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, F. M.</span><span> </span><span class="NLM_article-title">The interpretation of protein structures: Estimation of static accessibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm201217q&amp;key=10.1016%2F0022-2836%2871%2990324-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm201217q&amp;key=5551392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm201217q&amp;key=1%3ACAS%3A528%3ADyaE3MXht1Shtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1971&pages=379-400&author=B.+Leeauthor=F.+M.+Richards&title=The+interpretation+of+protein+structures%3A+Estimation+of+static+accessibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretation of protein structures:  estimation of static accessibility</span></div><div class="casAuthors">Lee, Byungkook; Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-400</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A program is described for drawing the van der Waals surface of a protein mol.  An extension of the program permits the accessibility of atoms, or groups of atoms, to solvent or solute mols. of specified size to be quant. assessed.  Accessibility is proportional to surface area.  The accessibility of all atoms in the 20 common amino acids in model tripeptides of the type Ala-X-Ala are given for defined conformation.  The accessibilities are also given for all atoms in RNase S, lysozyme, and myogoblin.  Internal cavities are defined and discussed.  Of the surface area of each protein 40-50% is occupied by nonpolar atoms.  The actual numerical results are sensitive to the values chosen for the van der Waals radii of the various groups.  Since there is uncertainty over the correct values for these radii, the derived nos. should only be used as qual. guide at this stage.  The av. change in accessibility for the atoms of all 3 proteins in going from a hypothetical extended chain to the folded conformation of the native protein is about a factor of 3.  This no. applies to both polar (N and O) and nonpolar (C and S) atoms considered sep.  The larger nonpolar amino acids tend to be more buried in the native form of all 3 proteins.  However, for all classes and for residues within a given class, the accessibility changes on folding tend to be highly variable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobxHItjODI-LVg90H21EOLACvtfcHk0ljIOnMcIx9Oiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXht1Shtrk%253D&md5=05fe695f57006e8a7f782e6c5a8db122</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2871%2990324-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252871%252990324-X%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DB.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DThe%2520interpretation%2520of%2520protein%2520structures%253A%2520Estimation%2520of%2520static%2520accessibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1971%26volume%3D55%26spage%3D379%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGA','PDB','3KGA'); return false;">PDB: 3KGA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYJ','PDB','3FYJ'); return false;">PDB: 3FYJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P3G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P3G','PDB','2P3G'); return false;">PDB: 2P3G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYK','PDB','3FYK'); return false;">PDB: 3FYK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a href="/doi/suppl/10.1021/jm201217q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_77425"></div></div></div></div></div><hr /></hr><p class="last">Tables (XLS) summarizing the heavy-atom rmsd values between the docked poses (step 1) and between the time-averaged poses (steps 3/4) of tautomer/species for individual compounds, published standard deviations of inhibitory activities of pyrrolopyridines (Table <a class="ref showTableEvent internalNav" href="#tbl2">2</a>), fractions of free tautomers/species and prevalences of bound tautomers/species for benzothiophenes (Table <a class="ref showTableEvent internalNav" href="#tbl1">1</a>) and pyrrolopyridines (Table <a class="ref showTableEvent internalNav" href="#tbl2">2</a>), respectively. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm201217q/suppl_file/jm201217q_si_001">jm201217q_si_001 (98.0 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm201217q&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Fjm201217q%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-5" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm201217q" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a2cc17b1a3c88","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
