
<html lang="en"     class="pb-page"  data-request-id="1476d83a-8c8b-4be4-8286-b302c95b58d9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.8b01960;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-6;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design of N-(5-Phenylthiazol-2-yl)acrylamides as Novel and Potent Glutathione S-Transferase Omega 1 Inhibitors" /></meta><meta name="dc.Creator" content="Weiyang  Dai" /></meta><meta name="dc.Creator" content="Soma  Samanta" /></meta><meta name="dc.Creator" content="Ding  Xue" /></meta><meta name="dc.Creator" content="Elyse M.  Petrunak" /></meta><meta name="dc.Creator" content="Jeanne A.  Stuckey" /></meta><meta name="dc.Creator" content="Yanyan  Han" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Yong  Wu" /></meta><meta name="dc.Creator" content="Nouri  Neamati" /></meta><meta name="dc.Description" content="Using reported glutathione S-transferase omega 1 (GSTO1-1) cocrystal structures, we designed and synthesized acrylamide-containing compounds that covalently bind to Cys32 on the catalytic site. Sta..." /></meta><meta name="Description" content="Using reported glutathione S-transferase omega 1 (GSTO1-1) cocrystal structures, we designed and synthesized acrylamide-containing compounds that covalently bind to Cys32 on the catalytic site. Sta..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 8, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01960" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01960" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01960" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01960" /></link>
        
    
    

<title>Structure-Based Design of N-(5-Phenylthiazol-2-yl)acrylamides as Novel and Potent Glutathione S-Transferase Omega 1 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01960" /></meta><meta property="og:title" content="Structure-Based Design of N-(5-Phenylthiazol-2-yl)acrylamides as Novel and Potent Glutathione S-Transferase Omega 1 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0023.jpeg" /></meta><meta property="og:description" content="Using reported glutathione S-transferase omega 1 (GSTO1-1) cocrystal structures, we designed and synthesized acrylamide-containing compounds that covalently bind to Cys32 on the catalytic site. Starting from a thiazole derivative 10 (GSTO1-1 IC50 = 0.6 μM), compound 18 was synthesized and cocrystallized with GSTO1. Modification on the amide moiety of hit compound 10 significantly increased the GSTO1-1 inhibitory potency. We solved the cocrystal structures of new derivatives, 37 and 44, bearing an amide side chain bound to GSTO1. These new structures showed a reorientation of the phenyl thiazole core of inhibitors, 37 and 44, when compared to 18. Guided by the cocrystal structure of GSTO1:44, analogue 49 was designed, resulting in the most potent GSTO1-1 inhibitor (IC50 = 0.22 ± 0.02 nM) known to date. We believe that our data will form the basis for future studies of developing GSTO1-1 as a new drug target for cancer therapy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01960"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01960">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01960&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01960&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01960&amp;href=/doi/10.1021/acs.jmedchem.8b01960" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3068-3087</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01946" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01971" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of <i>N</i>-(5-Phenylthiazol-2-yl)acrylamides as Novel and Potent Glutathione S-Transferase Omega 1 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Weiyang Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weiyang Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div><div class="loa-info-affiliations-info">Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, No. 17 People’s South Road, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weiyang++Dai">Weiyang Dai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3222-4661" title="Orcid link">http://orcid.org/0000-0002-3222-4661</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Soma Samanta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Soma Samanta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Soma++Samanta">Soma Samanta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ding Xue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ding Xue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ding++Xue">Ding Xue</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elyse M. Petrunak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elyse M. Petrunak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute and Department of Biological Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elyse+M.++Petrunak">Elyse M. Petrunak</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeanne A. Stuckey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanne A. Stuckey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute and Department of Biological Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne+A.++Stuckey">Jeanne A. Stuckey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanyan Han</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanyan Han</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Sciences, College of Pharmacy, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanyan++Han">Yanyan Han</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Duxin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duxin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Sciences, College of Pharmacy, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yong Wu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, No. 17 People’s South Road, Chengdu 610041, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#36414f5958517645554318535243185558"><span class="__cf_email__" data-cfemail="1e69677170795e6d7d6b307b7a6b307d70">[email protected]</span></a> (Y.W.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Wu">Yong Wu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0719-4963" title="Orcid link">http://orcid.org/0000-0003-0719-4963</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nouri Neamati</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nouri Neamati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, University of Michigan, North Campus Research Complex, 1600 Huron Parkway, Ann Arbor 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4c22292d212d38250c212928623921252f2462292839"><span class="__cf_email__" data-cfemail="640a01050905100d240901004a11090d070c4a010011">[email protected]</span></a> (N.N.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nouri++Neamati">Nouri Neamati</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01960&amp;href=/doi/10.1021%2Facs.jmedchem.8b01960" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3068–3087</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 8, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 December 2018</li><li><span class="item_label"><b>Published</b> online</span>8 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01960" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01960</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3068%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DWeiyang%2BDai%252C%2BSoma%2BSamanta%252C%2BDing%2BXue%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D6%26contentID%3Dacs.jmedchem.8b01960%26title%3DStructure-Based%2BDesign%2Bof%2BN-%25285-Phenylthiazol-2-yl%2529acrylamides%2Bas%2BNovel%2Band%2BPotent%2BGlutathione%2BS-Transferase%2BOmega%2B1%2BInhibitors%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3087%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01960"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2367</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01960" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of N-(5-Phenylthiazol-2-yl)acrylamides as Novel and Potent Glutathione S-Transferase Omega 1 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Weiyang&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Soma&quot;,&quot;last_name&quot;:&quot;Samanta&quot;},{&quot;first_name&quot;:&quot;Ding&quot;,&quot;last_name&quot;:&quot;Xue&quot;},{&quot;first_name&quot;:&quot;Elyse&quot;,&quot;last_name&quot;:&quot;M. Petrunak&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;A. Stuckey&quot;},{&quot;first_name&quot;:&quot;Yanyan&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Nouri&quot;,&quot;last_name&quot;:&quot;Neamati&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;3068-3087&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01960&quot;},&quot;abstract&quot;:&quot;Using reported glutathione S-transferase omega 1 (GSTO1-1) cocrystal structures, we designed and synthesized acrylamide-containing compounds that covalently bind to Cys32 on the catalytic site. Starting from a thiazole derivative 10 (GSTO1-1 IC50 = 0.6 μM), compound 18 was synthesized and cocrystallized with GSTO1. Modification on the amide moiety of hit compound 10 significantly increased the GSTO1-1 inhibitory potency. We solved the cocrystal structures of new derivatives, 37 and 44, bearing an amide side chain bound to GSTO1. These new structures showed a reorientation of the phenyl thiazole core of inhibitors, 37 and 44, when compared to 18. Guided by the cocrystal structure of GSTO1:44, analogue 49 was designed, resulting in the most potent GSTO1-1 inhibitor (IC50 = 0.22 ± 0.02 nM) known to date. We believe that our data will form the basis for future studies of developing GSTO1-1 as a new drug target for cancer therapy.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01960&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01960" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01960&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01960" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01960&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01960" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01960&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01960&amp;href=/doi/10.1021/acs.jmedchem.8b01960" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01960" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01960" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01960%26sid%3Dliteratum%253Aachs%26pmid%3D30735370%26genre%3Darticle%26aulast%3DDai%26date%3D2019%26atitle%3DStructure-Based%2BDesign%2Bof%2BN-%25285-Phenylthiazol-2-yl%2529acrylamides%2Bas%2BNovel%2Band%2BPotent%2BGlutathione%2BS-Transferase%2BOmega%2B1%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D6%26spage%3D3068%26epage%3D3087%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/jmcmar.2019.62.issue-6/20190328/jmcmar.2019.62.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Using reported glutathione S-transferase omega 1 (GSTO1-1) cocrystal structures, we designed and synthesized acrylamide-containing compounds that covalently bind to Cys32 on the catalytic site. Starting from a thiazole derivative <b>10</b> (GSTO1-1 IC<sub>50</sub> = 0.6 μM), compound <b>18</b> was synthesized and cocrystallized with GSTO1. Modification on the amide moiety of hit compound <b>10</b> significantly increased the GSTO1-1 inhibitory potency. We solved the cocrystal structures of new derivatives, <b>37</b> and <b>44</b>, bearing an amide side chain bound to GSTO1. These new structures showed a reorientation of the phenyl thiazole core of inhibitors, <b>37</b> and <b>44</b>, when compared to <b>18</b>. Guided by the cocrystal structure of GSTO1:<b>44</b>, analogue <b>49</b> was designed, resulting in the most potent GSTO1-1 inhibitor (IC<sub>50</sub> = 0.22 ± 0.02 nM) known to date. We believe that our data will form the basis for future studies of developing GSTO1-1 as a new drug target for cancer therapy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Glutathione S-transferases (GSTs) are a diverse family of cytosolic, mitochondrial, and microsomal enzymes that conjugate glutathione to a variety of exogenous compounds including drugs, industrial intermediates, pesticides, herbicides, environmental pollutants, and carcinogens, which mainly act as part of an integrated defense strategy.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> In addition, GSTs also play an important role in biotransformation of endogenous substrates and other biological functions.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The mammalian GSTs are classified into seven different classes, alpha, mu, pi, theta, sigma, zeta, and omega. Among them, the omega GSTs (GSTOs) belong to an atypical cytosolic class and two isozymes, GSTO1-1 and GSTO2-2, have been identified in humans that share low sequence identity with other GST classes.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> GSTO1-1 possesses S-phenacyl glutathione reductase and monomethylarsonate reductase activity, whereas GSTO2-2 catalyzes dehydroascorbate reduction.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a><i>S</i>-(4-Nitrophenacyl)glutathione (4-NPG) is also reported as a good substrate for GSTO1-1.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> They are also unique among GSTs in terms of their catalytic amino acid residues. Rather than a catalytic serine or tyrosine, GSTOs possess a reactive cysteine nucleophile in the active site.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Therefore, their enzymatic activities are sensitive to molecules that are able to conjugate with the thiol group, such as <i>N</i>-ethylmaleimides or haloacetamides.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> The crystal structure of GSTO1 revealed two important active sites (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EEM">1EEM</a>).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> One is the hydrophobic or electrophile binding site (H-site) that can easily accommodate lipophilic electrophiles and the other is a narrow strip consisting of a series of hydrophilic residues that can selectively bind with glutathione (G-site).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">GSTO1-1 is overexpressed in select human cancers such as pancreatic cancer,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> head and neck cancer,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> urinary bladder cancer,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and colorectal cancer,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which leads to an accelerated detoxification of drug substrates and thus is implicated in drug resistance.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a> We previously reported that silencing GSTO1-1 with siRNA impairs cancer cell viability, suggesting that GSTO1-1 is a potential target for cancer therapy.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> It is also reported that GSTO1-1 plays a proinflammatory role in models of inflammation, colitis, and obesity.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Only a limited number of GSTO1-1 inhibitors have been reported, and there is a strong correlation between thiol-alkylating groups in these compounds and their GSTO1-1 inhibitory activities (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The epoxide-bearing cytokine release inhibitory drug (CRID) has affinity toward GSTO1-1, which may be related to their inhibition of ATP-induced IL-1β posttranslational processing.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Previous screen of the Molecular Libraries Small Molecule Repository (MLSMR) library uncovered a class of selective α-chloroacetamide GSTO1-1 inhibitors. After further optimization, KT53 was selected as a potent GSTO1-1 inhibitor that enhanced cisplatin-induced cytotoxicity.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, KT53 showed instability in phosphate-buffered saline (PBS) within 6 h at room temperature because of its intramolecular cyclization reaction. A GSTO1 cocrystal with ML175 was subsequently reported (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V3Q">5V3Q</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Additionally, an aryl fluorosulfate compound was identified as a GSTO1-1 probe by an “inverse drug discovery” strategy and its X-ray cocrystal structure was also solved (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEH">5UEH</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Previously, we developed a series of chloroacetamide-containing GSTO1-1 inhibitors by high-throughput screening of our in-house compound library.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The lead compound C1-27 was discovered as a low-molecular-weight chloroacetamide derivative with good potency. The cocrystal structure of the GSTO1-C1-27 complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM">4YQM</a>) was also obtained, showing C1-27 binding in the H-site of GSTO1. C1-27 exhibited cytotoxicity in several cancer cell lines, but knockdown of GSTO1-1 did not significantly affect C1-27 cytotoxicity, suggesting that C1-27 has off-target effects at higher concentrations than those required for GSTO1-1 inhibition. All of GSTO1-1 inhibitors discussed above display similar structural features to match the GSTO1 protein structure. Their hydrophobic cores (usually electron-deficient aromatic rings) allow them to enter the H-site easily. Most importantly, they all contain electrophilic reactive groups (epoxide, chloroacetamide, benzyl chloride, aryl fluorosulfate) that can covalently bind with the Cys32 thiol group of GSTO1. We noticed that α,β-unsaturated carbonyls, such as acrylamides, acrylic esters, and chalcones, can also be served as covalent modifiers. Drugs or compounds with α,β-unsaturated carbonyls are mostly designed to undergo an irreversible hetero-Michael addition reaction with a specific residue (typically, unsubstituted cysteines) of the target protein. Compared with other covalent binding groups, α,β-unsaturated carbonyls, especially acrylamides, are considered to be mild and present in select Food Drug and Administration-approved drugs.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> Previous reports showed that a series of artemisinin derivatives containing 1,2,4-trioxane acrylates displayed GSTP1 inhibition activity, indicating that α,β-unsaturated carbonyl modifiers might be applicable for GST inhibitors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Currently, there is no reported GSTO1-1 inhibitor containing an α,β-unsaturated carbonyl moiety. Herein, we describe a structure-guided approach of developing novel GSTO1-1 inhibitors based on our previously published GSTO1 crystal structures.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of reported GSTO1-1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">The synthesis of compounds <b>1–12</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Compounds <b>1–3</b> were synthesized from a C1-27 intermediate 5-amino-2-chloro-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>4i</b> conjugated with acryloyl chloride, 2-chloroethylsulfonyl chloride, and methyl 4-bromocrotonate, respectively, as described in our previous study.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Commercially available heterocycle amines were treated with acryloyl chloride in dichloromethane (DCM) to give <b>5–12</b>. <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> describes the synthesis of compounds <b>16–27</b> with different substituents. Bromination of commercially available acetophenones <b>13a–13l</b> using copper bromide in ethyl acetate gave 2-bromoacetophenones <b>14a–14l</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Treatment of 2-bromoacetophenones with thiourea in ethanol yielded 4-phenyl-2-aminothiazole derivatives <b>15a–15l</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The amines were then conjugated with acryloyl chloride to give compounds <b>16–27</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1–12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acryloyl chloride, DCM, DIEA, rt, 2 h; (b) 2-chloroethylsulfonyl chloride, DCM, DIEA, rt, 2 h; (c) methyl (<i>E</i>)-4-bromocronate, potassium fluoride, CH<sub>3</sub>CN, 100 °C, 4 h.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Thiazole Acrylamide Analogues <b>16–27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CuBr<sub>2</sub>, EtOAc, reflux, 3 h; (b) thiourea, EtOH, reflux, 2 h; (c) acryloyl chloride, DCM, DIEA, rt, 2 h.</p></p></figure><div class="NLM_p">Compounds <b>29–34</b> were synthesized as described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> to investigate the influence of installing substituents on the thiazole ring and the amide nitrogen. Compounds with 5-substituents on the thiazole ring <b>29</b> and <b>30</b> were obtained by treating commercially available α-bromoketones <b>14o</b> and <b>14p</b> with thiourea, followed by coupling with acryloyl chloride. Compounds with N-substitutions on the acrylamide moiety were synthesized under two different conditions. Intermediates <b>28c</b> and <b>28d</b> were obtained by <b>4f</b> and corresponding bromides. N-substituted 4-phenylthiazol-2-amines <b>28c–28d</b> were heated with acryloyl chloride in toluene to give <b>33–34</b>. The total yield of <b>33–34</b> was relatively low because the bromination process resulted in bisubstituted side products. In order to overcome this limitation and generate more derivatives with an amide side chain efficiently, we developed another synthetic route for compounds <b>37–45</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The commercially available starting material 2-bromoacetophenone <b>14m</b> was treated with potassium thiocyanate followed by primary amines to give the intermediate <b>28e–28h</b>.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Acid hydrolysis of methyl ester intermediate <b>28e</b> provided the free carboxylic acid <b>35</b>, which was then conjugated with amines to give amides <b>36a–36d</b>. Compounds <b>28e–28h</b> and <b>36a–36d</b> were treated with acryloyl chloride to give <b>37–45</b>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Thiazole Acrylamide Analogues <b>29–34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) corresponding N-substituted thiourea, EtOH, reflux, 2 h; (b) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux, 8 h; (c) acryloyl chloride, DCM, rt, 2 h; (d) acryloyl chloride, toluene, 70 °C, 0.5 h.</p></p></figure><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Thiozale Acrylamide Analogues <b>37–45</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, EtOH, reflux, 3 h; (b) corresponding primary amines, EtOH, reflux, overnight; (c) HCl, THF/H<sub>2</sub>O, reflux, 2 h; (d) EDC, HOBt, DIEA, DMF, rt, overnight; (e) acryloyl chloride, toluene, 70 °C, 0.5 h; (f) TFA, DCM, rt, 2 h.</p></p></figure><div class="NLM_p">Compounds <b>47–52</b> were synthesized as described in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Hydroxyl protection of <b>13p</b> and <b>13q</b> gave intermediate <b>13r</b> and <b>13s</b>. Bromination and cyclization processes were similar with the protocols presented in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> to give <b>28i–28l</b> from <b>14e</b>, <b>14r</b>, <b>14s</b>, and <b>14l</b>, respectively. Compound <b>46c</b> was obtained by conjugating <b>28l</b> with 2-chloroacetyl chloride. Methyl ether deprotection of <b>46a–46c</b> and <b>48</b> gave the hydroxyl derivatives <b>49–52</b>.</div><figure id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Thiazole Acrylamide Analogues <b>49–52</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, acetone, reflux, 8 h; (b) CuBr<sub>2</sub>, EtOAc, reflux, 3 h; (c) KSCN, EtOH, reflux, 3 h; (d) (3-methylisoxazol-5-yl)methanamine, EtOH, reflux, 32 h; (d) acryloyl chloride, toluene, 70 °C, 0.5 h; (e) 2-chloroacetyl chloride, DIEA, toluene, 70 °C, 1 h; (f) BBr<sub>3</sub>, DCM, −78 °C to rt, 0.5–2 h, THF, TEA, reflux, 1 h.</p></p></figure><div class="NLM_p">The synthesis of compound <b>53</b> started with <i>tert</i>-butyl (4-bromothiophen-2-yl)carbamate <b>54</b> conjugated with 5-(bromomethyl)-3-methylisoxazole in the presence of NaH (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). The Boc group of intermediate <b>55</b> was cleaved before conjugating with acryloyl chloride. A Suzuki coupling with (2-hydroxyphenyl)boronic acid offered the final product <b>53</b>.</div><figure id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 5-(bromomethyl)-3-methylisoxazole, NaH, DMF, 0 °C to rt, 2 h; (b) TFA, DCM, rt, 2 h; (c) acryloyl chloride, toluene, 70 °C, 0.5 h; (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 80 °C, 8 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Design and SAR Studies</h3><div class="NLM_p">GSTO1-1 is considered a relatively new target for early stage drug discovery. Various assays can be used to identify GSTO1 inhibitors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> We used several assays to evaluate the GSTO1-1 inhibitor C1-27, including the 4-NPG substrate assay and in-gel competitive fluorescence assay. Because C1-27 and all other GSTO1-1 inhibitors bind with GSTO1-1 on the catalytic active site residues, we tested the GSTO1-1 inhibitory activity of all compounds by a competitive fluorescence assay. Using 5-chloromethylfluorescein diacetate (CMFDA), which has been reported to be a potent and irreversible GSTO1-1 inhibitor (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> we optimized the gel-based binding assay that efficiently measures the competitive inhibition of CMFDA binding to GSTO1; a representative example of this assay is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S1</a>. Recombinant GSTO1-1 was used to reduce assay interference caused by limitation of gel separation. It is of interest to note that for irreversible covalent inhibitor ideal measurement is Kinact because it considers the rate of inactivation and the binding affinity.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The limitation of our assay is that it cannot measure the rate of inactivation and the binding affinity. Thus, in this study, we determined % inhibition of the binding of CMFDA to GSTO1 by the compound, where compounds are preincubated with GSTO1 for 30 min, followed by addition of CMFDA for 30 min. Preincubation at various time points did not show notable difference in the IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S2</a>). In this study, we determined IC<sub>50</sub> for all compounds preincubated for 30 min.</div><div class="NLM_p">Initially, we modified the chloroacetamide group of C1-27 with other electrophilic covalent binding groups. Its replacement with an acrylamide or a vinyl sulfonamide moiety caused a significant decrease in either GSTO1-1 potency or cytotoxicity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). By analyzing the cocrystal structure of C1-27 and GSTO1-1, we observed an important H-bond interaction between the sulfonamide moiety and Trp180 that restricts compound’s conformation in the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The chloroacetamide moiety and acrylamide moiety have similar molecular shapes but different binding positions. The cysteine thiol group binds to the alpha-carbonyl carbon atom of C1-27 but with the beta-carbonyl carbon of the acrylamide analogue <b>1</b>. When we superimposed these two structures, the distance between the two active sites was measured as approximately 1.34 Å (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Direct replacement of chloroacetamide on C1-27 with an acrylamide moiety might result in the loss of H-bond interaction because of the different length of the covalent binding groups. We further synthesized methyl crotonate derivative <b>3</b> where the distance from the sulfonamide group to the covalent binding site was retained. We expected that the ester moiety of <b>3</b> could interact with any residues on the G-site of GSTO1. However, this modification resulted in a loss of activity, perhaps due to the bulky methyl crotonate motif decreasing the chance of Michael acceptor reaction with the thiol group of Cys32. The unsatisfying result indicated that the C1-27 core was not suitable for acrylamide replacement, and new cores or scaffolds needed to be developed.</div><figure id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Cocrystal structure of GSTO1:C1-27 (pink stick, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM">4YQM</a>). (B) Structural differences of <b>1</b> (cyan) and C1-27 (orange) under nucleophilic attack. 3D structures were generated, overlapped, and measured in Maestro. (C) Compound <b>5</b> (cyan) displayed similar binding pose with C4-10 (orange) at the GSTO1 H-site in a docking study perform by Glide. (D) Alignment of <b>5</b> (yellow green) and <b>10</b> (pink) showed the advantage of the thiazole core structure, and 3D structures were generated and overlapped in Maestro.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. GSTO1-1 Binding Assay Activity of C1-27 Analogues <b>1–3</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0012.gif" alt="" id="GRAPHIC-d7e828-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values are the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div><div class="NLM_p">We previously built a chloroacetamide-containing compound library to screen potential GSTO1-1 inhibitors.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> On the basis of this library, a GSTO1 covalent docking study was conducted to search for novel inhibitors. The chloroacetamide moieties of active compounds were replaced with arylamide, and the new library generated was docked on the GSTO1 active site. Taking into consideration the difference in physical and chemical properties between chloroacetamide and acrylamide, we selected C4-10, a phenylpyrazole derivative, that showed modest potency on the GSTO1-1 binding assay in our previous study as a model compound (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and the cocrystal structure of C4-10 with GSTO1 was also solved (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV">4YQV</a>).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> We observed that its acrylamide analogue <b>5</b> exhibited a very similar pose with C4-10 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) in our docking study. The acrylamide moiety was bound to Cys32, the <i>N</i>-phenylpyrazole moiety occupied the H-site, and the benzene ring was located deep into the hydrophobic pocket. Additionally, some other phenyl heterocyclic derivatives were docked onto the GSTO1 active site and showed similar binding poses. Thus, a series of analogues were synthesized and investigated in the GSTO1-1 binding assay to verify our docking results. The compounds with the pyrazole core were inactive (<b>5</b> and <b>6</b>). Compounds <b>7</b> and <b>8</b> with a nonaromatic piperidine core also did not possess significant GSTO1-1 binding activity. However, compounds with a thiazole core displayed potency and 4-phenyl (<b>10</b>) was more potent than 5-phenyl derivatives (<b>11</b>). Most of these heterocyclic compounds are rigid molecules, and their binding conformation and low energy conformation are likely to be quite similar. Therefore, a three-dimensional (3D) structure superimposition was conducted to analyze the structure–activity relationship (SAR). We observed that the benzene ring of the thiazole compounds displayed a shift compared with the pyrazole compound <b>5</b>, while other motifs were perfectly aligned (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D), which is probably due to the unique sulfur–carbon bond angle in the thiazole ring (88.6°).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This special angle allows the benzene ring to insert into the narrow hydrophobic cleft to improve potency. Two other compounds with a thiophene core structure (<b>11</b> and <b>12</b>) also showed good potencies, suggesting that the sulfur atom in the heterocyclic core displays positive contribution to potency. Thiophene analogue <b>11</b> showed slightly decreased potency as compared to <b>10</b>, perhaps due to the conformational restriction imposed by its intramolecular H-bond. Endogenous GSTO1-1 binding activity was also observed on <b>9</b> and <b>10</b> in a cell-based assay. Compound <b>10</b> exhibits most desirable ligand efficiency (LE)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and ligand-lipophilicity efficiency (LLE)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> among these analogues. Therefore, we chose compound <b>10</b> as a new GSTO1-1 inhibitor template for further studies and optimization. Compound <b>10</b> also displayed cytotoxicity on HCT116 and HT29 cell lines, but whether the cytotoxicity was resulted from GSTO1-1 inhibition needs further investigation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. GSTO1-1 Binding Assay Activity, Calculated Molecule Properties, and Cytotoxicity of Phenyl Heterocyclic Arylamides <b>5–12</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0014.gif" alt="" id="GRAPHIC-d7e945-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0015.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values represent the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">clog <i>P</i> values were calculated using ChemBioDraw Professional 16.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Calculated LE = −1.4 log IC<sub>50</sub>/<i>N</i>, where <i>N</i> is the number of nonhydrogen atom. LE > 0.25 means a good hit.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Calculated LLE = pIC<sub>50</sub> (M) – clog <i>P</i>. LLE > 3 means a good hit.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div><div class="NLM_p">The binding pose of compound <b>10</b> in our docking study showed that the benzene ring does not fully occupy the H-site and additional lipophilic groups can be accommodated. We investigated whether modifications on the phenyl group can improve potency. All compounds were tested in the GSTO1-1 binding assay and 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) cytotoxicity assay. Because the major interaction between GSTO1 and our compounds is hydrophobic interaction, the LLE of these compounds was also investigated. Substitution of the benzene ring with chlorine atoms showed a preference at 4-position (<b>18</b>, IC<sub>50</sub> = 0.17 μM) over other positions. This may be due to the fact that 4-chloro can fill additional space in the hydrophobic pocket. The classic bioisosteric replacement of chlorine with the methyl group resulted in a slightly decreased potency. We hypothesized that the activity change was related to the different electron inductive effects between chlorine and the methyl group on the benzene ring. When bound to GSTO1, the benzene ring located in the hydrophobic pocket is surrounded by Trp180, Phe31, Phe225, and Tyr229. These residues can form π–π interaction with the benzene ring, and the interaction intensity is significantly influenced by the electron density of the aromatic ring. In order to test the feasibility of this hypothesis, additional analogues with substitutions at 4-position were synthesized. As anticipated, analogues with strong electron-withdrawing groups (e.g., trifluoromethyl analogue <b>20</b>, IC<sub>50</sub> = 0.30 μM) displayed improved potency, whereas analogues with 2,3-dioxole, a strong electron-donating substituent, showed decreased potencies (IC<sub>50</sub> = 1.22 μM).</div><div class="NLM_p">To validate our design strategy and to guide future analogue design, we obtained the cocrystal structure of <b>18</b> bound to GSTO1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB">6MHB</a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The 2.75 Å structure of GSTO1:<b>18</b> demonstrates similarity to the cocrystal structure of GSTO1 with a chloroacetamide inhibitor C4-10 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV">4YQV</a>)<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> in both the overall structure of GSTO1 [an average root mean square deviation (RMSD) between two structures of 0.49 Å] as well as the orientation of the ligand. Compound <b>18</b> occupies the same deep hydrophobic cleft of the H-site aligned by aromatic residues including Trp180, Phe31, Phe225, and Tyr229 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The 4-chlorophenyl moiety makes van der Waals contacts with the side chains of Val127, Arg183, Met187, Trp222, Phe225, and Leu226 as well as the backbone amide of Phe31 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The propionamide and thiazole ring make hydrophobic contacts with Met29, Pro33, Phe34, and Tyr229. Although Tyr229 appears to interact with the thiazole, its orientation varies between the six chains of GSTO1 modeled as part of this structure, and a lack of electron density in some chains disallows modeling of the Tyr229 side chain altogether. In conjunction with the poor electron density of the thiazole ring, this seems to suggest that binding of <b>18</b> is likely driven by interactions of the 4-chlorophenyl moiety with GSTO1. The most significant difference between GSTO1:<b>18</b> and GSTO1:C4-10 is a change in conformation of the Leu226 side chain, which reorients to interact with the chlorine of the 4-chlorophenyl substituent of <b>18</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB">6MHB</a>, cyan ribbons:teal sticks). (A) Hydrophobic interactions of <b>18</b> within the H-site of GSTO1. (B) Structural comparison of GSTO1:<b>18</b> and GSTO1:C4-10 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV">4YQV</a>, gray ribbons:black sticks) demonstrates that these inhibitors bind with similar orientations within a hydrophobic cavity (gray surface) of the GSTO1 H-site. Average RMSD = 0.49 Å</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. GSTO1-1 Binding Assay, clog <i>P</i>, LLE, and Cytotoxicity of Phenyl Thiazole Arylamides <b>16–27</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0016.gif" alt="" id="GRAPHIC-d7e1098-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0017.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values represent the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">clog <i>P</i> values were calculated using ChemBioDraw Professional 16.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Calculated LLE = pIC<sub>50</sub> (M) – clog <i>P</i>.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div><div class="NLM_p">Interestingly, we found that the benzene ring was close to the Phe31 backbone amide (3.50 Å), which could be treated as a potential binding site. Therefore, we generated 2-hydroxyl analogue <b>24</b>. The potency was improved although not that significantly. Meanwhile, the other 2-position substitutions or 4-hydroxyl substitution led to a potency reduction. The 2-methoxyl analogue <b>26</b> was inactive. We speculated that the 2-hydroxyl group probably formed an H-bond with the backbone oxygen of Phe31, but adding ortho-position substituents altered the binding energy and offset the potency gained from the H-bonding. We also tested the cytotoxicity of these compounds in the MTT assay. Most of these analogues showed inhibition of cell viability, but their cytotoxicities did not show an obvious correlation with their GSTO1 binding potency.</div><div class="NLM_p">In order to extend our SAR study and improve potency, we analyzed the cocrystal structure of <b>18</b> bound to GSTO1 looking for other potential interaction sites. Two regions on the binding pocket can be identified as exploitable sites. One is a hydrophobic area near the “T cave” surrounded by aromatic ring residues such as Tyr229, Phe31, and Phe225. The other one is Trp180 close to the thiazole ring (distance = 5.46 Å), which plays an important role as an H-bond donor when bound with C1-27. Therefore, we designed analogues with substituents on the 5-position of the thiazole ring or adding side chain on amide nitrogen.</div><div class="NLM_p">Inserting a methyl on the 5-position of the thiazole ring led to a potency reduction (<b>29</b>, 3.1 ± 1.5 μM), whereas methyl 5-carboxylate analogue <b>30</b> exhibited slightly increased potency. We assumed that 5-substitutions generally affected the low energy conformation, which accounted for the potency decrease for <b>29</b>, whereas the carboxylate group in <b>30</b> improved the binding free energy by forming another H-bond with Trp180. Compounds <b>33</b> and <b>34</b> displayed increased potencies, suggesting that more area occupied in the H-site helps exclude “unfavorable water molecules” and contribute to potency. Therefore, additional compounds with amide side chains were synthesized to improve potency based on this theory. Because ethyl acetate derivative <b>34</b> showed improved potency, we further examined if its homologues <b>37</b>, <b>38</b>, and <b>39</b> could be accommodated within the cavity. A bulky tert-butyl group can be tolerated, whereas the free carboxylic acid replacement resulted in a significant activity loss, suggesting that lipophilic groups may be beneficial to improve potency as we expected. Additional derivatives were synthesized to investigate the tolerance of other isosteres. Replacement of the methyl ester group with more lipophilic cyclohexyl amides led to increased potency, which also validated our hypothesis. Considering that there are still polar residues around the molecule, we sought to design amide derivatives with potential H-bond acceptors. Compounds with a terminal oxygen, including <b>43</b> with a bulky N-(2-morpholinoethyl) group, exhibited moderate to good potencies. Compound <b>42</b> displayed the best potency up to this stage (IC<sub>50</sub> = 0.03 ± 0.02 μM). All analogues showed improved LLE, suggesting that our optimization was reasonable. In addition, we also synthesized compounds based on <b>33</b>. We hypothesized that the additional benzyl group forms edge-to-face π-stacking with Tyr229. However, the benzyl group installation made the compound greasy (clog <i>P</i> = 4.48). In order to reduce lipophilicity, the pyridine derivative was synthesized. Unfortunately, compound <b>45</b> showed decreased potency probably due to the electron-deficient pyridine ring impairing the π-interaction. Hence, we designed a molecule with the 3-methyl-isoxazole group, which was regarded as a preferred druglike and electron-rich heterocycle group. This modification resulted in fairly good activity (<b>44</b>, IC<sub>50</sub> = 0.051 μM) and improved clog <i>P</i> (2.96). When the potency of the GSTO1-1 binding activity improved, these derivatives, however, lost their cytotoxicity, indicating that inhibiting GSTO1-1 might not influence cell viability.</div><div class="NLM_p">The cocrystal structures of GSTO1 with <b>37</b> and <b>44</b> were solved to resolutions of 2.0 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC">6MHC</a>) and 2.15 Å (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>), respectively. The two structures reveal very little difference in the overall structure of GSTO1 (average RMSD = 0.33 Å) when compared to one another but demonstrate more substantial differences when compared with the structure of GSTO:<b>18</b> (average RMSD = 0.78 Å for GSTO1:<b>37</b> and 0.80 Å for GSTO1:<b>44</b>) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In particular, the structures of GSTO1 with <b>37</b> and <b>44</b> show a rearrangement of the H-site as a result of a shift in the α9 helix when compared to the GSTO1:<b>18</b> structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In this state, the side chain of Trp222 is reoriented to occupy the cavity that accommodates the 4-chlorophenyl ring of <b>18</b>. In both structures, the electron density of inhibitors <b>37</b> and <b>44</b> suggests modeling the thiazole ring such that it engages in π–π stacking interactions with the side chain of Tyr229, which appears well ordered in all chains of both structures, and to a lesser extent Phe225 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The electron density associated with the phenyl ring of both inhibitors is less definitive, suggesting that there is likely a degree of free rotation about the bond connecting the two ring systems. The phenyl substituent is poised to make weak hydrophobic contacts with α4 helix residues Gly128, Ile131, Arg132, as well as Leu226 and Tyr229, which form a shallow pocket in the H-site. In the GSTO1:<b>37</b> structure, the electron density did not support modeling the ester, which is projected into bulk solvent and consequently disordered (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In contrast, an interaction with the side chain of Phe34 orders the isoxazole ring of <b>44</b>, which extends this portion of the ligand toward the G-site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). This additional interaction likely contributes to the improvement in potency of <b>44</b> when compared to <b>37</b> because it may block glutathione to enter the G-site. This discovery led us to further optimization by adding more flexible substituents attached to functional groups to interact with residues at the G-site that may further improve potency (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><figure id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural comparison of GSTO1:<b>37</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC">6MHC</a>, <b>37</b> is shown in pink sticks), GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>, <b>44</b> is shown in green sticks) and GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHA">6MHA</a>, <b>18</b> is shown in teal sticks). (A) Structural differences are a result of the N-substituted inhibitors <b>37</b> (magenta sticks) and <b>44</b> (dark green sticks) adopting a different orientation within the GSTO1 H-site compared to <b>18</b> (teal sticks). Trp222 makes pi-stacking with the chlorophenyl group of <b>18</b> but has no contact with the phenyl groups of <b>37</b> and <b>44</b>. (B) Hydrophobic interactions of GSTO1 with inhibitors <b>37</b> and <b>44</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) N-substitution of thiazole acrylamide inhibitors, as demonstrated by the cocrystal structure of GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>, dark green sticks), may provide a handle to access hydrogen-bonding or water-mediated interactions with residues in the glutathione binding site (glutathione from PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EEM">1EEM</a> shown in black sticks). Residues that interact with both glutathione and <b>44</b> are highlighted in light blue, whereas sites of potential enthalpic interactions with extended N-substituted thiazole inhibitors are highlighted in yellow. (B) 2-Position of the phenyl ring of inhibitor <b>44</b> is directed into a pocket with a strong overall positive charge. The 2-hydroxyl substituent of <b>49</b>, bearing a partial negative charge, is likely stabilized through electrostatic interactions with the inhibitor binding site. Electrostatic maps were generated using APBS from −5.0 eV (red) to +5.0 eV (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. GSTO1-1 Binding Assay, clog <i>P</i>, LLE, and Cytotoxicity of Thiazole Arylamides <b>37–45</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0018.gif" alt="" id="GRAPHIC-d7e1390-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0019.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values represent the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">clog <i>P</i> values were calculated using ChemBioDraw Professional 16.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Calculated LLE = pIC<sub>50</sub> (M) – clog <i>P</i>.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div><div class="NLM_p">The reorientation of the phenyl thiazole core of inhibitors <b>37</b> and <b>44</b> when compared to <b>18</b> seems to suggest that the deep cleft in the H-site occupied by inhibitors C4-10 and <b>18</b> cannot accommodate much steric bulk associated with N-substitution of the acrylamide. This ultimately forces N-substituted ligands to occupy a more solvent-exposed pocket of the H-site, which is driven mostly by the hydrophobic packing of the thiazole, whereas the deep hydrophobic cleft occupied by inhibitors C4-10 and <b>18</b> locks the phenyl ring into a more ordered state.</div><div class="NLM_p">On the basis of the above preliminary SAR results, we continued to design analogues based on our new optimized compound <b>44</b>. We first combined the 4-trifluromethylphenyl group and 3-methyl-isoxazole side chain on acrylamide nitrogen to provide a hybrid compound <b>47</b>. This modification showed a little improvement in potency as compared to <b>44</b>. We then examined the isoxazole compound with the 2-hydroxylphenyl group to match the potential protein–ligand interaction requirement, and this molecule exhibits a remarkable picomole level potency (<b>49</b>, IC<sub>50</sub> = 0.00022 μM). Meanwhile, its 2-methoxylphenyl analogue <b>48</b> was 200-fold less potent than <b>49</b>, validating the hypothesis that the hydroxyl group did play an important role on binding interaction. From the cocrystal structure of GSTO1:<b>44</b> and docking studies, it was not apparent that the hydroxyl group would form H-bonds with any protein residues. We believe that the increase in potency is due to charge interaction. The partial negative charge of the oxygen atom would be highly attracted to the positive charge of the drug-binding pocket. Thus, compound <b>49</b> with the 2′-hydroxyl substitution would bind tighter than compound <b>44</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). We also noticed that additional space can be occupied at the 4-position on the benzene ring; therefore, analogues with 4-chloro or 4-fluoro were designed. Compound <b>50</b> also displayed excellent potency, whereas decreased potency was observed with the fluorinated analogue <b>51</b>. High LLE values and optimal clog <i>P</i> of these compounds also illustrated that they also have good druglike properties. These compounds showed remarkable potency in the GSTO1-1 assay but did not show significant cytotoxicity, indicating that the selectivity of these compounds can be further improved (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. GSTO1-1 Binding Assay, clog <i>P</i>, LLE, and Cytotoxicity of Thiazole Arylamides <b>47–51</b><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0020.gif" alt="" id="GRAPHIC-d7e1508-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0021.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values represent the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">clog <i>P</i> values were calculated using ChemBioDraw Professional 16.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Calculated LLE = pIC<sub>50</sub> (M) – clog <i>P</i>.</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div><div class="NLM_p">Compound <b>49</b>, designed based on the GSTO1:<b>44</b> co-crystal structure, is the most potent GSTO1 inhibitor (IC<sub>50</sub> = 0.22 ± 0.02 nM) known to date and did not show off-target-induced cytotoxicity. There is a possibility that 2-OH of this molecule may make the overall molecule a potential chelator. We tested the compound in the presence or absence of various metals to examine whether a chelation could give rise to assay interference. We did not see a significant effect by various metals in the activity of the compound (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S3</a>). This compound also potently inhibited the GSTO1 enzyme activity in the 4-nitrophenacyl glutathione (4NPG) reduction assay (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S4</a>). Although not tested in this study, due to 64% sequence identity of GSTO1 and GSTO2, we expect these inhibitors may also bind to GSTO2.</div><div class="NLM_p">Covalent inhibitors sometimes display instability in vitro or in vivo because of their reactivity.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compounds <b>18</b> and <b>49</b> were selected for stability studies. They showed good stability in 20% acetonitrile/PBS buffer solution at room temperature overnight as detected by liquid chromatography–mass spectrometry (LC–MS). An in vitro mouse and human liver microsome stability assay was used to evaluate their metabolic properties (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). The results showed that compounds were rapidly metabolized in both mouse and human microsomes perhaps due to the acrylamide moiety being the main metabolic liability.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Metabolic Stability and Half-Life of Compounds <b>18</b> and <b>49</b> in Mouse and Human Liver Microsomes</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">percentage remaining (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">time point (min)</th><th class="colsep0 rowsep0" align="center"><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>18 in MLM</th><th class="colsep0 rowsep0" align="center"><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a>18 in HLM</th><th class="colsep0 rowsep0" align="center"><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>49 in MLM</th><th class="colsep0 rowsep0" align="center"><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a>49 in HLM</th><th class="colsep0 rowsep0" align="center"><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a>Verapamil in MLM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">33.93</td><td class="colsep0 rowsep0" align="left">66.67</td><td class="colsep0 rowsep0" align="left">16.00</td><td class="colsep0 rowsep0" align="left">14.81</td><td class="colsep0 rowsep0" align="left">26.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">17.86</td><td class="colsep0 rowsep0" align="left">48.28</td><td class="colsep0 rowsep0" align="left">12.00</td><td class="colsep0 rowsep0" align="left">7.41</td><td class="colsep0 rowsep0" align="left">10.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">12.5</td><td class="colsep0 rowsep0" align="left">39.08</td><td class="colsep0 rowsep0" align="left">12.00</td><td class="colsep0 rowsep0" align="left">3.70</td><td class="colsep0 rowsep0" align="left">4.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">7.14</td><td class="colsep0 rowsep0" align="left">19.54</td><td class="colsep0 rowsep0" align="left">8.00</td><td class="colsep0 rowsep0" align="left">3.70</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">5.36</td><td class="colsep0 rowsep0" align="left">8.04</td><td class="colsep0 rowsep0" align="left">4.00</td><td class="colsep0 rowsep0" align="left">3.70</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">5.36</td><td class="colsep0 rowsep0" align="left">5.75</td><td class="colsep0 rowsep0" align="left">4.00</td><td class="colsep0 rowsep0" align="left">3.70</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">half-life (min)</td><td class="colsep0 rowsep0" align="left">3.21</td><td class="colsep0 rowsep0" align="left">9.52</td><td class="colsep0 rowsep0" align="left">1.81</td><td class="colsep0 rowsep0" align="left">1.89</td><td class="colsep0 rowsep0" align="left">2.70</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">MLM: mouse liver microsome</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">HLM: human liver microsome.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Verapamil was set as a positive control.</p></div></div></div><div class="NLM_p">Additional optimizations were conducted to improve the metabolic stability without major decrease in potency. Two more analogues were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Chloroacetamide, the most commonly used covalent reactive group of GSTO1-1 inhibitors, was conjugated with the scaffold of compound <b>49</b> to obtain compound <b>52</b>. However, the replacement displayed respectable potency loss, indicating that the scaffold is more suitable for acrylamide. Concerning the thiazole nitrogen of compound <b>49</b> may be attacked by its acrylamide structure forming an inactive cyclic product like KT53, we synthesized <b>53</b> by replacing the thiazole ring by thiophene but that resulted in a 1000-fold loss in potency. Future studies will focus on seeking analogues with better PK properties.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. GSTO1-1 Binding Assay, clog <i>P</i>, LLE, and Cytotoxicity of Compounds <b>52</b> and <b>53</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0022.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">GSTO IC<sub>50</sub> values were determined using the GSTO binding assay. Values represent the mean ± SD of at least three independent experiments.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">clog <i>P</i> values were calculated using ChemBioDraw Professional 16.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Calculated LLE = pIC<sub>50</sub> (M) – clog <i>P</i>.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">MTT IC<sub>50</sub> values are reported from two independent experiments.</p></div></div><div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This study reports our structure-based design efforts toward the development of novel GSTO1 inhibitors with better potency and druglike properties. Our efforts toward discovering new analogues with an acrylamide structure led to a potent phenylthiazole compound <b>10</b> (IC<sub>50</sub> = 0.5 ± 0.2 μM) possessing desirable clog <i>P</i>, LE, and LLE. Compound <b>10</b> was identified as a lead, and a series of analogues were synthesized based on a new cocrystal structure with GSTO1. A cocrystal structure of <b>18</b> and GSTO1 revealed that the compound inserted into a hydrophobic cave with the GSTO1 H-site. Derivatives with an amide side chain showed improved potency as compared to those without substitutions. Cocrystal structures of <b>37</b> and <b>44</b> with GSTO1 showed slightly different binding poses than with <b>18</b>, but the core structures were still located in the electrophilic pocket. The introduction of an amide side chain caused a shift in the α9 helix and resulted in a reorientation of the H-site. We also found that the isoxazole ring of compound <b>44</b> occupies a small portion of the G-site and contributed to improved potency.</div><div class="NLM_p last">The inhibitors disclosed in this study present suitable lead compounds for further development. Future studies will focus on further optimizations of <b>49</b> to acquire potent inhibitors with improved PK and pharmaceutical properties. Detailed in vitro and in vivo studies will also be performed to evaluate their efficacy and safety in cancer therapy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50301" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50301" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Chemistry</h3><div class="NLM_p">Reagents and anhydrous solvents were purchased from commercial sources and used without further purification. Reaction progress was monitored by UV absorbance using thin-layer chromatography (TLC) on aluminum-backed precoated silica plates from Silicycle (SiliaPlate, 200 μm thickness, F254). Purifications using flash chromatography were performed using a Biotage Isolera chromatography system equipped with 10 and 25 g Ultra-SNAP Cartridge columns (25 μM spherical silica). Glassware for reactions was oven-dried in reactions performed using nitrogen or argon atmosphere using standard inert conditions. <sup>1</sup>H NMR spectra were obtained using a Bruker NMR instrument (300 or 400 MHz), and spectral data are reported using the following abbreviations: (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublet of doublets); coupling constants are reported in hertz, followed by integration. A Shimadzu LCMS 2020 system was utilized for generating high-performance liquid chromatography (HPLC) traces and obtaining MS data. The system was equipped with a photodiode array (PDA) UV detector and Kinetex 2.6 μm, XB-C18 100 Å, 75 × 4.6 mm column, which was used at room temperature. The HPLC gradient method utilized was 10–95% MeCN in H<sub>2</sub>O with 0.1% formic acid over 15 min with a 0.50 mL/min flow rate. Reverse-phase preparative purifications were performed on a Shimadzu LC20 modular HPLC system utilizing a PDA detector and a Kinetex 5 μm XB-C18 100 Å, 150 × 21.2 mm column. The purification method used was 25 min gradient of 10–90% MeCN in H<sub>2</sub>O with 0.05% trifluoroacetic acid as the additive. The purity of final compounds (≥95%) was assessed at 254 nm using the described column and method.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-(4-Chloro-3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)phenyl)acrylamide (<b>1</b>)</h4><div class="NLM_p last">To a solution of 5-amino-2-chloro-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>4i</b> (50 mg, 0.21 mmol) in DCM (10 mL) was added <i>N,N</i>-diisopropylethylamine (DIEA) (89 μL, 0.63 mmol) and stirred at 0 °C for 20 min. Acryloyl chloride (16 μL, 0.60 mmol) in DCM (5 mL) was added slowly. The reaction mixture was stirred at room temperature for 2 h and then partitioned between DCM and saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM for another two times. The combined organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2–50% EtOAc in hexane to give a white solid (48 mg, 80% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.79 (d, <i>J</i> = 2.6 Hz, 1H), 7.55–7.35 (m, 2H), 7.28 (s, 1H), 7.02 (s, 1H), 6.60 (dd, <i>J</i> = 16.5, 9.8 Hz, 1H), 6.35 (d, <i>J</i> = 16.5 Hz, 1H), 6.07 (d, <i>J</i> = 9.8 Hz, 1H), 2.93 (s, 6H). MS (ESI): 289.0 [M + H]<sup>+</sup>. Purity: 95.8%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-Chloro-<i>N</i>,<i>N</i>-dimethyl-5-(vinylsulfonamido)benzenesulfonamide (<b>2</b>)</h4><div class="NLM_p last">To a solution of 5-amino-2-chloro-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>4i</b> (50 mg, 0.21 mmol) in DCM (10 mL) was added DIEA (89 μL, 0.63 mmol) and stirred at 0 °C for 20 min. 2-Chloroethylsulfonyl chloride (34 mg, 0.21 mmol) in 5 mL DCM was added slowly. The reaction mixture was stirred at room temperature for 2 h and then partitioned between DCM and saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM for another two times. The combined organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified via flash chromatography using a gradient method of 2–50% EtOAc in hexane to give a white solid (25 mg, 36% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.79 (d, <i>J</i> = 2.6 Hz, 1H), 7.55–7.35 (m, 2H), 7.28 (s, 1H), 7.02 (s, 1H), 6.60 (dd, <i>J</i> = 16.5, 9.8 Hz, 1H), 6.35 (d, <i>J</i> = 16.5 Hz, 1H), 6.07 (d, <i>J</i> = 9.8 Hz, 1H), 2.93 (s, 6H). MS (ESI): 325.0 [M + H]<sup>+</sup>. Purity: 95.7%.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Methyl (<i>E</i>)-4-((4-Chloro-3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)phenyl)amino)but-2-enoate (<b>3</b>)</h4><div class="NLM_p last">To a solution of 5-amino-2-chloro-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>4i</b> (100 mg, 0.42 mmol) in acetonitrile (10 mL) were added potassium fluoride (73 mg, 12.8 mmol) and methyl (<i>E</i>)-4-bromocronate (150 mg, 0.84 mmol). The solution was heated at 100 °C for 4 h using a microwave reactor and concentrated in vacuo. The residue was purified with flash chromatography using DCM as the eluent to give <b>3</b> as a white solid (73 mg, 53% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.36–7.25 (m, 2H), 6.93 (dt, <i>J</i> = 15.7, 4.3 Hz, 1H), 6.70 (dd, <i>J</i> = 8.8, 3.1 Hz, 1H), 5.87 (dt, <i>J</i> = 15.8, 2.0 Hz, 1H), 4.12 (dd, <i>J</i> = 4.6, 1.9 Hz, 2H), 3.74 (d, <i>J</i> = 0.8 Hz, 3H), 2.88 (d, <i>J</i> = 0.8 Hz, 6H). MS (ESI): 332.9 [M + H]<sup>+</sup>. Purity: 96.4%.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure A: Preparation of Acrylamides <b>5–12</b></h4><div class="NLM_p">To a solution of phenyl heterocyclic amines (0.50 mmol) in DCM (10 mL) was added DIEA (1.50 mmol) and stirred at 0 °C for 20 min. Acryloyl chloride (0.60 mmol) in DCM (10 mL) was added slowly. The reaction mixture was stirred at room temperature for 2 h and then partitioned between DCM and saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM for two more times. The combined organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified via flash chromatography using a gradient method of 2–50% EtOAc in hexane.</div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <i>N</i>-(1-Phenyl-1<i>H</i>-pyrazol-4-yl)acrylamide (<b>5</b>)</h5><div class="NLM_p last">Compound <b>5</b> was prepared from 1-phenyl-1<i>H</i>-pyrazol-4-amine <b>4a</b> (100 mg, 0.62 mmol), DIEA (210 μL, 1.26 mmol), and acryloyl chloride (56 mg, 0.62 mmol) according to the general procedure A as a yellow solid (100 mg, 75% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.25 (d, <i>J</i> = 1.4 Hz, 1H), 7.69 (d, <i>J</i> = 1.7 Hz, 1H), 7.39–7.21 (m, 3H), 6.43 (dt, <i>J</i> = 16.9, 1.6 Hz, 1H), 6.34–6.18 (m, 1H), 5.74 (ddd, <i>J</i> = 10.1, 2.7, 1.4 Hz, 1H). MS (ESI): 214.1 [M + H]<sup>+</sup>. Purity: 98.5%.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> <i>N</i>-(1-Benzyl-1<i>H</i>-pyrazol-4-yl)acrylamide (<b>6</b>)</h5><div class="NLM_p last">Compound <b>6</b> was prepared from 1-benzyl-1<i>H</i>-pyrazol-4-amine <b>4b</b> (100 mg, 0.58 mmol), DIEA (193 μL, 1.16 mmol), and acryloyl chloride (52 mg, 0.58 mmol) according to the general procedure A as a yellow solid (107 mg, 82% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.45 (s, 1H), 8.02 (s, 1H), 7.45 (s, 1H), 7.37–7.24 (m, 1H), 7.29 (s, 2H), 7.20 (dd, <i>J</i> = 7.4, 2.2 Hz, 2H), 6.37 (dd, <i>J</i> = 17.0, 1.6 Hz, 1H), 6.21 (dd, <i>J</i> = 16.9, 10.0 Hz, 1H), 5.67 (dd, <i>J</i> = 10.0, 1.6 Hz, 1H), 5.23 (s, 2H). MS (ESI): 227.9 [M + H]<sup>+</sup>. Purity: 98.5%.</div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 1-(4-Phenylpiperidin-1-yl)prop-2-en-1-one (<b>7</b>)</h5><div class="NLM_p last">Compound <b>7</b> was prepared from 4-phenylpiperidine <b>4c</b> (100 mg, 0.62 mmol), DIEA (207 μL, 1.24 mmol), and acryloyl chloride (56 mg, 0.62 mmol) according to the general procedure A as a colorless oil (117 mg, 88% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.40–7.17 (m, 5H), 6.65 (dd, <i>J</i> = 16.8, 10.5 Hz, 1H), 6.32 (dd, <i>J</i> = 16.8, 2.0 Hz, 1H), 5.72 (dd, <i>J</i> = 10.5, 2.0 Hz, 1H), 4.86 (d, <i>J</i> = 13.2 Hz, 1H), 4.21–4.10 (m, 1H), 3.20 (t, <i>J</i> = 13.0 Hz, 1H), 2.93–2.75 (m, 1H), 2.87–2.55 (m, 3H), 1.69 (td, <i>J</i> = 16.3, 12.6, 7.5 Hz, 2H). MS (ESI): 215.9 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 1-(4-Benzylpiperidin-1-yl)prop-2-en-1-one (<b>8</b>)</h5><div class="NLM_p last">Compound <b>8</b> was prepared from 4-benzylpiperidine <b>4d</b> (100 mg, 0.57 mmol), DIEA (190 μL, 1.14 mmol), and acryloyl chloride (51 mg, 0.57 mmol) according to the general procedure A as a colorless oil (117 mg, 90% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.37–7.19 (m, 2H), 7.16 (dd, <i>J</i> = 5.5, 3.3 Hz, 3H), 6.59 (dd, <i>J</i> = 16.8, 10.5 Hz, 1H), 6.27 (dd, <i>J</i> = 16.8, 2.0 Hz, 1H), 5.67 (dd, <i>J</i> = 10.6, 2.0 Hz, 1H), 4.14 (q, <i>J</i> = 7.1 Hz, 1H), 2.67–2.47 (m, 4H), 2.05 (d, <i>J</i> = 11.3 Hz, 2H), 1.71 (s, 4H). MS (ESI): 230.1 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> <i>N</i>-(5-Phenylthiazol-2-yl)acrylamide (<b>9</b>)</h5><div class="NLM_p last">Compound <b>9</b> was prepared from 5-phenylthiazol-2-amine <b>4e</b> (100 mg, 0.57 mmol), DIEA (190 μL, 1.14 mmol), and acryloyl chloride (51 mg, 0.57 mmol) according to the general procedure A as a white solid (58 mg, 45% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.71–7.55 (m, 3H), 7.51–7.30 (m, 3H), 6.68 (dd, <i>J</i> = 17.0, 1.3 Hz, 1H), 6.50 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.00 (dd, <i>J</i> = 10.1, 1.3 Hz, 1H). MS (ESI): 231.0 [M + H]<sup>+</sup>. Purity: 97.7%.</div></div><div id="sec4_1_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> <i>N</i>-(4-Phenylthiazol-2-yl)acrylamide (<b>10</b>)</h5><div class="NLM_p last">Compound <b>10</b> was prepared from 4-phenylthiazol-2-amine <b>4f</b> (100 mg, 0.57 mmol), DIEA (190 μL, 1.14 mmol), and acryloyl chloride (51 mg, 0.57 mmol) according to the general procedure A as a white solid (52 mg, 40% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.83 (d, <i>J</i> = 7.6 Hz, 2H), 7.41 (dt, <i>J</i> = 23.4, 7.1 Hz, 3H), 7.21 (s, 1H), 6.56–6.44 (m, 1H), 6.06 (ddd, <i>J</i> = 16.8, 10.4, 6.2 Hz, 1H), 5.75 (dd, <i>J</i> = 10.4, 5.5 Hz, 1H). MS (ESI): 231.0 [M + H]<sup>+</sup>. Purity: 98.0%.</div></div><div id="sec4_1_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Ethyl 2-Acrylamido-4-phenylthiophene-3-carboxylate (<b>11</b>)</h5><div class="NLM_p last">Compound <b>11</b> was prepared from ethyl 4-phenylthiophene-2-amino-3-carboxylate <b>4g</b> (100 mg, 0.40 mmol), DIEA (135 μL, 0.82 mmol), and acryloyl chloride (36 mg, 0.40 mmol) according to the general procedure A as a yellow solid (93 mg, 77% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.91 (s, 1H), 7.49 (d, <i>J</i> = 1.7 Hz, 5H), 6.46 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 6.34 (dd, <i>J</i> = 17.0, 9.8 Hz, 1H), 5.85 (dd, <i>J</i> = 9.9, 1.6 Hz, 1H), 4.26 (q, <i>J</i> = 7.1 Hz, 2H), 1.14 (t, <i>J</i> = 7.1 Hz, 3H), 0.09 (s, 4H). MS (ESI): 302.1 [M + H]<sup>+</sup>. Purity: 95.0%.</div></div><div id="sec4_1_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Ethyl 2-Acrylamido-5-phenylthiophene-3-carboxylate (<b>12</b>)</h5><div class="NLM_p last">Compound <b>11</b> was prepared from ethyl 5-phenylthiophene-2-amino-3-carboxylate <b>4h</b> (100 mg, 0.40 mmol), DIEA (135 μL, 0.82 mmol), and acryloyl chloride (36 mg, 0.40 mmol) according to the general procedure A as a yellow solid (97 mg, 80% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 11.23 (s, 1H), 7.63 (dd, <i>J</i> = 7.3, 1.7 Hz, 2H), 7.50–7.25 (m, 5H), 6.56 (dd, <i>J</i> = 17.0, 1.1 Hz, 1H), 6.39 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 5.93 (dd, <i>J</i> = 10.2, 1.2 Hz, 1H), 4.40 (q, <i>J</i> = 7.1 Hz, 2H), 1.44 (t, <i>J</i> = 7.1 Hz, 3H). MS (ESI): 302.1 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-Chloro-2-methoxyacetophenone (<b>13r</b>)</h4><div class="NLM_p last">To a solution of 4-chloro-2-hydroxylacetophenone <b>13p</b> (1.0 g, 5.86 mmol) in acetonitrile (20 mL) was added anhydrous K<sub>2</sub>CO<sub>3</sub> (1.61 g, 11.7 mmol) and stirred at room temperature for 10 min. Methyl iodine (0.98 g, 7.03 mmol) was then added, and the reaction mixture was heated to reflux overnight and concentrated. The residue was suspended in H<sub>2</sub>O and extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, and concentrated to give 4-chloro-2-methoxyacetophenone <b>13r</b> as a white solid (1.04 g, 97% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.72 (d, <i>J</i> = 8.2 Hz, 1H), 7.05–6.94 (m, 2H), 3.93 (s, 3H), 2.61 (d, <i>J</i> = 0.6 Hz, 3H). MS (ESI): 185.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-Fluoro-2-methoxyacetophenone (<b>13s</b>)</h4><div class="NLM_p last">To a solution of 4-fluoro-2-hydroxylacetophenone <b>13p</b> (1.0 g, 6.49 mmol) in acetonitrile (20 mL) was added anhydrous K<sub>2</sub>CO<sub>3</sub> (1.79 g, 13.0 mmol) and stirred at room temperature for 10 min. Methyl iodine (1.10 g, 7.78 mmol) was then added, and the reaction mixture was heated to reflux overnight and concentrated. The residue was suspended in H<sub>2</sub>O and extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, and concentrated to give 4-chloro-2-methoxyacetophenone <b>13s</b> as a white solid (1.05 g, 94% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88–7.77 (m, 1H), 6.77–6.63 (m, 2H), 3.92 (s, 3H), 2.60 (s, 3H). MS (ESI): 169.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> General Procedure B: Preparation of Compounds <b>14a–14t</b></h4><div class="NLM_p">To a solution of 2-bromoacetophenones (3.00 mmol) in EtOAc (25 mL) was added copper bromide (6.00 mmol), and the suspension was heated to reflux under vigorous stirring for 3 h. Then the resulting white suspension was cooled to room temperature, and the solid was removed by filtration. The filtrate was washed with water, dried with MgSO<sub>4</sub>, filtered, and concentrated to give compounds <b>14a</b>–<b>14t</b> (55–97%).</div><div id="sec4_1_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 2-Bromo-2′-chloroacetophenone (<b>14a</b>)</h5><div class="NLM_p last">Intermediate <b>14a</b> was prepared from 2-chloroacetophenone <b>13a</b> according to the general procedure B as a colorless oil (539 mg, 77% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.62–7.54 (m, 1H), 7.46 (d, <i>J</i> = 4.9 Hz, 2H), 7.39 (dq, <i>J</i> = 7.7, 4.6, 3.6 Hz, 1H), 4.54 (d, <i>J</i> = 0.9 Hz, 2H). MS (ESI): 232.9, 234.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 2-Bromo-3′-chloroacetophenone (<b>14b</b>)</h5><div class="NLM_p last">Intermediate <b>14b</b> was prepared from 3-chloroacetophenone <b>13b</b> according to the general procedure B as a white solid (616 mg, 88% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.98 (t, <i>J</i> = 1.9 Hz, 1H), 7.93–7.83 (m, 1H), 7.66–7.56 (m, 1H), 7.47 (t, <i>J</i> = 7.9 Hz, 1H), 4.44 (s, 2H). MS (ESI): 232.9, 234.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 2-Bromo-4′-chloroacetophenone (<b>14c</b>)</h5><div class="NLM_p last">Intermediate <b>14c</b> was prepared from 4-chloroacetophenone <b>13c</b> according to the general procedure B as a white solid (658 mg, 95% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.95 (d, <i>J</i> = 8.2 Hz, 2H), 7.50 (d, <i>J</i> = 8.6 Hz, 2H), 4.43 (s, 2H). MS (ESI): 232.9, 234.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 2-Bromo-4′-methylacetophenone (<b>14d</b>)</h5><div class="NLM_p last">Intermediate <b>14d</b> was prepared from 4-chloroacetophenone <b>13d</b> according to the general procedure B as a white solid (658 mg, 88% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.91 (d, <i>J</i> = 8.0 Hz, 1H), 7.31 (d, <i>J</i> = 8.0 Hz, 1H), 4.46 (d, <i>J</i> = 0.6 Hz, 2H), 2.45 (s, 3H). MS (ESI): 212.9, 214.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 2-Bromo-4′-trifluoromethylacetophenone (<b>14e</b>)</h5><div class="NLM_p last">Intermediate <b>14e</b> was prepared from 4-trifluoromethylacetophenone <b>13e</b> according to the general procedure B as a white solid (440 mg, 55% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.13 (d, <i>J</i> = 8.2 Hz, 2H), 7.79 (d, <i>J</i> = 8.2 Hz, 2H), 4.48 (s, 2H). MS (ESI): 266.9, 268.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 2-Bromo-4′-methoxyacetophenone (<b>14f</b>)</h5><div class="NLM_p last">Intermediate <b>14f</b> was prepared from 4-methoxyacetophenone <b>13f</b> according to the general procedure B as a gray solid (612 mg, 90% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.99 (d, <i>J</i> = 9.1 Hz, 1H), 6.98 (d, <i>J</i> = 8.7 Hz, 1H), 4.42 (s, 1H), 3.91 (s, 2H). MS (ESI): 228.2, 230.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 2-Bromo-3′,4′-dioxolacetophenone (<b>14g</b>)</h5><div class="NLM_p last">Intermediate <b>14g</b> was prepared from 3,4-dioxolacetophenone <b>13g</b> according to the general procedure B as a yellow solid (568 mg, 79% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.66–7.54 (m, 2H), 7.51–7.34 (m, 2H), 6.90 (d, <i>J</i> = 8.2 Hz, 1H), 6.09 (s, 2H), 4.40 (s, 2H). MS (ESI): 242.9, 244.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 2-Bromo-2′-acetonaphthone (<b>14h</b>)</h5><div class="NLM_p last">Intermediate <b>14h</b> was prepared from 2-acetonaphthone <b>13h</b> according to the general procedure B as a white solid (720 mg, 97% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.53 (d, <i>J</i> = 1.6 Hz, 1H), 8.05 (dd, <i>J</i> = 8.6, 1.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.0 Hz, 1H), 7.97–7.88 (m, 3H), 7.72–7.54 (m, 3H), 4.61 (s, 2H). MS (ESI): 248.9, 250.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 2-Bromo-2′-hydroxylacetophenone (<b>14i</b>)</h5><div class="NLM_p last">Intermediate <b>14i</b> was prepared from 2-hydroxylacetophenone <b>13i</b> according to the general procedure B as a yellow solid (460 mg, 72% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 11.76 (s, 1H), 7.77 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 7.55 (ddd, <i>J</i> = 8.7, 7.2, 1.6 Hz, 1H), 7.05 (d, <i>J</i> = 8.5 Hz, 1H), 7.03–6.90 (m, 1H), 4.47 (s, 2H). MS (ESI): 214.9, 216.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 2-Bromo-4′-hydroxylacetophenone (<b>14j</b>)</h5><div class="NLM_p last">Intermediate <b>14j</b> was prepared from 4-hydroxylacetophenone <b>13j</b> according to the general procedure B as a yellow solid (496 mg, 73% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.40 (s, 1H), 7.98 (d, <i>J</i> = 8.6 Hz, 2H), 6.98 (d, <i>J</i> = 8.7 Hz, 2H), 4.65 (s, 2H). MS (ESI): 214.9, 216.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 2-Bromo-2′-methoxyacetophenone (<b>14k</b>)</h5><div class="NLM_p last">Intermediate <b>14k</b> was prepared from 2-methoxyacetophenone <b>13k</b> according to the general procedure B as a yellow solid (612 mg, 91% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.85 (dd, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.61–7.48 (m, 1H), 7.12–6.97 (m, 2H), 4.64 (s, 2H), 3.97 (s, 3H). MS (ESI): 228.2, 230.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2-Bromo-2′-fluoroacetophenone (<b>14l</b>)</h5><div class="NLM_p last">Intermediate <b>14l</b> was prepared from 2-fluoroacetophenone <b>13l</b> according to the general procedure B as a colorless oil (611 mg, 94% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.96 (td, <i>J</i> = 7.6, 2.1 Hz, 1H), 7.61 (q, <i>J</i> = 7.1 Hz, 1H), 7.36–7.23 (m, 1H), 7.19 (dd, <i>J</i> = 11.0, 8.0 Hz, 1H), 4.55 (dt, <i>J</i> = 2.5, 1.3 Hz, 2H). MS (ESI): 216.9, 218.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-Bromo-2′-methoxy-4′-chloroacetophenone (<b>14r</b>)</h5><div class="NLM_p last">Intermediate <b>14r</b> was prepared from 2-methoxy-4-chloroacetophenone <b>13r</b> according to the general procedure B as a yellow solid (687 mg, 87% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.73 (dd, <i>J</i> = 8.3, 6.2 Hz, 1H), 7.02–6.97 (m, 2H), 4.56 (s, 2H), 3.95 (s, 3H). MS (ESI): 262.9, 264.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 2-Bromo-2′-methoxy-4′-fluoroacetophenone (<b>14s</b>)</h5><div class="NLM_p last">Intermediate <b>14s</b> was prepared from 2-methoxy-4-fluoroacetophenone <b>13s</b> according to the general procedure B as a yellow solid (666 mg, 90% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.89 (dd, <i>J</i> = 8.6, 6.9 Hz, 1H), 6.85–6.58 (m, 2H), 4.57 (s, 2H), 3.95 (s, 3H). MS (ESI): 246.9, 248.9 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> General Procedure C: Preparation of Compounds <b>15a–15o</b> and <b>28a–28b</b></h4><div class="NLM_p">To a solution of 2-bromoacetophenones (1.0 equiv) in EtOH (20 mL) was added corresponding thiourea (1.2 equiv). The reaction mixture was heated to reflux for 2 h. Then the resulting solution was cooled to room temperature and poured into 2 M ammonium hydroxide aqueous solution (40 mL). The solid was collected by filtration, washed with water, and dried to give compounds <b>15a–15p</b> and <b>28a–28b</b>.</div><div id="sec4_1_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 4-(2-Chlorophenyl)thiazol-2-amine (<b>15a</b>)</h5><div class="NLM_p last">Intermediate <b>15a</b> was prepared from 2-bromo-2′-chloroacetophenone <b>14a</b> (200 mg, 0.85 mmol) and thiourea (64 mg, 0.85 mmol) according to the general procedure C as a white solid (164 mg, 92% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.97 (dd, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.47 (dd, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.33 (dtd, <i>J</i> = 21.8, 7.4, 1.7 Hz, 3H), 7.15 (s, 1H), 6.50 (s, 2H). MS (ESI): 211.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 4-(3-Chlorophenyl)thiazol-2-amine (<b>15b</b>)</h5><div class="NLM_p last">Intermediate <b>15b</b> was prepared from 2-bromo-3′-chloroacetophenone <b>14b</b> (200 mg, 0.85 mmol) and thiourea (64 mg, 0.85 mmol) according to the general procedure C as a white solid (173 mg, 97% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.90 (t, <i>J</i> = 1.9 Hz, 1H), 7.85–7.76 (m, 1H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 7.33–7.24 (m, 1H), 7.09 (s, 1H), 6.52 (s, 2H). MS (ESI): 211.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 4-(4-Chlorophenyl)thiazol-2-amine (<b>15c</b>)</h5><div class="NLM_p last">Intermediate <b>15c</b> was prepared from 2-bromo-4′-chloroacetophenone <b>14c</b> (200 mg, 0.85 mmol) and thiourea (64 mg, 0.85 mmol) according to the general procedure C as a white solid (171 mg, 96% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.87 (t, <i>J</i> = 8.3 Hz, 2H), 7.39 (d, <i>J</i> = 8.5 Hz, 2H), 7.01 (s, 1H), 6.51 (s, 2H). MS (ESI): 211.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 4-(4-Methylphenyl)thiazol-2-amine (<b>15d</b>)</h5><div class="NLM_p last">Intermediate <b>15d</b> was prepared from 2-bromo-4′-methylacetophenone <b>14d</b> (200 mg, 0.86 mmol) and thiourea (65 mg, 0.86 mmol) according to the general procedure C as a white solid (161 mg, 99% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.75 (d, <i>J</i> = 8.2 Hz, 2H), 7.17 (t, <i>J</i> = 7.9 Hz, 2H), 6.87 (s, 1H), 6.42 (s, 2H), 2.33 (s, 3H). MS (ESI): 191.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 4-(4-Trifluoromethylphenyl)thiazol-2-amine (<b>15e</b>)</h5><div class="NLM_p last">Intermediate <b>15e</b> was prepared from 2-bromo-4′-trifluoromethylacetophenone <b>14e</b> (200 mg, 0.75 mmol) and thiourea (57 mg, 0.75 mmol) according to the general procedure C as a white solid (177 mg, 97% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8.5 Hz, 1H), 7.41 (d, <i>J</i> = 8.5 Hz, 1H), 7.11 (s, 2H), 7.07 (s, 1H). MS (ESI): 245.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 4-(4-Methoxyphenyl)thiazol-2-amine (<b>15f</b>)</h5><div class="NLM_p last">Intermediate <b>15f</b> was prepared from 2-bromo-4′-methoxyacetophenone <b>14f</b> (200 mg, 0.88 mmol) and thiourea (71 mg, 0.88 mmol) according to the general procedure C as a white solid (161 mg, 89% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.79 (d, <i>J</i> = 8.8 Hz, 2H), 6.90 (dd, <i>J</i> = 14.7, 8.6 Hz, 2H), 6.78 (s, 1H), 6.37 (s, 1H), 3.83 (d, <i>J</i> = 5.9 Hz, 3H). MS (ESI): 207.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 4-(3′,4′-Dioxolphenyl)thiazol-2-amine (<b>15g</b>)</h5><div class="NLM_p last">Intermediate <b>15g</b> was prepared from 2-bromo-3′,4′-dioxolacetophenone <b>14g</b> (200 mg, 0.82 mmol) and thiourea (62 mg, 0.85 mmol) according to the general procedure C as a white solid (142 mg, 79% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.41 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.35 (d, <i>J</i> = 1.6 Hz, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 6.81 (s, 1H), 6.41 (s, 2H), 6.01 (s, 2H). MS (ESI): 221.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 4-(Naphthalen-2-yl)thiazol-2-amine (<b>15h</b>)</h5><div class="NLM_p last">Intermediate <b>15h</b> was prepared from 2-bromo-2′-acetonaphthone <b>14h</b> (200 mg, 0.81 mmol) and thiourea (61 mg, 0.81 mmol) according to the general procedure C as a white solid (144 mg, 79% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.41 (s, 1H), 8.00 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.90 (t, <i>J</i> = 8.9 Hz, 3H), 7.49 (tt, <i>J</i> = 7.4, 5.6 Hz, 2H), 7.11 (s, 1H), 6.53 (s, 2H). MS (ESI): 227.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 4-(2-Hydroxylphenyl)thiazol-2-amine (<b>15i</b>)</h5><div class="NLM_p last">Intermediate <b>15i</b> was prepared from 2-bromo-2′-hydroxylacetophenone <b>14i</b> (200 mg, 0.93 mmol) and thiourea (71 mg, 0.93 mmol) according to the general procedure C as a white solid (119 mg, 67% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.93 (s, 1H), 7.67 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H), 7.22–7.09 (m, 1H), 7.00 (s, 1H), 6.99–6.92 (m, 2H), 6.88–6.74 (m, 2H). MS (ESI): 193.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 4-(4-Hydroxylphenyl)thiazol-2-amine (<b>15j</b>)</h5><div class="NLM_p last">Intermediate <b>15j</b> was prepared from 2-bromo-4′-hydroxylacetophenone <b>14j</b> (200 mg, 0.93 mmol) and thiourea (71 mg, 0.93 mmol) according to the general procedure C as a white solid (131 mg, 74% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.87 (s, 1H), 7.70 (d, <i>J</i> = 8.9 Hz, 2H), 6.83 (d, <i>J</i> = 8.9 Hz, 2H), 6.71 (s, 1H). MS (ESI): 193.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4-(2-Methoxyphenyl)thiazol-2-amine (<b>15k</b>)</h5><div class="NLM_p last">Intermediate <b>15k</b> was prepared from 2-bromo-2′-methoxyacetophenone <b>14k</b> (200 mg, 0.88 mmol) and thiourea (71 mg, 0.88 mmol) according to the general procedure C as a white solid (148 mg, 82% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.17 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.24 (dd, <i>J</i> = 15.5, 1.8 Hz, 1H), 7.24 (s, 2H), 7.10–6.92 (m, 2H), 6.47 (s, 2H), 3.94 (s, 3H). MS (ESI): 207.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 4-(2-Fluorophenyl)thiazol-2-amine (<b>15l</b>)</h5><div class="NLM_p last">Intermediate <b>15l</b> was prepared from 2-bromo-2′-fluoroacetophenone <b>14l</b> (200 mg, 0.92 mmol) and thiourea (71 mg, 0.92 mmol) according to the general procedure C as a white solid (171 mg, 96% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.11 (td, <i>J</i> = 1.5, 7.7 Hz, 1H), 7.38–7.11 (m, 3H), 7.02 (d, <i>J</i> = 2.5 Hz, 1H), 6.48 (s, 2H). MS (ESI): 195.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 5-Methyl-4-phenylthiazol-2-amine (<b>15o</b>)</h5><div class="NLM_p last">Intermediate <b>15o</b> was prepared from 2-bromo-1-phenylpropan-1-one <b>14o</b> (200 mg, 0.86 mmol) and thiourea (64 mg, 0.86 mmol) according to the general procedure C as a white solid (83 mg, 51% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.62–7.52 (m, 2H), 7.48–7.26 (m, 4H), 5.34 (s, 2H), 2.40 (s, 3H). MS (ESI): 191.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> General Procedure D: Preparation of Acrylamides <b>16–27</b>, <b>29</b>, <b>30</b></h4><div class="NLM_p">To a solution of thiazol-2-amines (1.0 equiv) in DCM (10 mL) was added DIEA (2.0 equiv) and stirred at 0 °C for 20 min. Acryloyl chloride (1.0 equiv) in DCM (10 mL) was added slowly. The reaction mixture was stirred at room temperature for 2 h and then partitioned between DCM and saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM for two more times. The combined organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2–50% EtOAc in hexane.</div><div id="sec4_1_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> <i>N</i>-(4-(2-Chlorophenyl)thiazol-2-yl)acrylamide (<b>16</b>)</h5><div class="NLM_p last">Compound <b>16</b> was prepared from 4-(2-chlorophenyl)thiazol-2-amine <b>15a</b> (50 mg, 0.24 mmol), DIEA (79 μL, 0.48 mmol), and acryloyl chloride (21 mg, 0.24 mmol) according to the general procedure D as a white solid (32 mg, 52% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.76 (dd, <i>J</i> = 7.2, 2.3 Hz, 1H), 7.55–7.41 (m, 2H), 7.40–7.25 (m, 3H), 6.43 (d, <i>J</i> = 16.9 Hz, 1H), 5.92 (dd, <i>J</i> = 16.9, 10.4 Hz, 1H), 5.66 (d, <i>J</i> = 10.3 Hz, 1H). MS (ESI): 265.0 [M + H]<sup>+</sup>. Purity: 95.6%.</div></div><div id="sec4_1_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>N</i>-(4-(3-Chlorophenyl)thiazol-2-yl)acrylamide (<b>17</b>)</h5><div class="NLM_p last">Compound <b>17</b> was prepared from 4-(3-chlorophenyl)thiazol-2-amine <b>15b</b> (50 mg, 0.24 mmol), DIEA (79 μL, 0.48 mmol), and acryloyl chloride (21 mg, 0.24 mmol) according to the general procedure D as a white solid (30 mg, 48% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.82 (s, 1H), 7.71 (d, <i>J</i> = 7.4 Hz, 1H), 7.40 (d, <i>J</i> = 8.1 Hz, 1H), 7.36 (s, 1H), 7.22 (s, 1H), 6.60 (d, <i>J</i> = 16.8 Hz, 1H), 6.29 (dd, <i>J</i> = 17.3, 10.2 Hz, 1H), 5.94 (d, <i>J</i> = 10.4 Hz, 1H). MS (ESI): 265.0 [M + H]<sup>+</sup>. Purity: 99.1%</div></div><div id="sec4_1_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> <i>N</i>-(4-(4-Chlorophenyl)thiazol-2-yl)acrylamide (<b>18</b>)</h5><div class="NLM_p last">Compound <b>18</b> was prepared from 4-(4-chlorophenyl)thiazol-2-amine <b>15c</b> (50 mg, 0.24 mmol), DIEA (79 μL, 0.48 mmol), and acryloyl chloride (21 mg, 0.24 mmol) according to the general procedure D as a white solid (31 mg, 50% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.10 (s, 1H), 7.77 (d, <i>J</i> = 8.4 Hz, 2H), 7.41 (d, <i>J</i> = 8.4 Hz, 2H), 7.20 (s, 1H), 6.53 (d, <i>J</i> = 16.9 Hz, 1H), 6.12 (dd, <i>J</i> = 17.0, 10.4 Hz, 1H), 5.82 (d, <i>J</i> = 10.3 Hz, 1H). MS (ESI): 265.0 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_9_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> <i>N</i>-(4-(4-Methylphenyl)thiazol-2-yl)acrylamide (<b>19</b>)</h5><div class="NLM_p last">Compound <b>19</b> was prepared from 4-(4-methylphenyl)thiazol-2-amine <b>15d</b> (50 mg, 0.26 mmol), DIEA (87 μL, 0.52 mmol), and acryloyl chloride (23 mg, 0.26 mmol) according to the general procedure D as a white solid (39 mg, 61% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.71 (d, <i>J</i> = 7.9 Hz, 2H), 7.25 (d, <i>J</i> = 7.8 Hz, 2H), 7.14 (s, 1H), 6.51 (d, <i>J</i> = 16.9 Hz, 1H), 6.08 (dd, <i>J</i> = 16.9, 10.4 Hz, 1H), 5.77 (d, <i>J</i> = 10.4 Hz, 1H), 2.40 (s, 3H). MS (ESI): 245.2 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_9_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> <i>N</i>-(4-(4-Trifluoromethylphenyl)thiazol-2-yl)acrylamide (<b>20</b>)</h5><div class="NLM_p last">Compound <b>20</b> was prepared from 4-(4-trifluoromethylphenyl)thiazol-2-amine <b>15e</b> (50 mg, 0.20 mmol), DIEA (68 μL, 0.40 mmol), and acryloyl chloride (18 mg, 0.20 mmol) according to the general procedure D as a white solid (20 mg, 33% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.95 (d, <i>J</i> = 8.2 Hz, 2H), 7.70 (d, <i>J</i> = 8.1 Hz, 2H), 7.28 (s, 2H), 6.58 (d, <i>J</i> = 16.9 Hz, 1H), 6.24 (dd, <i>J</i> = 16.9, 10.4 Hz, 1H), 5.90 (d, <i>J</i> = 10.4 Hz, 1H), 1.27 (s, 1H), 0.09 (s, 5H). MS (ESI): 299.0 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_9_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> <i>N</i>-(4-(4-Methoxyphenyl)thiazol-2-yl)acrylamide (<b>21</b>)</h5><div class="NLM_p last">Compound <b>21</b> was prepared from 4-(4-methoxyphenyl)thiazol-2-amine <b>15f</b> (50 mg, 0.24 mmol), DIEA (79 μL, 0.48 mmol), and acryloyl chloride (22 mg, 0.24 mmol) according to the general procedure D as a white solid (43 mg, 68% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.76 (d, <i>J</i> = 8.6 Hz, 2H), 7.07 (s, 1H), 6.97 (d, <i>J</i> = 8.6 Hz, 2H), 6.48 (d, <i>J</i> = 16.9 Hz, 1H), 6.05 (dd, <i>J</i> = 16.9, 10.4 Hz, 1H), 5.72 (d, <i>J</i> = 10.4 Hz, 1H), 3.86 (s, 3H). MS (ESI): 261.0 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_9_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> <i>N</i>-(4-(3,4-Dioxylphenyl)thiazol-2-yl)acrylamide (<b>22</b>)</h5><div class="NLM_p last">Compound <b>22</b> was prepared from 4-(3,4-dioxylphenyl)thiazol-2-amine <b>15g</b> (50 mg, 0.23 mmol), DIEA (76 μL, 0.45 mmol), and acryloyl chloride (21 mg, 0.23 mmol) according to the general procedure D as a white solid (41 mg, 66% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.29 (s, 1H), 7.52–7.35 (m, 3H), 6.88 (d, <i>J</i> = 8.1 Hz, 1H), 6.71 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.53 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 6.04 (s, 2H), 5.92 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H). MS (ESI): 275.0 [M + H]<sup>+</sup>. Purity: 98.0%.</div></div><div id="sec4_1_9_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> <i>N</i>-(4-(Naphthalen-2-yl)thiazol-2-yl)acrylamide (<b>23</b>)</h5><div class="NLM_p last">Compound <b>23</b> was prepared from 4-(naphthalen-2-yl)thiazol-2-amine <b>15h</b> (50 mg, 0.22 mmol), DIEA (74 μL, 0.44 mmol), and acryloyl chloride (20 mg, 0.22 mmol) according to the general procedure D as a white solid (49 mg, 79% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.40 (s, 1H), 8.46 (s, 1H), 8.07 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.99–7.86 (m, 3H), 7.67 (s, 1H), 7.51 (hept, <i>J</i> = 5.0 Hz, 2H), 6.76 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.56 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.95 (dd, <i>J</i> = 10.1, 1.8 Hz, 1H). MS (ESI): 281.1 [M + H]<sup>+</sup>. Purity: 96.8%.</div></div><div id="sec4_1_9_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> <i>N</i>-(4-(2-Hydroxylphenyl)thiazol-2-yl)acrylamide (<b>24</b>)</h5><div class="NLM_p last">Compound <b>24</b> was prepared from 4-(2-hydroxylphenyl)thiazol-2-amine <b>15i</b> (50 mg, 0.26 mmol), DIEA (87 μL, 0.52 mmol), and acryloyl chloride (24 mg, 0.26 mmol) according to the general procedure D as a white solid (9 mg, 15% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 9.55 (s, 1H), 7.64 (d, <i>J</i> = 8.3 Hz, 1H), 7.25 (d, <i>J</i> = 4.2 Hz, 1H), 7.10–6.87 (m, 2H), 6.64 (d, <i>J</i> = 16.8 Hz, 1H), 6.38 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 5.99 (d, <i>J</i> = 10.3 Hz, 1H). MS (ESI): 246.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> <i>N</i>-(4-(4-Hydroxylphenyl)thiazol-2-yl)acrylamide (<b>25</b>)</h5><div class="NLM_p last">Compound <b>25</b> was prepared from 4-(4-hydroxylphenyl)thiazol-2-amine <b>15j</b> (50 mg, 0.26 mmol), DIEA (87 μL, 0.52 mmol), and acryloyl chloride (24 mg, 0.26 mmol) according to the general procedure D as a white solid (14 mg, 22% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.64 (d, <i>J</i> = 8.7 Hz, 3H), 7.48 (s, 1H), 7.03–6.88 (m, 3H), 6.68 (d, <i>J</i> = 17.3 Hz, 1H), 6.55–6.40 (m, 1H), 6.01 (d, <i>J</i> = 10.2 Hz, 1H). MS (ESI): 246.1 [M + H]<sup>+</sup>. Purity: 98.2%.</div></div><div id="sec4_1_9_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> <i>N</i>-(4-(2-Methoxyphenyl)thiazol-2-yl)acrylamide (<b>26</b>)</h5><div class="NLM_p last">Compound <b>26</b> was prepared from 4-(2-methoxyphenyl)thiazol-2-amine <b>15k</b> (50 mg, 0.26 mmol), DIEA (88 μL, 0.52 mmol), and acryloyl chloride (21 mg, 0.26 mmol) according to the general procedure D as a white solid (30 mg, 48% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.25 (s, 1H), 8.15 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.76 (s, 1H), 7.37–7.24 (m, 1H), 7.12 (d, <i>J</i> = 8.1 Hz, 1H), 7.01 (t, <i>J</i> = 7.5 Hz, 1H), 6.72 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.53 (dd, <i>J</i> = 17.1, 1.8 Hz, 1H), 5.92 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H), 4.00 (s, 3H). MS (ESI): 261.0 [M + H]<sup>+</sup>. Purity: 97.2%.</div></div><div id="sec4_1_9_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> <i>N</i>-(4-(2-Fluorophenyl)thiazol-2-yl)acrylamide (<b>27</b>)</h5><div class="NLM_p last">Compound <b>27</b> was prepared from 4-(2-fluorophenyl)thiazol-2-amine <b>15l</b> (50 mg, 0.25 mmol), DIEA (86 μL, 0.50 mmol), and acryloyl chloride (23 mg, 0.25 mmol) according to the general procedure D as a white solid (38 mg, 60% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.29 (s, 1H), 8.07–7.95 (m, 1H), 7.48 (d, <i>J</i> = 2.1 Hz, 1H), 7.37–7.28 (m, 1H), 7.32–7.11 (m, 3H), 6.51 (d, <i>J</i> = 16.9 Hz, 1H), 6.11 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 5.78 (d, <i>J</i> = 10.3 Hz, 1H). MS (ESI): 248.0 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_9_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> <i>N</i>-Benzyl-4-phenylthiazol-2-amine (<b>28c</b>)</h5><div class="NLM_p last">To a solution of 4-phenylthiazol-2-amine <b>4f</b> (200 mg, 1.13 mmol) in acetonitrile (20 mL) was added anhydrous K<sub>2</sub>CO<sub>3</sub> (470 mg, 3.34 mmol) and stirred at room temperature for 30 min. Benzyl bromide (193 mg, 1.13 mmol) was added, and the reaction mixture was heated at reflux for 8 h before concentrated. The residue was suspended in saturated NaHCO<sub>3</sub> aqueous solution before being extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2–30% EtOAc in hexane to give <i>N</i>-benzyl-4-phenylthiazol-2-amine <b>28c</b> as a white solid (52 mg, 17% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.83 (d, <i>J</i> = 7.5 Hz, 2H), 7.43–7.27 (m, 8H), 6.72 (s, 1H), 6.14 (s, 1H), 4.52 (s, 2H). MS (ESI): 267.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> Ethyl (4-Phenylthiazol-2-yl)glycinate (<b>28d</b>)</h5><div class="NLM_p last">To a solution of 4-phenylthiazol-2-amine <b>4f</b> (200 mg, 1.13 mmol) in acetonitrile (20 mL) was added anhydrous K<sub>2</sub>CO<sub>3</sub> (470 mg, 3.34 mmol) and stirred at room temperature for 30 min. Ethyl bromoacetate (188 mg, 1.13 mmol) was added, and the reaction mixture was heated to reflux for 8 h. Acetonitrile was removed in vacuo after the reaction was completed. The residue was suspended in saturated NaHCO<sub>3</sub> aqueous solution before being extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2–35% EtOAc in hexane to give <i>N</i>-benzyl-4-phenylthiazol-2-amine <b>28d</b> as a white solid (77 mg, 26% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88–7.77 (m, 2H), 7.47–7.17 (m, 3H), 6.77 (d, <i>J</i> = 14.4 Hz, 1H), 4.39 (s, 2H), 4.28 (dq, <i>J</i> = 11.9, 7.3 Hz, 2H), 1.33 (q, <i>J</i> = 7.2 Hz, 3H). MS (ESI): 263.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> General Procedure E: Preparation of Compounds <b>28e–28l</b></h4><div class="NLM_p">To a solution of 2-bromoacetophenones (1.0 equiv) in 15 mL EtOH was added potassium thiocyanate (1.0 equiv) and was heated under reflux for 2 h. Then to the resulting solution were added corresponding primary amines (1.2 equiv) in EtOH (10 mL), and it was heated to reflux overnight before being concentrated. The residue was suspended in H<sub>2</sub>O and neutralized before extracting with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2 to 35% EtOAc in hexane.</div><div id="sec4_1_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> Methyl(4-phenylthiazol-2-yl)glycinate (<b>28e</b>)</h5><div class="NLM_p last">Compound <b>28e</b> was prepared from 2-bromoacetophenone <b>14m</b> (1.0 g, 5.03 mmol), potassium thiocyanate (488 mg, 5.03 mmol), and methyl glycinate hydrochloride (757 mg, 6.03 mmol) according to the general procedure E as a yellow solid (910 mg, 73% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88–7.78 (m, 2H), 7.46–7.31 (m, 3H), 7.28 (s, 1H), 6.75 (s, 1H), 5.84 (s, 1H), 4.25 (s, 2H), 3.83 (s, 3H). MS (ESI): 249.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>tert</i>-Butyl(4-phenylthiazol-2-yl)glycinate (<b>28f</b>)</h5><div class="NLM_p last">Compound <b>28f</b> was prepared from 2-bromoacetophenone <b>14m</b> (300 mg, 1.51 mmol), potassium thiocyanate (146 mg, 1.51 mmol), and <i>tert</i>-butyl glycinate hydrogen chloride (303 mg, 1.81 mmol) according to the general procedure E as a yellow solid (380 mg, 87% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.83 (d, <i>J</i> = 7.6 Hz, 2H), 7.46–7.30 (m, 3H), 7.28 (s, 1H), 6.74 (s, 1H), 5.91 (s, 1H), 4.11 (s, 2H), 1.52 (s, 9H). MS (ESI): 291.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-4-Phenylthiazol-2-amine (<b>28g</b>)</h5><div class="NLM_p last">Compound <b>28g</b> was prepared from 2-bromoacetophenone <b>14m</b> (300 mg, 1.51 mmol), potassium thiocyanate (146 mg, 1.51 mmol), and (3-methylisoxazol-5-yl)methanamine (203 mg, 1.81 mmol) according to the general procedure E as a yellow solid (229 mg, 56% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88–7.78 (m, 2H), 7.46–7.32 (m, 13H), 7.28 (s, 1H), 6.75 (s, 1H), 6.07 (s, 1H), 4.63 (s, 2H), 2.42 (s, 3H). MS (ESI): 272.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 4-Phenyl-<i>N</i>-(pyridin-3-ylmethyl)thiazol-2-amine (<b>28h</b>)</h5><div class="NLM_p last">Compound <b>28h</b> was prepared from 2-bromoacetophenone <b>14m</b> (300 mg, 1.51 mmol), potassium thiocyanate (146 mg, 1.51 mmol), and pyridin-3-ylmethanamine (194 mg, 1.81 mmol) according to the general procedure E as a yellow solid (133 mg, 33% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.58 (d, <i>J</i> = 2.3 Hz, 1H), 8.50 (dd, <i>J</i> = 4.9, 1.6 Hz, 1H), 7.74 (ddt, <i>J</i> = 21.5, 7.9, 1.8 Hz, 3H), 7.50–7.19 (m, 5H), 6.95 (s, 1H), 6.68 (s, 1H), 4.49 (s, 2H). MS (ESI): 268.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-4-(4-trifluoromethylphenyl)thiazol-2-amine (<b>28i</b>)</h5><div class="NLM_p last">Compound <b>28i</b> was prepared from 2-bromo-4′-trifluoromethylacetophenone <b>14e</b> (300 mg, 1.12 mmol), potassium thiocyanate (109 mg, 1.12 mmol), and (3-methylisoxazol-5-yl)methanamine (151 mg, 1.35 mmol) according to the general procedure E as a yellow solid (255 mg, 67% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.93 (d, <i>J</i> = 8.1 Hz, 2H), 7.65 (d, <i>J</i> = 8.1 Hz, 3H), 6.87 (s, 1H), 6.08 (s, 1H), 4.67 (s, 2H), 2.43 (d, <i>J</i> = 0.9 Hz, 3H). MS (ESI): 340.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-4-(4-chloro-2-methoxyphenyl)thiazol-2-amine (<b>28j</b>)</h5><div class="NLM_p last">Compound <b>28j</b> was prepared from 2-bromo-4′-chloro-2′-methoxyacetophenone <b>14r</b> (300 mg, 1.14 mmol), potassium thiocyanate (111 mg, 1.14 mmol), and (3-methylisoxazol-5-yl)methanamine (153 mg, 1.37 mmol) according to the general procedure E as a yellow solid (233 mg, 61% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.06 (d, <i>J</i> = 8.4 Hz, 1H), 7.21 (s, 1H), 7.05–6.91 (m, 2H), 6.09 (s, 1H), 4.66 (s, 2H), 3.94 (s, 3H), 2.28 (s, 3H). MS (ESI): 336.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-4-(4-fluoro-2-methoxyphenyl)thiazol-2-amine (<b>28k</b>)</h5><div class="NLM_p last">Compound <b>28k</b> was prepared from 2-bromo-4′-fluoro-2′-methoxyacetophenone <b>14s</b> (300 mg, 1.21 mmol), potassium thiocyanate (118 mg, 1.21 mmol), and (3-methylisoxazol-5-yl)methanamine (163 mg, 1.46 mmol) according to the general procedure E as a yellow solid (213 mg, 55% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.97 (d, <i>J</i> = 8.4 Hz, 1H), 7.12 (s, 1H), 6.91 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.85 (d, <i>J</i> = 2.0 Hz, 1H), 6.00 (s, 1H), 4.56 (s, 2H), 3.84 (s, 3H), 2.18 (s, 3H). MS (ESI): 320.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-4-(2-methoxylphenyl)thiazol-2-amine (<b>28l</b>)</h5><div class="NLM_p last">Compound <b>28l</b> was prepared from 2-bromo-2′-methoxyacetophenone <b>14l</b> (300 mg, 1.31 mmol), potassium thiocyanate (127 mg, 1.31 mmol), and (3-methylisoxazol-5-yl)methanamine (176 mg, 1.57 mmol) according to the general procedure E as a yellow solid (236 mg, 60% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.15 (d, <i>J</i> = 7.7 Hz, 1H), 7.34–7.21 (m, 1H), 7.15–6.93 (m, 2H), 6.06 (d, <i>J</i> = 4.1 Hz, 1H), 4.63 (s, 2H), 3.96 (d, <i>J</i> = 1.5 Hz, 3H), 2.40 (d, <i>J</i> = 2.0 Hz, 3H). MS (ESI): 302.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>N</i>-(4-Phenyl-5-methyl-thiazol-2-yl)acrylamide (<b>29</b>)</h5><div class="NLM_p last">Compound <b>29</b> was prepared from 5-methyl-4-phenylthiazol-2-amine <b>15o</b> (50 mg, 0.26 mmol), DIEA (88 μL, 0.52 mmol), and acryloyl chloride (23 mg, 0.26 mmol) according to the general procedure D as a white solid (35 mg, 55% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.60 (d, <i>J</i> = 7.4 Hz, 2H), 7.38 (dt, <i>J</i> = 27.7, 7.3 Hz, 3H), 6.23 (dd, <i>J</i> = 16.5, 1.5 Hz, 1H), 5.46 (dd, <i>J</i> = 16.7, 10.3 Hz, 1H), 5.31 (dd, <i>J</i> = 10.3, 1.6 Hz, 1H), 2.54 (s, 3H). MS (ESI): 245.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Ethyl 2-Acrylamido-4-phenylthiazole-5-carboxylate (<b>30</b>)</h5><div class="NLM_p last">Compound <b>16</b> was prepared from commercially available ethyl 2-amino-4-phenylthiazole-5-carboxylate <b>15p</b> (50 mg, 0.18 mmol), DIEA (67 μL, 0.36 mmol), and acryloyl chloride (18 mg, 0.18 mmol) according to the general procedure D as a white solid (37 mg, 61% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.75 (dd, <i>J</i> = 6.6, 2.9 Hz, 2H), 7.44 (dd, <i>J</i> = 5.3, 3.6 Hz, 2H), 6.29 (dd, <i>J</i> = 15.9, 2.2 Hz, 1H), 5.48–5.37 (m, 1H), 5.35 (dd, <i>J</i> = 15.9, 10.3 Hz, 1H), 4.31 (q, <i>J</i> = 7.1 Hz, 2H), 1.33 (t, <i>J</i> = 7.1 Hz, 3H). MS (ESI): 303.1 [M + H]<sup>+</sup>. Purity: 95.6%</div></div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> General Procedure G: Preparation of N-Substituted Acrylamides <b>33–34</b>, <b>37–45</b>, <b>46a–46b</b>, <b>47–48</b></h4><div class="NLM_p">To a solution of 2-aminethiazoles (0.25 mmol) in toluene (15 mL) was added DIEA (0.75 mmol) and stirred at room temperature for 10 min. Acryloyl chloride (0.37 mmol) was then added slowly to the solution. The reaction mixture was heated at 70 °C for 0.5 h, and the solvent was removed in vacuo. The residue was suspended in saturated NaHCO<sub>3</sub> aqueous solution and extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 2–35% EtOAc in hexane.</div><div id="sec4_1_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> <i>N</i>-Benzyl-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>33</b>)</h5><div class="NLM_p last">Compound <b>32</b> was prepared from <i>N</i>-benzyl-4-phenylthiazol-2-amine <b>28c</b> (50 mg, 0.17 mmol), DIEA (84 μL, 0.50 mmol), and acryloyl chloride (23 mg, 0.25 mmol) according to the general procedure G as a white solid (19 mg, 32% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88 (d, <i>J</i> = 1.6 Hz, 1H), 7.46–7.25 (m, 10H), 6.80–6.57 (m, 2H), 5.89 (dd, <i>J</i> = 9.4, 2.6 Hz, 1H), 5.71 (s, 2H). MS (ESI): 321.1 [M + H]<sup>+</sup>. Purity: 98.9%.</div></div><div id="sec4_1_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Ethyl <i>N</i>-Acryloyl-<i>N</i>-(4-phenylthiazol-2-yl)glycinate (<b>34</b>)</h5><div class="NLM_p last">Compound <b>34</b> was prepared from ethyl(4-phenylthiazol-2-yl)glycinate <b>28d</b> (50 mg, 0.19 mmol), DIEA (96 μL, 0.57 mmol), and acryloyl chloride (26 mg, 0.29 mmol) according to the general procedure G as a white solid (21 mg, 34% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.93–7.83 (m, 2H), 7.48–7.25 (m, 3H), 7.26 (s, 1H), 6.65 (d, <i>J</i> = 5.9 Hz, 2H), 5.98 (t, <i>J</i> = 5.9 Hz, 1H), 5.11 (s, 2H), 4.29 (q, <i>J</i> = 7.1 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H). MS (ESI): 317.1 [M + H]<sup>+</sup>. Purity: 96.5%.</div></div><div id="sec4_1_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> (4-Phenylthiazol-2-yl)glycine Hydrogen Chloride (<b>35</b>)</h5><div class="NLM_p last">To a solution of intermediate <b>28e</b> (1.0 g, 4.03 mmol) in tetrahydrofuran (THF) (20 mL) was added 1 M HCl aqueous solution (5 mL). The reaction mixture was heated under reflux for 2 h, and the solvent was removed to give (4-phenylthiazol-2-yl)glycine hydrogen chloride <b>35</b> as a light yellow solid (890 mg, 94%). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>): δ 7.99–7.80 (m, 2H), 7.44–7.32 (m, 3H), 7.01 (s, 1H), 4.48 (s, 1H), 4.27 (s, 2H). MS (ESI): 235.0 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> General Procedure F: Preparation of Compounds <b>36a–36d</b></h4><div class="NLM_p">To a solution of intermediate <b>35</b> (100 mg, 0.37 mmol) in DMF (10 mL) were added 1-ethyl-3-(3-(dimethylamino)propyl)-carbodiimide (EDCI) (105 mg, 0.55 mmol), hydroxybenzotriazole (HOBt) (74 mg, 0.55 mmol), amines (0.44 mmol), and DIEA (185 μL, 1.11 mmol). The reaction mixture was stirred at room temperature overnight. After the reaction was completed, the solution was poured into saturated NaHCO<sub>3</sub> aqueous solution (25 mL) and extracted with EtOAc (3×). The organic layer was dried with MgSO<sub>4</sub>, filtered, concentrated, and purified with flash chromatography using a gradient method of 0–10% methanol in DCM.</div><div id="sec4_1_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>N</i>-Cyclohexyl-2-((4-phenylthiazol-2-yl)amino)acetamide (<b>36a</b>)</h5><div class="NLM_p last">Compound <b>36a</b> was prepared from <b>35</b> and cyclohexylamine (43 mg, 0.44 mmol) according to the general procedure F as a yellow solid (92 mg, 67% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.80 (dd, <i>J</i> = 7.1, 1.7 Hz, 2H), 7.39 (dd, <i>J</i> = 8.2, 6.5 Hz, 3H), 7.37–7.25 (m, 2H), 6.75 (s, 1H), 4.04 (s, 2H), 1.90 (dd, <i>J</i> = 12.6, 4.0 Hz, 3H), 1.74–1.54 (m, 4H), 1.46–1.24 (m, 3H), 1.29–1.05 (m, 4H). MS (ESI): 316.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> 1-Morpholino-2-((4-phenylthiazol-2-yl)amino)ethan-1-one (<b>36b</b>)</h5><div class="NLM_p last">Compound <b>36b</b> was prepared from <b>35</b> and morpholine (38 mg, 0.44 mmol) according to the general procedure F as a yellow oil (103 mg, 78% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.75–7.65 (m, 2H), 7.49 (d, <i>J</i> = 7.2 Hz, 3H), 6.61 (s, 1H), 4.26 (s, 2H), 3.70 (dd, <i>J</i> = 18.7, 6.8 Hz, 6H), 3.52 (d, <i>J</i> = 5.1 Hz, 2H). MS (ESI): 304.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> <i>N</i>-(Oxetan-3-yl)-2-((4-phenylthiazol-2-yl)amino)acetamide (<b>36c</b>)</h5><div class="NLM_p last">Compound <b>36c</b> was prepared from <b>35</b> and oxetan-3-amine hydrogen chloride (48 mg, 0.44 mmol) according to the general procedure F as a yellow solid (126 mg, 88% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88–7.78 (m, 2H), 7.46–7.32 (m, 3H), 7.28 (s, 1H), 6.75 (s, 1H), 6.07 (s, 1H), 4.63 (s, 2H), 2.42 (s, 3H). MS (ESI): 289.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> <i>N</i>-(2-Morpholinoethyl)-2-((4-phenylthiazol-2-yl)amino)acetamide (<b>36d</b>)</h5><div class="NLM_p last">Compound <b>36d</b> was prepared from <b>35</b> and 2-morpholinoethan-1-amine (57 mg, 0.44 mmol) according to the general procedure F as a yellow oil (89 mg, 59% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.86–7.76 (m, 2H), 7.45–7.25 (m, 4H), 7.06 (s, 1H), 6.77 (s, 1H), 4.13 (s, 2H), 3.60 (t, <i>J</i> = 4.6 Hz, 4H), 3.43 (q, <i>J</i> = 5.6 Hz, 2H), 2.53 (t, <i>J</i> = 5.9 Hz, 2H). MS (ESI): 347.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> Methyl <i>N</i>-Acryloyl-<i>N</i>-(4-phenylthiazol-2-yl)glycinate (<b>37</b>)</h5><div class="NLM_p last">Compound <b>37</b> was prepared from methyl (4-phenylthiazol-2-yl) glycinate <b>28e</b> (50 mg, 0.20 mmol), DIEA (101 μL, 0.60 mmol), and acryloyl chloride (27 mg, 0.30 mmol) according to the general procedure G as a white solid (33 mg, 55% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.92–7.82 (m, 2H), 7.48–7.23 (m, 5H), 6.69–6.60 (m, 2H), 6.03–5.93 (m, 1H), 5.15 (s, 2H), 3.83 (s, 3H). MS (ESI): 303.1 [M + H]<sup>+</sup>. Purity: 95.0%.</div></div><div id="sec4_1_12_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> <i>tert</i>-Butyl <i>N</i>-Acryloyl-<i>N</i>-(4-phenylthiazol-2-yl)glycinate (<b>38</b>)</h5><div class="NLM_p last">Compound <b>38</b> was prepared from <i>t</i>-butyl(4-phenylthiazol-2-yl)glycinate <b>28f</b> (100 mg, 0.34 mmol), DIEA (173 μL, 1.03 mmol), and acryloyl chloride (47 mg, 0.52 mmol) according to the general procedure G as a white solid (71 mg, 60% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.04–7.80 (m, 3H), 7.56–7.18 (m, 5H), 6.69–6.60 (m, 2H), 6.03–5.92 (m, 1H), 4.98 (d, <i>J</i> = 15.5 Hz, 2H). MS (ESI): 345.1 [M + H]<sup>+</sup>. Purity: 94.2%.</div></div><div id="sec4_1_12_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> <i>N</i>-Acryloyl-<i>N</i>-(4-phenylthiazol-2-yl)glycine (<b>39</b>)</h5><div class="NLM_p last">To a solution of <i>tert</i>-butyl <i>N</i>-acryloyl-<i>N</i>-(4-phenylthiazol-2-yl)glycinate <b>38</b> (50 mg, 0.14 mmol) in 10 mL DCM was added 10 mL trifluoroacetic acid. The solution was stirred at room temperature for 2 h before it was concentrated. The residue was purified with preparative HPLC to give a white solid (31 mg, 79% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88 (d, <i>J</i> = 7.7 Hz, 2H), 7.42 (t, <i>J</i> = 7.5 Hz, 2H), 7.38–7.25 (m, 2H), 6.66 (d, <i>J</i> = 8.8 Hz, 2H), 5.97 (dd, <i>J</i> = 8.7, 3.4 Hz, 1H), 5.12 (s, 2H). MS (ESI): 289.06 [M + H]<sup>+</sup>. Purity: 95.6%.</div></div><div id="sec4_1_12_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> <i>N</i>-(2-(Cyclohexylamino)-2-oxoethyl)-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>40</b>)</h5><div class="NLM_p last">Compound <b>40</b> was prepared from <i>N</i>-cyclohexyl-2-((4-phenylthiazol-2-yl)amino)acetamide <b>36a</b> (50 mg, 0.16 mmol), DIEA (80 μL, 0.38 mmol), and acryloyl chloride (22 mg, 0.24 mmol) according to the general procedure G as a white solid (36 mg, 61% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.88 (d, <i>J</i> = 7.4 Hz, 2H), 7.50–7.25 (m, 4H), 7.00 (dd, <i>J</i> = 16.7, 10.4 Hz, 1H), 6.73–6.61 (m, 1H), 6.02 (d, <i>J</i> = 10.1 Hz, 1H), 4.92 (s, 2H), 3.81 (d, <i>J</i> = 8.0 Hz, 1H), 1.85 (d, <i>J</i> = 12.2 Hz, 2H), 1.63–1.44 (m, 3H), 1.41–1.24 (m, 3H), 1.11 (t, <i>J</i> = 11.0 Hz, 3H). MS (ESI): 370.1 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_12_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> <i>N</i>-(2-Morpholino-2-oxoethyl)-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>41</b>)</h5><div class="NLM_p last">Compound <b>41</b> was prepared from 1-morpholino-2-((4-phenylthiazol-2-yl)amino)ethan-1-one <b>36b</b> (50 mg, 0.17 mmol), DIEA (83 μL, 0.50 mmol), and acryloyl chloride (22 mg, 0.25 mmol) according to the general procedure G as a white solid (35 mg, 60% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.82 (d, <i>J</i> = 7.4 Hz, 2H), 7.42 (t, <i>J</i> = 7.5 Hz, 2H), 7.39–7.25 (m, 2H), 6.65 (d, <i>J</i> = 8.4 Hz, 2H), 5.94 (s, 1H), 5.20 (s, 2H), 3.85–3.67 (m, 8H). MS (ESI): 358.1 [M + H]<sup>+</sup>. Purity: 96.3%.</div></div><div id="sec4_1_12_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> <i>N</i>-(2-(Oxetan-3-ylamino)-2-oxoethyl)-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>42</b>)</h5><div class="NLM_p last">Compound <b>42</b> was prepared from <i>N</i>-(oxetan-3-yl)-2-((4-phenylthiazol-2-yl)amino)acetamide <b>36c</b> (50 mg, 0.15 mmol), DIEA (77 μL, 0.46 mmol), and acryloyl chloride (21 mg, 0.23 mmol) according to the general procedure G as a white solid (38 mg, 65% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.93–7.83 (m, 2H), 7.79 (s, 1H), 7.42 (dt, <i>J</i> = 27.7, 7.4 Hz, 4H), 7.28 (s, 1H), 6.95 (dd, <i>J</i> = 16.6, 10.3 Hz, 1H), 6.68 (dd, <i>J</i> = 16.6, 1.6 Hz, 1H), 6.08–5.98 (m, 1H), 5.17–4.99 (m, 1H), 5.04–4.83 (m, 4H), 4.44 (t, <i>J</i> = 6.5 Hz, 2H). MS (ESI): 344.1 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_12_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> <i>N</i>-(2-((2-Morpholinoethyl)amino)-2-oxoethyl)-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>43</b>)</h5><div class="NLM_p last">Compound <b>43</b> was prepared from <i>N</i>-(2-morpholinoethyl)-2-((4-phenylthiazol-2-yl)amino)acetamide <b>36d</b> (50 mg, 0.14 mmol), DIEA (72 μL, 0.43 mmol), and acryloyl chloride (20 mg, 0.22 mmol) according to the general procedure G as a white solid (26 mg, 45% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.94–7.84 (m, 2H), 7.48–7.27 (m, 4H), 7.22 (s, 1H), 6.94 (dd, <i>J</i> = 16.6, 10.3 Hz, 1H), 6.68 (dd, <i>J</i> = 16.6, 1.7 Hz, 1H), 6.02 (dd, <i>J</i> = 10.4, 1.6 Hz, 1H), 5.04 (s, 2H), 3.37 (dd, <i>J</i> = 11.4, 5.6 Hz, 6H), 2.41 (t, <i>J</i> = 5.8 Hz, 2H). MS (ESI): 401.1 [M + H]<sup>+</sup>. Purity: 98.2%.</div></div><div id="sec4_1_12_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-phenylthiazol-2-yl)acrylamide (<b>44</b>)</h5><div class="NLM_p last">Compound <b>44</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-4-phenylthiazol-2-amine <b>28g</b> (50 mg, 0.18 mmol), DIEA (92 μL, 0.55 mmol), and acryloyl chloride (25 mg, 0.28 mmol) according to the general procedure G as a white solid (25 mg, 42% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.99–7.83 (m, 2H), 7.51–7.25 (m, 4H), 7.03 (dd, <i>J</i> = 16.6, 10.3 Hz, 1H), 6.65 (dd, <i>J</i> = 16.6, 1.7 Hz, 1H), 6.08 (s, 1H), 5.99 (dd, <i>J</i> = 10.4, 1.7 Hz, 1H), 5.64 (s, 2H), 2.38 (d, <i>J</i> = 0.8 Hz, 3H). MS (ESI): 326.1 [M + H]<sup>+</sup>. Purity: 98.4%.</div></div><div id="sec4_1_12_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> <i>N</i>-(4-Phenylthiazol-2-yl)-<i>N</i>-(pyridin-3-ylmethyl)acrylamide (<b>45</b>)</h5><div class="NLM_p last">Compound <b>45</b> was prepared from 4-phenyl-<i>N</i>-(pyridin-3-ylmethyl)thiazol-2-amine <b>28h</b> (50 mg, 0.19 mmol), DIEA (94 μL, 0.56 mmol), and acryloyl chloride (25 mg, 0.28 mmol) according to the general procedure G as a white solid (19 mg, 31% yield). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.01–7.91 (m, 2H), 7.72 (s, 2H), 7.62 (s, 1H), 7.47–7.32 (m, 6H), 7.35–7.24 (m, 2H), 7.20 (d, <i>J</i> = 7.9 Hz, 1H), 6.97 (dd, <i>J</i> = 16.5, 10.4 Hz, 1H), 6.54 (dd, <i>J</i> = 16.6, 1.9 Hz, 1H), 5.93 (dd, <i>J</i> = 10.4, 2.0 Hz, 1H), 5.83 (s, 2H). MS (ESI): 322.1 [M + H]<sup>+</sup>. Purity: 97.2%.</div></div><div id="sec4_1_12_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-chloro-2-methoxyphenyl)thiazol-2-yl)acrylamide (<b>46a</b>)</h5><div class="NLM_p last">Compound <b>46a</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-4-(4-chloro-2-methoxyphenyl)thiazol-2-amine <b>28j</b> (100 mg, 0.29 mmol), DIEA (148 μL, 0.88 mmol), and acryloyl chloride (40 mg, 0.45 mmol) according to the general procedure G as a white solid (51 mg, 44% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.12 (d, <i>J</i> = 8.3 Hz, 1H), 7.72 (s, 1H), 7.09–6.91 (m, 2H), 6.94–6.82 (m, 1H), 6.68 (d, <i>J</i> = 16.5 Hz, 1H), 6.09 (s, 1H), 6.01 (d, <i>J</i> = 10.3 Hz, 1H), 5.65 (s, 2H), 3.98 (s, 3H), 2.26 (s, 3H). MS (ESI): 390.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-fluoro-2-methoxyphenyl)thiazol-2-yl)acrylamide (<b>46b</b>)</h5><div class="NLM_p last">Compound <b>46b</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-4-(4-fluoro-2-methoxyphenyl)thiazol-2-amine <b>28k</b> (100 mg, 0.30 mmol), DIEA (149 μL, 0.89 mmol), and acryloyl chloride (40 mg, 0.45 mmol) according to the general procedure G as a white solid (38 mg, 44% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.12 (d, <i>J</i> = 8.3 Hz, 1H), 7.72 (s, 1H), 7.09–6.82 (m, 3H), 6.68 (d, <i>J</i> = 16.5 Hz, 1H), 6.12–5.96 (m, 2H), 5.65 (s, 2H), 3.98 (s, 3H), 2.26 (s, 3H). MS (ESI): 374.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> 2-Chloro-<i>N</i>-(4-(2-methoxyphenyl)thiazol-2-yl)-<i>N</i>-((3-methylisoxazol-5-yl)methyl)acetamide (<b>46c</b>)</h5><div class="NLM_p last">A solution of 2-chloroacetyl chloride (57 mg, 0.50 mmol), <b>28l</b> (100 mg, 0.33 mmol), and DIEA (129 mg, 1.00 mmol) in toluene (2 mL) was heated at 70 °C for 1 h. The mixture was concentrated, and the residue was purified with flash chromatography using DCM as the eluent to give <b>2</b> as a white solid (93 mg, 74% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.10–8.04 (m, 1H), 7.80 (s, 1H), 7.31 (d, <i>J</i> = 8.8 Hz, 1H), 7.13 (d, <i>J</i> = 8.0 Hz, 1H), 7.04 (t, <i>J</i> = 7.6 Hz, 1H), 6.40 (s, 1H), 5.64 (s, 2H), 4.92 (s, 2H), 3.92 (s, 3H), 2.17 (s, 3H). MS (ESI): 377.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-trifluoromethylphenyl)thiazol-2-yl)acrylamide (<b>47</b>)</h5><div class="NLM_p last">Compound <b>47</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-4-(4-trifluoromethylphenyl)thiazol-2-amine <b>28i</b> (50 mg, 0.16 mmol), DIEA (78 μL, 0.47 mmol), and acryloyl chloride (21 mg, 0.23 mmol) according to the general procedure G as a white solid (20 mg, 35% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.03 (d, <i>J</i> = 8.2 Hz, 2H), 7.69 (d, <i>J</i> = 8.2 Hz, 2H), 7.41 (s, 1H), 7.03 (dd, <i>J</i> = 16.5, 10.4 Hz, 1H), 6.67 (d, <i>J</i> = 16.2 Hz, 1H), 6.08–5.96 (m, 2H), 5.64 (s, 2H), 2.39 (s, 3H). MS (ESI): 394.1 [M + H]<sup>+</sup>. Purity: 95.5%.</div></div><div id="sec4_1_12_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(2-methoxyphenyl)thiazol-2-yl)acrylamide (<b>48</b>)</h5><div class="NLM_p last">Compound <b>48</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-4-(2-methoxylphenyl)thiazol-2-amine <b>28l</b> (100 mg, 0.33 mmol), DIEA (166 μL, 1.00 mmol), and acryloyl chloride (45 mg, 0.50 mmol) according to the general procedure G as a white solid (59 mg, 50% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.25 (d, <i>J</i> = 7.9 Hz, 1H), 7.78 (s, 1H), 7.38–7.25 (m, 1H), 7.13–6.94 (m, 4H), 6.64 (d, <i>J</i> = 16.5 Hz, 1H), 6.07 (s, 1H), 5.97 (d, <i>J</i> = 10.4 Hz, 1H), 5.64 (s, 2H), 4.00 (s, 3H), 2.37 (s, 3H). MS (ESI): 356.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> General Procedure H: Preparation of Compounds <b>49–51</b></h4><div class="NLM_p">A solution of methoxy-containing acrylamides (1.0 equiv) in anhydrous DCM (20 mL) was stirred at −78 °C for 30 min, and 1 M BBr<sub>3</sub> solution in DCM (5.0 equiv) was added dropwise. The reaction mixture was slowly warmed to room temperature and stirred for 2 h. Methanol (1 mL) was slowly added at −30 °C to quench the reaction. The mixture was extracted with DCM three times. The combined organic layer was dried with MgSO<sub>4</sub>, filtered, and concentrated. The crude was refluxed in 5% DIEA THF solution (20 mL) for 1 h. The solvent was then removed in vacuo, and the residue was purified with preparative HPLC.</div><div id="sec4_1_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(2-hydroxylphenyl)thiazol-2-yl)acrylamide (<b>49</b>)</h5><div class="NLM_p last">Compound <b>49</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(2-methoxyphenyl)thiazol-2-yl)acrylamide <b>48</b> (30 mg, 0.084 mmol) and 1 M BBr<sub>3</sub> solution (0.25 mL, 0.25 mmol) according to the general procedure G as a white solid (10 mg, 35% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.65 (d, <i>J</i> = 8.0 Hz, 1H), 7.33 (s, 1H), 7.08–6.88 (m, 3H), 6.70 (d, <i>J</i> = 16.5 Hz, 1H), 6.05 (d, <i>J</i> = 9.7 Hz, 2H), 5.49 (s, 2H), 2.42 (s, 3H). MS (ESI): 342.1 [M + H]<sup>+</sup>. Purity: 98.0%.</div></div><div id="sec4_1_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-chloro-2-hydroxylphenyl)thiazol-2-yl)acrylamide (<b>50</b>)</h5><div class="NLM_p last">Compound <b>50</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-chloro-2-methoxyphenyl)thiazol-2-yl)acrylamide <b>46a</b> (30 mg, 0.076 mmol) and 1 M BBr<sub>3</sub> solution (0.22 mL, 0.22 mmol) according to the general procedure G as a white solid (10 mg, 36% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.05–6.85 (m, 3H), 6.73 (d, <i>J</i> = 16.4 Hz, 1H), 6.25 (s, 1H), 6.11 (d, <i>J</i> = 10.3 Hz, 1H), 5.49 (s, 2H), 2.31 (s, 3H). MS (ESI): 376.0 [M + H]<sup>+</sup>. Purity: 98.4%.</div></div><div id="sec4_1_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> <i>N</i>-((3-Methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-fluoro-2-hydroxylphenyl)thiazol-2-yl)acrylamide (<b>51</b>)</h5><div class="NLM_p last">Compound <b>51</b> was prepared from <i>N</i>-((3-methylisoxazol-5-yl)methyl)-<i>N</i>-(4-(4-fluoro-2-methoxyphenyl)thiazol-2-yl)acrylamide <b>46b</b> (30 mg, 0.077 mmol) and 1 M BBr<sub>3</sub> solution (0.23 mL, 0.23 mmol) according to the general procedure G as a white solid (7 mg, 25% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.57 (dd, <i>J</i> = 8.7, 6.4 Hz, 1H), 7.22 (s, 1H), 6.98 (dd, <i>J</i> = 16.3, 10.3 Hz, 1H), 6.82–6.59 (m, 3H), 6.24 (s, 1H), 6.11 (d, <i>J</i> = 10.3 Hz, 1H), 5.50 (s, 2H), 2.31 (s, 3H). MS (ESI): 374.1 [M + H]<sup>+</sup>. Purity: 95.0%.</div></div><div id="sec4_1_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> 2-Chloro-<i>N</i>-(4-(2-hydroxyphenyl)thiazol-2-yl)-<i>N</i>-((3-methylisoxazol-5-yl)methyl)acetamide (<b>52</b>)</h5><div class="NLM_p last">Compound <b>52</b> was prepared from <b>46c</b> (30 mg, 0.077 mmol) and 1 M BBr<sub>3</sub> solution (0.23 mL, 0.23 mmol) according to the general procedure G as a white solid (14 mg, 24% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.85 (s, 1H), 7.63 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H), 7.34 (s, 1H), 7.31–7.25 (m, 1H), 7.02 (dd, <i>J</i> = 8.2, 1.2 Hz, 1H), 6.94 (ddd, <i>J</i> = 7.8, 7.2, 1.3 Hz, 1H), 6.34 (s, 1H), 5.54 (s, 2H), 4.63 (s, 2H), 2.31 (s, 3H). MS (ESI): 363.9 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_13_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> <i>N</i>-(4-(2-Hydroxyphenyl)thiophen-2-yl)-<i>N</i>-((3-methylisoxazol-5-yl)methyl)acrylamide (<b>53</b>)</h5><div class="NLM_p last">To a solution of <b>57</b> (19 mg, 0.058 mmol), (2-hydroxyphenyl)boronic acid (12 mg, 0.087 mmol), and Na<sub>2</sub>CO<sub>3</sub> (12 mg, 0.116 mmol) in dioxane (1.5 mL) and H<sub>2</sub>O (0.5 mL) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (6.7 mg, 0.0058 mmol) under argon. The mixture was heated at 80 °C under argon for 8 h. The reaction mixture was diluted with EtOAc, washed with saturated NaHCO<sub>3</sub> and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and purified with preparative HPLC to give <b>53</b> as a white solid (8 mg, 41% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.48 (s, 1H), 7.36 (d, <i>J</i> = 7.6 Hz, 1H), 7.24 (t, <i>J</i> = 7.9 Hz, 1H), 7.17 (s, 1H), 7.00 (t, <i>J</i> = 7.5 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 6.54–6.37 (m, 2H), 6.16 (s, 1H), 5.72 (d, <i>J</i> = 9.9 Hz, 1H), 5.05 (s, 2H), 2.32 (s, 3H). MS (ESI): 341.1 [M + H]<sup>+</sup>. Purity: 100.0%.</div></div><div id="sec4_1_13_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> <i>tert</i>-Butyl(4-bromothiophen-2-yl)((3-methylisoxazol-5-yl)methyl)carbamate (<b>55</b>)</h5><div class="NLM_p last">To a solution of <i>tert</i>-butyl (4-bromothiophen-2-yl)carbamate (40 mg, 0.144 mmol) in DMF (1 mL) at 0 °C was added NaH (8.6 mg, 0.216 mmol, 60% in mineral oil). The mixture was stirred at room temperature for 15 min before 5-(bromomethyl)-3-methylisoxazole (32 mg, 0.188 mmol) was added. The mixture was stirred for another 4 h, quenched by H<sub>2</sub>O, and extracted with EtOAc. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified with flash chromatography (20% EtOAc in hexane) to give <b>55</b> as a white solid (43 mg, 80%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 6.86 (s, 1H), 6.53 (s, 1H), 6.00 (s, 1H), 4.94 (s, 2H), 2.30 (s, 3H), 1.53 (s, 9H).</div></div><div id="sec4_1_13_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i121" class="anchor-spacer"></div><h5 class="article-section__title" id="_i121"> 4-Bromo-<i>N</i>-((3-methylisoxazol-5-yl)methyl)thiophen-2-amine (<b>56</b>)</h5><div class="NLM_p last">To a solution of <b>55</b> in DCM (1.5 mL) was added TFA (0.3 mL) dropwise. The mixture was stirred at room temperature for 3 h and concentrated. <b>56</b> was obtained as a colorless gel, which was directly used in the next step.</div></div><div id="sec4_1_13_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> <i>N</i>-(4-Bromothiophen-2-yl)-<i>N</i>-((3-methylisoxazol-5-yl)methyl)acrylamide (<b>57</b>)</h5><div class="NLM_p last">A solution of acryloyl chloride (21 mg, 0.23 mmol), <b>56</b> (31 mg, 0.12 mmol), and DIEA (44 mg, 0.35 mmol) in toluene (2 mL) was heated at 70 °C for 1 h. The mixture was concentrated, and the residue was purified with flash chromatography (20% EtOAc in hexane) to give <b>57</b> as a white solid (19 mg, 51% yield over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.19 (s, 1H), 6.82 (s, 1H), 6.49 (d, <i>J</i> = 16.7 Hz, 1H), 6.32 (d, <i>J</i> = 12.0 Hz, 1H), 6.11 (s, 1H), 5.73 (d, <i>J</i> = 10.3 Hz, 1H), 4.97 (s, 2H), 2.30 (s, 3H).</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> Molecular Modeling</h3><div class="NLM_p last">The published GSTO1 structures (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV">4YQV</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM">4YQM</a>) and in-house structures were used for the modeling of potential inhibitor binding modes from the fragment and previously screening results, respectively. All crystallographic water molecules were deleted for calculations, as they were not deemed to be possible to confidently predict important water molecules up front across the different series modeled. Protein alignment and hydrophobic surface calculations were conducted in Maestro.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Molecular graphics and analyses were performed with UCSF Chimera, developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco, with support from NIH P41-GM103311.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> 3D small molecule structures and distance measurements were generated in Maestro. Crystal structures were prepared for docking using the protein preparation wizard in Maestro, which optimizes hydrogen placements. The active site was defined by the bound ligand. Glide Covalent Docking Protocol<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was used for covalent ligand docking to target Cys32. Multiple tautomers and protonation states were enumerated for ligands using LigPrep.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Docked binding modes were ranked using the docking score (GlideScore35) and visually inspected for the retention of key interactions along with other close contacts. Binding modes were deprioritized when the conformation of the docked molecule was considered to be unsatisfactory.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> GSTO1-1 Expression and Purification</h3><div class="NLM_p last">GSTO1-1 [residues 1–241 bearing an N-terminal 6x histidine tag and a tobacco etch virus (TEV) protease cleavage site] was expressed and purified as described previously.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> In-Gel Fluorescence Binding Assay</h3><div class="NLM_p last">The inhibitory activity of the compounds was determined using a competitive inhibition of CMFDA binding to GSTO1. Briefly, an in vitro binding assay with purified GSTO1-1 was performed using 1 μM GSTO1-1 in a reaction buffer [100 mM Tris (pH 8.0), 1.5 mM ethylenediaminetetraacetic acid (EDTA), and 1 μM dithiothreitol (DTT)] incubated with compounds for 30 min at 37 <b>°</b>C, followed by addition of CMFDA (500 nM) for 30 min. The reaction was quenched with a Laemmli sample buffer and resolved on a 4–20% polyacrylamide gel. Gels were immediately scanned on an i-Bright imaging system (Thermo Fisher Scientific). Quantification of fluorescent band intensity was done using ImageJ software. % decrease in band intensity is equivalent to the % inhibition of CMFDA binding to the GSTO1 upon compound addition. Hence, % decrease in band intensity is the measurement of the inhibitory effect of GSTO1 because of binding of the compound to GSTO1.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i126" class="anchor-spacer"></div><h3 class="article-section__title" id="_i126"> 4NPG Reductase Assay<a onclick="showRef(event, 'ref4 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref6">(4,6)</a></h3><div class="NLM_p last">Inhibition of GSTO1 enzyme activity was measured by monitoring the reduction of 4-NPG to 4-nitroacetophenone. Briefly, in a 200 mL reaction volume, 5 μg/mL purified GSTO1 in reaction buffer [100 mM Tris (pH 8.0), 1.5 mM EDTA, and 1 mM DTT] was incubated with DMSO or different concentrations of inhibitors for 30 min at 37 °C. A unit of 1 mM 4-NPG was added to the reaction, and decrease in absorbance at 305 nm was recorded on Synergy H1 plate reader (Biotek).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i127" class="anchor-spacer"></div><h3 class="article-section__title" id="_i127"> Cell Lines</h3><div class="NLM_p last">Colon cancer cell line HCT116 was generously provided by Dr. Bert Vogelstein, Johns Hopkins Medical Institutions, Baltimore, MD. HT29 cell line was purchased from the American Type Culture Collection (Manassas, VA). Cell lines were cultured in RPMI 1640 media supplemented with 10% fetal bovine serum at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub>. All cell lines were maintained in culture under 30 passages and tested regularly for <i>Mycoplasma</i> contamination using PlasmoTest (InvivoGen, San Diego, CA, USA).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i128" class="anchor-spacer"></div><h3 class="article-section__title" id="_i128"> Cell Viability Assays</h3><div class="NLM_p last">Cell proliferation was assessed by the MTT assay. Cancer cells were seeded in 96-well microtiter plates and, after overnight attachment, treated with synthesized compounds. After 72 h, MTT solution (3 mg/mL) was added to each well, and cells were incubated for 3 h at 37 °C. After incubation, media from each well were removed, and the dark blue formazan crystals formed by live cells were dissolved in DMSO (100 μL per well). The absorbance intensity was measured at 570 nm on a microplate reader (Molecular Devices, Sunnyvale, CA, USA).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i129" class="anchor-spacer"></div><h3 class="article-section__title" id="_i129"> Crystallization, Data Collection, and Structure Determination</h3><div class="NLM_p">The GSTO1:<b>18</b> complex was crystallized by mixing GSTO1-1 (2.66 mg/mL in 25 mM Tris pH 7.5, 60 mM NaCl, and 1 mM DTT) with <b>18</b> in a 1:1.5 molar ratio and incubating at 4 °C for 48 h. Protein was concentrated to 22 mg/mL and crystallized by sitting drop vapor diffusion at 20 °C by mixing protein in a 1:1 ratio with well solution containing 100 mM 2-(N-morpholino)ethanesulfonic acid (MES) pH 6.5, 25% poly(ethylene glycol) (PEG)-3350, and 3% methanol. Crystals were cryoprotected in a solution containing 100 mM MES pH 6.5 and 30% PEG-3350 and flash-frozen in liquid nitrogen.</div><div class="NLM_p">The GSTO1:<b>37</b> and GSTO1:<b>44</b> complexes were crystallized by mixing concentrated GSTO1 (18.5–23 mg/mL in 25 mM Tris pH 7.5, 60 mM NaCl, and 1 mM DTT) in a 1:1.5 or 1:1.8 protein to inhibitor ratio with inhibitors <b>37</b> and <b>44</b>, respectively. Protein was incubated with an inhibitor at 4 °C for 1 h prior to crystallization by sitting drop vapor diffusion at 16 °C. Crystals formed in drops containing protein in a 1:1 ratio with well solution containing 100 mM MES pH 6.5, 30% PEG-3350, and either 3% (+/−)-2-methyl-2,4-pentanediol for GSTO1:<b>37</b> or 4% acetone for GSTO1:<b>44</b>. After 24 h, crystals were cryoprotected in a solution containing 100 mM MES pH 6.5, 30% PEG-3350, and 1.25 mM inhibitor and then super-cooled in liquid nitrogen.</div><div class="NLM_p">Diffraction data were collected on Advanced Photon Source LS-CAT beamlines 21-ID-G (GSTO-1 complexes with <b>18</b> and <b>37</b>) or 21-ID-D (GSTO1:<b>44</b>) as described in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Table 1</a>. All data were processed using HKL2000,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and all three structures were solved via molecular replacement using a published structure of a GSTO1/inhibitor complex sans ligand (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQU">4YQU</a>).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Molecular replacement was performed in MOLREP<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (GSTO1:<b>18</b>) or Phaser<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (GSTO1:<b>37</b> and GSTO1:<b>44</b>). Iterative model building and refinement were performed using COOT<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and BUSTER,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> respectively. Coordinates and geometric restraints for each ligand were generated in Grade.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> All structures were validated using Molprobity<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and the PDB validation server.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> All calculated RMSD values are based on SSM superpositioning in COOT.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">The GSTO1 structure bound to <b>18</b> was solved to 2.75 Å in the space group P1. The structure includes six chains of protein per asymmetric unit with an average RMSD of 0.47 Å between chains. Residues 6 through 241 were modeled in each chain with the exception of a disordered loop that varied in length between chains (133–135 in chains A, B, and D, 134–135 in chain C, and 133–136 in chains E and F). In chain E, a number of additional residues including 10–12, 57–58, and 234–235 were not modeled because of disorder. Difference electron density maps supported covalent modification of residue Cys32 by inhibitor <b>18</b> in all six chains (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S1A</a>); however, weaker electron density in chain F allowed only partial modeling of the inhibitor. The electron density corresponding to MES, a component of the crystallization solution and cryoprotectant, was also observed in four of the six chains.</div><div class="NLM_p last">The GSTO1 structures with <b>37</b> and <b>44</b> were solved in the space group P12<sub>1</sub>1 at resolutions of 2.0 and 2.15 Å, respectively. Both structures contained two molecules of protein per asymmetric unit with an RMSD of 0.48 Å between chains. In both structures, residues 1–241 are modeled in each chain, as well as three residues (SNA) associated with the N-terminal TEV cleavage site. Difference electron density maps supported covalent modification of the GSTO1 residue Cys32 by <b>37</b> or <b>44</b> in both chains of their corresponding cocrystal structures (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">Figure S1B,C</a>). In the structure of GSTO1 with <b>37</b>, the methyl ester substituent was not ordered. As with the GSTO1:<b>18</b> structure, the electron density corresponding to MES was observed in both chains of the GSTO1 structures with <b>37</b> and <b>44</b>. In the structure of GSTO1:<b>44</b>, the electron density on the surface of both protein chains supported modeling a monomer of PEG.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> Microsome Stability</h3><div class="NLM_p last">The microsome incubation system was prepared as follows. Microsome (10 μL, 20 mg/mL) was diluted with 330 μL 0.1 M phosphate buffer (3.3 mM MgCl<sub>2</sub>), and 40 μL of 10 μM test compound phosphate buffer solution was added. The prepared master solution was prewarmed in 37 °C for 3 min. NADPH (3.5 mg) was dissolved in 210 μL of 0.1 M phosphate buffer (3.3 mM MgCl<sub>2</sub>). Then 20 μL NADPH was added to the above master solution to initiate the reaction for 0, 5, 10, 15, 30, 45, and 60 min. The final concentration of the test compound in the reaction system was 1 μM. An aliquot of 40 μL was pipetted from the reaction solution and stopped by the addition of 4 volumes of cold acetonitrile containing 50 nM of compounds <b>18</b> and <b>49</b> as an internal standard (IS) at the designated time points. The incubation solution was centrifuged at 3500<i>g</i> for 10 min to precipitate the protein. The supernatant was used for LC/MS/MS analysis. The natural log peak area ratio (compound peak area/IS peak area) was plotted against time, and the gradient of the line was determined.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01960" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05440" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05440" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01960" class="ext-link">10.1021/acs.jmedchem.8b01960</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings including screening data, X-ray diffraction data, NMR spectra, and HPLC chromatograms (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB ID of new crystal (X-ray) structures/homology models: GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB">6MHB</a>), GSTO1:<b>37</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC">6MHC</a>), and GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf">jm8b01960_si_001.pdf (25.32 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_002.csv">jm8b01960_si_002.csv (3.58 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01960" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03787" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03787" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Wu</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Drug Targeting and Drug Delivery System of Ministry
of Education, West China School of Pharmacy, Sichuan University, No. 17 People’s South Road, Chengdu 610041, P. R. China</span>; 
    <span>Present Address:
                        Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, No. 17 People’s South Road, Chengdu, 610041, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0719-4963" title="Orcid link">http://orcid.org/0000-0003-0719-4963</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#26515f4948416655455308434253084548"><span class="__cf_email__" data-cfemail="8bfcf2e4e5eccbf8e8fea5eeeffea5e8e5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nouri Neamati</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span>; 
    <span>Present Address:
                        North Campus Research Complex, Bldg 520, 1600 Huron Parkway, Ann Arbor, MI 48109, USA</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#563833373b37223f163b333278233b3f353e78333223"><span class="__cf_email__" data-cfemail="640a01050905100d240901004a11090d070c4a010011">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weiyang Dai</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, 
     and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Key
Laboratory of Drug Targeting and Drug Delivery System of Ministry
of Education, West China School of Pharmacy, Sichuan University, No. 17 People’s South Road, Chengdu 610041, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3222-4661" title="Orcid link">http://orcid.org/0000-0002-3222-4661</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Soma Samanta</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ding Xue</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elyse M. Petrunak</span> - <span class="hlFld-Affiliation affiliation">Life
Sciences Institute and Department of Biological Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne A. Stuckey</span> - <span class="hlFld-Affiliation affiliation">Life
Sciences Institute and Department of Biological Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanyan Han</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical Sciences, College
of Pharmacy and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical Sciences, College
of Pharmacy and , University of Michigan, North Campus Research Complex, 1600
Huron Parkway, Ann Arbor 48109, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i134">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45854" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45854" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported in part through a University of Michigan Developmental Research/GI SPORE (P50 CA130810) Pilot Award. N.N. and J.A.S. are grateful for the support of this work from NIH P30 CA046592. W.D. was supported by China Scholarship Council (201606240054). Use of the Advanced Photon Source was supported by the U. S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under contract no. DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor for the support of this research program (Grant 085P1000817).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">GSTO1</td><td class="NLM_def"><p class="first last">glutathione S-transferase omega 1</p></td></tr><tr><td class="NLM_term">CRID</td><td class="NLM_def"><p class="first last">cytokine release inhibitory drug</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">CMFDA</td><td class="NLM_def"><p class="first last">5-chloromethylfluorescein diacetate</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)-carbodiimide</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">BBr<sub>3</sub></td><td class="NLM_def"><p class="first last">boron tribromide</p></td></tr><tr><td class="NLM_term">MgSO<sub>4</sub></td><td class="NLM_def"><p class="first last">magnesium sulfate</p></td></tr><tr><td class="NLM_term">NaHCO<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium bicarbonate</p></td></tr><tr><td class="NLM_term">K<sub>2</sub>CO<sub>3</sub></td><td class="NLM_def"><p class="first last">potassium carbonate</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrogen chloride</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i136">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowsey, I. R.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferases</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.45.120403.095857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1146%2Fannurev.pharmtox.45.120403.095857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=15822171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=51-88&author=J.+D.+Hayesauthor=J.+U.+Flanaganauthor=I.+R.+Jowsey&title=Glutathione+transferases&doi=10.1146%2Fannurev.pharmtox.45.120403.095857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases</span></div><div class="casAuthors">Hayes, John D.; Flanagan, Jack U.; Jowsey, Ian R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-88, 1 plate</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The authors describe the 3 mammalian glutathione S-transferase (GST) families, namely cytosolic, mitochondrial, and microsomal GSTs, the latter now designated MAPEG (Membrane-Assocd. Proteins in Eicosanoid and Glutathione metab.).  In addn. to detoxifying electrophilic xenobiotics, such as chem. carcinogens, environmental pollutants, and antitumor agents, these GSTs inactivate endogenous α,β-unsatd. aldehydes, quinones, epoxides, and hydroperoxides formed as secondary metabolites during oxidative stress.  These enzymes are also intimately involved in the biosynthesis of leukotrienes, prostaglandins, testosterone, and progesterone, as well as the degrdn. of tyrosine.  Among their substrates, GSTs conjugate the signaling mols., 15-deoxy-Δ12,14-prostaglandin J2 (15d-PGJ2) and 4-hydroxynonenal, with glutathione, and consequently they antagonize expression of genes trans-activated by peroxisome proliferator-activated receptor γ (PPARγ) and nuclear factor-erythroid 2 p45-related factor 2 (Nrf2).  Through metab. of 15d-PGJ2, GST may enhance gene expression driven by nuclear factor-κB (NF-κB).  Cytosolic human GST exhibit genetic polymorphisms and this variation can increase susceptibility to carcinogenesis and inflammatory disease.  Polymorphisms in human MAPEG are assocd. with alterations in lung function and increased risk of myocardial infarction and stroke.  Targeted disruption of murine genes has demonstrated that cytosolic GST isoenzymes are broadly cytoprotective, whereas MAPEG proteins have pro-inflammatory activities.  Furthermore, knockout of mouse GSTA4 and GSTZ1 leads to overexpression of transferases in the Alpha, Mu, and Pi classes, an observation suggesting they are part of an adaptive mechanism that responds to endogenous chem. cues such as 4-hydroxynonenal and tyrosine degrdn. products.  Consistent with this hypothesis, the promoters of cytosolic GST and MAPEG genes contain antioxidant response elements through which they are transcriptionally activated during exposure to Michael reaction acceptors and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH__KbntnGGrVg90H21EOLACvtfcHk0ljmKEhwpi10tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrk%253D&md5=c74f17b1381f046770a7f6faf5c51291</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.45.120403.095857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.45.120403.095857%26sid%3Dliteratum%253Aachs%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DFlanagan%26aufirst%3DJ.%2BU.%26aulast%3DJowsey%26aufirst%3DI.%2BR.%26atitle%3DGlutathione%2520transferases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D45%26spage%3D51%26epage%3D88%26doi%3D10.1146%2Fannurev.pharmtox.45.120403.095857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferases, regulators of cellular metabolism and physiology</span>. <i>BBA-Gen Subjects</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1830</i></span>,  <span class="NLM_fpage">3267</span>– <span class="NLM_lpage">3288</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2012.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.bbagen.2012.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=23201197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1830&publication_year=2013&pages=3267-3288&author=P.+G.+Boardauthor=D.+Menon&title=Glutathione+transferases%2C+regulators+of+cellular+metabolism+and+physiology&doi=10.1016%2Fj.bbagen.2012.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases, regulators of cellular metabolism and physiology</span></div><div class="casAuthors">Board, Philip G.; Menon, Deepthi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1830</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3267-3288</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The cytosolic glutathione S-transferases (GSTs) comprise a superfamily of proteins that can be categorized into multiple classes with a mixt. of highly specific and overlapping functions.  Here, the authors cover the genetics, structure, and function of human cytosolic GSTs with particular attention to their emerging roles in cellular metab.  All of the catalytically active GSTs contribute to the glutathione conjugation or glutathione dependant-biotransformation of xenobiotics, and many catalyze glutathione peroxidase or thiol transferase reactions.  GSTs also catalyze glutathione-dependent isomerization reactions required for the synthesis of several prostaglandins and steroid hormones and the catabolism of tyrosine.  An increasing body of work has implicated several GSTs in the regulation of cell signaling pathways mediated by stress-activated kinases like JNK kinase.  In addn., some members of the cytosolic GST family have been shown to form ion channels in intracellular membranes and to modulate ryanodine receptor Ca2+ channels in skeletal and cardiac muscle.  In addn. to their well established roles in the conjugation and biotransformation of xenobiotics, GSTs have emerged as significant regulators of pathways detg. cell proliferation and survival and as regulators of ryanodine receptors that are essential for muscle function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoECjoqUckeMrVg90H21EOLACvtfcHk0ljmKEhwpi10tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFSluw%253D%253D&md5=d0cb57c5d49fc78e6bac46ff7f60104f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2012.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2012.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DMenon%26aufirst%3DD.%26atitle%3DGlutathione%2520transferases%252C%2520regulators%2520of%2520cellular%2520metabolism%2520and%2520physiology%26jtitle%3DBBA-Gen%2520Subjects%26date%3D2013%26volume%3D1830%26spage%3D3267%26epage%3D3288%26doi%3D10.1016%2Fj.bbagen.2012.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.46.120604.141046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1146%2Fannurev.pharmtox.46.120604.141046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=16968214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=89-116&author=T.+W.+Kenslerauthor=N.+Wakabayashiauthor=S.+Biswal&title=Cell+survival+responses+to+environmental+stresses+via+the+Keap1-Nrf2-ARE+pathway&doi=10.1146%2Fannurev.pharmtox.46.120604.141046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span></div><div class="casAuthors">Kensler, Thomas W.; Wakabayashi, Nobunao; Biswal, Shyam</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-116</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Keap1-Nrf2-ARE signaling plays a significant role in protecting cells from endogenous and exogenous stresses.  The development of Nrf2 knockout mice has provided key insights into the toxicol. importance of this pathway.  These mice are more sensitive to the hepatic, pulmonary, ovarian, and neurotoxic consequences of acute exposures to environmental agents and drugs, inflammatory stresses, as well as chronic exposures to cigarette smoke and other carcinogens.  Under quiescent conditions, the transcription factor Nrf2 interacts with the actin-anchored protein Keap1, largely localized in the cytoplasm.  This quenching interaction maintains low basal expression of Nrf2-regulated genes.  However, upon recognition of chem. signals imparted by oxidative and electrophilic mols., Nrf2 is released from Keap1, escapes proteasomal degrdn., translocates to the nucleus, and transactivates the expression of several dozen cytoprotective genes that enhance cell survival.  This review highlights the key elements in this adaptive response to protection against acute and chronic cell injury provoked by environmental stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAmRpcXFLXq7Vg90H21EOLACvtfcHk0ljmKEhwpi10tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D&md5=eee9260ebe6186b82da647701a10c7e2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.46.120604.141046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.46.120604.141046%26sid%3Dliteratum%253Aachs%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DCell%2520survival%2520responses%2520to%2520environmental%2520stresses%2520via%2520the%2520Keap1-Nrf2-ARE%2520pathway%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2007%26volume%3D47%26spage%3D89%26epage%3D116%26doi%3D10.1146%2Fannurev.pharmtox.46.120604.141046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramkumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziemke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, H.</span></span> <span> </span><span class="NLM_article-title">Mechanistic evaluation and transcriptional signature of a glutathione S-transferase omega 1 inhibitor</span>. <i>Nat. Commu.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13084</span>, <span class="refDoi"> DOI: 10.1038/ncomms13084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fncomms13084" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13084&author=K.+Ramkumarauthor=S.+Samantaauthor=A.+Kyaniauthor=S.+Yangauthor=S.+Tamuraauthor=E.+Ziemkeauthor=J.+A.+Stuckeyauthor=S.+Liauthor=K.+Chinnaswamyauthor=H.+Otake&title=Mechanistic+evaluation+and+transcriptional+signature+of+a+glutathione+S-transferase+omega+1+inhibitor&doi=10.1038%2Fncomms13084"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fncomms13084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13084%26sid%3Dliteratum%253Aachs%26aulast%3DRamkumar%26aufirst%3DK.%26aulast%3DSamanta%26aufirst%3DS.%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DZiemke%26aufirst%3DE.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DOtake%26aufirst%3DH.%26atitle%3DMechanistic%2520evaluation%2520and%2520transcriptional%2520signature%2520of%2520a%2520glutathione%2520S-transferase%2520omega%25201%2520inhibitor%26jtitle%3DNat.%2520Commu.%26date%3D2016%26volume%3D7%26spage%3D13084%26doi%3D10.1038%2Fncomms13084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">The omega-class glutathione transferases: structure, function, and genetics</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">235</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.561353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.3109%2F03602532.2011.561353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=21495794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=226-235&author=P.+G.+Board&title=The+omega-class+glutathione+transferases%3A+structure%2C+function%2C+and+genetics&doi=10.3109%2F03602532.2011.561353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The omega-class glutathione transferases: structure, function, and genetics</span></div><div class="casAuthors">Board, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">226-235</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The omega class of glutathione S-transferases (GSTs) is a relatively ancient member of the cytosolic GST superfamily, and the omega-class GSTs are found in plants, animals, and some microbial species.  The omega-class GSTs exhibit the canonical GST fold, but, unlike other GSTs, the omega-class GSTs have a Cys residue in their active site.  Consequently, the omega-class GSTs catalyze a range of thiol transferase and redn. reactions that are not catalyzed by members of the other classes.  Human GSTO1-1 can catalyze the redn. of monomethylarsonic acid(V), but this does not appear to be physiol. important in cases of high environmental arsenic exposure.  GSTO1-1 also plays an important role in the biotransformation of reactive α-haloketones to nontoxic acetophenones.  Genetic variation is common in the omega-class GST genes, and variants that result in deficiency of GSTO1-1 have been characterized.  Genetic linkage studies have discovered assocns. between GSTO genes and the age at onset of Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis.  The mechanism underlying this assocn. with neurol. diseases may derive from the capacity of omega-class GSTs to mitigate oxidative stress or their role in activating the proinflammatory cytokine, interleukin-1β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw-fnr-XyZH7Vg90H21EOLACvtfcHk0lhDKiLWPzmXKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCisbo%253D&md5=9653a63761ddfb11cfc959f7e100cac9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.561353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.561353%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DThe%2520omega-class%2520glutathione%2520transferases%253A%2520structure%252C%2520function%252C%2520and%2520genetics%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2011%26volume%3D43%26spage%3D226%26epage%3D235%26doi%3D10.3109%2F03602532.2011.561353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappello, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, M. W.</span></span> <span> </span><span class="NLM_article-title">S-(4-Nitrophenacyl)glutathione is a specific substrate for glutathione transferase omega 1-1</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2007.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ab.2007.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=18028863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVyhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2008&pages=25-30&author=P.+G.+Boardauthor=M.+Cogganauthor=J.+Cappelloauthor=H.+Zhouauthor=A.+J.+Oakleyauthor=M.+W.+Anders&title=S-%284-Nitrophenacyl%29glutathione+is+a+specific+substrate+for+glutathione+transferase+omega+1-1&doi=10.1016%2Fj.ab.2007.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">S-(4-Nitrophenacyl)glutathione is a specific substrate for glutathione transferase omega 1-1</span></div><div class="casAuthors">Board, Philip G.; Coggan, Marjorie; Cappello, Jean; Zhou, Huina; Oakley, Aaron J.; Anders, M. W.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-30</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Glutathione transferase omega 1-1 (GSTO1-1) catalyzes the biotransformation of arsenic and is implicated as a factor influencing the age-at-onset of Alzheimer's disease and the posttranslational activation of interleukin 1β (IL-1β).  Investigation of the biol. role of GSTO1-1 variants has been hampered by the lack of a specific assay for GSTO1-1 activity in tissue samples that contain other GSTs and other enzymes with similar catalytic specificities.  Previous studies have shown that GSTO1-1 catalyzes the redn. of S-(phenacyl)glutathiones to acetophenones.  A new substrate, S-(4-nitrophenacyl)glutathione (4NPG), has been prepd. and found to have a high turnover with GSTO1-1 but negligible activity with GSTO2-2 and other members of the glutathione transferase superfamily.  A spectrophotometric assay with 4NPG as a substrate has been used to det. GSTO1-1 activity in several human breast cancer cell lines and in mouse liver and brain tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs5SUvAbtGrbVg90H21EOLACvtfcHk0lhDKiLWPzmXKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVyhtb8%253D&md5=afc7bd7570bbae5b095c993edc3bb1ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2007.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2007.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DCappello%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DAnders%26aufirst%3DM.%2BW.%26atitle%3DS-%25284-Nitrophenacyl%2529glutathione%2520is%2520a%2520specific%2520substrate%2520for%2520glutathione%2520transferase%2520omega%25201-1%26jtitle%3DAnal.%2520Biochem.%26date%3D2008%26volume%3D374%26spage%3D25%26epage%3D30%26doi%3D10.1016%2Fj.ab.2007.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelvanayagam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easteal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermiin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danley, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span> <span> </span><span class="NLM_article-title">Identification, characterization, and crystal structure of the Omega class glutathione transferases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">24798</span>– <span class="NLM_lpage">24806</span>, <span class="refDoi"> DOI: 10.1074/jbc.m001706200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1074%2Fjbc.m001706200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=24798-24806&author=P.+G.+Boardauthor=M.+Cogganauthor=G.+Chelvanayagamauthor=S.+Eastealauthor=L.+S.+Jermiinauthor=G.+K.+Schulteauthor=D.+E.+Danleyauthor=L.+R.+Hothauthor=M.+C.+Grifforauthor=A.+V.+Kamath&title=Identification%2C+characterization%2C+and+crystal+structure+of+the+Omega+class+glutathione+transferases&doi=10.1074%2Fjbc.m001706200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m001706200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m001706200%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DChelvanayagam%26aufirst%3DG.%26aulast%3DEasteal%26aufirst%3DS.%26aulast%3DJermiin%26aufirst%3DL.%2BS.%26aulast%3DSchulte%26aufirst%3DG.%2BK.%26aulast%3DDanley%26aufirst%3DD.%2BE.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DGriffor%26aufirst%3DM.%2BC.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26atitle%3DIdentification%252C%2520characterization%252C%2520and%2520crystal%2520structure%2520of%2520the%2520Omega%2520class%2520glutathione%2520transferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D24798%26epage%3D24806%26doi%3D10.1074%2Fjbc.m001706200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitbread, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masoumi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetlow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmuck, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">Characterization of the omega class of glutathione transferases</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>401</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(05)01005-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fs0076-6879%2805%2901005-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2005&pages=78-99&author=A.+K.+Whitbreadauthor=A.+Masoumiauthor=N.+Tetlowauthor=E.+Schmuckauthor=M.+Cogganauthor=P.+G.+Board&title=Characterization+of+the+omega+class+of+glutathione+transferases&doi=10.1016%2Fs0076-6879%2805%2901005-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fs0076-6879%2805%2901005-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0076-6879%252805%252901005-0%26sid%3Dliteratum%253Aachs%26aulast%3DWhitbread%26aufirst%3DA.%2BK.%26aulast%3DMasoumi%26aufirst%3DA.%26aulast%3DTetlow%26aufirst%3DN.%26aulast%3DSchmuck%26aufirst%3DE.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DCharacterization%2520of%2520the%2520omega%2520class%2520of%2520glutathione%2520transferases%26jtitle%3DMethods%2520Enzymol.%26date%3D2005%26volume%3D401%26spage%3D78%26epage%3D99%26doi%3D10.1016%2Fs0076-6879%2805%2901005-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Disparate proteome reactivity profiles of carbon electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">405</span>, <span class="refDoi"> DOI: 10.1038/nchembio.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnchembio.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=18488014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=405&author=E.+Weerapanaauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=Disparate+proteome+reactivity+profiles+of+carbon+electrophiles&doi=10.1038%2Fnchembio.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disparate proteome reactivity profiles of carbon electrophiles</span></div><div class="casAuthors">Weerapana, Eranthie; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">405-407</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insights into the proteome reactivity of electrophiles are crucial for designing activity-based probes for enzymes lacking cognate affinity labels.  Here, the authors show that different classes of carbon electrophiles exhibit markedly distinct amino acid labeling profiles in proteomes, ranging from selective reactivity with cysteine to adducts with several amino acids.  These data specify electrophilic chemotypes with restricted and permissive reactivity profiles to guide the design of next-generation functional proteomics probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI4Jh6351SYLVg90H21EOLACvtfcHk0lhwGilIoRCW8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D&md5=69eb0ae589c50f46a8695dc6731366e2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.91%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDisparate%2520proteome%2520reactivity%2520profiles%2520of%2520carbon%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D405%26doi%3D10.1038%2Fnchembio.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Quantitative reactivity profiling predicts functional cysteines in proteomes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">790</span>, <span class="refDoi"> DOI: 10.1038/nature09472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnature09472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=21085121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=790&author=E.+Weerapanaauthor=C.+Wangauthor=G.+M.+Simonauthor=F.+Richterauthor=S.+Khareauthor=M.+B.+D.+Dillonauthor=D.+A.+Bachovchinauthor=K.+Mowenauthor=D.+Bakerauthor=B.+F.+Cravatt&title=Quantitative+reactivity+profiling+predicts+functional+cysteines+in+proteomes&doi=10.1038%2Fnature09472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative reactivity profiling predicts functional cysteines in proteomes</span></div><div class="casAuthors">Weerapana, Eranthie; Wang, Chu; Simon, Gabriel M.; Richter, Florian; Khare, Sagar; Dillon, Myles B. D.; Bachovchin, Daniel A.; Mowen, Kerri; Baker, David; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">790-795</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cysteine is the most intrinsically nucleophilic amino acid in proteins, where its reactivity is tuned to perform diverse biochem. functions.  The absence of a consensus sequence that defines functional cysteines in proteins has hindered their discovery and characterization.  Here we describe a proteomics method to profile quant. the intrinsic reactivity of cysteine residues en masse directly in native biol. systems.  Hyper-reactivity was a rare feature among cysteines and it was found to specify a wide range of activities, including nucleophilic and reductive catalysis and sites of oxidative modification.  Hyper-reactive cysteines were identified in several proteins of uncharacterized function, including a residue conserved across eukaryotic phylogeny that we show is required for yeast viability and is involved in iron-sulfur protein biogenesis.  We also demonstrate that quant. reactivity profiling can form the basis for screening and functional assignment of cysteines in computationally designed proteins, where it discriminated catalytically active from inactive cysteine hydrolase designs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe4hHN7L4cGbVg90H21EOLACvtfcHk0lhwGilIoRCW8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI&md5=277e436e7a8f4dfff0b548bce52ac40d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09472%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DKhare%26aufirst%3DS.%26aulast%3DDillon%26aufirst%3DM.%2BB.%2BD.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DMowen%26aufirst%3DK.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DQuantitative%2520reactivity%2520profiling%2520predicts%2520functional%2520cysteines%2520in%2520proteomes%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D790%26doi%3D10.1038%2Fnature09472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, R.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.-W.</span></span> <span> </span><span class="NLM_article-title">Comparative proteomic analysis of differentially expressed proteins in human pancreatic cancer tissue</span>. <i>Hepatobiliary Pancreat. Dis. Int.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">200</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=19357035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1Ghtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=193-200&author=J.-H.+Chenauthor=R.-Z.+Niauthor=M.-B.+Xiaoauthor=J.-G.+Guoauthor=J.-W.+Zhou&title=Comparative+proteomic+analysis+of+differentially+expressed+proteins+in+human+pancreatic+cancer+tissue"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative proteomic analysis of differentially expressed proteins in human pancreatic cancer tissue</span></div><div class="casAuthors">Chen, Jian-Hua; Ni, Run-Zhou; Xiao, Ming-Bing; Guo, Ji-Guang; Zhou, Jia-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Hepatobiliary & Pancreatic Diseases International</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-200</span>CODEN:
                <span class="NLM_cas:coden">HPDIAJ</span>;
        ISSN:<span class="NLM_cas:issn">1499-3872</span>.
    
            (<span class="NLM_cas:orgname">First Affiliated Hospital, Zhejiang University School of Medicine</span>)
        </div><div class="casAbstract">Pancreatic cancer is one of the most common malignant tumors.  Early diagnosis of pancreatic cancer is difficult because of the latent onset and lack of good biomarkers.  This study aimed to look for and identify differentially expressed proteins in tissues of pancreatic cancer and adjacent noncancerous tissues by proteomic approaches so as to provide information about possible pancreatic cancer markers and therapeutic targets.  Proteins extd. from 3 paired adjacent noncancerous and cancerous pancreatic tissue specimens were sepd. by two-dimensional gel electrophoresis (2-DE).  The protein spots exhibiting statistical alternations between the two groups through computerized image anal. were then identified by matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS).  In addn., Western blotting and immunohistochem. were performed to verify the expression of certain candidate proteins.  Twelve proteins were significantly upregulated and 4 were downregulated between cancerous and paired adjacent noncancerous pancreatic tissues.  Several proteins (S100A11, Ig gamma-1 chain C region, GSTO1 and peroxiredoxin 4) were found for the first time to be assocd. with pancreatic cancer.  Differential expression of some identified proteins was further confirmed by Western blotting anal. and/or immunohistochem. anal.  Comparative proteomic anal. using 2-DE and MALDI-TOF-MS is an effective method for identifying differentially expressed proteins that may be the potential diagnostic biomarkers and therapeutic targets for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYq2jNzP9Vi7Vg90H21EOLACvtfcHk0lhwGilIoRCW8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1Ghtbk%253D&md5=fa36d1a413dfcacccde4b591abbdd13d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.-H.%26aulast%3DNi%26aufirst%3DR.-Z.%26aulast%3DXiao%26aufirst%3DM.-B.%26aulast%3DGuo%26aufirst%3DJ.-G.%26aulast%3DZhou%26aufirst%3DJ.-W.%26atitle%3DComparative%2520proteomic%2520analysis%2520of%2520differentially%2520expressed%2520proteins%2520in%2520human%2520pancreatic%2520cancer%2520tissue%26jtitle%3DHepatobiliary%2520Pancreat.%2520Dis.%2520Int.%26date%3D2009%26volume%3D8%26spage%3D193%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalowicz, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volker, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pambuccian, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrey, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, P. M.</span></span> <span> </span><span class="NLM_article-title">Identification of a gene expression signature associated with recurrent disease in squamous cell carcinoma of the head and neck</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-03-2144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1158%2F0008-5472.can-03-2144" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=55-63&author=M.+A.+Ginosauthor=G.+P.+Pageauthor=B.+S.+Michalowiczauthor=K.+J.+Patelauthor=S.+E.+Volkerauthor=S.+E.+Pambuccianauthor=F.+G.+Ondreyauthor=G.+L.+Adamsauthor=P.+M.+Gaffney&title=Identification+of+a+gene+expression+signature+associated+with+recurrent+disease+in+squamous+cell+carcinoma+of+the+head+and+neck&doi=10.1158%2F0008-5472.can-03-2144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-03-2144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-03-2144%26sid%3Dliteratum%253Aachs%26aulast%3DGinos%26aufirst%3DM.%2BA.%26aulast%3DPage%26aufirst%3DG.%2BP.%26aulast%3DMichalowicz%26aufirst%3DB.%2BS.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26aulast%3DVolker%26aufirst%3DS.%2BE.%26aulast%3DPambuccian%26aufirst%3DS.%2BE.%26aulast%3DOndrey%26aufirst%3DF.%2BG.%26aulast%3DAdams%26aufirst%3DG.%2BL.%26aulast%3DGaffney%26aufirst%3DP.%2BM.%26atitle%3DIdentification%2520of%2520a%2520gene%2520expression%2520signature%2520associated%2520with%2520recurrent%2520disease%2520in%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520head%2520and%2520neck%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D55%26epage%3D63%26doi%3D10.1158%2F0008-5472.can-03-2144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djukic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pljesa-Ercegovac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suvakov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coric, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragicevic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savic-Radojevic, A.</span></span> <span> </span><span class="NLM_article-title">Upregulated glutathione transferase omega-1 correlates with progression of urinary bladder carcinoma</span>. <i>Redox Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1080/13510002.2017.1299909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1080%2F13510002.2017.1299909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=28288548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlKnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=486-492&author=T.+Djukicauthor=T.+Simicauthor=M.+Pljesa-Ercegovacauthor=M.+Maticauthor=S.+Suvakovauthor=V.+Coricauthor=D.+Dragicevicauthor=A.+Savic-Radojevic&title=Upregulated+glutathione+transferase+omega-1+correlates+with+progression+of+urinary+bladder+carcinoma&doi=10.1080%2F13510002.2017.1299909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulated glutathione transferase omega-1 correlates with progression of urinary bladder carcinoma</span></div><div class="casAuthors">Djukic, Tatjana; Simic, Tatjana; Pljesa-Ercegovac, Marija; Matic, Marija; Suvakov, Sonja; Coric, Vesna; Dragicevic, Dejan; Savic-Radojevic, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Redox Report</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">486-492</span>CODEN:
                <span class="NLM_cas:coden">RDRPE4</span>;
        ISSN:<span class="NLM_cas:issn">1351-0002</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: Newly discovered glutathione transferase omega 1 (GSTO1-1) plays an important role in the glutathionylation cycle, a significant mechanism of protein function regulation.  GSTO1-1 expression pattern has not been studied in transitional cell carcinoma (TCC), as yet.: A total of 56 TCC tumor and corresponding non-tumor specimens were investigated.  Glutathione content and thioltransferase activity were measured spectrophotometrically.  Protein-glutathione mixed disulfides were measured fluorimetrically.  GSTO1-1 expression was detd. by immunoblot and qPCR.  Immunopptn. with GSTO1-1 antibody was followed by immunoblot using anti-GSTO1, GSTP1, c-Jun, JNK, Akt, phospho-Akt, and ASK1 antibody, while for the total S-glutathionylation levels non-reducing electrophoresis was performed.: The contents of reduced glutathione and thioltransferase activity were significantly increased in tumor compared to non-tumor tissue.  The increased GSTO1 expression in tumor tissue showed clear correlation with grade and stage.  However, decreased total protein glutathionylation level in tumor compared to non-tumor samples was found.  Immunopptn. has shown an assocn. of GSTO1-1 with GSTP1, Akt, phospho-Akt, and ASK1 proteins.: GSTO1 deglutathionylase activity suggests its potential important role in redox perturbations present in TCC.  Increased GSTO1-1 expression might contribute to TCC development and/or progression supporting the notion that GSTO1-1 may be a promising novel cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsmBE2HWB_c7Vg90H21EOLACvtfcHk0lgLOtplJdtU2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlKnurY%253D&md5=75be44b48cbd1c6e2d24b1061776b6c3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1080%2F13510002.2017.1299909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13510002.2017.1299909%26sid%3Dliteratum%253Aachs%26aulast%3DDjukic%26aufirst%3DT.%26aulast%3DSimic%26aufirst%3DT.%26aulast%3DPljesa-Ercegovac%26aufirst%3DM.%26aulast%3DMatic%26aufirst%3DM.%26aulast%3DSuvakov%26aufirst%3DS.%26aulast%3DCoric%26aufirst%3DV.%26aulast%3DDragicevic%26aufirst%3DD.%26aulast%3DSavic-Radojevic%26aufirst%3DA.%26atitle%3DUpregulated%2520glutathione%2520transferase%2520omega-1%2520correlates%2520with%2520progression%2520of%2520urinary%2520bladder%2520carcinoma%26jtitle%3DRedox%2520Rep.%26date%3D2017%26volume%3D22%26spage%3D486%26epage%3D492%26doi%3D10.1080%2F13510002.2017.1299909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span> <span> </span><span class="NLM_article-title">Proteomic analysis of Tiam1-mediated metastasis in colorectal cancer</span>. <i>Cell Biol. Int.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">814</span>, <span class="refDoi"> DOI: 10.1016/j.cellbi.2007.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.cellbi.2007.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=17376711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvFSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2007&pages=805-814&author=L.+Liuauthor=L.+Zhaoauthor=Y.+Zhangauthor=Q.+Zhangauthor=Y.+Ding&title=Proteomic+analysis+of+Tiam1-mediated+metastasis+in+colorectal+cancer&doi=10.1016%2Fj.cellbi.2007.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analysis of Tiam1-mediated metastasis in colorectal cancer</span></div><div class="casAuthors">Liu, Li; Zhao, Liang; Zhang, Yanfei; Zhang, Qinling; Ding, Yanqing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biology International</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-814</span>CODEN:
                <span class="NLM_cas:coden">CBIIEV</span>;
        ISSN:<span class="NLM_cas:issn">1065-6995</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tiam1 (T lymphoma invasion and metastasis 1), a guanine nucleotide exchange factor that activates Rac, was recently identified as a novel colorectal cancer metastasis-related gene.  To better understand the mechanism underlying Tiam1-mediated metastasis, we applied two-dimensional polyacrylamide gel electrophoresis (2-DE) and matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF MS) anal. to identify differentially expressed proteins between Tiam1 transfected and mock transfected colorectal cancer HT29 cells.  Eleven differentially expressed proteins were identified and further validated by Western blot and/or real-time PCR.  The results revealed that Tiam1 transfection in colorectal cancer cells could upregulate the expression of Fascin-1, heat shock protein 27 (HSP27), high-mobility group box 1 (HMGB1), glutathione S-transferase omega 1 (GSTO1) and downregulate the expression of annexin IV.  These differentially expressed proteins may be directly or indirectly regulated by Tiam1 and be helpful in studying mechanisms that lead to the function of Tiam1.  These results give some clues to elucidate the mechanism of Tiam1-mediated metastasis for colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojn2cD4feTP7Vg90H21EOLACvtfcHk0lgLOtplJdtU2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvFSjtr8%253D&md5=377fe5a4b54ecc8f29d79fc7f6041e10</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cellbi.2007.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellbi.2007.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DDing%26aufirst%3DY.%26atitle%3DProteomic%2520analysis%2520of%2520Tiam1-mediated%2520metastasis%2520in%2520colorectal%2520cancer%26jtitle%3DCell%2520Biol.%2520Int.%26date%3D2007%26volume%3D31%26spage%3D805%26epage%3D814%26doi%3D10.1016%2Fj.cellbi.2007.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIlwain, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tew, K. D.</span></span> <span> </span><span class="NLM_article-title">Glutathione S-transferase polymorphisms: cancer incidence and therapy</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1639</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1209373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fsj.onc.1209373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=16550164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFyhtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1639&author=C.+C.+McIlwainauthor=D.+M.+Townsendauthor=K.+D.+Tew&title=Glutathione+S-transferase+polymorphisms%3A+cancer+incidence+and+therapy&doi=10.1038%2Fsj.onc.1209373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione S-transferase polymorphisms: cancer incidence and therapy</span></div><div class="casAuthors">McIlwain, C. C.; Townsend, D. M.; Tew, K. D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1639-1648</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The super family of glutathione S-transferases (GSTs) is composed of multiple isoenzymes with significant evidence of functional polymorphic variation.  Over the last three decades, data from cancer studies have linked aberrant expression of GST isoenzymes with the development and expression of resistance to a variety of chems., including cancer drugs.  This review addresses how differences in the human GST isoenzyme expression patterns influence cancer susceptibility, prognosis and treatment.  In addn. to the well-characterized catalytic activity, recent evidence has shown that certain GST isoenzymes can regulate mitogen-activated protein kinases or can facilitate the addn. of glutathione to cysteine residues in target proteins (S-glutathionylation).  These multiple functionalities have contributed to the recent efforts to target GSTs with novel small mol. therapeutics.  Presently, at least two drugs are in late-stage clin. testing.  The evolving functions of GST and their divergent expression patterns in individuals make them an attractive target for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkmAFQdntw6rVg90H21EOLACvtfcHk0lgLOtplJdtU2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFyhtbo%253D&md5=48a8873daaa35f6140dff9fa58ffcaac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1209373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1209373%26sid%3Dliteratum%253Aachs%26aulast%3DMcIlwain%26aufirst%3DC.%2BC.%26aulast%3DTownsend%26aufirst%3DD.%2BM.%26aulast%3DTew%26aufirst%3DK.%2BD.%26atitle%3DGlutathione%2520S-transferase%2520polymorphisms%253A%2520cancer%2520incidence%2520and%2520therapy%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D1639%26doi%3D10.1038%2Fsj.onc.1209373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oguztuzun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güler Simsek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ada, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocdogan, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaygın, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iscan, M.</span></span> <span> </span><span class="NLM_article-title">Expression of CYP and GST in human normal and colon tumor tissues</span>. <i>Biotech. Histochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1080/10520295.2018.1493220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1080%2F10520295.2018.1493220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=30092668" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1-9&author=H.+Bulusauthor=S.+Oguztuzunauthor=G.+G%C3%BCler+Simsekauthor=M.+Kilicauthor=A.+O.+Adaauthor=S.+G%C3%B6lauthor=A.+K.+Kocdoganauthor=P.+Kayg%C4%B1nauthor=B.+Bozerauthor=M.+Iscan&title=Expression+of+CYP+and+GST+in+human+normal+and+colon+tumor+tissues&doi=10.1080%2F10520295.2018.1493220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F10520295.2018.1493220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10520295.2018.1493220%26sid%3Dliteratum%253Aachs%26aulast%3DBulus%26aufirst%3DH.%26aulast%3DOguztuzun%26aufirst%3DS.%26aulast%3DG%25C3%25BCler%2BSimsek%26aufirst%3DG.%26aulast%3DKilic%26aufirst%3DM.%26aulast%3DAda%26aufirst%3DA.%2BO.%26aulast%3DG%25C3%25B6l%26aufirst%3DS.%26aulast%3DKocdogan%26aufirst%3DA.%2BK.%26aulast%3DKayg%25C4%25B1n%26aufirst%3DP.%26aulast%3DBozer%26aufirst%3DB.%26aulast%3DIscan%26aufirst%3DM.%26atitle%3DExpression%2520of%2520CYP%2520and%2520GST%2520in%2520human%2520normal%2520and%2520colon%2520tumor%2520tissues%26jtitle%3DBiotech.%2520Histochem.%26date%3D2018%26spage%3D1%26epage%3D9%26doi%3D10.1080%2F10520295.2018.1493220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piaggi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raggi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitzalis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascherpa, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saviozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casini, A. F.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferase omega 1-1 (GSTO1-1) plays an anti-apoptotic role in cell resistance to cisplatin toxicity</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">811</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1093%2Fcarcin%2Fbgq031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=20106899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVWjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=804-811&author=S.+Piaggiauthor=C.+Raggiauthor=A.+Cortiauthor=E.+Pitzalisauthor=M.+C.+Mascherpaauthor=M.+Saviozziauthor=A.+Pompellaauthor=A.+F.+Casini&title=Glutathione+transferase+omega+1-1+%28GSTO1-1%29+plays+an+anti-apoptotic+role+in+cell+resistance+to+cisplatin+toxicity&doi=10.1093%2Fcarcin%2Fbgq031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferase omega 1-1 (GSTO1-1) plays an anti-apoptotic role in cell resistance to cisplatin toxicity</span></div><div class="casAuthors">Piaggi, Simona; Raggi, Chiara; Corti, Alessandro; Pitzalis, Emanuela; Mascherpa, Marco C.; Saviozzi, Michela; Pompella, Alfonso; Casini, Alessandro F.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">804-811</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Several lines of evidence correlate the overexpression of glutathione S-transferase omega 1-1 (GSTO1-1) with the onset of drug resistance of cancer cells; however, no direct evidence is yet available.  In order to investigate the mechanisms involved, stable transfection with GSTO1-1 complementary DNA was performed in HeLa cells, which spontaneously express very low levels of GSTO1-1.  When transfected cells were seeded at low d., a sharp increase in GSTO1-1 expression was obsd. as compared with controls, along with an increased resistance against cisplatin cytotoxicity.  When seeded at increasing densities, control untransfected cells also presented with an increase in GSTO1-1 expression, again accompanied by cisplatin resistance; the latter was significantly reduced after transfection with GSTO1-1 small interfering RNA.  Cisplatin resistance of transfected cells was not accounted for by changes in the intracellular drug concn. nor in the amt. of DNA cross-links or content of glutathione.  Rather, transfected cells presented with a marked decrease of apoptosis as compared with controls, suggesting that GSTO1-1 overexpression may prevent cisplatin toxicity by interfering with the apoptotic process.  Cisplatin treatment was in fact followed at early times (1-2 h) by activation of both Akt kinase and extracellular signal-regulated kinase (ERK)-1/2 in the transfected cells but not in controls.  Conversely, in transfected cells, the strong activation of Jun N-terminal kinase (JNK)-1 induced by cisplatin at later times (10-20 h) was completely prevented.  In conclusion, GSTO1-1 overexpression appears to be assocd. with activation of survival pathways (Akt and ERK1/2) and inhibition of apoptotic pathways (JNK1), as well as protection against cisplatin-induced apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZYICy71RURrVg90H21EOLACvtfcHk0liTJHYml3kwIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVWjsbo%253D&md5=0856d5d696594a33977e2c04d1a3adc8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq031%26sid%3Dliteratum%253Aachs%26aulast%3DPiaggi%26aufirst%3DS.%26aulast%3DRaggi%26aufirst%3DC.%26aulast%3DCorti%26aufirst%3DA.%26aulast%3DPitzalis%26aufirst%3DE.%26aulast%3DMascherpa%26aufirst%3DM.%2BC.%26aulast%3DSaviozzi%26aufirst%3DM.%26aulast%3DPompella%26aufirst%3DA.%26aulast%3DCasini%26aufirst%3DA.%2BF.%26atitle%3DGlutathione%2520transferase%2520omega%25201-1%2520%2528GSTO1-1%2529%2520plays%2520an%2520anti-apoptotic%2520role%2520in%2520cell%2520resistance%2520to%2520cisplatin%2520toxicity%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D804%26epage%3D811%26doi%3D10.1093%2Fcarcin%2Fbgq031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.-d.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, N.</span></span> <span> </span><span class="NLM_article-title">Identification of platinum-resistance associated proteins through proteomic analysis of human ovarian cancer cells and their platinum-resistant sublines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1021/pr060402r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr060402r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlaru7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=772-780&author=X.-d.+Yanauthor=L.-y.+Panauthor=Y.+Yuanauthor=J.-h.+Langauthor=N.+Mao&title=Identification+of+platinum-resistance+associated+proteins+through+proteomic+analysis+of+human+ovarian+cancer+cells+and+their+platinum-resistant+sublines&doi=10.1021%2Fpr060402r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Platinum-Resistance Associated Proteins through Proteomic Analysis of Human Ovarian Cancer Cells and Their Platinum-Resistant Sublines</span></div><div class="casAuthors">Yan, Xue-dong; Pan, Ling-ya; Yuan, Ye; Lang, Jing-he; Mao, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">772-780</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemoresistance is a major therapeutic obstacle in cancer patients, and the mechanisms of drug resistance are not fully understood.  In the present study, the authors established platinum-resistant human ovarian cancer cell lines and identified differentially expressed proteins related to platinum resistance.  The total proteins of two sensitive (SKOV3 and A2780) and four resistant (SKOV3/CDDP, SKOV3/CBP, A2780/CDDP, and A2780/CBP) human ovarian cancer cell lines were isolated by two-dimensional gel electrophoresis (2-DE).  The differentially expressed proteins were identified using matrix-assisted laser desorption ionization-time-of-flight mass spectrometry (MALDI-TOF MS).  In total, 57 differential protein spots were identified, and five proteins, including annexin A3, destrin, cofilin 1, glutathione-S-transferase omega 1, and cytosolic NADP+-dependent isocitrate dehydrogenase, were found to be coinstantaneous significance compared with their parental cells.  The expression of the five proteins was validated by quant. PCR and Western blot, and the Western blot results showed complete consistency with proteomic techniques.  The five proteins are hopeful to become candidates for platinum resistance.  These may be useful for further study of resistance mechanisms and screening of resistant biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGBj_whnFuILVg90H21EOLACvtfcHk0liHy2h87oqZhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlaru7jO&md5=c4c2852eaf4df3fb4b6a133e4b5a80a9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fpr060402r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr060402r%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.-d.%26aulast%3DPan%26aufirst%3DL.-y.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DLang%26aufirst%3DJ.-h.%26aulast%3DMao%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520platinum-resistance%2520associated%2520proteins%2520through%2520proteomic%2520analysis%2520of%2520human%2520ovarian%2520cancer%2520cells%2520and%2520their%2520platinum-resistant%2520sublines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2007%26volume%3D6%26spage%3D772%26epage%3D780%26doi%3D10.1021%2Fpr060402r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Innes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlstrom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüstle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarotto, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">GSTO1-1 plays a pro-inflammatory role in models of inflammation, colitis and obesity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17832</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-17861-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fs41598-017-17861-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=29259211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjtlehtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=17832&author=D.+Menonauthor=A.+Innesauthor=A.+J.+Oakleyauthor=J.+E.+Dahlstromauthor=L.+M.+Jensenauthor=A.+Br%C3%BCstleauthor=P.+Tummalaauthor=M.+Rookeauthor=M.+G.+Casarottoauthor=J.+B.+Baell&title=GSTO1-1+plays+a+pro-inflammatory+role+in+models+of+inflammation%2C+colitis+and+obesity&doi=10.1038%2Fs41598-017-17861-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">GSTO1-1 plays a pro-inflammatory role in models of inflammation, colitis and obesity</span></div><div class="casAuthors">Menon Deepthi; Innes Ashlee; Jensen Lora M; Brustle Anne; Tummala Padmaja; Rooke Melissa; Casarotto Marco G; Burgio Gaetan; Board Philip G; Menon Deepthi; O'Neill Luke A J; Oakley Aaron J; Dahlstrom Jane E; Baell Jonathan B; Nguyen Nghi; Xie Yiyue; Baell Jonathan B; Cuellar Matthew; Strasser Jessica; Walters Michael A; Dahlin Jayme L</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17832</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glutathione transferase Omega 1 (GSTO1-1) is an atypical GST reported to play a pro-inflammatory role in response to LPS.  Here we show that genetic knockout of Gsto1 alters the response of mice to three distinct inflammatory disease models.  GSTO1-1 deficiency ameliorates the inflammatory response stimulated by LPS and attenuates the inflammatory impact of a high fat diet on glucose tolerance and insulin resistance.  In contrast, GSTO1-1 deficient mice show a more severe inflammatory response and increased escape of bacteria from the colon into the lymphatic system in a dextran sodium sulfate mediated model of inflammatory bowel disease.  These responses are similar to those of TLR4 and MyD88 deficient mice in these models and confirm that GSTO1-1 is critical for a TLR4-like pro-inflammatory response in vivo.  In wild-type mice, we show that a small molecule inhibitor that covalently binds in the active site of GSTO1-1 can be used to ameliorate the inflammatory response to LPS.  Our findings demonstrate the potential therapeutic utility of GSTO1-1 inhibitors in the modulation of inflammation and suggest their possible application in the treatment of a range of inflammatory conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeIFfZgL8lsjl5L8hwNX0fW6udTcc2ebAQSS0e_L4krntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjtlehtg%253D%253D&md5=360b424b4c6c9409e7c2836559f61ff1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-17861-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-17861-6%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DD.%26aulast%3DInnes%26aufirst%3DA.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DDahlstrom%26aufirst%3DJ.%2BE.%26aulast%3DJensen%26aufirst%3DL.%2BM.%26aulast%3DBr%25C3%25BCstle%26aufirst%3DA.%26aulast%3DTummala%26aufirst%3DP.%26aulast%3DRooke%26aufirst%3DM.%26aulast%3DCasarotto%26aufirst%3DM.%2BG.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DGSTO1-1%2520plays%2520a%2520pro-inflammatory%2520role%2520in%2520models%2520of%2520inflammation%252C%2520colitis%2520and%2520obesity%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D17832%26doi%3D10.1038%2Fs41598-017-17861-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laliberte, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perregaux, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peese, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombroski, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabel, C. A.</span></span> <span> </span><span class="NLM_article-title">Glutathiones-transferase omega 1-1 is a target of cytokine release inhibitory drugs and may be responsible for their effect on interleukin-1β posttranslational processing</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">16567</span>– <span class="NLM_lpage">16578</span>, <span class="refDoi"> DOI: 10.1074/jbc.m211596200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1074%2Fjbc.m211596200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=16567-16578&author=R.+E.+Laliberteauthor=D.+G.+Perregauxauthor=L.+R.+Hothauthor=P.+J.+Rosnerauthor=C.+K.+Jordanauthor=K.+M.+Peeseauthor=J.+F.+Egglerauthor=M.+A.+Dombroskiauthor=K.+F.+Geogheganauthor=C.+A.+Gabel&title=Glutathiones-transferase+omega+1-1+is+a+target+of+cytokine+release+inhibitory+drugs+and+may+be+responsible+for+their+effect+on+interleukin-1%CE%B2+posttranslational+processing&doi=10.1074%2Fjbc.m211596200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m211596200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m211596200%26sid%3Dliteratum%253Aachs%26aulast%3DLaliberte%26aufirst%3DR.%2BE.%26aulast%3DPerregaux%26aufirst%3DD.%2BG.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DRosner%26aufirst%3DP.%2BJ.%26aulast%3DJordan%26aufirst%3DC.%2BK.%26aulast%3DPeese%26aufirst%3DK.%2BM.%26aulast%3DEggler%26aufirst%3DJ.%2BF.%26aulast%3DDombroski%26aufirst%3DM.%2BA.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DGabel%26aufirst%3DC.%2BA.%26atitle%3DGlutathiones-transferase%2520omega%25201-1%2520is%2520a%2520target%2520of%2520cytokine%2520release%2520inhibitory%2520drugs%2520and%2520may%2520be%2520responsible%2520for%2520their%2520effect%2520on%2520interleukin-1%25CE%25B2%2520posttranslational%2520processing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D16567%26epage%3D16578%26doi%3D10.1074%2Fjbc.m211596200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vega, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of GlutathioneS-Transferase Omega 1 That Impair Cancer Drug Resistance</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">16605</span>– <span class="NLM_lpage">16616</span>, <span class="refDoi"> DOI: 10.1021/ja2066972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2066972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16605-16616&author=K.+Tsuboiauthor=D.+A.+Bachovchinauthor=A.+E.+Speersauthor=T.+P.+Spicerauthor=V.+Fernandez-Vegaauthor=P.+Hodderauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Potent+and+Selective+Inhibitors+of+GlutathioneS-Transferase+Omega+1+That+Impair+Cancer+Drug+Resistance&doi=10.1021%2Fja2066972"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja2066972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2066972%26sid%3Dliteratum%253Aachs%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DSpicer%26aufirst%3DT.%2BP.%26aulast%3DFernandez-Vega%26aufirst%3DV.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520GlutathioneS-Transferase%2520Omega%25201%2520That%2520Impair%2520Cancer%2520Drug%2520Resistance%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16605%26epage%3D16616%26doi%3D10.1021%2Fja2066972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferase omega 1 is required for the lipopolysaccharide-stimulated induction of NADPH oxidase 1 and the production of reactive oxygen species in macrophages</span>. <i>Free Radic. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2014.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.freeradbiomed.2014.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=24873723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1SksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2014&pages=318-327&author=D.+Menonauthor=R.+Collauthor=P.+G.+Board&title=Glutathione+transferase+omega+1+is+required+for+the+lipopolysaccharide-stimulated+induction+of+NADPH+oxidase+1+and+the+production+of+reactive+oxygen+species+in+macrophages&doi=10.1016%2Fj.freeradbiomed.2014.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferase Omega 1 is required for the lipopolysaccharide-stimulated induction of NADPH oxidase 1 and the production of reactive oxygen species in macrophages</span></div><div class="casAuthors">Menon, Deepthi; Coll, Rebecca; O'Neill, Luke A. J.; Board, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">318-327</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bacterial lipopolysaccharide (LPS) stimulation of macrophages and inflammation via the Toll-like receptor 4 (TLR4) signaling pathway through NF-κB generates reactive oxygen species (ROS) and proinflammatory cytokines such as IL-1β, IL-6, and TNFα.  Because glutathione transferase Omega 1-1 (GSTO1-1) can catalyze redox reactions such as the deglutathionylation of proteins and has also been implicated in the release of IL-1β the authors investigated its role in the development of LPS-mediated inflammation.  The data show that shRNA knockdown of GSTO1-1 in macrophage-like J774.1A cells blocks the expression of NADPH oxidase 1 and the generation of ROS after LPS stimulation.  Similar results were obtained with a GSTO1-1 inhibitor.  To maintain high ROS levels during an inflammatory response, LPS stimulation causes the suppression of enzymes such as catalase and glutathione peroxidase that protect against oxidative stress.  The knockdown of GSTO1-1 also attenuates this response.  The data indicate that GSTO1-1 needs to be catalytically active and mediates its effects on the LPS/TLR4 inflammatory pathway upstream of NF-κB.  These data suggest that GSTO1-1 is a novel target for anti-inflammatory intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZlg072KwT7Vg90H21EOLACvtfcHk0li9Xr19_aCbzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1SksrvE&md5=c7d253152bfa0c535a89f798a47a6f46</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2014.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2014.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DD.%26aulast%3DColl%26aufirst%3DR.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DGlutathione%2520transferase%2520omega%25201%2520is%2520required%2520for%2520the%2520lipopolysaccharide-stimulated%2520induction%2520of%2520NADPH%2520oxidase%25201%2520and%2520the%2520production%2520of%2520reactive%2520oxygen%2520species%2520in%2520macrophages%26jtitle%3DFree%2520Radic.%2520Biol.%2520Med.%26date%3D2014%26volume%3D73%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fj.freeradbiomed.2014.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brighty, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bare, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, E. T.</span></span> <span> </span><span class="NLM_article-title">“Inverse drug discovery” strategy to identify proteins that are targeted by latent electrophiles as exemplified by aryl fluorosulfates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b08366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2017&pages=200-210&author=D.+E.+Boardauthor=G.+J.+Brightyauthor=L.+Plateauthor=G.+Bareauthor=W.+Chenauthor=S.+Liauthor=H.+Wangauthor=B.+F.+Cravattauthor=S.+Forliauthor=E.+T.+Powers&title=%E2%80%9CInverse+drug+discovery%E2%80%9D+strategy+to+identify+proteins+that+are+targeted+by+latent+electrophiles+as+exemplified+by+aryl+fluorosulfates&doi=10.1021%2Fjacs.7b08366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08366%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DD.%2BE.%26aulast%3DBrighty%26aufirst%3DG.%2BJ.%26aulast%3DPlate%26aufirst%3DL.%26aulast%3DBare%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DPowers%26aufirst%3DE.%2BT.%26atitle%3D%25E2%2580%259CInverse%2520drug%2520discovery%25E2%2580%259D%2520strategy%2520to%2520identify%2520proteins%2520that%2520are%2520targeted%2520by%2520latent%2520electrophiles%2520as%2520exemplified%2520by%2520aryl%2520fluorosulfates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D140%26spage%3D200%26epage%3D210%26doi%3D10.1021%2Fjacs.7b08366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppenheimer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, T.</span></span> <span> </span><span class="NLM_article-title">Chemical and computational methods for the characterization of covalent reactive groups for the prospective design of irreversible inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10072</span>– <span class="NLM_lpage">10079</span>, <span class="refDoi"> DOI: 10.1021/jm501412a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501412a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10072-10079&author=M.+E.+Flanaganauthor=J.+A.+Abramiteauthor=D.+P.+Andersonauthor=A.+Aulabaughauthor=U.+P.+Dahalauthor=A.+M.+Gilbertauthor=C.+Liauthor=J.+Montgomeryauthor=S.+R.+Oppenheimerauthor=T.+Ryder&title=Chemical+and+computational+methods+for+the+characterization+of+covalent+reactive+groups+for+the+prospective+design+of+irreversible+inhibitors&doi=10.1021%2Fjm501412a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Computational Methods for the Characterization of Covalent Reactive Groups for the Prospective Design of Irreversible Inhibitors</span></div><div class="casAuthors">Flanagan, Mark E.; Abramite, Joseph A.; Anderson, Dennis P.; Aulabaugh, Ann; Dahal, Upendra P.; Gilbert, Adam M.; Li, Chao; Montgomery, Justin; Oppenheimer, Stacey R.; Ryder, Tim; Schuff, Brandon P.; Uccello, Daniel P.; Walker, Gregory S.; Wu, Yan; Brown, Matthew F.; Chen, Jinshan M.; Hayward, Matthew M.; Noe, Mark C.; Obach, R. Scott; Philippe, Laurence; Shanmugasundaram, Veerabahu; Shapiro, Michael J.; Starr, Jeremy; Stroh, Justin; Che, Ye</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10072-10079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interest in drugs that covalently modify their target is driven by the desire for enhanced efficacy that can result from the silencing of enzymic activity until protein resynthesis can occur, along with the potential for increased selectivity by targeting uniquely positioned nucleophilic residues in the protein.  However, covalent approaches carry addnl. risk for toxicities or hypersensitivity reactions that can result from covalent modification of unintended targets.  Here we describe methods for measuring the reactivity of covalent reactive groups (CRGs) with a biol. relevant nucleophile, glutathione (GSH), along with kinetic data for a broad array of electrophiles.  We also describe a computational method for predicting electrophilic reactivity, which taken together can be applied to the prospective design of thiol-reactive covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzTSCv_Nj6bVg90H21EOLACvtfcHk0li9Xr19_aCbzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM&md5=3ccf59ab7a494655185e5eb5becf8c48</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm501412a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501412a%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DOppenheimer%26aufirst%3DS.%2BR.%26aulast%3DRyder%26aufirst%3DT.%26atitle%3DChemical%2520and%2520computational%2520methods%2520for%2520the%2520characterization%2520of%2520covalent%2520reactive%2520groups%2520for%2520the%2520prospective%2520design%2520of%2520irreversible%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10072%26epage%3D10079%26doi%3D10.1021%2Fjm501412a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>– <span class="NLM_lpage">6262</span>, <span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0li9Xr19_aCbzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummond, K. M.</span></span> <span> </span><span class="NLM_article-title">Covalent modifiers: a chemical perspective on the reactivity of α,β-unsaturated carbonyls with thiols via hetero-michael addition reactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00788</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00788" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=839-885&author=P.+A.+Jacksonauthor=J.+C.+Widenauthor=D.+A.+Harkiauthor=K.+M.+Brummond&title=Covalent+modifiers%3A+a+chemical+perspective+on+the+reactivity+of+%CE%B1%2C%CE%B2-unsaturated+carbonyls+with+thiols+via+hetero-michael+addition+reactions&doi=10.1021%2Facs.jmedchem.6b00788"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00788%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DP.%2BA.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DBrummond%26aufirst%3DK.%2BM.%26atitle%3DCovalent%2520modifiers%253A%2520a%2520chemical%2520perspective%2520on%2520the%2520reactivity%2520of%2520%25CE%25B1%252C%25CE%25B2-unsaturated%2520carbonyls%2520with%2520thiols%2520via%2520hetero-michael%2520addition%2520reactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D60%26spage%3D839%26epage%3D885%26doi%3D10.1021%2Facs.jmedchem.6b00788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teusch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aricioglu, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowski, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neudörfl, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griesbeck, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietsch, M.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of glutathione transferase P1 with trioxane structure as anticancer agents</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201402553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fcmdc.201402553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=25694385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A280%3ADC%252BC2MrmslCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=629-639&author=M.+Br%C3%A4utigamauthor=N.+Teuschauthor=T.+Schenkauthor=M.+Sheikhauthor=R.+Z.+Ariciogluauthor=S.+H.+Borowskiauthor=J.-M.+Neud%C3%B6rflauthor=U.+Baumannauthor=A.+G.+Griesbeckauthor=M.+Pietsch&title=Selective+inhibitors+of+glutathione+transferase+P1+with+trioxane+structure+as+anticancer+agents&doi=10.1002%2Fcmdc.201402553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of glutathione transferase P1 with trioxane structure as anticancer agents</span></div><div class="casAuthors">Brautigam Maria; Teusch Nicole; Schenk Tobias; Sheikh Miriam; Aricioglu Rocky Z; Borowski Swantje H; Neudorfl Jorg-Martin; Baumann Ulrich; Griesbeck Axel G; Pietsch Markus</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">629-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The response to chemotherapy in cancer patients is frequently compromised by drug resistance.  Although chemoresistance is a multifactorial phenomenon, many studies have demonstrated that altered drug metabolism through the expression of phase II conjugating enzymes, including glutathione transferases (GSTs), in tumor cells can be directly correlated with resistance against a wide range of marketed anticancer drugs.  In particular, overexpression of glutathione transferase P1 (GSTP1) appears to be a factor for poor prognosis during cancer therapy.  Former and ongoing clinical trials have confirmed GSTP1 inhibition as a principle for antitumor therapy.  A new series of 1,2,4-trioxane GSTP1 inhibitors were designed via a type II photooxygenation route of allylic alcohols followed by acid-catalyzed peroxyacetalization with aldehydes.  A set of novel inhibitors exhibit low micromolar to high nanomolar inhibition of GSTP1, revealing preliminary SAR for further lead optimization.  Importantly, high selectivity over another two human GST classes (GSTA1 and GSTM2) has been achieved.  The trioxane GSTP1 inhibitors may therefore serve as a basis for the development of novel drug candidates in overcoming chemoresistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpuEoHSV8gu8uCWE5mhx1RfW6udTcc2eaumHYHpZO8Kbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrmslCjtQ%253D%253D&md5=194c663eb80b74f2444ec77f67bc5ce2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402553%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DM.%26aulast%3DTeusch%26aufirst%3DN.%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DSheikh%26aufirst%3DM.%26aulast%3DAricioglu%26aufirst%3DR.%2BZ.%26aulast%3DBorowski%26aufirst%3DS.%2BH.%26aulast%3DNeud%25C3%25B6rfl%26aufirst%3DJ.-M.%26aulast%3DBaumann%26aufirst%3DU.%26aulast%3DGriesbeck%26aufirst%3DA.%2BG.%26aulast%3DPietsch%26aufirst%3DM.%26atitle%3DSelective%2520inhibitors%2520of%2520glutathione%2520transferase%2520P1%2520with%2520trioxane%2520structure%2520as%2520anticancer%2520agents%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D629%26epage%3D639%26doi%3D10.1002%2Fcmdc.201402553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aeluri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polepalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antiproliferative activity of imidazo[1,2-a]pyrimidine Mannich bases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.05.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ejmech.2015.05.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=26067381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVejtbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=18-23&author=R.+Aeluriauthor=M.+Allaauthor=S.+Polepalliauthor=N.+Jain&title=Synthesis+and+antiproliferative+activity+of+imidazo%5B1%2C2-a%5Dpyrimidine+Mannich+bases&doi=10.1016%2Fj.ejmech.2015.05.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antiproliferative activity of imidazo[1,2-a]pyrimidine Mannich bases</span></div><div class="casAuthors">Aeluri, Raghunath; Alla, Manjula; Polepalli, Sowjanya; Jain, Nishant</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-23</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of imidazo[1,2-a]pyrimidine Mannich bases were designed, synthesized in two phases.  Mannich bases were obtained by one pot three component condensation of imidazo[1,2-a]pyrimidine with secondary amine or piperazine and excess of formaldehyde soln. in methanol.  The synthesized Mannich bases were screened for in vitro growth inhibition against a panel of 3 different human cancer cell lines.  Most of the synthesized compds. exhibited antiproliferative activity with GI50 values ranging from 0.01 to 79.4 μM.  Compds. 2-Phenyl-3-(pyrrolidin-1-ylmethyl)imidazo[1,2-a]pyrimidine, ((4-methylpiperazin-1-yl)methyl)-2-phenylimidazo[1,2-a]pyrimidine and 1,4-bis((2-(2-Tolyl)imidazo[1,2-a]pyrimidin-3-yl)methyl)piperazine are effective inhibitors of growth of all cell lines, with GI50 values similar to that of std. drug.  The structure and activity relation was disclosed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr59HvD-9vSybVg90H21EOLACvtfcHk0lirywBhSKNrqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVejtbfJ&md5=98d77ffbe40e8788f58548edbc10cf29</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.05.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.05.037%26sid%3Dliteratum%253Aachs%26aulast%3DAeluri%26aufirst%3DR.%26aulast%3DAlla%26aufirst%3DM.%26aulast%3DPolepalli%26aufirst%3DS.%26aulast%3DJain%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520antiproliferative%2520activity%2520of%2520imidazo%255B1%252C2-a%255Dpyrimidine%2520Mannich%2520bases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D100%26spage%3D18%26epage%3D23%26doi%3D10.1016%2Fj.ejmech.2015.05.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sravanthi, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manju, S. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Fluorescence Properties of Novel indol-3yl-thiazolo[3,2-a][1,3,5]triazines and indole-3-carbaldehyde Schiff Bases</span>. <i>J. Fluoresc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1738</span>, <span class="refDoi"> DOI: 10.1007/s10895-015-1659-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1007%2Fs10895-015-1659-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=26432553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1alu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1727-1738&author=T.+V.+Sravanthiauthor=S.+L.+Manju&title=Synthesis+and+Fluorescence+Properties+of+Novel+indol-3yl-thiazolo%5B3%2C2-a%5D%5B1%2C3%2C5%5Dtriazines+and+indole-3-carbaldehyde+Schiff+Bases&doi=10.1007%2Fs10895-015-1659-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Fluorescence Properties of Novel indol-3yl-thiazolo[3,2-a][1,3,5]triazines and indole-3-carbaldehyde Schiff Bases</span></div><div class="casAuthors">Sravanthi, T. V.; Manju, S. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorescence</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1727-1738</span>CODEN:
                <span class="NLM_cas:coden">JOFLEN</span>;
        ISSN:<span class="NLM_cas:issn">1053-0509</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Novel photoactive 4-(4-chlorophenyl)-2-(1H-indol-3-yl)-6-substituted phenyl-2H-thiazolo[3,2-a][1,3,5]triazines were synthesized by the conjugate addn. of ammonia to the indole-3-carbaldehyde Schiff bases followed by the condensation with 4-chlorobenzaldehyde.  All the synthesized compds. were characterized by FT-IR, NMR, mass spectra and elemental analyses.  Their antioxidant property, electrochem. and photophys. properties in different org. solvents were investigated.  Comparative discussion on the photophys. properties of indole-3-carbaldehyde Schiff bases and 4-(4-chlorophenyl)-2-(1H-indol-3-yl)-6-substituted phenyl-2H-thiazolo[3,2-a][1,3,5]triazines were described.  The fluorescence quantum yield of Schiff bases (φf = 0.66-0.70 in DMSO) found to be interestingly higher.  High fluorescence quantum yield, large molar extinction coeff., high stokes shift and smaller optical band gap positioning these new derivs. as an efficient metal free org. fluorescent and semiconductor material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFcI3h-RDgr7Vg90H21EOLACvtfcHk0ljehV6gJKY0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1alu7rF&md5=094949491a7e875d3970ca854bebea23</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10895-015-1659-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10895-015-1659-1%26sid%3Dliteratum%253Aachs%26aulast%3DSravanthi%26aufirst%3DT.%2BV.%26aulast%3DManju%26aufirst%3DS.%2BL.%26atitle%3DSynthesis%2520and%2520Fluorescence%2520Properties%2520of%2520Novel%2520indol-3yl-thiazolo%255B3%252C2-a%255D%255B1%252C3%252C5%255Dtriazines%2520and%2520indole-3-carbaldehyde%2520Schiff%2520Bases%26jtitle%3DJ.%2520Fluoresc.%26date%3D2015%26volume%3D25%26spage%3D1727%26epage%3D1738%26doi%3D10.1007%2Fs10895-015-1659-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strobel, H.</span>; <span class="NLM_string-name">Wohlfart, P.</span>; <span class="NLM_string-name">Zoller, G.</span>; <span class="NLM_string-name">Will, D.
W.</span></span> <span> </span><span class="NLM_article-title">Heteroaryl-substituted amides comprising a saturated linker group, and their use as pharmaceuticals</span>. U.S. Patent <span class="NLM_patent">851,897,6B2</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Strobel&author=P.+Wohlfart&author=G.+Zoller&author=D.%0AW.+Will&title=Heteroaryl-substituted+amides+comprising+a+saturated+linker+group%2C+and+their+use+as+pharmaceuticals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStrobel%26aufirst%3DH.%26atitle%3DHeteroaryl-substituted%2520amides%2520comprising%2520a%2520saturated%2520linker%2520group%252C%2520and%2520their%2520use%2520as%2520pharmaceuticals%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarotto, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">Reviewing hit discovery literature for difficult targets: glutathione transferase omega-1 as an example</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7448</span>– <span class="NLM_lpage">7470</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00318</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00318" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7448-7470&author=Y.+Xieauthor=J.+L.+Dahlinauthor=A.+J.+Oakleyauthor=M.+G.+Casarottoauthor=P.+G.+Boardauthor=J.+B.+Baell&title=Reviewing+hit+discovery+literature+for+difficult+targets%3A+glutathione+transferase+omega-1+as+an+example&doi=10.1021%2Facs.jmedchem.8b00318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reviewing Hit Discovery Literature for Difficult Targets: Glutathione Transferase Omega-1 as an Example</span></div><div class="casAuthors">Xie, Yiyue; Dahlin, Jayme L.; Oakley, Aaron J.; Casarotto, Marco G.; Board, Philip G.; Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7448-7470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Early stage drug discovery reporting on relatively new or difficult targets is often assocd. with insufficient hit triage.  Literature reviews of such targets seldom delve into the detail required to critically analyze the assocd. screening hits reported.  Here we take the enzyme glutathione transferase omega-1 (GSTO1-1) as an example of a relatively difficult target and review the assocd. literature involving small-mol. inhibitors.  As part of this process we deliberately pay closer-than-usual attention to assay interference and hit quality aspects.  We believe this Perspective will be a useful guide for future development of GSTO1-1 inhibitors, as well serving as a template for future review formats of new or difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4iyFIZWXQKLVg90H21EOLACvtfcHk0ljehV6gJKY0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSgsbY%253D&md5=31037759dc901cb8b1f5a78877fa237c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00318%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DCasarotto%26aufirst%3DM.%2BG.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DReviewing%2520hit%2520discovery%2520literature%2520for%2520difficult%2520targets%253A%2520glutathione%2520transferase%2520omega-1%2520as%2520an%2520example%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7448%26epage%3D7470%26doi%3D10.1021%2Facs.jmedchem.8b00318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Son, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-T.</span></span> <span> </span><span class="NLM_article-title">Isozyme-specific fluorescent inhibitor of glutathione s-transferase omega 1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1021/cb100007s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100007s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=449-453&author=J.+Sonauthor=J.-J.+Leeauthor=J.-S.+Leeauthor=A.+Sch%C3%BCllerauthor=Y.-T.+Chang&title=Isozyme-specific+fluorescent+inhibitor+of+glutathione+s-transferase+omega+1&doi=10.1021%2Fcb100007s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcb100007s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100007s%26sid%3Dliteratum%253Aachs%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.-J.%26aulast%3DLee%26aufirst%3DJ.-S.%26aulast%3DSch%25C3%25BCller%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DY.-T.%26atitle%3DIsozyme-specific%2520fluorescent%2520inhibitor%2520of%2520glutathione%2520s-transferase%2520omega%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D449%26epage%3D453%26doi%3D10.1021%2Fcb100007s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierschke, S. G.</span></span> <span> </span><span class="NLM_article-title">Comparison of computational methods applied to oxazole, thiazole, and other heterocyclic compounds</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1002/jcc.540140112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fjcc.540140112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=75-88&author=A.+A.+Shafferauthor=S.+G.+Wierschke&title=Comparison+of+computational+methods+applied+to+oxazole%2C+thiazole%2C+and+other+heterocyclic+compounds&doi=10.1002%2Fjcc.540140112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540140112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540140112%26sid%3Dliteratum%253Aachs%26aulast%3DShaffer%26aufirst%3DA.%2BA.%26aulast%3DWierschke%26aufirst%3DS.%2BG.%26atitle%3DComparison%2520of%2520computational%2520methods%2520applied%2520to%2520oxazole%252C%2520thiazole%252C%2520and%2520other%2520heterocyclic%2520compounds%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D75%26epage%3D88%26doi%3D10.1002%2Fjcc.540140112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency as a guide in fragment hit selection and optimization</span>. <i>Discovery Today: Technol</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e157</span>– <span class="NLM_lpage">e162</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ddtec.2010.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVegsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e157-e162&author=S.+Schultesauthor=C.+de+Graafauthor=E.+E.+J.+Haaksmaauthor=I.+J.+P.+de+Eschauthor=R.+Leursauthor=O.+Kr%C3%A4mer&title=Ligand+efficiency+as+a+guide+in+fragment+hit+selection+and+optimization&doi=10.1016%2Fj.ddtec.2010.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency as a guide in fragment hit selection and optimization</span></div><div class="casAuthors">Schultes, Sabine; de Graaf, Chris; Haaksma, Eric E. J.; de Esch, Iwan J. P.; Leurs, Rob; Kraemer, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E157-E162</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based screening (FBS) has become an established approach for hit identification.  Starting points identified by FBS, are small fragments that require substantial modification to become leads.  As fragments are different from classical hits a process tailored for fragment evolution is required.  Scores for ligand efficiency have been proposed as guides for this process.  Here we review how these have been applied to guide the selection and optimization of fragment hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfhwn68rv4bVg90H21EOLACvtfcHk0ljehV6gJKY0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVegsL0%253D&md5=a57c85869d5c829159be31449fd04796</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchultes%26aufirst%3DS.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DHaaksma%26aufirst%3DE.%2BE.%2BJ.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%26atitle%3DLigand%2520efficiency%2520as%2520a%2520guide%2520in%2520fragment%2520hit%2520selection%2520and%2520optimization%26jtitle%3DDiscovery%2520Today%253A%2520Technol%26date%3D2010%26volume%3D7%26spage%3De157%26epage%3De162%26doi%3D10.1016%2Fj.ddtec.2010.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Disco.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lj8STrkILSPVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Disco.%26date%3D2007%26volume%3D6%26spage%3D881%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span> <i>Maestro</i>, version 10.1; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+version+10.1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lj8STrkILSPVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span> <i>Glide</i>, version 6.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+version+6.8%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span> <i>LigPrep</i>, version 3.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+version+3.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <i>Methods Enzymol.</i>; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+Enzymol."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520Enzymol.%26pub%3DElsevier%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgQKcDfdfJENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgQKcDfdfJENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr. Sect. D. Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgQKcDfdfJENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D.%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER, 2.10.3</i>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER%2C+2.10.3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%252C%25202.10.3%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE, 1.2.13</i>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE%2C+1.2.13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%252C%25201.2.13%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D. Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D.%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="note"><p class="first last">wwPDB Validation Server.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':[]}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Chinatsu Otake, Takuya Namba, Hidetsugu Tabata, Kosho Makino, Kiriko Hirano, Tetsuta Oshitari, Hideaki Natsugari, Takenori Kusumi, <span class="NLM_string-name hlFld-ContribAuthor">Hideyo Takahashi</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Preference of 2′-Fluoro-Substituted Acetophenone Derivatives Revealed by Through-Space 1H–19F and 13C–19F Spin–Spin Couplings. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (6)
                                     , 4638-4645. <a href="https://doi.org/10.1021/acs.joc.1c00051" title="DOI URL">https://doi.org/10.1021/acs.joc.1c00051</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.1c00051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.1c00051%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DConformational%252BPreference%252Bof%252B2%2525E2%252580%2525B2-Fluoro-Substituted%252BAcetophenone%252BDerivatives%252BRevealed%252Bby%252BThrough-Space%252B1H%2525E2%252580%25259319F%252Band%252B13C%2525E2%252580%25259319F%252BSpin%2525E2%252580%252593Spin%252BCouplings%26aulast%3DOtake%26aufirst%3DChinatsu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D07012021%26date%3D01032021%26volume%3D86%26issue%3D6%26spage%3D4638%26epage%3D4645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yiyue Xie, Padmaja Tummala, Aaron J. Oakley, Girdhar Singh Deora, Yuji Nakano, Melissa Rooke, Matthew E. Cuellar, Jessica M. Strasser, Jayme L. Dahlin, Michael A. Walters, Marco G. Casarotto, Philip G. Board, <span class="NLM_string-name hlFld-ContribAuthor">Jonathan B. Baell</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Benzenesulfonamide Derivatives as Potent Glutathione Transferase Omega-1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 2894-2914. <a href="https://doi.org/10.1021/acs.jmedchem.9b01391" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01391%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BBenzenesulfonamide%252BDerivatives%252Bas%252BPotent%252BGlutathione%252BTransferase%252BOmega-1%252BInhibitors%26aulast%3DXie%26aufirst%3DYiyue%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D22082019%26date%3D16032020%26date%3D27022020%26volume%3D63%26issue%3D6%26spage%3D2894%26epage%3D2914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuncan  Chen</span>, <span class="hlFld-ContribAuthor ">Shan  Lv</span>, <span class="hlFld-ContribAuthor ">Ruizhi  Lai</span>, <span class="hlFld-ContribAuthor ">Yingying  Xu</span>, <span class="hlFld-ContribAuthor ">Xin  Huang</span>, <span class="hlFld-ContribAuthor ">Jianglian  Li</span>, <span class="hlFld-ContribAuthor ">Guanghui  Lv</span>, <span class="hlFld-ContribAuthor ">Yong  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 2-aminothiazoles via rhodium-catalyzed carbenoid insertion/annulation of sulfoxonium ylides with thioureas. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2021,</strong> <em>21 </em><a href="https://doi.org/10.1016/j.cclet.2021.02.052" title="DOI URL">https://doi.org/10.1016/j.cclet.2021.02.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2021.02.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2021.02.052%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DSynthesis%252Bof%252B2-aminothiazoles%252Bvia%252Brhodium-catalyzed%252Bcarbenoid%252Binsertion%25252Fannulation%252Bof%252Bsulfoxonium%252Bylides%252Bwith%252Bthioureas%26aulast%3DChen%26aufirst%3DYuncan%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qian  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiefei  Wu</span>, <span class="hlFld-ContribAuthor ">Zexi  Pan</span>, <span class="hlFld-ContribAuthor ">Wen  Zhang</span>, <span class="hlFld-ContribAuthor ">Wei  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">A one-pot synthesis of 2-aminothiazoles via the coupling of ketones and thiourea using I
              2
              /dimethyl sulfoxide as a catalytic oxidative system. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Research</span><span> <strong>2021,</strong> <em>45 </em>
                                    (1-2)
                                     , 89-94. <a href="https://doi.org/10.1177/1747519820930961" title="DOI URL">https://doi.org/10.1177/1747519820930961</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1747519820930961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1747519820930961%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Research%26atitle%3DA%252Bone-pot%252Bsynthesis%252Bof%252B2-aminothiazoles%252Bvia%252Bthe%252Bcoupling%252Bof%252Bketones%252Band%252Bthiourea%252Busing%252BI%252B2%252B%25252Fdimethyl%252Bsulfoxide%252Bas%252Ba%252Bcatalytic%252Boxidative%252Bsystem%26aulast%3DZhang%26aufirst%3DQian%26date%3D2021%26date%3D2020%26volume%3D45%26issue%3D1-2%26spage%3D89%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patrick E.  Hanna</span>, <span class="hlFld-ContribAuthor ">M. W.  Anders</span>. </span><span class="cited-content_cbyCitation_article-title">The mercapturic acid pathway. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Toxicology</span><span> <strong>2019,</strong> <em>49 </em>
                                    (10)
                                     , 819-929. <a href="https://doi.org/10.1080/10408444.2019.1692191" title="DOI URL">https://doi.org/10.1080/10408444.2019.1692191</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10408444.2019.1692191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10408444.2019.1692191%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Toxicology%26atitle%3DThe%252Bmercapturic%252Bacid%252Bpathway%26aulast%3DHanna%26aufirst%3DPatrick%2BE.%26date%3D2019%26date%3D2020%26volume%3D49%26issue%3D10%26spage%3D819%26epage%3D929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of reported GSTO1-1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1–12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acryloyl chloride, DCM, DIEA, rt, 2 h; (b) 2-chloroethylsulfonyl chloride, DCM, DIEA, rt, 2 h; (c) methyl (<i>E</i>)-4-bromocronate, potassium fluoride, CH<sub>3</sub>CN, 100 °C, 4 h.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Thiazole Acrylamide Analogues <b>16–27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CuBr<sub>2</sub>, EtOAc, reflux, 3 h; (b) thiourea, EtOH, reflux, 2 h; (c) acryloyl chloride, DCM, DIEA, rt, 2 h.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Thiazole Acrylamide Analogues <b>29–34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) corresponding N-substituted thiourea, EtOH, reflux, 2 h; (b) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux, 8 h; (c) acryloyl chloride, DCM, rt, 2 h; (d) acryloyl chloride, toluene, 70 °C, 0.5 h.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Thiozale Acrylamide Analogues <b>37–45</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, EtOH, reflux, 3 h; (b) corresponding primary amines, EtOH, reflux, overnight; (c) HCl, THF/H<sub>2</sub>O, reflux, 2 h; (d) EDC, HOBt, DIEA, DMF, rt, overnight; (e) acryloyl chloride, toluene, 70 °C, 0.5 h; (f) TFA, DCM, rt, 2 h.</p></p></figure><figure data-id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Thiazole Acrylamide Analogues <b>49–52</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, acetone, reflux, 8 h; (b) CuBr<sub>2</sub>, EtOAc, reflux, 3 h; (c) KSCN, EtOH, reflux, 3 h; (d) (3-methylisoxazol-5-yl)methanamine, EtOH, reflux, 32 h; (d) acryloyl chloride, toluene, 70 °C, 0.5 h; (e) 2-chloroacetyl chloride, DIEA, toluene, 70 °C, 1 h; (f) BBr<sub>3</sub>, DCM, −78 °C to rt, 0.5–2 h, THF, TEA, reflux, 1 h.</p></p></figure><figure data-id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 5-(bromomethyl)-3-methylisoxazole, NaH, DMF, 0 °C to rt, 2 h; (b) TFA, DCM, rt, 2 h; (c) acryloyl chloride, toluene, 70 °C, 0.5 h; (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane, H<sub>2</sub>O, 80 °C, 8 h.</p></p></figure><figure data-id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Cocrystal structure of GSTO1:C1-27 (pink stick, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM">4YQM</a>). (B) Structural differences of <b>1</b> (cyan) and C1-27 (orange) under nucleophilic attack. 3D structures were generated, overlapped, and measured in Maestro. (C) Compound <b>5</b> (cyan) displayed similar binding pose with C4-10 (orange) at the GSTO1 H-site in a docking study perform by Glide. (D) Alignment of <b>5</b> (yellow green) and <b>10</b> (pink) showed the advantage of the thiazole core structure, and 3D structures were generated and overlapped in Maestro.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB">6MHB</a>, cyan ribbons:teal sticks). (A) Hydrophobic interactions of <b>18</b> within the H-site of GSTO1. (B) Structural comparison of GSTO1:<b>18</b> and GSTO1:C4-10 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV">4YQV</a>, gray ribbons:black sticks) demonstrates that these inhibitors bind with similar orientations within a hydrophobic cavity (gray surface) of the GSTO1 H-site. Average RMSD = 0.49 Å</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural comparison of GSTO1:<b>37</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC">6MHC</a>, <b>37</b> is shown in pink sticks), GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>, <b>44</b> is shown in green sticks) and GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHA">6MHA</a>, <b>18</b> is shown in teal sticks). (A) Structural differences are a result of the N-substituted inhibitors <b>37</b> (magenta sticks) and <b>44</b> (dark green sticks) adopting a different orientation within the GSTO1 H-site compared to <b>18</b> (teal sticks). Trp222 makes pi-stacking with the chlorophenyl group of <b>18</b> but has no contact with the phenyl groups of <b>37</b> and <b>44</b>. (B) Hydrophobic interactions of GSTO1 with inhibitors <b>37</b> and <b>44</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/medium/jm-2018-01960z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) N-substitution of thiazole acrylamide inhibitors, as demonstrated by the cocrystal structure of GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>, dark green sticks), may provide a handle to access hydrogen-bonding or water-mediated interactions with residues in the glutathione binding site (glutathione from PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EEM">1EEM</a> shown in black sticks). Residues that interact with both glutathione and <b>44</b> are highlighted in light blue, whereas sites of potential enthalpic interactions with extended N-substituted thiazole inhibitors are highlighted in yellow. (B) 2-Position of the phenyl ring of inhibitor <b>44</b> is directed into a pocket with a strong overall positive charge. The 2-hydroxyl substituent of <b>49</b>, bearing a partial negative charge, is likely stabilized through electrostatic interactions with the inhibitor binding site. Electrostatic maps were generated using APBS from −5.0 eV (red) to +5.0 eV (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-6/acs.jmedchem.8b01960/20190321/images/large/jm-2018-01960z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01960&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i136">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowsey, I. R.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferases</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.45.120403.095857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1146%2Fannurev.pharmtox.45.120403.095857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=15822171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=51-88&author=J.+D.+Hayesauthor=J.+U.+Flanaganauthor=I.+R.+Jowsey&title=Glutathione+transferases&doi=10.1146%2Fannurev.pharmtox.45.120403.095857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases</span></div><div class="casAuthors">Hayes, John D.; Flanagan, Jack U.; Jowsey, Ian R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-88, 1 plate</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The authors describe the 3 mammalian glutathione S-transferase (GST) families, namely cytosolic, mitochondrial, and microsomal GSTs, the latter now designated MAPEG (Membrane-Assocd. Proteins in Eicosanoid and Glutathione metab.).  In addn. to detoxifying electrophilic xenobiotics, such as chem. carcinogens, environmental pollutants, and antitumor agents, these GSTs inactivate endogenous α,β-unsatd. aldehydes, quinones, epoxides, and hydroperoxides formed as secondary metabolites during oxidative stress.  These enzymes are also intimately involved in the biosynthesis of leukotrienes, prostaglandins, testosterone, and progesterone, as well as the degrdn. of tyrosine.  Among their substrates, GSTs conjugate the signaling mols., 15-deoxy-Δ12,14-prostaglandin J2 (15d-PGJ2) and 4-hydroxynonenal, with glutathione, and consequently they antagonize expression of genes trans-activated by peroxisome proliferator-activated receptor γ (PPARγ) and nuclear factor-erythroid 2 p45-related factor 2 (Nrf2).  Through metab. of 15d-PGJ2, GST may enhance gene expression driven by nuclear factor-κB (NF-κB).  Cytosolic human GST exhibit genetic polymorphisms and this variation can increase susceptibility to carcinogenesis and inflammatory disease.  Polymorphisms in human MAPEG are assocd. with alterations in lung function and increased risk of myocardial infarction and stroke.  Targeted disruption of murine genes has demonstrated that cytosolic GST isoenzymes are broadly cytoprotective, whereas MAPEG proteins have pro-inflammatory activities.  Furthermore, knockout of mouse GSTA4 and GSTZ1 leads to overexpression of transferases in the Alpha, Mu, and Pi classes, an observation suggesting they are part of an adaptive mechanism that responds to endogenous chem. cues such as 4-hydroxynonenal and tyrosine degrdn. products.  Consistent with this hypothesis, the promoters of cytosolic GST and MAPEG genes contain antioxidant response elements through which they are transcriptionally activated during exposure to Michael reaction acceptors and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH__KbntnGGrVg90H21EOLACvtfcHk0lhzj8FD4OJeXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVWjtrk%253D&md5=c74f17b1381f046770a7f6faf5c51291</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.45.120403.095857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.45.120403.095857%26sid%3Dliteratum%253Aachs%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DFlanagan%26aufirst%3DJ.%2BU.%26aulast%3DJowsey%26aufirst%3DI.%2BR.%26atitle%3DGlutathione%2520transferases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D45%26spage%3D51%26epage%3D88%26doi%3D10.1146%2Fannurev.pharmtox.45.120403.095857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferases, regulators of cellular metabolism and physiology</span>. <i>BBA-Gen Subjects</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1830</i></span>,  <span class="NLM_fpage">3267</span>– <span class="NLM_lpage">3288</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2012.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.bbagen.2012.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=23201197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1830&publication_year=2013&pages=3267-3288&author=P.+G.+Boardauthor=D.+Menon&title=Glutathione+transferases%2C+regulators+of+cellular+metabolism+and+physiology&doi=10.1016%2Fj.bbagen.2012.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases, regulators of cellular metabolism and physiology</span></div><div class="casAuthors">Board, Philip G.; Menon, Deepthi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1830</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3267-3288</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The cytosolic glutathione S-transferases (GSTs) comprise a superfamily of proteins that can be categorized into multiple classes with a mixt. of highly specific and overlapping functions.  Here, the authors cover the genetics, structure, and function of human cytosolic GSTs with particular attention to their emerging roles in cellular metab.  All of the catalytically active GSTs contribute to the glutathione conjugation or glutathione dependant-biotransformation of xenobiotics, and many catalyze glutathione peroxidase or thiol transferase reactions.  GSTs also catalyze glutathione-dependent isomerization reactions required for the synthesis of several prostaglandins and steroid hormones and the catabolism of tyrosine.  An increasing body of work has implicated several GSTs in the regulation of cell signaling pathways mediated by stress-activated kinases like JNK kinase.  In addn., some members of the cytosolic GST family have been shown to form ion channels in intracellular membranes and to modulate ryanodine receptor Ca2+ channels in skeletal and cardiac muscle.  In addn. to their well established roles in the conjugation and biotransformation of xenobiotics, GSTs have emerged as significant regulators of pathways detg. cell proliferation and survival and as regulators of ryanodine receptors that are essential for muscle function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoECjoqUckeMrVg90H21EOLACvtfcHk0lhzj8FD4OJeXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFSluw%253D%253D&md5=d0cb57c5d49fc78e6bac46ff7f60104f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2012.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2012.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DMenon%26aufirst%3DD.%26atitle%3DGlutathione%2520transferases%252C%2520regulators%2520of%2520cellular%2520metabolism%2520and%2520physiology%26jtitle%3DBBA-Gen%2520Subjects%26date%3D2013%26volume%3D1830%26spage%3D3267%26epage%3D3288%26doi%3D10.1016%2Fj.bbagen.2012.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.46.120604.141046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1146%2Fannurev.pharmtox.46.120604.141046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=16968214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=89-116&author=T.+W.+Kenslerauthor=N.+Wakabayashiauthor=S.+Biswal&title=Cell+survival+responses+to+environmental+stresses+via+the+Keap1-Nrf2-ARE+pathway&doi=10.1146%2Fannurev.pharmtox.46.120604.141046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span></div><div class="casAuthors">Kensler, Thomas W.; Wakabayashi, Nobunao; Biswal, Shyam</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-116</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Keap1-Nrf2-ARE signaling plays a significant role in protecting cells from endogenous and exogenous stresses.  The development of Nrf2 knockout mice has provided key insights into the toxicol. importance of this pathway.  These mice are more sensitive to the hepatic, pulmonary, ovarian, and neurotoxic consequences of acute exposures to environmental agents and drugs, inflammatory stresses, as well as chronic exposures to cigarette smoke and other carcinogens.  Under quiescent conditions, the transcription factor Nrf2 interacts with the actin-anchored protein Keap1, largely localized in the cytoplasm.  This quenching interaction maintains low basal expression of Nrf2-regulated genes.  However, upon recognition of chem. signals imparted by oxidative and electrophilic mols., Nrf2 is released from Keap1, escapes proteasomal degrdn., translocates to the nucleus, and transactivates the expression of several dozen cytoprotective genes that enhance cell survival.  This review highlights the key elements in this adaptive response to protection against acute and chronic cell injury provoked by environmental stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAmRpcXFLXq7Vg90H21EOLACvtfcHk0lhkzC_DFJjeTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D&md5=eee9260ebe6186b82da647701a10c7e2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.46.120604.141046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.46.120604.141046%26sid%3Dliteratum%253Aachs%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DCell%2520survival%2520responses%2520to%2520environmental%2520stresses%2520via%2520the%2520Keap1-Nrf2-ARE%2520pathway%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2007%26volume%3D47%26spage%3D89%26epage%3D116%26doi%3D10.1146%2Fannurev.pharmtox.46.120604.141046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramkumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziemke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, H.</span></span> <span> </span><span class="NLM_article-title">Mechanistic evaluation and transcriptional signature of a glutathione S-transferase omega 1 inhibitor</span>. <i>Nat. Commu.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13084</span>, <span class="refDoi"> DOI: 10.1038/ncomms13084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fncomms13084" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13084&author=K.+Ramkumarauthor=S.+Samantaauthor=A.+Kyaniauthor=S.+Yangauthor=S.+Tamuraauthor=E.+Ziemkeauthor=J.+A.+Stuckeyauthor=S.+Liauthor=K.+Chinnaswamyauthor=H.+Otake&title=Mechanistic+evaluation+and+transcriptional+signature+of+a+glutathione+S-transferase+omega+1+inhibitor&doi=10.1038%2Fncomms13084"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fncomms13084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13084%26sid%3Dliteratum%253Aachs%26aulast%3DRamkumar%26aufirst%3DK.%26aulast%3DSamanta%26aufirst%3DS.%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DZiemke%26aufirst%3DE.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DOtake%26aufirst%3DH.%26atitle%3DMechanistic%2520evaluation%2520and%2520transcriptional%2520signature%2520of%2520a%2520glutathione%2520S-transferase%2520omega%25201%2520inhibitor%26jtitle%3DNat.%2520Commu.%26date%3D2016%26volume%3D7%26spage%3D13084%26doi%3D10.1038%2Fncomms13084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">The omega-class glutathione transferases: structure, function, and genetics</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">235</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.561353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.3109%2F03602532.2011.561353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=21495794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=226-235&author=P.+G.+Board&title=The+omega-class+glutathione+transferases%3A+structure%2C+function%2C+and+genetics&doi=10.3109%2F03602532.2011.561353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The omega-class glutathione transferases: structure, function, and genetics</span></div><div class="casAuthors">Board, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">226-235</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The omega class of glutathione S-transferases (GSTs) is a relatively ancient member of the cytosolic GST superfamily, and the omega-class GSTs are found in plants, animals, and some microbial species.  The omega-class GSTs exhibit the canonical GST fold, but, unlike other GSTs, the omega-class GSTs have a Cys residue in their active site.  Consequently, the omega-class GSTs catalyze a range of thiol transferase and redn. reactions that are not catalyzed by members of the other classes.  Human GSTO1-1 can catalyze the redn. of monomethylarsonic acid(V), but this does not appear to be physiol. important in cases of high environmental arsenic exposure.  GSTO1-1 also plays an important role in the biotransformation of reactive α-haloketones to nontoxic acetophenones.  Genetic variation is common in the omega-class GST genes, and variants that result in deficiency of GSTO1-1 have been characterized.  Genetic linkage studies have discovered assocns. between GSTO genes and the age at onset of Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis.  The mechanism underlying this assocn. with neurol. diseases may derive from the capacity of omega-class GSTs to mitigate oxidative stress or their role in activating the proinflammatory cytokine, interleukin-1β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw-fnr-XyZH7Vg90H21EOLACvtfcHk0lhkzC_DFJjeTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCisbo%253D&md5=9653a63761ddfb11cfc959f7e100cac9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.561353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.561353%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DThe%2520omega-class%2520glutathione%2520transferases%253A%2520structure%252C%2520function%252C%2520and%2520genetics%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2011%26volume%3D43%26spage%3D226%26epage%3D235%26doi%3D10.3109%2F03602532.2011.561353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappello, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, M. W.</span></span> <span> </span><span class="NLM_article-title">S-(4-Nitrophenacyl)glutathione is a specific substrate for glutathione transferase omega 1-1</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2007.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ab.2007.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=18028863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVyhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2008&pages=25-30&author=P.+G.+Boardauthor=M.+Cogganauthor=J.+Cappelloauthor=H.+Zhouauthor=A.+J.+Oakleyauthor=M.+W.+Anders&title=S-%284-Nitrophenacyl%29glutathione+is+a+specific+substrate+for+glutathione+transferase+omega+1-1&doi=10.1016%2Fj.ab.2007.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">S-(4-Nitrophenacyl)glutathione is a specific substrate for glutathione transferase omega 1-1</span></div><div class="casAuthors">Board, Philip G.; Coggan, Marjorie; Cappello, Jean; Zhou, Huina; Oakley, Aaron J.; Anders, M. W.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-30</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Glutathione transferase omega 1-1 (GSTO1-1) catalyzes the biotransformation of arsenic and is implicated as a factor influencing the age-at-onset of Alzheimer's disease and the posttranslational activation of interleukin 1β (IL-1β).  Investigation of the biol. role of GSTO1-1 variants has been hampered by the lack of a specific assay for GSTO1-1 activity in tissue samples that contain other GSTs and other enzymes with similar catalytic specificities.  Previous studies have shown that GSTO1-1 catalyzes the redn. of S-(phenacyl)glutathiones to acetophenones.  A new substrate, S-(4-nitrophenacyl)glutathione (4NPG), has been prepd. and found to have a high turnover with GSTO1-1 but negligible activity with GSTO2-2 and other members of the glutathione transferase superfamily.  A spectrophotometric assay with 4NPG as a substrate has been used to det. GSTO1-1 activity in several human breast cancer cell lines and in mouse liver and brain tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs5SUvAbtGrbVg90H21EOLACvtfcHk0lgGMhSgsvN9tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVyhtb8%253D&md5=afc7bd7570bbae5b095c993edc3bb1ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2007.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2007.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DCappello%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DAnders%26aufirst%3DM.%2BW.%26atitle%3DS-%25284-Nitrophenacyl%2529glutathione%2520is%2520a%2520specific%2520substrate%2520for%2520glutathione%2520transferase%2520omega%25201-1%26jtitle%3DAnal.%2520Biochem.%26date%3D2008%26volume%3D374%26spage%3D25%26epage%3D30%26doi%3D10.1016%2Fj.ab.2007.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelvanayagam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easteal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermiin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danley, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span> <span> </span><span class="NLM_article-title">Identification, characterization, and crystal structure of the Omega class glutathione transferases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">24798</span>– <span class="NLM_lpage">24806</span>, <span class="refDoi"> DOI: 10.1074/jbc.m001706200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1074%2Fjbc.m001706200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=24798-24806&author=P.+G.+Boardauthor=M.+Cogganauthor=G.+Chelvanayagamauthor=S.+Eastealauthor=L.+S.+Jermiinauthor=G.+K.+Schulteauthor=D.+E.+Danleyauthor=L.+R.+Hothauthor=M.+C.+Grifforauthor=A.+V.+Kamath&title=Identification%2C+characterization%2C+and+crystal+structure+of+the+Omega+class+glutathione+transferases&doi=10.1074%2Fjbc.m001706200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m001706200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m001706200%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DChelvanayagam%26aufirst%3DG.%26aulast%3DEasteal%26aufirst%3DS.%26aulast%3DJermiin%26aufirst%3DL.%2BS.%26aulast%3DSchulte%26aufirst%3DG.%2BK.%26aulast%3DDanley%26aufirst%3DD.%2BE.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DGriffor%26aufirst%3DM.%2BC.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26atitle%3DIdentification%252C%2520characterization%252C%2520and%2520crystal%2520structure%2520of%2520the%2520Omega%2520class%2520glutathione%2520transferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D24798%26epage%3D24806%26doi%3D10.1074%2Fjbc.m001706200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitbread, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masoumi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetlow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmuck, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">Characterization of the omega class of glutathione transferases</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>401</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(05)01005-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fs0076-6879%2805%2901005-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2005&pages=78-99&author=A.+K.+Whitbreadauthor=A.+Masoumiauthor=N.+Tetlowauthor=E.+Schmuckauthor=M.+Cogganauthor=P.+G.+Board&title=Characterization+of+the+omega+class+of+glutathione+transferases&doi=10.1016%2Fs0076-6879%2805%2901005-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fs0076-6879%2805%2901005-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0076-6879%252805%252901005-0%26sid%3Dliteratum%253Aachs%26aulast%3DWhitbread%26aufirst%3DA.%2BK.%26aulast%3DMasoumi%26aufirst%3DA.%26aulast%3DTetlow%26aufirst%3DN.%26aulast%3DSchmuck%26aufirst%3DE.%26aulast%3DCoggan%26aufirst%3DM.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DCharacterization%2520of%2520the%2520omega%2520class%2520of%2520glutathione%2520transferases%26jtitle%3DMethods%2520Enzymol.%26date%3D2005%26volume%3D401%26spage%3D78%26epage%3D99%26doi%3D10.1016%2Fs0076-6879%2805%2901005-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Disparate proteome reactivity profiles of carbon electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">405</span>, <span class="refDoi"> DOI: 10.1038/nchembio.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnchembio.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=18488014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=405&author=E.+Weerapanaauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=Disparate+proteome+reactivity+profiles+of+carbon+electrophiles&doi=10.1038%2Fnchembio.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disparate proteome reactivity profiles of carbon electrophiles</span></div><div class="casAuthors">Weerapana, Eranthie; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">405-407</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insights into the proteome reactivity of electrophiles are crucial for designing activity-based probes for enzymes lacking cognate affinity labels.  Here, the authors show that different classes of carbon electrophiles exhibit markedly distinct amino acid labeling profiles in proteomes, ranging from selective reactivity with cysteine to adducts with several amino acids.  These data specify electrophilic chemotypes with restricted and permissive reactivity profiles to guide the design of next-generation functional proteomics probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI4Jh6351SYLVg90H21EOLACvtfcHk0lh7H2QQiYxy6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D&md5=69eb0ae589c50f46a8695dc6731366e2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.91%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDisparate%2520proteome%2520reactivity%2520profiles%2520of%2520carbon%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D405%26doi%3D10.1038%2Fnchembio.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Quantitative reactivity profiling predicts functional cysteines in proteomes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">790</span>, <span class="refDoi"> DOI: 10.1038/nature09472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnature09472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=21085121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=790&author=E.+Weerapanaauthor=C.+Wangauthor=G.+M.+Simonauthor=F.+Richterauthor=S.+Khareauthor=M.+B.+D.+Dillonauthor=D.+A.+Bachovchinauthor=K.+Mowenauthor=D.+Bakerauthor=B.+F.+Cravatt&title=Quantitative+reactivity+profiling+predicts+functional+cysteines+in+proteomes&doi=10.1038%2Fnature09472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative reactivity profiling predicts functional cysteines in proteomes</span></div><div class="casAuthors">Weerapana, Eranthie; Wang, Chu; Simon, Gabriel M.; Richter, Florian; Khare, Sagar; Dillon, Myles B. D.; Bachovchin, Daniel A.; Mowen, Kerri; Baker, David; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">790-795</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cysteine is the most intrinsically nucleophilic amino acid in proteins, where its reactivity is tuned to perform diverse biochem. functions.  The absence of a consensus sequence that defines functional cysteines in proteins has hindered their discovery and characterization.  Here we describe a proteomics method to profile quant. the intrinsic reactivity of cysteine residues en masse directly in native biol. systems.  Hyper-reactivity was a rare feature among cysteines and it was found to specify a wide range of activities, including nucleophilic and reductive catalysis and sites of oxidative modification.  Hyper-reactive cysteines were identified in several proteins of uncharacterized function, including a residue conserved across eukaryotic phylogeny that we show is required for yeast viability and is involved in iron-sulfur protein biogenesis.  We also demonstrate that quant. reactivity profiling can form the basis for screening and functional assignment of cysteines in computationally designed proteins, where it discriminated catalytically active from inactive cysteine hydrolase designs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe4hHN7L4cGbVg90H21EOLACvtfcHk0lh7H2QQiYxy6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI&md5=277e436e7a8f4dfff0b548bce52ac40d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09472%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DKhare%26aufirst%3DS.%26aulast%3DDillon%26aufirst%3DM.%2BB.%2BD.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DMowen%26aufirst%3DK.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DQuantitative%2520reactivity%2520profiling%2520predicts%2520functional%2520cysteines%2520in%2520proteomes%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D790%26doi%3D10.1038%2Fnature09472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, R.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.-W.</span></span> <span> </span><span class="NLM_article-title">Comparative proteomic analysis of differentially expressed proteins in human pancreatic cancer tissue</span>. <i>Hepatobiliary Pancreat. Dis. Int.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">200</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=19357035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1Ghtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=193-200&author=J.-H.+Chenauthor=R.-Z.+Niauthor=M.-B.+Xiaoauthor=J.-G.+Guoauthor=J.-W.+Zhou&title=Comparative+proteomic+analysis+of+differentially+expressed+proteins+in+human+pancreatic+cancer+tissue"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative proteomic analysis of differentially expressed proteins in human pancreatic cancer tissue</span></div><div class="casAuthors">Chen, Jian-Hua; Ni, Run-Zhou; Xiao, Ming-Bing; Guo, Ji-Guang; Zhou, Jia-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Hepatobiliary & Pancreatic Diseases International</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-200</span>CODEN:
                <span class="NLM_cas:coden">HPDIAJ</span>;
        ISSN:<span class="NLM_cas:issn">1499-3872</span>.
    
            (<span class="NLM_cas:orgname">First Affiliated Hospital, Zhejiang University School of Medicine</span>)
        </div><div class="casAbstract">Pancreatic cancer is one of the most common malignant tumors.  Early diagnosis of pancreatic cancer is difficult because of the latent onset and lack of good biomarkers.  This study aimed to look for and identify differentially expressed proteins in tissues of pancreatic cancer and adjacent noncancerous tissues by proteomic approaches so as to provide information about possible pancreatic cancer markers and therapeutic targets.  Proteins extd. from 3 paired adjacent noncancerous and cancerous pancreatic tissue specimens were sepd. by two-dimensional gel electrophoresis (2-DE).  The protein spots exhibiting statistical alternations between the two groups through computerized image anal. were then identified by matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS).  In addn., Western blotting and immunohistochem. were performed to verify the expression of certain candidate proteins.  Twelve proteins were significantly upregulated and 4 were downregulated between cancerous and paired adjacent noncancerous pancreatic tissues.  Several proteins (S100A11, Ig gamma-1 chain C region, GSTO1 and peroxiredoxin 4) were found for the first time to be assocd. with pancreatic cancer.  Differential expression of some identified proteins was further confirmed by Western blotting anal. and/or immunohistochem. anal.  Comparative proteomic anal. using 2-DE and MALDI-TOF-MS is an effective method for identifying differentially expressed proteins that may be the potential diagnostic biomarkers and therapeutic targets for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYq2jNzP9Vi7Vg90H21EOLACvtfcHk0lgv2WOr7-xLNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1Ghtbk%253D&md5=fa36d1a413dfcacccde4b591abbdd13d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.-H.%26aulast%3DNi%26aufirst%3DR.-Z.%26aulast%3DXiao%26aufirst%3DM.-B.%26aulast%3DGuo%26aufirst%3DJ.-G.%26aulast%3DZhou%26aufirst%3DJ.-W.%26atitle%3DComparative%2520proteomic%2520analysis%2520of%2520differentially%2520expressed%2520proteins%2520in%2520human%2520pancreatic%2520cancer%2520tissue%26jtitle%3DHepatobiliary%2520Pancreat.%2520Dis.%2520Int.%26date%3D2009%26volume%3D8%26spage%3D193%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalowicz, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volker, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pambuccian, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrey, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, P. M.</span></span> <span> </span><span class="NLM_article-title">Identification of a gene expression signature associated with recurrent disease in squamous cell carcinoma of the head and neck</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-03-2144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1158%2F0008-5472.can-03-2144" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=55-63&author=M.+A.+Ginosauthor=G.+P.+Pageauthor=B.+S.+Michalowiczauthor=K.+J.+Patelauthor=S.+E.+Volkerauthor=S.+E.+Pambuccianauthor=F.+G.+Ondreyauthor=G.+L.+Adamsauthor=P.+M.+Gaffney&title=Identification+of+a+gene+expression+signature+associated+with+recurrent+disease+in+squamous+cell+carcinoma+of+the+head+and+neck&doi=10.1158%2F0008-5472.can-03-2144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-03-2144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-03-2144%26sid%3Dliteratum%253Aachs%26aulast%3DGinos%26aufirst%3DM.%2BA.%26aulast%3DPage%26aufirst%3DG.%2BP.%26aulast%3DMichalowicz%26aufirst%3DB.%2BS.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26aulast%3DVolker%26aufirst%3DS.%2BE.%26aulast%3DPambuccian%26aufirst%3DS.%2BE.%26aulast%3DOndrey%26aufirst%3DF.%2BG.%26aulast%3DAdams%26aufirst%3DG.%2BL.%26aulast%3DGaffney%26aufirst%3DP.%2BM.%26atitle%3DIdentification%2520of%2520a%2520gene%2520expression%2520signature%2520associated%2520with%2520recurrent%2520disease%2520in%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520head%2520and%2520neck%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D55%26epage%3D63%26doi%3D10.1158%2F0008-5472.can-03-2144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djukic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pljesa-Ercegovac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suvakov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coric, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragicevic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savic-Radojevic, A.</span></span> <span> </span><span class="NLM_article-title">Upregulated glutathione transferase omega-1 correlates with progression of urinary bladder carcinoma</span>. <i>Redox Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1080/13510002.2017.1299909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1080%2F13510002.2017.1299909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=28288548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlKnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=486-492&author=T.+Djukicauthor=T.+Simicauthor=M.+Pljesa-Ercegovacauthor=M.+Maticauthor=S.+Suvakovauthor=V.+Coricauthor=D.+Dragicevicauthor=A.+Savic-Radojevic&title=Upregulated+glutathione+transferase+omega-1+correlates+with+progression+of+urinary+bladder+carcinoma&doi=10.1080%2F13510002.2017.1299909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulated glutathione transferase omega-1 correlates with progression of urinary bladder carcinoma</span></div><div class="casAuthors">Djukic, Tatjana; Simic, Tatjana; Pljesa-Ercegovac, Marija; Matic, Marija; Suvakov, Sonja; Coric, Vesna; Dragicevic, Dejan; Savic-Radojevic, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Redox Report</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">486-492</span>CODEN:
                <span class="NLM_cas:coden">RDRPE4</span>;
        ISSN:<span class="NLM_cas:issn">1351-0002</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: Newly discovered glutathione transferase omega 1 (GSTO1-1) plays an important role in the glutathionylation cycle, a significant mechanism of protein function regulation.  GSTO1-1 expression pattern has not been studied in transitional cell carcinoma (TCC), as yet.: A total of 56 TCC tumor and corresponding non-tumor specimens were investigated.  Glutathione content and thioltransferase activity were measured spectrophotometrically.  Protein-glutathione mixed disulfides were measured fluorimetrically.  GSTO1-1 expression was detd. by immunoblot and qPCR.  Immunopptn. with GSTO1-1 antibody was followed by immunoblot using anti-GSTO1, GSTP1, c-Jun, JNK, Akt, phospho-Akt, and ASK1 antibody, while for the total S-glutathionylation levels non-reducing electrophoresis was performed.: The contents of reduced glutathione and thioltransferase activity were significantly increased in tumor compared to non-tumor tissue.  The increased GSTO1 expression in tumor tissue showed clear correlation with grade and stage.  However, decreased total protein glutathionylation level in tumor compared to non-tumor samples was found.  Immunopptn. has shown an assocn. of GSTO1-1 with GSTP1, Akt, phospho-Akt, and ASK1 proteins.: GSTO1 deglutathionylase activity suggests its potential important role in redox perturbations present in TCC.  Increased GSTO1-1 expression might contribute to TCC development and/or progression supporting the notion that GSTO1-1 may be a promising novel cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsmBE2HWB_c7Vg90H21EOLACvtfcHk0lgv2WOr7-xLNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlKnurY%253D&md5=75be44b48cbd1c6e2d24b1061776b6c3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1080%2F13510002.2017.1299909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13510002.2017.1299909%26sid%3Dliteratum%253Aachs%26aulast%3DDjukic%26aufirst%3DT.%26aulast%3DSimic%26aufirst%3DT.%26aulast%3DPljesa-Ercegovac%26aufirst%3DM.%26aulast%3DMatic%26aufirst%3DM.%26aulast%3DSuvakov%26aufirst%3DS.%26aulast%3DCoric%26aufirst%3DV.%26aulast%3DDragicevic%26aufirst%3DD.%26aulast%3DSavic-Radojevic%26aufirst%3DA.%26atitle%3DUpregulated%2520glutathione%2520transferase%2520omega-1%2520correlates%2520with%2520progression%2520of%2520urinary%2520bladder%2520carcinoma%26jtitle%3DRedox%2520Rep.%26date%3D2017%26volume%3D22%26spage%3D486%26epage%3D492%26doi%3D10.1080%2F13510002.2017.1299909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span> <span> </span><span class="NLM_article-title">Proteomic analysis of Tiam1-mediated metastasis in colorectal cancer</span>. <i>Cell Biol. Int.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">814</span>, <span class="refDoi"> DOI: 10.1016/j.cellbi.2007.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.cellbi.2007.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=17376711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvFSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2007&pages=805-814&author=L.+Liuauthor=L.+Zhaoauthor=Y.+Zhangauthor=Q.+Zhangauthor=Y.+Ding&title=Proteomic+analysis+of+Tiam1-mediated+metastasis+in+colorectal+cancer&doi=10.1016%2Fj.cellbi.2007.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analysis of Tiam1-mediated metastasis in colorectal cancer</span></div><div class="casAuthors">Liu, Li; Zhao, Liang; Zhang, Yanfei; Zhang, Qinling; Ding, Yanqing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biology International</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-814</span>CODEN:
                <span class="NLM_cas:coden">CBIIEV</span>;
        ISSN:<span class="NLM_cas:issn">1065-6995</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tiam1 (T lymphoma invasion and metastasis 1), a guanine nucleotide exchange factor that activates Rac, was recently identified as a novel colorectal cancer metastasis-related gene.  To better understand the mechanism underlying Tiam1-mediated metastasis, we applied two-dimensional polyacrylamide gel electrophoresis (2-DE) and matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF MS) anal. to identify differentially expressed proteins between Tiam1 transfected and mock transfected colorectal cancer HT29 cells.  Eleven differentially expressed proteins were identified and further validated by Western blot and/or real-time PCR.  The results revealed that Tiam1 transfection in colorectal cancer cells could upregulate the expression of Fascin-1, heat shock protein 27 (HSP27), high-mobility group box 1 (HMGB1), glutathione S-transferase omega 1 (GSTO1) and downregulate the expression of annexin IV.  These differentially expressed proteins may be directly or indirectly regulated by Tiam1 and be helpful in studying mechanisms that lead to the function of Tiam1.  These results give some clues to elucidate the mechanism of Tiam1-mediated metastasis for colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojn2cD4feTP7Vg90H21EOLACvtfcHk0ljchw1vC3m0WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvFSjtr8%253D&md5=377fe5a4b54ecc8f29d79fc7f6041e10</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cellbi.2007.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellbi.2007.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DDing%26aufirst%3DY.%26atitle%3DProteomic%2520analysis%2520of%2520Tiam1-mediated%2520metastasis%2520in%2520colorectal%2520cancer%26jtitle%3DCell%2520Biol.%2520Int.%26date%3D2007%26volume%3D31%26spage%3D805%26epage%3D814%26doi%3D10.1016%2Fj.cellbi.2007.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIlwain, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tew, K. D.</span></span> <span> </span><span class="NLM_article-title">Glutathione S-transferase polymorphisms: cancer incidence and therapy</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1639</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1209373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fsj.onc.1209373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=16550164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFyhtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1639&author=C.+C.+McIlwainauthor=D.+M.+Townsendauthor=K.+D.+Tew&title=Glutathione+S-transferase+polymorphisms%3A+cancer+incidence+and+therapy&doi=10.1038%2Fsj.onc.1209373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione S-transferase polymorphisms: cancer incidence and therapy</span></div><div class="casAuthors">McIlwain, C. C.; Townsend, D. M.; Tew, K. D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1639-1648</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The super family of glutathione S-transferases (GSTs) is composed of multiple isoenzymes with significant evidence of functional polymorphic variation.  Over the last three decades, data from cancer studies have linked aberrant expression of GST isoenzymes with the development and expression of resistance to a variety of chems., including cancer drugs.  This review addresses how differences in the human GST isoenzyme expression patterns influence cancer susceptibility, prognosis and treatment.  In addn. to the well-characterized catalytic activity, recent evidence has shown that certain GST isoenzymes can regulate mitogen-activated protein kinases or can facilitate the addn. of glutathione to cysteine residues in target proteins (S-glutathionylation).  These multiple functionalities have contributed to the recent efforts to target GSTs with novel small mol. therapeutics.  Presently, at least two drugs are in late-stage clin. testing.  The evolving functions of GST and their divergent expression patterns in individuals make them an attractive target for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkmAFQdntw6rVg90H21EOLACvtfcHk0ljchw1vC3m0WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFyhtbo%253D&md5=48a8873daaa35f6140dff9fa58ffcaac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1209373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1209373%26sid%3Dliteratum%253Aachs%26aulast%3DMcIlwain%26aufirst%3DC.%2BC.%26aulast%3DTownsend%26aufirst%3DD.%2BM.%26aulast%3DTew%26aufirst%3DK.%2BD.%26atitle%3DGlutathione%2520S-transferase%2520polymorphisms%253A%2520cancer%2520incidence%2520and%2520therapy%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D1639%26doi%3D10.1038%2Fsj.onc.1209373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oguztuzun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güler Simsek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ada, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocdogan, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaygın, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iscan, M.</span></span> <span> </span><span class="NLM_article-title">Expression of CYP and GST in human normal and colon tumor tissues</span>. <i>Biotech. Histochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1080/10520295.2018.1493220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1080%2F10520295.2018.1493220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=30092668" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1-9&author=H.+Bulusauthor=S.+Oguztuzunauthor=G.+G%C3%BCler+Simsekauthor=M.+Kilicauthor=A.+O.+Adaauthor=S.+G%C3%B6lauthor=A.+K.+Kocdoganauthor=P.+Kayg%C4%B1nauthor=B.+Bozerauthor=M.+Iscan&title=Expression+of+CYP+and+GST+in+human+normal+and+colon+tumor+tissues&doi=10.1080%2F10520295.2018.1493220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F10520295.2018.1493220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10520295.2018.1493220%26sid%3Dliteratum%253Aachs%26aulast%3DBulus%26aufirst%3DH.%26aulast%3DOguztuzun%26aufirst%3DS.%26aulast%3DG%25C3%25BCler%2BSimsek%26aufirst%3DG.%26aulast%3DKilic%26aufirst%3DM.%26aulast%3DAda%26aufirst%3DA.%2BO.%26aulast%3DG%25C3%25B6l%26aufirst%3DS.%26aulast%3DKocdogan%26aufirst%3DA.%2BK.%26aulast%3DKayg%25C4%25B1n%26aufirst%3DP.%26aulast%3DBozer%26aufirst%3DB.%26aulast%3DIscan%26aufirst%3DM.%26atitle%3DExpression%2520of%2520CYP%2520and%2520GST%2520in%2520human%2520normal%2520and%2520colon%2520tumor%2520tissues%26jtitle%3DBiotech.%2520Histochem.%26date%3D2018%26spage%3D1%26epage%3D9%26doi%3D10.1080%2F10520295.2018.1493220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piaggi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raggi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitzalis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascherpa, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saviozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casini, A. F.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferase omega 1-1 (GSTO1-1) plays an anti-apoptotic role in cell resistance to cisplatin toxicity</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">811</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1093%2Fcarcin%2Fbgq031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=20106899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVWjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=804-811&author=S.+Piaggiauthor=C.+Raggiauthor=A.+Cortiauthor=E.+Pitzalisauthor=M.+C.+Mascherpaauthor=M.+Saviozziauthor=A.+Pompellaauthor=A.+F.+Casini&title=Glutathione+transferase+omega+1-1+%28GSTO1-1%29+plays+an+anti-apoptotic+role+in+cell+resistance+to+cisplatin+toxicity&doi=10.1093%2Fcarcin%2Fbgq031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferase omega 1-1 (GSTO1-1) plays an anti-apoptotic role in cell resistance to cisplatin toxicity</span></div><div class="casAuthors">Piaggi, Simona; Raggi, Chiara; Corti, Alessandro; Pitzalis, Emanuela; Mascherpa, Marco C.; Saviozzi, Michela; Pompella, Alfonso; Casini, Alessandro F.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">804-811</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Several lines of evidence correlate the overexpression of glutathione S-transferase omega 1-1 (GSTO1-1) with the onset of drug resistance of cancer cells; however, no direct evidence is yet available.  In order to investigate the mechanisms involved, stable transfection with GSTO1-1 complementary DNA was performed in HeLa cells, which spontaneously express very low levels of GSTO1-1.  When transfected cells were seeded at low d., a sharp increase in GSTO1-1 expression was obsd. as compared with controls, along with an increased resistance against cisplatin cytotoxicity.  When seeded at increasing densities, control untransfected cells also presented with an increase in GSTO1-1 expression, again accompanied by cisplatin resistance; the latter was significantly reduced after transfection with GSTO1-1 small interfering RNA.  Cisplatin resistance of transfected cells was not accounted for by changes in the intracellular drug concn. nor in the amt. of DNA cross-links or content of glutathione.  Rather, transfected cells presented with a marked decrease of apoptosis as compared with controls, suggesting that GSTO1-1 overexpression may prevent cisplatin toxicity by interfering with the apoptotic process.  Cisplatin treatment was in fact followed at early times (1-2 h) by activation of both Akt kinase and extracellular signal-regulated kinase (ERK)-1/2 in the transfected cells but not in controls.  Conversely, in transfected cells, the strong activation of Jun N-terminal kinase (JNK)-1 induced by cisplatin at later times (10-20 h) was completely prevented.  In conclusion, GSTO1-1 overexpression appears to be assocd. with activation of survival pathways (Akt and ERK1/2) and inhibition of apoptotic pathways (JNK1), as well as protection against cisplatin-induced apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZYICy71RURrVg90H21EOLACvtfcHk0ljchw1vC3m0WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVWjsbo%253D&md5=0856d5d696594a33977e2c04d1a3adc8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq031%26sid%3Dliteratum%253Aachs%26aulast%3DPiaggi%26aufirst%3DS.%26aulast%3DRaggi%26aufirst%3DC.%26aulast%3DCorti%26aufirst%3DA.%26aulast%3DPitzalis%26aufirst%3DE.%26aulast%3DMascherpa%26aufirst%3DM.%2BC.%26aulast%3DSaviozzi%26aufirst%3DM.%26aulast%3DPompella%26aufirst%3DA.%26aulast%3DCasini%26aufirst%3DA.%2BF.%26atitle%3DGlutathione%2520transferase%2520omega%25201-1%2520%2528GSTO1-1%2529%2520plays%2520an%2520anti-apoptotic%2520role%2520in%2520cell%2520resistance%2520to%2520cisplatin%2520toxicity%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D804%26epage%3D811%26doi%3D10.1093%2Fcarcin%2Fbgq031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.-d.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, N.</span></span> <span> </span><span class="NLM_article-title">Identification of platinum-resistance associated proteins through proteomic analysis of human ovarian cancer cells and their platinum-resistant sublines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1021/pr060402r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr060402r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlaru7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=772-780&author=X.-d.+Yanauthor=L.-y.+Panauthor=Y.+Yuanauthor=J.-h.+Langauthor=N.+Mao&title=Identification+of+platinum-resistance+associated+proteins+through+proteomic+analysis+of+human+ovarian+cancer+cells+and+their+platinum-resistant+sublines&doi=10.1021%2Fpr060402r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Platinum-Resistance Associated Proteins through Proteomic Analysis of Human Ovarian Cancer Cells and Their Platinum-Resistant Sublines</span></div><div class="casAuthors">Yan, Xue-dong; Pan, Ling-ya; Yuan, Ye; Lang, Jing-he; Mao, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">772-780</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemoresistance is a major therapeutic obstacle in cancer patients, and the mechanisms of drug resistance are not fully understood.  In the present study, the authors established platinum-resistant human ovarian cancer cell lines and identified differentially expressed proteins related to platinum resistance.  The total proteins of two sensitive (SKOV3 and A2780) and four resistant (SKOV3/CDDP, SKOV3/CBP, A2780/CDDP, and A2780/CBP) human ovarian cancer cell lines were isolated by two-dimensional gel electrophoresis (2-DE).  The differentially expressed proteins were identified using matrix-assisted laser desorption ionization-time-of-flight mass spectrometry (MALDI-TOF MS).  In total, 57 differential protein spots were identified, and five proteins, including annexin A3, destrin, cofilin 1, glutathione-S-transferase omega 1, and cytosolic NADP+-dependent isocitrate dehydrogenase, were found to be coinstantaneous significance compared with their parental cells.  The expression of the five proteins was validated by quant. PCR and Western blot, and the Western blot results showed complete consistency with proteomic techniques.  The five proteins are hopeful to become candidates for platinum resistance.  These may be useful for further study of resistance mechanisms and screening of resistant biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGBj_whnFuILVg90H21EOLACvtfcHk0lhmLw28n2X6Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlaru7jO&md5=c4c2852eaf4df3fb4b6a133e4b5a80a9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fpr060402r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr060402r%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.-d.%26aulast%3DPan%26aufirst%3DL.-y.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DLang%26aufirst%3DJ.-h.%26aulast%3DMao%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520platinum-resistance%2520associated%2520proteins%2520through%2520proteomic%2520analysis%2520of%2520human%2520ovarian%2520cancer%2520cells%2520and%2520their%2520platinum-resistant%2520sublines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2007%26volume%3D6%26spage%3D772%26epage%3D780%26doi%3D10.1021%2Fpr060402r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Innes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlstrom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüstle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarotto, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">GSTO1-1 plays a pro-inflammatory role in models of inflammation, colitis and obesity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17832</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-17861-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fs41598-017-17861-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=29259211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjtlehtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=17832&author=D.+Menonauthor=A.+Innesauthor=A.+J.+Oakleyauthor=J.+E.+Dahlstromauthor=L.+M.+Jensenauthor=A.+Br%C3%BCstleauthor=P.+Tummalaauthor=M.+Rookeauthor=M.+G.+Casarottoauthor=J.+B.+Baell&title=GSTO1-1+plays+a+pro-inflammatory+role+in+models+of+inflammation%2C+colitis+and+obesity&doi=10.1038%2Fs41598-017-17861-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">GSTO1-1 plays a pro-inflammatory role in models of inflammation, colitis and obesity</span></div><div class="casAuthors">Menon Deepthi; Innes Ashlee; Jensen Lora M; Brustle Anne; Tummala Padmaja; Rooke Melissa; Casarotto Marco G; Burgio Gaetan; Board Philip G; Menon Deepthi; O'Neill Luke A J; Oakley Aaron J; Dahlstrom Jane E; Baell Jonathan B; Nguyen Nghi; Xie Yiyue; Baell Jonathan B; Cuellar Matthew; Strasser Jessica; Walters Michael A; Dahlin Jayme L</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17832</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glutathione transferase Omega 1 (GSTO1-1) is an atypical GST reported to play a pro-inflammatory role in response to LPS.  Here we show that genetic knockout of Gsto1 alters the response of mice to three distinct inflammatory disease models.  GSTO1-1 deficiency ameliorates the inflammatory response stimulated by LPS and attenuates the inflammatory impact of a high fat diet on glucose tolerance and insulin resistance.  In contrast, GSTO1-1 deficient mice show a more severe inflammatory response and increased escape of bacteria from the colon into the lymphatic system in a dextran sodium sulfate mediated model of inflammatory bowel disease.  These responses are similar to those of TLR4 and MyD88 deficient mice in these models and confirm that GSTO1-1 is critical for a TLR4-like pro-inflammatory response in vivo.  In wild-type mice, we show that a small molecule inhibitor that covalently binds in the active site of GSTO1-1 can be used to ameliorate the inflammatory response to LPS.  Our findings demonstrate the potential therapeutic utility of GSTO1-1 inhibitors in the modulation of inflammation and suggest their possible application in the treatment of a range of inflammatory conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeIFfZgL8lsjl5L8hwNX0fW6udTcc2eYny8TqnsDkhLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjtlehtg%253D%253D&md5=360b424b4c6c9409e7c2836559f61ff1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-17861-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-17861-6%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DD.%26aulast%3DInnes%26aufirst%3DA.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DDahlstrom%26aufirst%3DJ.%2BE.%26aulast%3DJensen%26aufirst%3DL.%2BM.%26aulast%3DBr%25C3%25BCstle%26aufirst%3DA.%26aulast%3DTummala%26aufirst%3DP.%26aulast%3DRooke%26aufirst%3DM.%26aulast%3DCasarotto%26aufirst%3DM.%2BG.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DGSTO1-1%2520plays%2520a%2520pro-inflammatory%2520role%2520in%2520models%2520of%2520inflammation%252C%2520colitis%2520and%2520obesity%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D17832%26doi%3D10.1038%2Fs41598-017-17861-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laliberte, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perregaux, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peese, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombroski, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabel, C. A.</span></span> <span> </span><span class="NLM_article-title">Glutathiones-transferase omega 1-1 is a target of cytokine release inhibitory drugs and may be responsible for their effect on interleukin-1β posttranslational processing</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">16567</span>– <span class="NLM_lpage">16578</span>, <span class="refDoi"> DOI: 10.1074/jbc.m211596200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1074%2Fjbc.m211596200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=16567-16578&author=R.+E.+Laliberteauthor=D.+G.+Perregauxauthor=L.+R.+Hothauthor=P.+J.+Rosnerauthor=C.+K.+Jordanauthor=K.+M.+Peeseauthor=J.+F.+Egglerauthor=M.+A.+Dombroskiauthor=K.+F.+Geogheganauthor=C.+A.+Gabel&title=Glutathiones-transferase+omega+1-1+is+a+target+of+cytokine+release+inhibitory+drugs+and+may+be+responsible+for+their+effect+on+interleukin-1%CE%B2+posttranslational+processing&doi=10.1074%2Fjbc.m211596200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m211596200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m211596200%26sid%3Dliteratum%253Aachs%26aulast%3DLaliberte%26aufirst%3DR.%2BE.%26aulast%3DPerregaux%26aufirst%3DD.%2BG.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DRosner%26aufirst%3DP.%2BJ.%26aulast%3DJordan%26aufirst%3DC.%2BK.%26aulast%3DPeese%26aufirst%3DK.%2BM.%26aulast%3DEggler%26aufirst%3DJ.%2BF.%26aulast%3DDombroski%26aufirst%3DM.%2BA.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DGabel%26aufirst%3DC.%2BA.%26atitle%3DGlutathiones-transferase%2520omega%25201-1%2520is%2520a%2520target%2520of%2520cytokine%2520release%2520inhibitory%2520drugs%2520and%2520may%2520be%2520responsible%2520for%2520their%2520effect%2520on%2520interleukin-1%25CE%25B2%2520posttranslational%2520processing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D16567%26epage%3D16578%26doi%3D10.1074%2Fjbc.m211596200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vega, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of GlutathioneS-Transferase Omega 1 That Impair Cancer Drug Resistance</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">16605</span>– <span class="NLM_lpage">16616</span>, <span class="refDoi"> DOI: 10.1021/ja2066972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2066972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16605-16616&author=K.+Tsuboiauthor=D.+A.+Bachovchinauthor=A.+E.+Speersauthor=T.+P.+Spicerauthor=V.+Fernandez-Vegaauthor=P.+Hodderauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Potent+and+Selective+Inhibitors+of+GlutathioneS-Transferase+Omega+1+That+Impair+Cancer+Drug+Resistance&doi=10.1021%2Fja2066972"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja2066972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2066972%26sid%3Dliteratum%253Aachs%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DSpicer%26aufirst%3DT.%2BP.%26aulast%3DFernandez-Vega%26aufirst%3DV.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520GlutathioneS-Transferase%2520Omega%25201%2520That%2520Impair%2520Cancer%2520Drug%2520Resistance%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16605%26epage%3D16616%26doi%3D10.1021%2Fja2066972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span> <span> </span><span class="NLM_article-title">Glutathione transferase omega 1 is required for the lipopolysaccharide-stimulated induction of NADPH oxidase 1 and the production of reactive oxygen species in macrophages</span>. <i>Free Radic. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2014.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.freeradbiomed.2014.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=24873723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1SksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2014&pages=318-327&author=D.+Menonauthor=R.+Collauthor=P.+G.+Board&title=Glutathione+transferase+omega+1+is+required+for+the+lipopolysaccharide-stimulated+induction+of+NADPH+oxidase+1+and+the+production+of+reactive+oxygen+species+in+macrophages&doi=10.1016%2Fj.freeradbiomed.2014.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferase Omega 1 is required for the lipopolysaccharide-stimulated induction of NADPH oxidase 1 and the production of reactive oxygen species in macrophages</span></div><div class="casAuthors">Menon, Deepthi; Coll, Rebecca; O'Neill, Luke A. J.; Board, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">318-327</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bacterial lipopolysaccharide (LPS) stimulation of macrophages and inflammation via the Toll-like receptor 4 (TLR4) signaling pathway through NF-κB generates reactive oxygen species (ROS) and proinflammatory cytokines such as IL-1β, IL-6, and TNFα.  Because glutathione transferase Omega 1-1 (GSTO1-1) can catalyze redox reactions such as the deglutathionylation of proteins and has also been implicated in the release of IL-1β the authors investigated its role in the development of LPS-mediated inflammation.  The data show that shRNA knockdown of GSTO1-1 in macrophage-like J774.1A cells blocks the expression of NADPH oxidase 1 and the generation of ROS after LPS stimulation.  Similar results were obtained with a GSTO1-1 inhibitor.  To maintain high ROS levels during an inflammatory response, LPS stimulation causes the suppression of enzymes such as catalase and glutathione peroxidase that protect against oxidative stress.  The knockdown of GSTO1-1 also attenuates this response.  The data indicate that GSTO1-1 needs to be catalytically active and mediates its effects on the LPS/TLR4 inflammatory pathway upstream of NF-κB.  These data suggest that GSTO1-1 is a novel target for anti-inflammatory intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZlg072KwT7Vg90H21EOLACvtfcHk0ljmQ81p3eXriw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1SksrvE&md5=c7d253152bfa0c535a89f798a47a6f46</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2014.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2014.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DD.%26aulast%3DColl%26aufirst%3DR.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26atitle%3DGlutathione%2520transferase%2520omega%25201%2520is%2520required%2520for%2520the%2520lipopolysaccharide-stimulated%2520induction%2520of%2520NADPH%2520oxidase%25201%2520and%2520the%2520production%2520of%2520reactive%2520oxygen%2520species%2520in%2520macrophages%26jtitle%3DFree%2520Radic.%2520Biol.%2520Med.%26date%3D2014%26volume%3D73%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fj.freeradbiomed.2014.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Board, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brighty, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bare, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, E. T.</span></span> <span> </span><span class="NLM_article-title">“Inverse drug discovery” strategy to identify proteins that are targeted by latent electrophiles as exemplified by aryl fluorosulfates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b08366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2017&pages=200-210&author=D.+E.+Boardauthor=G.+J.+Brightyauthor=L.+Plateauthor=G.+Bareauthor=W.+Chenauthor=S.+Liauthor=H.+Wangauthor=B.+F.+Cravattauthor=S.+Forliauthor=E.+T.+Powers&title=%E2%80%9CInverse+drug+discovery%E2%80%9D+strategy+to+identify+proteins+that+are+targeted+by+latent+electrophiles+as+exemplified+by+aryl+fluorosulfates&doi=10.1021%2Fjacs.7b08366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08366%26sid%3Dliteratum%253Aachs%26aulast%3DBoard%26aufirst%3DD.%2BE.%26aulast%3DBrighty%26aufirst%3DG.%2BJ.%26aulast%3DPlate%26aufirst%3DL.%26aulast%3DBare%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DPowers%26aufirst%3DE.%2BT.%26atitle%3D%25E2%2580%259CInverse%2520drug%2520discovery%25E2%2580%259D%2520strategy%2520to%2520identify%2520proteins%2520that%2520are%2520targeted%2520by%2520latent%2520electrophiles%2520as%2520exemplified%2520by%2520aryl%2520fluorosulfates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D140%26spage%3D200%26epage%3D210%26doi%3D10.1021%2Fjacs.7b08366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppenheimer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, T.</span></span> <span> </span><span class="NLM_article-title">Chemical and computational methods for the characterization of covalent reactive groups for the prospective design of irreversible inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10072</span>– <span class="NLM_lpage">10079</span>, <span class="refDoi"> DOI: 10.1021/jm501412a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501412a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10072-10079&author=M.+E.+Flanaganauthor=J.+A.+Abramiteauthor=D.+P.+Andersonauthor=A.+Aulabaughauthor=U.+P.+Dahalauthor=A.+M.+Gilbertauthor=C.+Liauthor=J.+Montgomeryauthor=S.+R.+Oppenheimerauthor=T.+Ryder&title=Chemical+and+computational+methods+for+the+characterization+of+covalent+reactive+groups+for+the+prospective+design+of+irreversible+inhibitors&doi=10.1021%2Fjm501412a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Computational Methods for the Characterization of Covalent Reactive Groups for the Prospective Design of Irreversible Inhibitors</span></div><div class="casAuthors">Flanagan, Mark E.; Abramite, Joseph A.; Anderson, Dennis P.; Aulabaugh, Ann; Dahal, Upendra P.; Gilbert, Adam M.; Li, Chao; Montgomery, Justin; Oppenheimer, Stacey R.; Ryder, Tim; Schuff, Brandon P.; Uccello, Daniel P.; Walker, Gregory S.; Wu, Yan; Brown, Matthew F.; Chen, Jinshan M.; Hayward, Matthew M.; Noe, Mark C.; Obach, R. Scott; Philippe, Laurence; Shanmugasundaram, Veerabahu; Shapiro, Michael J.; Starr, Jeremy; Stroh, Justin; Che, Ye</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10072-10079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interest in drugs that covalently modify their target is driven by the desire for enhanced efficacy that can result from the silencing of enzymic activity until protein resynthesis can occur, along with the potential for increased selectivity by targeting uniquely positioned nucleophilic residues in the protein.  However, covalent approaches carry addnl. risk for toxicities or hypersensitivity reactions that can result from covalent modification of unintended targets.  Here we describe methods for measuring the reactivity of covalent reactive groups (CRGs) with a biol. relevant nucleophile, glutathione (GSH), along with kinetic data for a broad array of electrophiles.  We also describe a computational method for predicting electrophilic reactivity, which taken together can be applied to the prospective design of thiol-reactive covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzTSCv_Nj6bVg90H21EOLACvtfcHk0ljmQ81p3eXriw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM&md5=3ccf59ab7a494655185e5eb5becf8c48</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm501412a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501412a%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DOppenheimer%26aufirst%3DS.%2BR.%26aulast%3DRyder%26aufirst%3DT.%26atitle%3DChemical%2520and%2520computational%2520methods%2520for%2520the%2520characterization%2520of%2520covalent%2520reactive%2520groups%2520for%2520the%2520prospective%2520design%2520of%2520irreversible%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10072%26epage%3D10079%26doi%3D10.1021%2Fjm501412a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>– <span class="NLM_lpage">6262</span>, <span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lh3FjFWXFIStA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummond, K. M.</span></span> <span> </span><span class="NLM_article-title">Covalent modifiers: a chemical perspective on the reactivity of α,β-unsaturated carbonyls with thiols via hetero-michael addition reactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00788</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00788" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=839-885&author=P.+A.+Jacksonauthor=J.+C.+Widenauthor=D.+A.+Harkiauthor=K.+M.+Brummond&title=Covalent+modifiers%3A+a+chemical+perspective+on+the+reactivity+of+%CE%B1%2C%CE%B2-unsaturated+carbonyls+with+thiols+via+hetero-michael+addition+reactions&doi=10.1021%2Facs.jmedchem.6b00788"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00788%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DP.%2BA.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DBrummond%26aufirst%3DK.%2BM.%26atitle%3DCovalent%2520modifiers%253A%2520a%2520chemical%2520perspective%2520on%2520the%2520reactivity%2520of%2520%25CE%25B1%252C%25CE%25B2-unsaturated%2520carbonyls%2520with%2520thiols%2520via%2520hetero-michael%2520addition%2520reactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D60%26spage%3D839%26epage%3D885%26doi%3D10.1021%2Facs.jmedchem.6b00788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teusch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aricioglu, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowski, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neudörfl, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griesbeck, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietsch, M.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of glutathione transferase P1 with trioxane structure as anticancer agents</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201402553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fcmdc.201402553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=25694385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A280%3ADC%252BC2MrmslCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=629-639&author=M.+Br%C3%A4utigamauthor=N.+Teuschauthor=T.+Schenkauthor=M.+Sheikhauthor=R.+Z.+Ariciogluauthor=S.+H.+Borowskiauthor=J.-M.+Neud%C3%B6rflauthor=U.+Baumannauthor=A.+G.+Griesbeckauthor=M.+Pietsch&title=Selective+inhibitors+of+glutathione+transferase+P1+with+trioxane+structure+as+anticancer+agents&doi=10.1002%2Fcmdc.201402553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of glutathione transferase P1 with trioxane structure as anticancer agents</span></div><div class="casAuthors">Brautigam Maria; Teusch Nicole; Schenk Tobias; Sheikh Miriam; Aricioglu Rocky Z; Borowski Swantje H; Neudorfl Jorg-Martin; Baumann Ulrich; Griesbeck Axel G; Pietsch Markus</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">629-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The response to chemotherapy in cancer patients is frequently compromised by drug resistance.  Although chemoresistance is a multifactorial phenomenon, many studies have demonstrated that altered drug metabolism through the expression of phase II conjugating enzymes, including glutathione transferases (GSTs), in tumor cells can be directly correlated with resistance against a wide range of marketed anticancer drugs.  In particular, overexpression of glutathione transferase P1 (GSTP1) appears to be a factor for poor prognosis during cancer therapy.  Former and ongoing clinical trials have confirmed GSTP1 inhibition as a principle for antitumor therapy.  A new series of 1,2,4-trioxane GSTP1 inhibitors were designed via a type II photooxygenation route of allylic alcohols followed by acid-catalyzed peroxyacetalization with aldehydes.  A set of novel inhibitors exhibit low micromolar to high nanomolar inhibition of GSTP1, revealing preliminary SAR for further lead optimization.  Importantly, high selectivity over another two human GST classes (GSTA1 and GSTM2) has been achieved.  The trioxane GSTP1 inhibitors may therefore serve as a basis for the development of novel drug candidates in overcoming chemoresistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpuEoHSV8gu8uCWE5mhx1RfW6udTcc2eZlJ0Tvw-BPiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrmslCjtQ%253D%253D&md5=194c663eb80b74f2444ec77f67bc5ce2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402553%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DM.%26aulast%3DTeusch%26aufirst%3DN.%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DSheikh%26aufirst%3DM.%26aulast%3DAricioglu%26aufirst%3DR.%2BZ.%26aulast%3DBorowski%26aufirst%3DS.%2BH.%26aulast%3DNeud%25C3%25B6rfl%26aufirst%3DJ.-M.%26aulast%3DBaumann%26aufirst%3DU.%26aulast%3DGriesbeck%26aufirst%3DA.%2BG.%26aulast%3DPietsch%26aufirst%3DM.%26atitle%3DSelective%2520inhibitors%2520of%2520glutathione%2520transferase%2520P1%2520with%2520trioxane%2520structure%2520as%2520anticancer%2520agents%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D629%26epage%3D639%26doi%3D10.1002%2Fcmdc.201402553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aeluri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polepalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antiproliferative activity of imidazo[1,2-a]pyrimidine Mannich bases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.05.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ejmech.2015.05.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=26067381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVejtbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=18-23&author=R.+Aeluriauthor=M.+Allaauthor=S.+Polepalliauthor=N.+Jain&title=Synthesis+and+antiproliferative+activity+of+imidazo%5B1%2C2-a%5Dpyrimidine+Mannich+bases&doi=10.1016%2Fj.ejmech.2015.05.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antiproliferative activity of imidazo[1,2-a]pyrimidine Mannich bases</span></div><div class="casAuthors">Aeluri, Raghunath; Alla, Manjula; Polepalli, Sowjanya; Jain, Nishant</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-23</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of imidazo[1,2-a]pyrimidine Mannich bases were designed, synthesized in two phases.  Mannich bases were obtained by one pot three component condensation of imidazo[1,2-a]pyrimidine with secondary amine or piperazine and excess of formaldehyde soln. in methanol.  The synthesized Mannich bases were screened for in vitro growth inhibition against a panel of 3 different human cancer cell lines.  Most of the synthesized compds. exhibited antiproliferative activity with GI50 values ranging from 0.01 to 79.4 μM.  Compds. 2-Phenyl-3-(pyrrolidin-1-ylmethyl)imidazo[1,2-a]pyrimidine, ((4-methylpiperazin-1-yl)methyl)-2-phenylimidazo[1,2-a]pyrimidine and 1,4-bis((2-(2-Tolyl)imidazo[1,2-a]pyrimidin-3-yl)methyl)piperazine are effective inhibitors of growth of all cell lines, with GI50 values similar to that of std. drug.  The structure and activity relation was disclosed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr59HvD-9vSybVg90H21EOLACvtfcHk0ljFX33INPOUVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVejtbfJ&md5=98d77ffbe40e8788f58548edbc10cf29</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.05.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.05.037%26sid%3Dliteratum%253Aachs%26aulast%3DAeluri%26aufirst%3DR.%26aulast%3DAlla%26aufirst%3DM.%26aulast%3DPolepalli%26aufirst%3DS.%26aulast%3DJain%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520antiproliferative%2520activity%2520of%2520imidazo%255B1%252C2-a%255Dpyrimidine%2520Mannich%2520bases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D100%26spage%3D18%26epage%3D23%26doi%3D10.1016%2Fj.ejmech.2015.05.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sravanthi, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manju, S. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Fluorescence Properties of Novel indol-3yl-thiazolo[3,2-a][1,3,5]triazines and indole-3-carbaldehyde Schiff Bases</span>. <i>J. Fluoresc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1738</span>, <span class="refDoi"> DOI: 10.1007/s10895-015-1659-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1007%2Fs10895-015-1659-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=26432553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1alu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1727-1738&author=T.+V.+Sravanthiauthor=S.+L.+Manju&title=Synthesis+and+Fluorescence+Properties+of+Novel+indol-3yl-thiazolo%5B3%2C2-a%5D%5B1%2C3%2C5%5Dtriazines+and+indole-3-carbaldehyde+Schiff+Bases&doi=10.1007%2Fs10895-015-1659-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Fluorescence Properties of Novel indol-3yl-thiazolo[3,2-a][1,3,5]triazines and indole-3-carbaldehyde Schiff Bases</span></div><div class="casAuthors">Sravanthi, T. V.; Manju, S. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorescence</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1727-1738</span>CODEN:
                <span class="NLM_cas:coden">JOFLEN</span>;
        ISSN:<span class="NLM_cas:issn">1053-0509</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Novel photoactive 4-(4-chlorophenyl)-2-(1H-indol-3-yl)-6-substituted phenyl-2H-thiazolo[3,2-a][1,3,5]triazines were synthesized by the conjugate addn. of ammonia to the indole-3-carbaldehyde Schiff bases followed by the condensation with 4-chlorobenzaldehyde.  All the synthesized compds. were characterized by FT-IR, NMR, mass spectra and elemental analyses.  Their antioxidant property, electrochem. and photophys. properties in different org. solvents were investigated.  Comparative discussion on the photophys. properties of indole-3-carbaldehyde Schiff bases and 4-(4-chlorophenyl)-2-(1H-indol-3-yl)-6-substituted phenyl-2H-thiazolo[3,2-a][1,3,5]triazines were described.  The fluorescence quantum yield of Schiff bases (φf = 0.66-0.70 in DMSO) found to be interestingly higher.  High fluorescence quantum yield, large molar extinction coeff., high stokes shift and smaller optical band gap positioning these new derivs. as an efficient metal free org. fluorescent and semiconductor material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFcI3h-RDgr7Vg90H21EOLACvtfcHk0ljFX33INPOUVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1alu7rF&md5=094949491a7e875d3970ca854bebea23</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10895-015-1659-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10895-015-1659-1%26sid%3Dliteratum%253Aachs%26aulast%3DSravanthi%26aufirst%3DT.%2BV.%26aulast%3DManju%26aufirst%3DS.%2BL.%26atitle%3DSynthesis%2520and%2520Fluorescence%2520Properties%2520of%2520Novel%2520indol-3yl-thiazolo%255B3%252C2-a%255D%255B1%252C3%252C5%255Dtriazines%2520and%2520indole-3-carbaldehyde%2520Schiff%2520Bases%26jtitle%3DJ.%2520Fluoresc.%26date%3D2015%26volume%3D25%26spage%3D1727%26epage%3D1738%26doi%3D10.1007%2Fs10895-015-1659-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strobel, H.</span>; <span class="NLM_string-name">Wohlfart, P.</span>; <span class="NLM_string-name">Zoller, G.</span>; <span class="NLM_string-name">Will, D.
W.</span></span> <span> </span><span class="NLM_article-title">Heteroaryl-substituted amides comprising a saturated linker group, and their use as pharmaceuticals</span>. U.S. Patent <span class="NLM_patent">851,897,6B2</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Strobel&author=P.+Wohlfart&author=G.+Zoller&author=D.%0AW.+Will&title=Heteroaryl-substituted+amides+comprising+a+saturated+linker+group%2C+and+their+use+as+pharmaceuticals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStrobel%26aufirst%3DH.%26atitle%3DHeteroaryl-substituted%2520amides%2520comprising%2520a%2520saturated%2520linker%2520group%252C%2520and%2520their%2520use%2520as%2520pharmaceuticals%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarotto, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">Reviewing hit discovery literature for difficult targets: glutathione transferase omega-1 as an example</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7448</span>– <span class="NLM_lpage">7470</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00318</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00318" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7448-7470&author=Y.+Xieauthor=J.+L.+Dahlinauthor=A.+J.+Oakleyauthor=M.+G.+Casarottoauthor=P.+G.+Boardauthor=J.+B.+Baell&title=Reviewing+hit+discovery+literature+for+difficult+targets%3A+glutathione+transferase+omega-1+as+an+example&doi=10.1021%2Facs.jmedchem.8b00318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reviewing Hit Discovery Literature for Difficult Targets: Glutathione Transferase Omega-1 as an Example</span></div><div class="casAuthors">Xie, Yiyue; Dahlin, Jayme L.; Oakley, Aaron J.; Casarotto, Marco G.; Board, Philip G.; Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7448-7470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Early stage drug discovery reporting on relatively new or difficult targets is often assocd. with insufficient hit triage.  Literature reviews of such targets seldom delve into the detail required to critically analyze the assocd. screening hits reported.  Here we take the enzyme glutathione transferase omega-1 (GSTO1-1) as an example of a relatively difficult target and review the assocd. literature involving small-mol. inhibitors.  As part of this process we deliberately pay closer-than-usual attention to assay interference and hit quality aspects.  We believe this Perspective will be a useful guide for future development of GSTO1-1 inhibitors, as well serving as a template for future review formats of new or difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4iyFIZWXQKLVg90H21EOLACvtfcHk0ljFX33INPOUVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSgsbY%253D&md5=31037759dc901cb8b1f5a78877fa237c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00318%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DOakley%26aufirst%3DA.%2BJ.%26aulast%3DCasarotto%26aufirst%3DM.%2BG.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DReviewing%2520hit%2520discovery%2520literature%2520for%2520difficult%2520targets%253A%2520glutathione%2520transferase%2520omega-1%2520as%2520an%2520example%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7448%26epage%3D7470%26doi%3D10.1021%2Facs.jmedchem.8b00318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Son, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-T.</span></span> <span> </span><span class="NLM_article-title">Isozyme-specific fluorescent inhibitor of glutathione s-transferase omega 1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1021/cb100007s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100007s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=449-453&author=J.+Sonauthor=J.-J.+Leeauthor=J.-S.+Leeauthor=A.+Sch%C3%BCllerauthor=Y.-T.+Chang&title=Isozyme-specific+fluorescent+inhibitor+of+glutathione+s-transferase+omega+1&doi=10.1021%2Fcb100007s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcb100007s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100007s%26sid%3Dliteratum%253Aachs%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.-J.%26aulast%3DLee%26aufirst%3DJ.-S.%26aulast%3DSch%25C3%25BCller%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DY.-T.%26atitle%3DIsozyme-specific%2520fluorescent%2520inhibitor%2520of%2520glutathione%2520s-transferase%2520omega%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D449%26epage%3D453%26doi%3D10.1021%2Fcb100007s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierschke, S. G.</span></span> <span> </span><span class="NLM_article-title">Comparison of computational methods applied to oxazole, thiazole, and other heterocyclic compounds</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1002/jcc.540140112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fjcc.540140112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=75-88&author=A.+A.+Shafferauthor=S.+G.+Wierschke&title=Comparison+of+computational+methods+applied+to+oxazole%2C+thiazole%2C+and+other+heterocyclic+compounds&doi=10.1002%2Fjcc.540140112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540140112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540140112%26sid%3Dliteratum%253Aachs%26aulast%3DShaffer%26aufirst%3DA.%2BA.%26aulast%3DWierschke%26aufirst%3DS.%2BG.%26atitle%3DComparison%2520of%2520computational%2520methods%2520applied%2520to%2520oxazole%252C%2520thiazole%252C%2520and%2520other%2520heterocyclic%2520compounds%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D75%26epage%3D88%26doi%3D10.1002%2Fjcc.540140112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency as a guide in fragment hit selection and optimization</span>. <i>Discovery Today: Technol</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e157</span>– <span class="NLM_lpage">e162</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2Fj.ddtec.2010.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVegsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e157-e162&author=S.+Schultesauthor=C.+de+Graafauthor=E.+E.+J.+Haaksmaauthor=I.+J.+P.+de+Eschauthor=R.+Leursauthor=O.+Kr%C3%A4mer&title=Ligand+efficiency+as+a+guide+in+fragment+hit+selection+and+optimization&doi=10.1016%2Fj.ddtec.2010.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency as a guide in fragment hit selection and optimization</span></div><div class="casAuthors">Schultes, Sabine; de Graaf, Chris; Haaksma, Eric E. J.; de Esch, Iwan J. P.; Leurs, Rob; Kraemer, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E157-E162</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based screening (FBS) has become an established approach for hit identification.  Starting points identified by FBS, are small fragments that require substantial modification to become leads.  As fragments are different from classical hits a process tailored for fragment evolution is required.  Scores for ligand efficiency have been proposed as guides for this process.  Here we review how these have been applied to guide the selection and optimization of fragment hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfhwn68rv4bVg90H21EOLACvtfcHk0ljuZ5YAy9kn7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVegsL0%253D&md5=a57c85869d5c829159be31449fd04796</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchultes%26aufirst%3DS.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DHaaksma%26aufirst%3DE.%2BE.%2BJ.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%26atitle%3DLigand%2520efficiency%2520as%2520a%2520guide%2520in%2520fragment%2520hit%2520selection%2520and%2520optimization%26jtitle%3DDiscovery%2520Today%253A%2520Technol%26date%3D2010%26volume%3D7%26spage%3De157%26epage%3De162%26doi%3D10.1016%2Fj.ddtec.2010.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Disco.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljuZ5YAy9kn7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Disco.%26date%3D2007%26volume%3D6%26spage%3D881%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span> <i>Maestro</i>, version 10.1; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+version+10.1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lgtr_Exfy85JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span> <i>Glide</i>, version 6.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+version+6.8%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span> <i>LigPrep</i>, version 3.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+version+3.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <i>Methods Enzymol.</i>; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+Enzymol."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520Enzymol.%26pub%3DElsevier%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgtr_Exfy85JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljhtBv3xYHbYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr. Sect. D. Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljhtBv3xYHbYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D.%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER, 2.10.3</i>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER%2C+2.10.3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%252C%25202.10.3%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE, 1.2.13</i>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE%2C+1.2.13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%252C%25201.2.13%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D. Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D.%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="note"><p class="first last">wwPDB Validation Server.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EEM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EEM','PDB','1EEM'); return false;">PDB: 1EEM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V3Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V3Q','PDB','5V3Q'); return false;">PDB: 5V3Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEH','PDB','5UEH'); return false;">PDB: 5UEH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQM','PDB','4YQM'); return false;">PDB: 4YQM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQV','PDB','4YQV'); return false;">PDB: 4YQV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB','PDB','6MHB'); return false;">PDB: 6MHB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC','PDB','6MHC'); return false;">PDB: 6MHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD','PDB','6MHD'); return false;">PDB: 6MHD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHA','PDB','6MHA'); return false;">PDB: 6MHA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YQU','PDB','4YQU'); return false;">PDB: 4YQU</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i131"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01960">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31622"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01960">10.1021/acs.jmedchem.8b01960</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings including screening data, X-ray diffraction data, NMR spectra, and HPLC chromatograms (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB ID of new crystal (X-ray) structures/homology models: GSTO1:<b>18</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHB">6MHB</a>), GSTO1:<b>37</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHC">6MHC</a>), and GSTO1:<b>44</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MHD">6MHD</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_001.pdf">jm8b01960_si_001.pdf (25.32 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01960/suppl_file/jm8b01960_si_002.csv">jm8b01960_si_002.csv (3.58 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01960&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01960%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-6%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01960" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679abc29095cd1db","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
