
<html lang="en"     class="pb-page"  data-request-id="dfa2bc18-610e-4300-92e0-e38da01cc780"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.0c01332;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Evolution of Novartis’ Small Molecule Screening Deck Design" /></meta><meta name="dc.Creator" content="Ansgar  Schuffenhauer" /></meta><meta name="dc.Creator" content="Nadine  Schneider" /></meta><meta name="dc.Creator" content="Samuel  Hintermann" /></meta><meta name="dc.Creator" content="Douglas  Auld" /></meta><meta name="dc.Creator" content="Jutta  Blank" /></meta><meta name="dc.Creator" content="Simona  Cotesta" /></meta><meta name="dc.Creator" content="Caroline  Engeloch" /></meta><meta name="dc.Creator" content="Nikolas  Fechner" /></meta><meta name="dc.Creator" content="Christoph  Gaul" /></meta><meta name="dc.Creator" content="Jerome  Giovannoni" /></meta><meta name="dc.Creator" content="Johanna  Jansen" /></meta><meta name="dc.Creator" content="John  Joslin" /></meta><meta name="dc.Creator" content="Philipp  Krastel" /></meta><meta name="dc.Creator" content="Eugen  Lounkine" /></meta><meta name="dc.Creator" content="John  Manchester" /></meta><meta name="dc.Creator" content="Lauren G.  Monovich" /></meta><meta name="dc.Creator" content="Anna Paola  Pelliccioli" /></meta><meta name="dc.Creator" content="Manuel  Schwarze" /></meta><meta name="dc.Creator" content="Michael D.  Shultz" /></meta><meta name="dc.Creator" content="Nikolaus  Stiefl" /></meta><meta name="dc.Creator" content="Daniel K.  Baeschlin" /></meta><meta name="dc.Description" content="This article summarizes the evolution of the screening deck at the Novartis Institutes for BioMedical Research (NIBR). Historically, the screening deck was an assembly of all available compounds. I..." /></meta><meta name="Description" content="This article summarizes the evolution of the screening deck at the Novartis Institutes for BioMedical Research (NIBR). Historically, the screening deck was an assembly of all available compounds. I..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 3, 2020" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01332" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01332" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01332" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01332" /></link>
        
    
    

<title>Evolution of Novartis’ Small Molecule Screening Deck Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01332" /></meta><meta property="og:title" content="Evolution of Novartis’ Small Molecule Screening Deck Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0014.jpeg" /></meta><meta property="og:description" content="This article summarizes the evolution of the screening deck at the Novartis Institutes for BioMedical Research (NIBR). Historically, the screening deck was an assembly of all available compounds. In 2015, we designed a first deck to facilitate access to diverse subsets with optimized properties. We allocated the compounds as plated subsets on a 2D grid with property based ranking in one dimension and increasing structural redundancy in the other. The learnings from the 2015 screening deck were applied to the design of a next generation in 2019. We found that using traditional leadlikeness criteria (mainly MW, clogP) reduces the hit rates of attractive chemical starting points in subset screening. Consequently, the 2019 deck relies on solubility and permeability to select preferred compounds. The 2019 design also uses NIBR’s experimental assay data and inferred biological activity profiles in addition to structural diversity to define redundancy across the compound sets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01332"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01332">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01332&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01332&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01332&amp;href=/doi/10.1021/acs.jmedchem.0c01332" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14425-14447</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01312" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01357" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Evolution of Novartis’ Small Molecule Screening Deck Design</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ansgar Schuffenhauer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ansgar Schuffenhauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>A.S.: email, <a href="/cdn-cgi/l/email-protection#573639243036257924343f22313132393f36223225173938213625233e247934383a"><span class="__cf_email__" data-cfemail="a7c6c9d4c0c6d589d4c4cfd2c1c1c2c9cfc6d2c2d5e7c9c8d1c6d5d3ced489c4c8ca">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ansgar++Schuffenhauer">Ansgar Schuffenhauer</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6385-0414" title="Orcid link">http://orcid.org/0000-0001-6385-0414</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nadine Schneider</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nadine Schneider</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nadine++Schneider">Nadine Schneider</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5824-2764" title="Orcid link">http://orcid.org/0000-0001-5824-2764</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samuel Hintermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samuel Hintermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samuel++Hintermann">Samuel Hintermann</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-6803" title="Orcid link">http://orcid.org/0000-0001-6853-6803</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Douglas Auld</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Douglas Auld</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Douglas++Auld">Douglas Auld</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8164-9375" title="Orcid link">http://orcid.org/0000-0002-8164-9375</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jutta Blank</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jutta Blank</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jutta++Blank">Jutta Blank</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simona Cotesta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simona Cotesta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simona++Cotesta">Simona Cotesta</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Caroline Engeloch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline Engeloch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline++Engeloch">Caroline Engeloch</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath7">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nikolas Fechner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nikolas Fechner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nikolas++Fechner">Nikolas Fechner</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath8">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3852-3950" title="Orcid link">http://orcid.org/0000-0003-3852-3950</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christoph Gaul</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christoph Gaul</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christoph++Gaul">Christoph Gaul</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath9">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9162-553X" title="Orcid link">http://orcid.org/0000-0001-9162-553X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jerome Giovannoni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jerome Giovannoni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jerome++Giovannoni">Jerome Giovannoni</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath10">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johanna Jansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johanna Jansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research—Emeryville, 5300 Chiron Way, Emeryville, California 94608-2916, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johanna++Jansen">Johanna Jansen</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath11">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3937-6243" title="Orcid link">http://orcid.org/0000-0003-3937-6243</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Joslin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Joslin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Foundation, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Joslin">John Joslin</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath12">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philipp Krastel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philipp Krastel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philipp++Krastel">Philipp Krastel</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath13">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eugen Lounkine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eugen Lounkine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eugen++Lounkine">Eugen Lounkine</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath14">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Manchester</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Manchester</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Manchester">John Manchester</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath15">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lauren G. Monovich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lauren G. Monovich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lauren+G.++Monovich">Lauren G. Monovich</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath16">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8455-4171" title="Orcid link">http://orcid.org/0000-0001-8455-4171</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anna Paola Pelliccioli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anna Paola Pelliccioli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anna+Paola++Pelliccioli">Anna Paola Pelliccioli</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath17">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manuel Schwarze</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manuel Schwarze</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manuel++Schwarze">Manuel Schwarze</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath18">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael D. Shultz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael D. Shultz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+D.++Shultz">Michael D. Shultz</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath19">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7920-955X" title="Orcid link">http://orcid.org/0000-0001-7920-955X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nikolaus Stiefl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nikolaus Stiefl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nikolaus++Stiefl">Nikolaus Stiefl</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath20">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2562-7080" title="Orcid link">http://orcid.org/0000-0003-2562-7080</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Daniel K. Baeschlin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel K. Baeschlin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>D.B.: email, <a href="/cdn-cgi/l/email-protection#f89c9996919d94d69a999d8b9b90949196b896978e998a8c918bd69b9795"><span class="__cf_email__" data-cfemail="a8ccc9c6c1cdc486cac9cddbcbc0c4c1c6e8c6c7dec9dadcc1db86cbc7c5">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+K.++Baeschlin">Daniel K. Baeschlin</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath21">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2227-7452" title="Orcid link">http://orcid.org/0000-0002-2227-7452</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01332&amp;href=/doi/10.1021%2Facs.jmedchem.0c01332" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14425–14447</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 3, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 July 2020</li><li><span class="item_label"><b>Published</b> online</span>3 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01332" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01332</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D14425%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnsgar%2BSchuffenhauer%252C%2BNadine%2BSchneider%252C%2BSamuel%2BHintermann%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D23%26contentID%3Dacs.jmedchem.0c01332%26title%3DEvolution%2Bof%2BNovartis%25E2%2580%2599%2BSmall%2BMolecule%2BScreening%2BDeck%2BDesign%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14447%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01332"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4539</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01332" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Evolution of Novartis’ Small Molecule Screening Deck Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ansgar&quot;,&quot;last_name&quot;:&quot;Schuffenhauer&quot;},{&quot;first_name&quot;:&quot;Nadine&quot;,&quot;last_name&quot;:&quot;Schneider&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;Hintermann&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;Auld&quot;},{&quot;first_name&quot;:&quot;Jutta&quot;,&quot;last_name&quot;:&quot;Blank&quot;},{&quot;first_name&quot;:&quot;Simona&quot;,&quot;last_name&quot;:&quot;Cotesta&quot;},{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;Engeloch&quot;},{&quot;first_name&quot;:&quot;Nikolas&quot;,&quot;last_name&quot;:&quot;Fechner&quot;},{&quot;first_name&quot;:&quot;Christoph&quot;,&quot;last_name&quot;:&quot;Gaul&quot;},{&quot;first_name&quot;:&quot;Jerome&quot;,&quot;last_name&quot;:&quot;Giovannoni&quot;},{&quot;first_name&quot;:&quot;Johanna&quot;,&quot;last_name&quot;:&quot;Jansen&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Joslin&quot;},{&quot;first_name&quot;:&quot;Philipp&quot;,&quot;last_name&quot;:&quot;Krastel&quot;},{&quot;first_name&quot;:&quot;Eugen&quot;,&quot;last_name&quot;:&quot;Lounkine&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Manchester&quot;},{&quot;first_name&quot;:&quot;Lauren&quot;,&quot;last_name&quot;:&quot;G. Monovich&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Paola Pelliccioli&quot;},{&quot;first_name&quot;:&quot;Manuel&quot;,&quot;last_name&quot;:&quot;Schwarze&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;D. Shultz&quot;},{&quot;first_name&quot;:&quot;Nikolaus&quot;,&quot;last_name&quot;:&quot;Stiefl&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;K. Baeschlin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14425-14447&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01332&quot;},&quot;abstract&quot;:&quot;This article summarizes the evolution of the screening deck at the Novartis Institutes for BioMedical Research (NIBR). Historically, the screening deck was an assembly of all available compounds. In 2015, we designed a first deck to facilitate access to diverse subsets with optimized properties. We allocated the compounds as plated subsets on a 2D grid with property based ranking in one dimension and increasing structural redundancy in the other. The learnings from the 2015 screening deck were applied to the design of a next generation in 2019. We found that using traditional leadlikeness criteria (mainly MW, clogP) reduces the hit rates of attractive chemical starting points in subset screening. Consequently, the 2019 deck relies on solubility and permeability to select preferred compounds. The 2019 design also uses NIBR’s experimental assay data and inferred biological activity profiles in addition to structural diversity to define redundancy across the compound sets.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01332&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01332" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01332&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01332" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01332&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01332" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01332&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01332&amp;href=/doi/10.1021/acs.jmedchem.0c01332" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01332" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01332" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01332%26sid%3Dliteratum%253Aachs%26pmid%3D33140646%26genre%3Darticle%26aulast%3DSchuffenhauer%26date%3D2020%26atitle%3DEvolution%2Bof%2BNovartis%25E2%2580%2599%2BSmall%2BMolecule%2BScreening%2BDeck%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14425%26epage%3D14447%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0014.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">This article summarizes the evolution of the screening deck at the Novartis Institutes for BioMedical Research (NIBR). Historically, the screening deck was an assembly of all available compounds. In 2015, we designed a first deck to facilitate access to diverse subsets with optimized properties. We allocated the compounds as plated subsets on a 2D grid with property based ranking in one dimension and increasing structural redundancy in the other. The learnings from the 2015 screening deck were applied to the design of a next generation in 2019. We found that using traditional leadlikeness criteria (mainly MW, clogP) reduces the hit rates of attractive chemical starting points in subset screening. Consequently, the 2019 deck relies on solubility and permeability to select preferred compounds. The 2019 design also uses NIBR’s experimental assay data and inferred biological activity profiles in addition to structural diversity to define redundancy across the compound sets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63444" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63444" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Plate-based high throughput screening (HTS)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> is still a major source of small molecule hits in drug discovery,<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> despite the emergence of plateless encoded screening approaches, such as DNA-encoded libraries<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> and microfluidics-based methods,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> as well as the advancements in computational virtual screening methods.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Therefore, many pharmaceutical companies continue to invest in plate-based low molecular weight (LMW) screening decks and consider them key assets.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a></div><div class="NLM_p">Historically, high throughput screens aimed at the largest possible set of samples, irrespective of their quality, to identify all possible chemical matter and remove undesired hits thereafter. Industry-wide learning from property agnostic screens has found that screening of compound collections of poor sample quality (e.g., limited purity) and with poor compound properties (e.g., low aqueous solubility) leads to unsatisfactory results.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Consequently, most pharmaceutical companies rebuilt their compound management systems to enable quality control of compound collections and curate high-quality and targeted compound sets. This required “individualizing” compound samples from plate-based collections into tubes to allow for selecting single samples through automated high-throughput tube picking and reformatting hundreds of thousands of samples. At the Novartis Institutes of BioMedical Research (NIBR), we undertook these changes of our compound management capabilities from 2008 until 2012.</div><div class="NLM_p">These automation advances also enabled a move from classic, full deck HTS to alternative screening paradigms. Today, NIBR project teams commonly screen subsets of the total compound collection (more than 2 million unique compounds) in an iterative fashion. The “full screening deck” has been reduced to fewer than 1.5 million samples by removing low-quality samples or compounds with undesired chemical structures. There are multiple reasons for such alternative screening paradigms. In the past decade, phenotypic screening has again gained increasing importance in drug discovery, with the practical consequence that assays and screening cascades are becoming more complex, thereby limiting the number of compounds that can be screened.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Iterative screening can reduce the overall number of compounds screened, save compound inventories, shorten timelines and costs, and more importantly validate and/or optimize assay formats before committing to a larger scale screening campaign.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> More controversially, we, as well as others,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> realized that it is challenging to assess the potential of screening hit scaffolds for optimization toward successful clinical candidates. Consequently, one might argue that the success of hit to lead optimization is more dependent on the ingenuity of the drug discovery teams than the absolute number of hits. A large hit list does not automatically lead to better outcomes, so NIBR project teams may opt to work on early hits from a subset screen rather than invest in more resource- and time-intensive full deck screens. Nevertheless, full deck screening remains an important strategic option for a subset of NIBR projects, including targets (or pathways) with unusually low hit rates, highly competitive targets requiring exclusive chemical starting points, and parallel chemistry efforts on multiple scaffolds, as well as projects with previously truly validated assays and screening cascades.</div><div class="NLM_p">Consequently, it has become important to make accessible high value subsets of the compound library. In cases where target and pathway specific hypotheses are available, this is best done through high-throughput picking of customized compound subsets from the tube collection. NIBR’s tube picking and plating capacities allow teams to pick up to 100k compound samples for such screening strategies. However, despite a high level of automation, picking of compounds still involves a significant effort, prolonged timelines for compound access, and higher sample consumption compared to the use of preplated subsets, while offering little added benefit for screening large and frequently used, diversity driven sets (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information</a> for detailed discussion). For diversity driven subset screening of >50k compounds, we at NIBR therefore chose to rely on a plate-based screening deck which is produced every 3–4 years from tubes.</div><div class="NLM_p">In a classic HTS, where all compounds are tested, the distribution of compounds to screening plates within the deck has little impact on outcome. However, in iterative diversity-driven subset screening (as practiced at NIBR) the right distribution is key to achieve sensible results. As mentioned above, the hits of a first subset screen are normally used to confirm not only the robustness of the assay but also the biological relevance of the screening cascade and even the whole hit finding strategy before subsequent expansion to additional compounds of the screening collection. Therefore, it is essential to arrange the content of the deck in such a way that subsets (defined by diversity and molecular property criteria) can be easily accessed and built upon. Similar efforts have been reported recently by scientists at Merck (U.S.).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Their effort was mainly focused on creating small phenotypic screening subsets up to 50k compounds.</div><div class="NLM_p last">This Perspective describes how we designed and implemented a screening deck for diversity-driven subset screening at NIBR, initially in 2015 and a revised version in 2019.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Screening Deck Design Principles</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To define the design principles for the screening deck, we first assessed its future use. NIBR is a large drug discovery organization with a diverse project portfolio regarding the diseases tackled, the target classes addressed, and the molecular modalities employed. NIBR scientists employ compound screening not only for hit finding but also for pathway interrogation and target identification. The plate based screening deck described here is primarily targeted toward hit finding as most other screening activities require only smaller compound sets, which are easily generated on demand from the tube based collection.</div><div class="NLM_p">Hit finding screens can be either truly random or hypothesis-driven based on prior knowledge on target, pathway, or perturbants. Hypothesis-driven screens are best addressed by specifically selected compound sets enabled by advances in <i>in silico</i> methods for compound selections. These selections (mainly project-specific compound sets of hundreds to a few tens of thousands of compounds) are today readily picked from NIBR’s tube store and plated on demand.</div><div class="NLM_p">In addition to hypothesis-driven screening approaches, NIBR projects continue to use random screening to discover novel chemical matter interacting with unprecedented binding sites, targets, or pathways. Hit finding project teams often combine both approaches (hypothesis-driven and random) to overcome the limitations and biases (“search under the lamppost”) of virtual screening.<a onclick="showRef(event, 'ref8 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref8 ref20 ref21">(8,20,21)</a> Random screening with the aim to find unprecedented and likely very scarce chemical matter usually requires large numbers of compounds to be tested, widely known as the classic paradigm of HTS.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> To enable truly random screening for unprecedented chemical matter, we needed to design a deck as chemically diverse as possible and to include the dark chemical matter<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> of our compound collection. Such a deck would serve not only today’s known targets but also the unknown targets of tomorrow. This and the desire of scientists to advance their projects in an iterative way regarding the compound selection as well as the assay formats led us to the decision to focus the design of the screening deck toward <i>diversity-driven subset screening</i>.</div><div class="NLM_p">For an “ideal” diversity-driven screening deck, the two most important criteria are, first, it should contain compounds addressing all possible targets and mechanisms of action within a minimal subset and, second, the substance and sample properties should be of highest quality (i.e., no positive readouts of undesired nature, e.g., compound samples that induce protein precipitation). More than 20 years of HTS experience in industry have greatly increased the understanding of these two challenges but also made clear that they are inherently impossible to be completely overcome.<a onclick="showRef(event, 'ref2 ref24'); return false;" href="javascript:void(0);" class="ref ref2 ref24">(2,24)</a> Nevertheless, the design of our screening decks is based on these two prime characteristics: first, <i>biological diversity</i> as the ability to address as many targets as possible with as few compounds as necessary and, second, <i>optimal compound sample properties</i> to limit positive readouts of undesired nature to a minimum.</div><div class="NLM_p">Any implementation of such high-level objectives faces the conundrum that the screening deck design relies on the knowledge generated by historical screening. This has the practical consequence that there cannot be one perfect screening deck design valid for all times and that the composition should be adapted over time to the new data, changes in compound availability, new target classes and modalities, and learnings resulting from its use. Therefore, we decided to define a process in which we would redesign the screening deck every 3–4 years, enabling inclusion of the learnings over time. This process is outlined in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The compound-handling setup at NIBR was built according to these requirements and therefore allows redesigning and reproducing the screening deck in that time frame of 3–4 years from the DMSO stock stored tube collection (more details on this can be found in the next section).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Screening deck design at NIBR as an iterative process. The thick arrows indicate the main design cycle discussed in this manuscript, describing how choices are made during the design of plate-sets when the screening deck is due for renewal. At this time the available screening data are analyzed to define the high level design strategy, which will then be executed exploiting all available chemical matter and data. Besides the design strategy, the composition of the screening deck is influenced by the composition of the underlying compound bank. Effects influencing the compound bank are indicated by the thinner arrows. The learning from screening data analysis does not only influence the immediate plate design but also influence the continuous efforts of compound bank enhancement by library design and synthesis. Furthermore, adding a compound to a plate set screened preferentially may lead to faster depletion of that sample. And finally, medicinal chemistry projects generate compound samples that may be chosen for the screening deck, without otherwise being influenced by the screening deck design process described here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Compound-Handling Technology Enabling the Production of a Designed Screening Deck</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Before 2015, the setup used for maintaining the screening deck was largely as described by Schopfer et al.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This setup was designed in the late 1990s and implemented in 2000–2001 to enable large-scale screening of a complete screening deck for traditional HTS (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Compounds were solubilized and plated in 384-well source plates for primary screening in the sequence as they arrived in compound management which resulted in a screening deck of up to 1.2M samples. These screening deck plates served as source for all HTS and were renewed continuously from solid samples over a cycle time of ∼6 years. The automated picking function allowed for up to several thousand samples to be picked per week and was designed to enable hit validation of primary full deck HTS campaigns.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound management setup prior to 2015 designed for full deck HTS. Compounds are solubilized and plated at 2 mM for screening in order of arrival. This prevented removal of undesired samples and defining the distribution of compounds across screening plates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This initial HTS setup enabled effective and highly automated compound screening. However, on the basis of the learnings in hit and lead finding in the following years, it became evident that much higher sample picking would be required to enable a number of emerging concepts mentioned above. The fact that screening plates were produced continuously based on the sequence of incoming compounds had undesired consequences. In particular, it led to a situation in which many samples of high chemical similarity (e.g., from combinatorial libraries produced at a given time) were co-plated on the plates, leading to a low intraplate diversity of compounds. This setup supported only limited diversity selection by plate.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">Due to these shortcomings, the compound management groups at NIBR undertook a major effort from 2008 until 2012 to rebuild their compound logistics capabilities. This included two major aspects: (a) to produce 10 mM stock solution collections for all compound samples (>1.2M) from solid samples that are LC–MS quality controlled and (b) to install automation systems enabling the picking and processing from tubes to microtiter plates of up to 40k tubes in 24 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound management setup designed for diversity-driven subset screening since 2015. The HT-picking capability of the tube store allows replating the screening deck every 3–4 years in an organized fashion as well as providing large subsets for target or pathway driven selections.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">With the 10 mM stock collection and the automation setup described in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> in place, a first version of the NIBR screening deck was produced in 2015. In 2019, a second version was created, based on a further design iteration including the learnings and experiences from the 2015 deck.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Creation of a Screening Deck Diversity Grid</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As described above for the design of our screening deck, we mainly considered two objectives: The first objective is to maximize diversity in the screening deck subsets. Diversity in the bioactivity space is our primary aim. This space is described by the compound’s activity data in all medicinally relevant mechanisms of action, whether these are currently known or not. However, our structure–activity knowledge is incomplete, as we not only do not know the bioactivity of most compounds on the targets that are already characterized but lack the knowledge of entire dimensions of this space corresponding to mechanisms of actions not yet explored. Therefore, relying only on the diversity in the known bioactivity space would restrict us to search only “under the lamp post”. Therefore, we add diversity in the chemical structure space,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> assuming that novel chemical structures also have a higher potential for unprecedented biological activity.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Diversity is a collective property of a given subset and cannot be meaningfully assigned to an individual member compound. There is no ranking involved in diversity as coverage of each target is equally valuable in the context of a multipurpose discovery setting. In fact, for phenotypic screening the most relevant targets for modulation of the phenotype are typically not known, so imposing any ranking in the bioactivity space is not expected to be useful. There exists a wide range of descriptors and diversity metrics for chemical compounds, of which some are partially redundant.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> There is no straightforward way to combine those into one unified metric. The choice made is therefore to use several relevant metrics independently to define compound classes for each metric by clustering. Additional compound classes result from existing discrete compound annotations. Once the compounds are stratified into biological activity and chemical structure classes, the aim of the diversity selection process is to generate minimally sized subsets ensuring a preset minimal coverage for each class. The larger the coverage requirement per class will then translate into an overall higher subset size. Minimizing pairwise dissimilarity of the compounds in the selection is not the aim, as this is driven by the low ranges of similarity values not relevant for inferring biological activity.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">The second objective is the optimization of compound specific and mostly physicochemical properties, where a ranking is well-defined. For example, in the case of aqueous solubility, high solubility is preferred over low solubility given all other properties being equal. As there are a multiple of such properties to consider, they need to be combined into a multiobjective score. Such a score is an individual property of each compound, allowing comparing and ranking compounds on an individual basis.</div><div class="NLM_p">To design subsets with maximal diversity and optimal properties, we developed the following process: Given one has defined compound classes for stratification, and a multiobjective property-based ranking, then sampling the best ranking compound from each class results in a subset with optimal properties fulfilling the constraint that all classes need to be covered. This process can be repeated until finally all compounds have been selected, thereby keeping track of the selection round in which a compound was picked. As a result, each compound has two associated properties: the property rank and the selection round at which it was picked. These define a two-dimensional coordinate space, with the property rank as <i>X</i>-axis and the picking round as <i>Y</i>-axis. By appropriate binning strategies, this 2D-space can be partitioned into a 2D-grid of compounds plated together as a group of one or more plates, allowing the scientists to choose from this grid the plate groups for their assays. The sizes of the plate groups (grid cells) are defined by practicalities: they should be small enough such that the screening scientist can generate useful screening subsets and large enough such that the tube picking of each group for the plating remains efficient (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information</a> for a detailed explanation).</div><div class="NLM_p">By choice of the grid cells corresponding to the first of N selection rounds, a subset focused on maximal coverage can be obtained. By concentration on the grid cells with the top property ranks, a subset focused on good properties can be obtained. The whole concept is outlined in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. It is possible in this concept to support both subset selection strategies without any requirement for redundant plating. It is also worth noting at this stage that by working with preset and discrete classes as input, the process is agnostic to the exact classification method, and in fact, multiple classification schemes can be used jointly.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Diversity grid generation: (1) all compounds are annotated with property flags (positive or warning flags; see below for flag description); (2) all compounds are allocated to diversity classes; (3) compounds are selected and assigned to the grid cells by picking the most attractive compound from each class (compound A or B over H, G, J, or K in class 1) and further giving preference to compounds covering as many diversity classes as possible (compound A over B for class 1, since A also covers class 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The following sections describe in detail how the multiobjective property ranking is defined, how the compounds are stratified into diversity classes, how the picking algorithm works, and finally how the picking results are translated into plate maps for production.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Individual Properties for Ranking</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As previously discussed, the new screening deck was designed for diversity-driven hit finding with the main objective to identify <i>bona fide</i> binding partners or modulators of the target or pathway interrogated by the assay. These hits would serve as chemical starting points for the identification of tool compounds or medicinal chemistry programs toward a clinical candidate. Therefore, liabilities that negatively affect the chance of identifying a compound with a desired molecular interaction in the screening process need to be avoided. Liabilities that can be addressed during lead-optimization, such as activity on off-targets or high clearance, are generally acceptable at the screening stage but might be chosen to deprioritize a hit compound or hit class over another. Altogether the requirements for screening compounds are more in line with those for molecular probes, for which solubility and permeability have been pointed out as key properties,<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> than with those for drug candidates.</div><div class="NLM_p">One key property we use in annotating screening compounds is aqueous solubility, with low solubility leading to a warning flag and high solubility to a positive flag. There is empirical evidence that compounds can act as <i>bona fide</i> hits at a concentration above their reported aqueous solubility. This is counterintuitive, and we attribute this mainly to two factors. (1) The solubility under the distinct assay conditions can differ from the conditions used in standardized HT-solubility assays due to differences in buffer composition. (2) We normally transfer the compounds as DMSO solutions directly into the assay buffer rather than dry them down or perform an intermediate dilution in buffer. Therefore, under cellular assay conditions complex absorption/dissolution mechanisms apply as compounds transfer from DMSO to the nonhomogenous assay environment. Nevertheless, compounds of low solubility (<10 μM) are likely only available at subnominal concentration in most assays. This significantly decreases the chance of discovering them as hits.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In addition, low solubility compounds can negatively interfere with assay readouts if precipitating or aggregating at high concentrations.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> At NIBR, solubility is assessed primarily by high-throughput experimental solubility measurements under equilibrium conditions from dried-down stock solutions at pH 6.8.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Such measurements are available for >130 000 samples (7.8%) of our compound collection and can be used for compound annotation. Although solubility assay formats with direct addition of DMSO stock solution to buffer<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> reflect the HTS scenario more closely, it has to be noted here that the majority of the solubility measurements were taken during lead optimization, and the choice of the assay format as well as the compounds measured has been made accordingly. In case the experimentally determined solubility is not available, predictions from an internally trained machine-learning<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> model trained on the experimental data of the assay described above are used instead, provided they have been made with a sufficient level of confidence (see methods in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information</a>). Solubility below 10 μM is flagged as a warning (“solubility_flag”) and solubility of >100 μM is used as a positive flag.</div><div class="NLM_p">In addition to solubility, cell permeability is a key requirement to observe an intracellular effect in cellular assay systems. Given that cell permeability is not critical for other assay formats, we decided to flag high permeability as positive but not to flag low permeability as a warning. As in the case of solubility, the assessment is based on existing experimental data if available and reliable predictions from a machine-learning model as a second option. NIBR’s databases hold experimental data for >130 000 compounds from at least one of the following three assays: Caco-2,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> PAMPA,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and MDCK-LE.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> For prediction of permeability, a model based on data from the PAMPA assay was used (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information</a>), as this has the largest training set and as we primarily look for passive membrane diffusion.</div><div class="NLM_p">Frequent hitters are a problem that has gained great attention over the past decade.<a onclick="showRef(event, 'ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref43 ref44">(42−44)</a> Frequent hitters can be defined as compound samples, which show up repeatedly as “active” in many unrelated assays. This can be caused by interference with assay readouts, unspecific modification or sequestrations of proteins, or genuine polypharmacology.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> While true polypharmcoplogy, that is the unselective but specific binding to many proteins, is not a concern as such, it is hard to distinguish this from unspecific frequent hitter behavior. Compounds showing excessive polypharmacology, such as for example type-I kinase inhibitors, are still not very attractive starting points, and therefore, flagging such compounds as frequent hitters is considered acceptable, even more so as these compound classes are typically well-known in advance of an HTS. It is important to note that the frequent hitter character can be due to the compound itself but also due to impurities in the sample originating from synthesis procedures or compound instability. Repeatedly occurring examples of reactive impurities are reagents like acylating agents or metals.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_p">Here, we identified frequent hitters empirically from the screening history of a sample as described by Nissink et al. based on the number of assays it was screened in and the number of assays it hit without considering assay target and read-out methodology.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> Samples identified as frequent hitters in this way were assigned a warning flag (“frequent_hitter_flag”). It is worth noting here that frequent hitter behavior defined in this way can be partially related to low solubility. While 15% of the compounds predicted to have a low solubility were flagged as frequent hitters, this was only the case for 6% of the remaining compounds, indicating that avoiding low solubility compounds will also be beneficial from a frequent hitter point of view. This is in line with observations made by Guha et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p">Detecting frequent hitters in this empirical way requires that the sample has been exposed to a sufficient number of assays. Therefore, it makes sense to augment the empirical assessment by substructure filters known to indicate frequent hitter behavior. There is a wide range of publications describing such filters, some of them based on individual assay technology, such as α screen in case of PAINS.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> In a team of computational and medicinal chemists, a wide range of published<a onclick="showRef(event, 'ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59">(49−59)</a> and internally used filters were reviewed. We decided to exclude the PAINS filter family C<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> in this review since this set is based on very limited data (<15 compounds) and was developed for α screen assays only. Altogether 1270 unique substructure filters encoded in SMARTS<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> notation were reviewed, resulting in a final set of 444 filters that can be used for hit finding and triaging in NIBR. One major reason for the drastic reduction in the number of filters was their redundancy. Since the initial set was collected from many different sources (most of them were also collections of filters published by pharma companies and created for excluding compounds from screening campaigns), some duplicates or slight variations were introduced to these filters over time. The SMARTS representation is quite powerful to describe molecular patterns, but it is also very complex to interpret (compare the visualizations in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a) and especially complicated when comparing SMARTS against each other.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> Due to this complexity and the fact that the team wanted to include opinions from a broad community of medicinal chemists, several approaches like an online survey and guided discussions were used to define the final set. A similar approach was conducted by Lahue and colleagues at Merck to review substructure alerts.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The overall goal of this review was to decide if a filter should be retained for the final set and if so, which severity level it should have, high or low severity or just annotation. The high severity level led to exclusion of the compounds from the screening deck altogether (“substructure_OUT”), whereas the filters of low severity add a warning flag to the compound (“substructure_flag”). In addition, some filters were retained for annotation purposes only to guide project teams but without any deprioritization of the matched compounds. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b shows the distribution of the filters in the three different severity levels and the number of matches found for them for the molecules of the PubChem<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> version 2018 database. Since compounds can match more than one filter, it was necessary to calculate an overall severity score as the sum of all matching filters (high severity, 10; low severity, 1; annotation, 0). Compounds with at least one high severity flag were excluded from the screening deck. To avoid overpenalizing compounds, it was important to remove redundant filters in the reviewing process. All reviewed substructure filters originating from the literature and in-house are available on GitHub along with a Python script for matching and scoring a list of compounds (see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a> statement).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Exemplary filter (taken originally from ref <a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a>) to highlight the complexity of a SMARTS representation (top). The filter is visualized in a chemically more intuitive way using the SMARTSViewer<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (middle). Different matches of this filter to molecules found in the PubChem version 2018 database<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> are shown with the matching substructure highlighted (bottom). (b) Distribution of the different severity levels within the 444 filters (left). Number of matches of the different filters on the molecules in the PubChem database separated by severity level (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">These filters also encode substructures to recognize special molecule types like peptides, glycosides, isotopes, steroid, or nucleosides, which were excluded from the main screening deck. In addition to these SMARTS-based filters, we also have defined some additional rules to capture unfavorable structural characteristics: compounds that do not contain any oxygen or nitrogen atom were excluded; molecules that have a very low (<10%) or very high (>60%) fraction of nitrogen and oxygen atoms were flagged.</div><div class="NLM_p">Samples of very small structures with a molecular weight of <180 were penalized (“Low_MW_flag”), and samples without an assigned structure were excluded (“no_structure_out”). In both cases the motivation was that very little to no structure–activity information can be gained from these samples.</div><div class="NLM_p">Finally, sample quality was taken into account. This assessment was based on LC–MS quality control data of the DMSO stock solutions, which were available for most samples (89%). While samples with a purity below 50% were excluded (“QC_out”), samples with purities between 50% and 85% were assigned a warning flag (“QC_flag”). Samples with >85% purity, but with a mass different from the expected one, were assigned a warning flag as well.</div><div class="NLM_p">An overview of all the warning flags assigned to samples included in the 2019 deck is shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Exclusion reasons and warning flags assigned to the compound samples considered for the 2019 screening deck (multiple flags can apply to one sample).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A rule-based categorization was used to organize the compounds into three property categories based on the described properties as described in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The top category “high solubility & permeability” contains positively flagged compounds due to high solubility and high permeability. The second category “neutral properties” covers all compounds without negative flags. All remaining compounds with one or multiple warning flags were added to the “property warning flags” category. Within each category, a prioritization ranking was applied as defined in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Compound Ranking Criteria for 2019 Screening Deck<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">property category</th><th class="colsep0 rowsep0" align="center">content</th><th class="colsep0 rowsep0" align="center">ranking within category</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">High solubility and permeability</td><td class="colsep0 rowsep0" align="left">No warning flags</td><td class="colsep0 rowsep0" rowspan="2" align="left">No further ranking within category</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Two positive flags (good solubility AND good permeability)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">Neutral properties</td><td class="colsep0 rowsep0" align="left">No warning flags</td><td class="colsep0 rowsep0" align="left">(1) Compounds with either good permeability or good solubility and no low permeability</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">One or zero positive flags</td><td class="colsep0 rowsep0" align="left">(2) Compounds without low permeability</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">(3) Compounds with low permeability</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">Property warning flags</td><td class="colsep0 rowsep0" align="left">At least one warning flag</td><td class="colsep0 rowsep0" align="left">(1) Only one warning flag and high solubility</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Two, one, or zero positive flags</td><td class="colsep0 rowsep0" align="left">(2) Only one warning flag</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">(3) Two or more warning flags: ranked by number of warning flags</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">It is worth noting that the ranking defined in this table will produce many ties of equally ranked compounds. This is intentional.</p></div></div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Bioactivity and Chemical Structure Space Coverage</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">With the property ranking for the <i>X</i>-axis of the grid in place, we needed to define an approach toward diversity coverage for the picking rounds to generate the <i>Y</i>-axis. We used several classification methods that can be grouped into the following categories: individual biological target classes, biological compound profile space classes, and chemical space coverage classes.</div><div id="sec7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Individual Biological Target Classes</h3><div class="NLM_p last">Bioactivity data on individual biological targets were extracted from our in-house system “hithub”, which contains all in-house bioactivity data and is regularly updated with bioactivity data from major public data sources: ChEMBL,<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> Clarivate Integrity,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> GOSTAR.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Two compound classes are defined for each target containing all compounds interacting with this target irrespective of their chemical structure. The first one is a generous definition of activity requiring an AC<sub>50</sub> on-target of ≤10 μM, without any further conditions. The second class per target is based on a composite score including both potency and selectivity criteria, aiming at potent compounds with a high selectivity against other targets, combining evidence from multiple data sources.<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a> The first class will typically contain the second more stringently defined class. Using the more generous classification approach alongside the more stringent one allows identifying compounds for a target in case the second, more stringent rules yield an empty class. In addition, in cases where the stringently defined class contains only compounds with less attractive properties, it may still be possible to select soluble and permeable compounds from the more generously defined class.</div></div><div id="sec8" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Biological Compound Profile Space Classes</h3><div class="NLM_p">A complementary approach to grouping of compounds by individual target is the usage of biological profile data across multiple targets or assays. These can be either fully experimentally determined profiles such as HTS Fingerprints<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> or High-Content-Screening FPs.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> As such experimental profiles are not available for all compounds; inferred biological profile from predictive models are used in addition. In our case we used the inferred activity profiles from profile QSAR<a onclick="showRef(event, 'ref74 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref74 ref75 ref76">(74−76)</a> (pQSAR) which predicts AC<sub>50</sub> values in a quantitative way. In addition, binary activity prediction from a multicategory naive Bayesian model<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> was used. Prediction profiles were calculated on an individual target basis to obtain the Bayesian activity fingerprints (BAFP) as well as per protein family based on the protein–domain annotation in the pfam database<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> domain to obtain the Bayesian domain fingerprints (BDFP). In the latter model, each compound–target pair was expanded to all protein families annotated to the target. The underlying activity data come from the same database as used for the individual target activities. As a fourth set of activity profiles, docking scores derived from docking the compounds into the pocketome<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> were used.</div><div class="NLM_p last">In order to obtain discrete classes from these activity profiles, they need to be clustered. This was done using the Markov Cluster Algorithm (MCL).<a onclick="showRef(event, 'ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref81 ref82">(81,82)</a> MCL is a graph clustering algorithm. In the similarity graph for each bioactivity profile type an edge was created between two compounds if the Pearson correlation between two profiles exceeded the cutoff value chosen for each profile type individually to avoid chance correlations (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Table 1 in Supporting Information</a> for the values chosen). Each activity profile data set was clustered on its own; thus no explicit weighting was required. However, the more clusters were generated per profile type, the more compounds are required to be selected per selection round and thus the more implicit weight each profile type gets. In order to balance the weights across the profile types, the same number of clusters of ∼30–50k (to fit the limit of 50k in the first iteration) per profile class was targeted, and the “inflation” parameter of the MCL clustering controlling the granularity was chosen accordingly.</div></div><div id="sec9" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemical Space Coverage Classes</h3><div class="NLM_p">The coverage of the bioactivity space is the dominant objective. However, there are only assay data for a limited set of biological targets, and novel targets may require different chemotypes. At NIBR, a rule-based scaffold classification has been developed called Scaffold Tree<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> which was subsequently extended to the Scaffold Network.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> This network was used for the definition of classes purely based on chemical structure. Each (sub)scaffold in the scaffold network forms a class consisting of the compounds containing it as substructure. As larger scaffolds tend to be less informative since they produce too many small classes, only scaffolds up to three rings (not isolated ring systems) according to the definition used in Scaffold Network were included. Singleton scaffolds were ignored.</div><div class="NLM_p">Scaffold classification does not work well for structures that are devoid of any cyclic core. Therefore, two classifications based on the extended connectivity fingerprints with a diameter of four (ECFP_4) were added.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> A similarity graph was computed and clustered with the MCL algorithm. In addition to the MCL clustering, chemical topic modeling<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> was also used. Chemical topic modeling has been adopted from the text-mining field and is able to organize large sets of molecules into specific “topics”, e.g., a peptide topic or steroids or kinase scaffolds. We have seen that the method works well for identifying chemical series in a large set of compounds and that it is also able to find series where the scaffold is slightly modified or disconnected.</div><div class="NLM_p last">Scaffolds do not convey direct information on 3D shape, and therefore shape classification models were applied. In the 2015 version of the screening deck a shape classification based on the principle moments of inertia of a single, low energy conformation was used following the concept by Sauer and Schwarz.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> The shape triangle was partitioned into equidistant grid cells which served as shape classes. Such a classification is taking neither pharmacophore features beyond shape nor conformational flexibility into account. We therefore applied a more attractive approach in the 2019 version. The FastROCS software<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> in combination with highly parallelized computation made it possible to calculate a ROCS 3D combo score<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> similarity graph, which then could be clustered with MCL in the same way as the biological profile similarity matrices described above. This shape representation does take into account both conformational flexibility and pharmacophore features.</div></div><div id="sec10" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Manually Curated Classifications</h3><div class="NLM_p">All of the above classifications were generated computationally and may not always reflect the chemist’s contextual knowledge. Therefore, additional classifications assigned by chemists were used when available. These came from three distinct sources: The first one is the assignment of project codes according to a controlled vocabulary upon compound registration. This may not always describe a homogeneous chemical series but does reflect an anticipated, although not necessarily observed bioactivity. The second one is an in-house database called Compound Series and Favorites (CS&F).<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> This tool implements the concept of medicinal chemistry story-telling as described by Stahl and Baier.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> A key feature of this tool is to enable medicinal chemists to define chemical series and assign individual landmark compounds and one or more scaffold substructure queries to them. For each series, progress in optimization toward drug candidate is annotated, e.g., through the assignment of tags taken from a large set of standardized vocabulary as well as relationship definitions between compounds or scaffolds. Another key feature is that the medicinal chemists are asked to tag the starting point of a chemical series, which is the screening hit in case the series results from screening activities (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). In order to derive classes for diversity selection, a membership list is created for each CS&F series in which the individually assigned compounds and the compounds matching one of the series substructure queries are combined. As a third source, compounds from libraries synthesized by combinatorial or parallel approaches are assigned to a specific library code for each library. For each library code there is a class consisting of all compounds annotated with it.<named-content content-type="anchor" rid="tbl2" type="simple"></named-content></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Web UI of the Compound Series & Favorites Tool. Each series has a shorthand panel showing the last five elements added. The series can be expanded into the full network view visualizing the relationship between the individual elements and a status flag indicating the series status. In addition to individual compounds, a series can contain also scaffold substructure queries. The example in this case has been taken from Furet et al.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Series members can be marked with tags such as “starting point” indicating the compound to be the chemical seed of a series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Compound Classes Defined as Coverage Objectives</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" colspan="2" align="center">description</th><th class="colsep0 rowsep0" align="center">used in 2015 version</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Individual biological targets (experimental)</td><td class="colsep0 rowsep0" align="left">TARGET_AC50</td><td class="colsep0 rowsep0" align="left">Compounds with AC<sub>50</sub> ≤ 10 μM on given target</td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TARGET_SCORE</td><td class="colsep0 rowsep0" align="left">Compounds with a target score of ≥4 on a given target<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a></td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Biological compound profile space (experimental)</td><td class="colsep0 rowsep0" align="left">HTSFP</td><td class="colsep0 rowsep0" align="left">HTS fingerprint similarity. <i>p</i>-value derived from Pearson correlation<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a></td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCSFP</td><td class="colsep0 rowsep0" align="left">Feature vectors from high content cellular imaging assays<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">Biological compound profile space (inferred)</td><td class="colsep0 rowsep0" align="left">PQSAR</td><td class="colsep0 rowsep0" align="left">Profile QSAR including all models on all NIBR concentration response assays where sufficient model reliability was achieved<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BAFP</td><td class="colsep0 rowsep0" align="left">Bayesian affinity fingerprints for individual targets<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BDFP</td><td class="colsep0 rowsep0" align="left">Bayesian affinity fingerprints for targets aggregated by pfam domains annotated to target<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOCKING_FP</td><td class="colsep0 rowsep0" align="left">Docking score profiles generated by high-throughput docking against pocketome</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">Chemical space coverage</td><td class="colsep0 rowsep0" align="left">ECFP_4</td><td class="colsep0 rowsep0" align="left">Extended connectivity 2D chemical structure similarity<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CHEM_TOPIC</td><td class="colsep0 rowsep0" align="left">Class assignment generated by chemical topic modeling<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FAST_ROCS</td><td class="colsep0 rowsep0" align="left">FAST-ROCS 3D shape and pharmacophore similarity<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a></td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SCAFFOLDS</td><td class="colsep0 rowsep0" align="left">(Sub)Scaffolds up to 3 rings as assigned by scaffold network<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">Manually curated classifications</td><td class="colsep0 rowsep0" align="left">PROJECT_CODES</td><td class="colsep0 rowsep0" align="left">Assignment to project codes by medicinal chemist</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SERIES</td><td class="colsep0 rowsep0" align="left">Manually curated assignment to chemical series</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LIBNAME</td><td class="colsep0 rowsep0" align="left">Assignment to combinatorial hit discovery library</td><td class="colsep0 rowsep0" align="left">yes</td></tr></tbody></table></div></div></div></div><div id="sec11" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Iterative Compound Selection Process</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As mentioned above, the objective is now to iterate over the classes defined as coverage objectives, pick the best ranking compound sample from each class, and repeat this cycle multiple times. At this stage, the algorithm is agnostic to the source of the classification and class is simply a group of samples. The classes defined are not mutually exclusive in membership, and they may be highly redundant. Such redundancy needs to be handled appropriately, as otherwise highly overlapping classes from different classification schemes would lead to a redundant selection of compounds. Thus, whenever a compound is selected because of its membership to one class, all other classes this compound belongs to are considered as covered as well. This may lead to a higher number of compounds selected per class than targeted at in the selection round due to “collateral coverage”. In order to compensate for that, selection from the affected class is stopped in subsequent rounds until its count is back on target.</div><div class="NLM_p">Another point worth considering is the size of the classes. In any clustering, high numbers of singletons and small clusters are expected due to the typical power law distribution of chemical cluster sizes.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> This would lead to a high number of compounds being picked in the first iterations to cover all clusters. As a consequence, the number of compounds in the first row of the diversity grid would exceed a size practical for screening purposes (we targeted a practical size of around 50k compounds for the first iteration and 50k to 100k for the second iteration, see below). As a compromise, the sampling in the first iterations can be restricted to clusters above a certain minimal size. This is a reasonable choice, as representatives of larger clusters offer more opportunities to follow up the hits of an initial small-scale screen by similarity expansion in an iterative screening process.<a onclick="showRef(event, 'ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref95 ref96">(95,96)</a></div><div class="NLM_p">Once all compounds have been ranked by property and allocated to coverage classes of different types and the minimal cluster size for each iteration has been defined, the selection algorithm can be run to generate the diversity grid. The flowchart of the algorithm is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. A step by step toy example can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information (Figure S1)</a>, and the source code is provided as well.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Flowchart of the selection algorithm. The fraction of compounds at which picking is to be stopped is a parameter of the algorithm, which can be set according to practical purposes. As a fine-grained subset structure for the second half of the full deck is typically not needed, a value of 0.5 is sufficient here. When several compounds of a class tie for the top rank, the compound annotated with the most class types gets precedence. If that does not break the tie, a random number assigned to each compound is used to break the tie.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In principle, the selection algorithm would allow allocating the compounds to plates in a very granular way, as at each selection round compounds could be allocated to plates in the exact order of the sequence they were picked. However, such a fine-grained plating would be technically challenging and yield no benefit for screening. Experience at NIBR shows that, in practice, project teams prefer to screen plate subsets of stepwise increasing size. The most often used subset sizes are ∼50k, ∼125k, 250k, 500k, and finally the full deck (∼1.5M). On the basis of this, several picking rounds are grouped together in one plate group to achieve the targeted sizes.</div></div><div id="sec12" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Results: Screening Deck Grids 2015 and 2019</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As mentioned, the first screening deck grid for diversity driven subset screening was produced in 2015 and a modified version was designed in 2019 incorporating the learnings from four years of screening practices at NIBR. The resulting sizes of the plate subsets are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Subset Sizes of the 2015 and 2019 Screening Decks<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="5" align="center">2015 Screening Deck</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. of diversity selection rounds</th><th class="colsep0 rowsep0" align="center">A</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">C</th><th class="colsep0 rowsep0" align="center">total per row (summation)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 1 (01)</td><td class="colsep0 rowsep0" align="left">18k</td><td class="colsep0 rowsep0" align="left">8k</td><td class="colsep0 rowsep0" align="left">18k</td><td class="colsep0 rowsep0" align="left">45k (45k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 2 (02–04)</td><td class="colsep0 rowsep0" align="left">14k</td><td class="colsep0 rowsep0" align="left">11k</td><td class="colsep0 rowsep0" align="left">45k</td><td class="colsep0 rowsep0" align="left">70k (115k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 3 (05–10)</td><td class="colsep0 rowsep0" align="left">10k</td><td class="colsep0 rowsep0" align="left">14k</td><td class="colsep0 rowsep0" align="left">100k</td><td class="colsep0 rowsep0" align="left">124k (239k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 4 (11–35)</td><td class="colsep0 rowsep0" align="left">15k</td><td class="colsep0 rowsep0" align="left">21k</td><td class="colsep0 rowsep0" align="left">208k</td><td class="colsep0 rowsep0" align="left">245k (484k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 5 (>35)</td><td class="colsep0 rowsep0" align="left">71k</td><td class="colsep0 rowsep0" align="left">76k</td><td class="colsep0 rowsep0" align="left">474k</td><td class="colsep0 rowsep0" align="left">621k (1.11M)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total per column</td><td class="colsep0 rowsep0" align="left">129k</td><td class="colsep0 rowsep0" align="left">131k</td><td class="colsep0 rowsep0" align="left">825k</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="6" align="center">2019 Screening Deck</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. of diversity selection rounds</th><th class="colsep0 rowsep0" align="center">high solubility and permeability</th><th class="colsep0 rowsep0" align="center">neutral properties</th><th class="colsep0 rowsep0" align="center">property warning flags</th><th class="colsep0 rowsep0" align="center">natural products</th><th class="colsep0 rowsep0" align="center">total per row (summation)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 1 (01)</td><td class="colsep0 rowsep0" align="left">39k</td><td class="colsep0 rowsep0" align="left">8k</td><td class="colsep0 rowsep0" align="left">4.2k</td><td class="colsep0 rowsep0" align="left">1.4k</td><td class="colsep0 rowsep0" align="left">53k (53k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 2 (02–03)</td><td class="colsep0 rowsep0" align="left">55k</td><td class="colsep0 rowsep0" align="left">20k</td><td class="colsep0 rowsep0" align="left">8k</td><td class="colsep0 rowsep0" align="left">1.4k</td><td class="colsep0 rowsep0" align="left">84k (137k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 3 (04–06)</td><td class="colsep0 rowsep0" align="left">69k</td><td class="colsep0 rowsep0" align="left">32k</td><td class="colsep0 rowsep0" align="left">23k</td><td class="colsep0 rowsep0" align="left">2.9k</td><td class="colsep0 rowsep0" align="left">127k (264k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 4 (07–15)</td><td class="colsep0 rowsep0" align="left">128k</td><td class="colsep0 rowsep0" align="left">83k</td><td class="colsep0 rowsep0" align="left">49k</td><td class="colsep0 rowsep0" align="left">4.2k</td><td class="colsep0 rowsep0" align="left">264k (528k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 5 (>16)</td><td class="colsep0 rowsep0" align="left">111k</td><td class="colsep0 rowsep0" align="left">350k</td><td class="colsep0 rowsep0" align="left">438k</td><td class="colsep0 rowsep0" align="left">4.2k</td><td class="colsep0 rowsep0" align="left">903k (1.43M)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total per column</td><td class="colsep0 rowsep0" align="left">402k</td><td class="colsep0 rowsep0" align="left">493k</td><td class="colsep0 rowsep0" align="left">522k</td><td class="colsep0 rowsep0" align="left">14k</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">This tables show the screening deck grid. Each row–column combination corresponds to a plate group consisting of one or more screening plates, whose size is given in the table. The row numbers correspond to the iterations that have been made through the compound collection, which are given in parentheses after the row number. For example, in 2019 row 2 contains the compounds that were selected in the 2nd and 3rd pass through all the compound classes of the compound collection. Thus, if rows 1 and 2 are selected for screening, for each class there are at least 3 members present in the subset of altogether 137k compounds contained in these two rows.</p></div></div></div><div class="NLM_p">The 2015 version differed from the above-described approach used for the 2019 version. Instead of a coarse-grained classification focusing on the three categories property-based, unbiased, and property warning flags, a more fine-grained property ranking was used in 2015. Multiple objectives were combined by Pareto ranking,<a onclick="showRef(event, 'ref97 ref98'); return false;" href="javascript:void(0);" class="ref ref97 ref98">(97,98)</a> meaning that in order to achieve a better ranking than another compound, a compound must be superior in at least one objective and not be inferior in any other objective. Molecular weight and clogP were explicitly used in the ranking, with the objective to be as close as possible to, or ideally within, the ranges of MW 150 to 350 and clogP 1 to 3, which were at that time considered to be ideal for hit molecules. Additional objectives used as negative flags were undesirable substructures, poor solubility, and poor permeability; the last two properties were estimated using a combination of experimental data with predictive models as in 2019. It is important to note that in 2015 solubility and permeability were used only to penalize lowly solubility structures but not to give priority to structures with good solubility and permeability over neutral or inconclusively predicted ones. In addition, a frequent hitter flag similar to the one used in 2019 was already in place and was incorporated into the ranking. As molecular weight and clogP are continuous variables rather than discrete flags, they dominated the overall ranking. In order to receive an optimal Pareto rank, both molecular weight and clogP had to be in the ideal range. One key feature of Pareto ranking is also that there is no compensation between the parameters assessed. A tiny increase in molecular weight has the same detrimental effect as the presence of a solubility or frequent hitter flag.</div><div class="NLM_p">There were 129k compounds with an optimal Pareto rank, and these were considered as column A. As discussed above, 250k was considered a relevant screening subset size, and consequently the boundaries for the next tier (B column) in the property based ranking were set in a way that forced a size of 250k for the combined A and B columns. All remaining compounds were assigned to the C column, without implying that all of these compounds were problematic. This differs from the 2019 approach, where on the property axis no preset sizes were enforced, but rather the criteria for property categories were rationally set, and the number of compounds for each category was accepted as a consequence of these criteria.</div><div class="NLM_p">The property profiles of the subsets of the 2015 and 2019 decks are shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> (molecular weight) and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> (clogP). The 2015 screening deck grid showed a strong bias for MW < 350 Da and a clogP range of 1–3 in categories A and B, making these compounds almost fragment-like. As a consequence, hits from these categories were mostly attractive chemical starting points due to their low MW and lack of undesired substructures. However, we also observed much reduced hit rates for the categories A and B over category C for the 2015 screening deck, causing concern that the restricted MW and clogP ranges might have led to an imbalance of compound diversity across the deck.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Molecular weight profile of the 2015 and 2019 deck versions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. clogP profiles of the subsets of the 2015 and 2019 deck versions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In both versions of the screening deck the first column consists of highly soluble compounds, either exclusively as in the 2019 version or to a vast majority as in the 2015 version. Detailed distributions of the compounds according to the 2019 solubility and permeability categorization can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">Supporting Information (Figures S3 and S4)</a>. The difference between the 2015 and 2019 deck is not so much in the fraction of compounds classified as highly soluble in the left column (“A” or “highly soluble & permeable”) but in the total number of compounds that could be assigned to this column. In the 2019 “highly soluble & permeable” there are more compounds than in the 2015 A column, as high solubility and permeability can also be achieved outside the MW and clogP ranges used for the 2015 A column. The overlap between the 2015 and 2019 property column categories is shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Overlap of Property Categories in the 2015 and 2019 Deck<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">2019 deck column category</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">high solubility and permeability</th><th class="colsep0 rowsep0" align="center">neutral properties</th><th class="colsep0 rowsep0" align="center">property warning flags</th><th class="colsep0 rowsep0" align="center">excluded</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">2015 deck column category</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">87404</td><td class="colsep0 rowsep0" align="left">25842</td><td class="colsep0 rowsep0" align="left">6693</td><td class="colsep0 rowsep0" align="left">819</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">53678</td><td class="colsep0 rowsep0" align="left">34722</td><td class="colsep0 rowsep0" align="left">31190</td><td class="colsep0 rowsep0" align="left">3341</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">152680</td><td class="colsep0 rowsep0" align="left">277953</td><td class="colsep0 rowsep0" align="left">326385</td><td class="colsep0 rowsep0" align="left">25663</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">not part of 2015 deck</td><td class="colsep0 rowsep0" align="left">109638</td><td class="colsep0 rowsep0" align="left">160025</td><td class="colsep0 rowsep0" align="left">164256</td><td class="colsep0 rowsep0" align="left">60231</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Compounds without available sample at the design of the 2019 deck were not included in this table, as they have not been rated according to the 2019 screening deck property criteria.</p></div></div></div><div class="NLM_p">Primary hit rates are not a good quality measure for screening success,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> as they often are more influenced by the variable assay robustness against confounding effects than the presence of true hits in the compound set screened. We therefore decided to analyze the distribution of the chemical starting points of medicinal chemistry optimization projects across the grid. We used the CS&F system described above to identify these chemical starting points. We included in the initial analysis the starting points of chemical series that at the time of analysis were either actively pursued or successfully transformed into a development candidate. Ideally such an analysis should be limited only to starting points originating from the screening deck. While CS&F provides the possibility to annotate the sources of starting points by a controlled vocabulary, the level of annotation was not sufficiently complete to take this into account. In the rare case that a chemical series had more than one starting point, and these starting points were from different cells in the grid, only the starting point in the most upper-left direction of the grid was counted. As described above, we use CS&F series information in the selection process for the 2019 grid as coverage objective. This distorts any readout of the allocation of starting points as it forces the coverage of each chemical series in the top row of the grid. Therefore, for the purpose of this evaluation the selection algorithm was rerun excluding CS&F series as a classification method. The resulting distributions can be seen in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Even in case of a random distribution of starting points, grid cells with more compounds are expected to contain more starting points. Therefore, the relative enrichment of starting points was calculated, which is the observed fraction of starting points divided by the fraction of total screening allocated to each grid cell.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Distribution of Chemical Series Starting Points across the Plate Sets<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="9" align="center">2015 Screening Deck</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">A</th><th class="rowsep1 colsep0" colspan="2" align="center">B</th><th class="rowsep1 colsep0" colspan="2" align="center">C</th><th class="rowsep1 colsep0" colspan="2" align="center">per row</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. of diversity selection rounds</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 1 (01)</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">12.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 2 (02–04)</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 3 (05–10)</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">12.9</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">18.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 4 (11–35)</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">22.9</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">24.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 5 (>35)</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">31.4</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">39.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">per column</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">9.9</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">10.7</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">79.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="9" align="center">2019 Screening Deck</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">high solubility and permeability</th><th class="rowsep1 colsep0" colspan="2" align="center">neutral properties</th><th class="rowsep1 colsep0" colspan="2" align="center">property warning flags</th><th class="rowsep1 colsep0" colspan="2" align="center">per row</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. of diversity selection rounds</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th><th class="colsep0 rowsep0" align="center">enrich</th><th class="colsep0 rowsep0" align="center">% sp</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 1 (01)</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">7.3</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">9.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 2 (02–03)</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">10.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 3 (04–06)</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">6.1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">9.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 4 (07–15)</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">6.5</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">13.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 5 (>16)</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">25.6</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">28.0</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">57.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">per column</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">28.9</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">37.0</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">34.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The % sp columns describe the percentage of the total number of series starting points with respect to the total number of series starting points included in the deck, which is 140 for 2015 and 246 for 2019. Enrichment value are not additive.</p></div></div></div><div class="NLM_p">In an ideal case, the relative enrichment should be high in the top left corner and low in the bottom right corner as the recommendation is to preferentially screen the top rows. Somewhat surprisingly, this was not the case. In particular, columns A and B show a negative enrichment of chemical starting points vs column C in alignment with the reduced hit rates observed for our screens. We recognize that 140 chemical starting points are a limited set to draw any conclusions from, but one rationalization could be that our portfolio has evolved to include targeting protein–protein interactions (PPIs) for which larger and greasier molecules are required. On the basis of these observations, we decided to change the ranking criteria for the 2019 version of the grid: high solubility and high permeability were introduced as positive selection criteria for column A (newly termed high solubility and permeability), and MW and clogP were no longer considered directly. However, in order to achieve good permeability and solubility at the same time, lower MW and clogP still appear to be advantageous; the MW and clogP distributions for the first high solubility and permeability column in the 2019 deck are shifted to lower values compared to the other two columns as can be seen in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>. More importantly, the new design also appears to positively influence the enrichment of the screening deck grid 2019 with chemical starting points in the first two columns and the first three rows.</div><div class="NLM_p">With hindsight, one could ask the question why the decision was made to rely so heavily on molecular weight and log <i>P</i> in 2015 since the value of using these properties to predict drug discovery success was already questioned at that time.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> When we designed the 2015 deck, machine learning models such as those used to predict solubility and permeability were not yet commonly accepted by the chemistry community. This was partially due to the fact that in line with observation at other pharma companies,<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> our models had not yet reached the prediction performance they had in 2019 due to the smaller amount of training data but also because there was still a great level of uncertainty in the medicinal chemistry community about the appropriate usage of such models. Therefore, there was a wish to rely on simpler, easier to interpret rules based on molecular weight and clogP.<a onclick="showRef(event, 'ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref101 ref102">(101,102)</a> The introduction of a rigorous and transparent internal review process for global machine learning models and clear communication about scope and limitations of each model helped to increase the trust of the medicinal chemistry community in such models and resulted in a wide acceptance of these models for the 2019 version. Another motivation for focusing on low molecular weight compounds in 2015 was that due to the combinatorial explosion of the chemistry space with increasing molecular weight,<a onclick="showRef(event, 'ref103 ref104'); return false;" href="javascript:void(0);" class="ref ref103 ref104">(103,104)</a> the sampling of the space for larger molecules would be less complete, and therefore it was believed that the chances to find molecules optimally fitting a binding site in this enlarged “haystack” would be lower.<a onclick="showRef(event, 'ref101 ref105'); return false;" href="javascript:void(0);" class="ref ref101 ref105">(101,105)</a> It was originally anticipated that the A column of the deck could be screened at elevated concentrations in order to also detect the weaker binders expected from this set.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Especially for cellular screens, the higher ionic strength resulting from higher concentrations of charged compounds limited this approach in reality.</div><div class="NLM_p">Next, we analyzed the frequent hitter distribution (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). As expected, the percentage of frequent hitters increases from left to right as they were penalized in both versions of the deck. No frequent hitters were present in the first column of both decks. In the B column in the 2015 deck the number of frequent hitters is less well controlled compared to the “neutral properties” column of the 2019 deck. In the 2015 deck the ranking of the compounds was dominated by molecular weight and clogP, and therefore the existing penalty for frequent hitters did not come into full effect. In both decks, frequent hitters were accepted in the C and “property warning flags” sections, respectively. Project teams need to consider this and take special care when hits from these columns are advanced into hit validation. When comparing the upper two rows of the property warning flags column in the 2019 deck with the upper two rows of the C column in the 2015 deck, the 2019 deck has less frequent hitters present there. This might be a consequence of stronger deprioritization of frequent hitters in the property ranking in 2019.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Frequent hitter analysis as density plots for hit counts versus total screen count for 2015 and 2019 screening deck per plate subset. In this plot frequent hitters<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> are located in the upper right area, whereas dark chemical matter<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> is located to the right side at the bottom, whereas compounds on the left side have not yet been exposed to a sufficient number of assays to make any conclusions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For the 2019 version of the grid a wider range of computed biological profiles was included, increasing the number of classes to be covered. To keep the size of the first and second row within the size constraints for lower throughput, especially phenotypic assays (as described above), minimal sizes of the classes to be sampled were enforced at 15 and 5 in the initial two picking rounds, respectively. This limits the class coverage in the initial rounds but retains the representation of larger classes. The manually curated classes and the individual biological target classes as defined in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> were exempt from these limitations, as these were deemed highly relevant classifications. As a further exception for the scaffold classification, scaffolds with only one representative were ignored, as these can occur when rings were present as substituents to an otherwise acyclic backbone (for example, oligopeptides with tyrosine or histidine side chains). As the selection proceeded, the classes each selected compound belonged to were tracked. The distribution of the class types during picking can be seen in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> for the 2019 grid. The chemical scaffolds were the individual class type that contributed most to the selections at all rounds, as the chemical structure is always known, whereas the fraction of compounds that can be linked to biological activity by either experimental data or predictions is limited to the current state of bioactivity knowledge.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Influence of the class types on the selection process. For each grid cell the number of classes that triggered the selection of compounds in this cell is shown. As a compound may be selected simultaneously by multiple classes, these numbers do not add up to the total number of compounds per grid cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Besides the synthetic compound collection, NIBR also owns a substantial natural product collection. In the 2015 deck natural products have been plated without any structure as one plate group. However, like the synthetic compound collection, the natural product collection is too large to always be screened entirely. Therefore, in 2019, the same selection process was applied, with some modifications, such as the relaxation of a few substructure filters, which are not deemed meaningful for natural products. The much smaller size of the collection made it possible for our natural product experts to review the grouping created by the automated diversity picking and curate it manually. As the collection was overall smaller in size, it was decided to group all compounds together in one row, without distinguishing between the property columns. The resulting plate groups form the column “natural products”. As mentioned, the first screening deck grid for diversity driven subset screening was produced in 2015 and a modified version was designed in 2019 incorporating the learnings from four years of screening practices at NIBR. The resulting sizes of the plate subsets are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Also here the top row contained less problematic structures than the bottom rows.</div><div class="NLM_p">NIBR provides a Facilitated Access to Screening Technologies (FAST-lab) program, by which academic partners can access NIBR screening systems and a subset of our screening deck within the framework of a collaboration while bringing their assays and tools.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> This screening subset consists of 200 000 nonexclusive diverse screening compounds and was acquired by NIBR over the past 20 years from multiple nonexclusive, commercial compound collections. This compound set is called Screening Set for External Collaboration (SSEC). When replating the full deck, we took the opportunity to structure this collection according to the same principle. The one exception was that we did not make use of the manually curated classes that rely on partially proprietary NIBR information. We applied a more stringent minimum class size limit for the first picking round, as in many FAST-lab collaborations the number of compounds that can be screened is quite limited, and we aimed therefore for a smaller first picking iteration. The results are shown in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Diversity Picking of the SSEC Set</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. of diversity selection rounds</th><th class="colsep0 rowsep0" align="center">C1: high solubility and permeability</th><th class="colsep0 rowsep0" align="center">C2: neutral properties</th><th class="colsep0 rowsep0" align="center">C3: property warning flags</th><th class="colsep0 rowsep0" align="center">total per row (summation)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 1 (01)</td><td class="colsep0 rowsep0" align="left">5.4k</td><td class="colsep0 rowsep0" align="left">0.8k</td><td class="colsep0 rowsep0" align="left">0.3k</td><td class="colsep0 rowsep0" align="left">6.6k (6.6k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 2 (02–03)</td><td class="colsep0 rowsep0" align="left">12.0k</td><td class="colsep0 rowsep0" align="left">4.1k</td><td class="colsep0 rowsep0" align="left">1.4k</td><td class="colsep0 rowsep0" align="left">17.5k (24.1k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 3 (04–06)</td><td class="colsep0 rowsep0" align="left">15.0k</td><td class="colsep0 rowsep0" align="left">8.4k</td><td class="colsep0 rowsep0" align="left">3.8k</td><td class="colsep0 rowsep0" align="left">27.1k (51.2k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">row 4 (>7)</td><td class="colsep0 rowsep0" align="left">23.4k</td><td class="colsep0 rowsep0" align="left">73.3k</td><td class="colsep0 rowsep0" align="left">56.0k</td><td class="colsep0 rowsep0" align="left">151.7k (203k)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total per column</td><td class="colsep0 rowsep0" align="left">55.8k</td><td class="colsep0 rowsep0" align="left">85.6k</td><td class="colsep0 rowsep0" align="left">61.4k</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div></div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Molecular weight and log <i>P</i> grid for the SSEC collection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec13" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67886" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67886" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The 2019 screening deck grid is currently NIBR’s primary source for plate-based diversity-driven subset screening. It is used in 50–100 subset screens with greater than 50k compounds yearly across the NIBR sites in a broad set of biochemical and cellular assays. The two-dimensional diversity grid is based on the key characteristics of a screening compound collection: diversity coverage against as many targets as possible and the appropriate compound properties for perturbing the target as desired. This grid of appropriately sized compound plate sets provides flexibility for iterative and subset screening allowing plate set selections based on molecular properties, as well as chemical and biological diversity criteria. The regular reproduction from tubes (every 3–4 years) allows for institutional learning across all hit finding projects regarding both the composition and arrangement of compounds within the screening deck. One key learning from the 2015 screening deck was to rely on solubility and permeability as main determinants for compound attractiveness rather than MW and clogP ranges. This was enabled by the access to high quality predictive models for these properties. In addition, newly available machine-learning models played a key role in extending the diversity selection using categories based on inferred biological activity in the 2019 approach reducing the dominance of chemical scaffold categories in diversity selection. Specifically, adding compound set based clustering of inferred activity profiles enabled us to increase the weight of biological activity information on the selection process. During the use of the 2015 screening deck, it became apparent that continuous communication and education of the screeners and the project teams are critical. Due to the perceived attractiveness of starting points in columns A and B, project teams tended to screen mostly those, irrespective of the target class pursued. However, especially for difficult target classes, chances to identify starting points were likely highest in column C due to the hard cut-offs used in columns A and B for molecular weight and lipophilicity.</div><div class="NLM_p last">Overall, we believe that our 2019 plate-based screening deck grid is enabling for diversity-driven subset and iterative screening and that the current design provides a well-balanced compound distribution in the screening deck.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01332" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01332?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01332</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Computational methods for the predictive models and the different clustering methods (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf">jm0c01332_si_001.pdf (778.25 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01332" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88680" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88680" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ansgar Schuffenhauer</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6385-0414" title="Orcid link">http://orcid.org/0000-0001-6385-0414</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e6878895818794c895858e93808083888e87938394a68889908794928f95c885898b"><span class="__cf_email__" data-cfemail="a1c0cfd2c6c0d38fd2c2c9d4c7c7c4cfc9c0d4c4d3e1cfced7c0d3d5c8d28fc2cecc">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel K. Baeschlin</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2227-7452" title="Orcid link">http://orcid.org/0000-0002-2227-7452</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6400050a0d01084a06050117070c080d0a240a0b120516100d174a070b09"><span class="__cf_email__" data-cfemail="90f4f1fef9f5fcbef2f1f5e3f3f8fcf9fed0feffe6f1e2e4f9e3bef3fffd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nadine Schneider</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5824-2764" title="Orcid link">http://orcid.org/0000-0001-5824-2764</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel Hintermann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-6803" title="Orcid link">http://orcid.org/0000-0001-6853-6803</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas Auld</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8164-9375" title="Orcid link">http://orcid.org/0000-0002-8164-9375</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jutta Blank</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simona Cotesta</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline Engeloch</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nikolas Fechner</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3852-3950" title="Orcid link">http://orcid.org/0000-0003-3852-3950</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christoph Gaul</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9162-553X" title="Orcid link">http://orcid.org/0000-0001-9162-553X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jerome Giovannoni</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johanna Jansen</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for BioMedical Research—Emeryville, 5300 Chiron Way, Emeryville, California 94608-2916, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3937-6243" title="Orcid link">http://orcid.org/0000-0003-3937-6243</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Joslin</span> - <span class="hlFld-Affiliation affiliation">Genomics
Institute of the Novartis Foundation, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philipp Krastel</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eugen Lounkine</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span>Present Address:
                        E.L.: Merck & Co. Inc., 33 Avenue Louis Pasteur, Boston, MA 02115, U.S</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Manchester</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lauren G. Monovich</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8455-4171" title="Orcid link">http://orcid.org/0000-0001-8455-4171</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Paola Pelliccioli</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuel Schwarze</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael D. Shultz</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research Inc., 181 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7920-955X" title="Orcid link">http://orcid.org/0000-0001-7920-955X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nikolaus Stiefl</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2562-7080" title="Orcid link">http://orcid.org/0000-0003-2562-7080</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Source code of the diversity picking algorithm alongside example input and output for the SSEC set on github: <a href="https://github.com/Novartis/div_rank" class="extLink">https://github.com/Novartis/div_rank</a>. Substructure filters used which have been contributed to rdkit on github: <a href="https://github.com/rdkit/rdkit/tree/master/Contrib/NIBRSubstructureFilters" class="extLink">https://github.com/rdkit/rdkit/tree/master/Contrib/NIBRSubstructureFilters</a>.</p><br /></br><p class="inlineNote">All authors are/were Novartis employees, and this research was financed by Novartis.</p></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Ansgar Schuffenhauer</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2387-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Ansgar Schuffenhauer</b> completed his Ph.D. at the Georg-August University in Göttingen, Germany, in the Department of Organic Chemistry (L. F. Tietze). He joined subsequently the group of Peter Willett at the University of Sheffield as a postdoc. He started his pharmaceutical industry career at Novartis in 2001. Over the years he impacted as chemoinformatics expert to many hits hit discovery projects and had a leading rule in the curation of the screening collection. Currently he is in the data science group of the Chemical Biology and Therapeutics platform, with a special interest in high-throughput hit discovery methods such as encoded libraries and their synergy with machine learning.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Nadine Schneider</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2392-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Nadine Schneider</b> obtained a B.Sc. and M.Sc. in Bioinformatics from the Saarland University in Germany. She did her Ph.D. specializing in Molecular Modeling in the group of Prof. Dr. Matthias Rarey at the University of Hamburg, Germany. In 2014 she joined the Novartis Institutes for BioMedical Research (NIBR) in Basel as a postdoc focusing on cheminformatics and data science under the supervision of Dr. Gregory Landrum and Dr. Nikolaus Stiefl. Since 2017, she is a data scientist in the Computer-Aided Drug Design team in Global Discovery Chemistry in NIBR, Basel.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Samuel Hintermann</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2397-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Samuel Hintermann</b> studied at the ETH Zürich where he received a Ph.D. in Chemistry (D. Seebach) followed by postgradual studies at the Memorial Sloan-Kettering Cancer Center in New York (Samuel Danishefsky). He started his career in the pharmaceutical industry in 1998 in the Neuroscience Department of Novartis. Over the years, he led different projects in the neuroscience and autoimmunity disease area and acquired experience in PET imaging. Currently, he is a member of the Hit Generation Sciences group at Novartis with special interest in encoded libraries and Library Sciences in general.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Douglas Auld</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2402-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Douglas Auld</b> completed his Ph.D. at UNC Chapel Hill with Prof. Gary J. Pielak in the Department of Chemistry and postdoctoral training at Massachusetts Institute of Technology with Prof. Paul Schimmel. Following postdoctoral training, Dr. Auld joined Pharmacopeia, a fledging biotechnology company aimed at applying combinatorial chemistry to drug discovery. Dr. Auld was then hired by the U.S. National Institutes of Health to help start the NIH Chemical Genomics Center (NCGC). At the NCGC Dr. Auld worked to establish a center of scientific excellence that has now become part of the National Center for Advancing Translational Sciences at the NIH. At NIBR Dr. Auld’s lab provides expertise in high-throughput biology to support drug development efforts. Dr. Auld’s lab at NIBR also provides resources to enable academic collaborations.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Jutta Blank</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2407-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jutta Blank</b> holds a degree and Ph.D. in Biochemistry from the University of Bayreuth in Germany. She moved to the U.K. for postdoctoral research, first in Structural Studies at the MRC Laboratory for Molecular Biology in Cambridge and then in the Target Discovery and Infection Department of AstraZeneca Alderley Park. Her lead discovery career started at the Syngenta Jealott’s Hill Research Centre as a team leader for in vitro assays before she moved to Basel in 2002 to join the NIBR Drug Discovery Centre. Since then, Jutta impacted numerous lead finding projects and held various roles, mostly recently leading the Early Translational Biology group in Basel.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Simona Cotesta</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2412-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Simona Cotesta</b> received her Ph.D. in Biochemistry at the ETH, Zurich, Switzerland. During her postdoctoral research, she worked in the molecular modeling field in the computational chemistry group at Roche, Basel (postdoc under the supervision of M. Stahl). In 2005 she started her drug discovering career in NIBR (Basel) and joined the Computer-Aided Drug Design team in Global Discovery Chemistry. She is currently medicinal chemistry project leader in the oncology area and group leader in the Computer-Aided Drug Design team.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Caroline Engeloch</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2417-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Caroline Engeloch</b> studied at the Universities of Strasbourg and Mulhouse, France, and received a Ph.D. from the University of Neuchâtel, Switzerland. She joined the Novartis Compound Management group in Basel in 2000 to start the operations of the first generation of automated liquid stores and related liquid handling activities. She now leads the Compound Management Team in Basel.</p></figure></div><div class="bio" rid="ath8"><figure id="ath8" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Nikolas Fechner</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2422-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Nikolas Fechner</b> studied bioinformatics at the University of Tübingen, Germany, and received the diploma in 2006. Between 2006 and 2010, he did his Ph.D. studies also at the University of Tübingen in the context of machine learning methods for small molecules, finishing with a Dr. rer. nat. in Computer Science. After this, he went on to work in the pharmaceutical industry, first as a scientific programmer at Eli Lilly and Company in the U.K. In 2012, he joined the Informatics Department of the Novartis Institutes for BioMedical Research (NIBR) where he is working on machine learning applications for small molecules.</p></figure></div><div class="bio" rid="ath9"><figure id="ath9" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Christoph Gaul</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2427-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Christoph Gaul</b> studied Chemistry at the Ludwig-Maximilians-Universität München (B.A. in Chemistry), The University of Texas at Austin (M.A. in Chemistry, Stephen Martin), and the ETH Zürich (Ph.D. in Chemistry, Dieter Seebach) before he went on to conduct postdoctoral research at the Memorial Sloan Kettering Cancer Center in New York (Samuel Danishefsky). In 2004, Christoph started his career as a Medicinal Chemist at Novartis in Basel, Switzerland, first in the Hit to Lead group and later as a Project and Group Leader in the Oncology Chemistry area. Currently, Christoph is heading the Hit Generation Sciences group at Novartis, building and providing expertise in hit generation chemistry, discovery technologies, and library sciences.</p></figure></div><div class="bio" rid="ath10"><figure id="ath10" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Jerome Giovannoni</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2432-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jerome Giovannoni</b> received his Master of Physics and Chemistry from the ESPCI-Paris. He then moved to the University of Montpellier (France) where he obtained his Ph.D. in Organic Chemistry with Professor Martinez. Subsequently, he joined the company Chemspeed Technologies as an Application Chemist. In 2005, Jerome Giovannoni moved to the agrochemical company Syngenta Crop Protection as a laboratory head in analytical development, with a focus on seed treatment products. He joined Novartis in 2009 where he implemented a high-throughput LCMS quality control laboratory within the Compound Management Department. From 2013 to 2016, he took over responsibility for the Automated Solution Production team. Since 2016, Dr. Giovannoni has been leading a team focusing on mass-spectrometry based screening and profiling assays.</p></figure></div><div class="bio" rid="ath11"><figure id="ath11" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Johanna Jansen</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2437-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Johanna Jansen</b> received her Ph.D. in Computational Medicinal Chemistry from the School of Pharmacy, University of Groningen (The Netherlands) and completed a postdoctoral term at the BioMedical Center of Uppsala University (Sweden). She started her industrial drug discovery career at Astra (Sweden) and moved from there to Chiron Corporation in the United States. Johanna is currently a Director at the Novartis Institutes for BioMedical Research and heads the Computer-Aided Drug Discovery (CADD) group at their Emeryville campus. She has contributed to several clinical candidates and to new approaches in CADD. She has been a driver and leader for many initiatives and platforms around chemical space exploration and leveraging all available knowledge in order to discover new and high quality chemical matter for drug discovery.</p></figure></div><div class="bio" rid="ath12"><figure id="ath12" data-index="25" class="article__inlineFigure"><h2 class="fig-label">John Joslin</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2443-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>John Joslin</b> received his Ph.D. in Cancer Biology at the University of Chicago, modeling therapy-related leukemia in mice. He showed that functional loss of Egr1 is an initiating event in the pathogenesis of AML/MDS. After completing his postdoctoral training in immunology, Dr. Joslin joined the Assay Development and High-Throughput Screening group (AD/HTS) at the Genomics Institute of the Novartis Research Foundation (GNF). His early efforts focused on the development of a fully automated flow cytometry screening system. This was an industry first and is still regarded as the leading platform for automated flow cytometry. Dr. Joslin continues to innovate in high-throughput screening and is currently Director of AD/HTS at GNF. Present efforts are focused on highly multiplexed gene expression, 3D biology, and microfluidics.</p></figure></div><div class="bio" rid="ath13"><figure id="ath13" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Philipp Krastel</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2448-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Philipp Krastel</b> studied chemistry at the Universities in Kiel and Goettingen. He received a Ph.D. in 2001 specializing in natural product research. After one year of postdoctoral training at the Institute of Biomelecular Chemistry at the University of Göttingen he started in 2002 as a Research Investigator in the Natural Products Units of the Novartis Institutes of Biomedical Research in Basel focusing on the analytical and preparative characterization of secondary metabolites from microorganisms. Since 2008 he has been leading a team of chemists of the Natural Products and Bimolecular Chemistry group at Novartis.</p></figure></div><div class="bio" rid="ath14"><figure id="ath14" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Eugen Lounkine</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2453-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Eugen Lounkine</b>, Dr., is a data scientist and cheminformatician with 10+ years of experience in early drug discovery. He graduated in 2009 from University of Bonn in Molecular Biomedicine and Computational Life Sciences under Prof. Dr. Juergen Bajorath,. He then did his postdoc under Meir Glick, Jeremy Jenkins, and Laszlo Urban at Novartis in Cambridge, MA, where he looked at the connection between (predicted) drug off-target profiles and adverse reactions in the clinic. After his postdoc, he continued as a lab head (2012) and later group leader (2018) at Novartis within Chemical Biology and Therapeutics Data Science. In December 2019 Eugen joined the Computational and Structural Chemistry group at Merck, where he is responsible for Chemical Biology Data Strategy.</p></figure></div><div class="bio" rid="ath15"><figure id="ath15" data-index="28" class="article__inlineFigure"><h2 class="fig-label">John Manchester</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2458-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>John Manchester</b> is a computational chemist with 20 years of experience applying informatics and modeling in all stages of drug discovery. He has contributed to many drug candidates and coauthored numerous papers summarizing drug discovery programs. He has a particular interest in understanding mechanisms underlying disease processes through iterating between modeling and experiment. He obtained his Ph.D. from SUNY Buffalo/Pacific Northwest National Labs and after a postdoc at the University of Rochester worked at AstraZeneca for 15 years prior to joining Novartis, where he is currently a colead in the Computer-Aided Drug Discovery group.</p></figure></div><div class="bio" rid="ath16"><figure id="ath16" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Lauren G. Monovich</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2463-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Lauren G. Monovich</b> received a B.A. in Chemistry from Kalamazoo College before completing her doctoral studies with Professor Gary Molander (University of Colorado, Boulder). After joining Novartis in 1998, Lauren has been privileged to lead drug discovery projects across multiple disease areas, project stages, and modalities and to invent a marketed therapeutic agent (osilodrostat for Cushing’s disease). In 2018, she received the Novartis Leading Scientist award for her work on orally exposed macrocyclic peptides. She established and currently coleads the Integrated Library Team of experts on library technology, design, and strategy. She is an active recruiter, founding member of Novartis’s Women in Chemistry Group, colead of Novartis’s Inclusion Council, and long-standing contributor to educational community outreach.</p></figure></div><div class="bio" rid="ath17"><figure id="ath17" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Anna Paola Pelliccioli</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2468-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Anna Paola Pelliccioli</b> studied Chemistry at the University of Perugia, Italy and obtained her Ph.D. in Chemistry from the University of Basel, Switzerland. She then moved to the R&D Informatics Department within Syngenta, in Basel, where she was responsible for reporting of experimental data. In 2009 she joined NIBR Informatics (NX). For the past few years she has been leading an interdisciplinary team of analysts, scientists, and software developers focused on delivering and supporting applications for chemists. One such application is Compound Series & Favorites. She is currently transitioning into a new role within NX, where she will be responsible for scientific digital document management.</p></figure></div><div class="bio" rid="ath18"><figure id="ath18" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Manuel Schwarze</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2473-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Manuel Schwarze</b> received his Master’s degree in Computer Science from TU Berlin, Germany, and started his career pioneering the new field of Web conferencing at Astound in Toronto, Canada, in 1999. In 2002, he joined Novartis in Basel, Switzerland, developing the PKI Toolkit for applied security and providing global consultancy in this area before joining Novartis Institutes of BioMedical Research (NIBR) in 2009. After leading a major project to bring NIBR chemists the paperless lab, Manuel worked as an architect, technical lead, and software engineer on cheminformatics systems in both the data analytics and machine learning spaces with Dr. Gregory Landrum, Dr. Anna Pelliccioli, and Dr. Nikolas Fechner. He is the primary contributor for the open-source KNIME RDKit Extensions, widely used in drug discovery processes.</p></figure></div><div class="bio" rid="ath19"><figure id="ath19" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Michael D. Shultz</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2478-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Michael D. Shultz</b> earned a B.A. in Chemistry from Albion College and a Ph.D. in Organic Chemistry with Professor Jean A. Chmielewski (Purdue University) and then received postdoctoral training with Professor Laura L. Kiessling (University of Wisconsin). He joined Novartis (Summit, NJ) in 2001 in the Arthritis and Bone Metabolism Department. Since then, he worked in the areas of Oncology and Immuno-Oncology at NIBR Cambridge and Shanghai prior to joining the Cardiovascular and Metabolism disease area in 2017. He led projects targeting HDACs, tankyrase, and IDO and teams in Chemical Biology (NIBR, Shanghai) and Heart Failure. His contributions to four compounds in clinical trials helped spark an ongoing interest in how metrics and properties are used in high throughput screening and during molecular optimizations.</p></figure></div><div class="bio" rid="ath20"><figure id="ath20" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Nikolaus Stiefl</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2483-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Nikolaus Stiefl</b>’s scientific career focused on pharmaceutical research early on. He received his degree from the School of Pharmacy, University of Wuerzburg, Germany, which was followed by a one year internship at the molecular modeling group of GlaxoWellcome, Stevenage, U.K. After coming back to Wuerzburg for his Ph.D. studies in Chemoinformatics and Chemometrics, he joined Lilly in Hamburg, Germany, for his postgraduate studies. In 2005 he joined the Novartis Institutes of BioMedical Research (NIBR) where he has contributed to several clinical candidates, led multiple informatics projects, and developed novel scientific approaches in CADD. Today, he is responsible for multiple large data science projects and is a colead in the Computer-Aided Drug Discovery group.</p></figure></div><div class="bio" rid="ath21"><figure id="ath21" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Daniel K. Baeschlin</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=BIO-d7e2488-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Daniel Baeschlin</b> received a Master of Science from the ETH, Zurich, Switzerland, and a Ph.D. from the University of Cambridge, England, in Organic Synthesis prior to postgraduate studies at the Memorial Sloan-Kettering Cancer Center in New York. He started his drug discovery career in 2001 at Morphochem AG in Basel before he joined the Novartis Institutes of BioMedical Research (NIBR) as medicinal chemist in 2003, where he led a project resulting in a preclinical candidate for diabetes. Since 2008, he has been involved in hit finding at NIBR, first as leader of compound management and today as leader of the high throughput biology team.</p></figure></div><div class="ack" id="ACK-d7e2493-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Designing a screening deck is a team effort building on the contributions of many scientists. We acknowledge the consultancy from the members of the design team for the 2015 version of the screening deck. We thank the chemists having participated in the review of the substructure filters (Phil Alper, Nicole Blaquiere, Natalie Dales, Martin Eberle, Peter Ertl, Jacob Gora, Brian Hurley, Tom Kirrane, Thomas Knoepfel, Guillaume Lapointe, Philipp Lehr, Richard Lewis, Edwige Lorthiois, Rainer Machauer, Christian Markert, Lynn McGregor, Jeff McKenna, Berndt Oberhauser, Silvio Roggo, Chris Sarko, Tim Schuhmann, Oliver Simic, Finton Sirockin, Aleksandar Stojanovic, Thomas Troxler, Riccardo Vianello). We thank the members of the natural product chemistry team who have reviewed the plate layout for the natural products. We thank the originators of the biological classification schemes for contributing these data: Valery Polyakov and Eric Martin (pQSAR), Yuan Wang (BAFP und BDFP), Paul Selzer and Xian Hang (HTSFP and HCSFP). We acknowledge Bernard Faller, Jörg Berghausen, and Grégori Gerebtzoff for consultancy in the building of the PAMPA model, and Suzanne Skolnik and Stephane Rodde for consultancy in the solubility model building. We acknowledge the internal model review panel for ensuring the quality of these models. We acknowledge the builders and maintainers of our core data warehouses DART/MAGMA/CDF and Hithub for their continued efforts, which made the analyses described in this article possible. We thank all chemists who are contributing their ongoing project data to CS&F, and Werner Breitenstein for the extensive curation of old project data for CS&F. Last but not least our thanks go to the management of Chemical Biology and Therapeutics (CBT) and Global Discovery Chemistry (GDC) platforms for supporting the library efforts.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal activity concentration</p></td></tr><tr><td class="NLM_term">Caco-2</td><td class="NLM_def"><p class="first last">colorectal adenocarcinoma cell line</p></td></tr><tr><td class="NLM_term">clogP</td><td class="NLM_def"><p class="first last">calculated log <i>P</i></p></td></tr><tr><td class="NLM_term">cpd</td><td class="NLM_def"><p class="first last">compound</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fingerprint</p></td></tr><tr><td class="NLM_term">HT-picking</td><td class="NLM_def"><p class="first last">high-throughput-picking</p></td></tr><tr><td class="NLM_term">MDCK-LE</td><td class="NLM_def"><p class="first last">Madin–Darby canine kidney cells with limited efflux</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">QC</td><td class="NLM_def"><p class="first last">quality control</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 107 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wildey, M. J.</span>; <span class="NLM_string-name">Haunso, A.</span>; <span class="NLM_string-name">Tudor, M.</span>; <span class="NLM_string-name">Webb, M.</span>; <span class="NLM_string-name">Connick, J. H.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Screening</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier Inc.</span>, <span class="NLM_year">2017</span>; Vol.  <span class="NLM_volume">50</span>, pp  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/bs.armc.2017.08.004</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fbs.armc.2017.08.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=149-195&author=M.+J.+Wildey&author=A.+Haunso&author=M.+Tudor&author=M.+Webb&author=J.+H.+Connickauthor=R.+A.+Goodnow&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26atitle%3DHigh-Throughput%2520Screening%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DElsevier%2520Inc%26date%3D2017%26volume%3D50%26spage%3D149%26epage%3D195%26doi%3D10.1016%2Fbs.armc.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirovic, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garyantes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzberg, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paslay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G. S.</span></span> <span> </span><span class="NLM_article-title">Impact of High-Throughput Screening in Biomedical Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+High-Throughput+Screening+in+Biomedical+Research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0ljTMamnd46ByA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520High-Throughput%2520Screening%2520in%2520Biomedical%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">The Discovery of First-in-Class Drugs: Origins and Evolution</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&issue=8&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+Discovery+of+First-in-Class+Drugs%3A+Origins+and+Evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0liexKQ9MgXVvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520Discovery%2520of%2520First-in-Class%2520Drugs%253A%2520Origins%2520and%2520Evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26issue%3D8%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span> <span> </span><span class="NLM_article-title">Encoded Library Technologies as Integrated Lead Finding Platforms for Drug Discovery</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1629</span>, <span class="refDoi"> DOI: 10.3390/molecules24081629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.3390%2Fmolecules24081629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpslyju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1629&issue=8&author=J.+Ottlauthor=L.+Lederauthor=J.+V.+Schaeferauthor=C.+E.+Dumelin&title=Encoded+Library+Technologies+as+Integrated+Lead+Finding+Platforms+for+Drug+Discovery&doi=10.3390%2Fmolecules24081629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded library technologies as integrated lead finding platforms for drug discovery</span></div><div class="casAuthors">Ottl, Johannes; Leder, Lukas; Schaefer, Jonas V.; Dumelin, Christoph E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1629</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The scope of targets investigated in pharmaceutical research is continuously moving into uncharted territory.  Consequently, finding suitable chem. matter with current compd. collections is proving increasingly difficult.  Encoded library technologies enable the rapid exploration of large chem. space for the identification of ligands for such targets.  These binders facilitate drug discovery projects both as tools for target validation, structural elucidation and assay development as well as starting points for medicinal chem.  Novartis internalized two complementing encoded library platforms to accelerate the initiation of its drug discovery programs.  For the identification of low-mol. wt. ligands, we apply DNA-encoded libraries.  In addn., encoded peptide libraries are employed to identify cyclic peptides.  This review discusses how we apply these two platforms in our research and why we consider it beneficial to run both pipelines inhouse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8YbvIrAzf7rVg90H21EOLACvtfcHk0liexKQ9MgXVvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpslyju7w%253D&md5=8ed3e114146bf60f513b94d1a83f992a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24081629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24081629%26sid%3Dliteratum%253Aachs%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DSchaefer%26aufirst%3DJ.%2BV.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26atitle%3DEncoded%2520Library%2520Technologies%2520as%2520Integrated%2520Lead%2520Finding%2520Platforms%2520for%2520Drug%2520Discovery%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26issue%3D8%26spage%3D1629%26doi%3D10.3390%2Fmolecules24081629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannocci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimbacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">20 Years of DNA-Encoded Chemical Libraries</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">12747</span>– <span class="NLM_lpage">12753</span>, <span class="refDoi"> DOI: 10.1039/c1cc15634a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1039%2Fc1cc15634a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=22083211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFamsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=12747-12753&issue=47&author=L.+Mannocciauthor=M.+Leimbacherauthor=M.+Wichertauthor=J.+R.+Scheuermannauthor=D.+Neri&title=20+Years+of+DNA-Encoded+Chemical+Libraries&doi=10.1039%2Fc1cc15634a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">20 Years of DNA-encoded chemical libraries</span></div><div class="casAuthors">Mannocci, Luca; Leimbacher, Markus; Wichert, Moreno; Scheuermann, Joerg; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">12747-12753</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The identification of specific binding mols. is a central problem in chem., biol. and medicine.  Therefore, technologies, which facilitate ligand discovery, may substantially contribute to a better understanding of biol. processes and to drug discovery.  DNA-encoded chem. libraries represent a new inexpensive tool for the fast and efficient identification of ligands to target proteins of choice.  Such libraries consist of collections of org. mols., covalently linked to a unique DNA tag serving as an amplifiable identification bar code.  DNA-encoding enables the in vitro selection of ligands by affinity capture at sub-picomolar concns. on virtually any target protein of interest, in analogy to established selection methodologies like antibody phage display.  Multiple strategies have been investigated by several academic and industrial labs. for the construction of DNA-encoded chem. libraries comprising up to millions of DNA-encoded compds.  The implementation of next generation high-throughput sequencing enabled the rapid identification of binding mols. from DNA-encoded libraries of unprecedented size.  This article reviews the development of DNA-encoded library technol. and its evolution into a novel drug discovery tool, commenting on challenges, perspectives and opportunities for the different exptl. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawUGgkRIh-7Vg90H21EOLACvtfcHk0liexKQ9MgXVvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFamsL3O&md5=835ec84173e5a1343484cb3574a4ec4b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2Fc1cc15634a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc15634a%26sid%3Dliteratum%253Aachs%26aulast%3DMannocci%26aufirst%3DL.%26aulast%3DLeimbacher%26aufirst%3DM.%26aulast%3DWichert%26aufirst%3DM.%26aulast%3DScheuermann%26aufirst%3DJ.%2BR.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3D20%2520Years%2520of%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26issue%3D47%26spage%3D12747%26epage%3D12753%26doi%3D10.1039%2Fc1cc15634a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Lead Discovery by DNA-Encoded Chemical Libraries</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11–12</span>),  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2007.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=17532531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvVart7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=465-471&issue=11%E2%80%9312&author=S.+Melkkoauthor=C.+Dumelinauthor=J.+Scheuermannauthor=D.+Neri&title=Lead+Discovery+by+DNA-Encoded+Chemical+Libraries&doi=10.1016%2Fj.drudis.2007.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lead discovery by DNA-encoded chemical libraries</span></div><div class="casAuthors">Melkko, Samu; Dumelin, Christoph E.; Scheuermann, Joerg; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11 & 12</span>),
    <span class="NLM_cas:pages">465-471</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The isolation of specific binding mols. is a central problem in the drug discovery process and might be useful for the elucidation of the biol. function of proteins identified in genome and proteome research.  Libraries of org. mols., conjugated covalently to DNA tags that serve as identification bar codes, were proposed recently as a way to identify ligands of target proteins of choice efficiently.  Here, we analyze the different strategies for constructing DNA-encoded chem. libraries, and the potential and challenges of this promising technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8bZwa_dmVUrVg90H21EOLACvtfcHk0lhY8PAyyyk0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvVart7g%253D&md5=b8b73bc50d523e2465dbb3eb08290adc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DDumelin%26aufirst%3DC.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DLead%2520Discovery%2520by%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26issue%3D11%25E2%2580%259312%26spage%3D465%26epage%3D471%26doi%3D10.1016%2Fj.drudis.2007.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Toonder, J. M. J.</span></span> <span> </span><span class="NLM_article-title">Microfluidics for Cell-Based High Throughput Screening Platforms-A Review</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>903</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2015.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.aca.2015.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26709297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFagt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=903&publication_year=2016&pages=36-50&author=G.+Duauthor=Q.+Fangauthor=J.+M.+J.+den+Toonder&title=Microfluidics+for+Cell-Based+High+Throughput+Screening+Platforms-A+Review&doi=10.1016%2Fj.aca.2015.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidics for cell-based high throughput screening platforms-A review</span></div><div class="casAuthors">Du, Guansheng; Fang, Qun; den Toonder, Jaap M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">903</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-50</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the last decades, the basic techniques of microfluidics for the study of cells such as cell culture, cell sepn., and cell lysis, have been well developed.  Based on cell handling techniques, microfluidics has been widely applied in the field of PCR (Polymerase Chain Reaction), immunoassays, organ-on-chip, stem cell research, and anal. and identification of circulating tumor cells.  As a major step in drug discovery, high-throughput screening allows rapid anal. of thousands of chem., biochem., genetic or pharmacol. tests in parallel.  In this review, the authors summarize the application of microfluidics in cell-based high throughput screening.  The screening methods mentioned in this paper include approaches using the perfusion flow mode, the droplet mode, and the microarray mode.  The authors also discuss the future development of microfluidic based high throughput screening platform for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwdyXF1fDA1bVg90H21EOLACvtfcHk0lhY8PAyyyk0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFagt7fP&md5=1077740c0b8ae724d1b0e0d36d09e631</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2015.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2015.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DG.%26aulast%3DFang%26aufirst%3DQ.%26aulast%3Dden%2BToonder%26aufirst%3DJ.%2BM.%2BJ.%26atitle%3DMicrofluidics%2520for%2520Cell-Based%2520High%2520Throughput%2520Screening%2520Platforms-A%2520Review%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2016%26volume%3D903%26spage%3D36%26epage%3D50%26doi%3D10.1016%2Fj.aca.2015.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gimeno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojeda-Montes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomás-Hernández, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cereto-Massagué, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrán-Debón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujadas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Vallvé, S.</span></span> <span> </span><span class="NLM_article-title">The Light and Dark Sides of Virtual Screening: What Is There to Know?</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1375</span>, <span class="refDoi"> DOI: 10.3390/ijms20061375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.3390%2Fijms20061375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=1375&issue=6&author=A.+Gimenoauthor=M.+Ojeda-Montesauthor=S.+Tom%C3%A1s-Hern%C3%A1ndezauthor=A.+Cereto-Massagu%C3%A9author=R.+Beltr%C3%A1n-Deb%C3%B3nauthor=M.+Muleroauthor=G.+Pujadasauthor=S.+Garcia-Vallv%C3%A9&title=The+Light+and+Dark+Sides+of+Virtual+Screening%3A+What+Is+There+to+Know%3F&doi=10.3390%2Fijms20061375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The light and dark sides of virtual screening: what is there to know?</span></div><div class="casAuthors">Gimeno, Aleix; Ojeda-Montes, Maria Jose; Tomas-Hernandez, Sarah; Cereto-Massague, Adria; Beltran-Debon, Raul; Mulero, Miquel; Pujadas, Gerard; Garcia-Vallve, Santiago</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1375</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Virtual screening consists of using computational tools to predict potentially bioactive compds. from files contg. large libraries of small mols.  Virtual screening is becoming increasingly popular in the field of drug discovery as in silico techniques are continuously being developed, improved, and made available.  As most of these techniques are easy to use, both private and public organizations apply virtual screening methodologies to save resources in the lab.  However, it is often the case that the techniques implemented in virtual screening workflows are restricted to those that the research team knows.  Moreover, although the software is often easy to use, each methodol. has a series of drawbacks that should be avoided so that false results or artifacts are not produced.  Here, we review the most common methodologies used in virtual screening workflows in order to both introduce the inexperienced researcher to new methodologies and advise the experienced researcher on how to prevent common mistakes and the improper usage of virtual screening methodologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJWXWcy8LDELVg90H21EOLACvtfcHk0lhY8PAyyyk0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmtL8%253D&md5=e813e0bb5168713ae670d27bbf8060cf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Fijms20061375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20061375%26sid%3Dliteratum%253Aachs%26aulast%3DGimeno%26aufirst%3DA.%26aulast%3DOjeda-Montes%26aufirst%3DM.%26aulast%3DTom%25C3%25A1s-Hern%25C3%25A1ndez%26aufirst%3DS.%26aulast%3DCereto-Massagu%25C3%25A9%26aufirst%3DA.%26aulast%3DBeltr%25C3%25A1n-Deb%25C3%25B3n%26aufirst%3DR.%26aulast%3DMulero%26aufirst%3DM.%26aulast%3DPujadas%26aufirst%3DG.%26aulast%3DGarcia-Vallv%25C3%25A9%26aufirst%3DS.%26atitle%3DThe%2520Light%2520and%2520Dark%2520Sides%2520of%2520Virtual%2520Screening%253A%2520What%2520Is%2520There%2520to%2520Know%253F%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26issue%3D6%26spage%3D1375%26doi%3D10.3390%2Fijms20061375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haning, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, H.</span></span> <span> </span><span class="NLM_article-title">An Approach towards Enhancement of a Screening Library: The Next Generation Library Initiative (NGLI) at Bayer — against All Odds?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2018.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30562586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1SnsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=668-672&issue=3&author=M.+Follmannauthor=H.+Briemauthor=A.+Steinmeyerauthor=A.+Hillischauthor=M.+H.+Schmittauthor=H.+Haningauthor=H.+Meier&title=An+Approach+towards+Enhancement+of+a+Screening+Library%3A+The+Next+Generation+Library+Initiative+%28NGLI%29+at+Bayer+%E2%80%94+against+All+Odds%3F&doi=10.1016%2Fj.drudis.2018.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An approach towards enhancement of a screening library: The Next Generation Library Initiative (NGLI) at Bayer - against all odds?</span></div><div class="casAuthors">Follmann, Markus; Briem, Hans; Steinmeyer, Andreas; Hillisch, Alexander; Schmitt, Monika H.; Haning, Helmut; Meier, Heinrich</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">668-672</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmaceutical companies often refer to 'screening their library' when performing high-throughput screening (HTS) on a corporate compd. collection to identify lead structures for small-mol. drug discovery programs.  Characteristics of such a library, including the size, chem. space covered, and physicochem. properties, often det. the success of a screening campaign.  Therefore, strategies to maintain and enhance the overall quality of screening collections are crucial to stay competitive and to cope with the 'novelty erosion' that is obsd. gradually.  The Next Generation Library Initiative (NGLI), the enhancement of Bayer's HTS collection by 500 000 newly designed compds. within 5 years, is addressing exactly this challenge.  Here, we describe this collaborative project, which involves all internal medicinal chemists in a crowd-sourcing approach, as well as selected external partners, to reach this ambitious goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vZrpFbl7FrVg90H21EOLACvtfcHk0lg8NL7CERjrvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1SnsrrK&md5=b2a5aa9b0ff60f0fe751aa10a8019f83</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSteinmeyer%26aufirst%3DA.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DM.%2BH.%26aulast%3DHaning%26aufirst%3DH.%26aulast%3DMeier%26aufirst%3DH.%26atitle%3DAn%2520Approach%2520towards%2520Enhancement%2520of%2520a%2520Screening%2520Library%253A%2520The%2520Next%2520Generation%2520Library%2520Initiative%2520%2528NGLI%2529%2520at%2520Bayer%2520%25E2%2580%2594%2520against%2520All%2520Odds%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D668%26epage%3D672%26doi%3D10.1016%2Fj.drudis.2018.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmerlin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüthi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegrist, R.</span></span> <span> </span><span class="NLM_article-title">The Screening Compound Collection: A Key Asset for Drug Discovery</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2533%2Fchimia.2017.667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=29070412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=667-677&issue=10&author=C.+Bossauthor=J.+Hazemannauthor=T.+Kimmerlinauthor=M.+von+Korffauthor=U.+L%C3%BCthiauthor=O.+Peterauthor=T.+Sanderauthor=R.+Siegrist&title=The+Screening+Compound+Collection%3A+A+Key+Asset+for+Drug+Discovery&doi=10.2533%2Fchimia.2017.667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The screening compound collection: a key asset for drug discovery</span></div><div class="casAuthors">Boss, Christoph; Hazemann, Julien; Kimmerlin, Thierry; von Korff, Modest; Luthi, Urs; Peter, Oliver; Sander, Thomas; Siegrist, Romain</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">In this case study on an essential instrument of modern drug discovery, we summarize our successful efforts in the last four years toward enhancing the Actelion screening compd. collection.  A key organizational step was the establishment of the Compd. Library Committee (CLC) in Sept. 2013.  This cross-functional team consisting of computational scientists, medicinal chemists and a biologist was endowed with a significant annual budget for regular new compd. purchases.  Based on an initial library anal. performed in 2013, the CLC developed a New Library Strategy.  The established continuous library turn-over mode, and the screening library size of 300'000 compds. were maintained, while the structural library quality was increased.  This was achieved by shifting the selection criteria from 'druglike' to 'leadlike' structures, enriching for non-flat structures, aiming for compd. novelty, and increasing the ratio of higher cost 'Premium Compds.'.  Novel chem. space was gained by adding natural compds., macrocycles, designed and focused libraries to the collection, and through mutual exchanges of proprietary compds. with agrochem. companies.  A comparative anal. in 2016 provided evidence for the pos. impact of these measures.  Screening the improved library has provided several highly promising hits, including a macrocyclic compd., that are currently followed up in different Hit-to-Lead and Lead Optimization programs.  It is important to state that the goal of the CLC was not to achieve higher HTS hit rates, but to increase the chances of identified hits to serve as the basis of successful early drug discovery programs.  The experience gathered so far legitimates the New Library Strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNca95IroBS7Vg90H21EOLACvtfcHk0lg8NL7CERjrvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeiurc%253D&md5=e919cd047c256713c3b48d942c432b9c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.667%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DHazemann%26aufirst%3DJ.%26aulast%3DKimmerlin%26aufirst%3DT.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DPeter%26aufirst%3DO.%26aulast%3DSander%26aufirst%3DT.%26aulast%3DSiegrist%26aufirst%3DR.%26atitle%3DThe%2520Screening%2520Compound%2520Collection%253A%2520A%2520Key%2520Asset%2520for%2520Drug%2520Discovery%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26issue%3D10%26spage%3D667%26epage%3D677%26doi%3D10.2533%2Fchimia.2017.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varghese, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span> <span> </span><span class="NLM_article-title">An Analysis of Different Components of a High-Throughput Screening Library</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2057</span>– <span class="NLM_lpage">2068</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ymtbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=2057-2068&issue=10&author=A.+Sahaauthor=T.+Vargheseauthor=A.+Liuauthor=S.+J.+Allenauthor=T.+Mirzadeganauthor=M.+D.+Hack&title=An+Analysis+of+Different+Components+of+a+High-Throughput+Screening+Library&doi=10.1021%2Facs.jcim.8b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of Different Components of a High-Throughput Screening Library</span></div><div class="casAuthors">Saha, Arjun; Varghese, Teena; Liu, Annie; Allen, Samantha J.; Mirzadegan, Taraneh; Hack, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2057-2068</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since many projects at pharmaceutical organizations get their start from a high-throughput screening (HTS) campaign, improving the quality of the HTS deck can improve the likelihood of discovering a high-quality lead mol. that can be progressed to a drug candidate.  Over the past decade, Janssen has implemented several strategies for external compd. acquisition to augment the screening deck beyond the chem. space and no. of mols. synthesized for internal projects.  In this report, the authors analyzed the performance of each of those compd. collections in the screening campaigns performed internally within Janssen during the last five years.  The authors classified the screening library into two broad categories: Internal and External.  The comparison of the performance of these sets of libraries was done by considering the primary, confirmation, and dose response hit rates.  The anal. revealed that Internal compds. (resulting from numerous medicinal chem. efforts against diverse protein targets) have higher av. confirmation hit rates than External ones; however, actives from both categories show similar probabilities of hitting multiple distinct targets.  The authors also investigated the property landscape of both sets of libraries to identify the key elements which make a difference in these categories of compds.  From this anal., Janssen aims to understand the descriptor landscape of the compds. with the highest hit rates and to use them for improving its future acquisition strategies as well as to inform the plating strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSQmsLK83Y2bVg90H21EOLACvtfcHk0ljjO7G4hzWTEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ymtbbF&md5=5ab8e2c7116a16acc5f030f06036428c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00258%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DA.%26aulast%3DVarghese%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DAllen%26aufirst%3DS.%2BJ.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DHack%26aufirst%3DM.%2BD.%26atitle%3DAn%2520Analysis%2520of%2520Different%2520Components%2520of%2520a%2520High-Throughput%2520Screening%2520Library%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26issue%3D10%26spage%3D2057%26epage%3D2068%26doi%3D10.1021%2Facs.jcim.8b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span> <span> </span><span class="NLM_article-title">Novel Trends in High-Throughput Screening</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">580</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2009.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.coph.2009.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=19775937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=580-588&issue=5&author=L.+M.+Mayrauthor=D.+Bojanic&title=Novel+Trends+in+High-Throughput+Screening&doi=10.1016%2Fj.coph.2009.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Novel trends in high-throughput screening</span></div><div class="casAuthors">Mayr, Lorenz M.; Bojanic, Dejan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">580-588</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established process for lead discovery in Pharma and Biotech companies and is now also being used for basic and applied research in academia.  It comprises the screening of large chem. libraries for activity against biol. targets via the use of automation, miniaturized assays and large-scale data anal.  Since its first advent in the early to mid 1990s, the field of HTS has seen not only a continuous change in technol. and processes, but also an adaptation to various needs in lead discovery.  HTS has now evolved into a mature discipline that is a crucial source of chem. starting points for drug discovery.  Whereas in previous years much emphasis has been put on a steady increase in screening capacity (quant. increase') via automation and miniaturization, the past years have seen a much greater emphasis on content and quality (qual. increase').  Today, many experts in the field see HTS at a crossroad with the need to decide on either higher throughput/more experimentation or a greater focus on assays of greater physiol. relevance, both of which may lead to higher productivity in pharmaceutical R&D.  In this paper, we describe the development of HTS over the past decade and point out our own ideas for future directions of HTS in biomedical research.  We predict that the trend toward further miniaturization will slow down with the balanced implementation of 384 well, 1536 well, and 384 low vol. well plates.  Furthermore, we envisage that there will be much more emphasis on rigorous assay and chem. characterization, particularly considering that novel and more difficult target classes will be pursued.  In recent years we have witnessed a clear trend in the drug discovery community toward rigorous hit validation by the use of orthogonal readout technologies, label free and biophys. methodologies.  We also see a trend toward a more flexible use of the various screening approaches in lead discovery, i.e., the use of both full deck compd. screening as well as the use of focused screening and iterative screening approaches.  Moreover, we expect greater usage of target identification strategies downstream of phenotypic screening and the more effective implementation of affinity selection technologies as a result of advances in chem. diversity methodologies.  We predict that, ultimately, each hit finding strategy will be much more project-related, tailor-made, and better integrated into the broader drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP3Oo1Orzt87Vg90H21EOLACvtfcHk0ljjO7G4hzWTEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnL&md5=50284fdd9f4d610e81fb442b78b7b69e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2009.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2009.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DL.%2BM.%26aulast%3DBojanic%26aufirst%3DD.%26atitle%3DNovel%2520Trends%2520in%2520High-Throughput%2520Screening%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2009%26volume%3D9%26issue%3D5%26spage%3D580%26epage%3D588%26doi%3D10.1016%2Fj.coph.2009.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">The Essential Roles of Chemistry in High-Throughput Screening Triage</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1290</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.60</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.4155%2Ffmc.14.60" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25163000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVeisbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1265-1290&issue=11&author=J.+L.+Dahlinauthor=M.+A.+Walters&title=The+Essential+Roles+of+Chemistry+in+High-Throughput+Screening+Triage&doi=10.4155%2Ffmc.14.60"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The essential roles of chemistry in high-throughput screening triage</span></div><div class="casAuthors">Dahlin, Jayme L.; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1265-1290</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">It is increasingly clear that academic high-throughput screening (HTS) and virtual HTS triage suffers from a lack of scientists trained in the art and science of early drug discovery chem.  Many recent publications report the discovery of compds. by screening that are most likely artifacts or promiscuous bioactive compds., and these results are not placed into the context of previous studies.  For HTS to be most successful, it is our contention that there must exist an early partnership between biologists and medicinal chemists.  Their combined skill sets are necessary to design robust assays and efficient workflows that will weed out assay artifacts, false positives, promiscuous bioactive compds. and intractable screening hits, efforts that ultimately give projects a better chance at identifying truly useful chem. matter.  Expertise in medicinal chem., chemoinformatics and purifn. sciences (anal. chem.) can enhance the post-HTS triage process by quickly removing these problematic chemotypes from consideration, while simultaneously prioritizing the more promising chem. matter for follow-up testing.  It is only when biologists and chemists collaborate effectively that HTS can manifest its full promise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4L_qQtRFBUbVg90H21EOLACvtfcHk0ljjO7G4hzWTEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVeisbjF&md5=4ca8e43d7a85a1f67be930d40bfefad4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.60&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.60%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Essential%2520Roles%2520of%2520Chemistry%2520in%2520High-Throughput%2520Screening%2520Triage%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D11%26spage%3D1265%26epage%3D1290%26doi%3D10.4155%2Ffmc.14.60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haasen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antczak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">How Phenotypic Screening Influenced Drug Discovery: Lessons from Five Years of Practice</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1089/adt.2017.796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1089%2Fadt.2017.796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=28800248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOqtrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=239-246&issue=6&author=D.+Haasenauthor=U.+Schopferauthor=C.+Antczakauthor=C.+Guyauthor=F.+Fuchsauthor=P.+Selzer&title=How+Phenotypic+Screening+Influenced+Drug+Discovery%3A+Lessons+from+Five+Years+of+Practice&doi=10.1089%2Fadt.2017.796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How Phenotypic Screening Influenced Drug Discovery: Lessons from Five Years of Practice</span></div><div class="casAuthors">Haasen, Dorothea; Schopfer, Ulrich; Antczak, Christophe; Guy, Chantale; Fuchs, Florian; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">239-246</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Since 2011, phenotypic screening has been a trend in the pharmaceutical industry as well as in academia.  This renaissance was triggered by analyses that suggested that phenotypic screening is a superior strategy to discover first-in-class drugs.  Despite these promises and considerable investments, pharmaceutical research organizations have encountered considerable challenges with the approach.  Few success stories have emerged in the past 5 years and companies are questioning their investment in this area.  In this contribution, we outline what we have learned about success factors and challenges of phenotypic screening.  We then describe how our efforts in phenotypic screening have influenced our approach to drug discovery in general.  We predict that concepts from phenotypic screening will be incorporated into target-based approaches and will thus remain influential beyond the current trend.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUI8SiRaqgaLVg90H21EOLACvtfcHk0lh1UY9WTHIUhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOqtrjM&md5=b63967012bb77fa413e023794e944fd6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1089%2Fadt.2017.796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2017.796%26sid%3Dliteratum%253Aachs%26aulast%3DHaasen%26aufirst%3DD.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DAntczak%26aufirst%3DC.%26aulast%3DGuy%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DF.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DHow%2520Phenotypic%2520Screening%2520Influenced%2520Drug%2520Discovery%253A%2520Lessons%2520from%2520Five%2520Years%2520of%2520Practice%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2017%26volume%3D15%26issue%3D6%26spage%3D239%26epage%3D246%26doi%3D10.1089%2Fadt.2017.796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paricharak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span> <span> </span><span class="NLM_article-title">Analysis of Iterative Screening with Stepwise Compound Selection Based on Novartis In-House HTS Data</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1255</span>– <span class="NLM_lpage">1264</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00029</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1255-1264&issue=5&author=S.+Paricharakauthor=A.+P.+Ijzermanauthor=A.+Benderauthor=F.+Nigsch&title=Analysis+of+Iterative+Screening+with+Stepwise+Compound+Selection+Based+on+Novartis+In-House+HTS+Data&doi=10.1021%2Facschembio.6b00029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Iterative Screening with Stepwise Compound Selection Based on Novartis In-house HTS Data</span></div><div class="casAuthors">Paricharak, Shardul; Ijzerman, Adriaan P.; Bender, Andreas; Nigsch, Florian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1255-1264</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With increased automation and larger compd. collections, the development of high-throughput screening (HTS) started replacing previous approaches in drug discovery from around the 1980s onward.  However, even today it is not always appropriate, or even feasible, to screen large collections of compds. in a particular assay.  Here, we present an efficient method for iterative screening of small subsets of compd. libraries.  With this method, the retrieval of active compds. is optimized using their structural information and biol. activity fingerprints.  We validated this approach retrospectively on 34 Novartis inhouse HTS assays covering a wide range of assay biol., including cell proliferation, antibacterial activity, gene expression, and phosphorylation.  This method was employed to retrieve subsets of compds. for screening, where selected hits from any given round of screening were used as starting points to select chem. and biol. similar compds. for the next iteration.  By only screening ∼1% of the full screening collection (∼15 000 compds.), the method consistently retrieves diverse compds. belonging to the top 0.5% of the most active compds. for the HTS campaign.  For most of the assays, over half of the compds. selected by the method were found to be among the 5% most active compds. of the corresponding full-deck HTS.  In addn., the stringency of the iterative method can be modified depending on the no. of compds. one can afford to screen, making it a flexible tool to discover active compds. efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogdiZaBr3m77Vg90H21EOLACvtfcHk0lh1UY9WTHIUhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFShu7o%253D&md5=e8586e380292cc7194a06b832a80d9be</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00029%26sid%3Dliteratum%253Aachs%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DNigsch%26aufirst%3DF.%26atitle%3DAnalysis%2520of%2520Iterative%2520Screening%2520with%2520Stepwise%2520Compound%2520Selection%2520Based%2520on%2520Novartis%2520In-House%2520HTS%2520Data%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D1255%26epage%3D1264%26doi%3D10.1021%2Facschembio.6b00029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buendia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linusson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toccaceli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlberg, E.</span></span> <span> </span><span class="NLM_article-title">Accurate Hit Estimation for Iterative Screening Using Venn-ABERS Predictors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00724</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00724" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFCitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1230-1237&issue=3&author=R.+Buendiaauthor=T.+Kogejauthor=O.+Engkvistauthor=L.+Carlssonauthor=H.+Linussonauthor=U.+Johanssonauthor=P.+Toccaceliauthor=E.+Ahlberg&title=Accurate+Hit+Estimation+for+Iterative+Screening+Using+Venn-ABERS+Predictors&doi=10.1021%2Facs.jcim.8b00724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Hit Estimation for Iterative Screening Using Venn-ABERS Predictors</span></div><div class="casAuthors">Buendia, Ruben; Kogej, Thierry; Engkvist, Ola; Carlsson, Lars; Linusson, Henrik; Johansson, Ulf; Toccaceli, Paolo; Ahlberg, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1230-1237</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Iterative screening has emerged as a promising approach to increase the efficiency of high-throughput screening (HTS) campaigns in drug discovery.  By learning from a subset of the compd. library, inferences on what compds. to screen next can be made by predictive models.  One of the challenges of iterative screening is to decide how many iterations to perform.  This is mainly related to difficulties in estg. the prospective hit rate in any given iteration.  In this article, a novel method based on Venn-ABERS predictors is proposed.  The method provides accurate ests. of the no. of hits retrieved in any given iteration during an HTS campaign.  The ests. provide the necessary information to support the decision on the no. of iterations needed to maximize the screening outcome.  Thus, this method offers a prospective screening strategy for early-stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFM6qNl1oTyrVg90H21EOLACvtfcHk0lh1UY9WTHIUhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFCitrc%253D&md5=b6ea3dab102f23a95ea380376729f343</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00724%26sid%3Dliteratum%253Aachs%26aulast%3DBuendia%26aufirst%3DR.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DLinusson%26aufirst%3DH.%26aulast%3DJohansson%26aufirst%3DU.%26aulast%3DToccaceli%26aufirst%3DP.%26aulast%3DAhlberg%26aufirst%3DE.%26atitle%3DAccurate%2520Hit%2520Estimation%2520for%2520Iterative%2520Screening%2520Using%2520Venn-ABERS%2520Predictors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D3%26spage%3D1230%26epage%3D1237%26doi%3D10.1021%2Facs.jcim.8b00724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lajiness, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span> <span> </span><span class="NLM_article-title">Assessment of the Consistency of Medicinal Chemists in Reviewing Sets of Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">4891</span>– <span class="NLM_lpage">4896</span>, <span class="refDoi"> DOI: 10.1021/jm049740z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049740z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVejtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4891-4896&issue=20&author=M.+S.+Lajinessauthor=G.+M.+Maggioraauthor=V.+Shanmugasundaram&title=Assessment+of+the+Consistency+of+Medicinal+Chemists+in+Reviewing+Sets+of+Compounds&doi=10.1021%2Fjm049740z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of the Consistency of Medicinal Chemists in Reviewing Sets of Compounds</span></div><div class="casAuthors">Lajiness, Michael S.; Maggiora, Gerald M.; Shanmugasundaram, Veerabahu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4891-4896</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Medicinal chemists are frequently asked to review lists of compds. to assess their drug- or lead-like nature and to evaluate the suitability of lead compds. based on their "attractiveness" and/or synthetic feasibility as a basis for launching a drug-discovery campaign.  It is often felt that one medicinal chemist's opinion is as good as any other, but is it.  In an attempt to answer this question, an expt. was performed in conjunction with a recent compd. acquisition program (CAP) conducted at Pharmacia.  Historically, the CAP included a review of many thousands of compds. by medicinal chemists who eliminate anything deemed undesirable for any reason.  In a review conducted in 2002, about 22 000 compds. requiring review by medicinal chemists were broken down into 11 lists of approx. 2000 compds. each.  Unknown to the medicinal chemists, a subset of 250 compds., previously rejected by a very experienced senior medicinal chemist, was added to each of the lists.  Most of the 13 medicinal chemists who participated in this process reviewed two lists, although some only reviewed a single list and one reviewed three lists.  Those compds. that were deemed unacceptable were recorded and tabulated in various ways to assess the consistency of the reviews.  It was found that medicinal chemists were not very consistent in the compds. they rejected as being undesirable.  The inconsistency arises from the subjective anal. that all humans utilize when considering "data sets" of any kind.  This has important implications for pharmaceutical project teams where individual medicinal chemists review lists of primary screening hits to identify those compds. suitable for follow-up.  Once a compd. is removed from a list, it and other structurally similar compds. are effectively removed from further consideration.  This can also have an impact on computational chemists who are developing models for assessing the desirability or attractiveness of different classes of compds. for lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZPYRvqjXJjLVg90H21EOLACvtfcHk0liKC93XXi4Zcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVejtL0%253D&md5=d5f8cbd4a56bb4c22782a3beccb3eefd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm049740z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049740z%26sid%3Dliteratum%253Aachs%26aulast%3DLajiness%26aufirst%3DM.%2BS.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26atitle%3DAssessment%2520of%2520the%2520Consistency%2520of%2520Medicinal%2520Chemists%2520in%2520Reviewing%2520Sets%2520of%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D20%26spage%3D4891%26epage%3D4896%26doi%3D10.1021%2Fjm049740z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasilyeva, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coley, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span> <span> </span><span class="NLM_article-title">Inside the Mind of a Medicinal Chemist: The Role of Human Bias in Compound Prioritization during Drug Discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e48476</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0048476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1371%2Fjournal.pone.0048476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23185259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVagsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e48476&issue=11&author=P.+S.+Kutchukianauthor=N.+Y.+Vasilyevaauthor=J.+Xuauthor=M.+K.+Lindvallauthor=M.+P.+Dillonauthor=M.+Glickauthor=J.+D.+Coleyauthor=N.+Brooijmans&title=Inside+the+Mind+of+a+Medicinal+Chemist%3A+The+Role+of+Human+Bias+in+Compound+Prioritization+during+Drug+Discovery&doi=10.1371%2Fjournal.pone.0048476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inside the mind of a medicinal chemist: the role of human bias in compound prioritization during drug discovery</span></div><div class="casAuthors">Kutchukian, Peter S.; Vasilyeva, Nadya Y.; Xu, Jordan; Lindvall, Mika K.; Dillon, Michael P.; Glick, Meir; Coley, John D.; Brooijmans, Natasja</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e48476</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Medicinal chemists' "intuition" is crit. for success in modern drug discovery.  Early in the discovery process, chemists select a subset of compds. for further research, often from many viable candidates.  These decisions det. the success of a discovery campaign, and ultimately what kind of drugs are developed and marketed to the public.  Surprisingly little is known about the cognitive aspects of chemists' decision-making when they prioritize compds.  We investigate (1) how and to what extent chemists simplify the problem of identifying promising compds., (2) whether chemists agree with each other about the criteria used for such decisions, and (3) how accurately chemists report the criteria they use for these decisions.  Chemists were surveyed and asked to select chem. fragments that they would be willing to develop into a lead compd. from a set of ∼4,000 available fragments.  Based on each chemist's selections, computational classifiers were built to model each chemist's selection strategy.  Results suggest that chemists greatly simplified the problem, typically using only 1-2 of many possible parameters when making their selections.  Although chemists tended to use the same parameters to select compds., differing value preferences for these parameters led to an overall lack of consensus in compd. selections.  Moreover, what little agreement there was among the chemists was largely in what fragments were undesirable.  Furthermore, chemists were often unaware of the parameters (such as compd. size) which were statistically significant in their selections, and overestimated the no. of parameters they employed.  A crit. evaluation of the problem space faced by medicinal chemists and cognitive models of categorization were esp. useful in understanding the low consensus between chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB2xkFnkLMFbVg90H21EOLACvtfcHk0liKC93XXi4Zcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVagsr3N&md5=a94653786832db485777a60e05dbb848</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0048476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0048476%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DVasilyeva%26aufirst%3DN.%2BY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLindvall%26aufirst%3DM.%2BK.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DColey%26aufirst%3DJ.%2BD.%26aulast%3DBrooijmans%26aufirst%3DN.%26atitle%3DInside%2520the%2520Mind%2520of%2520a%2520Medicinal%2520Chemist%253A%2520The%2520Role%2520of%2520Human%2520Bias%2520in%2520Compound%2520Prioritization%2520during%2520Drug%2520Discovery%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3De48476%26doi%3D10.1371%2Fjournal.pone.0048476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahue, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diratsouian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span> <span> </span><span class="NLM_article-title">Diversity & Tractability Revisited in Collaborative Small Molecule Phenotypic Screening Library Design</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">115192</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmc.2019.115192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31837897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSlurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=115192&issue=1&author=B.+R.+Lahueauthor=M.+Glickauthor=M.+Tudorauthor=S.+A.+Johnsonauthor=J.+Diratsouianauthor=M.+J.+Wildeyauthor=M.+Burtonauthor=R.+Mazzolaauthor=A.+M.+Wassermann&title=Diversity+%26+Tractability+Revisited+in+Collaborative+Small+Molecule+Phenotypic+Screening+Library+Design&doi=10.1016%2Fj.bmc.2019.115192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity & tractability revisited in collaborative small molecule phenotypic screening library design</span></div><div class="casAuthors">Lahue, Brian R.; Glick, Meir; Tudor, Matthew; Johnson, Scott Arne; Diratsouian, Janet; Wildey, Mary Jo; Burton, Marybeth; Mazzola, Robert; Wassermann, Anne Mai</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115192</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Identification of purposeful chem. matter on a broad range of drug targets is of high importance to the pharmaceutical industry.  However, disease-relevant but more complex hit-finding plans require flexibility regarding the subset of the compds. that we screen.  Herein we describe a strategy to design high-quality small mol. screening subsets of two different sizes to cope with a rapidly changing early discovery portfolio.  The approach taken balances chem. tractability, chem. diversity and biol. target coverage.  Furthermore, using surveys, we actively involved chemists within our company in the selection process of the diversity decks to ensure current medicinal chem. principles were incorporated.  The chemist surveys revealed that not all published PAINS substructure alerts are considered productive by the medicinal chem. community and in agreement with previously published results from other institutions, QED scores tracked quite well with chemists' notions of chem. attractiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn6BdDYp7at7Vg90H21EOLACvtfcHk0lgKyZRtn0_ncw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSlurrJ&md5=e5483bbc3a1011dcceaede9bb0611441</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115192%26sid%3Dliteratum%253Aachs%26aulast%3DLahue%26aufirst%3DB.%2BR.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DTudor%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DDiratsouian%26aufirst%3DJ.%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26aulast%3DBurton%26aufirst%3DM.%26aulast%3DMazzola%26aufirst%3DR.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26atitle%3DDiversity%2520%2526%2520Tractability%2520Revisited%2520in%2520Collaborative%2520Small%2520Molecule%2520Phenotypic%2520Screening%2520Library%2520Design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26issue%3D1%26spage%3D115192%26doi%3D10.1016%2Fj.bmc.2019.115192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scior, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Mayorga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuanalo-Contreras, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span> <span> </span><span class="NLM_article-title">Recognizing Pitfalls in Virtual Screening: A Critical Review</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1021/ci200528d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200528d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=867-881&issue=4&author=T.+Sciorauthor=A.+Benderauthor=G.+Tresadernauthor=J.+L.+Medina-Francoauthor=K.+Mart%C3%ADnez-Mayorgaauthor=T.+Langerauthor=K.+Cuanalo-Contrerasauthor=D.+K.+Agrafiotis&title=Recognizing+Pitfalls+in+Virtual+Screening%3A+A+Critical+Review&doi=10.1021%2Fci200528d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing Pitfalls in Virtual Screening: A Critical Review</span></div><div class="casAuthors">Scior, Thomas; Bender, Andreas; Tresadern, Gary; Medina-Franco, Jose L.; Martinez-Mayorga, Karina; Langer, Thierry; Cuanalo-Contreras, Karina; Agrafiotis, Dimitris K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aim of virtual screening (VS) is to identify bioactive compds. through computational means, by employing knowledge about the protein target (structure-based VS) or known bioactive ligands (ligand-based VS).  In VS, a large no. of mols. are ranked according to their likelihood to be bioactive compds., with the aim to enrich the top fraction of the resulting list (which can be tested in bioassays afterward).  At its core, VS attempts to improve the odds of identifying bioactive mols. by maximizing the true pos. rate, i.e., by ranking the truly active mols. as high as possible (and, correspondingly, the truly inactive ones as low as possible).  In choosing the right approach, the researcher is faced with many questions: where does the optimal balance between efficiency and accuracy lie when evaluating a particular algorithm; do some methods perform better than others and in what particular situations; and what do retrospective results tell us about the prospective utility of a particular method.  Given the multitude of settings, parameters, and data sets the practitioner can choose from, there are many pitfalls that lurk along the way which might render VS less efficient or downright useless.  This review attempts to catalog published and unpublished problems, shortcomings, failures, and tech. traps of VS methods with the aim to avoid pitfalls by making the user aware of them in the first place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGois-3Bq6mm-LVg90H21EOLACvtfcHk0lgKyZRtn0_ncw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D&md5=ad61b3289039c03844085a8b8de9e1e4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci200528d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200528d%26sid%3Dliteratum%253Aachs%26aulast%3DScior%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMart%25C3%25ADnez-Mayorga%26aufirst%3DK.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DCuanalo-Contreras%26aufirst%3DK.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26atitle%3DRecognizing%2520Pitfalls%2520in%2520Virtual%2520Screening%253A%2520A%2520Critical%2520Review%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D4%26spage%3D867%26epage%3D881%26doi%3D10.1021%2Fci200528d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pascale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span> <span> </span><span class="NLM_article-title">Biased Complement Diversity Selection for Effective Exploration of Chemical Space in Hit-Finding Campaigns</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVahtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1709-1714&issue=5&author=J.+M.+Jansenauthor=G.+De+Pascaleauthor=S.+Fongauthor=M.+Lindvallauthor=H.+E.+Moserauthor=K.+Pfisterauthor=B.+Warneauthor=C.+Wartchow&title=Biased+Complement+Diversity+Selection+for+Effective+Exploration+of+Chemical+Space+in+Hit-Finding+Campaigns&doi=10.1021%2Facs.jcim.9b00048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Complement Diversity Selection for Effective Exploration of Chemical Space in Hit-Finding Campaigns</span></div><div class="casAuthors">Jansen, Johanna M.; De Pascale, Gianfranco; Fong, Susan; Lindvall, Mika; Moser, Heinz E.; Pfister, Keith; Warne, Bob; Wartchow, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1709-1714</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The success of hit-finding campaigns relies on many factors, including the quality and diversity of the set of compds. that is selected for screening.  This paper presents a generalized workflow that guides compd. selections from large compd. archives with opportunities to bias the selections with available knowledge in order to improve hit quality while still effectively sampling the accessible chem. space.  An optional flag in the workflow supports an explicit complement design function where diversity selections complement a given core set of compds.  Results from three project applications as well as a literature case study exemplify the effectiveness of the approach, which is available as a KNIME workflow named Biased Complement Diversity (BCD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQD_EVOq0WLVg90H21EOLACvtfcHk0li1L-Ovshgr5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVahtbc%253D&md5=1ff4e953be15cda3259a7339a5c09983</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00048%26sid%3Dliteratum%253Aachs%26aulast%3DJansen%26aufirst%3DJ.%2BM.%26aulast%3DDe%2BPascale%26aufirst%3DG.%26aulast%3DFong%26aufirst%3DS.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DPfister%26aufirst%3DK.%26aulast%3DWarne%26aufirst%3DB.%26aulast%3DWartchow%26aufirst%3DC.%26atitle%3DBiased%2520Complement%2520Diversity%2520Selection%2520for%2520Effective%2520Exploration%2520of%2520Chemical%2520Space%2520in%2520Hit-Finding%2520Campaigns%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D5%26spage%3D1709%26epage%3D1714%26doi%3D10.1021%2Facs.jcim.9b00048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">10742</span>– <span class="NLM_lpage">10772</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00523</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00523" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvFWqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10742-10772&issue=19&author=M.+D.+Lloyd&title=High-Throughput+Screening+for+the+Discovery+of+Enzyme+Inhibitors&doi=10.1021%2Facs.jmedchem.0c00523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Screening for the Discovery of Enzyme Inhibitors</span></div><div class="casAuthors">Lloyd, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10742-10772</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes are common targets in high-throughput screening and related campaigns.  An anal. of papers published between 1990 and 2018 showed that kinases were the most common enzymes investigated, fluorescence-based assays were the most common readout method, and cancer and bacterial infections were the most common therapeutic areas.  High-throughput screening and fragment-screening campaigns published between 2017 and 2019 were analyzed in more depth, giving 75 examples of hit to lead development.  Kinases, phosphatases, proteases, and peptidases were the most common targets, fluorescent assays were the most commonly used, and a wide variety of structural features were obsd. within the derived drugs.  Hit frequency was largely independent of library size and pos. correlated with Z' value for the assay.  Binding of metal ions to library compds. and substrates is an underappreciated source of false-pos. results and unreproducible behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc-U48b3MdtrVg90H21EOLACvtfcHk0li1L-Ovshgr5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvFWqsrg%253D&md5=3a7664f55f018d8a434c6fd2bc3721c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26atitle%3DHigh-Throughput%2520Screening%2520for%2520the%2520Discovery%2520of%2520Enzyme%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772%26doi%3D10.1021%2Facs.jmedchem.0c00523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoepfner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Goff, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltier, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grippo, M. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prindle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krastel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobos-Correa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Dark Chemical Matter as a Promising Starting Point for Drug Lead Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">958</span>– <span class="NLM_lpage">966</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.1936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26479441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yjtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=958-966&issue=12&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=D.+Hoepfnerauthor=G.+Le+Goffauthor=F.+J+Kingauthor=C.+Studerauthor=J.+M+Peltierauthor=M.+L+Grippoauthor=V.+Prindleauthor=J.+Taoauthor=A.+Schuffenhauerauthor=I.+M+Wallaceauthor=S.+Chenauthor=P.+Krastelauthor=A.+Cobos-Correaauthor=C.+N+Parkerauthor=J.+W+Daviesauthor=M.+Glick&title=Dark+Chemical+Matter+as+a+Promising+Starting+Point+for+Drug+Lead+Discovery&doi=10.1038%2Fnchembio.1936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Dark chemical matter as a promising starting point for drug lead discovery</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Hoepfner, Dominic; Le Goff, Gaelle; King, Frederick J.; Studer, Christian; Peltier, John M.; Grippo, Melissa L.; Prindle, Vivian; Tao, Jianshi; Schuffenhauer, Ansgar; Wallace, Iain M.; Chen, Shanni; Krastel, Philipp; Cobos-Correa, Amanda; Parker, Christian N.; Davies, John W.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">958-966</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is an integral part of early drug discovery.  Herein, the authors focused on those small mols. in a screening collection that have never shown biol. activity despite having been exhaustively tested in HTS assays.  These compds. are referred to as 'dark chem. matter' (DCM).  The authors quantified DCM, validated it in quality control expts., described its physicochem. properties and mapped it into chem. space.  Through anal. of prospective reporter-gene assay, gene expression and yeast chemogenomics expts., the authors evaluated the potential of DCM to show biol. activity in future screens.  The authors demonstrated that, despite the apparent lack of activity, occasionally these compds. can result in potent hits with unique activity and clean safety profiles, which makes them valuable starting points for lead optimization efforts.  Among the identified DCM hits was a new antifungal chemotype with strong activity against the pathogen Cryptococcus neoformans but little activity at targets relevant to human safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp21Fvx08O4r7Vg90H21EOLACvtfcHk0li1L-Ovshgr5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yjtrnM&md5=fdd233cf5a177504afb3acc37600cf88</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1936%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DHoepfner%26aufirst%3DD.%26aulast%3DLe%2BGoff%26aufirst%3DG.%26aulast%3DKing%26aufirst%3DF.%2BJ%26aulast%3DStuder%26aufirst%3DC.%26aulast%3DPeltier%26aufirst%3DJ.%2BM%26aulast%3DGrippo%26aufirst%3DM.%2BL%26aulast%3DPrindle%26aufirst%3DV.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DWallace%26aufirst%3DI.%2BM%26aulast%3DChen%26aufirst%3DS.%26aulast%3DKrastel%26aufirst%3DP.%26aulast%3DCobos-Correa%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DC.%2BN%26aulast%3DDavies%26aufirst%3DJ.%2BW%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DDark%2520Chemical%2520Matter%2520as%2520a%2520Promising%2520Starting%2520Point%2520for%2520Drug%2520Lead%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D12%26spage%3D958%26epage%3D966%26doi%3D10.1038%2Fnchembio.1936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Critical Review of the Role of HTS in Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7–8</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2006.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16580969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A280%3ADC%252BD287otlWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=277-279&issue=7%E2%80%938&author=R.+Macarron&title=Critical+Review+of+the+Role+of+HTS+in+Drug+Discovery&doi=10.1016%2Fj.drudis.2006.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Critical review of the role of HTS in drug discovery</span></div><div class="casAuthors">Macarron Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">277-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">'A wise use of lead discovery tactics will distinguish successful drug discovery engines.'</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd_nOrub-6MD69zv1FsJFMfW6udTcc2eb2s3ivQwt_wLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD287otlWjtQ%253D%253D&md5=ac1107aee5902bd961080f0a47df03c2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DCritical%2520Review%2520of%2520the%2520Role%2520of%2520HTS%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D7%25E2%2580%25938%26spage%3D277%26epage%3D279%26doi%3D10.1016%2Fj.drudis.2006.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engeloch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklin, P.</span></span> <span> </span><span class="NLM_article-title">The Novartis Compound Archive - From Concept to Reality</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.2174/1386207054867328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F1386207054867328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16178810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2005&pages=513-519&issue=6&author=U.+Schopferauthor=C.+Engelochauthor=J.+Stanekauthor=M.+Girodauthor=A.+Schuffenhauerauthor=E.+Jacobyauthor=P.+Acklin&title=The+Novartis+Compound+Archive+-+From+Concept+to+Reality&doi=10.2174%2F1386207054867328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Novartis Compound Archive - from concept to reality</span></div><div class="casAuthors">Schopfer, U.; Engeloch, C.; Stanek, J.; Girod, M.; Schuffenhauer, A.; Jacoby, E.; Acklin, P.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">513-519</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">As HTS technologies come of age, pharmaceutical companies are focusing increasingly on the quality of their screening collections.  Storage conditions and their influence on compd. stability and soly. are debated intensely.  At Novartis, a strategy was developed that is different to most other companies: (1) compds. unsuitable for storage in soln. are excluded by computational methods; (2) compds. are stored at 4°C/20% relative humidity in a DMSO/water mixt. to avoid freeze-thaw cycles and water uptake and to allow rapid plate replication; (3) resolubilization of compds. at regular intervals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr91vmOkqjoKbVg90H21EOLACvtfcHk0ljB7umt_d8E1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOnu74%253D&md5=81ccc4ee90ec09e655322c6bfb3738b9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F1386207054867328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207054867328%26sid%3Dliteratum%253Aachs%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DEngeloch%26aufirst%3DC.%26aulast%3DStanek%26aufirst%3DJ.%26aulast%3DGirod%26aufirst%3DM.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DAcklin%26aufirst%3DP.%26atitle%3DThe%2520Novartis%2520Compound%2520Archive%2520-%2520From%2520Concept%2520to%2520Reality%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2005%26volume%3D8%26issue%3D6%26spage%3D513%26epage%3D519%26doi%3D10.2174%2F1386207054867328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sukuru, S. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Plate-Based Diversity Selection Based on Empirical HTS Data to Enhance the Number of Hits and Their Chemical Diversity</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1177/1087057109335678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057109335678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=19531667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVaksr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=690-699&issue=6&author=S.+C.+K.+Sukuruauthor=J.+L.+Jenkinsauthor=R.+E.+J.+Beckwithauthor=J.+Scheiberauthor=A.+Benderauthor=D.+Mikhailovauthor=J.+W.+Daviesauthor=M.+Glick&title=Plate-Based+Diversity+Selection+Based+on+Empirical+HTS+Data+to+Enhance+the+Number+of+Hits+and+Their+Chemical+Diversity&doi=10.1177%2F1087057109335678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Plate-based diversity selection based on empirical HTS data to enhance the number of hits and their chemical diversity</span></div><div class="casAuthors">Sukuru, Sai Chetan K.; Jenkins, Jeremy L.; Beckwith, Rohan E. J.; Scheiber, Josef; Bender, Andreas; Mikhailov, Dmitri; Davies, John W.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">690-699</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Typically, screening collections of pharmaceutical companies contain more than a million compds. today.  However, for certain high-throughput screening (HTS) campaigns, constraints posed by the assay throughput and/or the reagent costs make it impractical to screen the entire deck.  Therefore, it is desirable to effectively screen subsets of the collection based on a hypothesis or a diversity selection.  How to select compd. subsets is a subject of ongoing debate.  The authors present an approach based on extended connectivity fingerprints to carry out diversity selection on a per plate basis (instead of a per compd. basis).  HTS data from 35 Novartis screens spanning 5 target classes were investigated to assess the performance of this approach.  The anal. shows that selecting a fingerprint-diverse subset of 250K compds., representing 20% of the screening deck, would have achieved significantly higher hit rates for 86% of the screens.  This measure also outperforms the Murcko scaffold-based plate selection described previously, where only 49% of the screens showed similar improvements.  Strikingly, the 2-fold improvement in av. hit rates obsd. for 3 of 5 target classes in the data set indicates a target bias of the plate (and thus compd.) selection method.  Even though the diverse subset selection lacks any target hypothesis, its application shows significantly better results for some targets-namely, G-protein-coupled receptors, proteases, and protein-protein interactions-but not for kinase and pathway screens.  The synthetic origin of the compds. in the diverse subset appears to influence the screening hit rates.  Natural products were the most diverse compd. class, with significantly higher hit rates compared to the compds. from the traditional synthetic and combinatorial libraries.  These results offer empirical guidelines for plate-based diversity selection to enhance hit rates, based on target class and the library type being screened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn0GZfRsZ097Vg90H21EOLACvtfcHk0ljB7umt_d8E1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVaksr%252FJ&md5=fce87f6137195c5b4a6e1e9823adaa3d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1177%2F1087057109335678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109335678%26sid%3Dliteratum%253Aachs%26aulast%3DSukuru%26aufirst%3DS.%2BC.%2BK.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DPlate-Based%2520Diversity%2520Selection%2520Based%2520on%2520Empirical%2520HTS%2520Data%2520to%2520Enhance%2520the%2520Number%2520of%2520Hits%2520and%2520Their%2520Chemical%2520Diversity%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26issue%3D6%26spage%3D690%26epage%3D699%26doi%3D10.1177%2F1087057109335678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Lessons Learned from the Design of Chemical Space Networks and Opportunities for New Applications</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9906-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1007%2Fs10822-016-9906-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26945865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=191-208&issue=3&author=M.+Vogtauthor=D.+Stumpfeauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Lessons+Learned+from+the+Design+of+Chemical+Space+Networks+and+Opportunities+for+New+Applications&doi=10.1007%2Fs10822-016-9906-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the design of chemical space networks and opportunities for new applications</span></div><div class="casAuthors">Vogt, Martin; Stumpfe, Dagmar; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-208</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The concept of chem. space is of fundamental relevance in chem. informatics and computer-aided drug discovery.  In a series of articles published in the Journal of Computer-Aided Mol. Design, principles of chem. space design were evaluated, mol. networks proposed as an alternative to conventional coordinate-based chem. ref. spaces, and different types of chem. space networks (CSNs) constructed and analyzed.  Central to the generation of CSNs was the way in which mol. similarity relationships were assessed and a primary focal point was the network-based representation of biol. relevant chem. space.  The design and comparison of CSNs based upon alternative similarity measures can be viewed as an evolutionary path with interesting lessons learned along the way.  CSN design has matured to the point that such chem. space representations can be used in practice.  In this contribution, highlights from the sequence of CSN design efforts are discussed in context, providing a perspective for future practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5JT9f_RfpbVg90H21EOLACvtfcHk0lgvJgZYI8tVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D&md5=e5607c1fe6b9a5e74a55e8e97464bb22</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9906-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9906-3%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLessons%2520Learned%2520from%2520the%2520Design%2520of%2520Chemical%2520Space%2520Networks%2520and%2520Opportunities%2520for%2520New%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26issue%3D3%26spage%3D191%26epage%3D208%26doi%3D10.1007%2Fs10822-016-9906-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Space and Biology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span> (<span class="NLM_issue">7019</span>),  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1038/nature03192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnature03192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=15602547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=824-828&issue=7019&author=C.+M.+Dobson&title=Chemical+Space+and+Biology&doi=10.1038%2Fnature03192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space and biology</span></div><div class="casAuthors">Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. space - which encompasses all possible small org. mols., including those present in biol. systems - is vast.  So vast, in fact, that so far only a tiny fraction of it has been explored.  Nevertheless, these explorations have greatly enhanced our understanding of biol., and have led to the development of many of today's drugs.  The discovery of new bioactive mols., facilitated by a deeper understanding of the nature of the regions of chem. space that are relevant to biol., will advance our knowledge of biol. processes and lead to new strategies to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNmRTVWoSDrVg90H21EOLACvtfcHk0lgvJgZYI8tVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN&md5=b951fff4aa4877682756fb1831bff09d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature03192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03192%26sid%3Dliteratum%253Aachs%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520Space%2520and%2520Biology%26jtitle%3DNature%26date%3D2004%26volume%3D432%26issue%3D7019%26spage%3D824%26epage%3D828%26doi%3D10.1038%2Fnature03192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D.</span></span> <span> </span><span class="NLM_article-title">(Research Papers) Design of a Compound Screening Collection for Use in High Throughput Screening</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.2174/138620704772884832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&issue=1&author=G.+Harperauthor=S.+Pickettauthor=D.+Green&title=%28Research+Papers%29+Design+of+a+Compound+Screening+Collection+for+Use+in+High+Throughput+Screening&doi=10.2174%2F138620704772884832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0lgvJgZYI8tVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DD.%26atitle%3D%2528Research%2520Papers%2529%2520Design%2520of%2520a%2520Compound%2520Screening%2520Collection%2520for%2520Use%2520in%2520High%2520Throughput%2520Screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26issue%3D1%26spage%3D63%26epage%3D70%26doi%3D10.2174%2F138620704772884832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koutsoukas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paricharak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galloway, W. R. J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span> <span> </span><span class="NLM_article-title">How Diverse Are Diversity Assessment Methods? A Comparative Analysis and Benchmarking of Molecular Descriptor Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1021/ci400469u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400469u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGnt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=230-242&issue=1&author=A.+Koutsoukasauthor=S.+Paricharakauthor=W.+R.+J.+D.+Gallowayauthor=D.+R.+Springauthor=A.+P.+IJzermanauthor=R.+C.+Glenauthor=D.+Marcusauthor=A.+Bender&title=How+Diverse+Are+Diversity+Assessment+Methods%3F+A+Comparative+Analysis+and+Benchmarking+of+Molecular+Descriptor+Space&doi=10.1021%2Fci400469u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">How Diverse Are Diversity Assessment Methods? A Comparative Analysis and Benchmarking of Molecular Descriptor Space</span></div><div class="casAuthors">Koutsoukas, Alexios; Paricharak, Shardul; Galloway, Warren R. J. D.; Spring, David R.; Ijzerman, Adriaan P.; Glen, Robert C.; Marcus, David; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">230-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. diversity is a widely applied approach to select structurally diverse subsets of mols., often with the objective of maximizing the no. of hits in biol. screening.  While many methods exist in the area, few systematic comparisons using current descriptors in particular with the objective of assessing diversity in bioactivity space have been published, and this shortage is what the current study is aiming to address.  13 Widely used mol. descriptors were compared, including fingerprint-based descriptors (ECFP4, FCFP4, MACCS keys), pharmacophore-based descriptors (TAT, TAD, TGT, TGD, GpiDAPH3), shape-based descriptors (rapid overlay of chem. structures (ROCS) and principal moments of inertia (PMI)), a connectivity-matrix-based descriptor (BCUT), physicochem.-property-based descriptors (prop2D), and a more recently introduced mol. descriptor type (namely, "Bayes Affinity Fingerprints").  The authors assessed both the similar behavior of the descriptors in assessing the diversity of chem. libraries, and their ability to select compds. from libraries that are diverse in bioactivity space, which is a property of much practical relevance in screening library design.  This is particularly evident, given that many future targets to be screened are not known in advance, but that the library should still maximize the likelihood of contg. bioactive matter also for future screening campaigns.  Overall, the authors' results showed that descriptors based on atom topol. (i.e., fingerprint-based descriptors and pharmacophore-based descriptors) correlate well in rank-ordering compds., both within and between descriptor types.  However, shape-based descriptors such as ROCS and PMI showed weak correlation with the other descriptors used demonstrating significantly different behavior.  The authors then applied eight of the mol. descriptors compared in this study to sample a diverse subset of sample compds. (4%) from an initial population of 2587 compds., covering the 25 largest human activity classes from ChEMBL and measured the coverage of activity classes by the subsets.  Here, "Bayes Affinity Fingerprints" achieved an av. coverage of 92% of activity classes.  Using the descriptors ECFP4, GpiDAPH3, TGT, and random sampling, 91%, 84%, 84%, and 84% of the activity classes were represented in the selected compds., resp., followed by BCUT, prop2D, MACCS, and PMI (in order of decreasing performance).  In addn., the authors were able to show that there is no visible correlation between compd. diversity in PMI space and in bioactivity space, despite frequent use of PMI plots to this end.  To summarize, the authors assessed which descriptors select compds. with high coverage of bioactivity space, and can hence be used for diverse compd. selection for biol. screening.  In cases where multiple descriptors are to be used for diversity selection, this work describes which descriptors behave complementarily and can hence be used jointly to focus on different aspects of diversity in chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojohapFOhhyLVg90H21EOLACvtfcHk0lgaAvuJN_khLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGnt7bL&md5=f960e2e1ef89966171400e47af2f6d21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci400469u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400469u%26sid%3Dliteratum%253Aachs%26aulast%3DKoutsoukas%26aufirst%3DA.%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DGalloway%26aufirst%3DW.%2BR.%2BJ.%2BD.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DMarcus%26aufirst%3DD.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DHow%2520Diverse%2520Are%2520Diversity%2520Assessment%2520Methods%253F%2520A%2520Comparative%2520Analysis%2520and%2520Benchmarking%2520of%2520Molecular%2520Descriptor%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D1%26spage%3D230%26epage%3D242%26doi%3D10.1021%2Fci400469u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span> <span> </span><span class="NLM_article-title">Chemical Diversity and Biological Activity</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2006.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.ddtec.2006.12.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=387-395&issue=4&author=A.+Schuffenhauerauthor=N.+Brown&title=Chemical+Diversity+and+Biological+Activity&doi=10.1016%2Fj.ddtec.2006.12.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2006.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2006.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DChemical%2520Diversity%2520and%2520Biological%2520Activity%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2006%26volume%3D3%26issue%3D4%26spage%3D387%26epage%3D395%26doi%3D10.1016%2Fj.ddtec.2006.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Probing the Probes: Fitness Factors For Small Molecule Tools</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2010.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20609406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=561-577&issue=6&author=P.+Workmanauthor=I.+Collins&title=Probing+the+Probes%3A+Fitness+Factors+For+Small+Molecule+Tools&doi=10.1016%2Fj.chembiol.2010.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Probes: Fitness Factors For Small Molecule Tools</span></div><div class="casAuthors">Workman, Paul; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">561-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. probes for interrogating biol. processes are of considerable current interest.  Cell permeable small mol. tools have a major role in facilitating the functional annotation of the human genome, understanding both physiol. and pathol. processes, and validating new mol. targets.  To be valuable, chem. tools must satisfy necessary criteria and recent publications have suggested objective guidelines for what makes a useful chem. probe.  Although recognizing that such guidelines may be valuable, we caution against overly restrictive rules that may stifle innovation in favor of a "fit-for-purpose" approach.  Reviewing the literature and providing examples from the cancer field, we recommend a series of "fitness factors" to be considered when assessing chem. probes.  We hope this will encourage innovative chem. biol. research while minimizing the generation of poor quality and misleading biol. data, thus increasing understanding of the particular biol. area, to the benefit of basic research and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQtty9oZD17rVg90H21EOLACvtfcHk0lgaAvuJN_khLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D&md5=16be01da2339173aae8207030bf83c12</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DProbing%2520the%2520Probes%253A%2520Fitness%2520Factors%2520For%2520Small%2520Molecule%2520Tools%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26issue%3D6%26spage%3D561%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2010.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The Promise and Peril of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+Promise+and+Peril+of+Chemical+Probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lh3rRY0DaWoBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520Promise%2520and%2520Peril%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span> <span> </span><span class="NLM_article-title">Biological Assay Challenges from Compound Solubility: Strategies for Bioassay Optimization</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9–10</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2006.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16635808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvVGns7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=446-451&issue=9%E2%80%9310&author=L.+Diauthor=E.+H.+Kerns&title=Biological+Assay+Challenges+from+Compound+Solubility%3A+Strategies+for+Bioassay+Optimization&doi=10.1016%2Fj.drudis.2006.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Biological assay challenges from compounds solubility: strategies for bioassay optimization</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9 & 10</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Compd. soly. in buffers and DMSO (DMSO) has emerged as an important issue.  Many discovery compds. have low soly. but are potentially valuable as leads.  Unfortunately, low soly. affects bioassays by causing underestimated activity, reduced HTS-hit rates, variable data, inaccurate SAR, discrepancies between enzyme and cell assays and inaccurate in vitro ADME-Tox testing.  Strategies for optimizing bioassays include: considering soly. in HTS-library design; early screening for soly.; improving storage and handling of DMSO stocks; optimizing diln. protocols; and ensuring that low-soly. compds. are fully solubilized in bioassays.  These approaches allow for adequate assessments of valuable pharmacophores for which soly. can be chem. optimized at a later date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWFI1UiXIDLVg90H21EOLACvtfcHk0lh3rRY0DaWoBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvVGns7g%253D&md5=1814b6b9d1d8cb77540c42c37509a736</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DBiological%2520Assay%2520Challenges%2520from%2520Compound%2520Solubility%253A%2520Strategies%2520for%2520Bioassay%2520Optimization%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D9%25E2%2580%259310%26spage%3D446%26epage%3D451%26doi%3D10.1016%2Fj.drudis.2006.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGovern, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helfand, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A Specific Mechanism of Nonspecific Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">4265</span>– <span class="NLM_lpage">4272</span>, <span class="refDoi"> DOI: 10.1021/jm030266r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030266r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4265-4272&issue=20&author=S.+L.+McGovernauthor=B.+T.+Helfandauthor=B.+Fengauthor=B.+K.+Shoichet&title=A+Specific+Mechanism+of+Nonspecific+Inhibition&doi=10.1021%2Fjm030266r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Specific Mechanism of Nonspecific Inhibition</span></div><div class="casAuthors">McGovern, Susan L.; Helfand, Brian T.; Feng, Brian; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4265-4272</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous small mols. plague screening libraries and hit lists.  Previous work has found that several nonspecific compds. form submicrometer aggregates, and it has been suggested that this aggregate species is responsible for the inhibition of many different enzymes.  It is not understood how aggregates inhibit their targets.  To address this question, biophys., kinetic, and microscopy methods were used to study the interaction of promiscuous, aggregate-forming inhibitors with model proteins.  By use of centrifugation and gel electrophoresis, aggregates and protein were found to directly interact.  This is consistent with a subsequent observation from confocal fluorescence microscopy that aggregates conc. green fluorescent protein.  β-Lactamase mutants with increased or decreased thermodn. stability relative to wild-type enzyme were equally inhibited by an aggregate-forming compd., suggesting that denaturation by unfolding was not the primary mechanism of interaction.  Instead, visualization by electron microscopy revealed that enzyme assocs. with the surface of inhibitor aggregates.  This assocn. could be reversed or prevented by the addn. of Triton X-100.  These observations suggest that the aggregates formed by promiscuous compds. reversibly sequester enzyme, resulting in apparent inhibition.  They also suggest a simple method to identify or reverse the action of aggregate-based inhibitors, which appear to be widespread.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppdLVG_GeLZbVg90H21EOLACvtfcHk0ljGFdcURlU8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D&md5=cd0fd6bf0a64ceb7d7219dae2406dd21</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm030266r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030266r%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DHelfand%26aufirst%3DB.%2BT.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520Specific%2520Mechanism%2520of%2520Nonspecific%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26issue%3D20%26spage%3D4265%26epage%3D4272%26doi%3D10.1021%2Fjm030266r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span> <span> </span><span class="NLM_article-title">High Throughput Solubility Measurement in Drug Discovery and Development</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.addr.2007.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=17604872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=546-567&issue=7&author=J.+Alsenzauthor=M.+Kansy&title=High+Throughput+Solubility+Measurement+in+Drug+Discovery+and+Development&doi=10.1016%2Fj.addr.2007.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput solubility measurement in drug discovery and development</span></div><div class="casAuthors">Alsenz, Jochem; Kansy, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">546-567</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Measurement of drug soly. in various solvents is one of the key elements of compd. characterization during the whole discovery and development process.  This review summarizes current exptl. approaches and addresses recent advances in the exptl. methods used to det. drug soly. in drug discovery and early development.  This paper focuses on high throughput methods designed to det. kinetic and thermodn. (equil.) soly. but traditional methods are also presented.  The focus, positioning, exptl. setup, pros and cons, and limitations of individual assays are discussed and differences in soly. studies in discovery and development environments are highlighted.  Finally, future needs and trends in soly. assay development designed to overcome current bottlenecks and trade-offs between speed and quality/quantity of measurements are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsvOlLd-yoxLVg90H21EOLACvtfcHk0ljGFdcURlU8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D&md5=8d55ead3ea0c13fa49b032db725842b0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3DKansy%26aufirst%3DM.%26atitle%3DHigh%2520Throughput%2520Solubility%2520Measurement%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2007%26volume%3D59%26issue%3D7%26spage%3D546%26epage%3D567%26doi%3D10.1016%2Fj.addr.2007.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Exploratory Analysis of Kinetic Solubility Measurements of a Small Molecule Library</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4127</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmc.2011.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21640593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4127-4134&issue=13&author=R.+Guhaauthor=T.+S.+Dexheimerauthor=A.+N.+Kestranekauthor=A.+Jadhavauthor=A.+M.+Chervenakauthor=M.+G.+Fordauthor=A.+Simeonovauthor=G.+P.+Rothauthor=C.+J.+Thomas&title=Exploratory+Analysis+of+Kinetic+Solubility+Measurements+of+a+Small+Molecule+Library&doi=10.1016%2Fj.bmc.2011.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploratory analysis of kinetic solubility measurements of a small molecule library</span></div><div class="casAuthors">Guha, Rajarshi; Dexheimer, Thomas S.; Kestranek, Aimee N.; Jadhav, Ajit; Chervenak, Andrew M.; Ford, Michael G.; Simeonov, Anton; Roth, Gregory P.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4127-4134</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Kinetic soly. measurements using prototypical assay buffer conditions are presented for a ∼58,000 member library of small mols.  Analyses of the data based upon phys. and calcd. properties of each individual mol. were performed and resulting trends were considered in the context of commonly held opinions of how physicochem. properties influence aq. soly.  The data was further analyzed using a decision tree model for soly. prediction and via a multi-dimensional assessment of physicochem. relationships to soly. in the context of specific rule-breakers relative to common dogma.  The role of soly. as a determinant of assay outcome is also considered based upon each compd.'s cross-assay activity score for a collection of publicly available screening results.  Further, the role of soly. as a governing factor for colloidal aggregation formation within a specified assay setting is examd. and considered as a possible cause of a high cross-assay activity score.  The results of this soly. profile should aid chemists during library design and optimization efforts and represent a useful training set for computational soly. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU9biLJPDeaLVg90H21EOLACvtfcHk0ljGFdcURlU8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D&md5=1f5b67268444cd3911665ea395bbf72f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DKestranek%26aufirst%3DA.%2BN.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%2BM.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DExploratory%2520Analysis%2520of%2520Kinetic%2520Solubility%2520Measurements%2520of%2520a%2520Small%2520Molecule%2520Library%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26issue%3D13%26spage%3D4127%26epage%3D4134%26doi%3D10.1016%2Fj.bmc.2011.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10594</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10520-10594&issue=18&author=X.+Yangauthor=Y.+Wangauthor=R.+Byrneauthor=G.+Schneiderauthor=S.+Yang&title=Concepts+of+Artificial+Intelligence+for+Computer-Assisted+Drug+Discovery&doi=10.1021%2Facs.chemrev.8b00728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span></div><div class="casAuthors">Yang, Xin; Wang, Yifei; Byrne, Ryan; Schneider, Gisbert; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10520-10594</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Artificial intelligence (AI), and, in particular, deep learning as a subcategory of AI, provides opportunities for the discovery and development of innovative drugs.  Various machine learning approaches have recently (re)emerged, some of which may be considered instances of domain-specific AI which have been successfully employed for drug discovery and design.  This review provides a comprehensive portrayal of these machine learning techniques and of their applications in medicinal chem.  After introducing the basic principles, alongside some application notes, of the various machine learning algorithms, the current state-of-the art of AI-assisted pharmaceutical discovery is discussed, including applications in structure- and ligand-based virtual screening, de novo drug design, physicochem. and pharmacokinetic property prediction, drug repurposing, and related aspects.  Finally, several challenges and limitations of the current methods are summarized, with a view to potential future directions for AI-assisted drug discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUfBzNv587rVg90H21EOLACvtfcHk0ljGFdcURlU8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN&md5=6e9816e30251cd4f15c831e2dd20f100</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00728%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DConcepts%2520of%2520Artificial%2520Intelligence%2520for%2520Computer-Assisted%2520Drug%2520Discovery%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26issue%3D18%26spage%3D10520%26epage%3D10594%26doi%3D10.1021%2Facs.chemrev.8b00728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Breemen, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Caco-2 Cell Permeability Assays to Measure Drug Absorption</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1517/17425255.1.2.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1517%2F17425255.1.2.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16922635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=175-185&issue=2&author=R.+B.+Van+Breemenauthor=Y.+Li&title=Caco-2+Cell+Permeability+Assays+to+Measure+Drug+Absorption&doi=10.1517%2F17425255.1.2.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Caco-2 cell permeability assays to measure drug absorption</span></div><div class="casAuthors">van Breemen, Richard B.; Li, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-185</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Caco-2 cells are a human colon epithelial cancer cell line used as a model of human intestinal absorption of drugs and other compds.  When cultured as a monolayer, Caco-2 cells differentiate to form tight junctions between cells to serve as a model of paracellular movement of compds. across the mono-layer.  In addn., Caco-2 cells express transporter proteins, efflux proteins, and Phase II conjugation enzymes to model a variety of transcellular pathways as well as metabolic transformation of test substances.  In many respects, the Caco-2 cell monolayer mimics the human intestinal epithelium.  One of the functional differences between normal cells and Caco-2 cells is the lack of expression of the cytochrome P 450 isoenzymes and in particular, CYP3A4, which is normally expressed at high levels in the intestine.  However, Caco-2 cells may be induced to express higher levels of CYP3A4 by treatment with vitamin D3.  Caco-2 cell monolayers are usually cultured on semipermeable plastic supports that may be fitted into the wells of multi-well culture plates.  Test compds. are then added to either the apical or basolateral sides of the monolayer.  After incubation for various lengths of time, aliquots of the buffer in opposite chambers are removed for the detn. of the concn. of test compds. and the computation of the rates of permeability for each compd. (called the apparent permeability coeffs.).  Although radiolabeled compds. were used in the original Caco-2 cells monolayer assays, radio-labeled compds. have been replaced in most labs. by the use of liq. chromatog.-mass spectrometry (LC-MS) and LC-tandem mass spectrometry (LC-MS-MS).  Mass spectrometry not only eliminates the need for radiolabeled compds., but permits the simultaneous measurement of multiple compds.  The measurement of multiple compds. per assay reduces the no. of incubations that need to be carried out, thereby increasing the throughput of the expts.  Furthermore, LC-MS and LC-MS-MS add another dimension to Caco-2 assays by facilitating the investigation of the metab. of compds. by Caco-2 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcV-AZvy78ILVg90H21EOLACvtfcHk0lhXI-BgYv-eXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK&md5=7bd0b875ffa9c840f904bab6513b7e3f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1517%2F17425255.1.2.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.1.2.175%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BBreemen%26aufirst%3DR.%2BB.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DCaco-2%2520Cell%2520Permeability%2520Assays%2520to%2520Measure%2520Drug%2520Absorption%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2005%26volume%3D1%26issue%3D2%26spage%3D175%26epage%3D185%26doi%3D10.1517%2F17425255.1.2.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&issue=7&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lhXI-BgYv-eXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D7%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitney-Pickett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhard, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federico, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyner, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempshall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-kattan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liston, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span> <span> </span><span class="NLM_article-title">Development of a New Permeability Assay Using Low-Efflux MDCKII Cells</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4974</span>– <span class="NLM_lpage">4985</span>, <span class="refDoi"> DOI: 10.1002/jps.22674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fjps.22674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21766308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovFGltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=4974-4985&issue=11&author=L.+Diauthor=C.+Whitney-Pickettauthor=J.+P.+Umlandauthor=H.+Zhangauthor=X.+Zhangauthor=D.+F.+Gebhardauthor=Y.+Laiauthor=J.+J.+Federicoauthor=R.+E.+Davidsonauthor=R.+Smithauthor=E.+L.+Reynerauthor=C.+Leeauthor=B.+Fengauthor=C.+Rotterauthor=M.+V.+Varmaauthor=S.+Kempshallauthor=K.+Fennerauthor=A.+F.+El-kattanauthor=T.+E.+Listonauthor=M.+D.+Troutman&title=Development+of+a+New+Permeability+Assay+Using+Low-Efflux+MDCKII+Cells&doi=10.1002%2Fjps.22674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a new permeability assay using low-efflux MDCKII cells</span></div><div class="casAuthors">Di, Li; Whitney-Pickett, Carrie; Umland, John P.; Zhang, Hui; Zhang, Xun; Gebhard, David F.; Lai, Yurong; Federico, James J.; Davidson, Ralph E.; Smith, Russ; Reyner, Eric L.; Lee, Caroline; Feng, Bo; Rotter, Charles; Varma, Manthena V.; Kempshall, Sarah; Fenner, Katherine; El-kattan, Ayman F.; Liston, Theodore E.; Troutman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4974-4985</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Permeability is an important property of drug candidates.  The Madin-Darby canine kidney cell line (MDCK) permeability assay is widely used and the primary concern of using MDCK cells is the presence of endogenous transporters of nonhuman origin.  The canine P-glycoprotein (Pgp) can interfere with permeability and transporter studies, leading to less reliable data.  A new cell line, MDCKII-LE (low efflux), has been developed by selecting a subpopulation of low-efflux cells from MDCKII-WT using an iterative fluorescence-activated cell sorting technique with calcein-AM as a Pgp and efflux substrate.  MDCKII-LE cells are a subpopulation of MDCKII cells with over 200-fold lower canine Pgp mRNA level and fivefold lower protein level than MDCKII-WT.  MDCKII-LE cells showed less functional efflux activity than MDCKII-WT based on efflux ratios.  Notably, MDCKII-MDR1 showed about 1.5-fold decreased expression of endogenous canine Pgp, suggesting that using the net flux ratio might not completely cancel out the background endogenous transporter activities.  MDCKII-LE cells offer clear advantages over the MDCKII-WT by providing less efflux transporter background signals and minimizing interference from canine Pgp.  The MDCKII-LE apparent permeability values well differentiates compds. from high to medium/low human intestinal absorption and can be used for Biopharmaceutical Classification System.  The MDCKII-LE permeability assay (4-in-1 cassette dosing) is high throughput with good precision, reproducibility, robustness, and cost-effective. © 2011 Wiley-Liss, Inc. and the American Pharmacists Assocn. J Pharm Sci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVtRmYnbF17Vg90H21EOLACvtfcHk0lhXI-BgYv-eXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovFGltb8%253D&md5=97f647fc215ea6d04882351ffc912a1e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjps.22674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22674%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DWhitney-Pickett%26aufirst%3DC.%26aulast%3DUmland%26aufirst%3DJ.%2BP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGebhard%26aufirst%3DD.%2BF.%26aulast%3DLai%26aufirst%3DY.%26aulast%3DFederico%26aufirst%3DJ.%2BJ.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DReyner%26aufirst%3DE.%2BL.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DRotter%26aufirst%3DC.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DKempshall%26aufirst%3DS.%26aulast%3DFenner%26aufirst%3DK.%26aulast%3DEl-kattan%26aufirst%3DA.%2BF.%26aulast%3DListon%26aufirst%3DT.%2BE.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520New%2520Permeability%2520Assay%2520Using%2520Low-Efflux%2520MDCKII%2520Cells%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26issue%3D11%26spage%3D4974%26epage%3D4985%26doi%3D10.1002%2Fjps.22674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">Frequent Hitters: Nuisance Artifacts in High-Throughput Screening</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">667</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2020.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31987936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=657-667&issue=4&author=Z.+Y.+Yangauthor=J.+H.+Heauthor=A.+P.+Luauthor=T.+J.+Houauthor=D.+S.+Cao&title=Frequent+Hitters%3A+Nuisance+Artifacts+in+High-Throughput+Screening&doi=10.1016%2Fj.drudis.2020.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Frequent hitters: nuisance artifacts in high-throughput screening</span></div><div class="casAuthors">Yang, Zi-Yi; He, Jun-Hong; Lu, Ai-Ping; Hou, Ting-Jun; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-667</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the major challenges in early drug discovery is the recognition of frequent hitters (FHs), i.e., compds. that nonspecifically bind to a range of macromol. targets or false positives caused by various types of assay interferences.  In this review, we survey the mechanisms underlying different types of FHs, including aggregators, spectroscopic interference compds. (i.e., luciferase inhibitors and fluorescent compds.), chem. reactive compds., and promiscuous compds.  We also review commonly used exptl. detection techniques and computational prediction models for FH identification.  In addn., the rational applications of these computational filters are discussed.  It is believed that, with the rational use of FH filters, the efficiency of drug discovery will be significantly improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA9bpJANksO7Vg90H21EOLACvtfcHk0lj_v-GO5Igy0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWjur0%253D&md5=9a5f8c8396815ce9cb84203d1d41dc47</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DHou%26aufirst%3DT.%2BJ.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DFrequent%2520Hitters%253A%2520Nuisance%2520Artifacts%2520in%2520High-Throughput%2520Screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26issue%3D4%26spage%3D657%26epage%3D667%26doi%3D10.1016%2Fj.drudis.2020.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dantas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, T. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Junior, F. P.</span></span> <span> </span><span class="NLM_article-title">Dealing with Frequent Hitters in Drug Discovery: A Multidisciplinary View on the Issue of Filtering Compounds on Biological Screenings</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1282</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1654453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1080%2F17460441.2019.1654453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31416369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyksbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1269-1282&issue=12&author=R.+F.+Dantasauthor=T.+C.+S.+Evangelistaauthor=B.+J.+Nevesauthor=M.+R.+Sengerauthor=C.+H.+Andradeauthor=S.+B.+Ferreiraauthor=F.+P.+Silva-Junior&title=Dealing+with+Frequent+Hitters+in+Drug+Discovery%3A+A+Multidisciplinary+View+on+the+Issue+of+Filtering+Compounds+on+Biological+Screenings&doi=10.1080%2F17460441.2019.1654453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dealing with frequent hitters in drug discovery: a multidisciplinary view on the issue of filtering compounds on biological screenings</span></div><div class="casAuthors">Dantas, Rafael Ferreira; Evangelista, Tereza Cristina Santos; Neves, Bruno, Jr.; Senger, Mario Roberto; Andrade, Carolina Horta; Ferreira, Sabrina Baptista; Silva-Junior, Floriano Paes</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1269-1282</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The timely identification biol. active chems., in disease relevant screening assays, is a major endeavor in drug discovery.  The existence of frequent hitters (FHs) in non-related assays poses a formidable challenge in terms of whether to consider these mols. as chem. gold or promiscuous non-selective reactive trash (also known as PAINS-pan assay interference compds.).  In this review, the authors bring together expertize in synthetic chem., cheminformatics and biochem., three key areas for dealing with FHs.  They discuss synthetic methods facilitating prepn. of chem. diverse mol. libraries, while favoring activity in the biol. space.  They also survey and discuss recent computational advances in the prediction of PAINS from chem. structures.  Finally, they review exptl. approaches for the validation of the biol. activity of screening hits and discuss alternatives for exploiting promiscuity and chem. reactivity.: It's essential to develop more efficient computational methods to reliably recognize PAINS in distinct mol. environments.  Accordingly, advances in synthetic chem. hold the promise to provide a better quality of chem. matter for drug discovery.  Medicinal chemists should be more open to screening for hits showing biol. complex mechanisms of action rather than discarding mols. that may prove valuable as innovative disease treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4hxv43QT26LVg90H21EOLACvtfcHk0lj_v-GO5Igy0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyksbzM&md5=d63f74505e6638ea1fba8486775215db</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1654453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1654453%26sid%3Dliteratum%253Aachs%26aulast%3DDantas%26aufirst%3DR.%2BF.%26aulast%3DEvangelista%26aufirst%3DT.%2BC.%2BS.%26aulast%3DNeves%26aufirst%3DB.%2BJ.%26aulast%3DSenger%26aufirst%3DM.%2BR.%26aulast%3DAndrade%26aufirst%3DC.%2BH.%26aulast%3DFerreira%26aufirst%3DS.%2BB.%26aulast%3DSilva-Junior%26aufirst%3DF.%2BP.%26atitle%3DDealing%2520with%2520Frequent%2520Hitters%2520in%2520Drug%2520Discovery%253A%2520A%2520Multidisciplinary%2520View%2520on%2520the%2520Issue%2520of%2520Filtering%2520Compounds%2520on%2520Biological%2520Screenings%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D12%26spage%3D1269%26epage%3D1282%26doi%3D10.1080%2F17460441.2019.1654453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakravorty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa-Burke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remlinger, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarrón, R.</span></span> <span> </span><span class="NLM_article-title">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1177/2472555218768497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F2472555218768497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=29699447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=532-545&issue=6&author=S.+J.+Chakravortyauthor=J.+Chanauthor=M.+N.+Greenwoodauthor=I.+Popa-Burkeauthor=K.+S.+Remlingerauthor=S.+D.+Pickettauthor=D.+V.+S.+Greenauthor=M.+C.+Fillmoreauthor=T.+W.+Deanauthor=J.+I.+Luengoauthor=R.+Macarr%C3%B3n&title=Nuisance+Compounds%2C+PAINS+Filters%2C+and+Dark+Chemical+Matter+in+the+GSK+HTS+Collection&doi=10.1177%2F2472555218768497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span></div><div class="casAuthors">Chakravorty, Subhas J.; Chan, James; Greenwood, Marie Nicole; Popa-Burke, Ioana; Remlinger, Katja S.; Pickett, Stephen D.; Green, Darren V. S.; Fillmore, Martin C.; Dean, Tony W.; Luengo, Juan I.; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">532-545</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) hits include compds. with undesirable properties.  Many filters have been described to identify such hits.  Notably, pan-assay interference compds. (PAINS) has been adopted by the community as the std. term to refer to such filters, and very useful guidelines have been adopted by the American Chem. Society (ACS) and subsequently triggered a healthy scientific debate about the pitfalls of draconian use of filters.  Using an inhibitory frequency index, we have analyzed in detail the promiscuity profile of the whole GlaxoSmithKline (GSK) HTS collection comprising more than 2 million unique compds. that have been tested in hundreds of screening assays.  We provide a comprehensive anal. of many previously published filters and newly described classes of nuisance structures that may serve as a useful source of empirical information to guide the design or growth of HTS collections and hit triaging strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5bHsBxkwOrVg90H21EOLACvtfcHk0lj_v-GO5Igy0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE&md5=9a620fb177aa001b0a36e90948f8e326</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F2472555218768497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218768497%26sid%3Dliteratum%253Aachs%26aulast%3DChakravorty%26aufirst%3DS.%2BJ.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreenwood%26aufirst%3DM.%2BN.%26aulast%3DPopa-Burke%26aufirst%3DI.%26aulast%3DRemlinger%26aufirst%3DK.%2BS.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DMacarr%25C3%25B3n%26aufirst%3DR.%26atitle%3DNuisance%2520Compounds%252C%2520PAINS%2520Filters%252C%2520and%2520Dark%2520Chemical%2520Matter%2520in%2520the%2520GSK%2520HTS%2520Collection%26jtitle%3DSLAS%2520Discovery%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D532%26epage%3D545%26doi%3D10.1177%2F2472555218768497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasial, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Highly Promiscuous Small Molecules from Biological Screening Assays Include Many Pan-Assay Interference Compounds but Also Candidates for Polypharmacology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">10285</span>– <span class="NLM_lpage">10290</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10285-10290&issue=22&author=E.+Gilbergauthor=S.+Jasialauthor=D.+Stumpfeauthor=D.+Dimovaauthor=J.+Bajorath&title=Highly+Promiscuous+Small+Molecules+from+Biological+Screening+Assays+Include+Many+Pan-Assay+Interference+Compounds+but+Also+Candidates+for+Polypharmacology&doi=10.1021%2Facs.jmedchem.6b01314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Promiscuous Small Molecules from Biological Screening Assays Include Many Pan-Assay Interference Compounds but Also Candidates for Polypharmacology</span></div><div class="casAuthors">Gilberg, Erik; Jasial, Swarit; Stumpfe, Dagmar; Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10285-10290</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In PubChem screening assays, 466 highly promiscuous compds. were identified that were examd. for known pan-assay interference compds. (PAINS) and aggregators using publicly available filters.  These filters detected 210 PAINS and 67 aggregators.  Compds. passing the filters included addnl. PAINS that were not detected, mostly due to tautomerism, and a variety of other potentially reactive compds. currently not encoded as PAINS.  For a subset of compds. passing the filters, there was no evidence of potential artifacts.  These compds. are considered candidates for further exploring multitarget activities and the mol. basis of polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCj2gzF-ovLVg90H21EOLACvtfcHk0lhRqygGfRtKmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOlsr7O&md5=55d062a51de53000f1de0952e631b0a9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01314%26sid%3Dliteratum%253Aachs%26aulast%3DGilberg%26aufirst%3DE.%26aulast%3DJasial%26aufirst%3DS.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DHighly%2520Promiscuous%2520Small%2520Molecules%2520from%2520Biological%2520Screening%2520Assays%2520Include%2520Many%2520Pan-Assay%2520Interference%2520Compounds%2520but%2520Also%2520Candidates%2520for%2520Polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D22%26spage%3D10285%26epage%3D10290%26doi%3D10.1021%2Facs.jmedchem.6b01314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleason, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beatini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Q. F.</span></span> <span> </span><span class="NLM_article-title">Metal Impurities Cause False Positives in High-Throughput Screening Campaigns</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1021/ml3003296</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3003296" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVanu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=197-200&issue=2&author=J.+C.+Hermannauthor=Y.+Chenauthor=C.+Wartchowauthor=J.+Menkeauthor=L.+Gaoauthor=S.+K.+Gleasonauthor=N.+E.+Haynesauthor=N.+Scottauthor=A.+Petersenauthor=S.+Gabrielauthor=B.+Vuauthor=K.+M.+Georgeauthor=A.+Narayananauthor=S.+H.+Liauthor=H.+Qianauthor=N.+Beatiniauthor=L.+Niuauthor=Q.+F.+Gan&title=Metal+Impurities+Cause+False+Positives+in+High-Throughput+Screening+Campaigns&doi=10.1021%2Fml3003296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Metal Impurities Cause False Positives in High-Throughput Screening Campaigns</span></div><div class="casAuthors">Hermann, Johannes C.; Chen, Yingsi; Wartchow, Charles; Menke, John; Gao, Lin; Gleason, Shelley K.; Haynes, Nancy-Ellen; Scott, Nathan; Petersen, Ann; Gabriel, Stephen; Vu, Binh; George, Kelly M.; Narayanan, Arjun; Li, Shirley H.; Qian, Hong; Beatini, Nanda; Niu, Linghao; Gan, Qing-Fen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-200</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Org. impurities in compd. libraries are known to often cause false-pos. signals in screening campaigns for new leads, but org. impurities do not fully account for all false-pos. results.  We discovered inorg. impurities in our screening library that can also cause pos. signals for a variety of targets and/or readout systems, including biochem. and biosensor assays.  We investigated in depth the example of zinc for a specific project and in retrospect in various HTS screens at Roche and propose a straight-forward counter screen using the chelator TPEN to rule out inhibition caused by zinc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF424s97a80bVg90H21EOLACvtfcHk0lhRqygGfRtKmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVanu77J&md5=24419d1ca51f21226f7774eb1d38b93f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml3003296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003296%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWartchow%26aufirst%3DC.%26aulast%3DMenke%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DGleason%26aufirst%3DS.%2BK.%26aulast%3DHaynes%26aufirst%3DN.%2BE.%26aulast%3DScott%26aufirst%3DN.%26aulast%3DPetersen%26aufirst%3DA.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DVu%26aufirst%3DB.%26aulast%3DGeorge%26aufirst%3DK.%2BM.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%2BH.%26aulast%3DQian%26aufirst%3DH.%26aulast%3DBeatini%26aufirst%3DN.%26aulast%3DNiu%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DQ.%2BF.%26atitle%3DMetal%2520Impurities%2520Cause%2520False%2520Positives%2520in%2520High-Throughput%2520Screening%2520Campaigns%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26issue%3D2%26spage%3D197%26epage%3D200%26doi%3D10.1021%2Fml3003296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of Frequent-Hitter Behavior Based on Historical High-Throughput Screening Data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&issue=10&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+Frequent-Hitter+Behavior+Based+on+Historical+High-Throughput+Screening+Data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0ljnjCJsPe4XBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520Frequent-Hitter%2520Behavior%2520Based%2520on%2520Historical%2520High-Throughput%2520Screening%2520Data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D10%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">David, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feierberg, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span> <span> </span><span class="NLM_article-title">Identification of Compounds That Interfere with High-Throughput Screening Assay Technologies</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">1795</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.201900395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31479198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsl2gtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1795-1802&issue=20&author=L.+Davidauthor=J.+Walshauthor=N.+Sturmauthor=I.+Feierbergauthor=J.+W.+M.+Nissinkauthor=H.+Chenauthor=J.+Bajorathauthor=O.+Engkvist&title=Identification+of+Compounds+That+Interfere+with+High-Throughput+Screening+Assay+Technologies&doi=10.1002%2Fcmdc.201900395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Compounds That Interfere with High-Throughput Screening Assay Technologies</span></div><div class="casAuthors">David, Laurianne; Walsh, Jarrod; Sturm, Noe; Feierberg, Isabella; Nissink, J. Willem M.; Chen, Hongming; Bajorath, Juergen; Engkvist, Ola</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1795-1802</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A significant challenge in high-throughput screening (HTS) campaigns is the identification of assay technol. interference compds.  A Compd. Interfering with an Assay Technol. (CIAT) gives false readouts in many assays.  CIATs are often considered viable hits and investigated in follow-up studies, thus impeding research and wasting resources.  In this study, the authors developed a machine-learning (ML) model to predict CIATs for three assay technologies.  The model was trained on known CIATs and non-CIATs (NCIATs) identified in artifact assays and described by their 2D structural descriptors.  Usual methods identifying CIATs are based on statistical anal. of historical primary screening data and do not consider exptl. assays identifying CIATs.  The results show successful prediction of CIATs for existing and novel compds. and provide a complementary and wider set of predicted CIATs compared to BSF, a published structure-independent model, and to the PAINS substructural filters.  The anal. is an example of how well-curated datasets can provide powerful predictive models despite their relatively small size.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLDqUbFKxxrVg90H21EOLACvtfcHk0ljnjCJsPe4XBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsl2gtb%252FP&md5=d40c09bf89d6e92de742ab3367d1caf8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900395%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DWalsh%26aufirst%3DJ.%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DFeierberg%26aufirst%3DI.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DIdentification%2520of%2520Compounds%2520That%2520Interfere%2520with%2520High-Throughput%2520Screening%2520Assay%2520Technologies%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D20%26spage%3D1795%26epage%3D1802%26doi%3D10.1002%2Fcmdc.201900395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljnjCJsPe4XBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017 - Utility and Limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&issue=1&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+Year+Itch%3A+Pan-Assay+Interference+Compounds+%28PAINS%29+in+2017+-+Utility+and+Limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0lhmzXBkzgz7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520Year%2520Itch%253A%2520Pan-Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520in%25202017%2520-%2520Utility%2520and%2520Limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D1%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunavs, M.</span></span> <span> </span><span class="NLM_article-title">Investigating the Behavior of Published PAINS Alerts Using a Pharmaceutical Company Data Set</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12jsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=792-796&issue=8&author=L.+R.+Vidlerauthor=I.+A.+Watsonauthor=B.+J.+Margolisauthor=D.+J.+Cumminsauthor=M.+Brunavs&title=Investigating+the+Behavior+of+Published+PAINS+Alerts+Using+a+Pharmaceutical+Company+Data+Set&doi=10.1021%2Facsmedchemlett.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating the Behavior of Published PAINS Alerts Using a Pharmaceutical Company Data Set</span></div><div class="casAuthors">Vidler, Lewis R.; Watson, Ian A.; Margolis, Brandon J.; Cummins, David J.; Brunavs, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">792-796</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Biochem. assay interference is becoming increasingly recognized as a significant waste of resource in drug discovery, both in industry and academia.  A seminal publication from Baell and Holloway raised the awareness of this issue, and they published a set of alerts to identify what they described as PAINS (pan-assay interference compds.).  These alerts have been taken up by drug discovery groups, even though the original paper had a somewhat limited data set.  Here, we have taken Lilly's far larger internal data set to assess the PAINS alerts on four criteria: promiscuity (over six assay formats including AlphaScreen), compd. stability, cytotoxicity, and presence of a high Hill slope as a surrogate for non-1:1 protein-ligand binding.  It was found that only three of the alerts show pan-assay promiscuity, and the alerts appear to encode primarily AlphaScreen promiscuous mols.  Although not enriching for pan-assay promiscuity, many of the alerts do encode mols. that are unstable, show cytotoxicity, and increase the prevalence of high Hill slopes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhzWMrqUkHVLVg90H21EOLACvtfcHk0lhmzXBkzgz7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12jsLbL&md5=794c7338ab2ef2dc8da60cb2696760ff</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DMargolis%26aufirst%3DB.%2BJ.%26aulast%3DCummins%26aufirst%3DD.%2BJ.%26aulast%3DBrunavs%26aufirst%3DM.%26atitle%3DInvestigating%2520the%2520Behavior%2520of%2520Published%2520PAINS%2520Alerts%2520Using%2520a%2520Pharmaceutical%2520Company%2520Data%2520Set%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D8%26spage%3D792%26epage%3D796%26doi%3D10.1021%2Facsmedchemlett.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenaigner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinshagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span> <span> </span><span class="NLM_article-title">Identification of Small-Molecule Frequent Hitters from Alphascreen High-Throughput Screens</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1177/1087057113516861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057113516861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=24371213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=715-726&issue=5&author=K.+Schorppauthor=I.+Rothenaignerauthor=E.+Salminaauthor=J.+Reinshagenauthor=T.+Lowauthor=J.+K.+Brenkeauthor=J.+Gopalakrishnanauthor=I.+V.+Tetkoauthor=S.+Gulauthor=K.+Hadian&title=Identification+of+Small-Molecule+Frequent+Hitters+from+Alphascreen+High-Throughput+Screens&doi=10.1177%2F1087057113516861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small-molecule frequent hitters from Alphascreen high-throughput screens</span></div><div class="casAuthors">Schorpp, Kenji; Rothenaigner, Ina; Salmina, Elena; Reinshagen, Jeanette; Low, Terence; Brenke, Jara K.; Gopalakrishnan, Jay; Tetko, Igor V.; Gul, Sheraz; Hadian, Kamyar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">715-726</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Although small-mol. drug discovery efforts have focused largely on enzyme, receptor, and ion-channel targets, there has been an increase in such activities to search for protein-protein interaction (PPI) disruptors by applying high-throughout screening (HTS)-compatible protein-binding assays.  However, a disadvantage of these assays is that many primary hits are frequent hitters regardless of the PPI being investigated.  We have used the AlphaScreen technol. to screen four different robust PPI assays each against 25,000 compds.  These activities led to the identification of 137 compds. that demonstrated repeated activity in all PPI assays.  These compds. were subsequently evaluated in two AlphaScreen counter assays, leading to classification of compds. that either interfered with the AlphaScreen chem. (60 compds.) or prevented the binding of the protein His-tag moiety to nickel chelate (Ni2+-NTA) beads of the AlphaScreen detection system (77 compds.).  To further triage the 137 frequent hitters, we subsequently confirmed by a time-resolved fluorescence resonance energy transfer assay that most of these compds. were only frequent hitters in AlphaScreen assays.  A chemoinformatics anal. of the apparent hits provided details of the compds. that can be flagged as frequent hitters of the AlphaScreen technol., and these data have broad applicability for users of these detection technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQj9hCbRqv7LVg90H21EOLACvtfcHk0lhmzXBkzgz7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM&md5=ea65e879f093202deb2266bad94f0208</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1177%2F1087057113516861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516861%26sid%3Dliteratum%253Aachs%26aulast%3DSchorpp%26aufirst%3DK.%26aulast%3DRothenaigner%26aufirst%3DI.%26aulast%3DSalmina%26aufirst%3DE.%26aulast%3DReinshagen%26aufirst%3DJ.%26aulast%3DLow%26aufirst%3DT.%26aulast%3DBrenke%26aufirst%3DJ.%2BK.%26aulast%3DGopalakrishnan%26aufirst%3DJ.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DGul%26aufirst%3DS.%26aulast%3DHadian%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520Small-Molecule%2520Frequent%2520Hitters%2520from%2520Alphascreen%2520High-Throughput%2520Screens%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26issue%3D5%26spage%3D715%26epage%3D726%26doi%3D10.1177%2F1087057113516861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sofia, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D. A.</span></span> <span> </span><span class="NLM_article-title">An Empirical Process for the Design of High-Throughput Screening Deck Filters</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1060</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/ci050504m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050504m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFWju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1060-1068&issue=3&author=B.+C.+Pearceauthor=M.+J.+Sofiaauthor=A.+C.+Goodauthor=D.+M.+Drexlerauthor=D.+A.+Stock&title=An+Empirical+Process+for+the+Design+of+High-Throughput+Screening+Deck+Filters&doi=10.1021%2Fci050504m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An Empirical Process for the Design of High-Throughput Screening Deck Filters</span></div><div class="casAuthors">Pearce, Bradley C.; Sofia, Michael J.; Good, Andrew C.; Drexler, Dieter M.; Stock, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1060-1068</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A process for objective identification and filtering of undesirable compds. that contribute to high-throughput screening (HTS) deck promiscuity is described.  Two methods of mapping hit promiscuity have been developed linking SMARTS-based structural queries with historical primary HTS data.  The first compares an expected assay hit rate to actual hit rates.  The second examines the propensity of an individual compd. to hit multiple assays.  Statistical evaluation of the data indicates a correlation between the resultant functional group filters and compd. promiscuity.  These data corroborate a no. of commonly applied filters as well as producing some unexpected results.  Application of these models to HTS collection triage reduced the no. of inhouse compds. considered for screening by 12%.  The implications of these findings are further discussed in the context of the HTS screening set and combinatorial library design as well as compd. acquisition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_rORr8svyLLVg90H21EOLACvtfcHk0lhmzXBkzgz7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFWju7s%253D&md5=8c6260a16996b2cbf5d3caec5a883b0b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci050504m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050504m%26sid%3Dliteratum%253Aachs%26aulast%3DPearce%26aufirst%3DB.%2BC.%26aulast%3DSofia%26aufirst%3DM.%2BJ.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DDrexler%26aufirst%3DD.%2BM.%26aulast%3DStock%26aufirst%3DD.%2BA.%26atitle%3DAn%2520Empirical%2520Process%2520for%2520the%2520Design%2520of%2520High-Throughput%2520Screening%2520Deck%2520Filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D3%26spage%3D1060%26epage%3D1068%26doi%3D10.1021%2Fci050504m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reutlinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Multi-Objective Molecular de Novo Design by Adaptive Fragment Prioritization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4244</span>– <span class="NLM_lpage">4248</span>, <span class="refDoi"> DOI: 10.1002/anie.201310864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fanie.201310864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Sls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=4244-4248&issue=16&author=M.+Reutlingerauthor=T.+Rodriguesauthor=P.+Schneiderauthor=G.+Schneider&title=Multi-Objective+Molecular+de+Novo+Design+by+Adaptive+Fragment+Prioritization&doi=10.1002%2Fanie.201310864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Objective Molecular De Novo Design by Adaptive Fragment Prioritization</span></div><div class="casAuthors">Reutlinger, Michael; Rodrigues, Tiago; Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4244-4248</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We present the development and application of a computational mol. de novo design method for obtaining bioactive compds. with desired on- and off-target binding.  The approach translates the nature-inspired concept of ant colony optimization to combinatorial building block selection.  By relying on publicly available structure-activity data, we developed a predictive quant. polypharmacol. model for 640 human drug targets.  By taking reductive amination as an example of a privileged reaction, we obtained novel subtype-selective and multitarget-modulating dopamine D4 antagonists, as well as ligands selective for the sigma-1 receptor with accurately predicted affinities.  The nanomolar potencies of the hits obtained, their high ligand efficiencies, and an overall success rate of 90 % demonstrate that this ligand-based computer-aided mol. design method may guide target-focused combinatorial chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQcmive3D0LVg90H21EOLACvtfcHk0ljQ7G-MKl3E1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Sls7k%253D&md5=031a8e079e55dd027f90ea89fc42c0e6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fanie.201310864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201310864%26sid%3Dliteratum%253Aachs%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DRodrigues%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DMulti-Objective%2520Molecular%2520de%2520Novo%2520Design%2520by%2520Adaptive%2520Fragment%2520Prioritization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26issue%3D16%26spage%3D4244%26epage%3D4248%26doi%3D10.1002%2Fanie.201310864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewell, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lifely, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsden, N.</span></span> <span> </span><span class="NLM_article-title">Strategic Pooling of Compounds for High-Throughput Screening</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1021/ci990423o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci990423o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK1MXksVWkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=897-902&issue=5&author=M.+Hannauthor=B.+Hudsonauthor=X.+Lewellauthor=R.+Lifelyauthor=L.+Millerauthor=N.+Ramsden&title=Strategic+Pooling+of+Compounds+for+High-Throughput+Screening&doi=10.1021%2Fci990423o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Pooling of Compounds for High-Throughput Screening</span></div><div class="casAuthors">Hann, Mike; Hudson, Brian; Lewell, Xiao; Lifely, Rob; Miller, Luke; Ramsden, Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">897-902</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bringing new medicines to the market depends on the rapid discovery of new and effective drugs, often initiated through the biol. testing of many thousands of compds. in high-throughput screening (HTS).  Mixing compds. together into pools for screening is one way to accelerate this process and reduce costs.  This paper contains both theor. and exptl. data which suggest that careful selection of compds. to be pooled together is necessary to reduce the risk of reactivity between compds. within the pools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTwF-1nNSb07Vg90H21EOLACvtfcHk0ljQ7G-MKl3E1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVWkt78%253D&md5=513eb71f84118c961dbadddc44f7af9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci990423o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci990423o%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%26aulast%3DHudson%26aufirst%3DB.%26aulast%3DLewell%26aufirst%3DX.%26aulast%3DLifely%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DL.%26aulast%3DRamsden%26aufirst%3DN.%26atitle%3DStrategic%2520Pooling%2520of%2520Compounds%2520for%2520High-Throughput%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26issue%3D5%26spage%3D897%26epage%3D902%26doi%3D10.1021%2Fci990423o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blake, J.</span></span> <span> </span><span class="NLM_article-title">Identification and Evaluation of Molecular Properties Related to Preclinical Optimization and Clinical Fate</span>. <i>Med. Chem. (Sharjah, United Arab Emirates)</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.2174/157340605774598081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F157340605774598081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=649-655&issue=6&author=J.+Blake&title=Identification+and+Evaluation+of+Molecular+Properties+Related+to+Preclinical+Optimization+and+Clinical+Fate&doi=10.2174%2F157340605774598081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.2174%2F157340605774598081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340605774598081%26sid%3Dliteratum%253Aachs%26aulast%3DBlake%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520Evaluation%2520of%2520Molecular%2520Properties%2520Related%2520to%2520Preclinical%2520Optimization%2520and%2520Clinical%2520Fate%26jtitle%3DMed.%2520Chem.%2520%2528Sharjah%252C%2520United%2520Arab%2520Emirates%2529%26date%3D2005%26volume%3D1%26issue%3D6%26spage%3D649%26epage%3D655%26doi%3D10.2174%2F157340605774598081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span> <span> </span><span class="NLM_article-title">Lessons Learnt from Assembling Screening Libraries for Drug Discovery for Neglected Diseases</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200700139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.200700139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=18064617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Krtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=435-444&issue=3&author=R.+Brenkauthor=A.+Schipaniauthor=D.+Jamesauthor=A.+Krasowskiauthor=I.+H.+Gilbertauthor=J.+Frearsonauthor=P.+G.+Wyatt&title=Lessons+Learnt+from+Assembling+Screening+Libraries+for+Drug+Discovery+for+Neglected+Diseases&doi=10.1002%2Fcmdc.200700139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from assembling screening libraries for drug discovery for neglected diseases</span></div><div class="casAuthors">Brenk, Ruth; Schipani, Alessandro; James, Daniel; Krasowski, Agata; Gilbert, Ian Hugh; Frearson, Julie; Wyatt, Paul Graham</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">To enable the establishment of a drug discovery operation for neglected diseases, out of 2.3 million com. available compds. 222 552 compds. were selected for an in silico library, 57 438 for a diverse general screening library, and 1 697 compds. for a focused kinase set.  Compiling these libraries required a robust strategy for compd. selection.  Rules for unwanted groups were defined and selection criteria to enrich for lead-like compds. which facilitate straightforward structure-activity relationship exploration were established.  Further, a literature and patent review was undertaken to ext. key recognition elements of kinase inhibitors ("core fragments") to assemble a focused library for hit discovery for kinases.  Computational and exptl. characterization of the general screening library revealed that the selected compds. (1) span a broad range of lead-like space, (2) show a high degree of structural integrity and purity, and (3) demonstrate appropriate soly. for the purposes of biochem. screening.  The implications of this study for compd. selection, esp. in an academic environment with limited resources, are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy_Ybsdg8QS7Vg90H21EOLACvtfcHk0ljEZkCnGVcqPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Krtb4%253D&md5=b804d0ac8766e954391798e24aa7fbff</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700139%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DSchipani%26aufirst%3DA.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26atitle%3DLessons%2520Learnt%2520from%2520Assembling%2520Screening%2520Libraries%2520for%2520Drug%2520Discovery%2520for%2520Neglected%2520Diseases%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26issue%3D3%26spage%3D435%26epage%3D444%26doi%3D10.1002%2Fcmdc.200700139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Phantom PAINS: Problems with the Utility of Alerts for P an- A Ssay in Terference Compound S</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFCgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=417-427&issue=3&author=S.+J.+Capuzziauthor=E.+N.+Muratovauthor=A.+Tropsha&title=Phantom+PAINS%3A+Problems+with+the+Utility+of+Alerts+for+P+an-+A+Ssay+in+Terference+Compound+S&doi=10.1021%2Facs.jcim.6b00465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Phantom PAINS: Problems with the Utility of Alerts for Pan-Assay INterference CompoundS</span></div><div class="casAuthors">Capuzzi, Stephen J.; Muratov, Eugene N.; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-427</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of substructural alerts to identify Pan-Assay INterference compds. (PAINS) has become a common component of the triage process in biol. screening campaigns.  These alerts, however, were originally derived from a proprietary library tested in just six assays measuring protein-protein interaction (PPI) inhibition using the AlphaScreen detection technol. only; moreover, 68% (328 out of the 480 alerts) were derived from four or fewer compds.  In an effort to assess the reliability of these alerts as indicators of pan-assay interference, the authors performed a large-scale anal. of the impact of PAINS alerts on compd. promiscuity in bioassays using publicly available data in PubChem.  The authors found that the majority (97%) of all compds. contg. PAINS alerts were actually infrequent hitters in AlphaScreen assays measuring PPI inhibition.  The authors also found that the presence of PAINS alerts, contrary to expectations, did not reflect any heightened assay activity trends across all assays in PubChem including AlphaScreen, luciferase, β-lactamase, or fluorescence-based assays.  In addn., 109 PAINS alerts were present in 3570 extensively assayed, but consistently inactive compds. Dark Chem. Matter.  Finally, the authors obsd. that 87 small mol. FDA-approved drugs contained PAINS alerts and profiled their bioassay activity.  Based on this detailed anal. of PAINS alerts in nonproprietary compd. libraries, the authors caution against the blind use of PAINS filters to detect and triage compds. with possible PAINS liabilities and recommend that such conclusions should be drawn only by conducting orthogonal expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoczoSnfEJzUrVg90H21EOLACvtfcHk0ljEZkCnGVcqPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFCgtbw%253D&md5=206e1d1f24df12644527ddbaa03b24b0</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00465%26sid%3Dliteratum%253Aachs%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DPhantom%2520PAINS%253A%2520Problems%2520with%2520the%2520Utility%2520of%2520Alerts%2520for%2520P%2520an-%2520A%2520Ssay%2520in%2520Terference%2520Compound%2520S%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D3%26spage%3D417%26epage%3D427%26doi%3D10.1021%2Facs.jcim.6b00465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, P.</span></span> Filtering Chemcial Libraries <a href="http://practicalcheminformatics.blogspot.com/2018/08/filtering-chemical-libraries.html" class="extLink">http://practicalcheminformatics.blogspot.com/2018/08/filtering-chemical-libraries.html</a> (accessed Jul 2, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Walters%2C+P.+Filtering+Chemcial+Libraries+http%3A%2F%2Fpracticalcheminformatics.blogspot.com%2F2018%2F08%2Ffiltering-chemical-libraries.html+%28accessed+Jul+2%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DP.%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span>Daylight Theory Manual:
4. SMARTS - A Language for Describing Molecular Patterns. <a href="https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed Jul 1, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight+Theory+Manual%3A%0A4.+SMARTS+-+A+Language+for+Describing+Molecular+Patterns.+https%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+Jul+1%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmki, E. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Comparing Molecular Patterns Using the Example of SMARTS: Applications and Filter Collection Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2572</span>– <span class="NLM_lpage">2586</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00249</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00249" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVais7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2572-2586&issue=6&author=E.+S.+R.+Ehmkiauthor=R.+Schmidtauthor=F.+Ohmauthor=M.+Rarey&title=Comparing+Molecular+Patterns+Using+the+Example+of+SMARTS%3A+Applications+and+Filter+Collection+Analysis&doi=10.1021%2Facs.jcim.9b00249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing Molecular Patterns Using the Example of SMARTS: Applications and Filter Collection Analysis</span></div><div class="casAuthors">Ehmki, Emanuel S. R.; Schmidt, Robert; Ohm, Farina; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2572-2586</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a recent work, an algorithm to compare chem. patterns, written for example in SMARTS, was presented.  This algorithm, called SMARTScompare, is able to assess the identity, subset relation, and similarity of a pair of patterns.  Here we used an implementation of SMARTScompare to analyze SMARTS filter sets that were published in the context of, for example, high-throughput screening.  We found that the difference in intentions with which the filter sets were designed is mirrored in the similarity values we calcd.  The anal. revealed which patterns from one filter set are covered by filters from another set.  In one case it became obvious that a filter set is more or less completely covered by another.  Furthermore, we analyzed pattern hierarchies for consistency, and we propose a method to remove redundant patterns.  SMARTScompare together with SMARTScompareView equips users with powerful methods to visualize, compare, and focus their filter sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGronGm2an_3jrVg90H21EOLACvtfcHk0ljEZkCnGVcqPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVais7%252FN&md5=20522c4eef1dbe10205c90cce245dbc1</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00249%26sid%3Dliteratum%253Aachs%26aulast%3DEhmki%26aufirst%3DE.%2BS.%2BR.%26aulast%3DSchmidt%26aufirst%3DR.%26aulast%3DOhm%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DComparing%2520Molecular%2520Patterns%2520Using%2520the%2520Example%2520of%2520SMARTS%253A%2520Applications%2520and%2520Filter%2520Collection%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D6%26spage%3D2572%26epage%3D2586%26doi%3D10.1021%2Facs.jcim.9b00249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmki, E. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashychev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Comparing Molecular Patterns Using the Example of SMARTS: Theory and Algorithms</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2560</span>– <span class="NLM_lpage">2571</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVWjt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2560-2571&issue=6&author=R.+Schmidtauthor=E.+S.+R.+Ehmkiauthor=F.+Ohmauthor=H.+C.+Ehrlichauthor=A.+Mashychevauthor=M.+Rarey&title=Comparing+Molecular+Patterns+Using+the+Example+of+SMARTS%3A+Theory+and+Algorithms&doi=10.1021%2Facs.jcim.9b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing Molecular Patterns Using the Example of SMARTS: Theory and Algorithms</span></div><div class="casAuthors">Schmidt, Robert; Ehmki, Emanuel S. R.; Ohm, Farina; Ehrlich, Hans-Christian; Mashychev, Andriy; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2560-2571</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. patterns are widely used for compd. filtering in mol. design endeavors.  They describe structural properties that are connected with unwanted phys. or chem. properties like reactivity or toxicity.  With filter sets comprising hundreds of structural filters, an analytic approach to compare those patterns is needed.  Here the authors present a novel approach to solve the generic pattern comparison problem.  The authors introduce chem. inspired fingerprints for pattern nodes and edges to derive an easy-to-compare pattern representation.  On two annotated pattern graphs the authors apply a max. common subgraph algorithm enabling the calcn. of pattern inclusion and similarity.  The resulting algorithm can be used in many different ways.  The authors can automatically derive pattern hierarchies or search in large pattern collections for more general or more specific patterns.  To the best of the knowledge, the presented algorithm is the first of its kind enabling these types of chem. pattern analytics.  The new tool named SMARTScompare is an implementation of the approach for the SMARTS language, which is the quasi-std. for structural filters.  The authors demonstrate the capabilities of SMARTScompare on a large collection of SMARTS patterns from real applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYr0R1czYJz7Vg90H21EOLACvtfcHk0ljqU120NBk_oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVWjt7zK&md5=5bf149130e7ac86e093bc96f7d38d963</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00250%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DR.%26aulast%3DEhmki%26aufirst%3DE.%2BS.%2BR.%26aulast%3DOhm%26aufirst%3DF.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DMashychev%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DComparing%2520Molecular%2520Patterns%2520Using%2520the%2520Example%2520of%2520SMARTS%253A%2520Theory%2520and%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D6%26spage%3D2560%26epage%3D2571%26doi%3D10.1021%2Facs.jcim.9b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaslavsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolton, E. E.</span></span> <span> </span><span class="NLM_article-title">PubChem 2019 Update: Improved Access to Chemical Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1102</span>– <span class="NLM_lpage">D1109</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30371825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1102-D1109&issue=D1&author=S.+Kimauthor=J.+Chenauthor=T.+Chengauthor=A.+Gindulyteauthor=J.+Heauthor=S.+Heauthor=Q.+Liauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=B.+Yuauthor=L.+Zaslavskyauthor=J.+Zhangauthor=E.+E.+Bolton&title=PubChem+2019+Update%3A+Improved+Access+to+Chemical+Data&doi=10.1093%2Fnar%2Fgky1033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1033%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZaslavsky%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26atitle%3DPubChem%25202019%2520Update%253A%2520Improved%2520Access%2520to%2520Chemical%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1102%26epage%3DD1109%26doi%3D10.1093%2Fnar%2Fgky1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schomburg, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Interactive Design of Generic Chemical Patterns</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">13–14</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23402846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFyqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=651-658&issue=13%E2%80%9314&author=K.+T.+Schomburgauthor=L.+Wetzerauthor=M.+Rarey&title=Interactive+Design+of+Generic+Chemical+Patterns&doi=10.1016%2Fj.drudis.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive design of generic chemical patterns</span></div><div class="casAuthors">Schomburg, Karen T.; Wetzer, Lars; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">651-658</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Every medicinal chemist has to create chem. patterns occasionally for querying databases, applying filters or describing functional groups.  However, the representations of chem. patterns have been so far limited to languages with highly complex syntax, handicapping the application of patterns.  Graphic pattern editors similar to chem. editors can facilitate the work with patterns.  In this article, we review the interfaces of frequently used web search engines for chem. patterns.  We take a look at pattern editing concepts of standalone chem. editors and finally present a completely new, unpublished graphical approach to pattern design, the SMARTSeditor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgM7nF8XasFrVg90H21EOLACvtfcHk0ljqU120NBk_oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFyqtrc%253D&md5=6f2b072249484a95d353caac44c050ea</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchomburg%26aufirst%3DK.%2BT.%26aulast%3DWetzer%26aufirst%3DL.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DInteractive%2520Design%2520of%2520Generic%2520Chemical%2520Patterns%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26issue%3D13%25E2%2580%259314%26spage%3D651%26epage%3D658%26doi%3D10.1016%2Fj.drudis.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Félix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magariños, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Marañón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: Towards Direct Deposition of Bioassay Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30398643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&issue=D1&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+F%C3%A9lixauthor=M.+P.+Magari%C3%B1osauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Mara%C3%B1%C3%B3nauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+Towards+Direct+Deposition+of+Bioassay+Data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: towards direct deposition of bioassay data</span></div><div class="casAuthors">Mendez, David; Gaulton, Anna; Bento, A. Patricia; Chambers, Jon; De Veij, Marleen; Felix, Eloy; Magarinos, Maria Paula; Mosquera, Juan F.; Mutowo, Prudence; Nowotka, Michal; Gordillo-Maranon, Maria; Hunter, Fiona; Junco, Laura; Mugumbate, Grace; Rodriguez-Lopez, Milagros; Atkinson, Francis; Bosc, Nicolas; Radoux, Chris J.; Cabrera, Aldo Segura; Hersey, Anne; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D930-D940</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is a large, open-access bioactivity database, previously described in the 2012, 2014 and 2017 Nucleic Acids Research Database Issues.  In the last two years, several important improvements have been made to the database and are described here.  These include more robust capture and representation of assay details; a new data deposition system, allowing updating of data sets and deposition of supplementary data; and a completely redesigned web interface, with enhanced search and filtering capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM7KYSXc4jerVg90H21EOLACvtfcHk0lip5eyhMLtwJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF&md5=d80660ce87d5b3e0773594e4fddec046</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DF%25C3%25A9lix%26aufirst%3DE.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Mara%25C3%25B1%25C3%25B3n%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520Towards%2520Direct%2520Deposition%2520of%2520Bioassay%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cibrian-Uhalte, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarinos, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Database in 2017</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D945</span>– <span class="NLM_lpage">D954</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgkw1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27899562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D945-D954&issue=D1&author=A.+Gaultonauthor=A.+Herseyauthor=M.+L.+Nowotkaauthor=A.+P.+Bentoauthor=J.+Chambersauthor=D.+Mendezauthor=P.+Mutowoauthor=F.+Atkinsonauthor=L.+J.+Bellisauthor=E.+Cibrian-Uhalteauthor=M.+Daviesauthor=N.+Dedmanauthor=A.+Karlssonauthor=M.+P.+Magarinosauthor=J.+P.+Overingtonauthor=G.+Papadatosauthor=I.+Smitauthor=A.+R.+Leach&title=The+ChEMBL+Database+in+2017&doi=10.1093%2Fnar%2Fgkw1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL database in 2017</span></div><div class="casAuthors">Gaulton, Anna; Hersey, Anne; Nowotka, Micha-l; Bento, A. Patricia; Chambers, Jon; Mendez, David; Mutowo, Prudence; Atkinson, Francis; Bellis, Louisa J.; Cibrian-Uhalte, Elena; Davies, Mark; Dedman, Nathan; Karlsson, Anneli; Magarinos, Maria Paula; Overington, John P.; Papadatos, George; Smit, Ines; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D945-D954</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 and 2014 Nucleic Acids Research Database Issues.  Since then, alongside the continued extn. of data from the medicinal chem. literature, new sources of bioactivity data have also been added to the database.  These include: deposited data sets from neglected disease screening; crop protection data; drug metab. and disposition data and bioactivity data from patents.  A no. of improvements and new features have also been incorporated.  These include the annotation of assays and targets using ontologies, the inclusion of targets and indications for clin. candidates, addn. of metabolic pathways for drugs and calcn. of structural alerts.  The ChEMBL data can be accessed via a web-interface, RDF distribution, data downloads and RESTful web-services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB0IuNShSHIrVg90H21EOLACvtfcHk0lip5eyhMLtwJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D&md5=1dcbb641f8c2d11410663f97505a4911</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1074%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DNowotka%26aufirst%3DM.%2BL.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DCibrian-Uhalte%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DMagarinos%26aufirst%3DM.%2BP.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DI.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DThe%2520ChEMBL%2520Database%2520in%25202017%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD945%26epage%3DD954%26doi%3D10.1093%2Fnar%2Fgkw1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span>Clarivate Integrity. <a href="https://integrity.clarivate.com/" class="extLink">https://integrity.clarivate.com/</a> (accessed Jun 9, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Clarivate+Integrity.+https%3A%2F%2Fintegrity.clarivate.com%2F+%28accessed+Jun+9%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span>GOSTAR. <a href="https://www.gostardb.com/gostar/index.jsp" class="extLink">https://www.gostardb.com/gostar/index.jsp</a> (accessed Jun 9, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR.+https%3A%2F%2Fwww.gostardb.com%2Fgostar%2Findex.jsp+%28accessed+Jun+9%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2016.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27427232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=862-874&issue=7&author=Y.+Wangauthor=A.+Cornettauthor=F.+J.+Kingauthor=Y.+Maoauthor=F.+Nigschauthor=C.+G.+Parisauthor=G.+McAllisterauthor=J.+L.+Jenkins&title=Evidence-Based+and+Quantitative+Prioritization+of+Tool+Compounds+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.chembiol.2016.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span></div><div class="casAuthors">Wang, Yuan; Cornett, Allen; King, Fred J.; Mao, Yi; Nigsch, Florian; Paris, C. Gregory; McAllister, Gregory; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">862-874</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The use of potent and selective chem. tools with well-defined targets can help elucidate biol. processes driving phenotypes in phenotypic screens.  However, identification of selective compds. en masse to create targeted screening sets is non-trivial.  A systematic approach is needed to prioritize probes, which prevents the repeated use of published but unselective compds.  Here we performed a meta-anal. of integrated large-scale, heterogeneous bioactivity data to create an evidence-based, quant. metric to systematically rank tool compds. for targets.  Our tool score (TS) was then tested on hundreds of compds. by assessing their activity profiles in a panel of 41 cell-based pathway assays.  We demonstrate that high-TS tools show more reliably selective phenotypic profiles than lower-TS compds.  Addnl. we highlight frequently tested compds. that are non-selective tools and distinguish target family polypharmacol. from cross-family promiscuity.  TS can therefore be used to prioritize compds. from heterogeneous databases for phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquaPu2t_l4ZbVg90H21EOLACvtfcHk0lhO7Mu7XQWQdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E&md5=b990346556ac032b8cecbfc9455f8458</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DKing%26aufirst%3DF.%2BJ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DEvidence-Based%2520and%2520Quantitative%2520Prioritization%2520of%2520Tool%2520Compounds%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26issue%3D7%26spage%3D862%26epage%3D874%26doi%3D10.1016%2Fj.chembiol.2016.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canham, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baeschlin, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patoor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaanderup, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapa, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspesi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labbé-Giguère, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamber, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushell, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Systematic Chemogenetic Library Assembly</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1124</span>– <span class="NLM_lpage">1129</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2020.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=32707038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A280%3ADC%252BB38jms1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=1124-1129&author=S.+M.+Canhamauthor=Y.+Wangauthor=A.+Cornettauthor=D.+S.+Auldauthor=D.+K.+Baeschlinauthor=M.+Patoorauthor=P.+R.+Skaanderupauthor=A.+Hondaauthor=L.+Llamasauthor=G.+Wendelauthor=F.+A.+Mapaauthor=P.+Aspesiauthor=N.+Labb%C3%A9-Gigu%C3%A8reauthor=G.+G.+Gamberauthor=D.+S.+Palaciosauthor=A.+Schuffenhauerauthor=Z.+Dengauthor=F.+Nigschauthor=M.+Frederiksenauthor=S.+M.+Bushellauthor=D.+Rothmanauthor=R.+K.+Jainauthor=H.+Hemmerleauthor=K.+Brinerauthor=J.+A.+Porterauthor=J.+A.+Tallaricoauthor=J.+L.+Jenkins&title=Systematic+Chemogenetic+Library+Assembly&doi=10.1016%2Fj.chembiol.2020.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Chemogenetic Library Assembly</span></div><div class="casAuthors">Canham Stephen M; Auld Douglas S; Jenkins Jeremy L; Wang Yuan; Cornett Allen; Honda Ayako; Llamas Luis; Wendel Greg; Mapa Felipa A; Aspesi Peter Jr; Labbe-Giguere Nancy; Gamber Gabriel G; Palacios Daniel S; Deng Zhan; Bushell Simon M; Rothman Deborah; Jain Rishi K; Hemmerle Horst; Briner Karin; Porter Jeffery A; Tallarico John A; Baeschlin Daniel K; Patoor Maude; Skaanderup Philip R; Schuffenhauer Ansgar; Nigsch Florian; Frederiksen Mathias</div><div class="citationInfo"><span class="NLM_cas:title">Cell chemical biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1124-1129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemogenetic libraries, collections of well-defined chemical probes, provide tremendous value to biomedical research but require substantial effort to ensure diversity as well as quality of the contents.  We have assembled a chemogenetic library by data mining and crowdsourcing institutional expertise.  We are sharing our approach, lessons learned, and disclosing our current collection of 4,185 compounds with their primary annotated gene targets (https://github.com/Novartis/MoaBox).  This physical collection is regularly updated and used broadly both within Novartis and in collaboration with external partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQemxh8whKe-C86Js-BCLiifW6udTcc2eawsMUcg90Czbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38jms1yjsg%253D%253D&md5=e566fa7acce73970a41251a22fc42259</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DCanham%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DBaeschlin%26aufirst%3DD.%2BK.%26aulast%3DPatoor%26aufirst%3DM.%26aulast%3DSkaanderup%26aufirst%3DP.%2BR.%26aulast%3DHonda%26aufirst%3DA.%26aulast%3DLlamas%26aufirst%3DL.%26aulast%3DWendel%26aufirst%3DG.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3DLabb%25C3%25A9-Gigu%25C3%25A8re%26aufirst%3DN.%26aulast%3DGamber%26aufirst%3DG.%2BG.%26aulast%3DPalacios%26aufirst%3DD.%2BS.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DBushell%26aufirst%3DS.%2BM.%26aulast%3DRothman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DR.%2BK.%26aulast%3DHemmerle%26aufirst%3DH.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DSystematic%2520Chemogenetic%2520Library%2520Assembly%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D1124%26epage%3D1129%26doi%3D10.1016%2Fj.chembiol.2020.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&issue=8&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+Molecular+Similarity%3A+Comparing+Compounds+on+the+Basis+of+Biological+Activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0lgrmoKpZUq5nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520Molecular%2520Similarity%253A%2520Comparing%2520Compounds%2520on%2520the%2520Basis%2520of%2520Biological%2520Activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26issue%3D8%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity to Discover Structurally Diverse Bioactive Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">692</span>– <span class="NLM_lpage">703</span>, <span class="refDoi"> DOI: 10.1021/ci300607r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300607r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=692-703&issue=3&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=M.+Glick&title=Bioturbo+Similarity+Searching%3A+Combining+Chemical+and+Biological+Similarity+to+Discover+Structurally+Diverse+Bioactive+Molecules&doi=10.1021%2Fci300607r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity To Discover Structurally Diverse Bioactive Molecules</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">692-703</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening using bioactivity profiles has become an integral part of currently applied hit finding methods in pharmaceutical industry.  However, a significant drawback of this approach is that it is only applicable to compds. that have been biol. tested in the past and have sufficient activity annotations for meaningful profile comparisons.  Although bioactivity data generated in pharmaceutical institutions are growing on an unprecedented scale, the no. of biol. annotated compds. still covers only a minuscule fraction of chem. space.  For a newly synthesized compd. or an isolated natural product to be biol. characterized across multiple assays, it may take a considerable amt. of time.  Consequently, this chem. matter will not be included in virtual screening campaigns based on bioactivity profiles.  To overcome this problem, we herein introduce bioturbo similarity searching that uses chem. similarity to map mols. without biol. annotations into bioactivity space and then searches for biol. similar compds. in this ref. system.  In benchmark calcns. on primary screening data, we demonstrate that our approach generally achieves higher hit rates and identifies structurally more diverse compds. than approaches using chem. information only.  Furthermore, our method is able to discover hits with novel modes of inhibition that traditional 2D and 3D similarity approaches are unlikely to discover.  Test calcns. on a set of natural products reveal the practical utility of the approach for identifying novel and synthetically more accessible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lay0o3odq7Vg90H21EOLACvtfcHk0lhSne4u9Pd-cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D&md5=0df2d6699f97e95b0f04e40ac28c17fe</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci300607r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300607r%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DBioturbo%2520Similarity%2520Searching%253A%2520Combining%2520Chemical%2520and%2520Biological%2520Similarity%2520to%2520Discover%2520Structurally%2520Diverse%2520Bioactive%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D3%26spage%3D692%26epage%3D703%26doi%3D10.1021%2Fci300607r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">Benchmarking of Multivariate Similarity Measures for High-Content Screening Fingerprints in Phenotypic Drug Discovery</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1177/1087057113501390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057113501390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=24045583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1284-1297&issue=10&author=F.+Reisenauthor=X.+Zhangauthor=D.+Gabrielauthor=P.+Selzer&title=Benchmarking+of+Multivariate+Similarity+Measures+for+High-Content+Screening+Fingerprints+in+Phenotypic+Drug+Discovery&doi=10.1177%2F1087057113501390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking of multivariate similarity measures for high-content screening fingerprints in phenotypic drug discovery</span></div><div class="casAuthors">Reisen, Felix; Zhang, Xian; Gabriel, Daniela; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1284-1297, 14</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-content screening (HCS) is a powerful tool for drug discovery being capable of measuring cellular responses to chem. disturbance in a high-throughput manner.  HCS provides an image-based readout of cellular phenotypes, including features such as shape, intensity, or texture in a highly multiplexed and quant. manner.  The corresponding feature vectors can be used to characterize phenotypes and are thus defined as HCS fingerprints.  Systematic analyses of HCS fingerprints allow for objective computational comparisons of cellular responses.  Such comparisons therefore facilitate the detection of different compds. with different phenotypic outcomes from high-throughput HCS campaigns.  Feature selection methods and similarity measures, as a basis for phenotype identification and clustering, are crit. for the quality of such computational analyses.  We systematically evaluated 16 different similarity measures in combination with linear and nonlinear feature selection methods for their potential to capture biol. relevant image features.  Nonlinear correlation-based similarity measures such as Kendall's τ and Spearman's ρ perform well in most evaluation scenarios, outperforming other frequently used metrics (such as the Euclidian distance).  We also present four novel modifications of the connectivity map similarity that surpass the original version, in our expts.  This study provides a basis for generic phenotypic anal. in future HCS campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqccj8_bUpPcbVg90H21EOLACvtfcHk0lhSne4u9Pd-cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitrvF&md5=ecbc36a09e21f2aa317170f1ea7d8045</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1177%2F1087057113501390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113501390%26sid%3Dliteratum%253Aachs%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGabriel%26aufirst%3DD.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DBenchmarking%2520of%2520Multivariate%2520Similarity%2520Measures%2520for%2520High-Content%2520Screening%2520Fingerprints%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26issue%3D10%26spage%3D1284%26epage%3D1297%26doi%3D10.1177%2F1087057113501390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, R. C.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2077-2088&issue=8&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=L.+Tianauthor=R.+C.+Perez&title=Profile-QSAR+2.0%3A+Kinase+Virtual+Screening+Accuracy+Comparable+to+Four-Concentration+IC50s+for+Realistically+Novel+Compounds&doi=10.1021%2Facs.jcim.7b00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Tian, Li; Perez, Rolando C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2077-2088</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While conventional random forest regression (RFR) virtual screening models appear to have excellent accuracy on random held-out test sets, they prove lacking in actual practice.  Anal. of 18 historical virtual screens showed that random test sets are far more similar to their training sets than are the compds. project teams actually order.  A new, cluster-based "realistic" training/test set split, which mirrors the chem. novelty of real-life virtual screens, recapitulates the poor predictive power of RFR models in real projects.  The original Profile-QSAR (pQSAR) method greatly broadened the domain of applicability over conventional models by using as independent variables a profile of activity predictions from all historical assays in a large protein family.  However, the accuracy still fell short of expt. on the realistic test sets.  The improved "pQSAR 2.0" method replaces probabilities of activity from naive Bayes categorical models at several thresholds with predicted IC50s from RFR models.  Unexpectedly, the high accuracy also requires removing the RFR model for the actual assay of interest from the independent variable profile.  With these improvements, pQSAR 2.0 activity predictions are now statistically comparable to medium-throughput 4-concn. IC50 measurements even on the realistic test set.  Beyond the yes/no activity predictions from a typical high-throughput screen (HTS) or conventional virtual screen, these semi-quant. IC50 predictions allow for predicted potency, ligand efficiency, lipophilic efficiency and selectivity against antitargets, greatly facilitating hit list triaging and enabling virtual screening panels such as toxicity panels and overall promiscuity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURHgPoAj7n7Vg90H21EOLACvtfcHk0lhSne4u9Pd-cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF&md5=3e215174b6d4b89f490f79e61c893ee1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DR.%2BC.%26atitle%3DProfile-QSAR%25202.0%253A%2520Kinase%2520Virtual%2520Screening%2520Accuracy%2520Comparable%2520to%2520Four-Concentration%2520IC50s%2520for%2520Realistically%2520Novel%2520Compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D8%26spage%3D2077%26epage%3D2088%26doi%3D10.1021%2Facs.jcim.7b00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR: A Novel Meta-QSAR Method That Combines Activities across the Kinase Family to Accurately Predict Affinity, Selectivity, and Cellular Activity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1942</span>– <span class="NLM_lpage">1956</span>, <span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+A+Novel+Meta-QSAR+Method+That+Combines+Activities+across+the+Kinase+Family+to+Accurately+Predict+Affinity%2C+Selectivity%2C+and+Cellular+Activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0lh_fvI4Ni_bDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520A%2520Novel%2520Meta-QSAR%2520Method%2520That%2520Combines%2520Activities%2520across%2520the%2520Kinase%2520Family%2520to%2520Accurately%2520Predict%2520Affinity%252C%2520Selectivity%252C%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">All-Assay-Max2 PQSAR: Activity Predictions as Accurate as Four-Concentration IC 50 s for 8558 Novartis Assays</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4450</span>– <span class="NLM_lpage">4459</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOqurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=4450-4459&issue=10&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=X.-W.+Zhuauthor=L.+Tianauthor=P.+Mukherjeeauthor=X.+Liu&title=All-Assay-Max2+PQSAR%3A+Activity+Predictions+as+Accurate+as+Four-Concentration+IC+50+s+for+8558+Novartis+Assays&doi=10.1021%2Facs.jcim.9b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">All-Assay-Max2 pQSAR: Activity Predictions as Accurate as Four-Concentration IC50s for 8558 Novartis Assays</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Zhu, Xiang-Wei; Tian, Li; Mukherjee, Prasenjit; Liu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4450-4459</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-quant. structure-activity relationship (pQSAR) is a massively multitask, two-step machine learning method with unprecedented scope, accuracy, and applicability domain.  In step one, a "profile" of conventional single-assay random forest regression models are trained on a very large no. of biochem. and cellular pIC50 assays using Morgan 2 substructural fingerprints as compd. descriptors.  In step two, a panel of partial least squares (PLS) models are built using the profile of pIC50 predictions from those random forest regression models as compd. descriptors (hence the name).  Previously described for a panel of 728 biochem. and cellular kinase assays, we have now built an enormous pQSAR from 11 805 diverse Novartis (NVS) IC50 and EC50 assays.  This large no. of assays, and hence of compd. descriptors for PLS, dictated reducing the profile by only including random forest regression models whose predictions correlate with the assay being modeled.  The random forest regression and pQSAR models were evaluated with our "realistically novel" held-out test set, whose median av. similarity to the nearest training set member across the 11 805 assays was only 0.34, comparable to the novelty of compds. actually selected from virtual screens.  For the 11 805 single-assay random forest regression models, the median correlation of prediction with the expt. was only rext2 = 0.05, virtually random, and only 8% of the models achieved our std. success threshold of rext2 = 0.30.  For pQSAR, the median correlation was rext2 = 0.53, comparable to four-concn. exptl. IC50s, and 72% of the models met our rext2 > 0.30 std., totaling 8558 successful models.  The successful models included assays from all of the 51 annotated target subclasses, as well as 4196 phenotypic assays, indicating that pQSAR can be applied to virtually any disease area.  Every month, all models are updated to include new measurements, and predictions are made for 5.5 million NVS compds., totaling 50 billion predictions.  Common uses have included virtual screening, selectivity design, toxicity and promiscuity prediction, mechanism-of-action prediction, and others.  Several such actual applications are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVUqZKKKZCCbVg90H21EOLACvtfcHk0lh_fvI4Ni_bDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOqurbK&md5=00bbbad59334754948074d6e1e5edccb</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00375%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DZhu%26aufirst%3DX.-W.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DAll-Assay-Max2%2520PQSAR%253A%2520Activity%2520Predictions%2520as%2520Accurate%2520as%2520Four-Concentration%2520IC%252050%2520s%2520for%25208558%2520Novartis%2520Assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D10%26spage%3D4450%26epage%3D4459%26doi%3D10.1021%2Facs.jcim.9b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettles, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span> <span> </span><span class="NLM_article-title">“Bayes Affinity Fingerprints” Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2445</span>– <span class="NLM_lpage">2456</span>, <span class="refDoi"> DOI: 10.1021/ci600197y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600197y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=2445-2456&issue=6&author=A.+Benderauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=Z.+Dengauthor=J.+H.+Nettlesauthor=J.+W.+Davies&title=%E2%80%9CBayes+Affinity+Fingerprints%E2%80%9D+Improve+Retrieval+Rates+in+Virtual+Screening+and+Define+Orthogonal+Bioactivity+Space%3A+When+Are+Multitarget+Drugs+a+Feasible+Concept%3F&doi=10.1021%2Fci600197y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">"Bayes Affinity Fingerprints" Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Glick, Meir; Deng, Zhan; Nettles, James H.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2445-2456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conventional similarity searching of mols. compares single (or multiple) active query structures to each other in a relative framework, by means of a structural descriptor and a similarity measure.  While this often works well, depending on the target, the authors show here that retrieval rates can be improved considerably by incorporating an external framework describing ligand bioactivity space for comparisons ("Bayes affinity fingerprints").  Structures are described by Bayes scores for a ligand panel comprising about 1000 activity classes extd. from the WOMBAT database.  The comparison of structures is performed via the Pearson correlation coeff. of activity classes, i.e., the order in which two structures are similar to the panel activity classes.  Compd. retrieval on a recently published data set could be improved by as much as 24% relative (9% abs.).  Knowledge about the shape of the "bioactive chem. universe" is thus beneficial to identifying similar bioactivities.  Principal component anal. was employed to further analyze activity space with the objective to define orthogonal ligand bioactive chem. space, leading to nine major (roughly orthogonal) activity axes.  Employing only those nine activity classes, retrieval rates are still comparable to original Bayes affinity fingerprints; thus, the concept of orthogonal bioactive ligand chem. space was validated as being an information-rich but low-dimensional representation of bioactivity space.  Correlations between activity classes are a major determinant to gauge whether the desired multitarget activity of drugs is (on the basis of current knowledge) a feasible concept because it measures the extent to which activities can be optimized independently, or only by strongly influencing one another.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6IjlYwYDTwLVg90H21EOLACvtfcHk0lgCMD1xGlzOrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D&md5=47adb720154f45bda1738963d54c70a5</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci600197y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600197y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3D%25E2%2580%259CBayes%2520Affinity%2520Fingerprints%25E2%2580%259D%2520Improve%2520Retrieval%2520Rates%2520in%2520Virtual%2520Screening%2520and%2520Define%2520Orthogonal%2520Bioactivity%2520Space%253A%2520When%2520Are%2520Multitarget%2520Drugs%2520a%2520Feasible%2520Concept%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D6%26spage%3D2445%26epage%3D2456%26doi%3D10.1021%2Fci600197y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span> <span> </span><span class="NLM_article-title">Pfam: A Comprehensive Database of Protein Domain Families Based on Seed Alignments</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199707)28:3<405::AID-PROT10>3.0.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=9223186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1997&pages=405-420&issue=3&author=E.+L.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=R.+Durbin&title=Pfam%3A+A+Comprehensive+Database+of+Protein+Domain+Families+Based+on+Seed+Alignments&doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Pfam: a comprehensive database of protein domain families based on seed alignments</span></div><div class="casAuthors">Sonnhammer, Erik L. L.; Eddy, Sean R.; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-420</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Databases of multiple sequence alignments are a valuable aid to protein sequence classification and anal.  One of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other.  The latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods.  Here, the authors present a database based on hidden Markov model profiles (HMMs), which combines high quality and completeness.  This database, Pfam, consists of parts A and B.  Pfam-A is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and HMMs to find and align all members.  Pfam-B contains sequence families that were generated automatically by applying the Domainer algorithm to cluster and align the remaining protein sequences after removal of Pfam-A domains.  By using Pfam, a large no. of previously unannotated proteins from the Caenorhabditis elegans genome project were classified.  The authors also identified many novel family memberships in known proteins, including new Kazal, fibronectin type III, and response regulator receiver domains.  Pfam-A families have permanent accession nos. and form a library of HMMs available for searching and automatic annotation of new protein sequences.  There are World Wide Web servers at http://www.sanger.ac.uk/Pfam and http://genome.wustl.edu/Pfam, which allow browsing and HMM searching against Pfam-A with a query sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre6VakACw-_rVg90H21EOLACvtfcHk0lgCMD1xGlzOrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D&md5=8fe760b424180de92f0c3816443d2d8d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199707%252928%253A3%253C405%253A%253AAID-PROT10%253E3.0.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DPfam%253A%2520A%2520Comprehensive%2520Database%2520of%2520Protein%2520Domain%2520Families%2520Based%2520on%2520Seed%2520Alignments%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D28%26issue%3D3%26spage%3D405%26epage%3D420%26doi%3D10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Gebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piovesan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tosatto, S. C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. D.</span></span> <span> </span><span class="NLM_article-title">The Pfam Protein Families Database in 2019</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D427</span>– <span class="NLM_lpage">D432</span>, <span class="refDoi"> DOI: 10.1093/nar/gky995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30357350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D427-D432&issue=D1&author=S.+El-Gebaliauthor=J.+Mistryauthor=A.+Batemanauthor=S.+R.+Eddyauthor=A.+Lucianiauthor=S.+C.+Potterauthor=M.+Qureshiauthor=L.+J.+Richardsonauthor=G.+A.+Salazarauthor=A.+Smartauthor=E.+L.+L.+Sonnhammerauthor=L.+Hirshauthor=L.+Paladinauthor=D.+Piovesanauthor=S.+C.+E.+Tosattoauthor=R.+D.+Finn&title=The+Pfam+Protein+Families+Database+in+2019&doi=10.1093%2Fnar%2Fgky995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database in 2019</span></div><div class="casAuthors">El-Gebali, Sara; Mistry, Jaina; Bateman, Alex; Eddy, Sean R.; Luciani, Aurelien; Potter, Simon C.; Qureshi, Matloob; Richardson, Lorna J.; Salazar, Gustavo A.; Smart, Alfredo; Sonnhammer, Erik L. L.; Hirsh, Layla; Paladin, Lisanna; Piovesan, Damiano; Tosatto, Silvio C. E.; Finn, Robert D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D427-D432</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The last few years have witnessed significant changes in Pfam (https://pfam.xfam.org).  The no. of families has grown substantially to a total of 17,929 in release 32.0.  New addns. have been coupled with efforts to improve existing families, including refinement of domain boundaries, their classification into Pfam clans, as well as their functional annotation.  We recently began to collaborate with the RepeatsDB resource to improve the definition of tandem repeat families within Pfam.  We carried out a significant comparison to the structural classification database, namely the Evolutionary Classification of Protein Domains (ECOD) that led to the creation of 825 new families based on their set of uncharacterized families (EUFs).  Furthermore, we also connected Pfam entries to the Sequence Ontol. (SO) through mapping of the Pfam type definitions to SO terms.  Since Pfam has many community contributors, we recently enabled the linking between authorship of all Pfam entries with the corresponding authors' ORCID identifiers.  This effectively permits authors to claim credit for their Pfam curation and link them to their ORCID record.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqONw_dWfZb87Vg90H21EOLACvtfcHk0lgCMD1xGlzOrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7rE&md5=18f34c80b41ffa575898a4a2aff0ec11</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky995%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Gebali%26aufirst%3DS.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DLuciani%26aufirst%3DA.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DQureshi%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DL.%2BJ.%26aulast%3DSalazar%26aufirst%3DG.%2BA.%26aulast%3DSmart%26aufirst%3DA.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%2BL.%26aulast%3DHirsh%26aufirst%3DL.%26aulast%3DPaladin%26aufirst%3DL.%26aulast%3DPiovesan%26aufirst%3DD.%26aulast%3DTosatto%26aufirst%3DS.%2BC.%2BE.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Pfam%2520Protein%2520Families%2520Database%2520in%25202019%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD427%26epage%3DD432%26doi%3D10.1093%2Fnar%2Fgky995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilatovskiy, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Pocketome: An Encyclopedia of Small-Molecule Binding Sites in 4D</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgkr825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=535-540&issue=D1&author=I.+Kufarevaauthor=A.+V.+Ilatovskiyauthor=R.+Abagyan&title=Pocketome%3A+An+Encyclopedia+of+Small-Molecule+Binding+Sites+in+4D&doi=10.1093%2Fnar%2Fgkr825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Pocketome: an encyclopedia of small-molecule binding sites in 4D</span></div><div class="casAuthors">Kufareva, Irina; Ilatovskiy, Andrey V.; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D535-D540</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The importance of binding site plasticity in protein-ligand interactions is well-recognized, and so are the difficulties in predicting the nature and the degree of this plasticity by computational means.  To assist in understanding the flexible protein-ligand interactions, we constructed the Pocketome, an encyclopedia of about one thousand exptl. solved conformational ensembles of druggable binding sites in proteins, grouped by location and consistent chain/cofactor compn.  The multiplicity of pockets within the ensembles adds an extra, fourth dimension to the Pocketome entry data.  Within each ensemble, the pockets were carefully classified by the degree of their pairwise similarity and compatibility with different ligands.  The core of the Pocketome is derived regularly and automatically from the current releases of the Protein Data Bank and the Uniprot Knowledgebase; this core is complemented by entries built from manually provided seed ligand locations.  The Pocketome website (www.pocketome.org) allows searching for the sites of interest, anal. of conformational clusters, important residues, binding compatibility matrixes and interactive visualization of the ensembles using the ActiveICM web browser plugin.  The Pocketome collection can be used to build multi-conformational docking and 3D activity models as well as to design cross-docking and virtual ligand screening benchmarks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDsTtd27fxHrVg90H21EOLACvtfcHk0lhTGQFA8_JD8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbrN&md5=921806bf8bf7dde1e1c89a5ca082142d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr825%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DIlatovskiy%26aufirst%3DA.%2BV.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DPocketome%253A%2520An%2520Encyclopedia%2520of%2520Small-Molecule%2520Binding%2520Sites%2520in%25204D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3D535%26epage%3D540%26doi%3D10.1093%2Fnar%2Fgkr825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Dongen, S.</span></span> <span> </span><span class="NLM_article-title">Graph Clustering via a Discrete Uncoupling Process</span>. <i>SIAM J. Matrix Anal. Appl.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1137/040608635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1137%2F040608635" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=121-141&issue=1&author=S.+Van+Dongen&title=Graph+Clustering+via+a+Discrete+Uncoupling+Process&doi=10.1137%2F040608635"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1137%2F040608635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1137%252F040608635%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDongen%26aufirst%3DS.%26atitle%3DGraph%2520Clustering%2520via%2520a%2520Discrete%2520Uncoupling%2520Process%26jtitle%3DSIAM%2520J.%2520Matrix%2520Anal.%2520Appl.%26date%3D2008%26volume%3D30%26issue%3D1%26spage%3D121%26epage%3D141%26doi%3D10.1137%2F040608635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enright, A. J.</span></span> <span> </span><span class="NLM_article-title">An Efficient Algorithm for Large-Scale Detection of Protein Families</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1575</span>– <span class="NLM_lpage">1584</span>, <span class="refDoi"> DOI: 10.1093/nar/30.7.1575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2F30.7.1575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=11917018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD38Xltlajsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=1575-1584&issue=7&author=A.+J.+Enright&title=An+Efficient+Algorithm+for+Large-Scale+Detection+of+Protein+Families&doi=10.1093%2Fnar%2F30.7.1575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient algorithm for large-scale detection of protein families</span></div><div class="casAuthors">Enright, A. J.; Van Dongen, S.; Ouzounis, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1575-1584</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Detection of protein families in large databases is one of the principal research objectives in structural and functional genomics.  Protein family classification can significantly contribute to the delineation of functional diversity of homologous proteins, the prediction of function based on domain architecture or the presence of sequence motifs as well as comparative genomics, providing valuable evolutionary insights.  We present a novel approach called TRIBE-MCL for rapid and accurate clustering of protein sequences into families.  The method relies on the Markov cluster (MCL) algorithm for the assignment of proteins into families based on precomputed sequence similarity information.  This novel approach does not suffer from the problems that normally hinder other protein sequence clustering algorithms, such as the presence of multi-domain proteins, promiscuous domains and fragmented proteins.  The method has been rigorously tested and validated on a no. of very large databases, including SwissProt, InterPro, SCOP and the draft human genome.  Our results indicate that the method is ideally suited to the rapid and accurate detection of protein families on a large scale.  The method has been used to detect and categorize protein families within the draft human genome and the resulting families have been used to annotate a large proportion of human proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd4C24eKihR7Vg90H21EOLACvtfcHk0lhTGQFA8_JD8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xltlajsrk%253D&md5=54b188a2c209dfd6b943c6a6492f164c</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F30.7.1575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F30.7.1575%26sid%3Dliteratum%253Aachs%26aulast%3DEnright%26aufirst%3DA.%2BJ.%26atitle%3DAn%2520Efficient%2520Algorithm%2520for%2520Large-Scale%2520Detection%2520of%2520Protein%2520Families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26issue%3D7%26spage%3D1575%26epage%3D1584%26doi%3D10.1093%2Fnar%2F30.7.1575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roggo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">The Scaffold Tree – Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&issue=1&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+Scaffold+Tree+%E2%80%93+Visualization+of+the+Scaffold+Universe+by+Hierarchical+Scaffold+Classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0lg8S6O616twhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520Scaffold%2520Tree%2520%25E2%2580%2593%2520Visualization%2520of%2520the%2520Scaffold%2520Universe%2520by%2520Hierarchical%2520Scaffold%2520Classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span> <span> </span><span class="NLM_article-title">Mining for Bioactive Scaffolds with Scaffold Networks: Improved Compound Set Enrichment from Primary Screening Data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1021/ci2000924</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2000924" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFSqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1528-1538&issue=7&author=T.+Varinauthor=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Renner&title=Mining+for+Bioactive+Scaffolds+with+Scaffold+Networks%3A+Improved+Compound+Set+Enrichment+from+Primary+Screening+Data&doi=10.1021%2Fci2000924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Mining for Bioactive Scaffolds with Scaffold Networks: Improved Compound Set Enrichment from Primary Screening Data</span></div><div class="casAuthors">Varin, Thibault; Schuffenhauer, Ansgar; Ertl, Peter; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1538</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of meaningful chem. patterns in the increasing amts. of high-throughput-generated bioactivity data available today is an increasingly important challenge for successful drug discovery.  Herein, we present the scaffold network as a novel approach for mapping and navigation of chem. and biol. space.  A scaffold network represents the chem. space of a library of mols. consisting of all mol. scaffolds and smaller "parent" scaffolds generated therefrom by the pruning of rings, effectively leading to a network of common scaffold substructure relationships.  This algorithm provides an extension of the scaffold tree algorithm that, instead of a network, generates a tree relationship between a heuristically rule-based selected subset of parent scaffolds.  The approach was evaluated for the identification of statistically significantly active scaffolds from primary screening data for which the scaffold tree approach has already been shown to be successful.  Because of the exhaustive enumeration of smaller scaffolds and the full enumeration of relationships between them, about twice as many statistically significantly active scaffolds were identified compared to the scaffold-tree-based approach.  We suggest visualizing scaffold networks as islands of active scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNnYBgKCNoh7Vg90H21EOLACvtfcHk0lg8S6O616twhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFSqsrY%253D&md5=14a0cd277e5f11d002daf6296c29404b</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fci2000924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2000924%26sid%3Dliteratum%253Aachs%26aulast%3DVarin%26aufirst%3DT.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DMining%2520for%2520Bioactive%2520Scaffolds%2520with%2520Scaffold%2520Networks%253A%2520Improved%2520Compound%2520Set%2520Enrichment%2520from%2520Primary%2520Screening%2520Data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26issue%3D7%26spage%3D1528%26epage%3D1538%26doi%3D10.1021%2Fci2000924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&issue=5&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lg8S6O616twhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D5%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fechner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiefl, N.</span></span> <span> </span><span class="NLM_article-title">Chemical Topic Modeling: Exploring Molecular Data Sets Using a Common Text-Mining Approach</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1816</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00249</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00249" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOkt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1816-1831&issue=8&author=N.+Schneiderauthor=N.+Fechnerauthor=G.+A.+Landrumauthor=N.+Stiefl&title=Chemical+Topic+Modeling%3A+Exploring+Molecular+Data+Sets+Using+a+Common+Text-Mining+Approach&doi=10.1021%2Facs.jcim.7b00249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Topic Modeling: Exploring Molecular Data Sets Using a Common Text-Mining Approach</span></div><div class="casAuthors">Schneider, Nadine; Fechner, Nikolas; Landrum, Gregory A.; Stiefl, Nikolaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1816-1831</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Big data is one of the key transformative factors which are increasingly influencing all aspects of modern life.  Although this transformation brings vast opportunities it also generates novel challenges, not the least of which is organizing and searching this data deluge.  The field of medicinal chem. is not different: more and more data are being generated, for instance by technologies such as DNA encoded libraries, peptide libraries, text mining of large literature corpora, and new in silico enumeration methods.  Handling those huge sets of mols. effectively is quite challenging and requires compromises that often come at the expense of the interpretability of the results.  In order to find an intuitive and meaningful approach to organizing large mol. datasets, we adopted a probabilistic framework called "topic modeling" from the text-mining field.  Here, we present the first chem.-related implementation of this method, which allows large mol. sets to be assigned to "chem. topics" and investigating the relationships between those.  In this first study, we thoroughly evaluate this novel method in different expts. and discuss both its disadvantages and advantages.  We show very promising results in reproducing human-assigned concepts by using the approach to identify and retrieve chem. series from sets of mols.  We have also created an intuitive visualization of the chem. topics output by the algorithm.  This is a huge benefit compared to other unsupervised machine-learning methods, like clustering, which are commonly used to group sets of mols.  Finally, we applied the new method to the 1.6 million mols. of the ChEMBL22 dataset to test its robustness and efficiency.  In about 1h we built a 100-topic model of this large dataset in which we could identify interesting topics like "proteins", "DNA" or "steroids".  Along with this publication, we provide our datasets and an open-source implementation of the new method (CheTo) which will be part of an upcoming version of the open-source cheminformatics toolkit RDKit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEAbB6V9wBNrVg90H21EOLACvtfcHk0lgCPQFMHwwcwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOkt7bK&md5=a737f0d5dbb81f02cb2b58269322b2cc</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00249%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DFechner%26aufirst%3DN.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26aulast%3DStiefl%26aufirst%3DN.%26atitle%3DChemical%2520Topic%2520Modeling%253A%2520Exploring%2520Molecular%2520Data%2520Sets%2520Using%2520a%2520Common%2520Text-Mining%2520Approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D8%26spage%3D1816%26epage%3D1831%26doi%3D10.1021%2Facs.jcim.7b00249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, W. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Shape Diversity of Combinatorial Libraries: A Prerequisite for Broad Bioactivity</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">1003</span>, <span class="refDoi"> DOI: 10.1021/ci025599w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025599w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvF2muro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=987-1003&issue=3&author=W.+H.+B.+Sauerauthor=M.+K.+Schwarz&title=Molecular+Shape+Diversity+of+Combinatorial+Libraries%3A+A+Prerequisite+for+Broad+Bioactivity&doi=10.1021%2Fci025599w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape Diversity of Combinatorial Libraries: A Prerequisite for Broad Bioactivity</span></div><div class="casAuthors">Sauer, Wolfgang H. B.; Schwarz, Matthias K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">987-1003</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A computational method to rapidly assess and visualize the diversity in mol. shape assocd. with a given compd. set has been developed.  Normalized ratios of principal moments of inertia are plotted into two-dimensional triangular graphs and then used to compare the shape space covered by different compd. sets, such as combinatorial libraries of varying size and compn.  We have further developed a computational method to analyze interset similarity in terms of shape space coverage, which allows the shape redundancy between the different subsets of a given compd. collection to be analyzed in a quant. way.  The shape space coverage has been found to originate mainly from the nature and the 3D-geometry (but not the size) of the central scaffold, while the no. and nature of the peripheral substituents and conformational aspects were shown to be of minor importance.  Substantial shape space coverage has been correlated with broad biol. activity by applying the same shape anal. to collections of known bioactive compds., such as MDDR and the GOLD-set.  The aggregate of our results corroborates the intuitive notion that mol. shape is intimately linked to biol. activity and that a high degree of shape (hence scaffold) diversity in screening collections will increase the odds of addressing a broad range of biol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKIZLeZoCDh7Vg90H21EOLACvtfcHk0lgCPQFMHwwcwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvF2muro%253D&md5=c07401742457e76b066bed4b43feca1d</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fci025599w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025599w%26sid%3Dliteratum%253Aachs%26aulast%3DSauer%26aufirst%3DW.%2BH.%2BB.%26aulast%3DSchwarz%26aufirst%3DM.%2BK.%26atitle%3DMolecular%2520Shape%2520Diversity%2520of%2520Combinatorial%2520Libraries%253A%2520A%2520Prerequisite%2520for%2520Broad%2520Bioactivity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26issue%3D3%26spage%3D987%26epage%3D1003%26doi%3D10.1021%2Fci025599w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span>FastROCS. <a href="http://www.eyesopen.com/fastrocs" class="extLink">http://www.eyesopen.com/fastrocs</a> (accessed Jun 21, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FastROCS.+http%3A%2F%2Fwww.eyesopen.com%2Ffastrocs+%28accessed+Jun+21%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rush, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosyak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1495</span>, <span class="refDoi"> DOI: 10.1021/jm040163o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040163o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1489-1495&issue=5&author=T.+S.+Rushauthor=J.+A.+Grantauthor=L.+Mosyakauthor=A.+Nicholls&title=A+Shape-Based+3-D+Scaffold+Hopping+Method+and+Its+Application+to+a+Bacterial+Protein-Protein+Interaction&doi=10.1021%2Fjm040163o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span></div><div class="casAuthors">Rush, Thomas S., III; Grant, J. Andrew; Mosyak, Lidia; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1489-1495</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors describe the first prospective application of the shape-comparison program ROCS (Rapid Overlay of Chem. Structures) to find new scaffolds for small mol. inhibitors of the ZipA-FtsZ protein-protein interaction, a proposed antibacterial target.  The shape comparisons are made relative to the crystallog. detd., bioactive conformation of a high-throughput screening (HTS) hit.  The use of ROCS led to the identification of a set of novel, weakly binding inhibitors with scaffolds presenting synthetic opportunities to further optimize biol. affinity and lacking development issues assocd. with the HTS lead.  These ROCS-identified scaffolds would have been missed using other structural similarity approaches such as ISIS 2D fingerprints.  X-ray crystallog. anal. of one of the new inhibitors bound to ZipA reveals that the shape comparison approach very accurately predicted the binding mode.  These exptl. results validate this use of ROCS for chemotype switching or "lead hopping" and suggest that it is of general interest for lead identification in drug discovery endeavors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4EX9jFtct37Vg90H21EOLACvtfcHk0lgCPQFMHwwcwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D&md5=05b6e54a657a3b8c768e63852d871ef6</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm040163o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040163o%26sid%3Dliteratum%253Aachs%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520Shape-Based%25203-D%2520Scaffold%2520Hopping%2520Method%2520and%2520Its%2520Application%2520to%2520a%2520Bacterial%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26issue%3D5%26spage%3D1489%26epage%3D1495%26doi%3D10.1021%2Fjm040163o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&issue=1&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+Shape-Matching+and+Docking+as+Virtual+Screening+Tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lg1JgXMzRiM7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Shape-Matching%2520and%2520Docking%2520as%2520Virtual%2520Screening%2520Tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26issue%3D1%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelliccioli, A.</span>; <span class="NLM_string-name">Chatterjee, R.</span>; <span class="NLM_string-name">Koch, G.</span>; <span class="NLM_string-name">Schuffenhauer, A.</span>; <span class="NLM_string-name">Schwarze, M.</span>; <span class="NLM_string-name">Smith, T. J.</span>; <span class="NLM_string-name">Page, M.</span>; <span class="NLM_string-name">Projects, L.</span></span> <span> </span><span class="NLM_article-title">Compound Series and Favorites: A Medicinal Chemistry Story Telling Tool at Novartis What the Data Warehouse Knows</span>. Presented at the Seventh Joint Sheffield Conference on Chemoinformatics, Sheffield, U.K., July 4–6, <span class="NLM_year">1996</span>; Poster 27.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=A.+Pelliccioli&author=R.+Chatterjee&author=G.+Koch&author=A.+Schuffenhauer&author=M.+Schwarze&author=T.+J.+Smith&author=M.+Page&author=L.+Projects&title=Compound+Series+and+Favorites%3A+A+Medicinal+Chemistry+Story+Telling+Tool+at+Novartis+What+the+Data+Warehouse+Knows"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPelliccioli%26aufirst%3DA.%26atitle%3DCompound%2520Series%2520and%2520Favorites%253A%2520A%2520Medicinal%2520Chemistry%2520Story%2520Telling%2520Tool%2520at%2520Novartis%2520What%2520the%2520Data%2520Warehouse%2520Knows%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baier, S.</span></span> <span> </span><span class="NLM_article-title">How Many Molecules Does It Take to Tell a Story? Case Studies, Language, and an Epistemic View of Medicinal Chemistry</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.201500091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25820774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVKlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=949-956&issue=6&author=M.+Stahlauthor=S.+Baier&title=How+Many+Molecules+Does+It+Take+to+Tell+a+Story%3F+Case+Studies%2C+Language%2C+and+an+Epistemic+View+of+Medicinal+Chemistry&doi=10.1002%2Fcmdc.201500091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">How Many Molecules Does It Take to Tell a Story? Case Studies, Language, and an Epistemic View of Medicinal Chemistry</span></div><div class="casAuthors">Stahl, Martin; Baier, Sabine</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-956</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Medicinal chem. has always been closer to the arts than other disciplines in the natural sciences.  Instead of searching for natural laws, medicinal chem. creates new mol. entities entailing desired pharmaceutical characteristics.  While the productive output of medicinal chem. is comprehensively documented, the epistemic paths of the creative process are less well described.  Here we show how such paths could be visualized and how these visualizations relate to images developed in the history and philosophy of science.  Based on the discussion of these visualizations, we argue that there is a need for a new language of creativity that can be employed during the very course of research, as opposed to its retrospective anal.  This language should be able to reflect both the status and directions in highly complex research processes that may have a clear goal, yet must remain open to unexpected moments of serendipity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLajk6L_irtbVg90H21EOLACvtfcHk0lg1JgXMzRiM7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVKlsL4%253D&md5=00e7fcd5e2b7ca7289c565fcc7c5a06b</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500091%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DBaier%26aufirst%3DS.%26atitle%3DHow%2520Many%2520Molecules%2520Does%2520It%2520Take%2520to%2520Tell%2520a%2520Story%253F%2520Case%2520Studies%252C%2520Language%252C%2520and%2520an%2520Epistemic%2520View%2520of%2520Medicinal%2520Chemistry%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26issue%3D6%26spage%3D949%26epage%3D956%26doi%3D10.1002%2Fcmdc.201500091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbach-Weese, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruce, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritsch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aichholz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caravatti, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of NVP-BYL719 a Potent and Selective Phosphatidylinositol-3 Kinase Alpha Inhibitor Selected for Clinical</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3741</span>– <span class="NLM_lpage">3748</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmcl.2013.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23726034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3741-3748&issue=13&author=P.+Furetauthor=V.+Guagnanoauthor=R.+A.+Fairhurstauthor=P.+Imbach-Weeseauthor=I.+Bruceauthor=M.+Knappauthor=C.+Fritschauthor=F.+Blascoauthor=J.+Blanzauthor=R.+Aichholzauthor=J.+Hamonauthor=D.+Fabbroauthor=G.+Caravatti&title=Discovery+of+NVP-BYL719+a+Potent+and+Selective+Phosphatidylinositol-3+Kinase+Alpha+Inhibitor+Selected+for+Clinical&doi=10.1016%2Fj.bmcl.2013.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of NVP-BYL719 a potent and selective phosphatidylinositol-3 kinase alpha inhibitor selected for clinical evaluation</span></div><div class="casAuthors">Furet, Pascal; Guagnano, Vito; Fairhurst, Robin A.; Imbach-Weese, Patricia; Bruce, Ian; Knapp, Mark; Fritsch, Christine; Blasco, Francesca; Blanz, Joachim; Aichholz, Reiner; Hamon, Jacques; Fabbro, Doriano; Caravatti, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3741-3748</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphatidylinositol-3-kinase α (PI3Kα) is a therapeutic target of high interest in anticancer drug research.  On the basis of a binding model rationalizing the high selectivity and potency of a particular series of 2-aminothiazole compds. in inhibiting PI3Kα, a medicinal chem. program has led to the discovery of the clin. candidate NVP-BYL719 I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraN93Vwrfb4bVg90H21EOLACvtfcHk0lhGh_Nqr8qz8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFaisrY%253D&md5=a0d2d69e01e7a82a5fed04114105925e</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DImbach-Weese%26aufirst%3DP.%26aulast%3DBruce%26aufirst%3DI.%26aulast%3DKnapp%26aufirst%3DM.%26aulast%3DFritsch%26aufirst%3DC.%26aulast%3DBlasco%26aufirst%3DF.%26aulast%3DBlanz%26aufirst%3DJ.%26aulast%3DAichholz%26aufirst%3DR.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DCaravatti%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520NVP-BYL719%2520a%2520Potent%2520and%2520Selective%2520Phosphatidylinositol-3%2520Kinase%2520Alpha%2520Inhibitor%2520Selected%2520for%2520Clinical%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26issue%3D13%26spage%3D3741%26epage%3D3748%26doi%3D10.1016%2Fj.bmcl.2013.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benz, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamidass, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldi, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Power-Laws in Chemical Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi"> DOI: 10.1021/ci700353m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700353m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXms1Gitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1138-1151&issue=6&author=R.+W.+Benzauthor=S.+J.+Swamidassauthor=P.+Baldi&title=Discovery+of+Power-Laws+in+Chemical+Space&doi=10.1021%2Fci700353m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of power-laws in chemical space</span></div><div class="casAuthors">Benz, Ryan W.; Swamidass, S. Joshua; Baldi, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1138-1151</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Power-law distributions have been obsd. in a wide variety of areas.  To our knowledge however, there has been no systematic observation of power-law distributions in chemoinformatics.  Here, we present several examples of power-law distributions arising from the features of small, org. mols.  The distributions of rigid segments and ring systems, the distributions of mol. paths and circular substructures, and the sizes of mol. similarity clusters all show linear trends on log-log rank/frequency plots, suggesting underlying power-law distributions.  The no. of unique features also follow Heaps'-like laws.  The characteristic exponents of the power-laws lie in the 1.5-3 range, consistently with the exponents obsd. in other power-law phenomena.  The power-law nature of these distributions leads to several applications including the prediction of the growth of available data through Heaps' law and the optimal allocation of exptl. or computational resources via the 80/20 rule.  More importantly, we also show how the power-laws can be leveraged to efficiently compress chem. fingerprints in a lossless manner, useful for the improved storage and retrieval of mols. in large chem. databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovyRIpleieC7Vg90H21EOLACvtfcHk0lhGh_Nqr8qz8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXms1Gitb4%253D&md5=111cda24a98cae8e6a1dcf672b20c942</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fci700353m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700353m%26sid%3Dliteratum%253Aachs%26aulast%3DBenz%26aufirst%3DR.%2BW.%26aulast%3DSwamidass%26aufirst%3DS.%2BJ.%26aulast%3DBaldi%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Power-Laws%2520in%2520Chemical%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26issue%3D6%26spage%3D1138%26epage%3D1151%26doi%3D10.1021%2Fci700353m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Rethinking Screening</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1038/nchembio.303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20154660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=162-165&issue=3&author=T.+Kodadek&title=Rethinking+Screening&doi=10.1038%2Fnchembio.303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking screening</span></div><div class="casAuthors">Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">162-165</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bioactive compds. are most frequently identified via high-throughput screening campaigns.  This article discusses the strengths and weaknesses of the most popular screening approaches and the utility of compds. derived from them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTJ8nq3RnwQrVg90H21EOLACvtfcHk0lhGh_Nqr8qz8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGmt78%253D&md5=ccd1f4f86d54d44294dc2e3af2c28199</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.303%26sid%3Dliteratum%253Aachs%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DRethinking%2520Screening%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26issue%3D3%26spage%3D162%26epage%3D165%26doi%3D10.1038%2Fnchembio.303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, W. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Which Aspects of HTS Are Empirically Correlated with Downstream Success?</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">337</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=18428086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=327-337&issue=3&author=A.+Benderauthor=D.+Bojanicauthor=J.+W.+Daviesauthor=T.+J.+Crismanauthor=D.+Mikhailovauthor=J.+Scheiberauthor=J.+L.+Jenkinsauthor=Z.+Dengauthor=W.+A.+G.+Hillauthor=M.+Popovauthor=E.+Jacobyauthor=M.+Glick&title=Which+Aspects+of+HTS+Are+Empirically+Correlated+with+Downstream+Success%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Which aspects of HTS are empirically correlated with downstream success?</span></div><div class="casAuthors">Bender, Andreas; Bojanic, Dejan; Davies, John W.; Crisman, Thomas J.; Mikhailov, Dmitri; Scheiber, Josef; Jenkins, Jeremy L.; Deng, Zhan; Hill, W. Adam G.; Popov, Maxim; Jacoby, Edgar; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established hit-finding approach used in the pharmaceutical industry.  In this article, recent experience at Novartis with respect to factors influencing the success of HTS campaigns is discussed.  An inherent measure of HTS quality could be defined by the assay Z and Z' factors, the no. of hits and their biol. potencies; however, such measures of quality do not always correlate with the advancement of hits to the later stages of drug discovery.  Also, for many target classes, such as kinases, it is easy to identify hits, but, as a result of selectivity, intellectual property and other issues, the projects do not result in lead declarations.  In this article, HTS success is defined as the fraction of HTS campaigns that advance into the later stages of drug discovery, and the major influencing factors are examd.  Interestingly, screening compds. in individual wells or in mixts. did not have a major impact on the HTS success and, equally interesting, there was no difference in the progression rates of biochem. and cell-based assays.  Particular target types, assay technologies, structure-activity relationships and powder availability had a much greater impact on success as defined above.  In addn., significant mutual dependencies can be obsd. - while one assay format works well with one target type, this situation might be completely reversed for a combination of the same readout technol. with a different target type.  The results and opinions presented here should be regarded as groundwork, and a plethora of factors that influence the fate of a project, such as biophys. measurements, chem. attractiveness of the hits, strategic reasons and safety pharmacol., are not covered here.  Nonetheless, it is hoped that this information will be used industry-wide to improve success rates in terms of hits progressing into exploratory chem. and beyond.  The support that can be obtained from new in silico approaches to phase transitions are also described, along with the gaps they are designed to fill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqddDbeDZXfNbVg90H21EOLACvtfcHk0lhGh_Nqr8qz8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D&md5=c65c4142d449fe752b37c397cd2a941e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DHill%26aufirst%3DW.%2BA.%2BG.%26aulast%3DPopov%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DWhich%2520Aspects%2520of%2520HTS%2520Are%2520Empirically%2520Correlated%2520with%2520Downstream%2520Success%253F%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26issue%3D3%26spage%3D327%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span> <span> </span><span class="NLM_article-title">Designing Focused Libraries Using MoSELECT</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(01)00150-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2FS1093-3263%2801%2900150-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=12071283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVSmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2002&pages=491-498&issue=6&author=V.+J.+Gilletauthor=P.+Willettauthor=P.+J.+Flemingauthor=D.+V.+S.+Green&title=Designing+Focused+Libraries+Using+MoSELECT&doi=10.1016%2FS1093-3263%2801%2900150-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Designing focused libraries using MoSELECT</span></div><div class="casAuthors">Gillet, Valerie J.; Willett, Peter; Fleming, Peter J.; Green, Darren V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">When designing a combinatorial library it is usually desirable to optimize multiple properties of the library simultaneously and often the properties are in competition with one another.  For example, a library that is designed to be focused around a given target mol. should ideally have min. cost and also contain mols. that are bioavailable.  In this paper, we describe the program MoSELECT for multiobjective library design that is based on a multiobjective genetic algorithm (MOGA).  MoSELECT searches the product-space of a virtual combinatorial library to generate a family of equiv. solns. where each soln. represents a combinatorial subset of the virtual library optimized over multiple objectives.  The family of solns. allows the relationships between the objectives to be explored and thus enables the library designer to make an informed choice on an appropriate compromise soln.  Expts. are reported where MoSELECT has been applied to the design of various focused libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0uN-S5AYd37Vg90H21EOLACvtfcHk0ljMGjXiOY-y-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVSmtb8%253D&md5=ce3ea314059eec39761b0289c1edd53c</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2801%2900150-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252801%252900150-4%26sid%3Dliteratum%253Aachs%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DFleming%26aufirst%3DP.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26atitle%3DDesigning%2520Focused%2520Libraries%2520Using%2520MoSELECT%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2002%26volume%3D20%26issue%3D6%26spage%3D491%26epage%3D498%26doi%3D10.1016%2FS1093-3263%2801%2900150-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannas, C. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Library Design Using Multi-Objective Optimization Methods</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>685</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1007/978-1-60761-931-4_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1007%2F978-1-60761-931-4_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20981518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSrs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=685&publication_year=2011&pages=53-69&author=C.+A.+Nicolaouauthor=C.+C.+Kannas&title=Molecular+Library+Design+Using+Multi-Objective+Optimization+Methods&doi=10.1007%2F978-1-60761-931-4_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular library design using multi-objective optimization methods</span></div><div class="casAuthors">Nicolaou, Christos A.; Kannas, Christos C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">685</span>
        (<span class="NLM_cas:issue">Chemical Library Design</span>),
    <span class="NLM_cas:pages">53-69</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Advancements in combinatorial chem. and high-throughput screening technol. have enabled the synthesis and screening of large mol. libraries for the purposes of drug discovery.  Contrary to initial expectations, the increase in screening library size, typically combined with an emphasis on compd. structural diversity, did not result in a comparable increase in the no. of promising hits found.  In an effort to improve the likelihood of discovering hits with greater optimization potential, more recent approaches attempt to incorporate addnl. knowledge to the library design process to effectively guide the search.  Multi-objective optimization methods capable of taking into account several chem. and biol. criteria have been used to design collections of compds. satisfying simultaneously multiple pharmaceutically relevant objectives.  In this chapter, we present our efforts to implement a multi-objective optimization method, MEGALib, custom-designed to the library design problem.  The method exploits existing knowledge, e.g. from previous biol. screening expts., to identify and profile mol. fragments used subsequently to design compds. compromising the various objectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav94mSUm1G7Vg90H21EOLACvtfcHk0ljMGjXiOY-y-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSrs7rF&md5=675d0bf691152566029b6501cca47d19</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-931-4_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-931-4_3%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DC.%2BA.%26aulast%3DKannas%26aufirst%3DC.%2BC.%26atitle%3DMolecular%2520Library%2520Design%2520Using%2520Multi-Objective%2520Optimization%2520Methods%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D685%26spage%3D53%26epage%3D69%26doi%3D10.1007%2F978-1-60761-931-4_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Improving the Plausibility of Success with Inefficient Metrics</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">5</span>, <span class="refDoi"> DOI: 10.1021/ml4004638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4004638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=2-5&issue=1&author=M.+D.+Shultz&title=Improving+the+Plausibility+of+Success+with+Inefficient+Metrics&doi=10.1021%2Fml4004638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Plausibility of Success with Inefficient Metrics</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-5</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  To increase the probability of success in drug discovery, the concept of drug-like properties was introduced.  Efficiency metrics that normalize potency against these properties could help reach drug-like space more efficiently.  Potential reasons for the inefficient use of metrics and suboptimal decision making are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUcqughsg0LVg90H21EOLACvtfcHk0ljMGjXiOY-y-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtLjP&md5=de1493d6fa4e894b078b8b020ad4b3eb</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fml4004638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4004638%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DImproving%2520the%2520Plausibility%2520of%2520Success%2520with%2520Inefficient%2520Metrics%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D1%26spage%3D2%26epage%3D5%26doi%3D10.1021%2Fml4004638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhnke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneckener, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span> <span> </span><span class="NLM_article-title">Bayer’s in Silico ADMET Platform: A Journey of Machine Learning over the Past Two Decades</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2020.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=32652309" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+H.+G%C3%B6llerauthor=L.+Kuhnkeauthor=F.+Montanariauthor=A.+Boninauthor=S.+Schneckenerauthor=A.+ter+Laakauthor=J.+Wichardauthor=M.+Lobellauthor=A.+Hillisch&title=Bayer%E2%80%99s+in+Silico+ADMET+Platform%3A+A+Journey+of+Machine+Learning+over+the+Past+Two+Decades&doi=10.1016%2Fj.drudis.2020.07.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DKuhnke%26aufirst%3DL.%26aulast%3DMontanari%26aufirst%3DF.%26aulast%3DBonin%26aufirst%3DA.%26aulast%3DSchneckener%26aufirst%3DS.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DWichard%26aufirst%3DJ.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26atitle%3DBayer%25E2%2580%2599s%2520in%2520Silico%2520ADMET%2520Platform%253A%2520A%2520Journey%2520of%2520Machine%2520Learning%2520over%2520the%2520Past%2520Two%2520Decades%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26doi%3D10.1016%2Fj.drudis.2020.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Pursuing the Leadlikeness Concept in Pharmaceutical Research</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2004.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.cbpa.2004.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=15183323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkslWqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=255-263&issue=3&author=M.+M.+Hannauthor=T.+I.+Oprea&title=Pursuing+the+Leadlikeness+Concept+in+Pharmaceutical+Research&doi=10.1016%2Fj.cbpa.2004.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Pursuing the leadlikeness concept in pharmaceutical research</span></div><div class="casAuthors">Hann, Mike M.; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-263</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Lipinski and others, through concepts such as drug-likeness, re-focussed drug discovery back to the principles of medicinal chem. in the high-throughput era as key to reducing attrition.  More recently, the need to go further in defining what makes a good lead has been recognized with the concept of leadlikeness.  Leadlikeness implies cut-off values in the physico-chem. profile of chem. libraries such that they have reduced complexity (e.g. MW below <400) and other more restricted properties.  We examine these concepts in the context of Virtual (theor. possible), Tangible (chem. feasible) and Real (phys. available) worlds of mols.  In a thought expt., we take the HTS concept to the extreme: screening an estd. 60 million Global Collection' on 5000 targets and realizing that perhaps millions of drug candidates might be found that could not possibly be handled in reality.  Sampling of the Virtual and Tangible worlds is therefore a necessity.  We show that the world of Reals is significantly under-sampled as the MW of compds. increases.  This supports the design and screening of reduced complexity' (leadlike) compd. libraries, preferably with synthetic handles available for rapid chem. iteration and detected as interesting by careful screening or biophys. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd8ecCArzZ2bVg90H21EOLACvtfcHk0lgfVl3qNSsfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkslWqsrk%253D&md5=c78783585a60d3deee3b0513fe7300e9</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DPursuing%2520the%2520Leadlikeness%2520Concept%2520in%2520Pharmaceutical%2520Research%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26issue%3D3%26spage%3D255%26epage%3D263%26doi%3D10.1016%2Fj.cbpa.2004.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of Five in 2015 and beyond: Target and Ligand Structural Limitations, Ligand Chemistry Structure and Drug Discovery Project Decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+Five+in+2015+and+beyond%3A+Target+and+Ligand+Structural+Limitations%2C+Ligand+Chemistry+Structure+and+Drug+Discovery+Project+Decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lgfVl3qNSsfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520Five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520Ligand%2520Structural%2520Limitations%252C%2520Ligand%2520Chemistry%2520Structure%2520and%2520Drug%2520Discovery%2520Project%2520Decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deursen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddigkeit, L.</span></span> <span> </span><span class="NLM_article-title">Chemical Space as a Source for New Drugs</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1039/c0md00020e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1039%2Fc0md00020e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeju7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=30-38&issue=1&author=J.+L.+Reymondauthor=R.+Van+Deursenauthor=L.+C.+Blumauthor=L.+Ruddigkeit&title=Chemical+Space+as+a+Source+for+New+Drugs&doi=10.1039%2Fc0md00020e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space as a source for new drugs</span></div><div class="casAuthors">Reymond, Jean-Louis; van Deursen, Ruud; Blum, Lorenz C.; Ruddigkeit, Lars</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-38</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The chem. space is the ensemble of all possible mols., which is believed to contain at least 1060 org. mols. below 500 Da of possible interest for drug discovery.  This review summarizes the development of the chem. space concept from enumerating acyclic hydrocarbons in the 1800's to the recent assembly of the chem. universe database GDB.  Chem. space travel algorithms can be used to explore defined regions of chem. space by generating focused virtual libraries.  Maps of the chem. space are produced from property spaces visualized by principal component anal. or by self-organizing maps, and from structural analyses such as the scaffold-tree or the MQN-system.  Virtual screening of virtual chem. space followed by synthesis and testing of the best hits leads to the discovery of new drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdkwSU4qu1irVg90H21EOLACvtfcHk0lgfVl3qNSsfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeju7rF&md5=579428edc79f706101ab58c505052c72</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1039%2Fc0md00020e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00020e%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DRuddigkeit%26aufirst%3DL.%26atitle%3DChemical%2520Space%2520as%2520a%2520Source%2520for%2520New%2520Drugs%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26issue%3D1%26spage%3D30%26epage%3D38%26doi%3D10.1039%2Fc0md00020e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">The Chemical Space Project</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1021/ar500432k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500432k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=722-730&issue=3&author=J.-L.+Reymond&title=The+Chemical+Space+Project&doi=10.1021%2Far500432k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemical Space Project</span></div><div class="casAuthors">Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-730</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the simplest questions that can be asked about mol. diversity is how many org. mols. are possible in total.  To answer this question, my research group has computationally enumerated all possible org. mols. up to a certain size to gain an unbiased insight into the entire chem. space.  Our latest database, GDB-17, contains 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens, by far the largest small mol. database reported to date.  Mols. allowed by valency rules but unstable or nonsynthesizable due to strained topologies or reactive functional groups were not considered, which reduced the enumeration by at least 10 orders of magnitude and was essential to arrive at a manageable database size.  Despite these restrictions, GDB-17 is highly relevant with respect to known mols.  Beyond enumeration, understanding and exploiting GDBs (generated databases) led us to develop methods for virtual screening and visualization of very large databases in the form of a "periodic system of mols." comprising six different fingerprint spaces, with web-browsers for nearest neighbor searches, and the MQN- and SMIfp-Mapplet application for exploring color-coded principal component maps of GDB and other large databases.  Proof-of-concept applications of GDB for drug discovery were realized by combining virtual screening with chem. synthesis and activity testing for neurotransmitter receptor and transporter ligands.  One surprising lesson from using GDB for drug analog searches is the incredible depth of chem. space, i.e., the fact that millions of very close analogs of any mol. can be readily identified by nearest-neighbor searches in the MQN-space of the various GDBs.  The chem. space project has opened an unprecedented door on chem. diversity.  Ongoing and yet unmet challenges concern enumerating mols. beyond 17 atoms and synthesizing GDB mols. with innovative scaffolds and pharmacophores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxJYiz7ZE3ALVg90H21EOLACvtfcHk0ljVVk4LhIdIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D&md5=d3199b2273a0e8d92a4b1ffa59c6dffc</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Far500432k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500432k%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3DThe%2520Chemical%2520Space%2520Project%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26issue%3D3%26spage%3D722%26epage%3D730%26doi%3D10.1021%2Far500432k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span> <span> </span><span class="NLM_article-title">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&issue=3&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+Complexity+and+Its+Impact+on+the+Probability+of+Finding+Leads+for+Drug+Discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0ljVVk4LhIdIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520Complexity%2520and%2520Its%2520Impact%2520on%2520the%2520Probability%2520of%2520Finding%2520Leads%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26issue%3D3%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenjans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Increasing the Delivery of next Generation Therapeutics from High Throughput Screening Libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2015.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.cbpa.2015.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25909818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=104-110&author=M.+J.+Wigglesworthauthor=D.+C.+Murrayauthor=C.+J.+Blackettauthor=M.+Kossenjansauthor=J.+W.+M.+Nissink&title=Increasing+the+Delivery+of+next+Generation+Therapeutics+from+High+Throughput+Screening+Libraries&doi=10.1016%2Fj.cbpa.2015.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span></div><div class="casAuthors">Wigglesworth, Mark J.; Murray, David C.; Blackett, Carolyn J.; Kossenjans, Michael; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-110</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry has historically relied on high throughput screening as a cornerstone to identify chem. equity for drug discovery projects.  However, with pharmaceutical companies moving through a phase of diminished returns and alternative hit identification strategies proving successful, it is more important than ever to understand how this approach can be used more effectively to increase the delivery of next generation therapeutics from high throughput screening libraries.  There is a wide literature that describes HTS and fragment based screening approaches which offer clear direction on the process for these two distinct activities.  However, few people have considered how best to identify medium to low mol. wt. compds. from large diversity screening sets and increase downstream success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFeNO1FIRBLVg90H21EOLACvtfcHk0ljVVk4LhIdIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D&md5=4b4bd0e7992796bd208a99dcb099c785</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2015.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DD.%2BC.%26aulast%3DBlackett%26aufirst%3DC.%2BJ.%26aulast%3DKossenjans%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DIncreasing%2520the%2520Delivery%2520of%2520next%2520Generation%2520Therapeutics%2520from%2520High%2520Throughput%2520Screening%2520Libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D26%26spage%3D104%26epage%3D110%26doi%3D10.1016%2Fj.cbpa.2015.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span>FASTLab. <a href="https://github.com/Novartis/MoaBox" class="extLink">https://github.com/Novartis/MoaBox</a> (accessed Jul 23, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FASTLab.+https%3A%2F%2Fgithub.com%2FNovartis%2FMoaBox+%28accessed+Jul+23%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Paul D. Leeson, A. Patricia Bento, Anna Gaulton, Anne Hersey, Emma J. Manners, Chris J. Radoux, <span class="NLM_string-name hlFld-ContribAuthor">Andrew R. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Target-Based Evaluation of “Drug-Like” Properties and Ligand Efficiencies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7210-7230. <a href="https://doi.org/10.1021/acs.jmedchem.1c00416" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00416%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget-Based%252BEvaluation%252Bof%252B%2525E2%252580%25259CDrug-Like%2525E2%252580%25259D%252BProperties%252Band%252BLigand%252BEfficiencies%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D13052021%26volume%3D64%26issue%3D11%26spage%3D7210%26epage%3D7230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neann  Mathai</span>, <span class="hlFld-ContribAuthor ">Conrad  Stork</span>, <span class="hlFld-ContribAuthor ">Johannes  Kirchmair</span>. </span><span class="cited-content_cbyCitation_article-title">BonMOLière: Small-Sized Libraries of Readily Purchasable Compounds, Optimized to Produce Genuine Hits in Biological Screens across the Protein Space. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (15)
                                     , 7773. <a href="https://doi.org/10.3390/ijms22157773" title="DOI URL">https://doi.org/10.3390/ijms22157773</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22157773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22157773%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBonMOLi%2525C3%2525A8re%25253A%252BSmall-Sized%252BLibraries%252Bof%252BReadily%252BPurchasable%252BCompounds%25252C%252BOptimized%252Bto%252BProduce%252BGenuine%252BHits%252Bin%252BBiological%252BScreens%252Bacross%252Bthe%252BProtein%252BSpace%26aulast%3DMathai%26aufirst%3DNeann%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D15%26spage%3D7773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Graff</span>, <span class="hlFld-ContribAuthor ">Eugene I.  Shakhnovich</span>, <span class="hlFld-ContribAuthor ">Connor W.  Coley</span>. </span><span class="cited-content_cbyCitation_article-title">Accelerating high-throughput virtual screening through molecular pool-based active learning. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (22)
                                     , 7866-7881. <a href="https://doi.org/10.1039/D0SC06805E" title="DOI URL">https://doi.org/10.1039/D0SC06805E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC06805E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC06805E%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DAccelerating%252Bhigh-throughput%252Bvirtual%252Bscreening%252Bthrough%252Bmolecular%252Bpool-based%252Bactive%252Blearning%26aulast%3DGraff%26aufirst%3DDavid%2BE.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D22%26spage%3D7866%26epage%3D7881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shin  Numao</span>, <span class="hlFld-ContribAuthor ">Gianluca  Etienne</span>, <span class="hlFld-ContribAuthor ">Goran  Malojcic</span>, <span class="hlFld-ContribAuthor ">Enrico  Schmidt</span>, <span class="hlFld-ContribAuthor ">Christoph E.  Dumelin</span>. </span><span class="cited-content_cbyCitation_article-title">Compound Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-820472-6.00078-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-820472-6.00078-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820472-6.00078-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820472-6.00078-5%26sid%3Dliteratum%253Aachs%26atitle%3DCompound%252BScreening%26aulast%3DNumao%26aufirst%3DShin%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BBiomedical%252BSciences%26date%3D2021%26volume%3D107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0014.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Screening deck design at NIBR as an iterative process. The thick arrows indicate the main design cycle discussed in this manuscript, describing how choices are made during the design of plate-sets when the screening deck is due for renewal. At this time the available screening data are analyzed to define the high level design strategy, which will then be executed exploiting all available chemical matter and data. Besides the design strategy, the composition of the screening deck is influenced by the composition of the underlying compound bank. Effects influencing the compound bank are indicated by the thinner arrows. The learning from screening data analysis does not only influence the immediate plate design but also influence the continuous efforts of compound bank enhancement by library design and synthesis. Furthermore, adding a compound to a plate set screened preferentially may lead to faster depletion of that sample. And finally, medicinal chemistry projects generate compound samples that may be chosen for the screening deck, without otherwise being influenced by the screening deck design process described here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound management setup prior to 2015 designed for full deck HTS. Compounds are solubilized and plated at 2 mM for screening in order of arrival. This prevented removal of undesired samples and defining the distribution of compounds across screening plates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound management setup designed for diversity-driven subset screening since 2015. The HT-picking capability of the tube store allows replating the screening deck every 3–4 years in an organized fashion as well as providing large subsets for target or pathway driven selections.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Diversity grid generation: (1) all compounds are annotated with property flags (positive or warning flags; see below for flag description); (2) all compounds are allocated to diversity classes; (3) compounds are selected and assigned to the grid cells by picking the most attractive compound from each class (compound A or B over H, G, J, or K in class 1) and further giving preference to compounds covering as many diversity classes as possible (compound A over B for class 1, since A also covers class 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Exemplary filter (taken originally from ref <a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a>) to highlight the complexity of a SMARTS representation (top). The filter is visualized in a chemically more intuitive way using the SMARTSViewer<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (middle). Different matches of this filter to molecules found in the PubChem version 2018 database<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> are shown with the matching substructure highlighted (bottom). (b) Distribution of the different severity levels within the 444 filters (left). Number of matches of the different filters on the molecules in the PubChem database separated by severity level (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Exclusion reasons and warning flags assigned to the compound samples considered for the 2019 screening deck (multiple flags can apply to one sample).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Web UI of the Compound Series & Favorites Tool. Each series has a shorthand panel showing the last five elements added. The series can be expanded into the full network view visualizing the relationship between the individual elements and a status flag indicating the series status. In addition to individual compounds, a series can contain also scaffold substructure queries. The example in this case has been taken from Furet et al.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Series members can be marked with tags such as “starting point” indicating the compound to be the chemical seed of a series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Flowchart of the selection algorithm. The fraction of compounds at which picking is to be stopped is a parameter of the algorithm, which can be set according to practical purposes. As a fine-grained subset structure for the second half of the full deck is typically not needed, a value of 0.5 is sufficient here. When several compounds of a class tie for the top rank, the compound annotated with the most class types gets precedence. If that does not break the tie, a random number assigned to each compound is used to break the tie.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Molecular weight profile of the 2015 and 2019 deck versions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. clogP profiles of the subsets of the 2015 and 2019 deck versions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Frequent hitter analysis as density plots for hit counts versus total screen count for 2015 and 2019 screening deck per plate subset. In this plot frequent hitters<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> are located in the upper right area, whereas dark chemical matter<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> is located to the right side at the bottom, whereas compounds on the left side have not yet been exposed to a sufficient number of assays to make any conclusions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Influence of the class types on the selection process. For each grid cell the number of classes that triggered the selection of compounds in this cell is shown. As a compound may be selected simultaneously by multiple classes, these numbers do not add up to the total number of compounds per grid cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/medium/jm0c01332_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Molecular weight and log <i>P</i> grid for the SSEC collection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01332/20201203/images/large/jm0c01332_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01332&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38882" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38882" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 107 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wildey, M. J.</span>; <span class="NLM_string-name">Haunso, A.</span>; <span class="NLM_string-name">Tudor, M.</span>; <span class="NLM_string-name">Webb, M.</span>; <span class="NLM_string-name">Connick, J. H.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Screening</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier Inc.</span>, <span class="NLM_year">2017</span>; Vol.  <span class="NLM_volume">50</span>, pp  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/bs.armc.2017.08.004</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fbs.armc.2017.08.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=149-195&author=M.+J.+Wildey&author=A.+Haunso&author=M.+Tudor&author=M.+Webb&author=J.+H.+Connickauthor=R.+A.+Goodnow&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26atitle%3DHigh-Throughput%2520Screening%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DElsevier%2520Inc%26date%3D2017%26volume%3D50%26spage%3D149%26epage%3D195%26doi%3D10.1016%2Fbs.armc.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirovic, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garyantes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzberg, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paslay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G. S.</span></span> <span> </span><span class="NLM_article-title">Impact of High-Throughput Screening in Biomedical Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+High-Throughput+Screening+in+Biomedical+Research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0ljXUr7gXREu3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520High-Throughput%2520Screening%2520in%2520Biomedical%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">The Discovery of First-in-Class Drugs: Origins and Evolution</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&issue=8&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+Discovery+of+First-in-Class+Drugs%3A+Origins+and+Evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0ljCGFXxasv2pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520Discovery%2520of%2520First-in-Class%2520Drugs%253A%2520Origins%2520and%2520Evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26issue%3D8%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span> <span> </span><span class="NLM_article-title">Encoded Library Technologies as Integrated Lead Finding Platforms for Drug Discovery</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1629</span>, <span class="refDoi"> DOI: 10.3390/molecules24081629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.3390%2Fmolecules24081629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpslyju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1629&issue=8&author=J.+Ottlauthor=L.+Lederauthor=J.+V.+Schaeferauthor=C.+E.+Dumelin&title=Encoded+Library+Technologies+as+Integrated+Lead+Finding+Platforms+for+Drug+Discovery&doi=10.3390%2Fmolecules24081629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded library technologies as integrated lead finding platforms for drug discovery</span></div><div class="casAuthors">Ottl, Johannes; Leder, Lukas; Schaefer, Jonas V.; Dumelin, Christoph E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1629</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The scope of targets investigated in pharmaceutical research is continuously moving into uncharted territory.  Consequently, finding suitable chem. matter with current compd. collections is proving increasingly difficult.  Encoded library technologies enable the rapid exploration of large chem. space for the identification of ligands for such targets.  These binders facilitate drug discovery projects both as tools for target validation, structural elucidation and assay development as well as starting points for medicinal chem.  Novartis internalized two complementing encoded library platforms to accelerate the initiation of its drug discovery programs.  For the identification of low-mol. wt. ligands, we apply DNA-encoded libraries.  In addn., encoded peptide libraries are employed to identify cyclic peptides.  This review discusses how we apply these two platforms in our research and why we consider it beneficial to run both pipelines inhouse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8YbvIrAzf7rVg90H21EOLACvtfcHk0ljCGFXxasv2pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpslyju7w%253D&md5=8ed3e114146bf60f513b94d1a83f992a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24081629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24081629%26sid%3Dliteratum%253Aachs%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DSchaefer%26aufirst%3DJ.%2BV.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26atitle%3DEncoded%2520Library%2520Technologies%2520as%2520Integrated%2520Lead%2520Finding%2520Platforms%2520for%2520Drug%2520Discovery%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26issue%3D8%26spage%3D1629%26doi%3D10.3390%2Fmolecules24081629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannocci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimbacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">20 Years of DNA-Encoded Chemical Libraries</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">12747</span>– <span class="NLM_lpage">12753</span>, <span class="refDoi"> DOI: 10.1039/c1cc15634a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1039%2Fc1cc15634a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=22083211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFamsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=12747-12753&issue=47&author=L.+Mannocciauthor=M.+Leimbacherauthor=M.+Wichertauthor=J.+R.+Scheuermannauthor=D.+Neri&title=20+Years+of+DNA-Encoded+Chemical+Libraries&doi=10.1039%2Fc1cc15634a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">20 Years of DNA-encoded chemical libraries</span></div><div class="casAuthors">Mannocci, Luca; Leimbacher, Markus; Wichert, Moreno; Scheuermann, Joerg; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">12747-12753</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The identification of specific binding mols. is a central problem in chem., biol. and medicine.  Therefore, technologies, which facilitate ligand discovery, may substantially contribute to a better understanding of biol. processes and to drug discovery.  DNA-encoded chem. libraries represent a new inexpensive tool for the fast and efficient identification of ligands to target proteins of choice.  Such libraries consist of collections of org. mols., covalently linked to a unique DNA tag serving as an amplifiable identification bar code.  DNA-encoding enables the in vitro selection of ligands by affinity capture at sub-picomolar concns. on virtually any target protein of interest, in analogy to established selection methodologies like antibody phage display.  Multiple strategies have been investigated by several academic and industrial labs. for the construction of DNA-encoded chem. libraries comprising up to millions of DNA-encoded compds.  The implementation of next generation high-throughput sequencing enabled the rapid identification of binding mols. from DNA-encoded libraries of unprecedented size.  This article reviews the development of DNA-encoded library technol. and its evolution into a novel drug discovery tool, commenting on challenges, perspectives and opportunities for the different exptl. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawUGgkRIh-7Vg90H21EOLACvtfcHk0lgZcbGq453IJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFamsL3O&md5=835ec84173e5a1343484cb3574a4ec4b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2Fc1cc15634a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc15634a%26sid%3Dliteratum%253Aachs%26aulast%3DMannocci%26aufirst%3DL.%26aulast%3DLeimbacher%26aufirst%3DM.%26aulast%3DWichert%26aufirst%3DM.%26aulast%3DScheuermann%26aufirst%3DJ.%2BR.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3D20%2520Years%2520of%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26issue%3D47%26spage%3D12747%26epage%3D12753%26doi%3D10.1039%2Fc1cc15634a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Lead Discovery by DNA-Encoded Chemical Libraries</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11–12</span>),  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2007.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=17532531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvVart7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=465-471&issue=11%E2%80%9312&author=S.+Melkkoauthor=C.+Dumelinauthor=J.+Scheuermannauthor=D.+Neri&title=Lead+Discovery+by+DNA-Encoded+Chemical+Libraries&doi=10.1016%2Fj.drudis.2007.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lead discovery by DNA-encoded chemical libraries</span></div><div class="casAuthors">Melkko, Samu; Dumelin, Christoph E.; Scheuermann, Joerg; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11 & 12</span>),
    <span class="NLM_cas:pages">465-471</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The isolation of specific binding mols. is a central problem in the drug discovery process and might be useful for the elucidation of the biol. function of proteins identified in genome and proteome research.  Libraries of org. mols., conjugated covalently to DNA tags that serve as identification bar codes, were proposed recently as a way to identify ligands of target proteins of choice efficiently.  Here, we analyze the different strategies for constructing DNA-encoded chem. libraries, and the potential and challenges of this promising technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8bZwa_dmVUrVg90H21EOLACvtfcHk0lgZcbGq453IJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvVart7g%253D&md5=b8b73bc50d523e2465dbb3eb08290adc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DDumelin%26aufirst%3DC.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DLead%2520Discovery%2520by%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26issue%3D11%25E2%2580%259312%26spage%3D465%26epage%3D471%26doi%3D10.1016%2Fj.drudis.2007.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Toonder, J. M. J.</span></span> <span> </span><span class="NLM_article-title">Microfluidics for Cell-Based High Throughput Screening Platforms-A Review</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>903</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2015.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.aca.2015.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26709297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFagt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=903&publication_year=2016&pages=36-50&author=G.+Duauthor=Q.+Fangauthor=J.+M.+J.+den+Toonder&title=Microfluidics+for+Cell-Based+High+Throughput+Screening+Platforms-A+Review&doi=10.1016%2Fj.aca.2015.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidics for cell-based high throughput screening platforms-A review</span></div><div class="casAuthors">Du, Guansheng; Fang, Qun; den Toonder, Jaap M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">903</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-50</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the last decades, the basic techniques of microfluidics for the study of cells such as cell culture, cell sepn., and cell lysis, have been well developed.  Based on cell handling techniques, microfluidics has been widely applied in the field of PCR (Polymerase Chain Reaction), immunoassays, organ-on-chip, stem cell research, and anal. and identification of circulating tumor cells.  As a major step in drug discovery, high-throughput screening allows rapid anal. of thousands of chem., biochem., genetic or pharmacol. tests in parallel.  In this review, the authors summarize the application of microfluidics in cell-based high throughput screening.  The screening methods mentioned in this paper include approaches using the perfusion flow mode, the droplet mode, and the microarray mode.  The authors also discuss the future development of microfluidic based high throughput screening platform for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwdyXF1fDA1bVg90H21EOLACvtfcHk0lgZcbGq453IJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFagt7fP&md5=1077740c0b8ae724d1b0e0d36d09e631</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2015.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2015.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DG.%26aulast%3DFang%26aufirst%3DQ.%26aulast%3Dden%2BToonder%26aufirst%3DJ.%2BM.%2BJ.%26atitle%3DMicrofluidics%2520for%2520Cell-Based%2520High%2520Throughput%2520Screening%2520Platforms-A%2520Review%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2016%26volume%3D903%26spage%3D36%26epage%3D50%26doi%3D10.1016%2Fj.aca.2015.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gimeno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojeda-Montes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomás-Hernández, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cereto-Massagué, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrán-Debón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujadas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Vallvé, S.</span></span> <span> </span><span class="NLM_article-title">The Light and Dark Sides of Virtual Screening: What Is There to Know?</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1375</span>, <span class="refDoi"> DOI: 10.3390/ijms20061375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.3390%2Fijms20061375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=1375&issue=6&author=A.+Gimenoauthor=M.+Ojeda-Montesauthor=S.+Tom%C3%A1s-Hern%C3%A1ndezauthor=A.+Cereto-Massagu%C3%A9author=R.+Beltr%C3%A1n-Deb%C3%B3nauthor=M.+Muleroauthor=G.+Pujadasauthor=S.+Garcia-Vallv%C3%A9&title=The+Light+and+Dark+Sides+of+Virtual+Screening%3A+What+Is+There+to+Know%3F&doi=10.3390%2Fijms20061375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The light and dark sides of virtual screening: what is there to know?</span></div><div class="casAuthors">Gimeno, Aleix; Ojeda-Montes, Maria Jose; Tomas-Hernandez, Sarah; Cereto-Massague, Adria; Beltran-Debon, Raul; Mulero, Miquel; Pujadas, Gerard; Garcia-Vallve, Santiago</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1375</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Virtual screening consists of using computational tools to predict potentially bioactive compds. from files contg. large libraries of small mols.  Virtual screening is becoming increasingly popular in the field of drug discovery as in silico techniques are continuously being developed, improved, and made available.  As most of these techniques are easy to use, both private and public organizations apply virtual screening methodologies to save resources in the lab.  However, it is often the case that the techniques implemented in virtual screening workflows are restricted to those that the research team knows.  Moreover, although the software is often easy to use, each methodol. has a series of drawbacks that should be avoided so that false results or artifacts are not produced.  Here, we review the most common methodologies used in virtual screening workflows in order to both introduce the inexperienced researcher to new methodologies and advise the experienced researcher on how to prevent common mistakes and the improper usage of virtual screening methodologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJWXWcy8LDELVg90H21EOLACvtfcHk0lgZcbGq453IJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmtL8%253D&md5=e813e0bb5168713ae670d27bbf8060cf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Fijms20061375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20061375%26sid%3Dliteratum%253Aachs%26aulast%3DGimeno%26aufirst%3DA.%26aulast%3DOjeda-Montes%26aufirst%3DM.%26aulast%3DTom%25C3%25A1s-Hern%25C3%25A1ndez%26aufirst%3DS.%26aulast%3DCereto-Massagu%25C3%25A9%26aufirst%3DA.%26aulast%3DBeltr%25C3%25A1n-Deb%25C3%25B3n%26aufirst%3DR.%26aulast%3DMulero%26aufirst%3DM.%26aulast%3DPujadas%26aufirst%3DG.%26aulast%3DGarcia-Vallv%25C3%25A9%26aufirst%3DS.%26atitle%3DThe%2520Light%2520and%2520Dark%2520Sides%2520of%2520Virtual%2520Screening%253A%2520What%2520Is%2520There%2520to%2520Know%253F%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26issue%3D6%26spage%3D1375%26doi%3D10.3390%2Fijms20061375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haning, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, H.</span></span> <span> </span><span class="NLM_article-title">An Approach towards Enhancement of a Screening Library: The Next Generation Library Initiative (NGLI) at Bayer — against All Odds?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2018.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30562586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1SnsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=668-672&issue=3&author=M.+Follmannauthor=H.+Briemauthor=A.+Steinmeyerauthor=A.+Hillischauthor=M.+H.+Schmittauthor=H.+Haningauthor=H.+Meier&title=An+Approach+towards+Enhancement+of+a+Screening+Library%3A+The+Next+Generation+Library+Initiative+%28NGLI%29+at+Bayer+%E2%80%94+against+All+Odds%3F&doi=10.1016%2Fj.drudis.2018.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An approach towards enhancement of a screening library: The Next Generation Library Initiative (NGLI) at Bayer - against all odds?</span></div><div class="casAuthors">Follmann, Markus; Briem, Hans; Steinmeyer, Andreas; Hillisch, Alexander; Schmitt, Monika H.; Haning, Helmut; Meier, Heinrich</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">668-672</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmaceutical companies often refer to 'screening their library' when performing high-throughput screening (HTS) on a corporate compd. collection to identify lead structures for small-mol. drug discovery programs.  Characteristics of such a library, including the size, chem. space covered, and physicochem. properties, often det. the success of a screening campaign.  Therefore, strategies to maintain and enhance the overall quality of screening collections are crucial to stay competitive and to cope with the 'novelty erosion' that is obsd. gradually.  The Next Generation Library Initiative (NGLI), the enhancement of Bayer's HTS collection by 500 000 newly designed compds. within 5 years, is addressing exactly this challenge.  Here, we describe this collaborative project, which involves all internal medicinal chemists in a crowd-sourcing approach, as well as selected external partners, to reach this ambitious goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vZrpFbl7FrVg90H21EOLACvtfcHk0ljUwTAjYUNJAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1SnsrrK&md5=b2a5aa9b0ff60f0fe751aa10a8019f83</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSteinmeyer%26aufirst%3DA.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DM.%2BH.%26aulast%3DHaning%26aufirst%3DH.%26aulast%3DMeier%26aufirst%3DH.%26atitle%3DAn%2520Approach%2520towards%2520Enhancement%2520of%2520a%2520Screening%2520Library%253A%2520The%2520Next%2520Generation%2520Library%2520Initiative%2520%2528NGLI%2529%2520at%2520Bayer%2520%25E2%2580%2594%2520against%2520All%2520Odds%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D668%26epage%3D672%26doi%3D10.1016%2Fj.drudis.2018.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmerlin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüthi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegrist, R.</span></span> <span> </span><span class="NLM_article-title">The Screening Compound Collection: A Key Asset for Drug Discovery</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2533%2Fchimia.2017.667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=29070412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=667-677&issue=10&author=C.+Bossauthor=J.+Hazemannauthor=T.+Kimmerlinauthor=M.+von+Korffauthor=U.+L%C3%BCthiauthor=O.+Peterauthor=T.+Sanderauthor=R.+Siegrist&title=The+Screening+Compound+Collection%3A+A+Key+Asset+for+Drug+Discovery&doi=10.2533%2Fchimia.2017.667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The screening compound collection: a key asset for drug discovery</span></div><div class="casAuthors">Boss, Christoph; Hazemann, Julien; Kimmerlin, Thierry; von Korff, Modest; Luthi, Urs; Peter, Oliver; Sander, Thomas; Siegrist, Romain</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">In this case study on an essential instrument of modern drug discovery, we summarize our successful efforts in the last four years toward enhancing the Actelion screening compd. collection.  A key organizational step was the establishment of the Compd. Library Committee (CLC) in Sept. 2013.  This cross-functional team consisting of computational scientists, medicinal chemists and a biologist was endowed with a significant annual budget for regular new compd. purchases.  Based on an initial library anal. performed in 2013, the CLC developed a New Library Strategy.  The established continuous library turn-over mode, and the screening library size of 300'000 compds. were maintained, while the structural library quality was increased.  This was achieved by shifting the selection criteria from 'druglike' to 'leadlike' structures, enriching for non-flat structures, aiming for compd. novelty, and increasing the ratio of higher cost 'Premium Compds.'.  Novel chem. space was gained by adding natural compds., macrocycles, designed and focused libraries to the collection, and through mutual exchanges of proprietary compds. with agrochem. companies.  A comparative anal. in 2016 provided evidence for the pos. impact of these measures.  Screening the improved library has provided several highly promising hits, including a macrocyclic compd., that are currently followed up in different Hit-to-Lead and Lead Optimization programs.  It is important to state that the goal of the CLC was not to achieve higher HTS hit rates, but to increase the chances of identified hits to serve as the basis of successful early drug discovery programs.  The experience gathered so far legitimates the New Library Strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNca95IroBS7Vg90H21EOLACvtfcHk0ljUwTAjYUNJAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeiurc%253D&md5=e919cd047c256713c3b48d942c432b9c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.667%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DHazemann%26aufirst%3DJ.%26aulast%3DKimmerlin%26aufirst%3DT.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DPeter%26aufirst%3DO.%26aulast%3DSander%26aufirst%3DT.%26aulast%3DSiegrist%26aufirst%3DR.%26atitle%3DThe%2520Screening%2520Compound%2520Collection%253A%2520A%2520Key%2520Asset%2520for%2520Drug%2520Discovery%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26issue%3D10%26spage%3D667%26epage%3D677%26doi%3D10.2533%2Fchimia.2017.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varghese, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span> <span> </span><span class="NLM_article-title">An Analysis of Different Components of a High-Throughput Screening Library</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2057</span>– <span class="NLM_lpage">2068</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ymtbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=2057-2068&issue=10&author=A.+Sahaauthor=T.+Vargheseauthor=A.+Liuauthor=S.+J.+Allenauthor=T.+Mirzadeganauthor=M.+D.+Hack&title=An+Analysis+of+Different+Components+of+a+High-Throughput+Screening+Library&doi=10.1021%2Facs.jcim.8b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of Different Components of a High-Throughput Screening Library</span></div><div class="casAuthors">Saha, Arjun; Varghese, Teena; Liu, Annie; Allen, Samantha J.; Mirzadegan, Taraneh; Hack, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2057-2068</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since many projects at pharmaceutical organizations get their start from a high-throughput screening (HTS) campaign, improving the quality of the HTS deck can improve the likelihood of discovering a high-quality lead mol. that can be progressed to a drug candidate.  Over the past decade, Janssen has implemented several strategies for external compd. acquisition to augment the screening deck beyond the chem. space and no. of mols. synthesized for internal projects.  In this report, the authors analyzed the performance of each of those compd. collections in the screening campaigns performed internally within Janssen during the last five years.  The authors classified the screening library into two broad categories: Internal and External.  The comparison of the performance of these sets of libraries was done by considering the primary, confirmation, and dose response hit rates.  The anal. revealed that Internal compds. (resulting from numerous medicinal chem. efforts against diverse protein targets) have higher av. confirmation hit rates than External ones; however, actives from both categories show similar probabilities of hitting multiple distinct targets.  The authors also investigated the property landscape of both sets of libraries to identify the key elements which make a difference in these categories of compds.  From this anal., Janssen aims to understand the descriptor landscape of the compds. with the highest hit rates and to use them for improving its future acquisition strategies as well as to inform the plating strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSQmsLK83Y2bVg90H21EOLACvtfcHk0ljUwTAjYUNJAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ymtbbF&md5=5ab8e2c7116a16acc5f030f06036428c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00258%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DA.%26aulast%3DVarghese%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DAllen%26aufirst%3DS.%2BJ.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DHack%26aufirst%3DM.%2BD.%26atitle%3DAn%2520Analysis%2520of%2520Different%2520Components%2520of%2520a%2520High-Throughput%2520Screening%2520Library%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26issue%3D10%26spage%3D2057%26epage%3D2068%26doi%3D10.1021%2Facs.jcim.8b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span> <span> </span><span class="NLM_article-title">Novel Trends in High-Throughput Screening</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">580</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2009.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.coph.2009.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=19775937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=580-588&issue=5&author=L.+M.+Mayrauthor=D.+Bojanic&title=Novel+Trends+in+High-Throughput+Screening&doi=10.1016%2Fj.coph.2009.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Novel trends in high-throughput screening</span></div><div class="casAuthors">Mayr, Lorenz M.; Bojanic, Dejan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">580-588</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established process for lead discovery in Pharma and Biotech companies and is now also being used for basic and applied research in academia.  It comprises the screening of large chem. libraries for activity against biol. targets via the use of automation, miniaturized assays and large-scale data anal.  Since its first advent in the early to mid 1990s, the field of HTS has seen not only a continuous change in technol. and processes, but also an adaptation to various needs in lead discovery.  HTS has now evolved into a mature discipline that is a crucial source of chem. starting points for drug discovery.  Whereas in previous years much emphasis has been put on a steady increase in screening capacity (quant. increase') via automation and miniaturization, the past years have seen a much greater emphasis on content and quality (qual. increase').  Today, many experts in the field see HTS at a crossroad with the need to decide on either higher throughput/more experimentation or a greater focus on assays of greater physiol. relevance, both of which may lead to higher productivity in pharmaceutical R&D.  In this paper, we describe the development of HTS over the past decade and point out our own ideas for future directions of HTS in biomedical research.  We predict that the trend toward further miniaturization will slow down with the balanced implementation of 384 well, 1536 well, and 384 low vol. well plates.  Furthermore, we envisage that there will be much more emphasis on rigorous assay and chem. characterization, particularly considering that novel and more difficult target classes will be pursued.  In recent years we have witnessed a clear trend in the drug discovery community toward rigorous hit validation by the use of orthogonal readout technologies, label free and biophys. methodologies.  We also see a trend toward a more flexible use of the various screening approaches in lead discovery, i.e., the use of both full deck compd. screening as well as the use of focused screening and iterative screening approaches.  Moreover, we expect greater usage of target identification strategies downstream of phenotypic screening and the more effective implementation of affinity selection technologies as a result of advances in chem. diversity methodologies.  We predict that, ultimately, each hit finding strategy will be much more project-related, tailor-made, and better integrated into the broader drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP3Oo1Orzt87Vg90H21EOLACvtfcHk0ljsAMk3bDhP0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnL&md5=50284fdd9f4d610e81fb442b78b7b69e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2009.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2009.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DL.%2BM.%26aulast%3DBojanic%26aufirst%3DD.%26atitle%3DNovel%2520Trends%2520in%2520High-Throughput%2520Screening%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2009%26volume%3D9%26issue%3D5%26spage%3D580%26epage%3D588%26doi%3D10.1016%2Fj.coph.2009.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">The Essential Roles of Chemistry in High-Throughput Screening Triage</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1290</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.60</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.4155%2Ffmc.14.60" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25163000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVeisbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1265-1290&issue=11&author=J.+L.+Dahlinauthor=M.+A.+Walters&title=The+Essential+Roles+of+Chemistry+in+High-Throughput+Screening+Triage&doi=10.4155%2Ffmc.14.60"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The essential roles of chemistry in high-throughput screening triage</span></div><div class="casAuthors">Dahlin, Jayme L.; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1265-1290</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">It is increasingly clear that academic high-throughput screening (HTS) and virtual HTS triage suffers from a lack of scientists trained in the art and science of early drug discovery chem.  Many recent publications report the discovery of compds. by screening that are most likely artifacts or promiscuous bioactive compds., and these results are not placed into the context of previous studies.  For HTS to be most successful, it is our contention that there must exist an early partnership between biologists and medicinal chemists.  Their combined skill sets are necessary to design robust assays and efficient workflows that will weed out assay artifacts, false positives, promiscuous bioactive compds. and intractable screening hits, efforts that ultimately give projects a better chance at identifying truly useful chem. matter.  Expertise in medicinal chem., chemoinformatics and purifn. sciences (anal. chem.) can enhance the post-HTS triage process by quickly removing these problematic chemotypes from consideration, while simultaneously prioritizing the more promising chem. matter for follow-up testing.  It is only when biologists and chemists collaborate effectively that HTS can manifest its full promise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4L_qQtRFBUbVg90H21EOLACvtfcHk0ljsAMk3bDhP0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVeisbjF&md5=4ca8e43d7a85a1f67be930d40bfefad4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.60&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.60%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Essential%2520Roles%2520of%2520Chemistry%2520in%2520High-Throughput%2520Screening%2520Triage%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D11%26spage%3D1265%26epage%3D1290%26doi%3D10.4155%2Ffmc.14.60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haasen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antczak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">How Phenotypic Screening Influenced Drug Discovery: Lessons from Five Years of Practice</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1089/adt.2017.796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1089%2Fadt.2017.796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=28800248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOqtrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=239-246&issue=6&author=D.+Haasenauthor=U.+Schopferauthor=C.+Antczakauthor=C.+Guyauthor=F.+Fuchsauthor=P.+Selzer&title=How+Phenotypic+Screening+Influenced+Drug+Discovery%3A+Lessons+from+Five+Years+of+Practice&doi=10.1089%2Fadt.2017.796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How Phenotypic Screening Influenced Drug Discovery: Lessons from Five Years of Practice</span></div><div class="casAuthors">Haasen, Dorothea; Schopfer, Ulrich; Antczak, Christophe; Guy, Chantale; Fuchs, Florian; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">239-246</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Since 2011, phenotypic screening has been a trend in the pharmaceutical industry as well as in academia.  This renaissance was triggered by analyses that suggested that phenotypic screening is a superior strategy to discover first-in-class drugs.  Despite these promises and considerable investments, pharmaceutical research organizations have encountered considerable challenges with the approach.  Few success stories have emerged in the past 5 years and companies are questioning their investment in this area.  In this contribution, we outline what we have learned about success factors and challenges of phenotypic screening.  We then describe how our efforts in phenotypic screening have influenced our approach to drug discovery in general.  We predict that concepts from phenotypic screening will be incorporated into target-based approaches and will thus remain influential beyond the current trend.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUI8SiRaqgaLVg90H21EOLACvtfcHk0liuFDe7RJticQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOqtrjM&md5=b63967012bb77fa413e023794e944fd6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1089%2Fadt.2017.796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2017.796%26sid%3Dliteratum%253Aachs%26aulast%3DHaasen%26aufirst%3DD.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DAntczak%26aufirst%3DC.%26aulast%3DGuy%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DF.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DHow%2520Phenotypic%2520Screening%2520Influenced%2520Drug%2520Discovery%253A%2520Lessons%2520from%2520Five%2520Years%2520of%2520Practice%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2017%26volume%3D15%26issue%3D6%26spage%3D239%26epage%3D246%26doi%3D10.1089%2Fadt.2017.796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paricharak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span> <span> </span><span class="NLM_article-title">Analysis of Iterative Screening with Stepwise Compound Selection Based on Novartis In-House HTS Data</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1255</span>– <span class="NLM_lpage">1264</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00029</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1255-1264&issue=5&author=S.+Paricharakauthor=A.+P.+Ijzermanauthor=A.+Benderauthor=F.+Nigsch&title=Analysis+of+Iterative+Screening+with+Stepwise+Compound+Selection+Based+on+Novartis+In-House+HTS+Data&doi=10.1021%2Facschembio.6b00029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Iterative Screening with Stepwise Compound Selection Based on Novartis In-house HTS Data</span></div><div class="casAuthors">Paricharak, Shardul; Ijzerman, Adriaan P.; Bender, Andreas; Nigsch, Florian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1255-1264</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With increased automation and larger compd. collections, the development of high-throughput screening (HTS) started replacing previous approaches in drug discovery from around the 1980s onward.  However, even today it is not always appropriate, or even feasible, to screen large collections of compds. in a particular assay.  Here, we present an efficient method for iterative screening of small subsets of compd. libraries.  With this method, the retrieval of active compds. is optimized using their structural information and biol. activity fingerprints.  We validated this approach retrospectively on 34 Novartis inhouse HTS assays covering a wide range of assay biol., including cell proliferation, antibacterial activity, gene expression, and phosphorylation.  This method was employed to retrieve subsets of compds. for screening, where selected hits from any given round of screening were used as starting points to select chem. and biol. similar compds. for the next iteration.  By only screening ∼1% of the full screening collection (∼15 000 compds.), the method consistently retrieves diverse compds. belonging to the top 0.5% of the most active compds. for the HTS campaign.  For most of the assays, over half of the compds. selected by the method were found to be among the 5% most active compds. of the corresponding full-deck HTS.  In addn., the stringency of the iterative method can be modified depending on the no. of compds. one can afford to screen, making it a flexible tool to discover active compds. efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogdiZaBr3m77Vg90H21EOLACvtfcHk0liuFDe7RJticQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFShu7o%253D&md5=e8586e380292cc7194a06b832a80d9be</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00029%26sid%3Dliteratum%253Aachs%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DNigsch%26aufirst%3DF.%26atitle%3DAnalysis%2520of%2520Iterative%2520Screening%2520with%2520Stepwise%2520Compound%2520Selection%2520Based%2520on%2520Novartis%2520In-House%2520HTS%2520Data%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D1255%26epage%3D1264%26doi%3D10.1021%2Facschembio.6b00029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buendia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linusson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toccaceli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlberg, E.</span></span> <span> </span><span class="NLM_article-title">Accurate Hit Estimation for Iterative Screening Using Venn-ABERS Predictors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00724</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00724" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFCitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1230-1237&issue=3&author=R.+Buendiaauthor=T.+Kogejauthor=O.+Engkvistauthor=L.+Carlssonauthor=H.+Linussonauthor=U.+Johanssonauthor=P.+Toccaceliauthor=E.+Ahlberg&title=Accurate+Hit+Estimation+for+Iterative+Screening+Using+Venn-ABERS+Predictors&doi=10.1021%2Facs.jcim.8b00724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Hit Estimation for Iterative Screening Using Venn-ABERS Predictors</span></div><div class="casAuthors">Buendia, Ruben; Kogej, Thierry; Engkvist, Ola; Carlsson, Lars; Linusson, Henrik; Johansson, Ulf; Toccaceli, Paolo; Ahlberg, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1230-1237</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Iterative screening has emerged as a promising approach to increase the efficiency of high-throughput screening (HTS) campaigns in drug discovery.  By learning from a subset of the compd. library, inferences on what compds. to screen next can be made by predictive models.  One of the challenges of iterative screening is to decide how many iterations to perform.  This is mainly related to difficulties in estg. the prospective hit rate in any given iteration.  In this article, a novel method based on Venn-ABERS predictors is proposed.  The method provides accurate ests. of the no. of hits retrieved in any given iteration during an HTS campaign.  The ests. provide the necessary information to support the decision on the no. of iterations needed to maximize the screening outcome.  Thus, this method offers a prospective screening strategy for early-stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFM6qNl1oTyrVg90H21EOLACvtfcHk0liuFDe7RJticQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFCitrc%253D&md5=b6ea3dab102f23a95ea380376729f343</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00724%26sid%3Dliteratum%253Aachs%26aulast%3DBuendia%26aufirst%3DR.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DLinusson%26aufirst%3DH.%26aulast%3DJohansson%26aufirst%3DU.%26aulast%3DToccaceli%26aufirst%3DP.%26aulast%3DAhlberg%26aufirst%3DE.%26atitle%3DAccurate%2520Hit%2520Estimation%2520for%2520Iterative%2520Screening%2520Using%2520Venn-ABERS%2520Predictors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D3%26spage%3D1230%26epage%3D1237%26doi%3D10.1021%2Facs.jcim.8b00724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lajiness, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span> <span> </span><span class="NLM_article-title">Assessment of the Consistency of Medicinal Chemists in Reviewing Sets of Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">4891</span>– <span class="NLM_lpage">4896</span>, <span class="refDoi"> DOI: 10.1021/jm049740z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049740z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVejtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4891-4896&issue=20&author=M.+S.+Lajinessauthor=G.+M.+Maggioraauthor=V.+Shanmugasundaram&title=Assessment+of+the+Consistency+of+Medicinal+Chemists+in+Reviewing+Sets+of+Compounds&doi=10.1021%2Fjm049740z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of the Consistency of Medicinal Chemists in Reviewing Sets of Compounds</span></div><div class="casAuthors">Lajiness, Michael S.; Maggiora, Gerald M.; Shanmugasundaram, Veerabahu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4891-4896</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Medicinal chemists are frequently asked to review lists of compds. to assess their drug- or lead-like nature and to evaluate the suitability of lead compds. based on their "attractiveness" and/or synthetic feasibility as a basis for launching a drug-discovery campaign.  It is often felt that one medicinal chemist's opinion is as good as any other, but is it.  In an attempt to answer this question, an expt. was performed in conjunction with a recent compd. acquisition program (CAP) conducted at Pharmacia.  Historically, the CAP included a review of many thousands of compds. by medicinal chemists who eliminate anything deemed undesirable for any reason.  In a review conducted in 2002, about 22 000 compds. requiring review by medicinal chemists were broken down into 11 lists of approx. 2000 compds. each.  Unknown to the medicinal chemists, a subset of 250 compds., previously rejected by a very experienced senior medicinal chemist, was added to each of the lists.  Most of the 13 medicinal chemists who participated in this process reviewed two lists, although some only reviewed a single list and one reviewed three lists.  Those compds. that were deemed unacceptable were recorded and tabulated in various ways to assess the consistency of the reviews.  It was found that medicinal chemists were not very consistent in the compds. they rejected as being undesirable.  The inconsistency arises from the subjective anal. that all humans utilize when considering "data sets" of any kind.  This has important implications for pharmaceutical project teams where individual medicinal chemists review lists of primary screening hits to identify those compds. suitable for follow-up.  Once a compd. is removed from a list, it and other structurally similar compds. are effectively removed from further consideration.  This can also have an impact on computational chemists who are developing models for assessing the desirability or attractiveness of different classes of compds. for lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZPYRvqjXJjLVg90H21EOLACvtfcHk0lgaMtNM0Iw1YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVejtL0%253D&md5=d5f8cbd4a56bb4c22782a3beccb3eefd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm049740z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049740z%26sid%3Dliteratum%253Aachs%26aulast%3DLajiness%26aufirst%3DM.%2BS.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26atitle%3DAssessment%2520of%2520the%2520Consistency%2520of%2520Medicinal%2520Chemists%2520in%2520Reviewing%2520Sets%2520of%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D20%26spage%3D4891%26epage%3D4896%26doi%3D10.1021%2Fjm049740z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasilyeva, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coley, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span> <span> </span><span class="NLM_article-title">Inside the Mind of a Medicinal Chemist: The Role of Human Bias in Compound Prioritization during Drug Discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e48476</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0048476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1371%2Fjournal.pone.0048476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23185259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVagsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e48476&issue=11&author=P.+S.+Kutchukianauthor=N.+Y.+Vasilyevaauthor=J.+Xuauthor=M.+K.+Lindvallauthor=M.+P.+Dillonauthor=M.+Glickauthor=J.+D.+Coleyauthor=N.+Brooijmans&title=Inside+the+Mind+of+a+Medicinal+Chemist%3A+The+Role+of+Human+Bias+in+Compound+Prioritization+during+Drug+Discovery&doi=10.1371%2Fjournal.pone.0048476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inside the mind of a medicinal chemist: the role of human bias in compound prioritization during drug discovery</span></div><div class="casAuthors">Kutchukian, Peter S.; Vasilyeva, Nadya Y.; Xu, Jordan; Lindvall, Mika K.; Dillon, Michael P.; Glick, Meir; Coley, John D.; Brooijmans, Natasja</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e48476</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Medicinal chemists' "intuition" is crit. for success in modern drug discovery.  Early in the discovery process, chemists select a subset of compds. for further research, often from many viable candidates.  These decisions det. the success of a discovery campaign, and ultimately what kind of drugs are developed and marketed to the public.  Surprisingly little is known about the cognitive aspects of chemists' decision-making when they prioritize compds.  We investigate (1) how and to what extent chemists simplify the problem of identifying promising compds., (2) whether chemists agree with each other about the criteria used for such decisions, and (3) how accurately chemists report the criteria they use for these decisions.  Chemists were surveyed and asked to select chem. fragments that they would be willing to develop into a lead compd. from a set of ∼4,000 available fragments.  Based on each chemist's selections, computational classifiers were built to model each chemist's selection strategy.  Results suggest that chemists greatly simplified the problem, typically using only 1-2 of many possible parameters when making their selections.  Although chemists tended to use the same parameters to select compds., differing value preferences for these parameters led to an overall lack of consensus in compd. selections.  Moreover, what little agreement there was among the chemists was largely in what fragments were undesirable.  Furthermore, chemists were often unaware of the parameters (such as compd. size) which were statistically significant in their selections, and overestimated the no. of parameters they employed.  A crit. evaluation of the problem space faced by medicinal chemists and cognitive models of categorization were esp. useful in understanding the low consensus between chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB2xkFnkLMFbVg90H21EOLACvtfcHk0lgaMtNM0Iw1YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVagsr3N&md5=a94653786832db485777a60e05dbb848</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0048476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0048476%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DVasilyeva%26aufirst%3DN.%2BY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLindvall%26aufirst%3DM.%2BK.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DColey%26aufirst%3DJ.%2BD.%26aulast%3DBrooijmans%26aufirst%3DN.%26atitle%3DInside%2520the%2520Mind%2520of%2520a%2520Medicinal%2520Chemist%253A%2520The%2520Role%2520of%2520Human%2520Bias%2520in%2520Compound%2520Prioritization%2520during%2520Drug%2520Discovery%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3De48476%26doi%3D10.1371%2Fjournal.pone.0048476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahue, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diratsouian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span> <span> </span><span class="NLM_article-title">Diversity & Tractability Revisited in Collaborative Small Molecule Phenotypic Screening Library Design</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">115192</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmc.2019.115192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31837897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSlurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=115192&issue=1&author=B.+R.+Lahueauthor=M.+Glickauthor=M.+Tudorauthor=S.+A.+Johnsonauthor=J.+Diratsouianauthor=M.+J.+Wildeyauthor=M.+Burtonauthor=R.+Mazzolaauthor=A.+M.+Wassermann&title=Diversity+%26+Tractability+Revisited+in+Collaborative+Small+Molecule+Phenotypic+Screening+Library+Design&doi=10.1016%2Fj.bmc.2019.115192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity & tractability revisited in collaborative small molecule phenotypic screening library design</span></div><div class="casAuthors">Lahue, Brian R.; Glick, Meir; Tudor, Matthew; Johnson, Scott Arne; Diratsouian, Janet; Wildey, Mary Jo; Burton, Marybeth; Mazzola, Robert; Wassermann, Anne Mai</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115192</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Identification of purposeful chem. matter on a broad range of drug targets is of high importance to the pharmaceutical industry.  However, disease-relevant but more complex hit-finding plans require flexibility regarding the subset of the compds. that we screen.  Herein we describe a strategy to design high-quality small mol. screening subsets of two different sizes to cope with a rapidly changing early discovery portfolio.  The approach taken balances chem. tractability, chem. diversity and biol. target coverage.  Furthermore, using surveys, we actively involved chemists within our company in the selection process of the diversity decks to ensure current medicinal chem. principles were incorporated.  The chemist surveys revealed that not all published PAINS substructure alerts are considered productive by the medicinal chem. community and in agreement with previously published results from other institutions, QED scores tracked quite well with chemists' notions of chem. attractiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn6BdDYp7at7Vg90H21EOLACvtfcHk0lgaMtNM0Iw1YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSlurrJ&md5=e5483bbc3a1011dcceaede9bb0611441</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115192%26sid%3Dliteratum%253Aachs%26aulast%3DLahue%26aufirst%3DB.%2BR.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DTudor%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DDiratsouian%26aufirst%3DJ.%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26aulast%3DBurton%26aufirst%3DM.%26aulast%3DMazzola%26aufirst%3DR.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26atitle%3DDiversity%2520%2526%2520Tractability%2520Revisited%2520in%2520Collaborative%2520Small%2520Molecule%2520Phenotypic%2520Screening%2520Library%2520Design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26issue%3D1%26spage%3D115192%26doi%3D10.1016%2Fj.bmc.2019.115192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scior, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Mayorga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuanalo-Contreras, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span> <span> </span><span class="NLM_article-title">Recognizing Pitfalls in Virtual Screening: A Critical Review</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1021/ci200528d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200528d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=867-881&issue=4&author=T.+Sciorauthor=A.+Benderauthor=G.+Tresadernauthor=J.+L.+Medina-Francoauthor=K.+Mart%C3%ADnez-Mayorgaauthor=T.+Langerauthor=K.+Cuanalo-Contrerasauthor=D.+K.+Agrafiotis&title=Recognizing+Pitfalls+in+Virtual+Screening%3A+A+Critical+Review&doi=10.1021%2Fci200528d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing Pitfalls in Virtual Screening: A Critical Review</span></div><div class="casAuthors">Scior, Thomas; Bender, Andreas; Tresadern, Gary; Medina-Franco, Jose L.; Martinez-Mayorga, Karina; Langer, Thierry; Cuanalo-Contreras, Karina; Agrafiotis, Dimitris K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aim of virtual screening (VS) is to identify bioactive compds. through computational means, by employing knowledge about the protein target (structure-based VS) or known bioactive ligands (ligand-based VS).  In VS, a large no. of mols. are ranked according to their likelihood to be bioactive compds., with the aim to enrich the top fraction of the resulting list (which can be tested in bioassays afterward).  At its core, VS attempts to improve the odds of identifying bioactive mols. by maximizing the true pos. rate, i.e., by ranking the truly active mols. as high as possible (and, correspondingly, the truly inactive ones as low as possible).  In choosing the right approach, the researcher is faced with many questions: where does the optimal balance between efficiency and accuracy lie when evaluating a particular algorithm; do some methods perform better than others and in what particular situations; and what do retrospective results tell us about the prospective utility of a particular method.  Given the multitude of settings, parameters, and data sets the practitioner can choose from, there are many pitfalls that lurk along the way which might render VS less efficient or downright useless.  This review attempts to catalog published and unpublished problems, shortcomings, failures, and tech. traps of VS methods with the aim to avoid pitfalls by making the user aware of them in the first place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGois-3Bq6mm-LVg90H21EOLACvtfcHk0lhRHpHSao2Y8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D&md5=ad61b3289039c03844085a8b8de9e1e4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci200528d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200528d%26sid%3Dliteratum%253Aachs%26aulast%3DScior%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMart%25C3%25ADnez-Mayorga%26aufirst%3DK.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DCuanalo-Contreras%26aufirst%3DK.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26atitle%3DRecognizing%2520Pitfalls%2520in%2520Virtual%2520Screening%253A%2520A%2520Critical%2520Review%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D4%26spage%3D867%26epage%3D881%26doi%3D10.1021%2Fci200528d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pascale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span> <span> </span><span class="NLM_article-title">Biased Complement Diversity Selection for Effective Exploration of Chemical Space in Hit-Finding Campaigns</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVahtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1709-1714&issue=5&author=J.+M.+Jansenauthor=G.+De+Pascaleauthor=S.+Fongauthor=M.+Lindvallauthor=H.+E.+Moserauthor=K.+Pfisterauthor=B.+Warneauthor=C.+Wartchow&title=Biased+Complement+Diversity+Selection+for+Effective+Exploration+of+Chemical+Space+in+Hit-Finding+Campaigns&doi=10.1021%2Facs.jcim.9b00048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Complement Diversity Selection for Effective Exploration of Chemical Space in Hit-Finding Campaigns</span></div><div class="casAuthors">Jansen, Johanna M.; De Pascale, Gianfranco; Fong, Susan; Lindvall, Mika; Moser, Heinz E.; Pfister, Keith; Warne, Bob; Wartchow, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1709-1714</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The success of hit-finding campaigns relies on many factors, including the quality and diversity of the set of compds. that is selected for screening.  This paper presents a generalized workflow that guides compd. selections from large compd. archives with opportunities to bias the selections with available knowledge in order to improve hit quality while still effectively sampling the accessible chem. space.  An optional flag in the workflow supports an explicit complement design function where diversity selections complement a given core set of compds.  Results from three project applications as well as a literature case study exemplify the effectiveness of the approach, which is available as a KNIME workflow named Biased Complement Diversity (BCD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQD_EVOq0WLVg90H21EOLACvtfcHk0lhRHpHSao2Y8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVahtbc%253D&md5=1ff4e953be15cda3259a7339a5c09983</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00048%26sid%3Dliteratum%253Aachs%26aulast%3DJansen%26aufirst%3DJ.%2BM.%26aulast%3DDe%2BPascale%26aufirst%3DG.%26aulast%3DFong%26aufirst%3DS.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DPfister%26aufirst%3DK.%26aulast%3DWarne%26aufirst%3DB.%26aulast%3DWartchow%26aufirst%3DC.%26atitle%3DBiased%2520Complement%2520Diversity%2520Selection%2520for%2520Effective%2520Exploration%2520of%2520Chemical%2520Space%2520in%2520Hit-Finding%2520Campaigns%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D5%26spage%3D1709%26epage%3D1714%26doi%3D10.1021%2Facs.jcim.9b00048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">10742</span>– <span class="NLM_lpage">10772</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00523</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00523" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvFWqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10742-10772&issue=19&author=M.+D.+Lloyd&title=High-Throughput+Screening+for+the+Discovery+of+Enzyme+Inhibitors&doi=10.1021%2Facs.jmedchem.0c00523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Screening for the Discovery of Enzyme Inhibitors</span></div><div class="casAuthors">Lloyd, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10742-10772</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes are common targets in high-throughput screening and related campaigns.  An anal. of papers published between 1990 and 2018 showed that kinases were the most common enzymes investigated, fluorescence-based assays were the most common readout method, and cancer and bacterial infections were the most common therapeutic areas.  High-throughput screening and fragment-screening campaigns published between 2017 and 2019 were analyzed in more depth, giving 75 examples of hit to lead development.  Kinases, phosphatases, proteases, and peptidases were the most common targets, fluorescent assays were the most commonly used, and a wide variety of structural features were obsd. within the derived drugs.  Hit frequency was largely independent of library size and pos. correlated with Z' value for the assay.  Binding of metal ions to library compds. and substrates is an underappreciated source of false-pos. results and unreproducible behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc-U48b3MdtrVg90H21EOLACvtfcHk0lhRHpHSao2Y8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvFWqsrg%253D&md5=3a7664f55f018d8a434c6fd2bc3721c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26atitle%3DHigh-Throughput%2520Screening%2520for%2520the%2520Discovery%2520of%2520Enzyme%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772%26doi%3D10.1021%2Facs.jmedchem.0c00523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoepfner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Goff, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltier, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grippo, M. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prindle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krastel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobos-Correa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Dark Chemical Matter as a Promising Starting Point for Drug Lead Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">958</span>– <span class="NLM_lpage">966</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.1936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26479441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yjtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=958-966&issue=12&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=D.+Hoepfnerauthor=G.+Le+Goffauthor=F.+J+Kingauthor=C.+Studerauthor=J.+M+Peltierauthor=M.+L+Grippoauthor=V.+Prindleauthor=J.+Taoauthor=A.+Schuffenhauerauthor=I.+M+Wallaceauthor=S.+Chenauthor=P.+Krastelauthor=A.+Cobos-Correaauthor=C.+N+Parkerauthor=J.+W+Daviesauthor=M.+Glick&title=Dark+Chemical+Matter+as+a+Promising+Starting+Point+for+Drug+Lead+Discovery&doi=10.1038%2Fnchembio.1936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Dark chemical matter as a promising starting point for drug lead discovery</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Hoepfner, Dominic; Le Goff, Gaelle; King, Frederick J.; Studer, Christian; Peltier, John M.; Grippo, Melissa L.; Prindle, Vivian; Tao, Jianshi; Schuffenhauer, Ansgar; Wallace, Iain M.; Chen, Shanni; Krastel, Philipp; Cobos-Correa, Amanda; Parker, Christian N.; Davies, John W.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">958-966</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is an integral part of early drug discovery.  Herein, the authors focused on those small mols. in a screening collection that have never shown biol. activity despite having been exhaustively tested in HTS assays.  These compds. are referred to as 'dark chem. matter' (DCM).  The authors quantified DCM, validated it in quality control expts., described its physicochem. properties and mapped it into chem. space.  Through anal. of prospective reporter-gene assay, gene expression and yeast chemogenomics expts., the authors evaluated the potential of DCM to show biol. activity in future screens.  The authors demonstrated that, despite the apparent lack of activity, occasionally these compds. can result in potent hits with unique activity and clean safety profiles, which makes them valuable starting points for lead optimization efforts.  Among the identified DCM hits was a new antifungal chemotype with strong activity against the pathogen Cryptococcus neoformans but little activity at targets relevant to human safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp21Fvx08O4r7Vg90H21EOLACvtfcHk0liZefIgnIAIwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yjtrnM&md5=fdd233cf5a177504afb3acc37600cf88</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1936%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DHoepfner%26aufirst%3DD.%26aulast%3DLe%2BGoff%26aufirst%3DG.%26aulast%3DKing%26aufirst%3DF.%2BJ%26aulast%3DStuder%26aufirst%3DC.%26aulast%3DPeltier%26aufirst%3DJ.%2BM%26aulast%3DGrippo%26aufirst%3DM.%2BL%26aulast%3DPrindle%26aufirst%3DV.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DWallace%26aufirst%3DI.%2BM%26aulast%3DChen%26aufirst%3DS.%26aulast%3DKrastel%26aufirst%3DP.%26aulast%3DCobos-Correa%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DC.%2BN%26aulast%3DDavies%26aufirst%3DJ.%2BW%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DDark%2520Chemical%2520Matter%2520as%2520a%2520Promising%2520Starting%2520Point%2520for%2520Drug%2520Lead%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D12%26spage%3D958%26epage%3D966%26doi%3D10.1038%2Fnchembio.1936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Critical Review of the Role of HTS in Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7–8</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2006.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16580969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A280%3ADC%252BD287otlWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=277-279&issue=7%E2%80%938&author=R.+Macarron&title=Critical+Review+of+the+Role+of+HTS+in+Drug+Discovery&doi=10.1016%2Fj.drudis.2006.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Critical review of the role of HTS in drug discovery</span></div><div class="casAuthors">Macarron Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">277-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">'A wise use of lead discovery tactics will distinguish successful drug discovery engines.'</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd_nOrub-6MD69zv1FsJFMfW6udTcc2eYoCUkoFmAFK7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD287otlWjtQ%253D%253D&md5=ac1107aee5902bd961080f0a47df03c2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DCritical%2520Review%2520of%2520the%2520Role%2520of%2520HTS%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D7%25E2%2580%25938%26spage%3D277%26epage%3D279%26doi%3D10.1016%2Fj.drudis.2006.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engeloch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklin, P.</span></span> <span> </span><span class="NLM_article-title">The Novartis Compound Archive - From Concept to Reality</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.2174/1386207054867328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F1386207054867328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16178810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2005&pages=513-519&issue=6&author=U.+Schopferauthor=C.+Engelochauthor=J.+Stanekauthor=M.+Girodauthor=A.+Schuffenhauerauthor=E.+Jacobyauthor=P.+Acklin&title=The+Novartis+Compound+Archive+-+From+Concept+to+Reality&doi=10.2174%2F1386207054867328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Novartis Compound Archive - from concept to reality</span></div><div class="casAuthors">Schopfer, U.; Engeloch, C.; Stanek, J.; Girod, M.; Schuffenhauer, A.; Jacoby, E.; Acklin, P.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">513-519</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">As HTS technologies come of age, pharmaceutical companies are focusing increasingly on the quality of their screening collections.  Storage conditions and their influence on compd. stability and soly. are debated intensely.  At Novartis, a strategy was developed that is different to most other companies: (1) compds. unsuitable for storage in soln. are excluded by computational methods; (2) compds. are stored at 4°C/20% relative humidity in a DMSO/water mixt. to avoid freeze-thaw cycles and water uptake and to allow rapid plate replication; (3) resolubilization of compds. at regular intervals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr91vmOkqjoKbVg90H21EOLACvtfcHk0liZefIgnIAIwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOnu74%253D&md5=81ccc4ee90ec09e655322c6bfb3738b9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F1386207054867328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207054867328%26sid%3Dliteratum%253Aachs%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DEngeloch%26aufirst%3DC.%26aulast%3DStanek%26aufirst%3DJ.%26aulast%3DGirod%26aufirst%3DM.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DAcklin%26aufirst%3DP.%26atitle%3DThe%2520Novartis%2520Compound%2520Archive%2520-%2520From%2520Concept%2520to%2520Reality%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2005%26volume%3D8%26issue%3D6%26spage%3D513%26epage%3D519%26doi%3D10.2174%2F1386207054867328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sukuru, S. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Plate-Based Diversity Selection Based on Empirical HTS Data to Enhance the Number of Hits and Their Chemical Diversity</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1177/1087057109335678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057109335678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=19531667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVaksr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=690-699&issue=6&author=S.+C.+K.+Sukuruauthor=J.+L.+Jenkinsauthor=R.+E.+J.+Beckwithauthor=J.+Scheiberauthor=A.+Benderauthor=D.+Mikhailovauthor=J.+W.+Daviesauthor=M.+Glick&title=Plate-Based+Diversity+Selection+Based+on+Empirical+HTS+Data+to+Enhance+the+Number+of+Hits+and+Their+Chemical+Diversity&doi=10.1177%2F1087057109335678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Plate-based diversity selection based on empirical HTS data to enhance the number of hits and their chemical diversity</span></div><div class="casAuthors">Sukuru, Sai Chetan K.; Jenkins, Jeremy L.; Beckwith, Rohan E. J.; Scheiber, Josef; Bender, Andreas; Mikhailov, Dmitri; Davies, John W.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">690-699</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Typically, screening collections of pharmaceutical companies contain more than a million compds. today.  However, for certain high-throughput screening (HTS) campaigns, constraints posed by the assay throughput and/or the reagent costs make it impractical to screen the entire deck.  Therefore, it is desirable to effectively screen subsets of the collection based on a hypothesis or a diversity selection.  How to select compd. subsets is a subject of ongoing debate.  The authors present an approach based on extended connectivity fingerprints to carry out diversity selection on a per plate basis (instead of a per compd. basis).  HTS data from 35 Novartis screens spanning 5 target classes were investigated to assess the performance of this approach.  The anal. shows that selecting a fingerprint-diverse subset of 250K compds., representing 20% of the screening deck, would have achieved significantly higher hit rates for 86% of the screens.  This measure also outperforms the Murcko scaffold-based plate selection described previously, where only 49% of the screens showed similar improvements.  Strikingly, the 2-fold improvement in av. hit rates obsd. for 3 of 5 target classes in the data set indicates a target bias of the plate (and thus compd.) selection method.  Even though the diverse subset selection lacks any target hypothesis, its application shows significantly better results for some targets-namely, G-protein-coupled receptors, proteases, and protein-protein interactions-but not for kinase and pathway screens.  The synthetic origin of the compds. in the diverse subset appears to influence the screening hit rates.  Natural products were the most diverse compd. class, with significantly higher hit rates compared to the compds. from the traditional synthetic and combinatorial libraries.  These results offer empirical guidelines for plate-based diversity selection to enhance hit rates, based on target class and the library type being screened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn0GZfRsZ097Vg90H21EOLACvtfcHk0lhI44__pF5nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVaksr%252FJ&md5=fce87f6137195c5b4a6e1e9823adaa3d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1177%2F1087057109335678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109335678%26sid%3Dliteratum%253Aachs%26aulast%3DSukuru%26aufirst%3DS.%2BC.%2BK.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DPlate-Based%2520Diversity%2520Selection%2520Based%2520on%2520Empirical%2520HTS%2520Data%2520to%2520Enhance%2520the%2520Number%2520of%2520Hits%2520and%2520Their%2520Chemical%2520Diversity%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26issue%3D6%26spage%3D690%26epage%3D699%26doi%3D10.1177%2F1087057109335678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Lessons Learned from the Design of Chemical Space Networks and Opportunities for New Applications</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9906-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1007%2Fs10822-016-9906-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26945865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=191-208&issue=3&author=M.+Vogtauthor=D.+Stumpfeauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Lessons+Learned+from+the+Design+of+Chemical+Space+Networks+and+Opportunities+for+New+Applications&doi=10.1007%2Fs10822-016-9906-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the design of chemical space networks and opportunities for new applications</span></div><div class="casAuthors">Vogt, Martin; Stumpfe, Dagmar; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-208</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The concept of chem. space is of fundamental relevance in chem. informatics and computer-aided drug discovery.  In a series of articles published in the Journal of Computer-Aided Mol. Design, principles of chem. space design were evaluated, mol. networks proposed as an alternative to conventional coordinate-based chem. ref. spaces, and different types of chem. space networks (CSNs) constructed and analyzed.  Central to the generation of CSNs was the way in which mol. similarity relationships were assessed and a primary focal point was the network-based representation of biol. relevant chem. space.  The design and comparison of CSNs based upon alternative similarity measures can be viewed as an evolutionary path with interesting lessons learned along the way.  CSN design has matured to the point that such chem. space representations can be used in practice.  In this contribution, highlights from the sequence of CSN design efforts are discussed in context, providing a perspective for future practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5JT9f_RfpbVg90H21EOLACvtfcHk0lhI44__pF5nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D&md5=e5607c1fe6b9a5e74a55e8e97464bb22</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9906-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9906-3%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLessons%2520Learned%2520from%2520the%2520Design%2520of%2520Chemical%2520Space%2520Networks%2520and%2520Opportunities%2520for%2520New%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26issue%3D3%26spage%3D191%26epage%3D208%26doi%3D10.1007%2Fs10822-016-9906-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Space and Biology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span> (<span class="NLM_issue">7019</span>),  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1038/nature03192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnature03192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=15602547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=824-828&issue=7019&author=C.+M.+Dobson&title=Chemical+Space+and+Biology&doi=10.1038%2Fnature03192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space and biology</span></div><div class="casAuthors">Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. space - which encompasses all possible small org. mols., including those present in biol. systems - is vast.  So vast, in fact, that so far only a tiny fraction of it has been explored.  Nevertheless, these explorations have greatly enhanced our understanding of biol., and have led to the development of many of today's drugs.  The discovery of new bioactive mols., facilitated by a deeper understanding of the nature of the regions of chem. space that are relevant to biol., will advance our knowledge of biol. processes and lead to new strategies to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNmRTVWoSDrVg90H21EOLACvtfcHk0lhI44__pF5nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN&md5=b951fff4aa4877682756fb1831bff09d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature03192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03192%26sid%3Dliteratum%253Aachs%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520Space%2520and%2520Biology%26jtitle%3DNature%26date%3D2004%26volume%3D432%26issue%3D7019%26spage%3D824%26epage%3D828%26doi%3D10.1038%2Fnature03192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D.</span></span> <span> </span><span class="NLM_article-title">(Research Papers) Design of a Compound Screening Collection for Use in High Throughput Screening</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.2174/138620704772884832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&issue=1&author=G.+Harperauthor=S.+Pickettauthor=D.+Green&title=%28Research+Papers%29+Design+of+a+Compound+Screening+Collection+for+Use+in+High+Throughput+Screening&doi=10.2174%2F138620704772884832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0ljt8qtIxbjw5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DD.%26atitle%3D%2528Research%2520Papers%2529%2520Design%2520of%2520a%2520Compound%2520Screening%2520Collection%2520for%2520Use%2520in%2520High%2520Throughput%2520Screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26issue%3D1%26spage%3D63%26epage%3D70%26doi%3D10.2174%2F138620704772884832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koutsoukas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paricharak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galloway, W. R. J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span> <span> </span><span class="NLM_article-title">How Diverse Are Diversity Assessment Methods? A Comparative Analysis and Benchmarking of Molecular Descriptor Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1021/ci400469u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400469u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGnt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=230-242&issue=1&author=A.+Koutsoukasauthor=S.+Paricharakauthor=W.+R.+J.+D.+Gallowayauthor=D.+R.+Springauthor=A.+P.+IJzermanauthor=R.+C.+Glenauthor=D.+Marcusauthor=A.+Bender&title=How+Diverse+Are+Diversity+Assessment+Methods%3F+A+Comparative+Analysis+and+Benchmarking+of+Molecular+Descriptor+Space&doi=10.1021%2Fci400469u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">How Diverse Are Diversity Assessment Methods? A Comparative Analysis and Benchmarking of Molecular Descriptor Space</span></div><div class="casAuthors">Koutsoukas, Alexios; Paricharak, Shardul; Galloway, Warren R. J. D.; Spring, David R.; Ijzerman, Adriaan P.; Glen, Robert C.; Marcus, David; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">230-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. diversity is a widely applied approach to select structurally diverse subsets of mols., often with the objective of maximizing the no. of hits in biol. screening.  While many methods exist in the area, few systematic comparisons using current descriptors in particular with the objective of assessing diversity in bioactivity space have been published, and this shortage is what the current study is aiming to address.  13 Widely used mol. descriptors were compared, including fingerprint-based descriptors (ECFP4, FCFP4, MACCS keys), pharmacophore-based descriptors (TAT, TAD, TGT, TGD, GpiDAPH3), shape-based descriptors (rapid overlay of chem. structures (ROCS) and principal moments of inertia (PMI)), a connectivity-matrix-based descriptor (BCUT), physicochem.-property-based descriptors (prop2D), and a more recently introduced mol. descriptor type (namely, "Bayes Affinity Fingerprints").  The authors assessed both the similar behavior of the descriptors in assessing the diversity of chem. libraries, and their ability to select compds. from libraries that are diverse in bioactivity space, which is a property of much practical relevance in screening library design.  This is particularly evident, given that many future targets to be screened are not known in advance, but that the library should still maximize the likelihood of contg. bioactive matter also for future screening campaigns.  Overall, the authors' results showed that descriptors based on atom topol. (i.e., fingerprint-based descriptors and pharmacophore-based descriptors) correlate well in rank-ordering compds., both within and between descriptor types.  However, shape-based descriptors such as ROCS and PMI showed weak correlation with the other descriptors used demonstrating significantly different behavior.  The authors then applied eight of the mol. descriptors compared in this study to sample a diverse subset of sample compds. (4%) from an initial population of 2587 compds., covering the 25 largest human activity classes from ChEMBL and measured the coverage of activity classes by the subsets.  Here, "Bayes Affinity Fingerprints" achieved an av. coverage of 92% of activity classes.  Using the descriptors ECFP4, GpiDAPH3, TGT, and random sampling, 91%, 84%, 84%, and 84% of the activity classes were represented in the selected compds., resp., followed by BCUT, prop2D, MACCS, and PMI (in order of decreasing performance).  In addn., the authors were able to show that there is no visible correlation between compd. diversity in PMI space and in bioactivity space, despite frequent use of PMI plots to this end.  To summarize, the authors assessed which descriptors select compds. with high coverage of bioactivity space, and can hence be used for diverse compd. selection for biol. screening.  In cases where multiple descriptors are to be used for diversity selection, this work describes which descriptors behave complementarily and can hence be used jointly to focus on different aspects of diversity in chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojohapFOhhyLVg90H21EOLACvtfcHk0ljt8qtIxbjw5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGnt7bL&md5=f960e2e1ef89966171400e47af2f6d21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci400469u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400469u%26sid%3Dliteratum%253Aachs%26aulast%3DKoutsoukas%26aufirst%3DA.%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DGalloway%26aufirst%3DW.%2BR.%2BJ.%2BD.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DMarcus%26aufirst%3DD.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DHow%2520Diverse%2520Are%2520Diversity%2520Assessment%2520Methods%253F%2520A%2520Comparative%2520Analysis%2520and%2520Benchmarking%2520of%2520Molecular%2520Descriptor%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D1%26spage%3D230%26epage%3D242%26doi%3D10.1021%2Fci400469u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span> <span> </span><span class="NLM_article-title">Chemical Diversity and Biological Activity</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2006.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.ddtec.2006.12.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=387-395&issue=4&author=A.+Schuffenhauerauthor=N.+Brown&title=Chemical+Diversity+and+Biological+Activity&doi=10.1016%2Fj.ddtec.2006.12.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2006.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2006.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DChemical%2520Diversity%2520and%2520Biological%2520Activity%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2006%26volume%3D3%26issue%3D4%26spage%3D387%26epage%3D395%26doi%3D10.1016%2Fj.ddtec.2006.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Probing the Probes: Fitness Factors For Small Molecule Tools</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2010.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20609406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=561-577&issue=6&author=P.+Workmanauthor=I.+Collins&title=Probing+the+Probes%3A+Fitness+Factors+For+Small+Molecule+Tools&doi=10.1016%2Fj.chembiol.2010.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Probes: Fitness Factors For Small Molecule Tools</span></div><div class="casAuthors">Workman, Paul; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">561-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. probes for interrogating biol. processes are of considerable current interest.  Cell permeable small mol. tools have a major role in facilitating the functional annotation of the human genome, understanding both physiol. and pathol. processes, and validating new mol. targets.  To be valuable, chem. tools must satisfy necessary criteria and recent publications have suggested objective guidelines for what makes a useful chem. probe.  Although recognizing that such guidelines may be valuable, we caution against overly restrictive rules that may stifle innovation in favor of a "fit-for-purpose" approach.  Reviewing the literature and providing examples from the cancer field, we recommend a series of "fitness factors" to be considered when assessing chem. probes.  We hope this will encourage innovative chem. biol. research while minimizing the generation of poor quality and misleading biol. data, thus increasing understanding of the particular biol. area, to the benefit of basic research and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQtty9oZD17rVg90H21EOLACvtfcHk0ljt8qtIxbjw5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D&md5=16be01da2339173aae8207030bf83c12</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DProbing%2520the%2520Probes%253A%2520Fitness%2520Factors%2520For%2520Small%2520Molecule%2520Tools%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26issue%3D6%26spage%3D561%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2010.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The Promise and Peril of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+Promise+and+Peril+of+Chemical+Probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lgIlRJzyNvRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520Promise%2520and%2520Peril%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span> <span> </span><span class="NLM_article-title">Biological Assay Challenges from Compound Solubility: Strategies for Bioassay Optimization</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9–10</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2006.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16635808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvVGns7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=446-451&issue=9%E2%80%9310&author=L.+Diauthor=E.+H.+Kerns&title=Biological+Assay+Challenges+from+Compound+Solubility%3A+Strategies+for+Bioassay+Optimization&doi=10.1016%2Fj.drudis.2006.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Biological assay challenges from compounds solubility: strategies for bioassay optimization</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9 & 10</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Compd. soly. in buffers and DMSO (DMSO) has emerged as an important issue.  Many discovery compds. have low soly. but are potentially valuable as leads.  Unfortunately, low soly. affects bioassays by causing underestimated activity, reduced HTS-hit rates, variable data, inaccurate SAR, discrepancies between enzyme and cell assays and inaccurate in vitro ADME-Tox testing.  Strategies for optimizing bioassays include: considering soly. in HTS-library design; early screening for soly.; improving storage and handling of DMSO stocks; optimizing diln. protocols; and ensuring that low-soly. compds. are fully solubilized in bioassays.  These approaches allow for adequate assessments of valuable pharmacophores for which soly. can be chem. optimized at a later date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWFI1UiXIDLVg90H21EOLACvtfcHk0lgIlRJzyNvRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvVGns7g%253D&md5=1814b6b9d1d8cb77540c42c37509a736</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DBiological%2520Assay%2520Challenges%2520from%2520Compound%2520Solubility%253A%2520Strategies%2520for%2520Bioassay%2520Optimization%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D9%25E2%2580%259310%26spage%3D446%26epage%3D451%26doi%3D10.1016%2Fj.drudis.2006.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGovern, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helfand, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A Specific Mechanism of Nonspecific Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">4265</span>– <span class="NLM_lpage">4272</span>, <span class="refDoi"> DOI: 10.1021/jm030266r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030266r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4265-4272&issue=20&author=S.+L.+McGovernauthor=B.+T.+Helfandauthor=B.+Fengauthor=B.+K.+Shoichet&title=A+Specific+Mechanism+of+Nonspecific+Inhibition&doi=10.1021%2Fjm030266r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Specific Mechanism of Nonspecific Inhibition</span></div><div class="casAuthors">McGovern, Susan L.; Helfand, Brian T.; Feng, Brian; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4265-4272</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous small mols. plague screening libraries and hit lists.  Previous work has found that several nonspecific compds. form submicrometer aggregates, and it has been suggested that this aggregate species is responsible for the inhibition of many different enzymes.  It is not understood how aggregates inhibit their targets.  To address this question, biophys., kinetic, and microscopy methods were used to study the interaction of promiscuous, aggregate-forming inhibitors with model proteins.  By use of centrifugation and gel electrophoresis, aggregates and protein were found to directly interact.  This is consistent with a subsequent observation from confocal fluorescence microscopy that aggregates conc. green fluorescent protein.  β-Lactamase mutants with increased or decreased thermodn. stability relative to wild-type enzyme were equally inhibited by an aggregate-forming compd., suggesting that denaturation by unfolding was not the primary mechanism of interaction.  Instead, visualization by electron microscopy revealed that enzyme assocs. with the surface of inhibitor aggregates.  This assocn. could be reversed or prevented by the addn. of Triton X-100.  These observations suggest that the aggregates formed by promiscuous compds. reversibly sequester enzyme, resulting in apparent inhibition.  They also suggest a simple method to identify or reverse the action of aggregate-based inhibitors, which appear to be widespread.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppdLVG_GeLZbVg90H21EOLACvtfcHk0ljv5EFfpVEGiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D&md5=cd0fd6bf0a64ceb7d7219dae2406dd21</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm030266r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030266r%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DHelfand%26aufirst%3DB.%2BT.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520Specific%2520Mechanism%2520of%2520Nonspecific%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26issue%3D20%26spage%3D4265%26epage%3D4272%26doi%3D10.1021%2Fjm030266r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span> <span> </span><span class="NLM_article-title">High Throughput Solubility Measurement in Drug Discovery and Development</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.addr.2007.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=17604872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=546-567&issue=7&author=J.+Alsenzauthor=M.+Kansy&title=High+Throughput+Solubility+Measurement+in+Drug+Discovery+and+Development&doi=10.1016%2Fj.addr.2007.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput solubility measurement in drug discovery and development</span></div><div class="casAuthors">Alsenz, Jochem; Kansy, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">546-567</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Measurement of drug soly. in various solvents is one of the key elements of compd. characterization during the whole discovery and development process.  This review summarizes current exptl. approaches and addresses recent advances in the exptl. methods used to det. drug soly. in drug discovery and early development.  This paper focuses on high throughput methods designed to det. kinetic and thermodn. (equil.) soly. but traditional methods are also presented.  The focus, positioning, exptl. setup, pros and cons, and limitations of individual assays are discussed and differences in soly. studies in discovery and development environments are highlighted.  Finally, future needs and trends in soly. assay development designed to overcome current bottlenecks and trade-offs between speed and quality/quantity of measurements are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsvOlLd-yoxLVg90H21EOLACvtfcHk0ljv5EFfpVEGiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrw%253D&md5=8d55ead3ea0c13fa49b032db725842b0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DAlsenz%26aufirst%3DJ.%26aulast%3DKansy%26aufirst%3DM.%26atitle%3DHigh%2520Throughput%2520Solubility%2520Measurement%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2007%26volume%3D59%26issue%3D7%26spage%3D546%26epage%3D567%26doi%3D10.1016%2Fj.addr.2007.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Exploratory Analysis of Kinetic Solubility Measurements of a Small Molecule Library</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4127</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmc.2011.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21640593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4127-4134&issue=13&author=R.+Guhaauthor=T.+S.+Dexheimerauthor=A.+N.+Kestranekauthor=A.+Jadhavauthor=A.+M.+Chervenakauthor=M.+G.+Fordauthor=A.+Simeonovauthor=G.+P.+Rothauthor=C.+J.+Thomas&title=Exploratory+Analysis+of+Kinetic+Solubility+Measurements+of+a+Small+Molecule+Library&doi=10.1016%2Fj.bmc.2011.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploratory analysis of kinetic solubility measurements of a small molecule library</span></div><div class="casAuthors">Guha, Rajarshi; Dexheimer, Thomas S.; Kestranek, Aimee N.; Jadhav, Ajit; Chervenak, Andrew M.; Ford, Michael G.; Simeonov, Anton; Roth, Gregory P.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4127-4134</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Kinetic soly. measurements using prototypical assay buffer conditions are presented for a ∼58,000 member library of small mols.  Analyses of the data based upon phys. and calcd. properties of each individual mol. were performed and resulting trends were considered in the context of commonly held opinions of how physicochem. properties influence aq. soly.  The data was further analyzed using a decision tree model for soly. prediction and via a multi-dimensional assessment of physicochem. relationships to soly. in the context of specific rule-breakers relative to common dogma.  The role of soly. as a determinant of assay outcome is also considered based upon each compd.'s cross-assay activity score for a collection of publicly available screening results.  Further, the role of soly. as a governing factor for colloidal aggregation formation within a specified assay setting is examd. and considered as a possible cause of a high cross-assay activity score.  The results of this soly. profile should aid chemists during library design and optimization efforts and represent a useful training set for computational soly. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU9biLJPDeaLVg90H21EOLACvtfcHk0lh4z1X7ApxCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D&md5=1f5b67268444cd3911665ea395bbf72f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DKestranek%26aufirst%3DA.%2BN.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%2BM.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DExploratory%2520Analysis%2520of%2520Kinetic%2520Solubility%2520Measurements%2520of%2520a%2520Small%2520Molecule%2520Library%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26issue%3D13%26spage%3D4127%26epage%3D4134%26doi%3D10.1016%2Fj.bmc.2011.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10594</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10520-10594&issue=18&author=X.+Yangauthor=Y.+Wangauthor=R.+Byrneauthor=G.+Schneiderauthor=S.+Yang&title=Concepts+of+Artificial+Intelligence+for+Computer-Assisted+Drug+Discovery&doi=10.1021%2Facs.chemrev.8b00728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span></div><div class="casAuthors">Yang, Xin; Wang, Yifei; Byrne, Ryan; Schneider, Gisbert; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10520-10594</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Artificial intelligence (AI), and, in particular, deep learning as a subcategory of AI, provides opportunities for the discovery and development of innovative drugs.  Various machine learning approaches have recently (re)emerged, some of which may be considered instances of domain-specific AI which have been successfully employed for drug discovery and design.  This review provides a comprehensive portrayal of these machine learning techniques and of their applications in medicinal chem.  After introducing the basic principles, alongside some application notes, of the various machine learning algorithms, the current state-of-the art of AI-assisted pharmaceutical discovery is discussed, including applications in structure- and ligand-based virtual screening, de novo drug design, physicochem. and pharmacokinetic property prediction, drug repurposing, and related aspects.  Finally, several challenges and limitations of the current methods are summarized, with a view to potential future directions for AI-assisted drug discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUfBzNv587rVg90H21EOLACvtfcHk0lh4z1X7ApxCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN&md5=6e9816e30251cd4f15c831e2dd20f100</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00728%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DConcepts%2520of%2520Artificial%2520Intelligence%2520for%2520Computer-Assisted%2520Drug%2520Discovery%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26issue%3D18%26spage%3D10520%26epage%3D10594%26doi%3D10.1021%2Facs.chemrev.8b00728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Breemen, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Caco-2 Cell Permeability Assays to Measure Drug Absorption</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1517/17425255.1.2.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1517%2F17425255.1.2.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=16922635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=175-185&issue=2&author=R.+B.+Van+Breemenauthor=Y.+Li&title=Caco-2+Cell+Permeability+Assays+to+Measure+Drug+Absorption&doi=10.1517%2F17425255.1.2.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Caco-2 cell permeability assays to measure drug absorption</span></div><div class="casAuthors">van Breemen, Richard B.; Li, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-185</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Caco-2 cells are a human colon epithelial cancer cell line used as a model of human intestinal absorption of drugs and other compds.  When cultured as a monolayer, Caco-2 cells differentiate to form tight junctions between cells to serve as a model of paracellular movement of compds. across the mono-layer.  In addn., Caco-2 cells express transporter proteins, efflux proteins, and Phase II conjugation enzymes to model a variety of transcellular pathways as well as metabolic transformation of test substances.  In many respects, the Caco-2 cell monolayer mimics the human intestinal epithelium.  One of the functional differences between normal cells and Caco-2 cells is the lack of expression of the cytochrome P 450 isoenzymes and in particular, CYP3A4, which is normally expressed at high levels in the intestine.  However, Caco-2 cells may be induced to express higher levels of CYP3A4 by treatment with vitamin D3.  Caco-2 cell monolayers are usually cultured on semipermeable plastic supports that may be fitted into the wells of multi-well culture plates.  Test compds. are then added to either the apical or basolateral sides of the monolayer.  After incubation for various lengths of time, aliquots of the buffer in opposite chambers are removed for the detn. of the concn. of test compds. and the computation of the rates of permeability for each compd. (called the apparent permeability coeffs.).  Although radiolabeled compds. were used in the original Caco-2 cells monolayer assays, radio-labeled compds. have been replaced in most labs. by the use of liq. chromatog.-mass spectrometry (LC-MS) and LC-tandem mass spectrometry (LC-MS-MS).  Mass spectrometry not only eliminates the need for radiolabeled compds., but permits the simultaneous measurement of multiple compds.  The measurement of multiple compds. per assay reduces the no. of incubations that need to be carried out, thereby increasing the throughput of the expts.  Furthermore, LC-MS and LC-MS-MS add another dimension to Caco-2 assays by facilitating the investigation of the metab. of compds. by Caco-2 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcV-AZvy78ILVg90H21EOLACvtfcHk0lh4z1X7ApxCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK&md5=7bd0b875ffa9c840f904bab6513b7e3f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1517%2F17425255.1.2.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.1.2.175%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BBreemen%26aufirst%3DR.%2BB.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DCaco-2%2520Cell%2520Permeability%2520Assays%2520to%2520Measure%2520Drug%2520Absorption%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2005%26volume%3D1%26issue%3D2%26spage%3D175%26epage%3D185%26doi%3D10.1517%2F17425255.1.2.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&issue=7&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lh4z1X7ApxCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D7%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitney-Pickett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhard, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federico, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyner, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempshall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-kattan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liston, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span> <span> </span><span class="NLM_article-title">Development of a New Permeability Assay Using Low-Efflux MDCKII Cells</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4974</span>– <span class="NLM_lpage">4985</span>, <span class="refDoi"> DOI: 10.1002/jps.22674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fjps.22674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=21766308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovFGltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=4974-4985&issue=11&author=L.+Diauthor=C.+Whitney-Pickettauthor=J.+P.+Umlandauthor=H.+Zhangauthor=X.+Zhangauthor=D.+F.+Gebhardauthor=Y.+Laiauthor=J.+J.+Federicoauthor=R.+E.+Davidsonauthor=R.+Smithauthor=E.+L.+Reynerauthor=C.+Leeauthor=B.+Fengauthor=C.+Rotterauthor=M.+V.+Varmaauthor=S.+Kempshallauthor=K.+Fennerauthor=A.+F.+El-kattanauthor=T.+E.+Listonauthor=M.+D.+Troutman&title=Development+of+a+New+Permeability+Assay+Using+Low-Efflux+MDCKII+Cells&doi=10.1002%2Fjps.22674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a new permeability assay using low-efflux MDCKII cells</span></div><div class="casAuthors">Di, Li; Whitney-Pickett, Carrie; Umland, John P.; Zhang, Hui; Zhang, Xun; Gebhard, David F.; Lai, Yurong; Federico, James J.; Davidson, Ralph E.; Smith, Russ; Reyner, Eric L.; Lee, Caroline; Feng, Bo; Rotter, Charles; Varma, Manthena V.; Kempshall, Sarah; Fenner, Katherine; El-kattan, Ayman F.; Liston, Theodore E.; Troutman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4974-4985</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Permeability is an important property of drug candidates.  The Madin-Darby canine kidney cell line (MDCK) permeability assay is widely used and the primary concern of using MDCK cells is the presence of endogenous transporters of nonhuman origin.  The canine P-glycoprotein (Pgp) can interfere with permeability and transporter studies, leading to less reliable data.  A new cell line, MDCKII-LE (low efflux), has been developed by selecting a subpopulation of low-efflux cells from MDCKII-WT using an iterative fluorescence-activated cell sorting technique with calcein-AM as a Pgp and efflux substrate.  MDCKII-LE cells are a subpopulation of MDCKII cells with over 200-fold lower canine Pgp mRNA level and fivefold lower protein level than MDCKII-WT.  MDCKII-LE cells showed less functional efflux activity than MDCKII-WT based on efflux ratios.  Notably, MDCKII-MDR1 showed about 1.5-fold decreased expression of endogenous canine Pgp, suggesting that using the net flux ratio might not completely cancel out the background endogenous transporter activities.  MDCKII-LE cells offer clear advantages over the MDCKII-WT by providing less efflux transporter background signals and minimizing interference from canine Pgp.  The MDCKII-LE apparent permeability values well differentiates compds. from high to medium/low human intestinal absorption and can be used for Biopharmaceutical Classification System.  The MDCKII-LE permeability assay (4-in-1 cassette dosing) is high throughput with good precision, reproducibility, robustness, and cost-effective. © 2011 Wiley-Liss, Inc. and the American Pharmacists Assocn. J Pharm Sci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVtRmYnbF17Vg90H21EOLACvtfcHk0lhEWf8OUqYsWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovFGltb8%253D&md5=97f647fc215ea6d04882351ffc912a1e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjps.22674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22674%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DWhitney-Pickett%26aufirst%3DC.%26aulast%3DUmland%26aufirst%3DJ.%2BP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGebhard%26aufirst%3DD.%2BF.%26aulast%3DLai%26aufirst%3DY.%26aulast%3DFederico%26aufirst%3DJ.%2BJ.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DReyner%26aufirst%3DE.%2BL.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DRotter%26aufirst%3DC.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DKempshall%26aufirst%3DS.%26aulast%3DFenner%26aufirst%3DK.%26aulast%3DEl-kattan%26aufirst%3DA.%2BF.%26aulast%3DListon%26aufirst%3DT.%2BE.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520New%2520Permeability%2520Assay%2520Using%2520Low-Efflux%2520MDCKII%2520Cells%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26issue%3D11%26spage%3D4974%26epage%3D4985%26doi%3D10.1002%2Fjps.22674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">Frequent Hitters: Nuisance Artifacts in High-Throughput Screening</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">667</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2020.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31987936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=657-667&issue=4&author=Z.+Y.+Yangauthor=J.+H.+Heauthor=A.+P.+Luauthor=T.+J.+Houauthor=D.+S.+Cao&title=Frequent+Hitters%3A+Nuisance+Artifacts+in+High-Throughput+Screening&doi=10.1016%2Fj.drudis.2020.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Frequent hitters: nuisance artifacts in high-throughput screening</span></div><div class="casAuthors">Yang, Zi-Yi; He, Jun-Hong; Lu, Ai-Ping; Hou, Ting-Jun; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-667</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the major challenges in early drug discovery is the recognition of frequent hitters (FHs), i.e., compds. that nonspecifically bind to a range of macromol. targets or false positives caused by various types of assay interferences.  In this review, we survey the mechanisms underlying different types of FHs, including aggregators, spectroscopic interference compds. (i.e., luciferase inhibitors and fluorescent compds.), chem. reactive compds., and promiscuous compds.  We also review commonly used exptl. detection techniques and computational prediction models for FH identification.  In addn., the rational applications of these computational filters are discussed.  It is believed that, with the rational use of FH filters, the efficiency of drug discovery will be significantly improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA9bpJANksO7Vg90H21EOLACvtfcHk0lhEWf8OUqYsWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWjur0%253D&md5=9a5f8c8396815ce9cb84203d1d41dc47</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DHou%26aufirst%3DT.%2BJ.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DFrequent%2520Hitters%253A%2520Nuisance%2520Artifacts%2520in%2520High-Throughput%2520Screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26issue%3D4%26spage%3D657%26epage%3D667%26doi%3D10.1016%2Fj.drudis.2020.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dantas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, T. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Junior, F. P.</span></span> <span> </span><span class="NLM_article-title">Dealing with Frequent Hitters in Drug Discovery: A Multidisciplinary View on the Issue of Filtering Compounds on Biological Screenings</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1282</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1654453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1080%2F17460441.2019.1654453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31416369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyksbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1269-1282&issue=12&author=R.+F.+Dantasauthor=T.+C.+S.+Evangelistaauthor=B.+J.+Nevesauthor=M.+R.+Sengerauthor=C.+H.+Andradeauthor=S.+B.+Ferreiraauthor=F.+P.+Silva-Junior&title=Dealing+with+Frequent+Hitters+in+Drug+Discovery%3A+A+Multidisciplinary+View+on+the+Issue+of+Filtering+Compounds+on+Biological+Screenings&doi=10.1080%2F17460441.2019.1654453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dealing with frequent hitters in drug discovery: a multidisciplinary view on the issue of filtering compounds on biological screenings</span></div><div class="casAuthors">Dantas, Rafael Ferreira; Evangelista, Tereza Cristina Santos; Neves, Bruno, Jr.; Senger, Mario Roberto; Andrade, Carolina Horta; Ferreira, Sabrina Baptista; Silva-Junior, Floriano Paes</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1269-1282</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The timely identification biol. active chems., in disease relevant screening assays, is a major endeavor in drug discovery.  The existence of frequent hitters (FHs) in non-related assays poses a formidable challenge in terms of whether to consider these mols. as chem. gold or promiscuous non-selective reactive trash (also known as PAINS-pan assay interference compds.).  In this review, the authors bring together expertize in synthetic chem., cheminformatics and biochem., three key areas for dealing with FHs.  They discuss synthetic methods facilitating prepn. of chem. diverse mol. libraries, while favoring activity in the biol. space.  They also survey and discuss recent computational advances in the prediction of PAINS from chem. structures.  Finally, they review exptl. approaches for the validation of the biol. activity of screening hits and discuss alternatives for exploiting promiscuity and chem. reactivity.: It's essential to develop more efficient computational methods to reliably recognize PAINS in distinct mol. environments.  Accordingly, advances in synthetic chem. hold the promise to provide a better quality of chem. matter for drug discovery.  Medicinal chemists should be more open to screening for hits showing biol. complex mechanisms of action rather than discarding mols. that may prove valuable as innovative disease treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4hxv43QT26LVg90H21EOLACvtfcHk0li7QPYB0_h8jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyksbzM&md5=d63f74505e6638ea1fba8486775215db</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1654453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1654453%26sid%3Dliteratum%253Aachs%26aulast%3DDantas%26aufirst%3DR.%2BF.%26aulast%3DEvangelista%26aufirst%3DT.%2BC.%2BS.%26aulast%3DNeves%26aufirst%3DB.%2BJ.%26aulast%3DSenger%26aufirst%3DM.%2BR.%26aulast%3DAndrade%26aufirst%3DC.%2BH.%26aulast%3DFerreira%26aufirst%3DS.%2BB.%26aulast%3DSilva-Junior%26aufirst%3DF.%2BP.%26atitle%3DDealing%2520with%2520Frequent%2520Hitters%2520in%2520Drug%2520Discovery%253A%2520A%2520Multidisciplinary%2520View%2520on%2520the%2520Issue%2520of%2520Filtering%2520Compounds%2520on%2520Biological%2520Screenings%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D12%26spage%3D1269%26epage%3D1282%26doi%3D10.1080%2F17460441.2019.1654453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakravorty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa-Burke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remlinger, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarrón, R.</span></span> <span> </span><span class="NLM_article-title">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1177/2472555218768497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F2472555218768497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=29699447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=532-545&issue=6&author=S.+J.+Chakravortyauthor=J.+Chanauthor=M.+N.+Greenwoodauthor=I.+Popa-Burkeauthor=K.+S.+Remlingerauthor=S.+D.+Pickettauthor=D.+V.+S.+Greenauthor=M.+C.+Fillmoreauthor=T.+W.+Deanauthor=J.+I.+Luengoauthor=R.+Macarr%C3%B3n&title=Nuisance+Compounds%2C+PAINS+Filters%2C+and+Dark+Chemical+Matter+in+the+GSK+HTS+Collection&doi=10.1177%2F2472555218768497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span></div><div class="casAuthors">Chakravorty, Subhas J.; Chan, James; Greenwood, Marie Nicole; Popa-Burke, Ioana; Remlinger, Katja S.; Pickett, Stephen D.; Green, Darren V. S.; Fillmore, Martin C.; Dean, Tony W.; Luengo, Juan I.; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">532-545</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) hits include compds. with undesirable properties.  Many filters have been described to identify such hits.  Notably, pan-assay interference compds. (PAINS) has been adopted by the community as the std. term to refer to such filters, and very useful guidelines have been adopted by the American Chem. Society (ACS) and subsequently triggered a healthy scientific debate about the pitfalls of draconian use of filters.  Using an inhibitory frequency index, we have analyzed in detail the promiscuity profile of the whole GlaxoSmithKline (GSK) HTS collection comprising more than 2 million unique compds. that have been tested in hundreds of screening assays.  We provide a comprehensive anal. of many previously published filters and newly described classes of nuisance structures that may serve as a useful source of empirical information to guide the design or growth of HTS collections and hit triaging strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5bHsBxkwOrVg90H21EOLACvtfcHk0li7QPYB0_h8jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE&md5=9a620fb177aa001b0a36e90948f8e326</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F2472555218768497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218768497%26sid%3Dliteratum%253Aachs%26aulast%3DChakravorty%26aufirst%3DS.%2BJ.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreenwood%26aufirst%3DM.%2BN.%26aulast%3DPopa-Burke%26aufirst%3DI.%26aulast%3DRemlinger%26aufirst%3DK.%2BS.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DMacarr%25C3%25B3n%26aufirst%3DR.%26atitle%3DNuisance%2520Compounds%252C%2520PAINS%2520Filters%252C%2520and%2520Dark%2520Chemical%2520Matter%2520in%2520the%2520GSK%2520HTS%2520Collection%26jtitle%3DSLAS%2520Discovery%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D532%26epage%3D545%26doi%3D10.1177%2F2472555218768497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasial, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Highly Promiscuous Small Molecules from Biological Screening Assays Include Many Pan-Assay Interference Compounds but Also Candidates for Polypharmacology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">10285</span>– <span class="NLM_lpage">10290</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10285-10290&issue=22&author=E.+Gilbergauthor=S.+Jasialauthor=D.+Stumpfeauthor=D.+Dimovaauthor=J.+Bajorath&title=Highly+Promiscuous+Small+Molecules+from+Biological+Screening+Assays+Include+Many+Pan-Assay+Interference+Compounds+but+Also+Candidates+for+Polypharmacology&doi=10.1021%2Facs.jmedchem.6b01314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Promiscuous Small Molecules from Biological Screening Assays Include Many Pan-Assay Interference Compounds but Also Candidates for Polypharmacology</span></div><div class="casAuthors">Gilberg, Erik; Jasial, Swarit; Stumpfe, Dagmar; Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10285-10290</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In PubChem screening assays, 466 highly promiscuous compds. were identified that were examd. for known pan-assay interference compds. (PAINS) and aggregators using publicly available filters.  These filters detected 210 PAINS and 67 aggregators.  Compds. passing the filters included addnl. PAINS that were not detected, mostly due to tautomerism, and a variety of other potentially reactive compds. currently not encoded as PAINS.  For a subset of compds. passing the filters, there was no evidence of potential artifacts.  These compds. are considered candidates for further exploring multitarget activities and the mol. basis of polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCj2gzF-ovLVg90H21EOLACvtfcHk0lh6KTlaAKtcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOlsr7O&md5=55d062a51de53000f1de0952e631b0a9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01314%26sid%3Dliteratum%253Aachs%26aulast%3DGilberg%26aufirst%3DE.%26aulast%3DJasial%26aufirst%3DS.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DHighly%2520Promiscuous%2520Small%2520Molecules%2520from%2520Biological%2520Screening%2520Assays%2520Include%2520Many%2520Pan-Assay%2520Interference%2520Compounds%2520but%2520Also%2520Candidates%2520for%2520Polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D22%26spage%3D10285%26epage%3D10290%26doi%3D10.1021%2Facs.jmedchem.6b01314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleason, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beatini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Q. F.</span></span> <span> </span><span class="NLM_article-title">Metal Impurities Cause False Positives in High-Throughput Screening Campaigns</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1021/ml3003296</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3003296" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVanu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=197-200&issue=2&author=J.+C.+Hermannauthor=Y.+Chenauthor=C.+Wartchowauthor=J.+Menkeauthor=L.+Gaoauthor=S.+K.+Gleasonauthor=N.+E.+Haynesauthor=N.+Scottauthor=A.+Petersenauthor=S.+Gabrielauthor=B.+Vuauthor=K.+M.+Georgeauthor=A.+Narayananauthor=S.+H.+Liauthor=H.+Qianauthor=N.+Beatiniauthor=L.+Niuauthor=Q.+F.+Gan&title=Metal+Impurities+Cause+False+Positives+in+High-Throughput+Screening+Campaigns&doi=10.1021%2Fml3003296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Metal Impurities Cause False Positives in High-Throughput Screening Campaigns</span></div><div class="casAuthors">Hermann, Johannes C.; Chen, Yingsi; Wartchow, Charles; Menke, John; Gao, Lin; Gleason, Shelley K.; Haynes, Nancy-Ellen; Scott, Nathan; Petersen, Ann; Gabriel, Stephen; Vu, Binh; George, Kelly M.; Narayanan, Arjun; Li, Shirley H.; Qian, Hong; Beatini, Nanda; Niu, Linghao; Gan, Qing-Fen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-200</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Org. impurities in compd. libraries are known to often cause false-pos. signals in screening campaigns for new leads, but org. impurities do not fully account for all false-pos. results.  We discovered inorg. impurities in our screening library that can also cause pos. signals for a variety of targets and/or readout systems, including biochem. and biosensor assays.  We investigated in depth the example of zinc for a specific project and in retrospect in various HTS screens at Roche and propose a straight-forward counter screen using the chelator TPEN to rule out inhibition caused by zinc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF424s97a80bVg90H21EOLACvtfcHk0lh6KTlaAKtcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVanu77J&md5=24419d1ca51f21226f7774eb1d38b93f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml3003296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003296%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWartchow%26aufirst%3DC.%26aulast%3DMenke%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DGleason%26aufirst%3DS.%2BK.%26aulast%3DHaynes%26aufirst%3DN.%2BE.%26aulast%3DScott%26aufirst%3DN.%26aulast%3DPetersen%26aufirst%3DA.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DVu%26aufirst%3DB.%26aulast%3DGeorge%26aufirst%3DK.%2BM.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%2BH.%26aulast%3DQian%26aufirst%3DH.%26aulast%3DBeatini%26aufirst%3DN.%26aulast%3DNiu%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DQ.%2BF.%26atitle%3DMetal%2520Impurities%2520Cause%2520False%2520Positives%2520in%2520High-Throughput%2520Screening%2520Campaigns%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26issue%3D2%26spage%3D197%26epage%3D200%26doi%3D10.1021%2Fml3003296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of Frequent-Hitter Behavior Based on Historical High-Throughput Screening Data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&issue=10&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+Frequent-Hitter+Behavior+Based+on+Historical+High-Throughput+Screening+Data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0lhlS61-i43Prw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520Frequent-Hitter%2520Behavior%2520Based%2520on%2520Historical%2520High-Throughput%2520Screening%2520Data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D10%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">David, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feierberg, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span> <span> </span><span class="NLM_article-title">Identification of Compounds That Interfere with High-Throughput Screening Assay Technologies</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">1795</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.201900395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=31479198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsl2gtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1795-1802&issue=20&author=L.+Davidauthor=J.+Walshauthor=N.+Sturmauthor=I.+Feierbergauthor=J.+W.+M.+Nissinkauthor=H.+Chenauthor=J.+Bajorathauthor=O.+Engkvist&title=Identification+of+Compounds+That+Interfere+with+High-Throughput+Screening+Assay+Technologies&doi=10.1002%2Fcmdc.201900395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Compounds That Interfere with High-Throughput Screening Assay Technologies</span></div><div class="casAuthors">David, Laurianne; Walsh, Jarrod; Sturm, Noe; Feierberg, Isabella; Nissink, J. Willem M.; Chen, Hongming; Bajorath, Juergen; Engkvist, Ola</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1795-1802</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A significant challenge in high-throughput screening (HTS) campaigns is the identification of assay technol. interference compds.  A Compd. Interfering with an Assay Technol. (CIAT) gives false readouts in many assays.  CIATs are often considered viable hits and investigated in follow-up studies, thus impeding research and wasting resources.  In this study, the authors developed a machine-learning (ML) model to predict CIATs for three assay technologies.  The model was trained on known CIATs and non-CIATs (NCIATs) identified in artifact assays and described by their 2D structural descriptors.  Usual methods identifying CIATs are based on statistical anal. of historical primary screening data and do not consider exptl. assays identifying CIATs.  The results show successful prediction of CIATs for existing and novel compds. and provide a complementary and wider set of predicted CIATs compared to BSF, a published structure-independent model, and to the PAINS substructural filters.  The anal. is an example of how well-curated datasets can provide powerful predictive models despite their relatively small size.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLDqUbFKxxrVg90H21EOLACvtfcHk0lhlS61-i43Prw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsl2gtb%252FP&md5=d40c09bf89d6e92de742ab3367d1caf8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900395%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DWalsh%26aufirst%3DJ.%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DFeierberg%26aufirst%3DI.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DIdentification%2520of%2520Compounds%2520That%2520Interfere%2520with%2520High-Throughput%2520Screening%2520Assay%2520Technologies%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D20%26spage%3D1795%26epage%3D1802%26doi%3D10.1002%2Fcmdc.201900395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhlS61-i43Prw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017 - Utility and Limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&issue=1&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+Year+Itch%3A+Pan-Assay+Interference+Compounds+%28PAINS%29+in+2017+-+Utility+and+Limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0lh2Jk-jnp7EIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520Year%2520Itch%253A%2520Pan-Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520in%25202017%2520-%2520Utility%2520and%2520Limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D1%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunavs, M.</span></span> <span> </span><span class="NLM_article-title">Investigating the Behavior of Published PAINS Alerts Using a Pharmaceutical Company Data Set</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12jsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=792-796&issue=8&author=L.+R.+Vidlerauthor=I.+A.+Watsonauthor=B.+J.+Margolisauthor=D.+J.+Cumminsauthor=M.+Brunavs&title=Investigating+the+Behavior+of+Published+PAINS+Alerts+Using+a+Pharmaceutical+Company+Data+Set&doi=10.1021%2Facsmedchemlett.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating the Behavior of Published PAINS Alerts Using a Pharmaceutical Company Data Set</span></div><div class="casAuthors">Vidler, Lewis R.; Watson, Ian A.; Margolis, Brandon J.; Cummins, David J.; Brunavs, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">792-796</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Biochem. assay interference is becoming increasingly recognized as a significant waste of resource in drug discovery, both in industry and academia.  A seminal publication from Baell and Holloway raised the awareness of this issue, and they published a set of alerts to identify what they described as PAINS (pan-assay interference compds.).  These alerts have been taken up by drug discovery groups, even though the original paper had a somewhat limited data set.  Here, we have taken Lilly's far larger internal data set to assess the PAINS alerts on four criteria: promiscuity (over six assay formats including AlphaScreen), compd. stability, cytotoxicity, and presence of a high Hill slope as a surrogate for non-1:1 protein-ligand binding.  It was found that only three of the alerts show pan-assay promiscuity, and the alerts appear to encode primarily AlphaScreen promiscuous mols.  Although not enriching for pan-assay promiscuity, many of the alerts do encode mols. that are unstable, show cytotoxicity, and increase the prevalence of high Hill slopes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhzWMrqUkHVLVg90H21EOLACvtfcHk0lh2Jk-jnp7EIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12jsLbL&md5=794c7338ab2ef2dc8da60cb2696760ff</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DMargolis%26aufirst%3DB.%2BJ.%26aulast%3DCummins%26aufirst%3DD.%2BJ.%26aulast%3DBrunavs%26aufirst%3DM.%26atitle%3DInvestigating%2520the%2520Behavior%2520of%2520Published%2520PAINS%2520Alerts%2520Using%2520a%2520Pharmaceutical%2520Company%2520Data%2520Set%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D8%26spage%3D792%26epage%3D796%26doi%3D10.1021%2Facsmedchemlett.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenaigner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinshagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span> <span> </span><span class="NLM_article-title">Identification of Small-Molecule Frequent Hitters from Alphascreen High-Throughput Screens</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1177/1087057113516861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057113516861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=24371213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=715-726&issue=5&author=K.+Schorppauthor=I.+Rothenaignerauthor=E.+Salminaauthor=J.+Reinshagenauthor=T.+Lowauthor=J.+K.+Brenkeauthor=J.+Gopalakrishnanauthor=I.+V.+Tetkoauthor=S.+Gulauthor=K.+Hadian&title=Identification+of+Small-Molecule+Frequent+Hitters+from+Alphascreen+High-Throughput+Screens&doi=10.1177%2F1087057113516861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small-molecule frequent hitters from Alphascreen high-throughput screens</span></div><div class="casAuthors">Schorpp, Kenji; Rothenaigner, Ina; Salmina, Elena; Reinshagen, Jeanette; Low, Terence; Brenke, Jara K.; Gopalakrishnan, Jay; Tetko, Igor V.; Gul, Sheraz; Hadian, Kamyar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">715-726</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Although small-mol. drug discovery efforts have focused largely on enzyme, receptor, and ion-channel targets, there has been an increase in such activities to search for protein-protein interaction (PPI) disruptors by applying high-throughout screening (HTS)-compatible protein-binding assays.  However, a disadvantage of these assays is that many primary hits are frequent hitters regardless of the PPI being investigated.  We have used the AlphaScreen technol. to screen four different robust PPI assays each against 25,000 compds.  These activities led to the identification of 137 compds. that demonstrated repeated activity in all PPI assays.  These compds. were subsequently evaluated in two AlphaScreen counter assays, leading to classification of compds. that either interfered with the AlphaScreen chem. (60 compds.) or prevented the binding of the protein His-tag moiety to nickel chelate (Ni2+-NTA) beads of the AlphaScreen detection system (77 compds.).  To further triage the 137 frequent hitters, we subsequently confirmed by a time-resolved fluorescence resonance energy transfer assay that most of these compds. were only frequent hitters in AlphaScreen assays.  A chemoinformatics anal. of the apparent hits provided details of the compds. that can be flagged as frequent hitters of the AlphaScreen technol., and these data have broad applicability for users of these detection technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQj9hCbRqv7LVg90H21EOLACvtfcHk0lh2Jk-jnp7EIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM&md5=ea65e879f093202deb2266bad94f0208</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1177%2F1087057113516861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516861%26sid%3Dliteratum%253Aachs%26aulast%3DSchorpp%26aufirst%3DK.%26aulast%3DRothenaigner%26aufirst%3DI.%26aulast%3DSalmina%26aufirst%3DE.%26aulast%3DReinshagen%26aufirst%3DJ.%26aulast%3DLow%26aufirst%3DT.%26aulast%3DBrenke%26aufirst%3DJ.%2BK.%26aulast%3DGopalakrishnan%26aufirst%3DJ.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DGul%26aufirst%3DS.%26aulast%3DHadian%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520Small-Molecule%2520Frequent%2520Hitters%2520from%2520Alphascreen%2520High-Throughput%2520Screens%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26issue%3D5%26spage%3D715%26epage%3D726%26doi%3D10.1177%2F1087057113516861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sofia, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D. A.</span></span> <span> </span><span class="NLM_article-title">An Empirical Process for the Design of High-Throughput Screening Deck Filters</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1060</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/ci050504m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050504m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFWju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1060-1068&issue=3&author=B.+C.+Pearceauthor=M.+J.+Sofiaauthor=A.+C.+Goodauthor=D.+M.+Drexlerauthor=D.+A.+Stock&title=An+Empirical+Process+for+the+Design+of+High-Throughput+Screening+Deck+Filters&doi=10.1021%2Fci050504m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An Empirical Process for the Design of High-Throughput Screening Deck Filters</span></div><div class="casAuthors">Pearce, Bradley C.; Sofia, Michael J.; Good, Andrew C.; Drexler, Dieter M.; Stock, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1060-1068</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A process for objective identification and filtering of undesirable compds. that contribute to high-throughput screening (HTS) deck promiscuity is described.  Two methods of mapping hit promiscuity have been developed linking SMARTS-based structural queries with historical primary HTS data.  The first compares an expected assay hit rate to actual hit rates.  The second examines the propensity of an individual compd. to hit multiple assays.  Statistical evaluation of the data indicates a correlation between the resultant functional group filters and compd. promiscuity.  These data corroborate a no. of commonly applied filters as well as producing some unexpected results.  Application of these models to HTS collection triage reduced the no. of inhouse compds. considered for screening by 12%.  The implications of these findings are further discussed in the context of the HTS screening set and combinatorial library design as well as compd. acquisition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_rORr8svyLLVg90H21EOLACvtfcHk0lh2Jk-jnp7EIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFWju7s%253D&md5=8c6260a16996b2cbf5d3caec5a883b0b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci050504m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050504m%26sid%3Dliteratum%253Aachs%26aulast%3DPearce%26aufirst%3DB.%2BC.%26aulast%3DSofia%26aufirst%3DM.%2BJ.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DDrexler%26aufirst%3DD.%2BM.%26aulast%3DStock%26aufirst%3DD.%2BA.%26atitle%3DAn%2520Empirical%2520Process%2520for%2520the%2520Design%2520of%2520High-Throughput%2520Screening%2520Deck%2520Filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D3%26spage%3D1060%26epage%3D1068%26doi%3D10.1021%2Fci050504m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reutlinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Multi-Objective Molecular de Novo Design by Adaptive Fragment Prioritization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4244</span>– <span class="NLM_lpage">4248</span>, <span class="refDoi"> DOI: 10.1002/anie.201310864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fanie.201310864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Sls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=4244-4248&issue=16&author=M.+Reutlingerauthor=T.+Rodriguesauthor=P.+Schneiderauthor=G.+Schneider&title=Multi-Objective+Molecular+de+Novo+Design+by+Adaptive+Fragment+Prioritization&doi=10.1002%2Fanie.201310864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Objective Molecular De Novo Design by Adaptive Fragment Prioritization</span></div><div class="casAuthors">Reutlinger, Michael; Rodrigues, Tiago; Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4244-4248</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We present the development and application of a computational mol. de novo design method for obtaining bioactive compds. with desired on- and off-target binding.  The approach translates the nature-inspired concept of ant colony optimization to combinatorial building block selection.  By relying on publicly available structure-activity data, we developed a predictive quant. polypharmacol. model for 640 human drug targets.  By taking reductive amination as an example of a privileged reaction, we obtained novel subtype-selective and multitarget-modulating dopamine D4 antagonists, as well as ligands selective for the sigma-1 receptor with accurately predicted affinities.  The nanomolar potencies of the hits obtained, their high ligand efficiencies, and an overall success rate of 90 % demonstrate that this ligand-based computer-aided mol. design method may guide target-focused combinatorial chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQcmive3D0LVg90H21EOLACvtfcHk0liB23QV0gzDQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Sls7k%253D&md5=031a8e079e55dd027f90ea89fc42c0e6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fanie.201310864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201310864%26sid%3Dliteratum%253Aachs%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DRodrigues%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DMulti-Objective%2520Molecular%2520de%2520Novo%2520Design%2520by%2520Adaptive%2520Fragment%2520Prioritization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26issue%3D16%26spage%3D4244%26epage%3D4248%26doi%3D10.1002%2Fanie.201310864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewell, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lifely, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsden, N.</span></span> <span> </span><span class="NLM_article-title">Strategic Pooling of Compounds for High-Throughput Screening</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1021/ci990423o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci990423o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK1MXksVWkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=897-902&issue=5&author=M.+Hannauthor=B.+Hudsonauthor=X.+Lewellauthor=R.+Lifelyauthor=L.+Millerauthor=N.+Ramsden&title=Strategic+Pooling+of+Compounds+for+High-Throughput+Screening&doi=10.1021%2Fci990423o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Pooling of Compounds for High-Throughput Screening</span></div><div class="casAuthors">Hann, Mike; Hudson, Brian; Lewell, Xiao; Lifely, Rob; Miller, Luke; Ramsden, Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">897-902</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bringing new medicines to the market depends on the rapid discovery of new and effective drugs, often initiated through the biol. testing of many thousands of compds. in high-throughput screening (HTS).  Mixing compds. together into pools for screening is one way to accelerate this process and reduce costs.  This paper contains both theor. and exptl. data which suggest that careful selection of compds. to be pooled together is necessary to reduce the risk of reactivity between compds. within the pools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTwF-1nNSb07Vg90H21EOLACvtfcHk0liB23QV0gzDQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVWkt78%253D&md5=513eb71f84118c961dbadddc44f7af9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci990423o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci990423o%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%26aulast%3DHudson%26aufirst%3DB.%26aulast%3DLewell%26aufirst%3DX.%26aulast%3DLifely%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DL.%26aulast%3DRamsden%26aufirst%3DN.%26atitle%3DStrategic%2520Pooling%2520of%2520Compounds%2520for%2520High-Throughput%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26issue%3D5%26spage%3D897%26epage%3D902%26doi%3D10.1021%2Fci990423o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blake, J.</span></span> <span> </span><span class="NLM_article-title">Identification and Evaluation of Molecular Properties Related to Preclinical Optimization and Clinical Fate</span>. <i>Med. Chem. (Sharjah, United Arab Emirates)</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.2174/157340605774598081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.2174%2F157340605774598081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=649-655&issue=6&author=J.+Blake&title=Identification+and+Evaluation+of+Molecular+Properties+Related+to+Preclinical+Optimization+and+Clinical+Fate&doi=10.2174%2F157340605774598081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.2174%2F157340605774598081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340605774598081%26sid%3Dliteratum%253Aachs%26aulast%3DBlake%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520Evaluation%2520of%2520Molecular%2520Properties%2520Related%2520to%2520Preclinical%2520Optimization%2520and%2520Clinical%2520Fate%26jtitle%3DMed.%2520Chem.%2520%2528Sharjah%252C%2520United%2520Arab%2520Emirates%2529%26date%3D2005%26volume%3D1%26issue%3D6%26spage%3D649%26epage%3D655%26doi%3D10.2174%2F157340605774598081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span> <span> </span><span class="NLM_article-title">Lessons Learnt from Assembling Screening Libraries for Drug Discovery for Neglected Diseases</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200700139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.200700139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=18064617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Krtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=435-444&issue=3&author=R.+Brenkauthor=A.+Schipaniauthor=D.+Jamesauthor=A.+Krasowskiauthor=I.+H.+Gilbertauthor=J.+Frearsonauthor=P.+G.+Wyatt&title=Lessons+Learnt+from+Assembling+Screening+Libraries+for+Drug+Discovery+for+Neglected+Diseases&doi=10.1002%2Fcmdc.200700139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from assembling screening libraries for drug discovery for neglected diseases</span></div><div class="casAuthors">Brenk, Ruth; Schipani, Alessandro; James, Daniel; Krasowski, Agata; Gilbert, Ian Hugh; Frearson, Julie; Wyatt, Paul Graham</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">To enable the establishment of a drug discovery operation for neglected diseases, out of 2.3 million com. available compds. 222 552 compds. were selected for an in silico library, 57 438 for a diverse general screening library, and 1 697 compds. for a focused kinase set.  Compiling these libraries required a robust strategy for compd. selection.  Rules for unwanted groups were defined and selection criteria to enrich for lead-like compds. which facilitate straightforward structure-activity relationship exploration were established.  Further, a literature and patent review was undertaken to ext. key recognition elements of kinase inhibitors ("core fragments") to assemble a focused library for hit discovery for kinases.  Computational and exptl. characterization of the general screening library revealed that the selected compds. (1) span a broad range of lead-like space, (2) show a high degree of structural integrity and purity, and (3) demonstrate appropriate soly. for the purposes of biochem. screening.  The implications of this study for compd. selection, esp. in an academic environment with limited resources, are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy_Ybsdg8QS7Vg90H21EOLACvtfcHk0lhJjLzFUlS2vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Krtb4%253D&md5=b804d0ac8766e954391798e24aa7fbff</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700139%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DSchipani%26aufirst%3DA.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26atitle%3DLessons%2520Learnt%2520from%2520Assembling%2520Screening%2520Libraries%2520for%2520Drug%2520Discovery%2520for%2520Neglected%2520Diseases%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26issue%3D3%26spage%3D435%26epage%3D444%26doi%3D10.1002%2Fcmdc.200700139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Phantom PAINS: Problems with the Utility of Alerts for P an- A Ssay in Terference Compound S</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFCgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=417-427&issue=3&author=S.+J.+Capuzziauthor=E.+N.+Muratovauthor=A.+Tropsha&title=Phantom+PAINS%3A+Problems+with+the+Utility+of+Alerts+for+P+an-+A+Ssay+in+Terference+Compound+S&doi=10.1021%2Facs.jcim.6b00465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Phantom PAINS: Problems with the Utility of Alerts for Pan-Assay INterference CompoundS</span></div><div class="casAuthors">Capuzzi, Stephen J.; Muratov, Eugene N.; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-427</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of substructural alerts to identify Pan-Assay INterference compds. (PAINS) has become a common component of the triage process in biol. screening campaigns.  These alerts, however, were originally derived from a proprietary library tested in just six assays measuring protein-protein interaction (PPI) inhibition using the AlphaScreen detection technol. only; moreover, 68% (328 out of the 480 alerts) were derived from four or fewer compds.  In an effort to assess the reliability of these alerts as indicators of pan-assay interference, the authors performed a large-scale anal. of the impact of PAINS alerts on compd. promiscuity in bioassays using publicly available data in PubChem.  The authors found that the majority (97%) of all compds. contg. PAINS alerts were actually infrequent hitters in AlphaScreen assays measuring PPI inhibition.  The authors also found that the presence of PAINS alerts, contrary to expectations, did not reflect any heightened assay activity trends across all assays in PubChem including AlphaScreen, luciferase, β-lactamase, or fluorescence-based assays.  In addn., 109 PAINS alerts were present in 3570 extensively assayed, but consistently inactive compds. Dark Chem. Matter.  Finally, the authors obsd. that 87 small mol. FDA-approved drugs contained PAINS alerts and profiled their bioassay activity.  Based on this detailed anal. of PAINS alerts in nonproprietary compd. libraries, the authors caution against the blind use of PAINS filters to detect and triage compds. with possible PAINS liabilities and recommend that such conclusions should be drawn only by conducting orthogonal expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoczoSnfEJzUrVg90H21EOLACvtfcHk0lhJjLzFUlS2vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFCgtbw%253D&md5=206e1d1f24df12644527ddbaa03b24b0</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00465%26sid%3Dliteratum%253Aachs%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DPhantom%2520PAINS%253A%2520Problems%2520with%2520the%2520Utility%2520of%2520Alerts%2520for%2520P%2520an-%2520A%2520Ssay%2520in%2520Terference%2520Compound%2520S%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D3%26spage%3D417%26epage%3D427%26doi%3D10.1021%2Facs.jcim.6b00465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, P.</span></span> Filtering Chemcial Libraries <a href="http://practicalcheminformatics.blogspot.com/2018/08/filtering-chemical-libraries.html" class="extLink">http://practicalcheminformatics.blogspot.com/2018/08/filtering-chemical-libraries.html</a> (accessed Jul 2, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Walters%2C+P.+Filtering+Chemcial+Libraries+http%3A%2F%2Fpracticalcheminformatics.blogspot.com%2F2018%2F08%2Ffiltering-chemical-libraries.html+%28accessed+Jul+2%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DP.%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span>Daylight Theory Manual:
4. SMARTS - A Language for Describing Molecular Patterns. <a href="https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed Jul 1, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight+Theory+Manual%3A%0A4.+SMARTS+-+A+Language+for+Describing+Molecular+Patterns.+https%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+Jul+1%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmki, E. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Comparing Molecular Patterns Using the Example of SMARTS: Applications and Filter Collection Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2572</span>– <span class="NLM_lpage">2586</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00249</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00249" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVais7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2572-2586&issue=6&author=E.+S.+R.+Ehmkiauthor=R.+Schmidtauthor=F.+Ohmauthor=M.+Rarey&title=Comparing+Molecular+Patterns+Using+the+Example+of+SMARTS%3A+Applications+and+Filter+Collection+Analysis&doi=10.1021%2Facs.jcim.9b00249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing Molecular Patterns Using the Example of SMARTS: Applications and Filter Collection Analysis</span></div><div class="casAuthors">Ehmki, Emanuel S. R.; Schmidt, Robert; Ohm, Farina; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2572-2586</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a recent work, an algorithm to compare chem. patterns, written for example in SMARTS, was presented.  This algorithm, called SMARTScompare, is able to assess the identity, subset relation, and similarity of a pair of patterns.  Here we used an implementation of SMARTScompare to analyze SMARTS filter sets that were published in the context of, for example, high-throughput screening.  We found that the difference in intentions with which the filter sets were designed is mirrored in the similarity values we calcd.  The anal. revealed which patterns from one filter set are covered by filters from another set.  In one case it became obvious that a filter set is more or less completely covered by another.  Furthermore, we analyzed pattern hierarchies for consistency, and we propose a method to remove redundant patterns.  SMARTScompare together with SMARTScompareView equips users with powerful methods to visualize, compare, and focus their filter sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGronGm2an_3jrVg90H21EOLACvtfcHk0lhJjLzFUlS2vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVais7%252FN&md5=20522c4eef1dbe10205c90cce245dbc1</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00249%26sid%3Dliteratum%253Aachs%26aulast%3DEhmki%26aufirst%3DE.%2BS.%2BR.%26aulast%3DSchmidt%26aufirst%3DR.%26aulast%3DOhm%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DComparing%2520Molecular%2520Patterns%2520Using%2520the%2520Example%2520of%2520SMARTS%253A%2520Applications%2520and%2520Filter%2520Collection%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D6%26spage%3D2572%26epage%3D2586%26doi%3D10.1021%2Facs.jcim.9b00249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmki, E. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashychev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Comparing Molecular Patterns Using the Example of SMARTS: Theory and Algorithms</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2560</span>– <span class="NLM_lpage">2571</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVWjt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2560-2571&issue=6&author=R.+Schmidtauthor=E.+S.+R.+Ehmkiauthor=F.+Ohmauthor=H.+C.+Ehrlichauthor=A.+Mashychevauthor=M.+Rarey&title=Comparing+Molecular+Patterns+Using+the+Example+of+SMARTS%3A+Theory+and+Algorithms&doi=10.1021%2Facs.jcim.9b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing Molecular Patterns Using the Example of SMARTS: Theory and Algorithms</span></div><div class="casAuthors">Schmidt, Robert; Ehmki, Emanuel S. R.; Ohm, Farina; Ehrlich, Hans-Christian; Mashychev, Andriy; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2560-2571</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. patterns are widely used for compd. filtering in mol. design endeavors.  They describe structural properties that are connected with unwanted phys. or chem. properties like reactivity or toxicity.  With filter sets comprising hundreds of structural filters, an analytic approach to compare those patterns is needed.  Here the authors present a novel approach to solve the generic pattern comparison problem.  The authors introduce chem. inspired fingerprints for pattern nodes and edges to derive an easy-to-compare pattern representation.  On two annotated pattern graphs the authors apply a max. common subgraph algorithm enabling the calcn. of pattern inclusion and similarity.  The resulting algorithm can be used in many different ways.  The authors can automatically derive pattern hierarchies or search in large pattern collections for more general or more specific patterns.  To the best of the knowledge, the presented algorithm is the first of its kind enabling these types of chem. pattern analytics.  The new tool named SMARTScompare is an implementation of the approach for the SMARTS language, which is the quasi-std. for structural filters.  The authors demonstrate the capabilities of SMARTScompare on a large collection of SMARTS patterns from real applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYr0R1czYJz7Vg90H21EOLACvtfcHk0lhAwAcEcAE8OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVWjt7zK&md5=5bf149130e7ac86e093bc96f7d38d963</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00250%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DR.%26aulast%3DEhmki%26aufirst%3DE.%2BS.%2BR.%26aulast%3DOhm%26aufirst%3DF.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DMashychev%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DComparing%2520Molecular%2520Patterns%2520Using%2520the%2520Example%2520of%2520SMARTS%253A%2520Theory%2520and%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D6%26spage%3D2560%26epage%3D2571%26doi%3D10.1021%2Facs.jcim.9b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaslavsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolton, E. E.</span></span> <span> </span><span class="NLM_article-title">PubChem 2019 Update: Improved Access to Chemical Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1102</span>– <span class="NLM_lpage">D1109</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30371825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1102-D1109&issue=D1&author=S.+Kimauthor=J.+Chenauthor=T.+Chengauthor=A.+Gindulyteauthor=J.+Heauthor=S.+Heauthor=Q.+Liauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=B.+Yuauthor=L.+Zaslavskyauthor=J.+Zhangauthor=E.+E.+Bolton&title=PubChem+2019+Update%3A+Improved+Access+to+Chemical+Data&doi=10.1093%2Fnar%2Fgky1033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1033%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZaslavsky%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26atitle%3DPubChem%25202019%2520Update%253A%2520Improved%2520Access%2520to%2520Chemical%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1102%26epage%3DD1109%26doi%3D10.1093%2Fnar%2Fgky1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schomburg, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Interactive Design of Generic Chemical Patterns</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">13–14</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23402846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFyqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=651-658&issue=13%E2%80%9314&author=K.+T.+Schomburgauthor=L.+Wetzerauthor=M.+Rarey&title=Interactive+Design+of+Generic+Chemical+Patterns&doi=10.1016%2Fj.drudis.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive design of generic chemical patterns</span></div><div class="casAuthors">Schomburg, Karen T.; Wetzer, Lars; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">651-658</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Every medicinal chemist has to create chem. patterns occasionally for querying databases, applying filters or describing functional groups.  However, the representations of chem. patterns have been so far limited to languages with highly complex syntax, handicapping the application of patterns.  Graphic pattern editors similar to chem. editors can facilitate the work with patterns.  In this article, we review the interfaces of frequently used web search engines for chem. patterns.  We take a look at pattern editing concepts of standalone chem. editors and finally present a completely new, unpublished graphical approach to pattern design, the SMARTSeditor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgM7nF8XasFrVg90H21EOLACvtfcHk0lgptQmtvPYnfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFyqtrc%253D&md5=6f2b072249484a95d353caac44c050ea</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchomburg%26aufirst%3DK.%2BT.%26aulast%3DWetzer%26aufirst%3DL.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DInteractive%2520Design%2520of%2520Generic%2520Chemical%2520Patterns%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26issue%3D13%25E2%2580%259314%26spage%3D651%26epage%3D658%26doi%3D10.1016%2Fj.drudis.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Félix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magariños, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Marañón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: Towards Direct Deposition of Bioassay Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30398643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&issue=D1&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+F%C3%A9lixauthor=M.+P.+Magari%C3%B1osauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Mara%C3%B1%C3%B3nauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+Towards+Direct+Deposition+of+Bioassay+Data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: towards direct deposition of bioassay data</span></div><div class="casAuthors">Mendez, David; Gaulton, Anna; Bento, A. Patricia; Chambers, Jon; De Veij, Marleen; Felix, Eloy; Magarinos, Maria Paula; Mosquera, Juan F.; Mutowo, Prudence; Nowotka, Michal; Gordillo-Maranon, Maria; Hunter, Fiona; Junco, Laura; Mugumbate, Grace; Rodriguez-Lopez, Milagros; Atkinson, Francis; Bosc, Nicolas; Radoux, Chris J.; Cabrera, Aldo Segura; Hersey, Anne; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D930-D940</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is a large, open-access bioactivity database, previously described in the 2012, 2014 and 2017 Nucleic Acids Research Database Issues.  In the last two years, several important improvements have been made to the database and are described here.  These include more robust capture and representation of assay details; a new data deposition system, allowing updating of data sets and deposition of supplementary data; and a completely redesigned web interface, with enhanced search and filtering capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM7KYSXc4jerVg90H21EOLACvtfcHk0lgptQmtvPYnfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF&md5=d80660ce87d5b3e0773594e4fddec046</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DF%25C3%25A9lix%26aufirst%3DE.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Mara%25C3%25B1%25C3%25B3n%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520Towards%2520Direct%2520Deposition%2520of%2520Bioassay%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cibrian-Uhalte, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarinos, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Database in 2017</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D945</span>– <span class="NLM_lpage">D954</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgkw1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27899562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D945-D954&issue=D1&author=A.+Gaultonauthor=A.+Herseyauthor=M.+L.+Nowotkaauthor=A.+P.+Bentoauthor=J.+Chambersauthor=D.+Mendezauthor=P.+Mutowoauthor=F.+Atkinsonauthor=L.+J.+Bellisauthor=E.+Cibrian-Uhalteauthor=M.+Daviesauthor=N.+Dedmanauthor=A.+Karlssonauthor=M.+P.+Magarinosauthor=J.+P.+Overingtonauthor=G.+Papadatosauthor=I.+Smitauthor=A.+R.+Leach&title=The+ChEMBL+Database+in+2017&doi=10.1093%2Fnar%2Fgkw1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL database in 2017</span></div><div class="casAuthors">Gaulton, Anna; Hersey, Anne; Nowotka, Micha-l; Bento, A. Patricia; Chambers, Jon; Mendez, David; Mutowo, Prudence; Atkinson, Francis; Bellis, Louisa J.; Cibrian-Uhalte, Elena; Davies, Mark; Dedman, Nathan; Karlsson, Anneli; Magarinos, Maria Paula; Overington, John P.; Papadatos, George; Smit, Ines; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D945-D954</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 and 2014 Nucleic Acids Research Database Issues.  Since then, alongside the continued extn. of data from the medicinal chem. literature, new sources of bioactivity data have also been added to the database.  These include: deposited data sets from neglected disease screening; crop protection data; drug metab. and disposition data and bioactivity data from patents.  A no. of improvements and new features have also been incorporated.  These include the annotation of assays and targets using ontologies, the inclusion of targets and indications for clin. candidates, addn. of metabolic pathways for drugs and calcn. of structural alerts.  The ChEMBL data can be accessed via a web-interface, RDF distribution, data downloads and RESTful web-services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB0IuNShSHIrVg90H21EOLACvtfcHk0lj_mzCcnIeVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D&md5=1dcbb641f8c2d11410663f97505a4911</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1074%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DNowotka%26aufirst%3DM.%2BL.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DCibrian-Uhalte%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DMagarinos%26aufirst%3DM.%2BP.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DI.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DThe%2520ChEMBL%2520Database%2520in%25202017%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD945%26epage%3DD954%26doi%3D10.1093%2Fnar%2Fgkw1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span>Clarivate Integrity. <a href="https://integrity.clarivate.com/" class="extLink">https://integrity.clarivate.com/</a> (accessed Jun 9, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Clarivate+Integrity.+https%3A%2F%2Fintegrity.clarivate.com%2F+%28accessed+Jun+9%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span>GOSTAR. <a href="https://www.gostardb.com/gostar/index.jsp" class="extLink">https://www.gostardb.com/gostar/index.jsp</a> (accessed Jun 9, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR.+https%3A%2F%2Fwww.gostardb.com%2Fgostar%2Findex.jsp+%28accessed+Jun+9%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2016.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27427232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=862-874&issue=7&author=Y.+Wangauthor=A.+Cornettauthor=F.+J.+Kingauthor=Y.+Maoauthor=F.+Nigschauthor=C.+G.+Parisauthor=G.+McAllisterauthor=J.+L.+Jenkins&title=Evidence-Based+and+Quantitative+Prioritization+of+Tool+Compounds+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.chembiol.2016.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span></div><div class="casAuthors">Wang, Yuan; Cornett, Allen; King, Fred J.; Mao, Yi; Nigsch, Florian; Paris, C. Gregory; McAllister, Gregory; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">862-874</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The use of potent and selective chem. tools with well-defined targets can help elucidate biol. processes driving phenotypes in phenotypic screens.  However, identification of selective compds. en masse to create targeted screening sets is non-trivial.  A systematic approach is needed to prioritize probes, which prevents the repeated use of published but unselective compds.  Here we performed a meta-anal. of integrated large-scale, heterogeneous bioactivity data to create an evidence-based, quant. metric to systematically rank tool compds. for targets.  Our tool score (TS) was then tested on hundreds of compds. by assessing their activity profiles in a panel of 41 cell-based pathway assays.  We demonstrate that high-TS tools show more reliably selective phenotypic profiles than lower-TS compds.  Addnl. we highlight frequently tested compds. that are non-selective tools and distinguish target family polypharmacol. from cross-family promiscuity.  TS can therefore be used to prioritize compds. from heterogeneous databases for phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquaPu2t_l4ZbVg90H21EOLACvtfcHk0lj_mzCcnIeVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E&md5=b990346556ac032b8cecbfc9455f8458</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DKing%26aufirst%3DF.%2BJ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DEvidence-Based%2520and%2520Quantitative%2520Prioritization%2520of%2520Tool%2520Compounds%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26issue%3D7%26spage%3D862%26epage%3D874%26doi%3D10.1016%2Fj.chembiol.2016.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canham, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baeschlin, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patoor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaanderup, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapa, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspesi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labbé-Giguère, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamber, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushell, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Systematic Chemogenetic Library Assembly</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1124</span>– <span class="NLM_lpage">1129</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.chembiol.2020.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=32707038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A280%3ADC%252BB38jms1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=1124-1129&author=S.+M.+Canhamauthor=Y.+Wangauthor=A.+Cornettauthor=D.+S.+Auldauthor=D.+K.+Baeschlinauthor=M.+Patoorauthor=P.+R.+Skaanderupauthor=A.+Hondaauthor=L.+Llamasauthor=G.+Wendelauthor=F.+A.+Mapaauthor=P.+Aspesiauthor=N.+Labb%C3%A9-Gigu%C3%A8reauthor=G.+G.+Gamberauthor=D.+S.+Palaciosauthor=A.+Schuffenhauerauthor=Z.+Dengauthor=F.+Nigschauthor=M.+Frederiksenauthor=S.+M.+Bushellauthor=D.+Rothmanauthor=R.+K.+Jainauthor=H.+Hemmerleauthor=K.+Brinerauthor=J.+A.+Porterauthor=J.+A.+Tallaricoauthor=J.+L.+Jenkins&title=Systematic+Chemogenetic+Library+Assembly&doi=10.1016%2Fj.chembiol.2020.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Chemogenetic Library Assembly</span></div><div class="casAuthors">Canham Stephen M; Auld Douglas S; Jenkins Jeremy L; Wang Yuan; Cornett Allen; Honda Ayako; Llamas Luis; Wendel Greg; Mapa Felipa A; Aspesi Peter Jr; Labbe-Giguere Nancy; Gamber Gabriel G; Palacios Daniel S; Deng Zhan; Bushell Simon M; Rothman Deborah; Jain Rishi K; Hemmerle Horst; Briner Karin; Porter Jeffery A; Tallarico John A; Baeschlin Daniel K; Patoor Maude; Skaanderup Philip R; Schuffenhauer Ansgar; Nigsch Florian; Frederiksen Mathias</div><div class="citationInfo"><span class="NLM_cas:title">Cell chemical biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1124-1129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemogenetic libraries, collections of well-defined chemical probes, provide tremendous value to biomedical research but require substantial effort to ensure diversity as well as quality of the contents.  We have assembled a chemogenetic library by data mining and crowdsourcing institutional expertise.  We are sharing our approach, lessons learned, and disclosing our current collection of 4,185 compounds with their primary annotated gene targets (https://github.com/Novartis/MoaBox).  This physical collection is regularly updated and used broadly both within Novartis and in collaboration with external partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQemxh8whKe-C86Js-BCLiifW6udTcc2eYJv5IA5yo8fLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38jms1yjsg%253D%253D&md5=e566fa7acce73970a41251a22fc42259</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DCanham%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DBaeschlin%26aufirst%3DD.%2BK.%26aulast%3DPatoor%26aufirst%3DM.%26aulast%3DSkaanderup%26aufirst%3DP.%2BR.%26aulast%3DHonda%26aufirst%3DA.%26aulast%3DLlamas%26aufirst%3DL.%26aulast%3DWendel%26aufirst%3DG.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3DLabb%25C3%25A9-Gigu%25C3%25A8re%26aufirst%3DN.%26aulast%3DGamber%26aufirst%3DG.%2BG.%26aulast%3DPalacios%26aufirst%3DD.%2BS.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DBushell%26aufirst%3DS.%2BM.%26aulast%3DRothman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DR.%2BK.%26aulast%3DHemmerle%26aufirst%3DH.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DSystematic%2520Chemogenetic%2520Library%2520Assembly%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D1124%26epage%3D1129%26doi%3D10.1016%2Fj.chembiol.2020.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&issue=8&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+Molecular+Similarity%3A+Comparing+Compounds+on+the+Basis+of+Biological+Activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0lj8wLNBa6GHpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520Molecular%2520Similarity%253A%2520Comparing%2520Compounds%2520on%2520the%2520Basis%2520of%2520Biological%2520Activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26issue%3D8%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity to Discover Structurally Diverse Bioactive Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">692</span>– <span class="NLM_lpage">703</span>, <span class="refDoi"> DOI: 10.1021/ci300607r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300607r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=692-703&issue=3&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=M.+Glick&title=Bioturbo+Similarity+Searching%3A+Combining+Chemical+and+Biological+Similarity+to+Discover+Structurally+Diverse+Bioactive+Molecules&doi=10.1021%2Fci300607r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity To Discover Structurally Diverse Bioactive Molecules</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">692-703</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening using bioactivity profiles has become an integral part of currently applied hit finding methods in pharmaceutical industry.  However, a significant drawback of this approach is that it is only applicable to compds. that have been biol. tested in the past and have sufficient activity annotations for meaningful profile comparisons.  Although bioactivity data generated in pharmaceutical institutions are growing on an unprecedented scale, the no. of biol. annotated compds. still covers only a minuscule fraction of chem. space.  For a newly synthesized compd. or an isolated natural product to be biol. characterized across multiple assays, it may take a considerable amt. of time.  Consequently, this chem. matter will not be included in virtual screening campaigns based on bioactivity profiles.  To overcome this problem, we herein introduce bioturbo similarity searching that uses chem. similarity to map mols. without biol. annotations into bioactivity space and then searches for biol. similar compds. in this ref. system.  In benchmark calcns. on primary screening data, we demonstrate that our approach generally achieves higher hit rates and identifies structurally more diverse compds. than approaches using chem. information only.  Furthermore, our method is able to discover hits with novel modes of inhibition that traditional 2D and 3D similarity approaches are unlikely to discover.  Test calcns. on a set of natural products reveal the practical utility of the approach for identifying novel and synthetically more accessible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lay0o3odq7Vg90H21EOLACvtfcHk0lj8wLNBa6GHpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D&md5=0df2d6699f97e95b0f04e40ac28c17fe</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci300607r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300607r%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DBioturbo%2520Similarity%2520Searching%253A%2520Combining%2520Chemical%2520and%2520Biological%2520Similarity%2520to%2520Discover%2520Structurally%2520Diverse%2520Bioactive%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D3%26spage%3D692%26epage%3D703%26doi%3D10.1021%2Fci300607r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">Benchmarking of Multivariate Similarity Measures for High-Content Screening Fingerprints in Phenotypic Drug Discovery</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1177/1087057113501390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1177%2F1087057113501390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=24045583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1284-1297&issue=10&author=F.+Reisenauthor=X.+Zhangauthor=D.+Gabrielauthor=P.+Selzer&title=Benchmarking+of+Multivariate+Similarity+Measures+for+High-Content+Screening+Fingerprints+in+Phenotypic+Drug+Discovery&doi=10.1177%2F1087057113501390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking of multivariate similarity measures for high-content screening fingerprints in phenotypic drug discovery</span></div><div class="casAuthors">Reisen, Felix; Zhang, Xian; Gabriel, Daniela; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1284-1297, 14</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-content screening (HCS) is a powerful tool for drug discovery being capable of measuring cellular responses to chem. disturbance in a high-throughput manner.  HCS provides an image-based readout of cellular phenotypes, including features such as shape, intensity, or texture in a highly multiplexed and quant. manner.  The corresponding feature vectors can be used to characterize phenotypes and are thus defined as HCS fingerprints.  Systematic analyses of HCS fingerprints allow for objective computational comparisons of cellular responses.  Such comparisons therefore facilitate the detection of different compds. with different phenotypic outcomes from high-throughput HCS campaigns.  Feature selection methods and similarity measures, as a basis for phenotype identification and clustering, are crit. for the quality of such computational analyses.  We systematically evaluated 16 different similarity measures in combination with linear and nonlinear feature selection methods for their potential to capture biol. relevant image features.  Nonlinear correlation-based similarity measures such as Kendall's τ and Spearman's ρ perform well in most evaluation scenarios, outperforming other frequently used metrics (such as the Euclidian distance).  We also present four novel modifications of the connectivity map similarity that surpass the original version, in our expts.  This study provides a basis for generic phenotypic anal. in future HCS campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqccj8_bUpPcbVg90H21EOLACvtfcHk0lgq_TSn4gsUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitrvF&md5=ecbc36a09e21f2aa317170f1ea7d8045</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1177%2F1087057113501390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113501390%26sid%3Dliteratum%253Aachs%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGabriel%26aufirst%3DD.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DBenchmarking%2520of%2520Multivariate%2520Similarity%2520Measures%2520for%2520High-Content%2520Screening%2520Fingerprints%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26issue%3D10%26spage%3D1284%26epage%3D1297%26doi%3D10.1177%2F1087057113501390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, R. C.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2077-2088&issue=8&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=L.+Tianauthor=R.+C.+Perez&title=Profile-QSAR+2.0%3A+Kinase+Virtual+Screening+Accuracy+Comparable+to+Four-Concentration+IC50s+for+Realistically+Novel+Compounds&doi=10.1021%2Facs.jcim.7b00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Tian, Li; Perez, Rolando C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2077-2088</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While conventional random forest regression (RFR) virtual screening models appear to have excellent accuracy on random held-out test sets, they prove lacking in actual practice.  Anal. of 18 historical virtual screens showed that random test sets are far more similar to their training sets than are the compds. project teams actually order.  A new, cluster-based "realistic" training/test set split, which mirrors the chem. novelty of real-life virtual screens, recapitulates the poor predictive power of RFR models in real projects.  The original Profile-QSAR (pQSAR) method greatly broadened the domain of applicability over conventional models by using as independent variables a profile of activity predictions from all historical assays in a large protein family.  However, the accuracy still fell short of expt. on the realistic test sets.  The improved "pQSAR 2.0" method replaces probabilities of activity from naive Bayes categorical models at several thresholds with predicted IC50s from RFR models.  Unexpectedly, the high accuracy also requires removing the RFR model for the actual assay of interest from the independent variable profile.  With these improvements, pQSAR 2.0 activity predictions are now statistically comparable to medium-throughput 4-concn. IC50 measurements even on the realistic test set.  Beyond the yes/no activity predictions from a typical high-throughput screen (HTS) or conventional virtual screen, these semi-quant. IC50 predictions allow for predicted potency, ligand efficiency, lipophilic efficiency and selectivity against antitargets, greatly facilitating hit list triaging and enabling virtual screening panels such as toxicity panels and overall promiscuity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURHgPoAj7n7Vg90H21EOLACvtfcHk0lgq_TSn4gsUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF&md5=3e215174b6d4b89f490f79e61c893ee1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DR.%2BC.%26atitle%3DProfile-QSAR%25202.0%253A%2520Kinase%2520Virtual%2520Screening%2520Accuracy%2520Comparable%2520to%2520Four-Concentration%2520IC50s%2520for%2520Realistically%2520Novel%2520Compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D8%26spage%3D2077%26epage%3D2088%26doi%3D10.1021%2Facs.jcim.7b00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR: A Novel Meta-QSAR Method That Combines Activities across the Kinase Family to Accurately Predict Affinity, Selectivity, and Cellular Activity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1942</span>– <span class="NLM_lpage">1956</span>, <span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+A+Novel+Meta-QSAR+Method+That+Combines+Activities+across+the+Kinase+Family+to+Accurately+Predict+Affinity%2C+Selectivity%2C+and+Cellular+Activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0lgq_TSn4gsUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520A%2520Novel%2520Meta-QSAR%2520Method%2520That%2520Combines%2520Activities%2520across%2520the%2520Kinase%2520Family%2520to%2520Accurately%2520Predict%2520Affinity%252C%2520Selectivity%252C%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">All-Assay-Max2 PQSAR: Activity Predictions as Accurate as Four-Concentration IC 50 s for 8558 Novartis Assays</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4450</span>– <span class="NLM_lpage">4459</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOqurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=4450-4459&issue=10&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=X.-W.+Zhuauthor=L.+Tianauthor=P.+Mukherjeeauthor=X.+Liu&title=All-Assay-Max2+PQSAR%3A+Activity+Predictions+as+Accurate+as+Four-Concentration+IC+50+s+for+8558+Novartis+Assays&doi=10.1021%2Facs.jcim.9b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">All-Assay-Max2 pQSAR: Activity Predictions as Accurate as Four-Concentration IC50s for 8558 Novartis Assays</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Zhu, Xiang-Wei; Tian, Li; Mukherjee, Prasenjit; Liu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4450-4459</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-quant. structure-activity relationship (pQSAR) is a massively multitask, two-step machine learning method with unprecedented scope, accuracy, and applicability domain.  In step one, a "profile" of conventional single-assay random forest regression models are trained on a very large no. of biochem. and cellular pIC50 assays using Morgan 2 substructural fingerprints as compd. descriptors.  In step two, a panel of partial least squares (PLS) models are built using the profile of pIC50 predictions from those random forest regression models as compd. descriptors (hence the name).  Previously described for a panel of 728 biochem. and cellular kinase assays, we have now built an enormous pQSAR from 11 805 diverse Novartis (NVS) IC50 and EC50 assays.  This large no. of assays, and hence of compd. descriptors for PLS, dictated reducing the profile by only including random forest regression models whose predictions correlate with the assay being modeled.  The random forest regression and pQSAR models were evaluated with our "realistically novel" held-out test set, whose median av. similarity to the nearest training set member across the 11 805 assays was only 0.34, comparable to the novelty of compds. actually selected from virtual screens.  For the 11 805 single-assay random forest regression models, the median correlation of prediction with the expt. was only rext2 = 0.05, virtually random, and only 8% of the models achieved our std. success threshold of rext2 = 0.30.  For pQSAR, the median correlation was rext2 = 0.53, comparable to four-concn. exptl. IC50s, and 72% of the models met our rext2 > 0.30 std., totaling 8558 successful models.  The successful models included assays from all of the 51 annotated target subclasses, as well as 4196 phenotypic assays, indicating that pQSAR can be applied to virtually any disease area.  Every month, all models are updated to include new measurements, and predictions are made for 5.5 million NVS compds., totaling 50 billion predictions.  Common uses have included virtual screening, selectivity design, toxicity and promiscuity prediction, mechanism-of-action prediction, and others.  Several such actual applications are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVUqZKKKZCCbVg90H21EOLACvtfcHk0lieLNWKo6gdnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOqurbK&md5=00bbbad59334754948074d6e1e5edccb</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00375%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DZhu%26aufirst%3DX.-W.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DAll-Assay-Max2%2520PQSAR%253A%2520Activity%2520Predictions%2520as%2520Accurate%2520as%2520Four-Concentration%2520IC%252050%2520s%2520for%25208558%2520Novartis%2520Assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D10%26spage%3D4450%26epage%3D4459%26doi%3D10.1021%2Facs.jcim.9b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettles, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span> <span> </span><span class="NLM_article-title">“Bayes Affinity Fingerprints” Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2445</span>– <span class="NLM_lpage">2456</span>, <span class="refDoi"> DOI: 10.1021/ci600197y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600197y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=2445-2456&issue=6&author=A.+Benderauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=Z.+Dengauthor=J.+H.+Nettlesauthor=J.+W.+Davies&title=%E2%80%9CBayes+Affinity+Fingerprints%E2%80%9D+Improve+Retrieval+Rates+in+Virtual+Screening+and+Define+Orthogonal+Bioactivity+Space%3A+When+Are+Multitarget+Drugs+a+Feasible+Concept%3F&doi=10.1021%2Fci600197y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">"Bayes Affinity Fingerprints" Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Glick, Meir; Deng, Zhan; Nettles, James H.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2445-2456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conventional similarity searching of mols. compares single (or multiple) active query structures to each other in a relative framework, by means of a structural descriptor and a similarity measure.  While this often works well, depending on the target, the authors show here that retrieval rates can be improved considerably by incorporating an external framework describing ligand bioactivity space for comparisons ("Bayes affinity fingerprints").  Structures are described by Bayes scores for a ligand panel comprising about 1000 activity classes extd. from the WOMBAT database.  The comparison of structures is performed via the Pearson correlation coeff. of activity classes, i.e., the order in which two structures are similar to the panel activity classes.  Compd. retrieval on a recently published data set could be improved by as much as 24% relative (9% abs.).  Knowledge about the shape of the "bioactive chem. universe" is thus beneficial to identifying similar bioactivities.  Principal component anal. was employed to further analyze activity space with the objective to define orthogonal ligand bioactive chem. space, leading to nine major (roughly orthogonal) activity axes.  Employing only those nine activity classes, retrieval rates are still comparable to original Bayes affinity fingerprints; thus, the concept of orthogonal bioactive ligand chem. space was validated as being an information-rich but low-dimensional representation of bioactivity space.  Correlations between activity classes are a major determinant to gauge whether the desired multitarget activity of drugs is (on the basis of current knowledge) a feasible concept because it measures the extent to which activities can be optimized independently, or only by strongly influencing one another.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6IjlYwYDTwLVg90H21EOLACvtfcHk0lieLNWKo6gdnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D&md5=47adb720154f45bda1738963d54c70a5</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci600197y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600197y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3D%25E2%2580%259CBayes%2520Affinity%2520Fingerprints%25E2%2580%259D%2520Improve%2520Retrieval%2520Rates%2520in%2520Virtual%2520Screening%2520and%2520Define%2520Orthogonal%2520Bioactivity%2520Space%253A%2520When%2520Are%2520Multitarget%2520Drugs%2520a%2520Feasible%2520Concept%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D6%26spage%3D2445%26epage%3D2456%26doi%3D10.1021%2Fci600197y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span> <span> </span><span class="NLM_article-title">Pfam: A Comprehensive Database of Protein Domain Families Based on Seed Alignments</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199707)28:3<405::AID-PROT10>3.0.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=9223186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1997&pages=405-420&issue=3&author=E.+L.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=R.+Durbin&title=Pfam%3A+A+Comprehensive+Database+of+Protein+Domain+Families+Based+on+Seed+Alignments&doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Pfam: a comprehensive database of protein domain families based on seed alignments</span></div><div class="casAuthors">Sonnhammer, Erik L. L.; Eddy, Sean R.; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-420</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Databases of multiple sequence alignments are a valuable aid to protein sequence classification and anal.  One of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other.  The latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods.  Here, the authors present a database based on hidden Markov model profiles (HMMs), which combines high quality and completeness.  This database, Pfam, consists of parts A and B.  Pfam-A is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and HMMs to find and align all members.  Pfam-B contains sequence families that were generated automatically by applying the Domainer algorithm to cluster and align the remaining protein sequences after removal of Pfam-A domains.  By using Pfam, a large no. of previously unannotated proteins from the Caenorhabditis elegans genome project were classified.  The authors also identified many novel family memberships in known proteins, including new Kazal, fibronectin type III, and response regulator receiver domains.  Pfam-A families have permanent accession nos. and form a library of HMMs available for searching and automatic annotation of new protein sequences.  There are World Wide Web servers at http://www.sanger.ac.uk/Pfam and http://genome.wustl.edu/Pfam, which allow browsing and HMM searching against Pfam-A with a query sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre6VakACw-_rVg90H21EOLACvtfcHk0lieLNWKo6gdnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D&md5=8fe760b424180de92f0c3816443d2d8d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199707%252928%253A3%253C405%253A%253AAID-PROT10%253E3.0.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DPfam%253A%2520A%2520Comprehensive%2520Database%2520of%2520Protein%2520Domain%2520Families%2520Based%2520on%2520Seed%2520Alignments%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D28%26issue%3D3%26spage%3D405%26epage%3D420%26doi%3D10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Gebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piovesan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tosatto, S. C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. D.</span></span> <span> </span><span class="NLM_article-title">The Pfam Protein Families Database in 2019</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D427</span>– <span class="NLM_lpage">D432</span>, <span class="refDoi"> DOI: 10.1093/nar/gky995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgky995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=30357350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D427-D432&issue=D1&author=S.+El-Gebaliauthor=J.+Mistryauthor=A.+Batemanauthor=S.+R.+Eddyauthor=A.+Lucianiauthor=S.+C.+Potterauthor=M.+Qureshiauthor=L.+J.+Richardsonauthor=G.+A.+Salazarauthor=A.+Smartauthor=E.+L.+L.+Sonnhammerauthor=L.+Hirshauthor=L.+Paladinauthor=D.+Piovesanauthor=S.+C.+E.+Tosattoauthor=R.+D.+Finn&title=The+Pfam+Protein+Families+Database+in+2019&doi=10.1093%2Fnar%2Fgky995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database in 2019</span></div><div class="casAuthors">El-Gebali, Sara; Mistry, Jaina; Bateman, Alex; Eddy, Sean R.; Luciani, Aurelien; Potter, Simon C.; Qureshi, Matloob; Richardson, Lorna J.; Salazar, Gustavo A.; Smart, Alfredo; Sonnhammer, Erik L. L.; Hirsh, Layla; Paladin, Lisanna; Piovesan, Damiano; Tosatto, Silvio C. E.; Finn, Robert D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D427-D432</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The last few years have witnessed significant changes in Pfam (https://pfam.xfam.org).  The no. of families has grown substantially to a total of 17,929 in release 32.0.  New addns. have been coupled with efforts to improve existing families, including refinement of domain boundaries, their classification into Pfam clans, as well as their functional annotation.  We recently began to collaborate with the RepeatsDB resource to improve the definition of tandem repeat families within Pfam.  We carried out a significant comparison to the structural classification database, namely the Evolutionary Classification of Protein Domains (ECOD) that led to the creation of 825 new families based on their set of uncharacterized families (EUFs).  Furthermore, we also connected Pfam entries to the Sequence Ontol. (SO) through mapping of the Pfam type definitions to SO terms.  Since Pfam has many community contributors, we recently enabled the linking between authorship of all Pfam entries with the corresponding authors' ORCID identifiers.  This effectively permits authors to claim credit for their Pfam curation and link them to their ORCID record.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqONw_dWfZb87Vg90H21EOLACvtfcHk0lhAm6rE_55P7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7rE&md5=18f34c80b41ffa575898a4a2aff0ec11</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky995%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Gebali%26aufirst%3DS.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DLuciani%26aufirst%3DA.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DQureshi%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DL.%2BJ.%26aulast%3DSalazar%26aufirst%3DG.%2BA.%26aulast%3DSmart%26aufirst%3DA.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%2BL.%26aulast%3DHirsh%26aufirst%3DL.%26aulast%3DPaladin%26aufirst%3DL.%26aulast%3DPiovesan%26aufirst%3DD.%26aulast%3DTosatto%26aufirst%3DS.%2BC.%2BE.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Pfam%2520Protein%2520Families%2520Database%2520in%25202019%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD427%26epage%3DD432%26doi%3D10.1093%2Fnar%2Fgky995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilatovskiy, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Pocketome: An Encyclopedia of Small-Molecule Binding Sites in 4D</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2Fgkr825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=535-540&issue=D1&author=I.+Kufarevaauthor=A.+V.+Ilatovskiyauthor=R.+Abagyan&title=Pocketome%3A+An+Encyclopedia+of+Small-Molecule+Binding+Sites+in+4D&doi=10.1093%2Fnar%2Fgkr825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Pocketome: an encyclopedia of small-molecule binding sites in 4D</span></div><div class="casAuthors">Kufareva, Irina; Ilatovskiy, Andrey V.; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D535-D540</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The importance of binding site plasticity in protein-ligand interactions is well-recognized, and so are the difficulties in predicting the nature and the degree of this plasticity by computational means.  To assist in understanding the flexible protein-ligand interactions, we constructed the Pocketome, an encyclopedia of about one thousand exptl. solved conformational ensembles of druggable binding sites in proteins, grouped by location and consistent chain/cofactor compn.  The multiplicity of pockets within the ensembles adds an extra, fourth dimension to the Pocketome entry data.  Within each ensemble, the pockets were carefully classified by the degree of their pairwise similarity and compatibility with different ligands.  The core of the Pocketome is derived regularly and automatically from the current releases of the Protein Data Bank and the Uniprot Knowledgebase; this core is complemented by entries built from manually provided seed ligand locations.  The Pocketome website (www.pocketome.org) allows searching for the sites of interest, anal. of conformational clusters, important residues, binding compatibility matrixes and interactive visualization of the ensembles using the ActiveICM web browser plugin.  The Pocketome collection can be used to build multi-conformational docking and 3D activity models as well as to design cross-docking and virtual ligand screening benchmarks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDsTtd27fxHrVg90H21EOLACvtfcHk0lhAm6rE_55P7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbrN&md5=921806bf8bf7dde1e1c89a5ca082142d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr825%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DIlatovskiy%26aufirst%3DA.%2BV.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DPocketome%253A%2520An%2520Encyclopedia%2520of%2520Small-Molecule%2520Binding%2520Sites%2520in%25204D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3D535%26epage%3D540%26doi%3D10.1093%2Fnar%2Fgkr825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Dongen, S.</span></span> <span> </span><span class="NLM_article-title">Graph Clustering via a Discrete Uncoupling Process</span>. <i>SIAM J. Matrix Anal. Appl.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1137/040608635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1137%2F040608635" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=121-141&issue=1&author=S.+Van+Dongen&title=Graph+Clustering+via+a+Discrete+Uncoupling+Process&doi=10.1137%2F040608635"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1137%2F040608635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1137%252F040608635%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDongen%26aufirst%3DS.%26atitle%3DGraph%2520Clustering%2520via%2520a%2520Discrete%2520Uncoupling%2520Process%26jtitle%3DSIAM%2520J.%2520Matrix%2520Anal.%2520Appl.%26date%3D2008%26volume%3D30%26issue%3D1%26spage%3D121%26epage%3D141%26doi%3D10.1137%2F040608635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enright, A. J.</span></span> <span> </span><span class="NLM_article-title">An Efficient Algorithm for Large-Scale Detection of Protein Families</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1575</span>– <span class="NLM_lpage">1584</span>, <span class="refDoi"> DOI: 10.1093/nar/30.7.1575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1093%2Fnar%2F30.7.1575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=11917018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD38Xltlajsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=1575-1584&issue=7&author=A.+J.+Enright&title=An+Efficient+Algorithm+for+Large-Scale+Detection+of+Protein+Families&doi=10.1093%2Fnar%2F30.7.1575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient algorithm for large-scale detection of protein families</span></div><div class="casAuthors">Enright, A. J.; Van Dongen, S.; Ouzounis, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1575-1584</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Detection of protein families in large databases is one of the principal research objectives in structural and functional genomics.  Protein family classification can significantly contribute to the delineation of functional diversity of homologous proteins, the prediction of function based on domain architecture or the presence of sequence motifs as well as comparative genomics, providing valuable evolutionary insights.  We present a novel approach called TRIBE-MCL for rapid and accurate clustering of protein sequences into families.  The method relies on the Markov cluster (MCL) algorithm for the assignment of proteins into families based on precomputed sequence similarity information.  This novel approach does not suffer from the problems that normally hinder other protein sequence clustering algorithms, such as the presence of multi-domain proteins, promiscuous domains and fragmented proteins.  The method has been rigorously tested and validated on a no. of very large databases, including SwissProt, InterPro, SCOP and the draft human genome.  Our results indicate that the method is ideally suited to the rapid and accurate detection of protein families on a large scale.  The method has been used to detect and categorize protein families within the draft human genome and the resulting families have been used to annotate a large proportion of human proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd4C24eKihR7Vg90H21EOLACvtfcHk0lhAm6rE_55P7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xltlajsrk%253D&md5=54b188a2c209dfd6b943c6a6492f164c</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F30.7.1575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F30.7.1575%26sid%3Dliteratum%253Aachs%26aulast%3DEnright%26aufirst%3DA.%2BJ.%26atitle%3DAn%2520Efficient%2520Algorithm%2520for%2520Large-Scale%2520Detection%2520of%2520Protein%2520Families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26issue%3D7%26spage%3D1575%26epage%3D1584%26doi%3D10.1093%2Fnar%2F30.7.1575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roggo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">The Scaffold Tree – Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&issue=1&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+Scaffold+Tree+%E2%80%93+Visualization+of+the+Scaffold+Universe+by+Hierarchical+Scaffold+Classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0ljtjk87ui5c4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520Scaffold%2520Tree%2520%25E2%2580%2593%2520Visualization%2520of%2520the%2520Scaffold%2520Universe%2520by%2520Hierarchical%2520Scaffold%2520Classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span> <span> </span><span class="NLM_article-title">Mining for Bioactive Scaffolds with Scaffold Networks: Improved Compound Set Enrichment from Primary Screening Data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1021/ci2000924</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2000924" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFSqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1528-1538&issue=7&author=T.+Varinauthor=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Renner&title=Mining+for+Bioactive+Scaffolds+with+Scaffold+Networks%3A+Improved+Compound+Set+Enrichment+from+Primary+Screening+Data&doi=10.1021%2Fci2000924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Mining for Bioactive Scaffolds with Scaffold Networks: Improved Compound Set Enrichment from Primary Screening Data</span></div><div class="casAuthors">Varin, Thibault; Schuffenhauer, Ansgar; Ertl, Peter; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1538</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of meaningful chem. patterns in the increasing amts. of high-throughput-generated bioactivity data available today is an increasingly important challenge for successful drug discovery.  Herein, we present the scaffold network as a novel approach for mapping and navigation of chem. and biol. space.  A scaffold network represents the chem. space of a library of mols. consisting of all mol. scaffolds and smaller "parent" scaffolds generated therefrom by the pruning of rings, effectively leading to a network of common scaffold substructure relationships.  This algorithm provides an extension of the scaffold tree algorithm that, instead of a network, generates a tree relationship between a heuristically rule-based selected subset of parent scaffolds.  The approach was evaluated for the identification of statistically significantly active scaffolds from primary screening data for which the scaffold tree approach has already been shown to be successful.  Because of the exhaustive enumeration of smaller scaffolds and the full enumeration of relationships between them, about twice as many statistically significantly active scaffolds were identified compared to the scaffold-tree-based approach.  We suggest visualizing scaffold networks as islands of active scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNnYBgKCNoh7Vg90H21EOLACvtfcHk0ljtjk87ui5c4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFSqsrY%253D&md5=14a0cd277e5f11d002daf6296c29404b</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fci2000924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2000924%26sid%3Dliteratum%253Aachs%26aulast%3DVarin%26aufirst%3DT.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DMining%2520for%2520Bioactive%2520Scaffolds%2520with%2520Scaffold%2520Networks%253A%2520Improved%2520Compound%2520Set%2520Enrichment%2520from%2520Primary%2520Screening%2520Data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26issue%3D7%26spage%3D1528%26epage%3D1538%26doi%3D10.1021%2Fci2000924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&issue=5&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljtjk87ui5c4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D5%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fechner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiefl, N.</span></span> <span> </span><span class="NLM_article-title">Chemical Topic Modeling: Exploring Molecular Data Sets Using a Common Text-Mining Approach</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1816</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00249</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00249" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOkt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1816-1831&issue=8&author=N.+Schneiderauthor=N.+Fechnerauthor=G.+A.+Landrumauthor=N.+Stiefl&title=Chemical+Topic+Modeling%3A+Exploring+Molecular+Data+Sets+Using+a+Common+Text-Mining+Approach&doi=10.1021%2Facs.jcim.7b00249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Topic Modeling: Exploring Molecular Data Sets Using a Common Text-Mining Approach</span></div><div class="casAuthors">Schneider, Nadine; Fechner, Nikolas; Landrum, Gregory A.; Stiefl, Nikolaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1816-1831</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Big data is one of the key transformative factors which are increasingly influencing all aspects of modern life.  Although this transformation brings vast opportunities it also generates novel challenges, not the least of which is organizing and searching this data deluge.  The field of medicinal chem. is not different: more and more data are being generated, for instance by technologies such as DNA encoded libraries, peptide libraries, text mining of large literature corpora, and new in silico enumeration methods.  Handling those huge sets of mols. effectively is quite challenging and requires compromises that often come at the expense of the interpretability of the results.  In order to find an intuitive and meaningful approach to organizing large mol. datasets, we adopted a probabilistic framework called "topic modeling" from the text-mining field.  Here, we present the first chem.-related implementation of this method, which allows large mol. sets to be assigned to "chem. topics" and investigating the relationships between those.  In this first study, we thoroughly evaluate this novel method in different expts. and discuss both its disadvantages and advantages.  We show very promising results in reproducing human-assigned concepts by using the approach to identify and retrieve chem. series from sets of mols.  We have also created an intuitive visualization of the chem. topics output by the algorithm.  This is a huge benefit compared to other unsupervised machine-learning methods, like clustering, which are commonly used to group sets of mols.  Finally, we applied the new method to the 1.6 million mols. of the ChEMBL22 dataset to test its robustness and efficiency.  In about 1h we built a 100-topic model of this large dataset in which we could identify interesting topics like "proteins", "DNA" or "steroids".  Along with this publication, we provide our datasets and an open-source implementation of the new method (CheTo) which will be part of an upcoming version of the open-source cheminformatics toolkit RDKit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEAbB6V9wBNrVg90H21EOLACvtfcHk0lgyZPwQW1Tp-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOkt7bK&md5=a737f0d5dbb81f02cb2b58269322b2cc</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00249%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DFechner%26aufirst%3DN.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26aulast%3DStiefl%26aufirst%3DN.%26atitle%3DChemical%2520Topic%2520Modeling%253A%2520Exploring%2520Molecular%2520Data%2520Sets%2520Using%2520a%2520Common%2520Text-Mining%2520Approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D8%26spage%3D1816%26epage%3D1831%26doi%3D10.1021%2Facs.jcim.7b00249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, W. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Shape Diversity of Combinatorial Libraries: A Prerequisite for Broad Bioactivity</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">1003</span>, <span class="refDoi"> DOI: 10.1021/ci025599w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025599w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvF2muro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=987-1003&issue=3&author=W.+H.+B.+Sauerauthor=M.+K.+Schwarz&title=Molecular+Shape+Diversity+of+Combinatorial+Libraries%3A+A+Prerequisite+for+Broad+Bioactivity&doi=10.1021%2Fci025599w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape Diversity of Combinatorial Libraries: A Prerequisite for Broad Bioactivity</span></div><div class="casAuthors">Sauer, Wolfgang H. B.; Schwarz, Matthias K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">987-1003</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A computational method to rapidly assess and visualize the diversity in mol. shape assocd. with a given compd. set has been developed.  Normalized ratios of principal moments of inertia are plotted into two-dimensional triangular graphs and then used to compare the shape space covered by different compd. sets, such as combinatorial libraries of varying size and compn.  We have further developed a computational method to analyze interset similarity in terms of shape space coverage, which allows the shape redundancy between the different subsets of a given compd. collection to be analyzed in a quant. way.  The shape space coverage has been found to originate mainly from the nature and the 3D-geometry (but not the size) of the central scaffold, while the no. and nature of the peripheral substituents and conformational aspects were shown to be of minor importance.  Substantial shape space coverage has been correlated with broad biol. activity by applying the same shape anal. to collections of known bioactive compds., such as MDDR and the GOLD-set.  The aggregate of our results corroborates the intuitive notion that mol. shape is intimately linked to biol. activity and that a high degree of shape (hence scaffold) diversity in screening collections will increase the odds of addressing a broad range of biol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKIZLeZoCDh7Vg90H21EOLACvtfcHk0lgyZPwQW1Tp-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvF2muro%253D&md5=c07401742457e76b066bed4b43feca1d</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fci025599w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025599w%26sid%3Dliteratum%253Aachs%26aulast%3DSauer%26aufirst%3DW.%2BH.%2BB.%26aulast%3DSchwarz%26aufirst%3DM.%2BK.%26atitle%3DMolecular%2520Shape%2520Diversity%2520of%2520Combinatorial%2520Libraries%253A%2520A%2520Prerequisite%2520for%2520Broad%2520Bioactivity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26issue%3D3%26spage%3D987%26epage%3D1003%26doi%3D10.1021%2Fci025599w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span>FastROCS. <a href="http://www.eyesopen.com/fastrocs" class="extLink">http://www.eyesopen.com/fastrocs</a> (accessed Jun 21, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FastROCS.+http%3A%2F%2Fwww.eyesopen.com%2Ffastrocs+%28accessed+Jun+21%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rush, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosyak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1495</span>, <span class="refDoi"> DOI: 10.1021/jm040163o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040163o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1489-1495&issue=5&author=T.+S.+Rushauthor=J.+A.+Grantauthor=L.+Mosyakauthor=A.+Nicholls&title=A+Shape-Based+3-D+Scaffold+Hopping+Method+and+Its+Application+to+a+Bacterial+Protein-Protein+Interaction&doi=10.1021%2Fjm040163o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span></div><div class="casAuthors">Rush, Thomas S., III; Grant, J. Andrew; Mosyak, Lidia; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1489-1495</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors describe the first prospective application of the shape-comparison program ROCS (Rapid Overlay of Chem. Structures) to find new scaffolds for small mol. inhibitors of the ZipA-FtsZ protein-protein interaction, a proposed antibacterial target.  The shape comparisons are made relative to the crystallog. detd., bioactive conformation of a high-throughput screening (HTS) hit.  The use of ROCS led to the identification of a set of novel, weakly binding inhibitors with scaffolds presenting synthetic opportunities to further optimize biol. affinity and lacking development issues assocd. with the HTS lead.  These ROCS-identified scaffolds would have been missed using other structural similarity approaches such as ISIS 2D fingerprints.  X-ray crystallog. anal. of one of the new inhibitors bound to ZipA reveals that the shape comparison approach very accurately predicted the binding mode.  These exptl. results validate this use of ROCS for chemotype switching or "lead hopping" and suggest that it is of general interest for lead identification in drug discovery endeavors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4EX9jFtct37Vg90H21EOLACvtfcHk0lgyZPwQW1Tp-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D&md5=05b6e54a657a3b8c768e63852d871ef6</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm040163o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040163o%26sid%3Dliteratum%253Aachs%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520Shape-Based%25203-D%2520Scaffold%2520Hopping%2520Method%2520and%2520Its%2520Application%2520to%2520a%2520Bacterial%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26issue%3D5%26spage%3D1489%26epage%3D1495%26doi%3D10.1021%2Fjm040163o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&issue=1&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+Shape-Matching+and+Docking+as+Virtual+Screening+Tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lgtY_1jGnPYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Shape-Matching%2520and%2520Docking%2520as%2520Virtual%2520Screening%2520Tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26issue%3D1%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelliccioli, A.</span>; <span class="NLM_string-name">Chatterjee, R.</span>; <span class="NLM_string-name">Koch, G.</span>; <span class="NLM_string-name">Schuffenhauer, A.</span>; <span class="NLM_string-name">Schwarze, M.</span>; <span class="NLM_string-name">Smith, T. J.</span>; <span class="NLM_string-name">Page, M.</span>; <span class="NLM_string-name">Projects, L.</span></span> <span> </span><span class="NLM_article-title">Compound Series and Favorites: A Medicinal Chemistry Story Telling Tool at Novartis What the Data Warehouse Knows</span>. Presented at the Seventh Joint Sheffield Conference on Chemoinformatics, Sheffield, U.K., July 4–6, <span class="NLM_year">1996</span>; Poster 27.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=A.+Pelliccioli&author=R.+Chatterjee&author=G.+Koch&author=A.+Schuffenhauer&author=M.+Schwarze&author=T.+J.+Smith&author=M.+Page&author=L.+Projects&title=Compound+Series+and+Favorites%3A+A+Medicinal+Chemistry+Story+Telling+Tool+at+Novartis+What+the+Data+Warehouse+Knows"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPelliccioli%26aufirst%3DA.%26atitle%3DCompound%2520Series%2520and%2520Favorites%253A%2520A%2520Medicinal%2520Chemistry%2520Story%2520Telling%2520Tool%2520at%2520Novartis%2520What%2520the%2520Data%2520Warehouse%2520Knows%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baier, S.</span></span> <span> </span><span class="NLM_article-title">How Many Molecules Does It Take to Tell a Story? Case Studies, Language, and an Epistemic View of Medicinal Chemistry</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1002%2Fcmdc.201500091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25820774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVKlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=949-956&issue=6&author=M.+Stahlauthor=S.+Baier&title=How+Many+Molecules+Does+It+Take+to+Tell+a+Story%3F+Case+Studies%2C+Language%2C+and+an+Epistemic+View+of+Medicinal+Chemistry&doi=10.1002%2Fcmdc.201500091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">How Many Molecules Does It Take to Tell a Story? Case Studies, Language, and an Epistemic View of Medicinal Chemistry</span></div><div class="casAuthors">Stahl, Martin; Baier, Sabine</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-956</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Medicinal chem. has always been closer to the arts than other disciplines in the natural sciences.  Instead of searching for natural laws, medicinal chem. creates new mol. entities entailing desired pharmaceutical characteristics.  While the productive output of medicinal chem. is comprehensively documented, the epistemic paths of the creative process are less well described.  Here we show how such paths could be visualized and how these visualizations relate to images developed in the history and philosophy of science.  Based on the discussion of these visualizations, we argue that there is a need for a new language of creativity that can be employed during the very course of research, as opposed to its retrospective anal.  This language should be able to reflect both the status and directions in highly complex research processes that may have a clear goal, yet must remain open to unexpected moments of serendipity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLajk6L_irtbVg90H21EOLACvtfcHk0lgtY_1jGnPYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVKlsL4%253D&md5=00e7fcd5e2b7ca7289c565fcc7c5a06b</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500091%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DBaier%26aufirst%3DS.%26atitle%3DHow%2520Many%2520Molecules%2520Does%2520It%2520Take%2520to%2520Tell%2520a%2520Story%253F%2520Case%2520Studies%252C%2520Language%252C%2520and%2520an%2520Epistemic%2520View%2520of%2520Medicinal%2520Chemistry%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26issue%3D6%26spage%3D949%26epage%3D956%26doi%3D10.1002%2Fcmdc.201500091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbach-Weese, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruce, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritsch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aichholz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caravatti, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of NVP-BYL719 a Potent and Selective Phosphatidylinositol-3 Kinase Alpha Inhibitor Selected for Clinical</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3741</span>– <span class="NLM_lpage">3748</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.bmcl.2013.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=23726034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3741-3748&issue=13&author=P.+Furetauthor=V.+Guagnanoauthor=R.+A.+Fairhurstauthor=P.+Imbach-Weeseauthor=I.+Bruceauthor=M.+Knappauthor=C.+Fritschauthor=F.+Blascoauthor=J.+Blanzauthor=R.+Aichholzauthor=J.+Hamonauthor=D.+Fabbroauthor=G.+Caravatti&title=Discovery+of+NVP-BYL719+a+Potent+and+Selective+Phosphatidylinositol-3+Kinase+Alpha+Inhibitor+Selected+for+Clinical&doi=10.1016%2Fj.bmcl.2013.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of NVP-BYL719 a potent and selective phosphatidylinositol-3 kinase alpha inhibitor selected for clinical evaluation</span></div><div class="casAuthors">Furet, Pascal; Guagnano, Vito; Fairhurst, Robin A.; Imbach-Weese, Patricia; Bruce, Ian; Knapp, Mark; Fritsch, Christine; Blasco, Francesca; Blanz, Joachim; Aichholz, Reiner; Hamon, Jacques; Fabbro, Doriano; Caravatti, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3741-3748</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphatidylinositol-3-kinase α (PI3Kα) is a therapeutic target of high interest in anticancer drug research.  On the basis of a binding model rationalizing the high selectivity and potency of a particular series of 2-aminothiazole compds. in inhibiting PI3Kα, a medicinal chem. program has led to the discovery of the clin. candidate NVP-BYL719 I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraN93Vwrfb4bVg90H21EOLACvtfcHk0lgtY_1jGnPYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFaisrY%253D&md5=a0d2d69e01e7a82a5fed04114105925e</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DImbach-Weese%26aufirst%3DP.%26aulast%3DBruce%26aufirst%3DI.%26aulast%3DKnapp%26aufirst%3DM.%26aulast%3DFritsch%26aufirst%3DC.%26aulast%3DBlasco%26aufirst%3DF.%26aulast%3DBlanz%26aufirst%3DJ.%26aulast%3DAichholz%26aufirst%3DR.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DCaravatti%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520NVP-BYL719%2520a%2520Potent%2520and%2520Selective%2520Phosphatidylinositol-3%2520Kinase%2520Alpha%2520Inhibitor%2520Selected%2520for%2520Clinical%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26issue%3D13%26spage%3D3741%26epage%3D3748%26doi%3D10.1016%2Fj.bmcl.2013.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benz, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamidass, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldi, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Power-Laws in Chemical Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi"> DOI: 10.1021/ci700353m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700353m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXms1Gitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1138-1151&issue=6&author=R.+W.+Benzauthor=S.+J.+Swamidassauthor=P.+Baldi&title=Discovery+of+Power-Laws+in+Chemical+Space&doi=10.1021%2Fci700353m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of power-laws in chemical space</span></div><div class="casAuthors">Benz, Ryan W.; Swamidass, S. Joshua; Baldi, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1138-1151</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Power-law distributions have been obsd. in a wide variety of areas.  To our knowledge however, there has been no systematic observation of power-law distributions in chemoinformatics.  Here, we present several examples of power-law distributions arising from the features of small, org. mols.  The distributions of rigid segments and ring systems, the distributions of mol. paths and circular substructures, and the sizes of mol. similarity clusters all show linear trends on log-log rank/frequency plots, suggesting underlying power-law distributions.  The no. of unique features also follow Heaps'-like laws.  The characteristic exponents of the power-laws lie in the 1.5-3 range, consistently with the exponents obsd. in other power-law phenomena.  The power-law nature of these distributions leads to several applications including the prediction of the growth of available data through Heaps' law and the optimal allocation of exptl. or computational resources via the 80/20 rule.  More importantly, we also show how the power-laws can be leveraged to efficiently compress chem. fingerprints in a lossless manner, useful for the improved storage and retrieval of mols. in large chem. databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovyRIpleieC7Vg90H21EOLACvtfcHk0ljafQv0ule7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXms1Gitb4%253D&md5=111cda24a98cae8e6a1dcf672b20c942</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fci700353m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700353m%26sid%3Dliteratum%253Aachs%26aulast%3DBenz%26aufirst%3DR.%2BW.%26aulast%3DSwamidass%26aufirst%3DS.%2BJ.%26aulast%3DBaldi%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Power-Laws%2520in%2520Chemical%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26issue%3D6%26spage%3D1138%26epage%3D1151%26doi%3D10.1021%2Fci700353m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Rethinking Screening</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1038/nchembio.303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1038%2Fnchembio.303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20154660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=162-165&issue=3&author=T.+Kodadek&title=Rethinking+Screening&doi=10.1038%2Fnchembio.303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking screening</span></div><div class="casAuthors">Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">162-165</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bioactive compds. are most frequently identified via high-throughput screening campaigns.  This article discusses the strengths and weaknesses of the most popular screening approaches and the utility of compds. derived from them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTJ8nq3RnwQrVg90H21EOLACvtfcHk0ljafQv0ule7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGmt78%253D&md5=ccd1f4f86d54d44294dc2e3af2c28199</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.303%26sid%3Dliteratum%253Aachs%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DRethinking%2520Screening%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26issue%3D3%26spage%3D162%26epage%3D165%26doi%3D10.1038%2Fnchembio.303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, W. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Which Aspects of HTS Are Empirically Correlated with Downstream Success?</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">337</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=18428086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=327-337&issue=3&author=A.+Benderauthor=D.+Bojanicauthor=J.+W.+Daviesauthor=T.+J.+Crismanauthor=D.+Mikhailovauthor=J.+Scheiberauthor=J.+L.+Jenkinsauthor=Z.+Dengauthor=W.+A.+G.+Hillauthor=M.+Popovauthor=E.+Jacobyauthor=M.+Glick&title=Which+Aspects+of+HTS+Are+Empirically+Correlated+with+Downstream+Success%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Which aspects of HTS are empirically correlated with downstream success?</span></div><div class="casAuthors">Bender, Andreas; Bojanic, Dejan; Davies, John W.; Crisman, Thomas J.; Mikhailov, Dmitri; Scheiber, Josef; Jenkins, Jeremy L.; Deng, Zhan; Hill, W. Adam G.; Popov, Maxim; Jacoby, Edgar; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established hit-finding approach used in the pharmaceutical industry.  In this article, recent experience at Novartis with respect to factors influencing the success of HTS campaigns is discussed.  An inherent measure of HTS quality could be defined by the assay Z and Z' factors, the no. of hits and their biol. potencies; however, such measures of quality do not always correlate with the advancement of hits to the later stages of drug discovery.  Also, for many target classes, such as kinases, it is easy to identify hits, but, as a result of selectivity, intellectual property and other issues, the projects do not result in lead declarations.  In this article, HTS success is defined as the fraction of HTS campaigns that advance into the later stages of drug discovery, and the major influencing factors are examd.  Interestingly, screening compds. in individual wells or in mixts. did not have a major impact on the HTS success and, equally interesting, there was no difference in the progression rates of biochem. and cell-based assays.  Particular target types, assay technologies, structure-activity relationships and powder availability had a much greater impact on success as defined above.  In addn., significant mutual dependencies can be obsd. - while one assay format works well with one target type, this situation might be completely reversed for a combination of the same readout technol. with a different target type.  The results and opinions presented here should be regarded as groundwork, and a plethora of factors that influence the fate of a project, such as biophys. measurements, chem. attractiveness of the hits, strategic reasons and safety pharmacol., are not covered here.  Nonetheless, it is hoped that this information will be used industry-wide to improve success rates in terms of hits progressing into exploratory chem. and beyond.  The support that can be obtained from new in silico approaches to phase transitions are also described, along with the gaps they are designed to fill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqddDbeDZXfNbVg90H21EOLACvtfcHk0ljafQv0ule7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D&md5=c65c4142d449fe752b37c397cd2a941e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DHill%26aufirst%3DW.%2BA.%2BG.%26aulast%3DPopov%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DWhich%2520Aspects%2520of%2520HTS%2520Are%2520Empirically%2520Correlated%2520with%2520Downstream%2520Success%253F%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26issue%3D3%26spage%3D327%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span> <span> </span><span class="NLM_article-title">Designing Focused Libraries Using MoSELECT</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(01)00150-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2FS1093-3263%2801%2900150-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=12071283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVSmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2002&pages=491-498&issue=6&author=V.+J.+Gilletauthor=P.+Willettauthor=P.+J.+Flemingauthor=D.+V.+S.+Green&title=Designing+Focused+Libraries+Using+MoSELECT&doi=10.1016%2FS1093-3263%2801%2900150-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Designing focused libraries using MoSELECT</span></div><div class="casAuthors">Gillet, Valerie J.; Willett, Peter; Fleming, Peter J.; Green, Darren V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">When designing a combinatorial library it is usually desirable to optimize multiple properties of the library simultaneously and often the properties are in competition with one another.  For example, a library that is designed to be focused around a given target mol. should ideally have min. cost and also contain mols. that are bioavailable.  In this paper, we describe the program MoSELECT for multiobjective library design that is based on a multiobjective genetic algorithm (MOGA).  MoSELECT searches the product-space of a virtual combinatorial library to generate a family of equiv. solns. where each soln. represents a combinatorial subset of the virtual library optimized over multiple objectives.  The family of solns. allows the relationships between the objectives to be explored and thus enables the library designer to make an informed choice on an appropriate compromise soln.  Expts. are reported where MoSELECT has been applied to the design of various focused libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0uN-S5AYd37Vg90H21EOLACvtfcHk0ljkgDlBd55Pzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVSmtb8%253D&md5=ce3ea314059eec39761b0289c1edd53c</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2801%2900150-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252801%252900150-4%26sid%3Dliteratum%253Aachs%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DFleming%26aufirst%3DP.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26atitle%3DDesigning%2520Focused%2520Libraries%2520Using%2520MoSELECT%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2002%26volume%3D20%26issue%3D6%26spage%3D491%26epage%3D498%26doi%3D10.1016%2FS1093-3263%2801%2900150-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannas, C. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Library Design Using Multi-Objective Optimization Methods</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>685</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1007/978-1-60761-931-4_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1007%2F978-1-60761-931-4_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=20981518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSrs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=685&publication_year=2011&pages=53-69&author=C.+A.+Nicolaouauthor=C.+C.+Kannas&title=Molecular+Library+Design+Using+Multi-Objective+Optimization+Methods&doi=10.1007%2F978-1-60761-931-4_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular library design using multi-objective optimization methods</span></div><div class="casAuthors">Nicolaou, Christos A.; Kannas, Christos C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">685</span>
        (<span class="NLM_cas:issue">Chemical Library Design</span>),
    <span class="NLM_cas:pages">53-69</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Advancements in combinatorial chem. and high-throughput screening technol. have enabled the synthesis and screening of large mol. libraries for the purposes of drug discovery.  Contrary to initial expectations, the increase in screening library size, typically combined with an emphasis on compd. structural diversity, did not result in a comparable increase in the no. of promising hits found.  In an effort to improve the likelihood of discovering hits with greater optimization potential, more recent approaches attempt to incorporate addnl. knowledge to the library design process to effectively guide the search.  Multi-objective optimization methods capable of taking into account several chem. and biol. criteria have been used to design collections of compds. satisfying simultaneously multiple pharmaceutically relevant objectives.  In this chapter, we present our efforts to implement a multi-objective optimization method, MEGALib, custom-designed to the library design problem.  The method exploits existing knowledge, e.g. from previous biol. screening expts., to identify and profile mol. fragments used subsequently to design compds. compromising the various objectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav94mSUm1G7Vg90H21EOLACvtfcHk0ljkgDlBd55Pzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSrs7rF&md5=675d0bf691152566029b6501cca47d19</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-931-4_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-931-4_3%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DC.%2BA.%26aulast%3DKannas%26aufirst%3DC.%2BC.%26atitle%3DMolecular%2520Library%2520Design%2520Using%2520Multi-Objective%2520Optimization%2520Methods%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D685%26spage%3D53%26epage%3D69%26doi%3D10.1007%2F978-1-60761-931-4_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Improving the Plausibility of Success with Inefficient Metrics</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">5</span>, <span class="refDoi"> DOI: 10.1021/ml4004638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4004638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=2-5&issue=1&author=M.+D.+Shultz&title=Improving+the+Plausibility+of+Success+with+Inefficient+Metrics&doi=10.1021%2Fml4004638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Plausibility of Success with Inefficient Metrics</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-5</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  To increase the probability of success in drug discovery, the concept of drug-like properties was introduced.  Efficiency metrics that normalize potency against these properties could help reach drug-like space more efficiently.  Potential reasons for the inefficient use of metrics and suboptimal decision making are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUcqughsg0LVg90H21EOLACvtfcHk0ljkgDlBd55Pzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtLjP&md5=de1493d6fa4e894b078b8b020ad4b3eb</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fml4004638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4004638%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DImproving%2520the%2520Plausibility%2520of%2520Success%2520with%2520Inefficient%2520Metrics%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D1%26spage%3D2%26epage%3D5%26doi%3D10.1021%2Fml4004638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhnke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneckener, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span> <span> </span><span class="NLM_article-title">Bayer’s in Silico ADMET Platform: A Journey of Machine Learning over the Past Two Decades</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.drudis.2020.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=32652309" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+H.+G%C3%B6llerauthor=L.+Kuhnkeauthor=F.+Montanariauthor=A.+Boninauthor=S.+Schneckenerauthor=A.+ter+Laakauthor=J.+Wichardauthor=M.+Lobellauthor=A.+Hillisch&title=Bayer%E2%80%99s+in+Silico+ADMET+Platform%3A+A+Journey+of+Machine+Learning+over+the+Past+Two+Decades&doi=10.1016%2Fj.drudis.2020.07.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DKuhnke%26aufirst%3DL.%26aulast%3DMontanari%26aufirst%3DF.%26aulast%3DBonin%26aufirst%3DA.%26aulast%3DSchneckener%26aufirst%3DS.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DWichard%26aufirst%3DJ.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26atitle%3DBayer%25E2%2580%2599s%2520in%2520Silico%2520ADMET%2520Platform%253A%2520A%2520Journey%2520of%2520Machine%2520Learning%2520over%2520the%2520Past%2520Two%2520Decades%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26doi%3D10.1016%2Fj.drudis.2020.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Pursuing the Leadlikeness Concept in Pharmaceutical Research</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2004.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.cbpa.2004.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=15183323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkslWqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=255-263&issue=3&author=M.+M.+Hannauthor=T.+I.+Oprea&title=Pursuing+the+Leadlikeness+Concept+in+Pharmaceutical+Research&doi=10.1016%2Fj.cbpa.2004.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Pursuing the leadlikeness concept in pharmaceutical research</span></div><div class="casAuthors">Hann, Mike M.; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-263</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Lipinski and others, through concepts such as drug-likeness, re-focussed drug discovery back to the principles of medicinal chem. in the high-throughput era as key to reducing attrition.  More recently, the need to go further in defining what makes a good lead has been recognized with the concept of leadlikeness.  Leadlikeness implies cut-off values in the physico-chem. profile of chem. libraries such that they have reduced complexity (e.g. MW below <400) and other more restricted properties.  We examine these concepts in the context of Virtual (theor. possible), Tangible (chem. feasible) and Real (phys. available) worlds of mols.  In a thought expt., we take the HTS concept to the extreme: screening an estd. 60 million Global Collection' on 5000 targets and realizing that perhaps millions of drug candidates might be found that could not possibly be handled in reality.  Sampling of the Virtual and Tangible worlds is therefore a necessity.  We show that the world of Reals is significantly under-sampled as the MW of compds. increases.  This supports the design and screening of reduced complexity' (leadlike) compd. libraries, preferably with synthetic handles available for rapid chem. iteration and detected as interesting by careful screening or biophys. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd8ecCArzZ2bVg90H21EOLACvtfcHk0lhSizG3KX8FjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkslWqsrk%253D&md5=c78783585a60d3deee3b0513fe7300e9</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DPursuing%2520the%2520Leadlikeness%2520Concept%2520in%2520Pharmaceutical%2520Research%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26issue%3D3%26spage%3D255%26epage%3D263%26doi%3D10.1016%2Fj.cbpa.2004.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of Five in 2015 and beyond: Target and Ligand Structural Limitations, Ligand Chemistry Structure and Drug Discovery Project Decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+Five+in+2015+and+beyond%3A+Target+and+Ligand+Structural+Limitations%2C+Ligand+Chemistry+Structure+and+Drug+Discovery+Project+Decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lhSizG3KX8FjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520Five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520Ligand%2520Structural%2520Limitations%252C%2520Ligand%2520Chemistry%2520Structure%2520and%2520Drug%2520Discovery%2520Project%2520Decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deursen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddigkeit, L.</span></span> <span> </span><span class="NLM_article-title">Chemical Space as a Source for New Drugs</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1039/c0md00020e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1039%2Fc0md00020e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeju7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=30-38&issue=1&author=J.+L.+Reymondauthor=R.+Van+Deursenauthor=L.+C.+Blumauthor=L.+Ruddigkeit&title=Chemical+Space+as+a+Source+for+New+Drugs&doi=10.1039%2Fc0md00020e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space as a source for new drugs</span></div><div class="casAuthors">Reymond, Jean-Louis; van Deursen, Ruud; Blum, Lorenz C.; Ruddigkeit, Lars</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-38</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The chem. space is the ensemble of all possible mols., which is believed to contain at least 1060 org. mols. below 500 Da of possible interest for drug discovery.  This review summarizes the development of the chem. space concept from enumerating acyclic hydrocarbons in the 1800's to the recent assembly of the chem. universe database GDB.  Chem. space travel algorithms can be used to explore defined regions of chem. space by generating focused virtual libraries.  Maps of the chem. space are produced from property spaces visualized by principal component anal. or by self-organizing maps, and from structural analyses such as the scaffold-tree or the MQN-system.  Virtual screening of virtual chem. space followed by synthesis and testing of the best hits leads to the discovery of new drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdkwSU4qu1irVg90H21EOLACvtfcHk0lhSizG3KX8FjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeju7rF&md5=579428edc79f706101ab58c505052c72</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1039%2Fc0md00020e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00020e%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DRuddigkeit%26aufirst%3DL.%26atitle%3DChemical%2520Space%2520as%2520a%2520Source%2520for%2520New%2520Drugs%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26issue%3D1%26spage%3D30%26epage%3D38%26doi%3D10.1039%2Fc0md00020e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">The Chemical Space Project</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1021/ar500432k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500432k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=722-730&issue=3&author=J.-L.+Reymond&title=The+Chemical+Space+Project&doi=10.1021%2Far500432k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemical Space Project</span></div><div class="casAuthors">Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-730</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the simplest questions that can be asked about mol. diversity is how many org. mols. are possible in total.  To answer this question, my research group has computationally enumerated all possible org. mols. up to a certain size to gain an unbiased insight into the entire chem. space.  Our latest database, GDB-17, contains 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens, by far the largest small mol. database reported to date.  Mols. allowed by valency rules but unstable or nonsynthesizable due to strained topologies or reactive functional groups were not considered, which reduced the enumeration by at least 10 orders of magnitude and was essential to arrive at a manageable database size.  Despite these restrictions, GDB-17 is highly relevant with respect to known mols.  Beyond enumeration, understanding and exploiting GDBs (generated databases) led us to develop methods for virtual screening and visualization of very large databases in the form of a "periodic system of mols." comprising six different fingerprint spaces, with web-browsers for nearest neighbor searches, and the MQN- and SMIfp-Mapplet application for exploring color-coded principal component maps of GDB and other large databases.  Proof-of-concept applications of GDB for drug discovery were realized by combining virtual screening with chem. synthesis and activity testing for neurotransmitter receptor and transporter ligands.  One surprising lesson from using GDB for drug analog searches is the incredible depth of chem. space, i.e., the fact that millions of very close analogs of any mol. can be readily identified by nearest-neighbor searches in the MQN-space of the various GDBs.  The chem. space project has opened an unprecedented door on chem. diversity.  Ongoing and yet unmet challenges concern enumerating mols. beyond 17 atoms and synthesizing GDB mols. with innovative scaffolds and pharmacophores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxJYiz7ZE3ALVg90H21EOLACvtfcHk0li49PcX7OkLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D&md5=d3199b2273a0e8d92a4b1ffa59c6dffc</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Far500432k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500432k%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3DThe%2520Chemical%2520Space%2520Project%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26issue%3D3%26spage%3D722%26epage%3D730%26doi%3D10.1021%2Far500432k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span> <span> </span><span class="NLM_article-title">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&issue=3&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+Complexity+and+Its+Impact+on+the+Probability+of+Finding+Leads+for+Drug+Discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0li49PcX7OkLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520Complexity%2520and%2520Its%2520Impact%2520on%2520the%2520Probability%2520of%2520Finding%2520Leads%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26issue%3D3%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenjans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Increasing the Delivery of next Generation Therapeutics from High Throughput Screening Libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2015.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=10.1016%2Fj.cbpa.2015.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=25909818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=104-110&author=M.+J.+Wigglesworthauthor=D.+C.+Murrayauthor=C.+J.+Blackettauthor=M.+Kossenjansauthor=J.+W.+M.+Nissink&title=Increasing+the+Delivery+of+next+Generation+Therapeutics+from+High+Throughput+Screening+Libraries&doi=10.1016%2Fj.cbpa.2015.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span></div><div class="casAuthors">Wigglesworth, Mark J.; Murray, David C.; Blackett, Carolyn J.; Kossenjans, Michael; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-110</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry has historically relied on high throughput screening as a cornerstone to identify chem. equity for drug discovery projects.  However, with pharmaceutical companies moving through a phase of diminished returns and alternative hit identification strategies proving successful, it is more important than ever to understand how this approach can be used more effectively to increase the delivery of next generation therapeutics from high throughput screening libraries.  There is a wide literature that describes HTS and fragment based screening approaches which offer clear direction on the process for these two distinct activities.  However, few people have considered how best to identify medium to low mol. wt. compds. from large diversity screening sets and increase downstream success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFeNO1FIRBLVg90H21EOLACvtfcHk0li49PcX7OkLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D&md5=4b4bd0e7992796bd208a99dcb099c785</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2015.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DD.%2BC.%26aulast%3DBlackett%26aufirst%3DC.%2BJ.%26aulast%3DKossenjans%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DIncreasing%2520the%2520Delivery%2520of%2520next%2520Generation%2520Therapeutics%2520from%2520High%2520Throughput%2520Screening%2520Libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D26%26spage%3D104%26epage%3D110%26doi%3D10.1016%2Fj.cbpa.2015.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span>FASTLab. <a href="https://github.com/Novartis/MoaBox" class="extLink">https://github.com/Novartis/MoaBox</a> (accessed Jul 23, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FASTLab.+https%3A%2F%2Fgithub.com%2FNovartis%2FMoaBox+%28accessed+Jul+23%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01332">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65656"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01332?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01332</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Computational methods for the predictive models and the different clustering methods (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01332/suppl_file/jm0c01332_si_001.pdf">jm0c01332_si_001.pdf (778.25 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01332&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01332%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01332" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a7af02c302362","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
