
<html lang="en"     class="pb-page"  data-request-id="40dfa4a9-a4f8-4eb6-a6ae-e868272574b4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c00978;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)" /></meta><meta name="dc.Creator" content="Toufike  Kanouni" /></meta><meta name="dc.Creator" content="Christophe  Severin" /></meta><meta name="dc.Creator" content="Robert W.  Cho" /></meta><meta name="dc.Creator" content="Natalie Y.-Y.  Yuen" /></meta><meta name="dc.Creator" content="Jiangchun  Xu" /></meta><meta name="dc.Creator" content="Lihong  Shi" /></meta><meta name="dc.Creator" content="Chon  Lai" /></meta><meta name="dc.Creator" content="Joselyn R.  Del Rosario" /></meta><meta name="dc.Creator" content="Ryan K.  Stansfield" /></meta><meta name="dc.Creator" content="Lee N.  Lawton" /></meta><meta name="dc.Creator" content="David  Hosfield" /></meta><meta name="dc.Creator" content="Shawn  O’Connell" /></meta><meta name="dc.Creator" content="Matt M.  Kreilein" /></meta><meta name="dc.Creator" content="Paula  Tavares-Greco" /></meta><meta name="dc.Creator" content="Zhe  Nie" /></meta><meta name="dc.Creator" content="Stephen W.  Kaldor" /></meta><meta name="dc.Creator" content="James M.  Veal" /></meta><meta name="dc.Creator" content="Jeffrey A.  Stafford" /></meta><meta name="dc.Creator" content="Young K.  Chen" /></meta><meta name="dc.Description" content="Histone demethylase LSDl (KDMlA) belongs to the flavin adenine dinucleotide (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biology. Dysregulation and overexpre..." /></meta><meta name="Description" content="Histone demethylase LSDl (KDMlA) belongs to the flavin adenine dinucleotide (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biology. Dysregulation and overexpre..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 9, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00978" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00978" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00978" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00978" /></link>
        
    
    

<title>Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00978" /></meta><meta property="og:title" content="Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0011.jpeg" /></meta><meta property="og:description" content="Histone demethylase LSDl (KDMlA) belongs to the flavin adenine dinucleotide (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biology. Dysregulation and overexpression of LSD1 are hallmarks of a number of human diseases, particularly cancers that are characterized as morphologically poorly differentiated. As such, inhibitors of LSD1 have potential to be beneficial as a cancer therapy. The most clinically advanced inhibitors of LSDl are covalent inhibitors derived from tranylcypromine (TCP). Herein, we report the discovery of a novel series of reversible and selective LSDl inhibitors. Exploration of structure–activity relationships (SARs) and optimization of ADME properties resulted in the identification of clinical candidate CC-90011. CC-90011 exhibits potent on-target induction of cellular differentiation in acute myeloid leukemia (AML) and small cell lung cancer (SCLC) cell lines, and antitumor efficacy in patient-derived xenograft (PDX) SCLC models. CC-90011 is currently in phase 2 trials in patients with first line, extensive stage SCLC (ClinicalTrials.gov identifier: NCT03850067)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00978"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00978">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00978&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00978&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00978&amp;href=/doi/10.1021/acs.jmedchem.0c00978" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14522-14529</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00797" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Drug Annotation</span><a href="/doi/10.1021/acs.jmedchem.0c01163" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Toufike Kanouni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Toufike Kanouni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fount Therapeutics, LLC, San Diego, California 92130, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Toufike++Kanouni">Toufike Kanouni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christophe Severin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christophe Severin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christophe++Severin">Christophe Severin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert W. Cho</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert W. Cho</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quanticel Pharmaceuticels, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+W.++Cho">Robert W. Cho</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natalie Y.-Y. Yuen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalie Y.-Y. Yuen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oric Pharmaceuticals, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalie+Y.-Y.++Yuen">Natalie Y.-Y. Yuen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiangchun Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiangchun Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiangchun++Xu">Jiangchun Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lihong Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihong Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihong++Shi">Lihong Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chon Lai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chon Lai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chon++Lai">Chon Lai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joselyn R. Del Rosario</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joselyn R. Del Rosario</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joselyn+R.++Del+Rosario">Joselyn R. Del Rosario</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ryan K. Stansfield</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ryan K. Stansfield</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">858 Therapeutics, Inc., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ryan+K.++Stansfield">Ryan K. Stansfield</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lee N. Lawton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lee N. Lawton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lee+N.++Lawton">Lee N. Lawton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Hosfield</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Hosfield</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">University of Chicago, Chicago, Illinois 60637, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Hosfield">David Hosfield</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shawn O’Connell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shawn O’Connell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Inc., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shawn++O%E2%80%99Connell">Shawn O’Connell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matt M. Kreilein</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matt M. Kreilein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, Summit, New Jersey 07901, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matt+M.++Kreilein">Matt M. Kreilein</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paula Tavares-Greco</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paula Tavares-Greco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, Summit, New Jersey 07901, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paula++Tavares-Greco">Paula Tavares-Greco</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhe Nie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhe Nie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Schrödinger, Inc., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhe++Nie">Zhe Nie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen W. Kaldor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen W. Kaldor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fount Therapeutics, LLC, San Diego, California 92130, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+W.++Kaldor">Stephen W. Kaldor</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James M. Veal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James M. Veal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">858 Therapeutics, Inc., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+M.++Veal">James M. Veal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeffrey A. Stafford</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeffrey A. Stafford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">858 Therapeutics, Inc., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+A.++Stafford">Jeffrey A. Stafford</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Young K. Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Young K. Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bristol Myers Squibb, 10300 Campus Point Drive, Suite 100, San Diego, California 92121, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3940564c575e175a515c57795b544a175a5654"><span class="__cf_email__" data-cfemail="e69f89938881c8858e8388a6848b95c885898b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Young+K.++Chen">Young K. Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9354-0302" title="Orcid link">http://orcid.org/0000-0001-9354-0302</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00978&amp;href=/doi/10.1021%2Facs.jmedchem.0c00978" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14522–14529</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 9, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 June 2020</li><li><span class="item_label"><b>Published</b> online</span>9 October 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00978" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00978</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 <span class="NLM_copyright-holder">American Chemical Society</span></strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00978"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">15858</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00978" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Toufike&quot;,&quot;last_name&quot;:&quot;Kanouni&quot;},{&quot;first_name&quot;:&quot;Christophe&quot;,&quot;last_name&quot;:&quot;Severin&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;W. Cho&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;Y.-Y. Yuen&quot;},{&quot;first_name&quot;:&quot;Jiangchun&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Lihong&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Chon&quot;,&quot;last_name&quot;:&quot;Lai&quot;},{&quot;first_name&quot;:&quot;Joselyn&quot;,&quot;last_name&quot;:&quot;R. Del Rosario&quot;},{&quot;first_name&quot;:&quot;Ryan&quot;,&quot;last_name&quot;:&quot;K. Stansfield&quot;},{&quot;first_name&quot;:&quot;Lee&quot;,&quot;last_name&quot;:&quot;N. Lawton&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Hosfield&quot;},{&quot;first_name&quot;:&quot;Shawn&quot;,&quot;last_name&quot;:&quot;O’Connell&quot;},{&quot;first_name&quot;:&quot;Matt&quot;,&quot;last_name&quot;:&quot;M. Kreilein&quot;},{&quot;first_name&quot;:&quot;Paula&quot;,&quot;last_name&quot;:&quot;Tavares-Greco&quot;},{&quot;first_name&quot;:&quot;Zhe&quot;,&quot;last_name&quot;:&quot;Nie&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;W. Kaldor&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;M. Veal&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;A. Stafford&quot;},{&quot;first_name&quot;:&quot;Young&quot;,&quot;last_name&quot;:&quot;K. Chen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14522-14529&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00978&quot;},&quot;abstract&quot;:&quot;Histone demethylase LSDl (KDMlA) belongs to the flavin adenine dinucleotide (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biology. Dysregulation and overexpression of LSD1 are hallmarks of a number of human diseases, particularly cancers that are characterized as morphologically poorly differentiated. As such, inhibitors of LSD1 have potential to be beneficial as a cancer therapy. The most clinically advanced inhibitors of LSDl are covalent inhibitors derived from tranylcypromine (TCP). Herein, we report the discovery of a novel series of reversible and selective LSDl inhibitors. Exploration of structure–activity relationships (SARs) and optimization of ADME properties resulted in the identification of clinical candidate CC-90011. CC-90011 exhibits potent on-target induction of cellular differentiation in acute myeloid leukemia (AML) and small cell lung cancer (SCLC) cell lines, and antitumor efficacy in patient-derived xenograft (PDX) SCLC models. CC-90011 is currently &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00978&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00978" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00978&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00978" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00978&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00978" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00978&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00978&amp;href=/doi/10.1021/acs.jmedchem.0c00978" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00978" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00978" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00978%26sid%3Dliteratum%253Aachs%26pmid%3D33034194%26genre%3Darticle%26aulast%3DKanouni%26date%3D2020%26atitle%3DDiscovery%2Bof%2BCC-90011%253A%2BA%2BPotent%2Band%2BSelective%2BReversible%2BInhibitor%2Bof%2BLysine%2BSpecific%2BDemethylase%2B1%2B%2528LSD1%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14522%26epage%3D14529%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291881" title="Differentiation">Differentiation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0011.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Histone demethylase LSDl (KDMlA) belongs to the flavin adenine dinucleotide (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biology. Dysregulation and overexpression of LSD1 are hallmarks of a number of human diseases, particularly cancers that are characterized as morphologically poorly differentiated. As such, inhibitors of LSD1 have potential to be beneficial as a cancer therapy. The most clinically advanced inhibitors of LSDl are covalent inhibitors derived from tranylcypromine (TCP). Herein, we report the discovery of a novel series of reversible and selective LSDl inhibitors. Exploration of structure–activity relationships (SARs) and optimization of ADME properties resulted in the identification of clinical candidate CC-90011. CC-90011 exhibits potent on-target induction of cellular differentiation in acute myeloid leukemia (AML) and small cell lung cancer (SCLC) cell lines, and antitumor efficacy in patient-derived xenograft (PDX) SCLC models. CC-90011 is currently in phase 2 trials in patients with first line, extensive stage SCLC (ClinicalTrials.gov identifier: NCT03850067).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduciton</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07146" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07146" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Differentiation Agents</h3><div class="NLM_p">Tumors that are characterized as poorly differentiated are among the most lethal and difficult to treat as a result of uncontrolled proliferation of immature cells. Efforts to combat poorly differentiated tumors with targeted cytotoxic agents have had limited success, as resistance often develops. While cytotoxic agents are generally effective in debulking tumors, they often leave quiescent population of tumor cells to evolve and recruit auxiliary pathways for survival and evasion, hence, rendering cytotoxic therapies ineffective with prolonged treatment. An alternative approach to targeting poorly differentiated tumor is by inducing progression to terminal differentiation, as terminally differentiated cells have a limited capacity to proliferate and a finite lifespan.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> An early example of a differentiation inducing agent is all-trans retinoic acid (ATRA). ATRA, a metabolite of retinol-A, functions by inducing immature leukemic blast cells to undergo terminal differentiation and apoptosis. This discovery has been transformative for patients suffering from acute-promyelocytic leukemia (APL), a subset of AML with a chromosomal translocation of retinoic acid receptor. Prior to the discovery of ATRA, the prognosis for patients with APL was extremely poor, with patients succumbing to rapid onset of hematological toxicity. The cure rates for ATRA in combination with arsenic trioxide treatment are as high as 80% for APL patients.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Isocitrate dehydrogenase inhibitors (IDH1 and IDH2) have also been shown to successfully induce differentiation in another subset of AML patients that harbor IDH mutations. The IDH mutants produce the oncometabolite <span class="smallcaps smallerCapital">d</span>-2-hydroxyglutarate that inhibits downstream 2-oxoglutarate dependent epigenetic modifiers, resulting in differentiation arrest of blast cells.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> More recently, the discovery of inhibitors of dihydroorotate dehydrogenase (DHODH), an enzyme involved in the de novo pyrimidine synthesis pathway, is anticipated to further broaden the repertoire of differentiation inducing agents for combating AML.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> While advances in differentiation agents are showing great promise for hematological malignancy, the same is not true for patients with solid tumors.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Herein, we report the discovery of clinical candidate CC-90011, a potent reversible inhibitor for lysine demethylase-1 LSD1 (KDM1A) that induces cellular differentiation of poorly differentiated and highly metastatic small cell lung carcinoma (SCLC).</div><div class="NLM_p">LSD1 is a member of a broad family of monoamine oxidases that requires FAD as a cofactor to catalyze demethylase activity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> LSD1 catalyzes demethylation at histone 3 lysine 4 methyl 1/2 (H3K4me1/2) resulting in a transcriptional repressive state. However, LSD1 can also demethylate histone 3 lysine 9 methyl 1/2 (H3K9me1/2) to act as a transcriptional activator in certain cellular context. LSD1 is also known to demethylate nonhistone protein p53 and DNMT1.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The LSD1 protein structure is comprised of three domains. The SWIRM and Tower domains function as scaffolding for multiprotein complex formation. The amine oxidase domain (AO) contains the active site, wherein a catalytic lysine residue (K661) acts to deprotonate a methylated histone lysine (e.g., H3K4me2), prompting hydride transfer to FAD.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> It is noteworthy that catalytic activity of the protein is lost upon mutations to K661. However, recent reports have demonstrated that the AO pocket also functions as scaffolding for transcription regulators of cellular differentiation and proliferation, such as GFI1 and INSM1.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> These proteins bind to the AO pocket via their SNAG domains.</div><div class="NLM_p last">Under normal physiological conditions, epigenetic machinery tightly regulates stem cell fate to self-renew, differentiate, and proliferate. When functioning properly, this process is orderly and self-limiting. However, dysregulation of epigenetic machinery has been implicated in pathological conditions found in various cancers. Dysregulation and overexpression of LSD1 have been implicated in multiple cancer types that are trapped in differentiation arrest. LSD1 is enriched in 60% of leukemic blasts with oncogenic fusion protein MLL-AF9.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In this context, genetic knock down experiment of LSD1 in a THP-1 cell line resulted in induction of cellular differentiation, as measured by CD11b and CD86 expression. LSD1 also regulates cellular differentiation and proliferation of AML with RUNX1-RUNX1T1 translocation (AML1-ETO).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In solid tumors, LSD1 regulates differentiation and proliferation of SCLC, a neuroendocrine tumor that is morphologically characterized as immature, small, round, blue cells with granular chromatin.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> SCLC is known to be highly proliferative and to have high metastatic potential. The expression level of LSD1 in primary SCLC samples is elevated in 98% of the samples that have been tested.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> High expression levels of LSD1 have also been found in breast cancer, neuroblastoma, colorectal cancer, and prostate cancer. Aberrant dysregulation of LSD1 causes differentiation arrest in both hematological and solid tumors, making it an attractive target for cancer therapy.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Clinical Stage LSD1 Inhibitors</h3><div class="NLM_p">Tranylcypromine (TCP), a MAO inhibitor marketed for the treatment of severe depression, and structurally related analogs of TCP have been developed to selectively and irreversibly inhibit LSD1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). These compounds belong to a class of inhibitors that covalently modify FAD in the AO pocket. ORY1001, the most clinically advanced inhibitor in this class, is currently in phase 2 trials for AML and SCLC.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> GSK2879552 is another irreversible inhibitor; however, clinical trials of GSK2879552 in AML and SCLC patients have recently been terminated (ClinicalTrials.gov identifier NCT02034123).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Additional clinical stage irreversible inhibitors IMG7289 (ClinicalTrials.gov identifier NCT02842827) and INCB059872 (ClinicalTrials.gov identifier NCT02712905) are in clinical trials for AML and solid tumors. A reversible <i>N</i>-acylhydrazone LSD1 inhibitor was reported and is currently being evaluated in a phase 1 trial for AML.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Clinical stage LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Rationale for Reversible Inhibitors</h3><div class="NLM_p last">Given our understanding of LSD1 biology, we hypothesized that reversible inhibition of LSD1 could provide improved therapeutic outcomes compared to irreversible inhibition. LSD1 plays critical roles in maintaining cellular homeostasis in both embryonic stem cells (ESCs) and adult hematopoiesis. Genetic deletion of LSD1 in murine ESCs resulted in early death, while knock down of LSD1 in hematopoietic in adults leads to pancytopenia, albeit clinically manageable.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Given these findings, we surmised that irreversibly disabling LSD1 could lead to increase of on-target toxicity. Furthermore, reversible differentiation agents, such as ATRA, have been successfully employed for treating APL patients with minimal toxicity. As such, a reversible inhibitor of LSD1 could provide a therapeutic window to induce differentiation of tumor cells while limiting the potential on-target toxicity to normal cells.</div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54513" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54513" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Identification of Lead Reversible Inhibitors</h3><div class="NLM_p">A high throughput screen of a 300 000 compound diversity library and a smaller structure-based design library resulted in 75 compounds of interest from several distinct chemotypes. Within this subset, we identified compound <b>1</b> (1 μM) as a highly attractive starting point for further SAR exploration. A binding model of compound <b>1</b> was created from a crystal structure of the LSD1 AO pocket (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HKO">2HKO</a>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) and used to guide SAR optimization.<a onclick="showRef(event, 'cit6b'); return false;" href="javascript:void(0);" class="ref cit6b">(6b)</a> In the model, compound <b>1</b> forms an important H-bonding interaction between the nitrogen of the benzonitrile and catalytic lysine (K661) in the FAD groove. A salt-bridge interaction between the positively charged pyrrolidine side chain and D555 further enhances binding in the AO pocket. The Cl substituent projects into a hydrophobic pocket (W695, I356, L677, L692, and F358) adjacent to the narrow groove of FAD.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Model of compound <b>1</b> in the AO pocket of LSD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HKO">2HKO</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Introduction of lipophilic groups such as <i>p</i>-tolyl resulted in pyrimidine <b>2</b> (LSD1 IC<sub>50</sub> = 2 nM, CD11b EC<sub>50</sub> = 28 nM; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which represents a 500-fold improvement in biochemical potency and potent induction of CD11b in a THP1 cell line, as assessed by fluorescence-activated cell sorting (FACS) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">Supporting Information</a>). CD11b is a myeloid differentiation surface marker that is directly regulated by LSD1, and it was employed as a biomarker,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> while we were pleased that pyrimidine <b>2</b> could induce CD11b expression in a pharmacologically useful range, supporting the hypothesis that reversible inhibition is a viable strategy. However, <b>2</b> was limited by poor oral bioavailability (AUC<sub>0–6h</sub> = 1 μM·h) and potent hERG inhibition (>95% at 10 μM). Replacement of the pyrrolidine of <b>2</b> with a fused bicyclic ring system served to rigidify and lock <b>2</b> in a low energy conformation for optimal engagement of D555 and simultaneously eliminated an unproductive H-bond donor. The resulting pyrimidine <b>3</b> had good biochemical and cellular potency (LSD1 IC<sub>50</sub> = 1.4 nM, CD11b IC<sub>50</sub> = 73 nM) with improved oral exposure (AUC<sub>0–6h</sub> = 15 μM·h). The propensity of lipophilic cationic amines to be potent hERG channel blockers is well-known and can be a major challenge to overcome.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In order to address this liability, we devised a strategy of incremental structural changes and judicious placement of heteroatoms to lower the overall lipophilicity, all while maintaining potent cellular activity and oral exposure. To this end, modification of the fused bicyclic ring in <b>3</b> to a structurally simple aminopiperidine <b>4</b> maintained good potency (LSD1 IC<sub>50</sub> = 3.0 nM, CD11b IC<sub>50</sub> = 28 nM) and lowered overall lipophilicity, as measured by cLogD (2.6). Replacement of the central pyrimidine core with pyrazinone <b>5</b> retained potency (LSD1 IC<sub>50</sub> = 1.4 nM, CD11b IC<sub>50</sub> = 47 nM) while maintaining good oral exposure (AUC<sub>0–6h</sub> = 7.3 μM·h). Opportunities for modification of the benzonitrile were limited due to lack of space in the narrow FAD groove. However, addition of a fluorine at the 3-position of the benzonitrile <b>6</b> was accommodated and provided a further boost in potency (LSD1 IC<sub>50</sub> = 0.5 nM, CD11b IC<sub>50</sub> = 18 nM). The positional isomer with a fluorine substituent in the 2-position (<b>7</b>) was disfavored, resulting in a significant loss in biochemical and cellular potency (LSD1 IC<sub>50</sub> = 11 nM, CD11b IC<sub>50</sub> = 120 nM). Pyrimidinone <b>8</b> compared favorably to pyrazinone <b>5</b> with respect to potency (LSD1 IC<sub>50</sub> = 0.5 nM, CD11b IC<sub>50</sub> = 18 nM), and replacement of <i>p</i>-Me with <i>p</i>-OMe (<b>9</b>) resulted in meaningful lowering of hERG channel inhibition (67%). Addition of fluorine in the 3-position of methoxyphenyl group (<b>10</b>) further reduced hERG channel inhibition (48%) while maintaining good overall properties. Addition of a fluorine substitution in the 3-position of the benzonitrile resulted in compound <b>11</b> (CC-90011). Compound <b>11</b> demonstrated potent inhibition of LSD1 at 0.3 nM and induction of on-target cellular differentiation marker CD11b in THP-1 cell line with EC<sub>50</sub> = 7 nM, antiproliferative activity in AML kasumi-1 cells with EC<sub>50</sub> = 2 nM, and no effect in normal human fibroblasts (IMR-90). The selectivity profile of compound <b>11</b> was tested against FAD dependent amine-oxidases LSD2, MOA-A, and MAO-B, and no enzymatic inhibition was observed below 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of SAR of LSD1 Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">LSD1 IC<sub>50</sub> was determined using a TR-Fret assay.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">CD11b assay was assessed by FACS in THP1 cell line.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">10 mg/kg dose.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Percent hERG channel inhibition at 10 μM.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Calculated log <i>D</i>.</p></div></div><div></div></div><div class="NLM_p">Mouse pharmacokinetic properties were evaluated in CD-1 mice by administration of a single 5 mg/kg dose of compound <b>11</b> intravenously (iv) and by oral gavage (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). After iv administration, compound <b>11</b> had systemic clearance of 32.4 mL min<sup>–1</sup> kg<sup>–1</sup>, elimination half-life of 2 h, and a high volume of distribution of 7.5 L/kg. Compound <b>11</b> was readily absorbed after oral administration with an AUC<sub>0–24h</sub> of 1.8 μM·h, <i>C</i><sub>max</sub> of 0.36 μM, and oral bioavailability of 32%. The effect of compound <b>11</b> on hERG potassium channels (HEK293 cells stably transfected with hERG cDNA) was examined by patch clamp. The IC<sub>50</sub> value for compound <b>11</b> was determined to be 3.4 μM which provided sufficient cardiotoxicity safety margins to advance the compound for in vivo studies.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Pharmacokinetics and hERG Profile of <b>11</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0010.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center"><b>11</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iv/po (mg/kg)</td><td class="colsep0 rowsep0" align="left">5/5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cl (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">32.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Vd<sub>ss</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–24h</sub> (μM·h)</td><td class="colsep0 rowsep0" align="left">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG (μM)</td><td class="colsep0 rowsep0" align="left">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse PPB (%)</td><td class="colsep0 rowsep0" align="left">92</td></tr></tbody></table></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Cocrystal Structure of CC-90011/LSD1</h3><div class="NLM_p">An X-ray crystal structure of compound <b>11</b> bound to the LSD1/CoREST complex was determined at 2.92 Å (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As expected, the inhibitor forms a rigid Y-shaped configuration with the central pyrimidinone core projecting the aminopiperidine moiety to form a salt-bridge with D555. The benzonitrile projects into a hydrophobic pocket formed by FAD and several amino acid side chains. An H-bond interaction of catalytic K661 with the nitrile appears to have displaced the catalytic water molecule. The 2-F-anisole ring forms an additional hydrophobic interaction within a shallow pocket adjacent to the FAD pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of compound <b>11</b> in the AO pocket of LSD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Synthesis of <b>11</b> or CC-90011</h3><div class="NLM_p">The synthesis of <b>11</b> was achieved in six steps starting from readily available perchloropyrimidine <b>12</b>, as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The chemoselective displacement of the chlorine in the 4-position of perchloropyrimidine was achieved using aqueous base (1.3 equiv of NaOH) at ambient temperature, resulting in 69% yield of pyrimidinone <b>13</b>. Treatment of <b>13</b> with iodomethane resulted in the thermodynamically favorable <i>N</i>-methylpyrimidinone adduct <b>14</b> in good yield (62%). Treatment of <i>N</i>-methylpyrimidinone <b>14</b> with <i>N</i>-Boc-aminopiperidine led to the displacement of the 2-position chlorine to give <b>15</b> in excellent yield (80%). Selective Suzuki cross coupling reaction with (4-cyano-3-fluorophenyl)boronic acid in ACN/water at 85 °C furnished intermediate <b>16</b> (57% yield). A second Suzuki cross coupling with intermediate <b>16</b> and (3-fluoro-4-methoxyphenyl)boronic acid in dioxane/water at 80 °C afforded <b>17</b> in excellent yield (92%). Finally, deprotection with BSA in the presence of formic acid furnished compound <b>11</b> (CC-90011) as the benzenesulfonate salt in 95% yield. Notably, each chlorine atom in perchloropyrimidine <b>12</b> was chemoselectively functionalized to furnish CC-90011 in six linear steps and 17% overall yield.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>11</b> or CC-90011<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) 1.3 equiv of NaOH, THF, rt, overnight, 69% yield; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, overnight, 62% yield; (c) <i>tert</i>-butyl piperidin-4-ylcarbamate, DIEA, DMF, rt, overnight, 80% yield; (d) 4-cyano-3-fluorophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O, 85 °C, 57% yield; (e) (3-fluoro-4-methoxyphenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 80 °C, 92% yield; (f) benzenesulfonic acid, formic acid.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Pharmacology of Compound <b>11</b></h3><div class="NLM_p">LSD1 inhibitors have been shown to suppress GRP expression in neuroendocrine tumors such as SCLC. ChIP-seq analysis in SCLC cell lines (H69 and H209) showed LSD1 co-occupies sites with the enhancer mark H3K4me1 surrounding the GRP locus (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">Supporting Information</a>) linking LSD1 to regulation of gastrin-releasing peptide (GRP) expression, which was used as a pharmacodynamic (PD) biomarker. Consistent with the ChIP-seq results, suppression of GRP was observed with treatment of compound <b>11</b> (4 days) in a dose-dependent manner and at pharmacologically useful concentrations (EC<sub>50</sub> = 3 nM, H209 and 4 nM, H1417) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In addition, treatment of SCLC cells with compound <b>11</b> for an extended incubation period (12 days) in vitro resulted in potent antiproliferative activity (EC<sub>50</sub> = 6 nM, H1417) that correlated with GRP suppression (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). It has been suggested that loss of GRP expression is a result of SCLC cells undergoing differentiation and losing their neuroendocrine status.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) GRP mRNA expression in H209 and H1417 cell lines with treatment of compound <b>11</b>. (B) Cell-Titer Glo viability assay in H1417 cell line with treatment using compound <b>11</b>. (C) In vivo effect of compound <b>11</b> treatment on human GRP mRNA expression in SCLC H1417 model at 2.5 mg/kg, 5 mg/kg, and 10 mg/kg QD dosing for 4 days, <i>n</i> = 3 mice/group. (D) In vivo efficacy of compound <b>11</b> in mice bearing SCLC xenograft model H1417 at 2.5 mg/kg and 5 mg/kg QD dosing for 65 days, <i>n</i> = 7 or 8 mice/group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the excellent on-target in vitro activity and PK properties, compound <b>11</b> was advanced into a SCLC human tumor xenograft (H1417) study in mice. Mice were inoculated with H1417 tumor cells and treated QD at 2.5, 5, and 10 mg/kg of compound <b>11</b> for 4 consecutive days. GRP levels were assessed by qPCR 24 h after the last dose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Treatment with compound <b>11</b> resulted in robust downregulation of GRP mRNA levels at 2.5 mg/kg and maximum suppression of GRP at 5 mg/kg. In a separate study, mice were implanted with H1417 SCLC tumors and treated with compound <b>11</b> daily at 2.5 and 5 mg/kg, which resulted in tumor growth regressions of 159% and 178%, respectively (<i>P</i>-value of 0.001 and 0.0001) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). Compound <b>11</b> was well tolerated for the duration of this study (65 days) with an average maximal body weight gain of up to 7% (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">Supporting Information</a>). Maximum efficacy was observed at 5 mg/kg QD in both xenograft studies. These results established 5 mg/kg QD of <b>11</b> as the efficacious dose for the in vivo studies.</div><div class="NLM_p">To assess efficacy of compound <b>11</b> in clinically relevant patient-derived xenograft models (PDX), mice were inoculated with human LXFS 615 SCLC cells and treated with compound <b>11</b> at 5 mg/kg daily for 30 consecutive days. Compound treatment resulted in 78% TGI (<i>P</i>-value of 0.001) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Compound <b>11</b> was well tolerated with mean body weight loss of <1% (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">Supporting Information</a>). The efficacy observed in this PDX model is in line with other SCLC PDX models that had been treated with compound <b>11</b>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo efficacy of compound <b>11</b> in mice bearing PDX model LXFS615 at 5 mg/kg QD dosing for 30 days, <i>n</i> = 10 mice/group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>11</b> was evaluated in a separate SCLC PDX model (LU2514) wherein treatment with compound <b>11</b> at 5 and 10 mg/kg achieved tumor growth inhibition of 39% and 56%, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Additionally, marked cellular morphology changes (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) were observed upon H&E staining of tumor cells collected 15 days after the final dose (10 mg/kg group). Vehicle-treated tumor cells maintained a classic poorly differentiated round cell morphology with granular chromatin. On the other hand, compound <b>11</b> treated tumor cells was shown to have a looser cellular structure, decreased nucleus to cytoplasm ratio, and higher number of apoptotic bodies (indicated by yellow arrows). These results clearly demonstrate a change in differentiation status of LU2514 cells consistent with the mechanistic hypothesis. The phenotypic effect upon inhibition LSD1 is generally cytostatic, but the underlining differentiation biology is pronounced and warrants further investigation.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) In vivo efficacy of compound <b>11</b> in mice bearing PDX model LU2514 at 5 and 10 mg/kg QD dosing for 28 days, <i>n</i> = 8 or 10 mice/group. (b) H&E staining of vehicle and compound <b>11</b> treated (10 mg/kg) tumor (LU2514) 14 days after last dose.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Conventional targeted agents for combating cancers have focused on promoting apoptosis in rapidly proliferating cancer cells. These agents have proven effective in reducing tumor bulk, but prolonged treatment has often resulted in tumors developing resistance. An alternative strategy is to suppress proliferation by inducing terminal differentiation of the cancer cells. This strategy is employed less frequently but has been highly effective in subsets of AML. The discovery of CC-90011, a highly potent and reversible inhibitor of LSD1, provides a novel differentiation strategy for the treatment of neuroendocrine tumors and AML. Phase 1 study of CC-90011 in patients with advanced solid tumors has been completed, and the safety, tolerability, and preliminary efficacy have been reported.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> CC-90011 is currently in phase 2 trials in patients with first line, extensive stage SCLC (ClinicalTrials.gov identifier NCT03850067).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Chemical Synthesis Methods</h3><div class="NLM_p last">All solvents were purchased commercially and utilized without further purification. All reagents were obtained from commercial suppliers and used without further purification. All reactions were conducted under an inert atmosphere at room temperature unless otherwise stated. <sup>1</sup>H NMR spectra were recorded on a Bruker Advance 400. Chemical shifts are presented in parts per million (ppm). Coupling constants are in units of hertz (Hz). Splitting patterns describe apparent multiplicities and are designated as s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), and br (broad). Low-resolution mass spectrometry (MS) and compound purity data were acquired on a Shimazu LC/MS single quadrupole system equipped with electrospray ionization (ESI) source and UV detector (220 and 254 nm). The purity of final tested compounds was ≥95% as determined by <sup>1</sup>HNMR, LC/MS, and HPLC.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2,5,6-Trichloropyrimidin-4-ol (<b>13</b>)</h3><div class="NLM_p last">To a solution of perchloropyrimidine <b>12</b> (1 kg, 4.63 mol) in THF (6 L) was added 1 N NaOH (6 L, 6.0 mol) dropwise, and the mixture was stirred overnight at room temperature. Upon completion, the solution was acidified with 1 N HCl and extracted with DCM (3×). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. The solids were suspended and stirred in Et<sub>2</sub>O for 30 min. The heterogeneous mixture was successively filtered, washed with Et<sub>2</sub>O, and dried to give 635 g (69%) of the title compound <b>13</b>. LCMS (C18 column, column size 4.6 mm × 50 mm; mobile phase 20–40%, acetonitrile–water–0.02% NH<sub>4</sub>OAc): <i>t</i><sub>R</sub> = 2.785 min; [M + H] calcd for C<sub>4</sub>HCl<sub>3</sub>N<sub>2</sub>O, 199; found, 199.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 2,5,6-Trichloro-3-methyl-3-hydropyrimidin-4-one (<b>14</b>)</h3><div class="NLM_p last">A solution of 2,5,6-trichloropyrimidin-4-ol <b>13</b> (670 g, 3.38 mol) and K<sub>2</sub>CO<sub>3</sub> (560 g, 4.06 mol) in DMF (5 L) was stirred at room temperature for 15 min and then cooled to 0 °C. Iodomethane (528 g, 3.72 mol) was added dropwise, and the mixture was stirred at room temperature for 17 h. The reaction mixture was taken up in ethyl acetate and the combined organic layers were successively washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. The residue was purified by silica gel chromatography (PE:EA, 10:1) to give 447 g (62%) of the title compound <b>14</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.74 (s, 3 H). LCMS (C18 column; column size 4.6 mm × 50 mm; mobile phase 20–95%, acetonitrile–water–0.02% NH<sub>4</sub>OAc): <i>t</i><sub>R</sub> = 2.986 min; [M + H] calcd for C<sub>5</sub>H<sub>3</sub>Cl<sub>3</sub>N<sub>2</sub>O, 213; found, 213.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>N</i>-[1-(5,6-Dichloro-3-methyl-4-oxo(3-hydropyrimidin-2-yl))(4-piperidyl)](<i>tert</i>-butoxy)carboxamide (<b>15</b>)</h3><div class="NLM_p last">A solution of 2,5,6-trichloro-3-methyl-3-hydropyrimidin-4-one <b>14</b> (532 g, 2.51 mol), DIEA (648 g, 5.02 mol), and (<i>tert</i>-butoxy)-<i>N</i>-(4-piperidyl)carboxamide (502 g, 2.51 mol) in DMF (800 mL) was heated to 120 °C for 1 h. The solvent was removed in vacuo and the residue was purified by silica gel chromatography (PE:EA, 1:1) to give 751 g (80%) of the title compound <b>15</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.45 (s, 9H), 1.50–1.58 (m, 2H), 2.06–2.10 (m, 2H), 2.98–3.05 (m, 2H), 3.48 (s, 3 H), 3.53–3.56 (m, 2H), 3.70 (s, 1H), 4.52 (s, 1H). LCMS (C18 column; column size 4.6 mm × 50 mm; mobile phase 20–95%, acetonitrile–water–0.02% NH<sub>4</sub>OAc): <i>t</i><sub>R</sub> = 4.006 min; [M + H] calcd for C<sub>15</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub>, 377; found, 321 (MW-tBu).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>tert</i>-Butyl 1-(5-Chloro-4-(3-fluoro-4-cyanophenyl)-1-methyl-6-oxo-1,6-dihydropyrimidin-2-yl)piperidin-4-ylcarbamate (<b>16</b>)</h3><div class="NLM_p last">To a solution of <i>N</i>-[1-(5,6-dichloro-3-methyl-4-oxo(3-hydropyrimidin-2-yl))(4-piperidyl)](<i>tert</i>-butoxy)carboxamide <b>15</b> (150 g, 0.40 mol) in ACN (4 L), under N<sub>2</sub> atmosphere, was added 3-fluoro-4-cyanophenylboronic acid (65.8 g, 0.40 mol), Pd(Ph<sub>3</sub>P)<sub>4</sub> (9.3 g, 8 mmol), and Na<sub>2</sub>CO<sub>3</sub> (0.4 N, 2 L, 0.80 mol). The mixture was stirred at 85 °C for 2 h. Upon completion, the reaction was allowed to cool to room temperature. Water (2 L) was added, and the aqueous mixture was extracted with ethyl acetate (3×). The combined organic layers were successively washed with water (3×), brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. The residue was purified by silica gel chromatography (PE:EA, 3:1) to give 95 g (57%) of the title compound <b>16</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.45 (s, 9 H), 1.54–1.61 (m, 2H), 2.05–2.13 (m, 2H), 2.99–3.08 (m, 2H), 3.53–3.58 (s, 5H), 3.70 (s, 1H), 4.54 (d, <i>J</i> = 6.0 Hz, 1H), 7.68–7.80 (m, 3 H). LCMS (C18 column; column size 4.6 mm × 50 mm; mobile phase 5%–95%, acetonitrile–water–0.1% TFA): <i>t</i><sub>R</sub> = 4.443 min; [M + H] calcd for C<sub>22</sub>H<sub>25</sub>ClFN<sub>5</sub>O<sub>3</sub>, 462; found, 462.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>tert</i>-Butyl <i>N</i>-[1-[4-(4-Cyano-3-fluorophenyl)-5-(3-fluoro-4-methoxyphenyl)-1-methyl-6-oxopyrimidin-2-yl]piperidin-4-yl]carbamate (<b>17</b>)</h3><div class="NLM_p last">A mixture of (<i>tert</i>-butoxy)-<i>N</i>-{1-[5-chloro-6-(4-cyano-3-fluorophenyl)-3-methyl-4-oxo(3-hydropyrimidin-2-yl)](4-piperidyl)}carboxamide <b>16</b> (295.8 g, 640 mmol), 3-fluoro-4-methoxybenzeneboronic acid (217.7 g, 1281 mmol), Pd(Ph<sub>3</sub>P)<sub>4</sub> (18.5 g, 16.0 mmol), and K<sub>2</sub>CO<sub>3</sub> (265.5 g, 1921 mmol) in degassed dioxane:H<sub>2</sub>O (3:1, 3550 mL:887 mL) was stirred at 80 °C under N<sub>2</sub> atmosphere for 2 h. Upon completion, the reaction mixture was allowed to cool to room temperature and water (11 L) was added. The slurry was stirred for 1 h and filtered. The solids were successively washed with water (4 L) followed by MeOH:water (1:1, 4 L) mixture. The filter cake was taken up and stirred in MeOH (4 L) for 10 min. The slurry was filtered, and the solids were rinsed with MeOH (4 L) followed by washing with MTBE (4 L). The solids were taken in DCM (16 L), and the reaction mixture was stirred for 15 min. 2-Mercaptoethyl ethyl sulfide silica (400 g) was then added, and the reaction mixture was stirred at room temperature for at least 1 h under N<sub>2</sub> atmosphere. The reaction mixture was filtered through Celite in a fritted filter, and the solids were rinsed with DCM (1 L). The volume of DCM filtrate was reduced to near dryness, and MeOH (4 L) was added and concentrated in vacuo. The solids were taken up in MeOH (4 L), and the slurry was cooled to 15 °C and filtered. The filter cake was washed further with MeOH (0.7 L) followed by MTBE (0.7 L) and dried in a vacuum oven (45 °C) to a constant weight of 324.6 g (91.9%) of the title compound (<b>17</b>). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.46 (s, 9 H), 1.60 (d, <i>J</i> = 10.11 Hz, 2 H), 2.11 (d, <i>J</i> = 11.62 Hz, 2 H), 3.06 (t, <i>J</i> = 12.00 Hz, 2 H), 3.54 (s, 3 H), 3.60 (d, <i>J</i> = 13.64 Hz, 2 H), 3.72 (br s, 1 H), 3.88 (s, 3 H), 4.52 (br s, 1 H), 6.79–6.89 (m, 2 H), 6.97 (d, <i>J</i> = 12.38 Hz, 1 H), 7.13 (d, <i>J</i> = 8.34 Hz, 1 H), 7.31 (d, <i>J</i> = 9.85 Hz, 1 H), 7.42 (br s, 1 H). LCMS (C18 column; column size 4.6 mm × 50 mm; mobile phase: 5%–95%, acetonitrile–water–0.1% TFA): <i>t</i><sub>R</sub> = 6.979; [M + H] calcd for C<sub>29</sub>H<sub>31</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>, 552; found, 552.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-[2-(4-Aminopiperidin-1-yl)-5-(3-fluoro-4-methoxyphenyl)-1-methyl-6-oxo-1,6-dihydropyrimidin-4-yl]-2-fluorobenzonitrile, Besylate Salt (<b>11</b> or CC90011)</h3><div class="NLM_p last">A solution of <i>tert</i>-butyl <i>N</i>-[1-[4-(4-cyano-3-fluorophenyl)-5-(3-fluoro-4-methoxyphenyl)-1-methyl-6-oxopyrimidin-2-yl]piperidin-4-yl]carbamate (<b>17</b>) (5.0 g, 9.06 mmol) and benzenesulfonic acid monohydrate (1.9 g, 10.9 mmol) in formic acid (41.5 mL) was stirred at room temperature until reaction completion. The solution was filtered through a 0.45 μm filter. Water (25 mL) was added to the formic acid solution. Seeds of 4-[2-(4-aminopiperidin-1-yl)-5-(3-fluoro-4-methoxyphenyl)-1-methyl-6-oxo-1,6-dihydropyrimidin-4-yl]-2-fluorobenzonitrile, besylate salt (0.05g) were introduced, and the solution was aged for 30 min. Water (up to 50 mL) was added to the mixture over 6 h. The batch was then allowed to age for at least 12 h. The batch was filtered, and the cake was washed with 80/20 water/formic acid (v/v) and dried at 40–50 °C in a vacuum oven with a nitrogen bleed to give the (5.25 g, 95%) of title compound (<b>11</b> or CC-90011). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 1.69 (q, 2H, <i>J</i> = 11.4 Hz), 2.00 (d, 2H, <i>J</i> = 10.2 Hz), 2.99 (t, 2H, <i>J</i> = 12.3 Hz), 3.31 (bs, 1H), 3.42 (s, 3H), 3.72 (d, 2H, <i>J</i> = 13.2 Hz), 3.81 (s, 3H), 6.78 (d, 1H, J= 8.4 Hz), 7.01–7.06 (m, 1H), 7.04–7.06 (m, 1H), 7.19 (dd, 1H, 1.2 Hz), 7.32 (m, 2H), 7.32 (m, 1H), 7.46 (dd, 1H, <i>J</i> = 10.5, 1.2 Hz), 7.61 (m, 2H), 7.82 (dd, 1H, <i>J</i> = 8.1, 7.2 Hz), 7.92 (bs, 1H), 7.92 (bs, 2H). LCMS (column, Agilent Zorbax SB-C8, 4.6 mm × 50 mm, 3.5 μm particle size; mobile phase 5%–95%, acetonitrile–water–0.1% TFA): <i>t</i><sub>R</sub> = 3.854; [M + H] calcd for C<sub>24</sub>H<sub>23</sub>F<sub>2</sub>N<sub>5</sub>O<sub>2</sub>, 452; found, 452. A second recrystallization from 80/20 water/formic acid was performed as described above to provide material greater than 99% pure as determined by LC analysis.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00978" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01183" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01183" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00978?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00978</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Biology and pharmacology of <b>11</b>, crystallization methods, and purity information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and standard error of measurement for all biological data present (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf">jm0c00978_si_001.pdf (532.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_002.csv">jm0c00978_si_002.csv (0.77 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates and experimental data upon article publication (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a> for compound <b>11</b> bound to LSD1/CoREST).</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00978" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75470" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75470" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Young K. Chen</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9354-0302" title="Orcid link">http://orcid.org/0000-0001-9354-0302</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#522b3d273c357c313a373c12303f217c313d3f"><span class="__cf_email__" data-cfemail="11687e647f763f7279747f51737c623f727e7c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toufike Kanouni</span> - <span class="hlFld-Affiliation affiliation">Fount
Therapeutics, LLC, San Diego, California 92130, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christophe Severin</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert W. Cho</span> - <span class="hlFld-Affiliation affiliation">Quanticel
Pharmaceuticels, San Francisco, California 94158, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie Y.-Y. Yuen</span> - <span class="hlFld-Affiliation affiliation">Oric
Pharmaceuticals, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiangchun Xu</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihong Shi</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chon Lai</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joselyn R. Del Rosario</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, 10300 Campus
Point Drive, Suite 100, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ryan K. Stansfield</span> - <span class="hlFld-Affiliation affiliation">858
Therapeutics, Inc., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lee N. Lawton</span> - <span class="hlFld-Affiliation affiliation">Dana-Farber
Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Hosfield</span> - <span class="hlFld-Affiliation affiliation">University
of Chicago, Chicago, Illinois 60637, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shawn O’Connell</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Inc., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matt M. Kreilein</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, Summit, New Jersey 07901, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paula Tavares-Greco</span> - <span class="hlFld-Affiliation affiliation">Bristol
Myers Squibb, Summit, New Jersey 07901, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhe Nie</span> - <span class="hlFld-Affiliation affiliation">Schrödinger, Inc., San
Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen W. Kaldor</span> - <span class="hlFld-Affiliation affiliation">Fount
Therapeutics, LLC, San Diego, California 92130, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James M. Veal</span> - <span class="hlFld-Affiliation affiliation">858
Therapeutics, Inc., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey A. Stafford</span> - <span class="hlFld-Affiliation affiliation">858
Therapeutics, Inc., San Diego, California 92121, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Y.K.C. is grateful to Laura D’Agostino and Robert Borzilleri for valuable discussions, proofreading, and unwavering support during the drafting of this manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine demethylase 1</p></td></tr><tr><td class="NLM_term">ATRA</td><td class="NLM_def"><p class="first last">all-trans retinoic acid</p></td></tr><tr><td class="NLM_term">IDH</td><td class="NLM_def"><p class="first last">isocitrate dehydrogenase</p></td></tr><tr><td class="NLM_term">DHODH</td><td class="NLM_def"><p class="first last">dihydroorotate dehydrogenase</p></td></tr><tr><td class="NLM_term">H3K4me1/2</td><td class="NLM_def"><p class="first last">histone 3 lysine 4 methyl 1/2</p></td></tr><tr><td class="NLM_term">H3K9 me1/2</td><td class="NLM_def"><p class="first last">histone 3 lysine 9 methyl 1/2</p></td></tr><tr><td class="NLM_term">GFI1</td><td class="NLM_def"><p class="first last">growth factor independent 1</p></td></tr><tr><td class="NLM_term">INSM1</td><td class="NLM_def"><p class="first last">insulinoma associated protein 1</p></td></tr><tr><td class="NLM_term">SNAG</td><td class="NLM_def"><p class="first last">Snail1</p></td></tr><tr><td class="NLM_term">ESC</td><td class="NLM_def"><p class="first last">embryonic stem cell</p></td></tr><tr><td class="NLM_term">H&E</td><td class="NLM_def"><p class="first last">hemotoxylin and eosin</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88485" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88485" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 18 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Enane, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunthararajah, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korc, M.</span></span> <span> </span><span class="NLM_article-title">Differentiation therapy and the mechanisms that terminate cancer cell proliferation without harming normal cells</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">912</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0919-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fs41419-018-0919-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30190481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BB3c3mtlCksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=912&issue=9&author=F.+O.+Enaneauthor=Y.+Saunthararajahauthor=M.+Korc&title=Differentiation+therapy+and+the+mechanisms+that+terminate+cancer+cell+proliferation+without+harming+normal+cells&doi=10.1038%2Fs41419-018-0919-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation therapy and the mechanisms that terminate cancer cell proliferation without harming normal cells</span></div><div class="casAuthors">Enane Francis O; Korc Murray; Saunthararajah Yogen; Saunthararajah Yogen; Korc Murray; Korc Murray</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">912</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemotherapeutic drugs have a common intent to activate apoptosis in tumor cells.  However, master regulators of apoptosis (e.g., p53, p16/CDKN2A) are frequently genetically inactivated in cancers, resulting in multidrug resistance.  An alternative, p53-independent method for terminating malignant proliferation is to engage terminal-differentiation.  Normally, the exponential proliferation of lineage-committed progenitors, coordinated by the master transcription factor (TF) MYC, is self-limited by forward-differentiation to terminal lineage-fates.  In cancers, however, this exponential proliferation is disengaged from terminal-differentiation.  The mechanisms underlying this decoupling are mostly unknown.  We performed a systematic review of published literature (January 2007-June 2018) to identify gene pathways linked to differentiation-failure in three treatment-recalcitrant cancers: hepatocellular carcinoma (HCC), ovarian cancer (OVC), and pancreatic ductal adenocarcinoma (PDAC).  We analyzed key gene alterations in various apoptosis, proliferation and differentiation pathways to determine whether it is possible to predict treatment outcomes and suggest novel therapies.  Poorly differentiated tumors were linked to poorer survival across histologies.  Our analyses suggested loss-of-function events to master TF drivers of lineage-fates and their cofactors as being linked to differentiation-failure: genomic data in TCGA and ICGC databases demonstrated frequent haploinsufficiency of lineage master TFs (e.g., GATA4/6) in poorly differentiated tumors; the coactivators that these TFs use to activate genes (e.g.  ARID1A, PBRM1) were also frequently inactivated by genetic mutation and/or deletion.  By contrast, corepressor components (e.g., DNMT1, EED, UHRF1, and BAZ1A/B), that oppose coactivators to repress or turn off genes, were frequently amplified instead, and the level of amplification was highest in poorly differentiated lesions.  This selection by neoplastic evolution towards unbalanced activity of transcriptional corepressors suggests these enzymes as candidate targets for inhibition aiming to re-engage forward-differentiation.  This notion is supported by both pre-clinical and clinical trial literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMrdazT79u5suCWE5mhx1RfW6udTcc2eZ-K_RNsKw7Sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3mtlCksw%253D%253D&md5=859bac67bb3bac116dc150979fa94f99</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0919-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0919-9%26sid%3Dliteratum%253Aachs%26aulast%3DEnane%26aufirst%3DF.%2BO.%26aulast%3DSaunthararajah%26aufirst%3DY.%26aulast%3DKorc%26aufirst%3DM.%26atitle%3DDifferentiation%2520therapy%2520and%2520the%2520mechanisms%2520that%2520terminate%2520cancer%2520cell%2520proliferation%2520without%2520harming%2520normal%2520cells%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D912%26doi%3D10.1038%2Fs41419-018-0919-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">de The, H.</span></span> <span> </span><span class="NLM_article-title">Differentiation therapy revisited</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fnrc.2017.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=29192213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=117-127&issue=2&author=H.+de+The&title=Differentiation+therapy+revisited&doi=10.1038%2Fnrc.2017.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation therapy revisited</span></div><div class="casAuthors">de The, Hugues</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">117-127</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The concept of differentiation therapy emerged from the fact that hormones or cytokines may promote differentiation ex vivo, thereby irreversibly changing the phenotype of cancer cells.  Its hallmark success has been the treatment of acute promyelocytic leukemia (APL), a condition that is now highly curable by the combination of retinoic acid (RA) and arsenic.  Recently, drugs that trigger differentiation in a variety of primary tumor cells have been identified, suggesting that they are clin. useful.  This Opinion article analyses the basis for the clin. successes of RA or arsenic in APL by assessing the resp. roles of terminal maturation and loss of self-renewal.  By reviewing other successful examples of drug-induced tumor cell differentiation, novel approaches to transform differentiating drugs into more efficient therapies are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr76NekKL66CrVg90H21EOLACvtfcHk0ljI7A_hEzxbdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtb%252FF&md5=f30632a18f99479ca8910bbe66c3f607</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.103%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DDifferentiation%2520therapy%2520revisited%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2018%26volume%3D18%26issue%3D2%26spage%3D117%26epage%3D127%26doi%3D10.1038%2Fnrc.2017.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoe, T.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of action and resistance to all-trans retinoic acid (ATRA) and arsenic trioxide (As<sub>2</sub>O<sub>3</sub>) in acute promyelocytic leukemia</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1007/s12185-013-1354-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1007%2Fs12185-013-1354-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=23670176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXps1yntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=717-725&issue=6&author=A.+Tomitaauthor=H.+Kiyoiauthor=T.+Naoe&title=Mechanisms+of+action+and+resistance+to+all-trans+retinoic+acid+%28ATRA%29+and+arsenic+trioxide+%28As2O3%29+in+acute+promyelocytic+leukemia&doi=10.1007%2Fs12185-013-1354-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of action and resistance to all-trans retinoic acid (ATRA) and arsenic trioxide (As2O3) in acute promyelocytic leukemia</span></div><div class="casAuthors">Tomita, Akihiro; Kiyoi, Hitoshi; Naoe, Tomoki</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">717-725</span>CODEN:
                <span class="NLM_cas:coden">IJHEEY</span>;
        ISSN:<span class="NLM_cas:issn">0925-5710</span>.
    
            (<span class="NLM_cas:orgname">Springer Japan</span>)
        </div><div class="casAbstract">A review.  Since the introduction of all-trans retinoic acid (ATRA) and arsenic trioxide (As2O3) for the treatment of acute promyelocytic leukemia (APL), the overall survival rate has improved dramatically.  However, relapse/refractory patients showing resistance to ATRA and/or As2O3 are recognized as a clin. significant problem.  Genetic mutations resulting in amino acid substitution in the retinoic acid receptor alpha (RARα) ligand binding domain (LBD) and the PML-B2 domain of PML-RARα, resp., have been reported as mol. mechanisms underlying resistance to ATRA and As2O3.  In the LBD mutation, ATRA binding with LBD is generally impaired, and ligand-dependent co-repressor dissocn. and degrdn. of PML-RARα by the proteasome pathway, leading to cell differentiation, are inhibited.  The PML-B2 mutation interferes with the direct binding of As2O3 with PML-B2, and PML-RARα SUMOylation with As2O3 followed by multimerization and degrdn. is impaired.  To overcome ATRA resistance, utilization of As2O3 provides a preferable outcome, and recently, a synthetic retinoid Am80, which has a higher binding affinity with PML-RARα than ATRA, has been tested in the clin. setting.  However, no strategy attempted to date has been successful in overcoming As2O3 resistance.  Detailed genomic analyses using patient samples harvested repeatedly may help in predicting the prognosis, selecting the effective targeting drugs, and designing new sophisticated strategies for the treatment of APL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRi89xz-8FzbVg90H21EOLACvtfcHk0ljI7A_hEzxbdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXps1yntLs%253D&md5=a762e823f1035ec451046d4e8de7159e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs12185-013-1354-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-013-1354-4%26sid%3Dliteratum%253Aachs%26aulast%3DTomita%26aufirst%3DA.%26aulast%3DKiyoi%26aufirst%3DH.%26aulast%3DNaoe%26aufirst%3DT.%26atitle%3DMechanisms%2520of%2520action%2520and%2520resistance%2520to%2520all-trans%2520retinoic%2520acid%2520%2528ATRA%2529%2520and%2520arsenic%2520trioxide%2520%2528As2O3%2529%2520in%2520acute%2520promyelocytic%2520leukemia%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2013%26volume%3D97%26issue%3D6%26spage%3D717%26epage%3D725%26doi%3D10.1007%2Fs12185-013-1354-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovici-Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salituro, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianchetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penard-Lacronique, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quivoron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saada, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Botton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaharski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of AG-120 (Ivosidenib): A first-in-class mutant IDH1 inhibitor for the treatment of IDH1 mutant cancers</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00421</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00421" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=300-305&issue=4&author=J.+Popovici-Mullerauthor=R.+M.+Lemieuxauthor=E.+Artinauthor=J.+O.+Saundersauthor=F.+G.+Salituroauthor=J.+Travinsauthor=G.+Cianchettaauthor=Z.+Caiauthor=D.+Zhouauthor=D.+Cuiauthor=P.+Chenauthor=K.+Straleyauthor=E.+Tobinauthor=F.+Wangauthor=M.+D.+Davidauthor=V.+Penard-Lacroniqueauthor=C.+Quivoronauthor=V.+Saadaauthor=S.+de+Bottonauthor=S.+Grossauthor=L.+Dangauthor=H.+Yangauthor=L.+Utleyauthor=Y.+Chenauthor=H.+Kimauthor=S.+Jinauthor=Z.+Guauthor=G.+Yaoauthor=Z.+Luoauthor=X.+Lvauthor=C.+Fangauthor=L.+Yanauthor=A.+Olaharskiauthor=L.+Silvermanauthor=S.+Billerauthor=S.+M.+Suauthor=K.+Yen&title=Discovery+of+AG-120+%28Ivosidenib%29%3A+A+first-in-class+mutant+IDH1+inhibitor+for+the+treatment+of+IDH1+mutant+cancers&doi=10.1021%2Facsmedchemlett.7b00421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of AG-120 (Ivosidenib): A First-in-Class Mutant IDH1 Inhibitor for the Treatment of IDH1 Mutant Cancers</span></div><div class="casAuthors">Popovici-Muller, Janeta; Lemieux, Rene M.; Artin, Erin; Saunders, Jeffrey O.; Salituro, Francesco G.; Travins, Jeremy; Cianchetta, Giovanni; Cai, Zhenwei; Zhou, Ding; Cui, Dawei; Chen, Ping; Straley, Kimberly; Tobin, Erica; Wang, Fang; David, Muriel D.; Penard-Lacronique, Virginie; Quivoron, Cyril; Saada, Veronique; de Botton, Stephane; Gross, Stefan; Dang, Lenny; Yang, Hua; Utley, Luke; Chen, Yue; Kim, Hyeryun; Jin, Shengfang; Gu, Zhiwei; Yao, Gui; Luo, Zhiyong; Lv, Xiaobing; Fang, Cheng; Yan, Liping; Olaharski, Andrew; Silverman, Lee; Biller, Scott; Su, Shin-San M.; Yen, Katharine</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">300-305</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Somatic point mutations at a key arginine residue (R132) within the active site of the metabolic enzyme isocitrate dehydrogenase 1 (IDH1) confer a novel gain of function in cancer cells, resulting in the prodn. of D-2-hydroxyglutarate (2-HG), an oncometabolite.  Elevated 2-HG levels are implicated in epigenetic alterations and impaired cellular differentiation.  IDH1 mutations have been described in an array of hematol. malignancies and solid tumors.  Here, we report the discovery of AG-120 (ivosidenib), an inhibitor of the IDH1 mutant enzyme that exhibits profound 2-HG lowering in tumor models and the ability to effect differentiation of primary patient AML samples ex vivo.  Preliminary data from phase 1 clin. trials enrolling patients with cancers harboring an IDH1 mutation indicate that AG-120 has an acceptable safety profile and clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmTLcR37BiPrVg90H21EOLACvtfcHk0lhQGXMgzZJing"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyrurw%253D&md5=730ac2faa6844e4abe3fd9a360d74d29</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00421%26sid%3Dliteratum%253Aachs%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DLemieux%26aufirst%3DR.%2BM.%26aulast%3DArtin%26aufirst%3DE.%26aulast%3DSaunders%26aufirst%3DJ.%2BO.%26aulast%3DSalituro%26aufirst%3DF.%2BG.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DCianchetta%26aufirst%3DG.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DStraley%26aufirst%3DK.%26aulast%3DTobin%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DDavid%26aufirst%3DM.%2BD.%26aulast%3DPenard-Lacronique%26aufirst%3DV.%26aulast%3DQuivoron%26aufirst%3DC.%26aulast%3DSaada%26aufirst%3DV.%26aulast%3Dde%2BBotton%26aufirst%3DS.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DUtley%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DOlaharski%26aufirst%3DA.%26aulast%3DSilverman%26aufirst%3DL.%26aulast%3DBiller%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DS.%2BM.%26aulast%3DYen%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520AG-120%2520%2528Ivosidenib%2529%253A%2520A%2520first-in-class%2520mutant%2520IDH1%2520inhibitor%2520for%2520the%2520treatment%2520of%2520IDH1%2520mutant%2520cancers%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D4%26spage%3D300%26epage%3D305%26doi%3D10.1021%2Facsmedchemlett.7b00421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kfoury, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercier, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schajnovitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolliday, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxcy, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobert, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szekely, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scadden, D. T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dihydroorotate dehydrogenase overcomes differentiation blockade in acute myeloid leukemia</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.08.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.cell.2016.08.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27641501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFamt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=171-186&issue=1&author=D.+B.+Sykesauthor=Y.+S.+Kfouryauthor=F.+E.+Mercierauthor=M.+J.+Wawerauthor=J.+M.+Lawauthor=M.+K.+Haynesauthor=T.+A.+Lewisauthor=A.+Schajnovitzauthor=E.+Jainauthor=D.+Leeauthor=H.+Meyerauthor=K.+A.+Pierceauthor=N.+J.+Tollidayauthor=A.+Wallerauthor=S.+J.+Ferraraauthor=A.+L.+Eheimauthor=D.+Stoeckigtauthor=K.+L.+Maxcyauthor=J.+M.+Cobertauthor=J.+Bachandauthor=B.+A.+Szekelyauthor=S.+Mukherjeeauthor=L.+A.+Sklarauthor=J.+D.+Kotzauthor=C.+B.+Clishauthor=R.+I.+Sadreyevauthor=P.+A.+Clemonsauthor=A.+Janzerauthor=S.+L.+Schreiberauthor=D.+T.+Scadden&title=Inhibition+of+dihydroorotate+dehydrogenase+overcomes+differentiation+blockade+in+acute+myeloid+leukemia&doi=10.1016%2Fj.cell.2016.08.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Dihydroorotate Dehydrogenase Overcomes Differentiation Blockade in Acute Myeloid Leukemia</span></div><div class="casAuthors">Sykes, David B.; Kfoury, Youmna S.; Mercier, Francois E.; Wawer, Mathias J.; Law, Jason M.; Haynes, Mark K.; Lewis, Timothy A.; Schajnovitz, Amir; Jain, Esha; Lee, Dongjun; Meyer, Hanna; Pierce, Kerry A.; Tolliday, Nicola J.; Waller, Anna; Ferrara, Steven J.; Eheim, Ashley L.; Stoeckigt, Detlef; Maxcy, Katrina L.; Cobert, Julien M.; Bachand, Jacqueline; Szekely, Brian A.; Mukherjee, Siddhartha; Sklar, Larry A.; Kotz, Joanne D.; Clish, Clary B.; Sadreyev, Ruslan I.; Clemons, Paul A.; Janzer, Andreas; Schreiber, Stuart L.; Scadden, David T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-186.e15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">While acute myeloid leukemia (AML) comprises many disparate genetic subtypes, one shared hallmark is the arrest of leukemic myeloblasts at an immature and self-renewing stage of development.  Therapies that overcome differentiation arrest represent a powerful treatment strategy.  The authors leveraged the observation that the majority of AML, despite their genetically heterogeneity, share in the expression of HoxA9, a gene normally downregulated during myeloid differentiation.  Using a conditional HoxA9 model system, the authors performed a high-throughput phenotypic screen and defined compds. that overcame differentiation blockade.  Target identification led to the unanticipated discovery that inhibition of the enzyme dihydroorotate dehydrogenase (DHODH) enables myeloid differentiation in human and mouse AML models.  In vivo, DHODH inhibitors reduced leukemic cell burden, decreased levels of leukemia-initiating cells, and improved survival.  These data demonstrate the role of DHODH as a metabolic regulator of differentiation and point to its inhibition as a strategy for overcoming differentiation blockade in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5JmGddhG2CbVg90H21EOLACvtfcHk0lhQGXMgzZJing"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFamt7nL&md5=19a4971c5a5c0a981ca39a1a6e31a20d</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.08.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.08.057%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DKfoury%26aufirst%3DY.%2BS.%26aulast%3DMercier%26aufirst%3DF.%2BE.%26aulast%3DWawer%26aufirst%3DM.%2BJ.%26aulast%3DLaw%26aufirst%3DJ.%2BM.%26aulast%3DHaynes%26aufirst%3DM.%2BK.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DSchajnovitz%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DPierce%26aufirst%3DK.%2BA.%26aulast%3DTolliday%26aufirst%3DN.%2BJ.%26aulast%3DWaller%26aufirst%3DA.%26aulast%3DFerrara%26aufirst%3DS.%2BJ.%26aulast%3DEheim%26aufirst%3DA.%2BL.%26aulast%3DStoeckigt%26aufirst%3DD.%26aulast%3DMaxcy%26aufirst%3DK.%2BL.%26aulast%3DCobert%26aufirst%3DJ.%2BM.%26aulast%3DBachand%26aufirst%3DJ.%26aulast%3DSzekely%26aufirst%3DB.%2BA.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DScadden%26aufirst%3DD.%2BT.%26atitle%3DInhibition%2520of%2520dihydroorotate%2520dehydrogenase%2520overcomes%2520differentiation%2520blockade%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DCell%26date%3D2016%26volume%3D167%26issue%3D1%26spage%3D171%26epage%3D186%26doi%3D10.1016%2Fj.cell.2016.08.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Christian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gradl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejeune, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzezinka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesche, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scadden, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span> <span> </span><span class="NLM_article-title">The novel dihydroorotate dehydrogenase (DHODH) inhibitor BAY 2402234 triggers differentiation and is effective in the treatment of myeloid malignancies</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2403</span>– <span class="NLM_lpage">2415</span>, <span class="refDoi"> DOI: 10.1038/s41375-019-0461-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fs41375-019-0461-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30940908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyqsr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=2403-2415&issue=10&author=S.+Christianauthor=C.+Merzauthor=L.+Evansauthor=S.+Gradlauthor=H.+Seidelauthor=A.+Fribergauthor=A.+Eheimauthor=P.+Lejeuneauthor=K.+Brzezinkaauthor=K.+Zimmermannauthor=S.+Ferraraauthor=H.+Meyerauthor=R.+Lescheauthor=D.+Stoeckigtauthor=M.+Bauserauthor=A.+Haegebarthauthor=D.+B.+Sykesauthor=D.+T.+Scaddenauthor=J.+A.+Losmanauthor=A.+Janzer&title=The+novel+dihydroorotate+dehydrogenase+%28DHODH%29+inhibitor+BAY+2402234+triggers+differentiation+and+is+effective+in+the+treatment+of+myeloid+malignancies&doi=10.1038%2Fs41375-019-0461-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">The novel dihydroorotate dehydrogenase (DHODH) inhibitor BAY 2402234 triggers differentiation and is effective in the treatment of myeloid malignancies</span></div><div class="casAuthors">Christian, Sven; Merz, Claudia; Evans, Laura; Gradl, Stefan; Seidel, Henrik; Friberg, Anders; Eheim, Ashley; Lejeune, Pascale; Brzezinka, Krzysztof; Zimmermann, Katja; Ferrara, Steven; Meyer, Hanna; Lesche, Ralf; Stoeckigt, Detlef; Bauser, Marcus; Haegebarth, Andrea; Sykes, David B.; Scadden, David T.; Losman, Julie-Aurore; Janzer, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2403-2415</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a devastating disease, with the majority of patients dying within a year of diagnosis.  For patients with relapsed/refractory AML, the prognosis is particularly poor with currently available treatments.  Although genetically heterogeneous, AML subtypes share a common differentiation arrest at hematopoietic progenitor stages.  Overcoming this differentiation arrest has the potential to improve the long-term survival of patients, as is the case in acute promyelocytic leukemia (APL), which is characterized by a chromosomal translocation involving the retinoic acid receptor alpha gene.  Treatment of APL with all-trans retinoic acid (ATRA) induces terminal differentiation and apoptosis of leukemic promyelocytes, resulting in cure rates of over 80%.  Unfortunately, similarly efficacious differentiation therapies have, to date, been lacking outside of APL.  Inhibition of dihydroorotate dehydrogenase (DHODH), a key enzyme in the de novo pyrimidine synthesis pathway, was recently reported to induce differentiation of diverse AML subtypes.  In this report we describe the discovery and characterization of BAY 2402234 - a novel, potent, selective and orally bioavailable DHODH inhibitor that shows monotherapy efficacy and differentiation induction across multiple AML subtypes.  Herein, we present the preclin. data that led to initiation of a phase I evaluation of this inhibitor in myeloid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpESpixEevQf7Vg90H21EOLACvtfcHk0lhWZp-bXY8uyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyqsr7J&md5=30e0ac086fb33617b5a4ecbbaa36d258</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fs41375-019-0461-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-019-0461-5%26sid%3Dliteratum%253Aachs%26aulast%3DChristian%26aufirst%3DS.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DEvans%26aufirst%3DL.%26aulast%3DGradl%26aufirst%3DS.%26aulast%3DSeidel%26aufirst%3DH.%26aulast%3DFriberg%26aufirst%3DA.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DLejeune%26aufirst%3DP.%26aulast%3DBrzezinka%26aufirst%3DK.%26aulast%3DZimmermann%26aufirst%3DK.%26aulast%3DFerrara%26aufirst%3DS.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DLesche%26aufirst%3DR.%26aulast%3DStoeckigt%26aufirst%3DD.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DScadden%26aufirst%3DD.%2BT.%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DJanzer%26aufirst%3DA.%26atitle%3DThe%2520novel%2520dihydroorotate%2520dehydrogenase%2520%2528DHODH%2529%2520inhibitor%2520BAY%25202402234%2520triggers%2520differentiation%2520and%2520is%2520effective%2520in%2520the%2520treatment%2520of%2520myeloid%2520malignancies%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26issue%3D10%26spage%3D2403%26epage%3D2415%26doi%3D10.1038%2Fs41375-019-0461-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span> <span> </span><span class="NLM_article-title">Differentiation-inducing therapy for solid tumors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.2174/138161206775201947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.2174%2F138161206775201947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=16454751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlCqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=379-385&issue=3&author=H.+Kawamataauthor=M.+Tachibanaauthor=T.+Fujimoriauthor=Y.+Imai&title=Differentiation-inducing+therapy+for+solid+tumors&doi=10.2174%2F138161206775201947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation-inducing therapy for solid tumors</span></div><div class="casAuthors">Kawamata, Hitoshi; Tachibana, Masatsugu; Fujimori, Takahiro; Imai, Yutaka</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-385</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Treating malignant tumor through the induction of cell differentiation has been an attractive concept, but clin. development of differentiation-inducing agents to treat malignant tumor, esp. for solid tumors has been limited to date.  Nerve growth factor, all trans retinoic acid, DMSO, active form vitamin D3, peroxisome proliferator-activated receptor γ, 12-0-tetradecanoylphorbol 13-acetate, hexamethylene-bis-acetamide, transforming growth factor-β, butyric acid, cAMP, and vesnarinone are known to have a differentiation-inducing capability on solid tumors in vitro and/or in vivo.  Moreover some of the differentiation-inducing agents have been used for treating patients with solid tumor, but the therapeutic effect of the differentiation-inducing agents on solid tumor is not strong when compared with that of conventional chemotherapeutic agents.  However, because most of the differentiation-inducing agents can potentiate the effect of conventional chemotherapy or radiation therapy, combination of differentiation-inducing therapy with conventional chemotherapy or radiation therapy might be used as a second- or third-line therapy in patients with advanced cancer.  Furthermore, anal. of the mol. mechanisms of the tumor differentiation therapy might provide selective and targeted mols. for novel cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuQmfkM8acUbVg90H21EOLACvtfcHk0lhWZp-bXY8uyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlCqtw%253D%253D&md5=0447b91f783b47bed4cbff034afa545e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F138161206775201947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161206775201947%26sid%3Dliteratum%253Aachs%26aulast%3DKawamata%26aufirst%3DH.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DFujimori%26aufirst%3DT.%26aulast%3DImai%26aufirst%3DY.%26atitle%3DDifferentiation-inducing%2520therapy%2520for%2520solid%2520tumors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2006%26volume%3D12%26issue%3D3%26spage%3D379%26epage%3D385%26doi%3D10.2174%2F138161206775201947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&issue=7&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lhWZp-bXY8uyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26issue%3D7%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human histone lysine-specific demethylase 1 (LSD1)</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">13956</span>– <span class="NLM_lpage">13961</span>, <span class="refDoi"> DOI: 10.1073/pnas.0606381103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1073%2Fpnas.0606381103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=16956976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCmtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=13956-13961&issue=38&author=Y.+Chenauthor=Y.+Yangauthor=F.+Wangauthor=K.+Wanauthor=K.+Yamaneauthor=Y.+Zhangauthor=M.+Lei&title=Crystal+structure+of+human+histone+lysine-specific+demethylase+1+%28LSD1%29&doi=10.1073%2Fpnas.0606381103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human histone lysine-specific demethylase 1 (LSD1)</span></div><div class="casAuthors">Chen, Yong; Yang, Yuting; Wang, Feng; Wan, Ke; Yamane, Kenichi; Zhang, Yi; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">13956-13961</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lysine-specific histone demethylase 1 (LSD1) was recently identified as the 1st histone demethylase that specifically demethylates monomethylated and dimethylated histone H3 at Lys-4.  It is a component of the CoREST and other corepressor complexes and plays an important role in silencing neuronal-specific genes in non-neuronal cells, but the mol. mechanisms of its action remain unclear.  Here, the 2.8-Å-resoln. crystal structure of human LSD1 is reported.  It revealed that LSD1 defines a new subfamily of FAD-dependent oxidases.  The active center of LSD1 was characterized by a remarkable 1245-Å3 substrate-binding cavity with a highly neg. electrostatic potential.  Although the protein core of LSD1 resembled other flavoenzymes, its enzymic activity and functions required 2 addnl. structural modules: an N-terminal SWIRM domain important for protein stability and a large insertion in the catalytic domain indispensable both for demethylase activity and the interaction with CoREST.  These results provide a framework for further probing of the catalytic mechanism and the functional roles of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Tqfaz7oaE7Vg90H21EOLACvtfcHk0ljC3Yf1XaSEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCmtbfK&md5=13da0bfe3f99d217eae73924dc3f6fcf</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0606381103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0606381103%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DCrystal%2520structure%2520of%2520human%2520histone%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26issue%3D38%26spage%3D13956%26epage%3D13961%26doi%3D10.1073%2Fpnas.0606381103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span> <span> </span><span class="NLM_article-title">LSD1: biologic roles and therapeutic targeting</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1103</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.2217%2Fepi-2016-0009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27479862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlCisr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1103-1116&issue=8&author=A.+Maiques-Diazauthor=T.+C.+Somervaille&title=LSD1%3A+biologic+roles+and+therapeutic+targeting&doi=10.2217%2Fepi-2016-0009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1: biologic roles and therapeutic targeting</span></div><div class="casAuthors">Maiques-Diaz, Alba; Somervaille, Tim CP</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1103-1116</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">LSD1 (KDM1A; BHC110; AOF2) was the first protein reported to exhibit histone demethylase activity and has since been shown to have multiple essential roles in mammalian biol.  Given its enzymic activity and its high-level expression in many human malignancies, a significant recent focus has been the development of pharmacol. inhibitors.  Here we summarize structural and biochem. knowledge of this important epigenetic regulator, with a particular emphasis on the functional and preclin. studies in oncol. that have provided justification for the evaluation of tranylcypromine deriv. LSD1 inhibitors in early phase clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroYKq_-3Ts2bVg90H21EOLACvtfcHk0ljC3Yf1XaSEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlCisr3P&md5=9bd062613a49f8527ee2b7f8e119e6ac</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0009%26sid%3Dliteratum%253Aachs%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%253A%2520biologic%2520roles%2520and%2520therapeutic%2520targeting%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26issue%3D8%26spage%3D1103%26epage%3D1116%26doi%3D10.2217%2Fepi-2016-0009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): A computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e25444</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25444&issue=9&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+A+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0ljC3Yf1XaSEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520A%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26issue%3D9%26spage%3De25444%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrello, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarelli, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellore, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1677</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&issue=8&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lgtiJTqv-hghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">LSD1 inhibitor T-3775440 inhibits SCLC cell proliferation by disrupting LSD1 interactions with SNAG domain proteins INSM1 and GFI1B</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">4652</span>– <span class="NLM_lpage">4662</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-3502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F0008-5472.CAN-16-3502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=28667074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=4652-4662&issue=17&author=S.+Takagiauthor=Y.+Ishikawaauthor=A.+Mizutaniauthor=S.+Iwasakiauthor=S.+Matsumotoauthor=Y.+Kamadaauthor=T.+Nomuraauthor=K.+Nakamura&title=LSD1+inhibitor+T-3775440+inhibits+SCLC+cell+proliferation+by+disrupting+LSD1+interactions+with+SNAG+domain+proteins+INSM1+and+GFI1B&doi=10.1158%2F0008-5472.CAN-16-3502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 Inhibitor T-3775440 Inhibits SCLC Cell Proliferation by Disrupting LSD1 Interactions with SNAG Domain Proteins INSM1 and GFI1B</span></div><div class="casAuthors">Takagi, Shinji; Ishikawa, Yoshinori; Mizutani, Akio; Iwasaki, Shinji; Matsumoto, Satoru; Kamada, Yusuke; Nomura, Toshiyuki; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4652-4662</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">T-3775440 is an irreversible inhibitor of the chromatin demethylase LSD1, which exerts antiproliferative effects by disrupting the interaction between LSD1 and GFI1B, a SNAG domain transcription factor, inducing leukemia cell transdifferentiation.  Here, we describe the anticancer effects and mechanism of action of T-3775440 in small-cell lung cancer (SCLC).  T-3775440 inhibited proliferation of SCLC cells in vitro and retarded SCLC tumor growth in vivo.  T-3775440 disrupted the interaction between LSD1 and the transcriptional repressor INSM1, thereby inhibiting expression of neuroendocrine-assocd. genes, such as ASCL1.  INSM1 silencing phenocopied the effects of T-3775440 on gene expression and cell proliferation, consistent with the likelihood T-3775440 mediated its effects in SCLC by inhibiting INSM1.  T-3775440 also inhibited proliferation of an SCLC cell line that overexpressed GFI1B, rather than INSM1, by disrupting the interaction between LSD1 and GFI1B.  Taken together, our results argue that LSD1 plays an important role in neuroendocrine-assocd. transcription and cell proliferation of SCLC via interactions with the SNAG domain proteins INSM1 and GFI1B.  Targeting these crit. interactions with LSD1 inhibitors offers a novel rational strategy to therapeutically manage SCLC.  Cancer Res; 77(17); 4652-62. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqygHUXf_CKq7Vg90H21EOLACvtfcHk0lgtiJTqv-hghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7rM&md5=8cd3ee5ab304690ec57e83fda605d7c1</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-3502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-3502%26sid%3Dliteratum%253Aachs%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DA.%26aulast%3DIwasaki%26aufirst%3DS.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DLSD1%2520inhibitor%2520T-3775440%2520inhibits%2520SCLC%2520cell%2520proliferation%2520by%2520disrupting%2520LSD1%2520interactions%2520with%2520SNAG%2520domain%2520proteins%2520INSM1%2520and%2520GFI1B%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D17%26spage%3D4652%26epage%3D4662%26doi%3D10.1158%2F0008-5472.CAN-16-3502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairiki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hikichi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">A novel LSD1 inhibitor T-3775440 disrupts GFI1B-containing complex leading to transdifferentiation and impaired growth of AML cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F1535-7163.MCT-16-0471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27903753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=273-284&issue=2&author=Y.+Ishikawaauthor=K.+Gamoauthor=M.+Yabukiauthor=S.+Takagiauthor=K.+Toyoshimaauthor=K.+Nakayamaauthor=A.+Nakayamaauthor=M.+Morimotoauthor=H.+Miyashitaauthor=R.+Dairikiauthor=Y.+Hikichiauthor=N.+Tomitaauthor=D.+Tomitaauthor=S.+Imamuraauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Matsumotoauthor=R.+Haraauthor=T.+Nomuraauthor=K.+Tsuchidaauthor=K.+Nakamura&title=A+novel+LSD1+inhibitor+T-3775440+disrupts+GFI1B-containing+complex+leading+to+transdifferentiation+and+impaired+growth+of+AML+cells&doi=10.1158%2F1535-7163.MCT-16-0471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel LSD1 Inhibitor T-3775440 Disrupts GFI1B-Containing Complex Leading to Transdifferentiation and Impaired Growth of AML Cells</span></div><div class="casAuthors">Ishikawa, Yoshinori; Gamo, Kanae; Yabuki, Masato; Takagi, Shinji; Toyoshima, Kosei; Nakayama, Kazuhide; Nakayama, Akiko; Morimoto, Megumi; Miyashita, Hitoshi; Dairiki, Ryo; Hikichi, Yukiko; Tomita, Naoki; Tomita, Daisuke; Imamura, Shinichi; Iwatani, Misa; Kamada, Yusuke; Matsumoto, Satoru; Hara, Ryujiro; Nomura, Toshiyuki; Tsuchida, Ken; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-284</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Dysregulation of lysine (K)-specific demethylase 1A (LSD1), also known as KDM1A, has been implicated in the development of various cancers, including leukemia.  Here, we describe the antileukemic activity and mechanism of action of T-3775440, a novel irreversible LSD1 inhibitor.  Cell growth anal. of leukemia cell lines revealed that acute erythroid leukemia (AEL) and acute megakaryoblastic leukemia cells (AMKL) were highly sensitive to this compd.  T-3775440 treatment enforced transdifferentiation of erythroid/megakaryocytic lineages into granulomonocytic-like lineage cells.  Mechanistically, T-3775440 disrupted the interaction between LSD1 and growth factor-independent 1B (GFI1B), a transcription factor crit. for the differentiation processes of erythroid and megakaryocytic lineage cells.  Knockdown of LSD1 and GFI1B recapitulated T-3775440-induced transdifferentiation and cell growth suppression, highlighting the significance of LSD1-GFI1B axis inhibition with regard to the anti-AML effects of T-3775440.  Moreover, T-3775440 exhibited significant antitumor efficacy in AEL and AMKL xenograft models.  Our findings provide a rationale for evaluating LSD1 inhibitors as potential treatments and indicate a novel mechanism of action against AML, particularly AEL and AMKL.  Mol Cancer Ther; 16(2); 273-84. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomcxpsluqnf7Vg90H21EOLACvtfcHk0lgtiJTqv-hghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D&md5=2335343b4d5e7bbc197a1d5aec54e3af</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0471%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DGamo%26aufirst%3DK.%26aulast%3DYabuki%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DToyoshima%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DDairiki%26aufirst%3DR.%26aulast%3DHikichi%26aufirst%3DY.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DImamura%26aufirst%3DS.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DA%2520novel%2520LSD1%2520inhibitor%2520T-3775440%2520disrupts%2520GFI1B-containing%2520complex%2520leading%2520to%2520transdifferentiation%2520and%2520impaired%2520growth%2520of%2520AML%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26issue%3D2%26spage%3D273%26epage%3D284%26doi%3D10.1158%2F1535-7163.MCT-16-0471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebestreit, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popescu, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelent, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1038/nm.2661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&issue=4&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+Gollnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+Muller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia&doi=10.1038%2Fnm.2661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID γ (with interleukin-2 (IL-2) receptor γ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0livAKwEaKyozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DGollner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26issue%3D4%26spage%3D605%26epage%3D611%26doi%3D10.1038%2Fnm.2661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of the histone lysine demethylase KDM1A suppresses the growth of multiple acute myeloid leukemia subtypes</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1975</span>– <span class="NLM_lpage">1988</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F0008-5472.CAN-15-2333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=26837761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlSktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1975-1988&issue=7&author=J.+P.+McGrathauthor=K.+E.+Williamsonauthor=S.+Balasubramanianauthor=S.+Odateauthor=S.+Aroraauthor=C.+Hattonauthor=T.+M.+Edwardsauthor=T.+O%E2%80%99Brienauthor=S.+Magnusonauthor=D.+Stokoeauthor=D.+L.+Danielsauthor=B.+M.+Bryantauthor=P.+Trojer&title=Pharmacological+inhibition+of+the+histone+lysine+demethylase+KDM1A+suppresses+the+growth+of+multiple+acute+myeloid+leukemia+subtypes&doi=10.1158%2F0008-5472.CAN-15-2333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Inhibition of the Histone Lysine Demethylase KDM1A Suppresses the Growth of Multiple Acute Myeloid Leukemia Subtypes</span></div><div class="casAuthors">McGrath, John P.; Williamson, Kaylyn E.; Balasubramanian, Srividya; Odate, Shobu; Arora, Shilpi; Hatton, Charlie; Edwards, Thomas M.; O'Brien, Thomas; Magnuson, Steven; Stokoe, David; Daniels, Danette L.; Bryant, Barbara M.; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1975-1988</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (KDM1A) is a transcriptional coregulator that can function in both the activation and repression of gene expression, depending upon context.  KDM1A plays an important role in hematopoiesis and was identified as a dependency factor in leukemia stem cell populations.  Therefore, we investigated the consequences of inhibiting KDM1A in a panel of cell lines representing all acute myelogenous leukemia (AML) subtypes using selective, reversible and irreversible KDM1A small-mol. inhibitors.  Cell models of AML, CML, and T-ALL were potently affected by KDM1A inhibition, and cells bearing RUNX1-RUNX1T1 (AML1-ETO) translocations were esp. among the most sensitive.  RNAi-mediated silencing of KDM1A also effectively suppressed growth of RUNX1-RUNX1T1-contg. cell lines.  Furthermore, pharmacol. inhibition of KDM1A resulted in complete abrogation of tumor growth in an AML xenograft model harboring RUNX1-RUNX1T1 translocations.  We unexpectedly found that KDM1A-targeting compds. not only inhibited the catalytic activity of the enzyme, but evicted KDM1A from target genes.  Accordingly, compd.-mediated KDM1A eviction was assocd. with elevated levels of local histone H3 lysine 4 dimethylation, and increased target gene expression, which was further accompanied by cellular differentiation and induction of cell death.  Finally, our finding that KDM1A inhibitors effectively synergize with multiple conventional as well as candidate anti-AML agents affords a framework for potential future clin. application.  Cancer Res; 76(7); 1975-88. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2rcSAA1n6MrVg90H21EOLACvtfcHk0livAKwEaKyozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlSktbo%253D&md5=19efc6c725b7bf869749f90b2b2cec6e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2333%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%2BP.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DEdwards%26aufirst%3DT.%2BM.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DPharmacological%2520inhibition%2520of%2520the%2520histone%2520lysine%2520demethylase%2520KDM1A%2520suppresses%2520the%2520growth%2520of%2520multiple%2520acute%2520myeloid%2520leukemia%2520subtypes%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26issue%3D7%26spage%3D1975%26epage%3D1988%26doi%3D10.1158%2F0008-5472.CAN-15-2333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parimi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poropatich, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. J.</span></span> <span> </span><span class="NLM_article-title">Neuroendocrine differentiation of prostate cancer: a review</span>. <i>Am. J. Clin. Exp. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=25606573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvoslejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2014&pages=273-285&issue=4&author=V.+Parimiauthor=R.+Goyalauthor=K.+Poropatichauthor=X.+J.+Yang&title=Neuroendocrine+differentiation+of+prostate+cancer%3A+a+review"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroendocrine differentiation of prostate cancer: a review</span></div><div class="casAuthors">Parimi Vamsi; Goyal Rajen; Poropatich Kate; Yang Ximing J</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical and experimental urology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">273-85</span>
        ISSN:<span class="NLM_cas:issn">2330-1910</span>.
    </div><div class="casAbstract">Neuroendocrine cells are one of the epithelial populations in the prostate.  Neuroendocrine differentiation (NED) has been observed in prostate cancer.  In addition to small cell neuroendocrine carcinomas and carcinoid tumors of the prostate, prostatic adenocarcinomas may have NED.  The incidence and clinical relevance of NED in prostatic adenocarcinoma is not clearly understood because of conflicting results in the reported studies, and evaluation of NED is not routinely performed in clinical practice.  This review is an overall synthesis with an aim to develop a more comprehensive understanding and practical approach towards the current knowledge of neuroendocrine differentiation.  In this review we are stratifying these lesions into separate subtypes based on histologic parameters such as tumor morphology, neuroendocrine cell density and distribution and clinical parameters.  We also want to identify current controversies and confusing issues not totally resolved in this topic for further investigations.  Eventually a clearer understanding of this phenomenon and appropriate handling NED in prostate cancer will benefit clinical practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrIfVnOV1rD_tos8R3JUhXfW6udTcc2eYwexIuQwPG1rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvoslejtw%253D%253D&md5=d2113a1b08865d57de600f5cf2a2e17e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParimi%26aufirst%3DV.%26aulast%3DGoyal%26aufirst%3DR.%26aulast%3DPoropatich%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%2BJ.%26atitle%3DNeuroendocrine%2520differentiation%2520of%2520prostate%2520cancer%253A%2520a%2520review%26jtitle%3DAm.%2520J.%2520Clin.%2520Exp.%2520Urol.%26date%3D2014%26volume%3D2%26issue%3D4%26spage%3D273%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&issue=1&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0liDTqkg_o8Awg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26issue%3D1%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Magliulo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, S.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1A as a promising target in acute myeloid leukemia</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">255</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.3389%2Ffonc.2018.00255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30073149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ks1GqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=255&author=D.+Magliuloauthor=R.+Bernardiauthor=S.+Messina&title=Lysine-specific+demethylase+1A+as+a+promising+target+in+acute+myeloid+leukemia&doi=10.3389%2Ffonc.2018.00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Specific Demethylase 1A as a Promising Target in Acute Myeloid Leukemia</span></div><div class="casAuthors">Magliulo Daniela; Magliulo Daniela; Bernardi Rosa; Messina Samantha</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">255</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous hematopoietic malignancy characterized by the accumulation of incompletely differentiated progenitor cells (blasts) in the bone marrow and blood, and by suppression of normal hematopoiesis.  It has recently become apparent that the AML genome is characterized by recurrent mutations and dysregulations in epigenetic regulators.  These mutations frequently occur before the onset of full blown leukemia, at the pre-leukemic phase, and persist in residual disease that remains after therapeutic intervention, thus suggesting that targeting the AML epigenome may help to eradicate minimal residual disease and prevent relapse.  Within the AML epigenome, lysine-specific demethylase 1 A (LSD1) is a histone demethylase that is found frequently overexpressed, albeit not mutated, in AML.  LSD1 is a required constituent of critical transcription repressor complexes like CoREST and nucleosome remodeling and deacetylase (NuRD), and abrogation of LSD1 expression results in impaired self-renewal and proliferation, and increased differentiation and apoptosis in AML models and primary cells, particularly in AMLs with MLL- and AML1-rearrangements, or erythroid and megakaryoblastic differentiation block.  On this basis, a number of LSD1 inhibitors have been developed in the past decade, and few of them are currently being tested in clinical trials for patients with AML, along with other malignancies.  To date, the most promising application of this therapeutic strategy appears to be combination therapy of LSD1 inhibitors with all-trans retinoic acid (ATRA) to reactivate myeloid differentiation in cells that are not spontaneously susceptible to ATRA treatment.  In this review, we provide an overview of LSD1 function in normal hematopoiesis and leukemia, and of the current clinical application of LSD1 inhibitors for the treatment of patients with AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZvJytNRFkvDC850By6mxpfW6udTcc2eZF6T1OTqPrT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ks1GqtQ%253D%253D&md5=7aaddbc35e569064c3fa8da358299840</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00255%26sid%3Dliteratum%253Aachs%26aulast%3DMagliulo%26aufirst%3DD.%26aulast%3DBernardi%26aufirst%3DR.%26aulast%3DMessina%26aufirst%3DS.%26atitle%3DLysine-specific%2520demethylase%25201A%2520as%2520a%2520promising%2520target%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D255%26doi%3D10.3389%2Ffonc.2018.00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&issue=3&author=T.+Maesauthor=C.+Mascaroauthor=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0ljTRBlRul8B5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26issue%3D3%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brohl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubitt, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, D. R.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of lysine-specific demethylase 1 (LSD1) and histone deacetylase (HDAC) alone and in combination in Ewing sarcoma cell lines</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e0222228</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0222228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1371%2Fjournal.pone.0222228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=31550266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitl2qsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=e0222228&issue=9&author=D.+Welchauthor=E.+Kahenauthor=B.+Fridleyauthor=A.+S.+Brohlauthor=C.+L.+Cubittauthor=D.+R.+Reed&title=Small+molecule+inhibition+of+lysine-specific+demethylase+1+%28LSD1%29+and+histone+deacetylase+%28HDAC%29+alone+and+in+combination+in+Ewing+sarcoma+cell+lines&doi=10.1371%2Fjournal.pone.0222228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of lysine-specific demethylase 1 (LSD1) and histone deacetylase (HDAC) alone and in combination in Ewing sarcoma cell lines</span></div><div class="casAuthors">Welch, Darcy; Kahen, Elliot; Fridley, Brooke; Brohl, Andrew S.; Cubitt, Christopher L.; Reed, Damon R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e0222228</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ewing Sarcoma (ES) is characterized by recurrent translocations between EWSR1 and members of the ETS family of transcription factors.  The transcriptional activity of the fusion oncoprotein is dependent on interaction with the nucleosome remodeling and deactylase (NuRD) co-repressor complex.  While inhibitors of both histone deacetylase (HDAC) and lysine-specific demethylase-1 (LSD1) subunits of the NuRD complex demonstrate single agent activity in preclin. models, combination strategies have not been investigated.  We selected 7 clin. utilized chemotherapy agents, or active metabolites thereof, for experimentation: doxorubicin, cyclophosphamide, vincristine, etoposide and irinotecan as well as the HDAC inhibitor romidepsin and the reversible LSD1 inhibitor SP2509.  All agents were tested at clin. achievable concns. in 4 ES cell lines.  All possible 2 drug combinations were then tested for potential synergy.  Order of addn. of second-line conventional combination therapy agents was tested with the addn. of SP2509.  In two drug expts., synergy was obsd. with several combinations, including when SP2509 was paired with topoisomerase inhibitors or romidepsin.  Addn. of SP2509 after treatment with second-line combination therapy agents enhanced treatment effect.  Our findings suggest promising combination treatment strategies that utilize epigenetic agents in ES.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPvzCMFjMDIrVg90H21EOLACvtfcHk0ljTRBlRul8B5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitl2qsrrN&md5=fa61558bd91ef8bf7fff6c344ab8f7fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0222228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0222228%26sid%3Dliteratum%253Aachs%26aulast%3DWelch%26aufirst%3DD.%26aulast%3DKahen%26aufirst%3DE.%26aulast%3DFridley%26aufirst%3DB.%26aulast%3DBrohl%26aufirst%3DA.%2BS.%26aulast%3DCubitt%26aufirst%3DC.%2BL.%26aulast%3DReed%26aufirst%3DD.%2BR.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520histone%2520deacetylase%2520%2528HDAC%2529%2520alone%2520and%2520in%2520combination%2520in%2520Ewing%2520sarcoma%2520cell%2520lines%26jtitle%3DPLoS%2520One%26date%3D2019%26volume%3D14%26issue%3D9%26spage%3De0222228%26doi%3D10.1371%2Fjournal.pone.0222228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerenyi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luc, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e00633</span>, <span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0ljTRBlRul8B5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M. P.</span></span> <span> </span><span class="NLM_article-title">Generation of a set of simple, interpretable ADMET rules of thumb</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">817</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1021/jm701122q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701122q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=817-34&issue=4&author=M.+P.+Gleeson&title=Generation+of+a+set+of+simple%2C+interpretable+ADMET+rules+of+thumb&doi=10.1021%2Fjm701122q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Set of Simple, Interpretable ADMET Rules of Thumb</span></div><div class="casAuthors">Gleeson, M. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">817-834</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A set of simple, consistent structure-property guides have been detd. from an anal. of a no. of key ADMET assays run within GSK: soly., permeability, bioavailability, vol. of distribution, plasma protein binding, CNS penetration, brain tissue binding, P-gp efflux, hERG inhibition, and cytochrome P 450 1A2/2C9/2C19/2D6/3A4 inhibition.  The rules have been formulated using mol. properties that chemists intuitively know how to alter in a mol., namely, mol. wt., logP, and ionization state.  The rules supplement the more predictive black-box models available to us by clearly illustrating the key underlying trends, which are in line with reports in the literature.  It is clear from the analyses reported herein that almost all ADMET parameters deteriorate with either increasing mol. wt., logP, or both, with ionization state playing either a beneficial or detrimental affect depending on the parameter in question.  This study reemphasizes the need to focus on a lower mol. wt. and logP area of physicochem. property space to obtain improved ADMET parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PTdPHoD-c7Vg90H21EOLACvtfcHk0ljTRBlRul8B5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D&md5=42db5918ad8e9ea3ecf3a13846b3a434</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm701122q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701122q%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26atitle%3DGeneration%2520of%2520a%2520set%2520of%2520simple%252C%2520interpretable%2520ADMET%2520rules%2520of%2520thumb%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D4%26spage%3D817%26epage%3D34%26doi%3D10.1021%2Fjm701122q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Romano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filvaroff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, Z.</span></span> <span> </span><span class="NLM_article-title">Phase I study of CC-90011 in patients with advanced solid tumors and relapsed/refractory non-Hodgkin lymphoma (R/R NHL)</span>. <i>Ann. Oncol</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">i4</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1093%2Fannonc%2Fmdz029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=i4&issue=Suppl.+1&author=A.+Hollebecqueauthor=J.+S.+de+Bonoauthor=R.+Plummerauthor=N.+Isambertauthor=P.+Martin-Romanoauthor=E.+Baudinauthor=S.+Moraauthor=E.+Filvaroffauthor=M.+Lambaauthor=Z.+Nikolova&title=Phase+I+study+of+CC-90011+in+patients+with+advanced+solid+tumors+and+relapsed%2Frefractory+non-Hodgkin+lymphoma+%28R%2FR+NHL%29&doi=10.1093%2Fannonc%2Fmdz029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz029%26sid%3Dliteratum%253Aachs%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DPlummer%26aufirst%3DR.%26aulast%3DIsambert%26aufirst%3DN.%26aulast%3DMartin-Romano%26aufirst%3DP.%26aulast%3DBaudin%26aufirst%3DE.%26aulast%3DMora%26aufirst%3DS.%26aulast%3DFilvaroff%26aufirst%3DE.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DNikolova%26aufirst%3DZ.%26atitle%3DPhase%2520I%2520study%2520of%2520CC-90011%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520and%2520relapsed%252Frefractory%2520non-Hodgkin%2520lymphoma%2520%2528R%252FR%2520NHL%2529%26jtitle%3DAnn.%2520Oncol%26date%3D2019%26volume%3D30%26issue%3DSuppl.%25201%26spage%3Di4%26doi%3D10.1093%2Fannonc%2Fmdz029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4a','cit4b'],'ref5':['cit5'],'ref6':['cit6a','cit6b'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9a','cit9b','cit9c'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14a','cit14b'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Tong  Yu</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of dual inhibitors targeting crosstalk between histone epigenetic modulators for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113588. <a href="https://doi.org/10.1016/j.ejmech.2021.113588" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113588%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252Bdual%252Binhibitors%252Btargeting%252Bcrosstalk%252Bbetween%252Bhistone%252Bepigenetic%252Bmodulators%252Bfor%252Bcancer%252Btherapy%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D222%26spage%3D113588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Chen</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>, <span class="hlFld-ContribAuthor ">Xing-Han  Yun</span>, <span class="hlFld-ContribAuthor ">Tian-Hui  Hao</span>, <span class="hlFld-ContribAuthor ">Sheng-Jiao  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-component cascade reaction of 3-formylchromones: highly selective synthesis of 4,5-dihydro-[4,5′-bipyrimidin]-6(1
              H
              )-one derivatives. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2021,</strong> <em>57 </em>
                                    (62)
                                     , 7657-7660. <a href="https://doi.org/10.1039/D1CC02437J" title="DOI URL">https://doi.org/10.1039/D1CC02437J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CC02437J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CC02437J%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DMulti-component%252Bcascade%252Breaction%252Bof%252B3-formylchromones%25253A%252Bhighly%252Bselective%252Bsynthesis%252Bof%252B4%25252C5-dihydro-%25255B4%25252C5%2525E2%252580%2525B2-bipyrimidin%25255D-6%2525281%252BH%252B%252529-one%252Bderivatives%26aulast%3DChen%26aufirst%3DLi%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D62%26spage%3D7657%26epage%3D7660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gaya  Punnia-Moorthy</span>, <span class="hlFld-ContribAuthor ">Peter  Hersey</span>, <span class="hlFld-ContribAuthor ">Abdullah Al  Emran</span>, <span class="hlFld-ContribAuthor ">Jessamy  Tiffen</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Demethylases: Promising Drug Targets in Melanoma and Other Cancers. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fgene.2021.680633" title="DOI URL">https://doi.org/10.3389/fgene.2021.680633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2021.680633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2021.680633%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DLysine%252BDemethylases%25253A%252BPromising%252BDrug%252BTargets%252Bin%252BMelanoma%252Band%252BOther%252BCancers%26aulast%3DPunnia-Moorthy%26aufirst%3DGaya%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ming-Jie  Huang</span>, <span class="hlFld-ContribAuthor ">Jia-Wen  Guo</span>, <span class="hlFld-ContribAuthor ">Yun-Dong  Fu</span>, <span class="hlFld-ContribAuthor ">Ya-Zhen  You</span>, <span class="hlFld-ContribAuthor ">Wen-Yu  Xu</span>, <span class="hlFld-ContribAuthor ">Ting-Yu  Song</span>, <span class="hlFld-ContribAuthor ">Ran  Li</span>, <span class="hlFld-ContribAuthor ">Zi-Tong  Chen</span>, <span class="hlFld-ContribAuthor ">Li-Hua  Huang</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new tranylcypromine derivatives as highly potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>41 </em>, 127993. <a href="https://doi.org/10.1016/j.bmcl.2021.127993" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127993</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127993%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnew%252Btranylcypromine%252Bderivatives%252Bas%252Bhighly%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DHuang%26aufirst%3DMing-Jie%26date%3D2021%26volume%3D41%26spage%3D127993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Wang</span>, <span class="hlFld-ContribAuthor ">Qiang  Xie</span>, <span class="hlFld-ContribAuthor ">Huidan  Tan</span>, <span class="hlFld-ContribAuthor ">Minru  Liao</span>, <span class="hlFld-ContribAuthor ">Shiou  Zhu</span>, <span class="hlFld-ContribAuthor ">Ling-Li  Zheng</span>, <span class="hlFld-ContribAuthor ">Haixia  Huang</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting cancer epigenetic pathways with small-molecule compounds: Therapeutic efficacy and combination therapies. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2021,</strong> <em>13 </em>, 105702. <a href="https://doi.org/10.1016/j.phrs.2021.105702" title="DOI URL">https://doi.org/10.1016/j.phrs.2021.105702</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2021.105702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2021.105702%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DTargeting%252Bcancer%252Bepigenetic%252Bpathways%252Bwith%252Bsmall-molecule%252Bcompounds%25253A%252BTherapeutic%252Befficacy%252Band%252Bcombination%252Btherapies%26aulast%3DWang%26aufirst%3DYi%26date%3D2021%26volume%3D13%26spage%3D105702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barry M.  Zee</span>, <span class="hlFld-ContribAuthor ">Kamrine E.  Poels</span>, <span class="hlFld-ContribAuthor ">Cong-Hui  Yao</span>, <span class="hlFld-ContribAuthor ">Kimihito C.  Kawabata</span>, <span class="hlFld-ContribAuthor ">Gongwei  Wu</span>, <span class="hlFld-ContribAuthor ">Cihangir  Duy</span>, <span class="hlFld-ContribAuthor ">William D.  Jacobus</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Senior</span>, <span class="hlFld-ContribAuthor ">Jennifer E.  Endress</span>, <span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Scott B.  Lovitch</span>, <span class="hlFld-ContribAuthor ">Jiexian  Ma</span>, <span class="hlFld-ContribAuthor ">Abhinav  Dhall</span>, <span class="hlFld-ContribAuthor ">Isaac S.  Harris</span>, <span class="hlFld-ContribAuthor ">M. Andres  Blanco</span>, <span class="hlFld-ContribAuthor ">David B.  Sykes</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Licht</span>, <span class="hlFld-ContribAuthor ">David M.  Weinstock</span>, <span class="hlFld-ContribAuthor ">Ari  Melnick</span>, <span class="hlFld-ContribAuthor ">Marcia C.  Haigis</span>, <span class="hlFld-ContribAuthor ">Franziska  Michor</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Combined epigenetic and metabolic treatments overcome differentiation blockade in acute myeloid leukemia. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2021,</strong> <em>24 </em>
                                    (6)
                                     , 102651. <a href="https://doi.org/10.1016/j.isci.2021.102651" title="DOI URL">https://doi.org/10.1016/j.isci.2021.102651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2021.102651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2021.102651%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DCombined%252Bepigenetic%252Band%252Bmetabolic%252Btreatments%252Bovercome%252Bdifferentiation%252Bblockade%252Bin%252Bacute%252Bmyeloid%252Bleukemia%26aulast%3DZee%26aufirst%3DBarry%2BM.%26date%3D2021%26volume%3D24%26issue%3D6%26spage%3D102651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Synthesis of CC-90011. </span><span class="cited-content_cbyCitation_journal-name">Synfacts</span><span> <strong>2021,</strong>,, 0242. <a href="https://doi.org/10.1055/s-0040-1719367" title="DOI URL">https://doi.org/10.1055/s-0040-1719367</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/s-0040-1719367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fs-0040-1719367%26sid%3Dliteratum%253Aachs%26jtitle%3DSynfacts%26atitle%3DSynthesis%252Bof%252BCC-90011%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D03%26spage%3D0242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Jun  Fu</span>, <span class="hlFld-ContribAuthor ">Jun  Li</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Annual review of LSD1/KDM1A inhibitors in 2020. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113254. <a href="https://doi.org/10.1016/j.ejmech.2021.113254" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113254%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAnnual%252Breview%252Bof%252BLSD1%25252FKDM1A%252Binhibitors%252Bin%252B2020%26aulast%3DFu%26aufirst%3DDong-Jun%26date%3D2021%26volume%3D214%26spage%3D113254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0011.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Clinical stage LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Model of compound <b>1</b> in the AO pocket of LSD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HKO">2HKO</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of compound <b>11</b> in the AO pocket of LSD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>11</b> or CC-90011<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) 1.3 equiv of NaOH, THF, rt, overnight, 69% yield; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, overnight, 62% yield; (c) <i>tert</i>-butyl piperidin-4-ylcarbamate, DIEA, DMF, rt, overnight, 80% yield; (d) 4-cyano-3-fluorophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O, 85 °C, 57% yield; (e) (3-fluoro-4-methoxyphenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 80 °C, 92% yield; (f) benzenesulfonic acid, formic acid.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) GRP mRNA expression in H209 and H1417 cell lines with treatment of compound <b>11</b>. (B) Cell-Titer Glo viability assay in H1417 cell line with treatment using compound <b>11</b>. (C) In vivo effect of compound <b>11</b> treatment on human GRP mRNA expression in SCLC H1417 model at 2.5 mg/kg, 5 mg/kg, and 10 mg/kg QD dosing for 4 days, <i>n</i> = 3 mice/group. (D) In vivo efficacy of compound <b>11</b> in mice bearing SCLC xenograft model H1417 at 2.5 mg/kg and 5 mg/kg QD dosing for 65 days, <i>n</i> = 7 or 8 mice/group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo efficacy of compound <b>11</b> in mice bearing PDX model LXFS615 at 5 mg/kg QD dosing for 30 days, <i>n</i> = 10 mice/group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/medium/jm0c00978_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) In vivo efficacy of compound <b>11</b> in mice bearing PDX model LU2514 at 5 and 10 mg/kg QD dosing for 28 days, <i>n</i> = 8 or 10 mice/group. (b) H&E staining of vehicle and compound <b>11</b> treated (10 mg/kg) tumor (LU2514) 14 days after last dose.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00978/20201203/images/large/jm0c00978_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00978&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46525" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46525" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 18 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Enane, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunthararajah, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korc, M.</span></span> <span> </span><span class="NLM_article-title">Differentiation therapy and the mechanisms that terminate cancer cell proliferation without harming normal cells</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">912</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0919-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fs41419-018-0919-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30190481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BB3c3mtlCksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=912&issue=9&author=F.+O.+Enaneauthor=Y.+Saunthararajahauthor=M.+Korc&title=Differentiation+therapy+and+the+mechanisms+that+terminate+cancer+cell+proliferation+without+harming+normal+cells&doi=10.1038%2Fs41419-018-0919-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation therapy and the mechanisms that terminate cancer cell proliferation without harming normal cells</span></div><div class="casAuthors">Enane Francis O; Korc Murray; Saunthararajah Yogen; Saunthararajah Yogen; Korc Murray; Korc Murray</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">912</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemotherapeutic drugs have a common intent to activate apoptosis in tumor cells.  However, master regulators of apoptosis (e.g., p53, p16/CDKN2A) are frequently genetically inactivated in cancers, resulting in multidrug resistance.  An alternative, p53-independent method for terminating malignant proliferation is to engage terminal-differentiation.  Normally, the exponential proliferation of lineage-committed progenitors, coordinated by the master transcription factor (TF) MYC, is self-limited by forward-differentiation to terminal lineage-fates.  In cancers, however, this exponential proliferation is disengaged from terminal-differentiation.  The mechanisms underlying this decoupling are mostly unknown.  We performed a systematic review of published literature (January 2007-June 2018) to identify gene pathways linked to differentiation-failure in three treatment-recalcitrant cancers: hepatocellular carcinoma (HCC), ovarian cancer (OVC), and pancreatic ductal adenocarcinoma (PDAC).  We analyzed key gene alterations in various apoptosis, proliferation and differentiation pathways to determine whether it is possible to predict treatment outcomes and suggest novel therapies.  Poorly differentiated tumors were linked to poorer survival across histologies.  Our analyses suggested loss-of-function events to master TF drivers of lineage-fates and their cofactors as being linked to differentiation-failure: genomic data in TCGA and ICGC databases demonstrated frequent haploinsufficiency of lineage master TFs (e.g., GATA4/6) in poorly differentiated tumors; the coactivators that these TFs use to activate genes (e.g.  ARID1A, PBRM1) were also frequently inactivated by genetic mutation and/or deletion.  By contrast, corepressor components (e.g., DNMT1, EED, UHRF1, and BAZ1A/B), that oppose coactivators to repress or turn off genes, were frequently amplified instead, and the level of amplification was highest in poorly differentiated lesions.  This selection by neoplastic evolution towards unbalanced activity of transcriptional corepressors suggests these enzymes as candidate targets for inhibition aiming to re-engage forward-differentiation.  This notion is supported by both pre-clinical and clinical trial literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMrdazT79u5suCWE5mhx1RfW6udTcc2eYZK8yF_SJ-z7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3mtlCksw%253D%253D&md5=859bac67bb3bac116dc150979fa94f99</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0919-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0919-9%26sid%3Dliteratum%253Aachs%26aulast%3DEnane%26aufirst%3DF.%2BO.%26aulast%3DSaunthararajah%26aufirst%3DY.%26aulast%3DKorc%26aufirst%3DM.%26atitle%3DDifferentiation%2520therapy%2520and%2520the%2520mechanisms%2520that%2520terminate%2520cancer%2520cell%2520proliferation%2520without%2520harming%2520normal%2520cells%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D912%26doi%3D10.1038%2Fs41419-018-0919-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">de The, H.</span></span> <span> </span><span class="NLM_article-title">Differentiation therapy revisited</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fnrc.2017.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=29192213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=117-127&issue=2&author=H.+de+The&title=Differentiation+therapy+revisited&doi=10.1038%2Fnrc.2017.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation therapy revisited</span></div><div class="casAuthors">de The, Hugues</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">117-127</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The concept of differentiation therapy emerged from the fact that hormones or cytokines may promote differentiation ex vivo, thereby irreversibly changing the phenotype of cancer cells.  Its hallmark success has been the treatment of acute promyelocytic leukemia (APL), a condition that is now highly curable by the combination of retinoic acid (RA) and arsenic.  Recently, drugs that trigger differentiation in a variety of primary tumor cells have been identified, suggesting that they are clin. useful.  This Opinion article analyses the basis for the clin. successes of RA or arsenic in APL by assessing the resp. roles of terminal maturation and loss of self-renewal.  By reviewing other successful examples of drug-induced tumor cell differentiation, novel approaches to transform differentiating drugs into more efficient therapies are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr76NekKL66CrVg90H21EOLACvtfcHk0lj3925uPeEmiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtb%252FF&md5=f30632a18f99479ca8910bbe66c3f607</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.103%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DDifferentiation%2520therapy%2520revisited%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2018%26volume%3D18%26issue%3D2%26spage%3D117%26epage%3D127%26doi%3D10.1038%2Fnrc.2017.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoe, T.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of action and resistance to all-trans retinoic acid (ATRA) and arsenic trioxide (As<sub>2</sub>O<sub>3</sub>) in acute promyelocytic leukemia</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1007/s12185-013-1354-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1007%2Fs12185-013-1354-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=23670176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXps1yntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=717-725&issue=6&author=A.+Tomitaauthor=H.+Kiyoiauthor=T.+Naoe&title=Mechanisms+of+action+and+resistance+to+all-trans+retinoic+acid+%28ATRA%29+and+arsenic+trioxide+%28As2O3%29+in+acute+promyelocytic+leukemia&doi=10.1007%2Fs12185-013-1354-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of action and resistance to all-trans retinoic acid (ATRA) and arsenic trioxide (As2O3) in acute promyelocytic leukemia</span></div><div class="casAuthors">Tomita, Akihiro; Kiyoi, Hitoshi; Naoe, Tomoki</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">717-725</span>CODEN:
                <span class="NLM_cas:coden">IJHEEY</span>;
        ISSN:<span class="NLM_cas:issn">0925-5710</span>.
    
            (<span class="NLM_cas:orgname">Springer Japan</span>)
        </div><div class="casAbstract">A review.  Since the introduction of all-trans retinoic acid (ATRA) and arsenic trioxide (As2O3) for the treatment of acute promyelocytic leukemia (APL), the overall survival rate has improved dramatically.  However, relapse/refractory patients showing resistance to ATRA and/or As2O3 are recognized as a clin. significant problem.  Genetic mutations resulting in amino acid substitution in the retinoic acid receptor alpha (RARα) ligand binding domain (LBD) and the PML-B2 domain of PML-RARα, resp., have been reported as mol. mechanisms underlying resistance to ATRA and As2O3.  In the LBD mutation, ATRA binding with LBD is generally impaired, and ligand-dependent co-repressor dissocn. and degrdn. of PML-RARα by the proteasome pathway, leading to cell differentiation, are inhibited.  The PML-B2 mutation interferes with the direct binding of As2O3 with PML-B2, and PML-RARα SUMOylation with As2O3 followed by multimerization and degrdn. is impaired.  To overcome ATRA resistance, utilization of As2O3 provides a preferable outcome, and recently, a synthetic retinoid Am80, which has a higher binding affinity with PML-RARα than ATRA, has been tested in the clin. setting.  However, no strategy attempted to date has been successful in overcoming As2O3 resistance.  Detailed genomic analyses using patient samples harvested repeatedly may help in predicting the prognosis, selecting the effective targeting drugs, and designing new sophisticated strategies for the treatment of APL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRi89xz-8FzbVg90H21EOLACvtfcHk0lj3925uPeEmiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXps1yntLs%253D&md5=a762e823f1035ec451046d4e8de7159e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs12185-013-1354-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-013-1354-4%26sid%3Dliteratum%253Aachs%26aulast%3DTomita%26aufirst%3DA.%26aulast%3DKiyoi%26aufirst%3DH.%26aulast%3DNaoe%26aufirst%3DT.%26atitle%3DMechanisms%2520of%2520action%2520and%2520resistance%2520to%2520all-trans%2520retinoic%2520acid%2520%2528ATRA%2529%2520and%2520arsenic%2520trioxide%2520%2528As2O3%2529%2520in%2520acute%2520promyelocytic%2520leukemia%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2013%26volume%3D97%26issue%3D6%26spage%3D717%26epage%3D725%26doi%3D10.1007%2Fs12185-013-1354-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovici-Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salituro, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianchetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penard-Lacronique, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quivoron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saada, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Botton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaharski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of AG-120 (Ivosidenib): A first-in-class mutant IDH1 inhibitor for the treatment of IDH1 mutant cancers</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00421</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00421" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=300-305&issue=4&author=J.+Popovici-Mullerauthor=R.+M.+Lemieuxauthor=E.+Artinauthor=J.+O.+Saundersauthor=F.+G.+Salituroauthor=J.+Travinsauthor=G.+Cianchettaauthor=Z.+Caiauthor=D.+Zhouauthor=D.+Cuiauthor=P.+Chenauthor=K.+Straleyauthor=E.+Tobinauthor=F.+Wangauthor=M.+D.+Davidauthor=V.+Penard-Lacroniqueauthor=C.+Quivoronauthor=V.+Saadaauthor=S.+de+Bottonauthor=S.+Grossauthor=L.+Dangauthor=H.+Yangauthor=L.+Utleyauthor=Y.+Chenauthor=H.+Kimauthor=S.+Jinauthor=Z.+Guauthor=G.+Yaoauthor=Z.+Luoauthor=X.+Lvauthor=C.+Fangauthor=L.+Yanauthor=A.+Olaharskiauthor=L.+Silvermanauthor=S.+Billerauthor=S.+M.+Suauthor=K.+Yen&title=Discovery+of+AG-120+%28Ivosidenib%29%3A+A+first-in-class+mutant+IDH1+inhibitor+for+the+treatment+of+IDH1+mutant+cancers&doi=10.1021%2Facsmedchemlett.7b00421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of AG-120 (Ivosidenib): A First-in-Class Mutant IDH1 Inhibitor for the Treatment of IDH1 Mutant Cancers</span></div><div class="casAuthors">Popovici-Muller, Janeta; Lemieux, Rene M.; Artin, Erin; Saunders, Jeffrey O.; Salituro, Francesco G.; Travins, Jeremy; Cianchetta, Giovanni; Cai, Zhenwei; Zhou, Ding; Cui, Dawei; Chen, Ping; Straley, Kimberly; Tobin, Erica; Wang, Fang; David, Muriel D.; Penard-Lacronique, Virginie; Quivoron, Cyril; Saada, Veronique; de Botton, Stephane; Gross, Stefan; Dang, Lenny; Yang, Hua; Utley, Luke; Chen, Yue; Kim, Hyeryun; Jin, Shengfang; Gu, Zhiwei; Yao, Gui; Luo, Zhiyong; Lv, Xiaobing; Fang, Cheng; Yan, Liping; Olaharski, Andrew; Silverman, Lee; Biller, Scott; Su, Shin-San M.; Yen, Katharine</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">300-305</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Somatic point mutations at a key arginine residue (R132) within the active site of the metabolic enzyme isocitrate dehydrogenase 1 (IDH1) confer a novel gain of function in cancer cells, resulting in the prodn. of D-2-hydroxyglutarate (2-HG), an oncometabolite.  Elevated 2-HG levels are implicated in epigenetic alterations and impaired cellular differentiation.  IDH1 mutations have been described in an array of hematol. malignancies and solid tumors.  Here, we report the discovery of AG-120 (ivosidenib), an inhibitor of the IDH1 mutant enzyme that exhibits profound 2-HG lowering in tumor models and the ability to effect differentiation of primary patient AML samples ex vivo.  Preliminary data from phase 1 clin. trials enrolling patients with cancers harboring an IDH1 mutation indicate that AG-120 has an acceptable safety profile and clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmTLcR37BiPrVg90H21EOLACvtfcHk0lj3925uPeEmiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyrurw%253D&md5=730ac2faa6844e4abe3fd9a360d74d29</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00421%26sid%3Dliteratum%253Aachs%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DLemieux%26aufirst%3DR.%2BM.%26aulast%3DArtin%26aufirst%3DE.%26aulast%3DSaunders%26aufirst%3DJ.%2BO.%26aulast%3DSalituro%26aufirst%3DF.%2BG.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DCianchetta%26aufirst%3DG.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DStraley%26aufirst%3DK.%26aulast%3DTobin%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DDavid%26aufirst%3DM.%2BD.%26aulast%3DPenard-Lacronique%26aufirst%3DV.%26aulast%3DQuivoron%26aufirst%3DC.%26aulast%3DSaada%26aufirst%3DV.%26aulast%3Dde%2BBotton%26aufirst%3DS.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DUtley%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DOlaharski%26aufirst%3DA.%26aulast%3DSilverman%26aufirst%3DL.%26aulast%3DBiller%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DS.%2BM.%26aulast%3DYen%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520AG-120%2520%2528Ivosidenib%2529%253A%2520A%2520first-in-class%2520mutant%2520IDH1%2520inhibitor%2520for%2520the%2520treatment%2520of%2520IDH1%2520mutant%2520cancers%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D4%26spage%3D300%26epage%3D305%26doi%3D10.1021%2Facsmedchemlett.7b00421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kfoury, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercier, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schajnovitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolliday, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxcy, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobert, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szekely, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scadden, D. T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dihydroorotate dehydrogenase overcomes differentiation blockade in acute myeloid leukemia</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.08.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.cell.2016.08.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27641501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFamt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=171-186&issue=1&author=D.+B.+Sykesauthor=Y.+S.+Kfouryauthor=F.+E.+Mercierauthor=M.+J.+Wawerauthor=J.+M.+Lawauthor=M.+K.+Haynesauthor=T.+A.+Lewisauthor=A.+Schajnovitzauthor=E.+Jainauthor=D.+Leeauthor=H.+Meyerauthor=K.+A.+Pierceauthor=N.+J.+Tollidayauthor=A.+Wallerauthor=S.+J.+Ferraraauthor=A.+L.+Eheimauthor=D.+Stoeckigtauthor=K.+L.+Maxcyauthor=J.+M.+Cobertauthor=J.+Bachandauthor=B.+A.+Szekelyauthor=S.+Mukherjeeauthor=L.+A.+Sklarauthor=J.+D.+Kotzauthor=C.+B.+Clishauthor=R.+I.+Sadreyevauthor=P.+A.+Clemonsauthor=A.+Janzerauthor=S.+L.+Schreiberauthor=D.+T.+Scadden&title=Inhibition+of+dihydroorotate+dehydrogenase+overcomes+differentiation+blockade+in+acute+myeloid+leukemia&doi=10.1016%2Fj.cell.2016.08.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Dihydroorotate Dehydrogenase Overcomes Differentiation Blockade in Acute Myeloid Leukemia</span></div><div class="casAuthors">Sykes, David B.; Kfoury, Youmna S.; Mercier, Francois E.; Wawer, Mathias J.; Law, Jason M.; Haynes, Mark K.; Lewis, Timothy A.; Schajnovitz, Amir; Jain, Esha; Lee, Dongjun; Meyer, Hanna; Pierce, Kerry A.; Tolliday, Nicola J.; Waller, Anna; Ferrara, Steven J.; Eheim, Ashley L.; Stoeckigt, Detlef; Maxcy, Katrina L.; Cobert, Julien M.; Bachand, Jacqueline; Szekely, Brian A.; Mukherjee, Siddhartha; Sklar, Larry A.; Kotz, Joanne D.; Clish, Clary B.; Sadreyev, Ruslan I.; Clemons, Paul A.; Janzer, Andreas; Schreiber, Stuart L.; Scadden, David T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-186.e15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">While acute myeloid leukemia (AML) comprises many disparate genetic subtypes, one shared hallmark is the arrest of leukemic myeloblasts at an immature and self-renewing stage of development.  Therapies that overcome differentiation arrest represent a powerful treatment strategy.  The authors leveraged the observation that the majority of AML, despite their genetically heterogeneity, share in the expression of HoxA9, a gene normally downregulated during myeloid differentiation.  Using a conditional HoxA9 model system, the authors performed a high-throughput phenotypic screen and defined compds. that overcame differentiation blockade.  Target identification led to the unanticipated discovery that inhibition of the enzyme dihydroorotate dehydrogenase (DHODH) enables myeloid differentiation in human and mouse AML models.  In vivo, DHODH inhibitors reduced leukemic cell burden, decreased levels of leukemia-initiating cells, and improved survival.  These data demonstrate the role of DHODH as a metabolic regulator of differentiation and point to its inhibition as a strategy for overcoming differentiation blockade in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5JmGddhG2CbVg90H21EOLACvtfcHk0lhCKwE-VpL5mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFamt7nL&md5=19a4971c5a5c0a981ca39a1a6e31a20d</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.08.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.08.057%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DKfoury%26aufirst%3DY.%2BS.%26aulast%3DMercier%26aufirst%3DF.%2BE.%26aulast%3DWawer%26aufirst%3DM.%2BJ.%26aulast%3DLaw%26aufirst%3DJ.%2BM.%26aulast%3DHaynes%26aufirst%3DM.%2BK.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DSchajnovitz%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DPierce%26aufirst%3DK.%2BA.%26aulast%3DTolliday%26aufirst%3DN.%2BJ.%26aulast%3DWaller%26aufirst%3DA.%26aulast%3DFerrara%26aufirst%3DS.%2BJ.%26aulast%3DEheim%26aufirst%3DA.%2BL.%26aulast%3DStoeckigt%26aufirst%3DD.%26aulast%3DMaxcy%26aufirst%3DK.%2BL.%26aulast%3DCobert%26aufirst%3DJ.%2BM.%26aulast%3DBachand%26aufirst%3DJ.%26aulast%3DSzekely%26aufirst%3DB.%2BA.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DScadden%26aufirst%3DD.%2BT.%26atitle%3DInhibition%2520of%2520dihydroorotate%2520dehydrogenase%2520overcomes%2520differentiation%2520blockade%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DCell%26date%3D2016%26volume%3D167%26issue%3D1%26spage%3D171%26epage%3D186%26doi%3D10.1016%2Fj.cell.2016.08.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Christian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gradl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejeune, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzezinka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesche, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scadden, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span> <span> </span><span class="NLM_article-title">The novel dihydroorotate dehydrogenase (DHODH) inhibitor BAY 2402234 triggers differentiation and is effective in the treatment of myeloid malignancies</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2403</span>– <span class="NLM_lpage">2415</span>, <span class="refDoi"> DOI: 10.1038/s41375-019-0461-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fs41375-019-0461-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30940908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyqsr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=2403-2415&issue=10&author=S.+Christianauthor=C.+Merzauthor=L.+Evansauthor=S.+Gradlauthor=H.+Seidelauthor=A.+Fribergauthor=A.+Eheimauthor=P.+Lejeuneauthor=K.+Brzezinkaauthor=K.+Zimmermannauthor=S.+Ferraraauthor=H.+Meyerauthor=R.+Lescheauthor=D.+Stoeckigtauthor=M.+Bauserauthor=A.+Haegebarthauthor=D.+B.+Sykesauthor=D.+T.+Scaddenauthor=J.+A.+Losmanauthor=A.+Janzer&title=The+novel+dihydroorotate+dehydrogenase+%28DHODH%29+inhibitor+BAY+2402234+triggers+differentiation+and+is+effective+in+the+treatment+of+myeloid+malignancies&doi=10.1038%2Fs41375-019-0461-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">The novel dihydroorotate dehydrogenase (DHODH) inhibitor BAY 2402234 triggers differentiation and is effective in the treatment of myeloid malignancies</span></div><div class="casAuthors">Christian, Sven; Merz, Claudia; Evans, Laura; Gradl, Stefan; Seidel, Henrik; Friberg, Anders; Eheim, Ashley; Lejeune, Pascale; Brzezinka, Krzysztof; Zimmermann, Katja; Ferrara, Steven; Meyer, Hanna; Lesche, Ralf; Stoeckigt, Detlef; Bauser, Marcus; Haegebarth, Andrea; Sykes, David B.; Scadden, David T.; Losman, Julie-Aurore; Janzer, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2403-2415</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a devastating disease, with the majority of patients dying within a year of diagnosis.  For patients with relapsed/refractory AML, the prognosis is particularly poor with currently available treatments.  Although genetically heterogeneous, AML subtypes share a common differentiation arrest at hematopoietic progenitor stages.  Overcoming this differentiation arrest has the potential to improve the long-term survival of patients, as is the case in acute promyelocytic leukemia (APL), which is characterized by a chromosomal translocation involving the retinoic acid receptor alpha gene.  Treatment of APL with all-trans retinoic acid (ATRA) induces terminal differentiation and apoptosis of leukemic promyelocytes, resulting in cure rates of over 80%.  Unfortunately, similarly efficacious differentiation therapies have, to date, been lacking outside of APL.  Inhibition of dihydroorotate dehydrogenase (DHODH), a key enzyme in the de novo pyrimidine synthesis pathway, was recently reported to induce differentiation of diverse AML subtypes.  In this report we describe the discovery and characterization of BAY 2402234 - a novel, potent, selective and orally bioavailable DHODH inhibitor that shows monotherapy efficacy and differentiation induction across multiple AML subtypes.  Herein, we present the preclin. data that led to initiation of a phase I evaluation of this inhibitor in myeloid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpESpixEevQf7Vg90H21EOLACvtfcHk0lgEn7tWvjp9IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyqsr7J&md5=30e0ac086fb33617b5a4ecbbaa36d258</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fs41375-019-0461-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-019-0461-5%26sid%3Dliteratum%253Aachs%26aulast%3DChristian%26aufirst%3DS.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DEvans%26aufirst%3DL.%26aulast%3DGradl%26aufirst%3DS.%26aulast%3DSeidel%26aufirst%3DH.%26aulast%3DFriberg%26aufirst%3DA.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DLejeune%26aufirst%3DP.%26aulast%3DBrzezinka%26aufirst%3DK.%26aulast%3DZimmermann%26aufirst%3DK.%26aulast%3DFerrara%26aufirst%3DS.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DLesche%26aufirst%3DR.%26aulast%3DStoeckigt%26aufirst%3DD.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DScadden%26aufirst%3DD.%2BT.%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DJanzer%26aufirst%3DA.%26atitle%3DThe%2520novel%2520dihydroorotate%2520dehydrogenase%2520%2528DHODH%2529%2520inhibitor%2520BAY%25202402234%2520triggers%2520differentiation%2520and%2520is%2520effective%2520in%2520the%2520treatment%2520of%2520myeloid%2520malignancies%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26issue%3D10%26spage%3D2403%26epage%3D2415%26doi%3D10.1038%2Fs41375-019-0461-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span> <span> </span><span class="NLM_article-title">Differentiation-inducing therapy for solid tumors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.2174/138161206775201947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.2174%2F138161206775201947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=16454751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlCqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=379-385&issue=3&author=H.+Kawamataauthor=M.+Tachibanaauthor=T.+Fujimoriauthor=Y.+Imai&title=Differentiation-inducing+therapy+for+solid+tumors&doi=10.2174%2F138161206775201947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation-inducing therapy for solid tumors</span></div><div class="casAuthors">Kawamata, Hitoshi; Tachibana, Masatsugu; Fujimori, Takahiro; Imai, Yutaka</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-385</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Treating malignant tumor through the induction of cell differentiation has been an attractive concept, but clin. development of differentiation-inducing agents to treat malignant tumor, esp. for solid tumors has been limited to date.  Nerve growth factor, all trans retinoic acid, DMSO, active form vitamin D3, peroxisome proliferator-activated receptor γ, 12-0-tetradecanoylphorbol 13-acetate, hexamethylene-bis-acetamide, transforming growth factor-β, butyric acid, cAMP, and vesnarinone are known to have a differentiation-inducing capability on solid tumors in vitro and/or in vivo.  Moreover some of the differentiation-inducing agents have been used for treating patients with solid tumor, but the therapeutic effect of the differentiation-inducing agents on solid tumor is not strong when compared with that of conventional chemotherapeutic agents.  However, because most of the differentiation-inducing agents can potentiate the effect of conventional chemotherapy or radiation therapy, combination of differentiation-inducing therapy with conventional chemotherapy or radiation therapy might be used as a second- or third-line therapy in patients with advanced cancer.  Furthermore, anal. of the mol. mechanisms of the tumor differentiation therapy might provide selective and targeted mols. for novel cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuQmfkM8acUbVg90H21EOLACvtfcHk0lgEn7tWvjp9IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlCqtw%253D%253D&md5=0447b91f783b47bed4cbff034afa545e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F138161206775201947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161206775201947%26sid%3Dliteratum%253Aachs%26aulast%3DKawamata%26aufirst%3DH.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DFujimori%26aufirst%3DT.%26aulast%3DImai%26aufirst%3DY.%26atitle%3DDifferentiation-inducing%2520therapy%2520for%2520solid%2520tumors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2006%26volume%3D12%26issue%3D3%26spage%3D379%26epage%3D385%26doi%3D10.2174%2F138161206775201947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&issue=7&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lgEn7tWvjp9IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26issue%3D7%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human histone lysine-specific demethylase 1 (LSD1)</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">13956</span>– <span class="NLM_lpage">13961</span>, <span class="refDoi"> DOI: 10.1073/pnas.0606381103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1073%2Fpnas.0606381103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=16956976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCmtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=13956-13961&issue=38&author=Y.+Chenauthor=Y.+Yangauthor=F.+Wangauthor=K.+Wanauthor=K.+Yamaneauthor=Y.+Zhangauthor=M.+Lei&title=Crystal+structure+of+human+histone+lysine-specific+demethylase+1+%28LSD1%29&doi=10.1073%2Fpnas.0606381103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human histone lysine-specific demethylase 1 (LSD1)</span></div><div class="casAuthors">Chen, Yong; Yang, Yuting; Wang, Feng; Wan, Ke; Yamane, Kenichi; Zhang, Yi; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">13956-13961</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lysine-specific histone demethylase 1 (LSD1) was recently identified as the 1st histone demethylase that specifically demethylates monomethylated and dimethylated histone H3 at Lys-4.  It is a component of the CoREST and other corepressor complexes and plays an important role in silencing neuronal-specific genes in non-neuronal cells, but the mol. mechanisms of its action remain unclear.  Here, the 2.8-Å-resoln. crystal structure of human LSD1 is reported.  It revealed that LSD1 defines a new subfamily of FAD-dependent oxidases.  The active center of LSD1 was characterized by a remarkable 1245-Å3 substrate-binding cavity with a highly neg. electrostatic potential.  Although the protein core of LSD1 resembled other flavoenzymes, its enzymic activity and functions required 2 addnl. structural modules: an N-terminal SWIRM domain important for protein stability and a large insertion in the catalytic domain indispensable both for demethylase activity and the interaction with CoREST.  These results provide a framework for further probing of the catalytic mechanism and the functional roles of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Tqfaz7oaE7Vg90H21EOLACvtfcHk0lgEn7tWvjp9IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCmtbfK&md5=13da0bfe3f99d217eae73924dc3f6fcf</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0606381103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0606381103%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DCrystal%2520structure%2520of%2520human%2520histone%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26issue%3D38%26spage%3D13956%26epage%3D13961%26doi%3D10.1073%2Fpnas.0606381103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span> <span> </span><span class="NLM_article-title">LSD1: biologic roles and therapeutic targeting</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1103</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.2217%2Fepi-2016-0009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27479862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlCisr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1103-1116&issue=8&author=A.+Maiques-Diazauthor=T.+C.+Somervaille&title=LSD1%3A+biologic+roles+and+therapeutic+targeting&doi=10.2217%2Fepi-2016-0009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1: biologic roles and therapeutic targeting</span></div><div class="casAuthors">Maiques-Diaz, Alba; Somervaille, Tim CP</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1103-1116</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">LSD1 (KDM1A; BHC110; AOF2) was the first protein reported to exhibit histone demethylase activity and has since been shown to have multiple essential roles in mammalian biol.  Given its enzymic activity and its high-level expression in many human malignancies, a significant recent focus has been the development of pharmacol. inhibitors.  Here we summarize structural and biochem. knowledge of this important epigenetic regulator, with a particular emphasis on the functional and preclin. studies in oncol. that have provided justification for the evaluation of tranylcypromine deriv. LSD1 inhibitors in early phase clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroYKq_-3Ts2bVg90H21EOLACvtfcHk0lgqrZ20m5H-_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlCisr3P&md5=9bd062613a49f8527ee2b7f8e119e6ac</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0009%26sid%3Dliteratum%253Aachs%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%253A%2520biologic%2520roles%2520and%2520therapeutic%2520targeting%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26issue%3D8%26spage%3D1103%26epage%3D1116%26doi%3D10.2217%2Fepi-2016-0009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): A computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e25444</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25444&issue=9&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+A+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0lgqrZ20m5H-_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520A%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26issue%3D9%26spage%3De25444%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrello, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarelli, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellore, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1677</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&issue=8&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lgqrZ20m5H-_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">LSD1 inhibitor T-3775440 inhibits SCLC cell proliferation by disrupting LSD1 interactions with SNAG domain proteins INSM1 and GFI1B</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">4652</span>– <span class="NLM_lpage">4662</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-3502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F0008-5472.CAN-16-3502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=28667074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=4652-4662&issue=17&author=S.+Takagiauthor=Y.+Ishikawaauthor=A.+Mizutaniauthor=S.+Iwasakiauthor=S.+Matsumotoauthor=Y.+Kamadaauthor=T.+Nomuraauthor=K.+Nakamura&title=LSD1+inhibitor+T-3775440+inhibits+SCLC+cell+proliferation+by+disrupting+LSD1+interactions+with+SNAG+domain+proteins+INSM1+and+GFI1B&doi=10.1158%2F0008-5472.CAN-16-3502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 Inhibitor T-3775440 Inhibits SCLC Cell Proliferation by Disrupting LSD1 Interactions with SNAG Domain Proteins INSM1 and GFI1B</span></div><div class="casAuthors">Takagi, Shinji; Ishikawa, Yoshinori; Mizutani, Akio; Iwasaki, Shinji; Matsumoto, Satoru; Kamada, Yusuke; Nomura, Toshiyuki; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4652-4662</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">T-3775440 is an irreversible inhibitor of the chromatin demethylase LSD1, which exerts antiproliferative effects by disrupting the interaction between LSD1 and GFI1B, a SNAG domain transcription factor, inducing leukemia cell transdifferentiation.  Here, we describe the anticancer effects and mechanism of action of T-3775440 in small-cell lung cancer (SCLC).  T-3775440 inhibited proliferation of SCLC cells in vitro and retarded SCLC tumor growth in vivo.  T-3775440 disrupted the interaction between LSD1 and the transcriptional repressor INSM1, thereby inhibiting expression of neuroendocrine-assocd. genes, such as ASCL1.  INSM1 silencing phenocopied the effects of T-3775440 on gene expression and cell proliferation, consistent with the likelihood T-3775440 mediated its effects in SCLC by inhibiting INSM1.  T-3775440 also inhibited proliferation of an SCLC cell line that overexpressed GFI1B, rather than INSM1, by disrupting the interaction between LSD1 and GFI1B.  Taken together, our results argue that LSD1 plays an important role in neuroendocrine-assocd. transcription and cell proliferation of SCLC via interactions with the SNAG domain proteins INSM1 and GFI1B.  Targeting these crit. interactions with LSD1 inhibitors offers a novel rational strategy to therapeutically manage SCLC.  Cancer Res; 77(17); 4652-62. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqygHUXf_CKq7Vg90H21EOLACvtfcHk0lg8ZgASrrU2wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7rM&md5=8cd3ee5ab304690ec57e83fda605d7c1</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-3502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-3502%26sid%3Dliteratum%253Aachs%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DA.%26aulast%3DIwasaki%26aufirst%3DS.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DLSD1%2520inhibitor%2520T-3775440%2520inhibits%2520SCLC%2520cell%2520proliferation%2520by%2520disrupting%2520LSD1%2520interactions%2520with%2520SNAG%2520domain%2520proteins%2520INSM1%2520and%2520GFI1B%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D17%26spage%3D4652%26epage%3D4662%26doi%3D10.1158%2F0008-5472.CAN-16-3502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairiki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hikichi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">A novel LSD1 inhibitor T-3775440 disrupts GFI1B-containing complex leading to transdifferentiation and impaired growth of AML cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F1535-7163.MCT-16-0471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=27903753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=273-284&issue=2&author=Y.+Ishikawaauthor=K.+Gamoauthor=M.+Yabukiauthor=S.+Takagiauthor=K.+Toyoshimaauthor=K.+Nakayamaauthor=A.+Nakayamaauthor=M.+Morimotoauthor=H.+Miyashitaauthor=R.+Dairikiauthor=Y.+Hikichiauthor=N.+Tomitaauthor=D.+Tomitaauthor=S.+Imamuraauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Matsumotoauthor=R.+Haraauthor=T.+Nomuraauthor=K.+Tsuchidaauthor=K.+Nakamura&title=A+novel+LSD1+inhibitor+T-3775440+disrupts+GFI1B-containing+complex+leading+to+transdifferentiation+and+impaired+growth+of+AML+cells&doi=10.1158%2F1535-7163.MCT-16-0471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel LSD1 Inhibitor T-3775440 Disrupts GFI1B-Containing Complex Leading to Transdifferentiation and Impaired Growth of AML Cells</span></div><div class="casAuthors">Ishikawa, Yoshinori; Gamo, Kanae; Yabuki, Masato; Takagi, Shinji; Toyoshima, Kosei; Nakayama, Kazuhide; Nakayama, Akiko; Morimoto, Megumi; Miyashita, Hitoshi; Dairiki, Ryo; Hikichi, Yukiko; Tomita, Naoki; Tomita, Daisuke; Imamura, Shinichi; Iwatani, Misa; Kamada, Yusuke; Matsumoto, Satoru; Hara, Ryujiro; Nomura, Toshiyuki; Tsuchida, Ken; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-284</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Dysregulation of lysine (K)-specific demethylase 1A (LSD1), also known as KDM1A, has been implicated in the development of various cancers, including leukemia.  Here, we describe the antileukemic activity and mechanism of action of T-3775440, a novel irreversible LSD1 inhibitor.  Cell growth anal. of leukemia cell lines revealed that acute erythroid leukemia (AEL) and acute megakaryoblastic leukemia cells (AMKL) were highly sensitive to this compd.  T-3775440 treatment enforced transdifferentiation of erythroid/megakaryocytic lineages into granulomonocytic-like lineage cells.  Mechanistically, T-3775440 disrupted the interaction between LSD1 and growth factor-independent 1B (GFI1B), a transcription factor crit. for the differentiation processes of erythroid and megakaryocytic lineage cells.  Knockdown of LSD1 and GFI1B recapitulated T-3775440-induced transdifferentiation and cell growth suppression, highlighting the significance of LSD1-GFI1B axis inhibition with regard to the anti-AML effects of T-3775440.  Moreover, T-3775440 exhibited significant antitumor efficacy in AEL and AMKL xenograft models.  Our findings provide a rationale for evaluating LSD1 inhibitors as potential treatments and indicate a novel mechanism of action against AML, particularly AEL and AMKL.  Mol Cancer Ther; 16(2); 273-84. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomcxpsluqnf7Vg90H21EOLACvtfcHk0lg8ZgASrrU2wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D&md5=2335343b4d5e7bbc197a1d5aec54e3af</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0471%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DGamo%26aufirst%3DK.%26aulast%3DYabuki%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DToyoshima%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DDairiki%26aufirst%3DR.%26aulast%3DHikichi%26aufirst%3DY.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DImamura%26aufirst%3DS.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DA%2520novel%2520LSD1%2520inhibitor%2520T-3775440%2520disrupts%2520GFI1B-containing%2520complex%2520leading%2520to%2520transdifferentiation%2520and%2520impaired%2520growth%2520of%2520AML%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26issue%3D2%26spage%3D273%26epage%3D284%26doi%3D10.1158%2F1535-7163.MCT-16-0471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebestreit, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popescu, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelent, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1038/nm.2661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&issue=4&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+Gollnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+Muller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia&doi=10.1038%2Fnm.2661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID γ (with interleukin-2 (IL-2) receptor γ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0lgSQE5DSxU3XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DGollner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26issue%3D4%26spage%3D605%26epage%3D611%26doi%3D10.1038%2Fnm.2661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of the histone lysine demethylase KDM1A suppresses the growth of multiple acute myeloid leukemia subtypes</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1975</span>– <span class="NLM_lpage">1988</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1158%2F0008-5472.CAN-15-2333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=26837761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlSktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1975-1988&issue=7&author=J.+P.+McGrathauthor=K.+E.+Williamsonauthor=S.+Balasubramanianauthor=S.+Odateauthor=S.+Aroraauthor=C.+Hattonauthor=T.+M.+Edwardsauthor=T.+O%E2%80%99Brienauthor=S.+Magnusonauthor=D.+Stokoeauthor=D.+L.+Danielsauthor=B.+M.+Bryantauthor=P.+Trojer&title=Pharmacological+inhibition+of+the+histone+lysine+demethylase+KDM1A+suppresses+the+growth+of+multiple+acute+myeloid+leukemia+subtypes&doi=10.1158%2F0008-5472.CAN-15-2333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Inhibition of the Histone Lysine Demethylase KDM1A Suppresses the Growth of Multiple Acute Myeloid Leukemia Subtypes</span></div><div class="casAuthors">McGrath, John P.; Williamson, Kaylyn E.; Balasubramanian, Srividya; Odate, Shobu; Arora, Shilpi; Hatton, Charlie; Edwards, Thomas M.; O'Brien, Thomas; Magnuson, Steven; Stokoe, David; Daniels, Danette L.; Bryant, Barbara M.; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1975-1988</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (KDM1A) is a transcriptional coregulator that can function in both the activation and repression of gene expression, depending upon context.  KDM1A plays an important role in hematopoiesis and was identified as a dependency factor in leukemia stem cell populations.  Therefore, we investigated the consequences of inhibiting KDM1A in a panel of cell lines representing all acute myelogenous leukemia (AML) subtypes using selective, reversible and irreversible KDM1A small-mol. inhibitors.  Cell models of AML, CML, and T-ALL were potently affected by KDM1A inhibition, and cells bearing RUNX1-RUNX1T1 (AML1-ETO) translocations were esp. among the most sensitive.  RNAi-mediated silencing of KDM1A also effectively suppressed growth of RUNX1-RUNX1T1-contg. cell lines.  Furthermore, pharmacol. inhibition of KDM1A resulted in complete abrogation of tumor growth in an AML xenograft model harboring RUNX1-RUNX1T1 translocations.  We unexpectedly found that KDM1A-targeting compds. not only inhibited the catalytic activity of the enzyme, but evicted KDM1A from target genes.  Accordingly, compd.-mediated KDM1A eviction was assocd. with elevated levels of local histone H3 lysine 4 dimethylation, and increased target gene expression, which was further accompanied by cellular differentiation and induction of cell death.  Finally, our finding that KDM1A inhibitors effectively synergize with multiple conventional as well as candidate anti-AML agents affords a framework for potential future clin. application.  Cancer Res; 76(7); 1975-88. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2rcSAA1n6MrVg90H21EOLACvtfcHk0lgSQE5DSxU3XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlSktbo%253D&md5=19efc6c725b7bf869749f90b2b2cec6e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2333%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%2BP.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DEdwards%26aufirst%3DT.%2BM.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DPharmacological%2520inhibition%2520of%2520the%2520histone%2520lysine%2520demethylase%2520KDM1A%2520suppresses%2520the%2520growth%2520of%2520multiple%2520acute%2520myeloid%2520leukemia%2520subtypes%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26issue%3D7%26spage%3D1975%26epage%3D1988%26doi%3D10.1158%2F0008-5472.CAN-15-2333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parimi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poropatich, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. J.</span></span> <span> </span><span class="NLM_article-title">Neuroendocrine differentiation of prostate cancer: a review</span>. <i>Am. J. Clin. Exp. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=25606573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvoslejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2014&pages=273-285&issue=4&author=V.+Parimiauthor=R.+Goyalauthor=K.+Poropatichauthor=X.+J.+Yang&title=Neuroendocrine+differentiation+of+prostate+cancer%3A+a+review"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroendocrine differentiation of prostate cancer: a review</span></div><div class="casAuthors">Parimi Vamsi; Goyal Rajen; Poropatich Kate; Yang Ximing J</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical and experimental urology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">273-85</span>
        ISSN:<span class="NLM_cas:issn">2330-1910</span>.
    </div><div class="casAbstract">Neuroendocrine cells are one of the epithelial populations in the prostate.  Neuroendocrine differentiation (NED) has been observed in prostate cancer.  In addition to small cell neuroendocrine carcinomas and carcinoid tumors of the prostate, prostatic adenocarcinomas may have NED.  The incidence and clinical relevance of NED in prostatic adenocarcinoma is not clearly understood because of conflicting results in the reported studies, and evaluation of NED is not routinely performed in clinical practice.  This review is an overall synthesis with an aim to develop a more comprehensive understanding and practical approach towards the current knowledge of neuroendocrine differentiation.  In this review we are stratifying these lesions into separate subtypes based on histologic parameters such as tumor morphology, neuroendocrine cell density and distribution and clinical parameters.  We also want to identify current controversies and confusing issues not totally resolved in this topic for further investigations.  Eventually a clearer understanding of this phenomenon and appropriate handling NED in prostate cancer will benefit clinical practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrIfVnOV1rD_tos8R3JUhXfW6udTcc2eb2q33mrRm307ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvoslejtw%253D%253D&md5=d2113a1b08865d57de600f5cf2a2e17e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParimi%26aufirst%3DV.%26aulast%3DGoyal%26aufirst%3DR.%26aulast%3DPoropatich%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%2BJ.%26atitle%3DNeuroendocrine%2520differentiation%2520of%2520prostate%2520cancer%253A%2520a%2520review%26jtitle%3DAm.%2520J.%2520Clin.%2520Exp.%2520Urol.%26date%3D2014%26volume%3D2%26issue%3D4%26spage%3D273%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&issue=1&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhmPTc3pqeQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26issue%3D1%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Magliulo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, S.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1A as a promising target in acute myeloid leukemia</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">255</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.3389%2Ffonc.2018.00255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=30073149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ks1GqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=255&author=D.+Magliuloauthor=R.+Bernardiauthor=S.+Messina&title=Lysine-specific+demethylase+1A+as+a+promising+target+in+acute+myeloid+leukemia&doi=10.3389%2Ffonc.2018.00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Specific Demethylase 1A as a Promising Target in Acute Myeloid Leukemia</span></div><div class="casAuthors">Magliulo Daniela; Magliulo Daniela; Bernardi Rosa; Messina Samantha</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">255</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous hematopoietic malignancy characterized by the accumulation of incompletely differentiated progenitor cells (blasts) in the bone marrow and blood, and by suppression of normal hematopoiesis.  It has recently become apparent that the AML genome is characterized by recurrent mutations and dysregulations in epigenetic regulators.  These mutations frequently occur before the onset of full blown leukemia, at the pre-leukemic phase, and persist in residual disease that remains after therapeutic intervention, thus suggesting that targeting the AML epigenome may help to eradicate minimal residual disease and prevent relapse.  Within the AML epigenome, lysine-specific demethylase 1 A (LSD1) is a histone demethylase that is found frequently overexpressed, albeit not mutated, in AML.  LSD1 is a required constituent of critical transcription repressor complexes like CoREST and nucleosome remodeling and deacetylase (NuRD), and abrogation of LSD1 expression results in impaired self-renewal and proliferation, and increased differentiation and apoptosis in AML models and primary cells, particularly in AMLs with MLL- and AML1-rearrangements, or erythroid and megakaryoblastic differentiation block.  On this basis, a number of LSD1 inhibitors have been developed in the past decade, and few of them are currently being tested in clinical trials for patients with AML, along with other malignancies.  To date, the most promising application of this therapeutic strategy appears to be combination therapy of LSD1 inhibitors with all-trans retinoic acid (ATRA) to reactivate myeloid differentiation in cells that are not spontaneously susceptible to ATRA treatment.  In this review, we provide an overview of LSD1 function in normal hematopoiesis and leukemia, and of the current clinical application of LSD1 inhibitors for the treatment of patients with AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZvJytNRFkvDC850By6mxpfW6udTcc2ebZ0-JKhhWwArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ks1GqtQ%253D%253D&md5=7aaddbc35e569064c3fa8da358299840</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00255%26sid%3Dliteratum%253Aachs%26aulast%3DMagliulo%26aufirst%3DD.%26aulast%3DBernardi%26aufirst%3DR.%26aulast%3DMessina%26aufirst%3DS.%26atitle%3DLysine-specific%2520demethylase%25201A%2520as%2520a%2520promising%2520target%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D255%26doi%3D10.3389%2Ffonc.2018.00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&issue=3&author=T.+Maesauthor=C.+Mascaroauthor=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0litYQPPIvHGDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26issue%3D3%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brohl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubitt, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, D. R.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of lysine-specific demethylase 1 (LSD1) and histone deacetylase (HDAC) alone and in combination in Ewing sarcoma cell lines</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e0222228</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0222228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1371%2Fjournal.pone.0222228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=31550266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitl2qsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=e0222228&issue=9&author=D.+Welchauthor=E.+Kahenauthor=B.+Fridleyauthor=A.+S.+Brohlauthor=C.+L.+Cubittauthor=D.+R.+Reed&title=Small+molecule+inhibition+of+lysine-specific+demethylase+1+%28LSD1%29+and+histone+deacetylase+%28HDAC%29+alone+and+in+combination+in+Ewing+sarcoma+cell+lines&doi=10.1371%2Fjournal.pone.0222228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of lysine-specific demethylase 1 (LSD1) and histone deacetylase (HDAC) alone and in combination in Ewing sarcoma cell lines</span></div><div class="casAuthors">Welch, Darcy; Kahen, Elliot; Fridley, Brooke; Brohl, Andrew S.; Cubitt, Christopher L.; Reed, Damon R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e0222228</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ewing Sarcoma (ES) is characterized by recurrent translocations between EWSR1 and members of the ETS family of transcription factors.  The transcriptional activity of the fusion oncoprotein is dependent on interaction with the nucleosome remodeling and deactylase (NuRD) co-repressor complex.  While inhibitors of both histone deacetylase (HDAC) and lysine-specific demethylase-1 (LSD1) subunits of the NuRD complex demonstrate single agent activity in preclin. models, combination strategies have not been investigated.  We selected 7 clin. utilized chemotherapy agents, or active metabolites thereof, for experimentation: doxorubicin, cyclophosphamide, vincristine, etoposide and irinotecan as well as the HDAC inhibitor romidepsin and the reversible LSD1 inhibitor SP2509.  All agents were tested at clin. achievable concns. in 4 ES cell lines.  All possible 2 drug combinations were then tested for potential synergy.  Order of addn. of second-line conventional combination therapy agents was tested with the addn. of SP2509.  In two drug expts., synergy was obsd. with several combinations, including when SP2509 was paired with topoisomerase inhibitors or romidepsin.  Addn. of SP2509 after treatment with second-line combination therapy agents enhanced treatment effect.  Our findings suggest promising combination treatment strategies that utilize epigenetic agents in ES.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPvzCMFjMDIrVg90H21EOLACvtfcHk0litYQPPIvHGDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitl2qsrrN&md5=fa61558bd91ef8bf7fff6c344ab8f7fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0222228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0222228%26sid%3Dliteratum%253Aachs%26aulast%3DWelch%26aufirst%3DD.%26aulast%3DKahen%26aufirst%3DE.%26aulast%3DFridley%26aufirst%3DB.%26aulast%3DBrohl%26aufirst%3DA.%2BS.%26aulast%3DCubitt%26aufirst%3DC.%2BL.%26aulast%3DReed%26aufirst%3DD.%2BR.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520histone%2520deacetylase%2520%2528HDAC%2529%2520alone%2520and%2520in%2520combination%2520in%2520Ewing%2520sarcoma%2520cell%2520lines%26jtitle%3DPLoS%2520One%26date%3D2019%26volume%3D14%26issue%3D9%26spage%3De0222228%26doi%3D10.1371%2Fjournal.pone.0222228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerenyi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luc, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e00633</span>, <span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0litYQPPIvHGDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M. P.</span></span> <span> </span><span class="NLM_article-title">Generation of a set of simple, interpretable ADMET rules of thumb</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">817</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1021/jm701122q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701122q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=817-34&issue=4&author=M.+P.+Gleeson&title=Generation+of+a+set+of+simple%2C+interpretable+ADMET+rules+of+thumb&doi=10.1021%2Fjm701122q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Set of Simple, Interpretable ADMET Rules of Thumb</span></div><div class="casAuthors">Gleeson, M. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">817-834</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A set of simple, consistent structure-property guides have been detd. from an anal. of a no. of key ADMET assays run within GSK: soly., permeability, bioavailability, vol. of distribution, plasma protein binding, CNS penetration, brain tissue binding, P-gp efflux, hERG inhibition, and cytochrome P 450 1A2/2C9/2C19/2D6/3A4 inhibition.  The rules have been formulated using mol. properties that chemists intuitively know how to alter in a mol., namely, mol. wt., logP, and ionization state.  The rules supplement the more predictive black-box models available to us by clearly illustrating the key underlying trends, which are in line with reports in the literature.  It is clear from the analyses reported herein that almost all ADMET parameters deteriorate with either increasing mol. wt., logP, or both, with ionization state playing either a beneficial or detrimental affect depending on the parameter in question.  This study reemphasizes the need to focus on a lower mol. wt. and logP area of physicochem. property space to obtain improved ADMET parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PTdPHoD-c7Vg90H21EOLACvtfcHk0lhRBXXL3TKr9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D&md5=42db5918ad8e9ea3ecf3a13846b3a434</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm701122q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701122q%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26atitle%3DGeneration%2520of%2520a%2520set%2520of%2520simple%252C%2520interpretable%2520ADMET%2520rules%2520of%2520thumb%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D4%26spage%3D817%26epage%3D34%26doi%3D10.1021%2Fjm701122q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Romano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filvaroff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, Z.</span></span> <span> </span><span class="NLM_article-title">Phase I study of CC-90011 in patients with advanced solid tumors and relapsed/refractory non-Hodgkin lymphoma (R/R NHL)</span>. <i>Ann. Oncol</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">i4</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;key=10.1093%2Fannonc%2Fmdz029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=i4&issue=Suppl.+1&author=A.+Hollebecqueauthor=J.+S.+de+Bonoauthor=R.+Plummerauthor=N.+Isambertauthor=P.+Martin-Romanoauthor=E.+Baudinauthor=S.+Moraauthor=E.+Filvaroffauthor=M.+Lambaauthor=Z.+Nikolova&title=Phase+I+study+of+CC-90011+in+patients+with+advanced+solid+tumors+and+relapsed%2Frefractory+non-Hodgkin+lymphoma+%28R%2FR+NHL%29&doi=10.1093%2Fannonc%2Fmdz029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz029%26sid%3Dliteratum%253Aachs%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DPlummer%26aufirst%3DR.%26aulast%3DIsambert%26aufirst%3DN.%26aulast%3DMartin-Romano%26aufirst%3DP.%26aulast%3DBaudin%26aufirst%3DE.%26aulast%3DMora%26aufirst%3DS.%26aulast%3DFilvaroff%26aufirst%3DE.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DNikolova%26aufirst%3DZ.%26atitle%3DPhase%2520I%2520study%2520of%2520CC-90011%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520and%2520relapsed%252Frefractory%2520non-Hodgkin%2520lymphoma%2520%2528R%252FR%2520NHL%2529%26jtitle%3DAnn.%2520Oncol%26date%3D2019%26volume%3D30%26issue%3DSuppl.%25201%26spage%3Di4%26doi%3D10.1093%2Fannonc%2Fmdz029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HKO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HKO','PDB','2HKO'); return false;">PDB: 2HKO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K','PDB','6W4K'); return false;">PDB: 6W4K</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00978">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81337"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00978?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00978</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Biology and pharmacology of <b>11</b>, crystallization methods, and purity information (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and standard error of measurement for all biological data present (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_001.pdf">jm0c00978_si_001.pdf (532.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00978/suppl_file/jm0c00978_si_002.csv">jm0c00978_si_002.csv (0.77 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates and experimental data upon article publication (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a> for compound <b>11</b> bound to LSD1/CoREST).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00978%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00978" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679955eece2524cb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
