
<html lang="en"     class="pb-page"  data-request-id="2dc3d71e-40e7-4050-a418-364fd228c5b2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2016.59.issue-16;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.6b00906"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles" /></meta><meta name="dc.Creator" content="Dagmar  Stumpfe" /></meta><meta name="dc.Creator" content="Dilyana  Dimova" /></meta><meta name="dc.Creator" content="Jürgen  Bajorath" /></meta><meta name="dc.Description" content="A computational methodology is introduced for detecting all unique series of analogs in large compound data sets, regardless of chemical relationships between analogs. No prior knowledge of core st..." /></meta><meta name="Description" content="A computational methodology is introduced for detecting all unique series of analogs in large compound data sets, regardless of chemical relationships between analogs. No prior knowledge of core st..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 8, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00906" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00906" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00906" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00906" /></link>
        
    
    

<title>Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00906" /></meta><meta property="og:title" content="Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0006.jpeg" /></meta><meta property="og:description" content="A computational methodology is introduced for detecting all unique series of analogs in large compound data sets, regardless of chemical relationships between analogs. No prior knowledge of core structures or R-groups is required, which are automatically determined. The approach is based upon the generation of retrosynthetic matched molecular pairs and analog networks from which distinct series are isolated. The methodology was applied to systematically extract more than 17 000 distinct series from the ChEMBL database. For comparison, analog series were also isolated from screening compounds and drugs. Known biological activities were mapped to series from ChEMBL, and in more than 13 000 of these series, key compounds were identified that represented substitution sites of all analogs within a series and its complete activity profile. The analog series, key compounds, and activity profiles are made freely available as a resource for medicinal chemistry applications." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00906"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00906">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00906&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00906&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00906&amp;href=/doi/10.1021/acs.jmedchem.6b00906" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7667-7676</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00879" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00247" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dagmar++Stumpfe">Dagmar Stumpfe</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dilyana++Dimova">Dilyana Dimova</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%BCrgen++Bajorath">Jürgen Bajorath</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Dahlmannstrasse 2, D-53113 Bonn, Germany</span></div><div class="corresp-info"><strong>*</strong>Phone: +49-228-2699-306. Fax: +49-228-2699-341. E-mail: <a href="/cdn-cgi/l/email-protection#adcfccc7c2dfccd9c5edcfc4d983d8c3c480cfc2c3c383c9c8"><span class="__cf_email__" data-cfemail="ceacafa4a1bcafbaa68eaca7bae0bba0a7e3aca1a0a0e0aaab">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00906&amp;href=/doi/10.1021%2Facs.jmedchem.6b00906" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7667–7676</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 8, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 June 2016</li><li><span class="item_label"><b>Published</b> online</span>8 August 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 August 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00906" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00906</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7667%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDagmar%2BStumpfe%252C%2BDilyana%2BDimova%252C%2BJ%25C3%25BCrgen%2BBajorath%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D16%26contentID%3Dacs.jmedchem.6b00906%26title%3DComputational%2BMethod%2Bfor%2Bthe%2BSystematic%2BIdentification%2Bof%2BAnalog%2BSeries%2Band%2BKey%2BCompounds%2BRepresenting%2BSeries%2Band%2BTheir%2BBiological%2BActivity%2BProfiles%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7676%26publicationDate%3DAugust%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00906"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1207</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">39</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00906" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dagmar&quot;,&quot;last_name&quot;:&quot;Stumpfe&quot;},{&quot;first_name&quot;:&quot;Dilyana&quot;,&quot;last_name&quot;:&quot;Dimova&quot;},{&quot;first_name&quot;:&quot;Jürgen&quot;,&quot;last_name&quot;:&quot;Bajorath&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;7667-7676&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00906&quot;},&quot;abstract&quot;:&quot;A computational methodology is introduced for detecting all unique series of analogs in large compound data sets, regardless of chemical relationships between analogs. No prior knowledge of core structures or R-groups is required, which are automatically determined. The approach is based upon the generation of retrosynthetic matched molecular pairs and analog networks from which distinct series are isolated. The methodology was applied to systematically extract more than 17 000 distinct series from the ChEMBL database. For comparison, analog series were also isolated from screening compounds and drugs. Known biological activities were mapped to series from ChEMBL, and in more than 13 000 of these series, key compounds were identified that represented substitution sites of all analogs within a series and its complete activity profile. The analog series, key compounds, and activity profiles are made freely available as a resource for medicinal chemistry applications.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00906&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00906" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00906&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00906" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00906&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00906" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00906&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00906&amp;href=/doi/10.1021/acs.jmedchem.6b00906" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00906" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00906" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00906%26sid%3Dliteratum%253Aachs%26pmid%3D27501131%26genre%3Darticle%26aulast%3DStumpfe%26date%3D2016%26atitle%3DComputational%2BMethod%2Bfor%2Bthe%2BSystematic%2BIdentification%2Bof%2BAnalog%2BSeries%2Band%2BKey%2BCompounds%2BRepresenting%2BSeries%2Band%2BTheir%2BBiological%2BActivity%2BProfiles%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D16%26spage%3D7667%26epage%3D7676%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291583" title="Mathematical methods">Mathematical methods</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/jmcmar.2016.59.issue-16/20160825/jmcmar.2016.59.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A computational methodology is introduced for detecting all unique series of analogs in large compound data sets, regardless of chemical relationships between analogs. No prior knowledge of core structures or R-groups is required, which are automatically determined. The approach is based upon the generation of retrosynthetic matched molecular pairs and analog networks from which distinct series are isolated. The methodology was applied to systematically extract more than 17 000 distinct series from the ChEMBL database. For comparison, analog series were also isolated from screening compounds and drugs. Known biological activities were mapped to series from ChEMBL, and in more than 13 000 of these series, key compounds were identified that represented substitution sites of all analogs within a series and its complete activity profile. The analog series, key compounds, and activity profiles are made freely available as a resource for medicinal chemistry applications.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02232" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02232" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Series of structurally analogous compounds produced in the course of chemical optimization are a natural focal point of medicinal chemistry. From analogs, structure–activity relationships (SARs) are derived to guide further compound optimization.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In the practice of medicinal chemistry, analog series are mostly generated on a case-by-case basis and conventional R-group tables continue to be the prevalent data format for monitoring evolving series and associated SAR information. Quantitative SAR (QSAR) analysis<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> has traditionally been used as a computational approach to support analog design by predicting the activity of new analogs prior to synthesis. Other computational methods have been developed to further extend the R-group table format<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> or hierarchically organize analog sets and visualize SAR patterns associated with large series.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> Furthermore, computational concepts have recently been introduced to monitor SAR progression during compound optimization.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a></div><div class="NLM_p">While series-centric efforts continue to dominate the generation of analogs in medicinal chemistry, only very few attempts have thus far been made to globally view accessible analog space and systematically search for analog series in compound repositories. In particular, no efficient computational approach is presently available to extract all available series from compound collections. From a computational perspective, a comprehensive search for analog series among structurally heterogeneous compounds is a nontrivial task. The situation differs for individual compounds. For example, a chemically intuitive way to detect analogs of a query compound is the calculation of matched molecular pairs (MMPs),<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> which are defined as pairs of compounds that only differ by a structural change at a single site (i.e., the exchange of a pair of substructures). As an extension of the MMP concept, matching molecular series (MMSs) were introduced as series of compounds that are only distinguished by chemical modifications at a single site.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Accordingly, MMSs can also be used to represent analog series.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For the generation of MMPs and MMSs, elegant and computationally efficient molecular fragmentation algorithms are available.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Furthermore, retrosynthetic combinatorial analysis procedure (RECAP) rules<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> have been applied to generate MMPs.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Following this approach, retrosynthetic rules replace random fragmentation of bonds. These RECAP-MMPs<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> are synthetically accessible and thus further increase the relevance of MMPs for the exploration of chemical relationships.</div><div class="NLM_p">An MMP-based analog search was carried out in the ChEMBL database,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> the major public repository for compounds and activity data from medicinal chemistry, for preselected query compounds.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> For each query, available analogs were identified. A surprising finding was that for more than a third of active compounds taken from ChEMBL, no analogs were detected.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Furthermore, the promiscuity of analogs within a set was determined and compared. Compound promiscuity was defined as the number of targets a compound was reported to be active against.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> So-defined promiscuity represents the molecular basis of polypharmacology<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19-21)</a> and is distinct from undesired promiscuity due to reactivity of compounds or other assay artifacts.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> For about three-quarters of all sets, identified analogs were found to have the same degree of promiscuity as the query compound.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">While an MMP-based analog search is useful to detect available analogs for given compounds that differ by modifications at a single site, it does not identify and distinguish between unique series of analogs taking all chemical modifications and compound relationships into account. This task goes much beyond individual analog searches.</div><div class="NLM_p">Herein we present, to our knowledge, a first generally applicable methodology to extract all available analog series from compound sources. The approach combines the generation of retrosynthetic MMPs and analog networks in which unique series emerge as disjoint clusters that can be isolated and visualized.</div><div class="NLM_p last">In a large-scale analysis of analog space across ChEMBL, 17 371 distinct series were identified and their target annotations studied. Rather unexpectedly, compounds were frequently detected that captured all analog relationships within a newly identified series, including different sites of chemical diversification, and shared all reported biological activities. Thus, these compounds were series-representative from a chemical and biological viewpoint. They delineate currently accessible analog space and provide a previously unconsidered source of knowledge for analog mapping and design including the consideration of multitarget activities. Herein we introduce the computational methodology and report the results of our large-scale application.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Methods and Materials</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Compound Activity Data</h3><div class="NLM_p last">Compounds and activity data were assembled from ChEMBL,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> version 21. Only compounds involved in direct interactions (target relationship type “D”) with human targets at the highest confidence level (target confidence score 9) were taken. Two different types of potency measurements were separately considered including assay-independent equilibrium constants (<i>K</i><sub>i</sub> values) and assay-dependent IC<sub>50</sub> values. Approximate measurements associated with “>”, “<”, or “∼” were discarded. If a compound had multiple <i>K</i><sub>i</sub> or IC<sub>50</sub> values for the same target, the geometric mean of the values was calculated as the final potency annotation. However, the mean was only used if all values fell into the same order of magnitude; if not, the values were discarded. Given these selection criteria, a total of 167 290 unique compounds were collected with activity against a total of 1594 targets. For comparison, analog series were also extracted from active screening compounds available in PubChem<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and drugs from DrugBank.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> From PubChem, a total of 304 956 compounds with available structures and activity against 772 targets were obtained, and from DrugBank, 6225 experimental or approved drugs directed against 4029 targets were obtained. Calculations were performed using in-house scripts with the aid of KNIME<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> protocols and the OpenEye chemistry toolkit.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Retrosynthetic MMPs</h3><div class="NLM_p last">RECAP-MMPs<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (in the following simply referred to as MMPs) were systematically generated for selected compounds using an adaptation of the method of Hussain and Rea<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> to fragment exocyclic single bonds according to retrosynthetic rules. Previously established fragment size restrictions were applied to limit MMPs to pairs of compounds consisting of typical analogs.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Specifically, the size of the conserved MMP core structure was required to be at least twice the size of the larger substituent, which was permitted to consist of at most 13 heavy atoms. These restrictions were applied to ensure that substituents were limited in size to maximally a condensed two-ring system with no more than three additional atoms, consistent with largest substitutions typically seen in analog series from medicinal chemistry.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Analog Series</h3><div class="NLM_p">Unique analog series were identified in a three-step procedure. First, for each compound, all MMPs were collected and the union of compounds involved in MMP relationships was determined, yielding 96 889 compounds (62.9%). Second, a global “analog network” was generated in which all qualifying compounds were nodes and edges indicated pairwise MMP relationships. Third, from the network, all disjoint clusters were isolated. Each cluster represented a unique series of analogs covering all available substitution sites and R-groups. <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> illustrates the methodology. Network representations were drawn with Cytoscape.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Identification of analog series. The computational approach to systematically identify distinct analog series is illustrated that combines MMP and network analysis, as described in the text. The cluster on the right (thick frame) resulting from the union of different compound-based analog subsets represents a disjoint cluster (unique series) from the global analog network.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Key Compounds</h3><div class="NLM_p">For compounds comprising each cluster, substitution sites were recorded and target annotations collected. A search was carried out for individual compounds that were substituted at all sites and active against all targets other cluster members were active against. The union of all target annotations represented the activity profile of a cluster. If available, such compounds were considered representatives (key compounds) of given analog series.</div><div class="NLM_p last">All analog series and key compounds are made freely available via an open access deposition on the Zenodo platform.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Methodological Concept</h3><div class="NLM_p last">We introduce an intuitive and effective computational methodology to extract distinct analog series from compound data sets of any source. The approach is outlined in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. Compound-based analog searches can be carried out in different ways, for example, by simple substructure searching using predefined scaffolds or, more objectively and systematically, by applying the MMP formalism, as discussed above. In order to further increase the chemical relevance of computationally identified analogs, core–substituent size relationships and retrosynthetic rules were applied for MMP generation. MMP calculations make it possible to identify all pairwise analog relationships for query compounds, but this is not sufficient to determine unique series. Therefore, pairwise relationships must be systematically organized, which is accomplished through the generation of a global network where compounds are represented as nodes and connected to each other if they form an MMP relationship. In the network, unique series of analogs emerge as disjoint clusters. Importantly, a cluster contains all compounds sharing a given core structure, all chemically explored substitution sites, and currently available R-groups. The core structure is not predefined but results from comprehensive molecular comparisons. Hence, no prior knowledge or structural definitions are required, which also sets the methodology apart from scaffold-based approaches. As illustrated in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, a three-step process is applied to identify unique series. In the first step, each compound is used once as a query to generate all possible compound-based analog subsets. In the second step, overlapping subsets are combined and the global network structure is constructed. In the third step, all disjoint clusters representing individual series are identified. A compound can only belong to one series. The network structure is computationally analyzed by searching the underlying relationship matrix and can be visualized in part, e.g., by focusing on sections or individual clusters, or as a whole. Visualization further supports the analysis of analog series, which can be applied to screening or optimization data sets of any source. If analog series are extracted from large compound collections with activities against many different targets, as presented in the following, target annotations of compounds are mapped to individual clusters once the structural organization has been finalized. This makes it possible to further distinguish between analog series with single- or multitarget activities and identify key compounds with high SAR information content.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Searching for Analog Series</h3><div class="NLM_p">In a large-scale application of the method described above, we have systematically searched for analog series in 167 290 ChEMBL compounds for which well-defined activity data were available. These compounds were active against a total of 1594 targets. Retrosynthetic analog relationships were detected for 96 889 compounds with activity against 1382 targets. The resulting global analog network consisted of a total of 17 371 clusters, each of which represented a unique series of analogs. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> shows exemplary clusters with different topology from the global analog network. Clusters included cliques and a variety of sparsely connected clusters. In cliques, all nodes are connected to each other; i.e., all possible pairwise MMP relationships are formed. Approximately 47% of cluster containing at least three analogs were cliques. <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> reports the cluster distribution. The number of compounds per series ranged from two (7815 analog pairs were detected) to more than 300 (in a single instance). There were 7498 series with three to 10 analogs (containing a total of 36 246 compounds), 2042 series with 11 to 100 analogs (with a total of 42 230 compounds), and 16 series within more than 100 analogs (a total of 2783 compounds). Hence, large numbers of series with up to 10 and also with 10 to 100 compounds were identified, which provide a rich source of SAR information.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cluster view of an analog series. Two exemplary clusters from the global analog network with different topology are shown. Nodes are color-coded according to activity profiles. The cluster on the left is a clique (completely connected subgraph) consisting of 10 inhibitors of MAP kinase p38α and tyrosine-protein kinase LCK. All inhibitors are active against one or the other target (single-target activity). The cluster on the right contains ligands of dopamine, norepinephrine, and serotonin transporters as well as the histamine H3 receptor. In this case, all analogs are promiscuous. From each cluster, two exemplary analogs are shown and substitution sites highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Network Statistics<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">no. targets</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cluster order</th><th class="colsep0 rowsep0" align="center" char=".">no. clusters (series)</th><th class="colsep0 rowsep0" align="center" char=".">no. CPDs</th><th class="colsep0 rowsep0" align="center" char=".">no. clusters with key CPDs</th><th class="colsep0 rowsep0" align="center" char=".">no. key CPDs</th><th class="colsep0 rowsep0" align="center" char=".">median</th><th class="colsep0 rowsep0" align="center">min–max</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">7815</td><td class="colsep0 rowsep0" align="char" char=".">15630</td><td class="colsep0 rowsep0" align="char" char=".">7056</td><td class="colsep0 rowsep0" align="char" char=".">12916</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">1–40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3–5</td><td class="colsep0 rowsep0" align="char" char=".">5222</td><td class="colsep0 rowsep0" align="char" char=".">19135</td><td class="colsep0 rowsep0" align="char" char=".">4121</td><td class="colsep0 rowsep0" align="char" char=".">10407</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">1–57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6–10</td><td class="colsep0 rowsep0" align="char" char=".">2276</td><td class="colsep0 rowsep0" align="char" char=".">17111</td><td class="colsep0 rowsep0" align="char" char=".">1345</td><td class="colsep0 rowsep0" align="char" char=".">5671</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="left">1–49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11–20</td><td class="colsep0 rowsep0" align="char" char=".">1369</td><td class="colsep0 rowsep0" align="char" char=".">19840</td><td class="colsep0 rowsep0" align="char" char=".">544</td><td class="colsep0 rowsep0" align="char" char=".">3646</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="left">1–38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">21–50</td><td class="colsep0 rowsep0" align="char" char=".">604</td><td class="colsep0 rowsep0" align="char" char=".">17816</td><td class="colsep0 rowsep0" align="char" char=".">114</td><td class="colsep0 rowsep0" align="char" char=".">1153</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="left">1–63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">51–100</td><td class="colsep0 rowsep0" align="char" char=".">69</td><td class="colsep0 rowsep0" align="char" char=".">4574</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="left">1–47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">101–200</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">1890</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="left">1–130</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">201–300</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">557</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">20.5</td><td class="colsep0 rowsep0" align="left">1–40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">>300</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">336</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="left">4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">The distribution of clusters, compounds (CPDs), and targets across the global analog network is reported. Each cluster represents a unique series of analogs. “Cluster order” gives the number of compounds per cluster. For each cluster order, the total number of key compounds and clusters with key compounds, the median number of target annotations, and the minimum and maximum (min–max) are provided.</p></div></div></div><div class="NLM_p">As a control calculation, compounds were also first retrieved on a per-target basis and then organized. This is often done when extracting active compounds from databases. Compared to our approach, the target-based retrieval was accompanied by a highly significant loss of information. Specifically, between one and 4570 analogs remained undetected for series originating from a total of 1006 target sets. Depending on the set, the missed analogs were active against one to 217 additional targets.</div><div class="NLM_p">For comparison, we also isolated analog series from 304 956 screening compounds from PubChem and 6225 drugs from DrugBank. From PubChem compounds, 32 375 analog series were extracted comprising 160 446 compounds. In addition, drugs yielded 441 analog sets with 1209 compounds. Hence, a large number of analog series was also obtained from PuBChem.</div><div class="NLM_p last">We determined the overlap of analog series from different sources. There was essentially no overlap between ChEMBL and DrugBank. Only one series from ChEMBL and one series from DrugBank shared a single compound. Similarly, three series from PubChem and three from DrugBank had four compounds in common. By contrast, 1204 series from ChEMBL and 1156 series from PubChem shared a total of 3033 compounds, reflecting the fact that there is data exchange between these large repositories.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Activity Profiles</h3><div class="NLM_p last"><a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> also reports target annotations for series of increasing size. For series with up to 50 compounds, median numbers of targets per series were only one or two, although between 38 and 63 target annotations were detected for individual series, depending on their size. For series with up to 100 and 200 compounds the median numbers of targets increased to three and four, respectively, with individual series having more than 100 target annotations. Thus, many series, especially series with up to 50 analogs, were active against single targets, but multitarget activities were also frequently observed.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Key Compounds</h3><div class="NLM_p">We also searched for compounds that covered all substitution sites within a series and all target annotations of analogs. These compounds were considered series-representative. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> illustrates the concept of key compounds for two small analog series. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a shows inhibitors of matrix metalloproteases, all of which share activity against matrix metalloprotease 12. Three analogs are also active against protease 13, and the key compound in the center is additionally active against 2, 9, and 14. The series consists of three subsets with varying substitution sites, all of which are present in the key compound. The series in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b also consists of three subsets, each of which contains a pair of inhibitors of different subunits of PI3-kinase p110. The key compound in the center combines the activity of the other analogs against the α, β, γ, and δ subunits.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analog series and key compounds. Venn diagrams show two small series of analogs with key compounds in the center that cover all MMP relationships and substitution sites within a series as well as all target annotations. In (a) and (b), inhibitors of different matrix metalloproteases and PI3 kinase p110 subunits are displayed, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A total of 33 793 key compounds were identified in 13 180 series with activity against 1120 targets (81.0%). Thus, there were often multiple key compounds per series. <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> reports the distribution of key compounds over series of increasing size. For series with three to five and six to 10 analogs, 10 407 and 5671 key compounds were detected, respectively. Larger series with more than 10–50 compounds still yielded a total of 4799 key compounds. For series with more than 50 analogs, no key compounds were detected.</div><div class="NLM_p">The overall large number of key compounds can be rationalized by considering analog series having only one or small numbers of substitution sites and activity against single targets. In these frequently occurring cases, multiple key compounds were typically present.</div><div class="NLM_p">By definition, key compounds are the most promiscuous analogs within a series, since no compound can have a higher degree of promiscuity. <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> reports the distribution of compounds and key compounds with increasing degrees of promiscuity. The majority of compounds (67 132) were only active against a single target, which also included 23 958 key compounds. In addition, 6018 key compounds were active against two targets and 2258, 1005, and 293 key compounds against three, four, and five targets, respectively. By contrast, only very small numbers of compounds had more than 10 target annotations. Taken together, the results in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> reveal that individual key compounds frequently characterized relatively large analog series with multitarget activities.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Promiscuity Degree of Key Compounds<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no. targets</th><th class="colsep0 rowsep0" align="center" char=".">no. CPDs</th><th class="colsep0 rowsep0" align="center" char=".">no. key CPDs</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">67132</td><td class="colsep0 rowsep0" align="char" char=".">23958</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">17893</td><td class="colsep0 rowsep0" align="char" char=".">6018</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">7205</td><td class="colsep0 rowsep0" align="char" char=".">2258</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">2831</td><td class="colsep0 rowsep0" align="char" char=".">1005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char=".">799</td><td class="colsep0 rowsep0" align="char" char=".">293</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">≥6, ≤10</td><td class="colsep0 rowsep0" align="char" char=".">823</td><td class="colsep0 rowsep0" align="char" char=".">217</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">≥11, ≤20</td><td class="colsep0 rowsep0" align="char" char=".">160</td><td class="colsep0 rowsep0" align="char" char=".">34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">≥21, ≤30</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">≥31, ≤40</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">≥41, ≤68</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The distribution of all compounds (CPDs) and key compounds over different promiscuity degrees (no. targets) is reported.</p></div></div></div><div class="NLM_p last">Only 3075 of the more than 32 000 analog series isolated from PubChem contained key compounds. A total of 3993 key compounds (2.4%) were identified, a much lower proportion than determined for ChEMBL, which might be expected for screening data. Furthermore, 185 of the 441 series from DrugBank contained 304 key compounds (25.1%).</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Targets</h3><div class="NLM_p">It is emphasized that the methodology does not attempt to predict novel targets of active compounds. Rather, available target annotations of compounds are systematically mapped to analog series. <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> reports targets with known active compounds present in many different analog series (with single activity or multitarget activity). Not surprisingly, popular therapeutic targets including, among others, a variety of G protein coupled receptors covered most analog series. The majority of these series contained one or more key compounds, often with multitarget activities.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Targets with Extensive Representation across Analog Series<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center" char=".">no. CPDs</th><th class="colsep0 rowsep0" align="center" char=".">no. analog series</th><th class="colsep0 rowsep0" align="center" char=".">no. key CPDs</th><th class="colsep0 rowsep0" align="center" char=".">no. analog series with key CPDs</th><th class="colsep0 rowsep0" align="center">additional targets</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serotonin transporter</td><td class="colsep0 rowsep0" align="char" char=".">1489</td><td class="colsep0 rowsep0" align="char" char=".">417</td><td class="colsep0 rowsep0" align="char" char=".">597</td><td class="colsep0 rowsep0" align="char" char=".">274</td><td class="colsep0 rowsep0" align="left">0–38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cannabinoid CB1 receptor</td><td class="colsep0 rowsep0" align="char" char=".">2007</td><td class="colsep0 rowsep0" align="char" char=".">397</td><td class="colsep0 rowsep0" align="char" char=".">716</td><td class="colsep0 rowsep0" align="char" char=".">292</td><td class="colsep0 rowsep0" align="left">0–5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dopamine D2 receptor</td><td class="colsep0 rowsep0" align="char" char=".">1562</td><td class="colsep0 rowsep0" align="char" char=".">380</td><td class="colsep0 rowsep0" align="char" char=".">544</td><td class="colsep0 rowsep0" align="char" char=".">244</td><td class="colsep0 rowsep0" align="left">0–38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cannabinoid CB2 receptor</td><td class="colsep0 rowsep0" align="char" char=".">1901</td><td class="colsep0 rowsep0" align="char" char=".">378</td><td class="colsep0 rowsep0" align="char" char=".">639</td><td class="colsep0 rowsep0" align="char" char=".">281</td><td class="colsep0 rowsep0" align="left">0–5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cytochrome P450 3A4</td><td class="colsep0 rowsep0" align="char" char=".">1046</td><td class="colsep0 rowsep0" align="char" char=".">365</td><td class="colsep0 rowsep0" align="char" char=".">326</td><td class="colsep0 rowsep0" align="char" char=".">182</td><td class="colsep0 rowsep0" align="left">0–31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">adenosine A2a receptor</td><td class="colsep0 rowsep0" align="char" char=".">2428</td><td class="colsep0 rowsep0" align="char" char=".">345</td><td class="colsep0 rowsep0" align="char" char=".">472</td><td class="colsep0 rowsep0" align="char" char=".">210</td><td class="colsep0 rowsep0" align="left">0–7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carbonic anhydrase II</td><td class="colsep0 rowsep0" align="char" char=".">1101</td><td class="colsep0 rowsep0" align="char" char=".">341</td><td class="colsep0 rowsep0" align="char" char=".">391</td><td class="colsep0 rowsep0" align="char" char=".">184</td><td class="colsep0 rowsep0" align="left">0–11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">vascular endothelial growth factor receptor 2</td><td class="colsep0 rowsep0" align="char" char=".">1591</td><td class="colsep0 rowsep0" align="char" char=".">338</td><td class="colsep0 rowsep0" align="char" char=".">590</td><td class="colsep0 rowsep0" align="char" char=".">204</td><td class="colsep0 rowsep0" align="left">0–18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serotonin 6 (5-HT6) receptor</td><td class="colsep0 rowsep0" align="char" char=".">1151</td><td class="colsep0 rowsep0" align="char" char=".">332</td><td class="colsep0 rowsep0" align="char" char=".">561</td><td class="colsep0 rowsep0" align="char" char=".">244</td><td class="colsep0 rowsep0" align="left">0–38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">adenosine A3 receptor</td><td class="colsep0 rowsep0" align="char" char=".">1946</td><td class="colsep0 rowsep0" align="char" char=".">324</td><td class="colsep0 rowsep0" align="char" char=".">391</td><td class="colsep0 rowsep0" align="char" char=".">182</td><td class="colsep0 rowsep0" align="left">0–38</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Reported are the top 10 targets with active compounds participating in the largest numbers of analog series. In each case, the total number of compounds, key compounds, and additional targets per series (min–max) is also provided.</p></div></div></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Prospective Applications</h3><div class="NLM_p">Our methodology can be prospectively applied in different ways. As described above in detail, a systematic search for all available analog series can be carried out in compound collections of any source, including in-house compound decks, leading to a global structural organization and the identification of all existing series. Then activity data can be mapped to these series. Furthermore, another immediate application of the methodology is the search for available analogs and activity data during the early stages of compound optimization campaigns. When only a few analogs are available, or even a single active compound, the method can be effectively used to search any compound database for all available analogs and associated activity information. This might reveal important information during early project stages, as illustrated in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. In the example in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a, a global analog search was carried out in ChEMBL for a single inhibitor of acetylcholinesterase. A total of 13 analogs were identified, five of which were also known inhibitors of acetylcholinesterase. In addition, three analogs were detected with activity against monoamine oxidase B and four others with dual activity against monoamine oxidase A and B. Furthermore, another analog was found to be active against an unrelated target, P-glycoprotein 1. In the example in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b, a global search was carried out for a small series of kinase inhibitors with activity against Aurora A and B. In this case, the series-based search also identified several other analogs with similar activity, one analog active against another kinase, and three others that were exclusively annotated with HERG antitarget activity, thus pointing at potential liabilities associated with the evolving series, which would merit careful follow-up investigation.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0005.jpeg" id="GRAPHIC-d7e861-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Global analog search. A prospective application of the computational approach introduced herein is illustrated. Global analog search calculations are carried out in ChEMBL for a (a) single query compound and (b) small series of analogs. In each case, identified analogs are shown and color-coded by known targets. For each target, the target name and the ChEMBL target identification (in parentheses) are given.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0004.jpeg jm-2016-00906c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">These examples illustrate that in addition to identifying all available structural analogs for given query compounds or series, the searches might also yield activity information for evolving series that is of immediate relevance for compound optimization.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In the practice of medicinal chemistry, analog series are typically defined on a case-by-case basis, which is sufficient for many applications. The study of large individual series can be computationally supported, for example, by applying the scaffold or maximum common substructure concepts, which provide formally consistent reference frames for structural analysis but have limited chemical relevance. Herein we have introduced a generally applicable computational approach to extract analog series from large compound data sets, which combines MMP and molecular network analysis and enables visualization of clusters representing different series. By design, the methodology is not series-centric but operates at a global level. It comprehensively and consistently accounts for all analog relationships that can be captured by the further extended and retrosynthetic MMP formalism. Following this approach, prior knowledge of core structures or substitution sites is not required, which are automatically determined on the basis of comprehensive compound comparisons, applying retrosynthetic rules and size restrictions for the generation of R-groups. Analog series can be extracted from data sets with multitarget activities, as reported herein, and key compounds be identified that contain substitution site and activity information representative for an entire series. In our analysis of the current release of ChEMBL, more than 17 000 analog series with varying composition, size, and activity annotations were identified. Rather unexpectedly, more than 13 000 of these series contained key compounds. As defined herein, key compounds delineate currently accessible analog space. All identified analog series and key compounds from ChEMBL, PubChem, and DrugBank are made freely available in an organized form as a part of our study,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> which provides a substantial knowledge base for different practical applications. For example, individual compounds can be searched against these series. Alternatively, global analog searches can be carried out for given queries in any compound repositories. Matching analogs provide hit expansion candidates and SAR information. Moreover, for a target of interest, all currently available series of analogs can be obtained, the SAR information associated with these series analyzed, and multitarget activities of compounds considered.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00906" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jürgen Bajorath</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Dahlmannstrasse 2, D-53113 Bonn, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5c3e3d36332e3d28341c3e352872293235713e333232723839"><span class="__cf_email__" data-cfemail="f99b9893968b988d91b99b908dd78c9790d49b969797d79d9c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dagmar Stumpfe</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Dahlmannstrasse 2, D-53113 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dilyana Dimova</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Dahlmannstrasse 2, D-53113 Bonn, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The contributions of D.S. and D.D. should be considered equal.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e880-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The use of OpenEye’s toolkits was made possible by their free academic licensing program. D.S. is supported by Sonderforschungsbereich 704 of the Deutsche Forschungsgemeinschaft.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i20" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i20"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i21" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i21"> Abbreviations Used</h2><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matched molecular pair</p></td></tr><tr><td class="NLM_term">MMS</td><td class="NLM_def"><p class="first last">matching molecular series</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">RECAP</td><td class="NLM_def"><p class="first last">retrosynthetic combinatorial analysis procedure</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span> <span class="citation_source-book">The Practice of Medicinal Chemistry</span>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_contrib-group">Wermuth, C. G.</span>, Ed.; <span class="NLM_publisher-name">Academic Press/Elsevier</span>: <span class="NLM_publisher-loc">San Diego, CA, U.S./London, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+G.+Wermuth&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press%252FElsevier%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Cherkasov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourches, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskin, I. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dearden, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramatica, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todeschini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consonni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuz’min, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benigni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rathman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terfloth, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">QSAR Modeling: Where Have You Been? Where Are You Going To?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4977</span><span class="NLM_x">–</span> <span class="NLM_lpage">5010</span><span class="refDoi"> DOI: 10.1021/jm4004285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4004285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4977-5010&author=A.+Cherkasovauthor=E.+N.+Muratovauthor=D.+Fourchesauthor=A.+Varnekauthor=I.+I.+Baskinauthor=M.+Croninauthor=J.+Deardenauthor=P.+Gramaticaauthor=Y.+C.+Martinauthor=R.+Todeschiniauthor=V.+Consonniauthor=V.+E.+Kuz%E2%80%99minauthor=R.+Cramerauthor=R.+Benigniauthor=C.+Yangauthor=J.+Rathmanauthor=L.+Terflothauthor=J.+Gasteigerauthor=A.+Richardauthor=A.+Tropsha&title=QSAR+Modeling%3A+Where+Have+You+Been%3F+Where+Are+You+Going+To%3F&doi=10.1021%2Fjm4004285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling: Where Have You Been? Where Are You Going To?</span></div><div class="casAuthors">Cherkasov, Artem; Muratov, Eugene N.; Fourches, Denis; Varnek, Alexandre; Baskin, Igor I.; Cronin, Mark; Dearden, John; Gramatica, Paola; Martin, Yvonne C.; Todeschini, Roberto; Consonni, Viviana; Kuzmin, Victor E.; Cramer, Richard; Benigni, Romualdo; Yang, Chihae; Rathman, James; Terfloth, Lothar; Gasteiger, Johann; Richard, Ann; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4977-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quant. structure-activity relationship modeling is one of the major computational tools employed in medicinal chem.  However, throughout its entire history it has drawn both praise and criticism concerning its reliability, limitations, successes, and failures.  In this paper, we discuss (i) the development and evolution of QSAR; (ii) the current trends, unsolved problems, and pressing challenges; and (iii) several novel and emerging applications of QSAR modeling.  Throughout this discussion, we provide guidelines for QSAR development, validation, and application, which are summarized in best practices for building rigorously validated and externally predictive QSAR models.  We hope that this Perspective will help communications between computational and exptl. chemists toward collaborative development and use of QSAR models.  We also believe that the guidelines presented here will help journal editors and reviewers apply more stringent scientific stds. to manuscripts reporting new QSAR studies, as well as encourage the use of high quality, validated QSARs for regulatory decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbLWydJbspbLVg90H21EOLACvtfcHk0lh-ylz2O8UbbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE&md5=a2d371ee33fbecf48ef59ecf914e4aa8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm4004285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4004285%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDearden%26aufirst%3DJ.%26aulast%3DGramatica%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DKuz%25E2%2580%2599min%26aufirst%3DV.%2BE.%26aulast%3DCramer%26aufirst%3DR.%26aulast%3DBenigni%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DRathman%26aufirst%3DJ.%26aulast%3DTerfloth%26aufirst%3DL.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520Modeling%253A%2520Where%2520Have%2520You%2520Been%253F%2520Where%2520Are%2520You%2520Going%2520To%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4977%26epage%3D5010%26doi%3D10.1021%2Fjm4004285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Agrafiotis, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shemanarev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farnum, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobanov, V. S.</span><span> </span><span class="NLM_article-title">SAR Maps: A New SAR Visualization Technique for Medicinal Chemists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5926</span><span class="NLM_x">–</span> <span class="NLM_lpage">5937</span><span class="refDoi"> DOI: 10.1021/jm070845m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070845m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5926-5937&author=D.+K.+Agrafiotisauthor=M.+Shemanarevauthor=P.+J.+Connollyauthor=M.+Farnumauthor=V.+S.+Lobanov&title=SAR+Maps%3A+A+New+SAR+Visualization+Technique+for+Medicinal+Chemists&doi=10.1021%2Fjm070845m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm070845m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070845m%26sid%3Dliteratum%253Aachs%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DShemanarev%26aufirst%3DM.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DFarnum%26aufirst%3DM.%26aulast%3DLobanov%26aufirst%3DV.%2BS.%26atitle%3DSAR%2520Maps%253A%2520A%2520New%2520SAR%2520Visualization%2520Technique%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5926%26epage%3D5937%26doi%3D10.1021%2Fjm070845m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Directed R-Group Combination Graph: A Methodology To Uncover Structure-Activity Relationship Patterns in a Series of Analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1215</span><span class="NLM_x">–</span> <span class="NLM_lpage">1226</span><span class="refDoi"> DOI: 10.1021/jm201362h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201362h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1215-1226&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Directed+R-Group+Combination+Graph%3A+A+Methodology+To+Uncover+Structure-Activity+Relationship+Patterns+in+a+Series+of+Analogues&doi=10.1021%2Fjm201362h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm201362h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201362h%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDirected%2520R-Group%2520Combination%2520Graph%253A%2520A%2520Methodology%2520To%2520Uncover%2520Structure-Activity%2520Relationship%2520Patterns%2520in%2520a%2520Series%2520of%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1215%26epage%3D1226%26doi%3D10.1021%2Fjm201362h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">AnalogExplorer: A New Method for Graphical Analysis of Analog Series and Associated Structure-Activity Relationship Information</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9184</span><span class="NLM_x">–</span> <span class="NLM_lpage">9194</span><span class="refDoi"> DOI: 10.1021/jm501391g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501391g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9184-9194&author=B.+Zhangauthor=Y.+Huauthor=J.+Bajorath&title=AnalogExplorer%3A+A+New+Method+for+Graphical+Analysis+of+Analog+Series+and+Associated+Structure-Activity+Relationship+Information&doi=10.1021%2Fjm501391g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm501391g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501391g%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAnalogExplorer%253A%2520A%2520New%2520Method%2520for%2520Graphical%2520Analysis%2520of%2520Analog%2520Series%2520and%2520Associated%2520Structure-Activity%2520Relationship%2520Information%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9184%26epage%3D9194%26doi%3D10.1021%2Fjm501391g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Maynard, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. D.</span><span> </span><span class="NLM_article-title">Quantifying, Visualizing, and Monitoring Lead Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4189</span><span class="NLM_x">–</span> <span class="NLM_lpage">4201</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4189-4201&author=A.+T.+Maynardauthor=C.+D.+Roberts&title=Quantifying%2C+Visualizing%2C+and+Monitoring+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DA.%2BT.%26aulast%3DRoberts%26aufirst%3DC.%2BD.%26atitle%3DQuantifying%252C%2520Visualizing%252C%2520and%2520Monitoring%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4189%26epage%3D4201%26doi%3D10.1021%2Facs.jmedchem.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Shanmugasundaram, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayastha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Vega de León, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Monitoring the Progression of Structure-Activity Relationship Information During Lead Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4235</span><span class="NLM_x">–</span> <span class="NLM_lpage">4244</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4235-4244&author=V.+Shanmugasundaramauthor=L.+Zhangauthor=S.+Kayasthaauthor=A.+de+la+Vega+de+Le%C3%B3nauthor=D.+Dimovaauthor=J.+Bajorath&title=Monitoring+the+Progression+of+Structure-Activity+Relationship+Information+During+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b01428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01428%26sid%3Dliteratum%253Aachs%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKayastha%26aufirst%3DS.%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMonitoring%2520the%2520Progression%2520of%2520Structure-Activity%2520Relationship%2520Information%2520During%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4235%26epage%3D4244%26doi%3D10.1021%2Facs.jmedchem.5b01428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Kenny, P. W.; Sadowski, J.</span><span> </span><span class="NLM_article-title">Structure Modification in Chemical Databases</span>. In  <span class="citation_source-book">Chemoinformatics in Drug Discovery</span>; <span class="NLM_contrib-group">Oprea, T. I.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span>, DOI: <span class="refDoi"> DOI: 10.1002/3527603743.ch11</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1002%2F3527603743.ch11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=271-285&author=P.+W.+Kenny&author=J.+Sadowskiauthor=T.+I.+Oprea&title=Chemoinformatics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F3527603743.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603743.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DStructure%2520Modification%2520in%2520Chemical%2520Databases%26btitle%3DChemoinformatics%2520in%2520Drug%2520Discovery%26aulast%3DOprea%26aufirst%3DT.%2BI.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D271%26epage%3D285%26doi%3D10.1002%2F3527603743.ch11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Griffen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, D. J.</span><span> </span><span class="NLM_article-title">Matched Molecular Pairs as a Medicinal Chemistry Tool</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7739</span><span class="NLM_x">–</span> <span class="NLM_lpage">7750</span><span class="refDoi"> DOI: 10.1021/jm200452d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200452d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Wks7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7739-7750&author=E.+Griffenauthor=A.+G.+Leachauthor=G.+R.+Robbauthor=D.+J.+Warner&title=Matched+Molecular+Pairs+as+a+Medicinal+Chemistry+Tool&doi=10.1021%2Fjm200452d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Matched Molecular Pairs as a Medicinal Chemistry Tool</span></div><div class="casAuthors">Griffen, Ed; Leach, Andrew G.; Robb, Graeme R.; Warner, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7739-7750</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">At the very heart of the role of a medicinal chemist or drug designer is the ability to link chem. structure to mol. properties.  This is the structure-activity relationship(SAR) or structure-property relationship(SPR).  Drug discovery projects often discover that a particular structural change causes a change to a property of interest.  The authors of this paper have been involved in developing and implementing some variations on the original theme of MMPA (matched mol. pairs anal.) and felt this to be the time to highlight some of these and to point toward future developments.  All of them are involved in the process of drug design, and it is within this context that they will set their comments.  In outline, they introduce matched mol. pair anal. and highlight why it fits so well with the kind of problem solving that is required in the process of drug design.  This will be followed by some comments concerning when the application of the methodol. is appropriate.  Then in a section intended for those with more computational interests, the authors describe the various ways that have been devised for identifying such matched pairs.  Throughout the paper, the authors have elected to use a publicly available data set to illustrate some of the comments: the ChEMBL database of inhibitors of the epidermal growth factor receptor tyrosine kinase (EGFR).  This consists of 2899 measurements of an IC50 annotated as a binding measurement on 2348 unique compds.  These measurements are drawn from a no. of sources, having been abstracted from the scientific literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMGFfO8Flm5bVg90H21EOLACvtfcHk0liYOrjv0b4NNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Wks7jK&md5=fab3344d790d1c0ae688ec65830d9143</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm200452d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200452d%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26atitle%3DMatched%2520Molecular%2520Pairs%2520as%2520a%2520Medicinal%2520Chemistry%2520Tool%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7739%26epage%3D7750%26doi%3D10.1021%2Fjm200452d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wawer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Local Structural Changes, Global Data Views: Graphical Substructure-Activity Relationship Trailing</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2944</span><span class="NLM_x">–</span> <span class="NLM_lpage">2951</span><span class="refDoi"> DOI: 10.1021/jm200026b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200026b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvV2hurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2944-2951&author=M.+Wawerauthor=J.+Bajorath&title=Local+Structural+Changes%2C+Global+Data+Views%3A+Graphical+Substructure-Activity+Relationship+Trailing&doi=10.1021%2Fjm200026b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Local structural changes, global data views: Graphical substructure-activity relationship trailing</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2944-2951</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The systematic extn. of structure-activity relation (SAR) information from large and diverse compd. data sets depends on the application of computational anal. methods.  Irresp. of the methodol. details, the ultimate goal of large-scale SAR anal. is to identify most informative compds. and rationalize structural changes that det. SAR behavior.  Such insights provide a basis for further chem. exploration.  Herein we introduce the first graphical SAR anal. method that globally organizes large compd. data sets on the basis of local structural relationships, hence providing an immediate access to important structural modifications and SAR determinants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPva8Il5aD1rVg90H21EOLACvtfcHk0liYOrjv0b4NNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvV2hurk%253D&md5=b8d6cb2d9d9bf7aa6e01ccf2f70006c6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm200026b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200026b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLocal%2520Structural%2520Changes%252C%2520Global%2520Data%2520Views%253A%2520Graphical%2520Substructure-Activity%2520Relationship%2520Trailing%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2944%26epage%3D2951%26doi%3D10.1021%2Fjm200026b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Ghosh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Classification of Matching Molecular Series on the Basis of SAR Phenotypes and Structural Relationships</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1039/C5MD00566C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC5MD00566C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1ylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=237-246&author=A.+Ghoshauthor=D.+Dimovaauthor=J.+Bajorath&title=Classification+of+Matching+Molecular+Series+on+the+Basis+of+SAR+Phenotypes+and+Structural+Relationships&doi=10.1039%2FC5MD00566C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of matching molecular series on the basis of SAR phenotypes and structural relationships</span></div><div class="casAuthors">Ghosh, Adhideb; Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">237-246</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In medicinal chem., matched mol. pairs (MMPs) have experienced increasing interest.  An MMP is defined as a pair of compds. that only differ by a structural change at a single site.  MMPs are often used to assoc. specific structural modifications with changes in mol. properties.  As an extension of the MMP concept, matching mol. series (MMSs) have been introduced.  An MMS is defined as a set of compds. with pairwise MMP relationships and thus represents a series of analogs with modifications at a single site.  We have aimed to systematically identify publicly available MMSs, characterize the SAR information encoded by MMSs and explore structural relationships between them.  Therefore, we have searched for MMSs in more than 700 compd. activity classes, systematically explored structural relationships between MMSs, and characterized the structure-activity relationship (SAR) information they contain.  Combining SAR and structural relationship information has made it possible to identify structurally related MMSs with similar or distinct SAR characteristics, which frequently shared compds.  Such MMSs combine series of analogs with different substitution sites and reveal how structural modifications influence SARs.  They can also be used to explore analog pathways that change SAR characteristics and provide addnl. SAR information.  As a part of our study, a comprehensive database of pairs of MMSs with different SAR characteristics is made freely available for medicinal chem. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcNEPVy4JWOLVg90H21EOLACvtfcHk0lgbdxi6nUXIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1ylsw%253D%253D&md5=2609b5fb4dc1b5fef3f0f9e4cbe98991</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FC5MD00566C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00566C%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DClassification%2520of%2520Matching%2520Molecular%2520Series%2520on%2520the%2520Basis%2520of%2520SAR%2520Phenotypes%2520and%2520Structural%2520Relationships%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D237%26epage%3D246%26doi%3D10.1039%2FC5MD00566C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+Efficient+Algorithm+to+Identify+Matched+Molecular+Pairs+%28MMPs%29+in+Large+Data+Sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lgbdxi6nUXIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520Efficient%2520Algorithm%2520to%2520Identify%2520Matched%2520Molecular%2520Pairs%2520%2528MMPs%2529%2520in%2520Large%2520Data%2520Sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span> </span><span class="NLM_article-title">RECAP – Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1021/ci970429i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+B.+Juddauthor=S.+P.+Watsonauthor=M.+M.+Hann&title=RECAP+%E2%80%93+Retrosynthetic+Combinatorial+Analysis+Procedure%3A+A+Powerful+New+Technique+for+Identifying+Privileged+Molecular+Fragments+with+Useful+Applications+in+Combinatorial+Chemistry&doi=10.1021%2Fci970429i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lgbdxi6nUXIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%2BB.%26aulast%3DWatson%26aufirst%3DS.%2BP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DRECAP%2520%25E2%2580%2593%2520Retrosynthetic%2520Combinatorial%2520Analysis%2520Procedure%253A%2520A%2520Powerful%2520New%2520Technique%2520for%2520Identifying%2520Privileged%2520Molecular%2520Fragments%2520with%2520Useful%2520Applications%2520in%2520Combinatorial%2520Chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522%26doi%3D10.1021%2Fci970429i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">de la Vega de León, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Matched Molecular Pairs Derived by Retrosynthetic Fragmentation</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1039/C3MD00259D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC3MD00259D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFGkt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=64-67&author=A.+de+la+Vega+de+Le%C3%B3nauthor=J.+Bajorath&title=Matched+Molecular+Pairs+Derived+by+Retrosynthetic+Fragmentation&doi=10.1039%2FC3MD00259D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Matched molecular pairs derived by retrosynthetic fragmentation</span></div><div class="casAuthors">de la Vega de Leon, Antonio; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-67</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Matched mol. pairs (MMPs) are defined as pairs of compds. that only differ by a chem. change at a single site.  MMPs have become popular in medicinal chem. to support lead optimization, absorption, distribution, metab., excretion, and toxicity (ADMET) anal., and other applications.  Thus far, MMPs have been algorithmically defined and not on the basis of reaction information.  This often limits the chem. interpretability and practical utility of MMPs.  Therefore, we introduce synthetically accessible MMPs that are automatically generated by applying reaction rules following the retrosynthetic combinatorial anal. procedure (RECAP).  A library of more than 92 000 RECAP-MMPs was generated from public domain compds. active against 435 different targets exclusively utilizing high-confidence activity data.  This library is made freely available for use in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqotALy5I0biLVg90H21EOLACvtfcHk0lgbdxi6nUXIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFGkt7fE&md5=5a336f03755927bed0289243caaa1e2f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC3MD00259D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00259D%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMatched%2520Molecular%2520Pairs%2520Derived%2520by%2520Retrosynthetic%2520Fragmentation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D64%26epage%3D67%26doi%3D10.1039%2FC3MD00259D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: A Large-Scale Bioactivity Database for Drug Discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lhiXQlJgk6AVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumpfe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Systematic Assessment of Analog Relationships between Bioactive Compounds and Promiscuity of Analog sets</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">230</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1039/C5MD00449G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC5MD00449G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=230-236&author=D.+Dimovaauthor=D.+Stumpfeauthor=J.+Bajorath&title=Systematic+Assessment+of+Analog+Relationships+between+Bioactive+Compounds+and+Promiscuity+of+Analog+sets&doi=10.1039%2FC5MD00449G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC5MD00449G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00449G%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSystematic%2520Assessment%2520of%2520Analog%2520Relationships%2520between%2520Bioactive%2520Compounds%2520and%2520Promiscuity%2520of%2520Analog%2520sets%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D230%26epage%3D236%26doi%3D10.1039%2FC5MD00449G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Compound Promiscuity - What Can We Learn From Current Data</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">644</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1016%2Fj.drudis.2013.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=23524195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=644-650&author=Y.+Huauthor=J.+Bajorath&title=Compound+Promiscuity+-+What+Can+We+Learn+From+Current+Data&doi=10.1016%2Fj.drudis.2013.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Compound promiscuity: what can we learn from current data?</span></div><div class="casAuthors">Hu, Ye; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">644-650</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The specificity paradigm that assigns central relevance to achieving target specificity of drug candidates has recently been revisited.  Evidence is mounting that polypharmacol. drug behavior is often responsible for therapeutic efficacy suggesting the consideration of new drug development strategies.  Target promiscuity of compds. is at the origin of polypharmacol.  For many bioactive compds., multiple target annotations are available indicating that compd. promiscuity is a general phenomenon.  However, careful anal. of compd. activity data reveals that the degree of apparent promiscuity is strongly influenced by data selection criteria and the type of activity measurements that are considered.  Furthermore, promiscuity involving unrelated targets is less common.  Apparent target promiscuity might often better be interpreted as activity promiscuity in different assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrQ19HLWfCarVg90H21EOLACvtfcHk0lhiXQlJgk6AVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGru7Y%253D&md5=1ec94bf333ae5da9d35aa60f06366436</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCompound%2520Promiscuity%2520-%2520What%2520Can%2520We%2520Learn%2520From%2520Current%2520Data%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D644%26epage%3D650%26doi%3D10.1016%2Fj.drudis.2013.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">High-Resolution View of Compound Promiscuity</span> <span class="citation_source-journal">F1000Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="refDoi"> DOI: 10.12688/f1000research.2-144.v2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.12688%2Ff1000research.2-144.v2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=24358872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A280%3ADC%252BC2c3psVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=144&author=Y.+Huauthor=J.+Bajorath&title=High-Resolution+View+of+Compound+Promiscuity&doi=10.12688%2Ff1000research.2-144.v2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution view of compound promiscuity</span></div><div class="casAuthors">Hu Ye; Bajorath Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144</span>
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    </div><div class="casAbstract">Compound promiscuity is defined as the ability of a small molecule to specifically interact with multiple biological targets.  So-defined promiscuity is relevant for drug discovery because it provides the molecular basis of polypharmacology, which is increasingly implicated in the therapeutic efficacy of drugs.  Recent studies have analyzed different aspects of compound promiscuity on the basis of currently available activity data.  In this commentary, we present take-home messages from these studies augmented with new results to generate a detailed picture of compound promiscuity that might serve as a reference for further discussions and research activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRhg3urNXDByO2kE7krGOBfW6udTcc2eYHBrHoBUjEdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3psVWrug%253D%253D&md5=f96841e071505f8e25de1ef0766dc1a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.2-144.v2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.2-144.v2%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DHigh-Resolution%2520View%2520of%2520Compound%2520Promiscuity%26jtitle%3DF1000Research%26date%3D2013%26volume%3D2%26spage%3D144%26doi%3D10.12688%2Ff1000research.2-144.v2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Network Pharmacology: The Next Paradigm in Drug Discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">682</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1038/nchembio.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1038%2Fnchembio.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=18936753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=682-690&author=A.+L.+Hopkins&title=Network+Pharmacology%3A+The+Next+Paradigm+in+Drug+Discovery&doi=10.1038%2Fnchembio.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Network pharmacology: the next paradigm in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is the concept of designing maximally selective ligands to act on individual drug targets.  However, many effective drugs act via modulation of multiple proteins rather than single targets.  Advances in systems biol. are revealing a phenotypic robustness and a network structure that strongly suggests that exquisitely selective compds., compared with multi-target drugs, may exhibit lower than desired clin. efficacy.  This new appreciation of the role of polypharmacol. has significant implications for tackling the two major sources of attrition in drug development, efficacy and toxicity.  Integrating network biol. and polypharmacol. holds the promise of expanding the current opportunity space for druggable targets.  However, the rational design of polypharmacol. faces considerable challenges in the need for new methods to validate target combinations and optimize multiple structure-activity relationships while maintaining drug-like properties.  Advances in these areas are creating the foundation of the next paradigm in drug discovery: network pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqHy4FqkK-h7Vg90H21EOLACvtfcHk0liTG8JIFxNTLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK&md5=03b8be3ddbcab7dc65afd17df7ee8395</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.118%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DNetwork%2520Pharmacology%253A%2520The%2520Next%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D682%26epage%3D690%26doi%3D10.1038%2Fnchembio.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Jalencas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">On the Origins of Drug Polypharmacology</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1039/C2MD20242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC2MD20242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=80-87&author=X.+Jalencasauthor=J.+Mestres&title=On+the+Origins+of+Drug+Polypharmacology&doi=10.1039%2FC2MD20242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the origins of drug polypharmacology</span></div><div class="casAuthors">Jalencas, Xavier; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-87</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The ability of small mols. to interact with multiple proteins is commonly referred to as polypharmacol.  The now widely accepted polypharmacol. of drugs is of particular interest for human health as it has implications beyond therapeutic efficacy, from anticipating adverse drug reactions to identifying potential repurposing opportunities.  There have been a no. of studies relating the extent of drug polypharmacol. to the physicochem. properties and fragment compn. of the drug itself, but also to the protein family and distant binding site similarities of the drug's primary target.  Taken together, all these observations lead to speculate that the origins of drug polypharmacol. may lie at the heart of protein evolution and that polypharmacol. may just be a reminiscent signature of some of the mechanisms of adaptation that primitive biol. systems developed to increase the chances of survival in a highly variable early chem. environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1qsCiSthHLVg90H21EOLACvtfcHk0liTG8JIFxNTLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI&md5=77fe84731536e5008cca3b0faf1e58dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC2MD20242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20242E%26sid%3Dliteratum%253Aachs%26aulast%3DJalencas%26aufirst%3DX.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DOn%2520the%2520Origins%2520of%2520Drug%2520Polypharmacology%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D80%26epage%3D87%26doi%3D10.1039%2FC2MD20242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Anighoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="NLM_x">–</span> <span class="NLM_lpage">7887</span><span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lgsOKBWnfcv3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzek, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karapetyan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dracheva, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem’s BioAssay Database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D400</span><span class="NLM_x">–</span> <span class="NLM_lpage">D412</span><span class="refDoi"> DOI: 10.1093/nar/gkr1132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkr1132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=22140110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D400-D412&author=Y.+Wangauthor=J.+Xiaoauthor=T.+O.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=Z.+Zhouauthor=L.+Hanauthor=K.+Karapetyanauthor=S.+Drachevaauthor=B.+A.+Shoemakerauthor=E.+Boltonauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=PubChem%E2%80%99s+BioAssay+Database&doi=10.1093%2Fnar%2Fgkr1132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem's BioAssay Database</span></div><div class="casAuthors">Wang, Yanli; Xiao, Jewen; Suzek, Tugba O.; Zhang, Jian; Wang, Jiyao; Zhou, Zhigang; Han, Lianyi; Karapetyan, Karen; Dracheva, Svetlana; Shoemaker, Benjamin A.; Bolton, Evan; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D400-D412</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem (http://pubchem.ncbi.nlm.nih.gov) is a public repository for biol. activity data of small mols. and RNAi reagents.  The mission of PubChem is to deliver free and easy access to all deposited data, and to provide intuitive data anal. tools.  The PubChem BioAssay database currently contains 500 000 descriptions of assay protocols, covering 5000 protein targets, 30 000 gene targets and providing over 130 million bioactivity outcomes.  PubChem's bioassay data are integrated into the NCBI Entrez information retrieval system, thus making PubChem data searchable and accessible by Entrez queries.  Also, as a repository, PubChem constantly optimizes and develops its deposition system answering many demands of both high- and low-vol. depositors.  The PubChem information platform allows users to search, review and download bioassay description and data.  The PubChem platform also enables researchers to collect, compare and analyze biol. test results through web-based and programmatic tools.  In this work, we provide an update for the PubChem BioAssay resource, including information content growth, data model extension and new developments of data submission, retrieval, anal. and download tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISH42-uje7LVg90H21EOLACvtfcHk0lgsOKBWnfcv3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzI&md5=f172bdd6db3d31af6aed30dde4d7eeee</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1132%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DSuzek%26aufirst%3DT.%2BO.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DKarapetyan%26aufirst%3DK.%26aulast%3DDracheva%26aufirst%3DS.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DBolton%26aufirst%3DE.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%25E2%2580%2599s%2520BioAssay%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD400%26epage%3DD412%26doi%3D10.1093%2Fnar%2Fgkr1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Law, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, A.</span><span> </span><span class="NLM_article-title">DrugBank 4.0: Shedding New Light on Drug Metabolism</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1097</span><span class="refDoi"> DOI: 10.1093/nar/gkt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=24203711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Kh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1091-D1097&author=V.+Lawauthor=C.+Knoxauthor=Y.+Djoumbouauthor=T.+Jewisonauthor=A.+C.+Guoauthor=Y.+Liuauthor=A.+Tang&title=DrugBank+4.0%3A+Shedding+New+Light+on+Drug+Metabolism&doi=10.1093%2Fnar%2Fgkt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 4.0: shedding new light on drug metabolism</span></div><div class="casAuthors">Law, Vivian; Knox, Craig; Djoumbou, Yannick; Jewison, Tim; Guo, An Chi; Liu, Yifeng; Maciejewski, Adam; Arndt, David; Wilson, Michael; Neveu, Vanessa; Tang, Alexandra; Gabriel, Geraldine; Ly, Carol; Adamjee, Sakina; Dame, Zerihun T.; Han, Beomsoo; Zhou, You; Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1091-D1097</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank (http://www.drugbank.ca) is a comprehensive online database contg. extensive biochem. and pharmacol. information about drugs, their mechanisms and their targets.  Since it was first described in 2006, DrugBank has rapidly evolved, both in response to user requests and in response to changing trends in drug research and development.  Previous versions of DrugBank have been widely used to facilitate drug and in silico drug target discovery.  The latest update, DrugBank 4.0, has been further expanded to contain data on drug metab., absorption, distribution, metab., excretion and toxicity (ADMET) and other kinds of quant. structure activity relationships (QSAR) information.  These enhancements are intended to facilitate research in xenobiotic metab. (both prediction and characterization), pharmacokinetics, pharmacodynamics and drug design/discovery.  For this release, >1200 drug metabolites (including their structures, names, activity, abundance and other detailed data) have been added along with >1300 drug metab. reactions (including metabolizing enzymes and reaction types) and dozens of drug metab. pathways.  Another 30 predicted or measured ADMET parameters have been added to each DrugCard, bringing the av. no. of quant. ADMET values for Food and Drug Administration-approved drugs close to 40.  Referential NMR and MS spectra have been added for almost 400 drugs as well as spectral and mass matching tools to facilitate compd. identification.  This expanded collection of drug information is complemented by a no. of new or improved search tools, including one that provides a simple analyses of drug-target, -enzyme and -transporter assocns. to provide insight on drug-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_BsUO1phm7Vg90H21EOLACvtfcHk0lgsOKBWnfcv3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Kh&md5=0da9f3ecfc2061f814803cf4b070e54e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1068%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DV.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DA.%26atitle%3DDrugBank%25204.0%253A%2520Shedding%2520New%2520Light%2520on%2520Drug%2520Metabolism%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1091%26epage%3DD1097%26doi%3D10.1093%2Fnar%2Fgkt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Berthold, M. R.; Cebron, N.; Dill, F.; Gabriel, T. R.; Kötter, T.; Meinl, T.; Ohl, P.; Sieb, C.; Thiel, K.; Wiswedel, B.</span><span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. In  <span class="citation_source-book">Studies in Classification, Data Analysis, and Knowledge Organization</span>; <span class="NLM_contrib-group">Preisach, C.; Burkhardt, H.; Schmidt-Thieme, L.; Decker, R.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-3-540-78246-9_38</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Berthold&author=N.+Cebron&author=F.+Dill&author=T.+R.+Gabriel&author=T.+K%C3%B6tter&author=T.+Meinl&author=P.+Ohl&author=C.+Sieb&author=K.+Thiel&author=B.+Wiswedelauthor=C.+Preisach&author=H.+Burkhardt&author=L.+Schmidt-Thieme&author=R.+Decker&title=Studies+in+Classification%2C+Data+Analysis%2C+and+Knowledge+Organization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26btitle%3DStudies%2520in%2520Classification%252C%2520Data%2520Analysis%252C%2520and%2520Knowledge%2520Organization%26aulast%3DPreisach%26aufirst%3DC.%26pub%3DSpringer%26date%3D2008%26spage%3D319%26epage%3D326%26doi%3D10.1007%2F978-3-540-78246-9_38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span>OEChem TK; <span class="NLM_publisher-name">OpenEye Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM, U.S.</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OEChem+TK%3B+OpenEye+Scientific+Software%2C+Inc.%3A+Santa+Fe%2C+NM%2C+U.S.%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOEChem%2520TK%26pub%3DOpenEye%2520Scientific%2520Software%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumpfe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">MMP-cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1138</span><span class="NLM_x">–</span> <span class="NLM_lpage">1145</span><span class="refDoi"> DOI: 10.1021/ci3001138</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001138" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1138-1145&author=X.+Huauthor=Y.+Huauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=MMP-cliffs%3A+Systematic+Identification+of+Activity+Cliffs+on+the+Basis+of+Matched+Molecular+Pairs&doi=10.1021%2Fci3001138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span></div><div class="casAuthors">Hu, Xiaoying; Hu, Ye; Vogt, Martin; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1138-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs are generally defined as pairs of structurally similar compds. having large differences in potency.  The anal. of activity cliffs is of general interest because structure-activity relationship (SAR) determinants can often be deduced from them.  Crit. questions for the study of activity cliffs include how similar compds. should be to qualify as cliff partners, how similarity should be assessed, and how large potency differences between participating compds. should be.  Thus far, activity cliffs have mostly been defined on the basis of calcd. Tanimoto similarity values using structural descriptors, esp. 2D fingerprints.  As any theor. assessment of mol. similarity, this approach has its limitations.  For example, calcd. Tanimoto similarities might often be difficult to reconcile and interpret from a chem. perspective, a point of critique frequently raised in medicinal chem.  Herein, we have explored activity cliffs by considering well-defined substructure replacements instead of calcd. similarity values.  For this purpose, the matched mol. pair (MMP) formalism has been applied.  MMPs were systematically derived from public domain compds., and activity cliffs were extd. from them, termed MMP-cliffs.  The frequency of cliff formation was detd. for compds. active against different targets, MMP-cliffs were analyzed in detail, and re-evaluated on the basis of Tanimoto similarity.  In many instances, chem. intuitive activity cliffs were only detected on the basis of MMPs, but not Tanimoto similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYSOxNqPjGbVg90H21EOLACvtfcHk0ljOmxueqscj_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D&md5=958a76d85d0a06424b43eb1b694cf3ca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci3001138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001138%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMMP-cliffs%253A%2520Systematic%2520Identification%2520of%2520Activity%2520Cliffs%2520on%2520the%2520Basis%2520of%2520Matched%2520Molecular%2520Pairs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1138%26epage%3D1145%26doi%3D10.1021%2Fci3001138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Shannon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markiel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozier, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baliga, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramage, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwikowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ideker, T.</span><span> </span><span class="NLM_article-title">Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks</span> <span class="citation_source-journal">Genome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2498</span><span class="NLM_x">–</span> <span class="NLM_lpage">2504</span><span class="refDoi"> DOI: 10.1101/gr.1239303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1101%2Fgr.1239303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=14597658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&author=P.+Shannonauthor=A.+Markielauthor=O.+Ozierauthor=N.+S.+Baligaauthor=J.+T.+Wangauthor=D.+Ramageauthor=N.+Aminauthor=B.+Schwikowskiauthor=T.+Ideker&title=Cytoscape%3A+A+Software+Environment+for+Integrated+Models+of+Biomolecular+Interaction+Networks&doi=10.1101%2Fgr.1239303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Cytoscape: A software environment for integrated models of biomolecular interaction networks</span></div><div class="casAuthors">Shannon, Paul; Markiel, Andrew; Ozier, Owen; Baliga, Nitin S.; Wang, Jonathan T.; Ramage, Daniel; Amin, Nada; Schwikowski, Benno; Ideker, Trey</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2504</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cytoscape is an open source software project for integrating biomol. interaction networks with high-throughput expression data and other mol. states into a unified conceptual framework.  Although applicable to any system of mol. components and interactions, Cytoscape is most powerful when used in conjunction with large databases of protein-protein, protein-DNA, and genetic interactions that are increasingly available for humans and model organisms.  Cytoscape's software Core provides basic functionality to layout and query the network; to visually integrate the network with expression profiles, phenotypes, and other mol. states; and to link the network to databases of functional annotations.  The Core is extensible through a straightforward plug-in architecture, allowing rapid development of addnl. computational analyses and features.  Several case studies of Cytoscape plug-ins are surveyed, including a search for interaction pathways correlating with changes in gene expression, a study of protein complexes involved in cellular recovery to DNA damage, inference of a combined phys./functional interaction network for Halobacterium, and an interface to detailed stochastic/kinetic gene regulatory models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_cu7mgf2RbVg90H21EOLACvtfcHk0ljOmxueqscj_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D&md5=2bcbca9a3bd04717761f0424c0209e43</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1101%2Fgr.1239303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.1239303%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DP.%26aulast%3DMarkiel%26aufirst%3DA.%26aulast%3DOzier%26aufirst%3DO.%26aulast%3DBaliga%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DJ.%2BT.%26aulast%3DRamage%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DIdeker%26aufirst%3DT.%26atitle%3DCytoscape%253A%2520A%2520Software%2520Environment%2520for%2520Integrated%2520Models%2520of%2520Biomolecular%2520Interaction%2520Networks%26jtitle%3DGenome%2520Res.%26date%3D2003%26volume%3D13%26spage%3D2498%26epage%3D2504%26doi%3D10.1101%2Fgr.1239303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span>Zenodo. <a href="https://zenodo.org/record/59688" class="extLink">https://zenodo.org/record/59688</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Zenodo.+https%3A%2F%2Fzenodo.org%2Frecord%2F59688."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 39 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Maheshwara Reddy Nadiveedhi, Poojith Nuthalapati, Mohan Gundluru, Mohan Reddy Yanamula, Saritha Venkatareddy Kallimakula, Visweswara Rao Pasupuleti, Vijaya Kumar Reddy Avula, Swetha Vallela, Grigory Vasilievich Zyryanov, Satheesh Krishna Balam, <span class="NLM_string-name hlFld-ContribAuthor">Suresh Reddy Cirandur</span>. </span><span class="cited-content_cbyCitation_article-title">Green Synthesis, Antioxidant, and Plant Growth Regulatory Activities of Novel α-Furfuryl-2-alkylaminophosphonates. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (4)
                                     , 2934-2948. <a href="https://doi.org/10.1021/acsomega.0c05302" title="DOI URL">https://doi.org/10.1021/acsomega.0c05302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c05302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c05302%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DGreen%252BSynthesis%25252C%252BAntioxidant%25252C%252Band%252BPlant%252BGrowth%252BRegulatory%252BActivities%252Bof%252BNovel%252B%2525CE%2525B1-Furfuryl-2-alkylaminophosphonates%26aulast%3DNadiveedhi%26aufirst%3DMaheshwara%2BReddy%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D30102020%26date%3D18122020%26date%3D21012021%26volume%3D6%26issue%3D4%26spage%3D2934%26epage%3D2948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Raquel Rodríguez-Pérez, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Interpretation of Compound Activity Predictions from Complex Machine Learning Models Using Local Approximations and Shapley Values. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8761-8777. <a href="https://doi.org/10.1021/acs.jmedchem.9b01101" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01101%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInterpretation%252Bof%252BCompound%252BActivity%252BPredictions%252Bfrom%252BComplex%252BMachine%252BLearning%252BModels%252BUsing%252BLocal%252BApproximations%252Band%252BShapley%252BValues%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D08072019%26date%3D26092019%26date%3D12092019%26volume%3D63%26issue%3D16%26spage%3D8761%26epage%3D8777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lorenzo Cianni, Christian Wolfgang Feldmann, Erik Gilberg, Michael Gütschow, Luiz Juliano, Andrei Leitão, Jürgen Bajorath, <span class="NLM_string-name hlFld-ContribAuthor">Carlos A. Montanari</span>. </span><span class="cited-content_cbyCitation_article-title">Can Cysteine Protease Cross-Class Inhibitors Achieve Selectivity?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10497-10525. <a href="https://doi.org/10.1021/acs.jmedchem.9b00683" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00683</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00683%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCan%252BCysteine%252BProtease%252BCross-Class%252BInhibitors%252BAchieve%252BSelectivity%25253F%26aulast%3DCianni%26aufirst%3DLorenzo%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25042019%26date%3D15082019%26date%3D30072019%26volume%3D62%26issue%3D23%26spage%3D10497%26epage%3D10525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Emilia Vásquez-Domínguez, Vinicio Danilo Armijos-Jaramillo, Eduardo Tejera, <span class="NLM_string-name hlFld-ContribAuthor">Humbert González-Díaz</span>. </span><span class="cited-content_cbyCitation_article-title">Multioutput Perturbation-Theory Machine Learning (PTML) Model of ChEMBL Data for Antiretroviral Compounds. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2019,</strong> <em>16 </em>
                                    (10)
                                     , 4200-4212. <a href="https://doi.org/10.1021/acs.molpharmaceut.9b00538" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.9b00538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.9b00538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.9b00538%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DMultioutput%252BPerturbation-Theory%252BMachine%252BLearning%252B%252528PTML%252529%252BModel%252Bof%252BChEMBL%252BData%252Bfor%252BAntiretroviral%252BCompounds%26aulast%3DV%25C3%25A1squez-Dom%25C3%25ADnguez%26aufirst%3DEmilia%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17052019%26date%3D19082019%26date%3D08082019%26date%3D30082019%26date%3D19082019%26volume%3D16%26issue%3D10%26spage%3D4200%26epage%3D4212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Erik Gilberg, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Progress in Structure-Based Evaluation of Compound Promiscuity. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (2)
                                     , 2758-2765. <a href="https://doi.org/10.1021/acsomega.8b03639" title="DOI URL">https://doi.org/10.1021/acsomega.8b03639</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b03639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b03639%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DRecent%252BProgress%252Bin%252BStructure-Based%252BEvaluation%252Bof%252BCompound%252BPromiscuity%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D27122018%26date%3D29012019%26volume%3D4%26issue%3D2%26spage%3D2758%26epage%3D2765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Raquel Rodríguez-Pérez, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Multitask Machine Learning for Classifying Highly and Weakly Potent Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (2)
                                     , 4367-4375. <a href="https://doi.org/10.1021/acsomega.9b00298" title="DOI URL">https://doi.org/10.1021/acsomega.9b00298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00298%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMultitask%252BMachine%252BLearning%252Bfor%252BClassifying%252BHighly%252Band%252BWeakly%252BPotent%252BKinase%252BInhibitors%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D01022019%26date%3D19022019%26volume%3D4%26issue%3D2%26spage%3D4367%26epage%3D4375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">J. Jesús Naveja, Martin Vogt, Dagmar Stumpfe, José L. Medina-Franco, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Extraction of Analogue Series from Large Compound Collections Using a New Computational Compound–Core Relationship Method. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     , 1027-1032. <a href="https://doi.org/10.1021/acsomega.8b03390" title="DOI URL">https://doi.org/10.1021/acsomega.8b03390</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b03390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b03390%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DSystematic%252BExtraction%252Bof%252BAnalogue%252BSeries%252Bfrom%252BLarge%252BCompound%252BCollections%252BUsing%252Ba%252BNew%252BComputational%252BCompound%2525E2%252580%252593Core%252BRelationship%252BMethod%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D03122018%26date%3D03012019%26volume%3D4%26issue%3D1%26spage%3D1027%26epage%3D1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Erik Gilberg, Michael Gütschow, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Promiscuous Ligands from Experimentally Determined Structures, Binding Conformations, and Protein Family-Dependent Interaction Hotspots. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     , 1729-1737. <a href="https://doi.org/10.1021/acsomega.8b03481" title="DOI URL">https://doi.org/10.1021/acsomega.8b03481</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b03481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b03481%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DPromiscuous%252BLigands%252Bfrom%252BExperimentally%252BDetermined%252BStructures%25252C%252BBinding%252BConformations%25252C%252Band%252BProtein%252BFamily-Dependent%252BInteraction%252BHotspots%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D12122018%26date%3D10012019%26volume%3D4%26issue%3D1%26spage%3D1729%26epage%3D1737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Vogt, Dimitar Yonchev, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Method to Evaluate Progress in Lead Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (23)
                                     , 10895-10900. <a href="https://doi.org/10.1021/acs.jmedchem.8b01626" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01626%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BMethod%252Bto%252BEvaluate%252BProgress%252Bin%252BLead%252BOptimization%26aulast%3DVogt%26aufirst%3DMartin%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D19102018%26date%3D30112018%26volume%3D61%26issue%3D23%26spage%3D10895%26epage%3D10900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dimitar Yonchev, Martin Vogt, Dagmar Stumpfe, Ryo Kunimoto, Tomoyuki Miyao, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Assessment of Chemical Saturation of Analogue Series under Varying Conditions. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (11)
                                     , 15799-15808. <a href="https://doi.org/10.1021/acsomega.8b02087" title="DOI URL">https://doi.org/10.1021/acsomega.8b02087</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b02087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b02087%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DComputational%252BAssessment%252Bof%252BChemical%252BSaturation%252Bof%252BAnalogue%252BSeries%252Bunder%252BVarying%252BConditions%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D18082018%26date%3D08112018%26volume%3D3%26issue%3D11%26spage%3D15799%26epage%3D15808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joana Ferreira da Costa, David Silva, Olga Caamaño, José M. Brea, Maria Isabel Loza, Cristian R. Munteanu, Alejandro Pazos, Xerardo García-Mera, <span class="NLM_string-name hlFld-ContribAuthor">Humbert González-Díaz</span>. </span><span class="cited-content_cbyCitation_article-title">Perturbation Theory/Machine Learning Model of ChEMBL Data for Dopamine Targets: Docking, Synthesis, and Assay of New l-Prolyl-l-leucyl-glycinamide Peptidomimetics. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2018,</strong> <em>9 </em>
                                    (11)
                                     , 2572-2587. <a href="https://doi.org/10.1021/acschemneuro.8b00083" title="DOI URL">https://doi.org/10.1021/acschemneuro.8b00083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.8b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.8b00083%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DPerturbation%252BTheory%25252FMachine%252BLearning%252BModel%252Bof%252BChEMBL%252BData%252Bfor%252BDopamine%252BTargets%25253A%252BDocking%25252C%252BSynthesis%25252C%252Band%252BAssay%252Bof%252BNew%252Bl-Prolyl-l-leucyl-glycinamide%252BPeptidomimetics%26aulast%3DFerreira%2Bda%2BCosta%26aufirst%3DJoana%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22022018%26date%3D23052018%26date%3D25062018%26date%3D23052018%26volume%3D9%26issue%3D11%26spage%3D2572%26epage%3D2587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Harbil Bediaga, Sonia Arrasate, <span class="NLM_string-name hlFld-ContribAuthor">Humbert González-Díaz</span>. </span><span class="cited-content_cbyCitation_article-title">PTML Combinatorial Model of ChEMBL Compounds Assays for Multiple Types of Cancer. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2018,</strong> <em>20 </em>
                                    (11)
                                     , 621-632. <a href="https://doi.org/10.1021/acscombsci.8b00090" title="DOI URL">https://doi.org/10.1021/acscombsci.8b00090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.8b00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.8b00090%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DPTML%252BCombinatorial%252BModel%252Bof%252BChEMBL%252BCompounds%252BAssays%252Bfor%252BMultiple%252BTypes%252Bof%252BCancer%26aulast%3DBediaga%26aufirst%3DHarbil%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01072018%26date%3D12092018%26date%3D03102018%26date%3D21092018%26volume%3D20%26issue%3D11%26spage%3D621%26epage%3D632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">X-ray-Structure-Based Identification of Compounds with Activity against Targets from Different Families and Generation of Templates for Multitarget Ligand Design. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (1)
                                     , 106-111. <a href="https://doi.org/10.1021/acsomega.7b01849" title="DOI URL">https://doi.org/10.1021/acsomega.7b01849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b01849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b01849%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DX-ray-Structure-Based%252BIdentification%252Bof%252BCompounds%252Bwith%252BActivity%252Bagainst%252BTargets%252Bfrom%252BDifferent%252BFamilies%252Band%252BGeneration%252Bof%252BTemplates%252Bfor%252BMultitarget%252BLigand%252BDesign%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D24112017%26date%3D18122017%26date%3D31012018%26volume%3D3%26issue%3D1%26spage%3D106%26epage%3D111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carmen  Cerchia</span>, <span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Antonio  Lavecchia</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Exploring Structural Relationships between Bioactive and Commercial Chemical Space and Developing Target Hypotheses for Compound Acquisition. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (11)
                                     , 7760-7766. <a href="https://doi.org/10.1021/acsomega.7b01338" title="DOI URL">https://doi.org/10.1021/acsomega.7b01338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b01338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b01338%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DExploring%252BStructural%252BRelationships%252Bbetween%252BBioactive%252Band%252BCommercial%252BChemical%252BSpace%252Band%252BDeveloping%252BTarget%252BHypotheses%252Bfor%252BCompound%252BAcquisition%26aulast%3DCerchia%26aufirst%3DCarmen%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09092017%26date%3D31102017%26date%3D30112017%26volume%3D2%26issue%3D11%26spage%3D7760%26epage%3D7766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryo  Kunimoto</span>, <span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Application of a New Scaffold Concept for Computational Target Deconvolution of Chemical Cancer Cell Line Screens. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (4)
                                     , 1463-1468. <a href="https://doi.org/10.1021/acsomega.7b00215" title="DOI URL">https://doi.org/10.1021/acsomega.7b00215</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00215%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DApplication%252Bof%252Ba%252BNew%252BScaffold%252BConcept%252Bfor%252BComputational%252BTarget%252BDeconvolution%252Bof%252BChemical%252BCancer%252BCell%252BLine%252BScreens%26aulast%3DKunimoto%26aufirst%3DRyo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24022017%26date%3D06042017%26date%3D30042017%26volume%3D2%26issue%3D4%26spage%3D1463%26epage%3D1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Swarit  Jasial</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Highly Promiscuous Small Molecules from Biological Screening Assays Include Many Pan-Assay Interference Compounds but Also Candidates for Polypharmacology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (22)
                                     , 10285-10290. <a href="https://doi.org/10.1021/acs.jmedchem.6b01314" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01314%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHighly%252BPromiscuous%252BSmall%252BMolecules%252Bfrom%252BBiological%252BScreening%252BAssays%252BInclude%252BMany%252BPan-Assay%252BInterference%252BCompounds%252Bbut%252BAlso%252BCandidates%252Bfor%252BPolypharmacology%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D01092016%26date%3D03112016%26date%3D23112016%26volume%3D59%26issue%3D22%26spage%3D10285%26epage%3D10290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raquel  Rodríguez-Pérez</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Feature importance correlation from machine learning indicates functional relationships between proteins and similar compound binding characteristics. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-93771-y" title="DOI URL">https://doi.org/10.1038/s41598-021-93771-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-93771-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-93771-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DFeature%252Bimportance%252Bcorrelation%252Bfrom%252Bmachine%252Blearning%252Bindicates%252Bfunctional%252Brelationships%252Bbetween%252Bproteins%252Band%252Bsimilar%252Bcompound%252Bbinding%252Bcharacteristics%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Candida  Manelfi</span>, <span class="hlFld-ContribAuthor ">Marica  Gemei</span>, <span class="hlFld-ContribAuthor ">Carmine  Talarico</span>, <span class="hlFld-ContribAuthor ">Carmen  Cerchia</span>, <span class="hlFld-ContribAuthor ">Anna  Fava</span>, <span class="hlFld-ContribAuthor ">Filippo  Lunghini</span>, <span class="hlFld-ContribAuthor ">Andrea Rosario  Beccari</span>. </span><span class="cited-content_cbyCitation_article-title">“Molecular Anatomy”: a new multi-dimensional hierarchical scaffold analysis tool. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00526-y" title="DOI URL">https://doi.org/10.1186/s13321-021-00526-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00526-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00526-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3D%2525E2%252580%25259CMolecular%252BAnatomy%2525E2%252580%25259D%25253A%252Ba%252Bnew%252Bmulti-dimensional%252Bhierarchical%252Bscaffold%252Banalysis%252Btool%26aulast%3DManelfi%26aufirst%3DCandida%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>, <span class="hlFld-ContribAuthor ">Norberto  Sánchez-Cruz</span>, <span class="hlFld-ContribAuthor ">Edgar  López-López</span>, <span class="hlFld-ContribAuthor ">Bárbara I.  Díaz-Eufracio</span>. </span><span class="cited-content_cbyCitation_article-title">Progress on open chemoinformatic tools for expanding and exploring the chemical space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>22 </em><a href="https://doi.org/10.1007/s10822-021-00399-1" title="DOI URL">https://doi.org/10.1007/s10822-021-00399-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00399-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00399-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DProgress%252Bon%252Bopen%252Bchemoinformatic%252Btools%252Bfor%252Bexpanding%252Band%252Bexploring%252Bthe%252Bchemical%252Bspace%26aulast%3DMedina-Franco%26aufirst%3DJos%25C3%25A9%2BL.%26date%3D2021%26date%3D2021%26volume%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raquel  Rodríguez-Pérez</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of multi-target deep neural network models for compound potency prediction under increasingly challenging test conditions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (3)
                                     , 285-295. <a href="https://doi.org/10.1007/s10822-021-00376-8" title="DOI URL">https://doi.org/10.1007/s10822-021-00376-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00376-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00376-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DEvaluation%252Bof%252Bmulti-target%252Bdeep%252Bneural%252Bnetwork%252Bmodels%252Bfor%252Bcompound%252Bpotency%252Bprediction%252Bunder%252Bincreasingly%252Bchallenging%252Btest%252Bconditions%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D3%26spage%3D285%26epage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">José L.  Medina‐Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Consistent Cell‐selective Analog Series as Constellation Luminaries in Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2020,</strong> <em>39 </em>
                                    (12)
                                     , 2000061. <a href="https://doi.org/10.1002/minf.202000061" title="DOI URL">https://doi.org/10.1002/minf.202000061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.202000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.202000061%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DConsistent%252BCell%2525E2%252580%252590selective%252BAnalog%252BSeries%252Bas%252BConstellation%252BLuminaries%252Bin%252BChemical%252BSpace%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D12%26spage%3D2000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raquel  Rodríguez-Pérez</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Interpretation of machine learning models using shapley values: application to compound potency and multi-target activity predictions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (10)
                                     , 1013-1026. <a href="https://doi.org/10.1007/s10822-020-00314-0" title="DOI URL">https://doi.org/10.1007/s10822-020-00314-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00314-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00314-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DInterpretation%252Bof%252Bmachine%252Blearning%252Bmodels%252Busing%252Bshapley%252Bvalues%25253A%252Bapplication%252Bto%252Bcompound%252Bpotency%252Band%252Bmulti-target%252Bactivity%252Bpredictions%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D10%26spage%3D1013%26epage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eugene N.  Muratov</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Robert P.  Sheridan</span>, <span class="hlFld-ContribAuthor ">Igor V.  Tetko</span>, <span class="hlFld-ContribAuthor ">Dmitry  Filimonov</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>, <span class="hlFld-ContribAuthor ">Tudor I.  Oprea</span>, <span class="hlFld-ContribAuthor ">Igor I.  Baskin</span>, <span class="hlFld-ContribAuthor ">Alexandre  Varnek</span>, <span class="hlFld-ContribAuthor ">Adrian  Roitberg</span>, <span class="hlFld-ContribAuthor ">Olexandr  Isayev</span>, <span class="hlFld-ContribAuthor ">Stefano  Curtalolo</span>, <span class="hlFld-ContribAuthor ">Denis  Fourches</span>, <span class="hlFld-ContribAuthor ">Yoram  Cohen</span>, <span class="hlFld-ContribAuthor ">Alan  Aspuru-Guzik</span>, <span class="hlFld-ContribAuthor ">David A.  Winkler</span>, <span class="hlFld-ContribAuthor ">Dimitris  Agrafiotis</span>, <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>, <span class="hlFld-ContribAuthor ">Alexander  Tropsha</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR without borders. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2020,</strong> <em>49 </em>
                                    (11)
                                     , 3525-3564. <a href="https://doi.org/10.1039/D0CS00098A" title="DOI URL">https://doi.org/10.1039/D0CS00098A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00098A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00098A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DQSAR%252Bwithout%252Bborders%26aulast%3DMuratov%26aufirst%3DEugene%2BN.%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D11%26spage%3D3525%26epage%3D3564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Introducing a new category of activity cliffs combining different compound similarity criteria. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     , 132-141. <a href="https://doi.org/10.1039/C9MD00463G" title="DOI URL">https://doi.org/10.1039/C9MD00463G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00463G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00463G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DIntroducing%252Ba%252Bnew%252Bcategory%252Bof%252Bactivity%252Bcliffs%252Bcombining%252Bdifferent%252Bcompound%252Bsimilarity%252Bcriteria%26aulast%3DHu%26aufirst%3DHuabin%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1%26spage%3D132%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">B. Angélica  Pilón-Jiménez</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">A general approach for retrosynthetic molecular core analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0380-5" title="DOI URL">https://doi.org/10.1186/s13321-019-0380-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0380-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0380-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DA%252Bgeneral%252Bapproach%252Bfor%252Bretrosynthetic%252Bmolecular%252Bcore%252Banalysis%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Introducing a new category of activity cliffs with chemical modifications at multiple sites and rationalizing contributions of individual substitutions. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (16)
                                     , 3605-3612. <a href="https://doi.org/10.1016/j.bmc.2019.06.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.06.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.06.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DIntroducing%252Ba%252Bnew%252Bcategory%252Bof%252Bactivity%252Bcliffs%252Bwith%252Bchemical%252Bmodifications%252Bat%252Bmultiple%252Bsites%252Band%252Brationalizing%252Bcontributions%252Bof%252Bindividual%252Bsubstitutions%26aulast%3DStumpfe%26aufirst%3DDagmar%26date%3D2019%26volume%3D27%26issue%3D16%26spage%3D3605%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Finding Constellations in Chemical Space Through Core Analysis. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00510" title="DOI URL">https://doi.org/10.3389/fchem.2019.00510</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00510%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFinding%252BConstellations%252Bin%252BChemical%252BSpace%252BThrough%252BCore%252BAnalysis%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filip  Miljković</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Data-Driven Exploration of Selectivity and Off-Target Activities of Designated Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (10)
                                     , 2434. <a href="https://doi.org/10.3390/molecules23102434" title="DOI URL">https://doi.org/10.3390/molecules23102434</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23102434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23102434%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DData-Driven%252BExploration%252Bof%252BSelectivity%252Band%252BOff-Target%252BActivities%252Bof%252BDesignated%252BChemical%252BProbes%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DFilip%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D10%26spage%3D2434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Improving the utility of molecular scaffolds for medicinal and computational chemistry. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2018,</strong> <em>10 </em>
                                    (14)
                                     , 1645-1648. <a href="https://doi.org/10.4155/fmc-2018-0106" title="DOI URL">https://doi.org/10.4155/fmc-2018-0106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0106%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DImproving%252Bthe%252Butility%252Bof%252Bmolecular%252Bscaffolds%252Bfor%252Bmedicinal%252Band%252Bcomputational%252Bchemistry%26aulast%3DBajorath%26aufirst%3DJ%25C3%25BCrgen%26date%3D2018%26volume%3D10%26issue%3D14%26spage%3D1645%26epage%3D1648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Collection of analog series-based scaffolds from public compound sources. </span><span class="cited-content_cbyCitation_journal-name">Future Science OA</span><span> <strong>2018,</strong> <em>4 </em>
                                    (4)
                                     , FSO287. <a href="https://doi.org/10.4155/fsoa-2017-0135" title="DOI URL">https://doi.org/10.4155/fsoa-2017-0135</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fsoa-2017-0135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffsoa-2017-0135%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Science%2520OA%26atitle%3DCollection%252Bof%252Banalog%252Bseries-based%252Bscaffolds%252Bfrom%252Bpublic%252Bcompound%252Bsources%26aulast%3DDimova%26aufirst%3DDilyana%26date%3D2018%26volume%3D4%26issue%3D4%26spage%3DFSO287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christin  Rakers</span>, <span class="hlFld-ContribAuthor ">Rifat Ara  Najnin</span>, <span class="hlFld-ContribAuthor ">Ahsan Habib  Polash</span>, <span class="hlFld-ContribAuthor ">Shunichi  Takeda</span>, <span class="hlFld-ContribAuthor ">J.B.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomic Active Learning's Domain of Applicability on Small, Sparse qHTS Matrices: A Study Using Cytochrome P450 and Nuclear Hormone Receptor Families. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 511-521. <a href="https://doi.org/10.1002/cmdc.201700677" title="DOI URL">https://doi.org/10.1002/cmdc.201700677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700677%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DChemogenomic%252BActive%252BLearning%252527s%252BDomain%252Bof%252BApplicability%252Bon%252BSmall%25252C%252BSparse%252BqHTS%252BMatrices%25253A%252BA%252BStudy%252BUsing%252BCytochrome%252BP450%252Band%252BNuclear%252BHormone%252BReceptor%252BFamilies%26aulast%3DRakers%26aufirst%3DChristin%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D6%26spage%3D511%26epage%3D521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Computational design of new molecular scaffolds for medicinal chemistry, part II: generalization of analog series-based scaffolds. </span><span class="cited-content_cbyCitation_journal-name">Future Science OA</span><span> <strong>2018,</strong> <em>4 </em>
                                    (2)
                                     , FSO267. <a href="https://doi.org/10.4155/fsoa-2017-0102" title="DOI URL">https://doi.org/10.4155/fsoa-2017-0102</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fsoa-2017-0102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffsoa-2017-0102%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Science%2520OA%26atitle%3DComputational%252Bdesign%252Bof%252Bnew%252Bmolecular%252Bscaffolds%252Bfor%252Bmedicinal%252Bchemistry%25252C%252Bpart%252BII%25253A%252Bgeneralization%252Bof%252Banalog%252Bseries-based%252Bscaffolds%26aulast%3DDimova%26aufirst%3DDilyana%26date%3D2018%26volume%3D4%26issue%3D2%26spage%3DFSO267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryo  Kunimoto</span>, <span class="hlFld-ContribAuthor ">Tomoyuki  Miyao</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Computational method for estimating progression saturation of analog series. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2018,</strong> <em>8 </em>
                                    (10)
                                     , 5484-5492. <a href="https://doi.org/10.1039/C7RA13748F" title="DOI URL">https://doi.org/10.1039/C7RA13748F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA13748F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA13748F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DComputational%252Bmethod%252Bfor%252Bestimating%252Bprogression%252Bsaturation%252Bof%252Banalog%252Bseries%26aulast%3DKunimoto%26aufirst%3DRyo%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D10%26spage%3D5484%26epage%3D5492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Is scaffold hopping a reliable indicator for the ability of computational methods to identify structurally diverse active compounds?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2017,</strong> <em>31 </em>
                                    (7)
                                     , 603-608. <a href="https://doi.org/10.1007/s10822-017-0032-7" title="DOI URL">https://doi.org/10.1007/s10822-017-0032-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-017-0032-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-017-0032-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIs%252Bscaffold%252Bhopping%252Ba%252Breliable%252Bindicator%252Bfor%252Bthe%252Bability%252Bof%252Bcomputational%252Bmethods%252Bto%252Bidentify%252Bstructurally%252Bdiverse%252Bactive%252Bcompounds%25253F%26aulast%3DDimova%26aufirst%3DDilyana%26date%3D2017%26date%3D2017%26volume%3D31%26issue%3D7%26spage%3D603%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Assessing Scaffold Diversity of Kinase Inhibitors Using Alternative Scaffold Concepts and Estimating the Scaffold Hopping Potential for Different Kinases. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2017,</strong> <em>22 </em>
                                    (5)
                                     , 730. <a href="https://doi.org/10.3390/molecules22050730" title="DOI URL">https://doi.org/10.3390/molecules22050730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules22050730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules22050730%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAssessing%252BScaffold%252BDiversity%252Bof%252BKinase%252BInhibitors%252BUsing%252BAlternative%252BScaffold%252BConcepts%252Band%252BEstimating%252Bthe%252BScaffold%252BHopping%252BPotential%252Bfor%252BDifferent%252BKinases%26aulast%3DDimova%26aufirst%3DDilyana%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D5%26spage%3D730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Swarit  Jasial</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Dark chemical matter in public screening assays and derivation of target hypotheses. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (11)
                                     , 2100-2104. <a href="https://doi.org/10.1039/C7MD00426E" title="DOI URL">https://doi.org/10.1039/C7MD00426E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00426E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00426E%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDark%252Bchemical%252Bmatter%252Bin%252Bpublic%252Bscreening%252Bassays%252Band%252Bderivation%252Bof%252Btarget%252Bhypotheses%26aulast%3DJasial%26aufirst%3DSwarit%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D11%26spage%3D2100%26epage%3D2104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic analysis of structural and activity relationships between conventional hierarchical and analog series-based scaffolds. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2017,</strong> <em>7 </em>
                                    (30)
                                     , 18718-18723. <a href="https://doi.org/10.1039/C7RA01416C" title="DOI URL">https://doi.org/10.1039/C7RA01416C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA01416C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA01416C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSystematic%252Banalysis%252Bof%252Bstructural%252Band%252Bactivity%252Brelationships%252Bbetween%252Bconventional%252Bhierarchical%252Band%252Banalog%252Bseries-based%252Bscaffolds%26aulast%3DStumpfe%26aufirst%3DDagmar%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D30%26spage%3D18718%26epage%3D18723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Activity profiles of analog series containing pan assay interference compounds. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2017,</strong> <em>7 </em>
                                    (57)
                                     , 35638-35647. <a href="https://doi.org/10.1039/C7RA06736D" title="DOI URL">https://doi.org/10.1039/C7RA06736D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA06736D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA06736D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DActivity%252Bprofiles%252Bof%252Banalog%252Bseries%252Bcontaining%252Bpan%252Bassay%252Binterference%252Bcompounds%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D57%26spage%3D35638%26epage%3D35647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Ye  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Analog series-based scaffolds: computational design and exploration of a new type of molecular scaffolds for medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Future Science OA</span><span> <strong>2016,</strong> <em>2 </em>
                                    (4)
                                     , FSO149. <a href="https://doi.org/10.4155/fsoa-2016-0058" title="DOI URL">https://doi.org/10.4155/fsoa-2016-0058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fsoa-2016-0058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffsoa-2016-0058%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Science%2520OA%26atitle%3DAnalog%252Bseries-based%252Bscaffolds%25253A%252Bcomputational%252Bdesign%252Band%252Bexploration%252Bof%252Ba%252Bnew%252Btype%252Bof%252Bmolecular%252Bscaffolds%252Bfor%252Bmedicinal%252Bchemistry%26aulast%3DDimova%26aufirst%3DDilyana%26date%3D2016%26volume%3D2%26issue%3D4%26spage%3DFSO149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Identification of analog series. The computational approach to systematically identify distinct analog series is illustrated that combines MMP and network analysis, as described in the text. The cluster on the right (thick frame) resulting from the union of different compound-based analog subsets represents a disjoint cluster (unique series) from the global analog network.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cluster view of an analog series. Two exemplary clusters from the global analog network with different topology are shown. Nodes are color-coded according to activity profiles. The cluster on the left is a clique (completely connected subgraph) consisting of 10 inhibitors of MAP kinase p38α and tyrosine-protein kinase LCK. All inhibitors are active against one or the other target (single-target activity). The cluster on the right contains ligands of dopamine, norepinephrine, and serotonin transporters as well as the histamine H3 receptor. In this case, all analogs are promiscuous. From each cluster, two exemplary analogs are shown and substitution sites highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analog series and key compounds. Venn diagrams show two small series of analogs with key compounds in the center that cover all MMP relationships and substitution sites within a series as well as all target annotations. In (a) and (b), inhibitors of different matrix metalloproteases and PI3 kinase p110 subunits are displayed, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/medium/jm-2016-00906c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0005.jpeg" id="rightTab-GRAPHIC-d7e861-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Global analog search. A prospective application of the computational approach introduced herein is illustrated. Global analog search calculations are carried out in ChEMBL for a (a) single query compound and (b) small series of analogs. In each case, identified analogs are shown and color-coded by known targets. For each target, the target name and the ChEMBL target identification (in parentheses) are given.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-16/acs.jmedchem.6b00906/20160819/images/large/jm-2016-00906c_0004.jpeg jm-2016-00906c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00906&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span> <span class="citation_source-book">The Practice of Medicinal Chemistry</span>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_contrib-group">Wermuth, C. G.</span>, Ed.; <span class="NLM_publisher-name">Academic Press/Elsevier</span>: <span class="NLM_publisher-loc">San Diego, CA, U.S./London, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+G.+Wermuth&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press%252FElsevier%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Cherkasov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourches, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskin, I. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dearden, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramatica, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todeschini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consonni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuz’min, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benigni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rathman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terfloth, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">QSAR Modeling: Where Have You Been? Where Are You Going To?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4977</span><span class="NLM_x">–</span> <span class="NLM_lpage">5010</span><span class="refDoi"> DOI: 10.1021/jm4004285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4004285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4977-5010&author=A.+Cherkasovauthor=E.+N.+Muratovauthor=D.+Fourchesauthor=A.+Varnekauthor=I.+I.+Baskinauthor=M.+Croninauthor=J.+Deardenauthor=P.+Gramaticaauthor=Y.+C.+Martinauthor=R.+Todeschiniauthor=V.+Consonniauthor=V.+E.+Kuz%E2%80%99minauthor=R.+Cramerauthor=R.+Benigniauthor=C.+Yangauthor=J.+Rathmanauthor=L.+Terflothauthor=J.+Gasteigerauthor=A.+Richardauthor=A.+Tropsha&title=QSAR+Modeling%3A+Where+Have+You+Been%3F+Where+Are+You+Going+To%3F&doi=10.1021%2Fjm4004285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling: Where Have You Been? Where Are You Going To?</span></div><div class="casAuthors">Cherkasov, Artem; Muratov, Eugene N.; Fourches, Denis; Varnek, Alexandre; Baskin, Igor I.; Cronin, Mark; Dearden, John; Gramatica, Paola; Martin, Yvonne C.; Todeschini, Roberto; Consonni, Viviana; Kuzmin, Victor E.; Cramer, Richard; Benigni, Romualdo; Yang, Chihae; Rathman, James; Terfloth, Lothar; Gasteiger, Johann; Richard, Ann; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4977-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quant. structure-activity relationship modeling is one of the major computational tools employed in medicinal chem.  However, throughout its entire history it has drawn both praise and criticism concerning its reliability, limitations, successes, and failures.  In this paper, we discuss (i) the development and evolution of QSAR; (ii) the current trends, unsolved problems, and pressing challenges; and (iii) several novel and emerging applications of QSAR modeling.  Throughout this discussion, we provide guidelines for QSAR development, validation, and application, which are summarized in best practices for building rigorously validated and externally predictive QSAR models.  We hope that this Perspective will help communications between computational and exptl. chemists toward collaborative development and use of QSAR models.  We also believe that the guidelines presented here will help journal editors and reviewers apply more stringent scientific stds. to manuscripts reporting new QSAR studies, as well as encourage the use of high quality, validated QSARs for regulatory decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbLWydJbspbLVg90H21EOLACvtfcHk0lhV2t5jOVr_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE&md5=a2d371ee33fbecf48ef59ecf914e4aa8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm4004285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4004285%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDearden%26aufirst%3DJ.%26aulast%3DGramatica%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DKuz%25E2%2580%2599min%26aufirst%3DV.%2BE.%26aulast%3DCramer%26aufirst%3DR.%26aulast%3DBenigni%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DRathman%26aufirst%3DJ.%26aulast%3DTerfloth%26aufirst%3DL.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520Modeling%253A%2520Where%2520Have%2520You%2520Been%253F%2520Where%2520Are%2520You%2520Going%2520To%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4977%26epage%3D5010%26doi%3D10.1021%2Fjm4004285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Agrafiotis, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shemanarev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farnum, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobanov, V. S.</span><span> </span><span class="NLM_article-title">SAR Maps: A New SAR Visualization Technique for Medicinal Chemists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5926</span><span class="NLM_x">–</span> <span class="NLM_lpage">5937</span><span class="refDoi"> DOI: 10.1021/jm070845m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070845m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5926-5937&author=D.+K.+Agrafiotisauthor=M.+Shemanarevauthor=P.+J.+Connollyauthor=M.+Farnumauthor=V.+S.+Lobanov&title=SAR+Maps%3A+A+New+SAR+Visualization+Technique+for+Medicinal+Chemists&doi=10.1021%2Fjm070845m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm070845m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070845m%26sid%3Dliteratum%253Aachs%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DShemanarev%26aufirst%3DM.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DFarnum%26aufirst%3DM.%26aulast%3DLobanov%26aufirst%3DV.%2BS.%26atitle%3DSAR%2520Maps%253A%2520A%2520New%2520SAR%2520Visualization%2520Technique%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5926%26epage%3D5937%26doi%3D10.1021%2Fjm070845m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Directed R-Group Combination Graph: A Methodology To Uncover Structure-Activity Relationship Patterns in a Series of Analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1215</span><span class="NLM_x">–</span> <span class="NLM_lpage">1226</span><span class="refDoi"> DOI: 10.1021/jm201362h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201362h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1215-1226&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Directed+R-Group+Combination+Graph%3A+A+Methodology+To+Uncover+Structure-Activity+Relationship+Patterns+in+a+Series+of+Analogues&doi=10.1021%2Fjm201362h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm201362h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201362h%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDirected%2520R-Group%2520Combination%2520Graph%253A%2520A%2520Methodology%2520To%2520Uncover%2520Structure-Activity%2520Relationship%2520Patterns%2520in%2520a%2520Series%2520of%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1215%26epage%3D1226%26doi%3D10.1021%2Fjm201362h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">AnalogExplorer: A New Method for Graphical Analysis of Analog Series and Associated Structure-Activity Relationship Information</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9184</span><span class="NLM_x">–</span> <span class="NLM_lpage">9194</span><span class="refDoi"> DOI: 10.1021/jm501391g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501391g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9184-9194&author=B.+Zhangauthor=Y.+Huauthor=J.+Bajorath&title=AnalogExplorer%3A+A+New+Method+for+Graphical+Analysis+of+Analog+Series+and+Associated+Structure-Activity+Relationship+Information&doi=10.1021%2Fjm501391g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm501391g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501391g%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAnalogExplorer%253A%2520A%2520New%2520Method%2520for%2520Graphical%2520Analysis%2520of%2520Analog%2520Series%2520and%2520Associated%2520Structure-Activity%2520Relationship%2520Information%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9184%26epage%3D9194%26doi%3D10.1021%2Fjm501391g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Maynard, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. D.</span><span> </span><span class="NLM_article-title">Quantifying, Visualizing, and Monitoring Lead Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4189</span><span class="NLM_x">–</span> <span class="NLM_lpage">4201</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4189-4201&author=A.+T.+Maynardauthor=C.+D.+Roberts&title=Quantifying%2C+Visualizing%2C+and+Monitoring+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DA.%2BT.%26aulast%3DRoberts%26aufirst%3DC.%2BD.%26atitle%3DQuantifying%252C%2520Visualizing%252C%2520and%2520Monitoring%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4189%26epage%3D4201%26doi%3D10.1021%2Facs.jmedchem.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Shanmugasundaram, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayastha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Vega de León, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Monitoring the Progression of Structure-Activity Relationship Information During Lead Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4235</span><span class="NLM_x">–</span> <span class="NLM_lpage">4244</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4235-4244&author=V.+Shanmugasundaramauthor=L.+Zhangauthor=S.+Kayasthaauthor=A.+de+la+Vega+de+Le%C3%B3nauthor=D.+Dimovaauthor=J.+Bajorath&title=Monitoring+the+Progression+of+Structure-Activity+Relationship+Information+During+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b01428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01428%26sid%3Dliteratum%253Aachs%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKayastha%26aufirst%3DS.%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMonitoring%2520the%2520Progression%2520of%2520Structure-Activity%2520Relationship%2520Information%2520During%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4235%26epage%3D4244%26doi%3D10.1021%2Facs.jmedchem.5b01428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Kenny, P. W.; Sadowski, J.</span><span> </span><span class="NLM_article-title">Structure Modification in Chemical Databases</span>. In  <span class="citation_source-book">Chemoinformatics in Drug Discovery</span>; <span class="NLM_contrib-group">Oprea, T. I.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span>, DOI: <span class="refDoi"> DOI: 10.1002/3527603743.ch11</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1002%2F3527603743.ch11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=271-285&author=P.+W.+Kenny&author=J.+Sadowskiauthor=T.+I.+Oprea&title=Chemoinformatics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F3527603743.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603743.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DStructure%2520Modification%2520in%2520Chemical%2520Databases%26btitle%3DChemoinformatics%2520in%2520Drug%2520Discovery%26aulast%3DOprea%26aufirst%3DT.%2BI.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D271%26epage%3D285%26doi%3D10.1002%2F3527603743.ch11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Griffen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, D. J.</span><span> </span><span class="NLM_article-title">Matched Molecular Pairs as a Medicinal Chemistry Tool</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7739</span><span class="NLM_x">–</span> <span class="NLM_lpage">7750</span><span class="refDoi"> DOI: 10.1021/jm200452d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200452d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Wks7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7739-7750&author=E.+Griffenauthor=A.+G.+Leachauthor=G.+R.+Robbauthor=D.+J.+Warner&title=Matched+Molecular+Pairs+as+a+Medicinal+Chemistry+Tool&doi=10.1021%2Fjm200452d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Matched Molecular Pairs as a Medicinal Chemistry Tool</span></div><div class="casAuthors">Griffen, Ed; Leach, Andrew G.; Robb, Graeme R.; Warner, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7739-7750</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">At the very heart of the role of a medicinal chemist or drug designer is the ability to link chem. structure to mol. properties.  This is the structure-activity relationship(SAR) or structure-property relationship(SPR).  Drug discovery projects often discover that a particular structural change causes a change to a property of interest.  The authors of this paper have been involved in developing and implementing some variations on the original theme of MMPA (matched mol. pairs anal.) and felt this to be the time to highlight some of these and to point toward future developments.  All of them are involved in the process of drug design, and it is within this context that they will set their comments.  In outline, they introduce matched mol. pair anal. and highlight why it fits so well with the kind of problem solving that is required in the process of drug design.  This will be followed by some comments concerning when the application of the methodol. is appropriate.  Then in a section intended for those with more computational interests, the authors describe the various ways that have been devised for identifying such matched pairs.  Throughout the paper, the authors have elected to use a publicly available data set to illustrate some of the comments: the ChEMBL database of inhibitors of the epidermal growth factor receptor tyrosine kinase (EGFR).  This consists of 2899 measurements of an IC50 annotated as a binding measurement on 2348 unique compds.  These measurements are drawn from a no. of sources, having been abstracted from the scientific literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMGFfO8Flm5bVg90H21EOLACvtfcHk0lhsiYB9HgLMKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Wks7jK&md5=fab3344d790d1c0ae688ec65830d9143</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm200452d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200452d%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26atitle%3DMatched%2520Molecular%2520Pairs%2520as%2520a%2520Medicinal%2520Chemistry%2520Tool%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7739%26epage%3D7750%26doi%3D10.1021%2Fjm200452d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wawer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Local Structural Changes, Global Data Views: Graphical Substructure-Activity Relationship Trailing</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2944</span><span class="NLM_x">–</span> <span class="NLM_lpage">2951</span><span class="refDoi"> DOI: 10.1021/jm200026b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200026b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvV2hurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2944-2951&author=M.+Wawerauthor=J.+Bajorath&title=Local+Structural+Changes%2C+Global+Data+Views%3A+Graphical+Substructure-Activity+Relationship+Trailing&doi=10.1021%2Fjm200026b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Local structural changes, global data views: Graphical substructure-activity relationship trailing</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2944-2951</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The systematic extn. of structure-activity relation (SAR) information from large and diverse compd. data sets depends on the application of computational anal. methods.  Irresp. of the methodol. details, the ultimate goal of large-scale SAR anal. is to identify most informative compds. and rationalize structural changes that det. SAR behavior.  Such insights provide a basis for further chem. exploration.  Herein we introduce the first graphical SAR anal. method that globally organizes large compd. data sets on the basis of local structural relationships, hence providing an immediate access to important structural modifications and SAR determinants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPva8Il5aD1rVg90H21EOLACvtfcHk0lhsiYB9HgLMKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvV2hurk%253D&md5=b8d6cb2d9d9bf7aa6e01ccf2f70006c6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm200026b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200026b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLocal%2520Structural%2520Changes%252C%2520Global%2520Data%2520Views%253A%2520Graphical%2520Substructure-Activity%2520Relationship%2520Trailing%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2944%26epage%3D2951%26doi%3D10.1021%2Fjm200026b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Ghosh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Classification of Matching Molecular Series on the Basis of SAR Phenotypes and Structural Relationships</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1039/C5MD00566C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC5MD00566C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1ylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=237-246&author=A.+Ghoshauthor=D.+Dimovaauthor=J.+Bajorath&title=Classification+of+Matching+Molecular+Series+on+the+Basis+of+SAR+Phenotypes+and+Structural+Relationships&doi=10.1039%2FC5MD00566C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of matching molecular series on the basis of SAR phenotypes and structural relationships</span></div><div class="casAuthors">Ghosh, Adhideb; Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">237-246</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In medicinal chem., matched mol. pairs (MMPs) have experienced increasing interest.  An MMP is defined as a pair of compds. that only differ by a structural change at a single site.  MMPs are often used to assoc. specific structural modifications with changes in mol. properties.  As an extension of the MMP concept, matching mol. series (MMSs) have been introduced.  An MMS is defined as a set of compds. with pairwise MMP relationships and thus represents a series of analogs with modifications at a single site.  We have aimed to systematically identify publicly available MMSs, characterize the SAR information encoded by MMSs and explore structural relationships between them.  Therefore, we have searched for MMSs in more than 700 compd. activity classes, systematically explored structural relationships between MMSs, and characterized the structure-activity relationship (SAR) information they contain.  Combining SAR and structural relationship information has made it possible to identify structurally related MMSs with similar or distinct SAR characteristics, which frequently shared compds.  Such MMSs combine series of analogs with different substitution sites and reveal how structural modifications influence SARs.  They can also be used to explore analog pathways that change SAR characteristics and provide addnl. SAR information.  As a part of our study, a comprehensive database of pairs of MMSs with different SAR characteristics is made freely available for medicinal chem. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcNEPVy4JWOLVg90H21EOLACvtfcHk0lhsiYB9HgLMKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1ylsw%253D%253D&md5=2609b5fb4dc1b5fef3f0f9e4cbe98991</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FC5MD00566C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00566C%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DClassification%2520of%2520Matching%2520Molecular%2520Series%2520on%2520the%2520Basis%2520of%2520SAR%2520Phenotypes%2520and%2520Structural%2520Relationships%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D237%26epage%3D246%26doi%3D10.1039%2FC5MD00566C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+Efficient+Algorithm+to+Identify+Matched+Molecular+Pairs+%28MMPs%29+in+Large+Data+Sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lg7XsXUrtDgUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520Efficient%2520Algorithm%2520to%2520Identify%2520Matched%2520Molecular%2520Pairs%2520%2528MMPs%2529%2520in%2520Large%2520Data%2520Sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span> </span><span class="NLM_article-title">RECAP – Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1021/ci970429i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+B.+Juddauthor=S.+P.+Watsonauthor=M.+M.+Hann&title=RECAP+%E2%80%93+Retrosynthetic+Combinatorial+Analysis+Procedure%3A+A+Powerful+New+Technique+for+Identifying+Privileged+Molecular+Fragments+with+Useful+Applications+in+Combinatorial+Chemistry&doi=10.1021%2Fci970429i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lg7XsXUrtDgUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%2BB.%26aulast%3DWatson%26aufirst%3DS.%2BP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DRECAP%2520%25E2%2580%2593%2520Retrosynthetic%2520Combinatorial%2520Analysis%2520Procedure%253A%2520A%2520Powerful%2520New%2520Technique%2520for%2520Identifying%2520Privileged%2520Molecular%2520Fragments%2520with%2520Useful%2520Applications%2520in%2520Combinatorial%2520Chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522%26doi%3D10.1021%2Fci970429i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">de la Vega de León, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Matched Molecular Pairs Derived by Retrosynthetic Fragmentation</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1039/C3MD00259D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC3MD00259D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFGkt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=64-67&author=A.+de+la+Vega+de+Le%C3%B3nauthor=J.+Bajorath&title=Matched+Molecular+Pairs+Derived+by+Retrosynthetic+Fragmentation&doi=10.1039%2FC3MD00259D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Matched molecular pairs derived by retrosynthetic fragmentation</span></div><div class="casAuthors">de la Vega de Leon, Antonio; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-67</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Matched mol. pairs (MMPs) are defined as pairs of compds. that only differ by a chem. change at a single site.  MMPs have become popular in medicinal chem. to support lead optimization, absorption, distribution, metab., excretion, and toxicity (ADMET) anal., and other applications.  Thus far, MMPs have been algorithmically defined and not on the basis of reaction information.  This often limits the chem. interpretability and practical utility of MMPs.  Therefore, we introduce synthetically accessible MMPs that are automatically generated by applying reaction rules following the retrosynthetic combinatorial anal. procedure (RECAP).  A library of more than 92 000 RECAP-MMPs was generated from public domain compds. active against 435 different targets exclusively utilizing high-confidence activity data.  This library is made freely available for use in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqotALy5I0biLVg90H21EOLACvtfcHk0lhmTa7__nvVJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFGkt7fE&md5=5a336f03755927bed0289243caaa1e2f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC3MD00259D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00259D%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMatched%2520Molecular%2520Pairs%2520Derived%2520by%2520Retrosynthetic%2520Fragmentation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D64%26epage%3D67%26doi%3D10.1039%2FC3MD00259D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: A Large-Scale Bioactivity Database for Drug Discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lhmTa7__nvVJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Dimova, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumpfe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Systematic Assessment of Analog Relationships between Bioactive Compounds and Promiscuity of Analog sets</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">230</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1039/C5MD00449G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC5MD00449G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=230-236&author=D.+Dimovaauthor=D.+Stumpfeauthor=J.+Bajorath&title=Systematic+Assessment+of+Analog+Relationships+between+Bioactive+Compounds+and+Promiscuity+of+Analog+sets&doi=10.1039%2FC5MD00449G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC5MD00449G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00449G%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSystematic%2520Assessment%2520of%2520Analog%2520Relationships%2520between%2520Bioactive%2520Compounds%2520and%2520Promiscuity%2520of%2520Analog%2520sets%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D230%26epage%3D236%26doi%3D10.1039%2FC5MD00449G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Compound Promiscuity - What Can We Learn From Current Data</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">644</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1016%2Fj.drudis.2013.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=23524195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=644-650&author=Y.+Huauthor=J.+Bajorath&title=Compound+Promiscuity+-+What+Can+We+Learn+From+Current+Data&doi=10.1016%2Fj.drudis.2013.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Compound promiscuity: what can we learn from current data?</span></div><div class="casAuthors">Hu, Ye; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">644-650</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The specificity paradigm that assigns central relevance to achieving target specificity of drug candidates has recently been revisited.  Evidence is mounting that polypharmacol. drug behavior is often responsible for therapeutic efficacy suggesting the consideration of new drug development strategies.  Target promiscuity of compds. is at the origin of polypharmacol.  For many bioactive compds., multiple target annotations are available indicating that compd. promiscuity is a general phenomenon.  However, careful anal. of compd. activity data reveals that the degree of apparent promiscuity is strongly influenced by data selection criteria and the type of activity measurements that are considered.  Furthermore, promiscuity involving unrelated targets is less common.  Apparent target promiscuity might often better be interpreted as activity promiscuity in different assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrQ19HLWfCarVg90H21EOLACvtfcHk0liYLY9-7LPiUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGru7Y%253D&md5=1ec94bf333ae5da9d35aa60f06366436</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCompound%2520Promiscuity%2520-%2520What%2520Can%2520We%2520Learn%2520From%2520Current%2520Data%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D644%26epage%3D650%26doi%3D10.1016%2Fj.drudis.2013.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">High-Resolution View of Compound Promiscuity</span> <span class="citation_source-journal">F1000Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="refDoi"> DOI: 10.12688/f1000research.2-144.v2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.12688%2Ff1000research.2-144.v2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=24358872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A280%3ADC%252BC2c3psVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=144&author=Y.+Huauthor=J.+Bajorath&title=High-Resolution+View+of+Compound+Promiscuity&doi=10.12688%2Ff1000research.2-144.v2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution view of compound promiscuity</span></div><div class="casAuthors">Hu Ye; Bajorath Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144</span>
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    </div><div class="casAbstract">Compound promiscuity is defined as the ability of a small molecule to specifically interact with multiple biological targets.  So-defined promiscuity is relevant for drug discovery because it provides the molecular basis of polypharmacology, which is increasingly implicated in the therapeutic efficacy of drugs.  Recent studies have analyzed different aspects of compound promiscuity on the basis of currently available activity data.  In this commentary, we present take-home messages from these studies augmented with new results to generate a detailed picture of compound promiscuity that might serve as a reference for further discussions and research activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRhg3urNXDByO2kE7krGOBfW6udTcc2eZC8glJQc1H9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3psVWrug%253D%253D&md5=f96841e071505f8e25de1ef0766dc1a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.2-144.v2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.2-144.v2%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DHigh-Resolution%2520View%2520of%2520Compound%2520Promiscuity%26jtitle%3DF1000Research%26date%3D2013%26volume%3D2%26spage%3D144%26doi%3D10.12688%2Ff1000research.2-144.v2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Network Pharmacology: The Next Paradigm in Drug Discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">682</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1038/nchembio.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1038%2Fnchembio.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=18936753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=682-690&author=A.+L.+Hopkins&title=Network+Pharmacology%3A+The+Next+Paradigm+in+Drug+Discovery&doi=10.1038%2Fnchembio.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Network pharmacology: the next paradigm in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is the concept of designing maximally selective ligands to act on individual drug targets.  However, many effective drugs act via modulation of multiple proteins rather than single targets.  Advances in systems biol. are revealing a phenotypic robustness and a network structure that strongly suggests that exquisitely selective compds., compared with multi-target drugs, may exhibit lower than desired clin. efficacy.  This new appreciation of the role of polypharmacol. has significant implications for tackling the two major sources of attrition in drug development, efficacy and toxicity.  Integrating network biol. and polypharmacol. holds the promise of expanding the current opportunity space for druggable targets.  However, the rational design of polypharmacol. faces considerable challenges in the need for new methods to validate target combinations and optimize multiple structure-activity relationships while maintaining drug-like properties.  Advances in these areas are creating the foundation of the next paradigm in drug discovery: network pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqHy4FqkK-h7Vg90H21EOLACvtfcHk0liYLY9-7LPiUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK&md5=03b8be3ddbcab7dc65afd17df7ee8395</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.118%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DNetwork%2520Pharmacology%253A%2520The%2520Next%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D682%26epage%3D690%26doi%3D10.1038%2Fnchembio.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Jalencas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">On the Origins of Drug Polypharmacology</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1039/C2MD20242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1039%2FC2MD20242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=80-87&author=X.+Jalencasauthor=J.+Mestres&title=On+the+Origins+of+Drug+Polypharmacology&doi=10.1039%2FC2MD20242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the origins of drug polypharmacology</span></div><div class="casAuthors">Jalencas, Xavier; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-87</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The ability of small mols. to interact with multiple proteins is commonly referred to as polypharmacol.  The now widely accepted polypharmacol. of drugs is of particular interest for human health as it has implications beyond therapeutic efficacy, from anticipating adverse drug reactions to identifying potential repurposing opportunities.  There have been a no. of studies relating the extent of drug polypharmacol. to the physicochem. properties and fragment compn. of the drug itself, but also to the protein family and distant binding site similarities of the drug's primary target.  Taken together, all these observations lead to speculate that the origins of drug polypharmacol. may lie at the heart of protein evolution and that polypharmacol. may just be a reminiscent signature of some of the mechanisms of adaptation that primitive biol. systems developed to increase the chances of survival in a highly variable early chem. environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1qsCiSthHLVg90H21EOLACvtfcHk0liGsSNvRB9jiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI&md5=77fe84731536e5008cca3b0faf1e58dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC2MD20242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20242E%26sid%3Dliteratum%253Aachs%26aulast%3DJalencas%26aufirst%3DX.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DOn%2520the%2520Origins%2520of%2520Drug%2520Polypharmacology%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D80%26epage%3D87%26doi%3D10.1039%2FC2MD20242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Anighoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="NLM_x">–</span> <span class="NLM_lpage">7887</span><span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0liGsSNvRB9jiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzek, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karapetyan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dracheva, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem’s BioAssay Database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D400</span><span class="NLM_x">–</span> <span class="NLM_lpage">D412</span><span class="refDoi"> DOI: 10.1093/nar/gkr1132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkr1132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=22140110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D400-D412&author=Y.+Wangauthor=J.+Xiaoauthor=T.+O.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=Z.+Zhouauthor=L.+Hanauthor=K.+Karapetyanauthor=S.+Drachevaauthor=B.+A.+Shoemakerauthor=E.+Boltonauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=PubChem%E2%80%99s+BioAssay+Database&doi=10.1093%2Fnar%2Fgkr1132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem's BioAssay Database</span></div><div class="casAuthors">Wang, Yanli; Xiao, Jewen; Suzek, Tugba O.; Zhang, Jian; Wang, Jiyao; Zhou, Zhigang; Han, Lianyi; Karapetyan, Karen; Dracheva, Svetlana; Shoemaker, Benjamin A.; Bolton, Evan; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D400-D412</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem (http://pubchem.ncbi.nlm.nih.gov) is a public repository for biol. activity data of small mols. and RNAi reagents.  The mission of PubChem is to deliver free and easy access to all deposited data, and to provide intuitive data anal. tools.  The PubChem BioAssay database currently contains 500 000 descriptions of assay protocols, covering 5000 protein targets, 30 000 gene targets and providing over 130 million bioactivity outcomes.  PubChem's bioassay data are integrated into the NCBI Entrez information retrieval system, thus making PubChem data searchable and accessible by Entrez queries.  Also, as a repository, PubChem constantly optimizes and develops its deposition system answering many demands of both high- and low-vol. depositors.  The PubChem information platform allows users to search, review and download bioassay description and data.  The PubChem platform also enables researchers to collect, compare and analyze biol. test results through web-based and programmatic tools.  In this work, we provide an update for the PubChem BioAssay resource, including information content growth, data model extension and new developments of data submission, retrieval, anal. and download tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISH42-uje7LVg90H21EOLACvtfcHk0liGsSNvRB9jiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzI&md5=f172bdd6db3d31af6aed30dde4d7eeee</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1132%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DSuzek%26aufirst%3DT.%2BO.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DKarapetyan%26aufirst%3DK.%26aulast%3DDracheva%26aufirst%3DS.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DBolton%26aufirst%3DE.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%25E2%2580%2599s%2520BioAssay%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD400%26epage%3DD412%26doi%3D10.1093%2Fnar%2Fgkr1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Law, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, A.</span><span> </span><span class="NLM_article-title">DrugBank 4.0: Shedding New Light on Drug Metabolism</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1097</span><span class="refDoi"> DOI: 10.1093/nar/gkt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1093%2Fnar%2Fgkt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=24203711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Kh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1091-D1097&author=V.+Lawauthor=C.+Knoxauthor=Y.+Djoumbouauthor=T.+Jewisonauthor=A.+C.+Guoauthor=Y.+Liuauthor=A.+Tang&title=DrugBank+4.0%3A+Shedding+New+Light+on+Drug+Metabolism&doi=10.1093%2Fnar%2Fgkt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 4.0: shedding new light on drug metabolism</span></div><div class="casAuthors">Law, Vivian; Knox, Craig; Djoumbou, Yannick; Jewison, Tim; Guo, An Chi; Liu, Yifeng; Maciejewski, Adam; Arndt, David; Wilson, Michael; Neveu, Vanessa; Tang, Alexandra; Gabriel, Geraldine; Ly, Carol; Adamjee, Sakina; Dame, Zerihun T.; Han, Beomsoo; Zhou, You; Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1091-D1097</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank (http://www.drugbank.ca) is a comprehensive online database contg. extensive biochem. and pharmacol. information about drugs, their mechanisms and their targets.  Since it was first described in 2006, DrugBank has rapidly evolved, both in response to user requests and in response to changing trends in drug research and development.  Previous versions of DrugBank have been widely used to facilitate drug and in silico drug target discovery.  The latest update, DrugBank 4.0, has been further expanded to contain data on drug metab., absorption, distribution, metab., excretion and toxicity (ADMET) and other kinds of quant. structure activity relationships (QSAR) information.  These enhancements are intended to facilitate research in xenobiotic metab. (both prediction and characterization), pharmacokinetics, pharmacodynamics and drug design/discovery.  For this release, >1200 drug metabolites (including their structures, names, activity, abundance and other detailed data) have been added along with >1300 drug metab. reactions (including metabolizing enzymes and reaction types) and dozens of drug metab. pathways.  Another 30 predicted or measured ADMET parameters have been added to each DrugCard, bringing the av. no. of quant. ADMET values for Food and Drug Administration-approved drugs close to 40.  Referential NMR and MS spectra have been added for almost 400 drugs as well as spectral and mass matching tools to facilitate compd. identification.  This expanded collection of drug information is complemented by a no. of new or improved search tools, including one that provides a simple analyses of drug-target, -enzyme and -transporter assocns. to provide insight on drug-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_BsUO1phm7Vg90H21EOLACvtfcHk0liGsSNvRB9jiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Kh&md5=0da9f3ecfc2061f814803cf4b070e54e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1068%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DV.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DA.%26atitle%3DDrugBank%25204.0%253A%2520Shedding%2520New%2520Light%2520on%2520Drug%2520Metabolism%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1091%26epage%3DD1097%26doi%3D10.1093%2Fnar%2Fgkt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Berthold, M. R.; Cebron, N.; Dill, F.; Gabriel, T. R.; Kötter, T.; Meinl, T.; Ohl, P.; Sieb, C.; Thiel, K.; Wiswedel, B.</span><span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. In  <span class="citation_source-book">Studies in Classification, Data Analysis, and Knowledge Organization</span>; <span class="NLM_contrib-group">Preisach, C.; Burkhardt, H.; Schmidt-Thieme, L.; Decker, R.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-3-540-78246-9_38</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Berthold&author=N.+Cebron&author=F.+Dill&author=T.+R.+Gabriel&author=T.+K%C3%B6tter&author=T.+Meinl&author=P.+Ohl&author=C.+Sieb&author=K.+Thiel&author=B.+Wiswedelauthor=C.+Preisach&author=H.+Burkhardt&author=L.+Schmidt-Thieme&author=R.+Decker&title=Studies+in+Classification%2C+Data+Analysis%2C+and+Knowledge+Organization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26btitle%3DStudies%2520in%2520Classification%252C%2520Data%2520Analysis%252C%2520and%2520Knowledge%2520Organization%26aulast%3DPreisach%26aufirst%3DC.%26pub%3DSpringer%26date%3D2008%26spage%3D319%26epage%3D326%26doi%3D10.1007%2F978-3-540-78246-9_38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span>OEChem TK; <span class="NLM_publisher-name">OpenEye Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM, U.S.</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OEChem+TK%3B+OpenEye+Scientific+Software%2C+Inc.%3A+Santa+Fe%2C+NM%2C+U.S.%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOEChem%2520TK%26pub%3DOpenEye%2520Scientific%2520Software%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumpfe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">MMP-cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1138</span><span class="NLM_x">–</span> <span class="NLM_lpage">1145</span><span class="refDoi"> DOI: 10.1021/ci3001138</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001138" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1138-1145&author=X.+Huauthor=Y.+Huauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=MMP-cliffs%3A+Systematic+Identification+of+Activity+Cliffs+on+the+Basis+of+Matched+Molecular+Pairs&doi=10.1021%2Fci3001138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span></div><div class="casAuthors">Hu, Xiaoying; Hu, Ye; Vogt, Martin; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1138-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs are generally defined as pairs of structurally similar compds. having large differences in potency.  The anal. of activity cliffs is of general interest because structure-activity relationship (SAR) determinants can often be deduced from them.  Crit. questions for the study of activity cliffs include how similar compds. should be to qualify as cliff partners, how similarity should be assessed, and how large potency differences between participating compds. should be.  Thus far, activity cliffs have mostly been defined on the basis of calcd. Tanimoto similarity values using structural descriptors, esp. 2D fingerprints.  As any theor. assessment of mol. similarity, this approach has its limitations.  For example, calcd. Tanimoto similarities might often be difficult to reconcile and interpret from a chem. perspective, a point of critique frequently raised in medicinal chem.  Herein, we have explored activity cliffs by considering well-defined substructure replacements instead of calcd. similarity values.  For this purpose, the matched mol. pair (MMP) formalism has been applied.  MMPs were systematically derived from public domain compds., and activity cliffs were extd. from them, termed MMP-cliffs.  The frequency of cliff formation was detd. for compds. active against different targets, MMP-cliffs were analyzed in detail, and re-evaluated on the basis of Tanimoto similarity.  In many instances, chem. intuitive activity cliffs were only detected on the basis of MMPs, but not Tanimoto similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYSOxNqPjGbVg90H21EOLACvtfcHk0lgn9VySyKugHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D&md5=958a76d85d0a06424b43eb1b694cf3ca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci3001138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001138%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMMP-cliffs%253A%2520Systematic%2520Identification%2520of%2520Activity%2520Cliffs%2520on%2520the%2520Basis%2520of%2520Matched%2520Molecular%2520Pairs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1138%26epage%3D1145%26doi%3D10.1021%2Fci3001138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Shannon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markiel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozier, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baliga, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramage, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwikowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ideker, T.</span><span> </span><span class="NLM_article-title">Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks</span> <span class="citation_source-journal">Genome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2498</span><span class="NLM_x">–</span> <span class="NLM_lpage">2504</span><span class="refDoi"> DOI: 10.1101/gr.1239303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=10.1101%2Fgr.1239303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=14597658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&author=P.+Shannonauthor=A.+Markielauthor=O.+Ozierauthor=N.+S.+Baligaauthor=J.+T.+Wangauthor=D.+Ramageauthor=N.+Aminauthor=B.+Schwikowskiauthor=T.+Ideker&title=Cytoscape%3A+A+Software+Environment+for+Integrated+Models+of+Biomolecular+Interaction+Networks&doi=10.1101%2Fgr.1239303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Cytoscape: A software environment for integrated models of biomolecular interaction networks</span></div><div class="casAuthors">Shannon, Paul; Markiel, Andrew; Ozier, Owen; Baliga, Nitin S.; Wang, Jonathan T.; Ramage, Daniel; Amin, Nada; Schwikowski, Benno; Ideker, Trey</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2504</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cytoscape is an open source software project for integrating biomol. interaction networks with high-throughput expression data and other mol. states into a unified conceptual framework.  Although applicable to any system of mol. components and interactions, Cytoscape is most powerful when used in conjunction with large databases of protein-protein, protein-DNA, and genetic interactions that are increasingly available for humans and model organisms.  Cytoscape's software Core provides basic functionality to layout and query the network; to visually integrate the network with expression profiles, phenotypes, and other mol. states; and to link the network to databases of functional annotations.  The Core is extensible through a straightforward plug-in architecture, allowing rapid development of addnl. computational analyses and features.  Several case studies of Cytoscape plug-ins are surveyed, including a search for interaction pathways correlating with changes in gene expression, a study of protein complexes involved in cellular recovery to DNA damage, inference of a combined phys./functional interaction network for Halobacterium, and an interface to detailed stochastic/kinetic gene regulatory models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_cu7mgf2RbVg90H21EOLACvtfcHk0lgn9VySyKugHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D&md5=2bcbca9a3bd04717761f0424c0209e43</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1101%2Fgr.1239303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.1239303%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DP.%26aulast%3DMarkiel%26aufirst%3DA.%26aulast%3DOzier%26aufirst%3DO.%26aulast%3DBaliga%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DJ.%2BT.%26aulast%3DRamage%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DIdeker%26aufirst%3DT.%26atitle%3DCytoscape%253A%2520A%2520Software%2520Environment%2520for%2520Integrated%2520Models%2520of%2520Biomolecular%2520Interaction%2520Networks%26jtitle%3DGenome%2520Res.%26date%3D2003%26volume%3D13%26spage%3D2498%26epage%3D2504%26doi%3D10.1101%2Fgr.1239303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span>Zenodo. <a href="https://zenodo.org/record/59688" class="extLink">https://zenodo.org/record/59688</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Zenodo.+https%3A%2F%2Fzenodo.org%2Frecord%2F59688."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-16%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00906" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00906" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ab3368fd12254","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
