
<html lang="en"     class="pb-page"  data-request-id="88efc7bc-7a1c-4978-812c-ebee2c1a3216"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00622;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="RNA G-Quadruplexes in Kirsten Ras (KRAS) Oncogene as Targets for Small Molecules Inhibiting Translation" /></meta><meta name="dc.Creator" content="Giulia  Miglietta" /></meta><meta name="dc.Creator" content="Susanna  Cogoi" /></meta><meta name="dc.Creator" content="Jessica  Marinello" /></meta><meta name="dc.Creator" content="Giovanni  Capranico" /></meta><meta name="dc.Creator" content="Alexander S.  Tikhomirov" /></meta><meta name="dc.Creator" content="Andrey  Shchekotikhin" /></meta><meta name="dc.Creator" content="Luigi E.  Xodo" /></meta><meta name="dc.Description" content="The human KRAS transcript contains a G-rich 5′-UTR sequence (77% GC) harboring several G4 motifs capable to form stable RNA G-quadruplex (RG4) structures that can serve as targets for small molecul..." /></meta><meta name="Description" content="The human KRAS transcript contains a G-rich 5′-UTR sequence (77% GC) harboring several G4 motifs capable to form stable RNA G-quadruplex (RG4) structures that can serve as targets for small molecul..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 1, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00622" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00622" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00622" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00622" /></link>
        
    
    

<title>RNA G-Quadruplexes in Kirsten Ras (KRAS) Oncogene as Targets for Small Molecules Inhibiting Translation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00622" /></meta><meta property="og:title" content="RNA G-Quadruplexes in Kirsten Ras (KRAS) Oncogene as Targets for Small Molecules Inhibiting Translation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0012.jpeg" /></meta><meta property="og:description" content="The human KRAS transcript contains a G-rich 5′-UTR sequence (77% GC) harboring several G4 motifs capable to form stable RNA G-quadruplex (RG4) structures that can serve as targets for small molecules. A biotin–streptavidin pull-down assay showed that 4,11-bis(2-aminoethylamino)anthra[2,3-b]furan-5,10-dione (2a) binds to RG4s in the KRAS transcript under low-abundance cellular conditions. Dual-luciferase assays demonstrated that 2a and its analogue 4,11-bis(2-aminoethylamino)anthra[2,3-b]thiophene-5,10-dione (2b) repress translation in a dose-dependent manner. The effect of the G4-ligands on Panc-1 cancer cells has also been examined. Both 2a and 2b efficiently penetrate the cells, suppressing protein p21KRAS to &lt;10% of the control. The KRAS down-regulation induces apoptosis together with a dramatic reduction of cell growth and colony formation. In summary, we report a strategy to suppress the KRAS oncogene in pancreatic cancer cells by means of small molecules binding to RG4s in the 5′-UTR of mRNA." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00622"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00622">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00622&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00622&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00622&amp;href=/doi/10.1021/acs.jmedchem.7b00622" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 9448-9461</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01520" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00646" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">RNA G-Quadruplexes in Kirsten Ras (<i>KRAS</i>) Oncogene as Targets for Small Molecules Inhibiting Translation</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giulia++Miglietta">Giulia Miglietta</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susanna++Cogoi">Susanna Cogoi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jessica++Marinello">Jessica Marinello</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giovanni++Capranico">Giovanni Capranico</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8708-6454" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+S.++Tikhomirov">Alexander S. Tikhomirov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrey++Shchekotikhin">Andrey Shchekotikhin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luigi+E.++Xodo">Luigi E. Xodo</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3344-7207" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Medicine, Biochemistry Laboratory, University of Udine, 33100 Udine, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmacy and Biotechnology, University of Bologna, 40100 Bologna, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Gause Institute of New Antibiotics, 119021 Moscow, Russia</span></div><div class="corresp-info"><strong>*</strong>Tel: +39.0432.494395. Fax: +39.0432.494301. E-mail: <a href="/cdn-cgi/l/email-protection#1c7069757b7532647378735c6972756978327568"><span class="__cf_email__" data-cfemail="3f534a5658561147505b507f4a51564a5b11564b">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00622&amp;href=/doi/10.1021%2Facs.jmedchem.7b00622" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 9448–9461</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 15, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 May 2017</li><li><span class="item_label"><b>Published</b> online</span>1 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 December 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00622" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00622</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9448%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGiulia%2BMiglietta%252C%2BSusanna%2BCogoi%252C%2BJessica%2BMarinello%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D23%26contentID%3Dacs.jmedchem.7b00622%26title%3DRNA%2BG-Quadruplexes%2Bin%2BKirsten%2BRas%2B%2528KRAS%2529%2BOncogene%2Bas%2BTargets%2Bfor%2BSmall%2BMolecules%2BInhibiting%2BTranslation%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9461%26publicationDate%3DDecember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00622"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2888</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00622" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;RNA G-Quadruplexes in Kirsten Ras (KRAS) Oncogene as Targets for Small Molecules Inhibiting Translation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Giulia&quot;,&quot;last_name&quot;:&quot;Miglietta&quot;},{&quot;first_name&quot;:&quot;Susanna&quot;,&quot;last_name&quot;:&quot;Cogoi&quot;},{&quot;first_name&quot;:&quot;Jessica&quot;,&quot;last_name&quot;:&quot;Marinello&quot;},{&quot;first_name&quot;:&quot;Giovanni&quot;,&quot;last_name&quot;:&quot;Capranico&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;S. Tikhomirov&quot;},{&quot;first_name&quot;:&quot;Andrey&quot;,&quot;last_name&quot;:&quot;Shchekotikhin&quot;},{&quot;first_name&quot;:&quot;Luigi&quot;,&quot;last_name&quot;:&quot;E. Xodo&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;9448-9461&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00622&quot;},&quot;abstract&quot;:&quot;The human KRAS transcript contains a G-rich 5′-UTR sequence (77% GC) harboring several G4 motifs capable to form stable RNA G-quadruplex (RG4) structures that can serve as targets for small molecules. A biotin–streptavidin pull-down assay showed that 4,11-bis(2-aminoethylamino)anthra[2,3-b]furan-5,10-dione (2a) binds to RG4s in the KRAS transcript under low-abundance cellular conditions. Dual-luciferase assays demonstrated that 2a and its analogue 4,11-bis(2-aminoethylamino)anthra[2,3-b]thiophene-5,10-dione (2b) repress translation in a dose-dependent manner. The effect of the G4-ligands on Panc-1 cancer cells has also been examined. Both 2a and 2b efficiently penetrate the cells, suppressing protein p21KRAS to &lt;10% of the control. The KRAS down-regulation induces apoptosis together with a dramatic reduction of cell growth and colony formation. In summary, we report a strategy to suppress the KRAS oncogene in pancreatic cancer cells by means of small molecules binding to RG4s in the 5′-UTR of mRNA.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00622&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00622" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00622&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00622" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00622&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00622" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00622&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00622&amp;href=/doi/10.1021/acs.jmedchem.7b00622" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00622" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00622" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00622%26sid%3Dliteratum%253Aachs%26pmid%3D29140695%26genre%3Darticle%26aulast%3DMiglietta%26date%3D2017%26atitle%3DRNA%2BG-Quadruplexes%2Bin%2BKirsten%2BRas%2B%2528KRAS%2529%2BOncogene%2Bas%2BTargets%2Bfor%2BSmall%2BMolecules%2BInhibiting%2BTranslation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D23%26spage%3D9448%26epage%3D9461%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290694" title="Bioluminescent probes">Bioluminescent probes</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/jmcmar.2017.60.issue-23/20171214/jmcmar.2017.60.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The human <i>KRAS</i> transcript contains a G-rich 5′-UTR sequence (77% GC) harboring several G4 motifs capable to form stable RNA G-quadruplex (RG4) structures that can serve as targets for small molecules. A biotin–streptavidin pull-down assay showed that 4,11-bis(2-aminoethylamino)anthra[2,3-<i>b</i>]furan-5,10-dione (<b>2a</b>) binds to RG4s in the <i>KRAS</i> transcript under low-abundance cellular conditions. Dual-luciferase assays demonstrated that <b>2a</b> and its analogue 4,11-bis(2-aminoethylamino)anthra[2,3-<i>b</i>]thiophene-5,10-dione (<b>2b</b>) repress translation in a dose-dependent manner. The effect of the G4-ligands on Panc-1 cancer cells has also been examined. Both <b>2a</b> and <b>2b</b> efficiently penetrate the cells, suppressing protein p21KRAS to <10% of the control. The <i>KRAS</i> down-regulation induces apoptosis together with a dramatic reduction of cell growth and colony formation. In summary, we report a strategy to suppress the <i>KRAS</i> oncogene in pancreatic cancer cells by means of small molecules binding to RG4s in the 5′-UTR of mRNA.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06218" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06218" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The three ras genes (<i>HRAS</i>, <i>KRAS</i>, and <i>NRAS</i>) encode for highly homologous (83–90% sequence identity) GTPases of 21 kDa that cycle between an active GTP-bound and an inactive GDP-bound state.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> This cycling is mediated by guanine nucleotides exchange factors (GEFs) and by GTPase activating proteins (GAPs).<a onclick="showRef(event, 'ref1 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref3">(1, 3)</a> In the GTP-bound state, the p21RAS protein interacts with downstream effectors, activating specific cellular processes including proliferation, survival, and differentiation.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Mutations in the ras genes are estimated to be present in ∼30% of all human cancers.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> However, in pancreatic ductal adenocarcinoma (PDAC), <i>KRAS</i> is mutated in ∼95% of patients.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> The mutant alleles carry a single missense point mutation in exon 1, codon 12, 13, or 61, which impairs GAP-mediated GTP-to-GDP hydrolysis. This results in an aberrant protein that is locked into the activated state, transmitting constitutively signals for proliferation to the nucleus.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> According to recent studies, mutations in the <i>KRAS</i> gene can be seen as primary genetic lesions that initiate the malignant transformation of pancreatic cells.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> Progression to invasive PDAC occurs through a stepwise accumulation of other genetic lesions, in particular those causing the inactivation of tumor suppressor genes.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p last">Recent studies have demonstrated that <i>KRAS</i> is essential for the maintenance of PDAC as it reprograms the metabolism of glucose and glutamine to fuel a high proliferation rate.<a onclick="showRef(event, 'ref9 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref12 ref13">(9, 12, 13)</a> The dependence of metabolic pathways on specific oncogenes has led to the concept of “oncogene addiction”, which means that, although cancer cells may depend on a number of genetic aberrations, they often develop a dependency on a particular oncogene.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Considering the central role played by <i>KRAS</i> on the pathogenesis of PDAC, <i>KRAS</i> is considered a crucial target for anticancer drugs. However, despite more than two decades of research, up to now no antiras drugs have reached the clinic, creating the impression that ras genes may be “undruggable”.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> Recently, the design of new inhibitors binding directly to protein p21RAS has fueled research in this direction.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Other strategies that are being pursued use drugs that inhibit either the association of p21RAS to the membrane or the activity of downstream pathways.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19-22)</a> In our laboratory, we developed an alternative anti <i>KRAS</i> strategy by focusing on two targets for small molecules: (i) a G4-motif located upstream of the transcription start site (TSS) (−320/–306), which is recognized by essential transcription factors (MAZ, Ku70, PARP-1, and hnRNP A1);<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (ii) G4-motifs located in the 5′-untranslated region (5′-UTR) of the <i>KRAS</i> transcript.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> Previous studies have shown that the presence of G4 motifs in 5′-UTR of mRNA inhibits translation, on the basis of luciferase assays.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> In this study, we demonstrate by a streptavidin–biotin pull-down assay that small molecules bind to RNA G-quadruplexes (RG4s) formed in the 5′-UTR of low-abundant cellular <i>KRAS</i> transcripts. These small molecules, anthrafurandiones (ATFD) and anthrathiophenediones (ATPD), suppress luciferase from specific vectors as well as p21RAS in pancreatic Panc-1 cancer cells. ATFD and ATPD strongly induce apoptosis and reduce the metabolic activity and colony formation of Panc-1 cells carrying mutant <i>KRAS</i> G12D. Our results demonstrate that ATPD and ATFD are promising therapeutic drugs to suppress oncogenic <i>KRAS</i> in pancreatic cancer cells, through their binding to RG4s located in the 5′-UTR of mRNA.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since Hurley and co-workers published their seminal study on the transcription of <i>CMYC</i> controlled by a regulatory mechanism involving a G4 DNA,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> a great number of papers on this unusual structure has been reported in the literature.<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33 ref34 ref35">(29-35)</a> The biological function of G4 DNA is rather complex: some studies indicate that it behaves as a transcriptional repressor,<a onclick="showRef(event, 'ref28 ref32 ref35'); return false;" href="javascript:void(0);" class="ref ref28 ref32 ref35">(28, 32, 35)</a> others that G4 DNA is associated with transcriptionally active chromatin.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Whatever its function is, G4 DNA is an attractive target for therapeutic small molecules.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> However, bioinformatics analyses showed that the transcript of certain oncogenes contains a 5′-UTR rich in guanines that can form G4 RNA structures (RG4s).<a onclick="showRef(event, 'ref26 ref27 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref37 ref38 ref39 ref40">(26, 27, 37-40)</a> The 5′-UTR region of mRNA plays an essential role in the initiation of translation. This occurs through three steps: (i) association of the 7-methylguanosine cap with the 43 S initiation complex; (ii) scanning of complex 43 S along 5′-UTR up to codon AUG; (iii) assembly of a larger 80 S ribosome that proceeds with protein synthesis. The process is normally regulated by eukaryotic translation initiation factors (eIFs). In addition, certain transcripts use a cap-independent translation initiation that depends on internal ribosomal entry sites.<a onclick="showRef(event, 'ref26 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref26 ref39 ref40">(26, 39, 40)</a> It has been hypothesized that excess secondary structure including RG4s in 5′-UTR may have a regulatory function on translation.<a onclick="showRef(event, 'ref26 ref27 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref37 ref38 ref39 ref40">(26, 27, 37-40)</a> Indeed, as observed with RNA hairpins, RG4s can either inhibit the assembly of the translation initiation machinery at the 5′-cap or interfere with the scanning of the ribosome toward the AUG codon (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S1">Supporting Information S1</a>).<a onclick="showRef(event, 'ref26 ref41 ref42 ref43 ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref26 ref41 ref42 ref43 ref44 ref45 ref46 ref47">(26, 41-47)</a> Luciferase data obtained with expression plasmids designed with G4 in specific positions upstream of AUG have been reported as a proof-of-principle.<a onclick="showRef(event, 'ref26 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref26 ref43 ref44">(26, 43, 44)</a> In our work we demonstrate that furan- and thiophene-fused anthraquinone derivatives bind to RG4s in the 5′-UTR of <i>KRAS</i> mRNA under low-abundance cellular conditions. These G4-specific molecules are found to suppress the <i>KRAS</i> oncogene in pancreatic cancer cells through a mechanism involving the inhibition of translation.</div><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Human <i>KRAS</i> 5′-UTR Forms RNA G-Quadruplex Structures</h3><div class="NLM_p">The human <i>KRAS</i> transcript contains a 5′-UTR of 192 nucleotides, characterized by a high GC content (77%) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Its putative secondary structure can be predicted by Mfold,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> a bioinformatic tool that generates a secondary structure by taking into consideration the orthologues of the RNA sequence. However, Mfold does not take into account G4 motifs; thus, one wonders if the secondary structure predicted for the <i>KRAS</i> 5′-UTR is actually formed, considering that it contains 33 GG runs which give rise to a multitude of G4 motifs. To identify the G4 motifs with the highest G-score (i.e., highest propensity to fold into a G4), we interrogated QGRS Mapper.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> We considered a consensus G4 motif composed by 2 G-tetrads and loop length up to 12 nt. The analysis gave three nonoverlapping G4 motifs with a G-score = 21 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). If the overlapping G4 motifs were included in the analysis, their number was >300, suggesting that the human <i>KRAS</i> 5′-UTR sequence has a high propensity to form RG4s. The three nonoverlapping G4 motifs are located within the first 80 nt of 5′-UTR, and each displays a circular dichroism spectrum characterized by a strong ellipticity at 265 nm and a negative ellipticity at 240 nm, typical of parallel G-quadruplexes, as does the well-known G4-motif located in the 5′-UTR of <i>NRAS</i> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> A).<a onclick="showRef(event, 'ref26 ref50'); return false;" href="javascript:void(0);" class="ref ref26 ref50">(26, 50)</a> The G4 motifs show UV-thermal difference spectra characterized by a negative peak at 295 nm in K<sup>+</sup>, but not in Li<sup>+</sup>, buffer, a distinctive feature of G4 structures (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S2">Supporting Information S2</a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> They show cooperative UV-melting curves at 296 nm (as well as at 260 nm), with <i>T</i><sub>M</sub> varying from 53 °C to 64 °C in K<sup>+</sup> buffer (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> B and <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The melting curves analyzed with a two-state model gave Δ<i>G</i> of quadruplex formation between ∼−3.7 and −5.6 kcal/mol (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. G4 Motifs in s-80, the First 80 nt in the <i>KRAS</i> 5′-UTR mRNA</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">position sequence (5′→3′)</th><th class="colsep0 rowsep0" align="center">G/C<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">G-score</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>M</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">G4 RNA Δ<i>G</i><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> (kcal/mol)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-1</td><td class="colsep0 rowsep0" align="left">16–30</td><td class="colsep0 rowsep0" align="left">GC<u class="uu">GG</u>C<u class="uu">GG</u>C<u class="uu">GG</u>A<u class="uu">GG</u>CA</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">–4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-z</td><td class="colsep0 rowsep0" align="left">36–55</td><td class="colsep0 rowsep0" align="left"><u class="uu">GG</u>CGGC<u class="uu">GG</u>CAGU<u class="uu">GG</u>CGGC<u class="uu">GG</u></td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">–3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-c</td><td class="colsep0 rowsep0" align="left">59–71</td><td class="colsep0 rowsep0" align="left">A<u class="uu">GG</u>U<u class="uu">GG</u>C<u class="uu">GG</u>C<u class="uu">GG</u>C</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">–5.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">G/C ratio of the G4 motifs (>1.5 favors G4 over loop-hairpin).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last"><i>T</i><sub>M</sub> obtained in 50 mM Na-cacodylate, pH 7.4, 100 mM KCl.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Δ<i>G</i>s (298 K) obtained from UV-melting curves, using a two-state model.</p></div></div></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Primary structure of the 5′-UTR of the human <i>KRAS</i> transcript. The sequence is very rich in guanines (77% GC) and contains 33 GG-runs. (B) Predicted secondary structure of the first 80 nt of 5′-UTR (s-80) given by Mfold.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In s-80 there are three putative nonoverlapping G4 motifs <i>utr</i>-1, <i>utr</i>-z, and <i>utr</i>-c, indicated with colored arrows on the stem-loop structure. According to QGRS, the G4 motifs have a G-score = 21. (C) Cartoon showing the three nonoverlapping G4 RNAs and the structure of a G-tetrad.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Circular dichroism spectra at 25 °C and 90 °C of 5 μM <i>NRAS</i>, <i>utr-</i>1, <i>utr</i>-c, and <i>utr</i>-z RG4s in 50 mM Tris-HCl, pH 7.4, 100 mM KCl (or 100 mM LiCl); for <i>NRAS</i>, the KCl or LiCl concentration was 20 mM. The spectra were obtained in 0.5 cm cuvettes. (B) UV-melting curves of 5 μM <i>utr-</i>1, <i>utr</i>-z, and <i>utr</i>-c in 50 mM Na-cacodylate, pH 7.4, 100 mM KCl or LiCl. The curves were obtained by measuring the absorbance at 296 nm. The absorbance was normalized with the value at 20 °C. (C) EMSA in 50 mM Tris-HCl, pH 7.4, 100 mM KCl of mixtures containing <i>NRAS</i> RG4 and BG4 (lanes 2 and 3 from left) or <i>utr</i>-z RG4 and BG4 (lanes 5 and 6 from left). BG4 was used at 1 and 2 μg, and the gel was 5% polyacrylamide in TBE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To confirm that the selected G4 motifs adopt a RG4 structure, we analyze by EMSA if they are recognized by BG4, an antibody specific for G-quadruplexes.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52, 53)</a> This analysis was performed with only the 20-mer <i>utr-z</i> G4 motif, as <i>utr-1</i> and <i>utr-c</i> are too short for binding to BG4. We found that BG4 formed a stable RNA–protein complex with <i>utr-z</i> as well as with the <i>NRAS</i> RG4, which was used as a positive control (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> C).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Next, we analyzed by EMSA the first 80-nt of 5′-UTR (s-80), which contains the three nonoverlapping G4 motifs. We wanted to find out if the G4 structures are also present in this longer RNA s-80 sequence, although it could form a mismatched stem-loop structure, according to Mfold (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> A). We found that in K<sup>+</sup>-buffer, BG4 clearly bound to s-80, while it did not essentially bind to single-stranded RNA or hairpin RNA sequences, thus suggesting that <i>KRAS</i> 5′-UTR forms indeed local G4 structures (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> B). We found that BG4 bound to s-80, although with a weaker intensity, also in Li<sup>+</sup>-buffer. This is probably due to the fact that s-80 exists as a hairpin = G4 equilibrium, which is more shifted to the right in K<sup>+</sup>- than Li<sup>+</sup>-buffer. BG4, upon binding to the existing quadruplex, is likely to push the equilibrium to the right. The capacity of RNA sequences to fold into G4s in the absence and presence of various metal ions, including K<sup>+</sup>, Na<sup>+</sup>, and Li<sup>+</sup>, has been examined by Miserachs et al.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Further evidence that G4 is present in s-80 was obtained by measuring the reactivity of the guanines with RNase T1,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> taking into account that the guanines involved in the formation of Watson–Crick or Hoogsteen hydrogen bonds do not react with RNase T1. If s-80 assumes the stem-loop structure of <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> A, the loop guanines G23-G24 and G26-G27 should be reactive. On the other hand, if s-80 forms the three nonoverlapping G4 structures, the loop guanines should not be reactive, as they should be involved in the formation of the G-tetrads. In contrast, guanines G30 and G33, falling between the <i>utr-1</i> and <i>utr-z</i> RG4 structures, should be reactive to RNase T1. The footprinting of s-80 over the loop region shows that there is a prevalence of stem-loop structure in Li<sup>+</sup>-buffer, as G30 and G33 are not or very slightly reactive (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> C, from left lanes 4 and 7). By contrast, in K<sup>+</sup>-buffer both G4 and stem-loop structures are present in solution, as G30 and G33 as well as G23-G24 and G26-G27 are clearly reactive with RNase T1 (lanes 3 and 6 from left). This behavior suggests that s-80 exists in solution in equilibrium between G4 and the stem-loop structure. To further support this conclusion, we tested the presence of RG4s in s-80 with thioflavin T (ThT), a fluorescence sensor specific for G4.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> As illustrated in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> A, ThT exhibited a strong increase of fluorescence emission upon binding to s-80 but not upon binding to a RNA hairpin (5′-GGCCGCCGCAGUGGCGGCGG). In <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> B we compared the fluorescence enhancement of ThT induced by s-80 or hairpin RNA, in K<sup>+</sup>- or Li<sup>+</sup>-buffer.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Stem-loop structure of the first 80 nt of <i>KRAS</i> 5′-UTR (s-80) given by Mfold. Sequence s-80 is in equilibrium with an alternative structure characterized by three nonoverlapping G4 RNAs. (B) EMSA in 5% polyacrylamide, TB 1X, of mixtures between BG4 and s-80 or RNA hairpin (hair) 5′-GGCCGCCGCAGUGGCGGCGG or ssRNA 5′-UGUAAACAUCCUACACUCAGCU in 50 mM Tris-HCl, pH 7.4, 100 mM KCl (K<sup>+</sup>-buffer) or 100 mM LiCl (Li<sup>+</sup>-buffer). Except mixture 4, all have been prepared in K<sup>+</sup>-buffer. Loading: s-80 (lane 1); s-80 + 1 μg BG4 (lane 2); s-80 + 2 μg BG4 (lane 3); s-80 + 1 μg BG4 in Li<sup>+</sup>-buffer (lane 4); hair (lane 5), hair +1 μg BG4 (lane 6); ssRNA (lane 7); ssRNA + 1 μg BG4 (lane 8). (C) RNase T1 footprinting of s-80. Loading from left to right: alkaline RNA fragmentation (Seq); RNase T1 reaction in urea (urea), in K<sup>+</sup>-buffer (K100), in Li<sup>+</sup>-buffer (Li100), in K<sup>+</sup>-buffer + <b>2a</b> (<i>r</i> = 4) to see if the hairpin = G4 equilibrium is shifted by the ligand (<b>2a</b>), in K<sup>+</sup>-buffer with 150 mM KCl (K150) in Li<sup>+</sup>-buffer with 150 mM (Li150). The bottom band (band 7) matches the mobility of a 7-mer fragment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Fluorescence emission spectra in 50 mM Tris-HCl pH 7.4, 100 mM KCl (K<sup>+</sup>-buffer), of 0.2 μM ThT in the presence of increasing amounts of 80-mer RNA fragment s-80 (top panel) or RNA hairpin 5′-GGCCGCCGCAGUGGCGGCGG (bottom panel). (B) Structure of thioflavin T (ThT) and increase of fluorescence quantum yield at 485 nm of ThT following addition of increasing amounts of s-80 or hairpin in K<sup>+</sup>-buffer and Li<sup>+</sup>-buffer (50 mM Tris-HCl pH 7.4, 100 mM LiCl).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">K<sup>+</sup>-buffer s-80 caused an increase of fluorescence (<i>F</i>/<i>F</i><sub>0</sub>) up to 25 while in Li<sup>+</sup>-buffer only up to 10, in agreement with the fact that the hairpin = G4 equilibrium is more shifted to the right in the K<sup>+</sup>-buffer. By comparison, a designed hairpin RNA increases <i>F</i>/<i>F</i><sub>0</sub> up to 5. These data strongly suggest that under physiological conditions, RG4 structures are certainly present in the first portion of <i>KRAS</i> 5′-UTR. Recently, Weldon et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> performed RNA footprintings on wild-type Bcl-x-681 transcript and its 7-deaza-G analogue, which cannot form RG4. They found that RG4 is present in the transcript, despite a possible formation of competing stem-loop structures. The Bcl-x G4 motifs have G-scores similar or lower (15 ≤ G-score ≤ 21) than those found in the 5′-UTR of <i>KRAS</i> (G-score = 21).</div></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> ATPD and ATFD: Uptake and Capacity To Stabilize RG4</h3><div class="NLM_p">As the 5′-UTR of <i>KRAS</i> is conserved in mammals (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S3">Supporting Information S3</a>) and forms RG4s, we hypothesized that these unusual structures could be involved in the mechanism regulating translation of <i>KRAS</i>. Previous studies have showed that translation can be modulated by small molecules targeting to RG4s.<a onclick="showRef(event, 'ref41 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref41 ref58 ref59">(41, 58, 59)</a> We therefore searched for molecules that obey the following criteria: (i) high capacity to penetrate cell membranes; (ii) binding to <i>KRAS</i> mRNA despite its typical cellular low-abundance; (iii) high affinity for RG4s. In previous studies we reported that anthrathiophenediones (ATPD) with alkyl side chains carrying either guanidine or chloroacetamidine terminal groups penetrate bladder cancer cells and bind to DNA and RNA G4s.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a> Because the side chains strongly impact the uptake of these molecules,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> we tested in human pancreatic Panc-1 cancer cells several ATPD analogues and focused on 4,11-bis[2-guanidinoethylamino)anthra[2,3-<i>b</i>]thiophene-5,10-dione (<b>1b</b>) and 4,11-bis(2-aminoethylamino)anthra[2,3-<i>b</i>]thiophene-5,10-dione (<b>2b</b>)<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> as well as on their furan analogues, anthrafurandiones (ATFD) <b>1a</b> and <b>2a</b><a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> A). Both types of ligands have alkyl side chains ending either with guanidine or amine groups. As the chromophore of these molecules emits fluorescence upon excitation at 488 nm, we investigated their cellular uptake by FACS (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> B). The results showed that compounds <b>2a</b> and <b>2b</b> with aminoethyl side chains are taken up 20- and 4-fold more efficiently than the corresponding guanidine analogues <b>1a</b> and <b>1b</b>, respectively. An explanation can be that compounds carrying a localized charge generally display lower membrane permeability than neutral compounds. Compound <b>2a</b> differs from <b>2b</b> only for an atom in the five-member ring: oxygen against sulfur. Nonetheless, the former is 5-fold more permeable to Panc-1 cells than the latter. The higher polarizability of sulfur compared to oxygen provides a rationale for this behavior.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Due to their high uptake in Panc-1 cells, we used <b>2a</b> and <b>2b</b> to design a strategy aiming at inhibiting translation of <i>KRAS</i> in Panc-1 cells. The guanidine analogues <b>1a</b> and <b>1b</b> were used as reference compounds: because their uptake is lower, they are expected to produce a weaker cellular effect. We first asked if the molecules bind to and stabilize the RG4 structures. UV-melting experiments showed that <b>2a</b> and, to a lesser extent, <b>2b</b> strongly stabilize the three nonoverlapping RG4s located in the <i>KRAS</i> 5′-UTR. The increase of <i>T</i><sub>M</sub> (Δ<i>T</i><sub>M</sub>) of the three RG4s in the presence of the G4 ligands at <i>r</i> = 1, 2, and 4 (<i>r</i> = [ligand]/[G4]) are reported in the histograms of <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> C. <b>2a</b> caused a <i>T</i><sub>M</sub> increase up to 32 °C, while <b>2b</b> up to 20 °C. The lower Δ<i>T</i><sub>M</sub> brought about by <b>2b</b> is probably due to the polarizability of sulfur that may reduce the stacking of the chromophore upon the G-tetrads. Our data show that when thiophene is replaced with furan, the ligand increases not only its capacity to stabilize RG4 but also the uptake in Panc-1 cells. We also investigated the quadruplex-to-duplex specificity of the G4 ligands by competition experiments. For instance, the <i>T</i><sub>M</sub> of <i>utr-1</i> RG4 in the presence of <b>2a</b> at <i>r</i> = 4 did not change when a 5- or 10-fold excess of 21-mer RNA duplex was added to the mixtures (not shown). This is in keeping with our previous data on analogue ligands.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Structures of ATFD <b>2a</b>, <b>1a</b> and ATPD <b>2b</b>, <b>1b</b>. Two amine- or guanidine-alkyl side chains have been attached to the ring system of the molecules. (B) Flow cytometry data showing the uptake of 3 μM compounds by Panc-1 cells after an incubation of 4 h. The two alkylamine compounds <b>2a</b> and <b>2b</b> are taken up more than the guanidine analogues. (C) Δ<i>T</i><sub>M</sub> is the <i>T</i><sub>M</sub> increase of <i>utr-</i>z, <i>utr-</i>1, and <i>utr-</i>c G4 RNAs caused by <b>2a</b> and <b>2b</b> at <i>r</i> = 1, 2, and 4, in 50 mM Tris-HCl, pH 7.4, 100 mM KCl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, the affinity of ligands <b>2a</b> and <b>2b</b> for various G4 RNAs located in 5′-UTR was determined by fluorescence titrations. A typical titration obtained with <i>utr</i>-1 and <b>2a</b> is shown in <a class="ref internalNav" href="#notes1" aria-label="Supporting Information S4">Supporting Information S4</a>. The various <i>K</i><sub>D</sub>s are reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. The average <i>K</i><sub>D</sub> value for <b>2a</b> is ∼140 nM and for <b>2b</b> is ∼156 nM. The <i>K</i><sub>D</sub> for the binding of the ligands to competing mismatched hairpin RNAs could not be measured, as we were unable to find experimental conditions in which these putative structures, predicted by Mfold, are stable. We also designed a non-natural RNA stem-loop structure and found that <b>2a</b> and <b>2b</b> have affinities for this hairpin from 2 to 6 times lower than that for the RG4s. Interestingly, the <i>K</i><sub>D</sub>s for the binding of the ligands to the critical G4 DNA formed in the <i>KRAS</i> promoter are 626 ± 71 nM for <b>2a</b> and 278 ± 21 nM for <b>2b</b>. Although the ligands can bind also to G4 DNA, the fact that they accumulate more in the cytoplasm than in the nucleus (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S5">Supporting Information S5</a>) and that there are many copies of mRNA RG4s, suggest that the main target, i.e., those producing a stronger cellular effect, is likely to be in mRNA.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>K</i><sub>D</sub> Values Relative to the Interaction between <i>KRAS</i> G4 RNAs and Ligands <b>2a</b> and <b>2b</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">sequence (5′→3′)</th><th class="colsep0 rowsep0" align="center"><b>2a</b> <i>K</i><sub>D</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><b>2b</b> <i>K</i><sub>D</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-1</td><td class="colsep0 rowsep0" align="left">GC<u class="uu">GG</u>C<u class="uu">GG</u>C<u class="uu">GG</u>A<u class="uu">GG</u>CA</td><td class="colsep0 rowsep0" align="left">75 ± 6</td><td class="colsep0 rowsep0" align="left">91 ± 15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-z</td><td class="colsep0 rowsep0" align="left"><u class="uu">GG</u>CGGC<u class="uu">GG</u>CAGU<u class="uu">GG</u>CGGC<u class="uu">GG</u></td><td class="colsep0 rowsep0" align="left">251 ± 151</td><td class="colsep0 rowsep0" align="left">294 ± 138</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>utr</i>-4<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">CAGCAGC<u class="uu">GG</u>C<u class="uu">GG</u>C<u class="uu">GG</u>CAGU<u class="uu">GG</u></td><td class="colsep0 rowsep0" align="left">96 ± 45</td><td class="colsep0 rowsep0" align="left">83 ± 27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WC hairpin<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">CCGCCGCAGUGGCGGCGG</td><td class="colsep0 rowsep0" align="left">514 ± 103</td><td class="colsep0 rowsep0" align="left">482 ± 109</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Obtained from fluorescence titrations in 50 mM Tris-HCl, pH 7.4, 100 mM KCl.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">G4 motif present in the <i>KRAS</i> 5′UTR that overlaps other G4 motifs;</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">WC RNA hairpin with 7-C:G stem and 4-nt loop (AGUG) .</p></div></div></div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> ATFDs Target the <i>KRAS</i> Transcript under Cellular Low-Abundance Conditions</h3><div class="NLM_p">Studies so far reported on the use of small molecules to inhibit translation are based on luciferase assays and on the assumption that the G4 ligands bind to mRNA.<a onclick="showRef(event, 'ref26 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref26 ref43 ref44 ref45">(26, 43-45)</a> The first issue that we addressed in our study was to demonstrate that <b>2a</b> (the compound showing the highest uptake) does indeed bind to <i>KRAS</i> mRNA, under its low-abundance cellular condition. To do this, we synthesized biotinylated <b>2a</b> (b-<b>2a</b>) and set up a biotin–streptavidin pull-down assay.</div><div id="sec2-3-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Synthesis of Biotinylated <b>2a</b> (b-<b>2a</b>)</h4><div class="NLM_p">The synthesis of compounds <b>1a</b>, <b>1b</b>, <b>2a</b>, and <b>2b</b>, based on the nucleophilic substitution of alkoxy groups in the peri-positions of heteroarene-fused anthracenediones, has been described previously.<a onclick="showRef(event, 'ref62 ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref62 ref63 ref64 ref65">(62-65)</a> To link biotin to <b>2a</b> we used as spacer 1,6-diaminohexane.<a onclick="showRef(event, 'ref66 ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref66 ref67 ref68">(66-68)</a> First, biotin (<b>1</b>) was coupled with mono-Boc-protected 1,6-diaminohexane using HATU as coupling reagent. The subsequent cleavage of the protecting group led to building block HDA–biotin (<b>3</b>) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> A).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> As starting compound for the biotinylation of <b>2a</b>, we used 4,11-dimethoxy-5,10-dioxoanthra[2,3-<i>b</i>]furan-2-carboxylic acid (<b>4</b>).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The condensation of the amine group of <b>3</b> with the carboxylic group of <b>4</b> in the presence of PyBOP yielded the biotinylated 4,11-dimethoxy intermediate <b>5</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> B). Finally, by treating 4,11-dimethoxyanthra[2,3-<i>b</i>]furan-2-carboxamide (<b>5)</b> with ethylenediamine in THF at 50 °C we obtained to biotinylated 4,11-bis(aminoalkylamino)anthra[2,3-<i>b</i>]furan-5,10-dione <b>6</b> (b-<b>2a</b>).</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Biotinylated Anthrafurandiones b-<b>2a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BocNH(CH<sub>2</sub>)<sub>6</sub>NH<sub>2</sub>, HATU, NMM, DMF, rt; overnight; yield 89%; (b) DCM, TFA, rt; 3 h; yield 87%; (c) <b>3</b>, PyBOP, DIPEA, DMSO, rt, 1 h, yield 80%; (d) ethylendiamine, THF, 50 °C, 1.5–2 h, yield 68%.</p></p></figure></div><div id="sec2-3-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Streptavidin–Biotin Pull-Down Assays with b-<b>2a</b></h4><div class="NLM_p">To demonstrate that the ligands bind to RG4s in <i>KRAS</i> mRNA, we first ask if, by means of a biotin–streptavidin assay, b-<b>2a</b> is capable to pull down a DNA containing a G4 structure (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> A). We designed two DNA strands of 115 and 89 nt of which the former contained in the middle a G4 motif (telomeric <i>htel</i><a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a>). Both DNA strands had the same 5′- and 3′-ends and could be amplified with the same couple of primers. A 1:1 mixture of the two strands was incubated with an excess b-<b>2a</b>, and the DNA bound to the biotinylated ligand was pulled down by streptavidin–magnetic beads and the recovered DNA amplified. On a calibration curve, the DNA recovered was dramatically enriched with the 115-nt sequence containing G4, suggesting that b-<b>2a</b> efficiently pulled down the DNA strand carrying G4: a prerequisite essential for the next step of the experiment (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S6">Supporting Information S6</a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Structure of b-<b>2a</b> conjugated with biotin. (B) Total cellular mRNAs contain <i>KRAS</i> mRNA with G4 RNA in the 5′-UTR. G4-RNA is bound by b-<b>2a</b>. <i>HPRT</i> is not bound or little bound by b-<b>2a</b>. (C) The streptavidin–biotin pull-down assay: b-<b>2a</b> pulls down preferentially G4-motif mRNAs. The recovered mRNA was used to determine by RT-qPCR the amounts of <i>KRAS</i> and <i>HPRT</i> mRNAs. (D) The histograms show the ratio of <i>KRAS</i>/<i>HPRT</i> mRNAs in the input, i.e., total cellular extract (fixed to 1), and in the recovered RNA from a total cellular extract treated with increasing amounts of b-<b>2a</b>. An enrichment of <i>KRAS</i> over <i>HPRT</i> of nearly 3 times was obtained, suggesting that the biotinylated ligands bind to <i>KRAS</i> mRNA within the total cellular extract.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The pull-down experiment was repeated by replacing the 1:1 mixture with total cellular RNA extracted from Panc-1 cells. Cellular RNA is composed by ribosomal, transfer, and, in minimal part, mRNA. As illustrated in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> B, the transcriptome contains mRNAs without G4 structures (such as <i>HPRT</i>) and with G4 structures in 5′-UTR as <i>KRAS</i>. We reasoned that while all transcripts will have weak binding sites for b-<b>2a</b> (RNA stem-loop secondary structures), only a fraction of transcripts will exhibit both weak (stem-loop) and strong (G4) binding sites for b-<b>2a</b>. On the basis of this assumption, total cellular RNA from Panc-1 cells was incubated with b-<b>2a</b> and the RNA bound to the ligand was pulled down with streptavidin-coated magnetic beads. The amounts of <i>KRAS</i> and <i>HPRT</i> transcripts in the recovered RNA were determined by quantitative RT-PCR and compared to the amounts of the same genes detected in the input (untreated cellular RNA). The results showed that <i>KRAS</i>/<i>HPRT</i> in the pulled-down RNA increased nearly 3-fold compared to the input, suggesting that b-<b>2a</b> binds indeed to RG4s in <i>KRAS</i> transcripts (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> C,D). The pull-down experiment was repeated with increasing concentrations of b-<b>2a</b> (from 400 to 800 nΜ). The highest enrichment in <i>KRAS</i> transcript was obtained at a concentration of <b>2a</b> of 600 nM. With higher concentrations of b-<b>2a</b>, the enrichment decreased as the ligand bound to both strong (G4) and weak (stem-loop) sites (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> B). Together, the data demonstrate that a small molecule such as <b>2a</b>, designed to inhibit gene expression, targets G4 RNA in the 5′-UTR of the <i>KRAS</i> transcript under cellular conditions in which the transcript is typically in low abundance.</div></div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Capacity of Compounds <b>2a</b> and <b>2b</b> To Repress Translation in Pancreatic Cancer Cells</h3><div class="NLM_p">After having demonstrated that ATPD and ATFD bind to RG4s in the <i>KRAS</i> transcript, we asked if they inhibit the translation of the oncogene in pancreatic cancer cells. First, we performed a dual-luciferase assay as a proof-of-principle. We used a plasmid where <i>Renilla</i> luciferase was driven by the <i>KRAS</i> promoter including 5′-UTR (pRL-KRAS). To determine the transfection efficiency, we used pHRAS-mutA-luc, in which <i>Firefly</i> luciferase is driven by a mutated <i>HRAS</i> promoter, which does not bear G4 motifs (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> A).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Panc-1 cells were first treated for 6 h with increasing amounts of <b>2a</b> and <b>2b</b>, with the guanidine analogues <b>1a</b> and <b>1b</b> (from 0.3 to 1.6 μM), and then with the luciferase vectors. The ratio between <i>Renilla</i> and <i>Firefly</i> luciferases was measured 48 h after transfection. <b>2a</b> and <b>2b</b> at concentrations between 0.3 and 1.6 μM dramatically lower luciferase, much more than that of the reference guanidine analogues (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> B). When <b>2a</b> was used at lower concentrations, between 0.07 and 0.33 μM, a clear dose–response inhibitory effect was observed (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> C). If one considers that the four ligands have similar affinity for <i>utr</i>-z (and also for the other RG4s) but different uptake in Panc-1 cells, there is a clear correlation between luciferase inhibition and uptake. The higher the uptake (<b>2a</b> and <b>2b</b>), the higher the luciferase inhibition.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A, B) Dual luciferase assay showing the effect of compounds <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> (0–1.6 μM) on <i>Renilla</i> luciferase driven by the <i>KRAS</i> promoter comprising the 5′-UTR (pRL-KRAS). As reference, we used a plasmid in which <i>Firefly</i> luciferase was driven by the <i>HRAS</i> promoter mutated to abolish its capacity to form quadruplex structures.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The assay shows that <b>2a</b> strongly inhibits luciferase expression. Also <b>2b</b> shows a strong inhibitory activity. (C) When Panc-1 cells are treated with <b>2a</b> at lower concentrations (0–0.33 μM), a dose–response reduction of luciferase is observed. T = <i>Renilla</i>/<i>Firefly</i> in treated cells, C = <i>Renilla</i>/<i>Firefly</i> in untreated cells. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It might be argued that the luciferase repression mediated by <b>2a</b> and <b>2b</b> is due to the binding of the ligands not only to the RG4s in 5′-UTR but also to G4 in the <i>KRAS</i> promoter.<a onclick="showRef(event, 'ref23 ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref23 ref72 ref73">(23, 72, 73)</a> To address this point, we prepared a vector where <i>Renilla</i> luciferase is driven by the cytomegalovirus (CMV) promoter. At the 3′-end of CMV promoter, we inserted the <i>KRAS</i> 5′-UTR (pRL-CMV-UTR). As the CMV promoter does not form G4 structures, the strong binding sites for <b>2a</b> are only located in the 5′-UTR sequence. As a reference vector, we used pHRAS-mutA-luc. The dual luciferase assay showed that <b>2a</b> represses <i>Renilla</i> luciferase expression in a dose–response manner, suggesting that the presence of <i>KRAS</i> 5′-UTR alone is sufficient to promote the repression of luciferase (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>, left panel). The inhibition is observed with higher ligand concentrations than those used with plasmid pRL-KRAS, because the CMV promoter is stronger than the <i>KRAS</i> promoter. We then measured the effect of <b>2a</b> on pRL-CMV, which lacks the <i>KRAS</i> 5′-UTR downstream of the CMV promoter. In this case <b>2a</b> did not repress <i>Renilla</i> luciferase, in agreement with the fact that G4 is not present in this construct (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>, right panel). This experiment supports the conclusion that <b>2a</b> upon binding to RG4 in the 5′-UTR of <i>KRAS</i> inhibits translation. We cannot rule out that the compounds also bind to the <i>KRAS</i> promoter. However, considering that (i) the G4 ligands accumulate more in the cytoplasm than in the nucleus (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S5">Supporting Information S5</a>) and that (ii) there are many copies of RNA targets in the cells, even under mRNA low-abundance cellular conditions, <b>2a</b> and <b>2b</b> are expected to bind more favorably to G4 in mRNA than in promoter DNA.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Dual luciferase assay with <i>Renilla</i> luciferase plasmid pRL-CMV-UTR or pRL-CMV and <i>Firefly</i> luciferase plasmid pHRAS-mutA-luc. Left histograms show a strong dose–response luciferase reduction caused by increasing amounts of <b>2a</b> (0–1.6 μM), while the right panel shows that when the <i>KRAS</i> 5′-UTR is removed, the luciferase reduction is not observed, suggesting that the inhibitory effect is mediated by 5′-UTR. T = <i>Renilla</i>/<i>Firefly</i> in treated cells, C = <i>Renilla</i>/<i>Firefly</i> in untreated cells. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Effect of <b>2a</b> and <b>2b</b> on <i>KRAS</i> Expression in Pancreatic Cancer Cells</h3><div class="NLM_p">As the designed compounds, in particular <b>2a</b>, decrease the luciferase from vectors containing the <i>KRAS</i> 5′-UTR, we asked if these G4 ligands are also able to inhibit the expression of genomic <i>KRAS</i> in pancreatic cancer cells. We measured by quantitative RT-PCR the level of <i>KRAS</i> transcript in Panc-1 cells after 6 and 24 h of treatment. The results showed that while <b>2b</b>, <b>1a</b>, and <b>1b</b> did not lower the level of mRNA, <b>2a</b> reduced it to about 50% of the control, at both time points. By contrast, all the compounds displayed a strong capacity to suppress the KRAS protein. In particular <b>2a</b> and <b>2b</b>, which are efficiently taken up by Panc-1 cells, brought the protein down to <10% of the control (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a> A,B). In light of these results, we concluded that <b>2a</b> and <b>2b</b> inhibit <i>KRAS</i> mainly at the translational level. Moreover, both luciferase and Western blot data show that compound <b>2a</b> is slightly more active than <b>2b</b>, possibly because <b>2a</b> has a higher capacity to penetrate the cell membrane than <b>2b</b>, and a fraction of it may also target G4 in the <i>KRAS</i> promoter.</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Quantitative RT-PCR of <i>KRAS</i> mRNA in Panc-1 cells treated for 6 and 24 h with 1.6 μM <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b>. Ordinate reports the level of KRAS mRNA normalized to β2-microglobulin and <i>HPRT</i>. (B) Western blot determination of <i>KRAS</i> protein and β-actin in Panc-1 cells treated with 1.6 μM <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> for 48 h. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pancreatic cancer cells, being addicted to <i>KRAS</i>, respond to the repression of the oncogene by activating apoptosis.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> We found indeed that <b>2a</b> and to a lesser extent <b>2b</b> caused a strong activation of caspases 3/7 (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> A). In contrast, the reference compounds <b>1a</b> and <b>1b</b> showed a much weaker caspase activation. To confirm the activation of apoptosis, we also performed an annexin/propidium iodide assay (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> B). This is based on the observation that an early event occurring in apoptosis is the translocation of phosphatidylserine from the inner to the outer leaflet of the plasma membrane, thus exposing the phospholipid to the external cell environment. Annexin V, a phosphatidylserine recognizing protein labeled with FITC, can be used to detect this event by FACS. Early and late apoptosis can be distinguished by using both annexin V and propidium iodide (PI).<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> The experiment showed that 1.6 μM <b>2a</b> and <b>2b</b> increased the population of apoptotic cells 72 h after treatment: untreated Panc-1 cells (apoptotic cells ∼1.2%); Panc-1 cells treated with <b>2a</b> (apoptotic cells ∼32.5%); Panc-1 cells treated with <b>2b</b> (apoptotic cells ∼20.6%).</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Activation of caspase 3/7 in Panc-1 cells treated with <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> (0.8 and 1.6 μM). Compounds <b>2a</b> and <b>2b</b> show activity higher than that of analogues <b>1a</b> and <b>1b</b>. (B) Annexin–propidium assay of Panc-1 cells treated for 72 h with <b>2a</b> (1.6 μM) and <b>2b</b> (1.6 μM). (C) Metabolic activity in Panc-1 cells treated with increasing concentrations of <b>2a</b>, <b>2b</b>, <b>1a</b>. and <b>1b</b>. Compound <b>2a</b> shows the highest activity. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Finally, the metabolic activity of Panc-1 cells treated with the designed compounds was evaluated by a resazurin assay. Compound <b>2a</b> causes an inhibition of the metabolic activity (IC<sub>50</sub> = 0.26 μM) stronger than <b>2b</b> (IC<sub>50</sub> = 0.9 μM) (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> C). Moreover, <b>2a</b> caused a strong inhibition (∼80%) of colony formation in two pancreatic cancer cells: Panc-1 and BxPC-3 (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S7">Supporting Information S7</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20433" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20433" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">By setting up a streptavidin–biotin pull-down assay, we have demonstrated that <b>2a</b>, an anthrafurandione with aminoethylamino side chains, which efficiently internalizes and accumulates in the cytoplasm of Panc-1 cancer cells, targets <i>KRAS</i> mRNA under low-abundance cellular conditions. Luciferase assays with specific vectors showed that <b>2a</b> and its anthrathiophenedione analogue <b>2b</b> repress translation in a dose–response manner, suggesting that they have a great potential in cancer therapy. Indeed, Western blots showed that these molecules strongly decrease the <i>KRAS</i> protein in Panc-1 cancer cells. Moreover, the compounds activate apoptosis, as indicated by the caspase 3/7 and annexin/propidium iodide assays, and reduce the metabolic activity as well as the colony formation of the cells.</div><div class="NLM_p">The mechanism of action of the designed compounds <b>2a</b> and <b>2b</b> is based on their capacity to bind to G4 structures located in the 5′-UTR of <i>KRAS</i> mRNA. The presence of these folded structures in mRNA has been demonstrated by immunostaining in fixed cells.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> A recent study by Guo and Bartel casts doubts on the existence of RG4s under in vivo conditions, as the authors found that RG4s are globally unfolded by single-stranded binding proteins.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> If this is also true for the <i>KRAS</i> transcripts in pancreatic cancer cells, compounds <b>2a</b> and <b>2b</b> could inhibit translation by competing with the binding of single stranded-binding proteins to the RNA G4-motifs.</div><div class="NLM_p last">Finally, having established in cancer cells that <b>2a</b> and <b>2b</b> have a potential as anticancer agents, the next step will be in vivo testing. As pancreatic cancer cells are addicted to <i>KRAS</i>, therapeutics targeting this oncogene should be much more injurious for the malignant cells than for normal cells.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15305" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15305" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Oligonucleotides</h3><div class="NLM_p last">The oligonucleotides used in this study have been purchased from Microsynth (Switzerland). Oligonucleotide solutions in DEPC-treated Milli-Q water have been conserved at −80 °C. The sequences are reported in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a class="ref internalNav" href="#notes1" aria-label="Supporting Information S8">Supporting Information S8</a>.</div></div><div id="sec4-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis of <b>2a</b>, <b>2b</b>, <b>1a</b>, <b>1b</b>, and Biotinylated Ligand b-<b>2a</b></h3><div id="sec4-2-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> General information</h4><div class="NLM_p last">NMR spectra were recorded on a Varian VXR-400 instrument operated at 400 MHz (<sup>1</sup>H NMR) and 100 MHz (<sup>13</sup>C NMR). Chemical shifts were measured in DMSO-<i>d</i><sub>6</sub> using tetramethylsilane as an internal standard. Analytical TLC was performed using silica gel F<sub>254</sub> plates (Merck) and column chromatography with SilicaGel Merck 60. Melting points were determined using a Buchi SMP-20 apparatus and are uncorrected. High-resolution mass spectra were recorded with electron-spray ionization using a Bruker Daltonics microOTOF-QII instrument. UV spectra were recorded on a Hitachi-U2000 spectrophotometer. HPLC was performed using a Shimadzu Class-VP V6.12SP1 system. All solutions were dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated at reduced pressure using a Buchi-R200 rotary evaporator at a temperature below 45 °C. All products were vacuum-dried at room temperature. All solvents, chemicals, and reagents were obtained commercially and used without purification. The ligands <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> have been synthesized as previously described.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60, 61)</a> The purity of final conjugates b-<b>2a</b> was >95% as determined by HPLC analysis (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information S9">Supporting Information S9</a>).</div></div><div id="sec4-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Synthetic Procedures</h4><div id="sec4-3-1" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> <i>tert</i>-Butyl (6-(5-((3a<i>S</i>,4<i>S</i>,6a<i>R</i>)-2-Oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl)pentanamido)hexyl) carbamate (<b>2</b>, <i>N</i>-Boc-HDA–Biotin)</h5><div class="NLM_p last"><i>N</i>-Boc-1,6-Diaminohexane (0.50 g, 2.05 mmol) was added to a stirred solution of biotin (<b>1</b>, 0.50 g, 2.05 mmol), <i>N</i>-methylmorpholine (NMM, 0.23 mL, 2.05 mmol) and 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU, 0.77 g, 2.05 mmol) in dry DMF (20 mL). The reaction mixture was stirred overnight at room temperature and diluted with water, and the product was extracted with ethyl acetate (2 × 25 mL). The extract was washed twice with water, dried, and evaporated. The residue was purified by using column chromatography with chloroform–methanol (1:0 → 3:1) as the eluting solvent. The solid precipitate was recrystallized from <i>n</i>-propanol to give 0.84 g (89%) of the white powder of <i>N</i>-Boc-HDA–biotin (<b>2</b>);<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> mp 173–176 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.73 (t, 1H, <i>J</i> = 5.4 Hz, NH); 6.77 (t, 1H, <i>J</i> = 5.0 Hz, NH); 6.43 (br s, 1H, NH-biotin); 6.36 (br s, 1H, NH-biotin); 4.31–4.28 (m, 1H, CH); 4.13–4.11 (m, 1H, CH); 3.11–3.06 (m, 1H, SC<i>H</i>CH<sub>2</sub>); 3.01 (dd, 2H, <sup>1</sup><i>J</i><i>=</i> 6.2, <sup>2</sup><i>J</i><i>=</i> 7.8 Hz, NCH<sub>2</sub>); 2.89 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 6.0, <sup>2</sup><i>J</i><i>=</i> 7.2 Hz, NCH<sub>2</sub>); 2.81 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 5.2, <sup>2</sup><i>J</i><i>=</i> 7.5 Hz, SCH<i>H</i>); 2.57 (d, 1H, <i>J</i> = 12.4 Hz, SC<i>H</i>H); 2.03 (t, 2H, <i>J</i> = 7.5 Hz, COC<i>H</i><sub>2</sub>CH<sub>2</sub>); 1.60–1.20 (m, 23H, C(CH<sub>3</sub>)<sub>3</sub>, 7CH<sub>2</sub>); HRMS (ESI) calculated for C<sub>21</sub>H<sub>39</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup> 443.2687, found 443.2673.</div></div><div id="sec4-3-2" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> <i>N</i>-(6-Aminohexyl)-5-((3a<i>S</i>,4<i>S</i>,6a<i>R</i>)-2-oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl)pentanamide Trifluoroacetate (HDA–Biotin, <b>3</b>)</h5><div class="NLM_p last">A solution of <i>N</i>-Boc-HDA–biotin (<b>2</b>, 0.80 g, 1.81 mmol) in a mixture of DCM (10.0 mL) and TFA (2.0 mL) was stirred for 3 h at the room temperature. The solvent was evaporated and residue reprecipitated from warm water with acetone. The precipitated crystals were filtered, washed with acetone, and dried to yield 0.72 g (87%) of HDA–biotin (<b>3</b>); mp 92–95 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.76 (t, 1H, <i>J</i> = 5.4 Hz, NH); 7.70 (br s, 3H, NH<sub>3</sub>); 6.42 (br s, 1H, NH-biotin); 6.37 (br s, 1H, NH-biotin); 4.32–4.29 (m, 1H, CH); 4.14–4.11 (m, 1H, CH); 3.11–3.06 (m, 1H, SC<i>H</i>CH<sub>2</sub>); 3.01 (dd, 2H, <sup>1</sup><i>J</i><i>=</i> 6.0, <sup>2</sup><i>J</i><i>=</i> 6.8 Hz, NCH<sub>2</sub>); 2.82 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 5.0, <sup>2</sup><i>J</i><i>=</i> 7.5 Hz, SCH<i>H</i>); 2.76 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 6.0, <sup>2</sup><i>J</i><i>=</i> 7.2 Hz, NCH<sub>2</sub>); 2.58 (d, 1H, <i>J</i> = 12.6 Hz, SC<i>H</i>H); 2.04 (t, 2H, <i>J</i> = 7.5 Hz, COC<i>H</i><sub>2</sub>CH<sub>2</sub>); 1.64–1.24 (m, 14H, 7CH<sub>2</sub>); HRMS (ESI) calculated for C<sub>16</sub>H<sub>31</sub>N<sub>4</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup> 343.2162, found 343.2156.</div></div><div id="sec4-3-3" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Biotinyl-<i>N</i>-(6-aminohexyl)-4,11-dimethoxy-5,10-dioxo-5,10-dihydroanthra[2,3-<i>b</i>]furan-2-carboxamide (<b>5</b>)</h5><div class="NLM_p last">A mixture of 4,11-dimethoxyanthra[2,3-<i>b</i>]furan-5,10-dione-2-carboxylic acid (<b>4</b>;<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> 0.30 g, 0.85 mmol), ethyldiisopropylamine (DIPEA, 0.5 mL, 3.00 mmol), biotinyl-<i>N</i>-(6-aminohexyl)amine trifluoroacetate (<b>3</b>, 0.39 g, 0.85 mmol), and benzotriazol-1-yloxytripyrrolidinophosphonium hexafluorophosphate (PyBOP, 0.52 g, 1.00 mmol) in DMSO (15 mL) was stirred at room temperature for 1 h. The reaction mixture was diluted with water, and the product was extracted with ethyl acetate (2 × 20 mL). The extract was washed twice with water, dried, and evaporated. The residue was purified by using column chromatography with chloroform–methanol (1:0 → 3:1) as the eluting solvent. The yield of the orange solid of <b>5</b> was 0.46 g (80%); mp 132–134 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.87 (t, 1H, <i>J</i> = 5.9 Hz, NH); 8.05–8.03 (m, 2H, 6,9-H); 7.88 (s, 1H, 3-H); 7.82–7.80 (m, 2H, 7,8-H); 7.76 (t, 1H, <i>J</i> = 5.3 Hz, NH); 6.44 (br s, 1H, NH-biotin); 6.37 (br s, 1H, NH-biotin); 4.31–4.27 (m, 1H, CH); 4.12 (s, 3H, OMe); 4.07 (s, 3H, OMe); 4.13–4.12–4.10 (m, 1H, CH); 3.28 (dd, 2H, <sup>1</sup><i>J</i><i>=</i> 6.2, <sup>2</sup><i>J</i><i>=</i> 6.8 Hz, NCH<sub>2</sub>); 3.10–3.06 (m, 1H, SC<i>H</i>CH<sub>2</sub>); 3.02 (dd, 2H, <sup>1</sup><i>J</i><i>=</i> 6.0, <sup>2</sup><i>J</i><i>=</i> 7.0 Hz, NCH<sub>2</sub>); 2.81 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 4.9, <sup>2</sup><i>J</i><i>=</i> 7.5 Hz, SCH<i>H</i>); 2.56 (d, 1H, <i>J</i> = 12.3 Hz, SC<i>H</i>H); 2.04 (t, 2H, <i>J</i> = 7.1 Hz, COC<i>H</i><sub>2</sub>CH<sub>2</sub>); 1.59–1.25 (m, 14H, 7CH<sub>2</sub>); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 182.05 (C═O); 181.65 (C═O); 171.87 (N–C═O); 162.75 (N–CO–N); 156.92 (N–C═O); 151.61 (C); 150.47 (2C); 142.60 (C); 133.99 (C); 133.80 (C); 133.64 (C); 127.30 (C); 123.33 (C); 120.43 (C); 133.98 (CH); 126.10 (CH); 126.00 (CH); 108.44 (CH); 61.81 (CH); 61.71 (CH); 61.07 (OCH<sub>3</sub>); 59.22 (OCH<sub>3</sub>); 55.47 (CH); 39.86 (CH<sub>2</sub>); 38.95 (CH<sub>2</sub>); 38.31 (CH2); 35.25 (CH<sub>2</sub>); 29.15 (CH<sub>2</sub>); 29.00 (CH<sub>2</sub>); 28.24 (CH<sub>2</sub>); 28.06 (CH<sub>2</sub>); 26.19 (CH<sub>2</sub>); 26.14 (CH<sub>2</sub>); 25.38 (CH<sub>2</sub>). HRMS (ESI) calculated for C<sub>35</sub>H<sub>41</sub>N<sub>4</sub>O<sub>8</sub>S<sup>+</sup> [M + H]<sup>+</sup> 677.2640, found 677.2612.</div></div><div id="sec4-3-4" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> 4,11-Bis((2-aminoethyl)amino)-biotinyl-<i>N</i>-(6-aminohexyl)-5,10-dioxo-5,10-dihydroanthra[2,3-<i>b</i>]furan-2-carboxamide (<b>6</b>, b-<b>2a</b>)</h5><div class="NLM_p last">A mixture of compound <b>5</b> (0.27 g, 0.40 mmol) and ethylenediamine (1.5 mL) in THF (5.0 mL) was heated at 50 °C for 2–3 h. During this time, the yellow color of the reaction mixture changed to dark blue, and after the complete conversion of <b>5</b> (as determined by TLC) the solution was cooled and quenched with water. An aqueous solution of HCl (1%) was added to make the pH = 8.0, the solution was saturated with NaCl, and the product was extracted with warm <i>n</i>-butanol (3 × 25 mL). The extract was washed twice with brine, dried, and evaporated. The residue was purified by column chromatography using chloroform–methanol–concentrated NH<sub>4</sub>OH (10:2:0 → 10:4:1) as the eluting solvent. The purified residue was dissolved in a warm aqueous solution of HCl (1 N) and reprecipitated with acetone. The precipitated crystals were filtered, washed with acetone, and dried to yield 0.22 g (68%) of the dark blue powder of dihydrochloride <b>6</b>; mp 204–206 °C (decomp); HPLC Kromasil-100-5-mkm C-18 column (4 × 250 mm, LW = 260 nm), eluent: A, H<sub>3</sub>PO<sub>4</sub> (0.01 M), B, MeCN; gradient B 20 → 60% (30 min), elution time 9.4 min, purity 96%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (t, 1H, <i>J</i> = 5.0 Hz, NH); 11.36 (t, 1H, <i>J</i> = 5.5 Hz, NH); 9.32 (t, 1H, <i>J</i> = 5.4 Hz, NH); 8.55 (s, 1H,3-H); 8.41 (br s, 3H, NH<sub>3</sub>); 8.25–8.22 (m, 2H, 6,9-H); 8.18 (br s, 3H, NH<sub>3</sub>); 7.82 (br s, 1H, NH); 7.80–7.78 (m, 2H, 7,8-H); 6.44 (br s, 1H, NH-biotin); 6.39 (br s, 1H, NH-biotin); 4.31–4.28 (m, 1H, CH); 4.13–4.10 (m, 3H, CH, NCH<sub>2</sub>); 4.08–4.06 (m, 2H, 2NCH<sub>2</sub>); 3.29–3.26 (m, 4H, 2NCH<sub>2</sub>); 3.10–3.06 (m, 1H, SC<i>H</i>CH<sub>2</sub>); 3.05–3.00 (m, 2H, NCH<sub>2</sub>); 2.81 (dd, 1H, <sup>1</sup><i>J</i><i>=</i> 5.1, <sup>2</sup><i>J</i><i>=</i> 7.3 Hz, SCH<i>H</i>); 2.57 (d, 1H, <i>J</i> = 12.4 Hz, SC<i>H</i>H); 2.05 (t, 2H, <i>J</i> = 7.3 Hz, COC<i>H</i><sub>2</sub>CH<sub>2</sub>); 1.60–1.30 (m, 14H, 7CH<sub>2</sub>). HRMS (ESI) calculated for C<sub>37</sub>H<sub>49</sub>N<sub>8</sub>O<sub>6</sub>S<sup>+</sup> [M + H]<sup>+</sup> 733.3490, found 733.3494.</div></div></div></div><div id="sec4-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cell Culture, Metabolic Activity, and Proliferation Assays</h3><div class="NLM_p">Panc-1 and BxPC-3 cells (human pancreatic cancer cells) were maintained in exponential growth in Dulbecco’s Modified Eagle’s Medium (DMEM) containing 100 U/mL penicillin, 100 mg/mL streptomycin, 20 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and 10% fetal bovine serum (Euroclone, Italy). The cell lines have been genotyped by Microsynth (Switzerland) to verify their identity. They matched 100% with the DNA-profiles of Panc-1 (ATCC CRL-1469TM) and BxPC-3 (ATCC CRL-1687TM). The metabolic activity assay was performed by seeding the cells (10 × 10<sup>3</sup> cells/well) in a 96-well plate. After 1 day, the cells have been treated with the compounds and after an incubation of 72 h the resazurin assay was performed following a standard procedure.</div><div class="NLM_p last">Colony forming assays have been carried out with Panc-1 and BxPC-3 cells plated on a 60 mm plate and treated with 0.25 and 0.5 μΜ <b>2a</b> or 1 and 1.5 μΜ <b>2b</b>. After 18 days, the cells were fixed and stained for 10 min with 2.5% methylene blue in 50% ethanol. Colonies with >50 cells were counted.</div></div><div id="sec4-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Circular Dichroism Spectra and UV-Melting Curves</h3><div class="NLM_p">CD spectra have been obtained on a JASCO J-600 spectropolarimeter, equipped with a thermostated cell holder, with 5 μM oligonucleotide solutions in 50 mM Na-cacodylate pH 7.4, 100 mM KCl or LiCl (RNase free). The CD spectra (for <i>KRAS utr-</i>1, <i>utr</i>-c, and <i>utr</i>-z) and <i>NRAS</i> were registered at 25 °C and 90 °C. Spectra were recorded in 0.5 cm quartz cuvette. The spectra were calculated with J-700 Standard Analysis software (Japan Spectroscopic Co, Ltd.). Each spectrum was recorded three times, smoothed, and the baseline subtracted.</div><div class="NLM_p last">UV melting curves were obtained by using JASCO V-750 UV–visible spectrophotometer equipped with a temperature control system that heats/cools the sample through Peltier technology (ETCS-761) (Jasco, Portland, OR). Melting curves were recorded at 260 and 296 nm in a 0.5 cm path length quartz cuvette, heating the sample from 20 °C to 100 °C. The samples were prepared at a final concentration of 5 μΜ in 100 mM KCl and 50 mM Na-cacodylate, pH 7.4. Incubation with increasing amounts of the molecules (<b>2a</b> and <b>2b</b>) (<i>r</i> = 1, 2, and 4) was performed for 1 h at room temperature.</div></div><div id="sec4-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> RNase T1 Footprinting and Electrophoretic Mobility-Shift Assay</h3><div class="NLM_p last">Single-stranded RNAs were purified by PAGE and 5′-end labeled with T4 polynucleotide kinase (ThermoFisher, Waltham, MA) and [<sup>32</sup>P]-ATP (PerkinElmer, Waltham, MA) for 1.5 h at 37 °C. RNase T1 footprinting was performed with 30 nM s-80 heated for 5 min at 85 °C and then incubated overnight at 25 °C in 10x Structure Buffer (RNase T1 Biochemistry grade, ThermoFisher) with 100 mM KCl or LiCl. The reactions were performed with 0.05 units of RNase T1 (ThermoFisher) for 10 min at 25 °C and stopped with 20 μL inactivation/precipitation buffer (ThermoFischer). RNA was precipitated at −80 °C for 2 h and centrifuged for 30 min at 13000 rpm. Precipitated RNA was resuspended with loading buffer (ThermoFisher), heated for 5 min at 95 °C, and electrophoresed on a 20% denaturing gel, pre-equilibrated at 55 °C in a Sequi-Gen GT Nucleic Acids Electrophoresis Apparatus (Bio-Rad, Hercules, CA), equipped with a thermocouple that allows precise temperature control. EMSA assays were performed with 20 nM <i>utr-z</i> or s-80 labeled at the 5′-end with [<sup>32</sup>P]-ATP. The mixture was incubated for 30 min at 37 °C with increasing concentrations of antibody BG4 (1 and 2 μg). BG4 was produced according to Studier et al.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The samples were run in a 5% TBE 1x gel for 2 h. After running, the gel was fixed in a solution containing 10% acetic acid and 10% methanol, dried at 80 °C, and exposed to Hyperfilm MP (GE Healthcare) for autoradiography.</div></div><div id="sec4-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Western Blots</h3><div class="NLM_p last">Total protein lysates (15 μg) were electrophoresed on 12% SDS-PAGE and transferred to a nitrocellulose membrane at 70 V for 2 h. The filter was blocked for 1 h with 5% BSA solution in PBS 0.05% Tween (Sigma-Aldrich, St. Louis, MO) at room temperature. The primary antibodies used are antiactin (clone JLA20, IgM mouse, 1 × 10<sup>–4</sup> μg/mL, Calbiochem, Merck Millipore, Germany) and anti-KRAS (IgG rabbit polyclonal antibody, diluted 1:250, ab 102007, Abcam, United Kingdom). Membranes with the samples were incubated overnight at 4 °C with primary antibodies. The filters were washed with 0.05% Tween in PBS and subsequently incubated for 1 h with the secondary antibodies, horseradish peroxidase conjugated: antimouse IgM (diluted 1:2000) and antirabbit IgG (diluted 1:5000) (Calbiochem, Merck Millipore, Germany). The signal of the proteins was developed with Super Signal West PICO and FEMTO (ThermoFisher) and detected with ChemiDOC XRS, Quantity One 4.6.5 software (BioRad Laboratories).</div></div><div id="sec4-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> RNA Extraction and Quantitative Real-Time PCR</h3><div class="NLM_p">Panc-1 cells were plated in a 96-well plate (18 × 10<sup>3</sup> cells/well). The following day (24 h), we treated the cells with <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b>, and total RNA was extracted by using iScript  RT-qPCR Sample Preparation Reagent (BioRad) 6 and 24 h after treatment, following the manufacturer’s instructions. For the cDNA synthesis, 1.25 μL of RNA was heated at 70 °C and placed on ice. The solution was added to 11.5 μL of a mix containing: 1x buffer, 0.01 M DTT (Invitrogen, Carlsbad, CA), 1.6 μM primer dT [MWG Biotech, Germany; d(T)<sub>16</sub>], 1.6 μM random hexamer primers (Mycrosynth, Switzerland), 0.4 mM dNTPs solution containing equimolar amounts of dATP, dCTP, dGTP, and dTTP (Euroclone, Italy), 0.8 U/μL RNase OUT, and 8 U/μL of M-MLV reverse transcriptase (Invitrogen). The mixtures were incubated for 1 h at 37 °C and stopped by heating at 95 °C for 5 min. As a negative control, the reverse transcription reaction was performed with a sample containing DEPC-treated water.</div><div class="NLM_p last">To determine the levels of <i>KRAS</i> and housekeeping genes hypoxanthine-guanine phosphoribosyltransferase (<i>HPRT</i>) and β2-microglobulin, quantitative real-time multiplex reactions were performed. We used 1x Kapa Probe fast qPCR kit (KAPA Biosystems, Wilmington, MA), 2.2 μL of cDNA, and primers/probes (sequences are reported in <a class="ref internalNav" href="#notes1" aria-label="Supporting Information S8">Supporting Information S8</a>). The PCR cycle was 3 min at 95 °C, 50 cycles 10 s at 95 °C, and 60 s at 58 °C. PCR reactions were carried out with a CFX-96 real-time PCR apparatus controlled by an Optical System software (version 3.1) (Bio-Rad Laboratories). The <i>KRAS</i> transcript level was normalized with the housekeeping genes.</div></div><div id="sec4-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Dual Luciferase Assays</h3><div class="NLM_p last">Panc-1 cell were plated (20 × 10<sup>3</sup>) and the following day treated with increasing concentrations of compounds <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> (0.3–1.6 μM). After 6 h of treatment, the cells were transfected with the plasmids. The 192 nt of <i>KRAS</i> 5′UTR (NM_033360) was cloned between Hind III and Nhe I of pRL-CMV plasmid (GenScript, Piscataway, NJ). Transfection was performed by mixing vectors (10 ng/well) p-light-SwitchGear KRAS (SwitchGear Genomics, Carlsbad, CA) (in the text pRL-KRAS) or pRL-CMV-UTR or pRL-CMV (<i>Renilla</i> luciferase) with 200 ng of control plasmid (pHRAS-mutA-luc) (<i>Firefly</i> luciferase) using jet-PEI (Polyplus, France) as transfectant agent. <i>Renilla</i> luciferase in cell lysates was measured and normalized by <i>Firefly</i> luciferase. Luciferase assays were performed 48 h after transfection with Dual-Glo Luciferase Assay System (Promega, Fitchburg, WI) following the supplier instructions. Samples were read on a Turner Luminometer and the relative luminescence expressed as (T/C × 100) where T = <i>Renilla</i> luciferase/<i>Firefly</i> luciferase in treated cells and C = <i>Renilla</i> luciferase/<i>Firefly</i> luciferase in untreated cells.</div></div><div id="sec4-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Uptake Analysis</h3><div class="NLM_p last">Panc-1 cells were plated in a 24-well plate at density of 5 × 10<sup>4</sup> cells/well. After 1 day, the cells were treated with the molecules: time and concentration are indicated in figure captions. The cells were trypsinized and pelleted. The pellets were resuspended in 200 μL of PBS and immediately analyzed by FACScan Flow Cytometer (Becton Dickinson, Franklin Lakes, NJ) equipped with a 488 nm argon laser. A minimum of 10<sup>4</sup> cells for each sample were acquired in list mode and analyzed using Cell Quest software. The cell population was analyzed by FSC light and SSC light. The signal was detected by FL3 (680 nm) channel in log scale.</div></div><div id="sec4-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Apoptosis Assays</h3><div class="NLM_p last">Caspase activity assay was performed with Apo-ONE Homogeneous Caspase-3/7 Assay (Promega,), according to the manufacturer’s protocol. Annexin V–propidium iodide assay was performed with Annexin V Apoptosis Detection Kit (Santa Cruz Biotechnology, Inc., Dallas, TX), following the manufacturer’s instructions. Flow cytometry measurements were performed with FACScan Flow Cytomer (Becton Dickinson).</div></div><div id="sec4-12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Biotin–Streptavidin Pull-Down Experiments</h3><div class="NLM_p">To construct the calibration plot for biotin–streptavidin pull-down assays, we prepared several mixtures of 115- and 89-nt sequences in 50 mM Tris-HCl, pH 7.4, 100 mM KCl, at a total concentration of 0.2 μM and 115-nt/89-nt ratio varying from 1 to 5000. The mixtures have been amplified with KAPA2G Robust HotStart PCR Kit (KAPA Biosystems) using the same couple of primers (sequences reported in <a class="ref internalNav" href="#notes1" aria-label="Supporting Information S8">Supporting Information S8</a>) (0.5 μM) and dNTPs (0.2 mM). The program used was 5 min at 95 °C, 15 s at 95 °C, 15 s at 55 °C, and 30 s at 72 °C, for 35 cycles. The products have been separated by 8% PAGE and the bands stained with ethidium bromide. Their intensities were measured with a ChemiDOC XRS, Quantity One 4.6.5 software (BioRad Laboratories). We reported in a plot the ratio of the intensities of the amplified bands (115-nt/89-nt) as a function of the logarithm of the ratio (<i>r</i>) of the two sequences in the mixtures. We obtained a straight line that correlated the band intensities with the mixture composition.</div><div class="NLM_p last">Total RNA was extracted from Panc-1 cells, and its concentration was measured by UV absorption. Eight micrograms of cellular RNA in 50 mM KCl, 50 mM Tris-HCl, pH 7.4, was incubated overnight with increasing concentrations of b-<b>2a</b> (from 0.4 to 0.8 μM). We incubated the magnetic beads, after saturation with ssDNA (vide infra), with cellular RNA treated with the biotinylated ligand for 20 min at 25 °C. The supernatant was removed, and the beads were washed twice with Tris buffer. We then recovered the bound RNA with a solution 0.8 M NaCl. The recovered RNA was retrotranscribed with 0.8 U/μL RNase OUT and 8 U/μL of M-MLV reverse transcriptase (Life Technologies, ThermoFisher) and amplified by quantitative real-time PCR (vide infra). We amplified <i>KRAS</i> and the housekeeping genes <i>HPRT</i> and β2-microglobulin (for primers, see <a class="ref internalNav" href="#notes2" aria-label="Supporting Information S8">Supporting Information S8</a>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00622">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00622" class="ext-link">10.1021/acs.jmedchem.7b00622</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Scheme illustrating how RG4 can regulate translation initiation, thermal difference spectra of RG4s, <i>KRAS</i> 5′-UTR sequences in mammals, fluorescence titrations, UV and NMR spectra and HPLC-chromatograms, confocal microscopy, colony forming assay, and DNA and RNA sequences used in the study (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_001.pdf">jm7b00622_si_001.pdf (926.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_002.csv">jm7b00622_si_002.csv (0.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00622" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luigi E. Xodo</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Biochemistry Laboratory, University
of Udine, 33100 Udine, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3344-7207" title="Orcid link">http://orcid.org/0000-0003-3344-7207</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#315d445856581f495e555e71445f5844551f5845"><span class="__cf_email__" data-cfemail="38544d515f511640575c57784d56514d5c16514c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giulia Miglietta</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Biochemistry Laboratory, University
of Udine, 33100 Udine, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susanna Cogoi</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Biochemistry Laboratory, University
of Udine, 33100 Udine, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jessica Marinello</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, 40100 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giovanni Capranico</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, 40100 Bologna, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8708-6454" title="Orcid link">http://orcid.org/0000-0002-8708-6454</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander S. Tikhomirov</span> - <span class="hlFld-Affiliation affiliation">Gause Institute
of New Antibiotics, 119021 Moscow, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrey Shchekotikhin</span> - <span class="hlFld-Affiliation affiliation">Gause Institute
of New Antibiotics, 119021 Moscow, Russia</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>This study was conceived and written by L.E.X. and G.M. A.S.T. and A.S. designed and synthesized the G4 ligands, and G.M. and S.C. performed the experiments. J.M. and G.C. prepared and purified BG4. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d129e3365-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54144" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54144" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work has been carried out with the financial support of AIRC (Italian association for cancer research). IG2013, project code 143.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i35" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i35"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i36" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i36"> Abbreviations Used</h2><tr><td class="NLM_term"><i>KRAS</i></td><td class="NLM_def"><p class="first last">Kirsten ras gene</p></td></tr><tr><td class="NLM_term">PDAC</td><td class="NLM_def"><p class="first last">pancreatic ductal adenocarcinoma cells</p></td></tr><tr><td class="NLM_term">RG4</td><td class="NLM_def"><p class="first last">RNA G-quadruplex</p></td></tr><tr><td class="NLM_term">Panc-1</td><td class="NLM_def"><p class="first last">pancreatic cancer cells</p></td></tr><tr><td class="NLM_term">BxPC-3</td><td class="NLM_def"><p class="first last">pancreatic cancer cells</p></td></tr><tr><td class="NLM_term">ATPD</td><td class="NLM_def"><p class="first last">antrathiophenediones</p></td></tr><tr><td class="NLM_term">5′-UTR</td><td class="NLM_def"><p class="first last">5′-untranslated region</p></td></tr><tr><td class="NLM_term">ATFD</td><td class="NLM_def"><p class="first last">antrafurandiones</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">benzotriazol-1-yl-oxytripyrrolidinophosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">ethyldiisopropylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">ThT</td><td class="NLM_def"><p class="first last">thioflavin T</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 78 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherfils, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossman, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Der, C. J.</span><span> </span><span class="NLM_article-title">Ras superfamily GEFs and GAPs: validated and tractable targets for cancer therapy?</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1038/nrc2960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrc2960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21102635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=842-857&author=D.+Vigilauthor=J.+Cherfilsauthor=K.+L.+Rossmanauthor=C.+J.+Der&title=Ras+superfamily+GEFs+and+GAPs%3A+validated+and+tractable+targets+for+cancer+therapy%3F&doi=10.1038%2Fnrc2960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ras superfamily GEFs and GAPs: validated and tractable targets for cancer therapy?</span></div><div class="casAuthors">Vigil, Dominico; Cherfils, Jacqueline; Rossman, Kent L.; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">842-857</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is now considerable and increasing evidence for a causal role for aberrant activity of the Ras superfamily of small GTPases in human cancers.  These GTPases function as GDP-GTP-regulated binary switches that control many fundamental cellular processes.  A common mechanism of GTPase deregulation in cancer is the deregulated expression and/or activity of their regulatory proteins, guanine nucleotide exchange factors (GEFs) that promote formation of the active GTP-bound state and GTPase-activating proteins (GAPs) that return the GTPase to its GDP-bound inactive state.  In this Review, we assess the assocn. of GEFs and GAPs with cancer and their druggability for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwbf8B7sLmb7Vg90H21EOLACvtfcHk0ljRAEQCN9H7uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsbrM&md5=b7ad81f2fe0f7f7eaa5f015188f62e85</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2960%26sid%3Dliteratum%253Aachs%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DCherfils%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DRas%2520superfamily%2520GEFs%2520and%2520GAPs%253A%2520validated%2520and%2520tractable%2520targets%2520for%2520cancer%2520therapy%253F%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D842%26epage%3D857%26doi%3D10.1038%2Fnrc2960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Rodriguez-Viciana, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabatier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormick, F.</span><span> </span><span class="NLM_article-title">Signaling specificity by Ras family GTPases is determined by the full spectrum of effectors they regulate</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">4943</span><span class="NLM_x">–</span> <span class="NLM_lpage">4945</span><span class="refDoi"> DOI: 10.1128/MCB.24.11.4943-4954.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1128%2FMCB.24.11.4943-4954.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=15143186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=4943-4945&author=P.+Rodriguez-Vicianaauthor=C.+Sabatierauthor=F.+McCormick&title=Signaling+specificity+by+Ras+family+GTPases+is+determined+by+the+full+spectrum+of+effectors+they+regulate&doi=10.1128%2FMCB.24.11.4943-4954.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling specificity by Ras family GTPases is determined by the full spectrum of effectors they regulate</span></div><div class="casAuthors">Rodriguez-Viciana, Pablo; Sabatier, Celine; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4943-4954</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Ras family GTPases (RFGs) regulate signaling pathways that control multiple biol. processes.  How signaling specificity among the closely related family members is achieved is poorly understood.  We have taken a proteomics approach to signaling by RFGs, and we have analyzed interactions of a panel of RFGs with a comprehensive group of known and potential effectors.  We have found remarkable differences in the ability of RFGs to regulate the various isoforms of known effector families.  We have also identified several proteins as novel effectors of RFGs with differential binding specificities to the various RFGs.  We propose that specificity among RFGs is achieved by the differential regulation of combinations of effector families as well as by the selective regulation of different isoforms within an effector family.  An understanding of this new level of complexity in the signaling pathways regulated by RFGs is necessary to understand how they carry out their many cellular functions.  It will also likely have crit. implications in the treatment of human diseases such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyMRsdLoV7GLVg90H21EOLACvtfcHk0ljRAEQCN9H7uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhsbw%253D&md5=472eb17841cba28ce667c34cd07cb25e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.4943-4954.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.4943-4954.2004%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Viciana%26aufirst%3DP.%26aulast%3DSabatier%26aufirst%3DC.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DSignaling%2520specificity%2520by%2520Ras%2520family%2520GTPases%2520is%2520determined%2520by%2520the%2520full%2520spectrum%2520of%2520effectors%2520they%2520regulate%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D4943%26epage%3D4945%26doi%3D10.1128%2FMCB.24.11.4943-4954.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wortzel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seger, R.</span><span> </span><span class="NLM_article-title">The ERK cascade: distinct functions within various subcellular organelles</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1177/1947601911407328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1177%2F1947601911407328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21779493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=195-209&author=I.+Wortzelauthor=R.+Seger&title=The+ERK+cascade%3A+distinct+functions+within+various+subcellular+organelles&doi=10.1177%2F1947601911407328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The ERK cascade: Distinct functions within various subcellular organelles</span></div><div class="casAuthors">Wortzel, Inbal; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">195-209</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  The extracellular signal-regulated kinase 1/2 (ERK1/2) cascade is a central signaling pathway that regulates a wide variety of stimulated cellular processes, including mainly proliferation, differentiation, and survival, but apoptosis and stress response as well.  The ability of this linear cascade to induce so many distinct and even opposing effects after various stimulations raises the question as to how the signaling specificity of the cascade is regulated.  Over the past years, several specificity-mediating mechanisms have been elucidated, including temporal regulation, scaffolding interactions, crosstalks with other signaling components, substrate competition, and multiple components in each tier of the cascade.  In addn., spatial regulation of various components of the cascade is probably one of the main ways by which signals can be directed to some downstream targets and not to others.  Here, te authors describe 1st the components of the ERK1/2 cascade and their mode of regulation by kinases, phosphatases, and scaffold proteins.  In the 2nd part, the authors focus on the role of MEK1/2 and ERK1/2 compartmentalization in the nucleus, mitochondria, endosomes, plasma membrane, cytoskeleton, and Golgi app.  The authors explain that this spatial distribution may direct ERK1/2 signals to regulate the organelles' activities.  However, it can also direct the activity of the cascade's components to the outer surface of the organelles in order to bring them to close proximity to specific cytoplasmic targets.  The authors conclude that the dynamic localization of the ERK1/2 cascade components is an important regulatory mechanism in detg. the signaling specificity of the cascade, and its understanding should shed new light on the understanding of many stimulus-dependent processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7L1MPZYcUQ7Vg90H21EOLACvtfcHk0ljpBUjtSiwBSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7w%253D&md5=0321854677455eb57552cfc371146324</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1177%2F1947601911407328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911407328%26sid%3Dliteratum%253Aachs%26aulast%3DWortzel%26aufirst%3DI.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DThe%2520ERK%2520cascade%253A%2520distinct%2520functions%2520within%2520various%2520subcellular%2520organelles%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D195%26epage%3D209%26doi%3D10.1177%2F1947601911407328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Castellano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downward, J.</span><span> </span><span class="NLM_article-title">Ras interaction with PI3K: more than just another effector pathway</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">274</span><span class="refDoi"> DOI: 10.1177/1947601911408079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1177%2F1947601911408079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21779497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=261-274&author=E.+Castellanoauthor=J.+Downward&title=Ras+interaction+with+PI3K%3A+more+than+just+another+effector+pathway&doi=10.1177%2F1947601911408079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ras interaction with P13K: more than just another effector pathway</span></div><div class="casAuthors">Castellano, Esther; Downward, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-274</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  RAS proteins are small GTPases known for their involvement in oncogenesis: around 25% of human tumors present mutations in a member of this family.  RAS operates in a complex signaling network with multiple activators and effectors, which allows them to regulate many cellular functions such as cell proliferation, differentiation, apoptosis, and senescence.  Phosphatidylinositol 3-kinase (PI3K) is one of the main effector pathways of RAS, regulating cell growth, cell cycle entry, cell survival, cytoskeleton reorganization, and metab.  However, it is the involvement of this pathway in human tumors that has attracted most attention.  PI3K has proven to be necessary for RAS-induced transformation in vitro, and more importantly, mice with mutations in the PI3K catalytic subunit p110α that block its ability to interact with RAS are highly resistant to endogenous oncogenic KRAS-induced lung tumorigenesis and HRAS-induced skin carcinogenesis.  These animals also have a delayed development of the lymphatic vasculature.  Many PI3K inhibitors have been developed that are now in clin. trials.  However, it is a complex pathway with many feedback loops, and interactions with other pathways make the results of its inhibition hard to predict.  Combined therapy with another RAS-regulated pathway such as RAF/MEK/ERK may be the most effective way to treat cancer, at least in animal models mimicking the human disease.  In this review, we will summarize current knowledge about how RAS regulates one of its best-known effectors, PI3K.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLhXMI50nEbrVg90H21EOLACvtfcHk0ljpBUjtSiwBSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7c%253D&md5=2293eaf0d990f7638b4949dc88e744d9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1947601911408079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911408079%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DE.%26aulast%3DDownward%26aufirst%3DJ.%26atitle%3DRas%2520interaction%2520with%2520PI3K%253A%2520more%2520than%2520just%2520another%2520effector%2520pathway%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D261%26epage%3D274%26doi%3D10.1177%2F1947601911408079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Schubbert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span> </span><span class="NLM_article-title">Hyperactive Ras in developmental disorders and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1038/nrc2109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrc2109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17384584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlyhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=295-308&author=S.+Schubbertauthor=K.+Shannonauthor=G.+Bollag&title=Hyperactive+Ras+in+developmental+disorders+and+cancer&doi=10.1038%2Fnrc2109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hyperactive Ras in developmental disorders and cancer</span></div><div class="casAuthors">Schubbert, Suzanne; Shannon, Kevin; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-308</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ras genes are the most common targets for somatic gain-of-function mutations in human cancer.  Recently, germline mutations that affect components of the Ras-Raf-mitogen-activated and extracellular-signal regulated kinase kinase (MEK)-extracellular signal-regulated kinase (ERK) pathway were shown to cause several developmental disorders, including Nonan, Costello and cardio-facio-cutaneous syndromes.  Many of these mutant alleles encode proteins with aberrant biochem. and functional properties.  Here we will discuss the implications of germline mutations in the Ras-Raf-MEK-ERK pathway for understanding normal developmental processes and cancer pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRvjqrp1ho7Vg90H21EOLACvtfcHk0ljpBUjtSiwBSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlyhtLs%253D&md5=25fcaa959dae5e8512b959936bcbb347</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc2109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2109%26sid%3Dliteratum%253Aachs%26aulast%3DSchubbert%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DK.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DHyperactive%2520Ras%2520in%2520developmental%2520disorders%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D295%26epage%3D308%26doi%3D10.1038%2Fnrc2109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Ryan, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardeesy, N.</span><span> </span><span class="NLM_article-title">Pancreatic adenocarcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">2140</span><span class="NLM_x">–</span> <span class="NLM_lpage">2141</span><span class="refDoi"> DOI: 10.1056/NEJMra1404198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1056%2FNEJMra1404198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25427123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A280%3ADC%252BC2MzhtFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2014&pages=2140-2141&author=D.+P.+Ryanauthor=T.+S.+Hongauthor=N.+Bardeesy&title=Pancreatic+adenocarcinoma&doi=10.1056%2FNEJMra1404198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic adenocarcinoma</span></div><div class="casAuthors">Ryan David P; Hong Theodore S; Bardeesy Nabeel</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2140-1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXWXu7vS1YVeR_TJajKxJDfW6udTcc2eYn9Z9Oelf51Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzhtFCkug%253D%253D&md5=4ca27851b40f9613b10577597abcf1ce</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1404198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1404198%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DD.%2BP.%26aulast%3DHong%26aufirst%3DT.%2BS.%26aulast%3DBardeesy%26aufirst%3DN.%26atitle%3DPancreatic%2520adenocarcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D371%26spage%3D2140%26epage%3D2141%26doi%3D10.1056%2FNEJMra1404198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Prior, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">A comprehensive survey of Ras mutations in cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">2457</span><span class="NLM_x">–</span> <span class="NLM_lpage">2467</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F0008-5472.CAN-11-2612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22589270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2457-2467&author=I.+A.+Priorauthor=P.+D.+Lewisauthor=C.+Mattos&title=A+comprehensive+survey+of+Ras+mutations+in+cancer&doi=10.1158%2F0008-5472.CAN-11-2612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Survey of Ras Mutations in Cancer</span></div><div class="casAuthors">Prior, Ian A.; Lewis, Paul D.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2457-2467</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Ras, that promote oncogenesis when they are mutationally activated at codon 12, 13, or 61.  Although there is a high degree of similarity among the isoforms, K-Ras mutations are far more frequently obsd. in cancer, and each isoform displays preferential coupling to particular cancer types.  We examd. the mutational spectra of Ras isoforms curated from large-scale tumor profiling and found that each isoform exhibits surprisingly distinctive codon mutation and amino-acid substitution biases.  These findings were unexpected given that these mutations occur in regions that share 100% amino-acid sequence identity among the 3 isoforms.  Of importance, many of these mutational biases were not due to differences in exposure to mutagens, because the patterns were still evident when compared within specific cancer types.  We discuss potential genetic and epigenetic mechanisms, as well as isoform-specific differences in protein structure and signaling, that may promote these distinct mutation patterns and differential coupling to specific cancers.  Cancer Res; 72(10); 2457-67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg2lqiicosOLVg90H21EOLACvtfcHk0lgORQ7UGu7PDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D&md5=6d1894847015468f77c8b08b57c6d62a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2612%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DI.%2BA.%26aulast%3DLewis%26aufirst%3DP.%2BD.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DA%2520comprehensive%2520survey%2520of%2520Ras%2520mutations%2520in%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2457%26epage%3D2467%26doi%3D10.1158%2F0008-5472.CAN-11-2612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Karnoub, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span> </span><span class="NLM_article-title">Ras oncogenes: split personalities</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">517</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1038/nrm2438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrm2438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18568040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFGnsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=517-531&author=A.+E.+Karnoubauthor=R.+A.+Weinberg&title=Ras+oncogenes%3A+split+personalities&doi=10.1038%2Fnrm2438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ras oncogenes: split personalities</span></div><div class="casAuthors">Karnoub, Antoine E.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-531</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Extensive research over the past 30 years has revealed the involvement of Ras not only in tumorigenesis but also in many developmental disorders.  The complexity of the mol. and cell biol. mechanisms of action of Ras proteins indicates that much remains to be learns.  Extensive research on the Ras proteins and their functions in cell physiol. over the past 30 years has led to numerous insights that have revealed the involvement of Ras not only in tumorigenesis but also in many developmental disorders.  Despite great strides in our understanding of the mol. and cellular mechanisms of action of the Ras proteins, the expanding roster of their downstream effectors and the complexity of the signaling cascades that they regulate indicate that much remains to be learns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocptUdlhqaM7Vg90H21EOLACvtfcHk0lgORQ7UGu7PDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFGnsr0%253D&md5=9de78b17b953b2560238d7a98f3bea6a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrm2438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2438%26sid%3Dliteratum%253Aachs%26aulast%3DKarnoub%26aufirst%3DA.%2BE.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DRas%2520oncogenes%253A%2520split%2520personalities%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D517%26epage%3D531%26doi%3D10.1038%2Fnrm2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Collins, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brisset, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbán, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbán, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rakshit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flannagan, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adsay, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasca di Magliano, M.</span><span> </span><span class="NLM_article-title">Oncogenic Kras is required for both the initiation and maintenance of pancreatic cancer in mice</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">653</span><span class="refDoi"> DOI: 10.1172/JCI59227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1172%2FJCI59227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22232209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XitVelsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2012&pages=639-653&author=M.+A.+Collinsauthor=F.+Bednarauthor=Y.+Zhangauthor=J.+C.+Brissetauthor=S.+Galb%C3%A1nauthor=C.+J.+Galb%C3%A1nauthor=S.+Rakshitauthor=K.+S.+Flannaganauthor=N.+V.+Adsayauthor=M.+Pasca+di+Magliano&title=Oncogenic+Kras+is+required+for+both+the+initiation+and+maintenance+of+pancreatic+cancer+in+mice&doi=10.1172%2FJCI59227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Kras is required for both the initiation and maintenance of pancreatic cancer in mice</span></div><div class="casAuthors">Collins, Meredith A.; Bednar, Filip; Zhang, Yaqing; Brisset, Jean-Christophe; Galban, Stefanie; Galban, Craig J.; Rakshit, Sabita; Flannagan, Karen S.; Adsay, N. Volkan; di Magliano, Marina Pasca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">639-653</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Pancreatic cancer is almost invariably assocd. with mutations in the KRAS gene, most commonly KRASG12D, that result in a dominant-active form of the KRAS GTPase.  However, how KRAS mutations promote pancreatic carcinogenesis is not fully understood, and whether oncogenic KRAS is required for the maintenance of pancreatic cancer has not been established.  To address these questions, we generated two mouse models of pancreatic tumorigenesis: mice transgenic for inducible KrasG12D, which allows for inducible, pancreas-specific, and reversible expression of the oncogenic KrasG12D, with or without inactivation of one allele of the tumor suppressor gene p53.  Here, we report that, early in tumorigenesis, induction of oncogenic KrasG12D reversibly altered normal epithelial differentiation following tissue damage, leading to precancerous lesions.  Inactivation of KrasG12D in established precursor lesions and during progression to cancer led to regression of the lesions, indicating that KrasG12D was required for tumor cell survival.  Strikingly, during all stages of carcinogenesis, KrasG12D upregulated Hedgehog signaling, inflammatory pathways, and several pathways known to mediate paracrine interactions between epithelial cells and their surrounding microenvironment, thus promoting formation and maintenance of the fibroinflammatory stroma that plays a pivotal role in pancreatic cancer.  Our data establish that epithelial KrasG12D influences multiple cell types to drive pancreatic tumorigenesis and is essential for tumor maintenance.  They also strongly support the notion that inhibiting KrasG12D, or its downstream effectors, could provide a new approach for the treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA_MsyfjFbgrVg90H21EOLACvtfcHk0lgORQ7UGu7PDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitVelsrc%253D&md5=576556a0569dd9d1517acbe405fe64c5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1172%2FJCI59227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI59227%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DM.%2BA.%26aulast%3DBednar%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBrisset%26aufirst%3DJ.%2BC.%26aulast%3DGalb%25C3%25A1n%26aufirst%3DS.%26aulast%3DGalb%25C3%25A1n%26aufirst%3DC.%2BJ.%26aulast%3DRakshit%26aufirst%3DS.%26aulast%3DFlannagan%26aufirst%3DK.%2BS.%26aulast%3DAdsay%26aufirst%3DN.%2BV.%26aulast%3DPasca%2Bdi%2BMagliano%26aufirst%3DM.%26atitle%3DOncogenic%2520Kras%2520is%2520required%2520for%2520both%2520the%2520initiation%2520and%2520maintenance%2520of%2520pancreatic%2520cancer%2520in%2520mice%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2012%26volume%3D122%26spage%3D639%26epage%3D653%26doi%3D10.1172%2FJCI59227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Kanda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthaei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maitra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goggins, M.</span><span> </span><span class="NLM_article-title">Presence of somatic mutations in most early-stage pancreatic intraepithelial neoplasia</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">142</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">733</span><span class="refDoi"> DOI: 10.1053/j.gastro.2011.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1053%2Fj.gastro.2011.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2012&pages=730-733&author=M.+Kandaauthor=H.+Matthaeiauthor=J.+Wuauthor=S.+M.+Hongauthor=J.+Yuauthor=M.+Borgesauthor=R.+H.+Hrubanauthor=A.+Maitraauthor=K.+Kinzlerauthor=B.+Vogelsteinauthor=M.+Goggins&title=Presence+of+somatic+mutations+in+most+early-stage+pancreatic+intraepithelial+neoplasia&doi=10.1053%2Fj.gastro.2011.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2011.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2011.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DKanda%26aufirst%3DM.%26aulast%3DMatthaei%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%2BM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DBorges%26aufirst%3DM.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DKinzler%26aufirst%3DK.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DGoggins%26aufirst%3DM.%26atitle%3DPresence%2520of%2520somatic%2520mutations%2520in%2520most%2520early-stage%2520pancreatic%2520intraepithelial%2520neoplasia%26jtitle%3DGastroenterology%26date%3D2012%26volume%3D142%26spage%3D730%26epage%3D733%26doi%3D10.1053%2Fj.gastro.2011.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Vincent, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goggins, M.</span><span> </span><span class="NLM_article-title">Pancreatic cancer</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">378</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0140-6736(10)62307-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FS0140-6736%2810%2962307-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21620466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A280%3ADC%252BC3MjlvFOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2011&pages=607-620&author=A.+Vincentauthor=J.+Hermanauthor=R.+Schulickauthor=R.+H.+Hrubanauthor=M.+Goggins&title=Pancreatic+cancer&doi=10.1016%2FS0140-6736%2810%2962307-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic cancer</span></div><div class="casAuthors">Vincent Audrey; Herman Joseph; Schulick Rich; Hruban Ralph H; Goggins Michael</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">9791</span>),
    <span class="NLM_cas:pages">607-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Substantial progress has been made in our understanding of the biology of pancreatic cancer, and advances in patients' management have also taken place.  Evidence is beginning to show that screening first-degree relatives of individuals with several family members affected by pancreatic cancer can identify non-invasive precursors of this malignant disease.  The incidence of and number of deaths caused by pancreatic tumours have been gradually rising, even as incidence and mortality of other common cancers have been declining.  Despite developments in detection and management of pancreatic cancer, only about 4% of patients will live 5 years after diagnosis.  Survival is better for those with malignant disease localised to the pancreas, because surgical resection at present offers the only chance of cure.  Unfortunately, 80-85% of patients present with advanced unresectable disease.  Furthermore, pancreatic cancer responds poorly to most chemotherapeutic agents.  Hence, we need to understand the biological mechanisms that contribute to development and progression of pancreatic tumours.  In this Seminar we will discuss the most common and deadly form of pancreatic cancer, pancreatic ductal adenocarcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_Px7QoMFLVGhHGZei4OW9fW6udTcc2eas657hYPXkzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjlvFOmsg%253D%253D&md5=fb7f79c8c02f2f2bad5e5ad7a9b09096</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2810%2962307-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252810%252962307-0%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DA.%26aulast%3DHerman%26aufirst%3DJ.%26aulast%3DSchulick%26aufirst%3DR.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DGoggins%26aufirst%3DM.%26atitle%3DPancreatic%2520cancer%26jtitle%3DLancet%26date%3D2011%26volume%3D378%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0140-6736%2810%2962307-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Ying, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmelman, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssiotis, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher-Sananikone, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locasale, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coloff, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paik, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guimaraes, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hezel, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissleder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span> </span><span class="NLM_article-title">Oncogenic Kras maintains pancreatic tumors through regulation of anabolic glucose metabolism</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">656</span><span class="NLM_x">–</span> <span class="NLM_lpage">670</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.cell.2012.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22541435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Kjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=656-670&author=H.+Yingauthor=A.+C.+Kimmelmanauthor=C.+A.+Lyssiotisauthor=S.+Huaauthor=G.+C.+Chuauthor=E.+Fletcher-Sananikoneauthor=J.+W.+Locasaleauthor=J.+Sonauthor=H.+Zhangauthor=J.+L.+Coloffauthor=H.+Yanauthor=W.+Wangauthor=S.+Chenauthor=A.+Vialeauthor=H.+Zhengauthor=J.+H.+Paikauthor=C.+Limauthor=A.+R.+Guimaraesauthor=E.+S.+Martinauthor=J.+Changauthor=A.+F.+Hezelauthor=S.+R.+Perryauthor=J.+Huauthor=B.+Ganauthor=Y.+Xiaoauthor=J.+M.+Asaraauthor=R.+Weisslederauthor=Y.+A.+Wangauthor=L.+Chinauthor=L.+C.+Cantleyauthor=R.+A.+DePinho&title=Oncogenic+Kras+maintains+pancreatic+tumors+through+regulation+of+anabolic+glucose+metabolism&doi=10.1016%2Fj.cell.2012.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Kras maintains pancreatic tumors through regulation of anabolic glucose metabolism</span></div><div class="casAuthors">Ying, Haoqiang; Kimmelman, Alec C.; Lyssiotis, Costas A.; Hua, Sujun; Chu, Gerald C.; Fletcher-Sananikone, Eliot; Locasale, Jason W.; Son, Jaekyoung; Zhang, Hailei; Coloff, Jonathan L.; Yan, Haiyan; Wang, Wei; Chen, Shujuan; Viale, Andrea; Zheng, Hongwu; Paik, Ji-hye; Lim, Carol; Guimaraes, Alexander R.; Martin, Eric S.; Chang, Jeffery; Hezel, Aram F.; Perry, Samuel R.; Hu, Jian; Gan, Boyi; Xiao, Yonghong; Asara, John M.; Weissleder, Ralph; Wang, Y. Alan; Chin, Lynda; Cantley, Lewis C.; DePinho, Ronald A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">656-670</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tumor maintenance relies on continued activity of driver oncogenes, although their rate-limiting role is highly context dependent.  Oncogenic Kras mutation is the signature event in pancreatic ductal adenocarcinoma (PDAC), serving a crit. role in tumor initiation.  Here, an inducible KrasG12D-driven PDAC mouse model establishes that advanced PDAC remains strictly dependent on KrasG12D expression.  Transcriptome and metabolomic analyses indicate that KrasG12D serves a vital role in controlling tumor metab. through stimulation of glucose uptake and channeling of glucose intermediates into the hexosamine biosynthesis and pentose phosphate pathways (PPP).  These studies also reveal that oncogenic Kras promotes ribose biogenesis.  Unlike canonical models, we demonstrate that KrasG12D drives glycolysis intermediates into the nonoxidative PPP, thereby decoupling ribose biogenesis from NADP/NADPH-mediated redox control.  Together, this work provides in vivo mechanistic insights into how oncogenic Kras promotes metabolic reprogramming in native tumors and illuminates potential metabolic targets that can be exploited for therapeutic benefit in PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0n15htmLPUrVg90H21EOLACvtfcHk0lgD8C9P44KWig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Kjs7k%253D&md5=c8474034804faf04fb8bd108dd958540</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DChu%26aufirst%3DG.%2BC.%26aulast%3DFletcher-Sananikone%26aufirst%3DE.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DColoff%26aufirst%3DJ.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DPaik%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DC.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DMartin%26aufirst%3DE.%2BS.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DHezel%26aufirst%3DA.%2BF.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DGan%26aufirst%3DB.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26atitle%3DOncogenic%2520Kras%2520maintains%2520pancreatic%2520tumors%2520through%2520regulation%2520of%2520anabolic%2520glucose%2520metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D656%26epage%3D670%26doi%3D10.1016%2Fj.cell.2012.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssiotis, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ying, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ligorio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrone, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullarky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyh-Chang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleming, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardeesy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haigis, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmelman, A. C.</span><span> </span><span class="NLM_article-title">Glutamine supports pancreatic cancer growth through a KRAS-regulated metabolic pathway</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">496</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature12040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnature12040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23535601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFyjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=101-105&author=J.+Sonauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=X.+Wangauthor=S.+Huaauthor=M.+Ligorioauthor=R.+M.+Pereraauthor=C.+R.+Ferroneauthor=E.+Mullarkyauthor=N.+Shyh-Changauthor=Y.+Kangauthor=J.+B.+Flemingauthor=N.+Bardeesyauthor=J.+M.+Asaraauthor=M.+C.+Haigisauthor=R.+A.+DePinhoauthor=L.+C.+Cantleyauthor=A.+C.+Kimmelman&title=Glutamine+supports+pancreatic+cancer+growth+through+a+KRAS-regulated+metabolic+pathway&doi=10.1038%2Fnature12040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine supports pancreatic cancer growth through a KRAS-regulated metabolic pathway</span></div><div class="casAuthors">Son, Jaekyoung; Lyssiotis, Costas A.; Ying, Haoqiang; Wang, Xiaoxu; Hua, Sujun; Ligorio, Matteo; Perera, Rushika M.; Ferrone, Cristina R.; Mullarky, Edouard; Ng, Shyh-Chang; Kang, Ya'an; Fleming, Jason B.; Bardeesy, Nabeel; Asara, John M.; Haigis, Marcia C.; DePinho, Ronald A.; Cantley, Lewis C.; Kimmelman, Alec C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7443</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells have metabolic dependencies that distinguish them from their normal counterparts.  Among these dependencies is an increased use of the amino acid glutamine to fuel anabolic processes.  Indeed, the spectrum of glutamine-dependent tumors and the mechanisms whereby glutamine supports cancer metab. remain areas of active investigation.  Here we report the identification of a non-canonical pathway of glutamine use in human pancreatic ductal adenocarcinoma (PDAC) cells that is required for tumor growth.  Whereas most cells use glutamate dehydrogenase (GLUD1) to convert glutamine-derived glutamate into α-ketoglutarate in the mitochondria to fuel the tricarboxylic acid cycle, PDAC relies on a distinct pathway in which glutamine-derived aspartate is transported into the cytoplasm where it can be converted into oxaloacetate by aspartate transaminase (GOT1).  Subsequently, this oxaloacetate is converted into malate and then pyruvate, ostensibly increasing the NADPH/NADP+ ratio which can potentially maintain the cellular redox state.  Importantly, PDAC cells are strongly dependent on this series of reactions, as glutamine deprivation or genetic inhibition of any enzyme in this pathway leads to an increase in reactive oxygen species and a redn. in reduced glutathione.  Moreover, knockdown of any component enzyme in this series of reactions also results in a pronounced suppression of PDAC growth in vitro and in vivo.  Furthermore, we establish that the reprogramming of glutamine metab. is mediated by oncogenic KRAS, the signature genetic alteration in PDAC, through the transcriptional upregulation and repression of key metabolic enzymes in this pathway.  The essentiality of this pathway in PDAC and the fact that it is dispensable in normal cells may provide novel therapeutic approaches to treat these refractory tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzWAEWZvCWcbVg90H21EOLACvtfcHk0lgD8C9P44KWig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFyjsrc%253D&md5=162af18c1102745226aa2096179f1cc6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature12040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12040%26sid%3Dliteratum%253Aachs%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DLigorio%26aufirst%3DM.%26aulast%3DPerera%26aufirst%3DR.%2BM.%26aulast%3DFerrone%26aufirst%3DC.%2BR.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DShyh-Chang%26aufirst%3DN.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DFleming%26aufirst%3DJ.%2BB.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DHaigis%26aufirst%3DM.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26atitle%3DGlutamine%2520supports%2520pancreatic%2520cancer%2520growth%2520through%2520a%2520KRAS-regulated%2520metabolic%2520pathway%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fnature12040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Weinstein, I. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsher, D.</span><span> </span><span class="NLM_article-title">Oncogene addiction</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3077</span><span class="NLM_x">–</span> <span class="NLM_lpage">3080</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-3293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F0008-5472.CAN-07-3293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18451130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltlSmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3077-3080&author=I.+B.+Weinsteinauthor=A.+Joeauthor=D.+Felsher&title=Oncogene+addiction&doi=10.1158%2F0008-5472.CAN-07-3293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogene Addiction</span></div><div class="casAuthors">Weinstein, I. Bernard; Joe, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3077-3080</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Cancer cells contain multiple genetic and epigenetic abnormalities.  Despite this complexity, their growth and survival can often be impaired by the inactivation of a single oncogene.  This phenomenon, called "oncogene addiction," provides a rationale for mol. targeted therapy.  The efficacy of this strategy requires novel methods, including integrative genomics and systems biol., to identify the state of oncogene addiction (i.e., the "Achilles heel") in specific cancers.  Combination therapy may also be required to prevent the escape of cancers from a given state of oncogene addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobVcTGxEyDNLVg90H21EOLACvtfcHk0lgD8C9P44KWig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltlSmtrk%253D&md5=fc62277a0200431335f79c420ebb02a6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-3293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-3293%26sid%3Dliteratum%253Aachs%26aulast%3DWeinstein%26aufirst%3DI.%2BB.%26aulast%3DJoe%26aufirst%3DA.%26aulast%3DFelsher%26aufirst%3DD.%26atitle%3DOncogene%2520addiction%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3077%26epage%3D3080%26doi%3D10.1158%2F0008-5472.CAN-07-3293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucharski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Selective inhibition of oncogenic KRAS output with small molecules targeting the inactive state</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+inhibition+of+oncogenic+KRAS+output+with+small+molecules+targeting+the+inactive+state&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0lhqPFQqYYRqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520oncogenic%2520KRAS%2520output%2520with%2520small%2520molecules%2520targeting%2520the%2520inactive%2520state%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wilson, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tolias, P.</span><span> </span><span class="NLM_article-title">Recent advances in cancer drug discovery targeting RAS</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1915</span><span class="NLM_x">–</span> <span class="NLM_lpage">1919</span><span class="refDoi"> DOI: 10.1016/j.drudis.2016.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.drudis.2016.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27506872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlaiu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1915-1919&author=C.+Y.+Wilsonauthor=P.+Tolias&title=Recent+advances+in+cancer+drug+discovery+targeting+RAS&doi=10.1016%2Fj.drudis.2016.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in cancer drug discovery targeting RAS</span></div><div class="casAuthors">Wilson, Candice Y.; Tolias, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1915-1919</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mutated RAS is present in 30% of human tumors, appearing in 90% of pancreatic, 45% of colon and 35% of lung cancers.  These high occurrences make RAS one of the most important drug targets in oncol.  Three decades of effort to target RAS have been unsuccessful in generating drug therapies suggesting that it might represent an 'undruggable' target.  However, recent reports highlighting new approaches for targeting RAS have uncovered more information on protein structure and identified new binding pockets.  Efforts to target the KRAS G12C mutation specifically have shown promising results whereas other approaches have targeted various protein complexes.  These advances could lead to development of new effective cancer drugs targeting RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJnoLMitHbsrVg90H21EOLACvtfcHk0lhqPFQqYYRqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlaiu73P&md5=5bcad00948b76112248f9ebc332a8d73</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DC.%2BY.%26aulast%3DTolias%26aufirst%3DP.%26atitle%3DRecent%2520advances%2520in%2520cancer%2520drug%2520discovery%2520targeting%2520RAS%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1915%26epage%3D1919%26doi%3D10.1016%2Fj.drudis.2016.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Stephen, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esposito, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagni, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormick, F.</span><span> </span><span class="NLM_article-title">Dragging ras back in the ring</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1016/j.ccr.2014.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.ccr.2014.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24651010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVWhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=272-281&author=A.+G.+Stephenauthor=D.+Espositoauthor=R.+K.+Bagniauthor=F.+McCormick&title=Dragging+ras+back+in+the+ring&doi=10.1016%2Fj.ccr.2014.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Dragging Ras Back in the Ring</span></div><div class="casAuthors">Stephen, Andrew G.; Esposito, Dominic; Bagni, Rachel K.; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">272-281</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Ras proteins play a major role in human cancers but have not yielded to therapeutic attack.  Ras-driven cancers are among the most difficult to treat and often excluded from therapies.  The Ras proteins have been termed "undruggable," based on failures from an era in which understanding of signaling transduction, feedback loops, redundancy, tumor heterogeneity, and Ras' oncogenic role was poor.  Structures of Ras oncoproteins bound to their effectors or regulators are unsolved, and it is unknown precisely how Ras proteins activate their downstream targets.  These knowledge gaps have impaired development of therapeutic strategies.  A better understanding of Ras biol. and biochem., coupled with new ways of targeting undruggable proteins, is likely to lead to new ways of defeating Ras-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsJEUXbjmDe7Vg90H21EOLACvtfcHk0lhqPFQqYYRqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVWhtLY%253D&md5=1ecb33c95db7d5d6f910b9e2af2ec44e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DStephen%26aufirst%3DA.%2BG.%26aulast%3DEsposito%26aufirst%3DD.%26aulast%3DBagni%26aufirst%3DR.%2BK.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DDragging%2520ras%2520back%2520in%2520the%2520ring%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D272%26epage%3D281%26doi%3D10.1016%2Fj.ccr.2014.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1038/nrd.2016.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrd.2016.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27469033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=771-785&author=J.+M.+Ostremauthor=K.+M.+Shokat&title=Direct+small-molecule+inhibitors+of+KRAS%3A+from+structural+insights+to+mechanism-based+design&doi=10.1038%2Fnrd.2016.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span></div><div class="casAuthors">Ostrem, Jonathan M. L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">771-785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer.  In addn. to holding this distinction, unsuccessful attempts to target this protein have led to the characterization of RAS as 'undruggable'.  However, recent advances in technol. and novel approaches to drug discovery have renewed hope that a direct KRAS inhibitor may be on the horizon.  In this Review, we provide an in-depth anal. of the structure, dynamics, mutational activation and inactivation, and signalling mechanisms of RAS.  From this perspective, we then consider potential mechanisms of action for effective RAS inhibitors.  Finally, we examine each of the many recent reports of direct RAS inhibitors and discuss promising avenues for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBobrhZLF3V7Vg90H21EOLACvtfcHk0likb2SECs_SaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK&md5=b0832c7ef45cc0555e11c59baccc5abd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.139%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DDirect%2520small-molecule%2520inhibitors%2520of%2520KRAS%253A%2520from%2520structural%2520insights%2520to%2520mechanism-based%2520design%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D771%26epage%3D785%26doi%3D10.1038%2Fnrd.2016.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kiessling, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogler, G.</span><span> </span><span class="NLM_article-title">Targeting the ras pathwayby mitogene-activated protein kinase inhibitors</span> <span class="citation_source-journal">Swiss Med. Wkly.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">w14207</span><span class="refDoi"> DOI: 10.4414/smw.2015.14207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.4414%2Fsmw.2015.14207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26691679" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2015&pages=w14207&author=M.+K.+Kiesslingauthor=G.+Rogler&title=Targeting+the+ras+pathwayby+mitogene-activated+protein+kinase+inhibitors&doi=10.4414%2Fsmw.2015.14207"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4414%2Fsmw.2015.14207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4414%252Fsmw.2015.14207%26sid%3Dliteratum%253Aachs%26aulast%3DKiessling%26aufirst%3DM.%2BK.%26aulast%3DRogler%26aufirst%3DG.%26atitle%3DTargeting%2520the%2520ras%2520pathwayby%2520mitogene-activated%2520protein%2520kinase%2520inhibitors%26jtitle%3DSwiss%2520Med.%2520Wkly.%26date%3D2015%26volume%3D145%26spage%3Dw14207%26doi%3D10.4414%2Fsmw.2015.14207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparano, J. A.</span><span> </span><span class="NLM_article-title">Farnesyl transferase inhibitors</span> <span class="citation_source-journal">Cancer Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">653</span><span class="NLM_x">–</span> <span class="NLM_lpage">661</span><span class="refDoi"> DOI: 10.1080/07357900801971016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1080%2F07357900801971016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=653-661&author=T.+Liauthor=J.+A.+Sparano&title=Farnesyl+transferase+inhibitors&doi=10.1080%2F07357900801971016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1080%2F07357900801971016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07357900801971016%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSparano%26aufirst%3DJ.%2BA.%26atitle%3DFarnesyl%2520transferase%2520inhibitors%26jtitle%3DCancer%2520Invest.%26date%3D2008%26volume%3D26%26spage%3D653%26epage%3D661%26doi%3D10.1080%2F07357900801971016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Cox, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Der, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philips, M. R.</span><span> </span><span class="NLM_article-title">Targeting RAs membrane association: back to the future for anti-RAS drug discovery</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1819</span><span class="NLM_x">–</span> <span class="NLM_lpage">1827</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-3214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F1078-0432.CCR-14-3214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25878363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsl2gt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1819-1827&author=A.+D.+Coxauthor=C.+J.+Derauthor=M.+R.+Philips&title=Targeting+RAs+membrane+association%3A+back+to+the+future+for+anti-RAS+drug+discovery&doi=10.1158%2F1078-0432.CCR-14-3214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RAS Membrane Association: Back to the Future for Anti-RAS Drug Discovery?</span></div><div class="casAuthors">Cox, Adrienne D.; Der, Channing J.; Philips, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1819-1827</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">RAS proteins require membrane assocn. for their biol. activity, making this assocn. a logical target for anti-RAS therapeutics.  Lipid modification of RAS proteins by a farnesyl isoprenoid is an obligate step in that assocn., and is an enzymic process.  Accordingly, farnesyltransferase inhibitors (FTI) were developed as potential anti-RAS drugs.  The lack of efficacy of FTIs as anticancer drugs was widely seen as indicating that blocking RAS membrane assocn. was a flawed approach to cancer treatment.  However, a deeper understanding of RAS modification and trafficking has revealed that this was an erroneous conclusion.  In the presence of FTIs, KRAS and NRAS, which are the RAS isoforms most frequently mutated in cancer, become substrates for alternative modification, can still assoc. with membranes, and can still function.  Thus, FTIs failed not because blocking RAS membrane assocn. is an ineffective approach, but because FTIs failed to accomplish that task.  Recent findings regarding RAS isoform trafficking and the regulation of RAS subcellular localization have rekindled interest in efforts to target these processes.  In particular, improved understanding of the palmitoylation/depalmitoylation cycle that regulates RAS interaction with the plasma membrane, endomembranes, and cytosol, and of the potential importance of RAS chaperones, have led to new approaches.  Efforts to validate and target other enzymically regulated posttranslational modifications are also ongoing.  In this review, we revisit lessons learned, describe the current state of the art, and highlight challenging but promising directions to achieve the goal of disrupting RAS membrane assocn. and subcellular localization for anti-RAS drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCImUyZbuCKrVg90H21EOLACvtfcHk0likb2SECs_SaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsl2gt74%253D&md5=6f41ac23a7d14b90633755e374b6ffea</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3214%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DPhilips%26aufirst%3DM.%2BR.%26atitle%3DTargeting%2520RAs%2520membrane%2520association%253A%2520back%2520to%2520the%2520future%2520for%2520anti-RAS%2520drug%2520discovery%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1819%26epage%3D1827%26doi%3D10.1158%2F1078-0432.CCR-14-3214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Tamanoi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Recent progress in developing small molecule inhibitors designed to interfere with ras membrane association: toward inhibiting KRAS and NRAS functions</span> <span class="citation_source-journal">Enzymes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1016/B978-0-12-420146-0.00008-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FB978-0-12-420146-0.00008-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25034105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=181-200&author=F.+Tamanoiauthor=J.+Lu&title=Recent+progress+in+developing+small+molecule+inhibitors+designed+to+interfere+with+ras+membrane+association%3A+toward+inhibiting+KRAS+and+NRAS+functions&doi=10.1016%2FB978-0-12-420146-0.00008-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in developing small molecule inhibitors designed to interfere with Ras membrane association: toward inhibiting K-Ras and N-Ras functions</span></div><div class="casAuthors">Tamanoi, Fuyuhiko; Lu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Enzymes</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">181-200</span>CODEN:
                <span class="NLM_cas:coden">25GLAS</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  K-Ras and N-Ras are mutated in a wide range of human cancers, thus making these proteins attractive targets of anticancer drug development.  However, no effective compds. have been obtained so far.  One of the approaches taken to inhibit the function of K-Ras and N-Ras is to interfere with their membrane assocn.  Various attempts have been taken.  In the first example, we examine the approach conceived in early 1990s to inhibit protein prenylation that is required for their membrane assocn.  The initial premise that the inhibition of Ras farnesylation leads to the inhibition of Ras was not realized, mainly due to alternative prenylation of K-Ras and N-Ras proteins.  This led to the idea that the combined inhibition of FTase and GGTase-I can block membrane assocn. of K-Ras and N-Ras.  Dual specificity inhibitors of FTase and GGTase-I (DPIs) were also developed.  These compds. were tested in preclin. and clin. studies.  It appears that sufficiently high concn. of the drug to inhibit K-Ras was not achieved in previous attempts.  In addn., dose-limiting toxicity has been obsd. and this was primarily ascribed to GGTase-I inhibition.  Strategies to confer cancer targeting capabilities to the inhibitors may overcome the dose-limiting toxicity.  In the second approach, postprenylation events were exploited.  This led to the development of various inhibitors including the ICMT inhibitors.  Finally, recent identification of compds. that inhibit the interaction between K-Ras and PDE-δ is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMwOKCVlBNLrVg90H21EOLACvtfcHk0likb2SECs_SaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSgsL4%253D&md5=843c567089ab46424549b1f183d50fba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-420146-0.00008-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-420146-0.00008-1%26sid%3Dliteratum%253Aachs%26aulast%3DTamanoi%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DRecent%2520progress%2520in%2520developing%2520small%2520molecule%2520inhibitors%2520designed%2520to%2520interfere%2520with%2520ras%2520membrane%2520association%253A%2520toward%2520inhibiting%2520KRAS%2520and%2520NRAS%2520functions%26jtitle%3DEnzymes%26date%3D2013%26volume%3D34%26spage%3D181%26epage%3D200%26doi%3D10.1016%2FB978-0-12-420146-0.00008-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paramasivam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3780</span><span class="refDoi"> DOI: 10.1093/nar/gkn120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkn120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18490377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3765-3780&author=S.+Cogoiauthor=M.+Paramasivamauthor=B.+Spolaoreauthor=L.+E.+Xodo&title=Structural+polymorphism+within+a+regulatory+element+of+the+human+KRAS+promoter%3A+formation+of+G4-DNA+recognized+by+nuclear+proteins&doi=10.1093%2Fnar%2Fgkn120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span></div><div class="casAuthors">Cogoi, Susanna; Paramasivam, Manikandan; Spolaore, Barbara; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3765-3780</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The human KRAS proto-oncogene contains a crit. nuclease hypersensitive element (NHE) upstream of the major transcription initiation site.  In this article, we demonstrate by primer-extension expts., PAGE, chem. footprinting, CD, UV and FRET expts. that the G-rich strand of NHE (32R) folds into intra-mol. G-quadruplex structures.  Fluorescence data show that 32R in 100 mM KCl melts with a biphasic profile, showing the formation of two distinct G-quadruplexes with Tm of ∼55°C (Q1) and ∼72°C (Q2).  DMS-footprinting and CD suggest that Q1 can be a parallel and Q2 a mixed parallel/antiparallel G-quadruplex.  When dsNHE (32R hybridized to its complementary) is incubated with a nuclear ext. from Panc-1 cells, three DNA-protein complexes are obsd. by EMSA.  The complex of slower mobility is competed by quadruplex 32R, but not by mutant oligonucleotides, which cannot form a quadruplex structure.  Using paramagnetic beads coupled with 32R, we pulled down from the Panc-1 ext. proteins with affinity for quadruplex 32R.  One of these is the heterogeneous nuclear ribonucleoprotein A1, which was previously reported to unfold quadruplex DNA.  These study suggests a role of quadruplex DNA in KRAS transcription and provides the basis for the rationale design of mol. strategies to inhibit the expression of KRAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5riE_L0077Vg90H21EOLACvtfcHk0lgnwJ_oXie78w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D&md5=73fcda8805e5f3989bcf6824805ed046</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn120%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DParamasivam%26aufirst%3DM.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DStructural%2520polymorphism%2520within%2520a%2520regulatory%2520element%2520of%2520the%2520human%2520KRAS%2520promoter%253A%2520formation%2520of%2520G4-DNA%2520recognized%2520by%2520nuclear%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3765%26epage%3D3780%26doi%3D10.1093%2Fnar%2Fgkn120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Faudale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Photoactivated cationic alkyl-substituted porphyrin binding to g4-RNA in the 5′-UTR of KRAS oncogene represses translation</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1039/C1CC15850C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1039%2FC1CC15850C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22127206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ersbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=874-876&author=M.+Faudaleauthor=S.+Cogoiauthor=L.+E.+Xodo&title=Photoactivated+cationic+alkyl-substituted+porphyrin+binding+to+g4-RNA+in+the+5%E2%80%B2-UTR+of+KRAS+oncogene+represses+translation&doi=10.1039%2FC1CC15850C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Photoactivated cationic alkyl-substituted porphyrin binding to g4-RNA in the 5'-UTR of KRAS oncogene represses translation</span></div><div class="casAuthors">Faudale, Mariangela; Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">874-876</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The KRAS transcript is characterized by a 192-nt 5'-UTR contg. repetitive runs of two-guanines which can fold in several G-quadruplexes.  These folded structures have a high affinity for the cationic porphyrin tri-meso(N-methyl-4-pyridyl), meso(N-tetradecyl-4-pyridyl) porphine (TMPyP4-C14), which efficiently penetrates cell membranes.  Upon photoactivation TMPyP4-C14 induces a dramatic down-regulation of oncogenic KRAS and cell growth arrest in pancreatic cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IgGfekwEpLVg90H21EOLACvtfcHk0lgnwJ_oXie78w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ersbnE&md5=2743606db2e9469d526a586af884d894</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC1CC15850C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC1CC15850C%26sid%3Dliteratum%253Aachs%26aulast%3DFaudale%26aufirst%3DM.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DPhotoactivated%2520cationic%2520alkyl-substituted%2520porphyrin%2520binding%2520to%2520g4-RNA%2520in%2520the%25205%25E2%2580%25B2-UTR%2520of%2520KRAS%2520oncogene%2520represses%2520translation%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2012%26volume%3D48%26spage%3D874%26epage%3D876%26doi%3D10.1039%2FC1CC15850C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacchigna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Anticancer activity of cationic porphyrins in melanoma tumour-bearing mice and mechanistic in vitro studies</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1186/1476-4598-13-75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1186%2F1476-4598-13-75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24684778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVWks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=75-92&author=V.+Rapozziauthor=S.+Zorzetauthor=M.+Zacchignaauthor=E.+Della+Pietraauthor=S.+Cogoiauthor=L.+E.+Xodo&title=Anticancer+activity+of+cationic+porphyrins+in+melanoma+tumour-bearing+mice+and+mechanistic+in+vitro+studies&doi=10.1186%2F1476-4598-13-75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer activity of cationic porphyrins in melanoma tumour-bearing mice and mechanistic in vitro studies</span></div><div class="casAuthors">Rapozzi, Valentina; Zorzet, Sonia; Zacchigna, Marina; Della Pietra, Emilia; Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75/1-75/17, 17</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Porphyrin TMPyP4 (P4) and its C14H28-alkyl deriv. (C14) are G-quadruplex binders and singlet oxygen (1O2) generators.  In contrast, TMPyP2 (P2) produces 1O2 but it is not a G-quadruplex binder.  As their photosensitizing activity is currently undefined, we report in this study their efficacy against a melanoma skin tumor and describe an in vitro mechanistic study which gives insights into their anticancer activity.  Methods: Uptake and antiproliferative activity of photoactivated P2, P4 and C14 have been investigated in murine melanoma B78-H1 cells by FACS, clonogenic and migration assays.  Apoptosis was investigated by PARP-1 cleavage and annexin-propidium iodide assays.  Biodistribution and in vivo anticancer activity were tested in melanoma tumor-bearing mice.  Porphyrin binding and photocleavage of G-rich mRNA regions were investigated by electrophoresis and RT-PCR.  Porphyrin effect on ERK pathway was explored by Western blots.  Results: Thanks to its higher lipophylicity C14 was taken up by murine melanoma B78-H1 cells up to 30-fold more efficiently than P4.  When photoactivated (7.2 J/cm2) in B78-H1 melanoma cells, P4 and C14, but not control P2, caused a strong inhibition of metabolic activity, clonogenic growth and cell migration.  Biodistribution studies on melanoma tumor-bearing mice showed that P4 and C14 localize in the tumor.  Upon irradn. (660 nm, 193 J/cm2), P4 and C14 retarded tumor growth and increased the median survival time of the treated mice by ∼50% (P <0.01 by ANOVA), whereas porphyrin P2 did not.  The light-dependent mechanism mediated by P4 and C14 is likely due to the binding to and photocleavage of G-rich quadruplex-forming sequences within the 5'-untranslated regions of the mitogenic ras genes.  This causes a decrease of RAS protein and inhibition of downstream ERK pathway, which stimulates proliferation.  Annexin V/propidium iodide and PARP-1 cleavage assays showed that the porphyrins arrested tumor growth by apoptosis and necrosis.  C14 also showed an intrinsic light-independent anticancer activity, as recently reported for G4-RNA binders.  Conclusions: Porphyrins P4 and C14 impair the clonogenic growth and migration of B78-H1 melanoma cells and inhibit melanoma tumor growth in vivo.  Evidence is provided that C14 acts through light-dependent (mRNA photocleavage) and light-independent (translation inhibition) mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCrHhSDBDwYrVg90H21EOLACvtfcHk0lgnwJ_oXie78w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVWks78%253D&md5=45bff23e8e4b99b15b3dabc82dc8833c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-13-75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-13-75%26sid%3Dliteratum%253Aachs%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DZacchigna%26aufirst%3DM.%26aulast%3DDella%2BPietra%26aufirst%3DE.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DAnticancer%2520activity%2520of%2520cationic%2520porphyrins%2520in%2520melanoma%2520tumour-bearing%2520mice%2520and%2520mechanistic%2520in%2520vitro%2520studies%26jtitle%3DMol.%2520Cancer%26date%3D2014%26volume%3D13%26spage%3D75%26epage%3D92%26doi%3D10.1186%2F1476-4598-13-75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">An RNA G-quadruplex in the 5′-UTR of the NRAS proto-oncogene modulates translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1038/nchembio864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchembio864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17322877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=218-221&author=S.+Kumariauthor=A.+Bugautauthor=J.+L.+Huppertauthor=S.+Balasubramanian&title=An+RNA+G-quadruplex+in+the+5%E2%80%B2-UTR+of+the+NRAS+proto-oncogene+modulates+translation&doi=10.1038%2Fnchembio864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA G-quadruplex in the 5' UTR of the NRAS proto-oncogene modulates translation</span></div><div class="casAuthors">Kumari, Sunita; Bugaut, Anthony; Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acid sequences can adopt noncanonical four-stranded secondary structures called guanine (G)-quadruplexes.  Bioinformatics anal. suggests that G-quadruplex motifs are prevalent in genomes, which raises the need to elucidate their function.  There is now evidence for the existence of DNA G-quadruplexes at telomeres with assocd. biol. function.  A recent hypothesis supports the notion that gene promoter elements contain DNA G-quadruplex motifs that control gene expression at the transcriptional level.  We discovered a highly conserved, thermodynamically stable RNA G-quadruplex in the 5' untranslated region (UTR) of the gene transcript of the human NRAS proto-oncogene.  Using a cell-free translation system coupled to a reporter gene assay, we have demonstrated that this NRAS RNA G-quadruplex modulates translation.  This is the first example of translational repression by an RNA G-quadruplex.  Bioinformatics anal. has revealed 2,922 other 5' UTR RNA G-quadruplex elements in the human genome.  We propose that RNA G-quadruplexes in the 5' UTR modulate gene expression at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZXUhKbJ4cLVg90H21EOLACvtfcHk0ljPhx83t9HzKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D&md5=c4dda371d46b95d25b285d1adeb599be</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio864%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DAn%2520RNA%2520G-quadruplex%2520in%2520the%25205%25E2%2580%25B2-UTR%2520of%2520the%2520NRAS%2520proto-oncogene%2520modulates%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnchembio864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">5′-UTR RNA G-quadruplexes: translation regulation and targeting</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4727</span><span class="NLM_x">–</span> <span class="NLM_lpage">4741</span><span class="refDoi"> DOI: 10.1093/nar/gks068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgks068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22351747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=4727-4741&author=A.+Bugautauthor=S.+Balasubramanian&title=5%E2%80%B2-UTR+RNA+G-quadruplexes%3A+translation+regulation+and+targeting&doi=10.1093%2Fnar%2Fgks068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR RNA G-quadruplexes: translation regulation and targeting</span></div><div class="casAuthors">Bugaut, Anthony; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4727-4741</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  RNA structures in the untranslated regions (UTRs) of mRNAs influence post-transcriptional regulation of gene expression.  Much of the knowledge in this area depends on canonical double-stranded RNA elements.  There has been considerable recent advancement of our understanding of guanine(G)-rich nucleic acids sequences that form four-stranded structures, called G-quadruplexes.  While much of the research has been focused on DNA G-quadruplexes, there has recently been a rapid emergence of interest in RNA G-quadruplexes, particularly in the 5'-UTRs of mRNAs.  Collectively, these studies suggest that RNA G-quadruplexes exist in the 5'-UTRs of many genes, including genes of clin. interest, and that such structural elements can influence translation.  This review features the progresses in the study of 5'-UTR RNA G-quadruplex-mediated translational control.  It covers computational anal., cell-free, cell-based and chem. biol. studies that have sought to elucidate the roles of RNA G-quadruplexes in both cap-dependent and -independent regulation of mRNA translation.  We also discuss protein trans-acting factors that have been implicated and the evidence that such RNA motifs have potential as small mol. target.  Finally, we close the review with a perspective on the future challenges in the field of 5'-UTR RNA G-quadruplex-mediated translation regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fxG5DFVSLrVg90H21EOLACvtfcHk0ljPhx83t9HzKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D&md5=d782ebe76a05937797a398ea20b3adf3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks068%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3D5%25E2%2580%25B2-UTR%2520RNA%2520G-quadruplexes%253A%2520translation%2520regulation%2520and%2520targeting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D4727%26epage%3D4741%26doi%3D10.1093%2Fnar%2Fgks068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0ljPhx83t9HzKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Bochman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paeschke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakian, V. A.</span><span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">780</span><span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0ljPhx83t9HzKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Existance and consequences of G-quadruplex structures in DNA</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.gde.2013.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.gde.2013.10.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=22-29&author=P.+Muratauthor=S.+Balasubramanian&title=Existance+and+consequences+of+G-quadruplex+structures+in+DNA&doi=10.1016%2Fj.gde.2013.10.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2013.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2013.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DExistance%2520and%2520consequences%2520of%2520G-quadruplex%2520structures%2520in%2520DNA%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2014%26volume%3D25%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.gde.2013.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Burge, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Quadruplex DNA: sequence, topology and structure</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">5402</span><span class="NLM_x">–</span> <span class="NLM_lpage">5415</span><span class="refDoi"> DOI: 10.1093/nar/gkl655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17012276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=5402-5415&author=S.+Burgeauthor=G.+N.+Parkinsonauthor=P.+Hazelauthor=A.+K.+Toddauthor=S.+Neidle&title=Quadruplex+DNA%3A+sequence%2C+topology+and+structure&doi=10.1093%2Fnar%2Fgkl655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex DNA: sequence, topology and structure</span></div><div class="casAuthors">Burge, Sarah; Parkinson, Gary N.; Hazel, Pascale; Todd, Alan K.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5402-5415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are higher-order DNA and RNA structures formed from G-rich sequences that are built around tetrads of hydrogen-bonded guanine bases.  Potential quadruplex sequences have been identified in G-rich eukaryotic telomeres, and more recently in non-telomeric genomic DNA, e.g. in nuclease-hypersensitive promoter regions.  The natural role and biol. validation of these structures is starting to be explored, and there is particular interest in them as targets for therapeutic intervention.  This survey focuses on the folding and structural features on quadruplexes formed from telomeric and non-telomeric DNA sequences, and examines fundamental aspects of topol. and the emerging relationships with sequence.  Emphasis is placed on information from the high-resoln. methods of x-ray crystallog. and NMR, and their scope and current limitations are discussed.  Such information, together with biol. insights, will be important for the discovery of drugs targeting quadruplexes from particular genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovf3pnf_GPorVg90H21EOLACvtfcHk0ljl5JCT37QFDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM&md5=63387d3c6c1e3567dd8cddd0f0d86c97</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl655%26sid%3Dliteratum%253Aachs%26aulast%3DBurge%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DQuadruplex%2520DNA%253A%2520sequence%252C%2520topology%2520and%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D5402%26epage%3D5415%26doi%3D10.1093%2Fnar%2Fgkl655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0ljl5JCT37QFDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">G4 motifs in human genes</span> <span class="citation_source-journal">Ann. N. Y. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">1267</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1111/j.1749-6632.2012.06586.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1111%2Fj.1749-6632.2012.06586.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1267&publication_year=2012&pages=53-60&author=N.+Maizels&title=G4+motifs+in+human+genes&doi=10.1111%2Fj.1749-6632.2012.06586.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2012.06586.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2012.06586.x%26sid%3Dliteratum%253Aachs%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DG4%2520motifs%2520in%2520human%2520genes%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2012%26volume%3D1267%26spage%3D53%26epage%3D60%26doi%3D10.1111%2Fj.1749-6632.2012.06586.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Mendoza, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourdoncle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulé, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span> </span><span class="NLM_article-title">G-quadruplexes and helicases</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2006</span><span class="refDoi"> DOI: 10.1093/nar/gkw079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkw079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26883636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Sqsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1989-2006&author=O.+Mendozaauthor=A.+Bourdoncleauthor=J.+B.+Boul%C3%A9author=R.+M.+Broshauthor=J.+L.+Mergny&title=G-quadruplexes+and+helicases&doi=10.1093%2Fnar%2Fgkw079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes and helicases</span></div><div class="casAuthors">Mendoza, Oscar; Bourdoncle, Anne; Boule, Jean-Baptiste; Brosh, Robert M., Jr.; Mergny, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1989-2006</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Guanine-rich DNA strands can fold in vitro into non-canonical DNA structures called G-quadruplexes.  These structures may be very stable under physiol. conditions.  Evidence suggests that G-quadruplex structures may act as 'knots' within genomic DNA, and it has been hypothesized that proteins may have evolved to remove these structures.  The first indication of how G-quadruplex structures could be unfolded enzymically came in the late 1990s with reports that some well-known duplex DNA helicases resolved these structures in vitro.  Since then, the no. of studies reporting G-quadruplex DNA unfolding by helicase enzymes has rapidly increased.  The present review aims to present a general overview of the helicase/G-quadruplex field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1DDktyH31aLVg90H21EOLACvtfcHk0lgMpmUPrRVUyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Sqsb7E&md5=0d424783aab817ee14c56597712477ba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw079%26sid%3Dliteratum%253Aachs%26aulast%3DMendoza%26aufirst%3DO.%26aulast%3DBourdoncle%26aufirst%3DA.%26aulast%3DBoul%25C3%25A9%26aufirst%3DJ.%2BB.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DG-quadruplexes%2520and%2520helicases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D1989%26epage%3D2006%26doi%3D10.1093%2Fnar%2Fgkw079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G4 DNA in ras genes and its potential in cancer therapy</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">1859</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2016.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bbagrm.2016.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26855080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Shurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1859&publication_year=2016&pages=663-674&author=S.+Cogoiauthor=L.+E.+Xodo&title=G4+DNA+in+ras+genes+and+its+potential+in+cancer+therapy&doi=10.1016%2Fj.bbagrm.2016.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">G4 DNA in ras genes and its potential in cancer therapy</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1859</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">663-674</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">It is now well established that in the human genome the canonical double helix coexists with folded G-quadruplex structures that are known to have important biol. functions.  In this review we summarize the current knowledge on quadruplex formation in the promoters of the ras genes that are mutated in about 30% of all human cancers.  We describe the nuclear proteins that recognize these unusual DNA structures and discuss their function in transcription.  We also examine the formation of G-quadruplexes in the 5'-untranslated region of the ras transcripts and conclude this review by reporting strategies that use either ras G-quadruplexes or proteins recognizing the ras G-quadruplexes as targets of anticancer small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOg06a0dRrALVg90H21EOLACvtfcHk0lgMpmUPrRVUyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Shurc%253D&md5=a3c0e6fcee18fb40fb5a88bd38901657</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2016.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2016.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG4%2520DNA%2520in%2520ras%2520genes%2520and%2520its%2520potential%2520in%2520cancer%2520therapy%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2016%26volume%3D1859%26spage%3D663%26epage%3D674%26doi%3D10.1016%2Fj.bbagrm.2016.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Hänsel-Hertsch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beraldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lensing, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsico, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zyner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">G-quadruplex structures mark human regulatory chromatin</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1267</span><span class="NLM_x">–</span> <span class="NLM_lpage">1272</span><span class="refDoi"> DOI: 10.1038/ng.3662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fng.3662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27618450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFSgurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=1267-1272&author=R.+H%C3%A4nsel-Hertschauthor=D.+Beraldiauthor=S.+V.+Lensingauthor=G.+Marsicoauthor=K.+Zynerauthor=A.+Parryauthor=M.+Di+Antonioauthor=J.+Pikeauthor=H.+Kimuraauthor=M.+Naritaauthor=D.+Tannahillauthor=S.+Balasubramanian&title=G-quadruplex+structures+mark+human+regulatory+chromatin&doi=10.1038%2Fng.3662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures mark human regulatory chromatin</span></div><div class="casAuthors">Hansel-Hertsch, Robert; Beraldi, Dario; Lensing, Stefanie V.; Marsico, Giovanni; Zyner, Katherine; Parry, Aled; Di Antonio, Marco; Pike, Jeremy; Kimura, Hiroshi; Narita, Masashi; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1267-1272</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplex (G4) structural motifs have been linked to transcription, replication and genome instability and are implicated in cancer and other diseases.  However, it is crucial to demonstrate the bona fide formation of G4 structures within an endogenous chromatin context.  Herein we address this through the development of G4 ChIP-seq, an antibody-based G4 chromatin immunopptn. and high-throughput sequencing approach.  We find ∼10,000 G4 structures in human chromatin, predominantly in regulatory, nucleosome-depleted regions.  G4 structures are enriched in the promoters and 5' UTRs of highly transcribed genes, particularly in genes related to cancer and in somatic copy no. amplifications, such as MYC.  Strikingly, de novo and enhanced G4 formation are assocd. with increased transcriptional activity, as shown by HDAC inhibitor-induced chromatin relaxation and obsd. in immortalized as compared to normal cellular states.  Our findings show that regulatory, nucleosome-depleted chromatin and elevated transcription shape the endogenous human G4 DNA landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYo8vsZd_UZLVg90H21EOLACvtfcHk0lgMpmUPrRVUyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFSgurvP&md5=898d19559a9ccbc0ff468558a653fc43</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fng.3662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3662%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4nsel-Hertsch%26aufirst%3DR.%26aulast%3DBeraldi%26aufirst%3DD.%26aulast%3DLensing%26aufirst%3DS.%2BV.%26aulast%3DMarsico%26aufirst%3DG.%26aulast%3DZyner%26aufirst%3DK.%26aulast%3DParry%26aufirst%3DA.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DPike%26aufirst%3DJ.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DNarita%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DG-quadruplex%2520structures%2520mark%2520human%2520regulatory%2520chromatin%26jtitle%3DNat.%2520Genet.%26date%3D2016%26volume%3D48%26spage%3D1267%26epage%3D1272%26doi%3D10.1038%2Fng.3662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Beaudoin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">5′-UTR G-quadruplex structures acting as translational repressors</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">7022</span><span class="NLM_x">–</span> <span class="NLM_lpage">7036</span><span class="refDoi"> DOI: 10.1093/nar/gkq557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkq557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20571090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVektrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=7022-7036&author=J.+D.+Beaudoinauthor=J.+P.+Perreault&title=5%E2%80%B2-UTR+G-quadruplex+structures+acting+as+translational+repressors&doi=10.1093%2Fnar%2Fgkq557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR G-quadruplex structures acting as translational repressors</span></div><div class="casAuthors">Beaudoin, Jean-Denis; Perreault, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7022-7036</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Given that greater than 90% of the human genome is expressed, it is logical to assume that post-transcriptional regulatory mechanisms must be the primary means of controlling the flow of information from mRNA to protein.  This report describes a robust approach that includes in silico, in vitro and in cellulo expts. permitting an in-depth evaluation of the impact of G-quadruplexes as translational repressors.  Sequences including potential G-quadruplexes were selected within nine distinct genes encoding proteins involved in various biol. processes.  Their abilities to fold into G-quadruplex structures in vitro were evaluated using CD, thermal denaturation and the novel use of in-line probing.  Six sequences were obsd. to fold into G-quadruplex structures in vitro, all of which exhibited translational inhibition in cellulo when linked to a reporter gene.  Sequence anal., direct mutagenesis and subsequent expts. were performed in order to define the rules governing the folding of G-quadruplexes.  In addn., the impact of single-nucleotide polymorphism was shown to be important in the formation of G-quadruplexes located within the 5'-untranslated region of an mRNA.  In light of these results, clearly the 5'-UTR G-quadruplexes represent a class of translational repressors that is broadly distributed in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRYMyH2yIct7Vg90H21EOLACvtfcHk0ljoMUDB-FNMKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVektrzO&md5=eed8abc1182def80967aa26632116d4a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq557%26sid%3Dliteratum%253Aachs%26aulast%3DBeaudoin%26aufirst%3DJ.%2BD.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3D5%25E2%2580%25B2-UTR%2520G-quadruplex%2520structures%2520acting%2520as%2520translational%2520repressors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D7022%26epage%3D7036%26doi%3D10.1093%2Fnar%2Fgkq557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Jodoin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garant, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdi Laaref, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phaneuf, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">The folding of 5′-UTR human G-quadruplexes possessing long central loop</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1141</span><span class="refDoi"> DOI: 10.1261/rna.044578.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1261%2Frna.044578.114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1129-1141&author=R.+Jodoinauthor=L.+Bauerauthor=J.+M.+Garantauthor=A.+Mahdi+Laarefauthor=F.+Phaneufauthor=J.+P.+Perreault&title=The+folding+of+5%E2%80%B2-UTR+human+G-quadruplexes+possessing+long+central+loop&doi=10.1261%2Frna.044578.114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1261%2Frna.044578.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.044578.114%26sid%3Dliteratum%253Aachs%26aulast%3DJodoin%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DL.%26aulast%3DGarant%26aufirst%3DJ.%2BM.%26aulast%3DMahdi%2BLaaref%26aufirst%3DA.%26aulast%3DPhaneuf%26aufirst%3DF.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520folding%2520of%25205%25E2%2580%25B2-UTR%2520human%2520G-quadruplexes%2520possessing%2520long%2520central%2520loop%26jtitle%3DRNA%26date%3D2014%26volume%3D20%26spage%3D1129%26epage%3D1141%26doi%3D10.1261%2Frna.044578.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Hinnebusch, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanov, I. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span> </span><span class="NLM_article-title">Translational control by 5′-untranslated regions of eukaryotic mRNAs</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1416</span><span class="refDoi"> DOI: 10.1126/science.aad9868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1126%2Fscience.aad9868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27313038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslOqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=1413-1416&author=A.+G.+Hinnebuschauthor=I.+P.+Ivanovauthor=N.+Sonenberg&title=Translational+control+by+5%E2%80%B2-untranslated+regions+of+eukaryotic+mRNAs&doi=10.1126%2Fscience.aad9868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Translational control by 5'-untranslated regions of eukaryotic mRNAs</span></div><div class="casAuthors">Hinnebusch, Alan G.; Ivanov, Ivaylo P.; Sonenberg, Nahum</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6292</span>),
    <span class="NLM_cas:pages">1413-1416</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The eukaryotic 5' untranslated region (UTR) is crit. for ribosome recruitment to the mRNA and start codon choice and plays a major role in the control of translation efficiency and shaping the cellular proteome.  The ribosomal initiation complex is assembled on the mRNA via a cap-dependent or cap-independent mechanism.  The authors describe various mechanisms controlling ribosome scanning and initiation codon selection by 5' upstream open reading frames, translation initiation factors, and primary and secondary structures of the 5'UTR, including particular sequence motifs.  The authors also discuss translational control via phosphorylation of eukaryotic initiation factor 2, which is implicated in learning and memory, neurodegenerative diseases, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogiiubG2iPLVg90H21EOLACvtfcHk0ljoMUDB-FNMKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslOqsbY%253D&md5=b50892c1043afb14c936a36a51dc8452</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad9868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad9868%26sid%3Dliteratum%253Aachs%26aulast%3DHinnebusch%26aufirst%3DA.%2BG.%26aulast%3DIvanov%26aufirst%3DI.%2BP.%26aulast%3DSonenberg%26aufirst%3DN.%26atitle%3DTranslational%2520control%2520by%25205%25E2%2580%25B2-untranslated%2520regions%2520of%2520eukaryotic%2520mRNAs%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D1413%26epage%3D1416%26doi%3D10.1126%2Fscience.aad9868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topisirovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">RNA G-quadruplexes and their potential regulatory roles in translation</span> <span class="citation_source-journal">Translation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_elocation-id">e1244031</span><span class="refDoi"> DOI: 10.1080/21690731.2016.1244031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1080%2F21690731.2016.1244031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&author=J.+Songauthor=J.+P.+Perreaultauthor=I.+Topisirovicauthor=S.+Richard&title=RNA+G-quadruplexes+and+their+potential+regulatory+roles+in+translation&doi=10.1080%2F21690731.2016.1244031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1080%2F21690731.2016.1244031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21690731.2016.1244031%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26aulast%3DTopisirovic%26aufirst%3DI.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DRNA%2520G-quadruplexes%2520and%2520their%2520potential%2520regulatory%2520roles%2520in%2520translation%26jtitle%3DTranslation%26date%3D2016%26volume%3D4%26doi%3D10.1080%2F21690731.2016.1244031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Gomez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guédin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salles, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teulade-Fichou, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calsou, P.</span><span> </span><span class="NLM_article-title">A G-quadruplex structure within the 5′-UTR of TRF2 mRNA represses translation in human cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">7187</span><span class="NLM_x">–</span> <span class="NLM_lpage">7198</span><span class="refDoi"> DOI: 10.1093/nar/gkq563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkq563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20571083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVektr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=7187-7198&author=D.+Gomezauthor=A.+Gu%C3%A9dinauthor=J.+L.+Mergnyauthor=B.+Sallesauthor=J.+F.+Riouauthor=M.+P.+Teulade-Fichouauthor=P.+Calsou&title=A+G-quadruplex+structure+within+the+5%E2%80%B2-UTR+of+TRF2+mRNA+represses+translation+in+human+cells&doi=10.1093%2Fnar%2Fgkq563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A G-quadruplex structure within the 5'-UTR of TRF2 mRNA represses translation in human cells</span></div><div class="casAuthors">Gomez, Dennis; Guedin, Aurore; Mergny, Jean-Louis; Salles, Bernard; Riou, Jean-Francois; Teulade-Fichou, Marie-Paule; Calsou, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7187-7198</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Telomeres protect chromosome ends from being recognized as double-stranded breaks.  Telomeric function is ensured by the shelterin complex in which TRF2 protein is an essential player.  The G-rich strand of telomere DNA can fold into G-quadruplex (G4) structure.  Small mols. stabilizing G4 structures, named G4 ligands, have been shown to alter telomeric functions in human cells.  This study shows that a guanine-rich RNA sequence located in the 5'-UTR region of the TRF2 mRNA (hereafter 91TRF2G) is capable of forming a stable quadruplex that causes a 2.8-fold decrease in the translation of a reporter gene in human cells, as compared to a mutant 5'-UTR unable to fold into G4.  Several highly selective G4 ligands, the pyridine dicarboxamide deriv. 360A and bisquinolinium compds. Phen-DC(3) and Phen-DC(6), are able to bind the 91TRF2G:RNA sequence and to modulate TRF2 protein translation in vitro.  Since the naturally occurring 5'-UTR TRF2:RNA G4 element was used here, which is conserved in several vertebrate orthologs, the present data substantiate a potential translational mechanism mediated by a G4 RNA motif for the downregulation of TRF2 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY7NhUZIRXSbVg90H21EOLACvtfcHk0lj3-fJtcc9zzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVektr7I&md5=e72914887bc8a119a521d3b55617833c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq563%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DD.%26aulast%3DGu%25C3%25A9din%26aufirst%3DA.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DSalles%26aufirst%3DB.%26aulast%3DRiou%26aufirst%3DJ.%2BF.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DCalsou%26aufirst%3DP.%26atitle%3DA%2520G-quadruplex%2520structure%2520within%2520the%25205%25E2%2580%25B2-UTR%2520of%2520TRF2%2520mRNA%2520represses%2520translation%2520in%2520human%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D7187%26epage%3D7198%26doi%3D10.1093%2Fnar%2Fgkq563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Kwok, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Assmann, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bevilacqua, P. C.</span><span> </span><span class="NLM_article-title">A stable RNA G-quadruplex within the 5′-UTR of Arabidopsis thaliana ATR mRNA inhibits translation</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1042/BJ20141063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1042%2FBJ20141063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=91-102&author=C.+K.+Kwokauthor=Y.+Dingauthor=S.+Shahidauthor=S.+M.+Assmannauthor=P.+C.+Bevilacqua&title=A+stable+RNA+G-quadruplex+within+the+5%E2%80%B2-UTR+of+Arabidopsis+thaliana+ATR+mRNA+inhibits+translation&doi=10.1042%2FBJ20141063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1042%2FBJ20141063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141063%26sid%3Dliteratum%253Aachs%26aulast%3DKwok%26aufirst%3DC.%2BK.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DShahid%26aufirst%3DS.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26atitle%3DA%2520stable%2520RNA%2520G-quadruplex%2520within%2520the%25205%25E2%2580%25B2-UTR%2520of%2520Arabidopsis%2520thaliana%2520ATR%2520mRNA%2520inhibits%2520translation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D91%26epage%3D102%26doi%3D10.1042%2FBJ20141063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Arora, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutkiewicz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaria, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hariharan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurreck, J.</span><span> </span><span class="NLM_article-title">Inhibition of translation in eukaryotic cells by an RNA G-quadruplex motif</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1290</span><span class="NLM_x">–</span> <span class="NLM_lpage">1296</span><span class="refDoi"> DOI: 10.1261/rna.1001708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1261%2Frna.1001708" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=1290-1296&author=A.+Aroraauthor=M.+Dutkiewiczauthor=V.+Scariaauthor=M.+Hariharanauthor=S.+Maitiauthor=J.+Kurreck&title=Inhibition+of+translation+in+eukaryotic+cells+by+an+RNA+G-quadruplex+motif&doi=10.1261%2Frna.1001708"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1261%2Frna.1001708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.1001708%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DDutkiewicz%26aufirst%3DM.%26aulast%3DScaria%26aufirst%3DV.%26aulast%3DHariharan%26aufirst%3DM.%26aulast%3DMaiti%26aufirst%3DS.%26aulast%3DKurreck%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520translation%2520in%2520eukaryotic%2520cells%2520by%2520an%2520RNA%2520G-quadruplex%2520motif%26jtitle%3DRNA%26date%3D2008%26volume%3D14%26spage%3D1290%26epage%3D1296%26doi%3D10.1261%2Frna.1001708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Position and stability are determining factors for translation repression by an RNA G-quaduplex forming sequence within 5′-UTR of the NRAS proto oncogene</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">12664</span><span class="NLM_x">–</span> <span class="NLM_lpage">12669</span><span class="refDoi"> DOI: 10.1021/bi8010797</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8010797" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=12664-12669&author=S.+Kumariauthor=A.+Bugautauthor=S.+Balasubramanian&title=Position+and+stability+are+determining+factors+for+translation+repression+by+an+RNA+G-quaduplex+forming+sequence+within+5%E2%80%B2-UTR+of+the+NRAS+proto+oncogene&doi=10.1021%2Fbi8010797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi8010797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8010797%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DPosition%2520and%2520stability%2520are%2520determining%2520factors%2520for%2520translation%2520repression%2520by%2520an%2520RNA%2520G-quaduplex%2520forming%2520sequence%2520within%25205%25E2%2580%25B2-UTR%2520of%2520the%2520NRAS%2520proto%2520oncogene%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D12664%26epage%3D12669%26doi%3D10.1021%2Fbi8010797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Morris, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Negishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pazsint, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonhoft, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span> </span><span class="NLM_article-title">A RNA G-quadruplex is essential for cap-independent translation initiation in human VEGF IRES</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">17831</span><span class="NLM_x">–</span> <span class="NLM_lpage">17839</span><span class="refDoi"> DOI: 10.1021/ja106287x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja106287x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=17831-17839&author=M.+J.+Morrisauthor=Y.+Negishiauthor=C.+Pazsintauthor=J.+D.+Schonhoftauthor=S.+Basu&title=A+RNA+G-quadruplex+is+essential+for+cap-independent+translation+initiation+in+human+VEGF+IRES&doi=10.1021%2Fja106287x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja106287x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja106287x%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DM.%2BJ.%26aulast%3DNegishi%26aufirst%3DY.%26aulast%3DPazsint%26aufirst%3DC.%26aulast%3DSchonhoft%26aufirst%3DJ.%2BD.%26aulast%3DBasu%26aufirst%3DS.%26atitle%3DA%2520RNA%2520G-quadruplex%2520is%2520essential%2520for%2520cap-independent%2520translation%2520initiation%2520in%2520human%2520VEGF%2520IRES%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D17831%26epage%3D17839%26doi%3D10.1021%2Fja106287x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Lammich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamp, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuscher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zilow, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haass, C.</span><span> </span><span class="NLM_article-title">Translational repression of the disintegrin and metallo protease ADAM10 by a stable G-quadruplex secondary structure in its 5′-untrasnlated region</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">45063</span><span class="NLM_x">–</span> <span class="NLM_lpage">45072</span><span class="refDoi"> DOI: 10.1074/jbc.M111.296921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1074%2Fjbc.M111.296921" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=45063-45072&author=S.+Lammichauthor=F.+Kampauthor=J.+Wagnerauthor=B.+Nuscherauthor=S.+Zilowauthor=A.+K.+Ludwigauthor=M.+Willemauthor=C.+Haass&title=Translational+repression+of+the+disintegrin+and+metallo+protease+ADAM10+by+a+stable+G-quadruplex+secondary+structure+in+its+5%E2%80%B2-untrasnlated+region&doi=10.1074%2Fjbc.M111.296921"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.296921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.296921%26sid%3Dliteratum%253Aachs%26aulast%3DLammich%26aufirst%3DS.%26aulast%3DKamp%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DJ.%26aulast%3DNuscher%26aufirst%3DB.%26aulast%3DZilow%26aufirst%3DS.%26aulast%3DLudwig%26aufirst%3DA.%2BK.%26aulast%3DWillem%26aufirst%3DM.%26aulast%3DHaass%26aufirst%3DC.%26atitle%3DTranslational%2520repression%2520of%2520the%2520disintegrin%2520and%2520metallo%2520protease%2520ADAM10%2520by%2520a%2520stable%2520G-quadruplex%2520secondary%2520structure%2520in%2520its%25205%25E2%2580%25B2-untrasnlated%2520region%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D45063%26epage%3D45072%26doi%3D10.1074%2Fjbc.M111.296921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Bolduc, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garant, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">Irregular G-quadruplexes found in untranslated regions of human mRNAs influences translation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">21751</span><span class="NLM_x">–</span> <span class="NLM_lpage">21750</span><span class="refDoi"> DOI: 10.1074/jbc.M116.744839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1074%2Fjbc.M116.744839" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=21751-21750&author=F.+Bolducauthor=J.+M.+Garantauthor=F.+Allardauthor=J.+P.+Perreault&title=Irregular+G-quadruplexes+found+in+untranslated+regions+of+human+mRNAs+influences+translation&doi=10.1074%2Fjbc.M116.744839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.744839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.744839%26sid%3Dliteratum%253Aachs%26aulast%3DBolduc%26aufirst%3DF.%26aulast%3DGarant%26aufirst%3DJ.%2BM.%26aulast%3DAllard%26aufirst%3DF.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3DIrregular%2520G-quadruplexes%2520found%2520in%2520untranslated%2520regions%2520of%2520human%2520mRNAs%2520influences%2520translation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D21751%26epage%3D21750%26doi%3D10.1074%2Fjbc.M116.744839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Zuker, M.</span><span> </span><span class="NLM_article-title">Mfold web server for nucleic acid folding and hybridization prediction</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3406</span><span class="NLM_x">–</span> <span class="NLM_lpage">3415</span><span class="refDoi"> DOI: 10.1093/nar/gkg595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkg595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=12824337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVWisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=3406-3415&author=M.+Zuker&title=Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction&doi=10.1093%2Fnar%2Fgkg595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mfold web server for nucleic acid folding and hybridization prediction</span></div><div class="casAuthors">Zuker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3406-3415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The abbreviated name, ' mfold web server', describes a no. of closely related software applications available on the World Wide Web (WWW) for the prediction of the secondary structure of single stranded nucleic acids.  The objective of this web server is to provide easy access to RNA and DNA folding and hybridization software to the scientific community at large.  By making use of universally available web GUIs (Graphical User Interfaces), the server circumvents the problem of portability of this software.  Detailed output, in the form of structure plots with or without reliability information, single strand frequency plots and energy dot plots', are available for the folding of single sequences.  A variety of bulk' servers give less information, but in a shorter time and for up to hundreds of sequences at once.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4cF1dfOCOjLVg90H21EOLACvtfcHk0lgQpyvOALH8gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVWisr8%253D&md5=4f49b0c2add0eedbcc41a1f058e285bb</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg595%26sid%3Dliteratum%253Aachs%26aulast%3DZuker%26aufirst%3DM.%26atitle%3DMfold%2520web%2520server%2520for%2520nucleic%2520acid%2520folding%2520and%2520hybridization%2520prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26spage%3D3406%26epage%3D3415%26doi%3D10.1093%2Fnar%2Fgkg595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Kikin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Antonio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagga, P. S.</span><span> </span><span class="NLM_article-title">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">W676</span><span class="NLM_x">–</span> <span class="NLM_lpage">W682</span><span class="refDoi"> DOI: 10.1093/nar/gkl253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16845096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W676-W682&author=O.+Kikinauthor=L.+D%E2%80%99Antonioauthor=P.+S.+Bagga&title=QGRS+Mapper%3A+a+web-based+server+for+predicting+G-quadruplexes+in+nucleotide+sequences&doi=10.1093%2Fnar%2Fgkl253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span></div><div class="casAuthors">Kikin, Oleg; D'Antonio, Lawrence; Bagga, Paramjeet S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W676-W682</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biol. processes and as therapeutic targets.  G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation.  Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts.  Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation.  Computational approaches to study G-quadruplexes allow detailed anal. of mammalian genomes.  There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences.  We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences.  It is a user-friendly application that provides many options for defining and studying G-quadruplexes.  It performs anal. of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences.  It is also useful for predicting G-quadruplex structures in oligonucleotides.  The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites.  This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing.  In addn. to providing data on compn. and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data.  The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yCz4ebPV-bVg90H21EOLACvtfcHk0lgQpyvOALH8gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D&md5=e8f69845c3df9217df7f346a06c3db44</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl253%26sid%3Dliteratum%253Aachs%26aulast%3DKikin%26aufirst%3DO.%26aulast%3DD%25E2%2580%2599Antonio%26aufirst%3DL.%26aulast%3DBagga%26aufirst%3DP.%2BS.%26atitle%3DQGRS%2520Mapper%253A%2520a%2520web-based%2520server%2520for%2520predicting%2520G-quadruplexes%2520in%2520nucleotide%2520sequences%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW676%26epage%3DW682%26doi%3D10.1093%2Fnar%2Fgkl253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Vorlickova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kejnovska, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednarova, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renciuk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kypr, J.</span><span> </span><span class="NLM_article-title">Circular dichroism spectroscopy of DNA: from duplexes to quadruplexes</span> <span class="citation_source-journal">Chirality</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1002/chir.22064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1002%2Fchir.22064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22696273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVeiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=691-698&author=M.+Vorlickovaauthor=I.+Kejnovskaauthor=K.+Bednarovaauthor=D.+Renciukauthor=J.+Kypr&title=Circular+dichroism+spectroscopy+of+DNA%3A+from+duplexes+to+quadruplexes&doi=10.1002%2Fchir.22064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Circular Dichroism Spectroscopy of DNA: From Duplexes to Quadruplexes</span></div><div class="casAuthors">Vorlickova, Michaela; Kejnovska, Iva; Bednarova, Klara; Renciuk, Daniel; Kypr, Jaroslav</div><div class="citationInfo"><span class="NLM_cas:title">Chirality</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">691-698</span>CODEN:
                <span class="NLM_cas:coden">CHRLEP</span>;
        ISSN:<span class="NLM_cas:issn">0899-0042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleic acids bear the genetic information and participate in its expression and evolution during replication, repair, recombination, transcription, and translation.  These phenomena are mostly based on recognition of nucleic acids by proteins.  The major factor enabling the specific recognition is structure.  CD spectroscopy is very useful to study secondary structures of nucleic acids, in general, and DNA, in particular.  CD sensitively reflects isomerizations among distinct conformational states.  The isomerizations may operate as mol. switches regulating various physiol. or pathol. processes.  Here, we review CD spectra of nucleic acids, beginning with early studies on natural DNA mols. through analyses of synthetic polynucleotides to study of selected genomic fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIb90iB7srw7Vg90H21EOLACvtfcHk0lgQpyvOALH8gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVeiurc%253D&md5=06d157cb05249eae6a6ba8c6f5f3e3be</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fchir.22064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchir.22064%26sid%3Dliteratum%253Aachs%26aulast%3DVorlickova%26aufirst%3DM.%26aulast%3DKejnovska%26aufirst%3DI.%26aulast%3DBednarova%26aufirst%3DK.%26aulast%3DRenciuk%26aufirst%3DD.%26aulast%3DKypr%26aufirst%3DJ.%26atitle%3DCircular%2520dichroism%2520spectroscopy%2520of%2520DNA%253A%2520from%2520duplexes%2520to%2520quadruplexes%26jtitle%3DChirality%26date%3D2012%26volume%3D24%26spage%3D691%26epage%3D698%26doi%3D10.1002%2Fchir.22064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Mergny, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaires, J. B.</span><span> </span><span class="NLM_article-title">Thermal difference spectra: a specific signature for nucleic acid structures</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_elocation-id">e138</span><span class="refDoi"> DOI: 10.1093/nar/gni134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgni134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16157860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&author=J.+L.+Mergnyauthor=J.+Liauthor=L.+Lacroixauthor=S.+Amraneauthor=J.+B.+Chaires&title=Thermal+difference+spectra%3A+a+specific+signature+for+nucleic+acid+structures&doi=10.1093%2Fnar%2Fgni134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal difference spectra: a specific signature for nucleic acid structures</span></div><div class="casAuthors">Mergny, Jean-Louis; Li, Jing; Lacroix, Laurent; Amrane, Samir; Chaires, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">e138/1-e138/6</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The authors show that nucleic acid structures may be conveniently and inexpensively characterized by their UV thermal difference spectra.  A thermal difference spectrum (TDS) is obtained for a nucleic acid by simply recording the UV absorbance spectra of the unfolded and folded states at temps. above and below its melting temp. (Tm).  The difference between these two spectra is the TDS.  The TDS has a specific shape that is unique for each type of nucleic acid structure, a conclusion that is based on a comparison of >900 spectra from 200 different sequences.  The shape of the TDS reflects the subtleties of base stacking interactions that occur uniquely within each type of nucleic acid structure.  TDS provides a simple, inexpensive and rapid method to obtain structural insight into nucleic acid structures, which is applicable to both DNA and RNA from short oligomers to polynucleotides.  TDS complements CD as a tool for the structural characterization of nucleic acids in soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZDiohxvD4qbVg90H21EOLACvtfcHk0ljyrCV3kP6YNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsrrF&md5=d8ed2fd2a0bbe38e4a04dfc19c999c17</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgni134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgni134%26sid%3Dliteratum%253Aachs%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26atitle%3DThermal%2520difference%2520spectra%253A%2520a%2520specific%2520signature%2520for%2520nucleic%2520acid%2520structures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26doi%3D10.1093%2Fnar%2Fgni134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Yangyuoru, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghimire, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, H.</span><span> </span><span class="NLM_article-title">Dual binding of an antibody and a small molecule increases the stability of TERRA G-quadruplex</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">910</span><span class="NLM_x">–</span> <span class="NLM_lpage">913</span><span class="refDoi"> DOI: 10.1002/anie.201408113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1002%2Fanie.201408113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=910-913&author=P.+M.+Yangyuoruauthor=M.+Di+Antonioauthor=C.+Ghimireauthor=G.+Biffiauthor=S.+Balasubramanianauthor=H.+Mao&title=Dual+binding+of+an+antibody+and+a+small+molecule+increases+the+stability+of+TERRA+G-quadruplex&doi=10.1002%2Fanie.201408113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fanie.201408113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201408113%26sid%3Dliteratum%253Aachs%26aulast%3DYangyuoru%26aufirst%3DP.%2BM.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DGhimire%26aufirst%3DC.%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DH.%26atitle%3DDual%2520binding%2520of%2520an%2520antibody%2520and%2520a%2520small%2520molecule%2520increases%2520the%2520stability%2520of%2520TERRA%2520G-quadruplex%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D910%26epage%3D913%26doi%3D10.1002%2Fanie.201408113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0ljyrCV3kP6YNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Miserachs, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johannsen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigel, R. K. O.</span><span> </span><span class="NLM_article-title">Distinct differences in metal ion specificity of RNA and DNA G-quadruplexes</span> <span class="citation_source-journal">JBIC, J. Biol. Inorg. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span><span class="refDoi"> DOI: 10.1007/s00775-016-1393-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs00775-016-1393-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27704222" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=975-986&author=H.+G.+Miserachsauthor=D.+Donghiauthor=R.+Bornerauthor=S.+Johannsenauthor=R.+K.+O.+Sigel&title=Distinct+differences+in+metal+ion+specificity+of+RNA+and+DNA+G-quadruplexes&doi=10.1007%2Fs00775-016-1393-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs00775-016-1393-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00775-016-1393-4%26sid%3Dliteratum%253Aachs%26aulast%3DMiserachs%26aufirst%3DH.%2BG.%26aulast%3DDonghi%26aufirst%3DD.%26aulast%3DBorner%26aufirst%3DR.%26aulast%3DJohannsen%26aufirst%3DS.%26aulast%3DSigel%26aufirst%3DR.%2BK.%2BO.%26atitle%3DDistinct%2520differences%2520in%2520metal%2520ion%2520specificity%2520of%2520RNA%2520and%2520DNA%2520G-quadruplexes%26jtitle%3DJBIC%252C%2520J.%2520Biol.%2520Inorg.%2520Chem.%26date%3D2016%26volume%3D21%26spage%3D975%26epage%3D986%26doi%3D10.1007%2Fs00775-016-1393-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">McPike, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodisman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dabrowiak, J. C.</span><span> </span><span class="NLM_article-title">Drug-RNA footprinting</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">449</span><span class="refDoi"> DOI: 10.1016/S0076-6879(01)40435-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FS0076-6879%2801%2940435-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2001&pages=431-449&author=M.+P.+McPikeauthor=J.+Goodismanauthor=J.+C.+Dabrowiak&title=Drug-RNA+footprinting&doi=10.1016%2FS0076-6879%2801%2940435-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2801%2940435-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252801%252940435-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcPike%26aufirst%3DM.%2BP.%26aulast%3DGoodisman%26aufirst%3DJ.%26aulast%3DDabrowiak%26aufirst%3DJ.%2BC.%26atitle%3DDrug-RNA%2520footprinting%26jtitle%3DMethods%2520Enzymol.%26date%3D2001%26volume%3D340%26spage%3D431%26epage%3D449%26doi%3D10.1016%2FS0076-6879%2801%2940435-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title">Thioflavin T as an efficient fluorescence sensor for selective recognition of RNA G-quadruplexes</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">24793</span><span class="refDoi"> DOI: 10.1038/srep24793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fsrep24793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27098781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Kntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=24793&author=S.+Xuauthor=Q.+Liauthor=J.+Xiangauthor=Q.+Yangauthor=H.+Sunauthor=A.+Guanauthor=L.+Wangauthor=Y.+Liuauthor=L.+Yuauthor=Y.+Shiauthor=H.+Chenauthor=Y.+Tang&title=Thioflavin+T+as+an+efficient+fluorescence+sensor+for+selective+recognition+of+RNA+G-quadruplexes&doi=10.1038%2Fsrep24793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Thioflavin T as an efficient fluorescence sensor for selective recognition of RNA G-quadruplexes</span></div><div class="casAuthors">Xu, Shujuan; Li, Qian; Xiang, Junfeng; Yang, Qianfan; Sun, Hongxia; Guan, Aijiao; Wang, Lixia; Liu, Yan; Yu, Lijia; Shi, Yunhua; Chen, Hongbo; Tang, Yalin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24793</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplexes (G4s) play important roles in translational regulation, mRNA processing events and gene expression.  Therefore, a fluorescent probe that is capable of efficiently recognizing RNA G-quadruplex structures among other RNA forms is highly desirable.  In this study, a water-sol. fluorogenic dye (i.e., Thioflavin T (ThT)) was employed to recognize RNA G-quadruplex structures using UV-Vis absorption spectra, fluorescence spectra and emission lifetime expts.  By stacking on the G-tetrad, the ThT probe exhibited highly specific recognition of RNA G-quadruplex structures with striking fluorescence enhancement compared with other RNA forms.  The specific binding demonstrates that ThT is an efficient fluorescence sensor that can distinguish G4 and non-G4 RNA structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpil8g_rgoR2LVg90H21EOLACvtfcHk0ljMzORLZWHsew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Kntr8%253D&md5=3ae08487d53162f02c83eb03b19e5b8d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fsrep24793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep24793%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DThioflavin%2520T%2520as%2520an%2520efficient%2520fluorescence%2520sensor%2520for%2520selective%2520recognition%2520of%2520RNA%2520G-quadruplexes%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D24793%26doi%3D10.1038%2Fsrep24793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Weldon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behm-Ansmant, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branlant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eperon, I. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, C.</span><span> </span><span class="NLM_article-title">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1038/nchembio.2228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchembio.2228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27820800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=18-20&author=C.+Weldonauthor=I.+Behm-Ansmantauthor=L.+H.+Hurleyauthor=G.+A.+Burleyauthor=C.+Branlantauthor=I.+C.+Eperonauthor=C.+Dominguez&title=Identification+of+G-quadruplexes+in+long+functional+RNAs+using+7-deazaguanine+RNA&doi=10.1038%2Fnchembio.2228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span></div><div class="casAuthors">Weldon, Carika; Behm-Ansmant, Isabelle; Hurley, Laurence H.; Burley, Glenn A.; Branlant, Christiane; Eperon, Ian C.; Dominguez, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-20</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplex (G4) structures are thought to affect biol. processes, including translation and pre-mRNA splicing, but it is not possible at present to demonstrate that they form naturally at specific sequences in long functional RNA mols.  We developed a new strategy, footprinting of long 7-deazaguanine-substituted RNAs (FOLDeR), that allows the formation of G4s to be confirmed in long RNAs and under functional conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEnw-nGuf4ZLVg90H21EOLACvtfcHk0ljMzORLZWHsew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN&md5=b1f547bacfc316a5544d6901bab73533</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2228%26sid%3Dliteratum%253Aachs%26aulast%3DWeldon%26aufirst%3DC.%26aulast%3DBehm-Ansmant%26aufirst%3DI.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBurley%26aufirst%3DG.%2BA.%26aulast%3DBranlant%26aufirst%3DC.%26aulast%3DEperon%26aufirst%3DI.%2BC.%26aulast%3DDominguez%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520G-quadruplexes%2520in%2520long%2520functional%2520RNAs%2520using%25207-deazaguanine%2520RNA%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D18%26epage%3D20%26doi%3D10.1038%2Fnchembio.2228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Small molecule-mediated inhibition of translation by targeting a native RNA G-quadruplex</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2771</span><span class="NLM_x">–</span> <span class="NLM_lpage">2776</span><span class="refDoi"> DOI: 10.1039/c002418j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1039%2Fc002418j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20436976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=2771-2776&author=A.+Bugautauthor=R.+Rodriguezauthor=S.+Kumariauthor=S.+T.+Hsuauthor=S.+Balasubramanian&title=Small+molecule-mediated+inhibition+of+translation+by+targeting+a+native+RNA+G-quadruplex&doi=10.1039%2Fc002418j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule-mediated inhibition of translation by targeting a native RNA G-quadruplex</span></div><div class="casAuthors">Bugaut, Anthony; Rodriguez, Raphael; Kumari, Sunita; Hsu, Shang-Te Danny; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2771-2776</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein, we show that a naturally occurring RNA G-quadruplex element within the 5' UTR of the human NRAS proto-oncogene is a target for a small mol. that inhibits translation in vitro.  The present study provides a first demonstration that natural 5' UTR mRNA G-quadruplexes have potential as mol. targets for small mols. that modulate translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppcJ1Ja-L2T7Vg90H21EOLACvtfcHk0ljMzORLZWHsew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWktL0%253D&md5=6bb9cf5b26189a75207c0721662082e2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2Fc002418j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc002418j%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DHsu%26aufirst%3DS.%2BT.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DSmall%2520molecule-mediated%2520inhibition%2520of%2520translation%2520by%2520targeting%2520a%2520native%2520RNA%2520G-quadruplex%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2010%26volume%3D8%26spage%3D2771%26epage%3D2776%26doi%3D10.1039%2Fc002418j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Katsuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furuta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagihara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesugi, M.</span><span> </span><span class="NLM_article-title">A small molecule that represses translation of G-quadruplex-containing mRNA</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">9037</span><span class="NLM_x">–</span> <span class="NLM_lpage">9040</span><span class="refDoi"> DOI: 10.1021/jacs.6b04506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=9037-9040&author=Y.+Katsudaauthor=S.+Satoauthor=L.+Asanoauthor=Y.+Morimuraauthor=T.+Furutaauthor=H.+Sugiyamaauthor=M.+Hagiharaauthor=M.+Uesugi&title=A+small+molecule+that+represses+translation+of+G-quadruplex-containing+mRNA&doi=10.1021%2Fjacs.6b04506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule that represses translation of G-quadruplex-containing mRNA</span></div><div class="casAuthors">Katsuda, Yousuke; Sato, Shin-ichi; Asano, Lisa; Morimura, Yoshitaka; Furuta, Tomoyuki; Sugiyama, Hiroshi; Hagihara, Masaki; Uesugi, Motonari</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9037-9040</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G-quadruplexes form highly stable nucleic acid structures, which are implicated in various biol. processes in both DNA and RNA.  Although DNA G-quadruplexes have been studied in great detail, biol. roles of RNA G-quadruplexes have received less attention.  Here, a screening of a chem. library permitted identification of a small-mol. tool that binds selectively to RNA G-quadruplex structures.  The polyarom. mol., RGB-1, stabilizes RNA G-quadruplex, but not DNA versions or other RNA structures.  RGB-1 intensified the G-quadruplex-mediated inhibition of RNA translation in mammalian cells, decreased expression of the NRAS proto-oncogene in breast cancer cells, and permitted identification of a novel sequence that forms G-quadruplex in NRAS mRNA.  RGB-1 may serve as a unique tool for understanding cellular roles of RNA G-quadruplex structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-OEuiG5IuLVg90H21EOLACvtfcHk0liGtnmpOpR7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP&md5=75ad58de109aa50cc964db46bc46f69b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04506%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DAsano%26aufirst%3DL.%26aulast%3DMorimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DHagihara%26aufirst%3DM.%26aulast%3DUesugi%26aufirst%3DM.%26atitle%3DA%2520small%2520molecule%2520that%2520represses%2520translation%2520of%2520G-quadruplex-containing%2520mRNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D9037%26epage%3D9040%26doi%3D10.1021%2Fjacs.6b04506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Membrino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinkevich, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Guanidino anthrathiophenediones as G-quadruplex binders: uptake, intracellular localization, and anti-Harvey-Ras gene activity in bladder cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2764</span><span class="NLM_x">–</span> <span class="NLM_lpage">2778</span><span class="refDoi"> DOI: 10.1021/jm3019063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3019063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2764-2778&author=S.+Cogoiauthor=A.+E.+Shchekotikhinauthor=A.+Membrinoauthor=Y.+B.+Sinkevichauthor=L.+E.+Xodo&title=Guanidino+anthrathiophenediones+as+G-quadruplex+binders%3A+uptake%2C+intracellular+localization%2C+and+anti-Harvey-Ras+gene+activity+in+bladder+cancer+cells&doi=10.1021%2Fjm3019063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm3019063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3019063%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DMembrino%26aufirst%3DA.%26aulast%3DSinkevich%26aufirst%3DY.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DGuanidino%2520anthrathiophenediones%2520as%2520G-quadruplex%2520binders%253A%2520uptake%252C%2520intracellular%2520localization%252C%2520and%2520anti-Harvey-Ras%2520gene%2520activity%2520in%2520bladder%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2764%26epage%3D2778%26doi%3D10.1021%2Fjm3019063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Potent apoptotic response induced by chloroacetamidine anthrathiophenediones in bladder cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5476</span><span class="NLM_x">–</span> <span class="NLM_lpage">5485</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00409</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00409" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5476-5485&author=S.+Cogoiauthor=S.+Zorzetauthor=A.+E.+Shchekotikhinauthor=L.+E.+Xodo&title=Potent+apoptotic+response+induced+by+chloroacetamidine+anthrathiophenediones+in+bladder+cancer+cells&doi=10.1021%2Facs.jmedchem.5b00409"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00409%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DPotent%2520apoptotic%2520response%2520induced%2520by%2520chloroacetamidine%2520anthrathiophenediones%2520in%2520bladder%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5476%26epage%3D5485%26doi%3D10.1021%2Facs.jmedchem.5b00409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glazunova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dezhenkova, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzikov, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinkevich, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovalenko, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buyanov, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shtil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">Synthesis and cytotoxic properties of 4,11-bis[(aminoethyl)amino]anthra[2,3-<i>b</i>]thiophene-5,10-diones, novel analogues of antitumor anthracene-9,10-diones</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1861</span><span class="NLM_x">–</span> <span class="NLM_lpage">1869</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.01.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bmc.2009.01.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1861-1869&author=A.+E.+Shchekotikhinauthor=V.+A.+Glazunovaauthor=L.+G.+Dezhenkovaauthor=Y.+N.+Luzikovauthor=Y.+B.+Sinkevichauthor=L.+V.+Kovalenkoauthor=V.+N.+Buyanovauthor=J.+Balzariniauthor=A.+A.+Shtilauthor=M.+N.+Preobrazhenskaya&title=Synthesis+and+cytotoxic+properties+of+4%2C11-bis%5B%28aminoethyl%29amino%5Danthra%5B2%2C3-b%5Dthiophene-5%2C10-diones%2C+novel+analogues+of+antitumor+anthracene-9%2C10-diones&doi=10.1016%2Fj.bmc.2009.01.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.01.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.01.047%26sid%3Dliteratum%253Aachs%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DGlazunova%26aufirst%3DV.%2BA.%26aulast%3DDezhenkova%26aufirst%3DL.%2BG.%26aulast%3DLuzikov%26aufirst%3DY.%2BN.%26aulast%3DSinkevich%26aufirst%3DY.%2BB.%26aulast%3DKovalenko%26aufirst%3DL.%2BV.%26aulast%3DBuyanov%26aufirst%3DV.%2BN.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DShtil%26aufirst%3DA.%2BA.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DSynthesis%2520and%2520cytotoxic%2520properties%2520of%25204%252C11-bis%255B%2528aminoethyl%2529amino%255Danthra%255B2%252C3-b%255Dthiophene-5%252C10-diones%252C%2520novel%2520analogues%2520of%2520antitumor%2520anthracene-9%252C10-diones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D1861%26epage%3D1869%26doi%3D10.1016%2Fj.bmc.2009.01.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span> </span><span class="NLM_article-title">Methods for the synthesis and modification of linear anthrafurandiones</span> <span class="citation_source-journal">Chem. Heterocycl. Compd.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1007/s10593-014-1459-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs10593-014-1459-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=171-184&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=M.+N.+Preobrazhenskaya&title=Methods+for+the+synthesis+and+modification+of+linear+anthrafurandiones&doi=10.1007%2Fs10593-014-1459-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for the Synthesis and Modification of Linear Anthrafurandiones (Review)</span></div><div class="casAuthors">Tikhomirov, A. S.; Shchekotikhin, A. E.; Preobrazhenskaya, M. N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-184</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Derivs. of anthra[2,3-b]furan-5,10-diones hold promise for various fields of science and technol., esp. in the search for biol. active compds.  This review covers all currently known methods for the cyclization of linear anthrafurandiones, the first examples of which were synthesized approx. 30 years ago.  The material has been arranged mainly in the order of increasing degree of substitution in anthrafurandiones.  All the literature methods for functional group transformations providing access to a range of anthrafurandiones unavailable by direct heterocyclization have been considered sep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE67DaXvt5zLVg90H21EOLACvtfcHk0liGtnmpOpR7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCktb4%253D&md5=d10d1fd2d3ed4f3e934d956386d14a29</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10593-014-1459-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-014-1459-6%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DMethods%2520for%2520the%2520synthesis%2520and%2520modification%2520of%2520linear%2520anthrafurandiones%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2014%26volume%3D50%26spage%3D171%26epage%3D184%26doi%3D10.1007%2Fs10593-014-1459-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Gronowitz, S.</span><span> </span><span class="NLM_article-title">Thiophene and Its Derivatives</span>, Part 2.  <span class="citation_source-book">The Chemistry of Heterocyclic Compounds</span>; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1986</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">44</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1986&author=S.+Gronowitz&title=The+Chemistry+of+Heterocyclic+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGronowitz%26aufirst%3DS.%26atitle%3DThiophene%2520and%2520Its%2520Derivatives%26btitle%3DThe%2520Chemistry%2520of%2520Heterocyclic%2520Compounds%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1986%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatarskiy, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dezhenkova, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glazunova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shtil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chueh, P. J.</span><span> </span><span class="NLM_article-title">Synthesis and characterization of 4,11-diaminoanthra[2,3-<i>b</i>]furan-5,10-diones: tumor cell apoptosis through tNOX-modulated NAD<sup>+</sup>/NADH ratio and SIRT1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9522</span><span class="NLM_x">–</span> <span class="NLM_lpage">9534</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00859</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00859" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9522-9534&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=Y.+H.+Leeauthor=Y.+A.+Chenauthor=C.+A.+Yehauthor=V.+V.+Tatarskiyauthor=L.+G.+Dezhenkovaauthor=V.+A.+Glazunovaauthor=J.+Balzariniauthor=A.+A.+Shtilauthor=M.+N.+Preobrazhenskayaauthor=P.+J.+Chueh&title=Synthesis+and+characterization+of+4%2C11-diaminoanthra%5B2%2C3-b%5Dfuran-5%2C10-diones%3A+tumor+cell+apoptosis+through+tNOX-modulated+NAD%2B%2FNADH+ratio+and+SIRT1&doi=10.1021%2Facs.jmedchem.5b00859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Characterization of 4,11-Diaminoanthra[2,3-b]furan-5,10-diones: Tumor Cell Apoptosis through tNOX-Modulated NAD+/NADH Ratio and SIRT1</span></div><div class="casAuthors">Tikhomirov, Alexander S.; Shchekotikhin, Andrey E.; Lee, Yi-Hui; Chen, Yi-Ann; Yeh, Chia-An; Tatarskiy, Victor V.; Dezhenkova, Lyubov G.; Glazunova, Valeria A.; Balzarini, Jan; Shtil, Alexander A.; Preobrazhenskaya, Maria N.; Chueh, Pin Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9522-9534</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 4,11-diaminoanthra[2,3-b]furan-5,10-dione derivs. with different side chains were synthesized.  Selected 2-unsubstituted derivs. I [R = NH2, NHMe, NMe2, NH(CH2)2OH ] showed high antiproliferative potency on a panel of mammalian tumor cell lines including multidrug resistance variants.  Compds. I [R = NH2, NHMe, NMe2, NH(CH2)2OH ] utilized multiple mechanisms of cytotoxicity including inhibition of Top1/Top2-mediated DNA relaxation, reduced NAD+/NADH ratio through tNOX inhibition, suppression of a NAD+-dependent sirtuin 1 (SIRT1) deacetylase activity, and activation of caspase-mediated apoptosis.  Here, for the first time, we report that tumor-assocd. NADH oxidase (tNOX) and SIRT1 are important cellular targets of antitumor anthracene-9,10-diones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFsylgKt5c07Vg90H21EOLACvtfcHk0lj_stgQ_nIyOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsLzI&md5=262e35de4b1c5b845e49c29da205e955</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00859%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DLee%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BA.%26aulast%3DYeh%26aufirst%3DC.%2BA.%26aulast%3DTatarskiy%26aufirst%3DV.%2BV.%26aulast%3DDezhenkova%26aufirst%3DL.%2BG.%26aulast%3DGlazunova%26aufirst%3DV.%2BA.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DShtil%26aufirst%3DA.%2BA.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26aulast%3DChueh%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520characterization%2520of%25204%252C11-diaminoanthra%255B2%252C3-b%255Dfuran-5%252C10-diones%253A%2520tumor%2520cell%2520apoptosis%2520through%2520tNOX-modulated%2520NAD%252B%252FNADH%2520ratio%2520and%2520SIRT1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9522%26epage%3D9534%26doi%3D10.1021%2Facs.jmedchem.5b00859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Sabatino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paganelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leone, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papini, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Luca, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginanneschi, M.</span><span> </span><span class="NLM_article-title">A new biotin derivative-DOTA conjugate as a candidate for pretargeted diagnosis and therapy of tumors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3170</span><span class="NLM_x">–</span> <span class="NLM_lpage">3173</span><span class="refDoi"> DOI: 10.1021/jm030789z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030789z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=3170-3173&author=G.+Sabatinoauthor=M.+Chinolauthor=G.+Paganelliauthor=S.+Papiauthor=M.+Chelliauthor=G.+Leoneauthor=A.+M.+Papiniauthor=A.+De+Lucaauthor=M.+Ginanneschi&title=A+new+biotin+derivative-DOTA+conjugate+as+a+candidate+for+pretargeted+diagnosis+and+therapy+of+tumors&doi=10.1021%2Fjm030789z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm030789z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030789z%26sid%3Dliteratum%253Aachs%26aulast%3DSabatino%26aufirst%3DG.%26aulast%3DChinol%26aufirst%3DM.%26aulast%3DPaganelli%26aufirst%3DG.%26aulast%3DPapi%26aufirst%3DS.%26aulast%3DChelli%26aufirst%3DM.%26aulast%3DLeone%26aufirst%3DG.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DDe%2BLuca%26aufirst%3DA.%26aulast%3DGinanneschi%26aufirst%3DM.%26atitle%3DA%2520new%2520biotin%2520derivative-DOTA%2520conjugate%2520as%2520a%2520candidate%2520for%2520pretargeted%2520diagnosis%2520and%2520therapy%2520of%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D3170%26epage%3D3173%26doi%3D10.1021%2Fjm030789z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Huang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span> </span><span class="NLM_article-title">Synthesis of biotin–AMP conjugate for 5′ biotin labeling of RNA through one-step in vitro transcription</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1848</span><span class="NLM_x">–</span> <span class="NLM_lpage">1861</span><span class="refDoi"> DOI: 10.1038/nprot.2008.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnprot.2008.185" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1848-1861&author=F.+Huangauthor=J.+Heauthor=Y.+Zhangauthor=Y.+Guo&title=Synthesis+of+biotin%E2%80%93AMP+conjugate+for+5%E2%80%B2+biotin+labeling+of+RNA+through+one-step+in+vitro+transcription&doi=10.1038%2Fnprot.2008.185"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.185%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520biotin%25E2%2580%2593AMP%2520conjugate%2520for%25205%25E2%2580%25B2%2520biotin%2520labeling%2520of%2520RNA%2520through%2520one-step%2520in%2520vitro%2520transcription%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1848%26epage%3D1861%26doi%3D10.1038%2Fnprot.2008.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Isaac-Lam, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammonds, D. M.</span><span> </span><span class="NLM_article-title">Biotinylated chlorin and its zinc and indium complexes: synthesis and in vitro biological evaluation for photodynamic therapy</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="refDoi"> DOI: 10.3390/ph10020041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.3390%2Fph10020041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCqsLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=41&author=M.+F.+Isaac-Lamauthor=D.+M.+Hammonds&title=Biotinylated+chlorin+and+its+zinc+and+indium+complexes%3A+synthesis+and+in+vitro+biological+evaluation+for+photodynamic+therapy&doi=10.3390%2Fph10020041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biotinylated chlorin and its zinc and indium complexes: synthesis and in vitro biological evaluation for photodynamic therapy</span></div><div class="casAuthors">Isaac-Lam, Meden F.; Hammonds, Dewana M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">41/1-41/12</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The synthesis and characterization of biotinylated chlorin photosensitizer and the corresponding zinc and indium complexes are described for potential applications in photodynamic therapy (PDT) for cancer.  Phototoxicity of the biotin-chlorin conjugate and the metalated complexes was detd. in colon carcinoma CT26 cell lines known to overexpress biotin (Vit B7) receptors.  Cell survival assay indicated that the biotinylated chlorin and indium complex showed increased cell growth inhibition than the zinc complex and the starting chlorin (Me pheophorbide).  Fluorescence microcopy studies revealed the generation of apoptotic cells upon light irradn. of colon cells treated with the indium complex.  Targeting biotin receptors in cancer cells can improve specificity of photosensitizers for PDT applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5wyaEGaU9LVg90H21EOLACvtfcHk0li3YiETZgGuXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCqsLfI&md5=41721d161fee8993cca4ea706d134482</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3390%2Fph10020041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph10020041%26sid%3Dliteratum%253Aachs%26aulast%3DIsaac-Lam%26aufirst%3DM.%2BF.%26aulast%3DHammonds%26aufirst%3DD.%2BM.%26atitle%3DBiotinylated%2520chlorin%2520and%2520its%2520zinc%2520and%2520indium%2520complexes%253A%2520synthesis%2520and%2520in%2520vitro%2520biological%2520evaluation%2520for%2520photodynamic%2520therapy%26jtitle%3DPharmaceuticals%26date%3D2017%26volume%3D10%26spage%3D41%26doi%3D10.3390%2Fph10020041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzikov, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korolev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span> </span><span class="NLM_article-title">Heterocyclic analogs of 5,12-naphthacene-Quinone. 11. A new method for preparing 4,11-dimethoxyanthra[2,3-<i>b</i>]furan-5,10-dione</span> <span class="citation_source-journal">Chem. Heterocycl. Compd.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1206</span><span class="NLM_x">–</span> <span class="NLM_lpage">1211</span><span class="refDoi"> DOI: 10.1007/s10593-012-0895-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs10593-012-0895-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=1206-1211&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=Y.+N.+Luzikovauthor=A.+M.+Korolevauthor=M.+N.+Preobrazhenskaya&title=Heterocyclic+analogs+of+5%2C12-naphthacene-Quinone.+11.+A+new+method+for+preparing+4%2C11-dimethoxyanthra%5B2%2C3-b%5Dfuran-5%2C10-dione&doi=10.1007%2Fs10593-012-0895-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2Fs10593-012-0895-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-012-0895-4%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DLuzikov%26aufirst%3DY.%2BN.%26aulast%3DKorolev%26aufirst%3DA.%2BM.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DHeterocyclic%2520analogs%2520of%25205%252C12-naphthacene-Quinone.%252011.%2520A%2520new%2520method%2520for%2520preparing%25204%252C11-dimethoxyanthra%255B2%252C3-b%255Dfuran-5%252C10-dione%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2011%26volume%3D47%26spage%3D1206%26epage%3D1211%26doi%3D10.1007%2Fs10593-012-0895-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Human telomeric G-quadruplex: structures of DNA and RNA sequences</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2009.07464.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1111%2Fj.1742-4658.2009.07464.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=19951353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivVSmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=1107-1117&author=A.+T.+Phan&title=Human+telomeric+G-quadruplex%3A+structures+of+DNA+and+RNA+sequences&doi=10.1111%2Fj.1742-4658.2009.07464.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Human telomeric G-quadruplex: structures of DNA and RNA sequences</span></div><div class="casAuthors">Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1107-1117</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Telomeres play an important role in cellular aging and cancer.  Human telomeric DNA and RNA G-rich sequences are capable of forming a 4-stranded structure, known as the G-quadruplex.  Such a structure might be important for telomere biol. and a good target for drug design.  Here, the author describes the structural diversity or conservation of DNA and RNA human telomeric G-quadruplexes, discusses structural views on targeting these G-quadruplexes, and presents some future challenges for structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjtvxCsiVbsrVg90H21EOLACvtfcHk0li3YiETZgGuXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivVSmsbc%253D&md5=341396c7c9e0600f09f9beaee1fc0879</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2009.07464.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2009.07464.x%26sid%3Dliteratum%253Aachs%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DHuman%2520telomeric%2520G-quadruplex%253A%2520structures%2520of%2520DNA%2520and%2520RNA%2520sequences%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D1107%26epage%3D1117%26doi%3D10.1111%2Fj.1742-4658.2009.07464.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">HRAS is silenced by two neighboring G-quadruplexes and activated by MAZ, a zinc-finger transcription factor with DNA unfolding property</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">8379</span><span class="NLM_x">–</span> <span class="NLM_lpage">8388</span><span class="refDoi"> DOI: 10.1093/nar/gku574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgku574" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=8379-8388&author=S.+Cogoiauthor=A.+E.+Shchekotikhinauthor=L.+E.+Xodo&title=HRAS+is+silenced+by+two+neighboring+G-quadruplexes+and+activated+by+MAZ%2C+a+zinc-finger+transcription+factor+with+DNA+unfolding+property&doi=10.1093%2Fnar%2Fgku574"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku574%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DHRAS%2520is%2520silenced%2520by%2520two%2520neighboring%2520G-quadruplexes%2520and%2520activated%2520by%2520MAZ%252C%2520a%2520zinc-finger%2520transcription%2520factor%2520with%2520DNA%2520unfolding%2520property%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8379%26epage%3D8388%26doi%3D10.1093%2Fnar%2Fgku574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cauci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Critical role of hnRNP A1 in activating KRAS transcription in pancreatic cancer cells: A molecular mechanism involving G4 DNA</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">1861</span><span class="NLM_x">, </span> <span class="NLM_fpage">1389</span><span class="NLM_x">–</span> <span class="NLM_lpage">1398</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2016.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bbagen.2016.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27888145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSmsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2017&pages=1389-1398&author=S.+Cogoiauthor=V.+Rapozziauthor=S.+Cauciauthor=L.+E.+Xodo&title=Critical+role+of+hnRNP+A1+in+activating+KRAS+transcription+in+pancreatic+cancer+cells%3A+A+molecular+mechanism+involving+G4+DNA&doi=10.1016%2Fj.bbagen.2016.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of hnRNP A1 in activating KRAS transcription in pancreatic cancer cells: A molecular mechanism involving G4 DNA</span></div><div class="casAuthors">Cogoi, Susanna; Rapozzi, Valentina; Cauci, Sabina; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">5_Part_B</span>),
    <span class="NLM_cas:pages">1389-1398</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KRAS is one of the most mutated genes in human cancer.  Its crucial role in the tumorigenesis of pancreatic ductal adenocarcinoma (PDAC) has been widely demonstrated.  As this deadly cancer does not sufficiently respond to conventional chemotherapies, it is important to increase our knowledge of pancreatic cancer biol., in particular how oncogenic KRAS is regulated.  The promoter of KRAS contains a GA-element composed of runs of guanines that fold into a G4 structure.  This unusual DNA conformation is recognized by several nuclear proteins, including MAZ and hnRNP A1.  Recent data have revealed that KRAS is interconnected to ILK and hnRNP A1 in a circuitry that enables pancreatic cancer cells to maintain an aggressive phenotype.  The present review illustrates recent advances on how KRAS is regulated in pancreatic cancer cells, focusing on the formation of G4 structures in the KRAS promoter and their interaction with hnRNP A1.  The newly discovered KRAS-ILK-hnRNP A1 regulatory loop is discussed, emphasizing its potential as a therapeutic target for PDAC-specific mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9A7QOe721LVg90H21EOLACvtfcHk0li3YiETZgGuXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSmsbzP&md5=43a9ecdbf65e6a7e027a063206738bb4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2016.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2016.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DCauci%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DCritical%2520role%2520of%2520hnRNP%2520A1%2520in%2520activating%2520KRAS%2520transcription%2520in%2520pancreatic%2520cancer%2520cells%253A%2520A%2520molecular%2520mechanism%2520involving%2520G4%2520DNA%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2017%26volume%3D1861%26spage%3D1389%26epage%3D1398%26doi%3D10.1016%2Fj.bbagen.2016.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0lh6J_OEQbQikA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Géci, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">MAZ-binding G4-decoy with locked nucleic acid and twisted intercalating nucleic acid modifications suppresses KRAS in pancreatic cancer cells and delays tumor growth in mice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4049</span><span class="NLM_x">–</span> <span class="NLM_lpage">4064</span><span class="refDoi"> DOI: 10.1093/nar/gkt127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkt127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23471001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=4049-4064&author=S.+Cogoiauthor=S.+Zorzetauthor=V.+Rapozziauthor=I.+G%C3%A9ciauthor=E.+B.+Pedersenauthor=L.+E.+Xodo&title=MAZ-binding+G4-decoy+with+locked+nucleic+acid+and+twisted+intercalating+nucleic+acid+modifications+suppresses+KRAS+in+pancreatic+cancer+cells+and+delays+tumor+growth+in+mice&doi=10.1093%2Fnar%2Fgkt127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MAZ-binding G4-decoy with locked nucleic acid and twisted intercalating nucleic acid modifications suppresses KRAS in pancreatic cancer cells and delays tumor growth in mice</span></div><div class="casAuthors">Cogoi, Susanna; Zorzet, Sonia; Rapozzi, Valentina; Geci, Imrich; Pedersen, Erik B.; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4049-4064</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">KRAS mutations are primary genetic lesions leading to pancreatic cancer.  The promoter of human KRAS contains a nuclease-hypersensitive element (NHE) that can fold in G4-DNA structures binding to nuclear proteins, including MAZ (myc-assocd. zinc-finger).  Here, we report that MAZ activates KRAS transcription.  To knockdown oncogenic KRAS in pancreatic cancer cells, we designed oligonucleotides that mimic one of the G-quadruplexes formed by NHE (G4-decoys).  To increase their nuclease resistance, two locked nucleic acid (LNA) modifications were introduced at the 3'-end, whereas to enhance the folding and stability, two polycyclic arom. hydrocarbon units (TINA or AMANY) were inserted internally, to cap the quadruplex.  The most active G4-decoy (2998), which had two para-TINAs, strongly suppressed KRAS expression in Panc-1 cells.  It also repressed their metabolic activity (IC50 = 520 nM), and it inhibited cell growth and colony formation by activating apoptosis.  We finally injected 2998 and control oligonucleotides 5153, 5154 (2 nmol/mouse) intratumorally in SCID mice bearing a Panc-1 xenograft.  After three treatments, 2998 reduced tumor xenograft growth by 64% compared with control and increased the Kaplan-Meier median survival time by 70%.  Together, our data show that MAZ-specific G4-decoys mimicking a KRAS quadruplex are promising for pancreatic cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoQEFAMt2lfrVg90H21EOLACvtfcHk0lh6J_OEQbQikA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKju70%253D&md5=461de60d1eafb94201a1bd600c4829f2</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt127%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DG%25C3%25A9ci%26aufirst%3DI.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DMAZ-binding%2520G4-decoy%2520with%2520locked%2520nucleic%2520acid%2520and%2520twisted%2520intercalating%2520nucleic%2520acid%2520modifications%2520suppresses%2520KRAS%2520in%2520pancreatic%2520cancer%2520cells%2520and%2520delays%2520tumor%2520growth%2520in%2520mice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D4049%26epage%3D4064%26doi%3D10.1093%2Fnar%2Fgkt127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Wlodkowic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skommer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darzynkiewicz, Z.</span><span> </span><span class="NLM_article-title">Flow cytometry-based apoptosis detection</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">559</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1007/978-1-60327-017-5_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2F978-1-60327-017-5_2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=19609746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=559&publication_year=2009&pages=19-32&author=D.+Wlodkowicauthor=J.+Skommerauthor=Z.+Darzynkiewicz&title=Flow+cytometry-based+apoptosis+detection&doi=10.1007%2F978-1-60327-017-5_2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Flow cytometry-based apoptosis detection</span></div><div class="casAuthors">Wlodkowic, Donald; Skommer, Joanna; Darzynkiewicz, Zbigniew</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">559</span>
        (<span class="NLM_cas:issue">Apoptosis</span>),
    <span class="NLM_cas:pages">19-32</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">An apoptosing cell demonstrates multitude of characteristic morphol. and biochem. features, which vary depending on the stimuli and the cell type.  The gross majority of classical apoptotic hallmarks can be rapidly examd. by flow and image cytometry.  Cytometry thus became a technol. of choice in diverse studies of cellular demise.  A large variety of cytometric methods designed to identify apoptotic cells and probe mechanisms assocd. with this mode of cell demise have been developed during the past two decades.  In the present chapter, we outline a handful of commonly used methods that are based on the assessment of: mitochondrial transmembrane potential, activation of caspases, plasma membrane alterations and DNA fragmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHAI-ImJTtYLVg90H21EOLACvtfcHk0lh6J_OEQbQikA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVylurk%253D&md5=648ed0c0eedf72b432173d8e9d112165</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1007%2F978-1-60327-017-5_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60327-017-5_2%26sid%3Dliteratum%253Aachs%26aulast%3DWlodkowic%26aufirst%3DD.%26aulast%3DSkommer%26aufirst%3DJ.%26aulast%3DDarzynkiewicz%26aufirst%3DZ.%26atitle%3DFlow%2520cytometry-based%2520apoptosis%2520detection%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2009%26volume%3D559%26spage%3D19%26epage%3D32%26doi%3D10.1007%2F978-1-60327-017-5_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1038/nchem.1805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchem.1805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24345950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=75-80&author=G.+Biffiauthor=M.+Di+Antonioauthor=D.+Tannahillauthor=S.+Balasubramanian&title=Visualization+and+selective+chemical+targeting+of+RNA+G-quadruplex+structures+in+the+cytoplasm+of+human+cells&doi=10.1038%2Fnchem.1805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span></div><div class="casAuthors">Biffi, Giulia; Di Antonio, Marco; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-80</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following extensive evidence for the formation of four-stranded DNA G-quadruplex structures in vitro, DNA G-quadruplexes have been obsd. within human cells.  Although chem. distinct, RNA can also fold in vitro into G-quadruplex structures that are highly stable because of the 2'-hydroxyl group.  However, RNA G-quadruplexes have not yet been reported in cells.  Here, we demonstrate the visualization of RNA G-quadruplex structures within the cytoplasm of human cells using a G-quadruplex structure-specific antibody.  We also demonstrate that small mols. that bind to G-quadruplexes in vitro can trap endogenous RNA G-quadruplexes when applied to cells.  Furthermore, a small mol. that exhibits a preference for RNA G-quadruplexes rather than DNA G-quadruplexes in biophys. expts. also shows the same selectivity within a cellular context.  Our findings provide substantive evidence for RNA G-quadruplex formation in the human transcriptome, and corroborate the selectivity and application of stabilizing ligands that target G-quadruplexes within a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmciXxoAihK7Vg90H21EOLACvtfcHk0ljPf0vdK6Kqzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N&md5=cc944f978b038bf42176a861a80b61d0</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1805%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DVisualization%2520and%2520selective%2520chemical%2520targeting%2520of%2520RNA%2520G-quadruplex%2520structures%2520in%2520the%2520cytoplasm%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D75%26epage%3D80%26doi%3D10.1038%2Fnchem.1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Guo, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartel, D. P.</span><span> </span><span class="NLM_article-title">RNA G-quadruplexes are globally unfolded in eukaryotiv cells and depleted in bacteria</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">353</span><span class="NLM_x">, </span> <span class="NLM_fpage">6306</span><span class="refDoi"> DOI: 10.1126/science.aaf5371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1126%2Fscience.aaf5371" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=6306&author=J.+U.+Guoauthor=D.+P.+Bartel&title=RNA+G-quadruplexes+are+globally+unfolded+in+eukaryotiv+cells+and+depleted+in+bacteria&doi=10.1126%2Fscience.aaf5371"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf5371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf5371%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%2BU.%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DRNA%2520G-quadruplexes%2520are%2520globally%2520unfolded%2520in%2520eukaryotiv%2520cells%2520and%2520depleted%2520in%2520bacteria%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D6306%26doi%3D10.1126%2Fscience.aaf5371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Studier, F. W.</span><span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">207</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span><span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0ljPf0vdK6Kqzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alexander S. Tikhomirov, Mahmoud A. S. Abdelhamid, Georgy Y. Nadysev, George V. Zatonsky, Eugene E. Bykov, Pin Ju Chueh, Zoë A. E. Waller, <span class="NLM_string-name hlFld-ContribAuthor">Andrey E. Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">Water-Soluble Heliomycin Derivatives to Target i-Motif DNA. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2021,</strong> <em>84 </em>
                                    (5)
                                     , 1617-1625. <a href="https://doi.org/10.1021/acs.jnatprod.1c00162" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.1c00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.1c00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.1c00162%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DWater-Soluble%252BHeliomycin%252BDerivatives%252Bto%252BTarget%252Bi-Motif%252BDNA%26aulast%3DTikhomirov%26aufirst%3DAlexander%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19022021%26date%3D11052021%26volume%3D84%26issue%3D5%26spage%3D1617%26epage%3D1625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Annalisa Ferino, Giulia Nicoletto, Francesca D’Este, Sonia Zorzet, Sara Lago, Sara N. Richter, Alexander Tikhomirov, Andrey Shchekotikhin, <span class="NLM_string-name hlFld-ContribAuthor">Luigi E. Xodo</span>. </span><span class="cited-content_cbyCitation_article-title">Photodynamic Therapy for ras-Driven Cancers: Targeting G-Quadruplex RNA Structures with Bifunctional Alkyl-Modified Porphyrins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (3)
                                     , 1245-1260. <a href="https://doi.org/10.1021/acs.jmedchem.9b01577" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01577</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01577%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPhotodynamic%252BTherapy%252Bfor%252Bras-Driven%252BCancers%25253A%252BTargeting%252BG-Quadruplex%252BRNA%252BStructures%252Bwith%252BBifunctional%252BAlkyl-Modified%252BPorphyrins%26aulast%3DFerino%26aufirst%3DAnnalisa%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D21092019%26date%3D24012020%26date%3D13012020%26volume%3D63%26issue%3D3%26spage%3D1245%26epage%3D1260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brittany S. Morgan, Bilva G. Sanaba, Anita Donlic, Diane B. Karloff, Jordan E. Forte, Yuqi Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Amanda E. Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">R-BIND: An Interactive Database for Exploring and Developing RNA-Targeted Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 2691-2700. <a href="https://doi.org/10.1021/acschembio.9b00631" title="DOI URL">https://doi.org/10.1021/acschembio.9b00631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00631%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DR-BIND%25253A%252BAn%252BInteractive%252BDatabase%252Bfor%252BExploring%252Band%252BDeveloping%252BRNA-Targeted%252BChemical%252BProbes%26aulast%3DMorgan%26aufirst%3DBrittany%2BS.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03082019%26date%3D07102019%26date%3D29102019%26date%3D07102019%26volume%3D14%26issue%3D12%26spage%3D2691%26epage%3D2700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Munir
A. Al-Zeer, <span class="NLM_string-name hlFld-ContribAuthor">Jens Kurreck</span>. </span><span class="cited-content_cbyCitation_article-title">Deciphering the Enigmatic Biological Functions of RNA Guanine-Quadruplex Motifs in Human Cells. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2019,</strong> <em>58 </em>
                                    (5)
                                     , 305-311. <a href="https://doi.org/10.1021/acs.biochem.8b00904" title="DOI URL">https://doi.org/10.1021/acs.biochem.8b00904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.8b00904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.8b00904%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DDeciphering%252Bthe%252BEnigmatic%252BBiological%252BFunctions%252Bof%252BRNA%252BGuanine-Quadruplex%252BMotifs%252Bin%252BHuman%252BCells%26aulast%3DAl-Zeer%26aufirst%3DMunir%2BA.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D29082018%26date%3D12102018%26date%3D24102018%26date%3D17102018%26volume%3D58%26issue%3D5%26spage%3D305%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matteo Nadai, Filippo Doria, Matteo Scalabrin, Valentina Pirota, Vincenzo Grande, Greta Bergamaschi, Valeria Amendola, Fernaldo Richtia Winnerdy, Anh Tuân Phan, Sara N. Richter, <span class="NLM_string-name hlFld-ContribAuthor">Mauro Freccero</span>. </span><span class="cited-content_cbyCitation_article-title">A Catalytic and Selective Scissoring Molecular Tool for Quadruplex Nucleic Acids. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (44)
                                     , 14528-14532. <a href="https://doi.org/10.1021/jacs.8b05337" title="DOI URL">https://doi.org/10.1021/jacs.8b05337</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b05337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b05337%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DA%252BCatalytic%252Band%252BSelective%252BScissoring%252BMolecular%252BTool%252Bfor%252BQuadruplex%252BNucleic%252BAcids%26aulast%3DNadai%26aufirst%3DMatteo%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22052018%26date%3D29102018%26date%3D14102018%26volume%3D140%26issue%3D44%26spage%3D14528%26epage%3D14532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nils  Kosiol</span>, <span class="hlFld-ContribAuthor ">Stefan  Juranek</span>, <span class="hlFld-ContribAuthor ">Peter  Brossart</span>, <span class="hlFld-ContribAuthor ">Annkristin  Heine</span>, <span class="hlFld-ContribAuthor ">Katrin  Paeschke</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplexes: a promising target for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer</span><span> <strong>2021,</strong> <em>20 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12943-021-01328-4" title="DOI URL">https://doi.org/10.1186/s12943-021-01328-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12943-021-01328-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12943-021-01328-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%26atitle%3DG-quadruplexes%25253A%252Ba%252Bpromising%252Btarget%252Bfor%252Bcancer%252Btherapy%26aulast%3DKosiol%26aufirst%3DNils%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yulia L.  Volodina</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Lyubov G.  Dezhenkova</span>, <span class="hlFld-ContribAuthor ">Alla A.  Ramonova</span>, <span class="hlFld-ContribAuthor ">Anastasia V.  Kononova</span>, <span class="hlFld-ContribAuthor ">Daria V.  Andreeva</span>, <span class="hlFld-ContribAuthor ">Dmitry N.  Kaluzhny</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Mikhail M.  Moisenovich</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Shtil</span>. </span><span class="cited-content_cbyCitation_article-title">Thiophene-2-carboxamide derivatives of anthraquinone: A new potent antitumor chemotype. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>221 </em>, 113521. <a href="https://doi.org/10.1016/j.ejmech.2021.113521" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113521%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThiophene-2-carboxamide%252Bderivatives%252Bof%252Banthraquinone%25253A%252BA%252Bnew%252Bpotent%252Bantitumor%252Bchemotype%26aulast%3DVolodina%26aufirst%3DYulia%2BL.%26date%3D2021%26volume%3D221%26spage%3D113521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huachun  Liu</span>, <span class="hlFld-ContribAuthor ">Simone  Rauch</span>, <span class="hlFld-ContribAuthor ">Bryan C.  Dickinson</span>. </span><span class="cited-content_cbyCitation_article-title">Programmable technologies to manipulate gene expression at the RNA level. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>64 </em>, 27-37. <a href="https://doi.org/10.1016/j.cbpa.2021.03.004" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DProgrammable%252Btechnologies%252Bto%252Bmanipulate%252Bgene%252Bexpression%252Bat%252Bthe%252BRNA%252Blevel%26aulast%3DLiu%26aufirst%3DHuachun%26date%3D2021%26volume%3D64%26spage%3D27%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victoria  Sanchez-Martin</span>, <span class="hlFld-ContribAuthor ">Miguel  Soriano</span>, <span class="hlFld-ContribAuthor ">Jose Antonio  Garcia-Salcedo</span>. </span><span class="cited-content_cbyCitation_article-title">Quadruplex Ligands in Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (13)
                                     , 3156. <a href="https://doi.org/10.3390/cancers13133156" title="DOI URL">https://doi.org/10.3390/cancers13133156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13133156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13133156%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DQuadruplex%252BLigands%252Bin%252BCancer%252BTherapy%26aulast%3DSanchez-Martin%26aufirst%3DVictoria%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D13%26spage%3D3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jared T.  Baisden</span>, <span class="hlFld-ContribAuthor ">Jessica L.  Childs-Disney</span>, <span class="hlFld-ContribAuthor ">Lucas S.  Ryan</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Affecting RNA biology genome-wide by binding small molecules and chemically induced proximity. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>62 </em>, 119-129. <a href="https://doi.org/10.1016/j.cbpa.2021.03.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DAffecting%252BRNA%252Bbiology%252Bgenome-wide%252Bby%252Bbinding%252Bsmall%252Bmolecules%252Band%252Bchemically%252Binduced%252Bproximity%26aulast%3DBaisden%26aufirst%3DJared%2BT.%26date%3D2021%26volume%3D62%26spage%3D119%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamini  Singh</span>, <span class="hlFld-ContribAuthor ">Jianan  Lin</span>, <span class="hlFld-ContribAuthor ">Nicolas  Lecomte</span>, <span class="hlFld-ContribAuthor ">Prathibha  Mohan</span>, <span class="hlFld-ContribAuthor ">Askan  Gokce</span>, <span class="hlFld-ContribAuthor ">Viraj R.  Sanghvi</span>, <span class="hlFld-ContribAuthor ">Man  Jiang</span>, <span class="hlFld-ContribAuthor ">Olivera  Grbovic-Huezo</span>, <span class="hlFld-ContribAuthor ">Antonija  Burčul</span>, <span class="hlFld-ContribAuthor ">Stefan G.  Stark</span>, <span class="hlFld-ContribAuthor ">Paul B.  Romesser</span>, <span class="hlFld-ContribAuthor ">Qing  Chang</span>, <span class="hlFld-ContribAuthor ">Jerry P.  Melchor</span>, <span class="hlFld-ContribAuthor ">Rachel K.  Beyer</span>, <span class="hlFld-ContribAuthor ">Mark  Duggan</span>, <span class="hlFld-ContribAuthor ">Yoshiyuki  Fukase</span>, <span class="hlFld-ContribAuthor ">Guangli  Yang</span>, <span class="hlFld-ContribAuthor ">Ouathek  Ouerfelli</span>, <span class="hlFld-ContribAuthor ">Agnes  Viale</span>, <span class="hlFld-ContribAuthor ">Elisa  de Stanchina</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Stamford</span>, <span class="hlFld-ContribAuthor ">Peter T.  Meinke</span>, <span class="hlFld-ContribAuthor ">Gunnar  Rätsch</span>, <span class="hlFld-ContribAuthor ">Steven D.  Leach</span>, <span class="hlFld-ContribAuthor ">Zhengqing  Ouyang</span>, <span class="hlFld-ContribAuthor ">Hans-Guido  Wendel</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting eIF4A-Dependent Translation of KRAS Signaling Molecules. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2021,</strong> <em>81 </em>
                                    (8)
                                     , 2002-2014. <a href="https://doi.org/10.1158/0008-5472.CAN-20-2929" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-20-2929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-20-2929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-20-2929%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DTargeting%252BeIF4A-Dependent%252BTranslation%252Bof%252BKRAS%252BSignaling%252BMolecules%26aulast%3DSingh%26aufirst%3DKamini%26date%3D2021%26date%3D2021%26volume%3D81%26issue%3D8%26spage%3D2002%26epage%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Loukiani  Savva</span>, <span class="hlFld-ContribAuthor ">Savvas N.  Georgiades</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments in Small-Molecule Ligands of Medicinal Relevance for Harnessing the Anticancer Potential of G-Quadruplexes. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 841. <a href="https://doi.org/10.3390/molecules26040841" title="DOI URL">https://doi.org/10.3390/molecules26040841</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040841%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRecent%252BDevelopments%252Bin%252BSmall-Molecule%252BLigands%252Bof%252BMedicinal%252BRelevance%252Bfor%252BHarnessing%252Bthe%252BAnticancer%252BPotential%252Bof%252BG-Quadruplexes%26aulast%3DSavva%26aufirst%3DLoukiani%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daria V.  Andreeva</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">Ligands of G-quadruplex nucleic acids. </span><span class="cited-content_cbyCitation_journal-name">Russian Chemical Reviews</span><span> <strong>2021,</strong> <em>90 </em>
                                    (1)
                                     , 1-38. <a href="https://doi.org/10.1070/RCR4968" title="DOI URL">https://doi.org/10.1070/RCR4968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1070/RCR4968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1070%2FRCR4968%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Chemical%2520Reviews%26atitle%3DLigands%252Bof%252BG-quadruplex%252Bnucleic%252Bacids%26aulast%3DAndreeva%26aufirst%3DDaria%2BV.%26date%3D2021%26volume%3D90%26issue%3D1%26spage%3D1%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prakash  Kharel</span>, <span class="hlFld-ContribAuthor ">Gertraud  Becker</span>, <span class="hlFld-ContribAuthor ">Vladimir  Tsvetkov</span>, <span class="hlFld-ContribAuthor ">Pavel  Ivanov</span>. </span><span class="cited-content_cbyCitation_article-title">Properties and biological impact of RNA G-quadruplexes: from order to turmoil and back. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (22)
                                     , 12534-12555. <a href="https://doi.org/10.1093/nar/gkaa1126" title="DOI URL">https://doi.org/10.1093/nar/gkaa1126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa1126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa1126%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DProperties%252Band%252Bbiological%252Bimpact%252Bof%252BRNA%252BG-quadruplexes%25253A%252Bfrom%252Border%252Bto%252Bturmoil%252Band%252Bback%26aulast%3DKharel%26aufirst%3DPrakash%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D22%26spage%3D12534%26epage%3D12555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Miglietta</span>, <span class="hlFld-ContribAuthor ">Marco  Russo</span>, <span class="hlFld-ContribAuthor ">Giovanni  Capranico</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplex–R-loop interactions and the mechanism of anticancer G-quadruplex binders. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (21)
                                     , 11942-11957. <a href="https://doi.org/10.1093/nar/gkaa944" title="DOI URL">https://doi.org/10.1093/nar/gkaa944</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa944%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DG-quadruplex%2525E2%252580%252593R-loop%252Binteractions%252Band%252Bthe%252Bmechanism%252Bof%252Banticancer%252BG-quadruplex%252Bbinders%26aulast%3DMiglietta%26aufirst%3DGiulia%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D21%26spage%3D11942%26epage%3D11957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neha</span>, <span class="hlFld-ContribAuthor ">Parimal  Das</span>, <span class="hlFld-ContribAuthor ">Shiv Prakash  Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Dual role of G-quadruplex in translocation renal cell carcinoma: Exploring plausible Cancer therapeutic innovation. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2020,</strong> <em>1864 </em>
                                    (12)
                                     , 129719. <a href="https://doi.org/10.1016/j.bbagen.2020.129719" title="DOI URL">https://doi.org/10.1016/j.bbagen.2020.129719</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2020.129719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2020.129719%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DDual%252Brole%252Bof%252BG-quadruplex%252Bin%252Btranslocation%252Brenal%252Bcell%252Bcarcinoma%25253A%252BExploring%252Bplausible%252BCancer%252Btherapeutic%252Binnovation%26aulast%3DNeha%26date%3D2020%26volume%3D1864%26issue%3D12%26spage%3D129719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Mies J.  van Steenbergen</span>, <span class="hlFld-ContribAuthor ">Nataliia  Beztsinna</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>, <span class="hlFld-ContribAuthor ">Twan  Lammers</span>, <span class="hlFld-ContribAuthor ">Cornelus F.  van Nostrum</span>, <span class="hlFld-ContribAuthor ">Wim E.  Hennink</span>. </span><span class="cited-content_cbyCitation_article-title">Biotin-decorated all-HPMA polymeric micelles for paclitaxel delivery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Controlled Release</span><span> <strong>2020,</strong> <em>328 </em>, 970-984. <a href="https://doi.org/10.1016/j.jconrel.2020.09.013" title="DOI URL">https://doi.org/10.1016/j.jconrel.2020.09.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jconrel.2020.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jconrel.2020.09.013%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Controlled%2520Release%26atitle%3DBiotin-decorated%252Ball-HPMA%252Bpolymeric%252Bmicelles%252Bfor%252Bpaclitaxel%252Bdelivery%26aulast%3DWang%26aufirst%3DYan%26date%3D2020%26volume%3D328%26spage%3D970%26epage%3D984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ai-Ming  Yu</span>, <span class="hlFld-ContribAuthor ">Young Hee  Choi</span>, <span class="hlFld-ContribAuthor ">Mei-Juan  Tu</span>, . </span><span class="cited-content_cbyCitation_article-title">RNA Drugs and RNA Targets for Small Molecules: Principles, Progress, and Challenges. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reviews</span><span> <strong>2020,</strong> <em>72 </em>
                                    (4)
                                     , 862-898. <a href="https://doi.org/10.1124/pr.120.019554" title="DOI URL">https://doi.org/10.1124/pr.120.019554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/pr.120.019554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fpr.120.019554%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reviews%26atitle%3DRNA%252BDrugs%252Band%252BRNA%252BTargets%252Bfor%252BSmall%252BMolecules%25253A%252BPrinciples%25252C%252BProgress%25252C%252Band%252BChallenges%26aulast%3DYu%26aufirst%3DAi-Ming%26date%3D2020%26date%3D2020%26volume%3D72%26issue%3D4%26spage%3D862%26epage%3D898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Josué  Carvalho</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Mergny</span>, <span class="hlFld-ContribAuthor ">Gilmar F.  Salgado</span>, <span class="hlFld-ContribAuthor ">João A.  Queiroz</span>, <span class="hlFld-ContribAuthor ">Carla  Cruz</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplex, Friend or Foe: The Role of the G-quartet in Anticancer Strategies. </span><span class="cited-content_cbyCitation_journal-name">Trends in Molecular Medicine</span><span> <strong>2020,</strong> <em>26 </em>
                                    (9)
                                     , 848-861. <a href="https://doi.org/10.1016/j.molmed.2020.05.002" title="DOI URL">https://doi.org/10.1016/j.molmed.2020.05.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmed.2020.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmed.2020.05.002%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Molecular%2520Medicine%26atitle%3DG-quadruplex%25252C%252BFriend%252Bor%252BFoe%25253A%252BThe%252BRole%252Bof%252Bthe%252BG-quartet%252Bin%252BAnticancer%252BStrategies%26aulast%3DCarvalho%26aufirst%3DJosu%25C3%25A9%26date%3D2020%26volume%3D26%26issue%3D9%26spage%3D848%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valeria A.  Litvinova</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Ivan V.  Ivanov</span>, <span class="hlFld-ContribAuthor ">Svetlana E.  Solovieva</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">A facile access to 2-substituted naphtho[2,3-g]quinoline-3-carboxylic acid esters via intramolecular cyclization and PyBOP-promoted functionalization. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2020,</strong> <em>76 </em>
                                    (36)
                                     , 131418. <a href="https://doi.org/10.1016/j.tet.2020.131418" title="DOI URL">https://doi.org/10.1016/j.tet.2020.131418</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2020.131418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2020.131418%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DA%252Bfacile%252Baccess%252Bto%252B2-substituted%252Bnaphtho%25255B2%25252C3-g%25255Dquinoline-3-carboxylic%252Bacid%252Besters%252Bvia%252Bintramolecular%252Bcyclization%252Band%252BPyBOP-promoted%252Bfunctionalization%26aulast%3DLitvinova%26aufirst%3DValeria%2BA.%26date%3D2020%26volume%3D76%26issue%3D36%26spage%3D131418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giorgio  Cinque</span>, <span class="hlFld-ContribAuthor ">Annalisa  Ferino</span>, <span class="hlFld-ContribAuthor ">Erik B.  Pedersen</span>, <span class="hlFld-ContribAuthor ">Luigi E.  Xodo</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Poly [ADP-ribose] Polymerase 1 in Activating the Kirsten ras (KRAS) Gene in Response to Oxidative Stress. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (17)
                                     , 6237. <a href="https://doi.org/10.3390/ijms21176237" title="DOI URL">https://doi.org/10.3390/ijms21176237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21176237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21176237%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DRole%252Bof%252BPoly%252B%25255BADP-ribose%25255D%252BPolymerase%252B1%252Bin%252BActivating%252Bthe%252BKirsten%252Bras%252B%252528KRAS%252529%252BGene%252Bin%252BResponse%252Bto%252BOxidative%252BStress%26aulast%3DCinque%26aufirst%3DGiorgio%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D17%26spage%3D6237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Valeria A.  Litvinova</span>, <span class="hlFld-ContribAuthor ">Daria V.  Andreeva</span>, <span class="hlFld-ContribAuthor ">Vladimir B.  Tsvetkov</span>, <span class="hlFld-ContribAuthor ">Lyubov G.  Dezhenkova</span>, <span class="hlFld-ContribAuthor ">Yulia L.  Volodina</span>, <span class="hlFld-ContribAuthor ">Dmitry N.  Kaluzhny</span>, <span class="hlFld-ContribAuthor ">Ivan D.  Treshalin</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Alla A.  Ramonova</span>, <span class="hlFld-ContribAuthor ">Mikhail M.  Moisenovich</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Shtil</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">Amides of pyrrole- and thiophene-fused anthraquinone derivatives: A role of the heterocyclic core in antitumor properties. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>199 </em>, 112294. <a href="https://doi.org/10.1016/j.ejmech.2020.112294" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112294%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAmides%252Bof%252Bpyrrole-%252Band%252Bthiophene-fused%252Banthraquinone%252Bderivatives%25253A%252BA%252Brole%252Bof%252Bthe%252Bheterocyclic%252Bcore%252Bin%252Bantitumor%252Bproperties%26aulast%3DTikhomirov%26aufirst%3DAlexander%2BS.%26date%3D2020%26volume%3D199%26spage%3D112294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  D'Aria</span>, <span class="hlFld-ContribAuthor ">Vincenzo Maria  D'Amore</span>, <span class="hlFld-ContribAuthor ">Francesco Saverio  Di Leva</span>, <span class="hlFld-ContribAuthor ">Jussara  Amato</span>, <span class="hlFld-ContribAuthor ">Marco  Caterino</span>, <span class="hlFld-ContribAuthor ">Pasquale  Russomanno</span>, <span class="hlFld-ContribAuthor ">Silvia  Salerno</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Barresi</span>, <span class="hlFld-ContribAuthor ">Marinella  De Leo</span>, <span class="hlFld-ContribAuthor ">Anna Maria  Marini</span>, <span class="hlFld-ContribAuthor ">Sabrina  Taliani</span>, <span class="hlFld-ContribAuthor ">Federico  Da Settimo</span>, <span class="hlFld-ContribAuthor ">Gilmar F.  Salgado</span>, <span class="hlFld-ContribAuthor ">Luca  Pompili</span>, <span class="hlFld-ContribAuthor ">Pasquale  Zizza</span>, <span class="hlFld-ContribAuthor ">Senji  Shirasawa</span>, <span class="hlFld-ContribAuthor ">Ettore  Novellino</span>, <span class="hlFld-ContribAuthor ">Annamaria  Biroccio</span>, <span class="hlFld-ContribAuthor ">Luciana  Marinelli</span>, <span class="hlFld-ContribAuthor ">Concetta  Giancola</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the KRAS oncogene: Synthesis, physicochemical and biological evaluation of novel G-Quadruplex DNA binders. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2020,</strong> <em>149 </em>, 105337. <a href="https://doi.org/10.1016/j.ejps.2020.105337" title="DOI URL">https://doi.org/10.1016/j.ejps.2020.105337</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2020.105337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2020.105337%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DTargeting%252Bthe%252BKRAS%252Boncogene%25253A%252BSynthesis%25252C%252Bphysicochemical%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252BG-Quadruplex%252BDNA%252Bbinders%26aulast%3DD%2527Aria%26aufirst%3DFederica%26date%3D2020%26volume%3D149%26spage%3D105337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prakash  Kharel</span>, <span class="hlFld-ContribAuthor ">Sumirtha  Balaratnam</span>, <span class="hlFld-ContribAuthor ">Nathan  Beals</span>, <span class="hlFld-ContribAuthor ">Soumitra  Basu</span>. </span><span class="cited-content_cbyCitation_article-title">The role of RNA G‐quadruplexes in human diseases and therapeutic strategies. </span><span class="cited-content_cbyCitation_journal-name">WIREs RNA</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/wrna.1568" title="DOI URL">https://doi.org/10.1002/wrna.1568</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wrna.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwrna.1568%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520RNA%26atitle%3DThe%252Brole%252Bof%252BRNA%252BG%2525E2%252580%252590quadruplexes%252Bin%252Bhuman%252Bdiseases%252Band%252Btherapeutic%252Bstrategies%26aulast%3DKharel%26aufirst%3DPrakash%26date%3D2020%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luigi E.  Xodo</span>. </span><span class="cited-content_cbyCitation_article-title">Quadruplex nucleic acids in KRAS targeted-cancer therapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 325-359. <a href="https://doi.org/10.1016/bs.armc.2020.04.004" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.04.004%26sid%3Dliteratum%253Aachs%26atitle%3DQuadruplex%252Bnucleic%252Bacids%252Bin%252BKRAS%252Btargeted-cancer%252Btherapy%26aulast%3DXodo%26aufirst%3DLuigi%2BE.%26date%3D2020%26spage%3D325%26epage%3D359%26pub%3DElsevier%26atitle%3DQuadruplex%252BNucleic%252BAcids%252BAs%252BTargets%252BFor%252BMedicinal%252BChemistry%26date%3D2020%26volume%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel  Jodoin</span>, <span class="hlFld-ContribAuthor ">Julie C  Carrier</span>, <span class="hlFld-ContribAuthor ">Nathalie  Rivard</span>, <span class="hlFld-ContribAuthor ">Martin  Bisaillon</span>, <span class="hlFld-ContribAuthor ">Jean-Pierre  Perreault</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplex located in the 5′UTR of the BAG-1 mRNA affects both its cap-dependent and cap-independent translation through global secondary structure maintenance. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (19)
                                     , 10247-10266. <a href="https://doi.org/10.1093/nar/gkz777" title="DOI URL">https://doi.org/10.1093/nar/gkz777</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz777%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DG-quadruplex%252Blocated%252Bin%252Bthe%252B5%2525E2%252580%2525B2UTR%252Bof%252Bthe%252BBAG-1%252BmRNA%252Baffects%252Bboth%252Bits%252Bcap-dependent%252Band%252Bcap-independent%252Btranslation%252Bthrough%252Bglobal%252Bsecondary%252Bstructure%252Bmaintenance%26aulast%3DJodoin%26aufirst%3DRachel%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D19%26spage%3D10247%26epage%3D10266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Ivan V.  Ivanov</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Korolev</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">β-Hydroxylation of anthraquinone derivatives with benzaldehyde oxime as a source of hydroxyl group. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2019,</strong> <em>75 </em>
                                    (43)
                                     , 130623. <a href="https://doi.org/10.1016/j.tet.2019.130623" title="DOI URL">https://doi.org/10.1016/j.tet.2019.130623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2019.130623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2019.130623%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3D%2525CE%2525B2-Hydroxylation%252Bof%252Banthraquinone%252Bderivatives%252Bwith%252Bbenzaldehyde%252Boxime%252Bas%252Ba%252Bsource%252Bof%252Bhydroxyl%252Bgroup%26aulast%3DTikhomirov%26aufirst%3DAlexander%2BS.%26date%3D2019%26volume%3D75%26issue%3D43%26spage%3D130623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. I.  Treshchalin</span>, <span class="hlFld-ContribAuthor ">E. V.  Neborak</span>, <span class="hlFld-ContribAuthor ">H. M.  Treshchalina</span>. </span><span class="cited-content_cbyCitation_article-title">PROSPECTS FOR SEARCHING MULTITARGET TOPOISOMERASE INHIBITORS WITH ANTITUMOR PROPERTIES. </span><span class="cited-content_cbyCitation_journal-name">Siberian journal of oncology</span><span> <strong>2019,</strong> <em>18 </em>
                                    (4)
                                     , 59-66. <a href="https://doi.org/10.21294/1814-4861-2019-18-4-59-66" title="DOI URL">https://doi.org/10.21294/1814-4861-2019-18-4-59-66</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.21294/1814-4861-2019-18-4-59-66&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.21294%2F1814-4861-2019-18-4-59-66%26sid%3Dliteratum%253Aachs%26jtitle%3DSiberian%2520journal%2520of%2520oncology%26atitle%3DPROSPECTS%252BFOR%252BSEARCHING%252BMULTITARGET%252BTOPOISOMERASE%252BINHIBITORS%252BWITH%252BANTITUMOR%252BPROPERTIES%26aulast%3DTreshchalin%26aufirst%3DM.%2BI.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D4%26spage%3D59%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joanna  Sztuba-Solinska</span>, <span class="hlFld-ContribAuthor ">Gabriela  Chavez-Calvillo</span>, <span class="hlFld-ContribAuthor ">Sabrina Elizabeth  Cline</span>. </span><span class="cited-content_cbyCitation_article-title">Unveiling the druggable RNA targets and small molecule therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (10)
                                     , 2149-2165. <a href="https://doi.org/10.1016/j.bmc.2019.03.057" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.03.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DUnveiling%252Bthe%252Bdruggable%252BRNA%252Btargets%252Band%252Bsmall%252Bmolecule%252Btherapeutics%26aulast%3DSztuba-Solinska%26aufirst%3DJoanna%26date%3D2019%26volume%3D27%26issue%3D10%26spage%3D2149%26epage%3D2165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Samuel E  Harvey</span>, <span class="hlFld-ContribAuthor ">Chonghui  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">A high-throughput screen identifies small molecule modulators of alternative splicing by targeting RNA G-quadruplexes. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (7)
                                     , 3667-3679. <a href="https://doi.org/10.1093/nar/gkz036" title="DOI URL">https://doi.org/10.1093/nar/gkz036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz036%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DA%252Bhigh-throughput%252Bscreen%252Bidentifies%252Bsmall%252Bmolecule%252Bmodulators%252Bof%252Balternative%252Bsplicing%252Bby%252Btargeting%252BRNA%252BG-quadruplexes%26aulast%3DZhang%26aufirst%3DJing%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D7%26spage%3D3667%26epage%3D3679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yulia L.  Volodina</span>, <span class="hlFld-ContribAuthor ">Lyubov G.  Dezhenkova</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Victor V.  Tatarskiy</span>, <span class="hlFld-ContribAuthor ">Dmitry N.  Kaluzhny</span>, <span class="hlFld-ContribAuthor ">Anastasia M.  Moisenovich</span>, <span class="hlFld-ContribAuthor ">Mikhail M.  Moisenovich</span>, <span class="hlFld-ContribAuthor ">Alexandra K.  Isagulieva</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Shtil</span>, <span class="hlFld-ContribAuthor ">Vladimir B.  Tsvetkov</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">New anthra[2,3-b]furancarboxamides: A role of positioning of the carboxamide moiety in antitumor properties. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>165 </em>, 31-45. <a href="https://doi.org/10.1016/j.ejmech.2018.12.068" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.12.068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.12.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.12.068%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNew%252Banthra%25255B2%25252C3-b%25255Dfurancarboxamides%25253A%252BA%252Brole%252Bof%252Bpositioning%252Bof%252Bthe%252Bcarboxamide%252Bmoiety%252Bin%252Bantitumor%252Bproperties%26aulast%3DVolodina%26aufirst%3DYulia%2BL.%26date%3D2019%26volume%3D165%26spage%3D31%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chia-Yang  Lin</span>, <span class="hlFld-ContribAuthor ">Atikul  Islam</span>, <span class="hlFld-ContribAuthor ">Claire J.  Su</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>, <span class="hlFld-ContribAuthor ">Show-Mei  Chuang</span>, <span class="hlFld-ContribAuthor ">Pin Ju  Chueh</span>, <span class="hlFld-ContribAuthor ">Yao Li  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Engagement with tNOX (ENOX2) to Inhibit SIRT1 and Activate p53-Dependent and -Independent Apoptotic Pathways by Novel 4,11-Diaminoanthra[2,3-b]furan-5,10-diones in Hepatocellular Carcinoma Cells. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (3)
                                     , 420. <a href="https://doi.org/10.3390/cancers11030420" title="DOI URL">https://doi.org/10.3390/cancers11030420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11030420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11030420%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DEngagement%252Bwith%252BtNOX%252B%252528ENOX2%252529%252Bto%252BInhibit%252BSIRT1%252Band%252BActivate%252Bp53-Dependent%252Band%252B-Independent%252BApoptotic%252BPathways%252Bby%252BNovel%252B4%25252C11-Diaminoanthra%25255B2%25252C3-b%25255Dfuran-5%25252C10-diones%252Bin%252BHepatocellular%252BCarcinoma%252BCells%26aulast%3DLin%26aufirst%3DChia-Yang%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D3%26spage%3D420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sunny Y.  Yang</span>, <span class="hlFld-ContribAuthor ">Pauline  Lejault</span>, <span class="hlFld-ContribAuthor ">Sandy  Chevrier</span>, <span class="hlFld-ContribAuthor ">Romain  Boidot</span>, <span class="hlFld-ContribAuthor ">A. Gordon  Robertson</span>, <span class="hlFld-ContribAuthor ">Judy M. Y.  Wong</span>, <span class="hlFld-ContribAuthor ">David  Monchaud</span>. </span><span class="cited-content_cbyCitation_article-title">Transcriptome-wide identification of transient RNA G-quadruplexes in human cells. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2018,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-07224-8" title="DOI URL">https://doi.org/10.1038/s41467-018-07224-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-07224-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-07224-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DTranscriptome-wide%252Bidentification%252Bof%252Btransient%252BRNA%252BG-quadruplexes%252Bin%252Bhuman%252Bcells%26aulast%3DYang%26aufirst%3DSunny%2BY.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Vladimir B.  Tsvetkov</span>, <span class="hlFld-ContribAuthor ">Dmitry N.  Kaluzhny</span>, <span class="hlFld-ContribAuthor ">Yulia L.  Volodina</span>, <span class="hlFld-ContribAuthor ">George V.  Zatonsky</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">Tri-armed ligands of G-quadruplex on heteroarene-fused anthraquinone scaffolds: Design, synthesis and pre-screening of biological properties. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>159 </em>, 59-73. <a href="https://doi.org/10.1016/j.ejmech.2018.09.054" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.09.054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.09.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.09.054%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTri-armed%252Bligands%252Bof%252BG-quadruplex%252Bon%252Bheteroarene-fused%252Banthraquinone%252Bscaffolds%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bpre-screening%252Bof%252Bbiological%252Bproperties%26aulast%3DTikhomirov%26aufirst%3DAlexander%2BS.%26date%3D2018%26volume%3D159%26spage%3D59%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anita  Donlic</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RNA in mammalian systems with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: RNA</span><span> <strong>2018,</strong> <em>9 </em>
                                    (4)
                                     , e1477. <a href="https://doi.org/10.1002/wrna.1477" title="DOI URL">https://doi.org/10.1002/wrna.1477</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wrna.1477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwrna.1477%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26atitle%3DTargeting%252BRNA%252Bin%252Bmammalian%252Bsystems%252Bwith%252Bsmall%252Bmolecules%26aulast%3DDonlic%26aufirst%3DAnita%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D4%26spage%3De1477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mekala  Gunaratnam</span>, <span class="hlFld-ContribAuthor ">Gavin W.  Collie</span>, <span class="hlFld-ContribAuthor ">Anthony P.  Reszka</span>, <span class="hlFld-ContribAuthor ">Alan K.  Todd</span>, <span class="hlFld-ContribAuthor ">Gary N.  Parkinson</span>, <span class="hlFld-ContribAuthor ">Stephen  Neidle</span>. </span><span class="cited-content_cbyCitation_article-title">A naphthalene diimide G-quadruplex ligand inhibits cell growth and down-regulates BCL-2 expression in an imatinib-resistant gastrointestinal cancer cell line. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (11)
                                     , 2958-2964. <a href="https://doi.org/10.1016/j.bmc.2018.04.050" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.04.050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.04.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.04.050%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DA%252Bnaphthalene%252Bdiimide%252BG-quadruplex%252Bligand%252Binhibits%252Bcell%252Bgrowth%252Band%252Bdown-regulates%252BBCL-2%252Bexpression%252Bin%252Ban%252Bimatinib-resistant%252Bgastrointestinal%252Bcancer%252Bcell%252Bline%26aulast%3DGunaratnam%26aufirst%3DMekala%26date%3D2018%26volume%3D26%26issue%3D11%26spage%3D2958%26epage%3D2964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael I.  Treshalin</span>, <span class="hlFld-ContribAuthor ">E. V  Neborak</span>. </span><span class="cited-content_cbyCitation_article-title">TOPOISOMERASES: FEATURES OF THE ACTION, CLASSIFICATION, CELL FUNCTIONS, INHIBITION, ANTHRAFURANDION. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Oncology</span><span> <strong>2018,</strong> <em>23 </em>
                                    (2)
                                     , 60-70. <a href="https://doi.org/10.18821/1028-9984-2018-23-2-60-70" title="DOI URL">https://doi.org/10.18821/1028-9984-2018-23-2-60-70</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18821/1028-9984-2018-23-2-60-70&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18821%2F1028-9984-2018-23-2-60-70%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Oncology%26atitle%3DTOPOISOMERASES%25253A%252BFEATURES%252BOF%252BTHE%252BACTION%25252C%252BCLASSIFICATION%25252C%252BCELL%252BFUNCTIONS%25252C%252BINHIBITION%25252C%252BANTHRAFURANDION%26aulast%3DTreshalin%26aufirst%3DMichael%2BI.%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D2%26spage%3D60%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Chia-Yang  Lin</span>, <span class="hlFld-ContribAuthor ">Yulia L.  Volodina</span>, <span class="hlFld-ContribAuthor ">Lyubov G.  Dezhenkova</span>, <span class="hlFld-ContribAuthor ">Victor V.  Tatarskiy</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Shtil</span>, <span class="hlFld-ContribAuthor ">Punit  Kaur</span>, <span class="hlFld-ContribAuthor ">Pin Ju  Chueh</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">New antitumor anthra[2,3-b]furan-3-carboxamides: Synthesis and structure-activity relationship. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>148 </em>, 128-139. <a href="https://doi.org/10.1016/j.ejmech.2018.02.027" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.02.027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.02.027%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNew%252Bantitumor%252Banthra%25255B2%25252C3-b%25255Dfuran-3-carboxamides%25253A%252BSynthesis%252Band%252Bstructure-activity%252Brelationship%26aulast%3DTikhomirov%26aufirst%3DAlexander%2BS.%26date%3D2018%26volume%3D148%26spage%3D128%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Primary structure of the 5′-UTR of the human <i>KRAS</i> transcript. The sequence is very rich in guanines (77% GC) and contains 33 GG-runs. (B) Predicted secondary structure of the first 80 nt of 5′-UTR (s-80) given by Mfold.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In s-80 there are three putative nonoverlapping G4 motifs <i>utr</i>-1, <i>utr</i>-z, and <i>utr</i>-c, indicated with colored arrows on the stem-loop structure. According to QGRS, the G4 motifs have a G-score = 21. (C) Cartoon showing the three nonoverlapping G4 RNAs and the structure of a G-tetrad.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Circular dichroism spectra at 25 °C and 90 °C of 5 μM <i>NRAS</i>, <i>utr-</i>1, <i>utr</i>-c, and <i>utr</i>-z RG4s in 50 mM Tris-HCl, pH 7.4, 100 mM KCl (or 100 mM LiCl); for <i>NRAS</i>, the KCl or LiCl concentration was 20 mM. The spectra were obtained in 0.5 cm cuvettes. (B) UV-melting curves of 5 μM <i>utr-</i>1, <i>utr</i>-z, and <i>utr</i>-c in 50 mM Na-cacodylate, pH 7.4, 100 mM KCl or LiCl. The curves were obtained by measuring the absorbance at 296 nm. The absorbance was normalized with the value at 20 °C. (C) EMSA in 50 mM Tris-HCl, pH 7.4, 100 mM KCl of mixtures containing <i>NRAS</i> RG4 and BG4 (lanes 2 and 3 from left) or <i>utr</i>-z RG4 and BG4 (lanes 5 and 6 from left). BG4 was used at 1 and 2 μg, and the gel was 5% polyacrylamide in TBE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Stem-loop structure of the first 80 nt of <i>KRAS</i> 5′-UTR (s-80) given by Mfold. Sequence s-80 is in equilibrium with an alternative structure characterized by three nonoverlapping G4 RNAs. (B) EMSA in 5% polyacrylamide, TB 1X, of mixtures between BG4 and s-80 or RNA hairpin (hair) 5′-GGCCGCCGCAGUGGCGGCGG or ssRNA 5′-UGUAAACAUCCUACACUCAGCU in 50 mM Tris-HCl, pH 7.4, 100 mM KCl (K<sup>+</sup>-buffer) or 100 mM LiCl (Li<sup>+</sup>-buffer). Except mixture 4, all have been prepared in K<sup>+</sup>-buffer. Loading: s-80 (lane 1); s-80 + 1 μg BG4 (lane 2); s-80 + 2 μg BG4 (lane 3); s-80 + 1 μg BG4 in Li<sup>+</sup>-buffer (lane 4); hair (lane 5), hair +1 μg BG4 (lane 6); ssRNA (lane 7); ssRNA + 1 μg BG4 (lane 8). (C) RNase T1 footprinting of s-80. Loading from left to right: alkaline RNA fragmentation (Seq); RNase T1 reaction in urea (urea), in K<sup>+</sup>-buffer (K100), in Li<sup>+</sup>-buffer (Li100), in K<sup>+</sup>-buffer + <b>2a</b> (<i>r</i> = 4) to see if the hairpin = G4 equilibrium is shifted by the ligand (<b>2a</b>), in K<sup>+</sup>-buffer with 150 mM KCl (K150) in Li<sup>+</sup>-buffer with 150 mM (Li150). The bottom band (band 7) matches the mobility of a 7-mer fragment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Fluorescence emission spectra in 50 mM Tris-HCl pH 7.4, 100 mM KCl (K<sup>+</sup>-buffer), of 0.2 μM ThT in the presence of increasing amounts of 80-mer RNA fragment s-80 (top panel) or RNA hairpin 5′-GGCCGCCGCAGUGGCGGCGG (bottom panel). (B) Structure of thioflavin T (ThT) and increase of fluorescence quantum yield at 485 nm of ThT following addition of increasing amounts of s-80 or hairpin in K<sup>+</sup>-buffer and Li<sup>+</sup>-buffer (50 mM Tris-HCl pH 7.4, 100 mM LiCl).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Structures of ATFD <b>2a</b>, <b>1a</b> and ATPD <b>2b</b>, <b>1b</b>. Two amine- or guanidine-alkyl side chains have been attached to the ring system of the molecules. (B) Flow cytometry data showing the uptake of 3 μM compounds by Panc-1 cells after an incubation of 4 h. The two alkylamine compounds <b>2a</b> and <b>2b</b> are taken up more than the guanidine analogues. (C) Δ<i>T</i><sub>M</sub> is the <i>T</i><sub>M</sub> increase of <i>utr-</i>z, <i>utr-</i>1, and <i>utr-</i>c G4 RNAs caused by <b>2a</b> and <b>2b</b> at <i>r</i> = 1, 2, and 4, in 50 mM Tris-HCl, pH 7.4, 100 mM KCl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Biotinylated Anthrafurandiones b-<b>2a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BocNH(CH<sub>2</sub>)<sub>6</sub>NH<sub>2</sub>, HATU, NMM, DMF, rt; overnight; yield 89%; (b) DCM, TFA, rt; 3 h; yield 87%; (c) <b>3</b>, PyBOP, DIPEA, DMSO, rt, 1 h, yield 80%; (d) ethylendiamine, THF, 50 °C, 1.5–2 h, yield 68%.</p></p></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Structure of b-<b>2a</b> conjugated with biotin. (B) Total cellular mRNAs contain <i>KRAS</i> mRNA with G4 RNA in the 5′-UTR. G4-RNA is bound by b-<b>2a</b>. <i>HPRT</i> is not bound or little bound by b-<b>2a</b>. (C) The streptavidin–biotin pull-down assay: b-<b>2a</b> pulls down preferentially G4-motif mRNAs. The recovered mRNA was used to determine by RT-qPCR the amounts of <i>KRAS</i> and <i>HPRT</i> mRNAs. (D) The histograms show the ratio of <i>KRAS</i>/<i>HPRT</i> mRNAs in the input, i.e., total cellular extract (fixed to 1), and in the recovered RNA from a total cellular extract treated with increasing amounts of b-<b>2a</b>. An enrichment of <i>KRAS</i> over <i>HPRT</i> of nearly 3 times was obtained, suggesting that the biotinylated ligands bind to <i>KRAS</i> mRNA within the total cellular extract.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A, B) Dual luciferase assay showing the effect of compounds <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> (0–1.6 μM) on <i>Renilla</i> luciferase driven by the <i>KRAS</i> promoter comprising the 5′-UTR (pRL-KRAS). As reference, we used a plasmid in which <i>Firefly</i> luciferase was driven by the <i>HRAS</i> promoter mutated to abolish its capacity to form quadruplex structures.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The assay shows that <b>2a</b> strongly inhibits luciferase expression. Also <b>2b</b> shows a strong inhibitory activity. (C) When Panc-1 cells are treated with <b>2a</b> at lower concentrations (0–0.33 μM), a dose–response reduction of luciferase is observed. T = <i>Renilla</i>/<i>Firefly</i> in treated cells, C = <i>Renilla</i>/<i>Firefly</i> in untreated cells. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Dual luciferase assay with <i>Renilla</i> luciferase plasmid pRL-CMV-UTR or pRL-CMV and <i>Firefly</i> luciferase plasmid pHRAS-mutA-luc. Left histograms show a strong dose–response luciferase reduction caused by increasing amounts of <b>2a</b> (0–1.6 μM), while the right panel shows that when the <i>KRAS</i> 5′-UTR is removed, the luciferase reduction is not observed, suggesting that the inhibitory effect is mediated by 5′-UTR. T = <i>Renilla</i>/<i>Firefly</i> in treated cells, C = <i>Renilla</i>/<i>Firefly</i> in untreated cells. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Quantitative RT-PCR of <i>KRAS</i> mRNA in Panc-1 cells treated for 6 and 24 h with 1.6 μM <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b>. Ordinate reports the level of KRAS mRNA normalized to β2-microglobulin and <i>HPRT</i>. (B) Western blot determination of <i>KRAS</i> protein and β-actin in Panc-1 cells treated with 1.6 μM <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> for 48 h. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/medium/jm-2017-006225_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Activation of caspase 3/7 in Panc-1 cells treated with <b>2a</b>, <b>2b</b>, <b>1a</b>, and <b>1b</b> (0.8 and 1.6 μM). Compounds <b>2a</b> and <b>2b</b> show activity higher than that of analogues <b>1a</b> and <b>1b</b>. (B) Annexin–propidium assay of Panc-1 cells treated for 72 h with <b>2a</b> (1.6 μM) and <b>2b</b> (1.6 μM). (C) Metabolic activity in Panc-1 cells treated with increasing concentrations of <b>2a</b>, <b>2b</b>, <b>1a</b>. and <b>1b</b>. Compound <b>2a</b> shows the highest activity. *<i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-23/acs.jmedchem.7b00622/20171215/images/large/jm-2017-006225_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00622&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 78 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherfils, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossman, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Der, C. J.</span><span> </span><span class="NLM_article-title">Ras superfamily GEFs and GAPs: validated and tractable targets for cancer therapy?</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1038/nrc2960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrc2960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21102635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=842-857&author=D.+Vigilauthor=J.+Cherfilsauthor=K.+L.+Rossmanauthor=C.+J.+Der&title=Ras+superfamily+GEFs+and+GAPs%3A+validated+and+tractable+targets+for+cancer+therapy%3F&doi=10.1038%2Fnrc2960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ras superfamily GEFs and GAPs: validated and tractable targets for cancer therapy?</span></div><div class="casAuthors">Vigil, Dominico; Cherfils, Jacqueline; Rossman, Kent L.; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">842-857</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is now considerable and increasing evidence for a causal role for aberrant activity of the Ras superfamily of small GTPases in human cancers.  These GTPases function as GDP-GTP-regulated binary switches that control many fundamental cellular processes.  A common mechanism of GTPase deregulation in cancer is the deregulated expression and/or activity of their regulatory proteins, guanine nucleotide exchange factors (GEFs) that promote formation of the active GTP-bound state and GTPase-activating proteins (GAPs) that return the GTPase to its GDP-bound inactive state.  In this Review, we assess the assocn. of GEFs and GAPs with cancer and their druggability for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwbf8B7sLmb7Vg90H21EOLACvtfcHk0lj_vB3vEJB7vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsbrM&md5=b7ad81f2fe0f7f7eaa5f015188f62e85</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2960%26sid%3Dliteratum%253Aachs%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DCherfils%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DRas%2520superfamily%2520GEFs%2520and%2520GAPs%253A%2520validated%2520and%2520tractable%2520targets%2520for%2520cancer%2520therapy%253F%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D842%26epage%3D857%26doi%3D10.1038%2Fnrc2960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Rodriguez-Viciana, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabatier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormick, F.</span><span> </span><span class="NLM_article-title">Signaling specificity by Ras family GTPases is determined by the full spectrum of effectors they regulate</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">4943</span><span class="NLM_x">–</span> <span class="NLM_lpage">4945</span><span class="refDoi"> DOI: 10.1128/MCB.24.11.4943-4954.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1128%2FMCB.24.11.4943-4954.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=15143186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=4943-4945&author=P.+Rodriguez-Vicianaauthor=C.+Sabatierauthor=F.+McCormick&title=Signaling+specificity+by+Ras+family+GTPases+is+determined+by+the+full+spectrum+of+effectors+they+regulate&doi=10.1128%2FMCB.24.11.4943-4954.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling specificity by Ras family GTPases is determined by the full spectrum of effectors they regulate</span></div><div class="casAuthors">Rodriguez-Viciana, Pablo; Sabatier, Celine; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4943-4954</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Ras family GTPases (RFGs) regulate signaling pathways that control multiple biol. processes.  How signaling specificity among the closely related family members is achieved is poorly understood.  We have taken a proteomics approach to signaling by RFGs, and we have analyzed interactions of a panel of RFGs with a comprehensive group of known and potential effectors.  We have found remarkable differences in the ability of RFGs to regulate the various isoforms of known effector families.  We have also identified several proteins as novel effectors of RFGs with differential binding specificities to the various RFGs.  We propose that specificity among RFGs is achieved by the differential regulation of combinations of effector families as well as by the selective regulation of different isoforms within an effector family.  An understanding of this new level of complexity in the signaling pathways regulated by RFGs is necessary to understand how they carry out their many cellular functions.  It will also likely have crit. implications in the treatment of human diseases such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyMRsdLoV7GLVg90H21EOLACvtfcHk0lj_vB3vEJB7vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhsbw%253D&md5=472eb17841cba28ce667c34cd07cb25e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.4943-4954.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.4943-4954.2004%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Viciana%26aufirst%3DP.%26aulast%3DSabatier%26aufirst%3DC.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DSignaling%2520specificity%2520by%2520Ras%2520family%2520GTPases%2520is%2520determined%2520by%2520the%2520full%2520spectrum%2520of%2520effectors%2520they%2520regulate%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D4943%26epage%3D4945%26doi%3D10.1128%2FMCB.24.11.4943-4954.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wortzel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seger, R.</span><span> </span><span class="NLM_article-title">The ERK cascade: distinct functions within various subcellular organelles</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1177/1947601911407328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1177%2F1947601911407328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21779493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=195-209&author=I.+Wortzelauthor=R.+Seger&title=The+ERK+cascade%3A+distinct+functions+within+various+subcellular+organelles&doi=10.1177%2F1947601911407328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The ERK cascade: Distinct functions within various subcellular organelles</span></div><div class="casAuthors">Wortzel, Inbal; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">195-209</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  The extracellular signal-regulated kinase 1/2 (ERK1/2) cascade is a central signaling pathway that regulates a wide variety of stimulated cellular processes, including mainly proliferation, differentiation, and survival, but apoptosis and stress response as well.  The ability of this linear cascade to induce so many distinct and even opposing effects after various stimulations raises the question as to how the signaling specificity of the cascade is regulated.  Over the past years, several specificity-mediating mechanisms have been elucidated, including temporal regulation, scaffolding interactions, crosstalks with other signaling components, substrate competition, and multiple components in each tier of the cascade.  In addn., spatial regulation of various components of the cascade is probably one of the main ways by which signals can be directed to some downstream targets and not to others.  Here, te authors describe 1st the components of the ERK1/2 cascade and their mode of regulation by kinases, phosphatases, and scaffold proteins.  In the 2nd part, the authors focus on the role of MEK1/2 and ERK1/2 compartmentalization in the nucleus, mitochondria, endosomes, plasma membrane, cytoskeleton, and Golgi app.  The authors explain that this spatial distribution may direct ERK1/2 signals to regulate the organelles' activities.  However, it can also direct the activity of the cascade's components to the outer surface of the organelles in order to bring them to close proximity to specific cytoplasmic targets.  The authors conclude that the dynamic localization of the ERK1/2 cascade components is an important regulatory mechanism in detg. the signaling specificity of the cascade, and its understanding should shed new light on the understanding of many stimulus-dependent processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7L1MPZYcUQ7Vg90H21EOLACvtfcHk0lhC7r-cy8h77w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7w%253D&md5=0321854677455eb57552cfc371146324</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1177%2F1947601911407328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911407328%26sid%3Dliteratum%253Aachs%26aulast%3DWortzel%26aufirst%3DI.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DThe%2520ERK%2520cascade%253A%2520distinct%2520functions%2520within%2520various%2520subcellular%2520organelles%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D195%26epage%3D209%26doi%3D10.1177%2F1947601911407328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Castellano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downward, J.</span><span> </span><span class="NLM_article-title">Ras interaction with PI3K: more than just another effector pathway</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">274</span><span class="refDoi"> DOI: 10.1177/1947601911408079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1177%2F1947601911408079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21779497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=261-274&author=E.+Castellanoauthor=J.+Downward&title=Ras+interaction+with+PI3K%3A+more+than+just+another+effector+pathway&doi=10.1177%2F1947601911408079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ras interaction with P13K: more than just another effector pathway</span></div><div class="casAuthors">Castellano, Esther; Downward, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-274</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  RAS proteins are small GTPases known for their involvement in oncogenesis: around 25% of human tumors present mutations in a member of this family.  RAS operates in a complex signaling network with multiple activators and effectors, which allows them to regulate many cellular functions such as cell proliferation, differentiation, apoptosis, and senescence.  Phosphatidylinositol 3-kinase (PI3K) is one of the main effector pathways of RAS, regulating cell growth, cell cycle entry, cell survival, cytoskeleton reorganization, and metab.  However, it is the involvement of this pathway in human tumors that has attracted most attention.  PI3K has proven to be necessary for RAS-induced transformation in vitro, and more importantly, mice with mutations in the PI3K catalytic subunit p110α that block its ability to interact with RAS are highly resistant to endogenous oncogenic KRAS-induced lung tumorigenesis and HRAS-induced skin carcinogenesis.  These animals also have a delayed development of the lymphatic vasculature.  Many PI3K inhibitors have been developed that are now in clin. trials.  However, it is a complex pathway with many feedback loops, and interactions with other pathways make the results of its inhibition hard to predict.  Combined therapy with another RAS-regulated pathway such as RAF/MEK/ERK may be the most effective way to treat cancer, at least in animal models mimicking the human disease.  In this review, we will summarize current knowledge about how RAS regulates one of its best-known effectors, PI3K.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLhXMI50nEbrVg90H21EOLACvtfcHk0lhC7r-cy8h77w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7c%253D&md5=2293eaf0d990f7638b4949dc88e744d9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1177%2F1947601911408079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911408079%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DE.%26aulast%3DDownward%26aufirst%3DJ.%26atitle%3DRas%2520interaction%2520with%2520PI3K%253A%2520more%2520than%2520just%2520another%2520effector%2520pathway%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D261%26epage%3D274%26doi%3D10.1177%2F1947601911408079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Schubbert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span> </span><span class="NLM_article-title">Hyperactive Ras in developmental disorders and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1038/nrc2109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrc2109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17384584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlyhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=295-308&author=S.+Schubbertauthor=K.+Shannonauthor=G.+Bollag&title=Hyperactive+Ras+in+developmental+disorders+and+cancer&doi=10.1038%2Fnrc2109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hyperactive Ras in developmental disorders and cancer</span></div><div class="casAuthors">Schubbert, Suzanne; Shannon, Kevin; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-308</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ras genes are the most common targets for somatic gain-of-function mutations in human cancer.  Recently, germline mutations that affect components of the Ras-Raf-mitogen-activated and extracellular-signal regulated kinase kinase (MEK)-extracellular signal-regulated kinase (ERK) pathway were shown to cause several developmental disorders, including Nonan, Costello and cardio-facio-cutaneous syndromes.  Many of these mutant alleles encode proteins with aberrant biochem. and functional properties.  Here we will discuss the implications of germline mutations in the Ras-Raf-MEK-ERK pathway for understanding normal developmental processes and cancer pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRvjqrp1ho7Vg90H21EOLACvtfcHk0lgnzWRibJ4rDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlyhtLs%253D&md5=25fcaa959dae5e8512b959936bcbb347</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc2109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2109%26sid%3Dliteratum%253Aachs%26aulast%3DSchubbert%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DK.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DHyperactive%2520Ras%2520in%2520developmental%2520disorders%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D295%26epage%3D308%26doi%3D10.1038%2Fnrc2109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Ryan, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardeesy, N.</span><span> </span><span class="NLM_article-title">Pancreatic adenocarcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">2140</span><span class="NLM_x">–</span> <span class="NLM_lpage">2141</span><span class="refDoi"> DOI: 10.1056/NEJMra1404198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1056%2FNEJMra1404198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25427123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A280%3ADC%252BC2MzhtFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2014&pages=2140-2141&author=D.+P.+Ryanauthor=T.+S.+Hongauthor=N.+Bardeesy&title=Pancreatic+adenocarcinoma&doi=10.1056%2FNEJMra1404198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic adenocarcinoma</span></div><div class="casAuthors">Ryan David P; Hong Theodore S; Bardeesy Nabeel</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2140-1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXWXu7vS1YVeR_TJajKxJDfW6udTcc2eaQCt78OkiUt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzhtFCkug%253D%253D&md5=4ca27851b40f9613b10577597abcf1ce</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1404198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1404198%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DD.%2BP.%26aulast%3DHong%26aufirst%3DT.%2BS.%26aulast%3DBardeesy%26aufirst%3DN.%26atitle%3DPancreatic%2520adenocarcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D371%26spage%3D2140%26epage%3D2141%26doi%3D10.1056%2FNEJMra1404198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Prior, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">A comprehensive survey of Ras mutations in cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">2457</span><span class="NLM_x">–</span> <span class="NLM_lpage">2467</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F0008-5472.CAN-11-2612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22589270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2457-2467&author=I.+A.+Priorauthor=P.+D.+Lewisauthor=C.+Mattos&title=A+comprehensive+survey+of+Ras+mutations+in+cancer&doi=10.1158%2F0008-5472.CAN-11-2612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Survey of Ras Mutations in Cancer</span></div><div class="casAuthors">Prior, Ian A.; Lewis, Paul D.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2457-2467</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Ras, that promote oncogenesis when they are mutationally activated at codon 12, 13, or 61.  Although there is a high degree of similarity among the isoforms, K-Ras mutations are far more frequently obsd. in cancer, and each isoform displays preferential coupling to particular cancer types.  We examd. the mutational spectra of Ras isoforms curated from large-scale tumor profiling and found that each isoform exhibits surprisingly distinctive codon mutation and amino-acid substitution biases.  These findings were unexpected given that these mutations occur in regions that share 100% amino-acid sequence identity among the 3 isoforms.  Of importance, many of these mutational biases were not due to differences in exposure to mutagens, because the patterns were still evident when compared within specific cancer types.  We discuss potential genetic and epigenetic mechanisms, as well as isoform-specific differences in protein structure and signaling, that may promote these distinct mutation patterns and differential coupling to specific cancers.  Cancer Res; 72(10); 2457-67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg2lqiicosOLVg90H21EOLACvtfcHk0lgnzWRibJ4rDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D&md5=6d1894847015468f77c8b08b57c6d62a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2612%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DI.%2BA.%26aulast%3DLewis%26aufirst%3DP.%2BD.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DA%2520comprehensive%2520survey%2520of%2520Ras%2520mutations%2520in%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2457%26epage%3D2467%26doi%3D10.1158%2F0008-5472.CAN-11-2612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Karnoub, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span> </span><span class="NLM_article-title">Ras oncogenes: split personalities</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">517</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1038/nrm2438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrm2438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18568040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFGnsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=517-531&author=A.+E.+Karnoubauthor=R.+A.+Weinberg&title=Ras+oncogenes%3A+split+personalities&doi=10.1038%2Fnrm2438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ras oncogenes: split personalities</span></div><div class="casAuthors">Karnoub, Antoine E.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-531</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Extensive research over the past 30 years has revealed the involvement of Ras not only in tumorigenesis but also in many developmental disorders.  The complexity of the mol. and cell biol. mechanisms of action of Ras proteins indicates that much remains to be learns.  Extensive research on the Ras proteins and their functions in cell physiol. over the past 30 years has led to numerous insights that have revealed the involvement of Ras not only in tumorigenesis but also in many developmental disorders.  Despite great strides in our understanding of the mol. and cellular mechanisms of action of the Ras proteins, the expanding roster of their downstream effectors and the complexity of the signaling cascades that they regulate indicate that much remains to be learns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocptUdlhqaM7Vg90H21EOLACvtfcHk0lg1Db5JspL-nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFGnsr0%253D&md5=9de78b17b953b2560238d7a98f3bea6a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrm2438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2438%26sid%3Dliteratum%253Aachs%26aulast%3DKarnoub%26aufirst%3DA.%2BE.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DRas%2520oncogenes%253A%2520split%2520personalities%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D517%26epage%3D531%26doi%3D10.1038%2Fnrm2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Collins, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brisset, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbán, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbán, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rakshit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flannagan, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adsay, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasca di Magliano, M.</span><span> </span><span class="NLM_article-title">Oncogenic Kras is required for both the initiation and maintenance of pancreatic cancer in mice</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">653</span><span class="refDoi"> DOI: 10.1172/JCI59227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1172%2FJCI59227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22232209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XitVelsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2012&pages=639-653&author=M.+A.+Collinsauthor=F.+Bednarauthor=Y.+Zhangauthor=J.+C.+Brissetauthor=S.+Galb%C3%A1nauthor=C.+J.+Galb%C3%A1nauthor=S.+Rakshitauthor=K.+S.+Flannaganauthor=N.+V.+Adsayauthor=M.+Pasca+di+Magliano&title=Oncogenic+Kras+is+required+for+both+the+initiation+and+maintenance+of+pancreatic+cancer+in+mice&doi=10.1172%2FJCI59227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Kras is required for both the initiation and maintenance of pancreatic cancer in mice</span></div><div class="casAuthors">Collins, Meredith A.; Bednar, Filip; Zhang, Yaqing; Brisset, Jean-Christophe; Galban, Stefanie; Galban, Craig J.; Rakshit, Sabita; Flannagan, Karen S.; Adsay, N. Volkan; di Magliano, Marina Pasca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">639-653</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Pancreatic cancer is almost invariably assocd. with mutations in the KRAS gene, most commonly KRASG12D, that result in a dominant-active form of the KRAS GTPase.  However, how KRAS mutations promote pancreatic carcinogenesis is not fully understood, and whether oncogenic KRAS is required for the maintenance of pancreatic cancer has not been established.  To address these questions, we generated two mouse models of pancreatic tumorigenesis: mice transgenic for inducible KrasG12D, which allows for inducible, pancreas-specific, and reversible expression of the oncogenic KrasG12D, with or without inactivation of one allele of the tumor suppressor gene p53.  Here, we report that, early in tumorigenesis, induction of oncogenic KrasG12D reversibly altered normal epithelial differentiation following tissue damage, leading to precancerous lesions.  Inactivation of KrasG12D in established precursor lesions and during progression to cancer led to regression of the lesions, indicating that KrasG12D was required for tumor cell survival.  Strikingly, during all stages of carcinogenesis, KrasG12D upregulated Hedgehog signaling, inflammatory pathways, and several pathways known to mediate paracrine interactions between epithelial cells and their surrounding microenvironment, thus promoting formation and maintenance of the fibroinflammatory stroma that plays a pivotal role in pancreatic cancer.  Our data establish that epithelial KrasG12D influences multiple cell types to drive pancreatic tumorigenesis and is essential for tumor maintenance.  They also strongly support the notion that inhibiting KrasG12D, or its downstream effectors, could provide a new approach for the treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA_MsyfjFbgrVg90H21EOLACvtfcHk0lg1Db5JspL-nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitVelsrc%253D&md5=576556a0569dd9d1517acbe405fe64c5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1172%2FJCI59227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI59227%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DM.%2BA.%26aulast%3DBednar%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBrisset%26aufirst%3DJ.%2BC.%26aulast%3DGalb%25C3%25A1n%26aufirst%3DS.%26aulast%3DGalb%25C3%25A1n%26aufirst%3DC.%2BJ.%26aulast%3DRakshit%26aufirst%3DS.%26aulast%3DFlannagan%26aufirst%3DK.%2BS.%26aulast%3DAdsay%26aufirst%3DN.%2BV.%26aulast%3DPasca%2Bdi%2BMagliano%26aufirst%3DM.%26atitle%3DOncogenic%2520Kras%2520is%2520required%2520for%2520both%2520the%2520initiation%2520and%2520maintenance%2520of%2520pancreatic%2520cancer%2520in%2520mice%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2012%26volume%3D122%26spage%3D639%26epage%3D653%26doi%3D10.1172%2FJCI59227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Kanda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthaei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maitra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goggins, M.</span><span> </span><span class="NLM_article-title">Presence of somatic mutations in most early-stage pancreatic intraepithelial neoplasia</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">142</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">733</span><span class="refDoi"> DOI: 10.1053/j.gastro.2011.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1053%2Fj.gastro.2011.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2012&pages=730-733&author=M.+Kandaauthor=H.+Matthaeiauthor=J.+Wuauthor=S.+M.+Hongauthor=J.+Yuauthor=M.+Borgesauthor=R.+H.+Hrubanauthor=A.+Maitraauthor=K.+Kinzlerauthor=B.+Vogelsteinauthor=M.+Goggins&title=Presence+of+somatic+mutations+in+most+early-stage+pancreatic+intraepithelial+neoplasia&doi=10.1053%2Fj.gastro.2011.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2011.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2011.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DKanda%26aufirst%3DM.%26aulast%3DMatthaei%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%2BM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DBorges%26aufirst%3DM.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DKinzler%26aufirst%3DK.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DGoggins%26aufirst%3DM.%26atitle%3DPresence%2520of%2520somatic%2520mutations%2520in%2520most%2520early-stage%2520pancreatic%2520intraepithelial%2520neoplasia%26jtitle%3DGastroenterology%26date%3D2012%26volume%3D142%26spage%3D730%26epage%3D733%26doi%3D10.1053%2Fj.gastro.2011.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Vincent, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruban, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goggins, M.</span><span> </span><span class="NLM_article-title">Pancreatic cancer</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">378</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0140-6736(10)62307-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FS0140-6736%2810%2962307-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21620466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A280%3ADC%252BC3MjlvFOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2011&pages=607-620&author=A.+Vincentauthor=J.+Hermanauthor=R.+Schulickauthor=R.+H.+Hrubanauthor=M.+Goggins&title=Pancreatic+cancer&doi=10.1016%2FS0140-6736%2810%2962307-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic cancer</span></div><div class="casAuthors">Vincent Audrey; Herman Joseph; Schulick Rich; Hruban Ralph H; Goggins Michael</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">9791</span>),
    <span class="NLM_cas:pages">607-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Substantial progress has been made in our understanding of the biology of pancreatic cancer, and advances in patients' management have also taken place.  Evidence is beginning to show that screening first-degree relatives of individuals with several family members affected by pancreatic cancer can identify non-invasive precursors of this malignant disease.  The incidence of and number of deaths caused by pancreatic tumours have been gradually rising, even as incidence and mortality of other common cancers have been declining.  Despite developments in detection and management of pancreatic cancer, only about 4% of patients will live 5 years after diagnosis.  Survival is better for those with malignant disease localised to the pancreas, because surgical resection at present offers the only chance of cure.  Unfortunately, 80-85% of patients present with advanced unresectable disease.  Furthermore, pancreatic cancer responds poorly to most chemotherapeutic agents.  Hence, we need to understand the biological mechanisms that contribute to development and progression of pancreatic tumours.  In this Seminar we will discuss the most common and deadly form of pancreatic cancer, pancreatic ductal adenocarcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_Px7QoMFLVGhHGZei4OW9fW6udTcc2eYrcRS4oQ-zYLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjlvFOmsg%253D%253D&md5=fb7f79c8c02f2f2bad5e5ad7a9b09096</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2810%2962307-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252810%252962307-0%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DA.%26aulast%3DHerman%26aufirst%3DJ.%26aulast%3DSchulick%26aufirst%3DR.%26aulast%3DHruban%26aufirst%3DR.%2BH.%26aulast%3DGoggins%26aufirst%3DM.%26atitle%3DPancreatic%2520cancer%26jtitle%3DLancet%26date%3D2011%26volume%3D378%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0140-6736%2810%2962307-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Ying, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmelman, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssiotis, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher-Sananikone, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locasale, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coloff, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paik, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guimaraes, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hezel, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissleder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span> </span><span class="NLM_article-title">Oncogenic Kras maintains pancreatic tumors through regulation of anabolic glucose metabolism</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">656</span><span class="NLM_x">–</span> <span class="NLM_lpage">670</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.cell.2012.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22541435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Kjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=656-670&author=H.+Yingauthor=A.+C.+Kimmelmanauthor=C.+A.+Lyssiotisauthor=S.+Huaauthor=G.+C.+Chuauthor=E.+Fletcher-Sananikoneauthor=J.+W.+Locasaleauthor=J.+Sonauthor=H.+Zhangauthor=J.+L.+Coloffauthor=H.+Yanauthor=W.+Wangauthor=S.+Chenauthor=A.+Vialeauthor=H.+Zhengauthor=J.+H.+Paikauthor=C.+Limauthor=A.+R.+Guimaraesauthor=E.+S.+Martinauthor=J.+Changauthor=A.+F.+Hezelauthor=S.+R.+Perryauthor=J.+Huauthor=B.+Ganauthor=Y.+Xiaoauthor=J.+M.+Asaraauthor=R.+Weisslederauthor=Y.+A.+Wangauthor=L.+Chinauthor=L.+C.+Cantleyauthor=R.+A.+DePinho&title=Oncogenic+Kras+maintains+pancreatic+tumors+through+regulation+of+anabolic+glucose+metabolism&doi=10.1016%2Fj.cell.2012.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Kras maintains pancreatic tumors through regulation of anabolic glucose metabolism</span></div><div class="casAuthors">Ying, Haoqiang; Kimmelman, Alec C.; Lyssiotis, Costas A.; Hua, Sujun; Chu, Gerald C.; Fletcher-Sananikone, Eliot; Locasale, Jason W.; Son, Jaekyoung; Zhang, Hailei; Coloff, Jonathan L.; Yan, Haiyan; Wang, Wei; Chen, Shujuan; Viale, Andrea; Zheng, Hongwu; Paik, Ji-hye; Lim, Carol; Guimaraes, Alexander R.; Martin, Eric S.; Chang, Jeffery; Hezel, Aram F.; Perry, Samuel R.; Hu, Jian; Gan, Boyi; Xiao, Yonghong; Asara, John M.; Weissleder, Ralph; Wang, Y. Alan; Chin, Lynda; Cantley, Lewis C.; DePinho, Ronald A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">656-670</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tumor maintenance relies on continued activity of driver oncogenes, although their rate-limiting role is highly context dependent.  Oncogenic Kras mutation is the signature event in pancreatic ductal adenocarcinoma (PDAC), serving a crit. role in tumor initiation.  Here, an inducible KrasG12D-driven PDAC mouse model establishes that advanced PDAC remains strictly dependent on KrasG12D expression.  Transcriptome and metabolomic analyses indicate that KrasG12D serves a vital role in controlling tumor metab. through stimulation of glucose uptake and channeling of glucose intermediates into the hexosamine biosynthesis and pentose phosphate pathways (PPP).  These studies also reveal that oncogenic Kras promotes ribose biogenesis.  Unlike canonical models, we demonstrate that KrasG12D drives glycolysis intermediates into the nonoxidative PPP, thereby decoupling ribose biogenesis from NADP/NADPH-mediated redox control.  Together, this work provides in vivo mechanistic insights into how oncogenic Kras promotes metabolic reprogramming in native tumors and illuminates potential metabolic targets that can be exploited for therapeutic benefit in PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0n15htmLPUrVg90H21EOLACvtfcHk0ljJVlGPNR3T_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Kjs7k%253D&md5=c8474034804faf04fb8bd108dd958540</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DChu%26aufirst%3DG.%2BC.%26aulast%3DFletcher-Sananikone%26aufirst%3DE.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DColoff%26aufirst%3DJ.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DPaik%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DC.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DMartin%26aufirst%3DE.%2BS.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DHezel%26aufirst%3DA.%2BF.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DGan%26aufirst%3DB.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26atitle%3DOncogenic%2520Kras%2520maintains%2520pancreatic%2520tumors%2520through%2520regulation%2520of%2520anabolic%2520glucose%2520metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D656%26epage%3D670%26doi%3D10.1016%2Fj.cell.2012.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssiotis, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ying, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ligorio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrone, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullarky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyh-Chang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleming, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardeesy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haigis, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmelman, A. C.</span><span> </span><span class="NLM_article-title">Glutamine supports pancreatic cancer growth through a KRAS-regulated metabolic pathway</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">496</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature12040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnature12040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23535601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFyjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=101-105&author=J.+Sonauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=X.+Wangauthor=S.+Huaauthor=M.+Ligorioauthor=R.+M.+Pereraauthor=C.+R.+Ferroneauthor=E.+Mullarkyauthor=N.+Shyh-Changauthor=Y.+Kangauthor=J.+B.+Flemingauthor=N.+Bardeesyauthor=J.+M.+Asaraauthor=M.+C.+Haigisauthor=R.+A.+DePinhoauthor=L.+C.+Cantleyauthor=A.+C.+Kimmelman&title=Glutamine+supports+pancreatic+cancer+growth+through+a+KRAS-regulated+metabolic+pathway&doi=10.1038%2Fnature12040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine supports pancreatic cancer growth through a KRAS-regulated metabolic pathway</span></div><div class="casAuthors">Son, Jaekyoung; Lyssiotis, Costas A.; Ying, Haoqiang; Wang, Xiaoxu; Hua, Sujun; Ligorio, Matteo; Perera, Rushika M.; Ferrone, Cristina R.; Mullarky, Edouard; Ng, Shyh-Chang; Kang, Ya'an; Fleming, Jason B.; Bardeesy, Nabeel; Asara, John M.; Haigis, Marcia C.; DePinho, Ronald A.; Cantley, Lewis C.; Kimmelman, Alec C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7443</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells have metabolic dependencies that distinguish them from their normal counterparts.  Among these dependencies is an increased use of the amino acid glutamine to fuel anabolic processes.  Indeed, the spectrum of glutamine-dependent tumors and the mechanisms whereby glutamine supports cancer metab. remain areas of active investigation.  Here we report the identification of a non-canonical pathway of glutamine use in human pancreatic ductal adenocarcinoma (PDAC) cells that is required for tumor growth.  Whereas most cells use glutamate dehydrogenase (GLUD1) to convert glutamine-derived glutamate into α-ketoglutarate in the mitochondria to fuel the tricarboxylic acid cycle, PDAC relies on a distinct pathway in which glutamine-derived aspartate is transported into the cytoplasm where it can be converted into oxaloacetate by aspartate transaminase (GOT1).  Subsequently, this oxaloacetate is converted into malate and then pyruvate, ostensibly increasing the NADPH/NADP+ ratio which can potentially maintain the cellular redox state.  Importantly, PDAC cells are strongly dependent on this series of reactions, as glutamine deprivation or genetic inhibition of any enzyme in this pathway leads to an increase in reactive oxygen species and a redn. in reduced glutathione.  Moreover, knockdown of any component enzyme in this series of reactions also results in a pronounced suppression of PDAC growth in vitro and in vivo.  Furthermore, we establish that the reprogramming of glutamine metab. is mediated by oncogenic KRAS, the signature genetic alteration in PDAC, through the transcriptional upregulation and repression of key metabolic enzymes in this pathway.  The essentiality of this pathway in PDAC and the fact that it is dispensable in normal cells may provide novel therapeutic approaches to treat these refractory tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzWAEWZvCWcbVg90H21EOLACvtfcHk0ljJVlGPNR3T_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFyjsrc%253D&md5=162af18c1102745226aa2096179f1cc6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature12040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12040%26sid%3Dliteratum%253Aachs%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DLigorio%26aufirst%3DM.%26aulast%3DPerera%26aufirst%3DR.%2BM.%26aulast%3DFerrone%26aufirst%3DC.%2BR.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DShyh-Chang%26aufirst%3DN.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DFleming%26aufirst%3DJ.%2BB.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DHaigis%26aufirst%3DM.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26atitle%3DGlutamine%2520supports%2520pancreatic%2520cancer%2520growth%2520through%2520a%2520KRAS-regulated%2520metabolic%2520pathway%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fnature12040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Weinstein, I. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsher, D.</span><span> </span><span class="NLM_article-title">Oncogene addiction</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3077</span><span class="NLM_x">–</span> <span class="NLM_lpage">3080</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-3293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F0008-5472.CAN-07-3293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18451130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltlSmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3077-3080&author=I.+B.+Weinsteinauthor=A.+Joeauthor=D.+Felsher&title=Oncogene+addiction&doi=10.1158%2F0008-5472.CAN-07-3293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogene Addiction</span></div><div class="casAuthors">Weinstein, I. Bernard; Joe, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3077-3080</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Cancer cells contain multiple genetic and epigenetic abnormalities.  Despite this complexity, their growth and survival can often be impaired by the inactivation of a single oncogene.  This phenomenon, called "oncogene addiction," provides a rationale for mol. targeted therapy.  The efficacy of this strategy requires novel methods, including integrative genomics and systems biol., to identify the state of oncogene addiction (i.e., the "Achilles heel") in specific cancers.  Combination therapy may also be required to prevent the escape of cancers from a given state of oncogene addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobVcTGxEyDNLVg90H21EOLACvtfcHk0lht0BI6sSkQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltlSmtrk%253D&md5=fc62277a0200431335f79c420ebb02a6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-3293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-3293%26sid%3Dliteratum%253Aachs%26aulast%3DWeinstein%26aufirst%3DI.%2BB.%26aulast%3DJoe%26aufirst%3DA.%26aulast%3DFelsher%26aufirst%3DD.%26atitle%3DOncogene%2520addiction%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3077%26epage%3D3080%26doi%3D10.1158%2F0008-5472.CAN-07-3293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucharski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Selective inhibition of oncogenic KRAS output with small molecules targeting the inactive state</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+inhibition+of+oncogenic+KRAS+output+with+small+molecules+targeting+the+inactive+state&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0lht0BI6sSkQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520oncogenic%2520KRAS%2520output%2520with%2520small%2520molecules%2520targeting%2520the%2520inactive%2520state%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wilson, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tolias, P.</span><span> </span><span class="NLM_article-title">Recent advances in cancer drug discovery targeting RAS</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1915</span><span class="NLM_x">–</span> <span class="NLM_lpage">1919</span><span class="refDoi"> DOI: 10.1016/j.drudis.2016.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.drudis.2016.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27506872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlaiu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1915-1919&author=C.+Y.+Wilsonauthor=P.+Tolias&title=Recent+advances+in+cancer+drug+discovery+targeting+RAS&doi=10.1016%2Fj.drudis.2016.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in cancer drug discovery targeting RAS</span></div><div class="casAuthors">Wilson, Candice Y.; Tolias, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1915-1919</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mutated RAS is present in 30% of human tumors, appearing in 90% of pancreatic, 45% of colon and 35% of lung cancers.  These high occurrences make RAS one of the most important drug targets in oncol.  Three decades of effort to target RAS have been unsuccessful in generating drug therapies suggesting that it might represent an 'undruggable' target.  However, recent reports highlighting new approaches for targeting RAS have uncovered more information on protein structure and identified new binding pockets.  Efforts to target the KRAS G12C mutation specifically have shown promising results whereas other approaches have targeted various protein complexes.  These advances could lead to development of new effective cancer drugs targeting RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJnoLMitHbsrVg90H21EOLACvtfcHk0ljNvYpxens6xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlaiu73P&md5=5bcad00948b76112248f9ebc332a8d73</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DC.%2BY.%26aulast%3DTolias%26aufirst%3DP.%26atitle%3DRecent%2520advances%2520in%2520cancer%2520drug%2520discovery%2520targeting%2520RAS%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1915%26epage%3D1919%26doi%3D10.1016%2Fj.drudis.2016.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Stephen, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esposito, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagni, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormick, F.</span><span> </span><span class="NLM_article-title">Dragging ras back in the ring</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1016/j.ccr.2014.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.ccr.2014.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24651010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVWhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=272-281&author=A.+G.+Stephenauthor=D.+Espositoauthor=R.+K.+Bagniauthor=F.+McCormick&title=Dragging+ras+back+in+the+ring&doi=10.1016%2Fj.ccr.2014.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Dragging Ras Back in the Ring</span></div><div class="casAuthors">Stephen, Andrew G.; Esposito, Dominic; Bagni, Rachel K.; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">272-281</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Ras proteins play a major role in human cancers but have not yielded to therapeutic attack.  Ras-driven cancers are among the most difficult to treat and often excluded from therapies.  The Ras proteins have been termed "undruggable," based on failures from an era in which understanding of signaling transduction, feedback loops, redundancy, tumor heterogeneity, and Ras' oncogenic role was poor.  Structures of Ras oncoproteins bound to their effectors or regulators are unsolved, and it is unknown precisely how Ras proteins activate their downstream targets.  These knowledge gaps have impaired development of therapeutic strategies.  A better understanding of Ras biol. and biochem., coupled with new ways of targeting undruggable proteins, is likely to lead to new ways of defeating Ras-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsJEUXbjmDe7Vg90H21EOLACvtfcHk0ljNvYpxens6xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVWhtLY%253D&md5=1ecb33c95db7d5d6f910b9e2af2ec44e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DStephen%26aufirst%3DA.%2BG.%26aulast%3DEsposito%26aufirst%3DD.%26aulast%3DBagni%26aufirst%3DR.%2BK.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DDragging%2520ras%2520back%2520in%2520the%2520ring%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D272%26epage%3D281%26doi%3D10.1016%2Fj.ccr.2014.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1038/nrd.2016.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrd.2016.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27469033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=771-785&author=J.+M.+Ostremauthor=K.+M.+Shokat&title=Direct+small-molecule+inhibitors+of+KRAS%3A+from+structural+insights+to+mechanism-based+design&doi=10.1038%2Fnrd.2016.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span></div><div class="casAuthors">Ostrem, Jonathan M. L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">771-785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer.  In addn. to holding this distinction, unsuccessful attempts to target this protein have led to the characterization of RAS as 'undruggable'.  However, recent advances in technol. and novel approaches to drug discovery have renewed hope that a direct KRAS inhibitor may be on the horizon.  In this Review, we provide an in-depth anal. of the structure, dynamics, mutational activation and inactivation, and signalling mechanisms of RAS.  From this perspective, we then consider potential mechanisms of action for effective RAS inhibitors.  Finally, we examine each of the many recent reports of direct RAS inhibitors and discuss promising avenues for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBobrhZLF3V7Vg90H21EOLACvtfcHk0ljNvYpxens6xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK&md5=b0832c7ef45cc0555e11c59baccc5abd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.139%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DDirect%2520small-molecule%2520inhibitors%2520of%2520KRAS%253A%2520from%2520structural%2520insights%2520to%2520mechanism-based%2520design%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D771%26epage%3D785%26doi%3D10.1038%2Fnrd.2016.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kiessling, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogler, G.</span><span> </span><span class="NLM_article-title">Targeting the ras pathwayby mitogene-activated protein kinase inhibitors</span> <span class="citation_source-journal">Swiss Med. Wkly.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">w14207</span><span class="refDoi"> DOI: 10.4414/smw.2015.14207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.4414%2Fsmw.2015.14207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26691679" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2015&pages=w14207&author=M.+K.+Kiesslingauthor=G.+Rogler&title=Targeting+the+ras+pathwayby+mitogene-activated+protein+kinase+inhibitors&doi=10.4414%2Fsmw.2015.14207"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4414%2Fsmw.2015.14207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4414%252Fsmw.2015.14207%26sid%3Dliteratum%253Aachs%26aulast%3DKiessling%26aufirst%3DM.%2BK.%26aulast%3DRogler%26aufirst%3DG.%26atitle%3DTargeting%2520the%2520ras%2520pathwayby%2520mitogene-activated%2520protein%2520kinase%2520inhibitors%26jtitle%3DSwiss%2520Med.%2520Wkly.%26date%3D2015%26volume%3D145%26spage%3Dw14207%26doi%3D10.4414%2Fsmw.2015.14207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparano, J. A.</span><span> </span><span class="NLM_article-title">Farnesyl transferase inhibitors</span> <span class="citation_source-journal">Cancer Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">653</span><span class="NLM_x">–</span> <span class="NLM_lpage">661</span><span class="refDoi"> DOI: 10.1080/07357900801971016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1080%2F07357900801971016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=653-661&author=T.+Liauthor=J.+A.+Sparano&title=Farnesyl+transferase+inhibitors&doi=10.1080%2F07357900801971016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1080%2F07357900801971016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07357900801971016%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSparano%26aufirst%3DJ.%2BA.%26atitle%3DFarnesyl%2520transferase%2520inhibitors%26jtitle%3DCancer%2520Invest.%26date%3D2008%26volume%3D26%26spage%3D653%26epage%3D661%26doi%3D10.1080%2F07357900801971016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Cox, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Der, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philips, M. R.</span><span> </span><span class="NLM_article-title">Targeting RAs membrane association: back to the future for anti-RAS drug discovery</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1819</span><span class="NLM_x">–</span> <span class="NLM_lpage">1827</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-3214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1158%2F1078-0432.CCR-14-3214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25878363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsl2gt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1819-1827&author=A.+D.+Coxauthor=C.+J.+Derauthor=M.+R.+Philips&title=Targeting+RAs+membrane+association%3A+back+to+the+future+for+anti-RAS+drug+discovery&doi=10.1158%2F1078-0432.CCR-14-3214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RAS Membrane Association: Back to the Future for Anti-RAS Drug Discovery?</span></div><div class="casAuthors">Cox, Adrienne D.; Der, Channing J.; Philips, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1819-1827</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">RAS proteins require membrane assocn. for their biol. activity, making this assocn. a logical target for anti-RAS therapeutics.  Lipid modification of RAS proteins by a farnesyl isoprenoid is an obligate step in that assocn., and is an enzymic process.  Accordingly, farnesyltransferase inhibitors (FTI) were developed as potential anti-RAS drugs.  The lack of efficacy of FTIs as anticancer drugs was widely seen as indicating that blocking RAS membrane assocn. was a flawed approach to cancer treatment.  However, a deeper understanding of RAS modification and trafficking has revealed that this was an erroneous conclusion.  In the presence of FTIs, KRAS and NRAS, which are the RAS isoforms most frequently mutated in cancer, become substrates for alternative modification, can still assoc. with membranes, and can still function.  Thus, FTIs failed not because blocking RAS membrane assocn. is an ineffective approach, but because FTIs failed to accomplish that task.  Recent findings regarding RAS isoform trafficking and the regulation of RAS subcellular localization have rekindled interest in efforts to target these processes.  In particular, improved understanding of the palmitoylation/depalmitoylation cycle that regulates RAS interaction with the plasma membrane, endomembranes, and cytosol, and of the potential importance of RAS chaperones, have led to new approaches.  Efforts to validate and target other enzymically regulated posttranslational modifications are also ongoing.  In this review, we revisit lessons learned, describe the current state of the art, and highlight challenging but promising directions to achieve the goal of disrupting RAS membrane assocn. and subcellular localization for anti-RAS drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCImUyZbuCKrVg90H21EOLACvtfcHk0lhCvkpG_mXoJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsl2gt74%253D&md5=6f41ac23a7d14b90633755e374b6ffea</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3214%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DPhilips%26aufirst%3DM.%2BR.%26atitle%3DTargeting%2520RAs%2520membrane%2520association%253A%2520back%2520to%2520the%2520future%2520for%2520anti-RAS%2520drug%2520discovery%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1819%26epage%3D1827%26doi%3D10.1158%2F1078-0432.CCR-14-3214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Tamanoi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Recent progress in developing small molecule inhibitors designed to interfere with ras membrane association: toward inhibiting KRAS and NRAS functions</span> <span class="citation_source-journal">Enzymes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1016/B978-0-12-420146-0.00008-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FB978-0-12-420146-0.00008-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=25034105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=181-200&author=F.+Tamanoiauthor=J.+Lu&title=Recent+progress+in+developing+small+molecule+inhibitors+designed+to+interfere+with+ras+membrane+association%3A+toward+inhibiting+KRAS+and+NRAS+functions&doi=10.1016%2FB978-0-12-420146-0.00008-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in developing small molecule inhibitors designed to interfere with Ras membrane association: toward inhibiting K-Ras and N-Ras functions</span></div><div class="casAuthors">Tamanoi, Fuyuhiko; Lu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Enzymes</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">181-200</span>CODEN:
                <span class="NLM_cas:coden">25GLAS</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  K-Ras and N-Ras are mutated in a wide range of human cancers, thus making these proteins attractive targets of anticancer drug development.  However, no effective compds. have been obtained so far.  One of the approaches taken to inhibit the function of K-Ras and N-Ras is to interfere with their membrane assocn.  Various attempts have been taken.  In the first example, we examine the approach conceived in early 1990s to inhibit protein prenylation that is required for their membrane assocn.  The initial premise that the inhibition of Ras farnesylation leads to the inhibition of Ras was not realized, mainly due to alternative prenylation of K-Ras and N-Ras proteins.  This led to the idea that the combined inhibition of FTase and GGTase-I can block membrane assocn. of K-Ras and N-Ras.  Dual specificity inhibitors of FTase and GGTase-I (DPIs) were also developed.  These compds. were tested in preclin. and clin. studies.  It appears that sufficiently high concn. of the drug to inhibit K-Ras was not achieved in previous attempts.  In addn., dose-limiting toxicity has been obsd. and this was primarily ascribed to GGTase-I inhibition.  Strategies to confer cancer targeting capabilities to the inhibitors may overcome the dose-limiting toxicity.  In the second approach, postprenylation events were exploited.  This led to the development of various inhibitors including the ICMT inhibitors.  Finally, recent identification of compds. that inhibit the interaction between K-Ras and PDE-δ is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMwOKCVlBNLrVg90H21EOLACvtfcHk0livWdz3mE2vXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSgsL4%253D&md5=843c567089ab46424549b1f183d50fba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-420146-0.00008-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-420146-0.00008-1%26sid%3Dliteratum%253Aachs%26aulast%3DTamanoi%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DRecent%2520progress%2520in%2520developing%2520small%2520molecule%2520inhibitors%2520designed%2520to%2520interfere%2520with%2520ras%2520membrane%2520association%253A%2520toward%2520inhibiting%2520KRAS%2520and%2520NRAS%2520functions%26jtitle%3DEnzymes%26date%3D2013%26volume%3D34%26spage%3D181%26epage%3D200%26doi%3D10.1016%2FB978-0-12-420146-0.00008-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paramasivam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3780</span><span class="refDoi"> DOI: 10.1093/nar/gkn120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkn120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=18490377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3765-3780&author=S.+Cogoiauthor=M.+Paramasivamauthor=B.+Spolaoreauthor=L.+E.+Xodo&title=Structural+polymorphism+within+a+regulatory+element+of+the+human+KRAS+promoter%3A+formation+of+G4-DNA+recognized+by+nuclear+proteins&doi=10.1093%2Fnar%2Fgkn120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span></div><div class="casAuthors">Cogoi, Susanna; Paramasivam, Manikandan; Spolaore, Barbara; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3765-3780</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The human KRAS proto-oncogene contains a crit. nuclease hypersensitive element (NHE) upstream of the major transcription initiation site.  In this article, we demonstrate by primer-extension expts., PAGE, chem. footprinting, CD, UV and FRET expts. that the G-rich strand of NHE (32R) folds into intra-mol. G-quadruplex structures.  Fluorescence data show that 32R in 100 mM KCl melts with a biphasic profile, showing the formation of two distinct G-quadruplexes with Tm of ∼55°C (Q1) and ∼72°C (Q2).  DMS-footprinting and CD suggest that Q1 can be a parallel and Q2 a mixed parallel/antiparallel G-quadruplex.  When dsNHE (32R hybridized to its complementary) is incubated with a nuclear ext. from Panc-1 cells, three DNA-protein complexes are obsd. by EMSA.  The complex of slower mobility is competed by quadruplex 32R, but not by mutant oligonucleotides, which cannot form a quadruplex structure.  Using paramagnetic beads coupled with 32R, we pulled down from the Panc-1 ext. proteins with affinity for quadruplex 32R.  One of these is the heterogeneous nuclear ribonucleoprotein A1, which was previously reported to unfold quadruplex DNA.  These study suggests a role of quadruplex DNA in KRAS transcription and provides the basis for the rationale design of mol. strategies to inhibit the expression of KRAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5riE_L0077Vg90H21EOLACvtfcHk0livWdz3mE2vXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D&md5=73fcda8805e5f3989bcf6824805ed046</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn120%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DParamasivam%26aufirst%3DM.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DStructural%2520polymorphism%2520within%2520a%2520regulatory%2520element%2520of%2520the%2520human%2520KRAS%2520promoter%253A%2520formation%2520of%2520G4-DNA%2520recognized%2520by%2520nuclear%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3765%26epage%3D3780%26doi%3D10.1093%2Fnar%2Fgkn120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Faudale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Photoactivated cationic alkyl-substituted porphyrin binding to g4-RNA in the 5′-UTR of KRAS oncogene represses translation</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1039/C1CC15850C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1039%2FC1CC15850C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22127206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ersbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=874-876&author=M.+Faudaleauthor=S.+Cogoiauthor=L.+E.+Xodo&title=Photoactivated+cationic+alkyl-substituted+porphyrin+binding+to+g4-RNA+in+the+5%E2%80%B2-UTR+of+KRAS+oncogene+represses+translation&doi=10.1039%2FC1CC15850C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Photoactivated cationic alkyl-substituted porphyrin binding to g4-RNA in the 5'-UTR of KRAS oncogene represses translation</span></div><div class="casAuthors">Faudale, Mariangela; Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">874-876</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The KRAS transcript is characterized by a 192-nt 5'-UTR contg. repetitive runs of two-guanines which can fold in several G-quadruplexes.  These folded structures have a high affinity for the cationic porphyrin tri-meso(N-methyl-4-pyridyl), meso(N-tetradecyl-4-pyridyl) porphine (TMPyP4-C14), which efficiently penetrates cell membranes.  Upon photoactivation TMPyP4-C14 induces a dramatic down-regulation of oncogenic KRAS and cell growth arrest in pancreatic cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IgGfekwEpLVg90H21EOLACvtfcHk0lhkSC-YOKy83Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ersbnE&md5=2743606db2e9469d526a586af884d894</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC1CC15850C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC1CC15850C%26sid%3Dliteratum%253Aachs%26aulast%3DFaudale%26aufirst%3DM.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DPhotoactivated%2520cationic%2520alkyl-substituted%2520porphyrin%2520binding%2520to%2520g4-RNA%2520in%2520the%25205%25E2%2580%25B2-UTR%2520of%2520KRAS%2520oncogene%2520represses%2520translation%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2012%26volume%3D48%26spage%3D874%26epage%3D876%26doi%3D10.1039%2FC1CC15850C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacchigna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Anticancer activity of cationic porphyrins in melanoma tumour-bearing mice and mechanistic in vitro studies</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1186/1476-4598-13-75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1186%2F1476-4598-13-75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24684778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVWks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=75-92&author=V.+Rapozziauthor=S.+Zorzetauthor=M.+Zacchignaauthor=E.+Della+Pietraauthor=S.+Cogoiauthor=L.+E.+Xodo&title=Anticancer+activity+of+cationic+porphyrins+in+melanoma+tumour-bearing+mice+and+mechanistic+in+vitro+studies&doi=10.1186%2F1476-4598-13-75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer activity of cationic porphyrins in melanoma tumour-bearing mice and mechanistic in vitro studies</span></div><div class="casAuthors">Rapozzi, Valentina; Zorzet, Sonia; Zacchigna, Marina; Della Pietra, Emilia; Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75/1-75/17, 17</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Porphyrin TMPyP4 (P4) and its C14H28-alkyl deriv. (C14) are G-quadruplex binders and singlet oxygen (1O2) generators.  In contrast, TMPyP2 (P2) produces 1O2 but it is not a G-quadruplex binder.  As their photosensitizing activity is currently undefined, we report in this study their efficacy against a melanoma skin tumor and describe an in vitro mechanistic study which gives insights into their anticancer activity.  Methods: Uptake and antiproliferative activity of photoactivated P2, P4 and C14 have been investigated in murine melanoma B78-H1 cells by FACS, clonogenic and migration assays.  Apoptosis was investigated by PARP-1 cleavage and annexin-propidium iodide assays.  Biodistribution and in vivo anticancer activity were tested in melanoma tumor-bearing mice.  Porphyrin binding and photocleavage of G-rich mRNA regions were investigated by electrophoresis and RT-PCR.  Porphyrin effect on ERK pathway was explored by Western blots.  Results: Thanks to its higher lipophylicity C14 was taken up by murine melanoma B78-H1 cells up to 30-fold more efficiently than P4.  When photoactivated (7.2 J/cm2) in B78-H1 melanoma cells, P4 and C14, but not control P2, caused a strong inhibition of metabolic activity, clonogenic growth and cell migration.  Biodistribution studies on melanoma tumor-bearing mice showed that P4 and C14 localize in the tumor.  Upon irradn. (660 nm, 193 J/cm2), P4 and C14 retarded tumor growth and increased the median survival time of the treated mice by ∼50% (P <0.01 by ANOVA), whereas porphyrin P2 did not.  The light-dependent mechanism mediated by P4 and C14 is likely due to the binding to and photocleavage of G-rich quadruplex-forming sequences within the 5'-untranslated regions of the mitogenic ras genes.  This causes a decrease of RAS protein and inhibition of downstream ERK pathway, which stimulates proliferation.  Annexin V/propidium iodide and PARP-1 cleavage assays showed that the porphyrins arrested tumor growth by apoptosis and necrosis.  C14 also showed an intrinsic light-independent anticancer activity, as recently reported for G4-RNA binders.  Conclusions: Porphyrins P4 and C14 impair the clonogenic growth and migration of B78-H1 melanoma cells and inhibit melanoma tumor growth in vivo.  Evidence is provided that C14 acts through light-dependent (mRNA photocleavage) and light-independent (translation inhibition) mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCrHhSDBDwYrVg90H21EOLACvtfcHk0lhkSC-YOKy83Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVWks78%253D&md5=45bff23e8e4b99b15b3dabc82dc8833c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-13-75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-13-75%26sid%3Dliteratum%253Aachs%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DZacchigna%26aufirst%3DM.%26aulast%3DDella%2BPietra%26aufirst%3DE.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DAnticancer%2520activity%2520of%2520cationic%2520porphyrins%2520in%2520melanoma%2520tumour-bearing%2520mice%2520and%2520mechanistic%2520in%2520vitro%2520studies%26jtitle%3DMol.%2520Cancer%26date%3D2014%26volume%3D13%26spage%3D75%26epage%3D92%26doi%3D10.1186%2F1476-4598-13-75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">An RNA G-quadruplex in the 5′-UTR of the NRAS proto-oncogene modulates translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1038/nchembio864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchembio864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17322877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=218-221&author=S.+Kumariauthor=A.+Bugautauthor=J.+L.+Huppertauthor=S.+Balasubramanian&title=An+RNA+G-quadruplex+in+the+5%E2%80%B2-UTR+of+the+NRAS+proto-oncogene+modulates+translation&doi=10.1038%2Fnchembio864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA G-quadruplex in the 5' UTR of the NRAS proto-oncogene modulates translation</span></div><div class="casAuthors">Kumari, Sunita; Bugaut, Anthony; Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acid sequences can adopt noncanonical four-stranded secondary structures called guanine (G)-quadruplexes.  Bioinformatics anal. suggests that G-quadruplex motifs are prevalent in genomes, which raises the need to elucidate their function.  There is now evidence for the existence of DNA G-quadruplexes at telomeres with assocd. biol. function.  A recent hypothesis supports the notion that gene promoter elements contain DNA G-quadruplex motifs that control gene expression at the transcriptional level.  We discovered a highly conserved, thermodynamically stable RNA G-quadruplex in the 5' untranslated region (UTR) of the gene transcript of the human NRAS proto-oncogene.  Using a cell-free translation system coupled to a reporter gene assay, we have demonstrated that this NRAS RNA G-quadruplex modulates translation.  This is the first example of translational repression by an RNA G-quadruplex.  Bioinformatics anal. has revealed 2,922 other 5' UTR RNA G-quadruplex elements in the human genome.  We propose that RNA G-quadruplexes in the 5' UTR modulate gene expression at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZXUhKbJ4cLVg90H21EOLACvtfcHk0lj4Ce7cm8C1Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D&md5=c4dda371d46b95d25b285d1adeb599be</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio864%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DAn%2520RNA%2520G-quadruplex%2520in%2520the%25205%25E2%2580%25B2-UTR%2520of%2520the%2520NRAS%2520proto-oncogene%2520modulates%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnchembio864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">5′-UTR RNA G-quadruplexes: translation regulation and targeting</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4727</span><span class="NLM_x">–</span> <span class="NLM_lpage">4741</span><span class="refDoi"> DOI: 10.1093/nar/gks068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgks068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22351747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=4727-4741&author=A.+Bugautauthor=S.+Balasubramanian&title=5%E2%80%B2-UTR+RNA+G-quadruplexes%3A+translation+regulation+and+targeting&doi=10.1093%2Fnar%2Fgks068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR RNA G-quadruplexes: translation regulation and targeting</span></div><div class="casAuthors">Bugaut, Anthony; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4727-4741</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  RNA structures in the untranslated regions (UTRs) of mRNAs influence post-transcriptional regulation of gene expression.  Much of the knowledge in this area depends on canonical double-stranded RNA elements.  There has been considerable recent advancement of our understanding of guanine(G)-rich nucleic acids sequences that form four-stranded structures, called G-quadruplexes.  While much of the research has been focused on DNA G-quadruplexes, there has recently been a rapid emergence of interest in RNA G-quadruplexes, particularly in the 5'-UTRs of mRNAs.  Collectively, these studies suggest that RNA G-quadruplexes exist in the 5'-UTRs of many genes, including genes of clin. interest, and that such structural elements can influence translation.  This review features the progresses in the study of 5'-UTR RNA G-quadruplex-mediated translational control.  It covers computational anal., cell-free, cell-based and chem. biol. studies that have sought to elucidate the roles of RNA G-quadruplexes in both cap-dependent and -independent regulation of mRNA translation.  We also discuss protein trans-acting factors that have been implicated and the evidence that such RNA motifs have potential as small mol. target.  Finally, we close the review with a perspective on the future challenges in the field of 5'-UTR RNA G-quadruplex-mediated translation regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fxG5DFVSLrVg90H21EOLACvtfcHk0lj4Ce7cm8C1Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D&md5=d782ebe76a05937797a398ea20b3adf3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks068%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3D5%25E2%2580%25B2-UTR%2520RNA%2520G-quadruplexes%253A%2520translation%2520regulation%2520and%2520targeting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D4727%26epage%3D4741%26doi%3D10.1093%2Fnar%2Fgks068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0ljV_Dyaxx9wMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Bochman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paeschke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakian, V. A.</span><span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">780</span><span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0ljV_Dyaxx9wMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Existance and consequences of G-quadruplex structures in DNA</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.gde.2013.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.gde.2013.10.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=22-29&author=P.+Muratauthor=S.+Balasubramanian&title=Existance+and+consequences+of+G-quadruplex+structures+in+DNA&doi=10.1016%2Fj.gde.2013.10.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2013.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2013.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DExistance%2520and%2520consequences%2520of%2520G-quadruplex%2520structures%2520in%2520DNA%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2014%26volume%3D25%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.gde.2013.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Burge, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Quadruplex DNA: sequence, topology and structure</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">5402</span><span class="NLM_x">–</span> <span class="NLM_lpage">5415</span><span class="refDoi"> DOI: 10.1093/nar/gkl655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=17012276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=5402-5415&author=S.+Burgeauthor=G.+N.+Parkinsonauthor=P.+Hazelauthor=A.+K.+Toddauthor=S.+Neidle&title=Quadruplex+DNA%3A+sequence%2C+topology+and+structure&doi=10.1093%2Fnar%2Fgkl655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex DNA: sequence, topology and structure</span></div><div class="casAuthors">Burge, Sarah; Parkinson, Gary N.; Hazel, Pascale; Todd, Alan K.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5402-5415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are higher-order DNA and RNA structures formed from G-rich sequences that are built around tetrads of hydrogen-bonded guanine bases.  Potential quadruplex sequences have been identified in G-rich eukaryotic telomeres, and more recently in non-telomeric genomic DNA, e.g. in nuclease-hypersensitive promoter regions.  The natural role and biol. validation of these structures is starting to be explored, and there is particular interest in them as targets for therapeutic intervention.  This survey focuses on the folding and structural features on quadruplexes formed from telomeric and non-telomeric DNA sequences, and examines fundamental aspects of topol. and the emerging relationships with sequence.  Emphasis is placed on information from the high-resoln. methods of x-ray crystallog. and NMR, and their scope and current limitations are discussed.  Such information, together with biol. insights, will be important for the discovery of drugs targeting quadruplexes from particular genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovf3pnf_GPorVg90H21EOLACvtfcHk0ljZo4zxhmsnpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM&md5=63387d3c6c1e3567dd8cddd0f0d86c97</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl655%26sid%3Dliteratum%253Aachs%26aulast%3DBurge%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DQuadruplex%2520DNA%253A%2520sequence%252C%2520topology%2520and%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D5402%26epage%3D5415%26doi%3D10.1093%2Fnar%2Fgkl655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0ljZo4zxhmsnpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">G4 motifs in human genes</span> <span class="citation_source-journal">Ann. N. Y. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">1267</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1111/j.1749-6632.2012.06586.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1111%2Fj.1749-6632.2012.06586.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1267&publication_year=2012&pages=53-60&author=N.+Maizels&title=G4+motifs+in+human+genes&doi=10.1111%2Fj.1749-6632.2012.06586.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2012.06586.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2012.06586.x%26sid%3Dliteratum%253Aachs%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DG4%2520motifs%2520in%2520human%2520genes%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2012%26volume%3D1267%26spage%3D53%26epage%3D60%26doi%3D10.1111%2Fj.1749-6632.2012.06586.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Mendoza, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourdoncle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulé, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span> </span><span class="NLM_article-title">G-quadruplexes and helicases</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2006</span><span class="refDoi"> DOI: 10.1093/nar/gkw079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkw079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26883636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Sqsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1989-2006&author=O.+Mendozaauthor=A.+Bourdoncleauthor=J.+B.+Boul%C3%A9author=R.+M.+Broshauthor=J.+L.+Mergny&title=G-quadruplexes+and+helicases&doi=10.1093%2Fnar%2Fgkw079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes and helicases</span></div><div class="casAuthors">Mendoza, Oscar; Bourdoncle, Anne; Boule, Jean-Baptiste; Brosh, Robert M., Jr.; Mergny, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1989-2006</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Guanine-rich DNA strands can fold in vitro into non-canonical DNA structures called G-quadruplexes.  These structures may be very stable under physiol. conditions.  Evidence suggests that G-quadruplex structures may act as 'knots' within genomic DNA, and it has been hypothesized that proteins may have evolved to remove these structures.  The first indication of how G-quadruplex structures could be unfolded enzymically came in the late 1990s with reports that some well-known duplex DNA helicases resolved these structures in vitro.  Since then, the no. of studies reporting G-quadruplex DNA unfolding by helicase enzymes has rapidly increased.  The present review aims to present a general overview of the helicase/G-quadruplex field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1DDktyH31aLVg90H21EOLACvtfcHk0ljZo4zxhmsnpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Sqsb7E&md5=0d424783aab817ee14c56597712477ba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw079%26sid%3Dliteratum%253Aachs%26aulast%3DMendoza%26aufirst%3DO.%26aulast%3DBourdoncle%26aufirst%3DA.%26aulast%3DBoul%25C3%25A9%26aufirst%3DJ.%2BB.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DG-quadruplexes%2520and%2520helicases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D1989%26epage%3D2006%26doi%3D10.1093%2Fnar%2Fgkw079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G4 DNA in ras genes and its potential in cancer therapy</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">1859</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2016.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bbagrm.2016.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=26855080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Shurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1859&publication_year=2016&pages=663-674&author=S.+Cogoiauthor=L.+E.+Xodo&title=G4+DNA+in+ras+genes+and+its+potential+in+cancer+therapy&doi=10.1016%2Fj.bbagrm.2016.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">G4 DNA in ras genes and its potential in cancer therapy</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1859</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">663-674</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">It is now well established that in the human genome the canonical double helix coexists with folded G-quadruplex structures that are known to have important biol. functions.  In this review we summarize the current knowledge on quadruplex formation in the promoters of the ras genes that are mutated in about 30% of all human cancers.  We describe the nuclear proteins that recognize these unusual DNA structures and discuss their function in transcription.  We also examine the formation of G-quadruplexes in the 5'-untranslated region of the ras transcripts and conclude this review by reporting strategies that use either ras G-quadruplexes or proteins recognizing the ras G-quadruplexes as targets of anticancer small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOg06a0dRrALVg90H21EOLACvtfcHk0li49xOUuuk71Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Shurc%253D&md5=a3c0e6fcee18fb40fb5a88bd38901657</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2016.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2016.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG4%2520DNA%2520in%2520ras%2520genes%2520and%2520its%2520potential%2520in%2520cancer%2520therapy%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2016%26volume%3D1859%26spage%3D663%26epage%3D674%26doi%3D10.1016%2Fj.bbagrm.2016.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Hänsel-Hertsch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beraldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lensing, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsico, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zyner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">G-quadruplex structures mark human regulatory chromatin</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1267</span><span class="NLM_x">–</span> <span class="NLM_lpage">1272</span><span class="refDoi"> DOI: 10.1038/ng.3662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fng.3662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27618450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFSgurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=1267-1272&author=R.+H%C3%A4nsel-Hertschauthor=D.+Beraldiauthor=S.+V.+Lensingauthor=G.+Marsicoauthor=K.+Zynerauthor=A.+Parryauthor=M.+Di+Antonioauthor=J.+Pikeauthor=H.+Kimuraauthor=M.+Naritaauthor=D.+Tannahillauthor=S.+Balasubramanian&title=G-quadruplex+structures+mark+human+regulatory+chromatin&doi=10.1038%2Fng.3662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures mark human regulatory chromatin</span></div><div class="casAuthors">Hansel-Hertsch, Robert; Beraldi, Dario; Lensing, Stefanie V.; Marsico, Giovanni; Zyner, Katherine; Parry, Aled; Di Antonio, Marco; Pike, Jeremy; Kimura, Hiroshi; Narita, Masashi; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1267-1272</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplex (G4) structural motifs have been linked to transcription, replication and genome instability and are implicated in cancer and other diseases.  However, it is crucial to demonstrate the bona fide formation of G4 structures within an endogenous chromatin context.  Herein we address this through the development of G4 ChIP-seq, an antibody-based G4 chromatin immunopptn. and high-throughput sequencing approach.  We find ∼10,000 G4 structures in human chromatin, predominantly in regulatory, nucleosome-depleted regions.  G4 structures are enriched in the promoters and 5' UTRs of highly transcribed genes, particularly in genes related to cancer and in somatic copy no. amplifications, such as MYC.  Strikingly, de novo and enhanced G4 formation are assocd. with increased transcriptional activity, as shown by HDAC inhibitor-induced chromatin relaxation and obsd. in immortalized as compared to normal cellular states.  Our findings show that regulatory, nucleosome-depleted chromatin and elevated transcription shape the endogenous human G4 DNA landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYo8vsZd_UZLVg90H21EOLACvtfcHk0li49xOUuuk71Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFSgurvP&md5=898d19559a9ccbc0ff468558a653fc43</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fng.3662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3662%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4nsel-Hertsch%26aufirst%3DR.%26aulast%3DBeraldi%26aufirst%3DD.%26aulast%3DLensing%26aufirst%3DS.%2BV.%26aulast%3DMarsico%26aufirst%3DG.%26aulast%3DZyner%26aufirst%3DK.%26aulast%3DParry%26aufirst%3DA.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DPike%26aufirst%3DJ.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DNarita%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DG-quadruplex%2520structures%2520mark%2520human%2520regulatory%2520chromatin%26jtitle%3DNat.%2520Genet.%26date%3D2016%26volume%3D48%26spage%3D1267%26epage%3D1272%26doi%3D10.1038%2Fng.3662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Beaudoin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">5′-UTR G-quadruplex structures acting as translational repressors</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">7022</span><span class="NLM_x">–</span> <span class="NLM_lpage">7036</span><span class="refDoi"> DOI: 10.1093/nar/gkq557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkq557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20571090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVektrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=7022-7036&author=J.+D.+Beaudoinauthor=J.+P.+Perreault&title=5%E2%80%B2-UTR+G-quadruplex+structures+acting+as+translational+repressors&doi=10.1093%2Fnar%2Fgkq557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR G-quadruplex structures acting as translational repressors</span></div><div class="casAuthors">Beaudoin, Jean-Denis; Perreault, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7022-7036</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Given that greater than 90% of the human genome is expressed, it is logical to assume that post-transcriptional regulatory mechanisms must be the primary means of controlling the flow of information from mRNA to protein.  This report describes a robust approach that includes in silico, in vitro and in cellulo expts. permitting an in-depth evaluation of the impact of G-quadruplexes as translational repressors.  Sequences including potential G-quadruplexes were selected within nine distinct genes encoding proteins involved in various biol. processes.  Their abilities to fold into G-quadruplex structures in vitro were evaluated using CD, thermal denaturation and the novel use of in-line probing.  Six sequences were obsd. to fold into G-quadruplex structures in vitro, all of which exhibited translational inhibition in cellulo when linked to a reporter gene.  Sequence anal., direct mutagenesis and subsequent expts. were performed in order to define the rules governing the folding of G-quadruplexes.  In addn., the impact of single-nucleotide polymorphism was shown to be important in the formation of G-quadruplexes located within the 5'-untranslated region of an mRNA.  In light of these results, clearly the 5'-UTR G-quadruplexes represent a class of translational repressors that is broadly distributed in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRYMyH2yIct7Vg90H21EOLACvtfcHk0li49xOUuuk71Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVektrzO&md5=eed8abc1182def80967aa26632116d4a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq557%26sid%3Dliteratum%253Aachs%26aulast%3DBeaudoin%26aufirst%3DJ.%2BD.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3D5%25E2%2580%25B2-UTR%2520G-quadruplex%2520structures%2520acting%2520as%2520translational%2520repressors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D7022%26epage%3D7036%26doi%3D10.1093%2Fnar%2Fgkq557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Jodoin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garant, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdi Laaref, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phaneuf, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">The folding of 5′-UTR human G-quadruplexes possessing long central loop</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1141</span><span class="refDoi"> DOI: 10.1261/rna.044578.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1261%2Frna.044578.114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1129-1141&author=R.+Jodoinauthor=L.+Bauerauthor=J.+M.+Garantauthor=A.+Mahdi+Laarefauthor=F.+Phaneufauthor=J.+P.+Perreault&title=The+folding+of+5%E2%80%B2-UTR+human+G-quadruplexes+possessing+long+central+loop&doi=10.1261%2Frna.044578.114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1261%2Frna.044578.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.044578.114%26sid%3Dliteratum%253Aachs%26aulast%3DJodoin%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DL.%26aulast%3DGarant%26aufirst%3DJ.%2BM.%26aulast%3DMahdi%2BLaaref%26aufirst%3DA.%26aulast%3DPhaneuf%26aufirst%3DF.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520folding%2520of%25205%25E2%2580%25B2-UTR%2520human%2520G-quadruplexes%2520possessing%2520long%2520central%2520loop%26jtitle%3DRNA%26date%3D2014%26volume%3D20%26spage%3D1129%26epage%3D1141%26doi%3D10.1261%2Frna.044578.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Hinnebusch, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanov, I. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span> </span><span class="NLM_article-title">Translational control by 5′-untranslated regions of eukaryotic mRNAs</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1416</span><span class="refDoi"> DOI: 10.1126/science.aad9868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1126%2Fscience.aad9868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27313038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslOqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=1413-1416&author=A.+G.+Hinnebuschauthor=I.+P.+Ivanovauthor=N.+Sonenberg&title=Translational+control+by+5%E2%80%B2-untranslated+regions+of+eukaryotic+mRNAs&doi=10.1126%2Fscience.aad9868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Translational control by 5'-untranslated regions of eukaryotic mRNAs</span></div><div class="casAuthors">Hinnebusch, Alan G.; Ivanov, Ivaylo P.; Sonenberg, Nahum</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6292</span>),
    <span class="NLM_cas:pages">1413-1416</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The eukaryotic 5' untranslated region (UTR) is crit. for ribosome recruitment to the mRNA and start codon choice and plays a major role in the control of translation efficiency and shaping the cellular proteome.  The ribosomal initiation complex is assembled on the mRNA via a cap-dependent or cap-independent mechanism.  The authors describe various mechanisms controlling ribosome scanning and initiation codon selection by 5' upstream open reading frames, translation initiation factors, and primary and secondary structures of the 5'UTR, including particular sequence motifs.  The authors also discuss translational control via phosphorylation of eukaryotic initiation factor 2, which is implicated in learning and memory, neurodegenerative diseases, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogiiubG2iPLVg90H21EOLACvtfcHk0lieZnanzeU6sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslOqsbY%253D&md5=b50892c1043afb14c936a36a51dc8452</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad9868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad9868%26sid%3Dliteratum%253Aachs%26aulast%3DHinnebusch%26aufirst%3DA.%2BG.%26aulast%3DIvanov%26aufirst%3DI.%2BP.%26aulast%3DSonenberg%26aufirst%3DN.%26atitle%3DTranslational%2520control%2520by%25205%25E2%2580%25B2-untranslated%2520regions%2520of%2520eukaryotic%2520mRNAs%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D1413%26epage%3D1416%26doi%3D10.1126%2Fscience.aad9868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topisirovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">RNA G-quadruplexes and their potential regulatory roles in translation</span> <span class="citation_source-journal">Translation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_elocation-id">e1244031</span><span class="refDoi"> DOI: 10.1080/21690731.2016.1244031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1080%2F21690731.2016.1244031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&author=J.+Songauthor=J.+P.+Perreaultauthor=I.+Topisirovicauthor=S.+Richard&title=RNA+G-quadruplexes+and+their+potential+regulatory+roles+in+translation&doi=10.1080%2F21690731.2016.1244031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1080%2F21690731.2016.1244031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21690731.2016.1244031%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26aulast%3DTopisirovic%26aufirst%3DI.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DRNA%2520G-quadruplexes%2520and%2520their%2520potential%2520regulatory%2520roles%2520in%2520translation%26jtitle%3DTranslation%26date%3D2016%26volume%3D4%26doi%3D10.1080%2F21690731.2016.1244031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Gomez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guédin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salles, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teulade-Fichou, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calsou, P.</span><span> </span><span class="NLM_article-title">A G-quadruplex structure within the 5′-UTR of TRF2 mRNA represses translation in human cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">7187</span><span class="NLM_x">–</span> <span class="NLM_lpage">7198</span><span class="refDoi"> DOI: 10.1093/nar/gkq563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkq563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20571083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVektr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=7187-7198&author=D.+Gomezauthor=A.+Gu%C3%A9dinauthor=J.+L.+Mergnyauthor=B.+Sallesauthor=J.+F.+Riouauthor=M.+P.+Teulade-Fichouauthor=P.+Calsou&title=A+G-quadruplex+structure+within+the+5%E2%80%B2-UTR+of+TRF2+mRNA+represses+translation+in+human+cells&doi=10.1093%2Fnar%2Fgkq563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A G-quadruplex structure within the 5'-UTR of TRF2 mRNA represses translation in human cells</span></div><div class="casAuthors">Gomez, Dennis; Guedin, Aurore; Mergny, Jean-Louis; Salles, Bernard; Riou, Jean-Francois; Teulade-Fichou, Marie-Paule; Calsou, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7187-7198</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Telomeres protect chromosome ends from being recognized as double-stranded breaks.  Telomeric function is ensured by the shelterin complex in which TRF2 protein is an essential player.  The G-rich strand of telomere DNA can fold into G-quadruplex (G4) structure.  Small mols. stabilizing G4 structures, named G4 ligands, have been shown to alter telomeric functions in human cells.  This study shows that a guanine-rich RNA sequence located in the 5'-UTR region of the TRF2 mRNA (hereafter 91TRF2G) is capable of forming a stable quadruplex that causes a 2.8-fold decrease in the translation of a reporter gene in human cells, as compared to a mutant 5'-UTR unable to fold into G4.  Several highly selective G4 ligands, the pyridine dicarboxamide deriv. 360A and bisquinolinium compds. Phen-DC(3) and Phen-DC(6), are able to bind the 91TRF2G:RNA sequence and to modulate TRF2 protein translation in vitro.  Since the naturally occurring 5'-UTR TRF2:RNA G4 element was used here, which is conserved in several vertebrate orthologs, the present data substantiate a potential translational mechanism mediated by a G4 RNA motif for the downregulation of TRF2 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY7NhUZIRXSbVg90H21EOLACvtfcHk0lieZnanzeU6sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVektr7I&md5=e72914887bc8a119a521d3b55617833c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq563%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DD.%26aulast%3DGu%25C3%25A9din%26aufirst%3DA.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DSalles%26aufirst%3DB.%26aulast%3DRiou%26aufirst%3DJ.%2BF.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DCalsou%26aufirst%3DP.%26atitle%3DA%2520G-quadruplex%2520structure%2520within%2520the%25205%25E2%2580%25B2-UTR%2520of%2520TRF2%2520mRNA%2520represses%2520translation%2520in%2520human%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D7187%26epage%3D7198%26doi%3D10.1093%2Fnar%2Fgkq563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Kwok, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Assmann, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bevilacqua, P. C.</span><span> </span><span class="NLM_article-title">A stable RNA G-quadruplex within the 5′-UTR of Arabidopsis thaliana ATR mRNA inhibits translation</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1042/BJ20141063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1042%2FBJ20141063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=91-102&author=C.+K.+Kwokauthor=Y.+Dingauthor=S.+Shahidauthor=S.+M.+Assmannauthor=P.+C.+Bevilacqua&title=A+stable+RNA+G-quadruplex+within+the+5%E2%80%B2-UTR+of+Arabidopsis+thaliana+ATR+mRNA+inhibits+translation&doi=10.1042%2FBJ20141063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1042%2FBJ20141063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141063%26sid%3Dliteratum%253Aachs%26aulast%3DKwok%26aufirst%3DC.%2BK.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DShahid%26aufirst%3DS.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26atitle%3DA%2520stable%2520RNA%2520G-quadruplex%2520within%2520the%25205%25E2%2580%25B2-UTR%2520of%2520Arabidopsis%2520thaliana%2520ATR%2520mRNA%2520inhibits%2520translation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D91%26epage%3D102%26doi%3D10.1042%2FBJ20141063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Arora, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutkiewicz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaria, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hariharan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurreck, J.</span><span> </span><span class="NLM_article-title">Inhibition of translation in eukaryotic cells by an RNA G-quadruplex motif</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1290</span><span class="NLM_x">–</span> <span class="NLM_lpage">1296</span><span class="refDoi"> DOI: 10.1261/rna.1001708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1261%2Frna.1001708" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=1290-1296&author=A.+Aroraauthor=M.+Dutkiewiczauthor=V.+Scariaauthor=M.+Hariharanauthor=S.+Maitiauthor=J.+Kurreck&title=Inhibition+of+translation+in+eukaryotic+cells+by+an+RNA+G-quadruplex+motif&doi=10.1261%2Frna.1001708"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1261%2Frna.1001708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.1001708%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DDutkiewicz%26aufirst%3DM.%26aulast%3DScaria%26aufirst%3DV.%26aulast%3DHariharan%26aufirst%3DM.%26aulast%3DMaiti%26aufirst%3DS.%26aulast%3DKurreck%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520translation%2520in%2520eukaryotic%2520cells%2520by%2520an%2520RNA%2520G-quadruplex%2520motif%26jtitle%3DRNA%26date%3D2008%26volume%3D14%26spage%3D1290%26epage%3D1296%26doi%3D10.1261%2Frna.1001708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Position and stability are determining factors for translation repression by an RNA G-quaduplex forming sequence within 5′-UTR of the NRAS proto oncogene</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">12664</span><span class="NLM_x">–</span> <span class="NLM_lpage">12669</span><span class="refDoi"> DOI: 10.1021/bi8010797</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8010797" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=12664-12669&author=S.+Kumariauthor=A.+Bugautauthor=S.+Balasubramanian&title=Position+and+stability+are+determining+factors+for+translation+repression+by+an+RNA+G-quaduplex+forming+sequence+within+5%E2%80%B2-UTR+of+the+NRAS+proto+oncogene&doi=10.1021%2Fbi8010797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi8010797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8010797%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DPosition%2520and%2520stability%2520are%2520determining%2520factors%2520for%2520translation%2520repression%2520by%2520an%2520RNA%2520G-quaduplex%2520forming%2520sequence%2520within%25205%25E2%2580%25B2-UTR%2520of%2520the%2520NRAS%2520proto%2520oncogene%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D12664%26epage%3D12669%26doi%3D10.1021%2Fbi8010797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Morris, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Negishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pazsint, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonhoft, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span> </span><span class="NLM_article-title">A RNA G-quadruplex is essential for cap-independent translation initiation in human VEGF IRES</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">17831</span><span class="NLM_x">–</span> <span class="NLM_lpage">17839</span><span class="refDoi"> DOI: 10.1021/ja106287x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja106287x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=17831-17839&author=M.+J.+Morrisauthor=Y.+Negishiauthor=C.+Pazsintauthor=J.+D.+Schonhoftauthor=S.+Basu&title=A+RNA+G-quadruplex+is+essential+for+cap-independent+translation+initiation+in+human+VEGF+IRES&doi=10.1021%2Fja106287x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja106287x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja106287x%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DM.%2BJ.%26aulast%3DNegishi%26aufirst%3DY.%26aulast%3DPazsint%26aufirst%3DC.%26aulast%3DSchonhoft%26aufirst%3DJ.%2BD.%26aulast%3DBasu%26aufirst%3DS.%26atitle%3DA%2520RNA%2520G-quadruplex%2520is%2520essential%2520for%2520cap-independent%2520translation%2520initiation%2520in%2520human%2520VEGF%2520IRES%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D17831%26epage%3D17839%26doi%3D10.1021%2Fja106287x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Lammich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamp, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuscher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zilow, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haass, C.</span><span> </span><span class="NLM_article-title">Translational repression of the disintegrin and metallo protease ADAM10 by a stable G-quadruplex secondary structure in its 5′-untrasnlated region</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">45063</span><span class="NLM_x">–</span> <span class="NLM_lpage">45072</span><span class="refDoi"> DOI: 10.1074/jbc.M111.296921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1074%2Fjbc.M111.296921" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=45063-45072&author=S.+Lammichauthor=F.+Kampauthor=J.+Wagnerauthor=B.+Nuscherauthor=S.+Zilowauthor=A.+K.+Ludwigauthor=M.+Willemauthor=C.+Haass&title=Translational+repression+of+the+disintegrin+and+metallo+protease+ADAM10+by+a+stable+G-quadruplex+secondary+structure+in+its+5%E2%80%B2-untrasnlated+region&doi=10.1074%2Fjbc.M111.296921"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.296921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.296921%26sid%3Dliteratum%253Aachs%26aulast%3DLammich%26aufirst%3DS.%26aulast%3DKamp%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DJ.%26aulast%3DNuscher%26aufirst%3DB.%26aulast%3DZilow%26aufirst%3DS.%26aulast%3DLudwig%26aufirst%3DA.%2BK.%26aulast%3DWillem%26aufirst%3DM.%26aulast%3DHaass%26aufirst%3DC.%26atitle%3DTranslational%2520repression%2520of%2520the%2520disintegrin%2520and%2520metallo%2520protease%2520ADAM10%2520by%2520a%2520stable%2520G-quadruplex%2520secondary%2520structure%2520in%2520its%25205%25E2%2580%25B2-untrasnlated%2520region%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D45063%26epage%3D45072%26doi%3D10.1074%2Fjbc.M111.296921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Bolduc, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garant, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perreault, J. P.</span><span> </span><span class="NLM_article-title">Irregular G-quadruplexes found in untranslated regions of human mRNAs influences translation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">21751</span><span class="NLM_x">–</span> <span class="NLM_lpage">21750</span><span class="refDoi"> DOI: 10.1074/jbc.M116.744839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1074%2Fjbc.M116.744839" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=21751-21750&author=F.+Bolducauthor=J.+M.+Garantauthor=F.+Allardauthor=J.+P.+Perreault&title=Irregular+G-quadruplexes+found+in+untranslated+regions+of+human+mRNAs+influences+translation&doi=10.1074%2Fjbc.M116.744839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.744839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.744839%26sid%3Dliteratum%253Aachs%26aulast%3DBolduc%26aufirst%3DF.%26aulast%3DGarant%26aufirst%3DJ.%2BM.%26aulast%3DAllard%26aufirst%3DF.%26aulast%3DPerreault%26aufirst%3DJ.%2BP.%26atitle%3DIrregular%2520G-quadruplexes%2520found%2520in%2520untranslated%2520regions%2520of%2520human%2520mRNAs%2520influences%2520translation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D21751%26epage%3D21750%26doi%3D10.1074%2Fjbc.M116.744839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Zuker, M.</span><span> </span><span class="NLM_article-title">Mfold web server for nucleic acid folding and hybridization prediction</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3406</span><span class="NLM_x">–</span> <span class="NLM_lpage">3415</span><span class="refDoi"> DOI: 10.1093/nar/gkg595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkg595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=12824337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVWisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=3406-3415&author=M.+Zuker&title=Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction&doi=10.1093%2Fnar%2Fgkg595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mfold web server for nucleic acid folding and hybridization prediction</span></div><div class="casAuthors">Zuker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3406-3415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The abbreviated name, ' mfold web server', describes a no. of closely related software applications available on the World Wide Web (WWW) for the prediction of the secondary structure of single stranded nucleic acids.  The objective of this web server is to provide easy access to RNA and DNA folding and hybridization software to the scientific community at large.  By making use of universally available web GUIs (Graphical User Interfaces), the server circumvents the problem of portability of this software.  Detailed output, in the form of structure plots with or without reliability information, single strand frequency plots and energy dot plots', are available for the folding of single sequences.  A variety of bulk' servers give less information, but in a shorter time and for up to hundreds of sequences at once.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4cF1dfOCOjLVg90H21EOLACvtfcHk0ljpZQgLRMFY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVWisr8%253D&md5=4f49b0c2add0eedbcc41a1f058e285bb</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg595%26sid%3Dliteratum%253Aachs%26aulast%3DZuker%26aufirst%3DM.%26atitle%3DMfold%2520web%2520server%2520for%2520nucleic%2520acid%2520folding%2520and%2520hybridization%2520prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26spage%3D3406%26epage%3D3415%26doi%3D10.1093%2Fnar%2Fgkg595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Kikin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Antonio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagga, P. S.</span><span> </span><span class="NLM_article-title">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">W676</span><span class="NLM_x">–</span> <span class="NLM_lpage">W682</span><span class="refDoi"> DOI: 10.1093/nar/gkl253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16845096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W676-W682&author=O.+Kikinauthor=L.+D%E2%80%99Antonioauthor=P.+S.+Bagga&title=QGRS+Mapper%3A+a+web-based+server+for+predicting+G-quadruplexes+in+nucleotide+sequences&doi=10.1093%2Fnar%2Fgkl253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span></div><div class="casAuthors">Kikin, Oleg; D'Antonio, Lawrence; Bagga, Paramjeet S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W676-W682</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biol. processes and as therapeutic targets.  G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation.  Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts.  Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation.  Computational approaches to study G-quadruplexes allow detailed anal. of mammalian genomes.  There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences.  We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences.  It is a user-friendly application that provides many options for defining and studying G-quadruplexes.  It performs anal. of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences.  It is also useful for predicting G-quadruplex structures in oligonucleotides.  The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites.  This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing.  In addn. to providing data on compn. and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data.  The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yCz4ebPV-bVg90H21EOLACvtfcHk0ljpZQgLRMFY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D&md5=e8f69845c3df9217df7f346a06c3db44</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl253%26sid%3Dliteratum%253Aachs%26aulast%3DKikin%26aufirst%3DO.%26aulast%3DD%25E2%2580%2599Antonio%26aufirst%3DL.%26aulast%3DBagga%26aufirst%3DP.%2BS.%26atitle%3DQGRS%2520Mapper%253A%2520a%2520web-based%2520server%2520for%2520predicting%2520G-quadruplexes%2520in%2520nucleotide%2520sequences%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW676%26epage%3DW682%26doi%3D10.1093%2Fnar%2Fgkl253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Vorlickova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kejnovska, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednarova, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renciuk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kypr, J.</span><span> </span><span class="NLM_article-title">Circular dichroism spectroscopy of DNA: from duplexes to quadruplexes</span> <span class="citation_source-journal">Chirality</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1002/chir.22064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1002%2Fchir.22064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=22696273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVeiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=691-698&author=M.+Vorlickovaauthor=I.+Kejnovskaauthor=K.+Bednarovaauthor=D.+Renciukauthor=J.+Kypr&title=Circular+dichroism+spectroscopy+of+DNA%3A+from+duplexes+to+quadruplexes&doi=10.1002%2Fchir.22064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Circular Dichroism Spectroscopy of DNA: From Duplexes to Quadruplexes</span></div><div class="casAuthors">Vorlickova, Michaela; Kejnovska, Iva; Bednarova, Klara; Renciuk, Daniel; Kypr, Jaroslav</div><div class="citationInfo"><span class="NLM_cas:title">Chirality</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">691-698</span>CODEN:
                <span class="NLM_cas:coden">CHRLEP</span>;
        ISSN:<span class="NLM_cas:issn">0899-0042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleic acids bear the genetic information and participate in its expression and evolution during replication, repair, recombination, transcription, and translation.  These phenomena are mostly based on recognition of nucleic acids by proteins.  The major factor enabling the specific recognition is structure.  CD spectroscopy is very useful to study secondary structures of nucleic acids, in general, and DNA, in particular.  CD sensitively reflects isomerizations among distinct conformational states.  The isomerizations may operate as mol. switches regulating various physiol. or pathol. processes.  Here, we review CD spectra of nucleic acids, beginning with early studies on natural DNA mols. through analyses of synthetic polynucleotides to study of selected genomic fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIb90iB7srw7Vg90H21EOLACvtfcHk0lj6grlzRYwPMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVeiurc%253D&md5=06d157cb05249eae6a6ba8c6f5f3e3be</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fchir.22064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchir.22064%26sid%3Dliteratum%253Aachs%26aulast%3DVorlickova%26aufirst%3DM.%26aulast%3DKejnovska%26aufirst%3DI.%26aulast%3DBednarova%26aufirst%3DK.%26aulast%3DRenciuk%26aufirst%3DD.%26aulast%3DKypr%26aufirst%3DJ.%26atitle%3DCircular%2520dichroism%2520spectroscopy%2520of%2520DNA%253A%2520from%2520duplexes%2520to%2520quadruplexes%26jtitle%3DChirality%26date%3D2012%26volume%3D24%26spage%3D691%26epage%3D698%26doi%3D10.1002%2Fchir.22064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Mergny, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaires, J. B.</span><span> </span><span class="NLM_article-title">Thermal difference spectra: a specific signature for nucleic acid structures</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_elocation-id">e138</span><span class="refDoi"> DOI: 10.1093/nar/gni134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgni134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16157860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&author=J.+L.+Mergnyauthor=J.+Liauthor=L.+Lacroixauthor=S.+Amraneauthor=J.+B.+Chaires&title=Thermal+difference+spectra%3A+a+specific+signature+for+nucleic+acid+structures&doi=10.1093%2Fnar%2Fgni134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal difference spectra: a specific signature for nucleic acid structures</span></div><div class="casAuthors">Mergny, Jean-Louis; Li, Jing; Lacroix, Laurent; Amrane, Samir; Chaires, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">e138/1-e138/6</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The authors show that nucleic acid structures may be conveniently and inexpensively characterized by their UV thermal difference spectra.  A thermal difference spectrum (TDS) is obtained for a nucleic acid by simply recording the UV absorbance spectra of the unfolded and folded states at temps. above and below its melting temp. (Tm).  The difference between these two spectra is the TDS.  The TDS has a specific shape that is unique for each type of nucleic acid structure, a conclusion that is based on a comparison of >900 spectra from 200 different sequences.  The shape of the TDS reflects the subtleties of base stacking interactions that occur uniquely within each type of nucleic acid structure.  TDS provides a simple, inexpensive and rapid method to obtain structural insight into nucleic acid structures, which is applicable to both DNA and RNA from short oligomers to polynucleotides.  TDS complements CD as a tool for the structural characterization of nucleic acids in soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZDiohxvD4qbVg90H21EOLACvtfcHk0lj6grlzRYwPMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsrrF&md5=d8ed2fd2a0bbe38e4a04dfc19c999c17</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgni134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgni134%26sid%3Dliteratum%253Aachs%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26atitle%3DThermal%2520difference%2520spectra%253A%2520a%2520specific%2520signature%2520for%2520nucleic%2520acid%2520structures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26doi%3D10.1093%2Fnar%2Fgni134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Yangyuoru, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghimire, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, H.</span><span> </span><span class="NLM_article-title">Dual binding of an antibody and a small molecule increases the stability of TERRA G-quadruplex</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">910</span><span class="NLM_x">–</span> <span class="NLM_lpage">913</span><span class="refDoi"> DOI: 10.1002/anie.201408113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1002%2Fanie.201408113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=910-913&author=P.+M.+Yangyuoruauthor=M.+Di+Antonioauthor=C.+Ghimireauthor=G.+Biffiauthor=S.+Balasubramanianauthor=H.+Mao&title=Dual+binding+of+an+antibody+and+a+small+molecule+increases+the+stability+of+TERRA+G-quadruplex&doi=10.1002%2Fanie.201408113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fanie.201408113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201408113%26sid%3Dliteratum%253Aachs%26aulast%3DYangyuoru%26aufirst%3DP.%2BM.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DGhimire%26aufirst%3DC.%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DH.%26atitle%3DDual%2520binding%2520of%2520an%2520antibody%2520and%2520a%2520small%2520molecule%2520increases%2520the%2520stability%2520of%2520TERRA%2520G-quadruplex%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D910%26epage%3D913%26doi%3D10.1002%2Fanie.201408113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0lj6grlzRYwPMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Miserachs, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johannsen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigel, R. K. O.</span><span> </span><span class="NLM_article-title">Distinct differences in metal ion specificity of RNA and DNA G-quadruplexes</span> <span class="citation_source-journal">JBIC, J. Biol. Inorg. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span><span class="refDoi"> DOI: 10.1007/s00775-016-1393-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs00775-016-1393-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27704222" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=975-986&author=H.+G.+Miserachsauthor=D.+Donghiauthor=R.+Bornerauthor=S.+Johannsenauthor=R.+K.+O.+Sigel&title=Distinct+differences+in+metal+ion+specificity+of+RNA+and+DNA+G-quadruplexes&doi=10.1007%2Fs00775-016-1393-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs00775-016-1393-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00775-016-1393-4%26sid%3Dliteratum%253Aachs%26aulast%3DMiserachs%26aufirst%3DH.%2BG.%26aulast%3DDonghi%26aufirst%3DD.%26aulast%3DBorner%26aufirst%3DR.%26aulast%3DJohannsen%26aufirst%3DS.%26aulast%3DSigel%26aufirst%3DR.%2BK.%2BO.%26atitle%3DDistinct%2520differences%2520in%2520metal%2520ion%2520specificity%2520of%2520RNA%2520and%2520DNA%2520G-quadruplexes%26jtitle%3DJBIC%252C%2520J.%2520Biol.%2520Inorg.%2520Chem.%26date%3D2016%26volume%3D21%26spage%3D975%26epage%3D986%26doi%3D10.1007%2Fs00775-016-1393-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">McPike, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodisman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dabrowiak, J. C.</span><span> </span><span class="NLM_article-title">Drug-RNA footprinting</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">449</span><span class="refDoi"> DOI: 10.1016/S0076-6879(01)40435-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2FS0076-6879%2801%2940435-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2001&pages=431-449&author=M.+P.+McPikeauthor=J.+Goodismanauthor=J.+C.+Dabrowiak&title=Drug-RNA+footprinting&doi=10.1016%2FS0076-6879%2801%2940435-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2801%2940435-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252801%252940435-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcPike%26aufirst%3DM.%2BP.%26aulast%3DGoodisman%26aufirst%3DJ.%26aulast%3DDabrowiak%26aufirst%3DJ.%2BC.%26atitle%3DDrug-RNA%2520footprinting%26jtitle%3DMethods%2520Enzymol.%26date%3D2001%26volume%3D340%26spage%3D431%26epage%3D449%26doi%3D10.1016%2FS0076-6879%2801%2940435-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title">Thioflavin T as an efficient fluorescence sensor for selective recognition of RNA G-quadruplexes</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">24793</span><span class="refDoi"> DOI: 10.1038/srep24793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fsrep24793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27098781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Kntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=24793&author=S.+Xuauthor=Q.+Liauthor=J.+Xiangauthor=Q.+Yangauthor=H.+Sunauthor=A.+Guanauthor=L.+Wangauthor=Y.+Liuauthor=L.+Yuauthor=Y.+Shiauthor=H.+Chenauthor=Y.+Tang&title=Thioflavin+T+as+an+efficient+fluorescence+sensor+for+selective+recognition+of+RNA+G-quadruplexes&doi=10.1038%2Fsrep24793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Thioflavin T as an efficient fluorescence sensor for selective recognition of RNA G-quadruplexes</span></div><div class="casAuthors">Xu, Shujuan; Li, Qian; Xiang, Junfeng; Yang, Qianfan; Sun, Hongxia; Guan, Aijiao; Wang, Lixia; Liu, Yan; Yu, Lijia; Shi, Yunhua; Chen, Hongbo; Tang, Yalin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24793</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplexes (G4s) play important roles in translational regulation, mRNA processing events and gene expression.  Therefore, a fluorescent probe that is capable of efficiently recognizing RNA G-quadruplex structures among other RNA forms is highly desirable.  In this study, a water-sol. fluorogenic dye (i.e., Thioflavin T (ThT)) was employed to recognize RNA G-quadruplex structures using UV-Vis absorption spectra, fluorescence spectra and emission lifetime expts.  By stacking on the G-tetrad, the ThT probe exhibited highly specific recognition of RNA G-quadruplex structures with striking fluorescence enhancement compared with other RNA forms.  The specific binding demonstrates that ThT is an efficient fluorescence sensor that can distinguish G4 and non-G4 RNA structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpil8g_rgoR2LVg90H21EOLACvtfcHk0lgsp3Jc5fj8Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Kntr8%253D&md5=3ae08487d53162f02c83eb03b19e5b8d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fsrep24793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep24793%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DThioflavin%2520T%2520as%2520an%2520efficient%2520fluorescence%2520sensor%2520for%2520selective%2520recognition%2520of%2520RNA%2520G-quadruplexes%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D24793%26doi%3D10.1038%2Fsrep24793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Weldon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behm-Ansmant, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branlant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eperon, I. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, C.</span><span> </span><span class="NLM_article-title">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1038/nchembio.2228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchembio.2228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27820800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=18-20&author=C.+Weldonauthor=I.+Behm-Ansmantauthor=L.+H.+Hurleyauthor=G.+A.+Burleyauthor=C.+Branlantauthor=I.+C.+Eperonauthor=C.+Dominguez&title=Identification+of+G-quadruplexes+in+long+functional+RNAs+using+7-deazaguanine+RNA&doi=10.1038%2Fnchembio.2228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span></div><div class="casAuthors">Weldon, Carika; Behm-Ansmant, Isabelle; Hurley, Laurence H.; Burley, Glenn A.; Branlant, Christiane; Eperon, Ian C.; Dominguez, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-20</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplex (G4) structures are thought to affect biol. processes, including translation and pre-mRNA splicing, but it is not possible at present to demonstrate that they form naturally at specific sequences in long functional RNA mols.  We developed a new strategy, footprinting of long 7-deazaguanine-substituted RNAs (FOLDeR), that allows the formation of G4s to be confirmed in long RNAs and under functional conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEnw-nGuf4ZLVg90H21EOLACvtfcHk0lhL0_dqO_3cSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN&md5=b1f547bacfc316a5544d6901bab73533</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2228%26sid%3Dliteratum%253Aachs%26aulast%3DWeldon%26aufirst%3DC.%26aulast%3DBehm-Ansmant%26aufirst%3DI.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBurley%26aufirst%3DG.%2BA.%26aulast%3DBranlant%26aufirst%3DC.%26aulast%3DEperon%26aufirst%3DI.%2BC.%26aulast%3DDominguez%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520G-quadruplexes%2520in%2520long%2520functional%2520RNAs%2520using%25207-deazaguanine%2520RNA%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D18%26epage%3D20%26doi%3D10.1038%2Fnchembio.2228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Small molecule-mediated inhibition of translation by targeting a native RNA G-quadruplex</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2771</span><span class="NLM_x">–</span> <span class="NLM_lpage">2776</span><span class="refDoi"> DOI: 10.1039/c002418j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1039%2Fc002418j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=20436976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=2771-2776&author=A.+Bugautauthor=R.+Rodriguezauthor=S.+Kumariauthor=S.+T.+Hsuauthor=S.+Balasubramanian&title=Small+molecule-mediated+inhibition+of+translation+by+targeting+a+native+RNA+G-quadruplex&doi=10.1039%2Fc002418j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule-mediated inhibition of translation by targeting a native RNA G-quadruplex</span></div><div class="casAuthors">Bugaut, Anthony; Rodriguez, Raphael; Kumari, Sunita; Hsu, Shang-Te Danny; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2771-2776</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein, we show that a naturally occurring RNA G-quadruplex element within the 5' UTR of the human NRAS proto-oncogene is a target for a small mol. that inhibits translation in vitro.  The present study provides a first demonstration that natural 5' UTR mRNA G-quadruplexes have potential as mol. targets for small mols. that modulate translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppcJ1Ja-L2T7Vg90H21EOLACvtfcHk0lhL0_dqO_3cSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWktL0%253D&md5=6bb9cf5b26189a75207c0721662082e2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2Fc002418j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc002418j%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DHsu%26aufirst%3DS.%2BT.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DSmall%2520molecule-mediated%2520inhibition%2520of%2520translation%2520by%2520targeting%2520a%2520native%2520RNA%2520G-quadruplex%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2010%26volume%3D8%26spage%3D2771%26epage%3D2776%26doi%3D10.1039%2Fc002418j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Katsuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furuta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagihara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesugi, M.</span><span> </span><span class="NLM_article-title">A small molecule that represses translation of G-quadruplex-containing mRNA</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">9037</span><span class="NLM_x">–</span> <span class="NLM_lpage">9040</span><span class="refDoi"> DOI: 10.1021/jacs.6b04506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=9037-9040&author=Y.+Katsudaauthor=S.+Satoauthor=L.+Asanoauthor=Y.+Morimuraauthor=T.+Furutaauthor=H.+Sugiyamaauthor=M.+Hagiharaauthor=M.+Uesugi&title=A+small+molecule+that+represses+translation+of+G-quadruplex-containing+mRNA&doi=10.1021%2Fjacs.6b04506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule that represses translation of G-quadruplex-containing mRNA</span></div><div class="casAuthors">Katsuda, Yousuke; Sato, Shin-ichi; Asano, Lisa; Morimura, Yoshitaka; Furuta, Tomoyuki; Sugiyama, Hiroshi; Hagihara, Masaki; Uesugi, Motonari</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9037-9040</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G-quadruplexes form highly stable nucleic acid structures, which are implicated in various biol. processes in both DNA and RNA.  Although DNA G-quadruplexes have been studied in great detail, biol. roles of RNA G-quadruplexes have received less attention.  Here, a screening of a chem. library permitted identification of a small-mol. tool that binds selectively to RNA G-quadruplex structures.  The polyarom. mol., RGB-1, stabilizes RNA G-quadruplex, but not DNA versions or other RNA structures.  RGB-1 intensified the G-quadruplex-mediated inhibition of RNA translation in mammalian cells, decreased expression of the NRAS proto-oncogene in breast cancer cells, and permitted identification of a novel sequence that forms G-quadruplex in NRAS mRNA.  RGB-1 may serve as a unique tool for understanding cellular roles of RNA G-quadruplex structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-OEuiG5IuLVg90H21EOLACvtfcHk0ljsWzjklPOXDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP&md5=75ad58de109aa50cc964db46bc46f69b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04506%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DAsano%26aufirst%3DL.%26aulast%3DMorimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DHagihara%26aufirst%3DM.%26aulast%3DUesugi%26aufirst%3DM.%26atitle%3DA%2520small%2520molecule%2520that%2520represses%2520translation%2520of%2520G-quadruplex-containing%2520mRNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D9037%26epage%3D9040%26doi%3D10.1021%2Fjacs.6b04506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Membrino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinkevich, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Guanidino anthrathiophenediones as G-quadruplex binders: uptake, intracellular localization, and anti-Harvey-Ras gene activity in bladder cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2764</span><span class="NLM_x">–</span> <span class="NLM_lpage">2778</span><span class="refDoi"> DOI: 10.1021/jm3019063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3019063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2764-2778&author=S.+Cogoiauthor=A.+E.+Shchekotikhinauthor=A.+Membrinoauthor=Y.+B.+Sinkevichauthor=L.+E.+Xodo&title=Guanidino+anthrathiophenediones+as+G-quadruplex+binders%3A+uptake%2C+intracellular+localization%2C+and+anti-Harvey-Ras+gene+activity+in+bladder+cancer+cells&doi=10.1021%2Fjm3019063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm3019063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3019063%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DMembrino%26aufirst%3DA.%26aulast%3DSinkevich%26aufirst%3DY.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DGuanidino%2520anthrathiophenediones%2520as%2520G-quadruplex%2520binders%253A%2520uptake%252C%2520intracellular%2520localization%252C%2520and%2520anti-Harvey-Ras%2520gene%2520activity%2520in%2520bladder%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2764%26epage%3D2778%26doi%3D10.1021%2Fjm3019063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Potent apoptotic response induced by chloroacetamidine anthrathiophenediones in bladder cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5476</span><span class="NLM_x">–</span> <span class="NLM_lpage">5485</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00409</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00409" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5476-5485&author=S.+Cogoiauthor=S.+Zorzetauthor=A.+E.+Shchekotikhinauthor=L.+E.+Xodo&title=Potent+apoptotic+response+induced+by+chloroacetamidine+anthrathiophenediones+in+bladder+cancer+cells&doi=10.1021%2Facs.jmedchem.5b00409"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00409%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DPotent%2520apoptotic%2520response%2520induced%2520by%2520chloroacetamidine%2520anthrathiophenediones%2520in%2520bladder%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5476%26epage%3D5485%26doi%3D10.1021%2Facs.jmedchem.5b00409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glazunova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dezhenkova, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzikov, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinkevich, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovalenko, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buyanov, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shtil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">Synthesis and cytotoxic properties of 4,11-bis[(aminoethyl)amino]anthra[2,3-<i>b</i>]thiophene-5,10-diones, novel analogues of antitumor anthracene-9,10-diones</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1861</span><span class="NLM_x">–</span> <span class="NLM_lpage">1869</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.01.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bmc.2009.01.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1861-1869&author=A.+E.+Shchekotikhinauthor=V.+A.+Glazunovaauthor=L.+G.+Dezhenkovaauthor=Y.+N.+Luzikovauthor=Y.+B.+Sinkevichauthor=L.+V.+Kovalenkoauthor=V.+N.+Buyanovauthor=J.+Balzariniauthor=A.+A.+Shtilauthor=M.+N.+Preobrazhenskaya&title=Synthesis+and+cytotoxic+properties+of+4%2C11-bis%5B%28aminoethyl%29amino%5Danthra%5B2%2C3-b%5Dthiophene-5%2C10-diones%2C+novel+analogues+of+antitumor+anthracene-9%2C10-diones&doi=10.1016%2Fj.bmc.2009.01.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.01.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.01.047%26sid%3Dliteratum%253Aachs%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DGlazunova%26aufirst%3DV.%2BA.%26aulast%3DDezhenkova%26aufirst%3DL.%2BG.%26aulast%3DLuzikov%26aufirst%3DY.%2BN.%26aulast%3DSinkevich%26aufirst%3DY.%2BB.%26aulast%3DKovalenko%26aufirst%3DL.%2BV.%26aulast%3DBuyanov%26aufirst%3DV.%2BN.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DShtil%26aufirst%3DA.%2BA.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DSynthesis%2520and%2520cytotoxic%2520properties%2520of%25204%252C11-bis%255B%2528aminoethyl%2529amino%255Danthra%255B2%252C3-b%255Dthiophene-5%252C10-diones%252C%2520novel%2520analogues%2520of%2520antitumor%2520anthracene-9%252C10-diones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D1861%26epage%3D1869%26doi%3D10.1016%2Fj.bmc.2009.01.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span> </span><span class="NLM_article-title">Methods for the synthesis and modification of linear anthrafurandiones</span> <span class="citation_source-journal">Chem. Heterocycl. Compd.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1007/s10593-014-1459-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs10593-014-1459-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=171-184&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=M.+N.+Preobrazhenskaya&title=Methods+for+the+synthesis+and+modification+of+linear+anthrafurandiones&doi=10.1007%2Fs10593-014-1459-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for the Synthesis and Modification of Linear Anthrafurandiones (Review)</span></div><div class="casAuthors">Tikhomirov, A. S.; Shchekotikhin, A. E.; Preobrazhenskaya, M. N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-184</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Derivs. of anthra[2,3-b]furan-5,10-diones hold promise for various fields of science and technol., esp. in the search for biol. active compds.  This review covers all currently known methods for the cyclization of linear anthrafurandiones, the first examples of which were synthesized approx. 30 years ago.  The material has been arranged mainly in the order of increasing degree of substitution in anthrafurandiones.  All the literature methods for functional group transformations providing access to a range of anthrafurandiones unavailable by direct heterocyclization have been considered sep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE67DaXvt5zLVg90H21EOLACvtfcHk0lhRC8tG6DAyCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCktb4%253D&md5=d10d1fd2d3ed4f3e934d956386d14a29</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10593-014-1459-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-014-1459-6%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DMethods%2520for%2520the%2520synthesis%2520and%2520modification%2520of%2520linear%2520anthrafurandiones%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2014%26volume%3D50%26spage%3D171%26epage%3D184%26doi%3D10.1007%2Fs10593-014-1459-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Gronowitz, S.</span><span> </span><span class="NLM_article-title">Thiophene and Its Derivatives</span>, Part 2.  <span class="citation_source-book">The Chemistry of Heterocyclic Compounds</span>; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1986</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">44</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1986&author=S.+Gronowitz&title=The+Chemistry+of+Heterocyclic+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGronowitz%26aufirst%3DS.%26atitle%3DThiophene%2520and%2520Its%2520Derivatives%26btitle%3DThe%2520Chemistry%2520of%2520Heterocyclic%2520Compounds%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1986%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatarskiy, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dezhenkova, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glazunova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shtil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chueh, P. J.</span><span> </span><span class="NLM_article-title">Synthesis and characterization of 4,11-diaminoanthra[2,3-<i>b</i>]furan-5,10-diones: tumor cell apoptosis through tNOX-modulated NAD<sup>+</sup>/NADH ratio and SIRT1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9522</span><span class="NLM_x">–</span> <span class="NLM_lpage">9534</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00859</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00859" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9522-9534&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=Y.+H.+Leeauthor=Y.+A.+Chenauthor=C.+A.+Yehauthor=V.+V.+Tatarskiyauthor=L.+G.+Dezhenkovaauthor=V.+A.+Glazunovaauthor=J.+Balzariniauthor=A.+A.+Shtilauthor=M.+N.+Preobrazhenskayaauthor=P.+J.+Chueh&title=Synthesis+and+characterization+of+4%2C11-diaminoanthra%5B2%2C3-b%5Dfuran-5%2C10-diones%3A+tumor+cell+apoptosis+through+tNOX-modulated+NAD%2B%2FNADH+ratio+and+SIRT1&doi=10.1021%2Facs.jmedchem.5b00859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Characterization of 4,11-Diaminoanthra[2,3-b]furan-5,10-diones: Tumor Cell Apoptosis through tNOX-Modulated NAD+/NADH Ratio and SIRT1</span></div><div class="casAuthors">Tikhomirov, Alexander S.; Shchekotikhin, Andrey E.; Lee, Yi-Hui; Chen, Yi-Ann; Yeh, Chia-An; Tatarskiy, Victor V.; Dezhenkova, Lyubov G.; Glazunova, Valeria A.; Balzarini, Jan; Shtil, Alexander A.; Preobrazhenskaya, Maria N.; Chueh, Pin Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9522-9534</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 4,11-diaminoanthra[2,3-b]furan-5,10-dione derivs. with different side chains were synthesized.  Selected 2-unsubstituted derivs. I [R = NH2, NHMe, NMe2, NH(CH2)2OH ] showed high antiproliferative potency on a panel of mammalian tumor cell lines including multidrug resistance variants.  Compds. I [R = NH2, NHMe, NMe2, NH(CH2)2OH ] utilized multiple mechanisms of cytotoxicity including inhibition of Top1/Top2-mediated DNA relaxation, reduced NAD+/NADH ratio through tNOX inhibition, suppression of a NAD+-dependent sirtuin 1 (SIRT1) deacetylase activity, and activation of caspase-mediated apoptosis.  Here, for the first time, we report that tumor-assocd. NADH oxidase (tNOX) and SIRT1 are important cellular targets of antitumor anthracene-9,10-diones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFsylgKt5c07Vg90H21EOLACvtfcHk0lhRC8tG6DAyCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsLzI&md5=262e35de4b1c5b845e49c29da205e955</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00859%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DLee%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BA.%26aulast%3DYeh%26aufirst%3DC.%2BA.%26aulast%3DTatarskiy%26aufirst%3DV.%2BV.%26aulast%3DDezhenkova%26aufirst%3DL.%2BG.%26aulast%3DGlazunova%26aufirst%3DV.%2BA.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DShtil%26aufirst%3DA.%2BA.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26aulast%3DChueh%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520characterization%2520of%25204%252C11-diaminoanthra%255B2%252C3-b%255Dfuran-5%252C10-diones%253A%2520tumor%2520cell%2520apoptosis%2520through%2520tNOX-modulated%2520NAD%252B%252FNADH%2520ratio%2520and%2520SIRT1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9522%26epage%3D9534%26doi%3D10.1021%2Facs.jmedchem.5b00859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Sabatino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paganelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leone, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papini, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Luca, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginanneschi, M.</span><span> </span><span class="NLM_article-title">A new biotin derivative-DOTA conjugate as a candidate for pretargeted diagnosis and therapy of tumors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3170</span><span class="NLM_x">–</span> <span class="NLM_lpage">3173</span><span class="refDoi"> DOI: 10.1021/jm030789z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030789z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=3170-3173&author=G.+Sabatinoauthor=M.+Chinolauthor=G.+Paganelliauthor=S.+Papiauthor=M.+Chelliauthor=G.+Leoneauthor=A.+M.+Papiniauthor=A.+De+Lucaauthor=M.+Ginanneschi&title=A+new+biotin+derivative-DOTA+conjugate+as+a+candidate+for+pretargeted+diagnosis+and+therapy+of+tumors&doi=10.1021%2Fjm030789z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm030789z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030789z%26sid%3Dliteratum%253Aachs%26aulast%3DSabatino%26aufirst%3DG.%26aulast%3DChinol%26aufirst%3DM.%26aulast%3DPaganelli%26aufirst%3DG.%26aulast%3DPapi%26aufirst%3DS.%26aulast%3DChelli%26aufirst%3DM.%26aulast%3DLeone%26aufirst%3DG.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DDe%2BLuca%26aufirst%3DA.%26aulast%3DGinanneschi%26aufirst%3DM.%26atitle%3DA%2520new%2520biotin%2520derivative-DOTA%2520conjugate%2520as%2520a%2520candidate%2520for%2520pretargeted%2520diagnosis%2520and%2520therapy%2520of%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D3170%26epage%3D3173%26doi%3D10.1021%2Fjm030789z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Huang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span> </span><span class="NLM_article-title">Synthesis of biotin–AMP conjugate for 5′ biotin labeling of RNA through one-step in vitro transcription</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1848</span><span class="NLM_x">–</span> <span class="NLM_lpage">1861</span><span class="refDoi"> DOI: 10.1038/nprot.2008.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnprot.2008.185" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1848-1861&author=F.+Huangauthor=J.+Heauthor=Y.+Zhangauthor=Y.+Guo&title=Synthesis+of+biotin%E2%80%93AMP+conjugate+for+5%E2%80%B2+biotin+labeling+of+RNA+through+one-step+in+vitro+transcription&doi=10.1038%2Fnprot.2008.185"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.185%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520biotin%25E2%2580%2593AMP%2520conjugate%2520for%25205%25E2%2580%25B2%2520biotin%2520labeling%2520of%2520RNA%2520through%2520one-step%2520in%2520vitro%2520transcription%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1848%26epage%3D1861%26doi%3D10.1038%2Fnprot.2008.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Isaac-Lam, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammonds, D. M.</span><span> </span><span class="NLM_article-title">Biotinylated chlorin and its zinc and indium complexes: synthesis and in vitro biological evaluation for photodynamic therapy</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="refDoi"> DOI: 10.3390/ph10020041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.3390%2Fph10020041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCqsLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=41&author=M.+F.+Isaac-Lamauthor=D.+M.+Hammonds&title=Biotinylated+chlorin+and+its+zinc+and+indium+complexes%3A+synthesis+and+in+vitro+biological+evaluation+for+photodynamic+therapy&doi=10.3390%2Fph10020041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biotinylated chlorin and its zinc and indium complexes: synthesis and in vitro biological evaluation for photodynamic therapy</span></div><div class="casAuthors">Isaac-Lam, Meden F.; Hammonds, Dewana M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">41/1-41/12</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The synthesis and characterization of biotinylated chlorin photosensitizer and the corresponding zinc and indium complexes are described for potential applications in photodynamic therapy (PDT) for cancer.  Phototoxicity of the biotin-chlorin conjugate and the metalated complexes was detd. in colon carcinoma CT26 cell lines known to overexpress biotin (Vit B7) receptors.  Cell survival assay indicated that the biotinylated chlorin and indium complex showed increased cell growth inhibition than the zinc complex and the starting chlorin (Me pheophorbide).  Fluorescence microcopy studies revealed the generation of apoptotic cells upon light irradn. of colon cells treated with the indium complex.  Targeting biotin receptors in cancer cells can improve specificity of photosensitizers for PDT applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5wyaEGaU9LVg90H21EOLACvtfcHk0lgSQpj_6Iswww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCqsLfI&md5=41721d161fee8993cca4ea706d134482</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3390%2Fph10020041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph10020041%26sid%3Dliteratum%253Aachs%26aulast%3DIsaac-Lam%26aufirst%3DM.%2BF.%26aulast%3DHammonds%26aufirst%3DD.%2BM.%26atitle%3DBiotinylated%2520chlorin%2520and%2520its%2520zinc%2520and%2520indium%2520complexes%253A%2520synthesis%2520and%2520in%2520vitro%2520biological%2520evaluation%2520for%2520photodynamic%2520therapy%26jtitle%3DPharmaceuticals%26date%3D2017%26volume%3D10%26spage%3D41%26doi%3D10.3390%2Fph10020041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Tikhomirov, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzikov, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korolev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preobrazhenskaya, M. N.</span><span> </span><span class="NLM_article-title">Heterocyclic analogs of 5,12-naphthacene-Quinone. 11. A new method for preparing 4,11-dimethoxyanthra[2,3-<i>b</i>]furan-5,10-dione</span> <span class="citation_source-journal">Chem. Heterocycl. Compd.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1206</span><span class="NLM_x">–</span> <span class="NLM_lpage">1211</span><span class="refDoi"> DOI: 10.1007/s10593-012-0895-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2Fs10593-012-0895-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=1206-1211&author=A.+S.+Tikhomirovauthor=A.+E.+Shchekotikhinauthor=Y.+N.+Luzikovauthor=A.+M.+Korolevauthor=M.+N.+Preobrazhenskaya&title=Heterocyclic+analogs+of+5%2C12-naphthacene-Quinone.+11.+A+new+method+for+preparing+4%2C11-dimethoxyanthra%5B2%2C3-b%5Dfuran-5%2C10-dione&doi=10.1007%2Fs10593-012-0895-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2Fs10593-012-0895-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-012-0895-4%26sid%3Dliteratum%253Aachs%26aulast%3DTikhomirov%26aufirst%3DA.%2BS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DLuzikov%26aufirst%3DY.%2BN.%26aulast%3DKorolev%26aufirst%3DA.%2BM.%26aulast%3DPreobrazhenskaya%26aufirst%3DM.%2BN.%26atitle%3DHeterocyclic%2520analogs%2520of%25205%252C12-naphthacene-Quinone.%252011.%2520A%2520new%2520method%2520for%2520preparing%25204%252C11-dimethoxyanthra%255B2%252C3-b%255Dfuran-5%252C10-dione%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2011%26volume%3D47%26spage%3D1206%26epage%3D1211%26doi%3D10.1007%2Fs10593-012-0895-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Human telomeric G-quadruplex: structures of DNA and RNA sequences</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2009.07464.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1111%2Fj.1742-4658.2009.07464.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=19951353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivVSmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=1107-1117&author=A.+T.+Phan&title=Human+telomeric+G-quadruplex%3A+structures+of+DNA+and+RNA+sequences&doi=10.1111%2Fj.1742-4658.2009.07464.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Human telomeric G-quadruplex: structures of DNA and RNA sequences</span></div><div class="casAuthors">Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1107-1117</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Telomeres play an important role in cellular aging and cancer.  Human telomeric DNA and RNA G-rich sequences are capable of forming a 4-stranded structure, known as the G-quadruplex.  Such a structure might be important for telomere biol. and a good target for drug design.  Here, the author describes the structural diversity or conservation of DNA and RNA human telomeric G-quadruplexes, discusses structural views on targeting these G-quadruplexes, and presents some future challenges for structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjtvxCsiVbsrVg90H21EOLACvtfcHk0lgSQpj_6Iswww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivVSmsbc%253D&md5=341396c7c9e0600f09f9beaee1fc0879</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2009.07464.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2009.07464.x%26sid%3Dliteratum%253Aachs%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DHuman%2520telomeric%2520G-quadruplex%253A%2520structures%2520of%2520DNA%2520and%2520RNA%2520sequences%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D1107%26epage%3D1117%26doi%3D10.1111%2Fj.1742-4658.2009.07464.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shchekotikhin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">HRAS is silenced by two neighboring G-quadruplexes and activated by MAZ, a zinc-finger transcription factor with DNA unfolding property</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">8379</span><span class="NLM_x">–</span> <span class="NLM_lpage">8388</span><span class="refDoi"> DOI: 10.1093/nar/gku574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgku574" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=8379-8388&author=S.+Cogoiauthor=A.+E.+Shchekotikhinauthor=L.+E.+Xodo&title=HRAS+is+silenced+by+two+neighboring+G-quadruplexes+and+activated+by+MAZ%2C+a+zinc-finger+transcription+factor+with+DNA+unfolding+property&doi=10.1093%2Fnar%2Fgku574"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku574%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DShchekotikhin%26aufirst%3DA.%2BE.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DHRAS%2520is%2520silenced%2520by%2520two%2520neighboring%2520G-quadruplexes%2520and%2520activated%2520by%2520MAZ%252C%2520a%2520zinc-finger%2520transcription%2520factor%2520with%2520DNA%2520unfolding%2520property%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8379%26epage%3D8388%26doi%3D10.1093%2Fnar%2Fgku574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cauci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Critical role of hnRNP A1 in activating KRAS transcription in pancreatic cancer cells: A molecular mechanism involving G4 DNA</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">1861</span><span class="NLM_x">, </span> <span class="NLM_fpage">1389</span><span class="NLM_x">–</span> <span class="NLM_lpage">1398</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2016.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.bbagen.2016.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=27888145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSmsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2017&pages=1389-1398&author=S.+Cogoiauthor=V.+Rapozziauthor=S.+Cauciauthor=L.+E.+Xodo&title=Critical+role+of+hnRNP+A1+in+activating+KRAS+transcription+in+pancreatic+cancer+cells%3A+A+molecular+mechanism+involving+G4+DNA&doi=10.1016%2Fj.bbagen.2016.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of hnRNP A1 in activating KRAS transcription in pancreatic cancer cells: A molecular mechanism involving G4 DNA</span></div><div class="casAuthors">Cogoi, Susanna; Rapozzi, Valentina; Cauci, Sabina; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">5_Part_B</span>),
    <span class="NLM_cas:pages">1389-1398</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KRAS is one of the most mutated genes in human cancer.  Its crucial role in the tumorigenesis of pancreatic ductal adenocarcinoma (PDAC) has been widely demonstrated.  As this deadly cancer does not sufficiently respond to conventional chemotherapies, it is important to increase our knowledge of pancreatic cancer biol., in particular how oncogenic KRAS is regulated.  The promoter of KRAS contains a GA-element composed of runs of guanines that fold into a G4 structure.  This unusual DNA conformation is recognized by several nuclear proteins, including MAZ and hnRNP A1.  Recent data have revealed that KRAS is interconnected to ILK and hnRNP A1 in a circuitry that enables pancreatic cancer cells to maintain an aggressive phenotype.  The present review illustrates recent advances on how KRAS is regulated in pancreatic cancer cells, focusing on the formation of G4 structures in the KRAS promoter and their interaction with hnRNP A1.  The newly discovered KRAS-ILK-hnRNP A1 regulatory loop is discussed, emphasizing its potential as a therapeutic target for PDAC-specific mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9A7QOe721LVg90H21EOLACvtfcHk0lgiF3B_Y8PXZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSmsbzP&md5=43a9ecdbf65e6a7e027a063206738bb4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2016.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2016.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DCauci%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DCritical%2520role%2520of%2520hnRNP%2520A1%2520in%2520activating%2520KRAS%2520transcription%2520in%2520pancreatic%2520cancer%2520cells%253A%2520A%2520molecular%2520mechanism%2520involving%2520G4%2520DNA%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2017%26volume%3D1861%26spage%3D1389%26epage%3D1398%26doi%3D10.1016%2Fj.bbagen.2016.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0lgiF3B_Y8PXZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorzet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Géci, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">MAZ-binding G4-decoy with locked nucleic acid and twisted intercalating nucleic acid modifications suppresses KRAS in pancreatic cancer cells and delays tumor growth in mice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4049</span><span class="NLM_x">–</span> <span class="NLM_lpage">4064</span><span class="refDoi"> DOI: 10.1093/nar/gkt127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1093%2Fnar%2Fgkt127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=23471001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=4049-4064&author=S.+Cogoiauthor=S.+Zorzetauthor=V.+Rapozziauthor=I.+G%C3%A9ciauthor=E.+B.+Pedersenauthor=L.+E.+Xodo&title=MAZ-binding+G4-decoy+with+locked+nucleic+acid+and+twisted+intercalating+nucleic+acid+modifications+suppresses+KRAS+in+pancreatic+cancer+cells+and+delays+tumor+growth+in+mice&doi=10.1093%2Fnar%2Fgkt127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MAZ-binding G4-decoy with locked nucleic acid and twisted intercalating nucleic acid modifications suppresses KRAS in pancreatic cancer cells and delays tumor growth in mice</span></div><div class="casAuthors">Cogoi, Susanna; Zorzet, Sonia; Rapozzi, Valentina; Geci, Imrich; Pedersen, Erik B.; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4049-4064</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">KRAS mutations are primary genetic lesions leading to pancreatic cancer.  The promoter of human KRAS contains a nuclease-hypersensitive element (NHE) that can fold in G4-DNA structures binding to nuclear proteins, including MAZ (myc-assocd. zinc-finger).  Here, we report that MAZ activates KRAS transcription.  To knockdown oncogenic KRAS in pancreatic cancer cells, we designed oligonucleotides that mimic one of the G-quadruplexes formed by NHE (G4-decoys).  To increase their nuclease resistance, two locked nucleic acid (LNA) modifications were introduced at the 3'-end, whereas to enhance the folding and stability, two polycyclic arom. hydrocarbon units (TINA or AMANY) were inserted internally, to cap the quadruplex.  The most active G4-decoy (2998), which had two para-TINAs, strongly suppressed KRAS expression in Panc-1 cells.  It also repressed their metabolic activity (IC50 = 520 nM), and it inhibited cell growth and colony formation by activating apoptosis.  We finally injected 2998 and control oligonucleotides 5153, 5154 (2 nmol/mouse) intratumorally in SCID mice bearing a Panc-1 xenograft.  After three treatments, 2998 reduced tumor xenograft growth by 64% compared with control and increased the Kaplan-Meier median survival time by 70%.  Together, our data show that MAZ-specific G4-decoys mimicking a KRAS quadruplex are promising for pancreatic cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoQEFAMt2lfrVg90H21EOLACvtfcHk0lhDaG5ONXI_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKju70%253D&md5=461de60d1eafb94201a1bd600c4829f2</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt127%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DZorzet%26aufirst%3DS.%26aulast%3DRapozzi%26aufirst%3DV.%26aulast%3DG%25C3%25A9ci%26aufirst%3DI.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DMAZ-binding%2520G4-decoy%2520with%2520locked%2520nucleic%2520acid%2520and%2520twisted%2520intercalating%2520nucleic%2520acid%2520modifications%2520suppresses%2520KRAS%2520in%2520pancreatic%2520cancer%2520cells%2520and%2520delays%2520tumor%2520growth%2520in%2520mice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D4049%26epage%3D4064%26doi%3D10.1093%2Fnar%2Fgkt127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Wlodkowic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skommer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darzynkiewicz, Z.</span><span> </span><span class="NLM_article-title">Flow cytometry-based apoptosis detection</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">559</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1007/978-1-60327-017-5_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1007%2F978-1-60327-017-5_2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=19609746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=559&publication_year=2009&pages=19-32&author=D.+Wlodkowicauthor=J.+Skommerauthor=Z.+Darzynkiewicz&title=Flow+cytometry-based+apoptosis+detection&doi=10.1007%2F978-1-60327-017-5_2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Flow cytometry-based apoptosis detection</span></div><div class="casAuthors">Wlodkowic, Donald; Skommer, Joanna; Darzynkiewicz, Zbigniew</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">559</span>
        (<span class="NLM_cas:issue">Apoptosis</span>),
    <span class="NLM_cas:pages">19-32</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">An apoptosing cell demonstrates multitude of characteristic morphol. and biochem. features, which vary depending on the stimuli and the cell type.  The gross majority of classical apoptotic hallmarks can be rapidly examd. by flow and image cytometry.  Cytometry thus became a technol. of choice in diverse studies of cellular demise.  A large variety of cytometric methods designed to identify apoptotic cells and probe mechanisms assocd. with this mode of cell demise have been developed during the past two decades.  In the present chapter, we outline a handful of commonly used methods that are based on the assessment of: mitochondrial transmembrane potential, activation of caspases, plasma membrane alterations and DNA fragmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHAI-ImJTtYLVg90H21EOLACvtfcHk0lhDaG5ONXI_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVylurk%253D&md5=648ed0c0eedf72b432173d8e9d112165</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1007%2F978-1-60327-017-5_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60327-017-5_2%26sid%3Dliteratum%253Aachs%26aulast%3DWlodkowic%26aufirst%3DD.%26aulast%3DSkommer%26aufirst%3DJ.%26aulast%3DDarzynkiewicz%26aufirst%3DZ.%26atitle%3DFlow%2520cytometry-based%2520apoptosis%2520detection%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2009%26volume%3D559%26spage%3D19%26epage%3D32%26doi%3D10.1007%2F978-1-60327-017-5_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1038/nchem.1805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1038%2Fnchem.1805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=24345950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=75-80&author=G.+Biffiauthor=M.+Di+Antonioauthor=D.+Tannahillauthor=S.+Balasubramanian&title=Visualization+and+selective+chemical+targeting+of+RNA+G-quadruplex+structures+in+the+cytoplasm+of+human+cells&doi=10.1038%2Fnchem.1805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span></div><div class="casAuthors">Biffi, Giulia; Di Antonio, Marco; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-80</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following extensive evidence for the formation of four-stranded DNA G-quadruplex structures in vitro, DNA G-quadruplexes have been obsd. within human cells.  Although chem. distinct, RNA can also fold in vitro into G-quadruplex structures that are highly stable because of the 2'-hydroxyl group.  However, RNA G-quadruplexes have not yet been reported in cells.  Here, we demonstrate the visualization of RNA G-quadruplex structures within the cytoplasm of human cells using a G-quadruplex structure-specific antibody.  We also demonstrate that small mols. that bind to G-quadruplexes in vitro can trap endogenous RNA G-quadruplexes when applied to cells.  Furthermore, a small mol. that exhibits a preference for RNA G-quadruplexes rather than DNA G-quadruplexes in biophys. expts. also shows the same selectivity within a cellular context.  Our findings provide substantive evidence for RNA G-quadruplex formation in the human transcriptome, and corroborate the selectivity and application of stabilizing ligands that target G-quadruplexes within a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmciXxoAihK7Vg90H21EOLACvtfcHk0lhDaG5ONXI_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N&md5=cc944f978b038bf42176a861a80b61d0</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1805%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DVisualization%2520and%2520selective%2520chemical%2520targeting%2520of%2520RNA%2520G-quadruplex%2520structures%2520in%2520the%2520cytoplasm%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D75%26epage%3D80%26doi%3D10.1038%2Fnchem.1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Guo, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartel, D. P.</span><span> </span><span class="NLM_article-title">RNA G-quadruplexes are globally unfolded in eukaryotiv cells and depleted in bacteria</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">353</span><span class="NLM_x">, </span> <span class="NLM_fpage">6306</span><span class="refDoi"> DOI: 10.1126/science.aaf5371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1126%2Fscience.aaf5371" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=6306&author=J.+U.+Guoauthor=D.+P.+Bartel&title=RNA+G-quadruplexes+are+globally+unfolded+in+eukaryotiv+cells+and+depleted+in+bacteria&doi=10.1126%2Fscience.aaf5371"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf5371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf5371%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%2BU.%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DRNA%2520G-quadruplexes%2520are%2520globally%2520unfolded%2520in%2520eukaryotiv%2520cells%2520and%2520depleted%2520in%2520bacteria%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D6306%26doi%3D10.1126%2Fscience.aaf5371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Studier, F. W.</span><span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span> <span class="citation_source-journal">Protein Expression Purif.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">207</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span><span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0liz5VgY1ki-Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i32"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00622">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71386"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00622">10.1021/acs.jmedchem.7b00622</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Scheme illustrating how RG4 can regulate translation initiation, thermal difference spectra of RG4s, <i>KRAS</i> 5′-UTR sequences in mammals, fluorescence titrations, UV and NMR spectra and HPLC-chromatograms, confocal microscopy, colony forming assay, and DNA and RNA sequences used in the study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_001.pdf">jm7b00622_si_001.pdf (926.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00622/suppl_file/jm7b00622_si_002.csv">jm7b00622_si_002.csv (0.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00622&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00622%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00622" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bb9198dc3d30","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
