
<html lang="en"     class="pb-page"  data-request-id="2ab33285-d48f-41ba-833d-ecfece69d58a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01154;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-3;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors" /></meta><meta name="dc.Creator" content="Xi  Jiang" /></meta><meta name="dc.Creator" content="Ashutosh  Kumar" /></meta><meta name="dc.Creator" content="Yasutaka  Motomura" /></meta><meta name="dc.Creator" content="Tian  Liu" /></meta><meta name="dc.Creator" content="Yong  Zhou" /></meta><meta name="dc.Creator" content="Kazuyo  Moro" /></meta><meta name="dc.Creator" content="Kam Y. J.  Zhang" /></meta><meta name="dc.Creator" content="Qing  Yang" /></meta><meta name="dc.Description" content="Chitinases not only play vital roles in the human innate immune system but are also essential for the development of pathogenic fungi and pests. Chitinase inhibitors are efficient tools to investig..." /></meta><meta name="Description" content="Chitinases not only play vital roles in the human innate immune system but are also essential for the development of pathogenic fungi and pests. Chitinase inhibitors are efficient tools to investig..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 13, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01154" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01154" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01154" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01154" /></link>
        
    
    

<title>A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01154" /></meta><meta property="og:title" content="A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0001.jpeg" /></meta><meta property="og:description" content="Chitinases not only play vital roles in the human innate immune system but are also essential for the development of pathogenic fungi and pests. Chitinase inhibitors are efficient tools to investigate the elusive role of human chitinases and to control pathogens and pests. Via hierarchical virtual screening, we have discovered a series of chitinase inhibitors with a novel scaffold that have high inhibitory activities and selectivities against human and insect chitinases. The most potent human chitotriosidase inhibitor, compound 40, exhibited a Ki of 49 nM, and the most potent inhibitor of the insect pest chitinase OfChi-h, compound 53, exhibited a Ki of 9 nM. The binding of these two most potent inhibitors was confirmed by X-ray crystallography. In a murine model of bleomycin-induced pulmonary fibrosis, compound 40 was found to suppress the chitotriosidase activity by 60%, leading to a significant increase in inflammatory cells and suggesting that chitotriosidase played a protective role." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01154"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01154">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01154&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01154&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01154&amp;href=/doi/10.1021/acs.jmedchem.9b01154" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 987-1001</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00993" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01172" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xi Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xi Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Bioengineering, Dalian University of Technology, 2 Linggong Road, Dalian 116024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xi++Jiang">Xi Jiang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ashutosh Kumar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ashutosh Kumar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Structural Bioinformatics, Center for Biosystems Dynamics Research, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ashutosh++Kumar">Ashutosh Kumar</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yasutaka Motomura</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yasutaka Motomura</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Innate Immune Systems, Center for Integrative Medical Sciences, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</div><div class="loa-info-affiliations-info">Laboratory for Innate Immune Systems, Department of Microbiology and Immunology, Graduate School of Medicine, Osaka University, 2-2 Yamadaoka, Suita-shi, Osaka 565-0871, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yasutaka++Motomura">Yasutaka Motomura</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Tian Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tian Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Bioengineering, Dalian University of Technology, 2 Linggong Road, Dalian 116024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tian++Liu">Tian Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9768-5496" title="Orcid link">http://orcid.org/0000-0001-9768-5496</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yong Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Software, Dalian University of Technology, 2 Linggong Road, Dalian 116024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Zhou">Yong Zhou</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kazuyo Moro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kazuyo Moro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Innate Immune Systems, Center for Integrative Medical Sciences, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</div><div class="loa-info-affiliations-info">Laboratory for Innate Immune Systems, Department of Microbiology and Immunology, Graduate School of Medicine, Osaka University, 2-2 Yamadaoka, Suita-shi, Osaka 565-0871, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kazuyo++Moro">Kazuyo Moro</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kam Y. J. Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kam Y. J. Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Structural Bioinformatics, Center for Biosystems Dynamics Research, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#7219131f081a131c1532001b19171c5c1802"><span class="__cf_email__" data-cfemail="7318121e091b121d1433011a18161d5d1903">[emailÂ protected]</span></a> (K.Y.J.Z.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kam+Y.+J.++Zhang">Kam Y. J. Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9282-8045" title="Orcid link">http://orcid.org/0000-0002-9282-8045</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Qing Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qing Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection and Shenzhen Agricultural Genome Research Institute, Chinese Academy of Agricultural Sciences, 2 West Yuanmingyuan Road, Beijing 100193, China</div><div class="loa-info-affiliations-info">School of Bioengineering, Dalian University of Technology, 2 Linggong Road, Dalian 116024, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#8dfce4e3eaf4ece3eacde9e1f8f9a3e8e9f8a3eee3"><span class="__cf_email__" data-cfemail="7c0d15121b051d121b3c1810090852191809521f12">[emailÂ protected]</span></a> (Q.Y.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qing++Yang">Qing Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8623-7532" title="Orcid link">http://orcid.org/0000-0001-8623-7532</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01154&amp;href=/doi/10.1021%2Facs.jmedchem.9b01154" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 987â1001</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 13, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 July 2019</li><li><span class="item_label"><b>Published</b> online</span>13 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01154</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D987%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXi%2BJiang%252C%2BAshutosh%2BKumar%252C%2BYasutaka%2BMotomura%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D3%26contentID%3Dacs.jmedchem.9b01154%26title%3DA%2BSeries%2Bof%2BCompounds%2BBearing%2Ba%2BDipyrido-Pyrimidine%2BScaffold%2BActing%2Bas%2BNovel%2BHuman%2Band%2BInsect%2BPest%2BChitinase%2BInhibitors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1001%26publicationDate%3DFebruary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01154"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1174</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01154" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xi&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Ashutosh&quot;,&quot;last_name&quot;:&quot;Kumar&quot;},{&quot;first_name&quot;:&quot;Yasutaka&quot;,&quot;last_name&quot;:&quot;Motomura&quot;},{&quot;first_name&quot;:&quot;Tian&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Kazuyo&quot;,&quot;last_name&quot;:&quot;Moro&quot;},{&quot;first_name&quot;:&quot;Kam&quot;,&quot;last_name&quot;:&quot;Y. J. Zhang&quot;},{&quot;first_name&quot;:&quot;Qing&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;987-1001&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01154&quot;},&quot;abstract&quot;:&quot;Chitinases not only play vital roles in the human innate immune system but are also essential for the development of pathogenic fungi and pests. Chitinase inhibitors are efficient tools to investigate the elusive role of human chitinases and to control pathogens and pests. Via hierarchical virtual screening, we have discovered a series of chitinase inhibitors with a novel scaffold that have high inhibitory activities and selectivities against human and insect chitinases. The most potent human chitotriosidase inhibitor, compound 40, exhibited a Ki of 49 nM, and the most potent inhibitor of the insect pest chitinase OfChi-h, compound 53, exhibited a Ki of 9 nM. The binding of these two most potent inhibitors was confirmed by X-ray crystallography. In a murine model of bleomycin-induced pulmonary fibrosis, compound 40 was found to suppress the chitotriosidase activity by 60%, leading to a significant increase in inflammatory cells and suggesting that chitotriosidase played a protective role.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01154" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01154&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01154" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01154&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01154" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01154&amp;href=/doi/10.1021/acs.jmedchem.9b01154" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01154" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01154" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01154%26sid%3Dliteratum%253Aachs%26pmid%3D31928006%26genre%3Darticle%26aulast%3DJiang%26date%3D2020%26atitle%3DA%2BSeries%2Bof%2BCompounds%2BBearing%2Ba%2BDipyrido-Pyrimidine%2BScaffold%2BActing%2Bas%2BNovel%2BHuman%2Band%2BInsect%2BPest%2BChitinase%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D3%26spage%3D987%26epage%3D1001%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (7)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/jmcmar.2020.63.issue-3/20200213/jmcmar.2020.63.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Chitinases not only play vital roles in the human innate immune system but are also essential for the development of pathogenic fungi and pests. Chitinase inhibitors are efficient tools to investigate the elusive role of human chitinases and to control pathogens and pests. Via hierarchical virtual screening, we have discovered a series of chitinase inhibitors with a novel scaffold that have high inhibitory activities and selectivities against human and insect chitinases. The most potent human chitotriosidase inhibitor, compound <b>40</b>, exhibited a <i>K</i><sub>i</sub> of 49 nM, and the most potent inhibitor of the insect pest chitinase <i>Of</i>Chi-h, compound <b>53</b>, exhibited a <i>K</i><sub>i</sub> of 9 nM. The binding of these two most potent inhibitors was confirmed by X-ray crystallography. In a murine model of bleomycin-induced pulmonary fibrosis, compound <b>40</b> was found to suppress the chitotriosidase activity by 60%, leading to a significant increase in inflammatory cells and suggesting that chitotriosidase played a protective role.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The glycosyl hydrolase family 18 chitinases catalyze the degradation of chitin, a homopolymeric form of Î²-(1,4)-linked <i>N</i>-acetyl-D-glucosamine (GlcNAc). As chitin constitutes the cell wall of fungi, the eggshells of nematodes, and the exoskeleton of arthropods, chitinases play essential roles in the growth and development of chitin-containing organisms as well as immunologically defensive roles in non-chitin-containing organisms such as humans and plants.</div><div class="NLM_p">Humans have two chitinases: chitotriosidase (Chit1) and acidic mammalian chitinase (AMCase). A number of studies indicate that elevation of Chit1 in plasma or tissues is associated with several diseases. A 10- to 1000-fold elevation of Chit1 has been reported in the blood of patients with lysosomal storage diseases, such as Gaucher disease,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> NiemannâPick syndrome, and Fabry disease,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> reflecting a particular activation state of macrophages.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Elevated levels of Chit1 are also observed in a variety of other diseases, including asthma and airway hyper-responsiveness,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> infections (fungal and bacterial infection, malaria), chronic inflammation (atherosclerosis,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> sarcoidosis<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>), liver diseases (nonalcoholic fatty liver disease steatohepatitis<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a>), neurodegenerative diseases (Alzheimerâs disease, ischemic cerebrovascular dementia), and interstitial lung diseases in systemic sclerosis.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> On the other hand, AMCase, which is distinguished from other chitinases by several unique features including a low optimum pH and the preference for the Î²-anomer of the substrate,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> is highly expressed in the lung tissue of asthmatic patients and in animal models of asthma.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9â12)</a> Moreover, it has been shown that AMCase is induced by a Th2-specific, IL-13-mediated pathway in a chitin or ovalbumin (OVA) allergy model.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Furthermore, it has been observed that treatment of allergen-challenged mice with AMCase antisera or AMCase inhibitors alleviated primary features of allergic inflammation.<a onclick="showRef(event, 'ref10 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref10 ref13 ref14 ref15">(10,13â15)</a> Although both human chitinases were the subject of a plethora of investigations in the last decade, the exact role of either Chit1 or AMCase has not yet been fully defined.</div><div class="NLM_p">Due to diverse functions of chitinases, chitinase inhibitors have a number of potential applications, including as chemical probes, immunomodulators, fungicides, and insecticides. Several classes of chitinase inhibitors have been reported (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) including the pseudotrisaccharide allosamidin (<b>1</b>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> the cyclic dipeptide CI-4 (<b>2</b>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> the cyclic pentapeptides argifin (<b>3</b>) and argadin (<b>4</b>),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> fully deacetylated chitooligosaccharides (<b>5</b> and <b>6</b>),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> acetazolamide (<b>7</b>),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> bisdionin C (<b>8</b>) and F (<b>9</b>),<a onclick="showRef(event, 'ref13 ref21'); return false;" href="javascript:void(0);" class="ref ref13 ref21">(13,21)</a> methylxanthines,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> theophylline (<b>10</b>), caffeine (<b>11</b>), pentoxifylline (<b>12</b>),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> psammaplins (<b>13</b>),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and styloguanidines (<b>14</b>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Many of these inhibitors are natural products. Some of the highly potent ones such as allosamidin, argifin, and argadin were employed to study the roles of Chit1 and AMCase in the development of various diseases.<a onclick="showRef(event, 'ref10 ref14 ref26'); return false;" href="javascript:void(0);" class="ref ref10 ref14 ref26">(10,14,26)</a> However, their poor synthetic accessibility and low abundance in nature have so far hampered the widespread use of these inhibitors, whereas their structural complexity makes them unsuitable as starting points for drug design. Initial efforts to develop drug-like inhibitors of Chit1 and AMCase were based on the 1,3-dimethylxanthine scaffold and only resulted in weak inhibitors.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Synthesis of dicaffeine scaffold derivatives (named bisdionins, <b>8</b> and <b>9</b>) with variable linker lengths produced moderately potent Chit1 and AMCase inhibitors.<a onclick="showRef(event, 'ref13 ref21'); return false;" href="javascript:void(0);" class="ref ref13 ref21">(13,21)</a> Interestingly, one compound, bisdionin F (<b>9</b>) was found to be 20-fold selective for AMCase.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Another inhibitor discovery effort involving the combination of high-throughput screening, fragment-based drug design, and in silico screening led to the identification of an aminotriazole-based compound, Wyeth-1 (<b>15</b>), which was a moderately potent and non-selective Chit1 and AMCase inhibitor.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Using structure guided design, Wyeth-1 was later optimized into a highly potent AMCase inhibitor (<b>16</b>) with about 20-fold selectivity against Chit1. However, this compound was not pursued further due to safety concerns and dopamine receptor off-target activity.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of a few chitinase inhibitors reported in the literature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Small-molecule drug-like inhibitors of human chitinases played a significant part in understanding their biological role and implications of inhibiting these proteins. However, investigations on the role of Chit1 and/or AMCase in the immune response are much more challenging. Apart from the design of suitable in vivo experiments, the specific inhibition of these proteins is required. All existing Chit1 or AMCase inhibitors are either not highly potent or lack selectivity toward these proteins and thus are not suitable to discriminate functional differences between Chit1 and AMCase. While the two most selective AMCase inhibitors are only 20-fold selective,<a onclick="showRef(event, 'ref13 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref15">(13,15)</a> no human Chit1 selective inhibitor has been reported. Recently, a highly potent inhibitor (<b>17</b>) of mouse Chit1 has also been reported with about 143-fold selectivity against mouse AMCase, but its inhibitory effect on human Chit1 is not known.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In light of these circumstances, the development of novel, highly potent, highly selective, and in vivo active human Chit1 inhibitors is required.</div><div class="NLM_p">Besides studies on human chitinases, many researchers pay great attention to chitinases originating from pathogenic fungi and agricultural pests. This is because the chitinases essential for the development of fungi and pests are potential targets for developing drugs and agrochemicals. Chi-h, for example, is a lepidoptera-exclusive insect chitinase and is involved in cuticular chitin degradation during pest molting. Suppression of Chi-h activity results in severe defects in pupation and then death.</div><div class="NLM_p last">Here, utilizing hierarchical virtual screening, we have identified a series of chitinase inhibitors with a common dipyrido-pyrimidine scaffold. A novel human Chit1-selective inhibitor with a <i>K<sub>i</sub></i> value of 49 nM and an insect pest Chi-h selective inhibitor with a <i>K<sub>i</sub></i> value of 9 nM were obtained. Cocrystal structures of both of these compounds with human Chit1 and chitinases from other species confirmed the binding and explained their potent activity and high selectivity. We further demonstrated the in vivo efficacy of most potent human Chit1-selective inhibitor using a murine model of bleomycin-induced pulmonary fibrosis, which suggested a protective role of Chit1 in inflammation under experimental conditions. Additionally, this study also provided lead compounds for the development of agrochemicals against destructive insect pests.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50516" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50516" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Inhibitory Activities and Selectivities of the Compounds Obtained by SBVS</h3><div class="NLM_p">Virtual screening to discover chitinase inhibitors led to the identification of nine compounds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <b>18</b>â<b>26</b> in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Table S1</a>). The evaluation of inhibitory activities of these nine compounds against <i>Hs</i>Chit1 resulted in the identification of the compound 2-amino-1-(furan-2-ylmethyl)-5-oxo-3-(((tetrahydrofuran-2-yl)methyl)carbamoyl)-5<i>H</i>-dipyrido[1,2-<i>a</i>:2â²,3â²-<i>d</i>]pyrimidin-1-ium (<b>24</b>) that demonstrated almost complete inhibition of <i>Hs</i>Chit1 enzymatic activity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Table S1</a>). The three-dimensional overlay of the query ligand (GlcNAc)<sub>3</sub> with <b>24</b> revealed a reasonable shape overlap even though the chemical structures of these two ligands are significantly different (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). The concentration-dependent assay revealed a <i>K<sub>i</sub></i> value of 6.8 Î¼M (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Further assessment against insect and bacterial chitinases revealed similar potencies against <i>Of</i>Chi-h and <i>Sm</i>ChiB (<i>K<sub>i</sub></i> values of 8 and 6 Î¼M, respectively).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Hierarchical virtual screening strategy used to identify dipyrido-pyrimidine derivatives as chitinase inhibitors. A combination of shape similarity calculations, electrostatic potential similarity calculations, and molecular docking was employed to prioritize compounds for the enzymatic assay. Three rounds of virtual screening were carried out. (B) Residual activity of first round virtual screening compounds against <i>Hs</i>Chit1, <i>Of</i>Chi-h, and <i>Sm</i>ChiB. (C) A 3D shape of the query (GlcNAc)<sub>3</sub>. (D) A 3D shape overlay of the query (GlcNAc)<sub>3</sub> with a hit compound <b>24</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>K<sub>i</sub></i> Values of <b>40</b> and <b>53</b> for Chitinases from Different Organisms</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><i>K<sub>i</sub></i> (Î¼M)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">chitinase</th><th class="colsep0 rowsep0" align="center" char="."><b>40</b></th><th class="colsep0 rowsep0" align="center" char="."><b>53</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>Hs</i>Chit1</td><td class="colsep0 rowsep0" align="char" char=".">0.049</td><td class="colsep0 rowsep0" align="char" char=".">1.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">hAMCase</td><td class="colsep0 rowsep0" align="char" char=".">3.96</td><td class="colsep0 rowsep0" align="char" char=".">9.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mChit1</td><td class="colsep0 rowsep0" align="char" char=".">0.035</td><td class="colsep0 rowsep0" align="char" char=".">1.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mAMCase</td><td class="colsep0 rowsep0" align="char" char=".">0.51</td><td class="colsep0 rowsep0" align="char" char=".">1.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>Of</i>Chi-h</td><td class="colsep0 rowsep0" align="char" char=".">0.39</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>Sm</i>ChiB</td><td class="colsep0 rowsep0" align="char" char=".">0.41</td><td class="colsep0 rowsep0" align="char" char=".">0.058</td></tr></tbody></table></div></div><div class="NLM_p">The dipyrido-pyrimidine-based scaffold represents a novel compound class with moderate activity, and to the best of our knowledge, this compound has not been previously described to possess activity against any chitinase. A similarity search against ChEMBL<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and Similarity Ensemble Approach (SEA) database<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> using <b>24</b> as a query revealed no known biological activity for this class of compounds. Moreover, scanning of this compound for PAINS<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and problematic compounds according to Eli Lilly MedChem Rules<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> exposed no major structural alert. The only warning given was the presence of a quaternary aryl nitrogen (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Supporting Information</a>). Although compounds with a quaternary aryl nitrogen may have some liabilities such as toxicities, we decided to pursue this scaffold for several reasons: (1) It is not predicted to interfere with assays, that is, passed the PAINS filter. (2) Several approved and marketed drugs contain a quaternary aryl nitrogen, for example, Ceftazidime (DrugBank code DB00438), pyridostigmine (DrugBank code DB00545), and berberine (DrugBank code DB04115). (3) The quaternary aryl nitrogen is present in a variety of natural products from marine sponges with antiviral and antitumor activities and acetylcholine esterase inhibitory activity.<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33â35)</a> (4) We wanted to develop this scaffold for a variety of uses such as biological probes and pesticides. Although <b>24</b> represents a novel and interesting chemical class with chitinase inhibitory activity, it is not potent enough to be used as a chemical probe or lead compound. However, <b>24</b> could be a suitable starting point for the development of highly potent, in vivo active inhibitor of <i>Hs</i>Chit1, insect or bacterial chitinases. Therefore, we decided to further improve the potency of the dipyrido-pyrimidine-based scaffold by identifying structural analogs. Seventy-five structural analogs of <b>24</b> were identified in two rounds of virtual screening (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <b>27</b>â<b>101</b> in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Tables S2 and S3</a>). The chemical structures of some of the relevant compounds are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Chemical structures of a few relevant compounds with the dipyrido-pyrimidine scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The inhibitory potency of these structural analogs was tested against <i>Hs</i>Chit1, hAMCase, <i>Of</i>Chi-h, and <i>Sm</i>ChiB. A majority of the tested analogs were found to be more potent than the parent compound <b>24</b> for all the tested enzymes (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Tables S2 and S3</a>). For <i>Hs</i>Chit1, <b>40</b> was found to be the most potent. It showed a <i>K<sub>i</sub></i> value of 49 nM, which was about 100-fold better than the parent compound <b>24</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The specificity of <b>40</b> against another human chitinase, hAMCase, was also evaluated. Compound <b>40</b> inhibited hAMCase with a <i>K<sub>i</sub></i> value of 3.9 Î¼M and was found to be 80-fold more potent against <i>Hs</i>Chit1 than hAMCase (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Evaluation of <i>Of</i>Chi-h and <i>Sm</i>ChiB inhibitory activities revealed that <b>40</b> was also a broad-spectrum chitinase inhibitor with <i>K<sub>i</sub></i> values of 0.39 and 0.41 Î¼M against <i>Of</i>Chi-h and <i>Sm</i>ChiB, respectively. Apart from <b>40</b>, compound <b>53</b> was another interesting compound displaying potent inhibition of <i>Of</i>Chi-h and <i>Sm</i>ChiB enzymatic activities with <i>K<sub>i</sub></i> values of 9 and 58 nM, respectively. Unexpectedly, <b>53</b> showed weak activity against <i>Hs</i>Chit1 with <i>K<sub>i</sub></i> value of 1.12 Î¼M (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The weak inhibition of <b>53</b> against <i>Hs</i>Chit1 suggested that the inhibitor might bind to <i>Hs</i>Chit1 by a different mechanism. The inhibitory activities of <b>40</b> and <b>53</b> were also measured against mouse Chit1 and AMCase. As expected, inhibitory activities of both <b>40</b> and <b>53</b> against mChit1 were similar to that of <i>Hs</i>Chit1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Interestingly, <b>40</b> was found to be only 15-fold more selective for mChit1 over mAMCase (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">We next analyzed the structure activity relationships (SAR) to understand the mechanism of inhibition and functional groups responsible for higher potency and selectivity among the tested chitinases. The substituents were available at four positions of the dipyrido-pyrimidine-based scaffold (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S1</a>). Among these 75 compounds, R2 and R3 positions were either unsubstituted or substituted with a methyl group. The comparison of a pair of compounds that differ only by the methyl group at the R2 or R3 position revealed no major differences in activity for a majority of the compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S2</a>). The major structural differences among these 75 analogs of <b>24</b> were in substituents at R1 and R4 positions. The effect of each of the substituents at R1 and R4 positions on chitinase enzymatic activity was analyzed by plotting the inhibition rates for compounds having a particular functional group (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figures S3 and S4</a>). As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S3</a>, a majority of substituents have a carboxamide group at the R1 position, and the compounds without it (e.g., <b>36</b>, <b>41</b>, <b>44</b>, <b>46</b>, and <b>51</b>) were found to be inactive for all the tested chitinases (see R1-11, R1-18, R1-19, and R1-24 in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S3</a>). The carboxamide group may be involved in hydrogen bonding contacts with conserved active site residues. Further, substitutions of carboxamide linked with cycloalkyl groups, charged piperizine, and piperidine also led to reduced chitinase activity in a majority of the compounds (e.g., <b>27</b>, <b>30</b>, <b>42</b>, <b>45</b>, and <b>48</b>) (see R1-14, R1-15, and R1-25 in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S3</a>). In contrast to the substitution of non-aromatic groups, substitution of aromatic carboxamide moieties such as 4-methylbenzyl carboxamide (R1-1), benzyl carboxamide (R1-2), 3-pyridinylmethyl carboxamide (R1-3), 4-methoxybenzyl carboxamide (R1-4), 4-fluorobenzyl carboxamide (R1-5), and phenylethyl carboxamide (R1-7) at the R1 position resulted in higher inhibition in a majority of compounds. Among these, compounds with phenylethyl carboxamide and 3-pyridinylmethyl carboxamide at the R1 position, such as <b>40</b>, <b>58</b>, <b>91</b>, <b>94</b>, <b>96</b>, <b>52</b>, <b>53</b>, <b>59</b>, <b>67</b>, and <b>99</b>, were found to be the most potent for all the tested chitinases (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S3, Tables S2 and S3</a>). In fact, a majority of the compounds with these two functional groups were active against <i>Hs</i>Chit1 and to some extent against other tested chitinases irrespective of substituents at other positions. It was interesting to see that compounds with 2-chlorobenzyl carboxamide (R1-9) at the R1 position (e.g., <b>69</b>, <b>77</b>, and <b>89</b>) were either inactive or moderately active even though an aromatic ring at the R1 position was present. At the R1 position of the dipyrido-pyrimidine scaffold, SAR analysis suggests the requirement of functional groups that can make hydrogen bonding and stacking interactions. However, substitutions on the aromatic ring are not favorable especially at the <i>ortho</i>-position of the aromatic ring, which might be due to the limited size of that subpocket.</div><div class="NLM_p last">SAR analysis of substituents at the R4 position revealed that all compounds with 3-pyridinylmethyl substitution (e.g., <b>36</b>, <b>43</b>, <b>44</b>, and <b>45</b>) did not possess chitinase inhibitory activity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S4, Table S2</a>) and were found to be inactive. Moreover, the compounds with alkyl substitutions such as butene, methoxyethyl, and methoxypropyl substitutions at the R4 position (e.g., <b>27</b>, <b>48</b>, <b>51</b>, <b>56</b>, <b>75</b>, <b>88</b>, and <b>93</b>) were also either less active or inactive in most cases (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S4, Tables S2 and S3</a>). Substitutions of benzyl (R4-1) and tetrahydrofuran (R4-2) at the R4 position resulted in higher <i>Hs</i>Chit1, hAMCase, and <i>Sm</i>ChiB inhibitory activities for a majority of compounds (e.g., <b>40</b>, <b>47</b>, <b>53</b>, <b>61</b>, <b>62</b>, <b>91</b>, <b>92</b>, <b>94</b>, <b>95</b>, <b>96</b>, and <b>99</b>). Some substitutions at the R4 position were favorable for inhibition of only a specific chitinase. For example, compounds with 2-chlorobenzyl (R4-6) were only active against <i>Hs</i>Chit1 and possessed no inhibitory activity against hAMCase, <i>Of</i>Chi-h, and <i>Sm</i>ChiB (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S4, Tables S2 and S3</a>). Similarly, 4-fluorobenzyl (R4-3) substitution is not favorable for <i>Of</i>Chi-h inhibition, while it is tolerable for hAMCase and <i>Sm</i>ChiB and to some extent for <i>Hs</i>Chit1. In addition to 2-chlorobenzyl (R4-6) and 4-fluorobenzyl (R4-3), substitution of benzyl (R4-1) at the R4 position also has a negative effect on <i>Of</i>Chi-h inhibitory activity, which might be due to a smaller R4 group binding subpocket than the ones in <i>Hs</i>Chit1, hAMCase, and <i>Sm</i>ChiB.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Crystal Complexes Reveal the Inhibitory Mechanism of Compound <b>40</b> against <i>Hs</i>Chit1 and <i>Sm</i>ChiB</h3><div class="NLM_p">To further investigate the inhibitory mechanism of <b>40</b>, it was cocrystallized with <i>Hs</i>Chit1-CAD. The structure of the <i>Hs</i>Chit1/<b>40</b> complex was refined to 1.8 Ã. The 2Fo-Fc map showed a clear electron density for <b>40</b> along the substrate-binding cleft. Crystallographic analysis demonstrated that the dipyrido-pyrimidine moiety of <b>40</b> binds at the +1 and +2 subsites of <i>Hs</i>Chit1. It is stacked between the indole groups of Trp99 and Trp218, forming strong hydrophobic interactions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). As these two residues are conserved not only in human chitinases but also in the other family 18 chitinases, a broad chitinase activity of the dipyrido-pyrimidine scaffold is expected. The carboxyl group of Asp213 forms a hydrogen bond with the 3-carbamoyl nitrogen atom. As Asp213 is conserved in all the tested chitinases (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S5</a>), the interaction seems critical for the inhibitory activity. This may be the reason behind the lack of chitinase inhibitory activity in the case of compounds without carboxamide-based moieties at the R1 position. Water-mediated hydrogen bonds are formed between the guanidine group of Arg269, carboxyl group of Glu297, 2-amino nitrogen atom, and 3-carbamoyl oxygen atom as well as between the carboxyl group of Glu140 and 3-carbamoyl nitrogen atom. The phenylethyl carboxamide moiety at the R1 position of the dipyrido-pyrimidine scaffold inserts into a subpocket that is surrounded by Tyr267 and Trp358 and forms ÏâÏ interactions. As both aromatic amino acids are conserved in <i>Hs</i>Chit1, hAMCase, <i>Of</i>Chi-h, and <i>Sm</i>ChiB, hydrophobic ÏâÏ stacking interactions seem critical for activity, and compounds lacking aromatic groups were not found to be active (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Tables S2 and S3</a>). Stacking interactions with Trp358 are especially important as Trp358 stacks with GlcNAc at the â1 subsite and is responsible for accommodating it in a boat conformation, which is critical for catalysis. Moreover, a benzyl at the R4 position stacks vertically with the indole group of Trp218 forming ÏâÏ T-shaped interactions. The details of the <i>Hs</i>Chit1/<b>40</b> structural complex reveal that the strong ÏâÏ stacking interactions of the tryptophan at +1 and +2 subsites and hydrogen bonding with key residues around the active site make <b>40</b> a potent inhibitor against <i>Hs</i>Chit1.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Details of the interaction of <b>40</b> with <i>Hs</i>Chit1-CAD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR">6JJR</a>) (A) and <i>Sm</i>ChiB (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9">6JK9</a>) (B). Compound <b>40</b> is shown in light yellow, <i>Hs</i>Chit1 is shown in cyan, <i>Sm</i>ChiB is shown in white, hydrogen bonds are shown as dashed lines, and water molecules are shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To understand the reason why <b>40</b> is a modest inhibitor against hAMCase, we tried to obtain a structure of the hAMCase/<b>40</b> complex; unfortunately, both soaking and cocrystallization efforts were unsuccessful. Hence, to understand the <i>Hs</i>Chit1 selectivity of <b>40</b>, a docking-predicted binding mode of <b>40</b> with hAMCase was analyzed and compared with the <i>Hs</i>Chit1/<b>40</b> crystal structure (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S6</a>). Compound <b>40</b> was predicted to bind at the +1 and +2 subsites of hAMCase and forms various ÏâÏ interactions with the residues around the active site similar to <i>Hs</i>Chit1. However, there were some subtle differences that might account for the selectivity of <b>40</b> for <i>Hs</i>Chit1. One critical difference may be the presence of His269 in hAMCase in place of Arg269 in <i>Hs</i>Chit1 even though the amino acid residues that form the binding pocket in both <i>Hs</i>Chit1 and hAMCase are highly conserved. In the case of the <i>Hs</i>Chit1/<b>40</b> crystal structure, the guanidine group of Arg269 forms a water-mediated hydrogen bond with <b>40</b> and may be responsible for perfectly placing the phenylethyl ring for edge-to-face Ï stacking interactions with the conserved Trp358. In hAMCase, on the other hand, although this type of interaction is missing, the phenylethyl carboxamide moiety can still make hydrophobic contacts with residues Trp360, Trp31, and Tyr267. Moreover, the comparison of binding pocket volume of all available <i>Hs</i>Chit1 and hAMCase crystal structures revealed slightly a bigger binding pocket in the case of <i>Hs</i>Chit1, which may allow the rotation of functional groups for protein-ligand interactions (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S7A</a>). Further, in the predicted binding mode, a benzyl ring at the R4 position of the dipyrido-pyrimidine scaffold is also not perfectly placed for edge-to-face Ï stacking interactions with another conserved residue Trp218. The distance between Trp99 and Trp218 in hAMCase is slightly longer than that in <i>Hs</i>Chit1 and might have contributed to the lower potency of <b>40</b> against hAMCase (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S7B</a>).</div><div class="NLM_p last"><i>Sm</i>ChiB was often used as a model to explore the binding mechanism. The crystal structure of <i>Sm</i>ChiB/<b>40</b> also reveals a more or less similar binding mechanism to <i>Hs</i>Chit1 with a root-mean-square deviation of 1 Ã (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Slight structural differences between the <i>Hs</i>Chit1 and <i>Sm</i>ChiB substrate binding sites might have caused the subtle differences in the <b>40</b> binding mode. The dipyrido-pyrimidine moiety is sandwiched between Trp97 and Trp220 (Trp99 and Trp218 in <i>Hs</i>Chit1) via the hydrophobic stacking interaction. The 3-carbamoyl nitrogen forms a hydrogen bond with the carboxyl group of Asp316, while the 2-amino nitrogen engages water-mediated hydrogen bonds with the carboxyl group of Asp215. The phenylethyl carboxamide moiety at the R1 position is located in a hydrophobic pocket consisting of Tyr214, Tyr292, and Trp403 side chains. The chitin insert domain of <i>Sm</i>ChiB covers the 1-benzyl ring though no subpocket interactions have been observed. The structural details reveal that, just like HsChit1, hydrogen bonds with Asp215 and hydrophobic stacking contacts with the dipyrido-pyrimidine moiety are also crucial for the inhibition of <i>Sm</i>ChiB enzymatic activity.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Crystal Complexes Reveal the Inhibitory Mechanism of Compound <b>53</b> against <i>Hs</i>Chit1, <i>Of</i>Chi-h, and <i>Sm</i>ChiB</h3><div class="NLM_p">To investigate the binding mode of <b>53</b>, the structures of <i>Hs</i>Chit1, <i>Of</i>Chi-h, and <i>Sm</i>ChiB in complex with <b>53</b> were refined to 1.6, 2.2, and 2.5 Ã, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The active site reveals clear electron density for <b>53</b>. Unexpectedly, the overall binding mode of <b>53</b> in <i>Of</i>Chi-h is different from those of <i>Hs</i>Chit1 and <i>Sm</i>ChiB. One molecule of <b>53</b> occupies +1 and +2 subsites of <i>Of</i>Chi-h, while electron density for two molecules of <b>53</b> was observed in <i>Hs</i>Chit1 and <i>Sm</i>ChiB +1 and +2 subsites. The <i>Of</i>Chi-h/<b>53</b> binding mode is more or less similar to <i>Hs</i>Chit1/<b>40</b> and <i>Sm</i>ChiB/<b>40</b>, except the position of the 3-pyridinylmethyl carboxamide moiety, which extends toward the â1 subsite in close proximity to catalytic residues Asp306 and Glu308. The 3-pyridinyl nitrogen forms a hydrogen bond with the catalytic residue Glu308, and this particular interaction may be responsible for higher activity of <b>53</b> against <i>Of</i>Chi-h. Other interactions are similar to <i>Hs</i>Chit1/<b>40</b> and <i>Sm</i>ChiB/<b>40</b> binding modes. The dipyrido-pyrimidine moiety is found to be sandwiched between Trp268 and Trp389 to form hydrophobic stacking interactions (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The 3-carbamoyl nitrogen atom forms a hydrogen bond with the side chain of Asp384. Moreover, the 1-tetrahydrofuran oxygen can form two hydrogen bonds with the side chain of Arg439 to stabilize this five-membered ring. All these stacking interactions and hydrogen bonds make <b>53</b> a potent inhibitor of <i>Of</i>Chi-h.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Details of the interaction of <b>53</b> with <i>Of</i>Chi-h (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN">6JMN</a>) (A), <i>Hs</i>Chit1-CAD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6">6JK6</a>) (B), and <i>Sm</i>ChiB (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF">6JKF</a>) (C). Compound <b>53</b> is shown in pale pink, <i>Of</i>Chi-h is in green, <i>Hs</i>Chit1 is shown in cyan, <i>Sm</i>ChiB is shown in white, hydrogen bonds are shown as dashed lines, and water molecules are shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Contrary to the binding mode of <b>53</b> in <i>Of</i>Chi-h, two <b>53</b> molecules occupying <i>Hs</i>Chit1 +1 and +2 subsites were observed (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The dipyrido-pyrimidine moieties were found sandwiched between Trp99 and Trp218. One molecule, the lower <b>53</b>, inserts deeply into the substrate binding cleft closer to Trp99. Due to the deeper placement of the dipyrido-pyrimidine ring, a critical hydrogen bond between 3-pyridinyl nitrogen and catalytic residue Glu140 was not observed as the pyridine ring flips in another direction. Interestingly, the loss of this hydrogen bond is compensated by the hydrogen bonding interaction with the hydroxyl of Tyr212 and ÏâÏ stacking interactions with Trp358. The catalytic residue Glu140 instead interacts with the 3-carbamoyl moiety. Moreover, a water-molecule-mediated hydrogen bonding interaction between the 3-carbamoyl moiety and the backbone nitrogen of Trp99 was also observed. Additionally, the 5-carbonyl oxygen atom forms a hydrogen bond with the hydroxyl of Tyr141. The other molecule, upper <b>53</b>, only forms stacking interactions with Trp218 and one hydrogen bond between 3-pyridine and the backbone carbonyl oxygen atom of Trp218. The hydrophobic stacking interactions of the four-layer sandwich structure stabilize the upper <b>53</b> molecule. The distance between the indole group of Trp99 and the dipyrido-pyrimidine ring of the lower <b>53</b> is 3.5 Ã, the same distance that is between the indole group of Trp218 and the dipyrido-pyrimidine ring of the upper <b>53</b> and between the two <b>53</b> dipyrido-pyrimidine rings. The +1 subsite Trp99 migrates downward 1.5 Ã to accommodate these two molecules. The unfavorable Gly98-Thr103 main chain conformational change is consistent with the decreased inhibitor activity.</div><div class="NLM_p">The asymmetric unit comprised two molecules of <i>Sm</i>ChiB/<b>53</b>, of which the inhibitor binding mode of each protein molecule is different (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Two <b>53</b> were bound in +1 and +2 subsites of one protein molecule; however, the other protein molecule accommodates only one <b>53</b>. In the case where two <b>53</b> occupy +1 and +2 subsites, the lower <b>53</b> 3-pyridinyl nitrogen atom forms a strong hydrogen bond with the oxygen atom of the catalytic residue Glu144 side chain. The tetrahydrofuran group at the R4 position is located near the chitin insert domain, and its oxygen atom forms a hydrogen bond with the side chain of Asp215. Additionally, the lower inhibitor engages two water-mediated hydrogen bonds, and the 3-carbamoyl nitrogen and oxygen form hydrogen bonds with the side chain of Asp316 and the backbone of Trp97, respectively. The upper inhibitor is only stabilized by hydrophobic stacking interactions between the lower <b>53</b> and Trp220. The two-molecule insertion causes a large conformational change where the distance between Trp97 and Trp220 increases from 7.5 to 10 Ã. Unlike the two-molecule inhibitor binding mode, binding of the one-molecule inhibitor almost has no influence on the distance between +1 and +2 subsite residues. This binding mechanism is similar to that of <i>Of</i>Chi-h/<b>53</b> where 3-carbamoyl nitrogen forms a hydrogen bond with the side chain of Asp215 and 3-pyridinyl nitrogen forms a hydrogen bond with Glu144. The 2-amino and 3-carbonyl also form water-mediated hydrogen bonds with carboxyl of Asp316 and backbone amide nitrogen of Trp97, respectively. This suggests that <b>53</b> inhibits <i>Sm</i>ChiB by two mechanisms. The inhibitor inserts into +1 and +2 subsites by two-molecule mode at high concentration, while the inhibitor inserts by one-molecule mode at low concentration. The details of the crystal structure complex are consistent with increased inhibitory activity of <b>53</b>.</div><div class="NLM_p">Although the binding of two molecules of <b>53</b> with <i>Hs</i>Chit1 and <i>Sm</i>ChiB at +1 and +2 subsites is peculiar, it has been previously observed in several chitinase crystal structures.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> To further study the binding mode of <b>53</b>, isothermal titration calorimetry (ITC) experiments were performed to determine the binding stoichiometry of <b>53</b> with <i>Hs</i>Chit1, <i>Of</i>Chi-h, and <i>Sm</i>ChiB in solution. The <i>K</i><sub>d</sub> values for <b>53</b> derived from the ITC experiment are in agreement with <i>K<sub>i</sub></i> values from the chitinase inhibition assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Figure S8</a>). Binding of <b>53</b> to <i>Of</i>Chi-h comprises mainly of hydrogen bond and hydrophobic contacts as indicated by negative or favorable Î<i>H</i> and Î<i>S</i> values. On the other hand, binding of <b>53</b> to both <i>Hs</i>Chit1 and <i>Sm</i>ChiB involves conformational changes as indicated by positive or unfavorable entropy values. Moreover, the thermodynamic data (Î<i>H</i> and Î<i>S</i>) expose similar binding mode of <b>53</b> to <i>Hs</i>Chit1 and <i>Sm</i>ChiB. This observation is consistent with our <i>Hs</i>Chit1/<b>53</b> and <i>Sm</i>ChiB/<b>53</b> crystal structures where a conformational change facilitates the binding of the second molecule. However, further analysis of ITC data revealed a stoichiometry of â¼1 for the number of <b>53</b> bound per enzyme molecule. The ITC results indicated the binding of only one <b>53</b> molecule to both <i>Hs</i>Chit1 and <i>Sm</i>ChiB. The differences in ITC and <i>Hs</i>Chit1/<b>53</b> and <i>Sm</i>ChiB/<b>53</b> crystal structures might be due to different experimental conditions as much higher concentrations of <b>53</b> were used to grow crystals.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Thermodynamic Values from ITC Experiment</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">chitinases</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">thermodynamic parameter</th><th class="colsep0 rowsep0" align="center" char="."><i>Hs</i>Chit1</th><th class="colsep0 rowsep0" align="center" char="."><i>Sm</i>ChiB</th><th class="colsep0 rowsep0" align="center" char="."><i>Of</i>Chi-h</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>N</i></td><td class="colsep0 rowsep0" align="char" char=".">1.04</td><td class="colsep0 rowsep0" align="char" char=".">0.96</td><td class="colsep0 rowsep0" align="char" char=".">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>K</i><sub>a</sub> (M)</td><td class="colsep0 rowsep0" align="char" char=".">8.95Â ÃÂ 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char=".">5.11Â ÃÂ 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char=".">1.17Â ÃÂ 10<sup>7</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> (Î¼M)</td><td class="colsep0 rowsep0" align="char" char=".">1.12</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td><td class="colsep0 rowsep0" align="char" char=".">0.085</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>H</i> (cal/mol)</td><td class="colsep0 rowsep0" align="char" char=".">â5925</td><td class="colsep0 rowsep0" align="char" char=".">â5903</td><td class="colsep0 rowsep0" align="char" char=".">â17260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Î<i>S</i> (cal/mol/deg)</td><td class="colsep0 rowsep0" align="char" char=".">7.36</td><td class="colsep0 rowsep0" align="char" char=".">10.9</td><td class="colsep0 rowsep0" align="char" char=".">â25.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Î<i>G</i> (cal/mol)</td><td class="colsep0 rowsep0" align="char" char=".">â8119.38</td><td class="colsep0 rowsep0" align="char" char=".">â9152.84</td><td class="colsep0 rowsep0" align="char" char=".">â9657.18</td></tr></tbody></table></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Bleomycin Challenge Increases Chit1 Expression</h3><div class="NLM_p">To test the in vivo efficacy of <b>40</b>, a well-established model of lung fibrosis was used, in which mice were intranasally sensitized with bleomycin. Eight days after the last of the three challenge doses were administered, the expression of Chit1 in the lung tissue was increased 3-fold and the chitinase activity of the bronchoalveolar lavage fluid (BALF) was also increased (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Since this time point is at the early stage of pulmonary fibrosis, the expressions of collagen gene <i>Col3</i> and essential cytokines during pathogenesis, including <i>Areg</i> and <i>TGFb</i>, were at similar levels in the bleomycin-challenged and control groups. As type 2 inflammation is involved in pulmonary fibrosis, leukocyte and gene expression of cytokines were also detected. In tissues (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), bleomycin did not have a significant impact on the total cell, eosinophil, neutrophil, monocyte, alveolar macrophage or interstitial macrophage numbers, or gene expression of the type 2 cytokines IL-5 and IL-13. The gene for type 2 initiators, <i>Il33</i>, was expressed at similar levels in both groups of mice. Bleomycin administration also had no significant impact on the total cell, eosinophil, neutrophil, and alveolar macrophage or interstitial macrophage numbers in the BALF at day 12. These data suggested that bleomycin increased Chit1 expression and chitinase activity in lung tissue before pulmonary immunity.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Expression levels of Chit1 and cytokines in lung tissue from the no-treatment, the bleomycin-challenged, and the <b>40</b>-treated bleomycin-challenged groups. The control contains 5% DMSO in PBS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cell recruitment in the lungs of bleomycin-induced mice treated with <b>40</b>. Graphs of the total cell, eosinophil, neutrophil, monocyte, alveolar macrophage, PD-L2<sup>+</sup> alveolar macrophage, interstitial macrophage, and PD-L2<sup>+</sup> interstitial macrophage numbers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Suppression of Chitinase Activity Enhances Lung Inflammation Induced by Tissue Damage</h3><div class="NLM_p">The mice were treated with bleomycin in the presence/absence of <b>40</b>. The chitinase activity in the BALF of these mice was then compared (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>)</a>. Administration of bleomycin alone induced an increase of chitinase activity by 40% compared with that of the control group; however, administration of bleomycin combined with <b>40</b> completely suppressed this elevation. In an in vitro assay, the addition of 10 Î¼g of <b>40</b> and 1 Î¼g of <b>40</b> reduced the chitinase activity in the BALF by 55 and 35%, respectively, suggesting that the suppression by <b>40</b> was dosage-dependent.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition activity of <b>40</b> in the BALF of mice treated with bleomycin in vivo and in vitro.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The impact of suppressing the chitinase activity was further investigated. In comparison with the bleomycin-treated groups, treatment with bleomycin combined with <b>40</b> did not alter the gene expression levels of <i>Chit1</i> and fibrosis cytokines or the amount of type 2 inflammation cytokines but did increase the number of macrophage cells in the lungs, including monocytes, PD-L2<sup>+</sup> alveolar macrophages and interstitial macrophages, and the expression level of the cytokine <i>Areg</i> in the lungs. This increase in the number of macrophage cells suggests that suppression of chitinase activity induced immunological responses. As <i>Areg</i> is a well-known epidermal growth factor that plays an essential role in the pathogenesis of pulmonary fibrosis, up-regulation of <i>Areg</i> is a sign of fibrosis.</div><div class="NLM_p">To confirm that <b>40</b> would have a similar activity against mChit1, inhibitory activities of <b>40</b> against mChit1 and mAMCase were determined in vitro (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As expected, <b>40</b> showed a similar activity against mChit1 as against the human enzyme with a <i>K<sub>i</sub></i> value of 35 nM and about 15-fold selectivity between mChit1 and mAMCase.</div><div class="NLM_p last">Additionally, <b>40</b> was tested against a selected set of key enzymes, transporters, and ion channels to characterize the activity profile.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> These include protein-tyrosine kinase (LCK), phosphodiesterase (PDE4D), cyclooxygenase (COX2), acetylcholinesterase (ACHE), monoamine oxidase (MAO-A), steroid nuclear receptors (GR), 5-hydroxytyramine transporter (5HTT), and K<sup>+</sup> channel (hERG). Compound <b>40</b> displayed no significant inhibitory activity against the majority of the proteins (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">Table S4</a>). However, <b>40</b> showed potential cardiotoxicity (98.8% inhibition at 10 Î¼M for hERG) and revealed inhibition of the phosphodiesterase enzyme (87.8% inhibition at 10 Î¼M for PDE4D2). This is concerning and should be addressed before further development of <b>40</b> for therapeutic purposes. Moreover, even though <b>40</b> inhibits mChit1 with high potency, it is not possible to completely exclude a scenario where inhibition of other proteins such as mAMCase might have contributed to the increase in inflammation, especially when <b>40</b> is only 15-fold selective for mChit1 as compared to 80-fold selectivity for <i>Hs</i>Chit1.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Structure-based virtual screening facilitated the identification of the dipyrido-pyrimidine-based scaffold that exhibited inhibitory activity against chitinases from various species such as humans, insects, and bacteria (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Among the dipyrido-pyrimidine derivatives, <b>40</b> was found to be the most potent <i>Hs</i>Chit1 inhibitor, and to the best of our knowledge, <b>40</b> is the most potent synthetic <i>Hs</i>Chit1 inhibitor reported to date. Natural products allosamidin (<b>1</b>) and demethylallosamidin are the only known <i>Hs</i>Chit1 inhibitors that are more potent than <b>40</b>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> However, their use in drug discovery was significantly impeded by their limited availability, difficulty in synthesis, and pharmacokinetic properties not suitable for in vivo studies. Furthermore, <b>40</b> demonstrated high selectivity against hAMCase and is the most selective <i>Hs</i>Chit1 inhibitor known to date. Recently, a highly potent inhibitor of mouse Chit1 (<b>17</b>) with 143-fold selectivity against mAMCase has also been reported, but its inhibitory effect on <i>Hs</i>Chit1 is not known.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The dipyrido-pyrimidine-based scaffold possessed broad chitinase inhibitory activity and also inhibited chitinases from insect pests and bacteria with high potency. Compound <b>53</b> was especially interesting, inhibiting <i>Of</i>Chi-h and <i>Sm</i>ChiB with potency in the low nM range. Importantly, this compound was only a weak inhibitor of <i>Hs</i>Chit1, making it an excellent candidate for agrochemical development.</div><div class="NLM_p">Binding of the most potent compounds <b>40</b> and <b>53</b> was confirmed by X-ray crystallography, and several crystal structures of human, insect, and bacterial chitinases were obtained in complex with these dipyrido-pyrimidine ligands, providing structural insight into the inhibitory mode and selectivity. Crystal structures revealed that both compounds bind to various chitinases in a similar binding mode forming similar interactions. The stacking interaction with two tryptophans at +1 and +2 subsites is the main driving force behind the binding of the dipyrido-pyrimidine scaffold to various chitinases, while substitutions at R1 and R4 positions control the potency and selectivity. Although compounds binding to both ân and +n subsites have previously been observed,<a onclick="showRef(event, 'ref13 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref13 ref33 ref34">(13,33,34)</a> dipyrido-pyrimidine ligands specifically bind to +1 and +2 subsites with functional groups extending toward the active site. This type of binding mode was not described previously in the literature and suggests a new subpocket to target for the development of highly potent inhibitors of family 18 chitinases.</div><div class="NLM_p">Comparison of the cocrystal structures of <i>Hs</i>Chit1/<b>40</b> and docking-predicted binding mode of <b>40</b> in the hAMCase binding pocket allowed us to elucidate the mechanism leading to the 80-fold selectivity of <b>40</b> for <i>Hs</i>Chit1 over hAMCase. Although the amino acid residues that form the binding pocket in both <i>Hs</i>Chit1 and hAMCase are highly conserved, the R269H substitution in hAMCase, which is one of the two different residues (the other is M300I in hAMCase) near the catalytic site may be responsible for the higher selectivity of <b>40</b> for <i>Hs</i>Chit1. SAR analysis suggested that all active compounds bear substituents with hydrogen bonding capabilities at R4 positions, which interact with residues in the vicinity. These interactions facilitated much favorable placement of the dipyrido-pyrimidine ring and other aromatic moieties for stronger ÏâÏ stacking with tryptophans not only at positive subsites but also near the catalytic subpocket. Reduction in overall protein-ligand contacts in hAMCase might have contributed to comparatively lower binding affinity of dipyrido-pyrimidine ligands against hAMCase. Our SAR analysis and comparison of crystal and docked <b>40</b> structures further provided guidelines for the development of hAMCase-selective compounds. It has been shown previously that a hydrophobic group facing the positively charged side chain of Arg269 in <i>Hs</i>Chit1 could have a negative effect on potency, whereas these groups can form van der Waals contacts with the hAMCase His269 imidazole ring.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Hence, to develop hAMCase-selective dipyrido-pyrimidine ligands, substitution of moieties such as ethylbenzyl carboxamide and propylbenzyl carboxamide at the R1 position could be preferred. Moreover, since the R1 group binding subpocket is slightly smaller in <i>Hs</i>Chit1, substitutions at the <i>meta</i>-position of the benzyl ring could be helpful in developing hAMCase-selective compounds.</div><div class="NLM_p">The cocrystal structures of <b>53</b> with <i>Of</i>Chi-h, <i>Hs</i>Chit1, and <i>Sm</i>ChiB outlined the unique binding mode of this compound. Binding of <b>53</b> to <i>Of</i>Chi-h was similar to those of <i>Hs</i>Chit1/<b>40</b> and <i>Sm</i>ChiB/<b>40</b> cocrystal structures with one molecule in +1 and +2 subsites. The dipyrido-pyrimidine moiety of <b>53</b> was found to occupy a similar position. However, the presence of a donor in the 3-pyridinyl moiety triggered the hydrogen bond formation with catalytic glutamate, which might be responsible for its high binding affinity against <i>Of</i>Chi-h. The crystal structure of <i>Sm</i>ChiB/<b>53</b> was a peculiar case as both types of binding modes were observed in each of the two <i>Sm</i>ChiB molecules in the asymmetric unit. The observation of two different binding modes in one enzyme suggests a concentration-dependent inhibitory mechanism.</div><div class="NLM_p">The discovery of <b>40</b> enabled us to understand the dynamic roles of Chit1 in inflammation by partially suppressing the activity of Chit1. In a bleomycin-induced lung fibrosis murine model, Chit1<sup>â/â</sup> mice appeared to have significantly reduced lung fibrosis, indicating that 100% suppression of Chitl protects mice from severe lung fibrosis. As indicated by inhibitory evaluation, <b>40</b> inhibited the chitinase activity of BALF by 50% in vivo compared with the elevated level of Chit1 induced by bleomycin. This attenuated level, which was comparable to that of bleomycin-untreated mice, led to a significant increase in inflammatory cells and factors. In our murine model, bleomycin treatment induced elevated expression levels of Chit1, but not collagen and fibrosis cytokine, type 2 inflammation initiator IL-33, and inflammation cytokines IL-5 and IL-13. These results indicated that Chit1 was one of the initiators to respond to bleomycin treatment. Partial suppression of Chit1 leading to inflammation suggested that Chit1 played a protective role. However, the possibility that the increased inflammation is due to the inhibition of other proteins such as AMCase could not be excluded. Moreover, as inflammation is a very complex process with multiple triggers, effectors, and mechanisms, the observed outcome (destructive or protective) may depend on several factors such as experiment design, dose, and method of administration, exposure duration, and markers. Therefore, additional investigations using <b>40</b> as a tool compound are required to determine the exact role of Chit1, destructive or protective. Based on our observations, we hypothesize that the homeostasis level of Chit1 under disease conditions determines its protective or destructive role and will be investigated in the future using tool compounds such as <b>40</b>.</div><div class="NLM_p">Our study resulted in the identification of compounds containing the dipyrido-pyrimidine scaffold with inhibitory activities against chitinases from human, insects, and bacteria. Among the reported dipyrido-pyrimidine derivatives, <b>40</b> and <b>53</b> are reasonably potent and selective against <i>Hs</i>Chit1 and <i>Of</i>Chi-h, respectively, and could be used as starting points for the development of useful tools for human therapy and pest controls. However, separate optimization efforts are needed for their utility as therapeutics or pesticides. For optimization efforts targeted toward therapeutics, safety is the most important factor. Since <b>40</b> showed inhibitory activities against hERG and PDE4D2, derivatives of <b>40</b> should be assessed not only for inhibition of human chitinases but also for their off-target activities. Although <b>40</b> does not violate Lipinskiâs rule of five,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> properties such as molecular weight and log <i>P</i> are toward the higher end and should be either reduced or maintained during optimization. Efforts should also be made to improve the solubility of compounds for oral bioavailability. Safety is also the most important factor in the development of pesticides, and future development efforts should be focused on improving the selectivity toward insect chitinase while avoiding any activity against human chitinase and other human and animal proteins. Cocrystal structures of <b>53</b> with both <i>Hs</i>Chit1 and <i>Of</i>Chi-h and differences in the active site can be exploited to improve selectivity. In addition, hydrophobicity of the dipyrido-pyrimidine scaffold should be maintained or improved during optimization as most pesticides are nonpolar hydrophobic molecules. Moreover, high water solubility is not a desired characteristic for the pesticide development due to the low soil adsorption coefficient and risk of polluting the ground water.</div><div class="NLM_p last">In summary, by utilizing a combination of in silico, in vitro and in vivo experiments, we have discovered and validated a chemical scaffold, which potently inhibited chitinases from humans, insects, and bacteria. Combining X-ray crystallography and computational modeling, we further revealed the molecular basis of inhibition and selectivity. The identified scaffold can be utilized to develop highly efficient and selective molecular tools to regulate chitinase, which is vital in human immunity, pathogenic fungi, and pest insect development.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Structure-Based Virtual Screening</h3><div class="NLM_p last">Structure-based virtual screening (SBVS) was carried out following a hierarchical virtual screening protocol,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> which is summarized in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A. Initially, a library of about 4 million commercially available compounds was filtered based on their shape and electrostatic potential similarities with three units of Î²-1,4-linked <i>N</i>-acetylglucosamine, (GlcNAc)<sub>3</sub>. The bioactive conformation of (GlcNAc)<sub>3</sub> was used as a query for shape and electrostatic comparisons (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WL1">3WL1</a>). The shape similarity calculations were performed using the ROCS program.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Ligand conformations for shape similarity calculations were generated employing the Omega program.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> A maximum number of 200 conformations were generated for each compound. TanimotoCombo, a combined Tanimoto coefficient<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> for the shape and chemical features, was used to rank-order the screening library. The 1000 top-ranked compounds in their ROCS-aligned conformations were then subjected to electrostatic potential comparisons using the EON program. The MMFF force field<a onclick="showRef(event, 'ref43 ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45 ref46 ref47">(43â47)</a> was used to add partial charges to the compounds. Similarity was assessed using a combined shape and electrostatic similarity metric (ET_Combo score). The top-ranking 500 compounds were then selected for further prioritization using molecular docking based on their interactions with the active site. Molecular docking was performed using the Glide program.<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48â50)</a> Ligands for molecular docking were prepared using LigPrep from SchrÃ¶dinger. Ionization and tautomeric states of compounds were generated at a target pH of 7.0 Â± 2.0 using the Epik program.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> The protein structure for molecular docking was prepared using protein preparation utility in Maestro (SchrÃ¶dinger Inc.) where hydrogens were added, bond orders were assigned, and protonation states of all charged residues were determined at the neutral pH. The grid for molecular docking was generated using the coordinates of (GlcNAc)<sub>3</sub> bound to <i>Of</i>Cht1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WL1">3WL1</a>). Glide âdocking scoreâ, which is a Glide score with Epik penalties, was used as a scoring function to rank-order compounds. The compound ranking for each method in the hierarchical virtual screening protocol is given in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv" class="ext-link">Table S5</a>. In addition, the 2D similarity between (GlcNAc)<sub>3</sub> and hit compounds using MACCS keys was also performed. The low Tanimoto scores for all hit compounds highlighted the importance of a hierarchical virtual screening protocol since a simple 2D similarity search would not yield comparable results. About 50 high-scoring compounds were visually inspected, and out of these, a set of 10 compounds were then cherry-picked to test their ability to block hydrolysis of substrate 4MU-Î²-(GlcNAc)<sub>2</sub> to 4-MU and (GlcNAc)<sub>2</sub> using a fluorometric assay. Out of these 10 selected compounds, one compound was not available commercially. After identifying the initial actives, two more rounds of virtual screening were carried out to identify structural analogs. A similar protocol to the first round was used in two subsequent rounds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The most active compound from each round was selected as a query for the shape and electrostatic similarity search. In addition, a substructure search was also used in the third round to identify structural analogs. Compound ranking for the second-round hits is given in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv" class="ext-link">Table S6</a>. Ranking was not employed to select compounds in the third round, and emphasis was given to compounds similar to the most potent ones in the previous round. Diversity and compound selection based on SAR exploration were also considered. Evaluation of the initial hit and its derivatives for PAINS<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and problematic compounds according to Eli Lilly MedChem Rules<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was performed using the Filter program from OpenEye Scientific Software and a ruby implementation of Eli Lilly MedChem Rules. Molecular docking of <b>40</b> to the AMCase crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RM8">3RM8</a>) was carried out using the Glide program. Ligand and protein structures were prepared using the ligand and protein structure preparation utility of Maestro (SchrÃ¶dinger Inc.) at a target pH of 2.0.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemicals</h3><div class="NLM_p last">Compounds were purchased from Topscience (<a href="http://www.tsbiochem.com" class="extLink">http://www.tsbiochem.com</a>), a local distributor of compounds from chemical vendors including Chembridge, Chemdiv, and Vitas-M. The purities of these compounds were verified to be >95%. All of the compounds were maintained as DMSO stock solutions.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Enzyme Preparation</h3><div class="NLM_p last">Human chitotriosidase catalytic domain (<i>Hs</i>Chit1-CAD), human acidic mammalian chitinase catalytic domain (hAMCase-CAD), mouse chitotriosidase catalytic domain (mChit1), mouse acidic mammalian chitinase catalytic domain (mAMCase-CAD), and <i>Ostrinia furnacalis</i> chitinase h (<i>Of</i>Chi-h) were overexpressed in <i>Pichia pastoris</i>, and <i>Sm</i>ChiB was overexpressed in <i>Escherichia coli</i> BL21 (DE3). The target proteins were purified using metal-chelating chromatography as described previously.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> All of the expressed proteins were quantified using a BCA protein assay kit (TaKaRa Biotech, China) with bovine serum albumin as the standard. The purity of these enzymes was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and found to be >95% in all cases.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chitinase Inhibition Assay and <i>K<sub>i</sub></i> Determination</h3><div class="NLM_p last">The chitinolytic activities of <i>Hs</i>Chit1-CAD, hAMCase-CAD, mChit1-CAD, mAMCase-CAD, <i>Of</i>Chi-h, and <i>Sm</i>ChiB were determined using 4-methylumbelliferyl Î²-D-<i>N</i>,<i>N</i>â²-diacetylchitobioside hydrate (4MU-(GlcNAc)<sub>2</sub>) (provided by Prof. Jianjun Zhang of China Agricultural University) as the substrate. The reaction mixture consisted of 100 Î¼L of 10 nM testing enzyme, 50 Î¼M 4MU-(GlcNAc)<sub>2</sub>, 10 Î¼M inhibitor, 1% DMSO, and 0.1% Triton X-100 in 20 mM sodium phosphate buffer (pH 6.0). The reaction mixture in the absence of compounds was used as a blank control. After incubation at 30 Â°C for 30 min, 0.5 M sodium carbonate was added to stop the reaction, and the fluorescence produced by the released 4-MU was quantified with a Varioskan Flash microplate reader (Thermo Fisher Scientific) using excitation and emission wavelengths of 360 and 440 nm, respectively. Experiments were performed in triplicate unless otherwise specified. For the determination of <i>K<sub>i</sub></i>, the substrate concentration was set at 50, 30, and 15 Î¼M. The inhibitor concentration varied according to the inhibition activity. <i>K<sub>i</sub></i> values were obtained by curve fitting using Prism software (GraphPad).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cocrystallization and Data Collection</h3><div class="NLM_p last"><i>Hs</i>Chit1-CAD, <i>Of</i>Chi-h, and <i>Sm</i>ChiB were crystallized as described previously.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53â55)</a> Briefly, <i>Hs</i>Chit1-CAD was concentrated to 9 mg/mL in 10 mM HEPES (pH 7.5) and 50 mM NaCl. Then the enzyme was incubated with <b>40</b> or <b>53</b> at a ratio of 1:10 at 277 K for 12 h. Crystals were grown by the hanging-drop vapor-diffusion method at 293 K by mixing 1 Î¼L of the protein solution and an equal volume of a well solution containing 25% (w/v) polyethylene glycol (PEG) 3350 and 0.2 M potassium sodium tartrate (PST) at pH 7.2. <i>Of</i>Chi-h incubated with <b>53</b> was spin-concentrated to 10 mg/mL in 20 mM bis-Tris (pH 6.5) containing 50 mM NaCl. The hanging-drop vapor-diffusion crystallization experiments were set up by mixing 1 Î¼L of <i>Of</i>Chi-h and 1 Î¼L of reservoir solution containing 100 mM HEPES pH 7.0 and 30% (w/v) Jeffamine ED-2001. <i>Sm</i>ChiB was incubated with a 10-fold molar excess of <b>40</b> or <b>53</b>. For crystallization, 1 Î¼L of 20 mg/mL protein solution was mixed with an equal volume of a well solution containing 50 mM citrate (pH 5.6), 0.5 M Li<sub>2</sub>SO<sub>4</sub>, and 0.25 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>. Crystals appeared within 1â2 weeks at 293 K. Afterward, the crystals were soaked for 10 s in a reservoir solution containing 10% (v/v) glycerol as a cryoprotectant and subsequently flash-cooled in liquid nitrogen. The X-ray diffraction data of the complexes were collected on the BL-19 U1 at Shanghai Synchrotron Radiation Facility in China. The diffraction data of the complexes were processed using the HKL-3000 package.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Structure Determination and Refinement</h3><div class="NLM_p">The structures of <i>Hs</i>Chit1/<b>40</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR">6JJR</a>), <i>Hs</i>Chit1/ <b>53</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6">6JK6</a>), <i>Of</i>Chi-h/<b>53</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN">6JMN</a>), <i>Sm</i>ChiB/<b>40</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9">6JK9</a>), and <i>Sm</i>ChiB/<b>53</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF">6JKF</a>) were solved by molecular replacement with Phaser.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The PHENIX suite of programs was used for structure refinement.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Coot was used to manually build and extend the molecular models.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The quality of the models was checked by PROCHECK.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Structural figures were generated using PyMOL. The statistics of the diffraction data and structure refinement are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Data Collection and Refinement Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center"><i>Hs</i>Chit1/<b>40</b></th><th class="colsep0 rowsep0" align="center"><i>Hs</i>Chit1/<b>53</b></th><th class="colsep0 rowsep0" align="center"><i>Sm</i>ChiB/<b>40</b></th><th class="colsep0 rowsep0" align="center"><i>Sm</i>ChiB/<b>53</b></th><th class="colsep0 rowsep0" align="center"><i>Of</i>Chi-h/<b>53</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Protein Data Bank entry</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR">6JJR</a></td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6">6JK6</a></td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9">6JK9</a></td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF">6JKF</a></td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN">6JMN</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="center"><i>P</i>22<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="center"><i>P</i>4<sub>3</sub></td><td class="colsep0 rowsep0" align="center"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="center"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="center"><i>P</i>22<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit-cell parameters</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i> (Ã)</td><td class="colsep0 rowsep0" align="center">42.295</td><td class="colsep0 rowsep0" align="center">66.726</td><td class="colsep0 rowsep0" align="center">55.375</td><td class="colsep0 rowsep0" align="center">55.767</td><td class="colsep0 rowsep0" align="center">48.496</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>b</i> (Ã)</td><td class="colsep0 rowsep0" align="center">86.161</td><td class="colsep0 rowsep0" align="center">66.726</td><td class="colsep0 rowsep0" align="center">103.667</td><td class="colsep0 rowsep0" align="center">104.997</td><td class="colsep0 rowsep0" align="center">108.684</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>c</i> (Ã)</td><td class="colsep0 rowsep0" align="center">105.134</td><td class="colsep0 rowsep0" align="center">88.363</td><td class="colsep0 rowsep0" align="center">185.438</td><td class="colsep0 rowsep0" align="center">185.868</td><td class="colsep0 rowsep0" align="center">122.442</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Î± (Â°)</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Î² (Â°)</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Î³ (Â°)</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90.0</td><td class="colsep0 rowsep0" align="center">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Ã)</td><td class="colsep0 rowsep0" align="center">0.97778</td><td class="colsep0 rowsep0" align="center">0.97778</td><td class="colsep0 rowsep0" align="center">0.97778</td><td class="colsep0 rowsep0" align="center">0.97778</td><td class="colsep0 rowsep0" align="center">0.97778</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">temperature (K)</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="center">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Ã)</td><td class="colsep0 rowsep0" align="center">43.08â1.834 (1.9â1.834)</td><td class="colsep0 rowsep0" align="center">29.84â1.571 (1.628â1.571)</td><td class="colsep0 rowsep0" align="center">29.62â2.312 (2.395â2.312)</td><td class="colsep0 rowsep0" align="center">29.71â1.99 (2.061â1.99)</td><td class="colsep0 rowsep0" align="center">27.17â2.502 (2.592â2.502)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="center">33762 (1783)</td><td class="colsep0 rowsep0" align="center">53786 (5407)</td><td class="colsep0 rowsep0" align="center">47308 (4405)</td><td class="colsep0 rowsep0" align="center">75464 (7177)</td><td class="colsep0 rowsep0" align="center">23042 (2269)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">observed reflections</td><td class="colsep0 rowsep0" align="center">203595</td><td class="colsep0 rowsep0" align="center">399280</td><td class="colsep0 rowsep0" align="center">479942</td><td class="colsep0 rowsep0" align="center">991829</td><td class="colsep0 rowsep0" align="center">296941</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="center">0.100 (0.909)</td><td class="colsep0 rowsep0" align="center">0.048 (0.132)</td><td class="colsep0 rowsep0" align="center">0.101 (0.469)</td><td class="colsep0 rowsep0" align="center">0.119 (0.469)</td><td class="colsep0 rowsep0" align="center">0.075 (0.735)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average multiplicity</td><td class="colsep0 rowsep0" align="center">6.0 (5.8)</td><td class="colsep0 rowsep0" align="center">7.4 (7.0)</td><td class="colsep0 rowsep0" align="center">12.8 (12.6)</td><td class="colsep0 rowsep0" align="center">13.1 (13.3)</td><td class="colsep0 rowsep0" align="center">12.8 (12.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>I</i>/Ï(<i>I</i>)</td><td class="colsep0 rowsep0" align="center">15.2 (1.8)</td><td class="colsep0 rowsep0" align="center">37.0 (16.2)</td><td class="colsep0 rowsep0" align="center">25.4 (4.5)</td><td class="colsep0 rowsep0" align="center">25.5 (4.4)</td><td class="colsep0 rowsep0" align="center">34.0 (2.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="center">85.06 (53.03)</td><td class="colsep0 rowsep0" align="center">99.96 (99.93)</td><td class="colsep0 rowsep0" align="center">98.98 (93.91)</td><td class="colsep0 rowsep0" align="center">99.57 (96.46)</td><td class="colsep0 rowsep0" align="center">99.72 (99.65)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="center">0.1674/0.1943</td><td class="colsep0 rowsep0" align="center">0.1662/0.1875</td><td class="colsep0 rowsep0" align="center">0.2059/0.2319</td><td class="colsep0 rowsep0" align="center">0.1821/0.2014</td><td class="colsep0 rowsep0" align="center">0.2143(0.2444)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein atoms</td><td class="colsep0 rowsep0" align="center">2923</td><td class="colsep0 rowsep0" align="center">2861</td><td class="colsep0 rowsep0" align="center">7804</td><td class="colsep0 rowsep0" align="center">7809</td><td class="colsep0 rowsep0" align="center">4197</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="center">366</td><td class="colsep0 rowsep0" align="center">355</td><td class="colsep0 rowsep0" align="center">215</td><td class="colsep0 rowsep0" align="center">691</td><td class="colsep0 rowsep0" align="center">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">other atoms</td><td class="colsep0 rowsep0" align="center">34</td><td class="colsep0 rowsep0" align="center">66</td><td class="colsep0 rowsep0" align="center">68</td><td class="colsep0 rowsep0" align="center">99</td><td class="colsep0 rowsep0" align="center">61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">r.m.s. deviation from ideal</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Ã)</td><td class="colsep0 rowsep0" align="center">0.009</td><td class="colsep0 rowsep0" align="center">0.008</td><td class="colsep0 rowsep0" align="center">0.006</td><td class="colsep0 rowsep0" align="center">0.005</td><td class="colsep0 rowsep0" align="center">0.015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (Â°)</td><td class="colsep0 rowsep0" align="center">0.78</td><td class="colsep0 rowsep0" align="center">0.98</td><td class="colsep0 rowsep0" align="center">0.80</td><td class="colsep0 rowsep0" align="center">0.79</td><td class="colsep0 rowsep0" align="center">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson <i>B</i> factor (Ã<sup>2</sup>)</td><td class="colsep0 rowsep0" align="center">19.38</td><td class="colsep0 rowsep0" align="center">12.21</td><td class="colsep0 rowsep0" align="center">35.50</td><td class="colsep0 rowsep0" align="center">26.57</td><td class="colsep0 rowsep0" align="center">62.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i> factor (Ã<sup>2</sup>)</td><td class="colsep0 rowsep0" align="center">21.71</td><td class="colsep0 rowsep0" align="center">17.70</td><td class="colsep0 rowsep0" align="center">46.00</td><td class="colsep0 rowsep0" align="center">30.95</td><td class="colsep0 rowsep0" align="center">68.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein atoms</td><td class="colsep0 rowsep0" align="center">20.55</td><td class="colsep0 rowsep0" align="center">16.02</td><td class="colsep0 rowsep0" align="center">45.95</td><td class="colsep0 rowsep0" align="center">30.34</td><td class="colsep0 rowsep0" align="center">68.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="center">30.53</td><td class="colsep0 rowsep0" align="center">30.04</td><td class="colsep0 rowsep0" align="center">46.51</td><td class="colsep0 rowsep0" align="center">36.64</td><td class="colsep0 rowsep0" align="center">69.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand molecules</td><td class="colsep0 rowsep0" align="center">26.67</td><td class="colsep0 rowsep0" align="center">24.42</td><td class="colsep0 rowsep0" align="center">50.42</td><td class="colsep0 rowsep0" align="center">39.29</td><td class="colsep0 rowsep0" align="center">69.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot (%)</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored</td><td class="colsep0 rowsep0" align="center">98.10</td><td class="colsep0 rowsep0" align="center">98.62</td><td class="colsep0 rowsep0" align="center">98.58</td><td class="colsep0 rowsep0" align="center">98.18</td><td class="colsep0 rowsep0" align="center">97.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed</td><td class="colsep0 rowsep0" align="center">1.90</td><td class="colsep0 rowsep0" align="center">1.38</td><td class="colsep0 rowsep0" align="center">1.42</td><td class="colsep0 rowsep0" align="center">1.82</td><td class="colsep0 rowsep0" align="center">2.81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">outliers</td><td class="colsep0 rowsep0" align="center">0.00</td><td class="colsep0 rowsep0" align="center">0.00</td><td class="colsep0 rowsep0" align="center">0.00</td><td class="colsep0 rowsep0" align="center">0.00</td><td class="colsep0 rowsep0" align="center">0.19</td></tr></tbody></table></div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Isothermal Titration Calorimetry</h3><div class="NLM_p last"><i>Hs</i>Chit1, <i>Sm</i>ChiB, and <i>Of</i>Chi-h were desalted into ITC buffer (50 mM NaH<sub>2</sub>PO<sub>4</sub>, 150 mM NaCl, pH 6.0). Compound <b>53</b> was dissolved in the same buffer. Measurements of <i>Hs</i>Chit1 were conducted at 25 Â°C with a stirring speed of 1000 rpm. After the baseline was balanced, 2.5 Î¼L of 1 mM <b>53</b> was injected into a calorimeter cell containing 100 Î¼M <i>Hs</i>Chit1. The injections were repeated 16 times over 180 s intervals. The background was measured by injecting the inhibitor into the cell containing only buffer. ITC experiments with <i>Sm</i>ChiB or <i>Of</i>Chi-h were performed as described for <i>Hs</i>Chit1 but with different proteins and inhibitor concentrations (40 Î¼M <i>Sm</i>ChiB, 400 Î¼M <b>53</b> or 10 Î¼M <i>Of</i>Chi-h, 100 Î¼M <b>53</b>). After subtracting the blank, the integrated heat of injection was used to fit one site binding model using Microcal Origin 7.0. Gibbs free energy (Î<i>G</i>) and the entropy change (Î<i>S</i>) can be calculated from the enthalpy change (Î<i>H</i>) and the binding constant <i>K</i> using Î<i>G</i> = â<i>RT</i> ln <i>K</i> and Î<i>G</i> = Î<i>H</i> â <i>T</i>Î<i>S</i>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Selectivity Profile</h3><div class="NLM_p last">The selectivity profile of <b>40</b> against a subset of proteins selected from the early in vitro mini safety panel of WuXi AppTec was measured by the vendor according to published protocols.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Biological Methods (in Vivo)</h3><div class="NLM_p last">All experiments were approved by the Animal Care and Use Committee of RIKEN and were performed in accordance with institutional guidelines.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cell Preparation and Flow Cytometry</h3><div class="NLM_p last">Lung and bronchoalveolar lavage fluid (BALF) cells were prepared as described previously.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Briefly, lungs were minced with scissors and incubated with 10% FBS/RPMI-1640 containing 50 Î¼g/mL Liberase TM (Roche, Basel, Switzerland) and 1 Î¼g/mL DNase I (Roche) for 1 h at 37 Â°C. Digested lung tissue was homogenized with GentleMACS (Miltenyi Biotec) and then passed through a strainer. After density-gradient centrifugation using a 30% (v/v) Percoll solution (GE Healthcare), cells were used for flow cytometric analysis. BALF cells were collected by washing the lungs thrice with 500 Î¼L of PBS. For flow cytometric analysis, FITC-conjugated anti-Gr-1 (RB6-8C5), Alexa647-conjugated anti-Siglec-F (E50-2440), PE-Cy7-conjugated anti-CD11c (HL3), BV421-conjugated anti-CD11b (M1/70), and V500-conjugated anti-CD45 (30-F11) antibodies were purchased from BD Biosciences. PE-conjugated anti-PD-L2 (TY25) and APC-Cy7-conjugated anti-F4/80 (BM8) antibodies were purchased from Biolegend. Lung and BALF cells were analyzed: CD45<sup>+</sup>SiglecF<sup>+</sup>CD11c<sup>lo</sup> cells as eosinophils, CD45<sup>+</sup> CD11b<sup>+</sup> Gr1<sup>hi</sup> cells as neutrophils, CD45<sup>+</sup> Autofluorescence<sup>+</sup>SiglecF<sup>+</sup>CD11c<sup>hi</sup> cells as alveolar macrophages, CD45<sup>+</sup>Autofluorescence<sup>+</sup>SiglecF<sup>lo</sup>F4/80<sup>+</sup> cells as interstitial macrophages, and CD45<sup>+</sup> Autofluorescence-MHC classII<sup>+</sup>CD11b<sup>+</sup> Gr1<sup>lo</sup> as monocytes.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Bleomycin-Induced Inflammation and Challenge</h3><div class="NLM_p last">All experiments used 8 week old female C57BL6/N mice from Charles River. Mice were intranasally (i.n.) administered with 6 Î¼g/mouse bleomycin for 3 consecutive days. Compound <b>40</b> or vehicle (5% DMSO in PBS) was intraperitoneally (i.p.) administered from day 0 to day 11 at 2 Î¼g/mouse. Necropsies were performed 24 h after the final challenge (day 12).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Quantitative Polymerase Chain Reaction (qPCR)</h3><div class="NLM_p last">Total RNA was extracted from lung tissue using the TRIzol reagent (Thermo Fischer Scientific), and cDNA was synthesized with the SuperScript III First-Strand Synthesis System for RT-PCR (Sigma). qPCR was carried out using an SYBR Premix Ex Taq II (Tli RNase H Plus, Takara Bio) on a StepOne Plus Real-Time PCR System qPCR instrument (Applied Biosystems) using the following primers: <i>Chit1</i>, forward 5â²-TGACTCTGCTGATGGTCCAG-3â² and reverse 5â²-TAGGCTGTTCAGCTCCTGGT-3â²; <i>Col3</i>, forward 5â²-TCTCTAGACTCATAGGACTGACC-3â² and reverse 5â²-TTCTTCTCACCCTTCTTCATCC-3â²; <i>Areg</i>, forward 5â²-GGTCTTAGGCTCAGGCCATTA-3â² and reverse 5â²-CGCTTATGGTGGAAACCTCTC-3â²; <i>Tgfb</i>, forward 5â²-CACCTGCAAGACCATCGACAT-3â² and reverse 5â²-GAGCCTTAGTTTGGACAGGATCTG-3â²; <i>Il33</i>, forward 5â²- CCTTCTCGCTGATTTCCAAG-3â² and reverse 5â²-CCGTTACGGATATGGTGGTC-3â²; <i>Il5</i>, forward 5â²-AGCAATGAGACGATGAGGCT-3â² and reverse 5â²-CCCACGGACAGTTTGATTC-3â²; <i>Il13</i>, forward 5â²-TGTGTCTCTCCCTCTGACCC-3â² and reverse 5â²-TCCAGGGCTACACAGAACCC-3â²; and internal control <i>Î²-actin</i>, forward 5â²-ACTATTGGCAACGAGCGGTTC-3â² and reverse 5â²-GGATGCCACAGGATTCCATAC-3â². Data were normalized against Î²-actin by the ÎÎ<i>Ct</i> method.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Chit1 Activity Assay</h3><div class="NLM_p last">10 Î¼L of BALF and supernatants of cultured cells were incubated with 10 Î¼L of 0.5 mM 4-MU-(GlcNAc)<sub>2</sub> substrate in 80 Î¼L of buffer (0.1 M citric acid and 0.2 M sodium phosphate (pH 6.0)) at 37 Â°C for 15 min. After incubation, the reaction was stopped by mixing with 100 Î¼L of 0.5 M Na<sub>2</sub>CO<sub>3</sub> buffer. Released 4-MU was quantified with a nano-drop 3000 (Thermo Fischer Scientific).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01154" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01154?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01154</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Docked complex of compound <b>40</b> in hAMCase (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Output file from PAINS filtering (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_003.txt" class="ext-link">TXT</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_004.txt" class="ext-link">TXT</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_005.txt" class="ext-link">TXT</a>)</p></li><li><p class="inline">Selectivity profile of compound <b>40</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_006.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Dipyrido-pyrimidine scaffold and chemical structures at R1 to R4 positions; inhibitory activities for pairs of compounds that differ at the R2 or R3 position; inhibitory activities for each functional group at the R1 position; inhibitory activities for each functional group at the R4 position; sequence alignment of <i>Hs</i>Chit1, hAMCase, <i>Of</i>Chi-h, and <i>Sm</i>ChiB; docking-predicted structure of <b>40</b> with hAMCase; distribution of binding pocket volumes of <i>Hs</i>Chit1 and hAMCase; ITC profiles of compound <b>53</b> binding to <i>Hs</i>Chit1, <i>Sm</i>ChiB, and <i>Of</i>Chi-h; inhibition rates of compounds identified in the first round; inhibition rates of compounds identified in the second round; inhibition rates of compounds identified in the third round; selectivity profile of <b>40</b> against a selected set of diverse proteins; ranking of the first-round hits for each method in the hierarchical virtual screening protocols; ranking of the second-round hits for each method in the hierarchical virtual screening protocol; the Dixon plots for <i>K<sub>i</sub></i> determination; <sup>1</sup>H NMR or MS spectra of tested compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv">jm9b01154_si_001.csv (7.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_002.pdb">jm9b01154_si_002.pdb (470.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_003.txt">jm9b01154_si_003.txt (6.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_004.txt">jm9b01154_si_004.txt (8.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_005.txt">jm9b01154_si_005.txt (8.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_006.xlsx">jm9b01154_si_006.xlsx (498.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf">jm9b01154_si_007.pdf (4.07 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the chitinase-inhibitor complexes described in this paper have been deposited with PDB with accession codes: 6JJR (<i>Hs</i>Chit1/<b>40</b>), 6JK6 (<i>Hs</i>Chit1/<b>53</b>), 6JK9 (<i>Sm</i>ChiB/<b>40</b>), 6JKF (<i>Sm</i>ChiB/<b>53</b>), and 6JMN (<i>Of</i>Chi-h/<b>53</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01154" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kam Y. J. Zhang</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Structural Bioinformatics, Center for Biosystems Dynamics Research, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9282-8045" title="Orcid link">http://orcid.org/0000-0002-9282-8045</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3259535f485a535c5572405b59575c1c5842"><span class="__cf_email__" data-cfemail="ea818b8790828b848daa9883818f84c4809a">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qing Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection and Shenzhen Agricultural
Genome Research Institute, Chinese Academy of Agricultural Sciences, 2 West Yuanmingyuan Road, Beijing 100193, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">School
of Bioengineering, Dalian University of
Technology, 2 Linggong Road, Dalian 116024, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8623-7532" title="Orcid link">http://orcid.org/0000-0001-8623-7532</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#98e9f1f6ffe1f9f6ffd8fcf4edecb6fdfcedb6fbf6"><span class="__cf_email__" data-cfemail="7001191e1709111e1730141c05045e1514055e131e">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xi Jiang</span> - <span class="hlFld-Affiliation affiliation">School
of Bioengineering, Dalian University of
Technology, 2 Linggong Road, Dalian 116024, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ashutosh Kumar</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Structural Bioinformatics, Center for Biosystems Dynamics Research, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasutaka Motomura</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Innate Immune Systems, Center for Integrative Medical Sciences, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Laboratory
for Innate Immune Systems, Department of Microbiology and Immunology,
Graduate School of Medicine, Osaka University, 2-2 Yamadaoka,
Suita-shi, Osaka 565-0871, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tian Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Bioengineering, Dalian University of
Technology, 2 Linggong Road, Dalian 116024, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9768-5496" title="Orcid link">http://orcid.org/0000-0001-9768-5496</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Zhou</span> - <span class="hlFld-Affiliation affiliation">School of Software, Dalian University of
Technology, 2 Linggong Road, Dalian 116024, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kazuyo Moro</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Innate Immune Systems, Center for Integrative Medical Sciences, RIKEN, 1-7-22 Suehiro, Tsurumi, Yokohama, Kanagawa 230-0045, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Laboratory
for Innate Immune Systems, Department of Microbiology and Immunology,
Graduate School of Medicine, Osaka University, 2-2 Yamadaoka,
Suita-shi, Osaka 565-0871, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.J. and A.K. contributed equally. The manuscript was written through contributions of all authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Key R&D Program of China (2017YFD0200501), the National Natural Science Foundation of China (31830076) and Shenzhen Science and Technology Program (GrantÂ No.Â KQTD20180411143628272). We thank the staff of BL18U/BL19U1 Beamline of National Facility for Protein Science Shanghai at Shanghai Synchrotron Radiation Facility for assistance during data collection. We also thank Natsuki Takeno and Miho Mochizuki for their technical support. We acknowledge RIKEN ACCC for the supercomputing resources at the Hokusai BigWaterfall supercomputer used in this study. Part of this research was supported by Platform Project for Supporting Drug Discovery and Life Science Research (Basis for Supporting Innovative Drug Discovery and Life Science Research (BINDS)) from Japan Agency for Medical Research and Development (AMED) under Grant Number JP18am0101082.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">GlcNAc</td><td class="NLM_def"><p class="first last"><i>N</i>-acetyl-D-glucosamine</p></td></tr><tr><td class="NLM_term">Chit1</td><td class="NLM_def"><p class="first last">chitotriosidase</p></td></tr><tr><td class="NLM_term">AMCase</td><td class="NLM_def"><p class="first last">acidic mammalian chitinase</p></td></tr><tr><td class="NLM_term">OVA</td><td class="NLM_def"><p class="first last">ovalbumin</p></td></tr><tr><td class="NLM_term">SBVS</td><td class="NLM_def"><p class="first last">structure-based virtual screening</p></td></tr><tr><td class="NLM_term"><i>Of</i>Chi-h</td><td class="NLM_def"><p class="first last"><i>Ostrinia furnacalis</i> chitinase h</p></td></tr><tr><td class="NLM_term">SDS-PAGE</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate-polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">4MU-(GlcNAc)<sub>2</sub></td><td class="NLM_def"><p class="first last">4-methylumbelliferyl Î²-D-<i>N</i>,<i>N</i>â²-diacetylchitobioside hydrate</p></td></tr><tr><td class="NLM_term">BALF</td><td class="NLM_def"><p class="first last">bronchoalveolar lavage fluid</p></td></tr><tr><td class="NLM_term">SEA</td><td class="NLM_def"><p class="first last">similarity ensemble approach</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure activity relationship</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollak, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Weely, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Oers, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span> <span> </span><span class="NLM_article-title">Marked elevation of plasma chitotriosidase activity. A novel hallmark of Gaucher disease</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1288</span>â <span class="NLM_lpage">1292</span>, <span class="refDoi">Â DOI: 10.1172/JCI117084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1172%2FJCI117084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=8132768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADyaK2c7nvFCkug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1994&pages=1288-1292&author=C.+E.+Hollakauthor=S.+van+Weelyauthor=M.+H.+van+Oersauthor=J.+M.+Aerts&title=Marked+elevation+of+plasma+chitotriosidase+activity.+A+novel+hallmark+of+Gaucher+disease&doi=10.1172%2FJCI117084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Marked elevation of plasma chitotriosidase activity. A novel hallmark of Gaucher disease</span></div><div class="casAuthors">Hollak C E; van Weely S; van Oers M H; Aerts J M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1288-92</span>
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">Gaucher disease (GD; glucosylceramidosis) is caused by a deficient activity of the enzyme glucocerebrosidase (GC).  Clinical manifestations are highly variable and cannot be predicted accurately on the basis of the properties of mutant GC.  Analysis of secondary abnormalities, such as elevated plasma levels of some hydrolases, may help to increase insight into the complicated pathophysiology of the disease and could also provide useful disease markers.  The recent availability of enzyme supplementation therapy for GD increases the need for markers as early predictors of the efficacy of treatment.  We report the finding of a very marked increase in chitotrisidase activity in plasma of 30 of 32 symptomatic type 1 GD patients studied: the median activity being > 600 times the median value in plasma of healthy volunteers.  In three GC-deficient individuals without clinical symptoms, only slight increases were noted.  Chitotriosidase activity was absent in plasma of three control subjects and two patients.  During enzyme supplementation therapy, chitotriosidase activity declined dramatically.  We conclude that plasma chitotriosidase levels can serve as a new diagnostic hallmark of GD and should prove to be useful in assessing whether clinical manifestations of GD are present and for monitoring the efficacy of therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHCFYsz4T-0TMsgLhIoICsfW6udTcc2eZY9OgWOwc-Vrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c7nvFCkug%253D%253D&md5=919539442ad855c2d21e7694637bb3cf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1172%2FJCI117084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI117084%26sid%3Dliteratum%253Aachs%26aulast%3DHollak%26aufirst%3DC.%2BE.%26aulast%3Dvan%2BWeely%26aufirst%3DS.%26aulast%3Dvan%2BOers%26aufirst%3DM.%2BH.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26atitle%3DMarked%2520elevation%2520of%2520plasma%2520chitotriosidase%2520activity.%2520A%2520novel%2520hallmark%2520of%2520Gaucher%2520disease%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1994%26volume%3D93%26spage%3D1288%26epage%3D1292%26doi%3D10.1172%2FJCI117084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kzhyshkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gratchev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goerdt, S.</span></span> <span> </span><span class="NLM_article-title">Human chitinases and chitinase-like proteins as indicators for inflammation and cancer</span>. <i>Biomarker Insights</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">128</span>â <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1177%2F117727190700200023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19662198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BD1MrktFSmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=128-146&author=J.+Kzhyshkowskaauthor=A.+Gratchevauthor=S.+Goerdt&title=Human+chitinases+and+chitinase-like+proteins+as+indicators+for+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human chitinases and chitinase-like proteins as indicators for inflammation and cancer</span></div><div class="casAuthors">Kzhyshkowska Julia; Gratchev Alexei; Goerdt Sergij</div><div class="citationInfo"><span class="NLM_cas:title">Biomarker insights</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">128-46</span>
        ISSN:<span class="NLM_cas:issn">1177-2719</span>.
    </div><div class="casAbstract">Human Glyco_18 domain-containing proteins constitute a family of chitinases and chitinase-like proteins.  Chitotriosidase and AMCase are true enzymes which hydrolyse chitin and have a C-terminal chitin-binding domain.  YKL-40, YKL-39, SI-CLP and murine YM1/2 proteins possess solely Glyco_18 domain and do not have the hydrolytic activity.  The major sources of Glyco_18 containing proteins are macrophages, neutrophils, epithelial cells, chondrocytes, synovial cells, and cancer cells.  Both macrophages and neutrophils use the regulated secretory mechanism for the release of Glyco_18 containing proteins.  Glyco_18 containing proteins are established biomarkers for human diseases.  Chitotriosidase is overproduced by lipid-laden macrophages and is a major marker for the inherited lysosomal storage Gaucher disease.  AMCase and murine lectin YM1 are upregulated in Th2-environment, and enzymatic activity of AMCase contributes to asthma pathogenesis.  YKL proteins act as soluble mediators for the cell proliferation and migration, and are also involved in rheumatoid arthritis, inflammatory bowel disease, hepatic fibrosis and cirrhosis.  Chitotriosidase and YKL-40 reflect the macrophage activation in atherosclerotic plaques.  Serum level of YKL-40 is a diagnostic and prognostic marker for numerous types of solid tumors.  YKL-39 is a marker for the activation of chondrocytes and the progression of the osteoarthritis in human.  Recently identified SI-CLP is upregulated by Th2 cytokine IL-4 as well as by glucocorticoids.  This unique feature of SI-CLP makes it an attractive candidate for the examination of individual sensitivity of patients to glucocorticoid treatment and prediction of side effects of glucocorticoid therapy.  Human chitinases and chitinase-like proteins are found in tissues and circulation, and can be detected by non-invasive technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZmzIeeqUN7ZdYoSR9eyvsfW6udTcc2eZY9OgWOwc-Vrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrktFSmtQ%253D%253D&md5=23bce5cbcab08e01a3e9ffc2948f30a3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1177%2F117727190700200023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F117727190700200023%26sid%3Dliteratum%253Aachs%26aulast%3DKzhyshkowska%26aufirst%3DJ.%26aulast%3DGratchev%26aufirst%3DA.%26aulast%3DGoerdt%26aufirst%3DS.%26atitle%3DHuman%2520chitinases%2520and%2520chitinase-like%2520proteins%2520as%2520indicators%2520for%2520inflammation%2520and%2520cancer%26jtitle%3DBiomarker%2520Insights%26date%3D2007%26volume%3D2%26spage%3D128%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boven, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Meurs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laman, J. D.</span></span> <span> </span><span class="NLM_article-title">Gaucher cells demonstrate a distinct macrophage phenotype and resemble alternatively activated macrophages</span>. <i>Am. J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">359</span>â <span class="NLM_lpage">369</span>, <span class="refDoi">Â DOI: 10.1309/BG5VA8JRDQH1M7HN</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1309%2FBG5VA8JRDQH1M7HN" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15362365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotFyhurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2004&pages=359-369&author=L.+A.+Bovenauthor=M.+van+Meursauthor=R.+G.+Bootauthor=A.+Mehtaauthor=L.+Boonauthor=J.+M.+Aertsauthor=J.+D.+Laman&title=Gaucher+cells+demonstrate+a+distinct+macrophage+phenotype+and+resemble+alternatively+activated+macrophages&doi=10.1309%2FBG5VA8JRDQH1M7HN"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Gaucher cells demonstrate a distinct macrophage phenotype and resemble alternatively activated macrophages</span></div><div class="casAuthors">Boven, Leonie A.; van Meurs, Marjan; Boot, Rolf G.; Mehta, Atul; Boon, Louis; Aerts, Johannes M.; Laman, Jon D.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">359-369</span>CODEN:
                <span class="NLM_cas:coden">AJCPAI</span>;
        ISSN:<span class="NLM_cas:issn">0002-9173</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Pathologists</span>)
        </div><div class="casAbstract">Although the existence of anti-inflammatory alternatively activated macrophages (aaMÎ¦) has been accepted widely based on in vitro studies, their in vivo location, phenotype, and function still are debated.  Gaucher disease (GD) is caused by a genetic deficiency in the lysosomal enzyme glucocerebrosidase and is characterized by accumulation of glycosphingolipids in so-called Gaucher cells (GCs), By using immunohistochem. anal., we investigated whether this results in an aaMÎ¦ phenotype.  GCs are macrophage-like cells, expressing acid phosphatase, CD68, CD14, and HLA class II, but not CD11b, CD40, or dendritic cell markers.  GCs show infrequent immunoreactivity for mannose receptor.  GCs did not express proinflammatory cytokines such as tumor necrosis factor Î± and monocyte chemoattractant protein 1, but did express the aaMÎ¦ markers CD163, CCL18, and interleukin-1 receptor antagonist.  Furthermore, CD36 and signal receptor protein Î±, involved in lipid uptake, also were obsd. on GCs.  Thus, GCs represent a distinctive population of myeloid cells that resemble aaMÎ¦ but differ from previously described in vitro aaMÎ¦.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG3PZoxUXeabVg90H21EOLACvtfcHk0lgsy2AdwoBfCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotFyhurg%253D&md5=2403ba7fd5c4d4921c14a1756fef30fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1309%2FBG5VA8JRDQH1M7HN&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1309%252FBG5VA8JRDQH1M7HN%26sid%3Dliteratum%253Aachs%26aulast%3DBoven%26aufirst%3DL.%2BA.%26aulast%3Dvan%2BMeurs%26aufirst%3DM.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DMehta%26aufirst%3DA.%26aulast%3DBoon%26aufirst%3DL.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26aulast%3DLaman%26aufirst%3DJ.%2BD.%26atitle%3DGaucher%2520cells%2520demonstrate%2520a%2520distinct%2520macrophage%2520phenotype%2520and%2520resemble%2520alternatively%2520activated%2520macrophages%26jtitle%3DAm.%2520J.%2520Clin.%2520Pathol.%26date%3D2004%26volume%3D122%26spage%3D359%26epage%3D369%26doi%3D10.1309%2FBG5VA8JRDQH1M7HN" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavala, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esnault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukreja, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertics, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chupp, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarjour, N. N.</span></span> <span> </span><span class="NLM_article-title">Segmental allergen challenge enhances chitinase activity and levels of CCL18 in mild atopic asthma</span>. <i>Clin. Exp. Allergy.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">197</span>, <span class="refDoi">Â DOI: 10.1111/cea.12032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1111%2Fcea.12032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23331560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Chtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2013&pages=187-197&author=M.+L.+Gavalaauthor=E.+A.+B.+Kellyauthor=S.+Esnaultauthor=S.+Kukrejaauthor=M.+D.+Evansauthor=P.+J.+Berticsauthor=G.+L.+Chuppauthor=N.+N.+Jarjour&title=Segmental+allergen+challenge+enhances+chitinase+activity+and+levels+of+CCL18+in+mild+atopic+asthma&doi=10.1111%2Fcea.12032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Segmental allergen challenge enhances chitinase activity and levels of CCL18 in mild atopic asthma</span></div><div class="casAuthors">Gavala, M. L.; Kelly, E. A. B.; Esnault, S.; Kukreja, S.; Evans, M. D.; Bertics, P. J.; Chupp, G. L.; Jarjour, N. N.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Allergy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-197</span>CODEN:
                <span class="NLM_cas:coden">CLEAEN</span>;
        ISSN:<span class="NLM_cas:issn">0954-7894</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Background : Allergic airway inflammation contributes to the airway remodelling that has been linked to increased obstruction and morbidity in asthma.  However, the mechanisms by which allergens contribute to airway remodelling in humans are not fully established.  CCL18, chitotriosidase (CHIT1) and YKL-40 are readily detectable in the lungs and contribute to remodelling in other fibrotic diseases, but their involvement in allergic asthma is unclear.  Objective : We hypothesized that CCL18, YKL-40 and CHIT1 bioactivity are enhanced in allergic asthma subjects after segmental allergen challenge and are related to increased pro-fibrotic and Th2-assocd. mediators in the lungs.  Methods : Levels of CCL18 and YKL-40 protein and chitotriosidase (CHIT1) bioactivity in bronchoalveolar lavage (BAL) fluid, as well as CCL18, YKL-40 and CHIT1 mRNA levels in BAL cells were evaluated in patients with asthma at baseline and 48 h after segmental allergen challenge.  We also examd. the correlation between CCL18 and YKL-40 levels and CHIT1 activity with the levels of other pro-fibrotic factors and chemokines previously shown to be up-regulated after allergen challenge.  Results : Chitotriosidase activity and YKL-40 and CCL18 levels were elevated after segmental allergen challenge and these levels correlated with those of other pro-fibrotic factors, T cell chemokines, and inflammatory cells after allergen challenge.  CCL18 and YKL-40 mRNA levels also increased in BAL cells after allergen challenge.  Conclusions and Clin. Relevance : Our results suggest that CCL18 and YKL-40 levels and CHIT1 activity are enhanced in allergic airway inflammation and thus may contribute to airway remodelling in asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEyjeaYu0j0LVg90H21EOLACvtfcHk0lh3FmgyzSR6Xw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Chtrw%253D&md5=815e295065739fa0ff69742683abd847</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcea.12032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcea.12032%26sid%3Dliteratum%253Aachs%26aulast%3DGavala%26aufirst%3DM.%2BL.%26aulast%3DKelly%26aufirst%3DE.%2BA.%2BB.%26aulast%3DEsnault%26aufirst%3DS.%26aulast%3DKukreja%26aufirst%3DS.%26aulast%3DEvans%26aufirst%3DM.%2BD.%26aulast%3DBertics%26aufirst%3DP.%2BJ.%26aulast%3DChupp%26aufirst%3DG.%2BL.%26aulast%3DJarjour%26aufirst%3DN.%2BN.%26atitle%3DSegmental%2520allergen%2520challenge%2520enhances%2520chitinase%2520activity%2520and%2520levels%2520of%2520CCL18%2520in%2520mild%2520atopic%2520asthma%26jtitle%3DClin.%2520Exp.%2520Allergy.%26date%3D2013%26volume%3D43%26spage%3D187%26epage%3D197%26doi%3D10.1111%2Fcea.12032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artieda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cenarro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GaÃ±Ã¡n, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">JericÃ³, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalvo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casado, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitoria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puzo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PocovÃ­, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civeira, F.</span></span> <span> </span><span class="NLM_article-title">Serum chitotriosidase activity is increased in subjects with atherosclerosis disease</span>. <i>Arterioscler., Thromb., Vasc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1645</span>â <span class="NLM_lpage">1652</span>, <span class="refDoi">Â DOI: 10.1161/01.ATV.0000089329.09061.07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1161%2F01.ATV.0000089329.09061.07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=12893688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntVWrt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=1645-1652&author=M.+Artiedaauthor=A.+Cenarroauthor=A.+Ga%C3%B1%C3%A1nauthor=I.+Jeric%C3%B3author=C.+Gonzalvoauthor=J.+M.+Casadoauthor=I.+Vitoriaauthor=J.+Puzoauthor=M.+Pocov%C3%ADauthor=F.+Civeira&title=Serum+chitotriosidase+activity+is+increased+in+subjects+with+atherosclerosis+disease&doi=10.1161%2F01.ATV.0000089329.09061.07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Serum chitotriosidase activity is increased in subjects With atherosclerosis disease</span></div><div class="casAuthors">Artieda, Marta; Cenarro, Ana; Ganan, Alberto; Jerico, Ivonne; Gonzalvo, Carmen; Casado, Juan M.; Vitoria, Isabel; Puzo, Jose; Pocovi, Miguel; Civeira, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1645-1652</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objective- This study was undertaken to analyze the relation between serum activity of chitotriosidase enzyme, a protein synthesized exclusively by activated macrophages, and atherosclerotic lesion extent in subjects with atherothrombotic stroke (ATS) and in subjects with ischemic heart disease (IHD).  Methods and Results- We assayed the serum chitotriosidase activity and a common chitotriosidase gene polymorphism that causes deficiency in chitotriosidase activity in 3 Spanish populations, ATS (n=153), IHD (n=124), and control (n=148) subjects.  Statistical differences were found in serum chitotriosidase activity between ATS (88.1Â±4.6 nmol/mL Â· h, P<0.0001) and IHD subjects (79.0Â±6.3, P=0.002) vs. control group (70.9Â±5.2).  These obsd. differences were not attributable to a distinct allelic or genotype distribution.  The extension of the atherosclerotic lesion in carotids of ATS subjects was measured by duplex sonog.  Chitotriosidase activities were 66.9Â±9.6, 88.7Â±8.3, and 107.7Â±11.8 for subjects with carotid stenosis â¤30%, 31% to 60%, and >60%, resp.  Statistical differences were obsd. between subjects with major and intermediate stenosis grade compared with subjects with minor stenosis, P=0.005 and P=0.016, resp.  Conclusions- Serum chitotriosidase activity is significantly increased in individuals suffering from atherosclerosis disease and is related to the severity of the atherosclerotic lesion, suggesting a possible role as atherosclerotic extent marker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjfaHPeIda2LVg90H21EOLACvtfcHk0lh3FmgyzSR6Xw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntVWrt7o%253D&md5=f060955236f7d0f0c317c335f521eb6f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1161%2F01.ATV.0000089329.09061.07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.ATV.0000089329.09061.07%26sid%3Dliteratum%253Aachs%26aulast%3DArtieda%26aufirst%3DM.%26aulast%3DCenarro%26aufirst%3DA.%26aulast%3DGa%25C3%25B1%25C3%25A1n%26aufirst%3DA.%26aulast%3DJeric%25C3%25B3%26aufirst%3DI.%26aulast%3DGonzalvo%26aufirst%3DC.%26aulast%3DCasado%26aufirst%3DJ.%2BM.%26aulast%3DVitoria%26aufirst%3DI.%26aulast%3DPuzo%26aufirst%3DJ.%26aulast%3DPocov%25C3%25AD%26aufirst%3DM.%26aulast%3DCiveira%26aufirst%3DF.%26atitle%3DSerum%2520chitotriosidase%2520activity%2520is%2520increased%2520in%2520subjects%2520with%2520atherosclerosis%2520disease%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D1645%26epage%3D1652%26doi%3D10.1161%2F01.ATV.0000089329.09061.07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollak, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alberts, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span> <span> </span><span class="NLM_article-title">Plasma chitotriosidase and CCL18 as surrogate markers for granulomatous macrophages in sarcoidosis</span>. <i>Clin. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">31</span>â <span class="NLM_lpage">36</span>, <span class="refDoi">Â DOI: 10.1016/j.cca.2009.09.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.cca.2009.09.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19808030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGrtbzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=411&publication_year=2010&pages=31-36&author=R.+G.+Bootauthor=C.+E.+M.+Hollakauthor=M.+Verhoekauthor=C.+Albertsauthor=R.+E.+Jonkersauthor=J.+M.+Aerts&title=Plasma+chitotriosidase+and+CCL18+as+surrogate+markers+for+granulomatous+macrophages+in+sarcoidosis&doi=10.1016%2Fj.cca.2009.09.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Plasma chitotriosidase and CCL18 as surrogate markers for granulomatous macrophages in sarcoidosis</span></div><div class="casAuthors">Boot, Rolf G.; Hollak, Carla E. M.; Verhoek, Marri; Alberts, C.; Jonkers, Rene E.; Aerts, Johannes M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinica Chimica Acta</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">31-36</span>CODEN:
                <span class="NLM_cas:coden">CCATAR</span>;
        ISSN:<span class="NLM_cas:issn">0009-8981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accumulation of macrophages in multiple organs is a common feature of sarcoidosis and Gaucher disease.  The vast no. of storage macrophages in Gaucher patients has facilitated the discovery of suitable plasma markers like chitotriosidase and CCL18.  Plasma specimens of patients with sarcoidosis were examd. on chitotriosidase activity and CCL18 protein levels.  Results: Chitotriosidase was markedly increased, being on av. 13.7-fold elevated (range: 1.1-43.3).  The sensitivity of demonstrating sarcoidosis using plasma chitotriosidase values exceeded that using serum angiotensin-converting enzyme values.  A 3.5-fold (range: 1-15) increase in CCL18 was also obsd.  The relative changes in chitotriosidase and CCL18 during the course of disease closely mimicked each other, suggesting an identical cellular source.  In situ hybridization anal. confirmed massive prodn. of chitotriosidase by sarcoid macrophages.  The increase in plasma chitotriosidase correlated with the stage of disease, being highest in active sarcoidosis with extrapulmonary involvement.  Therapy with steroids resulted in clear redn. of plasma chitotriosidase and CCL18 and relapse of disease activity was preceded by increases in these parameters.  Sarcoid macrophages secrete high quantities of chitotriosidase and CCL18.  Detn. of plasma chitotriosidase and CCL18 may be useful to monitor changes in granulomatous macrophages during the course of sarcoidosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbph6FD-qdc7Vg90H21EOLACvtfcHk0lgWRJ54Ze0sQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGrtbzN&md5=515b8f209c26a953d816a6f1ffe58f0a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cca.2009.09.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cca.2009.09.034%26sid%3Dliteratum%253Aachs%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DHollak%26aufirst%3DC.%2BE.%2BM.%26aulast%3DVerhoek%26aufirst%3DM.%26aulast%3DAlberts%26aufirst%3DC.%26aulast%3DJonkers%26aufirst%3DR.%2BE.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26atitle%3DPlasma%2520chitotriosidase%2520and%2520CCL18%2520as%2520surrogate%2520markers%2520for%2520granulomatous%2520macrophages%2520in%2520sarcoidosis%26jtitle%3DClin.%2520Chim.%2520Acta%26date%3D2010%26volume%3D411%26spage%3D31%26epage%3D36%26doi%3D10.1016%2Fj.cca.2009.09.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malaguarnera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Rosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambito, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dellâOmbra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicoletti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malaguarnera, M.</span></span> <span> </span><span class="NLM_article-title">Chitotriosidase gene expression in Kupffer cells from patients with non-alcoholic fatty liver disease</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1313</span>â <span class="NLM_lpage">1320</span>, <span class="refDoi">Â DOI: 10.1136/gut.2005.075697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1136%2Fgut.2005.075697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16825325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVehurfP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2006&pages=1313-1320&author=L.+Malaguarneraauthor=M.+Di+Rosaauthor=A.+M.+Zambitoauthor=N.+dell%E2%80%99Ombraauthor=F.+Nicolettiauthor=M.+Malaguarnera&title=Chitotriosidase+gene+expression+in+Kupffer+cells+from+patients+with+non-alcoholic+fatty+liver+disease&doi=10.1136%2Fgut.2005.075697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Chitotriosidase gene expression in Kupffer cells from patients with non-alcoholic fatty liver disease</span></div><div class="casAuthors">Malaguarnera, L.; Di Rosa, M.; Zambito, A. M.; dell'Ombra, N.; Nicoletti, F.; Malaguarnera, M.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1313-1320</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Non-alc. steatohepatitis (NASH) is a clinicopathol. condition characterized by a necroinflammatory disorder with fatty infiltration of the hepatocytes.  The mol. mechanisms involved in the anomalous behavior of liver cells have only partially been detd.  Human chitotriosidase (Chit) is a chitinolytic enzyme mainly produced by activated macrophages.  The aim of this study was to investigate the expression of the chitinase-like gene in Kupffer cells, to det. how chitotriosidase may be implicated in the progression from uncomplicated steatosis to steatohepatitis with progressive fibrosis.  Seventy-five subjects were studied: 40 with NASH, 20 with simple steatosis, and 15 normal controls.  Kupffer cells obtained from liver biopsies were used to detect CHIT expression, superoxide anion (O2-), lipid peroxidn., and tumor necrosis factor Î± (TNFÎ±) and ferritin levels.  CHIT expression differed markedly in livers from normal controls and in those from patients with simple steatosis or non-alc. steatohepatitis.  A significant correlation between mRNA CHIT and O2-, lipid peroxidn., TNFÎ±, and ferritin levels was obsd. in both NASH and simple steatosis.  Human Kupffer cells in NASH patients overproduce chitotriosidase.  At the highest levels of prodn., this enzyme may play a role in increasing the risk for a poor outcome in steatohepatitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZcSFMhHChYrVg90H21EOLACvtfcHk0lgWRJ54Ze0sQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVehurfP&md5=533961c0afd63a24cf44dd6ddb62e0f5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1136%2Fgut.2005.075697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.2005.075697%26sid%3Dliteratum%253Aachs%26aulast%3DMalaguarnera%26aufirst%3DL.%26aulast%3DDi%2BRosa%26aufirst%3DM.%26aulast%3DZambito%26aufirst%3DA.%2BM.%26aulast%3Ddell%25E2%2580%2599Ombra%26aufirst%3DN.%26aulast%3DNicoletti%26aufirst%3DF.%26aulast%3DMalaguarnera%26aufirst%3DM.%26atitle%3DChitotriosidase%2520gene%2520expression%2520in%2520Kupffer%2520cells%2520from%2520patients%2520with%2520non-alcoholic%2520fatty%2520liver%2520disease%26jtitle%3DGut%26date%3D2006%26volume%3D55%26spage%3D1313%26epage%3D1320%26doi%3D10.1136%2Fgut.2005.075697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommaart, E. F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swart, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghauharali-van der Vlugt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bijl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Place, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M. F. G.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel acidic mammalian chitinase distinct from chitotriosidase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">6770</span>â <span class="NLM_lpage">6778</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M009886200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M009886200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=11085997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSgsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=6770-6778&author=R.+G.+Bootauthor=E.+F.+C.+Blommaartauthor=E.+Swartauthor=K.+Ghauharali-van+der+Vlugtauthor=N.+Bijlauthor=C.+Moeauthor=A.+Placeauthor=J.+M.+F.+G.+Aerts&title=Identification+of+a+novel+acidic+mammalian+chitinase+distinct+from+chitotriosidase&doi=10.1074%2Fjbc.M009886200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel acidic mammalian chitinase distinct from chitotriosidase</span></div><div class="casAuthors">Boot, Rolf G.; Blommaart, Edward F. C.; Swart, Erwin; Ghuharali-Van der Vlugt, Karen; Bijl, Nora; Moe, Cassandra; Place, Allen; Aerts, Johannes M. F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6770-6778</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chitinases are ubiquitous chitin-fragmenting hydrolases.  Recently we discovered the first human chitinase, named chitotriosidase, that is specifically expressed by phagocytes.  We here report the identification, purifn., and subsequent cloning of a second mammalian chitinase.  This enzyme is characterized by an acidic isoelec. point and therefore named acidic mammalian chitinase (AMCase).  In rodents and man the enzyme is relatively abundant in the gastrointestinal tract and is found to a lesser extent in the lung.  Like chitotriosidase, AMCase is synthesized as a 50-kDa protein contg. a 39-kDa N-terminal catalytic domain, a hinge region, and a C-terminal chitin-binding domain.  In contrast to chitotriosidase, the enzyme is extremely acid stable and shows a distinct second pH optimum around pH 2.  AMCase is capable of cleaving artificial chitin-like substrates as well as crab shell chitin and chitin as present in the fungal cell wall.  Our study has revealed the existence of a chitinolytic enzyme in the gastrointestinal tract and lung that may play a role in digestion and/or defense.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK9zza8yUykLVg90H21EOLACvtfcHk0lgWRJ54Ze0sQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslSgsrs%253D&md5=cb1d31ccbf699384de3cb00105259993</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M009886200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M009886200%26sid%3Dliteratum%253Aachs%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DBlommaart%26aufirst%3DE.%2BF.%2BC.%26aulast%3DSwart%26aufirst%3DE.%26aulast%3DGhauharali-van%2Bder%2BVlugt%26aufirst%3DK.%26aulast%3DBijl%26aufirst%3DN.%26aulast%3DMoe%26aufirst%3DC.%26aulast%3DPlace%26aufirst%3DA.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%2BF.%2BG.%26atitle%3DIdentification%2520of%2520a%2520novel%2520acidic%2520mammalian%2520chitinase%2520distinct%2520from%2520chitotriosidase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D6770%26epage%3D6778%26doi%3D10.1074%2Fjbc.M009886200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of acidic mammalian chitinase by RNA interference suppresses ovalbumin-sensitized allergic asthma</span>. <i>Hum. Gene Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1597</span>â <span class="NLM_lpage">1606</span>, <span class="refDoi">Â DOI: 10.1089/hum.2008.092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1089%2Fhum.2008.092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19548841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1WhtL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=1597-1606&author=C.+J.+Yangauthor=Y.+K.+Liuauthor=C.+L.+Liuauthor=C.+N.+Shenauthor=M.+L.+Kuoauthor=C.+C.+Suauthor=C.+P.+Tsengauthor=T.+C.+Yenauthor=C.+R.+Shen&title=Inhibition+of+acidic+mammalian+chitinase+by+RNA+interference+suppresses+ovalbumin-sensitized+allergic+asthma&doi=10.1089%2Fhum.2008.092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of acidic mammalian chitinase by RNA interference suppresses ovalbumin-sensitized allergic asthma</span></div><div class="casAuthors">Yang, Ching-Jen; Liu, Yu-Kuo; Liu, Chao-Lin; Shen, Chia-Ning; Kuo, Ming-Ling; Su, Chien-Chang; Tseng, Ching-Ping; Yen, Tzu-Chen; Shen, Chia-Rui</div><div class="citationInfo"><span class="NLM_cas:title">Human Gene Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1597-1606</span>CODEN:
                <span class="NLM_cas:coden">HGTHE3</span>;
        ISSN:<span class="NLM_cas:issn">1043-0342</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Asthma, a chronic helper T cell type 2-mediated inflammatory disease, is characterized by airway hyperresponsiveness and inflammation.  Growing evidence suggests that increased expression of acidic mammalian chitinase (AMCase) may play a role in the pathogenesis of asthma.  In the present study, we sought to develop an RNA interference approach to suppress allergic asthma in mice through silencing of AMCase expression.  Mice sensitized with ovalbumin (OVA) were intratracheally administered a recombinant adeno-assocd. virus expressing short hairpin RNA (rAAV-shRNA) against AMCase.  In OVA-sensitized mice, the development of allergic symptoms was significantly assocd. with elevated AMCase expression.  After administration of rAAV-shRNA, there was a significant redn. of AMCase expression in the lung and in bronchoalveolar lavage fluid (BALF) cells of sensitized mice.  Sensitized mice receiving rAAV-shRNA showed a significant improvement in allergic symptoms, including airway hyperresponsiveness (AHR), eosinophil infiltration, eotaxin, interleukin-13 secretion in BALF, and serum OVA-specific IgE level.  Our data suggest the hyperexpression of AMCase in asthma can be suppressed by rAAV-mediated shRNA.  Silencing AMCase expression by shRNA may be a promising therapeutic strategy in asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyU9pcjbg2rLVg90H21EOLACvtfcHk0lgga4H4nvC6Fw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1WhtL%252FI&md5=ad60a3b1cdd8089e340a15455e19747c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1089%2Fhum.2008.092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fhum.2008.092%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%2BK.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26aulast%3DShen%26aufirst%3DC.%2BN.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26aulast%3DSu%26aufirst%3DC.%2BC.%26aulast%3DTseng%26aufirst%3DC.%2BP.%26aulast%3DYen%26aufirst%3DT.%2BC.%26aulast%3DShen%26aufirst%3DC.%2BR.%26atitle%3DInhibition%2520of%2520acidic%2520mammalian%2520chitinase%2520by%2520RNA%2520interference%2520suppresses%2520ovalbumin-sensitized%2520allergic%2520asthma%26jtitle%3DHum.%2520Gene%2520Ther.%26date%3D2009%26volume%3D20%26spage%3D1597%26epage%3D1606%26doi%3D10.1089%2Fhum.2008.092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. A.</span></span> <span> </span><span class="NLM_article-title">Acidic mammalian chitinase in asthmatic Th2 inflammation and IL-13 pathway activation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1678</span>â <span class="NLM_lpage">1682</span>, <span class="refDoi">Â DOI: 10.1126/science.1095336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1126%2Fscience.1095336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15192232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1Cgsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1678-1682&author=Z.+Zhuauthor=T.+Zhengauthor=R.+J.+Homerauthor=Y.+K.+Kimauthor=N.+Y.+Chenauthor=L.+Cohnauthor=Q.+Hamidauthor=J.+A.+Elias&title=Acidic+mammalian+chitinase+in+asthmatic+Th2+inflammation+and+IL-13+pathway+activation&doi=10.1126%2Fscience.1095336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic Mammalian Chitinase in Asthmatic Th2 Inflammation and IL-13 Pathway Activation</span></div><div class="casAuthors">Zhu, Zhou; Zheng, Tao; Homer, Robert J.; Kim, Yoon-Keun; Chen, Ning Yuan; Cohn, Lauren; Hamid, Qutayba; Elias, Jack A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5677</span>),
    <span class="NLM_cas:pages">1678-1682</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Chitin is a surface component of parasites and insects, and chitinases are induced in lower life forms during infections with these agents.  Although chitin itself does not exist in humans, chitinases are present in the human genome.  We show here that acidic mammalian chitinase (AMCase) is induced via a T helper-2 (Th2)-specific, interleukin-13 (IL-13)-mediated pathway in epithelial cells and macrophages in an aeroallergen asthma model and expressed in exaggerated quantities in human asthma.  AMCase neutralization ameliorated Th2 inflammation and airway hyperresponsiveness, in part by inhibiting IL-13 pathway activation and chemokine induction.  AMCase may thus be an important mediator of IL-13-induced responses in Th2-dominated disorders such as asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr11ubUlNbNs7Vg90H21EOLACvtfcHk0lgga4H4nvC6Fw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1Cgsr0%253D&md5=4358d813c473ef0d9b788a9dceb0f7f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095336%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DHomer%26aufirst%3DR.%2BJ.%26aulast%3DKim%26aufirst%3DY.%2BK.%26aulast%3DChen%26aufirst%3DN.%2BY.%26aulast%3DCohn%26aufirst%3DL.%26aulast%3DHamid%26aufirst%3DQ.%26aulast%3DElias%26aufirst%3DJ.%2BA.%26atitle%3DAcidic%2520mammalian%2520chitinase%2520in%2520asthmatic%2520Th2%2520inflammation%2520and%2520IL-13%2520pathway%2520activation%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1678%26epage%3D1682%26doi%3D10.1126%2Fscience.1095336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bierbaum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deichmann, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahn, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzmann, A.</span></span> <span> </span><span class="NLM_article-title">Polymorphisms and haplotypes of acid mammalian chitinase are associated with bronchial asthma</span>. <i>Am. J. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">1505</span>â <span class="NLM_lpage">1509</span>, <span class="refDoi">Â DOI: 10.1164/rccm.200506-890OC</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1164%2Frccm.200506-890OC" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16179638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BD2MnlsFehtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2005&pages=1505-1509&author=S.+Bierbaumauthor=R.+Nickelauthor=A.+Kochauthor=S.+Lauauthor=K.+A.+Deichmannauthor=U.+Wahnauthor=A.+Superti-Furgaauthor=A.+Heinzmann&title=Polymorphisms+and+haplotypes+of+acid+mammalian+chitinase+are+associated+with+bronchial+asthma&doi=10.1164%2Frccm.200506-890OC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Polymorphisms and haplotypes of acid mammalian chitinase are associated with bronchial asthma</span></div><div class="casAuthors">Bierbaum Sibylle; Nickel Renate; Koch Anja; Lau Susanne; Deichmann Klaus A; Wahn Ulrich; Superti-Furga Andrea; Heinzmann Andrea</div><div class="citationInfo"><span class="NLM_cas:title">American journal of respiratory and critical care medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1505-9</span>
        ISSN:<span class="NLM_cas:issn">1073-449X</span>.
    </div><div class="casAbstract">RATIONALE:  Chitinases are enzymes that cleave chitin, a polysaccharide contained in many parasites of humans.  Recent studies in mouse models of bronchial asthma have shown that acid mammalian chitinase (AMCase) is involved in the pathophysiology of asthma.  It acts downstream of interleukin-13; inhibition of AMCase leads to an abrogated T-helper cell 2 inflammation, less bronchial hyperreactivity, and fewer eosinophils.  OBJECTIVES:  The aim of this study was to identify common genetic variants in human AMCase and to use them to test for association of AMCase with pediatric asthma.  METHODS:  By sequencing the promotor region and all 11 exons on 30 individuals, 12 high-frequency polymorphisms were identified.  Genotyping of six variants in exons and one promotor polymorphism was performed on the following populations by means of restriction fragment length polymorphisms: 322 children with asthma, 270 randomly chosen adult controls, and a pediatric control population consisting of 565 children who, at age 10 yr, had never wheezed and never been diagnosed having asthma.  MEASUREMENTS AND MAIN RESULTS:  We identified three known and two new amino acid variants.  Analyses by the Armitage's trend test using both control populations showed association of the newly identified variant K17R and the nearby noncoding polymorphism rs3818822 with asthma (p = 0.0031 and p = 0.0003, respectively).  In addition, haplotype analyses revealed strong association of haplotypes with the disease (asthma population vs. pediatric control subjects, p < 10(-10)).  CONCLUSIONS:  This newly described association between AMCase polymorphisms and asthma adds further evidence supporting the involvement of AMCase in the development of asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ99nfQi50cwHWEEzDrymxmfW6udTcc2eaPlLTqjpiPH7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnlsFehtw%253D%253D&md5=d4f7c5393196e655ac38b019b2481b84</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1164%2Frccm.200506-890OC&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1164%252Frccm.200506-890OC%26sid%3Dliteratum%253Aachs%26aulast%3DBierbaum%26aufirst%3DS.%26aulast%3DNickel%26aufirst%3DR.%26aulast%3DKoch%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DS.%26aulast%3DDeichmann%26aufirst%3DK.%2BA.%26aulast%3DWahn%26aufirst%3DU.%26aulast%3DSuperti-Furga%26aufirst%3DA.%26aulast%3DHeinzmann%26aufirst%3DA.%26atitle%3DPolymorphisms%2520and%2520haplotypes%2520of%2520acid%2520mammalian%2520chitinase%2520are%2520associated%2520with%2520bronchial%2520asthma%26jtitle%3DAm.%2520J.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2005%26volume%3D172%26spage%3D1505%26epage%3D1509%26doi%3D10.1164%2Frccm.200506-890OC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. L.</span></span> <span> </span><span class="NLM_article-title">The correlation between chitin and acidic mammalian chitinase in animal models of allergic asthma</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">27371</span>â <span class="NLM_lpage">27377</span>, <span class="refDoi">Â DOI: 10.3390/ijms161126033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fijms161126033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=26580611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2ns7vM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=27371-27377&author=C.+R.+Shenauthor=H.+H.+Juangauthor=H.+S.+Chenauthor=C.+J.+Yangauthor=C.+J.+Wuauthor=M.+H.+Leeauthor=Y.+S.+Hwangauthor=M.+L.+Kuoauthor=Y.+S.+Chenauthor=J.+K.+Chenauthor=C.+L.+Liu&title=The+correlation+between+chitin+and+acidic+mammalian+chitinase+in+animal+models+of+allergic+asthma&doi=10.3390%2Fijms161126033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The correlation between chitin and acidic mammalian chitinase in animal models of allergic asthma</span></div><div class="casAuthors">Shen, Chia-Rui; Juang, Horng-Heng; Chen, Hui-Shan; Yang, Ching-Jen; Wu, Chia-Jen; Lee, Meng-Hua; Hwang, Yih-Shiou; Kuo, Ming-Ling; Chen, Ya-Shan; Chen, Jeen-Kuan; Liu, Chao-Lin</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">27371-27377</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Asthma is the result of chronic inflammation of the airways which subsequently results in airway hyper-responsiveness and airflow obstruction.  It has been shown that an elicited expression of acidic mammalian chitinase (AMCase) may be involved in the pathogenesis of asthma.  Our recent study has demonstrated that the specific suppression of elevated AMCase leads to reduced eosinophilia and Th2-mediated immune responses in an ovalbumin (OVA)-sensitized mouse model of allergic asthma.  In the current study, we show that the elicited expression of AMCase in the lung tissues of both ovalbumin- and Der P2-induced allergic asthma mouse models.  The effects of allergic mediated mols. on AMCase expression were evaluated by utilizing promoter assay in the lung cells.  In fact, the exposure of chitin, a polymd. sugar and the fundamental component of the major allergen mite and several of the inflammatory mediators, showed significant enhancement on AMCase expression.  Such obtained results contribute to the basis of developing a promising therapeutic strategy for asthma by silencing AMCase expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWk_pMAbsLTbVg90H21EOLACvtfcHk0lghXJDB9NsDXA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2ns7vM&md5=2ef2e141501a7e3fa6c5d29f07933ae0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fijms161126033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms161126033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.%2BR.%26aulast%3DJuang%26aufirst%3DH.%2BH.%26aulast%3DChen%26aufirst%3DH.%2BS.%26aulast%3DYang%26aufirst%3DC.%2BJ.%26aulast%3DWu%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DM.%2BH.%26aulast%3DHwang%26aufirst%3DY.%2BS.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BS.%26aulast%3DChen%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26atitle%3DThe%2520correlation%2520between%2520chitin%2520and%2520acidic%2520mammalian%2520chitinase%2520in%2520animal%2520models%2520of%2520allergic%2520asthma%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2015%26volume%3D16%26spage%3D27371%26epage%3D27377%26doi%3D10.3390%2Fijms161126033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maizels, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. E.</span></span> <span> </span><span class="NLM_article-title">Analyzing airway inflammation with chemical biology: dissection of acidic mammalian chitinase function with a selective drug-like inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">569</span>â <span class="NLM_lpage">579</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21609838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslOksL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=569-579&author=T.+E.+Sutherlandauthor=O.+A.+Andersenauthor=M.+Betouauthor=I.+M.+Egglestonauthor=R.+M.+Maizelsauthor=D.+van+Aaltenauthor=J.+E.+Allen&title=Analyzing+airway+inflammation+with+chemical+biology%3A+dissection+of+acidic+mammalian+chitinase+function+with+a+selective+drug-like+inhibitor&doi=10.1016%2Fj.chembiol.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing Airway Inflammation with Chemical Biology: Dissection of Acidic Mammalian Chitinase Function with a Selective Drug-like Inhibitor</span></div><div class="casAuthors">Sutherland, Tara E.; Andersen, Ole A.; Betou, Marie; Eggleston, Ian M.; Maizels, Rick M.; van Aalten, Daan; Allen, Judith E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">569-579</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is produced in the lung during allergic inflammation and asthma, and inhibition of enzymic activity has been considered as a therapeutic strategy.  However, most chitinase inhibitors are nonselective, addnl. inhibiting chitotriosidase activity.  Here, we describe bisdionin F, a competitive AMCase inhibitor with 20-fold selectivity for AMCase over chitotriosidase, designed by utilizing the AMCase crystal structure and dicaffeine scaffold.  In a murine model of allergic inflammation, bisdionin F-treatment attenuated chitinase activity and alleviated the primary features of allergic inflammation including eosinophilia.  However, selective AMCase inhibition by bisdionin F also caused dramatic and unexpected neutrophilia in the lungs.  This class of inhibitor will be a powerful tool to dissect the functions of mammalian chitinases in disease and represents a synthetically accessible scaffold to optimize inhibitory properties in terms of airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlYJOOAc9ksbVg90H21EOLACvtfcHk0lghXJDB9NsDXA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslOksL4%253D&md5=3b082d80515694618da2dd966c25a6f9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DT.%2BE.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DBetou%26aufirst%3DM.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3DMaizels%26aufirst%3DR.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%26aulast%3DAllen%26aufirst%3DJ.%2BE.%26atitle%3DAnalyzing%2520airway%2520inflammation%2520with%2520chemical%2520biology%253A%2520dissection%2520of%2520acidic%2520mammalian%2520chitinase%2520function%2520with%2520a%2520selective%2520drug-like%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D569%26epage%3D579%26doi%3D10.1016%2Fj.chembiol.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguchi-Tsuda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kan-O, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, Y.</span></span> <span> </span><span class="NLM_article-title">Demethylallosamidin, a chitinase inhibitor, suppresses airway inflammation and hyperresponsiveness</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2009.09.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bbrc.2009.09.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19782048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yltLbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=103-108&author=T.+Matsumotoauthor=H.+Inoueauthor=Y.+Satoauthor=Y.+Kitaauthor=T.+Nakanoauthor=N.+Nodaauthor=M.+Eguchi-Tsudaauthor=A.+Moriwakiauthor=K.+Kan-Oauthor=K.+Matsumotoauthor=T.+Shimizuauthor=H.+Nagasawaauthor=S.+Sakudaauthor=Y.+Nakanishi&title=Demethylallosamidin%2C+a+chitinase+inhibitor%2C+suppresses+airway+inflammation+and+hyperresponsiveness&doi=10.1016%2Fj.bbrc.2009.09.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylallosamidin, a chitinase inhibitor, suppresses airway inflammation and hyperresponsiveness</span></div><div class="casAuthors">Matsumoto, Takafumi; Inoue, Hiromasa; Sato, Yosuke; Kita, Yoshihiro; Nakano, Takako; Noda, Naotaka; Eguchi-Tsuda, Miyuki; Moriwaki, Atsushi; Kan-o, Keiko; Matsumoto, Koichiro; Shimizu, Takao; Nagasawa, Hiromichi; Sakuda, Shohei; Nakanishi, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase is upregulated in response to allergen exposure in the lung.  We investigated the effects of chitinase inhibitors, allosamidin (Allo) and demethylallosamidin (Dma), on asthmatic responses.  Mice were subjected to IL-13 instillation into the airways or to ovalbumin sensitization plus exposure with or without treatment of Allo or Dma.  Airway hyperresponsiveness (AHR) and inflammation were evaluated.  Allo and Dma attenuated airway eosinophilia and the upregulation of eotaxin after IL-13 instillation, while Dma, but not Allo, suppressed AHR in IL-13-induced asthma.  Allo or Dma suppressed the elevated chitinase activity in BAL fluids after IL-13 to similar levels.  The bronchoprotective PGE2 levels in BAL fluids were elevated after IL-13 instillation.  Allo, but not Dma, suppressed the overprodn. of PGE2 and the expression of COX-2 and PGE synthase-1 induced by IL-13.  In ovalbumin-induced asthma, Dma suppressed AHR more strongly than Allo.  These findings suggest that Dma attenuates asthmatic responses induced by IL-13 without affecting PGE2 synthesis.  Dma may have potential as therapeutic agents for asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xINGL5MfG7Vg90H21EOLACvtfcHk0lghXJDB9NsDXA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yltLbP&md5=f72224168fbb3362c5200a7c41ab53a6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2009.09.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2009.09.075%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DN.%26aulast%3DEguchi-Tsuda%26aufirst%3DM.%26aulast%3DMoriwaki%26aufirst%3DA.%26aulast%3DKan-O%26aufirst%3DK.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DShimizu%26aufirst%3DT.%26aulast%3DNagasawa%26aufirst%3DH.%26aulast%3DSakuda%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DY.%26atitle%3DDemethylallosamidin%252C%2520a%2520chitinase%2520inhibitor%252C%2520suppresses%2520airway%2520inflammation%2520and%2520hyperresponsiveness%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2009%26volume%3D390%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.bbrc.2009.09.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olczak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czestkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrzejczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzwonek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklepkiewicz, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboubi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheeler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hungerford, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podjarny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitschler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousido-Siah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fadel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, A.</span></span> <span> </span><span class="NLM_article-title">Targeting acidic mammalian chitinase is effective in animal model of asthma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">695</span>â <span class="NLM_lpage">710</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01051</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01051" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCms7nJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=695-710&author=M.+Mazurauthor=J.+Olczakauthor=S.+Olejniczakauthor=R.+Koralewskiauthor=W.+Czestkowskiauthor=A.+Jedrzejczakauthor=J.+Golabauthor=K.+Dzwonekauthor=B.+Dymekauthor=P.+L.+Sklepkiewiczauthor=A.+Zagozdzonauthor=T.+Noonanauthor=K.+Mahboubiauthor=B.+Conwayauthor=R.+Sheelerauthor=P.+Beckettauthor=W.+M.+Hungerfordauthor=A.+Podjarnyauthor=A.+Mitschlerauthor=A.+Cousido-Siahauthor=F.+Fadelauthor=A.+Golebiowski&title=Targeting+acidic+mammalian+chitinase+is+effective+in+animal+model+of+asthma&doi=10.1021%2Facs.jmedchem.7b01051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Acidic Mammalian chitinase Is Effective in Animal Model of Asthma</span></div><div class="casAuthors">Mazur, Marzena; Olczak, Jacek; Olejniczak, Sylwia; Koralewski, Robert; Czestkowski, Wojciech; Jedrzejczak, Anna; Golab, Jakub; Dzwonek, Karolina; Dymek, Barbara; Sklepkiewicz, Piotr L.; Zagozdzon, Agnieszka; Noonan, Tom; Mahboubi, Keyvan; Conway, Bruce; Sheeler, Ryan; Beckett, Paul; Hungerford, William M.; Podjarny, Alberto; Mitschler, Andre; Cousido-Siah, Alexandra; Fadel, Firas; Golebiowski, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">695-710</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article highlights the authors' work toward the identification of a potent, selective, and efficacious acidic mammalian chitinase (AMCase) inhibitor.  Rational design, guided by x-ray anal. of several inhibitors bound to human chitotriosidase (hCHIT1), led to the identification of compd. 7f (1-(3-amino-1H-1,2,4-triazol-5-yl)-N-(4-chlorophenethyl)-N-isobutylpiperidin-4-amine) as a highly potent AMCase inhibitor (IC50 values of 14 and 19 nM against human and mouse enzyme, resp.) and selective (>150Ã against mCHIT1) with very good PK properties.  This compd. dosed once daily at 30 mg/kg po showed significant anti-inflammatory efficacy in HDM-induced allergic airway inflammation in mice, reducing inflammatory cell influx in the BALF and total IgE concn. in plasma, which correlated with decrease of chitinolytic activity.  Therapeutic efficacy of compd. 7f in the clin. relevant aero-allergen-induced acute asthma model in mice provides a rationale for developing AMCase inhibitor for the treatment of asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n2hO0hKVarVg90H21EOLACvtfcHk0lgak6LhiU2HMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCms7nJ&md5=d93c7add7d10a66f03419a6c7fe59bfc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01051%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DM.%26aulast%3DOlczak%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DS.%26aulast%3DKoralewski%26aufirst%3DR.%26aulast%3DCzestkowski%26aufirst%3DW.%26aulast%3DJedrzejczak%26aufirst%3DA.%26aulast%3DGolab%26aufirst%3DJ.%26aulast%3DDzwonek%26aufirst%3DK.%26aulast%3DDymek%26aufirst%3DB.%26aulast%3DSklepkiewicz%26aufirst%3DP.%2BL.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DNoonan%26aufirst%3DT.%26aulast%3DMahboubi%26aufirst%3DK.%26aulast%3DConway%26aufirst%3DB.%26aulast%3DSheeler%26aufirst%3DR.%26aulast%3DBeckett%26aufirst%3DP.%26aulast%3DHungerford%26aufirst%3DW.%2BM.%26aulast%3DPodjarny%26aufirst%3DA.%26aulast%3DMitschler%26aufirst%3DA.%26aulast%3DCousido-Siah%26aufirst%3DA.%26aulast%3DFadel%26aufirst%3DF.%26aulast%3DGolebiowski%26aufirst%3DA.%26atitle%3DTargeting%2520acidic%2520mammalian%2520chitinase%2520is%2520effective%2520in%2520animal%2520model%2520of%2520asthma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D695%26epage%3D710%26doi%3D10.1021%2Facs.jmedchem.7b01051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isogai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, K.</span></span> <span> </span><span class="NLM_article-title">The structure of allosamidin, a novel insect chitinase inhibitor, produced by <i>Streptomyces Sp</i></span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2475</span>â <span class="NLM_lpage">2478</span>, <span class="refDoi">Â DOI: 10.1016/S0040-4039(00)84560-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0040-4039%2800%2984560-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaL28XkvFCiurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1986&pages=2475-2478&author=S.+Sakudaauthor=A.+Isogaiauthor=S.+Matsumotoauthor=A.+Suzukiauthor=K.+Koseki&title=The+structure+of+allosamidin%2C+a+novel+insect+chitinase+inhibitor%2C+produced+by+Streptomyces+Sp&doi=10.1016%2FS0040-4039%2800%2984560-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of allosamidin, a novel insect chitinase inhibitor, produced by Streptomyces sp</span></div><div class="casAuthors">Sakuda, Shohei; Isogai, Akira; Matsumoto, Shogo; Suzuki, Akinori; Koseki, Koshi</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2475-8</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Allosamidin (I), a novel insect chitinase inhibitor, was isolated from the mycelium of Streptomyces sp. and characterized.  I was a basic pseudotrisaccharide consisting of 2-acetamido-2-deoxy-D-allose(N-acetyl-D-allosamine) and a novel aminocyclitol deriv. termed allosamizoline.  I inhibited the chitinases of silkworm (Bombex mori) and retarded the growth of instar larvae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyNDCRkGaOibVg90H21EOLACvtfcHk0liAexxi3Ly5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XkvFCiurs%253D&md5=e5df9e4bf6613356b7061435223b2cfd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2984560-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252984560-8%26sid%3Dliteratum%253Aachs%26aulast%3DSakuda%26aufirst%3DS.%26aulast%3DIsogai%26aufirst%3DA.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DKoseki%26aufirst%3DK.%26atitle%3DThe%2520structure%2520of%2520allosamidin%252C%2520a%2520novel%2520insect%2520chitinase%2520inhibitor%252C%2520produced%2520by%2520Streptomyces%2520Sp%26jtitle%3DTetrahedron%2520Lett.%26date%3D1986%26volume%3D27%26spage%3D2475%26epage%3D2478%26doi%3D10.1016%2FS0040-4039%2800%2984560-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M. F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodkinson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Specificity and affinity of natural product cyclopentapeptide inhibitors against <i>A. fumigatus</i>, human, and bacterial chitinases</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2004.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2004.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15664516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=65-76&author=F.+V.+Raoauthor=D.+R.+Houstonauthor=R.+G.+Bootauthor=J.+M.+F.+G.+Aertsauthor=M.+Hodkinsonauthor=D.+J.+Adamsauthor=K.+Shiomiauthor=S.+Omuraauthor=D.+M.+F.+van+Aalten&title=Specificity+and+affinity+of+natural+product+cyclopentapeptide+inhibitors+against+A.+fumigatus%2C+human%2C+and+bacterial+chitinases&doi=10.1016%2Fj.chembiol.2004.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity and Affinity of Natural Product Cyclopentapeptide Inhibitors against A. fumigatus, Human, and Bacterial Chitinases</span></div><div class="casAuthors">Rao, Francesco V.; Houston, Douglas R.; Boot, Rolf G.; Aerts, Johannes M. F. G.; Hodkinson, Michael; Adams, David J.; Shiomi, Kazuro; Omura, Satoshi; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-76</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Family 18 chitinases play key roles in organisms ranging from bacteria to man.  There is a need for specific, potent inhibitors to probe the function of these chitinases in different organisms.  Such mols. could also provide leads for the development of chemotherapeuticals with fungicidal, insecticidal, or anti-inflammatory potential.  Recently, two natural product peptides, argifin and argadin, have been characterized, which structurally mimic chitinase-chitooligosaccharide interactions and inhibit a bacterial chitinase in the nM-mM range.  Here, we show that these inhibitors also act on human and Aspergillus fumigatus chitinases.  The structures of these enzymes in complex with argifin and argadin, together with mutagenesis, fluorescence, and enzymol., reveal that subtle changes in the binding site dramatically affect affinity and selectivity.  The data show that it may be possible to develop specific chitinase inhibitors based on the argifin/argadin scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRk5ZHk2BMa7Vg90H21EOLACvtfcHk0liAexxi3Ly5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsA%253D%253D&md5=5bec80bc8ff6be5f906ee13d4d8fc4b8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%2BF.%2BG.%26aulast%3DHodkinson%26aufirst%3DM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShiomi%26aufirst%3DK.%26aulast%3DOmura%26aufirst%3DS.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DSpecificity%2520and%2520affinity%2520of%2520natural%2520product%2520cyclopentapeptide%2520inhibitors%2520against%2520A.%2520fumigatus%252C%2520human%252C%2520and%2520bacterial%2520chitinases%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D65%26epage%3D76%26doi%3D10.1016%2Fj.chembiol.2004.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunazuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OÌMura, S.</span></span> <span> </span><span class="NLM_article-title">Recent development of two chitinase inhibitors, Argifin and Argadin, produced by soil microorganisms</span>. <i>Proc. Jpn. Acad. Ser. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">85</span>â <span class="NLM_lpage">102</span>, <span class="refDoi">Â DOI: 10.2183/pjab.86.85</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.2183%2Fpjab.86.85" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFOgtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2010&pages=85-102&author=T.+Hiroseauthor=T.+Sunazukaauthor=S.+O%CC%85Mura&title=Recent+development+of+two+chitinase+inhibitors%2C+Argifin+and+Argadin%2C+produced+by+soil+microorganisms&doi=10.2183%2Fpjab.86.85"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Recent development of two chitinase inhibitors, Argifin and Argadin, produced by soil microorganisms</span></div><div class="casAuthors">Hirose, Tomoyasu; Sunazuka, Toshiaki; Omura, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the Japan Academy, Series B: Physical and Biological Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-102</span>CODEN:
                <span class="NLM_cas:coden">PJABDW</span>;
        ISSN:<span class="NLM_cas:issn">0386-2208</span>.
    
            (<span class="NLM_cas:orgname">Nippon Gakushiin</span>)
        </div><div class="casAbstract">A review.  Chitin, the second most abundant polysaccharide in nature, occurs in fungi, some algae and many invertebrates, including insects.  Thus, chitin synthesis and degrdn. could represent specific targets for fungicides and insecticides.  Chitinases hydrolyze chitin into oligomers of N-acetyl-D-glucosamine at key points in the life cycles of organisms, consequently, chitinase inhibitors have become subject of increasing interest.  This review covers the development of two chitinase inhibitors of natural origin, Argifin and Argadin, isolated from the cultured broth of microorganisms in our lab.  In particular, the practical total synthesis of these natural products, the synthesis of lead compds. via computer-aided rational mol. design, and discovery methods that generate only highly-active compds. using a kinetic target(chitinase)-guided synthesis approach (termed in situ click chem.) are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-AuwyNwzeYbVg90H21EOLACvtfcHk0liAexxi3Ly5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFOgtr8%253D&md5=1b74ff18415c88080adcbb027030b51a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2183%2Fpjab.86.85&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2183%252Fpjab.86.85%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DT.%26aulast%3DSunazuka%26aufirst%3DT.%26aulast%3DO%25CC%2585Mura%26aufirst%3DS.%26atitle%3DRecent%2520development%2520of%2520two%2520chitinase%2520inhibitors%252C%2520Argifin%2520and%2520Argadin%252C%2520produced%2520by%2520soil%2520microorganisms%26jtitle%3DProc.%2520Jpn.%2520Acad.%2520Ser.%2520B%26date%3D2010%26volume%3D86%26spage%3D85%26epage%3D102%26doi%3D10.2183%2Fpjab.86.85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span> <span> </span><span class="NLM_article-title">Fully deacetylated chitooligosaccharides act as efficient glycoside hydrolase family 18 chitinase inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">17932</span>â <span class="NLM_lpage">17940</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M114.564534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M114.564534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=24828498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaisrvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=17932-17940&author=L.+Chenauthor=Y.+Zhouauthor=M.+Quauthor=Y.+Zhaoauthor=Q.+Yang&title=Fully+deacetylated+chitooligosaccharides+act+as+efficient+glycoside+hydrolase+family+18+chitinase+inhibitors&doi=10.1074%2Fjbc.M114.564534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fully deacetylated chitooligosaccharides act as efficient glycoside hydrolase family 18 chitinase inhibitors</span></div><div class="casAuthors">Chen, Lei; Zhou, Yong; Qu, Mingbo; Zhao, Yong; Yang, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">17932-17940</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors against chitinases have potential applications as pesticides, fungicides, and antiasthmatics.  Here, we report that a series of fully deacetylated chitooligosaccharides (GlcN)2-7 can act as inhibitors against the insect chitinase OfChtI, the human chitinase HsCht, and the bacterial chitinases SmChiA and SmChiB with IC50 values at micromolar to millimolar levels.  The injection of mixed (GlcN)2-7 into the fifth instar larvae of the insect Ostrinia furnacalis resulted in 85% of the larvae being arrested at the larval stage and death after 10 days, also suggesting that (GlcN)2-7 might inhibit OfChtI in vivo.  Crystal structures of the catalytic domain of OfChtI (OfChtI-CAD) complexed with (GlcN)5,6 were obtained at resolns. of 2.0 Ã.  These structures, together with mutagenesis and thermodn. anal., suggested that the inhibition was strongly related to the interaction between the -1 GlcN residue of the inhibitor and the catalytic Glu148 of the enzyme.  Structure-based comparison showed that the fully deacetylated chitooligosaccharides mimic the substrate chitooligosaccharides by binding to the active cleft.  This work first reports the inhibitory activity and proposed inhibitory mechanism of fully deacetylated chitooligosaccharides.  Because the fully deacetylated chitooligosaccharides can be easily derived from chitin, one of the most abundant materials in nature, this work also provides a platform for developing eco-friendly inhibitors against chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpogLcsC9w3CbVg90H21EOLACvtfcHk0liAexxi3Ly5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaisrvL&md5=6bf68fe4e174f0b0e1fa59545c67d76e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.564534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.564534%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DQ.%26atitle%3DFully%2520deacetylated%2520chitooligosaccharides%2520act%2520as%2520efficient%2520glycoside%2520hydrolase%2520family%252018%2520chitinase%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D17932%26epage%3D17940%26doi%3D10.1074%2Fjbc.M114.564534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Acetazolamide-based fungal chitinase inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">8334</span>â <span class="NLM_lpage">8340</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2010.09.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bmc.2010.09.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21044846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8334-8340&author=A.+W.+Sch%C3%BCttelkopfauthor=L.+Grosauthor=D.+E.+Blairauthor=J.+A.+Frearsonauthor=D.+M.+F.+van+Aaltenauthor=I.+H.+Gilbert&title=Acetazolamide-based+fungal+chitinase+inhibitors&doi=10.1016%2Fj.bmc.2010.09.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Acetazolamide-based fungal chitinase inhibitors</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; Gros, Ludovic; Blair, David E.; Frearson, Julie A.; van Aalten, Daan M. F.; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8334-8340</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chitin is an essential structural component of the fungal cell wall.  Chitinases are thought to be important for fungal cell wall remodelling, and inhibition of these enzymes has been proposed as a potential strategy for development of novel anti-fungals.  The fungal pathogen Aspergillus fumigatus possesses two distinct multi-gene chitinase families.  Here we explore acetazolamide as a chem. scaffold for the inhibition of an A. fumigatus plant-type' chitinase.  A co-crystal structure of AfChiA1 with acetazolamide was used to guide synthesis and screening of acetazolamide analogs that yielded SAR in agreement with these structural data.  Although acetazolamide and its analogs are weak inhibitors of the enzyme, they have a high ligand efficiency and as such are interesting leads for future inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0YAez21-yarVg90H21EOLACvtfcHk0lgM1cliPNokQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurjE&md5=6a8ffe1abf068b24fdabef52a72b6a13</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.09.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.09.062%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DGros%26aufirst%3DL.%26aulast%3DBlair%26aufirst%3DD.%2BE.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DAcetazolamide-based%2520fungal%2520chitinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8334%26epage%3D8340%26doi%3D10.1016%2Fj.bmc.2010.09.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Bisdionin C-A rationally designed, submicromolar inhibitor of family 18 chitinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">428</span>â <span class="NLM_lpage">432</span>, <span class="refDoi">Â DOI: 10.1021/ml200008b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200008b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=428-432&author=A.+W.+Sch%C3%BCttelkopfauthor=O.+A.+Andersenauthor=F.+V.+Raoauthor=M.+Allwoodauthor=C.+L.+Rushauthor=I.+M.+Egglestonauthor=D.+M.+F.+van+Aalten&title=Bisdionin+C-A+rationally+designed%2C+submicromolar+inhibitor+of+family+18+chitinases&doi=10.1021%2Fml200008b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml200008b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200008b%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAllwood%26aufirst%3DM.%26aulast%3DRush%26aufirst%3DC.%2BL.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DBisdionin%2520C-A%2520rationally%2520designed%252C%2520submicromolar%2520inhibitor%2520of%2520family%252018%2520chitinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D428%26epage%3D432%26doi%3D10.1021%2Fml200008b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vora, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Methylxanthine drugs are chitinase inhibitors: investigation of inhibition and binding modes</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">973</span>â <span class="NLM_lpage">980</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2005.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2005.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16183021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=973-980&author=F.+V.+Raoauthor=O.+A.+Andersenauthor=K.+A.+Voraauthor=J.+A.+DeMartinoauthor=D.+M.+F.+van+Aalten&title=Methylxanthine+drugs+are+chitinase+inhibitors%3A+investigation+of+inhibition+and+binding+modes&doi=10.1016%2Fj.chembiol.2005.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Methylxanthine Drugs Are Chitinase Inhibitors: Investigation of Inhibition and Binding Modes</span></div><div class="casAuthors">Rao, Francesco V.; Andersen, Ole A.; Vora, Kalpit A.; DeMartino, Julie A.; Van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Family 18 chitinases play key roles in a range of pathogenic organisms and are overexpressed in the asthmatic lung.  By screening a library of marketed drug mols., we have identified methylxanthine derivs. as possible inhibitor leads.  These derivs., theophylline, caffeine, and pentoxifylline, are used therapeutically as antiinflammatory agents, with pleiotropic mechanisms of action.  Here it is shown that they are also competitive inhibitors against a fungal family 18 chitinase, with pentoxifylline being the most potent (Ki of 37 Î¼M).  Crystallog. anal. of chitinase-inhibitor complexes revealed specific interactions with the active site, mimicking the reaction intermediate analog, allosamidin.  Mutagenesis identified the key active site residues, conserved in mammalian chitinases, which contribute to inhibitor affinity.  Enzyme assays also revealed that these methylxanthines are active against human chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKoH7MRm3ibVg90H21EOLACvtfcHk0lgM1cliPNokQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7%252FM&md5=1d26875c45626540ccac6683f8b0cd41</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DVora%26aufirst%3DK.%2BA.%26aulast%3DDeMartino%26aufirst%3DJ.%2BA.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DMethylxanthine%2520drugs%2520are%2520chitinase%2520inhibitors%253A%2520investigation%2520of%2520inhibition%2520and%2520binding%2520modes%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D973%26epage%3D980%26doi%3D10.1016%2Fj.chembiol.2005.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Screening-based discovery and structural dissection of a novel family 18 chitinase inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">27278</span>â <span class="NLM_lpage">27285</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M604048200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M604048200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16844689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlKmtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27278-27285&author=A.+W.+Sch%C3%BCttelkopfauthor=O.+A.+Andersenauthor=F.+V.+Raoauthor=M.+Allwoodauthor=C.+Lloydauthor=I.+M.+Egglestonauthor=D.+M.+F.+van+Aalten&title=Screening-based+discovery+and+structural+dissection+of+a+novel+family+18+chitinase+inhibitor&doi=10.1074%2Fjbc.M604048200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Screening-based Discovery and Structural Dissection of a Novel Family 18 Chitinase Inhibitor</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; Andersen, Ole A.; Rao, Francesco V.; Allwood, Matthew; Lloyd, Clare; Eggleston, Ian M.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">27278-27285</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Family 18 chitinases play key roles in the life cycles of a variety of organisms ranging from bacteria to man.  Very recently it has been shown that one of the mammalian chitinases is highly overexpressed in the asthmatic lung and contributes to the pathogenic process through recruitment of inflammatory cells.  Although several potent natural product chitinase inhibitors have been identified, their chemotherapeutic potential or their use as cell biol. tools is limited due to their size, complex chem., and limited availability.  We describe a virtual screening-based approach to identification of a novel, purine-based, chitinase inhibitor.  This inhibitor acts in the low micromolar (Ki = 2.8Â±0.2 Î¼M) range in a competitive mode.  Dissection of the binding mode by X-ray crystallog. reveals that the compd., which consists of two linked caffeine moieties, binds in the active site through extensive and not previously obsd. stacking interactions with conserved, solvent exposed tryptophans.  Such exposed aroms. are also present in the structures of many other carbohydrate processing enzymes.  The compd. exhibits favorable chem. properties and is likely to be useful as a general scaffold for development of pan-family 18 chitinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfEHLpyiMotrVg90H21EOLACvtfcHk0lhdSuOtocx34g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlKmtrs%253D&md5=66db5ed0f11883ce5d74c77d6830669b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M604048200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M604048200%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAllwood%26aufirst%3DM.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DScreening-based%2520discovery%2520and%2520structural%2520dissection%2520of%2520a%2520novel%2520family%252018%2520chitinase%2520inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27278%26epage%3D27285%26doi%3D10.1074%2Fjbc.M604048200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabudravu, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eijsink, V. G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gooday, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Synstad, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Psammaplin A, a chitinase inhibitor isolated from the Fijian marine sponge <i>Aplysinella rhax</i></span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1128</span>, <span class="refDoi">Â DOI: 10.1016/S0968-0896(01)00372-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0968-0896%2801%2900372-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=11836123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVKht70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1123-1128&author=J.+N.+Tabudravuauthor=V.+G.+H.+Eijsinkauthor=G.+W.+Goodayauthor=M.+Jasparsauthor=D.+Komanderauthor=M.+Leggauthor=B.+Synstadauthor=D.+M.+F.+van+Aalten&title=Psammaplin+A%2C+a+chitinase+inhibitor+isolated+from+the+Fijian+marine+sponge+Aplysinella+rhax&doi=10.1016%2FS0968-0896%2801%2900372-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Psammaplin A, a chitinase inhibitor isolated from the Fijian marine sponge Aplysinella rhax</span></div><div class="casAuthors">Tabudravu, J. N.; Eijsink, V. G. H.; Gooday, G. W.; Jaspars, M.; Komander, D.; Legg, M.; Synstad, B.; van Aalten, D. M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1123-1128</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Several brominated tyrosine derived compds., psammaplins A, K and L as well as bisaprasin were isolated from the Fijian marine sponge Aplysinella rhax during a bioassay guided isolation protocol.  Their structures were detd. using NMR and MS techniques.  Psammaplin A was found to moderately inhibit chitinase B from Serratia marcescens, the mode of inhibition being non-competitive.  Crystallog. studies suggest that a disordered psammaplin A mol. is bound near the active site.  Interestingly, psammaplin A was a potent antifungal agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKbW-239omDrVg90H21EOLACvtfcHk0lhdSuOtocx34g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVKht70%253D&md5=64a9babaf17409abed456014eca0f965</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2801%2900372-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252801%252900372-8%26sid%3Dliteratum%253Aachs%26aulast%3DTabudravu%26aufirst%3DJ.%2BN.%26aulast%3DEijsink%26aufirst%3DV.%2BG.%2BH.%26aulast%3DGooday%26aufirst%3DG.%2BW.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DLegg%26aufirst%3DM.%26aulast%3DSynstad%26aufirst%3DB.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPsammaplin%2520A%252C%2520a%2520chitinase%2520inhibitor%2520isolated%2520from%2520the%2520Fijian%2520marine%2520sponge%2520Aplysinella%2520rhax%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2002%26volume%3D10%26spage%3D1123%26epage%3D1128%26doi%3D10.1016%2FS0968-0896%2801%2900372-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shizuri, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo</span></span> <span> </span><span class="NLM_article-title">Styloguanidines, new chitinase inhibitors from the marine sponge <i>Stylotella aurantium</i></span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2133</span>â <span class="NLM_lpage">2136</span>, <span class="refDoi">Â DOI: 10.1016/0040-4039(95)00194-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2F0040-4039%2895%2900194-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK2MXkslOjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=2133-2136&author=T.+Katoauthor=Y.+Shizuriauthor=H.+Izumidaauthor=A.+Yokoyamaauthor=+Endo&title=Styloguanidines%2C+new+chitinase+inhibitors+from+the+marine+sponge+Stylotella+aurantium&doi=10.1016%2F0040-4039%2895%2900194-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Styloguanidines, new chitinase inhibitors from the marine sponge Stylotella aurantium</span></div><div class="casAuthors">Kato, Taketoshi; Shizuri, Yoshikazu; Izumida, Hitoshi; Yokoyama, Akihiro; Endo, Mamoru</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2133-6</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Three potent new chitinase inhibitors, styloguanidine (I), 3-bromostyloguanidine, and 2,3-dibromostyloguanidine were isolated from the sponge collected from the Yap Sea.  Their structures were detd. by spectral means.  These were a new type of inhibitor from natural sources and were proved to inhibit the molting of cyprid larvae of barnacles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorawLAF1tPNrVg90H21EOLACvtfcHk0li0UTsWC23tQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslOjsr4%253D&md5=a97fef3517b4255d828866bc2d2766bc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2900194-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252900194-H%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DT.%26aulast%3DShizuri%26aufirst%3DY.%26aulast%3DIzumida%26aufirst%3DH.%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DEndo%26atitle%3DStyloguanidines%252C%2520new%2520chitinase%2520inhibitors%2520from%2520the%2520marine%2520sponge%2520Stylotella%2520aurantium%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26spage%3D2133%26epage%3D2136%26doi%3D10.1016%2F0040-4039%2895%2900194-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchie, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankovich, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuff, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span> <span> </span><span class="NLM_article-title">Functional comparison of recombinant acidic mammalian chitinase with enzyme from murine bronchoalveolar lavage</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">55</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1016/j.pep.2010.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.pep.2010.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20826216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKrsrjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2011&pages=55-62&author=E.+R.+Goedkenauthor=R.+F.+O%E2%80%99Brienauthor=T.+Xiangauthor=D.+L.+Banachauthor=S.+C.+Marchieauthor=E.+H.+Barlowauthor=S.+Hubbardauthor=J.+A.+Mankovichauthor=J.+Jiangauthor=P.+L.+Richardsonauthor=C.+A.+Cuffauthor=A.+D.+Cherniack&title=Functional+comparison+of+recombinant+acidic+mammalian+chitinase+with+enzyme+from+murine+bronchoalveolar+lavage&doi=10.1016%2Fj.pep.2010.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional comparison of recombinant acidic mammalian chitinase with enzyme from murine bronchoalveolar lavage</span></div><div class="casAuthors">Goedken, Eric R.; O'Brien, Robert F.; Xiang, Tao; Banach, David L.; Marchie, Susan C.; Barlow, Eve H.; Hubbard, Stacy; Mankovich, John A.; Jiang, Jianjun; Richardson, Paul L.; Cuff, Carolyn A.; Cherniack, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression & Purification</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is an enzyme that selectively degrades the biopolymer chitin.  Several chitinase enzymes are utilized by mammals to hydrolyze chitin encountered by inhalation and ingestion.  AMCase is distinct from other mammalian chitinases as its activity is retained in strongly acidic conditions (pH < 2.0).  AMCase expression is induced by antigen-induced mouse models of allergic lung inflammation.  This protein has also been implicated in the pathogenesis of asthma although its precise role is poorly defined.  A novel way to express and purify active murine AMCase is described.  This material retains properties obsd. in mouse bronchoalveolar lavage (BAL) fluid with regard to pH preference of activity and its inhibition by cyclic peptide inhibitors argifin and argadin.  It was found that chitinase in BAL from both antigen-challenged and control animals have similar properties in this regard.  This strongly supports the notion the same enzyme (AMCase) gives rise to chitinase activity in both challenged and unchallenged animals.  Expression of active human AMCase is also described.  The methods described in this paper provide a reliable source of recombinant AMCase that can be utilized to expand understanding of AMCase's role in regulating allergic inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonWeoser5mN7Vg90H21EOLACvtfcHk0li0UTsWC23tQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKrsrjN&md5=93103913f8a5d169445b226e49f0034b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2010.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2010.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DR.%2BF.%26aulast%3DXiang%26aufirst%3DT.%26aulast%3DBanach%26aufirst%3DD.%2BL.%26aulast%3DMarchie%26aufirst%3DS.%2BC.%26aulast%3DBarlow%26aufirst%3DE.%2BH.%26aulast%3DHubbard%26aufirst%3DS.%26aulast%3DMankovich%26aufirst%3DJ.%2BA.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DCuff%26aufirst%3DC.%2BA.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26atitle%3DFunctional%2520comparison%2520of%2520recombinant%2520acidic%2520mammalian%2520chitinase%2520with%2520enzyme%2520from%2520murine%2520bronchoalveolar%2520lavage%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2011%26volume%3D75%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.pep.2010.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olland, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursavich, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeClercq, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph-McCarthy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingboe, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, D. H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of acidic mammalian chitinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6122</span>â <span class="NLM_lpage">6128</span>, <span class="refDoi">Â DOI: 10.1021/jm100533p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100533p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6122-6128&author=D.+C.+Coleauthor=A.+M.+Ollandauthor=J.+Jacobauthor=J.+Brooksauthor=M.+G.+Bursavichauthor=R.+Czerwinskiauthor=C.+DeClercqauthor=M.+Johnsonauthor=D.+Joseph-McCarthyauthor=J.+W.+Ellingboeauthor=L.+Linauthor=P.+Nowakauthor=E.+Presmanauthor=J.+Strandauthor=A.+Tamauthor=C.+M.+M.+Williamsauthor=S.+Yaoauthor=D.+H.+H.+Tsaoauthor=L.+J.+Fitz&title=Identification+and+characterization+of+acidic+mammalian+chitinase+inhibitors&doi=10.1021%2Fjm100533p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Acidic Mammalian Chitinase Inhibitors</span></div><div class="casAuthors">Cole, Derek C.; Olland, Andrea M.; Jacob, Jaison; Brooks, Jon; Bursavich, Matthew G.; Czerwinski, Robert; De Clercq, Charlene; Johnson, Mark; Joseph-McCarthy, Diane; Ellingboe, John W.; Lin, Laura; Nowak, Pawel; Presman, Ella; Strand, James; Tam, Amy; Williams, Cara M. M.; Yao, Shihua; Tsao, Desiree H. H.; Fitz, Lori J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6122-6128</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is a member of the glycosyl hydrolase 18 family (EC 3.2.1.14) that has been implicated in the pathophysiol. of allergic airway disease such as asthma.  Small mol. inhibitors of AMCase were identified using a combination of high-throughput screening, fragment screening, and virtual screening techniques and characterized by enzyme inhibition and NMR and Biacore binding expts.  X-ray structures of the inhibitors in complex with AMCase revealed that the larger more potent HTS hits, e.g. 5-(4-(2-(4-bromophenoxy)ethyl)piperazine-1-yl)-1H-1,2,4-triazol-3-amine 1, spanned from the active site pocket to a hydrophobic pocket.  Smaller fragments identified by FBS occupy both these pockets independently and suggest potential strategies for linking fragments.  Compd. 1 is a 200 nM AMCase inhibitor which reduced AMCase enzymic activity in the bronchoalveolar lavage fluid in allergen-challenged mice after oral dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq3-AOdwYs2LVg90H21EOLACvtfcHk0li0UTsWC23tQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChtL8%253D&md5=47d5010b73151d1ad4f641c47c3fb100</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100533p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100533p%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BC.%26aulast%3DOlland%26aufirst%3DA.%2BM.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DBrooks%26aufirst%3DJ.%26aulast%3DBursavich%26aufirst%3DM.%2BG.%26aulast%3DCzerwinski%26aufirst%3DR.%26aulast%3DDeClercq%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DEllingboe%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNowak%26aufirst%3DP.%26aulast%3DPresman%26aufirst%3DE.%26aulast%3DStrand%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DC.%2BM.%2BM.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%2BH.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520acidic%2520mammalian%2520chitinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6122%26epage%3D6128%26doi%3D10.1021%2Fjm100533p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoszewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andryianau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matyszewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pluta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefaniak, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurkiewicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czestkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedziejko, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruza, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzwonek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olczak, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective, orally bioavailable inhibitor of mouse chitotriosidase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">310</span>â <span class="NLM_lpage">314</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2017.12.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bmcl.2017.12.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=29292229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlSrsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=310-314&author=M.+Mazurauthor=A.+Bartoszewiczauthor=B.+Dymekauthor=M.+Salamonauthor=G.+Andryianauauthor=M.+Kowalskiauthor=S.+Olejniczakauthor=K.+Matyszewskiauthor=E.+Plutaauthor=B.+Borekauthor=F.+Stefaniakauthor=A.+Zagozdzonauthor=M.+Mazurkiewiczauthor=R.+Koralewskiauthor=W.+Czestkowskiauthor=M.+Piotrowiczauthor=P.+Niedziejkoauthor=M.+M.+Gruzaauthor=K.+Dzwonekauthor=A.+Golebiowskiauthor=J.+Golabauthor=J.+Olczak&title=Discovery+of+selective%2C+orally+bioavailable+inhibitor+of+mouse+chitotriosidase&doi=10.1016%2Fj.bmcl.2017.12.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective, orally bioavailable inhibitor of mouse chitotriosidase</span></div><div class="casAuthors">Mazur, Marzena; Bartoszewicz, Agnieszka; Dymek, Barbara; Salamon, Magdalena; Andryianau, Gleb; Kowalski, Michal; Olejniczak, Sylwia; Matyszewski, Krzysztof; Pluta, Elzbieta; Borek, Bartlomiej; Stefaniak, Filip; Zagozdzon, Agnieszka; Mazurkiewicz, Marcin; Koralewski, Robert; Czestkowski, Wojciech; Piotrowicz, Michal; Niedziejko, Piotr; Gruza, Mariusz M.; Dzwonek, Karolina; Golebiowski, Adam; Golab, Jakub; Olczak, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-314</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article describes the authors' work towards the identification of a potent and selective inhibitor of mouse chitotriosidase (mCHIT1).  A series of small mol. inhibitors of mCHIT1 and mAMCase have been developed from early lead compd. (I).  Examn. of synthesized analogs led to discovery of several novel highly potent compds.  Among them compd. 9 (OAT-2068) displays a remarkable 143-fold mCHIT1 vs. mAMCase selectivity.  To explain the obsd. SAR mol. docking expts. were performed, which were in line with the exptl. data from the enzymic assays.  Inhibitor 9 (OAT-2068) was found to have an excellent pharmacokinetic profile.  This, together with high activity and selectivity, makes the compd. an ideal and unique tool for studying the role of CHIT1 in biol. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot976-E0U9pLVg90H21EOLACvtfcHk0lhiw3Ea_aq74w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlSrsA%253D%253D&md5=420c689a31b501c151a1915ca37ab16b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.12.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.12.047%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DM.%26aulast%3DBartoszewicz%26aufirst%3DA.%26aulast%3DDymek%26aufirst%3DB.%26aulast%3DSalamon%26aufirst%3DM.%26aulast%3DAndryianau%26aufirst%3DG.%26aulast%3DKowalski%26aufirst%3DM.%26aulast%3DOlejniczak%26aufirst%3DS.%26aulast%3DMatyszewski%26aufirst%3DK.%26aulast%3DPluta%26aufirst%3DE.%26aulast%3DBorek%26aufirst%3DB.%26aulast%3DStefaniak%26aufirst%3DF.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DMazurkiewicz%26aufirst%3DM.%26aulast%3DKoralewski%26aufirst%3DR.%26aulast%3DCzestkowski%26aufirst%3DW.%26aulast%3DPiotrowicz%26aufirst%3DM.%26aulast%3DNiedziejko%26aufirst%3DP.%26aulast%3DGruza%26aufirst%3DM.%2BM.%26aulast%3DDzwonek%26aufirst%3DK.%26aulast%3DGolebiowski%26aufirst%3DA.%26aulast%3DGolab%26aufirst%3DJ.%26aulast%3DOlczak%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%252C%2520orally%2520bioavailable%2520inhibitor%2520of%2520mouse%2520chitotriosidase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D310%26epage%3D314%26doi%3D10.1016%2Fj.bmcl.2017.12.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CibriÃ¡n-Uhalte, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MagariÃ±os, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL database in 2017</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D945</span>â <span class="NLM_lpage">D954</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkw1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1093%2Fnar%2Fgkw1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=27899562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D945-D954&author=A.+Gaultonauthor=A.+Herseyauthor=M.+Nowotkaauthor=A.+P.+Bentoauthor=J.+Chambersauthor=D.+Mendezauthor=P.+Mutowoauthor=F.+Atkinsonauthor=L.+J.+Bellisauthor=E.+Cibri%C3%A1n-Uhalteauthor=M.+Daviesauthor=N.+Dedmanauthor=A.+Karlssonauthor=M.+P.+Magari%C3%B1osauthor=J.+P.+Overingtonauthor=G.+Papadatosauthor=I.+Smitauthor=A.+R.+Leach&title=The+ChEMBL+database+in+2017&doi=10.1093%2Fnar%2Fgkw1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL database in 2017</span></div><div class="casAuthors">Gaulton, Anna; Hersey, Anne; Nowotka, Micha-l; Bento, A. Patricia; Chambers, Jon; Mendez, David; Mutowo, Prudence; Atkinson, Francis; Bellis, Louisa J.; Cibrian-Uhalte, Elena; Davies, Mark; Dedman, Nathan; Karlsson, Anneli; Magarinos, Maria Paula; Overington, John P.; Papadatos, George; Smit, Ines; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D945-D954</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 and 2014 Nucleic Acids Research Database Issues.  Since then, alongside the continued extn. of data from the medicinal chem. literature, new sources of bioactivity data have also been added to the database.  These include: deposited data sets from neglected disease screening; crop protection data; drug metab. and disposition data and bioactivity data from patents.  A no. of improvements and new features have also been incorporated.  These include the annotation of assays and targets using ontologies, the inclusion of targets and indications for clin. candidates, addn. of metabolic pathways for drugs and calcn. of structural alerts.  The ChEMBL data can be accessed via a web-interface, RDF distribution, data downloads and RESTful web-services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB0IuNShSHIrVg90H21EOLACvtfcHk0lhiw3Ea_aq74w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D&md5=1dcbb641f8c2d11410663f97505a4911</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1074%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DCibri%25C3%25A1n-Uhalte%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DI.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DThe%2520ChEMBL%2520database%2520in%25202017%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD945%26epage%3DD954%26doi%3D10.1093%2Fnar%2Fgkw1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>, <span class="refDoi">Â DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, Î±2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lj8Fgf3sCoofQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>â <span class="NLM_lpage">2740</span>, <span class="refDoi">Â DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lj8Fgf3sCoofQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruns, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span> <span> </span><span class="NLM_article-title">Rules for identifying potentially reactive or promiscuous compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9763</span>â <span class="NLM_lpage">9772</span>, <span class="refDoi">Â DOI: 10.1021/jm301008n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301008n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSgs7bP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9763-9772&author=R.+F.+Brunsauthor=I.+A.+Watson&title=Rules+for+identifying+potentially+reactive+or+promiscuous+compounds&doi=10.1021%2Fjm301008n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Rules for Identifying Potentially Reactive or Promiscuous Compounds</span></div><div class="casAuthors">Bruns, Robert F.; Watson, Ian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9763-9772</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article describes a set of 275 rules, developed over an 18-yr period, used to identify compds. that may interfere with biol. assays, allowing their removal from screening sets.  Reasons for rejection include reactivity (e.g., acyl halides), interference with assay measurements (fluorescence, absorbance, quenching), activities that damage proteins (oxidizers, detergents), instability (e.g., latent aldehydes), and lack of druggability (e.g., compds. lacking both oxygen and nitrogen).  The structural queries were profiled for frequency of occurrence in druglike and nondruglike compd. sets and were extensively reviewed by a panel of experienced medicinal chemists.  As a means of profiling the rules and as a filter in its own right, an index of biol. promiscuity was developed.  The 584 gene targets with screening data at Lilly were assigned to 17 subfamilies, and the no. of subfamilies at which a compd. was active was used as a promiscuity index.  For certain compds., promiscuous activity disappeared after sample repurifn., indicating interference from occult contaminants.  Because this type of interference is not amenable to substructure search, a "nuisance list" was developed to flag interfering compds. that passed the substructure rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNvXr0QeH2l7Vg90H21EOLACvtfcHk0lj8Fgf3sCoofQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSgs7bP&md5=3c37f5513b3859e0180686e99d83d30c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm301008n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301008n%26sid%3Dliteratum%253Aachs%26aulast%3DBruns%26aufirst%3DR.%2BF.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26atitle%3DRules%2520for%2520identifying%2520potentially%2520reactive%2520or%2520promiscuous%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9763%26epage%3D9772%26doi%3D10.1021%2Fjm301008n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâRourke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremb, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duggan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sioud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbatia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emwas, A.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voolstra, C.</span></span> <span> </span><span class="NLM_article-title">Identification of a 3-Alkylpyridinium compound from the Red Sea sponge <i>Amphimedon chloros</i> with in vitro inhibitory activity against the West Nile virus NS3 protease</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1420</span>â <span class="NLM_lpage">3049</span>, <span class="refDoi">Â DOI: 10.3390/molecules23061472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fmolecules23061472" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1420-3049&author=A.+O%E2%80%99Rourkeauthor=S.+Krembauthor=B.+Dugganauthor=S.+Sioudauthor=N.+Kharbatiaauthor=M.+Rajiauthor=A.-H.+Emwasauthor=W.+Gerwickauthor=C.+Voolstra&title=Identification+of+a+3-Alkylpyridinium+compound+from+the+Red+Sea+sponge+Amphimedon+chloros+with+in+vitro+inhibitory+activity+against+the+West+Nile+virus+NS3+protease&doi=10.3390%2Fmolecules23061472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23061472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23061472%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DA.%26aulast%3DKremb%26aufirst%3DS.%26aulast%3DDuggan%26aufirst%3DB.%26aulast%3DSioud%26aufirst%3DS.%26aulast%3DKharbatia%26aufirst%3DN.%26aulast%3DRaji%26aufirst%3DM.%26aulast%3DEmwas%26aufirst%3DA.-H.%26aulast%3DGerwick%26aufirst%3DW.%26aulast%3DVoolstra%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%25203-Alkylpyridinium%2520compound%2520from%2520the%2520Red%2520Sea%2520sponge%2520Amphimedon%2520chloros%2520with%2520in%2520vitro%2520inhibitory%2520activity%2520against%2520the%2520West%2520Nile%2520virus%2520NS3%2520protease%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1420%26epage%3D3049%26doi%3D10.3390%2Fmolecules23061472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zovko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viktorsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewensohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KoloÅ¡a, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FilipiÄ, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kem, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paleari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, T.</span></span> <span> </span><span class="NLM_article-title">APS8, a polymeric alkylpyridinium salt blocks Î±7 nAChR and induces apoptosis in non-small cell lung carcinoma</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2574</span>â <span class="NLM_lpage">2594</span>, <span class="refDoi">Â DOI: 10.3390/md11072574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fmd11072574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23880932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1OntLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=2574-2594&author=A.+Zovkoauthor=K.+Viktorssonauthor=R.+Lewensohnauthor=K.+Kolo%C5%A1aauthor=M.+Filipi%C4%8Dauthor=H.+Xingauthor=W.+Kemauthor=L.+Paleariauthor=T.+Turk&title=APS8%2C+a+polymeric+alkylpyridinium+salt+blocks+%CE%B17+nAChR+and+induces+apoptosis+in+non-small+cell+lung+carcinoma&doi=10.3390%2Fmd11072574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">APS8, a polymeric alkylpyridinium salt blocks Î±7 nAChR and induces apoptosis in non-small cell lung carcinoma</span></div><div class="casAuthors">Zovko, Ana; Viktorsson, Kristina; Lewensohn, Rolf; Kolosa, Katja; Filipic, Metka; Xing, Hong; Kem, William R.; Paleari, Laura; Turk, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2574-2594, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Naturally occurring 3-alkylpyridinium polymers (poly-APS) from the marine sponge Reniera sarai, consisting of monomers contg. polar pyridinium and nonpolar alkyl chain moieties, have been demonstrated to exert a wide range of biol. activities, including a selective cytotoxicity against non-small cell lung cancer (NSCLC) cells.  APS8, an analog of poly-APS with defined alkyl chain length and mol. size, non-competitively inhibits Î±7 nicotinic acetylcholine receptors (nAChRs) at nanomolar concns. that are too low to be acetylcholinesterase (AChE) inhibitory or generally cytotoxic.  In the present study we show that APS8 inhibits NSCLC tumor cell growth and activates apoptotic pathways.  APS8 was not toxic for normal lung fibroblasts.  Furthermore, in NSCLC cells, APS8 reduced the adverse anti-apoptotic, proliferative effects of nicotine.  Our results suggest that APS8 or similar compds. might be considered as lead compds. to develop antitumor therapeutic agents for at least certain types of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1pxDm-wo0rLVg90H21EOLACvtfcHk0lgruP5KvwSQBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1OntLrK&md5=916e08d126a917ad1635cecad12ea82e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fmd11072574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd11072574%26sid%3Dliteratum%253Aachs%26aulast%3DZovko%26aufirst%3DA.%26aulast%3DViktorsson%26aufirst%3DK.%26aulast%3DLewensohn%26aufirst%3DR.%26aulast%3DKolo%25C5%25A1a%26aufirst%3DK.%26aulast%3DFilipi%25C4%258D%26aufirst%3DM.%26aulast%3DXing%26aufirst%3DH.%26aulast%3DKem%26aufirst%3DW.%26aulast%3DPaleari%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DT.%26atitle%3DAPS8%252C%2520a%2520polymeric%2520alkylpyridinium%2520salt%2520blocks%2520%25CE%25B17%2520nAChR%2520and%2520induces%2520apoptosis%2520in%2520non-small%2520cell%2520lung%2520carcinoma%26jtitle%3DMar.%2520Drugs%26date%3D2013%26volume%3D11%26spage%3D2574%26epage%3D2594%26doi%3D10.3390%2Fmd11072574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GrandiÄ, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ArÃ¡oz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MolgÃ³, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SepÄiÄ, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FrangeÅ¾, R.</span></span> <span> </span><span class="NLM_article-title">The non-competitive acetylcholinesterase inhibitor APS12-2 is a potent antagonist of skeletal muscle nicotinic acetylcholine receptors</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">221</span>â <span class="NLM_lpage">228</span>, <span class="refDoi">Â DOI: 10.1016/j.taap.2012.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.taap.2012.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23046821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslejtbbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=2012&pages=221-228&author=M.+Grandi%C4%8Dauthor=R.+Ar%C3%A1ozauthor=J.+Molg%C3%B3author=T.+Turkauthor=K.+Sep%C4%8Di%C4%87author=E.+Benoitauthor=R.+Frange%C5%BE&title=The+non-competitive+acetylcholinesterase+inhibitor+APS12-2+is+a+potent+antagonist+of+skeletal+muscle+nicotinic+acetylcholine+receptors&doi=10.1016%2Fj.taap.2012.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The non-competitive acetylcholinesterase inhibitor APS12-2 is a potent antagonist of skeletal muscle nicotinic acetylcholine receptors</span></div><div class="casAuthors">Grandic, Marjana; Araoz, Romulo; Molgo, Jordi; Turk, Tom; Sepcic, Kristina; Benoit, Evelyne; Frangez, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">221-228</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">APS12-2, a non-competitive acetylcholinesterase inhibitor, is one of the synthetic analogs of polymeric alkylpyridinium salts (poly-APS) isolated from the marine sponge Reniera sarai.  In the present work the effects of APS12-2 were studied on isolated mouse phrenic nerve-hemidiaphragm muscle prepns., using twitch tension measurements and electrophysiol. recordings.  APS12-2 in a concn.-dependent manner blocked nerve-evoked isometric muscle contraction (IC50 = 0.74 Î¼M), without affecting directly-elicited twitch tension up to 2.72 Î¼M.  The compd. (0.007-3.40 Î¼M) decreased the amplitude of miniature endplate potentials until a complete block by concns. higher than 0.68 Î¼M, without affecting their frequency.  Full size endplate potentials, recorded after blocking voltage-gated muscle sodium channels, were inhibited by APS12-2 in a concn.-dependent manner (IC50 = 0.36 Î¼M) without significant change in the resting membrane potential of the muscle fibers up to 3.40 Î¼M.  The compd. also blocked acetylcholine-evoked inward currents in Xenopus oocytes in which Torpedo (Î±12Î²1Î³Î´) muscle-type nicotinic acetylcholine receptors (nAChRs) have been incorporated (IC50 = 0.0005 Î¼M), indicating a higher affinity of the compd. for Torpedo (Î±12Î²1Î³Î´) than for the mouse (Î±12Î²1Î³Îµ) nAChR.  Our data show for the first time that APS12-2 blocks neuromuscular transmission by a non-depolarizing mechanism through an action on postsynaptic nAChRs of the skeletal neuromuscular junction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo91qjIVLLOgLVg90H21EOLACvtfcHk0lgruP5KvwSQBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslejtbbN&md5=7e22be7d20d8122fb8fe0db20cfa890b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2012.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2012.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DGrandi%25C4%258D%26aufirst%3DM.%26aulast%3DAr%25C3%25A1oz%26aufirst%3DR.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DT.%26aulast%3DSep%25C4%258Di%25C4%2587%26aufirst%3DK.%26aulast%3DBenoit%26aufirst%3DE.%26aulast%3DFrange%25C5%25BE%26aufirst%3DR.%26atitle%3DThe%2520non-competitive%2520acetylcholinesterase%2520inhibitor%2520APS12-2%2520is%2520a%2520potent%2520antagonist%2520of%2520skeletal%2520muscle%2520nicotinic%2520acetylcholine%2520receptors%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2012%26volume%3D265%26spage%3D221%26epage%3D228%26doi%3D10.1016%2Fj.taap.2012.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pantoom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suginta, W.</span></span> <span> </span><span class="NLM_article-title">Potent family-18 chitinase inhibitors X-ray structures, affinities, and binding mechanisms</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">24312</span>â <span class="NLM_lpage">24323</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M110.183376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M110.183376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21531720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1GltLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=24312-24323&author=S.+Pantoomauthor=I.+R.+Vetterauthor=H.+Prinzauthor=W.+Suginta&title=Potent+family-18+chitinase+inhibitors+X-ray+structures%2C+affinities%2C+and+binding+mechanisms&doi=10.1074%2Fjbc.M110.183376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Family-18 Chitinase Inhibitors: X-RAY STRUCTURES, AFFINITIES, AND BINDING MECHANISMS</span></div><div class="casAuthors">Pantoom, Supansa; Vetter, Ingrid R.; Prinz, Heino; Suginta, Wipa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">24312-24323</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Six novel inhibitors of Vibrio harveyi chitinase A (VhChiA), a family-18 chitinase homolog, were identified by in vitro screening of a library of pharmacol. active compds.  Unlike the previously identified inhibitors that mimicked the reaction intermediates, crystallog. evidence from 14 VhChiA-inhibitor complexes showed that all of the inhibitor mols. occupied the outer part of the substrate-binding cleft at two hydrophobic areas.  The interactions at the aglycon location are well defined and tightly assocd. with Trp-397 and Trp-275, whereas the interactions at the glycone location are patchy, indicating lower affinity and a loose interaction with two consensus residues, Trp-168 and Val-205.  When Trp-275 was substituted with glycine (W275G), the binding affinity toward all of the inhibitors dramatically decreased, and in most structures two inhibitor mols. were found to stack against Trp-397 at the aglycon site.  Such results indicate that hydrophobic interactions are important for binding of the newly identified inhibitors by the chitinase.  X-ray data and isothermal microcalorimetry showed that the inhibitors occupied the active site of VhChiA in three different binding modes, including single-site binding, independent two-site binding, and sequential two-site binding.  The inhibitory effect of dequalinium in the low nanomolar range makes this compd. an extremely attractive lead compd. for plausible development of therapeutics against human diseases involving chitinase-mediated pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwlrfx2sXPQrVg90H21EOLACvtfcHk0lgruP5KvwSQBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1GltLk%253D&md5=90d769da0f73c2b8fd3f6dab53c3b880</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.183376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.183376%26sid%3Dliteratum%253Aachs%26aulast%3DPantoom%26aufirst%3DS.%26aulast%3DVetter%26aufirst%3DI.%2BR.%26aulast%3DPrinz%26aufirst%3DH.%26aulast%3DSuginta%26aufirst%3DW.%26atitle%3DPotent%2520family-18%2520chitinase%2520inhibitors%2520X-ray%2520structures%252C%2520affinities%252C%2520and%2520binding%2520mechanisms%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D24312%26epage%3D24323%26doi%3D10.1074%2Fjbc.M110.183376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><a href="http://rsd.wuxiapptec.com/InvitrosafetyprofilingforMedchemSupport" class="extLink">http://rsd.wuxiapptec.com/InvitrosafetyprofilingforMedchemSupport</a> (Accessed on Dec. 31, 2019)</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Frsd.wuxiapptec.com%2FInvitrosafetyprofilingforMedchemSupport+%28Accessed+on+Dec.+31%2C+2019%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lhVPS0tVBxYPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Hierarchical virtual screening approaches in small molecule drug discovery</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">26</span>â <span class="NLM_lpage">37</span>, <span class="refDoi">Â DOI: 10.1016/j.ymeth.2014.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.ymeth.2014.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=25072167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12mt7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=26-37&author=A.+Kumarauthor=K.+Y.+J.+Zhang&title=Hierarchical+virtual+screening+approaches+in+small+molecule+drug+discovery&doi=10.1016%2Fj.ymeth.2014.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical virtual screening approaches in small molecule drug discovery</span></div><div class="casAuthors">Kumar, Ashutosh; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-37</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Virtual screening has played a significant role in the discovery of small mol. inhibitors of therapeutic targets in last two decades.  Various ligand and structure-based virtual screening approaches are employed to identify small mol. ligands for proteins of interest.  These approaches are often combined in either hierarchical or parallel manner to take advantage of the strength and avoid the limitations assocd. with individual methods.  Hierarchical combination of ligand and structure-based virtual screening approaches has received noteworthy success in numerous drug discovery campaigns.  In hierarchical virtual screening, several filters using ligand and structure-based approaches are sequentially applied to reduce a large screening library to a no. small enough for exptl. testing.  In this review, we focus on different hierarchical virtual screening strategies and their application in the discovery of small mol. modulators of important drug targets.  Several virtual screening studies are discussed to demonstrate the successful application of hierarchical virtual screening in small mol. drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI5IS_ELjp3rVg90H21EOLACvtfcHk0lhVPS0tVBxYPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12mt7bO&md5=37afd198e391774598a72214a377fd05</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DHierarchical%2520virtual%2520screening%2520approaches%2520in%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D26%26epage%3D37%26doi%3D10.1016%2Fj.ymeth.2014.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">82</span>, <span class="refDoi">Â DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lhVPS0tVBxYPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the protein databank and cambridge structural database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+protein+databank+and+cambridge+structural+database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lioSuirHRsoCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520protein%2520databank%2520and%2520cambridge%2520structural%2520database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimoto, T. T.</span></span> <span> </span><span class="NLM_article-title">A computer program for classifying plants</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1960</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">1115</span>â <span class="NLM_lpage">1118</span>, <span class="refDoi">Â DOI: 10.1126/science.132.3434.1115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1126%2Fscience.132.3434.1115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17790723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BC3cvgvFyntw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=1960&pages=1115-1118&author=D.+J.+Rogersauthor=T.+T.+Tanimoto&title=A+computer+program+for+classifying+plants&doi=10.1126%2Fscience.132.3434.1115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Computer Program for Classifying Plants</span></div><div class="casAuthors">Rogers D J; Tanimoto T T</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1960</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">3434</span>),
    <span class="NLM_cas:pages">1115-8</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL7dGsQdn6sO2qxNLjzXe8fW6udTcc2eYaG7lnOE0wr7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cvgvFyntw%253D%253D&md5=1b1906eaa3974e68c3933a464484041d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.132.3434.1115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.132.3434.1115%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%2BJ.%26aulast%3DTanimoto%26aufirst%3DT.%2BT.%26atitle%3DA%2520computer%2520program%2520for%2520classifying%2520plants%26jtitle%3DScience%26date%3D1960%26volume%3D132%26spage%3D1115%26epage%3D1118%26doi%3D10.1126%2Fscience.132.3434.1115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachbar, R. B.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. IV. conformational energies and geometries for MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">615</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<587::AID-JCC4>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=587-615&author=T.+A.+Halgrenauthor=R.+B.+Nachbar&title=Merck+molecular+force+field.+IV.+conformational+energies+and+geometries+for+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. IV. Conformational energies and geometries for MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.; Nachbar, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">587-615</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the parameterization and performance of MMFF94 for conformational energies, rotational barriers, and equil. torsion angles.  It describes the derivation of the torsion parameters from high-quality computational data and characterizes MMFF94's ability to reproduce the computational and exptl. data, the latter particularly in relation to MM3.  The computational data included: (i) â¼250 comparisons of conformational energy based on "MP4SDQ/TZP" calcns. (triple-zeta plus polarization calcns. at a defined approxn. to the highly correlated MP4SDQ level) at MP2/6-31G* geometries; and (ii) â¼1200 MP2/TZP comparisons of "torsion profile" structures at geometries derived in restrained least-squares fits that used the complete set of available computational data, thereby ensuring that a fully optimal set of parameters would be obtained.  The final parameters reproduce the "MP4SDQ/TZP" and MP2/TZP computational data with root mean square (rms) deviations of 0.31 and 0.50 kcal/mol, resp.  In addn., MMFF94 reproduces a set of 37 exptl. gas-phase and soln. conformational energies, enthalpies, and free energies with a rms deviation of 0.38 kcal/mol; for comparison, the "MP3SDQ/TZP" calcns. and MM3 each gives a rms deviation of 0.37 kcal/mol.  Furthermore, MMFF94 reproduces 28 exptl. detd. rotational barriers with a rms deviation of 0.39 kcal/mol.  Given the diverse nature of the exptl. conformational energies and rotational barriers and the clear indications of exptl. error in some cases, the MMFF94 results appear excellent.  Nevertheless, MMFF94 encounters somewhat greater difficulty in handling multi-functional compds. that place highly polar functional groups in close proximity, probably because it, like other commonly used force fields, too greatly simplifies the description of electrostatic interactions.  Some suggestions for enhancements to MMFF94's functional form are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTSvDsbv5rM7Vg90H21EOLACvtfcHk0lioSuirHRsoCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmtr4%253D&md5=fd00606ee59a83268c33e614220127f2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C587%253A%253AAID-JCC4%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DNachbar%26aufirst%3DR.%2BB.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520IV.%2520conformational%2520energies%2520and%2520geometries%2520for%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D587%26epage%3D615%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. III. Molecular geometries and vibrational frequencies for MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">553</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<553::AID-JCC3>3.0.CO;2-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=553-586&author=T.+A.+Halgren&title=Merck+molecular+force+field.+III.+Molecular+geometries+and+vibrational+frequencies+for+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. III. Molecular geometries and vibrational frequencies for MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">553-86</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the parameterization and performance of MMFF94 for mol. geometries and deformations.  It defines the form used for the valence-coordinate terms that represent variations in bond lengths and angles, and it describes the derivation of quadratic force consts. from HF/6-31G* data and the derivation of ref. bond lengths and angles from fits to MP2/6-31G*-optimized geometries.  Comparisons offered show that MMFF94 accurately reproduces the computational data used in its parameterization and demonstrate that its derivation from such data simultaneously confers the ability to reproduce expt.  In particular, MMFF94 reproduces exptl. detd. bond lengths and angles for 30 org. mols. with root mean square (rms) deviations of 0.014 Ã and 1.2Â°, resp.  MM3 reproduces bond angles to the same accuracy, but reproduces exptl. bond lengths more accurately, in part because it was fit directly to thermally averaged exptl. bond lengths; MMFF94, in contrast, was fit to (usually shorter) energy-min. values, as is proper for an anharmonic force field intended for use in mol.-dynamics simulations.  The comparisons also show that UFF and a recent version of CHARMm (QUANTA 3.3 parameterization) are less accurate for mol. geometries than either MMFF94 or MM3.  For vibrational frequencies, MMFF94 and MM3 give comparable overall rms deviations vs. expt. of 61 cm-1 and 57 cm-1, resp., for 15 small, mostly org. mols.  In a no. of instances, MM3's derivation employed obsd. frequencies that differ substantially-by nearly 400 cm cm-1 in one case-from other published frequencies which had themselves been confirmed theor. by good-quality ab initio calcns.  Overall, the comparisons to exptl. geometries and vibrational frequencies demonstrate that MMFF94 achieves MM3-like accuracy for org. systems for which MM3 has been parameterized.  Because MMFF94 is derived mainly from computational data, however, it has been possible to parameterize MMFF94 with equal rigor for a wide variety of addnl. systems for which little or no useful exptl. data exist.  Equally good performance can be expected for such systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWR5mBOf56TbVg90H21EOLACvtfcHk0lg31alcUN77qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbc%253D&md5=3ef990cab5dd2c6f9e179d99c066de5d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C553%253A%253AAID-JCC3%253E3.0.CO%253B2-T%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520III.%2520Molecular%2520geometries%2520and%2520vibrational%2520frequencies%2520for%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D553%26epage%3D586%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>â <span class="NLM_lpage">519</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+molecular+force+field.+I.+Basis%2C+form%2C+scope%2C+parameterization%2C+and+performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Ã root mean square [rms]), bond angles (1.2Â° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0lg31alcUN77qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520I.%2520Basis%252C%2520form%252C%2520scope%252C%2520parameterization%252C%2520and%2520performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. V. Extension of MMFF94 using experimental data, additional computational data, and empirical rules</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">616</span>â <span class="NLM_lpage">641</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<616::AID-JCC5>3.0.CO;2-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=616-641&author=T.+A.+Halgren&title=Merck+molecular+force+field.+V.+Extension+of+MMFF94+using+experimental+data%2C+additional+computational+data%2C+and+empirical+rules&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. V. Extension of MMFF94 using experimental data, additional computational data, and empirical rules</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">616-41</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the extension of the Merck Mol. Force Field (MMFF94) to a much broader range of org. systems.  It also describes a preliminary parameterization of MMFF94 for the hydronium and hydroxide ions and for various halide, alkali, and alkali earth ions as well as for such "protein" metals as Zn2+, Ca2+, Cu2+, Cu+, Fe2+, and Fe3+.  The extension employed computational data on charge distributions, mol. geometries, and conformational energies for a series of oxysulfur(particularly sulfonamide) and oxyphosphorous compds. and for a diverse set of small mols. and ions approx. 2800 good-quality structures extd. from the Cambridge Structural Database (CSD).  Some of the addnl. computational data were used to extend the explicit parametrization of electrostatic interactions and to more widely define a useful additive approxn. for the "bond polarity" and computational data served to define ref. bond lengths and angles that the extended force fields uses in conjunction with force consts. obtained from carefully calibrated empirical rules.  The extended torsion parameters consist partly of explicit parameters derived to reproduce MP2/6-31G* conformational energies and partly of "default parameters" provided by empirical rules patterned after those used in DREIDING and UFF but calibrated, where possible, against computationally derived MMFF94 torsion parameters.  Comparisons to exptl. data show that MMFF94 reproduces crystallog. bond lengths and bond angles with relatively modest root mean square (rms) deviations of â¼0.02 Ã and 2Â°, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4ynbnad1JgbVg90H21EOLACvtfcHk0lg31alcUN77qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmtr8%253D&md5=7741ad82491c06e849077b18c3dd6778</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C616%253A%253AAID-JCC5%253E3.0.CO%253B2-X%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520V.%2520Extension%2520of%2520MMFF94%2520using%2520experimental%2520data%252C%2520additional%2520computational%2520data%252C%2520and%2520empirical%2520rules%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D616%26epage%3D641%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. II. MMFF94 van der Waals and electrostatic parameters for intermolecular interactions</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">520</span>â <span class="NLM_lpage">552</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<520::AID-JCC2>3.0.CO;2-W</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=520-552&author=T.+A.+Halgren&title=Merck+molecular+force+field.+II.+MMFF94+van+der+Waals+and+electrostatic+parameters+for+intermolecular+interactions&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. II. MMFF94 van der Waals and electrostatic parameters for intermolecular interactions</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">520-52</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article defines the parameterization and performance of MMFF94 for intermol. interactions.  It specifies the novel "buffered" functional forms used for treating van der Waals (vdW) and electrostatic interactions, and describes the use of: (1) high quality ab initio data to parameterize vdW interactions involving aliph. hydrogens; and (2) HF/6-31 G* calcns. on hydrogen-bonded complexes to parameterize nonbonded interactions in polar systems.  Comparisons show that appropriate trends in the HF/6-31G* data are well reproduced by MMFF94 and that intermol. interaction energies and geometries closely parallel those given by the highly regarded OPLS force field.  A proper balance between solvent-solvent, solvent-solute, and solute-solute interactions, critically important for prospective success in aq. simulations, thus appears to be attained.  Comparison of MMFF94, OPLS, CHELPG electrostatic potential fit, QEq, Gasteiger, and Abraham charges for 20 small mols. and ions also shows the close correspondence between MMFF94 and OPLS.  As do OPLS and all current, widely used force fields, MMFF94 employs "effective pair potentials" which incorporate in an averaged way the increases in polarity which occur in high dielec. media.  Some limitations of this approach are discussed and suggestions for possible enhancements to MMFF94's functional form are noted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX0kuyu0af2rVg90H21EOLACvtfcHk0ljbW0xWm_ZPYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbY%253D&md5=b6f7f0fcddcee6aeb7cdba747f594070</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C520%253A%253AAID-JCC2%253E3.0.CO%253B2-W%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520II.%2520MMFF94%2520van%2520der%2520Waals%2520and%2520electrostatic%2520parameters%2520for%2520intermolecular%2520interactions%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D520%26epage%3D552%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljbW0xWm_ZPYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>â <span class="NLM_lpage">1749</span>, <span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljbW0xWm_ZPYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>â <span class="NLM_lpage">6196</span>, <span class="refDoi">Â DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0liijKVbZC5G_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK <sub>a</sub> prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0liijKVbZC5G_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">591</span>â <span class="NLM_lpage">604</span>, <span class="refDoi">Â DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=591-604&author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&doi=10.1007%2Fs10822-010-9349-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0liijKVbZC5G_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D591%26epage%3D604%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousido-Siah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitschler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podjarny, A.</span></span> <span> </span><span class="NLM_article-title">New insights into the enzymatic mechanism of human chitotriosidase (CHIT1) catalytic domain by atomic resolution X-ray diffraction and hybrid QM/MM</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1455</span>â <span class="NLM_lpage">1470</span>, <span class="refDoi">Â DOI: 10.1107/S139900471500783X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS139900471500783X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=26143917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gnu7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1455-1470&author=F.+Fadelauthor=Y.+Zhaoauthor=R.+Cachauauthor=A.+Cousido-Siahauthor=F.+X.+Ruizauthor=K.+Harlosauthor=E.+Howardauthor=A.+Mitschlerauthor=A.+Podjarny&title=New+insights+into+the+enzymatic+mechanism+of+human+chitotriosidase+%28CHIT1%29+catalytic+domain+by+atomic+resolution+X-ray+diffraction+and+hybrid+QM%2FMM&doi=10.1107%2FS139900471500783X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into the enzymatic mechanism of human chitotriosidase (CHIT1) catalytic domain by atomic resolution X-ray diffraction and hybrid QM/MM</span></div><div class="casAuthors">Fadel, Firas; Zhao, Yuguang; Cachau, Raul; Cousido-Siah, Alexandra; Ruiz, Francesc X.; Harlos, Karl; Howard, Eduardo; Mitschler, Andre; Podjarny, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1455-1470</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Chitotriosidase (CHIT1) is a human chitinase belonging to the highly conserved glycosyl hydrolase family 18 (GH18).  GH18 enzymes hydrolyze chitin, an N-acetylglucosamine polymer synthesized by lower organisms for structural purposes.  Recently, CHIT1 has attracted attention owing to its upregulation in immune-system disorders and as a marker of Gaucher disease.  The 39 kDa catalytic domain shows a conserved cluster of three acidic residues, Glu140, Asp138 and Asp136, involved in the hydrolysis reaction.  Under an excess concn. of substrate, CHIT1 and other homologues perform an addnl. activity, transglycosylation.  To understand the catalytic mechanism of GH18 chitinases and the dual enzymic activity, the structure and mechanism of CHIT1 were analyzed in detail.  The resoln. of the crystals of the catalytic domain was improved from 1.65 Ã (PDB entry 1waw) to 0.95-1.10 Ã for the apo and pseudo-apo forms and the complex with chitobiose, allowing the detn. of the protonation states within the active site.  This information was extended by hybrid quantum mechanics/mol. mechanics (QM/MM) calcns.  The results suggest a new mechanism involving changes in the conformation and protonation state of the catalytic triad, as well as a new role for Tyr27, providing new insights into the hydrolysis and transglycosylation activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ62bad1_u27Vg90H21EOLACvtfcHk0ljFJye2CtPW1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gnu7nP&md5=6ef30a3b71be3f2568d6e73be01b26d9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS139900471500783X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS139900471500783X%26sid%3Dliteratum%253Aachs%26aulast%3DFadel%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DCachau%26aufirst%3DR.%26aulast%3DCousido-Siah%26aufirst%3DA.%26aulast%3DRuiz%26aufirst%3DF.%2BX.%26aulast%3DHarlos%26aufirst%3DK.%26aulast%3DHoward%26aufirst%3DE.%26aulast%3DMitschler%26aufirst%3DA.%26aulast%3DPodjarny%26aufirst%3DA.%26atitle%3DNew%2520insights%2520into%2520the%2520enzymatic%2520mechanism%2520of%2520human%2520chitotriosidase%2520%2528CHIT1%2529%2520catalytic%2520domain%2520by%2520atomic%2520resolution%2520X-ray%2520diffraction%2520and%2520hybrid%2520QM%252FMM%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D1455%26epage%3D1470%26doi%3D10.1107%2FS139900471500783X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span> <span> </span><span class="NLM_article-title">Structure, catalysis and inhibition of <i>Of</i>Chi-h, the Lepidoptera-exclusive insect chitinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">2080</span>â <span class="NLM_lpage">2088</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M116.755330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M116.755330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=28053084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVGisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2080-2088&author=T.+Liuauthor=L.+Chenauthor=Y.+Zhouauthor=X.+Jiangauthor=Y.+Duanauthor=Q.+Yang&title=Structure%2C+catalysis+and+inhibition+of+OfChi-h%2C+the+Lepidoptera-exclusive+insect+chitinase&doi=10.1074%2Fjbc.M116.755330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, Catalysis, and Inhibition of OfChi-h, the Lepidoptera-exclusive Insect Chitinase</span></div><div class="casAuthors">Liu, Tian; Chen, Lei; Zhou, Yong; Jiang, Xi; Duan, Yanwei; Yang, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2080-2088</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chitinase-h (Chi-h) is of special interest among insect chitinases due to its exclusive distribution in lepidopteran insects and high sequence identity with bacterial and baculovirus homologs.  Here OfChi-h, a Chi-h from Ostrinia furnacalis, was investigated.  Crystal structures of both OfChi-h and its complex with chitoheptaose, (GlcN)7, reveal that OfChi-h possesses a long and asym. substrate binding cleft, which is a typical characteristics of a processive exo-chitinase.  The structural comparison between OfChi-h and its bacterial homolog SmChiA uncovered two phenylalanine-to-tryptophan site variants in OfChi-h at subsites +2 and possibly -7.  The F232W/F396W double mutant endowed SmChiA with higher hydrolytic activities toward insol. substrates, such as insect cuticle, Î±-chitin, and chitin nanowhisker.  An enzymic assay demonstrated that OfChi-h outperformed OfChtI, an insect endo-chitinase, toward the insol. substrates, but showed lower activity toward the sol. substrate ethylene glycol chitin.  Furthermore, OfChi-h was found to be inhibited by N,N',N''-trimethylglucosamine-N,N',N'',N'''-tetraacetylchitotetraose (TMG-(GlcNAc)4), a substrate analog which can be degraded into TMG-(GlcNAc)1-2.  Injection of TMG-(GlcNAc)4 into 5th-instar O. furnacalis larvae led to severe defects in pupation.  This work provides insights into a molting-indispensable insect chitinase that is phylogenetically closer to bacterial chitinases than insect chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQLeeZuN8zRrVg90H21EOLACvtfcHk0ljFJye2CtPW1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVGisb8%253D&md5=0a7fc8a49c6f9a8e57b9f924a6bcc0a5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.755330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.755330%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DQ.%26atitle%3DStructure%252C%2520catalysis%2520and%2520inhibition%2520of%2520OfChi-h%252C%2520the%2520Lepidoptera-exclusive%2520insect%2520chitinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D2080%26epage%3D2088%26doi%3D10.1074%2Fjbc.M116.755330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirono, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunazuka, T.</span></span> <span> </span><span class="NLM_article-title">Observation of the controlled assembly of preclick components in the in situ click chemistry generation of a chitinase inhibitor</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15892</span>â <span class="NLM_lpage">15897</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1315049110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1073%2Fpnas.1315049110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=24043811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15892-15897&author=T.+Hiroseauthor=N.+Maitaauthor=H.+Goudaauthor=J.+Kosekiauthor=T.+Yamamotoauthor=A.+Sugawaraauthor=H.+Nakanoauthor=S.+Hironoauthor=K.+Shiomiauthor=T.+Watanabeauthor=H.+Taniguchiauthor=K.+B.+Sharplessauthor=S.+Omuraauthor=T.+Sunazuka&title=Observation+of+the+controlled+assembly+of+preclick+components+in+the+in+situ+click+chemistry+generation+of+a+chitinase+inhibitor&doi=10.1073%2Fpnas.1315049110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Observation of the controlled assembly of preclick components in the in situ click chemistry generation of a chitinase inhibitor</span></div><div class="casAuthors">Hirose, Tomoyasu; Maita, Nobuo; Gouda, Hiroaki; Koseki, Jun; Yamamoto, Tsuyoshi; Sugawara, Akihiro; Nakano, Hirofumi; Hirono, Shuichi; Shiomi, Kazuro; Watanabe, Takeshi; Taniguchi, Hisaaki; Sharpless, K. Barry; Omura, Satoshi; Sunazuka, Toshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">15892-15897,S15892/1-S15892/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Huisgen cycloaddn. of azides and alkynes, accelerated by target biomols., termed "in situ click chem.," has been successfully exploited to discover highly potent enzyme inhibitors.  We have previously reported a specific Serratia marcescens chitinase B (SmChiB)-templated syn-triazole inhibitor generated in situ from an azide-bearing inhibitor and an alkyne fragment.  Several in situ click chem. studies have been reported.  Although some mechanistic evidence has been obtained, such as X-ray anal. of [protein]-["click ligand"] complexes, indicating that proteins act as both mold and template between unique pairs of azide and alkyne fragments, to date, observations have been based solely on "postclick" structural information.  Here, we describe crystal structures of SmChiB complexed with an azide ligand and an O-allyl oxime fragment as a mimic of a click partner, revealing a mechanism for accelerating syn-triazole formation, which allows generation of its own distinct inhibitor.  We have also performed d. functional theory calcns. based on the X-ray structure to explore the acceleration of the Huisgen cycloaddn. by SmChiB.  The d. functional theory calcns. reasonably support that SmChiB plays a role by the cage effect during the pre-translation and post-translation states of selective syn-triazole click formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvlga4V4v-ZbVg90H21EOLACvtfcHk0ljFJye2CtPW1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbrL&md5=b605dc38fae207c49cd915ac28f2d2f1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315049110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315049110%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DT.%26aulast%3DMaita%26aufirst%3DN.%26aulast%3DGouda%26aufirst%3DH.%26aulast%3DKoseki%26aufirst%3DJ.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DSugawara%26aufirst%3DA.%26aulast%3DNakano%26aufirst%3DH.%26aulast%3DHirono%26aufirst%3DS.%26aulast%3DShiomi%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DTaniguchi%26aufirst%3DH.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26aulast%3DOmura%26aufirst%3DS.%26aulast%3DSunazuka%26aufirst%3DT.%26atitle%3DObservation%2520of%2520the%2520controlled%2520assembly%2520of%2520preclick%2520components%2520in%2520the%2520in%2520situ%2520click%2520chemistry%2520generation%2520of%2520a%2520chitinase%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D15892%26epage%3D15897%26doi%3D10.1073%2Fpnas.1315049110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>â <span class="NLM_lpage">674</span>, <span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljEKejWvIbkHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkÃ³czi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. D.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>â <span class="NLM_lpage">221</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljEKejWvIbkHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520D.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr. D.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljEKejWvIbkHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%2520D.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>â <span class="NLM_lpage">291</span>, <span class="refDoi">Â DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lhxTT6qtbKACg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealey, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyasu, S.</span></span> <span> </span><span class="NLM_article-title">Isolation and analysis of group 2 innate lymphoid cells in mice</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">792</span>â <span class="NLM_lpage">806</span>, <span class="refDoi">Â DOI: 10.1038/nprot.2015.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1038%2Fnprot.2015.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=25927389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVWqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=792-806&author=K.+Moroauthor=K.+N.+Ealeyauthor=H.+Kabataauthor=S.+Koyasu&title=Isolation+and+analysis+of+group+2+innate+lymphoid+cells+in+mice&doi=10.1038%2Fnprot.2015.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and analysis of group 2 innate lymphoid cells in mice</span></div><div class="casAuthors">Moro, Kazuyo; Ealey, Kafi N.; Kabata, Hiroki; Koyasu, Shigeo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-806</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent studies have identified distinct subsets of innate lymphocytes, collectively called innate lymphoid cells (ILCs), which lack antigen receptor expression but produce various effector cytokines.  Group 2 ILCs (ILC2s) respond to epithelial cell-derived cytokines such as interleukin (IL)-25, IL-33 and thymic stromal lymphopoietin (TSLP), produce large amts. of type 2 cytokines, and have a key role in anti-helminth innate immunity and in the pathophysiol. of allergic inflammation.  The reported phenotypic characteristics of mouse ILC2s vary, depending on the tissue source and prepn. method.  This protocol describes improved methods for tissue-specific isolation and anal. of mouse ILC2s of high purity and yield from fat tissue, lung, bronchoalveolar lavage fluid (BALF) and small intestine.  These improved methods are the result of our thorough investigation of enzymes used for tissue digestion, methods for the elimination of undesired cells, and a combination of antibodies for the detection and isolation of ILC2s.  In addn., this new protocol now enables the isolation of ILC2s of high yield, even from inflamed tissues.  Depending on the tissue being analyzed, it takes â¼2-4 h for isolation and flow cytometric anal. of ILC2s from the various tissues of a single mouse and â¼4-8 h to sort purified ILC2s from pooled tissues of multiple mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoveg-q0y91HLVg90H21EOLACvtfcHk0lhxTT6qtbKACg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVWqtLY%253D&md5=0f2824429cf66c4b2edf3e1909c58b30</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.047%26sid%3Dliteratum%253Aachs%26aulast%3DMoro%26aufirst%3DK.%26aulast%3DEaley%26aufirst%3DK.%2BN.%26aulast%3DKabata%26aufirst%3DH.%26aulast%3DKoyasu%26aufirst%3DS.%26atitle%3DIsolation%2520and%2520analysis%2520of%2520group%25202%2520innate%2520lymphoid%2520cells%2520in%2520mice%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D792%26epage%3D806%26doi%3D10.1038%2Fnprot.2015.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ling Zhu, Lei Chen, Xusheng Shao, Jiagao Cheng, Qing Yang, <span class="NLM_string-name hlFld-ContribAuthor">Xuhong Qian</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Inhibitors of an Insect Pest Chitinase: Design and Optimization of 9-O-Aromatic and Heterocyclic Esters of Berberine. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2021,</strong> <em>69 </em>
                                    (27)
                                     , 7526-7533. <a href="https://doi.org/10.1021/acs.jafc.0c07401" title="DOI URL">https://doi.org/10.1021/acs.jafc.0c07401</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.0c07401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.0c07401%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DNovel%252BInhibitors%252Bof%252Ban%252BInsect%252BPest%252BChitinase%25253A%252BDesign%252Band%252BOptimization%252Bof%252B9-O-Aromatic%252Band%252BHeterocyclic%252BEsters%252Bof%252BBerberine%26aulast%3DZhu%26aufirst%3DLing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D23112020%26date%3D09062021%26date%3D16052021%26date%3D02072021%26volume%3D69%26issue%3D27%26spage%3D7526%26epage%3D7533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francois Berenger, Ashutosh Kumar, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Yoshihiro Yamanishi</span>. </span><span class="cited-content_cbyCitation_article-title">Lean-Docking: Exploiting Ligandsâ Predicted Docking Scores to Accelerate Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2341-2352. <a href="https://doi.org/10.1021/acs.jcim.0c01452" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01452%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLean-Docking%25253A%252BExploiting%252BLigands%2525E2%252580%252599%252BPredicted%252BDocking%252BScores%252Bto%252BAccelerate%252BMolecular%252BDocking%26aulast%3DBerenger%26aufirst%3DFrancois%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D17122020%26date%3D16042021%26volume%3D61%26issue%3D5%26spage%3D2341%26epage%3D2352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qi Chen, Wei Chen, Ashutosh Kumar, Xi Jiang, Matej Janezic, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Qing Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure and Structure-Based Discovery of Inhibitors of the Nematode Chitinase CeCht1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2021,</strong> <em>69 </em>
                                    (11)
                                     , 3519-3526. <a href="https://doi.org/10.1021/acs.jafc.1c00162" title="DOI URL">https://doi.org/10.1021/acs.jafc.1c00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.1c00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.1c00162%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DCrystal%252BStructure%252Band%252BStructure-Based%252BDiscovery%252Bof%252BInhibitors%252Bof%252Bthe%252BNematode%252BChitinase%252BCeCht1%26aulast%3DChen%26aufirst%3DQi%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D10012021%26date%3D02032021%26date%3D26022021%26date%3D11032021%26volume%3D69%26issue%3D11%26spage%3D3519%26epage%3D3526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert Koralewski, Barbara Dymek, Marzena Mazur, Piotr Sklepkiewicz, Sylwia Olejniczak, Wojciech Czestkowski, Krzysztof Matyszewski, Gleb Andryianau, Piotr Niedziejko, Michal Kowalski, Mariusz Gruza, BartÅomiej Borek, Karol Jedrzejczak, Agnieszka Bartoszewicz, ElÅ¼bieta Pluta, Aleksandra Rymaszewska, Magdalena Kania, Tomasz Rejczak, Sylwia Piasecka, Michal Mlacki, Marcin Mazurkiewicz, MichaÅ Piotrowicz, Magdalena Salamon, Agnieszka Zagozdzon, Agnieszka Napiorkowska-Gromadzka, Aneta Bartlomiejczak, Witold Mozga, PaweÅ DobrzaÅski, Karolina Dzwonek, Jakub Golab, Marcin Nowotny, Jacek Olczak, <span class="NLM_string-name hlFld-ContribAuthor">Adam Golebiowski</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of OATD-01, a First-in-Class Chitinase Inhibitor as Potential New Therapeutics for Idiopathic Pulmonary Fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15527-15540. <a href="https://doi.org/10.1021/acs.jmedchem.0c01179" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01179%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BOATD-01%25252C%252Ba%252BFirst-in-Class%252BChitinase%252BInhibitor%252Bas%252BPotential%252BNew%252BTherapeutics%252Bfor%252BIdiopathic%252BPulmonary%252BFibrosis%26aulast%3DKoralewski%26aufirst%3DRobert%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07072020%26date%3D20102020%26volume%3D63%26issue%3D24%26spage%3D15527%26epage%3D15540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pengtao Yuan, Xi Jiang, Siyu Wang, Xusheng Shao, Qing Yang, <span class="NLM_string-name hlFld-ContribAuthor">Xuhong Qian</span>. </span><span class="cited-content_cbyCitation_article-title">X-ray Structure and Molecular Docking Guided Discovery of Novel Chitinase Inhibitors with a Scaffold of Dipyridopyrimidine-3-carboxamide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2020,</strong> <em>68 </em>
                                    (47)
                                     , 13584-13593. <a href="https://doi.org/10.1021/acs.jafc.0c03742" title="DOI URL">https://doi.org/10.1021/acs.jafc.0c03742</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.0c03742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.0c03742%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DX-ray%252BStructure%252Band%252BMolecular%252BDocking%252BGuided%252BDiscovery%252Bof%252BNovel%252BChitinase%252BInhibitors%252Bwith%252Ba%252BScaffold%252Bof%252BDipyridopyrimidine-3-carboxamide%26aulast%3DYuan%26aufirst%3DPengtao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14062020%26date%3D20102020%26date%3D27102020%26date%3D05112020%26volume%3D68%26issue%3D47%26spage%3D13584%26epage%3D13593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gleb Andryianau, Michal Kowalski, Michal C. Piotrowicz, Adam A. Rajkiewicz, Barbara Dymek, Piotr L. Sklepkiewicz, Elzbieta Pluta, Filip Stefaniak, Wojciech Czestkowski, Sylwia Olejniczak, Marzena Mazur, Piotr Niedziejko, Robert Koralewski, Krzysztof Matyszewski, Mariusz Gruza, Agnieszka Zagozdzon, Magdalena Salamon, Aleksandra Rymaszewska, Mikolaj Welzer, Karolina Dzwonek, Jakub Golab, Jacek Olczak, Agnieszka Bartoszewicz, <span class="NLM_string-name hlFld-ContribAuthor">Adam Golebiowski</span>. </span><span class="cited-content_cbyCitation_article-title">Benzoxazepine-Derived Selective, Orally Bioavailable Inhibitor of Human Acidic Mammalian Chitinase. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1228-1235. <a href="https://doi.org/10.1021/acsmedchemlett.0c00092" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00092</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00092%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBenzoxazepine-Derived%252BSelective%25252C%252BOrally%252BBioavailable%252BInhibitor%252Bof%252BHuman%252BAcidic%252BMammalian%252BChitinase%26aulast%3DAndryianau%26aufirst%3DGleb%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21022020%26date%3D24042020%26date%3D01052020%26date%3D24042020%26volume%3D11%26issue%3D6%26spage%3D1228%26epage%3D1235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wei Chen, <span class="NLM_string-name hlFld-ContribAuthor">Qing Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Novel Pesticides Targeting Insect Chitinases: A Minireview and Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2020,</strong> <em>68 </em>
                                    (16)
                                     , 4559-4565. <a href="https://doi.org/10.1021/acs.jafc.0c00888" title="DOI URL">https://doi.org/10.1021/acs.jafc.0c00888</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.0c00888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.0c00888%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DDevelopment%252Bof%252BNovel%252BPesticides%252BTargeting%252BInsect%252BChitinases%25253A%252BA%252BMinireview%252Band%252BPerspective%26aulast%3DChen%26aufirst%3DWei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08022020%26date%3D28032020%26date%3D02042020%26date%3D08042020%26date%3D02042020%26volume%3D68%26issue%3D16%26spage%3D4559%26epage%3D4565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marzena  Mazur</span>, <span class="hlFld-ContribAuthor ">Anna  ZieliÅska</span>, <span class="hlFld-ContribAuthor ">Marcin M.  Grzybowski</span>, <span class="hlFld-ContribAuthor ">Jacek  Olczak</span>, <span class="hlFld-ContribAuthor ">Jakub  Fichna</span>. </span><span class="cited-content_cbyCitation_article-title">Chitinases and Chitinase-Like Proteins as Therapeutic Targets in Inflammatory Diseases, with a Special Focus on Inflammatory Bowel Diseases. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (13)
                                     , 6966. <a href="https://doi.org/10.3390/ijms22136966" title="DOI URL">https://doi.org/10.3390/ijms22136966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22136966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22136966%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DChitinases%252Band%252BChitinase-Like%252BProteins%252Bas%252BTherapeutic%252BTargets%252Bin%252BInflammatory%252BDiseases%25252C%252Bwith%252Ba%252BSpecial%252BFocus%252Bon%252BInflammatory%252BBowel%252BDiseases%26aulast%3DMazur%26aufirst%3DMarzena%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D13%26spage%3D6966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Chen</span>, <span class="hlFld-ContribAuthor ">Qi  Chen</span>, <span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Xi  Jiang</span>, <span class="hlFld-ContribAuthor ">Kam Y. J.  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based virtual screening of highly potent inhibitors of the nematode chitinase
              Ce
              Cht1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2021,</strong> <em>36 </em>
                                    (1)
                                     , 1198-1204. <a href="https://doi.org/10.1080/14756366.2021.1931862" title="DOI URL">https://doi.org/10.1080/14756366.2021.1931862</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2021.1931862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2021.1931862%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bvirtual%252Bscreening%252Bof%252Bhighly%252Bpotent%252Binhibitors%252Bof%252Bthe%252Bnematode%252Bchitinase%252BCe%252BCht1%26aulast%3DChen%26aufirst%3DWei%26date%3D2021%26date%3D2021%26volume%3D36%26issue%3D1%26spage%3D1198%26epage%3D1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Chen</span>, <span class="hlFld-ContribAuthor ">Xi  Jiang</span>, <span class="hlFld-ContribAuthor ">Qing  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Glycoside hydrolase family 18 chitinases: The known and the unknown. </span><span class="cited-content_cbyCitation_journal-name">Biotechnology Advances</span><span> <strong>2020,</strong> <em>43 </em>, 107553. <a href="https://doi.org/10.1016/j.biotechadv.2020.107553" title="DOI URL">https://doi.org/10.1016/j.biotechadv.2020.107553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biotechadv.2020.107553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biotechadv.2020.107553%26sid%3Dliteratum%253Aachs%26jtitle%3DBiotechnology%2520Advances%26atitle%3DGlycoside%252Bhydrolase%252Bfamily%252B18%252Bchitinases%25253A%252BThe%252Bknown%252Band%252Bthe%252Bunknown%26aulast%3DChen%26aufirst%3DWei%26date%3D2020%26volume%3D43%26spage%3D107553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of a few chitinase inhibitors reported in the literature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Hierarchical virtual screening strategy used to identify dipyrido-pyrimidine derivatives as chitinase inhibitors. A combination of shape similarity calculations, electrostatic potential similarity calculations, and molecular docking was employed to prioritize compounds for the enzymatic assay. Three rounds of virtual screening were carried out. (B) Residual activity of first round virtual screening compounds against <i>Hs</i>Chit1, <i>Of</i>Chi-h, and <i>Sm</i>ChiB. (C) A 3D shape of the query (GlcNAc)<sub>3</sub>. (D) A 3D shape overlay of the query (GlcNAc)<sub>3</sub> with a hit compound <b>24</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Chemical structures of a few relevant compounds with the dipyrido-pyrimidine scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Details of the interaction of <b>40</b> with <i>Hs</i>Chit1-CAD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR">6JJR</a>) (A) and <i>Sm</i>ChiB (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9">6JK9</a>) (B). Compound <b>40</b> is shown in light yellow, <i>Hs</i>Chit1 is shown in cyan, <i>Sm</i>ChiB is shown in white, hydrogen bonds are shown as dashed lines, and water molecules are shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Details of the interaction of <b>53</b> with <i>Of</i>Chi-h (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN">6JMN</a>) (A), <i>Hs</i>Chit1-CAD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6">6JK6</a>) (B), and <i>Sm</i>ChiB (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF">6JKF</a>) (C). Compound <b>53</b> is shown in pale pink, <i>Of</i>Chi-h is in green, <i>Hs</i>Chit1 is shown in cyan, <i>Sm</i>ChiB is shown in white, hydrogen bonds are shown as dashed lines, and water molecules are shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Expression levels of Chit1 and cytokines in lung tissue from the no-treatment, the bleomycin-challenged, and the <b>40</b>-treated bleomycin-challenged groups. The control contains 5% DMSO in PBS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cell recruitment in the lungs of bleomycin-induced mice treated with <b>40</b>. Graphs of the total cell, eosinophil, neutrophil, monocyte, alveolar macrophage, PD-L2<sup>+</sup> alveolar macrophage, interstitial macrophage, and PD-L2<sup>+</sup> interstitial macrophage numbers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/medium/jm9b01154_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition activity of <b>40</b> in the BALF of mice treated with bleomycin in vivo and in vitro.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01154/20200207/images/large/jm9b01154_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01154&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22128" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22128" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollak, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Weely, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Oers, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span> <span> </span><span class="NLM_article-title">Marked elevation of plasma chitotriosidase activity. A novel hallmark of Gaucher disease</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1288</span>â <span class="NLM_lpage">1292</span>, <span class="refDoi">Â DOI: 10.1172/JCI117084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1172%2FJCI117084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=8132768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADyaK2c7nvFCkug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1994&pages=1288-1292&author=C.+E.+Hollakauthor=S.+van+Weelyauthor=M.+H.+van+Oersauthor=J.+M.+Aerts&title=Marked+elevation+of+plasma+chitotriosidase+activity.+A+novel+hallmark+of+Gaucher+disease&doi=10.1172%2FJCI117084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Marked elevation of plasma chitotriosidase activity. A novel hallmark of Gaucher disease</span></div><div class="casAuthors">Hollak C E; van Weely S; van Oers M H; Aerts J M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1288-92</span>
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">Gaucher disease (GD; glucosylceramidosis) is caused by a deficient activity of the enzyme glucocerebrosidase (GC).  Clinical manifestations are highly variable and cannot be predicted accurately on the basis of the properties of mutant GC.  Analysis of secondary abnormalities, such as elevated plasma levels of some hydrolases, may help to increase insight into the complicated pathophysiology of the disease and could also provide useful disease markers.  The recent availability of enzyme supplementation therapy for GD increases the need for markers as early predictors of the efficacy of treatment.  We report the finding of a very marked increase in chitotrisidase activity in plasma of 30 of 32 symptomatic type 1 GD patients studied: the median activity being > 600 times the median value in plasma of healthy volunteers.  In three GC-deficient individuals without clinical symptoms, only slight increases were noted.  Chitotriosidase activity was absent in plasma of three control subjects and two patients.  During enzyme supplementation therapy, chitotriosidase activity declined dramatically.  We conclude that plasma chitotriosidase levels can serve as a new diagnostic hallmark of GD and should prove to be useful in assessing whether clinical manifestations of GD are present and for monitoring the efficacy of therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHCFYsz4T-0TMsgLhIoICsfW6udTcc2eYhloNWJexK3rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c7nvFCkug%253D%253D&md5=919539442ad855c2d21e7694637bb3cf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1172%2FJCI117084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI117084%26sid%3Dliteratum%253Aachs%26aulast%3DHollak%26aufirst%3DC.%2BE.%26aulast%3Dvan%2BWeely%26aufirst%3DS.%26aulast%3Dvan%2BOers%26aufirst%3DM.%2BH.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26atitle%3DMarked%2520elevation%2520of%2520plasma%2520chitotriosidase%2520activity.%2520A%2520novel%2520hallmark%2520of%2520Gaucher%2520disease%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1994%26volume%3D93%26spage%3D1288%26epage%3D1292%26doi%3D10.1172%2FJCI117084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kzhyshkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gratchev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goerdt, S.</span></span> <span> </span><span class="NLM_article-title">Human chitinases and chitinase-like proteins as indicators for inflammation and cancer</span>. <i>Biomarker Insights</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">128</span>â <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1177%2F117727190700200023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19662198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BD1MrktFSmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=128-146&author=J.+Kzhyshkowskaauthor=A.+Gratchevauthor=S.+Goerdt&title=Human+chitinases+and+chitinase-like+proteins+as+indicators+for+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human chitinases and chitinase-like proteins as indicators for inflammation and cancer</span></div><div class="casAuthors">Kzhyshkowska Julia; Gratchev Alexei; Goerdt Sergij</div><div class="citationInfo"><span class="NLM_cas:title">Biomarker insights</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">128-46</span>
        ISSN:<span class="NLM_cas:issn">1177-2719</span>.
    </div><div class="casAbstract">Human Glyco_18 domain-containing proteins constitute a family of chitinases and chitinase-like proteins.  Chitotriosidase and AMCase are true enzymes which hydrolyse chitin and have a C-terminal chitin-binding domain.  YKL-40, YKL-39, SI-CLP and murine YM1/2 proteins possess solely Glyco_18 domain and do not have the hydrolytic activity.  The major sources of Glyco_18 containing proteins are macrophages, neutrophils, epithelial cells, chondrocytes, synovial cells, and cancer cells.  Both macrophages and neutrophils use the regulated secretory mechanism for the release of Glyco_18 containing proteins.  Glyco_18 containing proteins are established biomarkers for human diseases.  Chitotriosidase is overproduced by lipid-laden macrophages and is a major marker for the inherited lysosomal storage Gaucher disease.  AMCase and murine lectin YM1 are upregulated in Th2-environment, and enzymatic activity of AMCase contributes to asthma pathogenesis.  YKL proteins act as soluble mediators for the cell proliferation and migration, and are also involved in rheumatoid arthritis, inflammatory bowel disease, hepatic fibrosis and cirrhosis.  Chitotriosidase and YKL-40 reflect the macrophage activation in atherosclerotic plaques.  Serum level of YKL-40 is a diagnostic and prognostic marker for numerous types of solid tumors.  YKL-39 is a marker for the activation of chondrocytes and the progression of the osteoarthritis in human.  Recently identified SI-CLP is upregulated by Th2 cytokine IL-4 as well as by glucocorticoids.  This unique feature of SI-CLP makes it an attractive candidate for the examination of individual sensitivity of patients to glucocorticoid treatment and prediction of side effects of glucocorticoid therapy.  Human chitinases and chitinase-like proteins are found in tissues and circulation, and can be detected by non-invasive technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZmzIeeqUN7ZdYoSR9eyvsfW6udTcc2eYhloNWJexK3rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrktFSmtQ%253D%253D&md5=23bce5cbcab08e01a3e9ffc2948f30a3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1177%2F117727190700200023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F117727190700200023%26sid%3Dliteratum%253Aachs%26aulast%3DKzhyshkowska%26aufirst%3DJ.%26aulast%3DGratchev%26aufirst%3DA.%26aulast%3DGoerdt%26aufirst%3DS.%26atitle%3DHuman%2520chitinases%2520and%2520chitinase-like%2520proteins%2520as%2520indicators%2520for%2520inflammation%2520and%2520cancer%26jtitle%3DBiomarker%2520Insights%26date%3D2007%26volume%3D2%26spage%3D128%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boven, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Meurs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laman, J. D.</span></span> <span> </span><span class="NLM_article-title">Gaucher cells demonstrate a distinct macrophage phenotype and resemble alternatively activated macrophages</span>. <i>Am. J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">359</span>â <span class="NLM_lpage">369</span>, <span class="refDoi">Â DOI: 10.1309/BG5VA8JRDQH1M7HN</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1309%2FBG5VA8JRDQH1M7HN" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15362365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotFyhurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2004&pages=359-369&author=L.+A.+Bovenauthor=M.+van+Meursauthor=R.+G.+Bootauthor=A.+Mehtaauthor=L.+Boonauthor=J.+M.+Aertsauthor=J.+D.+Laman&title=Gaucher+cells+demonstrate+a+distinct+macrophage+phenotype+and+resemble+alternatively+activated+macrophages&doi=10.1309%2FBG5VA8JRDQH1M7HN"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Gaucher cells demonstrate a distinct macrophage phenotype and resemble alternatively activated macrophages</span></div><div class="casAuthors">Boven, Leonie A.; van Meurs, Marjan; Boot, Rolf G.; Mehta, Atul; Boon, Louis; Aerts, Johannes M.; Laman, Jon D.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">359-369</span>CODEN:
                <span class="NLM_cas:coden">AJCPAI</span>;
        ISSN:<span class="NLM_cas:issn">0002-9173</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Pathologists</span>)
        </div><div class="casAbstract">Although the existence of anti-inflammatory alternatively activated macrophages (aaMÎ¦) has been accepted widely based on in vitro studies, their in vivo location, phenotype, and function still are debated.  Gaucher disease (GD) is caused by a genetic deficiency in the lysosomal enzyme glucocerebrosidase and is characterized by accumulation of glycosphingolipids in so-called Gaucher cells (GCs), By using immunohistochem. anal., we investigated whether this results in an aaMÎ¦ phenotype.  GCs are macrophage-like cells, expressing acid phosphatase, CD68, CD14, and HLA class II, but not CD11b, CD40, or dendritic cell markers.  GCs show infrequent immunoreactivity for mannose receptor.  GCs did not express proinflammatory cytokines such as tumor necrosis factor Î± and monocyte chemoattractant protein 1, but did express the aaMÎ¦ markers CD163, CCL18, and interleukin-1 receptor antagonist.  Furthermore, CD36 and signal receptor protein Î±, involved in lipid uptake, also were obsd. on GCs.  Thus, GCs represent a distinctive population of myeloid cells that resemble aaMÎ¦ but differ from previously described in vitro aaMÎ¦.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG3PZoxUXeabVg90H21EOLACvtfcHk0lgGiH9C1g281w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotFyhurg%253D&md5=2403ba7fd5c4d4921c14a1756fef30fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1309%2FBG5VA8JRDQH1M7HN&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1309%252FBG5VA8JRDQH1M7HN%26sid%3Dliteratum%253Aachs%26aulast%3DBoven%26aufirst%3DL.%2BA.%26aulast%3Dvan%2BMeurs%26aufirst%3DM.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DMehta%26aufirst%3DA.%26aulast%3DBoon%26aufirst%3DL.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26aulast%3DLaman%26aufirst%3DJ.%2BD.%26atitle%3DGaucher%2520cells%2520demonstrate%2520a%2520distinct%2520macrophage%2520phenotype%2520and%2520resemble%2520alternatively%2520activated%2520macrophages%26jtitle%3DAm.%2520J.%2520Clin.%2520Pathol.%26date%3D2004%26volume%3D122%26spage%3D359%26epage%3D369%26doi%3D10.1309%2FBG5VA8JRDQH1M7HN" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavala, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esnault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukreja, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertics, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chupp, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarjour, N. N.</span></span> <span> </span><span class="NLM_article-title">Segmental allergen challenge enhances chitinase activity and levels of CCL18 in mild atopic asthma</span>. <i>Clin. Exp. Allergy.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">197</span>, <span class="refDoi">Â DOI: 10.1111/cea.12032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1111%2Fcea.12032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23331560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Chtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2013&pages=187-197&author=M.+L.+Gavalaauthor=E.+A.+B.+Kellyauthor=S.+Esnaultauthor=S.+Kukrejaauthor=M.+D.+Evansauthor=P.+J.+Berticsauthor=G.+L.+Chuppauthor=N.+N.+Jarjour&title=Segmental+allergen+challenge+enhances+chitinase+activity+and+levels+of+CCL18+in+mild+atopic+asthma&doi=10.1111%2Fcea.12032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Segmental allergen challenge enhances chitinase activity and levels of CCL18 in mild atopic asthma</span></div><div class="casAuthors">Gavala, M. L.; Kelly, E. A. B.; Esnault, S.; Kukreja, S.; Evans, M. D.; Bertics, P. J.; Chupp, G. L.; Jarjour, N. N.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Allergy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-197</span>CODEN:
                <span class="NLM_cas:coden">CLEAEN</span>;
        ISSN:<span class="NLM_cas:issn">0954-7894</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Background : Allergic airway inflammation contributes to the airway remodelling that has been linked to increased obstruction and morbidity in asthma.  However, the mechanisms by which allergens contribute to airway remodelling in humans are not fully established.  CCL18, chitotriosidase (CHIT1) and YKL-40 are readily detectable in the lungs and contribute to remodelling in other fibrotic diseases, but their involvement in allergic asthma is unclear.  Objective : We hypothesized that CCL18, YKL-40 and CHIT1 bioactivity are enhanced in allergic asthma subjects after segmental allergen challenge and are related to increased pro-fibrotic and Th2-assocd. mediators in the lungs.  Methods : Levels of CCL18 and YKL-40 protein and chitotriosidase (CHIT1) bioactivity in bronchoalveolar lavage (BAL) fluid, as well as CCL18, YKL-40 and CHIT1 mRNA levels in BAL cells were evaluated in patients with asthma at baseline and 48 h after segmental allergen challenge.  We also examd. the correlation between CCL18 and YKL-40 levels and CHIT1 activity with the levels of other pro-fibrotic factors and chemokines previously shown to be up-regulated after allergen challenge.  Results : Chitotriosidase activity and YKL-40 and CCL18 levels were elevated after segmental allergen challenge and these levels correlated with those of other pro-fibrotic factors, T cell chemokines, and inflammatory cells after allergen challenge.  CCL18 and YKL-40 mRNA levels also increased in BAL cells after allergen challenge.  Conclusions and Clin. Relevance : Our results suggest that CCL18 and YKL-40 levels and CHIT1 activity are enhanced in allergic airway inflammation and thus may contribute to airway remodelling in asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEyjeaYu0j0LVg90H21EOLACvtfcHk0lhcJ01uzyLQ1w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Chtrw%253D&md5=815e295065739fa0ff69742683abd847</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcea.12032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcea.12032%26sid%3Dliteratum%253Aachs%26aulast%3DGavala%26aufirst%3DM.%2BL.%26aulast%3DKelly%26aufirst%3DE.%2BA.%2BB.%26aulast%3DEsnault%26aufirst%3DS.%26aulast%3DKukreja%26aufirst%3DS.%26aulast%3DEvans%26aufirst%3DM.%2BD.%26aulast%3DBertics%26aufirst%3DP.%2BJ.%26aulast%3DChupp%26aufirst%3DG.%2BL.%26aulast%3DJarjour%26aufirst%3DN.%2BN.%26atitle%3DSegmental%2520allergen%2520challenge%2520enhances%2520chitinase%2520activity%2520and%2520levels%2520of%2520CCL18%2520in%2520mild%2520atopic%2520asthma%26jtitle%3DClin.%2520Exp.%2520Allergy.%26date%3D2013%26volume%3D43%26spage%3D187%26epage%3D197%26doi%3D10.1111%2Fcea.12032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artieda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cenarro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GaÃ±Ã¡n, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">JericÃ³, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalvo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casado, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitoria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puzo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PocovÃ­, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civeira, F.</span></span> <span> </span><span class="NLM_article-title">Serum chitotriosidase activity is increased in subjects with atherosclerosis disease</span>. <i>Arterioscler., Thromb., Vasc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1645</span>â <span class="NLM_lpage">1652</span>, <span class="refDoi">Â DOI: 10.1161/01.ATV.0000089329.09061.07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1161%2F01.ATV.0000089329.09061.07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=12893688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntVWrt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=1645-1652&author=M.+Artiedaauthor=A.+Cenarroauthor=A.+Ga%C3%B1%C3%A1nauthor=I.+Jeric%C3%B3author=C.+Gonzalvoauthor=J.+M.+Casadoauthor=I.+Vitoriaauthor=J.+Puzoauthor=M.+Pocov%C3%ADauthor=F.+Civeira&title=Serum+chitotriosidase+activity+is+increased+in+subjects+with+atherosclerosis+disease&doi=10.1161%2F01.ATV.0000089329.09061.07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Serum chitotriosidase activity is increased in subjects With atherosclerosis disease</span></div><div class="casAuthors">Artieda, Marta; Cenarro, Ana; Ganan, Alberto; Jerico, Ivonne; Gonzalvo, Carmen; Casado, Juan M.; Vitoria, Isabel; Puzo, Jose; Pocovi, Miguel; Civeira, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1645-1652</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objective- This study was undertaken to analyze the relation between serum activity of chitotriosidase enzyme, a protein synthesized exclusively by activated macrophages, and atherosclerotic lesion extent in subjects with atherothrombotic stroke (ATS) and in subjects with ischemic heart disease (IHD).  Methods and Results- We assayed the serum chitotriosidase activity and a common chitotriosidase gene polymorphism that causes deficiency in chitotriosidase activity in 3 Spanish populations, ATS (n=153), IHD (n=124), and control (n=148) subjects.  Statistical differences were found in serum chitotriosidase activity between ATS (88.1Â±4.6 nmol/mL Â· h, P<0.0001) and IHD subjects (79.0Â±6.3, P=0.002) vs. control group (70.9Â±5.2).  These obsd. differences were not attributable to a distinct allelic or genotype distribution.  The extension of the atherosclerotic lesion in carotids of ATS subjects was measured by duplex sonog.  Chitotriosidase activities were 66.9Â±9.6, 88.7Â±8.3, and 107.7Â±11.8 for subjects with carotid stenosis â¤30%, 31% to 60%, and >60%, resp.  Statistical differences were obsd. between subjects with major and intermediate stenosis grade compared with subjects with minor stenosis, P=0.005 and P=0.016, resp.  Conclusions- Serum chitotriosidase activity is significantly increased in individuals suffering from atherosclerosis disease and is related to the severity of the atherosclerotic lesion, suggesting a possible role as atherosclerotic extent marker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjfaHPeIda2LVg90H21EOLACvtfcHk0lhcJ01uzyLQ1w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntVWrt7o%253D&md5=f060955236f7d0f0c317c335f521eb6f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1161%2F01.ATV.0000089329.09061.07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.ATV.0000089329.09061.07%26sid%3Dliteratum%253Aachs%26aulast%3DArtieda%26aufirst%3DM.%26aulast%3DCenarro%26aufirst%3DA.%26aulast%3DGa%25C3%25B1%25C3%25A1n%26aufirst%3DA.%26aulast%3DJeric%25C3%25B3%26aufirst%3DI.%26aulast%3DGonzalvo%26aufirst%3DC.%26aulast%3DCasado%26aufirst%3DJ.%2BM.%26aulast%3DVitoria%26aufirst%3DI.%26aulast%3DPuzo%26aufirst%3DJ.%26aulast%3DPocov%25C3%25AD%26aufirst%3DM.%26aulast%3DCiveira%26aufirst%3DF.%26atitle%3DSerum%2520chitotriosidase%2520activity%2520is%2520increased%2520in%2520subjects%2520with%2520atherosclerosis%2520disease%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D1645%26epage%3D1652%26doi%3D10.1161%2F01.ATV.0000089329.09061.07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollak, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alberts, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M.</span></span> <span> </span><span class="NLM_article-title">Plasma chitotriosidase and CCL18 as surrogate markers for granulomatous macrophages in sarcoidosis</span>. <i>Clin. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">31</span>â <span class="NLM_lpage">36</span>, <span class="refDoi">Â DOI: 10.1016/j.cca.2009.09.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.cca.2009.09.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19808030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGrtbzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=411&publication_year=2010&pages=31-36&author=R.+G.+Bootauthor=C.+E.+M.+Hollakauthor=M.+Verhoekauthor=C.+Albertsauthor=R.+E.+Jonkersauthor=J.+M.+Aerts&title=Plasma+chitotriosidase+and+CCL18+as+surrogate+markers+for+granulomatous+macrophages+in+sarcoidosis&doi=10.1016%2Fj.cca.2009.09.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Plasma chitotriosidase and CCL18 as surrogate markers for granulomatous macrophages in sarcoidosis</span></div><div class="casAuthors">Boot, Rolf G.; Hollak, Carla E. M.; Verhoek, Marri; Alberts, C.; Jonkers, Rene E.; Aerts, Johannes M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinica Chimica Acta</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">31-36</span>CODEN:
                <span class="NLM_cas:coden">CCATAR</span>;
        ISSN:<span class="NLM_cas:issn">0009-8981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accumulation of macrophages in multiple organs is a common feature of sarcoidosis and Gaucher disease.  The vast no. of storage macrophages in Gaucher patients has facilitated the discovery of suitable plasma markers like chitotriosidase and CCL18.  Plasma specimens of patients with sarcoidosis were examd. on chitotriosidase activity and CCL18 protein levels.  Results: Chitotriosidase was markedly increased, being on av. 13.7-fold elevated (range: 1.1-43.3).  The sensitivity of demonstrating sarcoidosis using plasma chitotriosidase values exceeded that using serum angiotensin-converting enzyme values.  A 3.5-fold (range: 1-15) increase in CCL18 was also obsd.  The relative changes in chitotriosidase and CCL18 during the course of disease closely mimicked each other, suggesting an identical cellular source.  In situ hybridization anal. confirmed massive prodn. of chitotriosidase by sarcoid macrophages.  The increase in plasma chitotriosidase correlated with the stage of disease, being highest in active sarcoidosis with extrapulmonary involvement.  Therapy with steroids resulted in clear redn. of plasma chitotriosidase and CCL18 and relapse of disease activity was preceded by increases in these parameters.  Sarcoid macrophages secrete high quantities of chitotriosidase and CCL18.  Detn. of plasma chitotriosidase and CCL18 may be useful to monitor changes in granulomatous macrophages during the course of sarcoidosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbph6FD-qdc7Vg90H21EOLACvtfcHk0lhoPkaV9JL1MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGrtbzN&md5=515b8f209c26a953d816a6f1ffe58f0a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cca.2009.09.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cca.2009.09.034%26sid%3Dliteratum%253Aachs%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DHollak%26aufirst%3DC.%2BE.%2BM.%26aulast%3DVerhoek%26aufirst%3DM.%26aulast%3DAlberts%26aufirst%3DC.%26aulast%3DJonkers%26aufirst%3DR.%2BE.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%26atitle%3DPlasma%2520chitotriosidase%2520and%2520CCL18%2520as%2520surrogate%2520markers%2520for%2520granulomatous%2520macrophages%2520in%2520sarcoidosis%26jtitle%3DClin.%2520Chim.%2520Acta%26date%3D2010%26volume%3D411%26spage%3D31%26epage%3D36%26doi%3D10.1016%2Fj.cca.2009.09.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malaguarnera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Rosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambito, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dellâOmbra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicoletti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malaguarnera, M.</span></span> <span> </span><span class="NLM_article-title">Chitotriosidase gene expression in Kupffer cells from patients with non-alcoholic fatty liver disease</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1313</span>â <span class="NLM_lpage">1320</span>, <span class="refDoi">Â DOI: 10.1136/gut.2005.075697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1136%2Fgut.2005.075697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16825325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVehurfP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2006&pages=1313-1320&author=L.+Malaguarneraauthor=M.+Di+Rosaauthor=A.+M.+Zambitoauthor=N.+dell%E2%80%99Ombraauthor=F.+Nicolettiauthor=M.+Malaguarnera&title=Chitotriosidase+gene+expression+in+Kupffer+cells+from+patients+with+non-alcoholic+fatty+liver+disease&doi=10.1136%2Fgut.2005.075697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Chitotriosidase gene expression in Kupffer cells from patients with non-alcoholic fatty liver disease</span></div><div class="casAuthors">Malaguarnera, L.; Di Rosa, M.; Zambito, A. M.; dell'Ombra, N.; Nicoletti, F.; Malaguarnera, M.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1313-1320</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Non-alc. steatohepatitis (NASH) is a clinicopathol. condition characterized by a necroinflammatory disorder with fatty infiltration of the hepatocytes.  The mol. mechanisms involved in the anomalous behavior of liver cells have only partially been detd.  Human chitotriosidase (Chit) is a chitinolytic enzyme mainly produced by activated macrophages.  The aim of this study was to investigate the expression of the chitinase-like gene in Kupffer cells, to det. how chitotriosidase may be implicated in the progression from uncomplicated steatosis to steatohepatitis with progressive fibrosis.  Seventy-five subjects were studied: 40 with NASH, 20 with simple steatosis, and 15 normal controls.  Kupffer cells obtained from liver biopsies were used to detect CHIT expression, superoxide anion (O2-), lipid peroxidn., and tumor necrosis factor Î± (TNFÎ±) and ferritin levels.  CHIT expression differed markedly in livers from normal controls and in those from patients with simple steatosis or non-alc. steatohepatitis.  A significant correlation between mRNA CHIT and O2-, lipid peroxidn., TNFÎ±, and ferritin levels was obsd. in both NASH and simple steatosis.  Human Kupffer cells in NASH patients overproduce chitotriosidase.  At the highest levels of prodn., this enzyme may play a role in increasing the risk for a poor outcome in steatohepatitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZcSFMhHChYrVg90H21EOLACvtfcHk0lhoPkaV9JL1MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVehurfP&md5=533961c0afd63a24cf44dd6ddb62e0f5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1136%2Fgut.2005.075697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.2005.075697%26sid%3Dliteratum%253Aachs%26aulast%3DMalaguarnera%26aufirst%3DL.%26aulast%3DDi%2BRosa%26aufirst%3DM.%26aulast%3DZambito%26aufirst%3DA.%2BM.%26aulast%3Ddell%25E2%2580%2599Ombra%26aufirst%3DN.%26aulast%3DNicoletti%26aufirst%3DF.%26aulast%3DMalaguarnera%26aufirst%3DM.%26atitle%3DChitotriosidase%2520gene%2520expression%2520in%2520Kupffer%2520cells%2520from%2520patients%2520with%2520non-alcoholic%2520fatty%2520liver%2520disease%26jtitle%3DGut%26date%3D2006%26volume%3D55%26spage%3D1313%26epage%3D1320%26doi%3D10.1136%2Fgut.2005.075697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommaart, E. F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swart, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghauharali-van der Vlugt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bijl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Place, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M. F. G.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel acidic mammalian chitinase distinct from chitotriosidase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">6770</span>â <span class="NLM_lpage">6778</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M009886200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M009886200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=11085997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSgsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=6770-6778&author=R.+G.+Bootauthor=E.+F.+C.+Blommaartauthor=E.+Swartauthor=K.+Ghauharali-van+der+Vlugtauthor=N.+Bijlauthor=C.+Moeauthor=A.+Placeauthor=J.+M.+F.+G.+Aerts&title=Identification+of+a+novel+acidic+mammalian+chitinase+distinct+from+chitotriosidase&doi=10.1074%2Fjbc.M009886200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel acidic mammalian chitinase distinct from chitotriosidase</span></div><div class="casAuthors">Boot, Rolf G.; Blommaart, Edward F. C.; Swart, Erwin; Ghuharali-Van der Vlugt, Karen; Bijl, Nora; Moe, Cassandra; Place, Allen; Aerts, Johannes M. F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6770-6778</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chitinases are ubiquitous chitin-fragmenting hydrolases.  Recently we discovered the first human chitinase, named chitotriosidase, that is specifically expressed by phagocytes.  We here report the identification, purifn., and subsequent cloning of a second mammalian chitinase.  This enzyme is characterized by an acidic isoelec. point and therefore named acidic mammalian chitinase (AMCase).  In rodents and man the enzyme is relatively abundant in the gastrointestinal tract and is found to a lesser extent in the lung.  Like chitotriosidase, AMCase is synthesized as a 50-kDa protein contg. a 39-kDa N-terminal catalytic domain, a hinge region, and a C-terminal chitin-binding domain.  In contrast to chitotriosidase, the enzyme is extremely acid stable and shows a distinct second pH optimum around pH 2.  AMCase is capable of cleaving artificial chitin-like substrates as well as crab shell chitin and chitin as present in the fungal cell wall.  Our study has revealed the existence of a chitinolytic enzyme in the gastrointestinal tract and lung that may play a role in digestion and/or defense.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK9zza8yUykLVg90H21EOLACvtfcHk0lic-Yxws0AIbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslSgsrs%253D&md5=cb1d31ccbf699384de3cb00105259993</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M009886200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M009886200%26sid%3Dliteratum%253Aachs%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DBlommaart%26aufirst%3DE.%2BF.%2BC.%26aulast%3DSwart%26aufirst%3DE.%26aulast%3DGhauharali-van%2Bder%2BVlugt%26aufirst%3DK.%26aulast%3DBijl%26aufirst%3DN.%26aulast%3DMoe%26aufirst%3DC.%26aulast%3DPlace%26aufirst%3DA.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%2BF.%2BG.%26atitle%3DIdentification%2520of%2520a%2520novel%2520acidic%2520mammalian%2520chitinase%2520distinct%2520from%2520chitotriosidase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D6770%26epage%3D6778%26doi%3D10.1074%2Fjbc.M009886200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of acidic mammalian chitinase by RNA interference suppresses ovalbumin-sensitized allergic asthma</span>. <i>Hum. Gene Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1597</span>â <span class="NLM_lpage">1606</span>, <span class="refDoi">Â DOI: 10.1089/hum.2008.092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1089%2Fhum.2008.092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19548841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1WhtL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=1597-1606&author=C.+J.+Yangauthor=Y.+K.+Liuauthor=C.+L.+Liuauthor=C.+N.+Shenauthor=M.+L.+Kuoauthor=C.+C.+Suauthor=C.+P.+Tsengauthor=T.+C.+Yenauthor=C.+R.+Shen&title=Inhibition+of+acidic+mammalian+chitinase+by+RNA+interference+suppresses+ovalbumin-sensitized+allergic+asthma&doi=10.1089%2Fhum.2008.092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of acidic mammalian chitinase by RNA interference suppresses ovalbumin-sensitized allergic asthma</span></div><div class="casAuthors">Yang, Ching-Jen; Liu, Yu-Kuo; Liu, Chao-Lin; Shen, Chia-Ning; Kuo, Ming-Ling; Su, Chien-Chang; Tseng, Ching-Ping; Yen, Tzu-Chen; Shen, Chia-Rui</div><div class="citationInfo"><span class="NLM_cas:title">Human Gene Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1597-1606</span>CODEN:
                <span class="NLM_cas:coden">HGTHE3</span>;
        ISSN:<span class="NLM_cas:issn">1043-0342</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Asthma, a chronic helper T cell type 2-mediated inflammatory disease, is characterized by airway hyperresponsiveness and inflammation.  Growing evidence suggests that increased expression of acidic mammalian chitinase (AMCase) may play a role in the pathogenesis of asthma.  In the present study, we sought to develop an RNA interference approach to suppress allergic asthma in mice through silencing of AMCase expression.  Mice sensitized with ovalbumin (OVA) were intratracheally administered a recombinant adeno-assocd. virus expressing short hairpin RNA (rAAV-shRNA) against AMCase.  In OVA-sensitized mice, the development of allergic symptoms was significantly assocd. with elevated AMCase expression.  After administration of rAAV-shRNA, there was a significant redn. of AMCase expression in the lung and in bronchoalveolar lavage fluid (BALF) cells of sensitized mice.  Sensitized mice receiving rAAV-shRNA showed a significant improvement in allergic symptoms, including airway hyperresponsiveness (AHR), eosinophil infiltration, eotaxin, interleukin-13 secretion in BALF, and serum OVA-specific IgE level.  Our data suggest the hyperexpression of AMCase in asthma can be suppressed by rAAV-mediated shRNA.  Silencing AMCase expression by shRNA may be a promising therapeutic strategy in asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyU9pcjbg2rLVg90H21EOLACvtfcHk0lic-Yxws0AIbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1WhtL%252FI&md5=ad60a3b1cdd8089e340a15455e19747c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1089%2Fhum.2008.092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fhum.2008.092%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%2BK.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26aulast%3DShen%26aufirst%3DC.%2BN.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26aulast%3DSu%26aufirst%3DC.%2BC.%26aulast%3DTseng%26aufirst%3DC.%2BP.%26aulast%3DYen%26aufirst%3DT.%2BC.%26aulast%3DShen%26aufirst%3DC.%2BR.%26atitle%3DInhibition%2520of%2520acidic%2520mammalian%2520chitinase%2520by%2520RNA%2520interference%2520suppresses%2520ovalbumin-sensitized%2520allergic%2520asthma%26jtitle%3DHum.%2520Gene%2520Ther.%26date%3D2009%26volume%3D20%26spage%3D1597%26epage%3D1606%26doi%3D10.1089%2Fhum.2008.092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. A.</span></span> <span> </span><span class="NLM_article-title">Acidic mammalian chitinase in asthmatic Th2 inflammation and IL-13 pathway activation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1678</span>â <span class="NLM_lpage">1682</span>, <span class="refDoi">Â DOI: 10.1126/science.1095336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1126%2Fscience.1095336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15192232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1Cgsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1678-1682&author=Z.+Zhuauthor=T.+Zhengauthor=R.+J.+Homerauthor=Y.+K.+Kimauthor=N.+Y.+Chenauthor=L.+Cohnauthor=Q.+Hamidauthor=J.+A.+Elias&title=Acidic+mammalian+chitinase+in+asthmatic+Th2+inflammation+and+IL-13+pathway+activation&doi=10.1126%2Fscience.1095336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic Mammalian Chitinase in Asthmatic Th2 Inflammation and IL-13 Pathway Activation</span></div><div class="casAuthors">Zhu, Zhou; Zheng, Tao; Homer, Robert J.; Kim, Yoon-Keun; Chen, Ning Yuan; Cohn, Lauren; Hamid, Qutayba; Elias, Jack A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5677</span>),
    <span class="NLM_cas:pages">1678-1682</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Chitin is a surface component of parasites and insects, and chitinases are induced in lower life forms during infections with these agents.  Although chitin itself does not exist in humans, chitinases are present in the human genome.  We show here that acidic mammalian chitinase (AMCase) is induced via a T helper-2 (Th2)-specific, interleukin-13 (IL-13)-mediated pathway in epithelial cells and macrophages in an aeroallergen asthma model and expressed in exaggerated quantities in human asthma.  AMCase neutralization ameliorated Th2 inflammation and airway hyperresponsiveness, in part by inhibiting IL-13 pathway activation and chemokine induction.  AMCase may thus be an important mediator of IL-13-induced responses in Th2-dominated disorders such as asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr11ubUlNbNs7Vg90H21EOLACvtfcHk0lic-Yxws0AIbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1Cgsr0%253D&md5=4358d813c473ef0d9b788a9dceb0f7f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095336%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DHomer%26aufirst%3DR.%2BJ.%26aulast%3DKim%26aufirst%3DY.%2BK.%26aulast%3DChen%26aufirst%3DN.%2BY.%26aulast%3DCohn%26aufirst%3DL.%26aulast%3DHamid%26aufirst%3DQ.%26aulast%3DElias%26aufirst%3DJ.%2BA.%26atitle%3DAcidic%2520mammalian%2520chitinase%2520in%2520asthmatic%2520Th2%2520inflammation%2520and%2520IL-13%2520pathway%2520activation%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1678%26epage%3D1682%26doi%3D10.1126%2Fscience.1095336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bierbaum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deichmann, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahn, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzmann, A.</span></span> <span> </span><span class="NLM_article-title">Polymorphisms and haplotypes of acid mammalian chitinase are associated with bronchial asthma</span>. <i>Am. J. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">1505</span>â <span class="NLM_lpage">1509</span>, <span class="refDoi">Â DOI: 10.1164/rccm.200506-890OC</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1164%2Frccm.200506-890OC" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16179638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BD2MnlsFehtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2005&pages=1505-1509&author=S.+Bierbaumauthor=R.+Nickelauthor=A.+Kochauthor=S.+Lauauthor=K.+A.+Deichmannauthor=U.+Wahnauthor=A.+Superti-Furgaauthor=A.+Heinzmann&title=Polymorphisms+and+haplotypes+of+acid+mammalian+chitinase+are+associated+with+bronchial+asthma&doi=10.1164%2Frccm.200506-890OC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Polymorphisms and haplotypes of acid mammalian chitinase are associated with bronchial asthma</span></div><div class="casAuthors">Bierbaum Sibylle; Nickel Renate; Koch Anja; Lau Susanne; Deichmann Klaus A; Wahn Ulrich; Superti-Furga Andrea; Heinzmann Andrea</div><div class="citationInfo"><span class="NLM_cas:title">American journal of respiratory and critical care medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1505-9</span>
        ISSN:<span class="NLM_cas:issn">1073-449X</span>.
    </div><div class="casAbstract">RATIONALE:  Chitinases are enzymes that cleave chitin, a polysaccharide contained in many parasites of humans.  Recent studies in mouse models of bronchial asthma have shown that acid mammalian chitinase (AMCase) is involved in the pathophysiology of asthma.  It acts downstream of interleukin-13; inhibition of AMCase leads to an abrogated T-helper cell 2 inflammation, less bronchial hyperreactivity, and fewer eosinophils.  OBJECTIVES:  The aim of this study was to identify common genetic variants in human AMCase and to use them to test for association of AMCase with pediatric asthma.  METHODS:  By sequencing the promotor region and all 11 exons on 30 individuals, 12 high-frequency polymorphisms were identified.  Genotyping of six variants in exons and one promotor polymorphism was performed on the following populations by means of restriction fragment length polymorphisms: 322 children with asthma, 270 randomly chosen adult controls, and a pediatric control population consisting of 565 children who, at age 10 yr, had never wheezed and never been diagnosed having asthma.  MEASUREMENTS AND MAIN RESULTS:  We identified three known and two new amino acid variants.  Analyses by the Armitage's trend test using both control populations showed association of the newly identified variant K17R and the nearby noncoding polymorphism rs3818822 with asthma (p = 0.0031 and p = 0.0003, respectively).  In addition, haplotype analyses revealed strong association of haplotypes with the disease (asthma population vs. pediatric control subjects, p < 10(-10)).  CONCLUSIONS:  This newly described association between AMCase polymorphisms and asthma adds further evidence supporting the involvement of AMCase in the development of asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ99nfQi50cwHWEEzDrymxmfW6udTcc2ebEfYxfzfvp1Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnlsFehtw%253D%253D&md5=d4f7c5393196e655ac38b019b2481b84</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1164%2Frccm.200506-890OC&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1164%252Frccm.200506-890OC%26sid%3Dliteratum%253Aachs%26aulast%3DBierbaum%26aufirst%3DS.%26aulast%3DNickel%26aufirst%3DR.%26aulast%3DKoch%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DS.%26aulast%3DDeichmann%26aufirst%3DK.%2BA.%26aulast%3DWahn%26aufirst%3DU.%26aulast%3DSuperti-Furga%26aufirst%3DA.%26aulast%3DHeinzmann%26aufirst%3DA.%26atitle%3DPolymorphisms%2520and%2520haplotypes%2520of%2520acid%2520mammalian%2520chitinase%2520are%2520associated%2520with%2520bronchial%2520asthma%26jtitle%3DAm.%2520J.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2005%26volume%3D172%26spage%3D1505%26epage%3D1509%26doi%3D10.1164%2Frccm.200506-890OC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. L.</span></span> <span> </span><span class="NLM_article-title">The correlation between chitin and acidic mammalian chitinase in animal models of allergic asthma</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">27371</span>â <span class="NLM_lpage">27377</span>, <span class="refDoi">Â DOI: 10.3390/ijms161126033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fijms161126033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=26580611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2ns7vM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=27371-27377&author=C.+R.+Shenauthor=H.+H.+Juangauthor=H.+S.+Chenauthor=C.+J.+Yangauthor=C.+J.+Wuauthor=M.+H.+Leeauthor=Y.+S.+Hwangauthor=M.+L.+Kuoauthor=Y.+S.+Chenauthor=J.+K.+Chenauthor=C.+L.+Liu&title=The+correlation+between+chitin+and+acidic+mammalian+chitinase+in+animal+models+of+allergic+asthma&doi=10.3390%2Fijms161126033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The correlation between chitin and acidic mammalian chitinase in animal models of allergic asthma</span></div><div class="casAuthors">Shen, Chia-Rui; Juang, Horng-Heng; Chen, Hui-Shan; Yang, Ching-Jen; Wu, Chia-Jen; Lee, Meng-Hua; Hwang, Yih-Shiou; Kuo, Ming-Ling; Chen, Ya-Shan; Chen, Jeen-Kuan; Liu, Chao-Lin</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">27371-27377</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Asthma is the result of chronic inflammation of the airways which subsequently results in airway hyper-responsiveness and airflow obstruction.  It has been shown that an elicited expression of acidic mammalian chitinase (AMCase) may be involved in the pathogenesis of asthma.  Our recent study has demonstrated that the specific suppression of elevated AMCase leads to reduced eosinophilia and Th2-mediated immune responses in an ovalbumin (OVA)-sensitized mouse model of allergic asthma.  In the current study, we show that the elicited expression of AMCase in the lung tissues of both ovalbumin- and Der P2-induced allergic asthma mouse models.  The effects of allergic mediated mols. on AMCase expression were evaluated by utilizing promoter assay in the lung cells.  In fact, the exposure of chitin, a polymd. sugar and the fundamental component of the major allergen mite and several of the inflammatory mediators, showed significant enhancement on AMCase expression.  Such obtained results contribute to the basis of developing a promising therapeutic strategy for asthma by silencing AMCase expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWk_pMAbsLTbVg90H21EOLACvtfcHk0lieg5YpgN9YoQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2ns7vM&md5=2ef2e141501a7e3fa6c5d29f07933ae0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fijms161126033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms161126033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.%2BR.%26aulast%3DJuang%26aufirst%3DH.%2BH.%26aulast%3DChen%26aufirst%3DH.%2BS.%26aulast%3DYang%26aufirst%3DC.%2BJ.%26aulast%3DWu%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DM.%2BH.%26aulast%3DHwang%26aufirst%3DY.%2BS.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BS.%26aulast%3DChen%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26atitle%3DThe%2520correlation%2520between%2520chitin%2520and%2520acidic%2520mammalian%2520chitinase%2520in%2520animal%2520models%2520of%2520allergic%2520asthma%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2015%26volume%3D16%26spage%3D27371%26epage%3D27377%26doi%3D10.3390%2Fijms161126033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maizels, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. E.</span></span> <span> </span><span class="NLM_article-title">Analyzing airway inflammation with chemical biology: dissection of acidic mammalian chitinase function with a selective drug-like inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">569</span>â <span class="NLM_lpage">579</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21609838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslOksL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=569-579&author=T.+E.+Sutherlandauthor=O.+A.+Andersenauthor=M.+Betouauthor=I.+M.+Egglestonauthor=R.+M.+Maizelsauthor=D.+van+Aaltenauthor=J.+E.+Allen&title=Analyzing+airway+inflammation+with+chemical+biology%3A+dissection+of+acidic+mammalian+chitinase+function+with+a+selective+drug-like+inhibitor&doi=10.1016%2Fj.chembiol.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing Airway Inflammation with Chemical Biology: Dissection of Acidic Mammalian Chitinase Function with a Selective Drug-like Inhibitor</span></div><div class="casAuthors">Sutherland, Tara E.; Andersen, Ole A.; Betou, Marie; Eggleston, Ian M.; Maizels, Rick M.; van Aalten, Daan; Allen, Judith E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">569-579</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is produced in the lung during allergic inflammation and asthma, and inhibition of enzymic activity has been considered as a therapeutic strategy.  However, most chitinase inhibitors are nonselective, addnl. inhibiting chitotriosidase activity.  Here, we describe bisdionin F, a competitive AMCase inhibitor with 20-fold selectivity for AMCase over chitotriosidase, designed by utilizing the AMCase crystal structure and dicaffeine scaffold.  In a murine model of allergic inflammation, bisdionin F-treatment attenuated chitinase activity and alleviated the primary features of allergic inflammation including eosinophilia.  However, selective AMCase inhibition by bisdionin F also caused dramatic and unexpected neutrophilia in the lungs.  This class of inhibitor will be a powerful tool to dissect the functions of mammalian chitinases in disease and represents a synthetically accessible scaffold to optimize inhibitory properties in terms of airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlYJOOAc9ksbVg90H21EOLACvtfcHk0lieg5YpgN9YoQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslOksL4%253D&md5=3b082d80515694618da2dd966c25a6f9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DT.%2BE.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DBetou%26aufirst%3DM.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3DMaizels%26aufirst%3DR.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%26aulast%3DAllen%26aufirst%3DJ.%2BE.%26atitle%3DAnalyzing%2520airway%2520inflammation%2520with%2520chemical%2520biology%253A%2520dissection%2520of%2520acidic%2520mammalian%2520chitinase%2520function%2520with%2520a%2520selective%2520drug-like%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D569%26epage%3D579%26doi%3D10.1016%2Fj.chembiol.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguchi-Tsuda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kan-O, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, Y.</span></span> <span> </span><span class="NLM_article-title">Demethylallosamidin, a chitinase inhibitor, suppresses airway inflammation and hyperresponsiveness</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2009.09.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bbrc.2009.09.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19782048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yltLbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=103-108&author=T.+Matsumotoauthor=H.+Inoueauthor=Y.+Satoauthor=Y.+Kitaauthor=T.+Nakanoauthor=N.+Nodaauthor=M.+Eguchi-Tsudaauthor=A.+Moriwakiauthor=K.+Kan-Oauthor=K.+Matsumotoauthor=T.+Shimizuauthor=H.+Nagasawaauthor=S.+Sakudaauthor=Y.+Nakanishi&title=Demethylallosamidin%2C+a+chitinase+inhibitor%2C+suppresses+airway+inflammation+and+hyperresponsiveness&doi=10.1016%2Fj.bbrc.2009.09.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylallosamidin, a chitinase inhibitor, suppresses airway inflammation and hyperresponsiveness</span></div><div class="casAuthors">Matsumoto, Takafumi; Inoue, Hiromasa; Sato, Yosuke; Kita, Yoshihiro; Nakano, Takako; Noda, Naotaka; Eguchi-Tsuda, Miyuki; Moriwaki, Atsushi; Kan-o, Keiko; Matsumoto, Koichiro; Shimizu, Takao; Nagasawa, Hiromichi; Sakuda, Shohei; Nakanishi, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase is upregulated in response to allergen exposure in the lung.  We investigated the effects of chitinase inhibitors, allosamidin (Allo) and demethylallosamidin (Dma), on asthmatic responses.  Mice were subjected to IL-13 instillation into the airways or to ovalbumin sensitization plus exposure with or without treatment of Allo or Dma.  Airway hyperresponsiveness (AHR) and inflammation were evaluated.  Allo and Dma attenuated airway eosinophilia and the upregulation of eotaxin after IL-13 instillation, while Dma, but not Allo, suppressed AHR in IL-13-induced asthma.  Allo or Dma suppressed the elevated chitinase activity in BAL fluids after IL-13 to similar levels.  The bronchoprotective PGE2 levels in BAL fluids were elevated after IL-13 instillation.  Allo, but not Dma, suppressed the overprodn. of PGE2 and the expression of COX-2 and PGE synthase-1 induced by IL-13.  In ovalbumin-induced asthma, Dma suppressed AHR more strongly than Allo.  These findings suggest that Dma attenuates asthmatic responses induced by IL-13 without affecting PGE2 synthesis.  Dma may have potential as therapeutic agents for asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xINGL5MfG7Vg90H21EOLACvtfcHk0lj6zWygxbGrDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yltLbP&md5=f72224168fbb3362c5200a7c41ab53a6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2009.09.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2009.09.075%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DN.%26aulast%3DEguchi-Tsuda%26aufirst%3DM.%26aulast%3DMoriwaki%26aufirst%3DA.%26aulast%3DKan-O%26aufirst%3DK.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DShimizu%26aufirst%3DT.%26aulast%3DNagasawa%26aufirst%3DH.%26aulast%3DSakuda%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DY.%26atitle%3DDemethylallosamidin%252C%2520a%2520chitinase%2520inhibitor%252C%2520suppresses%2520airway%2520inflammation%2520and%2520hyperresponsiveness%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2009%26volume%3D390%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.bbrc.2009.09.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olczak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czestkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrzejczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzwonek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklepkiewicz, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboubi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheeler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hungerford, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podjarny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitschler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousido-Siah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fadel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, A.</span></span> <span> </span><span class="NLM_article-title">Targeting acidic mammalian chitinase is effective in animal model of asthma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">695</span>â <span class="NLM_lpage">710</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01051</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01051" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCms7nJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=695-710&author=M.+Mazurauthor=J.+Olczakauthor=S.+Olejniczakauthor=R.+Koralewskiauthor=W.+Czestkowskiauthor=A.+Jedrzejczakauthor=J.+Golabauthor=K.+Dzwonekauthor=B.+Dymekauthor=P.+L.+Sklepkiewiczauthor=A.+Zagozdzonauthor=T.+Noonanauthor=K.+Mahboubiauthor=B.+Conwayauthor=R.+Sheelerauthor=P.+Beckettauthor=W.+M.+Hungerfordauthor=A.+Podjarnyauthor=A.+Mitschlerauthor=A.+Cousido-Siahauthor=F.+Fadelauthor=A.+Golebiowski&title=Targeting+acidic+mammalian+chitinase+is+effective+in+animal+model+of+asthma&doi=10.1021%2Facs.jmedchem.7b01051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Acidic Mammalian chitinase Is Effective in Animal Model of Asthma</span></div><div class="casAuthors">Mazur, Marzena; Olczak, Jacek; Olejniczak, Sylwia; Koralewski, Robert; Czestkowski, Wojciech; Jedrzejczak, Anna; Golab, Jakub; Dzwonek, Karolina; Dymek, Barbara; Sklepkiewicz, Piotr L.; Zagozdzon, Agnieszka; Noonan, Tom; Mahboubi, Keyvan; Conway, Bruce; Sheeler, Ryan; Beckett, Paul; Hungerford, William M.; Podjarny, Alberto; Mitschler, Andre; Cousido-Siah, Alexandra; Fadel, Firas; Golebiowski, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">695-710</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article highlights the authors' work toward the identification of a potent, selective, and efficacious acidic mammalian chitinase (AMCase) inhibitor.  Rational design, guided by x-ray anal. of several inhibitors bound to human chitotriosidase (hCHIT1), led to the identification of compd. 7f (1-(3-amino-1H-1,2,4-triazol-5-yl)-N-(4-chlorophenethyl)-N-isobutylpiperidin-4-amine) as a highly potent AMCase inhibitor (IC50 values of 14 and 19 nM against human and mouse enzyme, resp.) and selective (>150Ã against mCHIT1) with very good PK properties.  This compd. dosed once daily at 30 mg/kg po showed significant anti-inflammatory efficacy in HDM-induced allergic airway inflammation in mice, reducing inflammatory cell influx in the BALF and total IgE concn. in plasma, which correlated with decrease of chitinolytic activity.  Therapeutic efficacy of compd. 7f in the clin. relevant aero-allergen-induced acute asthma model in mice provides a rationale for developing AMCase inhibitor for the treatment of asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n2hO0hKVarVg90H21EOLACvtfcHk0lj6zWygxbGrDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCms7nJ&md5=d93c7add7d10a66f03419a6c7fe59bfc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01051%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DM.%26aulast%3DOlczak%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DS.%26aulast%3DKoralewski%26aufirst%3DR.%26aulast%3DCzestkowski%26aufirst%3DW.%26aulast%3DJedrzejczak%26aufirst%3DA.%26aulast%3DGolab%26aufirst%3DJ.%26aulast%3DDzwonek%26aufirst%3DK.%26aulast%3DDymek%26aufirst%3DB.%26aulast%3DSklepkiewicz%26aufirst%3DP.%2BL.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DNoonan%26aufirst%3DT.%26aulast%3DMahboubi%26aufirst%3DK.%26aulast%3DConway%26aufirst%3DB.%26aulast%3DSheeler%26aufirst%3DR.%26aulast%3DBeckett%26aufirst%3DP.%26aulast%3DHungerford%26aufirst%3DW.%2BM.%26aulast%3DPodjarny%26aufirst%3DA.%26aulast%3DMitschler%26aufirst%3DA.%26aulast%3DCousido-Siah%26aufirst%3DA.%26aulast%3DFadel%26aufirst%3DF.%26aulast%3DGolebiowski%26aufirst%3DA.%26atitle%3DTargeting%2520acidic%2520mammalian%2520chitinase%2520is%2520effective%2520in%2520animal%2520model%2520of%2520asthma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D695%26epage%3D710%26doi%3D10.1021%2Facs.jmedchem.7b01051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isogai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, K.</span></span> <span> </span><span class="NLM_article-title">The structure of allosamidin, a novel insect chitinase inhibitor, produced by <i>Streptomyces Sp</i></span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2475</span>â <span class="NLM_lpage">2478</span>, <span class="refDoi">Â DOI: 10.1016/S0040-4039(00)84560-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0040-4039%2800%2984560-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaL28XkvFCiurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1986&pages=2475-2478&author=S.+Sakudaauthor=A.+Isogaiauthor=S.+Matsumotoauthor=A.+Suzukiauthor=K.+Koseki&title=The+structure+of+allosamidin%2C+a+novel+insect+chitinase+inhibitor%2C+produced+by+Streptomyces+Sp&doi=10.1016%2FS0040-4039%2800%2984560-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of allosamidin, a novel insect chitinase inhibitor, produced by Streptomyces sp</span></div><div class="casAuthors">Sakuda, Shohei; Isogai, Akira; Matsumoto, Shogo; Suzuki, Akinori; Koseki, Koshi</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2475-8</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Allosamidin (I), a novel insect chitinase inhibitor, was isolated from the mycelium of Streptomyces sp. and characterized.  I was a basic pseudotrisaccharide consisting of 2-acetamido-2-deoxy-D-allose(N-acetyl-D-allosamine) and a novel aminocyclitol deriv. termed allosamizoline.  I inhibited the chitinases of silkworm (Bombex mori) and retarded the growth of instar larvae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyNDCRkGaOibVg90H21EOLACvtfcHk0lj6zWygxbGrDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XkvFCiurs%253D&md5=e5df9e4bf6613356b7061435223b2cfd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2984560-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252984560-8%26sid%3Dliteratum%253Aachs%26aulast%3DSakuda%26aufirst%3DS.%26aulast%3DIsogai%26aufirst%3DA.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DKoseki%26aufirst%3DK.%26atitle%3DThe%2520structure%2520of%2520allosamidin%252C%2520a%2520novel%2520insect%2520chitinase%2520inhibitor%252C%2520produced%2520by%2520Streptomyces%2520Sp%26jtitle%3DTetrahedron%2520Lett.%26date%3D1986%26volume%3D27%26spage%3D2475%26epage%3D2478%26doi%3D10.1016%2FS0040-4039%2800%2984560-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J. M. F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodkinson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Specificity and affinity of natural product cyclopentapeptide inhibitors against <i>A. fumigatus</i>, human, and bacterial chitinases</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2004.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2004.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=15664516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=65-76&author=F.+V.+Raoauthor=D.+R.+Houstonauthor=R.+G.+Bootauthor=J.+M.+F.+G.+Aertsauthor=M.+Hodkinsonauthor=D.+J.+Adamsauthor=K.+Shiomiauthor=S.+Omuraauthor=D.+M.+F.+van+Aalten&title=Specificity+and+affinity+of+natural+product+cyclopentapeptide+inhibitors+against+A.+fumigatus%2C+human%2C+and+bacterial+chitinases&doi=10.1016%2Fj.chembiol.2004.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity and Affinity of Natural Product Cyclopentapeptide Inhibitors against A. fumigatus, Human, and Bacterial Chitinases</span></div><div class="casAuthors">Rao, Francesco V.; Houston, Douglas R.; Boot, Rolf G.; Aerts, Johannes M. F. G.; Hodkinson, Michael; Adams, David J.; Shiomi, Kazuro; Omura, Satoshi; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-76</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Family 18 chitinases play key roles in organisms ranging from bacteria to man.  There is a need for specific, potent inhibitors to probe the function of these chitinases in different organisms.  Such mols. could also provide leads for the development of chemotherapeuticals with fungicidal, insecticidal, or anti-inflammatory potential.  Recently, two natural product peptides, argifin and argadin, have been characterized, which structurally mimic chitinase-chitooligosaccharide interactions and inhibit a bacterial chitinase in the nM-mM range.  Here, we show that these inhibitors also act on human and Aspergillus fumigatus chitinases.  The structures of these enzymes in complex with argifin and argadin, together with mutagenesis, fluorescence, and enzymol., reveal that subtle changes in the binding site dramatically affect affinity and selectivity.  The data show that it may be possible to develop specific chitinase inhibitors based on the argifin/argadin scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRk5ZHk2BMa7Vg90H21EOLACvtfcHk0lg66Sdrv0LLWg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsA%253D%253D&md5=5bec80bc8ff6be5f906ee13d4d8fc4b8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DAerts%26aufirst%3DJ.%2BM.%2BF.%2BG.%26aulast%3DHodkinson%26aufirst%3DM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShiomi%26aufirst%3DK.%26aulast%3DOmura%26aufirst%3DS.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DSpecificity%2520and%2520affinity%2520of%2520natural%2520product%2520cyclopentapeptide%2520inhibitors%2520against%2520A.%2520fumigatus%252C%2520human%252C%2520and%2520bacterial%2520chitinases%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D65%26epage%3D76%26doi%3D10.1016%2Fj.chembiol.2004.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunazuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OÌMura, S.</span></span> <span> </span><span class="NLM_article-title">Recent development of two chitinase inhibitors, Argifin and Argadin, produced by soil microorganisms</span>. <i>Proc. Jpn. Acad. Ser. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">85</span>â <span class="NLM_lpage">102</span>, <span class="refDoi">Â DOI: 10.2183/pjab.86.85</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.2183%2Fpjab.86.85" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFOgtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2010&pages=85-102&author=T.+Hiroseauthor=T.+Sunazukaauthor=S.+O%CC%85Mura&title=Recent+development+of+two+chitinase+inhibitors%2C+Argifin+and+Argadin%2C+produced+by+soil+microorganisms&doi=10.2183%2Fpjab.86.85"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Recent development of two chitinase inhibitors, Argifin and Argadin, produced by soil microorganisms</span></div><div class="casAuthors">Hirose, Tomoyasu; Sunazuka, Toshiaki; Omura, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the Japan Academy, Series B: Physical and Biological Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-102</span>CODEN:
                <span class="NLM_cas:coden">PJABDW</span>;
        ISSN:<span class="NLM_cas:issn">0386-2208</span>.
    
            (<span class="NLM_cas:orgname">Nippon Gakushiin</span>)
        </div><div class="casAbstract">A review.  Chitin, the second most abundant polysaccharide in nature, occurs in fungi, some algae and many invertebrates, including insects.  Thus, chitin synthesis and degrdn. could represent specific targets for fungicides and insecticides.  Chitinases hydrolyze chitin into oligomers of N-acetyl-D-glucosamine at key points in the life cycles of organisms, consequently, chitinase inhibitors have become subject of increasing interest.  This review covers the development of two chitinase inhibitors of natural origin, Argifin and Argadin, isolated from the cultured broth of microorganisms in our lab.  In particular, the practical total synthesis of these natural products, the synthesis of lead compds. via computer-aided rational mol. design, and discovery methods that generate only highly-active compds. using a kinetic target(chitinase)-guided synthesis approach (termed in situ click chem.) are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-AuwyNwzeYbVg90H21EOLACvtfcHk0lg66Sdrv0LLWg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFOgtr8%253D&md5=1b74ff18415c88080adcbb027030b51a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2183%2Fpjab.86.85&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2183%252Fpjab.86.85%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DT.%26aulast%3DSunazuka%26aufirst%3DT.%26aulast%3DO%25CC%2585Mura%26aufirst%3DS.%26atitle%3DRecent%2520development%2520of%2520two%2520chitinase%2520inhibitors%252C%2520Argifin%2520and%2520Argadin%252C%2520produced%2520by%2520soil%2520microorganisms%26jtitle%3DProc.%2520Jpn.%2520Acad.%2520Ser.%2520B%26date%3D2010%26volume%3D86%26spage%3D85%26epage%3D102%26doi%3D10.2183%2Fpjab.86.85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span> <span> </span><span class="NLM_article-title">Fully deacetylated chitooligosaccharides act as efficient glycoside hydrolase family 18 chitinase inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">17932</span>â <span class="NLM_lpage">17940</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M114.564534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M114.564534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=24828498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaisrvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=17932-17940&author=L.+Chenauthor=Y.+Zhouauthor=M.+Quauthor=Y.+Zhaoauthor=Q.+Yang&title=Fully+deacetylated+chitooligosaccharides+act+as+efficient+glycoside+hydrolase+family+18+chitinase+inhibitors&doi=10.1074%2Fjbc.M114.564534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fully deacetylated chitooligosaccharides act as efficient glycoside hydrolase family 18 chitinase inhibitors</span></div><div class="casAuthors">Chen, Lei; Zhou, Yong; Qu, Mingbo; Zhao, Yong; Yang, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">17932-17940</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors against chitinases have potential applications as pesticides, fungicides, and antiasthmatics.  Here, we report that a series of fully deacetylated chitooligosaccharides (GlcN)2-7 can act as inhibitors against the insect chitinase OfChtI, the human chitinase HsCht, and the bacterial chitinases SmChiA and SmChiB with IC50 values at micromolar to millimolar levels.  The injection of mixed (GlcN)2-7 into the fifth instar larvae of the insect Ostrinia furnacalis resulted in 85% of the larvae being arrested at the larval stage and death after 10 days, also suggesting that (GlcN)2-7 might inhibit OfChtI in vivo.  Crystal structures of the catalytic domain of OfChtI (OfChtI-CAD) complexed with (GlcN)5,6 were obtained at resolns. of 2.0 Ã.  These structures, together with mutagenesis and thermodn. anal., suggested that the inhibition was strongly related to the interaction between the -1 GlcN residue of the inhibitor and the catalytic Glu148 of the enzyme.  Structure-based comparison showed that the fully deacetylated chitooligosaccharides mimic the substrate chitooligosaccharides by binding to the active cleft.  This work first reports the inhibitory activity and proposed inhibitory mechanism of fully deacetylated chitooligosaccharides.  Because the fully deacetylated chitooligosaccharides can be easily derived from chitin, one of the most abundant materials in nature, this work also provides a platform for developing eco-friendly inhibitors against chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpogLcsC9w3CbVg90H21EOLACvtfcHk0lg66Sdrv0LLWg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaisrvL&md5=6bf68fe4e174f0b0e1fa59545c67d76e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.564534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.564534%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DQ.%26atitle%3DFully%2520deacetylated%2520chitooligosaccharides%2520act%2520as%2520efficient%2520glycoside%2520hydrolase%2520family%252018%2520chitinase%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D17932%26epage%3D17940%26doi%3D10.1074%2Fjbc.M114.564534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Acetazolamide-based fungal chitinase inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">8334</span>â <span class="NLM_lpage">8340</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2010.09.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bmc.2010.09.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21044846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8334-8340&author=A.+W.+Sch%C3%BCttelkopfauthor=L.+Grosauthor=D.+E.+Blairauthor=J.+A.+Frearsonauthor=D.+M.+F.+van+Aaltenauthor=I.+H.+Gilbert&title=Acetazolamide-based+fungal+chitinase+inhibitors&doi=10.1016%2Fj.bmc.2010.09.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Acetazolamide-based fungal chitinase inhibitors</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; Gros, Ludovic; Blair, David E.; Frearson, Julie A.; van Aalten, Daan M. F.; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8334-8340</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chitin is an essential structural component of the fungal cell wall.  Chitinases are thought to be important for fungal cell wall remodelling, and inhibition of these enzymes has been proposed as a potential strategy for development of novel anti-fungals.  The fungal pathogen Aspergillus fumigatus possesses two distinct multi-gene chitinase families.  Here we explore acetazolamide as a chem. scaffold for the inhibition of an A. fumigatus plant-type' chitinase.  A co-crystal structure of AfChiA1 with acetazolamide was used to guide synthesis and screening of acetazolamide analogs that yielded SAR in agreement with these structural data.  Although acetazolamide and its analogs are weak inhibitors of the enzyme, they have a high ligand efficiency and as such are interesting leads for future inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0YAez21-yarVg90H21EOLACvtfcHk0lgLp2U_RflQ0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurjE&md5=6a8ffe1abf068b24fdabef52a72b6a13</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.09.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.09.062%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DGros%26aufirst%3DL.%26aulast%3DBlair%26aufirst%3DD.%2BE.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DAcetazolamide-based%2520fungal%2520chitinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8334%26epage%3D8340%26doi%3D10.1016%2Fj.bmc.2010.09.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Bisdionin C-A rationally designed, submicromolar inhibitor of family 18 chitinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">428</span>â <span class="NLM_lpage">432</span>, <span class="refDoi">Â DOI: 10.1021/ml200008b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200008b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=428-432&author=A.+W.+Sch%C3%BCttelkopfauthor=O.+A.+Andersenauthor=F.+V.+Raoauthor=M.+Allwoodauthor=C.+L.+Rushauthor=I.+M.+Egglestonauthor=D.+M.+F.+van+Aalten&title=Bisdionin+C-A+rationally+designed%2C+submicromolar+inhibitor+of+family+18+chitinases&doi=10.1021%2Fml200008b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml200008b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200008b%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAllwood%26aufirst%3DM.%26aulast%3DRush%26aufirst%3DC.%2BL.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DBisdionin%2520C-A%2520rationally%2520designed%252C%2520submicromolar%2520inhibitor%2520of%2520family%252018%2520chitinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D428%26epage%3D432%26doi%3D10.1021%2Fml200008b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vora, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Methylxanthine drugs are chitinase inhibitors: investigation of inhibition and binding modes</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">973</span>â <span class="NLM_lpage">980</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2005.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.chembiol.2005.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16183021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=973-980&author=F.+V.+Raoauthor=O.+A.+Andersenauthor=K.+A.+Voraauthor=J.+A.+DeMartinoauthor=D.+M.+F.+van+Aalten&title=Methylxanthine+drugs+are+chitinase+inhibitors%3A+investigation+of+inhibition+and+binding+modes&doi=10.1016%2Fj.chembiol.2005.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Methylxanthine Drugs Are Chitinase Inhibitors: Investigation of Inhibition and Binding Modes</span></div><div class="casAuthors">Rao, Francesco V.; Andersen, Ole A.; Vora, Kalpit A.; DeMartino, Julie A.; Van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Family 18 chitinases play key roles in a range of pathogenic organisms and are overexpressed in the asthmatic lung.  By screening a library of marketed drug mols., we have identified methylxanthine derivs. as possible inhibitor leads.  These derivs., theophylline, caffeine, and pentoxifylline, are used therapeutically as antiinflammatory agents, with pleiotropic mechanisms of action.  Here it is shown that they are also competitive inhibitors against a fungal family 18 chitinase, with pentoxifylline being the most potent (Ki of 37 Î¼M).  Crystallog. anal. of chitinase-inhibitor complexes revealed specific interactions with the active site, mimicking the reaction intermediate analog, allosamidin.  Mutagenesis identified the key active site residues, conserved in mammalian chitinases, which contribute to inhibitor affinity.  Enzyme assays also revealed that these methylxanthines are active against human chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKoH7MRm3ibVg90H21EOLACvtfcHk0lgLp2U_RflQ0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7%252FM&md5=1d26875c45626540ccac6683f8b0cd41</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DVora%26aufirst%3DK.%2BA.%26aulast%3DDeMartino%26aufirst%3DJ.%2BA.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DMethylxanthine%2520drugs%2520are%2520chitinase%2520inhibitors%253A%2520investigation%2520of%2520inhibition%2520and%2520binding%2520modes%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D973%26epage%3D980%26doi%3D10.1016%2Fj.chembiol.2005.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼ttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Screening-based discovery and structural dissection of a novel family 18 chitinase inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">27278</span>â <span class="NLM_lpage">27285</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M604048200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M604048200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=16844689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlKmtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27278-27285&author=A.+W.+Sch%C3%BCttelkopfauthor=O.+A.+Andersenauthor=F.+V.+Raoauthor=M.+Allwoodauthor=C.+Lloydauthor=I.+M.+Egglestonauthor=D.+M.+F.+van+Aalten&title=Screening-based+discovery+and+structural+dissection+of+a+novel+family+18+chitinase+inhibitor&doi=10.1074%2Fjbc.M604048200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Screening-based Discovery and Structural Dissection of a Novel Family 18 Chitinase Inhibitor</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; Andersen, Ole A.; Rao, Francesco V.; Allwood, Matthew; Lloyd, Clare; Eggleston, Ian M.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">27278-27285</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Family 18 chitinases play key roles in the life cycles of a variety of organisms ranging from bacteria to man.  Very recently it has been shown that one of the mammalian chitinases is highly overexpressed in the asthmatic lung and contributes to the pathogenic process through recruitment of inflammatory cells.  Although several potent natural product chitinase inhibitors have been identified, their chemotherapeutic potential or their use as cell biol. tools is limited due to their size, complex chem., and limited availability.  We describe a virtual screening-based approach to identification of a novel, purine-based, chitinase inhibitor.  This inhibitor acts in the low micromolar (Ki = 2.8Â±0.2 Î¼M) range in a competitive mode.  Dissection of the binding mode by X-ray crystallog. reveals that the compd., which consists of two linked caffeine moieties, binds in the active site through extensive and not previously obsd. stacking interactions with conserved, solvent exposed tryptophans.  Such exposed aroms. are also present in the structures of many other carbohydrate processing enzymes.  The compd. exhibits favorable chem. properties and is likely to be useful as a general scaffold for development of pan-family 18 chitinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfEHLpyiMotrVg90H21EOLACvtfcHk0lj3SW4UEbpgkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlKmtrs%253D&md5=66db5ed0f11883ce5d74c77d6830669b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M604048200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M604048200%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DRao%26aufirst%3DF.%2BV.%26aulast%3DAllwood%26aufirst%3DM.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DEggleston%26aufirst%3DI.%2BM.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DScreening-based%2520discovery%2520and%2520structural%2520dissection%2520of%2520a%2520novel%2520family%252018%2520chitinase%2520inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27278%26epage%3D27285%26doi%3D10.1074%2Fjbc.M604048200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabudravu, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eijsink, V. G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gooday, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Synstad, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Psammaplin A, a chitinase inhibitor isolated from the Fijian marine sponge <i>Aplysinella rhax</i></span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1128</span>, <span class="refDoi">Â DOI: 10.1016/S0968-0896(01)00372-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0968-0896%2801%2900372-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=11836123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVKht70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1123-1128&author=J.+N.+Tabudravuauthor=V.+G.+H.+Eijsinkauthor=G.+W.+Goodayauthor=M.+Jasparsauthor=D.+Komanderauthor=M.+Leggauthor=B.+Synstadauthor=D.+M.+F.+van+Aalten&title=Psammaplin+A%2C+a+chitinase+inhibitor+isolated+from+the+Fijian+marine+sponge+Aplysinella+rhax&doi=10.1016%2FS0968-0896%2801%2900372-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Psammaplin A, a chitinase inhibitor isolated from the Fijian marine sponge Aplysinella rhax</span></div><div class="casAuthors">Tabudravu, J. N.; Eijsink, V. G. H.; Gooday, G. W.; Jaspars, M.; Komander, D.; Legg, M.; Synstad, B.; van Aalten, D. M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1123-1128</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Several brominated tyrosine derived compds., psammaplins A, K and L as well as bisaprasin were isolated from the Fijian marine sponge Aplysinella rhax during a bioassay guided isolation protocol.  Their structures were detd. using NMR and MS techniques.  Psammaplin A was found to moderately inhibit chitinase B from Serratia marcescens, the mode of inhibition being non-competitive.  Crystallog. studies suggest that a disordered psammaplin A mol. is bound near the active site.  Interestingly, psammaplin A was a potent antifungal agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKbW-239omDrVg90H21EOLACvtfcHk0lj3SW4UEbpgkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVKht70%253D&md5=64a9babaf17409abed456014eca0f965</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2801%2900372-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252801%252900372-8%26sid%3Dliteratum%253Aachs%26aulast%3DTabudravu%26aufirst%3DJ.%2BN.%26aulast%3DEijsink%26aufirst%3DV.%2BG.%2BH.%26aulast%3DGooday%26aufirst%3DG.%2BW.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DLegg%26aufirst%3DM.%26aulast%3DSynstad%26aufirst%3DB.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPsammaplin%2520A%252C%2520a%2520chitinase%2520inhibitor%2520isolated%2520from%2520the%2520Fijian%2520marine%2520sponge%2520Aplysinella%2520rhax%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2002%26volume%3D10%26spage%3D1123%26epage%3D1128%26doi%3D10.1016%2FS0968-0896%2801%2900372-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shizuri, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo</span></span> <span> </span><span class="NLM_article-title">Styloguanidines, new chitinase inhibitors from the marine sponge <i>Stylotella aurantium</i></span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2133</span>â <span class="NLM_lpage">2136</span>, <span class="refDoi">Â DOI: 10.1016/0040-4039(95)00194-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2F0040-4039%2895%2900194-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK2MXkslOjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=2133-2136&author=T.+Katoauthor=Y.+Shizuriauthor=H.+Izumidaauthor=A.+Yokoyamaauthor=+Endo&title=Styloguanidines%2C+new+chitinase+inhibitors+from+the+marine+sponge+Stylotella+aurantium&doi=10.1016%2F0040-4039%2895%2900194-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Styloguanidines, new chitinase inhibitors from the marine sponge Stylotella aurantium</span></div><div class="casAuthors">Kato, Taketoshi; Shizuri, Yoshikazu; Izumida, Hitoshi; Yokoyama, Akihiro; Endo, Mamoru</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2133-6</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Three potent new chitinase inhibitors, styloguanidine (I), 3-bromostyloguanidine, and 2,3-dibromostyloguanidine were isolated from the sponge collected from the Yap Sea.  Their structures were detd. by spectral means.  These were a new type of inhibitor from natural sources and were proved to inhibit the molting of cyprid larvae of barnacles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorawLAF1tPNrVg90H21EOLACvtfcHk0lj3SW4UEbpgkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslOjsr4%253D&md5=a97fef3517b4255d828866bc2d2766bc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2900194-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252900194-H%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DT.%26aulast%3DShizuri%26aufirst%3DY.%26aulast%3DIzumida%26aufirst%3DH.%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DEndo%26atitle%3DStyloguanidines%252C%2520new%2520chitinase%2520inhibitors%2520from%2520the%2520marine%2520sponge%2520Stylotella%2520aurantium%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26spage%3D2133%26epage%3D2136%26doi%3D10.1016%2F0040-4039%2895%2900194-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchie, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankovich, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuff, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span> <span> </span><span class="NLM_article-title">Functional comparison of recombinant acidic mammalian chitinase with enzyme from murine bronchoalveolar lavage</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">55</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1016/j.pep.2010.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.pep.2010.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20826216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKrsrjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2011&pages=55-62&author=E.+R.+Goedkenauthor=R.+F.+O%E2%80%99Brienauthor=T.+Xiangauthor=D.+L.+Banachauthor=S.+C.+Marchieauthor=E.+H.+Barlowauthor=S.+Hubbardauthor=J.+A.+Mankovichauthor=J.+Jiangauthor=P.+L.+Richardsonauthor=C.+A.+Cuffauthor=A.+D.+Cherniack&title=Functional+comparison+of+recombinant+acidic+mammalian+chitinase+with+enzyme+from+murine+bronchoalveolar+lavage&doi=10.1016%2Fj.pep.2010.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional comparison of recombinant acidic mammalian chitinase with enzyme from murine bronchoalveolar lavage</span></div><div class="casAuthors">Goedken, Eric R.; O'Brien, Robert F.; Xiang, Tao; Banach, David L.; Marchie, Susan C.; Barlow, Eve H.; Hubbard, Stacy; Mankovich, John A.; Jiang, Jianjun; Richardson, Paul L.; Cuff, Carolyn A.; Cherniack, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression & Purification</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is an enzyme that selectively degrades the biopolymer chitin.  Several chitinase enzymes are utilized by mammals to hydrolyze chitin encountered by inhalation and ingestion.  AMCase is distinct from other mammalian chitinases as its activity is retained in strongly acidic conditions (pH < 2.0).  AMCase expression is induced by antigen-induced mouse models of allergic lung inflammation.  This protein has also been implicated in the pathogenesis of asthma although its precise role is poorly defined.  A novel way to express and purify active murine AMCase is described.  This material retains properties obsd. in mouse bronchoalveolar lavage (BAL) fluid with regard to pH preference of activity and its inhibition by cyclic peptide inhibitors argifin and argadin.  It was found that chitinase in BAL from both antigen-challenged and control animals have similar properties in this regard.  This strongly supports the notion the same enzyme (AMCase) gives rise to chitinase activity in both challenged and unchallenged animals.  Expression of active human AMCase is also described.  The methods described in this paper provide a reliable source of recombinant AMCase that can be utilized to expand understanding of AMCase's role in regulating allergic inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonWeoser5mN7Vg90H21EOLACvtfcHk0lip3LnlGVAmEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKrsrjN&md5=93103913f8a5d169445b226e49f0034b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2010.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2010.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DR.%2BF.%26aulast%3DXiang%26aufirst%3DT.%26aulast%3DBanach%26aufirst%3DD.%2BL.%26aulast%3DMarchie%26aufirst%3DS.%2BC.%26aulast%3DBarlow%26aufirst%3DE.%2BH.%26aulast%3DHubbard%26aufirst%3DS.%26aulast%3DMankovich%26aufirst%3DJ.%2BA.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DCuff%26aufirst%3DC.%2BA.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26atitle%3DFunctional%2520comparison%2520of%2520recombinant%2520acidic%2520mammalian%2520chitinase%2520with%2520enzyme%2520from%2520murine%2520bronchoalveolar%2520lavage%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2011%26volume%3D75%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.pep.2010.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olland, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursavich, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeClercq, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph-McCarthy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingboe, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, D. H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of acidic mammalian chitinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6122</span>â <span class="NLM_lpage">6128</span>, <span class="refDoi">Â DOI: 10.1021/jm100533p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100533p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6122-6128&author=D.+C.+Coleauthor=A.+M.+Ollandauthor=J.+Jacobauthor=J.+Brooksauthor=M.+G.+Bursavichauthor=R.+Czerwinskiauthor=C.+DeClercqauthor=M.+Johnsonauthor=D.+Joseph-McCarthyauthor=J.+W.+Ellingboeauthor=L.+Linauthor=P.+Nowakauthor=E.+Presmanauthor=J.+Strandauthor=A.+Tamauthor=C.+M.+M.+Williamsauthor=S.+Yaoauthor=D.+H.+H.+Tsaoauthor=L.+J.+Fitz&title=Identification+and+characterization+of+acidic+mammalian+chitinase+inhibitors&doi=10.1021%2Fjm100533p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Acidic Mammalian Chitinase Inhibitors</span></div><div class="casAuthors">Cole, Derek C.; Olland, Andrea M.; Jacob, Jaison; Brooks, Jon; Bursavich, Matthew G.; Czerwinski, Robert; De Clercq, Charlene; Johnson, Mark; Joseph-McCarthy, Diane; Ellingboe, John W.; Lin, Laura; Nowak, Pawel; Presman, Ella; Strand, James; Tam, Amy; Williams, Cara M. M.; Yao, Shihua; Tsao, Desiree H. H.; Fitz, Lori J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6122-6128</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acidic mammalian chitinase (AMCase) is a member of the glycosyl hydrolase 18 family (EC 3.2.1.14) that has been implicated in the pathophysiol. of allergic airway disease such as asthma.  Small mol. inhibitors of AMCase were identified using a combination of high-throughput screening, fragment screening, and virtual screening techniques and characterized by enzyme inhibition and NMR and Biacore binding expts.  X-ray structures of the inhibitors in complex with AMCase revealed that the larger more potent HTS hits, e.g. 5-(4-(2-(4-bromophenoxy)ethyl)piperazine-1-yl)-1H-1,2,4-triazol-3-amine 1, spanned from the active site pocket to a hydrophobic pocket.  Smaller fragments identified by FBS occupy both these pockets independently and suggest potential strategies for linking fragments.  Compd. 1 is a 200 nM AMCase inhibitor which reduced AMCase enzymic activity in the bronchoalveolar lavage fluid in allergen-challenged mice after oral dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq3-AOdwYs2LVg90H21EOLACvtfcHk0lip3LnlGVAmEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChtL8%253D&md5=47d5010b73151d1ad4f641c47c3fb100</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100533p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100533p%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BC.%26aulast%3DOlland%26aufirst%3DA.%2BM.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DBrooks%26aufirst%3DJ.%26aulast%3DBursavich%26aufirst%3DM.%2BG.%26aulast%3DCzerwinski%26aufirst%3DR.%26aulast%3DDeClercq%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DEllingboe%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNowak%26aufirst%3DP.%26aulast%3DPresman%26aufirst%3DE.%26aulast%3DStrand%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DC.%2BM.%2BM.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%2BH.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520acidic%2520mammalian%2520chitinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6122%26epage%3D6128%26doi%3D10.1021%2Fjm100533p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoszewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andryianau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matyszewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pluta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefaniak, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurkiewicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czestkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedziejko, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruza, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzwonek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olczak, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective, orally bioavailable inhibitor of mouse chitotriosidase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">310</span>â <span class="NLM_lpage">314</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2017.12.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.bmcl.2017.12.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=29292229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlSrsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=310-314&author=M.+Mazurauthor=A.+Bartoszewiczauthor=B.+Dymekauthor=M.+Salamonauthor=G.+Andryianauauthor=M.+Kowalskiauthor=S.+Olejniczakauthor=K.+Matyszewskiauthor=E.+Plutaauthor=B.+Borekauthor=F.+Stefaniakauthor=A.+Zagozdzonauthor=M.+Mazurkiewiczauthor=R.+Koralewskiauthor=W.+Czestkowskiauthor=M.+Piotrowiczauthor=P.+Niedziejkoauthor=M.+M.+Gruzaauthor=K.+Dzwonekauthor=A.+Golebiowskiauthor=J.+Golabauthor=J.+Olczak&title=Discovery+of+selective%2C+orally+bioavailable+inhibitor+of+mouse+chitotriosidase&doi=10.1016%2Fj.bmcl.2017.12.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective, orally bioavailable inhibitor of mouse chitotriosidase</span></div><div class="casAuthors">Mazur, Marzena; Bartoszewicz, Agnieszka; Dymek, Barbara; Salamon, Magdalena; Andryianau, Gleb; Kowalski, Michal; Olejniczak, Sylwia; Matyszewski, Krzysztof; Pluta, Elzbieta; Borek, Bartlomiej; Stefaniak, Filip; Zagozdzon, Agnieszka; Mazurkiewicz, Marcin; Koralewski, Robert; Czestkowski, Wojciech; Piotrowicz, Michal; Niedziejko, Piotr; Gruza, Mariusz M.; Dzwonek, Karolina; Golebiowski, Adam; Golab, Jakub; Olczak, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-314</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article describes the authors' work towards the identification of a potent and selective inhibitor of mouse chitotriosidase (mCHIT1).  A series of small mol. inhibitors of mCHIT1 and mAMCase have been developed from early lead compd. (I).  Examn. of synthesized analogs led to discovery of several novel highly potent compds.  Among them compd. 9 (OAT-2068) displays a remarkable 143-fold mCHIT1 vs. mAMCase selectivity.  To explain the obsd. SAR mol. docking expts. were performed, which were in line with the exptl. data from the enzymic assays.  Inhibitor 9 (OAT-2068) was found to have an excellent pharmacokinetic profile.  This, together with high activity and selectivity, makes the compd. an ideal and unique tool for studying the role of CHIT1 in biol. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot976-E0U9pLVg90H21EOLACvtfcHk0lip3LnlGVAmEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlSrsA%253D%253D&md5=420c689a31b501c151a1915ca37ab16b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.12.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.12.047%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DM.%26aulast%3DBartoszewicz%26aufirst%3DA.%26aulast%3DDymek%26aufirst%3DB.%26aulast%3DSalamon%26aufirst%3DM.%26aulast%3DAndryianau%26aufirst%3DG.%26aulast%3DKowalski%26aufirst%3DM.%26aulast%3DOlejniczak%26aufirst%3DS.%26aulast%3DMatyszewski%26aufirst%3DK.%26aulast%3DPluta%26aufirst%3DE.%26aulast%3DBorek%26aufirst%3DB.%26aulast%3DStefaniak%26aufirst%3DF.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DMazurkiewicz%26aufirst%3DM.%26aulast%3DKoralewski%26aufirst%3DR.%26aulast%3DCzestkowski%26aufirst%3DW.%26aulast%3DPiotrowicz%26aufirst%3DM.%26aulast%3DNiedziejko%26aufirst%3DP.%26aulast%3DGruza%26aufirst%3DM.%2BM.%26aulast%3DDzwonek%26aufirst%3DK.%26aulast%3DGolebiowski%26aufirst%3DA.%26aulast%3DGolab%26aufirst%3DJ.%26aulast%3DOlczak%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%252C%2520orally%2520bioavailable%2520inhibitor%2520of%2520mouse%2520chitotriosidase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D310%26epage%3D314%26doi%3D10.1016%2Fj.bmcl.2017.12.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CibriÃ¡n-Uhalte, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MagariÃ±os, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL database in 2017</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D945</span>â <span class="NLM_lpage">D954</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkw1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1093%2Fnar%2Fgkw1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=27899562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D945-D954&author=A.+Gaultonauthor=A.+Herseyauthor=M.+Nowotkaauthor=A.+P.+Bentoauthor=J.+Chambersauthor=D.+Mendezauthor=P.+Mutowoauthor=F.+Atkinsonauthor=L.+J.+Bellisauthor=E.+Cibri%C3%A1n-Uhalteauthor=M.+Daviesauthor=N.+Dedmanauthor=A.+Karlssonauthor=M.+P.+Magari%C3%B1osauthor=J.+P.+Overingtonauthor=G.+Papadatosauthor=I.+Smitauthor=A.+R.+Leach&title=The+ChEMBL+database+in+2017&doi=10.1093%2Fnar%2Fgkw1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL database in 2017</span></div><div class="casAuthors">Gaulton, Anna; Hersey, Anne; Nowotka, Micha-l; Bento, A. Patricia; Chambers, Jon; Mendez, David; Mutowo, Prudence; Atkinson, Francis; Bellis, Louisa J.; Cibrian-Uhalte, Elena; Davies, Mark; Dedman, Nathan; Karlsson, Anneli; Magarinos, Maria Paula; Overington, John P.; Papadatos, George; Smit, Ines; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D945-D954</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 and 2014 Nucleic Acids Research Database Issues.  Since then, alongside the continued extn. of data from the medicinal chem. literature, new sources of bioactivity data have also been added to the database.  These include: deposited data sets from neglected disease screening; crop protection data; drug metab. and disposition data and bioactivity data from patents.  A no. of improvements and new features have also been incorporated.  These include the annotation of assays and targets using ontologies, the inclusion of targets and indications for clin. candidates, addn. of metabolic pathways for drugs and calcn. of structural alerts.  The ChEMBL data can be accessed via a web-interface, RDF distribution, data downloads and RESTful web-services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB0IuNShSHIrVg90H21EOLACvtfcHk0lifl5pn4ut7MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhurs%253D&md5=1dcbb641f8c2d11410663f97505a4911</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1074%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DCibri%25C3%25A1n-Uhalte%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DI.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DThe%2520ChEMBL%2520database%2520in%25202017%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD945%26epage%3DD954%26doi%3D10.1093%2Fnar%2Fgkw1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>, <span class="refDoi">Â DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, Î±2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lifl5pn4ut7MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>â <span class="NLM_lpage">2740</span>, <span class="refDoi">Â DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lifl5pn4ut7MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruns, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span> <span> </span><span class="NLM_article-title">Rules for identifying potentially reactive or promiscuous compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9763</span>â <span class="NLM_lpage">9772</span>, <span class="refDoi">Â DOI: 10.1021/jm301008n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301008n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSgs7bP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9763-9772&author=R.+F.+Brunsauthor=I.+A.+Watson&title=Rules+for+identifying+potentially+reactive+or+promiscuous+compounds&doi=10.1021%2Fjm301008n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Rules for Identifying Potentially Reactive or Promiscuous Compounds</span></div><div class="casAuthors">Bruns, Robert F.; Watson, Ian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9763-9772</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article describes a set of 275 rules, developed over an 18-yr period, used to identify compds. that may interfere with biol. assays, allowing their removal from screening sets.  Reasons for rejection include reactivity (e.g., acyl halides), interference with assay measurements (fluorescence, absorbance, quenching), activities that damage proteins (oxidizers, detergents), instability (e.g., latent aldehydes), and lack of druggability (e.g., compds. lacking both oxygen and nitrogen).  The structural queries were profiled for frequency of occurrence in druglike and nondruglike compd. sets and were extensively reviewed by a panel of experienced medicinal chemists.  As a means of profiling the rules and as a filter in its own right, an index of biol. promiscuity was developed.  The 584 gene targets with screening data at Lilly were assigned to 17 subfamilies, and the no. of subfamilies at which a compd. was active was used as a promiscuity index.  For certain compds., promiscuous activity disappeared after sample repurifn., indicating interference from occult contaminants.  Because this type of interference is not amenable to substructure search, a "nuisance list" was developed to flag interfering compds. that passed the substructure rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNvXr0QeH2l7Vg90H21EOLACvtfcHk0lh5y6prOOCnvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSgs7bP&md5=3c37f5513b3859e0180686e99d83d30c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm301008n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301008n%26sid%3Dliteratum%253Aachs%26aulast%3DBruns%26aufirst%3DR.%2BF.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26atitle%3DRules%2520for%2520identifying%2520potentially%2520reactive%2520or%2520promiscuous%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9763%26epage%3D9772%26doi%3D10.1021%2Fjm301008n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâRourke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremb, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duggan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sioud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbatia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emwas, A.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voolstra, C.</span></span> <span> </span><span class="NLM_article-title">Identification of a 3-Alkylpyridinium compound from the Red Sea sponge <i>Amphimedon chloros</i> with in vitro inhibitory activity against the West Nile virus NS3 protease</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1420</span>â <span class="NLM_lpage">3049</span>, <span class="refDoi">Â DOI: 10.3390/molecules23061472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fmolecules23061472" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1420-3049&author=A.+O%E2%80%99Rourkeauthor=S.+Krembauthor=B.+Dugganauthor=S.+Sioudauthor=N.+Kharbatiaauthor=M.+Rajiauthor=A.-H.+Emwasauthor=W.+Gerwickauthor=C.+Voolstra&title=Identification+of+a+3-Alkylpyridinium+compound+from+the+Red+Sea+sponge+Amphimedon+chloros+with+in+vitro+inhibitory+activity+against+the+West+Nile+virus+NS3+protease&doi=10.3390%2Fmolecules23061472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23061472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23061472%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DA.%26aulast%3DKremb%26aufirst%3DS.%26aulast%3DDuggan%26aufirst%3DB.%26aulast%3DSioud%26aufirst%3DS.%26aulast%3DKharbatia%26aufirst%3DN.%26aulast%3DRaji%26aufirst%3DM.%26aulast%3DEmwas%26aufirst%3DA.-H.%26aulast%3DGerwick%26aufirst%3DW.%26aulast%3DVoolstra%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%25203-Alkylpyridinium%2520compound%2520from%2520the%2520Red%2520Sea%2520sponge%2520Amphimedon%2520chloros%2520with%2520in%2520vitro%2520inhibitory%2520activity%2520against%2520the%2520West%2520Nile%2520virus%2520NS3%2520protease%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1420%26epage%3D3049%26doi%3D10.3390%2Fmolecules23061472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zovko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viktorsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewensohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KoloÅ¡a, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FilipiÄ, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kem, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paleari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, T.</span></span> <span> </span><span class="NLM_article-title">APS8, a polymeric alkylpyridinium salt blocks Î±7 nAChR and induces apoptosis in non-small cell lung carcinoma</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2574</span>â <span class="NLM_lpage">2594</span>, <span class="refDoi">Â DOI: 10.3390/md11072574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.3390%2Fmd11072574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23880932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1OntLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=2574-2594&author=A.+Zovkoauthor=K.+Viktorssonauthor=R.+Lewensohnauthor=K.+Kolo%C5%A1aauthor=M.+Filipi%C4%8Dauthor=H.+Xingauthor=W.+Kemauthor=L.+Paleariauthor=T.+Turk&title=APS8%2C+a+polymeric+alkylpyridinium+salt+blocks+%CE%B17+nAChR+and+induces+apoptosis+in+non-small+cell+lung+carcinoma&doi=10.3390%2Fmd11072574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">APS8, a polymeric alkylpyridinium salt blocks Î±7 nAChR and induces apoptosis in non-small cell lung carcinoma</span></div><div class="casAuthors">Zovko, Ana; Viktorsson, Kristina; Lewensohn, Rolf; Kolosa, Katja; Filipic, Metka; Xing, Hong; Kem, William R.; Paleari, Laura; Turk, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2574-2594, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Naturally occurring 3-alkylpyridinium polymers (poly-APS) from the marine sponge Reniera sarai, consisting of monomers contg. polar pyridinium and nonpolar alkyl chain moieties, have been demonstrated to exert a wide range of biol. activities, including a selective cytotoxicity against non-small cell lung cancer (NSCLC) cells.  APS8, an analog of poly-APS with defined alkyl chain length and mol. size, non-competitively inhibits Î±7 nicotinic acetylcholine receptors (nAChRs) at nanomolar concns. that are too low to be acetylcholinesterase (AChE) inhibitory or generally cytotoxic.  In the present study we show that APS8 inhibits NSCLC tumor cell growth and activates apoptotic pathways.  APS8 was not toxic for normal lung fibroblasts.  Furthermore, in NSCLC cells, APS8 reduced the adverse anti-apoptotic, proliferative effects of nicotine.  Our results suggest that APS8 or similar compds. might be considered as lead compds. to develop antitumor therapeutic agents for at least certain types of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1pxDm-wo0rLVg90H21EOLACvtfcHk0lh5y6prOOCnvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1OntLrK&md5=916e08d126a917ad1635cecad12ea82e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fmd11072574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd11072574%26sid%3Dliteratum%253Aachs%26aulast%3DZovko%26aufirst%3DA.%26aulast%3DViktorsson%26aufirst%3DK.%26aulast%3DLewensohn%26aufirst%3DR.%26aulast%3DKolo%25C5%25A1a%26aufirst%3DK.%26aulast%3DFilipi%25C4%258D%26aufirst%3DM.%26aulast%3DXing%26aufirst%3DH.%26aulast%3DKem%26aufirst%3DW.%26aulast%3DPaleari%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DT.%26atitle%3DAPS8%252C%2520a%2520polymeric%2520alkylpyridinium%2520salt%2520blocks%2520%25CE%25B17%2520nAChR%2520and%2520induces%2520apoptosis%2520in%2520non-small%2520cell%2520lung%2520carcinoma%26jtitle%3DMar.%2520Drugs%26date%3D2013%26volume%3D11%26spage%3D2574%26epage%3D2594%26doi%3D10.3390%2Fmd11072574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GrandiÄ, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ArÃ¡oz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MolgÃ³, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SepÄiÄ, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FrangeÅ¾, R.</span></span> <span> </span><span class="NLM_article-title">The non-competitive acetylcholinesterase inhibitor APS12-2 is a potent antagonist of skeletal muscle nicotinic acetylcholine receptors</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">221</span>â <span class="NLM_lpage">228</span>, <span class="refDoi">Â DOI: 10.1016/j.taap.2012.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.taap.2012.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=23046821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslejtbbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=2012&pages=221-228&author=M.+Grandi%C4%8Dauthor=R.+Ar%C3%A1ozauthor=J.+Molg%C3%B3author=T.+Turkauthor=K.+Sep%C4%8Di%C4%87author=E.+Benoitauthor=R.+Frange%C5%BE&title=The+non-competitive+acetylcholinesterase+inhibitor+APS12-2+is+a+potent+antagonist+of+skeletal+muscle+nicotinic+acetylcholine+receptors&doi=10.1016%2Fj.taap.2012.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The non-competitive acetylcholinesterase inhibitor APS12-2 is a potent antagonist of skeletal muscle nicotinic acetylcholine receptors</span></div><div class="casAuthors">Grandic, Marjana; Araoz, Romulo; Molgo, Jordi; Turk, Tom; Sepcic, Kristina; Benoit, Evelyne; Frangez, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">221-228</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">APS12-2, a non-competitive acetylcholinesterase inhibitor, is one of the synthetic analogs of polymeric alkylpyridinium salts (poly-APS) isolated from the marine sponge Reniera sarai.  In the present work the effects of APS12-2 were studied on isolated mouse phrenic nerve-hemidiaphragm muscle prepns., using twitch tension measurements and electrophysiol. recordings.  APS12-2 in a concn.-dependent manner blocked nerve-evoked isometric muscle contraction (IC50 = 0.74 Î¼M), without affecting directly-elicited twitch tension up to 2.72 Î¼M.  The compd. (0.007-3.40 Î¼M) decreased the amplitude of miniature endplate potentials until a complete block by concns. higher than 0.68 Î¼M, without affecting their frequency.  Full size endplate potentials, recorded after blocking voltage-gated muscle sodium channels, were inhibited by APS12-2 in a concn.-dependent manner (IC50 = 0.36 Î¼M) without significant change in the resting membrane potential of the muscle fibers up to 3.40 Î¼M.  The compd. also blocked acetylcholine-evoked inward currents in Xenopus oocytes in which Torpedo (Î±12Î²1Î³Î´) muscle-type nicotinic acetylcholine receptors (nAChRs) have been incorporated (IC50 = 0.0005 Î¼M), indicating a higher affinity of the compd. for Torpedo (Î±12Î²1Î³Î´) than for the mouse (Î±12Î²1Î³Îµ) nAChR.  Our data show for the first time that APS12-2 blocks neuromuscular transmission by a non-depolarizing mechanism through an action on postsynaptic nAChRs of the skeletal neuromuscular junction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo91qjIVLLOgLVg90H21EOLACvtfcHk0lhfxm09p5nrGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslejtbbN&md5=7e22be7d20d8122fb8fe0db20cfa890b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2012.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2012.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DGrandi%25C4%258D%26aufirst%3DM.%26aulast%3DAr%25C3%25A1oz%26aufirst%3DR.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DT.%26aulast%3DSep%25C4%258Di%25C4%2587%26aufirst%3DK.%26aulast%3DBenoit%26aufirst%3DE.%26aulast%3DFrange%25C5%25BE%26aufirst%3DR.%26atitle%3DThe%2520non-competitive%2520acetylcholinesterase%2520inhibitor%2520APS12-2%2520is%2520a%2520potent%2520antagonist%2520of%2520skeletal%2520muscle%2520nicotinic%2520acetylcholine%2520receptors%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2012%26volume%3D265%26spage%3D221%26epage%3D228%26doi%3D10.1016%2Fj.taap.2012.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pantoom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suginta, W.</span></span> <span> </span><span class="NLM_article-title">Potent family-18 chitinase inhibitors X-ray structures, affinities, and binding mechanisms</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">24312</span>â <span class="NLM_lpage">24323</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M110.183376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M110.183376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=21531720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1GltLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=24312-24323&author=S.+Pantoomauthor=I.+R.+Vetterauthor=H.+Prinzauthor=W.+Suginta&title=Potent+family-18+chitinase+inhibitors+X-ray+structures%2C+affinities%2C+and+binding+mechanisms&doi=10.1074%2Fjbc.M110.183376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Family-18 Chitinase Inhibitors: X-RAY STRUCTURES, AFFINITIES, AND BINDING MECHANISMS</span></div><div class="casAuthors">Pantoom, Supansa; Vetter, Ingrid R.; Prinz, Heino; Suginta, Wipa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">24312-24323</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Six novel inhibitors of Vibrio harveyi chitinase A (VhChiA), a family-18 chitinase homolog, were identified by in vitro screening of a library of pharmacol. active compds.  Unlike the previously identified inhibitors that mimicked the reaction intermediates, crystallog. evidence from 14 VhChiA-inhibitor complexes showed that all of the inhibitor mols. occupied the outer part of the substrate-binding cleft at two hydrophobic areas.  The interactions at the aglycon location are well defined and tightly assocd. with Trp-397 and Trp-275, whereas the interactions at the glycone location are patchy, indicating lower affinity and a loose interaction with two consensus residues, Trp-168 and Val-205.  When Trp-275 was substituted with glycine (W275G), the binding affinity toward all of the inhibitors dramatically decreased, and in most structures two inhibitor mols. were found to stack against Trp-397 at the aglycon site.  Such results indicate that hydrophobic interactions are important for binding of the newly identified inhibitors by the chitinase.  X-ray data and isothermal microcalorimetry showed that the inhibitors occupied the active site of VhChiA in three different binding modes, including single-site binding, independent two-site binding, and sequential two-site binding.  The inhibitory effect of dequalinium in the low nanomolar range makes this compd. an extremely attractive lead compd. for plausible development of therapeutics against human diseases involving chitinase-mediated pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwlrfx2sXPQrVg90H21EOLACvtfcHk0lhfxm09p5nrGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1GltLk%253D&md5=90d769da0f73c2b8fd3f6dab53c3b880</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.183376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.183376%26sid%3Dliteratum%253Aachs%26aulast%3DPantoom%26aufirst%3DS.%26aulast%3DVetter%26aufirst%3DI.%2BR.%26aulast%3DPrinz%26aufirst%3DH.%26aulast%3DSuginta%26aufirst%3DW.%26atitle%3DPotent%2520family-18%2520chitinase%2520inhibitors%2520X-ray%2520structures%252C%2520affinities%252C%2520and%2520binding%2520mechanisms%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D24312%26epage%3D24323%26doi%3D10.1074%2Fjbc.M110.183376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><a href="http://rsd.wuxiapptec.com/InvitrosafetyprofilingforMedchemSupport" class="extLink">http://rsd.wuxiapptec.com/InvitrosafetyprofilingforMedchemSupport</a> (Accessed on Dec. 31, 2019)</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Frsd.wuxiapptec.com%2FInvitrosafetyprofilingforMedchemSupport+%28Accessed+on+Dec.+31%2C+2019%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lhfxm09p5nrGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Hierarchical virtual screening approaches in small molecule drug discovery</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">26</span>â <span class="NLM_lpage">37</span>, <span class="refDoi">Â DOI: 10.1016/j.ymeth.2014.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1016%2Fj.ymeth.2014.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=25072167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12mt7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=26-37&author=A.+Kumarauthor=K.+Y.+J.+Zhang&title=Hierarchical+virtual+screening+approaches+in+small+molecule+drug+discovery&doi=10.1016%2Fj.ymeth.2014.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical virtual screening approaches in small molecule drug discovery</span></div><div class="casAuthors">Kumar, Ashutosh; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-37</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Virtual screening has played a significant role in the discovery of small mol. inhibitors of therapeutic targets in last two decades.  Various ligand and structure-based virtual screening approaches are employed to identify small mol. ligands for proteins of interest.  These approaches are often combined in either hierarchical or parallel manner to take advantage of the strength and avoid the limitations assocd. with individual methods.  Hierarchical combination of ligand and structure-based virtual screening approaches has received noteworthy success in numerous drug discovery campaigns.  In hierarchical virtual screening, several filters using ligand and structure-based approaches are sequentially applied to reduce a large screening library to a no. small enough for exptl. testing.  In this review, we focus on different hierarchical virtual screening strategies and their application in the discovery of small mol. modulators of important drug targets.  Several virtual screening studies are discussed to demonstrate the successful application of hierarchical virtual screening in small mol. drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI5IS_ELjp3rVg90H21EOLACvtfcHk0lix8qxT2ZzkyQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12mt7bO&md5=37afd198e391774598a72214a377fd05</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DHierarchical%2520virtual%2520screening%2520approaches%2520in%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D26%26epage%3D37%26doi%3D10.1016%2Fj.ymeth.2014.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">82</span>, <span class="refDoi">Â DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lix8qxT2ZzkyQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the protein databank and cambridge structural database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+protein+databank+and+cambridge+structural+database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lix8qxT2ZzkyQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520protein%2520databank%2520and%2520cambridge%2520structural%2520database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimoto, T. T.</span></span> <span> </span><span class="NLM_article-title">A computer program for classifying plants</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1960</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">1115</span>â <span class="NLM_lpage">1118</span>, <span class="refDoi">Â DOI: 10.1126/science.132.3434.1115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1126%2Fscience.132.3434.1115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17790723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A280%3ADC%252BC3cvgvFyntw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=1960&pages=1115-1118&author=D.+J.+Rogersauthor=T.+T.+Tanimoto&title=A+computer+program+for+classifying+plants&doi=10.1126%2Fscience.132.3434.1115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Computer Program for Classifying Plants</span></div><div class="casAuthors">Rogers D J; Tanimoto T T</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1960</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">3434</span>),
    <span class="NLM_cas:pages">1115-8</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL7dGsQdn6sO2qxNLjzXe8fW6udTcc2ebFfa1I6nKv-Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cvgvFyntw%253D%253D&md5=1b1906eaa3974e68c3933a464484041d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.132.3434.1115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.132.3434.1115%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%2BJ.%26aulast%3DTanimoto%26aufirst%3DT.%2BT.%26atitle%3DA%2520computer%2520program%2520for%2520classifying%2520plants%26jtitle%3DScience%26date%3D1960%26volume%3D132%26spage%3D1115%26epage%3D1118%26doi%3D10.1126%2Fscience.132.3434.1115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachbar, R. B.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. IV. conformational energies and geometries for MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">615</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<587::AID-JCC4>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=587-615&author=T.+A.+Halgrenauthor=R.+B.+Nachbar&title=Merck+molecular+force+field.+IV.+conformational+energies+and+geometries+for+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. IV. Conformational energies and geometries for MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.; Nachbar, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">587-615</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the parameterization and performance of MMFF94 for conformational energies, rotational barriers, and equil. torsion angles.  It describes the derivation of the torsion parameters from high-quality computational data and characterizes MMFF94's ability to reproduce the computational and exptl. data, the latter particularly in relation to MM3.  The computational data included: (i) â¼250 comparisons of conformational energy based on "MP4SDQ/TZP" calcns. (triple-zeta plus polarization calcns. at a defined approxn. to the highly correlated MP4SDQ level) at MP2/6-31G* geometries; and (ii) â¼1200 MP2/TZP comparisons of "torsion profile" structures at geometries derived in restrained least-squares fits that used the complete set of available computational data, thereby ensuring that a fully optimal set of parameters would be obtained.  The final parameters reproduce the "MP4SDQ/TZP" and MP2/TZP computational data with root mean square (rms) deviations of 0.31 and 0.50 kcal/mol, resp.  In addn., MMFF94 reproduces a set of 37 exptl. gas-phase and soln. conformational energies, enthalpies, and free energies with a rms deviation of 0.38 kcal/mol; for comparison, the "MP3SDQ/TZP" calcns. and MM3 each gives a rms deviation of 0.37 kcal/mol.  Furthermore, MMFF94 reproduces 28 exptl. detd. rotational barriers with a rms deviation of 0.39 kcal/mol.  Given the diverse nature of the exptl. conformational energies and rotational barriers and the clear indications of exptl. error in some cases, the MMFF94 results appear excellent.  Nevertheless, MMFF94 encounters somewhat greater difficulty in handling multi-functional compds. that place highly polar functional groups in close proximity, probably because it, like other commonly used force fields, too greatly simplifies the description of electrostatic interactions.  Some suggestions for enhancements to MMFF94's functional form are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTSvDsbv5rM7Vg90H21EOLACvtfcHk0lixZk9KeIs0yQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmtr4%253D&md5=fd00606ee59a83268c33e614220127f2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C587%253A%253AAID-JCC4%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DNachbar%26aufirst%3DR.%2BB.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520IV.%2520conformational%2520energies%2520and%2520geometries%2520for%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D587%26epage%3D615%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C587%3A%3AAID-JCC4%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. III. Molecular geometries and vibrational frequencies for MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">553</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<553::AID-JCC3>3.0.CO;2-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=553-586&author=T.+A.+Halgren&title=Merck+molecular+force+field.+III.+Molecular+geometries+and+vibrational+frequencies+for+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. III. Molecular geometries and vibrational frequencies for MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">553-86</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the parameterization and performance of MMFF94 for mol. geometries and deformations.  It defines the form used for the valence-coordinate terms that represent variations in bond lengths and angles, and it describes the derivation of quadratic force consts. from HF/6-31G* data and the derivation of ref. bond lengths and angles from fits to MP2/6-31G*-optimized geometries.  Comparisons offered show that MMFF94 accurately reproduces the computational data used in its parameterization and demonstrate that its derivation from such data simultaneously confers the ability to reproduce expt.  In particular, MMFF94 reproduces exptl. detd. bond lengths and angles for 30 org. mols. with root mean square (rms) deviations of 0.014 Ã and 1.2Â°, resp.  MM3 reproduces bond angles to the same accuracy, but reproduces exptl. bond lengths more accurately, in part because it was fit directly to thermally averaged exptl. bond lengths; MMFF94, in contrast, was fit to (usually shorter) energy-min. values, as is proper for an anharmonic force field intended for use in mol.-dynamics simulations.  The comparisons also show that UFF and a recent version of CHARMm (QUANTA 3.3 parameterization) are less accurate for mol. geometries than either MMFF94 or MM3.  For vibrational frequencies, MMFF94 and MM3 give comparable overall rms deviations vs. expt. of 61 cm-1 and 57 cm-1, resp., for 15 small, mostly org. mols.  In a no. of instances, MM3's derivation employed obsd. frequencies that differ substantially-by nearly 400 cm cm-1 in one case-from other published frequencies which had themselves been confirmed theor. by good-quality ab initio calcns.  Overall, the comparisons to exptl. geometries and vibrational frequencies demonstrate that MMFF94 achieves MM3-like accuracy for org. systems for which MM3 has been parameterized.  Because MMFF94 is derived mainly from computational data, however, it has been possible to parameterize MMFF94 with equal rigor for a wide variety of addnl. systems for which little or no useful exptl. data exist.  Equally good performance can be expected for such systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWR5mBOf56TbVg90H21EOLACvtfcHk0lixZk9KeIs0yQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbc%253D&md5=3ef990cab5dd2c6f9e179d99c066de5d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C553%253A%253AAID-JCC3%253E3.0.CO%253B2-T%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520III.%2520Molecular%2520geometries%2520and%2520vibrational%2520frequencies%2520for%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D553%26epage%3D586%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C553%3A%3AAID-JCC3%3E3.0.CO%3B2-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>â <span class="NLM_lpage">519</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+molecular+force+field.+I.+Basis%2C+form%2C+scope%2C+parameterization%2C+and+performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Ã root mean square [rms]), bond angles (1.2Â° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0ljACmm63yDKrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520I.%2520Basis%252C%2520form%252C%2520scope%252C%2520parameterization%252C%2520and%2520performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. V. Extension of MMFF94 using experimental data, additional computational data, and empirical rules</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">616</span>â <span class="NLM_lpage">641</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<616::AID-JCC5>3.0.CO;2-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=616-641&author=T.+A.+Halgren&title=Merck+molecular+force+field.+V.+Extension+of+MMFF94+using+experimental+data%2C+additional+computational+data%2C+and+empirical+rules&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. V. Extension of MMFF94 using experimental data, additional computational data, and empirical rules</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">616-41</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article describes the extension of the Merck Mol. Force Field (MMFF94) to a much broader range of org. systems.  It also describes a preliminary parameterization of MMFF94 for the hydronium and hydroxide ions and for various halide, alkali, and alkali earth ions as well as for such "protein" metals as Zn2+, Ca2+, Cu2+, Cu+, Fe2+, and Fe3+.  The extension employed computational data on charge distributions, mol. geometries, and conformational energies for a series of oxysulfur(particularly sulfonamide) and oxyphosphorous compds. and for a diverse set of small mols. and ions approx. 2800 good-quality structures extd. from the Cambridge Structural Database (CSD).  Some of the addnl. computational data were used to extend the explicit parametrization of electrostatic interactions and to more widely define a useful additive approxn. for the "bond polarity" and computational data served to define ref. bond lengths and angles that the extended force fields uses in conjunction with force consts. obtained from carefully calibrated empirical rules.  The extended torsion parameters consist partly of explicit parameters derived to reproduce MP2/6-31G* conformational energies and partly of "default parameters" provided by empirical rules patterned after those used in DREIDING and UFF but calibrated, where possible, against computationally derived MMFF94 torsion parameters.  Comparisons to exptl. data show that MMFF94 reproduces crystallog. bond lengths and bond angles with relatively modest root mean square (rms) deviations of â¼0.02 Ã and 2Â°, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4ynbnad1JgbVg90H21EOLACvtfcHk0ljACmm63yDKrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmtr8%253D&md5=7741ad82491c06e849077b18c3dd6778</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C616%253A%253AAID-JCC5%253E3.0.CO%253B2-X%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520V.%2520Extension%2520of%2520MMFF94%2520using%2520experimental%2520data%252C%2520additional%2520computational%2520data%252C%2520and%2520empirical%2520rules%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D616%26epage%3D641%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C616%3A%3AAID-JCC5%3E3.0.CO%3B2-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck molecular force field. II. MMFF94 van der Waals and electrostatic parameters for intermolecular interactions</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">520</span>â <span class="NLM_lpage">552</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<520::AID-JCC2>3.0.CO;2-W</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=520-552&author=T.+A.+Halgren&title=Merck+molecular+force+field.+II.+MMFF94+van+der+Waals+and+electrostatic+parameters+for+intermolecular+interactions&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. II. MMFF94 van der Waals and electrostatic parameters for intermolecular interactions</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">520-52</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article defines the parameterization and performance of MMFF94 for intermol. interactions.  It specifies the novel "buffered" functional forms used for treating van der Waals (vdW) and electrostatic interactions, and describes the use of: (1) high quality ab initio data to parameterize vdW interactions involving aliph. hydrogens; and (2) HF/6-31 G* calcns. on hydrogen-bonded complexes to parameterize nonbonded interactions in polar systems.  Comparisons show that appropriate trends in the HF/6-31G* data are well reproduced by MMFF94 and that intermol. interaction energies and geometries closely parallel those given by the highly regarded OPLS force field.  A proper balance between solvent-solvent, solvent-solute, and solute-solute interactions, critically important for prospective success in aq. simulations, thus appears to be attained.  Comparison of MMFF94, OPLS, CHELPG electrostatic potential fit, QEq, Gasteiger, and Abraham charges for 20 small mols. and ions also shows the close correspondence between MMFF94 and OPLS.  As do OPLS and all current, widely used force fields, MMFF94 employs "effective pair potentials" which incorporate in an averaged way the increases in polarity which occur in high dielec. media.  Some limitations of this approach are discussed and suggestions for possible enhancements to MMFF94's functional form are noted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX0kuyu0af2rVg90H21EOLACvtfcHk0ljACmm63yDKrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbY%253D&md5=b6f7f0fcddcee6aeb7cdba747f594070</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C520%253A%253AAID-JCC2%253E3.0.CO%253B2-W%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520molecular%2520force%2520field.%2520II.%2520MMFF94%2520van%2520der%2520Waals%2520and%2520electrostatic%2520parameters%2520for%2520intermolecular%2520interactions%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D520%26epage%3D552%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C520%3A%3AAID-JCC2%3E3.0.CO%3B2-W" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgFlxkX99CF_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>â <span class="NLM_lpage">1749</span>, <span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgFlxkX99CF_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>â <span class="NLM_lpage">6196</span>, <span class="refDoi">Â DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lgFlxkX99CF_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK <sub>a</sub> prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0liWDVkrsSv26A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">591</span>â <span class="NLM_lpage">604</span>, <span class="refDoi">Â DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=591-604&author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&doi=10.1007%2Fs10822-010-9349-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0liWDVkrsSv26A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D591%26epage%3D604%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousido-Siah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitschler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podjarny, A.</span></span> <span> </span><span class="NLM_article-title">New insights into the enzymatic mechanism of human chitotriosidase (CHIT1) catalytic domain by atomic resolution X-ray diffraction and hybrid QM/MM</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1455</span>â <span class="NLM_lpage">1470</span>, <span class="refDoi">Â DOI: 10.1107/S139900471500783X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS139900471500783X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=26143917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gnu7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1455-1470&author=F.+Fadelauthor=Y.+Zhaoauthor=R.+Cachauauthor=A.+Cousido-Siahauthor=F.+X.+Ruizauthor=K.+Harlosauthor=E.+Howardauthor=A.+Mitschlerauthor=A.+Podjarny&title=New+insights+into+the+enzymatic+mechanism+of+human+chitotriosidase+%28CHIT1%29+catalytic+domain+by+atomic+resolution+X-ray+diffraction+and+hybrid+QM%2FMM&doi=10.1107%2FS139900471500783X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into the enzymatic mechanism of human chitotriosidase (CHIT1) catalytic domain by atomic resolution X-ray diffraction and hybrid QM/MM</span></div><div class="casAuthors">Fadel, Firas; Zhao, Yuguang; Cachau, Raul; Cousido-Siah, Alexandra; Ruiz, Francesc X.; Harlos, Karl; Howard, Eduardo; Mitschler, Andre; Podjarny, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1455-1470</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Chitotriosidase (CHIT1) is a human chitinase belonging to the highly conserved glycosyl hydrolase family 18 (GH18).  GH18 enzymes hydrolyze chitin, an N-acetylglucosamine polymer synthesized by lower organisms for structural purposes.  Recently, CHIT1 has attracted attention owing to its upregulation in immune-system disorders and as a marker of Gaucher disease.  The 39 kDa catalytic domain shows a conserved cluster of three acidic residues, Glu140, Asp138 and Asp136, involved in the hydrolysis reaction.  Under an excess concn. of substrate, CHIT1 and other homologues perform an addnl. activity, transglycosylation.  To understand the catalytic mechanism of GH18 chitinases and the dual enzymic activity, the structure and mechanism of CHIT1 were analyzed in detail.  The resoln. of the crystals of the catalytic domain was improved from 1.65 Ã (PDB entry 1waw) to 0.95-1.10 Ã for the apo and pseudo-apo forms and the complex with chitobiose, allowing the detn. of the protonation states within the active site.  This information was extended by hybrid quantum mechanics/mol. mechanics (QM/MM) calcns.  The results suggest a new mechanism involving changes in the conformation and protonation state of the catalytic triad, as well as a new role for Tyr27, providing new insights into the hydrolysis and transglycosylation activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ62bad1_u27Vg90H21EOLACvtfcHk0liWDVkrsSv26A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gnu7nP&md5=6ef30a3b71be3f2568d6e73be01b26d9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS139900471500783X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS139900471500783X%26sid%3Dliteratum%253Aachs%26aulast%3DFadel%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DCachau%26aufirst%3DR.%26aulast%3DCousido-Siah%26aufirst%3DA.%26aulast%3DRuiz%26aufirst%3DF.%2BX.%26aulast%3DHarlos%26aufirst%3DK.%26aulast%3DHoward%26aufirst%3DE.%26aulast%3DMitschler%26aufirst%3DA.%26aulast%3DPodjarny%26aufirst%3DA.%26atitle%3DNew%2520insights%2520into%2520the%2520enzymatic%2520mechanism%2520of%2520human%2520chitotriosidase%2520%2528CHIT1%2529%2520catalytic%2520domain%2520by%2520atomic%2520resolution%2520X-ray%2520diffraction%2520and%2520hybrid%2520QM%252FMM%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D1455%26epage%3D1470%26doi%3D10.1107%2FS139900471500783X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span> <span> </span><span class="NLM_article-title">Structure, catalysis and inhibition of <i>Of</i>Chi-h, the Lepidoptera-exclusive insect chitinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">2080</span>â <span class="NLM_lpage">2088</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M116.755330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1074%2Fjbc.M116.755330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=28053084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVGisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2080-2088&author=T.+Liuauthor=L.+Chenauthor=Y.+Zhouauthor=X.+Jiangauthor=Y.+Duanauthor=Q.+Yang&title=Structure%2C+catalysis+and+inhibition+of+OfChi-h%2C+the+Lepidoptera-exclusive+insect+chitinase&doi=10.1074%2Fjbc.M116.755330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, Catalysis, and Inhibition of OfChi-h, the Lepidoptera-exclusive Insect Chitinase</span></div><div class="casAuthors">Liu, Tian; Chen, Lei; Zhou, Yong; Jiang, Xi; Duan, Yanwei; Yang, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2080-2088</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chitinase-h (Chi-h) is of special interest among insect chitinases due to its exclusive distribution in lepidopteran insects and high sequence identity with bacterial and baculovirus homologs.  Here OfChi-h, a Chi-h from Ostrinia furnacalis, was investigated.  Crystal structures of both OfChi-h and its complex with chitoheptaose, (GlcN)7, reveal that OfChi-h possesses a long and asym. substrate binding cleft, which is a typical characteristics of a processive exo-chitinase.  The structural comparison between OfChi-h and its bacterial homolog SmChiA uncovered two phenylalanine-to-tryptophan site variants in OfChi-h at subsites +2 and possibly -7.  The F232W/F396W double mutant endowed SmChiA with higher hydrolytic activities toward insol. substrates, such as insect cuticle, Î±-chitin, and chitin nanowhisker.  An enzymic assay demonstrated that OfChi-h outperformed OfChtI, an insect endo-chitinase, toward the insol. substrates, but showed lower activity toward the sol. substrate ethylene glycol chitin.  Furthermore, OfChi-h was found to be inhibited by N,N',N''-trimethylglucosamine-N,N',N'',N'''-tetraacetylchitotetraose (TMG-(GlcNAc)4), a substrate analog which can be degraded into TMG-(GlcNAc)1-2.  Injection of TMG-(GlcNAc)4 into 5th-instar O. furnacalis larvae led to severe defects in pupation.  This work provides insights into a molting-indispensable insect chitinase that is phylogenetically closer to bacterial chitinases than insect chitinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQLeeZuN8zRrVg90H21EOLACvtfcHk0lhjAxDACuUIpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVGisb8%253D&md5=0a7fc8a49c6f9a8e57b9f924a6bcc0a5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.755330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.755330%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DQ.%26atitle%3DStructure%252C%2520catalysis%2520and%2520inhibition%2520of%2520OfChi-h%252C%2520the%2520Lepidoptera-exclusive%2520insect%2520chitinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D2080%26epage%3D2088%26doi%3D10.1074%2Fjbc.M116.755330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirono, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunazuka, T.</span></span> <span> </span><span class="NLM_article-title">Observation of the controlled assembly of preclick components in the in situ click chemistry generation of a chitinase inhibitor</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15892</span>â <span class="NLM_lpage">15897</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1315049110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1073%2Fpnas.1315049110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=24043811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15892-15897&author=T.+Hiroseauthor=N.+Maitaauthor=H.+Goudaauthor=J.+Kosekiauthor=T.+Yamamotoauthor=A.+Sugawaraauthor=H.+Nakanoauthor=S.+Hironoauthor=K.+Shiomiauthor=T.+Watanabeauthor=H.+Taniguchiauthor=K.+B.+Sharplessauthor=S.+Omuraauthor=T.+Sunazuka&title=Observation+of+the+controlled+assembly+of+preclick+components+in+the+in+situ+click+chemistry+generation+of+a+chitinase+inhibitor&doi=10.1073%2Fpnas.1315049110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Observation of the controlled assembly of preclick components in the in situ click chemistry generation of a chitinase inhibitor</span></div><div class="casAuthors">Hirose, Tomoyasu; Maita, Nobuo; Gouda, Hiroaki; Koseki, Jun; Yamamoto, Tsuyoshi; Sugawara, Akihiro; Nakano, Hirofumi; Hirono, Shuichi; Shiomi, Kazuro; Watanabe, Takeshi; Taniguchi, Hisaaki; Sharpless, K. Barry; Omura, Satoshi; Sunazuka, Toshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">15892-15897,S15892/1-S15892/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Huisgen cycloaddn. of azides and alkynes, accelerated by target biomols., termed "in situ click chem.," has been successfully exploited to discover highly potent enzyme inhibitors.  We have previously reported a specific Serratia marcescens chitinase B (SmChiB)-templated syn-triazole inhibitor generated in situ from an azide-bearing inhibitor and an alkyne fragment.  Several in situ click chem. studies have been reported.  Although some mechanistic evidence has been obtained, such as X-ray anal. of [protein]-["click ligand"] complexes, indicating that proteins act as both mold and template between unique pairs of azide and alkyne fragments, to date, observations have been based solely on "postclick" structural information.  Here, we describe crystal structures of SmChiB complexed with an azide ligand and an O-allyl oxime fragment as a mimic of a click partner, revealing a mechanism for accelerating syn-triazole formation, which allows generation of its own distinct inhibitor.  We have also performed d. functional theory calcns. based on the X-ray structure to explore the acceleration of the Huisgen cycloaddn. by SmChiB.  The d. functional theory calcns. reasonably support that SmChiB plays a role by the cage effect during the pre-translation and post-translation states of selective syn-triazole click formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvlga4V4v-ZbVg90H21EOLACvtfcHk0lhjAxDACuUIpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbrL&md5=b605dc38fae207c49cd915ac28f2d2f1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315049110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315049110%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DT.%26aulast%3DMaita%26aufirst%3DN.%26aulast%3DGouda%26aufirst%3DH.%26aulast%3DKoseki%26aufirst%3DJ.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DSugawara%26aufirst%3DA.%26aulast%3DNakano%26aufirst%3DH.%26aulast%3DHirono%26aufirst%3DS.%26aulast%3DShiomi%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DTaniguchi%26aufirst%3DH.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26aulast%3DOmura%26aufirst%3DS.%26aulast%3DSunazuka%26aufirst%3DT.%26atitle%3DObservation%2520of%2520the%2520controlled%2520assembly%2520of%2520preclick%2520components%2520in%2520the%2520in%2520situ%2520click%2520chemistry%2520generation%2520of%2520a%2520chitinase%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D15892%26epage%3D15897%26doi%3D10.1073%2Fpnas.1315049110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>â <span class="NLM_lpage">674</span>, <span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhjAxDACuUIpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkÃ³czi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. D.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>â <span class="NLM_lpage">221</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgyMFyTCQekbQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520D.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr. D.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgyMFyTCQekbQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%2520D.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>â <span class="NLM_lpage">291</span>, <span class="refDoi">Â DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lgyMFyTCQekbQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealey, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyasu, S.</span></span> <span> </span><span class="NLM_article-title">Isolation and analysis of group 2 innate lymphoid cells in mice</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">792</span>â <span class="NLM_lpage">806</span>, <span class="refDoi">Â DOI: 10.1038/nprot.2015.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=10.1038%2Fnprot.2015.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=25927389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVWqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=792-806&author=K.+Moroauthor=K.+N.+Ealeyauthor=H.+Kabataauthor=S.+Koyasu&title=Isolation+and+analysis+of+group+2+innate+lymphoid+cells+in+mice&doi=10.1038%2Fnprot.2015.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and analysis of group 2 innate lymphoid cells in mice</span></div><div class="casAuthors">Moro, Kazuyo; Ealey, Kafi N.; Kabata, Hiroki; Koyasu, Shigeo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-806</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent studies have identified distinct subsets of innate lymphocytes, collectively called innate lymphoid cells (ILCs), which lack antigen receptor expression but produce various effector cytokines.  Group 2 ILCs (ILC2s) respond to epithelial cell-derived cytokines such as interleukin (IL)-25, IL-33 and thymic stromal lymphopoietin (TSLP), produce large amts. of type 2 cytokines, and have a key role in anti-helminth innate immunity and in the pathophysiol. of allergic inflammation.  The reported phenotypic characteristics of mouse ILC2s vary, depending on the tissue source and prepn. method.  This protocol describes improved methods for tissue-specific isolation and anal. of mouse ILC2s of high purity and yield from fat tissue, lung, bronchoalveolar lavage fluid (BALF) and small intestine.  These improved methods are the result of our thorough investigation of enzymes used for tissue digestion, methods for the elimination of undesired cells, and a combination of antibodies for the detection and isolation of ILC2s.  In addn., this new protocol now enables the isolation of ILC2s of high yield, even from inflamed tissues.  Depending on the tissue being analyzed, it takes â¼2-4 h for isolation and flow cytometric anal. of ILC2s from the various tissues of a single mouse and â¼4-8 h to sort purified ILC2s from pooled tissues of multiple mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoveg-q0y91HLVg90H21EOLACvtfcHk0liFIqDFhnyzmA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVWqtLY%253D&md5=0f2824429cf66c4b2edf3e1909c58b30</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.047%26sid%3Dliteratum%253Aachs%26aulast%3DMoro%26aufirst%3DK.%26aulast%3DEaley%26aufirst%3DK.%2BN.%26aulast%3DKabata%26aufirst%3DH.%26aulast%3DKoyasu%26aufirst%3DS.%26atitle%3DIsolation%2520and%2520analysis%2520of%2520group%25202%2520innate%2520lymphoid%2520cells%2520in%2520mice%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D792%26epage%3D806%26doi%3D10.1038%2Fnprot.2015.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJR','PDB','6JJR'); return false;">PDB: 6JJR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK9','PDB','6JK9'); return false;">PDB: 6JK9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JMN','PDB','6JMN'); return false;">PDB: 6JMN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JK6','PDB','6JK6'); return false;">PDB: 6JK6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JKF','PDB','6JKF'); return false;">PDB: 6JKF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WL1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WL1','PDB','3WL1'); return false;">PDB: 3WL1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RM8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RM8','PDB','3RM8'); return false;">PDB: 3RM8</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20168"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01154?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01154</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv">CSV</a>)</p></li><li><p class="inline">Docked complex of compound <b>40</b> in hAMCase (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Output file from PAINS filtering (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_003.txt">TXT</a>, <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_004.txt">TXT</a>, <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_005.txt">TXT</a>)</p></li><li><p class="inline">Selectivity profile of compound <b>40</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_006.xlsx">XLSX</a>)</p></li><li><p class="inline">Dipyrido-pyrimidine scaffold and chemical structures at R1 to R4 positions; inhibitory activities for pairs of compounds that differ at the R2 or R3 position; inhibitory activities for each functional group at the R1 position; inhibitory activities for each functional group at the R4 position; sequence alignment of <i>Hs</i>Chit1, hAMCase, <i>Of</i>Chi-h, and <i>Sm</i>ChiB; docking-predicted structure of <b>40</b> with hAMCase; distribution of binding pocket volumes of <i>Hs</i>Chit1 and hAMCase; ITC profiles of compound <b>53</b> binding to <i>Hs</i>Chit1, <i>Sm</i>ChiB, and <i>Of</i>Chi-h; inhibition rates of compounds identified in the first round; inhibition rates of compounds identified in the second round; inhibition rates of compounds identified in the third round; selectivity profile of <b>40</b> against a selected set of diverse proteins; ranking of the first-round hits for each method in the hierarchical virtual screening protocols; ranking of the second-round hits for each method in the hierarchical virtual screening protocol; the Dixon plots for <i>K<sub>i</sub></i> determination; <sup>1</sup>H NMR or MS spectra of tested compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_001.csv">jm9b01154_si_001.csv (7.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_002.pdb">jm9b01154_si_002.pdb (470.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_003.txt">jm9b01154_si_003.txt (6.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_004.txt">jm9b01154_si_004.txt (8.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_005.txt">jm9b01154_si_005.txt (8.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_006.xlsx">jm9b01154_si_006.xlsx (498.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01154/suppl_file/jm9b01154_si_007.pdf">jm9b01154_si_007.pdf (4.07 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the chitinase-inhibitor complexes described in this paper have been deposited with PDB with accession codes: 6JJR (<i>Hs</i>Chit1/<b>40</b>), 6JK6 (<i>Hs</i>Chit1/<b>53</b>), 6JK9 (<i>Sm</i>ChiB/<b>40</b>), 6JKF (<i>Sm</i>ChiB/<b>53</b>), and 6JMN (<i>Of</i>Chi-h/<b>53</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01154&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01154%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-3%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01154" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d2cd6ad13ddd","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
