
<html lang="en"     class="pb-page"  data-request-id="5383f546-ef71-4185-bcc1-029078e96194"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.8b01025;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-1;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Proline- and Arginine-Rich Peptides as Flexible Allosteric Modulators of Human Proteasome Activity" /></meta><meta name="dc.Creator" content="Małgorzata  Giżyńska" /></meta><meta name="dc.Creator" content="Julia  Witkowska" /></meta><meta name="dc.Creator" content="Przemysław  Karpowicz" /></meta><meta name="dc.Creator" content="Rafał  Rostankowski" /></meta><meta name="dc.Creator" content="Estrella S.  Chocron" /></meta><meta name="dc.Creator" content="Andrew M.  Pickering" /></meta><meta name="dc.Creator" content="Pawel  Osmulski" /></meta><meta name="dc.Creator" content="Maria  Gaczynska" /></meta><meta name="dc.Creator" content="Elżbieta  Jankowska" /></meta><meta name="dc.Description" content="Proline- and arginine-rich peptide PR11 is an allosteric inhibitor of 20S proteasome. We modified its sequence inter alia by introducing HbYX, RYX, or RHbX C-terminal extensions (Hb, hydrophobic mo..." /></meta><meta name="Description" content="Proline- and arginine-rich peptide PR11 is an allosteric inhibitor of 20S proteasome. We modified its sequence inter alia by introducing HbYX, RYX, or RHbX C-terminal extensions (Hb, hydrophobic mo..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 19, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01025" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01025" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01025" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01025" /></link>
        
    
    

<title>Proline- and Arginine-Rich Peptides as Flexible Allosteric Modulators of Human Proteasome Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01025" /></meta><meta property="og:title" content="Proline- and Arginine-Rich Peptides as Flexible Allosteric Modulators of Human Proteasome Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0010.jpeg" /></meta><meta property="og:description" content="Proline- and arginine-rich peptide PR11 is an allosteric inhibitor of 20S proteasome. We modified its sequence inter alia by introducing HbYX, RYX, or RHbX C-terminal extensions (Hb, hydrophobic moiety; R, arginine; Y, tyrosine; X, any residue). Consequently, we were able to improve inhibitory potency or to convert inhibitors into strong activators: the former with an aromatic penultimate Hb residue and the latter with the HbYX motif. The PR peptide activator stimulated 20S proteasome in vitro to efficiently degrade protein substrates, such as α-synuclein and enolase, but also activated proteasome in cultured fibroblasts. The positive and negative PR modulators differently influenced the proteasome conformational dynamics and affected opening of the substrate entry pore. The resolved crystal structure showed PR inhibitor bound far from the active sites, at the proteasome outer face, in the pocket used by natural activators. Our studies indicate the opportunity to tune proteasome activity by allosteric regulators based on PR peptide scaffold." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01025"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01025">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01025&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01025&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01025&amp;href=/doi/10.1021/acs.jmedchem.8b01025" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 359-370</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00994" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00192" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Proline- and Arginine-Rich Peptides as Flexible Allosteric Modulators of Human Proteasome Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Małgorzata Giżyńska</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Małgorzata Giżyńska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ma%C5%82gorzata++Gi%C5%BCy%C5%84ska">Małgorzata Giżyńska</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Witkowska</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Witkowska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Witkowska">Julia Witkowska</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Przemysław Karpowicz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Przemysław Karpowicz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Przemys%C5%82aw++Karpowicz">Przemysław Karpowicz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rafał Rostankowski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rafał Rostankowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rafa%C5%82++Rostankowski">Rafał Rostankowski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Estrella S. Chocron</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Estrella S. Chocron</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, The Barshop Institute for Longevity and Aging Studies, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Estrella+S.++Chocron">Estrella S. Chocron</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew M. Pickering</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew M. Pickering</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, The Barshop Institute for Longevity and Aging Studies, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+M.++Pickering">Andrew M. Pickering</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pawel Osmulski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pawel Osmulski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Institute of Biotechnology, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pawel++Osmulski">Pawel Osmulski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maria Gaczynska</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria Gaczynska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Institute of Biotechnology, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</div></div><span class="conrtib-corresp"><strong>*</strong>M.G.: e-mail, <a href="/cdn-cgi/l/email-protection#f49395978e8d9a879f95b481809c87978795da919081"><span class="__cf_email__" data-cfemail="cbacaaa8b1b2a5b8a0aa8bbebfa3b8a8b8aae5aeafbe">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Gaczynska">Maria Gaczynska</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Elżbieta Jankowska</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elżbieta Jankowska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</div></div><span class="conrtib-corresp"><strong>*</strong>E.J.: e-mail, <a href="/cdn-cgi/l/email-protection#98fdf4e2faf1fdecf9b6f2f9f6f3f7efebf3f9d8edffb6fdfcedb6e8f4"><span class="__cf_email__" data-cfemail="33565f49515a5647521d59525d585c444058527346541d5657461d435f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=El%C5%BCbieta++Jankowska">Elżbieta Jankowska</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1099-8885" title="Orcid link">http://orcid.org/0000-0002-1099-8885</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01025&amp;href=/doi/10.1021%2Facs.jmedchem.8b01025" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 359–370</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 19, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 June 2018</li><li><span class="item_label"><b>Published</b> online</span>19 November 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 January 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01025" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01025</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01025"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1768</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">14</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01025" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Proline- and Arginine-Rich Peptides as Flexible Allosteric Modulators of Human Proteasome Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Małgorzata&quot;,&quot;last_name&quot;:&quot;Giżyńska&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Witkowska&quot;},{&quot;first_name&quot;:&quot;Przemysław&quot;,&quot;last_name&quot;:&quot;Karpowicz&quot;},{&quot;first_name&quot;:&quot;Rafał&quot;,&quot;last_name&quot;:&quot;Rostankowski&quot;},{&quot;first_name&quot;:&quot;Estrella&quot;,&quot;last_name&quot;:&quot;S. Chocron&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;M. Pickering&quot;},{&quot;first_name&quot;:&quot;Pawel&quot;,&quot;last_name&quot;:&quot;Osmulski&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Gaczynska&quot;},{&quot;first_name&quot;:&quot;Elżbieta&quot;,&quot;last_name&quot;:&quot;Jankowska&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;359-370&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01025&quot;},&quot;abstract&quot;:&quot;Proline- and arginine-rich peptide PR11 is an allosteric inhibitor of 20S proteasome. We modified its sequence inter alia by introducing HbYX, RYX, or RHbX C-terminal extensions (Hb, hydrophobic moiety; R, arginine; Y, tyrosine; X, any residue). Consequently, we were able to improve inhibitory potency or to convert inhibitors into strong activators: the former with an aromatic penultimate Hb residue and the latter with the HbYX motif. The PR peptide activator stimulated 20S proteasome in vitro to efficiently degrade protein substrates, such as α-synuclein and enolase, but also activated proteasome in cultured fibroblasts. The positive and negative PR modulators differently influenced the proteasome conformational dynamics and affected opening of the substrate entry pore. The resolved crystal structure showed PR inhibitor bound far from the active sites, at the proteasome outer face, in the pocket used by natural activators. Our studies indicate the opportunity to tune proteasome activity by allosteric regulat&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01025&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01025" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01025&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01025" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01025&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01025" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01025&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01025&amp;href=/doi/10.1021/acs.jmedchem.8b01025" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01025" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01025" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01025%26sid%3Dliteratum%253Aachs%26pmid%3D30452262%26genre%3Darticle%26aulast%3DGi%25C5%25BCy%25C5%2584ska%26date%3D2019%26atitle%3DProline-%2Band%2BArginine-Rich%2BPeptides%2Bas%2BFlexible%2BAllosteric%2BModulators%2Bof%2BHuman%2BProteasome%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D1%26spage%3D359%26epage%3D370%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291631" title="Modulators">Modulators</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/jmcmar.2019.62.issue-1/20190110/jmcmar.2019.62.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Proline- and arginine-rich peptide PR11 is an allosteric inhibitor of 20S proteasome. We modified its sequence inter alia by introducing HbYX, RYX, or RHbX C-terminal extensions (Hb, hydrophobic moiety; R, arginine; Y, tyrosine; X, any residue). Consequently, we were able to improve inhibitory potency or to convert inhibitors into strong activators: the former with an aromatic penultimate Hb residue and the latter with the HbYX motif. The PR peptide activator stimulated 20S proteasome in vitro to efficiently degrade protein substrates, such as α-synuclein and enolase, but also activated proteasome in cultured fibroblasts. The positive and negative PR modulators differently influenced the proteasome conformational dynamics and affected opening of the substrate entry pore. The resolved crystal structure showed PR inhibitor bound far from the active sites, at the proteasome outer face, in the pocket used by natural activators. Our studies indicate the opportunity to tune proteasome activity by allosteric regulators based on PR peptide scaffold.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/62/1"><issue-title>Allosteric Modulators</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88384" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88384" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ubiquitin–proteasome system (UPS) is one of the two main proteolytic pathways existing in human cells.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It is involved in regulation of all aspects of cellular physiology, and aberrations in its function are intricately related to the well-being of cells, organs, and organisms.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Proteasome inhibition became an efficient way to kill rapidly proliferating, and thus addicted to proteasome activity, cancer cells. Three FDA-approved drugs specifically inhibiting the proteasome, bortezomib, carfilzomib, and ixazomib, revolutionized the therapy of hematological cancers.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Multiple other inhibitors are in clinical trials.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, killing oversensitive cells by abolition of the proteasome activity is not the only way to use proteasome-targeting compounds. For example, mild inhibition of the proteasome may protect muscle cells from excessive protein degradation that occurs in disease-related cachexia or aging-related sarcopenia.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> On the other hand, activation of the proteasome should help to attenuate these age-related diseases which result from the diminished proteasome activity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">The 26S proteasome, which is a central element of the UPS, comprises one or two 19S regulatory particles and a barrel-shaped catalytic core called 20S proteasome or core particle (CP). The 19S module enables recognition of proteins marked for degradation by polyubiquitin chains, their deubiquitination, unfolding, and translocation into the catalytic core.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The core consists of 28 subunits, which are arranged in a stack of four heptameric rings in an αββα fashion.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The two outer α rings provide binding sites for activating or regulatory particles, whereas the catalytic activity resides within the inner β subunits. In eukaryotic proteasomes the catalytically active β subunits exhibit three different substrate cleavage preferences: caspase-like post acidic (C-L; β1/1′), trypsin-like post basic (T-L; β2/2′), and chymotrypsin-like post hydrophobic (ChT-L; β5/5′).<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> The tightly packed N-termini of the α-subunits form the gate, which restricts access of substrates to the catalytic chamber. Opening of the gate is promoted by docking of proteasome activators: 19S (PAN in <i>Archaea</i>), 11S (PA28/REG), or PA200.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11−14)</a> All these additional modules attach to the surface of the α ring (the “α-face”), anchoring in the pockets between the α subunits, but through transduction of allosteric signals they can influence the performance of the active sites.<a onclick="showRef(event, 'ref7 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref13 ref14">(7,13,14)</a></div><div class="NLM_p">Although most intracellular proteins are selectively targeted for degradation through ubiquitin tagging, an increasing number of proteins have been identified as undergoing ubiquitin-independent cleavage by the 20S core itself. The pool of its substrates includes proteins that have been partially or completely unfolded due to aging, mutations, or oxidation and also native proteins, which are intrinsically disordered or encompass large (>30 residues) disordered regions.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> A growing body of evidence indicates that the 20S core plays a major role in the clearance of proteins that can be precursors of toxic oligomeric species implicated in the pathogenesis of severe neurological disorders, such as Parkinson’s, Alzheimer’s, and Huntington’s diseases and amyotrophic lateral sclerosis.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a></div><div class="NLM_p">The extensive involvement of proteasome in human health and disease requires precise regulation of its activity and causes that small molecules with such capacity are of great interest. However, while the field of competitive inhibitors of the enzyme is mature, the concepts for noncompetitive allosteric regulation are only starting to emerge, mainly because of structural complexity of the proteasomes, which offers not only unique opportunities but also challenges to the rational design of allosteric regulators. Relatively few noncompetitive/allosteric small molecule regulators of the CP activities have been described so far. Among them are several compounds with a quinoline or imidazoline scaffold (inhibitors), derivatives of rapamycin (inhibitors), chlorpromazine (activators), or betulinic acid (inhibitors/activators).<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24 ref25 ref26">(20−26)</a> A promising direction in the design of proteasome modulators could be peptidic structures,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> since peptides and peptidomimetics can offer higher specificity and lower toxicity than low molecular weight compounds.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Peptide modulators can be derived from the binding regions of proteins which are natural proteasome regulators. One example of such an approach is a short, 10-residue C-terminal fragment of the RP subunit, Rpt5, which has been reported to activate the core in trans.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> We designed a 14-mer peptide based on the C-terminal fragment of Blm10 (a yeast ortholog of PA200), which stimulated human 20S proteasome’s activity 3-fold at 1 μM concentration.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The key feature of both these peptides is the three-residue C-terminal “HbYX” motif (hydrophobic-Tyr-any residue) through which the modulator docks in the pocket between α subunits and probably allosterically affects catalytic activity.<a onclick="showRef(event, 'ref13 ref29'); return false;" href="javascript:void(0);" class="ref ref13 ref29">(13,29)</a> Another example of the peptidic approach to the regulation of proteasome activity is PAP1 peptide, described by Dal Vechio et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> This peptide reportedly activated the proteasome and was able to prevent protein aggregation in a cellular model of amyotropic lateral sclerosis. The distinct group of peptidic modulators of the 20S proteasome are peptides and mimetics derived by us from the proteasome-binding viral protein HIV-1 Tat, which in vitro potently inhibited the core.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p last">Here, we focus our attention on the proline- and arginine-rich (PR) porcine cathelicidine PR39 peptide, which is a proteasome inhibitor with unique molecular and intracellular effects.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> Since it was reported that the 11-residue N-terminal fragment of PR39, PR11 (<b>11</b>), was sufficient to convey allosteric actions on the CP,<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> we set out to explore peptide regulators based on the scaffold of <b>11</b> and constructed from Arg-rich, Pro-rich, and HbYX-inspired motifs. Modifications introduced to the sequence of <b>11</b> allowed us to discover inhibitors more potent than the parent peptide but also compounds able to efficiently <i>stimulate</i> human 20S proteasome (h20S). Our studies have shed light on allosteric interactions that may be critical for the positive and negative regulation of the proteasome.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of the Peptides</h3><div class="NLM_p">Compound <b>11</b> has been described before as a noncompetitive and allosteric inhibitor of human and yeast proteasomes.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As a basis for our design, we distinguished three modules in the canonical structure of <b>11</b>: the N-terminal triple-Arg, the Pro-rich module (PPPP or PRPP), and the variable C-terminal fragment (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The RRR module, which has been proved essential for proteasome targeting by <b>11</b>, was present in all peptides. Some of the peptides (<b>3</b>, <b>4</b>, <b>10</b>, and <b>12</b>) were devoid of the Pro-rich module but had instead an extended C-terminal part. In <b>6</b>, <b>7</b>, and <b>8</b> both the extended C-terminus and Pro-rich module were included. The extension of the original sequence of <b>11</b> was inspired by the fact that an aromatic residue at the ultimate or penultimate position has been indicated as crucial for binding of allosteric modulators to proteasome.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Another rationale for such a modification was reported by Anbanandam et al., better inhibitory capacity of <b>11</b> with a Trp residue appended at the C-terminus.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In <b>1</b> and <b>2</b> the C-terminus was modified by incorporation of the full HbYX motif (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which has been proved to be a critical element of proteasome activators.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Amino Acid Sequence and Corresponding IC<sub>50</sub> of the Studied PR Modulators</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">MW [Da]</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">sequence</th><th class="colsep0 rowsep0" align="center">calculated</th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> ± SEM [μM]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">RRR-PRPP-<i>YLPR</i>-OH</td><td class="colsep0 rowsep0" align="left">1462.8746</td><td class="colsep0 rowsep0" align="left">1462.7694</td><td class="colsep0 rowsep0" align="left">0.095 ± 0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">RRR-PPPP-<b>LYA</b>-OH</td><td class="colsep0 rowsep0" align="left">1221.7095</td><td class="colsep0 rowsep0" align="left">1221.6209</td><td class="colsep0 rowsep0" align="left">N/A<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">RRR-PPPP-<b>YYA</b>-OH</td><td class="colsep0 rowsep0" align="left">1271.6888</td><td class="colsep0 rowsep0" align="left">1271.5974</td><td class="colsep0 rowsep0" align="left">N/A<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">RRR-<i>YLPR</i>-WG-OH</td><td class="colsep0 rowsep0" align="left">1258.7160</td><td class="colsep0 rowsep0" align="left">1258.6235</td><td class="colsep0 rowsep0" align="left">0.111 ± 0.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">RRR-<i>YLPR</i>-WG-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1257.7160</td><td class="colsep0 rowsep0" align="left">1257.6425</td><td class="colsep0 rowsep0" align="left">0.082 ± 0.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">RRR-PRPP-<i>YLPR</i>-FG-OH</td><td class="colsep0 rowsep0" align="left">1666.9645</td><td class="colsep0 rowsep0" align="left">1666.8566</td><td class="colsep0 rowsep0" align="left">0.099 ± 0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">RRR-PRPP-<i>YLPR</i>-WG-OH</td><td class="colsep0 rowsep0" align="left">1705.9754</td><td class="colsep0 rowsep0" align="left">1705.8556</td><td class="colsep0 rowsep0" align="left">0.050 ± 0.006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">RRR-PRPP-<i>YLPR</i>-WG-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1704.9754</td><td class="colsep0 rowsep0" align="left">1704.8704</td><td class="colsep0 rowsep0" align="left">0.113 ± 0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">RRR-YPR-WG-OH</td><td class="colsep0 rowsep0" align="left">1145.6319</td><td class="colsep0 rowsep0" align="left">1145.5477</td><td class="colsep0 rowsep0" align="left">0.166 ± 0.017</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">RRR-<i>YLPR</i>-<b>YA</b>-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1248.7156</td><td class="colsep0 rowsep0" align="left">1248.6411</td><td class="colsep0 rowsep0" align="left">0.154 ± 0.003</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Not applicable.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> PR Peptides as Inhibitors of Human 20S Proteasome</h3><div class="NLM_p">All PR peptides described so far were inhibitors of human 20S proteasome.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Accordingly, we found that the ChT-L peptidase activity of SDS-activated 20S was inhibited by all but two of our modulators (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">S1</a>). The IC<sub>50</sub> (the concentration causing 50% inhibition of activity, compared to the vehicle-treated control) was respectably below 200 nM for all inhibitors, with the lowest values obtained for <b>7</b> and <b>4</b> (50 and 82 nM, respectively; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The common motif in the sequences of <b>7</b> and <b>4</b> was a WG moiety, added to the C-terminal module of <b>11</b>. Since glycine is devoid of any productive side chain, it may be speculated that the indole ring of tryptophan is solely responsible for this improvement in the inhibitory capacity. This conclusion is in line with the results of Anbanandam et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and proves the positive effect that may be exerted by tryptophan incorporated as an ultimate/penultimate residue into the sequence of PR-type inhibitors. Further analysis of the structure–activity relationships in <b>3</b>–<b>8</b> indicates that other elements that distinguish their sequences do not affect their inhibitory propensity in a consistent manner. While the lack of the Pro-rich module diminished the inhibitory capacity of the analog with a C-terminal carboxylate (<b>3</b> vs <b>7</b>), the same modification was rather advantageous for the peptide terminated with an amide group (<b>8</b> vs <b>4</b>). Compound <b>10</b>, which has almost the same sequence as <b>3</b> except for the lack of Leu residue in the YLPR segment, was the weakest inhibitor in the tested set of compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The importance of the Leu residue may result either from its ability to furnish hydrophobic interactions necessary for efficient proteasome inhibition or from its ability to position other moieties in such a way as to enable their interactions with the 20S proteasome. The most striking result of the kinetic assays utilizing the activated 20S was a complete lack of inhibitory capacity of <b>1</b> and <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). These two compounds were equipped with the HbYX motif, with a tyrosine residue occupying the penultimate position and preceded by either Leu or another Tyr. On the other hand, when the penultimate aromatic residue Trp (<b>3</b>, <b>4</b>, <b>7</b>, <b>10</b>), Phe (<b>6</b>), or Tyr (<b>12</b>) was flanked by Arg from the YLPR/YPR segment, the peptides retained their inhibitory propensity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. ChT-L peptidase activity of SDS-activated human 20S proteasome was inhibited by selected PR peptides in a dose dependent manner, with the exception of <b>1</b> and <b>2</b>. <b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S1</a>) and <b>7</b> were better inhibitors than <b>11</b> (red line), whereas <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S1</a>) and <b>12</b> were less efficient. The enzyme activity was probed with the substrate Suc-LLVY-AMC. Each titration was performed in three independent replicates. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Apart from the ChT-L, we also tested the trypsin- and caspase-like activities to detect if there are any selectivity in the influence of our PR-analogs on the SDS-activated h20S proteasome. We observed that compounds <b>3</b>–<b>12</b> inhibited both the C-L and T-L activities but the caspase-like peptidase responded to much lower concentrations of PRs (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S2</a>). This response generally resembled the sensitivity of the ChT-L peptidase (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">S1</a>). The T-L activity diminished significantly only when the modulators were applied at their highest tested concentration (10 μM). <b>1</b> and <b>2</b> displayed very weak inhibitory propensity against the C-L and the complete lack of capability to inhibit the T-L peptidase (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Mechanism of Inhibition</h3><div class="NLM_p">We attempted to determine the mechanism of inhibition for the best two inhibitors, <b>4</b> and <b>7</b>, using Suc-LLVY-AMC as a substrate for ChT-L active centers. We tested three concentrations of each inhibitor and noticed that with increasing concentration the dose–response curves adopt a more pronounced sigmoidal shape, indicating a cooperative digest of the substrate (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Such an effect is usually observed for an allosteric mode of inhibition.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Inhibition of the ChT-L peptidase activity of h20S by <b>4</b> (a) or <b>7</b> (b) followed the mechanism of noncompetitive mixed inhibition. In this mechanism, proteasome preserves partial catalytic activity, while the inhibitor binding affinity also depends on occupancy of the active centers. Each titration was performed in three independent replicates. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Numerical analysis of the data, performed with the enzyme kinetics applications of OriginPro, pointed at the partial noncompetitive mixed inhibition model as the most probable mechanism to explain the PR peptide interactions with proteasome (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Calculated <i>K</i><sub>i</sub> values were 0.0101 ± 0.0055 μM and 0.0048 ± 0.0026 μM for <b>4</b> and <b>7</b>, respectively, which closely matched the calculated IC<sub>50</sub> values (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Kinetic Parameters of Inhibition of SDS-Activated h20S Proteasome</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">compound <b>4</b></th><th class="rowsep1 colsep0" colspan="2" align="center">compound <b>7</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinetic parameter</th><th class="colsep0 rowsep0" align="center">value</th><th class="colsep0 rowsep0" align="center">SEM</th><th class="colsep0 rowsep0" align="center">value</th><th class="colsep0 rowsep0" align="center">SEM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>max</sub> [s<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">13.46</td><td class="colsep0 rowsep0" align="left">0.91</td><td class="colsep0 rowsep0" align="left">13.64</td><td class="colsep0 rowsep0" align="left">0.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>M</sub> [μM]</td><td class="colsep0 rowsep0" align="left">14.02</td><td class="colsep0 rowsep0" align="left">5.55</td><td class="colsep0 rowsep0" align="left">17.42</td><td class="colsep0 rowsep0" align="left">5.79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>i</sub> [μM]</td><td class="colsep0 rowsep0" align="left">0.0101</td><td class="colsep0 rowsep0" align="left">0.0055</td><td class="colsep0 rowsep0" align="left">0.0048</td><td class="colsep0 rowsep0" align="left">0.0026</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">β</td><td class="colsep0 rowsep0" align="left">4.98 × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="left">4.20 × 10<sup>8</sup></td><td class="colsep0 rowsep0" align="left">1347</td><td class="colsep0 rowsep0" align="left">1370</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α</td><td class="colsep0 rowsep0" align="left">180878</td><td class="colsep0 rowsep0" align="left">7.65 × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="left">4620</td><td class="colsep0 rowsep0" align="left">2963</td></tr></tbody></table></div></div><div class="NLM_p last">The mixed inhibition indicates that affinity of binding of a substrate to the active center and of an inhibitor to its regulatory pocket is mutually cross-dependent. Furthermore, partial noncompetitive inhibition may explain the substantial remaining catalytic activity of proteasome in the presence of PR peptides. Apparently, these inhibitors can significantly slow formation of the substrate–enzyme complex or dissociation of the product but cannot completely block the catalytic cycle. It is worth noting that this partial inhibition, without total abolishment of proteasome activity even at high concentrations of an inhibitor, is typical for allosteric modulators and may be very beneficial when the cells treated with the drug are meant to survive rather than die from apoptosis. Both tested peptides presented similar kinetic parameters with a notable difference in <i>K</i><sub>i</sub> value, which designated <b>7</b> as the better inhibitor of the two (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). The proposed mechanism of inhibition is in contrast to the previously studied case of PR39, which was classified as consistent with a pure noncompetitive mode of action.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> A putative penetration of the catalytic chamber by a 39-residue poly-Pro-rich peptide is less plausible than the similar action performed by the short peptides considered here. The lack of a competitive component in the case of PR39 could be rationalized in this way.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> PR Inhibitors Impair Activity of h20S Stimulated with Rpt5 C-Terminal Peptide</h3><div class="NLM_p">Activation of the core proteasome with SDS is an established intervention for studying inhibitor efficiency;<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> however it can be considered an artificial way of enzyme stimulation. Therefore, we decided to also test our PR peptides using h20S activated with the 10-aa C-terminal peptide of the Rpt5 subunit of the 19S regulatory particle (KKK-ANLQ-<b>YYA</b>-OH, named here “Rpt5”). Rpt5 activated proteasome with an AC<sub>50</sub> of 3.14 μM and can be considered a “minimal RP” model for activity tests with proteasome regulators.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We used two concentrations of Rpt5: 1 μM, which stimulated the ChT-L activity nearly 4-fold, and a saturating 10 μM concentration (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S3</a>), which left the proteasome activity about 8-fold higher than the latent control.</div><div class="NLM_p">As demonstrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, both <b>7</b> and the original peptide <b>11</b> inhibited the Rpt5-activated proteasome. <b>7</b> was a more potent inhibitor than <b>11</b>, with an IC<sub>50</sub> of about 0.5 μM (0.56 μM with 1 μM Rpt5 and 0.50 μM with 10 μM Rpt5, respectively). The IC<sub>50</sub> of <b>11</b> was approximated as 1.60 μM (1 μM Rpt5) and 1.90 μM (10 μM Rpt5). The effect of <b>2</b> was quite interesting: it did not affect the performance of CP activated by 10 μM Rpt5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b) but was able to stimulate 2-fold the activity of CP treated with 1 μM Rpt5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). As a result, the CP activity rose 8-fold (4-fold activation by Rpt5 × 2-fold activation by <b>2</b>), which is similar to the activation achieved with the saturating concentration of Rpt5. This newly observed activity of <b>2</b> prompted us to test the effects of our PR peptides on the latent h20S.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>7</b> and <b>11</b> inhibited 20S proteasome activated with 1 μM (a) or 10 μM Rpt5 (b). IC<sub>50</sub> calculated for <b>7</b> did not depend on Rpt5 concentration, whereas for <b>11</b> it was slightly increased at the higher concentration of Rpt5. In contrast, in the presence of 1 μM Rpt5, <b>2</b> additively activated proteasome (a). This effect was abolished at 10 μM Rpt5 (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> There Are Efficient Proteasome Activators among PR-Type Modulators</h3><div class="NLM_p">PR analogs that displayed distinct inhibitory capacity against SDS-activated h20S (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">S1, and S2</a>) differed in their influence on the latent enzyme. At low concentrations, clustering around 1 μM, <b>6</b>, <b>11</b>, and <b>12</b> activated the ChT-L peptidase of proteasome, with the best-performing <b>12</b> achieving nearly 5-fold activation at 5 μM concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The activating capacity of <b>4</b> was weaker (maximum 2-fold at 5 μM concentration), while <b>7</b> practically did not activate h20S at all. The transient nature of the activation observed for compounds <b>12</b>, <b>11</b>, and <b>6</b> may be explained by presence of a low affinity secondary binding site that exercises the inhibitory actions on ChT-L activity. An alternative explanation would call for a negative cooperativity between binding sites where the occupation of the first binding site would allosterically modify the second binding site to turn it into the inhibitory site upon binding of the same compound. So far our analyses of enzyme kinetics performed in the presence of the compounds exclude the possibility that the digest products may play a role of inhibitors leading to the bell shaped dose–response.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ChT-L activity of the latent 20S proteasome, probed with the substrate Suc-LLVY-AMC, was strongly and diversely regulated by PR peptides. At low concentrations <b>6</b>, <b>11</b>, and <b>12</b> transiently activated the proteasome. <b>1</b> and <b>2</b> proved to be strong activators of the latent CP and induced typical sigmoidal dose response stimulation. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, <b>1</b> and <b>2</b> that did not inhibit the SDS-activated 20S proved to be strong activators of the latent CP. When probed with the substrate Suc-LLVY-AMC (ChT-L activity) they stimulated the enzyme up to 7-fold at 25 μM concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The caspase-like peptidase was even more sensitive to activation by the compounds; they stimulated it 5-fold to 7-fold at 10 μM concentration (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p">Proteasome has a dynamic structure, and even in the latent state about 25% of its molecules adopt a partially open conformation, which enables small molecules to penetrate the catalytic channel and be degraded.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Therefore, using only small fluorogenic substrates as probes can produce misleading results, especially in the case of identification of an activator.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> To verify the results obtained with Suc-LLVY-AMC, we included in our kinetic studies the internally quenched LFP nonapeptide, which has been described as a substrate that is very slowly hydrolyzed by the nonactivated 20S proteasome.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Using this substrate, we did not detect any, even transient, stimulating propensity of <b>3</b>, <b>4</b>, <b>6</b>, <b>7</b>, <b>8</b>, <b>10</b>, <b>11</b>, and <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">S5b</a>). The only PR analogs capable of efficiently, and in a dose dependent manner, stimulating ChT-L peptidase activity of the latent h20S were <b>1</b> and <b>2</b>. At the highest tested concentration (25 μM) these two peptides accelerated LFP hydrolysis 8- and 14-fold, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). In kinetic tests with the SDS-activated h20S <b>1</b> and <b>2</b> also behaved differently from other PRs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). While almost all analogs strongly inhibited degradation of the LFP substrate, <b>1</b> and <b>2</b> did not exert any inhibitory capacity, consistent with the results obtained with Suc-LLVY-AMC substrate. <b>7</b> and <b>8</b> were better inhibitors when compared to <b>11</b>, whereas <b>3</b> and <b>10</b> were much less efficient (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">S5a</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) ChT-L activity of the SDS-activated proteasome, probed with LFP substrate. Almost all analogs, except for <b>1</b> and <b>2</b>, inhibited h20S, with <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S5a</a>) and <b>7</b> being more efficient and <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S5a</a>) and <b>3</b> significantly less efficient than <b>11</b> (red line). (b) <b>1</b> and <b>2</b> were the only compounds among the studied set of PR analogs able to stimulate the activity of the latent h20S. The maximum activation effect reached 8-fold for <b>1</b> and 14-fold for <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The observed diverse effects of PR peptides on proteasome activity cannot be explained based on conformational differences between the positive and negative modulators. Although some structural diversity was detected when comparing the modulators’ CD spectra, it was not helpful in identification of activators and inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S6</a>). We thus turned our attention to the primary structure differences. There are two regions that differentiate the positive and negative modulators’ sequences: the proline-rich module and the C-terminal sequence. To check the influence of the first one, we have synthesized an analog of inhibitor <b>7</b> with the PRPP segment substituted with the module PPPP, present in the sequence of <b>1</b> and <b>2</b> activators. The peptide <b>7-4P</b> inhibited the chymotrypsin-like activity of the SDS-activated proteasome with similar effectiveness as <b>7</b> and was not able to activate this peptidase in the latent h20S (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S7</a>). The possible explanations of the observed diverse effects of PR peptides on h20S activity should be thus sought within the C-terminal sequence. <b>1</b> and <b>2</b> are furnished with the HbYX module (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which they share with protein activators such as PA200/Blm10, PAN, or Rpt2, 3, and 5 subunits of 19S. This motif has been proved to be a key factor in the mechanism of proteasome activation, due to the contacts delivered by the hydroxyl group of the penultimate Tyr with Gly19. This interaction causes a shifting of the Pro17 reverse turn in one or more α subunits, which results in a partially or fully opened entrance leading to the catalytic chamber.<a onclick="showRef(event, 'ref7 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref13">(7,13)</a> The stimulating propensity of <b>1</b> and <b>2</b> may result from similar interactions. It is also tempting to speculate that the limited activating potency of the remaining PR analogs may originate from shortening their C-terminal motif to only HbX (with “Hb” denoting a hydrophobic Trp or Phe residue), which precludes the canonic contacts with Gly19. Compound <b>12</b>, which displayed quite significant (although transient) activating capacity when probed with Suc-LLVY-AMC (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), comprises the penultimate Tyr in its sequence. However, in contrast to <b>1</b> and <b>2</b>, not the hydrophobic but basic arginine residue precedes the penultimate Tyr in <b>12</b>, forming an alternative RYX motif, similar to RHbX present in all PR analogs which displayed inhibitory capacity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Arginine possesses a long side chain grafted with the highly basic guanidyl group and thus may provide interactions not available to <b>1</b> and <b>2</b>. The interactions of both Arg and the penultimate Tyr may be responsible for the ambivalent properties of <b>12</b>, i.e., the concomitant demonstration of considerable inhibitory and activating capacity against the latent h20S.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> PR Peptides Affect Degradation of Disorganized/Unfolded Proteins</h3><div class="NLM_p">The model peptide substrates provide a convenient way to probe proteasome activity. However, utilization of protein substrates has the advantage of a better approximation of the kinetic challenges encountered in the cellular context. Therefore, we tested degradation of α-synuclein (α-syn) which, as a natively unfolded protein, can be degraded by 20S alone, without the assistance of the 19S regulatory complex.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a shows that PR modulators influence α-syn proteolysis differently. At 1 μM concentration the most efficient inhibitors were <b>6</b> and <b>7</b>, but at higher concentration <b>4</b>, <b>11</b>, and <b>12</b> impeded α-syn degradation to similar extent. <b>1</b> and <b>2</b> did not display any inhibitory capacity at 1 μM concentration. On the contrary, at 10 μM concentration, both compounds increased the level of α-syn hydrolysis. In the presence of <b>2</b>, h20S was able to almost completely degrade this protein (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of protein substrates by human 20S proteasome was strongly affected by PR peptides. Levels of the remaining substrates were determined based on SDS–PAGE electrophoretic separation of proteins incubated with 20S proteasome. Almost 70% of α-synuclein (a) and 50% of enolase (b) were degraded by 20S proteasome under the applied experimental conditions. Representative SDS–PAGE electrophoregrams with Coomassie-stained substrate bands are presented below the columns. At 10 μM, <b>1</b> and <b>2</b> accelerated the digest to completely degrade enolase. Other PR peptides, especially at 10 μM concentration, efficiently blocked degradation of the model proteins, with <b>6</b> and <b>7</b> being the best inhibitors. NS = not statistically significant; all other cases are statistically significant (<i>p</i> < 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further survey the catalytic response of 20S to PRs, we expanded our tests to enolase. This 436-residue thermolabile protein is an established substrate of 20S proteasome,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> although we found that it was less efficiently degraded by CP than α-syn (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b).</div><div class="NLM_p last">The enolase degradation assay confirmed that <b>6</b> and <b>7</b> are efficient inhibitors of proteolysis, whereas <b>11</b>, <b>4</b>, and <b>12</b> influence this process slightly less effectively. In contrast to α-syn, there were no distinct differences in the level of inhibition at 1 and 10 μM concentration of the inhibitor. Importantly, <b>1</b> and <b>2</b> stimulated degradation of enolase, a poor substrate, much better than of α-syn. Indeed, stimulation with 1 μM concentration of <b>2</b> led to more than 2-fold acceleration of enolase digestion. Surprisingly, 20S proteasome activated with 10 μM of either peptide completely degraded enolase under the employed reaction conditions. HPLC analysis of the products showed short fragments of the hydrolyzed protein. The detection of such products strongly indicates that the proteolytic activity of h20S was responsible for the disappearance of the enolase band at the electropherograms.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Stimulation of Proteasome Activity in Cell Culture</h3><div class="NLM_p">Intrigued by the strong potency of <b>2</b> as an activator of the peptidase and proteinase activity of the proteasome, we decided to test the performance of this peptide in cell culture. We chose human primary fibroblasts, which provide an advantageous system for investigating dynamic molecular regulatory processes without the confounding effects of a disease state. The cells were treated with <b>2</b> at concentrations ranging from 0.1 μM to 20 μM. Even the highest concentration of <b>2</b> did not exert any cytotoxic effects (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). Importantly, the total activity of proteasomes was significantly elevated in total cell lysates prepared from cells treated with <b>2</b>, as compared with the vehicle-treated cells. The activation was nearly 3-fold when 10 μM concentration of <b>2</b> was present in the cell culture medium (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). This important result indicates that the activating PR derivatives can be safely used in cellulo to stimulate UPS.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Treatment of human cultured fibroblasts with compound <b>2</b> (24 h) did not affect viability of the cells, even at high peptide concentrations (a). ChT-L activity of proteasome in cell extracts prepared from human fibroblasts treated with <b>2</b> was activated up to 3-fold in a dose dependent manner (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Dynamics of the Gate Probed by Atomic Force Microscopy</h3><div class="NLM_p">We had established earlier that the latent 20S proteasome can switch freely between two major conformational states: a prevailing “closed” state with no detectable gate in the α-face and a less populous state with AFM-detectable indentation in the gate area (the “indented” or “open” state).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Interaction of control proteasomes with a substrate was followed by a switch to a majority of “open” particles (about 75%). Consequently, we proposed an allosteric model of the gate movements with a positive feedback loop running between the gate and the active centers and enabling efficient passage of substrates and products to and from the catalytic chamber.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Treatment of proteasomes with the canonical PR peptides, <b>11</b> and PR39, induced a “shaky” conformation, with multiple shallow indentations indicating destabilization of the α-face.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Moreover, we noticed that while the images of “closed” particles were very uniform, the conformers with detectable indentation presented two distinct morphologies of the α-face: with a deep indentation surrounded by a symmetrical rim and with a shallow dip and irregular rims. Computational analysis of sections through the α-face of imaged control proteasome particles allowed distinction between three conformers, “closed” (as previously) as well as “intermediate” and “open”, replacing the previously described state with the α-face indentation. We found the control proteasome particles in a conformational equilibrium of nearly three-quarters of closed (71% ± 2%), 22% ± 3% of intermediate, and 7% ± 3% of open particles (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Interestingly, this refined partition of conformers in the control sample closely resembles the recently proposed cryo-EM based model of conformational states of the gate in 26S proteasome, where only about 8% of particles assumed the fully open conformation.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. PR peptides influenced the abundance of the α-face conformations of 20S proteasome, as detected by AFM. (a) In control, untreated samples, about 70% of molecules had their central channel covered and classified as a closed gate conformation. About 7% of molecules had their gate completely open, and the remaining 22% were in the process of switching between these conformations and were classified as intermediates. In the presence of an activating 1 μM concentration of <b>2</b>, the abundance of open and intermediate conformations increased to 54% (18% and 36%, respectively). In contrast, a 1 μM concentration of <b>7</b> decreased the number of open conformers to less than 3%. The higher concentration of <b>7</b> pushed proteasome to open the gate in 18% of molecules, with a slightly higher contribution of intermediates (25%). (b) Representative AFM images of the three conformational forms of the core proteasome. The top-view images with the α-face exposed were zoomed-in from 1 μm × 1 μm fields. The images are raw and have been subjected only to planefitting/flattening, linear adjustments of brightness and contrast, and linear interpolation for viewing clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Treatment with 1 μM <b>2</b> increased the contribution of both open and intermediate conformers, which together amounted to half of the analyzed standing proteasomes (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). In contrast, treatment with 1 μM <b>7</b> resulted in a slightly decreased partition of open proteasomes and a high contribution of closed particles (73% ± 2%). Interestingly, treatment with a high concentration of <b>7</b> (10 μM) resulted in apparent destabilization of the α-face, with the content of the closed conformers decreasing to only 57% ± 6% (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The destabilization of the α-face by high concentrations of <b>7</b> may seem an unusual effect for an inhibitor; however it follows our previous observations with PR39 and <b>11</b>, when closed conformers were poorly detected in the peptide-treated sets of particles.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> We speculate that at higher concentrations PR inhibitors may occupy additional binding sites that include pockets between the α subunits. The pocket binding is likely to be critical for activation by the HbYX-containing <b>1</b> and <b>2</b>; however engagement of other sites may contribute to the effects displayed by the PR inhibitors.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Binding Site of PR Inhibitor in the Proteasome Molecule</h3><div class="NLM_p">Molecular modeling and yeast two-hybrid studies have indicated that PR peptides bind to the outer rim of the yeast proteasome α-ring.<a onclick="showRef(event, 'ref34 ref48'); return false;" href="javascript:void(0);" class="ref ref34 ref48">(34,48)</a> To elucidate the structural basis of the interaction, we performed trials to crystallize our modulators in complexes with both human and yeast 20S proteasomes. Crystals were obtained for both orthologous enzymes; however a complex was only successfully formed in the case of <b>6</b> and the yeast 20S. Since <b>6</b> is also an efficient inhibitor of the yeast enzyme (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S8</a>), the structural information provided by this complex may be extended to the human counterpart. The structure of the complex was determined at 2.7 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z">4X6Z</a>) and confirmed that PR peptides bind within the proteasome α-ring. The structure solved based on a single data collection showed <b>6</b> bound in the pocket between the α5 and α6 subunits, and the unidentified electron density was localized symmetrically between the α5′ and α6′ subunits. Merging the three data sets collected for the same complex reinforced the electron density which allows us to identify the peptide electron density also in the α5′/α6′ pocket. The three C-terminal amino acid residues of <b>6</b> that were visible in the crystal structure created the main-chain to main-chain hydrogen bonds with α6Ser33 and α6Gly76, and the salt bridge with the α6Lys62 side chain (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). The rest of the peptide sequence was not defined by the electron density probably because of its flexibility and the lack of stable interactions with the proteasome. Degradation of <b>6</b> by 20S, as other possible explanation, was excluded by the peptide stability tests (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S9</a>).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Interaction of <b>6</b> with yeast 20S proteasome (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z">4X6Z</a>). (a) General (left) and detailed (blow-up) localization of the inhibitor binding site between α5 (wheat color) and α6 (gray) subunits. It is visible in the crystal structure that the C-terminal fragment of <b>6</b> (green) binds to the ε-amine group of the conserved α6Lys62 through its carboxylate group. (b) Electron density defining the fragment of <b>6</b> included in the model (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map contoured at 1σ level). The cartoon of general proteasome structure in (a), left, has also been used in our earlier work.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Binding in the α5/α6 pocket and similar network of interactions utilized to anchor the modulator within the pocket were observed by us in the crystal structure of the Blm10-based proteasome activator, Blm-pep.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> This compound possesses the C-terminal HbYX motif (YYA) and was able to stimulate h20S 8-fold when probed with the LFP substrate. The same site of binding of two modulators, displaying the opposite influence on the proteasome activity, may indicate that the pockets between the α subunits may be universal binding places for allosteric modulators. The allosteric signal that transduces either inhibition or stimulation effects to the active sites may thus result from transient interactions, which due to their nature could be observed in the crystal structure of neither compound <b>6</b> nor Blm-pep. The importance of transient interactions in proteasome allosteric activation is supported by the cryo-EM structures of yeast and human 26S proteasomes, in which the opened entrance to the catalytic chamber was observed in only one of the detected few conformational states.<a onclick="showRef(event, 'ref7 ref47 ref49'); return false;" href="javascript:void(0);" class="ref ref7 ref47 ref49">(7,47,49)</a> It remains to be established which allosteric routes contribute to the effects of ligands of the α-face pockets. The nonexclusive options include direct allosteric signaling from the pockets to the active sites, the pocket-gate signaling resulting in conformational shifts favoring distinct states of the gate and influencing substrate uptake, as well as the complex pocket-gate-antechamber/catalytic chamber/active sites network. PR peptides constitute excellent tools for the exploration of proteasome allostery.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73558" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73558" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We demonstrated that manipulations of the structural modules of Pro- and Arg-rich peptides could provide compounds that exhibit diverse effects on the activity of the core proteasome particle, both in vitro and in cellulo. Mixing-and-matching of Pro-rich and HbYX or RHbX motifs produces strong activators or strong inhibitors. Studies of interactions of PR39 and <b>11</b> with yeast proteasome suggested binding to the outer rim of the α ring.<a onclick="showRef(event, 'ref34 ref48'); return false;" href="javascript:void(0);" class="ref ref34 ref48">(34,48)</a> While we cannot exclude and should still consider such binding site(s) for our PR derivatives, their binding to the intersubunit pockets, used by the HbYX-containing activators, is an intriguing opportunity. The plausible common binding sites for allosteric inhibitors and activators imply that the type of the signal that is transmitted to the binding pockets and allosterically transduced to the gate and/or the active sites may depend on small but precisely adjusted changes in the modulator design. Further exploration of the diverse effects of PR peptides on performance of the core proteasome will undoubtedly provide more information on the mechanism of proteasome degradation, the significance of the α-face pockets in modulating proteasome activity, and the usefulness of small peptides as universal proteasome regulators.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Information</h3><div class="NLM_p last">20S proteasome, isolated from human erythrocytes, was purchased from Enzo Life Sciences Inc. (Farmingdale, NY). All reagents used in tests with proteasome were of molecular biology grade. The pH of all buffers was determined at 20 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Peptide Synthesis</h3><div class="NLM_p last">Syntheses of all peptides were carried out on a solid support (TentaGel R PHB or TentaGel R RAM resin), using a Liberty Blue microwave peptide synthesizer (CEM). Coupling of orthogonally protected Fmoc-amino acid residues was carried out using, as a coupling agent, 1 M solution of <i>N</i>,<i>N</i>′-diisopropylcarbodiimide in dimethylformamide with 0.5 M ethyl cyano(hydroxyimino)acetate as an antiracemization additive. Crude peptides were purified by reverse phase high performance liquid chromatography (RP-HPLC), using a C12 semipreparative Jupiter Proteo column (21.2 mm × 250 mm, 4 μm, Phenomenex) and H<sub>2</sub>O/acetonitrile gradients. The purity of the peptides was assessed by HPLC analysis performed using a Luna C18 column (4.6 mm × 250 mm, 5 μm, 100 Å; Phenomenex) and a LC-20A HPLC system (Shimadzu). 60 min gradients of 100% A → 100% B (A, H<sub>2</sub>O + 0.1% TFA; B, 80% acetonitrile/H<sub>2</sub>O + 0.1% TFA) and a detection wavelength of λ = 223 nm were applied. The purity of the peptides has been determined based on the integration of the area under the peaks, using the Lab Solution software provided by the HPLC manufacturer (Shimadzu), The purity of all PRs was at least 95%. The identity of the pure products was confirmed based on <i>m</i>/<i>z</i> signals detected by a LCMS-ESI-IT-TOF Prominence mass spectrometer (Shimadzu). The MS spectra and the chromatograms, alongside with their quantitative analysis, are available in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Enzymatic Activity Tests</h3><div class="NLM_p">Boc-LRR-AMC and Z-LLE-AMC were used as a probes in the trypsin-like and caspase-like activity assays, respectively. Two peptide substrates were employed in the assessment of the proteasome chymotrypsin-like activity: the classic fluorogenic succinyl-Leu-Leu-Val-Tyr-4-methylcoumarin-7-amide (Suc-LLVY-AMC, Enzo Life Sciences Inc.) and a homemade FRET-type nonapeptide (LFP; mca-AKVYPYPME-dap(dnp)-amide) as an alternative which is more reliable for assessment of the activating propensities of the studied compounds.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Latent h20S proteasome was activated with 0.005% SDS. Stock solutions of the peptides (10 mM) were prepared in dimethyl sulfoxide (DMSO). The activity assays were performed in a 96-well plate in 50 mM TrisHCl, (pH 8.0) at a 100 μL final volume. The CP was used at a final concentration of 0.001 mg/mL (1.4 nM). Suc-LLVY-AMC, Boc-LRR-AMC, and Z-LLE-AMC were added at 100 μM and LFP at 20 μM final concentration. The release of aminomethylcoumarin (AMC) was followed by monitoring the fluorescence emission in the range 380–460 nm. LFP hydrolysis was detected by measuring the emission in the range 322–398 nm (Infinite M200 Pro, Tecan). Fluorescence was measured continuously every 2 min for up to 60 min, at 37 °C. All activity assays were performed in at least three independent replicates. The relative activity was calculated in relation to the catalytic activity of the vehicle (DMSO) treated proteasome.</div><div class="NLM_p last">Activity tests with Rpt5 peptide and Rpt5 combined with PRs were performed in 45 mM Tris/HCl (pH 8.0) buffer containing 100 mM KCl and 1 mM EDTA, to ensure latency of the h20S proteasome. The Fluoroskan Ascent plate reader (Thermo Fisher Scientific Inc., Waltham, MA) was used in these assays, with fluorescence measured every 1 min for 60 min at 37 °C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Determination of the Type of Inhibition</h3><div class="NLM_p last">To determine the type of inhibition induced by PRs, the ChT-L activity of h20S proteasome was tested in the presence of three different concentrations of <b>7</b> and <b>4</b> (0.05, 0.1, and 0.2 μM). The proteasome concentration was 1.4 nM. ChT-L activity was probed with Suc-LLVY-AMC in a concentration range 20–200 μM. The kinetic parameters were calculated with the kinetic module of OriginPro 2017 software (OriginLab, Northampton, MA).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Protein Substrate Degradation Assay</h3><div class="NLM_p last">Human α-synuclein (rPeptide) or yeast enolase (Sigma-Aldrich) was incubated with human 20S proteasome activated with 0.01% SDS. The h20S/protein substrate ratio was 1:100 pmol and 1:10 pmol for α-synuclein and enolase, respectively. Degradation experiments were carried out at 37 °C in 20 mM HEPES, pH 7.4, at a 10 μL total sample volume for 1 h (α-synuclein) or 4 h (enolase). Either DMSO (control) or compounds dissolved in DMSO were added to evaluate the influence of PR peptides on the degradation process. The DMSO concentration never exceeded 0.05%. The reaction was stopped with 4× Laemmli buffer, and then samples were boiled for 5 min at 95 °C and loaded (8 μL) onto a 12% (α-synuclein) or 10% (enolase) SDS–PAGE gel. The protein bands were detected with Coomassie Blue-based reagent (InstantBlue, Sigma-Aldrich). Quantitative image analysis was carried out with Quantity One 1-D analysis software (Bio-Rad). The amount of the nondegraded protein was calculated after reduction of the background intensity and expressed as a percentage of the control. Each value represents an average of at least three experiments. All results are presented as a mean ± SEM. Statistical analysis was performed using SigmaPlot 12.3 and one-way ANOVA followed by a Bonferroni post hoc test for pairwise comparison. A <i>P</i> value of <0.05 was considered statistically significant.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Fibroblast Culture</h3><div class="NLM_p last">Primary human fibroblasts (GM04390, Coriell cell repositories) were cultured in complete media made of Dulbecco’s modified Eagle medium (high-glucose variant, Gibco-Invitrogen, Carlsbad, CA) supplemented with 10% heat-inactivated fetal bovine serum and antibiotics (100 U/mL penicillin, 100 μg/mL streptomycin, and 0.25 μg/mL amphotericin B; Gibco-Invitrogen). Cells were incubated at 3% O<sub>2</sub>, hypoxic with respect to atmospheric O<sub>2</sub> concentration, to mimic their normal physiological environment; the incubators were also maintained at 5% CO<sub>2</sub> and 37 °C. The medium was replaced every 3–4 days. For experimental end points, cells were seeded at 100 000 cells/mL in either 6- or 96-well plates. Cells were washed with PBS and medium replaced with Optimem (Gibco-Invitrogen) 24 h after seeding. Cells were treated with 0.1–20 μM <b>2</b> (or an equal volume of DMSO diluent) 48 h after seeding. Experimental end points were typically performed 24 h after initiation of treatment with <b>2</b>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Proteolytic Activity Assay in Cell Lysates</h3><div class="NLM_p last">Cells were harvested through scraping in PBS and then lysed in proteolysis buffer (50 mM Tris/HCl, pH 7.4, 5 mM MgCl<sub>2</sub>, 1 mM DTT) through mechanical lysis followed by freeze fractionation. Protein content was measured using a Bradford assay, after which 1 μg of cell lysate was diluted to 100 μL and incubated with 50 μM Suc-LLVY-AMC (Sigma-Aldrich). Fluorescence emission was measured at 460 nm with excitation at 370 nm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> WST-1 Assay</h3><div class="NLM_p last">Cell viability was evaluated using a WST-1 assay following the Sigma recommended protocol. Cells were incubated with 10% WST-1 reagent (Sigma-Aldrich) for 4 h after which absorbance was measured at 450 nm.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Atomic Force Microscopy (AFM) Imaging</h3><div class="NLM_p last">We used our established procedures to image proteasome particles in AFM oscillating (tapping) mode in liquid using a MultiMode Nanoscope IIIa microscope (Bruker Corp., Billerica, MA).<a onclick="showRef(event, 'ref46 ref50'); return false;" href="javascript:void(0);" class="ref ref46 ref50">(46,50)</a> In short, about 50 ng of h20S proteasomes in 3 μL of 5 mM Tris/HCl buffer (pH 7.0) was deposited on freshly cleaved muscovite mica, which provides a flat, clean, and mildly negatively charged surface that electrostatically binds a majority of proteins. After 2 min of incubation at room temperature the enzyme molecules were overlaid with 30 μL of 5 mM Tris/HCl buffer, pH 8.0. Scanning was performed with SNL (Sharp Nitride Lever) probes with a nominal spring constant of 0.35 N/m, mounted in a liquid-mode chamber (Bruker Corp.) and tuned to 9–10 kHz. Fields of 1 μm<sup>2</sup> containing multiple proteasome particles, the majority of them standing on the α ring and not touching their neighbors, were scanned at a rate of 3.05 Hz, with a drive voltage of 200–600 mV and a setpoint ranging from 1.5 to 2.0 V. The relatively high setpoint ensured very gentle imaging conditions. The trace and retrace images were collected with a digital resolution of 512 × 512 pixels. Numerical values of the height of particles were collected in SPIP software. The analyzed data remained “raw”, since the images were subjected only to the standard order 1 flattening and planefitting in the Nanoscope software. To determine the conformational status of the α face, height values of the six-pixel scan across the center of the α face were harvested. When a plot of these values (a section through the center of the α face) revealed a local minimum, the particle was classified as “open”. When a plot presented a concave function devoid of a local minimum, the particle was considered to be an “intermediate” conformer. The remaining “closed” particles displayed convex functions in their section plots.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Crystallization</h3><div class="NLM_p last">The proteasome crystals were grown at 20 °C using the hanging drop vapor diffusion method. Drops contained a 1:1 mixture of the protein (2.5 mg/mL) and the reservoir solution (30 mM magnesium acetate, 100 mM 2-(<i>N</i>-morpholino)ethanesulfonic acid (MES), pH 7.2, and 12% (v/v) 2-methyl-2,4-pentanediol (MPD)). The proteasome–peptide complex was obtained by soaking the crystals with a solution of <b>6</b> at a final concentration of 1 μM for 24 h. Crystals were cryoprotected in the mother liquor and flash-frozen in liquid nitrogen.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Structure Determination and Refinement</h3><div class="NLM_p last">Diffraction data were collected at the beam 19 ID at Advanced Photon Source, Argonne National Laboratory, USA (λ = 0.979 Å). The images were indexed, integrated, and scaled using the HKL3000 program package.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The crystal structure was determined by molecular replacement using MOLREP<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and the coordinates of the yeast 20S proteasome (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RYP">1RYP</a>) as a search model. The model was refined using Refmac<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> from the CCP4 package,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> with stereochemical restraints and, in the later stages, TLS parameters for rigid-body segments established by the TLSMD server.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The model was constructed in Coot.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Progress of the refinement was monitored, and the model was validated using <i>R</i><sub>free</sub>.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The quality of the final structure was assessed using a MolProbity server.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Data collection and refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Table 1S</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01025" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01025" class="ext-link">10.1021/acs.jmedchem.8b01025</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">HPLC and HR MS analysis of the designed analogs, supplementary figures displaying the ChT-L, T-L, and C-L activity of human proteasome in the presence of the modulators, comparison of <b>7</b> and <b>7-4P</b> influence on the latent and SDS-activated h20S, CD spectra of the modulators, inhibitory capacity of <b>6</b> against yeast 20S proteasome, HPLC chromatogram confirming stability of <b>6</b> against proteasomal proteolytic activity, and crystallographic data statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf">jm8b01025_si_001.pdf (1.43 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for the yeast 20S proteasome in the complex with <b>6</b> modulator is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z">4X6Z</a>. The atomic coordinates and experimental data have been already released.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01025" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Gaczynska</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Institute of Biotechnology, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b5d2d4d6cfccdbc6ded4f5c0c1ddc6d6c6d49bd0d1c0"><span class="__cf_email__" data-cfemail="ec8b8d8f9695829f878dac9998849f8f9f8dc2898899">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elżbieta Jankowska</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1099-8885" title="Orcid link">http://orcid.org/0000-0002-1099-8885</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#60050c1a02090514014e0a010e0b0f17130b012015074e0504154e100c"><span class="__cf_email__" data-cfemail="0d6861776f6468796c23676c6366627a7e666c4d786a23686978237d61">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Małgorzata Giżyńska</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Witkowska</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Przemysław Karpowicz</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rafał Rostankowski</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdansk, Wita Stwosza 63, 80-308 Gdansk, Poland</span>; 
    <span>Present Address:
                        R.R.: Polpharma SA Pharmaceutical Works, Pelplińska 19, Starogard Gdański 83-200, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Estrella S. Chocron</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, The Barshop Institute for Longevity and Aging
Studies, University of Texas Health Science
Center, 15355 Lambda Drive, San Antonio, Texas 78245, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew M. Pickering</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, The Barshop Institute for Longevity and Aging
Studies, University of Texas Health Science
Center, 15355 Lambda Drive, San Antonio, Texas 78245, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pawel Osmulski</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Institute of Biotechnology, University of Texas Health Science Center, 15355 Lambda Drive, San Antonio, Texas 78245, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Author contributions were the following: M.Gi., synthesis and kinetic studies of modulators, proteolytic stability tests; J.W., synthesis and kinetic studies of modulators, X-ray structure refinement; P.K., protein substrates degradation tests; A.M.P. and E.S.C., cellular tests; P.O. and M.Ga., AFM and kinetic studies with Rpt5 peptide, design of the experiments, and data analysis; R.R., crystallization of 20S-6 complex; E.J., design of the experiments and data analysis. All authors contributed to the manuscript preparation.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e1913-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86444" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86444" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by National Science Center (Grants UMO-2014/15/B/NZ7/01014 (E.J.), UMO-2015/17/N/NZ1/01772 (J.W.), and UMO-2016/23/N/ST5/02812 (M.Gi.)), the William and Ella Owens Medical Research Foundation award (M.Ga. and P.O.), the San Antonio Nathan Shock Center of Excellence in the Biology of Aging Pilot Grants Program NIH/P30 (A.M.P.), and the Voelcker Fund New Investigator Award (A.M.P.). We thank Dr. Dominika Borek from the Department of Biochemistry, University of Texas Southwestern Medical Center (U.S.), for her assistance in the X-ray data refinement.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration causing 50% of maximal activation</p></td></tr><tr><td class="NLM_term">AFM</td><td class="NLM_def"><p class="first last">atomic force microscopy</p></td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">aminomethylcoumarin</p></td></tr><tr><td class="NLM_term">ChT-L</td><td class="NLM_def"><p class="first last">chymotrypsin-like activity</p></td></tr><tr><td class="NLM_term">CP</td><td class="NLM_def"><p class="first last">core particle</p></td></tr><tr><td class="NLM_term">h20S</td><td class="NLM_def"><p class="first last">human 20S proteasome</p></td></tr><tr><td class="NLM_term">Hb</td><td class="NLM_def"><p class="first last">hydrophobic</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration causing 50% inhibition of activity</p></td></tr><tr><td class="NLM_term">LFP</td><td class="NLM_def"><p class="first last">mca-AKVYPYPME-dap(dnp)-amide</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">PR peptide</td><td class="NLM_def"><p class="first last">proline- and arginine-rich peptide</p></td></tr><tr><td class="NLM_term">RP</td><td class="NLM_def"><p class="first last">regulatory particle</p></td></tr><tr><td class="NLM_term">Rpt5</td><td class="NLM_def"><p class="first last">regulatory particle ATPase 5 and its C-terminal fragment</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">UPS</td><td class="NLM_def"><p class="first last">ubiquitin–proteasome system</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30150" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30150" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, A. L.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin-Mediated Proteolytic Pathway: Mode of Action and Clinical Implications</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">Suppl. 34</span>),  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-4644(2000)77:34+<40::AID-JCB9>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2000&pages=40-51&issue=Suppl.+34&author=A.+Ciechanoverauthor=A.+Orianauthor=A.+L.+Schwartz&title=The+Ubiquitin-Mediated+Proteolytic+Pathway%3A+Mode+of+Action+and+Clinical+Implications&doi=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-4644%25282000%252977%253A34%252B%253C40%253A%253AAID-JCB9%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26aulast%3DOrian%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DA.%2BL.%26atitle%3DThe%2520Ubiquitin-Mediated%2520Proteolytic%2520Pathway%253A%2520Mode%2520of%2520Action%2520and%2520Clinical%2520Implications%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2000%26volume%3D77%26issue%3DSuppl.%252034%26spage%3D40%26epage%3D51%26doi%3D10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">The Proteasome in Health and Disease</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1028</span>, <span class="refDoi"> DOI: 10.2174/138161281906141012124358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.2174%2F138161281906141012124358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23016682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFeqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1010-1028&author=E.+Jankowskaauthor=J.+Stojauthor=P.+Karpowiczauthor=P.+A.+Osmulskiauthor=M.+Gaczynska&title=The+Proteasome+in+Health+and+Disease&doi=10.2174%2F138161281906141012124358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The proteasome in health and disease</span></div><div class="casAuthors">Jankowska, Elzbieta; Stoj, Julia; Karpowicz, Przemyslaw; Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1028</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The giant proteolytic factory called the proteasome came a long way from a biochem. curio to a major regulator of cellular physiol. and a renowned drug target within the ubiquitin proteasome pathway (UPP).  Thanks to availability of highly specific inhibitors of the proteasome, in less than twenty years it was possible to identify major transcription factors, cyclins, and products of oncogenes as crucial substrates for the UPP.  Nine years passed since the FDA speedily approved bortezomib, the inhibitor of proteasome, for treatment of multiple myeloma.  One year after its approval, the field was honored by awarding the Nobel Prize to Hershko, Ciechanover and Rose for introducing the concept of controlled proteolysis of ubiquitin-tagged substrates, with proteasome as the intracellular recycling facility.  Taking into consideration the universal involvement of the proteasome in the life of all cells in human body, it comes to no surprise that the enzyme is deeply implicated in etiol., progression, diagnosis or cure of multiple diseases.  Below we discuss some aspects of the involvement: from direct causative links to changes in proteasome properties that correlate with pathol. conditions.  We start with diseases collectively known as cancer, and with immune system-related pathologies.  Here, the proteasome inhibitors are either already used in clinics, or undergo advanced preclin. screening.  Then, we will continue with cardiovascular disorders, followed by aging.  Changes of the proteasome make-up during aging may be a priming factor for neurodegenerative diseases, described last.  We discuss the potential for proteasome regulation: inhibition, activation or specificity modulation, to successfully enter the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAvSvTc5q4wrVg90H21EOLACvtfcHk0liZo2QNQMCoiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFeqtrg%253D&md5=8b4a664827d4d79f4c9bf88212ec4160</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F138161281906141012124358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161281906141012124358%26sid%3Dliteratum%253Aachs%26aulast%3DJankowska%26aufirst%3DE.%26aulast%3DStoj%26aufirst%3DJ.%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DThe%2520Proteasome%2520in%2520Health%2520and%2520Disease%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D1010%26epage%3D1028%26doi%3D10.2174%2F138161281906141012124358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, A. E.</span></span> <span> </span><span class="NLM_article-title">Proteasome Inhibitors in Cancer Therapy</span>. <i>J. Cell Commun. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1007/s12079-011-0121-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2Fs12079-011-0121-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21484190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvosl2jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=101-110&issue=2&author=L.+J.+Crawfordauthor=B.+Walkerauthor=A.+E.+Irvine&title=Proteasome+Inhibitors+in+Cancer+Therapy&doi=10.1007%2Fs12079-011-0121-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors in cancer therapy</span></div><div class="casAuthors">Crawford Lisa J; Walker Brian; Irvine Alexandra E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cell communication and signaling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ubiquitin proteasome pathway plays a critical role in regulating many processes in the cell which are important for tumour cell growth and survival.  Inhibition of proteasome function has emerged as a powerful strategy for anti-cancer therapy.  Clinical validation of the proteasome as a therapeutic target was achieved with bortezomib and has prompted the development of a second generation of proteasome inhibitors with improved pharmacological properties.  This review summarises the main mechanisms of action of proteasome inhibitors in cancer, the development of proteasome inhibitors as therapeutic agents and the properties and progress of next generation proteasome inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFoYHRPE0q0okFb3mqPUYDfW6udTcc2ebZTa-HidqyTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvosl2jtQ%253D%253D&md5=5a3d94d5da47330ca700e3d61c1c9e14</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs12079-011-0121-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12079-011-0121-7%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DL.%2BJ.%26aulast%3DWalker%26aufirst%3DB.%26aulast%3DIrvine%26aufirst%3DA.%2BE.%26atitle%3DProteasome%2520Inhibitors%2520in%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Cell%2520Commun.%2520Signal.%26date%3D2011%26volume%3D5%26issue%3D2%26spage%3D101%26epage%3D110%26doi%3D10.1007%2Fs12079-011-0121-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, A. E.</span></span> <span> </span><span class="NLM_article-title">Targeting the Ubiquitin Proteasome System in Haematological Malignancies</span>. <i>Blood Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.blre.2013.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.blre.2013.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24183816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslahsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=297-304&issue=6&author=L.+J.+Crawfordauthor=A.+E.+Irvine&title=Targeting+the+Ubiquitin+Proteasome+System+in+Haematological+Malignancies&doi=10.1016%2Fj.blre.2013.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the ubiquitin proteasome system in haematological malignancies</span></div><div class="casAuthors">Crawford, Lisa J.; Irvine, Alexandra E.</div><div class="citationInfo"><span class="NLM_cas:title">Blood Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">297-304</span>CODEN:
                <span class="NLM_cas:coden">BLOREB</span>;
        ISSN:<span class="NLM_cas:issn">0268-960X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin proteasome system (UPS) plays a central role in cellular protein homeostasis through the targeted destruction of damaged/misfolded proteins and regulatory proteins that control crit. cellular functions.  The UPS comprises a sequential series of enzymic activities to covalently attach ubiquitin to proteins to target them for degrdn. through the proteasome.  Aberrancies within this system have been assocd. with transformation and tumorigenesis and thus, the UPS represents an attractive target for the development of anti-cancer therapies.  The use of the first-in-class proteasome inhibitor, bortezomib, in the treatment of Plasma Cell Myeloma and Mantle Cell Lymphoma has validated the UPS as a therapeutic target.  Following on its success, efforts are focused on the development of second-generation proteasome inhibitors and small mol. inhibitors of other components of the UPS.  This review will provide an overview of the UPS and discuss current and novel therapies targeting the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSIPslSGGo1bVg90H21EOLACvtfcHk0lgzkYr6Z7AxnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslahsLjM&md5=dc702c6f3e0e87c27955cfa11502bca5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.blre.2013.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.blre.2013.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DL.%2BJ.%26aulast%3DIrvine%26aufirst%3DA.%2BE.%26atitle%3DTargeting%2520the%2520Ubiquitin%2520Proteasome%2520System%2520in%2520Haematological%2520Malignancies%26jtitle%3DBlood%2520Rev.%26date%3D2013%26volume%3D27%26issue%3D6%26spage%3D297%26epage%3D304%26doi%3D10.1016%2Fj.blre.2013.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Muscle Wasting in Disease: Molecular Mechanisms and Promising Therapies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1038/nrd4467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fnrd4467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25549588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=58-74&issue=1&author=S.+Cohenauthor=J.+A.+Nathanauthor=A.+L.+Goldberg&title=Muscle+Wasting+in+Disease%3A+Molecular+Mechanisms+and+Promising+Therapies&doi=10.1038%2Fnrd4467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscle wasting in disease: molecular mechanisms and promising therapies</span></div><div class="casAuthors">Cohen, Shenhav; Nathan, James A.; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-74</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Atrophy occurs in specific muscles with inactivity (for example, during plaster cast immobilization) or denervation (for example, in patients with spinal cord injuries).  Muscle wasting occurs systemically in older people (a condition known as sarcopenia); as a physiol. response to fasting or malnutrition; and in many diseases, including chronic obstructive pulmonary disorder, cancer-assocd. cachexia, diabetes, renal failure, cardiac failure, Cushing syndrome, sepsis, burns and trauma.  The rapid loss of muscle mass and strength primarily results from excessive protein breakdown, which is often accompanied by reduced protein synthesis.  This loss of muscle function can lead to reduced quality of life, increased morbidity and mortality.  Exercise is the only accepted approach to prevent or slow atrophy.  However, several promising therapeutic agents are in development, and major advances in our understanding of the cellular mechanisms that regulate the protein balance in muscle include the identification of several cytokines, particularly myostatin, and a common transcriptional program that promotes muscle wasting.  Here, we discuss these new insights and the rationally designed therapies that are emerging to combat muscle wasting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwPC0gdbm5bVg90H21EOLACvtfcHk0lgzkYr6Z7AxnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSg&md5=3c66223fa688330f2d50812548df5396</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4467%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DNathan%26aufirst%3DJ.%2BA.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DMuscle%2520Wasting%2520in%2520Disease%253A%2520Molecular%2520Mechanisms%2520and%2520Promising%2520Therapies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26issue%3D1%26spage%3D58%26epage%3D74%26doi%3D10.1038%2Fnrd4467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">Proteasomal Dysfunction in Aging and Huntington Disease</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2010.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.nbd.2010.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21145396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFCktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=4-8&issue=1&author=X.-J.+Liauthor=S.+Li&title=Proteasomal+Dysfunction+in+Aging+and+Huntington+Disease&doi=10.1016%2Fj.nbd.2010.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasomal dysfunction in aging and Huntington disease</span></div><div class="casAuthors">Li, Xiao-Jiang; Li, Shihua</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Disease</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-8</span>CODEN:
                <span class="NLM_cas:coden">NUDIEM</span>;
        ISSN:<span class="NLM_cas:issn">0969-9961</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein degrdn. plays a central role in many cellular functions.  Misfolded and damaged proteins are removed from the cells to avoid toxicity.  Eukaryotic cells have two main routes for clearing misfolded or toxic proteins: the ubiquitin-proteasome and autophagy-lysosome pathways.  The ubiquitin-proteasome system (UPS) is ubiquitously present in the cytoplasm, nucleus, and various subcellular regions whereas autophagy predominantly functions in the cytoplasm.  The activity of the UPS often remains at a high level, whereas basal autophagy constitutively occurs at low levels in cells for the performance of homeostatic functions.  Because of the presence of the UPS in the nucleus, the UPS function may be more important for clearing misfolded proteins in the nucleus.  Polyglutamine diseases, including Huntington disease (HD), show the age-dependent neurol. symptoms and the accumulation of misfolded proteins that are often found in the nucleus.  The selective neuropathol. in HD is also found to assoc. with the preferential accumulation of the disease protein huntington in neuronal cells.  Although it is clear that the UPS is important for clearing mutant huntingtin, it remains unclear whether aging or HD affects the capacity of neuronal UPS to remove toxic and misfolded proteins.  In this review, we focus on the relationship between the UPS function and aging as well as Huntington disease.  We also discuss findings that suggest that aging is a more important factor that can neg. impact the function of the UPS.  This article is part of a Special Issue entitled "Autophagy and protein degrdn. in neurol. diseases.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnnV8623Fm-7Vg90H21EOLACvtfcHk0lgzkYr6Z7AxnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFCktb0%253D&md5=15ff156b2e745d0bc40080f588ef079d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2010.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2010.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-J.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DProteasomal%2520Dysfunction%2520in%2520Aging%2520and%2520Huntington%2520Disease%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2011%26volume%3D43%26issue%3D1%26spage%3D4%26epage%3D8%26doi%3D10.1016%2Fj.nbd.2010.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Dynamic Regulation of the Human 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">12991</span>– <span class="NLM_lpage">12996</span>, <span class="refDoi"> DOI: 10.1073/pnas.1614614113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.1614614113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27791164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslalsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=12991-12996&issue=46&author=S.+Chenauthor=J.+Wuauthor=Y.+Luauthor=Y.-B.+Maauthor=B.-H.+Leeauthor=Z.+Yuauthor=Q.+Ouyangauthor=D.+J.+Finleyauthor=M.+W.+Kirschnerauthor=Y.+Mao&title=Structural+Basis+for+Dynamic+Regulation+of+the+Human+26S+Proteasome&doi=10.1073%2Fpnas.1614614113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for dynamic regulation of the human 26S proteasome</span></div><div class="casAuthors">Chen, Shuobing; Wu, Jiayi; Lu, Ying; Ma, Yong-Bei; Lee, Byung-Hoon; Yu, Zhou; Ouyang, Qi; Finley, Daniel J.; Kirschner, Marc W.; Mao, Youdong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">12991-12996</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The proteasome is the major engine of protein degrdn. in all eukaryotic cells.  At the heart of this machine is a heterohexameric ring of AAA (ATPases assocd. with diverse cellular activities) proteins that unfolds ubiquitylated target proteins that are concurrently translocated into a proteolytic chamber and degraded into peptides.  Using cryoelectron microscopy, we detd. a near-at.-resoln. structure of the 2.5-MDa human proteasome in its ground state, as well as subnanometer-resoln. structures of the holoenzyme in three alternative conformational states.  The substrate-unfolding AAA-ATPase channel is narrowed by 10 inward-facing pore loops arranged into two helixes that run in parallel with each other, one hydrophobic in character and the other highly charged.  The gate of the core particle was unexpectedly found closed in the ground state and open in only one of the alternative states.  Coordinated, stepwise conformational changes of the regulatory particle couple ATP hydrolysis to substrate translocation and regulate gating of the core particle, leading to processive degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojT-6dAofYVbVg90H21EOLACvtfcHk0ljHOR4Fs4xIJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslalsLfF&md5=1824587478f94283add68652b6d5a263</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614614113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614614113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.-B.%26aulast%3DLee%26aufirst%3DB.-H.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DOuyang%26aufirst%3DQ.%26aulast%3DFinley%26aufirst%3DD.%2BJ.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DMao%26aufirst%3DY.%26atitle%3DStructural%2520Basis%2520for%2520Dynamic%2520Regulation%2520of%2520the%2520Human%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D46%26spage%3D12991%26epage%3D12996%26doi%3D10.1073%2Fpnas.1614614113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jap, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwickl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the 20S Proteasome from the Archaeon T. Acidophilum at 3.4 Å Resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>268</i></span> (<span class="NLM_issue">5210</span>),  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">539</span>, <span class="refDoi"> DOI: 10.1126/science.7725097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1126%2Fscience.7725097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=7725097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADyaK2M3kt1Sqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1995&pages=533-539&issue=5210&author=J.+L%C3%B6weauthor=D.+Stockauthor=B.+Japauthor=P.+Zwicklauthor=W.+Baumeisterauthor=R.+Huber&title=Crystal+Structure+of+the+20S+Proteasome+from+the+Archaeon+T.+Acidophilum+at+3.4+%C3%85+Resolution&doi=10.1126%2Fscience.7725097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the 20S proteasome from the archaeon T. acidophilum at 3.4 A resolution</span></div><div class="casAuthors">Lowe J; Stock D; Jap B; Zwickl P; Baumeister W; Huber R</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">5210</span>),
    <span class="NLM_cas:pages">533-9</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The three-dimensional structure of the proteasome from the archaebacterium Thermoplasma acidophilum has been elucidated by x-ray crystallographic analysis by means of isomorphous replacement and cyclic averaging.  The atomic model was built and refined to a crystallographic R factor of 22.1 percent.  The 673-kilodalton protease complex consists of 14 copies of two different subunits, alpha and beta, forming a barrel-shaped structure of four stacked rings.  The two inner rings consist of seven beta subunits each, and the two outer rings consist of seven alpha subunits each.  A narrow channel controls access to the three inner compartments.  The alpha 7 beta 7 beta 7 alpha 7 subunit assembly has 72-point group symmetry.  The structures of the alpha and beta subunits are similar, consisting of a core of two antiparallel beta sheets that is flanked by alpha helices on both sides.  The binding of a peptide aldehyde inhibitor marks the active site in the central cavity at the amino termini of the beta subunits and suggests a novel proteolytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSE36ig5vun65w7prLlxLOzfW6udTcc2ebmxyqRHXzMA7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M3kt1Sqtw%253D%253D&md5=c09bba67da07e6addad1fb8d49a04b20</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.7725097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7725097%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DStock%26aufirst%3DD.%26aulast%3DJap%26aufirst%3DB.%26aulast%3DZwickl%26aufirst%3DP.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520the%252020S%2520Proteasome%2520from%2520the%2520Archaeon%2520T.%2520Acidophilum%2520at%25203.4%2520%25C3%2585%2520Resolution%26jtitle%3DScience%26date%3D1995%26volume%3D268%26issue%3D5210%26spage%3D533%26epage%3D539%26doi%3D10.1126%2Fscience.7725097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ditzel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bochtler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartunik, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span> <span> </span><span class="NLM_article-title">Structure of 20S Proteasome from Yeast at 2.4Å Resolution</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">6624</span>),  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1038/386463a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F386463a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9087403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=1997&pages=463-471&issue=6624&author=M.+Grollauthor=L.+Ditzelauthor=J.+L%C3%B6weauthor=D.+Stockauthor=M.+Bochtlerauthor=H.+D.+Bartunikauthor=R.+Huber&title=Structure+of+20S+Proteasome+from+Yeast+at+2.4%C3%85+Resolution&doi=10.1038%2F386463a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of 20S proteasome from yeast at 2.4 Å resolution</span></div><div class="casAuthors">Groll, Michael; Ditzel, Lars; Loewe, Jan; Stock, Daniela; Bochtler, Matthias; Bartunik, Hans D.; Huber, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">6624</span>),
    <span class="NLM_cas:pages">463-471</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of the 20S proteasome from the yeast Saccharomyces cerevisiae shows that its 28 protein subunits are arranged as an (α1...α7,β1...β7)2 complex in four stacked rings and occupy unique locations.  The interior of the particle, which harbors the active sites, is only accessible by some very narrow side entrances.  The β-type subunits are synthesized as proproteins before being proteolytically processed for assembly into the particle.  The pro forms of three of the seven different β-type subunits, β1-PRE3, β2/PUP1 and β5/PRE2, are cleaved between the threonine at position 1 and the last glycine of the pro-sequence, with release of the active-site residue Thr 1.  These three β-type subunits have inhibitor-binding sites, indicating that PRE2 has a chymotrypsin-like and a trypsin-like activity and that PRE3 has peptidylglutamyl peptide hydrolytic specificity.  Other β-type subunits are processed to an intermediate form, indicating that an addnl. nonspecific endopeptidase activity may exist which is important for peptide hydrolysis and for the generation of ligands for class I mols. of the major histocompatibility complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUP7-BUWwo-bVg90H21EOLACvtfcHk0ljHOR4Fs4xIJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSqsr0%253D&md5=1796bb267e45cc5cdd2c97ecb426f39b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F386463a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F386463a0%26sid%3Dliteratum%253Aachs%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DDitzel%26aufirst%3DL.%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DStock%26aufirst%3DD.%26aulast%3DBochtler%26aufirst%3DM.%26aulast%3DBartunik%26aufirst%3DH.%2BD.%26aulast%3DHuber%26aufirst%3DR.%26atitle%3DStructure%2520of%252020S%2520Proteasome%2520from%2520Yeast%2520at%25202.4%25C3%2585%2520Resolution%26jtitle%3DNature%26date%3D1997%26volume%3D386%26issue%3D6624%26spage%3D463%26epage%3D471%26doi%3D10.1038%2F386463a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomisugi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukihara, T.</span></span> <span> </span><span class="NLM_article-title">The Structure of the Mammalian 20S Proteasome at 2.75 Å Resolution</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(02)00748-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2FS0969-2126%2802%2900748-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=12015144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvVWjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=609-618&issue=5&author=M.+Unnoauthor=T.+Mizushimaauthor=Y.+Morimotoauthor=Y.+Tomisugiauthor=K.+Tanakaauthor=N.+Yasuokaauthor=T.+Tsukihara&title=The+Structure+of+the+Mammalian+20S+Proteasome+at+2.75+%C3%85+Resolution&doi=10.1016%2FS0969-2126%2802%2900748-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of the mammalian 20S proteasome at 2.75 Å resolution</span></div><div class="casAuthors">Unno, Masaki; Mizushima, Tsunehiro; Morimoto, Yukio; Tomisugi, Yoshikazu; Tanaka, Keiji; Yasuoka, Noritake; Tsukihara, Tomitake</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">609-618</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The 20S proteasome is the catalytic portion of the 26S proteasome.  Constitutively expressed mammalian 20S proteasomes have 3 active subunits, β1, β2, and β5, which are replaced in the immunoproteasome by interferon-γ-inducible subunits β1i, β2i, and β5i, resp.  Here, the authors detd. the crystal structure of the bovine 20S proteasome at 2.75 Å resoln.  The structures of the α2, β1, β5, β6, and β7 subunits of the bovine enzyme were different from the yeast enzyme but enabled the bovine proteasome to accommodate either the constitutive or the inducible subunits.  A novel N-terminal nucleophile hydrolase activity was proposed for the β7 subunit.  The authors also detd. the site of the nuclear localization signals in the mol.  A model of the immunoproteasome was predicted from this constitutive structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVIKTjVKtIXbVg90H21EOLACvtfcHk0lh3QF7jWYNmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvVWjtLw%253D&md5=03c3997a62eae5c5fa791e791f3ccd2c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900748-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900748-7%26sid%3Dliteratum%253Aachs%26aulast%3DUnno%26aufirst%3DM.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DMorimoto%26aufirst%3DY.%26aulast%3DTomisugi%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYasuoka%26aufirst%3DN.%26aulast%3DTsukihara%26aufirst%3DT.%26atitle%3DThe%2520Structure%2520of%2520the%2520Mammalian%252020S%2520Proteasome%2520at%25202.75%2520%25C3%2585%2520Resolution%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26issue%3D5%26spage%3D609%26epage%3D618%26doi%3D10.1016%2FS0969-2126%2802%2900748-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambour, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amalric, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsarrat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlet-Schiltz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet-Dubouch, M.-P.</span></span> <span> </span><span class="NLM_article-title">Subcellular Distribution and Dynamics of Active Proteasome Complexes Unraveled by a Workflow Combining in Vivo Complex Cross-Linking and Quantitative Proteomics</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1074/mcp.M112.023317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fmcp.M112.023317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23242550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlOitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=687-699&issue=3&author=B.+Fabreauthor=T.+Lambourauthor=J.+Delobelauthor=F.+Amalricauthor=B.+Monsarratauthor=O.+Burlet-Schiltzauthor=M.-P.+Bousquet-Dubouch&title=Subcellular+Distribution+and+Dynamics+of+Active+Proteasome+Complexes+Unraveled+by+a+Workflow+Combining+in+Vivo+Complex+Cross-Linking+and+Quantitative+Proteomics&doi=10.1074%2Fmcp.M112.023317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Subcellular Distribution and Dynamics of Active Proteasome Complexes Unraveled by a Workflow Combining in Vivo Complex Cross-Linking and Quantitative Proteomics</span></div><div class="casAuthors">Fabre, Bertrand; Lambour, Thomas; Delobel, Julien; Amalric, Francois; Monsarrat, Bernard; Burlet-Schiltz, Odile; Bousquet-Dubouch, Marie-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Through protein degrdn., the proteasome plays fundamental roles in different cell compartments.  Although the compn. of the 20S catalytic core particle (CP) has been well documented, little is known about the compn. and dynamics of the regulatory complexes that play a crucial role in its activity, or about how they assoc. with the CP in different cell compartments, different cell lines, and in response to external stimuli.  Because of difficulties performing acceptable cell fractionation while maintaining complex integrity, it has been challenging to characterize proteasome complexes by proteomic approaches.  Here, we report an integrated protocol, combining a crosslinking procedure on intact cells with cell fractionation, proteasome immuno-purifn., and robust label-free quant. proteomic anal. by mass spectrometry to det. the distribution and dynamics of cellular proteasome complexes in leukemic cells.  Activity profiles of proteasomes were correlated fully with the compn. of protein complexes and stoichiometry.  Moreover, our results suggest that, at the subcellular level, proteasome function is regulated by dynamic interactions between the 20S CP and its regulatory proteins-which modulate proteasome activity, stability, localization, or substrate uptake-rather than by profound changes in 20S CP compn.  Proteasome plasticity was obsd. both in the 20S CP and in its network of interactions following IFNγ stimulation.  The fractionation protocol also revealed specific proteolytic activities and structural features of low-abundance microsomal proteasomes from U937 and KG1a cells.  These could be linked to their important roles in the endoplasmic reticulum assocd. degrdn. pathway in leukemic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopH0HunTPDfLVg90H21EOLACvtfcHk0lh3QF7jWYNmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlOitL4%253D&md5=93d90b61268072123e712a8861bde7c8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M112.023317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M112.023317%26sid%3Dliteratum%253Aachs%26aulast%3DFabre%26aufirst%3DB.%26aulast%3DLambour%26aufirst%3DT.%26aulast%3DDelobel%26aufirst%3DJ.%26aulast%3DAmalric%26aufirst%3DF.%26aulast%3DMonsarrat%26aufirst%3DB.%26aulast%3DBurlet-Schiltz%26aufirst%3DO.%26aulast%3DBousquet-Dubouch%26aufirst%3DM.-P.%26atitle%3DSubcellular%2520Distribution%2520and%2520Dynamics%2520of%2520Active%2520Proteasome%2520Complexes%2520Unraveled%2520by%2520a%2520Workflow%2520Combining%2520in%2520Vivo%2520Complex%2520Cross-Linking%2520and%2520Quantitative%2520Proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2013%26volume%3D12%26issue%3D3%26spage%3D687%26epage%3D699%26doi%3D10.1074%2Fmcp.M112.023317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambour, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrigues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducoux-Petit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amalric, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsarrat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlet-Schiltz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet-Dubouch, M. P.</span></span> <span> </span><span class="NLM_article-title">Label-Free Quantitative Proteomics Reveals the Dynamics of Proteasome Complexes Composition and Stoichiometry in a Wide Range of Human Cell Lines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3027</span>– <span class="NLM_lpage">3037</span>, <span class="refDoi"> DOI: 10.1021/pr500193k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr500193k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFeltbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3027-3037&issue=6&author=B.+Fabreauthor=T.+Lambourauthor=L.+Garriguesauthor=M.+Ducoux-Petitauthor=F.+Amalricauthor=B.+Monsarratauthor=O.+Burlet-Schiltzauthor=M.+P.+Bousquet-Dubouch&title=Label-Free+Quantitative+Proteomics+Reveals+the+Dynamics+of+Proteasome+Complexes+Composition+and+Stoichiometry+in+a+Wide+Range+of+Human+Cell+Lines&doi=10.1021%2Fpr500193k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Label-Free Quantitative Proteomics Reveals the Dynamics of Proteasome Complexes Composition and Stoichiometry in a Wide Range of Human Cell Lines</span></div><div class="casAuthors">Fabre, Bertrand; Lambour, Thomas; Garrigues, Luc; Ducoux-Petit, Manuelle; Amalric, Francois; Monsarrat, Bernard; Burlet-Schiltz, Odile; Bousquet-Dubouch, Marie-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3027-3037</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is the main proteolytic system involved in intracellular proteins homeostasis in eukaryotes.  Although the structure of proteasome complexes has been well characterized, the distribution of its activators and assocd. proteins are less studied.  Here, we det. the compn. and the stoichiometry of proteasome complexes and their assocd. proteins in a wide range of human cell lines using a one-step affinity purifn. method and a label-free quant. proteomic approach.  We show that proteasome complexes are highly dynamic protein assemblies, the activity of which being regulated at different levels by variations in the stoichiometry of bound regulators, in the compn. of catalytic subunits and assocd. proteins, and in the rate of the 20S catalytic core complex assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoabYGVXnBoT7Vg90H21EOLACvtfcHk0lh3QF7jWYNmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFeltbw%253D&md5=39f6ec1acbf38f21e3bb2dab145514ae</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fpr500193k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr500193k%26sid%3Dliteratum%253Aachs%26aulast%3DFabre%26aufirst%3DB.%26aulast%3DLambour%26aufirst%3DT.%26aulast%3DGarrigues%26aufirst%3DL.%26aulast%3DDucoux-Petit%26aufirst%3DM.%26aulast%3DAmalric%26aufirst%3DF.%26aulast%3DMonsarrat%26aufirst%3DB.%26aulast%3DBurlet-Schiltz%26aufirst%3DO.%26aulast%3DBousquet-Dubouch%26aufirst%3DM.%2BP.%26atitle%3DLabel-Free%2520Quantitative%2520Proteomics%2520Reveals%2520the%2520Dynamics%2520of%2520Proteasome%2520Complexes%2520Composition%2520and%2520Stoichiometry%2520in%2520a%2520Wide%2520Range%2520of%2520Human%2520Cell%2520Lines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2014%26volume%3D13%26issue%3D6%26spage%3D3027%26epage%3D3037%26doi%3D10.1021%2Fpr500193k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadre-Bazzaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Formosa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structure of a Blm10 Complex Reveals Common Mechanisms for Proteasome Binding and Gate Opening</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">728</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2010.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20227375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=728-735&issue=5&author=K.+Sadre-Bazzazauthor=F.+G.+Whitbyauthor=H.+Robinsonauthor=T.+Formosaauthor=C.+P.+Hill&title=Structure+of+a+Blm10+Complex+Reveals+Common+Mechanisms+for+Proteasome+Binding+and+Gate+Opening&doi=10.1016%2Fj.molcel.2010.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a Blm10 complex reveals common mechanisms for proteasome binding and gate opening</span></div><div class="casAuthors">Sadre-Bazzaz, Kianoush; Whitby, Frank G.; Robinson, Howard; Formosa, Tim; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">728-735</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The proteasome is an abundant protease that is critically important for numerous cellular pathways.  Proteasomes are activated in vitro by three known classes of proteins/complexes, including Blm10/PA200.  Here, we report a 3.4 Å resoln. crystal structure of a proteasome-Blm10 complex, which reveals that Blm10 surrounds the proteasome entry pore in the 1.2 MDa complex to form a largely closed dome that is expected to restrict access of potential substrates.  This architecture and the observation that Blm10 induces a disordered proteasome gate structure challenge the assumption that Blm10 functions as an activator of proteolysis in vivo.  The Blm10 C terminus binds in the same manner as seen for 11S activators and inferred for 19S/PAN activators and indicates a unified model for gate opening.  We also demonstrate that Blm10 acts to maintain mitochondrial function.  Consistent with the structural data, the C-terminal residues of Blm10 are needed for this activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp17uYMdi2WCLVg90H21EOLACvtfcHk0lgDUt1mlWD0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KrsLc%253D&md5=0215e288cb4a9fef177d3da3e3f2a94d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DSadre-Bazzaz%26aufirst%3DK.%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DFormosa%26aufirst%3DT.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructure%2520of%2520a%2520Blm10%2520Complex%2520Reveals%2520Common%2520Mechanisms%2520for%2520Proteasome%2520Binding%2520and%2520Gate%2520Opening%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D37%26issue%3D5%26spage%3D728%26epage%3D735%26doi%3D10.1016%2Fj.molcel.2010.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowlton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Activation of 20S Proteasomes by 11S Regulators</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>408</i></span> (<span class="NLM_issue">6808</span>),  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/35040607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F35040607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=11081519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVaksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2000&pages=115-120&issue=6808&author=F.+G.+Whitbyauthor=E.+I.+Mastersauthor=L.+Kramerauthor=J.+R.+Knowltonauthor=Y.+Yaoauthor=C.+C.+Wangauthor=C.+P.+Hill&title=Structural+Basis+for+the+Activation+of+20S+Proteasomes+by+11S+Regulators&doi=10.1038%2F35040607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the activation of 20S proteasomes by 11S regulators</span></div><div class="casAuthors">Whitby, Frank G.; Masters, Eugene I.; Kramer, Larissa; Knowlton, J. Randolph; Yao, Yi; Wang, Ching C.; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">6808</span>),
    <span class="NLM_cas:pages">115-120</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most of the non-lysosomal proteolysis that occurs in eukaryotic cells is performed by a nonspecific and abundant barrel-shaped complex called the 20S proteasome.  Substrates access the active sites, which are sequestered in an internal chamber, by traversing a narrow opening (α-annulus) that is blocked in the unliganded 20S proteasome by amino-terminal sequences of α-subunits.  Peptide products probably exit the 20S proteasome through the same opening.  11S regulators (also called PA26 (ref. 4), PA28 (ref. 5) and REG) are heptamers that stimulate 20S proteasome peptidase activity in vitro and may facilitate product release in vivo.  Here we report the co-crystal structure of yeast 20S proteasome with the 11S regulator from Trypanosoma brucei (PA26).  PA26 carboxy-terminal tails provide binding affinity by inserting into pockets on the 20S proteasome, and PA26 activation loops induce conformational changes in α-subunits that open the gate sepg. the proteasome interior from the intracellular environment.  The redn. in processivity expected for an open conformation of the exit gate may explain the role of 11S regulators in the prodn. of ligands for major histocompatibility complex class I mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_QJYjTvf0abVg90H21EOLACvtfcHk0lgDUt1mlWD0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVaksb4%253D&md5=f3ff3f296da463f4bc8f4c61cf5b8435</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F35040607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35040607%26sid%3Dliteratum%253Aachs%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DMasters%26aufirst%3DE.%2BI.%26aulast%3DKramer%26aufirst%3DL.%26aulast%3DKnowlton%26aufirst%3DJ.%2BR.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Activation%2520of%252020S%2520Proteasomes%2520by%252011S%2520Regulators%26jtitle%3DNature%26date%3D2000%26volume%3D408%26issue%3D6808%26spage%3D115%26epage%3D120%26doi%3D10.1038%2F35040607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baugh, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viktorova, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilipenko, E. V.</span></span> <span> </span><span class="NLM_article-title">Proteasomes Can Degrade a Significant Proportion of Cellular Proteins Independent of Ubiquitination</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">827</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.12.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.jmb.2008.12.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19162040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitVCqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=814-827&issue=3&author=J.+M.+Baughauthor=E.+G.+Viktorovaauthor=E.+V.+Pilipenko&title=Proteasomes+Can+Degrade+a+Significant+Proportion+of+Cellular+Proteins+Independent+of+Ubiquitination&doi=10.1016%2Fj.jmb.2008.12.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasomes Can Degrade a Significant Proportion of Cellular Proteins Independent of Ubiquitination</span></div><div class="casAuthors">Baugh, James M.; Viktorova, Ekaterina G.; Pilipenko, Evgeny V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">814-827</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The crit. role of the ubiquitin-26S proteasome system in regulation of protein homeostasis in eukaryotes is well established.  In contrast, the impact of the ubiquitin-independent proteolytic activity of proteasomes is poorly understood.  Through biochem. anal. of mammalian lysates, we find that the 20S proteasome, latent in peptide hydrolysis, specifically cleaves more than 20% of all cellular proteins.  Thirty intrinsic proteasome substrates (IPSs) were identified and in vitro studies of their processing revealed that cleavage occurs at disordered regions, generating stable products encompassing structured domains.  The mechanism of IPS recognition is remarkably well conserved in the eukaryotic kingdom, as mammalian and yeast 20S proteasomes exhibit the same target specificity.  Further, 26S proteasomes specifically recognize and cleave IPSs at similar sites, independent of ubiquitination, suggesting that disordered regions likely constitute the universal structural signal for IPS proteolysis by proteasomes.  Finally, we show that proteasomes contribute to physiol. regulation of IPS levels in living cells and the inactivation of ubiquitin-activating enzyme E1 does not prevent IPS degrdn.  Collectively, these findings suggest a significant contribution of the ubiquitin-independent proteasome degrdn. pathway to the regulation of protein homeostasis in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq20sAHAjVTGLVg90H21EOLACvtfcHk0lgDUt1mlWD0cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitVCqt7s%253D&md5=0a103389423bbcf9cdad614ac43776c9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.12.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.12.081%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DJ.%2BM.%26aulast%3DViktorova%26aufirst%3DE.%2BG.%26aulast%3DPilipenko%26aufirst%3DE.%2BV.%26atitle%3DProteasomes%2520Can%2520Degrade%2520a%2520Significant%2520Proportion%2520of%2520Cellular%2520Proteins%2520Independent%2520of%2520Ubiquitination%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D386%26issue%3D3%26spage%3D814%26epage%3D827%26doi%3D10.1016%2Fj.jmb.2008.12.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Lanzas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaño, J. G.</span></span> <span> </span><span class="NLM_article-title">Proteins Directly Interacting with Mammalian 20S Proteasomal Subunits and Ubiquitin-Independent Proteasomal Degradation</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.3390/biom4041140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.3390%2Fbiom4041140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25534281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvit12gsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1140-1154&issue=4&author=R.+S%C3%A1nchez-Lanzasauthor=J.+G.+Casta%C3%B1o&title=Proteins+Directly+Interacting+with+Mammalian+20S+Proteasomal+Subunits+and+Ubiquitin-Independent+Proteasomal+Degradation&doi=10.3390%2Fbiom4041140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins directly interacting with mammalian 20S proteasomal subunits and ubiquitin-independent proteasomal degradation</span></div><div class="casAuthors">Sanchez-Lanzas Raul; Castano Jose G</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1140-54</span>
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    </div><div class="casAbstract">The mammalian 20S proteasome is a heterodimeric cylindrical complex (α7β7β7α7), composed of four rings each composed of seven different α or β subunits with broad proteolytic activity.  We review the mammalian proteins shown to directly interact with specific 20S proteasomal subunits and those subjected to ubiquitin-independent proteasomal degradation (UIPD).  The published reports of proteins that interact with specific proteasomal subunits, and others found on interactome databases and those that are degraded by a UIPD mechanism, overlap by only a few protein members.  Therefore, systematic studies of the specificity of the interactions, the elucidation of the protein regions implicated in the interactions (that may or may not be followed by degradation) and competition experiments between proteins known to interact with the same proteasomal subunit, are needed.  Those studies should provide a coherent picture of the molecular mechanisms governing the interactions of cellular proteins with proteasomal subunits, and their relevance to cell proteostasis and cell functioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTolbzz8v6UYTAyhZbj3lKdfW6udTcc2eZdCI-XU12qO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvit12gsA%253D%253D&md5=a9bd42d8dcd99a40a922f436f6adf0bc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fbiom4041140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom4041140%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Lanzas%26aufirst%3DR.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DJ.%2BG.%26atitle%3DProteins%2520Directly%2520Interacting%2520with%2520Mammalian%252020S%2520Proteasomal%2520Subunits%2520and%2520Ubiquitin-Independent%2520Proteasomal%2520Degradation%26jtitle%3DBiomolecules%26date%3D2014%26volume%3D4%26issue%3D4%26spage%3D1140%26epage%3D1154%26doi%3D10.3390%2Fbiom4041140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raynes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomatto, L. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. J. A.</span></span> <span> </span><span class="NLM_article-title">Degradation of Oxidized Proteins by the Proteasome: Distinguishing between the 20S, 26S, and Immunoproteasome Proteolytic Pathways</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2016.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.mam.2016.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27155164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsVeqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2016&pages=41-55&author=R.+Raynesauthor=L.+C.+D.+Pomattoauthor=K.+J.+A.+Davies&title=Degradation+of+Oxidized+Proteins+by+the+Proteasome%3A+Distinguishing+between+the+20S%2C+26S%2C+and+Immunoproteasome+Proteolytic+Pathways&doi=10.1016%2Fj.mam.2016.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of oxidized proteins by the proteasome: Distinguishing between the 20S, 26S, and immunoproteasome proteolytic pathways</span></div><div class="casAuthors">Raynes, Rachel; Pomatto, Laura C. D.; Davies, Kelvin J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The proteasome is a ubiquitous and highly plastic multi-subunit protease with multi-catalytic activity that is conserved in all eukaryotes.  The most widely known function of the proteasome is protein degrdn. through the 26S ubiquitin-proteasome system, responsible for the vast majority of protein degrdn. during homeostasis.  However, the proteasome also plays an important role in adaptive immune responses and adaptation to oxidative stress.  The unbound 20S proteasome, the core common to all proteasome conformations, is the main protease responsible for degrading oxidized proteins.  During periods of acute stress, the 19S regulatory cap of the 26S proteasome disassocs. from the proteolytic core, allowing for immediate ATP/ubiquitin-independent protein degrdn. by the 20S proteasome.  Despite the abundance of unbound 20S proteasome compared to other proteasomal conformations, many publications fail to distinguish between the two proteolytic systems and often regard the 26S proteasome as the dominant protease.  Further confounding the issue are the differential roles these two proteolytic systems have in adaptation and aging.  In this review, we will summarize the increasing evidence that the 20S core proteasome constitutes the major conformation of the proteasome system and that it is far from a latent protease requiring activation by binding regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZCX5MieTHrVg90H21EOLACvtfcHk0ljzJ32ntB-rGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsVeqtL4%253D&md5=a347662fca5d811cda30294ca18f1f2a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2016.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2016.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DRaynes%26aufirst%3DR.%26aulast%3DPomatto%26aufirst%3DL.%2BC.%2BD.%26aulast%3DDavies%26aufirst%3DK.%2BJ.%2BA.%26atitle%3DDegradation%2520of%2520Oxidized%2520Proteins%2520by%2520the%2520Proteasome%253A%2520Distinguishing%2520between%2520the%252020S%252C%252026S%252C%2520and%2520Immunoproteasome%2520Proteolytic%2520Pathways%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2016%26volume%3D50%26spage%3D41%26epage%3D55%26doi%3D10.1016%2Fj.mam.2016.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, Y. T.</span></span> <span> </span><span class="NLM_article-title">Degradation of Misfolded Proteins in Neurodegenerative Diseases: Therapeutic Targets and Strategies</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e147</span>, <span class="refDoi"> DOI: 10.1038/emm.2014.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Femm.2014.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25766616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=e147&issue=3&author=A.+Ciechanoverauthor=Y.+T.+Kwon&title=Degradation+of+Misfolded+Proteins+in+Neurodegenerative+Diseases%3A+Therapeutic+Targets+and+Strategies&doi=10.1038%2Femm.2014.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of misfolded proteins in neurodegenerative diseases: therapeutic targets and strategies</span></div><div class="casAuthors">Ciechanover, Aaron; Kwon, Yong Tae</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e147</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Mammalian cells remove misfolded proteins using various proteolytic systems, including the ubiquitin (Ub)-proteasome system (UPS), chaperone mediated autophagy (CMA) and macroautophagy.  The majority of misfolded proteins are degraded by the UPS, in which Ub-conjugated substrates are deubiquitinated, unfolded and cleaved into small peptides when passing through the narrow chamber of the proteasome.  The substrates that expose a specific degrdn. signal, the KFERQ sequence motif, can be delivered to and degraded in lysosomes via the CMA.  Aggregation-prone substrates resistant to both the UPS and the CMA can be degraded by macroautophagy, in which cargoes are segregated into autophagosomes before degrdn. by lysosomal hydrolases.  Although most misfolded and aggregated proteins in the human proteome can be degraded by cellular protein quality control, some native and mutant proteins prone to aggregation into β-sheet-enriched oligomers are resistant to all known proteolytic pathways and can thus grow into inclusion bodies or extracellular plaques.  The accumulation of protease-resistant misfolded and aggregated proteins is a common mechanism underlying protein misfolding disorders, including neurodegenerative diseases such as Huntington's disease (HD), Alzheimer's disease (AD), Parkinson's disease (PD), prion diseases and Amyotrophic Lateral Sclerosis (ALS).  In this review, we provide an overview of the proteolytic pathways in neurons, with an emphasis on the UPS, CMA and macroautophagy, and discuss the role of protein quality control in the degrdn. of pathogenic proteins in neurodegenerative diseases.  Addnl., we examine existing putative therapeutic strategies to efficiently remove cytotoxic proteins from degenerating neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAcnolVc3nZ7Vg90H21EOLACvtfcHk0ljzJ32ntB-rGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFOhtL8%253D&md5=f4d7542fdd0e42b0d3e566585454261a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Femm.2014.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2014.117%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26aulast%3DKwon%26aufirst%3DY.%2BT.%26atitle%3DDegradation%2520of%2520Misfolded%2520Proteins%2520in%2520Neurodegenerative%2520Diseases%253A%2520Therapeutic%2520Targets%2520and%2520Strategies%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2015%26volume%3D47%26issue%3D3%26spage%3De147%26doi%3D10.1038%2Femm.2014.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilchez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillin, A.</span></span> <span> </span><span class="NLM_article-title">The Role of Protein Clearance Mechanisms in Organismal Ageing and Age-Related Diseases</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5659</span>, <span class="refDoi"> DOI: 10.1038/ncomms6659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fncomms6659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25482515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVentb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5659&author=D.+Vilchezauthor=I.+Saezauthor=A.+Dillin&title=The+Role+of+Protein+Clearance+Mechanisms+in+Organismal+Ageing+and+Age-Related+Diseases&doi=10.1038%2Fncomms6659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The role of protein clearance mechanisms in organismal ageing and age-related diseases</span></div><div class="casAuthors">Vilchez, David; Saez, Isabel; Dillin, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5659</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are subject to continuous and complex quality-control mechanisms, which ensure integrity of the proteome.  Vilchez et al. review how a demise in these processes, collectively referred to as proteostasis, is linked to organismal ageing and the development of age-assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0qgIqQnnmlbVg90H21EOLACvtfcHk0lhxn3gOplPsow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVentb8%253D&md5=7371b6f29c2605c7e0bbdaffaf0aa270</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fncomms6659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6659%26sid%3Dliteratum%253Aachs%26aulast%3DVilchez%26aufirst%3DD.%26aulast%3DSaez%26aufirst%3DI.%26aulast%3DDillin%26aufirst%3DA.%26atitle%3DThe%2520Role%2520of%2520Protein%2520Clearance%2520Mechanisms%2520in%2520Organismal%2520Ageing%2520and%2520Age-Related%2520Diseases%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5659%26doi%3D10.1038%2Fncomms6659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruschak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slassi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span> <span> </span><span class="NLM_article-title">Novel Proteasome Inhibitors to Overcome Bortezomib Resistance</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1093%2Fjnci%2Fdjr160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21606441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVKqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=1007-1017&issue=13&author=A.+M.+Ruschakauthor=M.+Slassiauthor=L.+E.+Kayauthor=A.+D.+Schimmer&title=Novel+Proteasome+Inhibitors+to+Overcome+Bortezomib+Resistance&doi=10.1093%2Fjnci%2Fdjr160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Proteasome Inhibitors to Overcome Bortezomib Resistance</span></div><div class="casAuthors">Ruschak, Amy M.; Slassi, Malik; Kay, Lewis E.; Schimmer, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The proteasome is an intracellular enzyme complex that degrades ubiquitin-tagged proteins and thereby regulates protein levels within the cell.  Given this important role in maintaining cellular homeostasis, it is perhaps somewhat surprising that proteasome inhibitors have a therapeutic window.  Proteasome inhibitors have demonstrated clin. efficacy in the treatment of multiple myeloma and mantle cell lymphoma and are under evaluation for the treatment of other malignancies.  Bortezomib is the first and only Food and Drug Administration-approved proteasome inhibitor that inhibits this enzyme complex in a reversible fashion.  Although bortezomib improves clin. outcomes when used as a single agent, most patients do not respond to this drug and those who do respond almost uniformly relapse.  As such, efforts are underway to develop proteasome inhibitors that act through mechanisms distinct from that of bortezomib.  Specifically, inhibitors that bind the active site of the proteasome and inhibit the complex irreversibly have been developed and are in advanced clin. trials.  Inhibitors that act on sites of the proteasome outside of the catalytic center have also been identified and are in preclin. development.  In this review, we discuss the structure and function of the proteasome.  We then focus on the mol. biol., chem., and the preclin. and clin. efficacy of novel proteasome inhibitors as strategies to inhibit this target and overcome some forms of bortezomib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWeUjDj5ezAbVg90H21EOLACvtfcHk0lhxn3gOplPsow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVKqsrs%253D&md5=52b263bbfd39bf18a543a382610c762f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr160%26sid%3Dliteratum%253Aachs%26aulast%3DRuschak%26aufirst%3DA.%2BM.%26aulast%3DSlassi%26aufirst%3DM.%26aulast%3DKay%26aufirst%3DL.%2BE.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26atitle%3DNovel%2520Proteasome%2520Inhibitors%2520to%2520Overcome%2520Bortezomib%2520Resistance%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26issue%3D13%26spage%3D1007%26epage%3D1017%26doi%3D10.1093%2Fjnci%2Fdjr160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansdell, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dissanayake, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Substituted Quinolines as Noncovalent Proteasome Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2441</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bmc.2016.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27112450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslSjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=2441-2450&issue=11&author=T.+J.+McDanielauthor=T.+A.+Lansdellauthor=A.+A.+Dissanayakeauthor=L.+M.+Azevedoauthor=J.+Claesauthor=A.+L.+Odomauthor=J.+J.+Tepe&title=Substituted+Quinolines+as+Noncovalent+Proteasome+Inhibitors&doi=10.1016%2Fj.bmc.2016.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Substituted quinolines as noncovalent proteasome inhibitors</span></div><div class="casAuthors">McDaniel, Tanner J.; Lansdell, Theresa A.; Dissanayake, Amila A.; Azevedo, Lauren M.; Claes, Jacob; Odom, Aaron L.; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2441-2450</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Screening of a library of diverse heterocyclic scaffolds identified substituted quinolines as inhibitors of the human proteasome.  The heterocyclic library was prepd. via a novel titanium-catalyzed multicomponent coupling reaction, which rendered a diverse set of isoxazoles, pyrimidines, pyrroles, pyrazoles and quinolines.  SAR of the parent lead compd. indicated that hydrophobic residues on the benzo-moiety significantly improved potency.  Lead compd. 3-cyclohexenyl-2-methyl-6-piperidinylquinoline (25) inhibits the chymotryptic-like proteolytic activity of the proteasome (IC50 5.4 μM), representing a new class of nonpeptidic, noncovalent proteasome inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc8639UcfzmLVg90H21EOLACvtfcHk0lhxn3gOplPsow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslSjtrg%253D&md5=4479d7a26d98b091aaf13428b4590843</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DT.%2BJ.%26aulast%3DLansdell%26aufirst%3DT.%2BA.%26aulast%3DDissanayake%26aufirst%3DA.%2BA.%26aulast%3DAzevedo%26aufirst%3DL.%2BM.%26aulast%3DClaes%26aufirst%3DJ.%26aulast%3DOdom%26aufirst%3DA.%2BL.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSubstituted%2520Quinolines%2520as%2520Noncovalent%2520Proteasome%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26issue%3D11%26spage%3D2441%26epage%3D2450%26doi%3D10.1016%2Fj.bmc.2016.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansdell, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurchla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hovde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weilbaecher, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Noncompetitive Modulation of the Proteasome by Imidazoline Scaffolds Overcomes Bortezomib Resistance and Delays MM Tumor Growth in Vivo</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1021/cb300568r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300568r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKhsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=578-587&issue=3&author=T.+A.+Lansdellauthor=M.+A.+Hurchlaauthor=J.+Xiangauthor=S.+Hovdeauthor=K.+N.+Weilbaecherauthor=R.+W.+Henryauthor=J.+J.+Tepe&title=Noncompetitive+Modulation+of+the+Proteasome+by+Imidazoline+Scaffolds+Overcomes+Bortezomib+Resistance+and+Delays+MM+Tumor+Growth+in+Vivo&doi=10.1021%2Fcb300568r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Noncompetitive Modulation of the Proteasome by Imidazoline Scaffolds Overcomes Bortezomib Resistance and Delays MM Tumor Growth in Vivo</span></div><div class="casAuthors">Lansdell, Theresa A.; Hurchla, Michelle A.; Xiang, Jingyu; Hovde, Stacy; Weilbaecher, Katherine N.; Henry, R. William; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-587</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) is a malignant disorder of differentiated B-cells for which std. care involves the inhibition of the proteasome.  All clin. used proteasome inhibitors, including the chemotherapeutic drug bortezomib, target the catalytic active sites of the proteasome and inhibit protein proteolysis by competing with substrate binding.  However, nearly all (∼97%) patients become intolerant or resistant to treatments within a few years, after which the av. survival time is less than 1 yr.  We describe herein the inhibition of the human proteasome via a noncompetitive mechanism by the imidazoline scaffold, TCH-13.  Consistent with a mechanism distinct from that of competitive inhibitors, TCH-013 acts additively with and overcomes resistance to bortezomib.  Importantly, TCH-013 induces apoptosis in a panel of myeloma and leukemia cell lines, but in contrast, normal lymphocytes, primary bone marrow stromal cells (hBMSC), and macrophages are resistant to its cytotoxic effects.  TCH-013 was equally effective in blocking MM cell growth in co-cultures of MM cells with hBMSC isolated from CD138 neg. bone marrow (BM) samples of MM patients.  The cellular activity translated well in vivo where TCH-013 delayed tumor growth in an MM xenograft model to a similar extent as bortezomib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ovzBZEXwn7Vg90H21EOLACvtfcHk0lh8bjlqsXGyIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKhsbbK&md5=70093e771713e6ee6a15f2291997b5a7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb300568r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300568r%26sid%3Dliteratum%253Aachs%26aulast%3DLansdell%26aufirst%3DT.%2BA.%26aulast%3DHurchla%26aufirst%3DM.%2BA.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DHovde%26aufirst%3DS.%26aulast%3DWeilbaecher%26aufirst%3DK.%2BN.%26aulast%3DHenry%26aufirst%3DR.%2BW.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DNoncompetitive%2520Modulation%2520of%2520the%2520Proteasome%2520by%2520Imidazoline%2520Scaffolds%2520Overcomes%2520Bortezomib%2520Resistance%2520and%2520Delays%2520MM%2520Tumor%2520Growth%2520in%2520Vivo%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D3%26spage%3D578%26epage%3D587%26doi%3D10.1021%2Fcb300568r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njomen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjögren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Enhancement of 20S Proteasome Activity Targets Intrinsically Disordered Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2240</span>– <span class="NLM_lpage">2247</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2240-2247&issue=9&author=C.+L.+Jonesauthor=E.+Njomenauthor=B.+Sj%C3%B6grenauthor=T.+S.+Dexheimerauthor=J.+J.+Tepe&title=Small+Molecule+Enhancement+of+20S+Proteasome+Activity+Targets+Intrinsically+Disordered+Proteins&doi=10.1021%2Facschembio.7b00489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Enhancement of 20S Proteasome Activity Targets Intrinsically Disordered Proteins</span></div><div class="casAuthors">Jones, Corey L.; Njomen, Evert; Sjogren, Benita; Dexheimer, Thomas S.; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2240-2247</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome is the main protease for the degrdn. of oxidatively damaged and intrinsically disordered proteins.  When accumulation of disordered or oxidatively damaged proteins exceeds proper clearance in neurons, imbalanced pathway signaling or aggregation occurs, which have been implicated in the pathogenesis of several neurol. disorders.  Screening of the NIH Clin. Collection and Prestwick libraries identified the neuroleptic agent chlorpromazine as a lead agent capable of enhancing 20S proteasome activity.  Chem. manipulation of chlorpromazine abrogated its D2R receptor binding affinity while retaining its ability to enhance 20S mediated proteolysis at low micromolar concns.  The resulting small mol. enhancers of 20S proteasome activity induced the degrdn. of intrinsically disordered proteins, α-synuclein, and tau but not structured proteins.  These small mol. 20S agonists can serve as leads to explore the therapeutic potential of 20S activation or as new tools to provide insight into the yet unclear mechanics of 20S-gate regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtP6cycR_6KbVg90H21EOLACvtfcHk0lh8bjlqsXGyIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jI&md5=52f7dcf5de303a4e8b992af6ed590171</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00489%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DC.%2BL.%26aulast%3DNjomen%26aufirst%3DE.%26aulast%3DSj%25C3%25B6gren%26aufirst%3DB.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSmall%2520Molecule%2520Enhancement%2520of%252020S%2520Proteasome%2520Activity%2520Targets%2520Intrinsically%2520Disordered%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3D2240%26epage%3D2247%26doi%3D10.1021%2Facschembio.7b00489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Rapamycin Allosterically Inhibits the Proteasome</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1124/mol.112.083873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1124%2Fmol.112.083873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23619386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=104-113&issue=1&author=P.+A.+Osmulskiauthor=M.+Gaczynska&title=Rapamycin+Allosterically+Inhibits+the+Proteasome&doi=10.1124%2Fmol.112.083873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin allosterically inhibits the proteasome</span></div><div class="casAuthors">Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">104-113</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Rapamycin is a canonical allosteric inhibitor of the mammalian tarpet of rapamycin (mTOR) kinase with immunosuppressive and proapoptotic activities.  We found that in vitro rapamycin also regulates the proteasome, which is an essential intracellular protease of the ubiquitin-proteasome pathway.  Rapamycin inhibits proteinase and selected peptidase activities of the catalytic core proteasome at low micromolar concns.  Moreover, the drug interferes with binding of the 19S cap essential for processing of polyubiquitinylated substrates and with the PA200 proteasome activator to the 20S catalytic core proteasome.  These protein complexes are known to bind to specific grooves on the α face region of the 20S core.  Treatment with rapamycin affects the conformational dynamics of the proteasomal gate, which is centrally positioned within the α face and allosterically regulated element responsible for the intake of substrates.  We showed that rapamycin shares all the proteasome targeting properties not only with other two-domain, closed-ring analogs (rapalogs) but also with its single domain mimics and seco-rapamycin, which is the first in vivo open-ring metabolite of rapamycin that does not affect mTOR.  We hypothesize that rapamycin and related compds. bind to the α face and allosterically impact proteasome function.  This article discusses the implications of our findings for the mechanism of in vivo actions of rapamycin and for the design of novel allosteric drugs targeting the proteasome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBS8ZCRQbJ7Vg90H21EOLACvtfcHk0lh8bjlqsXGyIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjF&md5=503e8e747496f74ba6942608932449d5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.083873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.083873%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DRapamycin%2520Allosterically%2520Inhibits%2520the%2520Proteasome%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26issue%3D1%26spage%3D104%26epage%3D113%26doi%3D10.1124%2Fmol.112.083873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Njomen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Modulation of Proteasome Assembly</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">28</span>),  <span class="NLM_fpage">4214</span>– <span class="NLM_lpage">4224</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00579</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00579" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWju7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4214-4224&issue=28&author=E.+Njomenauthor=P.+A.+Osmulskiauthor=C.+L.+Jonesauthor=M.+Gaczynskaauthor=J.+J.+Tepe&title=Small+Molecule+Modulation+of+Proteasome+Assembly&doi=10.1021%2Facs.biochem.8b00579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modulation of Proteasome Assembly</span></div><div class="casAuthors">Njomen, Evert; Osmulski, Pawel A.; Jones, Corey L.; Gaczynska, Maria; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4214-4224</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome is the main protease that directly targets intrinsically disordered proteins (IDPs) for proteolytic degrdn.  Mutations, oxidative stress or aging can induce the build-up of IDPs resulting in incorrect signaling or aggregation, assocd. with the pathogenesis of many cancers and neurodegenerative diseases.  Drugs that facilitate 20S-mediated proteolysis have therefore many potential therapeutic applications.  We report herein the modulation of proteasome assembly by the small mole-cule TCH-165, resulting in an increase in 20S levels.  The increase of free 20S corresponds to enhanced proteolysis of IDPs, including α-synuclein, tau, ornithine decarboxylase and c-Fos, but not structured proteins.  Clearance of ubiquitinated protein was largely maintained by single capped proteasome complexes (19S-20S), but accumulation occurs when all 19S capped proteasome complexes are depleted.  This study illustrates the first example of a small mol. capable of targeting disordered proteins for degrdn. by regulating the dynamic equil. between different proteasome complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWB-LPHAN7wLVg90H21EOLACvtfcHk0lh8bjlqsXGyIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWju7fJ&md5=8d8920859b36c1098de6452427b1baca</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00579%26sid%3Dliteratum%253Aachs%26aulast%3DNjomen%26aufirst%3DE.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DJones%26aufirst%3DC.%2BL.%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSmall%2520Molecule%2520Modulation%2520of%2520Proteasome%2520Assembly%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D28%26spage%3D4214%26epage%3D4224%26doi%3D10.1021%2Facs.biochem.8b00579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span> <span> </span><span class="NLM_article-title">Activation and Inhibition of Proteasomes by Betulinic Acid and Its Derivatives</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>581</i></span> (<span class="NLM_issue">25</span>),  <span class="NLM_fpage">4955</span>– <span class="NLM_lpage">4959</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2007.09.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.febslet.2007.09.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=17904555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGku7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2007&pages=4955-4959&issue=25&author=L.+Huangauthor=P.+Hoauthor=C.-H.+Chen&title=Activation+and+Inhibition+of+Proteasomes+by+Betulinic+Acid+and+Its+Derivatives&doi=10.1016%2Fj.febslet.2007.09.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Activation and inhibition of the proteasome by betulinic acid and its derivatives</span></div><div class="casAuthors">Huang, Li; Ho, Phong; Chen, Chin-Ho</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">581</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4955-4959</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This study discovered that betulinic acid (BA) is a potent proteasome activator that preferentially activates the chymotrypsin-like activity of the proteasome.  Chem. modifications can transform BA into proteasome inhibitors.  Chem. modifications at the C-3 position of BA resulted in compds., such as dimethylsuccinyl BA (DSB), with various inhibitory activities against the human 20S proteasome.  Interestingly, the proteasomal activation by BA and the inhibitory activity of DSB could be abrogated by introducing a side chain at the C-28 position.  In summary, this study discovered a class of small mols. that can either activate or inhibit human proteasome activity depending on side chain modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXCEaz3UPUE7Vg90H21EOLACvtfcHk0ljq4p4V7Mqb2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGku7nN&md5=031df2a57e72fefccc01a293e04e8bca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2007.09.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2007.09.031%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DC.-H.%26atitle%3DActivation%2520and%2520Inhibition%2520of%2520Proteasomes%2520by%2520Betulinic%2520Acid%2520and%2520Its%2520Derivatives%26jtitle%3DFEBS%2520Lett.%26date%3D2007%26volume%3D581%26issue%3D25%26spage%3D4955%26epage%3D4959%26doi%3D10.1016%2Fj.febslet.2007.09.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-E.</span></span> <span> </span><span class="NLM_article-title">Synthetic Peptide-Based Activators of the Proteasome</span>. <i>Mol. Biol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1023/A:1006851428691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1023%2FA%3A1006851428691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9228292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXks1amsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1997&pages=119-124&issue=1&author=S.+Wilkauthor=W.-E.+Chen&title=Synthetic+Peptide-Based+Activators+of+the+Proteasome&doi=10.1023%2FA%3A1006851428691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic peptide-based activators of the proteasome</span></div><div class="casAuthors">Wilk, Sherwin; Chen, Wei-Er</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology Reports</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">119-124</span>CODEN:
                <span class="NLM_cas:coden">MLBRBU</span>;
        ISSN:<span class="NLM_cas:issn">0301-4851</span>.
    
            (<span class="NLM_cas:orgname">Kluwer</span>)
        </div><div class="casAbstract">The development of small mol. peptide-based activators of the 20S proteasome or multicatalytic proteinase complex is initiated.  The enhancement of antigen presentation by transfection of the protein activator PA28α into a mouse fibroblast cell line supports the potential use of small mol. activators in stimulating the immune response.  Four classes of peptide-based activators were synthesized, i.e. peptidyl alcs., esters, p-nitroanilides and nitriles.  These compds. markedly and reversibly stimulated the hydrolysis of suc-LLVY-MCA, Z-LLE-NA and Z-GPALG-p-aminobenzoate as well as hydrolysis of the decapeptide angiotensin I.  Stimulation was due to a decrease in the Km and increase in the Vmax of the substrate.  In general, the EC50 for activation ranged from 50-150 mM and maximal stimulation varied from 3 to 15 fold depending on the activity measured.  Z-IE(O-tBu)AL-p-nitroanilide, a proteasome substrate, markedly stimulated the hydrolysis of Z-GPALG-pAB by binding to a saturable high affinity site distinct from its binding site as substrate.  Since all effective activators contain hydrophobic groups in positions P1-P5, low aq. soly. is a limitation of these compds.  Competition expts. suggest that these activators bind to the same site as PA28.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrijOvCsrxrqbVg90H21EOLACvtfcHk0ljq4p4V7Mqb2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXks1amsrY%253D&md5=4340498024abf6e450abfda061f02fc6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1006851428691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1006851428691%26sid%3Dliteratum%253Aachs%26aulast%3DWilk%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.-E.%26atitle%3DSynthetic%2520Peptide-Based%2520Activators%2520of%2520the%2520Proteasome%26jtitle%3DMol.%2520Biol.%2520Rep.%26date%3D1997%26volume%3D24%26issue%3D1%26spage%3D119%26epage%3D124%26doi%3D10.1023%2FA%3A1006851428691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics: Current Status and Future Directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&issue=1&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+Therapeutics%3A+Current+Status+and+Future+Directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0ljq4p4V7Mqb2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520Therapeutics%253A%2520Current%2520Status%2520and%2520Future%2520Directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D1%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Docking of the Proteasomal ATPases’ Carboxyl Termini in the 20S Proteasome’s Ring Opens the Gate for Substrate Entry</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2007.06.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=17803938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=731-744&issue=5&author=D.+M.+Smithauthor=S.+C.+Changauthor=S.+Parkauthor=D.+Finleyauthor=Y.+Chengauthor=A.+L.+Goldberg&title=Docking+of+the+Proteasomal+ATPases%E2%80%99+Carboxyl+Termini+in+the+20S+Proteasome%E2%80%99s+Ring+Opens+the+Gate+for+Substrate+Entry&doi=10.1016%2Fj.molcel.2007.06.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Docking of the proteasomal ATPases' carboxyl termini in the 20S proteasome's α ring opens the gate for substrate entry</span></div><div class="casAuthors">Smith, David M.; Chang, Shih-Chung; Park, Soyeon; Finley, Daniel; Cheng, Yifan; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The 20S proteasome functions in protein degrdn. in eukaryotes together with the 19S ATPases or in archaea with the homologous PAN ATPase complex.  These ATPases contain a conserved C-terminal hydrophobic-tyrosine-X motif (HbYX).  We show that these residues are essential for PAN to assoc. with the 20S and open its gated channel for substrate entry.  Upon ATP binding, these C-terminal residues bind to pockets between the 20S's α subunits.  Seven-residue or longer peptides from PAN's C terminus contg. the HbYX motif also bind to these sites and induce gate opening in the 20S.  Gate opening could be induced by C-terminal peptides from the 19S ATPase subunits, Rpt2, and Rpt5, but not by ones from PA28/26, which lack the HbYX motif and cause gate opening by distinct mechanisms.  C-terminal residues in the 19S ATPases were also shown to be crit. for gating and stability of 26S proteasomes.  Thus, the C termini of the proteasomal ATPases function like a "key in a lock" to induce gate opening and allow substrate entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq37nJVOXWjPrVg90H21EOLACvtfcHk0lgzILQZ_nlVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7J&md5=5837c2dafce8e1ce8e80c67487f51f17</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.033%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DFinley%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DDocking%2520of%2520the%2520Proteasomal%2520ATPases%25E2%2580%2599%2520Carboxyl%2520Termini%2520in%2520the%252020S%2520Proteasome%25E2%2580%2599s%2520Ring%2520Opens%2520the%2520Gate%2520for%2520Substrate%2520Entry%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26issue%3D5%26spage%3D731%26epage%3D744%26doi%3D10.1016%2Fj.molcel.2007.06.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gizyńska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grudnik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giełdoń, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Low Molecular Weight Activator Blm-Pep with Yeast 20S Proteasome - Insights into the Enzyme Activation Mechanism</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">6177</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-05997-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fs41598-017-05997-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28733623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC1cfgtFentA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6177&issue=1&author=J.+Witkowskaauthor=M.+Gizy%C5%84skaauthor=P.+Grudnikauthor=P.+Golikauthor=P.+Karpowiczauthor=A.+Gie%C5%82do%C5%84author=G.+Dubinauthor=E.+Jankowska&title=Crystal+Structure+of+a+Low+Molecular+Weight+Activator+Blm-Pep+with+Yeast+20S+Proteasome+-+Insights+into+the+Enzyme+Activation+Mechanism&doi=10.1038%2Fs41598-017-05997-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a low molecular weight activator Blm-pep with yeast 20S proteasome - insights into the enzyme activation mechanism</span></div><div class="casAuthors">Witkowska Julia; Gizynska Malgorzata; Karpowicz Przemyslaw; Gieldon Artur; Jankowska Elzbieta; Grudnik Przemyslaw; Golik Przemyslaw; Dubin Grzegorz; Grudnik Przemyslaw; Golik Przemyslaw; Dubin Grzegorz</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6177</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteasomes are responsible for protein turnover in eukaryotic cells, degrading short-lived species but also removing improperly folded or oxidatively damaged ones.  Dysfunction of a proteasome results in gradual accumulation of misfolded/damaged proteins, leading to their aggregation.  It has been postulated that proteasome activators may facilitate removal of such aggregation-prone proteins and thus prevent development of neurodegenerative disorders.  However, the discovery of pharmacologically relevant compounds is hindered by insufficient structural understanding of the activation process.  In this study we provide a model peptidic activator of human proteasome and analyze the structure-activity relationship within this novel scaffold.  The binding mode of the activator at the relevant pocket within the proteasome has been determined by X-ray crystallography.  This crystal structure provides an important basis for rational design of pharmacological compounds.  Moreover, by providing a novel insight into the proteasome gating mechanism, our results allow the commonly accepted model of proteasome regulation to be revisited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpbYP86F-n6LYtC3U5jeOifW6udTcc2eYphOmisIrBlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfgtFentA%253D%253D&md5=9daf28ff62212eb8765546eb8762c3bc</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-05997-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-05997-4%26sid%3Dliteratum%253Aachs%26aulast%3DWitkowska%26aufirst%3DJ.%26aulast%3DGizy%25C5%2584ska%26aufirst%3DM.%26aulast%3DGrudnik%26aufirst%3DP.%26aulast%3DGolik%26aufirst%3DP.%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DGie%25C5%2582do%25C5%2584%26aufirst%3DA.%26aulast%3DDubin%26aufirst%3DG.%26aulast%3DJankowska%26aufirst%3DE.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Low%2520Molecular%2520Weight%2520Activator%2520Blm-Pep%2520with%2520Yeast%252020S%2520Proteasome%2520-%2520Insights%2520into%2520the%2520Enzyme%2520Activation%2520Mechanism%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26issue%3D1%26spage%3D6177%26doi%3D10.1038%2Fs41598-017-05997-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dal
Vechio, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerqueira, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augusto, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demasi, M.</span></span> <span> </span><span class="NLM_article-title">Peptides That Activate the 20S Proteasome by Gate Opening Increased Oxidized Protein Removal and Reduced Protein Aggregation</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2013.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.freeradbiomed.2013.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24291399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlehu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=304-313&author=F.+H.+Dal%0AVechioauthor=F.+Cerqueiraauthor=O.+Augustoauthor=R.+Lopesauthor=M.+Demasi&title=Peptides+That+Activate+the+20S+Proteasome+by+Gate+Opening+Increased+Oxidized+Protein+Removal+and+Reduced+Protein+Aggregation&doi=10.1016%2Fj.freeradbiomed.2013.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides that activate the 20S proteasome by gate opening increased oxidized protein removal and reduced protein aggregation</span></div><div class="casAuthors">Dal Vechio, Francisco H.; Cerqueira, Fernanda; Augusto, Ohara; Lopes, Robson; Demasi, Marilene</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-313</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The proteasome is a multicatalytic protease that is responsible for the degrdn. of the majority of intracellular proteins.  Its role is correlated with several major regulatory pathways that are involved in cell cycle control, signaling, and antigen presentation, as well as in the removal of oxidatively damaged proteins.  Although several proteasomal catalytic inhibitors have been described, very few activators have been reported to date.  Some reports in the literature highlight the cellular protective effects of proteasome activation against oxidative stress and its effect on increased life span.  In this work, we describe a peptide named proteasome-activating peptide 1 (PAP1), which increases the chymotrypsin-like proteasomal catalytic activity and, consequently, proteolytic rates both in vitro and in culture.  PAP1 proteasomal activation is mediated by the opening of the proteasomal catalytic chamber.  We also demonstrate that the obsd. proteasomal activation protected cells from oxidative stress; further, PAP1 prevented protein aggregation in a cellular model of amyotrophic lateral sclerosis.  The role of 20SPT gate opening underlying protection against oxidative stress was also explored in yeast cells.  The present data indicate the importance of proteasomal activators as potential drugs for the treatment of pathologies assocd. with the impaired removal of damaged proteins, which is obsd. in many neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OVs0noe7YbVg90H21EOLACvtfcHk0lgzILQZ_nlVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlehu7g%253D&md5=d710cb1adc50097ae7a3cdf2f25ebd03</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2013.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2013.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DDal%2BVechio%26aufirst%3DF.%2BH.%26aulast%3DCerqueira%26aufirst%3DF.%26aulast%3DAugusto%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DR.%26aulast%3DDemasi%26aufirst%3DM.%26atitle%3DPeptides%2520That%2520Activate%2520the%252020S%2520Proteasome%2520by%2520Gate%2520Opening%2520Increased%2520Oxidized%2520Protein%2520Removal%2520and%2520Reduced%2520Protein%2520Aggregation%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2014%26volume%3D67%26spage%3D304%26epage%3D313%26doi%3D10.1016%2Fj.freeradbiomed.2013.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikorska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giżyńska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belczyk-Ciesielska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span> <span> </span><span class="NLM_article-title">Interplay between Structure and Charge as a Key to Allosteric Modulation of Human 20S Proteasome by the Basic Fragment of HIV-1 Tat Protein</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e0143038</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0143038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1371%2Fjournal.pone.0143038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=26575189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0143038&issue=11&author=P.+Karpowiczauthor=P.+A.+Osmulskiauthor=J.+Witkowskaauthor=E.+Sikorskaauthor=M.+Gi%C5%BCy%C5%84skaauthor=A.+Belczyk-Ciesielskaauthor=M.+E.+Gaczynskaauthor=E.+Jankowska&title=Interplay+between+Structure+and+Charge+as+a+Key+to+Allosteric+Modulation+of+Human+20S+Proteasome+by+the+Basic+Fragment+of+HIV-1+Tat+Protein&doi=10.1371%2Fjournal.pone.0143038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Interplay between structure and charge as a key to allosteric modulation of human 20S proteasome by the basic fragment of HIV-1 tat protein</span></div><div class="casAuthors">Karpowicz, Przemyslaw; Osmulski, Pawel A.; Witkowska, Julia; Sikorska, Emilia; Gizynska, Malgorzata; Belczyk-Ciesielska, Agnieszka; Gaczynska, Maria E.; Jankowska, Elzbieta</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0143038/1-e0143038/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The proteasome is a giant protease responsible for degrdn. of the majority of cytosolic proteins.  Competitive inhibitors of the proteasome are used against aggressive blood cancers.  However, broadening the use of proteasome-targeting drugs requires new mechanistic approaches to the enzyme's inhibition.  In our previous studies we described Tat1 peptide, an allosteric inhibitor of the proteasome derived from a fragment of the basic domain of HIV-Tat1 protein.  Here, we attempted to dissect the structural determinants of the proteasome inhibition by Tat1.  Single- and multiple- alanine walking scans were performed.  Tat1 analogs with stabilized beta-turn conformation at positions 4-5 and 8-9, pointed out by the mol. dynamics modeling and the alanine scan, were synthesized.  Structure of Tat1 analogs were analyzed by CD, Fourier transform IR and NMR spectroscopy studies, supplemented by mol. dynamics simulations.  Biol. activity tests and structural studies revealed that high flexibility and exposed pos. charge are hallmarks of Tat1 peptide.  Interestingly, stabilization of a beta-turn at the 8-9 position was necessary to significantly improve the inhibitory potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnCn9RQIMoq7Vg90H21EOLACvtfcHk0lgm_UhQx7shtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSjs7vP&md5=afdbf33be55eeba722755025a1d30ed7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0143038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0143038%26sid%3Dliteratum%253Aachs%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DWitkowska%26aufirst%3DJ.%26aulast%3DSikorska%26aufirst%3DE.%26aulast%3DGi%25C5%25BCy%25C5%2584ska%26aufirst%3DM.%26aulast%3DBelczyk-Ciesielska%26aufirst%3DA.%26aulast%3DGaczynska%26aufirst%3DM.%2BE.%26aulast%3DJankowska%26aufirst%3DE.%26atitle%3DInterplay%2520between%2520Structure%2520and%2520Charge%2520as%2520a%2520Key%2520to%2520Allosteric%2520Modulation%2520of%2520Human%252020S%2520Proteasome%2520by%2520the%2520Basic%2520Fragment%2520of%2520HIV-1%2520Tat%2520Protein%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D11%26spage%3De0143038%26doi%3D10.1371%2Fjournal.pone.0143038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metais, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aird, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hampton, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">PR39, a Peptide Regulator of Angiogenesis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/71527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F71527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10613823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXks1WlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=49-55&issue=1&author=J.+Liauthor=M.+Postauthor=R.+Volkauthor=Y.+Gaoauthor=M.+Liauthor=C.+Metaisauthor=K.+Satoauthor=J.+Tsaiauthor=W.+Airdauthor=R.+D.+Rosenbergauthor=T.+G.+Hamptonauthor=J.+Liauthor=F.+Sellkeauthor=P.+Carmelietauthor=M.+Simons&title=PR39%2C+a+Peptide+Regulator+of+Angiogenesis&doi=10.1038%2F71527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PR39, a peptide regulator of angiogenesis</span></div><div class="casAuthors">Li, Jian; Post, Mark; Volk, Rudiger; Gao, Youhe; Li, Min; Metais, Caroline; Sato, Kaori; Tsai, Jo; Aird, William; Rosenberg, Robert D.; Hampton, Thomas G.; Li, Jianyi; Sellke, Frank; Carmeliet, Peter; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-55</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Although tissue injury and inflammation are considered essential for the induction of angiogenesis, the mol. controls of this cascade are mostly unknown.  Here we show that a macrophage-derived peptide, PR39, inhibited the ubiquitin-proteasome-dependent degrdn. of hypoxia-inducible factor-1α protein, resulting in accelerated formation of vascular structures in vitro and increased myocardial vasculature in mice.  For the latter, coronary flow studies demonstrated that PR39-induced angiogenesis resulted in the prodn. of functional blood vessels.  These findings show that PR39 and related compds. can be used as potent inductors of angiogenesis, and that selective inhibition of hypoxia-inducible factor-1α degrdn. may underlie the mechanism of inflammation-induced angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmNGIiFzRNQ7Vg90H21EOLACvtfcHk0lgm_UhQx7shtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXks1WlsA%253D%253D&md5=832cbec47bef0ca645f59de00d2639df</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2F71527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F71527%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPost%26aufirst%3DM.%26aulast%3DVolk%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMetais%26aufirst%3DC.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DAird%26aufirst%3DW.%26aulast%3DRosenberg%26aufirst%3DR.%2BD.%26aulast%3DHampton%26aufirst%3DT.%2BG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSellke%26aufirst%3DF.%26aulast%3DCarmeliet%26aufirst%3DP.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DPR39%252C%2520a%2520Peptide%2520Regulator%2520of%2520Angiogenesis%26jtitle%3DNat.%2520Med.%26date%3D2000%26volume%3D6%26issue%3D1%26spage%3D49%26epage%3D55%26doi%3D10.1038%2F71527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hietaranta, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saluja, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steer, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ubiquitin-Proteasome Pathway-mediated IκBα Degradation by a Naturally Occurring Antibacterial Peptide</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.1172/JCI9826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1172%2FJCI9826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10930447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2000&pages=439-448&issue=3&author=Y.+Gaoauthor=S.+Leckerauthor=M.+J.+Postauthor=A.+J.+Hietarantaauthor=J.+Liauthor=R.+Volkauthor=M.+Liauthor=K.+Satoauthor=A.+K.+Salujaauthor=M.+L.+Steerauthor=A.+L.+Goldbergauthor=M.+Simons&title=Inhibition+of+Ubiquitin-Proteasome+Pathway-mediated+I%CE%BAB%CE%B1+Degradation+by+a+Naturally+Occurring+Antibacterial+Peptide&doi=10.1172%2FJCI9826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of ubiquitin-proteasome pathway-mediated IκBα degradation by a naturally occurring antibacterial peptide</span></div><div class="casAuthors">Gao, Youhe; Lecker, Stewart; Post, Mark J.; Hietaranta, Antti J.; Li, Jian; Volk, Rudiger; Li, Min; Sato, Kaori; Saluja, Ashok K.; Steer, Michael L.; Goldberg, Alfred L.; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">439-448</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Induction of NF-κB-dependent gene expression plays an important role in a no. of biol. processes including inflammation and ischemia-reperfusion injury.  However, few attempts aimed at selective regulation of this transcription factor have been successful.  We report here that a naturally occurring antibacterial peptide PR39 reversibly binds to the α7 subunit of the 26S proteasome and blocks degrdn. of NF-κB inhibitor IκBα by the ubiquitin-proteasome pathway without affecting overall proteasome activity.  IκBα phosphorylation and ubiquitination occur normally after PR39 treatment, and binding of valosin-contg. proteins is not impaired.  The inhibition of IκBα degrdn. abolishes induction of NF-κB-dependent gene expression in cell culture and in mouse models of acute pancreatitis and myocardial infarction, including upregulation of endothelial adhesion proteins VCAM-1 and ICAM-1.  In the latter model, sustained infusion of PR39 peptide resulted in significant redn. of myocardial infarct size.  PR39 and related peptides may provide novel means to regulate cellular function and to control of NF-κB-dependent gene expression for therapeutic purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-ImWsrwT5-bVg90H21EOLACvtfcHk0lgm_UhQx7shtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOrtb8%253D&md5=3deee114b2e7d008929f77a7e6c307b1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1172%2FJCI9826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI9826%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLecker%26aufirst%3DS.%26aulast%3DPost%26aufirst%3DM.%2BJ.%26aulast%3DHietaranta%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DVolk%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DSaluja%26aufirst%3DA.%2BK.%26aulast%3DSteer%26aufirst%3DM.%2BL.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520Ubiquitin-Proteasome%2520Pathway-mediated%2520I%25CE%25BAB%25CE%25B1%2520Degradation%2520by%2520a%2520Naturally%2520Occurring%2520Antibacterial%2520Peptide%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2000%26volume%3D106%26issue%3D3%26spage%3D439%26epage%3D448%26doi%3D10.1172%2FJCI9826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Proline- and Arginine-Rich Peptides Constitute a Novel Class of Allosteric Inhibitors of Proteasome Activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">29</span>),  <span class="NLM_fpage">8663</span>– <span class="NLM_lpage">8670</span>, <span class="refDoi"> DOI: 10.1021/bi034784f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi034784f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltV2jur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=8663-8670&issue=29&author=M.+Gaczynskaauthor=P.+A.+Osmulskiauthor=Y.+Gaoauthor=M.+J.+Postauthor=M.+Simons&title=Proline-+and+Arginine-Rich+Peptides+Constitute+a+Novel+Class+of+Allosteric+Inhibitors+of+Proteasome+Activity&doi=10.1021%2Fbi034784f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Proline- and arginine-rich peptides constitute a novel class of allosteric inhibitors of proteasome activity</span></div><div class="casAuthors">Gaczynska, Maria; Osmulski, Pawel A.; Gao, Youhe; Post, Mark J.; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">8663-8670</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substrate-specific inhibition of the proteasome has been unachievable despite great interest in proteasome inhibitors as drugs.  Recent studies demonstrated that PR39, a natural proline- and arginine-rich antibacterial peptide, stimulates angiogenesis and inhibits inflammatory responses by specifically blocking degrdn. of IκBα and HIF-1α by the proteasome.  However, mol. events involved in the PR39-proteasome interaction have not been elucidated.  Here the authors show that PR39 is a noncompetitive and reversible inhibitor of the proteasome function.  This effect is achieved by a unique allosteric mechanism allowing for specific inhibition of degrdn. of selected proteins without affecting total proteasome-dependent proteolysis.  At. force microscopy (AFM) studies demonstrate that 20S and 26S proteasomes treated with PR39 or its derivs. exhibit serious perturbations in their structure and their normal allosteric movements.  These effects are universal for proteasomes from yeast to human.  The shortest functional sequence derived from PR39 still showing the allosteric inhibitory effect consists of eleven N-terminal residues contg. essential three N-terminal arginines.  The noncompetitive and reversible in vitro action of PR39 and its truncated derivs. is matched by the ability of the peptides to induce angiogenesis in vivo.  The authors postulate that PR39 changes conformational dynamics of the proteasomes by interactions with the noncatalytic subunit α7 in a way that prevents the enzyme from cleaving the substrates of unique structural constraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3kqE8-uSDBLVg90H21EOLACvtfcHk0ljrt-2yVhGlpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltV2jur8%253D&md5=442f1c1a97fd83ae905f85c8134f79c7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi034784f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi034784f%26sid%3Dliteratum%253Aachs%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DPost%26aufirst%3DM.%2BJ.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DProline-%2520and%2520Arginine-Rich%2520Peptides%2520Constitute%2520a%2520Novel%2520Class%2520of%2520Allosteric%2520Inhibitors%2520of%2520Proteasome%2520Activity%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26issue%3D29%26spage%3D8663%26epage%3D8670%26doi%3D10.1021%2Fbi034784f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anbanandam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albarado, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirziu, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veeraraghavan, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Proline- and Arginine-Rich Peptide Inhibition of Proteasome</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.jmb.2008.09.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=18823992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSgs7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=219-227&issue=1&author=A.+Anbanandamauthor=D.+C.+Albaradoauthor=D.+C.+Tirziuauthor=M.+Simonsauthor=S.+Veeraraghavan&title=Molecular+Basis+for+Proline-+and+Arginine-Rich+Peptide+Inhibition+of+Proteasome&doi=10.1016%2Fj.jmb.2008.09.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for Proline- and Arginine-Rich Peptide Inhibition of Proteasome</span></div><div class="casAuthors">Anbanandam, Asokan; Albarado, Diana C.; Tirziu, Daniela C.; Simons, Michael; Veeraraghavan, Sudha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-227</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PR39, a naturally occurring and cell-permeable proline- and arginine-rich peptide, blocks the degrdn. of inhibitor of nuclear factor κB (IκBα), thereby attenuating inflammation.  It is a noncompetitive and reversible inhibitor of 20S proteasome.  To identify its basis of action, we used soln. NMR spectroscopy and mutational analyses of the active fragment, PR11, which identified amino acids required for human 20S proteasome inhibiting activity.  We then examd. PR11-mediated changes in the expression of nuclear factor κB-dependent genes in situ.  The results provide prerequisites for proteasome inhibition by proline- and arginine-rich peptides, providing a powerful new tool to investigate inflammatory processes.  These findings offer new leads in developing drugs to treat heart diseases or stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrujUG4EXG_s7Vg90H21EOLACvtfcHk0ljrt-2yVhGlpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSgs7nE&md5=2a2c8c1854f8a1445ce94947474bb0d8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.021%26sid%3Dliteratum%253Aachs%26aulast%3DAnbanandam%26aufirst%3DA.%26aulast%3DAlbarado%26aufirst%3DD.%2BC.%26aulast%3DTirziu%26aufirst%3DD.%2BC.%26aulast%3DSimons%26aufirst%3DM.%26aulast%3DVeeraraghavan%26aufirst%3DS.%26atitle%3DMolecular%2520Basis%2520for%2520Proline-%2520and%2520Arginine-Rich%2520Peptide%2520Inhibition%2520of%2520Proteasome%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26issue%3D1%26spage%3D219%26epage%3D227%26doi%3D10.1016%2Fj.jmb.2008.09.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myszka, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structural Models for Interactions between the 20S Proteasome and Its PAN/19S Activators</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1074/jbc.C109.070425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fjbc.C109.070425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19889631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=13-17&issue=1&author=B.+M.+Stadtmuellerauthor=K.+Ferrellauthor=F.+G.+Whitbyauthor=A.+Herouxauthor=H.+Robinsonauthor=D.+G.+Myszkaauthor=C.+P.+Hill&title=Structural+Models+for+Interactions+between+the+20S+Proteasome+and+Its+PAN%2F19S+Activators&doi=10.1074%2Fjbc.C109.070425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Models for Interactions between the 20S Proteasome and Its PAN/19S Activators</span></div><div class="casAuthors">Stadtmueller, Beth M.; Ferrell, Katherine; Whitby, Frank G.; Heroux, Annie; Robinson, Howard; Myszka, David G.; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Proteasome activity is regulated by sequestration of its proteolytic centers in a barrel-shaped structure that limits substrate access.  Substrates enter the proteasome by means of activator complexes that bind to the end rings of proteasome α subunits and induce opening of an axial entrance/exit pore.  The PA26 (proteasome activator 26 kDa) activator binds in a pocket on the proteasome surface using main chain contacts of its C-terminal residues and uses an internal activation loop to trigger gate opening by repositioning the proteasome Pro-17 reverse turn.  Subunits of the unrelated 19S/PAN (proteasome-activating nucleosidase) activators bind with their C termini in the same pockets but can induce proteasome gate opening entirely from interactions of their C-terminal peptides, which are reported to cause gate opening by inducing a rocking motion of proteasome α subunits rather than by directly contacting the Pro-17 turn.  Here we report crystal structures and binding studies of proteasome complexes with PA26 constructs that display modified C-terminal residues, including those corresponding to PAN.  These findings suggest that PA26 and PAN/19S C-terminal residues bind superimposably and that both classes of activator induce gate opening by using direct contacts to residues of the proteasome Pro-17 reverse turn.  In the case of the PAN and 19S activators, a penultimate tyrosine/phenylalanine residue contacts the proteasome Gly-19 carbonyl oxygen to stabilize the open conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQoFu-WcOgLLVg90H21EOLACvtfcHk0ljrt-2yVhGlpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbnO&md5=6e01cd7f1f713257c9b2e549ac94e75b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C109.070425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C109.070425%26sid%3Dliteratum%253Aachs%26aulast%3DStadtmueller%26aufirst%3DB.%2BM.%26aulast%3DFerrell%26aufirst%3DK.%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructural%2520Models%2520for%2520Interactions%2520between%2520the%252020S%2520Proteasome%2520and%2520Its%2520PAN%252F19S%2520Activators%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D1%26spage%3D13%26epage%3D17%26doi%3D10.1074%2Fjbc.C109.070425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Traut, T. W.</span></span> <i>Allosteric Regulatory Enzymes</i>; <span class="NLM_publisher-name">Springer US</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2F978-0-387-72891-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=T.+W.+Traut&title=Allosteric+Regulatory+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-72891-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-72891-9%26sid%3Dliteratum%253Aachs%26aulast%3DTraut%26aufirst%3DT.%2BW.%26btitle%3DAllosteric%2520Regulatory%2520Enzymes%26pub%3DSpringer%2520US%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melandri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Characterization of the Chymotryptic Activity of the 20S Proteasome</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3899</span>– <span class="NLM_lpage">3908</span>, <span class="refDoi"> DOI: 10.1021/bi952262x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi952262x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK28XhsFKqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=3899-3908&issue=13&author=R.+L.+Steinauthor=F.+Melandriauthor=L.+Dick&title=Kinetic+Characterization+of+the+Chymotryptic+Activity+of+the+20S+Proteasome&doi=10.1021%2Fbi952262x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of the chymotryptic activity of the 20S proteasome</span></div><div class="casAuthors">Stein, Ross L.; Melandri, Francesco; Dick, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3899-908</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors report kinetic studies for the chymotryptic activity of the 20S proteasome.  Reaction progress curves that are recorded at concns. of Suc-Leu-Leu-Val-Tyr-AMC greater than about 40 μM are biphasic and characterized by initial velocities that decay by a first-order process to final, steady-state velocities.  At [Suc-Leu-Leu-Val-Tyr-AMC] > 40 μM, initial and steady-state velocities are smaller than predicted from simple, Michaelis-Menten kinetics.  The first-order rate const. for the approach to steady-state has a complex dependence on substrate concn. and decreases sigmoidally as substrate concn. increases.  These results indicate that the 20S proteasome is a hysteretic enzyme and is subject to substrate inhibition.  To explain these observations the authors propose a minimal kinetic model with two crit. mechanistic features: (1) the 20S proteasome has two cooperative active sites for Suc-Leu-Leu-Val-Tyr-AMC and (2) there are two interconvertible conformers of active 20S proteasome.  To probe this mechanism in greater detail, the authors explored the kinetic mechanism of inhibition of the 20S proteasome-catalyzed hydrolysis of Suc-Leu-Leu-Val-Tyr-AMC by the peptide aldehyde, Ac-Leu-Leu-Nle-H.  The studies reveal a nonlinear dependence of steady-state velocity on inhibitor concn. (i.e., parabolic inhibition) as well as a nonlinear dependence of the apparent inhibitor dissocn. const. on substrate concn.  Both of these observations are explained by binding of inhibitor at multiple sites on the enzyme.  Taken together, the results of this study indicate that the 20S proteasome is a conformationally flexible protein that can adjust to the binding of ligands and that has multiple and cooperative active sites.  These results support a view of the proteasome's substrate specificity in which (1) substrates are recognized and hydrolyzed by more than one active site; (2) each active site can bind substrates that possess a variety of P1 residues; and (3) the P1 residue plays a relatively minor role as a specificity determinant.  Finally, the authors interpret the results of this study to suggest that, in vivo, the 20S proteasome requires conformational plasticity for its interactions with regulatory complexes and, after it has combined with appropriate regulatory complexes, to catalyze hydrolysis of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQf-rPQS50RrVg90H21EOLACvtfcHk0lhpjlcaQtGnZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFKqsrg%253D&md5=5cbce45f695898e7501d57b72159d24c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi952262x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi952262x%26sid%3Dliteratum%253Aachs%26aulast%3DStein%26aufirst%3DR.%2BL.%26aulast%3DMelandri%26aufirst%3DF.%26aulast%3DDick%26aufirst%3DL.%26atitle%3DKinetic%2520Characterization%2520of%2520the%2520Chymotryptic%2520Activity%2520of%2520the%252020S%2520Proteasome%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26issue%3D13%26spage%3D3899%26epage%3D3908%26doi%3D10.1021%2Fbi952262x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trader, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and Application of a Sensitive Peptide Reporter to Discover 20S Proteasome Stimulators</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.7b00193</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.7b00193" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltV2rtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=269-276&issue=5&author=R.%0AA.+Colemanauthor=D.+J.+Trader&title=Development+and+Application+of+a+Sensitive+Peptide+Reporter+to+Discover+20S+Proteasome+Stimulators&doi=10.1021%2Facscombsci.7b00193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Application of a Sensitive Peptide Reporter to Discover 20S Proteasome Stimulators</span></div><div class="casAuthors">Coleman, Rachel A.; Trader, Darci J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To attenuate an overabundance of cellular protein, it has been hypothesized that the 20S core particle (20S CP) of the proteasome can be chem. stimulated to degrade proteins into nontoxic peptides more quickly.  Screening for small mol. 20S CP stimulators is typically performed with a reporter peptide composed of four amino acids and a coumarin group that is released upon proteasome-mediated hydrolysis to generate a fluorescent signal.  Screening with this small reporter can lead to false negatives because the reporter peptide is rapidly turned-over without stimulation.  To improve the screening for 20S CP stimulators, the authors have developed a peptide FRET reporter nearly four times more sensitive to stimulation but still amenable for high throughput screening.  Through application of the authors' FRET reporter, the authors discovered two 20S CP gate-opening stimulators and also a mol. that elicits its mechanism of action through an interaction with a 20S CP active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaNdZtHuSLubVg90H21EOLACvtfcHk0lhpjlcaQtGnZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltV2rtr0%253D&md5=ba87e0eea43df05bc45d9e17dc240036</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.7b00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.7b00193%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DTrader%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520Application%2520of%2520a%2520Sensitive%2520Peptide%2520Reporter%2520to%2520Discover%252020S%2520Proteasome%2520Stimulators%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26issue%3D5%26spage%3D269%26epage%3D276%26doi%3D10.1021%2Facscombsci.7b00193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">ATP Binding to PAN or the 26S ATPases Causes Association with the 20S Proteasome, Gate Opening, and Translocation of Unfolded Proteins</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2005.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=16337593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtleqtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2005&pages=687-698&issue=5&author=D.+M.+Smithauthor=G.+Kafriauthor=Y.+Chengauthor=D.+Ngauthor=T.+Walzauthor=A.+L.+Goldberg&title=ATP+Binding+to+PAN+or+the+26S+ATPases+Causes+Association+with+the+20S+Proteasome%2C+Gate+Opening%2C+and+Translocation+of+Unfolded+Proteins&doi=10.1016%2Fj.molcel.2005.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ATP binding to PAN or the 26S ATPases causes association with the 20S proteasome, gate opening, and translocation of unfolded proteins</span></div><div class="casAuthors">Smith, David M.; Kafri, Galit; Cheng, Yifan; Ng, David; Walz, Thomas; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">687-698</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The archaeal ATPase complex PAN, the homolog of the eukaryotic 26S proteasome-regulatory ATPases, was shown to assoc. transiently with the 20S proteasome upon binding of ATP or ATPγS, but not ADP.  By electron microscopy (EM), PAN appears as a two-ring structure, capping the 20S, and resembles two densities in the 19S complex.  The N termini of the archaeal 20S α subunits were found to function as a gate that prevents entry of seven-residue peptides but allows entry of tetrapeptides.  Upon assocn. with the 20S particle, PAN stimulates gate opening.  Although degrdn. of globular proteins requires ATP hydrolysis, the PAN-20S complex with ATPγS translocates and degrades unfolded and denatured proteins.  Rabbit 26S proteasomes also degrade these unfolded proteins upon ATP binding, without hydrolysis.  Thus, although unfolding requires energy from ATP hydrolysis, ATP binding alone supports ATPase-20S assocn., gate opening, and translocation of unfolded substrates into the proteasome, which can occur by facilitated diffusion through the ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbfrsl1AGA9rVg90H21EOLACvtfcHk0lhpjlcaQtGnZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtleqtb%252FO&md5=3bf5c913e4880a0b27e098521092d3d0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DKafri%26aufirst%3DG.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DNg%26aufirst%3DD.%26aulast%3DWalz%26aufirst%3DT.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DATP%2520Binding%2520to%2520PAN%2520or%2520the%252026S%2520ATPases%2520Causes%2520Association%2520with%2520the%252020S%2520Proteasome%252C%2520Gate%2520Opening%252C%2520and%2520Translocation%2520of%2520Unfolded%2520Proteins%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D20%26issue%3D5%26spage%3D687%26epage%3D698%26doi%3D10.1016%2Fj.molcel.2005.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez-Castelao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goethals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaño, J. G.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Cleavage of Alpha-Synuclein by the 20S Proteasome and Modulation of Its Degradation by the RedOx State of the N-Terminal Methionines</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2013.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bbamcr.2013.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24315858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFei" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=352-365&issue=2&author=B.+Alvarez-Castelaoauthor=M.+Goethalsauthor=J.+Vandekerckhoveauthor=J.+G.+Casta%C3%B1o&title=Mechanism+of+Cleavage+of+Alpha-Synuclein+by+the+20S+Proteasome+and+Modulation+of+Its+Degradation+by+the+RedOx+State+of+the+N-Terminal+Methionines&doi=10.1016%2Fj.bbamcr.2013.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of cleavage of alpha-synuclein by the 20S proteasome and modulation of its degradation by the RedOx state of the N-terminal methionines</span></div><div class="casAuthors">Alvarez-Castelao, Beatriz; Goethals, Marc; Vandekerckhove, Joel; Castano, Jose G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">352-365</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Alpha-synuclein is a small protein implicated in the pathophysiol. of Parkinson's disease (PD).  We have investigated the mechanism of cleavage of alpha-synuclein by the 20S proteasome.  Alpha-synuclein interacts with the C8 (α7) subunit of the proteasome.  The N-terminal part of alpha-synuclein (amino acids 1-60) is essential for its proteasomal degrdn. and anal. of peptides released from proteasomal digestion allows concluding that initial cleavages occur within the N-terminal region of the mol.  Aggregated alpha-synucleins are also degraded by the proteasome with a reduced rate, likely due to Met oxidn.  In fact, mild oxidn. of alpha-synuclein with H2O2 resulted in the inhibition of its degrdn. by the proteasome, mainly due to oxidn. of Met 1 and 5 of alpha-synuclein.  The inhibition was reversed by treatment of the oxidized protein with methionine sulfoxide reductases (MsrA plus MsrB).  Similarly, treatment with H2O2 of N2A cells transfected with alpha-synuclein resulted in the inhibition of its degrdn. that was also reverted by co-transfection of MsrA plus MsrB.  These results clearly indicate that oxidative stress, a common feature of PD and other synucleinopathies, promotes a RedOx change in the proteostasis of alpha-synuclein due to Met oxidn. and reduced proteasomal degrdn.; compromised reversion of those oxidative changes would result in the accumulation of oxidative damaged alpha-synuclein likely contributing to the pathogenesis of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHZc32bql3P7Vg90H21EOLACvtfcHk0lhIeaT0h1IL0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFei&md5=9deadf1c5b5655efea0ce37e7c0aeee8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2013.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2013.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez-Castelao%26aufirst%3DB.%26aulast%3DGoethals%26aufirst%3DM.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DJ.%2BG.%26atitle%3DMechanism%2520of%2520Cleavage%2520of%2520Alpha-Synuclein%2520by%2520the%252020S%2520Proteasome%2520and%2520Modulation%2520of%2520Its%2520Degradation%2520by%2520the%2520RedOx%2520State%2520of%2520the%2520N-Terminal%2520Methionines%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26issue%3D2%26spage%3D352%26epage%3D365%26doi%3D10.1016%2Fj.bbamcr.2013.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trader, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Establishment of a Suite of Assays That Support the Discovery of Proteasome Stimulators</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1861</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">899</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bbagen.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28065760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1ChtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2017&pages=892-899&issue=4&author=D.+J.+Traderauthor=S.+Simanskiauthor=P.+Dicksonauthor=T.+Kodadek&title=Establishment+of+a+Suite+of+Assays+That+Support+the+Discovery+of+Proteasome+Stimulators&doi=10.1016%2Fj.bbagen.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of a suite of assays that support the discovery of proteasome stimulators</span></div><div class="casAuthors">Trader, Darci J.; Simanski, Scott; Dickson, Paige; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-899</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The proteasome catalyzes the degrdn. of many mis-folded proteins, which are otherwise cytotoxic.  There is interest in the discovery of proteasome agonists, but previous efforts to do so have been disappointing.  The cleavage of small fluorogenic peptides is used routinely as an assay to screen for proteasome modulators.  The authors have developed follow-on assays that employ more physiol. relevant substrates.  To demonstrate the efficacy of this workflow, the NIH Clin. Collection (NCC) was screened.  While many compds. stimulated proteasome-mediated proteolysis of the pro-fluorogenic peptide substrates, most failed to evince activity in assays with larger peptide or protein substrates.  The authors also show that two mols. claimed previously to be proteasome agonists, oleuropein and betulinic acid, indeed accelerate hydrolysis of the fluorogenic substrate, but have no effect on the turnover of a mis-folded protein in vitro or in cellulo.  However, two small mols. from the NCC, MK-866 and AM-404, stimulate the proteasome-mediated turnover of a mis-folded protein in living cells by 3- to 4-fold.  Assays that monitor the proteasome-mediated degrdn. of larger peptides and proteins can distinguish bona fide agonists from compds. only able to stimulate the cleavage of short, nonphysiol. relevant peptides.  A suite of assays has been established that allows the discovery of bona fide proteasome agonists.  AM-404 and MK-866 can be useful tools for cell culture expts., and can serve as scaffolds to generate more potent 20S stimulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKT3yI4GlcnbVg90H21EOLACvtfcHk0lhIeaT0h1IL0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1ChtLg%253D&md5=65e65acaca269de157eb121c81b1f626</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTrader%26aufirst%3DD.%2BJ.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DDickson%26aufirst%3DP.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DEstablishment%2520of%2520a%2520Suite%2520of%2520Assays%2520That%2520Support%2520the%2520Discovery%2520of%2520Proteasome%2520Stimulators%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2017%26volume%3D1861%26issue%3D4%26spage%3D892%26epage%3D899%26doi%3D10.1016%2Fj.bbagen.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilholz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemeyer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammensee, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schild, H.</span></span> <span> </span><span class="NLM_article-title">Cleavage Motifs of the Yeast 20S Proteasome Subunits Deduced from Digests of Enolase 1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">12504</span>– <span class="NLM_lpage">12509</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.21.12504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.95.21.12504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9770515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK1cXmsl2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=12504-12509&issue=21&author=A.+K.+Nussbaumauthor=T.+P.+Dickauthor=W.+Keilholzauthor=M.+Schirleauthor=S.+Stevanovicauthor=K.+Dietzauthor=W.+Heinemeyerauthor=M.+Grollauthor=D.+H.+Wolfauthor=R.+Huberauthor=H.-G.+Rammenseeauthor=H.+Schild&title=Cleavage+Motifs+of+the+Yeast+20S+Proteasome+Subunits+Deduced+from+Digests+of+Enolase+1&doi=10.1073%2Fpnas.95.21.12504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cleavage motifs of the yeast 20S proteasome β subunits deduced from digests of enolase 1</span></div><div class="casAuthors">Nussbaum, Alexander K.; Dick, Tobias P.; Keilholz, Wieland; Schirle, Markus; Stevanovic, Stefan; Dietz, Klaus; Heinemeyer, Wolfgang; Groll, Michael; Wolf, Dieter H.; Huber, Robert; Rammensee, Hans-Georg; Schild, Hansjorg</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12504-12509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 436-amino acid protein enolase 1 from yeast was degraded in vitro by purified wild-type and mutant yeast 20S proteasome particles.  Anal. of the cleavage products at different times revealed a processive degrdn. mechanism and a length distribution of fragments ranging from 3 to 25 amino acids with an av. length of 7 to 8 amino acids.  Surprisingly, the av. fragment length was very similar between wild-type and mutant 20S proteasomes with reduced nos. of active sites.  This implies that the fragment length is not influenced by the distance between the active sites, as previously postulated.  A detailed anal. of the cleavages also allowed the identification of certain amino acid characteristics in positions flanking the cleavage site that guide the selection of the P1 residues by the three active β subunits.  Because yeast and mammalian proteasomes are highly homologous, similar cleavage motifs might be used by mammalian proteasomes.  Therefore, the data provide a basis for predicting proteasomal degrdn. products from which peptides are sampled by major histocompatibility complex class I mols. for presentation to cytotoxic T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI-Ri_79ge-bVg90H21EOLACvtfcHk0lhIeaT0h1IL0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsl2qsLY%253D&md5=d5dcf9cce44a50d37bf3765906cc332d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.21.12504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.21.12504%26sid%3Dliteratum%253Aachs%26aulast%3DNussbaum%26aufirst%3DA.%2BK.%26aulast%3DDick%26aufirst%3DT.%2BP.%26aulast%3DKeilholz%26aufirst%3DW.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DStevanovic%26aufirst%3DS.%26aulast%3DDietz%26aufirst%3DK.%26aulast%3DHeinemeyer%26aufirst%3DW.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DWolf%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DRammensee%26aufirst%3DH.-G.%26aulast%3DSchild%26aufirst%3DH.%26atitle%3DCleavage%2520Motifs%2520of%2520the%2520Yeast%252020S%2520Proteasome%2520Subunits%2520Deduced%2520from%2520Digests%2520of%2520Enolase%25201%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26issue%3D21%26spage%3D12504%26epage%3D12509%26doi%3D10.1073%2Fpnas.95.21.12504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Atomic Force Microscopy Reveals Two Conformations of the 20 S Proteasome from Fission Yeast</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">13171</span>– <span class="NLM_lpage">13174</span>, <span class="refDoi"> DOI: 10.1074/jbc.C901035199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fjbc.C901035199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10747864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtFyksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=13171-13174&issue=18&author=P.+A.+Osmulskiauthor=M.+Gaczynska&title=Atomic+Force+Microscopy+Reveals+Two+Conformations+of+the+20+S+Proteasome+from+Fission+Yeast&doi=10.1074%2Fjbc.C901035199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic force microscopy reveals two conformations of the 20 S proteasome from fission yeast</span></div><div class="casAuthors">Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13171-13174</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The proteasome (multicatalytic proteinase) is a major cytosolic proteolytic complex, indispensable in eukaryotic cells.  The barrel-shaped core of this enzyme, the 20 S proteasome, is built from 28 subunits forming 4 stacked rings.  The 2 inner β-rings harbor active centers, whereas the 2 outer α-rings play a structural role.  The previously reported crystal structure of the yeast 20 S particle showed that the entrance to the central channel was sealed.  Because of this result, the path of substrates into the catalytic chamber has remained enigmatic.  Here, the authors used tapping mode at. force microscopy (AFM) in liq. to address the dynamic aspects of the 20 S proteasomes from fission yeast (Schizosaccharomyces pombe).  Evidence is presented that, when obsd. with AFM, the proteasome particles in top view position have either open or closed entrance to the central channel.  The preferred conformation depends on the ligands present.  Apparently, the addn. of a substrate to the uninhibited proteasome shifts the equil. toward the open conformation.  These results shed new light on the possible path of the substrate into the proteolytic chamber.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzu2Vm0qhUbVg90H21EOLACvtfcHk0ljIcgtdrFROLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtFyksrk%253D&md5=dc6794d80535297849227a03a1df0bbb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C901035199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C901035199%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAtomic%2520Force%2520Microscopy%2520Reveals%2520Two%2520Conformations%2520of%2520the%252020%2520S%2520Proteasome%2520from%2520Fission%2520Yeast%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26issue%3D18%26spage%3D13171%26epage%3D13174%26doi%3D10.1074%2Fjbc.C901035199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochstrasser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">A Tetrahedral Transition State at the Active Sites of the 20S Proteasome Is Coupled to Opening of the α-Ring Channel</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.str.2009.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19679091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1CltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1137-1147&issue=8&author=P.+A.+Osmulskiauthor=M.+Hochstrasserauthor=M.+Gaczynska&title=A+Tetrahedral+Transition+State+at+the+Active+Sites+of+the+20S+Proteasome+Is+Coupled+to+Opening+of+the+%CE%B1-Ring+Channel&doi=10.1016%2Fj.str.2009.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Tetrahedral Transition State at the Active Sites of the 20S Proteasome Is Coupled to Opening of the α-Ring Channel</span></div><div class="casAuthors">Osmulski, Pawel A.; Hochstrasser, Mark; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1137-1147</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Intrinsic conformational transitions contribute to the catalytic action of many enzymes.  Here we use a single-mol. approach to demonstrate how such transitions are linked to the catalytic sites of the eukaryotic proteasome, an essential protease of the ubiquitin pathway.  The active sites of the cylindrical proteasomal core particle are located in a central chamber accessible through gated entry channels.  By using at. force microscopy, we found continual alternation between open and closed gate conformations.  We analyzed the relative abundance of these conformers in wild-type and mutated yeast core particles upon exposure to substrates or inhibitors.  Our data indicate that the dynamic gate can be opened by allosteric coupling to a tetrahedral transition state at any of the working active centers.  The results point to the Nα-amine of the N-terminal active site threonyl residue as the major effector group responsible for triggering the essential conformational switch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiHKa1tntq-rVg90H21EOLACvtfcHk0ljIcgtdrFROLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1CltLo%253D&md5=73da479a8278c7a3b5dc434390bf4883</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DHochstrasser%26aufirst%3DM.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DA%2520Tetrahedral%2520Transition%2520State%2520at%2520the%2520Active%2520Sites%2520of%2520the%252020S%2520Proteasome%2520Is%2520Coupled%2520to%2520Opening%2520of%2520the%2520%25CE%25B1-Ring%2520Channel%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26issue%3D8%26spage%3D1137%26epage%3D1147%26doi%3D10.1016%2Fj.str.2009.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span> <span> </span><span class="NLM_article-title">Conformational Landscape of the p28-Bound Human Proteasome Regulatory Particle</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">333</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2017.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28689658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCgtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=322-333&issue=2&author=Y.+Luauthor=J.+Wuauthor=Y.+Dongauthor=S.+Chenauthor=S.+Sunauthor=Y.+B.+Maauthor=Q.+Ouyangauthor=D.+Finleyauthor=M.+W.+Kirschnerauthor=Y.+Mao&title=Conformational+Landscape+of+the+p28-Bound+Human+Proteasome+Regulatory+Particle&doi=10.1016%2Fj.molcel.2017.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Landscape of the p28-Bound Human Proteasome Regulatory Particle</span></div><div class="casAuthors">Lu, Ying; Wu, Jiayi; Dong, Yuanchen; Chen, Shuobing; Sun, Shuangwu; Ma, Yong-Bei; Ouyang, Qi; Finley, Daniel; Kirschner, Marc W.; Mao, Youdong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">322-333.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The proteasome holoenzyme is activated by its regulatory particle (RP) consisting of two subcomplexes, the lid and the base.  A key event in base assembly is the formation of a heterohexameric ring of AAA-ATPases, which is guided by at least four RP assembly chaperones in mammals: PAAF1, p28/gankyrin, p27/PSMD9, and S5b.  Using cryogenic electron microscopy, we analyzed the non-AAA structure of the p28-bound human RP at 4.5 Å resoln. and detd. seven distinct conformations of the Rpn1-p28-AAA subcomplex within the p28-bound RP at subnanometer resolns.  Remarkably, the p28-bound AAA ring does not form a channel in the free RP and spontaneously samples multiple "open" and "closed" topologies at the Rpt2-Rpt6 and Rpt3-Rpt4 interfaces.  Our anal. suggests that p28 assists the proteolytic core particle to select a specific conformation of the ATPase ring for RP engagement and is released in a shoehorn-like fashion in the last step of the chaperone-mediated proteasome assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3cD6iJtLSbLVg90H21EOLACvtfcHk0ljIcgtdrFROLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCgtbnE&md5=0863ec470f4dca12ef4ec78297dfdabb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DY.%2BB.%26aulast%3DOuyang%26aufirst%3DQ.%26aulast%3DFinley%26aufirst%3DD.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DMao%26aufirst%3DY.%26atitle%3DConformational%2520Landscape%2520of%2520the%2520p28-Bound%2520Human%2520Proteasome%2520Regulatory%2520Particle%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D67%26issue%3D2%26spage%3D322%26epage%3D333%26doi%3D10.1016%2Fj.molcel.2017.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Allosteric Regulators of the Proteasome: Potential Drugs and a Novel Approach for Drug Design</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.2174/092986706775197926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.2174%2F092986706775197926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=16472211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVSiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=155-165&issue=2&author=X.+Tanauthor=P.+A.+A.+Osmulskiauthor=M.+Gaczynska&title=Allosteric+Regulators+of+the+Proteasome%3A+Potential+Drugs+and+a+Novel+Approach+for+Drug+Design&doi=10.2174%2F092986706775197926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric regulators of the proteasome: Potential drugs and a novel approach for drug design</span></div><div class="casAuthors">Tan, Xiaolin; Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">155-165</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The proteasome recently gained an exceptional attention as a novel drug target, therefore its inhibitors became important subjects for rational drug design.  A synthetic competitive inhibitor Velcade was lately approved in a fast-track process to treat multiple myeloma and is tested with other types of cancers.  The proteasome is a major proteolytic assembly in eukaryotic cells responsible for the degrdn. of most intracellular proteins, including proteins crucial to cell cycle regulation and apoptosis.  The ubiquitin-proteasome pathway has been implicated in many diseases such as cancer, autoimmune diseases, inflammation, and stroke.  The activity of the proteasome can be blocked for therapeutic purposes with competitive inhibitors like Velcade, which trigger apoptosis in target cells.  However, much more versatile outcomes and a true control of the proteasome can be achieved with allosteric regulators.  Certain natural proteins and peptides bind to the catalytic core of the proteasome and allosterically induce a wide array of effects ranging from changes in product size to substrate-specific inhibition.  Designing small synthetic compds. allosterically interacting with the proteasome represents a novel approach that has enormous potential for the treatment of a wide range of diseases.  Below we provide a review of current knowledge about proteasomal allosteric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXUmGL0qOPSLVg90H21EOLACvtfcHk0liOhEtc-YzvKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVSiug%253D%253D&md5=e8acd061f5ef976b61b18cc40b1e2bef</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2174%2F092986706775197926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986706775197926%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DX.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAllosteric%2520Regulators%2520of%2520the%2520Proteasome%253A%2520Potential%2520Drugs%2520and%2520a%2520Novel%2520Approach%2520for%2520Drug%2520Design%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D155%26epage%3D165%26doi%3D10.2174%2F092986706775197926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wehmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aufderheide, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, E.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Functional Cycle of the ATPase Module of the 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1305</span>– <span class="NLM_lpage">1310</span>, <span class="refDoi"> DOI: 10.1073/pnas.1621129114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.1621129114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28115689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=1305-1310&issue=6&author=M.+Wehmerauthor=T.+Rudackauthor=F.+Beckauthor=A.+Aufderheideauthor=G.+Pfeiferauthor=J.+M.+Plitzkoauthor=F.+F%C3%B6rsterauthor=K.+Schultenauthor=W.+Baumeisterauthor=E.+Sakata&title=Structural+Insights+into+the+Functional+Cycle+of+the+ATPase+Module+of+the+26S+Proteasome&doi=10.1073%2Fpnas.1621129114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the functional cycle of the ATPase module of the 26S proteasome</span></div><div class="casAuthors">Wehmer, Marc; Rudack, Till; Beck, Florian; Aufderheide, Antje; Pfeifer, Guenter; Plitzko, Juergen M.; Foerster, Friedrich; Schulten, Klaus; Baumeister, Wolfgang; Sakata, Eri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1310</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In eukaryotic cells, the ubiquitin-proteasome system (UPS) is responsible for the regulated degrdn. of intracellular proteins.  The 26S holocomplex comprises the core particle (CP), where proteolysis takes place, and 1 or 2 regulatory particles (RPs).  The base of the RP is formed by a heterohexameric AAA+-ATPase module, which unfolds and translocates substrates into the CP.  Here, applying single-particle cryo-electron microscopy (cryo-EM) and image classification to samples in the presence of different nucleotides and nucleotide analogs, the authors were able to observe 4 distinct conformational states (s1-s4).  The resoln. of the 4 conformers allowed for the construction of at. models of the AAA+-ATPase module as it progresses through the functional cycle.  In a hitherto unobserved state (s4), the gate controlling access to the CP was open.  The structures described in this study allow the authors to put forward a model for the 26S functional cycle driven by ATP hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3MiQWEovbLrVg90H21EOLACvtfcHk0liOhEtc-YzvKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGjs78%253D&md5=9dfa75847017d5f973da07790229ca65</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1621129114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1621129114%26sid%3Dliteratum%253Aachs%26aulast%3DWehmer%26aufirst%3DM.%26aulast%3DRudack%26aufirst%3DT.%26aulast%3DBeck%26aufirst%3DF.%26aulast%3DAufderheide%26aufirst%3DA.%26aulast%3DPfeifer%26aufirst%3DG.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DF%25C3%25B6rster%26aufirst%3DF.%26aulast%3DSchulten%26aufirst%3DK.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DSakata%26aufirst%3DE.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Functional%2520Cycle%2520of%2520the%2520ATPase%2520Module%2520of%2520the%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D6%26spage%3D1305%26epage%3D1310%26doi%3D10.1073%2Fpnas.1621129114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span>; <span class="NLM_string-name">Osmulski, P. A.</span></span> <span> </span><span class="NLM_article-title">Atomic Force Microscopy of Proteasome Assemblies</span>. In  <i>Atomic Force Microscopy in Biomedical Research</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braga, P.</span>, <span class="NLM_string-name">Ricci, D.</span></span>, Eds.; Methods in Molecular Biology, Vol. 736; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-105-5_9</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2F978-1-61779-105-5_9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=117-132&author=M.+Gaczynska&author=P.+A.+Osmulskiauthor=P.+Braga&author=D.+Ricci&title=Atomic+Force+Microscopy+in+Biomedical+Research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-105-5_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-61779-105-5_9%26sid%3Dliteratum%253Aachs%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAtomic%2520Force%2520Microscopy%2520of%2520Proteasome%2520Assemblies%26btitle%3DAtomic%2520Force%2520Microscopy%2520in%2520Biomedical%2520Research%26aulast%3DBraga%26aufirst%3DP.%26pub%3DHumana%2520Press%26date%3D2011%26spage%3D117%26epage%3D132%26doi%3D10.1007%2F978-1-61779-105-5_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhS6bi7eZMi2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">IUCr. Molecular Replacement with <i>MOLREP</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&issue=1&author=A.+Vaginauthor=A.+Teplyakov&title=IUCr.+Molecular+Replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0lhS6bi7eZMi2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DIUCr.%2520Molecular%2520Replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D1%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">IUCr. <i>REFMAC</i> 5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&issue=4&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=IUCr.+REFMAC+5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lhS6bi7eZMi2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DIUCr.%2520REFMAC%25205%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3D4%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&issue=Part+4&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhS6bi7eZMi2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3DPart%25204%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">IUCr. <i>TLSMD</i> Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&issue=1&author=J.+Painterauthor=E.+A.+Merritt&title=IUCr.+TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0lhaJQOq9cfIgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DIUCr.%2520TLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26issue%3D1%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhaJQOq9cfIgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R Value: A Novel Statistical Quantity for Assessing the Accuracy of Crystal Structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span> (<span class="NLM_issue">6359</span>),  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&issue=6359&author=A.+T.+Br%C3%BCnger&title=Free+R+Value%3A+A+Novel+Statistical+Quantity+for+Assessing+the+Accuracy+of+Crystal+Structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value:  a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Bruenger, Axel T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A reliable and unbiased indicator of the accuracy is proposed for fitting at. models for macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCvKGOC-AKrVg90H21EOLACvtfcHk0lhaJQOq9cfIgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D&md5=a382d593937b2394d9354ccba49cdbc3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520Value%253A%2520A%2520Novel%2520Statistical%2520Quantity%2520for%2520Assessing%2520the%2520Accuracy%2520of%2520Crystal%2520Structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26issue%3D6359%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&issue=1&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljZbvfCAOmnjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D1%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 14 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Taylor J. Fiolek, Katarina L. Keel, <span class="NLM_string-name hlFld-ContribAuthor">Jetze J. Tepe</span>. </span><span class="cited-content_cbyCitation_article-title">Fluspirilene Analogs Activate the 20S Proteasome and Overcome Proteasome Impairment by Intrinsically Disordered Protein Oligomers. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (8)
                                     , 1438-1448. <a href="https://doi.org/10.1021/acschemneuro.1c00099" title="DOI URL">https://doi.org/10.1021/acschemneuro.1c00099</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.1c00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.1c00099%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DFluspirilene%252BAnalogs%252BActivate%252Bthe%252B20S%252BProteasome%252Band%252BOvercome%252BProteasome%252BImpairment%252Bby%252BIntrinsically%252BDisordered%252BProtein%252BOligomers%26aulast%3DFiolek%26aufirst%3DTaylor%2BJ.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20022021%26date%3D17032021%26date%3D31032021%26volume%3D12%26issue%3D8%26spage%3D1438%26epage%3D1448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Evert Njomen, <span class="NLM_string-name hlFld-ContribAuthor">Jetze J. Tepe</span>. </span><span class="cited-content_cbyCitation_article-title">Proteasome Activation as a New Therapeutic Approach To Target Proteotoxic Disorders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6469-6481. <a href="https://doi.org/10.1021/acs.jmedchem.9b00101" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00101%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteasome%252BActivation%252Bas%252Ba%252BNew%252BTherapeutic%252BApproach%252BTo%252BTarget%252BProteotoxic%252BDisorders%26aulast%3DNjomen%26aufirst%3DEvert%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15012019%26date%3D14032019%26date%3D06032019%26volume%3D62%26issue%3D14%26spage%3D6469%26epage%3D6481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  García-Viñuales</span>, <span class="hlFld-ContribAuthor ">Michele F.M.  Sciacca</span>, <span class="hlFld-ContribAuthor ">Valeria  Lanza</span>, <span class="hlFld-ContribAuthor ">Anna Maria  Santoro</span>, <span class="hlFld-ContribAuthor ">Giulia  Grasso</span>, <span class="hlFld-ContribAuthor ">Grazia R.  Tundo</span>, <span class="hlFld-ContribAuthor ">Diego  Sbardella</span>, <span class="hlFld-ContribAuthor ">Massimiliano  Coletta</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Grasso</span>, <span class="hlFld-ContribAuthor ">Carmelo  La Rosa</span>, <span class="hlFld-ContribAuthor ">Danilo  Milardi</span>. </span><span class="cited-content_cbyCitation_article-title">The interplay between lipid and Aβ amyloid homeostasis in Alzheimer’s Disease: risk factors and therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Chemistry and Physics of Lipids</span><span> <strong>2021,</strong> <em>236 </em>, 105072. <a href="https://doi.org/10.1016/j.chemphyslip.2021.105072" title="DOI URL">https://doi.org/10.1016/j.chemphyslip.2021.105072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemphyslip.2021.105072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemphyslip.2021.105072%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520and%2520Physics%2520of%2520Lipids%26atitle%3DThe%252Binterplay%252Bbetween%252Blipid%252Band%252BA%2525CE%2525B2%252Bamyloid%252Bhomeostasis%252Bin%252BAlzheimer%2525E2%252580%252599s%252BDisease%25253A%252Brisk%252Bfactors%252Band%252Btherapeutic%252Bopportunities%26aulast%3DGarc%25C3%25ADa-Vi%25C3%25B1uales%26aufirst%3DSara%26date%3D2021%26volume%3D236%26spage%3D105072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel A.  Coleman</span>, <span class="hlFld-ContribAuthor ">Rodrigo  Mohallem</span>, <span class="hlFld-ContribAuthor ">Uma K.  Aryal</span>, <span class="hlFld-ContribAuthor ">Darci J.  Trader</span>. </span><span class="cited-content_cbyCitation_article-title">Protein degradation profile reveals dynamic nature of 20S proteasome small molecule stimulation. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>2 </em>
                                    (2)
                                     , 636-644. <a href="https://doi.org/10.1039/D0CB00191K" title="DOI URL">https://doi.org/10.1039/D0CB00191K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CB00191K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CB00191K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DProtein%252Bdegradation%252Bprofile%252Breveals%252Bdynamic%252Bnature%252Bof%252B20S%252Bproteasome%252Bsmall%252Bmolecule%252Bstimulation%26aulast%3DColeman%26aufirst%3DRachel%2BA.%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D2%26spage%3D636%26epage%3D644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tapan  Behl</span>, <span class="hlFld-ContribAuthor ">Swati  Chadha</span>, <span class="hlFld-ContribAuthor ">Monika  Sachdeva</span>, <span class="hlFld-ContribAuthor ">Arun  Kumar</span>, <span class="hlFld-ContribAuthor ">Abdul  Hafeez</span>, <span class="hlFld-ContribAuthor ">Vineet  Mehta</span>, <span class="hlFld-ContribAuthor ">Simona  Bungau</span>. </span><span class="cited-content_cbyCitation_article-title">Ubiquitination in rheumatoid arthritis. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2020,</strong> <em>261 </em>, 118459. <a href="https://doi.org/10.1016/j.lfs.2020.118459" title="DOI URL">https://doi.org/10.1016/j.lfs.2020.118459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2020.118459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2020.118459%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DUbiquitination%252Bin%252Brheumatoid%252Barthritis%26aulast%3DBehl%26aufirst%3DTapan%26date%3D2020%26volume%3D261%26spage%3D118459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna Maria  Santoro</span>, <span class="hlFld-ContribAuthor ">Alessandro  D’Urso</span>, <span class="hlFld-ContribAuthor ">Alessandra  Cunsolo</span>, <span class="hlFld-ContribAuthor ">Danilo  Milardi</span>, <span class="hlFld-ContribAuthor ">Roberto  Purrello</span>, <span class="hlFld-ContribAuthor ">Diego  Sbardella</span>, <span class="hlFld-ContribAuthor ">Grazia R.  Tundo</span>, <span class="hlFld-ContribAuthor ">Donatella  Diana</span>, <span class="hlFld-ContribAuthor ">Roberto  Fattorusso</span>, <span class="hlFld-ContribAuthor ">Antonio Di  Dato</span>, <span class="hlFld-ContribAuthor ">Antonella  Paladino</span>, <span class="hlFld-ContribAuthor ">Marco  Persico</span>, <span class="hlFld-ContribAuthor ">Massimo  Coletta</span>, <span class="hlFld-ContribAuthor ">Caterina  Fattorusso</span>. </span><span class="cited-content_cbyCitation_article-title">Cooperative Binding of the Cationic Porphyrin Tris-T4 Enhances Catalytic Activity of 20S Proteasome Unveiling a Complex Distribution of Functional States. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (19)
                                     , 7190. <a href="https://doi.org/10.3390/ijms21197190" title="DOI URL">https://doi.org/10.3390/ijms21197190</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21197190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21197190%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DCooperative%252BBinding%252Bof%252Bthe%252BCationic%252BPorphyrin%252BTris-T4%252BEnhances%252BCatalytic%252BActivity%252Bof%252B20S%252BProteasome%252BUnveiling%252Ba%252BComplex%252BDistribution%252Bof%252BFunctional%252BStates%26aulast%3DSantoro%26aufirst%3DAnna%2BMaria%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D19%26spage%3D7190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">G.R.  Tundo</span>, <span class="hlFld-ContribAuthor ">D.  Sbardella</span>, <span class="hlFld-ContribAuthor ">A.M.  Santoro</span>, <span class="hlFld-ContribAuthor ">A.  Coletta</span>, <span class="hlFld-ContribAuthor ">F.  Oddone</span>, <span class="hlFld-ContribAuthor ">G.  Grasso</span>, <span class="hlFld-ContribAuthor ">D.  Milardi</span>, <span class="hlFld-ContribAuthor ">P.M.  Lacal</span>, <span class="hlFld-ContribAuthor ">S.  Marini</span>, <span class="hlFld-ContribAuthor ">R.  Purrello</span>, <span class="hlFld-ContribAuthor ">G.  Graziani</span>, <span class="hlFld-ContribAuthor ">M.  Coletta</span>. </span><span class="cited-content_cbyCitation_article-title">The proteasome as a druggable target with multiple therapeutic potentialities: Cutting and non-cutting edges. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2020,</strong> <em>213 </em>, 107579. <a href="https://doi.org/10.1016/j.pharmthera.2020.107579" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2020.107579</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2020.107579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2020.107579%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DThe%252Bproteasome%252Bas%252Ba%252Bdruggable%252Btarget%252Bwith%252Bmultiple%252Btherapeutic%252Bpotentialities%25253A%252BCutting%252Band%252Bnon-cutting%252Bedges%26aulast%3DTundo%26aufirst%3DG.R.%26date%3D2020%26volume%3D213%26spage%3D107579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dhiraj  Kumar</span>, <span class="hlFld-ContribAuthor ">Rashmi K.  Ambasta</span>, <span class="hlFld-ContribAuthor ">Pravir  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Ubiquitin biology in neurodegenerative disorders: From impairment to therapeutic strategies. </span><span class="cited-content_cbyCitation_journal-name">Ageing Research Reviews</span><span> <strong>2020,</strong> <em>61 </em>, 101078. <a href="https://doi.org/10.1016/j.arr.2020.101078" title="DOI URL">https://doi.org/10.1016/j.arr.2020.101078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.arr.2020.101078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.arr.2020.101078%26sid%3Dliteratum%253Aachs%26jtitle%3DAgeing%2520Research%2520Reviews%26atitle%3DUbiquitin%252Bbiology%252Bin%252Bneurodegenerative%252Bdisorders%25253A%252BFrom%252Bimpairment%252Bto%252Btherapeutic%252Bstrategies%26aulast%3DKumar%26aufirst%3DDhiraj%26date%3D2020%26volume%3D61%26spage%3D101078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Thomas  Meul</span>, <span class="hlFld-ContribAuthor ">Silke  Meiners</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the proteasome system: A novel concept of proteasome inhibition and regulation. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2020,</strong> <em>211 </em>, 107526. <a href="https://doi.org/10.1016/j.pharmthera.2020.107526" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2020.107526</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2020.107526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2020.107526%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DExploring%252Bthe%252Bproteasome%252Bsystem%25253A%252BA%252Bnovel%252Bconcept%252Bof%252Bproteasome%252Binhibition%252Band%252Bregulation%26aulast%3DWang%26aufirst%3DXinyuan%26date%3D2020%26volume%3D211%26spage%3D107526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paweł A.  Osmulski</span>, <span class="hlFld-ContribAuthor ">Przemysław  Karpowicz</span>, <span class="hlFld-ContribAuthor ">Elżbieta  Jankowska</span>, <span class="hlFld-ContribAuthor ">Jonathan  Bohmann</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Pickering</span>, <span class="hlFld-ContribAuthor ">Maria  Gaczyńska</span>. </span><span class="cited-content_cbyCitation_article-title">New Peptide-Based Pharmacophore Activates 20S Proteasome. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (6)
                                     , 1439. <a href="https://doi.org/10.3390/molecules25061439" title="DOI URL">https://doi.org/10.3390/molecules25061439</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25061439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25061439%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNew%252BPeptide-Based%252BPharmacophore%252BActivates%252B20S%252BProteasome%26aulast%3DOsmulski%26aufirst%3DPawe%25C5%2582%2BA.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D6%26spage%3D1439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna Maria  Santoro</span>, <span class="hlFld-ContribAuthor ">Valeria  Lanza</span>, <span class="hlFld-ContribAuthor ">Francesco  Bellia</span>, <span class="hlFld-ContribAuthor ">Diego  Sbardella</span>, <span class="hlFld-ContribAuthor ">Grazia R.  Tundo</span>, <span class="hlFld-ContribAuthor ">Alessandra  Cannizzo</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Grasso</span>, <span class="hlFld-ContribAuthor ">Mariaconcetta  Arizzi</span>, <span class="hlFld-ContribAuthor ">Vincenzo G.  Nicoletti</span>, <span class="hlFld-ContribAuthor ">Stefano  Alcaro</span>, <span class="hlFld-ContribAuthor ">Giosuè  Costa</span>, <span class="hlFld-ContribAuthor ">Adriana  Pietropaolo</span>, <span class="hlFld-ContribAuthor ">Gaetano  Malgieri</span>, <span class="hlFld-ContribAuthor ">Gianluca  D'Abrosca</span>, <span class="hlFld-ContribAuthor ">Roberto  Fattorusso</span>, <span class="hlFld-ContribAuthor ">Sara  García‐Viñuales</span>, <span class="hlFld-ContribAuthor ">Ikhlas M. M.  Ahmed</span>, <span class="hlFld-ContribAuthor ">Massimiliano  Coletta</span>, <span class="hlFld-ContribAuthor ">Danilo  Milardi</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrazolones Activate the Proteasome by Gating Mechanisms and Protect Neuronal Cells from β‐Amyloid Toxicity. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (3)
                                     , 302-316. <a href="https://doi.org/10.1002/cmdc.201900612" title="DOI URL">https://doi.org/10.1002/cmdc.201900612</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900612%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DPyrazolones%252BActivate%252Bthe%252BProteasome%252Bby%252BGating%252BMechanisms%252Band%252BProtect%252BNeuronal%252BCells%252Bfrom%252B%2525CE%2525B2%2525E2%252580%252590Amyloid%252BToxicity%26aulast%3DSantoro%26aufirst%3DAnna%2BMaria%26date%3D2020%26date%3D2019%26volume%3D15%26issue%3D3%26spage%3D302%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Corey L.  Jones</span>, <span class="hlFld-ContribAuthor ">Jetze J.  Tepe</span>. </span><span class="cited-content_cbyCitation_article-title">Proteasome Activation to Combat Proteotoxicity. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (15)
                                     , 2841. <a href="https://doi.org/10.3390/molecules24152841" title="DOI URL">https://doi.org/10.3390/molecules24152841</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24152841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24152841%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DProteasome%252BActivation%252Bto%252BCombat%252BProteotoxicity%26aulast%3DJones%26aufirst%3DCorey%2BL.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D15%26spage%3D2841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel A.  Coleman</span>, <span class="hlFld-ContribAuthor ">Darci J.  Trader</span>. </span><span class="cited-content_cbyCitation_article-title">Methods to Discover and Evaluate Proteasome Small Molecule Stimulators. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (12)
                                     , 2341. <a href="https://doi.org/10.3390/molecules24122341" title="DOI URL">https://doi.org/10.3390/molecules24122341</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24122341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24122341%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMethods%252Bto%252BDiscover%252Band%252BEvaluate%252BProteasome%252BSmall%252BMolecule%252BStimulators%26aulast%3DColeman%26aufirst%3DRachel%2BA.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D12%26spage%3D2341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew B.  Giletto</span>, <span class="hlFld-ContribAuthor ">Pawel A.  Osmulski</span>, <span class="hlFld-ContribAuthor ">Corey L.  Jones</span>, <span class="hlFld-ContribAuthor ">Maria E.  Gaczynska</span>, <span class="hlFld-ContribAuthor ">Jetze J.  Tepe</span>. </span><span class="cited-content_cbyCitation_article-title">Pipecolic esters as minimized templates for proteasome inhibition. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (10)
                                     , 2734-2746. <a href="https://doi.org/10.1039/C9OB00122K" title="DOI URL">https://doi.org/10.1039/C9OB00122K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB00122K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB00122K%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DPipecolic%252Besters%252Bas%252Bminimized%252Btemplates%252Bfor%252Bproteasome%252Binhibition%26aulast%3DGiletto%26aufirst%3DMatthew%2BB.%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D10%26spage%3D2734%26epage%3D2746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. ChT-L peptidase activity of SDS-activated human 20S proteasome was inhibited by selected PR peptides in a dose dependent manner, with the exception of <b>1</b> and <b>2</b>. <b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S1</a>) and <b>7</b> were better inhibitors than <b>11</b> (red line), whereas <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S1</a>) and <b>12</b> were less efficient. The enzyme activity was probed with the substrate Suc-LLVY-AMC. Each titration was performed in three independent replicates. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Inhibition of the ChT-L peptidase activity of h20S by <b>4</b> (a) or <b>7</b> (b) followed the mechanism of noncompetitive mixed inhibition. In this mechanism, proteasome preserves partial catalytic activity, while the inhibitor binding affinity also depends on occupancy of the active centers. Each titration was performed in three independent replicates. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>7</b> and <b>11</b> inhibited 20S proteasome activated with 1 μM (a) or 10 μM Rpt5 (b). IC<sub>50</sub> calculated for <b>7</b> did not depend on Rpt5 concentration, whereas for <b>11</b> it was slightly increased at the higher concentration of Rpt5. In contrast, in the presence of 1 μM Rpt5, <b>2</b> additively activated proteasome (a). This effect was abolished at 10 μM Rpt5 (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ChT-L activity of the latent 20S proteasome, probed with the substrate Suc-LLVY-AMC, was strongly and diversely regulated by PR peptides. At low concentrations <b>6</b>, <b>11</b>, and <b>12</b> transiently activated the proteasome. <b>1</b> and <b>2</b> proved to be strong activators of the latent CP and induced typical sigmoidal dose response stimulation. The results are presented as the mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) ChT-L activity of the SDS-activated proteasome, probed with LFP substrate. Almost all analogs, except for <b>1</b> and <b>2</b>, inhibited h20S, with <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S5a</a>) and <b>7</b> being more efficient and <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf" class="ext-link">Figure S5a</a>) and <b>3</b> significantly less efficient than <b>11</b> (red line). (b) <b>1</b> and <b>2</b> were the only compounds among the studied set of PR analogs able to stimulate the activity of the latent h20S. The maximum activation effect reached 8-fold for <b>1</b> and 14-fold for <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of protein substrates by human 20S proteasome was strongly affected by PR peptides. Levels of the remaining substrates were determined based on SDS–PAGE electrophoretic separation of proteins incubated with 20S proteasome. Almost 70% of α-synuclein (a) and 50% of enolase (b) were degraded by 20S proteasome under the applied experimental conditions. Representative SDS–PAGE electrophoregrams with Coomassie-stained substrate bands are presented below the columns. At 10 μM, <b>1</b> and <b>2</b> accelerated the digest to completely degrade enolase. Other PR peptides, especially at 10 μM concentration, efficiently blocked degradation of the model proteins, with <b>6</b> and <b>7</b> being the best inhibitors. NS = not statistically significant; all other cases are statistically significant (<i>p</i> < 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Treatment of human cultured fibroblasts with compound <b>2</b> (24 h) did not affect viability of the cells, even at high peptide concentrations (a). ChT-L activity of proteasome in cell extracts prepared from human fibroblasts treated with <b>2</b> was activated up to 3-fold in a dose dependent manner (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. PR peptides influenced the abundance of the α-face conformations of 20S proteasome, as detected by AFM. (a) In control, untreated samples, about 70% of molecules had their central channel covered and classified as a closed gate conformation. About 7% of molecules had their gate completely open, and the remaining 22% were in the process of switching between these conformations and were classified as intermediates. In the presence of an activating 1 μM concentration of <b>2</b>, the abundance of open and intermediate conformations increased to 54% (18% and 36%, respectively). In contrast, a 1 μM concentration of <b>7</b> decreased the number of open conformers to less than 3%. The higher concentration of <b>7</b> pushed proteasome to open the gate in 18% of molecules, with a slightly higher contribution of intermediates (25%). (b) Representative AFM images of the three conformational forms of the core proteasome. The top-view images with the α-face exposed were zoomed-in from 1 μm × 1 μm fields. The images are raw and have been subjected only to planefitting/flattening, linear adjustments of brightness and contrast, and linear interpolation for viewing clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/medium/jm-2018-010254_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Interaction of <b>6</b> with yeast 20S proteasome (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z">4X6Z</a>). (a) General (left) and detailed (blow-up) localization of the inhibitor binding site between α5 (wheat color) and α6 (gray) subunits. It is visible in the crystal structure that the C-terminal fragment of <b>6</b> (green) binds to the ε-amine group of the conserved α6Lys62 through its carboxylate group. (b) Electron density defining the fragment of <b>6</b> included in the model (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map contoured at 1σ level). The cartoon of general proteasome structure in (a), left, has also been used in our earlier work.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-1/acs.jmedchem.8b01025/20191119/images/large/jm-2018-010254_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01025&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28427" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28427" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, A. L.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin-Mediated Proteolytic Pathway: Mode of Action and Clinical Implications</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">Suppl. 34</span>),  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-4644(2000)77:34+<40::AID-JCB9>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2000&pages=40-51&issue=Suppl.+34&author=A.+Ciechanoverauthor=A.+Orianauthor=A.+L.+Schwartz&title=The+Ubiquitin-Mediated+Proteolytic+Pathway%3A+Mode+of+Action+and+Clinical+Implications&doi=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-4644%25282000%252977%253A34%252B%253C40%253A%253AAID-JCB9%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26aulast%3DOrian%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DA.%2BL.%26atitle%3DThe%2520Ubiquitin-Mediated%2520Proteolytic%2520Pathway%253A%2520Mode%2520of%2520Action%2520and%2520Clinical%2520Implications%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2000%26volume%3D77%26issue%3DSuppl.%252034%26spage%3D40%26epage%3D51%26doi%3D10.1002%2F%28SICI%291097-4644%282000%2977%3A34%2B%3C40%3A%3AAID-JCB9%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">The Proteasome in Health and Disease</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1028</span>, <span class="refDoi"> DOI: 10.2174/138161281906141012124358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.2174%2F138161281906141012124358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23016682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFeqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1010-1028&author=E.+Jankowskaauthor=J.+Stojauthor=P.+Karpowiczauthor=P.+A.+Osmulskiauthor=M.+Gaczynska&title=The+Proteasome+in+Health+and+Disease&doi=10.2174%2F138161281906141012124358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The proteasome in health and disease</span></div><div class="casAuthors">Jankowska, Elzbieta; Stoj, Julia; Karpowicz, Przemyslaw; Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1028</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The giant proteolytic factory called the proteasome came a long way from a biochem. curio to a major regulator of cellular physiol. and a renowned drug target within the ubiquitin proteasome pathway (UPP).  Thanks to availability of highly specific inhibitors of the proteasome, in less than twenty years it was possible to identify major transcription factors, cyclins, and products of oncogenes as crucial substrates for the UPP.  Nine years passed since the FDA speedily approved bortezomib, the inhibitor of proteasome, for treatment of multiple myeloma.  One year after its approval, the field was honored by awarding the Nobel Prize to Hershko, Ciechanover and Rose for introducing the concept of controlled proteolysis of ubiquitin-tagged substrates, with proteasome as the intracellular recycling facility.  Taking into consideration the universal involvement of the proteasome in the life of all cells in human body, it comes to no surprise that the enzyme is deeply implicated in etiol., progression, diagnosis or cure of multiple diseases.  Below we discuss some aspects of the involvement: from direct causative links to changes in proteasome properties that correlate with pathol. conditions.  We start with diseases collectively known as cancer, and with immune system-related pathologies.  Here, the proteasome inhibitors are either already used in clinics, or undergo advanced preclin. screening.  Then, we will continue with cardiovascular disorders, followed by aging.  Changes of the proteasome make-up during aging may be a priming factor for neurodegenerative diseases, described last.  We discuss the potential for proteasome regulation: inhibition, activation or specificity modulation, to successfully enter the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAvSvTc5q4wrVg90H21EOLACvtfcHk0lgX4R6EyV54Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFeqtrg%253D&md5=8b4a664827d4d79f4c9bf88212ec4160</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F138161281906141012124358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161281906141012124358%26sid%3Dliteratum%253Aachs%26aulast%3DJankowska%26aufirst%3DE.%26aulast%3DStoj%26aufirst%3DJ.%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DThe%2520Proteasome%2520in%2520Health%2520and%2520Disease%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D1010%26epage%3D1028%26doi%3D10.2174%2F138161281906141012124358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, A. E.</span></span> <span> </span><span class="NLM_article-title">Proteasome Inhibitors in Cancer Therapy</span>. <i>J. Cell Commun. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1007/s12079-011-0121-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2Fs12079-011-0121-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21484190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvosl2jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=101-110&issue=2&author=L.+J.+Crawfordauthor=B.+Walkerauthor=A.+E.+Irvine&title=Proteasome+Inhibitors+in+Cancer+Therapy&doi=10.1007%2Fs12079-011-0121-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors in cancer therapy</span></div><div class="casAuthors">Crawford Lisa J; Walker Brian; Irvine Alexandra E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cell communication and signaling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ubiquitin proteasome pathway plays a critical role in regulating many processes in the cell which are important for tumour cell growth and survival.  Inhibition of proteasome function has emerged as a powerful strategy for anti-cancer therapy.  Clinical validation of the proteasome as a therapeutic target was achieved with bortezomib and has prompted the development of a second generation of proteasome inhibitors with improved pharmacological properties.  This review summarises the main mechanisms of action of proteasome inhibitors in cancer, the development of proteasome inhibitors as therapeutic agents and the properties and progress of next generation proteasome inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFoYHRPE0q0okFb3mqPUYDfW6udTcc2eaw-eSmABYH7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvosl2jtQ%253D%253D&md5=5a3d94d5da47330ca700e3d61c1c9e14</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs12079-011-0121-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12079-011-0121-7%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DL.%2BJ.%26aulast%3DWalker%26aufirst%3DB.%26aulast%3DIrvine%26aufirst%3DA.%2BE.%26atitle%3DProteasome%2520Inhibitors%2520in%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Cell%2520Commun.%2520Signal.%26date%3D2011%26volume%3D5%26issue%3D2%26spage%3D101%26epage%3D110%26doi%3D10.1007%2Fs12079-011-0121-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, A. E.</span></span> <span> </span><span class="NLM_article-title">Targeting the Ubiquitin Proteasome System in Haematological Malignancies</span>. <i>Blood Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.blre.2013.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.blre.2013.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24183816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslahsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=297-304&issue=6&author=L.+J.+Crawfordauthor=A.+E.+Irvine&title=Targeting+the+Ubiquitin+Proteasome+System+in+Haematological+Malignancies&doi=10.1016%2Fj.blre.2013.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the ubiquitin proteasome system in haematological malignancies</span></div><div class="casAuthors">Crawford, Lisa J.; Irvine, Alexandra E.</div><div class="citationInfo"><span class="NLM_cas:title">Blood Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">297-304</span>CODEN:
                <span class="NLM_cas:coden">BLOREB</span>;
        ISSN:<span class="NLM_cas:issn">0268-960X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin proteasome system (UPS) plays a central role in cellular protein homeostasis through the targeted destruction of damaged/misfolded proteins and regulatory proteins that control crit. cellular functions.  The UPS comprises a sequential series of enzymic activities to covalently attach ubiquitin to proteins to target them for degrdn. through the proteasome.  Aberrancies within this system have been assocd. with transformation and tumorigenesis and thus, the UPS represents an attractive target for the development of anti-cancer therapies.  The use of the first-in-class proteasome inhibitor, bortezomib, in the treatment of Plasma Cell Myeloma and Mantle Cell Lymphoma has validated the UPS as a therapeutic target.  Following on its success, efforts are focused on the development of second-generation proteasome inhibitors and small mol. inhibitors of other components of the UPS.  This review will provide an overview of the UPS and discuss current and novel therapies targeting the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSIPslSGGo1bVg90H21EOLACvtfcHk0ligQmORCBTHqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslahsLjM&md5=dc702c6f3e0e87c27955cfa11502bca5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.blre.2013.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.blre.2013.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DL.%2BJ.%26aulast%3DIrvine%26aufirst%3DA.%2BE.%26atitle%3DTargeting%2520the%2520Ubiquitin%2520Proteasome%2520System%2520in%2520Haematological%2520Malignancies%26jtitle%3DBlood%2520Rev.%26date%3D2013%26volume%3D27%26issue%3D6%26spage%3D297%26epage%3D304%26doi%3D10.1016%2Fj.blre.2013.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Muscle Wasting in Disease: Molecular Mechanisms and Promising Therapies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1038/nrd4467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fnrd4467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25549588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=58-74&issue=1&author=S.+Cohenauthor=J.+A.+Nathanauthor=A.+L.+Goldberg&title=Muscle+Wasting+in+Disease%3A+Molecular+Mechanisms+and+Promising+Therapies&doi=10.1038%2Fnrd4467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscle wasting in disease: molecular mechanisms and promising therapies</span></div><div class="casAuthors">Cohen, Shenhav; Nathan, James A.; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-74</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Atrophy occurs in specific muscles with inactivity (for example, during plaster cast immobilization) or denervation (for example, in patients with spinal cord injuries).  Muscle wasting occurs systemically in older people (a condition known as sarcopenia); as a physiol. response to fasting or malnutrition; and in many diseases, including chronic obstructive pulmonary disorder, cancer-assocd. cachexia, diabetes, renal failure, cardiac failure, Cushing syndrome, sepsis, burns and trauma.  The rapid loss of muscle mass and strength primarily results from excessive protein breakdown, which is often accompanied by reduced protein synthesis.  This loss of muscle function can lead to reduced quality of life, increased morbidity and mortality.  Exercise is the only accepted approach to prevent or slow atrophy.  However, several promising therapeutic agents are in development, and major advances in our understanding of the cellular mechanisms that regulate the protein balance in muscle include the identification of several cytokines, particularly myostatin, and a common transcriptional program that promotes muscle wasting.  Here, we discuss these new insights and the rationally designed therapies that are emerging to combat muscle wasting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwPC0gdbm5bVg90H21EOLACvtfcHk0ligQmORCBTHqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSg&md5=3c66223fa688330f2d50812548df5396</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4467%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DNathan%26aufirst%3DJ.%2BA.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DMuscle%2520Wasting%2520in%2520Disease%253A%2520Molecular%2520Mechanisms%2520and%2520Promising%2520Therapies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26issue%3D1%26spage%3D58%26epage%3D74%26doi%3D10.1038%2Fnrd4467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">Proteasomal Dysfunction in Aging and Huntington Disease</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2010.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.nbd.2010.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21145396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFCktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=4-8&issue=1&author=X.-J.+Liauthor=S.+Li&title=Proteasomal+Dysfunction+in+Aging+and+Huntington+Disease&doi=10.1016%2Fj.nbd.2010.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasomal dysfunction in aging and Huntington disease</span></div><div class="casAuthors">Li, Xiao-Jiang; Li, Shihua</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Disease</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-8</span>CODEN:
                <span class="NLM_cas:coden">NUDIEM</span>;
        ISSN:<span class="NLM_cas:issn">0969-9961</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein degrdn. plays a central role in many cellular functions.  Misfolded and damaged proteins are removed from the cells to avoid toxicity.  Eukaryotic cells have two main routes for clearing misfolded or toxic proteins: the ubiquitin-proteasome and autophagy-lysosome pathways.  The ubiquitin-proteasome system (UPS) is ubiquitously present in the cytoplasm, nucleus, and various subcellular regions whereas autophagy predominantly functions in the cytoplasm.  The activity of the UPS often remains at a high level, whereas basal autophagy constitutively occurs at low levels in cells for the performance of homeostatic functions.  Because of the presence of the UPS in the nucleus, the UPS function may be more important for clearing misfolded proteins in the nucleus.  Polyglutamine diseases, including Huntington disease (HD), show the age-dependent neurol. symptoms and the accumulation of misfolded proteins that are often found in the nucleus.  The selective neuropathol. in HD is also found to assoc. with the preferential accumulation of the disease protein huntington in neuronal cells.  Although it is clear that the UPS is important for clearing mutant huntingtin, it remains unclear whether aging or HD affects the capacity of neuronal UPS to remove toxic and misfolded proteins.  In this review, we focus on the relationship between the UPS function and aging as well as Huntington disease.  We also discuss findings that suggest that aging is a more important factor that can neg. impact the function of the UPS.  This article is part of a Special Issue entitled "Autophagy and protein degrdn. in neurol. diseases.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnnV8623Fm-7Vg90H21EOLACvtfcHk0ligQmORCBTHqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFCktb0%253D&md5=15ff156b2e745d0bc40080f588ef079d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2010.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2010.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-J.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DProteasomal%2520Dysfunction%2520in%2520Aging%2520and%2520Huntington%2520Disease%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2011%26volume%3D43%26issue%3D1%26spage%3D4%26epage%3D8%26doi%3D10.1016%2Fj.nbd.2010.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Dynamic Regulation of the Human 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">12991</span>– <span class="NLM_lpage">12996</span>, <span class="refDoi"> DOI: 10.1073/pnas.1614614113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.1614614113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27791164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslalsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=12991-12996&issue=46&author=S.+Chenauthor=J.+Wuauthor=Y.+Luauthor=Y.-B.+Maauthor=B.-H.+Leeauthor=Z.+Yuauthor=Q.+Ouyangauthor=D.+J.+Finleyauthor=M.+W.+Kirschnerauthor=Y.+Mao&title=Structural+Basis+for+Dynamic+Regulation+of+the+Human+26S+Proteasome&doi=10.1073%2Fpnas.1614614113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for dynamic regulation of the human 26S proteasome</span></div><div class="casAuthors">Chen, Shuobing; Wu, Jiayi; Lu, Ying; Ma, Yong-Bei; Lee, Byung-Hoon; Yu, Zhou; Ouyang, Qi; Finley, Daniel J.; Kirschner, Marc W.; Mao, Youdong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">12991-12996</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The proteasome is the major engine of protein degrdn. in all eukaryotic cells.  At the heart of this machine is a heterohexameric ring of AAA (ATPases assocd. with diverse cellular activities) proteins that unfolds ubiquitylated target proteins that are concurrently translocated into a proteolytic chamber and degraded into peptides.  Using cryoelectron microscopy, we detd. a near-at.-resoln. structure of the 2.5-MDa human proteasome in its ground state, as well as subnanometer-resoln. structures of the holoenzyme in three alternative conformational states.  The substrate-unfolding AAA-ATPase channel is narrowed by 10 inward-facing pore loops arranged into two helixes that run in parallel with each other, one hydrophobic in character and the other highly charged.  The gate of the core particle was unexpectedly found closed in the ground state and open in only one of the alternative states.  Coordinated, stepwise conformational changes of the regulatory particle couple ATP hydrolysis to substrate translocation and regulate gating of the core particle, leading to processive degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojT-6dAofYVbVg90H21EOLACvtfcHk0ligQmORCBTHqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslalsLfF&md5=1824587478f94283add68652b6d5a263</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614614113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614614113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.-B.%26aulast%3DLee%26aufirst%3DB.-H.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DOuyang%26aufirst%3DQ.%26aulast%3DFinley%26aufirst%3DD.%2BJ.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DMao%26aufirst%3DY.%26atitle%3DStructural%2520Basis%2520for%2520Dynamic%2520Regulation%2520of%2520the%2520Human%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D46%26spage%3D12991%26epage%3D12996%26doi%3D10.1073%2Fpnas.1614614113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jap, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwickl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the 20S Proteasome from the Archaeon T. Acidophilum at 3.4 Å Resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>268</i></span> (<span class="NLM_issue">5210</span>),  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">539</span>, <span class="refDoi"> DOI: 10.1126/science.7725097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1126%2Fscience.7725097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=7725097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADyaK2M3kt1Sqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1995&pages=533-539&issue=5210&author=J.+L%C3%B6weauthor=D.+Stockauthor=B.+Japauthor=P.+Zwicklauthor=W.+Baumeisterauthor=R.+Huber&title=Crystal+Structure+of+the+20S+Proteasome+from+the+Archaeon+T.+Acidophilum+at+3.4+%C3%85+Resolution&doi=10.1126%2Fscience.7725097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the 20S proteasome from the archaeon T. acidophilum at 3.4 A resolution</span></div><div class="casAuthors">Lowe J; Stock D; Jap B; Zwickl P; Baumeister W; Huber R</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">5210</span>),
    <span class="NLM_cas:pages">533-9</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The three-dimensional structure of the proteasome from the archaebacterium Thermoplasma acidophilum has been elucidated by x-ray crystallographic analysis by means of isomorphous replacement and cyclic averaging.  The atomic model was built and refined to a crystallographic R factor of 22.1 percent.  The 673-kilodalton protease complex consists of 14 copies of two different subunits, alpha and beta, forming a barrel-shaped structure of four stacked rings.  The two inner rings consist of seven beta subunits each, and the two outer rings consist of seven alpha subunits each.  A narrow channel controls access to the three inner compartments.  The alpha 7 beta 7 beta 7 alpha 7 subunit assembly has 72-point group symmetry.  The structures of the alpha and beta subunits are similar, consisting of a core of two antiparallel beta sheets that is flanked by alpha helices on both sides.  The binding of a peptide aldehyde inhibitor marks the active site in the central cavity at the amino termini of the beta subunits and suggests a novel proteolytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSE36ig5vun65w7prLlxLOzfW6udTcc2eafbeo_iLESArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M3kt1Sqtw%253D%253D&md5=c09bba67da07e6addad1fb8d49a04b20</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.7725097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7725097%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DStock%26aufirst%3DD.%26aulast%3DJap%26aufirst%3DB.%26aulast%3DZwickl%26aufirst%3DP.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520the%252020S%2520Proteasome%2520from%2520the%2520Archaeon%2520T.%2520Acidophilum%2520at%25203.4%2520%25C3%2585%2520Resolution%26jtitle%3DScience%26date%3D1995%26volume%3D268%26issue%3D5210%26spage%3D533%26epage%3D539%26doi%3D10.1126%2Fscience.7725097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ditzel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bochtler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartunik, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span> <span> </span><span class="NLM_article-title">Structure of 20S Proteasome from Yeast at 2.4Å Resolution</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">6624</span>),  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1038/386463a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F386463a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9087403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=1997&pages=463-471&issue=6624&author=M.+Grollauthor=L.+Ditzelauthor=J.+L%C3%B6weauthor=D.+Stockauthor=M.+Bochtlerauthor=H.+D.+Bartunikauthor=R.+Huber&title=Structure+of+20S+Proteasome+from+Yeast+at+2.4%C3%85+Resolution&doi=10.1038%2F386463a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of 20S proteasome from yeast at 2.4 Å resolution</span></div><div class="casAuthors">Groll, Michael; Ditzel, Lars; Loewe, Jan; Stock, Daniela; Bochtler, Matthias; Bartunik, Hans D.; Huber, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">6624</span>),
    <span class="NLM_cas:pages">463-471</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of the 20S proteasome from the yeast Saccharomyces cerevisiae shows that its 28 protein subunits are arranged as an (α1...α7,β1...β7)2 complex in four stacked rings and occupy unique locations.  The interior of the particle, which harbors the active sites, is only accessible by some very narrow side entrances.  The β-type subunits are synthesized as proproteins before being proteolytically processed for assembly into the particle.  The pro forms of three of the seven different β-type subunits, β1-PRE3, β2/PUP1 and β5/PRE2, are cleaved between the threonine at position 1 and the last glycine of the pro-sequence, with release of the active-site residue Thr 1.  These three β-type subunits have inhibitor-binding sites, indicating that PRE2 has a chymotrypsin-like and a trypsin-like activity and that PRE3 has peptidylglutamyl peptide hydrolytic specificity.  Other β-type subunits are processed to an intermediate form, indicating that an addnl. nonspecific endopeptidase activity may exist which is important for peptide hydrolysis and for the generation of ligands for class I mols. of the major histocompatibility complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUP7-BUWwo-bVg90H21EOLACvtfcHk0lhQuvJt0RVyig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSqsr0%253D&md5=1796bb267e45cc5cdd2c97ecb426f39b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F386463a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F386463a0%26sid%3Dliteratum%253Aachs%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DDitzel%26aufirst%3DL.%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DStock%26aufirst%3DD.%26aulast%3DBochtler%26aufirst%3DM.%26aulast%3DBartunik%26aufirst%3DH.%2BD.%26aulast%3DHuber%26aufirst%3DR.%26atitle%3DStructure%2520of%252020S%2520Proteasome%2520from%2520Yeast%2520at%25202.4%25C3%2585%2520Resolution%26jtitle%3DNature%26date%3D1997%26volume%3D386%26issue%3D6624%26spage%3D463%26epage%3D471%26doi%3D10.1038%2F386463a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomisugi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukihara, T.</span></span> <span> </span><span class="NLM_article-title">The Structure of the Mammalian 20S Proteasome at 2.75 Å Resolution</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(02)00748-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2FS0969-2126%2802%2900748-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=12015144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvVWjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=609-618&issue=5&author=M.+Unnoauthor=T.+Mizushimaauthor=Y.+Morimotoauthor=Y.+Tomisugiauthor=K.+Tanakaauthor=N.+Yasuokaauthor=T.+Tsukihara&title=The+Structure+of+the+Mammalian+20S+Proteasome+at+2.75+%C3%85+Resolution&doi=10.1016%2FS0969-2126%2802%2900748-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of the mammalian 20S proteasome at 2.75 Å resolution</span></div><div class="casAuthors">Unno, Masaki; Mizushima, Tsunehiro; Morimoto, Yukio; Tomisugi, Yoshikazu; Tanaka, Keiji; Yasuoka, Noritake; Tsukihara, Tomitake</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">609-618</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The 20S proteasome is the catalytic portion of the 26S proteasome.  Constitutively expressed mammalian 20S proteasomes have 3 active subunits, β1, β2, and β5, which are replaced in the immunoproteasome by interferon-γ-inducible subunits β1i, β2i, and β5i, resp.  Here, the authors detd. the crystal structure of the bovine 20S proteasome at 2.75 Å resoln.  The structures of the α2, β1, β5, β6, and β7 subunits of the bovine enzyme were different from the yeast enzyme but enabled the bovine proteasome to accommodate either the constitutive or the inducible subunits.  A novel N-terminal nucleophile hydrolase activity was proposed for the β7 subunit.  The authors also detd. the site of the nuclear localization signals in the mol.  A model of the immunoproteasome was predicted from this constitutive structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVIKTjVKtIXbVg90H21EOLACvtfcHk0lhQuvJt0RVyig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvVWjtLw%253D&md5=03c3997a62eae5c5fa791e791f3ccd2c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900748-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900748-7%26sid%3Dliteratum%253Aachs%26aulast%3DUnno%26aufirst%3DM.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DMorimoto%26aufirst%3DY.%26aulast%3DTomisugi%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYasuoka%26aufirst%3DN.%26aulast%3DTsukihara%26aufirst%3DT.%26atitle%3DThe%2520Structure%2520of%2520the%2520Mammalian%252020S%2520Proteasome%2520at%25202.75%2520%25C3%2585%2520Resolution%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26issue%3D5%26spage%3D609%26epage%3D618%26doi%3D10.1016%2FS0969-2126%2802%2900748-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambour, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amalric, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsarrat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlet-Schiltz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet-Dubouch, M.-P.</span></span> <span> </span><span class="NLM_article-title">Subcellular Distribution and Dynamics of Active Proteasome Complexes Unraveled by a Workflow Combining in Vivo Complex Cross-Linking and Quantitative Proteomics</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1074/mcp.M112.023317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fmcp.M112.023317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23242550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlOitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=687-699&issue=3&author=B.+Fabreauthor=T.+Lambourauthor=J.+Delobelauthor=F.+Amalricauthor=B.+Monsarratauthor=O.+Burlet-Schiltzauthor=M.-P.+Bousquet-Dubouch&title=Subcellular+Distribution+and+Dynamics+of+Active+Proteasome+Complexes+Unraveled+by+a+Workflow+Combining+in+Vivo+Complex+Cross-Linking+and+Quantitative+Proteomics&doi=10.1074%2Fmcp.M112.023317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Subcellular Distribution and Dynamics of Active Proteasome Complexes Unraveled by a Workflow Combining in Vivo Complex Cross-Linking and Quantitative Proteomics</span></div><div class="casAuthors">Fabre, Bertrand; Lambour, Thomas; Delobel, Julien; Amalric, Francois; Monsarrat, Bernard; Burlet-Schiltz, Odile; Bousquet-Dubouch, Marie-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Through protein degrdn., the proteasome plays fundamental roles in different cell compartments.  Although the compn. of the 20S catalytic core particle (CP) has been well documented, little is known about the compn. and dynamics of the regulatory complexes that play a crucial role in its activity, or about how they assoc. with the CP in different cell compartments, different cell lines, and in response to external stimuli.  Because of difficulties performing acceptable cell fractionation while maintaining complex integrity, it has been challenging to characterize proteasome complexes by proteomic approaches.  Here, we report an integrated protocol, combining a crosslinking procedure on intact cells with cell fractionation, proteasome immuno-purifn., and robust label-free quant. proteomic anal. by mass spectrometry to det. the distribution and dynamics of cellular proteasome complexes in leukemic cells.  Activity profiles of proteasomes were correlated fully with the compn. of protein complexes and stoichiometry.  Moreover, our results suggest that, at the subcellular level, proteasome function is regulated by dynamic interactions between the 20S CP and its regulatory proteins-which modulate proteasome activity, stability, localization, or substrate uptake-rather than by profound changes in 20S CP compn.  Proteasome plasticity was obsd. both in the 20S CP and in its network of interactions following IFNγ stimulation.  The fractionation protocol also revealed specific proteolytic activities and structural features of low-abundance microsomal proteasomes from U937 and KG1a cells.  These could be linked to their important roles in the endoplasmic reticulum assocd. degrdn. pathway in leukemic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopH0HunTPDfLVg90H21EOLACvtfcHk0lgSMsf7Ptlw8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlOitL4%253D&md5=93d90b61268072123e712a8861bde7c8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M112.023317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M112.023317%26sid%3Dliteratum%253Aachs%26aulast%3DFabre%26aufirst%3DB.%26aulast%3DLambour%26aufirst%3DT.%26aulast%3DDelobel%26aufirst%3DJ.%26aulast%3DAmalric%26aufirst%3DF.%26aulast%3DMonsarrat%26aufirst%3DB.%26aulast%3DBurlet-Schiltz%26aufirst%3DO.%26aulast%3DBousquet-Dubouch%26aufirst%3DM.-P.%26atitle%3DSubcellular%2520Distribution%2520and%2520Dynamics%2520of%2520Active%2520Proteasome%2520Complexes%2520Unraveled%2520by%2520a%2520Workflow%2520Combining%2520in%2520Vivo%2520Complex%2520Cross-Linking%2520and%2520Quantitative%2520Proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2013%26volume%3D12%26issue%3D3%26spage%3D687%26epage%3D699%26doi%3D10.1074%2Fmcp.M112.023317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambour, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrigues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducoux-Petit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amalric, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsarrat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlet-Schiltz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet-Dubouch, M. P.</span></span> <span> </span><span class="NLM_article-title">Label-Free Quantitative Proteomics Reveals the Dynamics of Proteasome Complexes Composition and Stoichiometry in a Wide Range of Human Cell Lines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3027</span>– <span class="NLM_lpage">3037</span>, <span class="refDoi"> DOI: 10.1021/pr500193k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr500193k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFeltbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3027-3037&issue=6&author=B.+Fabreauthor=T.+Lambourauthor=L.+Garriguesauthor=M.+Ducoux-Petitauthor=F.+Amalricauthor=B.+Monsarratauthor=O.+Burlet-Schiltzauthor=M.+P.+Bousquet-Dubouch&title=Label-Free+Quantitative+Proteomics+Reveals+the+Dynamics+of+Proteasome+Complexes+Composition+and+Stoichiometry+in+a+Wide+Range+of+Human+Cell+Lines&doi=10.1021%2Fpr500193k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Label-Free Quantitative Proteomics Reveals the Dynamics of Proteasome Complexes Composition and Stoichiometry in a Wide Range of Human Cell Lines</span></div><div class="casAuthors">Fabre, Bertrand; Lambour, Thomas; Garrigues, Luc; Ducoux-Petit, Manuelle; Amalric, Francois; Monsarrat, Bernard; Burlet-Schiltz, Odile; Bousquet-Dubouch, Marie-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3027-3037</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is the main proteolytic system involved in intracellular proteins homeostasis in eukaryotes.  Although the structure of proteasome complexes has been well characterized, the distribution of its activators and assocd. proteins are less studied.  Here, we det. the compn. and the stoichiometry of proteasome complexes and their assocd. proteins in a wide range of human cell lines using a one-step affinity purifn. method and a label-free quant. proteomic approach.  We show that proteasome complexes are highly dynamic protein assemblies, the activity of which being regulated at different levels by variations in the stoichiometry of bound regulators, in the compn. of catalytic subunits and assocd. proteins, and in the rate of the 20S catalytic core complex assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoabYGVXnBoT7Vg90H21EOLACvtfcHk0lgSMsf7Ptlw8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFeltbw%253D&md5=39f6ec1acbf38f21e3bb2dab145514ae</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fpr500193k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr500193k%26sid%3Dliteratum%253Aachs%26aulast%3DFabre%26aufirst%3DB.%26aulast%3DLambour%26aufirst%3DT.%26aulast%3DGarrigues%26aufirst%3DL.%26aulast%3DDucoux-Petit%26aufirst%3DM.%26aulast%3DAmalric%26aufirst%3DF.%26aulast%3DMonsarrat%26aufirst%3DB.%26aulast%3DBurlet-Schiltz%26aufirst%3DO.%26aulast%3DBousquet-Dubouch%26aufirst%3DM.%2BP.%26atitle%3DLabel-Free%2520Quantitative%2520Proteomics%2520Reveals%2520the%2520Dynamics%2520of%2520Proteasome%2520Complexes%2520Composition%2520and%2520Stoichiometry%2520in%2520a%2520Wide%2520Range%2520of%2520Human%2520Cell%2520Lines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2014%26volume%3D13%26issue%3D6%26spage%3D3027%26epage%3D3037%26doi%3D10.1021%2Fpr500193k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadre-Bazzaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Formosa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structure of a Blm10 Complex Reveals Common Mechanisms for Proteasome Binding and Gate Opening</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">728</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2010.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20227375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=728-735&issue=5&author=K.+Sadre-Bazzazauthor=F.+G.+Whitbyauthor=H.+Robinsonauthor=T.+Formosaauthor=C.+P.+Hill&title=Structure+of+a+Blm10+Complex+Reveals+Common+Mechanisms+for+Proteasome+Binding+and+Gate+Opening&doi=10.1016%2Fj.molcel.2010.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a Blm10 complex reveals common mechanisms for proteasome binding and gate opening</span></div><div class="casAuthors">Sadre-Bazzaz, Kianoush; Whitby, Frank G.; Robinson, Howard; Formosa, Tim; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">728-735</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The proteasome is an abundant protease that is critically important for numerous cellular pathways.  Proteasomes are activated in vitro by three known classes of proteins/complexes, including Blm10/PA200.  Here, we report a 3.4 Å resoln. crystal structure of a proteasome-Blm10 complex, which reveals that Blm10 surrounds the proteasome entry pore in the 1.2 MDa complex to form a largely closed dome that is expected to restrict access of potential substrates.  This architecture and the observation that Blm10 induces a disordered proteasome gate structure challenge the assumption that Blm10 functions as an activator of proteolysis in vivo.  The Blm10 C terminus binds in the same manner as seen for 11S activators and inferred for 19S/PAN activators and indicates a unified model for gate opening.  We also demonstrate that Blm10 acts to maintain mitochondrial function.  Consistent with the structural data, the C-terminal residues of Blm10 are needed for this activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp17uYMdi2WCLVg90H21EOLACvtfcHk0lgSMsf7Ptlw8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KrsLc%253D&md5=0215e288cb4a9fef177d3da3e3f2a94d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DSadre-Bazzaz%26aufirst%3DK.%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DFormosa%26aufirst%3DT.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructure%2520of%2520a%2520Blm10%2520Complex%2520Reveals%2520Common%2520Mechanisms%2520for%2520Proteasome%2520Binding%2520and%2520Gate%2520Opening%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D37%26issue%3D5%26spage%3D728%26epage%3D735%26doi%3D10.1016%2Fj.molcel.2010.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowlton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Activation of 20S Proteasomes by 11S Regulators</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>408</i></span> (<span class="NLM_issue">6808</span>),  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/35040607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F35040607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=11081519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVaksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2000&pages=115-120&issue=6808&author=F.+G.+Whitbyauthor=E.+I.+Mastersauthor=L.+Kramerauthor=J.+R.+Knowltonauthor=Y.+Yaoauthor=C.+C.+Wangauthor=C.+P.+Hill&title=Structural+Basis+for+the+Activation+of+20S+Proteasomes+by+11S+Regulators&doi=10.1038%2F35040607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the activation of 20S proteasomes by 11S regulators</span></div><div class="casAuthors">Whitby, Frank G.; Masters, Eugene I.; Kramer, Larissa; Knowlton, J. Randolph; Yao, Yi; Wang, Ching C.; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">6808</span>),
    <span class="NLM_cas:pages">115-120</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most of the non-lysosomal proteolysis that occurs in eukaryotic cells is performed by a nonspecific and abundant barrel-shaped complex called the 20S proteasome.  Substrates access the active sites, which are sequestered in an internal chamber, by traversing a narrow opening (α-annulus) that is blocked in the unliganded 20S proteasome by amino-terminal sequences of α-subunits.  Peptide products probably exit the 20S proteasome through the same opening.  11S regulators (also called PA26 (ref. 4), PA28 (ref. 5) and REG) are heptamers that stimulate 20S proteasome peptidase activity in vitro and may facilitate product release in vivo.  Here we report the co-crystal structure of yeast 20S proteasome with the 11S regulator from Trypanosoma brucei (PA26).  PA26 carboxy-terminal tails provide binding affinity by inserting into pockets on the 20S proteasome, and PA26 activation loops induce conformational changes in α-subunits that open the gate sepg. the proteasome interior from the intracellular environment.  The redn. in processivity expected for an open conformation of the exit gate may explain the role of 11S regulators in the prodn. of ligands for major histocompatibility complex class I mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_QJYjTvf0abVg90H21EOLACvtfcHk0liNI67RcMmi6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVaksb4%253D&md5=f3ff3f296da463f4bc8f4c61cf5b8435</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F35040607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35040607%26sid%3Dliteratum%253Aachs%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DMasters%26aufirst%3DE.%2BI.%26aulast%3DKramer%26aufirst%3DL.%26aulast%3DKnowlton%26aufirst%3DJ.%2BR.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Activation%2520of%252020S%2520Proteasomes%2520by%252011S%2520Regulators%26jtitle%3DNature%26date%3D2000%26volume%3D408%26issue%3D6808%26spage%3D115%26epage%3D120%26doi%3D10.1038%2F35040607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baugh, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viktorova, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilipenko, E. V.</span></span> <span> </span><span class="NLM_article-title">Proteasomes Can Degrade a Significant Proportion of Cellular Proteins Independent of Ubiquitination</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">827</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.12.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.jmb.2008.12.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19162040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitVCqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=814-827&issue=3&author=J.+M.+Baughauthor=E.+G.+Viktorovaauthor=E.+V.+Pilipenko&title=Proteasomes+Can+Degrade+a+Significant+Proportion+of+Cellular+Proteins+Independent+of+Ubiquitination&doi=10.1016%2Fj.jmb.2008.12.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasomes Can Degrade a Significant Proportion of Cellular Proteins Independent of Ubiquitination</span></div><div class="casAuthors">Baugh, James M.; Viktorova, Ekaterina G.; Pilipenko, Evgeny V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">814-827</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The crit. role of the ubiquitin-26S proteasome system in regulation of protein homeostasis in eukaryotes is well established.  In contrast, the impact of the ubiquitin-independent proteolytic activity of proteasomes is poorly understood.  Through biochem. anal. of mammalian lysates, we find that the 20S proteasome, latent in peptide hydrolysis, specifically cleaves more than 20% of all cellular proteins.  Thirty intrinsic proteasome substrates (IPSs) were identified and in vitro studies of their processing revealed that cleavage occurs at disordered regions, generating stable products encompassing structured domains.  The mechanism of IPS recognition is remarkably well conserved in the eukaryotic kingdom, as mammalian and yeast 20S proteasomes exhibit the same target specificity.  Further, 26S proteasomes specifically recognize and cleave IPSs at similar sites, independent of ubiquitination, suggesting that disordered regions likely constitute the universal structural signal for IPS proteolysis by proteasomes.  Finally, we show that proteasomes contribute to physiol. regulation of IPS levels in living cells and the inactivation of ubiquitin-activating enzyme E1 does not prevent IPS degrdn.  Collectively, these findings suggest a significant contribution of the ubiquitin-independent proteasome degrdn. pathway to the regulation of protein homeostasis in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq20sAHAjVTGLVg90H21EOLACvtfcHk0liNI67RcMmi6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitVCqt7s%253D&md5=0a103389423bbcf9cdad614ac43776c9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.12.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.12.081%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DJ.%2BM.%26aulast%3DViktorova%26aufirst%3DE.%2BG.%26aulast%3DPilipenko%26aufirst%3DE.%2BV.%26atitle%3DProteasomes%2520Can%2520Degrade%2520a%2520Significant%2520Proportion%2520of%2520Cellular%2520Proteins%2520Independent%2520of%2520Ubiquitination%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D386%26issue%3D3%26spage%3D814%26epage%3D827%26doi%3D10.1016%2Fj.jmb.2008.12.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Lanzas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaño, J. G.</span></span> <span> </span><span class="NLM_article-title">Proteins Directly Interacting with Mammalian 20S Proteasomal Subunits and Ubiquitin-Independent Proteasomal Degradation</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.3390/biom4041140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.3390%2Fbiom4041140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25534281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvit12gsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1140-1154&issue=4&author=R.+S%C3%A1nchez-Lanzasauthor=J.+G.+Casta%C3%B1o&title=Proteins+Directly+Interacting+with+Mammalian+20S+Proteasomal+Subunits+and+Ubiquitin-Independent+Proteasomal+Degradation&doi=10.3390%2Fbiom4041140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins directly interacting with mammalian 20S proteasomal subunits and ubiquitin-independent proteasomal degradation</span></div><div class="casAuthors">Sanchez-Lanzas Raul; Castano Jose G</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1140-54</span>
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    </div><div class="casAbstract">The mammalian 20S proteasome is a heterodimeric cylindrical complex (α7β7β7α7), composed of four rings each composed of seven different α or β subunits with broad proteolytic activity.  We review the mammalian proteins shown to directly interact with specific 20S proteasomal subunits and those subjected to ubiquitin-independent proteasomal degradation (UIPD).  The published reports of proteins that interact with specific proteasomal subunits, and others found on interactome databases and those that are degraded by a UIPD mechanism, overlap by only a few protein members.  Therefore, systematic studies of the specificity of the interactions, the elucidation of the protein regions implicated in the interactions (that may or may not be followed by degradation) and competition experiments between proteins known to interact with the same proteasomal subunit, are needed.  Those studies should provide a coherent picture of the molecular mechanisms governing the interactions of cellular proteins with proteasomal subunits, and their relevance to cell proteostasis and cell functioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTolbzz8v6UYTAyhZbj3lKdfW6udTcc2eYT8G3m_0Gy1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvit12gsA%253D%253D&md5=a9bd42d8dcd99a40a922f436f6adf0bc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fbiom4041140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom4041140%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Lanzas%26aufirst%3DR.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DJ.%2BG.%26atitle%3DProteins%2520Directly%2520Interacting%2520with%2520Mammalian%252020S%2520Proteasomal%2520Subunits%2520and%2520Ubiquitin-Independent%2520Proteasomal%2520Degradation%26jtitle%3DBiomolecules%26date%3D2014%26volume%3D4%26issue%3D4%26spage%3D1140%26epage%3D1154%26doi%3D10.3390%2Fbiom4041140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raynes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomatto, L. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. J. A.</span></span> <span> </span><span class="NLM_article-title">Degradation of Oxidized Proteins by the Proteasome: Distinguishing between the 20S, 26S, and Immunoproteasome Proteolytic Pathways</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2016.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.mam.2016.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27155164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsVeqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2016&pages=41-55&author=R.+Raynesauthor=L.+C.+D.+Pomattoauthor=K.+J.+A.+Davies&title=Degradation+of+Oxidized+Proteins+by+the+Proteasome%3A+Distinguishing+between+the+20S%2C+26S%2C+and+Immunoproteasome+Proteolytic+Pathways&doi=10.1016%2Fj.mam.2016.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of oxidized proteins by the proteasome: Distinguishing between the 20S, 26S, and immunoproteasome proteolytic pathways</span></div><div class="casAuthors">Raynes, Rachel; Pomatto, Laura C. D.; Davies, Kelvin J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The proteasome is a ubiquitous and highly plastic multi-subunit protease with multi-catalytic activity that is conserved in all eukaryotes.  The most widely known function of the proteasome is protein degrdn. through the 26S ubiquitin-proteasome system, responsible for the vast majority of protein degrdn. during homeostasis.  However, the proteasome also plays an important role in adaptive immune responses and adaptation to oxidative stress.  The unbound 20S proteasome, the core common to all proteasome conformations, is the main protease responsible for degrading oxidized proteins.  During periods of acute stress, the 19S regulatory cap of the 26S proteasome disassocs. from the proteolytic core, allowing for immediate ATP/ubiquitin-independent protein degrdn. by the 20S proteasome.  Despite the abundance of unbound 20S proteasome compared to other proteasomal conformations, many publications fail to distinguish between the two proteolytic systems and often regard the 26S proteasome as the dominant protease.  Further confounding the issue are the differential roles these two proteolytic systems have in adaptation and aging.  In this review, we will summarize the increasing evidence that the 20S core proteasome constitutes the major conformation of the proteasome system and that it is far from a latent protease requiring activation by binding regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZCX5MieTHrVg90H21EOLACvtfcHk0lii7xr5RqqE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsVeqtL4%253D&md5=a347662fca5d811cda30294ca18f1f2a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2016.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2016.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DRaynes%26aufirst%3DR.%26aulast%3DPomatto%26aufirst%3DL.%2BC.%2BD.%26aulast%3DDavies%26aufirst%3DK.%2BJ.%2BA.%26atitle%3DDegradation%2520of%2520Oxidized%2520Proteins%2520by%2520the%2520Proteasome%253A%2520Distinguishing%2520between%2520the%252020S%252C%252026S%252C%2520and%2520Immunoproteasome%2520Proteolytic%2520Pathways%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2016%26volume%3D50%26spage%3D41%26epage%3D55%26doi%3D10.1016%2Fj.mam.2016.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, Y. T.</span></span> <span> </span><span class="NLM_article-title">Degradation of Misfolded Proteins in Neurodegenerative Diseases: Therapeutic Targets and Strategies</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e147</span>, <span class="refDoi"> DOI: 10.1038/emm.2014.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Femm.2014.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25766616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=e147&issue=3&author=A.+Ciechanoverauthor=Y.+T.+Kwon&title=Degradation+of+Misfolded+Proteins+in+Neurodegenerative+Diseases%3A+Therapeutic+Targets+and+Strategies&doi=10.1038%2Femm.2014.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of misfolded proteins in neurodegenerative diseases: therapeutic targets and strategies</span></div><div class="casAuthors">Ciechanover, Aaron; Kwon, Yong Tae</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e147</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Mammalian cells remove misfolded proteins using various proteolytic systems, including the ubiquitin (Ub)-proteasome system (UPS), chaperone mediated autophagy (CMA) and macroautophagy.  The majority of misfolded proteins are degraded by the UPS, in which Ub-conjugated substrates are deubiquitinated, unfolded and cleaved into small peptides when passing through the narrow chamber of the proteasome.  The substrates that expose a specific degrdn. signal, the KFERQ sequence motif, can be delivered to and degraded in lysosomes via the CMA.  Aggregation-prone substrates resistant to both the UPS and the CMA can be degraded by macroautophagy, in which cargoes are segregated into autophagosomes before degrdn. by lysosomal hydrolases.  Although most misfolded and aggregated proteins in the human proteome can be degraded by cellular protein quality control, some native and mutant proteins prone to aggregation into β-sheet-enriched oligomers are resistant to all known proteolytic pathways and can thus grow into inclusion bodies or extracellular plaques.  The accumulation of protease-resistant misfolded and aggregated proteins is a common mechanism underlying protein misfolding disorders, including neurodegenerative diseases such as Huntington's disease (HD), Alzheimer's disease (AD), Parkinson's disease (PD), prion diseases and Amyotrophic Lateral Sclerosis (ALS).  In this review, we provide an overview of the proteolytic pathways in neurons, with an emphasis on the UPS, CMA and macroautophagy, and discuss the role of protein quality control in the degrdn. of pathogenic proteins in neurodegenerative diseases.  Addnl., we examine existing putative therapeutic strategies to efficiently remove cytotoxic proteins from degenerating neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAcnolVc3nZ7Vg90H21EOLACvtfcHk0lii7xr5RqqE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFOhtL8%253D&md5=f4d7542fdd0e42b0d3e566585454261a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Femm.2014.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2014.117%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26aulast%3DKwon%26aufirst%3DY.%2BT.%26atitle%3DDegradation%2520of%2520Misfolded%2520Proteins%2520in%2520Neurodegenerative%2520Diseases%253A%2520Therapeutic%2520Targets%2520and%2520Strategies%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2015%26volume%3D47%26issue%3D3%26spage%3De147%26doi%3D10.1038%2Femm.2014.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilchez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillin, A.</span></span> <span> </span><span class="NLM_article-title">The Role of Protein Clearance Mechanisms in Organismal Ageing and Age-Related Diseases</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5659</span>, <span class="refDoi"> DOI: 10.1038/ncomms6659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fncomms6659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25482515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVentb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5659&author=D.+Vilchezauthor=I.+Saezauthor=A.+Dillin&title=The+Role+of+Protein+Clearance+Mechanisms+in+Organismal+Ageing+and+Age-Related+Diseases&doi=10.1038%2Fncomms6659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The role of protein clearance mechanisms in organismal ageing and age-related diseases</span></div><div class="casAuthors">Vilchez, David; Saez, Isabel; Dillin, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5659</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are subject to continuous and complex quality-control mechanisms, which ensure integrity of the proteome.  Vilchez et al. review how a demise in these processes, collectively referred to as proteostasis, is linked to organismal ageing and the development of age-assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0qgIqQnnmlbVg90H21EOLACvtfcHk0lghupME_gb2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVentb8%253D&md5=7371b6f29c2605c7e0bbdaffaf0aa270</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fncomms6659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6659%26sid%3Dliteratum%253Aachs%26aulast%3DVilchez%26aufirst%3DD.%26aulast%3DSaez%26aufirst%3DI.%26aulast%3DDillin%26aufirst%3DA.%26atitle%3DThe%2520Role%2520of%2520Protein%2520Clearance%2520Mechanisms%2520in%2520Organismal%2520Ageing%2520and%2520Age-Related%2520Diseases%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5659%26doi%3D10.1038%2Fncomms6659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruschak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slassi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span> <span> </span><span class="NLM_article-title">Novel Proteasome Inhibitors to Overcome Bortezomib Resistance</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1093%2Fjnci%2Fdjr160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21606441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVKqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=1007-1017&issue=13&author=A.+M.+Ruschakauthor=M.+Slassiauthor=L.+E.+Kayauthor=A.+D.+Schimmer&title=Novel+Proteasome+Inhibitors+to+Overcome+Bortezomib+Resistance&doi=10.1093%2Fjnci%2Fdjr160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Proteasome Inhibitors to Overcome Bortezomib Resistance</span></div><div class="casAuthors">Ruschak, Amy M.; Slassi, Malik; Kay, Lewis E.; Schimmer, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The proteasome is an intracellular enzyme complex that degrades ubiquitin-tagged proteins and thereby regulates protein levels within the cell.  Given this important role in maintaining cellular homeostasis, it is perhaps somewhat surprising that proteasome inhibitors have a therapeutic window.  Proteasome inhibitors have demonstrated clin. efficacy in the treatment of multiple myeloma and mantle cell lymphoma and are under evaluation for the treatment of other malignancies.  Bortezomib is the first and only Food and Drug Administration-approved proteasome inhibitor that inhibits this enzyme complex in a reversible fashion.  Although bortezomib improves clin. outcomes when used as a single agent, most patients do not respond to this drug and those who do respond almost uniformly relapse.  As such, efforts are underway to develop proteasome inhibitors that act through mechanisms distinct from that of bortezomib.  Specifically, inhibitors that bind the active site of the proteasome and inhibit the complex irreversibly have been developed and are in advanced clin. trials.  Inhibitors that act on sites of the proteasome outside of the catalytic center have also been identified and are in preclin. development.  In this review, we discuss the structure and function of the proteasome.  We then focus on the mol. biol., chem., and the preclin. and clin. efficacy of novel proteasome inhibitors as strategies to inhibit this target and overcome some forms of bortezomib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWeUjDj5ezAbVg90H21EOLACvtfcHk0lghupME_gb2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVKqsrs%253D&md5=52b263bbfd39bf18a543a382610c762f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr160%26sid%3Dliteratum%253Aachs%26aulast%3DRuschak%26aufirst%3DA.%2BM.%26aulast%3DSlassi%26aufirst%3DM.%26aulast%3DKay%26aufirst%3DL.%2BE.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26atitle%3DNovel%2520Proteasome%2520Inhibitors%2520to%2520Overcome%2520Bortezomib%2520Resistance%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26issue%3D13%26spage%3D1007%26epage%3D1017%26doi%3D10.1093%2Fjnci%2Fdjr160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansdell, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dissanayake, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Substituted Quinolines as Noncovalent Proteasome Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2441</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bmc.2016.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27112450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslSjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=2441-2450&issue=11&author=T.+J.+McDanielauthor=T.+A.+Lansdellauthor=A.+A.+Dissanayakeauthor=L.+M.+Azevedoauthor=J.+Claesauthor=A.+L.+Odomauthor=J.+J.+Tepe&title=Substituted+Quinolines+as+Noncovalent+Proteasome+Inhibitors&doi=10.1016%2Fj.bmc.2016.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Substituted quinolines as noncovalent proteasome inhibitors</span></div><div class="casAuthors">McDaniel, Tanner J.; Lansdell, Theresa A.; Dissanayake, Amila A.; Azevedo, Lauren M.; Claes, Jacob; Odom, Aaron L.; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2441-2450</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Screening of a library of diverse heterocyclic scaffolds identified substituted quinolines as inhibitors of the human proteasome.  The heterocyclic library was prepd. via a novel titanium-catalyzed multicomponent coupling reaction, which rendered a diverse set of isoxazoles, pyrimidines, pyrroles, pyrazoles and quinolines.  SAR of the parent lead compd. indicated that hydrophobic residues on the benzo-moiety significantly improved potency.  Lead compd. 3-cyclohexenyl-2-methyl-6-piperidinylquinoline (25) inhibits the chymotryptic-like proteolytic activity of the proteasome (IC50 5.4 μM), representing a new class of nonpeptidic, noncovalent proteasome inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc8639UcfzmLVg90H21EOLACvtfcHk0lghupME_gb2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslSjtrg%253D&md5=4479d7a26d98b091aaf13428b4590843</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DT.%2BJ.%26aulast%3DLansdell%26aufirst%3DT.%2BA.%26aulast%3DDissanayake%26aufirst%3DA.%2BA.%26aulast%3DAzevedo%26aufirst%3DL.%2BM.%26aulast%3DClaes%26aufirst%3DJ.%26aulast%3DOdom%26aufirst%3DA.%2BL.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSubstituted%2520Quinolines%2520as%2520Noncovalent%2520Proteasome%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26issue%3D11%26spage%3D2441%26epage%3D2450%26doi%3D10.1016%2Fj.bmc.2016.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansdell, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurchla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hovde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weilbaecher, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Noncompetitive Modulation of the Proteasome by Imidazoline Scaffolds Overcomes Bortezomib Resistance and Delays MM Tumor Growth in Vivo</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1021/cb300568r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300568r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKhsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=578-587&issue=3&author=T.+A.+Lansdellauthor=M.+A.+Hurchlaauthor=J.+Xiangauthor=S.+Hovdeauthor=K.+N.+Weilbaecherauthor=R.+W.+Henryauthor=J.+J.+Tepe&title=Noncompetitive+Modulation+of+the+Proteasome+by+Imidazoline+Scaffolds+Overcomes+Bortezomib+Resistance+and+Delays+MM+Tumor+Growth+in+Vivo&doi=10.1021%2Fcb300568r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Noncompetitive Modulation of the Proteasome by Imidazoline Scaffolds Overcomes Bortezomib Resistance and Delays MM Tumor Growth in Vivo</span></div><div class="casAuthors">Lansdell, Theresa A.; Hurchla, Michelle A.; Xiang, Jingyu; Hovde, Stacy; Weilbaecher, Katherine N.; Henry, R. William; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-587</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) is a malignant disorder of differentiated B-cells for which std. care involves the inhibition of the proteasome.  All clin. used proteasome inhibitors, including the chemotherapeutic drug bortezomib, target the catalytic active sites of the proteasome and inhibit protein proteolysis by competing with substrate binding.  However, nearly all (∼97%) patients become intolerant or resistant to treatments within a few years, after which the av. survival time is less than 1 yr.  We describe herein the inhibition of the human proteasome via a noncompetitive mechanism by the imidazoline scaffold, TCH-13.  Consistent with a mechanism distinct from that of competitive inhibitors, TCH-013 acts additively with and overcomes resistance to bortezomib.  Importantly, TCH-013 induces apoptosis in a panel of myeloma and leukemia cell lines, but in contrast, normal lymphocytes, primary bone marrow stromal cells (hBMSC), and macrophages are resistant to its cytotoxic effects.  TCH-013 was equally effective in blocking MM cell growth in co-cultures of MM cells with hBMSC isolated from CD138 neg. bone marrow (BM) samples of MM patients.  The cellular activity translated well in vivo where TCH-013 delayed tumor growth in an MM xenograft model to a similar extent as bortezomib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ovzBZEXwn7Vg90H21EOLACvtfcHk0ljlIoSy3vkSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKhsbbK&md5=70093e771713e6ee6a15f2291997b5a7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb300568r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300568r%26sid%3Dliteratum%253Aachs%26aulast%3DLansdell%26aufirst%3DT.%2BA.%26aulast%3DHurchla%26aufirst%3DM.%2BA.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DHovde%26aufirst%3DS.%26aulast%3DWeilbaecher%26aufirst%3DK.%2BN.%26aulast%3DHenry%26aufirst%3DR.%2BW.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DNoncompetitive%2520Modulation%2520of%2520the%2520Proteasome%2520by%2520Imidazoline%2520Scaffolds%2520Overcomes%2520Bortezomib%2520Resistance%2520and%2520Delays%2520MM%2520Tumor%2520Growth%2520in%2520Vivo%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D3%26spage%3D578%26epage%3D587%26doi%3D10.1021%2Fcb300568r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njomen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjögren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Enhancement of 20S Proteasome Activity Targets Intrinsically Disordered Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2240</span>– <span class="NLM_lpage">2247</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2240-2247&issue=9&author=C.+L.+Jonesauthor=E.+Njomenauthor=B.+Sj%C3%B6grenauthor=T.+S.+Dexheimerauthor=J.+J.+Tepe&title=Small+Molecule+Enhancement+of+20S+Proteasome+Activity+Targets+Intrinsically+Disordered+Proteins&doi=10.1021%2Facschembio.7b00489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Enhancement of 20S Proteasome Activity Targets Intrinsically Disordered Proteins</span></div><div class="casAuthors">Jones, Corey L.; Njomen, Evert; Sjogren, Benita; Dexheimer, Thomas S.; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2240-2247</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome is the main protease for the degrdn. of oxidatively damaged and intrinsically disordered proteins.  When accumulation of disordered or oxidatively damaged proteins exceeds proper clearance in neurons, imbalanced pathway signaling or aggregation occurs, which have been implicated in the pathogenesis of several neurol. disorders.  Screening of the NIH Clin. Collection and Prestwick libraries identified the neuroleptic agent chlorpromazine as a lead agent capable of enhancing 20S proteasome activity.  Chem. manipulation of chlorpromazine abrogated its D2R receptor binding affinity while retaining its ability to enhance 20S mediated proteolysis at low micromolar concns.  The resulting small mol. enhancers of 20S proteasome activity induced the degrdn. of intrinsically disordered proteins, α-synuclein, and tau but not structured proteins.  These small mol. 20S agonists can serve as leads to explore the therapeutic potential of 20S activation or as new tools to provide insight into the yet unclear mechanics of 20S-gate regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtP6cycR_6KbVg90H21EOLACvtfcHk0ljlIoSy3vkSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyhu7jI&md5=52f7dcf5de303a4e8b992af6ed590171</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00489%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DC.%2BL.%26aulast%3DNjomen%26aufirst%3DE.%26aulast%3DSj%25C3%25B6gren%26aufirst%3DB.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSmall%2520Molecule%2520Enhancement%2520of%252020S%2520Proteasome%2520Activity%2520Targets%2520Intrinsically%2520Disordered%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3D2240%26epage%3D2247%26doi%3D10.1021%2Facschembio.7b00489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Rapamycin Allosterically Inhibits the Proteasome</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1124/mol.112.083873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1124%2Fmol.112.083873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=23619386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=104-113&issue=1&author=P.+A.+Osmulskiauthor=M.+Gaczynska&title=Rapamycin+Allosterically+Inhibits+the+Proteasome&doi=10.1124%2Fmol.112.083873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin allosterically inhibits the proteasome</span></div><div class="casAuthors">Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">104-113</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Rapamycin is a canonical allosteric inhibitor of the mammalian tarpet of rapamycin (mTOR) kinase with immunosuppressive and proapoptotic activities.  We found that in vitro rapamycin also regulates the proteasome, which is an essential intracellular protease of the ubiquitin-proteasome pathway.  Rapamycin inhibits proteinase and selected peptidase activities of the catalytic core proteasome at low micromolar concns.  Moreover, the drug interferes with binding of the 19S cap essential for processing of polyubiquitinylated substrates and with the PA200 proteasome activator to the 20S catalytic core proteasome.  These protein complexes are known to bind to specific grooves on the α face region of the 20S core.  Treatment with rapamycin affects the conformational dynamics of the proteasomal gate, which is centrally positioned within the α face and allosterically regulated element responsible for the intake of substrates.  We showed that rapamycin shares all the proteasome targeting properties not only with other two-domain, closed-ring analogs (rapalogs) but also with its single domain mimics and seco-rapamycin, which is the first in vivo open-ring metabolite of rapamycin that does not affect mTOR.  We hypothesize that rapamycin and related compds. bind to the α face and allosterically impact proteasome function.  This article discusses the implications of our findings for the mechanism of in vivo actions of rapamycin and for the design of novel allosteric drugs targeting the proteasome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBS8ZCRQbJ7Vg90H21EOLACvtfcHk0ljlIoSy3vkSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjF&md5=503e8e747496f74ba6942608932449d5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.083873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.083873%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DRapamycin%2520Allosterically%2520Inhibits%2520the%2520Proteasome%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26issue%3D1%26spage%3D104%26epage%3D113%26doi%3D10.1124%2Fmol.112.083873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Njomen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepe, J. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Modulation of Proteasome Assembly</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">28</span>),  <span class="NLM_fpage">4214</span>– <span class="NLM_lpage">4224</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00579</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00579" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWju7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4214-4224&issue=28&author=E.+Njomenauthor=P.+A.+Osmulskiauthor=C.+L.+Jonesauthor=M.+Gaczynskaauthor=J.+J.+Tepe&title=Small+Molecule+Modulation+of+Proteasome+Assembly&doi=10.1021%2Facs.biochem.8b00579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modulation of Proteasome Assembly</span></div><div class="casAuthors">Njomen, Evert; Osmulski, Pawel A.; Jones, Corey L.; Gaczynska, Maria; Tepe, Jetze J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4214-4224</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome is the main protease that directly targets intrinsically disordered proteins (IDPs) for proteolytic degrdn.  Mutations, oxidative stress or aging can induce the build-up of IDPs resulting in incorrect signaling or aggregation, assocd. with the pathogenesis of many cancers and neurodegenerative diseases.  Drugs that facilitate 20S-mediated proteolysis have therefore many potential therapeutic applications.  We report herein the modulation of proteasome assembly by the small mole-cule TCH-165, resulting in an increase in 20S levels.  The increase of free 20S corresponds to enhanced proteolysis of IDPs, including α-synuclein, tau, ornithine decarboxylase and c-Fos, but not structured proteins.  Clearance of ubiquitinated protein was largely maintained by single capped proteasome complexes (19S-20S), but accumulation occurs when all 19S capped proteasome complexes are depleted.  This study illustrates the first example of a small mol. capable of targeting disordered proteins for degrdn. by regulating the dynamic equil. between different proteasome complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWB-LPHAN7wLVg90H21EOLACvtfcHk0lj-k1rIfumbvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWju7fJ&md5=8d8920859b36c1098de6452427b1baca</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00579%26sid%3Dliteratum%253Aachs%26aulast%3DNjomen%26aufirst%3DE.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DJones%26aufirst%3DC.%2BL.%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DTepe%26aufirst%3DJ.%2BJ.%26atitle%3DSmall%2520Molecule%2520Modulation%2520of%2520Proteasome%2520Assembly%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D28%26spage%3D4214%26epage%3D4224%26doi%3D10.1021%2Facs.biochem.8b00579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span> <span> </span><span class="NLM_article-title">Activation and Inhibition of Proteasomes by Betulinic Acid and Its Derivatives</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>581</i></span> (<span class="NLM_issue">25</span>),  <span class="NLM_fpage">4955</span>– <span class="NLM_lpage">4959</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2007.09.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.febslet.2007.09.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=17904555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGku7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2007&pages=4955-4959&issue=25&author=L.+Huangauthor=P.+Hoauthor=C.-H.+Chen&title=Activation+and+Inhibition+of+Proteasomes+by+Betulinic+Acid+and+Its+Derivatives&doi=10.1016%2Fj.febslet.2007.09.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Activation and inhibition of the proteasome by betulinic acid and its derivatives</span></div><div class="casAuthors">Huang, Li; Ho, Phong; Chen, Chin-Ho</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">581</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4955-4959</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This study discovered that betulinic acid (BA) is a potent proteasome activator that preferentially activates the chymotrypsin-like activity of the proteasome.  Chem. modifications can transform BA into proteasome inhibitors.  Chem. modifications at the C-3 position of BA resulted in compds., such as dimethylsuccinyl BA (DSB), with various inhibitory activities against the human 20S proteasome.  Interestingly, the proteasomal activation by BA and the inhibitory activity of DSB could be abrogated by introducing a side chain at the C-28 position.  In summary, this study discovered a class of small mols. that can either activate or inhibit human proteasome activity depending on side chain modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXCEaz3UPUE7Vg90H21EOLACvtfcHk0lj-k1rIfumbvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGku7nN&md5=031df2a57e72fefccc01a293e04e8bca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2007.09.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2007.09.031%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DC.-H.%26atitle%3DActivation%2520and%2520Inhibition%2520of%2520Proteasomes%2520by%2520Betulinic%2520Acid%2520and%2520Its%2520Derivatives%26jtitle%3DFEBS%2520Lett.%26date%3D2007%26volume%3D581%26issue%3D25%26spage%3D4955%26epage%3D4959%26doi%3D10.1016%2Fj.febslet.2007.09.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-E.</span></span> <span> </span><span class="NLM_article-title">Synthetic Peptide-Based Activators of the Proteasome</span>. <i>Mol. Biol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1023/A:1006851428691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1023%2FA%3A1006851428691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9228292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXks1amsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1997&pages=119-124&issue=1&author=S.+Wilkauthor=W.-E.+Chen&title=Synthetic+Peptide-Based+Activators+of+the+Proteasome&doi=10.1023%2FA%3A1006851428691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic peptide-based activators of the proteasome</span></div><div class="casAuthors">Wilk, Sherwin; Chen, Wei-Er</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology Reports</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">119-124</span>CODEN:
                <span class="NLM_cas:coden">MLBRBU</span>;
        ISSN:<span class="NLM_cas:issn">0301-4851</span>.
    
            (<span class="NLM_cas:orgname">Kluwer</span>)
        </div><div class="casAbstract">The development of small mol. peptide-based activators of the 20S proteasome or multicatalytic proteinase complex is initiated.  The enhancement of antigen presentation by transfection of the protein activator PA28α into a mouse fibroblast cell line supports the potential use of small mol. activators in stimulating the immune response.  Four classes of peptide-based activators were synthesized, i.e. peptidyl alcs., esters, p-nitroanilides and nitriles.  These compds. markedly and reversibly stimulated the hydrolysis of suc-LLVY-MCA, Z-LLE-NA and Z-GPALG-p-aminobenzoate as well as hydrolysis of the decapeptide angiotensin I.  Stimulation was due to a decrease in the Km and increase in the Vmax of the substrate.  In general, the EC50 for activation ranged from 50-150 mM and maximal stimulation varied from 3 to 15 fold depending on the activity measured.  Z-IE(O-tBu)AL-p-nitroanilide, a proteasome substrate, markedly stimulated the hydrolysis of Z-GPALG-pAB by binding to a saturable high affinity site distinct from its binding site as substrate.  Since all effective activators contain hydrophobic groups in positions P1-P5, low aq. soly. is a limitation of these compds.  Competition expts. suggest that these activators bind to the same site as PA28.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrijOvCsrxrqbVg90H21EOLACvtfcHk0lj-k1rIfumbvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXks1amsrY%253D&md5=4340498024abf6e450abfda061f02fc6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1006851428691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1006851428691%26sid%3Dliteratum%253Aachs%26aulast%3DWilk%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.-E.%26atitle%3DSynthetic%2520Peptide-Based%2520Activators%2520of%2520the%2520Proteasome%26jtitle%3DMol.%2520Biol.%2520Rep.%26date%3D1997%26volume%3D24%26issue%3D1%26spage%3D119%26epage%3D124%26doi%3D10.1023%2FA%3A1006851428691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics: Current Status and Future Directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&issue=1&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+Therapeutics%3A+Current+Status+and+Future+Directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0lj-k1rIfumbvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520Therapeutics%253A%2520Current%2520Status%2520and%2520Future%2520Directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D1%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">Docking of the Proteasomal ATPases’ Carboxyl Termini in the 20S Proteasome’s Ring Opens the Gate for Substrate Entry</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2007.06.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=17803938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=731-744&issue=5&author=D.+M.+Smithauthor=S.+C.+Changauthor=S.+Parkauthor=D.+Finleyauthor=Y.+Chengauthor=A.+L.+Goldberg&title=Docking+of+the+Proteasomal+ATPases%E2%80%99+Carboxyl+Termini+in+the+20S+Proteasome%E2%80%99s+Ring+Opens+the+Gate+for+Substrate+Entry&doi=10.1016%2Fj.molcel.2007.06.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Docking of the proteasomal ATPases' carboxyl termini in the 20S proteasome's α ring opens the gate for substrate entry</span></div><div class="casAuthors">Smith, David M.; Chang, Shih-Chung; Park, Soyeon; Finley, Daniel; Cheng, Yifan; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The 20S proteasome functions in protein degrdn. in eukaryotes together with the 19S ATPases or in archaea with the homologous PAN ATPase complex.  These ATPases contain a conserved C-terminal hydrophobic-tyrosine-X motif (HbYX).  We show that these residues are essential for PAN to assoc. with the 20S and open its gated channel for substrate entry.  Upon ATP binding, these C-terminal residues bind to pockets between the 20S's α subunits.  Seven-residue or longer peptides from PAN's C terminus contg. the HbYX motif also bind to these sites and induce gate opening in the 20S.  Gate opening could be induced by C-terminal peptides from the 19S ATPase subunits, Rpt2, and Rpt5, but not by ones from PA28/26, which lack the HbYX motif and cause gate opening by distinct mechanisms.  C-terminal residues in the 19S ATPases were also shown to be crit. for gating and stability of 26S proteasomes.  Thus, the C termini of the proteasomal ATPases function like a "key in a lock" to induce gate opening and allow substrate entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq37nJVOXWjPrVg90H21EOLACvtfcHk0ljLbmldfBOLXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7J&md5=5837c2dafce8e1ce8e80c67487f51f17</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.033%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DFinley%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DDocking%2520of%2520the%2520Proteasomal%2520ATPases%25E2%2580%2599%2520Carboxyl%2520Termini%2520in%2520the%252020S%2520Proteasome%25E2%2580%2599s%2520Ring%2520Opens%2520the%2520Gate%2520for%2520Substrate%2520Entry%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26issue%3D5%26spage%3D731%26epage%3D744%26doi%3D10.1016%2Fj.molcel.2007.06.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gizyńska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grudnik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giełdoń, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Low Molecular Weight Activator Blm-Pep with Yeast 20S Proteasome - Insights into the Enzyme Activation Mechanism</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">6177</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-05997-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2Fs41598-017-05997-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28733623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A280%3ADC%252BC1cfgtFentA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6177&issue=1&author=J.+Witkowskaauthor=M.+Gizy%C5%84skaauthor=P.+Grudnikauthor=P.+Golikauthor=P.+Karpowiczauthor=A.+Gie%C5%82do%C5%84author=G.+Dubinauthor=E.+Jankowska&title=Crystal+Structure+of+a+Low+Molecular+Weight+Activator+Blm-Pep+with+Yeast+20S+Proteasome+-+Insights+into+the+Enzyme+Activation+Mechanism&doi=10.1038%2Fs41598-017-05997-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a low molecular weight activator Blm-pep with yeast 20S proteasome - insights into the enzyme activation mechanism</span></div><div class="casAuthors">Witkowska Julia; Gizynska Malgorzata; Karpowicz Przemyslaw; Gieldon Artur; Jankowska Elzbieta; Grudnik Przemyslaw; Golik Przemyslaw; Dubin Grzegorz; Grudnik Przemyslaw; Golik Przemyslaw; Dubin Grzegorz</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6177</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteasomes are responsible for protein turnover in eukaryotic cells, degrading short-lived species but also removing improperly folded or oxidatively damaged ones.  Dysfunction of a proteasome results in gradual accumulation of misfolded/damaged proteins, leading to their aggregation.  It has been postulated that proteasome activators may facilitate removal of such aggregation-prone proteins and thus prevent development of neurodegenerative disorders.  However, the discovery of pharmacologically relevant compounds is hindered by insufficient structural understanding of the activation process.  In this study we provide a model peptidic activator of human proteasome and analyze the structure-activity relationship within this novel scaffold.  The binding mode of the activator at the relevant pocket within the proteasome has been determined by X-ray crystallography.  This crystal structure provides an important basis for rational design of pharmacological compounds.  Moreover, by providing a novel insight into the proteasome gating mechanism, our results allow the commonly accepted model of proteasome regulation to be revisited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpbYP86F-n6LYtC3U5jeOifW6udTcc2eYdVmEIH3_rZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfgtFentA%253D%253D&md5=9daf28ff62212eb8765546eb8762c3bc</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-05997-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-05997-4%26sid%3Dliteratum%253Aachs%26aulast%3DWitkowska%26aufirst%3DJ.%26aulast%3DGizy%25C5%2584ska%26aufirst%3DM.%26aulast%3DGrudnik%26aufirst%3DP.%26aulast%3DGolik%26aufirst%3DP.%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DGie%25C5%2582do%25C5%2584%26aufirst%3DA.%26aulast%3DDubin%26aufirst%3DG.%26aulast%3DJankowska%26aufirst%3DE.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Low%2520Molecular%2520Weight%2520Activator%2520Blm-Pep%2520with%2520Yeast%252020S%2520Proteasome%2520-%2520Insights%2520into%2520the%2520Enzyme%2520Activation%2520Mechanism%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26issue%3D1%26spage%3D6177%26doi%3D10.1038%2Fs41598-017-05997-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dal
Vechio, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerqueira, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augusto, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demasi, M.</span></span> <span> </span><span class="NLM_article-title">Peptides That Activate the 20S Proteasome by Gate Opening Increased Oxidized Protein Removal and Reduced Protein Aggregation</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2013.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.freeradbiomed.2013.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24291399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlehu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=304-313&author=F.+H.+Dal%0AVechioauthor=F.+Cerqueiraauthor=O.+Augustoauthor=R.+Lopesauthor=M.+Demasi&title=Peptides+That+Activate+the+20S+Proteasome+by+Gate+Opening+Increased+Oxidized+Protein+Removal+and+Reduced+Protein+Aggregation&doi=10.1016%2Fj.freeradbiomed.2013.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides that activate the 20S proteasome by gate opening increased oxidized protein removal and reduced protein aggregation</span></div><div class="casAuthors">Dal Vechio, Francisco H.; Cerqueira, Fernanda; Augusto, Ohara; Lopes, Robson; Demasi, Marilene</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-313</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The proteasome is a multicatalytic protease that is responsible for the degrdn. of the majority of intracellular proteins.  Its role is correlated with several major regulatory pathways that are involved in cell cycle control, signaling, and antigen presentation, as well as in the removal of oxidatively damaged proteins.  Although several proteasomal catalytic inhibitors have been described, very few activators have been reported to date.  Some reports in the literature highlight the cellular protective effects of proteasome activation against oxidative stress and its effect on increased life span.  In this work, we describe a peptide named proteasome-activating peptide 1 (PAP1), which increases the chymotrypsin-like proteasomal catalytic activity and, consequently, proteolytic rates both in vitro and in culture.  PAP1 proteasomal activation is mediated by the opening of the proteasomal catalytic chamber.  We also demonstrate that the obsd. proteasomal activation protected cells from oxidative stress; further, PAP1 prevented protein aggregation in a cellular model of amyotrophic lateral sclerosis.  The role of 20SPT gate opening underlying protection against oxidative stress was also explored in yeast cells.  The present data indicate the importance of proteasomal activators as potential drugs for the treatment of pathologies assocd. with the impaired removal of damaged proteins, which is obsd. in many neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OVs0noe7YbVg90H21EOLACvtfcHk0ljLbmldfBOLXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlehu7g%253D&md5=d710cb1adc50097ae7a3cdf2f25ebd03</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2013.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2013.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DDal%2BVechio%26aufirst%3DF.%2BH.%26aulast%3DCerqueira%26aufirst%3DF.%26aulast%3DAugusto%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DR.%26aulast%3DDemasi%26aufirst%3DM.%26atitle%3DPeptides%2520That%2520Activate%2520the%252020S%2520Proteasome%2520by%2520Gate%2520Opening%2520Increased%2520Oxidized%2520Protein%2520Removal%2520and%2520Reduced%2520Protein%2520Aggregation%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2014%26volume%3D67%26spage%3D304%26epage%3D313%26doi%3D10.1016%2Fj.freeradbiomed.2013.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karpowicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkowska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikorska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giżyńska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belczyk-Ciesielska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankowska, E.</span></span> <span> </span><span class="NLM_article-title">Interplay between Structure and Charge as a Key to Allosteric Modulation of Human 20S Proteasome by the Basic Fragment of HIV-1 Tat Protein</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e0143038</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0143038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1371%2Fjournal.pone.0143038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=26575189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0143038&issue=11&author=P.+Karpowiczauthor=P.+A.+Osmulskiauthor=J.+Witkowskaauthor=E.+Sikorskaauthor=M.+Gi%C5%BCy%C5%84skaauthor=A.+Belczyk-Ciesielskaauthor=M.+E.+Gaczynskaauthor=E.+Jankowska&title=Interplay+between+Structure+and+Charge+as+a+Key+to+Allosteric+Modulation+of+Human+20S+Proteasome+by+the+Basic+Fragment+of+HIV-1+Tat+Protein&doi=10.1371%2Fjournal.pone.0143038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Interplay between structure and charge as a key to allosteric modulation of human 20S proteasome by the basic fragment of HIV-1 tat protein</span></div><div class="casAuthors">Karpowicz, Przemyslaw; Osmulski, Pawel A.; Witkowska, Julia; Sikorska, Emilia; Gizynska, Malgorzata; Belczyk-Ciesielska, Agnieszka; Gaczynska, Maria E.; Jankowska, Elzbieta</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0143038/1-e0143038/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The proteasome is a giant protease responsible for degrdn. of the majority of cytosolic proteins.  Competitive inhibitors of the proteasome are used against aggressive blood cancers.  However, broadening the use of proteasome-targeting drugs requires new mechanistic approaches to the enzyme's inhibition.  In our previous studies we described Tat1 peptide, an allosteric inhibitor of the proteasome derived from a fragment of the basic domain of HIV-Tat1 protein.  Here, we attempted to dissect the structural determinants of the proteasome inhibition by Tat1.  Single- and multiple- alanine walking scans were performed.  Tat1 analogs with stabilized beta-turn conformation at positions 4-5 and 8-9, pointed out by the mol. dynamics modeling and the alanine scan, were synthesized.  Structure of Tat1 analogs were analyzed by CD, Fourier transform IR and NMR spectroscopy studies, supplemented by mol. dynamics simulations.  Biol. activity tests and structural studies revealed that high flexibility and exposed pos. charge are hallmarks of Tat1 peptide.  Interestingly, stabilization of a beta-turn at the 8-9 position was necessary to significantly improve the inhibitory potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnCn9RQIMoq7Vg90H21EOLACvtfcHk0ljZAp8Y8RiBIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSjs7vP&md5=afdbf33be55eeba722755025a1d30ed7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0143038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0143038%26sid%3Dliteratum%253Aachs%26aulast%3DKarpowicz%26aufirst%3DP.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DWitkowska%26aufirst%3DJ.%26aulast%3DSikorska%26aufirst%3DE.%26aulast%3DGi%25C5%25BCy%25C5%2584ska%26aufirst%3DM.%26aulast%3DBelczyk-Ciesielska%26aufirst%3DA.%26aulast%3DGaczynska%26aufirst%3DM.%2BE.%26aulast%3DJankowska%26aufirst%3DE.%26atitle%3DInterplay%2520between%2520Structure%2520and%2520Charge%2520as%2520a%2520Key%2520to%2520Allosteric%2520Modulation%2520of%2520Human%252020S%2520Proteasome%2520by%2520the%2520Basic%2520Fragment%2520of%2520HIV-1%2520Tat%2520Protein%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D11%26spage%3De0143038%26doi%3D10.1371%2Fjournal.pone.0143038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metais, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aird, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hampton, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">PR39, a Peptide Regulator of Angiogenesis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/71527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F71527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10613823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXks1WlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=49-55&issue=1&author=J.+Liauthor=M.+Postauthor=R.+Volkauthor=Y.+Gaoauthor=M.+Liauthor=C.+Metaisauthor=K.+Satoauthor=J.+Tsaiauthor=W.+Airdauthor=R.+D.+Rosenbergauthor=T.+G.+Hamptonauthor=J.+Liauthor=F.+Sellkeauthor=P.+Carmelietauthor=M.+Simons&title=PR39%2C+a+Peptide+Regulator+of+Angiogenesis&doi=10.1038%2F71527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PR39, a peptide regulator of angiogenesis</span></div><div class="casAuthors">Li, Jian; Post, Mark; Volk, Rudiger; Gao, Youhe; Li, Min; Metais, Caroline; Sato, Kaori; Tsai, Jo; Aird, William; Rosenberg, Robert D.; Hampton, Thomas G.; Li, Jianyi; Sellke, Frank; Carmeliet, Peter; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-55</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Although tissue injury and inflammation are considered essential for the induction of angiogenesis, the mol. controls of this cascade are mostly unknown.  Here we show that a macrophage-derived peptide, PR39, inhibited the ubiquitin-proteasome-dependent degrdn. of hypoxia-inducible factor-1α protein, resulting in accelerated formation of vascular structures in vitro and increased myocardial vasculature in mice.  For the latter, coronary flow studies demonstrated that PR39-induced angiogenesis resulted in the prodn. of functional blood vessels.  These findings show that PR39 and related compds. can be used as potent inductors of angiogenesis, and that selective inhibition of hypoxia-inducible factor-1α degrdn. may underlie the mechanism of inflammation-induced angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmNGIiFzRNQ7Vg90H21EOLACvtfcHk0ljZAp8Y8RiBIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXks1WlsA%253D%253D&md5=832cbec47bef0ca645f59de00d2639df</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2F71527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F71527%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPost%26aufirst%3DM.%26aulast%3DVolk%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMetais%26aufirst%3DC.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DAird%26aufirst%3DW.%26aulast%3DRosenberg%26aufirst%3DR.%2BD.%26aulast%3DHampton%26aufirst%3DT.%2BG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSellke%26aufirst%3DF.%26aulast%3DCarmeliet%26aufirst%3DP.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DPR39%252C%2520a%2520Peptide%2520Regulator%2520of%2520Angiogenesis%26jtitle%3DNat.%2520Med.%26date%3D2000%26volume%3D6%26issue%3D1%26spage%3D49%26epage%3D55%26doi%3D10.1038%2F71527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hietaranta, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saluja, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steer, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ubiquitin-Proteasome Pathway-mediated IκBα Degradation by a Naturally Occurring Antibacterial Peptide</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.1172/JCI9826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1172%2FJCI9826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10930447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2000&pages=439-448&issue=3&author=Y.+Gaoauthor=S.+Leckerauthor=M.+J.+Postauthor=A.+J.+Hietarantaauthor=J.+Liauthor=R.+Volkauthor=M.+Liauthor=K.+Satoauthor=A.+K.+Salujaauthor=M.+L.+Steerauthor=A.+L.+Goldbergauthor=M.+Simons&title=Inhibition+of+Ubiquitin-Proteasome+Pathway-mediated+I%CE%BAB%CE%B1+Degradation+by+a+Naturally+Occurring+Antibacterial+Peptide&doi=10.1172%2FJCI9826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of ubiquitin-proteasome pathway-mediated IκBα degradation by a naturally occurring antibacterial peptide</span></div><div class="casAuthors">Gao, Youhe; Lecker, Stewart; Post, Mark J.; Hietaranta, Antti J.; Li, Jian; Volk, Rudiger; Li, Min; Sato, Kaori; Saluja, Ashok K.; Steer, Michael L.; Goldberg, Alfred L.; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">439-448</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Induction of NF-κB-dependent gene expression plays an important role in a no. of biol. processes including inflammation and ischemia-reperfusion injury.  However, few attempts aimed at selective regulation of this transcription factor have been successful.  We report here that a naturally occurring antibacterial peptide PR39 reversibly binds to the α7 subunit of the 26S proteasome and blocks degrdn. of NF-κB inhibitor IκBα by the ubiquitin-proteasome pathway without affecting overall proteasome activity.  IκBα phosphorylation and ubiquitination occur normally after PR39 treatment, and binding of valosin-contg. proteins is not impaired.  The inhibition of IκBα degrdn. abolishes induction of NF-κB-dependent gene expression in cell culture and in mouse models of acute pancreatitis and myocardial infarction, including upregulation of endothelial adhesion proteins VCAM-1 and ICAM-1.  In the latter model, sustained infusion of PR39 peptide resulted in significant redn. of myocardial infarct size.  PR39 and related peptides may provide novel means to regulate cellular function and to control of NF-κB-dependent gene expression for therapeutic purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-ImWsrwT5-bVg90H21EOLACvtfcHk0ljZAp8Y8RiBIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOrtb8%253D&md5=3deee114b2e7d008929f77a7e6c307b1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1172%2FJCI9826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI9826%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLecker%26aufirst%3DS.%26aulast%3DPost%26aufirst%3DM.%2BJ.%26aulast%3DHietaranta%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DVolk%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DSaluja%26aufirst%3DA.%2BK.%26aulast%3DSteer%26aufirst%3DM.%2BL.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520Ubiquitin-Proteasome%2520Pathway-mediated%2520I%25CE%25BAB%25CE%25B1%2520Degradation%2520by%2520a%2520Naturally%2520Occurring%2520Antibacterial%2520Peptide%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2000%26volume%3D106%26issue%3D3%26spage%3D439%26epage%3D448%26doi%3D10.1172%2FJCI9826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Proline- and Arginine-Rich Peptides Constitute a Novel Class of Allosteric Inhibitors of Proteasome Activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">29</span>),  <span class="NLM_fpage">8663</span>– <span class="NLM_lpage">8670</span>, <span class="refDoi"> DOI: 10.1021/bi034784f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi034784f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltV2jur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=8663-8670&issue=29&author=M.+Gaczynskaauthor=P.+A.+Osmulskiauthor=Y.+Gaoauthor=M.+J.+Postauthor=M.+Simons&title=Proline-+and+Arginine-Rich+Peptides+Constitute+a+Novel+Class+of+Allosteric+Inhibitors+of+Proteasome+Activity&doi=10.1021%2Fbi034784f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Proline- and arginine-rich peptides constitute a novel class of allosteric inhibitors of proteasome activity</span></div><div class="casAuthors">Gaczynska, Maria; Osmulski, Pawel A.; Gao, Youhe; Post, Mark J.; Simons, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">8663-8670</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substrate-specific inhibition of the proteasome has been unachievable despite great interest in proteasome inhibitors as drugs.  Recent studies demonstrated that PR39, a natural proline- and arginine-rich antibacterial peptide, stimulates angiogenesis and inhibits inflammatory responses by specifically blocking degrdn. of IκBα and HIF-1α by the proteasome.  However, mol. events involved in the PR39-proteasome interaction have not been elucidated.  Here the authors show that PR39 is a noncompetitive and reversible inhibitor of the proteasome function.  This effect is achieved by a unique allosteric mechanism allowing for specific inhibition of degrdn. of selected proteins without affecting total proteasome-dependent proteolysis.  At. force microscopy (AFM) studies demonstrate that 20S and 26S proteasomes treated with PR39 or its derivs. exhibit serious perturbations in their structure and their normal allosteric movements.  These effects are universal for proteasomes from yeast to human.  The shortest functional sequence derived from PR39 still showing the allosteric inhibitory effect consists of eleven N-terminal residues contg. essential three N-terminal arginines.  The noncompetitive and reversible in vitro action of PR39 and its truncated derivs. is matched by the ability of the peptides to induce angiogenesis in vivo.  The authors postulate that PR39 changes conformational dynamics of the proteasomes by interactions with the noncatalytic subunit α7 in a way that prevents the enzyme from cleaving the substrates of unique structural constraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3kqE8-uSDBLVg90H21EOLACvtfcHk0liuZSLUmDe7kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltV2jur8%253D&md5=442f1c1a97fd83ae905f85c8134f79c7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi034784f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi034784f%26sid%3Dliteratum%253Aachs%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DPost%26aufirst%3DM.%2BJ.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DProline-%2520and%2520Arginine-Rich%2520Peptides%2520Constitute%2520a%2520Novel%2520Class%2520of%2520Allosteric%2520Inhibitors%2520of%2520Proteasome%2520Activity%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26issue%3D29%26spage%3D8663%26epage%3D8670%26doi%3D10.1021%2Fbi034784f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anbanandam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albarado, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirziu, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veeraraghavan, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Proline- and Arginine-Rich Peptide Inhibition of Proteasome</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.jmb.2008.09.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=18823992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSgs7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=219-227&issue=1&author=A.+Anbanandamauthor=D.+C.+Albaradoauthor=D.+C.+Tirziuauthor=M.+Simonsauthor=S.+Veeraraghavan&title=Molecular+Basis+for+Proline-+and+Arginine-Rich+Peptide+Inhibition+of+Proteasome&doi=10.1016%2Fj.jmb.2008.09.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for Proline- and Arginine-Rich Peptide Inhibition of Proteasome</span></div><div class="casAuthors">Anbanandam, Asokan; Albarado, Diana C.; Tirziu, Daniela C.; Simons, Michael; Veeraraghavan, Sudha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-227</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PR39, a naturally occurring and cell-permeable proline- and arginine-rich peptide, blocks the degrdn. of inhibitor of nuclear factor κB (IκBα), thereby attenuating inflammation.  It is a noncompetitive and reversible inhibitor of 20S proteasome.  To identify its basis of action, we used soln. NMR spectroscopy and mutational analyses of the active fragment, PR11, which identified amino acids required for human 20S proteasome inhibiting activity.  We then examd. PR11-mediated changes in the expression of nuclear factor κB-dependent genes in situ.  The results provide prerequisites for proteasome inhibition by proline- and arginine-rich peptides, providing a powerful new tool to investigate inflammatory processes.  These findings offer new leads in developing drugs to treat heart diseases or stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrujUG4EXG_s7Vg90H21EOLACvtfcHk0liuZSLUmDe7kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSgs7nE&md5=2a2c8c1854f8a1445ce94947474bb0d8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.021%26sid%3Dliteratum%253Aachs%26aulast%3DAnbanandam%26aufirst%3DA.%26aulast%3DAlbarado%26aufirst%3DD.%2BC.%26aulast%3DTirziu%26aufirst%3DD.%2BC.%26aulast%3DSimons%26aufirst%3DM.%26aulast%3DVeeraraghavan%26aufirst%3DS.%26atitle%3DMolecular%2520Basis%2520for%2520Proline-%2520and%2520Arginine-Rich%2520Peptide%2520Inhibition%2520of%2520Proteasome%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26issue%3D1%26spage%3D219%26epage%3D227%26doi%3D10.1016%2Fj.jmb.2008.09.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myszka, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span> <span> </span><span class="NLM_article-title">Structural Models for Interactions between the 20S Proteasome and Its PAN/19S Activators</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1074/jbc.C109.070425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fjbc.C109.070425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19889631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=13-17&issue=1&author=B.+M.+Stadtmuellerauthor=K.+Ferrellauthor=F.+G.+Whitbyauthor=A.+Herouxauthor=H.+Robinsonauthor=D.+G.+Myszkaauthor=C.+P.+Hill&title=Structural+Models+for+Interactions+between+the+20S+Proteasome+and+Its+PAN%2F19S+Activators&doi=10.1074%2Fjbc.C109.070425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Models for Interactions between the 20S Proteasome and Its PAN/19S Activators</span></div><div class="casAuthors">Stadtmueller, Beth M.; Ferrell, Katherine; Whitby, Frank G.; Heroux, Annie; Robinson, Howard; Myszka, David G.; Hill, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Proteasome activity is regulated by sequestration of its proteolytic centers in a barrel-shaped structure that limits substrate access.  Substrates enter the proteasome by means of activator complexes that bind to the end rings of proteasome α subunits and induce opening of an axial entrance/exit pore.  The PA26 (proteasome activator 26 kDa) activator binds in a pocket on the proteasome surface using main chain contacts of its C-terminal residues and uses an internal activation loop to trigger gate opening by repositioning the proteasome Pro-17 reverse turn.  Subunits of the unrelated 19S/PAN (proteasome-activating nucleosidase) activators bind with their C termini in the same pockets but can induce proteasome gate opening entirely from interactions of their C-terminal peptides, which are reported to cause gate opening by inducing a rocking motion of proteasome α subunits rather than by directly contacting the Pro-17 turn.  Here we report crystal structures and binding studies of proteasome complexes with PA26 constructs that display modified C-terminal residues, including those corresponding to PAN.  These findings suggest that PA26 and PAN/19S C-terminal residues bind superimposably and that both classes of activator induce gate opening by using direct contacts to residues of the proteasome Pro-17 reverse turn.  In the case of the PAN and 19S activators, a penultimate tyrosine/phenylalanine residue contacts the proteasome Gly-19 carbonyl oxygen to stabilize the open conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQoFu-WcOgLLVg90H21EOLACvtfcHk0liuZSLUmDe7kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbnO&md5=6e01cd7f1f713257c9b2e549ac94e75b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C109.070425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C109.070425%26sid%3Dliteratum%253Aachs%26aulast%3DStadtmueller%26aufirst%3DB.%2BM.%26aulast%3DFerrell%26aufirst%3DK.%26aulast%3DWhitby%26aufirst%3DF.%2BG.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26aulast%3DHill%26aufirst%3DC.%2BP.%26atitle%3DStructural%2520Models%2520for%2520Interactions%2520between%2520the%252020S%2520Proteasome%2520and%2520Its%2520PAN%252F19S%2520Activators%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D1%26spage%3D13%26epage%3D17%26doi%3D10.1074%2Fjbc.C109.070425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Traut, T. W.</span></span> <i>Allosteric Regulatory Enzymes</i>; <span class="NLM_publisher-name">Springer US</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2F978-0-387-72891-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=T.+W.+Traut&title=Allosteric+Regulatory+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-72891-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-72891-9%26sid%3Dliteratum%253Aachs%26aulast%3DTraut%26aufirst%3DT.%2BW.%26btitle%3DAllosteric%2520Regulatory%2520Enzymes%26pub%3DSpringer%2520US%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melandri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Characterization of the Chymotryptic Activity of the 20S Proteasome</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3899</span>– <span class="NLM_lpage">3908</span>, <span class="refDoi"> DOI: 10.1021/bi952262x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi952262x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK28XhsFKqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=3899-3908&issue=13&author=R.+L.+Steinauthor=F.+Melandriauthor=L.+Dick&title=Kinetic+Characterization+of+the+Chymotryptic+Activity+of+the+20S+Proteasome&doi=10.1021%2Fbi952262x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of the chymotryptic activity of the 20S proteasome</span></div><div class="casAuthors">Stein, Ross L.; Melandri, Francesco; Dick, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3899-908</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors report kinetic studies for the chymotryptic activity of the 20S proteasome.  Reaction progress curves that are recorded at concns. of Suc-Leu-Leu-Val-Tyr-AMC greater than about 40 μM are biphasic and characterized by initial velocities that decay by a first-order process to final, steady-state velocities.  At [Suc-Leu-Leu-Val-Tyr-AMC] > 40 μM, initial and steady-state velocities are smaller than predicted from simple, Michaelis-Menten kinetics.  The first-order rate const. for the approach to steady-state has a complex dependence on substrate concn. and decreases sigmoidally as substrate concn. increases.  These results indicate that the 20S proteasome is a hysteretic enzyme and is subject to substrate inhibition.  To explain these observations the authors propose a minimal kinetic model with two crit. mechanistic features: (1) the 20S proteasome has two cooperative active sites for Suc-Leu-Leu-Val-Tyr-AMC and (2) there are two interconvertible conformers of active 20S proteasome.  To probe this mechanism in greater detail, the authors explored the kinetic mechanism of inhibition of the 20S proteasome-catalyzed hydrolysis of Suc-Leu-Leu-Val-Tyr-AMC by the peptide aldehyde, Ac-Leu-Leu-Nle-H.  The studies reveal a nonlinear dependence of steady-state velocity on inhibitor concn. (i.e., parabolic inhibition) as well as a nonlinear dependence of the apparent inhibitor dissocn. const. on substrate concn.  Both of these observations are explained by binding of inhibitor at multiple sites on the enzyme.  Taken together, the results of this study indicate that the 20S proteasome is a conformationally flexible protein that can adjust to the binding of ligands and that has multiple and cooperative active sites.  These results support a view of the proteasome's substrate specificity in which (1) substrates are recognized and hydrolyzed by more than one active site; (2) each active site can bind substrates that possess a variety of P1 residues; and (3) the P1 residue plays a relatively minor role as a specificity determinant.  Finally, the authors interpret the results of this study to suggest that, in vivo, the 20S proteasome requires conformational plasticity for its interactions with regulatory complexes and, after it has combined with appropriate regulatory complexes, to catalyze hydrolysis of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQf-rPQS50RrVg90H21EOLACvtfcHk0ljNhUcQV_GB2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFKqsrg%253D&md5=5cbce45f695898e7501d57b72159d24c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi952262x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi952262x%26sid%3Dliteratum%253Aachs%26aulast%3DStein%26aufirst%3DR.%2BL.%26aulast%3DMelandri%26aufirst%3DF.%26aulast%3DDick%26aufirst%3DL.%26atitle%3DKinetic%2520Characterization%2520of%2520the%2520Chymotryptic%2520Activity%2520of%2520the%252020S%2520Proteasome%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26issue%3D13%26spage%3D3899%26epage%3D3908%26doi%3D10.1021%2Fbi952262x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trader, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and Application of a Sensitive Peptide Reporter to Discover 20S Proteasome Stimulators</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.7b00193</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.7b00193" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltV2rtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=269-276&issue=5&author=R.%0AA.+Colemanauthor=D.+J.+Trader&title=Development+and+Application+of+a+Sensitive+Peptide+Reporter+to+Discover+20S+Proteasome+Stimulators&doi=10.1021%2Facscombsci.7b00193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Application of a Sensitive Peptide Reporter to Discover 20S Proteasome Stimulators</span></div><div class="casAuthors">Coleman, Rachel A.; Trader, Darci J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To attenuate an overabundance of cellular protein, it has been hypothesized that the 20S core particle (20S CP) of the proteasome can be chem. stimulated to degrade proteins into nontoxic peptides more quickly.  Screening for small mol. 20S CP stimulators is typically performed with a reporter peptide composed of four amino acids and a coumarin group that is released upon proteasome-mediated hydrolysis to generate a fluorescent signal.  Screening with this small reporter can lead to false negatives because the reporter peptide is rapidly turned-over without stimulation.  To improve the screening for 20S CP stimulators, the authors have developed a peptide FRET reporter nearly four times more sensitive to stimulation but still amenable for high throughput screening.  Through application of the authors' FRET reporter, the authors discovered two 20S CP gate-opening stimulators and also a mol. that elicits its mechanism of action through an interaction with a 20S CP active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaNdZtHuSLubVg90H21EOLACvtfcHk0ljNhUcQV_GB2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltV2rtr0%253D&md5=ba87e0eea43df05bc45d9e17dc240036</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.7b00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.7b00193%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DTrader%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520Application%2520of%2520a%2520Sensitive%2520Peptide%2520Reporter%2520to%2520Discover%252020S%2520Proteasome%2520Stimulators%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26issue%3D5%26spage%3D269%26epage%3D276%26doi%3D10.1021%2Facscombsci.7b00193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, A. L.</span></span> <span> </span><span class="NLM_article-title">ATP Binding to PAN or the 26S ATPases Causes Association with the 20S Proteasome, Gate Opening, and Translocation of Unfolded Proteins</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2005.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=16337593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtleqtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2005&pages=687-698&issue=5&author=D.+M.+Smithauthor=G.+Kafriauthor=Y.+Chengauthor=D.+Ngauthor=T.+Walzauthor=A.+L.+Goldberg&title=ATP+Binding+to+PAN+or+the+26S+ATPases+Causes+Association+with+the+20S+Proteasome%2C+Gate+Opening%2C+and+Translocation+of+Unfolded+Proteins&doi=10.1016%2Fj.molcel.2005.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ATP binding to PAN or the 26S ATPases causes association with the 20S proteasome, gate opening, and translocation of unfolded proteins</span></div><div class="casAuthors">Smith, David M.; Kafri, Galit; Cheng, Yifan; Ng, David; Walz, Thomas; Goldberg, Alfred L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">687-698</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The archaeal ATPase complex PAN, the homolog of the eukaryotic 26S proteasome-regulatory ATPases, was shown to assoc. transiently with the 20S proteasome upon binding of ATP or ATPγS, but not ADP.  By electron microscopy (EM), PAN appears as a two-ring structure, capping the 20S, and resembles two densities in the 19S complex.  The N termini of the archaeal 20S α subunits were found to function as a gate that prevents entry of seven-residue peptides but allows entry of tetrapeptides.  Upon assocn. with the 20S particle, PAN stimulates gate opening.  Although degrdn. of globular proteins requires ATP hydrolysis, the PAN-20S complex with ATPγS translocates and degrades unfolded and denatured proteins.  Rabbit 26S proteasomes also degrade these unfolded proteins upon ATP binding, without hydrolysis.  Thus, although unfolding requires energy from ATP hydrolysis, ATP binding alone supports ATPase-20S assocn., gate opening, and translocation of unfolded substrates into the proteasome, which can occur by facilitated diffusion through the ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbfrsl1AGA9rVg90H21EOLACvtfcHk0ljNhUcQV_GB2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtleqtb%252FO&md5=3bf5c913e4880a0b27e098521092d3d0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DKafri%26aufirst%3DG.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DNg%26aufirst%3DD.%26aulast%3DWalz%26aufirst%3DT.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DATP%2520Binding%2520to%2520PAN%2520or%2520the%252026S%2520ATPases%2520Causes%2520Association%2520with%2520the%252020S%2520Proteasome%252C%2520Gate%2520Opening%252C%2520and%2520Translocation%2520of%2520Unfolded%2520Proteins%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D20%26issue%3D5%26spage%3D687%26epage%3D698%26doi%3D10.1016%2Fj.molcel.2005.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez-Castelao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goethals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaño, J. G.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Cleavage of Alpha-Synuclein by the 20S Proteasome and Modulation of Its Degradation by the RedOx State of the N-Terminal Methionines</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2013.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bbamcr.2013.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=24315858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFei" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=352-365&issue=2&author=B.+Alvarez-Castelaoauthor=M.+Goethalsauthor=J.+Vandekerckhoveauthor=J.+G.+Casta%C3%B1o&title=Mechanism+of+Cleavage+of+Alpha-Synuclein+by+the+20S+Proteasome+and+Modulation+of+Its+Degradation+by+the+RedOx+State+of+the+N-Terminal+Methionines&doi=10.1016%2Fj.bbamcr.2013.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of cleavage of alpha-synuclein by the 20S proteasome and modulation of its degradation by the RedOx state of the N-terminal methionines</span></div><div class="casAuthors">Alvarez-Castelao, Beatriz; Goethals, Marc; Vandekerckhove, Joel; Castano, Jose G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">352-365</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Alpha-synuclein is a small protein implicated in the pathophysiol. of Parkinson's disease (PD).  We have investigated the mechanism of cleavage of alpha-synuclein by the 20S proteasome.  Alpha-synuclein interacts with the C8 (α7) subunit of the proteasome.  The N-terminal part of alpha-synuclein (amino acids 1-60) is essential for its proteasomal degrdn. and anal. of peptides released from proteasomal digestion allows concluding that initial cleavages occur within the N-terminal region of the mol.  Aggregated alpha-synucleins are also degraded by the proteasome with a reduced rate, likely due to Met oxidn.  In fact, mild oxidn. of alpha-synuclein with H2O2 resulted in the inhibition of its degrdn. by the proteasome, mainly due to oxidn. of Met 1 and 5 of alpha-synuclein.  The inhibition was reversed by treatment of the oxidized protein with methionine sulfoxide reductases (MsrA plus MsrB).  Similarly, treatment with H2O2 of N2A cells transfected with alpha-synuclein resulted in the inhibition of its degrdn. that was also reverted by co-transfection of MsrA plus MsrB.  These results clearly indicate that oxidative stress, a common feature of PD and other synucleinopathies, promotes a RedOx change in the proteostasis of alpha-synuclein due to Met oxidn. and reduced proteasomal degrdn.; compromised reversion of those oxidative changes would result in the accumulation of oxidative damaged alpha-synuclein likely contributing to the pathogenesis of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHZc32bql3P7Vg90H21EOLACvtfcHk0lgnssJUqFWgYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFei&md5=9deadf1c5b5655efea0ce37e7c0aeee8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2013.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2013.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez-Castelao%26aufirst%3DB.%26aulast%3DGoethals%26aufirst%3DM.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DJ.%2BG.%26atitle%3DMechanism%2520of%2520Cleavage%2520of%2520Alpha-Synuclein%2520by%2520the%252020S%2520Proteasome%2520and%2520Modulation%2520of%2520Its%2520Degradation%2520by%2520the%2520RedOx%2520State%2520of%2520the%2520N-Terminal%2520Methionines%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26issue%3D2%26spage%3D352%26epage%3D365%26doi%3D10.1016%2Fj.bbamcr.2013.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trader, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Establishment of a Suite of Assays That Support the Discovery of Proteasome Stimulators</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1861</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">899</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.bbagen.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28065760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1ChtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2017&pages=892-899&issue=4&author=D.+J.+Traderauthor=S.+Simanskiauthor=P.+Dicksonauthor=T.+Kodadek&title=Establishment+of+a+Suite+of+Assays+That+Support+the+Discovery+of+Proteasome+Stimulators&doi=10.1016%2Fj.bbagen.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of a suite of assays that support the discovery of proteasome stimulators</span></div><div class="casAuthors">Trader, Darci J.; Simanski, Scott; Dickson, Paige; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-899</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The proteasome catalyzes the degrdn. of many mis-folded proteins, which are otherwise cytotoxic.  There is interest in the discovery of proteasome agonists, but previous efforts to do so have been disappointing.  The cleavage of small fluorogenic peptides is used routinely as an assay to screen for proteasome modulators.  The authors have developed follow-on assays that employ more physiol. relevant substrates.  To demonstrate the efficacy of this workflow, the NIH Clin. Collection (NCC) was screened.  While many compds. stimulated proteasome-mediated proteolysis of the pro-fluorogenic peptide substrates, most failed to evince activity in assays with larger peptide or protein substrates.  The authors also show that two mols. claimed previously to be proteasome agonists, oleuropein and betulinic acid, indeed accelerate hydrolysis of the fluorogenic substrate, but have no effect on the turnover of a mis-folded protein in vitro or in cellulo.  However, two small mols. from the NCC, MK-866 and AM-404, stimulate the proteasome-mediated turnover of a mis-folded protein in living cells by 3- to 4-fold.  Assays that monitor the proteasome-mediated degrdn. of larger peptides and proteins can distinguish bona fide agonists from compds. only able to stimulate the cleavage of short, nonphysiol. relevant peptides.  A suite of assays has been established that allows the discovery of bona fide proteasome agonists.  AM-404 and MK-866 can be useful tools for cell culture expts., and can serve as scaffolds to generate more potent 20S stimulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKT3yI4GlcnbVg90H21EOLACvtfcHk0lgnssJUqFWgYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1ChtLg%253D&md5=65e65acaca269de157eb121c81b1f626</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTrader%26aufirst%3DD.%2BJ.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DDickson%26aufirst%3DP.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DEstablishment%2520of%2520a%2520Suite%2520of%2520Assays%2520That%2520Support%2520the%2520Discovery%2520of%2520Proteasome%2520Stimulators%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2017%26volume%3D1861%26issue%3D4%26spage%3D892%26epage%3D899%26doi%3D10.1016%2Fj.bbagen.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilholz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemeyer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammensee, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schild, H.</span></span> <span> </span><span class="NLM_article-title">Cleavage Motifs of the Yeast 20S Proteasome Subunits Deduced from Digests of Enolase 1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">12504</span>– <span class="NLM_lpage">12509</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.21.12504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.95.21.12504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=9770515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK1cXmsl2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=12504-12509&issue=21&author=A.+K.+Nussbaumauthor=T.+P.+Dickauthor=W.+Keilholzauthor=M.+Schirleauthor=S.+Stevanovicauthor=K.+Dietzauthor=W.+Heinemeyerauthor=M.+Grollauthor=D.+H.+Wolfauthor=R.+Huberauthor=H.-G.+Rammenseeauthor=H.+Schild&title=Cleavage+Motifs+of+the+Yeast+20S+Proteasome+Subunits+Deduced+from+Digests+of+Enolase+1&doi=10.1073%2Fpnas.95.21.12504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cleavage motifs of the yeast 20S proteasome β subunits deduced from digests of enolase 1</span></div><div class="casAuthors">Nussbaum, Alexander K.; Dick, Tobias P.; Keilholz, Wieland; Schirle, Markus; Stevanovic, Stefan; Dietz, Klaus; Heinemeyer, Wolfgang; Groll, Michael; Wolf, Dieter H.; Huber, Robert; Rammensee, Hans-Georg; Schild, Hansjorg</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12504-12509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 436-amino acid protein enolase 1 from yeast was degraded in vitro by purified wild-type and mutant yeast 20S proteasome particles.  Anal. of the cleavage products at different times revealed a processive degrdn. mechanism and a length distribution of fragments ranging from 3 to 25 amino acids with an av. length of 7 to 8 amino acids.  Surprisingly, the av. fragment length was very similar between wild-type and mutant 20S proteasomes with reduced nos. of active sites.  This implies that the fragment length is not influenced by the distance between the active sites, as previously postulated.  A detailed anal. of the cleavages also allowed the identification of certain amino acid characteristics in positions flanking the cleavage site that guide the selection of the P1 residues by the three active β subunits.  Because yeast and mammalian proteasomes are highly homologous, similar cleavage motifs might be used by mammalian proteasomes.  Therefore, the data provide a basis for predicting proteasomal degrdn. products from which peptides are sampled by major histocompatibility complex class I mols. for presentation to cytotoxic T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI-Ri_79ge-bVg90H21EOLACvtfcHk0lgnssJUqFWgYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsl2qsLY%253D&md5=d5dcf9cce44a50d37bf3765906cc332d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.21.12504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.21.12504%26sid%3Dliteratum%253Aachs%26aulast%3DNussbaum%26aufirst%3DA.%2BK.%26aulast%3DDick%26aufirst%3DT.%2BP.%26aulast%3DKeilholz%26aufirst%3DW.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DStevanovic%26aufirst%3DS.%26aulast%3DDietz%26aufirst%3DK.%26aulast%3DHeinemeyer%26aufirst%3DW.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DWolf%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DRammensee%26aufirst%3DH.-G.%26aulast%3DSchild%26aufirst%3DH.%26atitle%3DCleavage%2520Motifs%2520of%2520the%2520Yeast%252020S%2520Proteasome%2520Subunits%2520Deduced%2520from%2520Digests%2520of%2520Enolase%25201%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26issue%3D21%26spage%3D12504%26epage%3D12509%26doi%3D10.1073%2Fpnas.95.21.12504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Atomic Force Microscopy Reveals Two Conformations of the 20 S Proteasome from Fission Yeast</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">13171</span>– <span class="NLM_lpage">13174</span>, <span class="refDoi"> DOI: 10.1074/jbc.C901035199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1074%2Fjbc.C901035199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10747864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtFyksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=13171-13174&issue=18&author=P.+A.+Osmulskiauthor=M.+Gaczynska&title=Atomic+Force+Microscopy+Reveals+Two+Conformations+of+the+20+S+Proteasome+from+Fission+Yeast&doi=10.1074%2Fjbc.C901035199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic force microscopy reveals two conformations of the 20 S proteasome from fission yeast</span></div><div class="casAuthors">Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13171-13174</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The proteasome (multicatalytic proteinase) is a major cytosolic proteolytic complex, indispensable in eukaryotic cells.  The barrel-shaped core of this enzyme, the 20 S proteasome, is built from 28 subunits forming 4 stacked rings.  The 2 inner β-rings harbor active centers, whereas the 2 outer α-rings play a structural role.  The previously reported crystal structure of the yeast 20 S particle showed that the entrance to the central channel was sealed.  Because of this result, the path of substrates into the catalytic chamber has remained enigmatic.  Here, the authors used tapping mode at. force microscopy (AFM) in liq. to address the dynamic aspects of the 20 S proteasomes from fission yeast (Schizosaccharomyces pombe).  Evidence is presented that, when obsd. with AFM, the proteasome particles in top view position have either open or closed entrance to the central channel.  The preferred conformation depends on the ligands present.  Apparently, the addn. of a substrate to the uninhibited proteasome shifts the equil. toward the open conformation.  These results shed new light on the possible path of the substrate into the proteolytic chamber.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzu2Vm0qhUbVg90H21EOLACvtfcHk0lhYiIGNfVtWQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtFyksrk%253D&md5=dc6794d80535297849227a03a1df0bbb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C901035199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C901035199%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAtomic%2520Force%2520Microscopy%2520Reveals%2520Two%2520Conformations%2520of%2520the%252020%2520S%2520Proteasome%2520from%2520Fission%2520Yeast%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26issue%3D18%26spage%3D13171%26epage%3D13174%26doi%3D10.1074%2Fjbc.C901035199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochstrasser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">A Tetrahedral Transition State at the Active Sites of the 20S Proteasome Is Coupled to Opening of the α-Ring Channel</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.str.2009.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=19679091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1CltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1137-1147&issue=8&author=P.+A.+Osmulskiauthor=M.+Hochstrasserauthor=M.+Gaczynska&title=A+Tetrahedral+Transition+State+at+the+Active+Sites+of+the+20S+Proteasome+Is+Coupled+to+Opening+of+the+%CE%B1-Ring+Channel&doi=10.1016%2Fj.str.2009.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Tetrahedral Transition State at the Active Sites of the 20S Proteasome Is Coupled to Opening of the α-Ring Channel</span></div><div class="casAuthors">Osmulski, Pawel A.; Hochstrasser, Mark; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1137-1147</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Intrinsic conformational transitions contribute to the catalytic action of many enzymes.  Here we use a single-mol. approach to demonstrate how such transitions are linked to the catalytic sites of the eukaryotic proteasome, an essential protease of the ubiquitin pathway.  The active sites of the cylindrical proteasomal core particle are located in a central chamber accessible through gated entry channels.  By using at. force microscopy, we found continual alternation between open and closed gate conformations.  We analyzed the relative abundance of these conformers in wild-type and mutated yeast core particles upon exposure to substrates or inhibitors.  Our data indicate that the dynamic gate can be opened by allosteric coupling to a tetrahedral transition state at any of the working active centers.  The results point to the Nα-amine of the N-terminal active site threonyl residue as the major effector group responsible for triggering the essential conformational switch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiHKa1tntq-rVg90H21EOLACvtfcHk0lhYiIGNfVtWQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1CltLo%253D&md5=73da479a8278c7a3b5dc434390bf4883</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%26aulast%3DHochstrasser%26aufirst%3DM.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DA%2520Tetrahedral%2520Transition%2520State%2520at%2520the%2520Active%2520Sites%2520of%2520the%252020S%2520Proteasome%2520Is%2520Coupled%2520to%2520Opening%2520of%2520the%2520%25CE%25B1-Ring%2520Channel%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26issue%3D8%26spage%3D1137%26epage%3D1147%26doi%3D10.1016%2Fj.str.2009.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span> <span> </span><span class="NLM_article-title">Conformational Landscape of the p28-Bound Human Proteasome Regulatory Particle</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">333</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2Fj.molcel.2017.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28689658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCgtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=322-333&issue=2&author=Y.+Luauthor=J.+Wuauthor=Y.+Dongauthor=S.+Chenauthor=S.+Sunauthor=Y.+B.+Maauthor=Q.+Ouyangauthor=D.+Finleyauthor=M.+W.+Kirschnerauthor=Y.+Mao&title=Conformational+Landscape+of+the+p28-Bound+Human+Proteasome+Regulatory+Particle&doi=10.1016%2Fj.molcel.2017.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Landscape of the p28-Bound Human Proteasome Regulatory Particle</span></div><div class="casAuthors">Lu, Ying; Wu, Jiayi; Dong, Yuanchen; Chen, Shuobing; Sun, Shuangwu; Ma, Yong-Bei; Ouyang, Qi; Finley, Daniel; Kirschner, Marc W.; Mao, Youdong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">322-333.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The proteasome holoenzyme is activated by its regulatory particle (RP) consisting of two subcomplexes, the lid and the base.  A key event in base assembly is the formation of a heterohexameric ring of AAA-ATPases, which is guided by at least four RP assembly chaperones in mammals: PAAF1, p28/gankyrin, p27/PSMD9, and S5b.  Using cryogenic electron microscopy, we analyzed the non-AAA structure of the p28-bound human RP at 4.5 Å resoln. and detd. seven distinct conformations of the Rpn1-p28-AAA subcomplex within the p28-bound RP at subnanometer resolns.  Remarkably, the p28-bound AAA ring does not form a channel in the free RP and spontaneously samples multiple "open" and "closed" topologies at the Rpt2-Rpt6 and Rpt3-Rpt4 interfaces.  Our anal. suggests that p28 assists the proteolytic core particle to select a specific conformation of the ATPase ring for RP engagement and is released in a shoehorn-like fashion in the last step of the chaperone-mediated proteasome assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3cD6iJtLSbLVg90H21EOLACvtfcHk0lhYiIGNfVtWQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCgtbnE&md5=0863ec470f4dca12ef4ec78297dfdabb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DY.%2BB.%26aulast%3DOuyang%26aufirst%3DQ.%26aulast%3DFinley%26aufirst%3DD.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DMao%26aufirst%3DY.%26atitle%3DConformational%2520Landscape%2520of%2520the%2520p28-Bound%2520Human%2520Proteasome%2520Regulatory%2520Particle%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D67%26issue%3D2%26spage%3D322%26epage%3D333%26doi%3D10.1016%2Fj.molcel.2017.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span> <span> </span><span class="NLM_article-title">Allosteric Regulators of the Proteasome: Potential Drugs and a Novel Approach for Drug Design</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.2174/092986706775197926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.2174%2F092986706775197926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=16472211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVSiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=155-165&issue=2&author=X.+Tanauthor=P.+A.+A.+Osmulskiauthor=M.+Gaczynska&title=Allosteric+Regulators+of+the+Proteasome%3A+Potential+Drugs+and+a+Novel+Approach+for+Drug+Design&doi=10.2174%2F092986706775197926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric regulators of the proteasome: Potential drugs and a novel approach for drug design</span></div><div class="casAuthors">Tan, Xiaolin; Osmulski, Pawel A.; Gaczynska, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">155-165</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The proteasome recently gained an exceptional attention as a novel drug target, therefore its inhibitors became important subjects for rational drug design.  A synthetic competitive inhibitor Velcade was lately approved in a fast-track process to treat multiple myeloma and is tested with other types of cancers.  The proteasome is a major proteolytic assembly in eukaryotic cells responsible for the degrdn. of most intracellular proteins, including proteins crucial to cell cycle regulation and apoptosis.  The ubiquitin-proteasome pathway has been implicated in many diseases such as cancer, autoimmune diseases, inflammation, and stroke.  The activity of the proteasome can be blocked for therapeutic purposes with competitive inhibitors like Velcade, which trigger apoptosis in target cells.  However, much more versatile outcomes and a true control of the proteasome can be achieved with allosteric regulators.  Certain natural proteins and peptides bind to the catalytic core of the proteasome and allosterically induce a wide array of effects ranging from changes in product size to substrate-specific inhibition.  Designing small synthetic compds. allosterically interacting with the proteasome represents a novel approach that has enormous potential for the treatment of a wide range of diseases.  Below we provide a review of current knowledge about proteasomal allosteric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXUmGL0qOPSLVg90H21EOLACvtfcHk0lhYiIGNfVtWQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVSiug%253D%253D&md5=e8acd061f5ef976b61b18cc40b1e2bef</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2174%2F092986706775197926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986706775197926%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DX.%26aulast%3DOsmulski%26aufirst%3DP.%2BA.%2BA.%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAllosteric%2520Regulators%2520of%2520the%2520Proteasome%253A%2520Potential%2520Drugs%2520and%2520a%2520Novel%2520Approach%2520for%2520Drug%2520Design%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D155%26epage%3D165%26doi%3D10.2174%2F092986706775197926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wehmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aufderheide, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, E.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Functional Cycle of the ATPase Module of the 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1305</span>– <span class="NLM_lpage">1310</span>, <span class="refDoi"> DOI: 10.1073/pnas.1621129114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1073%2Fpnas.1621129114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=28115689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=1305-1310&issue=6&author=M.+Wehmerauthor=T.+Rudackauthor=F.+Beckauthor=A.+Aufderheideauthor=G.+Pfeiferauthor=J.+M.+Plitzkoauthor=F.+F%C3%B6rsterauthor=K.+Schultenauthor=W.+Baumeisterauthor=E.+Sakata&title=Structural+Insights+into+the+Functional+Cycle+of+the+ATPase+Module+of+the+26S+Proteasome&doi=10.1073%2Fpnas.1621129114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the functional cycle of the ATPase module of the 26S proteasome</span></div><div class="casAuthors">Wehmer, Marc; Rudack, Till; Beck, Florian; Aufderheide, Antje; Pfeifer, Guenter; Plitzko, Juergen M.; Foerster, Friedrich; Schulten, Klaus; Baumeister, Wolfgang; Sakata, Eri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1310</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In eukaryotic cells, the ubiquitin-proteasome system (UPS) is responsible for the regulated degrdn. of intracellular proteins.  The 26S holocomplex comprises the core particle (CP), where proteolysis takes place, and 1 or 2 regulatory particles (RPs).  The base of the RP is formed by a heterohexameric AAA+-ATPase module, which unfolds and translocates substrates into the CP.  Here, applying single-particle cryo-electron microscopy (cryo-EM) and image classification to samples in the presence of different nucleotides and nucleotide analogs, the authors were able to observe 4 distinct conformational states (s1-s4).  The resoln. of the 4 conformers allowed for the construction of at. models of the AAA+-ATPase module as it progresses through the functional cycle.  In a hitherto unobserved state (s4), the gate controlling access to the CP was open.  The structures described in this study allow the authors to put forward a model for the 26S functional cycle driven by ATP hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3MiQWEovbLrVg90H21EOLACvtfcHk0ljlO5o9Ar_psQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGjs78%253D&md5=9dfa75847017d5f973da07790229ca65</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1621129114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1621129114%26sid%3Dliteratum%253Aachs%26aulast%3DWehmer%26aufirst%3DM.%26aulast%3DRudack%26aufirst%3DT.%26aulast%3DBeck%26aufirst%3DF.%26aulast%3DAufderheide%26aufirst%3DA.%26aulast%3DPfeifer%26aufirst%3DG.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DF%25C3%25B6rster%26aufirst%3DF.%26aulast%3DSchulten%26aufirst%3DK.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DSakata%26aufirst%3DE.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Functional%2520Cycle%2520of%2520the%2520ATPase%2520Module%2520of%2520the%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D6%26spage%3D1305%26epage%3D1310%26doi%3D10.1073%2Fpnas.1621129114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span>; <span class="NLM_string-name">Osmulski, P. A.</span></span> <span> </span><span class="NLM_article-title">Atomic Force Microscopy of Proteasome Assemblies</span>. In  <i>Atomic Force Microscopy in Biomedical Research</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braga, P.</span>, <span class="NLM_string-name">Ricci, D.</span></span>, Eds.; Methods in Molecular Biology, Vol. 736; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-105-5_9</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1007%2F978-1-61779-105-5_9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=117-132&author=M.+Gaczynska&author=P.+A.+Osmulskiauthor=P.+Braga&author=D.+Ricci&title=Atomic+Force+Microscopy+in+Biomedical+Research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-105-5_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-61779-105-5_9%26sid%3Dliteratum%253Aachs%26aulast%3DGaczynska%26aufirst%3DM.%26atitle%3DAtomic%2520Force%2520Microscopy%2520of%2520Proteasome%2520Assemblies%26btitle%3DAtomic%2520Force%2520Microscopy%2520in%2520Biomedical%2520Research%26aulast%3DBraga%26aufirst%3DP.%26pub%3DHumana%2520Press%26date%3D2011%26spage%3D117%26epage%3D132%26doi%3D10.1007%2F978-1-61779-105-5_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljlO5o9Ar_psQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">IUCr. Molecular Replacement with <i>MOLREP</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&issue=1&author=A.+Vaginauthor=A.+Teplyakov&title=IUCr.+Molecular+Replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0ljlO5o9Ar_psQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DIUCr.%2520Molecular%2520Replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D1%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">IUCr. <i>REFMAC</i> 5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&issue=4&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=IUCr.+REFMAC+5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lheTWBdbGsJ5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DIUCr.%2520REFMAC%25205%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3D4%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&issue=Part+4&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lheTWBdbGsJ5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3DPart%25204%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">IUCr. <i>TLSMD</i> Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&issue=1&author=J.+Painterauthor=E.+A.+Merritt&title=IUCr.+TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0lheTWBdbGsJ5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DIUCr.%2520TLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26issue%3D1%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lib213-XbGDdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R Value: A Novel Statistical Quantity for Assessing the Accuracy of Crystal Structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span> (<span class="NLM_issue">6359</span>),  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&issue=6359&author=A.+T.+Br%C3%BCnger&title=Free+R+Value%3A+A+Novel+Statistical+Quantity+for+Assessing+the+Accuracy+of+Crystal+Structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value:  a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Bruenger, Axel T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A reliable and unbiased indicator of the accuracy is proposed for fitting at. models for macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCvKGOC-AKrVg90H21EOLACvtfcHk0lib213-XbGDdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D&md5=a382d593937b2394d9354ccba49cdbc3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520Value%253A%2520A%2520Novel%2520Statistical%2520Quantity%2520for%2520Assessing%2520the%2520Accuracy%2520of%2520Crystal%2520Structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26issue%3D6359%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&issue=1&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lib213-XbGDdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D1%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z','PDB','4X6Z'); return false;">PDB: 4X6Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RYP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RYP','PDB','1RYP'); return false;">PDB: 1RYP</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01025">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33480"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01025">10.1021/acs.jmedchem.8b01025</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">HPLC and HR MS analysis of the designed analogs, supplementary figures displaying the ChT-L, T-L, and C-L activity of human proteasome in the presence of the modulators, comparison of <b>7</b> and <b>7-4P</b> influence on the latent and SDS-activated h20S, CD spectra of the modulators, inhibitory capacity of <b>6</b> against yeast 20S proteasome, HPLC chromatogram confirming stability of <b>6</b> against proteasomal proteolytic activity, and crystallographic data statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01025/suppl_file/jm8b01025_si_001.pdf">jm8b01025_si_001.pdf (1.43 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for the yeast 20S proteasome in the complex with <b>6</b> modulator is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6Z">4X6Z</a>. The atomic coordinates and experimental data have been already released.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01025&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01025%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-1%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01025" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a55b39fa1d19b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
